{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "9939 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7ca577f7-c6ef-43c0-a2e6-d72a2d69238f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=bb9b11c1-0e5a-43fc-a144-60eaddaae78d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c445da7d-34be-47ca-b1c2-9f407e20f5da&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=71dda9bb-504b-41fc-b9ba-210bce0922d0&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=29e475fb-0af4-4fb3-8dc9-3ac163d7a2d5&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=32d52e03-e1e6-40e0-8e17-0bfa5159f8ec&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=668e3b80-5bd5-48f2-b1e3-ddc57dc2e407&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2ff4792f-5927-4d23-b429-658ec7dd8c9f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=dc7f65d3-b5e5-4e7c-ad0f-ee27e2a79d17&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-30&until-occurred-date=2021-10-07\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20159228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21261912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.16.20023754...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.455288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.20172288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.21251670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.09.373258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/135582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.455988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.424510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20124461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.23.353045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.12.20230292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.439272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.441398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21261000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.07.21249419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.380741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.437224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.07.20145979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.03.075069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.28.20029272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.442687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.434390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.261586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.20167205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.428136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.213132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.13.039917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.335083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.20248932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.21249563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462603...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.419044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275308.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462438...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.458031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.07.030650...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/571380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.444292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/715482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.461806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.460651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.434040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435343...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.422737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460445...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.02.232710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452462...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.21261496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.332023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.440978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.20142703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.383083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.430920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.437441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.432552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.442971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20223891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.18.20070417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20151423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/292581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.451001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.15.20065995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.458941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.460530...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462344...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462347...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448347...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.10.036418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462109...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.458711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.440156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.444019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264219...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.449152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.447990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.21261083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.438781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.451285...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/037762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.27.20220905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.04.20122481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.20222661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.21253183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.20.20108183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.093195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.435068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.14.150490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.431090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.431255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462529...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462263...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.13.20211359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.214148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.06.20147199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.25.169946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.29.20081174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21254000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/782557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.11.934471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.26.20044826...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.433940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.21249998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.415687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.425825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.22.427830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21249554...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.09.30.462488', 'tweets': 423, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Durability of immune responses to the BNT162b2 mRNA vaccine', 'authors': \"[{'name': 'Mehul S. Suthar'}, {'name': 'Prabhu S. Arunachalam'}, {'name': 'Mengyun Hu'}, {'name': 'Noah Reis'}, {'name': 'Meera Trisal'}, {'name': 'Olivia Raeber'}, {'name': 'Sharon Chinthrajah'}, {'name': 'Meredith E. Davis-Gardner'}, {'name': 'Kelly Manning'}, {'name': 'Prakriti Mudvari'}, {'name': 'Eli Boritz'}, {'name': 'Sucheta Godbole'}, {'name': 'Amy R. Henry'}, {'name': 'Daniel C. Douek'}, {'name': 'Peter Halfmann'}, {'name': 'Yoshihiro Kawaoka'}, {'name': 'Veronika I. Zarnitsyna'}, {'name': 'Kari Nadeau'}, {'name': 'Bali Pulendran'}]\", 'abstract': '<p>The development of the highly efficacious mRNA vaccines in less than a year since the emergence of SARS-CoV-2 represents a landmark in vaccinology. However, reports of waning vaccine efficacy, coupled with the emergence of variants of concern that are resistant to antibody neutralization, have raised concerns about the potential lack of durability of immunity to vaccination. We recently reported findings from a comprehensive analysis of innate and adaptive immune responses in 56 healthy volunteers who received two doses of the BNT162b2 vaccination. Here, we analyzed antibody responses to the homologous Wu strain as well as several variants of concern, including the emerging Mu (B.1.621) variant, and T cell responses in a subset of these volunteers at six months (day 210 post-primary vaccination) after the second dose. Our data demonstrate a substantial waning of antibody responses and T cell immunity to SARS-CoV-2 and its variants, at 6 months following the second immunization with the BNT162b2 vaccine. Notably, a significant proportion of vaccinees have neutralizing titers below the detection limit, and suggest a 3<sup>rd</sup> booster immunization might be warranted to enhance the antibody titers and T cell responses.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.21264325', 'tweets': 220, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Assessing the Burden of COVID-19 in Developing Countries: Systematic Review, Meta-Analysis, and Public Policy Implications', 'authors': \"[{'name': 'Andrew Levin'}, {'name': 'Nana Owusu-Boaitey'}, {'name': 'Sierra Pugh'}, {'name': 'Bailey K. Fosdick'}, {'name': 'Anthony B. Zwi'}, {'name': 'Anup Malani'}, {'name': 'Satej Soman'}, {'name': 'Lonni Besançon'}, {'name': 'Ilya Kashnitsky'}, {'name': 'Sachin Ganesh'}, {'name': 'Aloysius McLaughlin'}, {'name': 'Gayeong Song'}, {'name': 'Rine Uhm'}, {'name': 'Gideon Meyerowitz-Katz'}]\", 'abstract': '<sec><title>Introduction</title><p>The infection-fatality rate (IFR) of COVID-19 has been carefully measured and analyzed in high-income countries, whereas there has been no systematic analysis of age-specific seroprevalence or IFR for developing countries. Indeed, it has been suggested that the death rate in developing countries may be far lower than in high-income countries—an outcome that would be starkly different from the typical pattern for many other infectious diseases.</p></sec><sec><title>Methods</title><p>We systematically reviewed the literature to identify all serology studies in developing countries that were conducted using representative samples of specimens collected by early 2021. For each of the antibody assays used in these serology studies, we identified data on assay characteristics, including the extent of seroreversion over time. We analyzed the serology data using a Bayesian model that incorporates conventional sampling uncertainty as well as uncertainties about assay sensitivity and specificity. We then calculated IFRs using individual case reports or aggregated public health updates, including age-specific estimates whenever feasible.</p></sec><sec><title>Results</title><p>Seroprevalence in many developing country locations was markedly higher than in high-income countries but still far short of herd immunity. In most locations, seroprevalence among older adults was similar to that of younger age-groups. Age-specific IFRs were 1.3-2.5x higher than in high-income countries. The median value of population IFR was 0.5% among developing countries with satisfactory death reporting as of 2016, compared to a median of 0.05% for other developing countries.</p></sec><sec><title>Conclusion</title><p>The burden of COVID-19 is far higher in developing countries than in high-income countries, reflecting a combination of elevated transmission to middle-aged and older adults as well as limited access to adequate healthcare. These results underscore the critical need to accelerate the provision of vaccine doses to vulnerable populations in developing countries.</p></sec><sec><title>Key Points</title><list list-type=\"simple\"><list-item><label>‐</label><p>Age-specific prevalence and infection fatality rate (IFR) of COVID-19 for developing countries has not been well assessed.</p></list-item><list-item><label>‐</label><p>Seroprevalence in developing countries (as measured by antibodies against SARS-CoV-2) is markedly higher than in high-income countries but still far short of herd immunity.</p></list-item><list-item><label>‐</label><p>Seroprevalence among older adults is broadly similar to that of younger age-groups.</p></list-item><list-item><label>‐</label><p>Age-specific IFRs in developing countries are roughly twice those of high-income countries.</p></list-item><list-item><label>‐</label><p>Population IFR in developing countries with satisfactory death reporting (based on UN/WHO data as of 2016) is ten times higher than in other developing countries.</p></list-item><list-item><label>‐</label><p>These results underscore the urgency of disseminating vaccines to vulnerable people in developing countries.</p></list-item></list></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.21264363', 'tweets': 197, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'SARS-CoV-2 infections elicit higher levels of original antigenic sin antibodies compared to SARS-CoV-2 mRNA vaccinations', 'authors': \"[{'name': 'Elizabeth M. Anderson'}, {'name': 'Theresa Eilola'}, {'name': 'Eileen Goodwin'}, {'name': 'Marcus J. Bolton'}, {'name': 'Sigrid Gouma'}, {'name': 'Rishi R. Goel'}, {'name': 'Mark M. Painter'}, {'name': 'Sokratis A. Apostolidis'}, {'name': 'Divij Mathew'}, {'name': 'Debora Dunbar'}, {'name': 'Danielle Fiore'}, {'name': 'Amanda Brock'}, {'name': 'JoEllen Weaver'}, {'name': 'John S. Millar'}, {'name': 'Stephanie DerOhannessian'}, {'name': 'Allison R. Greenplate'}, {'name': 'Ian Frank'}, {'name': 'Daniel J. Rader'}, {'name': 'E. John Wherry'}, {'name': 'Scott E. Hensley'}, {'name': ' '}]\", 'abstract': '<p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) mRNA vaccines elicit higher levels of antibodies compared to natural SARS-CoV-2 infections in most individuals; however, the specificities of antibodies elicited by vaccination versus infection remain incompletely understood. Here, we characterized the magnitude and specificity of SARS-CoV-2 spike-reactive antibodies from 10 acutely infected health care workers and 23 participants who received mRNA-based SARS-CoV-2 vaccines. We found that infection and primary mRNA vaccination elicited S1 and S2-reactive antibodies, while secondary vaccination boosted mostly S1 antibodies. Using magnetic bead-based absorption assays, we found that SARS-CoV-2 infections elicited a large proportion of original antigenic sin-like antibodies that bound efficiently to common seasonal human coronaviruses but poorly to SARS-CoV-2. In converse, vaccination only modestly boosted antibodies reactive to common seasonal human coronaviruses and these antibodies bound efficiently to SARS-CoV-2. Our data indicate that SARS-CoV-2 mRNA vaccinations elicit fundamentally different antibody responses compared to SARS-CoV-2 infections.</p><sec><title>Abstract Figure</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"21264363v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec><sec><title>HIGHLIGHTS</title><list list-type=\"bullet\"><list-item><p>SARS-CoV-2 mRNA vaccines elicit higher levels of antibodies compared to SARS-CoV-2 infections</p></list-item><list-item><p>The first dose of an mRNA vaccine generates both S1 and S2 responses while the second dose boosts primarily S1-specific antibodies</p></list-item><list-item><p>SARS-CoV-2 infections, but not mRNA vaccinations, elicit high levels of antibodies that bind strongly to seasonal coronaviruses but weakly to SARS-CoV-2</p></list-item></list></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.21264272', 'tweets': 145, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Associations between SARS-CoV-2 variants and risk of COVID-19 hospitalization among confirmed cases in Washington State: a retrospective cohort study', 'authors': \"[{'name': 'Miguel I. Paredes'}, {'name': 'Stephanie M. Lunn'}, {'name': 'Michael Famulare'}, {'name': 'Lauren A. Frisbie'}, {'name': 'Ian Painter'}, {'name': 'Roy Burstein'}, {'name': 'Pavitra Roychoudhury'}, {'name': 'Hong Xie'}, {'name': 'Shah A. Mohamed Bakhash'}, {'name': 'Ricardo Perez'}, {'name': 'Maria Lukes'}, {'name': 'Sean Ellis'}, {'name': 'Saraswathi Sathees'}, {'name': 'Patrick Mathias'}, {'name': 'Alexander Greninger'}, {'name': 'Lea M. Starita'}, {'name': 'Chris D. Frazar'}, {'name': 'Erica Ryke'}, {'name': 'Weizhi Zhong'}, {'name': 'Luis Gamboa'}, {'name': 'Machiko Threlkeld'}, {'name': 'Jover Lee'}, {'name': 'Deborah A. Nickerson'}, {'name': 'Daniel L. Bates'}, {'name': 'Matthew E. Hartman'}, {'name': 'Eric Haugen'}, {'name': 'Truong N. Nguyen'}, {'name': 'Joshua D. Richards'}, {'name': 'Jacob L. Rodriguez'}, {'name': 'John A. Stamatoyannopoulos'}, {'name': 'Eric Thorland'}, {'name': 'Geoff Melly'}, {'name': 'Philip E. Dykema'}, {'name': 'Drew C. MacKellar'}, {'name': 'Hannah K. Gray'}, {'name': 'Avi Singh'}, {'name': 'JohnAric MoonDance Peterson'}, {'name': 'Denny Russell'}, {'name': 'Laura Marcela Torres'}, {'name': 'Scott Lindquist'}, {'name': 'Trevor Bedford'}, {'name': 'Krisandra J. Allen'}, {'name': 'Hanna N. Oltean'}]\", 'abstract': '<sec><title>Background</title><p>The COVID-19 pandemic is now dominated by variant lineages; the resulting impact on disease severity remains unclear. Using a retrospective cohort study, we assessed the risk of hospitalization following infection with nine variants of concern or interest (VOC/VOI).</p></sec><sec><title>Methods</title><p>Our study includes individuals with positive SARS-CoV-2 RT-PCR in the Washington Disease Reporting System and with available viral genome data, from December 1, 2020 to July 30, 2021. The main analysis was restricted to cases with specimens collected through sentinel surveillance. Using a Cox proportional hazards model with mixed effects, we estimated hazard ratios (HR) for the risk of hospitalization following infection with a VOC/VOI, adjusting for age, sex, and vaccination status.</p></sec><sec><title>Findings</title><p>Of the 27,814 cases, 23,170 (83.3%) were sequenced through sentinel surveillance, of which 726 (3.1%) were hospitalized due to COVID-19. Higher hospitalization risk was found for infections with Gamma (HR 3.17, 95% CI 2.15-4.67), Beta (HR: 2.97, 95% CI 1.65–5.35), Delta (HR: 2.30, 95% CI 1.69-3.15), and Alpha (HR 1.59, 95% CI 1.26–1.99) compared to infections with an ancestral lineage. Following VOC infection, unvaccinated patients show a similar higher hospitalization risk, while vaccinated patients show no significant difference in risk, both when compared to unvaccinated, ancestral lineage cases.</p></sec><sec><title>Interpretation</title><p>Infection with a VOC results in a higher hospitalization risk, with an active vaccination attenuating that risk. Our findings support promoting hospital preparedness, vaccination, and robust genomic surveillance.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462148', 'tweets': 83, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Adeno-associated virus (AAV) reduces cortical dendritic complexity in a TLR9-dependent manner', 'authors': \"[{'name': 'Christos M. Suriano'}, {'name': 'Jessica L. Verpeut'}, {'name': 'Neerav Kumar'}, {'name': 'Jie Ma'}, {'name': 'Caroline Jung'}, {'name': 'Lisa M. Boulanger'}]\", 'abstract': '<p>Recombinant adeno-associated viruses (AAVs) allow rapid and efficient gene delivery in the nervous system. AAVs are widely used in research and are the basis of multiple FDA-approved gene therapies. Here, we find that the immune response to AAV’s genome reduces dendritic complexity in mammalian cortex. Dendritic loss associated with AAV-mediated gene delivery occurs at experimentally-relevant titers, cannot be explained by responses to transgene expression or surgery, and is not restricted to a particular capsid serotype, encoded transgene, promoter, or production facility. AAV-associated dendritic loss is accompanied by a decrease in the frequency and amplitude of miniature excitatory postsynaptic currents (mEPSCs) and upregulation of immune molecules that can limit dendritic complexity and synaptic transmission. Blocking detection of unmethylated CpG-rich DNA via Toll-like receptor 9 (TLR9) protects dendritic complexity, suggesting that immunodetection of a core feature of the AAV genome triggers dendritic loss. These results reveal previously unsuspected impacts of AAV on neuronal structure and function and identify TLR9 inhibitors as important tools to improve the safety and efficacy of AAV-mediated gene delivery in the nervous system.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462231', 'tweets': 75, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structures of core eukaryotic protein complexes', 'authors': \"[{'name': 'Ian R. Humphreys'}, {'name': 'Jimin Pei'}, {'name': 'Minkyung Baek'}, {'name': 'Aditya Krishnakumar'}, {'name': 'Ivan Anishchenko'}, {'name': 'Sergey Ovchinnikov'}, {'name': 'Jing Zhang'}, {'name': 'Travis J. Ness'}, {'name': 'Sudeep Banjade'}, {'name': 'Saket Bagde'}, {'name': 'Viktoriya G. Stancheva'}, {'name': 'Xiao-Han Li'}, {'name': 'Kaixian Liu'}, {'name': 'Zhi Zheng'}, {'name': 'Daniel J. Barrero'}, {'name': 'Upasana Roy'}, {'name': 'Israel S. Fernández'}, {'name': 'Barnabas Szakal'}, {'name': 'Dana Branzei'}, {'name': 'Eric C. Greene'}, {'name': 'Sue Biggins'}, {'name': 'Scott Keeney'}, {'name': 'Elizabeth A. Miller'}, {'name': 'J. Christopher Fromme'}, {'name': 'Tamara L. Hendrickson'}, {'name': 'Qian Cong'}, {'name': 'David Baker'}]\", 'abstract': '<p>Protein-protein interactions play critical roles in biology, but despite decades of effort, the structures of many eukaryotic protein complexes are unknown, and there are likely many interactions that have not yet been identified. Here, we take advantage of recent advances in proteome-wide amino acid coevolution analysis and deep-learning-based structure modeling to systematically identify and build accurate models of core eukaryotic protein complexes, as represented within the <italic>Saccharomyces cerevisiae</italic> proteome. We use a combination of RoseTTAFold and AlphaFold to screen through paired multiple sequence alignments for 8.3 million pairs of <italic>S. cerevisiae</italic> proteins and build models for strongly predicted protein assemblies with two to five components. Comparison to existing interaction and structural data suggests that these predictions are likely to be quite accurate. We provide structure models spanning almost all key processes in Eukaryotic cells for 104 protein assemblies which have not been previously identified, and 608 which have not been structurally characterized.</p><sec><title>One-sentence summary</title><p>We take advantage of recent advances in proteome-wide amino acid coevolution analysis and deep-learning-based structure modeling to systematically identify and build accurate models of core eukaryotic protein complexes.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.21264376', 'tweets': 42, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Estimates of pandemic excess mortality in India based on civil registration data', 'authors': \"[{'name': 'Murad Banaji'}, {'name': 'Aashish Gupta'}]\", 'abstract': '<sec><title>Background</title><p>The COVID-19 pandemic has had large impacts on population health. These impacts are less well understood in low-and middle-income countries, where mortality surveillance before the pandemic was patchy. Although limited all-cause mortality data are available in India, interpreting this data remains a challenge.</p></sec><sec><title>Objective</title><p>We use existing data on all-cause mortality from civil registration systems of twelve Indian states comprising around 60% of the national population to understand the scale and timing of excess deaths in India during the COVID-19 pandemic.</p></sec><sec><title>Methods</title><p>We characterize the available data, discuss the various reasons why these data are incomplete, and estimate the extent of coverage in the data. Comparing the pandemic period to 2019, we estimate excess mortality in twelve Indian states, and extrapolate our estimates to the rest of India. We explore sensitivity of the estimates to various assumptions, and present optimistic and pessimistic scenarios along with our central estimates.</p></sec><sec><title>Results</title><p>For the 12 states with available all-cause mortality data, we document an increase of 28% in deaths during April 2020-May 2021 relative to expectations from 2019. This level of increase in mortality, if it applies nationally, would imply 2.8-2.9 million excess deaths. More limited data from June 2021 increases national estimates of excess deaths during April 2020-June 2021 to 3.8 million. With more optimistic or pessimistic assumptions, excess deaths during this period could credibly lie between 2.8 million and 5.2 million. We find that the scale of estimated excess deaths is broadly consistent with expectations based on seroprevalence data and international data on COVID-19 fatality rates. Moreover, there is a strong association between the timing of excess deaths, and of recorded COVID-19 deaths.</p></sec><sec><title>Contribution</title><p>We show that the surveillance of pandemic mortality in India has been extremely poor, with around 8-10 times as many excess deaths as officially recorded COVID-19 deaths. Our findings highlight the utility of all-cause mortality data, as well as the significant challenges in interpreting such data from LMICs. These data reveal that India is among the countries most severely impacted by the pandemic. It is likely that in absolute terms India has seen the highest number of pandemic excess deaths of any country in the world.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.21263801', 'tweets': 30, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Analysis of overdispersion in airborne transmission of Covid-19', 'authors': \"[{'name': 'Swetaprovo Chaudhuri'}, {'name': 'Prasad Kasibhatla'}, {'name': 'Arnab Mukherjee'}, {'name': 'William Pan'}, {'name': 'Glenn Morrison'}, {'name': 'Sharmistha Mishra'}, {'name': 'Vijaya Kumar Murty'}]\", 'abstract': '<p>Superspreading events and overdispersion are hallmarks of the Covid-19 pandemic. To gain insight into the nature and controlling factors of these superspreading events and heterogeneity in transmission, we conducted mechanistic modeling of SARS-CoV-2 transmission by infectious aerosols using real-world occupancy data from a large number of full-service restaurants in ten US metropolises. Including a large number of factors that influence disease transmission in these settings, we demonstrate the emergence of a stretched tail in the probability density function of secondary infection numbers indicating strong heterogeneity in individual infectivity. Derived analytical results further demonstrate that variability in viral loads and variability in occupancy, together, lead to overdispersion in the number of secondary infections arising from individual index cases. Our analysis, connecting mechanistic understanding of SARS-CoV-2 transmission by aerosols with observed large-scale epidemiological characteristics of Covid-19 outbreaks, adds an important dimension to the mounting body of evidence with regards to the determinants of airborne transmission of SARS-CoV-2 by aerosols in indoor settings.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462538', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'New tools for automated cryo-EM single-particle analysis in RELION-4.0', 'authors': \"[{'name': 'Dari Kimanius'}, {'name': 'Liyi Dong'}, {'name': 'Grigory Sharov'}, {'name': 'Takanori Nakane'}, {'name': 'Sjors H.W. Scheres'}]\", 'abstract': '<label>1</label><title>Abstract</title><p>We describe new tools for the processing of electron cryo-microscopy (cryo-EM) images in the fourth major release of the RELION software. In particular, we introduce VDAM, a Variable-metric gradient Descent algorithm with Adaptive Moments estimation, for image refinement; a convolutional neural network for unsupervised selection of 2D classes; and a flexible framework for the design and execution of multiple jobs in pre-defined workflows. In addition, we present a stand-alone utility called MDCatch that links the execution of jobs within this framework with metadata gathering during microscope data acquisition. The new tools are aimed at providing fast and robust procedures for unsupervised cryo-EM structure determination, with potential applications for on-the-fly processing and the development of flexible, high-throughput structure determination pipelines. We illustrate their potential on twelve publicly available cryo-EM data sets.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462505', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'S. pombe wtf genes use dual transcriptional regulation and selective protein exclusion from spores to cause meiotic drive', 'authors': \"[{'name': 'Nicole L. Nuckolls'}, {'name': 'Ananya Nidamangala Srinivasa'}, {'name': 'Anthony C. Mok'}, {'name': 'María Angélica Bravo Núñez'}, {'name': 'Jeffrey J. Lange'}, {'name': 'Todd J. Gallagher'}, {'name': 'Chris W. Seidel'}, {'name': 'Sarah E. Zanders'}]\", 'abstract': '<p>Meiotic drivers bias gametogenesis to ensure their transmission into more than half the offspring of a heterozygote. In <italic>Schizosaccharomyces pombe</italic>, <italic>wtf</italic> meiotic drivers destroy the meiotic products (spores) that do not inherit the driver from a heterozygote, thereby reducing fertility. <italic>wtf</italic> drivers encode both a Wtf<sup>poison</sup> protein and a Wtf<sup>antidote</sup> protein using alternative transcriptional start sites. Here, we analyze how the expression and localization of the Wtf proteins are regulated to achieve drive. We show that transcriptional timing and selective protein exclusion from developing spores ensure that all spores are exposed to Wtf4<sup>poison</sup>, but only the spores that inherit <italic>wtf4</italic> receive a dose of Wtf4<sup>antidote</sup> sufficient for survival. In addition, we show that the Mei4 transcription factor, a master regulator of meiosis, controls the expression of the <italic>wtf4<sup>poison</sup></italic> transcript. This dual transcriptional regulation, which includes the use of a critical meiotic transcription factor, likely complicates the universal suppression of <italic>wtf</italic> genes without concomitantly disrupting spore viability. We propose that these features contribute to the evolutionary success of the <italic>wtf</italic> drivers.</p><sec><title>Author Summary</title><p>Killer meiotic drivers are one type of selfish DNA sequence. When only one copy of a killer meiotic driver is found in a genome, the driver is expected to be transmitted to only half of the gametes (e.g. eggs or sperm). Killer meiotic drivers, however, kill developing gametes that do not inherit them, giving the driver a transmission advantage into the next generation. The molecular mechanisms used by these killers are not well understood. In this work, we analyzed how one killer meiotic driver, <italic>wtf4</italic> from fission yeast, ensures targeted gamete (spore) killing. Previous work showed that <italic>wtf</italic> meiotic drivers encode a poison protein that is transmitted to all spores and an antidote protein that rescues only spores that inherit the locus. Here, we show that different timing of the expression of the two proteins, combined with differential inclusion of the proteins in developing spores, both contribute to targeted spore killing. We also demonstrate that <italic>wtf4</italic> exploits an essential gene expression pathway, making it difficult for the genome to prevent this locus from being expressed and killing. This extends our knowledge both of how these genetic parasites act and how they are equipped to evade host suppression mechanisms.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462449', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': True, 'title': 'Pyronaridine Protects Against SARS-CoV-2 in Mouse', 'authors': \"[{'name': 'Ana C. Puhl'}, {'name': 'Giovanni F. Gomes'}, {'name': 'Samara Damasceno'}, {'name': 'Andre S. Godoy'}, {'name': 'Gabriela D. Noske'}, {'name': 'Aline M. Nakamura'}, {'name': 'Victor O. Gawriljuk'}, {'name': 'Rafaela S. Fernandes'}, {'name': 'Natalia Monakhova'}, {'name': 'Olga Riabova'}, {'name': 'Thomas R. Lane'}, {'name': 'Vadim Makarov'}, {'name': 'Flavio P. Veras'}, {'name': 'Sabrina S. Batah'}, {'name': 'Alexandre T. Fabro'}, {'name': 'Glaucius Oliva'}, {'name': 'Fernando Q. Cunha'}, {'name': 'José C. Alves-Filho'}, {'name': 'Thiago M. Cunha'}, {'name': 'Sean Ekins'}]\", 'abstract': '<p>There are currently relatively few small-molecule antiviral drugs that are either approved or emergency approved for use against SARS-CoV-2. One of these is remdesivir, which was originally repurposed from its use against Ebola and functions by causing early RNA chain termination. We used this as justification to evaluate three molecules we had previously identified computationally with antiviral activity against Ebola and Marburg. Out of these we previously identified pyronaridine, which inhibited the SARS-CoV-2 replication in A549-ACE2 cells. Herein, the <italic>in vivo</italic> efficacy of pyronaridine has now been assessed in a K18-hACE transgenic mouse model of COVID-19. Pyronaridine treatment demonstrated a statistically significant reduction of viral load in the lungs of SARS CoV-2 infected mice. Furthermore, the pyronaridine treated group reduced lung pathology, which was also associated with significant reduction in the levels of pro-inflammatory cytokines/chemokine and cell infiltration. Notably, pyronaridine inhibited the viral PL<sup>pro</sup> activity <italic>in vitro</italic> (IC<sub>50</sub> of 1.8 µM) without any effect on M<sup>pro</sup>, indicating a possible molecular mechanism involved in its ability to inhibit SARS-CoV-2 replication. Interestingly, pyronaridine also selectively inhibits the host kinase CAMK1 (IC<sub>50</sub> of 2.4 µM). We have also generated several pyronaridine analogs to assist in understanding the structure activity relationship for PL<sup>pro</sup> inhibition. Our results indicate that pyronaridine is a potential therapeutic candidate for COVID-19.</p><sec><title>One sentence summary</title><p>There is currently intense interest in discovering small molecules with direct antiviral activity against the severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2). Pyronaridine, an antiviral drug with in vitro activity against Ebola, Marburg and SARS-CoV-2 has now statistically significantly reduced the viral load in mice along with IL-6, TNF-α, and IFN-β ultimately demonstrating a protective effect against lung damage by infection to provide a new potential treatment for testing clinically.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.21264206', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Enhancing the brain’s emotion regulation capacity with a randomised trial of a 5-week heart rate variability biofeedback intervention', 'authors': \"[{'name': 'Kaoru Nashiro'}, {'name': 'Jungwon Min'}, {'name': 'Hyun Joo Yoo'}, {'name': 'Christine Cho'}, {'name': 'Shelby L. Bachman'}, {'name': 'Shubir Dutt'}, {'name': 'Julian F. Thayer'}, {'name': 'Paul Lehrer'}, {'name': 'Tiantian Feng'}, {'name': 'Noah Mercer'}, {'name': 'Padideh Nasseri'}, {'name': 'Diana Wang'}, {'name': 'Catie Chang'}, {'name': 'Vasilis Z. Marmarelis'}, {'name': 'Shri Narayanan'}, {'name': 'Daniel A. Nation'}, {'name': 'Mara Mather'}]\", 'abstract': '<p>Heart rate variability is a robust biomarker of emotional well-being, consistent with the shared brain networks regulating emotion regulation and heart rate. While high heart rate oscillatory activity clearly indicates healthy regulatory brain systems, can increasing this oscillatory activity also enhance brain function? To test this possibility, we randomly assigned 106 young adult participants to one of two 5-week interventions involving daily biofeedback that either increased heart rate oscillations (Osc+ condition) or had little effect on heart rate oscillations (Osc- condition) and examined effects on brain activity during rest and during regulating emotion. In this healthy cohort, the two conditions did not differentially affect anxiety, depression or mood. However, the Osc+ intervention increased low-frequency heart rate variability and increased brain oscillatory dynamics and functional connectivity in emotion-related resting-state networks. It also increased down-regulation of activity in somatosensory brain regions during an emotion regulation task. The Osc- intervention did not have these effects. These findings indicate that heart rate oscillatory activity not only reflects the current state of regulatory brain systems but also changes how the brain operates beyond the moments of high oscillatory activity.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462068', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Here’s the twist: How the brain updates naturalistic event memories as our understanding of the past changes', 'authors': \"[{'name': 'Asieh Zadbood'}, {'name': 'Samuel A. Nastase'}, {'name': 'Janice Chen'}, {'name': 'Kenneth A. Norman'}, {'name': 'Uri Hasson'}]\", 'abstract': '<p>The brain actively reshapes past memories in light of new incoming information. In the current study, we ask how the brain supports this updatinge process during the encoding and recall of naturalistic stimuli. One group of participants watched a movie (“The Sixth Sense”) with a cinematic “twist” at the end that dramatically changed the interpretation of previous events. Next, participants were asked to verbally recall the movie events, taking into account the new “twist” information. Most participants updated their recall to incorporate the twist. Two additional groups recalled the movie without having to update their memories during recall: one group never saw the twist; another group was exposed to the twist prior to the beginning of the movie, and thus the twist information was incorporated both during encoding and recall. We found that providing participants with information about the twist beforehand altered neural response patterns during movie-viewing in the default mode network (DMN). Moreover, presenting participants with the twist at the end of the movie changed the neural representation of the previously-encoded information during recall in a subset of DMN regions. Further evidence for this transformation was obtained by comparing the neural activation patterns during encoding and recall and correlating them with behavioral signatures of memory updating. Our results demonstrate that neural representations of past events encoded in the DMN are dynamically integrated with new information that reshapes our memory in natural contexts.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462199', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Cell-ACDC: a user-friendly toolset embedding state-of-the-art neural networks for segmentation, tracking and cell cycle annotations of live-cell imaging data', 'authors': \"[{'name': 'Francesco Padovani'}, {'name': 'Benedikt Mairhörmann'}, {'name': 'Pascal Falter-Braun'}, {'name': 'Jette Lengefeld'}, {'name': 'Kurt M. Schmoller'}]\", 'abstract': '<p>Live-cell imaging is a powerful tool to study dynamic cellular processes on the level of single cells with quantitative detail. Microfluidics enables parallel high-throughput imaging, creating a downstream bottleneck at the stage of data analysis. Recent progress on deep learning image analysis dramatically improved cell segmentation and tracking. Nevertheless, manual data validation and correction is typically still required and broadly used tools spanning the complete range of live-cell imaging analysis, from cell segmentation to pedigree analysis and signal quantification, are still needed. Here, we present Cell-ACDC, a user-friendly graphical user-interface (GUI)-based framework written in Python, for segmentation, tracking and cell cycle annotation. We included two state-of-the-art and high-accuracy deep learning models for single-cell segmentation of yeast and mammalian cells implemented in the most used deep learning frameworks TensorFlow and PyTorch. Additionally, we developed and implemented a cell tracking method and embedded it into an intuitive, semi-automated workflow for label-free cell cycle annotation of single cells. The open-source and modularized nature of Cell-ACDC will enable simple and fast integration of new deep learning-based and traditional methods for cell segmentation or downstream image analysis.</p><sec><title>Source code</title><p><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/SchmollerLab/Cell_ACDC\">https://github.com/SchmollerLab/Cell_ACDC</ext-link></p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462010', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Drivers and Determinants of Strain Dynamics Following Faecal Microbiota Transplantation', 'authors': \"[{'name': 'Thomas SB Schmidt'}, {'name': 'Simone S Li'}, {'name': 'Oleksandr M Maistrenko'}, {'name': 'Wasiu Akanni'}, {'name': 'Luis Pedro Coelho'}, {'name': 'Sibasish Dolai'}, {'name': 'Anthony Fullam'}, {'name': 'Anna M Glazek'}, {'name': 'Rajna Hercog'}, {'name': 'Hilde Herrema'}, {'name': 'Ferris Jung'}, {'name': 'Stefanie Kandels'}, {'name': 'Askarbek Orakov'}, {'name': 'Thea Van Rossum'}, {'name': 'Vladimir Benes'}, {'name': 'Thomas J Borody'}, {'name': 'Willem M de Vos'}, {'name': 'Cyriel Y Ponsioen'}, {'name': 'Max Nieuwdorp'}, {'name': 'Peer Bork'}]\", 'abstract': '<p>Faecal microbiota transplantation (FMT) is an efficacious therapeutic intervention, but its clinical mode of action and underlying microbiome dynamics remain poorly understood. Here, we analysed the metagenomes associated with 142 FMTs, in a time series-based meta-study across five disease indications. We quantified strain-level dynamics of 1,089 microbial species based on their pangenome, complemented with 47,548 newly constructed metagenome-assembled genomes. Using subsets of procedural-, host- and microbiome-based variables, LASSO-regularised regression models accurately predicted the colonisation and resilience of donor and recipient microbes, as well as turnover of individual species. Linking this to putative ecological mechanisms, we found these sets of variables to be informative of the underlying processes that shape the post-FMT gut microbiome. Recipient factors and complementarity of donor and recipient microbiomes, encompassing entire communities to individual strains, were the main determinants of individual strain population dynamics, and mostly independent of clinical outcomes. Recipient community state and the degree of residual strain depletion provided a neutral baseline for donor strain colonisation success, in addition to inhibitive priority effects between species and conspecific strains, as well as putatively adaptive processes. Our results suggest promising tunable parameters to enhance donor flora colonisation or recipient flora displacement in clinical practice, towards the development of more targeted and personalised therapies.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.21264208', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Integration of genetic colocalizations with physiological and pharmacological perturbations identifies cardiometabolic disease genes', 'authors': \"[{'name': 'Michael J. Gloudemans'}, {'name': 'Brunilda Balliu'}, {'name': 'Daniel Nachun'}, {'name': 'Matthew G. Durrant'}, {'name': 'Erik Ingelsson'}, {'name': 'Martin Wabitsch'}, {'name': 'Thomas Quertermous'}, {'name': 'Stephen B. Montgomery'}, {'name': 'Joshua W. Knowles'}, {'name': 'Ivan Carcamo-Orive'}]\", 'abstract': '<sec><title>Background</title><p>Identification of causal genes for polygenic human diseases has been extremely challenging, and our understanding of how physiological and pharmacological stimuli modulate genetic risk at disease-associated loci is limited. Specifically, insulin resistance (IR), a common feature of cardiometabolic disease, including type 2 diabetes, obesity, and dyslipidemia, lacks well-powered GWAS, and therefore few associated loci and causal genes have been identified.</p></sec><sec><title>Results</title><p>Here, we perform and integrate LD-adjusted colocalization analyses across nine cardiometabolic traits combined with eQTLs and sQTLs from five metabolically relevant human tissues (subcutaneous and visceral adipose, skeletal muscle, liver, and pancreas). We identify 470 colocalized loci and prioritize 207 loci with a single colocalized gene. To elucidate upstream regulators and functional mechanisms for these genes, we integrate their transcriptional responses to 21 physiological and pharmacological cardiometabolic regulators in human adipocytes, hepatocytes, and skeletal muscle cells, and map their protein-protein interactions.</p></sec><sec><title>Conclusions</title><p>Our use of transcriptional responses under metabolic perturbations to contextualize genetic associations from our state-of-the-art colocalization approach provides a list of likely causal genes and their upstream regulators in the context of IR-associated cardiometabolic risk.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462548', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Prime Editing Efficiency and Fidelity are Enhanced in the Absence of Mismatch Repair', 'authors': \"[{'name': 'J Ferreira da Silva'}, {'name': 'GP Oliveira'}, {'name': 'EA Arasa-Verge'}, {'name': 'A Moretton'}, {'name': 'G Timelthaler'}, {'name': 'J Jiricny'}, {'name': 'JI Loizou'}]\", 'abstract': '<p>Prime editing is a powerful genome engineering approach that enables the introduction of base substitutions, insertions and deletions, into any given genomic locus. But prime editing, at even the same locus, can exhibit wildly different efficiencies in various cell backgrounds. It is unclear what determines these variations in efficiencies in a given cellular context. Through a focused genetic screen targeting DNA repair factors, we show that the efficiency of prime editing is attenuated by the mismatch repair pathway. The accumulation of the mismatch repair protein MLH1 at sites of prime editing, indicates that mismatch repair acts at these regions to directly counteract the insertion of the edit. Consequently, ablation of mismatch repair yields an up to 17-fold increase in prime editing efficiency across different human cell lines, several types of edits and multiple genomic loci. Our results shed new light on the cellular requirements for prime editing and identify that ablation of mismatch repair increases editing efficiency and fidelity.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.10.01.462730', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Deep mutational scan of a drug efflux pump reveals its structure-function landscape', 'authors': \"[{'name': 'Gianmarco Meier'}, {'name': 'Sujani Thavarasah'}, {'name': 'Kai Ehrenbolger'}, {'name': 'Cedric A. J. Hutter'}, {'name': 'Lea M. Hürlimann'}, {'name': 'Jonas Barandun'}, {'name': 'Markus A. Seeger'}]\", 'abstract': '<p>Drug efflux is a common resistance mechanism found in bacteria and cancer cells. Although several structures of drug efflux pumps are available, they provide only limited functional information on the phenomenon of drug efflux. Here, we performed deep mutational scanning (DMS) on the bacterial ATP binding cassette (ABC) transporter EfrCD to determine the drug efflux activity profile of more than 1500 single variants. These systematic measurements revealed that the introduction of negative charges at different locations within the large substrate binding pocket results in strongly increased efflux activity towards positively charged ethidium, while additional aromatic residues did not display the same effect. Data analysis in the context of an inward-facing cryo-EM structure of EfrCD uncovered a high affinity binding site, which releases bound drugs through a peristaltic transport mechanism as the transporter transits to its outward-facing conformation. Finally, we identified substitutions resulting in rapid Hoechst influx without affecting the efflux activity for ethidium and daunorubicin. Hence, single mutations can convert the ABC exporter EfrCD into a drug-specific ABC importer.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462379', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Modeling spatial, developmental, physiological, and topological constraints on human brain connectivity', 'authors': \"[{'name': 'S. Oldham'}, {'name': 'B. D. Fulcher'}, {'name': 'K. Aquino'}, {'name': 'A. Arnatkevičiūtė'}, {'name': 'C. Paquola'}, {'name': 'R. Shishegar'}, {'name': 'A. Fornito'}]\", 'abstract': '<p>The complex connectivity of nervous systems is thought to have been shaped by competitive selection pressures to minimize wiring costs and support adaptive function. Accordingly, recent modeling work indicates that stochastic processes, shaped by putative trade-offs between the cost and value of each connection, can successfully reproduce many topological properties of macroscale human connectomes measured with diffusion magnetic resonance imaging. Here, we derive a new formalism with the aim to more accurately capture the competing pressures of wiring cost minimization and topological complexity. We further show that model performance can be improved by accounting for developmental changes in brain geometry and associated wiring costs, and by using inter-regional transcriptional or microstructural similarity rather than topological wiring-rules. However, all models struggled to capture topologies spatial embedding. Our findings highlight an important role for genetics in shaping macroscale brain connectivity and indicate that stochastic models offer an incomplete account of connectome organization.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462485', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Repurposing CAR targets in solid tumors via a pan-cancer single-cell transcriptome analysis', 'authors': \"[{'name': 'Sanna Madan'}, {'name': 'Alejandro A. Schäffer'}, {'name': 'Eytan Ruppin'}]\", 'abstract': '<p>Chimeric antigen receptor (CAR) T cell therapy is a powerful and promising tool for unleashing lasting antitumor immunity, which is currently studied in numerous blood and solid cancers. Here, mining single cell transcriptomics datasets of solid tumors, we survey many existing CAR targets, aiming to new solid tumor types in which these targets are differentially expressed in tumor cells and not on non-tumor cells within the tumor microenvironment, thus providing new repurposing opportunities. Quite surprisingly, we find that many existing solid cancer CAR targets have a better discriminatory power in uncharted cancer types than in the ones they have been developed or being tested for. Furthermore, we show that several existing liquid cancer CAR targets may also hold promise in specific solid tumors. Taken together, this analysis uncovers a large potential of repurposing current CARs that is currently yet untapped.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.462132', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'An anti-inflammatory activation sequence governs macrophage transcriptional dynamics during tissue injury', 'authors': \"[{'name': 'Nicolas Denans'}, {'name': 'Nhung T. T. Tran'}, {'name': 'Madeleine E. Swall'}, {'name': 'Daniel C. Diaz'}, {'name': 'Jillian Blanck'}, {'name': 'Tatjana Piotrowski'}]\", 'abstract': '<p>Macrophages are essential for tissue repair and regeneration. Yet, the molecular programs, as well as the timing of their activation during and after tissue injury are poorly defined. Using a high spatio-temporal resolution single cell analysis of macrophages coupled with live imaging after sensory hair cell death in zebrafish, we find that the same population of macrophages transitions through a sequence of three major anti-inflammatory activation states. Macrophages first show a signature of glucocorticoid activation, then IL10 signaling and finally the induction of oxidative phosphorylation by IL4/Polyamine signaling. Importantly, loss-of-function of glucocorticoid and IL10 signaling shows that each step of the sequence is independently activated. Our results provide the first evidence that macrophages, in addition to a switch from M1 to M2, sequentially and independently transition though three anti-inflammatory pathways <italic>in vivo</italic> during tissue injury in a regenerating organ.</p><sec><title>One-Sentence Summary</title><p>We show that macrophages are sequentially activated by three different anti-inflammatory pathways during tissue injury.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462303', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Emerging mosquito resistance to piperonyl butoxide-synergized pyrethroid insecticide and its mechanism', 'authors': \"[{'name': 'Guofa Zhou'}, {'name': 'Yiji Li'}, {'name': 'Brook Jeang'}, {'name': 'Xiaoming Wang'}, {'name': 'Daibin Zhong'}, {'name': 'Guiyun Yan'}]\", 'abstract': '<p>Piperonyl butoxide (PBO)-synergized pyrethroid products are widely available for the control of pyrethroid-resistant mosquitoes. To date, no study has formally examined mosquito resistance to PBO-synergized insecticides. We used <italic>Culex quinquefasciatus</italic> as a model mosquito examined the insecticide resistance mechanisms of mosquitoes to PBO-synergized pyrethroid using modified World Health Organization tube bioassays and biochemical analysis of metabolic enzyme expressions prior- and post-PBO exposure. We measured mosquito mortalities and metabolic enzyme expressions in mosquitoes with/without pre-exposure to different PBO concentrations and exposure durations. We found that field <italic>Culex quinquefasciatus</italic> mosquitoes were resistant to all insecticides tested, including PBO-synergized pyrethroids (mortality ranged from 3.7±4.7% to 66.7±7.7%), except malathion. Field mosquitoes had elevated levels of carboxylesterase (COE, 3.8-fold) and monooxygenase (P450, 2.1-fold) but not glutathione S-transferase (GST) compared to susceptible mosquitoes. When the field mosquitoes were pre-exposed to 4% PBO, the 50% lethal concentration of deltamethrin was reduced from 0.22% to 0.10%, compare to 0.02% for susceptible mosquitoes. Knockdown resistance gene mutation (L1014F) rate was 62% in field mosquitoes. PBO pre-exposure suppressed P450 enzyme expression levels by 25∼34%, GST by 11%, and had no impact on COE enzyme expression. Even with the optimal PBO concentration and exposure duration, field mosquitoes had significantly higher P450 enzyme expression levels after PBO exposure compared to laboratory controls. These results demonstrate that PBO alone may not be enough to control highly pyrethroid resistant mosquitoes due to the multiple resistance mechanisms. Mosquito resistance to PBO-synergized insecticide should be closely monitored.</p><sec><title>Authors’ Summary</title><p>Mosquitoes are vectors of many major infectious diseases globally. Insecticides and related products are widely used for mosquito controls and disease preventions. Over time and following repeated use, mosquitoes (including <italic>Aedes, Anopheles</italic> and <italic>Culex</italic>) have developed very high resistance to multiple insecticides all over the world. Target site insensitivity due to mutations in the voltage-gated sodium channel gene and overproduction of metabolic detoxification enzymes such as cytochrome P450 (CYP) monooxygenases play critical role in insecticide resistance in mosquitoes. To enhance the killing power of insecticides, synergized insecticides were developed by mixing insecticide synergists with pyrethroids. Discovered in the 1940s, piperonyl butoxide (PBO) is one of the earliest and most commonly used insecticide synergists. Field application of PBO-synergized insecticides performed far better than mono-pyrethroids. PBO-treated long-lasting insecticidal nets (PBO-LLINs), which also use pyrethroids, outperformed regular LLIN for malaria control in many African countries. PBO-LLIN is soon to be rolled out on a large scale for malaria control in Africa. One important question regarding the use of synergized insecticides is whether they will select for vector population resistance to synergized insecticide products, in other words, are PBO-synergized pyrethroids effective against highly insecticide-resistant mosquitoes? To date, no study has formally examined mosquito resistance to PBO-synergized insecticides. Here, we used <italic>Culex quinquefasciatus</italic> as a model mosquito, we examined its resistance status to different insecticides including PBO-synergized pyrethrins and tested how PBO exposure affect mosquito mortality and the expressions of metabolic enzymes. We found that field <italic>Culex quinquefasciatus</italic> mosquitoes were resistant to multiple insecticides tested, including PBO-synergized pyrethroids. Field mosquitoes had elevated levels of carboxylesterase (COE) and monooxygenase (P450) but not glutathione S-transferase (GST) enzyme expressions compared to susceptible mosquitoes. Even with optimal PBO concentration and exposure duration, field mosquitoes had significantly higher P450 enzyme expression levels after PBO exposure compared to laboratory controls, and PBO exposure had no impact on COE enzyme expressions. The phenomena of the insecticide-resistant mosquitoes’ insensitivity to PBO exposure or PBO-synergized insecticides and multiple-resistance mechanisms have also been reported from <italic>Aedes</italic> and <italic>Anopheles</italic> mosquitoes in different countries. These results demonstrate that PBO alone is not enough to control highly pyrethroid resistant mosquitoes due to multiple resistance mechanisms. Mosquito resistance to PBO-synergized insecticide should be closely monitored</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462517', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Hippocampal CA1 pyramidal cells do not receive monosynaptic input from thalamic nucleus reuniens', 'authors': \"[{'name': 'Lilya Andrianova'}, {'name': 'Erica S Brady'}, {'name': 'Gabriella Margetts-Smith'}, {'name': 'Shivali Kohli'}, {'name': 'Chris J McBain'}, {'name': 'Michael T Craig'}]\", 'abstract': '<p>Thalamic nucleus reuniens (NRe) is located on the midline and acts as a hub to mediate interactions between hippocampal, entorhinal and prefrontal areas although these connections are poorly defined at the cellular and synaptic level. Using electrophysiology and monosynaptic circuit-tracing, we found that pyramidal cells in CA1 receive no direct input from NRe. This contrasts starkly with prefrontal cortex, subiculum and entorhinal cortex, and indicates that NRe inputs to CA1 primarily drive local inhibition and not excitation they do in the other regions.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462495', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Dominant Awn Inhibitor, which encodes an ALOG protein in sorghum, suppresses awn formation in rice', 'authors': \"[{'name': 'Hideki Takanashi'}, {'name': 'Hiromi Kajiya-Kanegae'}, {'name': 'Asuka Nishimura'}, {'name': 'Junko Yamada'}, {'name': 'Motoyuki Ishimori'}, {'name': 'Masaaki Kobayashi'}, {'name': 'Kentaro Yano'}, {'name': 'Hiroyoshi Iwata'}, {'name': 'Nobuhiro Tsutsumi'}, {'name': 'Wataru Sakamoto'}]\", 'abstract': '<p>The awn, a needle-like structure that extends from the tip of the lemma in grass species, is believed to play a role in environmental adaptation and fitness. Wild species of cereal crops tend to have longer awns, whereas modern cultivars display a variety of awn lengths and morphologies. Awns in some crops seem to have been eliminated during domestication, and genes controlling awn formation have been reported. Interestingly, a few single dominant factors that eliminate awns are already known; in sorghum (<italic>Sorghum bicolor</italic>), for example, the dominant awn inhibitor has been known for a century, though its molecular identity remains uncharacterized. In this study, we conducted quantitative trait locus (QTL) analysis and a genome-wide association study of awn-related traits in sorghum. Among the three QTLs controlling awn presence and length, one of them, mapped on chromosome 3, had an extraordinary effect on eliminating awns. We provide a line of evidence demonstrating that this QTL corresponds to <italic>Dominant Awn Inhibitor</italic> (<italic>DAI</italic>), encoding an ALOG family protein that potentially acts as a transcription factor. Detailed analysis of the ALOG protein family in cereals revealed that <italic>DAI</italic> was considered to originate from a gene duplication of its twin paralogue on chromosome 10. Interestingly, heterologous expression of <italic>DAI</italic> with its own promoter in rice inhibited awn formation in the awned cultivar Kasalath. Our finding that <italic>DAI</italic> was born by gene duplication provides an interesting example of gain-of-function, which occurs only in sorghum but shares its functionality with rice and sorghum.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462603', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Regulation of the Evolutionarily Conserved Muscle Myofibrillar Matrix by Cell Type Dependent and Independent Mechanisms', 'authors': \"[{'name': 'Peter T. Ajayi'}, {'name': 'Prasanna Katti'}, {'name': 'T. Bradley Willingham'}, {'name': 'Ye Sun'}, {'name': 'Christopher K.E. Bleck'}, {'name': 'Brian Glancy'}]\", 'abstract': '<p>Skeletal muscles play a central role in human movement through forces transmitted by contraction of the sarcomere. We recently showed that mammalian sarcomeres are connected through frequent branches forming a singular, mesh-like myofibrillar matrix. However, the extent to which myofibrillar connectivity is evolutionarily conserved as well as mechanisms which regulate the specific architecture of sarcomere branching remain unclear. Here, we demonstrate the presence of a myofibrillar matrix in the tubular, but not indirect flight (IF) muscles within <italic>Drosophila melanogaster</italic>. Moreover, we find that loss of transcription factor <italic>H15</italic> increases sarcomere branching frequency in the tubular jump muscles, and we show that sarcomere branching can be turned on in IF muscles by <italic>salm</italic>-mediated conversion to tubular muscles. Finally, we demonstrate that <italic>neurochondrin</italic> knockdown results in myofibrillar connectivity in IF muscles without conversion to tubular muscles. These data indicate an evolutionarily conserved myofibrillar matrix regulated by both cell-type dependent and independent mechanisms.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.462226', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Osteoblasts pattern endothelium and somatosensory axons during zebrafish caudal fin organogenesis', 'authors': \"[{'name': 'Rosalind G. Bump'}, {'name': 'Camille E. A. Goo'}, {'name': 'Emma C. Horton'}, {'name': 'Jeffrey P. Rasmussen'}]\", 'abstract': '<p>Skeletal elements frequently associate with vasculature and somatosensory nerves, which regulate bone development and homeostasis. However, the deep, internal location of bones in many vertebrates has limited <italic>in vivo</italic> exploration of the neurovascular-bone relationship. Here, we use the zebrafish caudal fin, an optically accessible organ formed of repeating bony ray skeletal units, to determine the cellular relationship between nerves, bones, and endothelium. In adults, we establish the presence of somatosensory axons running through the inside of the bony fin rays, juxtaposed with osteoblasts on the inner hemiray surface. During development, we show the caudal fin progresses through sequential stages of endothelial plexus formation, bony ray addition, ray innervation, and endothelial remodeling. Surprisingly, the initial stages of fin morphogenesis proceed normally in animals lacking either fin endothelium or somatosensory nerves. Instead, we find that <italic>sp7+</italic> osteoblasts are required for endothelial remodeling and somatosensory axon innervation in the developing fin. Overall, this study demonstrates that the proximal neurovascular-bone relationship in the adult caudal fin is established during fin organogenesis and suggests that ray-associated osteoblasts pattern axons and endothelium.</p><sec><title>Summary statement</title><p>Analysis of cellular interdependence during caudal fin development reveals roles for osteoblasts in patterning endothelium and somatosensory axon innervation.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462180', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Extracellular Electron Transfer Enables Cellular Control of Cu(I)-catalyzed Alkyne-Azide Cycloaddition', 'authors': \"[{'name': 'Gina Partipilo'}, {'name': 'Austin J. Graham'}, {'name': 'Brian Belardi'}, {'name': 'Benjamin K. Keitz'}]\", 'abstract': '<p>Extracellular electron transfer (EET) is an anaerobic respiration process that couples carbon oxidation to the reduction of metal species. In the presence of a suitable metal catalyst, EET allows for cellular metabolism to control a variety of synthetic transformations. Here, we report the use of EET from the model electroactive bacterium <italic>Shewanella oneidensis</italic> for metabolic and genetic control over Cu(I)-catalyzed Alkyne-Azide Cycloaddition (CuAAC). CuAAC conversion under anaerobic and aerobic conditions was dependent on live, actively respiring <italic>S. oneidensis</italic> cells. In addition, reaction progress and kinetics could be further manipulated by tailoring the central carbon metabolism of <italic>S. oneidensis</italic>. Similarly, CuAAC activity was dependent on specific EET pathways and could be manipulated using inducible genetic circuits controlling the expression of EET-relevant proteins including MtrC, MtrA, and CymA. EET-driven CuAAC also exhibited modularity and robustness in ligand tolerance and substrate scope. Furthermore, the living nature of this system could be exploited to perform multiple reaction cycles without requiring regeneration, something inaccessible to traditional chemical reductants. Finally, <italic>S. oneidensis</italic> enabled bioorthogonal CuAAC membrane labelling on live mammalian cells without affecting cell viability, suggesting that <italic>S. oneidensis</italic> can act as a dynamically tunable biocatalyst in complex environments. In summary, our results demonstrate how EET can expand the reaction scope available to living systems by enabling cellular control of CuAAC.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462680', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Evolutionary insights into primate skeletal gene regulation using a comparative cell culture model', 'authors': \"[{'name': 'Genevieve Housman'}, {'name': 'Emilie Briscoe'}, {'name': 'Yoav Gilad'}]\", 'abstract': '<p>The evolution of complex skeletal traits in primates was likely influenced by both genetic and environmental factors. Because skeletal tissues are notoriously challenging to study using functional genomic approaches, they remain poorly characterized even in humans, let alone across multiple species. The challenges involved in obtaining functional genomic data from the skeleton, combined with the difficulty of obtaining such tissues from nonhuman apes, motivated us to consider an alternative <italic>in vitro</italic> system with which to comparatively study gene regulation in skeletal cell types. Specifically, we differentiated six human and six chimpanzee induced pluripotent stem cell lines (<bold>iPSCs</bold>) into mesenchymal stem cells (<bold>MSCs</bold>) and subsequently into osteogenic cells (bone cells). We validated differentiation using standard methods and collected single-cell RNA sequencing data from over 100,000 cells across multiple samples and replicates at each stage of differentiation. While most genes that we examined display conserved patterns of expression across species, hundreds of genes are differentially expressed (<bold>DE</bold>) between humans and chimpanzees within and across stages of osteogenic differentiation. Some of these interspecific DE genes show functional enrichments relevant in skeletal tissue trait development. Moreover, topic modeling indicates that interspecific gene programs become more pronounced as cells mature. Overall, we propose that this <italic>in vitro</italic> model can be used to identify interspecific regulatory differences that may have contributed to skeletal trait differences between species.</p><sec><title>Author Summary</title><p>Primates display a range of skeletal morphologies and susceptibilities to skeletal diseases, but the molecular basis of these phenotypic differences is unclear. Studies of gene expression variation in primate skeletal tissues are extremely restricted due to the ethical and practical challenges associated with collecting samples. Nevertheless, the ability to study gene regulation in primate skeletal tissues is crucial for understanding how the primate skeleton has evolved. We therefore developed a comparative primate skeletal cell culture model that allows us to access a spectrum of human and chimpanzee cell types as they differentiate from stem cells into bone cells. While most gene expression patterns are conserved across species, we also identified hundreds of differentially expressed genes between humans and chimpanzees within and across stages of differentiation. We also classified cells by osteogenic stage and identified additional interspecific differentially expressed genes which may contribute to skeletal trait differences. We anticipate that this model will be extremely useful for exploring questions related to gene regulation variation in primate bone biology and development.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.462168', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Neurofibromin 1 controls metabolic balance and Notch-dependent quiescence of juvenile myogenic progenitors', 'authors': \"[{'name': 'Xiaoyan Wei'}, {'name': 'Matthias Lienhard'}, {'name': 'Arunima Murgai'}, {'name': 'Julia Franke'}, {'name': 'Sophie Pöhle-Kronawitter'}, {'name': 'Georgios Kotsaris'}, {'name': 'Hao Wu'}, {'name': 'Stefan Börno'}, {'name': 'Bernd Timmermann'}, {'name': 'Rainer Glauben'}, {'name': 'Sigmar Stricker'}]\", 'abstract': '<p>Patients affected by neurofibromatosis type 1 (NF1) frequently show muscle weakness with unknown etiology. Here we show that Neurofibromin-1 (Nf1) is not required in muscle fibers, but specifically in early postnatal myogenic progenitors (MPs), where Nf1 loss led to cell cycle exit and differentiation blockade, depleting the MP pool resulting in reduced myonuclear accrual as well as reduced muscle stem cell numbers. This was caused by precocious induction of stem cell quiescence coupled to metabolic reprogramming of MPs impinging on glycolytic shutdown, which was conserved in muscle fibers. We show that a Mek/Erk/NOS pathway hypersensitizes Nf1-deficient MPs to Notch signaling, consequently, early postnatal Notch pathway inhibition ameliorated premature quiescence, metabolic reprogramming and muscle growth. This reveals an unexpected role of Ras/Mek/Erk signaling supporting postnatal MP quiescence in concert with Notch signaling, which is controlled by Nf1 safeguarding coordinated muscle growth and muscle stem cell pool establishment. Furthermore, our data suggest transmission of metabolic reprogramming across cellular differentiation, affecting fiber metabolism and function in NF1.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.459010', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A whole-brain monosynaptic input connectome to neuron classes in mouse visual cortex', 'authors': \"[{'name': 'Shenqin Yao'}, {'name': 'Quanxin Wang'}, {'name': 'Karla E. Hirokawa'}, {'name': 'Benjamin Ouellette'}, {'name': 'Ruweida Ahmed'}, {'name': 'Jasmin Bomben'}, {'name': 'Krissy Brouner'}, {'name': 'Linzy Casal'}, {'name': 'Shiella Caldejon'}, {'name': 'Andy Cho'}, {'name': 'Nadezhda I. Dotson'}, {'name': 'Tanya L. Daigle'}, {'name': 'Tom Egdorf'}, {'name': 'Rachel Enstrom'}, {'name': 'Amanda Gary'}, {'name': 'Emily Gelfand'}, {'name': 'Melissa Gorham'}, {'name': 'Fiona Griffin'}, {'name': 'Hong Gu'}, {'name': 'Nicole Hancock'}, {'name': 'Robert Howard'}, {'name': 'Leonard Kuan'}, {'name': 'Sophie Lambert'}, {'name': 'Eric Kenji Lee'}, {'name': 'Jennifer Luviano'}, {'name': 'Kyla Mace'}, {'name': 'Michelle Maxwell'}, {'name': 'Marty T. Mortrud'}, {'name': 'Maitham Naeemi'}, {'name': 'Chelsea Nayan'}, {'name': 'Nhan-Kiet Ngo'}, {'name': 'Thuyanh Nguyen'}, {'name': 'Kat North'}, {'name': 'Shea Ransford'}, {'name': 'Augustin Ruiz'}, {'name': 'Sam Seid'}, {'name': 'Jackie Swapp'}, {'name': 'Michael J Taormina'}, {'name': 'Wayne Wakeman'}, {'name': 'Thomas Zhou'}, {'name': 'Philip R. Nicovich'}, {'name': 'Ali Williford'}, {'name': 'Lydia Potekhina'}, {'name': 'Medea McGraw'}, {'name': 'Lydia Ng'}, {'name': 'Peter A. Groblewski'}, {'name': 'Bosiljka Tasic'}, {'name': 'Stefan Mihalas'}, {'name': 'Julie A Harris'}, {'name': 'Ali Cetin'}, {'name': 'Hongkui Zeng'}]\", 'abstract': '<p>Identification of the structural connections between neurons is a prerequisite to understanding brain function. We developed a pipeline to systematically map brain-wide monosynaptic inputs to specific neuronal populations using Cre-driver mouse lines and the recombinant rabies tracing system. We first improved the rabies virus tracing strategy to accurately identify starter cells and to efficiently quantify presynaptic inputs. We then mapped brain-wide presynaptic inputs to different excitatory and inhibitory neuron subclasses in the primary visual cortex and seven higher visual areas. Our results reveal quantitative target-, layer- and cell-class-specific differences in the retrograde connectomes, despite similar global input patterns to different neuronal populations in the same anatomical area. The retrograde connectivity we define is consistent with the presence of the ventral and dorsal visual information processing streams and reveals further subnetworks within the dorsal stream. The hierarchical organization of the entire visual cortex can be derived from intracortical feedforward and feedback pathways mediated by upper- and lower-layer input neurons, respectively. This study expands our knowledge of the brain-wide inputs regulating visual areas and demonstrates that our improved rabies virus tracing strategy can be used to scale up the effort in dissecting connectivity of genetically defined cell populations in the whole mouse brain.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462384', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Bromodomain factor 5 is an essential transcriptional regulator of the Leishmania genome', 'authors': \"[{'name': 'Nathaniel G. Jones'}, {'name': 'Vincent Geoghegan'}, {'name': 'Gareth Moore'}, {'name': 'Juliana B. T. Carnielli'}, {'name': 'Katherine Newling'}, {'name': 'Félix Calderón'}, {'name': 'Raquel Gabarró'}, {'name': 'Julio Martín'}, {'name': 'Rab Prinjha'}, {'name': 'Inmaculada Rioja'}, {'name': 'Anthony J. Wilkinson'}, {'name': 'Jeremy C. Mottram'}]\", 'abstract': '<p><italic>Leishmania</italic> are unicellular parasites that cause human and animal disease. Alongside other organisms in kinetoplastida, they have evolved an unusual genome architecture that requires all RNA polymerase II transcribed genes to be expressed constitutively, with transcriptional start regions denoted by histone variants and histone lysine acetylation. However, the way these chromatin marks are interpreted by the cell is not understood. Seven predicted bromodomain factors (BDF1-7), the reader modules for acetyl-lysine, were identified across <italic>Leishmania</italic> genomes. Using <italic>L. mexicana</italic> as a model, Cas9-driven gene deletions indicate that BDF1-5 are essential for promastigote survival, whilst DiCre inducible gene deletion of the dual bromodomain factor <italic>BDF5</italic> identified it to be essential for both promastigotes and amastigotes. ChIP-seq assessment of BDF5s genomic distribution revealed it as highly enriched at transcriptional start sites. Using an optimised proximity proteomic and phosphoproteomic technique, XL-BioID, we defined the BDF5-proximal environment to be enriched for other bromodomain factors, histone acetyltransferase 2, and proteins essential for transcriptional activity and RNA processing. Inducible deletion of BDF5, led to a disruption of pol II transcriptional activity and global defects in gene expression. Our results indicate the requirement of <italic>Leishmania</italic> to interpret histone acetylation marks for normal levels of gene expression and thus cellular viability.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462081', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Performance-optimized neural networks as an explanatory framework for decision confidence', 'authors': \"[{'name': 'Taylor W. Webb'}, {'name': 'Kiyofumi Miyoshi'}, {'name': 'Tsz Yan So'}, {'name': 'Sivananda Rajananda'}, {'name': 'Hakwan Lau'}]\", 'abstract': '<p>Previous work has sought to understand decision confidence as a prediction of the probability that a decision will be correct, leading to debate over whether these predictions are optimal, and whether they rely on the same decision variable as decisions themselves. This work has generally relied on idealized, low-dimensional modeling frameworks, such as signal detection theory or Bayesian inference, leaving open the question of how decision confidence operates in the domain of high-dimensional, naturalistic stimuli. To address this, we developed a deep neural network model optimized to assess decision confidence directly given high-dimensional inputs such as images. The model naturally accounts for a number of puzzling dissociations between decisions and confidence, suggests a principled explanation of these dissociations in terms of optimization for the statistics of sensory inputs, and makes the surprising prediction that, despite these dissociations, decisions and confidence depend on a common decision variable.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.461806', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Progranulin deficiency results in reduced bis(monoacylglycero)phosphate (BMP) levels and gangliosidosis', 'authors': \"[{'name': 'Sebastian Boland'}, {'name': 'Sharan Swarup'}, {'name': 'Yohannes A. Ambaw'}, {'name': 'Ruth C. Richards'}, {'name': 'Alexander W. Fischer'}, {'name': 'Shubham Singh'}, {'name': 'Geetika Aggarwal'}, {'name': 'Salvatore Spina'}, {'name': 'Alissa L. Nana'}, {'name': 'Lea T. Grinberg'}, {'name': 'William W. Seeley'}, {'name': 'Michal A. Surma'}, {'name': 'Christian Klose'}, {'name': 'Joao A. Paulo'}, {'name': 'Andrew D. Nguyen'}, {'name': 'J. Wade Harper'}, {'name': 'Tobias C. Walther'}, {'name': 'Robert V. Farese'}]\", 'abstract': '<p>Homozygous mutations of granulin precursor (<italic>GRN</italic>) lead to neuronal ceroid lipofuscinosis<sup>1</sup>, a severe neurodevelopmental disease, in humans and neuroinflammation in mice<sup>2</sup>. Haploinsufficiency of <italic>GRN</italic> almost invariably causes frontotemporal dementia (FTD)<sup>3,4</sup>. The <italic>GRN</italic> locus produces progranulin (PGRN), a lysosomal precursor protein that is cleaved to granulin peptides<sup>5,6</sup>. Despite intensive investigation, the function of granulins and the reason why their absence causes neurodegeneration remain unclear. Here, we investigated PGRN function in lipid degradation, a major function of lysosomes. We show that PGRN-knockout human cells, PGRN-deficient murine brain, and frontal lobes of human brains from patients with <italic>GRN</italic> mutation-related FTD have increased levels of gangliosides, highly abundant sialic acid–containing glycosphingolipids (GSL) that are degraded in lysosomes. Probing how PGRN deficiency causes these changes, we found normal levels and activities of enzymes that catabolize gangliosides. However, levels of bis(monoacylglycero)phosphate (BMP), a lysosomal lipid required for ganglioside catabolism<sup>7</sup>, were markedly reduced in PGRN-deficient cells and patient brain tissues. These data indicate that granulins are required to maintain BMP levels, which regulate ganglioside catabolism, and that PGRN deficiency in lysosomes leads to gangliosidosis. This aberrant accumulation of gangliosides may contribute to neuroinflammation and neurodegeneration susceptibility.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462663', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A gut commensal niche regulates stable association of a multispecies microbiota', 'authors': \"[{'name': 'Ren Dodge'}, {'name': 'Eric W. Jones'}, {'name': 'Haolong Zhu'}, {'name': 'Benjamin Obadia'}, {'name': 'Daniel J. Martinez'}, {'name': 'Chenhui Wang'}, {'name': 'Andrés Aranda-Díaz'}, {'name': 'Kevin Aumiller'}, {'name': 'Zhexian Liu'}, {'name': 'Marco Voltolini'}, {'name': 'Eoin L. Brodie'}, {'name': 'Kerwyn Casey Huang'}, {'name': 'Jean M. Carlson'}, {'name': 'David A. Sivak'}, {'name': 'Allan C. Spradling'}, {'name': 'William B. Ludington'}]\", 'abstract': '<p>The intestines of animals are typically colonized by a complex, relatively stable microbiota that influences health and fitness, but the underlying mechanisms of colonization remain poorly understood. As a typical animal, the fruit fly, <italic>Drosophila</italic> melanogaster, is associated with a consistent set of commensal bacterial species, yet the reason for this consistency is unknown. Here, we use gnotobiotic flies, microscopy, and microbial pulse-chase protocols to show that a commensal niche exists within the proventriculus region of the <italic>Drosophila</italic> foregut that selectively binds bacteria with exquisite strain-level specificity. Primary colonizers saturate the niche and exclude secondary colonizers of the same strain, but initial colonization by <italic>Lactobacillus</italic> physically remodels the niche to favor secondary colonization by <italic>Acetobacter</italic>. Our results provide a mechanistic framework for understanding the establishment and stability of an intestinal microbiome.</p><sec><title>One-Sentence Summary</title><p>A strain-specific set of bacteria inhabits a defined spatial region of the <italic>Drosophila</italic> gut that forms a commensal niche.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.21264089', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Immune Memory Response After a Booster Injection of mRNA-1273 for Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2)', 'authors': \"[{'name': 'Laurence Chu'}, {'name': 'David Montefiori'}, {'name': 'Wenmei Huang'}, {'name': 'Biliana Nestorova'}, {'name': 'Ying Chang'}, {'name': 'Andrea Carfi'}, {'name': 'Darin K. Edwards'}, {'name': 'Judy Oestreicher'}, {'name': 'Holly Legault'}, {'name': 'Bethany Girard'}, {'name': 'Rolando Pajon'}, {'name': 'Jacqueline M. Miller'}, {'name': 'Rituparna Das'}, {'name': 'Brett Leav'}, {'name': 'Roderick McPhee'}]\", 'abstract': '<p>Rising breakthrough infections of coronavirus-2 (SARS-CoV-2) in previously immunized individuals has raised concerns for a booster to combat suspected waning immunity and new variants. Participants immunized 6-8 months earlier with a primary series of two doses of 50 or 100 µg of mRNA-1273 were administered a booster injection of 50 µg of mRNA-1273. Neutralizing antibody levels against wild-type virus and the Delta variant at one month after the booster were 1.7-fold and 2.1-fold higher, respectively, than those 28 days post primary series second injection indicating an immune memory response. The reactogenicity after the booster dose was similar to that after the second dose in the primary series of two doses of mRNA-1273 (50 or 100 µg) with no serious adverse events reported in the one-month follow-up period. These results demonstrate that a booster injection of mRNA-1273 in previously immunized individuals stimulated an immune response greater than the primary vaccination series.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462463', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Optogenetic EB1 inactivation shortens metaphase spindles by disrupting cortical force-producing interactions with astral microtubules', 'authors': \"[{'name': 'Alessandro Dema'}, {'name': 'Jeffrey van Haren'}, {'name': 'Torsten Wittmann'}]\", 'abstract': '<p>Chromosome segregation is accomplished by the mitotic spindle, a bipolar micromachine built primarily from microtubules. Different microtubule populations contribute to spindle function: Kinetochore microtubules attach and transmit forces to chromosomes, antiparallel interpolar microtubules support spindle structure, and astral microtubules connect spindle poles to the cell cortex [1,2]. In mammalian cells, End Binding (EB) proteins associate with all growing microtubule plus ends throughout the cell cycle and serve as adaptors for a diverse group of +TIPs that control microtubule dynamics and interactions with other intracellular structures [3]. Because binding of many +TIPs to EB1 and thus microtubule-end association is switched off by mitotic phosphorylation [4–6] the mitotic function of EBs remains poorly understood. To analyze how EB1 and associated +TIPs on different spindle microtubule populations contribute to mitotic spindle dynamics, we use a light sensitive EB1 variant, π-EB1, that allows local, acute and reversible inactivation of +TIP association with growing microtubule ends in live cells [7]. We find that acute π-EB1 photoinactivation results in rapid and reversible metaphase spindle shortening and transient relaxation of tension across the central spindle. However, in contrast to interphase, π-EB1 photoinactivation does not inhibit microtubule growth in metaphase, but instead increases astral microtubule length and number. Yet, in the absence of EB1 activity astral microtubules fail to engage the cortical dynein/dynactin machinery and spindle poles move away from regions of π-EB1 photoinactivation. In conclusion, our optogenetic approach reveals mitotic EB1 functions that remain hidden in genetic experiments likely due to compensatory molecular systems regulating vertebrate spindle dynamics.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.21264328', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Endemic Seasonal Coronavirus Neutralisation and COVID-19 severity', 'authors': \"[{'name': 'David A. Wells'}, {'name': 'Diego Cantoni'}, {'name': 'Martin Mayora-Neto'}, {'name': 'Cecilia Di Genova'}, {'name': 'Alexander Sampson'}, {'name': 'Matteo Ferrari'}, {'name': 'George Carnell'}, {'name': 'Angalee Nadesalingam'}, {'name': 'Peter Smith'}, {'name': 'Andrew Chan'}, {'name': 'Gianmarco Raddi'}, {'name': 'Javier Castillo-Olivares'}, {'name': 'Helen Baxendale'}, {'name': 'Nigel Temperton'}, {'name': 'Jonathan L. Heeney'}]\", 'abstract': '<p>The virus SARS-CoV-2, responsible for the global COVID-19 pandemic, spread rapidly around the world causing high morbidity and mortality because humans have no pre-existing immunity. However, there are four known, endemic seasonal coronaviruses in humans (HCoVs) and whether antibodies for these HCoVs play a role in severity of COVID-19 disease has generated a lot of interest. Of these seasonal viruses NL63 is of particular interest as it uses the same cell entry receptor as SARS-CoV-2.We use functional, neutralising assays to investigate cross reactive antibodies and their relationship with COVID-19 severity. We analysed neutralisation of SARS-CoV-2, NL63, HKU1, and 229E in 38 COVID-19 patients and 62 healthcare workers, and a further 182 samples to specifically study the relationship between SARS-CoV-2 and NL63.We found that although HCoV neutralisation was very common there was little evidence that these antibodies neutralised SARS-CoV-2. Despite no evidence in cross neutralisation, levels of NL63 neutralisating antibodies become elevated after exposure to SARS-CoV-2 through infection or following vaccination.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462364', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Multi-omics and deep learning provide a multifaceted view of cancer', 'authors': \"[{'name': 'Bora Uyar'}, {'name': 'Jonathan Ronen'}, {'name': 'Vedran Franke'}, {'name': 'Gaetano Gargiulo'}, {'name': 'Altuna Akalin'}]\", 'abstract': '<p>Cancer is a complex disease with a large financial and healthcare burden on society. One hallmark of the disease is the uncontrolled growth and proliferation of malignant cells. Unlike Mendelian diseases which may be explained by a few genomic loci, a deeper molecular and mechanistic understanding of the development of cancer is needed. Such an endeavor requires the integration of tens of thousands of molecular features across multiple layers of information encoded in the cells. In practical terms, this implies integration of multi omics information from the genome, transcriptome, epigenome, proteome, metabolome, and even micro-environmental factors such as the microbiome. Finding mechanistic insights and biomarkers in such a high dimensional space is a challenging task. Therefore, efficient machine learning techniques are needed to reduce the dimensionality of the data while simultaneously discovering complex but meaningful biomarkers. These markers then can lead to testable hypotheses in research and clinical applications. In this study, we applied advanced deep learning methods to uncover multi-omic fingerprints that are associated with a wide range of clinical and molecular features of tumor samples. Using these fingerprints, we can accurately classify different cancer types, and their subtypes. Non-linear multi-omic fingerprints can uncover clinical features associated with patient survival and response to treatment, ranging from chemotherapy to immunotherapy. In addition, multi-omic fingerprints may be deconvoluted into a meaningful subset of genes and genomic alterations to support clinically relevant decisions.</p><sec><title>Graphical Abstract</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"462364v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462608', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Cost-effective, high-throughput phenotyping system for 3D reconstruction of fruit form', 'authors': \"[{'name': 'Mitchell J. Feldmann'}, {'name': 'Amy Tabb'}]\", 'abstract': '<p>Reliable phenotyping methods that are simple to operate and in-expensive to deploy are critical for studying quantitative traits in plants. Traditional fruit shape phenotyping relies on human raters or 2D analyses to assess form, e.g., size and shape. Systems for 3D imaging using multi-view stereo have been implemented, but frequently rely on commercial software and/or specialized hardware, which can lead to limitations in accessibility and scalability. We present a complete system constructed of consumer-grade components for capturing, calibrating, and reconstructing the 3D form of small-to-moderate sized fruits and tubers. Data acquisition and image capture sessions are 9 seconds to capture 60 images. The initial prototype cost was $1600 USD. We measured accuracy by comparing reconstructed models of 3D printed ground truth objects to the original digital files of those same ground truth objects. The <italic>R</italic><sup>2</sup> between length of the primary, secondary, and tertiary axes, volume, and surface area of the ground-truth object and the reconstructed models was <italic>&gt;</italic> 0.97 and root-mean square error (RMSE) was <italic>&lt;</italic>3mm for objects without locally concave regions. Measurements from 1mm and 2mm resolution reconstructions were consistent (<italic>R</italic><sup>2</sup><italic>&gt;</italic> 0.99). Qualitative assessments were performed on 48 fruit and tubers, including 18 strawberries, 12 potatoes, 5 grapes, 7 peppers, and 4 Bosch and 2 red Anjou pears. Our proposed phenotyping system is fast, relatively low cost, and has demonstrated accuracy for certain shape classes, and could be used for the 3D analysis of fruit form.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.21264242', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Clinical Trial of Efficacy and Toxicity of Disoproxil Tenofovir Fumarate and Emtricitabine for Mild to Moderate SARS-CoV-2 Infections', 'authors': \"[{'name': 'E.A.G. Arruda'}, {'name': 'R.J. Pires-Neto'}, {'name': 'M.S. Medeiros'}, {'name': 'J. Quirino-Filho'}, {'name': 'M. Clementino'}, {'name': 'R.N.D.G. Gondim'}, {'name': 'L.M.V.C. Magalhães'}, {'name': 'K.F. Cavalcante'}, {'name': 'V.A.F. Viana'}, {'name': 'Liana Perdigão Mello'}, {'name': 'R.B Martins'}, {'name': 'A.A. Santos'}, {'name': 'P.J.C. Magalhães'}, {'name': 'A. Havt'}, {'name': 'N.P. Lopes'}, {'name': 'E. Arruda-Neto'}, {'name': 'A.A.M. Lima'}, {'name': ' '}]\", 'abstract': '<p>This study aimed to evaluate the efficacy and toxicity of tenofovir (TDF) and TDF combined with emtricitabine (TDF/FTC) in patients with mild to moderate COVID-19 infections. We conducted a randomized, double-blind, placebo-controlled clinical trial in patients with clinical suspicion of mild to moderate respiratory infection caused by SARS-CoV-2 who were treated at an outpatient clinic. Patients were randomly recruited to take 10 days of TDF (300 mg/day), TDF (300 mg/day) combined with FTC (200 mg/day) or placebo Vitamin C (500 mg/day). The primary parameter was the score of symptoms and predictive signs of COVID-19, assessed on the seventh day of patient follow-up. From a total of 309 patients with clinical suspicion of SARS-CoV-2, 227 met the inclusion criteria and were randomly distributed into the following groups: (a) 75 (one did not initiate treatment) in the TDF group; (b) 74 in the TDF combined with FTC group; and (c) 77 in the Vitamin C group (placebo). Of the 226 patients, 139 (62%) were positive for SARS-CoV-2. Fever (≥37.8°C), ageusia or dysgeusia, anosmia or dysosmia, and two or more clinical symptoms or signs were significantly associated with SARS-CoV-2 infection. There was no significant change in clinical score based on clinical symptoms and signs between treatment groups. Patients with mild to moderate infection by SARS-CoV-2 had higher concentrations of G-CSF, IL-1β, IL-6 and TNF-α compared to patients without infection. Patients with mild to moderate respiratory infection, with fever (≥37.8°C), loss of smell, loss of taste and two or more symptoms, have a better prediction for the diagnosis of COVID-19. Patients with SARS-CoV-2 showed higher and more persistent proinflammatory cytokines profile compared to patients not infected with SARS-CoV-2. Pharmacological intervention with TDF or TDF combined with FTC did not change the clinical signs and symptoms score in mild to moderate respiratory infection in patients with SARS-CoV-2 compared to the Vitamin C group (placebo).</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462138', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The extended Price equation: migration and sex', 'authors': \"[{'name': 'Jake Brown'}, {'name': 'Jared M. Field'}]\", 'abstract': '<p>The Price equation provides a general partition of evolutionary change into two components. The first is usually thought to represent natural selection and the second, transmission bias. Here, we provide a new derivation of the generalised equation, which contains a largely ignored third term. Unlike the original Price equation, this extension can account for migration and mixed asexual and sexual reproduction. The notation used here expresses the generalised equation explicitly in terms of fitness, rendering this otherwise difficult third term more open to biological interpretation and use. This re-derivation also permits fundamental results, derived from the Price equation, to be more easily generalised. We take Hamilton’s rule as a case study, and provide an exact, total expression that allows for population structures like haplodiploidy. Our analysis, more generally, makes clear the previously hidden assumptions in similar fundamental results, highlighting the caution that must be taken when interpreting them.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462203', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Numb reduces Tau levels and prevents neurodegeneration in mouse models of tauopathy in an isoform-specific manner', 'authors': \"[{'name': 'Marine Lacomme'}, {'name': 'Sarah C. Hales'}, {'name': 'Katarina Stevanovic'}, {'name': 'Christine Jolicoeur'}, {'name': 'Therence Bois'}, {'name': 'Jenny Cai'}, {'name': 'Melissa Desrosiers'}, {'name': 'Deniz Dalkara'}, {'name': 'Michel Cayouette'}]\", 'abstract': '<p>Accumulation of the microtubule-associated protein Tau is linked to neuronal cell death in tauopathies, but how exactly intraneuronal Tau levels are regulated in health and disease remains unclear. Here we identify the trafficking adaptor protein Numb as an essential regulator of Tau homeostasis. Conditional inactivation of Numb in retinal ganglion cells (RGCs) increases monomeric and oligomeric Tau levels, leading to axonal blebbing followed by neuronal cell loss in aged mice. Moreover, in a mouse model of tauopathy, inactivation of Numb in RGCs and spinal motoneurons accelerates neurodegeneration, and leads to precocious hindlimb paralysis. Conversely, overexpression of the long isoform of Numb (Numb-72), but not other isoforms, decreases intracellular Tau levels by promoting the extracellular release of monomeric Tau, and AAV-mediated delivery of Numb-72 in RGCs <italic>in vivo</italic> prevents neurodegeneration in two different mouse models of tauopathy. Taken together, these results uncover Numb as a modulator of intracellular Tau levels and identify Numb-72 as a novel therapeutic factor for tauopathies.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462487', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Endoplasmic reticulum stress activates human IRE1α through reversible assembly of inactive dimers into small oligomers', 'authors': \"[{'name': 'Vladislav Belyy'}, {'name': 'Iratxe Zuazo-Gaztelu'}, {'name': 'Andrew Alamban'}, {'name': 'Avi Ashkenazi'}, {'name': 'Peter Walter'}]\", 'abstract': '<p>Protein folding homeostasis in the endoplasmic reticulum (ER) is regulated by a signaling network, termed the unfolded protein response (UPR). Inositol-requiring enzyme 1 (IRE1) is an ER membrane-resident kinase/RNase that mediates signal transmission in the most evolutionarily conserved branch of the UPR. Dimerization and/or higher-order oligomerization of IRE1 are thought to be important for its activation mechanism, yet the actual oligomeric states of inactive, active, and attenuated mammalian IRE1 complexes remained unknown. We developed an automated two-color single-molecule tracking approach to dissect the oligomerization of tagged endogenous human IRE1 in live cells. In contrast to previous models, our data indicate that IRE1 exists as a constitutive homodimer at baseline and assembles into small oligomers upon ER stress. We demonstrate that the formation of inactive dimers and stress-dependent oligomers is fully governed by IRE1’s lumenal domain. Phosphorylation of IRE1’s kinase domain occurs more slowly than oligomerization and is retained after oligomers disassemble back into dimers. Our findings suggest that assembly of IRE1 dimers into larger oligomers specifically enables <italic>trans-</italic> autophosphorylation, which in turn drives IRE1’s RNase activity.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462468', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Distance estimation from monocular cues in an ethological visuomotor task', 'authors': \"[{'name': 'Philip R. L. Parker'}, {'name': 'Elliott T. T. Abe'}, {'name': 'Natalie T. Beatie'}, {'name': 'Emmalyn S. P. Leonard'}, {'name': 'Dylan M. Martins'}, {'name': 'Shelby L. Sharp'}, {'name': 'David G. Wyrick'}, {'name': 'Luca Mazzucato'}, {'name': 'Cristopher M. Niell'}]\", 'abstract': '<p>In natural contexts, sensory processing and motor output are closely coupled, which is reflected in the fact that many brain areas contain both sensory and movement signals. However, standard reductionist paradigms decouple sensory decisions from their natural motor consequences, and head-fixation prevents the natural sensory consequences of self-motion. In particular, movement through the environment provides a number of depth cues beyond stereo vision that are poorly understood. To study the integration of visual processing and motor output in a naturalistic task, we investigated distance estimation in freely moving mice. We found that mice use vision to accurately jump across a variable gap, thus directly coupling a visual computation to its corresponding ethological motor output. Monocular eyelid suture did not affect performance, thus mice can use cues that do not depend on binocular disparity and stereo vision. Under monocular conditions, mice performed more vertical head movements, consistent with the use of motion parallax cues, and optogenetic suppression of primary visual cortex impaired task performance. Together, these results show that mice can use monocular cues, relying on visual cortex, to accurately judge distance. Furthermore, this behavioral paradigm provides a foundation for studying how neural circuits convert sensory information into ethological motor output.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.461531', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Sex Differences in Brain Tumor Glutamine Metabolism Reveal Sex-Specific Vulnerabilities to Treatment', 'authors': \"[{'name': 'Jasmin Sponagel'}, {'name': 'Jill K. Jones'}, {'name': 'Cheryl Frankfater'}, {'name': 'Shanshan Zhang'}, {'name': 'Olivia Tung'}, {'name': 'Kevin Cho'}, {'name': 'Kelsey L. Tinkum'}, {'name': 'Hannah Gass'}, {'name': 'Elena Nunez'}, {'name': 'Douglas R. Spitz'}, {'name': 'Prakash Chinnaiyan'}, {'name': 'Jacob Schaefer'}, {'name': 'Gary J. Patti'}, {'name': 'Maya S. Graham'}, {'name': 'Audrey Mauguen'}, {'name': 'Milan Grkovski'}, {'name': 'Mark P. Dunphy'}, {'name': 'Simone Krebs'}, {'name': 'Jingqin Luo'}, {'name': 'Joshua B. Rubin'}, {'name': 'Joseph E. Ippolito'}]\", 'abstract': '<p>Sex differences in normal metabolism are well described, but whether they persist in cancerous tissue is unknown. We assessed metabolite abundance in glioblastoma surgical specimens and found that male glioblastomas are enriched for amino acids, including glutamine. Using PET imaging, we found that gliomas in male patients exhibit significantly higher glutamine uptake. These sex differences were well-modeled in murine transformed astrocytes, in which male cells imported and metabolized more glutamine and were more sensitive to glutaminase 1 (GLS1) inhibition. The sensitivity to GLS1 inhibition in males was driven by their dependence on glutamine-derived glutamate for α-ketoglutarate synthesis and TCA cycle replenishment. Females were resistant to GLS1 inhibition through greater pyruvate carboxylase-mediated TCA cycle replenishment. Thus, clinically important sex differences exist in targetable elements of metabolism. Recognition of sex-biased metabolism is an opportunity to improve treatments for all patients through further laboratory and clinical research.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462396', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Quantifying mediation between omics layers and complex traits', 'authors': \"[{'name': 'Marie C. Sadler'}, {'name': 'Chiara Auwerx'}, {'name': 'Eleonora Porcu'}, {'name': 'Zoltán Kutalik'}]\", 'abstract': '<sec><title>Background</title><p>High-dimensional omics datasets provide valuable resources to determine the causal role of molecular traits in mediating the path from genotype to phenotype. Making use of quantitative trait loci (QTL) and genome-wide association studies (GWASs) summary statistics, we developed a multivariable Mendelian randomization (MVMR) framework to quantify the connectivity between three omics layers (DNA methylome (DNAm), transcriptome and proteome) and their cascading causal impact on complex traits and diseases.</p></sec><sec><title>Results</title><p>Evaluating 50 complex traits, we found that on average 37.8% (95% CI: [36.0%-39.5%]) of DNAm-to-trait effects were mediated through transcripts in the <italic>cis</italic>-region, while only 15.8% (95% CI: [11.9%-19.6%]) are mediated through proteins in <italic>cis</italic>. DNAm sites typically regulate multiple transcripts, and while found to predominantly decrease gene expression, this was only the case for 53.4% across ≈ 47,000 significant DNAm-transcript pairs. The average mediation proportion for transcript-to-trait effects through proteins (encoded for by the assessed transcript or located in <italic>trans</italic>) was estimated to be 5.27% (95%CI: [4.11%-6.43%]). Notable differences in the transcript and protein QTL architectures were detected with only 22% of protein levels being causally driven by their corresponding transcript levels. Several regulatory mechanisms were hypothesized including an example where cg10385390 (chr1:8’022’505) increases the risk of irritable bowel disease by reducing <italic>PARK7</italic> transcript and protein expression.</p></sec><sec><title>Conclusions</title><p>The proposed integrative framework identified putative causal chains through omics layers providing a powerful tool to map GWAS signals. Quantification of causal effects between successive layers indicated that molecular mechanisms can be more complex than what the central dogma of biology would suggest.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462204', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'An Evolutionarily Conserved Regulatory Pathway of Muscle Mitochondrial Network Organization', 'authors': \"[{'name': 'Prasanna Katti'}, {'name': 'Peter T. Ajayi'}, {'name': 'Angel Aponte'}, {'name': 'Christopher K.E. Bleck'}, {'name': 'Brian Glancy'}]\", 'abstract': '<p>Mitochondrial networks provide coordinated energy distribution throughout muscle cells. However, pathways specifying mitochondrial network-type separately from contractile fiber-type remain unclear. Here, we show that natural energetic demands placed on <italic>Drosophila melanogaster</italic> muscles yield native cell-types among which contractile and mitochondrial network-types are regulated independently. Proteomic analyses of indirect flight, jump, and leg muscles together with muscles misexpressing known fiber-type specification factor <italic>salm</italic> identified transcription factors <italic>H15</italic> and <italic>cut</italic> as potential mitochondrial network regulators. We demonstrate <italic>H15</italic> operates downstream of <italic>salm</italic> regulating flight muscle contractile and mitochondrial network-type. Conversely, <italic>H15</italic> regulates mitochondrial network configuration but not contractile type in jump and leg muscles. Further, we find that <italic>cut</italic> regulates <italic>salm</italic> expression in flight muscles and mitochondrial network configuration in leg muscles. These data indicate cell type-specific regulation of muscle mitochondrial network organization separately from contractile type, mitochondrial content, and mitochondrial size through an evolutionarily conserved pathway involving <italic>cut, salm</italic>, and <italic>H15</italic>.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.462139', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The spread of the first introns in proto-eukaryotic paralogs', 'authors': \"[{'name': 'Julian Vosseberg'}, {'name': 'Michelle Schinkel'}, {'name': 'Sjoerd Gremmen'}, {'name': 'Berend Snel'}]\", 'abstract': '<p>Spliceosomal introns are a unique feature of eukaryotic genes. Previous studies have established that many introns were present in the protein-coding genes of the last eukaryotic common ancestor (LECA). Intron positions shared between genes that duplicated before LECA could in principle provide insight into the emergence of the first introns. In this study we use ancestral intron position reconstructions in two large sets of duplicated families to systematically identify these ancient paralogous intron positions. We found that 20-35% of introns inferred to have been present in LECA were shared between paralogs. These shared introns, which likely preceded ancient duplications, were widespread across different functions, with the notable exception of nuclear transport. Since we observed a clear signal of pervasive intron loss prior to LECA, it is likely that substantially more introns were shared at the time of duplication than we can detect in LECA. The large extent of shared introns indicates an early origin of introns during eukaryogenesis and suggests an early origin of a nuclear structure, before most of the other complex eukaryotic features were established.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462233', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Convergent mosaic brain evolution is associated with the evolution of novel electrosensory systems in teleost fishes', 'authors': \"[{'name': 'Erika L. Schumacher'}, {'name': 'Bruce A. Carlson'}]\", 'abstract': '<p>Brain region size generally scales allometrically with total brain size, but mosaic shifts in brain region size independent of brain size have been found in several lineages and may be related to the evolution of behavioral novelty. African weakly electric fishes (Mormyroidea) evolved a mosaically enlarged cerebellum and hindbrain, yet the relationship to their behaviorally novel electrosensory system remains unclear. We addressed this by studying South American weakly electric fishes (Gymnotiformes) and weakly electric catfishes (<italic>Synodontis</italic> spp.), which evolved varying aspects of electrosensory systems, independent of mormyroids. If the mormyroid mosaic increases are related to evolving an electrosensory system, we should find similar mosaic shifts in gymnotiforms and <italic>Synodontis</italic>. Using micro-computed tomography scans, we quantified brain region scaling for multiple electrogenic, electroreceptive, and non-electrosensing species. We found mosaic increases in cerebellum in all three electrogenic lineages relative to non-electric lineages and mosaic increases in torus semicircularis and hindbrain associated with the evolution of electrogenesis and electroreceptor type. These results show that evolving novel electrosensory systems is repeatedly and independently associated with changes in the sizes of individual brain regions independent of brain size, which suggests that selection can impact structural brain composition to favor specific regions involved in novel behaviors.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.27.21263187', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Comprehensive genetic diagnosis of tandem repeat expansion disorders with programmable targeted nanopore sequencing', 'authors': \"[{'name': 'Igor Stevanovski'}, {'name': 'Sanjog R. Chintalaphani'}, {'name': 'Hasindu Gamaarachchi'}, {'name': 'James M. Ferguson'}, {'name': 'Sandy S. Pineda'}, {'name': 'Carolin K. Scriba'}, {'name': 'Michel Tchan'}, {'name': 'Victor Fung'}, {'name': 'Karl Ng'}, {'name': 'Andrea Cortese'}, {'name': 'Henry Houlden'}, {'name': 'Carol Dobson-Stone'}, {'name': 'Lauren Fitzpatrick'}, {'name': 'Glenda Halliday'}, {'name': 'Gianina Ravenscroft'}, {'name': 'Mark R. Davis'}, {'name': 'Nigel G. Laing'}, {'name': 'Avi Fellner'}, {'name': 'Marina Kennerson'}, {'name': 'Kishore R. Kumar'}, {'name': 'Ira W. Deveson'}]\", 'abstract': '<p>Short-tandem repeat (STR) expansions are an important class of pathogenic genetic variants. Over forty neurological and neuromuscular diseases are caused by STR expansions, with 37 different genes implicated to date. Here we describe the use of programmable targeted long-read sequencing with Oxford Nanopore’s ReadUntil function for parallel genotyping of all known neuropathogenic STRs in a single, simple assay. Our approach enables accurate, haplotype-resolved assembly and DNA methylation profiling of expanded and non-expanded STR sites. In doing so, the assay correctly diagnoses all individuals in a cohort of patients (<italic>n</italic> = 27) with various neurogenetic diseases, including Huntington’s disease, fragile X syndrome and cerebellar ataxia (CANVAS) and others. Targeted long-read sequencing solves large and complex STR expansions that confound established molecular tests and short-read sequencing, and identifies non-canonical STR motif conformations and internal sequence interruptions. Even in our relatively small cohort, we observe a wide diversity of STR alleles of known and unknown pathogenicity, suggesting that long-read sequencing will redefine the genetic landscape of STR expansion disorders. Finally, we show how the flexible inclusion of pharmacogenomics (PGx) genes as secondary ReadUntil targets can identify clinically actionable PGx genotypes to further inform patient care, at no extra cost. Our study addresses the need for improved techniques for genetic diagnosis of STR expansion disorders and illustrates the broad utility of programmable long-read sequencing for clinical genomics.</p><sec><title>One sentence summary</title><p>This study describes the development and validation of a programmable targeted nanopore sequencing assay for parallel genetic diagnosis of all known pathogenic short-tandem repeats (STRs) in a single, simple test.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462432', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Natural and designed proteins inspired by extremotolerant organisms can form condensates and attenuate apoptosis in human cells', 'authors': \"[{'name': 'Mike T. Veling'}, {'name': 'Dan T. Nguyen'}, {'name': 'Nicole N. Thadani'}, {'name': 'Michela E. Oster'}, {'name': 'Nathan J. Rollins'}, {'name': 'Kelly P. Brock'}, {'name': 'Neville P. Bethel'}, {'name': 'David Baker'}, {'name': 'Jeffrey C. Way'}, {'name': 'Debora S. Marks'}, {'name': 'Roger L. Chang'}, {'name': 'Pamela A. Silver'}]\", 'abstract': '<p>Many organisms can survive extreme conditions and successfully recover to normal life. This extremotolerant behavior has been attributed in part to repetitive, amphipathic, and intrinsically disordered proteins that are upregulated in the protected state. Here, we assemble a library of approximately 300 naturally-occurring and designed extremotolerance-associated proteins to assess their ability to protect human cells from chemically-induced apoptosis. We show that proteins from tardigrades, nematodes, and the Chinese giant salamander are apoptosis protective. Notably, we identify a region of the human ApoE protein with similarity to extremotolerance-associated proteins that also protects against apoptosis. This region mirrors the phase separation behavior seen with such proteins, like the tardigrade protein CAHS2. Moreover, we identify a synthetic protein, DHR81, that shares this combination of elevated phase separation propensity and apoptosis protection. Finally, we demonstrate that driving protective proteins into the condensate state increases apoptosis protection, and highlight the ability for DHR81 condensates to sequester caspase-7. Taken together, this work draws a link between extremotolerance-associated proteins, condensate formation, and human cellular protection.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462436', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Single-cell and nucleus RNA-seq in a mouse model of AD reveal activation of distinct glial subpopulations in the presence of plaques and tangles', 'authors': \"[{'name': 'Gabriela Balderrama-Gutierrez'}, {'name': 'Heidi Liang'}, {'name': 'Narges Rezaie'}, {'name': 'Klebea Carvalho'}, {'name': 'Stefania Forner'}, {'name': 'Dina Matheos'}, {'name': 'Elisabeth Rebboah'}, {'name': 'Kim N. Green'}, {'name': 'Andrea J. Tenner'}, {'name': 'Frank LaFerla'}, {'name': 'Ali Mortazavi'}]\", 'abstract': '<p>Multiple mouse models have been generated that strive to recapitulate human Alzheimer’s disease (AD) pathological features to investigate disease mechanisms and potential treatments. The 3xTg-AD mouse presents the two major hallmarks of AD, which are plaques and tangles that increase during aging. While behavioral changes and the accumulation of plaques and tangles have been well described in the 3xTg-AD mice, the subpopulations of neurons and glial cells present throughout disease progression have not been characterized. Here, we used single-cell RNA-seq to investigate changes in subpopulations of microglia, and single-nucleus RNA-seq to explore subpopulations of neurons, astrocytes, and oligodendrocytes in the hippocampus and cortex of aging 3xTg-AD as well as 5xFAD mice for comparison. We recovered a common path of age-associated astrocyte activation between the 3xTg-AD and the 5xFAD models and found that 3xTg-AD-derived astrocytes seem to be less activated. We identified multiple subtypes of microglia, including a subpopulation with a distinct transcription factor expression profile that showed an early increase in <italic>Csf1</italic> expression before the switch to disease associated microglia (DAM). We used bulk RNA-seq in the hippocampus of 3xTg-AD mice across their lifespan to identify distinct modules of genes whose expression increases with aging and worsening pathology. Finally, scATAC-seq revealed multiple subpopulations of cells with accessible chromatin in regions around genes associated with glial activation. Overall, differences between the main glial groups point to a slower activation process in the 3xTg-AD model when compared to the 5xFAD. Our study contributes to the identification of progressive transcriptional changes of glial cells in a mouse model that has plaques and tangles, thus providing information to aid in targeted AD therapeutics that could translate into positive clinical outcomes.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462632', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Direct Reprogramming of Non-limb Fibroblasts to Cells with Properties of Limb Progenitors', 'authors': \"[{'name': 'Yuji Atsuta'}, {'name': 'Changhee Lee'}, {'name': 'Alan R. Rodrigues'}, {'name': 'Charlotte Colle'}, {'name': 'Reiko R. Tomizawa'}, {'name': 'Ernesto G. Lujan'}, {'name': 'Patrick Tschopp'}, {'name': 'Joshua M. Gorham'}, {'name': 'Jean-Pierre Vannier'}, {'name': 'Christine E. Seidman'}, {'name': 'Jonathan G. Seidman'}, {'name': 'Olivier Pourquié'}, {'name': 'Clifford J. Tabin'}]\", 'abstract': '<p>The early limb bud consists of mesenchymal progenitors (limb progenitors) derived from the lateral plate mesoderm (LPM) that produce most of the tissues of the mature limb bud. The LPM also gives rise to the mesodermal components of the trunk, flank and neck. However, the mesenchymal cells generated at these other axial levels cannot produce the variety of cell types found in the limb bud, nor can they be directed to form a patterned appendage-like structure, even when placed in the context of the signals responsible for organizing the limb bud. Here, by taking advantage of a direct reprogramming approach, we find a set of factors (Prdm16, Zbtb16, and Lin28) normally expressed in the early limb bud, that are capable of imparting limb progenitor-like properties to non-limb fibroblasts. Cells reprogrammed by these factors show similar gene expression profiles, and can differentiate into similar cell types, as endogenous limb progenitors. The further addition of Lin41 potentiates proliferation of the reprogrammed cells while suppressing differentiation. These results suggest that these same four key factors may play pivotal roles in the specification of endogenous limb progenitors.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462420', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'SARS-CoV-2 Beta variant infection elicits potent lineage-specific and cross-reactive antibodies', 'authors': \"[{'name': 'S Momsen Reincke'}, {'name': 'Meng Yuan'}, {'name': 'Hans-Christian Kornau'}, {'name': 'Victor M Corman'}, {'name': 'Scott van Hoof'}, {'name': 'Elisa Sánchez-Sendin'}, {'name': 'Melanie Ramberger'}, {'name': 'Wenli Yu'}, {'name': 'Yuanzi Hua'}, {'name': 'Henry Tien'}, {'name': 'Marie Luisa Schmidt'}, {'name': 'Tatjana Schwarz'}, {'name': 'Lara Maria Jeworowski'}, {'name': 'Sarah E Brandl'}, {'name': 'Helle Foverskov Rasmussen'}, {'name': 'Marie A Homeyer'}, {'name': 'Laura Stöffler'}, {'name': 'Martin Barner'}, {'name': 'Désirée Kunkel'}, {'name': 'Shufan Huo'}, {'name': 'Johannes Horler'}, {'name': 'Niels von Wardenburg'}, {'name': 'Inge Kroidl'}, {'name': 'Tabea M Eser'}, {'name': 'Andreas Wieser'}, {'name': 'Christof Geldmacher'}, {'name': 'Michael Hoelscher'}, {'name': 'Hannes Gänzer'}, {'name': 'Günter Weiss'}, {'name': 'Dietmar Schmitz'}, {'name': 'Christian Drosten'}, {'name': 'Harald Prüss'}, {'name': 'Ian A. Wilson'}, {'name': 'Jakob Kreye'}]\", 'abstract': '<p>SARS-CoV-2 Beta variant of concern (VOC) resists neutralization by major classes of antibodies from non-VOC COVID-19 patients and vaccinated individuals. Here, serum of Beta variant infected patients revealed reduced cross-neutralization of non-VOC virus. From these patients, we isolated Beta-specific and cross-reactive receptor-binding domain (RBD) antibodies. The Beta-specificity results from recruitment of novel VOC-specific clonotypes and accommodation of VOC-defining amino acids into a major non-VOC antibody class that is normally sensitive to these mutations. The Beta-elicited cross-reactive antibodies share genetic and structural features with non-VOC-elicited antibodies, including a public VH1-58 clonotype targeting the RBD ridge independent of VOC mutations. These findings advance our understanding of the antibody response to SARS-CoV-2 shaped by antigenic drift with implications for design of next-generation vaccines and therapeutics.</p><sec><title>One sentence summary</title><p>SARS-CoV-2 Beta variant elicits lineage-specific antibodies and antibodies with neutralizing breadth against wild-type virus and VOCs.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462340', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Single-cell transcriptomic analysis unveils spinal motor neuron subtype diversity underpinning the water-to-land transition in vertebrates', 'authors': \"[{'name': 'Ee Shan Liau'}, {'name': 'Suoqin Jin'}, {'name': 'Yen-Chung Chen'}, {'name': 'Wei-Szu Liu'}, {'name': 'Luok Wen Yong'}, {'name': 'Chang-Tai Tsai'}, {'name': 'Maëliss Calon'}, {'name': 'Jr-Kai Yu'}, {'name': 'Yi-Hsien Su'}, {'name': 'Stéphane Nedelec'}, {'name': 'Qing Nie'}, {'name': 'Jun-An Chen'}]\", 'abstract': '<p>Spinal motor neurons (MNs) integrate sensory stimuli and brain commands to generate motor movements in vertebrates. Distinct MN populations and their diversity has long been hypothesized to co-evolve with motor circuit to provide the neural basis from undulatory to ambulatory locomotion during aquatic-to-terrestrial transition of vertebrates. However, how these subtypes are evolved remains largely enigmatic. Using single-cell transcriptomics, we investigate heterogeneity in mouse MNs and discover novel segment-specific subtypes. Among limb-innervating MNs, we reveal a diverse neuropeptide code for delineating putative motor pool identities. We further uncovered that axial MNs are subdivided by three conserved and molecularly distinct subpopulations, defined by Satb2, Nr2f2 or Bcl11b expression. Although axial MNs are conserved from cephalochordates to humans, subtype diversity becomes prominent in land animals and appears to continue evolving in humans. Overall, our study provides a unified classification system for spinal MNs and paves the way towards deciphering how neuronal subtypes are evolved.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462347', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Characterization of morpho-functional traits in mesophotic corals reveals optimized light capture and photosynthesis', 'authors': \"[{'name': 'Netanel Kramer'}, {'name': 'Jiaao Guan'}, {'name': 'Shaochen Chen'}, {'name': 'Daniel Wangpraseurt'}, {'name': 'Yossi Loya'}]\", 'abstract': '<p>The morphology and skeleton architecture of photosynthetic corals modulates the light capture and functioning of the coral-algal symbiosis on shallow-water corals. Since corals can thrive on mesophotic reefs under extreme light-limited conditions, we hypothesized that microskeletal coral features optimize light capture under low-light environments. Using micro-computed tomography scanning, we conducted a comprehensive three-dimensional (3D) assessment of small-scale skeleton morphology of the depth-generalist coral <italic>Stylophora pistillata</italic> collected from shallow (5 m) and mesophotic (45 m) depths. We detected a high phenotypic diversity between depths, resulting in two distinct morphotypes, with calyx diameter, theca height, and corallite marginal spacing contributing to most of the variation between depths. To determine whether such depth-specific morphotypes affect coral light capture and photosynthesis on the corallite-scale, we developed 3D simulations of light propagation based on photosynthesis-irradiance parameters. We found that corals associated with shallow morphotypes dissipated excess light through self-shading microskeletal features; while mesophotic morphotypes facilitated enhanced light absorption and photosynthesis under low-light conditions. We conclude that the mesophotic coral architecture provides a greater ability to trap solar energy and efficiently exploit the limited light conditions, and suggest that morphological modifications play a key role in the photoadaptation response to low-light.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462361', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Higher-order olfactory neurons in the lateral horn supports odor valence and odor identity coding in Drosophila', 'authors': \"[{'name': 'Sudeshna Das Chakraborty'}, {'name': 'Hetan Chang'}, {'name': 'Bill S. Hansson'}, {'name': 'Silke Sachse'}]\", 'abstract': '<p>Understanding neuronal representations of odor-evoked activities and their progressive transformation from the sensory level to higher brain centers features one of the major aims in olfactory neuroscience. Here, we investigated how odor information is transformed and represented in higher-order neurons of the lateral horn, one of the higher olfactory centers implicated in determining innate behavior, using <italic>Drosophila melanogaster</italic>. We focused on a subset of third-order glutamatergic lateral horn neurons (LHNs) and characterized their odor coding properties in relation to their presynaptic partner neurons, the projection neurons (PNs) by two-photon functional imaging. We found that odors evoke reproducible, stereotypic and odor-specific response patterns in LHNs. Notably, odor-evoked responses in these neurons are valence-specific in a way that their response amplitude is positively correlated with innate odor preferences. We postulate that this valence-specific activity is the result of integrating inputs from multiple olfactory channels through second-order neurons. GRASP and micro-lesioning experiments provide evidence that glutamatergic LHNs obtain their major excitatory input from uniglomerular PNs, while they receive an odor-specific inhibition through inhibitory multiglomerular PNs. In summary, our study indicates that odor representations in glutamatergic LHNs encode hedonic valence and odor identity and primarily retain the odor coding properties of second-order neurons.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.462120', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Real-time decoding of 5 finger movements from 2 EMG channels for mixed reality human-computer interaction', 'authors': \"[{'name': 'Eric J. McDermott'}, {'name': 'Thimm Zwiener'}, {'name': 'Ulf Ziemann'}, {'name': 'Christoph Zrenner'}]\", 'abstract': '<p>The search for optimized forms of human-computer interaction (HCI) has intensified alongside the growing potential for the combination of biosignals with virtual reality (VR) and augmented reality (AR) to enable the next generation of personal computing. At the core, this requires decoding the user’s biosignals into digital commands. Electromyography (EMG) is a biosensor of particular interest due to the ease of data collection, the relatively high signal-to-noise-ratio, its non-invasiveness, and the ability to interpret the signal as being generated by (intentional) muscle activity. Here, we investigate the potential of using data taken from a simple 2-channel EMG setup to differentiate 5 distinct movements. In particular, EMG was recorded from two bipolar sensors over small hand muscles (<italic>extensor digitorum</italic>, <italic>flexor digitorum profundus</italic>) while a subject performed 50 trials of dorsal extension and return for each of the five digits. The maximum and the mean values across the trial were determined for each channel and used as features. A k-nearest neighbors (kNN) classification was performed and overall 5-class classification accuracy reached 94% when using the full trial’s time window, while simulated real-time classification reached 90.4% accuracy when using the constructed kNN model (k=3) with a 280ms sliding window. Additionally, unsupervised learning was performed and a homogeneity of 85% was achieved. This study demonstrates that reliable decoding of different natural movements is possible with fewer than one channel per class, even without taking into account temporal features of the signal. The technical feasibility of this approach in a real-time setting was validated by sending real-time EMG data to a custom Unity3D VR application through a Lab Streaming Layer to control a user interface. Further use-cases of gamification and rehabilitation were also examined alongside the integration of eye-tracking and gesture recognition for a sensor fusion approach to HCI and user intent.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462124', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Neuronal glucose metabolism sets cholinergic tone and controls thermo-regulated signaling at the neuromuscular junction', 'authors': \"[{'name': 'Yan Tang'}, {'name': 'Haihong Zong'}, {'name': 'Hyokjoon Kwon'}, {'name': 'Yunping Qiu'}, {'name': 'Jacob B. Pessin'}, {'name': 'Licheng Wu'}, {'name': 'Katherine A. Buddo'}, {'name': 'Ilya Boykov'}, {'name': 'Cameron A. Schmidt'}, {'name': 'Chien-Te Lin'}, {'name': 'P. Darrell Neufer'}, {'name': 'Gary J. Schwartz'}, {'name': 'Irwin J. Kurland'}, {'name': 'Jeffrey E. Pessin'}]\", 'abstract': '<p>Cholinergic and sympathetic counter-regulatory networks control numerous physiologic functions including learning/memory/cognition, stress responsiveness, blood pressure, heart rate and energy balance. As neurons primarily utilize glucose as their primary metabolic energy source, we generated mice with increased glycolysis in cholinergic neurons by specific deletion of the fructose-2,6-phosphatase protein TIGAR. Steady-state and stable isotope flux analyses demonstrated increased rates of glycolysis, acetyl-CoA production, acetylcholine levels and density of neuromuscular synaptic junction clusters with enhanced acetylcholine release. The increase in cholinergic signaling reduced blood pressure and heart rate with a remarkable resistance to cold-induced hypothermia. These data directly demonstrate that increased cholinergic signaling through the modulation of glycolysis has several metabolic benefits particularly to increase energy expenditure and heat production upon cold exposure.</p><sec><title>Highlights</title><list list-type=\"order\"><list-item><p>Deficiency of a negative regulator of glycolysis (TIGAR) in cholinergic neurons increases the biosynthesis and content of the neurotransmitter acetylcholine.</p></list-item><list-item><p>Increased cholinergic tone reduces blood pressure and heart rate while enhancing signaling at neuromuscular junction.</p></list-item><list-item><p>Upregulation of neuromuscular junction activation provides protection against the paralytic curare and cold-induced hypothermia.</p></list-item><list-item><p>Modulation of cholinergic neuron glycolysis may provide a novel therapeutic approach for treatment of diseases stemming from reduced acetylcholine signaling such as myasthenia gravis and sarcopenic pre-synaptic dysfunction.</p></list-item></list></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462143', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Hippocampal-prefrontal theta coupling develops as mice become proficient in associative odorant discrimination learning', 'authors': \"[{'name': 'Daniel Ramirez-Gordillo'}, {'name': 'Andrew A. Parra'}, {'name': 'K. Ulrich Bayer'}, {'name': 'Diego Restrepo'}]\", 'abstract': '<p>Learning and memory requires coordinated activity between different regions of the brain. Here we studied the interaction between medial prefrontal cortex (mPFC) and hippocampal dorsal CA1 during associative odorant discrimination learning in the mouse. We found that as the animal learns to discriminate odorants in a go-no go task the coupling of high frequency neural oscillations to the phase of theta oscillations (phase-amplitude coupling or PAC) changes in a manner that results in divergence between rewarded and unrewarded odorant-elicited changes in the theta-phase referenced power (tPRP) for beta and gamma oscillations. In addition, in the proficient animal there was a decrease in the coordinated oscillatory activity between CA1 and mPFC in the presence of the unrewarded odorant. Furthermore, the changes in PAC resulted in a marked increase in the accuracy for decoding odorant identity from tPRP when the animal became proficient. Finally, we studied the role of Ca<sup>2+</sup>/calmodulin-dependent protein kinase II α (CaMKIIα), a protein involved in learning and memory, in oscillatory neural processing in this task. We find that the accuracy for decoding the odorant identity from tPRP decreases in CaMKIIα knockout mice and that this accuracy correlates with behavioral performance. These results implicate a role for PAC and CaMKIIα in olfactory go-no go associative learning in the hippocampal-prefrontal circuit.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.21264126', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Allergy and Immunology', 'covid': True, 'title': 'Efficacy of COVID-19 vaccines in immunocompromised patients: A systematic review and meta-analysis', 'authors': \"[{'name': 'Ainsley Ryan Yan Bin Lee'}, {'name': 'Shi Yin Wong'}, {'name': 'Louis Yi Ann Chai'}, {'name': 'Soo Chin Lee'}, {'name': 'Matilda Lee'}, {'name': 'Mark Dhinesh Muthiah'}, {'name': 'Sen Hee Tay'}, {'name': 'Chong Boon Teo'}, {'name': 'Benjamin Kye Jyn Tan'}, {'name': 'Yiong Huak Chan'}, {'name': 'Raghav Sundar'}, {'name': 'Yu Yang Soon'}]\", 'abstract': '<sec><title>Objective</title><p>To compare the efficacy of COVID 19 vaccines between those with immunocompromised medical conditions and those who are immunocompetent</p></sec><sec><title>Design</title><p>Systematic review and meta-analysis</p></sec><sec><title>Data sources</title><p>PubMed, EMBASE, CENTRAL, CORD-19 and WHO COVID-19 research databases were searched for eligible comparative studies published between 1 December 2020 and 3 September 2021. ClinicalTrials.gov and the WHO International Clinical Trials Registry Platform were searched in September 2021 to identify registered yet unpublished or ongoing studies.</p></sec><sec><title>Study selection</title><p>Prospective observational studies which compared the efficacy of COVID-19 vaccination between those with immunocompromising medical conditions and those who were immunocompetent were included. Two reviewers independently screened for potentially eligible studies.</p></sec><sec><title>Data extraction</title><p>The primary outcomes of interest were cumulative incidence of seroconversion after first and second doses of COVID vaccination. Secondary outcomes included SARS-CoV-2 antibody titre level after first and second doses of COVID-19 vaccination. After duplicate data abstraction, a frequentist random effects meta-analysis was conducted. Risk of bias was assessed using the ROBINS-I tool. Certainty of evidence was assessed using the GRADE approach.</p></sec><sec><title>Results</title><p>After screening 3283 studies, 42 studies that met our inclusion criteria were identified. 18 immunocompromised cohorts from 17 studies reported seroconversion in immunocompromised patients compared to healthy controls after the first dose and 30 immunocompromised cohorts in 28 studies reporting data after the second dose.</p><p>Among immunocompromised groups, in incremental order, transplant recipients had the lowest pooled risk ratio of 0.06 (95%CI: 0.04 to 0.09, I^2=0%, p=0.81) (GRADE=Moderate) followed by haematological cancer patients at 0.36 (95%CI: 0.21 to 0.62, I^2 = 89%, p&lt;0.01) (GRADE=Moderate), solid cancer patients at 0.40 (95%CI: 0.31 to 0.52, I^2 = 63%, p=0.03) (GRADE=Moderate) and IMID patients at 0.66 (95%CI: 0.48 to 0.91, I^2=81%, p&lt;0.01) (GRADE=Moderate).</p><p>After the second dose, the lowest pooled risk ratio was again seen in transplant recipients at 0.29 (95%CI: 0.21 to 0.40, I^2=91%, p&lt;0.01) (GRADE=Moderate), haematological cancer patients at 0.68 (95%CI: 0.57 to 0.80, I^2=68%, p=0.02) (GRADE=Low), IMID patients at 0.79 (95%CI: 0.72 to 0.86, I^2=87%, p&lt;0.01) (GRADE=Low) and solid cancer at 0.92 (95%CI: 0.89 to 0.95, I^2=26%, p=0.25) (GRADE=Low).</p></sec><sec><title>Conclusion</title><p>Seroconversion rates and serological titres are significantly lower in immunocompromised patients with transplant recipients having the poorest outcomes. Additional strategies on top of the conventional 2-dose regimen will likely be warranted, such as a booster dose of the vaccine.</p></sec><sec><title>Systematic review registration</title><p>PROSPERO CRD42021272088</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.462062', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Experimental Studies of Carpenter Bee (Xylocopa: Apidae) Thorax Mechanics During Defensive Buzzing', 'authors': \"[{'name': 'Mark Jankauski'}, {'name': 'Cailin Casey'}, {'name': 'Chelsea Heveran'}, {'name': 'Kathryn Busby'}, {'name': 'Stephen Buchmann'}]\", 'abstract': '<p>Bees and other Hymenoptera utilize thorax vibration to realize an extensive range of behaviors ranging from flight to pollination. Strong indirect flight muscles contract to deform the thoracic walls and the resulting oscillation is sustained through a mechanism called stretch activation. While the mechanics of the insect thorax and muscles have been studied extensively during flight, relatively little is known about the thorax mechanics during non-flight behaviors. In this work, we investigate the thorax mechanics of the carpenter bee <italic>Xylocopa californica</italic> during defensive buzzing. During defensive buzzing, the insect folds its wings over its abdomen and rapidly fires it flight muscles, resulting in a loud audible buzz and large forces intended to deter predators. We devised a novel experiment to measure thorax oscillation and directional force production from a defensively buzzing carpenter bee. The largest peak forces were on average 175 mN and were oriented with the insect’s dorsal-ventral muscle group. Peak forces oriented with the insect’s dorsal-longitudinal muscle group averaged 117 mN. Thorax velocities were about 90 mm s^-1 p-p and velocity amplitude was positively correlated to peak force. Thorax oscillation frequency averaged 132 Hz but was highly variable both within individuals and across the tested population. From our measurements, we estimated the peak mechanical power required by defensive buzzing at 8.7 mW, which we hypothesize is greater than the power required during flight. Overall, this study provides insight into the function and capabilities of the Hymenopteran indirect flight muscle during non-flight behaviors.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.461909', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': True, 'title': 'Face masks impair reconstruction of acoustic speech features and higher-level segmentational features in the presence of a distractor speaker', 'authors': \"[{'name': 'Chandra Leon Haider'}, {'name': 'Nina Suess'}, {'name': 'Anne Hauswald'}, {'name': 'Hyojin Park'}, {'name': 'Nathan Weisz'}]\", 'abstract': '<p>Face masks have become a prevalent measure during the Covid-19 pandemic to counteract the transmission of SARS-CoV 2. An unintended “side-effect” of face masks is their adverse influence on speech perception especially in challenging listening situations. So far, behavioural studies have not pinpointed exactly which feature(s) of speech processing face masks affect in such listening situations. We conducted an audiovisual (AV) multi-speaker experiment using naturalistic speech (i.e. an audiobook). In half of the trials, the target speaker wore a (surgical) face mask, while we measured the brain activity of normal hearing participants via magnetoencephalography (MEG). A decoding model on the clear AV speech (i.e. no additional speaker and target speaker not wearing a face mask) was trained and used to reconstruct crucial speech features in each condition. We found significant main effects of face masks on the reconstruction of acoustic features, such as the speech envelope and spectral speech features (i.e. pitch and formant frequencies), while reconstruction of higher level features of speech segmentation (phoneme and word onsets) were especially impaired through masks in difficult listening situations, i.e. when a distracting speaker was also presented. Our findings demonstrate the detrimental impact face masks have on listening and speech perception, thus extending previous behavioural results. Supporting the idea of visual facilitation of speech is the fact that we used surgical face masks in our study, which only show mild effects on speech acoustics. This idea is in line with recent research, also by our group, showing that visual cortical regions track spectral modulations. Since hearing impairment usually affects higher frequencies, the detrimental effect of face masks might pose a particular challenge for individuals who likely need the visual information about higher frequencies (e.g. formants) to compensate.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462164', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'A yeast with muscle doesn’t run faster: full humanization of the glycolytic pathway in Saccharomyces cerevisiae', 'authors': \"[{'name': 'Francine J. Boonekamp'}, {'name': 'Ewout Knibbe'}, {'name': 'Marcel A. Vieira-Lara'}, {'name': 'Melanie Wijsman'}, {'name': 'Marijke A.H. Luttik'}, {'name': 'Karen van Eunen'}, {'name': 'Maxime den Ridder'}, {'name': 'Reinier Bron'}, {'name': 'Ana Maria Almonacid Suarez'}, {'name': 'Patrick van Rijn'}, {'name': 'Justina C. Wolters'}, {'name': 'Martin Pabst'}, {'name': 'Jean-Marc Daran'}, {'name': 'Barbara Bakker'}, {'name': 'Pascale Daran-Lapujade'}]\", 'abstract': '<p>While transplantation of single genes in yeast plays a key role in elucidating gene functionality in metazoans, technical challenges hamper the humanization of full pathways and processes. Empowered by advances in synthetic biology, this study demonstrates the feasibility and implementation of full humanization of glycolysis in yeast. Single gene and full pathway transplantation revealed the remarkable conservation of both glycolytic and moonlighting functions and, combined with evolutionary strategies, brought to light novel, context-dependent responses. Remarkably, human hexokinase 1 and 2, but not 4, required mutations in their catalytic or allosteric sites for functionality in yeast, while hexokinase 3 was unable to complement its yeast ortholog. Comparison with human tissues cultures showed the preservation of turnover numbers of human glycolytic enzymes in yeast and human cell cultures. This demonstration of transplantation of an entire, essential pathway paves the way to the establishment of species, tissue and disease-specific metazoan models.</p><sec><title>One Sentence Summary</title><p>This work demonstrates the successful humanization of an entire pathway in <italic>Saccharomyces cerevisiae</italic> and establishes an attractive strategy to study (human) glycolysis architecture and regulation.</p></sec><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>The successful humanization of the entire glycolytic pathway in yeast offers new microbial models for both fundamental and applied studies.</p></list-item><list-item><p>Both glycolytic and moonlighting functions and turnover numbers of glycolytic enzymes are highly conserved between yeast and human.</p></list-item><list-item><p>Functionality of human hexokinases 1 and 2 in yeast requires mutations in the catalytic or allosteric binding sites.</p></list-item><list-item><p>Combination of single gene and full transplantation with laboratory evolution reveals context-dependent activity and evolution of glycolytic enzymes.</p></list-item></list></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462156', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Modulation of immunosuppressant drug treatment to improve SARS-CoV-2 vaccine efficacy in mice', 'authors': \"[{'name': 'Amy V. Paschall'}, {'name': 'Ahmet Ozdilek'}, {'name': 'Sydney L. Briner'}, {'name': 'Melinda A. Brindley'}, {'name': 'Fikri Y. Avci'}]\", 'abstract': '<p>The COVID-19 pandemic dramatically demonstrated the need for improved vaccination strategies and therapeutic responses to combat infectious diseases. However, the efficacy of vaccines has not yet been demonstrated in combination with commonly used immunosuppressive drug regimens. We sought to determine how common pharmaceutical drugs used in autoimmune disorders can alter immune responses to the SARS-CoV-2 spike protein vaccination.</p><p>We treated mice with five immunosuppressant drugs (cyclophosphamide, leflunomide, methotrexate, methylprednisolone, and mycophenolate mofetil), each with various mechanisms of action prior to and following immunization with SARS-CoV-2 spike protein. We assessed the functionality of antibody responses to spike protein and compared immune cell populations in mice that received no treatment with those that received continuous or temporarily suspended immune suppressive therapy.</p><p>All tested immunosuppressants significantly reduced the antibody titers in serum and functional antibody response against SARS-CoV-2 spike protein in immunized mice. Temporarily halting selected immunosuppressants (methylprednisolone and methotrexate, but not cyclophosphamide) improved antibody responses significantly. Through proof-of-principle experiments utilizing a mouse model, we demonstrated that immune suppression in autoimmune disorders through pharmaceutical treatments may impair vaccine response to SARS-CoV-2, and temporary suspension of immunosuppressant treatment may be necessary to mount an effective antibody vaccine response. This work provides feasibility for future clinical assessment of the impact of immunosuppressants on vaccine efficacy in humans.</p><sec><title>Significance Statement</title><p>Immunosuppressant regimens are widely used as therapies for a variety of diseases, including autoimmune, inflammatory, and cancer. However, immunosuppressants can impair critical immune responses to vaccination. The impact of standard immunosuppressant use on the critical, developing SARS-CoV-2 vaccination strategies has not been well-described. In this study, we use a mouse model to determine how different immunosuppressant drugs that act through different mechanisms can impair the antibody response to SARS-CoV-2 spike protein, and how modulating these drug regimens may restore antibody levels and function.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.10.01.21264368', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': False, 'title': '“Jittered” Oculomotor Sensing Concomitant with Hypoactive Dorsal Parietal Systems during Live Eye-Contact in Autism Spectrum Disorder', 'authors': \"[{'name': 'Joy Hirsch'}, {'name': 'Xian Zhang'}, {'name': 'J. Adam Noah'}, {'name': 'Swethasri Dravida'}, {'name': 'Adam Naples'}, {'name': 'Mark Tiede'}, {'name': 'Julie M. Wolf'}, {'name': 'James C. McPartland'}]\", 'abstract': '<p>Reluctant eye contact and reduced social interactions characteristic of autism spectrum disorder (ASD) are consistent with deficits in oculomotor and face processing systems. We test the hypothesis that these deficits are interrelated ASD.</p><p>Eye-tracking and hyperscanning with functional near-infrared spectroscopy (fNIRS) were used to acquire neuroimaging data during live, dynamic eye-to-eye contacts in 17 ASD and 19 typically-developed (TD) adults. Real eye contact conditions were contrasted with conditions where eye gaze was directed at a comparable dynamic face video. These findings were regressed with eye-contact dwell-time, i.e., times when gaze of both partners was in the eye-box of the other, to confirm the relationship between visual sensing and neural coding. Visual fixations and positional variance were also determined.</p><p>Average gaze dwell-times in the “eye-box” did not vary between ASD and TD participants but were longer for the Real Eye than Video Eye condition for both groups. However, positional gaze variability, “jitter”, was higher for ASD in both conditions. Neural findings for TD [Real Eye &gt; Video Eye] were consistent with previous findings for interactive face-gaze with activity in right temporal and dorsal parietal regions. However, in ASD ventral temporal regions were observed for this contrast without evidence for dorsal parietal activity. This neural difference was enhanced when regressed by eye-contact dwell-times.</p><p>Together findings are consistent with the hypothesis that unstable bottom-up oculomotor signals contribute to deficits in live face processing and reduced dorsal stream activity in ASD.</p><sec><title>Significance</title><p>Eye contact avoidance is a diagnostic feature of autism spectrum disorder (ASD). However, the underlying neural mechanisms are unknown. Using a two-person neural imaging system and a face-to-face paradigm with eye tracking, we found that the neural systems activated by live eye contact differed between typically developed (TD) and ASD groups. In ASD, the ventral occipital parietal systems were engaged, whereas in TD, the dorsal posterior parietal systems were engaged. Positional variation of eye gaze, “jitter”, was higher in ASD than TD. These findings are consistent with the hypothesis that bottom-up variations in visual sensing are components of altered interactive face processing and dorsal stream mechanisms and may contribute to the neural underpinnings of reluctant eye contact behaviors in ASD.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.21264298', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'A HOME-TREATMENT ALGORITHM BASED ON ANTI-INFLAMMATORY DRUGS TO PREVENT HOSPITALIZATION OF PATIENTS WITH EARLY COVID-19: A MATCHED-COHORT STUDY (COVER 2)', 'authors': \"[{'name': 'Elena Consolaro'}, {'name': 'Fredy Suter'}, {'name': 'Nadia Rubis'}, {'name': 'Stefania Pedroni'}, {'name': 'Chiara Moroni'}, {'name': 'Elena Pastò'}, {'name': 'Maria Vittoria Paganini'}, {'name': 'Grazia Pravettoni'}, {'name': 'Umberto Cantarelli'}, {'name': 'Norberto Perico'}, {'name': 'Annalisa Perna'}, {'name': 'Tobia Peracchi'}, {'name': 'Piero Ruggenenti'}, {'name': 'Giuseppe Remuzzi'}]\", 'abstract': '<sec><title>Background and Aim</title><p>While considerable success has been achieved in the management of patients hospitalized with severe coronavirus disease 2019 (COVID-19), far less progress has been made with early outpatient treatment. We assessed whether the implementation of a home treatment algorithm – designed based upon on a pathophysiologic and pharmacologic rationale - during the initial, mild phase of COVID-19, could effectively reduce hospital admissions.</p></sec><sec><title>Methods</title><p>This fully academic, matched-cohort study evaluated outcomes in 108 consecutive consenting patients with mild COVID-19 managed at home by their family doctors from January 2021 to May 2021, according to the proposed treatment algorithm and in 108 age-, sex-, and comorbidities-matched patients who were given other therapeutic schedules (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://ClinicalTrials.gov\">ClinicalTrials.gov</ext-link>: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04854824\">NCT04854824</ext-link>). The primary outcome was COVID-19-related hospitalization. Analyses were by intention-to-treat.</p></sec><sec><title>Results</title><p>One (0.9%) patient in the ‘recommended’ cohort and 12 (11.1%) in the ‘control’ cohort were admitted to hospital (P=0.0136). The proposed algorithm reduced, by 85%, the cumulative length of hospital stays (from 141 to 19 days) and related costs (from € 60.316 to € 9.058). Only 9.8 patients needed to be treated with the recommended algorithm to prevent one hospitalization event. The rate of resolution of major symptoms was numerically, but not significantly, higher in the ‘recommended’ compared to the ‘control’ cohort (97.2% versus 93.5%, respectively; P=0.322). Other symptoms lingered in a lower proportion of patients in the ‘recommended’ than in the‘control’ cohort (20.4% versus 63.9%, respectively; P&lt;0.001), and for a shorter period.</p></sec><sec><title>Conclusion</title><p>The adoption of the proposed outpatient treatment algorithm during the early, mild phase of COVID-19 reduced the incidence of subsequent hospitalization and related costs.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462733', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A novel locally c-di-GMP-controlled exopolysaccharide synthase required for N4 phage infection of E. coli', 'authors': \"[{'name': 'Eike H. Junkermeier'}, {'name': 'Regine Hengge'}]\", 'abstract': '<p>A major target of c-di-GMP signaling is the production of biofilm-associated extracellular polymeric substances (EPS), which in <italic>Escherichia coli</italic> K-12 include amyloid curli fibres, phosphoethanolamine-modified (pEtN-)cellulose and poly-N-acetyl-glucosamine (PGA). However, the characterized c-di-GMP-binding effector systems are largely outnumbered by the 12 diguanylate cyclases (DGCs) and 13 phosphodiesterases (PDEs), which synthetize and degrade c-di-GMP, respectively. <italic>E. coli</italic> possesses a single protein with a potentially c-di-GMP-binding MshEN domain, NfrB, which – together with the outer membrane protein NfrA – is known to serve as a receptor system for phage N4. Here, we show that NfrB not only binds c-di-GMP with high affinity, but as a novel c-di-GMP-controlled glycosyltransferase synthesizes a secreted EPS, which can impede motility and is required as an initial receptor for phage N4 infection. In addition, a systematic screening of the 12 DGCs of <italic>E. coli</italic> K-12 revealed that specifically DgcJ is required for the infection with phage N4 and interacts directly with NfrB. This is in line with local signaling models, where specific DGCs and/or PDEs form protein complexes with particular c-di-GMP effector/target systems. Our findings thus provide further evidence that intracellular signaling pathways, which all use the same diffusible second messenger, can act in parallel in a highly specific manner.</p><sec><title>Importance</title><p>Key findings in model organisms led to the concept of ‘local’ signaling, challenging the dogma of a gradually increasing global intracellular c-di-GMP concentration driving the motile-sessile transition in bacteria. In our current model, bacteria dynamically combine global as well as local signaling modes, in which specific DGCs and/or PDEs team up with effector/target systems in multiprotein complexes. Our present study highlights a novel example of how specificity in c-di-GMP signaling can be achieved by showing NfrB as a novel c-di-GMP binding effector in <italic>E. coli</italic>, which is controlled in a local manner specifically by DgcJ. We further show that NfrB (which was initially found as a part of a receptor system for phage N4) is involved in the production of a novel exopolysaccharide. Finally, our data shine new light on host interaction of phage N4, which uses this exopolysaccharide as an initial receptor for adsorption.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.21264312', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'A cross-sectional study of COVID-19 knowledge, beliefs and prevention behaviors among adults in Senegal', 'authors': \"[{'name': 'Matthew D. Kearney'}, {'name': 'Marta Bornstein'}, {'name': 'Marieme Fall'}, {'name': 'Roch Nianogo'}, {'name': 'Deborah Glik'}, {'name': 'Philip M. Massey'}]\", 'abstract': '<sec><title>Background</title><p>COVID-19 is an ongoing threat to global public health since its emergence at the end of 2019, yet little is known about how populations in francophone West Africa have responded to COVID-19 in their daily lives. Senegal, in particular, has been noted for its relative success in mitigating the spread and impact of COVID-19. We report original research findings on COVID-19 beliefs and prevention behaviors in a sample of Senegalese adults.</p></sec><sec><title>Methods</title><p>A multi-modal cross-sectional study was conducted to describe COVID-19 beliefs and prevention behaviors in a sample of Senegalese adults and to identify potential predictors of prevention behaviors. Univariate, bivariate, and multivariate statistics were generated to describe the sample and explore potential correlations.</p></sec><sec><title>Results</title><p>Mask wearing, hand washing, and use of hand sanitizer were most frequently reported. Social distancing and staying at home were also reported albeit to a lower degree. We also identified a range of psychosocial and demographic predictors for COVID-19 prevention behaviors. Men, compared to women, had lower odds (OR=0.59) of reporting prevention behaviors. Rural residents (vs. urban; OR=1.49) and participants with at least a high school education (vs. less than high school education; OR=1.33) were more likely to report COVID-19 prevention behaviors.</p></sec><sec><title>Discussion</title><p>Stakeholders and decision makers in Senegal and across Africa can use place-based evidence like ours to address COVID-19 risk factors and intervene effectively with policies and programming. Use of both phone and online surveys enhances representation and study generalizability and should be considered in future research with hard-to-reach populations.</p></sec><sec><title>Article Summary</title><sec><title>Strengths and limitations of this study</title><list list-type=\"bullet\"><list-item><p>The main strength of our study is the use of a multi-modal data collection strategy, online and via telephone. Had we relied on a single method, our sample’s demographic characteristics would likely have differed while also introducing selection bias.</p></list-item><list-item><p>Our recruitment strategy may have also increased the potential for selection bias because participants were recruited online and on-the-ground in Senegal; thus, all participants, by the nature of the recruitment methods, had access to the internet and/or a cell phone. To address potential confounding between recruitment methods, we controlled for recruitment method in our multivariate regression modelling.</p></list-item><list-item><p>Prior to our study, little evidence was available about how populations in francophone West Africa have responded to COVID-19 in their daily lives. We adapted pre-existing research panels and developed novel data collection instruments to capture information about COVID-19, and this may serve as a model for the responsiveness of ongoing scholarship to future global events.</p></list-item><list-item><p>A study with a larger sample may have been able to identify relationships between knowledge and behaviors, and our study was not design with enough statistical power to detect significant differences.</p></list-item><list-item><p>The findings of our study may not be generalizable beyond Senegal, or more specifically Senegalese adults.</p></list-item></list></sec></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.10.01.462769', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Mechanical loading and hyperosmolarity as a daily resetting cue for skeletal circadian clocks', 'authors': \"[{'name': 'Michal Dudek'}, {'name': 'Dharshika Pathiranage'}, {'name': 'Cátia F. Gonçalves'}, {'name': 'Craig Lawless'}, {'name': 'Dong Wang'}, {'name': 'Zhuojing Luo'}, {'name': 'Liu Yang'}, {'name': 'Judith Hoyland'}, {'name': 'Qing-Jun Meng'}]\", 'abstract': '<p>In mammals, temporally coordinated daily rhythms of behaviour and physiology are generated by a multi-oscillatory circadian system, entrained through cyclic environmental cues (e.g. light). Presence of niche-dependent physiological time cues has been proposed, which would allow local tissues flexibility of adopting a different phase relationship if circumstances require. Up till now, such tissue-unique stimuli have remained elusive. Here we show that cycles of mechanical loading and osmotic stimuli within physiological range drive rhythmic expression of clock genes and reset clock phase and amplitude in cartilage and intervertebral disc tissues. Hyperosmolarity (and not hypo-osmolarity) resets clocks in young and ageing skeletal tissues through mTORC2-AKT-GSK3β pathway, leading to genome-wide induction of rhythmic genes. These results suggest diurnal patterns of mechanical loading and consequent daily surges in extracellular osmolarity as a bona fide tissue niche-specific time cue to maintain skeletal circadian rhythms in sync.</p><sec><title>One-Sentence Summary</title><p>Circadian clocks in aneural skeletal tissues sense the passage of time through rhythmic patterns of loading and osmolarity.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462459', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Structural mapping of antibody landscapes to human betacoronavirus spike proteins', 'authors': \"[{'name': 'Sandhya Bangaru'}, {'name': 'Aleksandar Antanasijevic'}, {'name': 'Nurgun Kose'}, {'name': 'Leigh M. Sewall'}, {'name': 'Abigail M. Jackson'}, {'name': 'Naveenchandra Suryadevara'}, {'name': 'Xiaoyan Zhan'}, {'name': 'Jonathan L. Torres'}, {'name': 'Jeffrey Copps'}, {'name': 'Alba Torrents de la Peña'}, {'name': 'James E. Crowe'}, {'name': 'Andrew B. Ward'}]\", 'abstract': '<p>Preexisting immunity against seasonal coronaviruses (CoV) represents an important variable in predicting antibody responses and disease severity to Severe Acute Respiratory Syndrome CoV-2 (SARS-2) infections. We used electron microscopy based polyclonal epitope mapping (EMPEM) to characterize the antibody specificities against β-CoV spike proteins in sera from healthy donors (HDs) or SARS-2 convalescent donors (CDs). We observed that most HDs possessed antibodies specific to seasonal human CoVs (HCoVs) OC43 and HKU1 spike proteins while the CDs showed reactivity across all human β-CoVs. Detailed molecular mapping of spike-antibody complexes revealed epitopes that were differentially targeted by antibodies in preexisting and convalescent serum. Our studies provide an antigenic landscape to β-HCoV spikes in the general population serving as a basis for cross-reactive epitope analyses in SARS-2 -infected individuals.</p><sec><title>One-Sentence summary</title><p>We present the epitope mapping of polyclonal antibodies against beta-coronavirus spike proteins in human sera.</p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462800', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Molecular structure and conformation of stereocilia tip-links elucidated by cryo-electron tomography', 'authors': \"[{'name': 'Johannes Elferich'}, {'name': 'Sarah Clark'}, {'name': 'Jingpeng Ge'}, {'name': 'April Goehring'}, {'name': 'Aya Matsui'}, {'name': 'Eric Gouaux'}]\", 'abstract': '<p>Mechanosensory transduction (MT), the conversion of mechanical stimuli into electrical signals, underpins hearing and balance and is carried out within hair cells in the inner ear. Hair cells harbor actin-filled stereocilia, arranged in rows of descending heights, where the tips of stereocilia are connected to their taller neighbors by a filament composed of protocadherin 15 (PCDH15) and cadherin 23 (CDH23), deemed the ‘tip-link’. Tension exerted on the tip-link opens an ion channel at the tip of the shorter stereocilia, thus converting mechanical force into an electrical signal. While biochemical and structural studies have provided insights into the molecular composition and structure of isolated portions of the tip-link, the architecture, location and conformational states of intact tip-links, on stereocilia, remains unknown. Here we report <italic>in situ</italic> cryo-electron microscopy imaging of the tip-link in mouse stereocilia. We observe individual PCDH15 molecules at the tip and shaft of stereocilia and determine their stoichiometry, conformational heterogeneity, and their complexes with CDH23. The PCDH15/CDH23 complexes occur in clusters, frequently with more than one copy of PCDH15 at the tip of stereocilia, suggesting that tip-links might consist of more than one copy of the PCDH15/CDH23 heterotetramer and by extension, might include multiple MT complexes.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462803', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Improving the global dimensions of intrinsically disordered proteins in Martini 3', 'authors': \"[{'name': 'F. Emil Thomasen'}, {'name': 'Francesco Pesce'}, {'name': 'Mette Ahrensback Roesgaard'}, {'name': 'Giulio Tesei'}, {'name': 'Kresten Lindorff-Larsen'}]\", 'abstract': '<p>Coarse-grained molecular dynamics simulations are a useful tool to determine conformational ensembles of intrinsically disordered proteins (IDPs). Here, we show that the coarse-grained force field Martini 3 underestimates the global dimensions of IDPs when compared with small angle X-ray scattering (SAXS) data. Increasing the strength of protein-water interactions favors more expanded conformations, improving agreement with SAXS data and alleviating problems with overestimated IDP-IDP interactions.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462262', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Open Imputation Server provides secure Imputation services with provable genomic privacy', 'authors': \"[{'name': 'Arif O. Harmanci'}, {'name': 'Miran Kim'}, {'name': 'Su Wang'}, {'name': 'Wentao Li'}, {'name': 'Yongsoo Song'}, {'name': 'Kristin E. Lauter'}, {'name': 'Xiaoqian Jiang'}]\", 'abstract': '<sec><title>Summary</title><p>As DNA sequencing data is available for personal use, genomic privacy is becoming a major challenge. Nevertheless, high-throughput genomic data analysis outsourcing is performed using pipelines that tend to overlook these challenges.</p></sec><sec><title>Results</title><p>We present a client-server-based outsourcing framework for genotype imputation, an important step in genomic data analyses. Genotype data is encrypted by the client and encrypted data are used by the server that never observes the data in plain. Cloud-based framework can benefit from virtually unlimited computational resources while providing provable confidentiality. We demonstrate server’s utility from several aspects using genotype dataset from the 1000 Genomes datasets. First, we benchmark the accuracy of common variant imputation in comparison to BEAGLE, a state-of-the-art imputation method. We also provide the detailed time requirements of the server to showcase scaling of time usage in different steps of imputation. We also present a simple correlation metric that can be used to estimate imputation accuracy using only the reference panels. This is important for filtering the variants in downstream analyses. As a further demonstration and a different use case, we performed a simulated genomewide association study (GWAS) using imputed and known genotypes and highlight potential utility of the server for association studies. Overall, our study present multiple lines of evidence for usability of secure imputation service.</p></sec><sec><title>Availability</title><p>Server is publicly available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://www.secureomics.org/OpenImpute\">https://www.secureomics.org/OpenImpute</ext-link>. Users can anonymously test and use imputation server without registration.</p></sec><sec><title>Contact</title><p><email>Arif.O.Harmanci@uth.tmc.edu</email></p></sec>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462724', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Whale shark rhodopsin adapted to its vertically wide-ranging lifestyle', 'authors': \"[{'name': 'Kazuaki Yamaguchi'}, {'name': 'Mitsumasa Koyanagi'}, {'name': 'Keiichi Sato'}, {'name': 'Akihisa Terakita'}, {'name': 'Shigehiro Kuraku'}]\", 'abstract': '<p>Spectral tuning of visual pigments often facilitates adaptation to new environments, and it is intriguing to study the visual ecology of pelagic sharks with expanded habitats. The whale shark, which dives into the deep sea of nearly 2,000 meters besides near-surface filter-feeding, was previously shown to possesses the ‘blue-shifted’ rhodopsin (RHO). In this study, our spectroscopy of recombinant whale shark RHO mutants revealed the dominant effect of the novel spectral tuning amino acid site 94, which is implicated in congenital stationary night blindness of humans, accounting for the blue shift. Thermal decay profiling revealed the reduction of the thermal stability of whale shark RHO, as typically observed for cone opsins, which was experimentally shown to be achieved by the site 178, as well as 94. The results suggest that these two sites cooperatively enhance the visual capacity in both the deep sea and the sea surface, enabling exceptionally wide vertical migration of this species.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462562', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'A standardized method to purify cardiomyocytes from individual mouse hearts of any age', 'authors': \"[{'name': 'Amy M Nicks'}, {'name': 'Sara R Holman'}, {'name': 'Andrea Y Chan'}, {'name': 'Michael Tsang'}, {'name': 'Paul E Young'}, {'name': 'David T Humphreys'}, {'name': 'Nawazish Naqvi'}, {'name': 'Ahsan Husain'}, {'name': 'Ming Li'}, {'name': 'Nicola J Smith'}, {'name': 'Siiri E Iismaa'}, {'name': 'Robert M Graham'}]\", 'abstract': '<sec><title>Rationale</title><p>Primary cardiomyocytes are invaluable for understanding postnatal heart development and elucidating disease mechanisms in genetic and pharmacological models, however, a method to obtain freshly purified cardiomyocytes at any postnatal age, without using different age-dependent isolation procedures and cell culture, is lacking.</p></sec><sec><title>Objective</title><p>To develop a standardized method that allows rapid isolation and purification of cardiomyocytes in high yield and viability from individual neonatal, infant, and adult mice.</p></sec><sec><title>Methods and Results</title><p>Hearts of C57BL/6J mice were cannulated using a novel <italic>in situ</italic> aortic cannulation procedure optimized to allow cannulation of even the very small vessel of neonates (postnatal day 0-2, P0-2). Hearts were then subjected to Langendorff retrograde perfusion and enzymatic digestion. Cardiomyocytes were isolated after subsequent tissue disaggregation and filtration, in high yield (1.56-2.2×10<sup>6</sup> cardiomyocytes/heart) and viability (~70-100%). The larger size of infant (P10 and P13) and adult (P70), but not neonatal, cardiomyocytes relative to non-myocytes, allowed enrichment by differential centrifugation. Cardiomyocytes from all ages were further purified by immunomagnetic bead-based depletion of non-myocytes. Together, these procedures resulted in the isolation of highly purified cardiomyocytes (~94%) within 1 hour, enabling experiments using individual replicates. For example, RNA-sequencing of cardiomyocytes purified from one P2 male and female heart per litter (n=4 litters) showed distinct clustering by litters and sex differences for nine differentially expressed genes (FDR&lt;0.005). <italic>In situ</italic> fixation via coronary perfusion, performed immediately after tissue digestion, preserved the cytoarchitecture of isolated cardiomyocytes (yielding ~94% rod-shaped cardiomyocytes at all ages), allowing capture of spindle-shaped neonatal cells undergoing mitosis, as well as enabling accurate quantitation of cardiomyocyte area and nucleation state.</p></sec><sec><title>Conclusion</title><p>The procedures developed here provide a universal protocol for the rapid isolation and purification of high-quality cardiomyocytes from hearts of any postnatal age, even those of neonates, thereby enabling direct comparisons between individual hearts.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462554', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Isotopic evidence for population dynamics in the Central Italian Copper Age and Bronze Age', 'authors': \"[{'name': 'Marco Romboni'}, {'name': 'Ilenia Arienzo'}, {'name': 'Mauro Antonio Di Vito'}, {'name': 'Carmine Lubritto'}, {'name': 'Monica Piochi'}, {'name': 'Maria Rosa Di Cicco'}, {'name': 'Olga Rickards'}, {'name': 'Mario Federico Rolfo'}, {'name': 'Jan Sevink'}, {'name': 'Flavio De Angelis'}, {'name': 'Luca Alessandri'}]\", 'abstract': '<p>The mobility patterns in the Italian peninsula during prehistory are still relatively unknown. The excavation of the Copper Age and Bronze Age deposits in La Sassa cave (Sonnino, Italy) allowed to broaden the knowledge about some local and regional dynamics. We employed a multi-disciplinary approach, including stable (carbon and nitrogen – C and N, respectively) and radiogenic (strontium-Sr) isotopes analyses and the identification of the cultural traits in the material culture to identify mobility patterns that took place in the region. The Sr isotopic analyses on the human bones show that in the Copper Age and at the beginning of the Bronze Age, the cave was used as a burial place by different villages, perhaps spread in a radius of no more than 5 km. Stable isotopes analyses suggest the introduction of C4 plants in the diet of the Middle Bronze Age (MBA) communities in the area. Remarkably, in the same period, the material culture shows increasing influxes coming from the North. This evidence is consistent with the recent genomic findings tracing the arrival of people carrying a Steppe-related ancestry in Central Italy in MBA.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462291', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'easyfm: An easy software suite for file manipulation of Next Generation Sequencing data on desktops', 'authors': \"[{'name': 'Hyungtaek Jung'}, {'name': 'Brendan Jeon'}, {'name': 'Daniel Ortiz-Barrientos'}]\", 'abstract': '<p>Storing and manipulating Next Generation Sequencing (NGS) file formats is an essential but difficult task in biological data analysis. The <italic>easyfm</italic> (<bold>easy f</bold>ile <bold>m</bold>anipulation) toolkit (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/TaekAndBrendan/easyfm\">https://github.com/TaekAndBrendan/easyfm</ext-link>) makes manipulating commonly used NGS files more accessible to biologists. It enables them to perform end-to-end reproducible data analyses using a free standalone desktop application (available on Windows, Mac and Linux). Unlike existing tools (e.g. Galaxy), the Graphical User Interface (GUI)-based <italic>easyfm</italic> is not dependent on any high-performance computing (HPC) system and can be operated without an internet connection. This specific benefit allow <italic>easyfm</italic> to seamlessly integrate visual and interactive representations of NGS files, supporting a wider scope of bioinformatics applications in the life sciences.</p><sec><title>Author summary</title><p>The analysis and manipulation of NGS data for understanding biological phenomena is an increasingly important aspect in the life sciences. Yet, most methods for analysing, storing and manipulating NGS data require complex command-line tools in HPC or web-based servers and have not yet been implemented in comprehensive, easy-to-use software. This is a major hurdle preventing more general application in the field of NGS data analysis and file manipulation. Here we present <italic>easyfm</italic>, a free standalone Graphical User Interface (GUI) software with Python support that can be used to facilitate the rapid discovery of target sequences (or user’s interest) in NGS datasets for novice users. For user-friendliness and convenience, <italic>easyfm</italic> was developed with four work modules and a secondary GUI window (herein secondary window), covering different aspects of NGS data analysis (mainly focusing on FASTA files), including post-processing, filtering, format conversion, generating results, real-time log, and help. In combination with the executable tools (BLAST+ and BLAT) and Python, <italic>easyfm</italic> allows the user to set analysis parameters, select/extract regions of interest, examine the input and output results, and convert to a wide range of file formats. To help augment the functionality of existing web-based and command-line tools, <italic>easyfm</italic>, a self-contained program, comes with extensive documentation (hosted at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/TaekAndBrendan/easyfm\">https://github.com/TaekAndBrendan/easyfm</ext-link>) including a comprehensive step-by-step guide.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.30.462529', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Dynamical modeling of the H3K27 epigenetic landscape in mouse embryonic stem cells', 'authors': \"[{'name': 'Kapil Newar'}, {'name': 'Eric Fanchon'}, {'name': 'Daniel Jost'}]\", 'abstract': '<p>The Polycomb system via the methylation of the lysine 27 of histone H3 (H3K27) plays central roles in the silencing of many lineage-specific genes during development. Recent experimental evidence suggested that the recruitment of histone modifying enzymes like the Polycomb repressive complex 2 (PRC2) at specific sites and their spreading capacities from these sites are key to the establishment and maintenance of a proper epigenomic landscape around Polycomb-target genes. Here, based on previous biochemical knowledge, we turned this hypothesis into a mathematical model that can predict the locus-specific distributions of H3K27 modifications. Within the biological context of mouse embryonic stem cells, our model showed quantitative agreement with experimental profiles of H3K27 acetylation and methylation around Polycomb-target genes in wild-type and mutants. In particular, we demonstrated the key role of the reader-writer module of PRC2 and of the competition between the binding of activating and repressing enzymes in shaping the H3K27 landscape around transcriptional start sites. The predicted dynamics of establishment and maintenance of the repressive trimethylated H3K27 state suggest a slow accumulation, in perfect agreement with experiments. Our approach represents a first step towards a quantitative description of PcG regulation in various cellular contexts and provides a generic framework to better characterize epigenetic regulation in normal or disease situations.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.29.462289', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Computational design of peptide therapeutics: how useful are sequence-based protein-protein interaction predictors?', 'authors': \"[{'name': 'François Charih'}, {'name': 'Kyle K. Biggar'}, {'name': 'James R. Green'}]\", 'abstract': '<p>Engineering peptides to achieve a desired therapeutic effect through the inhibition of a specific target activity or protein interaction is a non-trivial task. Few of the existing <italic>in silico</italic> peptide design algorithms generate target-specific peptides. Instead, many methods produce peptides that achieve a desired effect through an unknown mechanism. In contrast with resource-intensive high-throughput experiments, <italic>in silico</italic> screening is a cost-effective alternative that can prune the space of candidates when engineering target-specific peptides. Using a set of FDA-approved peptides we curated specifically for this task, we assess the applicability of several sequence-based protein-protein interaction predictors as a screening tool within the context of peptide therapeutic engineering. We show that similarity-based protein-protein interaction predictors are more suitable for this purpose than the current state-of-the-art deep learning methods. We also show that this approach is mostly useful when designing new peptides against targets for which naturally-occurring interactors are already known, and that deploying it for <italic>de novo</italic> peptide engineering tasks may require gathering additional target-specific training data. Taken together, this work offers evidence that supports the use of similarity-based protein-protein interaction predictors for peptide therapeutic engineering, especially peptide analogs.</p><sec><title>Author summary</title><p>Peptides are short chains of amino acids that represent an alternative to small molecule drugs to inhibit enzymatic activity or protein-protein interactions. Designing small peptides to target a protein specifically is challenging, though a number of computational approaches do exist. These methods must to evaluate the quality of a peptide in terms of its ability to bind specifically to a target protein, which can be done <italic>in silico</italic> with sequence-based protein-protein interaction predictors. It isn’t clear, however, how suitable the different types of predictors are for the task of predicting interactions between peptides and proteins. Herein, we apply state-of-the-art protein-protein interaction predictors to a set of FDA-approved peptide therapeutics to determine how the different protein-protein interaction predictors compare for the task of assessing peptide quality, and to determine under which circumstances these methods may be successful. Our results show that similarity-based predictors outperform deep learning methods, and that these methods may be more suitable for peptide analog engineering than for the design of peptide therapeutics <italic>de novo</italic>.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462277', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'The use of barcoded Asaia bacteria in mosquito in vivo screens for identification of systemic insecticides and inhibitors of malaria transmission', 'authors': \"[{'name': 'Angelika Sturm'}, {'name': 'Martijn W. Vos'}, {'name': 'Rob Henderson'}, {'name': 'Maarten Eldering'}, {'name': 'Karin M.J. Koolen'}, {'name': 'Avinash Sheshachalam'}, {'name': 'Guido Favia'}, {'name': 'Kirandeep Samby'}, {'name': 'Esperanza Herreros'}, {'name': 'Koen J. Dechering'}]\", 'abstract': '<p>This work addresses the need for new chemical matter in product development for control of pest insects and vector-borne diseases. We present a barcoding strategy that enables phenotypic screens of blood-feeding insects against small molecules in microtiter plate-based arrays and apply this to discovery of novel systemic insecticides and compounds that block malaria parasite development in the mosquito vector. Encoding of the bloodmeals was achieved through recombinant DNA-tagged <italic>Asaia</italic> bacteria that successfully colonized <italic>Aedes</italic> and <italic>Anopheles</italic> mosquitoes. An arrayed screen of a collection of pesticides showed that chemical classes of avermectins, phenylpyrazoles and neonicotinoids were enriched for compounds with systemic adulticide activity against <italic>Anopheles</italic>. Using a luminescent <italic>Plasmodium falciparum</italic> reporter strain, barcoded screens identified 48 drug-like transmission blocking compounds from a 400-compound antimicrobial library. The approach significantly increases the throughput in phenotypic screening campaigns using adult insects, and identifies novel candidate small molecules for disease control.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462263', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Enthalpic and Entropic Contributions to Interleaflet Coupling Drive Domain Registration and Anti-registration in Biological Membrane', 'authors': \"[{'name': 'Akshara Sharma'}, {'name': 'Aniruddha Seal'}, {'name': 'Sahithya S. Iyer'}, {'name': 'Anand Srivastava'}]\", 'abstract': '<p>Biological membrane is a complex self-assembly of lipids, sterols and proteins organized as a fluid bilayer of two closely stacked lipid leaflets. Differential molecular interactions among its diverse constituents give rise to heterogeneities in the membrane lateral organization. Under certain conditions, heterogeneities in the two leaflets can be spatially synchronised and exist as registered domains across the bilayer. Several contrasting theories behind mechanisms that induce registration of nanoscale domains have been suggested[1–3]. Following a recent study[4] showing the effect of position of lipid tail unsaturation on domain registration behavior, we decided to develop an analytical theory to elucidate the driving forces that create and maintain domain registry across leaflets. Towards this, we formulated a Hamiltonian for a stacked lattice system where site variables encapsulate the lipid molecular properties including the position of unsaturation and various other interactions that could drive phase separation and interleaflet coupling. We solve the Hamiltonian using Monte Carlo simulations and create a complete phase diagram that reports the presence or absence of registered domains as a function of various Hamiltonian parameters. We find that the interleaflet coupling should be described as a competing enthalpic contribution due to interaction of lipid tail termini, primarily due to saturated-saturated interactions, and an interleaflet entropic contribution from overlap of unsaturated tail termini. We find that higher position of unsaturation provides weaker interleaflet coupling. We also find points in our parameter space that allow thermodynamically stable nanodomains in our bilayer model, which we have verified by carrying out extended Monte Carlo simulations. These persistent non-coalescing registered nanodomains close to the lower end of the accepted nanodomain size range also point towards a possible “nanoscale” emulsion description of lateral heterogeneities in biological membrane leaflets.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462177', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Towards real-world neuroscience using mobile EEG and augmented reality', 'authors': \"[{'name': 'Alexandra Krugliak'}, {'name': 'Alex Clarke'}]\", 'abstract': '<p>Our visual environment impacts multiple aspects of cognition including perception, attention and memory, yet most studies traditionally remove or control the external environment. As a result, we have a limited understanding of neurocognitive processes beyond the controlled lab environment. Here, we aim to study neural processes in real-world environments, while also maintaining a degree of control over perception. To achieve this, we combined mobile EEG (mEEG) and augmented reality (AR), which allows us to place virtual objects into the real world. We validated this AR and mEEG approach using a well-characterised cognitive response - the face inversion effect. Participants viewed upright and inverted faces in three EEG tasks (1) a lab-based computer task, (2) walking through an indoor environment while seeing face photographs, and (3) walking through an indoor environment while seeing virtual faces. We find greater low frequency EEG activity for inverted compared to upright faces in all experimental tasks, demonstrating that cognitively relevant signals can be extracted from mEEG and AR paradigms. This was established in both an epoch-based analysis aligned to face events, and a GLM-based approach that incorporates continuous EEG signals and face perception states. Together, this research helps pave the way to exploring neurocognitive processes in real-world environments while maintaining experimental control using AR.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462712', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Mycobacterial resistance to zinc poisoning requires assembly of P-ATPase-containing membrane metal efflux platforms', 'authors': \"[{'name': 'Yves-Marie Boudehen'}, {'name': 'Marion Faucher'}, {'name': 'Xavier Maréchal'}, {'name': 'Roger Miras'}, {'name': 'Jérôme Rech'}, {'name': 'Olivier Sénèque'}, {'name': 'Maximilian Wallat'}, {'name': 'Pascal Demange'}, {'name': 'Jean-Yves Bouet'}, {'name': 'Olivier Saurel'}, {'name': 'Patrice Catty'}, {'name': 'Claude Gutierrez'}, {'name': 'Olivier Neyrolles'}]\", 'abstract': '<p>Transition metals are toxic at high concentrations. The P<sub>1B</sub>-ATPase metal exporter CtpC/Rv3270 is required for resistance to zinc poisoning in the human pathogen <italic>Mycobacterium tuberculosis</italic>. Here, we discovered that zinc resistance also depends on the chaperone-like protein PacL1/Rv3269. PacL1 bound Zn<sup>2+</sup>, but unlike PacL1 and CtpC, the PacL1 metal-binding motif (MBM) was required only at high zinc concentrations. PacL1 co-localized with CtpC in dynamic microdomains within the mycobacterial plasma membrane. Microdomain formation did not require flotillins nor the PacL1 MBM. Instead, loss of the PacL1 Glutamine/Alanine repeats led to loss of CtpC and sensitivity to zinc. PacL1 and CtpC are within the same operon, and homologous PacL1-P<sub>1B</sub>-ATPase pairs are widely distributed within and across prokaryotes. PacL1 colocalized and functioned redundantly with PacL orthologs in <italic>Mycobacterium tuberculosis</italic>. Overall, our study suggests that PacL proteins are scaffolds that assemble P-ATPase-containing metal efflux platforms, a novel type of functional membrane microdomain that underlies bacterial resistance to metal poisoning.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.10.01.462786', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Chi3l1 is a modulator of glioma stem cell states and a therapeutic vulnerability for glioblastoma', 'authors': \"[{'name': 'Charlotte Guetta-Terrier'}, {'name': 'David Karambizi'}, {'name': 'Bedia Akosman'}, {'name': 'Jia-Shu Chen'}, {'name': 'Suchitra Kamle'}, {'name': 'Eduardo Fajardo'}, {'name': 'Andras Fiser'}, {'name': 'Ritambhara Singh'}, {'name': 'Steven A. Toms'}, {'name': 'Chun Geun Lee'}, {'name': 'Jack A. Elias'}, {'name': 'Nikos Tapinos'}]\", 'abstract': '<p>Chi3l1 (Chitinase 3-like 1) is a secreted protein highly expressed in glioblastoma. Here, we show that exposure of glioma stem cells (GSCs) to Chi3l1 reduces the CD133<sup>+</sup>/SOX2<sup>+</sup> cells and increases the CD44<sup>+</sup>/Chi3l1<sup>+</sup> cells. Chi3l1 binds to CD44 and induces phosphorylation and nuclear translocation of beta-catenin, Akt and STAT3. Single cell RNA-seq and RNA velocity following incubation of GSCs with Chi3l1 show significant changes in GSC state dynamics driving GSCs towards a mesenchymal expression profile and reducing transition probabilities towards terminal cellular states. ATAC-seq reveals that Chi3l1 increases accessibility of promoters containing MAZ transcription factor footprint. Inhibition of MAZ directly regulates genes with highest expression in cellular clusters exhibiting significant cell state transitions. Finally, targeting Chi3l1 in vivo with a blocking antibody, resets the transcriptomic profile of glioblastoma and inhibits tumor growth. Our work implicates Chi3l1 as modulator of GSC cellular states and demonstrates pre-clinical efficacy of anti-Chi3l1 antibody treatment.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.30.462573', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structural Insights into the BRAF Monomer-to-dimer Transition Mediated by RAS Binding', 'authors': \"[{'name': 'Juliana A. Martinez Fiesco'}, {'name': 'David E. Durrant'}, {'name': 'Deborah K. Morrison'}, {'name': 'Ping Zhang'}]\", 'abstract': '<p>An unresolved issue in RAF kinase signaling is how binding of autoinhibited RAF monomers to activated RAS initiates the conformational changes required to form active RAF dimers. Here, we present cryo-electron microscopy structures of full-length BRAF complexes derived from mammalian cells: autoinhibited monomeric BRAF:14-3-3<sub>2</sub>:MEK and BRAF:14-3-3<sub>2</sub> complexes and an inhibitor-bound, dimeric BRAF<sub>2</sub>:14-3-3<sub>2</sub> complex, at 3.7, 4.1, and 3.9 Å resolution, respectively. The RAS binding domain (RBD) of BRAF is resolved in the autoinhibited structures, and we find that neither MEK nor ATP binding is required to stabilize the autoinhibited complexes. Notably, the RBD was found to interact extensively with the 14-3-3 protomer bound to the BRAF C-terminal site. Moreover, through structure-guided mutational studies, our findings indicate that RAS-RAF binding is a dynamic process and that RBD residues at the 14-3-3 interface have a dual function, first stabilizing RBD orientation in the autoinhibited state and then contributing to full RAS contact.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462440', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Loss of the cleaved-protamine 2 domain leads to incomplete histone-to-protamine exchange and infertility in mice', 'authors': \"[{'name': 'Lena Arévalo'}, {'name': 'Gina Esther Merges'}, {'name': 'Simon Schneider'}, {'name': 'Franka Enow Oben'}, {'name': 'Isabelle Neumann'}, {'name': 'Hubert Schorle'}]\", 'abstract': '<p>Protamines are unique sperm-specific proteins that package and protect paternal chromatin until fertilization. A subset of mammalian species expresses two protamines (PRM1 and PRM2), while in others PRM1 is sufficient for sperm chromatin packaging. Alterations of the species-specific ratio between PRM1 and PRM2 are associated with infertility. Unlike PRM1, PRM2 is generated as a precursor protein consisting of a highly conserved N-terminal domain, termed cleaved PRM2 (cP2), which is consecutively trimmed off during chromatin condensation. The carboxyterminal part, called mature PRM2 (mP2), interacts with DNA and mediates chromatin-hypercondensation, together with PRM1. The removal of the cP2 domain is believed to be imperative for proper chromatin condensation and the prevention of DNA damage. Yet, the role of cP2 is not yet understood. Using CRISPR-Cas9 mediated gene editing, we generated mice lacking the cP2 domain while the mP2 is still expressed. We show that deletion of one allele of the cP2 domain is sufficient to render male mice infertile. cP2 deficient sperm show incomplete PRM2 incorporation, retention of transition proteins and a severely altered protamine ratio. During epididymal transit, cP2 deficient sperm seem to undergo ROS mediated degradation leading to complete DNA fragmentation, inviability and immotility of mature sperm. The cP2 domain therefore seems to be necessary for the complex crosstalk leading to the successive and complete removal of transition proteins and complete protamination of sperm chromatin. Overall, we present the first step towards understanding the role of the cP2 domain in paternal chromatin packaging and open up avenues for further research.</p>', 'posted': '2021-10-01'}, {'doi': '10.1101/2021.09.28.462232', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Evaluating the reliability, validity, and utility of overlapping networks: Implications for cognitive control', 'authors': \"[{'name': 'Savannah L. Cookson'}, {'name': 'Mark D’Esposito'}]\", 'abstract': '<p>Brain regions can be consistently clustered together into a small number of large-scale networks based on the correlation of their activity across time. However, the methods typically used to assign regions to these networks typically force non- or minimal overlap between the networks, losing information about how particular regions may connect to multiple networks. However, new methods are emerging that allow networks to overlap. Here, we examined the reliability and validity of one such assignment method, the mixed membership algorithm, and the utility of overlapping assignment for identifying potential gaps in existing network models of behavior. Analyses were conducted on resting state data from a large set of participants. First, we split the data into two halves and generated group-level overlapping networks for each subgroup. We then used a bootstrapped Dice similarity analysis to demonstrate that the binarized network topographies showed good agreement between subgroups. Next, we generated nonoverlapping networks in the combined data using a typical k-means clustering method. Using the same Dice Similarity analysis, we compared these nonoverlapping networks both with previously published nonoverlapping networks (Yeo et al, 2011) and with overlapping networks likewise generated for the combined data. The nonoverlapping networks generally recapitulated known networks in the literature, and the overlapping networks captured large portions of one to three nonoverlapping network topographies. Following this, we used a relative entropy analysis to assess the uniformity of each region’s distribution of connections across the overlapping networks. While many regions were indeed assigned to one network, more were assigned to multiple networks. Furthermore, regions assigned to multiple networks did not necessarily show stronger connection to one network over the others. Finally, we assessed the network membership of regions of the fronto-parietal and cingulo-opercular networks of the Dual-Networks model of cognitive control. In combination, the regions were members of all of the overlapping networks, and the majority were assigned to multiple networks. Together, these results indicate that overlapping assignment produces consistent and biologically plausible network topographies, and that a complete network model of cognitive control, or indeed any network model of behavior needs to be updated to account for multiple-network membership and the complex interactions that occur between the relevant networks as a function of their overlapping nature.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462236', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Tracking chromatin state changes using μMap photo-proximity labeling', 'authors': \"[{'name': 'Ciaran P. Seath'}, {'name': 'Antony J. Burton'}, {'name': 'David W. C. MacMillan'}, {'name': 'Tom W. Muir'}]\", 'abstract': '<p>Interactions between biomolecules, particularly proteins, underlie all cellular processes, and ultimately control cell fate. Perturbation of native interactions through mutation, changes in expression levels, or external stimuli leads to altered cellular physiology and can result in either disease or therapeutic effects.<sup>1,2</sup> Mapping these interactions and determining how they respond to stimulus is the genesis of many drug development efforts, leading to new therapeutic targets and improvements in human health.<sup>1</sup> However, in the complex environment of the nucleus it is challenging to determine protein-protein interactions due to low abundance, transient or multi-valent binding, and a lack of technologies that are able to interrogate these interactions without disrupting the protein binding surface under study.<sup>3</sup> Chromatin remodelers, modifying enzymes, interactors, and transcription factors can all be redirected by subtle changes to the microenvironment, causing global changes in protein expression levels and subsequent physiology. Here, we describe the Chroma-<italic>μ</italic>Map method for the traceless incorporation of Ir-photosensitizers into the nuclear microenvironment using engineered split inteins. These Ir-catalysts can activate diazirine warheads to form reactive carbenes within a ~10 nm radius, cross-linking with proteins within the immediate microenvironment for analysis via quantitative chemoproteomics.<sup>4</sup> We demonstrate this concept on nine different nuclear proteins with varied function and in each case, elucidating their microenvironments. Additionally, we show that this short-range proximity labeling method can reveal the critical changes in interactomes in the presence of cancer-associated mutations, as well as treatment with small-molecule inhibitors. Chroma-<italic>μ</italic>Map improves our fundamental understanding of nuclear protein-protein interactions, as well as the effects that small molecule therapeutics have on the local chromatin environment, and in doing so is expected to have a significant impact on the field of epigenetic drug discovery in both academia and industry.</p>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462237', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Regulation of Blos1 by IRE1 prevents the accumulation of Huntingtin protein aggregates', 'authors': \"[{'name': 'Donghwi Bae'}, {'name': 'Rachel Elizabeth Jones'}, {'name': 'Julie Hollien'}]\", 'abstract': '<p>Huntington’s Disease is characterized by accumulation of the aggregation-prone mutant Huntingtin (mHTT) protein. Here, we show that expression of mHTT in mouse cultured cells activates IRE1, the transmembrane sensor of stress in the endoplasmic reticulum, leading to degradation of the <italic>Blos1</italic> mRNA and repositioning of lysosomes and late endosomes toward the microtubule organizing center. Overriding <italic>Blos1</italic> degradation results in accumulation of larger mHTT aggregates and increased cell death. Although mHTT is degraded by macroautophagy when highly expressed, we show that prior to the formation of large aggregates, mHTT is degraded via an ESCRT-dependent, endosomal microautophagy pathway. This pathway is enhanced by <italic>Blos1</italic> degradation and appears to protect cells from a toxic, less aggregated form of mHTT.</p><sec><title>Condensed title</title><p><italic>Blos1</italic> regulation protects from Huntingtin protein aggregation</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.28.462241', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Using TrackMate to Analyze Drosophila Larval and Adult Locomotion', 'authors': \"[{'name': 'Alisa A. Omelchenko'}, {'name': 'Ainul Huda'}, {'name': 'Allison N. Castaneda'}, {'name': 'Thomas J. Vaden'}, {'name': 'Lina Ni'}]\", 'abstract': '<p><italic>Drosophila</italic> adult and larvae exhibit sophisticated behaviors that are widely used in development, synaptic transmission, sensory physiology, and learning and memory research. Many of these behaviors depend on locomotion, the ability of an animal to move. However, the statistical analysis of locomotion is not trivial. Here we use an open-source Fiji plugin TrackMate to track the locomotion of <italic>Drosophila</italic> adults and larvae. We build optimal experimental setups to rapidly process recordings by Fiji and analyze by TrackMate. We also provide tips for analyzing non-optimal recordings. TrackMate extracts the X and Y positions of an animal on each frame of an image sequence or a video. This information allows for generating moving trajectories, calculating moving distances, and determining preference indices in two-choice assays. Notably, this free-cost analysis method does not require programming skills.</p><sec><title>Summary statement</title><p>This study uses an open-source Fiji plugin TrackMate to computationally analyze <italic>Drosophila</italic> adult and larval behavioral assays, which does not require programming skills.</p></sec>', 'posted': '2021-09-30'}, {'doi': '10.1101/2021.09.29.462410', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Moderate prenatal alcohol exposure modifies sex-specific CRFR1 activity in the central amygdala and anxiety-like behavior in adolescent offspring', 'authors': \"[{'name': 'Siara Kate Rouzer'}, {'name': 'Marvin R. Diaz'}]\", 'abstract': '<p>Anxiety disorders are highly prevalent among individuals with a history of prenatal alcohol exposure (PAE), and adolescent rodents demonstrate anxiety-like behavior following moderate PAE on Gestational Day (G) 12. A likely systemic target of PAE is the stress peptide corticotrophin-releasing factor (CRF), as activation of CRF receptor 1 (CRFR1) in the medial nucleus of the central amygdala (CeM) is known to increase anxiety-like behavior in adults. To determine if CRF-CRFR1 interactions underly PAE-induced anxiety, functional changes in CRF system activity were investigated in adolescent male and female Sprague Dawley rats following G12 PAE. Compared to air-exposed controls, PAE increased basal spontaneous (s) inhibitory post-synaptic current (IPSC) frequency in the CeM of males, but not females. Furthermore, PAE blunted CRFR1-regulated miniature (m) IPSCs in a sex- and dose-specific manner, and only PAE males demonstrated tonic CRFR1 activity in the CeM. It was further determined that G12 PAE decreased CRFR1 mRNA in the CeM of males while increasing regional expression in females. Finally, infusion of a CRFR1 agonist into the CeM of adolescents produced a blunted expression of CRFR1-induced anxiety-like behavior exclusively in PAE males, mirroring the blunted physiology demonstrated by PAE males. Cumulatively, these data suggest that CRFR1 function within the CeM is age- and sex-specific, and PAE not only increases the expression of anxiety-like behavior, but may reduce the efficacy of treatment for PAE-induced anxiety through CRFR1-associated mechanisms. Therefore, future research will be necessary to develop targeted treatment of anxiety disorders in individuals with a history of PAE.</p>', 'posted': '2021-10-01'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.09.30.462488</td>\n",
       "      <td>423</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Durability of immune responses to the BNT162b2...</td>\n",
       "      <td>[{'name': 'Mehul S. Suthar'}, {'name': 'Prabhu...</td>\n",
       "      <td>&lt;p&gt;The development of the highly efficacious m...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.09.29.21264325</td>\n",
       "      <td>220</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Assessing the Burden of COVID-19 in Developing...</td>\n",
       "      <td>[{'name': 'Andrew Levin'}, {'name': 'Nana Owus...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Introduction&lt;/title&gt;&lt;p&gt;The infecti...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.09.30.21264363</td>\n",
       "      <td>197</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 infections elicit higher levels of ...</td>\n",
       "      <td>[{'name': 'Elizabeth M. Anderson'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;Severe acute respiratory syndrome coronavir...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.09.29.21264272</td>\n",
       "      <td>145</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Associations between SARS-CoV-2 variants and r...</td>\n",
       "      <td>[{'name': 'Miguel I. Paredes'}, {'name': 'Step...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The COVID-19 ...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.09.28.462148</td>\n",
       "      <td>83</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Adeno-associated virus (AAV) reduces cortical ...</td>\n",
       "      <td>[{'name': 'Christos M. Suriano'}, {'name': 'Je...</td>\n",
       "      <td>&lt;p&gt;Recombinant adeno-associated viruses (AAVs)...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.09.30.462231</td>\n",
       "      <td>75</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Structures of core eukaryotic protein complexes</td>\n",
       "      <td>[{'name': 'Ian R. Humphreys'}, {'name': 'Jimin...</td>\n",
       "      <td>&lt;p&gt;Protein-protein interactions play critical ...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.09.30.21264376</td>\n",
       "      <td>42</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>Estimates of pandemic excess mortality in Indi...</td>\n",
       "      <td>[{'name': 'Murad Banaji'}, {'name': 'Aashish G...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The COVID-19 ...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.09.28.21263801</td>\n",
       "      <td>30</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Analysis of overdispersion in airborne transmi...</td>\n",
       "      <td>[{'name': 'Swetaprovo Chaudhuri'}, {'name': 'P...</td>\n",
       "      <td>&lt;p&gt;Superspreading events and overdispersion ar...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.09.30.462538</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>New tools for automated cryo-EM single-particl...</td>\n",
       "      <td>[{'name': 'Dari Kimanius'}, {'name': 'Liyi Don...</td>\n",
       "      <td>&lt;label&gt;1&lt;/label&gt;&lt;title&gt;Abstract&lt;/title&gt;&lt;p&gt;We d...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.09.30.462505</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>S. pombe wtf genes use dual transcriptional re...</td>\n",
       "      <td>[{'name': 'Nicole L. Nuckolls'}, {'name': 'Ana...</td>\n",
       "      <td>&lt;p&gt;Meiotic drivers bias gametogenesis to ensur...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.09.30.462449</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pharmacology and Toxicology</td>\n",
       "      <td>True</td>\n",
       "      <td>Pyronaridine Protects Against SARS-CoV-2 in Mouse</td>\n",
       "      <td>[{'name': 'Ana C. Puhl'}, {'name': 'Giovanni F...</td>\n",
       "      <td>&lt;p&gt;There are currently relatively few small-mo...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.09.28.21264206</td>\n",
       "      <td>12</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>False</td>\n",
       "      <td>Enhancing the brain’s emotion regulation capac...</td>\n",
       "      <td>[{'name': 'Kaoru Nashiro'}, {'name': 'Jungwon ...</td>\n",
       "      <td>&lt;p&gt;Heart rate variability is a robust biomarke...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.09.28.462068</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Here’s the twist: How the brain updates natura...</td>\n",
       "      <td>[{'name': 'Asieh Zadbood'}, {'name': 'Samuel A...</td>\n",
       "      <td>&lt;p&gt;The brain actively reshapes past memories i...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.09.28.462199</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Cell-ACDC: a user-friendly toolset embedding s...</td>\n",
       "      <td>[{'name': 'Francesco Padovani'}, {'name': 'Ben...</td>\n",
       "      <td>&lt;p&gt;Live-cell imaging is a powerful tool to stu...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.09.30.462010</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Drivers and Determinants of Strain Dynamics Fo...</td>\n",
       "      <td>[{'name': 'Thomas SB Schmidt'}, {'name': 'Simo...</td>\n",
       "      <td>&lt;p&gt;Faecal microbiota transplantation (FMT) is ...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.09.28.21264208</td>\n",
       "      <td>10</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Integration of genetic colocalizations with ph...</td>\n",
       "      <td>[{'name': 'Michael J. Gloudemans'}, {'name': '...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Identificatio...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.09.30.462548</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Prime Editing Efficiency and Fidelity are Enha...</td>\n",
       "      <td>[{'name': 'J Ferreira da Silva'}, {'name': 'GP...</td>\n",
       "      <td>&lt;p&gt;Prime editing is a powerful genome engineer...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.10.01.462730</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Deep mutational scan of a drug efflux pump rev...</td>\n",
       "      <td>[{'name': 'Gianmarco Meier'}, {'name': 'Sujani...</td>\n",
       "      <td>&lt;p&gt;Drug efflux is a common resistance mechanis...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.09.29.462379</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Modeling spatial, developmental, physiological...</td>\n",
       "      <td>[{'name': 'S. Oldham'}, {'name': 'B. D. Fulche...</td>\n",
       "      <td>&lt;p&gt;The complex connectivity of nervous systems...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.09.29.462485</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Repurposing CAR targets in solid tumors via a ...</td>\n",
       "      <td>[{'name': 'Sanna Madan'}, {'name': 'Alejandro ...</td>\n",
       "      <td>&lt;p&gt;Chimeric antigen receptor (CAR) T cell ther...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.09.28.462132</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>An anti-inflammatory activation sequence gover...</td>\n",
       "      <td>[{'name': 'Nicolas Denans'}, {'name': 'Nhung T...</td>\n",
       "      <td>&lt;p&gt;Macrophages are essential for tissue repair...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.09.29.462303</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Emerging mosquito resistance to piperonyl buto...</td>\n",
       "      <td>[{'name': 'Guofa Zhou'}, {'name': 'Yiji Li'}, ...</td>\n",
       "      <td>&lt;p&gt;Piperonyl butoxide (PBO)-synergized pyrethr...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.09.30.462517</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Hippocampal CA1 pyramidal cells do not receive...</td>\n",
       "      <td>[{'name': 'Lilya Andrianova'}, {'name': 'Erica...</td>\n",
       "      <td>&lt;p&gt;Thalamic nucleus reuniens (NRe) is located ...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.09.29.462495</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Dominant Awn Inhibitor, which encodes an ALOG ...</td>\n",
       "      <td>[{'name': 'Hideki Takanashi'}, {'name': 'Hirom...</td>\n",
       "      <td>&lt;p&gt;The awn, a needle-like structure that exten...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.10.01.462603</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Regulation of the Evolutionarily Conserved Mus...</td>\n",
       "      <td>[{'name': 'Peter T. Ajayi'}, {'name': 'Prasann...</td>\n",
       "      <td>&lt;p&gt;Skeletal muscles play a central role in hum...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.09.28.462226</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Osteoblasts pattern endothelium and somatosens...</td>\n",
       "      <td>[{'name': 'Rosalind G. Bump'}, {'name': 'Camil...</td>\n",
       "      <td>&lt;p&gt;Skeletal elements frequently associate with...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.09.28.462180</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioengineering</td>\n",
       "      <td>False</td>\n",
       "      <td>Extracellular Electron Transfer Enables Cellul...</td>\n",
       "      <td>[{'name': 'Gina Partipilo'}, {'name': 'Austin ...</td>\n",
       "      <td>&lt;p&gt;Extracellular electron transfer (EET) is an...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.09.30.462680</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Evolutionary insights into primate skeletal ge...</td>\n",
       "      <td>[{'name': 'Genevieve Housman'}, {'name': 'Emil...</td>\n",
       "      <td>&lt;p&gt;The evolution of complex skeletal traits in...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.09.28.462168</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Neurofibromin 1 controls metabolic balance and...</td>\n",
       "      <td>[{'name': 'Xiaoyan Wei'}, {'name': 'Matthias L...</td>\n",
       "      <td>&lt;p&gt;Patients affected by neurofibromatosis type...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.09.29.459010</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>A whole-brain monosynaptic input connectome to...</td>\n",
       "      <td>[{'name': 'Shenqin Yao'}, {'name': 'Quanxin Wa...</td>\n",
       "      <td>&lt;p&gt;Identification of the structural connection...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.09.29.462384</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Bromodomain factor 5 is an essential transcrip...</td>\n",
       "      <td>[{'name': 'Nathaniel G. Jones'}, {'name': 'Vin...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Leishmania&lt;/italic&gt; are unicellular...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.09.28.462081</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Performance-optimized neural networks as an ex...</td>\n",
       "      <td>[{'name': 'Taylor W. Webb'}, {'name': 'Kiyofum...</td>\n",
       "      <td>&lt;p&gt;Previous work has sought to understand deci...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.09.30.461806</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Progranulin deficiency results in reduced bis(...</td>\n",
       "      <td>[{'name': 'Sebastian Boland'}, {'name': 'Shara...</td>\n",
       "      <td>&lt;p&gt;Homozygous mutations of granulin precursor ...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.09.30.462663</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>A gut commensal niche regulates stable associa...</td>\n",
       "      <td>[{'name': 'Ren Dodge'}, {'name': 'Eric W. Jone...</td>\n",
       "      <td>&lt;p&gt;The intestines of animals are typically col...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.09.29.21264089</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Immune Memory Response After a Booster Injecti...</td>\n",
       "      <td>[{'name': 'Laurence Chu'}, {'name': 'David Mon...</td>\n",
       "      <td>&lt;p&gt;Rising breakthrough infections of coronavir...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.09.29.462463</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Optogenetic EB1 inactivation shortens metaphas...</td>\n",
       "      <td>[{'name': 'Alessandro Dema'}, {'name': 'Jeffre...</td>\n",
       "      <td>&lt;p&gt;Chromosome segregation is accomplished by t...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.09.29.21264328</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Endemic Seasonal Coronavirus Neutralisation an...</td>\n",
       "      <td>[{'name': 'David A. Wells'}, {'name': 'Diego C...</td>\n",
       "      <td>&lt;p&gt;The virus SARS-CoV-2, responsible for the g...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.09.29.462364</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Multi-omics and deep learning provide a multif...</td>\n",
       "      <td>[{'name': 'Bora Uyar'}, {'name': 'Jonathan Ron...</td>\n",
       "      <td>&lt;p&gt;Cancer is a complex disease with a large fi...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.09.30.462608</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Cost-effective, high-throughput phenotyping sy...</td>\n",
       "      <td>[{'name': 'Mitchell J. Feldmann'}, {'name': 'A...</td>\n",
       "      <td>&lt;p&gt;Reliable phenotyping methods that are simpl...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.09.28.21264242</td>\n",
       "      <td>5</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Clinical Trial of Efficacy and Toxicity of Dis...</td>\n",
       "      <td>[{'name': 'E.A.G. Arruda'}, {'name': 'R.J. Pir...</td>\n",
       "      <td>&lt;p&gt;This study aimed to evaluate the efficacy a...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.09.28.462138</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The extended Price equation: migration and sex</td>\n",
       "      <td>[{'name': 'Jake Brown'}, {'name': 'Jared M. Fi...</td>\n",
       "      <td>&lt;p&gt;The Price equation provides a general parti...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.09.28.462203</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Numb reduces Tau levels and prevents neurodege...</td>\n",
       "      <td>[{'name': 'Marine Lacomme'}, {'name': 'Sarah C...</td>\n",
       "      <td>&lt;p&gt;Accumulation of the microtubule-associated ...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.09.29.462487</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Endoplasmic reticulum stress activates human I...</td>\n",
       "      <td>[{'name': 'Vladislav Belyy'}, {'name': 'Iratxe...</td>\n",
       "      <td>&lt;p&gt;Protein folding homeostasis in the endoplas...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.09.29.462468</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Distance estimation from monocular cues in an ...</td>\n",
       "      <td>[{'name': 'Philip R. L. Parker'}, {'name': 'El...</td>\n",
       "      <td>&lt;p&gt;In natural contexts, sensory processing and...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.09.29.461531</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Sex Differences in Brain Tumor Glutamine Metab...</td>\n",
       "      <td>[{'name': 'Jasmin Sponagel'}, {'name': 'Jill K...</td>\n",
       "      <td>&lt;p&gt;Sex differences in normal metabolism are we...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.09.29.462396</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Quantifying mediation between omics layers and...</td>\n",
       "      <td>[{'name': 'Marie C. Sadler'}, {'name': 'Chiara...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;High-dimensio...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.09.30.462204</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>An Evolutionarily Conserved Regulatory Pathway...</td>\n",
       "      <td>[{'name': 'Prasanna Katti'}, {'name': 'Peter T...</td>\n",
       "      <td>&lt;p&gt;Mitochondrial networks provide coordinated ...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.09.28.462139</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The spread of the first introns in proto-eukar...</td>\n",
       "      <td>[{'name': 'Julian Vosseberg'}, {'name': 'Miche...</td>\n",
       "      <td>&lt;p&gt;Spliceosomal introns are a unique feature o...</td>\n",
       "      <td>2021-09-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.09.29.462233</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Convergent mosaic brain evolution is associate...</td>\n",
       "      <td>[{'name': 'Erika L. Schumacher'}, {'name': 'Br...</td>\n",
       "      <td>&lt;p&gt;Brain region size generally scales allometr...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.09.27.21263187</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Comprehensive genetic diagnosis of tandem repe...</td>\n",
       "      <td>[{'name': 'Igor Stevanovski'}, {'name': 'Sanjo...</td>\n",
       "      <td>&lt;p&gt;Short-tandem repeat (STR) expansions are an...</td>\n",
       "      <td>2021-10-01</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.09.30.462488     423  bioRxiv   \n",
       "1   10.1101/2021.09.29.21264325     220  medRxiv   \n",
       "2   10.1101/2021.09.30.21264363     197  medRxiv   \n",
       "3   10.1101/2021.09.29.21264272     145  medRxiv   \n",
       "4     10.1101/2021.09.28.462148      83  bioRxiv   \n",
       "5     10.1101/2021.09.30.462231      75  bioRxiv   \n",
       "6   10.1101/2021.09.30.21264376      42  medRxiv   \n",
       "7   10.1101/2021.09.28.21263801      30  medRxiv   \n",
       "8     10.1101/2021.09.30.462538      22  bioRxiv   \n",
       "9     10.1101/2021.09.30.462505      21  bioRxiv   \n",
       "10    10.1101/2021.09.30.462449      14  bioRxiv   \n",
       "11  10.1101/2021.09.28.21264206      12  medRxiv   \n",
       "12    10.1101/2021.09.28.462068      12  bioRxiv   \n",
       "13    10.1101/2021.09.28.462199      11  bioRxiv   \n",
       "14    10.1101/2021.09.30.462010      11  bioRxiv   \n",
       "15  10.1101/2021.09.28.21264208      10  medRxiv   \n",
       "16    10.1101/2021.09.30.462548       9  bioRxiv   \n",
       "17    10.1101/2021.10.01.462730       9  bioRxiv   \n",
       "18    10.1101/2021.09.29.462379       8  bioRxiv   \n",
       "19    10.1101/2021.09.29.462485       8  bioRxiv   \n",
       "20    10.1101/2021.09.28.462132       8  bioRxiv   \n",
       "21    10.1101/2021.09.29.462303       8  bioRxiv   \n",
       "22    10.1101/2021.09.30.462517       8  bioRxiv   \n",
       "23    10.1101/2021.09.29.462495       7  bioRxiv   \n",
       "24    10.1101/2021.10.01.462603       7  bioRxiv   \n",
       "25    10.1101/2021.09.28.462226       7  bioRxiv   \n",
       "26    10.1101/2021.09.28.462180       7  bioRxiv   \n",
       "27    10.1101/2021.09.30.462680       7  bioRxiv   \n",
       "28    10.1101/2021.09.28.462168       6  bioRxiv   \n",
       "29    10.1101/2021.09.29.459010       6  bioRxiv   \n",
       "30    10.1101/2021.09.29.462384       6  bioRxiv   \n",
       "31    10.1101/2021.09.28.462081       6  bioRxiv   \n",
       "32    10.1101/2021.09.30.461806       6  bioRxiv   \n",
       "33    10.1101/2021.09.30.462663       6  bioRxiv   \n",
       "34  10.1101/2021.09.29.21264089       6  medRxiv   \n",
       "35    10.1101/2021.09.29.462463       6  bioRxiv   \n",
       "36  10.1101/2021.09.29.21264328       6  medRxiv   \n",
       "37    10.1101/2021.09.29.462364       5  bioRxiv   \n",
       "38    10.1101/2021.09.30.462608       5  bioRxiv   \n",
       "39  10.1101/2021.09.28.21264242       5  medRxiv   \n",
       "40    10.1101/2021.09.28.462138       5  bioRxiv   \n",
       "41    10.1101/2021.09.28.462203       5  bioRxiv   \n",
       "42    10.1101/2021.09.29.462487       5  bioRxiv   \n",
       "43    10.1101/2021.09.29.462468       5  bioRxiv   \n",
       "44    10.1101/2021.09.29.461531       5  bioRxiv   \n",
       "45    10.1101/2021.09.29.462396       5  bioRxiv   \n",
       "46    10.1101/2021.09.30.462204       5  bioRxiv   \n",
       "47    10.1101/2021.09.28.462139       4  bioRxiv   \n",
       "48    10.1101/2021.09.29.462233       4  bioRxiv   \n",
       "49  10.1101/2021.09.27.21263187       4  medRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0                              Immunology   True   \n",
       "1                            Epidemiology   True   \n",
       "2   Infectious Diseases (except HIV/AIDS)   True   \n",
       "3   Infectious Diseases (except HIV/AIDS)   True   \n",
       "4                            Neuroscience  False   \n",
       "5                            Biochemistry  False   \n",
       "6                Public and Global Health   True   \n",
       "7                            Epidemiology   True   \n",
       "8                              Biophysics  False   \n",
       "9                                Genetics  False   \n",
       "10            Pharmacology and Toxicology   True   \n",
       "11                              Neurology  False   \n",
       "12                           Neuroscience  False   \n",
       "13                           Cell Biology  False   \n",
       "14                           Microbiology  False   \n",
       "15           Genetic and Genomic Medicine  False   \n",
       "16                      Molecular Biology  False   \n",
       "17                           Biochemistry  False   \n",
       "18                           Neuroscience  False   \n",
       "19                         Bioinformatics  False   \n",
       "20                             Immunology  False   \n",
       "21                                Ecology  False   \n",
       "22                           Neuroscience  False   \n",
       "23                          Plant Biology  False   \n",
       "24                           Cell Biology  False   \n",
       "25                  Developmental Biology  False   \n",
       "26                         Bioengineering  False   \n",
       "27                               Genomics  False   \n",
       "28                           Cell Biology  False   \n",
       "29                           Neuroscience  False   \n",
       "30                           Cell Biology  False   \n",
       "31                           Neuroscience  False   \n",
       "32                           Cell Biology  False   \n",
       "33                           Microbiology  False   \n",
       "34  Infectious Diseases (except HIV/AIDS)   True   \n",
       "35                           Cell Biology  False   \n",
       "36  Infectious Diseases (except HIV/AIDS)   True   \n",
       "37                         Bioinformatics  False   \n",
       "38                          Plant Biology  False   \n",
       "39  Infectious Diseases (except HIV/AIDS)   True   \n",
       "40                   Evolutionary Biology  False   \n",
       "41                           Neuroscience  False   \n",
       "42                             Biophysics  False   \n",
       "43                           Neuroscience  False   \n",
       "44                         Cancer Biology  False   \n",
       "45                               Genomics  False   \n",
       "46                           Cell Biology  False   \n",
       "47                   Evolutionary Biology  False   \n",
       "48                   Evolutionary Biology  False   \n",
       "49           Genetic and Genomic Medicine  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Durability of immune responses to the BNT162b2...   \n",
       "1   Assessing the Burden of COVID-19 in Developing...   \n",
       "2   SARS-CoV-2 infections elicit higher levels of ...   \n",
       "3   Associations between SARS-CoV-2 variants and r...   \n",
       "4   Adeno-associated virus (AAV) reduces cortical ...   \n",
       "5     Structures of core eukaryotic protein complexes   \n",
       "6   Estimates of pandemic excess mortality in Indi...   \n",
       "7   Analysis of overdispersion in airborne transmi...   \n",
       "8   New tools for automated cryo-EM single-particl...   \n",
       "9   S. pombe wtf genes use dual transcriptional re...   \n",
       "10  Pyronaridine Protects Against SARS-CoV-2 in Mouse   \n",
       "11  Enhancing the brain’s emotion regulation capac...   \n",
       "12  Here’s the twist: How the brain updates natura...   \n",
       "13  Cell-ACDC: a user-friendly toolset embedding s...   \n",
       "14  Drivers and Determinants of Strain Dynamics Fo...   \n",
       "15  Integration of genetic colocalizations with ph...   \n",
       "16  Prime Editing Efficiency and Fidelity are Enha...   \n",
       "17  Deep mutational scan of a drug efflux pump rev...   \n",
       "18  Modeling spatial, developmental, physiological...   \n",
       "19  Repurposing CAR targets in solid tumors via a ...   \n",
       "20  An anti-inflammatory activation sequence gover...   \n",
       "21  Emerging mosquito resistance to piperonyl buto...   \n",
       "22  Hippocampal CA1 pyramidal cells do not receive...   \n",
       "23  Dominant Awn Inhibitor, which encodes an ALOG ...   \n",
       "24  Regulation of the Evolutionarily Conserved Mus...   \n",
       "25  Osteoblasts pattern endothelium and somatosens...   \n",
       "26  Extracellular Electron Transfer Enables Cellul...   \n",
       "27  Evolutionary insights into primate skeletal ge...   \n",
       "28  Neurofibromin 1 controls metabolic balance and...   \n",
       "29  A whole-brain monosynaptic input connectome to...   \n",
       "30  Bromodomain factor 5 is an essential transcrip...   \n",
       "31  Performance-optimized neural networks as an ex...   \n",
       "32  Progranulin deficiency results in reduced bis(...   \n",
       "33  A gut commensal niche regulates stable associa...   \n",
       "34  Immune Memory Response After a Booster Injecti...   \n",
       "35  Optogenetic EB1 inactivation shortens metaphas...   \n",
       "36  Endemic Seasonal Coronavirus Neutralisation an...   \n",
       "37  Multi-omics and deep learning provide a multif...   \n",
       "38  Cost-effective, high-throughput phenotyping sy...   \n",
       "39  Clinical Trial of Efficacy and Toxicity of Dis...   \n",
       "40     The extended Price equation: migration and sex   \n",
       "41  Numb reduces Tau levels and prevents neurodege...   \n",
       "42  Endoplasmic reticulum stress activates human I...   \n",
       "43  Distance estimation from monocular cues in an ...   \n",
       "44  Sex Differences in Brain Tumor Glutamine Metab...   \n",
       "45  Quantifying mediation between omics layers and...   \n",
       "46  An Evolutionarily Conserved Regulatory Pathway...   \n",
       "47  The spread of the first introns in proto-eukar...   \n",
       "48  Convergent mosaic brain evolution is associate...   \n",
       "49  Comprehensive genetic diagnosis of tandem repe...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Mehul S. Suthar'}, {'name': 'Prabhu...   \n",
       "1   [{'name': 'Andrew Levin'}, {'name': 'Nana Owus...   \n",
       "2   [{'name': 'Elizabeth M. Anderson'}, {'name': '...   \n",
       "3   [{'name': 'Miguel I. Paredes'}, {'name': 'Step...   \n",
       "4   [{'name': 'Christos M. Suriano'}, {'name': 'Je...   \n",
       "5   [{'name': 'Ian R. Humphreys'}, {'name': 'Jimin...   \n",
       "6   [{'name': 'Murad Banaji'}, {'name': 'Aashish G...   \n",
       "7   [{'name': 'Swetaprovo Chaudhuri'}, {'name': 'P...   \n",
       "8   [{'name': 'Dari Kimanius'}, {'name': 'Liyi Don...   \n",
       "9   [{'name': 'Nicole L. Nuckolls'}, {'name': 'Ana...   \n",
       "10  [{'name': 'Ana C. Puhl'}, {'name': 'Giovanni F...   \n",
       "11  [{'name': 'Kaoru Nashiro'}, {'name': 'Jungwon ...   \n",
       "12  [{'name': 'Asieh Zadbood'}, {'name': 'Samuel A...   \n",
       "13  [{'name': 'Francesco Padovani'}, {'name': 'Ben...   \n",
       "14  [{'name': 'Thomas SB Schmidt'}, {'name': 'Simo...   \n",
       "15  [{'name': 'Michael J. Gloudemans'}, {'name': '...   \n",
       "16  [{'name': 'J Ferreira da Silva'}, {'name': 'GP...   \n",
       "17  [{'name': 'Gianmarco Meier'}, {'name': 'Sujani...   \n",
       "18  [{'name': 'S. Oldham'}, {'name': 'B. D. Fulche...   \n",
       "19  [{'name': 'Sanna Madan'}, {'name': 'Alejandro ...   \n",
       "20  [{'name': 'Nicolas Denans'}, {'name': 'Nhung T...   \n",
       "21  [{'name': 'Guofa Zhou'}, {'name': 'Yiji Li'}, ...   \n",
       "22  [{'name': 'Lilya Andrianova'}, {'name': 'Erica...   \n",
       "23  [{'name': 'Hideki Takanashi'}, {'name': 'Hirom...   \n",
       "24  [{'name': 'Peter T. Ajayi'}, {'name': 'Prasann...   \n",
       "25  [{'name': 'Rosalind G. Bump'}, {'name': 'Camil...   \n",
       "26  [{'name': 'Gina Partipilo'}, {'name': 'Austin ...   \n",
       "27  [{'name': 'Genevieve Housman'}, {'name': 'Emil...   \n",
       "28  [{'name': 'Xiaoyan Wei'}, {'name': 'Matthias L...   \n",
       "29  [{'name': 'Shenqin Yao'}, {'name': 'Quanxin Wa...   \n",
       "30  [{'name': 'Nathaniel G. Jones'}, {'name': 'Vin...   \n",
       "31  [{'name': 'Taylor W. Webb'}, {'name': 'Kiyofum...   \n",
       "32  [{'name': 'Sebastian Boland'}, {'name': 'Shara...   \n",
       "33  [{'name': 'Ren Dodge'}, {'name': 'Eric W. Jone...   \n",
       "34  [{'name': 'Laurence Chu'}, {'name': 'David Mon...   \n",
       "35  [{'name': 'Alessandro Dema'}, {'name': 'Jeffre...   \n",
       "36  [{'name': 'David A. Wells'}, {'name': 'Diego C...   \n",
       "37  [{'name': 'Bora Uyar'}, {'name': 'Jonathan Ron...   \n",
       "38  [{'name': 'Mitchell J. Feldmann'}, {'name': 'A...   \n",
       "39  [{'name': 'E.A.G. Arruda'}, {'name': 'R.J. Pir...   \n",
       "40  [{'name': 'Jake Brown'}, {'name': 'Jared M. Fi...   \n",
       "41  [{'name': 'Marine Lacomme'}, {'name': 'Sarah C...   \n",
       "42  [{'name': 'Vladislav Belyy'}, {'name': 'Iratxe...   \n",
       "43  [{'name': 'Philip R. L. Parker'}, {'name': 'El...   \n",
       "44  [{'name': 'Jasmin Sponagel'}, {'name': 'Jill K...   \n",
       "45  [{'name': 'Marie C. Sadler'}, {'name': 'Chiara...   \n",
       "46  [{'name': 'Prasanna Katti'}, {'name': 'Peter T...   \n",
       "47  [{'name': 'Julian Vosseberg'}, {'name': 'Miche...   \n",
       "48  [{'name': 'Erika L. Schumacher'}, {'name': 'Br...   \n",
       "49  [{'name': 'Igor Stevanovski'}, {'name': 'Sanjo...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>The development of the highly efficacious m...  2021-09-30  \n",
       "1   <sec><title>Introduction</title><p>The infecti...  2021-10-01  \n",
       "2   <p>Severe acute respiratory syndrome coronavir...  2021-10-01  \n",
       "3   <sec><title>Background</title><p>The COVID-19 ...  2021-09-30  \n",
       "4   <p>Recombinant adeno-associated viruses (AAVs)...  2021-09-30  \n",
       "5   <p>Protein-protein interactions play critical ...  2021-09-30  \n",
       "6   <sec><title>Background</title><p>The COVID-19 ...  2021-10-01  \n",
       "7   <p>Superspreading events and overdispersion ar...  2021-09-30  \n",
       "8   <label>1</label><title>Abstract</title><p>We d...  2021-09-30  \n",
       "9   <p>Meiotic drivers bias gametogenesis to ensur...  2021-10-01  \n",
       "10  <p>There are currently relatively few small-mo...  2021-09-30  \n",
       "11  <p>Heart rate variability is a robust biomarke...  2021-09-30  \n",
       "12  <p>The brain actively reshapes past memories i...  2021-09-30  \n",
       "13  <p>Live-cell imaging is a powerful tool to stu...  2021-09-30  \n",
       "14  <p>Faecal microbiota transplantation (FMT) is ...  2021-09-30  \n",
       "15  <sec><title>Background</title><p>Identificatio...  2021-09-30  \n",
       "16  <p>Prime editing is a powerful genome engineer...  2021-09-30  \n",
       "17  <p>Drug efflux is a common resistance mechanis...  2021-10-01  \n",
       "18  <p>The complex connectivity of nervous systems...  2021-10-01  \n",
       "19  <p>Chimeric antigen receptor (CAR) T cell ther...  2021-10-01  \n",
       "20  <p>Macrophages are essential for tissue repair...  2021-09-30  \n",
       "21  <p>Piperonyl butoxide (PBO)-synergized pyrethr...  2021-10-01  \n",
       "22  <p>Thalamic nucleus reuniens (NRe) is located ...  2021-10-01  \n",
       "23  <p>The awn, a needle-like structure that exten...  2021-10-01  \n",
       "24  <p>Skeletal muscles play a central role in hum...  2021-10-01  \n",
       "25  <p>Skeletal elements frequently associate with...  2021-09-30  \n",
       "26  <p>Extracellular electron transfer (EET) is an...  2021-09-30  \n",
       "27  <p>The evolution of complex skeletal traits in...  2021-10-01  \n",
       "28  <p>Patients affected by neurofibromatosis type...  2021-09-30  \n",
       "29  <p>Identification of the structural connection...  2021-10-01  \n",
       "30  <p><italic>Leishmania</italic> are unicellular...  2021-09-30  \n",
       "31  <p>Previous work has sought to understand deci...  2021-09-30  \n",
       "32  <p>Homozygous mutations of granulin precursor ...  2021-09-30  \n",
       "33  <p>The intestines of animals are typically col...  2021-09-30  \n",
       "34  <p>Rising breakthrough infections of coronavir...  2021-10-01  \n",
       "35  <p>Chromosome segregation is accomplished by t...  2021-10-01  \n",
       "36  <p>The virus SARS-CoV-2, responsible for the g...  2021-10-01  \n",
       "37  <p>Cancer is a complex disease with a large fi...  2021-10-01  \n",
       "38  <p>Reliable phenotyping methods that are simpl...  2021-10-01  \n",
       "39  <p>This study aimed to evaluate the efficacy a...  2021-09-30  \n",
       "40  <p>The Price equation provides a general parti...  2021-09-30  \n",
       "41  <p>Accumulation of the microtubule-associated ...  2021-09-30  \n",
       "42  <p>Protein folding homeostasis in the endoplas...  2021-09-30  \n",
       "43  <p>In natural contexts, sensory processing and...  2021-10-01  \n",
       "44  <p>Sex differences in normal metabolism are we...  2021-10-01  \n",
       "45  <sec><title>Background</title><p>High-dimensio...  2021-10-01  \n",
       "46  <p>Mitochondrial networks provide coordinated ...  2021-10-01  \n",
       "47  <p>Spliceosomal introns are a unique feature o...  2021-09-30  \n",
       "48  <p>Brain region size generally scales allometr...  2021-10-01  \n",
       "49  <p>Short-tandem repeat (STR) expansions are an...  2021-10-01  "
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "92 preprints (including 17 covering SARS-CoV-2, 77 from bioRxiv and 15 from medRxiv) published in the last 7 days before 2021-10-07 had been tweeted at least 3 times (maximum 423)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
