{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "17430 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c1d71dfa-bbc6-4c7a-a7a1-b499b7313891&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d36bf7ad-3f1d-4b62-9dfd-a2c5742a0e86&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=19851a04-ac99-4cc5-994b-c94fdd14a9c4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=202887b1-5616-45aa-a7fb-f7f98e413e1b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a9f19cec-860d-434f-9d42-baa2f389c6bd&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e16bfe3f-5934-4a23-9b0b-bddd09eedc4c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b756a70b-661c-49f0-8a88-4104ac5636fb&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=817cd53d-c235-4d14-bd6b-d6df2c7b4aef&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=33b5c852-3293-4d3a-894a-4a73971af4d9&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=16594b0f-2bf8-4adc-8b05-7a72535f1004&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9c67e68a-cf49-4ee8-98f8-2ac0c382af71&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7c801257-bd37-4e98-adc6-d6532630ef5f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=873a7943-3ca4-4770-90d3-da57d17cdab3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=8383dd51-6d90-44e7-a618-9b1ef3f78a0d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b3ed2703-bafd-43b2-86f2-145fc5615000&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c764728c-241f-423f-8ab5-b59b79021795&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=58694150-dba8-4dc6-9a40-2a2c03ba3c31&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-22&until-occurred-date=2021-11-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-10-18.csv\n",
      "failed to load preprint_tweets_2021-10-25.csv\n",
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load preprint_tweets_2021-10-23.csv\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-11-01.csv\n",
      "failed to load preprint_tweets_2021-11-15.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-11-08.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load preprint_tweets_2021-11-22.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21263608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.21266484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469078...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469770...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266775...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469394...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468749...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.468228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275373.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.21265383...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.468420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.27.469863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469412...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.465477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.468884...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.174888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.440409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.438036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.420448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466306...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.468910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273300.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469669...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.21266229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/240317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455968...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.424451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.28.923532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.468202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.466958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469335...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439842...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469860...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275569.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.459575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.372219...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469104...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469797...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.469928...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.264705.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264626...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.468539...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469695...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.367649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.05.20088757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.413609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21258385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.13.414557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.241414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469682...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.469979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469435...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470178...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.21266863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469458...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470087...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21266262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20043919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265397...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275380.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21262911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.352914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.468818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469585...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275752.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.21262938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266681...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.21262020...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.31.116061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.21254323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.427276...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266512...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.437258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470088...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468777...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467405...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275325.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469570...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.163816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.469984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.21266552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469572...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.314831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/530097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469276...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469842...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20213546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20222778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469476...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.21263630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266676...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469446...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.437719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.21261741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469506...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469541...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.382143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.21265038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464700...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.08.191304...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.223024...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.11.451967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.413229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468360...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/306019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.424082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.431294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.21263149...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.469673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469043...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455235...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.05.447065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266688...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21262579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.21249324...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469626...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.15.275891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.468991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.27.470071...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.10.20021675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/789289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.12.380311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.429177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.30.066209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.17.20176636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.198549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469434...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.465626...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469775...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469172...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.20245993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470073...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.468873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469039...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/743294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.469203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.18.20024364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20120857...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.21266571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.14.297143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.388504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470170...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468761...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21266441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469221...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268482.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21266461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470078...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21250921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.26.424423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.424510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266682...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.225193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.399154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.08.139329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.21255690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266761...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.21263783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469750...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.27.470186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275308.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.462574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.021006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470104...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469676...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.442971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.16.20037135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.29.20115352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.462716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.20204693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.335083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261955...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469674...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458476...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.06.028712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21263014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/769083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.385401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.389031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.09.374488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21258418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.21266090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.469965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.469972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467238...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.21249236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460099...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.464426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468963...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.20086439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21261811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.21263359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.469692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.449999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469529...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469650...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.21258079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.235549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.21264927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.437037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.21266655...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470264...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.434372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265094...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.03.11.21253207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.468946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266713...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.20142703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.21252928...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.442295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.459429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470085...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.30.272237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.21.914044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/135582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21258780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469612...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.440671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275784.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.461806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.11.21255260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.459925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.12.21251599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.440511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.460970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.073262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20088260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.464936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.18.255570...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/768663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.353698...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.27.470183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.10.377358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468056...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.468142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.03.20167395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469894...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.468236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.464002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.26.062158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.467798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458273...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20124461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.10.079236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262955...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.298166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.21265575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.20192815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.14.20212555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20131243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461071...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.21263548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.470057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466797...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/276717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20198796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20130294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466884...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458953...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20216895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/613141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467773...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.30.20143289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265945...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/708255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.469847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21256632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21265758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.449462...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469700...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.447579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.21264092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.21.21251403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.21266534...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.21266561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.21266594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.426911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469777...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.426437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469088...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469749...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.261305.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.21250507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.451212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.441209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21253893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441655...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275648.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.27.21254452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.441050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/038232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.432674...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21265452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.444811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.434390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266683...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.29.926022...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.463791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468904...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21266051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.20248876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.462870...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.20241927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.17.386995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.20.052019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.21262393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468676...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.14.20062463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.21266265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.02.21249146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453601...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.11.24.21266735', 'tweets': 1266, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Increased risk of infection with SARS-CoV-2 Beta, Gamma, and Delta variant compared to Alpha variant in vaccinated individuals', 'authors': \"[{'name': 'Stijn P. Andeweg'}, {'name': 'Harry Vennema'}, {'name': 'Irene Veldhuijzen'}, {'name': 'Naomi Smorenburg'}, {'name': 'Dennis Schmitz'}, {'name': 'Florian Zwagemaker'}, {'name': 'Arianne B. van Gageldonk-Lafeber'}, {'name': 'Susan J.M. Hahn'}, {'name': 'Chantal Reusken'}, {'name': 'Mirjam J. Knol'}, {'name': 'Dirk Eggink'}, {'name': ' '}, {'name': ' '}]\", 'abstract': '<p>The extent to which severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants of concern (VOC) break through infection- or vaccine-induced immunity is not well understood. Here, we analyze 28,578 sequenced SARS-CoV-2 samples from individuals with known immune status obtained through national community testing in the Netherlands from March to August 2021. We find evidence for an increased risk of infection by the Beta (B.1.351), Gamma (P.1), or Delta (B.1.617.2) variants compared to the Alpha (B.1.1.7) variant after vaccination. No clear differences were found between vaccines. However, the effect was larger in the first 14-59 days after complete vaccination compared to 60 days and longer. In contrast to vaccine-induced immunity, no increased risk for reinfection with Beta, Gamma or Delta variants relative to Alpha variant was found in individuals with infection-induced immunity.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.21266484', 'tweets': 233, 'archive': 'medRxiv', 'subject-area': 'Pediatrics', 'covid': True, 'title': 'Durability of SARS-CoV-2 Antibodies from Natural Infection in Children and Adolescents', 'authors': \"[{'name': 'Sarah E. Messiah'}, {'name': 'Frances Brito'}, {'name': 'Harold W. Kohl'}, {'name': 'Stacia DeSantis'}, {'name': 'Melissa Valerio-Shewmaker'}, {'name': 'Jessica Ross'}, {'name': 'Michael D. Swartz'}, {'name': 'Ashraf Yaseen'}, {'name': 'Steven H. Kelder'}, {'name': 'Shiming Zhang'}, {'name': 'Onyinye S Omega-Njemnobi'}, {'name': 'Michael O. Gonzalez'}, {'name': 'Leqing Wu'}, {'name': 'Eric Boerwinkle'}, {'name': 'David Lakey'}, {'name': 'Jennifer A. Shuford'}, {'name': 'Stephen J. Pont'}]\", 'abstract': '<sec><title>Background</title><p>Recent data suggest the SARS-CoV-2 Delta (B.1.617.2) variant is more transmissible among children compared to the Alpha (B.1.1.7) variant. The true incidence and longitudinal presence of antibody response to SARS-CoV-2 infection is not known, however. We provided estimates of antibody response using Texas Coronavirus Antibody REsponse Survey (Texas CARES) data, a prospective population-based seroprevalence project designed to assess antibody status over time among the general population throughout the state.</p></sec><sec><title>Methods</title><p>In October 2020 Texas CARES began enrolling adults (aged 20-80 years) and children (aged 5-19 years). Participants were offered a series of three SARS-CoV-2 antibody tests over 6-8 months, or every 2-3 months that includes the immunoassay for detection of antibodies to the SARS-CoV-2 nucleocapsid protein (Roche N-test). Descriptive characteristics and COVID-19 infection-related symptom status was determined by questionnaire at the time of enrollment and prior to each successive blood draw. This analysis included participants ages 5-to-19 years old who have completed all three antibody assessments.</p></sec><sec><title>Results</title><p>From our sample (n=159; mean age 12.5 years, SD 3.6), 96% of those with evidence of nucleocapsid antibodies at baseline assessment continued to have antibodies &gt; six months later (mean 7.0 months, SD 0.97). There was no difference in the presence of antibodies by symptom status (asymptotic versus symptomatic) or severity (mild-moderate versus severe), sex, age group, or body mass index group (underweight, healthy weight, overweight, obesity) over the three antibody measurement timepoints.</p></sec><sec><title>Conclusions</title><p>These results suggest that infection-induced antibodies persist and thus may provide some protection against future infection for at least half a year. 57.9% of the sample were negative for infection-induced antibodies at their third measurement point, suggesting a significant proportion of children have still not acquired natural infection.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469536', 'tweets': 179, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Sampling the conformational landscapes of transporters and receptors with AlphaFold2', 'authors': \"[{'name': 'Diego del Alamo'}, {'name': 'Davide Sala'}, {'name': 'Hassane S. Mchaourab'}, {'name': 'Jens Meiler'}]\", 'abstract': '<p>Equilibrium fluctuations and triggered conformational changes often underlie the functional cycles of membrane proteins. For example, transporters mediate the passage of molecules across cell membranes by alternating between inward-facing (IF) and outward-facing (OF) states, while receptors undergo intracellular structural rearrangements that initiate signaling cascades. Although the conformational plasticity of these proteins has historically posed a challenge for traditional <italic>de novo</italic> protein structure prediction pipelines, the recent success of AlphaFold2 (AF2) in CASP14 culminated in the modeling of a transporter in multiple conformations to high accuracy. Given that AF2 was designed to predict static structures of proteins, it remains unclear if this result represents an underexplored capability to accurately predict multiple conformations and/or structural heterogeneity. Here, we present an approach to drive AF2 to sample alternative conformations of topologically diverse transporters and G-protein coupled receptors (GPCRs) that are absent from the AF2 training set. Whereas models generated using the default AF2 pipeline are conformationally homogeneous and nearly identical to one another, reducing the depth of the input multiple sequence alignments (MSAs) led to the generation of accurate models in multiple conformations. In our benchmark, these conformations were observed to span the range between two experimental structures of interest, suggesting that our protocol allows sampling of the conformational landscape at the energy minimum. Nevertheless, our results also highlight the need for the next generation of deep learning algorithms to be designed to predict ensembles of biophysically relevant states.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469762', 'tweets': 146, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'In vitro and in vivo CRISPR-Cas9 screens reveal drivers of aging in neural stem cells of the brain', 'authors': \"[{'name': 'Tyson J. Ruetz'}, {'name': 'Chloe M. Kashiwagi'}, {'name': 'Bhek Morton'}, {'name': 'Robin W. Yeo'}, {'name': 'Dena S. Leeman'}, {'name': 'David W. Morgens'}, {'name': 'C. Kimberly Tsui'}, {'name': 'Amy Li'}, {'name': 'Michael C. Bassik'}, {'name': 'Anne Brunet'}]\", 'abstract': '<p>Aging impairs the ability of neural stem cells to transition from quiescence to activation (proliferation) in the adult mammalian brain. Neural stem cell (NSC) functional decline results in decreased production of new neurons and defective regeneration upon injury during aging<sup>19</sup>, and this is exacerbated in Alzheimers disease<sup>10</sup>. Many genes are upregulated with age in NSCs<sup>3, 1113</sup>, and the knockout of some of these boosts old NSC activation and rejuvenates aspects of old brain function<sup>1418</sup>. But systematic functional testing of genes in old NSCs  and more generally in old cells  has not been done. This has been a major limiting factor in identifying the most promising rejuvenation interventions. Here we develop <italic>in vitro</italic> and <italic>in vivo</italic> high-throughput CRISPR-Cas9 screening platforms to systematically uncover gene knockouts that boost NSC activation in old mice. Our genome-wide screening pipeline in primary cultures of young and old NSCs identifies over 300 gene knockouts that specifically restore old NSC activation. Interestingly, the top gene knockouts are involved in glucose import, cilium organization and ribonucleoprotein structures. To determine which gene knockouts have a rejuvenating effect for the aging brain, we establish a scalable CRISPR-Cas9 screening platform <italic>in vivo</italic> in old mice. Of the 50 gene knockouts we tested <italic>in vivo</italic>, 23 boost old NSC activation and production of new neurons in old brains. Notably, the knockout of <italic>Slc2a4</italic>, which encodes for the GLUT4 glucose transporter, is a top rejuvenating intervention for old NSCs. GLUT4 protein expression increases in the stem cell niche during aging, and we show that old NSCs indeed uptake 2-fold more glucose than their young counterparts. Transient glucose starvation increases the ability of old NSCs to activate, which is not further improved by knockout of <italic>Slc2a4/</italic>GLUT4. Together, these results indicate that a shift in glucose uptake contributes to the decline in NSC activation with age, but that it can be reversed by genetic or external interventions. Importantly, our work provides scalable platforms to systematically identify genetic interventions that boost old NSC function, including <italic>in vivo</italic> in old brains, with important implications for regenerative and cognitive decline during aging.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469533', 'tweets': 143, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The Unique Cytoarchitecture and Wiring of the Human Default Mode Network', 'authors': \"[{'name': 'Casey Paquola'}, {'name': 'Margaret Garber'}, {'name': 'Stefan Frssle'}, {'name': 'Jessica Royer'}, {'name': 'Shahin Tavakol'}, {'name': 'Raul Rodriguez-Cruces'}, {'name': 'Elizabeth Jeffries'}, {'name': 'Jonathan Smallwood'}, {'name': 'Boris Bernhardt'}]\", 'abstract': '<p>It is challenging to specify the role of the default mode network (DMN) in human behaviour. Contemporary theories, based on functional magnetic resonance imaging (MRI), suggest that the DMN is insulated from the external world, which allows it to support perceptually-decoupled states and to integrate external and internal information in the construction of abstract meanings. To date, the neuronal architecture of the DMN has received relatively little attention. Understanding the cytoarchitectural composition and connectional layout of the DMN will provide novel insights into its role in brain function. We mapped cytoarchitectural variation within the DMN using a cortical type atlas and a histological model of the entire human brain. Next, we used MRI acquired in healthy young adults to explicate structural wiring and effective connectivity. We discovered profound diversity of DMN cytoarchitecture. Connectivity is organised along the most dominant cytoarchitectural axis. One side of the axis is the prominent receiver, whereas the other side remains more insulated, especially from sensory areas. The structural heterogeneity of the DMN engenders a network-level balance in communication with external and internal sources, which is distinctive, relative to other functional communities. The neuronal architecture of the DMN suggests it is a protuberance from the core cortical processing hierarchy and holds a unique role in information integration.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469481', 'tweets': 107, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'All-optical electrophysiology with improved genetically encoded voltage indicators reveals interneuron network dynamics in vivo', 'authors': \"[{'name': 'He Tian'}, {'name': 'Hunter C. Davis'}, {'name': 'J. David Wong-Campos'}, {'name': 'Linlin Z. Fan'}, {'name': 'Benjamin Gmeiner'}, {'name': 'Shahinoor Begum'}, {'name': 'Christopher A. Werley'}, {'name': 'Gabriel B. Borja'}, {'name': 'Hansini Upadhyay'}, {'name': 'Himali Shah'}, {'name': 'Jane Jacques'}, {'name': 'Pojeong Park'}, {'name': 'Yitong Qi'}, {'name': 'Vicente Parot'}, {'name': 'Karl Deisseroth'}, {'name': 'Adam E. Cohen'}]\", 'abstract': '<p>All-optical electrophysiology can be a powerful tool for studying neural dynamics <italic>in vivo</italic>, as it offers the ability to image and perturb membrane voltage in multiple cells simultaneously. The Optopatch constructs combine a red-shifted archaerhodopsin (Arch)-derived genetically encoded voltage indicator (GEVI) with a blue-shifted channelrhodopsin actuator (ChR). We used a video-based pooled screen to evolve Arch-derived GEVIs with improved signal-to-noise ratio (QuasAr6a) and kinetics (QuasAr6b). By combining optogenetic stimulation of individual cells with high-precision voltage imaging in neighboring cells, we mapped inhibitory and gap junction-mediated connections, <italic>in vivo</italic>. Optogenetic activation of a single NDNF-expressing neuron in visual cortex Layer 1 significantly suppressed the spike rate in some neighboring NDNF interneurons. Hippocampal PV cells showed near-synchronous spikes across multiple cells at a frequency significantly above what one would expect from independent spiking, suggesting that collective inhibitory spikes may play an important signaling role <italic>in vivo</italic>. By stimulating individual cells and recording from neighbors, we quantified gap junction coupling strengths. Together, these results demonstrate powerful new tools for all-optical microcircuit dissection in live mice.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469770', 'tweets': 101, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Characterization of the immune resistance of SARS-CoV-2 Mu variant and the immunity induced by Mu infection', 'authors': \"[{'name': 'Keiya Uriu'}, {'name': 'Pal Crdenas'}, {'name': 'Erika Muoz'}, {'name': 'Veronica Barragan'}, {'name': 'Yusuke Kosugi'}, {'name': 'Kotaro Shirakawa'}, {'name': 'Akifumi Takaori-Kondo'}, {'name': 'Kei Sato'}, {'name': ' '}, {'name': ' '}]\", 'abstract': '<p>We have revealed that the SARS-CoV-2 Mu variant is highly resistant to COVID-19 convalescent sera and vaccine sera.<sup>1</sup> However, it remains unclear how the immune resistance of the Mu variant is determined. Also, although the Mu variant is highly resistant to the sera obtained from COVID-19 convalescent during early pandemic (i.e., infected with prototypic virus) and vaccinated individuals (i.e., immunized based on prototypic virus), it was unaddressed how the convalescent sera from Mu-infected individuals function. In this study, we revealed that the two mutations in the spike protein of Mu variant, YY144-145TSN and E484K, are responsible for the potent immune resistance of Mu variant. Additionally, we showed that the convalescent sera obtained from the Mu-infected individuals can be broadly antiviral against the Mu variant as well as other SARS-CoV-2 variants of concern/interest. Our findings suggest that developing novel vaccines based on the Mu variant can be more effective against a broad range of SARS-CoV-2 variants.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.21266775', 'tweets': 94, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Recurring Spatiotemporal Patterns of COVID-19 in the United States', 'authors': \"[{'name': 'Hawre Jalal'}, {'name': 'Kyueun Lee'}, {'name': 'Donald S. Burke'}]\", 'abstract': '<p>We analyzed the waxing and waning patterns (surges) of reported SARS-CoV-2 cases from January 1, 2020 through Oct 31, 2021 in all states and provinces (n = 93) in the USA, Mexico, and Canada, and across all counties (N = 3142) in the USA. A correlation matrix of the 576  576 daily case incidence rates in the 50 US states generates a distinctive checkerboard pattern showing that the epidemic has consisted of seven distinct internally coherent spatiotemporal wave patterns, four in the first year of the epidemic, and three thus far in the second year. Geoclustering of state case rate trajectories reveals three dominant co-varying spatial clusters of similar case rate trajectories, in the northeastern, southeastern and central/western regions of the USA. The spatiotemporal patterns of epidemic year 1 have thus far been repeated (p&lt;.001) in epidemic year 2. The checkerboard pattern of the correlation matrix of case trajectories can be closely simulated as three sets of interacting sine waves with annual frequencies of 1:1:2 major cycles per year, corresponding to the northeastern, central/western, and southeastern state clusters. Case incidence patterns in Mexico and Canada have been similar to nearby regions in the southern US and the northern US, respectively. Time lapse videos allow visualization of the wave patterns. These highly structured geographical and temporal patterns, coupled with emerging evidence of annual repetition of these same patterns, show that SARS-CoV-2 case rates are driven at least in part by predictable seasonal factors.</p><sec><title>Significance Statement</title><p>Local COVID-19 rates wax and wane. Often these epidemic changes are attributed to localized human behavioral factors. Our finding of highly structured continental scale spatiotemporal patterns that cross state and national boundaries, coupled with emerging evidence of annual repetition of these same patterns, shows that COVID-19 transmission is driven at least in part by seasonal factors. Other epidemic factors such as vaccine coverage rates, or emergence of new strains like the Delta variant of SARS-CoV-2 appear to modify, but not totally eclipse, these underlying seasonal patterns. COVID-19 seasonal transmission patterns are associated with, and may be driven by, seasonal weather patterns. Predictability of these patterns can provide opportunities for forecasting the epidemic and for guiding public health preparedness and control efforts.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.469394', 'tweets': 84, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Miniscope-LFOV: A large field of view, single cell resolution, miniature microscope for wired and wire-free imaging of neural dynamics in freely behaving animals', 'authors': \"[{'name': 'Changliang Guo'}, {'name': 'Garrett J. Blair'}, {'name': 'Megha Sehgal'}, {'name': 'Federico N. Sangiuliano Jimka'}, {'name': 'Arash Bellafard'}, {'name': 'Alcino J. Silva'}, {'name': 'Peyman Golshani'}, {'name': 'Michele A. Basso'}, {'name': 'H. Tad Blair'}, {'name': 'Daniel Aharoni'}]\", 'abstract': '<p>We present a large field of view (FOV) open-source miniature microscope (MiniLFOV) designed to extend the capabilities of the UCLA Miniscope platform to large-scale, single cell resolution neural imaging in freely behaving large rodents and head-fixed mice. This system is capable of multiple imaging configurations, including deep brain imaging using implanted optical probes and cortical imaging through cranial windows. The MiniLFOV interfaces with existing open-source UCLA Miniscope DAQ hardware and software, can achieve single cell resolution imaging across a 3.6  2.7 mm field of view at 23 frames per second, has an electrically adjustable working distance of up to 3.5 mm150 m using an onboard electrowetting lens, incorporates an absolute head-orientation sensor, and weighs under 14 grams. The MiniLFOV provides a 30-fold larger FOV and yields 20-fold better sensitivity than Miniscope V3, and a 12-fold larger FOV with 2-fold better sensitivity than Miniscope V4. Power and data transmission are handled through a single, flexible coaxial cable down to 0.3 mm in diameter facilitating naturalistic behavior. We validated the MiniLFOV in freely behaving rats by simultaneously imaging &gt;1000 GCaMP7s expressing neurons in the CA1 layer of the hippocampus and in head-fixed mice by simultaneously imaging 2000 neurons in the mouse dorsal cortex through a 4  4 mm cranial window. For freely behaving experiments, the MiniLFOV supports optional wire-free operation using a 3.5 g wire-free data acquisition expansion board which enables close to 1-hour of wire-free recording with a 400 mAh (7.5 g) on-board single-cell lithium-polymer battery and expands wire-free imaging techniques to larger animal models. We expect this new open-source implementation of the UCLA Miniscope platform will enable researchers to address novel hypotheses concerning brain function in freely behaving animals.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.24.21266401', 'tweets': 71, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Effectiveness of BNT162b2 and ChAdOx1 against SARS-CoV-2 household transmission: a prospective cohort study in England', 'authors': \"[{'name': 'Samuel Clifford'}, {'name': 'Pauline Waight'}, {'name': 'Jada Hackman'}, {'name': 'Stephane Hu'}, {'name': 'Charlotte M Gower'}, {'name': 'Freja CM Kirsebom'}, {'name': 'Catriona Skarnes'}, {'name': 'Louise Letley'}, {'name': 'Jamie Lopez Bernal'}, {'name': 'Nick Andrews'}, {'name': 'Stefan Flasche'}, {'name': 'Elizabeth Miller'}]\", 'abstract': '<sec><title>Background</title><p>The ability of SARS-CoV-2 vaccines to protect against infection and onward transmission determines whether immunisation can control global circulation. We estimated effectiveness of BNT162b2 and ChAdOx1 vaccines against acquisition and transmission of the Alpha and Delta variants in a prospective household study in England.</p></sec><sec><title>Methods</title><p>Adult index cases in the community and their household contacts took oral-nasal swabs on days 1, 3 and 7 after enrolment. Swabs were tested by RT-qPCR with genomic sequencing conducted on a subset. We used Bayesian logistic regression to infer vaccine effectiveness against acquisition and transmission, adjusted for age, vaccination history and variant.</p></sec><sec><title>Findings</title><p>Between 2 February 2021 and 10 September 2021 213 index cases and 312 contacts were followed up. After excluding households lacking genomic proximity (N=2) or with unlikely serial intervals (N=16), 195 households with 278 contacts remained of whom 113 (41%) became PCR positive. Delta lineages had 1.64 times the risk (95% Credible Interval: 1.15  2.44) of transmission than Alpha; contacts older than 18 years were 1.19 times (1.04 - 1.52) more likely to acquire infection than children. Effectiveness of two doses of BNT162b2 against transmission of Delta was 31% (3%, 61%) and 42% (14%, 69%) for ChAdOx1, similar to their effectiveness for Alpha. Protection against infection with Alpha was higher than for Delta, 71% (12%,95%) vs 24% (2%, 64%) respectively for BNT162b2 and 26% (39%, 73%) vs 14% (5%, 46%) respectively for ChAdOx1.</p></sec><sec><title>Interpretation</title><p>BNT162b2 and ChAdOx1 reduce transmission of the Delta variant from breakthrough infections in the household setting though their protection against infection is low.</p></sec><sec><title>Funding</title><p>This study was funded by the UK Health Security Agency (formerly Public Health England) as part of the COVID-19 response.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469877', 'tweets': 59, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Young transposable elements rewired gene regulatory networks in human and chimpanzee hippocampal intermediate progenitors', 'authors': \"[{'name': 'Sruti Patoori'}, {'name': 'Samantha Barnada'}, {'name': 'Marco Trizzino'}]\", 'abstract': '<p>The hippocampus is associated with essential brain functions such as learning and memory. Human hippocampal volume is significantly greater than expected when compared to non-human apes, suggesting a recent expansion. Intermediate progenitors, which are able to undergo multiple rounds of proliferative division before a final neurogenic division, may have played a role in the evolutionary hippocampal expansion. To investigate the evolution of gene regulatory networks underpinning hippocampal neurogenesis in apes, we leveraged the differentiation of human and chimpanzee induced Pluripotent Stem Cells into TBR2-positive hippocampal intermediate progenitors (hpIPCs). We find that the gene networks active in hpIPCs are significantly different between humans and chimpanzees, with ~2,500 genes differentially expressed. We demonstrate that species-specific transposon-derived enhancers contribute to these transcriptomic differences. Young transposons, predominantly Endogenous Retroviruses (ERVs) and SINE-Vntr-Alus (SVAs), were co-opted as enhancers in a species-specific manner. Human-specific SVAs provided substrates for thousands of novel TBR2 binding sites, and CRISPR-mediated repression of these SVAs attenuates the expression of ~25% of the genes that are upregulated in human intermediate progenitors relative to the same cell population in the chimpanzee.</p><sec><title>Summary statement</title><p>Evolution of human and chimpanzee hippocampal development was mediated by co-option of young retrotransposons into species-specific enhancers.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.21266779', 'tweets': 59, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': True, 'title': 'Severe COVID-19 induces molecular signatures of aging in the human brain', 'authors': \"[{'name': 'Maria Mavrikaki'}, {'name': 'Jonathan D. Lee'}, {'name': 'Isaac H. Solomon'}, {'name': 'Frank J. Slack'}]\", 'abstract': '<p>Coronavirus disease 2019 (COVID-19) is predominantly an acute respiratory disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and remains a significant threat to public health. COVID-19 is accompanied by neurological symptoms and cognitive decline, but the molecular mechanisms underlying this effect remain unclear. As aging induces distinct molecular signatures in the brain associated with cognitive decline in healthy populations, we hypothesized that COVID-19 may induce molecular signatures of aging. Here, we performed whole transcriptomic analysis of human frontal cortex, a critical area for cognitive function, in 12 COVID-19 cases and age- and sex-matched uninfected controls. COVID-19 induces profound changes in gene expression, despite the absence of detectable virus in brain tissue. Pathway analysis shows downregulation of genes involved in synaptic function and cognition and upregulation of genes involved in immune processes. Comparison with five independent transcriptomic datasets of aging human frontal cortex reveals striking similarities between aged individuals and severe COVID-19 patients. Critically, individuals below 65 years of age exhibit profound transcriptomic changes not observed among older individuals in our patient cohort. Our data indicate that severe COVID-19 induces molecular signatures of aging in the human brain and emphasize the value of neurological follow-up in recovered individuals.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.468420', 'tweets': 54, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Plasticity of extrachromosomal and intrachromosomal BRAF amplifications in mediating targeted therapy dosage challenges', 'authors': \"[{'name': 'Kai Song'}, {'name': 'Jenna K. Minami'}, {'name': 'William P. Crosson'}, {'name': 'Jesus Salazar'}, {'name': 'Eli Pazol'}, {'name': 'Niroshi T. Senaratne'}, {'name': 'Nagesh Rao'}, {'name': 'Kim Paraiso'}, {'name': 'Thomas G. Graeber'}]\", 'abstract': '<p>Cancer cells display two modes of focal amplifications (FAs), extrachromosomal DNA/double-minutes (ecDNA/DMs) and intrachromosomal homogenously staining regions (HSRs). Understanding the plasticity of these two modes is critical for preventing targeted therapy resistance. We developed a combined BRAF plus MEK inhibitor resistance melanoma model that bears high BRAF amplifications through both DM and HSR modes, and investigated FA dynamics in the context of drug resistance plasticity. Cells harboring FAs displayed mode switching between DMs and HSRs, from both de novo genetic changes and selection of preexisting subpopulations. We found that copy number plasticity is not exclusive to DMs. Single cell-derived clones with HSRs also exhibit BRAF copy number and corresponding HSR length plasticity that allows them to respond to dose reduction and recover from drug addiction. Upon kinase inhibitor escalation, we observed reproducible selection for cells with BRAF kinase domain duplications residing on DMs. In sum, the plasticity of FAs allows cancer cells to respond to drug dose changes through a myriad of mechanisms. These mechanisms include increases or decreases in DMs, shortening of HSRs, acquisition of secondary resistance mechanisms, and expression of alternative slicing oncogene variants. These results highlight the challenges in targeting the cellular vulnerabilities tied to focal amplifications.</p><sec><title>Statement of Significance</title><p>Understanding the dynamics of oncogene amplifications is critical for appreciating tumorigenesis and preventing anticancer drug resistance. We found melanoma cells harboring BRAF amplifications in either DM or HSR formats to have high plasticity under different kinase inhibitor dosage challenges with evidence supporting de novo alterations, clonal selection, and coupling to additional resistance mechanisms. In in the absence of DMs, HSRs can offer comparable levels of plasticity as DMs.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.27.469863', 'tweets': 51, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A chemical-genetic map of the pathways controlling drug potency in Mycobacterium tuberculosis', 'authors': \"[{'name': 'Shuqi Li'}, {'name': 'Nicholas C Poulton'}, {'name': 'Jesseon S Chang'}, {'name': 'Zachary A Azadian'}, {'name': 'Michael A Dejusus'}, {'name': 'Nadine Ruecker'}, {'name': 'Matthew D Zimmerman'}, {'name': 'Kathryn Eckartt'}, {'name': 'Barbara Bosch'}, {'name': 'Curtis A. Engelhart'}, {'name': 'Daniel Sullivan'}, {'name': 'Martin Gengenbacher'}, {'name': 'Vronique A Dartois'}, {'name': 'Dirk Schnappinger'}, {'name': 'Jeremy M Rock'}]\", 'abstract': '<p><italic>Mycobacterium tuberculosis</italic> infection is notoriously difficult to treat. To define the bacterial determinants that limit treatment efficacy, we developed a CRISPRi chemical genetics platform to titrate the expression of nearly all Mtb genes and quantify bacterial fitness in the presence of different drugs. Mining this dataset, we discovered diverse mechanisms of intrinsic drug resistance, unveiling hundreds of potential targets for synergistic drug combinations. Combining our data with comparative genomics of Mtb clinical isolates, we further identified new mechanisms of acquired drug resistance, one of which is associated with the emergence of a multidrug-resistant tuberculosis (TB) outbreak in South America. Lastly, we make the unexpected discovery of loss-of-function mutations in the intrinsic resistance factor <italic>whiB7</italic> in an entire Mtb sublineage endemic to Southeast Asia, presenting an opportunity to repurpose macrolides to treat TB. This chemical-genetic map provides a rich resource to understand drug efficacy and guide future TB drug development and treatment.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.22.21266690', 'tweets': 49, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Favipiravir for treatment of outpatients with asymptomatic or uncomplicated COVID-19: a double-blind randomized, placebo-controlled, phase 2 trial', 'authors': \"[{'name': 'Marisa Holubar'}, {'name': 'Aruna Subramanian'}, {'name': 'Natasha Purington'}, {'name': 'Haley Hedlin'}, {'name': 'Bryan Bunning'}, {'name': 'Katharine S. Walter'}, {'name': 'Hector Bonilla'}, {'name': 'Athanasia Boumis'}, {'name': 'Michael Chen'}, {'name': 'Kimberly Clinton'}, {'name': 'Liisa Dewhurst'}, {'name': 'Carol Epstein'}, {'name': 'Prasanna Jagannathan'}, {'name': 'Richard H. Kaszynski'}, {'name': 'Lori Panu'}, {'name': 'Julie Parsonnet'}, {'name': 'Elizabeth L. Ponder'}, {'name': 'Orlando Quintero'}, {'name': 'Elizabeth Sefton'}, {'name': 'Upinder Singh'}, {'name': 'Luke Soberanis'}, {'name': 'Henry Truong'}, {'name': 'Jason R. Andrews'}, {'name': 'Manisha Desai'}, {'name': 'Chaitan Khosla'}, {'name': 'Yvonne Maldonado'}]\", 'abstract': '<sec><title>Background</title><p>Favipiravir is an oral, RNA-dependent RNA polymerase inhibitor with <italic>in vitro</italic> activity against SARS-CoV2. Despite limited data, favipiravir is administered to patients with COVID-19 in several countries.</p></sec><sec><title>Methods</title><p>We conducted a phase 2 double-blind randomized controlled outpatient trial of favipiravir in asymptomatic or mildly symptomatic adults with a positive SARS-CoV2 RT-PCR within 72 hours of enrollment. Participants were randomized 1:1 to receive placebo or favipiravir (1800 mg BID Day 1, 800mg BID Days 2-10). The primary outcome was SARS-CoV-2 shedding cessation in a modified intention-to-treat (mITT) cohort of participants with positive enrollment RT-PCRs. Using SARS-CoV-2 deep sequencing, we assessed favipiravirs impact on mutagenesis.</p></sec><sec><title>Results</title><p>From July 8, 2020 - March 23, 2021, we randomized 149 participants with 116 included in the mITT cohort. The participants mean age was 43 years (SD 12.5) and 57 (49%) were women. We found no difference in time to shedding cessation by treatment arm overall (HR 0.76 favoring placebo, 95% confidence interval [CI] 0.48  1.20) or in sub-group analyses (age, sex, high-risk comorbidities, seropositivity or symptom duration at enrollment). We observed no difference in time to symptom resolution (initial: HR 0.84, 95% CI 0.54  1.29; sustained: HR 0.87, 95% CI 0.52  1.45). We detected no difference in accumulation of transition mutations in the viral genome during treatment.</p></sec><sec><title>Conclusions</title><p>Our data do not support favipiravir use at commonly used doses in outpatients with uncomplicated COVID-19. Further research is needed to ascertain if higher doses of favipiravir are effective and safe for patients with COVID-19.</p></sec><sec><title>Trial registration number</title><p>NCT04346628</p></sec><sec><title>Summary</title><p>In this phase 2 double-blind randomized controlled outpatient trial of favipiravir in asymptomatic or uncomplicated patients with COVID-19, we found no difference in time to shedding cessation or time to symptom resolution by treatment arm.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469671', 'tweets': 47, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Inferring kinetic rate constants from single-molecule FRET trajectories  a blind benchmark of kinetic analysis tools', 'authors': \"[{'name': 'Markus Gtz'}, {'name': 'Anders Barth'}, {'name': 'Sren S.-R. Bohr'}, {'name': 'Richard Brner'}, {'name': 'Jixin Chen'}, {'name': 'Thorben Cordes'}, {'name': 'Dorothy A. Erie'}, {'name': 'Christian Gebhardt'}, {'name': 'Mlodie C.A.S. Hadzic'}, {'name': 'George L. Hamilton'}, {'name': 'Nikos S. Hatzakis'}, {'name': 'Thorsten Hugel'}, {'name': 'Lydia Kisley'}, {'name': 'Don C. Lamb'}, {'name': 'Carlos de Lannoy'}, {'name': 'Chelsea Mahn'}, {'name': 'Dushani Dunukara'}, {'name': 'Dick de Ridder'}, {'name': 'Hugo Sanabria'}, {'name': 'Julia Schimpf'}, {'name': 'Claus A.M. Seidel'}, {'name': 'Roland K.O. Sigel'}, {'name': 'Magnus Berg Sletfjerding'}, {'name': 'Johannes Thomsen'}, {'name': 'Leonie Vollmar'}, {'name': 'Simon Wanninger'}, {'name': 'Keith R. Weninger'}, {'name': 'Pengning Xu'}, {'name': 'Sonja Schmid'}]\", 'abstract': '<p>Single-molecule FRET (smFRET) is a versatile technique to study the dynamics and function of biomolecules since it makes nanoscale movements detectable as fluorescence signals. The powerful ability to infer quantitative kinetic information from smFRET data is, however, complicated by experimental limitations. Diverse analysis tools have been developed to overcome these hurdles but a systematic comparison is lacking. Here, we report the results of a blind benchmark study assessing eleven analysis tools used to infer kinetic rate constants from smFRET trajectories. We tested them against simulated and experimental data containing the most prominent difficulties encountered in analyzing smFRET experiments: different noise levels, varied model complexity, non-equilibrium dynamics, and kinetic heterogeneity. Our results highlight the current strengths and limitations in inferring kinetic information from smFRET trajectories. In addition, we formulate concrete recommendations and identify key targets for future developments, aimed to advance our understanding of biomolecular dynamics through quantitative experiment-derived models.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469668', 'tweets': 46, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A bioactive phlebovirus-like envelope protein in a hookworm endogenous virus', 'authors': \"[{'name': 'Monique Merchant'}, {'name': 'Carlos P. Mata'}, {'name': 'Yangci Liu'}, {'name': 'Haoming Zhai'}, {'name': 'Anna V. Protasio'}, {'name': 'Yorgo Modis'}]\", 'abstract': '<p>Endogenous viral elements (EVEs), accounting for 15% of our genome, serve as a genetic reservoir from which new genes can emerge. Nematode EVEs are particularly diverse and informative of virus evolution. We identify Atlas virus  an intact retrovirus-like EVE in the human hookworm Ancylostoma ceylanicum, with an envelope protein genetically related to GN- GC glycoproteins from phleboviruses. A cryo-EM structure of Atlas GC reveals a class II viral membrane fusion protein fold not previously seen in retroviruses. Atlas GC has the structural hallmarks of an active fusogen. Atlas GC trimers insert into membranes with endosomal lipid compositions and low pH. When expressed on the plasma membrane, Atlas GC has cell-cell fusion activity. RNA-Seq data analysis detected transcripts mapping to Atlas virus at different stages of hookworm development. With its preserved biological activities, Atlas GC has the potential to acquire a cellular function. Our work reveals structural plasticity in reverse-transcribing RNA viruses.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266692', 'tweets': 44, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Serological responses to COVID-19 booster vaccine in England', 'authors': \"[{'name': 'Georgina Ireland'}, {'name': 'Heather Whitaker'}, {'name': 'Shamez N Ladhani'}, {'name': 'Frances Baawuah'}, {'name': 'Vani Subbarao'}, {'name': 'Ezra Linley'}, {'name': 'Lenesha Warrener'}, {'name': 'Michelle OBrien'}, {'name': 'Corinne Whillock'}, {'name': 'Paul Moss'}, {'name': 'Mary E Ramsay'}, {'name': 'Gayatri Amirthalingam'}, {'name': 'Kevin E Brown'}]\", 'abstract': '<sec><title>Introduction</title><p>There are limited data on immune responses after COVID-19 vaccine boosters in individuals receiving primary immunisation with BNT162b2 (Pfizer-BioNTech) or AZD1222 (AstraZeneca).</p></sec><sec><title>Methods</title><p>A prospective, cohort study to assess SARS-CoV-2 antibody responses before and after booster vaccination with BNT162b2 in adults receiving either (i) two BNT162b2 doses &lt;30 days apart (BNT162b2-control), (ii) two BNT162b2 doses 30 days apart (BNT162b2-extended) or (iii) two AZD1222 doses 30 days apart (AZD1222-extended) in London, England. SARS-CoV-2 spike protein antibody geometric mean titres (GMTs) before and 2-4 weeks after booster were compared.</p></sec><sec><title>Results</title><p>Of 750 participants, 626 provided serum samples for up to 38 weeks after their second vaccine dose. Antibody GMTs peaked at 2-4 weeks after the second dose, before declining by 68% at 36-38 weeks after dose 2 for BNT162b2-control participants, 85% at 24-29 weeks for BNT162b2-extended participants and 78% at 24-29 weeks for AZD1222-extended participants. Antibody GMTs was highest in BNT162b2-extended participants (942 [95%CI, 797-1113]) than AZD1222-extended (183 [124-268]) participants at 24-29 weeks or BNT162b2-control participants at 36-38 weeks (208; 95%CI, 150-289). At 2-4 weeks after booster, GMTs were significantly higher than after primary vaccination in all three groups: 18,104 (95%CI, 13,911-23,560; n=47) in BNT162b2-control (76.3-fold), 13,980 (11,902-16,421; n=118) in BNT162b2-extended (15.9-fold) and 10,799 (8,510-13,704; n=43) in AZD1222-extended (57.2-fold) participants. BNT162b2-control participants (median:262 days) had a longer interval between primary and booster doses than BNT162b2-extended or AZD1222-extended (both median:186 days) participants.</p></sec><sec><title>Conclusions</title><p>We observed rapid serological responses to boosting with BNT162b2, irrespective of vaccine type or schedule used for primary immunisation, with higher post-booster responses with longer interval between primary immunisation and boosting. Boosters will not only provide additional protection for those at highest risk of severe COVID-19 but also prevent infection and, therefore, interrupt transmission, thereby reducing infections rates in the population. Ongoing surveillance will be important for monitoring the duration of protection after the booster.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469599', 'tweets': 42, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': True, 'title': 'A metagenomic DNA sequencing assay that is robust against environmental DNA contamination', 'authors': \"[{'name': 'Omary Mzava'}, {'name': 'Alexandre Pellan Cheng'}, {'name': 'Adrienne Chang'}, {'name': 'Sami Smalling'}, {'name': 'Liz-Audrey Djomnang Kounatse'}, {'name': 'Joan Lenz'}, {'name': 'Randy Longman'}, {'name': 'Amy Steadman'}, {'name': 'Mirella Salvatore'}, {'name': 'Manikkam Suthanthiran'}, {'name': 'John R. Lee'}, {'name': 'Christopher E. Mason'}, {'name': 'Darshana Dadhania'}, {'name': 'Iwijn De Vlaminck'}]\", 'abstract': '<p>Metagenomic DNA sequencing is a powerful tool to characterize microbial communities but is sensitive to environmental DNA contamination, in particular when applied to samples with low microbial biomass. Here, we present contamination-free metagenomic DNA sequencing (Coffee-seq), a metagenomic sequencing assay that is robust against environmental contamination. The core idea of Coffee-seq is to tag the DNA in the sample prior to DNA isolation and library preparation with a label that can be recorded by DNA sequencing. Any contaminating DNA that is introduced in the sample after tagging can then be bioinformatically identified and removed. We applied Coffee-seq to screen for infections from microorganisms with low burden in blood and urine, to identify COVID-19 co-infection, to characterize the urinary microbiome, and to identify microbial DNA signatures of inflammatory bowel disease in blood.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.21266767', 'tweets': 42, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'An immune correlate of SARS-CoV-2 infection and severity of reinfections', 'authors': \"[{'name': 'Hannah E. Maier'}, {'name': 'Angel Balmaseda'}, {'name': 'Sergio Ojeda'}, {'name': 'Cristiam Cerpas'}, {'name': 'Nery Sanchez'}, {'name': 'Miguel Plazaola'}, {'name': 'Harm van Bakel'}, {'name': 'John Kubale'}, {'name': 'Roger Lopez'}, {'name': 'Saira Saborio'}, {'name': 'Carlos Barilla'}, {'name': 'Eva Harris'}, {'name': 'Guillermina Kuan'}, {'name': 'Aubree Gordon'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>An immune correlate of protection from SARS-CoV-2 infection is urgently needed.</p></sec><sec><title>Methods</title><p>We used an ongoing household cohort with an embedded transmission study that closely monitors participants regardless of symptom status. Real-time reverse-transcription polymerase chain reaction (RT-PCR) and Enzyme-linked immunosorbent assays (ELISAs) were used to measure infections and seropositivity. Sequencing was performed to determine circulating strains of SARS-CoV-2. We investigated the protection associated with seropositivity resulting from prior infection, the anti-spike antibody titers needed for protection, and we compared the severity of first and second infections.</p></sec><sec><title>Results</title><p>In March 2021, 62.3% of the cohort was seropositive. After March 2021, gamma and delta variants predominated. Seropositivity was associated with 69.2% protection from any infection (95% CI: 60.7%-75.9%), with higher protection against moderate or severe infection (79.4%, 95% CI: 64.9%-87.9%). Anti-spike titers of 327 and 2,551 were associated with 50% and 80% protection from any infection; titers of 284 and 656 were sufficient for protection against moderate or severe disease. Second infections were less severe than first infections (Relative Risk (RR) of moderated or severe disease: 0.6, 95% CI: 0.38-0.98; RR of subclinical disease:1.9, 95% CI: 1.33-2.73).</p></sec><sec><title>Conclusions</title><p>Prior infection-induced immunity is protective against infection when predominantly gamma and delta SARS-CoV-2 circulated. The protective antibody titers presented may be useful for vaccine policy and control measures. While second infections were somewhat less severe, they were not as mild as ideal. A strategy involving vaccination will be needed to ease the burden of the SARS-CoV-2 pandemic.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.26.470086', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Multi-state Modeling of G-protein Coupled Receptors at Experimental Accuracy', 'authors': \"[{'name': 'Lim Heo'}, {'name': 'Michael Feig'}]\", 'abstract': '<p>The family of G-protein coupled receptors (GPCRs) is one of the largest protein families in the human genome. GPCRs transduct chemical signals from extracellular to intracellular regions via a conformational switch between active and inactive states upon ligand binding. While experimental structures of GPCRs remain limited, high-accuracy computational predictions are now possible with AlphaFold2. However, AlphaFold2 only predicts one state and is biased towards the inactive conformation. Here, a multi-state prediction protocol is introduced that extends AlphaFold2 to predict either active or inactive states at very high accuracy using state-annotated templated GPCR databases. The predicted models accurately capture the main structural changes upon activation of the GPCR at the atomic level. The models were also highly successful in predicting ligand binding poses via protein-ligand docking. We expect that high accuracy GPCR models in both activation states will promote understanding in GPCR activation mechanisms and drug discovery for GPCRs. At the time, the new protocol paves the way towards capturing the dynamics of proteins at high-accuracy via machine-learning methods.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.24.469859', 'tweets': 38, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'High prevalence of Klebsiella pneumoniae in European food products: a multicentric study comparing culture and molecular detection methods', 'authors': \"[{'name': 'Carla Rodrigues'}, {'name': 'Kathrin Hauser'}, {'name': 'Niamh Cahill'}, {'name': 'Magorzata Ligowska-Marzta'}, {'name': 'Gabriella Centorotola'}, {'name': 'Alessandra Cornacchia'}, {'name': 'Raquel Garcia Fierro'}, {'name': 'Marisa Haenni'}, {'name': 'Eva Mller Nielsen'}, {'name': 'Pascal Piveteau'}, {'name': 'Elodie Barbier'}, {'name': 'Dearbhile Morris'}, {'name': 'Francesco Pomilio'}, {'name': 'Sylvain Brisse'}]\", 'abstract': '<p><italic>Klebsiella pneumoniae</italic> species complex (KpSC) is a leading cause of multidrug-resistant human infections. To better understand the potential contribution of food as a vehicle of KpSC, we conducted a multicentric study to define an optimal culture method for its recovery from food matrices, and to characterize food isolates phenotypically and genotypically. Chicken meat (n=160) and salad (n=145) samples were collected in five European countries and screened for KpSC presence using culture-based and ZKIR qPCR methods. Enrichment using buffered peptone water followed by streaking on Simmons citrate agar with inositol (44C/48h) was defined as the most suitable selective culture method for KpSC recovery. High prevalence of KpSC was found in chicken meat (60% and 52% by ZKIR qPCR and culture approach, respectively) and salad (30% and 21%, respectively) samples. Genomic analyses revealed high genetic diversity with the dominance of phylogroups Kp1 (91%) and Kp3 (6%). 82% of isolates presented a natural antimicrobial susceptibility phenotype and genotype, with only four CTX-M-15-producing isolates detected. Notably, identical genotypes were found across samples: same food type and same country (15 cases); different food types and same country (1); same food type and two countries (1), suggesting high rates of transmission of KpSC within the food sector. Our study provides a novel isolation strategy for KpSC from food matrices and reinforces the view of food as a potential source of KpSC colonization in humans.</p><sec><title>Importance</title><p>Bacteria of the <italic>Klebsiella pneumoniae</italic> species complex (KpSC) are ubiquitous and <italic>K. pneumoniae</italic> (Kp) is a leading cause of antibiotic-resistant infections in humans and animals. Despite the urgent public health threat represented by Kp, there is a lack of knowledge on the contribution of food sources to colonization and subsequent infection in humans. This is partly due to the absence of standardized methods for characterizing KpSC presence in food matrices. Our multicentric study provides and implements a novel isolation strategy for KpSC from food matrices and shows that KpSC members are highly prevalent in salads and chicken meat, reinforcing the view of food as a potential source of KpSC colonization in humans. Despite the large genetic diversity and the low-levels of resistance detected, the occurrence of identical genotypes across samples suggests high rates of transmission of KpSC within the food sector, which need to be further explored to define possible control strategies.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.469166', 'tweets': 36, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Population Imaging of Central Sensitization', 'authors': \"[{'name': 'Charles Warwick'}, {'name': 'Joseph Salsovic'}, {'name': 'Junichi Hachisuka'}, {'name': 'Kelly M. Smith'}, {'name': 'Haichao Chen'}, {'name': 'James Ibinson'}, {'name': 'H. Richard Koerber'}, {'name': 'Sarah E. Ross'}]\", 'abstract': '<p>Capsaicin applied locally to the skin causes central sensitization that results in allodynia, a state in which pain is elicited by innocuous stimuli. Here, we used two-photon calcium imaging of neurons in the dorsal spinal cord to visualize central sensitization across excitatory interneurons and spinal projection neurons. To distinguish among excitatory neuron subtypes, we developed CICADA, a cell profiling approach that leverages the expression of distinct Gq-coupled receptors. We then identified capsaicin-responsive and capsaicin-sensitized neuronal populations. Capsaicin-sensitized neurons showed emergent responses to low threshold input and increased receptive field sizes consistent with the psychophysical phenomenon that allodynia is observed across an extended secondary zone. Finally, we identified spinal projection neurons that showed a shift in tuning toward low threshold input. These experiments provide a population-level view of central sensitization and a framework with which to model somatosensory integration in the dorsal horn.</p><sec><title>Highlight</title><p>Warwick et al. use two-photon calcium imaging coupled with pharmacological profiling to identify neuronal populations in the spinal dorsal horn that mediate capsaicin-induced central sensitization.</p></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.25.468910', 'tweets': 32, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Sex-specific transcriptomic and epitranscriptomic signatures of PTSD-like fear acquisition', 'authors': \"[{'name': 'Andre Martins Reis'}, {'name': 'Jillian Hammond'}, {'name': 'Igor Stevanovski'}, {'name': 'Jonathon C Arnold'}, {'name': 'Iain S. McGregor'}, {'name': 'Ira Deveson'}, {'name': 'Anand Gururajan'}]\", 'abstract': '<p>Our understanding of the molecular pathology of posttraumatic stress disorder (PTSD) is rapidly evolving and is being driven by advances in sequencing techniques. Conventional short-read RNA sequencing (RNA-seq) is a central tool in transcriptomics research that enables unbiased gene expression profiling. With the recent emergence of Oxford Nanopore direct RNA-seq (dRNA-seq), it is now also possible to interrogate diverse RNA modifications, collectively known as the epitranscriptome. Here, we present our analyses of the male and female mouse amygdala transcriptome and epitranscriptome, obtained using parallel Illumina RNA-seq and Oxford Nanopore dRNA-seq, associated with the acquisition of PTSD-like fear induced by Pavlovian cued-fear conditioning. We report significant sex-specific differences in the amygdala transcriptional response during fear acquisition, and a range of shared and dimorphic epitranscriptomic signatures. Differential RNA modifications are enriched among mRNA transcripts associated with neurotransmitter regulation and mitochondrial function, many of which have been previously implicated in PTSD. Very few differentially modified transcripts are also differentially expressed, suggesting an influential, expression-independent role for epitranscriptional regulation in PTSD-like fear-acquisition. Overall, our application of conventional and newly developed methods provides a platform for future work that will lead to new insights into and therapeutics for PTSD.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.21.469478', 'tweets': 32, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Real-time and remote MCMC trace inspection with Beastiary', 'authors': \"[{'name': 'Wytamma Wirth'}, {'name': 'Sebastian Duchene'}]\", 'abstract': '<label>1</label><title>Abstract</title><p>Bayesian phylogenetic methods have gained substantial popularity in the last decade, due to their ability to incorporate independent information and fit complex models. Most Bayesian implementations rely on Markov chain Monte Carlo (MCMC), which in turn requires careful interpretation of the output to assess the statistical validity of any resulting inferences. Here we describe Beastiary, a package for real-time and remote inspection of log flies generated by MCMC analysis commonly utilised in Bayesian phylogenetic analysis. Beastiary is an easily deployed web-sever that can be used to summarise and visualise the output of many popular software packages including BEAST, BEAST2, RevBayes, and MrBayes. We describe the overall design and implementation of Beastiary and some typical use cases, with a particular focus on the capability of monitoring analyses from remote servers.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469684', 'tweets': 32, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Cryo-EM structure of the octameric pore of Clostridium perfringens -toxin', 'authors': \"[{'name': 'Julia Bruggisser'}, {'name': 'Ioan Iacovache'}, {'name': 'Samuel C. Musson'}, {'name': 'Matteo T. Degiacomi'}, {'name': 'Horst Posthaus'}, {'name': 'Benot Zuber'}]\", 'abstract': '<p>We describe the cryo-EM structure of <italic>Clostridium perfringens</italic> -toxin (CBP) in styrene maleic acid (SMA) discs, which represents the membrane-inserted pore form, at near atomic resolution. We show that CPB forms an octamer, which though having a similar conformation to the hetero-oligomeric pores of bicomponent leukocidins, features a different receptor binding region and a novel N-terminal -barrel. The latter contains an additional selectivity filter and creates a bipolar pore. We propose that the N-terminal -barrel domain may regulate oligomerization and solubility of the complex and influence channel conductance and monomer stability. In addition, we show that the -barrel protrusion domain can be modified or exchanged without affecting the pore forming ability, thus making the pore particularly attractive for macromolecule sensing and nanotechnology. The cryo-EM structure of the octameric pore of CPB will facilitate future developments in both nanotechnology and basic research.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469669', 'tweets': 31, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Alpha power and stimulus-evoked activity dissociate metacognitive reports of attention, visibility and confidence in a visual detection task', 'authors': \"[{'name': 'Matthew James Davidson'}, {'name': 'James Macdonald'}, {'name': 'Nick Yeung'}]\", 'abstract': '<p>Variability in the detection and discrimination of weak visual stimuli has been linked to prestimulus neural activity. In particular, the power of oscillatory activity in the alpha-band (8-12 Hz) has been shown to impact upon the objective likelihood of stimulus detection, as well as measures of subjective visibility, attention, and decision confidence. We aimed to clarify how prestimulus alpha influences performance and phenomenology, by recording simultaneous subjective measures of attention and confidence (Experiment 1), or attention and visibility (Experiment  2) on a trial-by-trial basis in a visual detection task. Across both experiments, prestimulus alpha power was negatively and linearly correlated with the intensity of subjective attention. In contrast to this linear relationship, we observed a quadratic relationship between the strength of prestimulus alpha power and subjective ratings of confidence and visibility. We find that this same quadratic relationship links prestimulus alpha power to the strength of stimulus evoked responses. Visibility and confidence judgements corresponded to the strength of evoked responses, but confidence, uniquely, incorporated information about attentional state. As such, our findings reveal distinct psychological and neural correlates of metacognitive judgements of attentional state, stimulus visibility, and decision confidence.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469591', 'tweets': 30, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Chemo-mechanical Diffusion Waves Orchestrate Collective Dynamics of Immune Cell Podosomes', 'authors': \"[{'name': 'Ze Gong'}, {'name': 'Koen van den Dries'}, {'name': 'Alessandra Cambi'}, {'name': 'Vivek B. Shenoy'}]\", 'abstract': '<p>Immune cells, such as macrophages and dendritic cells, can utilize podosomes, actin-rich protrusions, to generate forces, migrate, and patrol for foreign antigens. In these cells, individual podosomes exhibit periodic protrusion and retraction cycles (vertical oscillations) to probe their microenvironment, while multiple podosomes arranged in clusters demonstrate coordinated wave-like spatiotemporal dynamics. However, the mechanisms governing both the individual vertical oscillations and the coordinated oscillation waves in clusters remain unclear. By integrating actin polymerization, myosin contractility, actin diffusion, and mechanosensitive signaling, we develop a chemo-mechanical model for both the oscillatory growth of individual podosomes and wave-like dynamics in clusters. Our model reveals that podosomes show oscillatory growth when the actin polymerization-associated protrusion and the signaling-associated myosin contraction occur at similar rates, while the diffusion of actin monomers within the cluster drives mesoscale coordination of individual podosome oscillations in an apparent wave-like fashion. Our model predicts the influence of different pharmacological treatments targeting myosin activity, actin polymerization, and mechanosensitive pathways, as well as the impact of the microenvironment stiffness on the wavelengths, frequencies, and speeds of the chemo-mechanical waves. Overall, our integrated theoretical and experimental approach reveals how collective wave dynamics arise due to the coupling between chemo-mechanical signaling and actin diffusion, shedding light on the role of podosomes in immune cell mechanosensing within the context of wound healing and cancer immunotherapy.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.20.21266229', 'tweets': 30, 'archive': 'medRxiv', 'subject-area': 'Intensive Care and Critical Care Medicine', 'covid': True, 'title': 'Efficacy and safety of a novel antiviral preparation in ICU-admitted patients with COVID-19: a phase III randomized controlled trial', 'authors': \"[{'name': 'Hossein Faramarzi'}, {'name': 'Amirhossein Sahebkar'}, {'name': 'Ahmad Hosseinpour'}, {'name': 'Vahid Khaloo'}, {'name': 'Parisa Chamanpara'}, {'name': 'Mohammad Reza Heydari'}, {'name': 'Sajad Najafi'}, {'name': 'Fatemeh Fotoohi Khankahdany'}, {'name': 'Ahmad Movahedpour'}]\", 'abstract': '<sec><title>Introduction</title><p>Despite an increasing number of studies, there is as yet no efficient antiviral treatment developed for the disease. In this clinical trial, we examined the efficacy of a novel herbal antiviral preparation comprising <italic>Zataria multiflora Boiss, Glycyrrhiza glabra, Cinnamomum Vermont, Allium sativuml</italic>, and <italic>Syzygium aromaticum</italic> in critically ill patients with COVID-19 patients.</p></sec><sec><title>Methods</title><p>A total number of 120 ICU-admitted patients requiring pulmonary support with a diagnosis of COVID-19 pneumonia were recruited to the trial. Participants were equally randomized to receive either the novel antiviral preparation sublingually, for up to two consecutive weeks or till discharge, or normal saline as the matching placebo. Clinical and laboratory parameters as well as survival rates were compared between the two groups at the study end.</p></sec><sec><title>Results</title><p>The cumulative incidence of death throughout the study period was 8.33% in the medication group and 60% in the placebo group (risk ratio: 0.14; 95% confidence interval [CI], 0.05 to 0.32; P&lt;0.001). Survival rates were significantly higher in the treatment group. Additionally, on day 7, several laboratory factors including white blood cells (WBCs) count, C-reactive protein (CRP), and SpO<sub>2</sub> were improved in patients treated with the novel antiviral preparation compared with the placebo group.</p></sec><sec><title>Conclusion</title><p>The novel antiviral preparation tested in this trial significantly improved the survival rate and reduced mortality in critically ill patients with COVID-19. Thus, this preparation might be suggested as a potentially promising COVID-19 treatment.</p><p>Funded by Shimi Teb Salamat Co., Shiraz, Iran, and registered on the Iranian registry of clinical trials (registration No. IRCT20200509047373N2).</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.21266565', 'tweets': 30, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Impacts of vaccination and asymptomatic testing on SARS-CoV-2 transmission dynamics in a university setting', 'authors': \"[{'name': 'Emily Nixon'}, {'name': 'Amy Thomas'}, {'name': 'Daniel Stocks'}, {'name': 'Antoine M.G. Barreaux'}, {'name': 'Gibran Hemani'}, {'name': 'Adam Trickey'}, {'name': 'Rachel Kwiatkowska'}, {'name': 'Josephine Walker'}, {'name': 'David Ellis'}, {'name': 'Leon Danon'}, {'name': 'Caroline Relton'}, {'name': 'Hannah Christensen'}, {'name': 'Ellen Brooks-Pollock'}]\", 'abstract': '<p>We investigate the impact of vaccination and asymptomatic testing uptake on SARS-CoV-2 transmission in a university student population using a stochastic compartmental model. We find that the magnitude and timing of outbreaks is highly variable under different vaccine uptake levels. With low level interventions (no asymptomatic testing, 30% vaccinated), 53-71% of students become infected during the first term; with high interventions (90% using asymptomatic testing, 90% vaccinated) cumulative incidence is 7-9%, with around 80% of these cases estimated to be asymptomatic. Asymptomatic testing is most useful when vaccine uptake is low: when 30% of students are vaccinated, 90% uptake of asymptomatic testing leads to almost half the case numbers. Under high levels of vaccine uptake (70-90%), case numbers in the student population are largely driven by community importation. Our findings suggest that vaccination is critical for controlling SARS-CoV-2 transmission in university settings with asymptomatic testing being a useful supporting measure.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.25.470006', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Cryo-EM structure of the Smc5/6 holo-complex', 'authors': \"[{'name': 'Stephen T. Hallett'}, {'name': 'Isabella Campbell Harry'}, {'name': 'Pascale Schellenberger'}, {'name': 'Lihong Zhou'}, {'name': 'Nora B. Cronin'}, {'name': 'Jonathan Baxter'}, {'name': 'Thomas J. Etheridge'}, {'name': 'Johanne M. Murray'}, {'name': 'Antony W. Oliver'}]\", 'abstract': '<p>The Smc5/6 complex plays an essential role in the resolution of recombination intermediates formed during mitosis or meiosis, or as a result of the cellular response to replication stress. It also functions as a restriction factor preventing viral integration. Here, we report the cryo-EM structure of the six-subunit budding yeast Smc5/6 holo-complex, reconstituted from recombinant proteins expressed in insect cells  providing a full overview of the complex in its apo / non-liganded form, and revealing how the Nse1/3/4 subcomplex binds to the hetero-dimeric SMC protein core. In addition, we demonstrate that a region within the head domain of Smc5, equivalent to the W-loop of Smc4 or F-loop of Smc1, mediates an essential interaction with Nse1. Taken together, these data confirm a degree of functional equivalence between the structurally unrelated KITE and HAWK accessory subunits associated with SMC complexes.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.22.469588', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Single organoid RNA-sequencing reveals high organoid-to-organoid variability', 'authors': \"[{'name': 'Kristin Gehling'}, {'name': 'Swati Parekh'}, {'name': 'Farina Schneider'}, {'name': 'Marcel Kirchner'}, {'name': 'Vangelis Kondylis'}, {'name': 'Chrysa Nikopoulou'}, {'name': 'Peter Tessarz'}]\", 'abstract': '<p>Over the last decades, organoids have been established from the majority of tissue resident stem and iPS cells. They hold great promise for our understanding of mammalian organ development, but also for the study of disease or even personalized medicine. In recent years, several reports hinted at intraculture organoid variability, but a systematic analysis of such a heterogeneity has not been performed before. Here, we used RNA-seq of individual organoids to address this question. Importantly, we find that batch-to-batch variation is very low, even when prepared by different researchers. On the other hand, there is organoid-to-organoid variability within a culture. Using differential gene expression, we did not identify specific pathways that drive this variability, pointing towards possible effects of the microenvironment within the culture condition. Taken together, our study provides a framework for organoid researchers to properly consider experimental design.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469546', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Long distance migration is a major factor driving local adaptation at continental scale in Coho Salmon', 'authors': \"[{'name': 'Quentin Rougemont'}, {'name': 'Amanda Xuereb'}, {'name': 'Xavier Dallaire'}, {'name': 'Jean-Sebastien Moore'}, {'name': 'Eric Normandeau'}, {'name': 'Eric B Rondeau'}, {'name': 'Ruth Withler'}, {'name': 'Donald Van Doornik'}, {'name': 'Penelope Crane'}, {'name': 'Kerry Naish'}, {'name': 'John-Carlos Garza'}, {'name': 'Terry Beacham'}, {'name': 'Ben F. Koop'}, {'name': 'Louis Bernatchez'}]\", 'abstract': '<p>Inferring the genomic basis of local adaptation is a long-standing goal of evolutionary biology. Beyond its fundamental evolutionary implications, such knowledge can guide conservation decisions for populations of conservation and management concern. Here, we investigated the genomic basis of local adaptation in the Coho salmon (Oncorhynchus kisutch) across its entire North American range. We hypothesized that extensive spatial variation in environmental conditions and the species homing behavior may promote the establishment of local adaptation. We genotyped 7,829 individuals representing 217 sampling locations at more than 100,000 high-quality RADseq loci to investigate how recombination might affect the detection of loci putatively under selection and took advantage of the precise description of the demographic history of the species from our previous work to draw accurate population genomic inferences about local adaptation. Results indicated that genetic differentiation scans and genetic-environment association analyses were both significantly affected by variation in recombination rate as low recombination regions displayed an increased number of outliers. By taking these confounding factors into consideration, we revealed that migration distance was the primary selective factor driving local adaptation and partial parallel divergence among distant populations. Moreover, we identified several candidates SNP associated with long distance migration and altitude including a gene known to be involved in adaptation to altitude in other species. The evolutionary implications of our findings are discussed along with conservation applications.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469675', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Improved peptide backbone fragmentation is the primary advantage of MS-cleavable crosslinkers', 'authors': \"[{'name': 'Lars Kolbowski'}, {'name': 'Swantje Lenz'}, {'name': 'Lutz Fischer'}, {'name': 'Ludwig R Sinn'}, {'name': 'Francis J OReilly'}, {'name': 'Juri Rappsilber'}]\", 'abstract': '<p>Proteome-wide crosslinking mass spectrometry studies have coincided with the advent of MS-cleavable crosslinkers that can reveal the individual masses of the two crosslinked peptides. However, recently such studies have also been published with non-cleavable crosslinkers suggesting that MS-cleavability is not essential. We therefore examined in detail the advantages and disadvantages of using the most popular MS-cleavable crosslinker, DSSO. Indeed, DSSO gave rise to signature peptide fragments with a distinct mass difference (doublet) for nearly all identified crosslinked peptides. Surprisingly, we could show that it was not these peptide masses that proved the main advantage of MS-cleavability of the crosslinker, but improved peptide backbone fragmentation that allowed for more confident peptide identification. We also show that the more intricate MS3-based data acquisition approaches lack sensitivity and specificity, causing them to be outperformed by the simpler and faster stepped HCD method. This understanding will guide future developments and applications of proteome-wide crosslinking mass spectrometry.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.26.470139', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Recurrent evolution of cross-resistance in response to selection for improved post-infection survival in Drosophila melanogaster', 'authors': \"[{'name': 'Aparajita Singh'}, {'name': 'Aabeer Basu'}, {'name': 'Biswajit Shit'}, {'name': 'Tejashwini Hegde'}, {'name': 'Nitin Bansal'}, {'name': 'Nagaraj Guru Prasad'}]\", 'abstract': '<p>The host susceptibility to one pathogen can decrease, increase, or remain unaffected by virtue of the host evolving resistance towards a second pathogen. Negative correlations between a host susceptibility to different pathogens is an often-cited explanation for maintenance of genetic variation in immune function determining traits in a host population. In this study, we investigated the change in susceptibility of Drosophila melanogaster flies to various novel bacterial pathogens after being experimentally selected for increased resistance to one particular bacterial pathogen. We independently selected flies to become more resistant towards Enterococcus faecalis and Pseudomonas entomophila, and baring a few exceptions the evolved populations exhibited cross-resistance against the range of pathogens tested in the study. Neither the identity of the native pathogen nor the host sex was major determining factors in predicting the pattern of cross-resistance exhibited by the selected populations. We therefore report that a generalized cross-resistance to novel pathogens can repeatedly evolve in response to selection for resistance against a single pathogen.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.24.469969', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Transcriptional profiles of a foliar fungal endophyte (Pestalotiopsis, Ascomycota) and its endohyphal bacterium (Luteibacter, Gammaproteobacteria) in co-culture support sulfur exchange and growth regulation', 'authors': \"[{'name': 'Justin P. Shaffer'}, {'name': 'Morgan E. Carter'}, {'name': 'Joseph E. Spraker'}, {'name': 'Meara Clark'}, {'name': 'Brian A. Smith'}, {'name': 'Kevin L. Hockett'}, {'name': 'David A. Baltrus'}, {'name': 'A. Elizabeth Arnold'}]\", 'abstract': '<p>Symbiosis with bacteria is widespread among eukaryotes, including fungi. Bacteria that live within fungal mycelia (endohyphal bacteria) occur in many plant-associated fungi, including diverse Mucoromycota and Dikarya. <italic>Pestalotiopsis</italic> sp. 9143 is a filamentous ascomycete isolated originally as a foliar endophyte of <italic>Platycladus orientalis</italic> (Cupressaceae). It is infected naturally with the endohyphal bacterium <italic>Luteibacter</italic> sp. 9143, which influences auxin and enzyme production by its fungal host. Previous studies have used transcriptomics to examine similar symbioses between endohyphal bacteria and root-associated fungi such as arbuscular mycorrhizal fungi and plant pathogens. However, currently there are no gene expression studies of endohyphal bacteria of Ascomycota, the most species-rich fungal phylum. We developed methods for assessing gene expression by <italic>Pestalotiopsis</italic> sp. and <italic>Luteibacter</italic> sp. when grown in co-culture and when each was grown axenically. Our assays showed that the density of <italic>Luteibacter</italic> sp. in co-culture was greater than in axenic culture, but the opposite was true for the <italic>Pestalotiopsis</italic> sp. Dual RNA-seq data demonstrate that growing in co-culture modulates developmental and metabolic processes in both the fungus and bacterium, potentially through changes in the balance of organic sulfur via methionine acquisition. Our analyses also suggest an unexpected, potential role of the bacterial type VI secretion system in symbiosis establishment, expanding current understanding of the scope and dynamics of fungal-bacterial symbioses.</p><sec><title>TWEET</title><p>When in co-culture, <italic>Luteibacter</italic> downregulates motility and upregulates a T6SS. Gene expression changes in its host, <italic>Pestalotiopsis</italic>, suggest the bacterium impacts fungal cell structure and methionine availability.</p></sec><sec><title>IMPORTANCE</title><p>Interactions between microbes and their hosts have important outcomes for host- and environmental health. Foliar fungal endophytes that infect healthy plants can harbor facultative endosymbionts called endohyphal bacteria, which can influence the outcome of plant-fungus interactions. These bacterial-fungal interactions can be influential but are poorly understood, particularly from a transcriptome perspective. Here, we report on a comparative, dual RNA-seq study examining the gene expression patterns of a foliar fungal endophyte and a facultative endohyphal bacterium when cultured together vs. separately. Our findings support a role for the fungus in providing organic sulfur to the bacterium, potentially through methionine acquisition, and potential involvement of a bacterial type VI secretion system in symbiosis establishment.</p><p>This work adds to the growing body of literature characterizing endohyphal bacterial-fungal interactions, with a focus on a model facultative bacterial-fungal symbiosis in two species-rich lineages, the Ascomycota and Proteobacteria.</p></sec>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.24.21266807', 'tweets': 26, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'A population framework for predicting the proportion of people infected by the far-field airborne transmission of SARS-CoV-2 indoors', 'authors': \"[{'name': 'Christopher Iddon'}, {'name': 'Benjamin Jones'}, {'name': 'Patrick Sharpe'}, {'name': 'Muge Cevik'}, {'name': 'Shaun Fitzgerald'}]\", 'abstract': '<p>The number of occupants in a space influences the risk of far-field airborne transmission of the SARS-CoV-2 virus because the likelihood of having infectious and susceptible people both scale with the number of occupants. Mass-balance and dose-response models determine far-field transmission risks for an individual person and a population of people after sub-dividing a large <italic>reference</italic> space into 10 identical <italic>comparator</italic> spaces.</p><p>For a single infected person when the <italic>per capita</italic> ventilation rate is preserved, the dose received by an individual person in the <italic>comparator</italic> space is 10-times higher because the equivalent ventilation rate per infected person is lower.</p><p>However, accounting for population dispersion, such as the community infection rate, the probability of an infected person being present and uncertainty in their viral load, shows the probability of transmission increases with occupancy. Also, far-field transmission is likely to be a rare event that requires a set of Goldilocks conditions that are <italic>just right</italic>, when mitigation measures have limited effect.</p><p>Therefore, resilient buildings should deliver the equivalent ventilation rate required by standards and increase the space volume per person, but also require reductions in the viral loads and the infection rate of the wider population.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.466958', 'tweets': 25, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'amica: an interactive and user-friendly web-platform for the analysis of proteomics data', 'authors': \"[{'name': 'Sebastian Didusch'}, {'name': 'Moritz Madern'}, {'name': 'Markus Hartl'}, {'name': 'Manuela Baccarini'}]\", 'abstract': '<sec><title>Summary</title><p>Quantitative proteomics has become an increasingly prominent tool in the study of life sciences. A substantial hurdle for many biologists are, however, the intricacies involved in the associated high troughput data analysis. In order to facilitate this task for users with little background knowledge in proteomics, we have developed amica, a freely available open-source web-based software that accepts proteomic input files from different sources and provides quality control, differential expression, biological network and over-representation analysis on the basis of minimal user input. Scientists can use amica interactively to compare proteins across multiple groups, create customized output graphics, and ultimately export the results in a tab-separated format that can be shared with collaborators.</p></sec><sec><title>Availability and Implementation</title><p>The code for the application, input data and documentation can be accessed online at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/tbaccata/amica\">https://github.com/tbaccata/amica</ext-link> and is also incorporated in the web application. A freely available version of amica is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://bioapps.maxperutzlabs.ac.at/app/amica\">https://bioapps.maxperutzlabs.ac.at/app/amica</ext-link>.</p></sec><sec><title>Contact</title><p><email>sebastian.didusch@univie.ac.at</email></p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.19.469335', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Pathology', 'covid': True, 'title': 'Acute SARS-CoV-2 infection in pregnancy is associated with placental ACE-2 shedding', 'authors': \"[{'name': 'Elizabeth S. Taglauer'}, {'name': 'Elisha M. Wachman'}, {'name': 'Lillian Juttukonda'}, {'name': 'Timothy Klouda'}, {'name': 'Jiwon Kim'}, {'name': 'Qiong Wang'}, {'name': 'Asuka Ishiyama'}, {'name': 'David J. Hackam'}, {'name': 'Ke Yuan'}, {'name': 'Hongpeng Jia'}]\", 'abstract': '<p>Human placental tissues have variable rates of SARS-CoV-2 invasion resulting in consistently low rates of fetal transmission suggesting a unique physiologic blockade against SARS-CoV-2. Angiotensin-converting enzyme (ACE)-2, the main receptor for SARS-CoV-2, is expressed as cell surface and soluble forms regulated by a metalloprotease cleavage enzyme, ADAM17. ACE-2 is expressed in the human placenta, but the regulation of placental ACE-2 expression in relation to timing of maternal SARS-CoV-2 infection in pregnancy is not well understood. In this study, we evaluated ACE-2 expression, ADAM17 activity and serum ACE-2 abundance in a cohort of matched villous placental and maternal serum samples from Control pregnancies (SARS-CoV-2 negative, n=8) and pregnancies affected by symptomatic maternal SARS-CoV-2 infections in the 2<sup>nd</sup> trimester (2<sup>nd</sup>Tri COVID, n=8) and 3rd trimester (3<sup>rd</sup>Tri COVID, n=8). In 3<sup>rd</sup>Tri COVID as compared to control and 2<sup>nd</sup>Tri-COVID villous placental tissues ACE-2 mRNA expression was remarkably elevated, however, ACE-2 protein expression was significantly decreased with a parallel increase in ADAM17 activity. Soluble ACE-2 was also significantly increased in the maternal serum from 3<sup>rd</sup>Tri COVID infections as compared to control and 2<sup>nd</sup>Tri-COVID pregnancies. These data suggest that in acute maternal SARS-CoV-2 infections, decreased placental ACE-2 protein may be the result of ACE-2 shedding. Overall, this work highlights the importance of ACE-2 for ongoing studies on SARS-CoV-2 responses at the maternal-fetal interface.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.24.21266741', 'tweets': 24, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': True, 'title': 'A genome-wide association study of COVID-19 related hospitalization in Spain reveals genetic disparities among sexes', 'authors': \"[{'name': 'ngel Carracedo'}, {'name': ' '}]\", 'abstract': '<p>We describe the results of the Spanish Coalition to Unlock Research on Host Genetics on COVID-19 (SCOURGE). In sex-disaggregated genome-wide studies of COVID-19 hospitalization, we found two known loci associated among males (<italic>SLC6A20</italic>-<italic>LZTFL1</italic> and <italic>IFNAR2</italic>), and a novel one among females (<italic>TLE1</italic>). Meta-analyses with independent studies revealed two novel associations (<italic>AQP3</italic> and <italic>ARHGAP33</italic>) and replicated <italic>ELF5</italic>. A genetic risk score predicted COVID-19 severity, especially among younger males. We found less SNP-heritability and larger heritability differences by age (&lt;60/60 years) among males than females. Inbreeding depression was associated with COVID-19 hospitalization and severity, and the effect was stronger among older males.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469860', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': True, 'title': \"Nanopore ReCappable Sequencing maps SARS-CoV-2 5' capping sites and provides new insights into the structure of sgRNAs\", 'authors': \"[{'name': 'Camilla Ugolini'}, {'name': 'Logan Mulroney'}, {'name': 'Adrien Leger'}, {'name': 'Matteo Castelli'}, {'name': 'Elena Criscuolo'}, {'name': 'Maia Kavanagh Williamson'}, {'name': 'Andrew D Davidson'}, {'name': 'Abdulaziz Almuqrin'}, {'name': 'Roberto Giambruno'}, {'name': 'Miten Jain'}, {'name': 'Gianmaria Frig'}, {'name': 'Hugh Olsen'}, {'name': 'George Tzertzinis'}, {'name': 'Ira Schildkraut'}, {'name': 'Madalee F Wulf'}, {'name': 'Ivan R. Corra'}, {'name': 'Laurence Ettwiller'}, {'name': 'Nicola Clementi'}, {'name': 'Massimo Clementi'}, {'name': 'Nicasio Mancini'}, {'name': 'Ewan Birney'}, {'name': 'Mark Akeson'}, {'name': 'Francesco Nicassio'}, {'name': 'David A Matthews'}, {'name': 'Tommaso Leonardi'}]\", 'abstract': '<p>The SARS-CoV-2 virus has a complex transcriptome characterised by multiple, nested sub genomic RNAs used to express structural and accessory proteins. Long-read sequencing technologies such as nanopore direct RNA sequencing can recover full-length transcripts, greatly simplifying the assembly of structurally complex RNAs. However, these techniques do not detect the 5 cap, thus preventing reliable identification and quantification of full-length, coding transcript models. Here we used Nanopore ReCappable Sequencing (NRCeq), a new technique that can identify capped full-length RNAs, to assemble a complete annotation of SARS-CoV-2 sgRNAs and annotate the location of capping sites across the viral genome. We obtained robust estimates of sgRNA expression across cell lines and viral isolates and identified novel canonical and non-canonical sgRNAs, including one that uses a previously un-annotated leader-to-body junction site. The data generated in this work constitute a useful resource for the scientific community and provide important insights into the mechanisms that regulate the transcription of SARS-CoV-2 sgRNAs.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.22.21266651', 'tweets': 24, 'archive': 'medRxiv', 'subject-area': 'Cardiovascular Medicine', 'covid': False, 'title': 'The Frequency, Penetrance and Variable Expressivity of Dilated Cardiomyopathy-Associated Putative Pathogenic Gene Variants in UK Biobank Participants', 'authors': \"[{'name': 'Ravi Shah'}, {'name': 'Babken Asatryan'}, {'name': 'Ghaith Sharaf Dabbagh'}, {'name': 'Nay Aung'}, {'name': 'Mohammed Y Khanji'}, {'name': 'Luis R. Lopes'}, {'name': 'Stefan van Duijvenboden'}, {'name': 'Anthony Holmes'}, {'name': 'Daniele Muser'}, {'name': 'Andrew P. Landstrom'}, {'name': 'Aaron Mark Lee'}, {'name': 'Pankaj Arora'}, {'name': 'Christopher Semsarian'}, {'name': 'Virend K. Somers'}, {'name': 'Anjali Owens'}, {'name': 'Patricia B Munroe'}, {'name': 'Steffen E. Petersen'}, {'name': 'C. Anwar A. Chahal'}]\", 'abstract': '<sec><title>Background</title><p>There is a paucity of data regarding the phenotype of dilated cardiomyopathy (DCM) gene variants in the general population. We aimed to determine the frequency and penetrance of DCM-associated putative pathogenic gene variants in a general, adult population, with a focus on the expression of clinical and subclinical phenotype, including structural, functional and arrhythmic disease features.</p></sec><sec><title>Methods</title><p>UK Biobank participants who had undergone whole exome sequencing (WES), ECG and cardiovascular magnetic resonance (CMR) imaging were selected for study. Three different variant calling strategies (one primary and two secondary) were used to identify subjects with putative pathogenic variants in 44 DCM genes. The observed phenotype was graded to either 1) DCM (clinical or CMR diagnosis); 2) early DCM features, including arrhythmia and/or conduction disease, isolated ventricular dilation, and hypokinetic non-dilated cardiomyopathy; or 3) phenotype-negative.</p></sec><sec><title>Results</title><p>Among 18,665 individuals included in the study, 1,463 (7.8%) subjects possessed 1 putative pathogenic variant in 44 DCM genes by the main variant calling strategy. A clinical diagnosis of DCM was present in 0.34% and early DCM features in 5.7% of individuals with putative pathogenic variants. ECG and CMR analysis revealed evidence of subclinical DCM in an additional 1.6% and early DCM features in 15.9% of individuals with putative pathogenic variants. Arrhythmias and/or conduction disease (15.2%) were the most common early DCM features, followed by hypokinetic non-dilated cardiomyopathy (4%). The combined clinical/subclinical penetrance was 30% with all three variant filtering strategies. Clinical DCM was slightly more prevalent among participants with putative pathogenic variants in definitive/strong evidence genes, as compared to those with variants in moderate/limited evidence genes.</p></sec><sec><title>Conclusions</title><p>In the UK Biobank, approximately 1/6 of adults with putative pathogenic variants in DCM genes exhibited a subclinical phenotype based on ECG and/or CMR, most commonly manifesting with arrhythmias in the absence of substantial ventricular dilation/dysfunction.</p></sec><sec><title>Clinical Perspective</title><sec><title>What is new?</title><list list-type=\"bullet\"><list-item><p>Among individuals with putative pathogenic DCM gene variants, subclinical DCM and early DCM features, detected by ECG and/or CMR, were nearly four times more common than clinically manifest DCM or early features (23.7% vs. 6.1%).</p></list-item><list-item><p>Over 90% of subjects with a putative pathogenic variant in DCM-associated genes did not have a prior history of DCM.</p></list-item><list-item><p>Clinical DCM was slightly more prevalent among participants with putative pathogenic variants in definitive/strong evidence genes (13.9% for clinical and subclinical), as compared to those with variants in moderate/limited evidence genes, but there was no significant difference in combined clinical/subclinical phenotype by cluster.</p></list-item><list-item><p>The overall clinical/subclinical penetrance of DCM-associated single putative pathogenic variants was highly variable between genes, ranging from 0 to 66.7%.</p></list-item></list></sec><sec><title>What are the clinical implications?</title><list list-type=\"bullet\"><list-item><p>Arrhythmias and cardiac conduction disease are the most common early manifestation of putative pathogenic variants implicated in DCM, mostly occurring prior to the development of structural/functional abnormalities.</p></list-item><list-item><p>A genotype-first screening approach for DCM using a large genetic panel is currently not suitable in the general population due to incomplete understanding of DCM genetic architecture and reduced penetrance of DCM-associated putative pathogenic variants.</p></list-item></list></sec></sec><sec><title>Journal Subject Terms</title><p>Cardiomyopathy; Genetics; Sudden Cardiac Death</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469913', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Synchrony and idiosyncrasy in the gut microbiome of wild primates', 'authors': \"[{'name': 'Johannes R. Bjrk'}, {'name': 'Mauna R. Dasari'}, {'name': 'Kim Roche'}, {'name': 'Laura Grieneisen'}, {'name': 'Trevor J. Gould'}, {'name': 'Jean-Christophe Grenier'}, {'name': 'Vania Yotova'}, {'name': 'Neil Gottel'}, {'name': 'David Jansen'}, {'name': 'Laurence R. Gesquiere'}, {'name': 'Jacob B. Gordon'}, {'name': 'Niki H. Learn'}, {'name': 'Tim L. Wango'}, {'name': 'Raphael S. Mututua'}, {'name': 'J. Kinyua Warutere'}, {'name': 'Longida Siodi'}, {'name': 'Sayan Mukherjee'}, {'name': 'Luis B. Barreiro'}, {'name': 'Susan C. Alberts'}, {'name': 'Jack A. Gilbert'}, {'name': 'Jenny Tung'}, {'name': 'Ran Blekhman'}, {'name': 'Elizabeth A. Archie'}]\", 'abstract': '<p>Human gut microbial dynamics are highly individualized, making it challenging to link microbiota to health and to design universal microbiome therapies. This individuality is typically attributed to variation in diets, environments, and medications, but it could also emerge from fundamental ecological forces that shape primate microbiota more generally. Here we leverage extensive gut microbiome time series from wild baboonshosts who experience little interindividual dietary and environmental heterogeneityto test whether gut microbial dynamics are synchronized across hosts or largely idiosyncratic. Despite their shared lifestyles, we find strong evidence for idiosyncrasy. Over time, samples from the same baboon were much more similar than samples from different baboons, and host-specific factors collectively explained 30% of the deviance in microbiome dynamics, compared to just 3% for factors shared across hosts. Hence, individualization may be common to mammalian gut microbiota, and designing universal microbiome interventions may face challenges beyond heterogeneity in human lifestyles.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469613', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Chronic Physical and Vicarious Psychosocial Stress Alter Fentanyl Consumption and Nucleus Accumbens Rho GTPases in Male and Female C57BL/6 Mice', 'authors': \"[{'name': 'Daniela Franco'}, {'name': 'Andreas B Wulff'}, {'name': 'Mary Kay Lobo'}, {'name': 'Megan E Fox'}]\", 'abstract': '<p>Chronic stress can increase the risk of developing a substance use disorder in vulnerable individuals. Numerous models have been developed to probe the underlying neurobiological mechanisms, however most prior work has been restricted to male rodents, conducted only in rats, or introduces physical injury that can complicate opioid studies. Here we sought to establish how chronic psychosocial stress influences fentanyl consumption in male and female C57BL/6 mice. We used chronic social defeat stress (CSDS), or the modified vicarious chronic witness defeat stress (CWDS), and used social interaction to stratify mice as stress-susceptible or resilient. We then subjected mice to a 15 day fentanyl drinking paradigm in the home cage that consisted of alternating forced and choice periods with increasing fentanyl concentrations. Male mice susceptible to either CWDS or CSDS consumed more fentanyl relative to unstressed mice, and exhibited increased preference for fentanyl. CWDS-susceptible female mice did not differ from unstressed mice during the forced periods, but showed increased preference for fentanyl. We also found decreased expression of nucleus accumbens Rho GTPases in male, but not female mice following stress and fentanyl drinking. We also compare fentanyl drinking behavior in mice that had free access to plain water throughout. Our results indicate that stress-sensitized fentanyl consumption is dependent on both sex and behavioral outcomes to stress.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469797', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Using AnnoTree to get more assignments, faster, in DIAMOND+MEGAN microbiome analysis', 'authors': \"[{'name': 'Anupam Gautam'}, {'name': 'Hendrik Felderhoff'}, {'name': 'Caner Baci'}, {'name': 'Daniel H. Huson'}]\", 'abstract': '<p>In microbiome analysis, one main approach is to align metagenomic sequencing reads against a protein-reference database such as NCBI-nr, and then to perform taxonomic and functional binning based on the alignments. This approach is embodied, for example, in the standard DIAMOND+MEGAN analysis pipeline, which first aligns reads against NCBI-nr using DIAMOND and then performs taxonomic and functional binning using MEGAN. Here we propose the use of the AnnoTree protein database, rather than NCBI-nr, in such alignment-based analyses to determine the prokaryotic content of metagenomic samples. We demonstrate a 2-fold speedup over the usage of the prokaryotic part of NCBI-nr, and increased assignment rates, in particular, assigning twice as many reads to KEGG. In addition to binning to the NCBI taxonomy, MEGAN now also bins to the GTDB taxonomy.</p><sec><title>IMPORTANCE</title><p>The NCBI-nr database is not explicitly designed for the purpose of microbiome analysis and its increasing size makes its unwieldy and computationally expensive for this purpose. The AnnoTree protein database is only one quarter the size of the full NCBI-nr database and is explicitly designed for metagenomic analysis, and so should be supported by alignment-based pipelines.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469587', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The energy requirements of ion homeostasis determine the lifespan of starving bacteria', 'authors': \"[{'name': 'Severin Schink'}, {'name': 'Mark Polk'}, {'name': 'Edward Athaide'}, {'name': 'Avik Mukherjee'}, {'name': 'Constantin Ammar'}, {'name': 'Xili Liu'}, {'name': 'Seungeun Oh'}, {'name': 'Yu-Fang Chang'}, {'name': 'Markus Basan'}]\", 'abstract': '<p>The majority of microbes on earth, whether they live in the ocean, the soil or in animals, are not growing, but instead struggling to survive starvation<sup>16</sup>. Some genes and environmental conditions affecting starvation survival have been identified<sup>713</sup>, but despite almost a century of study<sup>1416</sup>, we do not know which processes lead to irreversible loss of viability, which maintenance processes counteract them and how lifespan is determined from the balance of these opposing processes. Here, we used time-lapse microscopy to capture and characterize the cell death process of <italic>E. coli</italic> during carbon starvation for the first time. We found that a lack of nutrients results in the collapse of ion homeostasis, triggering a positive-feedback cascade of osmotic swelling and membrane permeabilization that ultimately results in lysis. Based on these findings, we hypothesized that ion transport is the major energetic requirement for starving cells and the primary determinant of the timing of lysis. We therefore developed a mathematical model that integrates ion homeostasis and cannibalistic nutrient recycling from perished cells<sup>16,17</sup> to predict lifespan changes under diverse conditions, such as changes of cell size, medium composition, and prior growth conditions. Guided by model predictions, we found that cell death during starvation could be dramatically slowed by replacing inorganic ions from the medium with a non-permeating osmoprotectant, removing the cost of ion homeostasis and preventing lysis. Our quantitative and predictive model explains how survival kinetics are determined in starvation and elucidates the mechanistic underpinnings of starvation survival.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.25.469928', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Systemic LSD1 inhibition prevents aberrant remodeling of metabolism in obesity', 'authors': \"[{'name': 'Bastian Ramms'}, {'name': 'Dennis P. Pollow'}, {'name': 'Han Zhu'}, {'name': 'Chelsea Nora'}, {'name': 'Austin R. Harrington'}, {'name': 'Ibrahim Omar'}, {'name': 'Philip L.S.M. Gordts'}, {'name': 'Matthew Wortham'}, {'name': 'Maike Sander'}]\", 'abstract': '<p>The transition from lean to obese states involves systemic metabolic remodeling that impacts insulin sensitivity, lipid partitioning, inflammation, and glycemic control. Here, we have taken a pharmacological approach to test the role of a nutrient-regulated chromatin modifier, lysine-specific demethylase (LSD1), in obesity-associated metabolic reprogramming. We show that systemic administration of an LSD1 inhibitor (GSK-LSD1) reduces food intake and body weight, ameliorates non-alcoholic fatty liver disease (NAFLD), and improves insulin sensitivity and glycemic control in mouse models of obesity. GSK-LSD1 has little effect on systemic metabolism of lean mice, suggesting LSD1 has a context-dependent role in promoting maladaptive changes in obesity. Analysis of insulin target tissues identified white adipose tissue as the major site of insulin sensitization by GSK-LSD1, where it reduces adipocyte inflammation and lipolysis. We demonstrate that GSK-LSD1 reverses NAFLD in a non-hepatocyte-autonomous manner, suggesting an indirect mechanism via inhibition of adipocyte lipolysis and subsequent effects on lipid partitioning. Pair-feeding experiments further revealed that effects of GSK-LSD1 on hyperglycemia and NAFLD are not a consequence of reduced food intake and weight loss. These findings suggest that targeting LSD1 could be a strategy for treatment of obesity and its associated complications including type 2 diabetes and NAFLD.\\n<fig id=\"ufig1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"469928v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.23.469703', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PerSVade: Personalized Structural Variation detection in your species of interest', 'authors': \"[{'name': 'Miquel ngel Schikora-Tamarit'}, {'name': 'Toni Gabaldn'}]\", 'abstract': '<p>Structural variants (SVs) like translocations, deletions, and other rearrangements underlie genetic and phenotypic variation. SVs are often overlooked due to difficult detection from short-read sequencing. Most algorithms yield low recall on humans, but the performance in other organisms is unclear. Similarly, despite remarkable differences across species genomes, most approaches use parameters optimized for humans. To overcome this and enable species-tailored approaches, we developed perSVade (personalized Structural Variation Detection), a pipeline that identifies SVs in a way that is optimized for any input sample. Starting from short reads, perSVade uses simulations on the reference genome to choose the best SV calling parameters. The output includes the optimally-called SVs and the accuracy, useful to assess the confidence in the results. In addition, perSVade can call small variants and copy-number variations. In summary, perSVade automatically identifies several types of genomic variation from short reads using sample-optimized parameters. We validated that perSVade increases the SV calling accuracy on simulated variants for six diverse eukaryotes, and on datasets of validated human variants. Importantly, we found no universal set of optimal parameters, which underscores the need for species-specific parameter optimization. PerSVade will improve our understanding about the role of SVs in non-human organisms.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469693', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A target expression threshold dictates invader defense and autoimmunity by CRISPR-Cas13', 'authors': \"[{'name': 'Elena Vialetto'}, {'name': 'Yanying Yu'}, {'name': 'Scott P. Collins'}, {'name': 'Katharina G. Wandera'}, {'name': 'Lars Barquist'}, {'name': 'Chase L. Beisel'}]\", 'abstract': '<p>Immune systems must recognize and clear foreign invaders without eliciting autoimmunity. CRISPR-Cas immune systems in prokaryotes manage this task by following two criteria: extensive guide:target complementarity and a defined target-flanking motif. Here we report an additional requirement for RNA-targeting CRISPR-Cas13 systems: expression of the target transcript exceeding a threshold. This finding is based on targeting endogenous non-essential transcripts, which rarely elicited dormancy through collateral RNA degradation. Instead, eliciting dormancy required over-expressing targeted transcripts above a threshold. A genome-wide screen confirmed target expression levels as the principal determinant of cytotoxic autoimmunity and revealed that the threshold shifts with the guide:target pair. This expression threshold ensured defense against a lytic bacteriophage yet allowed tolerance of a targeted beneficial gene expressed from an invading plasmid. These findings establish target expression levels as a third criterion for immune activation by RNA-targeting CRISPR-Cas systems, buffering against autoimmunity and distinguishing pathogenic and benign invaders.</p><sec id=\"s1\"><title>HIGHLIGHTS</title><list list-type=\"bullet\"><list-item><p>Cas13-induced dormancy requires RNA target levels to exceed an expression threshold</p></list-item><list-item><p>The expression threshold can prevent cytotoxic self-targeting for endogenous transcripts</p></list-item><list-item><p>The threshold shifts depending on the CRISPR RNA guide:target pair</p></list-item><list-item><p>The threshold allows cells to distinguish pathogenic and benign infections</p></list-item></list></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469601', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genome context influences evolutionary flexibility of nearly identical type III effectors in two phytopathogenic Pseudomonads', 'authors': \"[{'name': 'David A. Baltrus'}, {'name': 'Qian Feng'}, {'name': 'Brian H Kvitko'}]\", 'abstract': '<p>Integrative Conjugative Elements (ICEs) are replicons that can insert and excise from chromosomal locations in a site specific manner, can conjugate across strains, and which often carry a variety of genes useful for bacterial growth and survival under specific conditions. Although ICEs have been identified and vetted within certain clades of the agricultural pathogen <italic>Pseudomonas syringae</italic>, the impact of ICE carriage and transfer across the entire <italic>P. syringae</italic> species complex remains underexplored. Here we identify and vet an ICE (PmaICE-DQ) from <italic>P. syringae</italic> pv. maculicola ES4326, a strain commonly used for laboratory virulence experiments, demonstrate that this element can excise and conjugate across strains, and contains loci encoding multiple type III effector proteins. Moreover, genome context suggests that another ICE (PmaICE-AOAB) is highly similar in comparison with and found immediately adjacent to PmaICE-DQ within the chromosome of strain ES4326, and also contains multiple type III effectors. Lastly, we present passage data from <italic>in planta</italic> experiments that suggests that genomic plasticity associated with ICEs may enable strains to more rapidly lose type III effectors that trigger R-gene mediated resistance in comparison to strains where nearly isogenic effectors are not present in ICEs. Taken together, our study sheds light on a set of ICE elements from <italic>P. syringae</italic> pv. maculicola ES4326 and highlights how genomic context may lead to different evolutionary dynamics for shared virulence genes between strains.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.21266709', 'tweets': 19, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'COVID-19 vaccination and menstrual cycle changes: A United Kingdom (UK) retrospective case-control study', 'authors': \"[{'name': 'Alexandra Alvergne'}, {'name': 'Gabriella Kountourides'}, {'name': 'M. Austin Argentieri'}, {'name': 'Lisa Agyen'}, {'name': 'Natalie Rogers'}, {'name': 'Dawn Knight'}, {'name': 'Gemma C Sharp'}, {'name': 'Jacqueline A Maybin'}, {'name': 'Zuzanna Olszewska'}]\", 'abstract': '<sec><title>Background</title><p>There has been increasing public concern that COVID-19 vaccines cause menstrual cycle disturbances, yet there is currently limited data to evaluate the impact of vaccination on menstrual health. Our objectives were (1) to evaluate the prevalence of menstrual changes following vaccination against COVID-19, (2) to test potential risk factors for any such changes, and (3) to identify patterns of symptoms in participants written accounts.</p></sec><sec><title>Methods</title><p>We performed a secondary analysis of a retrospective online survey titled The Covid-19 Pandemic and Womens Reproductive Health, conducted in March 2021 in the UK before widespread media attention regarding potential impacts of SARS-CoV-2 vaccination on menstruation. Participants were recruited via a Facebook ad campaign in the UK and eligibility criteria for survey completion were age greater than 18 years, having ever menstruated and currently living in the UK. In total, 26,710 people gave consent and completed the survey. For this analysis we selected 4,989 participants who were pre-menopausal and vaccinated. These participants were aged 28 to 43, predominantly from England (81%), of white background (95%) and not using hormonal contraception (58%).</p></sec><sec><title>Findings</title><p>Among pre-menopausal vaccinated individuals (n=4,989), 80% did not report any menstrual cycle changes up to 4 months after their first COVID-19 vaccine injection. Current use of combined oral contraceptives was associated with lower odds of reporting any changes by 48% (OR = 0.52, 95CI = [0.34 to 0.78], <italic>P</italic>&lt;0.001). Odds of reporting any menstrual changes were increased by 44% for current smokers (OR = 1.16, 95CI = [1.06 to 1.26], <italic>P</italic>&lt;0.01) and by more than 50% for individuals with a positive COVID status [Long Covid (OR = 1.61, 95CI = [1.28 to 2.02], <italic>P</italic>&lt;0.001), acute COVID (OR = 1.54, 95CI = [1.27 to 1.86], <italic>P</italic>&lt;0.001)]. The effects remain after adjusting for self-reported magnitude of menstrual cycle changes over the year preceding the survey. Written accounts report diverse symptoms; the most common words include cramps, late, early, spotting, heavy and irregular, with a low level of clustering among them.</p></sec><sec><title>Conclusions</title><p>Following vaccination for COVID-19, menstrual disturbance occurred in 20% of individuals in a UK sample. Out of 33 variables investigated, smoking and a previous history of SARS-CoV-2 infection were found to be risk factors while using oestradiol-containing contraceptives was found to be a protective factor. Diverse experiences were reported, from menstrual bleeding cessation to heavy menstrual bleeding.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469560', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A Field-Deployable Diagnostic Assay for the Visual Detection of Misfolded Prions', 'authors': \"[{'name': 'Peter R. Christenson'}, {'name': 'Manci Li'}, {'name': 'Gage Rowden'}, {'name': 'Marc Schwabenlander'}, {'name': 'Tiffany M. Wolf'}, {'name': 'Sang-Hyun Oh'}, {'name': 'Peter A. Larsen'}]\", 'abstract': '<p>Diagnostic tools for the detection of protein-misfolding diseases (i.e., proteopathies) are limited. Gold nanoparticles (AuNPs) facilitate sensitive diagnostic techniques via visual color change for the detection of a variety of targets. In parallel, recently developed quaking-induced conversion (QuIC) assays leverage protein-amplification and fluorescent signaling for the accurate detection of misfolded proteins. Here, we combine AuNP and QuIC technologies for the visual detection of amplified misfolded prion proteins from tissues of wild white-tailed deer infected with chronic wasting disease (CWD), a prion disease of cervids. Our newly developed assay, MN-QuIC, enables both naked-eye and light-absorbance measurements for detection of misfolded prions. MN-QuIC leverages basic laboratory equipment that is cost-effective and portable, thus facilitating real-time prion diagnostics across a variety of settings. To test the portability of our assay, we deployed to a rural field-station in southeastern Minnesota and tested for CWD on site. We successfully demonstrated that MN-QuIC is functional in a non-traditional laboratory setting by performing a blinded analysis in the field and correctly identifying all CWD positive and CWD not detected deer at the field site in less than 24 hours, thus documenting the portability of the assay. Additionally, we show that electrostatic forces help govern the AuNP/prion interactions. Importantly, all of our white-tailed deer (n=37) were independently tested using ELISA, IHC, and/or RT-QuIC technologies and results secured with MN-QuIC were 100% consistent with these tests. We conclude that hybrid AuNP and QuIC assays, such as MN-QuIC, have great potential for sensitive, field-deployable diagnostics of a variety of protein misfolding diseases.</p><sec><title>Statement of Significance</title><p>Portable diagnostic tools for the detection of prion and protein-misfolding diseases are limited. Using chronic wasting disease (CWD) as a model, we demonstrate the functionality of a gold nanoparticle (AuNP)-based assay for the field-based detection of prion diseases. CWD is spreading through cervid populations across North America and the disease poses significant threats to a number of economic sectors, with global implications. Our newly developed diagnostic assay leverages the visible color change of AuNPs to detect the presence of CWD-causing prions. The assay can be performed using lowcost equipment and reagents, thus having great potential to expand prion disease surveillance efforts and prevent infectious prions from entering the human food chain.</p></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.468539', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'BrainPalmSeq: A curated RNA-seq database of palmitoylating and de-palmitoylating enzyme expression in the mouse brain', 'authors': \"[{'name': 'AR Wild'}, {'name': 'PW Hogg'}, {'name': 'S Flibotte'}, {'name': 'G Nasseri'}, {'name': 'R Hollman'}, {'name': 'K Haas'}, {'name': 'SX Bamji'}]\", 'abstract': '<p>Protein <italic>S</italic>-palmitoylation is a reversible post translational lipid modification that plays a critical role in neuronal development and plasticity, while dysregulated <italic>S</italic>-palmitoylation underlies a number of severe neurological disorders. Dynamic <italic>S</italic>-palmitoylation is regulated by a large family of ZDHHC palmitoylating enzymes, their accessory proteins, and a small number of known de-palmitoylating enzymes. Here, we curated and analyzed expression data for the proteins that mediate <italic>S</italic>-palmitoylation from publicly available RNAseq datasets, providing a comprehensive overview of their distribution in the mouse nervous system. We developed a web-tool that enables interactive visualization of the expression patterns for these proteins in the nervous system (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://brainpalmseq.med.ubc.ca/\">http://brainpalmseq.med.ubc.ca/</ext-link>), and explored this resource to find region and cell-type specific expression patterns that give insight into the function of palmitoylating and de-palmitoylating enzymes in the brain and neurological disorders. We found coordinated expression of ZDHHC enzymes with their accessory proteins, de-palmitoylating enzymes and other brain-expressed genes that included an enrichment of <italic>S</italic>-palmitoylation substrates. Finally, we utilized ZDHHC expression patterns to predict and validate palmitoylating enzyme-substrate interactions.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.25.470029', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Directed evolution of colE1 plasmid replication compatibility: a fast tractable tunable model for investigating biological orthogonality', 'authors': \"[{'name': 'Santiago Chaillou'}, {'name': 'Eleftheria-Pinelopi Stamou'}, {'name': 'Leticia Torres'}, {'name': 'Ana B. Riesco'}, {'name': 'Warren Hazelton'}, {'name': 'Vitor B. Pinheiro'}]\", 'abstract': '<p>Plasmids of the ColE1 family are among the most frequently used plasmids in molecular biology. They were adopted early in the field for many biotechnology applications, and as model systems to study plasmid biology. The mechanism of replication of ColE1 plasmids is well understood, involving the interaction between a plasmid-encoded sense-antisense gene pair (RNAI and RNAII). Because of its mechanism of replication, bacterial cells cannot maintain two different plasmids with the same origin, with one being rapidly lost from the population  a process known as plasmid incompatibility. While mutations in the regulatory genes RNAI and RNAII have been reported to make colE1 plasmids more compatible, there has been no attempt to engineer compatible colE1 origins, which can be used for multi-plasmid applications and that can bypass design constrains created by the current limited plasmid origin repertoire available. Here, we show that by targeting sequence diversity to the loop regions of RNAI (and RNAII), it is possible to select new viable colE1 origins that are compatible with the wild-type one. We demonstrate origin compatibility is not simply determined by sequence divergence in the loops, and that pairwise compatibility is not an accurate guide for higher order interactions. We identify potential principles to engineer plasmid copy number independently from other regulatory strategies and we propose plasmid compatibility as a tractable model to study biological orthogonality. New characterised plasmid origins increase flexibility and accessible complexity of design for challenging synthetic biology applications where biological circuits can be dispersed between multiple independent genetic elements.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.24.469925', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evolutionary features of a prolific subtype of avian influenza A virus in European waterfowl', 'authors': \"[{'name': 'Michelle Wille'}, {'name': 'Conny Tolf'}, {'name': 'Neus Latorre-Margalef'}, {'name': 'Ron A. M. Fouchier'}, {'name': 'Rebecca A. Halpin'}, {'name': 'David E. Wentworth'}, {'name': 'Jayna Ragwani'}, {'name': 'Oliver G. Pybus'}, {'name': 'Bjrn Olsen'}, {'name': 'Jonas Waldenstrm'}]\", 'abstract': '<p>Avian influenza A virus (AIV) is ubiquitous in waterfowl, and detected annually at high prevalence in waterfowl during the Northern Hemipshere autumn. Some AIV subtypes are globally common in waterfowl, such as H3N8, H4N6, and H6N2, and are detected in the same populations at high frequency, annually. In order to investigate genetic features associated to the long-term maintenance of common subtypes in migratory ducks, we sequenced 248 H4 viruses isolated across 8 years (2002-2009) from Mallards (<italic>Anas platyrhynchos</italic>) sampled in southeast Sweden. Phylogenetic analyses showed that both H4 and N6 sequences fell into in three distinct lineages, structured by year of isolation. Specifically, across the eight years of the study, we observed lineage replacement, whereby a different HA lineage circulated in the population each year. Analysis of deduced amino acid sequences of the HA lineages illustrated key differences in regions of the globular head of hemagglutinin that overlap with established antigentic sites in homologous hemagglutinin H3, suggesting the possibility of antigenic differences among these HA lineages. Beyond HA, lineage replacement was common to all segments, such that novel genome constellations were detected across years. A dominant genome constellation would rapidly amplify in the duck population, followed by unlinking of gene segments as a result of reassortment within 2-3 weeks following introduction. These data help reveal the evolutionary dynamics exhibited by AIV on both annual and decadal scales in an important reservoir host.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469695', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Antiviral activity of molnupiravir precursor NHC against Variants of Concern (VOCs) and its therapeutic window in a human lung cell model', 'authors': \"[{'name': 'Tessa Prince'}, {'name': 'Iah Donovan-Banfield'}, {'name': 'Hannah Goldswain'}, {'name': 'Rebekah Penrice-Randal'}, {'name': 'Lance Turtle'}, {'name': 'Tom Fletcher'}, {'name': 'Saye Khoo'}, {'name': 'Julian A. Hiscox'}]\", 'abstract': '<sec><title>Background</title><p>The UK Medicines and Regulatory Healthcare Agency (MHRA) have recently licensed the anti-viral drug, molnupiravir, for use in patients with mild-moderate COVID-19 disease with one or more risk factors for serious illness. Treatment with anti-viral drugs is best initiated early to prevent progression to severe disease, although the therapeutic window for intervention has not yet been fully defined.</p></sec><sec><title>Objectives</title><p>This study aimed to determine the activity of the molnupiravir parent drug (NHC) to different SARS-CoV-2 Variants of Concern (VoCs), and to establish the therapeutic window in human lung cell model.</p></sec><sec><title>Methods</title><p>Dose response assays were performed in parallel to determine the IC50 (the concentration of drug required to inhibit virus titre by 50%) of NHC against different variants. Human ACE-2 A549 cells were treated with NHC at different time points either before, during or after infection with SARS-CoV-2.</p></sec><sec><title>Results</title><p>Here we demonstrate that -D-N4-hydroxycytidine (NHC), the active metabolite of molnupiravir, has equivalent activity against four variants of SARS-CoV-2 in a human lung cell line ranging 0.04-0.16M IC50. Furthermore, we demonstrate that activity of the drug begins to drop after 48 hours post-infection.</p></sec><sec><title>Conclusions</title><p>One of the main advantages of molnupiravir is that it can be administered orally, and thus given to patients in an out-patient setting. These results support giving the drug early on after diagnosis or even in prophylaxis for individuals with high risk of developing severe disease.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469565', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Transcriptional and mutational signatures of the aging germline', 'authors': \"[{'name': 'Evan Witt'}, {'name': 'Christopher B Langer'}, {'name': 'Li Zhao'}]\", 'abstract': '<p>Aging is a complex biological process which is accompanied by changes in gene expression and mutational load. In many species including humans, old fathers pass on more paternally-derived <italic>de novo</italic> mutations, however, the cellular basis and cell types driving this pattern are still unclear. To understand the root causes of this phenomenon, we performed single-cell RNA-sequencing (scRNA-seq) on testes from young and old male <italic>Drosophila</italic>, as well as genomic sequencing (DNA-seq) on somatic tissue from the same flies. We found that early germ cells from old and young flies have similar mutational loads, but older flies are less able to remove mutations during spermatogenesis. This indicates that germline mutations arise from primarily non-replicative factors, and that the increased mutational load of older males is due to differences in genome maintenance activities such as repairs to DNA damage. We also found that T&gt;A mutations are enriched in older flies, and transcription-related enrichment terms are depleted in older males. Early spermatogenesis-enriched genes have lower dN/dS than late spermatogenesis-enriched genes, supporting the hypothesis that late spermatogenesis is the source of evolutionary innovation. This transcriptional disruption is reflected in the decreased expression of genome maintenance genes in early germ cells of older flies, as well as potentially aberrant transcription of transposable elements in the aging germline. Our results provide novel insights into the transcriptional and mutational signatures of the male germline.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.21266574', 'tweets': 18, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Reinfection with SARS-CoV-2: outcome, risk factors and vaccine efficacy in a Scottish cohort', 'authors': \"[{'name': 'Paul M McKeigue'}, {'name': 'David A McAllister'}, {'name': 'Chris Robertson'}, {'name': 'Diane Stockton'}, {'name': 'Helen M Colhoun'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>The objective of this study was to investigate how protection against COVID-19 conferred by previous infection is modified by vaccination.</p></sec><sec><title>Methods</title><p>In a cohort of all 152655 individuals in Scotland alive at 90 days after a positive test for SARS-CoV-2 (confirmed by cycle threshold &lt; 30, or two tests) followed till 22 September 2021, rate ratios for reinfection were estimated with calendar time or tests as timescale.</p></sec><sec><title>Findings</title><p>Rates of detected and hospitalised reinfection with COVID-19 while unvaccinated were respectively 6.8 (95% CI 6.4 to 7.2) and 0.18 (95% CI 0.12 to 0.25) per 1000 person-months. These rates were respectively 68% and 74% lower than in a matched cohort of individuals who had not previously tested positive. Efficacy of two doses of vaccine in those with previous infection was estimated as as 84% (95 percent CI 81% to 86%) against detected reinfection and 71% (95 percent CI 29% to 88%) against hospitalised or fatal reinfection. The rate of detected reinfection after two doses of vaccine was 1.35 (95% CI 1.02 to 1.78) times higher in those vaccinated before first infection than in those unvaccinated at first infection.</p></sec><sec><title>Interpretation</title><p>The combination of natural infection and vaccination provides maximal protection against new infection with SARS-CoV-2: prior vaccination does not impair this protection.</p></sec><sec><title>Funding</title><p>No specific funding was received for this work.</p></sec><sec><title>Research in context</title><sec><title>Evidence before this study</title><p>In a recent systematic review of cohort studies reported up to July 2021, the average reduction in COVID-19 infection rates in those with previous infection compared with those without evidence of previous infection was 90%. There is little information about the protective effect of previous infection against severe COVID-19, or about how the protective effects of previous infection against reinfection and severe disease are modified by vaccination.</p></sec><sec><title>What this paper adds</title><p>In unvaccinated individuals the protection against hospitalised COVID-19 conferred by previous infection is similar to that induced by vaccination. In those with previous infection, vaccination reduces the rates of reinfection and hospitalised COVID-19 by about 70%.</p></sec><sec><title>Implications of all the available evidence</title><p>The combination of natural infection and vaccination provides maximal protection against COVID-19: prior vaccination does not seriously impair this protection.</p></sec></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.20.469409', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Airway epithelial interferon response to SARS-CoV-2 is inferior to rhinovirus and heterologous rhinovirus infection suppresses SARS-CoV-2 replication', 'authors': \"[{'name': 'Elizabeth R. Vanderwall'}, {'name': 'Kaitlyn A. Barrow'}, {'name': 'Lucille M. Rich'}, {'name': 'David F. Read'}, {'name': 'Cole Trapnell'}, {'name': 'Oghenemega Okoloko'}, {'name': 'Steven F. Ziegler'}, {'name': 'Teal S. Hallstrand'}, {'name': 'Maria P. White'}, {'name': 'Jason S. Debley'}]\", 'abstract': '<sec><title>Introduction</title><p>Common alphacoronaviruses and human rhinoviruses (HRV) induce type I and III interferon (IFN) responses important to limiting viral replication in the airway epithelium. In contrast, highly pathogenic betacoronaviruses including SARS-CoV-2 may evade or antagonize RNA-induced IFN I/III responses.</p></sec><sec><title>Methods</title><p>In airway epithelial cells (AECs) from children and older adults we compared IFN I/III responses to SARS-CoV-2 and HRV-16, and assessed whether pre-infection with HRV-16, or pretreatment with recombinant IFN- or IFN-, modified SARS-CoV-2 replication. Bronchial AECs from children (ages 6-18 yrs.) and older adults (ages 60-75 yrs.) were differentiated <italic>ex vivo</italic> to generate organotypic cultures. In a biosafety level 3 (BSL-3) facility, cultures were infected with SARS-CoV-2 or HRV-16, and RNA and protein was harvested from cell lysates 96 hrs. following infection and supernatant was collected 48 and 96 hrs. following infection. In additional experiments cultures were pre-infected with HRV-16, or pre-treated with recombinant IFN-1 or IFN-2 before SARS-CoV-2 infection.</p></sec><sec><title>Results</title><p>Despite significant between-donor heterogeneity SARS-CoV-2 replicated 100 times more efficiently than HRV-16. IFNB1, INFL2, and CXCL10 gene expression and protein production following HRV-16 infection was significantly greater than following SARS-CoV-2. IFN gene expression and protein production were inversely correlated with SARS-CoV-2 replication. Treatment of cultures with recombinant IFN1 or IFN2, or pre-infection of cultures with HRV-16, markedly reduced SARS-CoV-2 replication.</p></sec><sec><title>Discussion</title><p>In addition to marked between-donor heterogeneity in IFN responses and viral replication, SARS-CoV-2 elicits a less robust IFN response in primary AEC cultures than does rhinovirus, and heterologous rhinovirus infection, or treatment with recombinant IFN-1 or IFN-2, markedly reduces SARS-CoV-2 replication.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469610', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Electrophysiological correlates of confidence differ across correct and erroneous perceptual decisions', 'authors': \"[{'name': 'Daniel Feuerriegel'}, {'name': 'Mackenzie Murphy'}, {'name': 'Alexandra Konski'}, {'name': 'Vinay Mepani'}, {'name': 'Jie Sun'}, {'name': 'Robert Hester'}, {'name': 'Stefan Bode'}]\", 'abstract': '<p>Every decision we make is accompanied by an estimate of the likelihood that our decision is accurate or appropriate. This likelihood estimate is termed our degree of decision confidence. Recent work has uncovered event-related potential (ERP) correlates of confidence both during decision formation and after a decision has been made. However, the interpretation of these findings is complicated by methodological issues related to ERP amplitude measurement that are prevalent across existing studies. To more accurately characterise the neural correlates of confidence, we presented participants with a difficult perceptual decision task that elicited a broad range of confidence ratings. We identified a frontal ERP component within an onset prior to the behavioural response, which exhibited more positive-going amplitudes in trials with higher confidence ratings. This frontal effect also biased measures of the centro-parietal positivity (CPP) component at parietal electrodes via volume conduction. Amplitudes of the error positivity (Pe) component that followed each decision were negatively associated with confidence for trials with decision errors, but not for trials with correct decisions. We provide evidence for both pre- and post-decisional neural correlates of decision confidence that are observed in trials with correct and erroneous decisions, respectively. Our findings suggest that certainty in having made a correct response is associated with frontal activity during decision formation, whereas certainty in having committed an error is instead associated with the post-decisional Pe component. We additionally show that some previously reported associations between decision confidence and CPP/Pe component amplitudes are (at least partly) a consequence of ERP amplitude measurement-related confounds.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469715', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Contractile force of transplanted cardiomyocytes contributes to heart function after injury', 'authors': \"[{'name': 'Tim Stdemann'}, {'name': 'Judith Rssinger'}, {'name': 'Christoph Manthey'}, {'name': 'Birgit Geertz'}, {'name': 'Rajiven Srikantharajah'}, {'name': 'Constantin von Bibra'}, {'name': 'Aya Shibamiya'}, {'name': 'Maria Khne'}, {'name': 'Antonius Wiehler'}, {'name': 'J. Simon Wiegert'}, {'name': 'Thomas Eschenhagen'}, {'name': 'Florian Weinberger'}]\", 'abstract': '<p>Transplantation of pluripotent stem cell-derived cardiomyocytes represents an innovative therapeutic strategy for heart failure. Studies in small and large animals have demonstrated functional recovery of left ventricular function after cardiomyocyte transplantation<sup>14</sup>, and first clinical studies are currently underway<sup>5</sup>. Yet, the mechanism of action underlying graft-induced benefit is unknown<sup>6</sup>. Here we demonstrate that transplanted cardiomyocytes actively contribute to heart function. We transplanted cardiomyocytes with an optogenetic off-on switch in a guinea pig cardiac injury model. Light-induced inhibition of engrafted cardiomyocyte contractility resulted in a rapid decrease of left ventricular function that was fully reversible with the offset of photostimulation. Hence, our optogenetic approach demonstrated that transplanted cardiomyocytes actively participate in heart function, supporting the hypothesis that the delivery of new force-generating myocardium can serve as a regenerative therapeutic strategy.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469568', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'A flexible workflow for building spectral libraries from narrow window data independent acquisition mass spectrometry data', 'authors': \"[{'name': 'Lilian R. Heil'}, {'name': 'William E. Fondrie'}, {'name': 'Christopher D. McGann'}, {'name': 'Alexander J. Federation'}, {'name': 'William S. Noble'}, {'name': 'Michael J. MacCoss'}, {'name': 'Uri Keich'}]\", 'abstract': '<p>Advances in library-based methods for peptide detection from data independent acquisition (DIA) mass spectrometry have made it possible to detect and quantify tens of thousands of peptides in a single mass spectrometry run. However, many of these methods rely on a comprehensive, high quality spectral library containing information about the expected retention time and fragmentation patterns of peptides in the sample. Empirical spectral libraries are often generated through data-dependent acquisition and may suffer from biases as a result. Spectral libraries can be generated <italic>in silico</italic> but these models are not trained to handle all possible post-translational modifications. Here, we propose a false discovery rate controlled spectrum-centric search workflow to generate spectral libraries directly from gas-phase fractionated DIA tandem mass spectrometry data. We demonstrate that this strategy is able to detect phosphorylated peptides and can be used to generate a spectral library for accurate peptide detection and quantitation in wide window DIA data. We compare the results of this search workflow to other library-free approaches and demonstrate that our search is competitive in terms of accuracy and sensitivity. These results demonstrate that the proposed workflow has the capacity to generate spectral libraries while avoiding the limitations of other methods.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.25.470048', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'The structure of the enigmatic ripple phase in saturated bilayers resolved: Machine learning reveals four lipid populations', 'authors': \"[{'name': 'Matthew Davies'}, {'name': 'A. D. Reyes-Figueroa'}, {'name': 'Andrey A. Gurtovenko'}, {'name': 'Daniel Frankel'}, {'name': 'Mikko Karttunen'}]\", 'abstract': '<p>A new mixed radial-angular, three-particle correlation function method in combination with unsupervised machine learning (ML) was applied to examine the emergence of the ripple phase in dipalmitoyphosphatidylcholine (DPPC) lipid bilayers using data from atomistic molecular dynamics (MD) simulations of system sizes ranging from 128 to 4,096 lipids. Based on the acyl tail conformations, the analysis revealed the presence of four distinct conformational populations of lipids in the ripple phases of the DPPC lipid bilayers. The expected gel- (ordered; <italic>L</italic><sub>o</sub>) and fluid-like (disordered; <italic>L</italic><sub>d</sub>) lipids are found along with their splayed tail equivalents (<italic>L</italic><sub>o,s</sub> and <italic>L</italic><sub>d,s</sub>). These lipids differ based on their gauche distribution and tail packing. The disordered (<italic>L</italic><sub>d</sub>) and disordered splayed (<italic>L</italic><sub>d,s</sub>) lipids spatially cluster in the ripple in the groove side, that is, in an asymmetric manner across the bilayer leaflets. The ripple phase does not contain large numbers of <italic>L</italic><sub>d</sub> lipids, instead they only exist on the interface of the groove side of the undulation. The bulk of the groove side is a complex coexistence of <italic>L</italic><sub>o</sub>, <italic>L</italic><sub>o,s</sub> and <italic>L</italic><sub>d,s</sub> lipids. The convex side of the undulation contains predominantly <italic>L</italic><sub>o</sub> lipids. Thus, the structure of the ripple phase is neither a simple coexistence of ordered and disordered lipids nor a coexistence of ordered interdigitating gel-like (<italic>L</italic><sub>o</sub>) and ordered splayed (<italic>L</italic><sub>o,s</sub>) lipids, but instead a coexistence of an ordered phase and a complex mixed phase. Principal component analysis (PCA) further confirmed the existence of the four lipid groups.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.23.469614', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Peptidoglycan recycling contributes to outer membrane integrity and carbapenem tolerance in Acinetobacter baumannii', 'authors': \"[{'name': 'Nowrosh Islam'}, {'name': 'Misha I. Kazi'}, {'name': 'Katie N. Kang'}, {'name': 'Jacob Biboy'}, {'name': 'Joe Gray'}, {'name': 'Feroz Ahmed'}, {'name': 'Richard D. Schargel'}, {'name': 'Cara C. Boutte'}, {'name': 'Tobias Drr'}, {'name': 'Waldemar Vollmer'}, {'name': 'Joseph M. Boll'}]\", 'abstract': '<p>The Gram-negative cell envelope is an essential structure that not only protects the cell against lysis from the internal turgor, but also forms a barrier to limit entry of antibiotics. Some of our most potent bactericidal antibiotics, the -lactams, exploit the essentiality of the cell envelope by inhibiting its biosynthesis, typically inducing lysis and rapid death. However, many Gram-negative bacteria exhibit antibiotic tolerance, the ability to sustain viability in the presence of -lactams for extended time periods. Despite several studies showing that antibiotic tolerance contributes directly to treatment failure, and is a steppingstone in acquisition of true resistance, the molecular factors that promote intrinsic tolerance are not well-understood. <italic>Acinetobacter baumannii</italic> is a critical-threat nosocomial pathogen notorious for its ability to rapidly develop multidrug resistance. While typically reserved to combat multidrug resistant infections, carbapenem -lactam antibiotics (i.e., meropenem) are first-line prescriptions to treat <italic>A. baumannii</italic> infections. Meropenem tolerance in Gram-negative pathogens is characterized by morphologically distinct populations of spheroplasts, but the impact of spheroplast formation is not fully understood. Here, we show that susceptible <italic>A. baumannii</italic> clinical isolates demonstrate high intrinsic tolerance to meropenem, form spheroplasts with the antibiotic and revert to normal growth after antibiotic removal. Using transcriptomics and genetics screens, we characterized novel tolerance factors and found that outer membrane integrity maintenance, drug efflux and peptidoglycan homeostasis collectively contribute to meropenem tolerance in <italic>A. baumannii</italic>. Furthermore, outer membrane integrity and peptidoglycan recycling are tightly linked in their contribution to meropenem tolerance in <italic>A. baumannii</italic>.</p><sec><title>Importance</title><p>Carbapenem treatment failure associated with superbug infections has rapidly increased in prevalence, highlighting an urgent need to develop new therapeutic strategies. Antibiotic tolerance can directly lead to treatment failure but has also been shown to promote acquisition of true resistance within a population. While some studies have addressed mechanisms that promote tolerance, factors that underlie Gram-negative bacterial survival during carbapenem treatment are not well-understood. Here, we characterized a role for peptidoglycan recycling in outer membrane integrity maintenance and carbapenem tolerance in <italic>A. baumannii</italic>. These studies suggest that the pathogen limits antibiotic concentrations in the periplasm and highlights physiological processes that could be targeted to improve antimicrobial treatment.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469682', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Categorical encoding of voice in human superior temporal cortex', 'authors': \"[{'name': 'Kyle Rupp'}, {'name': 'Jasmine Hect'}, {'name': 'Madison Remick'}, {'name': 'Avniel Ghuman'}, {'name': 'Bharath Chandrasekaran'}, {'name': 'Lori L. Holt'}, {'name': 'Taylor J. Abel'}]\", 'abstract': '<p>The ability to recognize abstract features of voice during auditory perception is a complex, yet poorly understood, feat of human audition. For the listener, this occurs in near-automatic fasion to seamlessly extract complex cues from a highly variable auditory signal. Voice perception depends on specialized regions of auditory cortex, including superior temporal gyrus (STG) and superior temporal sulcus (STS). However, the nature of voice encoding at the cortical level remains poorly understoood. We leverage intracerebral recordings across human auditory cortex during presentation of voice and non-voice acoustic stimuli to examine voice encoding in auditory cortex, in eight patient-participants undergoing epilepsy surgery evaluation. We show that voice-selectivity increases along the auditory hierarchy from supratemporal plane (STP) to the STG and STS. Results show accurate decoding of vocalizations from human auditory cortical activity even in the complete absence of linguistic content. These findings show an early, less-selective temporal window of neural activity in the STG and STS followed by a sustained, strongly voice-selective window. We then developed encoding models that demonstrate divergence in the encoding of acoustic features along the auditory hierarchy, wherein STG/STS responses were best explained by voice category as opposed to the acoustic features of voice stimuli. This is in contrast to neural activity recorded from STP, in which responses were accounted for by acoustic features. These findings support a model of voice perception that engages categorical encoding mechanisms within STG and STS.</p><sec><title>Significance Statement</title><p>Voice perception occurs via specialized networks in higher order auditory cortex, yet how voice features are encoded remains a central unanswered question. With human intracerebral recordings of auditory cortex, we provide evidence for categorical encoding of voice in STG and STS and occurs in the absence of linguistic content. This selectivity strengthens after an initial onset response and cannot be explained by simple acoustic features. Together, these data support the existence of sites within STG and STS that are specialized for voice perception.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.26.469979', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Input to layer 1 of somatosensory cortex: Local input outweighs long-range and can modulate the activity of layer 1 interneurons', 'authors': \"[{'name': 'Julia Ledderose'}, {'name': 'Timothy A Zolnik'}, {'name': 'Maria Toumazou'}, {'name': 'Thorsten Trimbuch'}, {'name': 'Christian Rosenmund'}, {'name': 'Britta J Eickholt'}, {'name': 'Dieter Jaeger'}, {'name': 'Matthew E Larkum'}, {'name': 'Robert Sachdev'}]\", 'abstract': '<p>Neocortical layer (L) 1 is a locus for interactions between long-range inputs, L1 interneurons and apical tuft dendrites of pyramidal neurons. Even though we have a wealth of information about L1, the level and effect of local input to this layer have not been quantified. Here we characterized the input to L1 of mouse somatosensory cortex with fast blue, monosynaptic rabies and optogenetics. Our work shows that most of the input to L1 is local, and that both local and long-range inputs to this layer arise predominantly from L2/3 and L5 neurons. Subtypes of L5 and L6b neurons project to the overlying L1 with different probabilities. VIP and SST interneurons in L2/3 and L5 also innervate L1. A subset of local L5, the intratelencephalic, pyramidal neurons, drive L1 interneurons but have no effect on L5 apical tuft dendrites. Monosynaptic rabies-based retrograde labelling reveals presynaptic boutons covering the entire somato-dendritic axis of pyramidal neurons, including in L1. When fast blue application was combined with rabies virus, we found that only a fraction of local and long-range neurons was both presynaptic to L5 neurons and projected to L1. These results demonstrate that L1 receives a large proportion of its input from local neurons, and that some of these inputs specifically target interneurons. We conclude that L1 is not just a site for interaction between long-range feedback and apical tuft dendrites of pyramidal cells, it is also a site for complex modulation of pyramidal neurons and interneurons by local inputs.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.24.21266740', 'tweets': 16, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Smoking-dependent expression alterations in nasal epithelium reveal immune impairment linked to germline variation and lung cancer risk', 'authors': \"[{'name': 'Maria Stella de Biase'}, {'name': 'Florian Massip'}, {'name': 'Tzu-Ting Wei'}, {'name': 'Federico M. Giorgi'}, {'name': 'Rory Stark'}, {'name': 'Amanda Stone'}, {'name': 'Amy Gladwell'}, {'name': 'Martin OReilly'}, {'name': 'Ines de Santiago'}, {'name': 'Kerstin Meyer'}, {'name': 'Florian Markowetz'}, {'name': 'Bruce A J Ponder'}, {'name': 'Robert C Rintoul'}, {'name': 'Roland F Schwarz'}]\", 'abstract': '<p>Lung cancer is the leading cause of cancer-related death in the world. In contrast to many other cancers, a direct connection to lifestyle risk in the form of cigarette smoke has long been established. More than 50% of all smoking-related lung cancers occur in former smokers, often many years after smoking cessation. Despite extensive research, the molecular processes for persistent lung cancer risk are unclear. CT screening of current and former smokers has been shown to reduce lung cancer mortality by up to 26%.</p><p>To examine whether clinical risk stratification can be improved upon by the addition of genetic data, and to explore the mechanisms of the persisting risk in former smokers, we have analyzed transcriptomic data from accessible airway tissues of 487 subjects. We developed a model to assess smoking associated gene expression changes and their reversibility after smoking is stopped, in both healthy subjects and clinic patients. We find persistent smoking-associated immune alterations to be a hallmark of the clinic patients. Integrating previous GWAS data using a transcriptional network approach, we demonstrate that the same immune and interferon related pathways are strongly enriched for genes linked to known genetic risk factors, demonstrating a causal relationship between immune alteration and lung cancer risk. Finally, we used accessible airway transcriptomic data to derive a non-invasive lung cancer risk classifier.</p><p>Our results provide initial evidence for germline-mediated personalised smoke injury response and risk in the general population, with potential implications for managing long-term lung cancer incidence and mortality.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469663', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'SARS-CoV-2 variants of concern Alpha, Beta, Gamma and Delta have extended ACE2 receptor host-ranges', 'authors': \"[{'name': 'Nazia Thakur'}, {'name': 'Giulia Gallo'}, {'name': 'Joseph Newman'}, {'name': 'Thomas P. Peacock'}, {'name': 'Luca Biasetti'}, {'name': 'Catherine N. Hall'}, {'name': 'Edward Wright'}, {'name': 'Wendy Barclay'}, {'name': 'Dalan Bailey'}]\", 'abstract': '<p>Following the emergence of SARS-CoV-2 in China in late 2019 a number of variants have emerged, with two of these  Alpha and Delta  subsequently growing to global prevalence. One characteristic of these variants are changes within the Spike protein, in particular the receptor binding domain (RBD). From a public health perspective these changes have important implications for increased transmissibility and immune escape; however, their presence could also modify the intrinsic host-range of the virus. Using viral pseudotyping we examined whether the variants of concern (VOCs) Alpha, Beta, Gamma and Delta have differing host ACE2 receptor usage patterns, focusing on a range of relevant mammalian ACE2 proteins. All four VOCs were able to overcome a previous restriction for mouse ACE2, with demonstrable differences also seen for individual VOCs with rat, ferret or civet ACE2 receptors, changes which we subsequently attribute to N501Y and E484K substitutions within the Spike RBD.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.26.470178', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': \"Couching the brain's multiple-demand and language-specific systems within a macroscale gradient of cortical connectivity\", 'authors': \"[{'name': 'Rocco Chiou'}, {'name': 'Elizabeth Jefferies'}, {'name': 'John Duncan'}, {'name': 'Gina F. Humphreys'}, {'name': 'Matthew Lambon Ralph'}]\", 'abstract': \"<p>The cerebrum comprises a set of specialised systems that tile across the cortical sheet, forming a tapestry-like configuration. For example, the multiple-demand and language-specific systems occupy largely separate neural estates and exhibit disparate functional profiles. Although delimiting the boundary between systems informs where cortical sheet functionally fractionates, it remains unclear why different systems' topographical placements are spatially configured in typical manners and how a macroscale architecture arises from this topography. Novel approaches have tackled this challenge by condensing the topography into a principal gradient, which represents the workflow of information processing from sensory-motoric to abstract-cognitive. To understand how the multiple-demand and language-specific systems are accommodated in the gradient framework, here we used fMRI to probe cognitive operations in semantic vs. visuospatial domains and projected functional activities onto the principal gradient. We found that the two systems showed distinct trajectories of distribution along gradient tiers, suggesting different roles in the transition from sensation to cognition. Critically, when semantic processing became difficult, the brain recruited a specialised 'semantic-control' system that was a functional and anatomical 'hybrid' juxtaposed between the multi-demand and language systems. We discuss how the brain's modular division can be better understood through the lens of a dimensionality-reduced gradient-like architecture.</p>\", 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.25.21266863', 'tweets': 16, 'archive': 'medRxiv', 'subject-area': 'Pain Medicine', 'covid': False, 'title': 'Predicting Post-operative Pain in Lung Cancer Patients using Pre-operative Peak Alpha Frequency', 'authors': \"[{'name': 'Samantha K. Millard'}, {'name': 'Andrew J. Furman'}, {'name': 'Amy Kerr'}, {'name': 'David A. Seminowicz'}, {'name': 'Fang Gao-Smith'}, {'name': 'Babu V. Naidu'}, {'name': 'Ali Mazaheri'}]\", 'abstract': '<sec><title>Aims and Objectives</title><p>Experimental models of neuropathic pain suggest that individual peak alpha frequency (PAF), measured using electroencephalography (EEG), can predict future pain sensitivity in experimental settings. Here, we tested whether PAF could predict future pain severity in a clinical setting in patients undergoing thoracotomy.</p></sec><sec><title>Methods</title><p>Recorded using wearable around the ear electrodes (cEEGrids), the feasibility and efficacy of pre-operative PAF as a neuro-marker for post-operative pain was assessed in 16 patients undergoing thoracic surgery for lung cancer (age = 67.53  4.38 [SD]). Patients also provided numerical ratings (0-10) of current, average, and worst pain pre-operatively as well as within three days post-operatively</p></sec><sec><title>Results and Significance</title><p>Pre-operative PAF of less than 9 Hz was highly sensitive (1.0) and specific (0.86) in identifying patients who would go on to experience severe (&gt;7/10) worst pain. Moreover, PAF was negatively correlated with patients current, average, and worst post-operative pain. PAF was significantly higher for those reporting lower pain severity compared to those reporting higher pain severity in the immediate post-operative period. This suggests that PAF is a promising neuro-marker to pre-operatively assess individual susceptibility to severe pain in the immediate post-operative period, possibly enabling more informed assessment of an individuals suitability for surgery.</p></sec>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.26.470087', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'High resolution species detection: accurate long read eDNA metabarcoding of North Sea fish using Oxford Nanopore sequencing', 'authors': \"[{'name': 'Karlijn Doorenspleet'}, {'name': 'Lara Jansen'}, {'name': 'Saskia Oosterbroek'}, {'name': 'Oscar Bos'}, {'name': 'Pauline Kamermans'}, {'name': 'Max Janse'}, {'name': 'Erik Wurz'}, {'name': 'Albertinka Murk'}, {'name': 'Reindert Nijland'}]\", 'abstract': '<p>To monitor the effect of nature restoration projects in North Sea ecosystems, accurate and intensive biodiversity assessments are vital. DNA based techniques and especially environmental DNA (eDNA) metabarcoding from seawater is becoming a powerful monitoring tool. However, current approaches are based on genetic target regions of &lt;500 nucleotides, which offer limited taxonomic resolution. This study aims to develop and validate a long read nanopore sequencing method for eDNA that enables improved identification of fish species. We designed a universal primer pair targeting a 2kb region covering the 12S and 16S rRNA genes of fish mitochondria. eDNA was amplified and sequenced using the Oxford Nanopore MiniON. Sequence data was processed using the new pipeline Decona, and accurate consensus identities of above 99.9% were retrieved. The primer set efficiency was tested with eDNA from a 3.000.000 L zoo aquarium with 31 species of bony fish and elasmobranchs. Over 55% of the species present were identified on species level and over 75% on genus level. Next, our long read eDNA metabarcoding approach was applied to North Sea eDNA field samples collected at ship wreck sites, the Gemini Offshore Wind Farm, the Borkum Reef Grounds and a bare sand bottom. Here, location specific fish and vertebrate communities were obtained. Incomplete reference databases still form a major bottleneck in further developing high resolution long read metabarcoding. Yet, the method has great potential for rapid and accurate fish species monitoring in marine field studies.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.23.469707', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'The RNA-binding protein Orb2 is associated with microcephaly and supports centrosome asymmetry in neural stem cells', 'authors': \"[{'name': 'Beverly V. Robinson'}, {'name': 'Junnan Fang'}, {'name': 'Dipen S. Mehta'}, {'name': 'Joseph Buehler'}, {'name': 'Dorothy A. Lerit'}]\", 'abstract': '<p>To maintain a balance of self-renewal versus neurogenesis, neural stem cells (NSCs) undergo repeated cycles of asymmetric cell division along an invariant polarity axis instructed by centrosomes. During interphase, the NSC centrosomes are defined by marked asymmetries in protein composition and functional activity as microtubule-organizing centers. Here, we show a conserved RNA-binding protein, Orb2, supports centrosome asymmetry in interphase NSCs. While Orb2 localizes to the active apical centrosome, it promotes the transient inactivation of the basal centrosome required for centrosome segregation and spindle morphogenesis. Orb2 is required cell autonomously within NSCs to support centrosome asymmetry and maintenance of the stem cell pool. Conversely, loss of <italic>orb2</italic> manifests in microcephaly independent of Orb2 function in NSCs. We suggest Orb2 plays opposing roles in centrosome activation and inactivation, possibly through the translational regulation of multiple mRNA substrates. Bioinformatics uncovers a significant overlap among RNA targets between <italic>Drosophila</italic> Orb2 and human CPEB4, consistent with a conserved role for CPEB proteins in centrosome regulation and neurodevelopment.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.24.469537', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': True, 'title': 'Neurotoxic Amyloidogenic Peptides Identified in the Proteome of SARS-COV2: Potential Implications for Neurological Symptoms in COVID-19', 'authors': \"[{'name': 'Saba Islam'}, {'name': 'Mirren Charnley'}, {'name': 'Guneet Bindra'}, {'name': 'Julian Ratcliffe'}, {'name': 'Jiangtao Zhou'}, {'name': 'Raffaele Mezzenga'}, {'name': 'Mark Hulett'}, {'name': 'Kyunghoon Han'}, {'name': 'Joshua T. Berryman'}, {'name': 'Nicholas P. Reynolds'}]\", 'abstract': '<p>COVID-19 is primarily known as a respiratory disease caused by the virus SARS-CoV-2. However, neurological symptoms such as memory loss, sensory confusion, cognitive and psychiatric issues, severe headaches, and even stroke are reported in as many as 30% of cases and can persist even after the infection is over (so-called long COVID). These neurological symptoms are thought to be caused by brain inflammation, triggered by the virus infecting the central nervous system of COVID-19 patients, however we still dont fully understand the mechanisms for these symptoms. The neurological effects of COVID-19 share many similarities to neurodegenerative diseases such as Alzheimers and Parkinsons in which the presence of cytotoxic protein-based amyloid aggregates is a common etiological feature. Following the hypothesis that some neurological symptoms of COVID-19 may also follow an amyloid etiology we performed a bioinformatic scan of the SARS-CoV-2 proteome, detecting peptide fragments that were predicted to be highly amyloidogenic. We selected two of these peptides and discovered that they do rapidly self-assemble into amyloid. Furthermore, these amyloid assemblies were shown to be highly toxic to a neuronal cell line. We introduce and support the idea that cytotoxic amyloid aggregates of SARS-CoV-2 proteins are causing some of the neurological symptoms commonly found in COVID-19 and contributing to long COVID, especially those symptoms which are novel to long COVID in contrast to other post-viral syndromes.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.20.469369', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Decidual immune response following COVID-19 during pregnancy varies by timing of maternal SARS-CoV-2 infection', 'authors': \"[{'name': 'Lillian J. Juttukonda'}, {'name': 'Elisha M. Wachman'}, {'name': 'Jeffery Boateng'}, {'name': 'Mayuri Jain'}, {'name': 'Yoel Benarroch'}, {'name': 'Elizabeth S. Taglauer'}]\", 'abstract': '<p>While COVID-19 infection during pregnancy is common, fetal transmission is rare, suggesting that intrauterine mechanisms form an effective blockade against SARS-CoV-2. Key among these is the decidual immune environment of the placenta. We hypothesized that decidual leukocytes are altered by maternal SARS-CoV-2 infection in pregnancy and that this decidual immune resonse is shaped by the timing of infection during gestation. To address this hypothesis, we collected decidua basalis tissues at delivery from women with symptomatic COVID-19 during second (2<sup>nd</sup> Tri COVID, n=8) or third trimester (3<sup>rd</sup> Tri COVID, n=8) and SARS-CoV-2-negative controls (Control, n=8). Decidual natural killer (NK) cells, macrophages and T cells were evaluated using quantitative microscopy, and pro- and anti-inflammatory cytokine mRNA expression was evaluated using quantitative reverse transcriptase PCR (qRT-PCR). When compared with the Control group, decidual tissues from 3<sup>rd</sup> Tri COVID exhibited significantly increased macrophages, NK cells and T cells, whereas 2<sup>nd</sup> Tri COVID only had significantly increased T cells. In evaluating decidual cytokine expression, we noted that IL-6, IL-8, IL-10 and TNF- were significantly correlated with macrophage cell abundance. However, in 2<sup>nd</sup> Tri COVID tissues, there was significant downregulation of IL-6, IL-8, IL-10, and TNF-. Taken together, these results suggest innate and adaptive immune responses are present at the maternal-fetal interface in maternal SARS-CoV-2 infections late in pregnancy, and that infections earlier in pregnancy show evidence of a resolving immune response. Further studies are warranted to characterize the full scope of intrauterine immune responses in pregnancies affected by maternal COVID-19.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.21.469432', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Patient brain organoids identify a link between the 16p11.2 copy number variant and the RBFOX1 gene', 'authors': \"[{'name': 'Milos Kostic'}, {'name': 'Joseph J. Raymond'}, {'name': 'Beata Henry'}, {'name': 'Tayfun Tumkaya'}, {'name': 'Jivan Khlghatyan'}, {'name': 'Jill Dvornik'}, {'name': 'Jack S. Hsiao'}, {'name': 'Seon Hye Cheon'}, {'name': 'Jonathan Chung'}, {'name': 'Yishan Sun'}, {'name': 'Ricardo E. Dolmetsch'}, {'name': 'Kathleen A. Worringer'}, {'name': 'Robert J. Ihry'}]\", 'abstract': '<p>Copy number variants (CNVs) that delete or duplicate 30 genes within the 16p11.2 genomic region give rise to a range of neurodevelopmental phenotypes with high penetrance in humans. Despite the identification of this small region, the mechanisms by which 16p11.2 CNVs lead to disease are unclear. Relevant models, like human cortical organoids (hCOs), are needed to understand the human-specific mechanisms of neurodevelopmental disease. We generated hCOs from 18 patients and controls, profiling 167,958 cells with single cell (sc)RNA-seq. Analysis revealed neuronal-specific differential expression of genes outside of the 16p11.2 region that were related to cell-cell adhesion, neuronal projection growth, and neurodevelopmental disorders. Furthermore, 16p11.2 deletion syndrome organoids exhibited reduced mRNA and protein levels of RBFOX1, a gene which can also harbor CNVs linked to neurodevelopmental phenotypes. We found that many genes previously shown to be regulated by RBFOX1 are also perturbed in organoids from patients with 16p11.2 deletion syndrome, and thus identified a novel link between independent CNVs associated with neuronal development and autism. Overall, this work suggests convergent signaling, which indicates the possibility of a common therapeutic mechanism across multiple rare neuronal diseases.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469329', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Directing iPSC Differentiation into iTenocytes using Combined Scleraxis Overexpression and Cyclic Loading', 'authors': \"[{'name': 'Angela Papalamprou'}, {'name': 'Victoria Yu'}, {'name': 'Angel Chen'}, {'name': 'Tina Stefanovic'}, {'name': 'Giselle Kaneda'}, {'name': 'Khosrowdad Salehi'}, {'name': 'Chloe Castaneda'}, {'name': 'Arkadiusz Gertych'}, {'name': 'Juliane D Glaeser'}, {'name': 'Dmitriy Sheyn'}]\", 'abstract': '<p>Regenerative therapies for tendon are falling behind other tissues due to the lack of an appropriate and potent cell therapeutic candidate. This study aimed to induce cell tenogenesis using stable Scleraxis (Scx) overexpression in combination with uniaxial mechanical stretch of mesenchymal stromal cells (MSCs) of different origins. Scleraxis (Scx) is the single direct molecular regulator of tendon differentiation known so far. Mechanoregulation is known to be a central element guiding tendon development and healing. Cells explored were bone marrow-derived (BM-)MSCs as well as MSCs differentiated from induced pluripotent stem cells (iMSCs). Upregulation of early and late tendon markers, increased collagen deposition, and morphometric and cytoskeleton-related changes in mechanically stimulated Scx-overexpressing iMSCs compared to BM-MSCs and controls. Our findings suggest that these cells can be differentiated into tenocytes and may be a better candidate for tendon cell therapy applications than BM-MSCs.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.24.469176', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Single-cell RNA sequencing of Plasmodium vivax sporozoites reveals stage- and species-specific transcriptomic signatures', 'authors': \"[{'name': 'Anthony A. Ruberto'}, {'name': 'Caitlin Bourke'}, {'name': 'Amlie Vantaux'}, {'name': 'Steven P. Maher'}, {'name': 'Aaron Jex'}, {'name': 'Benoit Witkowski'}, {'name': 'Georges Snounou'}, {'name': 'Ivo Mueller'}]\", 'abstract': '<p><italic>Plasmodium vivax</italic> sporozoites reside in the salivary glands of a mosquito before infecting a human host. Previous transcriptome-wide studies in populations of these forms were limited in their ability to elucidate cell-to-cell variation, thereby masking cellular states potentially important in understanding transmission outcomes. In this study, we performed transcription profiling on 9,947 <italic>P. vivax</italic> sporozoites to assess the extent to which they differ at single-cell resolution. We show that sporozoites residing in the mosquitos salivary glands exist in distinct developmental states, as defined by their transcriptomic signatures. Additionally, relative to <italic>P. falciparum, P. vivax</italic> displays overlapping and unique gene usage patterns, highlighting conserved and species-specific gene programs. Notably, distinguishing <italic>P. vivax</italic> from <italic>P. falciparum</italic> were a subset of <italic>P. vivax</italic> sporozoites expressing genes associated with translational regulation and repression. Finally, our comparison of single-cell transcriptomic data from <italic>P. vivax</italic> sporozoite and erythrocytic forms reveals gene usage patterns unique to sporozoites. In defining the transcriptomic signatures of individual <italic>P. vivax</italic> sporozoites, our work provides new insights into the factors driving their developmental trajectory and lays the groundwork for a more comprehensive <italic>P. vivax</italic> cell atlas.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469628', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'IReNA: integrated regulatory network analysis of single-cell transcriptomes', 'authors': \"[{'name': 'Junyao Jiang'}, {'name': 'Seth Blackshaw'}, {'name': 'Jiang Qian'}, {'name': 'Jie Wang'}]\", 'abstract': '<p>While single-cell RNA sequencing (scRNA-seq) is widely used to profile gene expression, few methods are available to infer gene regulatory networks using scRNA-seq data. Here, we developed and extended IReNA (Integrated Regulatory Network Analysis) to perform regulatory network analysis using scRNA-seq profiles. Four features are developed for IReNA. First, regulatory networks are divided into different modules which represent distinct biological functions. Second, transcription factors significantly regulating each gene module can be identified. Third, regulatory relationships among modules can be inferred. Fourth, IReNA can integrate ATAC-seq data into regulatory network analysis. If ATAC-seq data is available, both transcription factor footprints and binding motifs are used to refine regulatory relationships among co-expressed genes. Using public datasets, we showed that integrated network analysis of scRNA-seq data with ATAC-seq data identified a higher fraction of known regulators than scRNA-seq data alone. Moreover, IReNA provided a better performance of network analysis than currently available methods. Beyond the reconstruction of regulatory networks, IReNA can modularize regulatory networks, and identify key regulators and significant regulatory relationships for modules, facilitating the systems-level understanding of biological regulatory mechanisms. The R package IReNA is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/jiang-junyao/IReNA\">https://github.com/jiang-junyao/IReNA</ext-link>.</p><sec><title>Key points</title><list list-type=\"order\"><list-item><p>We have developed IReNA, a new method for reconstructing regulatory networks using either scRNA-seq and ATAC-seq data or scRNA-seq data alone.</p></list-item><list-item><p>IReNA can establish modular regulatory networks to identify key regulatory factors and statistically significant regulatory relationships among modules.</p></list-item><list-item><p>Through analyzing three public scRNA-seq datasets, IReNA shows better performance on identifying known regulators than Rcistarget, the most widely used alternative method.</p></list-item></list></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469564', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Effects of the expression of random sequence clones on growth and transcriptome regulation in Escherichia coli', 'authors': \"[{'name': 'Devika Bhave'}, {'name': 'Diethard Tautz'}]\", 'abstract': '<p>Comparative genomic analyses have provided evidence that new genetic functions can emerge out of random nucleotide sequences. Here, we apply a direct experimental approach to study the effects of plasmids harbouring random sequence inserts under the control of an inducible promoter. Based on data from previously described experiments dealing with the growth of clones within whole libraries, we extracted specific clones that had shown either negative or positive effects on relative cell growth. We analysed these individually with respect to growth characteristics and impact on the transcriptome. We find that candidate clones for negative peptides lead to growth arrest by eliciting a general stress response. Overexpression of positive clones, on the other hand, does not change the exponential growth rates of hosts, but they show a growth advantage over a neutral candidate clone when tested in direct competition experiments. Transcriptomic changes in positive clones are relatively moderate and specific to each clone. We conclude from our experiments that random sequence peptides are indeed a suitable source for the <italic>de novo</italic> evolution of genetic functions.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469642', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Unambiguous detection of SARS-CoV-2 subgenomic mRNAs with single cell RNA sequencing', 'authors': \"[{'name': 'Phillip Cohen'}, {'name': 'Emma J DeGrace'}, {'name': 'Oded Danziger'}, {'name': 'Roosheel S Patel'}, {'name': 'Brad R Rosenberg'}]\", 'abstract': '<p>Single cell RNA sequencing (scRNAseq) studies have provided critical insight into the pathogenesis of Severe Acute Respiratory Syndrome CoronaVirus 2 (SARS-CoV-2), the causative agent of COronaVIrus Disease 2019 (COVID-19). scRNAseq workflows are generally designed for the detection and quantification of eukaryotic host mRNAs and not viral RNAs. The performance of different scRNAseq methods to study SARS-CoV-2 RNAs has not been thoroughly evaluated. Here, we compare different scRNAseq methods for their ability to quantify and detect SARS-CoV-2 RNAs with a focus on subgenomic mRNAs (sgmRNAs), which are produced only during active viral replication and not present in viral particles. We present a data processing strategy, single cell CoronaVirus sequencing (scCoVseq), which quantifies reads unambiguously assigned to sgmRNAs or genomic RNA (gRNA). Compared to standard 10X Genomics Chromium Next GEM Single Cell 3 (10X 3) and Chromium Next GEM Single Cell V(D)J (10X 5) sequencing, we find that 10X 5 with an extended R1 sequencing strategy maximizes the unambiguous detection of sgmRNAs by increasing the number of reads spanning leader-sgmRNA junction sites. Differential gene expression testing and KEGG enrichment analysis of infected cells compared with bystander or mock cells showed an enrichment for COVID19-associated genes, supporting the ability of our method to accurately identify infected cells. Our method allows for quantification of coronavirus sgmRNA expression at single-cell resolution, and thereby supports high resolution studies of the dynamics of coronavirus RNA synthesis.</p><sec><title>Importance</title><p>Single cell RNA sequencing (scRNAseq) has emerged as a valuable tool to study host-viral interactions particularly in the context of COronaVIrus Disease-2019 (COVID-19). scRNAseq has been developed and optimized for analyzing eukaryotic mRNAs, and the ability of scRNAseq to measure RNAs produced by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) has not been fully characterized. Here we compare the performance of different scRNAseq methods to detect and quantify SARS-CoV-2 RNAs and develop an analysis workflow to specifically quantify unambiguous reads derived from SARS-CoV-2 genomic RNA and subgenomic mRNAs. Our work demonstrates the strengths and limitations of scRNAseq to measure SARS-CoV-2 RNA and identifies experimental and analytical approaches that allow for SARS-CoV-2 RNA detection and quantification. These developments will allow for studies of coronavirus RNA biogenesis at single-cell resolution to improve our understanding of viral pathogenesis.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.25.470047', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Development of a novel glycoengineering platform for the rapid production of conjugate vaccines', 'authors': \"[{'name': 'Sherif Abouelhadid'}, {'name': 'Elizabeth Atkins'}, {'name': 'Emily Kay'}, {'name': 'Ian Passmore'}, {'name': 'Simon J North'}, {'name': 'Burhan Lehri'}, {'name': 'Paul Hitchen'}, {'name': 'Eirik Bakke'}, {'name': 'Mohammed Rahman'}, {'name': 'Janine Bosse'}, {'name': 'Yanwen Li'}, {'name': 'Vanessa S. Terra'}, {'name': 'Paul Langford'}, {'name': 'Anne Dell'}, {'name': 'Brendan W Wren'}, {'name': 'Jon Cuccui'}]\", 'abstract': '<p>Antimicrobial resistance (AMR) is threatening the lives of millions worldwide. Antibiotics which once saved countless lives, are now failing, ushering in vaccines development as a current global imperative. Conjugate vaccines produced either by chemical synthesis or biologically in <italic>Escherichia coli</italic> cells, have been demonstrated to be safe and efficacious in protection against several deadly bacterial diseases. However, conjugate vaccines assembly and production have several shortcomings which hinders their wider availability. Here, we developed a tool, Mobile-element Assisted Glycoconjugation by Insertion on Chromosome, MAGIC, a novel method that overcomes the limitations of the current conjugate vaccine design method(s). We demonstrate at least 2-fold increase in glycoconjugate yield via MAGIC when compared to conventional bioconjugate method(s). Furthermore, the modularity of the MAGIC platform also allowed us to perform glycoengineering in genetically intractable bacterial species other than <italic>E. coli</italic>. The MAGIC system promises a rapid, robust and versatile method to develop vaccines against bacteria, especially AMR pathogens, and could be applied for biopreparedness.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.26.470094', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Chloroplast transit peptides often require downstream unstructured sequence in Chlamydomonas reinhardtii', 'authors': \"[{'name': 'Oliver D Caspari'}]\", 'abstract': '<p>The N-terminal sequence stretch that defines subcellular targeting for most nuclear encoded chloroplast proteins is usually considered identical to the sequence that is cleaved upon import. Yet here this study shows that for nine out of ten tested Chlamydomonas chloroplast transit peptides, additional sequence past the cleavage site is required to enable chloroplast targeting. Using replacements of native post-cleavage residues with alternative sequences points to a role for unstructured sequence at mature protein N-termini.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.25.470051', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Genetic perturbation enhances functional heterogeneity in alkaline phosphatase', 'authors': \"[{'name': 'Morito Sakuma'}, {'name': 'Shingo Honda'}, {'name': 'Hiroshi Ueno'}, {'name': 'Kentaro Miyazaki'}, {'name': 'Nobuhiko Tokuriki'}, {'name': 'Hiroyuki Noji'}]\", 'abstract': '<p>Enzymes inherently exhibit molecule-to-molecule heterogeneity in catalytic activity or function, which underlies the acquisition of new functions in evolutionary processes. However, correlations between the functional heterogeneity of an enzyme and its multi-functionality or promiscuity remain elusive. In addition, the modulation of functional heterogeneity upon genetic perturbation is currently unexplored. Here, we quantitatively analyzed functional heterogeneity in the wild-type and 69 single-point mutants of <italic>Escherichia coli</italic> alkaline phosphatase (AP) by employing single-molecule assay with a femtoliter reactor array device. Most mutant enzymes exhibited higher functional heterogeneity than the wild-type enzyme, irrespective of catalytic activity. These results indicated that the wild-type AP minimizes functional heterogeneity, and single-point mutations can significantly expand the span of functional heterogeneity in AP. Moreover, we identified a clear correlation between functional heterogeneity and promiscuous activities. These findings suggest that enzymes can acquire greater functional heterogeneity following marginal genetic perturbations that concomitantly promote catalytic promiscuity.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.23.21266766', 'tweets': 13, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Vaccination strategies in structured populations under partial immunity and reinfection', 'authors': \"[{'name': 'Gabriel Rodriguez-Maroto'}, {'name': 'Iker Atienza-Diez'}, {'name': 'Sal Ares'}, {'name': 'Susanna Manrubia'}]\", 'abstract': '<p>Optimal protocols of vaccine administration to minimize the effects of infectious diseases depend on a number of variables that admit different degrees of control. Examples include the characteristics of the disease and how it impacts on different groups of individuals as a function of sex, age or socioeconomic status, its transmission mode, or the demographic structure of the affected population. Here we introduce a compartmental model of infection propagation with vaccination and reinfection and analyse the effect that variations on the rates of these two processes have on the progression of the disease and on the number of fatalities. The population is split into two groups to highlight the overall effects on disease caused by different relationships between vaccine administration and various demographic structures. We show that optimal administration protocols depend on the vaccination rate, a variable severely conditioned by vaccine supply and acceptance. As a practical example, we study COVID-19 dynamics in various countries using real demographic data. The model can be easily applied to any other disease and demographic structure through a suitable estimation of parameter values. Simulations of the general model can be carried out at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://mybinder.org/v2/gh/IkerAtienza/SIYRD/main?urlpath=%2Fvoila%2Frender%2FSimulator.ipynb\">this interactive webpage</ext-link> [1].</p><sec><title>Author summary</title><p>Vaccination campaigns can have varying degrees of success in minimizing the effects of an infectious disease. It is often very difficult to assess <italic>a priori</italic> the importance and effect of different relevant factors. To gain insight into this problem, we present a model of infection propagation with vaccination and use it to study the effects of vaccination rate and population structure. We find that when the disease affects in different ways distinct population groups, the best vaccination strategy depends non-trivially on the rate at which vaccines can be administered. The application of our analysis to COVID-19 reveals that, in countries with aged populations, the best strategy is always to vaccinate first the elderly, while for youthful populations maximizing vaccination rate regardless of other considerations may save more lives.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.21266734', 'tweets': 13, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Establishing and characterising large COVID-19 cohorts after mapping the Information System for Research in Primary Care in Catalonia to the OMOP Common Data Model', 'authors': \"[{'name': 'Edward Burn'}, {'name': 'Sergio Fernndez-Bertoln'}, {'name': 'Erica A Voss'}, {'name': 'Clair Blacketer'}, {'name': 'Maria Aragn'}, {'name': 'Martina Recalde'}, {'name': 'Elena Roel'}, {'name': 'Andrea Pistillo'}, {'name': 'Berta Ravents'}, {'name': 'Carlen Reyes'}, {'name': 'Sebastiaan van Sandijk'}, {'name': 'Lars Halvorsen'}, {'name': 'Peter R Rijnbeek'}, {'name': 'Talita Duarte-Salles'}]\", 'abstract': '<sec><title>Background</title><p>Few datasets have been established that capture the full breadth of COVID-19 patient interactions with a health system. Our first objective was to create a COVID-19 dataset that linked primary care data to COVID-19 testing, hospitalisation, and mortality data at a patient level. Our second objective was to provide a descriptive analysis of COVID-19 outcomes among the general population and describe the characteristics of the affected individuals.</p></sec><sec><title>Methods</title><p>We mapped patient-level data from Catalonia, Spain, to the Observational Medical Outcomes Partnership (OMOP) Common Data Model (CDM). More than 3,000 data quality checks were performed to assess the readiness of the database for research. Subsequently, to summarise the COVID-19 population captured, we established a general population cohort as of the 1<sup>st</sup> March 2020 and identified outpatient COVID-19 diagnoses or positive test results for SARS-CoV-2, hospitalisations with COVID-19, and COVID-19 deaths during follow-up, which went up until 30<sup>th</sup> June 2021.</p></sec><sec><title>Findings</title><p>Mapping data to the OMOP CDM was performed and high data quality was observed. The mapped database was used to identify a total of 5,870,274 individuals, who were included in the general population cohort as of 1<sup>st</sup> March 2020. Over follow up, 604,472 had either an outpatient COVID-19 diagnosis or positive test result, 58,991 had a hospitalisation with COVID-19, 5,642 had an ICU admission with COVID-19, and 11,233 had a COVID-19 death. People who were hospitalised or died were more commonly older, male, and with more comorbidities. Those admitted to ICU with COVID-19 were generally younger and more often male than those hospitalised in general and those who died.</p></sec><sec><title>Interpretation</title><p>We have established a comprehensive dataset that captures COVID-19 diagnoses, test results, hospitalisations, and deaths in Catalonia, Spain. Extensive data checks have shown the data to be fit for use. From this dataset, a general population cohort of 5.9 million individuals was identified and their COVID-19 outcomes over time were described.</p></sec><sec><title>Funding</title><p>Generalitat de Catalunya and European Health Data and Evidence Network (EHDEN).</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469549', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Comparative genomics and pangenomics of vancomycin resistant and susceptible Enterococcus faecium from Irish hospitals across 20 years', 'authors': \"[{'name': 'R.J. Leigh'}, {'name': 'C. McKenna'}, {'name': 'R. McWade'}, {'name': 'B. Lynch'}, {'name': 'F. Walsh'}]\", 'abstract': '<p><italic>Enterococcus faecium</italic> has emerged as an important nosocomial pathogen, which is increasingly difficult to treat due to the genetic acquisition of vancomycin resistance. Ireland exhibits a recalcitrant vancomycin resistant bloodstream infection rate compared to other developed countries. A set of 28 vancomycin resistant isolates was sequenced to construct a dataset alongside 61 other publicly available Irish genomes. This dataset was extensively analysed using <italic>in-silico</italic> methodologies and uncovered distinct evolutionary, coevolutionary, and clinically relevant population trends. These results suggest that a stable (in terms of genome size, GC%, and number of genes), yet genetically diverse population (in terms of gene content) of <italic>Enterococcus faecium</italic> persist in Ireland with acquired resistance arising <italic>via</italic> plasmid acquisition (<italic>vanA</italic>) or to a lesser extent, chromosomal recombination (<italic>vanB</italic>). Population analysis described five clusters with one cluster partitioned into four clades which transcend isolation dates. Pangenomic and recombination analyses revealed an open (whole genome and chromosomal specific) pangenome illustrating a rampant evolutionary pattern. Comparative resistomics and virulomics uncovered distinct chromosomal and mobilomal propensity for multidrug resistance, widespread chromosomal point-mutation mediated resistance, and chromosomal harboured arsenals of virulence factors. Comparative phagomics revealed a core prophagome of three prophages throughout the dataset. Interestingly, a potential difference in biofilm formation strategies was highlighted by coevolutionary analysis, suggesting differential biofilm genotypes between <italic>vanA</italic> and <italic>vanB</italic> isolates. These results highlight the evolutionary history of Irish <italic>Enterococcus faecium</italic> isolates and may provide an insight into underlying infection dynamics in a clinical setting.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469585', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Improving behavior monitoring of free moving dairy cows using non invasive wireless EEG approach and digital signal processing techniques', 'authors': \"[{'name': 'Ana Carolina de Sousa Silva'}, {'name': 'Aldo Ivan Cspedes Arce'}, {'name': 'Hubert Luzdemio Arteaga Miano'}, {'name': 'Gustavo Voltani von Atzingen'}, {'name': 'Valeria Cristina Rodrigues Sarnighausen'}, {'name': 'Ernane Jos Xavier Costa'}]\", 'abstract': '<sec><title>Background</title><p>Electroencephalography (EEG) is the most common method to access brain information. Techniques to monitor and to extract brain signal characteristics in farm animals are not as developed as in humans and laboratory animals.</p></sec><sec><title>New method</title><p>The method comprised two steps. In the first step, the signals were acquired after the telemetric equipment was developed, the electrodes were positioned and fixed, the sample frequency was defined, the equipment was positioned, and artifacts and other acquisition problems were dealt with. Brain signals from six Holstein heifers that could move freely in free stalls were acquired. The control group consisted in the same number of bovines, contained in a climatic chamber (restrained group). In the second step, the signals were characterized by Power Spectral Density, Short-Time Fourier Transform, and Lempel-Ziv complexity.</p></sec><sec><title>Results</title><p>The results indicated that there was an ideal position to attach the electrodes to the front of the bovines head, so that longer artifact-free signal sections were acquired. The signals showed typical EEG frequency bands, like the bands found in humans. The Lempel-Ziv complexity values indicated that the bovine brain signals contained random and chaotic components. As expected, the signals acquired from the retained bovine group displayed sections with a larger number of artifacts.</p></sec><sec><title>Comparison with existing methods</title><p>We present the first method that helps to monitor and to extract brain signal features in unrestrained bovines.</p></sec><sec><title>Conclusions</title><p>The method could be applied to investigate changes in brain electrical activity during animal farming, to monitor brain activity related with animal behavior.</p></sec><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>A method that allows brain signals to be monitored in freely moving dairy cows is described</p></list-item><list-item><p>The method uses noninvasive electrodes to minimize stress during EEG monitoring and allows bovines to behave normally during the process</p></list-item><list-item><p>The method establishes the frequency sampling rate, electrodes positioning and fixation, equipment holding, artifact extraction, and signal characterization</p></list-item><list-item><p>The brain signals are characterized by PSD, STFT, and Lempel-Ziv normalized complexity</p></list-item><list-item><p>The method can be applied to relate EEG to animal behavior under normal handling conditions</p></list-item></list></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.24.469855', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Fibroblastic reticular cells provide a supportive niche for lymph node-resident macrophages', 'authors': \"[{'name': 'Joshua DRozario'}, {'name': 'Konstantin Knoblich'}, {'name': 'Mechthild Ltge'}, {'name': 'Christian Prez Shibayama'}, {'name': 'Hung-Wei Cheng'}, {'name': 'Yannick O. Alexandre'}, {'name': 'David Roberts'}, {'name': 'Joana Campos'}, {'name': 'Jillian Astarita'}, {'name': 'Emma Dutton'}, {'name': 'Muath Suliman'}, {'name': 'Alice E. Denton'}, {'name': 'Shannon J. Turley'}, {'name': 'Richard L. Boyd'}, {'name': 'Scott Mueller'}, {'name': 'Burkhard Ludewig'}, {'name': 'Tracy Heng'}, {'name': 'Anne L Fletcher'}]\", 'abstract': '<p>The lymph node (LN) is home to resident macrophage populations that are essential for immune function and homeostasis. The T cell paracortical zone is a major site of macrophage efferocytosis of apoptotic cells, but key factors controlling this niche are undefined. Here we show that fibroblastic reticular cells (FRCs) are an essential component of the LN macrophage niche. Macrophages co-localised with FRCs in human LNs, and murine single-cell RNA-sequencing revealed that most reticular cells expressed master macrophage regulator CSF1. Functional assays showed that CSF1R signalling was sufficient to support macrophage development. In the presence of LPS, FRCs underwent a mechanistic switch and maintained support through CSF1R-independent mechanisms. These effects were conserved between mouse and human systems. Rapid loss of macrophages and monocytes from LNs was observed upon genetic ablation of FRCs. These data reveal a critically important role for FRCs in the creation of the parenchymal macrophage niche within LNs.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266681', 'tweets': 13, 'archive': 'medRxiv', 'subject-area': 'Allergy and Immunology', 'covid': True, 'title': 'In vitro, classical complement activation differs by disease severity and between SARS-CoV-2 antigens', 'authors': \"[{'name': 'Rachel E Lamerton'}, {'name': 'Edith Marcial Juarez'}, {'name': 'Sian E Faustini'}, {'name': 'Marisol E Perez-Toledo'}, {'name': 'Margaret Goodall'}, {'name': 'Sian E Jossi'}, {'name': 'Maddy L Newby'}, {'name': 'Iain Chapple'}, {'name': 'Thomas Dietrich'}, {'name': 'Tonny Veenith'}, {'name': 'Adrian M Shields'}, {'name': 'Lorraine Harper'}, {'name': 'Ian R Henderson'}, {'name': 'Julie Rayes'}, {'name': 'David C Wraith'}, {'name': 'Steve P Watson'}, {'name': 'Max Crispin'}, {'name': 'Mark T Drayson'}, {'name': 'Alex G Richter'}, {'name': 'Adam F Cunningham'}]\", 'abstract': '<p>Antibodies specific for the spike glycoprotein (S) and nucleocapsid (N) SARS-CoV-2 proteins are typically present during severe COVID-19, and induced to S after vaccination. The binding of viral antigens by antibody can initiate the classical complement pathway. Since complement could play pathological or protective roles at distinct times during SARS-CoV-2 infection we determined levels of antibody-dependent complement activation along the complement cascade. Here, we used an ELISA assay to assess complement protein binding (C1q) and the deposition of C4b, C3b, and C5b to S and N antigens in the presence of anti-SARS-CoV-2 antibodies from different test groups: non-infected, single and double vaccinees, non-hospitalised convalescent (NHC) COVID-19 patients and convalescent hospitalised (ITU-CONV) COVID-19 patients. C1q binding correlates strongly with antibody responses, especially IgG1 levels. However, detection of downstream complement components, C4b, C3b and C5b shows some variability associated with the antigen and subjects studied. In the ITU-CONV, detection of C3b-C5b to S was observed consistently, but this was not the case in the NHC group. This is in contrast to responses to N, where median levels of complement deposition did not differ between the NHC and ITU-CONV groups. Moreover, for S but not N, downstream complement components were only detected in sera with higher IgG1 levels. Therefore, the classical pathway is activated by antibodies to multiple SARS-CoV-2 antigens, but the downstream effects of this activation may differ depending on the specific antigen targeted and the disease status of the subject.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469619', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Thermodynamic Constraints on Electromicrobial Protein Production', 'authors': \"[{'name': 'Lucas Wise'}, {'name': 'Sabrina Marecos'}, {'name': 'Katie Randolph'}, {'name': 'Eric Nshimyumukiza'}, {'name': 'Mohamed Hassan'}, {'name': 'Jacob Strouse'}, {'name': 'Farshid Salimijazi'}, {'name': 'Buz Barstow'}]\", 'abstract': '<p>Global consumption of protein is projected to double by the middle of the 21<sup>st</sup> century<sup>1</sup>. However, protein production is one of the most energy intensive and environmentally damaging parts of the food supply system today. Electromicrobial production technologies that combine renewable electricity and CO<sub>2</sub>-fixing microbial metabolism could dramatically increase the energy efficiency of commodity chemical production<sup>25</sup>. Here we present a molecular-scale model that sets an upper limit on the performance of any organism performing electromicrobial protein production. We show that engineered microbes that fix CO<sub>2</sub> and N<sub>2</sub> using reducing equivalents produced by H<sub>2</sub>-oxidation or extracellular electron uptake could produce amino acids with energy inputs as low as 64 MJ kg<sup>-1</sup>. This work provides a roadmap for development of engineered microbes that could significantly expand access to proteins produced with a low environmental footprint.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469714', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'LRRC15 is an inhibitory receptor blocking SARS-CoV-2 spike-mediated entry in trans', 'authors': \"[{'name': 'Jaewon Song'}, {'name': 'Ryan D. Chow'}, {'name': 'Mario Pena-Hernandez'}, {'name': 'Li Zhang'}, {'name': 'Skylar A. Loeb'}, {'name': 'Eui-Young So'}, {'name': 'Olin D. Liang'}, {'name': 'Craig B. Wilen'}, {'name': 'Sanghyun Lee'}]\", 'abstract': '<p>SARS-CoV-2 infection is mediated by the entry receptor ACE2. Although attachment factors and co-receptors facilitating entry are extensively studied, cellular entry factors inhibiting viral entry are largely unknown. Using a <italic>surface</italic>ome CRISPR activation screen, we identified human LRRC15 as an inhibitory receptor for SARS-CoV-2 entry. LRRC15 directly binds to the receptor-binding domain (RBD) of spike protein with a moderate affinity and inhibits spike-mediated entry. Analysis of human lung single cell RNA sequencing dataset reveals that expression of LRRC15 is primarily detected in fibroblasts and particularly enriched in pathological fibroblasts in COVID-19 patients. <italic>ACE2</italic> and <italic>LRRC15</italic> are not co-expressed in the same cell types in the lung. Strikingly, expression of LRRC15 in ACE2-negative cells blocks spike-mediated viral entry in ACE2+ cell <italic>in trans</italic>, suggesting a protective role of LRRC15 in a physiological context. Therefore, LRRC15 represents an inhibitory receptor for SARS-CoV-2 regulating viral entry <italic>in trans</italic>.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469725', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Fecal microbiota transplantation derived from Alzheimers disease mice worsens brain trauma outcomes in young C57BL/6 mice', 'authors': \"[{'name': 'Sirena Soriano'}, {'name': 'Kristen Curry'}, {'name': 'Qi Wang'}, {'name': 'Elsbeth Chow'}, {'name': 'Todd Treangen'}, {'name': 'Sonia Villapol'}]\", 'abstract': '<p>Traumatic brain injury (TBI) cause neuroinflammation, exaggerated immune response, and, consequently, neurodegeneration. The gut microbiome is an essential modulator of the immune system, impacting in the brain. There are not effective treatments for TBI, therefore, modulating the gut microbiome may shed novel therapeutics for the damaged brain. Also, in patients with Alzheimers disease (AD), the microbiota has been associated with a lack of diversity, which negatively modulates the immune system. This study aimed to determine whether the gut microbiota from AD mice exacerbates neurological deficits after TBI in young mice. For this purpose, we performed fecal microbiota transplants from AD (FMT-AD) mice into young C57BL/6 (wild-type, WT) mice following TBI. Thus, FMT-AD and fecal microbiota transplants from healthy controls (FMT-young) were administered orally to young WT mice after the TBI occurred. We first determined the gut microbiota diversity and composition by analyzing fulllength 16S rRNA sequences from mouse fecal samples using the Oxford Nanopore MinION technology. We collected the blood, brain, and gut tissues for protein and immunohistochemical analysis. Our results showed that FMT-AD treatment stimulates a higher relative abundance of <italic>Muribaculum intestinal</italic> and a decrease in <italic>Lactobacillus johnsonii</italic> compared FMT-young treatment in WT mice. Furthermore, WT mice exhibited larger lesion volumes, increased the number of activated microglia/macrophages cells, and reduced motor recovery after FMT-AD compared to FMT-young one day after TBI. Thus, the gut microbiota from AD mice not only aggravates the neuroinflammatory response and motor recovery, but also increases the lesion size after TBI in young WT mice.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.21.469456', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'NetrinG1+ cancer-associated fibroblasts generate unique extracellular vesicles that support the survival of pancreatic cancer cells under nutritional stress', 'authors': \"[{'name': 'Kristopher S. Raghavan'}, {'name': 'Ralph Francescone'}, {'name': 'Janusz Franco-Barraza'}, {'name': 'Jaye C. Gardiner'}, {'name': 'Dbora Barbosa Vendramini-Costa'}, {'name': 'Tiffany Luong'}, {'name': 'Narges Pourmandi'}, {'name': 'Anthony Andren'}, {'name': 'Alison Kurimchak'}, {'name': 'Charline Ogier'}, {'name': 'James S. Duncan'}, {'name': 'Costas A. Lyssiotis'}, {'name': 'Lucia R. Languino'}, {'name': 'Edna Cukierman'}]\", 'abstract': '<p>It is projected that, in 5 years, pancreatic cancer will become the second deadliest cancer in the United States. A unique aspect of pancreatic ductal adenocarcinoma (PDAC) is its stroma; rich in cancer-associated fibroblasts (CAFs) and a dense CAF-generated extracellular matrix (ECM). This fibrous stroma, known as desmoplasia, causes the collapse of local blood vessels rendering a nutrient-deprived milieu. Hence, PDAC cells are nurtured by local CAF-secreted products, which include, among others, CAF-generated small extracellular vesicles (sEVs). It is well-accepted that upon culturing functionally tumor-promoting CAFs under pathophysiological-relevant conditions (e.g., within self-produced ECM), these cells express NetrinG1 (NetG1) and sustain endosomal pools rich in active 51-integrin; traits indicative of poor patient survival. We herein report that NetG1<sup>+</sup> CAFs generate sEVs that rescue PDAC cells from nutrient-deprived induced apoptosis. Two unique sEVs, NetG1<sup>+</sup> and 51-integrin<sup>+</sup>, were uncovered. The former constitutes cargo of CAF-generated exomeres, and the latter is detected in classic exosomes. Proteomic and metabolomic analyses showed that the sEV-dependent PDAC survival is, at least in part, dictated by the cargo packaged within sEVs in a NetG1-dependent manner. Indeed, despite producing a similar number of vesicles, selected key proteins and metabolites (e.g., glutamine) were incorporated within the unique sEVs. Finally, we found that NetG1 and 51-integrin were detected in sEVs collected from plasma of PDAC patients, while their concomitant levels were significantly lower in plasma of sex/age-matched healthy donors. The discovery of these tumor-supporting CAF sEVs opens a new investigative avenue in tumor-stroma interactions and stroma staging detection.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266512', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Association of COVID-19 with arterial and venous vascular diseases: a population-wide cohort study of 48 million adults in England and Wales', 'authors': \"[{'name': 'Rochelle Knight'}, {'name': 'Venexia Walker'}, {'name': 'Samantha Ip'}, {'name': 'Jennifer A Cooper'}, {'name': 'Thomas Bolton'}, {'name': 'Spencer Keene'}, {'name': 'Rachel Denholm'}, {'name': 'Ashley Akbari'}, {'name': 'Hoda Abbasizanjani'}, {'name': 'Fatemeh Torabi'}, {'name': 'Efosa Omigie'}, {'name': 'Sam Hollings'}, {'name': 'Teri-Louise North'}, {'name': 'Renin Toms'}, {'name': 'Emanuele Di Angelantonio'}, {'name': 'Spiros Denaxas'}, {'name': 'Johan H Thygesen'}, {'name': 'Christopher Tomlinson'}, {'name': 'Ben Bray'}, {'name': 'Craig J Smith'}, {'name': 'Mark Barber'}, {'name': 'George Davey Smith'}, {'name': 'Nishi Chaturvedi'}, {'name': 'Cathie Sudlow'}, {'name': 'William N Whiteley'}, {'name': 'Angela Wood'}, {'name': 'Jonathan A C Sterne'}, {'name': ' '}]\", 'abstract': '<sec><title>Importance</title><p>The long-term effects of COVID-19 on the incidence of vascular diseases are unclear.</p></sec><sec><title>Objective</title><p>To quantify the association between time since diagnosis of COVID-19 and vascular disease, overall and by age, sex, ethnicity, and pre-existing disease.</p></sec><sec><title>Design</title><p>Cohort study based on population-wide linked electronic health records, with follow up from January 1<sup>st</sup> to December 7<sup>th</sup> 2020.</p></sec><sec><title>Setting and participants</title><p>Adults registered with an NHS general practice in England or Wales and alive on January 1<sup>st</sup> 2020.</p></sec><sec><title>Exposures</title><p>Time since diagnosis of COVID-19 (categorised as 0-6 days, 1-2 weeks, 3-4, 5-8, 9-12, 13-26 and 27-49 weeks since diagnosis), with and without hospitalisation within 28 days of diagnosis.</p></sec><sec><title>Main outcomes and measures</title><p>Primary outcomes were arterial thromboses (mainly acute myocardial infarction and ischaemic stroke) and venous thromboembolic events (VTE, mainly pulmonary embolism and lower limb deep vein thrombosis). We also studied other vascular events (transient ischaemic attack, haemorrhagic stroke, heart failure and angina). Hazard ratios were adjusted for demographic characteristics, previous disease diagnoses, comorbidities and medications.</p></sec><sec><title>Results</title><p>Among 48 million adults, 130,930 were and 1,315,471 were not hospitalised within 28 days of COVID-19. In England, there were 259,742 first arterial thromboses and 60,066 first VTE during 41.6 million person-years follow-up. Adjusted hazard ratios (aHRs) for first arterial thrombosis compared with no COVID-19 declined rapidly from 21.7 (95% CI 21.0-22.4) to 3.87 (3.58-4.19) in weeks 1 and 2 after COVID-19, 2.80 (2.61-3.01) during weeks 3-4 then to 1.34 (1.21-1.48) during weeks 27-49. aHRs for first VTE declined from 33.2 (31.3-35.2) and 8.52 (7.59-9.58) in weeks 1 and 2 to 7.95 (7.28-8.68) and 4.26 (3.86-4.69) during weeks 3-4 and 5-8, then 2.20 (1.99-2.44) and 1.80 (1.50-2.17) during weeks 13-26 and 27-49 respectively. aHRs were higher, for longer after diagnosis, after hospitalised than non-hospitalised COVID-19. aHRs were also higher among people of Black and Asian than White ethnicity and among people without than with a previous event. Across the whole population estimated increases in risk of arterial thromboses and VTEs were 2.5% and 0.6% respectively 49 weeks after COVID-19, corresponding to 7,197 and 3,517 additional events respectively after 1.4 million COVID-19 diagnoses.</p></sec><sec><title>Conclusions and Relevance</title><p>High rates of vascular disease early after COVID-19 diagnosis decline more rapidly for arterial thromboses than VTEs but rates remain elevated up to 49 weeks after COVID_19. These results support continued policies to avoid COVID-19 infection with effective COVID-19 vaccines and use of secondary preventive agents in high-risk patients.</p></sec><sec><title>Key points</title><sec><title>Question</title><p>Is COVID-19 associated with higher long-term incidence of vascular diseases?</p></sec><sec><title>Findings</title><p>In this cohort study of 48 million adults in England and Wales, COVID-19 was associated with higher incidence, that declined with time since diagnosis, of both arterial thromboses [week 1: adjusted HR [aHR] 21.7 (95% CI 21.0-22.4) weeks 27-49: aHR 1.34 (1.21-1.48)] and venous thromboembolism [week 1: aHR 33.2 (31.3-35.2), weeks 2749 1.80 (1.50-2.17)]. aHRs were higher, for longer, after hospitalised than non-hospitalised COVID-19. The estimated excess number of arterial thromboses and venous thromboembolisms was 10,500.</p></sec><sec><title>Meaning</title><p>Avoidance of COVID-19 infection through vaccination, and use of secondary preventive agents after infection in high-risk patients, may reduce post-COVID-19 acute vascular diseases.</p></sec></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469129', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Fast surveillance response and genome sequencing reveal the circulation of a new Yellow Fever Virus sublineage in 2021, in Minas Gerais, Brazil', 'authors': \"[{'name': 'Miguel S. Andrade'}, {'name': 'Fabrcio S. Campos'}, {'name': 'Cirilo H. de Oliveira'}, {'name': 'Ramon Silva Oliveira'}, {'name': 'Aline A. S. Campos'}, {'name': 'Marco A. B. Almeida'}, {'name': 'Danilo Simonini-Teixeira'}, {'name': 'Anai da P. Sev'}, {'name': 'Andrea Oliveira Dias Temponi'}, {'name': 'Fernando Maria Magalhes'}, {'name': 'Danielle Costa Capistrano Chaves'}, {'name': 'Maira Alves Pereira'}, {'name': 'Ludmila Oliveira Lamounier'}, {'name': 'Givaldo Gomes de Menezes'}, {'name': 'Sandy Micaele Aquino Teixeira'}, {'name': 'Maria Eduarda Gonalves dos Santos'}, {'name': 'Sofa Bernal-Valle'}, {'name': 'Nicolas F. D. Mller'}, {'name': 'Jader da C. Cardoso'}, {'name': 'Edmilson dos Santos'}, {'name': 'Maria A. Mares-Guia'}, {'name': 'George R. Albuquerque'}, {'name': 'Alessandro P. M. Romano'}, {'name': 'Ana C. Franco'}, {'name': 'Bergmann M. Ribeiro'}, {'name': 'Paulo M. Roehe'}, {'name': 'Filipe V. S. Abreu'}]\", 'abstract': '<p>Yellow fever virus (YFV) exhibits a sylvatic cycle of transmission involving wild mosquitoes and non-human primates (NHP). In Brazil, YFV is endemic in the Amazon region, from where waves of epidemic expansion towards other Brazilian states eventually occur. During such waves, the virus usually follows the route from North to the Central-West and Southeast Brazilian regions. Amidst these journeys, outbreaks of Yellow Fever (YF) in NHPs, with spillovers to humans have been observed. In the present work, we describe a surveillance effort encompassing the technology of smartphone applications and the coordinated action of several research institutions and health services that succeeded in the first confirmation of YFV in NHPs in the state of Minas Gerais (MG), Southeast region, in 2021, followed by genome sequencing in an interval of only ten days. Samples from two NHPs (one of the species <italic>Alouatta caraya</italic> in the municipality of Icara de Minas and the other of the species <italic>Callithrix penicillata</italic> in the municipality of Uba) were collected and the presence of YFV was confirmed by RT-qPCR. We generated three near-complete by Nanopore sequencer MinION. Phylogenetic analysis revealed that all viral genomes recovered are equal and related to lineage South America 1, clustering with a genome detected in the Amazon region (Par state) in 2017. These findings reveal the occurrence of a new wave of viral expansion in MG, six years after the beginning of the major outbreak in the state, between 2015-2018. No human cases were reported to date, showing the importance of coordinated work between local surveillance based on available technologies and support laboratories to ensure a quick response and implementation of contingency measures towards avoiding the occurrence of YF cases in humans.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.26.470088', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evolution of Human-specific Alleles Protecting Cognitive Function of Grandmothers', 'authors': \"[{'name': 'Sudeshna Saha'}, {'name': 'Naazneen Khan'}, {'name': 'Troy Comi'}, {'name': 'Andrea Verhagen'}, {'name': 'Aniruddha Sasmal'}, {'name': 'Sandra Diaz'}, {'name': 'Hai Yu'}, {'name': 'Xi Chen'}, {'name': 'Joshua M. Akey'}, {'name': 'Martin Frank'}, {'name': 'Pascal Gagneux'}, {'name': 'Ajit Varki'}]\", 'abstract': '<p>Late-onset Alzheimers Disease (LOAD) pathology is rare in our closest living evolutionary relatives (chimpanzees), which also express much lower microglial levels of CD33(Siglec-3), a myelomonocytic receptor inhibiting innate immune reactivity by extracellular V-set domain recognition of sialic acid(Sia)-containing self-associated molecular patterns (SAMPs). We earlier showed that V-set domain-deficient CD33-variant allele, protective against LOAD, is derived and specific to hominin-lineage. We now report that CD33 also harbors multiple hominin-specific V-set domain mutations and explore selection forces that may have favored such genomic changes. N-glycolylneuraminic acid (Neu5Gc), the preferred Sia-ligand of ancestral CD33 is absent in humans, due to hominin-specific, fixed loss-of-function mutation in CMAH, which generates CMP-Neu5Gc from its precursor, CMP-N-acetylneuraminic acid (Neu5Ac). Extensive mutational analysis and MD-simulations indicate that fixed change in amino acid 21 of hominin V-set domain and conformational changes related to His45 corrected for Neu5Gc-loss by switching to Neu5Ac-recognition. Considering immune-evasive molecular mimicry of SAMPs by pathogens, we found that human-specific pathogens <italic>Neisseria gonorrhoeae</italic> and Group B <italic>Streptococcus</italic> (affecting fertility and fetuses/neonates respectively) selectively bind huCD33 and this binding is significantly impacted by amino acid 21 modification. Alongside LOAD-protective CD33 alleles, humans harbor additional, derived, population-universal, cognition-protective variants absent in great ape genomes. Interestingly, 11 of 13 SNPs in these human genes (including CD33), that protect the cognitive health of elderly populations, are not shared by genomes of archaic hominins: Neanderthals and Denisovans. Finally, we present a plausible evolutionary scenario to compile, correlate and comprehend existing knowledge about huCD33 evolution and suggest that grandmothering emerged in humans.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.25.470062', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Spatial resolution of an integrated C4+CAM photosynthetic metabolism', 'authors': \"[{'name': 'Jose J Moreno-Villena'}, {'name': 'Haoran Zhou'}, {'name': 'Ian S Gilman'}, {'name': 'S. Lori Tausta'}, {'name': 'C. Y. Maurice Cheung'}]\", 'abstract': '<p>C<sub>4</sub> and CAM photosynthesis have repeatedly evolved in plants over the past 30 million years. Because both repurpose the same set of enzymes but differ in their spatial and temporal deployment, they have long been considered as distinct and incompatible adaptations. Remarkably, Portulaca contains multiple C<sub>4</sub> species that perform CAM when droughted. Spatially explicit analyses of gene expression reveal that C<sub>4</sub> and CAM systems are completely integrated in P. oleracea, with CAM and C<sub>4</sub> carbon fixation occurring in the same cells and CAM-generated metabolites likely incorporated directly into the C<sub>4</sub> cycle. Flux balance analysis corroborates the gene expression and predicts an integrated C<sub>4</sub>+CAM system under drought. This first spatially explicit description of a C<sub>4</sub>+CAM photosynthetic metabolism presents a new blueprint for crop improvement.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.24.469816', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genome sequences of Arthrobacter spp. that use a modified sulfoglycolytic Embden-Meyerhof-Parnas pathway', 'authors': \"[{'name': 'Arashdeep Kaur'}, {'name': 'Phillip L. van der Peet'}, {'name': 'Janice W.-Y. Mui'}, {'name': 'Marion Herisse'}, {'name': 'Sacha Pidot'}, {'name': 'Spencer J. Williams'}]\", 'abstract': '<sec><title>Background</title><p>Sulfoglycolysis pathways enable the breakdown of the sulfosugar sulfoquinovose and environmental recycling of its carbon and sulfur content. Several pathways exist for the breakdown of sulfoquinovose that usually lead to production of C3-sulfonates (sulfolactate and 2,3-dihydroxypropanesulfonate) that are excreted and sustain secondary bacterial communities. The prototypical sulfoglycolytic pathway is a variant of the classical Embden-Meyerhof-Parnas pathway that has been described in gram-negative <italic>Escherichia coli</italic> and results in production of 2,3-dihydroxypropanesulfonate.</p></sec><sec><title>Results</title><p>We used enrichment cultures to discover new sulfoglycolytic bacteria from Australian soil samples. Two gram-positive <italic>Arthrobacter</italic> spp. were isolated that produced sulfolactate as the metabolic end-product. Genome sequences identified a modified sulfoglycolytic Embden-Meyerhof-Parnas (sulfo-EMP) gene cluster that retained the core sulfoglycolysis genes encoding metabolic enzymes, but featuring the replacement of the gene encoding sulfolactaldehyde (SLA) reductase with SLA dehydrogenase, and the absence of sulfoquinovosidase and sulfoquinovose mutarotase genes. The gene clusters were broadly conserved across a range of other sequenced Actinobacteria.</p></sec><sec><title>Conclusions</title><p>We report the first gram-positive soil bacteria that utilize sulfo-EMP pathways to metabolize SQ. Excretion of sulfolactate is consistent with an aerobic saprophytic lifestyle. This work broadens our knowledge of the sulfo-EMP pathway to include soil bacteria.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469739', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Long-term potentiation at pyramidal cell to somatostatin interneuron synapses controls hippocampal network plasticity and memory', 'authors': \"[{'name': 'Azam Asgarihafshejani'}, {'name': 've Honor'}, {'name': 'Franois-Xavier Michon'}, {'name': 'Isabel Laplante'}, {'name': 'Jean-Claude Lacaille'}]\", 'abstract': '<p>Hippocampal somatostatin (SOM) cells are dendrite-projecting inhibitory interneurons. CA1 SOM cells receive major excitatory inputs from pyramidal cells (PC-SOM synapses) which show mGluR1a- and mTORC1-mediated long-term potentiation (LTP). PC-SOM synapse LTP contributes to CA1 network metaplasticity and memory consolidation, but whether it is sufficient to regulate these processes remains unknown. Here we used optogenetic stimulation of CA1 pyramidal cells and whole cell recordings in slices to show that optogenetic theta burst stimulation (TBS<sub>opto</sub>) produces LTP at PC-SOM synapses. At the network level, we found that TBS<sub>opto</sub> differentially regulates metaplasticity of pyramidal cell inputs: enhancing LTP at Schaffer collateral synapses and depressing LTP at temporo-ammonic synapses. At the behavioral level, we uncovered that in vivo TBS<sub>opto</sub> regulates learning-induced LTP at PC-SOM synapses, as well as contextual fear memory. Thus, LTP of PC-SOM synapses is a long-term feedback mechanism controlling pyramidal cell synaptic plasticity, sufficient to regulate memory consolidation.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469570', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Polyelectrolyte Nanocomplex Formation Combined with Electrostatic Self-Assembly Enables the Co-Delivery of Synergistic Antimicrobials to Treat Bacterial Biofilms', 'authors': \"[{'name': 'Joel A. Finbloom'}, {'name': 'Preethi Raghavan'}, {'name': 'Bhushan N. Kharbikar'}, {'name': 'Michelle A. Yu'}, {'name': 'Tejal A. Desai'}]\", 'abstract': '<p>New approaches are needed to treat bacterial biofilm infections, particularly those of <italic>Pseudomonas aeruginosa</italic> (PA). PA biofilms have high rates of antimicrobial resistance and are commonly found in chronic wound and cystic fibrosis lung infections. The use of combination therapeutics that act synergistically is a promising treatment strategy; however, the delivery of multiple therapeutics at relevant dosages is challenging in the clinic. We therefore developed a new nanoscale drug carrier by combining approaches from both polyelectrolyte nanocomplex (NC) formation and layer-by-layer electrostatic self-assembly. This strategy led to NC drug carriers functionalized with both tobramycin antibiotics and antimicrobial silver nanoparticles with high loading efficiencies. AgTob-NCs displayed synergistic enhancements in antimicrobial activity against planktonic and biofilm PA cultures, with positively charged NCs demonstrating complete biofilm eradication. Overall, this approach shows promise in the co-delivery of diverse classes of antimicrobials to overcome antimicrobial resistance and treat bacterial biofilm infections.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469509', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'A computationally efficient clustering linear combination approach to jointly analyze multiple phenotypes for GWAS', 'authors': \"[{'name': 'Meida Wang'}, {'name': 'Shuanglin Zhang'}, {'name': 'Qiuying Sha'}]\", 'abstract': '<p>There has been an increasing interest in joint analysis of multiple phenotypes in genome-wide association studies (GWAS) because jointly analyzing multiple phenotypes may increase statistical power to detect genetic variants associated with complex diseases or traits. Recently, many statistical methods have been developed for joint analysis of multiple phenotypes in genetic association studies, including the Clustering Linear Combination (CLC) method. The CLC method works particularly well with phenotypes that have natural groupings, but due to the unknown number of clusters for a given data, the final test statistic of CLC method is the minimum p-value among all p-values of the CLC test statistics obtained from each possible number of clusters. Therefore, a simulation procedure must be used to evaluate the p-value of the final test statistic. This makes the CLC method computationally demanding. We develop a new method called computationally efficient CLC (ceCLC) to test the association between multiple phenotypes and a genetic variant. Instead of using the minimum p-value as the test statistic in the CLC method, ceCLC uses the Cauchy combination test to combine all p-values of the CLC test statistics obtained from each possible number of clusters. The test statistic of ceCLC approximately follows a standard Cauchy distribution, so the p-value can be obtained from the cumulative density function without the need for the simulation procedure. Through extensive simulation studies and application on the COPDGene data, the results demonstrate that the type I error rates of ceCLC are effectively controlled in different simulation settings and ceCLC either outperforms all other methods or has statistical power that is very close to the most powerful method with which it has been compared.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.25.469984', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'DeepSpot: a deep neural network for RNA spot enhancement in smFISH microscopy images', 'authors': \"[{'name': 'Emmanuel Bouilhol'}, {'name': 'Edgar Lefevre'}, {'name': 'Benjamin Dartigues'}, {'name': 'Robyn Brackin'}, {'name': 'Anca Flavia Savulescu'}, {'name': 'Macha Nikolski'}]\", 'abstract': '<p>Detection of RNA spots in single molecule FISH microscopy images remains a difficult task especially when applied to large volumes of data. The small size of RNA spots combined with high noise level of images often requires a manual adaptation of the spot detection thresholds for each image. In this work we introduce DeepSpot, a Deep Learning based tool specifically designed to enhance RNA spots which enables spot detection without need to resort to image per image parameter tuning. We show how our method can enable the downstream accurate detection of spots. The architecture of DeepSpot is inspired by small object detection approaches. It incorporates dilated convolutions into a module specifically designed for the Context Aggregation for Small Object (CASO) and uses Residual Convolutions to propagate this information along the network. This enables DeepSpot to enhance all RNA spots to the same intensity and thus circumvents the need for parameter tuning. We evaluated how easily spots can be detected in images enhanced by our method, by training DeepSpot on 20 simulated and 1 experimental datasets, and have shown that more than 97% accuracy is achieved. Moreover, comparison with alternative deep learning approaches for mRNA spot detection (deepBlink) indicated that DeepSpot allows more precise mRNA detection. In addition, we generated smFISH images from mouse fibroblasts in a wound healing assay to evaluate whether DeepSpot enhancement can enable seamless mRNA spot detection and thus streamline studies of localized mRNA expression in cells.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.22.21266522', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Health Economics', 'covid': True, 'title': 'A comparison of health care use after severe COVID-19, respiratory syncytial virus, and influenza in children', 'authors': \"[{'name': 'Vilde Bergstad Larsen'}, {'name': 'Ketil Strdal'}, {'name': 'Kjetil Telle'}, {'name': 'Fredrik Methi'}, {'name': 'Karin Magnusson'}]\", 'abstract': '<sec><title>Background</title><p>We aimed to explore whether children in hospital care with COVID-19 have increased post-discharge health care use when compared to children in hospital care with 1) RSV infection, and 2) other RTIs.</p></sec><sec><title>Methods</title><p>In 34,214 children aged 1 month to 5 years who were registered with one or more hospital contacts with COVID-19 (N=128), RSV infection (N=4,009) or other RTIs (upper- and lower unspecified RTI as well as influenza) (N=34,457) (January 1<sup>st</sup>, 2017 to September 20<sup>th</sup>, 2021), we used a pre-post study design to investigate the individual all-cause primary and specialist health care use from 12 weeks prior to hospital admission, to 12 weeks after hospital discharge, stratified on infants (1-12 months) and toddlers (1-5 years).</p></sec><sec><title>Findings</title><p>We found a slight increase in primary health care use in the first four weeks after discharge for children aged 1-12 months with COVID-19 when compared to children with RSV infection (0.064 percentage points, 95% CI 0.02-0.126, 0.52% relative increase). For children aged 1-5 years, COVID-19 discharge was associated with a 1-4 weeks increase in primary health care use when compared to children with RSV infection (0.068 percentage points, 95% CI 0.022-0.144, 0.53% relative increase) and other RTI (0.046 percentage points, 95% CI 0.002-0.091, 0.45% relative increase). For children aged 1-12 months in hospital care with COVID-19, we found a similar-magnitude-increase in post-discharge inpatient specialist care use, which lasted for 12 weeks.</p></sec><sec><title>Interpretation</title><p>Our findings imply no severe impact on health care use after hospitalization with COVID-19 compared to hospitalization with RSV infection or other RTIs. The etiological mechanisms for potentially worse post-hospitalization complaints or health-seeking behavior for COVID-19 than for other RTIs in children should be further explored.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469620', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'In Diverse Conditions Intrinsic Chromatin Condensates Have Liquid-like Material Properties', 'authors': \"[{'name': 'Bryan A. Gibson'}, {'name': 'Claudia Blaukopf'}, {'name': 'Tracy Lou'}, {'name': 'Lynda K. Doolittle'}, {'name': 'Ilya Finkelstein'}, {'name': 'Geeta J. Narlikar'}, {'name': 'Daniel W. Gerlich'}, {'name': 'Michael K. Rosen'}]\", 'abstract': '<p>Eukaryotic nuclear DNA is wrapped around histone proteins to form nucleosomes, which further assemble to package and regulate the genome. Understanding of the physical mechanisms that contribute to higher order chromatin organization is limited. Previously, we reported the intrinsic capacity of chromatin to undergo phase separation and form dynamic liquid-like condensates, which can be regulated by cellular factors. Recent work from Hansen, Hendzel, and colleagues suggested these intrinsic chromatin condensates are solid in all but a specific set of conditions. Here we show that intrinsic chromatin condensates are fluid in diverse solutions, without need for specific buffering components. Exploring experimental differences in sample preparation and imaging between these two studies, we suggest what may have led Hansen, Hendzel, and colleagues to mischaracterize the innate properties of chromatin condensates. We also describe how liquid-like in vitro behaviors can translate to the locally dynamic but globally constrained movement of chromatin in cells.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469572', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Influenza A virus hemagglutinin prevents extensive membrane damage upon dehydration', 'authors': \"[{'name': 'Maiara A. Iriarte-Alonso'}, {'name': 'Alexander M. Bittner'}, {'name': 'Salvatore Chiantia'}]\", 'abstract': '<p>While the molecular mechanisms of virus infectivity are rather well known, the detailed consequences of environmental factors on virus biophysical properties are poorly understood. Seasonal influenza outbreaks are usually connected to the low winter temperature, but also to the low relative air humidity. Indeed, transmission rates increase in cold regions during winter. While low temperature must slow degradation processes, the role of low humidity is not clear. We studied the effect of relative humidity on a model of influenza A H1N1 virus envelope, a supported lipid bilayer containing the surface glycoprotein hemagglutinin (HA), which is present in the viral envelope in very high density. For complete cycles of hydration, dehydration and rehydration, we evaluate the membrane properties in terms of structure and dynamics, which we assess by combining confocal fluorescence microscopy, raster image correlation spectroscopy, line-scan fluorescence correlation spectroscopy and atomic force microscopy. Our findings indicate that the presence of HA prevents macroscopic membrane damage after dehydration. Without HA, fast membrane disruption is followed by irreversible loss of lipid and protein mobility. Although our model is principally limited by the membrane composition, the macroscopic effects of HA under dehydration stress reveal new insights on the stability of the virus at low relative humidity.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469516', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'CyanoHABs: unavoidable evolutionary ecological consequence for low nutrient-requiring cyanobacteria in eutrophic water', 'authors': \"[{'name': 'Wenduo Cheng'}, {'name': 'Somin Hwang'}, {'name': 'Qisen Guo'}, {'name': 'Haibei Zhang'}, {'name': 'Leyuan Qian'}, {'name': 'Weile Liu'}, {'name': 'Yang Yu'}, {'name': 'Yi Tao'}, {'name': 'Huansheng Cao'}]\", 'abstract': '<p>The mechanism of cyanobacterial harmful algal blooms (CyanoHABs) is complicated and confusing. One major reason is they are studied primarily from an ecological perspective and on bloom-forming species only. This narrow angle loses a broader evolutionary and ecological context in which CyanoHABs occur and fails to provide information on relevant components to achieve a wholistic understanding. To derive a comprehensive mechanism of CyanoHABs, we examine CyanoHABs through the overlooked evolutionary and ecological lenses: evolutionary radiation, ecological comparison with co-living algae, and recently identified genomic functional repertoire between blooming and non-blooming species. We found key factors: (1) elaborate diverse functional repertoire and low nutrient requirement in cyanobacteria molded by early adaptive evolution, (2) cyanobacteria having lower nutrient requirements than green algae indeed, (3) there is no directed evolution in biological functions toward water eutrophication in cyanobacteria, (4) the CyanoHAB-associated functional repertoire are more abundant and complete in blooming than non-blooming species. These factors lead us to postulate a preliminary mechanism of CyanoHABs as a synergistic quad: superior functional repertoire, established with long adaptive radiation under nutrient-deficient conditions and not evolved toward eutrophic conditions, enables cyanobacteria to efficiently utilize elevated nutrients under current eutrophic regime for excess growth and CyanoHABs thereof, due to their lower nutrient requirements than co-living algae. This preliminary synthesis without doubt needs further empirical testing, which can be undertaken with more comparative studies of multiple species using integrated systems biology approaches.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.19.469276', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': True, 'title': 'Hypertonic saline and aprotinin based blockage of SARS-CoV-2 specific furin site cleavage by inhibition of nasal protease activity', 'authors': \"[{'name': 'Clarissa Li'}, {'name': 'Adam W. Li'}]\", 'abstract': '<p>SARS-CoV-2 enters into the human body mainly through the nasal epithelial cells. Cell entry of SARS-CoV-2 needs to be pre-activated by S1/S2 boundary furin motif cleavage by furin and/or relevant proteases. It is important to locally block SARS-CoV-2 S1/S2 site cleavage caused by furin and other relevant protease activity in the nasal cavity. We tested hypertonic saline and aprotinin-based blockage of SARS-CoV-2 specific furin site cleavage by furin, trypsin and nasal swab samples containing nasal proteases. Our results show that saline and aprotinin block SARS-Cov-2 specific furin site cleavage and that a saline and aprotinin combination could significantly reduce SARS-Cov-2 wild-type and P681R mutant furin site cleavage by inhibition of nasal protease activity.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469708', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'A newly-evolved chimeric lysin motif receptor-like kinase in Medicago truncatula spp. tricycla R108 extends its Rhizobia symbiotic partnership', 'authors': \"[{'name': 'Thi-Bich Luu'}, {'name': 'Anna Ourth'}, {'name': 'Ccile Pouzet'}, {'name': 'Nicolas Pauly'}, {'name': 'Julie Cullimore'}]\", 'abstract': '<p><list list-type=\"bullet\"><list-item><p>Rhizobial lipochitooligosaccharidic Nod factors, specified by <italic>nod</italic> genes, are the primary determinants of host specificity in the legume-Rhizobia symbiosis. A <italic>Sinorhizobium meliloti nodF/nodL</italic> mutant produces Nod factors that differ from wild-type ones in lacking an <italic>O</italic>-acetate, and with a different acyl chain on the terminal non-reducing sugar. This mutant is defective in nodulation with various <italic>Medicago</italic> hosts.</p></list-item><list-item><p>We examined the nodulation ability of <italic>M. truncatula</italic> cv Jemalong A17 and <italic>M. truncatula</italic> ssp. <italic>tricycla</italic> R108 with the <italic>nodF/nodL</italic> mutant. We then applied genetic and functional approaches to study the genetic basis and mechanism of nodulation of R108 by this mutant.</p></list-item><list-item><p>We show that the <italic>nodF/nodL</italic> mutant can nodulate R108 but not A17. Using genomics and reverse genetics, we identified a newly-evolved gene in R108, <italic>LYK2bis</italic>, which is responsible for the phenotype. Transformation with <italic>LYK2bis</italic> allows A17 to gain nodulation with the <italic>nodF/nodL</italic> mutant. We found that LYK2bis is involved in specific NF signalling and interacts with the key receptor protein NFP.</p></list-item><list-item><p>Our findings reveal that a newly-evolved gene in R108, <italic>LYK2bis</italic>, extends nodulation specificity to strains producing non-<italic>O</italic>-acetylated NFs. Interaction between LYK2bis and NFP provides a means of integrating the nodulation signalling pathways.</p></list-item></list></p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469851', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Prefrontal, striatal, and VTA subnetwork dynamics during novelty and exploration', 'authors': \"[{'name': 'Adam J.O. Dede'}, {'name': 'Nader Marzban'}, {'name': 'Ashutosh Mishra'}, {'name': 'Robert Reichert'}, {'name': 'Paul M. Anderson'}, {'name': 'Michael X Cohen'}]\", 'abstract': '<p>Multiple distinct brain areas have been implicated in memory including the prefrontal cortex (PFC), striatum (STR), and ventral tegmental area (VTA). Information-exchange across these widespread networks requires flexible coordination at a fine time-scale. In the present study, we collected high-density recordings from the PFC, STR, and VTA of male rats during baseline, encoding, consolidation, and retrieval stages of memory formation. Novel sub-regional clustering analyses identified patterns of spatially restricted, temporally coherent, and frequency specific signals that were reproducible across days and were modulated by behavioral states. Clustering identified miniscule patches of neural tissue. Generalized eigen decomposition (GED) reduced each cluster to a single time series. Amplitude envelope correlation of the cluster time series was used to assess functional connectivity between clusters. Dense intra- and inter regional functional connectivity characterized the baseline period, with delta oscillations playing an outsized role. There was a dramatic pruning of network connectivity during encoding. Connectivity rebounded during consolidation, but connections in the theta band became stronger, and those in the delta band were weaker. Finally, during retrieval, connections were not as severely reduced as they had been during encoding, and specifically theta and higher-frequency connections were stronger. Underlying these connectivity changes, the anatomical extent of clusters observed in the gamma band in the PFC and in both the gamma and delta bands in the VTA changed markedly across behavioral conditions. These results demonstrate the brains ability to reorganize functionally at both the intra- and inter-regional levels during different stages of memory processing.</p><sec><title>SIGNIFICANCE STATEMENT</title><p>The brain is often thought of as a mosaic of areas each with static functions that activate or deactivate with task demands. Here, we used large-scale recordings (196 simultaneous electrodes) and developed a multivariate analysis approach to analyze data from all our recording locations simultaneously. This analysis revealed that the brain dramatically reorganized itself at both local and long-distance spatial scales during different stages of memory processing. These results demonstrate an extreme degree of flexibility in functional anatomy. Rather than thinking about the brain as a set of static mosaic tiles, it may better be characterized as a quickly moldable piece of clay where each parts function changes as the whole is reshaped from moment to moment.</p></sec>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.24.469842', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Single-domain antibodies efficiently neutralize SARS-CoV-2 variants of concern', 'authors': \"[{'name': 'Irina A. Favorskaya'}, {'name': 'Dmitry V. Shcheblyakov'}, {'name': 'Ilias B. Esmagambetov'}, {'name': 'Inna V. Dolzhikova'}, {'name': 'Irina A. Alekseeva'}, {'name': 'Anastasia I. Korobkova'}, {'name': 'Daria V. Voronina'}, {'name': 'Ekaterina I. Ryabova'}, {'name': 'Artem A. Derkaev'}, {'name': 'Anna V. Kovyrshina'}, {'name': 'Anna A. Iliukhina'}, {'name': 'Andrey G. Botikov'}, {'name': 'Olga L. Voronina'}, {'name': 'Daria A. Egorova'}, {'name': 'Olga V. Zubkova'}, {'name': 'Natalia N. Ryzhova'}, {'name': 'Ekaterina I. Aksenova'}, {'name': 'Marina S. Kunda'}, {'name': 'Denis Y. Logunov'}, {'name': 'Boris S. Naroditsky'}, {'name': 'Alexandr L. Gintsburg'}]\", 'abstract': '<p>Virus-neutralizing antibodies are one of the few treatment options for COVID-19. The evolution of SARS-CoV-2 virus has led to the emergence of virus variants with reduced sensitivity to some antibody-based therapies. The development of potent antibodies with a broad spectrum of neutralizing activity is urgently needed. Here we isolated a panel of single-domain antibodies that specifically bind to the receptor-binding domain of SARS-CoV-2 S glycoprotein. Three of the selected antibodies exhibiting most robust neutralization potency were used to generate dimeric molecules. We observed that these modifications resulted in up to a 200-fold increase in neutralizing activity. The most potent heterodimeric molecule efficiently neutralized each of SARS-CoV-2 variant of concern, including Alpha, Beta, Gamma and Delta variants. This heterodimeric molecule could be a promising drug candidate for a treatment for COVID-19 caused by virus variants of concern.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469689', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Is categorization in visual working memory a way to reduce mental effort? A pupillometry study', 'authors': \"[{'name': 'Cherie Zhou'}, {'name': 'Monicque M. Lorist'}, {'name': 'Sebastiaan Matht'}]\", 'abstract': '<p>Recent studies on visual working memory (VWM) have shown that visual information can be stored in VWM as continuous (e.g., a specific shade of red) as well as categorical representations (e.g., the general category red). It has been widely assumed, yet never directly tested, that continuous representations require more VWM mental effort than categorical representations; given limited VWM capacity, this would mean that fewer continuous, as compared to categorical, representations can be maintained simultaneously. We tested this assumption by measuring pupil size, as a proxy for mental effort, in a delayed estimation task. Participants memorized one to four ambiguous (boundaries between adjacent color categories) or prototypical colors to encourage continuous or categorical representations, respectively; after a delay, a probe indicated the location of the to-be-reported color. We found that, for set size 1, pupil size was larger while maintaining ambiguous as compared to prototypical colors, but without any difference in memory precision; this suggests that participants relied on an effortful continuous representation to maintain a single ambiguous color, thus resulting in pupil dilation while preserving precision. In contrast, for set size 2 and higher, pupil size was equally large while maintaining ambiguous and prototypical colors, but memory precision was now substantially reduced for ambiguous colors; this suggests that participants now also relied on categorical representations for ambiguous colors (which are by definition a poor fit to any category), thus reducing memory precision but not resulting in pupil dilation. Taken together, our results suggest that continuous representations are more effortful than categorical representations, and that very few continuous representations (perhaps only one) can be maintained simultaneously.</p><sec><title>Significance statement</title><p>Visual working memory (VWM) can store visual information in two more-or-less distinct formats: continuous and categorical representations. It is widely assumed that VWM capacity for continuous representations is more limited than for categorical representations, yet this assumption has never been explicitly tested. Here we address this crucial question of whether continuous representations cost more resources, and as a consequence limits VWM capacity, as compared to categorical representations. To do so, we measured pupil size (as a proxy for mental effort) and memory performance in a delayed estimation task. Crucially, our results suggest that continuous representations are more effortful than categorical representations, and that only one continuous representation can be maintained in VWM at one time.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469679', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Interplay Between KRAS and LZTR1 Protein Turnover, Controlled by CUL3/LZTR1 E3 Ubiquitin Ligase, is Disrupted by KRAS Mutations', 'authors': \"[{'name': 'Andreas Damianou'}, {'name': 'Zhu Liang'}, {'name': 'Frederik Lassen'}, {'name': 'George Vere'}, {'name': 'Svenja Hester'}, {'name': 'Philip D Charles'}, {'name': 'Adan Pinto-Fernandez'}, {'name': 'Alberto Santos-Delgado'}, {'name': 'Roman Fischer'}, {'name': 'Benedikt M Kessler'}]\", 'abstract': '<p>KRAS is a proto-oncogene encoding a small GTPase. Mutations contribute up to 30% of human solid tumours including lung adenocarcinoma, pancreatic and colorectal carcinomas. Most KRAS activating mutations interfere with GTP hydrolysis, essential for its role as a molecular switch, leading to alterations in their molecular environment and oncogenic signalling. Here, APEX-2 proximity labelling was used to profile the molecular environment of wild type and G12D, G13D and Q61H activating mutants of KRAS under both, starvation and stimulation conditions. We demonstrate by quantitative proteomics the presence of known interactors of KRAS including a-RAF and LZTR1, which varied in abundance with wildtype and KRAS mutants. Notably, the KRAS mutations G12D, G13D and Q61H abrogate association with LZTR1. Wildtype KRAS and LZTR1, as part of the CUL3 ubiquitin E3 ligase complex, affect each others protein stability, revealing a direct feedback loop mechanism. KRAS mutations disconnect this regulatory circuit, thereby contributing to oncogenesis.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.26.470093', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Paleozoic Protein Fossils Illuminate the Evolution of Vertebrate Genomes and Transposable Elements', 'authors': \"[{'name': 'Martin C Frith'}]\", 'abstract': '<p>Genomes hold a treasure trove of protein fossils: fragments of formerly protein-coding DNA, which mainly come from transposable elements (TEs) or host genes. These fossils reveal ancient evolution of TEs and genomes, and many fossils have been exapted to perform diverse functions important for the host\\'s fitness. However, old and highly-degraded fossils are hard to identify, and standard methods (e.g. BLAST) are not optimized for this task. Here, a recently optimized method is used to find protein fossils in vertebrate genomes. It finds Paleozoic fossils predating the amphibian/amniote divergence from most major TE categories, including virus-related Polinton and Gypsy elements. It finds 10 fossils in the human genome (8 from TEs and 2 from host genes) that predate the last common ancestor of all jawed vertebrates, probably from the Ordovician period. It also finds types of transposon and retrotransposon not found in human before. These fossils have extreme sequence conservation, indicating exaptation: some have evidence of gene-regulatory function, and they tend to lie nearest to developmental genes. Some ancient fossils suggest \"genome tectonics\", where two fragments of one TE have drifted apart by up to megabases, possibly explaining gene deserts and large introns. This paints a picture of great TE diversity in our aquatic ancestors, with patchy TE inheritance by later vertebrates, producing new genes and regulatory elements on the way. Host-gene fossils too have contributed anciently-conserved DNA segments. This paves the way to further studies of ancient protein fossils.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.23.469029', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Characterization of the grape powdery mildew genetic resistance loci in Muscadinia rotundifolia Trayshed', 'authors': \"[{'name': 'Mlanie Massonnet'}, {'name': 'Amanda M. Vondras'}, {'name': 'No Cochetel'}, {'name': 'Summaira Riaz'}, {'name': 'Dniel Pap'}, {'name': 'Andrea Minio'}, {'name': 'Rosa Figueroa-Balderas'}, {'name': 'M. Andrew Walker'}, {'name': 'Dario Cantu'}]\", 'abstract': '<p><italic>Muscadinia rotundifolia</italic> cv. Trayshed is a valuable source of resistance to grape powdery mildew (PM). It carries two PM resistance (<italic>R</italic>) loci, <italic>Run1.2</italic> on chromosome 12 and <italic>Run2.2</italic> on chromosome 18. This study identified the nucleotide-binding leucine-rich repeat (NLR) genes composing each <italic>R</italic> locus and their associated defense mechanisms. Evaluation of PM disease development showed that introgression of both loci confers resistance to PM in a <italic>V. vinifera</italic> background, but with varying speed and intensity of the response. To better understand the effect of NLR composition on PM resistance, both haplotypes of each <italic>R</italic> locus were reconstructed and the gene models within each haplotype were manually refined. We found that the number and classes of NLR genes differed between <italic>Run1.2</italic> and <italic>Run2.2</italic> loci and between the haplotypes of each <italic>R</italic> locus. In addition, NLR genes composing <italic>Run1.2b</italic> or <italic>Run2.2</italic> loci exhibited different levels of gene expression, pointing to candidate NLR genes responsible for PM resistance in Trayshed. Finally, a transcriptomic analysis that included six additional <italic>R</italic> loci showed differences in the defense mechanisms associated with <italic>Run1.2b</italic> and <italic>Run2.2</italic> in response to PM and at constitutive level. Altogether, our results reveal that Traysheds <italic>R</italic> loci are composed of distinct NLRs that trigger different plant defense mechanisms in response to PM and at constitutive level, which would explain the variation of pathogen restriction between the two loci.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469476', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Novel RNA viruses associated with avian haemosporidian parasites', 'authors': \"[{'name': 'Jose Roberto Rodrigues'}, {'name': 'Scott William Roy'}, {'name': 'Ravinder Sehgal'}]\", 'abstract': '<p>Avian haemosporidian parasites can cause malaria-like symptoms in their hosts and have been implicated in the demise of some bird species. The newly described Matryoshka RNA viruses (MaRNAV1 and MaRNAV2) infect parasites that in turn infect their vertebrate hosts. MaRNAV2 was the first RNA virus discovered associated with parasites of the genus <italic>Leucocytozoon</italic>. By analyzing metatranscriptomes from the NCBI SRA database with local sequence alignment tools, we detected two novel RNA viruses; we describe them as MaRNAV3 associated with <italic>Leucocytozoon</italic> and MaRNAV4 associated with <italic>Parahaemoproteus</italic>. These had ~40-60% amino acid identity to the RNA-dependent RNA-polymerase (RdRp) of MaRNAV1 and 2, respectively. These findings lead us to hypothesize that MaRNAV_like viruses are widespread and tightly associated with the order Haemosporida since they have been described in human <italic>Plasmodium vivax</italic>, and now two genera of avian haemosporidians.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469580', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Cholinergic control of striatal GABAergic microcircuits', 'authors': \"[{'name': 'S. Kocaturk'}, {'name': 'F. Shah'}, {'name': 'B. Guven'}, {'name': 'J.M. Tepper'}, {'name': 'M. Assous'}]\", 'abstract': '<p>Cholinergic interneurons (CINs) are essential elements of striatal circuits and behaviors. While acetylcholine signaling via muscarinic receptors (mAChRs) have been well studied, more recent data indicate that postsynaptic nicotinic receptors (nAChRs) located on GABAergic interneurons (GINs) are equally critical. One demonstration is that CINs stimulation induces large disynaptic inhibition of SPNs mediated by nAChR activation of striatal GINs. While these circuits are ideally positioned to modulate striatal output activity, the neurons involved are not definitively identified due largely to an incomplete mapping of CINs-GINs interconnections. Here, we show that CINs optogenetic activation evokes an intricate dual mechanism involving co-activation of pre- and postsynaptic mAChRs and nAChRs on four GINs populations. Using multi-optogenetics, we demonstrate the participation of tyrosine hydroxylase-expressing GINs in the disynaptic inhibition of SPNs likely via heterotypic electrical coupling with neurogliaform interneurons. Altogether, our results highlight the importance of CINs in regulating GINs microcircuits via complex synaptic/heterosynaptic mechanisms.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.24.21266768', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Radiology and Imaging', 'covid': False, 'title': 'Insights from an autism imaging biomarker challenge: promises and threats to biomarker discovery', 'authors': \"[{'name': 'Nicolas Traut'}, {'name': 'Katja Heuer'}, {'name': 'Guillaume Lematre'}, {'name': 'Anita Beggiato'}, {'name': 'David Germanaud'}, {'name': 'Monique Elmaleh'}, {'name': 'Alban Bethegnies'}, {'name': 'Laurent Bonnasse-Gahot'}, {'name': 'Weidong Cai'}, {'name': 'Stanislas Chambon'}, {'name': 'Freddy Cliquet'}, {'name': 'Ayoub Ghriss'}, {'name': 'Nicolas Guigui'}, {'name': 'Amicie de Pierrefeu'}, {'name': 'Meng Wang'}, {'name': 'Valentina Zantedeschi'}, {'name': 'Alexandre Boucaud'}, {'name': 'Joris van den Bossche'}, {'name': 'Balzs Kegl'}, {'name': 'Richard Delorme'}, {'name': 'Thomas Bourgeron'}, {'name': 'Roberto Toro'}, {'name': 'Gal Varoquaux'}]\", 'abstract': '<p>MRI has been extensively used to identify anatomical and functional differences in Autism Spectrum Disorder (ASD). Yet, many of these findings have proven difficult to replicate because studies rely on small cohorts and are built on many complex, undisclosed, analytic choices. We conducted an international challenge to predict ASD diagnosis from MRI data, where we provided preprocessed anatomical and functional MRI data from &gt; 2,000 individuals. Evaluation of the predictions was rigorously blinded. 146 challengers submitted prediction algorithms, which were evaluated at the end of the challenge using unseen data and an additional acquisition site. On the best algorithms, we studied the importance of MRI modalities, brain regions, and sample size. We found evidence that MRI could predict ASD diagnosis: the 10 best algorithms reliably predicted diagnosis with AUC0.80  far superior to what can be currently obtained using genotyping data in cohorts 20-times larger. We observed that functional MRI was more important for prediction than anatomical MRI, and that increasing sample size steadily increased prediction accuracy, providing an efficient strategy to improve biomarkers. We also observed that despite a strong incentive to generalise to unseen data, model development on a given dataset faces the risk of overfitting: performing well in cross-validation on the data at hand, but not generalising. Finally, we were able to predict ASD diagnosis on an external sample added after the end of the challenge (EU-AIMS), although with a lower prediction accuracy (AUC=0.72). This indicates that despite being based on a large multisite cohort, our challenge still produced biomarkers fragile in the face of dataset shifts.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>MRI can provide reliable biomarkers for ASD</p></list-item><list-item><p>Because of overfitting, models with excellent apparent performance on public data failed to generalise</p></list-item><list-item><p>The 10 best algorithms used classic machine learning methods</p></list-item><list-item><p>Biomarker development needs blinded evaluation.</p></list-item><list-item><p>Prediction performance should reach AUC=0.83 on samples with 10k subjects</p></list-item></list></sec>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.22.469543', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'ME-ACP: Multi-view Neural Networks with Ensemble Model for Identification of Anticancer Peptides', 'authors': \"[{'name': 'Guanwen Feng'}, {'name': 'Hang Yao'}, {'name': 'Chaoneng Li'}, {'name': 'Ruyi Liu'}, {'name': 'Rungen Huang'}, {'name': 'Xiaopeng Fan'}, {'name': 'Ruiquan Ge'}, {'name': 'Qiguang Miao'}]\", 'abstract': '<p>Cancer remains one of the most threatening diseases, which kills millions of lives every year. As a promising perspective for cancer treatments, anticancer peptides (ACPs) overcome a lot of disadvantages of traditional treatments. However, it is time-consuming and expensive to identify ACPs through conventional experiments. Hence, it is urgent and necessary to develop highly effective approaches to accurately identify ACPs in large amounts of protein sequences. In this work, we proposed a novel and effective method named ME-ACP which employed multi-view neural networks with ensemble model to identify ACPs. Firstly, we employed residue level and peptide level features preliminarily with ensemble models based on lightGBMs. Then, the outputs of lightGBM classifiers were fed into a hybrid deep neural network (HDNN) to identify ACPs. The experiments on independent test datasets demonstrated that ME-ACP achieved competitive performance on common evaluation metrics.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469574', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Correlations from structure and phylogeny combine constructively in the inference of protein partners from sequences', 'authors': \"[{'name': 'Andonis Gerardos'}, {'name': 'Nicola Dietler'}, {'name': 'Anne-Florence Bitbol'}]\", 'abstract': '<p>Inferring protein-protein interactions from sequences is an important task in computational biology. Recent methods based on Direct Coupling Analysis (DCA) or Mutual Information (MI) allow to find interaction partners among paralogs of two protein families. Does successful inference mainly rely on correlations from structural contacts or from phylogeny, or both? Do these two types of signal combine constructively or hinder each other? To address these questions, we generate and analyze synthetic data produced using a minimal model that allows us to control the amounts of structural constraints and phylogeny. We show that correlations from these two sources combine constructively to increase the performance of partner inference by DCA or MI. Furthermore, signal from phylogeny can rescue partner inference when signal from contacts becomes less informative, including in the realistic case where inter-protein contacts are restricted to a small subset of sites. We also demonstrate that DCA-inferred couplings between non-contact pairs of sites improve partner inference in the presence of strong phylogeny, while deteriorating it otherwise. Moreover, restricting to non-contact pairs of sites preserves inference performance in the presence of strong phylogeny. In a natural dataset, as well as in realistic synthetic data based on it, we find that non-contact pairs of sites contribute positively to partner inference performance, and that restricting to them preserves performance, evidencing an important role of phylogeny.</p><sec><title>Author summary</title><p>In protein sequence data, the amino acid usages at different sites of a protein or of two interacting proteins can be correlated because of functional constraints. For instance, the need to maintain physico-chemical complementarity among two sites that are in contact in the three-dimensional structure of a protein complex causes such correlations. However, correlations can also arise due to shared evolutionary history, even in the absence of any functional constraint. While these phylogenetic correlations are known to obscure the inference of structural contacts, we show, using controlled synthetic data, that correlations from structure and phylogeny combine constructively to allow the inference of protein partners from sequences. We also show that pairs of amino acids that are not in contact in the structure have a major impact on partner inference in a natural dataset and in realistic synthetic ones. These findings explain the success of methods based on pairwise maximum-entropy models or on information theory at predicting protein partners from sequences.</p></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.26.470137', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Staphylococcus aureus -toxin exerts anti-angiogenic effects by inhibiting re-endothelialization and neovessel formation', 'authors': \"[{'name': 'Phuong M Tran'}, {'name': 'Sharon S Tang'}, {'name': 'Wilmara Salgado-Pabn'}]\", 'abstract': '<p><italic>Staphylococcus aureus</italic> is the causative agent of numerous severe human infections associated with significant morbidity and mortality worldwide. <italic>S. aureus</italic> often targets the vascular endothelium to interfere with proper host responses during invasive infections. In this study, we provide evidence that <italic>S. aureus</italic> -toxin inhibits wound repair mechanisms in human endothelial cells by preventing cell proliferation and migration. These findings were confirmed in a rabbit aortic explant model where -toxin impedes sprout formation. Decreased cell proliferation was accompanied by decreased production of the angiogenic proteins endothelin-1, IGFBP-3, thrombospondin-1, TIMP-1, and TIMP-4. Meanwhile, inhibited wound repair was marked by increased HGF secretion from endothelial cells, likely a marker of endothelial cell damage. Together, these findings establish a mechanistic role for -toxin where it inhibits proper tissue repair processes that likely promote <italic>S. aureus</italic> infective niche.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.25.470010', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Lipid droplets are required for lipid mediator production and cancer cell proliferation', 'authors': \"[{'name': 'Toni Petan'}, {'name': 'Thomas O. Eichmann'}, {'name': 'Robert Zimmermann'}, {'name': 'Anja Pucer Jane'}, {'name': 'Jana Gerstmeier'}, {'name': 'Grard Lambeau'}, {'name': 'Eva Jarc Jovii'}, {'name': 'Paul Mike Jordan'}, {'name': 'Oliver Werz'}, {'name': 'Vesna Brglez'}]\", 'abstract': '<p>Lipid droplets are dynamic organelles with a central role in fatty acid metabolism. They protect cells from lipotoxicity by sequestering excess fatty acids but also provide fatty acids for metabolic reactions and signalling events. Here we show that lipid droplet turnover in cancer cells is required for production of -3 and -6 polyunsaturated fatty acid (PUFA)-derived inflammatory lipid mediators, including eicosanoids and specialised pro-resolving mediators. We show that incorporation of PUFAs into triglycerides mediated by diacylglycerol acyltransferase 1 (DGAT1), and their release by adipose triglyceride lipase (ATGL), are required for cyclooxygenase- and lipoxygenase-dependent lipid mediator production and cancer cell proliferation. The human group X secreted phospholipase A2 (hGX sPLA2) drives the delivery of membrane-derived PUFAs into lipid droplets, while ATGL promotes the incorporation of lipid droplet-derived PUFAs into phospholipids. The group IVA cytosolic PLA2 (cPLA2) acts on membrane phospholipids and complements ATGL in the regulation of PUFA trafficking between phospholipids and triglycerides. This study identifies lipid droplets as essential cellular hubs that control PUFA availability for production of lipid mediators involved in inflammation and tumorigenesis.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.26.470105', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Mars, a molecule archive suite for reproducible analysis and reporting of single molecule properties from bioimages', 'authors': \"[{'name': 'Nadia M Huisjes'}, {'name': 'Thomas M Retzer'}, {'name': 'Matthias J Scherr'}, {'name': 'Rohit Agarwal'}, {'name': 'Barbara Safaric'}, {'name': 'Anita Minnen'}, {'name': 'Karl E Duderstadt'}]\", 'abstract': '<p>The rapid development of new imaging approaches is generating larger and more complex datasets revealing the time evolution of individual cells and biomolecules. Single-molecule techniques, in particular, provide access to rare intermediates in complex, multistage molecular pathways, but few standards exist for processing these information-rich datasets, posing challenges for wider dissemination. Here, we present Mars, an open-source platform for storage and processing of image-derived properties of biomolecules. Mars provides Fiji/ImageJ2 commands written in Java for common single-molecule analysis tasks using a Molecule Archive architecture that is easily adapted to complex, multistep analysis workflows. Three diverse workflows involving molecule tracking, multichannel fluorescence imaging, and force spectroscopy, demonstrate the range of analysis applications. A comprehensive graphical user interface written in JavaFX enhances biomolecule feature exploration by providing charting, tagging, region highlighting, scriptable dashboards, and interactive image views. The interoperability of ImageJ2 ensures Molecule Archives can easily be opened in multiple environments, including those written in Python using PyImageJ, for interactive scripting and visualization. Mars provides a flexible solution for reproducible analysis of image-derived properties facilitating the discovery and quantitative classification of new biological phenomena with an open data format accessible to everyone.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.22.21266676', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Investigating SARS-CoV-2 breakthrough infections per variant and vaccine type', 'authors': \"[{'name': 'Jozef Dingemans'}, {'name': 'Brian M.J.W. van der Veer'}, {'name': 'Koen M.F. Gorgels'}, {'name': 'Volker Hackert'}, {'name': 'Audrey Y.J. Hensels'}, {'name': 'Casper D.J. den Heijer'}, {'name': 'Christian J.P.A Hoebe'}, {'name': 'Paul H.M. Savelkoul'}, {'name': 'Lieke B. van Alphen'}]\", 'abstract': '<p>Breakthrough SARS-CoV-2 infections have been reported in fully vaccinated individuals, in spite of the high efficacy of the currently available vaccines, proven in trials and real-world studies. Several variants of concern (VOC) have been proffered to be associated with breakthrough infections following immunization. In this study, we investigated 378 breakthrough infections recorded between January and July 2021 and compared the distribution of SARS-CoV-2 genotypes identified in 225 fully vaccinated individuals to the frequency of circulating community lineages in the region of South Limburg (The Netherlands) in a week-by-week comparison. Although the proportion of breakthrough infections was relatively low and stable when the Alpha variant was predominant, the rapid emergence of the Delta variant lead to a strong increase in breakthrough infections, with a higher relative proportion of individuals vaccinated with Oxford-AstraZeneca or J&amp;J/Janssen being infected compared to those immunized with mRNA-based vaccines. A significant difference in median age was observed when comparing fully vaccinated individuals with severe symptoms (83 years) to asymptomatic cases (46.5 years) or individuals with mild-to-moderate symptoms (42 years). There was no association between SARS-CoV-2 genotype or vaccine type and disease symptoms. Interestingly, symptomatic individuals harbored significantly higher SARS-CoV-2 loads than asymptomatic vaccinated individuals and breakthrough infections caused by the Delta variant are associated with increased viral loads compared to those caused by the Alpha variant. Altogether, these results indicate that the emergence of the Delta variant might have lowered the efficiency of particular vaccine types to prevent SARS-CoV-2 infections and that, although rare, the elderly are particularly at risk of becoming severely infected as the consequence of a breakthrough infection.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469656', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Parent-offspring inference in inbred populations', 'authors': \"[{'name': 'Jan-Niklas Runge'}, {'name': 'Barbara Knig'}, {'name': 'Anna K. Lindholm'}, {'name': 'Andres Bendesky'}]\", 'abstract': '<p>Genealogical relationships are fundamental components of genetic studies. However, it is often challenging to infer correct and complete pedigrees even when genome-wide information is available. For example, inbreeding can obfuscate genetic differences between individuals, making it difficult to even distinguish first-degree relatives such as parent-offspring from full siblings. Similarly, genotyping errors can interfere with the detection of genetic similarity between parents and their offspring. Inbreeding is common in natural, domesticated, and experimental populations and genotyping of these populations often has more errors than in human datasets, so efficient methods for building pedigrees under these conditions are necessary. Here, we present a new method for parent-offspring inference in inbred pedigrees called <italic>SPORE</italic> (Specific Parent-Offspring Relationship Estimation). <italic>SPORE</italic> is vastly superior to existing pedigree-inference methods at detecting parent-offspring relationships, in particular when inbreeding is high or in the presence of genotyping errors, or both. <italic>SPORE</italic> therefore fills an important void in the arsenal of pedigree inference tools.</p><sec><title>Author Summary</title><p>Knowing the genealogical relationships among individuals is critical for genetic analyses, such as for identifying the mutations that cause diseases or that contribute to valuable agricultural traits such as milk production. Although many tools exist for establishing pedigrees using genetic information, these tools fail when individuals are highly inbred, such as in domesticated animals, or in groups of people in which consanguineous matings are common. Furthermore, existing tools do not work well when genetic information has errors at levels observed in modern datasets. Here, we present a novel approach to solve these problems. Our method is significantly more accurate than existing tools and more tolerant of errors in the genetic data. We expect that our method, which is simple to use and computationally efficient, will be a useful tool in a diversity of settings, from the studies of human and natural populations, to agricultural and experimental settings.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.26.470134', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Allelic correlation is a marker of tradeoffs between barriers to transmission of expression variability and signal responsiveness in genetic networks', 'authors': \"[{'name': 'Ryan H Boe'}, {'name': 'Vinay Ayyappan'}, {'name': 'Lea Schuh'}, {'name': 'Arjun Raj'}]\", 'abstract': '<p>Accurately functioning genetic networks should be responsive to signals but prevent transmission of stochastic bursts of expression. Existing data in mammalian cells suggests that such transcriptional \"noise\" is transmitted by some genes and not others, suggesting that noise transmission is tunable, perhaps at the expense of other signal processing capabilities. However, systematic claims about noise transmission in genetic networks have been limited by the inability to directly measure noise transmission. Here we build a mathematical framework capable of modeling allelic correlation and noise transmission. We find that allelic correlation and noise transmission correspond across a broad range of model parameters and network architectures. We further find that limiting noise transmission comes with the trade-off of being unresponsive to signals, and that within the parameter regimes that are responsive to signals, there is a further trade-off between response time and basal noise transmission. Using a published allele specific single cell RNA-sequencing dataset, we found that genes with high allelic odds ratios are enriched for cell-type specific functions, and that within multiple signaling pathways, factors which are upstream in the pathway have higher allelic odds ratios than downstream factors. Overall, our findings suggest that some degree of noise transmission is required to be responsive to signals, but that minimization of noise transmission can be accomplished by trading-off for a slower response time.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.22.469506', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'NGSpop: A desktop software that supports population studies by identifying sequence variations from next-generation sequencing data', 'authors': \"[{'name': 'Dong-Jun Lee'}, {'name': 'Taesoo Kwon'}, {'name': 'Hye-Jin Lee'}, {'name': 'Yun-Ho Oh'}, {'name': 'Jin-Hyun Kim'}, {'name': 'Tae-Ho Lee'}]\", 'abstract': '<p>Next-generation sequencing (NGS) is widely used in all areas of genetic research, such as for genetic disease diagnosis and breeding, and it can produce massive amounts of data. The identification of sequence variants is an important step when processing large NGS datasets; however, currently, the process is complicated, repetitive, and requires concentration, which can be taxing on the researcher. Therefore, to support researchers who are not familiar with bioinformatics in identifying sequence variations regularly from large datasets, we have developed a fully automated desktop software, NGSpop. NGSpop includes functionalities for all the variant calling and visualization procedures used when processing NGS data, such as quality control, mapping, filtering details, and variant calling. In the variant calling step, the user can select the GATK or DeepVariant algorithm for variant calling. These algorithms can be executed using pre-set pipelines and options or customized with the user-specified options. NGSpop is implemented using JavaFX (version 1.8) and can thus be run on Unix like operating systems such as Ubuntu Linux (version 16.04, 18.0.4). Although there are several pipelines and visualization tools available for NGS data analysis, most integrated environments do not support batch processes; thus, variant detection cannot be automated for population-level studies. The NGSpop software, developed in this study, has an easy-to-use interface and helps in rapid analysis of multiple NGS data from population studies.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469541', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Quantification of vascular networks in photoacoustic mesoscopy', 'authors': \"[{'name': 'Emma L. Brown'}, {'name': 'Thierry L. Lefebvre'}, {'name': 'Paul W. Sweeney'}, {'name': 'Bernadette J. Stolz'}, {'name': 'Janek Grhl'}, {'name': 'Lina Hacker'}, {'name': 'Ziqiang Huang'}, {'name': 'Dominique-Laurent Couturier'}, {'name': 'Heather A. Harrington'}, {'name': 'Helen M. Byrne'}, {'name': 'Sarah E. Bohndiek'}]\", 'abstract': '<p>Mesoscopic photoacoustic imaging (PAI) enables non-invasive visualisation of tumour vasculature and has the potential to assess prognosis and therapeutic response. Currently, evaluating vasculature using mesoscopic PAI involves visual or semi-quantitative 2D measurements, which fail to capture 3D vessel network complexity, and lack robust ground truths for assessment of segmentation accuracy. Here, we developed an <italic>in silico</italic>, phantom, <italic>in vivo</italic>, and <italic>ex vivo</italic>-validated end-to-end framework to quantify 3D vascular networks captured using mesoscopic PAI. We applied our framework to evaluate the capacity of rule-based and machine learning-based segmentation methods, with or without vesselness image filtering, to preserve blood volume and network structure by employing topological data analysis. We first assessed segmentation performance against ground truth data of <italic>in silico</italic> synthetic vasculatures and a photoacoustic string phantom. Our results indicate that learning-based segmentation best preserves vessel diameter and blood volume at depth, while rule-based segmentation with vesselness image filtering accurately preserved network structure in superficial vessels. Next, we applied our framework to breast cancer patient-derived xenografts (PDXs), with corresponding <italic>ex vivo</italic> immunohistochemistry. We demonstrated that the above segmentation methods can reliably delineate the vasculature of 2 breast PDX models from mesoscopic PA images. Our results underscore the importance of evaluating the choice of segmentation method when applying mesoscopic PAI as a tool to evaluate vascular networks <italic>in vivo</italic>.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.21266731', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Radiology and Imaging', 'covid': False, 'title': 'In vivo myelin imaging and tissue microstructure in white matter hyperintensities and perilesional white matter', 'authors': \"[{'name': 'Jennifer K. Ferris'}, {'name': 'Brian Greeley'}, {'name': 'Irene M. Vavasour'}, {'name': 'Sarah N. Kraeutner'}, {'name': 'Shie Rinat'}, {'name': 'Joel Ramirez'}, {'name': 'Sandra E. Black'}, {'name': 'Lara A. Boyd'}]\", 'abstract': '<sec><title>Introduction</title><p>White Matter hyperintensities (WMHs) relate to cognitive decline in aging, due to their deleterious effect on white matter structure. Diffusion tensor imaging (DTI) detects changes to white matter microstructure, both within the WMH and extending in a penumbra-like pattern in surrounding (perilesional) normal appearing white matter (NAWM). However, DTI markers are not specific to tissue components, complicating interpretation of previous DTI findings. Myelin water imaging is a novel imaging technique that provides specific markers of myelin content (myelin water fraction: MWF) and interstitial fluid (geometric mean T2: GMT2). Here we combined DTI and myelin water imaging to examine tissue characteristics in WMHs and perilesional NAWM. Our goal was to establish a multimodal neuroimaging approach to track WMH progression.</p></sec><sec><title>Methods</title><p>80 individuals (47 older adults and 33 individuals with chronic stroke) underwent neuroimaging and tissue segmentation. To measure perilesional NAWM, WMH masks were dilated in 2mm segments up to 10mm in distance from the WMH to measure perilesional NAWM. Fractional anisotropy (FA), mean diffusivity (MD), MWF, and GMT2 were extracted from WMHs and perilesional NAWM. We examined whether white matter metrics showed a spatial gradient of effects in perilesional NAWM, as a function of Distance from the WMH, and Group. We tested whether white matter metrics in the WMH lesion related to severity of cerebrovascular disease across the whole sample, indexed by whole brain WMH volume.</p></sec><sec><title>Results</title><p>We observed a spatial gradient of higher MD and GMT2, and lower FA, in perilesional NAWM and the WMH. In the chronic stroke group, MWF was reduced in the WMH lesion but did not show a spatial gradient in perilesional NAWM. Across the whole sample, white matter metrics within the WMH lesion related to whole-brain WMH volume; MD and GMT2 increased, and MWF decreased, with increasing WMH volume.</p></sec><sec><title>Conclusions</title><p>NAWM adjacent to WMHs exhibits characteristics of a transitional stage between healthy NAWM tissue and WMH lesions. This was observed in markers sensitive to interstitial fluid, but not in the specific marker of myelin concentration (MWF). Within the WMH, interstitial fluid was higher and myelin concentration was lower in individuals with more severe cerebrovascular disease. Our data suggests that fluid-sensitive imaging metrics (such as DTI) can identify NAWM at high risk of conversion to a WMH. In contrast, specific markers of myelin concentration (e.g., MWF) can be used to measure levels of demyelination in the WMH itself, which may be a useful marker for disease staging of advanced WMHs.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469911', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'The ADAR1 editome reveals drivers of editing-specificity for ADAR1-isoforms', 'authors': \"[{'name': 'Renata Kleinova'}, {'name': 'Alina F. Leuchtenberger'}, {'name': 'Claudio Lo Giudice'}, {'name': 'Andrea Tanzer'}, {'name': 'Sophia Derdak'}, {'name': 'Ernesto Picardi'}, {'name': 'Michael F. Jantsch'}]\", 'abstract': '<p>Adenosine deaminase acting on RNA (ADAR) (also known as ADAR1) promotes A-to-I conversion in double-stranded and highly structured RNAs. ADAR1 has two isoforms transcribed from different promoters: ADAR1p150, which is mainly cytoplasmic and interferon-inducible, and constitutively expressed ADAR1p110 that is primarily localized in the nucleus.</p><p>Mutations in ADAR1 cause Aicardi  Goutires syndrome (AGS), a severe autoinflammatory disease in humans associated with aberrant IFN production. In mice, deletion of ADAR1 or selective knockout of the p150 isoform alone leads to embryonic lethality driven by overexpression of interferon-stimulated genes. This phenotype can be rescued by concurrent deletion of cytoplasmic dsRNA-sensor MDA5. These findings indicate that the interferon- inducible p150 isoform is indispensable and cannot be rescued by the ADAR1p110 isoform. Nevertheless, editing sites uniquely targeted by ADAR1p150 but also mechanisms of isoform- specificity remain elusive.</p><p>Here we combine RIP-seq on human cells expressing ADAR1 isoforms and combine this with analysis of isoform-specific editing patterns in genetically modified mouse cells to extensively investigate ADAR1-isoform binding- and editing characteristics.</p><p>Moreover, using mutated ADAR variants, we examine the effect of two unique features of ADAR1p150 on its target specificity: 1) cytoplasmic localization and 2) Z-DNA binding domain <italic></italic>. Our findings indicate that ZBD<italic></italic> contributes only minimally to p150 editing-specificity and that isoform-specific editing is directed mainly by the cytoplasmic localization of the editase.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266584', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Behaviour, booster vaccines and waning vaccine protection: modelling the medium-term dynamics of SARS-CoV-2 transmission in England', 'authors': \"[{'name': 'Rosanna C. Barnard'}, {'name': 'Nicholas G. Davies'}, {'name': 'Mark Jit'}, {'name': 'W. John Edmunds'}, {'name': ' '}]\", 'abstract': '<p>England has experienced a heavy burden of COVID-19, with high infection levels observed throughout the summer months of 2021. Alongside the emergence of evidence suggesting that COVID-19 vaccine protection wanes over time, booster vaccinations began for individuals aged 50 and above in September 2021. Using a model fitted to 18 months of epidemiological data, we project potential dynamics of SARS-CoV-2 transmission in England to September 2022. We consider key uncertainties including behavioural change, waning vaccine protection, strategies for vaccination, and the reintroduction of public health and social measures. We project the current wave of transmission will peak in Autumn 2021, with low levels of transmission in early 2022. The extent to which SARS-CoV-2 transmission resurges in 2022 depends largely on assumptions around waning vaccine protection and booster vaccinations. Widespread booster vaccinations or the reimposition of mild public health and social measures such as work-from-home policies could largely mitigate the wave of COVID-19 transmission projected to occur in England in Spring/Summer 2022.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266711', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Germanys low SARS-CoV-2 seroprevalence confirms effective containment in 2020: Results of the nationwide RKI-SOEP study', 'authors': \"[{'name': 'Hannelore Neuhauser'}, {'name': 'Angelika Schaffrath Rosario'}, {'name': 'Hans Butschalowsky'}, {'name': 'Sebastian Haller'}, {'name': 'Jens Hoebel'}, {'name': 'Janine Michel'}, {'name': 'Andreas Nitsche'}, {'name': 'Christina Poethko-Mller'}, {'name': 'Franziska Prtz'}, {'name': 'Martin Schlaud'}, {'name': 'Hans W. Steinhauer'}, {'name': 'Hendrik Wilking'}, {'name': 'Lothar H. Wieler'}, {'name': 'Lars Schaade'}, {'name': 'Stefan Liebig'}, {'name': 'Antje Gwald'}, {'name': 'Markus M. Grabka'}, {'name': 'Sabine Zinn'}, {'name': 'Thomas Ziese'}]\", 'abstract': '<p>Pre-vaccine SARS-CoV-2 seroprevalence data from Germany are scarce outside hotspots, and socioeconomic disparities remained largely unexplored. The nationwide RKI-SOEP study with 15,122 adult participants investigated seroprevalence and testing in a supplementary wave of the Socio-Economic-Panel conducted predominantly in October-November 2020. Self-collected oral-nasal swabs were PCR-positive in 0.4% and Euroimmun anti-SARS-CoV-2-S1-IgG ELISA from dry capillary blood in 1.3% (95% CI 0.9-1.7%, population-weighted, corrected for sensitivity=0.811, specificity=0.997). Seroprevalence was 1.7% (95% CI 1.2-2.3%) when additionally adjusting for antibody decay. Overall infection prevalence including self-reports was 2.1%. We estimate 45% (95% CI 21-60%) undetected cases and analyses suggest lower detection in socioeconomically deprived districts. Prior SARS-CoV-2 testing was reported by 18% from the lower educational group compared to 25% and 26% from the medium and high educational group (p&lt;0.0001). Symptom-triggered test frequency was similar across educational groups. However, routine testing was more common in low-educated adults, whereas travel-related testing and testing after contact with an infected person was more common in highly educated groups. In conclusion, pre-vaccine SARS-CoV-2-seroprevalence in Germany was very low. Notified cases appear to capture more than half of infections but may underestimate infections in lower socioeconomic groups. These data confirm the successful containment strategy of Germany until winter 2020.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469854', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Long-term culture system for deep-sea mussels Gigantidas childressi', 'authors': \"[{'name': 'Claas Hiebenthal'}, {'name': 'Finn-Ole Gehlert'}, {'name': 'Mark Schmidt'}, {'name': 'Thorsten B. H. Reusch'}, {'name': 'Frank Melzner'}]\", 'abstract': '<p>The simulation of deep-sea conditions in laboratories is technically challenging but necessary for experiments that aim at a deeper understanding of physiological mechanisms or host-symbiont interactions of deep-sea organisms. In a proof-of-concept study, we designed a recirculating system for long-term culture (&gt;2 years) of deep-sea mussels <italic>Gigantidas childressi</italic> (previously <italic>Bathymodiolus childressi</italic>). Mussels were automatically (and safely) supplied with a maximum stable level of ~60 M methane in seawater using a novel methane-air mixing system. Experimental animals also received daily doses of live microalgae. Condition indices of cultured <italic>G. childressi</italic> remained high over years, and low shell thickness growth could be detected, which is indicative of positive energy budgets. Using stable isotope data, we demonstrate that <italic>G. childressi</italic> in our culture system gained energy, both, from digestion of methane oxidizing endosymbionts and from digesting particulate food (microalgae). Limitations of the system, as well as opportunities for future experimental approaches involving deep-sea mussels are discussed.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.28.470272', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Analysis of 8000 proteins and reduced carry over significantly increase the throughput of single-shot proteomics', 'authors': \"[{'name': 'Karel Stejskal'}, {'name': 'Jeff Op de Beeck'}, {'name': 'Manuel Matzinger'}, {'name': 'Gerhard Duernberger'}, {'name': 'Oleksandr Boychenko'}, {'name': 'Paul Jacobs'}, {'name': 'Karl Mechtler'}]\", 'abstract': '<p>In the field of LC-MS based proteomics, increases in sampling depth and proteome coverage have mainly been accomplished by rapid advances in mass spectrometer technology. The comprehensiveness and quality of data that can be generated do however also depend on the performance provided by nano liquid chromatography (nanoLC) separations. Proper selection of reversed-phase separation columns can be of paramount importance to provide the MS instrument with peptides at the highest possible concentration and separated at the highest possible resolution. As an alternative to traditional packed bed LC column technology that uses beads packed into capillary tubing, we present a novel LC column format based on photolithographic definition and Deep Reactive Ion Etching (DRIE) into silicon wafers. With a next generation pillar array column designed for universal use in bottom-up proteomics, the critical dimensions of the stationary phase support structures have been reduced by a factor of 2 to provide further increases in separation power. To demonstrate the potential for single-shot proteomics workflows, we report on a series of optimization and benchmarking experiments where we combine LC separation on a new generation of pillar array columns using Vanquish Neo UHPLC with fast Orbitrap Tribrid MS data-dependent acquisition (DDA) and High-Field Asymmetric Waveform Ion Mobility Spectrometry (FAIMS). In addition to providing superior proteome coverage, robust operation over more than 1 month with a single nanoESI emitter and reduction of the column related sample carry over are additional figures of merit that can help improve proteome research sensitivity, productivity and standardization.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.23.469755', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Distinct SARS-CoV-2 sensing pathways in pDCs driving TLR7-antiviral vs. TLR2-immunopathological responses in COVID-19', 'authors': \"[{'name': 'Rene M. van der Sluis'}, {'name': 'Lamin B. Cham'}, {'name': 'Albert Gris Oliver'}, {'name': 'Kristine R. Gammelgaard'}, {'name': 'Jesper G. Pedersen'}, {'name': 'Manja Idorn'}, {'name': 'Ulvi Ahmadov'}, {'name': 'Sabine Sanches Hernandez'}, {'name': 'Ena Cmalovic'}, {'name': 'Stine H. Godsk'}, {'name': 'Jacob Thyrsted'}, {'name': 'Jesper D. Gunst'}, {'name': 'Silke D. Nielsen'}, {'name': 'Janni J. Jrgensen'}, {'name': 'Tobias Wang Bjerg'}, {'name': 'Anders Laustsen'}, {'name': 'Line S. Reinert'}, {'name': 'David Olagnier'}, {'name': 'Rasmus O. Bak'}, {'name': 'Mads Kjolby'}, {'name': 'Christian K. Holm'}, {'name': 'Martin Tolstrup'}, {'name': 'Sren R. Paludan'}, {'name': 'Lasse S. Kristensen'}, {'name': 'Ole S. Sgaard'}, {'name': 'Martin R. Jakobsen'}]\", 'abstract': '<p>Understanding the molecular pathways driving the acute antiviral and inflammatory response to SARS-CoV-2 infection is critical for developing treatments for severe COVID-19. Here we show that in COVID-19 patients, circulating plasmacytoid dendritic cells (pDCs) decline early after symptom onset and this correlated with COVID-19 disease severity. This transient depletion coincides with decreased expression of antiviral type I IFN and the systemic inflammatory cytokines CXCL10 and IL-6. Importantly, COVID-19 disease severity correlated with decreased pDC frequency in peripheral blood. Using an <italic>in vitro</italic> stem cell-based human pDC model, we demonstrate that pDCs directly sense SARS-CoV-2 and in response produce multiple antiviral (IFN and IFN1) and inflammatory (IL-6, IL-8, CXCL10) cytokines. This immune response is sufficient to protect epithelial cells from de novo SARS-CoV-2 infection. Targeted deletion of specific sensing pathways identified TLR7-MyD88 signaling as being crucial for production of the antiviral IFNs, whereas TLR2 is responsible for the inflammatory IL-6 response. Surprisingly, we found that SARS-CoV-2 engages the neuropilin-1 receptor on pDCs to mitigate the antiviral IFNs but not the IL-6 response. These results demonstrate distinct sensing pathways used by pDCs to elicit antiviral vs. immunopathological responses to SARS-CoV-2 and suggest that targeting neuropilin-1 on pDCs may be clinically relevant for mounting TLR7-mediated antiviral protection.</p><sec><title>One Sentence Summary</title><p>pDCs sense SARS-CoV-2 and elicit antiviral protection of lung epithelial cells through TLR7, while recognition of TLR2 elicits an IL-6 inflammatory response associated with immunopathology.<fig id=\"ufig1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\"><label>Graphical abstract:</label><caption><title>SARS-CoV-2 sensing by plasmacytoid dendritic cells.</title><p>SARS-CoV-2 is internalized by pDCs via a yet unknown endocytic mechanism. The intracellular TLR7 sensor detects viral RNA and induces a signaling cascade involving MyD88-IRAK4-TRAF6 (1) to induce CXCL10 and, via IRF7 phosphorylation and translocation, inducing type I and III Interferons (2). Once secreted, type I and III IFNs initiate autocrine and paracrine signals that induce the expression of IFN stimulated genes (ISGs), thereby facilitating an antiviral response that can protect the cell against infection. However, SARS-CoV-2, has the intrinsic property to facilitate CD304 signaling, potentially by interfering with IRF7 nuclear translocation, thereby inhibiting type I IFN production and thus reducing the antiviral response generated by the pDC (4). Furthermore, the SARS-CoV-2 envelope (E) glycoprotein is sensed by the extracellular TLR2/6 heterodimer and this facilitates production of the inflammatory IL-6 cytokine (5). Illustration was created with BioRender.com</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"469755v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469742', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Molecular Basis of Antibiotic Self-Resistance in a Bee Larvae Pathogen', 'authors': \"[{'name': 'Tam Dang'}, {'name': 'Bernhard Loll'}, {'name': 'Sebastian Mller'}, {'name': 'Ranko Skobalj'}, {'name': 'Julia Ebeling'}, {'name': 'Timur Bulatov'}, {'name': 'Sebastian Gensel'}, {'name': 'Josefine Gbel'}, {'name': 'Markus C. Wahl'}, {'name': 'Elke Genersch'}, {'name': 'Andi Mainz'}, {'name': 'Roderich D. Sssmuth'}]\", 'abstract': '<p><italic>Paenibacillus larvae</italic>, the causative agent of the devastating honey-bee disease American Foulbrood, produces the cationic polyketide-peptide hybrid paenilamicin that displays high antibacterial and antifungal activity. Its biosynthetic gene cluster contains a gene coding for the <italic>N</italic>-acetyltransferase PamZ. We show that PamZ acts as self-resistance factor in <italic>P. larvae</italic> by deactivation of paenilamicin. Using tandem MS, NMR spectroscopy and synthetic diastereomers, we identified the N-terminal amino group of the agmatinamic acid as the <italic>N</italic>-acetylation site. These findings highlight the pharmacophore region of paenilamicin, which we very recently identified as a new ribosome inhibitor. Here, we further elucidated the crystal structure of PamZ:acetyl-CoA complex at 1.34  resolution. An unusual tandem-domain architecture provides a well-defined substrate-binding groove decorated with negatively-charged residues to specifically attract the cationic paenilamicin. Our results will help to understand the mode of action of paenilamicin and its role in pathogenicity of <italic>P. larvae</italic> to fight American Foulbrood.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.21.469423', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Conserved recombination patterns across coronavirus subgenera', 'authors': \"[{'name': 'Arn de Klerk'}, {'name': 'Phillip Swanepoel'}, {'name': 'Rentia Lourens'}, {'name': 'Mpumelelo Zondo'}, {'name': 'Isaac Abodunran'}, {'name': 'Spyros Lytras'}, {'name': 'Oscar A MacLean'}, {'name': 'David Robertson'}, {'name': 'Sergei L Kosakovsky Pond'}, {'name': 'Jordan D Zehr'}, {'name': 'Venkatesh Kumar'}, {'name': 'Michael J. Stanhope'}, {'name': 'Gordon Harkins'}, {'name': 'Ben Murrell'}, {'name': 'Darren P Martin'}]\", 'abstract': '<p>Recombination contributes to the genetic diversity found in coronaviruses and is known to be a prominent mechanism whereby they evolve. It is apparent, both from controlled experiments and in genome sequences sampled from nature, that patterns of recombination in coronaviruses are non-random and that this is likely attributable to a combination of sequence features that favour the occurrence of recombination breakpoints at specific genomic sites, and selection disfavouring the survival of recombinants within which favourable intra-genome interactions have been disrupted. Here we leverage available whole-genome sequence data for six coronavirus subgenera to identify specific patterns of recombination that are conserved between multiple subgenera and then identify the likely factors that underlie these conserved patterns. Specifically, we confirm the non-randomness of recombination breakpoints across all six tested coronavirus subgenera, locate conserved recombination hot- and cold-spots, and determine that the locations of transcriptional regulatory sequences are likely major determinants of conserved recombination breakpoint hot-spot locations. We find that while the locations of recombination breakpoints are not uniformly associated with degrees of nucleotide sequence conservation, they display significant tendencies in multiple coronavirus subgenera to occur in low guanine-cytosine content genome regions, in non-coding regions, at the edges of genes, and at sites within the Spike gene that are predicted to be minimally disruptive of Spike protein folding. While it is apparent that sequence features such as transcriptional regulatory sequences are likely major determinants of where the template-switching events that yield recombination breakpoints most commonly occur, it is evident that selection against misfolded recombinant proteins also strongly impacts observable recombination breakpoint distributions in coronavirus genomes sampled from nature.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.28.469673', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Does the brain care about averages? A simple test.', 'authors': \"[{'name': 'Alejandro Tlaie'}, {'name': 'Katharine A Shapcott'}, {'name': 'Paul Tiesinga'}, {'name': 'Marieke Schlvinck'}, {'name': 'Martha N Havenith'}]\", 'abstract': '<p>Trial-averaged metrics, e.g. in the form of tuning curves and population response vectors, are a basic and widely accepted way of characterizing neuronal activity. But how relevant are such trial-averaged responses to neuronal computation itself? Here we present a simple test to estimate whether average responses reflect aspects of neuronal activity that contribute to neuronal processing in a specific context. The test probes two assumptions inherent in the usage of average neuronal metrics: 1) Reliability: Neuronal responses repeat consistently enough across single stimulus instances that the average response template they relate to remains recognizable to downstream regions. 2) Behavioural relevance: If a single-trial response is more similar to the average template, this should make it easier for the animal to identify the correct stimulus or action. We apply this test to a large publicly available data set featuring electrophysiological recordings from 42 cortical areas in behaving mice. In this data set, we show that single-trial responses were less correlated to the average response template than one would expect if they simply represented discrete versions of the template, down-sampled to a finite number of spikes. Moreover, single-trial responses were barely stimulus-specific  they could not be clearly assigned to the average response template of one stimulus. Most importantly, better-matched single-trial responses did not predict accurate behaviour for any of the recorded cortical areas. We conclude that in this data set, average responses do not seem particularly relevant to neuronal computation in a majority of brain areas, and we encourage other researchers to apply similar tests when using trial-averaged neuronal metrics.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.22.469473', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genomic regions associated with chocolate spot (Botrytis fabae Sard.) resistance in faba bean (Vicia faba L.)', 'authors': \"[{'name': 'Tadesse S. Gela'}, {'name': 'Margaret Bruce'}, {'name': 'Wei Chang'}, {'name': 'Frederick L. Stoddard'}, {'name': 'Alan H. Schulman'}, {'name': 'Albert Vandenberg'}, {'name': 'Hamid Khazaei'}]\", 'abstract': '<p>Chocolate spot (CS), caused by <italic>Botrytis fabae</italic> Sard., is an important threat to global faba bean production. Growing resistant faba bean cultivars is, therefore, paramount to preventing yield loss. To date, there have been no reported quantitative trait loci (QTLs) associated with CS resistance in faba bean. The objective of this study was to identify genomic regions associated with CS resistance using a recombinant inbred line (RIL) population derived from resistant accession ILB 938. A total of 165 RILs from the cross Mlodie/2  ILB 938/2 were genotyped and evaluated for CS reactions under replicated controlled climate conditions. QTL analysis identified five loci contributing to CS resistance on faba bean chromosomes 1 and 6, accounting for 5.023.4% of the total phenotypic variance. The sequences of SNP markers linked to resistance QTLs on chromosome 1 that have the largest effects encode multiple classes of putative disease and/or defense-related genes. The results of this study not only provide insight into disease-resistance QTLs, but also can be used as potential targets for marker-assisted breeding in faba bean genetic improvement for CS resistance.</p><sec><title>Key message</title><p>QTL mapping identified key genomic regions associated with chocolate spot resistance on faba bean chromosomes 1 and 6, which may serve as novel genetic tools to develop disease-resistant faba bean cultivars.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469463', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Jointly Optimized Spatial Histogram UNET Architecture (JOSHUA) for Adipose Tissue Segmentation', 'authors': \"[{'name': 'Joshua K. Peeples'}, {'name': 'Julie F. Jameson'}, {'name': 'Nisha M. Kotta'}, {'name': 'Jonathan M. Grasman'}, {'name': 'Whitney L. Stoppel'}, {'name': 'Alina Zare'}]\", 'abstract': '<sec><title>Objective</title><p>We quantify adipose tissue deposition at surgical sites as a function of biomaterial implantation.</p></sec><sec><title>Impact Statement</title><p>To our knowledge, this study is the first investigation to apply convolutional neural network (CNN) models to identify and segment adipose tissue in histological images from silk fibroin biomaterial implants.</p></sec><sec><title>Introduction</title><p>When designing biomaterials for the treatment of various soft tissue injuries and diseases, one must consider the extent of adipose tissue deposition. In this work, we implant silk fibroin biomaterials in a rodent subcutaneous injury model. Current strategies for quantifying adipose tissue after biomaterial implantation are often tedious and prone to human bias during analysis.</p></sec><sec><title>Methods</title><p>We used CNN models with novel spatial histogram layer(s) that can more accurately identify and segment regions of adipose tissue in hematoxylin and eosin (H&amp;E) and Massons Trichrome stained images, allowing for determination of the optimal biomaterial formulation. We compared the method, Jointly Optimized Spatial Histogram UNET Architecture (JOSHUA), to the baseline UNET model and an extension of the baseline model, Attention UNET, as well as to versions of the models with a supplemental attention-inspired mechanism (JOSHUA+ and UNET+).</p></sec><sec><title>Results</title><p>The inclusion of histogram layer(s) in our models shows improved performance through qualitative and quantitative evaluation.</p></sec><sec><title>Conclusion</title><p>Our results demonstrate that the proposed methods, JOSHUA and JOSHUA+, are highly beneficial for adipose tissue identification and localization. The new histological dataset and code for our experiments are publicly available.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.21266688', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'SARS-CoV-2 genome-based severity predictions correspond to lower qPCR values and higher viral load', 'authors': \"[{'name': 'Martin Skarzynski'}, {'name': 'Erin M. McAuley'}, {'name': 'Ezekiel J. Maier'}, {'name': 'Anthony C. Fries'}, {'name': 'Jameson D. Voss'}, {'name': 'Richard R. Chapleau'}]\", 'abstract': '<p>The 2019 coronavirus disease (COVID-19) pandemic has demonstrated the importance of predicting, identifying, and tracking mutations throughout a pandemic event. As the COVID-19 global pandemic surpassed one year, several variants had emerged resulting in increased severity and transmissibility. In order to reduce the impact on human life, it is critical to rapidly identify which genetic variants result in increased virulence or transmission. To address the former, we evaluated if a genome-based predictive algorithm designed to predict clinical severity could predict polymerase chain reaction (PCR) results, as a surrogate for viral load and severity. Using a previously published algorithm, we compared the viral genome-based severity predictions to clinically-derived PCR-based viral load of 716 viral genomes. For those samples predicted to be severe (predicted severity score &gt; 0.5), we observed an average cycle threshold (Ct) of 18.3, whereas those in in the mild category (severity prediction &lt; 0.5) had an average Ct of 20.4 (<italic>P</italic> = 0.0017). We found a non-trivial correlation between predicted severity probability and cycle threshold (<italic>r</italic> = 0.199). Additionally, when divided into quartiles by prediction severity probability, the most probable quartile (75% probability) had a Ct of 16.6 (n=10) as compared to those least probable to be severe (&lt;25%) of 21.4 (n=350) (<italic>P</italic> = 0.0045). Taken together, our results suggest that the severity predicted by a genome-based algorithm can be related to the metrics from the clinical diagnostic test, and that relative severity may be inferred from diagnostic values.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.469461', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Impaired astrocytic Ca2+ signalling in awake Alzheimers disease transgenic mice', 'authors': \"[{'name': 'Knut Sindre bjrsbrten'}, {'name': 'Gry H. E. Syverstad Skaaraas'}, {'name': 'Cline Cunen'}, {'name': 'Daniel M. Bjrnstad'}, {'name': 'Kristin M. Gullestad Binder'}, {'name': 'Vidar Jensen'}, {'name': 'Lars N.G. Nilsson'}, {'name': 'Shreyas B. Rao'}, {'name': 'Wannan Tang'}, {'name': 'Gudmund Horn Hermansen'}, {'name': 'Erlend A. Nagelhus'}, {'name': 'Ole Petter Ottersen'}, {'name': 'Reidun Torp'}, {'name': 'Rune Enger'}]\", 'abstract': '<p>Increased astrocytic Ca<sup>2+</sup> signaling related to amyloid plaques has been shown in Alzheimers disease mouse models, but to date no reports have characterized behaviorally induced astrocytic Ca<sup>2+</sup> signalling in such mice without the confounding effects of anesthesia. Here, we employ an event-based algorithm to assess astrocytic Ca<sup>2+</sup> signals in the neocortex of awake-behaving tg-ArcSwe mice and non-transgenic wildtype littermates while monitoring pupil responses and behavior. We demonstrate an attenuated astrocytic Ca<sup>2+</sup> response to locomotion and an uncoupling of pupil responses and astrocytic Ca<sup>2+</sup> signalling in 15-months old plaque-bearing mice. This points to a potential decoupling of neuromodulatory activation and astrocytic Ca<sup>2+</sup> activity, which may account for some of the cognitive dysfunctions observed in Alzheimers disease.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469511', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Adaptive coding of stimulus information in human fronto-parietal cortex during visual classification', 'authors': \"[{'name': 'David Wisniewski'}, {'name': 'Carlos Gonzlez-Garca'}, {'name': 'Silvia Formica'}, {'name': 'Alexandra Woolgar'}, {'name': 'Marcel Brass'}]\", 'abstract': '<p>Our ability to flexibly adapt to changing demands is supported by flexible coding of task-relevant information in frontal and parietal brain regions. Converging evidence suggest that coding of stimuli and task rules in these regions become stronger as task difficulty increases. Here, we tested whether there is a corresponding change in the representational format as well, an issue that has rarely been addressed directly in past research. Participants performed a visual classification task under varying levels of perceptual difficulty, while we acquired fMRI. Using a model-based representational similarity approach, we tested whether stimulus representations retain exemplar-level information. We expected representations to drop such exemplar-level information as perceptual difficulty increases, which would indicate a focus on representing behaviorally relevant category information. Counter to these expectations, and in contrast to previous research, we found frontal and parietal brain regions contained exemplar-level stimulus information. Interestingly, the anterior intraparietal sulcus (aIPS) retained exemplar-level stimulus information even in perceptually difficult trials, and these representations were directly related to performance. Overall, these findings call for a reassessment of the neural mechanisms underlying human adaptive behavior during visual classification.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469492', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'A multi-class gene classifier for SARS-CoV-2 variants based on convolutional neural network', 'authors': \"[{'name': 'Junyu Fan'}, {'name': 'Chutao Chen'}, {'name': 'Chen Song'}, {'name': 'Jiajie Pan'}, {'name': 'Guifu Wu'}]\", 'abstract': '<p>Surveillance of circulating variants of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is of great importance in controlling the coronavirus disease 2019 (COVID-19) pandemic. We propose an alignment-free in silico approach for classifying SARS-CoV-2 variants based on their genomic sequences. A deep learning model was constructed utilizing a stacked 1-D convolutional neural network and multilayer perceptron (MLP). The pre-processed genomic sequencing data of the four SARS-CoV-2 variants were first fed to three stacked convolution-pooling nets to extract local linkage patterns in the sequences. Then a 2-layer MLP was used to compute the correlations between the input and output. Finally, a logistic regression model transformed the output and returned the probability values. Learning curves and stratified 10-fold cross-validation showed that the proposed classifier enables robust variant classification. External validation of the classifier showed an accuracy of 0.9962, precision of 0.9963, recall of 0.9963 and F1 score of 0.9962, outperforming other machine learning methods, including logistic regression, K-nearest neighbor, support vector machine, and random forest. By comparing our model with an MLP model without the convolution-pooling network, we demonstrate the essential role of convolution in extracting viral variant features. Thus, our results indicate that the proposed convolution-based multi-class gene classifier is efficient for the variant classification of SARS-CoV-2.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469626', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Host tissue proteomics reveal insights into the molecular basis of Schistosoma haematobium-induced bladder pathology', 'authors': \"[{'name': 'Derick N. M. Osakunor'}, {'name': 'Kenji Ishida'}, {'name': 'Olivia K. Lamanna'}, {'name': 'Mario Rossi'}, {'name': 'Michael H. Hsieh'}]\", 'abstract': '<sec><title>Background</title><p>Urogenital schistosomiasis remains a major public health concern worldwide. In response to egg deposition, the host bladder undergoes gross and molecular morphological changes relevant for disease manifestation. However, limited mechanistic studies to date imply that the molecular mechanisms underlying pathology have not been well-defined. We leveraged a mouse model of urogenital schistosomiasis to perform for the first time, proteome profiling of the early molecular events that occur in the bladder after exposure to <italic>S. haematobium</italic> eggs, and to elucidate the protein pathways involved in urogenital schistosomiasis-induced pathology.</p></sec><sec><title>Methods</title><p>Purified <italic>S. haematobium</italic> eggs or control vehicle were microinjected into the bladder walls of mice. Mice were sacrificed seven days post-injection and bladder proteins isolated and processed for proteome profiling using mass spectrometry.</p></sec><sec><title>Results</title><p>We demonstrate that biological processes including carcinogenesis, immune and inflammatory responses, increased protein translation or turnover, oxidative stress responses, reduced cell adhesion and epithelial barrier integrity, and increased glucose metabolism were significantly enriched in <italic>S. haematobium</italic> infection.</p></sec><sec><title>Conclusion</title><p><italic>S. haematobium</italic> egg deposition in the bladder results in significant changes in proteins and pathways that play a role in pathology. Our findings highlight the potential bladder protein indicators for host-parasite interplay and provide new insights into the complex dynamics of pathology and characteristic bladder tissue changes in urogenital schistosomiasis. The findings will be relevant for development of improved interventions for disease control.</p></sec><sec><title>Author summary</title><p>The molecular mechanisms underlying the urinary and genital pathology from urogenital schistosomiasis have not been well-defined. This has mainly been due to limited mechanistic studies and the lack of a suitable animal model for <italic>S. haematobium</italic> infection. We leveraged a mouse model of urogenital schistosomiasis, along with proteomics analysis, to determine the early molecular events that occur in the bladder after exposure to <italic>S. haematobium</italic> eggs, and to define protein pathways that may be involved in morbidity from urogenital schistosomiasis infection. Our results show that <italic>S. haematobium</italic> egg deposition in the bladder results in significant changes in proteins and pathways that play a role in pathology. Biological processes including carcinogenesis, immune and inflammatory responses, increased protein translation or turnover, oxidative stress responses, reduced cell adhesion and epithelial barrier integrity, and increased glucose metabolism are significantly enriched in <italic>S. haematobium</italic> infection. Our findings highlight important protein indicators of host-parasite interactions in the bladder and their association with tissue changes in urogenital schistosomiasis. The findings presented here will be relevant for development of improved interventions for disease control, and contribute to realizing the 2021-2030 NTD goals by demonstrating evidence on the early events that occur during schistosome infection..</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.24.469921', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A fully automated FAIMS-DIA proteomic pipeline for high-throughput characterization of iPSC-derived neurons', 'authors': \"[{'name': 'Luke Reilly'}, {'name': 'Lirong Peng'}, {'name': 'Erika Lara'}, {'name': 'Daniel Ramos'}, {'name': 'Michael Fernandopulle'}, {'name': 'Caroline B. Pantazis'}, {'name': 'Julia Stadler'}, {'name': 'Marianita Santiana'}, {'name': 'Anant Dadu'}, {'name': 'James Iben'}, {'name': 'Faraz Faghri'}, {'name': 'Mike A. Nalls'}, {'name': 'Steven L. Coon'}, {'name': 'Priyanka Narayan'}, {'name': 'Andrew B. Singleton'}, {'name': 'Mark R. Cookson'}, {'name': 'Michael E. Ward'}, {'name': 'Yue A. Qi'}]\", 'abstract': '<p>Fully automated proteomic pipelines have the potential to achieve deep coverage of cellular proteomes with high throughput and scalability. However, it is important to evaluate performance, including both reproducibility and ability to provide meaningful levels of biological insight. Here, we present an approach combining high field asymmetric waveform ion mobility spectrometer (FAIMS) interface and data independent acquisition (DIA) proteomics approach developed as part of the induced pluripotent stem cell (iPSC) Neurodegenerative Disease Initiative (iNDI), a large-scale effort to understand how inherited diseases may manifest in neuronal cells. Our FAIMS-DIA approach identified more than 8000 proteins per mass spectrometry (MS) acquisition as well as superior total identification, reproducibility, and accuracy compared to other existing DIA methods. Next, we applied this approach to perform a longitudinal proteomic profiling of the differentiation of iPSC-derived neurons from the KOLF2.1J parental line used in iNDI. This analysis demonstrated a steady increase in expression of mature cortical neuron markers over the course of neuron differentiation. We validated the performance of our proteomics pipeline by comparing it to single cell RNA-Seq datasets obtained in parallel, confirming expression of key markers and cell type annotations. An interactive webapp of this temporal data is available for aligned-UMAP visualization and data browsing (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://share.streamlit.io/anant-droid/singlecellumap\">https://share.streamlit.io/anant-droid/singlecellumap</ext-link>). In summary, we report an extensively optimized and validated proteomic pipeline that will be suitable for large-scale studies such as iNDI.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.23.469651', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'SmdA is a novel cell morphology determinant in Staphylococcus aureus', 'authors': \"[{'name': 'Ine S. Myrbrten'}, {'name': 'Gro A. Stamss'}, {'name': 'Helena Chan'}, {'name': 'Danae Morales Angeles'}, {'name': 'Tiril Mathiesen Knutsen'}, {'name': 'Zhian Salehian'}, {'name': 'Volha Shapaval'}, {'name': 'Daniel Straume'}, {'name': 'Morten Kjos'}]\", 'abstract': '<p>Cell division and cell wall synthesis in staphylococci need to be precisely coordinated and controlled to allow the cell to multiply while maintaining their nearly spherical shape. The mechanisms ensuring correct placement of the division plane and synthesis of new cell wall have been studied intensively, however, hitherto unknown factors and proteins are likely to play key roles in this complex interplay. We here identified and investigated a protein with major influence on cell morphology in <italic>Staphylococcus aureus</italic>. The protein, named SmdA (for staphylococcal morphology determinant A), is a membrane-protein with septum-enriched localization. By CRISPRi knockdown and overexpression combined with different microscopy techniques, we demonstrate that proper levels of SmdA is necessary for cell division, including septum formation and cell splitting. We also identified conserved residues in SmdA that are critical for its functionality. Pulldown- and bacterial two-hybrid interaction experiments showed that SmdA interacts with several known cell division- and cell wall synthesis proteins, including penicillin binding proteins (PBPs) and EzrA. Notably, SmdA also affects susceptibility to cell wall targeting antibiotics, particularly in methicillin-resistant <italic>S. aureus</italic> (MRSA). Together, our results show that <italic>S. aureus</italic> is dependent on balanced amounts of membrane-attached SmdA in order to carry out proper cell division.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.468991', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'An allelic series of spontaneous mutations in Rorb cause a gait phenotype, retinal abnormalities, and transcriptomic changes relevant to human neurodevelopmental conditions', 'authors': \"[{'name': 'George C Murray'}, {'name': 'Jason Bubier'}, {'name': 'Oraya J Zinder'}, {'name': 'Belinda Harris'}, {'name': 'James Clark'}, {'name': 'Mia-Cara Christopher'}, {'name': 'Courtany Hanley'}, {'name': 'Harianto Tjong'}, {'name': 'Meihong Li'}, {'name': 'Chew Yee Ngan'}, {'name': 'Laura Reinholdt'}, {'name': 'Robert W Burgess'}, {'name': 'Abby LD Tadenev'}]\", 'abstract': '<p>Rorb encodes the Retinoic Acid Receptor-related orphan receptor beta. Mutations in either of the two transcripts of Rorb cause defects in multiple systems, including abnormal photoreceptor abundance and morphology in the retina and a characteristic high-stepper or duck-like gait arising from dysfunction of interneurons in the spinal cord. Rorb is also important for cortical development and cell fate specification in mice. Rorb variants segregate with epilepsy and comorbidities such as intellectual disability in numerous clinical cases. Here we describe five mouse strains with spontaneous mutations in Rorb identified by their gait phenotype. These mutations affect different domains and isoforms of Rorb, which correspond to the spectrum of anatomical and physiological phenotypes exhibited by these mice. Gene set analysis in Rorb mutants implicates pathways associated with development and nervous system function, and differential gene expression analysis indicates changes in numerous genes related to epilepsy, bipolar disorder, and autism spectrum disorder (ASD). Many of these genes and their protein products are known to interact during synapse formation and neuronal activity. These findings further illuminate the role of Rorb in nervous system development, provide further evidence for an association between Rorb and several neurological conditions, and describe an allelic series of Rorb mutant mice that will be useful for dissecting thalamocortical afferent(TCA) development, neural cell fate determination, and as animal models exhibiting transcriptomic shifts in neurological conditions such as epilepsy, bipolar disorder, and ASD.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.27.470071', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Archaic introgression and variation in pharmacogenes and implications for local adaptation', 'authors': \"[{'name': 'Tadeusz H Wroblewski'}, {'name': 'Kelsey E Witt'}, {'name': 'Seung-been Lee'}, {'name': 'Ripan S Malhi'}, {'name': 'Emilia Huerta-Sanchez'}, {'name': 'Fernando Villanea'}, {'name': 'Katrina G Claw'}]\", 'abstract': '<p>Modern humans carry Neanderthal and Denisovan (archaic) genome elements which may have been a result of environmental adaptation. These effects may be particularly evident in pharmacogenes - genes responsible for the processing of exogenous substances such as food, pollutants, and medications. However, the health implications and contribution of archaic ancestry in pharmacogenes of modern humans remains understudied. We characterize eleven key cytochrome P450 (CYP450) genes involved in drug metabolizing reactions in three Neanderthal and one Denisovan individuals and examine archaic introgression in modern human populations. We infer the metabolizing efficiency of these eleven genes in archaic individuals and show important genetic differences relative to modern human variants. We identify archaic-specific SNVs in each CYP450 gene, including some that are potentially damaging, which may result in altered metabolism in modern human people carrying these variants. We highlight four genes which display interesting patterns of archaic variation: CYP2B6 - we find a large number of unique variants in the Vindija Neanderthal, some of which are shared with a small subset of African modern humans; CYP2C9 - containing multiple variants that are shared between Europeans and Neanderthals; CYP2A6*12 - a variant defined by a hybridization event that was found in humans and Neanderthals, suggesting the recombination event predates both species; and CYP2J2 - in which we hypothesize a Neanderthal variant was re-introduced in non-African populations by archaic admixture. The genetic variation identified in archaic individuals imply environmental pressures that may have driven CYP450 gene evolution.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.23.21266717', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Addiction Medicine', 'covid': False, 'title': 'Identifying high-risk comorbidities of short and long-term opioid prescription use', 'authors': \"[{'name': 'Mariela V Jennings'}, {'name': 'Hyunjoon Lee'}, {'name': 'Daniel B Rocha'}, {'name': 'Sevim B Bianchi'}, {'name': 'Brandon J Coombes'}, {'name': 'Richard C Crist'}, {'name': 'Annika Faucon'}, {'name': 'Yirui Hu'}, {'name': 'Rachel L Kember'}, {'name': 'Travis T Mallard'}, {'name': 'Maria Niarchou'}, {'name': 'Melissa N Poulsen'}, {'name': 'Peter Straub'}, {'name': 'Richard D Urman'}, {'name': 'Colin G Walsh'}, {'name': 'Lea K Davis'}, {'name': 'Jordan W Smoller'}, {'name': 'Vanessa Troiani'}, {'name': 'Sandra Sanchez-Roige'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>Electronic health records (EHR) are useful tools for understanding complex medical phenotypes, but they have been underutilized for opioid use disorders (OUD). Patterns of prescription opioid use might provide an objective measure of OUD risk.</p></sec><sec><title>Methods</title><p>We extracted data for over 2.6 million patients across three health registries (Vanderbilt University Medical Center, Mass General Brigham, Geisinger) between 2005 and 2018. We defined three groups based on levels of opioid exposure: No Prescription, Minimal Exposure (2 prescriptions within 90 days at least once, but never 3 prescriptions &lt;90 days apart), and Chronic Exposure (10 opioid prescriptions in a year), and compared them to the full registries and to patients with OUD diagnostic codes. We extracted demographic and clinical characteristics known to co-occur with OUD, including psychiatric and substance use disorders, pain-related diagnoses, HIV, and hepatitis C.</p></sec><sec><title>Results</title><p>The prevalence of substance (alcohol, tobacco, cannabis) use disorders was higher in patients with OUD and Chronic Exposure than those with No Prescription or Minimal Exposure. Patients in the OUD and Chronic Exposure groups had more psychiatric (anxiety, depression, schizophrenia, bipolar disorder) and medical comorbidities (pain, hepatitis C, HIV) than those in the Minimal Exposure group. Notably, patients in the Minimal Exposure group had different comorbidity profiles (higher rates of substance use and psychiatric disorders, more pain conditions) than those in the Unscreened or No Prescription groups, highlighting the value of including opioid exposure in studies of OUD.</p></sec><sec><title>Conclusions</title><p>Long-term opioid prescription use may serve as an additional tool to characterize OUD risk.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469474', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'A chromosome-level reference genome of Ensete glaucum gives insight into diversity, chromosomal and repetitive sequence evolution in the Musaceae', 'authors': \"[{'name': 'Ziwei Wang'}, {'name': 'Mathieu Rouard'}, {'name': 'Manosh Kumar Biswas'}, {'name': 'Gaetan Droc'}, {'name': 'Dongli Cui'}, {'name': 'Nicolas Roux'}, {'name': 'Franc-Christophe Baurens'}, {'name': 'Xue-Jun Ge'}, {'name': 'Trude Schwarzacher'}, {'name': 'Pat (J.S.) Heslop-Harrison'}, {'name': 'Qing Liu'}]\", 'abstract': '<sec><title>Background</title><p><italic>Ensete glaucum</italic> (2<italic>n</italic> = 2<italic>x</italic> = 18) is a giant herbaceous monocotyledonous plant in the small Musaceae family along with banana (<italic>Musa</italic>). A high-quality reference genome sequence of <italic>E. glaucum</italic> offers a vital genomic resource for functional and evolutionary studies of <italic>Ensete</italic>, the Musaceae, and more widely in the Zingiberales.</p></sec><sec><title>Findings</title><p>Using a combination of Illumina and Oxford Nanopore Technologies (ONT) sequencing, genome-wide chromosome conformation capture (Hi-C), and RNA survey sequence, we report a high-quality assembly of the 481.5Mb genome with 9 pseudochromosomes and 36,836 genes (BUSCO 94.7%). A total of 55% of the genome is composed of repetitive sequences with LTR-retroelements (37%) and DNA transposons (7%) predominant. The 5S and 45S rDNA were each present at one locus, and the 5S rDNA had an exceptionally long monomer length of c.1,056 bp, contrasting with the c. 450 bp monomer at multiple loci in <italic>Musa</italic>. A tandemly repeated c. 134 bp satellite, 1.1% of the genome (with no similar sequence in <italic>Musa</italic>), was present around all nine centromeres, with a LINE retroelement also found at <italic>Musa</italic> centromeres. The assembly, including centromeric positions, enabled us to characterize in detail the chromosomal rearrangements occurring between the <italic>x</italic> = 9 species and <italic>x</italic> = 11 species of <italic>Musa</italic>. Only one chromosome has the same gene content as <italic>M. acuminata</italic> (ma). Three ma chromosomes represent part of only one <italic>E. glaucum</italic> (eg) chromosome, while the remaining seven ma chromosomes are fusions of parts of two, three, or four eg chromosomes, demonstrating complex and multiple evolutionary rearrangements in the change between <italic>x</italic> = 9 and <italic>x</italic> = 11.</p></sec><sec><title>Conclusions</title><p>The advance towards a Musaceae pangenome including <italic>E. glaucum,</italic> tolerant of extreme environments, makes a complete set of gene alleles available for crop breeding and understanding environmental responses. The chromosome-scale genome assembly show how chromosome number evolves, and features of the rapid evolution of repetitive sequences.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469765', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'SARS-CoV-2 Delta Spike Protein Enhances the Viral Fusogenicity and Inflammatory Cytokine Production', 'authors': \"[{'name': 'Zhujun Ao'}, {'name': 'Maggie Jing Ouyang'}, {'name': 'Titus Abiola Olukitibi'}, {'name': 'Xiaojian Yao'}]\", 'abstract': '<p>The Delta variant is now the most dominant and virulent SARS-CoV-2 variant of concern (VOC). In this study, we investigated several virological features of Delta spike protein (SP<sub>Delta</sub>), including protein maturation and its impact on viral entry of cell-free pseudotyped virus, cell-cell fusion ability and its induction of inflammatory cytokine production in human macrophages and dendritic cells. The results showed that SPC<sub>Delta</sub> exhibited enhanced S1/S2 cleavage in cells and pseudotyped virus-like particles (PVLPs). We further showed that SPC<sub>Delta</sub> elevated pseudovirus infection in human lung cell lines and mediated significantly enhanced syncytia formation. Furthermore, we revealed that SPC<sub>Delta</sub>-PVLPs had stronger effects on stimulating NF-B and AP-1 signaling in human monocytic THP1 cells and induced significantly higher levels of pro-inflammatory cytokine, such as TNF-, IL-1 and IL-6, released from human macrophages and dendritic cells. Overall, these studies provide evidence to support the important role of SPC<sub>Delta</sub> during virus infection, transmission and pathogenesis.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469702', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'High-quality reference genome for an arid-adapted mammal, the banner-tailed kangaroo rat (Dipodomys spectabilis)', 'authors': \"[{'name': 'Avril M. Harder'}, {'name': 'Kimberly K.O. Walden'}, {'name': 'Nicholas J. Marra'}, {'name': 'Janna R. Willoughby'}]\", 'abstract': '<p>Kangaroo rats in the genus <italic>Dipodomys</italic> are found in a variety of habitat types in western North America, including deserts, arid and semi-arid grasslands, and scrublands. Many <italic>Dipodomys</italic> species are experiencing strong population declines due to increasing habitat fragmentation, with two species listed as federally endangered. The precarious state of many <italic>Dipodomys</italic> populations, including those occupying extreme environments, make species of this genus valuable subjects for studying the impacts of habitat degradation and fragmentation on population genomic patterns and for characterizing the genomic bases of adaptation to harsh conditions. To facilitate exploration of such questions, we assembled and annotated a reference genome for the banner-tailed kangaroo rat (<italic>D. spectabilis</italic>) using PacBio HiFi sequencing reads, providing a more contiguous genomic resource than two previously assembled <italic>Dipodomys</italic> genomes. Using the HiFi data for <italic>D. spectabilis</italic> and publicly available sequencing data for two other <italic>Dipodomys</italic> species (<italic>D. ordii</italic> and <italic>D. stephensi</italic>), we demonstrate the utility of this new assembly for studies of congeners by conducting inference of historic effective population sizes (<italic>N</italic><sub>e</sub>) and linking these patterns to the species current extinction risk statuses. The genome assembly presented here will serve as a valuable resource for population and conservation genomic studies of <italic>Dipodomys</italic> species, comparative genomic research within mammals and rodents, and investigations into genomic adaptation to extreme environments and changing landscapes.</p><sec><title>Significance statement</title><p>Kangaroo rats in the genus <italic>Dipodomys</italic> occur in a wide variety of habitat types, ranging from scrublands to arid deserts, and are increasingly impacted by habitat fragmentation with populations of many species in strong decline. To facilitate population and conservation genomic studies of <italic>Dipodomys</italic> species, we generated the first reference genome assembly for the extensively studied banner-tailed kangaroo rat (<italic>D. spectabilis</italic>) from long read PacBio sequencing data. The genome assembly presented here will serve as a valuable resource for studies of <italic>Dipodomys</italic> specieswhich have long served as ecological and physiological models for the study of osmoregulationcomparative genomic surveys of mammals and rodents, and investigations into genomic adaptation to extreme environments and changing landscapes.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469804', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Three heads are better than one: Cooperative learning brains wire together when a consensus is reached', 'authors': \"[{'name': 'Yafeng Pan'}, {'name': 'Xiaojun Cheng'}, {'name': 'Yi Hu'}]\", 'abstract': '<p>Theories of human learning converge on the view that individuals working together learn better than do those working alone. Little is known, however, about the neural mechanisms of learning through cooperation. We addressed this research gap by leveraging functional near-infrared spectroscopy (fNIRS) to record the brain activity of triad members in a group simultaneously. Triads were instructed to analyze an ancient Chinese poem either cooperatively or independently. Four main findings emerged. First, we observed significant within-group neural synchronization (GNS) in the left superior temporal cortex, supramarginal gyrus, and postcentral gyrus during cooperative learning compared to independent learning. Second, the enhancement of GNS in triads was amplified when a consensus was reached (vs. elaboration or argument) during cooperative learning. Third, GNS was predictive of learning outcome at an early stage (156-170 s after learning was initiated). Fourth, social factors such as social closeness (e.g., how much learners liked one other) were reflected in GNS and co-varied with learning engagement. These results provide neurophysiological support for Piagets theory of cognitive development and favor the notion that successful learning through cooperation involves dynamic consensus building, which is captured in neural patterns shared across learners in a group.</p><sec><title>Significance Statement</title><p>Converging evidence has shown that cooperative learning is more effective than independent learning. An influential pedagogical theory postulates that learners benefit from cooperation through different forms of cognitive elaboration, such as providing elaborated clarifications to others. Alternatively, Piagets theory of cognitive development posits that cooperation encourages learners with diverse opinions to reach a consensus during the learning process. Here, we report that unlike individuals who worked alone, the brains of students who worked cooperatively with one another became synchronized. This within-group neural synchronization (GNS) was magnified when learners built mutual consensuses. These findings suggest that successful cooperative learning involves dynamic consensus building, which is reflected in the interpersonal coordination of cerebral activity.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469557', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Activation of peripheral neuronal FLT3 promotes exaggerated sensorial and emotional pain-related behaviors facilitating the transition from acute to chronic pain', 'authors': \"[{'name': 'Adrien Tassou'}, {'name': 'Maxime Thouaye'}, {'name': 'Damien Gilabert'}, {'name': 'Antoine Jouvenel'}, {'name': 'Jean-Philippe Leyris'}, {'name': 'Corinne Sonrier'}, {'name': 'Lucie Diouloufet'}, {'name': 'Ilana Mechaly'}, {'name': 'Myriam Chentouf'}, {'name': 'Madeline Neiveyans'}, {'name': 'Martine Pugnire'}, {'name': 'Pierre Martineau'}, {'name': 'Bruno Robert'}, {'name': 'Xavier Capdevila'}, {'name': 'Jean Valmier'}, {'name': 'Cyril Rivat'}]\", 'abstract': '<sec><title>Background</title><p>Acute pain events have been associated with persistent pain sensitization of nociceptive pathways increasing the risk of transition from acute to chronic pain. However, it is unclear whether injury-induced persistent pain sensitization can promote long-term mood disorders. The receptor tyrosine kinase FLT3 is causally required for peripheral nerve injury-induced pain chronification, questioning its role in the development of pain-induced mood alterations.</p></sec><sec><title>Methods</title><p>In a model of paw incisional pain, mice underwent single (SI) or double incision (DI) and went through behavioral and molecular phenotyping with the evaluation of both sensorial and emotional pain components. The role of FLT3 was then investigated either by inhibition using transgenic knock-out mice and functional antibodies or by activation with FLT3 ligand (FL) administrations.</p></sec><sec><title>Results</title><p>DI mice showed significant anxiodepressive-like and spontaneous pain behaviors while SI mice did not. DI also promoted and extended mechanical pain hypersensitivity compared to SI. This emotional and sensorial pain exaggeration correlated with a potentiation of spinal microglial activation after DI versus SI. Intrathecal minocycline, a microglial inhibitor, specifically reversed DI induced-mechanical hypersensitivity. Finally, FL injections in naive animals provoked mechanical allodynia and anxiodepressive-like disorders concomitant with a significant microglial activation while FLT3 inhibition blunted the development of persistent pain and depression after DI.</p></sec><sec><title>Conclusions</title><p>Our results show for the first time that the repetition of a peripheral lesion facilitates not only exaggerated nociceptive behaviors but also anxiodepressive disorders. The inhibition of FLT3 could become a promising therapy in the management of pain sensitization and related mood alterations.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469507', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Biomechanical characteristics of scapula and glenohumeral movements during pitching motion in injury-prone college baseball pitchers', 'authors': \"[{'name': 'Koji Miyashita'}, {'name': 'Sentaro Koshida'}, {'name': 'Taro Koyama'}, {'name': 'Kenicihro Ota'}, {'name': 'Yusuke Tani'}, {'name': 'Ryoji Okamune'}]\", 'abstract': '<p>Coordination of glenohumeral and scapular movements plays an important role in the injury prevention of baseball pitchers. However, there is no objective data establishing the direct relationship between pitching injuries and associated glenohumeral and scapular movements. Therefore, the objectives of the present study were to demonstrate biomechanical differences in scapular and glenohumeral movements during pitching between injury-prone pitchers and healthy college baseball pitchers. Thirty collegiate baseball pitchers were classified into two groups according to their injury status: injury-prone group (N=15, 20.71.4 years, 180.16.5 cm, 78.95.4 kg) and control group (N=15, 20.91.1 years, 177.16.6 cm, 72.36.7 kg). We obtained the pitching motion data using the three-dimensional motion analysis technique with four high-speed cameras. The horizontal abduction angles of the glenohumeral joint during cocking and acceleration phases were significantly greater in injury-prone pitchers [19.0 (95% CI: 14.423.6) at foot contact, 4.0 (95% CI: 7.7 to 0.2) at maximum external rotation (MER), and 0.3 (95% CI: 4.8 to 4.2) at ball release] than in healthy controls [11.7 (95%CI:7.1 to 16.3) at foot contact, 10.0(95%CI: 13.7 to 6.3) at MER, and 6.9(95%CI: 11.4 to 2.4)](<italic>p</italic>&lt;0.01). In addition, the external rotation angle (ER) of the scapula at MER was significantly greater in the injury-prone group [0.1 (95% CI: 5.0 to 4.8)] than in the control group [12.3 (95% CI: 17.2 to 7.4)] (p&lt;0.01), but there was no difference in the scapular ER during foot contact between the two groups. These results suggests that injury-prone pitchers have less internal rotation of the scapula and more horizontal abduction of the glenohumeral joint during cocking and acceleration phases. Therefore, sports medicine practitioners may need to pay more attention to coordination of scapular and glenohumeral movements during the cocking and acceleration phases of pitching for prevention of shoulder injuries.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469117', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Preclinical efficacy, safety, and immunogenicity of PHH-1V, a second-generation COVID-19 vaccine candidate based on a novel recombinant RBD fusion heterodimer of SARS-CoV-2', 'authors': \"[{'name': 'Antonio Barreiro'}, {'name': 'Antoni Prenafeta'}, {'name': 'Gregori Bech-Sabat'}, {'name': 'Merc Roca'}, {'name': 'Ricard March'}, {'name': 'Luis Gonzlez'}, {'name': 'Laia Madrenas'}, {'name': 'Jlia Corominas'}, {'name': 'lex Fernndez'}, {'name': 'Merc Molas'}, {'name': 'Jlia Vergara-Alert'}, {'name': 'Cristina Lorca-Or'}, {'name': 'Nria Roca'}, {'name': 'Leira Fernndez-Bastit'}, {'name': 'Jordi Rodon'}, {'name': 'Mnica Prez'}, {'name': 'Joaquim Segals'}, {'name': 'Edwards Pradenas'}, {'name': 'Silvia Marfil'}, {'name': 'Benjamin Trinit'}, {'name': 'Raquel Ortiz'}, {'name': 'Bonaventura Clotet'}, {'name': 'Juli Blanco'}, {'name': 'Jorge Daz Pedroza'}, {'name': 'Rosa Ampudia Carrasco'}, {'name': 'Yaiza Rosales Salgado'}, {'name': 'Jordina Loubat Casanovas'}, {'name': 'Sara Capdevila Larripa'}, {'name': 'Julia Garcia Prado'}, {'name': 'Jordi Barretina'}, {'name': 'Marta Sister-Or'}, {'name': 'Paula Cebollada Rica'}, {'name': 'Andreas Meyerhans'}, {'name': 'Laura Ferrer'}, {'name': 'Elia Torroella'}]\", 'abstract': '<p>Since the genetic sequence of SARS-CoV-2 became available in January 2020, new vaccines have been developed at an unprecedented speed. The current vaccines have been directly associated with a decline in new infection rates, prevention of severe disease and an outstanding decrease in mortality rates. However, the pandemic is still far from being over. New Variants of Concern (VoCs) are continuously evolving. Thus, it is essential to develop accessible second-generation COVID-19 vaccines against known and future VoCs to mitigate the current pandemic. Here, we provide preclinical data showing the immunogenicity, efficacy, and safety results in mice of a receptor-binding domain (RBD)-based recombinant protein vaccine candidate (PHH-1V) which consists of a novel RBD fusion heterodimer containing the B.1.1.7 (alpha) and B.1.351 (beta) variants of SARS-CoV-2, formulated with an oil-based adjuvant equivalent to MF59C.1. BALB/c and K18-hACE2 mice were immunized with different doses of recombinant RBD fusion heterodimer, following a two-dose prime-and-boost schedule. Upon 20 g RBD fusion heterodimer/dose immunization, BALB/c mice produced RBD-binding antibodies with neutralising activity against the alpha, beta, gamma, and delta variants. Furthermore, vaccination elicited robust activation of CD4<sup>+</sup> and CD8<sup>+</sup> T cells with early expression of Th1 cytokines upon <italic>in vitro</italic> restimulation, along with a good tolerability profile. Importantly, vaccination with 10 g or 20 g RBD fusion heterodimer/dose conferred 100% efficacy preventing mortality and bodyweight loss upon SARS-CoV-2 challenge in K18-hACE2 mice. These findings demonstrate the feasibility of this novel recombinant vaccine strategy, allowing the inclusion of up to 2 different RBD proteins in the same vaccine. Most importantly, this new platform is easy to adapt to future VoCs and has a good stability profile, thus ensuring its global distribution.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.24.469775', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': True, 'title': 'Atazanavir is a competitive inhibitor of SARS-CoV-2 Mpro, impairing variants replication in vitro and in vivo', 'authors': \"[{'name': 'Otvio Augusto Chaves'}, {'name': 'Carolina Q. Sacramento'}, {'name': 'Andr C. Ferreira'}, {'name': 'Mayara Mattos'}, {'name': 'Natalia Fintelman-Rodrigues'}, {'name': 'Jairo R. Temerozo'}, {'name': 'Douglas Pereira Pinto'}, {'name': 'Gabriel P. E. da Silveira'}, {'name': 'Las Bastos da Fonseca'}, {'name': 'Heliana Martins Pereira'}, {'name': 'Aluana Santana Carlos'}, {'name': 'Joana da Costa Pinto dvila'}, {'name': 'Joo P.B. Viola'}, {'name': 'Robson Q. Monteiro'}, {'name': 'Leonardo Vazquez'}, {'name': 'Patrcia T. Bozza'}, {'name': 'Hugo Caire Castro-Faria-Neto'}, {'name': 'Thiago Moreno L. Souza'}]\", 'abstract': '<p>Atazanavir (ATV) has already been considered as a potential repurposing drug to 2019 coronavirus disease (COVID-19), however, there are controversial reports on its mechanism of action and effectiveness as anti-severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Through the pre-clinical chain of experiments: enzymatic, molecular docking, cell-based, and <italic>in vivo</italic> assays, it is demonstrated here that both SARS-CoV-2 B.1 lineage and variant of concern gamma are susceptible to this antiretroviral. Enzymatic assays and molecular docking calculations showed that SARS-CoV-2 main protease (M<sup>pro</sup>) was inhibited by ATV, with Morrisons inhibitory constant (Ki) 1.5-fold higher than boceprevir (GC376, a positive control). ATV was a competitive inhibition, increasing the M<sup>pro</sup>s Michaelis-Menten (K<sub>m</sub>) more than 6-fold. Cell-based assays indicated that SARS-CoV-2 gamma is more susceptible to ATV than its predecessor strain B.1. Using oral administration of ATV in mice to reach plasmatic exposure similar to humans, transgenic mice expression in human angiotensin converting enzyme 2 (K18-hACE2) were partially protected against lethal challenge with SARS-CoV-2 gamma. Moreover, less cell death and inflammation were observed in the lung from infected and treated mice. Our studies may contribute to a better comprehension of the M<sup>pro</sup>/ATV interaction, which could pave the way to the development of specific inhibitors of this viral protease.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.28.470242', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Rethinking protein drug design with highly accurate structure prediction of anti-CRISPR proteins', 'authors': \"[{'name': 'Ho-min Park'}, {'name': 'Yunseol Park'}, {'name': 'Joris Vankerschaver'}, {'name': 'Arnout Van Messem'}, {'name': 'Wesley De Neve'}, {'name': 'Hyunjin Shim'}]\", 'abstract': '<p>Protein therapeutics play an important role in controlling the functions and activities of disease-causing proteins in modern medicine. Despite protein therapeutics having several advantages over traditional small-molecule therapeutics, further development has been hindered by drug complexity and delivery issues. However, recent progress in deep learning-based protein structure prediction approaches such as AlphaFold opens new opportunities to exploit the complexity of these macro-biomolecules for highly-specialised design to inhibit, regulate or even manipulate specific disease-causing proteins. Anti-CRISPR proteins are small proteins from bacteriophages that counter-defend against the prokaryotic adaptive immunity of CRISPR-Cas systems. They are unique examples of natural protein therapeutics that have been optimized by the host-parasite evolutionary arms race to inhibit a wide variety of host proteins. Here, we show that these Anti-CRISPR proteins display diverse inhibition mechanisms through accurate structural prediction and functional analysis. We find that these phage-derived proteins are extremely distinct in structure, some of which have no homologues in the current protein structure domain. Furthermore, we find a novel family of Anti-CRISPR proteins which are structurally homologous to the recently-discovered mechanism of manipulating host proteins through enzymatic activity, rather than through direct inference. Using highly accurate structure prediction, we present a wide variety of protein-manipulating strategies of anti-CRISPR proteins for future protein drug design.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.23.469670', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Transection injury differentially alters the proteome of the human sural nerve', 'authors': \"[{'name': 'Monica J. Chau'}, {'name': 'Jorge E. Quintero'}, {'name': 'Eric Blalock'}, {'name': 'Christopher Samaan'}, {'name': 'Greg Gerhardt'}, {'name': 'Craig van Horne'}]\", 'abstract': '<p>Regeneration after severe peripheral nerve injury is often poor. Knowledge of human nerve regeneration and the growth microenvironment is greatly lacking. We aimed to identify the regenerative proteins in human peripheral nerve by comparing the proteome before and after a transection injury. In a unique study design, we collected from the same participants, samples from nave and degenerating sural nerve. Nave and degenerating (two weeks after injury) samples were analyzed using mass spectrometry and immunoassays. Using a correlation matrix, we found significantly altered levels following the nerve injury. Mass spectrometry revealed that post-injury samples had 672 proteins significantly upregulated and 661 significantly downregulated compared to nave samples (q &lt; 0.05, |FC| &gt; 2). We used Gene Ontology pathways to highlight groups of proteins that were significantly upregulated or downregulated with injury-induced degeneration and regeneration. Significant protein changes in key pathways were identified including growth factor levels, Schwann cell de-differentiation, myelination downregulation, epithelial-mesenchymal transition, and axonal regeneration pathways. Having proteome signatures of human peripheral nerves of both the uninjured and the degenerating/regenerating state may serve as biomarkers to aid in the future development of repair strategies and in monitoring neural tissue regeneration.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.28.470237', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'High-throughput phenotyping reveals a link between transpiration efficiency and transpiration restriction under high evaporative demand and new loci controlling water use-related traits in African rice, Oryza glaberrima Steud.', 'authors': \"[{'name': 'Pablo Affortit'}, {'name': 'Branly Effa Effa'}, {'name': 'Mame Sokhatil Ndoye'}, {'name': 'Daniel Moukouanga'}, {'name': 'Nathalie Luchaire'}, {'name': 'Llorenc Cabrera-Bosquet'}, {'name': 'Marie Carmen Peralvarez'}, {'name': 'Raphael Pilloni'}, {'name': 'Claude Welcker'}, {'name': 'Antony Champion'}, {'name': 'Pascal Gantet'}, {'name': 'Abdala Gamby Diedhiou'}, {'name': 'Baboucarr Manneh'}, {'name': 'Ricardo Aroca'}, {'name': 'Vincent Vadez'}, {'name': 'Laurent Laplaze'}, {'name': 'Philippe Cubry'}, {'name': 'Alexandre Grondin'}]\", 'abstract': '<p>Because water availability is the most important environmental factor limiting crop production, improving water use efficiency, the amount of carbon fixed per water used, is a major target for crop improvement. In rice, the genetic bases of transpiration efficiency, the derivation of water use efficiency at the whole-plant scale, and its putative component trait transpiration restriction under high evaporative demand, remain unknown. These traits were measured in a panel of 147 African rice Oryza glaberrima genotypes, known as potential sources of tolerance genes to biotic and abiotic stresses. Our results reveal that higher transpiration efficiency is associated with transpiration restriction in African rice. Detailed measurements in a subset of highly differentiated genotypes confirmed these associations and suggested that the root to shoot ratio played an important role in transpiration restriction. Genome wide association studies identified marker-trait associations for transpiration response to evaporative demand, transpiration efficiency and its residuals, that links to genes involved in water transport and cell wall patterning. Our data suggest that root shoot partitioning is an important component of transpiration restriction that has a positive effect on transpiration efficiency in African rice. Both traits are heritable and define targets for breeding rice with improved water use strategies.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.21.469172', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Identification of a promiscuous conserved CTL epitope within the SARS-CoV-2 spike protein', 'authors': \"[{'name': 'Sheng Jiang'}, {'name': 'Shuting Wu'}, {'name': 'Gan Zhao'}, {'name': 'Yue He'}, {'name': 'Xinrong Guo'}, {'name': 'Zhiyu Zhang'}, {'name': 'Jiawang Hou'}, {'name': 'Yuan Ding'}, {'name': 'Alex Cheng'}, {'name': 'Bin Wang'}]\", 'abstract': '<p>The COVID-19 disease caused by infection with SARS-CoV-2 and its variants is devastating to the global public health and economy. To date, over a hundred COVID-19 vaccines are known to be under development and the few that have been approved to fight the disease are using the spike protein as the primary target antigen. Although virus neutralizing epitopes are mainly located within the RBD of the spike protein, the presence of T cell epitopes, particularly the CTL epitopes that are likely to be needed for killing infected cells, has received comparatively little attention. In this study, we predicted several potential T cell epitopes with web-based analytic tools, and narrowed them down from several potential MHC-I and MHC-II epitopes by ELIspot and cytolytic assays to a conserved MHC-I epitope. The epitope is highly conserved in current viral variants and compatible with presentation by most HLA alleles worldwide. In conclusion, we identified a CTL epitope suitable for evaluating the CD8+ T cell-mediated cellular response and potentially for addition into future COVID-19 vaccine candidates to maximize CTL responses against SARS-CoV-2.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.28.470222', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Inference of Rift Valley Fever pathogenesis in Bos taurus using a gene co-expression network', 'authors': \"[{'name': 'John Kaniaru Gitau'}, {'name': 'Rosaline Wanjiru Macharia'}, {'name': 'Kennedy Wanjau Mwangi'}, {'name': 'Nehemiah Mosioma Ongeso'}, {'name': 'Edwin Murungi'}]\", 'abstract': \"<p>Background: Rift Valley Fever (RVF) is a viral disease caused by the Rift Valley Fever virus and spread mainly by the Aedes and Culex mosquito species. The disease primarily infects domestic animals such as sheep, goats, and cattle, resulting in a spectrum of clinical outcomes including morbidity, massive storm abortions, and neonatal fatalities. RVF outbreaks are closely linked to above-average rainfall and flooding, which create an ideal environment for mosquitos to breed, multiply, and transmit the virus to animals. The outcomes of human RVF infection range from self-limiting febrile illness to potentially fatal hemorrhagic diatheses and miscarriage in pregnant women. Collectively, the economic losses due to the zoonotic RVF disease is immense. Methods: Using the Weighted Gene Co-expression Network Analysis (WGCNA) package, RNA-Seq data generated from five healthy Bos taurus steer calves aged 4-6 months was obtained from the Gene Expression Omnibus (GEO) database (Accession number GSE71417). The data was utilized to construct a gene co-expression network. Enriched modules containing genes potentially involved in RVF infection progression were identified. Moreover, using the Multiple Expectation Maximizations for Motif Elicitation (MEME) suite, consensus regulatory motifs of enriched gene clusters were deciphered and the most abundant putative regulatory motif in each enriched module unveiled by comparative analysis with publicly available motifs using the TOMTOM motif comparison tool. The potential roles of the identified regulatory motifs were inferred by literature mining. Results: The constructed gene co-expression network revealed thirty-three (33) modules, nine of which were enriched for Gene Ontology terms linked to RVF pathogenesis. Functional enrichment in two (red and turquoise) of the nine modules was significant. ASH1-like histone lysine methyltransferase and Astrotactin were the hub genes for the red and turquoise modules respectively. ASH1-like histone lysine methyltransferase gene is involved in chromatin epigenetic modification while Astrotactin is a vertebrate-specific gene that plays an important role in neurodevelopment. Additionally, consensus regulatory motifs located on the 3' end of genes in each enriched module was identified. Conclusions: In this study, we have developed a gene co-expression network that has aided in the unveiling of functionally related genes, intramodular hub genes, and immunity genes potentially involved in RVF pathogenesis. The discovery of functional genes with putative critical roles in the establishment of RVF infection establishment will contribute to the understanding of the molecular mechanism of RVF pathogenesis. Importantly, the putative regulatory motifs identified are plausible targets for RVF drug and vaccine development. Keywords: Rift Valley Fever, Bos taurus, Gene co-expression network, modules, hub genes, Regulatory motifs.</p>\", 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.26.470090', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Heterocomplexes between the Atypical Chemokine MIF and the CXC-Motif Chemokine CXCL4L1 Regulate Inflammation and Thrombus Formation', 'authors': \"[{'name': 'Markus Brandhofer'}, {'name': 'Adrian Hoffmann'}, {'name': 'Xavier Blanchet'}, {'name': 'Elena Siminkovitch'}, {'name': 'Anne-Katrin Rohlfing'}, {'name': 'Omar El Bounkari'}, {'name': 'Jeremy A. Nestele'}, {'name': 'Alexander Bild'}, {'name': 'Christos Kontos'}, {'name': 'Katleen Hille'}, {'name': 'Vanessa Rohde'}, {'name': 'Adrian Froehlich'}, {'name': 'Jona Golemi'}, {'name': 'Oezguen Goekce'}, {'name': 'Patrick Scheiermann'}, {'name': 'Nikolaos Tsilimparis'}, {'name': 'Wolfgang E. Kempf'}, {'name': 'Lars Maegdefessel'}, {'name': 'Remco T.A. Megens'}, {'name': 'Hans Ippel'}, {'name': 'Rory R. Koenen'}, {'name': 'Kevin Mayo'}, {'name': 'Meinrad Gawaz'}, {'name': 'Aphrodite Kapurniotu'}, {'name': 'Christian Weber'}, {'name': 'Philipp von Hundelshausen'}, {'name': 'Juergen Bernhagen'}]\", 'abstract': '<p>To fulfil their orchestrating function in immune cell trafficking in homeostasis and disease, a network of 49 chemokines and 23 receptors capitalizes on features of specificity, redundancy, and functional selectivity such as biased agonism. The discovery of the chemokine interactome, i.e. heteromeric chemokine-chemokine interactions, even across CC- and CXC-class borders, has further expanded the complexity within the network. Moreover, some inflammatory mediators, which are not structurally linked to classical CC-, CXC-, CX3C-, or C-chemokines, can bind to chemokine receptors and behave as atypical chemokines (ACKs). We identified the cytokine macrophage migration inhibitory factor (MIF) as an ACK that binds to the chemokine receptors CXCR2 and CXCR4 to promote atherogenic leukocyte recruitment. Here, we hypothesized that chemokine-chemokine interactions extend to ACKs and that MIF may form heterocomplexes with classical chemokines. We tested this hypothesis, applying an unbiased chemokine protein binding array. The platelet chemokine CXCL4L1, but not its variant CXCL4 or the CXCR2/CXCR4 ligands CXCL8 or CXCL12, was identified as a candidate interactor. MIF/CXCL4L1 complexation was verified by co-immunoprecipitation, surface plasmon-resonance analysis, and microscale thermophoresis, which also established high-affinity binding (KD=100-150 nM). The binding interface was predicted by peptide array-based mapping and molecular docking. We next determined whether heterocomplex formation modulates inflammatory and atherogenic activities of MIF. MIF-elicited T-cell chemotaxis as assessed in a 3D-matrix-based live cell-imaging set-up was abrogated, when cells were co-incubated with MIF and CXCL4L1. Heterocomplexation also blocked MIF-triggered migration of Egfp+ microglia in cortical cultures in situ. Of note, CXCL4L1 blocked the binding of Alexa-MIF to a soluble ectodomain mimic of CXCR4 and co-incubation with CXCL4L1 attenuated MIF-triggered dynamic mass redistribution in HEK293-CXCR4 transfectants, indicating that complex formation interferes with MIF/CXCR4 pathways. As MIF and CXCL4L1 are abundant platelet products, we finally tested their role in platelet activation. Multi-photon microscopy, FLIM-FRET, and proximity ligation assay visualized heterocomplexes in platelet aggregates and clinical human thrombus sections. Moreover, heterocomplex formation inhibited MIF-stimulated thrombus formation under flow and skewed the morphology of adhering platelets from a large to a small lamellipodia phenotype. Together, our study establishes a novel molecular interaction, adding to the complexity of the chemokine interactome and chemokine/receptor network. MIF/CXCL4L1, or more generally, ACK/CXC-motif chemokine heterocomplexes may be promising target structures to modulate inflammation and thrombosis.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.25.470073', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Nociceptive stimuli activate the hypothalamus-habenula circuit to inhibit the mesolimbic reward system', 'authors': \"[{'name': 'Hee Young Kim'}, {'name': 'Soo Min Lee'}, {'name': 'Yu Fan'}, {'name': 'Bonghyo Lee'}, {'name': 'Sang Chan Kim'}, {'name': 'Kyle B. Bills'}, {'name': 'Scott C. Steffensen'}]\", 'abstract': '<p>Nociceptive signals interact with various regions of the brain, including those involved in physical sensation, reward, cognition, and emotion. Emerging evidence points to a role of nociception in the modulation of the mesolimbic reward system. The mechanism by which nociception affects dopamine (DA) signaling and reward is unclear. The lateral hypothalamus (LH) and the lateral habenula (LHb) receive somatosensory inputs and are structurally connected with the mesolimbic DA system. Here we show that the LH-LHb pathway is necessary for nociceptive modulation of this system. Our extracellular single-unit recordings and head-mounted microendoscopic calcium imaging revealed that nociceptive stimulation by tail-pinch excited LHb and LH neurons, which was inhibited by chemical lesion of the LH. Tail-pinch decreased extracellular DA release in the nucleus accumbens ventrolateral shell, which was blocked by disruption of the LH. Furthermore, tail-pinch attenuated cocaine-induced locomotor activity, 50-kHz ultrasonic vocalizations and reinstatement of cocaine-seeking behavior, which was inhibited by chemogenetic silencing of the LH-LHb pathway. Our findings suggest that nociceptive stimulation recruits the LH-LHb pathway to inhibit mesolimbic DA system and drug reinstatement.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.28.470262', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Optimizing multiplexed imaging experimental design through tissue spatial segregation estimation', 'authors': \"[{'name': 'Pierre Bost'}, {'name': 'Daniel Schulz'}, {'name': 'Stefanie Engler'}, {'name': 'Clive Wasserfall'}, {'name': 'Bernd Bodenmiller'}]\", 'abstract': '<p>Recent advances in multiplexed imaging methods allow simultaneous detection of dozens of proteins or RNAs enabling deep spatial characterization of both healthy and tumor tissues. Parameters for design of optimal sequencing-based experiments have been established, but such parameters are lacking for multiplex imaging studies. Here, using a spatial transcriptomic atlas of healthy and tumor human tissues, we developed a new statistical framework that determines the number of fields of view necessary to accurately identify all cell types that are part of the tissue. Using this strategy on imaging mass cytometry data, we identified a measurement of tissue spatial segregation that enables optimal experimental design and that is technology invariant. This strategy will enable significantly improved design of multiplexed imaging studies.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.24.468873', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Microbial community analysis of biofilters reveals a dominance of either comammox Nitrospira or archaea as ammonia oxidizers in freshwater aquaria', 'authors': \"[{'name': 'Michelle M. McKnight'}, {'name': 'Josh D. Neufeld'}]\", 'abstract': '<p>Nitrification by aquarium biofilters transforms toxic ammonia waste (NH<sub>3</sub>/NH<sub>4</sub><sup>+</sup>) to less toxic nitrate (NO<sub>3</sub><sup>-</sup>) via nitrite (NO<sub>2</sub><sup>-</sup>). Ammonia oxidation is mediated by ammonia-oxidizing bacteria (AOB), ammonia-oxidizing archaea (AOA), and the recently discovered complete ammonia oxidizing (comammox) <italic>Nitrospira</italic>. Prior to the discovery of comammox <italic>Nitrospira</italic>, previous research revealed that AOA dominate among ammonia oxidizers in freshwater biofilters. Here, we characterized the composition of aquarium filter microbial communities and quantified the abundance of all three known groups of ammonia oxidizers. Aquarium biofilter and water samples were collected from representative freshwater and saltwater systems in Southwestern Ontario, Canada. Using extracted DNA, we performed 16S rRNA gene sequencing and quantitative PCR (qPCR) to assess community composition and quantify the abundance of <italic>amoA</italic> genes, respectively. Our results show that aquarium biofilter microbial communities were consistently represented by putative heterotrophs of the <italic>Proteobacteria</italic> and <italic>Bacteroides</italic> phyla, with distinct profiles associated with fresh versus saltwater biofilters. Among nitrifiers, comammox <italic>Nitrospira amoA</italic> genes were detected in all 38 freshwater aquarium biofilter samples and were the most abundant ammonia oxidizer in 30 of these samples, with the remaining biofilters dominated by AOA, based on <italic>amoA</italic> gene abundances. In saltwater biofilters, AOA or AOB were differentially abundant, with no comammox <italic>Nitrospira</italic> detected. These results demonstrate that comammox <italic>Nitrospira</italic> play an important role in biofilter nitrification that has been previously overlooked and such microcosms are useful for exploring the ecology of nitrification for future research.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469631', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genetic and chemical validation of Plasmodium falciparum aminopeptidase PfA-M17 as a drug target in the hemoglobin digestion pathway', 'authors': \"[{'name': 'Rebecca C.S. Edgar'}, {'name': 'Ghizal Siddiqui'}, {'name': 'Kathryn Hjerrild'}, {'name': 'Tess R. Malcolm'}, {'name': 'Natalie B. Vinh'}, {'name': 'Chaille T. Webb'}, {'name': 'Christopher A. MacRaild'}, {'name': 'Natalie A. Counihan'}, {'name': 'Darren J. Creek'}, {'name': 'Peter J Scammells'}, {'name': 'Sheena McGowan'}, {'name': 'Tania F. de Koning-Ward'}]\", 'abstract': '<p><italic>Plasmodium falciparum,</italic> a causative agent of malaria, continues to remain a global health threat since these parasites have developed increasing resistance to all anti-malaria drugs used throughout the world. Accordingly, drugs with novel modes of action are desperately required to combat malaria. <italic>P. falciparum</italic> parasites infect human red blood cells where they digest the hosts main protein constituent, hemoglobin. Leucine aminopeptidase <italic>Pf</italic>A-M17 is one of several aminopeptidases that have been implicated in the last step of this digestive pathway. Here we utilize both reverse genetics and a compound specifically designed to inhibit the activity of <italic>Pf</italic>A-M17 to show that <italic>Pf</italic>A-M17 is essential for <italic>P. falciparum</italic> survival as it provides parasites with free amino acids for growth, many of which are highly likely to originate from hemoglobin. We further show that our inhibitor is on-target for <italic>Pf</italic>A-M17 and has the ability to kill parasites at nanomolar concentrations. Thus, in contrast to other hemoglobin-degrading proteases that have overlapping redundant functions, we validate <italic>Pf</italic>A-M17 as a potential novel drug target.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.28.470254', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Metabolic analysis of a bacterial synthetic community from maize roots provides new mechanistic insights into microbiome stability', 'authors': \"[{'name': 'Jenna Krumbach'}, {'name': 'Patrizia Kroll'}, {'name': 'Vera Wewer'}, {'name': 'Sabine Metzger'}, {'name': 'Till Ischebeck'}, {'name': 'Richard P Jacoby'}]\", 'abstract': '<p>Stability is a desirable property for agricultural microbiomes, but there is a poor understanding of the mechanisms that mediate microbial community stability. Recently, a representative bacterial synthetic community from maize roots has been proposed as a model system to study microbiome stability (Niu 2017, PNAS, 114:E2450). This SynCom assembles stably when all seven members are present, but community diversity collapses without the keystone E. cloacae strain. The aim of this study was to assess the role of metabolites for the stability of this SynCom, by defining the metabolic niches occupied by each strain, as well as their cross-feeding phenotypes and B-vitamin dependencies. We show that the individual member strains occupy complementary metabolic niches, measured by the depletion of distinct metabolites in exometabolomic experiments, as well as contrasting growth phenotypes on diverse carbon substrates. Minimal medium experiments show that the established seven-member community comprises a mixture of prototrophic and auxotrophic strains. Correspondingly, experimental cross-feeding phenotypes showed that spent media harvested from the prototrophic strains can sustain growth of two auxotrophs. We suggest that the metabolic mechanisms exhibited by this SynCom could serve as design principles to inform the rational assembly of stable plant-associated microbial communities.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.23.21266759', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Social Class, Race/Ethnicity, and COVID-19 Mortality Among Working Age Adults in the United States', 'authors': \"[{'name': 'Elizabeth B. Pathak'}, {'name': 'Janelle Menard'}, {'name': 'Rebecca B. Garcia'}, {'name': 'Jason L. Salemi'}]\", 'abstract': '<sec><title>Importance</title><p>Substantial racial/ethnic disparities in COVID-19 mortality have been documented. Social class is a likely explanation of mortality disparities across and within racial/ethnic groups. This is the first U.S. study of social class and COVID-19 mortality in working age adults.</p></sec><sec><title>Objectives</title><p>To determine the joint effects of social class, race/ethnicity, and gender on the burden of COVID-19 mortality. A secondary objective was to determine whether differences in opportunities for remote work were correlated with COVID-19 death rates for sociodemographic groups.</p></sec><sec><title>Design</title><p>Annual mortality study which used a special government tabulation of 2020 COVID-19 related deaths stratified by decedents social class (educational attainment) and race/ethnicity.</p></sec><sec><title>Setting</title><p>United States in 2020.</p></sec><sec><title>Participants</title><p>COVID-19 decedents aged 25 to 64 years old (n=69,001).</p></sec><sec><title>Exposures</title><p>Social class (working class, some college, college graduate), race/ethnicity (Hispanic, Black, Asian, Indigenous, multiracial, and non-Hispanic white), and gender (women, men). Detailed census data on occupations held by adults in 2020 in each of the 36 sociodemographic groups studied were used to quantify the possibility of remote work for each group.</p></sec><sec><title>Main Outcomes and Measures</title><p>Age-adjusted COVID-19 death rates for 36 sociodemographic groups defined by social class, race/ethnicity, and gender. Disparities were quantified by relative risks and 95% confidence intervals. College graduates were the (low risk) referent group for all relative risk calculations.</p></sec><sec><title>Results</title><p>A higher proportion of Hispanics, Blacks, and Indigenous people were working class in 2020. COVID-19 mortality was five times higher in the working class vs. college graduates (72.2 vs. 14.6 deaths per 100,000, RR=4.94, 95% CI 4.82-5.05). The joint detriments of lower socioeconomic position, Hispanic ethnicity, and male gender resulted in a COVID-19 death rate which was over 27 times higher (178.0 vs. 6.5 deaths/100,000, RR=27.4, 95%CI 25.9-28.9) for working class Hispanic men vs. college graduate white women. In regression modeling, percent employed in never remote jobs explained 72% of the variance in COVID-19 death rates.</p></sec><sec><title>Conclusions and Relevance</title><p>SARS-CoV-2 infection control efforts should prioritize the working class (i.e. those with no college education), particularly those employed in never remote jobs with inflexible and unsafe working conditions (i.e. blue collar, service, and retail sales workers).</p></sec><sec><title>KEY POINTS</title><sec><title>Question</title><p>Did COVID-19 mortality rates among non-elderly adults vary significantly by social class, race/ethnicity, and gender in 2020?</p></sec><sec><title>Findings</title><p>Among 69,001 COVID-19 decedents, age-adjusted COVID-19 deaths rates were 5 times higher in working class vs. college graduate adults 25-64 years old. Working class Hispanic, Black, and Indigenous men suffered the highest burden of COVID-19 mortality, while college graduate white women experienced the lowest death rate.</p></sec><sec><title>Meaning</title><p>COVID-19 mitigation efforts should prioritize the working class (i.e. those with no college education), particularly blue collar, service, and retail sales workers.</p></sec></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266712', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': True, 'title': 'Human phospho-signaling networks of SARS-CoV-2 infection are rewired by population genetic variants', 'authors': \"[{'name': 'Diogo Pellegrina'}, {'name': 'Alexander T. Bahcheli'}, {'name': 'Michal Krassowski'}, {'name': 'Jri Reimand'}]\", 'abstract': '<p>SARS-CoV-2 infection hijacks signaling pathways and induces protein-protein interactions between human and viral proteins. Human genetic variation may impact SARS-CoV-2 infection and COVID-19 pathology; however, the role of genetic variation in these signaling networks remains uncharacterized. We studied human single nucleotide variants (SNVs) affecting phosphorylation sites modulated by SARS-CoV-2 infection, using machine learning to identify amino acid changes altering kinase-bound sequence motifs. We found 2033 infrequent phosphorylation-associated SNVs (pSNVs) that are enriched in sequence motif alterations, potentially reflecting the evolution of signaling networks regulating host defenses. Proteins with pSNVs are involved in viral life cycle processes and host responses, including regulators of RNA splicing and interferon response, as well as glucose homeostasis pathways with potential associations with COVID-19 co-morbidities. Certain pSNVs disrupt CDK and MAPK substrate motifs and replace these with motifs recognized by Tank Binding Kinase 1 (TBK1) involved in innate immune responses, indicating consistent rewiring of infection signaling networks. Our analysis highlights potential genetic factors contributing to the variation of SARS-CoV-2 infection and COVID-19 and suggests leads for mechanistic and translational studies.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469039', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Fuzzy set intersection based paired-end short-read alignment', 'authors': \"[{'name': 'William J. Bolosky'}, {'name': 'Arun Subramaniyan'}, {'name': 'Matei Zaharia'}, {'name': 'Ravi Pandya'}, {'name': 'Taylor Sittler'}, {'name': 'David Patterson'}]\", 'abstract': '<p>Much genomic data comes in the form of paired-end reads: two reads that represent genetic material with a small gap between. We present a new algorithm for aligning both reads in a pair simultaneously by fuzzily intersecting the sets of candidate alignment locations for each read. This algorithm is often much faster and produces alignments that result in variant calls having roughly the same concordance as the best competing aligners.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.25.469203', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'High Resolution Single Cell Maps Reveals Distinct Cell Organization and Function Across Different Regions of the Human Intestine', 'authors': \"[{'name': 'John W. Hickey'}, {'name': 'Winston R. Becker'}, {'name': 'Stephanie A. Nevins'}, {'name': 'Aaron Horning'}, {'name': 'Almudena Espin Perez'}, {'name': 'Roxanne Chiu'}, {'name': 'Derek C. Chen'}, {'name': 'Daniel Cotter'}, {'name': 'Edward D. Esplin'}, {'name': 'Annika K. Weimer'}, {'name': 'Chiara Caraccio'}, {'name': 'Vishal Venkataraaman'}, {'name': 'Christian M. Schrch'}, {'name': 'Sarah Black'}, {'name': 'Maria Brbi'}, {'name': 'Kaidi Cao'}, {'name': 'Jure Leskovec'}, {'name': 'Zhengyan Zhang'}, {'name': 'Shin Lin'}, {'name': 'Teri Longacre'}, {'name': 'Sylvia K. Plevitis'}, {'name': 'Yiing Lin'}, {'name': 'Garry P. Nolan'}, {'name': 'William J. Greenleaf'}, {'name': 'Michael Snyder'}]\", 'abstract': '<p>The colon is a complex organ that promotes digestion, extracts nutrients, participates in immune surveillance, maintains critical symbiotic relationships with microbiota, and affects overall health. To better understand its organization, functions, and its regulation at a single cell level, we performed CODEX multiplexed imaging, as well as single nuclear RNA and open chromatin assays across eight different intestinal sites of four donors. Through systematic analyses we find cell compositions differ dramatically across regions of the intestine, demonstrate the complexity of epithelial subtypes, and find that the same cell types are organized into distinct neighborhoods and communities highlighting distinct immunological niches present in the intestine. We also map gene regulatory differences in these cells suggestive of a regulatory differentiation cascade, and associate intestinal disease heritability with specific cell types. These results describe the complexity of the cell composition, regulation, and organization for this organ, and serve as an important reference map for understanding human biology and disease.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.23.469760', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Comparison of neuromuscular junction dynamics following ischemic and aged skeletal muscle', 'authors': \"[{'name': 'Berna Aliya'}, {'name': 'Mahir Mohiuddin'}, {'name': 'Jeongmoon J. Choi'}, {'name': 'Gunjae Jeong'}, {'name': 'Innie Kang'}, {'name': 'Hannah Castels'}, {'name': 'Cade Jones'}, {'name': 'Young C. Jang'}]\", 'abstract': '<p>Both aging and neuromuscular diseases lead to significant changes in the morphology and functionality of the neuromuscular synapse. Skeletal muscles display a remarkable regenerative capacity, however, are still susceptible to diseases of aging and peripheral nerve perturbations. In this study, we assessed how neuromuscular synapses differ in aged and injured skeletal muscle using an improved neuromuscular junction (NMJ) staining and imaging method. We found that both aged and ischemic skeletal muscle display Wallerian degeneration of the presynaptic motor axons and fragmentation of postsynaptic acetylcholine receptors (AChRs). Quantifiable measurements of various metrics of the NMJs provide a more concrete idea of the dynamics that are occurring in the muscle microenvironment. We questioned whether neuronal degradation precedes myofiber atrophy or vice versa. Previously, it was shown that a cellular crosstalk exists among the motor neurons, myofibers, vasculature, and mitochondria within the muscle microdomain. It is apparent that lack of blood flow to motor neurons in ischemic skeletal muscle disrupts the structure of NMJs, however it is unclear if the aging condition experiences similar dynamics. We demonstrated that both aged and ischemic skeletal muscle demonstrate similar patterns of degeneration, characterized by a smaller percentage overlap of presynaptic and postsynaptic sides, greater fragmentation of AChRs, and a smaller area of AChR clusters. Together, these results reveal high resolution, precise parallels between the aged and ischemic NMJs.</p><sec><title>Impact Statement</title><p>The goal of this study was to assess changes in presynaptic motor neurons and postsynaptic acetylcholine receptors following an ischemic injury model and compare this with an aging model. This was accomplished by characterizing key components of NMJ morphology, including overlap and size of the receptors. There is currently limited research investigating the cellular communication between skeletal muscle fibers and motor neurons. Additionally, there is limited work comparing neuromuscular remodeling in aged and young models. With the substantial prevalence of neuromuscular disorders, especially in the aging population, it is essential to understand nerve-muscle interactions in order to promote increased mobility and improved quality of life in both injury and aging models.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469579', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Identification of biological processes and signaling pathways for the knockout of REV-ERB in mouse brain', 'authors': \"[{'name': 'Jing Li'}, {'name': 'Wei Wang'}, {'name': 'Hanming Gu'}]\", 'abstract': '<p>REV-ERB is an orphan nuclear receptor that is widely expressed in the brain and inhibits transcriptional activities. A variety of genes affect the activity and expression of REV-ERB. In this study, our objective is to identify significant signaling pathways and biological processes in the knockout of the REV-ERB mouse brain. The GSE152919 dataset was originally created by using the Illumina HiSeq 4000 (Mus musculus). The KEGG and GO analyses suggested that biological processes PPAR signaling, Hippo signaling, and Hypertrophic cardiomyopathy (HCM) are mostly affected in the knockout of REV-ERB. Furthermore, we identified a number of genes according to the PPI network including NPAS2, CRY2, BMAL1, and CRY1 which were involved in the lack of REV-ERB in the brain. Therefore, our study provides further insights into the study of circadian clocks.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469576', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'A modified porous silicon microparticle promotes mucosal delivery of SARS-CoV-2 antigen and induction of potent and durable systemic and mucosal T helper 1 skewed protective immunity', 'authors': \"[{'name': 'Awadalkareem Adam'}, {'name': 'Qing Shi'}, {'name': 'Binbin Wang'}, {'name': 'Jing Zou'}, {'name': 'Junhua Mai'}, {'name': 'Samantha R Osman'}, {'name': 'Wenzhe Wu'}, {'name': 'Xuping Xie'}, {'name': 'Patricia V Aguilar'}, {'name': 'Xiaoyong Bao'}, {'name': 'Pei-Yong Shi'}, {'name': 'Haifa Shen'}, {'name': 'Tian Wang'}]\", 'abstract': '<p>Development of optimal SARS-CoV-2 vaccines to induce potent, long-lasting immunity and provide cross-reactive protection against emerging variants remains a high priority. Here, we report that a modified porous silicon microparticle (mPSM)-adjuvanted SARS-CoV-2 receptor-binding domain (RBD) vaccine activated dendritic cells and generated more potent and durable SARS-CoV-2-specific systemic humoral and type 1 helper T (Th) cell-mediated immune responses than alum-formulated RBD following parenteral vaccination, and protected mice from SARS-CoV-2 and Beta variant infection. mPSM facilitated the uptake of SARS-CoV-2 RBD antigens by nasal and airway epithelial cells. Parenteral and intranasal prime and boost vaccinations with mPSM-RBD elicited potent systemic and lung resident memory T and B cells and SARS-CoV-2 specific IgA responses, and markedly diminished viral loads and inflammation in the lung following SARS-CoV-2 Delta variant infection. Our results suggest that mPSM can serve as potent adjuvant for SARS-CoV-2 subunit vaccine which is effective for systemic and mucosal vaccination.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469878', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Optimizing bulk segregant analysis of drug resistance using Plasmodium falciparum genetic crosses conducted in humanized mice', 'authors': \"[{'name': 'Katelyn Vendrely Brenneman'}, {'name': 'Xue Li'}, {'name': 'Sudhir Kumar'}, {'name': 'Elizabeth Delgado'}, {'name': 'Lisa A. Checkley'}, {'name': 'Douglas A. Shoue'}, {'name': 'Ann Reyes'}, {'name': 'Biley Abatiyow'}, {'name': 'Meseret Haile'}, {'name': 'Rupam Tripura'}, {'name': 'Tom Peto'}, {'name': 'Dysoley Lek'}, {'name': 'Katrina A. Button-Simons'}, {'name': 'Stefan H.I. Kappe'}, {'name': 'Mehul Dhorda'}, {'name': 'Francois Nosten'}, {'name': 'Standwell C. Nkhoma'}, {'name': 'Ian H Cheeseman'}, {'name': 'Ashley M Vaughan'}, {'name': 'Michael T Ferdig'}, {'name': 'Tim JC Anderson'}]\", 'abstract': '<p>Background: Classical genetic crosses in malaria parasites involve isolation, genotyping, and phenotyping of multiple progeny parasites, which is time consuming and laborious. Bulk segregant analysis (BSA) offers a powerful and efficient alternative to identify loci underlying complex traits in the human malaria parasite, Plasmodium falciparum. \\nMethods: We have used BSA, which combines genetic crosses using humanized mice with pooled sequencing of progeny populations to measure changes in allele frequency following selection with antimalarial drugs. We used dihydroartemisinin (DHA) drug selection in two genetic crosses (Mal31xKH004 and NF54xNHP1337). We specifically investigated how synchronization, cryopreservation, and the drug selection regimen of progeny pools impacted the success of BSA experiments.\\nFindings: We detected a strong and repeatable quantitative trait locus (QTL) at chr13 kelch13 locus in both crosses, but did not detect QTLs at ferredoxin (fd), the apicoplast ribosomal protein S10 (arps10), multidrug resistance protein 2 (mdr2). QTLs were detected using synchronized, but not unsynchronized pools, consistent with the stage-specific action of DHA. We also successfully applied BSA to cryopreserved progeny pools.\\nInterpretation: Our results provide proof-of-principal of the utility of BSA for rapid, robust genetic mapping of drug resistance loci. Use of cryopreserved progeny pools expands the utility of BSA because we can conduct experiments using archived progeny pools from previous genetic crosses. BSA provides a powerful approach that complements traditional QTL methods for investigating the genetic architecture of resistance to antimalarials, and to reveal new or accessory loci contributing to artemisinin resistance.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.25.470059', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Single-cell generalized trend model (scGTM): a flexible and interpretable model of gene expression trend along cell pseudotime', 'authors': \"[{'name': 'Elvis Han Cui'}, {'name': 'Weng Kee Wong'}, {'name': 'Dongyuan Song'}, {'name': 'Jingyi Jessica Li'}]\", 'abstract': \"<p>Modeling single-cell gene expression trends along cell pseudotime is a crucial analysis for exploring biological processes. Most existing methods rely on nonparametric regression models for their flexibility; however, nonparametric models often provide trends too complex to interpret. Other existing methods use interpretable but restrictive models. Since model interpretability and flexibility are both indispensable for understanding biological processes, the single-cell field needs a model that improves the interpretability and largely maintains the flexibility of nonparametric regression models. Here we propose the single-cell generalized trend model (scGTM) for capturing a gene's expression trend, which may be monotone, hill-shaped, or valley-shaped, along cell pseudotime. The scGTM has three advantages: (1) it can capture non-monotonic trends that are still easy to interpret, (2) its parameters are biologically interpretable and trend informative, and (3) it can flexibly accommodate common distributions for modeling gene expression counts. To tackle the complex optimization problems, we use the particle swarm optimization algorithm to find the constrained maximum likelihood estimates for the scGTM parameters. As an application, we analyze several single-cell gene expression data sets using the scGTM and show that it can capture interpretable gene expression trends along cell pseudotime and reveal molecular insights underlying the biological processes. We also provide an open-access Python package for fitting the scGTM at https://github. com/ElvisCuiHan/scGTM.</p>\", 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.23.469697', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Modulation of PKM1/2 levels by steric blocking morpholinos alters the metabolic and pluripotent state of murine pluripotent stem cells', 'authors': \"[{'name': 'Joshua G. Dierolf'}, {'name': 'Hailey L.M. Hunter'}, {'name': 'Andrew J. Watson'}, {'name': 'Dean H. Betts'}]\", 'abstract': '<p>Cellular metabolism plays both an active and passive role in embryonic development, pluripotency, and cell-fate decisions. However, little is known regarding the role of metabolism in regulating the recently described formative pluripotent state. The pluripotent developmental continuum features a metabolic switch from a bivalent metabolism (both glycolysis and oxidative phosphorylation) in nave cells, to predominantly glycolysis in primed cells. We investigated the role of pyruvate kinase muscle isoforms (PKM1/2) in nave, formative, and primed mouse embryonic stem cells through modulation of PKM1/2 mRNA transcripts using steric blocking morpholinos that downregulate PKM2 and upregulate PKM1. We have examined these effects in nave, formative, and primed cells by quantifying the effects of PKM1/2 modulation on pluripotent and metabolic transcripts and by measuring shifts in the population frequencies of cells expressing nave and primed cell surface markers by flow cytometry. Our results demonstrate that modulating PKM1 and PKM2 levels alters the transition from the nave state into a primed pluripotent state by enhancing the proportion of the affected cells seen in the formative state. Therefore, we conclude that PKM1/2 actively contributes to mechanisms that oversee early stem pluripotency and their progression towards a primed pluripotent state.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469747', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Genome-wide characterization of SARS-CoV-2 cytopathogenic proteins in the search of antiviral targets', 'authors': \"[{'name': 'Jiantao Zhang'}, {'name': 'Qi Li'}, {'name': 'Ruth S. Cruz Cosme'}, {'name': 'Vladimir Gerzanich'}, {'name': 'Qiyi Tang'}, {'name': 'J. Marc Simard'}, {'name': 'Richard Y. Zhao'}]\", 'abstract': '<p>Therapeutic inhibition of critical viral functions is important for curtailing coronavirus disease-2019 (COVID-19). We sought to identify antiviral targets through genome-wide characterization of SARS-CoV-2 proteins that are crucial for viral pathogenesis and that cause harmful cytopathic effects. All twenty-nine viral proteins were tested in a fission yeast cell-based system using inducible gene expression. Twelve proteins including eight non-structural proteins (NSP1, NSP3, NSP4, NSP5, NSP6, NSP13, NSP14 and NSP15) and four accessory proteins (ORF3a, ORF6, ORF7a and ORF7b) were identified that altered cellular proliferation and integrity, and induced cell death. Cell death correlated with the activation of cellular oxidative stress. Of the twelve proteins, ORF3a was chosen for further study in mammalian cells. In human pulmonary and kidney epithelial cells, ORF3a induced cellular oxidative stress associated with apoptosis and necrosis, and caused activation of pro-inflammatory response with production of the cytokines TNF-, IL-6, and IFN-1, possibly through the activation of NF-B. To further characterize the mechanism, we tested a natural ORF3a Beta variant, Q57H, and a mutant with deletion of the highly conserved residue, G188. Compared to wild type ORF3a, the G188 variant yielded more robust activation of cellular oxidative stress, cell death, and innate immune response. Since cellular oxidative stress and inflammation contribute to cell death and tissue damage linked to the severity of COVID-19, our findings suggest that ORF3a is a promising, novel therapeutic target against COVID-19.</p><sec><title>Significance</title><p>The ongoing SARS-CoV-2 pandemic has claimed over 5 million lives with more than 250 million people infected world-wide. While vaccines are effective, the emergence of new viral variants could jeopardize vaccine protection. Antiviral drugs provide an alternative to battle against COVID-19. Our goal was to identify viral therapeutic targets that can be used in antiviral drug discovery. Utilizing a genome-wide functional analysis in a fission yeast cell-based system, we identified twelve viral candidates, including ORF3a, which cause cellular oxidative stress, inflammation and apoptosis and necrosis that contribute to COVID-19. Our findings indicate that antiviral agents targeting ORF3a could greatly impact COVID-19.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469607', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Local adaptation mediated niche expansion in correlation with genetic richness', 'authors': \"[{'name': 'Masaomi Kurokawa'}, {'name': 'Issei Nishimura'}, {'name': 'Bei-Wen Ying'}]\", 'abstract': '<p>As a central issue in evolution and ecology, the quantitative relationship among the genome, adaptation and the niche was investigated. Local adaptation of five <italic>Escherichia coli</italic> strains carrying either the wild-type genome or reduced genomes was achieved by experimental evolution. A high-throughput fitness assay of the ancestor and evolved populations across an environmental gradient of eight niches resulted in a total of 80 fitness curves generated from 2,220 growth curves. Further analyses showed that the increases in both local adaptiveness and niche broadness were negatively correlated with genetic richness. Local adaptation caused common niche expansion, whereas niche expansion for generality or speciality was decided by genetic richness. The order of the mutations accumulated stepwise was correlated with the magnitude of the fitness increase attributed to mutation accumulation. Pre-adaptation probably participated in coordination among genetic richness, local adaptation and niche expansion.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469605', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Firefly bioluminescence outshines aerial predators', 'authors': \"[{'name': 'Gareth S. Powell'}, {'name': 'Natalie A. Saxton'}, {'name': 'Yelena M. Pacheco'}, {'name': 'Kathrin F. Stanger-Hall'}, {'name': 'Gavin J. Martin'}, {'name': 'Dominik Kusy'}, {'name': 'Ladislav Bocak'}, {'name': 'Marc A. Branham'}, {'name': 'Seth M. Bybee'}]\", 'abstract': '<p>Bioluminescence is found across life and has many functions. Yet we understand very little about its timing and origins, particularly as a predator avoidance strategy. Understanding the timing between bioluminescence and predator origins has yet to be examined and can help elucidate the evolution of the ecologically important signal aposematism. Using the most prevalent bioluminescent group, fireflies, where bioluminescence primarily functions as aposematic and sexual signals, the timing for the origins of both potential predators of fireflies and bioluminescence is explored. Divergence time estimations were performed using a genomic-scale phylogenetic reconstruction Lampyridae, and multiple fossil calibration points, allowing for a robust estimate for the origin of beetle bioluminescence as both a terrestrial and aerial signal. Our results recover the origins of terrestrial beetle bioluminescence at 141 mya and aerial bioluminescence at 133 mya. These ages predate the origins of all known extant aerial predators (i.e., bats and birds) and support the much older terrestrial predators (frogs, ground beetles, lizards, snakes, and hunting spiders) as the most likely drivers of bioluminescence in beetles.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469627', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'The Mutational Dynamics of Short Tandem Repeats in Large, Multigenerational Families', 'authors': \"[{'name': 'Cody J. Steely'}, {'name': 'W. Scott Watkins'}, {'name': 'Lisa Baird'}, {'name': 'Lynn B. Jorde'}]\", 'abstract': '<p>Short tandem repeats (STRs) are tandemly repeated sequences of 1-6 bp motifs. STRs compose approximately 3% of the genome, and mutations at STR loci have been linked to dozens of human diseases including amyotrophic lateral sclerosis, Friedreich ataxia, Huntington disease, and fragile X syndrome. Improving our understanding of these mutations would increase our knowledge of the mutational dynamics of the genome and may uncover additional loci that contribute to disease. Here, to estimate the genome-wide pattern of mutations at STR loci, we analyzed blood-derived whole-genome sequencing data for 544 individuals from 29 three-generation CEPH pedigrees. These pedigrees contain both sets of grandparents, the parents, and an average of 9 grandchildren per family. Using HipSTR we identified <italic>de novo</italic> STR mutations in the 2<sup>nd</sup> generation of these pedigrees. Analyzing ~1.6 million STR loci, we estimate the empircal <italic>de novo</italic> STR mutation rate to be 5.24*10<sup>5</sup> mutations per locus per generation. We find that perfect repeats mutate ~2x more often than imperfect repeats. <italic>De novo</italic> STRs are significantly enriched in <italic>Alu</italic> elements (p &lt; 2.2e-16). Approximately 30% of STR mutations occur within <italic>Alu</italic> elements, which compose only ~11% of the genome, and ~10% are found in LINE-1 insertions, which compose ~17% of the genome. Phasing these <italic>de novo</italic> mutations to the parent of origin shows that parental transmission biases vary among families. We estimate the average number of <italic>de novo</italic> genome-wide STR mutations per individual to be ~85, which is similar to the average number of observed <italic>de novo</italic> single nucleotide variants.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469686', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Using light-sheet microscopy to study spontaneous activity in the developing lateral-line system', 'authors': \"[{'name': 'Qiuxiang Zhang'}, {'name': 'Katie Kindt'}]\", 'abstract': '<p>Hair cells are the sensory receptors in the auditory and vestibular systems of all vertebrates, and in the lateral-line system of aquatic vertebrates. During development, spontaneous activity in hair cells shapes the formation of these sensory systems. In auditory hair cells of mice, coordinated waves of spontaneous activity can be triggered by concomitant activity in nearby supporting cells. But in mammals, developing auditory and vestibular hair cells can also autonomously generate spontaneous events independent of supporting cell activity. To date, significant progress has been made studying spontaneous activity in the auditory and vestibular systems of mammals, in isolated cultures. The purpose of this work is to explore the zebrafish lateral-line system as a model to study and understand spontaneous activity <italic>in vivo</italic>. Our work applies genetically encoded calcium indicators along with light-sheet fluorescence microscopy to visualize spontaneous calcium activity in the developing lateral-line system. Consistent with our previous work, we show that spontaneous calcium activity is present in developing lateral-line hair cells. We now show that supporting cells that surround hair cells, and cholinergic efferent terminals that directly contact hair cells are also spontaneously active. Using two-color functional imaging we demonstrate that spontaneous activity in hair cells does not correlate with activity in either supporting cells or cholinergic terminals. We find that during lateral-line development, hair cells autonomously generate spontaneous events. Using localized calcium indicators, we show that within hair cells, spontaneous calcium activity occurs in two distinct domainsthe mechanosensory bundle and the presynapse. Further, spontaneous activity in the mechanosensory bundle ultimately drives spontaneous calcium influx at the presynapse. Comprehensively, our results indicate that in developing lateral-line hair cells, autonomously generated spontaneous activity originates with spontaneous mechanosensory events. Overall, with robust spontaneous activity three different cell types, the developing lateral line is a rich model to study these activities in an intact sensory organ. Future work studying this model may further our understanding of these activities and their role in sensory system formation, function and regeneration.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469544', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The role of indirect effects in coevolution as mutualism transitions into antagonism', 'authors': \"[{'name': 'Fernando Pedraza'}, {'name': 'Hanlun Liu'}, {'name': 'Klementyna A. Gawecka'}, {'name': 'Jordi Bascompte'}]\", 'abstract': '<p>Species interactions have evolved from antagonistic to mutualistic and back several times throughout lifes history. Yet, it is unclear how changes in the type of interaction between species alter the coevolutionary dynamics of entire communities. This is a pressing matter, as transitions from mutualisms to antagonisms may be becoming more common with human-induced global change. Here, we combine network and evolutionary theory to simulate how shifts in interaction types alter the coevolution of empirical communities. We show that as mutualistic networks shift to antagonistic, selection imposed by direct partners begins to outweigh that imposed by indirect partners. This weakening of indirect effects is associated with communities losing their tight integration of traits and increasing their rate of adaptation. The above changes are more pronounced when specialist consumers are the first species to switch to antagonism. A shift in the outcome of species interactions may therefore reverberate across communities and alter the direction and speed of coevolution.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469527', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genome Assembly and Population Resequencing Reveal the Geographical Divergence of Shanmei (Rubus corchorifolius)', 'authors': \"[{'name': 'Yinqing Yang'}, {'name': 'Kang Zhang'}, {'name': 'Ya Xiao'}, {'name': 'Lingkui Zhang'}, {'name': 'Yile Huang'}, {'name': 'Xing Li'}, {'name': 'Shumin Chen'}, {'name': 'Yansong Peng'}, {'name': 'Shuhua Yang'}, {'name': 'Yongbo Liu'}, {'name': 'Feng Cheng'}]\", 'abstract': '<p><bold><italic>Rubus corchorifolius</italic></bold> (<bold>Shanmei</bold> or mountain berry, 2n =14) is widely distributed in China, and its fruit has high nutritional and medicinal values. Here, we report a high-quality chromosome-scale <bold>genome assembly</bold> of Shanmei, with a size of 215.69 Mb and encompassing 26,696 genes. Genome comparisons among Rosaceae species show that Shanmei and Fupenzi (<italic>Rubus chingii</italic> Hu) are most closely related, and then is blackberry (<italic>Rubus occidentalis</italic>). Further <bold>resequencing</bold> of 101 samples of Shanmei collected from four regions in provinces of Yunnan, Hunan, Jiangxi, and Sichuan in South China reveals that the Hunan population of Shanmei possesses the highest diversity and may represent the relatively more ancestral population. Moreover, the Yunnan population undergoes strong selection based on nucleotide diversity, linkage disequilibrium, and the historical effective population size analyses. Furthermore, genes from candidate genomic regions that show strong <bold>divergence</bold> are significantly enriched in flavonoid biosynthesis and plant hormone signal transduction, indicating the genetic basis of adaptation of Shanmei to the local environments. The high-quality genome sequences and the variome dataset of Shanmei provide valuable resources for breeding applications and for elucidating the <bold>genome evolution</bold> and ecological adaptation of <italic>Rubus</italic> species.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.28.470265', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'X chromosomes show a faster evolutionary rate and complete somatic dosage compensation across Timema stick insect species', 'authors': \"[{'name': 'Darren J Parker'}, {'name': 'Kamil S Jaron'}, {'name': 'Zo Dumas'}, {'name': 'Marc Robinson-Rechavi'}, {'name': 'Tanja Schwander'}]\", 'abstract': '<p>Sex chromosomes have evolved repeatedly across the tree of life. As they are present in different copy numbers in males and females, they are expected to experience different selection pressures than the autosomes, with consequences including a faster rate of evolution, increased accumulation of sexually antagonistic alleles, and the evolution of dosage compensation. Whether these consequences are general or linked to idiosyncrasies of specific taxa is not clear as relatively few taxa have been studied thus far. Here we use whole-genome sequencing to identify and characterize the evolution of the X chromosome in five species of Timema stick insects with XX:X0 sex determination. The X chromosome had a similar size (approximately 11% of the genome) and gene content across all five species, suggesting that the X chromosome originated prior to the diversification of the genus. Genes on the X showed evidence of a faster evolutionary rate than genes on the autosomes, likely due to less effective purifying selection. Genes on the X also showed almost complete dosage compensation in somatic tissues (heads and legs), but dosage compensation was absent in the reproductive tracts. Contrary to prediction, sex-biased genes showed little enrichment on the X, suggesting that the advantage X-linkage provides to the accumulation of sexually antagonistic alleles is weak. Overall, we found the consequences of X-linkage on gene sequences and expression to be similar across Timema species, showing the characteristics of the X chromosome are surprisingly consistent over 30 million years of evolution.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.26.470170', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Two Liberibacter Effectors Combine to Suppress Critical Innate Immune Defenses and Facilitate Huanglongbing Pathogenesis in Citrus', 'authors': \"[{'name': 'Supratim Basu'}, {'name': 'Loan Huynh'}, {'name': 'Shujian Zhang'}, {'name': 'Roel Rabara'}, {'name': 'jeanette velasquez'}, {'name': 'Hau Nguyen'}, {'name': 'Guixia Hao'}, {'name': 'Godfrey Miles'}, {'name': 'Qingchun Shi'}, {'name': 'Ed Stover'}, {'name': 'Goutam Gupta'}]\", 'abstract': '<p>Genome sequence analyses predicted the presence of effectors in the gram-negative Candidatus Liberibacter asiaticus (CLas) even without the presence of a classical type III secretion system.  Since CLas is not culturable, it is not possible to perform traditional gene knockout experiments to determine the role of various effectors in Huanglongbing (HLB) pathogenesis.  Therefore, we followed an alternative functional genomics approach to examine the role of the CLas effectors in HLB pathogenesis in general and more specifically in suppressing citrus innate immune response.  Here, we focused on the CLas effectors, P235 and Effector 3, to perform the following studies.  First, proteomic studies by LC-MS/MS were conducted to screen the putative interacting citrus protein partners of P235 and Effector 3 from the healthy and CLas-infected Hamlin extracts and the most probable candidates were identified based upon their high protein scores from LC-MS/MS.  Second, a transgenic tobacco split GFP system was designed for in planta detection of the most probable citrus interacting protein partners of P235 and Effector 3.  Third, in vitro and in planta studies were performed to show that each of two effectors interacts with and inhibits the functions of multiple citrus proteins belonging to the innate immune pathways.  These inhibitory interactions led to a high level of reactive oxygen species (ROS), blocking of bactericidal lipid binding protein (LTP), and induction of premature programmed cell death (PCD), thereby supporting CLas infection and HLB pathogenesis.  Finally, an LTP mimic was designed to sequester and block the CLas effector and to rescue the bactericidal activity of LTP.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.22.469535', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'EZH2 inhibition in glioblastoma stem cells increases the expression of neuronal genes and the neuronal developmental regulators ZIC2, ZNF423 and MAFB', 'authors': \"[{'name': 'Bnar Abdul Kader'}, {'name': 'Rebecca Distefano'}, {'name': 'Katherine L. West'}, {'name': 'Adam G. West'}]\", 'abstract': '<p>Glioblastoma multiforme (GBM) is an aggressive brain cancer with a very poor prognosis. It has been shown that GBM stem cells within a GBM tumour have increased resistance to standard therapies, so new approaches are needed to increase the range of treatment options available. Here we use two GBM stem cell lines, representing the classical/pro-neural and mesenchymal GBM subtypes, to investigate the effects of three different EZH2 inhibitors on GBM stem cell survival and gene expression: EPZ6438, GSK343 and UNC1999. EZH2 is the catalytic component of the PRC2 chromatin repressor complex, which represses transcription through methylation of histone H3 at lysine 27. Both cell lines showed significantly reduced colony formation after 48-hour exposure to the inhibitors, indicating they were sensitive to all three EZH2 inhibitors. RNA-seq analysis revealed that all three EZH2 inhibitors led to increased expression of genes related to neurogenesis and/or neuronal structure in both GBM stem cell lines. Chromatin immunoprecipitation (ChIP-Seq) was used to identify potential direct targets of the histone methylation activity of EZH2 that might be driving the increase in neuronal gene expression. Three genes were identified as candidate regulatory targets common to both cell lines: MAFB, ZIC2 and ZNF423. These transcription factors all have known roles in regulating neurogenesis, brain development and/or neuronal function. Through analysis of three different EZH2 inhibitors and two GBM stem cell lines, this study demonstrates a common underlying mechanism for how inhibition of EZH2 activity reduces GBM stem cell proliferation and survival.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469552', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': True, 'title': 'Stabilization of the SARS-CoV-2 Receptor Binding Domain by Protein Core Redesign and Deep Mutational Scanning', 'authors': \"[{'name': 'Alison C. Leonard'}, {'name': 'Jonathan J. Weinstein'}, {'name': 'Paul J. Steiner'}, {'name': 'Annette H. Erbse'}, {'name': 'Sarel J. Fleishman'}, {'name': 'Timothy A. Whitehead'}]\", 'abstract': '<p>Stabilizing antigenic proteins as vaccine immunogens or diagnostic reagents is a stringent case of protein engineering and design as the exterior surface must maintain recognition by receptor(s) and antigenspecific antibodies at multiple distinct epitopes. This is a challenge, as stability-enhancing mutations must be focused on the protein core, whereas successful computational stabilization algorithms typically select mutations at solvent-facing positions. In this study we report the stabilization of SARS-CoV-2 Wuhan Hu-1 Spike receptor binding domain (S RBD) using a combination of deep mutational scanning and computational design, including the FuncLib algorithm. Our most successful design encodes I358F, Y365W, T430I, and I513L RBD mutations, maintains recognition by the receptor ACE2 and a panel of different anti-RBD monoclonal antibodies, is between 1-2C more thermally stable than the original RBD using a thermal shift assay, and is less proteolytically sensitive to chymotrypsin and thermolysin than the original RBD. Our approach could be applied to the computational stabilization of a wide range of proteins without requiring detailed knowledge of active sites or binding epitopes, particularly powerful for cases when there are multiple or unknown binding sites.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469940', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Cryo-EM structure of hexameric yeast Lon (PIM1) highlights importance of conserved structural elements', 'authors': \"[{'name': 'Jie Yang'}, {'name': 'Albert S. Song'}, {'name': 'R. Luke Wiseman'}, {'name': 'Gabriel C. Lander'}]\", 'abstract': '<p>Lon protease is a conserved ATP-dependent serine protease composed of an AAA+ domain that mechanically unfolds substrates and a serine protease domain that degrades unfolded substrates. In yeast, dysregulation of Lon protease (PIM1) attenuates lifespan and leads to gross mitochondrial morphologic perturbations. Although structures of bacterial and human Lon protease reveal a hexameric assembly, PIM1 was speculated to form a heptameric assembly, and is uniquely characterized by a ~50 residue insertion between the ATPase and protease domains. To understand the yeast-specific properties of PIM1, we determined a high-resolution cryo-EM structure of PIM1 in a substrate-translocating state. Here, we reveal that PIM1 forms a hexamer, conserved with that of bacterial and human Lon proteases, wherein the ATPase domains form a canonical closed spiral that enables pore loop residues to translocate substrate to the protease chamber. In the substrate-translocating state, PIM1 protease domains form a planar protease chamber in an active conformation and are uniquely characterized by a ~15 residue C-terminal extension. These additional C-terminal residues form an alpha-helix that is located along the base of the protease domain. Finally, we did not observe density for the yeast-specific insertion between the ATPase and protease domains, likely due to high conformational flexibility. Biochemical studies to investigate the insertion using constructs that truncated or replaced the insertion with a glycine-serine linker suggest that the yeast-specific insertion is dispensable for PIM1s enzymatic function. Altogether, our structural and biochemical studies highlight unique components of PIM1 machinery and demonstrate evolutionary conservation of Lon protease function.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.17.21266441', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Rheumatology', 'covid': True, 'title': 'Pausing methotrexate improves immunogenicity of COVID-19 vaccination in patients with rheumatic diseases', 'authors': \"[{'name': 'AN Arumahandi de Silva'}, {'name': 'LM Frommert'}, {'name': 'FN Albach'}, {'name': 'J Klotsche'}, {'name': 'V Scholz'}, {'name': 'LM Jeworowski'}, {'name': 'T Schwarz'}, {'name': 'A ten Hagen'}, {'name': 'J Zernicke'}, {'name': 'VM Corman'}, {'name': 'C Drosten'}, {'name': 'GR Burmester'}, {'name': 'R Biesen'}]\", 'abstract': '<sec><title>Objective</title><p>To study the effect of methotrexate (MTX) and its discontinuation on the humoral immune response after COVID-19 vaccination in patients with autoimmune rheumatic diseases (AIRD).</p></sec><sec><title>Methods</title><p>Neutralising SARS-CoV-2 antibodies were measured after second vaccination in 64 rheumatic patients on methotrexate therapy, 31 of whom temporarily paused medication without a fixed regimen. The control group consisted of 21 AIRD patients without immunosuppressive medication.</p></sec><sec><title>Results</title><p>MTX patients showed a significantly lower median antibody response compared to AIRD patients without immunosuppressive therapy (p&lt; 0.001). Young age (&lt;60 years) and MTX-hold after vaccination were found to be the main factors influencing antibody response after vaccination, while BMI or MTX dose demonstrated no effect. For patients taking MTX, age correlated negatively with immune response (r=-0.49; p&lt;0.001) and all patients with antibody levels (14 %) below the cut-off were older than 60 years. Patients who held MTX during at least one vaccination showed significantly higher median neutralising antibody levels after second vaccination, compared to patients who continued MTX therapy during both vaccinations (68.82 %, 92.73 %, p=&lt;0.001). This effect was particularly pronounced in patients older than 60 years (p=0.0016). The impact of the time period after vaccination was greater than of the time before vaccination with the critical cut-off being 10 days.</p></sec><sec><title>Conclusion</title><p>MTX reduces the immunogenicity of SARS-CoV-2 vaccination in an age-dependent manner. Our data further suggest that holding MTX for at least 10 days after vaccination significantly improves the antibody response in patients over 60 years of age.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469798', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Microbial consortiums of putative degraders of low-density polyethylene-associated compounds in the ocean', 'authors': \"[{'name': 'Maria Pinto'}, {'name': 'Zihao Zhao'}, {'name': 'Katja Klun'}, {'name': 'Eugen Libowitzky'}, {'name': 'Gerhard J Herndl'}]\", 'abstract': '<p>Polyethylene (PE) is one of the most abundant plastics in the ocean. The development of a biofilm on PE in the ocean has been reported, yet whether some of the biofilm-forming organisms can biodegrade this plastic in the environment remains unknown. Via metagenomics analysis, we taxonomically and functionally analysed three biofilm communities using low-density polyethylene (LDPE) as their sole carbon source for two years. Several of the taxa that increased in relative abundance over time were closely related to known degraders of alkane and other hydrocarbons. Alkane degradation has been proposed to be involved in PE degradation, and most of the organisms increasing in relative abundance over time harboured genes encoding proteins essential in alkane degradation, such as the genes alkB and CYP153, encoding an alkane monooxygenase and a cytochrome P450 alkane hydroxylase. Weight loss of PE sheets when incubated with these communities and chemical and electron microscopic analyses provided evidence for alteration of the PE surface over time. Taken together, these results provide evidence for the utilization of LDPE-associated compounds by the prokaryotic communities. This study identifies a group of genes potentially involved in the degradation of the LDPE polymeric structure and/or associated plastic additives in the ocean and describes a phylogenetically diverse community of plastic biofilm-dwelling microbes with the potential of utilizing LDPE-associated compounds as carbon and energy source.</p><sec><title>Importance</title><p>Low-density polyethylene (LDPE) is one of the most used plastics worldwide and a large portion of it ends up in the ocean. Very little is known about its fate in the ocean and whether it can be biodegraded by microorganisms. By combining 2-year incubations with metagenomics, respiration measurements and LDPE surface analysis, we identified bacteria and associated genes and metabolic pathways potentially involved in LDPE biodegradation. After two years of incubation, two of the microbial communities exhibited a very similar taxonomic composition mediating changes to the LDPE pieces they were incubated with. We provide evidence that there are plastic-biofilm dwelling bacteria in the ocean, that might have the potential to degrade LDPE-associated compounds, and that alkane degradation pathways might be involved.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469779', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Loss of CEP162 function at the primary cilium delays ciliogenesis and causes retinal ciliopathy in humans', 'authors': \"[{'name': 'Nafisa Nuzhat'}, {'name': 'Kristof Van Schil'}, {'name': 'Sandra Liakopoulos'}, {'name': 'Miriam Bauwens'}, {'name': 'Alfredo Duenas Rey'}, {'name': 'Stephan Kseberg'}, {'name': 'Melanie Jger'}, {'name': 'Jason R. Willer'}, {'name': 'Jennifer Winter'}, {'name': 'Hanh Truong'}, {'name': 'Nuria Gruartmoner'}, {'name': 'Mattias Van Heetvelde'}, {'name': 'Joachim Wolf'}, {'name': 'Robert Merget'}, {'name': 'Sabine Grasshoff-Derr'}, {'name': 'Jo Van Dorpe'}, {'name': 'Anne Hoorens'}, {'name': 'Heidi Sthr'}, {'name': 'Luke Mansard'}, {'name': 'Anne-Franoise Roux'}, {'name': 'Thomas Langmann'}, {'name': 'Katharina Dannhausen'}, {'name': 'David Rosenkranz'}, {'name': 'Karl Martin Wissing'}, {'name': 'Michel Van Lint'}, {'name': 'Heidi Rossmann'}, {'name': 'Friederike Huser'}, {'name': 'Peter Nrnberg'}, {'name': 'Holger Thiele'}, {'name': 'Ulrich Zechner'}, {'name': 'Jillian N. Pearring'}, {'name': 'Elfride De Baere'}, {'name': 'Hanno J. Bolz'}]\", 'abstract': '<p>Ciliopathies often comprise retinal degeneration since the photoreceptors outer segment is an adapted primary cilium. CEP162 is a distal end centriolar protein required for proper transition zone assembly during ciliogenesis and whose loss causes ciliopathy in zebrafish. CEP162 has so far not been implicated in human disease. Here, we identified a homozygous <italic>CEP162</italic> frameshift variant, c.1935dupA (p.(E646R*5)), in retinitis pigmentosa patients from two unrelated Moroccan families, likely representing a founder allele. We found that even though mRNA levels were reduced, the truncated CEP162-E646R*5 protein was expressed and localized to the mitotic spindle during mitosis, but not at the basal body of the cilium. In <italic>CEP162</italic> knockdown cells, expression of the truncated CEP162-E646R*5 protein is unable to restore ciliation indicating its loss of function at the cilium. In patient fibroblasts, cilia overcome the absence of CEP162 from the primary cilium by delaying ciliogenesis through the persistence of CP110 at the mother centriole. The patient fibroblasts are ultimately able to extend some abnormally long cilia that are missing key transition zone components. Defective transition zone formation likely disproportionately affects the long-living ciliary outer segment of photoreceptors resulting in retinal dystrophy. CEP162 is expressed in human retina, and we show that wild-type CEP162, but not truncated CEP162-E646R*5, specifically localizes to the distal end of centrioles of mouse photoreceptor cilia. Together, our genetic, cell-based, and <italic>in vivo</italic> modeling establish that CEP162 deficiency causes retinal ciliopathy in humans.</p><sec><title>SIGNIFICANCE STATEMENT</title><p>Ciliopathies often comprise retinal degeneration since the photoreceptors outer segment is an adapted primary cilium. CEP162 is a basal body protein required for proper transition zone assembly during ciliogenesis and has so far not been implicated in human disease. Using genetic, cell-based, and <italic>in vivo</italic> studies, we show that a biallelic mutation of <italic>CEP162</italic> causes late-onset human retinal degeneration. This mutation specifically hinders CEP162 function at the cilia, leading to impaired transition zone assembly and delayed formation of dysmorphic cilia. Studies in mouse photoreceptors support that absence of CEP162 could lead to defective outer segments. The discovery of novel ciliopathy genes, such as <italic>CEP162,</italic> advances our insight into cell-specific functions of the primary cilium.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.17.21266461', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Space-time Classification Index for Assessing COVID-19 Hotspots', 'authors': \"[{'name': 'David Haynes'}, {'name': 'Chetan Tiwari'}]\", 'abstract': '<sec><title>Objectives</title><p>To develop new methods to address problems associated with use of traditional measures of disease surveillance, including prevalence and positivity rates.</p></sec><sec><title>Methods</title><p>We use data from the public New York Times Github repository to develop a space-time classification index of COVID-19 hotspots. The Local Indicator of Spatial Association (LISA) statistic is applied to identify daily clusters of COVID-19 cases, from July 4th to July 19th.</p></sec><sec><title>Results</title><p>The classification index is a spatial and temporal assessment tool that seeks to incorporate temporal trends of the clusters that are high-high and high-low. Two classifications support the index: severity and temporal duration. We define severity as the number of times a county is statistically significant and temporal duration captures the number of consecutive days a county is a hotspot.</p></sec><sec><title>Conclusions</title><p>The space-time classification index provides a statistically robust measure of the spatial patterns of COVID-19 hotspots. Spatial information is not captured through measures like the positivity rate, which merely divides the number of cases by tests conducted. The index proposed in this paper can guide intervention efforts by classifying counties with six-levels of importance.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.26.470078', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Reversible brain edema in experimental cerebral malaria is associated with transcellular blood-brain barrier disruption and delayed microhemorrhages', 'authors': \"[{'name': 'Angelika Hoffmann'}, {'name': 'Jessica Jin'}, {'name': 'Mame Aida'}, {'name': 'Chi Ho Wai'}, {'name': 'Sanjib Mohanty'}, {'name': 'Praveen K. Sahu'}, {'name': 'Rajyabardham Pattnaik'}, {'name': 'Lukas Pirpamer'}, {'name': 'Manuel Fischer'}, {'name': 'Sabine Heiland'}, {'name': 'Michael Lanzer'}, {'name': 'Friedrich Frischknecht'}, {'name': 'Ann-Kristin Mueller'}, {'name': 'Johannes Pfeil'}, {'name': 'Megharay Majhi'}, {'name': 'Marek Cyrklaff'}, {'name': 'Samuel C. Wassmer'}, {'name': 'Martin Bendszus'}]\", 'abstract': '<p>Brain swelling occurs in cerebral malaria (CM) and may either reverse or result in fatal outcome. It is currently unknown how brain swelling in CM reverses, as investigations have been hampered by inadequate animal models. In this study, we show that reversible brain swelling in experimental murine cerebral malaria (ECM) can be induced reliably after single vaccination with radiation-attenuated sporozoites as revealed by  in vivo  high-field (9.4T) magnetic resonance imaging. Our results provide evidence that parenchymal fluid increase and consecutive brain swelling results from transcellular blood-brain barrier disruption (BBBD), as revealed by electron microscopy. This mechanism enables reversal of brain swelling but does not prevent persistent focal brain damage, evidenced by microhemorrhages, in areas of most severe BBBD. In a cohort of 27 pediatric and adult CM patients (n=4 fatal, n=23 non-fatal) two out of four fatal CM patients (50%) and 8 out of 23 non-fatal CM patients (35%) showed microhemorrhages on MRI at clinical field strength of 1.5T, emphasizing the translational potential of the experimental model.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.21.469450', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Reduction in population size and not a shift in parasite community affect evolution of immune genes in island birds', 'authors': \"[{'name': 'Mathilde Barthe'}, {'name': 'Claire Doutrelant'}, {'name': 'Rita Covas'}, {'name': 'Martim Melo'}, {'name': 'Juan Carlos Illera'}, {'name': 'Marie-Ka Tilak'}, {'name': 'Constance Colombier'}, {'name': 'Thibault Leroy'}, {'name': 'Claire Loiseau'}, {'name': 'Benoit Nabholz'}]\", 'abstract': '<p>Shared ecological conditions encountered by species that colonize islands often lead to the evolution of convergent phenotypes, commonly referred to as island syndrome. Reduced immune functions have been previously proposed to be part of the island syndrome, as a consequence of the reduced diversity of pathogens on island ecosystems. According to this hypothesis, immune genes are expected to exhibit genomic signatures of relaxed selection pressure in island species. In this study, we used comparative genomic methods to study immune genes in island species (N = 20) and their mainland relatives (N = 14). We gathered public data as well as generated new data on innate (Toll-Like Receptors, Beta Defensins) and acquired immune genes (Major Histocompatibility Complex classes I and II), but also on hundreds of genes annotated as involved in various immune functions. As a control, we used a set of 97 genes not involved in immune functions, to account for the lower effective population sizes in island species. We used synonymous and non-synonymous variations to estimate the selection pressure acting on immune genes. For the genes evolving under balancing selection, we used simulation to estimate the impact of population size variation. We found a significant effect of drift on immune genes of island species leading to a reduction in genetic diversity and efficacy of selection. However, the intensity of relaxed selection was not significantly different from control genes, except for MHC class II genes. These genes exhibit a significantly higher level of non-synonymous loss of polymorphism than expected assuming only drift and an evolution under frequency dependent selection, possibly due to a reduction of extracellular parasite communities on islands. Overall, our results showed that demographic effects lead to a decrease in the immune functions of island species, but the relaxed selection caused by a reduced parasite pressure may only occur in some immune genes categories.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.19.469282', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The gendered self: Evidence for differences in whole-brain dynamics', 'authors': \"[{'name': 'Carme Uribe'}, {'name': 'Anira Escrichs'}, {'name': 'Eleonora de Filippi'}, {'name': 'Yonatan Sanz-Perl'}, {'name': 'Carme Junque'}, {'name': 'Esther Gomez-Gil'}, {'name': 'Morten L Kringelbach'}, {'name': 'Antonio Guillamon'}, {'name': 'Gustavo Deco'}]\", 'abstract': '<p>How the brain constructs gender identity is largely unknown, but some neural differences have recently been discovered. Here, we used an intrinsic-ignition framework to investigate if gender identity changes the propagation of the neural activity across the whole-brain network and within resting-state networks. Studying 29 transmen and 17 transwomen with gender incongruence, 22 ciswomen, and 19 cismen, we computed the capability of a given brain area in space to propagate activity to other areas (mean-ignition) and its variability across time (node-metastability). We found that both measures differentiated all four groups across the whole-brain network. Furthermore, at the network level, we found that compared to the other groups, cismen showed higher mean-ignition of the dorsal attention network and node-metastability of the dorsal and ventral attention, executive control, and temporal parietal networks. We also found mean-ignition differences between cismen and ciswomen within the executive control network, but higher in ciswomen than cismen and transmen for the default-mode network. For the node-metastability, this was higher in cismen compared to ciswomen in the somatomotor network, while both mean-ignition and node-metastability were higher for cismen than transmen in the limbic network. Finally, we computed correlations between both measures and their body image scores. Transmen dissatisfaction, cismen, and ciswomen satisfaction towards their own body image were distinctively associated with specific networks per group. Overall, the study of the whole-brain network dynamical complexity discriminates binary gender identity groups, and functional connectivity dynamics approaches are needed to disentangle the complex understanding of the gendered self.</p><sec><title>Significance statement</title><p>The study of sex/gender differences may be enriched by the heterogeneity of other gender minority groups, such as transgender. Functional connectivity measures capturing the spatio-temporal oscillations of the brain can provide insights on how the brain cooperates. This is the first study investigating how the whole-brain network propagates information across the brain, spatially and temporally, in binary gender groups (cisgender and transgender) by means of the intrinsic-ignition framework. We found four whole-brain unique phenotypes pertaining to each gender group, namely cismen, ciswomen, transmen and transwomen. Novel functional connectivity dynamics frameworks can contribute to disentangle the complex experience of a <italic>gendered</italic>-self.</p></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.24.21266682', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Uptake of Covid-19 preventive measures among 10 immigrant ethnic groups in Norway', 'authors': \"[{'name': 'Abdi Gele'}, {'name': 'Naima Said Sheikh'}, {'name': 'Prabhjot Kour'}, {'name': 'Samera A Qureshi'}]\", 'abstract': '<sec><title>Background</title><p>A pessimistic view of the impact of Covid-19 on immigrants has generated an interest in exploring the role of socio-economic and cultural factors on excess infection, hospitalization and death among immigrants. Nowhere in the world is such interest more palpable than in Western countries, including Norway. An expanding amount of literature has demonstrated that preexisting socio-economic inequalities have affected Covid-19 control programs through a disruption of immigrants uptake to preventive measures. Nonetheless, until very recently, no qualitative research has been conducted to address the impact of socio-economic and socio-cultural factors on immigrants uptake on preventive measures of Covid-19 in Norway.</p></sec><sec><title>Methods</title><p>An interview-based qualitative study consisting of 88 participants (49 women and 39 men) from 10 immigrant ethnic groups were carried out. Participants were recruited through purposive sampling and snowballing. In-depth interviews were held through telephone or online for those who have experience in the use of zoom or teams. Data were analyzed using thematic analysis</p></sec><sec><title>Results</title><p>We found that participants attitudes toward the pandemic in general, and more specifically their adherence to preventive measures, have increased over time. However, the number of barriers that hinder immigrants from adhering to preventive measures were identified and classified more broadly into three main subthemes: 1) socio-economic barriers; 2) socio-cultural barriers, and 3) other barriers. Socio-economic barriers include overcrowded households, working in first-line jobs, education and language. Socio-cultural barriers include collectivist culture, religious fatalism and risk perception toward the pandemic.</p></sec><sec><title>Conclusion</title><p>To reduce the health inequality that arises from overcrowded housing, there is a need for a long-term strategy to help improve the housing situation of low-income immigrant families that live in overcrowded households. In addition, increasing health literacy and more generally, the integration of immigrants, may also reduce the effect of socio-cultural factors on an immigrants uptake of preventive measures.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.21266761', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': True, 'title': 'Higher Limbic and Basal Ganglia volumes in surviving COVID-negative patients and the relations to fatigue', 'authors': \"[{'name': 'Rakibul Hafiz'}, {'name': 'Tapan Kumar Gandhi'}, {'name': 'Sapna Mishra'}, {'name': 'Alok Prasad'}, {'name': 'Vidur Mahajan'}, {'name': 'Xin Di'}, {'name': 'Benjamin H. Natelson'}, {'name': 'Bharat B. Biswal'}]\", 'abstract': '<p>Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) has caused a global pandemic. Among several systemic abnormalities, little is known about the critical attack on the central nervous system (CNS). Several patient reports with multiple pathologies  ischemic strokes, mild infarcts, encephalitis, cerebro-vascular abnormalities, cerebral inflammation, and loss of consciousness, indicate CNS involvement. However, due to limited neuroimaging studies, conclusive group level effects are scarce in the literature and replication studies are necessary to verify if these effects persist in surviving acute-COVID patients. Furthermore, recent reports indicate fatigue is highly prevalent among slowly recovering patients. How early structural changes relate to fatigue need to be investigated. Our goal was to address this by scanning COVID subjects two weeks after hospital discharge. We hypothesized these surviving patients will demonstrate altered gray matter volume (GMV) when compared to healthy controls and further demonstrate correlation of GMV with fatigue. Voxel-based morphometry was applied to T1-weighted MRI images between 46 patients with COVID and 32 healthy controls. Significantly higher GMV in the <italic>Limbic System</italic> and <italic>Basal Ganglia</italic> regions were observed in surviving COVID-19 patients when compared to healthy controls. Moreover, within the patient group, there was a significant positive correlation between GMV and self-reported fatigue scores during work, within the <italic>ventral Basal Ganglia</italic> and <italic>Ventromedial Prefrontal Cortex</italic> regions. Therefore, our results align with both single case acute patient reports and current group level neuroimaging findings. Finally, we newly report a positive correlation of GMV with fatigue in COVID survivors.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469505', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genome-wide association studies and genomic selection assays made in a large sample of cacao (Theobroma cacao L.) germplasm reveal significant marker-trait associations and good predictive value for improving yield potential', 'authors': \"[{'name': 'Frances L. Bekele'}, {'name': 'Gillian G. Bidaisee'}, {'name': 'Mathilde Allegre'}, {'name': 'Xavier Argout'}, {'name': 'Olivier Fouet'}, {'name': 'Michel Boccara'}, {'name': 'Duraisamy Saravanakumar'}, {'name': 'Isaac Bekele'}, {'name': 'Claire Lanaud'}]\", 'abstract': '<p>A genome-wide association study was undertaken to unravel marker-trait associations (MTAs) between SNP markers and yield-related traits. It involved a subset of 421 cacao accessions from the large and diverse collection conserved <italic>ex situ</italic> at the International Cocoa Genebank Trinidad. An average linkage disequilibrium (r<sup>2</sup>) of 0.10 at 5.2 Mb was found across several chromosomes. Seventeen significant (<italic>P</italic>  8.17  10<sup>-5</sup> (log10 (p) = 4.088)) MTAs of interest, which accounted for 5 to 17% of the explained phenotypic variation, were identified using a Mixed Linear Model in TASSEL version 5.2.50. The most significant MTAs identified were related to seed number and seed length on chromosome 7 and seed number on chromosome 1. Other significant MTAs involved seed length to width ratio on chromosomes 3 and 5 and seed length on chromosomes 4 and 9. It was noteworthy that several yield-related traits, <italic>viz</italic>., seed length, seed length to width ratio and seed number were associated with markers on different chromosomes, indicating their polygenic nature. Approximately 40 candidate genes that encode embryo and seed development, protein synthesis, carbohydrate transport and lipid biosynthesis and transport were identified in this study. A significant association of fruit surface anthocyanin intensity co-localised with MYB-related protein 308 on chromosome 4. Testing of a genomic selection approach revealed good predictive value (GEBV) for economic traits such as seed number (GEBV = 0.611), seed length (0.6199), seed width (0.5435), seed length to width ratio (0.5503), seed/cotyledon mass (0.6014) and ovule number (0.6325). The findings of this study could facilitate genomic selection and marker-assisted breeding of cacao thereby expediting improvement in the yield potential of cacao planting material.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469750', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Treatment of infection-induced vascular pathologies is protective against persistent rough morphotype Mycobacterium abscessus infection in zebrafish', 'authors': \"[{'name': 'Julia Y Kam'}, {'name': 'Kathryn Wright'}, {'name': 'Warwick J Britton'}, {'name': 'Stefan H Oehlers'}]\", 'abstract': '<p><italic>Mycobacterium abscessus</italic> infections are of increasing global prevalence and are often difficult to treat due to complex antibiotic resistance profiles. While there are similarities between the pathogenesis of <italic>M. abscessus</italic> and tuberculous mycobacteria, including granuloma formation and stromal remodeling, there are distinct molecular differences at the host-pathogen interface. Here we have used a zebrafish-<italic>M. abscessus</italic> model and host-directed therapies that were previously identified in the zebrafish-<italic>M. marinum</italic> model to identify potential host-directed therapies against <italic>M. abscessus</italic> infection. We find efficacy of anti-angiogenic and vascular normalizing therapies against rough <italic>M. abscessus</italic> infection, but no effect of anti-platelet drugs.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469498', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Stochastic Simulation Algorithm for effective spreading dynamics on Time-evolving Adaptive NetworX (SSATAN-X)', 'authors': \"[{'name': 'Nadezdha Malysheva'}, {'name': 'Max von Kleist'}]\", 'abstract': '<p>Modelling and simulating the dynamics of pathogen spreading has been proven crucial to inform public heath decisions, containment strategies, as well as cost-effectiveness calculations. Pathogen spreading is often modelled as a stochastic process that is driven by pathogen exposure on time-evolving contact networks. In <italic>adaptive networks</italic>, the spreading process depends not only on the dynamics of a contact network, but vice versa, infection dynamics may alter risk behaviour and thus feed back onto contact dynamics, leading to emergent complex dynamics. However, stochastic simulation of pathogen spreading processes on <italic>adaptive networks</italic> is currently computationally prohibitive.</p><p>In this manuscript, we propose SSATAN-X, a new algorithm for the accurate stochastic simulation of pathogen spreading on <italic>adaptive networks</italic>. The key idea of SSATAN-X is to only capture the contact dynamics that are relevant to the spreading process. We show that SSATAN-X captures the contact dynamics and consequently the spreading dynamics accurately. The algorithm achieves up to 100 fold speed-up over the state-of-art stochastic simulation algorithm (SSA). The speed-up with SSATAN-X further increases when the contact dynamics are fast in relation to the spreading process, i.e. if contacts are short-lived and per-exposure infection risks are small, as applicable to most infectious diseases.</p><p>We envision that SSATAN-X may extend the scope of analysis of pathogen spreading on <italic>adaptive networks</italic>. Moreover, it may serve to create benchmark data sets to validate novel numerical approaches for simulation, or for the data-driven analysis of the spreading dynamics on <italic>adaptive networks</italic>. A C++ implementation of the algorithm is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/nmalysheva/SSATAN-X\">https://github.com/nmalysheva/SSATAN-X</ext-link>.</p><sec><title>Author summary</title><p>Modelling and simulating of infectious disease spreading supports public heath decisions, such as prevention and containment strategies and allows to perform cost-effectiveness calculations. Detailed modelling approaches consider stochastic pathogen spreading on time-evolving contact networks. In <italic>adaptive networks</italic>, the spreading process depends not only on the dynamics of a contact network, but vice versa, infection dynamics may alter risk behaviour and thus feed back onto contact dynamics.</p><p>Stochastic simulation of these complex dynamics is currently computationally prohibitive.</p><p>We propose a new algorithm (SSATAN-X) that can significantly speed up stochastic simulations on <italic>adaptive networks</italic>, while being accurate at the same time. Our algorithm achieves this speed-up by only considering the contact dynamics that are relevant to the spreading process. The benefit of algorithm is particularly pronounced when contacts are short-lived and per-exposure infection risks are small, which is applicable to most infectious diseases.</p><p>We envision that SSATAN-X may allow simulation and analysis of pathogen spreading on more complex <italic>adaptive networks</italic> than previously possible. Moreover, data sets may be created with SSATAN-X that are useful for benchmarking novel numerical schemes and analytic approaches.</p></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469748', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Demyelination induces transcriptional reprograming in proprioceptive and A rapidly adapting low-threshold-mechanoreceptor neurons', 'authors': \"[{'name': 'Benayahu Elbaz'}, {'name': 'Lite Yang'}, {'name': 'Braesen Lee Rader'}, {'name': 'Riki Kawaguchi'}, {'name': 'Maria Traka'}, {'name': 'Clifford J Woolf'}, {'name': 'William Renthal'}, {'name': 'Brian Popko'}]\", 'abstract': '<p>Schwann cells, the main glial cell in the peripheral nervous system (PNS), ensheath bundles of small unmyelinated axons or form myelin on larger axons. PNS injuries initiate transcriptional reprograming in both Schwann cells and sensory neurons that promotes regeneration. While the factors that initiate the transcriptional reprograming in Schwann cells are well characterized, the full range of stimuli that initiate this reprograming in sensory neurons remain elusive. Here, using a genetic model of Schwann cell ablation, we find that Schwann cell loss results in transient PNS demyelination without overt axonal loss. By profiling sensory ganglia at single-cell resolution we show that this demyelination induces transcriptional reprogramming preferably in proprioceptive and A RA-LTMR neurons. Transcriptional reprograming is assumed to be a cell autonomous response of sensory neurons to mechanical axonal injury. By identifying similar reprograming in non-injured, demyelinated neurons, our study suggests that this reprograming represents a non-cell autonomous transcriptional response of sensory neurons to the loss of axon-Schwann cell interactions.</p><sec><title>Highlights</title><list list-type=\"order\"><list-item><p>Ablation of Schwann cells results in transient PNS demyelination, without overt axonal loss.</p></list-item><list-item><p>Schwann cell loss results in transcriptional reprograming in specific sensory neurons.</p></list-item><list-item><p>Spinal nerve transection (mechanical injury of axons) and demyelination (intact axons) induces similar transcriptional responses in DRG neurons.</p></list-item><list-item><p>The transcriptional response to demyelination among DRG neurons is specific to the large myelinated proprioceptive and A RA-LTMR neurons.</p></list-item></list></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.24.469858', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'High diversity in the regulatory region of Stx-converting bacteriophage genomes', 'authors': \"[{'name': 'Annette Fagerlund'}, {'name': 'Marina Aspholm'}, {'name': 'Grzegorz Wgrzyn'}, {'name': 'Toril Lindbck'}]\", 'abstract': '<p>Shiga toxin (Stx) is the major virulence factor of enterohemorrhagic <italic>Escherichia coli</italic> (EHEC), and the <italic>stx</italic> genes are carried by temperate bacteriophages (Stx phages). The switch between lysogenic and lytic life cycle of the phage, which is crucial for Stx production and for severity of the disease, is regulated by the CI repressor. CI maintain latency by preventing transcription of the replication proteins. Three <underline>E</underline>HEC phage <underline>r</underline>eplication <underline>u</underline>nits (Eru1-3) in addition to the classical lambdoid replication region have been described previously, and Stx phages carrying the Eru1 replication region were associated with highly virulent EHEC strains. In this study, we have classified the Eru replication region of 419 Stx phages. In addition to the lambdoid replication region and the three already described Erus, ten novel Erus (named Eru4 to Eru13) were detected. The lambdoid type, Eru1, Eru4 and Eru7 seem to be widely distributed in Western Europe. Notably, EHEC strains involved in severe outbreaks in England and Norway carry Stx phages with Eru1, Eru2, Eru5 and Eru7 replication regions. Phylogenetic analysis of CI repressors from Stx phages revealed eight major clades that largely separate according to Eru type. The classification of replication regions and CI proteins of Stx phages provides an important platform for further studies aimed to assess how characteristics of the replication region influence the regulation of phage life cycle and, consequently, the virulence potential of the host EHEC strain.</p><sec><title>IMPORTANCE</title><p>EHEC is an emerging health challenge worldwide and outbreaks caused by this pathogen tend to be more frequent and severe. Increased knowledge on how characteristics of the replication region influence the virulence of <italic>E. coli</italic> may be used for more precise identification of high-risk EHEC strains.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469602', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Loss of hepatic Lgr4 and Lgr5 promotes nonalcoholic fatty liver disease', 'authors': \"[{'name': 'Enrica Saponara'}, {'name': 'Carlos Penno'}, {'name': 'Meztli L. Matadamas Guzmn'}, {'name': 'Virginie Brun'}, {'name': 'Benoit Fischer'}, {'name': 'Margaret Brousseau'}, {'name': 'Zhong-Yi Wang'}, {'name': 'Peter ODonnell'}, {'name': 'Jonathan Turner'}, {'name': 'Alexandra Graff Meyer'}, {'name': 'Laura Bollepalli'}, {'name': 'Giovanni dArio'}, {'name': 'Guglielmo Roma'}, {'name': 'Walter Carbone'}, {'name': 'Vanessa Orsini'}, {'name': 'Stefano Annunziato'}, {'name': 'Michael Obrecht'}, {'name': 'Nicolau Beckmann'}, {'name': 'Chandra Saravanan'}, {'name': 'Arnaud Osmont'}, {'name': 'Philipp Tropberger'}, {'name': 'Shola Richards'}, {'name': 'Christel Genoud'}, {'name': 'Alexandra Aebi'}, {'name': 'Svenja Ley'}, {'name': 'Iwona Ksiazek'}, {'name': 'Florian Nigsch'}, {'name': 'Luigi Terraciano'}, {'name': 'Tewis Bouwmeester'}, {'name': 'Jan Tchorz'}, {'name': 'Heinz Ruffner'}]\", 'abstract': '<sec><title>Background &amp; Aims</title><p>The Rspo-Lgr4/5-Znrf3/Rnf43 module is a master regulator of hepatic Wnt/-catenin signaling and metabolic zonation, but its impact on nonalcoholic fatty liver disease (NAFLD) remains unclear. We studied whether liver-specific loss of the Wnt/-catenin modulators Leucine-Rich Repeat-Containing G Protein-Coupled Receptor 4/5 (Lgr4/5) promotes nonalcoholic fatty liver disease (NAFLD).</p></sec><sec><title>Methods</title><p>Mice with liver-specific deletion of both receptors Lgr4/5 (Lgr4/5dLKO) were fed with normal diet (ND) or high fat diet (HFD). Livers of these mice were analyzed for lipid and fibrotic content by tissue staining and immunohistochemistry (IHC), and lipoproteins, inflammation and liver enzyme markers were measured in blood. Mechanistic insights into hepatic lipid accumulation were obtained by using <italic>ex vivo</italic> primary hepatocyte cultures derived from the Lgr4/5dLKO mice. Lipid analysis of mouse livers was performed by mass spectrometry (MS)-based untargeted lipidomic analysis.</p></sec><sec><title>Results</title><p>We demonstrated that liver-specific ablation of Lgr4/5-mediated Wnt signaling resulted in hepatic steatosis, impaired bile acid (BA) secretion and predisposition to liver fibrosis. Under HFD conditions, we observed progressive intrahepatic fat accumulation, developing into macro-vesicular steatosis. Serum lipoprotein levels in HFD-fed Lgr4/5dLKO mice were decreased, rather than increased, suggesting that accumulation of fat in the liver was due to impaired lipid secretion by hepatocytes. Our lipidome analysis revealed a severe alteration of several lipid species in livers of Lgr4/5dLKO mice, including triacylglycerol estolides (TG-EST), a storage form of bioactive free fatty acid (FA) esters of hydroxy FAs (FAHFAs).</p></sec><sec><title>Conclusions</title><p>Loss of hepatic Wnt/-catenin activity by Lgr4/5 deletion led to deregulation of lipoprotein pathways, loss of BA secretion, intrinsic alterations of lipid homeostasis and the onset of NAFLD.</p></sec><sec><title>Lay summary</title><p>The Wnt/-catenin pathway plays an important role during development and tissue homeostasis. Loss of Wnt/-catenin activity in mouse liver leads to loss of liver zonation, but the impact on nonalcoholic fatty liver disease (NAFLD) remains unclear. We show that livers of mice developed steatosis upon deletion of the positive pathway regulators Lgr4/5. Livers of knock-out (KO) mice exhibited altered lipid composition due to impaired lipid secretion. Furthermore, livers of these mice developed a nonalcoholic steatohepatitis (NASH)-like phenotype and fibrotic features derived from activated hepatic stellate cells. Our data demonstrate a protective role of Wnt/-catenin pathway activity towards the development of NAFLD.</p></sec><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Abrogation of hepatic Wnt/-catenin activity and liver zonation upon Lgr4/5 deletion in mice led to hepatic steatosis.</p></list-item><list-item><p>Liver fat accumulation was caused by impaired lipid secretion from hepatocytes.</p></list-item><list-item><p>Steatotic livers contained increased levels of diverse lipid species, including polyunsaturated fatty acids and triglycerol-estolides.</p></list-item><list-item><p>These data confirmed that a decrease in Wnt/-catenin signaling led to the development of nonalcoholic fatty liver disease (NAFLD) in mice.</p></list-item></list></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.27.470186', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Dopamine release in nucleus accumbens is under tonic inhibition by adenosine A1 receptors regulated by astrocytic ENT1 and dysregulated by ethanol', 'authors': \"[{'name': 'Bradley M Roberts'}, {'name': 'Elizabeth Lambert'}, {'name': 'Jessica A Livesey'}, {'name': 'Zhaofa Wu'}, {'name': 'Yulong Li'}, {'name': 'Stephanie J M Cragg'}]\", 'abstract': '<p>Striatal adenosine A1 receptor (A1R) activation can inhibit dopamine release. A1Rs on other striatal neurons are activated by an adenosine tone that is limited by equilibrative nucleoside transporter 1 (ENT1) that is enriched on astrocytes and is ethanol-sensitive. We explored whether dopamine release in nucleus accumbens core is under tonic inhibition by A1Rs, and is regulated by astrocytic ENT1 and ethanol. In ex vivo striatal slices from male and female mice, A1R agonists inhibited dopamine release evoked electrically or optogenetically and detected using fast-scan cyclic voltammetry, most strongly for lower stimulation frequencies and pulse numbers, thereby enhancing the activity-dependent contrast of dopamine release. Conversely, A1R antagonists reduced activity-dependent contrast but enhanced evoked dopamine release levels, even for single optogenetic pulses indicating an underlying tonic inhibition. The ENT1 inhibitor NBTI reduced dopamine release and promoted A1R-mediated inhibition, and conversely, virally-mediated astrocytic overexpression of ENT1 enhanced dopamine release and relieved A1R-mediated inhibition. By imaging the genetically encoded fluorescent adenosine sensor GRAB-Ado, we identified a striatal extracellular adenosine tone that was elevated by the ENT1 inhibitor and sensitive to gliotoxin fluorocitrate. Finally, we identified that ethanol (50 mM) promoted A1R-mediated inhibition of dopamine release, through diminishing adenosine uptake via ENT1. Together, these data reveal that dopamine output dynamics are gated by a striatal adenosine tone, limiting amplitude but promoting contrast, regulated by ENT1, and promoted by ethanol. These data add to the diverse mechanisms through which ethanol modulates striatal dopamine, and to emerging datasets supporting astrocytic transporters as important regulators of striatal function.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.22.469581', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Mechanically induced nuclear shuttling of -catenin requires co-transfer of actin', 'authors': \"[{'name': 'Buer Sen'}, {'name': 'Zhihui Xie'}, {'name': 'Sean Howard'}, {'name': 'Maya Styner'}, {'name': 'Andre J van Wijnen'}, {'name': 'Gunes Uzer'}, {'name': 'Janet Rubin'}]\", 'abstract': '<p>Mesenchymal stem cells (MSC) respond to environmental forces with both cytoskeletal re-structuring and activation of protein chaperones of mechanical information, -catenin and Yes-Associated Protein 1 (YAP1). To function, MSCs must differentiate between dynamic forces such as cyclic strains of extracellular matrix due to physical activity and static strains due to ECM stiffening. To delineate how MSCs recognize and respond differently to both force types, we compared effects of dynamic (200 cycles x 2%) and static (1  2% hold) strain on nuclear translocation of -catenin and YAP1 at 3h after force application. Dynamic strain induced nuclear accumulation of -catenin, and increased cytoskeletal actin structure and cell stiffness, but had no effect on nuclear YAP1 levels. Critically, both nuclear actin and nuclear stiffness increased along with dynamic strain-induced -catenin transport. Augmentation of cytoskeletal structure using either static strain or lysophosphatidic acid (LPA) did not increase nuclear content of catenin or actin, but induced robust nuclear increase in YAP1. As actin binds -catenin, we considered whether -catenin, which lacks a nuclear localization signal, was dependent on actin to gain entry to the nucleus. Knockdown of cofilin-1 (<italic>Cfl1</italic>) or importin-9 (<italic>Ipo9</italic>), which co-mediate nuclear transfer of G-actin, prevented dynamic strain-mediated nuclear transfer of both -catenin and actin. In sum, dynamic strain induction of actin re-structuring promotes nuclear transport of G-actin, concurrently supporting nuclear access of -catenin via mechanisms utilized for actin transport. Thus, dynamic and static strain activate alternative mechanoresponses reflected by differences in the cellular distributions of actin, -catenin and YAP1.</p><sec><title>Significance statement</title><p>Cells integrate both static and dynamic mechanical signals through the actin cytoskeleton which is attached to the nuclear envelope, affecting nuclear transport of -catenin and YAP1. Dynamic strain induces nuclear translocation of -catenin, but not YAP1, while static strain causes nuclear translocation of YAP1, but not -catenin. Importantly, nuclear transport of actin is induced by dynamic but not static force. Furthermore, nuclear import of -catenin depends on cofilin/importin-9 dependent actin transport mechanisms. Thus the presence of -catenin and YAP1 in the nucleus represent specific responses to regulatory mechanical signals.</p></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.26.470138', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Noise in a metabolic pathway leads to persister formation in Mycobacterium tuberculosis', 'authors': \"[{'name': 'Jeffrey Quigley'}, {'name': 'Kim Lewis'}]\", 'abstract': '<p>Tuberculosis is difficult to treat due to dormant cells in hypoxic granulomas, and stochastically-formed persisters tolerant of antibiotics. Bactericidal antibiotics kill by corrupting their energy-dependent targets. We reasoned that noise in the expression of an energy-generating component will produce rare persister cells. In sorted low ATP <italic>M. tuberculosis</italic> grown on acetate there is considerable cell-to-cell variation in the level of mRNA coding for AckA, the acetate kinase. Quenching the noise by overexpressing ackA sharply decreases persisters, showing that it acts as the main persister gene under these conditions. This demonstrates that a low energy mechanism is responsible for the formation of <italic>M. tuberculosis</italic> persisters and suggests that the mechanism of their antibiotic tolerance is similar to that of dormant cells in a granuloma. Entrance into a low energy state driven by stochastic variation in expression of energy-producing enzymes is likely a general mechanism by which bacteria produce persisters.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.22.469495', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The different brain areas occupied for integrating information of hierarchical linguistic units: a study based on EEG and TMS', 'authors': \"[{'name': 'Changfu Pei'}, {'name': 'Yuan Qiu'}, {'name': 'Fali Li'}, {'name': 'Xunan Huang'}, {'name': 'Yajing Si'}, {'name': 'Yuqin Li'}, {'name': 'Xiabing Zhang'}, {'name': 'Chunli Chen'}, {'name': 'Qiang Liu'}, {'name': 'Zehong Cao'}, {'name': 'Nai Ding'}, {'name': 'Shan Gao'}, {'name': 'Kimmo Alho'}, {'name': 'Dezhong Yao'}, {'name': 'Peng Xu'}]\", 'abstract': '<p>Human linguistic units are hierarchical, and our brain responds differently when processing linguistic units during sentence comprehension, especially when the modality of the received signal is different (auditory, visual, or audio-visual). However, it is unclear how the brain processes and integrates language information at different linguistic units (words, phrases, and sentences) provided simultaneously in audio and visual modalities. To address the issue, we presented participants with sequences of short Chinese sentences through auditory or visual or combined audio- visual modalities, while electroencephalographic responses were recorded. With a frequency tagging approach, we analyzed the neural representations of basic linguistic units (i.e., characters/monosyllabic words) and higher-level linguistic structures (i.e., phrases and sentences) across the three modalities separately. We found that audio-visual integration occurs at all linguistic units, and the brain areas involved in the integration varied across different linguistic levels. In particular, the integration of sentences activated the local left prefrontal area. Therefore, we used continuous theta-burst stimulation (cTBS) to verify that the left prefrontal cortex plays a vital role in the audio-visual integration of sentence information. Our findings suggest the advantage of bimodal language comprehension at hierarchical stages in language-related information processing and provide evidence for the causal role of the left prefrontal regions in processing information of audio-visual sentences.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.26.470089', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Homologous recombination repair creates mutations in the non-coding genome that alter Topoisomerase-1 cleavage sites & orchestrates irinotecan resistance', 'authors': \"[{'name': 'Santosh Kumar'}, {'name': 'Valid Gahramanov'}, {'name': 'Julia Yaglom'}, {'name': 'Shivani Patel'}, {'name': 'Lukasz Kaczmarczyk'}, {'name': 'Ivan Alexandrov'}, {'name': 'Gabi Gerlitz'}, {'name': 'Mali Salmon-Divon'}, {'name': 'Michael Y. Sherman'}]\", 'abstract': '<p>The selection of drug-resistant mammalian cell mutants requires multiple drug exposures. When cloned genetically identical cells are exposed to the drug, resistance is unlikely to result from the selection of pre-existent mutations. Therefore, adaptation must involve the generation of drug-resistant mutations de-novo. Understanding how adaptive mutations are generated and protect cells is important for our knowledge of cancer biology and evolution. Here, we studied the adaptation of cancer cells to topoisomerase (Top1) inhibitor irinotecan, which triggers DNA breaks, resulting in cytotoxicity. The resistance mechanism was based on the gradual accumulation of hundreds of thousands of recurrent mutations in non-coding DNA at sequence-specific Top1 cleavage sites. Repair of DSBs at these sites following initial irinotecan exposures created mutant sequences that were resistant to further Top1 cleavage. Therefore, by creating DNA breaks Top1 increases the rate of highly protective mutations specifically at such spots, thus explaining a puzzling need of dose escalation in resistance development.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.24.469962', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Deciphering heterogeneity of circulating epithelial cells in breast cancer patients', 'authors': \"[{'name': 'Maxim E. Menyailo'}, {'name': 'Viktoria R. Zainullina'}, {'name': 'Liubov A. Tashireva'}, {'name': 'Sofia Yu. Zolotareva'}, {'name': 'Tatiana S. Gerashchenko'}, {'name': 'Vladimir V. Alifanov'}, {'name': 'Olga E. Savelieva'}, {'name': 'Evgeniya S. Grigoryeva'}, {'name': 'Nataliya A. Tarabanovskaya'}, {'name': 'Nataliya O. Popova'}, {'name': 'Anna A. Khozyainova'}, {'name': 'Evgeny L. Choinzonov'}, {'name': 'Nadezhda V. Cherdyntseva'}, {'name': 'Vladimir M. Perelmuter'}, {'name': 'Evgeny V. Denisov'}]\", 'abstract': '<p>Circulating tumor cells (CTCs) and tumor hybrid cells, being the leading players in metastasis, have prognostic relevance and are potential antimetastatic targets. CTCs are identified as epithelial-positive and CD45 (leukocyte)-negative cells, whereas tumor hybrid cells usually have epithelial and leukocyte components. However, epithelial and hybrid cells are also observed in healthy subjects that complicate the detection of CTCs and tumor hybrid cells in cancer patients. This study evaluated the diversity of CD45-negative and CD45-positive circulating epithelial cells (CECs) in breast cancer patients (n=20) using single-cell RNA sequencing. We also tried to detect CTCs and tumor hybrid cells among CD45<sup></sup> CECs and CD45<sup>+</sup> CECs by analyzing DNA ploidy since aneuploidy is a hallmark of cancer cells. Aneuploid cells were predominantly detected in CD45<sup></sup> CECs, whereas CD45<sup>+</sup> CECs were mainly diploid. Most transcriptional cancer features, including many cancer-associated signaling pathways, were specific to aneuploid cells of one CD45<sup></sup> CEC cluster. These cells were also enriched by platelet genes and signaling pathways that may indicate their increased potential to adhere with thrombocytes. In CD45<sup>+</sup> CECs, only one cluster had many aneuploid cells that were surprisingly negative for transcriptional cancer features. Thus, CD45<sup></sup> and CD45<sup>+</sup> CECs are highly heterogeneous in breast cancer patients and consist of transcriptionally-distinct cell populations with varying degrees of DNA ploidy where aneuploid cells are likely CTCs and tumor hybrid cells.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.22.469359', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Visual Attention in The Fovea and The Periphery during Visual Search', 'authors': \"[{'name': 'Jie Zhang'}, {'name': 'Xiaocang Zhu'}, {'name': 'Shanshan Wang'}, {'name': 'Hossein Esteky'}, {'name': 'Yonghong Tian'}, {'name': 'Robert Desimone'}, {'name': 'Huihui Zhou'}]\", 'abstract': '<p>Visual search depends on both the foveal and peripheral visual system, yet the foveal attention mechanisms is still lack of insights. We simultaneously recorded the foveal and peripheral activities in V4, IT and LPFC, while monkeys performed a category-based visual search task. Feature attention enhanced responses of Face-selective, House-selective, and Non-selective foveal cells in visual cortex. While foveal attention effects appeared no matter the peripheral attention effects, paying attention to the foveal stimulus dissipated the peripheral feature attentional effects, and delayed the peripheral spatial attentional effects. When target features appeared both in the foveal and the peripheral, feature attention effects seemed to occur predominately in the foveal, which might not distribute across the visual field according to common view of distributed feature attention effects. As a result, the parallel attentive process seemed to occur during distractor fixations, while the serial process predominated during target fixations in visual search.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.26.470122', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'The role of four cholesterol-recognition motifs localized between amino acid residues 400-550 in regulating translocation and lytic activity of Adenylate Cyclase Toxin', 'authors': \"[{'name': 'Jone Amuategi'}, {'name': 'Rocio Alonso'}, {'name': 'Helena Ostolaza'}]\", 'abstract': '<p>Adenylate Cyclase Toxin (ACT or CyaA) is an important virulence factor secreted by Bordetella pertussis, the bacterium causative of whooping cough, playing an essential role in the establishment of infection in the respiratory tract. ACT is a pore-forming cytolysin belonging to the RTX (Repeats in ToXin) family of leukotoxins, capable of permeabilizing several cell types and pure lipid vesicles. Besides, the toxin delivers its N-terminal adenylate cyclase domain into the target cytosol, where catalyzes the conversion of ATP into cAMP, which affects cell signalling. In this study we have made two major observations. First, we show that ACT binds free cholesterol, and identify in its sequence 38 potential cholesterol-recognition motifs. Second, we reveal that four of those motifs are real, functional cholesterol-binding sites. Mutations of the central phenylalanine residues in said motifs have an important impact on the ACT lytic and translocation activities, suggesting their direct intervention in cholesterol recognition and toxin functionality. From our data a likely transmembrane topology can be inferred for the ACT helices constituting the translocation and the hydrophobic regions. From this topology a simple and plausible mechanism emerges by which ACT could translocate its AC domain into target cells, challenging previous views in the field. Blocking the ACT-cholesterol interactions might thus be an effective approach for inhibiting ACT toxicity on cells, and this could help in mitigating the severity of pertussis disease in humans.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.22.469296', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Long-term decreased cannabinoid type-1 receptor activity restores specific neurological phenotypes in the Ts65Dn mouse model of Down syndrome', 'authors': \"[{'name': 'Anna Vzquez-Oliver'}, {'name': 'Silvia Prez-Garca'}, {'name': 'Nieves Pizarro'}, {'name': 'Laura Molina-Porcel'}, {'name': 'Rafael de la Torre'}, {'name': 'Rafael Maldonado'}, {'name': 'Andrs Ozaita'}]\", 'abstract': '<p>Intellectual disability is the most prevalent and limiting hallmark of Down syndrome (DS), without any pharmacological treatment available. Neurodegeneration and neuroinflammation are relevant neurological features of DS reaching to early development of Alzheimers disease. Preclinical evidence suggests that the endocannabinoid system, an important neuromodulator on cognition and neuroinflammation, could act as beneficial target in DS. Indeed, cannabinoid type-1 receptor (CB1R) activity was enhanced in the hippocampus of young-adult trisomic Ts65Dn mice, a well-characterized surrogate model of DS. In previous studies, inhibition of CB1R, was able to restore key neurological deficits in this mouse model. To determine the possible clinical relevance of this target, it is mandatory to evaluate the long-term consequences of attenuated CB1R activity and to minimize the possible side-effects associated to this mechanism. We found that CB1R expression was significantly enhanced in the hippocampus brains of aged DS subjects. Similarly, middle-aged trisomic mice showed enhanced CB1R expression. Long-term oral administration of a low dose of the CB1R specific antagonist rimonabant was administered to male and female Ts65Dn trisomic and wild-type mice from the time of weaning to 10 months, an age when signs of neurodegeneration have been described in the model. CB1R inhibition resulted in significant cognitive improvement in novel object-recognition memory in trisomic male and female mice, reaching a similar performance to that of wild-type littermates. Interestingly, this long-term rimonabant treatment modify locomotor activity, anxiety-like behavior, body weight or survival rates. Brain analysis at 10 months of age revealed noradrenergic and cholinergic neurodegeneration signs in trisomic mice that were not modified by the treatment, although the alterations in hippocampal microglia morphology shown by vehicle-treated trisomic mice was normalized in trisomic mice exposed to rimonabant. Altogether, our results demonstrate a sustained pro-cognitive effect of CB1R inhibition at doses that do not produce major side effects that could be associated to an anti-inflammatory action, suggesting a potential interest in this target of to preserve cognitive functionality in DS.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469156', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Simultaneous assessment of eight phosphorylated STAT residues in T-cells by flow cytometry.', 'authors': \"[{'name': 'Emily Monk'}, {'name': 'Melinda Vassallo'}, {'name': 'Paulo Burke'}, {'name': 'Jeffrey Weber'}, {'name': 'Pratip Chattopadhyay'}, {'name': 'David Woods'}]\", 'abstract': '<p>Signal transducer and activator of transcription (STAT) proteins are a family of transcription factors controlling functions in immune responses and other cell types. Given their importance, we developed a flow cytometry panel to assess eight phosphorylated STAT residues in human T-cells, including six tyrosine residues across six STAT proteins (STAT1, STAT2, STAT3, STAT4, STAT5a, STAT6) and additional serine residues on STAT1 and STAT3. We applied this protocol to test the in vitro induction of pSTATs in response to CD3/CD28 activation and a panel of recombinant cytokines. We also assessed the pSTAT expression profiles of naive CD4+ T-cells polarized to Th1, Th2, Th17 or iTregs.  pSTAT1(S727), pSTAT2(Y689) and pSTAT3(S727) were constitutively expressed in most T-cells, even in the absence of stimulation. For pSTAT1(S727) and pSTAT3(S727), we observed two positive states, high and low. Conversely, expression of pSTAT1(Y701), pSTAT3(Y705), pSTAT4(Y693) and pSTAT6(Y641) were absent in resting T-cells and only expressed with CD3/CD28 activation or with specific cytokines. Variable frequencies of pSTAT5a(Y694) expression were observed in resting T-cells, which increased with activation or specific cytokine stimulation (e.g. IL-2). IFN-beta stimulation enhanced frequencies of expressing cells for all pSTATs. Correlations among several pSTATs, particularly pSTAT1(S727)high and pSTAT3(S727)high were observed. While polarization resulted in increases in canonically associated pSTATs, other non-canonical pSTAT changes were also observed. Collectively, we developed, optimized, and tested a sensitive and rapid approach for simultaneously assessing phosphorylation of six STAT proteins. Using this approach, we made several novel observations of T-cell pSTAT induction in response to stimuli.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.26.470104', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Proteomic Characterization of Serum Small Extracellular Vesicles in Human Breast Cancer', 'authors': \"[{'name': 'Ganfei Xu'}, {'name': 'Weiyi Huang'}, {'name': 'Shaoqian Du'}, {'name': 'Minjing Huang'}, {'name': 'Jiacheng Lyu'}, {'name': 'Fei Zhou'}, {'name': 'Rongxuan Zhu'}, {'name': 'Yuan Cao'}, {'name': 'Jingxuan Xv'}, {'name': 'Ning Li'}, {'name': 'Guoying Yu'}, {'name': 'Binghua Jiang'}, {'name': 'Olivier Gires'}, {'name': 'Lei Zhou'}, {'name': 'Hongwei Zhang'}, {'name': 'Chen Ding'}, {'name': 'Hongxia Wang'}]\", 'abstract': '<p>There is a lack of comprehensive understanding of breast cancer (BC) specific sEVs characteristics and composition on BC unique proteomic information from human samples. Here, we interrogated the proteomic landscape of sEVs in 167 serum samples from patients with BC, benign mammary disease (BD) and from healthy donors (HD). The analysis provides a comprehensive landscape of serum sEVs with totally 9,589 proteins identified, considerably expanding the panel of sEVs markers. Of note, serum BC-sEVs protein signatures were distinct from those of BD and HD, representing stage- and molecular subtype-specific patterns. We constructed specific sEVs protein identifiers that could serve as a liquid biopsy tool for diagnosis and classification of BC from benign mammary disease, molecular subtypes, as well as assessment of lymph node metastasis. We also identified 11 potential survival biomarkers for distant metastasis. This work may provide reference value for the accurate diagnosis and monitoring of BC progression using serum sEVs.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.24.469676', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Single-cell Transcriptomic Analysis Reveals the Cellular Heterogeneity of Mesenchymal Stem Cells', 'authors': \"[{'name': 'Chen Zhang'}, {'name': 'Xueshuai Han'}, {'name': 'Jingkun Liu'}, {'name': 'Lei Chen'}, {'name': 'Ying Lei'}, {'name': 'Kunying Chen'}, {'name': 'Jia Si'}, {'name': 'Tian-yi Wang'}, {'name': 'Hui Zhou'}, {'name': 'Xiaoyun Zhao'}, {'name': 'Xiaohui Zhang'}, {'name': 'Yihua An'}, {'name': 'Yueying Li'}, {'name': 'Qian-fei Wang'}]\", 'abstract': '<p><italic>Ex vivo</italic>-expanded mesenchymal stem cells (MSCs) have been demonstrated to be a heterogeneous mixture of cells exhibiting varying proliferative, multipotential, and immunomodulatory capacities. However, the exact characteristics of MSCs remain largely unknown. By single-cell RNA sequencing of 61,296 MSCs derived from bone marrow and Whartons jelly, we revealed five distinct subpopulations. The developmental trajectory of these five MSC subpopulations were mapped, revealing a differentiation path from stem-like active proliferative cells (APCs) to multipotent progenitor cells, followed by the branching into two paths  adipogenesis or osteochondrogenesis  and subsequent differentiation into unipotent prechondrocytes. The stem-like APCs, expressing the perivascular mesodermal progenitor markers <italic>CSPG4/MCAM/NES</italic>, uniquely exhibited strong proliferation and stemness signatures. Remarkably, the prechondrocyte subpopulation specifically expressed immunomodulatory genes and was able to suppress activated CD3<sup>+</sup> T cell proliferation <italic>in vitro</italic>, supporting the role of this population in immunoregulation. In summary, our analysis mapped the heterogeneous subpopulations of MSCs and identified two subpopulations with potential functions in self-renewal and immunoregulation. Our findings advance the definition of MSCs by identifying the specific functions of its heterogeneous cellular composition, allowing for more specific and effective MSC application through the purification of its functional subpopulations.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.26.470102', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Time course of EEG power during creative problem-solving with insight or remote thinking', 'authors': \"[{'name': 'Theophile Bieth'}, {'name': 'Marcela Ovando-Tellez'}, {'name': 'Alizee Lopez-Persem'}, {'name': 'Beatrice Garcin'}, {'name': 'Laurent Hugueville'}, {'name': 'Katia Lehongre'}, {'name': 'Richard Levy'}, {'name': 'Nathalie George'}, {'name': 'Emmanuelle Volle'}]\", 'abstract': '<p>Problem-solving often requires creativity and is critical in everyday life. However, the neurocognitive mechanisms underlying creative problem-solving remain poorly understood. Two mechanisms have been highlighted: forming new connections from and between the problem elements and insight solving (with a sudden realization of a solution). We examined EEG activity during an adapted version of a classical insight problem task, the Remote Associates Test, that requires finding a word connecting three words. It allowed us to explore remoteness in semantic connections (by varying the remoteness of the solution word across trials) and insight solving (identified as a \"Eureka\" moment reported by the participants). Semantic remoteness was associated with a power increase in alpha band (8-12Hz) in a left parieto-temporal cluster, beta band (13-30Hz) in a right fronto-temporal cluster in the early phase of the task, and theta band (3-7Hz) in frontal cluster before the participants responded. Insight solving was associated with power increase preceding the response in alpha and gamma band (31-60Hz) in left temporal clusters and theta band in a frontal cluster. Source reconstructions show the brain regions associated with these clusters. Overall, our findings shed new light on the dynamic of some of the mechanisms involved in creative problem-solving.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.23.469751', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Seed bio-priming with phosphate solubilizing bacteria strains to improve rice (Oryza sativa L. var. FARO 44) growth under ferruginous ultisol conditions', 'authors': \"[{'name': 'Musa Saheed Ibrahim'}, {'name': 'Beckley Ikhajiagbe'}]\", 'abstract': '<p>The research investigated the possibility of phosphate solubilizing bacteria (PSB) with plant growth-promoting (PGP) capabilities to improve growth properties of rice plant under ferruginous ultisol (FU) condition through bio-priming strategy. The PSB with PGP properties used in this research were <italic>Bacillus cereus</italic> strain GGBSU-1, <italic>Proteus mirabilis</italic> strain TL14-1 and <italic>Klebsiella variicola</italic> strain AUH-KAM-9 that were previously isolated and characterized following the 16S rRNA gene sequencing. Biosafety analysis of the PSB isolates was conducted using blood agar. The rice seeds were then bio-primed with the PSBs at 3, 12 and 24 hours priming durations and then sown in a composite FU soil sample. Differences in germination bioassay involving SEM, morphology, physiology and biomass parameters were investigated for 15 weeks after bio-priming. The composite FU soil used in the study had high pH, low bioavailable phosphorus, low water holding capacity and high iron levels which has led to a low growth properties of rice seeds without bio-priming in FU soil. Germination parameters was better in seeds bio-primed with the PSBs, especially at 12h priming duration as against seeds without priming. SEM showed more bacterial colonization in the PSB bio-primed seeds. Seed bio-priming of rice seed with <italic>Bacillus cereus</italic> strain GGBSU-1, <italic>Proteus mirabilis</italic> strain TL14-1 and <italic>Klebsiella variicola</italic> strain AUH-KAM-9 under FU soil condition significantly improved seed microbiome, rhizocolonization and soil nutrient properties, thereby enhancing growth properties of the rice plant. This suggest the ability of PSB to solubilize and mineralize soil phosphate and improve its availability and soil property for optimum plant usage in phosphate stressed and iron toxic soils.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469757', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'TFAP2 paralogs pioneer chromatin access for MITF and directly inhibit genes associated with cell migration', 'authors': \"[{'name': 'Colin Kenny'}, {'name': 'Ramile Dilshat'}, {'name': 'Hannah Seberg'}, {'name': 'Eric Van Otterloo'}, {'name': 'Gregory Bonde'}, {'name': 'Annika Helverson'}, {'name': 'Eirkur Steingrmsson'}, {'name': 'Robert A. Cornell'}]\", 'abstract': '<p>Transcription factors in the Activating-enhancer-binding Protein 2 (TFAP2) family redundantly regulate gene expression in melanocytes and melanoma. Previous ChIP-seq experiments indicate that TFAP2A and Microphthalmia-associated Transcription Factor (MITF), a master regulator in these cell types, co-activate enhancers of genes promoting pigmentation. Evidence that TFAP2 paralogs can serve as pioneer factors supports the possibility that TFAP2 facilitates MITF binding at co-bound enhancers, although this model has not been tested. In addition, while MITF and TFAP2 paralogs both appear to repress genes that promote invasion, whether they do so by co-repressing enhancers is unknown. To address these questions we evaluated gene expression, chromatin accessibility, TFAP2A and MITF binding, and chromatin marks characteristic of active enhancers in SK-MEL-28 melanoma cells that were wild-type or deleted of the two <italic>TFAP2</italic> paralogs with highest expression, <italic>TFAP2A</italic> and <italic>TFAP2C</italic> (i.e., <italic>TFAP2</italic>-KO cells). Integrated analyses revealed distinct subsets of enhancers bound by TFAP2A in WT cells that are inactivated and activated, respectively, in <italic>TFAP2</italic>-KO cells. At enhancers bound by both MITF and TFAP2A, MITF is generally lost in <italic>TFAP2A/TFAP2C</italic> double mutants, but not vice versa, implying TFAP2 pioneers chromatin access for MITF. There is a strong correlation between the sets of genes inhibited by MITF and TFAP2, although we did not find evidence that TFAP2 and MITF inhibit enhancers cooperatively. The findings imply that MITF and TFAP2 paralogs cooperatively affect the melanoma phenotype.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.24.469776', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Engineered ACE2-Fc counters murine lethal SARS-CoV-2 infection through direct neutralization and Fc-effector activities', 'authors': \"[{'name': 'Yaozong Chen'}, {'name': 'Lulu Sun'}, {'name': 'Irfan Ullah'}, {'name': 'Guillaume Beaudoin-Bussires'}, {'name': 'Sai Priya Anand'}, {'name': 'Andrew P. Hederman'}, {'name': 'William D. Tolbert'}, {'name': 'Rebekah Sherburn'}, {'name': 'Dung N. Nguyen'}, {'name': 'Lorie Marchitto'}, {'name': 'Shilei Ding'}, {'name': 'Di Wu'}, {'name': 'Yuhong Luo'}, {'name': 'Suneetha Gottumukkala'}, {'name': 'Sean Moran'}, {'name': 'Priti Kumar'}, {'name': 'Grzegorz Piszczek'}, {'name': 'Walther Mothes'}, {'name': 'Margaret E. Ackerman'}, {'name': 'Andrs Finzi'}, {'name': 'Pradeep D. Uchil'}, {'name': 'Frank J. Gonzalez'}, {'name': 'Marzena Pazgier'}]\", 'abstract': '<p>Soluble Angiotensin-Converting Enzyme 2 (ACE2) constitutes an attractive antiviral capable of targeting a wide range of coronaviruses utilizing ACE2 as their receptor. Here, using structure-guided approaches, we developed divalent ACE2 molecules by grafting the extracellular ACE2-domain onto a human IgG1 or IgG3 (ACE2-Fc). These ACE2-Fcs harbor structurally validated mutations that enhance spike (S) binding and remove angiotensin enzymatic activity. The lead variant bound tightly to S, mediated <italic>in vitro</italic> neutralization of SARS-CoV-2 variants of concern (VOCs) with sub-nanomolar IC<sub>50</sub> and was capable of robust Fc-effector functions, including antibody-dependent-cellular cytotoxicity, phagocytosis and complement deposition. When tested in a stringent K18-hACE2 mouse model, it delayed death or effectively resolved lethal SARS-CoV-2 infection in a prophylactic or therapeutic setting utilizing the combined effect of neutralization and Fc-effector functions. These data confirm the utility of ACE2-Fcs as valuable agents in preventing and eliminating SARS-CoV-2 infection and demonstrate that ACE2-Fc therapeutic activity require Fc-effector functions.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469674', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Interplay of adherens junctions and matrix proteolysis determines the invasive pattern and growth of squamous cell carcinoma', 'authors': \"[{'name': 'Takuya Kato'}, {'name': 'Robert P. Jenkins'}, {'name': 'Stefanie Derzsi'}, {'name': 'Melda Tozluoglu'}, {'name': 'Antonio Rullan'}, {'name': 'Steven Hooper'}, {'name': 'Raphael A. G. Chaleil'}, {'name': 'Xiao Fu'}, {'name': 'Holly Joyce'}, {'name': 'Selvam Thavaraj'}, {'name': 'Paul A. Bates'}, {'name': 'Erik Sahai'}]\", 'abstract': '<p>Cancers, such as squamous cell carcinoma, frequently invade as multicellular units. However, these invading units can be organized in a variety of ways, ranging from thin discontinuous strands to thick pushing collectives. Here we employ an integrated experimental and computational approach to identify the factors that determine the mode of collective cancer cell invasion. We find that matrix proteolysis is linked to the formation of wide strands, but has little effect on the maximum extent of invasion. Cell-cell junctions also favour wide strands, but our analysis also reveals a requirement for cell-cell junctions for efficient invasion in response to uniform directional cues. Unexpectedly, the ability to generate wide invasive strands is coupled to the ability to grow effectively when surrounded by ECM in 3D assays. Combinatorial perturbation of both matrix proteolysis and cell-cell adhesion demonstrates that the most aggressive cancer behaviour, both in terms of invasion and growth, is achieved at high levels of cell-cell adhesion and high levels of proteolysis. Contrary to expectation, cells with canonical mesenchymal traits  no cell-cell junctions and high proteolysis  exhibit reduced growth and lymph node metastasis. Thus, we conclude that the ability of squamous cell carcinoma cells to invade effectively is also linked to their ability to generate space for proliferation in confined contexts. These data provide an explanation for the apparent advantage of retaining cell-cell junctions in SCC.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469532', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Evaluation of new polymer-iodine complexes for the fabrication of medical devices', 'authors': \"[{'name': 'Marina Lpez-lvarez'}, {'name': 'Herb Ulmer'}, {'name': 'Nico Klay'}, {'name': 'Jan Maarten van Dijl'}]\", 'abstract': '<p>Povidone-iodine has been a true success story in the fight against infections by harnessing the antimicrobial and antiviral properties of elemental iodine for water-based applications. However, to date there has been little success in implementing iodine attributes in water-insoluble engineering plastics. Here, we describe the first development of biocidal active polyamide- and polyurethane-iodine complexes at laboratory and commercially relevant scales. These polymer-iodine materials are active against a broad range of microorganisms, including bacteria, yeast and fungi, and can be used as base materials for medical devices. The use of new polymer-iodine complexes for infection prevention in medical devices, such as sutures, catheters and drains, or wound care is expected to have significant positive effects at reducing healthcare-acquired infections. In addition, the materials are expected to find significant applications in other fields, such as air handling with the production of biocidal face masks and air filters to control spread of pathogens.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469864', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'An exploratory study of EEG connectivity during the first year of life in preterm and full-term infants', 'authors': \"[{'name': 'Eduardo Gonzalez-Moreira'}, {'name': 'Deirel Paz-Linares'}, {'name': 'Lourdes Cubero-Rego'}, {'name': 'Ariosky Areces-Gonzalez'}, {'name': 'Pedro Antonio Valdes-Sosa'}, {'name': 'Thalia Harmony'}]\", 'abstract': '<p>Aim: to evaluate EEG connectivity during the first year of age in healthy full-term infants and preterm infants with prenatal and perinatal risk factors for perinatal brain damage.\\nMethods: Three groups of infants were studied: healthy at full-term infants (n = 71), moderate and late preterm infants (n = 54), and very preterm infants (n = 56). All preterm infants had perinatal or/and perinatal risk factors for brain damage.  EEG was obtained during phase II of natural NREM sleep. EEG analysis was performed in 24 segments of 2.56 s free of artifacts. For the calculation of EEG sources, the spectral Structured Sparse Bayesian Learning (sSSBL) was used. Connectivity was computed by the phase-lag index.\\nResults: In healthy full-term infants, EEG interhemispheric connectivity in the different frequency bands followed similar trends with age to those reported in each frequency band: delta connectivity decreases, theta increases at the end of the year, in the alpha band, different trends were observed according to the region studied, and beta interhemispheric connectivity decreases with age. EEG connectivity in preterm infants showed differences from the results of the term group.\\nDiscussion: Important structural findings may explain the differences observed in EEG connectivity between the term and preterm groups.\\nConclusion: The study of EEG connectivity during the first year of age gives essential information on normal and abnormal brain development.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.24.469825', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Global non-random abundance of short tandem repeats in rodents and primates', 'authors': \"[{'name': 'Masoud Arabfard'}, {'name': 'Mahmood Salesi'}, {'name': 'Yazdan Hassani Nourian'}, {'name': 'Iman Arabipour'}, {'name': 'Ali Mohammad Ali Maddi'}, {'name': 'Kaveh Kavousi'}, {'name': 'Mina Ohadi'}]\", 'abstract': '<sec><title>Background</title><p>While of predominant abundance across vertebrate genomes and significant biological implications, the relevance of short tandem repeat (STR) abundance to speciation remains largely elusive and attributed to random coincidence for the most part. In a model study, here we collected whole-genome abundance of mono-, di-, and trinucleotide STRs in nine species, encompassing rodents and primates, including rat, mouse, olive baboon, gelada, macaque, gorilla, chimpanzee, bonobo, and human. The obtained unnormalized and normalized data were used to analyze hierarchical clustering of the STR abundances in the selected species.</p></sec><sec><title>Results</title><p>We found massive differential abundances between the rodent and primate orders. In addition, while numerous STRs had random abundance across the nine selected species, the global abundance conformed to three consistent &lt;clusters&gt;, as follows: &lt;rat, mouse&gt;, &lt;gelada, macaque, olive baboon&gt;, &lt;gorilla, chimpanzee, bonobo, human&gt;, which coincided with the phylogenetic distances of the selected species (p&lt; 4E-05). Exceptionally, in the trinucleotide STR compartment, human was significantly distant from all other species.</p></sec><sec><title>Conclusion</title><p>We propose that the global abundance of STRs is non-random in rodents and primates, and probably had a determining impact on the speciation of the two orders. We also propose the STRs and STR lengths which specifically coincided with the phylogeny of the selected species.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.25.21266090', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': True, 'title': 'Synthesis of High-Resolution Research-Quality MRI Data from Clinical MRI Data in Patients with COVID-19', 'authors': \"[{'name': 'Ryan J. Cali'}, {'name': 'Holly J. Freeman'}, {'name': 'Benjamin Billot'}, {'name': 'Megan E. Barra'}, {'name': 'David Fischer'}, {'name': 'William R. Sanders'}, {'name': 'Susie Y. Huang'}, {'name': 'John Conklin'}, {'name': 'Bruce Fischl'}, {'name': 'Juan Eugenio Iglesias'}, {'name': 'Brian L. Edlow'}]\", 'abstract': '<p>Pathophysiological mechanisms of neurological disorders in patients with coronavirus disease 2019 (COVID-19) are poorly understood, partly because of a lack of high-resolution neuroimaging data. We applied SynthSR, a convolutional neural network that synthesizes high-resolution isotropic research-quality data from thick-slice clinical MRI data, to a cohort of 11 patients with severe COVID-19. SynthSR successfully synthesized T1-weighted MPRAGE data at 1 mm spatial resolution for all 11 patients, each of whom had at least one brain lesion. Correlations between volumetric measures derived from synthesized and acquired MPRAGE data were strong for the cortical grey matter, subcortical grey matter, brainstem, hippocampus, and hemispheric white matter (<italic>r</italic>=0.84 to 0.96, p0.001), but absent for the cerebellar white matter and corpus callosum (<italic>r</italic>=0.04 to 0.17, p&gt;0.61). SynthSR creates an opportunity to quantitatively study clinical MRI scans and elucidate the pathophysiology of neurological disorders in patients with COVID-19, including those with focal lesions.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.23.469799', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Prevalence and correlates of phenazine resistance in culturable bacteria from a dryland wheat field', 'authors': \"[{'name': 'Elena K. Perry'}, {'name': 'Dianne K. Newman'}]\", 'abstract': '<p>Phenazines are a class of bacterially-produced redox-active natural antibiotics that have demonstrated potential as a sustainable alternative to traditional pesticides for the biocontrol of fungal crop diseases. However, the prevalence of bacterial resistance to agriculturally-relevant phenazines is poorly understood, limiting both the understanding of how these molecules might shape rhizosphere bacterial communities and the ability to perform risk assessment for off-target effects. Here, we describe profiles of susceptibility to the antifungal agent phenazine-1-carboxylic acid (PCA) across more than 100 bacterial strains isolated from a wheat field where PCA producers are indigenous and abundant. We find that Gram-positive bacteria are typically more sensitive to PCA than Gram-negative bacteria, but that there is also significant variability in susceptibility both within and across phyla. Phenazine-resistant strains are more likely to be isolated from the wheat rhizosphere, where PCA producers are also more abundant, compared to bulk soil. Furthermore, PCA toxicity is pH-dependent for most susceptible strains and broadly correlates with PCA reduction rates, suggesting that uptake and redox-cycling are important determinants of phenazine toxicity. Our results shed light on which classes of bacteria are most likely to be susceptible to phenazine toxicity in acidic or neutral soils. In addition, the taxonomic and phenotypic diversity of our strain collection represents a valuable resource for future studies on the role of natural antibiotics in shaping wheat rhizosphere communities.</p><sec><title>Importance</title><p>Microbial communities contribute to crop health in important ways. For example, phenazine metabolites are a class of redox-active molecules made by diverse soil bacteria that underpin the biocontrol of wheat and other crops. Their physiological functions are nuanced: in some contexts they are toxic, in others, beneficial. While much is known about phenazine production and the effect of phenazines on producing strains, our ability to predict how phenazines might shape the composition of environmental microbial communities is poorly constrained; that phenazine prevalence in the rhizosphere is predicted to increase in arid soils as the climate changes provides an impetus for further study. As a step towards gaining a predictive understanding of phenazine-linked microbial ecology, we document the effects of phenazines on diverse bacteria that were co-isolated from a wheat rhizosphere and identify conditions and phenotypes that correlate with how a strain will respond to phenazines.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.25.469965', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Molecular architecture of nucleosome remodeling and deacetylase sub-complexes by integrative structure determination', 'authors': \"[{'name': 'Shreyas Arvindekar'}, {'name': 'Matthew J. Jackman'}, {'name': 'Jason K.K. Low'}, {'name': 'Michael J. Landsberg'}, {'name': 'Joel P. Mackay'}, {'name': 'Shruthi Viswanath'}]\", 'abstract': '<p>The Nucleosome Remodeling and Deacetylase (NuRD) complex is a chromatin-modifying assembly that regulates gene expression and DNA damage repair. Despite its importance, limited structural information is available on the complex and a detailed understanding of its mechanism is lacking. We investigated the molecular architecture of three NuRD sub-complexes: MTA1-HDAC1-RBBP4 (MHR), MTA1<sup>N</sup>-HDAC1-MBD3<sup>GATAD2CC</sup> (MHM), and MTA1-HDAC1-RBBP4-MBD3-GATAD2 (NuDe) using Bayesian integrative structure determination with IMP (Integrative Modeling Platform), drawing on information from SEC-MALLS, DIA-MS, XLMS, negative stain EM, X-ray crystallography, NMR spectroscopy, secondary structure and homology predictions. The structures were corroborated by independent cryo-EM maps, biochemical assays, and known cancer-associated mutations. Our integrative structure of the 2:2:2 MHM complex shows asymmetric binding of MBD3, whereas our structure of the NuDe complex shows MBD3 localized precisely to a single position distant from the MTA1 dimerization interface. Our models suggest a possible mechanism by which asymmetry is introduced in NuRD, and indicate three previously unrecognized subunit interfaces in NuDe: HDAC1<sup>C</sup>-MTA1<sup>BAH</sup>, MTA1<sup>BAH</sup>-MBD3, and HDAC1<sup>60-100</sup>-MBD3. We observed that a significant number of cancer-associated mutations mapped to protein-protein interfaces in NuDe. Our approach also allows us to localize regions of unknown structure, such as HDAC1<sup>C</sup> and MBD3<sup>IDR</sup>, thereby resulting in the most complete structural characterization of these NuRD sub-complexes so far.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.25.469972', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A LRRK2/dLRRK-mediated lysosomal pathway that contributes to glial cell death and DA neuron survival', 'authors': \"[{'name': 'Linfang Wang'}, {'name': 'Honglei Wang'}, {'name': 'Margaret S. Ho'}]\", 'abstract': '<p>Mutations in <italic>leucine-rich repeat kinase 2</italic> (<italic>LRRK2</italic>) are the most common cause of familial and sporadic Parkinsons disease (PD). A plethora of evidence has indicated a role for LRRK2 in endolysosomal trafficking in neurons, while LRRK2 function in glia, although highly expressed, remains largely unknown. Here we present evidence that LRRK2/dLRRK mediates a glial lysosomal pathway that contributes to the mechanism of PD. Independent of its kinase activity, glial LRRK2/dLRRK knockdown in the immortalized microglial cells or flies results in enlarged and swelling lysosomes fewer in number. These lysosomes are less mobile, wrongly acidified, and exhibit defective membrane permeability and reduced activity of the lysosome hydrolase cathespin B. In addition, microglial LRRK2 depletion causes increased <italic>Caspase</italic> 3 levels, leading to glial apoptosis, dopaminergic neurodegeneration, and locomotor deficits in an age-dependent manner. Taken together, these findings demonstrate a functional role of LRRK2/dLRRK in regulating the glial lysosomal pathway; deficits in lysosomal biogenesis and function linking to glial apoptosis potentially underlie the mechanism of DA neurodegeneration, contributing to the progression of PD.</p>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.25.469692', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'The DOMINO web-server for active module identification analysis', 'authors': \"[{'name': 'Hagai Levi'}, {'name': 'Nima Rahmanian'}, {'name': 'Ran Elkon'}, {'name': 'Ron Shamir'}]\", 'abstract': '<p>Active module identification (AMI) is an essential step in many omics analyses. Such algorithms receive a gene network and a gene activity profile as input and report subnetworks that show significant over-representation of accrued activity signal (active modules). Such modules can point out key molecular processes in the analyzed biological conditions.</p><sec><title>Results</title><p>We recently introduced a novel AMI algorithm called DOMINO, and demonstrated that it detects active modules that capture biological signals with markedly improved rate of empirical validation. Here, we provide an online server that executes DOMINO, making it more accessible and user-friendly. To help the interpretation of solutions, the server provides GO enrichment analysis, module visualizations, and accessible output formats for customized downstream analysis. It also enables running DOMINO with various gene identifiers of different organisms.</p></sec><sec><title>Availability and implementation</title><p>The server is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://domino.cs.tau.ac.il\">http://domino.cs.tau.ac.il</ext-link>. Its codebase is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/Shamir-Lab\">https://github.com/Shamir-Lab</ext-link>.</p></sec>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.24.21266320', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Cardiovascular Medicine', 'covid': False, 'title': 'Measure by Measure: Resting Heart Rate Across the 24 Hour Cycle.', 'authors': \"[{'name': 'Cathy A Speed'}, {'name': 'Tom Arneil'}, {'name': 'Robert K Harle'}, {'name': 'Alex Wilson'}, {'name': 'Alan Karthikesalingam'}, {'name': 'Michael V McConnell'}, {'name': 'Justin P Phillips'}]\", 'abstract': '<p><bold>Background:</bold> Photoplethysmography (PPG) sensors, typically found in wrist-worn devices, can continuously monitor heart rate (HR) in large populations in real-world settings. Resting heart rate (RHR) is an important biomarker of morbidities and mortality, but no universally accepted definition nor measurement criteria exist. In this study, we provide a working definition of RHR and describe a method for measurement of RHR recorded using PPG derived from wristband measurement across a 24-hour cycle.\\n<bold>Methods:</bold> 433 healthy subjects wore a wrist device that measured activity and HR for up to 3 months. HR during inactivity was recorded and the duration of inactivity needed for HR to stabilise was ascertained. We identified the lowest HR during each 24-hour cycle (true RHR) and examined the time of day or night this occurred. The variation of HR during inactivity through the 24-hour cycle was also assessed. The sample was also subdivided according to daily activity levels for subset analysis. \\n<bold>Findings:</bold> Adequate data was obtained for 19,242 days and 18,520 nights. HR stabilised in most subjects after 4 minutes of inactivity. Mean (SD) RHR for the sample was 54.5 (8.0) bpm (day) and 50.5 (7.6) bpm (night). RHR values were highest in the least active group (lowest MET quartile) . A circadian variation of HR during inactivity was confirmed, with the lowest values being between 0300 and 0700 hours for most subjects.\\n<bold>Interpretation:</bold> RHR measured using a PPG-based wrist-worn device is significantly lower at night than in the day, and a circadian rhythm of HR during inactivity was confirmed.  Since RHR is such an important health metric, clarity on the definition and measurement methodology used is important. A minimum rest time of 4 minutes provides a reliable measurement of HR during inactivity and true RHR in a 24-hour cycle is best measured between 0300 and 0700 hours. \\n<bold>Funding:</bold> This study was funded by Google.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469550', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Human visual gamma for color stimuli: When LGN drive is equalized, red is not special', 'authors': \"[{'name': 'Benjamin Johannes Stauch'}, {'name': 'Alina Peter'}, {'name': 'Isabelle Ehrlich'}, {'name': 'Zora Nolte'}, {'name': 'Pascal Fries'}]\", 'abstract': '<p>Strong gamma-band oscillations in primate early visual cortex can be induced by spatially homogeneous, high-contrast stimuli, such as color surfaces. Compared to other hues, particularly strong gamma oscillations have been reported for red stimuli. However, precortical color processing and the resultant strength of input to V1 has often not been fully controlled for. This leaves the possibility that stronger responses to some hues were due to differences in V1 input strength. We presented stimuli that had equal luminance and color contrast levels in a color coordinate system based on color responses of the lateral geniculate nucleus, the main input source for area V1. With these stimuli, we recorded magnetoencephalography in 30 human subjects. We found narrowband color-induced gamma oscillations in early visual cortex, which, contrary to previous reports, did not differ between red and green stimuli of equal L-M cone contrast. Notably, blue stimuli with contrast exclusively on the S-cone axis induced very weak gamma responses, as well as smaller event-related fields and poorer change detection performance. The strength of human color gamma responses could be well explained by the strength of thalamic input induced by each hue and does not show a clear red bias when this input strength is properly equalized.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.26.470168', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'More than meets the Kappa for Antibody Superantigen Protein L (PpL)', 'authors': \"[{'name': 'Wei Li Ling'}, {'name': 'Joshua Yi Yeo'}, {'name': 'Yuen Ling Ng'}, {'name': 'Anil Wipat'}, {'name': 'Samuel KE Gan'}]\", 'abstract': '<p>Immunoglobulin superantigens play an important role in the affinity purification of antibodies and underlie the microbiota-immune axis at mucosal areas Focussing on the Staphylococcal Protein A (SpA), Streptococcal Protein G (SpG), and the Finegoldia Protein L (PpL) that were previously thought to bind to only specific regions of human antibodies, a systematic and holistic analysis of the antibody regions using 63 antibody permutations involving six V and seven VH region IgG1 revealed showed novel PpL-antibody interactions. While SpA and SpG showed relatively consistent interactions with the antibodies, our findings showed PpL binding to certain VH-V2, 5 and 6 interactions had contribution by other antibody regions. The findings of this have implications on PpL-based affinity antibody purifications and antibody design as well as provides novel insights to PpL-based microbiota-immune axis effects.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.23.21266747', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Health Informatics', 'covid': True, 'title': 'The experiences of 33 national COVID-19 dashboard teams during the first year of the pandemic in the WHO European Region: a qualitative study', 'authors': \"[{'name': 'Erica Barbazza'}, {'name': 'Damir Ivankovi'}, {'name': 'Karapet Davtyan'}, {'name': 'Mircha Poldrugovac'}, {'name': 'Zhamin Yelgezekova'}, {'name': 'Claire Willmington'}, {'name': 'Bernardo Meza-Torres'}, {'name': 'Vronique L.L.C. Bos'}, {'name': 'scar Brito Fernandes'}, {'name': 'Alexandru Rotar'}, {'name': 'Sabina Nuti'}, {'name': 'Milena Vainieri'}, {'name': 'Fabrizio Carinci'}, {'name': 'Natasha Azzopardi-Muscat'}, {'name': 'Oliver Groene'}, {'name': 'David Novillo-Ortiz'}, {'name': 'Niek Klazinga'}, {'name': 'Dionne Kringos'}]\", 'abstract': '<sec><title>Background</title><p>Governments across the WHO European Region prioritized dashboards for reporting COVID-19 data. The ubiquitous use of dashboards for public reporting is novel. This study explores the development of COVID-19 dashboards during the pandemics first year and common barriers, enablers and lessons from the experiences of teams responsible for their development.</p></sec><sec><title>Methods</title><p>Multiple methods were applied to identify and recruit COVID-19 dashboard teams using a purposive, quota sampling approach. Semi-structured group interviews were conducted between April June 2021. Using elaborative coding and thematic analysis, descriptive and explanatory themes were derived from interview data. A validation workshop with study participants was held in June 2021.</p></sec><sec><title>Results</title><p>Eighty informants, representing 33 national COVID-19 dashboard teams across the WHO European Region participated. Most dashboards were launched swiftly in the first months of the pandemic, between FebruaryMay 2020. The urgency, intense workload, limited human resources, data and privacy constraints, and public scrutiny were common to the initial development stage. Themes related to barriers or enablers were identified pertaining to the pre-pandemic context, pandemic itself, people and processes, software, data, and users. Lessons emerged around the themes of simplicity, trust, partnership, software and data, and change.</p></sec><sec><title>Conclusions</title><p>COVID-19 dashboards were developed in a learning-by-doing approach. The experiences of teams signal initial under-preparedness was compensated by high-level political endorsement, the professionalism of teams, accelerated data improvements, and immediate support of commercial software solutions. To leverage the full potential of dashboards, investments are needed at team-, national- and pan-European-level.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469529', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Simulation of aerosol dispersion during medical examinations', 'authors': \"[{'name': 'Sebastian Falk'}, {'name': 'Sarina K Mueller'}, {'name': 'Stefan Kniesburges'}, {'name': 'Michael Dllinger'}]\", 'abstract': '<p>The main route of transmission of the SARS-CoV2 virus has been shown to be airborne. The objective of this study is to analyze the aerosol dispersion and potential exposure to medical staff within a typical medical examination room during classical airway procedures. The multiphase simulation of the aerosol particles in the airflow is based on a Lagrangian-Eulerian approach. All simulation cases with surgical mask show partially but significantly reduced maxi-mum dispersion distances of the aerosol particles compared to the cases without surgical mask. The simulations have shown that medical examiner are exposed to large amount of aerosol particles, especially during procedures such as laryngoscopy where the examiners head is directly in front of the the patients face. However, exposure can be drastically reduced if the patient wears a mask which is possible for the most of the procedures studied, such as otoscopy, sonography, or anamnesis.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.26.470074', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Comparative analysis of Mycobacterium abscessus clinical isolate virulence using an invertebrate infection model', 'authors': \"[{'name': 'Yasuhiko Matsumoto'}, {'name': 'Hanako Fukano'}, {'name': 'Naoki Hasegawa'}, {'name': 'Yoshihiko Hoshino'}, {'name': 'Takashi Sugita'}]\", 'abstract': '<p>Aims: <italic>Mycobacterium abscessus</italic> causes chronic skin infections, lung diseases, and systemic or disseminated infections. We investigated the quantitative evaluation of the virulence of <italic>M. abscessus</italic> clinical isolates by calculating the LD50 using a silkworm infection model. Methods and Results: <italic>M. abscessus</italic> subsp. <italic>abscessus</italic> cells were injected into the hemolymph of silkworms. Silkworms died within two days post-infection with <italic>M. abscessus</italic> subsp. <italic>abscessus</italic> when reared at 37C. Viable cell numbers of <italic>M. abscessus</italic> subsp. <italic>abscessus</italic> increased in the hemolymph of the silkworms injected with <italic>M. abscessus</italic> subsp. <italic>abscessus</italic>. Silkworms were not killed following injections with heat-killed <italic>M. abscessus</italic> subsp. <italic>abscessus</italic> cells. The administration of clarithromycin, an antibacterial drug used for the treatment of the infection, prolonged the survival time of silkworms injected with <italic>M. abscessus</italic> subsp. <italic>abscessus</italic>. The LD50 values of seven clinical isolates were determined using the silkworm infection model and differed by up to nine-fold.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.23.469650', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Astrocytic expression of ALS-causative mutant FUS leads to TNF-dependent neurodegeneration in vivo', 'authors': \"[{'name': 'Brigid K. Jensen'}, {'name': 'Kevin J. McAvoy'}, {'name': 'Nicolette M. Heinsinger'}, {'name': 'Angelo C. Lepore'}, {'name': 'Hristelina Ilieva'}, {'name': 'Aaron R. Haeusler'}, {'name': 'Davide Trotti'}, {'name': 'Piera Pasinelli'}]\", 'abstract': '<p>Genetic mutations that cause Amyotrophic Lateral Sclerosis (ALS), a progressively lethal motor neuron disease, are commonly found in ubiquitously expressed genes. In addition to direct defects within motor neurons, growing evidence suggests that dysfunction of non-neuronal cells is also an important driver of disease. Previously, we demonstrated that mutations in DNA/RNA binding protein Fused in Sarcoma (FUS) induce neurotoxic phenotypes in astrocytes <italic>in vitro,</italic> via activation of the NF-B pathway and release of pro-inflammatory cytokine TNF. Here, we developed an intraspinal cord injection model to test whether astrocyte-specific expression of ALS-causative FUS<sup>R521G</sup> variant (mtFUS) causes neuronal damage <italic>in vivo</italic>. We show that mtFUS expression causes TNF upregulation, motor function deficits, and spinal motor neuron loss. We further demonstrate a lack of phenotype in TNF knockout animals expressing mtFUS, and prevention of neurodegeneration in mtFUS-transduced animals through administration of TNF neutralizing antibodies. Together, these studies strengthen evidence that astrocytes contribute to disease in ALS, establish that FUS-ALS astrocytes induce pathogenic changes to motor neurons <italic>in vivo</italic>, and provide insights identifying FUS-ALS specific potential therapeutic targets.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.21264927', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Development and Evaluation of AccuPower COVID-19 Multiplex Real-Time RT-PCR Kit and AccuPower SARS-CoV-2 Multiplex Real-Time RT-PCR Kit for SARS-CoV-2 Detection in Sputum, NPS/OPS, Saliva and Pooled Samples', 'authors': \"[{'name': 'In Bum Suh'}, {'name': 'Jaegyun Lim'}, {'name': 'Hyo Seon Kim'}, {'name': 'Guil Rhim'}, {'name': 'Kim Heebum'}, {'name': 'Kim Hana'}, {'name': 'Lee Sae-Mi'}, {'name': 'Park Hyun-sang'}, {'name': 'Song Hyun Ju'}, {'name': 'Hong MyungKook'}, {'name': 'Shin Gyung Sook'}, {'name': 'Moon Jung Kim'}]\", 'abstract': '<p>Rapid and accurate detection of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is essential for the successful control of the current global COVID-19 pandemic. The real-time reverse transcription polymerase chain reaction (Real-time RT-PCR) is the most widely used detection technique. This research describes the development of two novel multiplex real-time RT-PCR kits, <italic>AccuPower</italic><sup></sup> COVID-19 Multiplex Real-Time RT-PCR Kit (NCVM) specifically designed for use with the <italic>ExiStation</italic>48 system (comprised of <italic>ExiPrep</italic>48 Dx and <italic>Exicycler</italic>96 by BIONEER, Korea) for sample RNA extraction and PCR detection, and <italic>AccuPower</italic><sup></sup> SARS-CoV-2 Multiplex Real-Time RT-PCR Kit (SCVM) designed to be compatible with manufacturers on-market PCR instruments. The limit of detection (LoD) of NCVM was 120 copies/L and the LoD of the SCVM was 2 copies/mL for both the gene and the SARS-CoV-2 gene (N gene and RdRp gene). The <italic>AccuPower</italic><sup></sup> kits demonstrated high precision with no cross reactivity to other respiratory-related microorganisms. The clinical performance of <italic>AccuPower</italic><sup></sup> kits was evaluated using the following clinical samples: sputum and nasopharyngeal/oropharyngeal swab (NPS/OPS) samples. Overall agreement of the <italic>AccuPower</italic><sup></sup> kits with a Food and Drug Administration (FDA) approved emergency use authorized commercial kit (STANDARD M nCoV Real-Time Detection kit, SD BIOSENSOR, Korea) was above 95% (Cohens kappa coefficient  0.95), with a sensitivity of over 95%. The NPS/OPS specimen pooling experiment was conducted to verify the usability of <italic>AccuPower</italic><sup></sup> kits on pooled samples and the results showed greater than 90% agreement with individual NPS/OPS samples. The clinical performance of <italic>AccuPower</italic><sup></sup> kits with saliva samples was also compared with NPS/OPS samples and demonstrated over 95% agreement (Cohens kappa coefficient &gt; 0.95). This study shows the BIONEER NCVM and SCVM assays are comparable with the current standard confirmation assay and are suitable for effective clinical management and control of SARS-CoV-2.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.21266655', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Determining international spread of novel B.1.1.523 SARS-CoV-2 lineage', 'authors': \"[{'name': 'Lukas Zemaitis'}, {'name': 'Gediminas Alzbutas'}, {'name': 'Dovydas Gecys'}, {'name': 'Andrey Komissarov'}, {'name': 'Arnoldas Pautienius'}, {'name': 'Rasa Ugenskiene'}, {'name': 'Marius Sukys'}, {'name': 'Vaiva Lesauskaite'}]\", 'abstract': '<p>Here we report the emergence of variant lineage B.1.1.523 that contains a set of mutations including 156_158del, E484K and S494P in Spike protein. E484K and S494P are known to significantly reduce SARS-CoV-2 neutralization by convalescent and vaccinee sera and are considered as mutations of concern. Lineage B.1.1.523 has presumably originated in Russian Federation and spread across European countries with the peak of transmission in April  May 2021. The B.1.1.523 lineage has now been reported from 27 countries.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.28.470264', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Joint Learning of Full-structure Noise in Hierarchical Bayesian Regression Models', 'authors': \"[{'name': 'Ali Hashemi'}, {'name': 'Chang Cai'}, {'name': 'Yijing Gao'}, {'name': 'Sanjay Ghosh'}, {'name': 'Klaus-Robert Mller'}, {'name': 'Srikantan Nagarajan'}, {'name': 'Stefan Haufe'}]\", 'abstract': '<p>We consider the reconstruction of brain activity from electroencephalography (EEG). This inverse problem can be formulated as a linear regression with independent Gaussian scale mixture priors for both the source and noise components. Crucial factors influencing accuracy of source estimation are not only the noise level but also its correlation structure, but existing approaches have not addressed estimation of noise covariance matrices with full structure. To address this shortcoming, we develop hierarchical Bayesian (type-II maximum likelihood) models for observations with latent variables for source and noise, which are estimated jointly from data. As an extension to classical sparse Bayesian learning (SBL), where across-sensor observations are assumed to be independent and identically distributed, we consider Gaussian noise with full covariance structure. Using the majorization-maximization framework and Riemannian geometry, we derive an efficient algorithm for updating the noise covariance along the manifold of positive definite matrices. We demonstrate that our algorithm has guaranteed and fast convergence and validate it in simulations and with real MEG data. Our results demonstrate that the novel framework significantly improves upon state-of-the-art techniques in the real-world scenario where the noise is indeed non-diagonal and fully-structured. Our method has applications in many domains beyond biomagnetic inverse problems.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.22.469497', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Fine-grained, Nonlinear Image Registration of Live Cell Movies Reveals Spatiotemporal Organization of Diffuse Molecular Processes', 'authors': \"[{'name': 'Xuexia Jiang'}, {'name': 'Tadamoto Isogai'}, {'name': 'Joseph Chi'}, {'name': 'Gaudenz Danuser'}]\", 'abstract': '<p>We present an application of non-linear Image registration that allows spatiotemporal analysis of extremely noisy and diffuse molecular processes across the entire cell. To produce meaningful local tracking of the spatially coherent portion of diffuse protein dynamics, we improved upon existing non-linear image registration to compensate for cell movement and deformation. The registration relies on a subcellular fiducial marker, a cell motion mask, and a topological regularization that enforces diffeomorphism on the registration without significant loss of granularity. We demonstrate the potential of this approach in conjunction with stochastic time-series analysis through the discovery of distinct zones of coherent Profillin dynamics in symmetry-breaking U2OS cells. Further analysis of the resulting Profilin dynamics revealed strong relationships with the underlying actin organization. This study thus provides a framework for extracting functional interactions between cell morphodynamics, protein distributions, and signaling in cells undergoing continuous shape changes.</p><sec><title>Author Summary</title><p>By adapting optical flow based nonlinear image registration we created a method specific for live cell movies that preserves the dynamics of a signal of interest by remapping using a separate measurable subcellular location fiducial. This is an extension as well on our labs previously published method of cell edge-based referencing of subcellular locations that was incapable of extracting interpretable subcellular time series more than a few microns away from the cell edge. We showed that our method overcomes this key limitation and allows sampling of subcellular time series from every subcellular location through our discovery of organized profilin dynamics in moving cell and that these profilin dynamics are related to actin dynamics due to their ability to bind growing actin structures likely through actin polymerizing factors. Most importantly, our method is applicable to discovering subcellular organization and coordination in a widely used form of live cell microscopy data that hitherto has been largely limited to anecdotal analysis.</p></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.28.470260', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Oncogenic RAS commandeers amino acid sensing machinery to aberrantly activate mTORC1 in multiple myeloma', 'authors': \"[{'name': 'Yandan Yang'}, {'name': 'Thomas Oellerich'}, {'name': 'Ping Chen'}, {'name': 'Arnold Bolomsky'}, {'name': 'Michele Ceribelli'}, {'name': 'Bjorn Haupl'}, {'name': 'George W. Wright'}, {'name': 'James D. Phelan'}, {'name': 'Da Wei Huang'}, {'name': 'James W. Lord'}, {'name': 'Callie K. Van Winkle'}, {'name': 'Xin Yu'}, {'name': 'Jan Wisnieski'}, {'name': 'James Q. Wang'}, {'name': 'Frances A. Tosto'}, {'name': 'Erin Beck'}, {'name': 'Kelli Wilson'}, {'name': 'Crystal McKnight'}, {'name': 'Jameson Travers'}, {'name': 'Carleen Klumpp-Thomas'}, {'name': 'Grace A. Smith'}, {'name': 'Stefania Pittaluga'}, {'name': 'Irina Maric'}, {'name': 'Dickran Kazandjian'}, {'name': 'Craig J. Thomas'}, {'name': 'Ryan M. Young'}]\", 'abstract': '<p>Oncogenic mutations within the RAS pathway are common in multiple myeloma (MM), an incurable malignancy of plasma cells.  However, the mechanisms of pathogenic RAS signaling in this disease remain enigmatic and difficult to inhibit therapeutically.  We employed an unbiased proteogenomic approach to dissect RAS signaling in MM by combining genome-wide CRISPR-Cas9 screening with quantitative mass spectrometry focused on RAS biology.  We discovered that mutant isoforms of RAS organized a signaling complex with the amino acid transporter, SLC3A2, and MTOR on endolysosomes, which directly activated mTORC1 by co-opting amino acid sensing pathways.  MM tumors with high expression of mTORC1-dependent genes were more aggressive and enriched in RAS mutations, and we detected interactions between RAS and MTOR in MM patient tumors harboring mutant RAS isoforms.  Inhibition of RAS-dependent mTORC1 activity synergized with MEK and ERK inhibitors to quench pathogenic RAS signaling in MM cells.  This study redefines the RAS pathway in MM and provides a mechanistic and rational basis to target this novel mode of RAS signaling.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.22.468946', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'FLI1 and FRA1 transcription factors drive the transcriptional regulatory networks characterizing muscle invasive bladder cancer', 'authors': \"[{'name': 'Perihan Yagmur Guneri-Sozeri'}, {'name': 'Gulden Ozden Yilmaz'}, {'name': 'Asli Kisim'}, {'name': 'Aleyna Eray'}, {'name': 'Hamdiye Uzuner'}, {'name': 'Gkhan Karaklah'}, {'name': 'Devrim Pesen Okvur'}, {'name': 'Serif Senturk'}, {'name': 'Serap Erkek-Ozhan'}]\", 'abstract': '<p>Bladder cancer is mostly present in the form of urothelium carcinoma, causing over 150.000 deaths each year. Its histopathological classification as muscle invasive (MIBC) and non-muscle invasive (NMIBC) is the most prominent aspect, affecting the prognosis and progression of this disease. In this study, we defined the active regulatory landscape of MIBC and NMIBC cell lines using H3K27ac-seq and used an integrative data approach to combine our findings with existing data. Our analysis revealed FRA1 and FLI1 as the two critical transcription factors differentially regulating MIBC regulatory landscape. Importantly, we show that FRA1 and FLI1 regulate the genes involved in epithelial cell migration and cell junction organization. Knock-down of FRA1 and FLI1 in MIBC revealed the downregulation of several EMT-related genes such as <italic>MAP4K4</italic> and <italic>FLOT1</italic>. Further, ChIP-SICAP performed for FRA1 and FLI1 enabled us to infer chromatin binding partners of these two transcription factors and link this information with their target genes, providing a comprehensive regulatory circuit for the genes implicated in invasive ability of the bladder cancer cells. Finally, for the first time we show that knock-down of FRA1 and FRA1-FLI1 double knock-down results in significant reduction of invasion capacity of MIBC cells towards muscle microenvironment using IC-CHIP assays. Our results collectively highlight the role of these two transcription factors in invasive characteristics of bladder cancer in selection and design of targeted options for treatment of MIBC.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.21266713', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'In Vitro Nasal Tissue Model for the Validation of Nasopharyngeal and Mid-turbinate Swabs for SARS-CoV-2 Testing', 'authors': \"[{'name': 'Devon R. Hartigan'}, {'name': 'Miryam Adelfio'}, {'name': 'Molly E. Shutt'}, {'name': 'Stephanie M. Jones'}, {'name': 'Shreya Patel'}, {'name': 'Joshua T. Marchand'}, {'name': 'Pamela D. McGuinness'}, {'name': 'Bryan O. Buchholz'}, {'name': 'Chiara E. Ghezzi'}]\", 'abstract': '<p>Large-scale population testing is a key tool to mitigate the spread of respiratory pathogens, as in the current COVID-19 pandemic, where swabs are used to collect samples in the upper airways (e.g. nasopharyngeal and mid-turbinate nasal cavities) for diagnostics. However, the high volume of supplies required to achieve large-scale population testing has posed unprecedented challenges for swab manufacturing and distribution, resulting in a global shortage that has heavily impacted testing capacity world-wide and prompted the development of new swabs suitable for large-scale production. Newly designed swabs require rigorous pre-clinical and clinical validation studies that are costly and time consuming (<italic>i</italic>.<italic>e</italic>. months to years long); reducing the risks associated with swab validation is therefore paramount for their rapid deployment. To address these shortages, we developed a 3D-printed tissue model that mimics the nasopharyngeal and mid-turbinate nasal cavities, and we validated its use as a new tool to rapidly test swab performance. In addition to the nasal architecture, the tissue model mimics the soft nasal tissue with a silk-based sponge lining, and the physiological nasal fluid with asymptomatic and symptomatic viscosities of synthetic mucus. We performed several assays comparing standard flocked and injection-molded swabs. We quantified the swab pick-up and release, and determined the effect of viral load and mucus viscosity on swab efficacy by spiking the synthetic mucus with heat-inactivated SARS-CoV-2 virus. By molecular assays, we found that injected molded swabs performed similarly or superiorly in comparison to standard flocked swabs and we underscored a viscosity-dependent difference in cycle threshold values between the asymptomatic and symptomatic mucus for both swabs. To conclude, we developed an <italic>in vitro</italic> nasal tissue model, that corroborated previous swab performance data from clinical studies, with the potential of providing researchers with a clinically relevant, reproducible, safe, and cost-effective validation tool for the rapid development of newly designed swabs.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469508', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Effect of therapeutic ultrasound on the mechanical and biological properties of fibroblasts', 'authors': \"[{'name': 'Rosy P. Crdenas-Sandoval'}, {'name': 'Homero F. Pastrana-Rendn'}, {'name': 'Alba G. vila-Bernal'}, {'name': 'Anglica M. Ramrez-Martnez'}, {'name': 'Myriam L. Navarrete-Jimenez'}, {'name': 'Alejandro O. Ondo-Mendez'}, {'name': 'Diego A. Garzn-Alvarado'}]\", 'abstract': '<p>Conventional doses of therapeutic ultrasound alter the mechanical behavior of ligament fibroblasts to improve the regenerative and remodeling stages of the wound healing process. Using a multidisciplinary approach, we applied ultrasound doses of 1.0 and 2.0 W/cm<sup>2</sup> at 1 MHz frequency for five days on ligament fibroblasts. Atomic force microscopy showed a decrease in cell elastic modulus for both doses, but the treated cells were still viable based on flow cytometry. Finite element method analysis exhibited visible cytoskeleton displacements and decreased harmonics in treated cells. Colorimetric assay revealed increased cell proliferation, while scratch assay showed increased migration at low doses. An increase in collagen and fibronectin was detected by enzyme-linked immunoassay at high doses, and -actin expression for both treatments was visualized through immunofluorescence imaging. Both doses of ultrasound altered the fibroblast mechanical properties due to cytoskeletal reorganization and enhanced the early and late stages of cell repair.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.26.470085', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'IL-37 expression reduces acute and chronic neuroinflammation and rescues cognitive impairment in an Alzheimer s disease mouse model', 'authors': \"[{'name': 'Niklas Lonnemann'}, {'name': 'Shirin Hosseini'}, {'name': 'Melanie Ohm'}, {'name': 'Karsten Hiller'}, {'name': 'Charles A. Dinarello'}, {'name': 'Martin Korte'}]\", 'abstract': \"<p>The anti-inflammatory cytokine interleukin-37 (IL-37) is a member of the IL-1 family but not expressed in mice. We used a human IL 37 (hIL-37tg) expressing mouse, which has been subjected to various models of local and systemic inflammation as well as immunological challenges. Those studies demonstrate an immune-modulatory role of IL-37 which can be characterized as an important suppressor of innate immunity. We investigated the functions of IL-37 in the CNS and explored the effects of IL-37 on neuronal architecture and function, microglia phenotype, cytokine production and behavior after inflammatory challenge by intraperitoneal LPS-injection. Reduced spine density, activated microglia phenotype and impaired long-term potentiation (LTP) were observed in wild-type mice after LPS injection, whereas hIL-37tg mice showed no impairment. In addition, we crossed the hIL-37tg mouse with an animal model of Alzheimer's disease (APP/PS1) to investigate the anti-inflammatory properties of IL-37 under chronic neuroinflammatory conditions. Our results show that IL-37 is able to limit inflammation in the brain after acute inflammatory events and prevent the loss of cognitive abilities in a mouse model of AD.</p>\", 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.23.469578', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Resting state functional connectivity in relapsing remitting multiple sclerosis with mild disability  a data driven, whole brain multivoxel pattern analysis study', 'authors': \"[{'name': 'Gowthami Nair'}, {'name': 'Sruthi S. Nair'}, {'name': 'K. M. Arun'}, {'name': 'Paul Camacho'}, {'name': 'Elshal Bava'}, {'name': 'Priya Ajayaghosh'}, {'name': 'Ramshekhar N. Menon'}, {'name': 'Muralidharan Nair'}, {'name': 'Chandrasekharan Kesavadas'}, {'name': 'Sheeba Arnold Anteraper'}]\", 'abstract': '<p>Multivoxel pattern analysis (MVPA) has emerged as a powerful unbiased approach for generating seed regions of interest (ROIs) in resting-state functional connectivity (RSFC) analysis in a data-driven manner. The aim of the present study was to investigate RSFC differences between persons with relapsing-remitting multiple sclerosis (RRMS) and healthy controls (HCs). We performed a whole-brain connectome-wide MVPA in 50 RRMS patients with expanded disability status scale 4 and 50 age and gender-matched HCs. Significant group differences were noted in RSFC in 9 clusters distributed in 7 regions; right middle frontal gyrus, frontal medial cortex, left frontal pole, anterior cingulate gyrus, right middle temporal gyrus, left posterior middle temporal gyrus and right lateral occipital cortex. Whole-brain seed-to-voxel RSFC characterization of these clusters as seed ROIs revealed significantly increased RSFC to the posterior brain regions (bilateral superior lateral occipital cortices, right lingual gyrus and left occipital pole) and reduced connectivity in the anterior and medial regions (right paracingulate gyrus, anterior cingulate gyrus, left amygdala and left frontal orbital cortex) in RRMS compared to HCs. The results of this study agree with the previous reports on abnormalities of RSFC in RRMS, the cognitive and clinical implications of which are discussed herein.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469706', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'High-resolution mapping and digital atlas of subcortical regions in the macaque monkey based on matched MAP-MRI and histology', 'authors': \"[{'name': 'Kadharbatcha S Saleem'}, {'name': 'Alexandru V Avram'}, {'name': 'Daniel Glen'}, {'name': 'Cecil Chern-Chyi Yen'}, {'name': 'Frank Q Ye'}, {'name': 'Michal Komlosh'}, {'name': 'Peter J Basser'}]\", 'abstract': '<p>Subcortical nuclei and other deep brain structures are known to play an important role in the regulation of the central and peripheral nervous systems. It can be difficult to identify and delineate many of these nuclei and their finer subdivisions in conventional MRI due to their small size, buried location, and often subtle contrast compared to neighboring tissue. To address this problem, we applied a multi-modal approach in <italic>ex vivo</italic> non-human primate (NHP) brain that includes high-resolution mean apparent propagator (MAP)-MRI and five different histological stains imaged with high-resolution microscopy in the brain of the same subject. By registering these high-dimensional MRI data to high-resolution histology data, we can map the location, boundaries, subdivisions, and micro-architectural features of subcortical gray matter regions in the macaque monkey brain. At high spatial resolution, diffusion MRI in general, and MAP-MRI in particular, can distinguish a large number of deep brain structures, including the larger and smaller white matter fiber tracts as well as architectonic features within various nuclei. Correlation with histology from the same brain enables a thorough validation of the structures identified with MAP-MRI. Moreover, anatomical details that are evident in images of MAP-MRI parameters are not visible in conventional T1-weighted images. We also derived subcortical template SC21 from segmented MRI slices in three-dimensions and registered this volume to a previously published anatomical template with cortical parcellation (Reveley et al., 2017; Saleem and Logothetis, 2012), thereby integrating the 3D segmentation of both cortical and subcortical regions into the same volume. This newly updated three-dimensional D99 digital brain atlas (V2.0) is intended for use as a reference standard for macaque neuroanatomical, functional, and connectional imaging studies, involving both cortical and subcortical targets. The SC21 and D99 digital templates are available as volumes and surfaces in standard NIFTI and GIFTI formats.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469596', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Fine-tuning of deep language models as a computational framework of modeling listeners perspective during language comprehension', 'authors': \"[{'name': 'Refael Tikochinski'}, {'name': 'Ariel Goldstein'}, {'name': 'Yaara Yeshurun'}, {'name': 'Uri Hasson'}, {'name': 'Roi Reichart'}]\", 'abstract': '<p>Computational Deep Language Models (DLMs) have been shown to be effective in predicting neural responses during natural language processing. This study introduces a novel computational framework, based on the concept of fine-tuning (Hinton, 2007), for modeling differences in interpretation of narratives based on the listeners perspective (i.e. their prior knowledge, thoughts, and beliefs). We draw on an fMRI experiment conducted by Yeshurun et al. (2017), in which two groups of listeners were listening to the same narrative but with two different perspectives (cheating versus paranoia). We collected a dedicated dataset of ~3000 stories, and used it to create two modified (fine-tuned) versions of a pre-trained DLM, each representing the perspective of a different group of listeners. Information extracted from each of the two fine-tuned models was better fitted with neural responses of the corresponding group of listeners. Furthermore, we show that the degree of difference between the listeners interpretation of the story - as measured both neurally and behaviorally - can be approximated using the distances between the representations of the story extracted from these two fine-tuned models. These models-brain associations were expressed in many language-related brain areas, as well as in several higher-order areas related to the default-mode and the mentalizing networks, therefore implying that computational fine-tuning reliably captures relevant aspects of human language comprehension across different levels of cognitive processing.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469612', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Copy number alterations and epithelial-mesenchymal transition genes in diffuse and intestinal gastric cancers in Mexican patients', 'authors': \"[{'name': 'Violeta Larios-Serrato'}, {'name': 'Jos-Daro Martnez-Ezquerro'}, {'name': 'Hilda-Alicia Valdez-Salazar'}, {'name': 'Javier Torres'}, {'name': 'Margarita Camorlinga-Ponce'}, {'name': 'Patricia Pia-Snchez'}, {'name': 'Martha-Eugenia Ruiz-Tachiqun'}]\", 'abstract': '<p>Gastric cancer (GC) is a malignancy with the highest mortality among diseases of the digestive system worldwide. The study of GC-alterations is crucial to understand tumor biology, to establish important aspects of cancer prognosis and treatment response. Here, we purified DNA and performed whole-genome analysis with high-density arrays in samples from Mexican patients diagnosed with GC: diffuse (DGC) or intestinal (IGC), or non-atrophic gastritis (NAG) samples that served as controls. We identified shared and unique copy number alterations (CNA) between these altered tissues involving key genes and signaling pathways associated with cancer, allowing their molecular distinction and identification of the most relevant molecular functions impacted. When focused on epithelial-mesenchymal transition (EMT) genes, our bioinformatic analysis revealed that the altered network associated with chromosomal alterations included 11 genes shared between DGC, IGC, and NAG, as well as 19 DGC- and 7 IGC-exclusive genes, whose main molecular functions included adhesion, angiogenesis, migration, metastasis, morphogenesis, proliferation, and survival. This study presents the first whole-genome high-density array study in GC from Mexican patients and reveals shared and exclusive CNA-genes in DGC and IGC. In addition, we provide a bioinformatically predicted network focused on CNA-altered genes involved in the EMT, associated with the hallmarks of cancer, as well as precancerous alterations that could lead to gastric cancer.</p><sec><title>Implications</title><p>Molecular signatures of diffuse and intestinal GC, predicted bioinformatically, involve common and distinct CNA-EMT genes related to the hallmarks of cancer that are potential candidates for screening GC biomarkers, including early stages.</p></sec>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.23.469712', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A novel TREM2 splice isoform lacking the ligand binding is expressed in brain and shares localization', 'authors': \"[{'name': 'Benjamin C. Shaw'}, {'name': 'Henry C. Snider'}, {'name': 'Andrew K. Turner'}, {'name': 'Diana J. Zajac'}, {'name': 'James F. Simpson'}, {'name': 'Steven Estus'}]\", 'abstract': '<sec><title>Background</title><p>Genetic variants in <italic>TREM2</italic> are strongly associated with Alzheimers Disease (AD) risk but alternative splicing in <italic>TREM2</italic> transcripts has not been comprehensively described.</p></sec><sec><title>Objective</title><p>Recognizing that alternative splice variants can result in reduced gene expression and/or altered function, we sought to fully characterize splice variation in <italic>TREM2</italic>.</p></sec><sec><title>Methods</title><p>Human blood and anterior cingulate autopsy tissue from 61 donors were used for genotyping and cDNA synthesis followed by both end-point and quantitative PCR to identify and quantify novel <italic>TREM2</italic> isoforms.</p></sec><sec><title>Results</title><p>In addition to previously described transcripts lacking exon 3 or exon 4, or retaining part of intron 3, we identified novel isoforms lacking exon 2, along with isoforms lacking multiple exons. Isoforms lacking exon 2 were predominant at approximately 10% of TREM2 mRNA in the brain. Expression of <italic>TREM2</italic> and frequency of exon 2 skipping did not differ between AD samples and non-AD controls (p = 0.1268 and p = 0.4909, respectively). Further, these novel splice isoforms were also observed across multiple tissues with similar frequency (range 5.3  13.0%). Using ectopic expression, we found that the exon 2 skipped isoform <italic>D2-TREM2</italic> is translated to protein and localizes similarly to full-length TREM2 protein, and that both proteins are primarily retained in the Golgi complex.</p></sec><sec><title>Conclusion</title><p>Since the TREM2 ligand binding domain is encoded by exon 2, and skipping this exon retains reading frame while conserving localization, we hypothesize that <italic>D2-TREM2</italic> acts as an inhibitor of <italic>TREM2</italic> and that targeting <italic>TREM2</italic> splicing may be a novel therapeutic pathway for AD.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469645', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genomic variation across a clinical Cryptococcus population linked to disease outcome', 'authors': \"[{'name': 'Poppy Sephton-Clark'}, {'name': 'Jennifer L. Tenor'}, {'name': 'Dena L. Toffaletti'}, {'name': 'Nancy Meyers'}, {'name': 'Charles Giamberardino'}, {'name': 'Sle F. Molloy'}, {'name': 'Adrienne Chan'}, {'name': 'Tarsizio Chikaonda'}, {'name': 'Robert Heyderman'}, {'name': 'Mina Hosseinipour'}, {'name': 'Newton Kalata'}, {'name': 'Cecilia Kanyama'}, {'name': 'Christopher Kukacha'}, {'name': 'Duncan Lupiya'}, {'name': 'Henry C. Mwandumba'}, {'name': 'Thomas Harrison'}, {'name': 'Tihana Bicanic'}, {'name': 'John R. Perfect'}, {'name': 'Christina A. Cuomo'}]\", 'abstract': '<p><italic>Cryptococcus neoformans</italic> is the causative agent of cryptococcosis, a disease with poor patient outcomes, accounting for approximately 180,000 deaths each year. Patient outcomes may be impacted by the underlying genetics of the infecting isolate, however, our current understanding of how genetic diversity contributes to clinical outcomes is limited. Here, we leverage clinical, in vitro growth and genomic data for 284 <italic>C. neoformans</italic> isolates to identify clinically relevant pathogen variants within a population of clinical isolates from patients with HIV-associated cryptococcosis in Malawi. Through a genome-wide association study (GWAS) approach, we identify variants associated with fungal burden and growth rate. We also find both small and large-scale variation, including aneuploidy, associated with alternate growth phenotypes, which may impact the course of infection. Genes impacted by these variants are involved in transcriptional regulation, signal transduction, glycolysis, sugar transport, and glycosylation. When combined with clinical data, we show that growth within the CNS is reliant upon glycolysis in an animal model, and likely impacts patient mortality, as CNS burden modulates patient outcome. Additionally, we find genes with roles in sugar transport are under selection in the majority of these clinical isolates. Further, we demonstrate that two hypothetical proteins identified by GWAS impact virulence in animal models. Our approach illustrates links between genetic variation and clinically relevant phenotypes, shedding light on survival mechanisms within the CNS and pathways involved in this persistence.</p><sec><title>Importance</title><p>Infection outcomes for cryptococcosis, most commonly caused by <italic>C. neoformans</italic>, are influenced by host immune responses, as well as host and pathogen genetics. Infecting yeast isolates are genetically diverse, however, we lack a deep understanding of how this diversity impacts patient outcomes. To better understand both clinical isolate diversity and how diversity contributes to infection outcome, we utilize a large collection of clinical <italic>C. neoformans</italic> samples, isolated from patients enrolled in a clinical trial across 3 hospitals in Malawi. By combining whole-genome sequence data, clinical data, and in vitro growth data, we utilize genome-wide association approaches to examine the genetic basis of virulence. Genes with significant associations show virulence phenotypes in both murine and rabbit models, demonstrating that our approach successfully identifies links between genetic variation and biologically significant phenotypes.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469597', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Identification of a genomic DNA sequence that quantitatively modulates KLF1 expression in differentiating human hematopoietic cells', 'authors': \"[{'name': 'Merlin Nithya Gnanapragasam'}, {'name': 'Antanas Planutis'}, {'name': 'Jeffery A Glassberg'}, {'name': 'James J Bieker'}]\", 'abstract': '<p>Expression of the  -like globin genes is under strict developmental control, with both direct and indirect inputs responsible for this effect. One of the major players regulating their transition is KLF1/EKLF, where even a two-fold difference in its level alters the regulation of globin switching. We have reproduced this change in KLF1 expression in both cell lines and primary human cells, thus demonstrating that directed, quantitative control of KLF1 expression can be attained by genomic manipulation, and suggest a new way in which modulation of transcription factor levels may be used for clinical benefit.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.28.470275', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'DUAL FUNCTION OF ZIKA VIRUS NS2B-NS3 PROTEASE', 'authors': \"[{'name': 'Alexey Terskikh'}, {'name': 'Sergey Shiryaev'}, {'name': 'Anton Cheltsov'}, {'name': 'Robert C Liddington'}]\", 'abstract': '<p>Zika virus (ZIKV) serine protease, indispensable for viral polyprotein processing and replication, is composed of an NS2B polypeptide that associates with a proteolytic N terminal fragment of NS3 polypeptide (NS3pro) to form NS2B-NS3pro. The larger C-terminal fragment of NS3 polypeptide contains helicase activity. In the present study, we discovered that ZIKV NS2BNS3pro efficiently binds single-stranded (ss) RNA (Kd ~0.3 uM), suggesting that the protease may have a novel function. We tested an array of NS2B-NS3pro modifications and found that NS2B NS3pro constructs that adopt the recently discovered super-open conformation could not bind ssRNA. Likewise, stabilization of NS2B-NS3pro in the closed (proteolytically active) conformation by substrate-like inhibitors abolished ssRNA binding. Therefore, we suggest that ssRNA binding occurs when ZIKV protease adopts the open conformation, which could be modeled using dengue NS2B-NS3pro in the open conformation. ssRNA binding competes with ZIKV NS2B-NS3pro protease activity, likely by shifting the complex into the open conformation. Modeling of ZIKV NS3 helicase activity based on homologous crystal structures suggests that the open conformation of NS3pro domains provides a positively charged surface contiguous with the NS3 helicase domain. Such a positively charged surface is well poised to bind ssRNA, providing an explanation for the previously observed requirement of NS3pro for RNA processivity by viral helicase. Our structure-function analyses suggest that binding of ssRNA by the protease domain of NS3 is likely to be a universal feature of Flaviviridae, given the high level of homology between NS3 protease-helicase proteins in this family.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.23.21266487', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Cardiovascular Medicine', 'covid': False, 'title': 'Intersecting single-cell transcriptomics and genome-wide association studies identifies crucial cell populations and candidate genes for atherosclerosis', 'authors': \"[{'name': 'Lotte Slenders'}, {'name': 'Lennart P. L. Landsmeer'}, {'name': 'Kai Cui'}, {'name': 'Marie A.C. Depuydt'}, {'name': 'Maarten Verwer'}, {'name': 'Joost Mekke'}, {'name': 'Nathalie Timmerman'}, {'name': 'Noortje A.M. van den Dungen'}, {'name': 'Johan Kuiper'}, {'name': 'Menno P.J. Winther'}, {'name': 'Koen H.M. Prange'}, {'name': 'Wei Feng Ma'}, {'name': 'Clint L. Miller'}, {'name': 'Redouane Aherrahrou'}, {'name': 'Mete Civelek'}, {'name': 'Gert J. de Borst'}, {'name': 'Dominique P.V. de Kleijn'}, {'name': 'Folkert W. Asselbergs'}, {'name': 'Hester M. den Ruijter'}, {'name': 'Arjan Boltjes'}, {'name': 'Gerard Pasterkamp'}, {'name': 'Sander W. van der Laan'}, {'name': 'Michal Mokry'}]\", 'abstract': '<sec><title>Background</title><p>Genome-wide association studies have discovered hundreds of common genetic variants for atherosclerotic disease and cardiovascular risk factors. The translation of susceptibility loci into biological mechanisms and targets for drug discovery remains challenging. Intersecting genetic and gene expression data has led to the identification of candidate genes. However, previously studied tissues are often non-diseased and heterogeneous in cell composition, hindering accurate candidate prioritization. Therefore, we analyzed single-cell transcriptomics from atherosclerotic plaques for cell-type-specific expression to identify atherosclerosis-associated candidate gene-cell pairs.</p></sec><sec><title>Methods and Results</title><p>To identify disease-associated genes, we applied gene-based analyses using GWAS summary statistics from 46 atherosclerotic and cardiovascular disease, risk factors, and other traits. We then intersected these candidates with scRNA-seq data to identify genes specific for individual cell (sub)populations in atherosclerotic plaques. The coronary artery disease loci demonstrated a prominent signal in plaque smooth muscle cells (<italic>SKI</italic>, <italic>KANK2</italic>, <italic>SORT1</italic>) p-adj. = 0.0012, and endothelial cells (<italic>SLC44A1</italic>, <italic>ATP2B1</italic>) p-adj. = 0.0011. Further sub clustering revealed genes in risk loci for coronary calcification specifically enriched in a synthetic smooth muscle cell population. Finally, we used liver-derived scRNA-seq data and showed hepatocyte-specific enrichment of genes involved in serum lipid levels.</p></sec><sec><title>Conclusion</title><p>We discovered novel gene-cell pairs, on top of known pairs, pointing to new biological mechanisms of atherosclerotic disease. We highlight that loci associated with coronary artery disease reveal prominent association levels in mainly plaque smooth muscle and endothelial cell populations. We present an intuitive single-cell transcriptomics-driven workflow rooted in human large-scale genetic studies to identify putative candidate genes and affected cells associated with cardiovascular traits. Collectively, our workflow allows for the identification of cell-specific targets relevant for atherosclerosis and can be universally applied to other complex genetic diseases and traits.</p></sec><sec><title>Translational perspective</title><p>GWAS identified a large number of genomic loci associated with atherosclerotic disease. The translation of these results into drug development and faster diagnostics remains challenging. With our approach, we cross-reference the GWAS findings for atherosclerotic disease with scRNA-seq data of disease-relevant tissue and bring the GWAS findings closer to the functional and mechanistic studies.</p></sec><sec><title>Abstract Figure</title><fig id=\"ufig1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"21266487v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469691', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Global Structure of the Intrinsically Disordered Protein Tau Emerges from its Local Structure', 'authors': \"[{'name': 'Lukas S. Stelzl'}, {'name': 'Lisa M. Pietrek'}, {'name': 'Andrea Holla'}, {'name': 'Javier Oroz'}, {'name': 'Mateusz Sikora'}, {'name': 'Jrgen Kfinger'}, {'name': 'Benjamin Schuler'}, {'name': 'Markus Zweckstetter'}, {'name': 'Gerhard Hummer'}]\", 'abstract': '<p>The paradigmatic disordered protein tau plays an important role in neuronal function and neurodegenerative diseases. To disentangle the factors controlling the balance between functional and disease-associated conformational states, we build a structural ensemble of the tau K18 fragment containing the four pseudorepeat domains involved in both microtubule binding and amyloid fibril formation. We assemble 129-residue-long tau K18 chains at atomic resolution from an extensive fragment library constructed with molecular dynamics simulations. We introduce a reweighted hierarchical chain growth (RHCG) algorithm that integrates experimental data reporting on the local structure into the assembly process in a systematic manner. By combining Bayesian ensemble refinement with importance sampling, we obtain well-defined ensembles and overcome the problem of exponentially varying weights in the integrative modeling of long-chain polymeric molecules. The resulting tau K18 ensembles capture nuclear magnetic resonance (NMR) chemical shift and J-coupling measurements. Without further fitting, we achieve excellent agreement with measurements of NMR residual dipolar couplings. The good agreement with experimental measures of global structures such as singlemolecule Frster resonance energy transfer (FRET) efficiencies is improved further by ensemble refinement. By comparing wild-type and mutant ensembles, we show that pathogenic single-point P301 mutations shift the population from the turn-like conformations of the functional microtubule-bound state to the extended conformations of disease-associated tau fibrils. RHCG thus provides us with an atomically resolved view of the population equilibrium between functional and aggregation-prone states of tau K18, and demonstrates that global structural characteristics of this intrinsically disordered protein emerge from its local structure.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.28.470209', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'A conserved long intergenic non-coding RNA containing snoRNA sequences, lncCOBRA1, affects Arabidopsis germination and development', 'authors': \"[{'name': 'Marianne C Kramer'}, {'name': 'Hee Jong Kim'}, {'name': 'Kyle R Palos'}, {'name': 'Benjamin A Garcia'}, {'name': 'Eric Lyons'}, {'name': 'Mark A Beilstein'}, {'name': 'Andrew D.L. Nelson'}, {'name': 'Brian D Gregory'}]\", 'abstract': '<p>Long non-coding RNAs (lncRNAs) are an increasingly studied group of non-protein-coding transcripts with a wide variety of molecular functions gaining attention for their roles in numerous biological processes. Nearly 6,000 lncRNAs have been identified in Arabidopsis thaliana but many have yet to be studied. Here, we examine a class of previously uncharacterized lncRNAs termed CONSERVED IN BRASSICA RAPA (lncCOBRA) transcripts that were previously identified for their high level of sequence conservation in the related crop species Brassica rapa, their nuclear-localization and protein-bound nature. In particular, we focus on lncCOBRA1 and demonstrate that its abundance is highly tissue and developmental specific, with particularly high levels early in germination. lncCOBRA1 contains two snoRNAs domains within it, making it the first sno-lincRNA example in a non-mammalian system. However, we find that it is processed differently than its mammalian counterparts. We further show that plants lacking lncCOBRA1 display patterns of delayed gemination and are overall smaller than wild-type plants. Lastly, we identify the proteins that interact with lncCOBRA1 and examine the protein-protein interaction network of lncCOBRA1-interacting proteins.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.27.470183', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Obesity impairs therapeutic efficacy of mesenchymal stem cells by inhibiting cardiolipin-dependent mitophagy and intercellular mitochondrial transfer in mouse models of airway allergic inflammation', 'authors': \"[{'name': 'Shakti Sagar'}, {'name': 'Md. Imam Faizan'}, {'name': 'Nisha Chaudhary'}, {'name': 'Atish Gheware'}, {'name': 'Khushboo Sharma'}, {'name': 'Iqbal Azmi'}, {'name': 'Vijay Pal Singh'}, {'name': 'Gaurav Kharya'}, {'name': 'Ulaganathan Mabalirajan'}, {'name': 'Anurag Agrawal'}, {'name': 'Tanveer Ahmad'}, {'name': 'Soumya Sinha Roy'}]\", 'abstract': '<p>Mesenchymal stem cell (MSC) transplantation alleviates metabolic defects in diseased recipient cells by intercellular mitochondrial transport (IMT). However, the effect of host metabolic conditions on MSCs in general, and IMT in particular, has largely remained unexplored. This study has identified a molecular pathway that primarily governs the metabolic function and IMT of MSCs. We found underlying mitochondrial dysfunction, impaired mitophagy, and reduced IMT in MSCs derived from high-fat diet (HFD)-induced obese mice (MSC-Ob). Mechanistically, MSC-Ob failed to sequester their damaged mitochondria into LC3-dependent autophagosomes due to decrease in mitochondrial cardiolipin content, which we propose as a putative mitophagy receptor for LC3 in MSCs. Functionally, MSC-Ob exhibited diminished potential to rescue metabolic deficits and cell death in stress-induced epithelial cells. In a small molecule screen, we found pyrroloquinoline quinone (PQQ) as a regulator of mitophagy and IMT. Long-term culture of MSC-Ob with PQQ (MSC-ObPQQ) restored cardiolipin content and sequestration of mitochondria to autophagosomes with concomitant activation of mitophagy. Upon co-culture, MSC-ObPQQ rescued cell death in stress-induced epithelial cells by enhancing IMT. The beneficial effect of PQQ was also evident in MSCs derived from human subjects in an in vitro model. In two independent mice models, the transplantation of MSC-ObPQQ restored IMT to airway epithelial cells, improved their mitochondrial metabolism and attenuated features of allergic airway inflammation (AAI). However, unmodulated MSC-Ob failed to do so. In summary, we uncover the molecular mechanism leading to the therapeutic decline of obese-derived MSCs and highlight the importance of pharmacological modulation of these cells for therapeutic intervention.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.24.469914', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Over 400 million years of cooperation: Untangling the chondrocranium-dermatocranium connection', 'authors': \"[{'name': 'Susan M. Motch Perrine'}, {'name': 'M. Kathleen Pitirri'}, {'name': 'Emily L. Durham'}, {'name': 'Mizuho Kawasaki'}, {'name': 'Hao Zheng'}, {'name': 'Danny Z. Chen'}, {'name': 'Kazuhiko Kawasaki'}, {'name': 'Joan T. Richtsmeier'}]\", 'abstract': '<p>The cranial endo- and dermal skeletons, which comprise the vertebrate skull, evolved independently and form separately during embryogenesis. In mammals, the mostly cartilaginous cranial endoskeleton forms prior to the bony dermatocranium. Many features of the chondrocranium are transient, undergoing endochondral ossification or disappearing, so its role in skull morphogenesis is not understood The fibroblast growth factor (FGF) and receptor (FGFR) signaling pathway contributes significantly to the regulation of osteochondroprogenitor cell function. Mutations in FGFR genes are associated with diseases that impact the skull including dwarfing chondrodyplasia and craniosynostosis syndromes. We investigate the developing chondrocranium and dermatocranium using a mouse model for craniosynostosis carrying a gain of function mutation in Fgfr2 to assess development of these cranial skeleton systems. Dermatocrania and chondrocrania of <italic>Fgfr2c</italic><sup><italic>C342Y/+</italic></sup> mice and their <italic>Fgfr2c</italic><sup><italic>+/+</italic></sup> littermates were quantified in 3D from microcomputed tomography images of mouse embryos. Chondrocrania of embryonic mice carrying the Fgfr2 mutation are larger than their <italic>Fgfr2c</italic><sup><italic>+/+</italic></sup> littermates and include novel extensions of cartilage over the lateral and dorsal aspect of the brain. Like the forming chondrocranium, the embryonic dermatocranium is larger in <italic>Fgfr2c</italic><sup><italic>C342Y/+</italic></sup> mice throughout embryogenesis but after disappearance of much of the chondrocranium, the dermatocranium becomes progressively smaller relative to <italic>Fgfr2c</italic><sup><italic>+/+</italic></sup> littermates during postnatal growth. Results reveal the direct effects of this Fgfr2c mutation on embryonic cranial cartilage, the impact of chondrocranial structure on developing dermatocranial elements, the importance of the chondrocranium in decoding the impact of specific genetic variants on head morphogenesis, and the potential for harnessing these effects as therapeutic targets.</p><sec><title>Significance Statement</title><p>We present the first fully complete three-dimensional (3D) reconstructions of the mouse embryonic chondrocranium using a novel methodology of uncertainty guided segmentation of microcomputed tomography images with sparse annotation. We provide 3D reconstructions of chondrocrania of the <italic>Fgfr2c</italic><sup><italic>C342Y/+</italic></sup> Crouzon syndrome mouse and typically developing littermates for embryonic days 13.5, 14.5, 15.5, 16.5, and 17.5. This is the first study of the effects of an FGFR2 mutation on embryonic chondrocranial cartilage. 3D reconstructions of embryonic dermal bones reveal that the dermatocranium develops outside of, and in shapes that conform to the chondrocranium. Our findings have implications for the study and treatment of human craniofacial disease and for understanding the impact of chondrocranial morphology on the evolution of skull morphology.</p></sec>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.24.469894', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'RyR2/IRBIT regulates insulin gene transcription, insulin content, and secretion in the insulinoma cell line INS-1', 'authors': \"[{'name': 'Kyle E. Harvey'}, {'name': 'Emily K. LaVigne'}, {'name': 'Mohd Saleem Dar'}, {'name': 'Amy E. Salyer'}, {'name': 'Evan P.S. Pratt'}, {'name': 'Paxton A. Sample'}, {'name': 'Uma Aryal'}, {'name': 'Humaira Gowher'}, {'name': 'Gregory H. Hockerman'}]\", 'abstract': '<p>The role of ER Ca<sup>2+</sup> release via ryanodine receptors (RyR) in pancreatic -cell function is not well defined. Deletion of RyR2 from the rat insulinoma INS-1 (RyR2<sup>KO</sup>) enhanced the Ca<sup>2+</sup> integral (AUC) stimulated by 7.5 mM glucose, and rendered it sensitive to block by the IP<sub>3</sub> receptor inhibitor xestospongin C, coincident with reduced levels of the protein IP<sub>3</sub><underline>R</underline>eceptor <underline>B</underline>inding protein released with Inositol 1,4,5 <underline>T</underline>risphosphate (IRBIT; aka AHCYL1). Deletion of IRBIT from INS-1 cells (IRBIT<sup>KO</sup>) increased the Ca<sup>2+</sup> AUC in response to 7.5 mM glucose and induced xestospongin sensitivity. Insulin content and basal (2.5 mM glucose) and 7.5 mM glucose-stimulated insulin secretion were reduced in RyR2<sup>KO</sup> cells and more modestly reduced in IRBIT<sup>KO</sup> cells compared to controls. <italic>INS2</italic> mRNA levels were reduced in both RyR2<sup>KO</sup> and IRBIT<sup>KO</sup> cells, but <italic>INS1</italic> mRNA levels were specifically decreased in RyR2<sup>KO</sup> cells. Nuclear localization of S-adenosylhomocysteinase (AHCY) was increased in RyR2<sup>KO</sup> and IRBIT<sup>KO</sup> cells. DNA methylation of the <italic>INS1</italic> and <italic>INS2</italic> gene promotor regions was very low, and not different among RyR2<sup>KO</sup>, IRBIT<sup>KO</sup>, and controls. In contrast, exon 2 of the <italic>INS1</italic> and <italic>INS2</italic> genes was more extensively methylated in RyR2<sup>KO</sup> and IRBIT<sup>KO</sup> cells than in controls. Proteomics analysis using LC-MS/MS revealed that deletion of RyR2 or IRBIT resulted in differential regulation of 314 and 137 proteins, respectively, with 41 in common. These results suggest that RyR2 regulates IRBIT levels and activity in INS-1 cells, and together maintain insulin content and secretion, and regulate the proteome, perhaps via DNA methylation.</p><sec><title>One sentence Summary</title><p>Deletion of RyR2 from INS-1 cells had the unanticipated effect of reducing IRBIT proteins levels, and both RyR2 and IRBIT contribute to maintenance of glucose- stimulated insulin secretion.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.26.470012', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Patients with recurrent PVL+-Staphylococcus aureus infections show enhanced sensitivity to PVL-mediated formation of atypical NETs', 'authors': \"[{'name': 'Hina Jhelum'}, {'name': 'Dora erina'}, {'name': 'Christopher J Harbort'}, {'name': 'Andreas Lindner'}, {'name': 'Leif Gunnar Hanitsch'}, {'name': 'Rasmus Leistner'}, {'name': 'Jennyver-Tabea Schrder'}, {'name': 'Horst von Bernuth'}, {'name': 'Miriam Songa Stegemann'}, {'name': 'Mariana Schrmann'}, {'name': 'Arturo Zychlinsky'}, {'name': 'Renate Krger'}, {'name': 'Gerben Marsman'}]\", 'abstract': '<p>Panton-Valentine leukocidin (PVL) is a Staphylococcus aureus (S. aureus) toxin that binds to and kills human neutrophils, resulting in the formation of neutrophil extracellular traps (NETs). Some individuals colonized with PVL-positive S. aureus (PVL-SA) suffer from recurring infections whereas others are asymptomatically colonized. We found that neutrophils from affected patients express higher levels of CD45, one of the PVL receptors, and are more susceptible to killing at a low concentration of recombinant PVL than control neutrophils. We verified that PVL induces the formation of NETs and provide genetic and pharmacological evidence that PVL-induced NET formation is independent of NADPH-oxidase and reactive oxygen species (ROS) production. Through NET proteome analysis we identified that the protein content of PVL-induced NETs is different from NETs induced by mitogen or the microbial toxin nigericin. The abundance of the proteins cathelicidin (CAMP), elastase (NE), and proteinase 3 (PRTN3) was lower on PVL-induced NETs, which were inefficient in killing S. aureus. Neutrophils from patients that suffer from recurring PVL-positive infections may be more sensitive to PVL-induced NET formation, which may impair their ability to combat the infection.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.28.470255', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Functionalizing lipid sponge droplets with DNA', 'authors': \"[{'name': 'Christy Cho'}, {'name': 'Henrike Niederholtmeyer'}, {'name': 'Hyeonglim Seo'}, {'name': 'Ahanjit Bhattacharya'}, {'name': 'Neal K. Devaraj'}]\", 'abstract': '<p>Nucleic acids are among the most versatile molecules for the construction of biomimetic systems because they can serve as information carriers and programmable construction materials. How nucleic acids interact with membranous coacervate compartments such as lipid sponge droplets is not known. Here we systematically characterize the potential of DNA to functionalize lipid sponge droplets and demonstrate a strong size dependence for sequestration into the sponge phase. Double stranded DNA molecules of more than 300 bp are excluded and form a corona on the surface of droplets they are targeted to. Shorter DNA molecules partition efficiently into the lipid sponge phase and can direct DNA-templated reactions to droplets. We demonstrate repeated capture and release of labeled DNA strands by dynamic hybridization and strand displacement reactions that occur inside droplets. Our system opens new opportunities for DNA-encoded functions in lipid sponge droplets such as cargo control and signaling.</p>', 'posted': '2021-11-28'}, {'doi': '10.1101/2021.11.25.470057', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Experimental Investigation Of Pulse Sterilization Of Pathogens', 'authors': \"[{'name': 'Volodymyr I Chumakov'}, {'name': 'Mykhailo Ostryzhnyi'}, {'name': 'Oksana Kharchenko'}, {'name': 'Nataliya Rybalchenko'}, {'name': 'Vasiliy Muraveinyk'}, {'name': 'Aleksandr Tarasevich'}]\", 'abstract': '<p>The results of experimental investigations of the effect of high-intensity pulsed UV radiation on reference strains of opportunistic pathogens S, aureus and E. coli are presented. The modified pulse UV sterilizer -300-3 based on an end-face plasma accelerator was used as a radiation source, which provides a power pulsed discharge in an open atmosphere. The high efficiency of inactivation of the pathogens was provided within a short period of time. The possibility of providing urgent 100% sterilization of a pathogens has been shown. The prospects for the application of pulse sterilization technology to combat pathogens are considered.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.26.470055', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Pathology', 'covid': False, 'title': 'Bagaza virus in wild birds, Portugal, 2021', 'authors': \"[{'name': 'Joao Queiros'}, {'name': 'Silvia Barros'}, {'name': 'Alberto Sanchez-Cano'}, {'name': 'Margarida Henriques'}, {'name': 'Teresa Fagulha'}, {'name': 'Fabio Abade dos Santos'}, {'name': 'Margarida Duarte'}, {'name': 'Catarina Goncalves'}, {'name': 'David Goncalves'}, {'name': 'Marisa Rodrigues'}, {'name': 'Teresa Cardona Cabrera'}, {'name': 'Isabel G. Fernandez de Mera'}, {'name': 'Christian Gortazar'}, {'name': 'Ursula Hofle'}, {'name': 'Paulo Celio Alves'}]\", 'abstract': '<p>Bagaza virus (BAGV) emerged in Spain, in 2010. Since then, BAGV was not reported in other European countries until September 2021, when BAGV was diagnosed by molecular methods in one corn bunting and several red-legged partridges, after abnormal mortality in Southern Portugal. Sequencing revealed high similarity with strains reported previously.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.26.470101', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Simultaneous pathway engagement of translation in astrocytes, circadian rhythm in GABAergic neurons and cytoskeleton in glutamatergic neurons precede electroencephalographic changes in neurodegenerative prion disease', 'authors': \"[{'name': 'Lech Tadeusz Kaczmarczyk'}, {'name': 'Melvin Schleif'}, {'name': 'Lars Dittrich'}, {'name': 'Rhiannan Williams'}, {'name': 'Marusa Koderman'}, {'name': 'Vikas Bansal'}, {'name': 'Ashish Rajput'}, {'name': 'Theresa Schulte'}, {'name': 'Maria Jonson'}, {'name': 'Clemens Krost'}, {'name': 'Fabio Testaquadra'}, {'name': 'Stefan Bonn'}, {'name': 'Walker Scot Jackson'}]\", 'abstract': '<p>Selective vulnerability is an enigmatic feature of neurodegenerative diseases (NDs), whereby a widely expressed protein causes lesions in specific brain regions and cell types. This selectivity may arise from cells possessing varying capacities to regain proteostasis when stressed by cytotoxic protein conformers. Using the RiboTag method in mice, translational responses of five neural subtypes to acquired prion disease (PrD) were measured. Pre-onset and disease onset timepoints were chosen based on longitudinal electroencephalography (EEG) that revealed a gradual increase in theta power between 10- and 18-weeks after prion injection, resembling a clinical feature of human PrD. At disease onset, marked by significantly increased theta power and histopathological lesions, mice had pronounced translatome changes in all five cell types despite having a normal outward appearance. Remarkably, at a pre-onset stage, prior to EEG and neuropathological changes, we found that 1) translatomes of astrocytes indicated a sharply reduced synthesis of ribosomal and mitochondrial components, 2) excitatory neurons showed increased expression of cytoskeletal genes, and 3) inhibitory neurons revealed reduced expression of circadian rhythm network genes. Further assessment for the role of circadian rhythms using a jet lag paradigm modestly exacerbated disease. These data demonstrate that early translatome responses to neurodegeneration emerge prior to other signs of disease and are unique to different cell types. Therapeutic strategies may need to target multiple pathways, each in specific populations of cells, early in the disease process.</p>', 'posted': '2021-11-27'}, {'doi': '10.1101/2021.11.25.469847', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Centripetal nuclear shape fluctuations associate with chromatin condensation towards mitosis', 'authors': \"[{'name': 'Viola Introini'}, {'name': 'Gururaj Rao Kidiyoor'}, {'name': 'Giancarlo Porcella'}, {'name': 'Marco Foiani'}, {'name': 'Pietro Cicuta'}, {'name': 'Marco Cosentino Lagomarsino'}]\", 'abstract': '<p>The cell nucleus plays a central role in several key cellular processes, including chromosome organisation, replication and transcription. Recent work intriguingly suggests an association between nuclear mechanics and cell-cycle progression, but many aspects of this connection remain unexplored. Here, by monitoring nuclear shape fluctuations at different cell cycle stages, we uncover increasing inward fluctuations in late G2 and early mitosis, which are initially transient, but develop into instabilities that culminate into nuclear-envelope breakdown in mitosis. Perturbation experiments and correlation analysis reveal an association of these processes with chromatin condensation. We propose that the contrasting forces between an extensile stress and centripetal pulling from chromatin condensation could link mechanically chromosome condensation and nuclear- envelope breakdown, the two main nuclear processes during mitosis.</p><sec><title>Significance Statement</title><p>The nucleus was recently shown to exhibit shape fluctuations that vary with cell-cycle stage, but we know very little about the possible links between nuclear mechanics and cell cycle- progression. Through flickering analysis, this study monitors radius and nuclear envelope fluctuations across the cell cycle. The authors discover that as the cell cycle progresses towards mitosis, localised inward invaginations of the nuclear shape form initially transiently and gradually increasing their amplitude, in association with chromatin condensation. This phenomenon develops into nuclear envelope breakdown, suggesting a novel link between cell cycle, chromatin mechanics and nuclear shape fluctuations.</p></sec>', 'posted': '2021-11-25'}, {'doi': '10.1101/2021.11.23.469711', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Single virus assay reveals membrane determinants and mechanistic features of Sendai virus binding', 'authors': \"[{'name': 'Amy Lam'}, {'name': 'Orville O. Kirkland'}, {'name': 'Papa Freduah Anderson'}, {'name': 'Nandini Seetharaman'}, {'name': 'Dragan Vujovic'}, {'name': 'Patricia A. Thibault'}, {'name': 'Kristopher D. Azarm'}, {'name': 'Benhur Lee'}, {'name': 'Robert J. Rawle'}]\", 'abstract': '<p>Sendai virus (SeV, formally murine respirovirus) is a membrane-enveloped, negative sense RNA virus in the <italic>Paramyxoviridae</italic> family, and is closely related to human parainfluenza viruses. SeV has long been utilized as a model paramyxovirus and has recently gained attention as a viral vector candidate for both laboratory and clinical applications. To infect host cells, SeV must first bind to sialic-acid glycolipid or glycoprotein receptors on the host cell surface via its hemagglutinin-neuraminidase (HN) protein. Receptor binding induces a conformational change in HN, which allosterically triggers the viral fusion (F) protein to catalyze membrane fusion. While it is known that SeV binds to 2,3-linked sialic acid receptors, and there has been some study into the chemical requirements of those receptors, key mechanistic features of SeV binding remain unknown, in part because traditional approaches often convolve binding and fusion. Here, we develop and employ a fluorescence microscopy-based assay to observe SeV binding to supported lipid bilayers (SLBs) at the single particle level, which easily disentangles binding from fusion. Using this assay, we investigate mechanistic questions of SeV binding. We identify chemical structural features of ganglioside receptors that influence viral binding and demonstrate that binding is cooperative with respect to receptor density. We measure the characteristic decay time of unbinding and provide evidence supporting a rolling mechanism of viral mobility following receptor binding. We also study the dependence of binding on target cholesterol concentration. Interestingly, we find that while SeV binding shows striking parallels in cooperative binding with a prior report of Influenza A virus, it does not demonstrate a similar sensitivity to cholesterol concentration and receptor nano-cluster formation.</p><sec><title>STATEMENT OF SIGNIFICANCE</title><p>Paramyxoviruses are a family of membrane-enveloped viruses with many notable human and animal pathogens. In this study, we develop and use an assay to observe the initial step of infection  virus binding to the host membrane  for Sendai virus, the prototypical paramyxovirus, at the single virus level. This assay uses cell membrane mimics  supported lipid bilayers  as targets for virus binding to enable easy control of the membrane components with which the virus interacts. Using our assay, we gain insight into basic biophysical questions about Sendai virus binding, including the chemical characteristics of the receptor, the cooperative nature of binding, the influence of cholesterol, and the mechanism of viral mobility following binding.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.19.469291', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Beamforming seizures from the temporal lobe', 'authors': \"[{'name': 'Luis Garcia Dominguez'}, {'name': 'Apameh Tarazi'}, {'name': 'Taufik Valiante'}, {'name': 'Richard Wennberg'}]\", 'abstract': '<sec><title>Background</title><p>Surgical treatment of drug-resistant temporal lobe epilepsy (TLE) depends on proper identification of the seizure onset zone (SOZ), and differentiation of mesial, temporolimbic seizure onsets from temporal neocortical seizure onsets. Non-invasive source imaging using electroencephalography (EEG) and magnetoencephalography (MEG) can provide accurate information on interictal spike localization; however, EEG and MEG have low sensitivity for epileptiform activity restricted to deep temporolimbic structures. Moreover, in mesial temporal lobe epilepsy (MTLE), interictal spikes frequently arise in neocortical foci distant from the SOZ, rendering interictal spike localization potentially misleading for presurgical planning.</p></sec><sec><title>Methods</title><p>In this study, we used two different beamformer techniques applied to the MEG signal of ictal events acquired during EEG-MEG recordings in six patients with TLE (three neocortical, three MTLE). The ictal source localization results were compared to the patients ground truth SOZ localizations determined from intracranial EEG and/or clinical, neuroimaging and postsurgical outcome evidence.</p></sec><sec><title>Results</title><p>Beamformer analysis proved to be highly accurate in all cases and able to reliably identify focal seizure onsets localized to mesial, temporolimbic structures. In three patients, interictal spikes were either absent, too complex for inverse dipole modeling, or localized to anterolateral temporal neocortex distant to a mesial temporal SOZ.</p></sec><sec><title>Conclusions</title><p>This report demonstrates the suitability of MEG beamformer analysis of ictal events in TLE, which can supersede or complement the traditional analysis of interictal spikes. The method outlined is applicable to any type of epileptiform event, greatly expanding the information value of MEG and broadening its utility for presurgical recording in epilepsy.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469730', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Identification of biological processes and signaling pathways in the stretched nucleus pulposus cells', 'authors': \"[{'name': 'Min Zhang'}, {'name': 'Jing Wang'}, {'name': 'Hanming Gu'}]\", 'abstract': '<p>Low back pain is mostly caused by disc degeneration, which is due to the alterations in the osmotic pressure of nucleus pulposus cells. However, the knowledge about the mechanism and therapies for disc degeneration is not fully understood. Here, our objective is to identify significantly changed genes and biological processes in the stretched nucleus pulposus cells. The GSE175710 dataset was originally produced by using the Illumina HiSeq 4000 (Rattus norvegicus). The KEGG and GO analyses indicated that MAPK signaling, TNF signaling, IL17 signaling, and the NF-B signaling pathway are mostly affected in the stretched nucleus pulposus cells. Moreover, we identified several genes according to the PPI network such as Mmp9, Cxcl12, Col1a1, and Col3a1 in the stretched nucleus pulposus cells. Thus, our study provides further insights into the study of disc deterioration.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469700', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Zika virus-specific IgM elicited during pregnancy exhibits ultrapotent neutralization', 'authors': \"[{'name': 'Tulika Singh'}, {'name': 'Kwan-Ki Hwang'}, {'name': 'Andrew S. Miller'}, {'name': 'Rebecca L. Jones'}, {'name': 'Cesar A. Lopez'}, {'name': 'Camila Giuberti'}, {'name': 'Morgan A. Gladden'}, {'name': 'Itzayana Miller'}, {'name': 'Helen S. Webster'}, {'name': 'Joshua A. Eudailey'}, {'name': 'Kan Luo'}, {'name': 'Tarra Von Holle'}, {'name': 'Robert J. Edwards'}, {'name': 'Sarah Valencia'}, {'name': 'Katherine E. Burgomaster'}, {'name': 'Summer Zhang'}, {'name': 'Jesse F. Mangold'}, {'name': 'Joshua J. Tu'}, {'name': 'Maria Dennis'}, {'name': 'S. Munir Alam'}, {'name': 'Lakshmanane Premkumar'}, {'name': 'Reynaldo Dietze'}, {'name': 'Theodore C. Pierson'}, {'name': 'Eng Eong Ooi'}, {'name': 'Helen M. Lazear'}, {'name': 'Richard J. Kuhn'}, {'name': 'Sallie R. Permar'}, {'name': 'Mattia Bonsignori'}]\", 'abstract': '<p>Congenital Zika virus (ZIKV) infection results in neurodevelopmental deficits in up to 14% of infants born to ZIKV-infected mothers. Neutralizing antibodies are a critical component of protective immunity. Here, we demonstrate that plasma IgM responses contribute to ZIKV immunity in pregnancy, mediating neutralization up to three months post symptoms. From a ZIKV-infected pregnant woman, we established a B cell line secreting a pentameric ZIKV-specific IgM (DH1017.IgM) that exhibited ultrapotent ZIKV neutralization dependent on the IgM isotype. DH1017.IgM targets a novel envelope dimer epitope within Domain II. The arrangement of this epitope on the virion is compatible with concurrent engagement of all ten antigen-binding sites of DH1017.IgM, a solution not achievable by IgG antibodies. DH1017.IgM protected against lethal ZIKV challenge in mice. Our findings identify a unique role of antibodies of the IgM isotype in protection against ZIKV and posit DH1017.IgM as a safe and effective candidate immunoprophylactic, particularly during pregnancy.</p><sec><title>Key points</title><list list-type=\"bullet\"><list-item><p>Plasma IgM contributes to early ZIKV neutralization during pregnancy</p></list-item><list-item><p>Ultrapotent neutralization by pentameric DH1017.IgM mAb depends on isotype</p></list-item><list-item><p>DH1017.IgM can engage all binding sites concurrently through different angles of approach</p></list-item><list-item><p>DH1017.IgM protects mice against lethal ZIKV challenge</p></list-item></list></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469678', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'KEEPING IN TOUCH WITH OUR HIDDEN SIDE', 'authors': \"[{'name': 'Benjamin Mathieu'}, {'name': 'Antonin Abillama'}, {'name': 'Malvina Martinez'}, {'name': 'Laurence Mouchnino'}, {'name': 'Jean Blouin'}]\", 'abstract': '<p>Previous studies have shown that the sensory modality used to identify the position of proprioceptive targets hidden from sight, but frequently viewed, influences the type of the body representation employed for reaching them with the finger. The question then arises as to whether this observation also applies to proprioceptive targets which are hidden from sight, and rarely, if ever, viewed. We used an established technique for pinpointing the type of body representation used for the spatial encoding of targets which consisted of assessing the effect of peripheral gaze fixation on the pointing accuracy. More precisely, an exteroceptive, visually dependent, body representation is thought to be used if gaze deviation induces a deviation of the pointing movement. Three light-emitting diodes (LEDs) were positioned at the participants eye level at 25 deg, 0 deg and +25 deg with respect to the cyclopean eye. Without moving the head, the participant fixated the lit LED before the experimenter indicated one of the three target head positions: topmost point of the head (vertex) and two other points located at the front and back of the head. These targets were either verbal-cued or tactile-cued. The goal of the subjects (n=27) was to reach the target with their index finger. We analysed the accuracy of the movements directed to the topmost point of the head, which is a well-defined, yet out of view anatomical point. Based on the possibility of the brain to create visual representations of the body areas that remain out of view, we hypothesized that the position of the vertex is encoded using an exteroceptive body representation, both when verbally or tactile-cued. Results revealed that the pointing errors were biased in the opposite direction of gaze fixation for both verbal-cued and tactile-cued targets, suggesting the use of a vision-dependent exteroceptive body representation. The enhancement of the visual body representations by sensorimotor processes was suggested by the greater pointing accuracy when the vertex was identified by tactile stimulation compared to verbal instruction. Moreover, we found in a control condition that participants were more accurate in indicating the position of their own vertex than the vertex of other people. This result supports the idea that sensorimotor experiences increase the spatial resolution of the exteroceptive body representation. Together, our results suggest that the position of rarely viewed body parts are spatially encoded by an exteroceptive body representation and that non-visual sensorimotor processes are involved in the constructing of this representation.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469672', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Tom70-based transcriptional regulation of mitochondrial biogenesis and aging', 'authors': \"[{'name': 'Qingqing Liu'}, {'name': 'Catherine E. Chang'}, {'name': 'Alexandra C. Wooldredge'}, {'name': 'Benjamin Fong'}, {'name': 'Brian K. Kennedy'}, {'name': 'Chuankai Zhou'}]\", 'abstract': '<p>Mitochondrial biogenesis has two major steps: the transcriptional activation of nuclear genome-encoded mitochondrial proteins and the import of nascent mitochondrial proteins that are synthesized in the cytosol. These nascent mitochondrial proteins are aggregation-prone and can cause cytosolic proteostasis stress. The transcription factor-dependent transcriptional regulations and the TOM-TIM complex-dependent import of nascent mitochondrial proteins have been extensively studied. Yet, little is known regarding how these two steps of mitochondrial biogenesis coordinate with each other to avoid the cytosolic accumulation of these aggregation-prone nascent mitochondrial proteins. Here we show that in budding yeast, Tom70, a conserved receptor of the TOM complex, moonlights to regulate the transcriptional activity of mitochondrial proteins. Tom70s transcription regulatory role is conserved in <italic>Drosophila</italic>. The dual roles of Tom70 in both transcription/biogenesis and import of mitochondrial proteins allow the cells to accomplish mitochondrial biogenesis without compromising cytosolic proteostasis. The age-related reduction of Tom70, caused by reduced biogenesis and increased degradation of Tom70, is associated with the loss of mitochondrial membrane potential, mtDNA, and mitochondrial proteins. While loss of Tom70 accelerates aging and age-related mitochondrial defects, overexpressing TOM70 delays these mitochondrial dysfunctions and extends the replicative lifespan. Our results reveal unexpected roles of Tom70 in mitochondrial biogenesis and aging.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.21.21264092', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Distribution of confirmed with COVID-19 by age and gender in Mexico', 'authors': \"[{'name': 'Adalberto Maldonado'}, {'name': 'Marco Reyes'}]\", 'abstract': '<p>Social, economic, and cultural factors can influence the likelihood of exposure to the virus of each person in sanitary emergencies like those of COVID-19. In this sense, parallel to the biological vulnerability to infection with SARS-CoV-2, said factors determine a complementary vulnerability to infection. Hence, they can influence in gender and age distributions of those confirmed, which is not entirely comprehended yet. The effect that age and gender can have on total vulnerability in Mexico thus far is not understood. A better understanding of such dependence can help policy optimization and decision-making to future similar emergencies. We aim to study the age and gender distributions of those confirmed with COVID-19 in Mexico. We also investigate the vulnerability to the infection with SARS-CoV-2 depending on such features. Two different samples of the Mexican population are analyzed in this non-experimental study to compare each other and evaluate the association of the result of the COVID-19 test with gender and age. Data up until the beginning of the vaccination are considered. The percentage of confirmed males is higher than females in both samples, and most tested and confirmed are working-age. Age distributions are positively skewed, with the peak in [30,39] years, which disagrees with the distribution of the Mexican population. The data suggest that the vulnerability to infection weakly depends on gender and age. Males were identified as the most vulnerable gender, and the age group [70,79] showed a higher vulnerability to infection.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.21266534', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'Determination of NK cell immunophenotype and cytokine profile in patients with Rocky Mountain spotted fever', 'authors': \"[{'name': 'Carolina Maynez-Prieto'}, {'name': 'Jaime R. Adame-Gallegos'}, {'name': 'Moiss Ramrez-Lpez'}, {'name': 'Estefana Quintana-Mendas'}, {'name': 'Blanca E. Estrada-Aguirre'}, {'name': 'Maribel Baquera-Arteaga'}, {'name': 'Diana Manzanera-Ornelas'}, {'name': 'Enrique Guevara-Macas'}, {'name': 'Gerardo P. Espino-Sols'}]\", 'abstract': '<p>The intracellular pathogen <italic>Rickettsia rickettsii</italic> causes tick-borne spotted fever (also called Rocky Mountain spotted fever (RMSF) and is increasingly recognized as an emerging cause of febrile illness in Mexico. However, little is known about the early immune responses to infection. Four RMSF pediatric patients on acute phase and eight healthy controls from Chihuahua, Mexico were recruited. The natural killer cell (NK) immunophenotype and the cytokine profile in peripheral blood were characterized by flow cytometry. The population of cytotoxic NK cells expressing NKG2D was significantly decreased in patients on 3rd day of hospitalization compared to the first sampling on admission. Interleukins IL-6, IL-8, and IL-10 levels were significantly increased in patients upon admission compared to controls. This study shows that circulating NK cells are numerically decreased, while cytokines induce a pro-inflammatory process in patients.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.21266561', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Nephrology', 'covid': True, 'title': 'Reactogenicity of COVID-19 vaccine in hemodialysis patients: a single-center retrospective study', 'authors': \"[{'name': 'Gaetano Alfano'}, {'name': 'Niccol Morisi'}, {'name': 'Francesco Fontana'}, {'name': 'Roberta Scarmignan'}, {'name': 'Laura Tonelli'}, {'name': 'Camilla Ferri'}, {'name': 'Martina Montani'}, {'name': 'Silvia Giovanella'}, {'name': 'Giulia Ligabue'}, {'name': 'Giacomo Mori'}, {'name': 'Erica Franceschini'}, {'name': 'Giovanni Guaraldi'}, {'name': 'Gianni Cappelli'}, {'name': 'Riccardo Magistroni'}, {'name': 'Gabriele Donati'}]\", 'abstract': '<sec><title>Introduction</title><p>Some hemodialysis patients are reluctant to COVID-19 for the development of adverse events (AEs). The aim of this study was to verify the safety of mRNA-1273 vaccine in hemodialysis patients.</p></sec><sec><title>Methods</title><p>We conducted a retrospective analysis of in-center hemodialysis patients who underwent mRNA-1273 vaccine from March 1<sup>st</sup> to April 30<sup>th</sup>, 2021. All AEs occurring after the first and the second doses were collected and classified as local or systemic.</p></sec><sec><title>Results</title><p>Overall, 126 patients on chronic maintenance dialysis without a prior COVID-19 diagnosis were vaccinated with two doses of mRNA-1273 vaccine. Mean age was 68 (IQR, 54,7-76) years and 53.6% of patients were aged  65 years. During the observational period of 68 (IQR, 66-70) days, AEs occurred in 57.9% and 61.9% of patients after the first dose and second dose, respectively. The most common AEs were: injection-site pain (61.9%), erythema (4.8%), itching (4.8%), swelling (16.7%), axillary swelling/tenderness (2.4%), fever (17.5%) headache (7.9%), fatigue (23.8%), myalgia (17.5%), arthralgia (12.7%), dyspnoea (2.4%), nausea/vomiting (7.1%), diarrhoea (5.6%), shivers (4%) and vertigo (1.6%).</p><p>The rates of local AEs were similar after the first and second doses (P=0.8), whereas systemic AEs occurred more frequently after the second dose (P=0.001). Fever (P=0.03), fatigue (P=0.02) and nausea/vomiting (P=0.03) were significantly more frequent after the second dose of the vaccine. There were no age-related differences in the rate of AEs. Overall, vaccine-related AEs in hemodialysis patients seem to be lower than in the general population.</p></sec><sec><title>Conclusion</title><p>RNA-1273 vaccine was associated with the development of transient AEs after the first (57.9%) and second doses (61.9%) in patients on chronic maintenance hemodialysis. Systemic AEs were more common after the second dose. Overall, all AEs lasted for a few days, without any apparent sequelae.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.21266594', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': True, 'title': 'Sustained negative mental health outcomes among healthcare workers over the first year of the COVID-19 pandemic: a prospective cohort study', 'authors': \"[{'name': 'Roberto Mediavilla'}, {'name': 'Eduardo Fernndez-Jimnez'}, {'name': 'Irene Martinez-Morata'}, {'name': 'Fabiola Jaramillo'}, {'name': 'Jorge Andreo-Jover'}, {'name': 'Ins Morn-Snchez'}, {'name': 'Franco Mascayano'}, {'name': 'Berta Moreno-Kstner'}, {'name': 'Sergio Minu'}, {'name': 'Jos Luis Ayuso-Mateos'}, {'name': 'Richard A. Bryant'}, {'name': 'Mara-Fe Bravo-Ortiz'}, {'name': 'Gonzalo Martnez-Als'}, {'name': ' '}]\", 'abstract': '<sec><title>Objective</title><p>To characterize the evolution of healthcare workers mental health status over the 1-year period following the initial COVID-19 pandemic outbreak and to examine baseline characteristics associated with resolution or persistence of mental health problems over time.</p></sec><sec><title>Methods</title><p>We conducted an 8-month follow-up cohort study. Eligible participants were healthcare workers working in Spain. Baseline data were collected during the initial pandemic outbreak. Survey-based self-reported measures included COVID-19-related exposures, sociodemographic characteristics, and three mental health outcomes (psychological distress, depression symptoms, and posttraumatic stress disorder symptoms). We examined three longitudinal trajectories in mental health outcomes between baseline and follow-up assessments (namely <italic>asymptomatic/stable, recovering</italic>, and <italic>persistently symptomatic/worsening</italic>).</p></sec><sec><title>Results</title><p>We recruited 1,807 participants. Between baseline and follow-up assessments, the proportion of respondents screening positive for psychological distress and probable depression decreased, respectively, from 74% to 56% and from 28% to 21%. Two-thirds remained asymptomatic/stable in terms of depression symptoms and 56% remained symptomatic or worsened over time in terms of psychological distress.</p></sec><sec><title>Conclusions</title><p>Poor mental health outcomes among healthcare workers persisted over time. Occupational programs and mental health strategies should be put in place.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469778', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genome assembly of the JD17 soybean provides a new reference genome for Comparative genomics', 'authors': \"[{'name': 'Xinxin Yi'}, {'name': 'Jing Liu'}, {'name': 'Shengcai Chen'}, {'name': 'Hao Wu'}, {'name': 'Min Liu'}, {'name': 'Qing Xu'}, {'name': 'Lingshan Lei'}, {'name': 'Seunghee Lee'}, {'name': 'Bao Zhang'}, {'name': 'Dave Kudrna'}, {'name': 'Wei Fan'}, {'name': 'Rod A. Wing'}, {'name': 'Chunyan Yang'}, {'name': 'Mengchen Zhang'}, {'name': 'Jianwei Zhang'}, {'name': 'Xuelu Wang'}, {'name': 'Nansheng Chen'}]\", 'abstract': '<p>Cultivated soybean (<italic>Glycine max</italic>) is an important source for protein and oil. Many elite cultivars with different traits have been developed for different conditions. Each soybean strain has its own genetic diversity, and the availability of more high-quality soybean genomes can enhance comparative genomic analysis for identifying genetic underpinnings for its unique traits. In this study, we constructed a high-quality <italic>de novo</italic> assembly of an elite soybean cultivar Jidou 17 (JD17) with chromsome contiguity and high accuracy. We annotated 52,840 gene models and reconstructed 74,054 high-quality full-length transcripts. We performed a genome-wide comparative analysis based on the reference genome of JD17 with three published soybeans (WM82, ZH13 and W05), which identified five large inversions and two large translocations specific to JD17, 20,984 - 46,912 PAVs spanning 13.1 - 46.9 Mb in size, and 5 - 53 large PAV clusters larger than 500kb. 1,695,741 - 3,664,629 SNPs and 446,689 - 800,489 Indels were identified and annotated between JD17 and them. Symbiotic nitrogen fixation (SNF) genes were identified and the effects from these variants were further evaluated. It was found that the coding sequences of 9 nitrogen fixation-related genes were greatly affected. The high-quality genome assembly of JD17 can serve as a valuable reference for soybean functional genomics research.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469777', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Propagation of seminal toxins through binary expression gene drives can suppress polyandrous populations', 'authors': \"[{'name': 'Juan Hurtado'}, {'name': 'Santiago Revale'}, {'name': 'Luciano M Matzkin'}]\", 'abstract': '<p>Gene drives can be highly effective in controlling a target population by disrupting a female fertility gene. To spread across a population, these drives require that disrupted alleles be largely recessive so as not to impose too high of a fitness penalty. We argue that this restriction may be relaxed by using a double gene drive design to spread a split binary expression system. One drive carries a dominant lethal/toxic effector alone and the other a transactivator factor, without which the effector will not act. Only after the drives reach sufficiently high frequencies would individuals have the chance to inherit both system components and the effector be expressed. We explore through mathematical modeling the potential of this design to spread dominant lethal/toxic alleles and suppress populations. We show that this system could be implemented to spread engineered seminal proteins designed to kill females, making it highly effective against polyandrous populations.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469768', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'How increased cognitive load affects the dual-task cost in healthy young adults? A randomized, double-blind sham-controlled study', 'authors': \"[{'name': 'Shabnam Behrangrad'}, {'name': 'Maryam Zoghi'}, {'name': 'Farshad Mansouri'}, {'name': 'Shapour Jaberzadeh'}]\", 'abstract': '<p>Our ability to interact flexibly with the surrounding environment and achieve an adaptive goal-directed response is one of the necessities of balance control. This study aimed to examine the interaction between cognitive demand and the necessity for keeping balance in unstable conditions. We examined the effects of performing two cognitive tasks, namely the Stroop test and Wisconsin Card Sorting Test (WCST), on postural balance in healthy young adults. Stroop and the WCST test assess selective attention and cognitive flexibility in shifting between rules, respectively. Thirty-two healthy adults were included in two experimental conditions (control and treatment) in random order, separated by at least seven days. Standing balance was evaluated by the Sway Medical Mobile application in eyes open (EO) and eyes closed (EC) in different stance positions: feet apart, feet together, semi-tandem, tandem, and single-leg stance (SLS). Balance was evaluated before and after the cognitive test in each experimental condition. Our findings indicate that performing cognitively demanding tasks adversely affected the balance ability in more demanding balance tests such as the SLS with EC (P  0.05). However, no significant changes were seen in other balance tests (P  0.05). Additionally, no significant changes were seen in balance ability after the Stroop or Wisconsin card sorting test alone. These results confirm that performing cognitively demanding tasks significantly reduced the ability to keep balance in less stable conditions. These findings have significant implications in understanding and preventing falls and incidents resulting from an impaired balance in complex and cognitively demanding conditions.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469749', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Influence of selected plant growth stimulators in enhancing germinability and germination parameters of Zea mays L. under microgravity conditions simulated by a two-dimensional clinostat', 'authors': \"[{'name': 'Beckley Ikhajiagbe'}, {'name': 'Geoffrey O. Anoliefo'}, {'name': 'Alexander O. Orukpe'}, {'name': 'Saheed I. Musa'}]\", 'abstract': '<p>The earth has become increasingly overcrowded as a result of rapid urbanization and population growth, predicting that its carrying capacity could be overstretched. As a result, it is important to test the possibilities of growing plants under space exploration conditions, especially gravitational balance. Since microgravity impedes plant development, to what extent can plant growth stimulators reverse or enhance this trend? A total of 12 maize seeds were weighed and placed sideways in petri dish and inoculated with plant growth stimulators such as indole acetic acid (IAA), gibberellic acid (GA), and ascorbate (AA) and the clinorotated at different rates (0.5, 1.0 and 2.0 rpm), while control seeds were just placed on a table. Results showed that at 72 hrs, the maize seeds under microgravity showed reduced germination percentage with increasing clinorotation rates as against the control. But when stimulated with IAA, GA and AA, improved germination percentage was observed as against the control even under microgravity condition. The seedling dry weight, germination time and other germination parameters also showed similar improvements. Comparatively, the three growth stimulators showed no major variations in their ability to improve germination percentage under microgravitational impact. However, IAA showed more improvement on seedling vigor as against others, while GA showed more effect on the peak time and rate of germination. This research confirmed the possibilities of improving germinability of maize seeds under space exploration condition.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469737', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Plants species influence on rhizosphere microbial communities depends on N availability', 'authors': \"[{'name': 'Teal S. Potter'}, {'name': 'Brian L. Anacker'}, {'name': 'Amber C. Churchill'}, {'name': 'William D. Bowman'}]\", 'abstract': '<sec><title>Purpose</title><p>Plants and soil microbes both influence how ecosystems respond to environmental change. Yet, we lack the ability to generalize how plants and soil microbes influence each other in the same or varying soil conditions. This limitation thwarts ecologists ability to understand and predict effects of environmental changes such and elevated anthropogenic nitrogen (N) deposition. Accordingly, we examined the specificity of plant species influence on soil microbial community composition.</p></sec><sec><title>Methods</title><p>We tested (1) whether congeneric grass species have unique effects on soil microbial communities, (2) how relative abundances of microbial taxa can be explained by <italic>Poa</italic> phylogeny, plant traits, and range-wide traits (annual temperature and soil pH), and (3) whether N addition alters associations between <italic>Poa</italic> species and soil microbes, and (4) whether the magnitude of microbial community change in response to elevated N can be explained by plant growth responses to N. We conducted a greenhouse experiment with seven <italic>Poa</italic> species and native soils.</p></sec><sec><title>Results</title><p>We found that individual <italic>Poa</italic> species were associated with different soil fungi but similar soil bacteria. Differences in microbial composition were not attributable to <italic>Poa</italic> phylogeny, plant traits, or range-wide traits. Nitrogen addition enhanced the unique effects of <italic>Poa</italic> species on fungal and bacterial community compositions.</p></sec><sec><title>Conclusion</title><p>These results demonstrate how ecological interactions of related plant species vary depending on resource supply, revealing important context dependency for accurately predicting microbially-mediated nutrient cycling and ecosystem responses to changes in nutrient availability.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469734', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Quasi-universal scaling in mouse-brain neuronal activity stems from edge-of-instability critical dynamics', 'authors': \"[{'name': 'Guillermo B. Morales'}, {'name': 'Serena Di Santo'}, {'name': 'Miguel A. Muoz'}]\", 'abstract': '<p>The brain is in a state of perpetual reverberant neural activity, even in the absence of specific tasks or stimuli. Shedding light on the origin and functional significance of such activity is essential to understanding how the brain transmits, processes, and stores information. An inspiring, albeit controversial, conjecture proposes that some statistical characteristics of empirically observed neuronal activity can be understood by assuming that brain networks operate in a dynamical regime near the edge of a phase transition. Moreover, the resulting critical behavior, with its concomitant scale invariance, is assumed to carry crucial functional advantages. Here, we present a data-driven analysis based on simultaneous high-throughput recordings of the activity of thousands of individual neurons in various regions of the mouse brain. To analyze these data, we construct a unified theoretical framework that synergistically combines cutting-edge methods for the study of brain activity (such as a phenomenological renormalization group approach and techniques that infer the general dynamical state of a neural population), while designing complementary tools. This unified approach allows us to uncover strong signatures of scale invariance that is quasi-universal across brain regions and reveal that these areas operate, to a greater or lesser extent, at the edge of instability. Furthermore, this framework allows us to distinguish between quasi-universal background activity and non-universal input-related activity. Taken together, the following study provides strong evidence that brain networks actually operate in a critical regime which, among other functional advantages, provides them with a scale-invariant substrate of activity in which optimal input representations can be sustained.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469732', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Directed Evolution of Aptamer Discovery Technologies', 'authors': \"[{'name': 'Diana Wu'}, {'name': 'Chelsea K.L. Gordon'}, {'name': 'John H. Shin'}, {'name': 'Michael Eisenstein'}, {'name': 'H. Tom Soh'}]\", 'abstract': '<sec><title>Conspectus</title><p>Although antibodies are a powerful tool for molecular biology and clinical diagnostics, there are many emerging applications for which nucleic acid-based aptamers can be advantageous. However, generating high-quality aptamers with sufficient affinity and specificity for biomedical applications is a challenging feat for most research laboratories. In this <italic>Account</italic>, we describe four techniques developed in our lab to accelerate the discovery of high quality aptamer reagents that can achieve robust binding even for challenging molecular targets. The first method is particle display, in which we convert solution-phase aptamers into aptamer particles that can be screened via fluorescence-activated cell sorting (FACS) to quantitatively isolate individual aptamer particles based on their affinity. This enables the efficient isolation of high-affinity aptamers in fewer selection rounds than conventional methods, thereby minimizing selection biases and reducing the emergence of artifacts in the final aptamer pool. We subsequently developed the multi-parametric particle display (MPPD) method, which employs two-color FACS to isolate aptamer particles based on both affinity and specificity, yielding aptamers that exhibit excellent target binding even in complex matrices like serum. The third method is a click chemistry-based particle display (click-PD) that enables the generation and high-throughput screening of non-nattural aptamers with a wide range of base modifications. We have shown that these base-modified aptamers can achieve robust affinity and specificity for targets that have proven challenging or inaccessible with natural nucleotide-based aptamer libraries. Lastly, we describe the non-natural aptamer array (N2A2) platform, in which a modified benchtop sequencing instrument is used to characterize base-modified aptamers in a massively parallel fashion, enabling the efficient identification of molecules with excellent affinity and specificity for their targets. This system first generates aptamer clusters on the flow-cell surface that incorporate alkyne-modified nucleobases, and then performs a click reaction to couple those nucleobases to an azide-modified chemical moiety. This yields a sequence-defined array of tens of millions of base-modified sequences, which can then be characterized in a high-throughput fashion. Collectively, we believe that these advancements are helping to make aptamer technology more accessible, efficient, and robust, thereby enabling the use of these affinity reagents for a wider range of molecular recognition and detection-based applications.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.21.469426', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Towards environmental DNA-based bioassessment of freshwater reservoirs with small volumes of water: robust molecular protocols', 'authors': \"[{'name': 'Rebecca K. Loh'}, {'name': 'Sujatha N. Kutty'}, {'name': 'Darren C. J. Yeo'}, {'name': 'Rudolf Meier'}]\", 'abstract': '<p>Bioassessment of freshwater quality via eDNA is rapidly developing into a powerful alternative to traditional methods involving collecting, sorting, and identifying macroinvertebrates based on morphology. Particularly attractive would be methods that can use remote-controlled boats for sampling because it would allow for cost-effective, and frequent monitoring at multiple sites. The latter will be particularly important for tropical reservoirs that require year-around surveillance. We here optimize molecular protocols for capturing reservoir-specific differences in metazoan communities based on small water volumes (15 mL). The optimization is based on samples from two freshwater reservoirs with very different water qualities (reservoir signal). Each reservoir was sampled at three sites (biological replicates). For each water sample, the DNA was extracted twice (technical replicates). We then tested how much DNA template (0.1 ng to 15 ng) and how many PCR cycles (25 or 35) minimized variance between technical replicates. We find that 15 mL is sufficient for capturing the reservoir signal regardless of sampling time, template amounts, or PCR cycle numbers. Indeed, extrapolation from our results suggests that &lt;1 mL would be sufficient because only 17 of 59 metazoan mOTUs (mainly planktonic crustaceans and rotifers) detected with a 313bp COI minibarcode were shared. We find that the use of 35 PCR cycles significantly lowered the number of detected species and that template amounts &lt;0.5 ng yielded somewhat higher variance between technical replicates. Despite extensive trials, the variance between technical replicates remained high (Bray-Curtis: 520%; Jaccard: 1040%) and we predict that it will be difficult to reduce this variance further. However, the overall reservoir differences are so strong that all biological and technical replicates can be correctly assigned.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.21266785', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Rheumatology', 'covid': True, 'title': 'A RANDOMIZED CLINICAL TRIAL OF 2-WEEK METHOTREXATE DISCONTINUATION IN RHEUMATOID ARTHRITIS PATIENTS VACCINATED WITH INACTIVATED SARS-COV-2 VACCINE', 'authors': \"[{'name': 'Carlo S R Araujo'}, {'name': 'Ana C Medeiros-Ribeiro'}, {'name': 'Carla G S Saad'}, {'name': 'Karina R Bonfiglioli'}, {'name': 'Diogo S Domiciano'}, {'name': 'Andrea Y Shimabuco'}, {'name': 'Matheus R Silva'}, {'name': 'Emily F N Yuki'}, {'name': 'Sandra G Pasoto'}, {'name': 'Tatiana N Pedrosa'}, {'name': 'Leonard Kupa'}, {'name': 'Gioanna Zou'}, {'name': 'Rosa M R Pereira'}, {'name': 'Clovis A Silva'}, {'name': 'Nadia E Aikawa'}, {'name': 'Eloisa Bonfa'}]\", 'abstract': '<sec><title>Objectives</title><p>To evaluate the effect on immunogenicity and safety of 2-week methotrexate(MTX) discontinuation after each dose of the Sinovac-CoronaVac vaccine versus MTX maintenance in rheumatoid arthritis(RA) patients.</p></sec><sec><title>Methods</title><p>This was a single-center, prospective, randomized, investigator-blinded, intervention study (#<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04754698\">NCT04754698</ext-link>, CoronavRheum), including adult RA patients(stable CDAI&lt;10, prednisone&lt;7.5mg/day), randomized(1:1) to withdraw MTX(MTX-hold) for 2 weeks after each vaccine dose or maintain MTX(MTX-maintain), evaluated at D0, D28 and D69. Co-primary outcomes were anti-SARS-CoV-2 S1/S2 IgG seroconversion(SC) and neutralizing antibody(NAb) positivity at D69. Secondary outcomes were GMT and changes in disease activity scores. For immunogenicity analyses, we excluded patients with baseline positive IgG/NAb, and, for safety reasons, those unable to hold MTX twice(CDAI&gt;10 at D28).</p></sec><sec><title>Results</title><p>Randomization included 138 patients with 9 exclusions(5 COVID-19, 4 protocol violations). Safety evaluation included 60(MTX-hold) and 69(MTX-maintain) patients. Further exclusions consisted of 27 patients[13(21.7%) vs. 14(20.3%),p=0.848] with positive baseline IgG/NAb and 10 patients(21.3%) in MTX-hold with CDAI&gt;10 at D28. At D69, a higher increase in SC[29(78.4%) vs 30(54.5%),p=0.019] was observed in MTX-hold(n=37) in comparison to MTX-maintain(n=55), with parallel augmentation in GMT[34.2(25.2-46.4) <italic>vs</italic> 16.8(11.9-23.6),p=0.006]. No differences were observed for NAb positivity[23(62.2%) <italic>vs</italic> 27(49.1%),p=0.217]. Longitudinal variations in disease activity scores were alike in both groups(CDAI,p=0.144; DAS28-CRP,p=0.718).</p></sec><sec><title>Conclusion</title><p>We provided novel data that 2-week MTX withdrawal after each vaccine dose improves anti-SARS-CoV-2 immunogenicity. The comparable longitudinal variations of disease activity in both groups suggest that discontinuation is a feasible and efficient strategy in well-controlled RA patients, and may be even safer for vaccines with longer interval between doses or single dose schedules.</p></sec><sec><title>Funding</title><p>FAPESP/CNPq/B3-Bolsa de Valores-Brasil.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.21266755', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Genome sequencing in the Parkinsons disease clinic', 'authors': \"[{'name': 'Emily J. Hill'}, {'name': 'Laurie A. Robak'}, {'name': 'Rami Al-Ouran'}, {'name': 'Jennifer Deger'}, {'name': 'Jamie C. Fong'}, {'name': 'Paul Jerrod Vandeventer'}, {'name': 'Emily Schulman'}, {'name': 'Sindhu Rao'}, {'name': 'Hiba Saade'}, {'name': 'Rainer von Coelln'}, {'name': 'Harshavardhan Doddapaneni'}, {'name': 'Sejal Salvi'}, {'name': 'Shannon Dugan-Perez'}, {'name': 'Donna M. Muzny'}, {'name': 'Amy McGuire'}, {'name': 'Zhandong Liu'}, {'name': 'Richard Gibbs'}, {'name': 'Chad Shaw'}, {'name': 'Joseph Jankovic'}, {'name': 'Lisa M. Shulman'}, {'name': 'Joshua M. Shulman'}]\", 'abstract': '<sec><title>Background and Objectives</title><p>Genetic variants impact both Parkinsons disease (PD) risk and manifestations. While genetic information is of potential interest to patients and clinicians, genetic testing is rarely performed during routine PD clinical care. The goal of this study was to perform genome sequencing and examine patient interest in comprehensive genetic testing for PD in 2 academic movement disorder clinics.</p></sec><sec><title>Methods</title><p>In 208 subjects with PD (age=63 years, 67% male), genome sequencing was performed and filtered using a custom panel, including 49 genes associated with PD, parkinsonism, or related disorders, as well as a 90-variant PD genetic risk score. Separately, 231 patients (age=67 years, 63% male) were surveyed on interest in genetic testing at baseline and in response to vignettes covering (i) familial risk of PD (<italic>LRRK2</italic>); (ii) risk of PD dementia (<italic>GBA</italic>); (iii) PD genetic risk score; and (iv) secondary, medically-actionable variants (<italic>BRCA1</italic>).</p></sec><sec><title>Results</title><p>Genome sequencing revealed a <italic>LRRK2</italic> variant in 3.4% and a <italic>GBA</italic> risk variant in 10.1% of our clinical sample. The genetic risk score was normally distributed, identifying 42 subjects with high risk of PD. Medically-actionable findings were discovered in 2 subjects (1%). In our survey, the majority (82%) responded they would share a <italic>LRRK2</italic> variant with relatives. Most registered unchanged or increased interest in testing when confronted with potential risk for dementia or medically- actionable findings, and most (75%) expressed interest in learning their PD genetic risk score.</p></sec><sec><title>Discussion</title><p>Our results highlight broad interest in comprehensive genetic testing among patients with PD and may facilitate integration of genome sequencing in clinical practice.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.469413', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Perceived facial happiness during conversation correlates with insular and hypothalamus activity for humans, not robots: An investigation of the somatic marker hypothesis applied to social interactions', 'authors': \"[{'name': 'Thierry Chaminade'}, {'name': 'Nicolas Spatola'}]\", 'abstract': '<p>The somatic marker hypothesis posits that perceiving emotions entails reenacting markers of self emotions, in particular in the autonomous nervous system. Well studied in decision-making tasks, it has not been tested in a social cognitive neuroscience framework, and in particular for the automatic processing of positive emotions during natural interactions. Here, we address this question using a unique corpus of brain activity recorded during unconstrained conversations between participants and a human or a humanoid robot. fMRI recordings are used to test whether activity in the most important brain regions in relation to the autonomic system, the amygdala, hypothalamus and insula, is affected by the level of happiness expressed by the human and robot agents. Results indicate that for the hypothalamus and the insula, in particular the anterior agranular region strongly involved in processing social emotions, activity in the right hemisphere increases with the level of happiness expressed by the human, but not the robot. Results indicate that perceiving positive emotions in social interactions induces the local brain responses predicted by the somatic marker hypothesis, but only when the interacting agent is a real human.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469553', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Causal contributions of neural noise to left frontal networks subtending the modulation of conscious visual perception in the human brain', 'authors': \"[{'name': 'Chlo Stengel'}, {'name': 'Juli L. Amengual'}, {'name': 'Tristan Moreau'}, {'name': 'Antoni Valero-Cabr'}]\", 'abstract': '<p>For several decades, the field of human neurophysiology has focused on the role played by cortical oscillations in enabling brain function underpinning behaviors. In parallel, a less visible but robust body of work on the stochastic resonance phenomenon has also theorized contributions of neural noise  hence more heterogeneous, complex and less predictable activity  in brain coding. The latter notion has received indirect causal support via improvements of visual function during non-regular or random brain stimulation patterns. Nonetheless, direct evidence demonstrating an impact of brain stimulation on direct measures of neural noise is still lacking. Here we evaluated the impact of three <italic>non frequency-specific</italic> TMS bursts, compared to a control pure high-beta TMS rhythm, delivered to the left FEF during a visual detection task, on the heterogeneity, predictability and complexity of ongoing brain activity recorded with scalp EEG. Our data showed surprisingly that the three <italic>non frequency-specific</italic> TMS patterns did not prevent a build-up of local high-beta activity. Nonetheless, they increased power across broader or in multiple frequency bands compared to control purely <italic>rhythmic</italic> high-beta bursts tested along. Importantly, <italic>non frequency-specific</italic> patterns enhanced signal entropy over multiple time-scales, suggesting higher complexity and an overall induction of higher levels of cortical noise than <italic>rhythmic</italic> TMS bursts. Our outcomes provide indirect evidence on a potential modulatory role played by sources of stochastic noise on brain oscillations and synchronization. Additionally, they pave the way towards the development of novel neurostimulation approaches to manipulate cortical sources of noise and further investigate their causal role in neural coding.</p>', 'posted': '2021-11-22'}, {'doi': '10.1101/2021.11.22.469555', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Characterizing the short-latency evoked response to intracortical microstimulation across a multi-electrode array', 'authors': \"[{'name': 'Joseph Sombeck'}, {'name': 'Juliet Heye'}, {'name': 'Karthik Kumaravelu'}, {'name': 'Stefan M. Goetz'}, {'name': 'Angel V. Peterchev'}, {'name': 'Warren M. Grill'}, {'name': 'Sliman Bensmaia'}, {'name': 'Lee E. Miller'}]\", 'abstract': '<sec><title>Objective</title><p>Persons with tetraplegia can use brain-machine interfaces to make visually guided reaches with robotic arms. Without somatosensory feedback, these movements will likely be slow and imprecise, like those of persons who retain movement but have lost proprioception. Intracortical microstimulation (ICMS) has promise for providing artificial somatosensory feedback. If ICMS can mimic naturally occurring neural activity, afferent interfaces may be more informative and easier to learn than interfaces that evoke unnaturalistic activity. To develop such biomimetic stimulation patterns, it is important to characterize the responses of neurons to ICMS.</p></sec><sec><title>Approach</title><p>Using a Utah multi-electrode array, we recorded activity evoked by single pulses, and short (~0.2 s) and long (~4 s) trains of ICMS at a wide range of amplitudes and frequencies. As the electrical artifact caused by ICMS typically prevents recording for many milliseconds, we deployed a custom rapid-recovery amplifier with nonlinear gain to limit signal saturation on the stimulated electrode. Across all electrodes after stimulation, we removed the remaining slow return to baseline with acausal high-pass filtering of time-reversed recordings. With these techniques, we could record ~0.7 ms after stimulation offset even on the stimulated electrode.</p></sec><sec><title>Main results</title><p>We recorded likely transsynaptically-evoked activity as early as ~0.7 ms after single pulses of stimulation that was immediately followed by suppressed neural activity lasting 10150 ms. Instead of this long-lasting inhibition, neurons increased their firing rates for ~100 ms after trains. During long trains, the evoked response on the stimulated electrode decayed rapidly while the response was maintained on non-stimulated channels.</p></sec><sec><title>Significance</title><p>The detailed description of the spatial and temporal response to ICMS can be used to better interpret results from experiments that probe circuit connectivity or function of cortical areas. These results can also contribute to the design of stimulation patterns to improve afferent interfaces for artificial sensory feedback.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469644', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Comparative analysis of Bacillus cereus group isolates resistance using disk diffusion and broth microdilution and the correlation between resistance phenotypes and genotypes', 'authors': \"[{'name': 'Emma Mills'}, {'name': 'Erin Sullivan'}, {'name': 'Jasna Kovac'}]\", 'abstract': '<p>A collection of 85 <italic>Bacillus cereus</italic> group isolates were screened for phenotypic resistance to nine antibiotics using disk diffusion and broth microdilution. The broth microdilution antimicrobial results were interpreted using the CLSI M45 breakpoints for <italic>Bacillus</italic> spp. Due to the lack of <italic>Bacillus</italic> spp. disk diffusion breakpoints, the results obtained with the disk diffusion assay were interpreted using the CLSI M100 breakpoints for <italic>Staphylococcus</italic> spp. We identified significant (p &lt; 0.05) discrepancies in resistance interpretation between the two methods for ampicillin, gentamicin, rifampicin, tetracycline, and trimethoprim/sulfamethoxazole. Antimicrobial resistance genes were detected using unassembled and assembled whole-genome sequences with Ariba and Abricate, respectively, to assess the sensitivity and specificity for predicting phenotypic resistance based on the presence of antimicrobial resistance genes. We found antimicrobial resistance gene presence to be a poor indicator for phenotypic resistance, calling for further investigation of mechanisms underlying antimicrobial resistance in the <italic>B. cereus</italic> group. Genes with poor sensitivity and/or specificity, as determined based on broth microdilution results included <italic>rph</italic> (rifampicin, 0%, 95%), <italic>mph</italic> genes (erythromycin, 0%, 96%), and all <italic>van</italic> genes (vancomycin, 100%, 35%). However, <italic>Bc</italic> (ampicillin, 64%, 100%) and <italic>tet</italic> genes (tetracycline, 67%, 100%) were highly specific, albeit moderately sensitive indicators of phenotypic resistance based on broth microdilution results. Only beta-lactam resistance genes (<italic>Bc, BcII</italic>, and <italic>blaTEM</italic>) were highly sensitive (94%) and specific (100%) markers of resistance to ceftriaxone based on the disk diffusion results, providing further evidence of these beta-lactams role in nonsusceptibility of <italic>Bacillus cereus</italic> group isolates to ceftriaxone.</p><sec><title>IMPORTANCE</title><p><italic>Bacillus cereus</italic> group includes human pathogens that can cause severe infections requiring antibiotic treatment. Screening of environmental and food isolates for antimicrobial resistance can provide insight into what antibiotics may be more effective therapeutic options based on the lower prevalence of resistance. Currently, interpretation of antimicrobial susceptibility testing results using the disk diffusion method is complicated by the fact that there are no standard disk diffusion resistance breakpoints defined for <italic>Bacillus</italic> spp. Hence, the breakpoints for <italic>Staphylococcus</italic> are often used in research studies. By comparing the results of disk diffusion interpreted using the <italic>Staphylococcus</italic> spp. breakpoints against broth microdilution interpreted using <italic>Bacillus</italic> spp. breakpoints, this study demonstrated that disk diffusion results interpretation with <italic>Staphylococcus</italic> spp. breakpoints are inconsistent. This study also provides new insight into the poor associations between antimicrobial resistance genotypes and phenotypes for the <italic>B. cereus</italic> group.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469643', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Spatial metabolomics reveals localized impact of influenza virus infection on the lung tissue metabolome', 'authors': \"[{'name': 'Danya A. Dean'}, {'name': 'London Klechka'}, {'name': 'Ekram Hossain'}, {'name': 'Adwaita R. Parab'}, {'name': 'Krystin Eaton'}, {'name': 'Myron Hinsdale'}, {'name': 'Laura-Isobel McCall'}]\", 'abstract': '<p>The influenza virus (IAV) is a major cause of respiratory disease, with significant infection increases in pandemic years. Vaccines are a mainstay of IAV prevention, but are complicated by consideration of IAVs vast strain diversity, manufacturing and vaccine uptake limitations. While antivirals may be used for treatment of IAV, they are most effective in early stages of the infection and several virus strains have become drug resistant. Therefore, there is a need for advances in IAV treatment, especially host-directed, personalized therapeutics.Given the spatial dynamics of IAV infection and the relationship between viral spatial distribution and disease severity, a spatial approach is necessary to expand our understanding of IAV pathogenesis. We used spatial metabolomics to address this issue. Spatial metabolomics combines liquid chromatography-tandem mass spectrometry of metabolites extracted from systematic organ sections, 3D models and computational techniques, to develop spatial models of metabolite location and their role in organ function and disease pathogenesis. In this project, we analyzed plasma and systematically sectioned lung tissue samples from uninfected or infected mice. Spatial mapping of sites of metabolic perturbations revealed significantly lower metabolic perturbation in the trachea compared to other lung tissue sites. Using random forest machine learning, we identified metabolites that responded differently in each lung position based on infection, including specific amino acids, lipids and lipid-like molecules, and nucleosides. These results support the implementation of spatial metabolomics to understand metabolic changes upon IAV infection and to identify candidate pathways to be targeted for IAV therapeutics.</p><sec><title>Importance</title><p>The influenza virus is a major health concern. Over 1 billion people become infected annually despite the wide distribution of vaccines, and antiviral agents are insufficient to address current clinical needs. In this study, we used spatial metabolomics to understand changes in the lung and plasma metabolome of mice infected with influenza A virus, compared to uninfected controls. We determined metabolites altered by infection in specific lung tissue sites and distinguished metabolites perturbed by infection between lung tissue and plasma samples. Our findings highlight the importance of a spatial approach to understanding the intersection between lung metabolome, viral infection and disease severity. Ultimately, this approach will expand our understanding of respiratory disease pathogenesis and guide the development of novel host-directed therapeutics.</p></sec>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469634', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Discovering Novel Antimicrobial Peptides in Generative Adversarial Network', 'authors': \"[{'name': 'Tzu-Tang Lin'}, {'name': 'Li-Yen Yang'}, {'name': 'Ching-Tien Wang'}, {'name': 'Ga-Wen Lai'}, {'name': 'Chi-Fong Ko'}, {'name': 'Yang-Hsin Shih'}, {'name': 'Shu-Hwa Chen'}, {'name': 'Chung-Yen Lin'}]\", 'abstract': '<p>Due to the growing number of clinical antibiotic resistance cases in recent years, novel antimicrobial peptides (AMPs) can become ideal for next-generation antibiotics. This study trained a deep convolutional generative adversarial network (GAN) with known AMPs to generate novel AMP candidates. The quality of the GAN-designed peptides was evaluated <italic>in silico</italic>, and eight of them named GAN-pep 18 were chosen to be synthesized for further experiments. Disk diffusion testing and minimum inhibitory concentration (MIC) determination were used to determine the antibacterial effects of the synthesized GAN-designed peptides. Seven out of the eight synthesized GAN-designed peptides showed antibacterial activities.</p><p>Additionally, GAN-pep 3 and GAN-pep 8 had a broad spectrum of antibacterial effects. Both of them were also effective against antibiotic-resistant bacteria strains such as methicillin-resistant <italic>Staphylococcus aureus</italic> (<italic>S. aureus</italic>) and carbapenem-resistant <italic>Pseudomonas aeruginosa</italic> (<italic>P. aeruginosa</italic>). GAN-pep 3, the most promising GAN-designed peptide candidate, had low MICs against all the tested bacteria.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.469479', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Early age-related atrophy of cutaneous lymph nodes precipitates an early functional decline in skin immunity in mice with aging', 'authors': \"[{'name': 'Sandip Ashok Sonar'}, {'name': 'Jennifer L. Uhrlaub'}, {'name': 'Christopher P. Coplen'}, {'name': 'Gregory D. Sempowski'}, {'name': 'Jarrod A. Dudakov'}, {'name': 'Marcel R.M. van den Brink'}, {'name': 'Bonnie J. LaFleur'}, {'name': 'Mladen Jergovic'}, {'name': 'Janko Nikolich-ugich'}]\", 'abstract': '<p>Secondary lymphoid organs (SLO; including the spleen and lymph nodes) are critical both for the maintenance of nave T (T<sub>N</sub>) lymphocytes and for the initiation and coordination of immune responses. How they age, including the exact timing, extent, physiological relevance, and the nature of age-related changes, remains incompletely understood. We used time-stamping to indelibly mark cohorts of newly generated nave T cells (a.k.a. recent thymic emigrants - RTE) in mice, and followed their presence, phenotype and retention in SLO. We found that SLO involute asynchronously. Skin-draining lymph nodes (LN) atrophied early (6-9 months) in life and deeper tissue-draining LN and the spleen late (18-20 months), as measured by the loss of both T<sub>N</sub> numbers and the fibroblastic reticular cell (FRC) network. Time-stamped RTE cohorts of all ages entered SLO and successfully completed post-thymic differentiation. However, in older mice, these cells were poorly retained, and those found in SLO exhibited an emigration phenotype (CCR7<sup>lo</sup>S1P1<sup>hi</sup>). Transfers of adult RTE into recipients of different ages formally demonstrated that the defect segregates with the age of the SLO microenvironment and not with the age of T cells. Finally, upon intradermal immunization, RTE generated in mice as early as 6-7 months of age barely participated in <italic>de novo</italic> immune responses and failed to produce well-armed effector cells. These results highlight changes in structure and function of superficial secondary lymphoid organs in laboratory mice that are earlier than expected and are consistent with the long-appreciated and pronounced reduction of cutaneous immunity with aging.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469664', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Matrix-assisted laser desorption ionization time-of-flight mass spectrometry identification of methanogens of clinical interest', 'authors': \"[{'name': 'Cheick Oumar Guindo'}, {'name': 'Lynda Amir'}, {'name': 'Carine Couderc'}, {'name': 'Michel Drancourt'}, {'name': 'Ghiles Grine'}]\", 'abstract': '<p>Methanogens, the archaea uniquely detoxifying fermentative hydrogen into methane in the digestive tract, are increasingly detected in pathology situations, rendering their rapid identification mandatory. We improved the experimental protocol to identify broth-cultured methanogens by matrix-assisted laser desorption time-of-flight mass spectrometry (MALDI- TOF-MS). A database incorporating 34 reference spectra derived from 16 methanogen reference strains representative of eight species, supported further identification of 21 <italic>Methanobrevibacter smithii</italic> and 14 <italic>Methanobrevibacter oralis</italic> isolates broth-cultured from human stool and oral fluid, respectively, with scores &gt; 2. In addition, MALDI-TOF-MS differentiated five <italic>Methanobrevibacter smithii</italic> genotypes incorporated in the study. Data here reported found MALDI-TOF-MS as a first line identification method for methanogens recovered from microbiota and clinical samples.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.22.21265452', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Obstetrics and Gynecology', 'covid': False, 'title': 'Deep learning to estimate gestational age from blind ultrasound sweeps of the gravid abdomen', 'authors': \"[{'name': 'Teeranan Pokaprakarn'}, {'name': 'Juan C. Prieto'}, {'name': 'Joan T. Price'}, {'name': 'Margaret P. Kasaro'}, {'name': 'Ntazana Sindano'}, {'name': 'Hina R. Shah'}, {'name': 'Marc Peterson'}, {'name': 'Mutinta M. Akapelwa'}, {'name': 'Filson M. Kapilya'}, {'name': 'Yuri V. Sebastio'}, {'name': 'William Goodnight'}, {'name': 'Elizabeth M. Stringer'}, {'name': 'Bethany L. Freeman'}, {'name': 'Lina M. Montoya'}, {'name': 'Benjamin H. Chi'}, {'name': 'Dwight J. Rouse'}, {'name': 'Stephen R. Cole'}, {'name': 'Bellington Vwalika'}, {'name': 'Michael R. Kosorok'}, {'name': 'Jeffrey S. A. Stringer'}]\", 'abstract': '<sec><title>Background</title><p>Ultrasound is indispensable to gestational age estimation, and thus to quality obstetric care, yet high equipment cost and need for trained sonographers limit its use in low-resource settings.</p></sec><sec><title>Methods</title><p>From September 2018 through June 2021, we recruited 4,695 pregnant volunteers in North Carolina and Zambia and obtained blind ultrasound sweeps (cineloops) of the gravid abdomen alongside standard fetal biometry. We trained a neural network to estimate gestational age from the sweeps and, in three test sets, assessed performance of the model and biometry against previously established gestational age.</p></sec><sec><title>Results</title><p>In our main test set, model mean absolute error (MAE) was 3.9 days (standard error [SE] 0.12) vs. 4.7 days (SE 0.15) for biometry (difference -0.8 days; 95% CI -1.1, -0.5; p&lt;0.001). Results were similar in North Carolina (difference -0.6 days, 95% CI -0.9, -0.2) and Zambia (1.0 days, 95% CI -1.5, -0.5). Findings were supported in the test set of women who conceived by in vitro fertilization (model MAE 2.8 days [SE 0.28] vs. 3.6 days [SE 0.53] for biometry; difference -0.8 days, 95% CI -1.7, 0.2), and in the set of women from whom sweeps were collected by untrained users with low-cost, battery-powered devices (model MAE 4.9 days [SE 0.29] vs. 5.4 days [SE 0.28] for biometry; difference -0.6, 95% CI -1.3, 0.1).</p></sec><sec><title>Conclusions</title><p>Our model estimated gestational age more accurately from blindly obtained ultrasound sweeps than did trained sonographers performing fetal biometry. These results presage a future where all pregnant people  not just those in rich countries  can access the diagnostic benefits of sonography.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469621', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Age Differences in Diffusivity in the Locus Coeruleus and its Ascending Noradrenergic Tract', 'authors': \"[{'name': 'Shai Porat'}, {'name': 'Francesca Sibilia'}, {'name': 'Josephine Yoon'}, {'name': 'Yonggang Shi'}, {'name': 'Martin J. Dahl'}, {'name': 'Markus Werkle-Bergner'}, {'name': 'Sandra Dzel'}, {'name': 'Nils Bodammer'}, {'name': 'Ulman Lindenberger'}, {'name': 'Simone Khn'}, {'name': 'Mara Mather'}]\", 'abstract': '<p>The noradrenergic locus coeruleus (LC) is a small brainstem nucleus that promotes arousal and attention. Recent studies have examined the microstructural properties of the LC using diffusion-weighted magnetic resonance imaging and found unexpected age-related differences in fractional anisotropy - a measure of white matter integrity. Here, we used three datasets (Berlin Aging Study-II, N = 301, the Leipzig Study for Mind-Body-Emotion Interactions, N = 220, and Stockholm Sleepy Brain, N = 49), to replicate published findings and expand them by investigating diffusivity in the LCs ascending noradrenergic bundle. In younger adults, LC fractional anisotropy was significantly lower, compared to older adults. However, in the LCs ascending noradrenergic bundle, we observed significantly higher fractional anisotropy in younger adults, relative to older adults. These findings indicate that diffusivity in the LC versus the ascending noradrenergic bundle are both susceptible to microstructural changes in aging that have opposing effects on fractional anisotropy.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Fractional anisotropy in the locus coeruleus was lower in younger adults</p></list-item><list-item><p>Fractional anisotropy in the noradrenergic bundle was higher in younger adults</p></list-item><list-item><p>Sleep deprivation may affect diffusivity in younger adults more than older adults</p></list-item></list></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266559', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Factors associated with COVID-19 vaccine acceptance and hesitancy among residents of Northern California jails', 'authors': \"[{'name': 'Yiran E Liu'}, {'name': 'Jillian Oto'}, {'name': 'John Will'}, {'name': 'Christopher LeBoa'}, {'name': 'Alexis Doyle'}, {'name': 'Neil Rens'}, {'name': 'Shelley Aggarwal'}, {'name': 'Iryna Kalish'}, {'name': 'Marcela Rodriguez'}, {'name': 'Beruk Sherif'}, {'name': 'Chrisele Trinidad'}, {'name': 'Michael Del Rosario'}, {'name': 'Sophie Allen'}, {'name': 'Robert Spencer'}, {'name': 'Carlos Morales'}, {'name': 'Alexander Chyorny'}, {'name': 'Jason R Andrews'}]\", 'abstract': '<sec><title>Background</title><p>Carceral facilities are high-risk settings for COVID-19 transmission. Understanding of factors associated with COVID-19 vaccine acceptance and hesitancy among incarcerated individuals is incomplete, especially for people living in jails.</p></sec><sec><title>Methods</title><p>We conducted a retrospective review of COVID-19 vaccination data from the electronic health record (EHR) of residents in two Northern California county jails to examine factors associated with vaccine uptake in this population. We additionally administered a survey in four jails to assess reasons for vaccine hesitancy, sources of COVID-19 information, and medical mistrust. We performed multivariate logistic regression to determine associations with vaccine uptake or hesitancy.</p></sec><sec><title>Results</title><p>Of 2,584 jail residents offered a COVID-19 vaccine between March 19, 2021 and June 30, 2021, 1,464 (56.7%) accepted at least one dose. Among vaccinated residents, 538 (36.7%) initially refused the vaccine. Vaccine uptake was higher among older individuals, women, those with recent flu vaccination, and those living in shared cells or open dorms. Leading reasons for vaccine hesitancy included concerns around side effects and suboptimal efficacy. Television and friends/family were the most commonly cited and the most trusted sources of COVID-19 information, respectively. Vaccine acceptance was associated with increased trust in COVID-19 information sources and in medical personnel both in and out of jail.</p></sec><sec><title>Conclusion</title><p>Ongoing evidence-based COVID-19 vaccination efforts are needed in high-risk carceral settings. Effective interventions to improve vaccination rates in this population should utilize accessible and trusted sources of information to address concerns about vaccine side effects and efficacy and foster medical trust.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266673', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Neutralization of SARS-CoV-2 variants by rVSV-G-spike-elicited human sera', 'authors': \"[{'name': 'Yfat Yahalom-Ronen'}, {'name': 'Noam Erez'}, {'name': 'Morly Fisher'}, {'name': 'Hadas Tamir'}, {'name': 'Boaz Politi'}, {'name': 'Hagit Achdout'}, {'name': 'Sharon Melamed'}, {'name': 'Itai Glinert'}, {'name': 'Shay Weiss'}, {'name': 'Inbar Cohen-Gihon'}, {'name': 'Ofir Israeli'}, {'name': 'Marina Izak'}, {'name': 'Michal Mandelboim'}, {'name': 'Yoseph Caraco'}, {'name': 'Noa Madar-Balakirski'}, {'name': 'Adva Mechaly'}, {'name': 'Eilat Shinar'}, {'name': 'Daniel Cohen'}, {'name': 'Adi Beth-Din'}, {'name': 'Anat Zvi'}, {'name': 'Hadar Marcus'}, {'name': 'Tomer Israely'}, {'name': 'Nir Paran'}]\", 'abstract': '<p>BriLife (rVSV-G-spike) is a SARS-CoV-2 vaccine candidate based on vesicular stomatitis virus (VSV) platform. We show that sera from BriLife vaccinees maintain neutralization capacity against alpha, beta, gamma and delta SARS-CoV-2 variants. BriLife spontaneously-acquired spike mutations, corresponding with key SARS-CoV-2 variants mutations, may contribute to its efficacy against SARS-CoV-2 variants.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.23.469540', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Interferometric imaging of thermal expansion for temperature control in retinal laser therapy', 'authors': \"[{'name': 'David Veysset'}, {'name': 'Tong Ling'}, {'name': 'Yueming Zhuo'}, {'name': 'Daniel Palanker'}]\", 'abstract': '<p>Precise control of the temperature rise is a prerequisite for proper photothermal therapy. In retinal laser therapy, the heat deposition is primarily governed by the melanin concentration, which can significantly vary across the retina and from patient to patient. In this work, we present a method for determining the optical and thermal properties of layered materials, directly applicable to the retina, using low-energy laser heating and phase-resolved optical coherence tomography (pOCT). The method is demonstrated on a polymer-based tissue phantom heated with a laser pulse focused onto an absorbing layer buried below the phantoms surface. Using a line-scan spectral-domain pOCT, optical path length changes induced by the thermal expansion were extracted from sequential B-scans. The material properties were then determined by matching the optical path length changes to a thermo-mechanical model developed for fast computation. This method determined the absorption coefficient with a precision of 2.5% and the temperature rise with a precision of about 0.2C from a single laser exposure, while the peak did not exceed 8C during 1 ms pulse, which is well within the tissue safety range and significantly more precise than other methods.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266683', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Final sizes and durations of new COVID-19 pandemic waves in Ukraine and around the world predicted by generalized SIR model', 'authors': \"[{'name': 'Igor Nesteruk'}]\", 'abstract': '<p>New waves of the COVID-19 pandemic in Ukraine, which began in the summer of 2021, and after holidays in the middle of October 2021, were characterized by almost exponential growth of smoothed daily numbers of new cases. This is a matter of great concern and the need to immediately predict the epidemic dynamics in order to assess the possible maximum values of new cases, the risk of infection and the number of deaths. The generalized SIR-model and corresponding parameter identification procedure was used to simulate and predict the dynamics of two new epidemic waves in Ukraine and one in the whole world. Results of calculations show that new cases in Ukraine will not stop to appear before November 2022. If the global situation with vaccination, testing and treatment will not change, the pandemic could continue for another ten years.</p>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.22.21266721', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Simulating the impact of vaccination rates on the initial stages of a COVID-19 outbreak in New Zealand (Aotearoa) with a stochastic model', 'authors': \"[{'name': 'Leighton M. Watson'}]\", 'abstract': '<sec><title>Aim</title><p>The August 2021 COVID-19 outbreak in Auckland has caused the New Zealand government to transition from an elimination strategy to suppression, which relies heavily on high vaccination rates in the population. As restrictions are eased and as COVID-19 leaks through the Auckland boundary, there is a need to understand how different levels of vaccination will impact the initial stages of COVID-19 outbreaks that are seeded around the country.</p></sec><sec><title>Method</title><p>A stochastic branching process model is used to simulate the initial spread of a COVID-19 outbreak for different vaccination rates.</p></sec><sec><title>Results</title><p>High vaccination rates are effective at minimizing the number of infections and hospitalizations. Increasing vaccination rates from 20% (approximate value at the start of the August 2021 outbreak) to 80% (approximate proposed target) of the total population can reduce the median number of infections that occur within the first four weeks of an outbreak from 1011 to 14 (25<sup>th</sup> and 75<sup>th</sup> quantiles of 545-1602 and 2-32 for V=20% and V=80%, respectively). As the vaccination rate increases, the number of breakthrough infections (infections in fully vaccinated individuals) and hospitalizations of vaccinated individuals increases. Unvaccinated individuals, however, are 3.3x more likely to be infected with COVID-19 and 25x more likely to be hospitalized.</p></sec><sec><title>Conclusion</title><p>This work demonstrates the importance of vaccination in protecting individuals from COVID-19, preventing high caseloads, and minimizing the number of hospitalizations and hence limiting the pressure on the healthcare system.</p></sec>', 'posted': '2021-11-24'}, {'doi': '10.1101/2021.11.24.469808', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Zoology', 'covid': False, 'title': 'Viral dynamics and immune responses to foot-and-mouth disease virus in African buffalo (Syncerus caffer).', 'authors': \"[{'name': 'Eva Perez-Martin'}, {'name': 'Brianna Beechler'}, {'name': 'Katherine Scott'}, {'name': 'Lin-Mari de Klerk-Lorist'}, {'name': 'Fuquan Zhang'}, {'name': 'Georgina Limon'}, {'name': 'Brian Dugovich'}, {'name': 'Simon Gubbins'}, {'name': 'Arista Botha'}, {'name': 'Nicholas Juleff'}, {'name': 'Robyn Hetem'}, {'name': 'Louis van Schalkwyk'}, {'name': 'Francois Maree'}, {'name': 'Anna Jolles'}, {'name': 'Bryan Charleston'}]\", 'abstract': '<p>Foot-and-mouth disease (FMD) is one of the most important livestock diseases restricting international trade. While it is clear that African buffalo (Syncerus caffer) act as the main wildlife reservoir, viral and immune response dynamics during FMD virus acute infection have not been described before in this species. We used experimental needle inoculation and contact infections with three Southern African Territories serotypes to assess clinical, virological and immunological dynamics for thirty days post infection. Clinical FMD in the needle inoculated buffaloes was mild and characterised by pyrexia. Despite the absence of generalised vesicles, all contact animals were readily infected with their respective serotypes within the first 2-9 days after being mixed with needle challenged buffaloes. Irrespective of the route of infection or serotype there were positive associations between the viral loads in blood and the induction of host innate pro-inflammatory cytokines and acute phase proteins. Viral loads in blood and tonsils were tightly correlated during the acute phase of the infection, however, viraemia significantly declined after a peak at 4 days post infection (dpi), which correlated with the presence of detectable neutralising antibodies. In contrast, infectious virus was isolated in the tonsils until the last sampling point (30 dpi) in most animals. The pattern of virus detection in serum and tonsil swabs was similar for all three serotypes in the direct challenged and contact challenged animals. We have demonstrated for the first time, that African buffalo are indeed systemically affected by FMD virus and clinical FMD in buffalo is characterized by a transient pyrexia.  Despite the lack of FMD lesions, infection of African buffalo was characterised by high viral loads in blood and oropharynx, rapid and strong host innate and adaptive immune responses and high transmissibility.</p>', 'posted': '2021-11-26'}, {'doi': '10.1101/2021.11.23.469780', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'A chemical method to sequence 5-formylcytosine on RNA', 'authors': \"[{'name': 'Ang Li'}, {'name': 'Xuemeng Sun'}, {'name': 'A. Emilia Arguello'}, {'name': 'Ralph E. Kleiner'}]\", 'abstract': '<p>Epitranscriptomic RNA modifications can regulate biological processes, but there remains a major gap in our ability to identify and measure individual modifications at nucleotide resolution. Here we present Mal-Seq, a chemical method to sequence 5-formylcytosine (f<sup>5</sup>C) modifications on RNA based upon selective and efficient malononitrile-mediated labeling of f<sup>5</sup>C residues to generate adducts that are read as C-to-T mutations upon reverse transcription and PCR amplification. We apply Mal-Seq to characterize the prevalence of f<sup>5</sup>C at the wobble position of mt-tRNA(Met) in different organisms and tissue types and find that high-level f<sup>5</sup>C modification is present in mammals but lacking in lower eukaryotes. Our work sheds light on mitochondrial tRNA modifications throughout eukaryotic evolution and provides a general platform for characterizing the f<sup>5</sup>C epitranscriptome.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469781', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Inhibition of infection-induced vascular permeability modulates host leukocyte recruitment to Mycobacterium marinum granulomas in zebrafish', 'authors': \"[{'name': 'Julia Y Kam'}, {'name': 'Tina Cheng'}, {'name': 'Danielle C Garland'}, {'name': 'Warwick J Britton'}, {'name': 'David M Tobin'}, {'name': 'Stefan H Oehlers'}]\", 'abstract': '<p>Mycobacterial granuloma formation involves significant stromal remodeling and the growth of leaky, granuloma-associated vasculature. These permeable blood vessels aid mycobacterial growth, as anti-angiogenic or vascular normalizing therapies are beneficial host-directed therapies in pre-clinical models of tuberculosis. Here we demonstrate that vascular normalization through vascular endothelial-protein tyrosine phosphatase inhibition decreases granuloma hypoxia, the opposite effect of hypoxia-inducing anti-angiogenic therapy. Vascular normalization leads to increased T cell and decreased neutrophil recruitment to granulomas, correlates of a protective immune response against mycobacterial infection.</p>', 'posted': '2021-11-23'}, {'doi': '10.1101/2021.11.23.469785', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Ribonucleotide Reductase Regulatory Subunit M2 as a Driver of Glioblastoma TMZ-Resistance through Modulation of dNTP Production', 'authors': \"[{'name': 'Ella N Perrault'}, {'name': 'Jack M Shireman'}, {'name': 'Eunus S Ali'}, {'name': 'Isabelle Preddy'}, {'name': 'Peiyu Lin'}, {'name': 'Cheol Park'}, {'name': 'Luke Tomes'}, {'name': 'Andrew J Zolp'}, {'name': 'Shreya Budhiraja'}, {'name': 'Shivani Baisiwala'}, {'name': 'C. David James'}, {'name': 'Issam Ben-Sahra'}, {'name': 'Sebastian Pott'}, {'name': 'Anindita Basu'}, {'name': 'Atique U Ahmed'}]\", 'abstract': '<p>Glioblastoma (GBM) remains one of the most resistant and fatal forms of cancer. Previous studies have examined primary and recurrent GBM tumors, but it is difficult to study tumor evolution during therapy where resistance develops. To investigate this, we performed an <italic>in vivo</italic> single-cell RNA sequencing screen in a patient-derived xenograft (PDX) model. Primary GBM was modeled by mice treated with DMSO control, recurrent GBM was modeled by mice treated with temozolomide (TMZ), and during therapy GBM was modeled by mice euthanized after two of five TMZ treatments. Our analysis revealed the cellular population present during therapy to be distinct from primary and recurrent GBM. We found the Ribonucleotide Reductase gene family to exhibit a unique signature in our data due to an observed subunit switch to favor RRM2 during therapy. GBM cells were shown to rely on RRM2 during therapy causing RRM2-knockdown (KD) cells to be TMZ-sensitive. Using targeted metabolomics, we found RRM2-KDs to produce less dGTP and dCTP than control cells in response to TMZ (<italic>p&lt;0.0001</italic>). Supplementing RRM2-KDs with deoxycytidine and deoxyguanosine rescued TMZ-sensitivity, suggesting an RRM2-driven mechanism of chemoresistance, established by regulating the production of these nucleotides. <italic>In vivo</italic>, tumor-bearing mice treated with the RRM2-inhibitor, Triapine, in combination with TMZ, survived longer than mice treated with TMZ alone (<italic>p&lt;0.01</italic>), indicating promising clinical opportunities in targeting RRM2. Our data present a novel understanding of RRM2 activity, and its alteration during therapeutic stress as response to TMZ-induced DNA damage.</p>', 'posted': '2021-11-24'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.11.24.21266735</td>\n",
       "      <td>1266</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Increased risk of infection with SARS-CoV-2 Be...</td>\n",
       "      <td>[{'name': 'Stijn P. Andeweg'}, {'name': 'Harry...</td>\n",
       "      <td>&lt;p&gt;The extent to which severe acute respirator...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.11.21.21266484</td>\n",
       "      <td>233</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Pediatrics</td>\n",
       "      <td>True</td>\n",
       "      <td>Durability of SARS-CoV-2 Antibodies from Natur...</td>\n",
       "      <td>[{'name': 'Sarah E. Messiah'}, {'name': 'Franc...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Recent data s...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.11.22.469536</td>\n",
       "      <td>179</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Sampling the conformational landscapes of tran...</td>\n",
       "      <td>[{'name': 'Diego del Alamo'}, {'name': 'Davide...</td>\n",
       "      <td>&lt;p&gt;Equilibrium fluctuations and triggered conf...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.11.23.469762</td>\n",
       "      <td>146</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>In vitro and in vivo CRISPR-Cas9 screens revea...</td>\n",
       "      <td>[{'name': 'Tyson J. Ruetz'}, {'name': 'Chloe M...</td>\n",
       "      <td>&lt;p&gt;Aging impairs the ability of neural stem ce...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.11.22.469533</td>\n",
       "      <td>143</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>The Unique Cytoarchitecture and Wiring of the ...</td>\n",
       "      <td>[{'name': 'Casey Paquola'}, {'name': 'Margaret...</td>\n",
       "      <td>&lt;p&gt;It is challenging to specify the role of th...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.11.22.469481</td>\n",
       "      <td>107</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>All-optical electrophysiology with improved ge...</td>\n",
       "      <td>[{'name': 'He Tian'}, {'name': 'Hunter C. Davi...</td>\n",
       "      <td>&lt;p&gt;All-optical electrophysiology can be a powe...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.11.23.469770</td>\n",
       "      <td>101</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Characterization of the immune resistance of S...</td>\n",
       "      <td>[{'name': 'Keiya Uriu'}, {'name': 'Pal Crden...</td>\n",
       "      <td>&lt;p&gt;We have revealed that the SARS-CoV-2 Mu var...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.11.23.21266775</td>\n",
       "      <td>94</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Recurring Spatiotemporal Patterns of COVID-19 ...</td>\n",
       "      <td>[{'name': 'Hawre Jalal'}, {'name': 'Kyueun Lee...</td>\n",
       "      <td>&lt;p&gt;We analyzed the waxing and waning patterns ...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.11.21.469394</td>\n",
       "      <td>84</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Miniscope-LFOV: A large field of view, single ...</td>\n",
       "      <td>[{'name': 'Changliang Guo'}, {'name': 'Garrett...</td>\n",
       "      <td>&lt;p&gt;We present a large field of view (FOV) open...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.11.24.21266401</td>\n",
       "      <td>71</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Effectiveness of BNT162b2 and ChAdOx1 against ...</td>\n",
       "      <td>[{'name': 'Samuel Clifford'}, {'name': 'Paulin...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The ability o...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.11.24.469877</td>\n",
       "      <td>59</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Young transposable elements rewired gene regul...</td>\n",
       "      <td>[{'name': 'Sruti Patoori'}, {'name': 'Samantha...</td>\n",
       "      <td>&lt;p&gt;The hippocampus is associated with essentia...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.11.24.21266779</td>\n",
       "      <td>59</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>True</td>\n",
       "      <td>Severe COVID-19 induces molecular signatures o...</td>\n",
       "      <td>[{'name': 'Maria Mavrikaki'}, {'name': 'Jonath...</td>\n",
       "      <td>&lt;p&gt;Coronavirus disease 2019 (COVID-19) is pred...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.11.23.468420</td>\n",
       "      <td>54</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Plasticity of extrachromosomal and intrachromo...</td>\n",
       "      <td>[{'name': 'Kai Song'}, {'name': 'Jenna K. Mina...</td>\n",
       "      <td>&lt;p&gt;Cancer cells display two modes of focal amp...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.11.27.469863</td>\n",
       "      <td>51</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>A chemical-genetic map of the pathways control...</td>\n",
       "      <td>[{'name': 'Shuqi Li'}, {'name': 'Nicholas C Po...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Mycobacterium tuberculosis&lt;/italic&gt;...</td>\n",
       "      <td>2021-11-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.11.22.21266690</td>\n",
       "      <td>49</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Favipiravir for treatment of outpatients with ...</td>\n",
       "      <td>[{'name': 'Marisa Holubar'}, {'name': 'Aruna S...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Favipiravir i...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.11.23.469671</td>\n",
       "      <td>47</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Inferring kinetic rate constants from single-m...</td>\n",
       "      <td>[{'name': 'Markus Gtz'}, {'name': 'Anders Bar...</td>\n",
       "      <td>&lt;p&gt;Single-molecule FRET (smFRET) is a versatil...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.11.23.469668</td>\n",
       "      <td>46</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A bioactive phlebovirus-like envelope protein ...</td>\n",
       "      <td>[{'name': 'Monique Merchant'}, {'name': 'Carlo...</td>\n",
       "      <td>&lt;p&gt;Endogenous viral elements (EVEs), accountin...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.11.22.21266692</td>\n",
       "      <td>44</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Serological responses to COVID-19 booster vacc...</td>\n",
       "      <td>[{'name': 'Georgina Ireland'}, {'name': 'Heath...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Introduction&lt;/title&gt;&lt;p&gt;There are l...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.11.22.469599</td>\n",
       "      <td>42</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>True</td>\n",
       "      <td>A metagenomic DNA sequencing assay that is rob...</td>\n",
       "      <td>[{'name': 'Omary Mzava'}, {'name': 'Alexandre ...</td>\n",
       "      <td>&lt;p&gt;Metagenomic DNA sequencing is a powerful to...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.11.23.21266767</td>\n",
       "      <td>42</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>An immune correlate of SARS-CoV-2 infection an...</td>\n",
       "      <td>[{'name': 'Hannah E. Maier'}, {'name': 'Angel ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;An immune cor...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.11.26.470086</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Multi-state Modeling of G-protein Coupled Rece...</td>\n",
       "      <td>[{'name': 'Lim Heo'}, {'name': 'Michael Feig'}]</td>\n",
       "      <td>&lt;p&gt;The family of G-protein coupled receptors (...</td>\n",
       "      <td>2021-11-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.11.24.469859</td>\n",
       "      <td>38</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>High prevalence of Klebsiella pneumoniae in Eu...</td>\n",
       "      <td>[{'name': 'Carla Rodrigues'}, {'name': 'Kathri...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Klebsiella pneumoniae&lt;/italic&gt; spec...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.11.21.469166</td>\n",
       "      <td>36</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Population Imaging of Central Sensitization</td>\n",
       "      <td>[{'name': 'Charles Warwick'}, {'name': 'Joseph...</td>\n",
       "      <td>&lt;p&gt;Capsaicin applied locally to the skin cause...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.11.25.468910</td>\n",
       "      <td>32</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Sex-specific transcriptomic and epitranscripto...</td>\n",
       "      <td>[{'name': 'Andre Martins Reis'}, {'name': 'Jil...</td>\n",
       "      <td>&lt;p&gt;Our understanding of the molecular patholog...</td>\n",
       "      <td>2021-11-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.11.21.469478</td>\n",
       "      <td>32</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Real-time and remote MCMC trace inspection wit...</td>\n",
       "      <td>[{'name': 'Wytamma Wirth'}, {'name': 'Sebastia...</td>\n",
       "      <td>&lt;label&gt;1&lt;/label&gt;&lt;title&gt;Abstract&lt;/title&gt;&lt;p&gt;Baye...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.11.23.469684</td>\n",
       "      <td>32</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Cryo-EM structure of the octameric pore of Clo...</td>\n",
       "      <td>[{'name': 'Julia Bruggisser'}, {'name': 'Ioan ...</td>\n",
       "      <td>&lt;p&gt;We describe the cryo-EM structure of &lt;itali...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.11.23.469669</td>\n",
       "      <td>31</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Alpha power and stimulus-evoked activity disso...</td>\n",
       "      <td>[{'name': 'Matthew James Davidson'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Variability in the detection and discrimina...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.11.23.469591</td>\n",
       "      <td>30</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Chemo-mechanical Diffusion Waves Orchestrate C...</td>\n",
       "      <td>[{'name': 'Ze Gong'}, {'name': 'Koen van den D...</td>\n",
       "      <td>&lt;p&gt;Immune cells, such as macrophages and dendr...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.11.20.21266229</td>\n",
       "      <td>30</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Intensive Care and Critical Care Medicine</td>\n",
       "      <td>True</td>\n",
       "      <td>Efficacy and safety of a novel antiviral prepa...</td>\n",
       "      <td>[{'name': 'Hossein Faramarzi'}, {'name': 'Amir...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Introduction&lt;/title&gt;&lt;p&gt;Despite an ...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.11.22.21266565</td>\n",
       "      <td>30</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Impacts of vaccination and asymptomatic testin...</td>\n",
       "      <td>[{'name': 'Emily Nixon'}, {'name': 'Amy Thomas...</td>\n",
       "      <td>&lt;p&gt;We investigate the impact of vaccination an...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.11.25.470006</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Cryo-EM structure of the Smc5/6 holo-complex</td>\n",
       "      <td>[{'name': 'Stephen T. Hallett'}, {'name': 'Isa...</td>\n",
       "      <td>&lt;p&gt;The Smc5/6 complex plays an essential role ...</td>\n",
       "      <td>2021-11-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.11.22.469588</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Single organoid RNA-sequencing reveals high or...</td>\n",
       "      <td>[{'name': 'Kristin Gehling'}, {'name': 'Swati ...</td>\n",
       "      <td>&lt;p&gt;Over the last decades, organoids have been ...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.11.22.469546</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Long distance migration is a major factor driv...</td>\n",
       "      <td>[{'name': 'Quentin Rougemont'}, {'name': 'Aman...</td>\n",
       "      <td>&lt;p&gt;Inferring the genomic basis of local adapta...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.11.23.469675</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Improved peptide backbone fragmentation is the...</td>\n",
       "      <td>[{'name': 'Lars Kolbowski'}, {'name': 'Swantje...</td>\n",
       "      <td>&lt;p&gt;Proteome-wide crosslinking mass spectrometr...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.11.26.470139</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Recurrent evolution of cross-resistance in res...</td>\n",
       "      <td>[{'name': 'Aparajita Singh'}, {'name': 'Aabeer...</td>\n",
       "      <td>&lt;p&gt;The host susceptibility to one pathogen can...</td>\n",
       "      <td>2021-11-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.11.24.469969</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Transcriptional profiles of a foliar fungal en...</td>\n",
       "      <td>[{'name': 'Justin P. Shaffer'}, {'name': 'Morg...</td>\n",
       "      <td>&lt;p&gt;Symbiosis with bacteria is widespread among...</td>\n",
       "      <td>2021-11-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.11.24.21266807</td>\n",
       "      <td>26</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>A population framework for predicting the prop...</td>\n",
       "      <td>[{'name': 'Christopher Iddon'}, {'name': 'Benj...</td>\n",
       "      <td>&lt;p&gt;The number of occupants in a space influenc...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.11.23.466958</td>\n",
       "      <td>25</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>amica: an interactive and user-friendly web-pl...</td>\n",
       "      <td>[{'name': 'Sebastian Didusch'}, {'name': 'Mori...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Summary&lt;/title&gt;&lt;p&gt;Quantitative pro...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.11.19.469335</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pathology</td>\n",
       "      <td>True</td>\n",
       "      <td>Acute SARS-CoV-2 infection in pregnancy is ass...</td>\n",
       "      <td>[{'name': 'Elizabeth S. Taglauer'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;Human placental tissues have variable rates...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.11.24.21266741</td>\n",
       "      <td>24</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>True</td>\n",
       "      <td>A genome-wide association study of COVID-19 re...</td>\n",
       "      <td>[{'name': 'ngel Carracedo'}, {'name': ' '}]</td>\n",
       "      <td>&lt;p&gt;We describe the results of the Spanish Coal...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.11.24.469860</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>True</td>\n",
       "      <td>Nanopore ReCappable Sequencing maps SARS-CoV-2...</td>\n",
       "      <td>[{'name': 'Camilla Ugolini'}, {'name': 'Logan ...</td>\n",
       "      <td>&lt;p&gt;The SARS-CoV-2 virus has a complex transcri...</td>\n",
       "      <td>2021-11-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.11.22.21266651</td>\n",
       "      <td>24</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Cardiovascular Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>The Frequency, Penetrance and Variable Express...</td>\n",
       "      <td>[{'name': 'Ravi Shah'}, {'name': 'Babken Asatr...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;There is a pa...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.11.24.469913</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Synchrony and idiosyncrasy in the gut microbio...</td>\n",
       "      <td>[{'name': 'Johannes R. Bjrk'}, {'name': 'Maun...</td>\n",
       "      <td>&lt;p&gt;Human gut microbial dynamics are highly ind...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.11.22.469613</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Chronic Physical and Vicarious Psychosocial St...</td>\n",
       "      <td>[{'name': 'Daniela Franco'}, {'name': 'Andreas...</td>\n",
       "      <td>&lt;p&gt;Chronic stress can increase the risk of dev...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.11.23.469797</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Using AnnoTree to get more assignments, faster...</td>\n",
       "      <td>[{'name': 'Anupam Gautam'}, {'name': 'Hendrik ...</td>\n",
       "      <td>&lt;p&gt;In microbiome analysis, one main approach i...</td>\n",
       "      <td>2021-11-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.11.22.469587</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>The energy requirements of ion homeostasis det...</td>\n",
       "      <td>[{'name': 'Severin Schink'}, {'name': 'Mark Po...</td>\n",
       "      <td>&lt;p&gt;The majority of microbes on earth, whether ...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.11.25.469928</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Physiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Systemic LSD1 inhibition prevents aberrant rem...</td>\n",
       "      <td>[{'name': 'Bastian Ramms'}, {'name': 'Dennis P...</td>\n",
       "      <td>&lt;p&gt;The transition from lean to obese states in...</td>\n",
       "      <td>2021-11-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.11.23.469703</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>PerSVade: Personalized Structural Variation de...</td>\n",
       "      <td>[{'name': 'Miquel ngel Schikora-Tamarit'}, {'...</td>\n",
       "      <td>&lt;p&gt;Structural variants (SVs) like translocatio...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.11.23.469693</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A target expression threshold dictates invader...</td>\n",
       "      <td>[{'name': 'Elena Vialetto'}, {'name': 'Yanying...</td>\n",
       "      <td>&lt;p&gt;Immune systems must recognize and clear for...</td>\n",
       "      <td>2021-11-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.11.22.469601</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Genome context influences evolutionary flexibi...</td>\n",
       "      <td>[{'name': 'David A. Baltrus'}, {'name': 'Qian ...</td>\n",
       "      <td>&lt;p&gt;Integrative Conjugative Elements (ICEs) are...</td>\n",
       "      <td>2021-11-22</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2021.11.24.21266735    1266  medRxiv   \n",
       "1   10.1101/2021.11.21.21266484     233  medRxiv   \n",
       "2     10.1101/2021.11.22.469536     179  bioRxiv   \n",
       "3     10.1101/2021.11.23.469762     146  bioRxiv   \n",
       "4     10.1101/2021.11.22.469533     143  bioRxiv   \n",
       "5     10.1101/2021.11.22.469481     107  bioRxiv   \n",
       "6     10.1101/2021.11.23.469770     101  bioRxiv   \n",
       "7   10.1101/2021.11.23.21266775      94  medRxiv   \n",
       "8     10.1101/2021.11.21.469394      84  bioRxiv   \n",
       "9   10.1101/2021.11.24.21266401      71  medRxiv   \n",
       "10    10.1101/2021.11.24.469877      59  bioRxiv   \n",
       "11  10.1101/2021.11.24.21266779      59  medRxiv   \n",
       "12    10.1101/2021.11.23.468420      54  bioRxiv   \n",
       "13    10.1101/2021.11.27.469863      51  bioRxiv   \n",
       "14  10.1101/2021.11.22.21266690      49  medRxiv   \n",
       "15    10.1101/2021.11.23.469671      47  bioRxiv   \n",
       "16    10.1101/2021.11.23.469668      46  bioRxiv   \n",
       "17  10.1101/2021.11.22.21266692      44  medRxiv   \n",
       "18    10.1101/2021.11.22.469599      42  bioRxiv   \n",
       "19  10.1101/2021.11.23.21266767      42  medRxiv   \n",
       "20    10.1101/2021.11.26.470086      41  bioRxiv   \n",
       "21    10.1101/2021.11.24.469859      38  bioRxiv   \n",
       "22    10.1101/2021.11.21.469166      36  bioRxiv   \n",
       "23    10.1101/2021.11.25.468910      32  bioRxiv   \n",
       "24    10.1101/2021.11.21.469478      32  bioRxiv   \n",
       "25    10.1101/2021.11.23.469684      32  bioRxiv   \n",
       "26    10.1101/2021.11.23.469669      31  bioRxiv   \n",
       "27    10.1101/2021.11.23.469591      30  bioRxiv   \n",
       "28  10.1101/2021.11.20.21266229      30  medRxiv   \n",
       "29  10.1101/2021.11.22.21266565      30  medRxiv   \n",
       "30    10.1101/2021.11.25.470006      29  bioRxiv   \n",
       "31    10.1101/2021.11.22.469588      27  bioRxiv   \n",
       "32    10.1101/2021.11.22.469546      27  bioRxiv   \n",
       "33    10.1101/2021.11.23.469675      26  bioRxiv   \n",
       "34    10.1101/2021.11.26.470139      26  bioRxiv   \n",
       "35    10.1101/2021.11.24.469969      26  bioRxiv   \n",
       "36  10.1101/2021.11.24.21266807      26  medRxiv   \n",
       "37    10.1101/2021.11.23.466958      25  bioRxiv   \n",
       "38    10.1101/2021.11.19.469335      24  bioRxiv   \n",
       "39  10.1101/2021.11.24.21266741      24  medRxiv   \n",
       "40    10.1101/2021.11.24.469860      24  bioRxiv   \n",
       "41  10.1101/2021.11.22.21266651      24  medRxiv   \n",
       "42    10.1101/2021.11.24.469913      22  bioRxiv   \n",
       "43    10.1101/2021.11.22.469613      22  bioRxiv   \n",
       "44    10.1101/2021.11.23.469797      21  bioRxiv   \n",
       "45    10.1101/2021.11.22.469587      21  bioRxiv   \n",
       "46    10.1101/2021.11.25.469928      21  bioRxiv   \n",
       "47    10.1101/2021.11.23.469703      21  bioRxiv   \n",
       "48    10.1101/2021.11.23.469693      20  bioRxiv   \n",
       "49    10.1101/2021.11.22.469601      19  bioRxiv   \n",
       "\n",
       "                                 subject-area  covid  \\\n",
       "0                                Epidemiology   True   \n",
       "1                                  Pediatrics   True   \n",
       "2                                  Biophysics  False   \n",
       "3                                Neuroscience  False   \n",
       "4                                Neuroscience  False   \n",
       "5                                Neuroscience  False   \n",
       "6                                Microbiology   True   \n",
       "7                                Epidemiology   True   \n",
       "8                                Neuroscience  False   \n",
       "9                                Epidemiology   True   \n",
       "10                      Developmental Biology  False   \n",
       "11                                  Neurology   True   \n",
       "12                             Cancer Biology  False   \n",
       "13                               Microbiology  False   \n",
       "14      Infectious Diseases (except HIV/AIDS)   True   \n",
       "15                                 Biophysics  False   \n",
       "16                          Molecular Biology  False   \n",
       "17      Infectious Diseases (except HIV/AIDS)   True   \n",
       "18                                   Genomics   True   \n",
       "19      Infectious Diseases (except HIV/AIDS)   True   \n",
       "20                                 Biophysics  False   \n",
       "21                               Microbiology  False   \n",
       "22                               Neuroscience  False   \n",
       "23                               Neuroscience  False   \n",
       "24                       Evolutionary Biology  False   \n",
       "25                                 Biophysics  False   \n",
       "26                               Neuroscience  False   \n",
       "27                                 Biophysics  False   \n",
       "28  Intensive Care and Critical Care Medicine   True   \n",
       "29      Infectious Diseases (except HIV/AIDS)   True   \n",
       "30                               Biochemistry  False   \n",
       "31                               Cell Biology  False   \n",
       "32                       Evolutionary Biology  False   \n",
       "33                            Systems Biology  False   \n",
       "34                       Evolutionary Biology  False   \n",
       "35                               Microbiology  False   \n",
       "36                   Public and Global Health   True   \n",
       "37                             Bioinformatics  False   \n",
       "38                                  Pathology   True   \n",
       "39               Genetic and Genomic Medicine   True   \n",
       "40                                   Genomics   True   \n",
       "41                    Cardiovascular Medicine  False   \n",
       "42                                    Ecology  False   \n",
       "43                               Neuroscience  False   \n",
       "44                             Bioinformatics  False   \n",
       "45                               Microbiology  False   \n",
       "46                                 Physiology  False   \n",
       "47                             Bioinformatics  False   \n",
       "48                          Molecular Biology  False   \n",
       "49                               Microbiology  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Increased risk of infection with SARS-CoV-2 Be...   \n",
       "1   Durability of SARS-CoV-2 Antibodies from Natur...   \n",
       "2   Sampling the conformational landscapes of tran...   \n",
       "3   In vitro and in vivo CRISPR-Cas9 screens revea...   \n",
       "4   The Unique Cytoarchitecture and Wiring of the ...   \n",
       "5   All-optical electrophysiology with improved ge...   \n",
       "6   Characterization of the immune resistance of S...   \n",
       "7   Recurring Spatiotemporal Patterns of COVID-19 ...   \n",
       "8   Miniscope-LFOV: A large field of view, single ...   \n",
       "9   Effectiveness of BNT162b2 and ChAdOx1 against ...   \n",
       "10  Young transposable elements rewired gene regul...   \n",
       "11  Severe COVID-19 induces molecular signatures o...   \n",
       "12  Plasticity of extrachromosomal and intrachromo...   \n",
       "13  A chemical-genetic map of the pathways control...   \n",
       "14  Favipiravir for treatment of outpatients with ...   \n",
       "15  Inferring kinetic rate constants from single-m...   \n",
       "16  A bioactive phlebovirus-like envelope protein ...   \n",
       "17  Serological responses to COVID-19 booster vacc...   \n",
       "18  A metagenomic DNA sequencing assay that is rob...   \n",
       "19  An immune correlate of SARS-CoV-2 infection an...   \n",
       "20  Multi-state Modeling of G-protein Coupled Rece...   \n",
       "21  High prevalence of Klebsiella pneumoniae in Eu...   \n",
       "22        Population Imaging of Central Sensitization   \n",
       "23  Sex-specific transcriptomic and epitranscripto...   \n",
       "24  Real-time and remote MCMC trace inspection wit...   \n",
       "25  Cryo-EM structure of the octameric pore of Clo...   \n",
       "26  Alpha power and stimulus-evoked activity disso...   \n",
       "27  Chemo-mechanical Diffusion Waves Orchestrate C...   \n",
       "28  Efficacy and safety of a novel antiviral prepa...   \n",
       "29  Impacts of vaccination and asymptomatic testin...   \n",
       "30       Cryo-EM structure of the Smc5/6 holo-complex   \n",
       "31  Single organoid RNA-sequencing reveals high or...   \n",
       "32  Long distance migration is a major factor driv...   \n",
       "33  Improved peptide backbone fragmentation is the...   \n",
       "34  Recurrent evolution of cross-resistance in res...   \n",
       "35  Transcriptional profiles of a foliar fungal en...   \n",
       "36  A population framework for predicting the prop...   \n",
       "37  amica: an interactive and user-friendly web-pl...   \n",
       "38  Acute SARS-CoV-2 infection in pregnancy is ass...   \n",
       "39  A genome-wide association study of COVID-19 re...   \n",
       "40  Nanopore ReCappable Sequencing maps SARS-CoV-2...   \n",
       "41  The Frequency, Penetrance and Variable Express...   \n",
       "42  Synchrony and idiosyncrasy in the gut microbio...   \n",
       "43  Chronic Physical and Vicarious Psychosocial St...   \n",
       "44  Using AnnoTree to get more assignments, faster...   \n",
       "45  The energy requirements of ion homeostasis det...   \n",
       "46  Systemic LSD1 inhibition prevents aberrant rem...   \n",
       "47  PerSVade: Personalized Structural Variation de...   \n",
       "48  A target expression threshold dictates invader...   \n",
       "49  Genome context influences evolutionary flexibi...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Stijn P. Andeweg'}, {'name': 'Harry...   \n",
       "1   [{'name': 'Sarah E. Messiah'}, {'name': 'Franc...   \n",
       "2   [{'name': 'Diego del Alamo'}, {'name': 'Davide...   \n",
       "3   [{'name': 'Tyson J. Ruetz'}, {'name': 'Chloe M...   \n",
       "4   [{'name': 'Casey Paquola'}, {'name': 'Margaret...   \n",
       "5   [{'name': 'He Tian'}, {'name': 'Hunter C. Davi...   \n",
       "6   [{'name': 'Keiya Uriu'}, {'name': 'Pal Crden...   \n",
       "7   [{'name': 'Hawre Jalal'}, {'name': 'Kyueun Lee...   \n",
       "8   [{'name': 'Changliang Guo'}, {'name': 'Garrett...   \n",
       "9   [{'name': 'Samuel Clifford'}, {'name': 'Paulin...   \n",
       "10  [{'name': 'Sruti Patoori'}, {'name': 'Samantha...   \n",
       "11  [{'name': 'Maria Mavrikaki'}, {'name': 'Jonath...   \n",
       "12  [{'name': 'Kai Song'}, {'name': 'Jenna K. Mina...   \n",
       "13  [{'name': 'Shuqi Li'}, {'name': 'Nicholas C Po...   \n",
       "14  [{'name': 'Marisa Holubar'}, {'name': 'Aruna S...   \n",
       "15  [{'name': 'Markus Gtz'}, {'name': 'Anders Bar...   \n",
       "16  [{'name': 'Monique Merchant'}, {'name': 'Carlo...   \n",
       "17  [{'name': 'Georgina Ireland'}, {'name': 'Heath...   \n",
       "18  [{'name': 'Omary Mzava'}, {'name': 'Alexandre ...   \n",
       "19  [{'name': 'Hannah E. Maier'}, {'name': 'Angel ...   \n",
       "20    [{'name': 'Lim Heo'}, {'name': 'Michael Feig'}]   \n",
       "21  [{'name': 'Carla Rodrigues'}, {'name': 'Kathri...   \n",
       "22  [{'name': 'Charles Warwick'}, {'name': 'Joseph...   \n",
       "23  [{'name': 'Andre Martins Reis'}, {'name': 'Jil...   \n",
       "24  [{'name': 'Wytamma Wirth'}, {'name': 'Sebastia...   \n",
       "25  [{'name': 'Julia Bruggisser'}, {'name': 'Ioan ...   \n",
       "26  [{'name': 'Matthew James Davidson'}, {'name': ...   \n",
       "27  [{'name': 'Ze Gong'}, {'name': 'Koen van den D...   \n",
       "28  [{'name': 'Hossein Faramarzi'}, {'name': 'Amir...   \n",
       "29  [{'name': 'Emily Nixon'}, {'name': 'Amy Thomas...   \n",
       "30  [{'name': 'Stephen T. Hallett'}, {'name': 'Isa...   \n",
       "31  [{'name': 'Kristin Gehling'}, {'name': 'Swati ...   \n",
       "32  [{'name': 'Quentin Rougemont'}, {'name': 'Aman...   \n",
       "33  [{'name': 'Lars Kolbowski'}, {'name': 'Swantje...   \n",
       "34  [{'name': 'Aparajita Singh'}, {'name': 'Aabeer...   \n",
       "35  [{'name': 'Justin P. Shaffer'}, {'name': 'Morg...   \n",
       "36  [{'name': 'Christopher Iddon'}, {'name': 'Benj...   \n",
       "37  [{'name': 'Sebastian Didusch'}, {'name': 'Mori...   \n",
       "38  [{'name': 'Elizabeth S. Taglauer'}, {'name': '...   \n",
       "39       [{'name': 'ngel Carracedo'}, {'name': ' '}]   \n",
       "40  [{'name': 'Camilla Ugolini'}, {'name': 'Logan ...   \n",
       "41  [{'name': 'Ravi Shah'}, {'name': 'Babken Asatr...   \n",
       "42  [{'name': 'Johannes R. Bjrk'}, {'name': 'Maun...   \n",
       "43  [{'name': 'Daniela Franco'}, {'name': 'Andreas...   \n",
       "44  [{'name': 'Anupam Gautam'}, {'name': 'Hendrik ...   \n",
       "45  [{'name': 'Severin Schink'}, {'name': 'Mark Po...   \n",
       "46  [{'name': 'Bastian Ramms'}, {'name': 'Dennis P...   \n",
       "47  [{'name': 'Miquel ngel Schikora-Tamarit'}, {'...   \n",
       "48  [{'name': 'Elena Vialetto'}, {'name': 'Yanying...   \n",
       "49  [{'name': 'David A. Baltrus'}, {'name': 'Qian ...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>The extent to which severe acute respirator...  2021-11-24  \n",
       "1   <sec><title>Background</title><p>Recent data s...  2021-11-24  \n",
       "2   <p>Equilibrium fluctuations and triggered conf...  2021-11-22  \n",
       "3   <p>Aging impairs the ability of neural stem ce...  2021-11-23  \n",
       "4   <p>It is challenging to specify the role of th...  2021-11-22  \n",
       "5   <p>All-optical electrophysiology can be a powe...  2021-11-22  \n",
       "6   <p>We have revealed that the SARS-CoV-2 Mu var...  2021-11-24  \n",
       "7   <p>We analyzed the waxing and waning patterns ...  2021-11-24  \n",
       "8   <p>We present a large field of view (FOV) open...  2021-11-22  \n",
       "9   <sec><title>Background</title><p>The ability o...  2021-11-24  \n",
       "10  <p>The hippocampus is associated with essentia...  2021-11-24  \n",
       "11  <p>Coronavirus disease 2019 (COVID-19) is pred...  2021-11-24  \n",
       "12  <p>Cancer cells display two modes of focal amp...  2021-11-24  \n",
       "13  <p><italic>Mycobacterium tuberculosis</italic>...  2021-11-27  \n",
       "14  <sec><title>Background</title><p>Favipiravir i...  2021-11-24  \n",
       "15  <p>Single-molecule FRET (smFRET) is a versatil...  2021-11-23  \n",
       "16  <p>Endogenous viral elements (EVEs), accountin...  2021-11-24  \n",
       "17  <sec><title>Introduction</title><p>There are l...  2021-11-24  \n",
       "18  <p>Metagenomic DNA sequencing is a powerful to...  2021-11-23  \n",
       "19  <sec><title>Background</title><p>An immune cor...  2021-11-24  \n",
       "20  <p>The family of G-protein coupled receptors (...  2021-11-27  \n",
       "21  <p><italic>Klebsiella pneumoniae</italic> spec...  2021-11-24  \n",
       "22  <p>Capsaicin applied locally to the skin cause...  2021-11-22  \n",
       "23  <p>Our understanding of the molecular patholog...  2021-11-25  \n",
       "24  <label>1</label><title>Abstract</title><p>Baye...  2021-11-23  \n",
       "25  <p>We describe the cryo-EM structure of <itali...  2021-11-23  \n",
       "26  <p>Variability in the detection and discrimina...  2021-11-23  \n",
       "27  <p>Immune cells, such as macrophages and dendr...  2021-11-24  \n",
       "28  <sec><title>Introduction</title><p>Despite an ...  2021-11-23  \n",
       "29  <p>We investigate the impact of vaccination an...  2021-11-24  \n",
       "30  <p>The Smc5/6 complex plays an essential role ...  2021-11-25  \n",
       "31  <p>Over the last decades, organoids have been ...  2021-11-22  \n",
       "32  <p>Inferring the genomic basis of local adapta...  2021-11-22  \n",
       "33  <p>Proteome-wide crosslinking mass spectrometr...  2021-11-23  \n",
       "34  <p>The host susceptibility to one pathogen can...  2021-11-27  \n",
       "35  <p>Symbiosis with bacteria is widespread among...  2021-11-25  \n",
       "36  <p>The number of occupants in a space influenc...  2021-11-24  \n",
       "37  <sec><title>Summary</title><p>Quantitative pro...  2021-11-23  \n",
       "38  <p>Human placental tissues have variable rates...  2021-11-22  \n",
       "39  <p>We describe the results of the Spanish Coal...  2021-11-24  \n",
       "40  <p>The SARS-CoV-2 virus has a complex transcri...  2021-11-26  \n",
       "41  <sec><title>Background</title><p>There is a pa...  2021-11-24  \n",
       "42  <p>Human gut microbial dynamics are highly ind...  2021-11-24  \n",
       "43  <p>Chronic stress can increase the risk of dev...  2021-11-22  \n",
       "44  <p>In microbiome analysis, one main approach i...  2021-11-24  \n",
       "45  <p>The majority of microbes on earth, whether ...  2021-11-22  \n",
       "46  <p>The transition from lean to obese states in...  2021-11-25  \n",
       "47  <p>Structural variants (SVs) like translocatio...  2021-11-23  \n",
       "48  <p>Immune systems must recognize and clear for...  2021-11-23  \n",
       "49  <p>Integrative Conjugative Elements (ICEs) are...  2021-11-22  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "305 preprints (including 65 covering SARS-CoV-2, 252 from bioRxiv and 53 from medRxiv) published in the last 7 days before 2021-11-29 had been tweeted at least 3 times (maximum 1266)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
