{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "14181 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2ccbe6d7-a09b-4a41-8174-c0af31c01d24&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=fc2d57ed-7929-4a82-ada2-63fc8a23bbb6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=72cf07f6-1a10-4edb-86b8-3cdc6fc5c3b6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f094983c-648d-406a-8975-678723db561f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=fcd67939-e134-4929-9b3f-1187129eda7c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=cb5cfbbf-b196-45d6-b5e4-cf7f0cec7a50&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b9b34ce7-bf3b-4419-b512-4be411e7d341&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=0281ad14-bee8-40ff-863d-9ca80b5524fc&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=03f8e813-fd57-44bd-b19e-aeaa4dddf3f4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a25af283-585e-4d66-952e-bde8c1021a42&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5db63a22-6c87-418c-8676-d707770e850e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=02635167-f06b-49d3-9eab-37c47df9c07c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d541d9e0-e26f-4c5b-a815-8e8de4b344f3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b81773fb-9950-4822-b6bf-04c3bcf263e4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-11&until-occurred-date=2021-10-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.420448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.02.424917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.382143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/240317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464039...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/530097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275223.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/sqb.2018.83.037325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275569.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461278...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464465...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.463965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.27.175067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.413609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.206680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.10.986059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273300.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459623...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.463730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463750...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.28.923532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.17.156190...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.21261741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464035...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.463767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/407627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.459575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268037.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463953...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.28.066878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.27.175455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.437596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.03.932046...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.22.427801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.341602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.463948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.450922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21258385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.367649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20213546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.31.928630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.09.032995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/616565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.21264267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.372151...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.214536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.02.454825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.431115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462346...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464285...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.462913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21263834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.438087...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.20142703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.426870...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.442903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.21263548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464073...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.12.20193342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.163816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/458992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.31.275644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.09.287508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268722.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.17.156752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.03.20167395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.21255394...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.11.451967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/092205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.462574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.28.20219014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443947...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462435...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.21257248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.413229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.327114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.27.400788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.23.449573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460480...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.17.20037689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21262468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.356089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/768663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.463977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.14.20062463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.21263783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.222299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.460959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.21259273...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.11.293530...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.446027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.462405...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.08.191106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.21258086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.449395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.20204693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.462716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.20245993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.21251080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21262647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.461806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.434937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268581.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.21254540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256753...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.443897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.447448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.437224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.21259359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20088260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.20209114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.01.20166553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262263...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.463740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.451805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.462639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20159228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.14.21251715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464107...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.460971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.456972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21263008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.21262923...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/580456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463713...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.20235747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.21260079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.13.448249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.20.052027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.26.424423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.20245266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.07.238766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275308.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.08.20124792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431890...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.21251839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21258418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.25.20162107...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262955...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464638...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.389031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.463719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.20162420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.457912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.31.20221374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.342923...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.15.383323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.09.972307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.324368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.371005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/238527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.438828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.438792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.20093336...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.431364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.20086439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.459411...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.439712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.451597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.28.273029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.30.272237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.459429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425305...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.07.20056424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21263014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/765685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.458228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.21256049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.459925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/028522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.460811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.235549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.460970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458476...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458411...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457350...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.21261501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.451701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462172...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.21258567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.02.21261499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21258780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.21249563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.29.424619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461347...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.424510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.20248876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.399154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.395079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.17.386995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.385401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.10.374371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.21250957...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.434203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.21.21251403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.462880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21256632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.21251247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21263384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21249256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.335083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.18.20135210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.04.105700...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.31.116061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/276717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.088500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.04.20090076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/596718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/717801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/760819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.26.20044826...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.23.20039446...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.186684.114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461654...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.04.893909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.11.873398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464646...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.02.20186734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.464003...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.20205633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.20202267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464058...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.184507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.30.273920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.257758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.06.238915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.211482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.434982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257337...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.21255502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.439482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.445065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462458...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.21256193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21256946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.437190...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.21254026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.455586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.28.437432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462348...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21253866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445384...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.10.11.21264863', 'tweets': 226, 'archive': 'medRxiv', 'subject-area': 'Obstetrics and Gynecology', 'covid': True, 'title': 'Characterizing menstrual bleeding changes occurring after SARS-CoV-2 vaccination', 'authors': \"[{'name': 'Katharine MN Lee'}, {'name': 'Eleanor J Junkins'}, {'name': 'Urooba A Fatima'}, {'name': 'Maria L Cox'}, {'name': 'Kathryn BH Clancy'}]\", 'abstract': '<p>Many people began sharing that they experienced unexpected menstrual bleeding after SARS-CoV-2 inoculation. This emerging phenomenon was undeniable yet understudied. We investigated menstrual bleeding patterns among currently and formerly menstruating people, with a research design based off our expectations that these bleeding changes related to changes in clotting or inflammation, affecting normal menstrual repair. In this sample, 42% of people with regular menstrual cycles bled more heavily than usual, while 44% reported no change, after being vaccinated. Among people who typically do not menstruate, 71% of people on long-acting reversible contraceptives, 39% of people on gender-affirming hormones, and 66% of post-menopausal people reported breakthrough bleeding. We found increased/breakthrough bleeding was significantly associated with age, other vaccine side effects (fever, fatigue), history of pregnancy or birth, and ethnicity. Changes to menstrual bleeding are not uncommon nor dangerous, yet attention to these experiences is necessary to build trust in medicine.</p><sec><title>Teaser</title><p>Increased bleeding can occur post SARS-CoV-2 vaccines; this study characterizes patterns to enable future hypothesis testing.</p></sec>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.10.463851', 'tweets': 95, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The Japanese wolf is most closely related to modern dogs and its ancestral genome has been widely inherited by dogs throughout East Eurasia', 'authors': \"[{'name': 'Jun Gojobori'}, {'name': 'Nami Arakawa'}, {'name': 'Xiaokaiti Xiayire'}, {'name': 'Yuki Matsumoto'}, {'name': 'Shuichi Matsumura'}, {'name': 'Hitomi Hongo'}, {'name': 'Naotaka Ishiguro'}, {'name': 'Yohey Terai'}]\", 'abstract': '<p>The Japanese wolf (<italic>Canis lupus hodophilax</italic> Temminck, 1839) was a subspecies of the gray wolf that inhabited the Japanese Archipelago and became extinct 100-120 years ago. In this study, we determined the whole genomes of nine Japanese wolves from the 19<sup>th</sup>- early 20<sup>th</sup> centuries and 11 Japanese dogs and analyzed them along with both modern and ancient wolves and dogs. Genomic analyses indicate that the Japanese wolf was a unique subspecies of the gray wolf that was genetically distinct from both modern and ancient gray wolves, lacking gene flow with other gray wolves. A Phylogenetic tree that minimizes the effects of introgression shows that Japanese wolves are closest to the dog monophyletic group among the gray wolves. Moreover, Japanese wolves show significant genetic affinities with East Eurasian dogs. We estimated the level of introgression from the ancestor of the Japanese wolves to the ancestor of East Eurasian dogs that had occurred in the transitional period from the Pleistocene to the Holocene, at an early stage after divergence from West Eurasian dog lineages. Because of this introgression, Japanese wolf ancestry has been inherited by many dogs through admixture between East Eurasian dog lineages. As a result of this heredity, up to 5.5% of modern dog genomes throughout East Eurasia are derived from Japanese wolf ancestry.</p>', 'posted': '2021-10-11'}, {'doi': '10.1101/2021.10.16.464636', 'tweets': 88, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Automatic identification and annotation of MYB gene family members in plants', 'authors': \"[{'name': 'Boas Pucker'}]\", 'abstract': '<p>Background: MYBs are among the largest transcription factor families in plants. Consequently, members of this family are involved in a plethora of processes including development and specialized metabolism. The MYB families of many plant species were investigated in the last two decades since the first investigation looked at Arabidopsis thaliana. This body of knowledge and characterized sequences provide the basis for the identification, classification, and functional annotation of candidate sequences in new genome and transcriptome assemblies.\\nResults: A pipeline for the automatic identification and functional annotation of MYBs in a given sequence data set was implemented in Python. MYB candidates are identified, screened for the presence of a MYB domain and other motifs, and finally placed in a phylogenetic context with well characterized sequences. In addition to technical benchmarking based on existing annotation, the transcriptome assembly of Croton tiglium and the annotated genome sequence of Castanea crenata were screened for MYBs. Results of both analyses are presented in this study to illustrate the potential of this application. The analysis of one species takes only a few minutes depending on the number of predicted sequences and the size of the MYB gene family. This pipeline, the required bait sequences, and reference sequences for a classification are freely available on github: https://github.com/bpucker/MYB_annotator.\\nConclusions: This automatic annotation of the MYB gene family in novel assemblies makes genome-wide investigations consistent and paves the way for comparative studies in the future. Candidate genes for in-depth analyses are presented based on their orthology to previously characterized sequences which allows the functional annotation of the newly identified MYBs with high confidence. The identification of orthologs can also be harnessed to detect duplication and deletion events.</p>', 'posted': '2021-10-16'}, {'doi': '10.1101/2021.10.12.464039', 'tweets': 72, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Histone H1 regulates non-coding RNA turnover on chromatin in a m6A-dependent manner', 'authors': \"[{'name': 'Jos Miguel Fernndez-Justel'}, {'name': 'Cristina Santa-Mara'}, {'name': 'Alberto Ferrera-Lagoa'}, {'name': 'Mnica Salinas-Pena'}, {'name': 'Magdalena M. Maslon'}, {'name': 'Albert Jordan'}, {'name': 'Javier F. Cceres'}, {'name': 'Mara Gmez'}]\", 'abstract': '<p>Linker histones are highly abundant chromatin-associated proteins with well-established structural roles in chromatin and as general transcriptional repressors. In addition, it has been long proposed that histone H1 exerts context-specific effects on gene expression. Here, we have identified a new function of histone H1 in chromatin structure and transcription using a range of genomic approaches. We show that histone H1-depleted cells accumulate nascent non-coding RNAs on chromatin, suggesting that histone H1 prevents non-coding RNA transcription and regulates non-coding transcript turnover on chromatin. Accumulated non-coding transcripts have reduced levels of m6A modification and cause replication-transcription conflicts. Accordingly, altering the m6A RNA methylation pathway rescues the replicative phenotype of H1 loss. This work unveils unexpected regulatory roles of histone H1 on non-coding RNA turnover and m6A deposition, highlighting the intimate relationship between chromatin conformation, RNA metabolism and DNA replication to maintain genome performance.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.08.463334', 'tweets': 44, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': True, 'title': 'Covid-19 genomic analysis reveals clusters of emerging sublineages within the delta variant', 'authors': \"[{'name': 'Evans K. Rono'}]\", 'abstract': '<p>The emerging SARS-CoV-2 variants may potentially have enhanced transmissibility and virulence of the virus, and impacts on performance of diagnostic tools and efficacy of vaccines. Genomic surveillance provides an opportunity to detect and characterize new mutations early enough for effective deployment of control strategies. Here, genomic data from Germany and United Kingdom were examined for genetic diversity by assessing gene mutations and inferring phylogeny. Delta variant sublineages were grouped into seven distinct clusters of spike mutations located in N-terminal domain of S1 region (T95I, D138H, *D142G, Y145H and A222V) and S2 region (T719I and *N950D). The most predominant cluster was T95I mutation, with the highest frequencies (71.1% - 83.9%) in Wales, England and Scotland, and the least frequencies (8.9% - 12.1%) in Germany. Two mutations, *D142G and *N950D here described as *reverse mutations and T719I mutation, were largely unique to Germany. In a month, frequencies of D142G had increased from 55.6% to 67.8 % in Germany. Additionally, a cluster of D142G+T719I/T mutation went up from 27.7% to 34.1%, while a T95I+ D142G+N950D/N cluster rose from 19.2% to 26.2%. Although, two distinct clusters of T95I+D138H (2.6% - 3.8%) and T95I+Y145H+A222V (2.5% - 8.5%) mutations were present in all the countries, they were most predominant in Wales and Scotland respectively. Results suggest divergent evolutionary trajectories between the clusters of D142G mutation and those of T95I mutation. These findings provide insights into underlying dynamics of evolution of the delta variant. Future studies may evaluate the epidemiological and biological implications of these sublineages.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.14.464465', 'tweets': 42, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Polypolish: short-read polishing of long-read bacterial genome assemblies', 'authors': \"[{'name': 'Ryan R Wick'}, {'name': 'Kathryn E Holt'}]\", 'abstract': '<p>Long-read-only bacterial genome assemblies usually contain residual errors, most commonly homopolymer-length errors. Short-read polishing tools can use short reads to fix these errors, but most rely on short-read alignment which is unreliable in repeat regions. Errors in such regions are therefore challenging to fix and often remain after short-read polishing. Here we introduce Polypolish, a new short-read polisher which uses all-per-read alignments to repair errors in repeat sequences that other polishers cannot. In benchmarking tests using both simulated and real reads, we find that Polypolish performs well, and the best results are achieved by using Polypolish in combination with other short-read polishers.</p>', 'posted': '2021-10-16'}, {'doi': '10.1101/2021.10.12.464066', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Tissue-wide coordination of calcium signaling regulates the epithelial stem cell pool during homeostasis', 'authors': \"[{'name': 'Jessica L Moore'}, {'name': 'Feng Gao'}, {'name': 'Catherine Matte-Martone'}, {'name': 'Shuangshuang Du'}, {'name': 'Elizabeth Lathrop'}, {'name': 'Smirthy Ganesan'}, {'name': 'Lin Shao'}, {'name': 'Dhananjay Bhaskar'}, {'name': 'Andy Cox'}, {'name': 'Caroline Hendry'}, {'name': 'Bastian Rieck'}, {'name': 'Smita Krishnaswamy'}, {'name': 'Valentina Greco'}]\", 'abstract': '<p>Regenerative processes in the mammalian skin require coordinated cell-cell communication. Ca<sup>2+</sup> signaling can coordinate tissue-level responses in developing and wounded epithelia in tissue explants and invertebrates[1, 2, 3, 4]. However, its role in the homeostatic, regenerative basal layer of the skin epithelium is unknown due to significant challenges in studying signaling dynamics in a spatially complex tissue context in live mice. Here we combine <italic>in vivo</italic> imaging of dynamic Ca<sup>2+</sup> signaling at the single cell level across thousands of cells with a novel computational approach, Geometric Scattering Trajectory Homology (GSTH). GSTH models Ca<sup>2+</sup> as signals over a cell adjacency graph and uses a multi-level wavelet-like transform (called a scattering transform) to extract signaling patterns from our high dimensional <italic>in vivo</italic> datasets. We discover local Ca<sup>2+</sup> signaling patterns are orchestrated so that signals flow in a coordinated and directed manner across the tissue, distinct from topographically uncoordinated Ca<sup>2+</sup> signaling in excitatory tissues. Directed Ca<sup>2+</sup> signaling is regulated by the major gap junction protein in the epidermal stem cell layer, Connexin 43 (Cx43). Cx43 gap junctions are dissociated as cells progress through the cell cycle out of G1 and play an essential role in the progression of stem cells from G2 towards mitosis. Finally, G2 cells display related signaling patterns and are essential for tissue-level signaling coordination. Together, our results provide insight into how such a ubiquitous signaling pathway regulates highly specific behaviors and outcomes at a tissue-wide level to maintain proper homeostasis.</p>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.12.463965', 'tweets': 39, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Ferroptosis Regulation by the NGLY1/NFE2L1 Pathway', 'authors': \"[{'name': 'Giovanni C. Forcina'}, {'name': 'Lauren Pope'}, {'name': 'Magdalena Murray'}, {'name': 'Wentao Dong'}, {'name': 'Monther Abu-Remaileh'}, {'name': 'Carolyn R. Bertozzi'}, {'name': 'Scott J. Dixon'}]\", 'abstract': '<p>Ferroptosis is an oxidative form of non-apoptotic cell death whose transcriptional regulation is poorly understood. Capncollar (CNC) transcription factors including Nuclear Factor Erythroid-2 Related Factor 1 (NFE2L1/NRF1) and NFE2L2 (NRF2) are important regulators of oxidative stress responses. Here, we report that NFE2L1 expression inhibits ferroptosis, independent of NFE2L2. NFE2L1 inhibits ferroptosis by promoting expression of the key anti-ferroptotic lipid hydroperoxidase glutathione peroxidase 4 (GPX4). NFE2L1 abundance and function are regulated post-translationally by N-glycosylation. Functional maturation of NFE2L1 requires deglycosylation by cytosolic peptide:N-glycanase 1 (NGLY1). We find that loss of NGLY1 or NFE2L1 enhances ferroptosis sensitivity. Expression of wild-type NGLY1 but not a disease-associated NGLY1 mutant inhibits ferroptosis, and this effect is dependent on the presence of NFE2L1. Enhanced ferroptosis sensitivity in NFE2L1 and NFE2L2 knockout cells can be potently reverted by expression of an NFE2L1 mutant containing eight asparagine-to-aspartate protein sequence substitutions, which mimic NGLY1-catalyzed sequence editing. Enhanced ferroptosis sensitivity in NGLY1/NFE2L1 pathway mutants could also be reversed by overexpression of NFE2L2. These results suggest that ferroptosis sensitivity is regulated by NGLY1-catalyzed NFE2L1 deglycosylation, and highlight a broad role for CNC transcription factors in ferroptosis regulation.</p><sec><title>Significance Statement</title><p>Ferroptosis is an oxidative form of cell death whose biochemical regulation remains incompletely understood. NFE2L1/NRF1 is a capncollar (CNC) transcription factor whose role in ferroptosis regulation is unclear. Unlike the CNC family member NFE2L2/NRF2, NFE2L1 is an N-glycoprotein whose abundance is regulated by post-translational deglycosylation catalyzed by the enzyme peptide:N-glycanase 1 (NGLY1). Our results indicate that NGLY1-mediated NFE2L1 deglycosylation, resulting in editing of the NFE2L1 amino acid sequence, is necessary for NFE2L1 to inhibit ferroptosis. Mechanistically, NFE2L1 inhibits ferroptosis by via the anti-ferroptotic protein GPX4. This work demonstrates that CNC transcription factors beyond NFE2L2 can regulate ferroptosis. This work may suggest a role of misregulation of ferroptosis in NGLY1 deficiency, an ultrarare genetic disorder.</p></sec>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.13.21264960', 'tweets': 36, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Evaluation of real-life use of Point-Of-Care Rapid Antigen TEsting for SARS-CoV-2 in schools for outbreak control (EPOCRATES)', 'authors': \"[{'name': 'Ana C. Blanchard'}, {'name': 'Marc Desforges'}, {'name': 'Annie-Claude Labb'}, {'name': 'Cat Tuong Nguyen'}, {'name': 'Yves Petit'}, {'name': 'Dominic Besner'}, {'name': 'Kate Zinszer'}, {'name': 'Olivier Sguin'}, {'name': 'Zineb Laghdir'}, {'name': 'Kelsey Adams'}, {'name': 'Marie-ve Benoit'}, {'name': 'Genevive Leduc'}, {'name': 'Jean Longtin'}, {'name': 'Ioannis Ragoussis'}, {'name': 'David L. Buckeridge'}, {'name': 'Caroline Quach'}]\", 'abstract': '<p>We evaluated the use of rapid antigen detection tests (RADT) for the diagnosis of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection in school settings to determine RADTs performance characteristics compared to PCR.</p><sec><title>Methods</title><p>We did a real-world, prospective observational cohort study where recruited high-school students and staff from two high-schools in Montreal (Canada) were followed from January 25<sup>th</sup> to June 10<sup>th</sup>, 2021. Twenty-five percent of asymptomatic participants were tested weekly by RADT (nasal) and PCR (gargle). Class contacts of a case were tested. Symptomatic participants were tested by RADT (nasal) and PCR (nasal and gargle). The number of cases/outbreak and number of outbreaks were compared to other high schools in the same area.</p></sec><sec><title>Results</title><p>Overall, 2,099 students and 286 school staff members consented to participate. The overall RADTs specificity varied from 99.8 to 100%, with a lower sensitivity, varying from 28.6% in asymptomatic to 83.3% in symptomatic participants. The number of outbreaks was not different in the 2 participating schools compared to other high schools in the same area, but included a greater proportion of asymptomatic cases. Returning students to school after a 7-day quarantine, with a negative PCR on D6-7 after exposure, did not lead to subsequent outbreaks, as shown by serial testing. Of cases for whom the source was known, 37 of 57 (72.5%) were secondary to household transmission, 13 (25%) to intra-school transmission and one to community contacts between students in the same school.</p></sec><sec><title>Conclusion</title><p>RADT did not perform well as a screening tool in asymptomatic individuals. Reinforcing policies for symptom screening when entering schools and testing symptomatic individuals with RADT on the spot may avoid subsequent significant exposures in class.</p></sec><sec><title>Research in context</title><sec><title>Evidence before this study</title><p>Rapid antigen detection tests (RADT) have been used for years to diagnose respiratory pathogens, such as influenza and respiratory syncytial virus. These tests detect the presence of a specific viral antigen and are usually performed on nasopharyngeal or nasal specimens. RADT are relatively inexpensive and can be used at the point-of-care. Their performance characteristics vary, but they usually have high specificity and moderate sensitivity compared with PCR. For SARS-CoV-2, RADTs specificity has been quite high (99.5%), but with sensitivity ranging from 28.9% to 98.3%.</p></sec><sec><title>Added value of this study</title><p>This study determined the sensitivity and specificity of RADT against PCR for the diagnosis of SARS-CoV-2 in a real-life setting in high school students. The test performed well, in terms of specificity, but had a sensitivity that was as low as 28.6% in asymptomatic individuals to 83.3% in symptomatic individuals. The longitudinal aspect of the study also allowed to determine the impact of RADT on school outbreaks.</p></sec><sec><title>Implication of all available evidence</title><p>RADT does not perform well as a screening tool for asymptomatic individuals. Reinforcing policies for symptom screening when entering schools and testing symptomatic individuals with RADT on the spot may avoid subsequent significant exposures in class.</p></sec></sec>', 'posted': '2021-10-14'}, {'doi': '10.1101/2021.10.10.21264827', 'tweets': 36, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Heterologous SARS-CoV-2 Booster Vaccinations  Preliminary Report', 'authors': \"[{'name': 'Robert L. Atmar'}, {'name': 'Kirsten E. Lyke'}, {'name': 'Meagan E. Deming'}, {'name': 'Lisa A. Jackson'}, {'name': 'Angela R. Branche'}, {'name': 'Hana M. El Sahly'}, {'name': 'Christina A. Rostad'}, {'name': 'Judith M. Martin'}, {'name': 'Christine Johnston'}, {'name': 'Richard E. Rupp'}, {'name': 'Mark J. Mulligan'}, {'name': 'Rebecca C. Brady'}, {'name': 'Robert W. Frenck'}, {'name': 'Martn Bcker'}, {'name': 'Angelica C. Kottkamp'}, {'name': 'Tara M. Babu'}, {'name': 'Kumaravel Rajakumar'}, {'name': 'Srilatha Edupuganti'}, {'name': 'Ann R. Falsey'}, {'name': 'Christine M. Posavad'}, {'name': 'Janet I. Archer'}, {'name': 'Sonja Crandon'}, {'name': 'Seema U. Nayak'}, {'name': 'Daniel Szydlo'}, {'name': 'Jillian Zemanek'}, {'name': 'Clara P. Dominguez Islas'}, {'name': 'Elizabeth R. Brown'}, {'name': 'Mehul S. Suthar'}, {'name': 'M. Juliana McElrath'}, {'name': 'Adrian B. McDermott'}, {'name': 'Sarah E. OConnell'}, {'name': 'David C. Montefiori'}, {'name': 'Amanda Eaton'}, {'name': 'Kathleen M. Neuzil'}, {'name': 'David S. Stephens'}, {'name': 'Paul C. Roberts'}, {'name': 'John H. Beigel'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>While Coronavirus disease 2019 (Covid-19) vaccines are highly effective, breakthrough infections are occurring. Booster vaccinations have recently received emergency use authorization (EUA) for certain populations but are restricted to homologous mRNA vaccines. We evaluated homologous and heterologous booster vaccination in persons who had received an EUA Covid-19 vaccine regimen.</p></sec><sec><title>Methods</title><p>In this phase 1/2 open-label clinical trial conducted at ten U.S. sites, adults who received one of three EUA Covid-19 vaccines at least 12 weeks prior to enrollment and had no reported history of SARS-CoV-2 infection received a booster injection with one of three vaccines (Moderna mRNA-1273 100-g, Janssen Ad26.COV2.S 510<sup>10</sup> virus particles, or Pfizer-BioNTech BNT162b2 30-g; nine combinations). The primary outcomes were safety, reactogenicity, and humoral immunogenicity on study days 15 and 29.</p></sec><sec><title>Results</title><p>458 individuals were enrolled: 154 received mRNA-1273, 150 received Ad26.CoV2.S, and 154 received BNT162b2 booster vaccines. Reactogenicity was similar to that reported for the primary series. Injection site pain, malaise, headache, and myalgia occurred in more than half the participants. Booster vaccines increased the neutralizing activity against a D614G pseudovirus (4.2-76-fold) and binding antibody titers (4.6-56-fold) for all combinations; homologous boost increased neutralizing antibody titers 4.2-20-fold whereas heterologous boost increased titers 6.2-76-fold. Day 15 neutralizing and binding antibody titers varied by 28.7-fold and 20.9-fold, respectively, across the nine prime-boost combinations.</p></sec><sec><title>Conclusion</title><p>Homologous and heterologous booster vaccinations were well-tolerated and immunogenic in adults who completed a primary Covid-19 vaccine regimen at least 12 weeks earlier.</p><p>(Funded by National Institute of Allergy and Infectious Diseases; <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://ClinicalTrials.gov\">ClinicalTrials.gov</ext-link> number, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04889209\">NCT04889209</ext-link>)</p></sec>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.15.464546', 'tweets': 32, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Airpart: Interpretable statistical models for analyzing allelic imbalance in single-cell datasets', 'authors': \"[{'name': 'Wancen Mu'}, {'name': 'Hirak Sarkar'}, {'name': 'Avi Srivastava'}, {'name': 'Kwangbom Choi'}, {'name': 'Rob Patro'}, {'name': 'Michael I Love'}]\", 'abstract': '<p>Motivation: Allelic expression analysis aids in detection of cis-regulatory mechanisms of genetic variation which produce allelic imbalance (AI) in heterozygotes. Measuring AI in bulk data lacking time or spatial resolution has the limitation that cell-type-specific (CTS), spatial-, or time-dependent AI signals may be dampened or not detected. Results: We introduce a statistical method airpart for identifying differential CTS AI from single-cell RNA-sequencing (scRNA-seq) data, or other spatially- or time-resolved datasets. airpart outputs discrete partitions of data, pointing to groups of genes and cells under common mechanisms of cis-genetic regulation. In order to account for low counts in single-cell data, our method uses a Generalized Fused Lasso with Binomial likelihood for partitioning groups of cells by AI signal, and a hierarchical Bayesian model for AI statistical inference. In simulation, airpart accurately detected partitions of cell types by their AI and had lower RMSE of allelic ratio estimates than existing methods. In real data, airpart identified differential AI patterns across cell states and could be used to define trends of AI signal over spatial or time axes. Availability: The airpart package is available as a R/Bioconductor package at https://bioconductor.org/packages/airpart.</p>', 'posted': '2021-10-16'}, {'doi': '10.1101/2021.10.13.463730', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Dual polarity voltage imaging reveals subthreshold dynamics and concurrent spiking patterns of multiple neuron-types', 'authors': \"[{'name': 'Madhuvanthi Kannan'}, {'name': 'Ganesh Vasan'}, {'name': 'Simon Haziza'}, {'name': 'Cheng Huang'}, {'name': 'Radek Chrapkiewicz'}, {'name': 'Junjie Luo'}, {'name': 'Jessica A Cardin'}, {'name': 'Mark J Schnitzer'}, {'name': 'Vincent A Pieribone'}]\", 'abstract': '<p>Genetically encoded fluorescent voltage indicators are ideally suited to reveal the millisecond-scale interactions among and between distinct, targeted cell populations. However, current indicator families lack the requisite sensitivity for in vivo multipopulation imaging. We describe high-performance green and red sensors, Ace-mNeon2 and VARNAM2, and their reverse response-polarity variants, pAce and pAceR. Our indicators enable 0.4-1 kHz voltage recordings from &gt;50 neurons per field-of-view in awake mice and ~30-min continuous imaging in flies. Using dual-polarity multiplexed imaging, we uncovered behavioral state-dependent interactions between distinct neocortical subclasses, as well as contributions to hippocampal field potentials from non-overlapping projection neuronal ensembles. By combining three mutually compatible indicators, we demonstrate concurrent triple-population voltage imaging. Our approach will empower investigations of the dynamic interplay between neuronal subclasses at single-spike resolution.</p>', 'posted': '2021-10-15'}, {'doi': '10.1101/2021.10.12.464035', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Brain endothelial antigen presentation detains CD8+ T cells at the blood-brain barrier leading to its breakdown', 'authors': \"[{'name': 'Sidar Aydin'}, {'name': 'Javier Pareja'}, {'name': 'Vivianne M. Schallenberg'}, {'name': 'Armelle Klopstein'}, {'name': 'Thomas Gruber'}, {'name': 'Nicolas Page'}, {'name': 'Elisa Kaba'}, {'name': 'Urban Deutsch'}, {'name': 'Aaron J. Johnson'}, {'name': 'Mirjam Schenk'}, {'name': 'Doron Merkler'}, {'name': 'Britta Engelhardt'}]\", 'abstract': '<p>Blood-brain barrier (BBB) breakdown and immune cell infiltration into the central nervous system (CNS) are early hallmarks of multiple sclerosis (MS). High numbers of CD8<sup>+</sup> T cells are found in MS lesions and antigen (Ag)-presentation at the BBB was proposed to promote CD8<sup>+</sup> T-cell entry into the CNS. Employing live cell imaging and primary mouse brain microvascular endothelial cells (pMBMECs) as <italic>in vitro</italic> model of the BBB and a mouse model of CNS autoimmunity, we here show that pMBMECs process and present antigens leading to effector CD8<sup>+</sup> T-cell differentiation. Under physiological flow, endothelial Ag-presentation prohibited CD8<sup>+</sup> T-cell crawling and diapedesis leading to pMBMEC apoptosis. Reduced motility of Ag-specific CD8<sup>+</sup> T cells was also observed in CNS microvessels in neuroinflammation <italic>in vivo.</italic> Luminal MHC class I Ag-presentation at the BBB thus prohibits CD8<sup>+</sup> T-cell entry into the CNS and rather triggers CD8<sup>+</sup> T cell mediated focal BBB breakdown.</p>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.16.463767', 'tweets': 25, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Genomic insights into the population history and biological adaptation of Southwestern Chinese Hmong-Mien people', 'authors': \"[{'name': 'Yan Yan Liu'}, {'name': 'Mengge Wang'}, {'name': 'Changhui Liu'}, {'name': 'Jingrong Zhu'}, {'name': 'Xing Zou'}, {'name': 'Wenshan Li'}, {'name': 'Lin Lin Wang'}, {'name': 'Cuo Leng'}, {'name': 'Quyi Xu'}, {'name': 'Hui-Yuan Yeh'}, {'name': 'Chuan-Chao Wang'}, {'name': 'Xiaohong Wen'}, {'name': 'Chao Liu'}, {'name': 'Guanglin He'}]\", 'abstract': '<p>Hmong-Mien-speaking (HM) populations, widely distributed in South China, North of Thailand, Laos and Vietnam, have experienced different settlement environments, dietary habits and pathogen exposure. However, their specific biological adaptation also remained largely uncharacterized, which is important in the population evolutionary genetics and Trans-Omics for regional Precision Medicine. Besides, the origin and genetic diversity of HM people and their phylogenetic relationship with surrounding modern and ancient populations are unknown. Here, we reported genome-wide SNPs in 52 representative Miao people and combined them with 144 HM people from 13 geographically representative populations to characterize the full genetic admixture and adaptive landscape of HM speakers. We found that obvious genetic substructures existed in geographically different HM populations and also identified one new ancestral lineage specifically exited in HM people, which spatially distributed from Sichuan and Guizhou in the North to Thailand in the South and temporally dated to at least 500 years. The sharing patterns of the newly-identified homogeneous ancestry component combined the estimated admixture times via the decay of Linkage Disequilibrium and haplotype sharing in GLOBETROTTER suggested that the modern HM-speaking populations originated from Southwest China and migrated southward recently, which is consistent with the reconstructed phenomena of linguistic and archeological documents. Additionally, we identified specific adaptive signatures associated with several important human nervous system biological functions. Our pilot work emphasized the importance of anthropologically-informed sampling and deeply genetic structure reconstruction via whole-genome sequencing in the next step in the deep Chinese population genomic diversity project (CPGDP), especially in the ethnolinguistic regions.</p>', 'posted': '2021-10-16'}, {'doi': '10.1101/2021.10.11.463953', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Coping with oxidative stress in extreme environments: the distinctive roles played by Acinetobacter sp. Ver3 superoxide dismutases', 'authors': \"[{'name': 'Bruno Alejandro Steimbrch'}, {'name': 'Mariana G. Sartorio'}, {'name': 'Nstor Cortez'}, {'name': 'Daniela Albanesi'}, {'name': 'Mara-Natalia Lisa'}, {'name': 'Guillermo Daniel Repizo'}]\", 'abstract': '<p><italic>Acinetobacter sp</italic>. Ver3 is a polyextremophilic strain characterized by a high tolerance to radiation and pro-oxidants. The Ver3 genome comprises the <italic>sodB</italic> and <italic>sodC</italic> genes encoding an iron (<sup>AV3</sup>SodB) and a copper/zinc superoxide dismutase (<sup>AV3</sup>SodC), respectively; however, the specific role(s) of these genes has remained elusive. We show that the expression of <italic>sodB</italic> remained unaltered in different oxidative stress conditions whereas <italic>sodC</italic> was up-regulated in the presence of blue light. Besides, we studied the changes in the in vitro activity of each SOD enzyme in response to diverse agents and solved the crystal structure of <sup>AV3</sup>SodB at 1.34 , one of the highest resolutions achieved for a SOD. Cell fractionation studies interestingly revealed that <sup>AV3</sup>SodB is located in the cytosol whereas <sup>AV3</sup>SodC is also found in the periplasm. Consistently, a bioinformatic analysis of the genomes of 53 <italic>Acinetobacter</italic> species pointed out the presence of at least one SOD type in each compartment, suggesting that these enzymes are separately required to cope with oxidative stress. Surprisingly, <sup>AV3</sup>SodC was found in an active state also in outer membrane vesicles, probably exerting a protective role. Overall, our multidisciplinary approach highlights the relevance of SOD enzymes when <italic>Acinetobacter</italic> spp. are confronted with oxidizing agents.</p>', 'posted': '2021-10-11'}, {'doi': '10.1101/2021.10.10.463858', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A curated collection of Klebsiella metabolic models reveals variable substrate usage and gene essentiality', 'authors': \"[{'name': 'Jane Hawkey'}, {'name': 'Ben Vezina'}, {'name': 'Jonathan M. Monk'}, {'name': 'Louise M. Judd'}, {'name': 'Taylor Harshegyi'}, {'name': 'Sebastin Lpez-Fernndez'}, {'name': 'Carla Rodrigues'}, {'name': 'Sylvain Brisse'}, {'name': 'Kathryn E. Holt'}, {'name': 'Kelly L. Wyres'}]\", 'abstract': '<p>The <italic>Klebsiella pneumoniae</italic> species complex (KpSC) is a set of seven <italic>Klebsiella</italic> taxa which are found in a variety of niches, and are an important cause of opportunistic healthcare-associated infections in humans. Due to increasing rates of multi-drug resistance within the KpSC, there is a growing interest in better understanding the biology and metabolism of these organisms to inform novel control strategies. We collated 37 sequenced KpSC isolates isolated from a variety of niches, representing all seven taxa. We generated strain-specific genome scale metabolic models (GEMs) for all 37 isolates and simulated growth phenotypes on 511 distinct carbon, nitrogen, sulphur and phosphorus substrates. Models were curated and their accuracy assessed using matched phenotypic growth data for 94 substrates (median accuracy of 96%). We explored species-specific growth capabilities and examined the impact of all possible single gene deletions on growth in 145 core carbon substrates. These analyses revealed multiple strain-specific differences, within and between species and highlight the importance of selecting a diverse range of strains when exploring KpSC metabolism. This diverse set of highly accurate GEMs could be used to inform novel drug design, enhance genomic analyses, and identify novel virulence and resistance determinants. We envisage that these 37 curated strain-specific GEMs, covering all seven taxa of the KpSC, provide a valuable resource to the <italic>Klebsiella</italic> research community.</p>', 'posted': '2021-10-11'}, {'doi': '10.1101/2021.10.12.463948', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Pathology', 'covid': False, 'title': 'Prion-like -synuclein pathology in the brains of infants: Krabbe disease as a novel seed-competent -synucleinopathy', 'authors': \"[{'name': 'Christopher Hatton'}, {'name': 'Simona S. Ghanem'}, {'name': 'David J. Koss'}, {'name': 'Ilham Y. Abdi'}, {'name': 'Elizabeth Gibbons'}, {'name': 'Rita Guerreiro'}, {'name': 'Jose Bras'}, {'name': 'Lauren Walker'}, {'name': 'Ellen Gelpi'}, {'name': 'Wendy Heywood'}, {'name': 'Tiago F. Outeiro'}, {'name': 'Johannes Attems'}, {'name': 'Bobby McFarland'}, {'name': 'Rob Forsyth'}, {'name': 'Omar M. El-Agnaf'}, {'name': 'Daniel Erskine'}, {'name': ' '}]\", 'abstract': '<p>Krabbe disease (KD) is an infantile neurodegenerative disorder resulting from pathogenic variants in the <italic>GALC</italic> gene which causes accumulation of the toxic sphingolipid psychosine. <italic>GALC</italic> variants are associated with increased risk of Lewy body diseases (LBD), an umbrella term for age-associated neurodegenerative diseases in which the protein -synuclein aggregates into Lewy bodies. To explore whether -synuclein in KD has pathological similarities to that in LBD, we compared <italic>post-mortem</italic> KD tissue to that of infant control cases and identified alterations to -synuclein localisation and expression of modifications associated with LBD. To determine whether -synuclein in KD displayed pathogenic properties associated with LBD we evaluated its seeding capacity using the real-time quaking-induced conversion assay. Strikingly, seeded aggregation of -synuclein resulted in the formation of fibrillar aggregates similar to those observed in LBD, confirming the prion-like capacity of KD-derived -synuclein. These observations constitute the first report of prion-like -synuclein in the brain tissue of infants and challenge the putative view that -synuclein pathology is merely an age-associated phenomenon, instead suggesting it can result from alterations to biological processes such as sphingolipid homeostasis. Our findings have important implications for understanding the mechanisms underlying Lewy body formation in LBD.</p>', 'posted': '2021-10-14'}, {'doi': '10.1101/2021.10.13.464217', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Disentangling choice value and choice conflict in sequential decisions under risk', 'authors': \"[{'name': 'Laura Fontanesi'}, {'name': 'Amitai Shenhav'}, {'name': 'Sebastian Gluth'}]\", 'abstract': '<p>Recent years have witnessed a surge of interest in understanding the neural and cognitive dynamics that drive sequential decision making in general and foraging behavior in particular. Due to the intrinsic properties of most sequential decision-making paradigms, however, previous research in this area has suffered from the difficulty to disentangle properties of the decision related to (a) the value of switching to a new patch versus (b) the conflict experienced between choosing to stay or leave. Here, we show how the same problems arise in studies of sequential decision-making under risk, and how they can be overcome, taking as a specific example recent research on the pig dice game. In each round of the pig dice game, people roll a die and accumulate rewards until they either decide to proceed to the next round or lose all rewards. By combining simulation-based dissections of the task structure with two experiments, we show how an extension of the standard paradigm, together with cognitive modeling of decision-making processes, disentangles value-from conflict-related choice properties. Our study elucidates the cognitive mechanisms of sequential decision making and underscores the importance of avoiding potential pitfalls of paradigms that are commonly used in this research area.</p>', 'posted': '2021-10-14'}, {'doi': '10.1101/2021.10.12.464028', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'HORmon: automated annotation of human centromeres', 'authors': \"[{'name': 'Olga Kunyavskaya'}, {'name': 'Tatiana Dvorkina'}, {'name': 'Andrey V. Bzikadze'}, {'name': 'Ivan A. Alexandrov'}, {'name': 'Pavel A. Pevzner'}]\", 'abstract': '<p>Recent advances in long-read sequencing opened a possibility to address the long-standing questions about the architecture and evolution of human centromeres. They also emphasized the need for centromere annotation (partitioning human centromeres into monomers and higher-order repeats (HORs)). Even though there was a half-century-long series of semi-manual studies of centromere architecture, a rigorous centromere annotation algorithm is still lacking. Moreover, an automated centromere annotation is a prerequisite for studies of genetic diseases associated with centromeres, and evolutionary studies of centromeres across multiple species. Although the monomer decomposition (transforming a centromere into a monocentromere written in the monomer alphabet) and the HOR decomposition (representing a monocentromere in the alphabet of HORs) are currently viewed as two separate problems, we demonstrate that they should be integrated into a single framework in such a way that HOR (monomer) inference affects monomer (HOR) inference. We thus developed the HORmon algorithm that integrates the monomer/HOR inference and automatically generates the human monomers/HORs that are largely consistent with the previous semi-manual inference.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.08.21264727', 'tweets': 16, 'archive': 'medRxiv', 'subject-area': 'Rehabilitation Medicine and Physical Therapy', 'covid': False, 'title': 'Exoskeleton-assisted walking improves pulmonary and exercise performances more than conventional exercise program in individuals with spinal cord injury: a randomised controlled study', 'authors': \"[{'name': 'Xiao-Na Xiang'}, {'name': 'Li-Ming Zhang'}, {'name': 'Hui-Yan Zong'}, {'name': 'Yi Ou'}, {'name': 'Xi Yu'}, {'name': 'Yan Liu'}, {'name': 'Hong-Ying Jiang'}, {'name': 'Hong Cheng'}, {'name': 'Hong-Chen He'}, {'name': 'Cheng-Qi He'}]\", 'abstract': '<sec><title>Question</title><p>In people with spinal cord injury, does exoskeleton-assisted walking training improve pulmonary ventilation function, motor function and related body structure, walking, and activities of daily life equally comparing those with conventional exercise program?</p></sec><sec><title>Design</title><p>Randomised controlled trial with concealed allocation, assessor blinding and intention-to-treat analysis.</p></sec><sec><title>Intervention</title><p>Both groups undertook 16 sessions of 50-60min training (4 days/week, 4 weeks). Participants in the experimental group received EAW trainings using AIDER system, which assisted standing, walking, and climbing the stairs. The control group received a conventional exercise program which combined aerobic, resistance, flexibility and walking training.</p></sec><sec><title>Outcome measures</title><p>The primary outcome was the pulmonary function test. The secondary outcomes included: a 6-minute walk test with Borg scale (0 to 10) rating of exertion, 10-metre walk test, basic activities of daily living, trunk control test, lower extremity motor score, muscle tone of lower limb, bone mineral density, and distal femoral cartilage at baseline and upon completion of treatment.</p></sec><sec><title>Results</title><p>After 4 weeks of trainings, the experimental group improved more on the forced vital capacity (MD 0.53, 95%CI 0.01 to 1.06), predicted FVC% (MD 19.59, 95%CI 6.63 to 32.54) and forced expiratory volume in 1s (MD 0.61, 95%CI 0.15 to 1.07); BADL (MD 19.75, 95%CI 10.88 to 28.62); and distal femoral cartilage than the control group. Participants completed 6-minute walk test with median 17.3 meters while wearing the exoskeleton. There was no difference in trunk control test, lower extremity motor score, muscle tone, bone mineral density and adverse event.</p></sec><sec><title>Conclusions</title><p>In people with lower thoracic neurological level of spinal cord injury, exoskeleton-assisted walking training instead of a conventional excise program has potential benefits to facilitate pulmonary ventilation function, walking, basic activities of daily living and thickness of cartilage.</p></sec><sec><title>Trial registration</title><p>ChiCTR2000034623</p></sec>', 'posted': '2021-10-11'}, {'doi': '10.1101/2021.10.14.464400', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Intratumoral and extratumoral synapses are required for glioblastoma progression in Drosophila', 'authors': \"[{'name': 'Maria Losada-Perez'}, {'name': 'Mamen Hernandez Garcia-Moreno'}, {'name': 'Sergio Casas-Tinto'}]\", 'abstract': '<p>Glioblastoma (GB) is the most aggressive, lethal and frequent primary brain tumor. It originates from glial cells and is characterized by rapid expansion through infiltration. GB cells interact with the microenvironment and healthy surrounding tissues, mostly neurons and vessels. GB cells project tumor microtubes (TMs) that contact with neurons and exchange signaling molecules related to Wingless/WNT, JNK, Insulin or Neuroligin-3 pathways. This cell to cell communication promotes GB expansion and neurodegeneration. Moreover, healthy neurons form glutamatergic functional synapses with GB cells which facilitate GB expansion and premature death in mouse GB xerograph models. Targeting signaling and synaptic components of GB progression may become a suitable strategy against glioblastoma. In a Drosophila GB model, we have determined the post-synaptic nature of GB cells with respect to neurons, and the contribution of post-synaptic genes expressed in GB cells to tumor progression. In addition, we document the presence of intratumoral synapses between GB cells, and the functional contribution of pre-synaptic genes to GB calcium dependent activity and expansion. Finally, we explore the relevance of synaptic genes in GB cells to the lifespan reduction caused by GB advance. Our results indicate that both presynaptic and postsynaptic proteins play a role in GB progression and lethality.</p>', 'posted': '2021-10-16'}, {'doi': '10.1101/2021.10.13.464136', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Replication Fork Uncoupling Causes Nascent Strand Degradation and Fork Reversal', 'authors': \"[{'name': 'Tamar Kavlashvili'}, {'name': 'James M Dewar'}]\", 'abstract': '<p>Genotoxins cause nascent strand degradation (NSD) and fork reversal during DNA replication. NSD and fork reversal are crucial for genome stability and exploited by chemotherapeutic approaches. However, it is unclear how NSD and fork reversal are triggered. Additionally, the fate of the replicative helicase during these processes is unknown. We developed a biochemical approach to study synchronous, localized NSD and fork reversal using <italic>Xenopus</italic> egg extracts. We show that replication fork uncoupling stimulates NSD of both nascent strands and progressive conversion of uncoupled forks to reversed forks. The replicative helicase remains bound during NSD and fork reversal, indicating that both processes take place behind the helicase. Unexpectedly, NSD occurs before and after fork reversal, indicating that multiple degradation steps take place. Overall, our data show that uncoupling causes NSD and fork reversal and identify key steps involved in these processes.</p>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.12.464098', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The Hair Cell Analysis Toolbox: A machine learning-based whole cochlea analysis pipeline', 'authors': \"[{'name': 'Christopher J. Buswinka'}, {'name': 'David B. Rosenberg'}, {'name': 'Artur A. Indzhykulian'}]\", 'abstract': '<p>Auditory hair cells, the whole length of the cochlea, are routinely visualized using light microscopy techniques. It is common, therefore, for one to collect more data than is practical to analyze manually. There are currently no widely accepted tools for unsupervised, unbiased, and comprehensive analysis of cells in an entire cochlea. This represents a stark gap between image-based data and other tests of cochlear function. To close this gap, we present a machine learning-based hair cell analysis toolbox, for the analysis of whole cochleae, imaged with confocal microscopy. The software presented here allows the automation of common image analysis tasks such as counting hair cells, determining their best frequency, as well as quantifying single cell immunofluorescence intensities along the entire cochlear coil. We hope these automated tools will remove a considerable barrier in cochlear image analysis, allowing for more informative and less selective data analysis practices.</p>', 'posted': '2021-10-14'}, {'doi': '10.1101/2021.10.12.464064', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'New from old: discovery of the novel antibiotic actinomycin L in Streptomyces sp. MBT27', 'authors': \"[{'name': 'Nataliia Machushynets'}, {'name': 'Somayah S. Elsayed'}, {'name': 'Chao Du'}, {'name': 'Maxime A. Siegler'}, {'name': 'Mercedes de la Cruz'}, {'name': 'Olga Genilloud'}, {'name': 'Thomas Hankemeier'}, {'name': 'Gilles P. van Wezel'}]\", 'abstract': '<p>Streptomycetes are major producers of bioactive natural products, including the majority of the antibiotics. While much if the low-hanging fruit has been discovered, it is predited that less than 5% of the chemical space has been mined. Here, we describe the novel actinomycins L<sub>1</sub> and L<sub>2</sub>, which are produced by <italic>Streptomyces</italic> sp. MBT27. The molecules were discovered via metabolic analysis combined with molecular networking of cultures grown with different combinations of carbon sources. Actinomycins L<sub>1</sub> and L<sub>2</sub> are diastereoisomers, and the structure of actinomycin L<sub>2</sub> was resolved using NMR and single crystal X-ray crystallography. Actinomycin L is formed via a unique spirolinkage of anthranilamide to the 4-oxoproline moiety of actinomycin X<sub>2,</sub> prior to the condensation of the actinomycin halves. Feeding anthranilamide to cultures of <italic>Streptomyces antibioticus</italic>, which has the same biosynthetic gene cluster as <italic>Streptomyces</italic> sp. MBT27 but only produces actinomycin X<sub>2</sub>, resulted in the production of actinomycin L. This shows that actinomycin L results from joining two distinct metabolic pathways, namely those for actinomycin X<sub>2</sub> and for anthranilamide. Actinomycins L<sub>1</sub> and L<sub>2</sub> showed significant antimicrobial activity against Gram- positive bacteria. Our work shows how new molecules can still be identified even in the oldest of natural product families.</p><sec><title>IMPORTANCE</title><p>Actinomycin was the first antibiotic discovered in an actinobacterium by Selman Waksman and colleagues, as early as 1940. This period essentially marks the start of the golden era of antibiotic discovery. Over time, emerging antimicrobial resistance (AMR) and the declining success rate of antibiotic discovery resulted in the current antibiotic crisis. We surprisingly discovered that under some growth conditions, <italic>Streptomyces</italic> sp. MBT27 can produce actinomycins that are significantly different from those that have been published so far. The impact of this work is not only that we have discovered a novel molecule with very interesting chemical modifications in one of the oldest antibiotics ever described, but also that this requires the combined action of primary and secondary metabolic pathways, namely the biosynthesis of anthranilamide and of actinomycin X<sub>2</sub>, respectively. The implication of the discovery is that even the most well-studied families of natural products may still have surprises in store for us.</p></sec>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.13.464196', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'A linear programming-based strategy to save pipette tips in automated DNA assembly', 'authors': \"[{'name': 'Kirill Sechkar'}, {'name': 'Zoltan A Tuza'}, {'name': 'Guy-Bart V Stan'}]\", 'abstract': '<p>Laboratory automation and mathematical optimisation are key to improving the efficiency of synthetic biology research. While there are algorithms optimising the construct designs and synthesis strategies for DNA assembly, the optimisation of how DNA assembly reaction mixes are prepared remains largely unexplored. Here, we focus on reducing the pipette tip consumption of a liquid-handling robot as it delivers DNA parts across a multi-well plate where several constructs are being assembled in parallel. We propose a linear programming formulation of this problem based on the capacitated vehicle routing problem, along with an algorithm which applies a linear programming solver to our formulation, hence providing a strategy to prepare a given set of DNA assembly mixes using fewer pipette tips. The algorithm performed well in randomly generated and real-life scenarios concerning several modular DNA assembly standards, proving capable of reducing the pipette tip consumption by up to 61% in large-scale cases. Combining automatic process optimisation and robotic liquid-handling, our strategy promises to greatly improve the efficiency of DNA assembly, either used alone or in combination with other algorithmic methods.</p>', 'posted': '2021-10-15'}, {'doi': '10.1101/2021.10.13.464285', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Respiratory syncytial virus matrix protein assembles as a lattice with local and extended order that coordinates the position of the fusion glycoprotein', 'authors': \"[{'name': 'Bryan S Sibert'}, {'name': 'Joseph Y Kim'}, {'name': 'Jie E Yang'}, {'name': 'Zunlong Ke'}, {'name': 'Christopher C. Stobart'}, {'name': 'Martin M Moore'}, {'name': 'Elizabeth R Wright'}]\", 'abstract': '<p>Respiratory syncytial virus (RSV) is a significant cause of respiratory illness in young children and adults worldwide. There is currently no vaccine or targeted antiviral for RSV. RSV is an enveloped, filamentous, negative-strand RNA virus. Individual virions vary in both diameter and length, with an average diameter of ~130 nm and ranging from ~500 nm to over 10 m in length. The RSV matrix (M) protein is peripherally associated with the interior of the viral membrane. Though the general arrangement of structural proteins within the virion is known, the molecular organization of M and other structural proteins was previously unknown. Here, using whole-cell cryo-electron tomography and sub-tomogram averaging, we show that M is arranged in a packed helical-like lattice of M-dimers ordered at an angle of ~47 to the viral long axis. Sub-tomogram averages including F and M indicate that the position of F on the viral surface is correlated with the underlying M lattice. Finally, we report that RSV F is frequently observed as pairs, with the F trimers oriented in an anti-parallel conformation to support potential interaction between trimers. These results provide insight into RSV assembly and virion organization and may aid in the identification and development of RSV vaccines and anti-viral targets.</p>', 'posted': '2021-10-15'}, {'doi': '10.1101/2021.10.11.463377', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Environmentally dependent and independent control of cell shape determination by Rho GTPase regulators in melanoma', 'authors': \"[{'name': 'L. G. Dent'}, {'name': 'N. Curry'}, {'name': 'H. Sparks'}, {'name': 'V. Bousgouni'}, {'name': 'V. Maioli'}, {'name': 'S. Kumar'}, {'name': 'I. Munro'}, {'name': 'C. Dunsby'}, {'name': 'C. Bakal'}]\", 'abstract': '<p>In order to invade 3D tissues, cancer cells dynamically change cell morphology in response to geometric and mechanical cues in the environment. But how cells determine their shape in 3D versus 2D environments is poorly understood. Studying 2D versus 3D single cell shape determination has historically been technically difficult due to the lack of methodologies to directly compare the two environments. We developed an approach to study cell shape in 2D versus 3D by measuring cell shape at different depths in collagen using stage-scanning oblique plane microscopy (ssOPM). We find characteristic shape changes occur in melanoma cells depending on whether a cell is attached to a 2D surface or 3D environment, and that these changes can be modulated by Rho GTPase regulatory proteins. Our data suggest that regulation of cell protrusivity undergoes a switch of control between different Rho GTPase regulators depending on the physical microenvironment.</p>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.11.463956', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'SARS-CoV-2 variants exhibit increased kinetic stability of open spike conformations as an evolutionary strategy', 'authors': \"[{'name': 'Ziwei Yang'}, {'name': 'Yang Han'}, {'name': 'Shilei Ding'}, {'name': 'Andrs Finzi'}, {'name': 'Walther Mothes'}, {'name': 'Maolin Lu'}]\", 'abstract': '<p>SARS-CoV-2 variants of concern harbor mutations in the Spike (S) glycoprotein that confer more efficient transmission and dampen the efficacy of COVID-19 vaccines and antibody therapies. S mediates virus entry and is the primary target for antibody responses. Structural studies of soluble S variants have revealed an increased propensity towards conformations accessible to receptor human Angiotensin-Converting Enzyme 2 (hACE2). However, real-time observations of conformational dynamics that govern the structural equilibriums of the S variants have been lacking. Here, we report single-molecule Frster Resonance Energy Transfer (smFRET) studies of S variants containing critical mutations, including D614G and E484K, in the context of virus particles. Investigated variants predominantly occupied more open hACE2-accessible conformations, agreeing with previous structures of soluble trimers. Additionally, these S variants exhibited decelerated transitions in hACE2-accessible/bound states. Our finding of increased S kinetic stability in the open conformation provides a new perspective on SARS-CoV-2 adaptation to the human population.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.13.464175', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Modular, robust and extendible multicellular circuit design in yeast', 'authors': \"[{'name': 'Alberto Carignano'}, {'name': 'Dai Hua Chen'}, {'name': 'Cannon Mallory'}, {'name': 'Clay Wright'}, {'name': 'Georg Seelig'}, {'name': 'Eric Klavins'}]\", 'abstract': '<p>Division of labor between cells is ubiquitous in biology but the use of multi-cellular consortia for engineering applications is only beginning to be explored. A significant advantage of multi-cellular circuits is their potential to be modular with respect to composition but this claim has not yet been extensively tested using experiments and quantitative modeling. Here, we construct a library of 24 yeast strains capable of sending, receiving or responding to three molecular signals, characterize them experimentally and build quantitative models of their input-output relationships. We then compose these strains into two- and three-strain cascades as well a four-strain bistable switch and show that experimentally measured consortia dynamics can be predicted from the models of the constituent parts. To further explore the achievable range of behaviors, we perform a fully automated computational search over all two-, three- and four-strain consortia to identify combinations that realize target behaviors including logic gates, band-pass filters and time pulses. Strain combinations that are predicted to map onto a target behavior are further computationally optimized and then experimentally tested. Experiments closely track computational predictions. The high reliability of these model descriptions further strengthens the feasibility and highlights the potential for distributed computing in synthetic biology.</p>', 'posted': '2021-10-14'}, {'doi': '10.1101/2021.10.13.464073', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Rates of primary production in groundwater rival those in oligotrophic marine systems', 'authors': \"[{'name': 'Will A. Overholt'}, {'name': 'Susan Trumbore'}, {'name': 'Xiaomei Xu'}, {'name': 'Till L.V. Bornemann'}, {'name': 'Alexander J. Probst'}, {'name': 'Markus Krger'}, {'name': 'Martina Herrmann'}, {'name': 'Bo Thamdrup'}, {'name': 'Laura Bristow'}, {'name': 'Martin Taubert'}, {'name': 'Valrie F. Schwab'}, {'name': 'Martin Hlzer'}, {'name': 'Manja Marz'}, {'name': 'Kirsten Ksel'}]\", 'abstract': '<p>The terrestrial subsurface contains nearly all of Earths freshwater reserves<sup>1</sup> and harbors upwards of 60% of our planets total prokaryotic biomass<sup>2,3</sup>. While genetic surveys suggest these organisms rely on <italic>in situ</italic> carbon fixation, rather than the translocation of photosynthetically derived organic carbon<sup>46</sup>, corroborating measurements of carbon fixation in the subsurface are absent. Using a novel ultra-low level <sup>14</sup>C-labeling technique, we show that <italic>in situ</italic> carbon fixation rates in a carbonate aquifer reached 10% of the median rates measured in oligotrophic marine surface waters, and were up to six-fold greater than those observed in lower euphotic zone waters where deep chlorophyll levels peak. Empirical carbon fixation rates were substantiated by both nitrification and anammox rate data. Metagenomic analyses revealed a remarkable abundance of putative chemolithoautotrophic members of an uncharacterized order of Nitrospiria  the first representatives of this class expected to fix carbon via the Wood-Ljungdahl pathway. Based on these fixation rates, we extrapolate global primary production in carbonate groundwaters to be 0.11 Pg of carbon per year.</p>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.12.464113', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Single-particle cryo-EM structures from iDPC-STEM at near-atomic resolution', 'authors': \"[{'name': 'Ivan Lazi'}, {'name': 'Maarten Wirix'}, {'name': 'Max Leo Leidl'}, {'name': 'Felix de Haas'}, {'name': 'Maximilian Beckers'}, {'name': 'Evgeniya V. Pechnikova'}, {'name': 'Knut Mller-Caspary'}, {'name': 'Ricardo Egoavil'}, {'name': 'Eric G.T. Bosch'}, {'name': 'Carsten Sachse'}]\", 'abstract': '<p>Electron cryo-microscopy (cryo-EM) is becoming one of the routine tools for structure determination of biological macromolecules. Commonly, molecular images are obtained by conventional transmission electron microcopy (CTEM) using underfocus and subsequently computationally combined into a high-resolution 3D structure. Here, we apply scanning transmission electron microscopy (STEM) using the integrated differential phase contrast mode also known as iDPC-STEM to the cryo-EM test specimen of tobacco mosaic virus (TMV). The micrographs show complete contrast transfer to high resolution and enable the cryo-EM structure determination at 3.5  resolution using single-particle based helical reconstruction. A series of cryo-EM TMV maps was resolved at near-atomic resolution taken at different convergence semi-angle (CSA) beams and share identical features with maps obtained by CTEM of a previously acquired same-sized TMV data set. The associated map B-factors from iDPC-STEM match those obtained by CTEM recordings using 2<sup>nd</sup> generation direct electron detection devices. These data show that STEM imaging in general, and in particular the iDPC-STEM approach, can be applied to vitrified single-particle specimens to determine near-atomic resolution cryo-EM structures of biological macromolecules.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.13.464275', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Benchmarking imputation methods for network inference using a novel method of synthetic scRNA-seq data generation', 'authors': \"[{'name': 'Ayoub Lasri'}, {'name': 'Vahid Shahrezaei'}, {'name': 'Marc Sturrock'}]\", 'abstract': '<p>Single cell RNA-sequencing (scRNA-seq) has very rapidly become the new workhorse of modern biology providing an unprecedented global view on cellular diversity and heterogeneity. In particular, the structure of gene-gene expression correlation contains information on the underlying gene regulatory networks. However, interpretation of scRNA-seq data is challenging due to specific experimental error and biases that are unique to this kind of data including drop-out (or technical zeros). To deal with this problem several methods for imputation of zeros for scRNA-seq have been developed. However, it is not clear how these processing steps affect inference of genetic networks from single cell data. Here, we introduce Biomodelling.jl, a tool for generation of synthetic scRNA-seq data using multiscale modelling of stochastic gene regulatory networks in growing and dividing cells. Our tool produces realistic transcription data with a known ground truth network topology that can be used to benchmark different approaches for gene regulatory network inference. Using this tool we investigate the impact of different imputation methods on the performance of several network inference algorithms. Biomodelling.jl provides a versatile and useful tool for future development and benchmarking of network inference approaches using scRNA-seq data.</p>', 'posted': '2021-10-14'}, {'doi': '10.1101/2021.10.12.464112', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'In vivo versus in silico assessment of potentially pathogenic missense variants in human reproductive genes', 'authors': \"[{'name': 'Xinbao Ding'}, {'name': 'Priti Singh'}, {'name': 'Tina N. Tran'}, {'name': 'Robert Fragoza'}, {'name': 'Haiyuan Yu'}, {'name': 'John C. Schimenti'}]\", 'abstract': '<p>Infertility is a highly heterogeneous condition, with genetic causes estimated to be involved in approximately half of the cases. High-throughput sequencing (HTS) approaches are becoming an increasingly important tool for genetic diagnosis of diseases, including idiopathic infertility. Nevertheless, most rare or minor alleles revealed by HTS are classified as variants of uncertain significance (VUS). Interpreting the functional impacts of VUS is challenging but profoundly important for clinical management and genetic counseling. To determine the consequences of segregating polymorphisms in key fertility genes, we functionally evaluated 8 missense variants in the genes <italic>ANKRD31</italic>, <italic>BRDT</italic>, <italic>DMC1</italic>, <italic>EXOI</italic>, <italic>FKBP6</italic>, <italic>MSH4</italic> and <italic>SEPT12</italic> by generating genome-edited mouse models. Six variants were classified as deleterious by most functional prediction algorithms, and two disrupted a protein-protein interaction in the yeast 2 hybrid assay. Even though these genes are known to be essential for normal meiosis or spermiogenesis in mice, none of the tested human variants compromised fertility or gametogenesis in the mouse models. These results should be useful for genetic diagnoses of infertile patients, but they also underscore the need for more effective VUS categorization. To this end, we evaluated the performance of 10 widely used pathogenicity prediction algorithms in classifying missense variants within fertility-related genes from two sources: 1) the ClinVar database, and 2) those functionally tested in mice. We found that all the algorithms performed poorly in terms of predicting phenotypes of mouse-modeled variants. These studies highlight the importance of functional validation of potential infertility-causing genetic variants.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.12.463977', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'The Nup2 meiotic-autonomous region relieves autoinhibition of Nup60 to promote progression of meiosis and sporulation in Saccharomyces cerevisiae', 'authors': \"[{'name': 'Kelly Komachi'}, {'name': 'Sean M. Burgess'}]\", 'abstract': '<p>During meiosis, chromosomes undergo dramatic changes in structural organization, nuclear positioning, and motion. Although the nuclear pore complex has been shown to affect genome organization and function in vegetative cells, its role in meiotic chromosome dynamics has remained largely unexplored. Recent work in the budding yeast <italic>Saccharomyces cerevisiae</italic> demonstrated that the mobile nucleoporin Nup2 is required for normal progression through meiosis I prophase and sporulation in strains where telomere-led chromosome movement has been compromised. The meiotic autonomous region (MAR), a short fragment of Nup2 responsible for its role in meiosis, was shown to localize to the nuclear envelope via Nup60 and to bind to meiotic chromosomes. To understand the relative contribution these two activities have on MAR function, we first carried out a screen for MAR mutants defective in sporulation and found that all the mutations disrupt interaction with both Nup60 and meiotic chromosomes. Moreover, <italic>nup60</italic> mutants phenocopy <italic>nup2</italic> mutants, exhibiting similar nuclear division kinetics, sporulation efficiencies, and genetic interactions with mutations that affect the telomere bouquet. Although full-length Nup60 requires Nup2 for function, removal of Nup60s C-terminus allows Nup60 to bind meiotic chromosomes and promote sporulation without Nup2. In contrast, binding of the MAR to meiotic chromosomes is completely dependent on Nup60. Our findings uncover an inhibitory function for the Nup60 C-terminus and suggest that Nup60 mediates recruitment of meiotic chromosomes to the nuclear envelope, while Nup2 plays a secondary role counteracting Nup60s autoinhibition.</p>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.15.464515', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Spatial signatures of anesthesia-induced burst-suppression differ between primates and rodents', 'authors': \"[{'name': 'Nikoloz Sirmpilatze'}, {'name': 'Judith Mylius'}, {'name': 'Michael Ortiz-Rios'}, {'name': 'Jrgen Baudewig'}, {'name': 'Jaakko Paasonen'}, {'name': 'Daniel Golkowski'}, {'name': 'Andreas Ranft'}, {'name': 'Rdiger Ilg'}, {'name': 'Olli Grhn'}, {'name': 'Susann Boretius'}]\", 'abstract': '<p>During deep anesthesia, the electroencephalographic (EEG) signal of the brain alternates between bursts of activity and periods of relative silence (suppressions). The origin of burst-suppression and its distribution across the brain remain matters of debate. In this work, we used functional magnetic resonance imaging (fMRI) to map the brain areas involved in anesthesia-induced burst-suppression across four mammalian species: humans, long-tailed macaques, common marmosets, and rats. At first, we determined the fMRI signatures of burst-suppression in human EEG-fMRI data. Applying this method to animal fMRI datasets, we found distinct burst-suppression signatures in all species. The burst-suppression maps revealed a marked inter-species difference: in rats the entire neocortex engaged in burst-suppression, while in primates most sensory areas were excludedpredominantly the primary visual cortex. We anticipate that the identified species-specific fMRI signatures and whole-brain maps will guide future targeted studies investigating the cellular and molecular mechanisms of burst-suppression in unconscious states.</p>', 'posted': '2021-10-16'}, {'doi': '10.1101/2021.10.12.463740', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'RNAi screening of RING/U-box domain ubiquitin ligases identifies critical regulators of stem cells and tissue regeneration in planarians', 'authors': \"[{'name': 'John M. Allen'}, {'name': 'Madison Balagtas'}, {'name': 'Elizabeth Barajas'}, {'name': 'Carolina Cano Macip'}, {'name': 'Sarai Alvarez Zepeda'}, {'name': 'Ionit Iberkleid'}, {'name': 'Elizabeth M. Duncan'}, {'name': 'Ricardo M. Zayas'}]\", 'abstract': '<p>Regenerative processes depend on the interpretation of signals to coordinate cell behaviors. The role of ubiquitin-mediated signaling in regeneration is not well understood. To investigate how ubiquitylation might specifically impact tissue regeneration, we are studying planarians that are capable of regenerating from nearly any injury using a population of stem cells. Here we used RNAi to screen RING/U-box E3 ubiquitin ligases that are highly expressed in planarian stem cells and stem cell progeny. RNAi screening identified nine genes with functions in regeneration, including the spliceosomal factor <italic>prpf19</italic> and epigenetic regulator <italic>rnf2</italic>; based on roles in developmental processes, we further explored these two genes. We found that <italic>prpf19</italic> was required for survival but not for stem cell maintenance, suggesting a role in promoting cell differentiation. Because RNF2 is the catalytic subunit of the Polycomb Repressive Complex 1 (PRC1), we also examined other cofactors of <italic>rnf2</italic> and observed a striking phenotype of regional tissue misspecification in <italic>cbx</italic> and <italic>phc</italic> RNAi planarians. To identify genes regulated by PRC1, we performed RNA-seq after knocking down <italic>rnf2</italic> or <italic>phc</italic> and found that the set of genes differentially expressed were largely non-overlapping despite being predicted to function in the same complex. Using <italic>in situ</italic> hybridization, we showed that <italic>rnf2</italic> regulates gene expression levels within a tissue type, whereas <italic>phc</italic> is necessary for the spatial restriction of gene expression. This work uncovered roles for RING/U-box E3 ligases in stem cell regulation and regeneration and identified differential gene targets for PRC1 factors required for maintaining cell-type-specific gene expression in planarians.</p>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.08.21264719', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Intensive Care and Critical Care Medicine', 'covid': True, 'title': 'Trans Sodium Crocetinate (TSC) to Improve Oxygenation in COVID-19', 'authors': \"[{'name': 'Anca Streinu-Cercel'}, {'name': 'Oana Sndulescu'}, {'name': 'Victor Daniel Miron'}, {'name': 'Alina-Alexandra Oan'}, {'name': 'Maria Magdalena Motoi'}, {'name': 'Christopher D. Galloway'}, {'name': 'Adrian Streinu-Cercel'}]\", 'abstract': '<sec><title>Background</title><p>Trans Sodium Crocetinate (TSC) is a bipolar synthetic carotenoid under development as a drug to enhance oxygenation to hypoxic tissue in addition to standard of care. TSC acts via a novel mechanism of action, improving the diffusivity of oxygen in blood plasma. Thus, it is based on physical-chemical principles, unlike most drugs which are based on biochemistry-based mechanisms. We explored the use of escalating doses and multiple daily dosing of TSC as a potential therapeutic for patients suffering from hypoxemia due to SARS-CoV-2 infection.</p></sec><sec><title>Methods</title><p>Individuals 18 years who were hospitalized with confirmed SARS-CoV-2 infection and hypoxemia, defined as SpO<sub>2</sub> &lt; 94% on room air or requiring supplemental oxygen, WHO ordinal scale 3 through 7 (exclusive of Extra Corporeal Membrane Oxygenation [ECMO]) were enrolled in cohorts of six subjects, each of whom received the same dose (0.25, 0.5, 1.0, or 1.5 mg/kg) of TSC via intravenous bolus every 6 hours in addition to standard of care (SOC).</p><p>This report describes the safety and efficacy results from the lead-in phase of the study and the population pharmacokinetics (PK) analyses. Safety was assessed as the number of serious adverse events and dose-limiting toxicities (DLTs) observed with each dose. Several efficacy parameters were examined in the lead-in phase and descriptive statistics of efficacy parameters are provided. No formal statistical analyses were performed. The population PK analyses were based on previous analyses and examination of the concentration profiles, and two-compartment linear pharmacokinetic models were evaluated and validated. Covariates, including body size, age, sex, organ function, and dose level, were evaluated for inclusion into the model.</p></sec><sec><title>Results</title><p>TSC was well tolerated. There were no treatment emergent adverse events (TEAEs) reported. There were 2 serious adverse events (SAEs) reported during the study, neither were considered treatment-related. A total of 24 (96%) subjects survived. One subject (4.0%) died during the study as a result of an SAE (respiratory failure), and that event was determined to be due to COVID-19 complications and not related to study drug.</p><p>There was an observed reduction in the time to improvement in WHO Ordinal Scale with increasing dose. The median time to 1-point reduction in subjects receiving 0.25 mg/kg was 11.5 days versus 7.5 days in the 1.5 mg/kg treatment cohort. The overall range across all doses was 1 day to 28 days. A total of 36.0% of subjects had a 1-point improvement in WHO Ordinal Scale to Day 7. The 1.5 mg/kg dose resulted in observed superior outcomes for multiple secondary clinical outcomes: time to 1-point WHO Ordinal Score improvement through Day 29/discharge, 1-point improvement by Day 7, days to return to room air, and hospital length of stay.</p><p>The PK results showed that the two-compartment model fit the data well. Clearance decreased with increasing dose level and there was no evidence that clearance was affected by covariates other than dose level.</p></sec><sec><title>Conclusions</title><p>These findings suggest that TSC administration every 6 hours at doses up to 1.5 mg/kg for up to 15 days is safe and well tolerated with predictable pharmacokinetics and demonstrated an observed clinical benefit in the treatment of COVID-19-related hypoxemia.</p><p>(<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://ClinicalTrials.gov\">ClinicalTrials.gov</ext-link> number, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04573322\">NCT04573322</ext-link>)</p></sec>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.12.464107', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Topoisomerase II poisons inhibit vertebrate DNA replication through distinct mechanisms', 'authors': \"[{'name': 'Sabrina X. Van Ravenstein'}, {'name': 'Kavi P. Mehta'}, {'name': 'Tamar Kavlashvili'}, {'name': 'Jo Ann Byl'}, {'name': 'Runxiang Zhao'}, {'name': 'Neil Osheroff'}, {'name': 'David Cortez'}, {'name': 'James M. Dewar'}]\", 'abstract': '<p>Topoisomerase II (Top2) unlinks chromosomes during vertebrate DNA replication. Top2 poisons are widely-used chemotherapeutics that stabilize Top2 complexes on DNA, leading to cytotoxic DNA breaks. However, it is unclear how these drugs affect DNA replication, which is a major target of Top2 poisons. Using <italic>Xenopus</italic> egg extracts, we show that the Top2 poisons etoposide and doxorubicin both inhibit DNA replication through different mechanisms. Etoposide induces Top2-dependent DNA breaks and induces Top2-dependent fork stalling by trapping Top2 behind replication forks. In contrast, doxorubicin does not lead to appreciable break formation and instead intercalates into parental DNA to inhibit replication fork progression. In human cells, etoposide stalls replication forks in a Top2-dependent manner, while doxorubicin stalls forks independently of Top2. However, both drugs exhibit Top2-dependent cytotoxicity. Thus, despite shared genetic requirements for cytotoxicity etoposide and doxorubicin inhibit DNA replication through distinct mechanisms.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.11.463689', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': True, 'title': 'Mutating novel interaction sites in NRP1 reduces SARS-CoV-2 spike protein internalization', 'authors': \"[{'name': 'Debjani Pal'}, {'name': 'Kuntal De'}, {'name': 'Timothy B. Yates'}, {'name': 'Wellington Muchero'}]\", 'abstract': '<p>The global pandemic of Coronavirus disease 2019 caused by severe acute respiratory syndrome coronavirus 2 has become a severe global health problem because of its rapid spread(<italic>1</italic>). Both angiotensin-converting enzyme 2 and neuropilin 1 provide initial viral binding sites for SARS-CoV-2 (<italic>2, 3</italic>). Here, we show that three cysteine residues located in a1/a2 and b1 domains of neuropilin 1 are necessary for SARS-CoV-2 spike protein internalization in human cells. Mutating cysteines C82, C104 and C147 altered neuropilin 1 stability and binding ability as well as cellular internalization and lysosomal translocation of the spike protein. This resulted in up to 4 times reduction in spike protein load in cells for the original, alpha and delta SARS-CoV-2 variants even in the presence of the endogenous angiotensin-converting enzyme 2 receptor. Transcriptome analysis of cells transfected with mutated NRP1 revealed significantly reduced expression of genes involved in viral infection and replication, including eight members of the ribosomal protein L, ten members of ribosomal protein S and five members of the proteasome  subunit family proteins. We also observed higher expression of genes involved in suppression of inflammation and endoplasmic reticulum associated degradation. These observations suggest that these cysteines offer viable targets for therapies against COVID-19.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.07.21264710', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Nephrology', 'covid': False, 'title': 'ATP-citrate lyase as a therapeutic target in chronic kidney disease: a Mendelian Randomization analysis', 'authors': \"[{'name': 'Pedrum Mohammadi-Shemirani'}, {'name': 'Michael Chong'}, {'name': 'Nicolas Perrot'}, {'name': 'Marie Pigeyre'}, {'name': 'Gregory R. Steinberg'}, {'name': 'Guillaume Par'}, {'name': 'Joan C. Krepinsky'}, {'name': 'Matthew B. Lanktree'}]\", 'abstract': '<sec><title>Background</title><p>ATP-citrate lyase (ACLY) inhibition is a promising therapeutic target for dyslipidemia, atherosclerotic cardiovascular disease, non-alcoholic steatohepatitis, and metabolic syndrome. Genetic analysis of its role in chronic kidney disease (CKD) has not been performed.</p></sec><sec><title>Methods</title><p>We constructed a genetic instrument by selecting variants associated with <italic>ACLY</italic> expression level in the expression quantitative trait loci genetics consortium (eQTLGen) that includes blood samples from 31,684 participants. In a two-sample Mendelian randomization analysis, we then evaluated the effect of genetically predicted <italic>ACLY</italic> expression on risk of CKD, estimated glomerular filtration rate (eGFR), and microalbuminuria using the CKD Genetics consortium (CKDGen), United Kingdom biobank, and the Finnish Genetics consortium (FinnGen) totaling 66,396 CKD cases and 958,517 controls.</p></sec><sec><title>Results</title><p><italic>ACLY</italic> is constitutively expressed in all cell types including in whole blood. The genetic instrument included 13 variants and explained 1.5% of variation in whole blood <italic>ACLY</italic> gene expression. A 34% reduction in genetically predicted <italic>ACLY</italic> expression was associated with a 0.04 mmol/L reduced low-density lipoprotein cholesterol (<italic>P</italic> = 3.4  10<sup>4</sup>) and a 9% reduced risk of CKD (stage 3,4,5, dialysis or eGFR below 60 ml/min/1.73m<sup>2</sup>) (OR = 0.91, 95% C.I. 0.85-0.98, <italic>P</italic> = 0.008), but no association was observed with eGFR nor microalbuminuria.</p></sec><sec><title>Conclusion</title><p>Mendelian Randomization analysis provides cautious optimism regarding the possibility of ACLY as a therapeutic target for CKD.</p></sec>', 'posted': '2021-10-11'}, {'doi': '10.1101/2021.10.13.464316', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'CAR modulates plasma membrane nano-organization and immune signaling downstream of RALF1-FERONIA ligand-receptor signaling system', 'authors': \"[{'name': 'Weijun Chen'}, {'name': 'Huina Zhou'}, {'name': 'Fan Xu'}, {'name': 'Meng Yu'}, {'name': 'Alberto Coego'}, {'name': 'Lesia Rodriguez'}, {'name': 'Yuqing Lu'}, {'name': 'Qijun Xie'}, {'name': 'Qiong Fu'}, {'name': 'Jia Chen'}, {'name': 'Lan Li'}, {'name': 'Guoyun Xu'}, {'name': 'Dousheng Wu'}, {'name': 'Xiushan Li'}, {'name': 'Shaojun Dai'}, {'name': 'Xiaojuan Li'}, {'name': 'Yvon Jaillais'}, {'name': 'Pedro L. Rodriguez'}, {'name': 'Feng Yu'}, {'name': 'Sirui Zhu'}]\", 'abstract': '<p>The highly selective segregation of molecules at the plasma membrane (PM) to form domains is a widespread phenomenon. But how distinct signaling platforms are established and maintained by specific signals remains unclear. Here, we show that the receptor of Rapid Alkalinization Factor 1 (RALF1), the FERONIA (FER) receptor kinase, physically interacts with C2 domain ABA-related (CAR) proteins to control the nano-organization of the PM and to regulate extracellular signal transduction in Arabidopsis. During this process, the RALF1-FER pathway upregulates CAR protein synthesis, which then controls the membrane lipid order. This might act as a rapid feedforward loop to stabilize FER in PM nanodomains. FER then interacts with and phosphorylates CARs, reducing their lipid-binding ability, which might break the feedback regulation and downregulate CAR activity at latter time point. Similar to fer mutant, a pentuple car14569 mutant inhibits flg22-induced FLS2-BAK1 complex formation, which ultimately impacts plant immunity. Together, we proposed that the FER-CAR module controls the dynamics of the PM nano-organization during RALF signaling through a self-contained amplifying loop including both positive and negative feedbacks.</p>', 'posted': '2021-10-15'}, {'doi': '10.1101/2021.10.17.464158', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Distinct phases within large RNP granules of C. elegans oocytes are associated with differential stress responses of RNA binding proteins', 'authors': \"[{'name': 'Brooklynne Watkins'}, {'name': 'Mohamed T. Elaswad'}, {'name': 'Chloe Pestrue'}, {'name': 'Katherine Sharp'}, {'name': 'Elizabeth Breton'}, {'name': 'Jennifer A Schisa'}]\", 'abstract': '<p>One emerging paradigm of cellular organization of RNA and RNA binding proteins is the formation of membraneless organelles (MLOs). Examples of MLOs include several types of RNP (ribonucleoprotein) granules that form via phase separation. Proper regulation of the phase transitions of RNA binding proteins is critical as dysregulation can lead to disease states. Germ granules are small RNP granules conserved across metazoa. In <italic>C. elegans</italic>, when oogenesis undergoes an extended meiotic arrest, germ granules assemble into much larger, more complex RNP granules whose hypothesized function is to regulate RNA metabolism and maintain oocyte quality. As a step towards gaining insight into the function of RNP granules, in this report we characterize distinct phases for four RNA binding proteins in arrested oocytes. We find that PGL-1 is dynamic and has liquid-like properties, while MEG-3 has gel-like properties, both similar to their properties in small germ granules of embryos. We also show that MEX-3 exhibits gel-like properties in many regards but is more dynamic than MEG-3. We find CGH-1 is dynamic but does not consistently behave liquid-like, and may be an intermediate phase within RNP granules. We also show that the distinct phases of the RNA binding proteins are associated with differential responses to stress.</p>', 'posted': '2021-10-17'}, {'doi': '10.1101/2021.10.16.464638', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Ionizing radiation improves RIG-I mediated immunotherapy through enhanced p53 activation in malignant melanoma', 'authors': \"[{'name': 'Silke Lambing'}, {'name': 'Stefan Holdenrieder'}, {'name': 'Patrick Mller'}, {'name': 'Christian Hagen'}, {'name': 'Stephan Garbe'}, {'name': 'Martin Schlee'}, {'name': 'Jasper G. van den Boorn'}, {'name': 'Eva Bartok'}, {'name': 'Gunther Hartmann'}, {'name': 'Marcel Renn'}]\", 'abstract': '<p>The activation of the innate immune receptor RIG-I is a promising approach in immunooncology and currently under investigation in clinical trials. RIG-I agonists elicit a strong immune activation in both tumor and immune cells and induce both direct and indirect immune cell-mediated tumor cell death which involves tumor-specific cytotoxic T-cell response and type I interferon-driven innate cytotoxic immunity. Besides RIG-I, irradiation is known to induce cytotoxic DNA damage resulting in tumor debulking followed by the induction of tumor-specific immunity. To date, it is unclear whether the molecular antitumor effects of RIG-I and irradiation are additive or even synergize. Here, we investigated the combination of RIG-I activation with radiotherapy in melanoma. We found that low dose x-ray irradiation enhanced the extent and immunogenicity of RIG-I mediated tumor cell death in human and murine melanoma cell lines and in the murine B16 melanoma model in vivo. Pathway analysis of transcriptomic data revealed a central role for p53 downstream of the combined treatment, which was corroborated using p53-/- B16 cells. In vivo, the additional effect of irradiation on immune cell activation and inhibition of tumor growth was lost in mice carrying p53-knockout B16 tumors, while the response to RIG-I stimulation in those mice was maintained. Thus, our results identify p53 as pivotal for the synergy of RIG-I with irradiation, resulting in potent induction of immunogenic tumor cell death. Consequently, low dose radiotherapy holds great promise to further improve the efficacy or RIG-I ligands especially in patients with malignant melanoma or other tumors exhibiting a functional p53 pathway.</p>', 'posted': '2021-10-16'}, {'doi': '10.1101/2021.10.17.463719', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'The quantitative contribution of different Photosystem II compartments to non-photochemical quenching in Arabidopsis', 'authors': \"[{'name': 'Lauren Nicol'}, {'name': 'Vincenzo Mascoli'}, {'name': 'Herbert van Amerongen'}, {'name': 'Roberta Croce'}]\", 'abstract': '<p>Excess excitation energy in the light-harvesting antenna of Photosystem II (PSII) can cause irreversible damage to the photosynthetic apparatus. In periods of high light intensity, a feedback mechanism known as non-photochemical quenching (NPQ), induces the formation of quenchers which can safely dissipate excess excitation energy as heat. Although quenchers have been identified in more than one compartment of the PSII supercomplex, there is currently no quantitative description of how much NPQ is occurring at each of these locations. Here, we perform time-resolved fluorescence measurements on WT and antenna mutants lacking LHCII (NoL) and all peripheral antenna (Ch1 and Ch1lhcb5). By combining the results with those of steady-state fluorescence experiments we are able to estimate the intrinsic rate of NPQ for each plant and each PSII compartment. It is concluded that 60-70% of quenching occurs in LHCII, 15-20% in the minor antenna and 15-20% in the PSII core.</p>', 'posted': '2021-10-17'}, {'doi': '10.1101/2021.10.17.464727', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'T7Max transcription system', 'authors': \"[{'name': 'Christopher Deich'}, {'name': 'Brock Cash'}, {'name': 'Wakana Sato'}, {'name': 'Judee Sharon'}, {'name': 'Lauren Aufdembrink'}, {'name': 'Nathaniel J Gaut'}, {'name': 'Joseph Heili'}, {'name': 'Kaitlin Stokes'}, {'name': 'Aaron E Engelhart'}, {'name': 'Katarzyna P Adamala'}]\", 'abstract': '<p>Efficient cell-free protein expression from linear DNA templates has remained a challenge primarily due to template degradation. Here we present a modified T7 RNA polymerase promoter that acts to significantly increase the yields of both transcription and translation within in vitro systems. The modified promoter, termed T7Max, recruits standard T7 RNA polymerase, so no protein engineering is needed to take advantage of this method. This technique could be used with any T7 RNA polymerase- based in vitro protein expression system. Unlike other methods of limiting linear template degradation, the T7Max promoter increases transcript concentration in a T7 transcription reaction, providing more mRNA for translation.</p>', 'posted': '2021-10-17'}, {'doi': '10.1101/2021.10.17.464590', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'TRPV1-lineage somatosensory fibers communicate with taste neurons in the mouse parabrachial nucleus', 'authors': \"[{'name': 'Jinrong Li'}, {'name': 'Md Sams Sazzad Ali'}, {'name': 'Christian H Lemon'}]\", 'abstract': '<p>Trigeminal neurons supply somatosensation to craniofacial tissues. In mouse brain, ascending projections from medullary trigeminal neurons arrive at taste neurons in the autonomic parabrachial nucleus, suggesting taste neurons participate in somatosensory processing. However, the genetic cell types that support this convergence were undefined. Using Cre-directed optogenetics and in vivo neurophysiology in anesthetized mice of both sexes, here we studied whether TRPV1-lineage nociceptive and thermosensory fibers are primary neurons that drive trigeminal circuits reaching parabrachial taste cells. We monitored spiking activity in individual parabrachial neurons during photoexcitation of the terminals of TRPV1-lineage fibers that arrived at the dorsal spinal trigeminal nucleus pars caudalis, which relays orofacial somatosensory messages to the parabrachial area. Parabrachial neural responses to oral delivery of taste, chemesthetic, and thermal stimuli were also recorded. We found that optical excitation of TRPV1-lineage fibers frequently stimulated traditionally defined taste neurons in lateral parabrachial nuclei. The tuning of neurons across diverse tastes associated with their sensitivity to excitation of TRPV1-lineage fibers, which only sparingly engaged neurons oriented to preferred tastes like sucrose. Moreover, neurons that responded to photostimulation of TRPV1-lineage afferents showed strong responses to temperature including noxious heat, which predominantly excited parabrachial bitter taste cells. Multivariate analyses revealed the parabrachial confluence of TRPV1-lineage signals with taste captured sensory valence information shared across aversive gustatory, nociceptive, and thermal stimuli. Our results reveal that trigeminal fibers with defined roles in thermosensation and pain communicate with parabrachial taste neurons. This multisensory convergence supports dependencies between gustatory and somatosensory hedonic representations in the brain.</p>', 'posted': '2021-10-17'}, {'doi': '10.1101/2021.10.09.21264758', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'The military as a neglected pathogen transmitter and its implications for COVID-19: A systematic review', 'authors': \"[{'name': 'Claudia Chaufan'}, {'name': 'Ilinca A. Dutescu'}, {'name': 'Hanah Fekre'}, {'name': 'Saba Marzabadi'}, {'name': 'K.J. Noh'}]\", 'abstract': '<sec><title>Background</title><p>The risk of outbreaks escalating into pandemics has soared with globalization. Therefore, understanding transmission mechanisms of infectious diseases has become critical to formulating global public health policy. This systematic review assessed the evidence for the military as a disease vector, an historically relevant one, yet overlooked in times of COVID-19.</p></sec><sec><title>Methods</title><p>We searched 3 electronic databases without temporal restrictions. We identified 2010 of 6477 studies spanning over two centuries (1810  2020) that met our inclusion criteria and provided evidence for the military as a pathogen transmitter, within itself or between it and civilians. Two researchers independently extracted study data using a standardized form. Through team discussions, studies were grouped according to their type of transmission mechanism and direct quotes were extracted to generate themes and sub-themes. A content analysis was later performed and frequency distributions for each theme were generated.</p></sec><sec><title>Results</title><p>Biological mechanisms driving transmission included person-to-person transmission, contaminated food and water, vector-borne, and airborne routes. Social mechanisms facilitating transmission included crowded living spaces, unhygienic conditions, strenuous working, training conditions, absent or inadequate vaccination programs, pressure from military leadership, poor compliance with public health advice, contractor mismanagement, high-risk behaviours, and occupation-specific freedom of movement. Contaminated food and/or water was the most common biological transmission route. Living conditions were the most common social transmission mechanism, with young, low ranking military personnel repeatedly reported as the most affected group. Certain social mechanisms, such as employment-related freedom of movement, were unique to the military as a social institution. While few studies explicitly studied civilian populations, considerably more contained information that implied that civilians were likely impacted by outbreaks described in the military.</p></sec><sec><title>Conclusions</title><p>Features of the military identified in this study pose a significant public health threat, especially to countries with substantial military presence or underdeveloped health systems. Many social transmission mechanisms, unlike biological ones, were unique to the military, facilitating large-spreader events and affecting civilian health. As an increasingly interconnected world faces the challenges of COVID-19 and future infectious diseases, the identified features of the military may exacerbate current and similar challenges and impair attempts to implement successful and equitable pandemic policies.</p></sec>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.06.459411', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'GEOexplorer: an R/Bioconductor package for gene expression analysis and visualisation', 'authors': \"[{'name': 'Guy P Hunt'}, {'name': 'Rafael Henkin'}, {'name': 'Fabrizio Smeraldi'}, {'name': 'Michael R Barnes'}]\", 'abstract': '<sec><title>Background</title><p>Over the past three decades there have been numerous molecular biology developments that have led to an explosion in the number of gene expression studies being performed. Many of these gene expression studies publish their data to the public database GEO, making them freely available. By analysing gene expression datasets, researchers can identify genes that are differentially expressed between two groups. This can provide insights that lead to the development of new tests and treatments for diseases. Despite the wide availability of gene expression datasets, analysing them is difficult for several reasons. These reasons include the fact that most methods for performing gene expression analysis require programming proficiency.</p></sec><sec><title>Results</title><p>We developed the GEOexplorer software package to overcome several of the difficulties in performing gene expression analysis. GEOexplorer was therefore developed as a web application, that can perform interactive and reproducible microarray gene expression analysis, while producing a wealth of interactive visualisations to facilitate result exploration. GEOexplorer is implemented in R using the Shiny framework and is fully integrated with the existing core structures of the Bioconductor project. Users can perform the essential steps of exploratory data analysis and differential gene expression analysis intuitively and generate a broad spectrum of publication ready outputs.</p></sec><sec><title>Conclusion</title><p>GEOexplorer is distributed as an R package in the Bioconductor project (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://bioconductor.org/packages/GEOexplorer/\">http://bioconductor.org/packages/GEOexplorer/</ext-link>). GEOexplorer provides a solution for performing interactive and reproducible analyses of microarray gene expression data, empowering life scientists to perform exploratory data analysis and differential gene expression analysis on GEO microarray datasets.</p></sec>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.12.464059', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'New cycle, same old mistakes? Overlapping vs. discrete generations in long-term recurrent selection', 'authors': \"[{'name': 'Marlee R. Labroo'}, {'name': 'Jessica E. Rutkoski'}]\", 'abstract': '<sec><title>Background</title><p>Recurrent selection is a foundational breeding method for quantitative trait improvement. It typically features rapid breeding cycles that can lead to high rates of genetic gain. In recurrent phenotypic selection, generations do not overlap, which means that breeding candidates are evaluated and considered for selection for only one cycle. With recurrent genomic selection, candidates can be evaluated based on genomic estimated breeding values indefinitely, therefore facilitating overlapping generations. Candidates with true high breeding values that were discarded in one cycle due to underestimation of breeding value could be identified and selected in subsequent cycles. The consequences of allowing generations to overlap in recurrent selection are unknown. We assessed whether maintaining overlapping and discrete generations led to differences in genetic gain for phenotypic, genomic truncation, and genomic optimum contribution recurrent selection by simulation of traits with various heritabilities and genetic architectures across fifty breeding cycles. We also assessed differences of overlapping and discrete generations in a conventional breeding scheme with multiple stages and cohorts.</p></sec><sec><title>Results</title><p>With phenotypic selection, overlapping generations led to decreased genetic gain compared to discrete generations due to increased selection error bias. Selected individuals, which were in the upper tail of the distribution of phenotypic values, tended to also have high absolute error relative to their true breeding value compared to the overall population. Without repeated phenotyping, these individuals erroneously believed to have high value were repeatedly selected across cycles, leading to decreased genetic gain. With genomic truncation selection, overlapping and discrete generations performed similarly as updating breeding values precluded repeatedly selecting individuals with inaccurately high estimates of breeding values in subsequent cycles. Overlapping generations did not outperform discrete generations in the presence of a positive genetic trend with genomic truncation selection, as past generations had lower mean genetic values than the current generation of selection candidates. With genomic optimum contribution selection, overlapping and discrete generations performed similarly, but overlapping generations slightly outperformed discrete generations in the long term if the targeted inbreeding rate was extremely low.</p></sec><sec><title>Conclusions</title><p>Maintaining discrete generations in recurrent phenotypic selection leads to increased genetic gain, especially at low heritabilities, by preventing selection error bias. With genomic truncation selection and genomic optimum contribution selection, genetic gain does not differ between discrete and overlapping generations assuming non-genetic effects are not present. Overlapping generations may increase genetic gain in the long term with very low targeted rates of inbreeding in genomic optimum contribution selection.</p></sec>', 'posted': '2021-10-14'}, {'doi': '10.1101/2021.10.10.463866', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'STAG2 promotes the myelination transcriptional program in oligodendrocytes', 'authors': \"[{'name': 'Ningyan Cheng'}, {'name': 'Mohammed Kanchwala'}, {'name': 'Bret M. Evers'}, {'name': 'Chao Xing'}, {'name': 'Hongtao Yu'}]\", 'abstract': '<p>Cohesin folds chromosomes via DNA loop extrusion. Cohesin-mediated chromosome loops regulate transcription by shaping long-range enhancer-promoter interactions, among other mechanisms. Mutations of cohesin subunits and regulators cause human developmental diseases termed cohesinopathy. Vertebrate cohesin consists of SMC1, SMC3, RAD21, and either STAG1 or STAG2. To probe the physiological functions of cohesin, we created conditional knockout (cKO) mice with <italic>Stag2</italic> deleted in the nervous system. <italic>Stag2</italic> cKO mice exhibit growth retardation, neurological defects, and premature death, in part due to insufficient myelination of nerve fibers. <italic>Stag2</italic> cKO oligodendrocytes exhibit delayed maturation and downregulation of myelination-related genes. <italic>Stag2</italic> loss reduces promoter-anchored loops at downregulated genes in oligodendrocytes. Thus, STAG2-cohesin generates promoter-anchored loops at myelination-promoting genes to facilitate their transcription. Our study implicates defective myelination as a contributing factor to cohesinopathy and establishes oligodendrocytes as a relevant cell type to explore the mechanisms by which cohesin regulates transcription.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.16.464670', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Old and ancient trees are life history lottery winners and act as evolutionary buffers against long-term environmental change', 'authors': \"[{'name': 'Charles H Cannon'}, {'name': 'Gianluca Piovesan'}, {'name': 'Sergi Munne-Bosch'}]\", 'abstract': '<p>Trees can live many centuries with sustained fecundity and death is largely stochastic. We use a neutral stochastic model to examine the demographic patterns that emerge over time, across a range of population sizes and empirically observed mortality rates. A small proportion of trees (~1% at 1.5% mortality) are life-history lottery winners, achieving ages &gt;10-20x median age. Maximum age increases with bigger populations and lower mortality rates. One quarter of trees (~24%) achieve ages that are 3-4 times greater than median age. Three age classes (Mature, Old, and Ancient) contribute unique historical diversity across complex environmental cycles. Ancient trees are an emergent property of forests that requires many centuries to generate. They radically change generation time variance and population fitness, bridging infrequent environmental cycles. These life-history lottery winners are vital to future forest dynamics and invaluable data about environmental history and individual longevity. Old-growth forests contain trees that cannot be replaced through restoration or regeneration in the near future. They simply must be protected to preserve their unique diversity.</p>', 'posted': '2021-10-17'}, {'doi': '10.1101/2021.10.16.464646', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Pharmacological mechanisms on the anti-breast cancer property of Coix lacryma-jobi: A network-based analysis', 'authors': \"[{'name': 'Angelu Mae A Ferrer'}, {'name': 'Janella Rayne A David'}, {'name': 'Arvin A Taquiqui'}, {'name': 'Arci A Bautista'}, {'name': 'Custer C Deocaris'}, {'name': 'Malona Velasco Alinsug'}]\", 'abstract': '<p>Breast cancer is considered as one of the three most common cancers around the world and the second leading cause of cancer deaths among women. Coix lachrymal jobi, commonly known as Jobs tears or adlay has been reported to possess anti-cancer properties. Despite evidences provided by clinical data, the usage of Coix lacryma-jobi in treating cancer, particularly breast cancer, has been scarce. Thus, this study was conducted to determine the pharmacological mechanisms underlying its anti-breast cancer property using various network pathway analyses. Bioactive compounds from Coix lacryma-jobi and its potential target genes were obtained from SymMap. Breast cancer-related target genes were collected from CTD. Protein-protein interaction network was analyzed using the STRING database. GO and KEGG pathway enrichment analyses were performed using DAVID to further explore the mechanisms of Coix lacryma-jobi in treating breast cancer. PPI and compound-target-pathway were visualized using Cytoscape. A total of 26 bioactive compounds, 201 corresponding targets, 36625 breast cancer-associated targets were obtained, and 200 common targets were considered potential therapeutic targets. The 9 bioactive compounds identified were berberine, oleic acid, beta-sitosterol, sitosterol, linoleic acid, berberrubine, jatrorrhizine, thalifendine, and stigmasterol. The identified 5 core targets were ESR1, JUN, MAPK14, and RXRA.  Coix lacryma-jobi targets enriched in GO terms were mostly involved in regulation of transcription from RNA polymerase II promoter, drug response, steroid hormone receptor activity, and protein binding. This study elucidates on the pharmacological underpinnings on the potency of adlay against breast cancer.  Its subsequent drug development will be worth a step forward for a breast cancer-free society.</p>', 'posted': '2021-10-16'}, {'doi': '10.1101/2021.10.11.463729', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Deep Learning Genome-wide Linkage Association Study for Wheat Fusarium Head Blight Resistance Genes Discovery', 'authors': \"[{'name': 'Wayne Xu'}, {'name': 'Andriy Bilichak'}, {'name': 'Raman Dhariwal'}, {'name': 'Maria A. Henriquez'}, {'name': 'Harpinder Randhawa'}]\", 'abstract': '<sec><title>Background</title><p><italic>Fusarium</italic> head blight (FHB) is one of the most devastating diseases of wheat worldwide and artificial intelligence can assist with understanding resistance to the disease. Considering different sample populations, marker types, reference maps, and statistical methods, we developed a Deep Learning Genome-wide Linkage Association Study (dpGLAS) of FHB resistance in wheat.</p></sec><sec><title>Results</title><p>The dpGLAS was first applied to two bi-parental population datasets in which the cultivar AC Barrie was a common parent for FHB resistance. Eight candidate gene markers were discovered in the one AC Barrie population and 10 in the other associated with FHB resistance. Eight of these markers were also supported by the conventional QTL mapping. Most of these candidate marker genes were found associated with the Reactive Oxygen Species (ROS) and Abscisic acid (ABA) axes. These ROS and ABA pathways were further supported by RNA-seq transcriptome data of FHB resistant cv. AAC Tenacious, a parent of the third bi-parental population. In this dataset, the ROS-centered Panther protein families were significantly enriched in those genes that had most different response to FHB when compared the resistance Tenacious and the susceptible Roblin.</p></sec><sec><title>Conclusions</title><p>This study developed the framework of dpGLAS and identified candidate genes for FHB resistance in the Canadian spring wheat cultivars AC Barrie and AAC Tenacious.</p></sec>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.11.463917', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Secondary structure of subgenomic RNA M of SARS-CoV-2', 'authors': \"[{'name': 'Marta Soszynska-Jozwiak'}, {'name': 'Ryszard Kierzek'}, {'name': 'Elzbieta Kierzek'}]\", 'abstract': '<p>SARS-CoV-2 belongs the <italic>Coronavirinae</italic> family. As other coronaviruses, SARS-CoV-2 is enveloped and possesses positive-sense, single-stranded RNA genome of  30 kb. Genome RNA is used as the template for replication and transcription. During these processes, positive-sense genomic RNA (gRNA) and subgenomic RNAs (sgRNAs) are created. Several studies showed importance of genomic RNA secondary structure in SARS-CoV-2 replication. However, the structure of sgRNAs have remained largely unsolved so far. In this study, we performed probing of sgRNA M of SARS-CoV-2 <italic>in vitro</italic>. This is the first experimentally informed secondary structure model of sgRNA M, which presents features likely to be important in sgRNA M function. The knowledge about sgRNA M provides insights to better understand virus biology and could be used for designing new therapeutics.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.11.464003', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A bacterial Argonaute with efficient DNA and RNA cleavage activity guided by small DNA and RNA', 'authors': \"[{'name': 'Longyu Wang'}, {'name': 'Xiaochen Xie'}, {'name': 'Yang Liu'}, {'name': 'Wenqiang Li'}, {'name': 'Bin Lv'}, {'name': 'Zhiwei Zhang'}, {'name': 'Jun Yang'}, {'name': 'Guangbo Yan'}, {'name': 'Wanping Chen'}, {'name': 'Cheng Zhang'}, {'name': 'Fei Wang'}, {'name': 'Lixin Ma'}]\", 'abstract': '<p>Argonaute proteins are widespread in prokaryotes and eukaryotes. Most prokaryotic Argonaute proteins (pAgos) use 5P-gDNA to target complementary DNA. However, more and more studies on the properties of pAgos make their functions more diversified. Previously reported pAgos only possess several forms of high activity in all eight cleavage patterns, which limits their practical applications. Here, we described a unique pAgo from <italic>Marinitoga hydrogenitolerans</italic> (MhAgo) with eight cleavage activities. MhAgo can utilize all four types of guides (5OH-gDNA, 5P-gDNA, 5OH-gRNA, and 5P-gRNA) for ssDNA and RNA cleavage. Further studies demonstrated that MhAgo had high activities with 16-21 nt guides and no obvious preferences for the 5-end nucleotides of 5OH-guides. Unexpectedly, MhAgo had different preferences for the 5-end nucleotides of 5P-guides depending on the types of targets. Although the specificity of MhAgo was related to the types of guides, single mismatches in the central and 3-supplementary regions of guides greatly reduced the cleavage efficiency. Additionally, the electrophoretic mobility shift assay (EMSA) demonstrated MhAgo had the weakest affinity for 5P-gRNA:tRNA duplex, which was consistent with its cleavage efficiency. In conclusion, MhAgo is highly active under a wide range of conditions and can be used for programmable endonucleolytic cleavage of both ssDNA and RNA substrates. The abundant biochemical characteristics of MhAgo broaden our understanding of pAgos and expand the potential application in nucleic acids manipulations.</p>', 'posted': '2021-10-12'}, {'doi': '10.1101/2021.10.12.464058', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Control of protein activity by photoinduced spin polarized charge reorganization', 'authors': \"[{'name': 'Shirsendu Ghosh'}, {'name': 'Koyel Banerjee-Ghosh'}, {'name': 'Dorit Levy'}, {'name': 'David Scheerer'}, {'name': 'Inbal Riven'}, {'name': 'Jieun Shin'}, {'name': 'Harry B. Gray'}, {'name': 'Ron Naaman'}, {'name': 'Gilad Haran'}]\", 'abstract': '<p>Biomolecules within the living cell are subject to extensive electrical fields, particularly next to membranes. Indeed, a role for bioelectricity has been well established at the organismal level. While the importance of electrostatics in protein functions such as protein-protein association and enzymatic activity has been well documented, very little is known on how biomolecules respond to external electric fields, or in other words, what may be the potential contribution of polarizability to protein function. Here we use phototriggered charge injection from a site-specifically attached ruthenium photosensitizer to directly demonstrate the effects of charge redistribution within a protein. We find that binding of an antibody to phosphoglycerate kinase (PGK) is increased two folds under illumination. Remarkably, illumination is found to suppress the enzymatic activity of PGK by a factor as large as three. These responses are sensitive to the photosensitizer position on the protein. Surprisingly, left (but not right) circularly polarized light elicits these responses, indicating that the electrons involved in the observed dynamics are spin polarized, due to spin filtration by protein chiral structures. Our results directly establish the contribution of electrical polarization and the importance of the spin-dependent charge reorganization in the function of proteins. Future experiments with phototriggered charge injection will allow delineation of charge rearrangement pathways within proteins and will further depict their effects on protein function.</p>', 'posted': '2021-10-13'}, {'doi': '10.1101/2021.10.12.464079', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'A multi-omics approach identifies pancreatic cancer cell extracellular vesicles as mediators of the unfolded protein response in normal pancreatic epithelial cells', 'authors': \"[{'name': 'Charles P. Hinzman'}, {'name': 'Shivani Bansal'}, {'name': 'Yaoxiang Li'}, {'name': 'Anton Iliuk'}, {'name': 'Michael Girgis'}, {'name': 'Baldev Singh'}, {'name': 'Kelly M. Herremans'}, {'name': 'Jose G. Trevino'}, {'name': 'Vijay K. Singh'}, {'name': 'Partha P. Banerjee'}, {'name': 'Amrita K. Cheema'}]\", 'abstract': '<p>Although cancer-derived extracellular vesicles (cEVs) are thought to play a pivotal role in promoting cancer progression events, their precise effect on neighboring normal cells is unknown. In this study, we investigated the impact of pancreatic cancer ductal adenocarcinoma (PDAC) derived EVs on recipient non-tumorigenic pancreatic normal epithelial cells upon internalization. We show that PDAC cEVs increase the proliferation and invasive capability of treated normal cells. We further demonstrate that cEVs induce endoplasmic reticulum (ER) stress and the unfolded protein response (UPR) in treated normal pancreatic epithelial cells within 24 hours. Subsequently, these cells release several inflammatory cytokines. Leveraging a layered multi-omics approach, we analyzed EV cargo from a panel of 6 PDAC and 2 normal pancreas cell lines, using multiple EV isolation methods. We found that cEVs were enriched for an array of biomolecules which can induce or regulate ER stress and the UPR, including palmitic acid, sphingomyelins, metabolic regulators of tRNA charging and proteins which regulate trafficking and degradation. We further show that palmitic acid, at doses relevant to those found in cEVs, is sufficient to induce ER stress in normal pancreas cells. These results suggest that cEV cargo packaging may be designed to disseminate proliferative and invasive characteristics upon internalization by distant recipient normal cells, hitherto unreported. This study is among the first to highlight a major role for PDAC cEVs to induce stress in treated normal pancreas cells that may modulate a systemic response leading to altered phenotypes. For the first time, our study implicates cEV transported palmitic acid as a potential driver in this process. These findings highlight the importance of EVs in mediating disease etiology and open potential areas of investigation toward understanding the role of cEV lipids in promoting cell transformation in the surrounding microenvironment.</p>', 'posted': '2021-10-14'}, {'doi': '10.1101/2021.10.14.464379', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'The -amylase7 gene in Zea mays encodes a protein with structural and catalytic properties similar to Arabidopsis BAM2', 'authors': \"[{'name': 'Claire M Ravenburg'}, {'name': 'McKayla B Riney'}, {'name': 'Jonathan D Monroe'}, {'name': 'Christopher E Berndsen'}]\", 'abstract': '<p>Starch accumulates in the plastids of green plant tissue during the day to provide carbon for metabolism at night. Starch hydrolysis is catalyzed by members of the -amylase (BAM) family, which in Arabidopsis thaliana (At), includes nine structurally and functionally diverse members. One of these enzymes, AtBAM2, is a plastid-localized enzyme that is unique among characterized -amylases since it is tetrameric and exhibits sigmoidal kinetics. Sequence alignments show that the BAM domains of AtBAM7, a catalytically inactive, nuclear-localized transcription factor with an N-terminal DNA binding domain, and AtBAM2 are more closely related to each other than they are to any other AtBAM. Since BAM2 is found in more ancient lineages, it was hypothesized that BAM7 evolved from BAM2. However, analysis of the genomes of 48 flowering plants revealed 12 species that appear to have a BAM7 gene but lack a BAM2 gene. Upon closer inspection, these BAM7 proteins have a greater percent identity to AtBAM2 than to AtBAM7, and they share all of the AtBAM2 functional residues that BAM7 proteins normally lack. We hypothesize that these genes may encode a BAM2-like protein although they are currently annotated as BAM7-like genes. To test this hypothesis, we designed a cDNA of the short form of corn BAM7 (ZmBAM7-S) for expression in E. coli. Small Angle X-Ray Scattering data indicate that ZmBAM7-S has a tetrameric solution structure more similar to that of AtBAM2 than AtBAM1. In addition, partially purified ZmBAM7-S is catalytically active and exhibits sigmoidal kinetics. Together these data suggest that some BAM7 genes may encode a functional BAM2. Exploring and understanding -amylase gene structure could have impacts on the current annotation of genes.</p>', 'posted': '2021-10-15'}, {'doi': '10.1101/2021.10.15.464501', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Identification of binding interaction between Curcumin derivative and PERK13 Proline rich receptor like protein kinase protein using In silico docking techniques  to help plants tackle salt water', 'authors': \"[{'name': 'Prakash Angadipuram Vaithyanathan'}]\", 'abstract': '<p>Introduction: Arabidopsis thaliana, mouse ear cress or thale cress are small flowering plants included in the cruciferae family. They comprise of various characteristics such as diploid genetics, small genome size, rapid growth cycle, and relatively low repetitive DNA content, making it a perfect model for plant genome projects. Objective: The aim of this present Insilico research study is to carry out molecular drug docking studies between PERK13  Proline rich receptor like protein kinase and De O Acetylated Curcumin Di Galactose or digalactosylated curcumin, a derivative of curcumin. PERK13 protein is considered to play a significant role in helping plants tackle salinity levels in water. Methods: In this study, protein modelling tools and servers are used to model the 3D structure and the same is validated using Protein structure validation tools. Automated drug docking servers were used to dock the modelled protein with the chemical compound to analyse the electrostatic (H bond) interaction between PERK13 and De O Acetylated Curcumin Di Galactose, a better water soluble compound than curcumin. The docked structure was visualized using an advanced molecular visualization tool. Results and Discussion: The overall results obtained from this study on De O Acetylated Curcumin Di Galactose and PERK13 protein shows that De O Acetylated Curcumin Di Galactose, directly binds with the active site and other potentially binding regions of PERK13. Hence, it is concluded that De O Acetylated Curcumin Di Galactose could potentially play a vital role in future research related to the problem of helping the plants tackle increased saline levels in water. Keywords: A. thaliana, De O Acetylated Curcumin Di Galactose, Drug Docking, Insilico, PERK13, Protein Modelling.</p>', 'posted': '2021-10-16'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.10.11.21264863</td>\n",
       "      <td>226</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Obstetrics and Gynecology</td>\n",
       "      <td>True</td>\n",
       "      <td>Characterizing menstrual bleeding changes occu...</td>\n",
       "      <td>[{'name': 'Katharine MN Lee'}, {'name': 'Elean...</td>\n",
       "      <td>&lt;p&gt;Many people began sharing that they experie...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.10.10.463851</td>\n",
       "      <td>95</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The Japanese wolf is most closely related to m...</td>\n",
       "      <td>[{'name': 'Jun Gojobori'}, {'name': 'Nami Arak...</td>\n",
       "      <td>&lt;p&gt;The Japanese wolf (&lt;italic&gt;Canis lupus hodo...</td>\n",
       "      <td>2021-10-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.10.16.464636</td>\n",
       "      <td>88</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Automatic identification and annotation of MYB...</td>\n",
       "      <td>[{'name': 'Boas Pucker'}]</td>\n",
       "      <td>&lt;p&gt;Background: MYBs are among the largest tran...</td>\n",
       "      <td>2021-10-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.10.12.464039</td>\n",
       "      <td>72</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Histone H1 regulates non-coding RNA turnover o...</td>\n",
       "      <td>[{'name': 'Jos Miguel Fernndez-Justel'}, {'n...</td>\n",
       "      <td>&lt;p&gt;Linker histones are highly abundant chromat...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.10.08.463334</td>\n",
       "      <td>44</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>True</td>\n",
       "      <td>Covid-19 genomic analysis reveals clusters of ...</td>\n",
       "      <td>[{'name': 'Evans K. Rono'}]</td>\n",
       "      <td>&lt;p&gt;The emerging SARS-CoV-2 variants may potent...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.10.14.464465</td>\n",
       "      <td>42</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Polypolish: short-read polishing of long-read ...</td>\n",
       "      <td>[{'name': 'Ryan R Wick'}, {'name': 'Kathryn E ...</td>\n",
       "      <td>&lt;p&gt;Long-read-only bacterial genome assemblies ...</td>\n",
       "      <td>2021-10-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.10.12.464066</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Tissue-wide coordination of calcium signaling ...</td>\n",
       "      <td>[{'name': 'Jessica L Moore'}, {'name': 'Feng G...</td>\n",
       "      <td>&lt;p&gt;Regenerative processes in the mammalian ski...</td>\n",
       "      <td>2021-10-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.10.12.463965</td>\n",
       "      <td>39</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Ferroptosis Regulation by the NGLY1/NFE2L1 Pat...</td>\n",
       "      <td>[{'name': 'Giovanni C. Forcina'}, {'name': 'La...</td>\n",
       "      <td>&lt;p&gt;Ferroptosis is an oxidative form of non-apo...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.10.13.21264960</td>\n",
       "      <td>36</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Evaluation of real-life use of Point-Of-Care R...</td>\n",
       "      <td>[{'name': 'Ana C. Blanchard'}, {'name': 'Marc ...</td>\n",
       "      <td>&lt;p&gt;We evaluated the use of rapid antigen detec...</td>\n",
       "      <td>2021-10-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.10.10.21264827</td>\n",
       "      <td>36</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Heterologous SARS-CoV-2 Booster Vaccinations ...</td>\n",
       "      <td>[{'name': 'Robert L. Atmar'}, {'name': 'Kirste...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;While Coronav...</td>\n",
       "      <td>2021-10-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.10.15.464546</td>\n",
       "      <td>32</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Airpart: Interpretable statistical models for ...</td>\n",
       "      <td>[{'name': 'Wancen Mu'}, {'name': 'Hirak Sarkar...</td>\n",
       "      <td>&lt;p&gt;Motivation: Allelic expression analysis aid...</td>\n",
       "      <td>2021-10-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.10.13.463730</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Dual polarity voltage imaging reveals subthres...</td>\n",
       "      <td>[{'name': 'Madhuvanthi Kannan'}, {'name': 'Gan...</td>\n",
       "      <td>&lt;p&gt;Genetically encoded fluorescent voltage ind...</td>\n",
       "      <td>2021-10-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.10.12.464035</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Brain endothelial antigen presentation detains...</td>\n",
       "      <td>[{'name': 'Sidar Aydin'}, {'name': 'Javier Par...</td>\n",
       "      <td>&lt;p&gt;Blood-brain barrier (BBB) breakdown and imm...</td>\n",
       "      <td>2021-10-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.10.16.463767</td>\n",
       "      <td>25</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Genomic insights into the population history a...</td>\n",
       "      <td>[{'name': 'Yan Yan Liu'}, {'name': 'Mengge Wan...</td>\n",
       "      <td>&lt;p&gt;Hmong-Mien-speaking (HM) populations, widel...</td>\n",
       "      <td>2021-10-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.10.11.463953</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Coping with oxidative stress in extreme enviro...</td>\n",
       "      <td>[{'name': 'Bruno Alejandro Steimbrch'}, {'nam...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Acinetobacter sp&lt;/italic&gt;. Ver3 is ...</td>\n",
       "      <td>2021-10-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.10.10.463858</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>A curated collection of Klebsiella metabolic m...</td>\n",
       "      <td>[{'name': 'Jane Hawkey'}, {'name': 'Ben Vezina...</td>\n",
       "      <td>&lt;p&gt;The &lt;italic&gt;Klebsiella pneumoniae&lt;/italic&gt; ...</td>\n",
       "      <td>2021-10-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.10.12.463948</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pathology</td>\n",
       "      <td>False</td>\n",
       "      <td>Prion-like -synuclein pathology in the brains...</td>\n",
       "      <td>[{'name': 'Christopher Hatton'}, {'name': 'Sim...</td>\n",
       "      <td>&lt;p&gt;Krabbe disease (KD) is an infantile neurode...</td>\n",
       "      <td>2021-10-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.10.13.464217</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Animal Behavior and Cognition</td>\n",
       "      <td>False</td>\n",
       "      <td>Disentangling choice value and choice conflict...</td>\n",
       "      <td>[{'name': 'Laura Fontanesi'}, {'name': 'Amitai...</td>\n",
       "      <td>&lt;p&gt;Recent years have witnessed a surge of inte...</td>\n",
       "      <td>2021-10-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.10.12.464028</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>HORmon: automated annotation of human centromeres</td>\n",
       "      <td>[{'name': 'Olga Kunyavskaya'}, {'name': 'Tatia...</td>\n",
       "      <td>&lt;p&gt;Recent advances in long-read sequencing ope...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.10.08.21264727</td>\n",
       "      <td>16</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Rehabilitation Medicine and Physical Therapy</td>\n",
       "      <td>False</td>\n",
       "      <td>Exoskeleton-assisted walking improves pulmonar...</td>\n",
       "      <td>[{'name': 'Xiao-Na Xiang'}, {'name': 'Li-Ming ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Question&lt;/title&gt;&lt;p&gt;In people with ...</td>\n",
       "      <td>2021-10-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.10.14.464400</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Intratumoral and extratumoral synapses are req...</td>\n",
       "      <td>[{'name': 'Maria Losada-Perez'}, {'name': 'Mam...</td>\n",
       "      <td>&lt;p&gt;Glioblastoma (GB) is the most aggressive, l...</td>\n",
       "      <td>2021-10-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.10.13.464136</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Replication Fork Uncoupling Causes Nascent Str...</td>\n",
       "      <td>[{'name': 'Tamar Kavlashvili'}, {'name': 'Jame...</td>\n",
       "      <td>&lt;p&gt;Genotoxins cause nascent strand degradation...</td>\n",
       "      <td>2021-10-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.10.12.464098</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>The Hair Cell Analysis Toolbox: A machine lear...</td>\n",
       "      <td>[{'name': 'Christopher J. Buswinka'}, {'name':...</td>\n",
       "      <td>&lt;p&gt;Auditory hair cells, the whole length of th...</td>\n",
       "      <td>2021-10-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.10.12.464064</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>New from old: discovery of the novel antibioti...</td>\n",
       "      <td>[{'name': 'Nataliia Machushynets'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;Streptomycetes are major producers of bioac...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.10.13.464196</td>\n",
       "      <td>13</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A linear programming-based strategy to save pi...</td>\n",
       "      <td>[{'name': 'Kirill Sechkar'}, {'name': 'Zoltan ...</td>\n",
       "      <td>&lt;p&gt;Laboratory automation and mathematical opti...</td>\n",
       "      <td>2021-10-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.10.13.464285</td>\n",
       "      <td>13</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Respiratory syncytial virus matrix protein ass...</td>\n",
       "      <td>[{'name': 'Bryan S Sibert'}, {'name': 'Joseph ...</td>\n",
       "      <td>&lt;p&gt;Respiratory syncytial virus (RSV) is a sign...</td>\n",
       "      <td>2021-10-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.10.11.463377</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Environmentally dependent and independent cont...</td>\n",
       "      <td>[{'name': 'L. G. Dent'}, {'name': 'N. Curry'},...</td>\n",
       "      <td>&lt;p&gt;In order to invade 3D tissues, cancer cells...</td>\n",
       "      <td>2021-10-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.10.11.463956</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 variants exhibit increased kinetic ...</td>\n",
       "      <td>[{'name': 'Ziwei Yang'}, {'name': 'Yang Han'},...</td>\n",
       "      <td>&lt;p&gt;SARS-CoV-2 variants of concern harbor mutat...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.10.13.464175</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Modular, robust and extendible multicellular c...</td>\n",
       "      <td>[{'name': 'Alberto Carignano'}, {'name': 'Dai ...</td>\n",
       "      <td>&lt;p&gt;Division of labor between cells is ubiquito...</td>\n",
       "      <td>2021-10-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.10.13.464073</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Rates of primary production in groundwater riv...</td>\n",
       "      <td>[{'name': 'Will A. Overholt'}, {'name': 'Susan...</td>\n",
       "      <td>&lt;p&gt;The terrestrial subsurface contains nearly ...</td>\n",
       "      <td>2021-10-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.10.12.464113</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Single-particle cryo-EM structures from iDPC-S...</td>\n",
       "      <td>[{'name': 'Ivan Lazi'}, {'name': 'Maarten Wir...</td>\n",
       "      <td>&lt;p&gt;Electron cryo-microscopy (cryo-EM) is becom...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.10.13.464275</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Benchmarking imputation methods for network in...</td>\n",
       "      <td>[{'name': 'Ayoub Lasri'}, {'name': 'Vahid Shah...</td>\n",
       "      <td>&lt;p&gt;Single cell RNA-sequencing (scRNA-seq) has ...</td>\n",
       "      <td>2021-10-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.10.12.464112</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>In vivo versus in silico assessment of potenti...</td>\n",
       "      <td>[{'name': 'Xinbao Ding'}, {'name': 'Priti Sing...</td>\n",
       "      <td>&lt;p&gt;Infertility is a highly heterogeneous condi...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.10.12.463977</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>The Nup2 meiotic-autonomous region relieves au...</td>\n",
       "      <td>[{'name': 'Kelly Komachi'}, {'name': 'Sean M. ...</td>\n",
       "      <td>&lt;p&gt;During meiosis, chromosomes undergo dramati...</td>\n",
       "      <td>2021-10-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.10.15.464515</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Spatial signatures of anesthesia-induced burst...</td>\n",
       "      <td>[{'name': 'Nikoloz Sirmpilatze'}, {'name': 'Ju...</td>\n",
       "      <td>&lt;p&gt;During deep anesthesia, the electroencephal...</td>\n",
       "      <td>2021-10-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.10.12.463740</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>RNAi screening of RING/U-box domain ubiquitin ...</td>\n",
       "      <td>[{'name': 'John M. Allen'}, {'name': 'Madison ...</td>\n",
       "      <td>&lt;p&gt;Regenerative processes depend on the interp...</td>\n",
       "      <td>2021-10-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.10.08.21264719</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Intensive Care and Critical Care Medicine</td>\n",
       "      <td>True</td>\n",
       "      <td>Trans Sodium Crocetinate (TSC) to Improve Oxyg...</td>\n",
       "      <td>[{'name': 'Anca Streinu-Cercel'}, {'name': 'Oa...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Trans Sodium ...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.10.12.464107</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Topoisomerase II poisons inhibit vertebrate DN...</td>\n",
       "      <td>[{'name': 'Sabrina X. Van Ravenstein'}, {'name...</td>\n",
       "      <td>&lt;p&gt;Topoisomerase II (Top2) unlinks chromosomes...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.10.11.463689</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>True</td>\n",
       "      <td>Mutating novel interaction sites in NRP1 reduc...</td>\n",
       "      <td>[{'name': 'Debjani Pal'}, {'name': 'Kuntal De'...</td>\n",
       "      <td>&lt;p&gt;The global pandemic of Coronavirus disease ...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.10.07.21264710</td>\n",
       "      <td>5</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Nephrology</td>\n",
       "      <td>False</td>\n",
       "      <td>ATP-citrate lyase as a therapeutic target in c...</td>\n",
       "      <td>[{'name': 'Pedrum Mohammadi-Shemirani'}, {'nam...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;ATP-citrate l...</td>\n",
       "      <td>2021-10-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.10.13.464316</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>CAR modulates plasma membrane nano-organizatio...</td>\n",
       "      <td>[{'name': 'Weijun Chen'}, {'name': 'Huina Zhou...</td>\n",
       "      <td>&lt;p&gt;The highly selective segregation of molecul...</td>\n",
       "      <td>2021-10-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.10.17.464158</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Distinct phases within large RNP granules of C...</td>\n",
       "      <td>[{'name': 'Brooklynne Watkins'}, {'name': 'Moh...</td>\n",
       "      <td>&lt;p&gt;One emerging paradigm of cellular organizat...</td>\n",
       "      <td>2021-10-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.10.16.464638</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Ionizing radiation improves RIG-I mediated imm...</td>\n",
       "      <td>[{'name': 'Silke Lambing'}, {'name': 'Stefan H...</td>\n",
       "      <td>&lt;p&gt;The activation of the innate immune recepto...</td>\n",
       "      <td>2021-10-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.10.17.463719</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The quantitative contribution of different Pho...</td>\n",
       "      <td>[{'name': 'Lauren Nicol'}, {'name': 'Vincenzo ...</td>\n",
       "      <td>&lt;p&gt;Excess excitation energy in the light-harve...</td>\n",
       "      <td>2021-10-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.10.17.464727</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>T7Max transcription system</td>\n",
       "      <td>[{'name': 'Christopher Deich'}, {'name': 'Broc...</td>\n",
       "      <td>&lt;p&gt;Efficient cell-free protein expression from...</td>\n",
       "      <td>2021-10-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.10.17.464590</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>TRPV1-lineage somatosensory fibers communicate...</td>\n",
       "      <td>[{'name': 'Jinrong Li'}, {'name': 'Md Sams Saz...</td>\n",
       "      <td>&lt;p&gt;Trigeminal neurons supply somatosensation t...</td>\n",
       "      <td>2021-10-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.10.09.21264758</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>The military as a neglected pathogen transmitt...</td>\n",
       "      <td>[{'name': 'Claudia Chaufan'}, {'name': 'Ilinca...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The risk of o...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.10.06.459411</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>GEOexplorer: an R/Bioconductor package for gen...</td>\n",
       "      <td>[{'name': 'Guy P Hunt'}, {'name': 'Rafael Henk...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Over the past...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.10.12.464059</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>New cycle, same old mistakes? Overlapping vs. ...</td>\n",
       "      <td>[{'name': 'Marlee R. Labroo'}, {'name': 'Jessi...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Recurrent sel...</td>\n",
       "      <td>2021-10-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.10.10.463866</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>STAG2 promotes the myelination transcriptional...</td>\n",
       "      <td>[{'name': 'Ningyan Cheng'}, {'name': 'Mohammed...</td>\n",
       "      <td>&lt;p&gt;Cohesin folds chromosomes via DNA loop extr...</td>\n",
       "      <td>2021-10-12</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2021.10.11.21264863     226  medRxiv   \n",
       "1     10.1101/2021.10.10.463851      95  bioRxiv   \n",
       "2     10.1101/2021.10.16.464636      88  bioRxiv   \n",
       "3     10.1101/2021.10.12.464039      72  bioRxiv   \n",
       "4     10.1101/2021.10.08.463334      44  bioRxiv   \n",
       "5     10.1101/2021.10.14.464465      42  bioRxiv   \n",
       "6     10.1101/2021.10.12.464066      41  bioRxiv   \n",
       "7     10.1101/2021.10.12.463965      39  bioRxiv   \n",
       "8   10.1101/2021.10.13.21264960      36  medRxiv   \n",
       "9   10.1101/2021.10.10.21264827      36  medRxiv   \n",
       "10    10.1101/2021.10.15.464546      32  bioRxiv   \n",
       "11    10.1101/2021.10.13.463730      29  bioRxiv   \n",
       "12    10.1101/2021.10.12.464035      26  bioRxiv   \n",
       "13    10.1101/2021.10.16.463767      25  bioRxiv   \n",
       "14    10.1101/2021.10.11.463953      22  bioRxiv   \n",
       "15    10.1101/2021.10.10.463858      19  bioRxiv   \n",
       "16    10.1101/2021.10.12.463948      19  bioRxiv   \n",
       "17    10.1101/2021.10.13.464217      17  bioRxiv   \n",
       "18    10.1101/2021.10.12.464028      17  bioRxiv   \n",
       "19  10.1101/2021.10.08.21264727      16  medRxiv   \n",
       "20    10.1101/2021.10.14.464400      16  bioRxiv   \n",
       "21    10.1101/2021.10.13.464136      16  bioRxiv   \n",
       "22    10.1101/2021.10.12.464098      14  bioRxiv   \n",
       "23    10.1101/2021.10.12.464064      14  bioRxiv   \n",
       "24    10.1101/2021.10.13.464196      13  bioRxiv   \n",
       "25    10.1101/2021.10.13.464285      13  bioRxiv   \n",
       "26    10.1101/2021.10.11.463377      11  bioRxiv   \n",
       "27    10.1101/2021.10.11.463956      11  bioRxiv   \n",
       "28    10.1101/2021.10.13.464175      11  bioRxiv   \n",
       "29    10.1101/2021.10.13.464073      11  bioRxiv   \n",
       "30    10.1101/2021.10.12.464113      10  bioRxiv   \n",
       "31    10.1101/2021.10.13.464275      10  bioRxiv   \n",
       "32    10.1101/2021.10.12.464112       8  bioRxiv   \n",
       "33    10.1101/2021.10.12.463977       8  bioRxiv   \n",
       "34    10.1101/2021.10.15.464515       7  bioRxiv   \n",
       "35    10.1101/2021.10.12.463740       6  bioRxiv   \n",
       "36  10.1101/2021.10.08.21264719       6  medRxiv   \n",
       "37    10.1101/2021.10.12.464107       6  bioRxiv   \n",
       "38    10.1101/2021.10.11.463689       6  bioRxiv   \n",
       "39  10.1101/2021.10.07.21264710       5  medRxiv   \n",
       "40    10.1101/2021.10.13.464316       5  bioRxiv   \n",
       "41    10.1101/2021.10.17.464158       5  bioRxiv   \n",
       "42    10.1101/2021.10.16.464638       5  bioRxiv   \n",
       "43    10.1101/2021.10.17.463719       4  bioRxiv   \n",
       "44    10.1101/2021.10.17.464727       4  bioRxiv   \n",
       "45    10.1101/2021.10.17.464590       4  bioRxiv   \n",
       "46  10.1101/2021.10.09.21264758       4  medRxiv   \n",
       "47    10.1101/2021.10.06.459411       4  bioRxiv   \n",
       "48    10.1101/2021.10.12.464059       4  bioRxiv   \n",
       "49    10.1101/2021.10.10.463866       3  bioRxiv   \n",
       "\n",
       "                                    subject-area  covid  \\\n",
       "0                      Obstetrics and Gynecology   True   \n",
       "1                           Evolutionary Biology  False   \n",
       "2                                  Plant Biology  False   \n",
       "3                              Molecular Biology  False   \n",
       "4                                       Genomics   True   \n",
       "5                                 Bioinformatics  False   \n",
       "6                                   Cell Biology  False   \n",
       "7                                   Cell Biology  False   \n",
       "8          Infectious Diseases (except HIV/AIDS)   True   \n",
       "9          Infectious Diseases (except HIV/AIDS)   True   \n",
       "10                                Bioinformatics  False   \n",
       "11                                  Neuroscience  False   \n",
       "12                                    Immunology  False   \n",
       "13                                      Genetics  False   \n",
       "14                                  Microbiology  False   \n",
       "15                                  Microbiology  False   \n",
       "16                                     Pathology  False   \n",
       "17                 Animal Behavior and Cognition  False   \n",
       "18                                Bioinformatics  False   \n",
       "19  Rehabilitation Medicine and Physical Therapy  False   \n",
       "20                                  Neuroscience  False   \n",
       "21                                  Biochemistry  False   \n",
       "22                                  Neuroscience  False   \n",
       "23                                  Microbiology  False   \n",
       "24                             Synthetic Biology  False   \n",
       "25                                    Biophysics  False   \n",
       "26                                  Cell Biology  False   \n",
       "27                                  Microbiology   True   \n",
       "28                             Synthetic Biology  False   \n",
       "29                                  Microbiology  False   \n",
       "30                                    Biophysics  False   \n",
       "31                               Systems Biology  False   \n",
       "32                                      Genetics  False   \n",
       "33                                      Genetics  False   \n",
       "34                                  Neuroscience  False   \n",
       "35                         Developmental Biology  False   \n",
       "36     Intensive Care and Critical Care Medicine   True   \n",
       "37                                  Biochemistry  False   \n",
       "38                                      Genetics   True   \n",
       "39                                    Nephrology  False   \n",
       "40                                 Plant Biology  False   \n",
       "41                                  Cell Biology  False   \n",
       "42                                    Immunology  False   \n",
       "43                                 Plant Biology  False   \n",
       "44                             Synthetic Biology  False   \n",
       "45                                  Neuroscience  False   \n",
       "46                      Public and Global Health   True   \n",
       "47                                Bioinformatics  False   \n",
       "48                                      Genetics  False   \n",
       "49                         Developmental Biology  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Characterizing menstrual bleeding changes occu...   \n",
       "1   The Japanese wolf is most closely related to m...   \n",
       "2   Automatic identification and annotation of MYB...   \n",
       "3   Histone H1 regulates non-coding RNA turnover o...   \n",
       "4   Covid-19 genomic analysis reveals clusters of ...   \n",
       "5   Polypolish: short-read polishing of long-read ...   \n",
       "6   Tissue-wide coordination of calcium signaling ...   \n",
       "7   Ferroptosis Regulation by the NGLY1/NFE2L1 Pat...   \n",
       "8   Evaluation of real-life use of Point-Of-Care R...   \n",
       "9   Heterologous SARS-CoV-2 Booster Vaccinations ...   \n",
       "10  Airpart: Interpretable statistical models for ...   \n",
       "11  Dual polarity voltage imaging reveals subthres...   \n",
       "12  Brain endothelial antigen presentation detains...   \n",
       "13  Genomic insights into the population history a...   \n",
       "14  Coping with oxidative stress in extreme enviro...   \n",
       "15  A curated collection of Klebsiella metabolic m...   \n",
       "16  Prion-like -synuclein pathology in the brains...   \n",
       "17  Disentangling choice value and choice conflict...   \n",
       "18  HORmon: automated annotation of human centromeres   \n",
       "19  Exoskeleton-assisted walking improves pulmonar...   \n",
       "20  Intratumoral and extratumoral synapses are req...   \n",
       "21  Replication Fork Uncoupling Causes Nascent Str...   \n",
       "22  The Hair Cell Analysis Toolbox: A machine lear...   \n",
       "23  New from old: discovery of the novel antibioti...   \n",
       "24  A linear programming-based strategy to save pi...   \n",
       "25  Respiratory syncytial virus matrix protein ass...   \n",
       "26  Environmentally dependent and independent cont...   \n",
       "27  SARS-CoV-2 variants exhibit increased kinetic ...   \n",
       "28  Modular, robust and extendible multicellular c...   \n",
       "29  Rates of primary production in groundwater riv...   \n",
       "30  Single-particle cryo-EM structures from iDPC-S...   \n",
       "31  Benchmarking imputation methods for network in...   \n",
       "32  In vivo versus in silico assessment of potenti...   \n",
       "33  The Nup2 meiotic-autonomous region relieves au...   \n",
       "34  Spatial signatures of anesthesia-induced burst...   \n",
       "35  RNAi screening of RING/U-box domain ubiquitin ...   \n",
       "36  Trans Sodium Crocetinate (TSC) to Improve Oxyg...   \n",
       "37  Topoisomerase II poisons inhibit vertebrate DN...   \n",
       "38  Mutating novel interaction sites in NRP1 reduc...   \n",
       "39  ATP-citrate lyase as a therapeutic target in c...   \n",
       "40  CAR modulates plasma membrane nano-organizatio...   \n",
       "41  Distinct phases within large RNP granules of C...   \n",
       "42  Ionizing radiation improves RIG-I mediated imm...   \n",
       "43  The quantitative contribution of different Pho...   \n",
       "44                         T7Max transcription system   \n",
       "45  TRPV1-lineage somatosensory fibers communicate...   \n",
       "46  The military as a neglected pathogen transmitt...   \n",
       "47  GEOexplorer: an R/Bioconductor package for gen...   \n",
       "48  New cycle, same old mistakes? Overlapping vs. ...   \n",
       "49  STAG2 promotes the myelination transcriptional...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Katharine MN Lee'}, {'name': 'Elean...   \n",
       "1   [{'name': 'Jun Gojobori'}, {'name': 'Nami Arak...   \n",
       "2                           [{'name': 'Boas Pucker'}]   \n",
       "3   [{'name': 'Jos Miguel Fernndez-Justel'}, {'n...   \n",
       "4                         [{'name': 'Evans K. Rono'}]   \n",
       "5   [{'name': 'Ryan R Wick'}, {'name': 'Kathryn E ...   \n",
       "6   [{'name': 'Jessica L Moore'}, {'name': 'Feng G...   \n",
       "7   [{'name': 'Giovanni C. Forcina'}, {'name': 'La...   \n",
       "8   [{'name': 'Ana C. Blanchard'}, {'name': 'Marc ...   \n",
       "9   [{'name': 'Robert L. Atmar'}, {'name': 'Kirste...   \n",
       "10  [{'name': 'Wancen Mu'}, {'name': 'Hirak Sarkar...   \n",
       "11  [{'name': 'Madhuvanthi Kannan'}, {'name': 'Gan...   \n",
       "12  [{'name': 'Sidar Aydin'}, {'name': 'Javier Par...   \n",
       "13  [{'name': 'Yan Yan Liu'}, {'name': 'Mengge Wan...   \n",
       "14  [{'name': 'Bruno Alejandro Steimbrch'}, {'nam...   \n",
       "15  [{'name': 'Jane Hawkey'}, {'name': 'Ben Vezina...   \n",
       "16  [{'name': 'Christopher Hatton'}, {'name': 'Sim...   \n",
       "17  [{'name': 'Laura Fontanesi'}, {'name': 'Amitai...   \n",
       "18  [{'name': 'Olga Kunyavskaya'}, {'name': 'Tatia...   \n",
       "19  [{'name': 'Xiao-Na Xiang'}, {'name': 'Li-Ming ...   \n",
       "20  [{'name': 'Maria Losada-Perez'}, {'name': 'Mam...   \n",
       "21  [{'name': 'Tamar Kavlashvili'}, {'name': 'Jame...   \n",
       "22  [{'name': 'Christopher J. Buswinka'}, {'name':...   \n",
       "23  [{'name': 'Nataliia Machushynets'}, {'name': '...   \n",
       "24  [{'name': 'Kirill Sechkar'}, {'name': 'Zoltan ...   \n",
       "25  [{'name': 'Bryan S Sibert'}, {'name': 'Joseph ...   \n",
       "26  [{'name': 'L. G. Dent'}, {'name': 'N. Curry'},...   \n",
       "27  [{'name': 'Ziwei Yang'}, {'name': 'Yang Han'},...   \n",
       "28  [{'name': 'Alberto Carignano'}, {'name': 'Dai ...   \n",
       "29  [{'name': 'Will A. Overholt'}, {'name': 'Susan...   \n",
       "30  [{'name': 'Ivan Lazi'}, {'name': 'Maarten Wir...   \n",
       "31  [{'name': 'Ayoub Lasri'}, {'name': 'Vahid Shah...   \n",
       "32  [{'name': 'Xinbao Ding'}, {'name': 'Priti Sing...   \n",
       "33  [{'name': 'Kelly Komachi'}, {'name': 'Sean M. ...   \n",
       "34  [{'name': 'Nikoloz Sirmpilatze'}, {'name': 'Ju...   \n",
       "35  [{'name': 'John M. Allen'}, {'name': 'Madison ...   \n",
       "36  [{'name': 'Anca Streinu-Cercel'}, {'name': 'Oa...   \n",
       "37  [{'name': 'Sabrina X. Van Ravenstein'}, {'name...   \n",
       "38  [{'name': 'Debjani Pal'}, {'name': 'Kuntal De'...   \n",
       "39  [{'name': 'Pedrum Mohammadi-Shemirani'}, {'nam...   \n",
       "40  [{'name': 'Weijun Chen'}, {'name': 'Huina Zhou...   \n",
       "41  [{'name': 'Brooklynne Watkins'}, {'name': 'Moh...   \n",
       "42  [{'name': 'Silke Lambing'}, {'name': 'Stefan H...   \n",
       "43  [{'name': 'Lauren Nicol'}, {'name': 'Vincenzo ...   \n",
       "44  [{'name': 'Christopher Deich'}, {'name': 'Broc...   \n",
       "45  [{'name': 'Jinrong Li'}, {'name': 'Md Sams Saz...   \n",
       "46  [{'name': 'Claudia Chaufan'}, {'name': 'Ilinca...   \n",
       "47  [{'name': 'Guy P Hunt'}, {'name': 'Rafael Henk...   \n",
       "48  [{'name': 'Marlee R. Labroo'}, {'name': 'Jessi...   \n",
       "49  [{'name': 'Ningyan Cheng'}, {'name': 'Mohammed...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>Many people began sharing that they experie...  2021-10-12  \n",
       "1   <p>The Japanese wolf (<italic>Canis lupus hodo...  2021-10-11  \n",
       "2   <p>Background: MYBs are among the largest tran...  2021-10-16  \n",
       "3   <p>Linker histones are highly abundant chromat...  2021-10-12  \n",
       "4   <p>The emerging SARS-CoV-2 variants may potent...  2021-10-12  \n",
       "5   <p>Long-read-only bacterial genome assemblies ...  2021-10-16  \n",
       "6   <p>Regenerative processes in the mammalian ski...  2021-10-13  \n",
       "7   <p>Ferroptosis is an oxidative form of non-apo...  2021-10-12  \n",
       "8   <p>We evaluated the use of rapid antigen detec...  2021-10-14  \n",
       "9   <sec><title>Background</title><p>While Coronav...  2021-10-13  \n",
       "10  <p>Motivation: Allelic expression analysis aid...  2021-10-16  \n",
       "11  <p>Genetically encoded fluorescent voltage ind...  2021-10-15  \n",
       "12  <p>Blood-brain barrier (BBB) breakdown and imm...  2021-10-13  \n",
       "13  <p>Hmong-Mien-speaking (HM) populations, widel...  2021-10-16  \n",
       "14  <p><italic>Acinetobacter sp</italic>. Ver3 is ...  2021-10-11  \n",
       "15  <p>The <italic>Klebsiella pneumoniae</italic> ...  2021-10-11  \n",
       "16  <p>Krabbe disease (KD) is an infantile neurode...  2021-10-14  \n",
       "17  <p>Recent years have witnessed a surge of inte...  2021-10-14  \n",
       "18  <p>Recent advances in long-read sequencing ope...  2021-10-12  \n",
       "19  <sec><title>Question</title><p>In people with ...  2021-10-11  \n",
       "20  <p>Glioblastoma (GB) is the most aggressive, l...  2021-10-16  \n",
       "21  <p>Genotoxins cause nascent strand degradation...  2021-10-13  \n",
       "22  <p>Auditory hair cells, the whole length of th...  2021-10-14  \n",
       "23  <p>Streptomycetes are major producers of bioac...  2021-10-12  \n",
       "24  <p>Laboratory automation and mathematical opti...  2021-10-15  \n",
       "25  <p>Respiratory syncytial virus (RSV) is a sign...  2021-10-15  \n",
       "26  <p>In order to invade 3D tissues, cancer cells...  2021-10-13  \n",
       "27  <p>SARS-CoV-2 variants of concern harbor mutat...  2021-10-12  \n",
       "28  <p>Division of labor between cells is ubiquito...  2021-10-14  \n",
       "29  <p>The terrestrial subsurface contains nearly ...  2021-10-13  \n",
       "30  <p>Electron cryo-microscopy (cryo-EM) is becom...  2021-10-12  \n",
       "31  <p>Single cell RNA-sequencing (scRNA-seq) has ...  2021-10-14  \n",
       "32  <p>Infertility is a highly heterogeneous condi...  2021-10-12  \n",
       "33  <p>During meiosis, chromosomes undergo dramati...  2021-10-13  \n",
       "34  <p>During deep anesthesia, the electroencephal...  2021-10-16  \n",
       "35  <p>Regenerative processes depend on the interp...  2021-10-13  \n",
       "36  <sec><title>Background</title><p>Trans Sodium ...  2021-10-12  \n",
       "37  <p>Topoisomerase II (Top2) unlinks chromosomes...  2021-10-12  \n",
       "38  <p>The global pandemic of Coronavirus disease ...  2021-10-12  \n",
       "39  <sec><title>Background</title><p>ATP-citrate l...  2021-10-11  \n",
       "40  <p>The highly selective segregation of molecul...  2021-10-15  \n",
       "41  <p>One emerging paradigm of cellular organizat...  2021-10-17  \n",
       "42  <p>The activation of the innate immune recepto...  2021-10-16  \n",
       "43  <p>Excess excitation energy in the light-harve...  2021-10-17  \n",
       "44  <p>Efficient cell-free protein expression from...  2021-10-17  \n",
       "45  <p>Trigeminal neurons supply somatosensation t...  2021-10-17  \n",
       "46  <sec><title>Background</title><p>The risk of o...  2021-10-12  \n",
       "47  <sec><title>Background</title><p>Over the past...  2021-10-12  \n",
       "48  <sec><title>Background</title><p>Recurrent sel...  2021-10-14  \n",
       "49  <p>Cohesin folds chromosomes via DNA loop extr...  2021-10-12  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "59 preprints (including 9 covering SARS-CoV-2, 52 from bioRxiv and 7 from medRxiv) published in the last 7 days before 2021-10-18 had been tweeted at least 3 times (maximum 226)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
