{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29579 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=84a98e46-71ef-4c6c-b72a-3fb94438f2a1&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b1b490c8-4f9e-4e46-9cac-549f89a5dd39&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=bcc6d72a-765d-493d-91ba-433a4c7bee59&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=3db3d7ce-6b2d-4071-91c7-cee9469db2e2&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5232e182-9e25-441b-85f0-fab30dee9aea&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d0ac5209-2240-427a-9a89-b47ce2462dde&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d76174c3-4695-48b5-be41-0a0bc07d84d7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9557ea8d-b635-4f19-ad6a-eccab7c9623c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=441ae12b-e56b-4a6c-8e8d-01bb16e5c778&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d4703a23-678c-40a7-83ce-6578983c8cdf&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=63ea4243-4dec-43a9-bcf3-ccc05ac68958&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=1b89ef58-b4c4-4053-973d-7850f5eea736&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=0b7173c4-8dcc-4a13-ab8d-acd1051b3bc2&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=675b9902-1e68-4107-b4e0-bcb37d954e8e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4a3af725-be38-4f80-8263-905821770341&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=1500b31a-036f-4411-bb0b-0cb5939b8d9c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a0660110-2542-4b3b-9a90-8f8a54613460&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=79b037ce-e059-4523-b510-529206d52333&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0018.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=36d9567a-20fe-4d3c-ba76-f1d1819013e5&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0019.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9db98139-8cb4-4e67-ab28-d2f792f18f9d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0020.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=29fbc2a2-2ae9-478e-8160-945c4c111da4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0021.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=55146a41-5b2e-4ff2-a3e1-703b18fc0200&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0022.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=3d3a71ab-c96f-48a6-a123-04a451faf641&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0023.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a2a54334-c377-461b-9473-b8dc0fc1dda4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0024.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5ff524bf-105e-4928-998b-beab4813ce82&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0025.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=183c0f5c-481c-4ad0-807a-5e24d6157e5a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0026.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=0fccea42-1526-45cd-a295-ec4ed24d2546&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0027.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=92a57fc3-b5af-42ad-b522-d8b7314cfc07&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0028.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e7a41879-6a9c-48b2-ac6a-3bbc8c852c14&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-13&until-occurred-date=2021-12-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0029.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-10-18.csv\n",
      "failed to load preprint_tweets_2021-10-25.csv\n",
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load preprint_tweets_2021-10-23.csv\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-11-01.csv\n",
      "failed to load preprint_tweets_2021-11-15.csv\n",
      "failed to load preprint_tweets_2021-11-29.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-11-08.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load preprint_tweets_2021-11-22.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-12-06.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-12-13.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.12.06.471446...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.21267114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.21266068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21250921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.470392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21265796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21263608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21266055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.465626...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.470907...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.471005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21267155...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.21267677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.21267432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/507590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.446922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.26.062927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.434611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.31.429053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.21267684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472585...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.472009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472330...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.409144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.26.470134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457513...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.24.436901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472438...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.21267301...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.21250959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.442980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.30.468210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471870...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.470817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472793...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253770...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.276056.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.471765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.27.20018929...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266824...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.12.293498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472698...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.470764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472674...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.420331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.426428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.29.470428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.459575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472968...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.21267319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21265380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273300.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21266069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.28.923532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.469125...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.471754...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267454...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21266341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.30.21267047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472530...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472383...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.469105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472285...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472773...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.423161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472149...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.21266792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472216...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.15.275891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21265944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.06.21265920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/721662...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.21267516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265397...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267793...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472640...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21258385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.21254057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.20241927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.21250840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.472005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.469460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.21267094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21267350...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472466...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.21267573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.09.20187625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.453321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.427598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.21267312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21261243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472681...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.306464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.21262859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458411...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.367649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.22.351486...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21265753...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458238...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.30.470367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.21266853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.470920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21267146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473118...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473088...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472476...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472322...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.20.259937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472618...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.05.20123745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.465418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.21267295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.471045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21267993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472185...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.07.414938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.472861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.444295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.29.470445...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.20249061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472278...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.470946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21256376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.471323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.439412...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.356774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472688...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.428751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.04.451053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.470649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.467646...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267698...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.429164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.21267233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.470697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.440450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.425362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.455606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.471237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456336...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472350...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469640...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21267655...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267039...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.21256151...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.19.346353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.11.451967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.21267943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.01.181719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471659...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.08.20095604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.455568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21266035...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258518...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.21262914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.468693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469506...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265945...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/603928...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472153...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472107...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.472408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472435...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.427406...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.471511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.429684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472238...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21249256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.11.20229765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472281...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.470823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.472038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472884...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.441323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.471074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21267281...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.470717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.429211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.456256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.20204693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.444009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.07.20180448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.21263359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472078...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.453783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.31.21250868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267773...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472572...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.21251499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469860...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472293...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21266776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472054...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.09.20246306...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453860...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.432517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.27.21265906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20131243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.413229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.07.029009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.17.426875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472761...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.471280...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472427...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472422...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446454...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.21265536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.459429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21264964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.437719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472660...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21267986...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.471847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472339...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472348...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466347...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.471245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472529...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.423122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423434...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.12.20129254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472434...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262216...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.20248876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.01.405662...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.453704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.174888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472753...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.225854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459749...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.163816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.214486...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.30.228544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461046...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.112474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.20.162701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.10.20097543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.24.353722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.28.21262763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472826...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.471281...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.19.304188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.20202457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.05.459045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472676...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.313833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.16.951301...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.27.21259601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471777...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471623...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.466586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.459925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.21254026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473085...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472071...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.04.20031112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471945...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.471186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.21266229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.471190...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266775...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.21256571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.21256597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257337...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.442916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.29.469904...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.470767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.09.443246...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.470493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.21267132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.21266089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.06.20020974...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471682...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.471131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21252978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.12.874230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.468006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.468204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.471284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.446283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21266784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472347...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.21251475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.452958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.026559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.30.428333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454085...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.451501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.06.028712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21252086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.451983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20050336...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.472233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462597...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.12.14.472630', 'tweets': 646, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Considerable escape of SARS-CoV-2 variant Omicron to antibody neutralization', 'authors': \"[{'name': 'Delphine Planas'}, {'name': 'Nell Saunders'}, {'name': 'Piet Maes'}, {'name': 'Florence Guivel-Benhassine'}, {'name': 'Cyril Planchais'}, {'name': 'Julian Buchrieser'}, {'name': 'William-Henry Bolland'}, {'name': 'Franoise Porrot'}, {'name': 'Isabelle Staropoli'}, {'name': 'Frederic Lemoine'}, {'name': 'Hlne Pr'}, {'name': 'David Veyer'}, {'name': 'Julien Puech'}, {'name': 'Julien Rodary'}, {'name': 'Guy Baela'}, {'name': 'Simon Dellicour'}, {'name': 'Joren Raymenants'}, {'name': 'Sarah Gorissen'}, {'name': 'Caspar Geenen'}, {'name': 'Bert Vanmechelen'}, {'name': 'Tony Wawina-Bokalanga'}, {'name': 'Joan Mart-Carrerasi'}, {'name': 'Lize Cuypers'}, {'name': 'Aymeric Sve'}, {'name': 'Laurent Hocqueloux'}, {'name': 'Thierry Prazuck'}, {'name': 'Flix Rey'}, {'name': 'Etienne Simon-Lorrire'}, {'name': 'Timothe Bruel'}, {'name': 'Hugo Mouquet'}, {'name': 'Emmanuel Andr'}, {'name': 'Olivier Schwartz'}]\", 'abstract': '<p>The SARS-CoV-2 Omicron variant was first identified in November 2021 in Botswana and South Africa<sup>1,2</sup>. It has in the meantime spread to many countries and is expected to rapidly become dominant worldwide. The lineage is characterized by the presence of about 32 mutations in the Spike, located mostly in the N-terminal domain (NTD) and the receptor binding domain (RBD), which may enhance viral fitness and allow antibody evasion. Here, we isolated an infectious Omicron virus in Belgium, from a traveller returning from Egypt. We examined its sensitivity to 9 monoclonal antibodies (mAbs) clinically approved or in development<sup>3</sup>, and to antibodies present in 90 sera from COVID-19 vaccine recipients or convalescent individuals. Omicron was totally or partially resistant to neutralization by all mAbs tested. Sera from Pfizer or AstraZeneca vaccine recipients, sampled 5 months after complete vaccination, barely inhibited Omicron. Sera from COVID-19 convalescent patients collected 6 or 12 months post symptoms displayed low or no neutralizing activity against Omicron. Administration of a booster Pfizer dose as well as vaccination of previously infected individuals generated an anti-Omicron neutralizing response, with titers 5 to 31 fold lower against Omicron than against Delta. Thus, Omicron escapes most therapeutic monoclonal antibodies and to a large extent vaccine-elicited antibodies.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.21267755', 'tweets': 520, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'mRNA-based COVID-19 vaccine boosters induce neutralizing immunity against SARS-CoV-2 Omicron variant', 'authors': \"[{'name': 'Wilfredo F. Garcia-Beltran'}, {'name': 'Kerri J. St. Denis'}, {'name': 'Angelique Hoelzemer'}, {'name': 'Evan C. Lam'}, {'name': 'Adam D. Nitido'}, {'name': 'Maegan L. Sheehan'}, {'name': 'Cristhian Berrios'}, {'name': 'Onosereme Ofoman'}, {'name': 'Christina C. Chang'}, {'name': 'Blake M. Hauser'}, {'name': 'Jared Feldman'}, {'name': 'David J. Gregory'}, {'name': 'Mark C. Poznansky'}, {'name': 'Aaron G. Schmidt'}, {'name': 'A. John Iafrate'}, {'name': 'Vivek Naranbhai'}, {'name': 'Alejandro B. Balazs'}]\", 'abstract': '<p>Recent surveillance has revealed the emergence of the SARS-CoV-2 Omicron variant (BA.1/B.1.1.529) harboring up to 36 mutations in spike protein, the target of vaccine-induced neutralizing antibodies. Given its potential to escape vaccine-induced humoral immunity, we measured neutralization potency of sera from 88 mRNA-1273, 111 BNT162b, and 40 Ad26.COV2.S vaccine recipients against wild type, Delta, and Omicron SARS-CoV-2 pseudoviruses. We included individuals that were vaccinated recently (&lt;3 months), distantly (6-12 months), or recently boosted, and accounted for prior SARS-CoV-2 infection. Remarkably, neutralization of Omicron was undetectable in most vaccinated individuals. However, individuals boosted with mRNA vaccines exhibited potent neutralization of Omicron only 4-6-fold lower than wild type, suggesting that boosters enhance the cross-reactivity of neutralizing antibody responses. In addition, we find Omicron pseudovirus is more infectious than any other variant tested. Overall, this study highlights the importance of boosters to broaden neutralizing antibody responses against highly divergent SARS-CoV-2 variants.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.12.472242', 'tweets': 291, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Dynamics of CTCF and cohesin mediated chromatin looping revealed by live-cell imaging', 'authors': \"[{'name': 'Michele Gabriele'}, {'name': 'Hugo B. Brando'}, {'name': 'Simon Grosse-Holz'}, {'name': 'Asmita Jha'}, {'name': 'Gina M. Dailey'}, {'name': 'Claudia Cattoglio'}, {'name': 'Tsung-Han S. Hsieh'}, {'name': 'Leonid Mirny'}, {'name': 'Christoph Zechner'}, {'name': 'Anders S. Hansen'}]\", 'abstract': '<p>Animal genomes are folded into loops and topologically associating domains (TADs) by CTCF and cohesin, but whether these loops are stable or dynamic is unknown. Here, we directly visualize chromatin looping at the <italic>Fbn2</italic> TAD in mouse embryonic stem cells using super-resolution live-cell imaging and quantify looping dynamics by Bayesian inference. Our results are consistent with cohesin-mediated loop extrusion in cells, and with CTCF both stopping and stabilizing cohesin. Surprisingly, the <italic>Fbn2</italic> loop is both rare and dynamic, with a looped fraction of ~3-6.5% and a median loop lifetime of ~10-30 minutes. Instead of a stable loop, our results establish a highly dynamic view of TADs and loops where the <italic>Fbn2</italic> TAD exists predominantly in a partially extruded conformation. This dynamic and quantitative view of TADs may facilitate a mechanistic understanding of their functions.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472469', 'tweets': 171, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Giant Starship elements mobilize accessory genes in fungal genomes', 'authors': \"[{'name': 'Emile Gluck-Thaler'}, {'name': 'Timothy Ralston'}, {'name': 'Zachary Konkel'}, {'name': 'Cristhian Grabowski Ocampos'}, {'name': 'Veena Devi Ganeshan'}, {'name': 'Anne E. Dorrance'}, {'name': 'Terry L. Niblack'}, {'name': 'Corlett W. Wood'}, {'name': 'Jason C. Slot'}, {'name': 'Horacio D. Lopez-Nicora'}, {'name': 'Aaron A. Vogan'}]\", 'abstract': '<p>Accessory genes are variably present among members of a species and are a reservoir of adaptive functions. In bacteria, differences in gene distributions among individuals largely result from mobile elements that acquire and disperse accessory genes as cargo. In contrast, the impact of cargo-carrying elements on eukaryotic evolution remains largely unknown. Here, we show that variation in genome content within multiple fungal species is facilitated by <italic>Starships,</italic> a novel group of massive mobile elements that are 110 kb long on average, share conserved components, and carry diverse arrays of accessory genes. We identified hundreds of <italic>Starship-</italic>like regions across every major class of filamentous Ascomycetes, including 28 distinct <italic>Starships</italic> that range from 27-393 kb and last shared a common ancestor ca. 400 mya. Using new long-read assemblies of the plant pathogen <italic>Macrophomina phaseolina</italic>, we characterize 4 additional <italic>Starships</italic> whose past and ongoing activities contribute to standing variation in genome structure and content. One of these elements, <italic>Voyager</italic>, inserts into 5S rDNA and contains a candidate virulence factor whose increasing copy number has contrasting associations with pathogenic and saprophytic growth, suggesting <italic>Voyager</italic>s activity underlies an ecological trade-off. We propose that <italic>Starships</italic> are eukaryotic analogs of bacterial integrative and conjugative elements based on parallels between their conserved components and may therefore represent the first known agents of active gene transfer in eukaryotes. Our results suggest that <italic>Starships</italic> have shaped the content and structure of fungal genomes for millions of years and reveal a new concerted route for evolution throughout an entire eukaryotic phylum.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.10.21267523', 'tweets': 142, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Human serum from SARS-CoV-2 vaccinated and COVID-19 patients shows reduced binding to the RBD of SARS-CoV-2 Omicron variant', 'authors': \"[{'name': 'Maren Schubert'}, {'name': 'Federico Bertoglio'}, {'name': 'Stephan Steinke'}, {'name': 'Philip Alexander Heine'}, {'name': 'Mario Alberto Ynga-Durand'}, {'name': 'Fanglei Zuo'}, {'name': 'Likun Du'}, {'name': 'Janin Korn'}, {'name': 'Marko Miloevi'}, {'name': 'Esther Veronika Wenzel'}, {'name': 'Henrike Maass'}, {'name': 'Fran Krstanovi'}, {'name': 'Saskia Polten'}, {'name': 'Marina Pribani-Matei'}, {'name': 'Ilija Brizi'}, {'name': 'Antonio Piralla'}, {'name': 'Fausto Baldanti'}, {'name': 'Lennart Hammarstrm'}, {'name': 'Stefan Dbel'}, {'name': 'Alan usti'}, {'name': 'Harold Marcotte'}, {'name': 'Monika Strengert'}, {'name': 'Alen Proti'}, {'name': 'Qiang Pan-Hammarstrm'}, {'name': 'Luka iin-ain'}, {'name': 'Michael Hust'}]\", 'abstract': '<p>The COVID-19 pandemic is caused by the betacoronavirus SARS-CoV-2. In November 2021, the Omicron variant was discovered and classified as a variant of concern (VOC). Omicron shows substantially more mutations in the spike protein than any previous variant, mostly in the receptor binding domain (RBD). We analyzed the binding of the Omicron RBD to the human ACE2 receptor and the ability of human sera from COVID-19 patients or vaccinees in comparison to Wuhan, Beta or Delta RBDs variants.</p><p>All RBDs were produced in insect cells. RBD binding to hACE was analyzed by ELISA and microscale thermophoresis (MST). Similarly, sera from 27 COVID-19 patients, 21 fully vaccinated individuals and 16 booster recipients were titrated by ELISA on the fixed RBDs from the original Wuhan strain, Beta, Delta and Omicron VOC.</p><p>Surprisingly, the Omicron RBD showed the weakest binding to ACE2, arguing that improved ACE2 binding is not a likely driver of Omicron evolution. Serum antibody titers were significantly lower against Omicron RBD than against the original Wuhan strain. Serum titers against Beta and Delta RBDs were also reduced, but to a lesser degree. The reduced binding of sera to Omicron RBD adds evidence that current vaccination protocols may be less efficient against the Omicron variant.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472668', 'tweets': 142, 'archive': 'bioRxiv', 'subject-area': 'Pathology', 'covid': True, 'title': 'SARS-CoV-2 Spike protein activates TMEM16F-mediated platelet pro-coagulant activity', 'authors': \"[{'name': 'Ambra Cappelletto'}, {'name': 'Harriet E. Allan'}, {'name': 'Marilena Crescente'}, {'name': 'Edoardo Schneider'}, {'name': 'Rossana Bussani'}, {'name': 'Hashim Ali'}, {'name': 'Ilaria Secco'}, {'name': 'Simone Vodret'}, {'name': 'Roberto Simeone'}, {'name': 'Luca Mascaretti'}, {'name': 'Serena Zacchigna'}, {'name': 'Timothy D. Warner'}, {'name': 'Mauro Giacca'}]\", 'abstract': '<sec><title>Background</title><p>Thrombosis of the lung micro-vasculature is a characteristic of COVID-19 disease, which is observed in large excess compared to other forms of acute respiratory distress syndrome and thus suggests a trigger for thrombosis endogenous to the lung. Our recent work has shown that the SARS-CoV-2 Spike protein activates the cellular TMEM16F chloride channel and scramblase. Through a screening on &gt;3,000 FDA/EMA approved drugs, we identified Niclosamide and Clofazimine as the most effective molecules at inhibiting this activity. As TMEM16F plays an important role in the stimulation of the pro-coagulant activity of platelets, and considering that platelet abnormalities are common in COVID-19 patients, we investigated whether Spike directly affects platelet activation and pro-thrombotic function and tested the effect of Niclosamide and Clofazimine on these processes.</p></sec><sec><title>Methods</title><p>We produced SARS-CoV-2 Spike or VSV-G protein-pseudotyped virions, or generated cells expressing Spike on their plasma membrane, and tested their effects on platelet adhesion (fluorescence), aggregation (absorbance), exposure of phosphatidylserine (flow cytometry for annexin V binding), calcium flux (flow cytometry for fluo-4 AM), and clot formation and retraction. These experiments were also conducted in the presence of the TMEM16F activity inhibitors Niclosamide and Clofazimine.</p></sec><sec><title>Results</title><p>Here we show that exposure to SARS-CoV-2 Spike promotes platelet activation, adhesion and spreading, both when present on the envelope of virions or upon expression on the plasma membrane of cells. Spike was effective both as a sole agonist or by enhancing the effect of known platelet activators, such as collagen and collagen-related peptide. In particular, Spike exerted a noticeable effect on the procoagulant phenotype of platelets, by enhancing calcium flux, phosphatidylserine externalisation, and thrombin generation. Eventually, this resulted in a striking increase in thrombin-induced clot formation and retraction. Both Niclosamide and Clofazimine almost abolished this Spike-induced pro-coagulant response.</p></sec><sec><title>Conclusions</title><p>Together, these findings provide a pathogenic mechanism to explain thrombosis associated to COVID-19 lung disease, by which Spike present in SARS-CoV-2 virions or exposed on the surface of infected cells, leads to local platelet stimulation and subsequent activation of the coagulation cascade. As platelet TMEM16F is central in this process, these findings reinforce the rationale of repurposing drugs targeting this protein, such as Niclosamide, for COVID-19 therapy.</p></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.472950', 'tweets': 130, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': \"Assembly of recombinant tau into filaments identical to those of Alzheimer's disease and chronic traumatic encephalopathy\", 'authors': \"[{'name': 'Sofia Lvestam'}, {'name': 'Fujiet Adrian Koh'}, {'name': 'Bart van Knippenberg'}, {'name': 'Abhay Kotecha'}, {'name': 'Alexey G Murzin'}, {'name': 'Michel Goedert'}, {'name': 'Sjors Scheres'}]\", 'abstract': \"<p>Abundant filamentous inclusions of tau are characteristic of more than 20 neurodegenerative diseases that are collectively termed tauopathies. Electron cryo-microscopy (cryo-EM) structures of tau amyloid filaments from human brain revealed that different tau folds characterise many different diseases. A lack of laboratory-based model systems to generate these structures has hampered efforts to uncover the molecular mechanisms that underlie tauopathies. Here, we report in vitro assembly conditions with recombinant tau that replicate the structures of filaments from both Alzheimer's disease (AD) and chronic traumatic encephalopathy (CTE), as determined by cryo-EM. Our results suggest that post-translational modifications of tau modulate filament assembly, and that previously observed additional densities in AD and CTE filaments may arise from the presence of inorganic salts, like phosphates and sodium chloride. In vitro assembly of tau into disease-relevant filaments will facilitate studies to determine their roles in different diseases, as well as the development of compounds that specifically bind to these structures or prevent their formation.</p>\", 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.21267791', 'tweets': 100, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Global epidemiology of SARS-CoV-2 infection: a systematic review and meta-analysis of standardized population-based seroprevalence studies, Jan 2020-Oct 2021', 'authors': \"[{'name': 'Isabel Bergeri'}, {'name': 'Mairead Whelan'}, {'name': 'Harriet Ware'}, {'name': 'Lorenzo Subissi'}, {'name': 'Anthony Nardone'}, {'name': 'Hannah C Lewis'}, {'name': 'Zihan Li'}, {'name': 'Xiaomeng Ma'}, {'name': 'Marta Valenciano'}, {'name': 'Brianna Cheng'}, {'name': 'Lubna Al Ariqi'}, {'name': 'Arash Rashidian'}, {'name': 'Joseph Okeibunor'}, {'name': 'Tasnim Azim'}, {'name': 'Pushpa Wijesinghe'}, {'name': 'Linh-Vi Le'}, {'name': 'Aisling Vaughan'}, {'name': 'Richard Pebody'}, {'name': 'Andrea Vicari'}, {'name': 'Tingting Yan'}, {'name': 'Mercedes Yanes-Lane'}, {'name': 'Christian Cao'}, {'name': 'Matthew P Cheng'}, {'name': 'Jesse Papenburg'}, {'name': 'David Buckeridge'}, {'name': 'Niklas Bobrovitz'}, {'name': 'Rahul K Arora'}, {'name': 'Maria D van Kerkhove'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>COVID-19 case data underestimates infection and immunity, especially in low- and middle-income countries (LMICs). We meta-analyzed standardized SARS-CoV-2 seroprevalence studies to estimate global seroprevalence.</p></sec><sec><title>Objectives/Methods</title><p>We conducted a systematic review and meta-analysis, searching MEDLINE, Embase, Web of Science, preprints, and grey literature for SARS-CoV-2 seroprevalence studies aligned with the WHO UNITY protocol published between 2020-01-01 and 2021-10-29. Eligible studies were extracted and critically appraised in duplicate. We meta-analyzed seroprevalence by country and month, pooling to estimate regional and global seroprevalence over time; compared seroprevalence from infection to confirmed cases to estimate under-ascertainment; meta-analyzed differences in seroprevalence between demographic subgroups; and identified national factors associated with seroprevalence using meta-regression. PROSPERO: CRD42020183634.</p></sec><sec><title>Results</title><p>We identified 396 full texts reporting 736 distinct seroprevalence studies (41% LMIC), including 355 low/moderate risk of bias studies with national/sub-national scope in further analysis. By April 2021, global SARS-CoV-2 seroprevalence was 26.1%, 95% CI [24.6-27.6%]. Seroprevalence rose steeply in the first half of 2021 due to infection in some regions (e.g., 18.2% to 45.9% in Africa) and vaccination and infection in others (e.g., 11.3% to 57.4% in the Americas high-income countries), but remained low in others (e.g., 0.3% to 1.6% in the Western Pacific). In 2021 Q1, median seroprevalence to case ratios were 1.9:1 in HICs and 61.9:1 in LMICs. Children 0-9 years and adults 60+ were at lower risk of seropositivity than adults 20-29. In a multivariate model using data pre-vaccination, more stringent public health and social measures were associated with lower seroprevalence.</p></sec><sec><title>Conclusions</title><p>Global seroprevalence has risen considerably over time and with regional variation, however much of the global population remains susceptible to SARS-CoV-2 infection. True infections far exceed reported COVID-19 cases. Standardized seroprevalence studies are essential to inform COVID-19 control measures, particularly in resource-limited regions.</p></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472704', 'tweets': 91, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': True, 'title': 'Simulation of the omicron variant of SARS-CoV-2 shows broad antibody escape, weakened ACE2 binding, and modest increase in furin binding', 'authors': \"[{'name': 'M. Zaki Jawaid'}, {'name': 'A. Baidya'}, {'name': 'R. Mahboubi-Ardakani'}, {'name': 'Richard L. Davis'}, {'name': 'Daniel L. Cox'}]\", 'abstract': '<p>The recent emergence of the omicron variant of the SARS-CoV-2 virus with large numbers of mutations has raised concern about a potential new surge in infections. Here we use molecular dynamics to study the biophysics of the interface of the omicron spike protein binding to (i) the ACE2 receptor protein, (ii) antibodies from all known binding regions, and (iii) the furin binding domain. Our simulations suggest that while there is significant reduction of antibody binding strength corresponding to escape, the omicron spike pays a cost in terms of weaker receptor binding. The furin cleavage domain is the same or weaker binding than the alpha variant, suggesting less viral load and disease intensity than the extant delta variant.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472703', 'tweets': 86, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Linear plasmids in Klebsiella and other Enterobacteriaceae', 'authors': \"[{'name': 'Jane Hawkey'}, {'name': 'Hugh Cottingham'}, {'name': 'Alex Tokolyi'}, {'name': 'Ryan R. Wick'}, {'name': 'Louise M. Judd'}, {'name': 'Louise Cerdeira'}, {'name': 'Doroti de Oliveira Garcia'}, {'name': 'Kelly L. Wyres'}, {'name': 'Kathryn E. Holt'}]\", 'abstract': '<p>Linear plasmids are extrachromosomal DNA that have been found in a small number of bacterial species. To date, the only linear plasmids described in the Enterobacteriaceae family belong to <italic>Salmonella,</italic> first found in <italic>Salmonella</italic> Typhi. Here, we describe a collection of 12 isolates of the <italic>Klebsiella pneumoniae</italic> species complex in which we identified linear plasmids. We used this collection to search public sequence databases and discovered an additional 74 linear plasmid sequences in a variety of Enterobacteriaceae species. Gene content analysis divided these plasmids into five distinct phylogroups, with very few genes shared across more than two phylogroups. The majority of linear plasmid-encoded genes are of unknown function, however each phylogroup carried its own unique toxin-antitoxin system and genes with homology to those encoding the ParAB plasmid stability system. Passage <italic>in vitro</italic> of the 12 linear plasmid-carrying <italic>Klebsiella</italic> isolates in our collection (which include representatives of all five phylogroups) indicated that these linear plasmids can be stably maintained, and our data suggest they can transmit between <italic>K. pneumoniae</italic> strains (including members of globally disseminated multidrug resistant clones) and also between diverse Enterobacteriaceae species. The linear plasmid sequences, and representative isolates harbouring them, are made available as a resource to facilitate future studies on the evolution and function of these novel plasmids.</p><sec><title>Significance as a BioResource to the community</title><p>This study provides the first report of linear plasmids identified within the <italic>Klebsiella pneumoniae</italic> species complex and the first report in Enterobacteriaceae besides <italic>Salmonella.</italic>We present the first comparative analysis of linear plasmid sequences in Enterobacteriaceae, however whilst this family is highly clinically significant, the functional and/or evolutionary importance of these plasmids is not yet clear. To facilitate future studies to address these questions, we have publicly deposited (i) the collection of linear plasmid sequence data; (ii) isolates representative of each of the distinct linear plasmid phylogroups.</p></sec><sec><title>Data Summary</title><p><bold>The authors confirm all supporting data, code and protocols have been provided within the article or through supplementary information</bold></p><list list-type=\"order\"><list-item><p>Whole genome sequence reads from <italic>Klebsiella pneumoniae</italic> isolates sequenced in this study have been deposited in NCBI SRA under the accession numbers listed in <bold>Table S1</bold>.</p></list-item><list-item><p>Representative annotated sequences of one linear plasmid per phylogroup have been deposited in FigShare, <bold>doi 10.26180/16729126</bold>.</p></list-item><list-item><p>A copy of all linear plasmid sequences that we assembled from publicly available genome sequence reads are available in FigShare, <bold>doi 10.26180/16531365</bold>. Read accessions for these are given in <bold>Table S1</bold>.</p></list-item><list-item><p>Eleven representative <italic>K. pneumoniae</italic> isolates harbouring linear plasmids described in this study have been deposited with the National Collection of Type Cultures (NCTC) and are available for purchase under the NCTC accession numbers listed in <bold>Table S1.</bold><italic>K. pneumoniae</italic> 1194/11 (representative of phylogroup B) has been deposited in the Microorganisms Collection Center, Adolfo Lutz Institute, So Paulo, Brazil. To request strain 1194/11 (IAL 3063, SISGEN ABBF09B), contact:\\n<list list-type=\"simple\"><list-item><p>Microorganisms Collection Center</p></list-item><list-item><p>Culture Collection Laboratory</p></list-item><list-item><p>Instituto Adolfo Lutz, Sao Paulo State Department of Health</p></list-item><list-item><p>Address: Av Dr Arnaldo, 351, 10 floor, room 1020</p></list-item><list-item><p>Phone number: +55 11 3068-2884</p></list-item><list-item><p>Zip code 01246-000, So Paulo, Brazil</p></list-item><list-item><p>E-mail: <email>colecaoial@ial.sp.gov.br</email></p></list-item></list></p></list-item><list-item><p>Alignments of terminal inverted repeat sequences for each phylogroup can be found in <bold>Data S1</bold>, available on FigShare, <bold>doi 10.26180/16531371</bold>.</p></list-item></list></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.472920', 'tweets': 80, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': True, 'title': 'Amyloidogenesis of SARS-CoV-2 Spike Protein', 'authors': \"[{'name': 'Sofie Nystrom'}, {'name': 'Per Hammarstrom'}]\", 'abstract': '<p>SARS-CoV-2 infection is associated with a surprising number of morbidities. Uncanny similarities with amyloid-disease associated blood coagulation and fibrinolytic disturbances together with neurologic and cardiac problems led us to investigate the amyloidogenicity of the SARS-CoV-2 Spike protein (S-protein). Amyloid fibril assays of peptide library mixtures and theoretical predictions identified seven amyloidogenic sequences within the S-protein. All seven peptides in isolation formed aggregates during incubation at 37C. Three 20-amino acid long synthetic Spike peptides (sequence 191-210, 599-618, 1165-1184) fulfilled three amyloid fibril criteria: nucleation dependent polymerization kinetics by ThT, Congo red positivity and ultrastructural fibrillar morphology. Full-length folded S-protein did not form amyloid fibrils, but amyloid-like fibrils with evident branching were formed during 24 hours of S-protein co-incubation with the protease neutrophil elastase (NE) in vitro. NE efficiently cleaved S-protein rendering exposure of amyloidogenic segments and accumulation of the peptide 193-202, part of the most amyloidogenic synthetic Spike peptide. NE is overexpressed at inflamed sites of viral infection and at vaccine injection sites. Our data propose a molecular mechanism for amyloidogenesis of SARS-CoV-2 S-protein in humans facilitated by endoproteolysis. The potential implications of S-protein amyloidogenesis in COVID-19 disease associated pathogenesis and consequences following S-protein based vaccines should be addressed in understanding the disease, long COVID-19, and vaccine side effects.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.10.472112', 'tweets': 72, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'SARS-CoV-2 Spike triggers barrier dysfunction and vascular leak via integrins and TGF- signaling', 'authors': \"[{'name': 'Scott B. Biering'}, {'name': 'Francielle Tramontini Gomes de Sousa'}, {'name': 'Laurentia V. Tjang'}, {'name': 'Felix Pahmeier'}, {'name': 'Richard Ruan'}, {'name': 'Sophie F. Blanc'}, {'name': 'Trishna S. Patel'}, {'name': 'Caroline M. Worthington'}, {'name': 'Dustin R. Glasner'}, {'name': 'Bryan Castillo-Rojas'}, {'name': 'Venice Servellita'}, {'name': 'Nicholas T.N. Lo'}, {'name': 'Marcus P. Wong'}, {'name': 'Colin M. Warnes'}, {'name': 'Daniel R. Sandoval'}, {'name': 'Thomas Mandel Clausen'}, {'name': 'Yale A. Santos'}, {'name': 'Victoria Ortega'}, {'name': 'Hector C. Aguilar'}, {'name': 'Jeffrey D. Esko'}, {'name': 'Charles Y. Chui'}, {'name': 'John E. Pak'}, {'name': 'P. Robert Beatty'}, {'name': 'Eva Harris'}]\", 'abstract': '<p>Severe COVID-19 is associated with epithelial and endothelial barrier dysfunction within the lung as well as in distal organs. While it is appreciated that an exaggerated inflammatory response is associated with barrier dysfunction, the triggers of this pathology are unclear. Here, we report that cell-intrinsic interactions between the Spike (S) glycoprotein of SARS-CoV-2 and epithelial/endothelial cells are sufficient to trigger barrier dysfunction <italic>in vitro</italic> and vascular leak <italic>in vivo</italic>, independently of viral replication and the ACE2 receptor. We identify an S-triggered transcriptional response associated with extracellular matrix reorganization and TGF- signaling. Using genetic knockouts and specific inhibitors, we demonstrate that glycosaminoglycans, integrins, and the TGF- signaling axis are required for S-mediated barrier dysfunction. Our findings suggest that S interactions with barrier cells are a contributing factor to COVID-19 disease severity and offer mechanistic insight into SARS-CoV-2 triggered vascular leak, providing a starting point for development of therapies targeting COVID-19 pathogenesis.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472649', 'tweets': 70, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Areas Important for Ecological Connectivity Throughout Canada', 'authors': '[{\\'name\\': \\'Richard Pither\\'}, {\\'name\\': \"Paul O\\'Brien\"}, {\\'name\\': \\'Angela Brennan\\'}, {\\'name\\': \\'Kristen Hirsh-Pearson\\'}, {\\'name\\': \\'Jeff Bowman\\'}]', 'abstract': '<p>Governments around the world have acknowledged the importance of conserving ecological connectivity to help reverse the decline of biodiversity. In this study we employed recent methodological developments in circuit theory to conduct the first pan-Canadian analysis of multi-species connectivity for all terrestrial regions of the country, at a spatial grain sufficient to support local land-management decisions. We developed a movement cost surface with a limited number of thematic categories using the most recently updated land cover data available for the country. We divided the country into 17 tiles and used a wall-to-wall, omnidirectional mode of Circuitscape on each tile in order to assess ecological connectivity throughout entire landscapes as opposed to strictly among protected areas. The resulting raw current density map of Canada revealed heterogenous patterns of current density across the country, strongly influenced by geography, natural barriers, and human development. We included a validation analysis of the output current density map with independent wildlife data from across the country and found that mammal and herpetofauna locations were predicted by areas of high current density. We believe our current density map can be used to identify areas important for connectivity throughout Canada and thereby contribute to efforts to conserve biodiversity.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.12.472252', 'tweets': 66, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Improved neutralization of the SARS-CoV-2 Omicron variant after Pfizer-BioNTech BNT162b2 COVID-19 vaccine boosting', 'authors': \"[{'name': 'Kerri Basile'}, {'name': 'Rebecca J Rockett'}, {'name': 'Kenneth McPhie'}, {'name': 'Michael Fennell'}, {'name': 'Jessica Johnson-Mackinnon'}, {'name': 'Jessica E Agius'}, {'name': 'Winkie Fong'}, {'name': 'Hossinur Rahman'}, {'name': 'Danny Ko'}, {'name': 'Linda Donavan'}, {'name': 'Linda Hueston'}, {'name': 'Connie Lam'}, {'name': 'Alicia Arnott'}, {'name': 'Sharon C-A Chen'}, {'name': 'Susan Maddocks'}, {'name': 'Matthew V OSullivan'}, {'name': 'Dominic E Dwyer'}, {'name': 'Vitali Sintchenko'}, {'name': 'Jen Kok'}]\", 'abstract': '<p>In late November 2021, the World Health Organization declared the SARS-CoV-2 lineage B.1.1.529 the fifth variant of concern, Omicron. This variant has acquired 15 mutations in the receptor binding domain of the spike protein, raising concerns that Omicron could evade naturally acquired and vaccine-derived immunity. We utilized an authentic virus, multicycle neutralisation assay to demonstrate that sera collected 1, 3, and 6 months post-two doses of Pfizer-BioNTech BNT162b2 has a limited ability to neutralise SARS-CoV-2. However, four weeks after a third dose, neutralizing antibody titres are boosted. Despite this increase, neutralising antibody titres are reduced 4-fold for Omicron compared to lineage A.2.2 SARS-CoV-2.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472392', 'tweets': 52, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Characterizing the Behavioral and Neuroendocrine Features of Susceptibility and Resilience to Social Stress', 'authors': \"[{'name': 'Dalia Murra'}, {'name': 'Kathryn L. Hilde'}, {'name': 'Anne Fitzpatrick'}, {'name': 'Pamela M. Maras'}, {'name': 'Stanley J. Watson'}, {'name': 'Huda Akil'}]\", 'abstract': '<p>Evaluating and coping with stressful social events as they unfold is a critical strategy in overcoming them without long-lasting detrimental effects. Individuals display a wide range of responses to stress, which can manifest in a variety of outcomes for the brain as well as subsequent behavior. Chronic Social Defeat Stress (CSDS) in mice has been widely used to model individual variation following a social stressor. Following a course of repeated intermittent psychological and physical stress, mice diverge into separate populations of social reactivity: resilient (socially interactive) and susceptible (socially avoidant) animals. A rich body of work reveals distinct neurobiological and behavioral consequences of this experience that map onto the resilient and susceptible groups. However, the range of factors that emerge over the course of defeat have not been fully described. Therefore, in the current study, we focused on characterizing behavioral, physiological, and neuroendocrine profiles of mice in three separate phases: before, during, and following CSDS. We found that following CSDS, traditional read-outs of anxiety-like and depression-like behaviors do not map on to the resilient and susceptible groups. By contrast, behavioral coping strategies used during the initial social stress encounter better predict which mice will eventually become resilient or susceptible. In particular, mice that will emerge as susceptible display greater escape behavior on Day 1 of social defeat than those that will emerge as resilient, indicating early differences in coping mechanisms used between the two groups. We further show that the social avoidance phenotype in susceptible mice is specific to the aggressor strain and does not generalize to conspecifics or other strains, indicating that there may be features of threat discrimination that are specific to the susceptible mice. Our findings suggest that there are costs and benefits to both the resilient and susceptible outcomes, reflected in their ability to cope and adapt to the social stressor.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.12.21267677', 'tweets': 51, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Artificial intelligence for diagnosis and prognosis in neuroimaging for dementia; a systematic review', 'authors': \"[{'name': 'R Borchert'}, {'name': 'T Azevedo'}, {'name': 'A Badhwar'}, {'name': 'J Bernal'}, {'name': 'M Betts'}, {'name': 'R Bruffaerts'}, {'name': 'MC Burkhart'}, {'name': 'I Dewachter'}, {'name': 'HM Gellersen'}, {'name': 'A Low'}, {'name': 'L Machado'}, {'name': 'CR Madan'}, {'name': 'M Malpetti'}, {'name': 'J Mejia'}, {'name': 'S Michopoulou'}, {'name': 'C Muoz-Neira'}, {'name': 'M Peres'}, {'name': 'V Phillips'}, {'name': 'S Ramanan'}, {'name': 'S Tamburin'}, {'name': 'H Tantiangco'}, {'name': 'L Thakur'}, {'name': 'A Tomassini'}, {'name': 'A Vipin'}, {'name': 'E Tang'}, {'name': 'D Newby'}, {'name': 'J Ranson'}, {'name': 'D.J. Llewellyn'}, {'name': 'M Veldsman'}, {'name': 'T Rittman'}]\", 'abstract': '<sec><title>Introduction</title><p>Recent developments in artificial intelligence (AI) and neuroimaging offer new opportunities for improving diagnosis and prognosis of dementia. To synthesise the available literature, we performed a systematic review.</p></sec><sec><title>Methods</title><p>We systematically reviewed primary research publications up to January 2021, using AI for neuroimaging to predict diagnosis and/or prognosis in cognitive neurodegenerative diseases. After initial screening, data from each study was extracted, including: demographic information, AI methods, neuroimaging features, and results.</p></sec><sec><title>Results</title><p>We found 2709 reports, with 252 eligible papers remaining following screening. Most studies relied on the Alzheimers Disease Neuroimaging Initiative (ADNI) dataset (n=178) with no other individual dataset used more than 5 times. Algorithmic classifiers, such as support vector machine (SVM), were the most commonly used AI method (47%) followed by discriminative (32%) and generative (11%) classifiers. Structural MRI was used in 71% of studies with a wide range of accuracies for the diagnosis of neurodegenerative diseases and predicting prognosis. Lower accuracy was found in studies using a multi-class classifier or an external cohort as the validation group. There was improvement in accuracy when neuroimaging modalities were combined, e.g. PET and structural MRI. Only 17 papers studied non-Alzheimers disease dementias.</p></sec><sec><title>Conclusion</title><p>The use of AI with neuroimaging for diagnosis and prognosis in dementia is a rapidly emerging field. We make a number of recommendations addressing the definition of key clinical questions, heterogeneity of AI methods, and the availability of appropriate and representative data. We anticipate that addressing these issues will enable the field to move towards meaningful clinical translation.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.472691', 'tweets': 50, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A reusable benchmark of brain-age prediction from M/EEG resting-state signals', 'authors': \"[{'name': 'Denis A Engemann'}, {'name': 'Apolline Mellot'}, {'name': 'Richard Hoechenberger'}, {'name': 'Hubert Banville'}, {'name': 'David Sabbagh'}, {'name': 'Lukas Gemein'}, {'name': 'Tonio Ball'}, {'name': 'Alexandre Gramfort'}]\", 'abstract': '<p>Population-level modeling can define quantitative measures of individual aging by applying machine learning to large volumes of brain images. These measures of brain age, obtained from the general population, helped characterize disease severity in neurological populations, improving estimates of diagnosis or prognosis. Magnetoencephalography (MEG) and Electroencephalography (EEG) have the potential to further generalize this approach towards prevention and public health by enabling assessments of brain health at large scales in socioeconomically diverse environments. However, more research is needed to define methods that can handle the complexity and diversity of M/EEG signals across diverse real-world contexts. To catalyse this effort, here we propose reusable benchmarks of competing machine learning approaches for brain age modeling. We benchmarked popular classical machine learning pipelines and deep learning architectures previously used for pathology decoding or brain age estimation in 4 international M/EEG cohorts from diverse countries and cultural contexts, including recordings from more than 2500 participants. Our benchmarks were built on top of the M/EEG adaptations of the BIDS standard, providing tools that can be applied with minimal modification on any M/EEG dataset provided in the BIDS format. Our results suggest that, regardless of whether classical machine learning or deep learning was used, the highest performance was reached by pipelines and architectures involving spatially aware representations of the M/EEG signals, leading to R^2 scores between 0.60-0.71. Hand-crafted features paired with random forest regression provided robust benchmarks even in situations in which other approaches failed. Taken together, this set of benchmarks, accompanied by open-source software and high-level Python scripts, can serve as a starting point and quantitative reference for future efforts at developing M/EEG-based measures of brain aging. The generality of the approach renders this benchmark reusable for other related objectives such as modeling specific cognitive variables or clinical endpoints.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472719', 'tweets': 48, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Striking Antibody Evasion Manifested by the Omicron Variant of SARS-CoV-2', 'authors': \"[{'name': 'Lihong Liu'}, {'name': 'Sho Iketani'}, {'name': 'Yicheng Guo'}, {'name': 'Jasper Fuk-Woo Chan'}, {'name': 'Maple Wang'}, {'name': 'Liyuan Liu'}, {'name': 'Yang Luo'}, {'name': 'Hin Chu'}, {'name': 'Yiming Huang'}, {'name': 'Manoj S. Nair'}, {'name': 'Jian Yu'}, {'name': 'Kenn Ka-Heng Chik'}, {'name': 'Terrence Tsz-Tai Yuen'}, {'name': 'Chaemin Yoon'}, {'name': 'Kelvin Kai-Wang To'}, {'name': 'Honglin Chen'}, {'name': 'Michael T. Yin'}, {'name': 'Magdalena E. Sobieszczyk'}, {'name': 'Yaoxing Huang'}, {'name': 'Harris H. Wang'}, {'name': 'Zizhang Sheng'}, {'name': 'Kwok-Yung Yuen'}, {'name': 'David D. Ho'}]\", 'abstract': '<p>The Omicron (B.1.1.529) variant of SARS-CoV-2 was only recently detected in southern Africa, but its subsequent spread has been extensive, both regionally and globally.  It is expected to become dominant in the coming weeks, probably due to enhanced transmissibility.  A striking feature of this variant is the large number of spike mutations that pose a threat to the efficacy of current COVID-19 vaccines and antibody therapies. This concern is amplified by the findings from our study.  We found B.1.1.529 to be markedly resistant to neutralization by serum not only from convalescent patients, but also from individuals vaccinated with one of the four widely used COVID-19 vaccines. Even serum from persons vaccinated and boosted with mRNA-based vaccines exhibited substantially diminished neutralizing activity against B.1.1.529. By evaluating a panel of monoclonal antibodies to all known epitope clusters on the spike protein, we noted that the activity of 18 of the 19 antibodies tested were either abolished or impaired, including ones currently authorized or approved for use in patients. In addition, we also identified four new spike mutations (S371L, N440K, G446S, and Q493R) that confer greater antibody resistance to B.1.1.529. The Omicron variant presents a serious threat to many existing COVID-19 vaccines and therapies, compelling the development of new interventions that anticipate the evolutionary trajectory of SARS-CoV-2.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472096', 'tweets': 40, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'The ParB clamp docks onto Smc for DNA loading via a joint-ParB interface', 'authors': \"[{'name': 'Florian P. Bock'}, {'name': 'Anna Anchimiuk'}, {'name': 'Marie-Laure Diebold-Durand'}, {'name': 'Stephan Gruber'}]\", 'abstract': '<p>Chromosomes readily unlink from one another and segregate to daughter cells during cell division highlighting a remarkable ability of cells to organize long DNA molecules. SMC complexes mediate chromosome folding by DNA loop extrusion. In most bacteria, SMC complexes start loop extrusion at the ParB/<italic>parS</italic> partition complex formed near the replication origin. Whether they are recruited by recognizing a specific DNA structure in the partition complex or a protein component is unknown. By replacing genes in <italic>Bacillus subtilis</italic> with orthologous sequences from <italic>Streptococcus pneumoniae</italic>, we show that the three subunits of the bacterial Smc complex together with the ParB protein form a functional module that can organize and segregate chromosomes when transplanted into another organism. Using chimeric proteins and chemical cross-linking, we find that ParB binds to the Smc subunit directly. We map a binding interface to the Smc joint and the ParB CTP-binding domain. Structure prediction indicates how the ParB clamp presents DNA to the Smc complex to initiate DNA loop extrusion.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.12.21267684', 'tweets': 38, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': True, 'title': 'The impact of the COVID-19 school closure on adolescents use of mental healthcare services in Sweden', 'authors': \"[{'name': 'Helena Svaleryd'}, {'name': 'Evelina Bjrkegren'}, {'name': 'Jonas Vlachos'}]\", 'abstract': '<sec><title>Background</title><p>School closures used to contain the COVID-19 pandemic may have negative impacts on students mental health but credible evidence is scarce. Sweden moved upper-secondary students to remote learning but, as the only country in the OECD, kept schools at lower levers open throughout the pandemic.</p></sec><sec><title>Methods</title><p>Using nationwide register data, we estimate the difference in the use of mental healthcare services between upper- and lower-secondary students during the pandemic, and relate this to the same group difference prior to the pandemic. For the main analysis, this difference-in-difference approach is applied to the period April-June 2020 when upper-secondary schools were fully closed. We also study the periods July-December 2020 when upper-secondary schools were largely open, and January-March 2021 when they were partially closed. We study the impact on all contacts with hospitals and specialist psychiatric care due to mental and behavioral disorder, as well as prescriptions for antidepressants, insomnia, and ADHD drugs.</p></sec><sec><title>Findings</title><p>Compared with expected rates, the use of mental healthcare services among upper-secondary students fell by -3.71 [CI95 -5.52 to -1.91] cases per 1000 during April-June 2020, largely due to a reduction in depression and anxiety-related diagnoses (-1.49; CI95 [-2.36 to -0.63]) and prescriptions (-1.80; CI95 [-2.93 to -0.68]). This reduction in the use of mental healthcare services corresponds to a 4.36% CI95 [-6.47 to -2.25]) decrease compared to the level prior to the pandemic. The decrease compared to expected rates persists through July-December 2020 (-3.55%; CI95 [-5.38 to -1.71]) and January-March 2021 (-5.23%; CI95 [-7.24 to -3.21]). The reduction is stronger among students in the 2<sup>nd</sup> (-5.06%; CI95 [-8.02 to -2.09]) and 3<sup>rd</sup> (-4.86%; CI95 [-8.19 to -1.53]) year of upper-secondary school. The decrease is concentrated to students who was not in contact with mental healthcare services earlier in the academic year (-16.70%; CI95 [-22.20 to -11.20]). The relative reduction is largest for unplanned care (-13.88%; CI95 [-19.35 to -8.42]) and care at emergency units (-18.19%; CI95 [-26.44 to -9.92]).</p></sec><sec><title>Interpretation</title><p>Closing upper-secondary schools in Sweden reduced use of mental healthcare services. There is no indication of this being due to reduced accessibility. In a setting with no strict lockdown, moving to online teaching for a limited period did not worsen mental health among students in upper-secondary schools.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.10.472164', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'EraSOR: Erase Sample Overlap in polygenic score analyses', 'authors': \"[{'name': 'Shing Wan Choi'}, {'name': 'Timothy Shin Heng Mak'}, {'name': 'Clive J. Hoggart'}, {'name': 'Paul F. OReilly'}]\", 'abstract': '<sec><title>Background</title><p>Polygenic risk score (PRS) analyses are now routinely applied in biomedical research, with great hope that they will aid in our understanding of disease aetiology and contribute to personalized medicine. The continued growth of multi-cohort genome-wide association studies (GWASs) and large-scale biobank projects has provided researchers with a wealth of GWAS summary statistics and individual-level data suitable for performing PRS analyses. However, as the size of these studies increase, the risk of inter-cohort sample overlap and close relatedness increases. Ideally sample overlap would be identified and removed directly, but this is typically not possible due to privacy laws or consent agreements. This sample overlap, whether known or not, is a major problem in PRS analyses because it can lead to inflation of type 1 error and, thus, erroneous conclusions in published work.</p></sec><sec><title>Results</title><p>Here, for the first time, we report the scale of the sample overlap problem for PRS analyses by generating known sample overlap across sub-samples of the UK Biobank data, which we then use to produce GWAS and target data to mimic the effects of inter-cohort sample overlap. We demonstrate that inter-cohort overlap results in a significant and often substantial inflation in the observed PRS-trait association, coefficient of determination (R<sup>2</sup>) and false-positive rate. This inflation can be high even when the absolute number of overlapping individuals is small if this makes up a notable fraction of the target sample. We develop and introduce EraSOR (<underline>Era</underline>se <underline>S</underline>ample <underline>O</underline>verlap and <underline>R</underline>elatedness), a software for adjusting inflation in PRS prediction and association statistics in the presence of sample overlap or close relatedness between the GWAS and target samples. A key component of the EraSOR approach is inference of the degree of sample overlap from the intercept of a bivariate LD score regression applied to the GWAS and target data, making it powered in settings where both have sample sizes over 1,000 individuals. Through extensive benchmarking using UK Biobank and HapGen2 simulated genotype-phenotype data, we demonstrate that PRSs calculated using EraSOR-adjusted GWAS summary statistics are robust to inter-cohort overlap in a wide range of realistic scenarios and are even robust to high levels of residual genetic and environmental stratification.</p></sec><sec><title>Conclusion</title><p>The results of all PRS analyses for which sample overlap cannot be definitively ruled out should be considered with caution given high type 1 error observed in the presence of even low overlap between base and target cohorts. Given the strong performance of EraSOR in eliminating inflation caused by sample overlap in PRS studies with large (&gt;5k) target samples, we recommend that EraSOR be used in all future such PRS studies to mitigate the potential effects of inter-cohort overlap and close relatedness.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472585', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Compared with SARS-CoV2 wild types spike protein, the SARS-CoV2 omicrons receptor binding motif has adopted a more SARS-CoV1 and/or bat/civet-like structure', 'authors': \"[{'name': 'Michael O. Glocker'}, {'name': 'Kwabena F.M. Opuni'}, {'name': 'Hans-Juergen Thiesen'}]\", 'abstract': '<p>Our study focuses on free energy calculations of SARS-CoV2 spike protein receptor binding motives (RBMs) from wild type and variants-of-concern with particular emphasis on currently emerging SARS- CoV2 omicron variants of concern (VOC). Our computational free energy analysis underlines the occurrence of positive selection processes that specify omicron host adaption and bring changes on the molecular level into context with clinically relevant observations. Our free energy calculations studies regarding the interaction of omicrons RBM with human ACE2 shows weaker binding to ACE2 than alphas, deltas, or wild types RBM. Thus, less virus is predicted to be generated in time per infected cell. Our mutant analyses predict with focus on omicron variants a reduced spike-protein binding to ACE2-receptor protein possibly enhancing viral fitness / transmissibility and resulting in a delayed induction of danger signals as trade-off. Finally, more virus is produced but less per cell accompanied with delayed Covid-19 immunogenicity and pathogenicity. Regarding the latter, more virus is assumed to be required to initiate inflammatory immune responses.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.21267729', 'tweets': 36, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Interdependencies between cellular and humoral immune responses in heterologous and homologous SARS-CoV-2 vaccination', 'authors': \"[{'name': 'Moritz M. Hollstein'}, {'name': 'Lennart Mnsterktter'}, {'name': 'Michael P. Schn'}, {'name': 'Armin Bergmann'}, {'name': 'Thea M. Husar'}, {'name': 'Anna Abratis'}, {'name': 'Abass Eidizadeh'}, {'name': 'Meike Schaffrinski'}, {'name': 'Karolin Zachmann'}, {'name': 'Anne Schmitz'}, {'name': 'Jason Scott Holsapple'}, {'name': 'Hedwig Stanisz-Bogeski'}, {'name': 'Julie Schanz'}, {'name': 'Uwe Gro'}, {'name': 'Andreas Leha'}, {'name': 'Andreas E. Zautner'}, {'name': 'Moritz Schnelle'}, {'name': 'Luise Erpenbeck'}]\", 'abstract': '<sec><title>Background</title><p>Homologous and heterologous SARS-CoV-2-vaccinations yield different spike protein-directed humoral and cellular immune responses. However, their interdependencies remain elusive.</p></sec><sec><title>Methods</title><p>COV-ADAPT is a prospective, observational cohort study of 417 healthcare workers who received homologous vaccination with Astra (ChAdOx1-S; AstraZeneca) or BNT (BNT162b2; Biontech/Pfizer) or heterologous vaccination with Astra/BNT. We assessed the humoral (anti-spike-RBD-IgG, neutralizing antibodies, antibody avidity) and cellular (spike-induced T cell interferon- release) immune response in blood samples up to 2 weeks before (T1) and 2 to 12 weeks following secondary immunization (T2).</p></sec><sec><title>Findings</title><p>Initial vaccination with Astra resulted in lower anti-spike-RBD-IgG responses compared to BNT (70114 vs. 226279 BAU/ml, p&lt;0.01) at T1, whereas T cell activation did not differ significantly. Booster vaccination with BNT proved superior to Astra at T2 (anti-spike-RBD-IgG: Astra/BNT 23871627 and BNT/BNT 32022184 vs. Astra/Astra 413461 BAU/ml, both p&lt;0.001; spike-induced T cell interferon-<italic></italic> release: Astra/BNT 50696733 and BNT/BNT 48807570 vs. Astra/Astra 11522243 mIU/ml, both p&lt;0.001). No significant differences were detected between BNT-boostered groups at T2. For Astra, we observed no booster effect on T cell activation. We found associations between anti-spike-RBD-IgG levels (Astra/BNT and BNT/BNT) and T cell responses (Astra/Astra and Astra/BNT) from T1 to T2. There were also links between levels of anti-spike-RBD-IgG and T cell at both time points (all groups combined). All regimes yielded neutralizing antibodies and increased antibody avidity at T2.</p></sec><sec><title>Interpretation</title><p>Interdependencies between humoral and cellular immune responses differ between common SARS-CoV-2 vaccination regimes. T cell activation is unlikely to compensate for poor humoral responses.</p></sec><sec><title>Funding</title><p>Deutsche Forschungsgemeinschaft (DFG), ER723/3-1</p></sec><sec><title>Research in context</title><sec><title>Evidence before this study</title><p>We searched Pubmed for papers published between 01/01/2019 and 14/05/2021 with the search terms covid-19 combined with vaccination and heterologous, excluding BCG. Of the 41 papers found, none addressed the combination of ChAdOx1-S by AstraZeneca (Astra) and BNT162b2 by Biontech/Pfizer (BNT). After our study was initiated, the CombiVacS trial reported a significant booster effect when BNT was given after initial vaccination with Astra.<sup>1</sup> The investigators of the CoCo trial subsequently published data on heterologous immunization in comparison to homologous Astra in a small population (n=87), with the heterologous immunization scheme showing a superior humoral and cellular immune response.<sup>2</sup> Further studies investigated heterologous vaccinations with Astra and BNT as well as homologous Astra and BNT regimes and also found superior humoral and cellular immune responses in the heterologous regimes compared to homologous Astra, and comparable or slightly superior immune responses when compared to homologous BNT vaccination.<sup>36</sup> The body of research covering the effects of heterologous immunization regimes has recently been aggregated in a systematic review.<sup>7</sup></p></sec><sec><title>Added value of this study</title><p>To our knowledge, this is the first study that evaluates the interdependencies of cellular and humoral immune responses following heterologous vaccination with Astra/BNT in a large group of individuals. Our data show strong correlations between humoral and cellular immune responses with the prime-boost combination Astra/BNT. The findings suggest that individuals with a robust initial response developed strong humoral and cellular immune responses after booster immunization.</p></sec><sec><title>Implications of all the available evidence</title><p>Our study and the available data suggest that due to its superior capacity to elicit a humoral and cellular immune response, mRNA-based vaccines such as BNT should be chosen for booster vaccination rather than Astra. This seems to be particularly important in individuals whose immune response was poor after initial vaccination with Astra.</p><p>We demonstrate here an association between humoral and cellular immune responses following vaccination. Our findings suggest that distinct differences between common COVID-19 vaccination regimes should be taken into account in population-based vaccine programs. The present data indicate that a poor humoral immune response is unlikely to be mitigated by a strong cellular immune response.</p></sec></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472419', 'tweets': 35, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Generative Language Modeling for Antibody Design', 'authors': \"[{'name': 'Richard W. Shuai'}, {'name': 'Jeffrey A. Ruffolo'}, {'name': 'Jeffrey J. Gray'}]\", 'abstract': '<p>Successful development of monoclonal antibodies (mAbs) for therapeutic applications is hindered by developability issues such as low solubility, low thermal stability, high aggregation, and high immunogenicity. The discovery of more developable mAb candidates relies on high-quality antibody libraries for isolating candidates with desirable properties. We present Immunoglobulin Language Model (IgLM), a deep generative language model for generating synthetic libraries by re-designing variable-length spans of antibody sequences. IgLM formulates anti-body design as an autoregressive sequence generation task based on text-infilling in natural language. We trained IgLM on approximately 558M antibody heavy- and light-chain variable sequences, conditioning on each sequences chain type and species-of-origin. We demonstrate that IgLM can be applied to generate synthetic libraries that may accelerate the discovery of therapeutic antibody candidates.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.09.472009', 'tweets': 34, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A somatosensory computation that unifies limbs and tools', 'authors': \"[{'name': 'Luke E. Miller'}, {'name': 'Ccile Fabio'}, {'name': 'Frdrique de Vignemont'}, {'name': 'Alice Roy'}, {'name': 'W. Pieter Medendorp'}, {'name': 'Alessandro Farn'}]\", 'abstract': '<p>It is often claimed that tools are embodied by the user, but whether the brain actually repurposes its body-based computations to perform similar tasks with tools is not known. A fundamental body-based computation used by the somatosensory system is <italic>trilateration</italic>. Here, the location of touch on a limb is computed by integrating estimates of the distance between sensory input and its boundaries (e.g., elbow and wrist of the forearm). As evidence of this computational mechanism, tactile localization on a limb is most precise near its boundaries and lowest in the middle. If the brain repurposes trilateration to localize touch on a tool, we should observe this computational signature in behavior. In a large sample of participants, we indeed found that localizing touch on a tool produced the signature of trilateration, with highest precision close to the base and tip of the tool. A computational model of trilateration provided a good fit to the observed localization behavior. Importantly, model selection demonstrated that trilateration better explained each participants behavior than an alternative model of localization. These results have important implications for how trilateration may be implemented by somatosensory neural populations. In sum, the present study suggests that tools are indeed embodied at a computational level, repurposing a fundamental spatial computation.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.12.472330', 'tweets': 34, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Yield is negatively correlated with nucleotide-binding leucine-rich repeat gene content in soybean', 'authors': \"[{'name': 'Philipp E. Bayer'}, {'name': 'Haifei Hu'}, {'name': 'Jakob Petereit'}, {'name': 'Mark C Derbyshire'}, {'name': 'Rajeev K. Varshney'}, {'name': 'Babu Valliyodan'}, {'name': 'Henry T. Nguyen'}, {'name': 'Jacqueline Batley'}, {'name': 'David Edwards'}]\", 'abstract': '<p>The availability of increasing quantities of crop pangenome data permits the detailed association of gene content with agronomic traits. Here, we investigate disease resistance gene content of diverse soybean cultivars and report a significant negative correlation between the number of NLR resistance (<italic>R</italic>) genes and yield. We find no association between <italic>R-</italic>genes with seed weight, oil or protein content, and we find no correlation between yield and the number of RLK, RLP genes, or the total number of genes. These results suggest that recent yield improvement in soybean may be partially associated with the selective loss of NLR genes. Three quarters of soybean NLR genes do not show presence/absence variation, limiting the ability to select for their absence, and so the deletion or disabling of select NLR genes may support future yield improvement.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.21267756', 'tweets': 32, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'A spectrum of recessiveness among Mendelian disease variants in UK Biobank', 'authors': \"[{'name': 'Alison R. Barton'}, {'name': 'Margaux L.A. Hujoel'}, {'name': 'Ronen E. Mukamel'}, {'name': 'Maxwell A. Sherman'}, {'name': 'Po-Ru Loh'}]\", 'abstract': '<p>Recent work has found increasing evidence of mitigated, incompletely penetrant phenotypes in heterozygous carriers of recessive Mendelian disease variants. We leveraged whole-exome imputation within the full UK Biobank cohort (<italic>N</italic>500K) to extend such analyses to 3,481 rare variants curated from ClinVar and OMIM. Testing these variants for association with 57 quantitative traits yielded 103 significant associations involving variants previously implicated in 35 different diseases. Notable examples included a <italic>POR</italic> missense variant implicated in Antley-Bixler syndrome that associated with a 1.76 (s.e. 0.27) cm increase in height, and an <italic>ABCA3</italic> missense variant implicated in interstitial lung disease that associated with reduced FEV1/FVC ratio. Association analyses with 1,257 disease traits yielded five additional variant-disease associations. We also observed contrasting levels of recessiveness between two more-common, classical Mendelian diseases. Carriers of cystic fibrosis variants exhibited increased risk of several mitigated disease phenotypes, whereas carriers of spinal muscular atrophy alleles showed no evidence of altered phenotypes. Incomplete penetrance of cystic fibrosis carrier phenotypes did not appear to be mediated by common allelic variation on the functional haplotype. Our results show that many disease-associated recessive variants can produce mitigated phenotypes in heterozygous carriers and motivate further work exploring penetrance mechanisms.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.15.472746', 'tweets': 31, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Astrocytes in the dorsal vagal complex are not activated by systemic glucoprivation and their chemogenetic activation does not elicit homeostatic glucoregulatory responses in mice', 'authors': \"[{'name': 'Alastair J MacDonald'}, {'name': 'Katherine R Pye'}, {'name': 'Craig Beall'}, {'name': 'Kate LJ Ellacott'}]\", 'abstract': '<p>The dorsal vagal complex (DVC) is a brainstem site regulating diverse aspects of physiology including food intake and blood glucose homeostasis. Astrocytes are purported to play an active role in regulating DVC function and, by extension, physiological parameters. Previous work has demonstrated that DVC astrocytes directly sense hormones that regulate food intake and blood glucose and are critical for their effect. In addition, DVC astrocytes in ex vivo slices respond to low tissue glucose. The response of neurons, including catecholaminergic neurons, to low glucose is conditional on intact astrocyte signalling in slice preparations suggesting astrocytes are possibly the primary sensors of glucose deprivation (glucoprivation). Based on these findings we hypothesised that if DVC astrocytes act as glucoprivation sensors in vivo they would both show a response to systemic glucoprivation and drive physiological responses to restore blood glucose. We found that 2 hours of systemic glucoprivation induced neither FOS nor glial fibrillary acidic protein (GFAP)-immunoreactivity in DVC astrocytes, specifically those in the nucleus of the solitary tract (NTS). Furthermore, we found that while chemogenetic activation of DVC astrocytes suppressed food intake by reducing both meal size and meal number, this manipulation also suppressed food intake under conditions of glucoprivation. Chemogenetic activation of DVC astrocytes did not increase basal blood glucose nor protect against insulin-induced hypoglycaemia. In male mice chemogenetic DVC astrocyte activation did not alter glucose tolerance, in female mice the initial glucose excursion was reduced, suggesting enhanced glucose absorption. Taken together this suggests that as a whole-population DVC astrocytes do not function as glucoprivation sensors in vivo in mice. Instead, we propose that DVC astrocytes play an indispensable, homeostatic role to maintain the function of glucoregulatory neuronal circuitry.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472866', 'tweets': 31, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Understanding the source of METTL3-independent m6A in mRNA', 'authors': \"[{'name': 'Hui Xian Poh'}, {'name': 'Aashiq H Mirza'}, {'name': 'Brian F Pickering'}, {'name': 'Samie R Jaffrey'}]\", 'abstract': \"<p><italic>N</italic><sup>6</sup>-methyladenosine (m<sup>6</sup>A) is a highly prevalent mRNA modification which promotes degradation of transcripts encoding proteins that have roles in cell development, differentiation, and other pathways. METTL3 is the major methyltransferase that catalyzes the formation of m<sup>6</sup>A in mRNA. As 3080% of m<sup>6</sup>A can remain in mRNA after METTL3 depletion by CRISPR/Cas9-based methods, other enzymes are thought to catalyze a sizable fraction of m<sup>6</sup>A. Here, we re-examined the source of m<sup>6</sup>A in the mRNA transcriptome. We characterized mouse embryonic stem cell lines which continue to have m<sup>6</sup>A in their mRNA after <italic>Mettl3</italic> knockout. We show that these cells express alternatively spliced <italic>Mettl3</italic> transcript isoforms that bypass the CRISPR/Cas9 mutations and produce functionally active methyltransferases. We similarly show that other reported <italic>METTL3</italic> knockout cell lines express altered METTL3 proteins. We find that gene dependency datasets show that most cell lines fail to proliferate after <italic>METTL3</italic> deletion, suggesting that reported <italic>METTL3</italic> knockout cell lines express altered METTL3 proteins rather than have full knockout. Finally, we reassessed METTL3's role in synthesizing m<sup>6</sup>A using a genomic deletion of <italic>Mettl3</italic>, and found that METTL3 is responsible for &gt;95% of m<sup>6</sup>A in mRNA. Overall, these studies suggest that METTL3 is responsible for the vast majority of m<sup>6</sup>A in the transcriptome, and that remaining m<sup>6</sup>A in putative <italic>METTL3</italic> knockout cell lines is due to the expression of altered but functional METTL3 isoforms.</p>\", 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.473016', 'tweets': 30, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Centromere identity is dependent on nuclear spatial organization', 'authors': \"[{'name': 'Weifang Wu'}, {'name': 'Toni McHugh'}, {'name': 'David A Kelly'}, {'name': 'Alison L Pidoux'}, {'name': 'Robin C Allshire'}]\", 'abstract': '<p>The establishment of centromere-specific CENP-A chromatin is influenced by epigenetic and genetic processes. Central domain sequences from fission yeast centromeres are preferred substrates for CENP-A<sup>Cnp1</sup> incorporation, but their use is context dependent, requiring adjacent heterochromatin. CENP-A<sup>Cnp1</sup> overexpression bypasses heterochromatin dependency, suggesting heterochromatin ensures exposure to conditions or locations permissive for CENP-A<sup>Cnp1</sup> assembly. Centromeres cluster around spindle-pole bodies (SPBs). We show that heterochromatin-bearing minichromosomes localize close to SPBs, consistent with this location promoting CENP-A<sup>Cnp1</sup> incorporation. We demonstrate that heterochromatin-independent de novo CENP-A<sup>Cnp1</sup> chromatin assembly occurs when central domain DNA is placed near, but not far from, endogenous centromeres or neocentromeres. Moreover, direct tethering of central domain DNA at SPBs permits CENP-A<sup>Cnp1</sup> assembly, suggesting that the nuclear compartment surrounding SPBs is permissive for CENP-A<sup>Cnp1</sup> incorporation because target sequences are exposed to high levels of CENP-A<sup>Cnp1</sup> and associated assembly factors. Thus, nuclear spatial organization is a key epigenetic factor that influences centromere identity.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.472438', 'tweets': 30, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Artificial selection of communities drives the emergence of structured interactions', 'authors': \"[{'name': 'Jules Fraboul'}, {'name': 'Giulio Biroli'}, {'name': 'Silvia De Monte'}]\", 'abstract': '<p>Species-rich communities, such as the microbiota or environmental microbial assemblages, provide key functions for human health and ecological resilience. Increasing effort is being dedicated to design experimental protocols for selecting community-level functions of interest. These experiments typically involve selection acting on populations of communities, each of which is composed of multiple species. Numerical explorations allowed to link the evolutionary dynamics to the multiple parameters involved in this complex, multi-scale evolutionary process. However, a comprehensive theoretical understanding of artificial selection of communities is still lacking. Here, we propose a general model for the evolutionary dynamics of species-rich communities, each described by disordered generalized Lotka-Volterra equations, that we study analytically and by numerical simulations. Our results reveal that a generic response to selection for larger total community abundance is the emergence of an isolated eigenvalue of the interaction matrix that can be understood as an effective cross-feeding term. In this way, selection imprints a structure on the community, which results in a global increase of both the level of mutualism and the diversity of interactions. Our approach moreover allows to disentangle the role of intraspecific competition, interspecific interactions symmetry and number of selected communities in the evolutionary process, and can thus be used as a guidance in optimizing artificial selection protocols.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472426', 'tweets': 30, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Absolute Scaling of Single-Cell Transcriptomes Reveals Pervasive Hypertranscription in Adult Stem and Progenitor Cells', 'authors': \"[{'name': 'Yun-Kyo Kim'}, {'name': 'Miguel Ramalho-Santos'}]\", 'abstract': '<p>Hypertranscription facilitates biosynthetically demanding cellular state transitions through global upregulation of the nascent transcriptome. Despite its potential widespread relevance, documented examples of hypertranscription remain few and limited predominantly to early development. This limitation is in large part due to the fact that modern sequencing approaches, including single-cell RNA sequencing (scRNA-seq), generally assume similar levels of transcriptional output per cell. Here, we use molecule counting and spike-in normalization to develop absolute scaling of single-cell RNA sequencing data. Absolute scaling enables an estimation of total transcript abundances per cell, which we validate in embryonic stem cell (ESC) and germline data and apply to adult mouse organs at steady-state or during regeneration. The results reveal a remarkable dynamic range in transcriptional output among adult cell types. We find that many different multipotent stem and progenitor cell populations are in a state of hypertranscription, including in the hematopoietic system, intestine and skin. Hypertranscription marks cells with multilineage potential in adult organs, is redeployed in conditions of tissue injury, and can precede by 1-2 days bursts of proliferation during regeneration. In addition to the association between hypertranscription and the stem/progenitor cell state, we dissect the relationship between transcriptional output and cell cycle, ploidy and secretory behavior. Our analyses reveal a common set of molecular pathways associated with hypertranscription across adult organs, including chromatin remodeling, DNA repair, ribosome biogenesis and translation. Our findings introduce an approach towards maximizing single-cell RNA-seq profiling. By applying this methodology across a diverse collection of cell states and contexts, we put forth hypertranscription as a general and dynamic cellular program that is pervasively employed during development, organ maintenance and regeneration.</p><sec><title>SUMMARY STATEMENT</title><p>Absolute scaling of single-cell transcriptomic data reveals highly dynamic global levels of transcription across adult cell lineages</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472377', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'An intrinsic endothelial dysfunction causes cerebral small vessel disease', 'authors': \"[{'name': 'Sophie Quick'}, {'name': 'Tessa V. Procter'}, {'name': 'Jonathan Moss'}, {'name': 'Angus Lawson'}, {'name': 'Serena Baker'}, {'name': 'Marc Walton'}, {'name': 'Mehreen Mohammad'}, {'name': 'Will Mungall'}, {'name': 'Ami Onishi'}, {'name': 'Zuzanna Tobola'}, {'name': 'Michael Stringer'}, {'name': 'Maurits A. Jansen'}, {'name': 'Antoine Vallatos'}, {'name': 'Ylenia Giarratano'}, {'name': 'Miguel O. Bernabeu'}, {'name': 'Joanna M. Wardlaw'}, {'name': 'Anna C. Williams'}]\", 'abstract': '<p>Small Vessel Disease (SVD) is the leading cause of vascular dementia, causes a quarter of strokes, and worsens stroke outcomes(1, 2). The disease is characterised by cerebral small vessel and white matter pathology, but the underlying mechanisms are poorly understood. Classically, the microvascular and tissue damage has been considered secondary to extrinsic factors, such as hypertension, consisting of microvessel stiffening, impaired vasoreactivity and blood-brain barrier dysfunction identified in human sporadic SVDs. However, increasing evidence points to an underlying vulnerability to SVD-related brain damage, not just extrinsic factors. Here, in a novel normotensive transgenic rat model where the phospholipase flippase <italic>Atp11b</italic> is deleted, we show pathological, imaging and behavioural changes typical of those in human sporadic SVD, but that occur without hypertension. These changes are due to an intrinsic endothelial cell dysfunction, identified in vessels of the brain white matter and the retina, with pathological evidence of vasoreactivity and blood-brain barrier deficits, which precipitate a secondary maturation block in oligodendroglia and myelin disruption around the small vessels. This highlights that an intrinsic endothelial dysfunction may underlie vulnerability to human sporadic SVD, providing alternative therapeutic targets to prevent a major cause of stroke and dementia.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.473018', 'tweets': 28, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'vRhyme enables binning of viral genomes from metagenomes', 'authors': \"[{'name': 'Kristopher Kieft'}, {'name': 'Alyssa Adams'}, {'name': 'Rauf Salamzade'}, {'name': 'Lindsay Kalan'}, {'name': 'Karthik Anantharaman'}]\", 'abstract': '<p>Genome binning has been essential for characterization of bacteria, archaea, and even eukaryotes from metagenomes. Yet, no approach exists for viruses. We developed vRhyme, a fast and precise software for construction of viral metagenome-assembled genomes (vMAGs). vRhyme utilizes single- or multi-sample coverage effect size comparisons between scaffolds and employs supervised machine learning to identity nucleotide feature similarities, which are compiled into iterations of weighted networks and refined bins. Using simulated viromes, we displayed superior performance of vRhyme compared to available binning tools in constructing more complete and uncontaminated vMAGs. When applied to 10,601 viral scaffolds from human skin, vRhyme advanced our understanding of resident viruses, highlighted by identification of a Herelleviridae vMAG comprised of 22 scaffolds, and another vMAG encoding a nitrate reductase metabolic gene, representing near-complete genomes post-binning. vRhyme will enable a convention of binning uncultivated viral genomes and has the potential to transform metagenome-based viral ecology.</p>', 'posted': '2021-12-18'}, {'doi': '10.1101/2021.12.15.21267834', 'tweets': 28, 'archive': 'medRxiv', 'subject-area': 'Allergy and Immunology', 'covid': True, 'title': 'Comparison of the Immunogenicity of five COVID-19 vaccines in Sri Lanka', 'authors': \"[{'name': 'Chandima Jeewandara'}, {'name': 'Inoka Sepali Aberathna'}, {'name': 'Saubhagya Danasekara'}, {'name': 'Laksiri Gomes'}, {'name': 'Suranga Fernando'}, {'name': 'Dinuka Guruge'}, {'name': 'Thushali Ranasinghe'}, {'name': 'Banuri Gunasekera'}, {'name': 'Achala Kamaladasa'}, {'name': 'Heshan Kuruppu'}, {'name': 'Gayasha Somathilake'}, {'name': 'Jeewantha Jayamali'}, {'name': 'Deshni Jayathilaka'}, {'name': 'Helanka Dinesh Kumara Wijayatilake'}, {'name': 'Pradeep Darshana Pushpakumara'}, {'name': 'Michael Harvie'}, {'name': 'Thashmi Nimasha'}, {'name': 'Shiromi Devika Grace de Silva'}, {'name': 'Ruwan Wijayamuni'}, {'name': 'Lisa Schimanski'}, {'name': 'Pramila Rijal'}, {'name': 'Jack Tan'}, {'name': 'Alain Townsend'}, {'name': 'Graham S. Ogg'}, {'name': 'Gathsaurie Neelika Malavige'}]\", 'abstract': '<p>We assessed antibody responses 3 months post-vaccination in those who received mRNA-1273 (n=225), Sputnik V (n=128) or the first dose of Gam-COVID-Vac (n=184) and compared the results with previously reported data of Sinopharm and AZD1222 vaccinees. 99.5% of Moderna &gt;94% of AZD1222 or Sputnik V, 72% to 76% of Gam-COVID-Vac (first dose) and 38.1% to 68.3% of Sinopharm vaccinees had ACE2 blocking antibodies above the positive threshold. The ACE2 blocking antibody levels were highest to lowest was Moderna &gt; Sputnik V/ AZD1222 (had equal levels)&gt; first dose of Gam-COVID-Vac &gt; Sinopharm. All Moderna recipients had antibodies above the positive threshold to the ancestral (WT), B.1.1.7, B.1.351.1 and 80% positivity rate for B.1.617.2. Positivity rates of Sputnik V vaccinees for WT and variants, were higher than AZD1222 vaccinees, while Sinopharm vaccinees had the lowest positivity rates (&lt;16.7%). These findings highlight the need for further studies to understand the effects on clinical outcomes.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472599', 'tweets': 28, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Sir2-domain associated short prokaryotic Argonautes provide defence against invading mobile genetic elements through NAD+ depletion', 'authors': \"[{'name': 'Mindaugas Zaremba'}, {'name': 'Donata Dakineviciene'}, {'name': 'Edvardas Golovinas'}, {'name': 'Edvinas Stankunas'}, {'name': 'Anna Lopatina'}, {'name': 'Rotem Sorek'}, {'name': 'Elena Manakova'}, {'name': 'Audra Ruksenaite'}, {'name': 'Arunas Silanskas'}, {'name': 'Simonas Asmontas'}, {'name': 'Rokas Grigaitis'}, {'name': 'Kstutis Timinskas'}, {'name': 'eslovas Venclovas'}, {'name': 'Virginijus Siksnys'}]\", 'abstract': '<p>Argonaute (Ago) proteins are found in all three domains of life. The so-called long Agos are composed of four major domains (N, PAZ, MID, and PIWI) and contribute to RNA silencing in eukaryotes (eAgos) or defence against invading mobile genetic elements in prokaryotes (pAgos). Intriguingly, the majority (60%) of prokaryotic Agos (pAgos) identified bioinformatically are shorter (comprised of only MID and PIWI domains) and are typically associated with Sir2, Mrr or TIR domain-containing proteins. The cellular function and mechanism of short pAgos remain enigmatic. Here, we show that short pAgos from <italic>Geobacter sulfurreducens, Caballeronia cordobensis</italic> and <italic>Paraburkholderia graminis</italic>, together with the NAD<sup>+</sup>-bound Sir2-proteins form a stable heterodimeric Sir2/Ago complex that recognizes invading plasmid or phage DNA through the pAgos subunit and activates Sir2 subunit triggering the endogenous NAD<sup>+</sup> depletion and cell death thus preventing the propagation of invading DNA. This is the first demonstration that short Sir2-associated pAgos provide defence against phages and plasmids and underscores the diversity of mechanisms of prokaryotic Agos.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.472499', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'The structural repertoire of Fusarium oxysporum f. sp. lycopersici effectors revealed by experimental and computational studies', 'authors': \"[{'name': 'Daniel S. Yu'}, {'name': 'Megan A. Outram'}, {'name': 'Ashley Smith'}, {'name': 'Carl L. McCombe'}, {'name': 'Pravin B. Khambalkar'}, {'name': 'Sharmin A. Rima'}, {'name': 'Xizhe Sun'}, {'name': 'Lisong Ma'}, {'name': 'Daniel J. Ericsson'}, {'name': 'David A. Jones'}, {'name': 'Simon J. Williams'}]\", 'abstract': '<p>Plant pathogens secrete proteins, known as effectors, that function in the apoplast and inside plant cells to promote virulence. Effectors can also be detected by cell-surface and cytosolic receptors, resulting in the activation of defence pathways and plant immunity. Our understanding of fungal effector function and detection by immunity receptors is limited largely due to high sequence diversity and lack of identifiable sequence motifs precluding prediction of structure or function. Recent studies have demonstrated that fungal effectors can be grouped into structural classes despite significant sequence variation. Using protein x-ray crystallography, we identify a new structural class of effectors hidden within the secreted in xylem (SIX) effectors from <italic>Fusarium oxysporum f. sp. lycopersici</italic> (<italic>Fol</italic>). The recognised effectors Avr1 (SIX4) and Avr3 (SIX1) represent the founding members of the Fol dual-domain (FOLD) effector class. Using AlphaFold ab initio protein structure prediction, benchmarked against the experimentally determined structures, we demonstrate SIX6 and SIX13 are FOLD effectors. We show that the conserved N-domain of Avr1 and Avr3 is sufficient for recognition by their corresponding, but structurally-distinct, immunity receptors. Additional structural prediction and comparison indicate that 11 of the 14 SIX effectors group into four structural families. This revealed that genetically linked effectors are related structurally, and we provide direct evidence for a physical association between one divergently-transcribed effector pair. Collectively, these data indicate that Fol secretes groups of structurally-related molecules during plant infection, an observation that has broad implications for our understanding of pathogen virulence and the engineering of plant immunity receptors.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472793', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Dynamic epistasis analysis reveals how chromatin remodeling regulates transcriptional bursting', 'authors': \"[{'name': 'Ineke Brouwer'}, {'name': 'Emma Kerklingh'}, {'name': 'Fred van Leeuwen'}, {'name': 'Tineke Laura Lenstra'}]\", 'abstract': '<p>Transcriptional bursting has been linked to the stochastic positioning of nucleosomes. However, how bursting is regulated by remodeling of promoter nucleosomes is unknown. Here, we use single-molecule live-cell imaging of <italic>GAL10</italic> transcription in budding yeast to measure how transcriptional bursting changes upon single and double perturbations of chromatin remodeling factors, the transcription factor Gal4 and preinitiation complex (PIC) components. Using dynamic epistasis analysis, we reveal how remodeling of different nucleosomes regulates individual transcriptional bursting parameters. At the nucleosome covering the Gal4 binding sites, RSC acts synergistically with Gal4 binding to facilitate each burst. Conversely, nucleosome remodeling at the TATA box controls only the first burst upon galactose induction. In the absence of remodelers, nucleosomes at canonical TATA boxes are displaced by TBP binding to allow for transcription activation. Overall, our results reveal how promoter nucleosome remodeling, together with transcription factor and PIC binding regulates the kinetics of transcriptional bursting.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472556', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Simulated visual hallucinations in virtual reality enhance cognitive flexibility', 'authors': \"[{'name': 'Clara Rastelli'}, {'name': 'Antonino Greco'}, {'name': 'Yoed N. Kenett'}, {'name': 'Chiara Finocchiaro'}, {'name': 'Nicola De Pisapia'}]\", 'abstract': '<p>Historically, psychedelic drugs are known to modulate cognitive flexibility, a central aspect of cognition permitting adaptation to changing environmental demands. Despite proof suggesting phenomenological similarities between artificially-induced and actual psychedelic altered perception, experimental evidence is still lacking about whether the former is also able to modulate cognitive flexibility. To address this, we measure participants cognitive flexibility through behavioral tasks after the exposure to virtual reality panoramic videos and their hallucinatory-like counterparts generated by the DeepDream algorithm. Results show that the estimated semantic network has a flexible structure when preceded by altered videos. Crucially, following the simulated psychedelic exposure, individuals also show an attenuated contribution of the automatic process and chaotic dynamics underlying the decision process. This suggests that simulated altered perceptual phenomenology enhances cognitive flexibility, presumably due to a reorganization in the cognitive dynamics that facilitates the exploration of uncommon decision strategies and inhibits automated choices.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472574', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Combined SHP2 and ERK inhibition for the treatment of KRAS-driven Pancreatic Ductal Adenocarcinoma', 'authors': \"[{'name': 'Katrin J. Ciecielski'}, {'name': 'Antonio Mulero-Snchez'}, {'name': 'Alexandra Berninger'}, {'name': 'Laura Ruiz Caas'}, {'name': 'Astrid Bosma'}, {'name': 'Kvan Grgl'}, {'name': 'Nan Wu'}, {'name': 'Kalliope N. Diakopoulos'}, {'name': 'Ezgi Kaya-Aksoy'}, {'name': 'Dietrich A. Ruess'}, {'name': 'Derya Kabacaolu'}, {'name': 'Frnze Schmidt'}, {'name': 'Larissa Heinemann'}, {'name': 'Yuhui Fan'}, {'name': 'Bram Thijssen'}, {'name': 'Marieke van de Ven'}, {'name': 'Natalie Proost'}, {'name': 'Susanne Kossatz'}, {'name': 'Wolfgang A. Weber'}, {'name': 'Bruno Sainz'}, {'name': 'Rene Bernards'}, {'name': 'Hana Algl'}, {'name': 'Marina Lesina'}, {'name': 'Sara Mainardi'}]\", 'abstract': '<p>Mutant KRAS is present in over 90% of pancreatic as well as 30-40% of lung and colorectal cancers and is one of the most common oncogenic drivers. Despite decades of research and the recent emergence of isoform-specific KRAS<sup>G12C</sup>-inhibitors, most mutant KRAS isoforms, including the ones frequently associated with pancreatic ductal adenocarcinoma (PDAC), cannot be targeted directly. Moreover, targeting single RAS downstream effectors induces adaptive mechanisms leading to tumor recurrence or resistance. We report here on the combined inhibition of SHP2, a non-receptor tyrosine phosphatase upstream of KRAS, and ERK, a serine/threonine kinase and a key molecule downstream of KRAS in PDAC. This combination shows synergistic anticancer activity <italic>in vitro</italic>, superior disruption of the MAPK pathway, and significantly increased apoptosis induction compared to single-agent treatments. <italic>In vivo</italic>, we demonstrate good tolerability and efficacy of the combination. Concurrent inhibition of SHP2 and ERK induces significant tumor regression in multiple PDAC mouse models. Finally, we show evidence that <sup>18</sup>F-FDG PET scans can be used to detect and predict early drug responses in animal models. Based on these compelling results, we will investigate this drug combination in a clinical trial (SHERPA, <bold>SH</bold>P2 and <bold>ER</bold>K inhibition in <bold>pa</bold>ncreatic cancer, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04916236\">NCT04916236</ext-link>), enrolling patients with <italic>KRAS</italic>-mutant PDAC.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472358', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Lessons from the meiotic recombination landscape of the ZMM deficient budding yeast Lachancea waltii', 'authors': \"[{'name': 'Fabien Dutreux'}, {'name': 'Abhishek Dutta'}, {'name': 'Emilien Peltier'}, {'name': 'Sabrina Bibi-Triki'}, {'name': 'Anne Friedrich'}, {'name': 'Bertrand Llorente'}, {'name': 'Joseph Schacherer'}]\", 'abstract': '<p>Meiotic recombination has been deeply characterized in a few model species only, notably in the budding yeast <italic>Saccharomyces cerevisiae</italic>. Interestingly, most members of the ZMM pathway that implements meiotic crossover interference in <italic>S. cerevisiae</italic> have been lost in <italic>Lachancea</italic> yeast species after the divergence of <italic>Lachancea kluyveri</italic> from the rest of the clade. This suggests major differences in the control of crossover distribution. After investigating meiosis in <italic>L. kluyveri</italic>, we determined the meiotic recombination landscape of <italic>Lachancea waltii</italic> and identified several characteristics that should help understand better the underlying mechanisms. Such characteristics include systematic regions of loss of heterozygosity (LOH) in <italic>L. waltii</italic> hybrids, compatible with dysregulated Spo11-mediated DNA double strand breaks (DSB) independently of meiosis. They include a higher recombination rate in <italic>L. waltii</italic> than in <italic>L. kluyveri</italic> despite the lack of multiple ZMM pro-crossover factors. <italic>L. waltii</italic> exhibits an elevated frequency of zero-crossover bivalents as <italic>L. kluyveri</italic> but opposite to <italic>S. cerevisiae. L. waltii</italic> gene conversion tracts lengths are comparable to those observed in <italic>S. cerevisiae</italic> and shorter than in <italic>L. kluyveri</italic> despite the lack of Mlh2, a factor limiting conversion tracts size in <italic>S. cerevisiae. L. waltii</italic> recombination hotspots are not shared with either <italic>S. cerevisiae</italic> or <italic>L. kluyveri</italic>, showing that meiotic recombination hotspots can evolve at a rather limited evolutionary scale within budding yeasts. Finally, in line with the loss of several ZMM genes, we found only residual crossover interference in <italic>L. waltii</italic> likely coming from the modest interference existing between recombination precursors.</p><sec><title>Significance statement</title><p>Studying non-model species is relevant to understand better biological processes by shedding light on their evolutionary variations. Here we chose the non-model budding yeast <italic>Lachancea waltii</italic> to study meiotic recombination. In sexually reproducing organisms, meiotic recombination shuffles parental genetic combinations notably by crossovers that cluster in hotspots at the population level. We found remarkable variations compared to both the canonical <italic>Saccharomyces cerevisiae</italic> model and another close relative <italic>Lachancea kluyveri</italic>. Such variations notably include the loss in <italic>L. waltii</italic> of a layer of regulation of crossover distribution that is otherwise conserved in budding yeasts and mammals. They also include the lack of conservation of crossover hotspots across the <italic>Lachancea</italic> species while crossover hotspots are remarkably stable across the <italic>Saccharomyces</italic> species.</p></sec><sec><title>Highlights</title><list list-type=\"simple\"><list-item><label>-</label><p>Extensive LOH events in <italic>L. waltii</italic> intraspecific hybrids</p></list-item><list-item><label>-</label><p>No conservation of the recombination hotspots across the <italic>Lachancea</italic> genus</p></list-item><list-item><label>-</label><p>Reduced but not suppressed crossover interference in the absence of the ZMM pathway</p></list-item><list-item><label>-</label><p>Similar gene conversion tract lengths in <italic>L. waltii, S. cerevisiae</italic>, and <italic>L. kluyveri</italic> despite the lack of <italic>MLH2</italic> in <italic>L. waltii</italic></p></list-item></list></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.15.472838', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'A third vaccination with a single T cell epitope protects against SARS-CoV-2 infection in the absence of neutralizing antibodies', 'authors': \"[{'name': 'Iris N. Pardieck'}, {'name': 'Esme T.I. van der Gracht'}, {'name': 'Dominique M.B. Veerkamp'}, {'name': 'Felix M. Behr'}, {'name': 'Suzanne van Duikeren'}, {'name': 'Guillaume Beyrend'}, {'name': 'Jasper Rip'}, {'name': 'Reza Nadafi'}, {'name': 'Tetje C. van der Sluis'}, {'name': 'Elham Beyranvand Nejad'}, {'name': 'Nils Mulling'}, {'name': 'Dena J. Brasem'}, {'name': 'Marcel G.M. Camps'}, {'name': 'Sebenzile K. Myeni'}, {'name': 'Peter J. Bredenbeek'}, {'name': 'Marjolein Kikkert'}, {'name': 'Yeonsu Kim'}, {'name': 'Luka Cicin-Sain'}, {'name': 'Tamim Abdelaal'}, {'name': 'Klaas P.J.M. van Gisbergen'}, {'name': 'Kees L.M.C. Franken'}, {'name': 'Jan Wouter Drijfhout'}, {'name': 'Cornelius J.M. Melief'}, {'name': 'Gerben C.M. Zondag'}, {'name': 'Ferry Ossendorp'}, {'name': 'Ramon Arens'}]\", 'abstract': '<p>Understanding the mechanisms and impact of booster vaccinations can facilitate decisions on vaccination programmes. This study shows that three doses of the same synthetic peptide vaccine eliciting an exclusive CD8+ T cell response against one SARS-CoV-2 Spike epitope protected all mice against lethal SARS-CoV-2 infection in the K18-hACE2 transgenic mouse model in the absence of neutralizing antibodies, while only a second vaccination with this T cell vaccine was insufficient to provide protection. The third vaccine dose of the single T cell epitope peptide resulted in superior generation of effector-memory T cells in the circulation and tissue-resident memory T (TRM) cells, and these tertiary vaccine-specific CD8+ T cells were characterized by enhanced polyfunctional cytokine production. Moreover, fate mapping showed that a substantial fraction of the tertiary effector-memory CD8+ T cells developed from remigrated TRM cells. Thus, repeated booster vaccinations quantitatively and qualitatively improve the CD8+ T cell response leading to protection against otherwise lethal SARS-CoV-2 infection.</p>', 'posted': '2021-12-18'}, {'doi': '10.1101/2021.12.15.472619', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Transcriptomic responses of the human kidney to acute injury at single cell resolution', 'authors': \"[{'name': 'Christian Hinze'}, {'name': 'Christine Kocks'}, {'name': 'Janna Leiz'}, {'name': 'Nikos Karaiskos'}, {'name': 'Anastasiya Boltengagen'}, {'name': 'Christopher Mark Skopnik'}, {'name': 'Jan Klocke'}, {'name': 'Jan-Hendrik Hardenberg'}, {'name': 'Helena Stockmann'}, {'name': 'Inka Gotthardt'}, {'name': 'Benedikt Obermayer'}, {'name': 'Laleh Haghverdi'}, {'name': 'Emanuel Wyler'}, {'name': 'Markus Landthaler'}, {'name': 'Sebastian Bachmann'}, {'name': 'Andreas C. Hocke'}, {'name': 'Victor Corman'}, {'name': 'Jonas Busch'}, {'name': 'Wolfgang Schneider'}, {'name': 'Nina Himmerkus'}, {'name': 'Markus Bleich'}, {'name': 'Kai-Uwe Eckardt'}, {'name': 'Philipp Enghard'}, {'name': 'Nikolaus Rajewsky'}, {'name': 'Kai M Schmidt-Ott'}]\", 'abstract': '<p>Background: Acute kidney injury (AKI) occurs frequently in critically ill patients and is associated with adverse outcomes. Cellular mechanisms underlying AKI and kidney cell responses to injury remain incompletely understood. Methods: We performed single-nuclei transcriptomics, bulk transcriptomics, molecular imaging studies, and conventional histology on kidney tissues from 8 individuals with severe AKI (stage 2 or 3 according to Kidney Disease: Improving Global Outcomes (KDIGO) criteria). Specimens were obtained within 1-2 hours after individuals had succumbed to critical illness associated with respiratory infections, with 4 of 8 individuals diagnosed with COVID-19. Control kidney tissues were obtained post-mortem or after nephrectomy from individuals without AKI. Results: High-depth single cell-resolved gene expression data of human kidneys affected by AKI revealed enrichment of novel injury-associated cell states within the major cell types of the tubular epithelium, in particular in proximal tubules, thick ascending limbs and distal convoluted tubules. Four distinct, hierarchically interconnected injured cell states were distinguishable and characterized by transcriptome patterns associated with oxidative stress, hypoxia, interferon response, and epithelial-to-mesenchymal transition, respectively. Transcriptome differences between individuals with AKI were driven primarily by the cell type-specific abundance of these four injury subtypes rather than by private molecular responses. AKI-associated changes in gene expression between individuals with and without COVID-19 were similar. Conclusion: The study provides an extensive resource of the cell type-specific transcriptomic responses associated with critical illness-associated AKI in humans, highlighting recurrent disease-associated signatures and inter-individual heterogeneity. Personalized molecular disease assessment in human AKI may foster the development of tailored therapies.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.472475', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Non-rapid eye movement sleep and wake neurophysiology in schizophrenia', 'authors': \"[{'name': 'Nataliia Kozhemiako'}, {'name': 'Jun Wang'}, {'name': 'Chenguang Jiang'}, {'name': 'Lei A Wang'}, {'name': 'Guan-chen Gai'}, {'name': 'Kai Zou'}, {'name': 'Zhe Wang'}, {'name': 'Xiao-man Yu'}, {'name': 'Lin Zhou'}, {'name': 'Shen Li'}, {'name': 'Zhenglin Guo'}, {'name': 'Robert G Law'}, {'name': 'James Coleman'}, {'name': 'Dimitrios Mylonas'}, {'name': 'Lu Shen'}, {'name': 'Guoqiang Wang'}, {'name': 'Shuping Tan'}, {'name': 'Shengying Qin'}, {'name': 'Hailiang Huang'}, {'name': 'Michael Murphy'}, {'name': 'Robert Stickgold'}, {'name': 'Dara S Manoach'}, {'name': 'Zhenhe Zhou'}, {'name': 'Wei Zhu'}, {'name': 'Mei-Hua Hall'}, {'name': 'Shaun M Purcell'}, {'name': 'Jen Pan'}]\", 'abstract': '<p>Motivated by the potential of objective neurophysiological markers to index thalamocortical function in patients with severe psychiatric illnesses, we comprehensively characterized key NREM sleep parameters across multiple domains, their interdependencies, and their relationship to waking event-related potentials and symptom severity. In 130 schizophrenia (SCZ) patients and controls, we confirmed a marked reduction in sleep spindle density in SCZ and extended these findings to show that only slow spindles predicted symptom severity, and that fast and slow spindle properties were largely uncorrelated. We also describe a novel measure of slow oscillation and spindle interaction that was attenuated in SCZ. The main sleep findings were replicated in a demographically distinct sample, and a joint model, based on multiple NREM components, predicted disease status in the replication cohort. Although also altered in patients, auditory event-related potentials elicited during wake were unrelated to NREM metrics. Consistent with a growing literature implicating thalamocortical dysfunction in SCZ, our characterization identifies independent NREM and wake EEG biomarkers that may index distinct aspects of SCZ pathophysiology and point to multiple neural mechanisms underlying disease heterogeneity. This study lays the groundwork for evaluating these neurophysiological markers, individually or in combination, to guide efforts at treatment and prevention as well as identifying individuals most likely to benefit from specific interventions.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.10.21267582', 'tweets': 23, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Echinacea purpurea for the Long-term Prevention of Viral Respiratory Tract Infections during COVID-19 Pandemic: A Randomized, Open, Controlled, Exploratory Clinical Study', 'authors': \"[{'name': 'Emil Kolev'}, {'name': 'Lilyana Mircheva'}, {'name': 'Michael R. Edwards'}, {'name': 'Sebastian L. Johnston'}, {'name': 'Krassimir Kalinov'}, {'name': 'Rainer Stange'}, {'name': 'Giuseppe Gancitano'}, {'name': 'Wim Vanden Berghe'}, {'name': 'Samo Kreft'}]\", 'abstract': '<sec><title>Introduction</title><p>SARS-CoV-2 vaccination is effective in preventing severe COVID-19, but efficacy in reducing viral load and transmission wanes over time. In addition, the emergence of novel SARS-CoV-2 variants increases the threat of uncontrolled dissemination and additional antiviral therapies are urgently needed for effective containment. In previous <italic>in vitro</italic> studies <italic>Echinacea purpurea</italic> demonstrated strong antiviral activity against enveloped viruses, including SARS-CoV-2. In this study, we examined the potential of <italic>Echinacea purpurea</italic> in preventing and treating respiratory tract infections (RTIs) and in particular, SARS-CoV-2 infections.</p></sec><sec><title>Methods</title><p>120 healthy volunteers (m,f, 18  75 years) were randomly assigned to <italic>Echinacea</italic> prevention or control group without any intervention. After a run-in week, participants went through 3 prevention cycles of 2, 2 and 1 months with daily 2400mg <italic>Echinacea purpurea</italic> extract (Echinaforce, EF). The prevention cycles were interrupted by breaks of 1 week. Acute respiratory symptoms were treated with 4000 mg EF for up to 10 days, and their severity assessed via a diary. Naso/oropharyngeal swabs and venous blood samples were routinely collected every month and during acute illnesses for detection and identification of respiratory viruses, including SARS-CoV-2 via RT-qPCR and serology.</p></sec><sec><title>Results</title><p>Summarized over all phases of prevention, 21 and 29 samples tested positive for any virus in the EF and control group, of which 5 and 14 samples tested SARS-CoV-2 positive (RR=0.37, Chi-square test, p=0.03). Overall, 10 and 14 symptomatic episodes occurred, of which 5 and 8 were COVID-19 (RR=0.70, Chi-square test, p&gt;0.05). EF treatment when applied during acute episodes significantly reduced the overall virus load by at least 2.12 log<sub>10</sub> or approx. 99% (t-test, p&lt;0.05), the time to virus clearance by 8.0 days for all viruses (Wilcoxon test, p=0.02) and by 4.8 days for SARS-CoV-2 (p&gt;0.05) in comparison to control. Finally, EF treatment significantly reduced fever days (1 day vs 11 days, Chi-square test, p=0.003) but not the overall symptom severity. There were fewer COVID-19 related hospitalizations in the EF treatment group (N=0 vs N=2).</p></sec><sec><title>Discussion/Conclusion</title><p>EF exhibited antiviral effects and reduced the risk of viral RTIs, including SARS-CoV-2. By substantially reducing virus loads in infected subjects, EF offers a supportive addition to existing mandated treatments like vaccinations. Future confirmatory studies are warranted.</p></sec><sec><title>Clinical Trials registration Nr</title><p>NCT05002179</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.472698', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evolution of stickleback spines through independent cis-regulatory changes at HOXDB', 'authors': \"[{'name': 'Julia I. Wucherpfennig'}, {'name': 'Timothy R. Howes'}, {'name': 'Jessica N. Au'}, {'name': 'Eric H. Au'}, {'name': 'Garrett A. Roberts Kingman'}, {'name': 'Shannon D. Brady'}, {'name': 'Amy L. Herbert'}, {'name': 'Thomas E. Reimchen'}, {'name': 'Michael A. Bell'}, {'name': 'Craig B. Lowe'}, {'name': 'Anne C. Dalziel'}, {'name': 'David M. Kingsley'}]\", 'abstract': '<p>Understanding the genetic mechanisms leading to new traits is a fundamental goal of evolutionary biology. We show that <italic>HOXDB</italic> regulatory changes have been used repeatedly in different stickleback fish species to alter the length and number of bony dorsal spines. In <italic>Gasterosteus aculeatus</italic>, a variant <italic>HOXDB</italic> allele is genetically linked to shortening an existing spine and adding a spine. In <italic>Apeltes quadracus</italic>, a variant allele is associated with lengthening an existing spine and adding a spine. The alleles alter the same conserved non-coding <italic>HOXDB</italic> enhancer by diverse molecular mechanisms, including SNPs, deletions, and transposable element insertions. The independent <italic>cis</italic>-acting regulatory changes are linked to anterior expansion or contraction of <italic>HOXDB</italic> expression. Our findings support the long-standing hypothesis that natural <italic>Hox</italic> gene variation underlies key morphological patterning changes in wild populations and illustrate how different mutational mechanisms affecting the same region may produce opposite gene expression changes with similar phenotypic outcomes.</p><sec><title>Abstract Figure</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"472698v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472416', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Ecology drives the degree of convergence in the gene expression of extremophile fishes', 'authors': \"[{'name': 'Michael Tobler'}, {'name': 'Ryan Greenway'}, {'name': 'Joanna L. Kelley'}]\", 'abstract': '<p>Convergent evolution, where independent lineages evolve similar traits when adapting to similar habitats, is a common phenomenon and testament to the repeatability of evolutionary processes. Still, non-convergence is also common, and a major question is whether apparently idiosyncratic, lineage-specific evolutionary changes are reflective of chance events inherent to evolutionary processes, or whether they are also influenced by deterministic genetic or ecological factors. To address this question, we quantified the degree of convergence in genome-wide patterns of gene expression across lineages of livebearing fishes (family Poeciliidae) that span 40 million years of evolution and have colonized extreme environments in the form of toxic, hydrogen-sulfide-rich springs. We specifically asked whether the degree of convergence across lineage pairs was related to their phylogenetic relatedness or the ecological similarity of the habitats they inhabit. Using phylogenetic comparative analyses, we showed that the degree of convergence was highly variable across lineage pairs residing in sulfide springs. While closely related lineages did not exhibit higher degrees of convergence than distantly related ones, we uncovered a strong relationship between degree of convergence and ecological similarity. Our results indicate that variation in the degree of convergence is not merely noise associated with evolutionary contingency. Rather, cryptic environmental variation that is frequently ignored when we employ reductionist approaches can significantly contribute to adaptive evolution. This study highlights the importance of multivariate approaches that capture the complexities of both selective regimes and organismal design when assessing the roles of determinism and contingency in evolution.</p><sec><title>Significance Statement</title><p>When different species adapt to similar environmental conditions, we frequently observe a mix between shared (convergent) and lineage-specific (nonconvergent) evolutionary changes. Shared changes provide evidence for the repeatability and predictability of evolution. However, it remains unclear whether lineage-specific changes are caused by random forces that limit the predictability of evolution, or whether they reflect deterministic processes shaped by unidentified genetic and environmental factors. By analyzing patterns of gene expression across fishes in extreme environments, we show that the degree of convergence between lineages is related to ecology, indicating that lineage-specific evolutionary changes are not just noise caused by random processes. Thus, acknowledging the complexity of nature in empirical research is critical if we want to predict evolution.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.17.473116', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Intra-colony channel morphology in Escherichia coli biofilms is governed by nutrient availability and substrate stiffness', 'authors': \"[{'name': 'Beatrice Bottura'}, {'name': 'Liam M Rooney'}, {'name': 'Paul A Hoskisson'}, {'name': 'Gail McConnell'}]\", 'abstract': '<p>Nutrient-transporting channels are found throughout mature Escherichia coli biofilms, however the influence of environmental conditions on intra-colony channel formation is poorly understood. We report the effect of different substrate nutrient concentrations and agar stiffness on the structure and distribution of intra-colony channels in mature E. coli colony biofilms using fluorescence mesoscopy and quantitative image analysis. Intra-colony channel width was observed to increase non-linearly with radial distance from the centre of the biofilm and channels were, on average, 50% wider at the centre of carbon-limited biofilms compared to nitrogen-limited biofilms. Channel density also differed in colonies grown on rich and minimal medium substrates, with the former creating a network of tightly packed channels and the latter leading to well-separated, wider channels with easily identifiable edges. We conclude that intra-colony channel morphology in E. coli biofilms is influenced by both substrate composition and nutrient availability.</p>', 'posted': '2021-12-18'}, {'doi': '10.1101/2021.12.14.472674', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Qualifying a human Liver-Chip for predictive toxicology: Performance assessment and economic implications', 'authors': \"[{'name': 'Lorna Ewart'}, {'name': 'Athanasia Apostolou'}, {'name': 'Skyler A. Briggs'}, {'name': 'Christopher V. Carman'}, {'name': 'Jake T. Chaff'}, {'name': 'Anthony R. Heng'}, {'name': 'Sushma Jadalannagari'}, {'name': 'Jeshina Janardhanan'}, {'name': 'Kyung-Jin Jang'}, {'name': 'Sannidhi R. Joshipura'}, {'name': 'Mahika Kadam'}, {'name': 'Marianne Kanellias'}, {'name': 'Ville J. Kujala'}, {'name': 'Gauri Kulkarni'}, {'name': 'Christopher Y. Le'}, {'name': 'Carolina Lucchesi'}, {'name': 'Dimitris V. Manatakis'}, {'name': 'Kairav K. Maniar'}, {'name': 'Meaghan E. Quinn'}, {'name': 'Joseph S. Ravan'}, {'name': 'Ann Catherine Rizos'}, {'name': 'John F.K. Sauld'}, {'name': 'Josiah Sliz'}, {'name': 'William Tien-Street'}, {'name': 'Dennis Ramos Trinidad'}, {'name': 'James Velez'}, {'name': 'Max Wendell'}, {'name': 'Prathap Kumar Mahalingaiah'}, {'name': 'Donald E. Ingber'}, {'name': 'Daniel Levner'}]\", 'abstract': '<p>Human organ-on-a-chip (Organ-Chip) technology has the potential to disrupt preclinical drug discovery and improve success in drug development pipelines as it can recapitulate organ-level pathophysiology and clinical responses. The Innovation and Quality (IQ) consortium formed by multiple pharmaceutical and biotechnology companies, however, systematic and quantitative evaluation of the predictive value of Organ-Chips has not yet been reported. Here, 780 Liver-Chips were analyzed to determine their ability to predict drug-induced liver injury (DILI) caused by small molecules identified as benchmarks by the IQ consortium. The Liver-Chip met the qualification guidelines across a blinded set of 27 known hepatotoxic and non-toxic drugs with a sensitivity of 80% and a specificity of 100%. With this performance, a computational economic value analysis suggests that the Liver-Chip could generate $3 billion annually for the pharmaceutical industry due to increased R&amp;D productivity.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.12.472255', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Broad antiviral effects of Echinacea purpurea against SARS-CoV-2 variants of concern and potential mechanism of action', 'authors': \"[{'name': 'Selvarani Vimalanathan'}, {'name': 'Mahmoud Shehata'}, {'name': 'Kannan Sadasivam'}, {'name': 'Serena Delbue'}, {'name': 'Maria Dolci'}, {'name': 'Elena Pariani'}, {'name': 'Sarah DAlessandro'}, {'name': 'Stephan Pleschka'}]\", 'abstract': '<sec><title>Background</title><p>SARS-CoV-2 variants of concern (VOC) represent an alarming threat as they show altered biological behavior and may escape vaccination effectiveness. Some exhibit increased pathogenicity and transmissibility compared to the original wild type WUHAN (Hu-1). Broad-spectrum antivirals could complement and further enhance preventive benefits achieved through SARS-CoV-2 vaccination campaigns</p></sec><sec><title>Methods</title><p>The anti-coronavirus activity of <italic>Echinacea purpurea</italic> (Echinaforce extract, EF) against (i) VOCs B1.1.7 (alpha), B.1.351.1 (beta), P.1 (gamma), B1.617.2 (delta), AV.1 (Scottish) and B1.525 (eta), (ii) SARS-CoV-2 spike (S) protein-pseudotyped viral particles and reference strain OC43 as well as (iii) wild-type SARS-CoV-2 (Hu-1) were analyzed. Molecular dynamics (MD) were applied to study interaction of Echinaceas phytochemical markers with known pharmacological viral and host cell targets.</p></sec><sec><title>Results</title><p>EF extract broadly inhibited propagation of all investigated SARS-CoV-2 VOCs as well as entry of SARS-CoV-2 pseudoparticles at EC50s ranging from 3.62 to 12.03 g/ml. Preventive addition of 20 g/ml EF to epithelial cells significantly reduced sequential infection with SARS-CoV-2 (Hu-1) as well as with the common human strain OC43. MD analyses showed constant binding affinities to Hu-1, B1.1.7, B.1.351, P.1 and B1.617.2-typic S protein variants for alkylamides, caftaric acidand feruoyl-tartaric acid in EF extract. They further indicated that the EF extract could possibly interact with TMPRSS-2, a serine protease required for virus endocytosis.</p></sec><sec><title>Conclusions</title><p>EF extract demonstrated stable antiviral activity across 6 tested VOCs, which is likely due to the constant affinity of the contained phytochemical marker substances to all spike variants. A possible interaction of EF with TMPRSS-2 partially would explain cell protective benefits of the extract by inhibition of endocytosis. EF may therefore offer a supportive addition to vaccination endeavors in the control of existing and future SARS-CoV-2 virus mutations.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472645', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Retinoic acid signaling mediates peripheral cone photoreceptor survival in a mouse model of retina degeneration', 'authors': \"[{'name': 'Ryoji Amamoto'}, {'name': 'Grace K Wallick'}, {'name': 'Constance Cepko'}]\", 'abstract': '<p>Retinitis Pigmentosa (RP) is a wide array of progressive, debilitating visual disorders caused by mutations in a diverse set of genes. In both human patients and mouse models of RP, rod photoreceptor dysfunction leads to loss of night vision, and is followed by secondary cone photoreceptor dysfunction and degeneration, leading to loss of daylight color vision. A strategy to prevent secondary cone death could provide a generalized RP therapy to preserve daylight color vision regardless of the underlying mutation. In mouse models of RP, cones in the far peripheral retina survive long-term, despite complete rod loss. The mechanism for such peripheral cone survival had not been explored. Here, we found that active retinoic acid (RA) signaling in peripheral Muller glia is both sufficient and necessary for the extended cone survival. RA depletion by conditional knockout of RA synthesis enzymes, or overexpression of an RA degradation enzyme, abrogated peripheral cone survival. Conversely, constitutive activation of RA signaling in the central retina promoted long-term cone survival. These results indicate that RA signaling mediates the prolonged peripheral cone survival in the rd1 mouse model of retinal degeneration, and provide a basis for a generic strategy for cone survival in the many diseases that lead to loss of cone-mediated vision.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472968', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structure of human CST-pol-/primase bound to a telomeric overhang poised for initiation of telomere C-strand synthesis', 'authors': \"[{'name': 'Qixiang He'}, {'name': 'Xiuhua Lin'}, {'name': 'Bianca L Chavez'}, {'name': 'Benjamin L Lusk'}, {'name': 'Ci Ji Lim'}]\", 'abstract': \"<p>Telomere replication and regulation protect mammalian chromosome ends and promote genome stability. An essential step in telomere maintenance is the C-strand fill-in process, which is the <italic>de novo</italic> synthesis of the complementary strand of the telomere overhang. This step is catalyzed by polymerase-alpha/primase complex (pol-/primase) and coordinated by an accessory factor, CTC1-STN1-TEN1 (CST). Using cryogenic-electron microscopy single-particle analysis, we report the structure of the human telomere C-strand fill-in preinitiation complex (PIC) at 3.9  resolution. The structure reveals a CST and a pol-/primase co-bound to a single telomere overhang, poised for <italic>de novo </italic> RNA primer synthesis. Upon PIC assembly, the pol-/primase undergoes large conformation change from its apo-state; CST partitions the DNA and RNA catalytic centers of pol-/primase into two separate domains and positions the 3' end of an extended telomere single-stranded DNA template towards the RNA catalytic center (PRIM1 or p49). The telomeric single-stranded DNA template is further positioned by the POLA1 (or p180) catalytically dead exonuclease domain. Together with CST, the exonuclease domain forms a tight-fit molecular tunnel for template direction. Given the structural homology of CST to Replication Protein A (RPA), our structure provides the structural basis for a new model of how pol-/primase lagging-strand DNA synthesis is coordinated by single-stranded DNA-binding accessory factors.</p>\", 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.12.472321', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Insights into the role of lipoteichoic acids in Bacillus subtilis- a new function for MprF', 'authors': \"[{'name': 'Aurlie Guyet'}, {'name': 'Amirah Alofi'}, {'name': 'Richard A. Daniel'}]\", 'abstract': '<p>In <italic>Bacillus subtilis,</italic> the cell is protected from the environment by a cell envelope, which comprises of layers of peptidoglycan that maintain the cell shape and anionic teichoic acids polymers whose biological function remains unclear. In <italic>B. subtilis</italic>, loss of all Class A Penicillin-Binding Proteins (aPBPs) which function in peptidoglycan synthesis is conditionally lethal. Here we show that this lethality is associated with an alteration of the lipoteichoic acids (LTA) and the accumulation of the major autolysin LytE in the cell wall. We provide the first evidence that the length and abundance of LTA acts to regulate the cellular level of LytE. Importantly, we identify a novel function for the aminoacyl-phosphatidylglycerol synthase MprF which acts to modulate LTA biosynthesis in <italic>B. subtilis</italic> and in the pathogen <italic>Staphylococcus aureus</italic>. This finding has implications for our understanding of antimicrobial peptide resistance (particularly daptomycin) in clinically relevant bacteria and MprF-associated virulence in pathogens, such as methicillin resistant <italic>S. aureus</italic>.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472555', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Innate heuristics and fast learning support escape route selection in mice', 'authors': \"[{'name': 'Federico Claudi'}, {'name': 'Dario Campagner'}, {'name': 'Tiago Branco'}]\", 'abstract': '<p>When faced with imminent danger, animals must rapidly take defensive actions to reach safety. Mice can react to innately threatening stimuli in less than 250 milliseconds [1] and, in simple environments, use spatial memory to quickly escape to shelter [2,3]. Natural habitats, however, often offer multiple routes to safety which animals must rapidly identify and choose from to maximize the chances of survival [4]. This is challenging because while rodents can learn to navigate complex mazes to obtain rewards [5,6], learning the value of different routes through trial-and-error during escape from threat would likely be deadly. Here we have investigated how mice learn to choose between different escape routes to shelter. By using environments with paths to shelter of varying length and geometry we find that mice prefer options that minimize both path distance and path angle relative to the shelter. This choice strategy is already present during the first threat encounter and after only ~10 minutes of exploration in a novel environment, indicating that route selection does not require experience of escaping. Instead, an innate heuristic is used to assign threat survival value to alternative paths after rapidly learning the spatial environment. This route selection process is flexible and allows quick adaptation to arenas with dynamic geometries. Computational modelling of different classes of reinforcement learning agents shows that the observed behavior can be replicated by model-based agents acting in an environment where the shelter location is rewarding during exploration. These results show that mice combine fast spatial learning with innate heuristics to choose escape routes with the highest survival value. They further suggest that integrating priors acquired through evolution with knowledge learned from experience supports adaptation to changing environments while minimizing the need for trial-and-error when the errors are very costly.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.21267810', 'tweets': 17, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': True, 'title': 'Early detection and improved genomic surveillance of SARS-CoV-2 variants from deep sequencing data', 'authors': \"[{'name': 'Daniele Ramazzotti'}, {'name': 'Davide Maspero'}, {'name': 'Fabrizio Angaroni'}, {'name': 'Marco Antoniotti'}, {'name': 'Rocco Piazza'}, {'name': 'Alex Graudenzi'}]\", 'abstract': '<p>In the definition of fruitful strategies to contrast the worldwide diffusion of SARS-CoV-2, maximum efforts must be devoted to the early detection of dangerous variants. An effective help to this end is granted by the analysis of deep sequencing data of viral samples, which are typically discarded after the creation of consensus sequences. Indeed, only with deep sequencing data it is possible to identify intra-host low-frequency mutations, which are a direct footprint of mutational processes that may eventually lead to the origination of functionally advantageous variants. Accordingly, a timely and statistically robust identification of such mutations might inform political decision-making with significant anticipation with respect to standard analyses based on con-sensus sequences.</p><p>To support our claim, we here present the largest study to date of SARS-CoV-2 deep sequencing data, which involves 220,788 high quality samples, collected over 20 months from 137 distinct studies. Importantly, we show that a rele-vant number of spike and nucleocapsid mutations of interest associated to the most circulating variants, including Beta, Delta and Omicron, might have been intercepted several months in advance, possibly leading to different public-health decisions. In addition, we show that a refined genomic surveillance system involving high- and low-frequency mutations might allow one to pin-point possibly dangerous emerging mutation patterns, providing a data-driven automated support to epidemiologists and virologists.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.12.472296', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Personalized models of breast cancer desmoplasia reveal biomechanical determinants of drug penetration', 'authors': \"[{'name': 'Giovanni S. Offeddu'}, {'name': 'Kristina Haase'}, {'name': 'Zhengpeng Wan'}, {'name': 'Luca Possenti'}, {'name': 'Huu Tuan Nguyen'}, {'name': 'Mark R. Gillrie'}, {'name': 'Dean Hickman'}, {'name': 'Charles G. Knutson'}, {'name': 'Roger D. Kamm'}]\", 'abstract': '<p>Breast cancer desmoplasia heterogeneity contributes to high disease mortality due to discrepancies in treatment efficacy between patients. Personalized <italic>in vitro</italic> breast cancer models can be used for high throughput testing and ranking of therapeutic strategies to normalize the aberrant microenvironment in a patient-specific manner. Here, tumoroids assembled from patient-derived cells cultured in microphysiological systems including perfusable microvasculature reproduce key aspects of stromal and vascular dysfunction. Increased hyaluronic acid and collagen deposition, loss of vascular glycocalyx and reduced perfusion, and elevated interstitial fluid pressure in the models result in impaired drug distribution to tumor cells. We demonstrate the application of these personalized models as tools to rank molecular therapies for the normalization of the tumoroid microenvironment and to discover new therapeutic targets such as IL8 and CD44, which may ultimately improve drug efficacy in breast cancer patients.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.16.469125', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'imageseg: an R package for deep learning-based image segmentation', 'authors': \"[{'name': 'Jrgen Niedballa'}, {'name': 'Jan Axtner'}, {'name': 'Timm Fabian Dbert'}, {'name': 'Andrew Tilker'}, {'name': 'An Nguyen'}, {'name': 'Seth Timothy Wong'}, {'name': 'Christian Fiderer'}, {'name': 'Marco Heurich'}, {'name': 'Andreas Wilting'}]\", 'abstract': '<p>Convolutional neural networks (CNNs) and deep learning are powerful and robust tools for ecological applications. CNNs can perform very well in various tasks, especially for visual tasks and image data. Image segmentation (the classification of all pixels in images) is one such task and can for example be used to assess forest vertical and horizontal structure. While such methods have been suggested, widespread adoption in ecological research has been slow, likely due to technical difficulties in implementation of CNNs and lack of toolboxes for ecologists. Here, we present R package imageseg which implements a workflow for general-purpose image segmentation using CNNs and the U-Net architecture in R. The workflow covers data (pre)processing, model training, and predictions. We illustrate the utility of the package with two models for forest structural metrics: tree canopy density and understory vegetation density. We trained the models using large and diverse training data sets from a variety of forest types and biomes, consisting of 3288 canopy images (both canopy cover and hemispherical canopy closure photographs) and 1468 understory vegetation images. Overall classification accuracy of the models was high with a Dice score of 0.91 for the canopy model and 0.89 for the understory vegetation model (assessed with 821 and 367 images, respectively), indicating robustness to variation in input images and good generalization strength across forest types and biomes. The package and its workflow allow simple yet powerful assessments of forest structural metrics using pre-trained models. Furthermore, the package facilitates custom image segmentation with multiple classes and based on color or grayscale images, e.g. in cell biology or for medical images. Our package is free, open source, and available from CRAN. It will enable easier and faster implementation of deep learning-based image segmentation within R for ecological applications and beyond.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.472386', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Acute and chronic physical activity improve spatial pattern separation in humans', 'authors': \"[{'name': 'Daniela Ramirez Butavand'}, {'name': 'Mara F. Rodrguez'}, {'name': 'Mara V. Cifuentes'}, {'name': 'Magdalena Miranda'}, {'name': 'Cristian Garca Bauza'}, {'name': 'Pedro Bekinschtein'}, {'name': 'Fabricio Ballarini'}]\", 'abstract': '<p>Physical activity benefits both fitness and cognition. However, its effect on long-term memory is unclear. Successful memory involves not only remembering information over time but also keeping memories distinct and less confusing. The ability to separate similar experiences into distinct memories is one of the main features of episodic memory. In this work, we evaluated the effect of acute and chronic physical activity on a new task to assess spatial pattern separation in a 3D virtual reality environment. We manipulated the load of memory similarity and found that 25 minutes of cycling after encoding - but not before retrieval - was sufficient to improve similar, but not dissimilar memories, 24 hours after encoding. Furthermore, we found that participants who engaged in regular physical activity, but not sedentary subjects, showed memory for the similar condition the next day. Thus, physical activity could be a simple way to improve discrimination of spatial memories in humans.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.473006', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The probability of joint monophyly of all species in an arbitrary species tree.', 'authors': \"[{'name': 'Rohan S Mehta'}, {'name': 'Mike Steel'}, {'name': 'Noah A Rosenberg'}]\", 'abstract': '<p>Monophyly is a feature of a set of genetic lineages in which every lineage in the set is more closely related to all other members of the set than it is to any lineage outside the set. Multiple sets of lineages that are separately monophyletic are said to be reciprocally monophyletic, or jointly monophyletic. The prevalence of reciprocal monophyly, or joint monophyly, has been used to evaluate phylogenetic and phylogeographic hypotheses, as well as to delimit species. These applications often make use of a probability of joint monophyly under models of gene lineage evolution. Studies in coalescent theory have computed this joint monophyly probability for small numbers of separate groups in arbitrary species trees, and for arbitrary numbers of separate groups in trivial species trees. Here, generalizing existing results on monophyly probabilities under the multispecies coalescent, we derive the probability of joint monophyly for arbitrary numbers of separate groups in arbitrary species trees. We illustrate how our result collapses to previously examined cases. We also study the effect of tree height, sample size, and number of species on the probability of joint monophyly. The result also enables computation of relatively simple lower and upper bounds on the joint monophyly probability. Our results expand the scope of joint monophyly calculations beyond small numbers of species, subsuming past formulas that have been used in simpler cases.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.472146', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'AutoMicroED: A semi-automated MicroED processing pipeline', 'authors': \"[{'name': 'Samantha M Powell'}, {'name': 'Irina V Novikova'}, {'name': 'Doo Nam Kim'}, {'name': 'James E Evans'}]\", 'abstract': '<p>Despite rapid adaptation of micro-electron diffraction (MicroED) for protein and small molecule structure determination to sub-angstrom resolution, the lack of automation tools for easy MicroED data processing remains a challenge for expanding to the broader scientific community. In particular, automation tools, which are novice user friendly, compatible with heterogenous datasets and can be run in unison with data collection to judge the quality of incoming data (similar to cryosparc LIVE for single particle cryoEM) do not exist. Here, we present AutoMicroED, a cohesive and semi-automatic MicroED data processing pipeline that runs through image conversion, indexing, integration and scaling of data, followed by merging of successful datasets that are pushed through phasing and final structure determination. AutoMicroED is compatible with both small molecule and protein datasets and creates a straightforward and reproducible method to solve single structures from pure samples, or multiple structures from mixed populations. The immediate feedback on data quality, data completeness and more parameters, aids users to identify whether they have collected enough data for their needs. Overall, AutoMicroED permits efficient structure elucidation for both novice and experienced users with comparable results to more laborious manual processing.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472370', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'A convergent malignant phenotype in B-cell acute lymphoblastic leukemia involving the splicing factor SRRM1', 'authors': \"[{'name': 'Adria Closa'}, {'name': 'Marina Reixachs-Sol'}, {'name': 'Antonio C. Fuentes-Fayos'}, {'name': 'Katharina E. Hayer'}, {'name': 'Juan Luis Melero'}, {'name': 'Fabienne R. S. Adriaanse'}, {'name': 'Romy S. Bos'}, {'name': 'Manuel Torres-Diz'}, {'name': 'Stephen Hunger'}, {'name': 'Kathryn G. Roberts'}, {'name': 'Charles Mullighan'}, {'name': 'Ronald W. Stam'}, {'name': 'Andrei Thomas-Tikhonenko'}, {'name': 'Justo P. Castao'}, {'name': 'Ral M. Luque'}, {'name': 'Eduardo Eyras'}]\", 'abstract': '<p>A significant proportion of B-cell acute lymphoblastic leukemia (B-ALL) patients remains with a dismal prognosis due to yet undetermined mechanisms. We performed a comprehensive multicohort analysis of gene fusions, gene expression, and RNA splicing alterations to uncover molecular signatures potentially linked to the observed poor outcome. We identified 84 fusions with significant allele frequency across patients. We identified an expression signature that predicts high risk independently of the gene fusion background. This signature includes the upregulation of the splicing factor <italic>SRRM1</italic>, which potentially impacts splicing events associated with poor outcomes through protein-protein interactions with other splicing factors. Experiments in B-ALL cell lines provided further evidence for the role of SRRM1 on cell survival, proliferation, and invasion. Our findings reveal a convergent mechanism of aberrant RNA processing that sustains a malignant phenotype independently of gene fusions and could complement current clinical strategies in B-ALL.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472622', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': True, 'title': 'Computational analysis of the effect of SARS-CoV-2 variant Omicron Spike protein mutations on dynamics, ACE2 binding and propensity for immune escape', 'authors': \"[{'name': 'Natalia Teruel'}, {'name': 'Matthew Crown'}, {'name': 'Matthew Bashton'}, {'name': 'Rafael Najmanovich'}]\", 'abstract': '<p>The recently reported Omicron (B.1.1.529) SARS-CoV-2 variant has a large number of mutations in the Spike (S) protein compared to previous variants. Here we evaluate the potential effect of Omicron S mutations on S protein dynamics and ACE2 binding as contributing factors to infectivity as well as propensity for immune escape. We define a consensus set of mutations from 77 sequences assigned as Omicron in GISAID as of November 25. We create structural models of the Omicron S protein in the open and closed states, as part of a complex with ACE2 and for each of 77 complexes of S bound to different antibodies with known structures. We have previously utilized Dynamical Signatures (DS) and the Vibrational Entropy Score (VDS) to evaluate the propensity of S variants to favour the open state. Here, we introduce the Binding Influence Score (BIS) to evaluate the influence of mutations on binding affinity based on the net gain or loss of interactions within the protein-protein interface. BIS shows excellent correlation with experimental data (Pearson correlation coefficient of 0.87) on individual mutations in the ACE2 interface for the Alpha, Beta, Gamma, Delta and Omicron variants combined. On the one hand, the DS of Omicron highly favours a more rigid open state and a more flexible closed state with the largest VDS of all variants to date, suggesting a large increase in the chances to interact with ACE2. On the other hand, the BIS shows that apart from N501Y, all other mutations in the interface reduce ACE2 binding affinity. VDS and BIS show opposing effects on the overall effectiveness of Omicron mutations to promote binding to ACE2 and therefore initiate infection. To evaluate the propensity for immune escape we calculated the net change of favourable and unfavourable interactions within each S-antibody interface. The net change of interactions shows a positive score (a net increase of favourable interactions and decrease of unfavourable ones) for 41 out of 77 antibodies, a nil score for 15 and a negative score for 21 antibodies. Therefore, in only 28% of S-antibody complexes (21/77) we predict some level of immune escape due to a weakening of the interactions with Omicron S. Considering that most antibody epitopes and the mutations are within the S-ACE2 interface our results suggest that mutations within the RBD of Omicron may give rise to only partial immune escape, which comes at the expense of reduced ACE2 binding affinity. However, this reduced ACE2 affinity appears to have been offset by increasing the occupancy of the open state of the Spike protein.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472367', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A neuron, microglia, and astrocyte triple coculture model to study Alzheimer disease', 'authors': \"[{'name': 'Celia Luchena'}, {'name': 'Jone Zuazo-Ibarra'}, {'name': 'Jorge Valero'}, {'name': 'Carlos Matute'}, {'name': 'Elena Alberdi'}, {'name': 'Estibaliz Capetillo-Zarate'}]\", 'abstract': '<p>Glial cells are essential to understand Alzheimers disease (AD) progression, given their role in neuroinflammation and neurodegeneration. There is a need for reliable and easy to manipulate models that allow studying the mechanisms behind neuron and glia communication. Currently available models such as cocultures require complex methodologies and/or might not be affordable for all laboratories. With this in mind, we aimed to establish a straightforward <italic>in vitro</italic> setting with neurons and glial cells to study AD.</p><p>We generated a triple co-culture with neurons, microglia and astrocytes. Immunofluorescence, western blot and ELISA techniques were used to characterize the effects of oligomeric A (oA) in this model.</p><p>We found that, in the triple co-culture, microglia increased the expression of anti-inflammatory markers Arginase I and TGF-1, and reduced pro-inflammatory iNOS and IL-1, compared with microglia alone. Astrocytes reduced expression of pro-inflammatory A1 markers AMIGO2 and C3, and displayed a ramified morphology resembling physiological conditions. Lastly, neurons increased post-synaptic markers, and developed more and longer branches than in individual primary cultures. Addition of oA in the triple coculture reduced synaptic markers and increased microglial activation, which are hallmarks of AD.</p><p>Consequently, we developed a reliable model, where cells better resemble physiological conditions: microglia are less inflammatory, astrocytes are less reactive and neurons display a more mature morphology than in individual primary cultures. Moreover, we were able to recapitulate A-induced synaptic loss and inflammation. This model emerges as a powerful tool to study neurodegeneration and inflammation in the context of AD and other neurodegenerative diseases.</p><sec><title>Table of content image</title><fig id=\"ufig1\" position=\"anchor\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"472367v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec><sec><title>Main points</title><list list-type=\"bullet\"><list-item><p>In our model, microglia and astrocytes are less reactive, and neurons have a more mature morphology than in primary cultures.</p></list-item><list-item><p>oA reduced synaptic markers and increased microglial activation.</p></list-item><list-item><p>This triple co-culture is a reliable tool to study neurodegeneration and gliosis <italic>in vitro</italic>.</p></list-item></list></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.15.472756', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Short senolytic or senostatic interventions rescue progression of radiation-induced frailty and premature ageing in mice', 'authors': \"[{'name': 'Edward Fielder'}, {'name': 'Tengfei Wan'}, {'name': 'Ghazaleh Alimohammadiha'}, {'name': 'Abbas Ishaq'}, {'name': 'Evon Low'}, {'name': 'Melanie B Weigand'}, {'name': 'George Kelly'}, {'name': 'Craig Parker'}, {'name': 'Brigid Griffin'}, {'name': 'Diana Jurk'}, {'name': 'Viktor Korolchuk'}, {'name': 'Thomas von Zglinicki'}, {'name': 'Satomi Miwa'}]\", 'abstract': '<p>Cancer survivors suffer from progressive frailty, multimorbidity and premature morbidity. We hypothesize that therapy-induced senescence and senescence progression via bystander effects is a significant cause of this premature ageing phenotype. Accordingly, the study addresses the question whether a short anti-senescence intervention is able to block progression of radiation-induced frailty and disability in a pre-clinical setting. Male mice were sub-lethally irradiated at 5 months of age and treated (or not) with either a senolytic drug (Navitoclax or dasatinib + quercetin) for 10 days or with the senostatic metformin for 10 weeks. Follow up was for one year. Treatments commencing within a month after irradiation effectively reduced frailty progression (p&lt;0.05) and improved muscle (p&lt;0.01) and liver (p&lt;0.05) function as well as short-term memory (p&lt;0.05) until advanced age with no need for repeated interventions. Senolytic interventions that started late, after radiation-induced premature frailty was manifest, still had beneficial effects on frailty (p&lt;0.05) and short-term memory (p&lt;0.05). Metformin was similarly effective as senolytics. At therapeutically achievable concentrations metformin acted as a senostatic neither via inhibition of mitochondrial complex I, nor via improvement of mitophagy or mitochondrial function, but by reducing non-mitochondrial ROS production via NOX4 inhibition in senescent cells. Our study suggests that the progression of adverse long-term health and quality-of-life effects of radiation exposure, as experienced by cancer survivors, might be rescued by short-term adjuvant antisenescence interventions.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472942', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Controlling homology-directed repair outcomes in human stem cells with dCas9', 'authors': \"[{'name': 'William C Skarnes'}, {'name': 'Gang Ning'}, {'name': 'Sofia Giansiracusa'}, {'name': 'Alexander S Cruz'}, {'name': 'Cornelis Blauwendraat'}, {'name': 'Brandon Saavedra'}, {'name': 'Kevin Holden'}, {'name': 'Mark R Cookson'}, {'name': 'Michael E Ward'}, {'name': 'Justin A McDonough'}]\", 'abstract': '<p>Modeling human disease in human stem cells requires precise, scarless editing of single nucleotide variants (SNV) on one or both chromosomes.  Here we describe improved conditions for Cas9 RNP editing of SNVs that yield high rates of biallelic homology-directed repair.  To recover both heterozygous and homozygous SNV clones, catalytically inactive dCas9 was added to moderate high activity Cas9 RNPs. dCas9 can also block re-cutting and damage to SNV alleles engineered with non-overlapping guide RNAs.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.471754', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Namco: A microbiome explorer', 'authors': \"[{'name': 'Alexander Dietrich'}, {'name': 'Monica Steffi Matchado'}, {'name': 'Maximilian Zwiebel'}, {'name': 'Benjamin lke'}, {'name': 'Michael Lauber'}, {'name': 'Ilias Lagkouvardos'}, {'name': 'Jan Baumbach'}, {'name': 'Dirk Haller'}, {'name': 'Beate Brandl'}, {'name': 'Thomas Skurk'}, {'name': 'Hans Hauner'}, {'name': 'Sandra Reitmeier'}, {'name': 'Markus List'}]\", 'abstract': \"<p>Background: 16S rRNA gene profiling is currently the most widely used technique in microbiome research and allows for studying microbial diversity, taxonomic profiling, phylogenetics, functional and network analysis. While a plethora of tools have been developed for the analysis of 16S rRNA gene data, only few platforms offer a user-friendly interface and none comprehensively cover the whole analysis pipeline from raw data processing down to complex analysis. Results: We introduce Namco, an R shiny application that offers a streamlined interface and serves as a one-stop solution for microbiome analysis. We demonstrate Namco's capabilities by studying the association between rich fibre diet and the gut microbiota composition. Namco helped to prove the hypothesis that butyrate-producing bacteria are prompted by fibre-enriched intervention. Conclusion: Namco provides a broad range of features from raw data processing and basic statistics down to machine learning and network analysis, thus covering complex data analysis tasks that are not comprehensively covered elsewhere. Namco is freely available at https://exbio.wzw.tum.de/namco/.</p>\", 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472606', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A world of viruses nested within parasites: Unraveling viral diversity within parasitic flatworms (Platyhelminthes)', 'authors': \"[{'name': 'Nolwenn M Dheilly'}, {'name': 'Pierrick Lucas'}, {'name': 'Yannick Blanchard'}, {'name': 'Karyna Rosario'}]\", 'abstract': '<p>Because parasites have an inextricable relationship with their host, they have the potential to serve as viral reservoirs or facilitate virus host-shifts. Yet, little is known about viruses infecting parasitic hosts except for blood-feeding arthropods that are well-known vectors of zoonotic viruses. Herein we uncover viruses of flatworms (Phylum Platyhelminthes, group Neodermata) that specialize in parasitizing vertebrates and their ancestral free-living relatives. We discovered 115 novel viral sequences, including 1 in Macrostomorpha, 5 in Polycladida, 44 in Tricladida, 1 in Monogenea, 15 in Cestoda and 49 in Trematoda, through data mining. The majority of newly identified viruses constitute novel families or genera. Phylogenetic analyses show that the virome of flatworms changed dramatically during the transition of Neodermatans to a parasitic lifestyle. Most Neodermatan viruses seem to co-diversify with their host, with the exception of rhabdoviruses which may switch host more often, based on phylogenetic relationships. Neodermatan rhabodviruses also have an ancestral position to vertebrate-associated viruses, including Lyssaviruses, suggesting that vertebrate rhabdoviruses emerged from a flatworm rhabdovirus in a parasitized host. This study reveals an extensive diversity of viruses in Platyhelminthes and highlights the need to evaluate the role of viral infection in flatworm-associated diseases.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472530', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The ColR/S two-component system is a conserved determinant of host association across Pseudomonas species', 'authors': \"[{'name': 'Christina L. Wiesmann'}, {'name': 'Yue Zhang'}, {'name': 'Morgan Alford'}, {'name': 'David Thoms'}, {'name': 'Melanie Dostert'}, {'name': 'Andrew Wilson'}, {'name': 'Daniel Pletzer'}, {'name': 'Robert E. W. Hancock'}, {'name': 'Cara H. Haney'}]\", 'abstract': '<p>Members of the bacterial genus <italic>Pseudomonas</italic> form mutualistic, commensal and pathogenic associations with diverse hosts. The prevalence of host association across the genus suggests that symbiosis may be a conserved ancestral trait and that distinct symbiotic lifestyles may be more recently evolved. Here we show that the ColR/S two-component system, part of the <italic>Pseudomonas</italic> core genome, is functionally conserved between <italic>Pseudomonas aeruginosa</italic> and <italic>Pseudomonas fluorescens</italic>. Using plant rhizosphere colonization and virulence in a murine abscess model, we show that <italic>colR</italic> is required for commensalism with plants and virulence in animals. Comparative transcriptomics revealed that the ColR regulon has diverged between <italic>P. aeruginosa</italic> and <italic>P. fluorescens</italic> and deleting components of the ColR regulon revealed strain-specific, but not hostspecific, requirements for ColR-dependent genes. Collectively, our results suggest that ColR/S allows <italic>Pseudomonas</italic> to sense and respond to a host, but that the ColR-regulon has diverged between <italic>Pseudomonas</italic> strains with distinct lifestyles.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472383', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Differentiation signals from glia are fine-tuned to set neuronal numbers during development', 'authors': \"[{'name': 'Anadika R. Prasad'}, {'name': 'Matthew P. Bostock'}, {'name': 'Ins Lago-Baldaia'}, {'name': 'Zaynab Housseini'}, {'name': 'Vilaiwan M. Fernandes'}]\", 'abstract': '<p>Precise neuronal numbers are required for circuit formation and function. Known strategies to control neuronal numbers involve regulating either cell proliferation or survival. In the developing <italic>Drosophila</italic> visual system photoreceptors from the eye-disc induce their target field, the lamina, one column at a time. Although each column initially contains 6 precursors, only 5 differentiate into neurons of unique identities (L1-L5); the extra precursor undergoes apoptosis. We uncovered that Hedgehog signalling patterns columns, such that the 2 precursors experiencing the lowest signalling activity are specified as L5s; only one differentiates while the other extra precursor dies. We showed that a glial population called the outer chiasm giant glia (xg<sup>O</sup>), which reside below the lamina, relays differentiation signals from photoreceptors to induce L5 differentiation. The precursors nearest to xg<sup>O</sup> differentiate into L5s and antagonise inductive signalling to prevent the extra precursors from differentiating, resulting in their death. Thus, tissue architecture and feedback from young neurons fine-tune differentiation signals from glia to limit the number of neurons induced.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.469105', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Intrinsic instability of the dysbiotic microbiome revealed through dynamical systems inference at scale', 'authors': \"[{'name': 'Travis E Gibson'}, {'name': 'Younhun Kim'}, {'name': 'Sawal Acharya'}, {'name': 'David E Kaplan'}, {'name': 'Nicholas DiBenedetto'}, {'name': 'Richard Lavin'}, {'name': 'Bonnie Berger'}, {'name': 'Jessica R Allegretti'}, {'name': 'Lynn Bry'}, {'name': 'Georg K Gerber'}]\", 'abstract': '<p>Despite the importance of microbial dysbiosis in human disease, the phenomenon remains poorly understood. We provide the first comprehensive and predictive model of dysbiosis at ecosystem-scale, leveraging our new machine learning method for efficiently inferring compact and interpretable dynamical systems models. Coupling this approach with the most densely temporally sampled interventional study of the microbiome to date, using microbiota from healthy and dysbiotic human donors that we transplanted into mice subjected to antibiotic and dietary interventions, we demonstrate superior predictive performance of our method over state-of-the-art techniques. Moreover, we demonstrate that our approach uncovers intrinsic dynamical properties of dysbiosis driven by destabilizing competitive cycles, in contrast to stabilizing interaction chains in the healthy microbiome, which have implications for restoration of the microbiome to treat disease.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.12.472285', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Adaptive discrimination of antigen risk by predictive coding in immune system', 'authors': \"[{'name': 'Kana Yoshido'}, {'name': 'Honda Naoki'}]\", 'abstract': '<p>The immune system discriminates between harmful and harmless antigens based on past experiences; however, the underlying mechanism is largely unknown. From the viewpoint of machine learning, the learning system predicts the observation and updates the prediction based on prediction error, a process known as predictive coding. Here, we modeled the population dynamics of T cells by adopting the concept of predictive coding; helper and regulatory T cells predict the antigen amount and excessive immune response, respectively. Their prediction error signals, possibly via cytokines, induce their differentiation to memory T cells. Through numerical simulations, we found that the immune system identifies antigen risks depending on the concentration and input rapidness of the antigen. Further, our model reproduced history-dependent discrimination, as in allergy onset and subsequent therapy. Together, this study provided a novel framework to improve our understanding of how the immune system adaptively learns the risks of diverse antigens.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.15.472830', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'An unexpected role for the conserved ADAM-family metalloprotease ADM-2 in Caenorhabditis elegans molting', 'authors': \"[{'name': 'Braveen B Joseph'}, {'name': 'Sarina Meadows'}, {'name': 'Phillip T Edeen'}, {'name': 'Shaonil Binti'}, {'name': 'David S Fay'}]\", 'abstract': '<p>Molting is a widespread developmental process in which the external extracellular matrix (ECM), the cuticle, is remodeled to allow for organismal growth and environmental adaptation. Studies in the nematode Caenorhabditis elegans have identified a diverse set of molting-associated factors including signaling molecules, intracellular trafficking regulators, ECM components, and ECM-modifying enzymes such as matrix metalloproteases. C. elegans NEKL-2 and NEKL-3, two conserved members of the NEK family of protein kinases, are essential for molting and promote the endocytosis of environmental steroid-hormone precursors by the epidermis. Steroids in turn drive the cyclic induction of many genes required for molting. Here we report a novel role for the sole C. elegans ADAM-meltrin metalloprotease family member, ADM-2, as a negative regulator of molting. Whereas loss of adm-2 led to strong suppression of molting defects in partial loss-of-function nekl mutants, overexpression of ADM-2 induced molting defects in wild-type animals. CRISPR genome editing implicated the Zn-binding motif within the metalloprotease domain as critical for mediating the effects of ADM-2 on molting. ADM-2 is expressed in the epidermis, and its trafficking through the endo-lysosomal network was disrupted after NEKL depletion. We also identified the epidermally expressed low-density lipoprotein receptor-related protein, LRP-1, as a candidate target of ADM-2 regulation. Whereas loss of ADM-2 activity led to the upregulation of LRP-1, ADM-2 overexpression caused a reduction in LRP-1 abundance, suggesting that ADM-2 may function as a sheddase for LRP-1. We propose that loss of adm-2 suppresses molting defects in nekl mutants by eliminating a negative regulator of LRP-1, thereby compensating for defects in the efficiency of LRP-1 and sterol uptake. Our findings emphasize the importance of endocytic trafficking for both the internalization of factors that promote molting and the removal of proteins that would otherwise be deleterious to the molting process.\\n\\u2003</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472773', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Regulation of effector gene expression as concerted waves in Leptosphaeria maculans: a two-players game', 'authors': \"[{'name': 'Colin Clairet'}, {'name': 'Elise J Gay'}, {'name': 'Antoine Porquier'}, {'name': 'Franoise Blaise'}, {'name': 'Claire-Line Marais'}, {'name': 'Marie-Hlne Balesdent'}, {'name': 'Thierry Rouxel'}, {'name': 'Jessica L. Soyer'}, {'name': 'Isabelle Fudal'}]\", 'abstract': '<p>During infection, plant pathogenic fungi secrete a set of molecules collectively known as effectors, involved in overcoming the host immune system and in disease establishment. Effector genes are concertedly expressed as waves all along plant pathogenic fungi lifecycle. However, little is known about how coordinated expression of effector genes is regulated. Since many effector genes are located in repeat-rich regions, the role of chromatin remodeling in the regulation of effector expression was recently investigated. In <italic>Leptosphaeria maculans</italic>, causing stem canker of oilseed rape, we established that the repressive histone modification H3K9me3 (trimethylation of Lysine 9 of Histone H3), deposited by the histone methyltransferase KMT1, was involved in the regulation of expression of genes highly expressed during infection, including effectors. Nevertheless, inactivation of <italic>KMT1</italic> did not induce expression of these genes at the same level as observed during infection of oilseed rape, suggesting that a second regulator, such as a transcription factor (TF), might be involved. Pf2, a TF belonging to the Zn2Cys6 fungal specific TF family, was described in several Dothideomycete species as essential for pathogenicity and effector gene expression. We identified the orthologue of Pf2 in <italic>L. maculans</italic>, LmPf2, and investigated the role of LmPf2 together with KMT1, by inactivating and over-expressing LmPf2 in a wild type (WT) strain and a <italic>kmt1</italic> mutant. Functional analyses of the corresponding transformants highlighted an essential role of LmPf2 in the establishment of pathogenesis. Transcriptomic analyses during axenic growth showed that LmPf2 is involved in the control of effector gene expression. We observed an enhanced effect of the over-expression of <italic>LmPf2</italic> on effector gene expression in a <italic>kmt1</italic> background, suggesting an antagonist role between KMT1 and LmPf2.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472809', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'A Precision Medicine Approach to Metabolic Therapy for Breast Cancer in Mice', 'authors': \"[{'name': 'Ngozi D Akingbesote'}, {'name': 'Aaron Norman'}, {'name': 'Wanling Zhu'}, {'name': 'Alexandra A Halberstam'}, {'name': 'Xinyi Zhang'}, {'name': 'Julia R Foldi'}, {'name': 'Maryam B Lustberg'}, {'name': 'Rachel J. Perry'}]\", 'abstract': '<p>Increasing evidence highlights the possibility for approaches targeting metabolism as potential adjuvants to cancer therapy. Sodium-glucose transport protein 2 (SGLT2) inhibitors are the newest class of antihyperglycemic therapies and have recently been highlighted as a novel therapeutic approach to breast cancer. To our knowledge, however, SGLT2 inhibitors have not been applied in the neoadjuvant setting as a precision medicine approach to combining metabolic therapy with standard of care therapy for this devastating disease. In this study, we combine the SGLT2 inhibitor dapagliflozin with paclitaxel chemotherapy in both lean and obese mice. We show that dapagliflozin enhances the efficacy of paclitaxel, reducing tumor glucose uptake and prolonging survival in an insulin-dependent manner in some but not all breast tumors. Our data find a genetic signature for breast tumors most likely to respond to dapagliflozin in combination with paclitaxel. Tumors driven by mutations upstream of canonical insulin signaling pathways are likely to respond to such treatment, whereas tumors driven by mutations downstream of canonical insulin signaling are not. These data demonstrate that dapagliflozin enhances the response to chemotherapy in mice with breast cancer and suggest that breast cancer patients with driver mutations upstream of canonical insulin signaling may be most likely to benefit from this neoadjuvant approach. A clinical trial is currently in preparation, with an application recently submitted for Yale Human Investigations Committee approval, to test this hypothesis in breast cancer patients.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.472149', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Oxoglutarate dehydrogenase coordinates myofibril growth by maintaining amino acid homeostasis', 'authors': \"[{'name': 'Nicanor Gonzlez Morales'}, {'name': 'Ocane Marescal'}, {'name': 'Szilrd Szikora'}, {'name': 'Miklos Erdelyi'}, {'name': 'Pter Br'}, {'name': 'Tuana Mesquita'}, {'name': 'Jzsef Mihly'}, {'name': 'Frieder Schck'}]\", 'abstract': '<p>Myofibrils are long intracellular cables specific to muscles, composed mainly of actin and myosin filaments. The actin and myosin filaments are organized into repeated units called sarcomeres, which form the myofibril cables. Muscle contraction is achieved by the simultaneous shortening of sarcomeres and for a highly coordinated contraction to occur all sarcomeres should have the same size. Muscles have evolved a variety of ways to ensure sarcomere homogeneity, one example being the controlled oligomerization of Zasp proteins that sets the diameter of the myofibril. To understand how Zasp proteins effect myofibril growth, we looked for Zasp-binding proteins at the Z-disc. We found that the E1 subunit of the oxoglutarate dehydrogenase complex is recruited to the Z-disc by Zasp52 and is required to sustain myofibril growth. By making specific mutants, we show that its enzymatic activity is important for myofibril growth, and that the other two subunits of the complex are also required for myofibril formation. Using super resolution microscopy, we revealed the overall organization of the complex at the Z-disc. Then, using metabolomic analysis, we uncovered an amino acid balance defect affecting protein synthesis, that we also confirmed by genetic tools. In summary, we show that Zasp controls the local amino acid pool responsible for myofibril growth by recruiting the OGDH complex to the Z-disc.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.12.21266792', 'tweets': 14, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'Rapid fluoroquinolone resistance detection in Pseudomonas aeruginosa using mismatch amplification mutation assay-based real-time PCR', 'authors': \"[{'name': 'Danielle E. Madden'}, {'name': 'Kate L. McCarthy'}, {'name': 'Scott C. Bell'}, {'name': 'Olusola Olagoke'}, {'name': 'Timothy Baird'}, {'name': 'Jane Neill'}, {'name': 'Kay A. Ramsay'}, {'name': 'Timothy J. Kidd'}, {'name': 'Adam G. Stewart'}, {'name': 'Shradha Subedi'}, {'name': 'Keat Choong'}, {'name': 'Tamieka A. Fraser'}, {'name': 'Derek S. Sarovich'}, {'name': 'Erin P. Price'}]\", 'abstract': '<p>Antimicrobial resistance (AMR) is an ever-increasing global health concern. Here, we developed two SYBR Green-based mismatch amplification mutation assays (SYBR-MAMAs) targeting GyrA T83I, D87N, D87Y, and D87H, the predominant causes of fluoroquinolone AMR in <italic>Pseudomonas aeruginosa</italic>. SYBR-MAMAs were tested on 85 isolates, 45 with intermediate/full ciprofloxacin resistance. Assays identified the correct phenotype in 84% intermediate/full resistant and 100% sensitive strains. Clinical implementation of our rapid SYBR-MAMAs will permit timely treatment alterations to improve patient outcomes.</p>', 'posted': '2021-12-19'}, {'doi': '10.1101/2021.12.09.471935', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'No evidence of paralogous loci or new bona fide microRNAs in telomere to telomere (T2T) genomic data', 'authors': \"[{'name': 'Arun H. Patil'}, {'name': 'Marc K. Halushka'}, {'name': 'Bastian Fromm'}]\", 'abstract': '<p>The telomere to telomere (T2T) genome project discovered and mapped 240 million additional base pairs of primarily telomeric and centromeric reads. Much of this sequence was comprised of satellite sequences and large segmental duplications. We evaluated the extent to which human <italic>bona fide</italic> microRNAs (miRNAs) may be found in additional paralogous genomic loci or if previously undescribed microRNAs are present in these newly sequenced regions of the human genome. New genomic regions of the T2T project spanning 240 million bp of sequence were obtained and evaluated by blastn for the human miRNAs contained in MirGeneDB2.0 (N=556) and miRBase (N = 1917) along with all species of MirGeneDB2.0 miRNAs (N=10,899). Additionally, bowtie was used to compare unmapped reads from &gt;4,000 primary cell samples to the new T2T sequence. Based on sequence and structure, no <italic>bona fide</italic> miRNAs were identified. Ninety-seven miRNAs of questionable authenticity (frequently known repeat elements) were identified from the miRBase dataset across the newly described regions of the human genome. These 97 represent only 51 miRNA families due to paralogy of highly similar miRNAs such as 24 members of the hsa-mir-548 family. Altogether, this data strongly supports our having identified widely expressed <italic>bona fide</italic> miRNAs in the human genome and move us further toward the completion of human miRNA discovery.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472380', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'PTEN inhibits AMPK to control collective migration', 'authors': \"[{'name': 'Florent Peglion'}, {'name': 'Lavinia Capuana'}, {'name': 'Isabelle Perfettini'}, {'name': 'Ben Braithwaite'}, {'name': 'Emie Quissac'}, {'name': 'Karin Forsberg-Nilsson'}, {'name': 'Flora Llense'}, {'name': 'Sandrine Etienne-Manneville'}]\", 'abstract': '<p>PTEN is one of the most frequently mutated tumor suppressor gene in cancer. PTEN is generally altered in invasive cancers such as glioblastomas, but its function in collective cell migration and invasion is not fully characterized. Herein, we report that the loss of PTEN increases cell speed during collective migration of non-tumourous cells both in vitro and in vivo. We further show that loss of PTEN promotes LKB1-dependent phosphorylation and activation of the major metabolic regulator AMPK. In turn AMPK increases VASP phosphorylation, reduces VASP localization at cell-cell junctions and decreases the interjunctional transverse actin arcs at the leading front, provoking a weakening of cell-cell contacts and increasing migration speed. Targeting AMPK activity not only slows down PTEN-depleted cells, it also limits PTEN-null glioblastoma cell invasion, opening new opportunities to treat glioblastoma lethal invasiveness.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.15.21267793', 'tweets': 13, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Antibody-mediated Immunogenicity against SARS-CoV-2 following priming, boosting and hybrid immunity: insights from 11 months of follow-up of a healthcare worker cohort in Israel, December 2020-October 2021', 'authors': \"[{'name': 'Michael Edelstein'}, {'name': 'Karine Wiegler Beiruti'}, {'name': 'Hila Ben-Amram'}, {'name': 'Naor Bar-Zeev'}, {'name': 'Christian Sussan'}, {'name': 'Hani Asulin'}, {'name': 'David Strauss'}, {'name': 'Younes Bathish'}, {'name': 'Salman Zarka'}, {'name': 'Kamal Abu Jabal'}]\", 'abstract': '<sec><title>Background</title><p>We determined circulating anti-S SARS-CoV-2 IgG antibody titres in a vaccinated healthcare workers (HCWs) cohort from Northern Israel in the 11 months following primary vaccination according to age, ethnicity, boosting timing and previous infection status.</p></sec><sec><title>Methods</title><p>All consenting HCWs were invited to have their circulating IgG levels measured before vaccination and at 6 subsequent timepoints. All HCWs with suspected COVID-19 were PCR tested. We described trends in circulating IgG geometric mean concentration by age, ethnicity, timing of boosting and previous infection status and compared strata using Kruskall-Wallis tests.</p></sec><sec><title>Results</title><p>Among 985 vaccinated HCWs. IgG titres gradually decreased in all groups over the study duration. Younger or previously infected individuals had higher initial IgG levels (p&lt;0.001 in both cases); differences substantially decreased or disappeared at 7-9 months, before boosting. Pre-infection IgG levels in infected participants were similar to levels measured at the same timepoint in HCWs who remained uninfected (p&gt;0.3). IgG GMC in those boosted 6-7 months after dose 2 was lower compared with those boosted 8-9 months after (1999-vs 2736, p=0.02).</p></sec><sec><title>Conclusions</title><p>Immunity waned 6 months post-priming in all age groups and in previously infected individuals, reversed by boosting. IgG titres decrease among previously infected individuals and the proportion of reinfected individuals in this group, comparable to the proportion of breakthrough infection in previously uninfected individuals suggests individuals with hybrid immunity (infection+vaccination) may also require further doses. Our study also highlights the difficulty in determining protective IgG levels and the need to clarify the optimal timing in 3 dose regimens</p></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472254', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Individualized isometric neuromuscular electrical stimulation training promotes myonuclear accretion in mouse skeletal muscle', 'authors': \"[{'name': 'Aliki Zavoriti'}, {'name': 'Aurlie Fessard'}, {'name': 'Masoud Rahmati'}, {'name': 'Peggy Del Carmine'}, {'name': 'Bndicte Chazaud'}, {'name': 'Julien Gondin'}]\", 'abstract': '<p>Skeletal muscle is a plastic tissue that adapts to exercise through fusion of muscle stem cells (MuSCs) with myofibers, a physiological process referred to as myonuclear accretion. However, it is still unclear whether myonuclear accretion is driven by increased mechanical loading <italic>per se</italic>, or occurs, at least in part, in response to exercise-induced muscle injury. Here, we developed a carefully monitored and individualized neuromuscular electrical stimulation (NMES) training protocol of the mouse plantar flexor muscles. Each NMES training session consisted of 80 isometric contractions at a submaximal mechanical intensity corresponding to ~ 15% of maximal tetanic force to avoid muscle damage. NMES trained mice were stimulated for 2  3 consecutive days separated by one day of rest, for a total of 6 sessions. Experiments were conducted on C57BL/6J and BALB/c males at 10-12 weeks of age. NMES led to a robust myonuclear accretion and higher MuSC content in <italic>gastrocnemius</italic> muscle of both mouse lines, without overt signs of muscle damage/regeneration or muscle hypertrophy or force improvement. This new mouse model of myonuclear accretion relying on the main function of skeletal muscles, <italic>i.e.</italic>, force production in response to electrical stimuli, will be of utmost interest to further understand the role of MuSCs in skeletal muscle adaptations.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472361', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Dynamic Boolean modelling reveals the influence of energy supply on bacterial efflux pump expression', 'authors': \"[{'name': 'Ryan Kerr'}, {'name': 'Sara Jabbari'}, {'name': 'Jessica M. A. Blair'}, {'name': 'Iain G. Johnston'}]\", 'abstract': '<p>Antimicrobial resistance (AMR) is a global health issue. One key factor contributing to AMR is the ability of bacteria to export drugs through efflux pumps, which relies on the ATP-dependent expression and interaction of several controlling genes. Recent studies have shown significant cell-to-cell ATP variability exists within clonal bacterial populations, but the contribution of intrinsic cell-to-cell ATP heterogeneity is generally overlooked in understanding efflux pumps. Here, we consider how ATP variability influences gene regulatory networks controlling expression of efflux pump genes in two bacterial species. We develop and apply a generalisable Boolean modelling framework, developed to incorporate the dependence of gene expression dynamics on available cellular energy supply. Theoretical results show differences in energy availability can cause pronounced downstream heterogeneity in efflux gene expression. Cells with higher energy availability have a superior response to stressors. Further, in the absence of stress, model bacteria develop heterogeneous pulses of efflux pump gene expression which contribute to a sustained sub-population of cells with increased efflux expression activity, potentially conferring a continuous pool of intrinsically resistant bacteria. This modelling approach thus reveals an important source of heterogeneity in cell responses to antimicrobials and sheds light on potentially targetable aspects of efflux pump-related antimicrobial resistance.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.472640', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Role of immigrants and muzzle contacts in the uptake of a novel food by wild vervet monkeys', 'authors': \"[{'name': 'Pooja Dongre'}, {'name': 'Charlotte Canteloup'}, {'name': 'Galle Lant'}, {'name': 'Mathieu Cantat'}, {'name': 'Erica van de Waal'}]\", 'abstract': '<p>The entry into and uptake of information in social groups is critical for behavioural adaptation by long-lived species in rapidly changing environments. We exposed five groups of wild vervet monkeys to a novel food to investigate innovation of processing and consuming it. We observed whether dispersing males could introduce this innovation into groups and compare uptake between groups. We report that immigrant males innovated in two groups, and an infant innovated in one group. In two other groups, immigrant males imported the innovation from another group. Uptake of the novel food was fastest in groups where immigrant males ate first and younger individuals were most likely to acquire the novel food. We also investigated the role of muzzle contact behaviour in information seeking around the novel food. Muzzle contacts decreased in frequency over repeated exposures to the novel food, were targeted towards knowledgeable adults the most, and juveniles and nave individuals initiated the most. Knowledgeable adult males were targeted but rarely initiated muzzle contacts, whereas knowledgeable adult females were targeted and initiated them, emphasising an adult sex difference in seeking social information. We highlight the potential of immigrants as vectors of information, enabling rapid behavioural adaptation and population level culture.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.472433', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': True, 'title': 'Production and secretion of functional full-length SARS-CoV-2 spike protein in Chlamydomonas reinhardtii', 'authors': \"[{'name': 'Anna Kiefer'}, {'name': 'Justus Niemeyer'}, {'name': 'Anna Probst'}, {'name': 'Gerhard Erkel'}, {'name': 'Michael Schroda'}]\", 'abstract': '<p>The spike protein is the major protein on the surface of coronaviruses. It is therefore the prominent target of neutralizing antibodies and consequently the antigen of all currently admitted vaccines against SARS-CoV-2. Since it is a 1273-amino acids glycoprotein with 22 N-linked glycans, the production of functional, full-length spike protein was limited to mammalian and insect cells, requiring complex culture media. Here we report the production of full-length SARS-CoV-2 spike protein  lacking the C-terminal membrane anchor  as a secreted protein in the prefusion-stabilized conformation in the unicellular green alga <italic>Chlamydomonas reinhardtii</italic>. We show that the spike protein is efficiently cleaved at the furin cleavage site during synthesis in the alga and that cleavage is abolished upon mutation of the multi-basic cleavage site. We could enrich the spike protein from culture medium by ammonium sulfate precipitation and demonstrate its functionality based on its interaction with recombinant ACE2 and ACE2 expressed on human 293T cells. <italic>Chlamydomonas reinhardtii</italic> is a GRAS organism that can be cultivated at low cost in simple media at a large scale, making it an attractive production platform for recombinant spike protein and other biopharmaceuticals in low-income countries.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.15.472848', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Flv3A facilitates O2 photoreduction and affects H2 photoproduction independently of Flv1A in diazotrophic Anabaena filaments', 'authors': \"[{'name': 'Anita Santana Sanchez'}, {'name': 'Lauri Nikkanen'}, {'name': 'Gabor Toth'}, {'name': 'Maria Ermakova'}, {'name': 'Sergey Kosourov'}, {'name': 'Julia Walter'}, {'name': 'Meilin He'}, {'name': 'Eva-Mari Aro'}, {'name': 'Yagut Allahverdiyeva'}]\", 'abstract': '<p>The model heterocyst-forming filamentous cyanobacterium, Anabaena sp. PCC 7120 (Anabaena) represents multicellular organisms capable of simultaneously performing oxygenic photosynthesis in vegetative cells and the O2-sensitive N2-fixation inside the heterocysts. The flavodiiron proteins (FDPs) have been shown to participate in photoprotection of photosynthesis by driving excess electrons to O2 (Mehler-like reaction). Here, we addressed the physiological relevance of the vegetative cell-specific Flv1A and Flv3A on the bioenergetic processes occurring in the diazotrophic Anabaena under variable CO2. We demonstrate that both Flv1A and Flv3A are required for proper induction of the Mehler-like reaction upon a sudden change in light intensity, which is likely important for the activation of carbon-concentrating mechanisms (CCM) and CO2 fixation. Nevertheless, Flv3A showed a more important role in photoprotection than Flv1A. Under low CO2 diazotrophic conditions, Flv3A is capable of mediating moderate O2 photoreduction, independently of Flv1A, but in coordination with Flv2 and Flv4. Strikingly, the lack of Flv3A resulted in strong downregulation of the heterocyst-specific uptake hydrogenase, which led to enhanced H2 photoproduction under both oxic and micro-oxic conditions. These results reveal a novel regulatory network between the Mehler-like reaction and the H2 metabolism, which is of great interest for future photobiological production of H2 in Anabaena.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.17.21267350', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Risk of SARS-CoV-2 testing, PCR-confirmed infections and COVID-19-related hospital admissions in children and young people: birth cohort study', 'authors': \"[{'name': 'Pia Hardelid'}, {'name': 'Graziella Favarato'}, {'name': 'Linda Wijlaars'}, {'name': 'Lynda Fenton'}, {'name': 'Jim McMenamin'}, {'name': 'Tom Clemens'}, {'name': 'Chris DIBBEN'}, {'name': 'Ai Milojevic'}, {'name': 'Alison Macfarlane'}, {'name': 'Jonathon Taylor'}, {'name': 'Steve Cunningham'}, {'name': 'Rachael Wood'}]\", 'abstract': '<p>Background\\nThere have been no population-based studies of SARS-CoV-2 testing, PCR-confirmed infections and COVID-19-related hospital admissions across the full paediatric age range. We examine the epidemiology of SARS-CoV-2 in children and young people (CYP) aged &lt;23 years. \\n\\nMethods\\nWe used a birth cohort of all children born in Scotland since 1997, constructed via linkage between vital statistics, hospital records and SARS-CoV-2 surveillance data. We calculated risks of tests and PCR-confirmed infections per 1000 CYP-years between August and December 2020, and COVID-19-related hospital admissions per 100,000 CYP-years between February and December 2020. We used Poisson and Cox proportional hazards regression models to determine risk factors.\\n\\nResults\\nAmong the 1226855 CYP in the cohort, there were 378402 tests, 19005 PCR confirmed infections and 346 admissions, corresponding to rates of 770.8/1000 (95% confidence interval 768.4-773.3), 179.4 (176.9-182.0) and 29.4/100,000 (26.3-32.8) CYP-years respectively. Infants had the highest COVID-19-related admission rates. Chronic conditions, particularly multiple types of conditions, was strongly associated with COVID-19-related admissions across all ages. The hazard ratio for &gt;1 chronic condition type was 12.2 (7.9-18.82) compared to children with no chronic conditions. 89% of admitted children had no chronic conditions recorded.\\n\\nConclusions\\nInfants, and CYP with chronic conditions are at highest risk of admission with COVID-19, however the majority of admitted CYP have no chronic conditions. These results provide evidence to support risk/benefit analyses for paediatric COVID-19 vaccination programmes. Studies examining whether maternal vaccine during pregnancy prevents COVID-19 admissions in infants are urgently needed. \\n\\nFunding\\nUK Research and Innovation-Medical Research Council</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.472566', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Embryo-scale epithelial buckling forms a propagating furrow that initiates gastrulation', 'authors': \"[{'name': 'Julien Fierling'}, {'name': 'Alphy John'}, {'name': 'Barthlmy Delorme'}, {'name': 'Alexandre Torzynski'}, {'name': 'Guy Blanchard'}, {'name': 'Claire Lye'}, {'name': 'Grgoire Malandain'}, {'name': 'Bndicte Sanson'}, {'name': 'Jocelyn tienne'}, {'name': 'Philippe Marmottant'}, {'name': 'Catherine Quilliet'}, {'name': 'Matteo Rauzi'}]\", 'abstract': '<p>Cell apical constriction driven by actomyosin contraction forces is a conserved mechanism during tissue folding in embryo development. While much effort has been made to better understand the molecular mechanisms responsible for apical constriction, it is still not clear if apical actomyosin contraction forces are necessary or sufficient <italic>per se</italic> to drive tissue folding. To tackle this question, we use the <italic>Drosophila</italic> embryo model system that forms a furrow on the ventral side, initiating mesoderm internalization. Past computational models support the idea that cell apical contraction forces may not be sufficient and that active or passive cell apico-basal forces may be necessary to drive cell wedging and tissue furrowing. By using 3D computational modelling and <italic>in toto</italic> embryo image analysis and manipulation, we now challenge this idea and show that embryo-scale force balance of the tissue surface, rather than cell-autonomous shape changes, is necessary and sufficient to drive a buckling of the epithelial surface forming a furrow which propagates and initiates embryo gastrulation.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.21267847', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'COVID-19 vaccination, from first dose to booster: New insights into the frequency of most common systemic adverse events and possible booster nocebo effects based on a systematic review', 'authors': \"[{'name': 'Claudia Behrens'}, {'name': 'Maria Samii-Moghadam'}, {'name': 'Tatiana Gasperazzo'}, {'name': 'Anna M. Gross'}, {'name': 'Jack Mitchell'}, {'name': 'Johannes B. Lampe'}]\", 'abstract': '<sec><title>Background</title><p>Based on placebo data, it has been recently demonstrated that the frequencies of most common adverse events (AEs) of COVID-19 vaccination are overestimated due to negative expectation bias of vaccine recipients (nocebo effect). Since booster studies lack comparators, estimating the extent of the nocebo effect is difficult. We aimed to overcome this obstacle through a systematic comparison of most common AE frequencies across vaccine doses (first, second, booster), age groups, and vaccine vs. placebo arms.</p></sec><sec><title>Methods</title><p>We systematically assessed systemic AEs in approved COVID-19 vaccines according to the PRISMA guidelines. All documents regarding COVID-19 vaccines with a booster dose authorized by the FDA (cutoff date 19 November 2021) were systematically searched on PubMed and the FDA website. Solicited systemic AEs from all documents supporting approval/authorization were collected. After standardization of doses and age groups, AE frequencies were compared between vaccine and placebo.</p></sec><sec><title>Findings</title><p>Two trials were identified for BNT162b2 (n=21,785 participants), two for mRNA-1273 (n=22,324), and one for Ad26.COV2.S (n=4,085). Fever cases dropped to about half with the booster dose in all vaccines, whereas all other systemic AE frequencies were similar to the preceding dose. Almost no fever cases occurred with placebo (first/second dose); all other systemic AEs occurred at high frequencies. After subtracting placebo arm values from vaccine values, the frequencies for the various AEs were roughly comparable within each dose for each vaccine.</p></sec><sec><title>Interpretation</title><p>Fever is the only solicited systemic AE that can be assessed objectively. It occurs about 50% less often with the booster than with the preceding dose. This may indirectly indicate a considerable overestimation of systemic AEs in the case of booster vaccinations and a pronounced nocebo effect. The nocebo effect appears to substantially contribute to the differences in the frequencies of the various systemic AEs.</p></sec><sec><title>Funding</title><p>No funding received.</p></sec><sec><title>Research in context</title><sec><title>Evidence before this study</title><p>A high nocebo effect has recently been shown for the most common solicited adverse events (AEs) based on the randomized controlled trials of several vaccines approved for COVID-19. To date, there has been no systematic review of systemic AEs in COVID-19 vaccines with a booster dose approved by the FDA. Normally, assessing the extent of the nocebo effect requires the implementation of randomized, placebo-controlled studies. However, for ethical reasons, studies underlying the authorization of COVID-19 booster vaccines are not controlled. Therefore, reported AEs lack any reference parameter.</p><p>We searched PubMed and medRxiv for nocebo effects regarding the objectively measurable AE fever using the terms (COVID) AND (nocebo) AND (fever) with no language restrictions; no hits were found on PubMed and six were found on medRxiv. The search without any indication, using only the search terms (nocebo) AND (fever), resulted in two hits on PubMed and six hits on medRxiv. An additional search was performed for (vaccine or COVID) AND nocebo; 21 hits were found on PubMed and 12 on medRxiv. None of the hits obtained with the three searches showed evidence of relevance for the present topic of a direct association of fever with the nocebo effect.</p></sec><sec><title>Added value of this study</title><p>Instead of placebo AE data as reference parameter, we suggest fever as an objectively measurable value to estimate the extent of the nocebo effect in uncontrolled vaccination studies. To our knowledge, this is a new concept with which evidence for the extent of a possible nocebo effect in COVID-19 booster vaccination can be provided. With this approach we directly supplement information on the most recently reported nocebo effects in first, second and booster vaccine doses.</p><p>To the best of our knowledge, this is the first study to indirectly compare the solicited systemic AE frequencies of approved COVID-19 vaccines across all doses, with easy-to-understand graphs and based on all approval-relevant trials. This is astonishing, as in the context of a pandemic it is essential to communicate scientific data in a generally understandable manner and according to the highest scientific standards. The graphs provided in this work could serve as an example for such clear communication.</p></sec><sec><title>Implications of all the available evidence</title><p>Using the example of systemic AEs, we were able to show that the frequency of AEs in COVID-19 booster vaccination may be overestimated. Fever as an objective measure to estimate the nocebo effect could help to optimize public awareness campaigns in the future. The graphic presentation of results facilitates a deeper understanding of complex scientific data and provides new insights. It is therefore ideal for a highly dynamic scientific field and may also be applied to other challenges in the COVID-19 pandemic and beyond.</p></sec></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472466', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'Enrichment analysis on regulatory subspaces: a novel direction for the superior description of cellular responses to SARS-CoV-2', 'authors': \"[{'name': 'Pedro P. Rodrigues'}, {'name': 'Rafael S. Costa'}, {'name': 'Rui Henriques'}]\", 'abstract': '<p>Statement: The enrichment analysis of discriminative cell transcriptional responses to SARS-CoV-2 infection using biclustering produces a broader set of superiorly enriched GO terms and KEGG pathways against alternative state-of-the-art machine learning approaches, unraveling novel knowledge. Motivation and methods: The comprehensive understanding of the impacts of the SARS-CoV-2 virus on infected cells is still incomplete. This work identifies and analyses the main cell regulatory processes affected and induced by SARS-CoV-2, using transcriptomic data from several infectable cell lines available in public databases and in vivo samples. We propose a new class of statistical models to handle three major challenges, namely the scarcity of observations, the high dimensionality of the data, and the complexity of the interactions between genes. Additionally, we analyse the function of these genes and their interactions within cells to compare them to ones affected by IAV (H1N1), RSV and HPIV3 in the target cell lines. Results: Gathered results show that, although clustering and predictive algorithms aid classic functional enrichment analysis, recent pattern-based biclustering algorithms significantly improve the number and quality of the detected biological processes. Additionally, a comparative analysis of these processes is performed to identify potential pathophysiological characteristics of COVID-19. These are further compared to those identified by other authors for the same virus as well as related ones such as SARS-CoV-1. This approach is particularly relevant due to a lack of other works utilizing more complex machine learning tools within this context.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.21267809', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'mRNA COVID-19 vaccine effectiveness against SARS-CoV-2 infection in a prospective community cohort, rural Wisconsin, November 2020-December 2021', 'authors': \"[{'name': 'Huong Q McLean'}, {'name': 'David L McClure'}, {'name': 'Jennifer P King'}, {'name': 'Jennifer K Meece'}, {'name': 'David Pattinson'}, {'name': 'Gabriele Neumann'}, {'name': 'Yoshihiro Kawaoka'}, {'name': 'Melissa A Rolfes'}, {'name': 'Edward A Belongia'}]\", 'abstract': '<p>Reduced COVID-19 vaccine effectiveness (VE) has been observed with increasing predominance of the Delta variant. In a prospective rural community cohort of 1265 participants, VE against symptomatic and asymptomatic SARS-CoV-2 infection was 56% for mRNA COVID-19 vaccines.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472657', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Development and Characterization of Recombinant Vesicular Stomatitis Virus (rVSV)-based Bivalent Vaccine Against COVID-19 Delta Variant and Influenza Virus', 'authors': \"[{'name': 'Zhujun Ao'}, {'name': 'Maggie Jing Ouyang'}, {'name': 'Titus Abiola Olukitibi'}, {'name': 'Bryce Warner'}, {'name': 'Robert Vendramelli'}, {'name': 'Thang Truong'}, {'name': 'Manli Zhang'}, {'name': 'Sam Kung'}, {'name': 'Keith R Fowke'}, {'name': 'Darwyn Kobasa'}, {'name': 'Xiaojian Yao'}]\", 'abstract': '<p>COVID-19 and influenza are both highly contagious respiratory diseases with a wide range of severe symptoms and cause great disease burdens globally. It has become very urgent and important to develop a bivalent vaccine that is able to target these two infectious diseases simultaneously. In this study, we generated three attenuated replicating recombinant VSV (rVSV) vaccine candidates. These rVSV-based vaccines co-express SARS-CoV-2 Delta variant spike protein (SP) or the receptor binding domain (RBD) and four copies of the highly conserved M2 ectodomain (M2e) of influenza A fused with the Ebola glycoprotein DC-targeting/activation domain. Animal studies have shown that immunization with these bivalent rVSV vaccines induced efficient but variable levels of humoral and cell-mediated immune responses against both SARS-CoV-2 and influenza M2e protein. Significantly, our vaccine candidates induced production of high levels of neutralizing antibodies that protected cells against SARS-CoV-2 Delta and other SP-pseudovirus infections in culture. Furthermore, vaccination with the bivalent VSV vaccine via either intramuscular or intranasal route efficiently protected mice from the lethal challenge of H1N1 and H3N2 influenza viruses and significantly reduced viral load in the lungs. These studies provide convincing evidence for the high efficacy of this bivalent vaccine to prevent influenza replication and initiate robust immune responses against SARS-CoV-2 Delta variants. Further investigation of its efficacy to protect against SARS-CoV-2 Delta variants will provide substantial evidence for new avenues to control two contagious respiratory infections, COVID-19 and influenza.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.12.21267573', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'The age-dependent immunogenicity after two doses of MVC-COV1901 vaccine', 'authors': \"[{'name': 'Chia En Lien'}, {'name': 'Yi-Jiun Lin'}, {'name': 'Yi-Ling Lin'}, {'name': 'I-Chen Tai'}, {'name': 'Charles Chen'}]\", 'abstract': '<p>A post-hoc analysis of the phase 2 data was performed for the SARS-COV-2 subunit protein vaccine MVC-COV1901. Anti-spike IgG, neutralization assays with live virus and pseudovirus were used to demonstrate age-dependent vaccine-induced antibody response to the vaccine. Results showed that an association exists between age and immune responses to the vaccine, providing further support for the need of booster shots, especially for the older age groups.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.10.472134', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'SARS-CoV2 variant-specific replicating RNA vaccines protect from disease and pathology and reduce viral shedding following challenge with heterologous SARS-CoV2 variants of concern', 'authors': \"[{'name': 'David W. Hawman'}, {'name': 'Kimberly Meade-White'}, {'name': 'Jacob Archer'}, {'name': 'Shanna Leventhal'}, {'name': 'Drew Wilson'}, {'name': 'Carl Shaia'}, {'name': 'Samantha Randall'}, {'name': 'Amit P. Khandhar'}, {'name': 'Tien-Ying Hsiang'}, {'name': 'Michael Gale'}, {'name': 'Peter Berglund'}, {'name': 'Deborah Heydenburg Fuller'}, {'name': 'Heinz Feldmann'}, {'name': 'Jesse H. Erasmus'}]\", 'abstract': '<p>Despite mass public health efforts, the SARS-CoV2 pandemic continues as of late-2021 with resurgent case numbers in many parts of the world. The emergence of SARS-CoV2 variants of concern (VoC) and evidence that existing vaccines that were designed to protect from the original strains of SARS-CoV-2 may have reduced potency for protection from infection against these VoC is driving continued development of second generation vaccines that can protect against multiple VoC. In this report, we evaluated an alphavirus-based replicating RNA vaccine expressing Spike proteins from the original SARS-CoV-2 Alpha strain and recent VoCs delivered <italic>in vivo</italic> via a lipid inorganic nanoparticle. Vaccination of both mice and Syrian Golden hamsters showed that vaccination induced potent neutralizing titers against each homologous VoC but reduced neutralization against heterologous challenges. Vaccinated hamsters challenged with homologous SARS-CoV2 variants exhibited complete protection from infection. In addition, vaccinated hamsters challenged with heterologous SARS-CoV-2 variants exhibited significantly reduced shedding of infectious virus. Our data demonstrate that this vaccine platform elicits significant protective immunity against SARS-CoV2 variants and supports continued development of this platform.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.15.472751', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Polymorphism-aware estimation of species trees and evolutionary forces from genomic sequences with RevBayes', 'authors': \"[{'name': 'Rui Borges'}, {'name': 'Bastien Boussau'}, {'name': 'Sebastian Hoehna'}, {'name': 'Ricardo J Pereira'}, {'name': 'Carolin Kosiol'}]\", 'abstract': '<p>The availability of population genomic data through new sequencing technologies gives unprecedented opportunities for estimating important evolutionary forces such as genetic drift, selection, and mutation biases across organisms. Yet, analytical methods that can handle polymorphisms jointly with sequence divergence across species are rare and not easily accessible to empiricists. We implemented polymorphism-aware phylogenetic models (PoMos), an alternative approach for species tree estimation, in the Bayesian phylogenetic software RevBayes. PoMos naturally account for incomplete lineage sorting, which is known to cause difficulties for phylogenetic inference in species radiations, and scale well with genome-wide data. Simultaneously, PoMos can estimate mutation and selection biases. We have applied our methods to resolve the complex phylogenetic relationships of a young radiation of Chorthippus grasshoppers, based on coding sequences. In addition to establishing a well-supported species tree, we found a mutation bias favoring AT alleles and selection bias promoting the fixation of GC alleles, the latter consistent with GC-biased gene conversion. The selection bias is two orders of magnitude lower than genetic drift, validating the critical role of nearly neutral evolutionary processes in species radiation. PoMos offer a wide range of models to reconstruct phylogenies and can be easily combined with existing models in RevBayes - e.g., relaxed clock and divergence time estimation - offering new insights into the evolutionary processes underlying molecular evolution and, ultimately, species diversification.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.473053', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Prediction of prophages and their host ranges in pathogenic and commensal Neisseria species', 'authors': \"[{'name': 'Giulia Orazi'}, {'name': 'Alan J Collins'}, {'name': 'Rachel J Whitaker'}]\", 'abstract': '<p>The genus <italic>Neisseria</italic> includes two pathogenic species, <italic>N. gonorrhoeae</italic> and <italic>N. meningitidis</italic>, and numerous commensal species. <italic>Neisseria</italic> species frequently exchange DNA with one other, primarily via transformation and homologous recombination, and via multiple types of mobile genetic elements (MGEs). Few <italic>Neisseria</italic> bacteriophages (phages) have been identified and their impact on bacterial physiology is poorly understood. Furthermore, little is known about the range of species that <italic>Neisseria</italic> phages can infect. In this study, we used three virus prediction tools to scan 248 genomes of 21 different <italic>Neisseria</italic> species and identified 1302 unique predicted prophages. Using comparative genomics, we found that many predictions are dissimilar from other prophages and MGEs previously described to infect <italic>Neisseria</italic> species. We also identified similar predicted prophages in genomes of different <italic>Neisseria</italic> species. Additionally, we examined CRISPR-Cas targeting of each <italic>Neisseria</italic> genome and predicted prophage. While CRISPR targeting of chromosomal DNA appears to be common among several <italic>Neisseria</italic> species, we found that 20% of the prophages we predicted are targeted significantly more than the rest of the bacterial genome in which they were identified (i.e., backbone). Furthermore, many predicted prophages are targeted by CRISPR spacers encoded by other species. We then used these results to infer additional host species of known <italic>Neisseria</italic> prophages and predictions that are highly targeted relative to the backbone. Together, our results suggest that we have identified novel <italic>Neisseria</italic> prophages, several of which may infect multiple <italic>Neisseria</italic> species. These findings have important implications for understanding horizontal gene transfer between members of this genus. IMPORTANCE: Drug-resistant <italic>N. gonorrhoeae</italic> is a major threat to human health. Commensal <italic>Neisseria</italic> species are thought to serve as reservoirs of antibiotic resistance and virulence genes for the pathogenic species <italic>N. gonorrhoeae</italic> and <italic>N. meningitidis</italic>. Therefore, it is important to understand both the diversity of mobile genetic elements (MGEs) that can mediate horizontal gene transfer within this genus, and the breadth of species these MGEs can infect. In particular, few bacteriophages (phages) have been identified and characterized in <italic>Neisseria</italic> species. In this study, we identified a large number of candidate phages integrated within the genomes of commensal and pathogenic <italic>Neisseria</italic> species, many of which appear to be novel phages. Importantly, we discovered extensive interspecies targeting of predicted phages by <italic>Neisseria</italic> CRISPR-Cas systems, which may reflect their movement between different species. Uncovering the diversity and host range of phages is essential for understanding how they influence the evolution of their microbial hosts.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.473081', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Biology and engineering of integrative and conjugative elements: Construction and analyses of hybrid ICEs reveal element functions that affect species-specific efficiencies', 'authors': \"[{'name': 'Emily L Bean'}, {'name': 'Calvin Herman'}, {'name': 'Alan D. Grossman'}]\", 'abstract': '<p>Integrative and conjugative elements (ICEs) are mobile genetic elements that reside in a bacterial host chromosome and are prominent drivers of bacterial evolution. They are also powerful tools for genetic analyses and engineering. Transfer of an ICE to a new host involves many steps, including excision from the chromosome, DNA processing and replication, transfer across the envelope of the donor and recipient, processing of the DNA, and eventual integration into the chromosome of the new host (now a stable transconjugant). Interactions between an ICE and its hosts throughout the life cycle likely influence the efficiencies of acquisition by new hosts. Here, we investigated how different functional modules of two ICEs, Tn<italic>916</italic> and ICE<italic>Bs1</italic>, affect the transfer efficiencies into different host bacteria. We constructed hybrid elements that utilize the high-efficiency regulatory and excision modules of ICE<italic>Bs1</italic> and the conjugation genes of Tn<italic>916</italic>. These elements produced more transconjugants than Tn<italic>916</italic>, likely due to increased excision frequencies. We also found that several Tn<italic>916</italic> and ICE<italic>Bs1</italic> components can substitute for one other. Using <italic>B. subtilis</italic> donors and three <italic>Enterococcus</italic> species as recipients, we found that different hybrid elements were more readily acquired by some species than others, demonstrating species-specific interactions in steps of the ICE life cycle. This work demonstrates that hybrid elements utilizing the efficient regulatory functions of ICE<italic>Bs1</italic> can be built to enable efficient transfer into and engineering of a variety of other species.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.472681', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'pop-1/TCF, ref-2/ZIC and T-box factors regulate the development of anterior cells in the C. elegans embryo', 'authors': \"[{'name': 'Jonathan D. Rumley'}, {'name': 'Elicia A. Preston'}, {'name': 'Dylan Cook'}, {'name': 'Felicia L. Peng'}, {'name': 'Amanda L. Zacharias'}, {'name': 'Lucy Wu'}, {'name': 'Ilona Jileaeva'}, {'name': 'John Isaac Murray'}]\", 'abstract': '<p>Patterning of the anterior-posterior axis is fundamental to animal development. The Wnt pathway plays a major role in this process by activating the expression of posterior genes in animals from worms to humans. This observation raises the question of whether the Wnt pathway or other regulators control the expression of the many anterior-expressed genes. We found that the expression of five anterior-specific genes in <italic>Caenorhabditis elegans</italic> embryos depends on the Wnt pathway effectors <italic>pop-1/TCF</italic> and <italic>sys-1/</italic>-catenin. We focused further on one of these anterior genes, <italic>ref-2/ZIC,</italic> a conserved transcription factor expressed in multiple anterior lineages. Live imaging of <italic>ref-2</italic> mutant embryos identified defects in cell division timing and position in anterior lineages. <italic>Cis-</italic>regulatory dissection identified three <italic>ref-2</italic> transcriptional enhancers, one of which is necessary and sufficient for anterior-specific expression. This enhancer is activated by the T-box transcription factors TBX-37 and TBX-38, and surprisingly, concatemerized TBX-37/38 binding sites are sufficient to drive anterior-biased expression alone, despite the broad expression of TBX-37 and TBX-38. Taken together, our results highlight the diverse mechanisms used to regulate anterior expression patterns in the embryo.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.473064', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Nuclear lamins promote protrusion dynamics and collective, confined migration in vivo', 'authors': \"[{'name': 'Lauren Penfield'}, {'name': 'Denise Montell'}]\", 'abstract': '<p>Cells migrate collectively through confined environments during development and cancer metastasis. While the nucleus, a large and stiff organelle, impedes cell migration between non-deformable pillars in vitro, its function in vivo may vary depending on the microenvironment. Further, it is unknown how nuclei contribute to collective migration in vivo and whether nuclei in different positions within cell collectives experience different forces. Here, we use border cell migration in the fly ovary as an in vivo model to investigate the effects of confined, collective migration on nuclei and the contribution of nuclear lamins to migration. We found severe yet transient nuclear deformations occur, particularly in the leading cell, as border cells squeeze through tiny crevices between germline cells, termed nurse cells. Leading cells extend protrusions between nurse cells, which may pry open space to allow the cluster to advance. Here we report that the leading cell nuclei deformed as they moved into leading protrusions. Then as protrusions widened, the nucleus recovered a more circular shape. These data suggest that lead cell nuclei may help protrusions expand and thereby enlarge the migration path. To test how nuclei might promote or impede border cell migration, we investigated nuclear lamins, proteins that assemble into intermediate filaments and structurally support the nuclear envelope. Depletion of the Drosophila B-type lamin, Lam, from the outer, motile border cells, but not the inner, nonmotile polar cells, impeded border cell migration, whereas perturbations of the A-type lamin, LamC, did not. While wild type border cell clusters typically have one large leading protrusion as they delaminate from the anterior follicular epithelium, clusters depleted of B-type lamin had multiple, short-lived protrusions, resulting in unproductive cluster movement and failure to progress along the migration path. Further, border cell nuclei depleted of B-type lamins were small, formed blebs, and ruptured. Together, these data indicate that B-type lamin is requied for nuclear integrity, which in turn stabilizes the leading protrusion and promotes overall cluster polarization and collective movement through confined spaces.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.21267798', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Obstetrics and Gynecology', 'covid': False, 'title': 'The association between midwifery staffing levels and the experiences of mothers on postnatal wards: cross sectional analysis of routine data', 'authors': \"[{'name': 'Lesley Turner'}, {'name': 'David Culliford'}, {'name': 'Jane Ball'}, {'name': 'Ellen Kitson-Reynolds'}, {'name': 'Peter Griffiths'}]\", 'abstract': '<sec><title>Background</title><p>Women have consistently reported lower satisfaction with postnatal care compared with antenatal and labour care. The aim of this research was to examine whether womens experience of inpatient postnatal care in England is associated with variation in midwifery staffing levels.</p></sec><sec><title>Methods</title><p>Analysis of data from the National Maternity Survey in 2018 including 17,611 women from 129 organisations. This was linked to hospital midwifery staffing numbers from the National Health Service (NHS) Workforce Statistics and the number of births from Hospital Episode Statistics. A two-level logistic regression model was created to examine the association of midwifery staffing levels and experiences in post-natal care.</p></sec><sec><title>Results</title><p>The median full time equivalent midwives per 100 births was 3.55 (interquartile range 3.26 to 3.78). Higher staffing levels were associated with less likelihood of women reporting delay in discharge (adjusted odds ratio [aOR] 0.849, 95% CI 0.753 to 0.959, p=0.008), increased chances of women reporting that staff always helped in a reasonable time aOR1.200 (95% CI 1.052, 1.369, p=0.007) and that they always had the information or explanations they needed aOR 1.150 (95% CI 1.040, 1.271, p=0.006). Women were more likely to report being treated with kindness and understanding with higher staffing, but the difference was small and not statistically significant aOR 1.059 (0.949, 1.181, p=0.306).</p></sec><sec><title>Conclusions</title><p>Negative experiences for women on postnatal wards were more likely to occur in trusts with fewer midwives. Low staffing could be contributing to discharge delays and lack of support and information, which may in turn have implications for longer term outcomes for maternal and infant wellbeing.</p></sec><sec><title>Explanation</title><p>While we recognise that not all gestational parents identify as women; this term was chosen as it has been used in the data source which was accessed for this study and represents most people having maternities.</p></sec><sec><title>Statement of significance (problem)</title><p>Women report negative experiences of postnatal care compared with antenatal care and birth. There is a recognised shortage of midwives in maternity services, and this may be impacting on the quality of postnatal care.</p></sec><sec><title>What is already known?</title><p>There is evidence that midwifery staffing levels are associated with birth outcomes but little empirical evidence on the impact of midwifery staffing levels in postnatal care</p></sec><sec><title>What this paper adds?</title><p>This analysis of survey data supports previous findings that increased midwifery staffing is associated with benefits. This is the first study to examine the effects of staffing on womens experience of postnatal care.</p></sec>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472799', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Discovery of lipid-mediated protein-protein interactions in living cells using metabolic labeling with photoactivatable clickable probes', 'authors': \"[{'name': 'Roman O. Fedoryshchak'}, {'name': 'Andrii Gorelik'}, {'name': 'Mengjie Shen'}, {'name': 'Maria M. Shchepinova'}, {'name': 'Inmaculada Prez-Dorado'}, {'name': 'Edward W. Tate'}]\", 'abstract': '<p>Protein-protein interactions (PPIs) are essential and pervasive regulatory elements in cell biology. Despite development of a range of techniques to probe PPIs in living systems, there is a dearth of approaches to capture interactions driven by specific post-translational modifications (PTMs). Myristoylation is a lipid PTM added to more than 200 human proteins, where it may regulate membrane localization, stability or activity. Here we report design and synthesis of a panel of novel photocrosslinkable and clickable myristic acid analog probes, and their characterization as efficient substrates for human <italic>N</italic>-myristoyltransferases NMT1 and NMT2, both biochemically and through X-ray co-crystallography. We demonstrate metabolic incorporation of probes to label NMT substrates in cell culture and <italic>in situ</italic> intracellular photoactivation to form a covalent crosslink between modified proteins and their interactors, capturing a snapshot of interactions driven by the presence of the lipid PTM. Proteomic analyses revealed both known and multiple novel interactors of a series of myristoylated proteins, including ferroptosis suppressor protein FSP1 and spliceosome-associated RNA helicase DDX46. The concept exemplified by these probes offers an efficient approach for exploring the PTM-specific interactome, which may prove broadly applicable to other PTMs.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472559', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Prefrontal top-down projections control context-dependent strategy selection', 'authors': \"[{'name': 'Olivier Gschwend'}, {'name': 'Tao Yang'}, {'name': 'Danille van de Lisdonk'}, {'name': 'Xian Zhang'}, {'name': 'Radhashree Sharma'}, {'name': 'Bo Li'}]\", 'abstract': '<p>The rules governing behavior often vary with behavioral contexts. As a consequence, an action rewarded in one context may be discouraged in another. Animals and humans are capable of switching between behavioral strategies under different contexts and acting adaptively according to the variable rules, a flexibility that is thought to be mediated by the prefrontal cortex (PFC)<sup>14</sup>. However, how the PFC orchestrates context-dependent switch of strategies remains unclear. Here we show that pathway-specific projection neurons in the medial PFC (mPFC) differentially contribute to context-instructed strategy selection. In a decision-making task in which mice have been trained to flexibly switch between a previously established rule and a newly learned rule in a context-dependent manner, the activity of mPFC neurons projecting to the dorsomedial striatum encodes the contexts, and further represents decision strategies conforming to the old and new rules. Moreover, the activity of these neuron is required for context-instructed strategy selection. In contrast, the activity of mPFC neurons projecting to the ventral midline thalamus does not discriminate between the contexts, and represents the old rule even if mice have adopted the new one; furthermore, these neurons act to prevent the strategy switch under the new rule. Our results suggest that the mPFCstriatum pathway promotes flexible strategy selection guided by contexts, whereas the mPFCthalamus pathway favors fixed strategy selection by preserving old rules. Balanced activity between the two pathways may be critical for adaptive behaviors.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.472999', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': '3D-Strudel - a novel model-dependent map-feature validation method for high-resolution cryo-EM structures', 'authors': \"[{'name': 'Andrei Istrate'}, {'name': 'Zhe Wang'}, {'name': 'Garib N Murshudov'}, {'name': 'Ardan Patwardhan'}, {'name': 'Gerard J Kleywegt'}]\", 'abstract': '<p>Recent technological advances in electron cryo-microscopy (cryo-EM) have led to significant improvements in the resolution of many single-particle reconstructions and a sharp increase in the number of entries released in the Electron Microscopy Data Bank (EMDB) every year, which in turn has opened new possibilities for data mining. Here we present a resolution-dependent library of rotamer-specific amino-acid map motifs mined from entries in the EMDB archive with reported resolution between 2.0 and 4.0A. We further describe 3D-Strudel, a method for map/model validation based on these libraries. 3D-Strudel calculates linear correlation coefficients between the map values of a map-motif from the library and the experimental map values around a target residue. We also present \"Strudel Score\", a plug-in for ChimeraX, as a user-friendly tool for visualisation of 3D-Strudel validation results.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.21267743', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'HIV/AIDS', 'covid': False, 'title': 'HIV Phylogenetics Reveals Overlapping Transmission Networks among Cities and Key Populations in Pakistan', 'authors': \"[{'name': 'Adriano de Bernardi Schneider'}, {'name': 'Franois Cholette'}, {'name': 'Yann Pelcat'}, {'name': 'Aaron G. Lim'}, {'name': 'Peter T. Vickerman'}, {'name': 'Adiba Hassan'}, {'name': 'Laura H. Thompson'}, {'name': 'James F. Blanchard'}, {'name': 'Faran Emmanuel'}, {'name': 'Tahira Reza'}, {'name': 'Nosheen Dar'}, {'name': 'Nadeem Ikram'}, {'name': 'Richard Pilon'}, {'name': 'Chris Archibald'}, {'name': 'Jeffrey B. Joy'}, {'name': 'Paul Sandstrom'}, {'name': 'Joel O. Wertheim'}]\", 'abstract': '<p>The first case of HIV in Pakistan was documented in 1987, with multiple subtypes and circulating recombinant forms being introduced and currently circulating in the country. Since then, there has been a shift in the country from a low prevalence/high risk to a high-risk concentrated epidemic. Pakistans epidemic is concentrated among key populations at greater risk of HIV infection including people who inject drugs (PWID), Hijra sex workers (HSW), female sex workers (FSW), male sex workers (MSW), and men who have sex with men (MSM). This study focused on the geographical aspect as well as on the interactions between key populations at higher risk of contracting HIV. We aimed at understanding the behavior of these key populations at a molecular level with high granularity as well as investigating the possibility of multiple HIV-1 introductions in Pakistan. In this cross-sectional biological and behavioral survey, we collected dried blood spots (DBS) for the purposes of seroprevalence estimates and molecular epidemiology from individuals in 17 cities in Pakistan representing four key populations: PWID, HSW, MSW, and FSW. A total of 1153 envelope sequences (reference positions in HXB2: 7860-8274) of HIV were sequenced using a Sanger-based sequencing approach. To identify clusters based on the introduction of the virus in Pakistan from foreign countries we added 3623 publicly available HIV envelope sequences to our dataset. Phylogeographic inference suggests at least 15 independent introductions of the virus into Pakistan, with a total of 12 clusters ranging from 3 to 675 sequences in size containing sequences from Pakistan and neighboring countries exclusively. Our phylogenetic analysis shows a significant degree of connectivity and directionality suggesting broad and overlapping networks of HIV-1 transmission among cities and key populations in Pakistan.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472652', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Using Piecewise Regression to Identify Biological Phenomena in Biotelemetry Datasets', 'authors': \"[{'name': 'David Wolfson'}, {'name': 'John Fieberg'}, {'name': 'David E Andersen'}]\", 'abstract': \"<p>Technological advances in the field of animal tracking have greatly expanded the potential to remotely monitor animals, opening the door to exploring how animals shift their behavior over time or respond to external stimuli. A wide variety of animal-borne sensors can provide information on an animal's location, movement characteristics, external environmental conditions, and internal physiological status. Here, we demonstrate how piecewise regression can be used to identify the presence and timing of potential shifts in a variety of biological responses using GPS telemetry and other biologging data streams. Different biological latent states can be inferred by partitioning a time-series into multiple segments based on changes in modeled responses (e.g., their mean, variance, trend, degree of autocorrelation) and specifying a unique model structure for each interval. We provide five example applications highlighting a variety of taxonomic species, data streams, timescales, and biological phenomena. These examples include a short-term behavioral response (flee and return) by a trumpeter swan (Cygnus buccinator) immediately following a GPS collar deployment; remote identification of parturition based on movements by a pregnant moose (Alces alces); a physiological response (spike in heart-rate) in a black bear (Ursus americanus) to a stressful stimulus (presence of a drone); a mortality event of a trumpeter swan signaled by changes in collar temperature and Overall Dynamic Body Acceleration; and an unsupervised method for identifying the onset, return, duration, and staging use of sandhill crane (Antigone canadensis) migration. We implement analyses using the mcp package in R, which provides functionality for specifying and fitting a wide variety of user-defined model structures in a Bayesian framework and methods for assessing and comparing models using information criterion and cross-validation measures. This approach uses simple modeling approaches that are accessible to a wide audience and is a straightforward means of assessing a variety of biologically relevant changes in animal behavior.</p>\", 'posted': '2021-12-16'}, {'doi': '10.1101/2021.08.30.458238', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Transient population dynamics drive the spread of invasive wild pigs in North America', 'authors': \"[{'name': 'Ryan S Miller'}, {'name': 'Michael A Tabak'}, {'name': 'Christopher L Burdett'}, {'name': 'David W Wolfson'}]\", 'abstract': '<p>Invasion of nonindigenous species is considered one of the most urgent problems affecting native ecosystems and agricultural systems. Mechanistic models that account for short-term population dynamics can improve prediction because they incorporate differing demographic processes that link the environmental conditions of a spatial location explicitly with the invasion process. Yet short-term population dynamics are rarely accounted for in spatial models of invasive species spread.</p><p>Accounting for transient population dynamics, we predict the population growth rate and establishment probability of wild pigs following introduction into any location in North America. We compared predicted population growth rate with observed geographic rates of spread and found significant relationships between the annual rate of spread and population growth rates.</p><p>We used geospatial data on the distribution of mast producing tree species (a principle forage resource of wild pigs) and agricultural crops that can replace mast in their diets to predict population dynamics using transient population simulations. We simulated populations under different initial population sizes (i.e. number of introduced individuals, often termed propagule size) and for different amounts of time following introduction. By varying the initial population size and simulation time, we were able to identify areas in North America with high probability for establishment of wild pigs if introduced. Our findings can be used to inform surveillance and removal efforts to reduce the potential for establishment and spread of wild pigs.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.472906', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The genome sequence of the lesser marbled fritillary, Brenthis ino, and evidence for a segregating neo-Z chromosome', 'authors': \"[{'name': 'Alexander Mackintosh'}, {'name': 'Dominik Laetsch'}, {'name': 'Tobias Baril'}, {'name': 'Robert Foster'}, {'name': 'Vlad Dinc'}, {'name': 'Roger Vila'}, {'name': 'Alex Hayward'}, {'name': 'Konrad Lohse'}]\", 'abstract': '<p>The lesser marbled fritillary, <italic>Brenthis ino</italic> (Rottemburg, 1775), is a species of Palearctic butterfly. Male <italic>B. ino</italic> individuals have been reported to have between 12 and 14 pairs of chromosomes, a much reduced chromosome number than is typical in butterflies. Here we present a chromosome-level genome assembly for <italic>B. ino</italic>, as well as gene and transposable element annotations. The assembly is 411.8 Mb in span with contig and scaffold N50s of 9.6 and 29.5 Mb respectively. We also show evidence that the male individual from which we generated HiC data was heterozygous for a neo-Z chromosome, consistent with inheriting 14 chromosomes from one parent and 13 from the other. This genome assembly will be a valuable resource for studying chromosome evolution in Lepidoptera, as well as for comparative and population genomics more generally.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.472726', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Artificial intelligence to solve the X-ray crystallography phase problem: a case study report', 'authors': \"[{'name': 'Irne Barbarin-Bocahu'}, {'name': 'Marc Graille'}]\", 'abstract': '<p>The determination of three dimensional structures of macromolecules is one of the actual challenge in biology with the ultimate objective of understanding their function. So far, X-ray crystallography is the most popular method to solve structure, but this technique relies on the generation of diffracting crystals. Once a correct data set has been obtained, the calculation of electron density maps requires to solve the so-called phase problem  using different approaches. The most frequently used technique is molecular replacement, which relies on the availability of the structure of a protein sharing strong structural similarity with the studied protein. Its success rate is directly correlated with the quality of the models used for the molecular replacement trials. The availability of models as accurate as possible is then definitely critical.</p><p>Very recently, a breakthrough step has been made in the field of protein structure prediction thanks to the use of machine learning approaches as implemented in the AlphaFold or RoseTTAFold structure prediction programs. Here, we describe how these recent improvements helped us to solve the crystal structure of a protein involved in the nonsense-mediated mRNA decay pathway (NMD), an mRNA quality control pathway dedicated to the elimination of eukaryotic mRNAs harboring premature stop codons.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472800', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'The MyLo CRISPR-Cas9 Toolkit: A Markerless Yeast Localization and Overexpression CRISPR-Cas9 Toolkit', 'authors': \"[{'name': 'Bjorn DM Bean'}, {'name': 'Malcolm Whiteway'}, {'name': 'Vincent JJ Martin'}]\", 'abstract': '<p>The genetic tractability of the yeast <italic>Saccharomyces cerevisiae</italic> has made it a key model organism for basic research and a target for metabolic engineering. To streamline the introduction of tagged genes and compartmental markers with powerful CRISPR-Cas9-based genome editing tools we constructed a Markerless Yeast Localization and Overexpression (MyLO) CRISPR-Cas9 Toolkit with three components: (i) a set of optimized <italic>S. pyogenes</italic> Cas9-guide RNA (gRNA) expression vectors with five selectable markers and the option to either pre-clone or co-transform the gRNAs; (ii) vectors for the one-step construction of integration cassettes expressing an untagged or GFP/RFP/HA-tagged gene of interest at one of three levels, supporting localization and overexpression studies; and (iii) integration cassettes containing moderately expressed GFP- or RFP-tagged compartmental markers for colocalization experiments. These components allow rapid, high efficiency genomic integrations and modifications with only transient selection for the Cas9 vector, resulting in markerless transformations. Thus, the MyLO toolkit packages CRISPR-Cas9 technology into a flexible, optimized bundle to simplify yeast research</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472810', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Interferon- Exposure of Human iPSC-derived Neurons Alters Major Histocompatibility Complex I and Synapsin I Protein Expression', 'authors': \"[{'name': 'Adam Pavelinek'}, {'name': 'Rugile Matuleviciute'}, {'name': 'Laura Sichlinger'}, {'name': 'Lucia Dutan Polit'}, {'name': 'Nikos Armeniakos'}, {'name': 'Anthony C Vernon'}, {'name': 'Deepak P Srivastava'}]\", 'abstract': '<p>Human epidemiological data links maternal immune activation during gestation with increased risk for neurodevelopmental disorders including schizophrenia. Animal models of maternal immune activation (MIA) provide causal evidence for this association and strongly suggest that inflammatory cytokines act is a critical link between maternal infection and aberrant offspring brain and behavior development. This includes evidence for reduced synapse formation, consistent with post-mortem and in vivo evidence of reduced synaptic density in schizophrenia. However, to what extent specific cytokines are necessary and sufficient for these effects remains unclear. Using a human cellular model, we recently demonstrated that acute exposure to interferon- (IFN) recapitulates molecular and cellular phenotypes associated with neurodevelopmental disorders. Here, we extend this work to test whether IFN affects synapse formation in an induced neuron model that generates forebrain glutamatergic neurons. Using immunocytochemistry and quantitative PCR, we demonstrate that acute IFN exposure results in significantly increased MHCI expression at the mRNA and protein level. Furthermore, acute IFN exposure decreases synapsin I protein in neurons but does not affect synaptic gene mRNA levels. Interestingly, complement component 4A (C4A) mRNA is also significantly increased following acute IFN exposure. This study builds on our previous work by showing that IFN-mediated disruption of relevant synaptic proteins can occur at early stages of synapse formation, potentially contributing to neurodevelopmental disorder phenotypes such as schizophrenia.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.17.473118', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Population coding and oscillatory subspace synchronization integrate context into actions', 'authors': \"[{'name': 'Jan Weber'}, {'name': 'Anne-Kristin Solbakk'}, {'name': 'Alejandro Blenkmann'}, {'name': 'Anais Llorens'}, {'name': 'Ingrid Funderud'}, {'name': 'Sabine Leske'}, {'name': 'Pal Gunnar Larsson'}, {'name': 'Jugoslav Ivanovic'}, {'name': 'Robert Thomas Knight'}, {'name': 'Tor Endestad'}, {'name': 'Randolph Helfrich'}]\", 'abstract': '<p>Contextual cues and prior evidence guide human goal-directed behavior. To date, the neurophysiological mechanisms that implement contextual priors to guide subsequent actions remain unclear. Here we demonstrate that increasing behavioral uncertainty introduces a shift from an oscillatory to a continuous processing mode in human prefrontal cortex. At the population level, we found that oscillatory and continuous dynamics reflect dissociable signatures that support distinct aspects of encoding, transmission and execution of context-dependent action plans. We show that prefrontal population activity encodes predictive context and action plans in serially unfolding orthogonal subspaces, while prefrontal-motor theta oscillations synchronize action-encoding population subspaces to mediate the hand-off of action plans. Collectively, our results reveal how two key features of large-scale population activity, namely continuous population trajectories and oscillatory synchrony, operate in concert to guide context-dependent human behavior.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.473088', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Structural, functional and computational studies of membrane recognition by Plasmodium Perforin-Like Proteins 1 and 2.', 'authors': \"[{'name': 'Sophie Williams'}, {'name': 'Xiulian Yu'}, {'name': 'Tao Ni'}, {'name': 'Robert Gilbert'}, {'name': 'Phillip Stansfeld'}]\", 'abstract': '<p>Perforin-like proteins (PLPs) play key roles in the mechanisms associated with parasitic disease caused by apicomplexans such as  Plasmodium  (malaria) and Toxoplasma. The  T. gondii  PLP1 (TgPLP1) mediates tachyzoite egress from cells, while the five  Plasmodium  PLPs carry out various roles in the life cycle of the parasite and with respect to the molecular basis of disease. Here we focus on  Plasmodium vivax  PLP1 and PLP2 (PvPLP1 and PvPLP2) compared to TgPLP1; PvPLP1 is important for invasion of mammalian hosts by the parasite and establishment of a chronic infection, PvPLP2 is important during the symptomatic blood stage of the parasite life cycle. Determination of the crystal structure of the membrane-binding APC domain of PvPLP1 reveals notable differences with that of TgPLP1, which are reflected in its inability to bind lipid bilayers in the way that TgPLP1 and PvPLP2 can be shown to. Molecular dynamics simulations combined with site-directed mutagenesis and functional assays allow a dissection of the binding interactions of TgPLP1 and PvPLP2 on lipid bilayers, and reveal a similar tropism for lipids found enriched in the inner leaflet of the mammalian plasma membrane. In addition to this shared mode of membrane binding PvPLP2 displays a secondary synergistic interaction side-on from its principal bilayer interface. This study underlines the substantial differences between the biophysical properties of the APC domains of Apicomplexan PLPs, which reflect their significant sequence diversity. Such differences will be important factors in determining the cell targeting and membrane-binding activity of the different proteins, in their different developmental roles within parasite life cycles.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.472476', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Potential cross-protection against SARS-CoV-2 from previous exposure to bovine coronavirus', 'authors': \"[{'name': 'Lana Bazan Peters Querne'}, {'name': 'Fernanda Zettel Bastos'}, {'name': 'Breno Castello Branco Beiro'}]\", 'abstract': '<p>Humans have long shared infectious agents with cattle, and the common cold OC-43 CoV is a not-so-distant example of cross-species viral spillover. Human exposure to BCoV is certainly common, as the virus is endemic in cattle-raising regions. This article shows an <italic>in silico</italic> investigation of shared viral epitopes between BCoV and SARS-CoV-2. HLA recognition and lymphocyte reactivity were assessed using freely-available resources. Several epitopes were shared between BCoV and SARS-CoV-2, both for B and T lymphocytes. These data demonstrate that possible cross-protection is being induced by human exposure to cattle.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472322', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Spatially distributed computation in cortical circuits', 'authors': \"[{'name': 'Sergei Gepshtein'}, {'name': 'Ambarish Pawar'}, {'name': 'Sunwoo Kwon'}, {'name': 'Sergey Savelev'}, {'name': 'Thomas D. Albright'}]\", 'abstract': '<p>The traditional view of neural computation in the cerebral cortex holds that sensory neurons are specialized, i.e., selective for certain dimensions of sensory stimuli. This view was challenged by evidence of contextual interactions between stimulus dimensions in which a neurons response to one dimension strongly depends on other dimensions. Here we use methods of mathematical modeling, psychophysics, and electrophysiology to address shortcomings of the traditional view. Using a model of a generic cortical circuit, we begin with the simple demonstration that cortical responses are always distributed among neurons, forming characteristic waveforms, which we call neural waves. When stimulated by patterned stimuli, circuit responses arise by interference of neural waves. Resulting patterns of interference depend on interaction between stimulus dimensions. Comparison of these modeled responses with responses of biological vision makes it clear that the framework of neural wave interference provides a useful alternative to the standard concept of neural computation.</p><sec><title>One-sentence summary</title><p>Investigating interference of neural waves helps to overcome limitations of the traditional view of cortical computation.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.10.21267574', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Safety and immunogenicity of a heterologous booster of protein subunit vaccine MVC-COV1901 after two doses of adenoviral vector vaccine AZD1222', 'authors': \"[{'name': 'Shu-Hsing Cheng'}, {'name': 'Yi-Chun Lin'}, {'name': 'Cheng-Pin Chen'}, {'name': 'Chien-Yu Cheng'}]\", 'abstract': '<p>We report the interim safety and immunogenicity results in participants administrated with a booster dose of protein subunit vaccine MVC-COV1901 at 12 or 24 weeks after two doses of AZD1222 (ChAdOx1 nCoV-19). In subjects fully vaccinated with two doses of AZD1222, waning antibody immunity was apparent within six months of the second dose of AZD1222. At one month after the MVC-COV1901 booster dose, anti-SARS-CoV-2 spike IgG antibody titers and neutralizing antibody titers were 14- and 8.6-fold increased, respectively, when compared to the titer levels on the day of the booster dose. These interim results support the use of MVC-COV1901 as a heterologous booster for individuals vaccinated with AZD1222.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.15.472764', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Bacterial origin of thymidylate and folate metabolism in Asgard Archaea', 'authors': \"[{'name': 'Jonathan File'}, {'name': 'Hubert F. Becker'}, {'name': 'Lucille Mellottee'}, {'name': 'Zhihui Li'}, {'name': 'Jean-Christophe Lambry'}, {'name': 'Ursula Liebl'}, {'name': 'Hannu Myllykallio'}]\", 'abstract': '<p>Little is known about the evolution and biosynthetic function of DNA precursor and the folate metabolism in the Asgard group of archaea. As Asgard occupy a key position in the archaeal and eukaryotic phylogenetic trees, we have exploited very recently emerged genome and metagenome sequence information to investigate these central metabolic pathways. Our genome-wide analyses revealed that the recently cultured Asgard archaeon <italic>Candidatus</italic> Prometheoarchaeum syntrophicum strain MK-D1 (<italic>Psyn</italic>) contains a complete folate-dependent network for the biosynthesis of DNA/RNA precursors, amino acids and syntrophic amino acid utilization. Altogether our experimental and computational data suggest that phylogenetic incongruences of functional folate-dependent enzymes from Asgard archaea reflect their persistent horizontal transmission from various bacterial groups, which has rewired the key metabolic reactions in an important and recently identified archaeal phylogenetic group. We also experimentally validated the functionality of the lateral gene transfer of <italic>Psyn</italic> thymidylate synthase ThyX. This enzyme uses bacterial-like folates efficiently and is inhibited by mycobacterial ThyX inhibitors. Our data raise the possibility that the thymidylate metabolism, required for <italic>de novo</italic> DNA synthesis, originated in bacteria and has been independently transferred to archaea and eukaryotes. In conclusion, our study has revealed that recent prevalent lateral gene transfer has markedly shaped the evolution of Asgard archaea by allowing them to adapt to specific ecological niches.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472618', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Generating Electricity from Water Evaporation Through Microbial Biofilms', 'authors': \"[{'name': 'Xiaomeng Liu'}, {'name': 'Toshiyuki Ueki'}, {'name': 'Hongyan Gao'}, {'name': 'Trevor L. Woodard'}, {'name': 'Kelly P. Nevin'}, {'name': 'Tianda Fu'}, {'name': 'Shuai Fu'}, {'name': 'Lu Sun'}, {'name': 'Derek R. Lovley'}, {'name': 'Jun Yao'}]\", 'abstract': '<p>Sustainable strategies for energy production are required to reduce reliance on fossil fuels and to power electronics without generating toxic waste.<sup>1-7</sup> Generating electricity from water evaporation through engineered materials is a promising approach,<sup>8,9</sup> but power outputs have been low and the materials employed were not sustainably produced. Microorganisms can be mass produced with renewable feedstocks. Here, we demonstrate that it is possible to engineer microbial biofilms as a cohesive, flexible material for long-term continuous electricity production from evaporating water. The biofilm sheets were the functional component in devices that continuously produced power densities (1 W/cm<sup>2</sup>) higher than that achieved with non-biological materials. Current production scaled directly with biofilm-sheet size and skin-patch devices harvested sufficient electricity from the moisture on skin to continuously power wearable devices. The results demonstrate that appropriately engineered biofilms can perform as robust functional materials without the need for further processing or maintaining cell viability. Biofilm-based hydroelectric current production was comparable to that achieved with similar sized biofilms catalyzing current production in microbial fuel cells,<sup>10,11</sup> without the need for an organic feedstock or maintaining cell viability. The ubiquity of biofilms in nature suggests the possibility of additional sources of biomaterial for evaporation-based electricity generation and the possibility of harvesting electricity from diverse aqueous environments.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.21267732', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Health Policy', 'covid': True, 'title': 'Modelling the recovery of elective waiting lists following COVID-19: scenario projections for England', 'authors': \"[{'name': 'Nicholas C Howlett'}, {'name': 'Richard M Wood'}]\", 'abstract': '<sec><title>Background</title><p>A significant indirect impact of COVID-19 has been the increasing elective waiting times observed in many countries. In Englands National Health Service, the waiting list has grown from 4.4 million in February 2020 to 5.7m by August 2021.</p></sec><sec><title>Aims</title><p>The objective of this study was to estimate the trajectory of future waiting list size and waiting times to December 2025.</p></sec><sec><title>Methods</title><p>A scenario analysis was performed using computer simulation and publicly available data as of November 2021. Future demand assumed a phased return of various proportions (0, 25, 50 and 75%) of the estimated 7.1 million referrals missed during the pandemic. Future capacity assumed 90, 100 and 110% of that provided in the 12 months immediately before the pandemic.</p></sec><sec><title>Results</title><p>As a worst case, the waiting list would reach 13.6m (95% CI: 12.4m to 15.6m) by Autumn 2022, if 75% of missed referrals returned and only 90% of pre pandemic capacity could be achieved. Under this scenario, the proportion of patients waiting under 18 weeks would reduce from 67.6% in August 2021 to 42.2% (37.4% to 46.2%) with the number waiting over 52 weeks reaching 1.6m (0.8m to 3.1m) by Summer 2023. At this time, 29.0% (21.3% to 36.8%) of patients would be leaving the waiting list before treatment. Waiting lists would remain pressured under even the most optimistic of scenarios considered, with 18-week performance struggling to maintain 60% (against the 92% constitutional target).</p></sec><sec><title>Conclusions</title><p>This study reveals the long-term challenge for the NHS in recovering elective waiting lists as well as potential implications for patient outcomes and experience.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.21267549', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'The impact of environmental mycobiomes on geographic variation in COVID-19 mortality', 'authors': \"[{'name': 'Joshua Ladau'}, {'name': 'Katrina Abuabara'}, {'name': 'Angelica M. Walker'}, {'name': 'Marcin P. Joachimiak'}, {'name': 'Ishan Bansal'}, {'name': 'Yulun Wu'}, {'name': 'Elijah B. Hoffman'}, {'name': 'Chaincy Kuo'}, {'name': 'Nicola Falco'}, {'name': 'Jared Streich'}, {'name': 'Mark J. van der Laan'}, {'name': 'Haruko M. Wainwright'}, {'name': 'Eoin L. Brodie'}, {'name': 'Matthias Hess'}, {'name': 'Daniel Jacobson'}, {'name': 'James B. Brown'}]\", 'abstract': '<p>Mortality rates during the COVID-19 pandemic have varied by orders of magnitude across communities in the United States<sup>1</sup>. Individual, socioeconomic, and environmental factors have been linked to health outcomes of COVID-19<sup>2,3,4,5</sup>. It is now widely appreciated that the environmental microbiome, composed of microbial communities associated with soil, water, atmosphere, and the built environment, impacts immune system development and susceptibility to immune-mediated disease<sup>6,7,8</sup>. The human microbiome has been linked to individual COVID-19 disease outcomes<sup>9</sup>, but there are limited data on the influence of the environmental microbiome on geographic variation in COVID-19 across populations<sup>10</sup>. To fill this knowledge gap, we used taxonomic profiles of fungal communities associated with 1,135 homes in 494 counties from across the United States in a machine learning analysis to predict COVID-19 Infection Fatality Ratios (the number of deaths caused by COVID-19 per 1000 SARS-CoV-2 infections<sup>1</sup>; IFR). Here we show that increased fungal diversity, and in particular indoor exposure to outdoor fungi, is associated with reduced SARS-CoV-2 IFR. Further, we identify seven fungal genera that are the predominant drivers of this protective signal and may play a role in suppressing COVID-19 mortality. This relationship is strongest in counties where human populations have remained stable over at least the previous decade, consistent with the importance of early-life microbial exposures<sup>11</sup>. We also assessed the explanatory power of 754 other environmental and socioeconomic factors, and found that indoor-outdoor fungal beta-diversity is amongst the strongest predictors of county-level IFR, on par with the most important known COVID-19 risk factors, including age<sup>12</sup>. We anticipate that our study will be a starting point for further integration of environmental mycobiome data with population health information, providing an important missing link in our capacity to identify vulnerable populations. Ultimately, our identification of specific genera predicted to be protective against COVID-19 mortality may point toward novel, proactive therapeutic approaches to infectious disease.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.473092', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'p166 links membrane and intramitochondrial modules of the trypanosomal tripartite attachment complex', 'authors': \"[{'name': 'Bernd Schimanski'}, {'name': 'Salome Aeschlimann'}, {'name': 'Sandro Kser'}, {'name': 'Maria Gomez-Fabra Gala'}, {'name': 'Nora Vgtle'}, {'name': 'Andr Schneider'}]\", 'abstract': '<p>The protist parasite Trypanosoma brucei has a single mitochondrion with a single unit genome termed kinetoplast DNA (kDNA). Faithfull segregation of replicated kDNA is ensured by a complicated structure termed tripartite attachment complex (TAC). The TAC physically links the basal body of the flagellum with the kDNA spanning the two mitochondrial membranes. Here, we characterized p166 as the only TAC subunit that is anchored in the inner membrane. Its C-terminal transmembrane domain separates the protein into a large N-terminal region that interacts with the kDNA-localized TAC102 and a 34 aa C-tail that binds to the intermembrane space-exposed loop of the integral outer membrane protein TAC60. Thus, in contrast to the outer membrane TAC region which requires four essential subunits for proper function a single inner membrane TAC subunit is sufficient to bridge the distance from the OM to the kDNA. Surprisingly, non-functional p166 lacking the C-terminal 34 aa still localizes to the TAC region. This suggests the existence of nonessential TAC-associated proteins in the OM. These proteins can loosely bind to non-functional p166 lacking the C-terminal 34 aa and keep it at the TAC but their binding would not be strong enough to withstand the mechanical force upon kDNA segregation.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.472509', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Milking it for all its worth: The effects of environmental enrichment on maternal nurturance, lactation quality, and offspring social behavior', 'authors': \"[{'name': 'Holly DeRosa'}, {'name': 'Hieu Tran'}, {'name': 'Amanda C. Kentner'}]\", 'abstract': '<p>While alterations in the neonatal environment can have dramatic effects on offspring outcomes, the mechanisms that drive this phenomenon remain unclear. Breastfeeding confers robust benefits to offspring development, including those related to growth, immunity, and neurophysiology. Similarly, improving environmental complexity (i.e., environmental enrichment; EE) contributes developmental advantages to both humans and laboratory animal models. However, the impact of environmental context on maternal care and milk quality has not been thoroughly evaluated, nor are the biological underpinnings of EE on offspring development understood. Here, Sprague-Dawley rats were housed and bred in either EE or standard-housed (SD) conditions. Although EE dams gave birth to a larger number of pups, litters were standardized and cross-fostered across groups on postnatal day (P)1. Maternal milk samples were then collected on P1 (transitional milk phase) and P10 (mature milk phase) for analysis. While EE dams spent less time nursing, postnatal enrichment exposure was associated with heavier offspring bodyweights. Moreover, milk from EE dams had increased triglyceride levels compared to SD dams. Milk from EE mothers also contained a greater microbiome diversity and a significantly higher abundance of bacterial families related to bodyweight and energy metabolism. In addition to changes in lactational quality, we observed sex- and time-dependent effects of EE on offspring social behavior. Specifically, prenatal EE exposure was associated with greater sociability in females, while postnatal EE was associated with greater sociability in male offspring. Together, these results underscore the multidimensional impact of the combined neonatal and maternal environments on offspring development. These data also identify potential deficiencies in the quality of the gold standard laboratory housing condition and its impact on the welfare and design of translationally relevant animal models in biomedical research.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.17.21267993', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Predictive value of isolated symptoms for diagnosis of SARS-CoV-2 infection in children tested during peak circulation of the delta variant', 'authors': \"[{'name': 'Adrianna L. Westbrook'}, {'name': 'Laura C. Benedit'}, {'name': 'Jennifer K. Frediani'}, {'name': 'Mark A. Griffiths'}, {'name': 'Nabeel Y. Khan'}, {'name': 'Joshua M. Levy'}, {'name': 'Claudia R. Morris'}, {'name': 'Christina A. Rostad'}, {'name': 'Cheryl L. Stone'}, {'name': 'Julie Sullivan'}, {'name': 'Miriam B. Vos'}, {'name': 'Jean Welsh'}, {'name': 'Anna Wood'}, {'name': 'Greg S. Martin'}, {'name': 'Wilbur Lam'}, {'name': 'Nira R. Pollock'}]\", 'abstract': '<p>Background: COVID-19 testing policies for symptomatic children attending U.S. schools or daycare vary, and whether isolated symptoms should prompt testing is unclear. We evaluated children presenting for SARS-CoV-2 testing to determine if the likelihood of having a positive SARS-CoV-2 test differed between participants with one versus  2 symptoms, and to examine the predictive capability of isolated symptoms. \\nMethods: Participants  18 years presenting for clinical SARS-CoV-2 molecular testing in six sites in urban/suburban/rural Georgia (July-October, 2021; delta variant predominant) were queried about individual symptoms. Participants were classified into three groups: asymptomatic, one symptom only, or  2 symptoms. SARS-CoV-2 test results and clinical characteristics of the three groups were compared. Sensitivity, specificity, and positive/negative predictive values (PPV/NPV) for isolated symptoms were calculated by fitting a saturated Poisson model. \\nResults: Of 602 participants, 21.8% tested positive and 48.7% had a known or suspected close contact. Children reporting one symptom (n=82; OR=6.00, 95% CI: 2.70-13.33) and children reporting  2 symptoms (n=365; OR=5.25: 2.66-10.38) were significantly more likely to have a positive COVID-19 test than asymptomatic children (n=155), but they were not significantly different from each other (OR=0.88: 0.52-1.49). Sensitivity/PPV were highest for isolated fever (33%/57%), cough (25%/32%), and sore throat (21%/45%); headache had low sensitivity (8%) but higher PPV (33%). Sensitivity/PPV of isolated congestion/rhinorrhea were 8%/9%. \\nConclusions:  With high delta variant prevalence, children with isolated symptoms were as likely as those with multiple symptoms to test positive for COVID-19. Isolated fever, cough, sore throat, or headache, but not congestion/rhinorrhea, offered highest predictive value.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.472185', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'learnMET: an R package to apply machine learning methods for genomic prediction using multi-environment trial data', 'authors': \"[{'name': 'Cathy C. Westhues'}, {'name': 'Henner Simianer'}, {'name': 'Timothy M. Beissinger'}]\", 'abstract': '<p>We introduce the R-package <italic>learnMET</italic>, developed as a flexible framework to enable a collection of analyses on multi-environment trial (MET) breeding data with machine learning-based models. <italic>learnMET</italic> allows the combination of genomic information with environmental data such as climate and/or soil characteristics. Notably, the package offers the possibility of incorporating weather data from field weather stations, or can retrieve global meteorological datasets from a NASA database. Daily weather data can be aggregated over specific periods of time based on naive (for instance, non-overlapping 10-day windows) or phenological approaches. Different machine learning methods for genomic prediction are implemented, including gradient boosted trees, random forests, stacked ensemble models, and multi-layer perceptrons. These prediction models can be evaluated via a collection of cross-validation schemes that mimic typical scenarios encountered by plant breeders working with MET experimental data in a user-friendly way. The package is fully open source and accessible on GitHub.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.17.472861', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'OPEN leaf: an open-source cloud-based phenotyping system for tracking dynamic changes at leaf-specific resolution in Arabidopsis', 'authors': \"[{'name': 'Landon Gary Alan Swartz'}, {'name': 'Suxing Liu'}, {'name': 'Drew Dahlquist'}, {'name': 'Emily S Walter'}, {'name': 'Skyler Kramer'}, {'name': 'Sam McInturf'}, {'name': 'Alexander Bucksch'}, {'name': 'David Mendoza-Cozatl'}]\", 'abstract': '<p>The first draft of the Arabidopsis genome was released more than 20 years ago and despite intensive molecular research, more than 30% of Arabidopsis genes remained uncharacterized or without an assigned function. This is in part due to gene redundancy within gene families or the essential nature of genes, where their deletion results in lethality (i.e., the dark genome). High-throughput plant phenotyping (HTPP) offers an automated and unbiased approach to characterize subtle or transient phenotypes resulting from gene redundancy or inducible gene silencing; however, commercial HTPP platforms remain unaffordable. Here we describe the design and implementation of OPEN leaf, an open-source HTPP system with cloud connectivity and remote bilateral communication to facilitate data collection, sharing and processing. OPEN leaf, coupled with the SMART imaging processing package was able to consistently document and quantify dynamic morphological changes over time at the whole rosette level and also at leaf-specific resolution when plants experienced changes in nutrient availability. The modular design of OPEN leaf allows for additional sensor integration. Notably, our data demonstrate that VIS sensors remain underutilized and can be used in high-throughput screens to identify characterize previously unidentified phenotypes in a leaf-specific manner.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.473000', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Characterizing the variation in chromosome structure ensembles in the context of the nuclear microenvironment', 'authors': \"[{'name': 'Priyojit Das'}, {'name': 'Tongye Shen'}, {'name': 'Rachel Patton McCord'}]\", 'abstract': '<p>Inside the nucleus, chromosomes are subjected to direct physical interaction between different components, active forces, and thermal noise, leading to the formation of an ensemble of three-dimensional structures. However, it is still not well understood to what extent and how the structural ensemble varies from one chromosome region or cell-type to another. We designed a statistical analysis technique and applied it to single-cell chromosome imaging data to reveal the fluctuation of individual chromosome structures. By analyzing the resulting structural landscape, we find that the largest dynamic variation is the overall radius of gyration of the chromatin region, followed by domain reorganization within the region. By comparing different human cell-lines and experimental perturbations data using this statistical analysis technique and a network entropy approach, we identify both cell-type and condition-specific features of the structural landscapes. We identify a relationship between epigenetic state and the properties of chromosome structure fluctuation and validate this relationship through polymer simulations. Overall, our study suggests that the types of variation in a chromosome structure ensemble are cell-type as well as region-specific and can be attributed to constraints placed on the structure by factors such as variation in epigenetic state.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.15.472744', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Autophagy-related protein Atg11 preserves asymmetric inheritance of spindle microtubule-organizing centers', 'authors': \"[{'name': 'Md. Hashim Reza'}, {'name': 'Jigyasa Verma'}, {'name': 'Ratul Chowdhury'}, {'name': 'Ravi Manjithaya'}, {'name': 'Kaustuv Sanyal'}]\", 'abstract': '<p>Asymmetric spindle pole body (SPB) inheritance requires a cascade of events that involve kinases, phosphatases and structural scaffold proteins including molecular motors and microtubule-associated proteins present in the nucleus and/or the cytoplasm. Higher levels of an SPB component Spc72 and the spindle positioning factor Kar9 at the old SPB, which migrates to the daughter cell, ensure asymmetric SPB inheritance. Timely SPB duplication followed by its asymmetric inheritance is a key to correct spindle alignment leading to high-fidelity chromosome segregation. By combining <italic>in silico</italic> analysis of known protein-protein interactions of autophagy (Atg)-related proteins with those that constitute the chromosome segregation machinery, and growth dynamics of 35 <italic>atg</italic> mutants in the presence of a microtubule poison, we identified Atg11 as a potential regulator of chromosome transmission. Cells lacking Atg11 did not show any kinetochore defects but displayed a high rate of chromosome loss and delayed anaphase onset. Atg11 positively interacted with Kar9 and Kip2 and negatively with Dyn1 and Kar3 in mediating proper chromosome segregation suggesting a role of Atg11 in spindle positioning. Indeed, <italic>atg11</italic> cells displayed an inverted SPB inheritance. We further show that Atg11 promotes asymmetric localization of Spc72 and Kar9 on the old SPB. Atg11 physically interacted with Spc72 and transiently localized close to the old SPB during metaphase-to-anaphase progression. Taken together, our study uncovers an autophagy-independent role of Atg11 in spindle alignment and emphasizes the importance of unbiased screens to identify factors mediating the complex and intricate crosstalk between processes fundamental to genomic integrity.</p><sec><title>Significance statement</title><p>Dysregulation in spindle alignment during cell division causes decreased lifespan and contributes to diseased states related to aneuploidy such as cancer. The alignment of the mitotic spindle along the mother-bud axis in the budding yeast relies on the differential distribution of proteins between existing old and freshly formed new spindle pole bodies (SPBs). Non-random distribution of Spc72 and Kar9 facilitates the asymmetric inheritance of SPBs, vital for proper spindle elongation and alignment. Here, we identified a non-canonical role of an autophagy-related protein Atg11 in the Kar9-mediated spindle positioning and faithful chromosome segregation. Loss of Atg11 leads to randomized Spc72 distribution. Atg11, like other autophagy proteins, is evolutionarily conserved. Thus, this study reiterates the importance of crosstalk among major biological processes.</p></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472554', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'Relationship between monomer packing and receptor binding domain pocket status in the spike trimer of SARS-CoV-2 variants', 'authors': \"[{'name': 'Jim Warwicker'}]\", 'abstract': '<p>Existence of a SARS-CoV-2 spike protein trimer form with closer packing between monomers when receptor binding domains (RBDs) are all down, locked as opposed to closed, has been associated with linoleic acid (LA) binding at neutral pH, or can occur at acidic pH in the absence of LA binding.  The relationship between degree of closure of the LA binding pocket of the RBD, and monomer burial in the trimer, is examined for a range of spike protein structures, including those with D614G mutation, and that of the Delta variant (which also carries D614G).  Some spike protein structures with this aspartic acid mutation show monomer packing approaching that of the locked form (at neutral pH, without LA binding) for two segments, a third (around the RBD) remains less closely packed.  Analysis of other coronavirus RBD structures suggests that mutation of the RBD in spike protein of the Omicron variant could lead to LA binding pocket changes.  It is proposed that these changes could lead to one of two consequences for the Omicron variant spike protein (which also has the D614G mutation), at neutral pH and without LA binding, either easier access to a locked form throughout that leads to cooperative transitions between all RBD down and all RBD up, or maintenance of a spike trimer with locked characteristics C-terminal to the RBD at the same time as the RBD is free to transit between down and up states.  The situation may also be impacted by spike protein charge mutations in the Omicron lineage that alter pH-dependence around the RBD, in a similar way to the changes induced elsewhere by D614G.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472278', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Using polyacrylamide hydrogels to model physiological aortic stiffness reveals that microtubules are critical regulators of isolated smooth muscle cell morphology and contractility', 'authors': \"[{'name': 'Sultan Ahmed'}, {'name': 'Robert. T. Johnson'}, {'name': 'Reesha Solanki'}, {'name': 'Teclino Afewerki'}, {'name': 'Finn Wostear'}, {'name': 'Derek T. Warren'}]\", 'abstract': '<p>Vascular smooth muscle cells (VSMCs) are the predominant cell type in the medial layer of the aortic wall and normally exist in a quiescent, contractile phenotype where actomyosin-derived contractile forces maintain vascular tone. However, VSMCs are not terminally differentiated and can dedifferentiate into a proliferative, synthetic phenotype. Actomyosin force generation is essential for the function of both phenotypes. Whilst much is already known about the mechanisms of VSMC actomyosin force generation, existing assays are either low throughput and time consuming, or qualitative and inconsistent. In this study, we use polyacrylamide hydrogels, tuned to mimic the physiological stiffness of the aortic wall, in a VSMC contractility assay. Isolated VSMC area decreases following stimulation with the contractile agonists angiotensin II or carbachol. Importantly, the angiotensin II induced reduction in cell area correlated with increased traction stress generation. Inhibition of actomyosin activity using blebbistatin or Y-27632 prevented angiotensin II mediated changes in VSMC morphology, suggesting that changes in VSMC morphology and actomyosin activity are core components of the contractile response. Furthermore, we show that microtubule stability is an essential regulator of isolated VSMC contractility. Treatment with either colchicine or paclitaxel uncoupled the morphological and/or traction stress responses of angiotensin II stimulated VSMCs. Our findings support the tensegrity model and we demonstrate that microtubules act to balance the actomyosin-derived traction stress generation and regulate the morphological responses of VSMCs.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472759', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'GiPS: Genomics-informed parent selection uncovers the breeding value of wheat genetic resources', 'authors': \"[{'name': 'Albert W. Schulthess'}, {'name': 'Sandip M. Kale'}, {'name': 'Fang Liu'}, {'name': 'Yusheng Zhao'}, {'name': 'Norman Philipp'}, {'name': 'Maximilian Rembe'}, {'name': 'Yong Jiang'}, {'name': 'Ulrike Beukert'}, {'name': 'Albrecht Serfling'}, {'name': 'Axel Himmelbach'}, {'name': 'Jrg Fuchs'}, {'name': 'Markus Oppermann'}, {'name': 'Stephan Weise'}, {'name': 'Philipp H. G. Boeven'}, {'name': 'Johannes Schacht'}, {'name': 'C. Friedrich H. Longin'}, {'name': 'Sonja Kollers'}, {'name': 'Nina Pfeiffer'}, {'name': 'Viktor Korzun'}, {'name': 'Matthias Lange'}, {'name': 'Uwe Scholz'}, {'name': 'Nils Stein'}, {'name': 'Martin Mascher'}, {'name': 'Jochen C. Reif'}]\", 'abstract': '<p>The great efforts spent in the maintenance of past diversity in genebanks are rationalized by the potential role of plant genetic resources in future crop improvement  a concept whose practical implementation has fallen short of expectations. Here, we implement genomics-informed parent selection to expedite pre-breeding without discriminating against non-adapted germplasm. We collect dense genetic profiles for a large winter wheat collection and evaluate grain yield and resistance to yellow rust in representative coresets. Genomic prediction within and across genebanks identified the best parents for PGR x elite derived crosses that outyielded current elite cultivars in multiple field trials.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472983', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': \"Low Frequency Oscillations drive EEG's complexity changes during wakefulness and sleep\", 'authors': \"[{'name': 'Joaquin Gonzalez'}, {'name': 'Diego M. Mateos'}, {'name': 'Matias Cavelli'}, {'name': 'Alejandra Mondino'}, {'name': 'Claudia Pascovich'}, {'name': 'Pablo Torterolo'}, {'name': 'Nicolas Rubido'}]\", 'abstract': \"<p>Recently, the sleep-wake states have been analysed using novel complexity measures, complementing the classical analysis of EEGs by frequency bands. This new approach consistently shows a decrease in EEG's complexity during slow-wave sleep, yet it is unclear how cortical oscillations shape these complexity variations. In this work, we analyse how the frequency content of brain signals affects the complexity estimates in freely moving rats. We find that the low-frequency spectrum - including the Delta, Theta, and Sigma frequency bands - drives the complexity changes during the sleep-wake states. This happens because low-frequency oscillations emerge from neuronal population patterns, as we show by recovering the complexity variations during the sleep-wake cycle from micro, meso, and macroscopic recordings. Moreover, we find that the lower frequencies reveal synchronisation patterns across the neocortex, such as a sensory-motor decoupling that happens during REM sleep. Overall, our works shows that EEG's low frequencies are critical in shaping the sleep-wake states' complexity across cortical scales.</p>\", 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.473019', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evolutionary traits of Tick-borne encephalitis virus: Pervasive non-coding RNA structure conservation and molecular epidemiology', 'authors': \"[{'name': 'Lena S. Kutschera'}, {'name': 'Michael T. Wolfinger'}]\", 'abstract': \"<p>Tick-borne encephalitis virus (TBEV) is the etiological agent of tick-borne encephalitis, an infectious disease of the central nervous system that is often associated with severe sequelae in humans. While TBEV is typically classified into three subtypes, recent evidence suggests a more varied range of TBEV subtypes and lineages that differ substantially in their 3'UTR architecture. Building on comparative genomics approaches and thermodynamic modeling, we characterize the TBEV 3'UTR structureome diversity and propose a unified picture of pervasive non-coding RNA (ncRNA) structure conservation. Moreover, we provide an updated phylogeny of TBEV, building on more than 220 publicly available complete genomes, and investigate the molecular epidemiology and phylodynamics with Nextstrain, a web-based visualization framework for real-time pathogen evolution.</p>\", 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.472951', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The costs and benefits of dispersal in small populations', 'authors': \"[{'name': 'Jitka Polechova'}]\", 'abstract': '<p>Dispersal has three major effects on adaptation. First, the gene flow mixes alleles adapted to different environments, potentially hindering (swamping) adaptation. Second, it inflates genetic variance: this aids adaptation to spatially (and temporally) varying environments but if selection is hard, it lowers the mean fitness of the population. Third, neighbourhood size, which determines how weak genetic drift is, increases with dispersal -- when genetic drift is strong, increase of neighbourhood size with dispersal aids adaptation. In this note I focus on the role of dispersal in environments which change smoothly across space, and when local populations are quite small such that genetic drift has a significant effect. Using individual-based simulations, I show that in small populations, even leptokurtic dispersal benefits adaptation, by reducing the power of genetic drift. This has implications for management of small marginal populations: increased gene flow appears beneficial as long as adaptations involves a quantitative, rather than a discrete, trait. However, heavily leptokurtic dispersal will swamp continuous adaptation along steep environmental gradients so that only patches of locally adapted subpopulations remain.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.12.472307', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Scalable batch-correction approach for integrating large-scale single-cell transcriptomes', 'authors': \"[{'name': 'Xilin Shen'}, {'name': 'Hongru Shen'}, {'name': 'Dan Wu'}, {'name': 'Mengyao Feng'}, {'name': 'Jiani Hu'}, {'name': 'Jilei Liu'}, {'name': 'Yichen Yang'}, {'name': 'Meng Yang'}, {'name': 'Yang Li'}, {'name': 'Lei Shi'}, {'name': 'Kexin Chen'}, {'name': 'Xiangchun Li'}]\", 'abstract': '<p>Integration of the evolving large-scale single-cell transcriptomes requires scalable batch-correction approaches. Here we propose a simple batch-correction method that is scalable for integrating super large-scale single-cell transcriptomes from diverse sources. The core idea of the method is encoding batch information of each cell as a trainable parameter and added to its expression profile; subsequently, a contrastive learning approach is used to learn feature representation of the additive expression profile. We demonstrate the scalability of the proposed method by integrating 18 million cells obtained from the Human Cell Atlas. Our benchmark comparisons with current state-of-the-art single-cell integration methods demonstrated that our method could achieve comparable data alignment and cluster preservation. Our study would facilitate the integration of super large-scale single-cell transcriptomes. The source code is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/xilinshen/Fugue\">https://github.com/xilinshen/Fugue</ext-link>.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472688', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Mice identify subgoal locations through an action-driven mapping process', 'authors': \"[{'name': 'Philip Shamash'}, {'name': 'Tiago Branco'}]\", 'abstract': \"<p>Mammals instinctively explore and form mental maps of their spatial environments. Models of cognitive mapping in neuroscience mostly depict map-learning as a process of random or biased diffusion. In practice, however, animals explore spaces using structured, purposeful, sensory-guided actions. Here we test the hypothesis that executing specific exploratory actions is a key strategy for building a cognitive map. Previous work has shown that in arenas with obstacles and a shelter, mice spontaneously learn efficient multi-step escape routes by memorizing allocentric subgoal locations. We thus used threat-evoked escape to probe the relationship between ethological exploratory behavior and allocentric spatial memory. Using closed-loop neural manipulations to interrupt running movements during exploration, we found that blocking runs targeting an obstacle edge abolished subgoal learning. In contrast, blocking other movements while sparing edge-directed runs had no effect on memorizing subgoals. Finally, spatial analyses suggest that the decision to use a subgoal during escape takes into account the mouse's starting position relative to the layout of the environment. We conclude that mice use an action-driven learning process to identify subgoals and that these subgoals are then integrated into a map-based planning process. We suggest a conceptual framework for spatial learning that is compatible with the successor representation from reinforcement learning and sensorimotor enactivism from cognitive science.</p>\", 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.472491', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Transcription Factor RFX3 Stabilizes Mammary Basal Cell Identity', 'authors': \"[{'name': 'Erica M. Tross'}, {'name': 'Christian R. de Caestecker'}, {'name': 'Ken S. Lau'}, {'name': 'Ian G. Macara'}]\", 'abstract': '<p>The myoepithelial cell compartment of the murine postnatal mammary gland is generated from basal cap cells in the terminal end bud and maintained by self-renewal. Transdifferentiation to the luminal lineage does not normally occur but can be induced by DNA damage, luminal cell death or transplantation into a recipient mammary fat pad. Myoepithelial cells cultivated in vitro can also transdifferentiate towards the luminal lineage. Little is known about the molecular mechanisms and gene regulatory networks underlying this plasticity. Using a transgenic mouse (Tg11.5kb-GFP) that marks cap cells with GFP, we discovered that mature myoepithelial cells placed in culture begin to express GFP within 24 hrs and later express the Keratin 8 (K8) luminal marker. Cell tracking showed that most K8+ cells arose from GFP+ cells, suggesting that myoepithelial cells de-differentiate towards a progenitor state before changing lineage. Differential gene expression analysis, comparing pure GFP+ cap cells with mature myoepithelial cells, identified multiple transcription factors that iRegulon predicted might regulate the myoepithelial to cap cell transition. Knockout of one of these genes, Regulatory Factor 3 (Rfx3), significantly reduced the population of GFP+ cells and increased differentiation to the K8+ luminal lineage. Rfx3 knockout also reduced mammosphere growth and mammary gland regeneration efficiency in a transplantation assay, but had no effect on proliferation in vitro. Together, these data support a key role for Rfx3 in the stabilization of the mammary basal cell lineages.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472705', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Pseudomonas response regulators produced in an E. coli heterologous expression host exhibit host-derived post-translational phosphorylation', 'authors': \"[{'name': 'Megan E. Garber'}, {'name': 'Rodrigo Fregoso'}, {'name': 'Julie Lake'}, {'name': 'Anne Kakouridis'}, {'name': 'Aindrila Mukhopadhyay'}]\", 'abstract': '<p>In this report, we systematically characterize 32 response regulators (RRs) from a metal tolerant groundwater isolate, <italic>Pseudomonas stutzeri</italic> RCH2 to assess the impact of host-derived post-translational phosphorylation. As observed by distinct shifted bands in a phos-tag gel, 12 of the 24 detected RRs show homogenous mixtures of phosphorylated proteins or heterogenous mixtures of unphosphorylated and phosphorylated proteins. By evaluating the phosphorylation state of CzcR and CopR II under varying assay parameters, we found that changes to pH and exogenous addition of phospho-donors (e.g. acetyl phosphate) have little to no effect on phosphorylation state. By applying protein production conditions that decrease the pool of intracellular acetyl-phosphate in <italic>E. coli</italic>, we found a reduction in the phosphorylated population of CopR II when magnesium was added to the media, but observed no change in phosphorylated population when CopR II is expressed in <italic>E. coli</italic> BL21 (DE3) <italic>pta</italic>, a mutant with a metabolic disruption to the acetyl-phosphate pathway. Therefore, the specific mechanism of post-translational phosphorylation of RRs in <italic>E. coli</italic> remains obscure. These findings show the importance of characterizing the phosphorylations state of proteins when heterologously expressed, since their biochemical and physiological properties are dependent on post-translational modification.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472418', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Modelling biochemical gradients in vitro to control cell compartmentalization in a microengineered 3D model of the intestinal epithelium', 'authors': \"[{'name': 'Gizem Altay'}, {'name': 'Aina Abad-Lzaro'}, {'name': 'Emilio J. Gualda'}, {'name': 'Jordi Folch'}, {'name': 'Claudia Insa'}, {'name': 'Sbastien Tosi'}, {'name': 'Xavier Hernando-Momblona'}, {'name': 'Eduard Batlle'}, {'name': 'Pablo Loza-lvarez'}, {'name': 'Vanesa Fernndez-Majada'}, {'name': 'Elena Martinez'}, {'name': 'Matteo Bernardello'}, {'name': 'Gustavo Castro'}, {'name': 'Jordi Andilla'}]\", 'abstract': '<p>Gradients of signaling pathways within the intestinal stem cell (ISC) niche are instrumental for cellular compartmentalization and tissue function, yet how are they formed and sensed by the epithelium is still not fully understood. Here we present a new in vitro model of the small intestine based on primary epithelial cells (i), apically accessible (ii), with native tissue mechanical properties and controlled mesh size (iii), 3D villus-like architecture (iv), and biomolecular gradients of the ISC niche (v). Biochemical gradients are formed through the hydrogel-based scaffolds by free diffusion from a source to a sink chamber. To confirm the establishment of precise spatiotemporally controlled gradients, we employ light-sheet fluorescence microscopy and in-silico modelling. The ISC niche biochemical gradients applied along the villus axis lead to the in vivo-like compartmentalization of the proliferative and differentiated cells, while changing the composition and concentration of the biochemical factors affects the cellular organization along the villus axis. This novel 3D in vitro intestinal model derived from organoids recapitulates both the villus-like architecture and the gradients of ISC biochemical factors, thus opening the possibility to study in vitro the nature of such gradients and the resulting cellular response.</p><sec><title>Table of contents (ToC)</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"472418v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig><p>Biomolecular gradients are fundamental in intestinal epithelial tissue organization and function. The microengineered platform developed, with well-controlled spatiotemporal gradients of intestinal stem cell niche factors, 3D architecture and organoid-derived cells retrieves an in vivo-like cell compartmentalization. The system is easily incorporated in standard cell culture inserts where one can systematically test factors critical for intestinal tissue dynamics.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.15.470649', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Comparison of extraction methods for intracellular metabolomics', 'authors': \"[{'name': 'Carolin Andresen'}, {'name': 'Tobias Boch'}, {'name': 'Hagen M. Gegner'}, {'name': 'Nils Mechtel'}, {'name': 'Andreas Narr'}, {'name': 'Emrullah Birgin'}, {'name': 'Erik Rasbach'}, {'name': 'Nuh Rahbari'}, {'name': 'Andreas Trumpp'}, {'name': 'Gernot Poschet'}, {'name': 'Daniel Hbschmann'}]\", 'abstract': '<p>Measurements of metabolic compounds inside cells or tissues are of high informative potential since they represent the endpoint of biological information flow and a snapshot of the integration of many regulatory processes. However, it requires careful extraction to quantify their abundance. Here we present a comprehensive study using ten extraction protocols on four human sample types (liver tissue, bone marrow, HL60 and HEK cells) targeting 630 metabolites of different chemical classes. We show that the extraction efficiency and stability is highly variable across protocols and tissues by using different quality metrics including the limit of detection and variability between replicates as well as the sum of concentration as a global estimate of extraction stability. The profile of extracted metabolites depends on the used solvents - an observation which has implications for measurements of different sample types and metabolic compounds of interest. To identify the optimal extraction method for future metabolomics studies, the benchmark dataset was implemented in an easy-to-use, interactive and flexible online resource (R/shiny app MetaboExtract).</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.21267368', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Acute COVID-19 severity and 16-month mental morbidity trajectories in patient populations of six nations', 'authors': \"[{'name': 'Ingibjorg Magnusdottir'}, {'name': 'Aniko Lovik'}, {'name': 'Anna Bara Unnarsdottir'}, {'name': 'Daniel L. McCartney'}, {'name': 'Helga Ask'}, {'name': 'Kadri Koiv'}, {'name': 'Lea Arregui Nordahl Christoffersen'}, {'name': 'Sverre Urnes Johnson'}, {'name': 'Andrew M McIntosh'}, {'name': 'Anna K. Kahler'}, {'name': 'Archie Campbell'}, {'name': 'Arna Hauksdottir'}, {'name': 'Chloe Fawns-Ritchie'}, {'name': 'Christian Erikstrup'}, {'name': 'Dorte Helenius'}, {'name': 'Drew Altschul'}, {'name': 'Edda Bjork Thordardottir'}, {'name': 'Elias Eythorsson'}, {'name': 'Emma M. Frans'}, {'name': 'Gunnar Tomasson'}, {'name': 'Harpa Lind Jonsdottir'}, {'name': 'Harpa Runarsdottir'}, {'name': 'Henrik Hjalgrim'}, {'name': 'Hronn Hardardottir'}, {'name': 'Juan Gonzalez-Hijon'}, {'name': 'Karina Banasik'}, {'name': 'Khoa Manh Dinh'}, {'name': 'Li Lu'}, {'name': 'Lili Milani'}, {'name': 'Lill Trogstad'}, {'name': 'Maria Didriksen'}, {'name': 'Omid V. Ebrahimi'}, {'name': 'Patrick F. Sullivan'}, {'name': 'Per Minor Magnus'}, {'name': 'Qing Shen'}, {'name': 'Ragnar Nesvag'}, {'name': 'Reedik Magi'}, {'name': 'Runolfur Palsson'}, {'name': 'Sisse Rye Ostrowski'}, {'name': 'Thomas Werge'}, {'name': 'Asle Hoffart'}, {'name': 'David J. Porteous'}, {'name': 'Fang Fang'}, {'name': 'Johanna Jakobsdottir'}, {'name': 'Kelli Lehto'}, {'name': 'Ole A. Andreassen'}, {'name': 'Ole B.V. Pedersen'}, {'name': 'Thor Aspelund'}, {'name': 'Unnur Anna Valdimarsdottir'}]\", 'abstract': '<p>BACKGROUND\\nThe aim of this multinational study was to assess the development of adverse mental health symptoms among individuals diagnosed with COVID-19 in the general population by acute infection severity up to 16 months after diagnosis.\\nMETHODS\\nParticipants consisted of 247 249 individuals from seven cohorts across six countries (Denmark, Estonia, Iceland, Norway, Scotland, and Sweden) recruited from April 2020 through August 2021. We used multivariable Poisson regression to contrast symptom-prevalence of depression, anxiety, COVID-19 related distress, and poor sleep quality among individuals with and without a diagnosis of COVID-19 at entry to respective cohorts by time (0-16 months) from diagnosis. We also applied generalised estimating equations (GEE) analysis to test differences in repeated measures of mental health symptoms before and after COVID-19 diagnosis among individuals ever diagnosed with COVID-19 over time.\\nFINDINGS\\nA total of 9979 individuals (4%) were diagnosed with COVID-19 during the study period and presented overall with a higher symptom burden of depression (prevalence ratio [PR] 1.18, 95% confidence interval [95% CI] 1.03-1.36) and poorer sleep quality (1.13, 1.03-1.24) but not with higher levels of symptoms of anxiety or COVID-19 related distress compared with individuals without a COVID-19 diagnosis. While the prevalence of depression and COVID-19 related distress attenuated with time, the trajectories varied significantly by COVID-19 acute infection severity. Individuals diagnosed with COVID-19 but never bedridden due to their illness were consistently at lower risks of depression and anxiety (PR 0.83, 95% CI 0.75-0.91 and 0.77, 0.63-0.94, respectively), while patients bedridden for more than 7 days were persistently at higher risks of symptoms of depression and anxiety (PR 1.61, 95% CI 1.27-2.05 and 1.43, 1.26-1.63, respectively) throughout the 16-month study period.\\nCONCLUSION\\nAcute infection severity is a key determinant of long-term mental morbidity among COVID-19 patients.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472369', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Single-molecule sizing through nano-cavity confinement', 'authors': \"[{'name': 'Raphael P. B. Jacquat'}, {'name': 'Georg Krainer'}, {'name': 'Quentin Peter'}, {'name': 'Ali Nawaz Babar'}, {'name': 'Oliver Vanderpoorten'}, {'name': 'Catherine K. Xu'}, {'name': 'Clemens F. Kaminski'}, {'name': 'Ulrich F. Keyser'}, {'name': 'Jeremy J. Baumberg'}, {'name': 'Tuomas P. J. Knowles'}]\", 'abstract': '<p>An approach relying on nano-cavity confinement is developed in this paper for the sizing of nanoscale particles and single biomolecules in solution. The approach, termed nano-cavity diffusional sizing (NDS), measures particle residence times within fluidic nano-cavities to determine their hydrodynamic radii. Using theoretical modelling and simulation, we show that the residence time of particles within nano-cavities above a critical timescale depends on the diffusion coefficient of the particle, which allows estimation of the particles size. We demonstrate this approach experimentally through measurement of particle residence times within nano-fluidic cavities using single-molecule confocal microscopy. Our data show that the residence times scale linearly with the sizes of nanoscale colloids, protein aggregates and single DNA oligonucleotides. NDS thus constitutes a new single molecule optofluidic approach that allows rapid and quantitative sizing of nanoscale objects for potential application in nanobiotechnology, biophysics, and clinical diagnostics.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.21267698', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Development and external validation of the DOAT and DOATS scores: simple decision support tools to identify disease progression among nonelderly patients with mild/moderate COVID-19', 'authors': \"[{'name': 'Yoko Shibata'}, {'name': 'Hiroyuki Minemura'}, {'name': 'Yasuhito Suzuki'}, {'name': 'Takefumi Nikaido'}, {'name': 'Yoshinori Tanino'}, {'name': 'Atsuro Fukuhara'}, {'name': 'Ryuzo Kanno'}, {'name': 'Hiroyuki Saito'}, {'name': 'Shuzo Suzuki'}, {'name': 'Taeko Ishii'}, {'name': 'Yayoi Inokoshi'}, {'name': 'Eiichiro Sando'}, {'name': 'Hirofumi Sakuma'}, {'name': 'Tatsuho Kobayashi'}, {'name': 'Hiroaki Kume'}, {'name': 'Masahiro Kamimoto'}, {'name': 'Hideko Aoki'}, {'name': 'Akira Takama'}, {'name': 'Takamichi Kamiyama'}, {'name': 'Masaru Nakayama'}, {'name': 'Kiyoshi Saito'}, {'name': 'Koichi Tanigawa'}, {'name': 'Masahiko Sato'}, {'name': 'Toshiyuki Kanbe'}, {'name': 'Norio Kanzaki'}, {'name': 'Teruhisa Azuma'}, {'name': 'Keiji Sakamoto'}, {'name': 'Yuichi Nakamura'}, {'name': 'Hiroshi Otani'}, {'name': 'Mitsuru Waragai'}, {'name': 'Shinsaku Maeda'}, {'name': 'Tokiya Ishida'}, {'name': 'Keishi Sugino'}, {'name': 'Minoru Inage'}, {'name': 'Noriyuki Hirama'}, {'name': 'Kodai Furuyama'}, {'name': 'Shigeyuki Fukushima'}, {'name': 'Hiroshi Saito'}, {'name': 'Jun-ichi Machiya'}, {'name': 'Hiroyoshi Machida'}, {'name': 'Koya Abe'}, {'name': 'Katsuyoshi Iwabuchi'}, {'name': 'Yuji Katagiri'}, {'name': 'Yasuko Aida'}, {'name': 'Yuki Abe'}, {'name': 'Takahito Ota'}, {'name': 'Yuki Ishizawa'}, {'name': 'Yasuhiko Tsukada'}, {'name': 'Ryuki Yamada'}, {'name': 'Riko Sato'}, {'name': 'Takumi Omuna'}, {'name': 'Hikaru Tomita'}, {'name': 'Mikako Saito'}, {'name': 'Natsumi Watanabe'}, {'name': 'Mami Rikimaru'}, {'name': 'Takaya Kawamata'}, {'name': 'Takashi Umeda'}, {'name': 'Julia Morimoto'}, {'name': 'Ryuichi Togawa'}, {'name': 'Yuki Sato'}, {'name': 'Junpei Saito'}, {'name': 'Kenya Kanazawa'}, {'name': 'Kenji Omae'}, {'name': 'Kurita Noriaki'}, {'name': 'Ken Iseki'}]\", 'abstract': '<sec><title>BACKGROUND</title><p>Due to the dissemination of vaccination against severe acute respiratory syndrome coronavirus 2 in the elderly, the virus-susceptible subjects have shifted to unvaccinated non-elderlies. The risk factors of COVID-19 deterioration in non-elderly patients without respiratory failure have not yet been determined. This study was aimed to create simple predicting method to identify such patients who have high risk for exacerbation.</p></sec><sec><title>METHODS</title><p>We analyzed the data of 1,675 patients aged under 65 years who were admitted to hospitals with mild-to-moderate COVID-19. For validation, 324 similar patients were enrolled. Disease progression was defined as administration of medication, oxygen inhalation and mechanical ventilator starting one day or longer after admission.</p></sec><sec><title>RESULTS</title><p>The patients who exacerbated tended to be older, male, had histories of smoking, and had high body temperatures, lower oxygen saturation, and comorbidities such as diabetes/obesity and hypertension. Stepwise logistic regression analyses revealed that comorbidities of <underline>d</underline>iabetes/<underline>o</underline>besity, <underline>a</underline>ge  40 years, body temperature  38C, and oxygen saturation &lt; 96% (DOATS) were independent risk factors of worsening COVID-19. As a result two predictive scores were created: DOATS score, which includes all the above risk factors; and DOAT score, which includes all factors except for oxygen saturation. In the original cohort, the areas under the receiver operating characteristic curve of the DOATS and DOAT scores were 0.789 and 0.771, respectively. In the validation, the areas were 0.702 and 0.722, respectively.</p></sec><sec><title>CONCLUSION</title><p>We established two simple prediction scores that can quickly evaluate the risk of progression of COVID-19 in non-elderly, mild/moderate patients.</p></sec><sec><title>Summary</title><p>The risk stratification models using independent risks, namely comorbidity of <underline>d</underline>iabetes or <underline>o</underline>besity, <underline>a</underline>ge  40 years, high body temperature  38, and oxygen saturation &lt; 96%, DOATS and DOAT scores, predicted worsening COVID-19 in patients with mild-to-moderate cases.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.21267711', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Occupational and Environmental Health', 'covid': True, 'title': 'Relationship Between Alcohol Consumption and Telecommuting Preference-Practice Mismatch During the COVID-19 Pandemic', 'authors': \"[{'name': 'Chihiro Watanabe'}, {'name': 'Yusuke Konno'}, {'name': 'Reiji Yoshimura'}, {'name': 'Seiichiro Tateishi'}, {'name': 'Ayako Hino'}, {'name': 'Mayumi Tsuji'}, {'name': 'Akira Ogami'}, {'name': 'Masako Nagata'}, {'name': 'Keiji Muramatsu'}, {'name': 'Yoshihisa Fujino'}, {'name': ' '}]\", 'abstract': '<sec><title>Introduction</title><p>This study examined the association between increased alcohol consumption and telecommuting, comparing employees who expressed a preference for telecommuting and those who did not.</p></sec><sec><title>Methods</title><p>We conducted an internet monitor survey. Responses from 20,395 of the 33,302 participants were included in the final sample. Participants were asked about their desire for and frequency of telecommuting, and about changes in alcohol consumption under the COVID-19 pandemic. Data were analyzed by logistic regression analysis.</p></sec><sec><title>Results</title><p>Participants who telecommuted despite preferring not to do so reported significantly increased alcohol consumption, as revealed by a multivariate analysis (OR=1.62, 95% CI 1.25-2.12). Participants who expressed a preference for telecommuting showed no such increase.</p></sec><sec><title>Conclusions</title><p>Under the COVID-19 pandemic, telecommuting that involves a mismatch with employee preference for way of working may be a new risk factor for problematic drinking.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.21267800', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Sports Medicine', 'covid': True, 'title': 'A Cloth Facemask Causes No Major Respiratory or Cardiovascular Perturbations during Moderate to Heavy Exercise', 'authors': \"[{'name': 'Natlia Mendes Guardieiro'}, {'name': 'Gabriel Barreto'}, {'name': 'Felipe Miguel Marticorena'}, {'name': 'Tamires Nunes Oliveira'}, {'name': 'Luana Farias de Oliveira'}, {'name': 'Ana Lucia de S Pinto'}, {'name': 'Danilo Marcelo Leite do Prado'}, {'name': 'Bryan Saunders'}, {'name': 'Bruno Gualano'}]\", 'abstract': '<sec><title>Objectives</title><p>To investigate whether wearing a cloth facemask could affect physiological and perceptual responses to exercise at distinct exercise intensities in non-trained men and women.</p></sec><sec><title>Methods</title><p>In a crossover design, participants (17 men and 18 women) underwent a progressive square-wave test at four intensities (i. at 80% of the ventilatory anerobic threshold [80%VAT]; ii. at VAT; iii. at the respiratory compensation point [RCP]; iv. at exercise peak [Peak] to exhaustion), with or without a triple-layered cloth mask (Mask or No-Mask). Several physiological, metabolic and perceptual measures were analyzed.</p></sec><sec><title>Results</title><p>Mask reduced inspiratory capacity at all exercise intensities vs. No-Mask (p&lt;0.0001), irrespective of sex. Mask reduced respiratory frequency vs. No-Mask (p=0.001) at Peak (8.3 breathsmin<sup>-1</sup>; CI: -5.8, -10.8), RCP (6.9 breathsmin<sup>-1</sup>; CI: -4.6, -9.2) and VAT (6.5 breathsmin<sup>-1</sup>; CI: -4.1, -8.8), but not at Baseline or at 80%VAT. Mask also reduced tidal volume (p&lt;0.0001) at both RCP (0.5L; CI: -0.3, -0.6) and Peak (0.8L; CI: -0.6, -0.9), but not at Baseline, 80%VAT or VAT. Shallow breathing index was increased with Mask at Peak compared to No-Mask (11.3; CI: 7.5, 15.1), but not at any other intensities. Mask did not change heart rate, lactate, ratings of perceived exertion, blood pressure or oxygen saturation.</p></sec><sec><title>Conclusions</title><p>Wearing a cloth facemask during exercise at moderate to heavy intensities is unlikely to incur significant respiratory or cardiovascular changes, irrespective of sex. These data can inform new exercise recommendations for health during the COVID-19 pandemic and debunk unfounded allegations of harmful effects of masks during exercise. ClinicalTrials.gov: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04887714\">NCT04887714</ext-link></p></sec><sec><title>What are the new findings?</title><list list-type=\"simple\"><list-item><label></label><p>Using a progressive square-wave test, we showed that wearing a cloth facemask during exercise increased breathing difficulty, but this was dependent upon the exercise intensity.</p></list-item><list-item><label></label><p>Respiratory variables (e.g., inspiratory capacity, respiratory frequency, shallow breathing index) were affected at higher rather than lower intensities.</p></list-item><list-item><label></label><p>Mask wearing did not change heart rate, lactate, ratings of perceived exertion, blood pressure or oxygen saturation at any exercise intensity.</p></list-item><list-item><label></label><p>There were no substantial sex differences on the effects of mask wearing during exercise.</p></list-item></list></sec><sec><title>How might it impact on clinical practice in the future?</title><list list-type=\"simple\"><list-item><label></label><p>These data can debunk unfounded allegations on harmful effects of masks during exercise, and help inform new exercise recommendations for health during the COVID-19 pandemic, particularly where facemasks remain necessary.</p></list-item></list></sec>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472537', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Key features of inhibitor binding to the human mitochondrial pyruvate carrier hetero-dimer', 'authors': \"[{'name': 'Sotiria Tavoulari'}, {'name': 'Tom J.J. Schirris'}, {'name': 'Vasiliki Mavridou'}, {'name': 'Chancievan Thangaratnarajah'}, {'name': 'Martin S. King'}, {'name': 'Daniel T.D. Jones'}, {'name': 'Shujing Ding'}, {'name': 'Ian M. Fearnley'}, {'name': 'Edmund R.S. Kunji'}]\", 'abstract': '<p>The mitochondrial pyruvate carrier (MPC) has emerged as a promising drug target for metabolic disorders, including non-alcoholic steatohepatitis and diabetes, metabolically dependent cancers and neurodegenerative diseases. Human MPC is a protein complex, but the composition of its active form is debated and the mechanisms of transport and inhibition are not resolved. We have recombinantly expressed and purified the human hetero-complex MPC1L/MPC2 and demonstrate that it is a functional hetero-dimer, like the yeast MPC hetero-dimers. Unlike the latter, human MPC1L/MPC2 binds the known inhibitors with high potencies. We identify the essential chemical features shared between these structurally diverse inhibitors and demonstrate that high affinity binding is not attributed to covalent bond formation with MPC cysteines, as previously thought. We also identify 14 new inhibitors of MPC, one outperforming the most potent compound UK5099 by tenfold. Two of them are the commonly prescribed drugs entacapone and nitrofurantoin, suggesting possible off-target mechanisms associated with their adverse effects. This work advances our understanding of MPC inhibition and will accelerate the development of clinically relevant MPC modulators.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.12.472303', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Dehalobium species implicated in 2,3,7,8-tetrachloro-p-dioxin dechlorination in the contaminated sediments of Sydney Harbour Estuary', 'authors': \"[{'name': 'Matthew Lee'}, {'name': 'Gan Liang'}, {'name': 'Sophie I. Holland'}, {'name': 'Casey OFarrell'}, {'name': 'Keith Osbourne'}, {'name': 'Michael J. Manefield'}]\", 'abstract': '<p>Polychlorinated dibenzo-<italic>p</italic>-dioxins and furans (PCDD/F) are some of the most environmentally recalcitrant and toxic compounds. They are naturally occurring and by-products of anthropogenic activity. Sydney Harbour Estuary (Sydney, Australia), is heavily contaminated with PCDD/F. Analysis of sediment cores revealed that the contamination source in Homebush Bay continues to have one of the highest levels of PCDD/F contamination in the world (5207 pg WHO-TEQ g<sup>-1</sup>) with &gt;50% of the toxicity attributed to 2,3,7,8-tetrachlorodibenzo-<italic>p</italic>-dioxin (2,3,7,8-TCDD) the most toxic and concerning of the PCDD/F congeners. Comparison of congener profiles at the contamination source with surrounding bays and historical data provided evidence for the attenuation of 2,3,7,8-TCDD and other congeners at the source. This finding was supported by the detection of di-, mono- and unchlorinated dibenzo-<italic>p</italic>-dioxin. Microbial community analysis of sediments by 16S amplicon sequencing revealed an abundance of lineages from the class Dehalococcoidia (up to 15% of the community), including the genus <italic>Dehalobium</italic> (up to 0.5%). Anaerobic seawater enrichment cultures using perchloroethene as a more amenable growth substrate enriched only the <italic>Dehalobium</italic> population by more than six-fold. The enrichment culture then proved capable of reductively dechlorinating 2,3,7,8-TCDD to 2,3,7-TCDD and octachlorodibenzo-<italic>p</italic>-dibenzodioxin to hepta and hexa congeners. This work is the first to show microbial reductive dehalogenation of 2,3,7,8-TCDD with a bacterium from outside the <italic>Dehalococcoides</italic> genus, and one of only a few that demonstrates PCDD/F degradation in a marine environment.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472366', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Modeling of microRNA-derived disease network repurposes methotrexate for the prevention and therapy of abdominal aortic aneurysm in mice', 'authors': \"[{'name': 'Yicong Shen'}, {'name': 'Yuanxu Gao'}, {'name': 'Jiangcheng Shi'}, {'name': 'Zhou Huang'}, {'name': 'Rongbo Dai'}, {'name': 'Yi Fu'}, {'name': 'Yuan Zhou'}, {'name': 'Wei Kong'}, {'name': 'Qinghua Cui'}]\", 'abstract': '<p>Abdominal aortic aneurysm (AAA) is a highly lethal vascular disease characterized by permanent dilatation of the abdominal aorta. The main purpose of the current study is to search for noninvasive medical therapies for abdominal aortic aneurysm (AAA), for which there is currently no effective drug therapy. Network medicine represents a cutting-edge technology, as analysis and modeling of disease networks can provide critical clues regarding the etiology of specific diseases and which therapeutics may be effective. Here, we proposed a novel algorithm to quantify disease relations based on a large accumulated microRNA-disease association dataset and then built a disease network that covered 15 disease classes and included 304 diseases. Analysis revealed a number of patterns for these diseases; for example, diseases tended to be clustered and coherent in the network. Surprisingly, we found that AAA showed the strongest similarity with rheumatoid arthritis and systemic lupus erythematosus, both of which are autoimmune diseases, suggesting that AAA could be one type of autoimmune disease in etiology. Based on this observation, we further hypothesized that drugs for autoimmune disease could be repurposed for the prevention and therapy of AAA. Finally, animal experiments confirmed that methotrexate, a drug for autoimmune disease, was able to prevent the formation and inhibit the development of AAA.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472443', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Glutathione binding to the plant AtAtm3 transporter and implications for the conformational coupling of ABC transporters', 'authors': \"[{'name': 'Chengcheng Fan'}, {'name': 'Douglas C. Rees'}]\", 'abstract': '<p>The ATP Binding Cassette (ABC) transporter of mitochondria (Atm) from <italic>Arabidopsis thaliana</italic> (<italic>At</italic>Atm3) has been implicated in the maturation of cytosolic iron-sulfur proteins and heavy metal detoxification, plausibly by exporting glutathione derivatives. Using single-particle cryo-electron microscopy, we have determined structures of <italic>At</italic>Atm3 in multiple conformational states. These structures not only provide a structural framework for defining the alternating access transport cycle, but also highlight an unappreciated feature of the glutathione binding site, namely the paucity of cysteine residues that could potentially form inhibitory mixed disulfides with glutathione. Despite extensive efforts, we were unable to prepare the ternary complex of <italic>At</italic>Atm3 with bound GSSG and MgATP. A survey of structurally characterized type IV ABC transporters that includes <italic>At</italic>Atm3 establishes that while nucleotides are found associated with all conformational states, they are effectively required to stabilize occluded and outward-facing conformations. In contrast, transport substrates have only been observed associated with inward-facing conformations. The absence of structures containing both nucleotide and transport substrate suggests that this ternary complex exists only transiently during the transport cycle.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472350', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'An amphipathic and cationic antimicrobial peptide kills colistin resistant Gram-negative pathogens in vivo', 'authors': \"[{'name': 'Thomas T. Thomsen'}, {'name': 'Mette Kolpen'}, {'name': 'Vinoth Wigneswaran'}, {'name': 'Ulrik Kromann'}, {'name': 'Anna E. Ebbensgaard'}, {'name': 'Anette M. Hammerum'}, {'name': 'Henrik Hasman'}, {'name': 'Stine Radmer'}, {'name': 'Kasper N. Kragh'}, {'name': 'Rasmus Hartmann-Petersen'}, {'name': 'Paul Robert Hansen'}, {'name': 'Anders Folkesson'}, {'name': 'Niels Frimodt-Mller'}, {'name': 'Anders Lbner-Olesen'}]\", 'abstract': '<p>New antibiotics are needed against multidrug resistant Gram-negative pathogens that have compromised global health systems. Antimicrobial peptides are generally considered promising lead candidates for the next generation of antibiotics but have not fulfilled this expectation. Here we demonstrate activity of a cationic amphipathic undecapeptide (ChIP; <underline>Ch</underline>arge change <underline>I</underline>ndependent <underline>P</underline>eptide) against a wide panel of multidrug resistant Gram-negative pathogens. Importantly, the antimicrobial activity of ChIP is independent of the surface charge changes that confer colistin resistance through modification of Lipid A, while decreased activity of ChIP correlates with GlcN1 tri-acylation of Lipid A. In an <italic>in vivo</italic> peritonitis mouse model ChIP displays excellent activity against both colistin sensitive and resistant <italic>Escherichia coli</italic> and <italic>Acinetobacter baumannii</italic> strains.</p><sec><title>Author Summary</title><p>Antimicrobial peptides hold promise as novel treatment options for diseases caused by multidrug resistant bacteria. Here we present evidence that the ChIP peptide, comprised of 11 D-amino acids, is active against a variety of Gram-negative bacteria that ranks high on the WHO list of critically important pathogens. ChIP initially interacts with the Gram-negative outer membrane, independent of its surface charge, followed by entry into the periplasm and permeabilization of the inner membrane, leading to bacterial cell death. Detailed analyses of the outer membrane indicate that the acylation pattern of lipopolysaccharides plays an important role for ChIP activity. In a mouse infection model, ChIP display excellent activity in reducing bacterial numbers for both <italic>Escherichia coli</italic> and <italic>Acinetobacter baumannii</italic>. Importantly, ChIP is highly efficient against bacteria resistant to colistin, an antibiotic normally considered as a last resort treatment of infections caused by multidrug resistant Gram-negative bacteria.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472468', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A library for fMRI real-time processing systems in python (RTPSpy) with comprehensive online noise reduction, fast and accurate anatomical image processing, and online processing simulation', 'authors': \"[{'name': 'Masaya Misaki'}, {'name': 'Jerzy Bodurka'}, {'name': 'Martin P Paulus'}]\", 'abstract': '<p>We introduce a python library for real-time fMRI (rtfMRI) data processing systems, Real-Time Processing System in python (RTPSpy), to provide building blocks for a custom rtfMRI application with extensive and advanced functionalities. RTPSpy is a library package including 1) a fast, comprehensive, and flexible online fMRI denoising pipeline comparable to offline processing, 2) utilities for fast and accurate anatomical image processing to define a target region on-site, 3) a simulation system of online fMRI processing to optimize a pipeline and target signal calculation, 4) interface to an external application for feedback presentation, and 5) a boilerplate graphical user interface (GUI) integrating operations with RTPSpy library. Since online fMRI data processing cannot be equivalent to offline, we discussed the limitations of online analysis and their solutions in the RTPSpy implementation. We developed a fast and accurate anatomical image processing script with fast tissue segmentation (FastSeg), image alignment, and spatial normalization, utilizing the FastSurfer, AFNI, and ANTs. We confirmed that the FastSeg output was comparable with FreeSurfer, and could complete all the anatomical image processing in a few minutes. Thanks to its highly modular architecture, RTPSpy can easily be used for a simulation analysis to optimize a processing pipeline and target signal calculation. We present a sample script for building a real-time processing pipeline and running a simulation using RTPSpy. The library also offers a simple signal exchange mechanism with an external application. An external application can receive a real-time neurofeedback signal from RTPSpy in a background thread with a few lines of script. While the main components of the RTPSpy are the library modules, we also provide a GUI class for easy access to the RTPSpy functions. The boilerplate GUI application provided with the package allows users to develop a customized rtfMRI application with minimum scripting labor. Finally, we discussed the limitations of the package regarding environment-specific implementations. We believe that RTPSpy is an attractive option for developing rtfMRI applications highly optimized for individual purposes. The package is available from GitHub (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/mamisaki/RTPSpy\">https://github.com/mamisaki/RTPSpy</ext-link>) with GPL3 license.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.17.21267655', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Intensive Care and Critical Care Medicine', 'covid': True, 'title': 'Presentation, characteristics, treatments and outcomes of mechanically ventilated patients with COVID-19 in Bulgaria', 'authors': \"[{'name': 'Neda Bakalova'}, {'name': 'Ivats Natsev'}, {'name': 'Hristo Damov'}, {'name': 'Irina Yatsenko'}, {'name': 'Stefanija Jovinska'}, {'name': 'Rostislav Enev'}, {'name': 'Mikhail Cholakov'}, {'name': 'Filip Abedinov'}, {'name': 'Denitza Kalendjieva'}]\", 'abstract': '<p>Background: The first surge of coronavirus disease 2019 (COVID-19) cases in Bulgaria occurred in the fall of 2020. To accommodate the rising number of critically ill patients, new intensive care units were formed in several hospitals. Here we describe the clinical presentation, patient characteristics, treatments and outcomes of mechanically ventilated COVID-19 patients in a newly formed COVID-19 ICU at a tertiary cardiac center in Sofia, Bulgaria.  \\n\\nMethods: This is a retrospective observational study of mechanically ventilated COVID-19 patients admitted to Sveta Ekaterina University Hospital in Sofia, Bulgaria, between November 4th, 2020 and January 6th, 2021. Data were collected from electronic and written patient records and charts. \\n\\nResults: We identified 38 critical care patients admitted with respiratory failure and treated with mechanical ventilation at our COVID-19 ICU during this period. The median age was 66 (IQR 57-76, range 27-89) and 74% were male. Most patients, 36 (95%), had at least one comorbidity. The most common comorbidities were hypertension, valvular heart disease, ischemic heart disease and diabetes mellitus. Overall, 27 (71%) patients had a concomitant cardiac disease other than hypertension and 24% were recent cardiac surgical patients. Inotropic support was required in 29 (76%) patients, renal replacement therapy in 12 (32%) patients and prone positioning and ECMO were used in 5 (13%) and 2 (5%) patients respectively. The median duration of mechanical ventilation was 7.5 (IQR 5-14) days overall and 9 (IQR 6-13) days for survivors. At 30-days 28 (74%) of patients had died. Overall, 32 (84%) patients died in hospital and only 6 (16%) patients were discharged home. \\n\\nConclusions: During the first major surge of COVID-19 cases in Bulgaria, despite the wave arriving later than in other countries, the healthcare system was largely unprepared. In our setting, mortality in critically ill patients requiring mechanical ventilation was very high at 85%. There may be several factors contributing to these results, namely the predominance of cardiovascular comorbidities in our patient population, the strained ICU capacity and the lack of medical personnel to provide adequate intensive care to such complex patients.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.472997', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Dopamine and GPCR-mediated modulation of DN1 clock neurons gates the circadian timing of sleep', 'authors': \"[{'name': 'Matthias Schlichting'}, {'name': 'Shlesha Richhariya'}, {'name': 'Nicholas Herndon'}, {'name': 'Dingbang Ma'}, {'name': 'Jason Xin'}, {'name': 'William Lenh'}, {'name': 'Katharine Abruzzi'}, {'name': 'Michael Rosbash'}]\", 'abstract': '<p>The metronome-like circadian regulation of sleep timing must still adapt to an uncertain environment. Recent studies in Drosophila indicate that neuromodulation not only plays a key role in clock neuron synchronization but also affects interactions between the clock network and brain sleep centers. We show here that the targets of neuromodulators, G-Protein Coupled Receptors (GPCRs), are highly enriched in the fly brain circadian clock network. Single cell sequencing indicates that they are not only differentially expressed but also define clock neuron identity. We generated a comprehensive guide library to mutagenize individual GPCRs in specific neurons and verified the strategy with a targeted sequencing approach. Combined with a behavioral screen, the mutagenesis strategy revealed a novel role of dopamine in sleep regulation by identifying two dopamine receptors and a clock neuron subpopulation that gate the timing of sleep.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.21267039', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Allergy and Immunology', 'covid': True, 'title': 'Anti-SARS-CoV-2 cellular immunity in 571 vaccinees assessed using an interferon- release assay', 'authors': \"[{'name': 'Yoshifumi Uwamino'}, {'name': 'Masatoshi Wakui'}, {'name': 'Yoko Yatabe'}, {'name': 'Terumichi Nakagawa'}, {'name': 'Akiko Sakai'}, {'name': 'Toshinobu Kurafuji'}, {'name': 'Ayako Shibata'}, {'name': 'Yukari Tomita'}, {'name': 'Masayo Noguchi'}, {'name': 'Akiko Tanabe'}, {'name': 'Tomoko Arai'}, {'name': 'Akemi Ohno'}, {'name': 'Hiromitsu Yokota'}, {'name': 'Shunsuke Uno'}, {'name': 'Wakako Yamasawa'}, {'name': 'Yasunori Sato'}, {'name': 'Mari Ikeda'}, {'name': 'Akihiko Yoshimura'}, {'name': 'Naoki Hasegawa'}, {'name': 'Hideyuki Saya'}, {'name': 'Mitsuru Murata'}]\", 'abstract': '<p>Generation of antigen-specific memory T cells has been analyzed only for few coronavirus disease 2019 (COVID-19) vaccinees, whereas antibody titers have been serologically measured for a large number of individuals. Here, we assessed the anti-severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) cellular immune response in a large cohort using interferon (IFN)- release assays (IGRAs) based on short-term whole blood culture. The study included 571 individuals who received the viral spike (S) protein-expressing BNT162b2 mRNA SARS-CoV-2 vaccine. Serum IgG titers against the receptor-binding domain (RBD) of S protein were measured. Samples of 28 vaccinees were subjected to flow cytometry analysis of T cells derived from short-term whole blood culture. IFN- production triggered by S antigens was observed in most individuals 8 weeks after receiving the second dose of the vaccine, indicating acquisition of T cell memory responses. The frequencies of activated T cell subsets were strongly correlated with IFN- levels, supporting the usability of our approach. S antigen-stimulated IFN- levels were weakly correlated with anti-RBD IgG titers and associated with pre-vaccination infection and adverse reactions after the second dose. Our approach revealed cellular immunity acquired after COVID-19 vaccination, providing insights regarding the effects and adverse reactions of vaccination.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.473091', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'The Trypanosoma cruzi kinetoplast DNA minicircle sequences transfer biomarker of the multidrug treatment of Chagas disease', 'authors': \"[{'name': 'Antonio R. L. Teixeira'}, {'name': 'Alessandro O Sousa'}, {'name': 'Clever C Gomes'}, {'name': 'Adriana A S'}, {'name': 'Rubens J Nascimento'}, {'name': 'Liana L Pires'}, {'name': 'Ana M Castro'}, {'name': 'Francisco B Moreno'}]\", 'abstract': '<p>Background: The Trypanosoma cruzi infection renders the transfer of the mitochondrion kinetoplast DNA minicircle sequences into the hosts genome. The Aves are refractory to the infection, but chicks hatched from the T. cruzi inoculated eggs integrate the DNA minicircle sequences into the germ line cells. Rabbits, mice and chickens with the minicircle sequences mutations develop the Chagas cardiomyopathy and the DNA transfer underpins the heart disease.\\nMethodology: The PCR with the specific primer sets revealed the Protist nuclear DNA and the kinetoplast DNA in the agarose gels bands probed with the radiolabel specific sequences from tissues of the T. cruzi-infected rabbits and of the mice. A targetprimer TAIL-PCR amplification employing primer sets from the chickens, rabbits and mice, in combination with primer sets from the the T. cruzi kinetoplast minicircle sequences was used. This approach led us to disclose the integration sites of the kinetoplast DNA biomarker, then, used to monitor the effect of multidrug treatment of the T. cruzi infected mice.\\nPrincipal findings: The Southern hybridization, clone and sequence of the amplification products revealed the DNA minicircle sequences integrations sites in the LINE transposable elements. An array of inhibitors of eukaryote cells division was used to arrest the DNA transfer. It was shown that nine out of 12 inhibitors prevented the kinetoplast DNA integration into the macrophage genome. The multidrug treatment of the acutely T. cruzi-infected mice with Benznidazole, Azidothymidine and Ofloxacin lessened circa 2.5-fold the rate of the minicircle sequences integrations in the mouse genome and inhibited the rejection of the target heart cells.\\nConclusion and significance: The T. cruzi mitochondrion kinetoplast minicircle sequences transfer driven pathogenesis of Chagas disease is an ancient Cross-Kingdom DNA phenomenon of evolution and, therefore, paradigm research with effective purposing inhibitors is needed.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.21267562', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Real-world data on immune responses following heterologous prime-boost COVID-19 vaccination schedule with Pfizer and AstraZeneca vaccines in England', 'authors': \"[{'name': 'Samantha J Westrop'}, {'name': 'Heather J Whitaker'}, {'name': 'Annabel A Powell'}, {'name': 'Linda Power'}, {'name': 'Corinne Whillock'}, {'name': 'Helen Campbell'}, {'name': 'Ruth Simmons'}, {'name': 'Lenesha Warrener'}, {'name': 'Mary E Ramsay'}, {'name': 'Shamez N Ladhani'}, {'name': 'Kevin E Brown'}, {'name': 'Gayatri Amirthalingham'}]\", 'abstract': '<p><bold>Background</bold>\\nThere are limited data on immune responses to heterologous COVID19 immunisation schedules, especially following an extended 12week interval between doses.\\n<bold>Methods</bold>\\nSARSCoV2 infectionnave and previouslyinfected adults receiving ChAdBNT (ChAdOx1 nCoV19, AstraZeneca followed by BNT162b2, PfizerBioNTech) or BNTChAd as part of the UK national immunisation programme provided blood samples at 30 days and 12 weeks after their second dose. Geometric mean concentrations (GMC) of antiSARSCoV2 spike (S-antibody) and nucleoprotein (N-antibody) IgG antibodies and geometric mean ratios (GMR) were compared with a contemporaneous cohort receiving homologous ChAdChAd or BNTBNT.\\n<bold>Results</bold>\\nDuring MarchOctober 2021, 75,827 individuals were identified as having received heterologous vaccination, 9,489 invited to participate, 1,836 responded (19.3%) and 656 were eligible. In previouslyuninfected adults, Santibody GMC at 30 days postsecond dose were lowest for ChAdChAd (862 (95%CI, 694 1069)) and significantly higher for ChAdBNT (6233 (5522 7035); GMR 6.29; (5.04 7.85); p&lt;0.001), BNT-ChAd (4776 (4066 5610); GMR 4.55 (3.56 5.81); p&lt;0.001) and BNTBNT (5377 (4596 6289); GMR 5.66 (4.49 7.15); p&lt;0.001). By 12 weeks after dose two, Santibody GMC had declined in all groups and remained significantly lower for ChAdChAd compared to ChAdBNT (GMR 5.12 (3.79 6.92); p&lt;0.001), BNTChAd (GMR 4.1 (2.96 5.69); p&lt;0.001) and BNTBNT (GMR 6.06 (4.32 8.50); p&lt;0.001). Previously infected adults had higher Santibody GMC compared to infectionnave adults at all timepoints and with all vaccine schedules. \\n<bold>Conclusions</bold>\\nThese realworld findings demonstrate heterologous schedules with adenoviralvector and mRNA vaccines are highly immunogenic and may be recommended after a serious adverse reaction to one vaccine product, or to increase programmatic flexibility where vaccine supplies are constrained.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.21267714', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'Hepatitis C subtyping assay failure in UK patients born in Sub-Saharan Africa: implications for global treatment and elimination', 'authors': \"[{'name': 'Kazeem Adeboyejo'}, {'name': 'Barnabas J. King'}, {'name': 'Theocharis Tsoleridis'}, {'name': 'Alexander W. Tarr'}, {'name': 'John McLauchlan'}, {'name': 'William L. Irving'}, {'name': 'Jonathan K. Ball'}, {'name': 'C. Patrick McClure'}]\", 'abstract': '<p>The newly developed direct-acting antivirals (DAAs) have revolutionised the treatment of chronic hepatitis C virus (HCV), where cohort studies have shown that cure rates as high as 98% can be achieved. Whilst genome sequencing has demonstrated that some subtypes of HCV naturally harbour drug resistance associated substitutions (RAS), these have not been considered important as previous molecular epidemiological studies have suggested that such difficult-to-treat subtypes are rare. Therefore, to optimise and streamline molecular detection and sequence-based typing of diverse RAS-containing subtypes, a novel panel of single round PCR assays was applied to HCV derived from 146 individuals, whose likely source of infection was from regions of sub-Saharan Africa (SSA). Partial NS5A and NS5B sequences were obtained from 135 HCV-positive patients born in 19 different countries from SSA but attending clinics in the UK.</p><p>Virus subtype assignments were determined by pairwise-distance analysis and compared to both diagnostic laboratory assignments and free-to-use online typing tools. We determined that routine clinical diagnostic methods incorrectly subtyped 59.0% of samples, with a further 6.8% incorrectly genotyped. Of five commonly used online tools, Geno2Pheno performed most effectively in determining a subtype in agreement with pairwise distance analysis. Considering the estimated number of HCV infections to have occurred in across Africa, this study provides a simple low-cost pathway to guide regional therapeutic choice and assist global eradication programmes.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472579', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Effects of habitat destruction on coevolving metacommunities', 'authors': \"[{'name': 'Klementyna A. Gawecka'}, {'name': 'Fernando Pedraza'}, {'name': 'Jordi Bascompte'}]\", 'abstract': '<p>Habitat destruction is a growing threat to biodiversity and ecosystem services. The ecological consequences of habitat loss and fragmentation involve reductions in species abundance and even the extinction of species and interactions. However, we do not yet understand how habitat loss can alter the coevolutionary trajectories of the remaining species or how coevolution, in turn, affects their response to habitat loss. To investigate this, we develop a spatially explicit model which couples metacommunity and coevolutionary dynamics. We show that, by changing the size, composition and structure of local networks, habitat destruction increases the diversity of coevolutionary outcomes across the landscape. Furthermore, we show that while coevolution dampens the negative effects of habitat destruction in mutualistic networks, its effects on the persistence of antagonistic communities are less predictable.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.11.472230', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Peptide Aggregation Induced Immunogenic Rupture (PAIIR)', 'authors': \"[{'name': 'Gokhan Gunay'}, {'name': 'Seren Hamsici'}, {'name': 'Gillian A. Lang'}, {'name': 'Mark L. Lang'}, {'name': 'Susan Kovats'}, {'name': 'Handan Acar'}]\", 'abstract': '<p>Under the influence of stress and membrane damage, cells undergo immunogenic cell death (ICD), which involves the release of damage associated molecular patterns (DAMPs), natural adjuvants for enhancing an immune response. In the presence of an antigen, released DAMPs can determine the type and magnitude of the immune response, and therefore the longevity and efficacy of an antigen-specific immunity. In the last decade, the immune response effect of ICD has been shown, yet there is no tool that can induce controlled ICD with predictable results, regardless of the cell type. We designed a peptide-based tool, called [II], for controlled damage to cell membrane to induce ICD and DAMPs release. Herein we describe a series of experiments that determine that the mechanism of action of [II] includes a caspase-dependent ICD and subsequent release of immune stimulating DAMPs, on various cell types. Moreover, we tested the hypothesis that controlled DAMP release via [II] in vivo was associated with enhancement of antigen-specific adaptive immunity with influenza hemagglutinin (HA) subunit vaccine. HA and [II] showed significantly higher HA specific IgG1 and IgG2a antibodies, compared to HA-only immunized mice, while the peptide itself did not elicit antibodies. In this paper, we demonstrate the first peptide-aggregation induced immunogenic rupture (PAIIR) approach as vaccine adjuvants for increasing both humoral and cellular immunity. In consideration of its ability to enhance IgG2a responses that are associated with heterosubtypic influenza virus protection, PAIIR is a promising adjuvant to promote universal protection upon influenza HA vaccination.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472696', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Epigenetic remodeling during monolayer cell expansion reduces therapeutic potential', 'authors': \"[{'name': 'Adrienne Scott'}, {'name': 'Eduard Casas'}, {'name': 'Stephanie Ellyse Schneider'}, {'name': 'Alison Swearingen'}, {'name': 'Courtney Van Den Elzen'}, {'name': 'Benjamin Seelbinder'}, {'name': 'Jeanne Barthold'}, {'name': 'Jennifer Kugel'}, {'name': 'Josh Lewis Stern'}, {'name': 'Kyla Foster'}, {'name': 'Nancy Emery'}, {'name': 'Justin Brumbaugh'}, {'name': 'Corey Neu'}]\", 'abstract': '<p>Understanding how cells remember previous mechanical environments to influence their fate, or mechanical memory, informs the design of biomaterials and therapies in medicine. Current regeneration therapies require two-dimensional (2D) cell expansion processes to achieve large cell populations critical for the repair of damaged (e.g. connective and musculoskeletal) tissues. However, the influence of mechanical memory on cell fate following expansion is unknown, and mechanisms defining how physical environments influence the therapeutic potential of cells remain poorly understood. Here, we show that the organization of histone H3 trimethylated at lysine 9 (H3K9me3) and expression of tissue-identifying genes in primary cartilage cells (chondrocytes) transferred to three-dimensional (3D) hydrogels depends on the number of previous population doublings on tissue culture plastic during 2D cell expansion. Decreased levels of H3K9me3 occupying promoters of dedifferentiation genes after the 2D culture were also retained in 3D culture. Suppression of H3K9me3 during expansion of cells isolated from a murine model similarly resulted in the loss of the chondrocyte phenotype and global remodeling of nuclear architecture. In contrast, increasing levels of H3K9me3 through inhibiting H3K9 demethylases partially rescued the chondrogenic nuclear architecture and gene expression, which has important implications for tissue repair therapies, where expansion of large numbers of phenotypically-suitable cells is required. Overall, our findings indicate mechanical memory in primary cells is encoded in the chromatin architecture, which impacts cell fate and the phenotype of expanded cells.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.11.472202', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': True, 'title': 'Epistatic models predict mutable sites in SARS-CoV-2 proteins and epitopes', 'authors': \"[{'name': 'Juan Rodriguez-Rivas'}, {'name': 'Giancarlo Croce'}, {'name': 'Maureen Muscat'}, {'name': 'Martin Weigt'}]\", 'abstract': '<p>The emergence of new variants of SARS-CoV-2 is a major concern given their potential impact on the transmissibility and pathogenicity of the virus as well as the efficacy of therapeutic interventions. Here, we predict the mutability of all positions in SARS-CoV-2 protein domains to forecast the appearance of unseen variants. Using sequence data from other coronaviruses, pre-existing to SARS-CoV-2, we build statistical models that do not only capture amino-acid conservation but more complex patterns resulting from epistasis. We show that these models are notably superior to conservation profiles in estimating the already observable SARS-CoV-2 variability. In the receptor binding domain of the spike protein, we observe that the predicted mutability correlates well with experimental measures of protein stability and that both are reliable mutability predictors (ROC AUC ~0.8). Most interestingly, we observe an increasing agreement between our model and the observed variability as more data become available over time, proving the anticipatory capacity of our model. When combined with data concerning the immune response, our approach identifies positions where current variants of concern are highly overrepresented. These results could assist studies on viral evolution, future viral outbreaks and, in particular, guide the exploration and anticipation of potentially harmful future SARS-CoV-2 variants.</p><sec><title>Significance statement</title><p>During the COVID pandemic, new SARS-CoV-2 variants emerge and spread, some being of major concern due to their increased infectivity or their capacity to reduce vaccine efficiency. Anticipating new mutations, which might give rise to new variants, would be of great interest. Here we construct sequence models predicting how mutable SARS-CoV-2 positions are, using a single SARS-CoV-2 sequence and databases of other coronaviruses. Predictions are tested against available mutagenesis data and the observed variability of SARS-CoV-2 proteins. Interestingly, our predictions agree increasingly with observations, as more SARS-CoV-2 sequences become available. Combining predictions with immunological data, we find a clear overrepresentation of mutations in current variants of concern. The approach may become relevant for potential outbreaks of future viral diseases.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.15.21267794', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Transplantation', 'covid': True, 'title': 'Changing of therapeutic trends between the 1st and 2nd wave did not reduce COVID-19 related mortality of renal transplant recipients: a national registry study', 'authors': \"[{'name': 'Bastien Berger'}, {'name': 'Marc Hazzan'}, {'name': 'Nassim Kamar'}, {'name': 'Hlne Francois'}, {'name': 'Marie Matignon'}, {'name': 'Clarisse Greze'}, {'name': 'Philippe Gatault'}, {'name': 'Luc Frimat'}, {'name': 'Pierre F Westeel'}, {'name': 'Valentin Goutaudier'}, {'name': 'Renaud Snanoudj'}, {'name': 'Charlotte Colosio'}, {'name': 'Antoine Sicard'}, {'name': 'Dominique Bertrand'}, {'name': 'Christiane Mousson'}, {'name': 'Jamal Bamoulid'}, {'name': 'Antoine Thierry'}, {'name': 'Dany Anglicheau'}, {'name': 'Lionel Couzi'}, {'name': 'Jonathan M Chemouny'}, {'name': 'Agnes Duveau'}, {'name': 'Valerie Moal'}, {'name': 'Yannick Le Meur'}, {'name': 'Gilles Blancho'}, {'name': 'Jrme Tourret'}, {'name': 'Paolo Malvezzi'}, {'name': 'Christophe Mariat'}, {'name': 'Jean-Philippe Rerolle'}, {'name': 'Nicolas Bouvier'}, {'name': 'Sophie Caillard'}, {'name': 'Olivier Thaunat'}, {'name': ' '}]\", 'abstract': '<p>SARS-CoV-2 pandemic evolved in two consecutive waves over 2020 (for France: 1st wave from March 1 to July 31; and 2nd wave from August 1 to December 31). Improvements in the management of COVID-19 led to a reduction of mortality rates in hospitalized patients during the second wave. Whether this progress also benefited to kidney transplant recipients (KTR), a population particularly vulnerable to severe COVID-19, remained unclear.</p><p>In France, 957 KTR were hospitalized for COVID-19 in 2020 and their data were prospectively collected in the French SOT COVID registry. The presentation, management, and outcomes of the 359 KTR diagnosed during the 1st wave were compared to those of the 598 of the 2nd wave. Baseline comorbidities were largely similar between KTR of the 2 waves. Maintenance immunosuppression was reduced in most patients but withdrawal of antimetabolite (73.7% vs 58.4%, p&lt;0.001) or CNI (32.1% vs 16.6%, p&lt;0.001) was less frequent during the 2nd wave. Hydroxychloroquine and azithromycin that were commonly used during the 1st wave (21.7% and 30.9%, respectively) were almost abandoned during the 2nd. In contrast, the use of high dose corticosteroids doubled (19.5% vs. 41.6%, p&lt;0.001). Despite these changing trends in COVID-19 management, 60-day mortality was not statistically different between the 2 waves (25.3% vs. 23.9%; Log Rank, p=0.48).</p><p>We conclude that changing of therapeutic trends during 2020 did not reduce COVID-19 related mortality in KTR. Our data indirectly support the importance of vaccination and monoclonal neutralizing anti-SARS-CoV-2 antibodies to protect KTR from severe COVID-19.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472627', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Clustering and visualization of single-cell RNA-seq data using path metrics', 'authors': \"[{'name': 'Andriana Manousidaki'}, {'name': 'Anna Little'}, {'name': 'Yuying Xie'}]\", 'abstract': '<p>Recent advances in single-cell technologies have enabled high-resolution characterization of tissue and cancer compositions. Although numerous tools for dimension reduction and clustering are available for single-cell data analyses, these methods often fail to simultaneously preserve local cluster structure and global data geometry. This article explores the application of power-weighted path metrics for the analysis of single cell RNA data.\\nExtensive experiments on single cell RNA sequencing data sets confirm the usefulness of path metrics for dimension reduction and clustering. Distances between cells are measured in a data- driven way which is both density sensitive (decreasing distances across high density regions) and respects the underlying data geometry. By combining path metrics with multidimensional scaling, a low dimensional embedding of the data is obtained which respects both the global geometry of the data and preserves cluster structure. We evaluate the method both for clustering quality and geometric fidelity, and it outperforms other algorithms on a wide range of bench marking data sets.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.21267943', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'The quorum sensing com system regulates pneumococcal colonisation and invasive disease in a pseudo-stratified airway tissue model', 'authors': \"[{'name': 'Christian R. Kahlert'}, {'name': 'Susanne Nigg'}, {'name': 'Lucas Onder'}, {'name': 'Ronald Dijkman'}, {'name': 'Liliane Diener'}, {'name': 'Regulo Rodriguez'}, {'name': 'Pietro Vernazza'}, {'name': 'Volker Thiel'}, {'name': 'Jorge E. Vidal'}, {'name': 'Werner C. Albrich'}]\", 'abstract': '<p>Streptococcus pneumoniae (Spn) colonises respiratory epithelia but can also invade lung cells causing pneumonia. We developed an ex vivo model with human airway epithelial (HAE) cells harvested from lung biopsies to study Spn colonisation and translocation. Flow-cytometry, confocal imaging and electron microscopy studies identified the epithelial lineage with signs of differentiation (beating cilia, mucus, and tight junctions). HAE cells were challenged with Spn wild-type TIGR4 (wtSpn) or its isogenic comC quorum sensing-deficient mutant. comC mutant colonised significantly less than wtSpn at 6 h post-inoculation but at significantly higher levels at 19 h and 30 h. Translocation correlated inversely with colonisation density. Transepithelial electric resistance (TEER) decreased after pneumococcal infection and correlated with increased translocation for both strains.\\nConfocal imaging illustrated colocalisation of intracellular Spn with both cilia and zonulin-1 and prominent microcolony formation with wtSpn but disintegration of microcolony structures over time with comC mutant. comC caused a more pronounced release of both zonulin-1 and lactate dehydrogenase into the supernatant at later time points than wtSpn, suggesting that cytotoxicity is likely not the mechanism leading to translocation. There was a density- and time-dependent increase of inflammatory cytokines from human HAE cells infected with comC compared with wtSpn, including increased levels of the NLRP3 inflammasome-related IL-18. \\nIn conclusion, our experiments indicate that ComC system allows a higher organisational level of population structure resulting in microcolony formation, increased early colonisation and subsequent translocation. We propose that ComC inactivation unleashes a very different and possibly more virulent phenotype that merits further investigation.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.472958', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Real-time imaging of Arc/Arg3.1 transcription ex vivo reveals input-specific immediate early gene dynamics', 'authors': \"[{'name': 'Pablo J Lituma'}, {'name': 'Robert H Singer'}, {'name': 'Sulagna Das'}, {'name': 'Pablo E Castillo'}]\", 'abstract': '<p>The ability of neurons to process and store salient environmental features underlies information processing in the brain. Long-term information storage requires synaptic plasticity and regulation of gene expression. While distinct patterns of activity have been linked to synaptic plasticity, their impact on immediate early gene (IEG) expression remains poorly understood. The activity regulated cytoskeleton associated (Arc) gene has received wide attention as an IEG implicated in synaptic plasticity and memory. Yet, to date, the transcriptional dynamics of Arc in response to compartment and input-specific activity is unclear. By developing a knock-in mouse to fluorescently tag Arc alleles, we studied real-time transcription dynamics after stimulation of dentate granule cells (GCs) in acute hippocampal slices. To our surprise, we found that Arc transcription displayed distinct temporal kinetics depending on the activation of excitatory inputs that convey functionally distinct information, i.e. medial and lateral perforant paths (MPP and LPP, respectively). Moreover, the transcriptional dynamics of Arc after synaptic stimulation was similar to direct activation of GCs, although the contribution of ionotropic glutamate receptors, L-type voltage gated calcium channel, and the endoplasmic reticulum (ER) differed. Specifically, we observed an ER-mediated synapse-to-nucleus signal that supported elevations in nuclear calcium, and rapid induction of Arc transcription following MPP stimulation. However, activation of LPP inputs displayed lower nuclear calcium rise, which could underlie the delayed transcriptional onset of Arc. Our findings highlight how input-specific activity distinctly impacts transcriptional dynamics of an IEG linked to learning and memory.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.21267740', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Determinants of RSV epidemiology following suppression through pandemic contact restrictions', 'authors': \"[{'name': 'Mihaly Koltai'}, {'name': 'Fabienne Krauer'}, {'name': 'David Hodgson'}, {'name': 'Edwin van Leeuwen'}, {'name': 'Marina Treskova-Schwarzbach'}, {'name': 'Mark Jit'}, {'name': 'Stefan Flasche'}]\", 'abstract': '<sec><title>Introduction</title><p>COVID-19 related non-pharmaceutical interventions (NPIs) led to a suppression of RSV circulation in winter 2020/21 throughout Europe and an off-season resurgence in Summer 2021 in several European countries. We explore how such temporary interruption may shape future RSV epidemiology and what factors drive the associated uncertainty.</p></sec><sec><title>Methods</title><p>We developed an age-structured dynamic transmission model to simulate pre-pandemic RSV infections and hospitalisations. We sampled parameters governing RSV seasonality, immunity acquisition and duration of post-infection immunity and retained those simulations that qualitatively fit the UKs pre-pandemic epidemiology. From Spring 2020 to Summer 2021 we assumed a 50% reduced contact frequency, returning to pre-pandemic levels from mid-May 2021. We simulated transmission forwards until 2023 and evaluated the impact of the sampled parameters on the projected trajectories of RSV hospitalisations.</p></sec><sec><title>Results</title><p>Following a lifting of contact restrictions in summer 2021 the model replicated an out-of-season resurgence of RSV. If unmitigated, paediatric RSV hospitalisation incidence in the 2021/22 season was projected to increase by 32% to 67% compared to pre-pandemic levels. The size of the increase depended most on whether infection risk was primarily determined by immunity acquired from previous exposure or general immune maturation. While infants were less affected, the increase in seasonal hospitalisation incidence exceeded 100% in 1-2 year old children and 275% in 2-5 year old children, respectively, in some simulations where immunity from previous exposure dominated. Consequently, the average age of a case increased by 1 to 5 months, most markedly if there was strong immunity acquisition from previous exposure. If immunity to infection was largely determined by age rather than previous exposure, the 2021/22 season started earlier and lasted longer but with a peak incidence lower or similar to pre-pandemic levels. For subsequent seasons, simulations suggested a quick return to pre-pandemic epidemiology, with some slight oscillating behaviour possible depending on the strength of post-exposure immunity.</p></sec><sec><title>Conclusion</title><p>COVID-19 mitigation measures stopped RSV circulation in the 2020/21 season and generated immunity debt that will likely lead to a temporary increase in RSV burden in the season following the lifting of restrictions, particularly in 1 to 5 year old children. A more accurate understanding of immunity drivers for RSV is needed to better predict the size of such an increase and plan a potential expansion of pharmaceutical and non-pharmaceutical mitigation measures.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.10.472105', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Following the track: accuracy and reproducibility of predation assessment on artificial caterpillars', 'authors': \"[{'name': 'Elena Valds Correcher'}, {'name': 'Elina Mntyl'}, {'name': 'Luc Barbaro'}, {'name': 'Thomas Damestoy'}, {'name': 'Katerina Sam'}, {'name': 'Bastien Castagneyrol'}]\", 'abstract': '<p>Experimental studies of biotic interactions in real field conditions are essential to understand the structure and functioning of ecological networks. The use of artificial caterpillars to mimic actual prey availability is generally seen as a standard approach to compare the activity and diversity of predators along environmental gradients. Yet, even with standardized material and procedures, biases may still affect data from multiple observers with different expertise. We used pictures of artificial caterpillars with or without various predation marks, in an online survey that was targeted for the participants of the project, to evaluate the reliability of predation marks identification made by non-scientists and by scientists with and without previous experience in predation mark identification. Pictures of artificial caterpillars displayed typical marks left by birds, mammals and arthropods, as well as non-predation marks (false positive). 357 respondents scanned 7140 pictures of these pictures. Self-declared scientists were more confident and accurate in their observations than non-scientists, but the differences in correct identifications among scientists and non-scientists were low. Self-declared scientists with previous experience were also more accurate than scientists without previous experience, while there were no differences in self-confidence among scientists with and without previous experience. Accuracy in predation mark identification did not differ among types of predators, but respondents were more keen to identify marks left by birds or mammals than arthropods. Our results have practical implications for the design of multi-observer projects relying on artificial caterpillars as a proxy to assess predation intensity, in particular in the context of citizen science.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.10.21267630', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Polygenic adaptation is not a major driver of disparities in disease mortality across global populations', 'authors': \"[{'name': 'Ujani Hazra'}, {'name': 'Joseph Lachance'}]\", 'abstract': '<sec><title>Background and objectives</title><p>Health disparities are due to a range of socioeconomic and biological causes, and many common diseases have a genetic basis. Divergent evolutionary histories cause allele frequencies at disease-associated loci to differ across global populations. To what extent are differences in disease risks due to natural selection?</p></sec><sec><title>Methodology</title><p>Examining a panel of nine global populations, we identified which of the 20 most common causes of death have the largest health disparities. Polygenic risk scores were computed and compared for 11 common diseases for the same nine populations. We then used <italic>PolyGraph</italic> to test whether differences in disease risk can be attributed to polygenic adaptation. Finally, we compared human development index statistics and polygenic risk scores to mortality rates for each population.</p></sec><sec><title>Results</title><p>Among common causes of death, HIV/AIDS and tuberculosis exhibited the greatest disparities in mortality rates. Focusing on common polygenic diseases, we found that genetic predictions of disease risk varied across global populations (including elevated risks of lung cancer in Europeans). However, polygenic adaptation tests largely yielded negative results when applied to common diseases. Our analyses revealed that natural selection was not a major cause of differences in disease risks across global populations. We also found that correlations between mortality rates and human development index statistics were stronger than correlations between mortality rates and polygenic predictions of disease risks.</p></sec><sec><title>Conclusions and implications</title><p>Although evolutionary history contributes to differences in disease risks, health disparities are largely due to socioeconomic and other environmental factors.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.472971', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A high-throughput neutralization assay for yellow fever serodiagnostics', 'authors': \"[{'name': 'Madina Rasulova'}, {'name': 'Thomas Vercruysse'}, {'name': 'Jasmine Paulissen'}, {'name': 'Catherina Coun'}, {'name': 'Vanessa Suin'}, {'name': 'Leo Heyndrickx'}, {'name': 'Ji Ma'}, {'name': 'Katrien Geerts'}, {'name': 'Jolien Timmermans'}, {'name': 'Niraj Mishra'}, {'name': 'Li-Hsin Li'}, {'name': 'Dieudonn Buh Kum'}, {'name': 'Lotte Coelmont'}, {'name': 'Steven Van Gucht'}, {'name': 'Hadi Karimzadeh'}, {'name': 'Julia Thorn-Seshold'}, {'name': 'Simon Rothenfuer'}, {'name': 'Kevin K. Arin'}, {'name': 'Johan Neyts'}, {'name': 'Kai Dallmeier'}, {'name': 'Hendrik Jan Thibaut'}]\", 'abstract': '<p>Quick and accurate detection of neutralizing antibodies (nAbs) against yellow fever is essential in  serodiagnosis during outbreaks, for surveillance and to evaluate vaccine efficacy in population-wide studies. All this requires serological assays that can process a large number of samples in a highly standardized format. Albeit being laborious, time-consuming and limited in throughput, classical plaque reduction neutralization test (PRNT) is still considered gold standard for the detection and quantification of nAbs due to its sensitivity and specificity. Here we report the development of an alternative fluorescence-based serological assay (SNTFLUO) with an equally high sensitivity and specificity that is fit for high-throughput testing with the potential for automation. Finally, our novel SNTFLUO was cross-validated in several reference laboratories and against international WHO standards showing its potential to be implemented in clinical use. SNTFLUO assays with similar performance are available for the Japanese encephalitis, Zika and dengue viruses amenable for differential diagnostics.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.472952', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'The zinc-finger transcription factor LSL-1 is a major regulator of the germline transcriptional program in C. elegans', 'authors': \"[{'name': 'David Rodriguez-Crespo'}, {'name': 'Magali Nanchen'}, {'name': 'Shweta Rajopadhye'}, {'name': 'Chantal Wicky'}]\", 'abstract': '<p>Specific gene transcriptional programs are required to ensure proper proliferation and differentiation processes underlying the production of specialized cells during development. Gene activity is mainly regulated by the concerted action of transcription factors and chromatin proteins. In the nematode <italic>C. elegans</italic>, mechanisms that silence improper transcriptional programs in germline and somatic cells have been well studied, however, how are tissue specific sets of genes turned on is less known. LSL-1 is herein defined as a novel crucial transcriptional regulator of germline genes in <italic>C. elegans</italic>. LSL-1 is first detected in the P4 blastomere and remains present at all stages of germline development, from primordial germ cell proliferation to the end of meiotic prophase. <italic>lsl-1</italic> loss-of-function mutants exhibit many defects including meiotic prophase progression delay, a high level of germline apoptosis, and production of almost no functional gametes. Transcriptomic analysis and ChIP-seq data show that LSL-1 binds to promoters and acts as a transcriptional activator of germline genes involved in various processes, including homologous chromosome pairing, recombination, and genome stability. Furthermore, we show that LSL-1 functions by antagonizing the action of the heterochromatin proteins HPL-2/HP1 and LET-418/Mi2 known to be involved in the repression of germline genes in somatic cells. Based on our results, we propose LSL-1 to be a major regulator of the germline transcriptional program during development.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472732', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Pharmacokinetics and Pharmacodynamics of Vancomycin derivative LYSC98 in a Murine Thigh Infection Model against Staphylococcus aureus', 'authors': \"[{'name': 'Peng He'}, {'name': 'Xin Li'}, {'name': 'Xiaohan Guo'}, {'name': 'Xingchen Bian'}, {'name': 'Meiqing Feng'}]\", 'abstract': '<p><bold>LYSC98</bold> is a vancomycin derivative used for gram-positive bacterial infections therapy. We reported the pharmacokinetic/pharmacodynamic (PK/PD) targets of LYSC98 against <italic>Staphylococcus aureus</italic> using a murine thigh infection model. Three <italic>Staphylococcus aureus</italic> strains were utilized. Single-dose plasma pharmacokinetics of LYSC98 were determined in infected mice after the tail vein injection of 2, 4, and 8mg/kg. The results showed maximum plasma concentration (C<sub>max</sub>) 11466.67 - 48866.67 ng/mL, area under the concentration-time curve from 0 to 24 h(AUC<sub>0-24</sub>) 14788.42 -91885.93 ng/mLh, and elimination half-life(T<sub>1/2</sub>) 1.70-2.64 h, respectively. The C<sub>max</sub> (R<sup>2</sup> 0.9994) and AUC<sub>0-24</sub> (R<sup>2</sup> 0.981) were positively correlated with the dose of LYSC98 in the range of 2-8 mg/kg.</p><p>Dose fractionation studies using total doses of 2 to 8 mg/kg administered with q6h, q8h, q12h, and q24h were performed to evaluate the correlation of different PK/PD indices with efficacy. Sigmoid model analysis showed C<sub>max</sub>/MIC (R<sup>2</sup> 0.8941) was the best PK/PD index to predict the efficacy of LYSC98. In the dose ranging studies, two Methicillin-resistant <italic>Staphylococcus aureus</italic> (MRSA) clinical strains were used to infect the mice and 2-fold-increasing doses (1 to 16 mg/kg) of LYSC98 were administered. The magnitude of LYSC98 C<sub>max</sub>/MIC associated with net stasis, 1, 2, 3 and 4 - log10 kill were 5.78, 8.17, 11.14, 15.85 and 30.58, respectively. The results of this study showed LYSC98 a promising antibiotic with in vivo potency against MRSA, and will help in the dose design of phase one study for LYSC98.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.21267816', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'COVID-19 and Indian population: a comparative genetic analysis', 'authors': \"[{'name': 'Saurabh Patil'}, {'name': 'Sandhya Kiran Pemmasani'}, {'name': 'Neelima Chitturi'}, {'name': 'Ishita Bhatnagar'}, {'name': 'Anuradha Acharya'}, {'name': 'Lingareddy Venkata Subash'}]\", 'abstract': '<sec><title>Background</title><p>Major risk factors of COVID-19 include older age, male gender, and comorbidities. In addition, host genetic makeup is also known to play a major role in COVID-19 susceptibility and severity. To assess the genetic predisposition of the Indian population to COVID-19, a comparative analysis of the frequencies of polymorphisms directly or potentially associated with COVID-19 susceptibility, severity, immune response, and fatal outcomes was done between the Indian population and other major populations (European, African, East Asian, South Asian, and American).</p></sec><sec><title>Materials and methods</title><p>Polymorphisms directly or potentially associated with COVID-19 susceptibility, severity, immune response, and mortality were mined from genetic association studies, comparative genetic studies, expression quantitative trait loci studies among others. Genotype data of these polymorphisms were either sourced from the GenomegaDB database of Mapmygenome India Ltd. (sample size = 3054; Indian origin) or were imputed. Polymorphisms with minor allele frequency &gt;= 0.05 and that are in Hardy-Weinberg equilibrium in the Indian population were considered for allele frequency comparison between the Indian population and 1000 Genome population groups.</p></sec><sec><title>Results</title><p>Allele frequencies of 421 polymorphisms were found to be significantly different in the Indian population compared to European, African, East Asian, South Asian, and American populations. 128 polymorphisms were shortlisted based on linkage disequilibrium and were analyzed in detail. Apart from well-studied genes, like ACE2, TMPRSS2, ADAM17, and FURIN, variants from AHSG, IFITM3, PTPN2, CD209, CCL5, HEATR9, SELENBP9, AGO1, HLA-G, MX1, ICAM3, MUC5B, CRP, C1GALT1, and other genes were also found to be significantly different in Indian population. These variants might be implicated in COVID-19 susceptibility and progression.</p></sec><sec><title>Conclusion</title><p>Our comparative study unraveled multiple genetic variants whose allele frequencies were significantly different in the Indian population and might have a potential role in COVID-19 susceptibility, its severity, and fatal outcomes. This study can be very useful for selecting candidate genes/variants for future COVID-19 related genetic association studies.</p></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.473094', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genomic characteristics of recently recognized Vibrio cholerae El Tor lineages associated with cholera in Bangladesh, 1991-2017', 'authors': \"[{'name': 'Md Mamun Monir'}, {'name': 'Talal Hossain'}, {'name': 'Masatomo Morita'}, {'name': 'Makoto Ohnishi'}, {'name': 'Fatema-tuj Johura'}, {'name': 'Marzia Sultana'}, {'name': 'Shirajum Monira'}, {'name': 'Tahmeed Ahmed'}, {'name': 'Nicholas Thomson'}, {'name': 'Haruo Watanabe'}, {'name': 'Anwar Huq'}, {'name': 'Rita R. Colwell'}, {'name': 'Kimberley D Seed'}, {'name': 'Munirul Alam'}]\", 'abstract': '<p>Comparative genomic analysis of Vibrio cholerae El Tor associated with endemic cholera in Asia revealed two distinct lineages, one dominant in Bangladesh and the other in India. An in depth whole genome study of V. cholerae El Tor clinical strains isolated during endemic cholera in Bangladesh (1991  2017) included reference genome sequence data obtained online. Core genome phylogeny established using single nucleotide polymorphisms (SNPs) showed V. choleraeEl Tor strains comprised two lineages, BD-1 and BD-2, which, according to Bayesian phylodynamic analysis, originated from paraphyletic group BD-0 around 1981. BD-1 and BD-2 lineages overlapped temporally but were negatively associated as causative agents of cholera 2004-2017. Genome wide association study (GWAS) revealed 140 SNPs and 31 indels, resulting in gene alleles unique to BD-1 and BD-2. Regression analysis of root to tip distance and year of isolation indicated early BD-0 strains at the base, whereas BD-1 and BD-2 subsequently emerged and progressed by accumulating SNPs. Pangenome analysis provided evidence of gene acquisition by both BD-1 and BD-2, of which six crucial proteins of known function were predominant in BD-2. BD-1 and BD-2 diverged and have distinctively different genomic traits, namely heterogeneity in VSP-2, VPI-1, mobile elements, toxin encoding elements, and total gene abundance. In addition, the observed phage-inducible chromosomal island-like element (PLE1), and SXT ICE elements (ICE<sup>TET</sup>) in BD-2 presumably provided a fitness advantage for the lineage to outcompete BD-1 as the etiological agent of the endemic cholera in Bangladesh, with implications for global cholera epidemiology.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.15.472107', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Attenuated transcriptional response to pro-inflammatory cytokines in schizophrenia hiPSC-derived neural progenitor cells', 'authors': \"[{'name': 'Anjali Bhat'}, {'name': 'Haritz Irizar'}, {'name': 'Pooja Raval'}, {'name': 'Rodrigo RR Duarte'}, {'name': 'Lucia Dutan Polit'}, {'name': 'Timothy Powell'}, {'name': 'P.J. Michael Deans'}, {'name': 'Carol Shum'}, {'name': 'Roland Nagy'}, {'name': 'Grainne McAlonan'}, {'name': 'Conrad O Iyegbe'}, {'name': 'Jack Price'}, {'name': 'Elvira Bramon'}, {'name': 'Sagnik Bhattacharyya'}, {'name': 'Anthony C Vernon'}, {'name': 'Deepak P Srivastava'}]\", 'abstract': '<p>Maternal immune activation (MIA) during prenatal development is an environmental risk factor for psychiatric disorders including schizophrenia (SZ). This link is particularly well established in the case of schizophrenia. Converging lines of evidence from human and animal model studies have suggested that elevated cytokine levels in the maternal and fetal compartments are an important indication of the mechanisms driving this association. However, there is variability in susceptibility to the psychiatric risk conferred by MIA, likely influenced by genetic factors. How MIA interacts with a genetic profile susceptible to SZ is challenging to test in animal models. To address this gap, we examined whether differential gene expression responses occur in forebrain-lineage neural progenitor cells (NPCs) derived from human induced pluripotent stem cells (hiPSC) generated from three individuals with a diagnosis of schizophrenia and three healthy controls. Following acute (24 hour) treatment with either interferon-gamma (IFN; 25 ng/l) or interleukin (IL)-1 (10 ng/l), we identified, by RNA sequencing, 3380 differentially expressed genes (DEGs) in the IFN-treated control lines (compared to untreated controls), and 1980 DEGs in IFN-treated SZ lines (compared to untreated SZ lines). Of these, 359 genes responded differently to IFN treatment in SZ relative to control lines, which were enriched for genes differentially expressed in adult SZ cases and in pathways related to the regulation of antigen processing, neuronal post-synaptic density, and the presynapse. There were no differentially expressed genes in the IL-1-treatment conditions after Benjamini-Hochberg correction. Gene set enrichment analysis however showed that IL-1 impacts immune function and neuronal differentiation. Collectively, our data suggest that a) SZ NPCs show an attenuated transcriptional response to IFN treatment compared to control NPCs; b) IL-1 may be less relevant to NPC cultures than IFN and c) the genes differentially regulated in schizophrenia lines are primarily mitochondrial, loss-of-function, pre- and post-synaptic genes. Our findings particularly highlight early synaptic development as a key target for future research into the mechanisms of association between maternal immune activation and schizophrenia risk.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472647', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Attention enhances category representations across the brain with strengthened residual correlations to ventral temporal cortex', 'authors': \"[{'name': 'Arielle S Keller'}, {'name': 'Akshay V Jagadeesh'}, {'name': 'Lior Bugatus'}, {'name': 'Leanne M Williams'}, {'name': 'Kalanit Grill-Spector'}]\", 'abstract': \"<p>How does attention enhance neural representations of goal-relevant stimuli while suppressing representations of ignored stimuli across regions of the brain? While prior studies have shown that attention enhances visual responses, we lack a cohesive understanding of how selective attention modulates visual representations across the brain. Here, we used functional magnetic resonance imaging (fMRI) while participants performed a selective attention task on superimposed stimuli from multiple categories and used a data-driven approach to test how attention affects both decodability of category information and residual correlations (after regressing out stimulus-driven variance) with category-selective regions of ventral temporal cortex (VTC). Our data reveal three main findings. First, when two objects are simultaneously viewed, the category of the attended object can be decoded more readily than the category of the ignored object, with the greatest attentional enhancements observed in occipital and temporal lobes. Second, after accounting for the response to the stimulus, the correlation in the residual brain activity between a cortical region and a category-selective region of VTC was elevated when that region's preferred category was attended vs. ignored, and more so in the right occipital, parietal, and frontal cortices. Third, we found that the stronger the residual correlations between a given region of cortex and VTC, the better visual category information could be decoded from that region. These findings suggest that heightened residual correlations by selective attention may reflect the sharing of information between sensory regions and higher-order cortical regions to provide attentional enhancement of goal-relevant information.</p>\", 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.17.472408', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Fully Automatic Cell Segmentation with Fourier Descriptors', 'authors': \"[{'name': 'Dominik Hirling'}, {'name': 'Peter Horvath'}]\", 'abstract': '<p>Cell segmentation is a fundamental problem in biology for which convolutional neural networks yield the best results nowadays. In this paper, we present HarmonicNet, a network, which is a modification of the popular StarDist and SplineDist architectures. While StarDist and SplineDist describe an object by the lengths of equiangular rays and control points respectively, our network utilizes Fourier descriptors, predicting a coefficient vector for every pixel on the image, which implicitly define the resulting segmentation. We evaluate our model on three different datasets, and show that Fourier descriptors can achieve a high level of accuracy with a small number of coefficients. HarmonicNet is also capable of accurately segmenting objects that are not star-shaped, a case where StarDist performs suboptimally according to our experiments.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.15.472741', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Age determination through DNA methylation patterns of fingernails and toenails', 'authors': \"[{'name': 'Kristina Fokias'}, {'name': 'Lotte Dierckx'}, {'name': 'Wim Van de Voorde'}, {'name': 'Bram Bekaert'}]\", 'abstract': '<p>Over the past decade, age prediction based on DNA methylation has become a vastly investigated topic; many age prediction models have been developed based on different DNAm markers and using various tissues. However, the potential of using nails to this end has not yet been explored. Their inherent resistance to decay and ease of sampling would offer an advantage in cases where post-mortem degradation poses challenges concerning sample collection and DNA-extraction. In the current study, clippings from both fingernails and toenails were collected from 108 living test subjects (age range: 0  96 years). The methylation status of 15 CpGs located in 4 previously established age-related markers (ASPA, EDARADD, PDE4C, ELOVL2) was investigated through pyrosequencing of bisulphite converted DNA. Significant dissimilarities in methylation levels were observed between all four limbs, hence limb-specific age prediction models were developed using ordinary least squares, weighted least squares and quantile regression analysis. When applied to their respective test sets, these models yielded a mean absolute deviation between predicted and chronological age ranging from 6.71 to 8.48 years. In addition, the assay was tested on methylation data derived from 5 nail samples collected from deceased individuals, demonstrating its feasibility for application in post-mortem cases. In conclusion, this study provides the first proof that chronological age can be assessed through DNA methylation patterns in nails.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472743', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Nest aggregations of wild bees and apoid wasps in urban pavements: a street life to be promoted in urban planning', 'authors': \"[{'name': 'Gregoire Noel'}, {'name': 'Violette Van Keymeulen'}, {'name': 'Yvan Barbier'}, {'name': 'Sylvie Smets'}, {'name': 'Olivier Van Damme'}, {'name': 'Gilles Colinet'}, {'name': 'Julien Ruelle'}, {'name': 'Frederic Francis'}]\", 'abstract': '<p>In the last 10 years, knowledges of wild bees and apoid wasps community dynamics have gained interest in urban ecology focusing on the availability of floral resources in cities. Although understudied, the urban environment impacts the conditions of their nesting sites. Recent observations in the Brussels-Capital Region (Belgium) showed that urban pavements can be a novel nesting opportunity for Hymenoptera ground-nesting species such as wild bees and apoid wasps. Here, using citizen science, we investigated the richness of ground-nesting species living under urban pavements, the preferences of the sidewalk joint size related to ground-nesting species size and for sidewalk type or for soils texture under the pavements on the nesting site selection. A total of 22 species belonging to 10 Hymenoptera families of wild bees and digger wasps with their associated kleptoparasites were identified on 89 sites in Brussels. Sandstone setts or concrete slabs with an unbound joint size around 1 cm were found to be best suitable urban pavements for the ground-nesting species. The soil texture under the pavement was highly sandy among our samples. Finally, we also suggest engineering management guidelines to support bee and wasp species nesting under urban pavement in highly urbanized areas. Such observations pave the way for much research in the field of urban ecology to conceive multifunctional pavement promoting biodiversity.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.473079', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Assessment and Comparative Study of Biofilm Formation with frequency of Multi Drug Resistance in strains of Staphylococcus aureus', 'authors': \"[{'name': 'Kiran Fatima'}, {'name': 'Kashif Ali'}]\", 'abstract': \"<p>Background: The study was conducted to identify the role of biofilms in the antibiotic susceptibility in the strains of Staphylococcus aureus. A total of 19 non repeated pus/wound swab samples from different anatomic locations and 17 samples that were previously identified as S. aureus and preserved in the labs were included in the study. The Staphylococcus aureus was identified on the basis of colony morphology, Gram's stain, biochemical tests (catalase and coagulase tests) and molecular identification through PCR amplification. Methodology: A total of 26 samples were recovered out of the 31 samples. Kirby-Bauer disk diffusion susceptibility test was used to determine the sensitivity or resistance of S. aureus to methicillin. Out of the 26 strains, 4 were highly resistant, 10 were moderately resistant and 12 strains were sensitive. Three different protocols (Tube Method, Congo Red Agar Method and Tissue Culture plate method) were used for the detection of biofilm formation for both resistant and sensitive strains. Result: Comparative analysis of the antibiotic susceptibility and biofilm formation by different protocols showed that 70% strains that are resistant to antibiotic methicillin produced moderate-strong biofilms. 50% have produced the moderate-strong biofilms in all 3 protocols. In case of sensitive, 50% strains had produced none-weak biofilms in all 3 protocols. Decisions: The strains that had zone of inhibition of close to resistance produced weak-strong biofilms but they all produced weak biofilms in CRA method. It can be concluded that the strains of S. aureus that have the ability to produce biofilms become methicillin resistant. Keywords: Biofilm, antibiotic susceptibility, Congo Red Agar, Tube Method, Tissue culture plats.</p>\", 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.21267783', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Dynamics of anti-Spike IgG antibody level after full BNT162b2 COVID-19 vaccination in health care workers', 'authors': \"[{'name': 'Hiroaki Ikezaki'}, {'name': 'Ryoko Nakashima'}, {'name': 'Kahori Miyoshi'}, {'name': 'Yuichi Hara'}, {'name': 'Jun Hayashi'}, {'name': 'Hiroshi Hara'}, {'name': 'Hideyuki Nomura'}, {'name': 'Nobuyuki Shimono'}]\", 'abstract': '<sec><title>Introduction</title><p>The administration of a third vaccine is ongoing in many countries, but the evaluation of vaccine-induced immunity is still insufficient. This study evaluated anti-spike IgG levels in 373 health care workers six months after the BNT162b2 vaccination.</p></sec><sec><title>Methods</title><p>Dynamics of anti-spike IgG levels six months after the 2nd vaccination were assessed in 49 participants (Analysis-1). A cross-sectional assessment of anti-spike IgG level was performed in 373 participants (Analysis-2). Participants positive for anti-nucleocapsid IgG or IgM and receiving immunosuppressants were excluded from Analysis-2.</p></sec><sec><title>Results</title><p>In Analysis 1, the median anti-spike IgG level was lower in the older age group and decreased consistently after the second vaccination regardless of age. In Analysis-2, the anti-spike IgG level was significantly negatively associated with age (r = 0.35, p &lt; 0.01). This correlation remained statistically significant (r = 0.28, p &lt; 0.01) even after adjusting for sex, BMI, smoking habits, alcohol drinking habits, allergies, and the presence of fever or other adverse reactions at the time of the vaccination. Alcohol drinking habit was also associated with the anti-spike IgG level; daily alcohol drinkers had significantly lower anti-spike IgG levels than never alcohol drinkers. Sex, smoking habit, allergy, and fever and other side effects after the vaccination were not associated with anti-spike IgG levels six months after the 2nd vaccination.</p></sec><sec><title>Conclusions</title><p>Six months after the vaccination, the anti-spike IgG level was substantially low among older persons and daily alcohol drinkers.</p></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472435', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Spike initiation properties in the axon support high-fidelity signal transmission', 'authors': \"[{'name': 'Mohammad Amin Kamaleddin'}, {'name': 'Nooshin Abdollahi'}, {'name': 'Stphanie Ratt'}, {'name': 'Steven A Prescott'}]\", 'abstract': '<p>The axon initial segment (AIS) converts graded depolarization into all-or-none spikes that are transmitted by the axon to downstream neurons. Analog-to-digital transduction and digital signal transmission call for distinct spike initiation properties (filters) and those filters should, therefore, differ between the AIS and distal axon. Here we show that unlike the AIS, which spikes repetitively during sustained depolarization, the axon spikes transiently and only if depolarization reaches threshold before K<sub>V</sub>1 channels activate. Rate of depolarization is critical. This was shown by optogenetically evoking spikes in the distal axon of CA1 pyramidal neurons using different photostimulus waveforms and pharmacological conditions while recording antidromically propagated spikes at the soma, thus circumventing the prohibitive difficulty of patching intact axons. Computational modeling shows that K<sub>V</sub>1 channels in the axon implement a high-pass filter that is matched to the axial current waveform associated with spike propagation, thus maximizing the signal-to-noise ratio to ensure high-fidelity transmission of spike-based signals.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.473004', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Galvanotaxis of ciliates: Spatiotemporal dynamics of Coleps hirtus under electric fields', 'authors': \"[{'name': 'Anna Daul'}, {'name': 'Marie-Louise Lemloh'}, {'name': 'Marcel Hrning'}]\", 'abstract': '<p>Galvanotaxis describes the functional response of organisms to electric fields. In ciliates, the electric field influences the electrophysiology and thus the cilia beat dynamics. This leads to a change of the swimming direction towards the cathode. The dynamical response to electric fields of Coleps hirtus has not been studied since the observations of Verworn in 1890 (1). While galvanotaxis has been studied in other cilitates, C. hirtus exhibit properties not found elsewhere, such as biomineralization-processes of alveolar plates with impact on the intracellular calcium regulation and a bimodal resting membrane potential, which leads unique electrophysiological driven bimodal swimming dynamics. Here, we statistically analyze the galvanotactic dynamics of C. hirtus by automated cell tracking routines. We found that the number of cells that show a galvanotactic response, increases with the increase of the applied electric field strength with a mean at about 2.1 V/cm. The spatiotemporal swimming dynamics change and lead to a statistical increase of linear elongated cell trajectories that point toward the cathode. Further, the increase of the electric fields decreases the mean velocity variance for electric fields larger than about 1.3 V/cm, while showing no significant change in the absolute velocity for any applied electric field. Fully functional galvanotactic responses were observed at a minimum extracellular calcium concentration of 20 M. The results add important insights to the current understanding of cellular dynamics of ciliates and suggest that the currently accepted model lags the inclusion of the swimming dynamics and the complex calcium regulatory system of the cell. The results of this study do not only extend the fundamental understanding of C. hirtus dynamics, but also open possibilities for technical applications, such as biosensors or microrobots in the future.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.21267696', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Impact Of an Immune Modulator Mycobacterium-w On Adaptive Natural Killer Cells and Protection Against COVID-19', 'authors': \"[{'name': 'Sarita Rani Jaiswal'}, {'name': 'Jaganath Arunachalam'}, {'name': 'Ashraf Saifullah'}, {'name': 'Anupama Mehta'}, {'name': 'Gitali Bhagawati'}, {'name': 'Rohit Lakhchaura'}, {'name': 'Hemamalini Aiyer'}, {'name': 'Bakulesh Khamar'}, {'name': 'Suparno Chakrabarti'}]\", 'abstract': '<p>The kinetics of NKG2C<sup>+</sup> adaptive natural killer (ANK) cells and NKG2A<sup>+</sup>inhibitory NK (iNK) cells with respect to the incidence of SARS-CoV-2 infection were studied in a cohort of health-care workers following administration of heat killed <italic>Mycobacterium w</italic> (Mw group) in comparison to a control group. COVID-19 infection correlated with a lower NKG2C<sup>+</sup>ANK cells at baseline. NKG2C<sup>+</sup>ANK cells at baseline did not differ, but there was a significant upregulation of NKG2C expression and cytokine release in the Mw group (p=0.0009), particularly in those with lower baseline NKG2C expression (&lt;15%), along with marked downregulation of NKG2A<sup>+</sup>iNK cells (p&lt;0.0001), and an increase in the NKG2C<sup>+</sup>ANK/NKG2A<sup>+</sup>iNK ratio. This translated to a significant reduction in COVID-19 and its severity in the Mw group. No impact was observed on T cell subsets. Mw was observed to have a salutary impact on the ANK cell profile which might have provided protection against COVID-19 over a prolonged period in a non-immune high-risk population.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.471511', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A core salivary microbiome shows the high prevalence of bacterial members yet variability across human populations', 'authors': \"[{'name': 'Xinwei Ruan'}, {'name': 'Jiaqiang Luo'}, {'name': 'Pangzhen Zhang'}, {'name': 'Kate Howell'}]\", 'abstract': '<sec><title>Background</title><p>Human saliva contains diverse bacterial communities, reflecting human health status, dietary patterns and contributing to variability in the sensory perception of food. Many descriptions of salivary microbiome diversity compare commonalities and differences with reference to a diseased state, but the composition of healthy saliva has not been described.</p></sec><sec><title>Results</title><p>Here, we use a meta-analysis approach to define and explore the core membership of the human salivary microbial community by collecting and re-analysing raw 16S rRNA amplicon sequencing data from 47 studies with 2206 saliva samples. We found 68 core bacterial taxa that were consistently detected. Differences induced by various host intrinsic and behaviour factors, including gender, age, geographic location, tobacco usage, and alcohol consumption, were evident. The core of the salivary microbiome was verified by collecting and analysing saliva in an independent study.</p></sec><sec><title>Conclusion</title><p>These results suggest that the methods used can effectively define a core microbial community in human saliva with high prevalence. The core salivary microbiome demonstrated both stability and variability among populations. Geographic location was identified as the host factor with the largest effect in shaping salivary microbiota. The independent analysis underlined that the impact of geographic variation is likely due to diet.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.21267757', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'A systematic literature review of individual-level psychological and behavioral responses to the health information of COVID-19 from social media and legacy media', 'authors': \"[{'name': 'Wei Zhai'}, {'name': 'Jian Bian'}, {'name': 'Jing Zhang'}, {'name': 'Xueyin Bai'}]\", 'abstract': '<p>Covid-19 has been recognized as a terrifying global health threat since its detection, with far-reaching consequences that are unprecedented in the modern era. Since the outbreak of the pandemic, social media and legacy media have collectively delivered health information related to COVID-19 to the public as a catalyst to community perception of risk. However, the existing literature exhibits different viewpoints toward the role of social media and legacy media in disseminating health information of COVID-19. In this regard, this article conducted a systematic literature review to provide an overview of the current state of research concerning individuals-level psychological and behavioral response to COVID-19 related information from different sources, as well as presents the challenges and future research directions.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.21267561', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Identification of Genetic Loci Simultaneously Associated with Multiple Cardiometabolic Traits', 'authors': \"[{'name': 'Alexis C. Wood'}, {'name': 'Amit Arora'}, {'name': 'Michelle Newell'}, {'name': 'Victoria L. Bland'}, {'name': 'Jin Zhou'}, {'name': 'Nicola Pirastu'}, {'name': 'Jose M. Ordovas'}, {'name': 'Yann C. Klimentidis'}]\", 'abstract': '<sec><title>Background and Aims</title><p>Cardiometabolic disorders (CMD) arise from a constellation of features such as increased adiposity, hyperlipidemia, hypertension and compromised glucose control. Many genetic loci have shown associations with individual CMD-related traits, but no investigations have focused on simultaneously identifying loci showing associations across all domains. We therefore sought to identify loci associated with risk across seven continuous CMD-related traits.</p></sec><sec><title>Methods and Results</title><p>We conducted separate genome-wide association studies (GWAS) for systolic and diastolic blood pressure (SBP/DBP), hemoglobin A1c (HbA1c), low- and high-density lipoprotein cholesterol (LDL-C/HDL-C), waist-to-hip-ratio (WHR), and triglycerides (TGs) in the UK Biobank (N=356,574456,823). Multiple loci reached genome-wide levels of significance (N=145-333) for each trait, but only four loci (in/near <italic>VEGFA, GRB14-COBLL1, KLF14</italic>, and <italic>RGS19-OPRL1)</italic> were associated with risk across all seven traits (P&lt;510<sup>8</sup>). We sought replication of these four loci in an independent set of seven trait-specific GWAS meta-analyses. <italic>GRB14-COBLL1</italic> showed the most consistent replication, revealing nominally significant associations (P&lt;0.05) with all traits except DBP.</p></sec><sec><title>Conclusions</title><p>Our analyses suggest that very few loci are associated in the same direction of risk with traits representing the full spectrum of CMD features. We identified four such loci, and an understanding of the pathways between these loci and CMD risk may eventually identify factors that can be used to identify pathologic disturbances that represent broadly beneficial therapeutic targets.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.12.472320', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Biocatalytic Formation of Novel Polyesters with para-Hydroxyphenyl groups in the Backbone  Engineering Cupriavidus necator for production of high-performance materials from CO2 and electricity', 'authors': \"[{'name': 'Nils JH Averesch'}, {'name': 'Vince E Pane'}, {'name': 'Frauke Kracke'}, {'name': 'Marika Ziesack'}, {'name': 'Shannon N Nangle'}, {'name': 'Pamela A Silver'}, {'name': 'Robert W Waymouth'}, {'name': 'Craig S Criddle'}]\", 'abstract': '<p>Synthetic materials are integral components of consumables and durable goods and indispensable in our modern world. Polyesters are the most versatile bulk- and specialty-polymers, but their production is not sustainable, and their fate at end-of-life of great concern. Bioplastics are highly regarded alternatives but have shortcomings in material properties and commercial competitiveness with conventional synthetic plastics. These constraints have limited the success in global markets. Enabling bio-production of advanced bioplastics with superior properties from waste-derived feedstocks could change this.</p><p>We have created microbial cell factories that can produce a range of aliphatic and aromatic polyesters. A <italic>phaC1</italic> mutant of <italic>Cupriavidus necator</italic> H16 was complemented with hydroxyacyl-CoA transferases from either <italic>Clostridium propionicum</italic> (<italic>pct540</italic>) or <italic>Clostridium difficile</italic> (<italic>hadA</italic>), respectively. These were combined with a mutant PHA synthase (<italic>phaC1437</italic>) from <italic>Pseudomonas</italic> sp. MBEL 6-19, which rescued the PHA<sup></sup> phenotype of the knock-out mutant and allowed polymerization of various hydroxy carboxylates, including phloretic acid. This is the first-time, incorporation of an aromatic ring in the backbone of a biological polyester was achieved. Polymers contain <italic>para</italic>-hydroxyphenyl subunits are structurally analogous to synthetic aromatic polyesters like PET and high-strength polyarylates.</p><p>In a further advance, the transgenic strain was cultivated in a bio-electrochemical system under autotrophic conditions, enabling synthesis of aromatic bio-polyesters from H<sub>2</sub> and O<sub>2</sub> generated <italic>in situ</italic>, while assimilating CO<sub>2</sub>. Follow-up elementary flux-mode analysis established the feasibility of <italic>de novo</italic> production of twenty different polyesters from five different carbon- and energy-sources. This comprehensive study opens the door to sustainable bio-production of high-performance thermoplastics and thermosets.</p><sec><title>Significance statement</title><p>New biomaterials can facilitate transition to a carbon-neutral chemical industry and a circular economy while at the same time preventing accumulation of plastic wastes in the environment. This can be accomplished by developing drop-in replacements for existing fossil carbon-based plastics. To that end, this work demonstrates for the first time that biocatalytic polymerization can incorporate an aromatic <italic>para</italic>-hydroxyphenyl carbonic acid into the backbone of a bio-polyester. This was accomplished using a genetically engineered microbial cell factory that assimilates carbon dioxide using hydrogen that is produced electrochemically <italic>in situ</italic>. A bio-electrochemical system eliminates the need for external supply of oxyhydrogen and avoids explosive mixtures, opening the door for sustainable production of biomaterials analogous to aromatic bulk-polyesters such as PET and high-performance liquid-crystal polymers.</p></sec><sec><title>Highlights</title><list list-type=\"simple\"><list-item><p>- Biocatalytic formation of aromatic polyesters with structural analogy to polyarylates</p></list-item><list-item><p>- Production of novel PHAs from CO<sub>2</sub> and H<sub>2</sub>+O<sub>2</sub> produced <italic>in situ</italic> in a bio-electrochemical system</p></list-item><list-item><p>- Expression-level of PHA synthase and molecular weight of polyesters are inversely correlated</p></list-item><list-item><p>- <italic>In silico</italic> design and analysis of pathways towards novel PHAs from different carbon-sources</p></list-item></list></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472442', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Using deep convolutional neural networks to forecast spatial patterns of Amazonian deforestation', 'authors': \"[{'name': 'James Ball'}, {'name': 'Katerina Petrova'}, {'name': 'David A. Coomes'}, {'name': 'Seth Flaxman'}]\", 'abstract': '<p><list list-type=\"order\"><list-item><p>Tropical forests are subject to diverse deforestation pressures but their conservation is essential to achieve global climate goals. Predicting the location of deforestation is challenging due to the complexity of the natural and human systems involved but accurate and timely forecasts could enable effective planning and on-the-ground enforcement practices to curb deforestation rates. New computer vision technologies based on deep learning can be applied to the increasing volume of Earth observation data to generate novel insights and make predictions with unprecedented accuracy.</p></list-item><list-item><p>Here, we demonstrate the ability of deep convolutional neural networks to learn spatiotemporal patterns of deforestation from a limited set of freely available global data layers, including multispectral satellite imagery, the Hansen maps of historic deforestation (2001-2020) and the ALOS JAXA digital surface model, to forecast future deforestation (2021). We designed four original deep learning model architectures, based on 2D Convolutional Neural Networks (2DCNN), 3D Convolutional Neural Networks (3DCNN), and Long Short-Term Memory (LSTM) Recurrent Neural Networks (RNN) to produce spatial maps that indicate the risk to each forested pixel (~30 m) in the landscape of becoming deforested within the next year. They were trained and tested on data from two ~80,000 km<sup>2</sup> tropical forest regions in the Southern Peruvian Amazon.</p></list-item><list-item><p>We found that the networks could predict the likely location of future deforestation to a high degree of accuracy. Our best performing model  a 3DCNN  had the highest pixel-wise accuracy (80-90%) when validated on 2020 deforestation based 2014-2019 training. Visual examination of the forecasts indicated that the 3DCNN network could automatically discern the drivers of forest loss from the input data. For example, pixels around new access routes (e.g. roads) were assigned high risk whereas this was not the case for recent, concentrated natural loss events (e.g. remote landslides).</p></list-item><list-item><p>CNNs can harness limited time-series data to predict near-future deforestation patterns, an important step in using the growing volume of satellite remote sensing data to curb global deforestation. The modelling framework can be readily applied to any tropical forest location and used by governments and conservation organisations to prevent deforestation and plan protected areas.</p></list-item></list></p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.12.472299', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Age dependent changes in circulating Tfh cells influence the development of functional antibodies to malaria in children', 'authors': \"[{'name': 'Jo-Anne Chan'}, {'name': 'Jessica R Loughland'}, {'name': 'Lauren de la Parte'}, {'name': 'Satomi Okano'}, {'name': 'Isaac Ssewanyana'}, {'name': 'Mayimuna Nalubega'}, {'name': 'Felistas Nankya'}, {'name': 'Kenneth Musinguzi'}, {'name': 'John Rek'}, {'name': 'Emmanuel Arinaitwe'}, {'name': 'Peta Tipping'}, {'name': 'Peter Bourke'}, {'name': 'Dean Andrew'}, {'name': 'Nicholas Dooley'}, {'name': 'Arya SheelaNair'}, {'name': 'Bruce D Wines'}, {'name': 'Mark Hogarth'}, {'name': 'James Beeson'}, {'name': 'Bryan Greenhouse'}, {'name': 'Grant Dorsey'}, {'name': 'Moses Kamya'}, {'name': 'Gunter Hartel'}, {'name': 'Gabriela Minigo'}, {'name': 'Margaret Feeney'}, {'name': 'Prasanna Jagannathan'}, {'name': 'Michelle J Boyle'}]\", 'abstract': '<p>T-follicular helper (Tfh) cells are key drivers of antibodies that protect from malaria. However, little is known regarding the host and parasite factors that influence Tfh and functional antibody development. Here, we use samples from a large cross-sectional study of children residing in an area of high malaria transmission in Uganda to characterize Tfh cells and functional antibodies to multiple parasites stages. We identify a dramatic re-distribution of the Tfh cell compartment with age that is independent of malaria exposure, with Th2-Tfh cells predominating in early childhood, while Th1-Tfh cell gradually increase to adult levels over the first decade of life. Functional antibody acquisition is age-dependent and hierarchical acquired based on parasite stage, with merozoite responses followed by sporozoite and gametocyte antibodies. Antibodies were boosted in children with current infection, and were higher in females. The children with the very highest antibody levels had increased Tfh cell activation and proliferation, consistent with a key role of Tfh cells in antibody development. Together, these data reveal a complex relationship between the circulating Tfh compartment, antibody development and protection from malaria.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.472564', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A salvaging strategy enables stable metabolite provisioning among free-living bacteria', 'authors': \"[{'name': 'Sebastian Gude'}, {'name': 'Gordon J. Pherribo'}, {'name': 'Michiko E. Taga'}]\", 'abstract': '<p>All organisms rely on complex metabolites such as amino acids, nucleotides, and cofactors for essential metabolic processes. Some microbes synthesize these fundamental ingredients of life <italic>de novo</italic>, while others rely on uptake to fulfill their metabolic needs. Although certain metabolic processes are inherently leaky, the mechanisms enabling stable metabolite provisioning among microbes in the absence of a host remain largely unclear. In particular, how can metabolite provisioning among free-living bacteria be maintained under the evolutionary pressure to economize resources? Salvaging, the process of recycling and reusing, can be a metabolically efficient route to obtain access to required resources. Here, we show experimentally how precursor salvaging in engineered <italic>Escherichia coli</italic> populations can lead to stable, long-term metabolite provisioning. We find that salvaged cobamides (vitamin B<sub>12</sub> and related enzyme cofactors) are readily made available to non-productive population members, yet salvagers are strongly protected from overexploitation due to partial metabolite privatization. We also describe a previously unnoted benefit of precursor salvaging, namely the removal of the non-functional, proliferation-inhibiting precursor. As long as compatible precursors are present, any microbe possessing the terminal steps of a biosynthetic process can, in principle, forgo <italic>de novo</italic> biosynthesis in favor of salvaging. Consequently, precursor salvaging likely represents a potent, yet overlooked, alternative to <italic>de novo</italic> biosynthesis for the acquisition and provisioning of metabolites in free-living bacterial populations.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.12.472204', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Black chromatin is indispensable for accurate simulations of Drosophila melanogaster chromatin structure', 'authors': \"[{'name': 'Irina Tuszynska'}, {'name': 'Pawel Bednarz'}, {'name': 'Bartek Wilczynski'}]\", 'abstract': '<p>The interphase chromatin structure is extremely complex, precise and dynamic. Experimental methods can only show the frequency of interaction of the various parts of the chromatin. Therefore, it is extremely important to develop theoretical methods to predict the chromatin structure. In this publication, we describe the necessary factors for the effective modeling of the chromatin structure in <italic>Drosophila melanogaster</italic>. We also compared Monte Carlo with Molecular Dynamic methods. We showed that incorporating black, non-reactive chromatin is necessary for successfully prediction of chromatin structure, while the loop extrusion model or using Hi-C data as input are not essential for the basic structure reconstruction.</p><sec><title>Abstract Figure</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"472204v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.12.470823', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Stressed out underground?: Transcriptomics illuminates the genome-wide response to heat stress in surface and subterranean diving beetles', 'authors': \"[{'name': 'Perry G. Beasley-Hall'}, {'name': 'Terry Bertozzi'}, {'name': 'Tessa M. Bradford'}, {'name': 'Charles S. P. Foster'}, {'name': 'Karl Jones'}, {'name': 'Simon M. Tierney'}, {'name': 'William F. Humphreys'}, {'name': 'Andrew D. Austin'}, {'name': 'Steven J. B. Cooper'}]\", 'abstract': '<p>Subterranean habitats are environmentally stable with respect to temperature, humidity, and the absence of light. The transition to a subterranean lifestyle might therefore be expected to cause considerable shifts in an organisms physiology; here, we investigate how subterranean colonisation affects thermal tolerance. Subterranean organisms might be at an increased risk of decline in the face of global temperature rises, but robust data on the fauna is lacking, particularly at the molecular level. In this study we compare the heat shock response of two species of diving beetle in the genus <italic>Paroster</italic>: one surface-dwelling (<italic>P. nigroadumbratus</italic>), the other restricted to a single aquifer (<italic>P. macrosturtensis</italic>). <italic>P. macrosturtensis</italic> has been previously established as having a lower thermal tolerance compared to surface-dwelling relatives, but the genomic basis of this difference is unknown. By sequencing transcriptomes of experimentally heat-shocked individuals and performing differential expression analysis, we demonstrate both species can mount a heat shock response at high temperatures (35C), in agreement with past survival experiments. However, the genes involved in these responses differ between species, and far greater genes are differentially expressed in the surface species, which may explain its more robust response to heat stress. In contrast, the subterranean species significantly upregulated the heat shock protein gene <italic>Hsp68</italic> in the experimental setup under conditions it would likely encounter in nature (25C), suggesting it may be more sensitive to ambient stressors, e.g. handling. The results presented here contribute to an emerging narrative concerning weakened thermal tolerances in obligate subterranean organisms at the molecular level.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.15.472778', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The African Swine Fever Isolate ASFV-Kenya-1033-IX is highly virulent and stable after growth in the wild boar cell line WSL', 'authors': \"[{'name': 'Johanneke D. Hemmink'}, {'name': 'Hussein M. Abkallo'}, {'name': 'Sonal P. Henson'}, {'name': 'Emmanuel M. Khazalwa'}, {'name': 'Bernard Oduor'}, {'name': 'Anna Lacasta'}, {'name': 'Edward Okoth'}, {'name': 'Walter Fuchs'}, {'name': 'Richard P. Bishop'}, {'name': 'Lucilla Steinaa'}]\", 'abstract': '<p>In this study, we describe an African swine fever genotype IX virus (ASFV-Kenya-1033-IX), which was isolated from a domestic pig in Western Kenya during a reported outbreak, including efficiency of virus replication, <italic>in vivo</italic> virulence, and genome stability in pulmonary alveolar macrophages (PAM) and in a wild boar cell line (WSL). The ASFV-Kenya-1033-IX stock, which underwent multiple passages in WSL (more than 20), retained its ability to replicate in primary macrophages and it also retained the virulence <italic>in vivo</italic>. At the genomic level, only a few single nucleotide differences were observed between the macrophage and WSL-grown virus. Thus, we propose that the WSL cell line is suitable to produce live attenuated ASFV vaccine candidates based on this isolate and probably of similar viruses. The genome sequences for ASFV-Kenya-1033-IX grown in macrophages and in WSL cells was submitted to GenBank and a challenge model based on this isolate was set up, which will aid the development of vaccines against genotype IX ASFV circulating in Eastern and Central Africa.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.472884', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Emergence of compensatory mutations reveal the importance of electrostatic interactions between HIV-1 integrase and genomic RNA', 'authors': \"[{'name': 'Christian Shema Mugisha'}, {'name': 'Tung Dinh'}, {'name': 'Kasyap Tenneti'}, {'name': 'Jenna Eve Eschbach'}, {'name': 'Keanu Davis'}, {'name': 'Robert Gifford'}, {'name': 'Mamuka Kvaratskhelia'}, {'name': 'Sebla Kutluay'}]\", 'abstract': '<p>Independent of its catalytic activity, HIV-1 integrase (IN) enzyme regulates proper particle maturation by binding to and packaging the viral RNA genome (gRNA) inside the mature capsid lattice. Allosteric integrase inhibitors (ALLINIs) and class II IN substitutions inhibit the binding of IN to the gRNA and cause the formation of non-infectious virions characterized by mislocalization of the viral ribonucleoprotein complexes between the translucent conical capsid lattice and the viral lipid envelope. To gain insight into the molecular nature of IN-gRNA interactions, we have isolated compensatory substitutions in the background of a class II IN (R269A/K273A) variant that directly inhibits IN binding to the gRNA. We found that additional D256N and D270N substitutions in the C-terminal domain (CTD) of IN restored its ability to bind gRNA and led to the formation of infectious particles with correctly matured morphology. Furthermore, reinstating the overall positive electrostatic potential of the CTD through individual D256R or D256K substitutions was sufficient to restore IN-RNA binding and infectivity for the R269A/K273A as well as the R262A/R263A class II IN mutants. The compensatory mutations did not impact functional IN oligomerization, suggesting that they directly contributed to IN binding to the gRNA. Interestingly, HIV-1 IN R269A/K273A, but not IN R262A/R263A, bearing compensatory mutations was more sensitive to ALLINIs providing key genetic evidence that specific IN residues required for RNA binding also influence ALLINI activity. Structural modeling provided further insight into the molecular nature of IN-gRNA interactions and ALLINI mechanism of action. Taken together, our findings highlight an essential role of IN-gRNA interactions for proper virion maturation and reveal the importance of electrostatic interactions between the IN CTD and the gRNA.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.21267748', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'A meta-analysis of Early Results to predict Vaccine efficacy against Omicron', 'authors': \"[{'name': 'David S Khoury'}, {'name': 'Megan Steain'}, {'name': 'James Triccas'}, {'name': 'Alex Sigal'}, {'name': 'Miles Philip Davenport'}, {'name': 'Deborah Cromer'}]\", 'abstract': '<p>In the studies to date, the estimated fold-drop in neutralisation titre against Omicron ranges from 2- to over 20-fold depending on the study and serum tested. Collating data from the first four of these studies results in a combined estimate of the drop in neutralisation titre against Omicron of 9.7-fold (95% CI 5.5-17.1). We use our previously established model to predict that six months after primary immunisation with an mRNA vaccine, efficacy for Omicron is estimated to have waned to around 40% against symptomatic and 80% against severe disease. A booster dose with an existing mRNA vaccine (even though it targets the ancestral spike) has the potential to raise efficacy for Omicron to 86.2% (95% CI: 72.6-94) against symptomatic infection and 98.2% (95% CI: 90.2-99.7) against severe infection.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.472993', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'SMAD4 and TGF are architects of inverse genetic programs during fate-determination of antiviral memory CD8 T cells', 'authors': \"[{'name': 'Karthik Chandiran'}, {'name': 'Jenny Suarez-Ramirez'}, {'name': 'Yinghong Hu'}, {'name': 'Zeynep Ugur'}, {'name': 'Evan R Jellison'}, {'name': 'Jun-Siong low'}, {'name': 'Bryan McDonald'}, {'name': 'Susan M Kaech'}, {'name': 'Linda S Cauley'}]\", 'abstract': '<p>Transforming growth factor  (TGF) is a morphogenic protein that augments antiviral immunity by altering the functional properties of pathogen-specific memory CD8 T cells. During infection, TGF inhibits formation of effector (TEFF) and circulating memory CD8 T cells, while encouraging tissue resident memory CD8 T cells (TRM) to settle in peripheral tissues. SMAD proteins are signaling intermediates that are used by members of the TGF cytokine family to modify gene expression. Using RNA-sequencing we determined that SMAD4 altered the transcriptional profile of antiviral CTLs during respiratory infection. Our data show that SMAD4 and TGF use alternate signaling pathways to cooperatively regulate a collection of genes that determine whether pathogen-specific memory CD8 T cells localize in peripheral or lymphoid tissues. During infection, SMAD4 acts independently of TGF to inhibit TRM development, while inducing genes that support formation of circulating memory CD8 T cells. The genes that are modulated by SMAD4 include several homing receptors (CD103, KLRG1 and CD62L) and transcription factors (Hobit and EOMES) that support memory formation.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472734', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Manganese modulates metabolic activity and redox homeostasis in translationally-blocked Lactococcus cremoris, impacting metabolic persistence, cell-culturability, and flavor formation', 'authors': \"[{'name': 'Avis Dwi Wahyu Nugroho'}, {'name': 'Berdien van Olst'}, {'name': 'Stephanie Agnes Bachtiar'}, {'name': 'Sjef Boeren'}, {'name': 'Michiel Kleerebezem'}, {'name': 'Herwig Bachmann'}]\", 'abstract': '<p>Manganese (Mn) is an essential trace element that is supplemented in microbial media with varying benefits across species and growth conditions. We found that growth of <italic>Lactococcus cremoris</italic> was unaffected by manganese omission from the growth medium. The main proteome adaptation to manganese omission involved increased manganese transporter production (up to 2000-fold), while the remaining 10 significant proteome changes were between 1.4 and 4 fold. Further investigation in translationally-blocked (TB), non-growing cells showed that Mn supplementation (20 M) led to approximately 1.5X faster acidification compared to Mn-free conditions. However, this faster acidification stagnated within 24 hours, likely due to draining of intracellular NADH that coincides with substantial loss of culturability. Conversely, without manganese, non-growing cells persisted to acidify for weeks, albeit at a reduced rate, but maintaining redox balance and culturability. Strikingly, despite being unculturable, -keto acid-derived aldehydes continued to accumulate in cells incubated in the presence of manganese, whereas without manganese cells predominantly formed the corresponding alcohols. This is most likely reflecting NADH availability for the alcohol dehydrogenase-catalyzed conversion. Overall, manganese influences the lactococcal acidification rate, and flavor formation capacity in a redox dependent manner. These are important industrial traits especially during cheese ripening, where cells are in a non-growing, often unculturable state.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472078', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'LKB1 Is Physiologically Required for Sleep from Drosophila to the Mouse', 'authors': \"[{'name': 'Ziyi Liu'}, {'name': 'Lifen Jiang'}, {'name': 'Chaoyi Li'}, {'name': 'Chengang Li'}, {'name': 'Jingqun Yang'}, {'name': 'Dan Wang'}, {'name': 'Renbo Mao'}, {'name': 'Yi Rao'}]\", 'abstract': '<p>LKB1 is known as a master kinase for 14 kinases related to the adenosine monophosphate (AMP)-activated protein kinase (AMPK). Two of them (SIK3 and AMPK) have previously been implicated in sleep regulation. We generated loss-of-function (LOF) mutants for Lkb1 in both Drosophila and mice. Sleep was reduced in Lkb1-mutant flies and in flies with neuronal deletion of Lkb1. Sleep was reduced in mice after virally mediated reduction of Lkb1 in the brain. Electroencephalography (EEG) analysis showed that non-rapid eye movement (NREM) sleep and sleep need were both reduced in Lkb1-mutant mice. These results indicate that LKB1 plays a physiological role in sleep regulation conserved from flies to mice.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.472716', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Potential and action mechanism of favipiravir as an antiviral against Junin virus', 'authors': \"[{'name': 'Vahid Rajabali Zadeh'}, {'name': 'Shuzo Urata'}, {'name': 'Tosin Oladipo Afowowe'}, {'name': 'Jiro Yasuda'}]\", 'abstract': '<p>Favipiravir is a nucleoside analogue that inhibits the replication and transcription of a broad spectrum of RNA viruses, including pathogenic arenaviruses. In this study, we isolated a favipiravir-resistant mutant of Junin virus (JUNV), which is the causative agent of Argentine hemorrhagic fever, and analyzed the antiviral mechanism of favipiravir against JUNV. Two amino acid substitutions, N462D in the RNA-dependent RNA polymerase (RdRp) and A168T in the glycoprotein precursor GPC, were identified in the mutant. GPC-A168T substitution enhanced the efficiency of JUNV internalization, which explains the robust replication kinetics of the mutant in the virus growth analysis. Although RdRp-N462D substitution did not affect polymerase activity levels in a minigenome system, comparisons of RdRp error frequencies showed that the virus with RdRp-D462 possessed a significantly higher fidelity. We also provided experimental evidence for the first time that favipiravir inhibited JUNV through the accumulation of transition mutations, confirming its role as a purine analogue against arenaviruses. Moreover, we showed that treatment with a combination of favipiravir and either ribavirin or remdesivir inhibited JUNV replication in a synergistic manner, blocking the generation of the drug-resistant mutant. Our findings provide new insights for the clinical management and treatment of Argentine hemorrhagic fever.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.21267773', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Health Policy', 'covid': True, 'title': 'COVID-19 vaccine hesitancy in diverse groups in the UK - is the driver economic or cultural in student populations?', 'authors': \"[{'name': 'Francis Drobniewski'}, {'name': 'Dian Kusuma'}, {'name': 'Agnieszka Broda'}, {'name': 'Enrique Castro-Snchez'}, {'name': 'Raheelah Ahmad'}]\", 'abstract': '<p>Studies have identified a greater reluctance for members of the Black, Asian, and minority ethnic communities to be vaccinated against COVID-19 despite a higher probability of greater harm from COVID-19. We conducted an anonymised questionnaire-based study of students (recruiting primarily before first reports of embolic events) at two London universities to identify whether economic or educational levels were primarily responsible for this reluctance: a postgraduate core group (PGCC) n=860 and a pilot study of undergraduate medical and nursing students (n=103). Asian and Black students were 2.0 and 3.2 times (PGCC) less likely to accept the COVID vaccine than White British students. Similar findings were noted in the pilot study students. As students were studying for Masters or PhD degrees and voluntarily paying high fees, educational and economic reasons were unlikely to be the underlying cause, and wider cultural reservations were more likely. Politicians exerted a strong negative influence, suggesting that campaigns should omit politicians.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.473069', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Enhancer RNA-based modeling of adverse events and objective responses of immunotherapy', 'authors': \"[{'name': 'Mengbiao Guo'}, {'name': 'Zhiya Lu'}, {'name': 'Yuanyan Xiong'}]\", 'abstract': '<p>Immune checkpoint inhibitors (ICI) targeting PD-1/PD-L1 or CTLA-4 are emerging and effective immunotherapy strategies. However, ICI treated patients present heterogeneous responses and adverse events, thus demanding effective ways to assess benefit over risk before treatment. Here, by integrating pan-cancer clinical and molecular data, we tried to predict immune-related adverse events (irAEs, risk) and objective response rates (ORRs, benefit) based on enhancer RNAs (eRNAs) expression among patients receiving anti-PD-1/PD-L1 therapy. We built two effective regression models, explaining 71% variance (R=0.84) of irAEs with three eRNAs and 79% (R=0.89) of ORRs with five eRNAs. Interestingly, target genes of irAE-related enhancers, including upstream regulators of MYC, were involved in metabolism, inflammation, and immune activation, while ORR-related enhancers target PAK2 and DLG1 which directly participate in T cell activation. Our study provides references for the identification of immunotherapy-related biomarkers and potential therapeutic targets during immunotherapy.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.472581', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Adaptive strategies under prolonged starvation and role of slow growth in bacterial fitness', 'authors': \"[{'name': 'Pabitra Nandy'}]\", 'abstract': '<label>1.</label><title>Abstract</title><p>Adaptive evolution has the power to illuminate genetic mechanisms under a pre-defined set of selection factors in a controlled environment. Laboratory evolution of bacteria under long-term starvation has gained importance in recent years because of its ability to uncover adaptive strategies to overcome prolonged nutrient limitation- a condition thought to be encountered often by natural microbial isolates. In this evolutionary paradigm, bacteria are maintained in an energy-restricted environment in the growth phase called as long-term stationary phase or LTSP. This phase is characterized by a stable viable population size and highly dynamic genetic changes. Multiple independent iterations of LTSP evolution experiments have given rise to mutants that are slow-growing compared to the ancestor. Although the antagonistic regulation between rapid growth and stress response is fairly well-known in bacteria (especially <italic>Escherichia coli</italic>), the reason behind the growth deficit of many LTSP-adapted mutants has not been explored in detail. In this review, I revisit the trade-off between growth and stress response and delve into the regulatory mechanisms currently known to control growth under nutrient deficiency. Focusing on the theme of sigma-factor competition I try to search for the evolutionary reasoning of slow growth amongst mutants adapted to prolonged starvation. Additionally, I present novel experimental data indicating the dynamics of four such slow-growing variants that evolved during a 30-day long LTSP evolution experiment with <italic>Escherichia coli</italic>.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.14.472572', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'High content screening and computational prediction reveal viral genes that suppress innate immune response', 'authors': \"[{'name': 'Tai L. Ng'}, {'name': 'Erika J. Olson'}, {'name': 'Tae Yeon Yoo'}, {'name': 'H. Sloane Weiss'}, {'name': 'Yukiye Koide'}, {'name': 'Peter D. Koch'}, {'name': 'Nathan J. Rollins'}, {'name': 'Pia Mach'}, {'name': 'Tobias Meisinger'}, {'name': 'Trenton Bricken'}, {'name': 'Timothy Z. Chang'}, {'name': 'Colin Molloy'}, {'name': 'Jrme Zrcher'}, {'name': 'Timothy J. Mitchison'}, {'name': 'John I. Glass'}, {'name': 'Debora S. Marks'}, {'name': 'Jeffrey C. Way'}, {'name': 'Pamela A. Silver'}]\", 'abstract': '<p>Suppression of the host innate immune response is a critical aspect of viral replication. Upon infection, viruses may introduce one or more proteins that inhibit key immune pathways, such as the type I interferon pathway. However, the ability to predict and evaluate viral protein bioactivity on targeted pathways remains challenging and is typically done on a single virus/gene basis. Here, we present a medium-throughput high-content cell-based assay to reveal the immunosuppressive effects of viral proteins. To test the predictive power of our approach, we developed a library of 800 genes encoding known, predicted, and uncharacterized human viral genes. We find that previously known immune suppressors from numerous viral families such as <italic>Picornaviridae</italic> and <italic>Flaviviridae</italic> recorded positive responses. These include a number of viral proteases for which we further confirmed that innate immune suppression depends on protease activity. A class of predicted inhibitors encoded by <italic>Rhabdoviridae</italic> viruses was demonstrated to block nuclear transport, and several previously uncharacterized proteins from uncultivated viruses were shown to inhibit nuclear transport of the transcription factors NF-B and IRF3. We propose that this pathway-based assay, together with early sequencing, gene synthesis, and viral infection studies, could partly serve as the basis for rapid <italic>in vitro</italic> characterization of novel viral proteins.</p><sec><title>IMPORTANCE</title><p>Infectious diseases caused by viral pathogens exacerbate healthcare and economic burdens. Numerous viral biomolecules suppress the human innate immune system, enabling viruses to evade an immune response from the host. Despite our current understanding of viral replications and immune evasion, new viral proteins, including those encoded by uncultivated viruses or emerging viruses, are being unearthed at a rapid pace from large scale sequencing and surveillance projects. The use of medium- and high-throughput functional assays to characterize immunosuppressive functions of viral proteins can advance our understanding of viral replication and possibly treatment of infections. In this study we assembled a large viral gene library from diverse viral families and developed a high content assay to test for inhibition of innate immunity pathways. Our work expands the tools that can rapidly link sequence and protein function, representing a practical step towards early-stage evaluation of emerging and understudied viruses.</p></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472569', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'The blackgrass genome reveals patterns of divergent evolution of non-target site resistance to herbicides', 'authors': \"[{'name': 'Lichun Cai'}, {'name': 'David Comont'}, {'name': 'Dana MacGregor'}, {'name': 'Claudia Lowe'}, {'name': 'Roland Beffa'}, {'name': 'Christopher Saski'}, {'name': 'Paul Neve'}]\", 'abstract': '<p>Globally, weedy plants result in more crop yield loss than plant pathogens and insect pests combined. Much of the success of weeds rests with their ability to rapidly adapt in the face of human-mediated environmental management and change. The evolution of resistance to herbicides is an emblematic example of this rapid adaptation. Here, we focus on <italic>Alopecurus myosuroides</italic> (blackgrass), the most impactful agricultural weed in Europe. To gain insights into the evolutionary history and genomic mechanisms underlying adaptation in blackgrass, we assembled and annotated its large, complex genome. We show that non-target site herbicide resistance is oligogenic and likely evolves from standing genetic variation. We present evidence for divergent selection of resistance at the level of the genome in wild, evolved populations, though at the transcriptional level, resistance mechanisms are underpinned by similar patterns of up-regulation of stress- and defence-responsive gene families. These gene families are expanded in the blackgrass genome, suggesting that the large, duplicated, and dynamic genome plays a role in enabling rapid adaptation in blackgrass. These observations have wide significance for understanding rapid plant adaptation in novel stressful environments.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.12.472291', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'genomicSimulation: fast R functions for stochastic simulation of breeding programs', 'authors': \"[{'name': 'Kira Villiers'}, {'name': 'Eric Dinglasan'}, {'name': 'Ben J. Hayes'}, {'name': 'Kai P. Voss-Fels'}]\", 'abstract': '<p>Simulation tools are key to designing and optimising breeding programs that are many-year, high-effort endeavours. Tools that operate on real genotypes and integrate easily with other analysis software are needed for users to integrate simulated data into their analysis and decision-making processes. This paper presents genomicSimulation, a fast and flexible tool for the stochastic simulation of crossing and selection on real genotypes. It is fully written in C for high execution speeds, has minimal dependencies, and is available as an R package for integration with Rs broad range of analysis and visualisation tools. Comparisons of a simulated recreation of a breeding program to the real data shows that the tools simulated offspring correctly show key population features. Both versions of genomicSimulation are freely available on GitHub: The R package version at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/vllrs/genomicSimulation/\">https://github.com/vllrs/genomicSimulation/</ext-link> and the C library version at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/vllrs/genomicSimulationC\">https://github.com/vllrs/genomicSimulationC</ext-link></p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.16.472873', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Transcription factor lineages in plant-pathogenic fungi, connecting diversity with fungal virulence', 'authors': \"[{'name': 'Evan John'}, {'name': 'Kar-Chun Tan'}, {'name': 'Richard Peter Oliver'}, {'name': 'Karam Singh'}]\", 'abstract': '<p>Plant-pathogenic fungi span diverse taxonomic lineages. Their host-infection strategies are often specialised and require the coordinated regulation of molecular virulence factors. Transcription factors (TFs) are fundamental regulators of gene expression, controlling development and virulence in plant pathogenic fungi. Recent research has established regulatory roles for several taxonomically conserved fungal TFs, but the evolution of specific virulence regulators is not well understood. This study sought to explore the representation of TFs across a taxonomically-diverse range of fungi, with a focus on plant pathogens. A significant trend was observed among the obligate, host-associated pathogens, which possess a reduced overall TF repertoire, alluding to a lack of pressure for maintaining diversity. A novel orthology-based analysis is then presented that refined TF classifications, traditionally based on the nature of the DNA-binding domains. Using this analysis, cases of TF over/underrepresentation across fungal pathogen lineages are systematically highlighted. Specific examples are then explored and discussed that included the TF orthologues of Ste12, Pf2 and EBR1, plus phytotoxic secondary-metabolite cluster regulators, which all presented novel and distinct evolutionary insights. Ultimately, as the examples presented demonstrate, this resource can be interrogated to guide functional studies that seek to characterise virulence-specific regulators and shed light on the factors underpinning plant pathogenicity.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.21267812', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Analytical Solution of a New SEIR Model Based on Latent Period-Infectious Period Chronological Order', 'authors': \"[{'name': 'Xiaoping Liu'}]\", 'abstract': '<p>The Susceptible-Infectious-Recovered (<italic>SIR</italic>) and <italic>SIR</italic> derived epidemic models have been commonly used to analyze the spread of infectious diseases. The underlying assumption in these models, such as Susceptible-Exposed-Infectious-Recovered (<italic>SEIR</italic>) model, is that the change in variables <italic>E, I</italic> or <italic>R</italic> at time <italic>t</italic> is dependent on a fraction of <italic>E</italic> and <italic>I</italic> at time <italic>t</italic>. This means that after exposed on a day, this individual may become contagious or even recover on the same day. However, the real situation is different: an exposed individual will become infectious after a latent period (<italic>l</italic>) and then recover after an infectious period (<italic>i</italic>). In this study, we proposed a new <italic>SEIR</italic> model based on the latent period-infectious period chronological order (Liu X., Results Phys. 2021; 20:103712). An analytical solution to equations of this new <italic>SEIR</italic> model was derived. From this new <italic>SEIR</italic> model, we obtained a propagated curve of infectious cases under conditions <italic>l</italic>&gt;<italic>i</italic>. Similar propagated epidemic curves were reported in literature. However, the conventional <italic>SEIR</italic> model failed to simulate the propagated epidemic curves under the same conditions. For <italic>l</italic>&lt;<italic>i</italic>, the new <italic>SEIR</italic> models generated bell-shaped curves for infectious cases, and the curve is near symmetrical to the vertical line passing the curve peak. This characteristic can be found in many epidemic curves of daily COVID-19 cases reported from different countries. However, the curve generated from the conventional <italic>SEIR</italic> model is a right-skewed bell-shaped curve. An example for applying the analytical solution of the new <italic>SEIR</italic> model equations to simulate the reported daily COVID-19 cases was also given in this paper.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.12.472295', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Protective effect of peroxisome proliferator-activated receptor- in regulating the early inflammation response to extended hepatectomy from a comparative transcriptomic analysis', 'authors': \"[{'name': 'Cheng-Cheng Shi'}, {'name': 'Yang Bai'}, {'name': 'Xin Yan'}, {'name': 'Nuo Cheng'}, {'name': 'Wen-Zhi Guo'}, {'name': 'Shui-Jun Zhang'}, {'name': 'Ji-Hua Shi'}]\", 'abstract': '<p>The regulation mechanism of small-for-size syndrome remains unclear. Thus, we aimed to analyze the molecular profiles following extended hepatectomy and identify the therapeutic target. Major hepatectomy and extended hepatectomy were performed in the rat model, and the remnant livers were obtained dynamically for the high-throughput transcriptome analysis to identify the differentially expressed genes (DEGs). The general framework for weighted gene co-expression network analysis (WGCNA) was employed to explore the expression patterns of DEGs. As result, WGCNA identified 10 distinct gene co-expression modules according to the correlation between module eigengene and different postoperative time-points. The magenta module (gene count: 289) and the lightcyan module (gene count: 484) were found positively correlated with major hepatectomy instead of extended hepatectomy. In the lightcyan module, peroxisome proliferator-activated receptor- (PPAR) was selected and found the down-regulation in the remnant liver following extended (marginal) hepatectomy in rats and humans. Besides, administration of PPAR agonist attenuated hepatic inflammation injury while PPAR antagonist increased liver inflammation injury after extended hepatectomy in rats, marked by the significantly changed aminotransferases, tumor necrosis factor- and interleukin<italic>-</italic>6 levels in the plasm, and histological Suzuki criteria. Consequently, DEGs and their molecular profiles after extended hepatectomy were identified, and PPAR might be a potential therapy target for small-for-size syndrome.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.12.472329', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Disruption of c-di-GMP signaling networks unlocks cryptic expression of secondary metabolites during biofilm growth in Burkholderia pseudomallei', 'authors': \"[{'name': 'Grace I. Borlee'}, {'name': 'Mihnea R. Mangalea'}, {'name': 'Kevin H. Martin'}, {'name': 'Brooke A. Plumley'}, {'name': 'Samuel J. Golon'}, {'name': 'Bradley R. Borlee'}]\", 'abstract': '<p>The regulation and production of secondary metabolites during biofilm growth of <italic>Burkholderia</italic> spp. is not well understood. To learn more about the crucial role and regulatory control of cryptic molecules produced during biofilm growth, we disrupted c-di-GMP signaling in <italic>Burkholderia pseudomallei,</italic> a soil-borne bacterial saprophyte and the etiologic agent of melioidosis. Our approach to these studies combined transcriptional profiling with genetic deletions that targeted key c-di-GMP regulatory components to characterize responses to changes in temperature. Mutational analyses and conditional expression studies of c-di-GMP genes demonstrates their contribution to phenotypes such as biofilm formation, colony morphology, motility, and expression of secondary metabolite biosynthesis when grown as a biofilm at different temperatures. RNA-seq analysis was performed at varying temperatures in a II2523 mutant background that is responsive to temperature alterations resulting in hypo- and hyper-biofilm forming phenotypes. Differential regulation of genes was observed for polysaccharide biosynthesis, secretion systems, and nonribosomal peptide and polyketide synthase (NRPS/PKS) clusters in response to temperature changes. Deletion mutations of biosynthetic gene clusters (BGCs) clusters 2, 11, 14 (syrbactin), and 15 (malleipeptin) in wild-type and II2523 backgrounds also reveals the contribution of these BGCs to biofilm formation and colony morphology in addition to inhibition of <italic>Bacillus subtilis</italic> and <italic>Rhizoctonia solani</italic>. Our findings suggest that II2523 impacts the regulation of genes that contribute to biofilm formation and competition.</p><p>Characterization of cryptic BGCs under differing environmental conditions will allow for a better understanding of the role of secondary metabolites in the context of biofilm formation and microbe-microbe interactions.</p><sec><title>Importance</title><p><italic>Burkholderia pseudomallei</italic> is a saprophytic bacterium residing in the environment that switches to a pathogenic lifestyle during infection of a wide range of hosts. The environmental cues that serve as the stimulus to trigger this change are largely unknown. However, it is well established that the cellular level of c-di-GMP, a secondary signal messenger, controls the switch from growth as planktonic cells to growth as a biofilm. Disrupting the signaling mediated by c-di-GMP allows for a better understanding of the regulation and the contribution of the surface associated and secreted molecules that contribute to the various lifestyles of this organism. The genome of <italic>B. pseudomallei</italic> also encodes cryptic biosynthetic gene clusters predicted to encode small molecules that potentially contribute to growth as a biofilm, adaptation, and interactions with other organisms. A better understanding of the regulation of these molecules is crucial to understanding how this versatile pathogen alters its lifestyle.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.14.21267818', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Health Systems and Quality Improvement', 'covid': True, 'title': 'Performances of community pharmacists and community pharmacies during COVID-19 outbreak: a cross-sectional study', 'authors': \"[{'name': 'Afsaneh Sadeghinejad'}, {'name': 'Naemeh Nikvarz'}]\", 'abstract': '<sec><title>Background</title><p>Community pharmacists can play an important role in the management of coronavirus disease-19 (COVID-19) outbreak by providing pharmaceutical care and education services. In addition to providing drug-related services, community pharmacies must take measures to reduce the risk of COVID-19 transmission to customers and technicians in pharmacy. Therefore, a study was designed to assess community pharmacists performance and community pharmacies preparedness during COVID-19 outbreak.</p></sec><sec><title>Materials &amp; Methods</title><p>A cross-sectional study was conducted in Kerman, Iran. A 20-item checklist consisting of nine items assessing the practice of pharmacists regarding use of personal protective equipment (PPE) and education of people about routes of transmission, prophylaxis and treatment of COVID-19 and eleven items evaluating the adherence of pharmacies to principles of reducing the risk of COVID-19 transmission in public places was used.</p></sec><sec><title>Results</title><p>Of 95 enrolled pharmacists, 55 (57.9%) were female and 78.9% had work experience  10 years. More than 90% of pharmacists used appropriate PPE and provided information about medications for prevention and treatment of COVID-19 and routs of its transmission to the pharmacy customers. Phone consultation was done by 71.6% of pharmacists. Concerning the preparedness of pharmacies, 92% of personnel used PPE, and hand sanitizers were available in 89.5% of pharmacies. However, a clear sheet was not placed at counters in the majority of pharmacies, and home delivery services were not provided by most pharmacies.</p></sec><sec><title>Conclusions</title><p>The practice of most community pharmacists was acceptable. However, stricter measures should be taken to diminish the probability of COVID-19 transmission in community pharmacies.</p></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472604', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Evolution of miRNA binding sites and regulatory networks in cichlids', 'authors': \"[{'name': 'Tarang K. Mehta'}, {'name': 'Luca Penso-Dolfin'}, {'name': 'Will Nash'}, {'name': 'Sushmita Roy'}, {'name': 'Federica Di-Palma'}, {'name': 'Wilfried Haerty'}]\", 'abstract': '<p>The divergence of regulatory regions and gene regulatory network (GRN) rewiring is a key driver of cichlid phenotypic diversity. However, the contribution of miRNA binding site turnover has yet to be linked to GRN evolution across cichlids. Here, we extend our previous studies by analysing the selective constraints driving evolution of miRNA and transcription factor binding sites (TFBSs) of target genes, to infer instances of cichlid GRN rewiring associated with regulatory binding site turnover. Comparative analyses identified increased species-specific networks that are functionally associated to traits of cichlid phenotypic diversity. The evolutionary rewiring is associated with differential models of transcription factor (TF) and miRNA binding site turnover, driven by a high proportion of fast-evolving polymorphic sites in adaptive trait genes compared to subsets of random genes. Positive selection acting upon discrete mutations in these regulatory regions is likely to be an important mechanism in rewiring GRNs in rapidly radiating cichlids. Regulatory variants of functionally associated TF and miRNA binding sites of visual opsin genes differentially segregate according to phylogeny and ecology of Lake Malawi species, identifying both rewired e.g. clade-specific and conserved network motifs of adaptive trait associated GRNs. Our approach revealed several novel candidate regulators, regulatory regions and three-node motifs across cichlid genomes with previously reported associations to known adaptive evolutionary traits.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.472598', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Cell polarity determinant Dlg1 facilitates epithelial invagination by promoting tissue-scale mechanical coordination', 'authors': \"[{'name': 'Melisa Andrea Fuentes'}, {'name': 'Bing He'}]\", 'abstract': '<p>Epithelial folding mediated by apical constriction serves as a fundamental mechanism to convert flat epithelial sheets into multilayered structures. It remains elusive whether additional mechanical inputs are required for folding mediated by apical constriction. Using <italic>Drosophila</italic> mesoderm invagination as a model, we identified an important role for the non-constricting, lateral mesodermal cells adjacent to the constriction domain (\"flanking cells\") in facilitating epithelial folding. We found that depletion of the basolateral determinant, Dlg1, disrupts the transition between apical constriction and invagination without affecting the rate of apical constriction. Strikingly, the observed delay in invagination is associated with ineffective apical myosin contractions in the flanking cells that lead to overstretching of their apical domain. The defects in the flanking cells impede ventral-directed movement of the lateral ectoderm, suggesting reduced mechanical coupling between tissues. Specifically disrupting the flanking cells in wildtype embryos by laser ablation or optogenetic depletion of cortical actin is sufficient to delay the apical constriction-to-invagination transition. Our findings indicate that effective mesoderm invagination requires intact flanking cells and suggest a role for tissue-scale mechanical coupling during epithelial folding.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.21267787', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': False, 'title': 'Social isolation and the aging brain Social isolation is linked to declining grey matter structure and cognitive functions in the LIFE-Adult panel study', 'authors': \"[{'name': 'L Lammer'}, {'name': 'F Beyer'}, {'name': 'M Luppa'}, {'name': 'C Sander'}, {'name': 'R Baber'}, {'name': 'C Engel'}, {'name': 'K Wirkner'}, {'name': 'M Loeffler'}, {'name': 'SG Riedel-Heller'}, {'name': 'A Villringer'}, {'name': 'AV Witte'}]\", 'abstract': '<sec><title>Background</title><p>Social isolation is a risk factor for dementia, a devastating disease with a rapidly growing global prevalence. However, the link between social isolation and changes in brain structure and function is poorly understood, as studies are scarce in number, methodologically inconsistent and small in size. In this pre-registered analysis of a large population-based panel study, we aimed to determine the impact of social isolation on brain structures and cognitive functions central to age associated decline and dementia.</p></sec><sec><title>Methods and findings</title><p>We analysed data of 1992 cognitively healthy participants of the LIFE-Adult study at baseline (age range: 50-82 years) and of 1409 particpants at follow-up (average change in age: 5.89 years). We measured social isolation using the 30-point Lubben Social Network Scale (LSNS) and derived measures of grey matter structure from anatomical 3T MRIs. We employed covariate adjusted linear mixed models to test the associations of baseline social isolation and change in social isolation with hippocampal volume, cognitive functions (executive functions, memory, processing speed) and cortical thickness.</p><p>We found stronger baseline social isolation to be significantly associated with smaller hippocampal volumes ( = 5.5 mm<sup>3</sup>/LSNS point(pt), FDR q = 0.004, BF = 14.6) and lower cognitive functions (all  &lt; 0.014 SD/pt, FDR q &lt; 0.003, BF &gt; 49). Increases in social isolation over time were linked to hippocampal volume decline ( = 4.9 mm<sup>3</sup>/pt, FDR q = 0.01, BF = 2.9) and worse memory performance ( = 0.013 SD/pt, FDR q = 0.04, BF = 1.1). Furthermore, we detected a significant interaction of baseline social isolation with change in age on hippocampal volume ( = 0.556 mm<sup>3</sup>/pt*a, q = 0.04, BF = 0.5), indicating accelerated brain aging in more isolated individuals. Moreover, social isolation cross-sectionally and longitudinally correlated with lower cortical thickness in multiple clusters in the orbitofrontal cortex, precuneus and other areas (FDR q &lt; 0.05).</p></sec><sec><title>Conclusions</title><p>Here, we provide evidence that social isolation contributes to hippocampal and cortical atrophy and subtle cognitive decline in non-demented mid-to late-life adults. Importantly, within-subject effects of social isolation were similar to between-subject effects, indicating an opportunity for targeting social isolation to reduce dementia risk.</p></sec>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472595', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'NAD+ capping of RNA in Archaea and Mycobacteria', 'authors': \"[{'name': 'Olatz Ruiz-Larrabeiti'}, {'name': 'Roberto Benoni'}, {'name': 'Viacheslav Zemlianski'}, {'name': 'Nikola Hanikov'}, {'name': 'Marek Schwarz'}, {'name': 'Barbora Brezovsk'}, {'name': 'Barbora Benoni'}, {'name': 'Jarmila Hnilicov'}, {'name': 'Vladimir R. Kaberdin'}, {'name': 'Hana Cahov'}, {'name': 'Monika Vtzov'}, {'name': 'Martin Pevorovsk'}, {'name': 'Libor Krsn'}]\", 'abstract': '<p>Chemical modifications of RNA affect essential properties of transcripts, such as their translation, localization and stability. 5-end RNA capping with the ubiquitous redox cofactor nicotinamide adenine dinucleotide (NAD<sup>+</sup>) has been discovered in organisms ranging from bacteria to mammals. However, the hypothesis that NAD<sup>+</sup> capping might be universal in all domains of life has not been proven yet, as information on this RNA modification is missing for Archaea. Likewise, this RNA modification has not been studied in the clinically important <italic>Mycobacterium</italic> genus. Here, we demonstrate that NAD<sup>+</sup> capping occurs in the archaeal and mycobacterial model organisms <italic>Methanosarcina barkeri</italic> and <italic>Mycobacterium smegmatis.</italic> Moreover, we identify the NAD<sup>+</sup>-capped transcripts in <italic>M. smegmatis,</italic> showing that this modification is more prevalent in stationary phase, and revealing that mycobacterial NAD<sup>+</sup>-capped transcripts include non-coding small RNAs, such as Ms1. Furthermore, we show that mycobacterial RNA polymerase incorporates NAD<sup>+</sup> into RNA, and that the genes of NAD<sup>+</sup>-capped transcripts are preceded by promoter elements compatible with <sup>A</sup>/<sup>F</sup> dependent expression. Taken together, our findings demonstrate that NAD<sup>+</sup> capping exists in the archaeal domain of life, suggesting that it is universal to all living organisms, and define the NAD<sup>+</sup>-capped RNA landscape in mycobacteria, providing a basis for its future exploration.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.473023', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Auxin-driven ecophysiological diversification of leaves in domesticated tomato', 'authors': \"[{'name': 'Juliene dos Reis Moreira'}, {'name': 'Bruno Luan Rosa'}, {'name': 'Bruno S. Lira'}, {'name': 'Joni E. Lima'}, {'name': 'Ludmila N. Souza'}, {'name': 'Wagner C. Otoni'}, {'name': 'Antonio Figueira'}, {'name': 'Luciano Freschi'}, {'name': 'Tetsu Sakamoto'}, {'name': 'Lazaro Peres'}, {'name': 'Magdalena Rossi'}, {'name': 'Agustin Zsogon'}]\", 'abstract': '<p>The study of crop diversification has focussed mainly on the genetic changes underlying traits favoured by humans. However, the passage from natural habitats to agronomic settings probably operated changes beyond those comprising the classical domestication syndrome. A deeper understanding of these traits and their genetic signature would be valuable to inform conventional crop breeding and de novo domestication of crop wild relatives. Heterobaric leaves have bundle sheath extensions (BSEs) that compartmentalise the sub-stomatal cavity whereas homobaric leaves do not; BSE development is known to be controlled by the OBSCURAVENOSA (OBV) locus and the obv mutant lacks BSEs whereas leaves carrying the wild type allele have BSEs. Here we identify a non-synonymous amino acid change in the OBV gene that exists as a rare balanced polymorphism in the natural range of wild tomatoes, but has increased in frequency in domesticated tomatoes, suggesting that the latter diversified into heterobaric and homobaric leaf types. The mutation disrupts a C2H2 zinc finger motif in the OBV protein, resulting in the absence of BSEs in leaves and here we show that this and other pleiotropic effects, including changes in leaf insertion angle, leaf margin serration, minor vein density and fruit shape, are controlled by OBV via changes in auxin signalling. Loss of function of the transcriptional regulator AUXIN RESPONSE FACTOR (ARF4) also results in defective BSE development, revealing an additional component of a novel genetic module controlling aspects of leaf development important for ecological adaptation and subject to breeding selections.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.15.472782', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'DNMT3A knockouts in human iPSCs prevent de novo DNA methylation and reveal growth advantage during hematopoietic differentiation', 'authors': \"[{'name': 'Olivia Cypris'}, {'name': 'Julia Franzen'}, {'name': 'Joana Frobel'}, {'name': 'Philipp Glck'}, {'name': 'Chao-Chung Kuo'}, {'name': 'Stephani Schmitz'}, {'name': 'Selina Nchtern'}, {'name': 'Martin Zenke'}, {'name': 'Wolfgang Wagner'}]\", 'abstract': '<p>DNA methyltransferase 3A (<italic>DNMT3A</italic>) is a frequently mutated gene in many hematological malignancies, indicating that it may be essential for hematopoietic differentiation. Here, we addressed the functional relevance of DNMT3A for differentiation of human induced pluripotent stem cells (iPSCs) by knocking out exon 2, 19, or 23. Exon 19<sup>-/-</sup> and 23<sup>-/-</sup> lines revealed absence of almost the entire <italic>de novo</italic> DNA methylation during mesenchymal and hematopoietic differentiation. Yet, differentiation was only slightly reduced in exon 19<sup>-/-</sup> and increased in exon 23<sup>-/-</sup> lines, whereas there was no significant impact on gene expression in hematopoietic progenitors (iHPCs). Notably, <italic>DNMT3A</italic><sup>-/-</sup> iHPCs recapitulate some DNA methylation differences of acute myeloid leukemia with <italic>DNMT3A</italic> mutations. Furthermore, multicolor genetic barcoding revealed competitive growth advantage of exon 23<sup>-/-</sup> iHPCs. Our results demonstrate that <italic>de novo</italic> DNA methylation during hematopoietic differentiation of iPSCs is almost entirely dependent on DNMT3A and exon 23<sup>-/-</sup> iHPCs even gained growth advantage.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472453', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Physical model of electrical synapses in a neural network', 'authors': \"[{'name': 'M. Pekker'}, {'name': 'M.N. Shneider'}]\", 'abstract': '<p>A theoretical model of electrical synapses is proposed, in which connexons play the role of nails that hold unmyelinated areas of neurons at a distance of about 3.5 nm, and the electrical connection between them is provided by charging the membrane of an inactive neuron with currents generated in the intercellular electrolyte (saline) by the action potential in the active neuron. This mechanism is similar to the salutatory conduction of the action potential between the nodes of Ranvier in myelinated axons and the ephaptic coupling of sufficiently close spaced neurons.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.15.472761', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The neural cell adhesion molecule NrCAM regulates development of hypothalamic tanycytes', 'authors': \"[{'name': 'Alex Moore'}, {'name': 'Kavitha Chinnaiya'}, {'name': 'Dong Won Kim'}, {'name': 'Sarah Brown'}, {'name': 'Iain Stewart'}, {'name': 'Sarah Robins'}, {'name': 'Georgina Dowsett'}, {'name': 'Charlotte Muir'}, {'name': 'Marco Travaglio'}, {'name': 'Jo E Lewis'}, {'name': 'Fran Ebling'}, {'name': 'Seth Blackshaw'}, {'name': 'Andrew Furley'}, {'name': 'Marysia Placzek'}]\", 'abstract': '<p>Hypothalamic tanycytes are neural stem and progenitor cells, but little is known of how they are regulated. Here we provide evidence that the cell adhesion molecule, NrCAM, regulates tanycytes in the adult niche. NrCAM is strongly expressed in adult mouse tanycytes. Immunohistochemical and in situ hybridization analysis revealed that NrCAM loss of function leads to both a reduced number of tanycytes and reduced expression of tanycyte-specific cell markers, along with a small reduction in tyrosine hydroxylase-positive arcuate neurons.  Similar analyses of NrCAM mutants at E16 identify few changes in gene expression or cell composition, indicating that NrCAM regulates tanycytes, rather than early embryonic hypothalamic development. Neurosphere and organotypic assays support the idea that NrCAM governs cellular homeostasis. Single-cell RNA sequencing (scRNA-Seq) shows that tanycyte-specific genes, including a number that are implicated in thyroid hormone metabolism, show reduced expression in the mutant mouse. However, the mild tanycyte depletion and loss of markers observed in NrCAM-deficient mice were associated with only a subtle metabolic phenotype.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.472455', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Identifying Protein Features and Pathways Responsible for Toxicity using Machine learning, CANDO, and Tox21 datasets: Implications for Predictive Toxicology', 'authors': \"[{'name': 'Lama Moukheiber'}, {'name': 'William Mangione'}, {'name': 'Saeed Maleki'}, {'name': 'Zackary Falls'}, {'name': 'Mingchen Gao'}, {'name': 'Ram Samudrala'}]\", 'abstract': '<p>Humans are exposed to numerous compounds daily, some of which have adverse effects on health. Computational approaches for modeling toxicological data in conjunction with machine learning algorithms have gained popularity over the last few years. Machine learning methods have been used to predict toxicity-related biological activities using chemical structure descriptors. However, proteomic features have not been fully investigated. In this study, we construct a computational model using machine learning for selecting the most important proteins representing features in predicting the toxicity of the compounds in the Tox21 dataset using the multiscale Computational Analysis of Novel Drug Opportunities (CANDO) platform for therapeutic discovery. Tox21 is a highly imbalanced dataset consisting of twelve in-vitro assays, seven from the nuclear receptor (NR) signaling pathway and five from the stress response (SR) pathway, for more than 10,000 compounds. For our computational model, we employed a random forest (RF) with the combination of Synthetic Minority Oversampling Technique (SMOTE) and Edited Nearest Neighbor (ENN) method, aka SMOTE+ENN, which is resampling method to balance the activity class distribution. Within the NR and SR pathways, the activity of the aryl hydrocarbon receptor (NR-AhR), toxicity mediating transcription factor, and microchondrial membrane potential (SR-MMP) were two of the top-performing twelve toxicity endpoints with AUROCs of 0.90 and 0.92, respectively. The top extracted features for evaluating compound toxicity were passed into enrichment analysis to highlight the implicated biological pathways and proteins. We validated our enrichment results for the activity of the AhR using a thorough literature search. Our case study showed that the selected enriched pathways and proteins from our computational pipeline are not only correlated with NR-AhR toxicity but also form a cascading upstream/downstream arrangement. Our work elucidates significant relationships between protein and compound interactions computed using CANDO and the associated biological pathways to which the proteins belong, with twelve toxicity endpoints. This novel study uses machine learning not only to predict and understand toxicity but also elucidates therapeutic mechanisms at a proteomic level for a variety of toxicity endpoints.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.15.472760', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Simple contextual cueing prevents retroactive interference in short-term perceptual training of orientation detection tasks', 'authors': \"[{'name': 'Hui Huang'}, {'name': 'Yangming Zhang'}, {'name': 'Sheng Li'}]\", 'abstract': '<p>Perceptual training of multiple tasks suffers from interference between the trained tasks. Here, we conducted four psychophysical experiments with separate groups of participants to investigate the possibility of preventing the interference in short-term perceptual training. We trained the participants to detect two orientations of Gabor stimuli in two adjacent days at the same retinal location and examined the interference of training effects between the two orientations. The results showed significant retroactive interference from the second orientation to the first orientation (Experiments 1 and 2). Introducing a 6-hour interval between the pre-test and training of the second orientation did not eliminate the interference effect, excluding the interpretation of disrupted reconsolidation as the pre-test of the second orientation may reactivate and destabilize the representation of the first orientation (Experiment 3). Finally, the training of the two orientations was accompanied by fixations in two colors, each served as a contextual cue for one orientation. The results showed that the retroactive interference was not evident after introducing these passively perceived contextual cues (Experiment 4). Our findings suggest that the retroactive interference effect in short-term perceptual training of orientation detection tasks was likely the result of higher-level factors such as shared contextual cues embedded in the tasks. The effect of multiple perceptual training could be facilitated by associating the trained tasks with different contextual cues.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.472427', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The influence of Janthinobacterium sp. strain SLB01 on the cell culture of primmorphs of the sponge Lubomirskia baicalensis', 'authors': \"[{'name': 'Lubov I. Chernogor'}, {'name': 'Marina G. Eliseikina'}, {'name': 'Ivan S. Petrushin'}, {'name': 'Ekaterina A. Chernogor'}, {'name': 'Igor V. Khanaev'}, {'name': 'Sergei I. Belikov'}]\", 'abstract': '<p>Sponges (phylum Porifera) are ancient, filtering, multicellular metazoans. Freshwater sponges (Demosponges, Lubomirskiidae) dominate the fauna of the littoral zone of Lake Baikal. Over the last years, there have been mass diseases and death of endemic sponges. Previously, the strain <italic>Janthinobacterium</italic> sp. SLB01 was isolated from the diseased sponge <italic>Lubomirskia baicalensis</italic>. The studies of the pathogenicity of the strain <italic>Janthinobacterium</italic> sp. SLB01 for Baikal sponges has not been carried out. Therefore we infected experimentally <italic>in vitro</italic> to determine its pathogenicity by the cell culture of the primmorphs with subsequent isolation, sequencing, and analysis of the genomes. The purpose of the study is to show that the strain <italic>Janthinobacterium</italic> sp. SLB01 isolated from the diseased sponge <italic>L. baicalensis</italic> is a pathogen for the cell culture of primmorphs. The bacteria from the infected samples of the primmorphs were isolated and identified as strain <italic>Janthinobacterium</italic> sp. PLB02. A comparative analysis of the genomes of the strains <italic>Janthinobacterium</italic> sp. SLB01 and <italic>Janthinobacterium</italic> sp. PLB02 showed that they are practically identical. The genomes of both strains contain genes vioABCDE violacein, flok formation, and strong biofilm, and the type VI secretion system (T6SS), as the primary virulence factor. The bacterial strains <italic>Janthinobacterium</italic> sp. based on a comparison of complete genomes showed similarity with strain <italic>Janthinobacterium lividum</italic> MTR. Isolated our strains of <italic>Janthinobacterium</italic> sp. are pathogens for cell cultures of primmorphs and <italic>L. baicalensis</italic> sponges. The results of the study will help to expand the understanding of microbial relationships in the development of disease and the death of Baikal sponges.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472422', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Zoology', 'covid': False, 'title': 'Gross anatomy of the gluteal and posterior thigh muscles in koalas based on their innervations', 'authors': \"[{'name': 'Sayaka Tojima'}, {'name': 'Hidaka Anetai'}, {'name': 'Kaito Koike'}, {'name': 'Saori Anetai'}, {'name': 'Kounosuke Tokita'}, {'name': 'Chris Leigh'}, {'name': 'Jaliya Kumaratilake'}]\", 'abstract': '<p>There are not many descriptions of the muscle morphology of marsupials, despite the fact that they should show diversity according to the adaptation and dispersal to a variety of environments. Most of the previous studies regarding the gross anatomy of marsupials were conducted in the 1800 - 1900s, and many issues still remain that need to be reexamined. For instance, the muscle identification had been performed based only on their attachments and thus, muscle descriptions are often inconsistent among the studies. These classic studies often do not include figures or photographs, so the discrepancies in the descriptions of the muscles could only be verified by performing the muscle identification again with a more reliable method. This problem can be solved by performing muscle identification by innervation. This method, which focuses on the ontogenic origin of the muscle as opposed to the attachment site, is prone to individual and interspecies variation and is a common technique in recent anatomical research. This technique is more reliable than previous methods and is suitable for comparison with other taxa (i.e., eutherians). In this study, we first conducted muscle identification based on innervation in the gluteal and posterior thighs of koalas in order to reorganize the anatomical knowledge of marsupials. This is because the gluteus and posterior thighs of koalas are the areas where previous studies have been particularly inconsistent. We dissected five individual koalas and clarified discrepancies in previous studies, as well as investigated the unique muscle morphology and their function in koalas. Specifically, the koalas gluteal muscle group is suitable for abduction, while the posterior thigh muscles are particularly suitable for flexion. In the future, we will update the anatomical findings of marsupials in the same way to clarify the adaptive dissipation process of marsupials, as well as to contribute to the understanding of the evolutionary morphology of mammals.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.15.472660', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Differential Impact of rhamnolipids and TWEEN80 on composition and functionality of the gut microbiota in the SHIME system', 'authors': \"[{'name': 'Lisa Miclotte'}, {'name': 'Ellen De Paepe'}, {'name': 'Qiqiong Li'}, {'name': 'John Van Camp'}, {'name': 'Andreja Raikovic'}, {'name': 'Tom Van de Wiele'}]\", 'abstract': '<p>Dietary emulsifiers have been shown to affect the composition and function of the gut microbial community, both <italic>in vivo</italic> and <italic>in vitro</italic>. Yet, several knowledge gaps remain to be addressed: the impact from a longer timeframe exposure on the gut microbiota, interindividual variability in microbiome response and the putative impact from novel clean label alternatives for current food emulsifiers.</p><p>In the present study, the impact of one conventional dietary emulsifier, TWEEN80, and one potential novel alternative, rhamnolipids, on the human gut microbiota was investigated using the Mucosal Simulator of the Human Intestinal Microbial Ecosystem (M-SHIME). The faecal microbiota from two human donors, with high and low responsiveness to the emulsifiers, were exposed to 0,05 m% and 0,5 m% of the emulsifiers for 7 days.</p><p>The results confirmed previous observations that the effects on the composition and functionality are both emulsifier- and donor dependent. The effects reached an equilibrium after about 3 days of exposure. Overall, TWEEN80 and rhamnolipids displayed opposite effects: TWEEN80 increased cell counts, reduced propionate concentration, increased butyrate levels, increased a.o. <italic>Faecalibacterium, Blautia</italic> and <italic>Hungatella</italic> abundance, while rhamnolipids did the opposite. Rhamnolipids also sharply increased the abundance of unclassified Lachnospiraceae. On the other hand, both emulsifiers increased the relative abundance of unclassified Enterobacteriaceae. Both emulsifiers also altered the microbial metabolome in different ways and a pathway enrichment analysis tool revealed that the metabolome alterations could be reminiscent of gut issues and obesity.</p><p>Overall, the impact from the rhamnolipids was larger than that of TWEEN80 at similar concentrations, indicating that the former may not necessarily be a safer alternative for the latter. The microbiotas response also depended on its original composition and the sensitivity status for which the faecal donors were selected, was preserved. Whether the same donor-diversity and longitudinal impact can be expected in the human colon as well and what impact this has on the host will have to be further investigated.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.17.21267986', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Magnitude, global variation, and temporal development of the COVID-19 infection fatality burden', 'authors': \"[{'name': 'Christina Bohk-Ewald'}, {'name': 'Enrique Acosta'}, {'name': 'Tim Riffe'}, {'name': 'Christian Dudel'}, {'name': 'Mikko Myrskyla'}]\", 'abstract': '<p>How deadly is an infection with SARS-CoV-2 worldwide over time? This information is critical for developing and assessing public health responses on the country and global levels. However, imperfect data have been the most limiting factor for estimating the COVID-19 infection fatality burden during the first year of the pandemic. Here we leverage recently emerged compelling data sources and broadly applicable modeling strategies to estimate the crude infection fatality rate (cIFR) in 77 countries from 28 March 2020 to 31 March 2021, using 2.4 million reported deaths and estimated 435 million infections by age, sex, country, and date. The global average of all cIFR estimates is 1.2% (10th to 90th percentile: 0.2% to 2.4%). The cIFR varies strongly across countries, but little within countries over time, and it is often lower for women than men. Cross-country differences in cIFR are largely driven by the age structures of both the general and the truly infected population. While the broad trends and patterns of the cIFR estimates are more robust, we show that their levels are uncertain and sensitive to input data and modeling choices. In consequence, increased efforts at collecting high-quality data are essential for accurately estimating the cIFR, which is a key indicator for better understanding the health and mortality consequences of this pandemic.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.472113', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Tissue Optimisation Strategies for High Quality Ex Vivo Diffusion Imaging', 'authors': \"[{'name': 'Rachel L. C. Barrett'}, {'name': 'Diana Cash'}, {'name': 'Camilla Simmons'}, {'name': 'Eugene Kim'}, {'name': 'Tobias C. Wood'}, {'name': 'Richard Stones'}, {'name': 'Anthony C. Vernon'}, {'name': 'Marco Catani'}, {'name': 'Flavio DellAcqua'}]\", 'abstract': '<p><italic>Ex vivo</italic> diffusion imaging can be used to study healthy and pathological tissue microstructure in the rodent brain with microscopic resolution, providing a link between <italic>in vivo</italic> MRI and <italic>ex vivo</italic> microscopy techniques. A major challenge for the successful acquisition of <italic>ex vivo</italic> diffusion imaging data however are changes in the relaxivity and diffusivity of brain tissue following perfusion fixation. In this study we address this question by examining the combined effects of tissue preparation factors that influence image quality, including tissue rehydration time, fixative concentration and contrast agent concentration. We present an optimisation strategy combining these factors to manipulate the T1 and T2 of fixed tissue and maximise signal-to-noise ratio (SNR) efficiency. Applying this strategy in the rat brain resulted in a doubling of SNR and an increase in SNR per unit time by 135% in grey matter and 88% in white matter. This enabled the acquisition of excellent quality high-resolution (78 m isotropic voxel size) diffusion data in less than 4 days, with a <italic>b</italic>-value of 4000 s/mm<sup>2</sup>, 30 diffusion directions and a field of view of 40  13  18 mm, using a 9.4 Tesla scanner with a standard 39 mm volume coil and a 660 mT/m 114 mm gradient insert. It was also possible to achieve comparable data quality for a standard resolution (150 m) diffusion dataset in 2<sup>1</sup>/<sub>4</sub> hours. In conclusion, the optimisation strategy presented here may be used to improve signal quality, increase spatial resolution and/or allow faster acquisitions in preclinical <italic>ex vivo</italic> diffusion MRI experiments.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.15.471847', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Dynamic interplay between structural variations and 3D chromosome organization in pancreatic cancer', 'authors': \"[{'name': 'Yongxing Du'}, {'name': 'Zongting Gu'}, {'name': 'Zongze Li'}, {'name': 'Zan Yuan'}, {'name': 'Yue Zhao'}, {'name': 'Xiaohao Zheng'}, {'name': 'Xiaochen Bo'}, {'name': 'Hebing Chen'}, {'name': 'Chengfeng Wang'}]\", 'abstract': '<p>Structural variations (SVs) are the greatest source of variation in the genome and can lead to oncogenesis. However, the identification and interpretation of SVs in human pancreatic cancer remain largely undefined due to technological limitations. Here, we investigate the spectrum of SVs and three-dimensional (3D) chromatin architecture in human pancreatic ductal epithelial cell carcinogenesis by using state-of-the-art long-read single-molecule real-time (SMRT) and high-throughput chromosome conformation capture (Hi-C) sequencing techniques. We find that the 3D genome organization is remodeled and correlated with gene expressional change. The bulk remodeling effect of cross-boundary SVs in the 3D genome partly depends on intercellular genomic heterogeneity. Meanwhile, contact domains tend to minimize these disrupting effects of SVs within local adjacent genomic regions to maintain overall stability of 3D genome organization. Moreover, our data also demonstrates complex genomic rearrangements involving two key driver genes CDKN2A and SMAD4, and elucidates their influence on cancer-related gene expression from both linear view and 3D perspective. Overall, this study provides a valuable resource and highlights the impact, complexity and dynamicity of the interplay between SVs and 3D genome organization, which further expands our understanding of pathogenesis of SVs in human pancreatic cancer.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.13.472498', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Arousal elevation drives the development of oscillatory vocal output', 'authors': \"[{'name': 'Yisi S. Zhang'}, {'name': 'John L. Alvarez'}, {'name': 'Asif A. Ghazanfar'}]\", 'abstract': '<p>Adult behaviors, such as vocal production, often exhibit temporal regularity. In contrast, their immature forms are more irregular. We ask whether the coupling of motor behaviors with arousal changes give rise to temporal regularity and drive the transition from variable to regular motor output over the course of development. We used marmoset monkey vocal production to explore this putative influence of arousal on the nonlinear changes in their developing vocal output patterns. Based on a detailed analysis of vocal and arousal dynamics in marmosets, we put forth a general model incorporating arousal and auditory-feedback loops for spontaneous vocal production. Using this model, we show that a stable oscillation can emerge as the baseline arousal increases, predicting the transition from stochastic to periodic oscillations occurring in marmoset vocal development. We further provide a solution for how this model can explain vocal development as the joint consequence of energetic growth and social feedback. Together, our model offers a plausible mechanism for the development of arousal-mediated adaptive behavior.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472339', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Stratified microbial communities are highly sensitive towards multiple combined global change factors, revealing antagonistic and synergistic effects', 'authors': \"[{'name': 'Marcel Suleiman'}, {'name': 'Yves Choffat'}, {'name': 'Xue Zheng'}, {'name': 'Owen L Petchey'}]\", 'abstract': '<p>Microbial communities in many ecosystems are facing a broad range of global change scenarios, resulting in microbial changes and possibly regime shifts with unknown ecological consequences. While the influence of single stressors is already described in numerous studies, the effects of multiple stressors working simultaneously are still poorly understood. In this study, we used 240 highly replicable oxic/anoxic aquatic lab micro-ecosystems to understand the influence of four stressors (fertilizer, glyphosate, metal pollution, antibiotics) in all possible combinations at three different temperatures (20 C, 24 C, and 28 C) to shed light into consequences of multiple stressors on different levels of organization, ranging from species abundance to community and ecosystem parameters. Our data reveal that (i) combination of specific stressors can change the biological consequence and direction compared to single stressors in all levels of organisation (ii), effects of stressor combinations are modified by temperature, and (iii) that the number of stressors applied also lead to significant changes. In sum, our study confirmed the need of investigating multiple stressors working simultaneously across different ecological levels of organisation.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472348', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Integrative Interactome and Ubiquitinome Analyses Reveal Multiple Regulatory Pathways Targeted by ICP0 in HSV-1 Infected Neuronal Cells', 'authors': \"[{'name': 'Fujun Hou'}, {'name': 'Zeyu Sun'}, {'name': 'Yue Deng'}, {'name': 'Siyu Chen'}, {'name': 'Xiyuan Yang'}, {'name': 'Feiyang Ji'}, {'name': 'Menghao Zhou'}, {'name': 'Keyi Ren'}, {'name': 'Dongli Pan'}]\", 'abstract': '<p>Herpes simplex virus 1 (HSV-1) is a neurotropic virus that can undergo both productive and latent infection in neurons. ICP0 is an HSV-1 E3 ubiquitin ligase crucial for productive infection and reactivation from latency. However, its targets have not been systematically investigated in neuronal cells. After confirming the importance of ICP0 in HSV-1 neuronal replication using an ICP0-null virus, we identified many ICP0-interacting proteins in infected neuronal and non-neuronal cells by mass-spectrometry-based interactome analysis. Co-immunoprecipitation assays validated ICP0 interactions with ACOT8, C1QBP, OTUD4, SNX9 and VIM in both Neuro-2a and 293T cells. Overexpression and knockdown experiments showed that SNX9 restricted replication of the ICP0-null but not wild-type virus in Neuro-2a cells. Ubiquitinome analysis by immunoprecipitating the trypsin digested ubiquitin reminant followed by mass spectrometry identified numerous candidate ubiquitination substrates of ICP0 in infected Neuro-2a cells, among which OTUD4 and VIM were novel substrates confirmed to be ubiquitinated by transfected ICP0 in Neuro-2a cells despite no evidence of their degradation by ICP0. Expression of OTUD4 was induced independently of ICP0 during HSV-1 infection. Overexpressed OTUD4 enhanced type I interferon expression during infection with the ICP0-null but not wild-type virus. In summary, by combining two proteomic approaches followed by confirmatory and functional experiments, we identified and validated multiple novel targets of ICP0 in neuronal cells, and revealed potential restrictive activities of SNX9 and OTUD4 as well as ICP0-dependent antagonism of these activities.</p><sec><title>Author Summary</title><p>Herpes simplex virus 1 (HSV-1) establishes latent infection in neurons. ICP0 is known for its critical role in antagonizing cellular restrictive functions thereby initiating productive infection. It has been demonstrated to be important for both acute infection and reactivation from latency in neurons. However, little is known about its targets in neuronal cells. Here we combined two proteomic approaches, interactome and ubiquitinome analyses, to integratively identify interaction partners and substrates of ICP0 in HSV-1 infected neuronal cells. The results identified many novel targets as well as confirming previously reported ones. We also further validated some of the binding interactions and ubiquitin modifications. Functional studies revealed that the ICP0-interacting protein SNX9 restricted HSV-1 replication and the ICP0 substrate OTUD4 was induced to enhance type I interferon expression during HSV-1 neuronal infection. Moreover, the activities of these proteins appeared to be antagonized by ICP0-dependent mechanisms. This study provided comprehensive insight into ICP0 targets in neuronal cells and might prompt further investigation into the newly identified targets of ICP0.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.15.472736', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Photoaffinity capture compounds to profile the Magic Spot Nucleotide interactomes', 'authors': \"[{'name': 'Thomas M. Haas'}, {'name': 'Benot-Joseph Laventie'}, {'name': 'Simon Lagies'}, {'name': 'Caroline Harter'}, {'name': 'Isabel Prucker'}, {'name': 'Danilo Ritz'}, {'name': 'Raspudin Saleem Batcha'}, {'name': 'Danye Qiu'}, {'name': 'Wolfgang Httel'}, {'name': 'Jennifer Andexer'}, {'name': 'Urs Jenal'}, {'name': 'Henning J. Jessen'}]\", 'abstract': '<p>Magic Spot Nucleotides (MSN) regulate the stringent response, a highly conserved bacterial stress adaptation mechanism, enabling survival when confronted with adverse external challenges. In times of antibiotic crisis, a detailed understanding of the stringent response is of critical importance, as potentially new targets for pharmacological intervention could be identified. In this study, we delineate the MSN interactome in <italic>Escherichia coli</italic> and <italic>Salmonella typhimurium</italic> cell lysates applying a family of trifunctional photoaffinity capture compounds. We introduce different MSN probes covering diverse phosphorylation patterns, such as pppGpp, ppGpp, and pGpp. Our chemical proteomics approach provides datasets of diverse putative MSN receptors both from cytosolic and membrane fractions that, upon validation, unveil new MSN targets. We find, for example, that the dinucleoside polyphosphate hydrolase activity of the non-Nudix hydrolase ApaH is potently inhibited by pppGpp, which itself is converted to pGpp by ApaH. The photoaffinity capture compounds described herein will be useful to identify MSN interactomes under varying conditions and across bacterial species.</p><sec id=\"s4\"><title>TOC</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><caption><p><bold><italic>Molecular fishing:</italic></bold> a family of trifunctional photoaffinity capture compounds enables the identification of Magic Spot Nucleotide receptors by a chemoproteomics approach.</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"472736v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.13.472529', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Simulated colonic fluid replicates the in vivo growth capabilities of Citrobacter rodentium mutants and highlights a critical role for the Cpx envelope stress response in mediating stressors encountered in the gastrointestinal tract', 'authors': \"[{'name': 'Ashley Gilliland'}, {'name': 'Christina Gavino'}, {'name': 'Samantha Gruenheid'}, {'name': 'Tracy Raivio'}]\", 'abstract': '<p><italic>Citrobacter rodentium</italic> is an attaching and effacing (A/E) pathogen used as a model for enteropathogenic and enterohemorrhagic <italic>Escherichia coli</italic> infections in mice. While in the host, <italic>C. rodentium</italic> must adapt to stresses in the gastrointestinal tract such as antimicrobial peptides, pH changes, and bile salts. The Cpx envelope stress response (ESR) is a two-component system used by some bacteria to remediate stress by modulating gene expression and is necessary for <italic>C. rodentium</italic> pathogenesis in mice. To investigate genes in the Cpx regulon that may contribute to <italic>C. rodentium</italic> pathogenesis, RNA-Seq, SILAC, and microarray data from previous research was mined and the genes <italic>yebE, ygiB, bssR,</italic> and <italic>htpX</italic> were confirmed to be strongly upregulated by the presence of CpxRA using <italic>lux</italic> reporter constructs. To determine the function of these genes <italic>in vivo,</italic> knockout mutants were tested in C57Bl/6J and C3H/HeJ mice. Although none of the mutants exhibited marked virulence phenotypes, the <italic>cpxRA</italic> mutant had reduced colonization and attenuated virulence, as previously determined. We also found that the absence of the Cpx ESR resulted in higher expression of the LEE master regulator, <italic>ler.</italic> In addition, we determined that the <italic>cpxRA</italic> mutant had a growth defect in medium simulating the colon, as did several of the mutants bearing deletions in Cpx-upregulated genes. Overall, these results indicate that the <italic>cpxRA</italic> virulence defect is not due to any single Cpx regulon gene examined. Instead, attenuation may be the result of defective growth in the colonic environment resulting from the collective impact of multiple Cpx-regulated genes.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472434', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Adenine Base Editing in vivo with a Single Adeno-Associated Virus Vector', 'authors': \"[{'name': 'Han Zhang'}, {'name': 'Nathan Bamidele'}, {'name': 'Pengpeng Liu'}, {'name': 'Ogooluwa Ojelabi'}, {'name': 'Xin D. Gao'}, {'name': 'Tomas Rodriguez'}, {'name': 'Haoyang Cheng'}, {'name': 'Scot A. Wolfe'}, {'name': 'Wen Xue'}, {'name': 'Erik J. Sontheimer'}]\", 'abstract': '<p>Base editors (BEs) have opened new avenues for the treatment of genetic diseases. However, advances in delivery approaches are needed to enable disease targeting of a broad range of tissues and cell types. Adeno-associated virus (AAV) vectors remain one of the most promising delivery vehicles for gene therapies. Currently, most BE/guide combinations and their promoters exceed the packaging limit (~5 kb) of AAVs. Dual-AAV delivery strategies often require high viral doses that impose safety concerns. In this study, we engineered an adenine base editor using a compact Cas9 from Neisseria meningitidis (Nme2Cas9). Compared to the well-characterized Streptococcus pyogenes Cas9-containing ABEs, Nme2-ABE possesses a distinct PAM (N4CC) and editing window, exhibits fewer off-target effects, and can efficiently install therapeutically relevant mutations in both human and mouse genomes. Importantly, we show that in vivo delivery of Nme2-ABE and its guide RNA by a single-AAV vector can efficiently edit mouse genomic loci and revert the disease mutation and phenotype in an adult mouse model of tyrosinemia. We anticipate that Nme2-ABE, by virtue of its compact size and broad targeting range, will enable a range of therapeutic applications with improved safety and efficacy due in part to packaging in a single-vector system.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.16.472840', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Chronic Seizures Worsen Cognitive Function with Loss of Presenilin 2 Function in a Sex-Related Manner', 'authors': \"[{'name': 'Kevin M Knox'}, {'name': 'Megan Beckman'}, {'name': 'Carole L Smith'}, {'name': 'Suman Jayadev'}, {'name': 'Melissa Barker-Haliski'}]\", 'abstract': \"<p>Patients with early-onset Alzheimer's disease (EOAD) are at elevated risk for seizures, including patients with variants in presenilin 2 (PSEN2). Like patients with epilepsy, untreated seizures may also worsen cognitive function in AD. We therefore sought to define the impact of loss of normal PSEN2 function and chronic seizures on cognitive function in the aged brain. Male and female PSEN2 KO and age- and sex-matched wild-type (WT) mice were sham or corneal kindled beginning at 6-months-old. Kindled and sham-kindled mice were then challenged up to 6 weeks later in a battery of cognitive tests: non-habituated open field (OF), T-maze spontaneous alternation (TM), and Barnes maze (BM), followed by immunohistochemistry for markers of neuroinflammation and neuroplasticity. PSEN2 KO mice required significantly more stimulations to kindle (males: p&lt;0.02; females: p&lt;0.02). Chronic seizures more significantly worsened anxiety-like behaviors of female PSEN2 KO mice as measured by percentage of total time exploring the center of an OF. TM performance was significantly worsened in kindled female (p=0.024), but not male, PSEN2 KO mice. Male BM performance was generally worsened by seizures (p=0.038), but not by genotype. Conversely, kindled PSEN2 KO females made the most BM errors (p=0.007). Chronic seizures also significantly altered expression of hippocampal neuroinflammation and neuroplasticity markers in a sex-specific manner. This study demonstrates that hippocampus-dependent cognitive function may be only worsened by uncontrolled chronic seizures in female, but not male, PSEN2 KO mice. Our work aligns with clinical evidence of sex-specific worsening of AD burden and supports sex-specific anticonvulsant interventions in AD.</p>\", 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.21267804', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Emerging from the COVID-19 pandemic: impacts of variants, vaccines, and duration of immunity', 'authors': \"[{'name': 'Ken Newcomb'}, {'name': 'Edwin Michael'}]\", 'abstract': '<p>The advent of vaccines against SARS-CoV-2 and the roll out of mass vaccination programs are thought to present the most effective means to control and even end the ongoing pandemic. However, uncertainties connected with the partial effectiveness of present vaccines, duration of immunity against SARS-CoV-2, and potential impact of variant dynamics, mean that it is still possible that the contagion could follow different future paths in different communities. Here, we use an extended SEIR for SARS-COV-2 transmission sequentially calibrated to data on cases and interventions implemented in the state of Florida to explore how these factors may interact to govern potential pandemic futures. Our data-driven forecasts indicate while the introduction of vaccinations could lead to the permanent, albeit drawn-out, ending of the pandemic if the immunity generated through vaccinations and natural infections acts over the long-term, additional futures could become possible if this immunity wanes over time. These futures will be marked by repeated waves of infection, the amplitude and periodicity of which will depend on the duration over which the immunity generated in a population will operate. We conclude that the possibility of these complex futures will require continual vigilance and perhaps fundamental changes in societal responses if we are to effectively control SARS-CoV-2.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.14.453704', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Identifying Uncertainty States during Wayfinding in Indoor Environments: An EEG Classification Study', 'authors': \"[{'name': 'Bingzhao Zhu'}, {'name': 'Jesus G Cruz-Garza'}, {'name': 'Mahsa Shoaran'}, {'name': 'Saleh Kalantari'}]\", 'abstract': '<p>The researchers used a machine-learning classification approach to better understand neurological features  associated with periods of wayfinding uncertainty. The participants (n=30) were asked to complete wayfinding tasks of varying difficulty in a virtual reality (VR) hospital environment. Time segments when participants experienced navigational uncertainty were first identified using a combination of objective measurements (frequency of inputs into the VR controller) and behavioral annotations from two independent observers. Uncertainty time-segments during navigation were ranked on a scale from 1 (low) to 5 (high). The machine-learning model, a random forest classifier implemented using scikit-learn in Python, was used to evaluate common spatial patterns of EEG spectral power across the theta, alpha, and beta bands associated with the researcher-identified uncertainty states. The overall predictive power of the resulting model was 0.70 in terms of the area under the Receiver Operating Characteristics curve (ROC-AUC). These findings indicate that EEG data can potentially be used as a metric for identifying navigational uncertainty states, which may provide greater rigor and efficiency in studies of human responses to architectural design variables and wayfinding cues.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472753', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'ZEB2 transduces HIF1 dependent regulation of Transglutaminase 2 in glomerular podocytes', 'authors': \"[{'name': 'Lakshmi P Kolligundla'}, {'name': 'Ashish K Singh'}, {'name': 'Rajesh Kavvuri'}, {'name': 'Anil K Pasupulati'}]\", 'abstract': '<p>Glomerular podocytes are instrumental in ensuring glomerular permselectivity and regulating the integrity of glomerular biology. However, podocytes are vulnerable to various noxious stimuli such as hypoxia, and podocyte injury presented with glomerulosclerosis and impaired kidney function. The mechanism of hypoxia-induced podocyte injury vis-a-vis glomerulosclerosis has remained enigmatic. Hypoxia inducible factor 1 (HIF1) that transduces hypoxic adaptations, induces Transglutaminase 2 (TG2), a calcium dependent enzyme that catalyzes intramolecular -(-glutamyl) lysine cross-links of extracellular matrix (ECM) proteins. In this study, we investigated the mechanism of regulation of TG2 by HIF1. Stabilization of HIF1 by FG4592 (Roxadustat) and physiological hypoxia, resulted in elevated expression of ZEB2 (zinc-finger E-box-homeobox 2) and its downstream target TRPC6 (transient receptor potential channel 6). ZEB2 transcriptionally activates TG2 expression, whereas, via TRPC6, it induces calcium influx, inturn it increases the TG2 activity. Blocking the TRPC6 action or suppressing its expression only partially attenuated FG4592 induced TG2 activity, whereas suppression of ZEB2 expression significantly abolished TG2 activity. This study demonstrates that stabilization of HIF1 stimulates both TG2 expression and activity, whereas abrogation of HIF1 by metformin prevented HIF1 regulated TG2 and consequent glomerular injury.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472768', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': \"Metabolic alterations in a Drosophila model of Parkinson's disease based on DJ-1 deficiency\", 'authors': \"[{'name': 'Cristina Solana-Manrique'}, {'name': 'Francisco Jose Sanz'}, {'name': 'Isabel Torregrosa'}, {'name': 'Martina Palomino-Schatzlein'}, {'name': 'Carolina Hernandez-Oliver'}, {'name': 'Antonio Pineda'}, {'name': 'Nuria Paricio'}]\", 'abstract': \"<p>Parkinson's disease (PD) is the second most common neurodegenerative disorder whose physiopathology is still unclear. Besides, it is urgent to discover new biomarkers and therapeutic targets to facilitate its diagnosis and treatment. Previous studies performed in PD models and samples from PD patients already demonstrated that metabolic alterations were associated with this disease. In this context, the aim of this study is to give a better understanding of metabolic disturbances underlying PD pathogenesis. To achieve this goal, we used a <italic>Drosophila</italic> PD model based on inactivation of the <italic>DJ-1</italic> gene (ortholog of human <italic>DJ-1</italic>). Metabolomic analyses were performed in 1-day-old and 15-day-old <italic>DJ-1</italic> mutants and control flies using <sup>1</sup>H nuclear magnetic resonance spectroscopy, combined with expression and enzymatic activity assays of proteins implicated in altered pathways. Our results showed that PD model flies exhibit protein metabolism alterations, a shift from tricarboxylic acid cycle to glycolytic pathway to obtain ATP, together with an increase in the expression of some urea cycle enzymes. Thus, these metabolic changes could be contributing to PD pathogenesis and might constitute possible therapeutic targets and/or biomarkers for this disease.</p>\", 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472925', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'The p97 cofactor Ubxn7 facilitates replisome disassembly during S-phase', 'authors': \"[{'name': 'Zeynep Tarcan'}, {'name': 'Divyasree Poovathumkadavil'}, {'name': 'Aggeliki Skagia'}, {'name': 'Agnieszka Gambus'}]\", 'abstract': '<p>Complex cellular processes are driven by the regulated assembly and disassembly of large multi-protein complexes. In eukaryotic DNA replication, whilst we are beginning to understand the molecular mechanism for assembly of the replication machinery (replisome), we still know relatively little about the regulation of its disassembly at replication termination. Over recent years, the first elements of this process have emerged, revealing that the replicative helicase, at the heart of the replisome, is polyubiquitylated prior to unloading and that this unloading requires p97 segregase activity. Two different E3 ubiquitin ligases are now known to ubiquitylate the helicase under different conditions: Cul2Lrr1 and TRAIP. Here we have found two p97 cofactors, Ubxn7 and Faf1, which can interact with p97 during replisome disassembly in S-phase. Only Ubxn7 however facilitates efficient replisome disassembly through its interaction with both Cul2Lrr1 and p97. Our data therefore characterise Ubxn7 as the first substrate-specific p97 cofactor regulating replisome disassembly in vertebrates.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472944', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Understanding p300-transcription factor interactions using sequence variation and hybridization', 'authors': \"[{'name': 'Fruzsina Hobor'}, {'name': 'Zsofia Hegedus'}, {'name': 'Amaurys Avila Ibarra'}, {'name': 'Vencel L. Petrovicz'}, {'name': 'Gail J. Bartlett'}, {'name': 'Richard B Sessions'}, {'name': 'Andrew J. Wilson'}, {'name': 'Thomas A. Edwards'}]\", 'abstract': '<p>The hypoxic response is central to cell function and plays a significant role in the growth and survival of solid tumours. HIF-1 regulates the hypoxic response by activating over 100 genes responsible for adaptation to hypoxia, making it a potential target for anticancer drug discovery. Although there is significant structural and mechanistic understanding of the interaction between HIF-1 and p300 alongside negative regulators of HIF-1 such as CITED2, there remains a need to further understand the sequence determinants of binding. In this work we use a combination of protein expression, chemical synthesis, fluorescence anisotropy and isothermal titration calorimetry for HIF-1 sequence variants and a HIF-1- CITED hybrid sequence which we term CITIF. We show the HIF-1 sequence is highly tolerant to sequence variation through reduced enthalpic and less unfavourable entropic contributions, These data imply backbone as opposed to side chain interactions and ligand folding control the binding interaction and that sequence variations are tolerated as a result of adopting a more disordered bound interaction or fuzzy complex.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.21267851', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Endocrinology (including Diabetes Mellitus and Metabolic Disease)', 'covid': True, 'title': 'Home monitoring of HbA1c in diabetes mellitus: A protocol for systematic review and narrative synthesis on reliability, accuracy, and patient acceptability', 'authors': \"[{'name': 'Jack Colley'}, {'name': 'Hajira Dambha-Miller'}, {'name': 'Beth Stuart'}, {'name': 'Jazz Bartholomew'}, {'name': 'Hermione Price'}]\", 'abstract': '<sec><title>Introduction</title><p>Worldwide there are an estimated 463 million people with diabetes. [1] In the UK people with diabetes are offered an annual review including monitoring of Haemoglobin A1c (HbA1c). [2] [3] This can identify people with diabetes who are not meeting their glycaemic targets, enabling early intervention. Those who do not attend these reviews often have higher HbA1c levels and poorer health outcomes. [4] During the Coronavirus disease of 2019 (COVID-19) pandemic, there was a 77% reduction in monitoring of HbA1c in the UK. [5] We hypothesise that people with diabetes could take finger-prick samples at home for the measurement of HbA1c.</p></sec><sec><title>Method and Analysis</title><p>We will perform a systematic review of current evidence for capillary blood collected at home for the measurement of HbA1c. We will examine the validity, reliability, safety, and patient acceptability of the use of capillary blood compared with the usual standard of care of venous blood. We will explore variables which affect validity of results. Using core terms of Diabetes, HbA1c and Capillary sampling we will search MEDLINE, Embase, CINAHL, Web of Science Core Collection, Google Scholar, Open Grey and other grey literature from database inception until 2021. Risk of bias will be assessed using the COSMIN risk of bias tool to assess the quality of studies on reliability and measurement error. Database searches and data extraction for primary outcomes will be conducted in duplicate. We will produce a narrative synthesis exploring how variables of capillary blood collection impact on validity, as well as exploring the safety and acceptability of patient self-collection.</p></sec><sec><title>Ethics and Dissemination</title><p>This review will be submitted for publication in a peer-reviewed open-access journal. We will present our results at both national and international conferences. As a systematic review with no primary participant data or involvement, ethical approval is not applicable.</p></sec><sec><title>PROSPERO registration number</title><p>CRD42021225606</p></sec><sec><title>Strengths and limitations of this study</title><list list-type=\"bullet\"><list-item><p>To our knowledge this is the first systematic review to explore all postal methods of capillary blood collection for the measurement of HbA1c.</p></list-item><list-item><p>This review follows the Preferred Reporting Items for Systematic Reviews and Meta-Analyses guidelines, which offers transparency and enhances reproducibility.</p></list-item><list-item><p>Due to anticipated heterogeneity in statistical approaches, summary analyses, sample storage, transportation, extraction, and assay, meta-analysis is unlikely to be appropriate and therefore narrative synthesis will be used.</p></list-item><list-item><p>Due to the exclusion criteria, our findings may not be generalisable to a wider population including children, and those with haemoglobinopathies, high erythrocyte turnover or other conditions likely to affect HbA1c result.</p></list-item></list></sec>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472834', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Novel mechanism of plasmid-DNA transfer mediated by heterologous cell fusion in syntrophic coculture of Clostridium organisms', 'authors': \"[{'name': 'Kamil Charubin'}, {'name': 'Gwendolyn J Gregory'}, {'name': 'ELEFTHERIOS TERRY PAPOUTSAKIS'}]\", 'abstract': '<p>The evolution of bacteria is driven by random genetic mutations and horizontal gene transfer (HGT) of genetic material from other bacteria. HGT can occur via transformation, transduction, and conjugation. Here, we present a potential new mechanism of HGT which occurs in a syntrophic Clostridium coculture. We have previously shown that in syntrophic cocultures of Clostridium acetobutylicum and Clostridium ljungdahlii, the two organisms undergo heterologous cell fusion, which includes fusion of the peptidoglycan cell walls and membranes. Heterologous cell fusion facilitated a large-scale exchange of cytoplasmic protein and RNA between the two organisms, leading to the formation of hybrid bacterial cells containing cytoplasmic material of the two parent organisms. Here we present new evidence that cell fusion events also facilitate the exchange of plasmid DNA between the two organisms of the syntrophic coculture. Through the use of a selective subculturing process, we successfully isolated wild-type C. acetobutylicum clones which have acquired a portion of the plasmid DNA, containing the antibiotic resistance marker, from a recombinant strain of C. ljungdahlii. Fusion events led to formation of persistent aberrant hybrid cells with distinct morphogenetic characteristics. Furthermore, our data support the concept of a novel, interspecies, mechanism of acquiring antibiotic resistance. Since neither organism contains any known conjugation machinery or mechanism, these findings expand our understanding of multi-species microbiomes, their survival strategies, and evolution.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472887', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Nanoscale patterning of in vitro neuronal circuits', 'authors': \"[{'name': 'Jose C. Mateus'}, {'name': 'Sean Weaver'}, {'name': 'Dirk van Swaay'}, {'name': 'Aline Renz'}, {'name': 'Julian Hengsteler'}, {'name': 'Paulo Aguiar'}, {'name': 'Jnos Vrs'}]\", 'abstract': '<p>Methods for patterning neurons <italic>in vitro</italic> have gradually improved and are used to investigate questions difficult to address <italic>in</italic> or <italic>ex vivo</italic>. Though these techniques guide axons between groups of neurons, multiscale control of neuronal connectivity, from circuits to synapses, is yet to be achieved <italic>in vitro</italic>. As studying neuronal circuits with synaptic resolution <italic>in vivo</italic> poses significant challenges, an <italic>in vitro</italic> alternative could serve as a testbed for <italic>in vivo</italic> experiments or as a platform for validating biophysical and computational models. In this work we use a combination of electron beam and photolithography to create polydimethylsiloxane (PDMS) structures with features ranging from 150 nanometers to a few millimeters. Leveraging the difference between average axon and dendritic spine diameters, we restrict axon growth while allowing spines to pass through nanochannels to guide synapse formation between small groups of neurons (i.e.  nodes). We show this technique can be used to generate large numbers of isolated feed-forward circuits where connections between nodes are restricted to regions connected by nanochannels. Using a genetically encoded calcium indicator in combination with fluorescently tagged post synaptic protein, PSD-95, we demonstrate functional synapses can form in this region. Although more work needs to be done to control connectivity <italic>in vitro</italic>, we believe this is a significant step in that direction.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.15.472795', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Inferring the DNA replication origin density landscape in human cells from replication timing and fork directionality data', 'authors': \"[{'name': 'Jean-Michel Arbona'}, {'name': 'Hadi Kabalane'}, {'name': 'Arach Goldar'}, {'name': 'Olivier Hyrien'}, {'name': 'Benjamin Audit'}]\", 'abstract': '<p>The determinants of the locations and firing times of the multiple replication origins are still elusive in human and other metazoan organisms. Experiments can independently profile mean replication timing (MRT) and replication fork directionality (RFD) genome-wide. In the hypothesis of a constant replication fork speed, MRT and RFD are related to each other by an analytical formula so are a priori equivalent. However, we show here that experimental noises result in MRT and RFD profiles containing information at different spatial frequencies. We further demonstrate that one can compute an origin density landscape that, when inserted in an appropriate simulation framework, jointly predicts experimental MRT and RFD profiles with an unprecedented precision. We also extract an analytical formula linking intrinsic origin efficiency with observed origin efficiency and MRT. We then compare the computed origin density landscape with experimental distributions of potential origins (ORC, MCM) or actual initiation events (Bubble-seq, SNS-seq, OK-seq). The results indicate that MRT and RFD data are highly consistent with each other, that our simple model suffices to capture the replication dynamics during S phase given an appropriate initiation probability landscape, but that the density of potential origins is not the sole determinant of this landscape.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472863', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Mitotic spindle formation in the absence of Polo kinase', 'authors': \"[{'name': 'Juyoung Kim'}, {'name': 'Gohta Goshima'}]\", 'abstract': '<p>Mitosis is a fundamental process in every eukaryote, in which chromosomes are segregated into two daughter cells by the action of the microtubule (MT)-based spindle. Despite this common principle, genes essential for mitosis are variable among organisms. This indicates that the loss of essential genes or bypass-of-essentiality (BOE) occurred multiple times during evolution. While many BOE relationships have been recently revealed experimentally, the BOE of mitosis regulators (BOE-M) has been scarcely reported and how this occurs remains largely unknown. Here, by mutagenesis and subsequent evolutionary repair experiments, we isolated viable fission yeast strains that lacked the entire coding region of Polo-like kinase (Plk), a versatile essential mitotic kinase. The BOE of Plk was enabled by specific mutations in the downstream machinery, including the MT-nucleating gamma-tubulin complex, and more surprisingly, through downregulation of glucose uptake, which is not readily connected to mitosis. The latter bypass was dependent on casein kinase I (CK1), which has not been considered as a major mitotic regulator. Our genetic and phenotypic data suggest that CK1 constitutes an alternative mechanism of MT nucleation, which is normally dominated by Plk. A similar relationship was observed in a human colon cancer cell line. Thus, our study shows that BOE-M can be achieved by simple genetic or environmental changes, consistent with the occurrence of BOE-M during evolution. Furthermore, the identification of BOE-M constitutes a powerful means to uncover a hitherto under-studied mechanism driving mitosis and also hints at the limitations and solutions for selecting chemotherapeutic compounds targeting mitosis.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472789', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Soluble MAC is primarily released from MAC-resistant bacteria that potently convert complement component C5', 'authors': \"[{'name': 'Dennis J. Doorduijn'}, {'name': 'Marie V. Lukassen'}, {'name': 'Marije F.L. van t Wout'}, {'name': 'Vojtech Franc'}, {'name': 'Maartje Ruyken'}, {'name': 'Bart W. Bardoel'}, {'name': 'Albert J.R. Heck'}, {'name': 'Suzan H. M. Rooijakkers'}]\", 'abstract': '<p>The Membrane Attack Complex (MAC or C5b-9) is an important effector of the immune system to kill invading microbes. MAC is formed when complement enzymes on the bacterial surface convert complement component C5 into C5b. Although the MAC is a membrane-inserted complex, soluble forms of MAC (sMAC, or terminal complement complex (TCC)) are often detected in sera of patients suffering from infections. Consequently, sMAC has been proposed as a biomarker, but it remains unclear when and how it is formed during infections. Here, we studied mechanisms of MAC formation on bacteria and found that sMAC is primarily formed in human serum by bacteria resistant to MAC-dependent killing. Surprisingly, C5 was converted into C5b more potently by MAC-resistant compared to MAC-sensitive <italic>Escherichia coli</italic> strains. Both the increase in C5 conversion and sMAC generation were linked to the expression of lipopolysaccharide (LPS) O-Antigen in the bacterial outer membrane. In addition, we found that MAC precursors are released from the surface of MAC-resistant bacteria during MAC assembly. Release of MAC precursors from bacteria induced lysis of bystander human erythrocytes in the absence of other serum components. However, serum regulators vitronectin (Vn) and clusterin (Clu) can prevent this bystander lysis. Combining size exclusion chromatography with mass spectrometry profiling, we show that sMAC released from bacteria in serum is a heterogeneous mixture of complexes composed of C5b-8, up to 3 copies of C9 and multiple copies of Vn and Clu. Altogether, our data provide molecular insight into how sMAC is generated during bacterial infections. This fundamental knowledge could form the basis for exploring the use of sMAC as biomarker.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.15.472788', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Macrogenetics reveals multifaceted influences of environmental variation on vertebrate population genetic diversity across the Americas', 'authors': \"[{'name': 'Elizabeth R Lawrence'}, {'name': 'Dylan J Fraser'}]\", 'abstract': '<p>Relative to species diversity gradients, the broad scale distribution of population-specific genetic diversity (PGD) across taxa remains understudied. We used nuclear DNA data collected from 6285 vertebrate populations across the Americas to assess the role environmental variables play in structuring the spatial/latitudinal distribution of PGD, a key component of adaptive potential in the face of environmental change. Our results provide key evidence for taxa-specific responses and that temperature variability in addition to mean temperature may be a primary driver of PGD. Additionally, we found some positive influence of precipitation, productivity, and elevation on PGD; identified trends were dependent on the metric of PGD. In contrast to the classic negative relationship between species diversity and latitude, we report either a positive or taxa-dependent relationship between PGD and latitude, depending on the metric of PGD. The inconsistent latitudinal gradient in different metrics of PGD may be due to opposing processes diminishing patterns across latitudes that operate on different timescales, as well as the flattening of large-scale genetic gradients when assessing across species versus within species. Our study highlights the nuance required to assess broad patterns in genetic diversity, and the need for developing balanced conservation strategies that ensure population, species, and community persistence.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.15.472780', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'A simple and direct method to define clonal selection in somatic mosaicism', 'authors': \"[{'name': 'Verena Krber'}, {'name': 'Naser Ansari-Pour'}, {'name': 'Niels Asger Jakobsen'}, {'name': 'Rachel Moore'}, {'name': 'Nina Claudino'}, {'name': 'Marlen Metzner'}, {'name': 'Batchimeg Usukhbayar'}, {'name': 'Mirian Angulo Salazar'}, {'name': 'Simon Newman'}, {'name': 'Benjamin JL Kendrick'}, {'name': 'Adrian H Taylor'}, {'name': 'Rasheed Afinowi-Luitz'}, {'name': 'Roger Gundle'}, {'name': 'Bridget Watkins'}, {'name': 'Kim Wheway'}, {'name': 'Debra Beazley'}, {'name': 'Andrew J Carr'}, {'name': 'Paresh Vyas'}, {'name': 'Thomas Hfer'}]\", 'abstract': '<p>Dividing somatic stem cells acquire DNA changes marking different clones. With time, clones can become large, either stochastically through neutral drift, or increased fitness and consequent selection. We present a simple, direct, and general approach that distinguishes between these two processes in normal somatic tissue in individuals. The method relies on single time point whole genome sequencing to study somatic mosaicism as tissues age. Using this method, we show that in human clonal hemopoiesis (CH), clones with CH driver mutations, that comprise a median of 24% of hematopoiesis originate decades before they are detected. They expand, through selection by a median of 26% per year. Overall, there is a 3-fold increased rate of stem cell division and an 8.6-fold increase in active long-term stem cells.</p><sec><title>One sentence summary</title><p>We present an approach to distinguish genetic drift from selection, as subclones arise in somatic tissues, in individuals.</p></sec>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472985', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Classification and Regression Models for Genomic Selection of Skewed Phenotypes: A Case for Disease Resistance in Winter Wheat (Triticum aestivum L.)', 'authors': \"[{'name': 'Lance F Merrick'}, {'name': 'Dennis N Lozada'}, {'name': 'Xianming Chen'}, {'name': 'Arron H Carter'}]\", 'abstract': '<p>Most genomic prediction models are linear regression models that assume continuous and normally distributed phenotypes, but responses to diseases such as stripe rust (caused by Puccinia striiformis f. sp. tritici) are commonly recorded in ordinal scales and percentages. Disease severity (SEV) and infection type (IT) data in germplasm screening nurseries generally do not follow these assumptions. On this regard, researchers may ignore the lack of normality, transform the phenotypes, use generalized linear models, or use supervised learning algorithms and classification models with no restriction on the distribution of response variables, which are less sensitive when modeling ordinal scores. The goal of this research was to compare classification and regression genomic selection models for skewed phenotypes using stripe rust SEV and IT in winter wheat. We extensively compared both regression and classification prediction models using two training populations composed of breeding lines phenotyped in four years (2016-2018, and 2020) and a diversity panel phenotyped in four years (2013-2016). The prediction models used 19,861 genotyping-by-sequencing single-nucleotide polymorphism markers. Overall, square root transformed phenotypes using rrBLUP and support vector machine regression models displayed the highest combination of accuracy and relative efficiency across the regression and classification models. Further, a classification system based on support vector machine and ordinal Bayesian models with a 2-Class scale for SEV reached the highest class accuracy of 0.99. This study showed that breeders can use linear and non-parametric regression models within their own breeding lines over combined years to accurately predict skewed phenotypes.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.472594', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Neural Readout of a Latency Code in the Active Electrosensory System', 'authors': \"[{'name': 'Krista Perks'}, {'name': 'Nathaniel B Sawtell'}]\", 'abstract': '<p>The latency of spikes relative to a stimulus conveys sensory information across modalities. However, in most cases it remains unclear whether and how such latency codes are utilized by postsynaptic neurons. In the active electrosensory system of mormyrid fish, a latency code for stimulus amplitude in electroreceptor afferent nerve fibers (EAs) is hypothesized to be read out by a central reference provided by motor corollary discharge (CD). Here we demonstrate that CD enhances sensory responses in postsynaptic granular cells of the electrosensory lobe, but is not required for reading out EA input. Instead, diverse latency and spike count tuning across the EA population gives rise to graded information about stimulus amplitude that can be read out by standard integration of converging excitatory synaptic inputs. Inhibitory control over the temporal window of integration renders two granular cell subclasses differentially sensitive to information derived from relative spike latency versus spike count.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472991', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Tryptophol acetate and tyrosol acetate, metabolites secreted by a probiotic yeast, halt cytokine storm', 'authors': \"[{'name': 'orit malka'}, {'name': 'ravit malishev'}, {'name': 'marina bersudsky'}, {'name': 'manikand rajendran'}, {'name': 'mathumathi krishnamohan'}, {'name': 'jakeer shaik'}, {'name': 'evgeni tikhonov'}, {'name': 'eliya sultan'}, {'name': 'omry koren'}, {'name': 'ron apte'}, {'name': 'benyamin rosental'}, {'name': 'elena voronov'}, {'name': 'raz jelinek'}]\", 'abstract': \"<p>Probiotic fermented foods are perceived as contributing to human health and capable of protecting against inflammation, however solid mechanistic evidence for the presumptive therapeutic benefits is lacking. Here we report that tryptophol acetate and tyrosol acetate, small molecule metabolites secreted by the probiotic milk-fermented yeast Kluyveromyces marxianus exhibit remarkable anti-inflammatory properties. Comprehensive in vivo, ex vivo and in vitro experiments, employing LPS-induced 'cytokine storm' models, reveal dramatic effects of the two molecules, added in tandem, on mice morbidity, laboratory parameters and mortality. In parallel, significant attenuation of pro-inflammatory cytokines including IL-6, IL-1, IL-1 and TNF-B, and reduction of reactive oxygen species were recorded. Importantly, tryptophol acetate and tyrosol acetate did not completely suppress cytokine generation, but rather brought their concentrations back to baseline levels, further maintaining core immune functions, including phagocytosis. The anti-inflammatory effects of tryptophol acetate and tyrosol acetate were mediated through downregulation of TLR4, IL-1R, and TNFR signaling pathways and increased A20 expression, attenuating NF-kB level. In addition, the two molecules had a significant impact on mice microbiome, increasing the abundance of the genus Bactericides, known to exhibit anti-inflammatory properties. Overall, this work illuminates pronounced and broad-based immune modulation properties of probiotic yeast-secreted metabolites, uncovering their mechanism of action and underscoring potential new therapeutic avenues for severe inflammation.</p>\", 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.472798', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Long term exposure of human gut microbiota with high and low emulsifier sensitivity to soy lecithin in M-SHIME model.', 'authors': \"[{'name': 'Lisa Miclotte'}, {'name': 'Ellen De Paepe'}, {'name': 'Qiqiong Li'}, {'name': 'Andreja Rajkovic'}, {'name': 'John Van Camp'}, {'name': 'Tom Van de Wiele'}]\", 'abstract': '<p>In the context of the potential health hazards related to food processing, dietary emulsifiers have been shown to alter the structure and function of the gut microbial community, both in vivo and in vitro. In mouse models, these emulsifier exposed gut microbiota were shown to contribute to gut inflammation. Several knowledge gaps remain to be addressed though. As such, the impact from a longer timeframe of exposure on the gut microbiota is not known and interindividual variability in microbiome response needs to be measured. To answer these research questions, in this study the faecal microbiota from two individuals, previously selected for high and low emulsifier sensitivity, were exposed to two concentrations of soy lecithin during a 7 day treatment phase in the dynamic mucosal simulator of the human intestinal microbial ecosystem (M-SHIME). The results showed mild effects from soy lecithin on the composition and functionality of these microbial communities, which depended on the original microbial composition. The effects also mostly levelled off after 3 days of exposure. The emulsifier sensitivity for which the microbiota were selected, was preserved. Some potentially concerning effects were also registered: butyrate levels, positively correlating with Faecalibacterium abundance, were lowered by soy lecithin. Also the abundance of the beneficial Bifidobacterium genus was lowered, while the abundance of the notorious unclassified Enterobacteriaceae was increased. Within the family of the unclassified Lachnospiraceae, several genera were either suppressed or stimulated. The effects that these microbial alterations would have on a living host is not yet certain, especially given the fact that large fractions of soy lecithins constituents can be absorbed. Nevertheless, choline and phosphatidylcholine, both primary and absorbable constituents of soy lecithin, have recently been linked to cardiovascular disease via the generation of TMA by the gut microbiota. Further studies that validate our findings and link them to potential health outcomes are thus justified.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472542', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Multi-virulence and phenotypic spread of Campylobacter jejuni carried by chicken meat in Brazil', 'authors': \"[{'name': 'Phelipe Augusto Borba Martins Peres'}, {'name': 'Roberta Torres de Melo'}, {'name': 'Paulo Marcel Armendaris'}, {'name': 'Fabiano Barreto'}, {'name': 'Tiago Follmann Perin'}, {'name': 'Ana Laura Grazziotin'}, {'name': 'Guilherme Paz Monteiro'}, {'name': 'Eliane Pereira Mendona'}, {'name': 'Eduarda Cristina Alves Lourenzatto'}, {'name': 'Arthur Slompo Muniz Bicalho'}, {'name': 'Marcelo de Vito Filho'}, {'name': 'Ana Beatriz Garcez Buiatte'}, {'name': 'Daise Aparecida Rossi'}]\", 'abstract': '<p><italic>Campylobacter jejuni</italic> is the most incriminated pathogen in bacterial gastroenteritis, and therefore, characteristics of its epidemiology must be continuously investigated to support possible mitigating measures. This is particularly important when evaluating representative strains of the worlds leading chicken meat exporter, Brazil. We evaluated a panel of 14 virulence genes in 359 strains of <italic>C. jejuni</italic> isolated from chilled broiler carcasses of Brazil. The genes were classified into five virulence categories (B: biofilm/motility; SS: secretion/cytotoxicity system; CI: invasion/colonization; GB: Guillain-Barr and AE: adaptation to stress). The percentage of strains with stress adaptation genes (86.07%) indicates the potential to adapt to unfavorable environmental conditions and <italic>hcp</italic> gene in 97.77%, indicates the ability to cause serious infections in humans. Genes related to GBS in 77.44% of strains are an additional concern, which must be monitored. The gene panel showed the presence of 124 virulence profiles. Individual analyzes by carcass, slaughter establishment, and municipalities where they were located showed high I.Var., of 0.82, 0.87 and 0.78, respectively. Georeferencing indicated state A as a hotspot for virulent strains. Higher levels of isolation and multi-virulence were identified in the summer, which in Brazil is hot and humid. Proteomics was able to discriminate the strains, but due to the high heterogeneity between them, it did not allow to explain their dissemination. Together, our results showed that the studied strains are a potential danger to public health and that there is an urgent need for their surveillance and the adoption of control measures, especially in state A.</p><sec><title>AUTHOR SUMMARY</title><p><italic>Campylobacter jejuni</italic> is a bacterium considered one of the main causes of foodborne illnesses and the consumption of undercooked chicken meat is one of the main sources of human infection. In Brazil, epidemiological studies of this pathogen are still scarce, when compared to countries with structured surveillance, as well as, its analysis is not required by public health agencies in any group of foods intended for human consumption. Here we investigate the epidemiology of C. jejuni strains isolated from chilled chicken carcasses in Brazil, determining virulent and multivirulent strains, by the origin of the sample and its phenotypic patterns. The strains showed a high potential for adaptation to the environment, being classified as virulent and multivirulent, with a seasonal pattern in the hottest and humid periods of the year. In state A, the strains with the highest evolutionary level were isolated, when compared to the other states in the region. We hope that this study will help to better understand the potential risks that C. jejuni poses to the population and support surveillance agencies in tracking and adopting measures to minimize the dangers that this pathogen poses to public health.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.472826', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Neural signature of everyday function in older adults at-risk of cognitive impairment', 'authors': \"[{'name': 'Pierfilippo De Sanctis'}, {'name': 'Johanna Wagner'}, {'name': 'Sophie Molholm'}, {'name': 'John J. Foxe'}, {'name': 'Helena Blumen'}, {'name': 'Douwe Horsthuis'}]\", 'abstract': '<p>Assessment of everyday activities are central to the diagnosis of pre-dementia and dementia. Yet, little is known about the brain substrates and processes that contribute to everyday functional impairment, particularly during early stages of cognitive decline. We investigated everyday function using a complex gait task in normal older adults stratified by risk of cognitive impairment. We applied a novel EEG approach, which combines electroencephalographic with 3D-body tracking technology to measure brain-gait dynamics with millisecond precision while participants are in motion. Twenty-six participants (mean age = 74.9 years) with cognitive and everyday functional profiles within the normal range for their age and sex were ranked for risk of cognitive impairment. We used the Montreal Cognitive Assessment battery, a global index of cognition with a range from 0 to 30, to classify individuals as being at higher (22-26) and lower risk (27+). Individuals walking on a treadmill were exposed to visual perturbation designed to destabilize gait. Assuming that brain changes precede behavioral decline, we predicted that older adults increase step width to gain stability, yet the underlying neural signatures would be different for lower versus higher risk individuals. When pooling across risk groups, we found that step width increased and fronto-parietal activation shifted from transient, during swing phases, to sustained across the gait cycle during visually perturbed input. As predicted, step width increased in both groups but underlying neural signatures were different. Fronto-medial theta (3-7Hz) power of gait-related brain oscillations were increased in higher risk individuals during both perturbed and unperturbed inputs. On the other hand, left central gyri beta (13-28Hz) power was decreased in lower risk individuals, specifically during visually perturbed input. Finally, relating MoCA scores to spectral power pooled across fronto-parietal regions, we found associations between increased theta power and worse MoCA scores and between decreased beta power and better MoCA scores. Able-bodied older adults at-risk of cognitive impairment are characterized by unique neural signatures of mobility. Stronger reliance on frontomedial theta activation in at-risk individuals may reflect higher-order compensatory responses for deterioration of basic sensorimotor processes. Region and spectral-specific signatures of mobility may provide brain targets for early intervention against everyday functional decline.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.14.471281', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Link between aging and atheroprotection in Mif-deficient atherosclerotic mice', 'authors': \"[{'name': 'Christine Krammer'}, {'name': 'Bishan Yang'}, {'name': 'Sabrina Reichl'}, {'name': 'Verena Bolini'}, {'name': 'Corinna Schulte'}, {'name': 'Heidi Noels'}, {'name': 'Omar El Bounkari'}, {'name': 'Aphrodite Kapurniotu'}, {'name': 'Christian Weber'}, {'name': 'Sarajo Mohanta'}, {'name': 'Jrgen Bernhagen'}]\", 'abstract': '<p>Atherosclerosis is a lipid-triggered chronic inflammatory condition of our arteries and the main underlying pathology of myocardial infarction and stroke. Pathogenesis is age-dependent, but the mechanistic links between disease progression, age, and atherogenic cytokines and chemokines are incompletely understood. Here, we studied the chemokine-like inflammatory cytokine macrophage migration inhibitory factor (MIF) in atherogenic <italic>Apoe</italic><sup><italic>/</italic></sup> mice across different stages of aging and cholesterol-rich high-fat diet (HFD). MIF promotes atherosclerosis by mediating atherogenic monocyte and T-cell recruitment, amplifying lesional inflammation, and suppressing atheroprotective B-cell responses. However, age-related links between atherogenesis and MIF and its role in advanced atherosclerosis in aged mice have not been systematically explored. We compared effects of global <italic>Mif</italic>-gene deficiency in 30-, 42-, and 48-week-old <italic>Apoe</italic><sup><italic>/</italic></sup> mice on HFD for 24, 36, or 42 weeks, respectively, and in 52-week-old mice on a 6-week HFD. While a regio-specific atheroprotective phenotype of <italic>Mif-</italic>deficiency was observed in the 30/24-week-old group, atheroprotection was not detected in the 48/42- and 52/6-week-old groups, suggesting that atheroprotection afforded by global <italic>Mif</italic>-gene deletion differs across aging stages and atherogenic diet duration. We identify a combination of mechanisms that could explain this phenotype: i) <italic>Mif</italic>-deficiency promotes lesional Trem2<sup>+</sup> macrophage numbers in younger but not aged mice; ii) <italic>Mif</italic>-deficiency favors formation of lymphocyte-rich stage-I/II ATLOs in younger mice but ATLO numbers equalize with those in <italic>Apoe</italic><sup><italic>/</italic></sup> controls in the older mice; and iii) plasma anti-oxLDL-IgM antibody levels are decreased in aged <italic>Mif</italic>-deficient mice. Of note, these three markers (Trem2<sup>+</sup> macrophages, ATLOs, anti-oxLDL-IgM antibodies) have been previously linked to atheroprotection. Together, our study thus suggests that regio-specific atheroprotection due to global <italic>Mif</italic>-gene deficiency in atherogenic <italic>Apoe</italic><sup><italic>/</italic></sup> mice is lost upon advanced aging and identifies mechanisms that could explain this phenotype shift. These observations may have implications for translational MIF- directed strategies.</p>', 'posted': '2021-12-15'}, {'doi': '10.1101/2021.12.16.472883', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Specific vulnerability of long telomeres to undergo end fusions revealed by mutational analysis of Rap1', 'authors': \"[{'name': 'Peter Baumann'}, {'name': 'Lili Pan'}, {'name': 'Duncan Tormey'}, {'name': 'Nadine Bobon'}]\", 'abstract': '<p>The conserved Rap1 protein is part of the shelterin complex that plays critical roles in chromosome end protection and telomere length homeostasis. Previous studies addressed how fission yeast Rap1 contributes to telomere length maintenance, but the mechanism by which the protein inhibits end fusions has remained elusive. Here, we use a genetic screen in combination with high throughput sequencing to identify several amino acid positions in Rap1 that have a key role in end protection.  Interestingly, mutations at these sites render cells susceptible to genome instability in a conditional manner with longer telomeres being prone to undergoing end fusions, while short telomeres are sufficiently protected.  The protection of long telomeres requires their nuclear envelope attachment mediated by the Rap1-Bqt4 interaction. Our data demonstrates that longer telomeres pose an additional challenge for the maintenance of genome integrity and provides an explanation for a species-specific upper limit in telomere length.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.472014', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Dynamic structural cell responses in the thymus to acute injury, regeneration, and age', 'authors': '[{\\'name\\': \\'Lorenz L Jahn\\'}, {\\'name\\': \\'Anastasia I Kousa\\'}, {\\'name\\': \\'Lisa Sikkema\\'}, {\\'name\\': \\'Angel E Flores\\'}, {\\'name\\': \\'Kimon V Argyropoulos\\'}, {\\'name\\': \\'Jennifer Tsai\\'}, {\\'name\\': \\'Amina Lazrak\\'}, {\\'name\\': \\'Katherine Nichols\\'}, {\\'name\\': \\'Nicole Lee\\'}, {\\'name\\': \\'Florent Malard\\'}, {\\'name\\': \\'Hana Andrlova\\'}, {\\'name\\': \\'Antonio LC Gomes\\'}, {\\'name\\': \\'Enrico Velardi\\'}, {\\'name\\': \\'Salma Youssef\\'}, {\\'name\\': \\'Marina B da Silva\\'}, {\\'name\\': \\'Melissa Docampo\\'}, {\\'name\\': \\'Roshan Sharma\\'}, {\\'name\\': \\'Linus Mazoutis\\'}, {\\'name\\': \\'Manu Setty\\'}, {\\'name\\': \"Dana Pe\\'er\"}, {\\'name\\': \\'Nancy R Manley\\'}, {\\'name\\': \\'Jarrod A Dudakov\\'}, {\\'name\\': \\'Marcel RM van den Brink\\'}]', 'abstract': '<p>The thymus, the primary site of T cell development, is extremely sensitive to insult but also harbors tremendous capacity for repair. Using single cell sequencing of thymic structural cells, as well as functional and structural analyses, we revealed distinct regenerative programs by endothelial and mesenchymal subsets after injury that stimulated epithelial repair; the compartment primarily supporting T cell development. Thymic function not only declined over lifespan, contributing to immune aging, but the capacity of the thymus to regenerate after damage also declined in old mice. This could be attributed to an inability of the old microenvironment to induce reparative programs; leading to reduced ability to restore tissue structure and function. These findings provide a detailed framework for the response of structural cells to aging and acute damage, which could have considerable implications for our understanding of aging immunity and recovery from treatments such as chemotherapy and bone marrow transplant.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.472676', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Discrete LAT condensates encode antigen information from single pMHC:TCR binding events', 'authors': '[{\\'name\\': \\'Darren B. McAffee\\'}, {\\'name\\': \"Mark K. O\\'Dair\"}, {\\'name\\': \\'Jenny J. Lin\\'}, {\\'name\\': \\'Shalini T. Low-Nam\\'}, {\\'name\\': \\'Kiera B. Wilhelm\\'}, {\\'name\\': \\'Sungi Kim\\'}, {\\'name\\': \\'Shumpei Morita\\'}, {\\'name\\': \\'Jay T. Groves\\'}]', 'abstract': '<p>LAT assembly into a two-dimensional protein condensate is a prominent feature of antigen discrimination by T cells.  Here, we use single-molecule imaging techniques to resolve the spatial position and temporal duration of each pMHC:TCR molecular binding event while simultaneously monitoring LAT condensation at the membrane. An individual binding event is sufficient to trigger a LAT condensate, which is self-limiting, and neither its size nor lifetime is correlated with the duration of the originating pMHC:TCR binding event.  Only the probability of the LAT condensate forming is related to the pMHC:TCR binding dwell time.  LAT condenses abruptly, but after an extended delay from the originating binding event.  A LAT mutation that facilitates phosphorylation at the PLC-1 recruitment site shortens the delay time to LAT condensation and alters T cell antigen specificity.  These results identify a role for the LAT protein condensation phase transition in setting antigen discrimination thresholds in T cells.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.14.472679', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Cdc42 GTPase activating proteins Rga4 and Rga6 coordinate septum synthesis and membrane trafficking at the division plane during cytokinesis', 'authors': \"[{'name': 'Bethany F. Campbell'}, {'name': 'Brian S. Hercyk'}, {'name': 'Ashlei R. Williams'}, {'name': 'Ema San Miguel'}, {'name': 'Haylee G. Young'}, {'name': 'Maitreyi E. Das'}]\", 'abstract': '<p>Fission yeast cytokinesis is driven by simultaneous septum synthesis, membrane furrowing and actomyosin ring constriction. The septum consists of a primary septum flanked by secondary septa. First, delivery of the glucan synthase Bgs1 and membrane vesicles initiate primary septum synthesis and furrowing. Next, Bgs4 is delivered for secondary septum formation. It is unclear how septum synthesis is coordinated with membrane furrowing. Cdc42 promotes delivery of Bgs1 but not Bgs4. We find that after primary septum initiation, Cdc42 inactivators Rga4 and Rga6 localize to the division site. In <italic>rga4rga6</italic> mutants Cdc42 activity is enhanced during late cytokinesis and cells take longer to separate. Electron micrographs of the division site in these mutants exhibit malformed septum with irregular membrane structures. These mutants have a larger division plane with enhanced Bgs1 delivery but fail to enhance accumulation of Bgs4 and several exocytic proteins. Additionally, these mutants show endocytic defects at the division site. This suggests that Cdc42 regulates only specific membrane trafficking events. Our data indicate that while active Cdc42 promotes primary septum synthesis, as cytokinesis progresses Rga4 and Rga6 localize to the division site to decrease Cdc42 activity. This couples specific membrane trafficking events with septum formation to allow proper septum morphology.</p><sec><title>Abstract Figure</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"472679v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec><sec><title>Synopsis</title><p>The GTPase Cdc42 regulates cytokinesis in cell-walled fission yeast. Active Cdc42 promotes the initiation of septum (new cell wall) synthesis to physically divide daughter cells. Here we show that Cdc42 activity must be decreased at the later stages of cytokinesis to enable proper septum formation. Mutants lacking Cdc42 inactivators, Rga4 and Rga6, lead to membrane trafficking defects and malformed septa consequently delaying cell separation.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.472979', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Chemoproteomics of microbiota metabolites reveals small-molecule agonists for orphan receptor GPRC5A', 'authors': \"[{'name': 'Xiaohui Zhao'}, {'name': 'Kathryn R. Stein'}, {'name': 'Victor Chen'}, {'name': 'Matthew E. Griffin'}, {'name': 'Howard C. Hang'}]\", 'abstract': '<p>The microbiota generates diverse metabolites that can engage multiple pathways to modulate host physiology and disease, but their protein targets and mechanism(s) of action have not been fully elucidated. To address this challenge, we focused on indole-3-acetic acid (IAA), a prominent microbiota metabolite, and developed IAA-based chemical reporters for proteomic studies. We discovered that IAA interacts with many proteins in host cells, including small-molecule transporters, receptors and metabolic enzymes. Notably, our functional studies revealed that IAA binds to orphan G protein-coupled receptors such as GPRC5A, but only aromatic monoamines were capable of inducing GPRC5A signaling. Functional profiling of microbiota uncovered specific bacterial species and enzymes that generate GPRC5A agonists. Finally, biochemical characterization of GPRC5A activation identified more potent synthetic agonists as well as key amino acid residues involved in ligand binding. These studies highlight the utility of chemoproteomics to dissect protein targets and mechanisms of action for microbiota metabolites.</p>', 'posted': '2021-12-16'}, {'doi': '10.1101/2021.12.16.473009', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Towards a global perspective for Salvia L: Phylogeny, diversification, and floral evolution', 'authors': \"[{'name': 'Fatemeh Moein'}, {'name': 'Ziba Jamzad'}, {'name': 'Mohammadreza Rahiminejad'}, {'name': 'Jacob Brian Landis'}, {'name': 'Mansour Mirtadzadini'}, {'name': 'Douglas Edward Soltis'}, {'name': 'Pamela Sue Soltis'}]\", 'abstract': '<p>Premise of this study: Salvia is the most species-rich genus in Lamiaceae, encompassing approximately 1000 species distributed all over the world. We sought a new evolutionary perspective for Salvia by employing macroevolutionary analyses to address the tempo and mode of diversification. To study the association of floral traits with speciation and extinction, we modeled and explored the evolution of corolla length and the lever-mechanism pollination system across our Salvia phylogeny. \\nMethods: We reconstructed a multigene phylogeny for 366 species of Salvia in the broad sense including all major recognized lineages and numerous species from Iran, a region previously overlooked in studies of the genus. Our phylogenetic data in combination with divergence time estimates were used to examine the evolution of corolla length, woody vs. herbaceous habit, and presence vs. absence of a lever mechanism. We investigated the timing and dependence of Salvia diversification related to corolla length evolution through a disparity test and BAMM analysis. A HiSSE model was used to evaluate the dependency of diversification on the lever-mechanism pollination system in Salvia.\\nKey Results: Based on recent investigations and classifications, Salvia is monophyletic and comprises ~1000 species. Our inclusion, for the first time, of a comprehensive sampling for Iranian species of Salvia provides higher phylogenetic resolution for southwestern Asian species than obtained in previous studies. A medium corolla length (15-18mm) was reconstructed as the ancestral state for Salvia with multiple shifts to shorter and longer corollas. Macroevolutionary model analyses indicate that corolla length disparity is high throughout Salvia evolution, significantly different from expectations under a Brownian motion model during the last 28 million years of evolution. Our analyses show evidence of a higher diversification rate of corolla length for some Andean species of Salvia compared to other members of the genus. Based on our tests of diversification models, we reject the hypothesis of a direct effect of the lever mechanism on Salvia diversification.\\nConclusions: Using a broader species sampling than previous studies, we obtained a well-resolved phylogeny for southwest Asian species of Salvia. Corolla length is an adaptive trait throughout the Salvia phylogeny with a higher rate of diversification in the South American clade. Our results suggest caution in considering the lever-mechanism pollination system as one of the main drivers of speciation in Salvia.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.16.473085', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Amino acid substitutions in human growth hormone affect coiled-coil content and receptor binding', 'authors': \"[{'name': 'Andrei Rajkovic'}, {'name': 'Sandesh Kanchugal'}, {'name': 'Eldar Abdurakhmanov'}, {'name': 'Rebecca Howard'}, {'name': 'Astrid Grslund'}, {'name': 'Sebastian Wrmlnder'}, {'name': 'Joseph Erwin'}, {'name': 'Helena Ursula Danielson'}, {'name': 'Samuel Coulbourn Flores'}]\", 'abstract': '<p>The interaction between human Growth Hormone (hGH) and hGH Receptor (hGHR) has great relevance to human diseases such as acromegaly and cancer. HGH has been extensively engineered by other workers to improve binding and other properties. We used a computational screen to select substitutions at single hGH positions within the hGHR-binding site. We find that, while many successfully slow down dissociation of the hGH-hGHR complex once bound, they also slow down the association of hGH to hGHR. We are particularly interested in E174 which belongs to the hGH zinc-binding triad, and which spans coiled-coil helices and obeys the coiled-coil heptad pattern. Surprisingly, substituting E174 with A leads to substantial increase in an experimental measure of coiled-coil content. E174A is known to increase affinity of hGH against hGHR; here we show that this is simply because the off-rate is slowed down more than the on-rate, in line with what has been found for other affinity-improving mutations. For E174Y (and mutations at other sites) the slowdown in on-rate was greater, leading to decreased affinity. The results point to a link between coiled-coiling, zinc binding, and hGHR-binding affinity in hGH, and also suggest rules for choosing affinity-increasing substitutions.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.17.473117', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'LONG-TERM MONITORING OF THE EUROPEAN ROLLER (CORACIAS GARRULUS) IN UKRAINE: IS CLIMATE BEHIND THE CHANGES?', 'authors': \"[{'name': 'Tatiana Shupova'}, {'name': 'Volodymyr Tytar'}]\", 'abstract': '<p>Since the 1980s there has been a long-term decline in numbers and contraction of range in Europe, including Ukraine. Our specific goals were to reconstruct the climatically suitable range of the species in Ukraine before the 1980s,  gain better knowledge on its requirements,  compare the past and current suitable areas, infer the regional and environmental variables that best explain its occurrence, and quantify the overall range change in the country. For these purposes we created a database consisting of 347 records of the roller made ever in Ukraine. We employed a species distribution modeling (SDM) approach to hindcast changes in the suitable range of the roller during historical times across Ukraine and to derive spatially explicit predictions of climatic suitability for the species under current climate. SDMs were created for three time intervals (before 1980, 1985-2009, 2010-2021) using corresponding climate data extracted from the TerraClim database. SDMs show a decline of suitable for rollers areas in the country from 85 to 46%. Several factors, including land cover and use, human population density and climate, that could have contributed to the decline of the species in Ukraine were considered. We suggest climate change and its speed (velocity) have been responsible for shaping the contemporary home range of the European roller. Key words: Coracias garrulus; species distribution modelling; ecological niche; climate change; velocity of climate change</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.21266784', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Nephrology', 'covid': False, 'title': 'Pre-Operative Kidney Biomarkers and Risks for Death, Cardiovascular and Chronic Kidney Disease Events: The TRIBE-AKI Study', 'authors': \"[{'name': 'George Vasquez-Rios'}, {'name': 'Dennis G. Moledina'}, {'name': 'Yaqi Jia'}, {'name': 'Eric McArthur'}, {'name': 'Sherry G. Mansour'}, {'name': 'Heather Thiessen-Philbrook'}, {'name': 'Michael G. Shlipak'}, {'name': 'Jay L. Koyner'}, {'name': 'Amit X. Garg'}, {'name': 'Chirag R. Parikh'}, {'name': 'Steven G. Coca'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>Soluble tumor necrosis factor receptor (sTNFR)1, sTNFR2, and plasma kidney injury molecule-1 (KIM-1) are associated with kidney events in patients with and without diabetes. However, their associations with clinical outcomes when obtained pre-operatively have not been explored.</p></sec><sec><title>Methods</title><p>The TRIBE-AKI cohort study is a prospective, multicenter, cohort study of high-risk adults undergoing cardiac surgery. We assessed the associations between pre-operative concentrations of plasma sTNFR1, sTNFR2, and KIM-1 and post-operative long-term outcomes including mortality, cardiovascular events, and chronic kidney disease (CKD) incidence or progression, ascertained after discharge.</p></sec><sec><title>Results</title><p>Among 1378 participants included in the analysis with a median follow-up period was 6.7 (IQR 4.0-7.9), 434 (31%) patients died, 256 (19%) experienced cardiovascular events and out of 837 with available long-term kidney function data, 30% developed CKD. After adjustment for clinical covariates, each log increase in biomarker concentration was independently associated with mortality with 95%CI adjusted hazard ratios (aHRs) of 3.0 (2.3-4.0), 2.3 (1.8-2.9) and 2.0 (CI 1.6-2.4) for sTNFR1, sTNFR2 and KIM-1, respectively. For cardiovascular events, the 95%CI aHRs were 2.1 (1.5  3.1), 1.9 (1.4  2.6) and 1.6 (1.2  2.1) for sTNFR1, sTNFR2 and KIM-1, respectively. For CKD events, the aHRs were 2.2 (1.5  3.1) for sTNFR1, 1.9 (1.3  2.7) for sTNFR2, and 1.7 (1.3  2.3) for KIM-1. Despite the associations, each of the biomarkers alone or in combination failed to result in robust discrimination on an absolute basis or compared to a clinical model.</p></sec><sec><title>Conclusion</title><p>sTNFR1, sTNFR2, and KIM-1 were independently associated with longitudinal outcomes after discharge from a cardiac surgery hospitalization including death, cardiovascular and CKD events when obtained pre-operatively in high-risk individuals. Pre-operative plasma biomarkers could serve to assist during the evaluation of patients in whom cardiac surgery is planned.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472347', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Molecular Epidemiology and Microbiological Characteristics of Cryptococcus gattii VGII isolates from China', 'authors': \"[{'name': 'Xuelei Zang'}, {'name': 'weixin ke'}, {'name': 'Lifeng Wang'}, {'name': 'Hua Wu'}, {'name': 'Yemei Huang'}, {'name': 'Hengyu Deng'}, {'name': 'Meng Zhou'}, {'name': 'Ningxin Wu'}, {'name': 'Dingxia Shen'}, {'name': 'Xinying Xue'}]\", 'abstract': '<p><italic>Cryptococcus gattii</italic> (<italic>C. gattii</italic>) is a fungal pathogen that once caused an outbreak of cryptococcosis on Vancouver Island, and had spread worldwide, while few data were available in China. In this study, seven clinical isolates of <italic>C. gattii</italic> VGII were collected from 19 hospitals, Multi-locus Sequence Typing (MLST) analysis and whole-genome sequencing (WGS) was performed, and combined with published data for phylogenetic analysis. In addition, in vitro antifungal susceptibility testing, phenotypic analysis, and in vivo virulence studies were performed, subsequently, histopathological analysis of lung tissue was performed. <italic>C.gattii</italic> VGII infected patients were mainly immunocompetent male, and most of them had symptoms of central nervous system (CNS) involvement. MLST results showed that isolates from china exhibited high genetic diversity, and sequence type (ST) 7 was the major ST among the isolates. Some clinical isolates showed a close phylogenetic relationship with strains from Australia and South America. All clinical isolates did not show resistance to antifungal drugs. In addition, there was no correlation between virulence factors (temperature, melanin production, and capsule size) and virulence while in vivo experiments showed significant differences in virulence among strains. Lung fungal burden and damage to lung tissue correlated with virulence, and degree of damage to lung tissue in mice may highlight differences in virulence. Our work seeks to provide useful data for molecular epidemiology, antifungal susceptibility, and virulence differences of <italic>C. gattii</italic> VGII in China.</p><sec><title>Author Summary</title><p><italic>Cryptococcus gattii</italic> is one of the life-threatening fungal pathogens that can infect immunocompetent individuals. <italic>C.gattii</italic> can be invaded through the respiratory tract and can be spread to the brain, causing meningoencephalitis and even death. In 1999, an outbreak of <italic>C. gattii</italic> VGII occurred on Vancouver Island in Canada and spread in the Pacific Northwest region, which attracted more attention. Compared with other nations, the data of <italic>C gattii</italic> VGII in China was limited. To better understand the molecular epidemiology and microbiological characteristics of <italic>C.gattii</italic> VGII isolates from China, Bioinformatics analysis and in vivo and in vitro experiments were performed for clinical strains. Multi-locus Sequence Typing and whole-genome sequencing reveal the genetic diversity of the isolates and genetic relationships with isolates from other countries. We also investigated the antifungal drug susceptibility and virulence differences of the isolates. The authors expect that our work can provide useful data for the prevention and control of <italic>C. gattii</italic> VGII in China and to draw attention to <italic>C. gattii</italic>.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.21267077', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Change in Finances, Peer Access, and Mental Health Among Trans and Non-binary People in Canada During COVID-19', 'authors': \"[{'name': 'Monica A. Ghabrial'}, {'name': 'Ayden I. Scheim'}, {'name': 'Caiden Chih'}, {'name': 'Heather Santos'}, {'name': 'Noah James Adams'}, {'name': 'Greta R. Bauer'}]\", 'abstract': '<sec><title>Purpose</title><p>COVID-related stressors associated with loss of income and community are compounded with gender minority stress among trans and non-binary people (TNB)  which may result in mental health burden. The present study examined the effect of COVID-related change in finances and TNB gathering access on anxiety and depression among TNB people.</p></sec><sec><title>Methods</title><p>Participants were 18 years and older (<italic>M</italic> age = 30) who completed both pre-pandemic baseline (Fall 2019) and pandemic follow-up (Fall 2020) surveys in the Trans PULSE Canada study. Multivariable regression analyses examined associations between change in (1) finances and (2) access to TNB peers and mental health during the pandemic.</p></sec><sec><title>Results</title><p>Of 780 participants, 50% reported that COVID had a negative effect on personal income and 58.3% reported loss in access to TNB peer or friend gatherings (in person or online). Depression and anxiety symptoms increased from pre-pandemic to follow-up, and most participants were above measurement cut-offs for clinical levels during the pandemic. Changes in finances and access to peer gatherings were associated with depression symptoms during the pandemic, but effects depended on level of pre-pandemic depression. For participants with high pre-pandemic depression, financial stability was not protective against increased depression at follow-up. Participants experiencing unprecedentedly high levels of depression during COVID may have pursued more TNB gatherings. Neither financial change nor access to TNB gatherings were associated with pandemic anxiety.</p></sec><sec><title>Conclusion</title><p>Findings suggest need for a multifaceted approach to mental health programmes and services to address structural barriers, including financial support and meaningful TNB community engagement.</p></sec>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472325', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Monosomy X in isogenic human iPSC-derived trophoblast model impacts expression modules preserved in human placenta', 'authors': \"[{'name': 'Darcy T. Ahern'}, {'name': 'Prakhar Bansal'}, {'name': 'Isaac Faustino'}, {'name': 'Yuvabharath Kondaveeti'}, {'name': 'Heather R. Glatt-Deeley'}, {'name': 'Erin C. Banda'}, {'name': 'Stefan F. Pinter'}]\", 'abstract': '<p>Mammalian sex chromosomes encode homologous X/Y gene pairs that were retained on the male Y and escape X chromosome inactivation (XCI) in females. Inferred to reflect X/Y-pair dosage sensitivity, monosomy X is a leading cause of miscarriage in humans with near full penetrance. This phenotype is shared with many other mammals but not the mouse, which offers sophisticated genetic tools to generate sex chromosomal aneuploidy but also tolerates its developmental impact. To address this critical gap, we generated X-monosomic human induced pluripotent stem cells (hiPSCs) alongside otherwise isogenic euploid controls from male and female mosaic samples. Phased genomic variants of these hiPSC panels enable systematic investigation of X/Y dosage-sensitive features using <italic>in vitro</italic> models of human development.</p><p>Here, we demonstrate the utility of these validated hiPSC lines to test how X/Y-linked gene dosage impacts a widely-used model for the human syncytiotrophoblast. While these isogenic panels trigger a <italic>GATA2/3</italic> and <italic>TFAP2A/C</italic> -driven trophoblast gene circuit irrespective of karyotype, differential expression implicates monosomy X in altered levels of placental genes, and in secretion of placental growth factor (PlGF) and human chorionic gonadotropin (hCG). Remarkably, weighted gene co-expression network modules that significantly reflect these changes are also preserved in first-trimester chorionic villi and term placenta. Our results suggest monosomy X may skew trophoblast cell type composition, and that the pseudoautosomal region likely plays a key role in these changes, which may facilitate prioritization of haploinsufficient drivers of 45,X extra-embryonic phenotypes.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.21267727', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': True, 'title': 'Adult life-course trajectories of psychological distress and economic outcomes in midlife during the COVID-19 pandemic', 'authors': \"[{'name': 'V Moulton'}, {'name': 'A Sullivan'}, {'name': 'A Goodman'}, {'name': 'S Parsons'}, {'name': 'G Ploubidis'}]\", 'abstract': '<p>This study used two British birth cohorts to examine whether pre-pandemic trajectories of psychological distress were associated with a greater risk of changes in financial and employment situation during the pandemic, as well as increased need for government support and use of other methods to mitigate their economic situation. We identified 5 differential life-course trajectories of psychological distress from adolescence to midlife and explored their relation to changes in financial and employment circumstances at different stages during the pandemic from May 2020 to March 2021, applying multinomial logistic regression and controlling for numerous early life covariates. In addition, we ran modified Poisson models with robust standard errors to identify whether different trajectories were more likely to have been supported by the benefit system, payment holidays, borrowing and other methods of mitigating the economic shock. We found that despite the UK governments economic response package economic inequalities for pre-pandemic psychological distress trajectories with differential onset, severity and chronicity across the life-course were exacerbated by the COVID-19 economic shock. Furthermore, the subsequent cut in government support, alongside increases in the cost of living may widen economic inequalities for differential pre-pandemic psychological distress trajectories, which in turn may also worsen mental health. This work highlights, different pre-pandemic trajectories of psychological distress were more vulnerable to economic shock.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.473074', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Caveolae mediated endocytosis of VLDL particles in macrophages requires NPC1 and STARD3 for further lysosomal processing', 'authors': \"[{'name': 'Lei Deng'}, {'name': 'Frank Vrieling'}, {'name': 'Rinke Stienstra'}, {'name': 'Guido Hooiveld'}, {'name': 'Anouk L. Feitsma'}, {'name': 'Sander Kersten'}]\", 'abstract': '<p>Macrophages accumulate triglycerides under certain pathological conditions such as atherosclerosis. Triglycerides are carried in the bloodstream as part of very low-density lipoproteins (VLDL) and chylomicrons. How macrophages take up and process VLDL-lipids is not very well known. Here, using VLDL-sized triglyceride-rich emulsion particles, we aimed to study the mechanism by which VLDL-triglycerides are taken up, processed, and stored in macrophages. Our results show that macrophage uptake of emulsion particles mimicking VLDL (VLDLm) is dependent on lipoproteins lipase (LPL) and requires the lipoprotein-binding C-terminal domain of LPL but not the catalytic N-terminal domain. Subsequent internalization of VLDLm-triglycerides by macrophages is carried out by caveolae-mediated endocytosis, followed by triglyceride hydrolysis catalyzed by lysosomal acid lipase. Transfer of lysosomal fatty acids to the ER for subsequent storage as triglycerides is mediated by Stard3, whereas NPC1 was found to promote the extracellular efflux of fatty acids from lysosomes. Our data provide novel insights into how macrophages process VLDL-derived triglycerides and suggest that macrophages have the remarkable capacity to excrete part of the internalized triglycerides as fatty acids.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.13.472378', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Water droplet-in-oil digestion method for single-cell proteomics', 'authors': \"[{'name': 'Takeshi Masuda'}, {'name': 'Yuma Inamori'}, {'name': 'Arisu Furukawa'}, {'name': 'Kazuki Momosaki'}, {'name': 'Chih-Hsiang Chang'}, {'name': 'Daiki Kobayashi'}, {'name': 'Hiroto Ohguchi'}, {'name': 'Yawara Kawano'}, {'name': 'Shingo Ito'}, {'name': 'Norie Araki'}, {'name': 'Shao-En Ong'}, {'name': 'Sumio Ohtsuki'}]\", 'abstract': '<p>Recent advances in single-cell proteomics highlight the promise of sensitive analyses in limited cell populations. However, technical challenges remain for sample recovery, throughput, and versatility. Here, we first report a water droplet-in-oil digestion (WinO) method based on carboxyl-coated beads and phase transfer surfactants for proteomic analysis using limited sample amounts. This method was developed to minimize the contact area between the sample solution and the container to reduce the loss of proteins and peptides by adsorption. This method increased protein and peptide recovery 10-fold as well as the number of quantified transmembrane proteins compared to an in-solution digestion (ISD) method. The proteome profiles obtained from 100 cells using the WinO method highly correlated with those from 10000 cells using the ISD method. We successfully applied the WinO method to single-cell proteomics and quantified 462 proteins. Using the WinO method, samples can be easily prepared in a multi-well plate, making it a widely applicable and suitable method for single-cell proteomics.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.13.472381', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A rare human centenarian variant of SIRT6 enhances genome stability and interaction with Lamin A', 'authors': \"[{'name': 'Matthew Simon'}, {'name': 'Jiping Yang'}, {'name': 'Jonathan Gigas'}, {'name': 'Eric J. Earley'}, {'name': 'Tory M. Schaff'}, {'name': 'Lei Zhang'}, {'name': 'Maria Zagorulya'}, {'name': 'Greg Tombline'}, {'name': 'Michael Gilbert'}, {'name': 'Samantha L. Yuen'}, {'name': 'Alexis Pope'}, {'name': 'Michael Van Meter'}, {'name': 'Stephan Emmrich'}, {'name': 'Jeehae Han'}, {'name': 'Seungjin Ryu'}, {'name': 'Archana Tare'}, {'name': 'Yizhou Zhu'}, {'name': 'Adam Hudgins'}, {'name': 'Gil Atzmon'}, {'name': 'Nir Barzilai'}, {'name': 'Aaron Wolfe'}, {'name': 'Kelsey Moody'}, {'name': 'Benjamin A. Garcia'}, {'name': 'David D. Thomas'}, {'name': 'Paul D. Robbins'}, {'name': 'Jan Vijg'}, {'name': 'Andrei Seluanov'}, {'name': 'Yousin Suh'}, {'name': 'Vera Gorbunova'}]\", 'abstract': '<p>Sirtuin 6 (SIRT6) is a deacylase and mono-ADP ribosyl transferase (mADPr) enzyme involved in multiple cellular pathways implicated in the regulation of aging and metabolism. Targeted sequencing identified a SIRT6 allele containing two linked substitutions (N308K/A313S) as enriched in Ashkenazi Jewish (AJ) centenarians as compared to AJ control individuals. Characterization of this SIRT6 (centSIRT6) allele demonstrated it to be a stronger suppressor of LINE1 retrotransposons, confer enhanced stimulation of DNA double strand break repair, and more robust cancer cell killing compared to the wild type. Surprisingly, centSIRT6 displayed weaker deacetylase activity, but stronger mADPr activity, over a range of NAD<sup>+</sup> concentrations and substrates. Additionally, centSIRT6 displayed a stronger interaction with Lamin A/C (LMNA), which correlated with enhanced ribosylation of LMNA. Our results suggest that enhanced SIRT6 function contributes to human longevity by improving genome maintenance via increased mADPr activity and enhanced interaction with LMNA.</p>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472399', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The mating type transcription factor MAT1-1-1 from the fungal human pathogen Aspergillus fumigatus: synthesis, purification, and crystallization of the DNA binding domain', 'authors': \"[{'name': 'Barbara Ramak'}, {'name': 'Ulrich Kck'}, {'name': 'Eckhard Hofmann'}]\", 'abstract': '<p>Mating type (<italic>MAT</italic>) loci are the most important and significant regulators of sexual reproduction and development in ascomycetous fungi. Usually, they encode two transcription factors (TFs), named MAT1-1-1 or MAT1-2-1. Mating-type strains carry only one of the two TF genes, which control expression of pheromone and pheromone receptor genes, involved in the cell-cell recognition process. The present work presents the crystallization for the alpha1 (1) domain of MAT1-1-1 from the human pathogenic fungus <italic>Aspergillus fumigatus</italic> (AfMAT1-1-1). Crystals were obtained for the complex between a polypeptide containing the 1 domain and DNA carrying the AfMAT1-1-1 recognition sequence. A streak seeding technique was applied to improve native crystal quality, resulting in diffraction data to 3.2  resolution. Further, highly redundant data sets were collected from the crystals of selenomethionine-substituted AfMAT1-1-1 with a maximum resolution of 3.2 . This is the first report of structural studies on the 1 domain MAT regulator involved in the mating of ascomycetes.</p><sec><title>Synopsis</title><p>An optimized purification and crystallization protocol together with initial X-ray datasets are described for this mating type transcription factor from human pathogenic fungus <italic>Aspergillus fumigatus</italic>.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.11.472233', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Video-Based Deep Learning to Detect Dyssynergic Defecation with 3D High-Definition Anorectal Manometry', 'authors': \"[{'name': 'Joshua J. Levy'}, {'name': 'Christopher M. Navas'}, {'name': 'Joan A. Chandra'}, {'name': 'Brock C. Christensen'}, {'name': 'Louis J. Vaickus'}, {'name': 'Michael Curley'}, {'name': 'William D. Chey'}, {'name': 'Jason R. Baker'}, {'name': 'Eric D. Shah'}]\", 'abstract': '<sec><title>BACKGROUND AND AIMS</title><p>Evaluation for dyssynergia is the most common reason that gastroenterologists refer patients for anorectal manometry, because dyssynergia is amenable to biofeedback by physical therapists. High-definition anorectal manometry (3D-HDAM) is a promising technology to evaluate anorectal physiology, but adoption remains limited by its sheer complexity. We developed a 3D-HDAM deep learning algorithm to evaluate for dyssynergia.</p></sec><sec><title>METHODS</title><p>Spatial-temporal data were extracted from consecutive 3D-HDAM studies performed between 2018-2020 at a tertiary institution. The technical procedure and gold standard definition of dyssynergia were based on the London consensus, adapted to the needs of 3D-HDAM technology. Three machine learning models were generated: (1) traditional machine learning informed by conventional anorectal function metrics, (2) deep learning, and (3) a hybrid approach. Diagnostic accuracy was evaluated using bootstrap sampling to calculate area-under-the-curve (AUC). To evaluate overfitting, models were validated by adding 502 simulated defecation maneuvers with diagnostic ambiguity.</p></sec><sec><title>RESULTS</title><p>302 3D-HDAM studies representing 1,208 simulated defecation maneuvers were included (average age 55.2 years; 80.5% women). The deep learning model had comparable diagnostic accuracy (AUC=0.91 [95% confidence interval 0.89-0.93]) to traditional (AUC=0.93[0.92-0.95]) and hybrid (AUC=0.96[0.94-0.97]) predictive models in training cohorts. However, the deep learning model handled ambiguous tests more cautiously than other models; the deep learning model was more likely to designate an ambiguous test as inconclusive (odds ratio=4.21[2.78-6.38]) versus traditional/hybrid approaches.</p></sec><sec><title>CONCLUSIONS</title><p>By considering complex spatial-temporal information beyond conventional anorectal function metrics, deep learning on 3D-HDAM technology may enable gastroenterologists to reliably identify and manage dyssynergia in broader practice.</p></sec>', 'posted': '2021-12-13'}, {'doi': '10.1101/2021.12.13.472362', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Combining 13C and 15N from bone and dentine in marine mammal palaeoecological research: insights from toothed whales', 'authors': \"[{'name': 'Alba Rey-Iglesia'}, {'name': 'Tess Wilson'}, {'name': 'Jennifer Routledge'}, {'name': 'Mikkel Skovrind'}, {'name': 'Eva Garde'}, {'name': 'Mads Peter Heide-Jrgensen'}, {'name': 'Paul Szpak'}, {'name': 'Eline D. Lorenzen'}]\", 'abstract': '<p>Stable carbon (<sup>13</sup>C) and nitrogen (<sup>15</sup>N) isotope compositions of bone and dentine collagen extracted from museum specimens have been widely used to study the paleoecology of past populations. Due to possible systematic differences in stable isotope values between bone and dentine, dentine values can be transformed into bone-collagen equivalent using a correction factor. Here, we provide correction factors to transform dentine <sup>13</sup>C and <sup>15</sup>N values into bone-collagen equivalent for two toothed whales: narwhal and beluga. We sampled bone and tooth dentine from the skulls of 11 narwhals and 26 belugas. In narwhals, dentine was sampled from tusk and embedded tooth; in beluga, dentine was sampled from tooth. <sup>13</sup>C and <sup>15</sup>N were measured and intraindividual bone and dentine isotopic compositions were used to calculate correction factors for each species. We detected differences in <sup>13</sup>C and <sup>15</sup>N. In narwhals, we found (i) lower average <sup>13</sup>C and <sup>15</sup>N in bone compared with dentine; (ii) no difference in dentine <sup>13</sup>C between tusk and embedded tooth. For belugas, we also detected lower <sup>13</sup>C and <sup>15</sup>N in bone compared with tooth dentine. The correction factors provided by the study enable the combined analysis of stable isotope data from bone and dentine in these species.</p>', 'posted': '2021-12-14'}, {'doi': '10.1101/2021.12.16.473072', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Optimal spatial allocation of enzymes as an investment problem', 'authors': \"[{'name': 'Giovanni Giunta'}, {'name': 'Filipe Tostevin'}, {'name': 'Sorin Tanase-Nicola'}, {'name': 'Ulrich Gerland'}]\", 'abstract': '<p>Given a limited number of molecular components, cells face various allocation problems demanding decisions on how to distribute their resources. For instance, cells decide which enzymes to produce at what quantity, but also where to position them. Here we focus on the spatial allocation problem of how to distribute enzymes such as to maximize the total reaction flux produced by them in a system with given geometry and boundary conditions. So far, such distributions have been studied by computational optimization, but a deeper theoretical understanding was lacking. We derive an optimal allocation principle, which demands that the available enzymes are distributed such that the marginal flux returns at each occupied position are equal. This homogeneous marginal returns criterion (HMR criterion) corresponds to a portfolio optimization criterion in a scenario where each investment globally feeds back onto all payoffs. The HMR criterion allows us to analytically understand and characterize a localization-delocalization transition in the optimal enzyme distribution that was previously observed numerically. In particular, our analysis reveals the generality of the transition, and produces a practical test for the optimality of enzyme localization by comparing the reaction flux to the influx of substrate. Based on these results, we devise an additive construction algorithm, which builds up optimal enzyme arrangements systematically rather than by trial and error. Taken together, our results reveal a common principle in allocation problems from biology and economics, which can also serve as a design principle for synthetic biomolecular systems.</p>', 'posted': '2021-12-17'}, {'doi': '10.1101/2021.12.12.472268', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Contrastive Cycle Adversarial Autoencoders for Single-cell Multi-omics Alignment and Integration', 'authors': \"[{'name': 'Xuesong Wang'}, {'name': 'Zhihang Hu'}, {'name': 'Tingyang Yu'}, {'name': 'Ruijie Wang'}, {'name': 'Yumeng Wei'}, {'name': 'Juan Shu'}, {'name': 'Jianzhu Ma'}, {'name': 'Yu Li'}]\", 'abstract': '<p>Muilti-modality data are ubiquitous in biology, especially that we have entered the multi-omics era, when we can measure the same biological object (cell) from different aspects (omics) to provide a more comprehensive insight into the cellular system. When dealing with such multi-omics data, the first step is to determine the correspondence among different modalities. In other words, we should match data from different spaces corresponding to the same object. This problem is particularly challenging in the single-cell multi-omics scenario because such data are very sparse with extremely high dimensions. Secondly, matched single-cell multi-omics data are rare and hard to collect. Furthermore, due to the limitations of the experimental environment, the data are usually highly noisy. To promote the single-cell multi-omics research, we overcome the above challenges, proposing a novel framework to align and integrate single-cell RNA-seq data and single-cell ATAC-seq data. Our approach can efficiently map the above data with high sparsity and noise from different spaces to a low-dimensional manifold in a unified space, making the downstream alignment and integration straightforward. Compared with the other state-of-the-art methods, our method performs better in both simulated and real single-cell data. The proposed method is helpful for the single-cell multi-omics research. The improvement for integration on the simulated data is significant.</p>', 'posted': '2021-12-13'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.12.14.472630</td>\n",
       "      <td>646</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Considerable escape of SARS-CoV-2 variant Omic...</td>\n",
       "      <td>[{'name': 'Delphine Planas'}, {'name': 'Nell S...</td>\n",
       "      <td>&lt;p&gt;The SARS-CoV-2 Omicron variant was first id...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.12.14.21267755</td>\n",
       "      <td>520</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>mRNA-based COVID-19 vaccine boosters induce ne...</td>\n",
       "      <td>[{'name': 'Wilfredo F. Garcia-Beltran'}, {'nam...</td>\n",
       "      <td>&lt;p&gt;Recent surveillance has revealed the emerge...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.12.12.472242</td>\n",
       "      <td>291</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Dynamics of CTCF and cohesin mediated chromati...</td>\n",
       "      <td>[{'name': 'Michele Gabriele'}, {'name': 'Hugo ...</td>\n",
       "      <td>&lt;p&gt;Animal genomes are folded into loops and to...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.12.13.472469</td>\n",
       "      <td>171</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Giant Starship elements mobilize accessory gen...</td>\n",
       "      <td>[{'name': 'Emile Gluck-Thaler'}, {'name': 'Tim...</td>\n",
       "      <td>&lt;p&gt;Accessory genes are variably present among ...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.12.10.21267523</td>\n",
       "      <td>142</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Human serum from SARS-CoV-2 vaccinated and COV...</td>\n",
       "      <td>[{'name': 'Maren Schubert'}, {'name': 'Federic...</td>\n",
       "      <td>&lt;p&gt;The COVID-19 pandemic is caused by the beta...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.12.14.472668</td>\n",
       "      <td>142</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pathology</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 Spike protein activates TMEM16F-med...</td>\n",
       "      <td>[{'name': 'Ambra Cappelletto'}, {'name': 'Harr...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Thrombosis of...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.12.16.472950</td>\n",
       "      <td>130</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Assembly of recombinant tau into filaments ide...</td>\n",
       "      <td>[{'name': 'Sofia Lvestam'}, {'name': 'Fujiet ...</td>\n",
       "      <td>&lt;p&gt;Abundant filamentous inclusions of tau are ...</td>\n",
       "      <td>2021-12-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.12.14.21267791</td>\n",
       "      <td>100</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Global epidemiology of SARS-CoV-2 infection: a...</td>\n",
       "      <td>[{'name': 'Isabel Bergeri'}, {'name': 'Mairead...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;COVID-19 case...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.12.14.472704</td>\n",
       "      <td>91</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>True</td>\n",
       "      <td>Simulation of the omicron variant of SARS-CoV-...</td>\n",
       "      <td>[{'name': 'M. Zaki Jawaid'}, {'name': 'A. Baid...</td>\n",
       "      <td>&lt;p&gt;The recent emergence of the omicron variant...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.12.14.472703</td>\n",
       "      <td>86</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Linear plasmids in Klebsiella and other Entero...</td>\n",
       "      <td>[{'name': 'Jane Hawkey'}, {'name': 'Hugh Cotti...</td>\n",
       "      <td>&lt;p&gt;Linear plasmids are extrachromosomal DNA th...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.12.16.472920</td>\n",
       "      <td>80</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>True</td>\n",
       "      <td>Amyloidogenesis of SARS-CoV-2 Spike Protein</td>\n",
       "      <td>[{'name': 'Sofie Nystrom'}, {'name': 'Per Hamm...</td>\n",
       "      <td>&lt;p&gt;SARS-CoV-2 infection is associated with a s...</td>\n",
       "      <td>2021-12-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.12.10.472112</td>\n",
       "      <td>72</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 Spike triggers barrier dysfunction ...</td>\n",
       "      <td>[{'name': 'Scott B. Biering'}, {'name': 'Franc...</td>\n",
       "      <td>&lt;p&gt;Severe COVID-19 is associated with epitheli...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.12.14.472649</td>\n",
       "      <td>70</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Areas Important for Ecological Connectivity Th...</td>\n",
       "      <td>[{'name': 'Richard Pither'}, {'name': \"Paul O'...</td>\n",
       "      <td>&lt;p&gt;Governments around the world have acknowled...</td>\n",
       "      <td>2021-12-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.12.12.472252</td>\n",
       "      <td>66</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Improved neutralization of the SARS-CoV-2 Omic...</td>\n",
       "      <td>[{'name': 'Kerri Basile'}, {'name': 'Rebecca J...</td>\n",
       "      <td>&lt;p&gt;In late November 2021, the World Health Org...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.12.13.472392</td>\n",
       "      <td>52</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Characterizing the Behavioral and Neuroendocri...</td>\n",
       "      <td>[{'name': 'Dalia Murra'}, {'name': 'Kathryn L....</td>\n",
       "      <td>&lt;p&gt;Evaluating and coping with stressful social...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.12.12.21267677</td>\n",
       "      <td>51</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>False</td>\n",
       "      <td>Artificial intelligence for diagnosis and prog...</td>\n",
       "      <td>[{'name': 'R Borchert'}, {'name': 'T Azevedo'}...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Introduction&lt;/title&gt;&lt;p&gt;Recent deve...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.12.14.472691</td>\n",
       "      <td>50</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>A reusable benchmark of brain-age prediction f...</td>\n",
       "      <td>[{'name': 'Denis A Engemann'}, {'name': 'Apoll...</td>\n",
       "      <td>&lt;p&gt;Population-level modeling can define quanti...</td>\n",
       "      <td>2021-12-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.12.14.472719</td>\n",
       "      <td>48</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Striking Antibody Evasion Manifested by the Om...</td>\n",
       "      <td>[{'name': 'Lihong Liu'}, {'name': 'Sho Iketani...</td>\n",
       "      <td>&lt;p&gt;The Omicron (B.1.1.529) variant of SARS-CoV...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.12.15.472096</td>\n",
       "      <td>40</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>The ParB clamp docks onto Smc for DNA loading ...</td>\n",
       "      <td>[{'name': 'Florian P. Bock'}, {'name': 'Anna A...</td>\n",
       "      <td>&lt;p&gt;Chromosomes readily unlink from one another...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.12.12.21267684</td>\n",
       "      <td>38</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Psychiatry and Clinical Psychology</td>\n",
       "      <td>True</td>\n",
       "      <td>The impact of the COVID-19 school closure on a...</td>\n",
       "      <td>[{'name': 'Helena Svaleryd'}, {'name': 'Evelin...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;School closur...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.12.10.472164</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>EraSOR: Erase Sample Overlap in polygenic scor...</td>\n",
       "      <td>[{'name': 'Shing Wan Choi'}, {'name': 'Timothy...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Polygenic ris...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.12.14.472585</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>Compared with SARS-CoV2 wild types spike prot...</td>\n",
       "      <td>[{'name': 'Michael O. Glocker'}, {'name': 'Kwa...</td>\n",
       "      <td>&lt;p&gt;Our study focuses on free energy calculatio...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.12.13.21267729</td>\n",
       "      <td>36</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Interdependencies between cellular and humoral...</td>\n",
       "      <td>[{'name': 'Moritz M. Hollstein'}, {'name': 'Le...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Homologous an...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.12.13.472419</td>\n",
       "      <td>35</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioengineering</td>\n",
       "      <td>False</td>\n",
       "      <td>Generative Language Modeling for Antibody Design</td>\n",
       "      <td>[{'name': 'Richard W. Shuai'}, {'name': 'Jeffr...</td>\n",
       "      <td>&lt;p&gt;Successful development of monoclonal antibo...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.12.09.472009</td>\n",
       "      <td>34</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>A somatosensory computation that unifies limbs...</td>\n",
       "      <td>[{'name': 'Luke E. Miller'}, {'name': 'Ccile ...</td>\n",
       "      <td>&lt;p&gt;It is often claimed that tools are embodied...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.12.12.472330</td>\n",
       "      <td>34</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Yield is negatively correlated with nucleotide...</td>\n",
       "      <td>[{'name': 'Philipp E. Bayer'}, {'name': 'Haife...</td>\n",
       "      <td>&lt;p&gt;The availability of increasing quantities o...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.12.13.21267756</td>\n",
       "      <td>32</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>A spectrum of recessiveness among Mendelian di...</td>\n",
       "      <td>[{'name': 'Alison R. Barton'}, {'name': 'Marga...</td>\n",
       "      <td>&lt;p&gt;Recent work has found increasing evidence o...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.12.15.472746</td>\n",
       "      <td>31</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Astrocytes in the dorsal vagal complex are not...</td>\n",
       "      <td>[{'name': 'Alastair J MacDonald'}, {'name': 'K...</td>\n",
       "      <td>&lt;p&gt;The dorsal vagal complex (DVC) is a brainst...</td>\n",
       "      <td>2021-12-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.12.15.472866</td>\n",
       "      <td>31</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Understanding the source of METTL3-independent...</td>\n",
       "      <td>[{'name': 'Hui Xian Poh'}, {'name': 'Aashiq H ...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;N&lt;/italic&gt;&lt;sup&gt;6&lt;/sup&gt;-methyladenos...</td>\n",
       "      <td>2021-12-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.12.16.473016</td>\n",
       "      <td>30</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Centromere identity is dependent on nuclear sp...</td>\n",
       "      <td>[{'name': 'Weifang Wu'}, {'name': 'Toni McHugh...</td>\n",
       "      <td>&lt;p&gt;The establishment of centromere-specific CE...</td>\n",
       "      <td>2021-12-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.12.13.472438</td>\n",
       "      <td>30</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Artificial selection of communities drives the...</td>\n",
       "      <td>[{'name': 'Jules Fraboul'}, {'name': 'Giulio B...</td>\n",
       "      <td>&lt;p&gt;Species-rich communities, such as the micro...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.12.13.472426</td>\n",
       "      <td>30</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Absolute Scaling of Single-Cell Transcriptomes...</td>\n",
       "      <td>[{'name': 'Yun-Kyo Kim'}, {'name': 'Miguel Ram...</td>\n",
       "      <td>&lt;p&gt;Hypertranscription facilitates biosynthetic...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.12.13.472377</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>An intrinsic endothelial dysfunction causes ce...</td>\n",
       "      <td>[{'name': 'Sophie Quick'}, {'name': 'Tessa V. ...</td>\n",
       "      <td>&lt;p&gt;Small Vessel Disease (SVD) is the leading c...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.12.16.473018</td>\n",
       "      <td>28</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>vRhyme enables binning of viral genomes from m...</td>\n",
       "      <td>[{'name': 'Kristopher Kieft'}, {'name': 'Alyss...</td>\n",
       "      <td>&lt;p&gt;Genome binning has been essential for chara...</td>\n",
       "      <td>2021-12-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.12.15.21267834</td>\n",
       "      <td>28</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Allergy and Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Comparison of the Immunogenicity of five COVID...</td>\n",
       "      <td>[{'name': 'Chandima Jeewandara'}, {'name': 'In...</td>\n",
       "      <td>&lt;p&gt;We assessed antibody responses 3 months pos...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.12.14.472599</td>\n",
       "      <td>28</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Sir2-domain associated short prokaryotic Argon...</td>\n",
       "      <td>[{'name': 'Mindaugas Zaremba'}, {'name': 'Dona...</td>\n",
       "      <td>&lt;p&gt;Argonaute (Ago) proteins are found in all t...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.12.14.472499</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The structural repertoire of Fusarium oxysporu...</td>\n",
       "      <td>[{'name': 'Daniel S. Yu'}, {'name': 'Megan A. ...</td>\n",
       "      <td>&lt;p&gt;Plant pathogens secrete proteins, known as ...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.12.15.472793</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Dynamic epistasis analysis reveals how chromat...</td>\n",
       "      <td>[{'name': 'Ineke Brouwer'}, {'name': 'Emma Ker...</td>\n",
       "      <td>&lt;p&gt;Transcriptional bursting has been linked to...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.12.14.472556</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Animal Behavior and Cognition</td>\n",
       "      <td>False</td>\n",
       "      <td>Simulated visual hallucinations in virtual rea...</td>\n",
       "      <td>[{'name': 'Clara Rastelli'}, {'name': 'Antonin...</td>\n",
       "      <td>&lt;p&gt;Historically, psychedelic drugs are known t...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.12.14.472574</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Combined SHP2 and ERK inhibition for the treat...</td>\n",
       "      <td>[{'name': 'Katrin J. Ciecielski'}, {'name': 'A...</td>\n",
       "      <td>&lt;p&gt;Mutant KRAS is present in over 90% of pancr...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.12.13.472358</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Lessons from the meiotic recombination landsca...</td>\n",
       "      <td>[{'name': 'Fabien Dutreux'}, {'name': 'Abhishe...</td>\n",
       "      <td>&lt;p&gt;Meiotic recombination has been deeply chara...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.12.15.472838</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>A third vaccination with a single T cell epito...</td>\n",
       "      <td>[{'name': 'Iris N. Pardieck'}, {'name': 'Esme ...</td>\n",
       "      <td>&lt;p&gt;Understanding the mechanisms and impact of ...</td>\n",
       "      <td>2021-12-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.12.15.472619</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>Transcriptomic responses of the human kidney t...</td>\n",
       "      <td>[{'name': 'Christian Hinze'}, {'name': 'Christ...</td>\n",
       "      <td>&lt;p&gt;Background: Acute kidney injury (AKI) occur...</td>\n",
       "      <td>2021-12-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.12.13.472475</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Non-rapid eye movement sleep and wake neurophy...</td>\n",
       "      <td>[{'name': 'Nataliia Kozhemiako'}, {'name': 'Ju...</td>\n",
       "      <td>&lt;p&gt;Motivated by the potential of objective neu...</td>\n",
       "      <td>2021-12-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.12.10.21267582</td>\n",
       "      <td>23</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Echinacea purpurea for the Long-term Preventio...</td>\n",
       "      <td>[{'name': 'Emil Kolev'}, {'name': 'Lilyana Mir...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Introduction&lt;/title&gt;&lt;p&gt;SARS-CoV-2 ...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.12.14.472698</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Evolution of stickleback spines through indepe...</td>\n",
       "      <td>[{'name': 'Julia I. Wucherpfennig'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Understanding the genetic mechanisms leadin...</td>\n",
       "      <td>2021-12-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.12.13.472416</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Ecology drives the degree of convergence in th...</td>\n",
       "      <td>[{'name': 'Michael Tobler'}, {'name': 'Ryan Gr...</td>\n",
       "      <td>&lt;p&gt;Convergent evolution, where independent lin...</td>\n",
       "      <td>2021-12-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.12.17.473116</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Intra-colony channel morphology in Escherichia...</td>\n",
       "      <td>[{'name': 'Beatrice Bottura'}, {'name': 'Liam ...</td>\n",
       "      <td>&lt;p&gt;Nutrient-transporting channels are found th...</td>\n",
       "      <td>2021-12-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.12.14.472674</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pharmacology and Toxicology</td>\n",
       "      <td>False</td>\n",
       "      <td>Qualifying a human Liver-Chip for predictive t...</td>\n",
       "      <td>[{'name': 'Lorna Ewart'}, {'name': 'Athanasia ...</td>\n",
       "      <td>&lt;p&gt;Human organ-on-a-chip (Organ-Chip) technolo...</td>\n",
       "      <td>2021-12-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.12.12.472255</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Broad antiviral effects of Echinacea purpurea ...</td>\n",
       "      <td>[{'name': 'Selvarani Vimalanathan'}, {'name': ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;SARS-CoV-2 va...</td>\n",
       "      <td>2021-12-13</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.12.14.472630     646  bioRxiv   \n",
       "1   10.1101/2021.12.14.21267755     520  medRxiv   \n",
       "2     10.1101/2021.12.12.472242     291  bioRxiv   \n",
       "3     10.1101/2021.12.13.472469     171  bioRxiv   \n",
       "4   10.1101/2021.12.10.21267523     142  medRxiv   \n",
       "5     10.1101/2021.12.14.472668     142  bioRxiv   \n",
       "6     10.1101/2021.12.16.472950     130  bioRxiv   \n",
       "7   10.1101/2021.12.14.21267791     100  medRxiv   \n",
       "8     10.1101/2021.12.14.472704      91  bioRxiv   \n",
       "9     10.1101/2021.12.14.472703      86  bioRxiv   \n",
       "10    10.1101/2021.12.16.472920      80  bioRxiv   \n",
       "11    10.1101/2021.12.10.472112      72  bioRxiv   \n",
       "12    10.1101/2021.12.14.472649      70  bioRxiv   \n",
       "13    10.1101/2021.12.12.472252      66  bioRxiv   \n",
       "14    10.1101/2021.12.13.472392      52  bioRxiv   \n",
       "15  10.1101/2021.12.12.21267677      51  medRxiv   \n",
       "16    10.1101/2021.12.14.472691      50  bioRxiv   \n",
       "17    10.1101/2021.12.14.472719      48  bioRxiv   \n",
       "18    10.1101/2021.12.15.472096      40  bioRxiv   \n",
       "19  10.1101/2021.12.12.21267684      38  medRxiv   \n",
       "20    10.1101/2021.12.10.472164      37  bioRxiv   \n",
       "21    10.1101/2021.12.14.472585      37  bioRxiv   \n",
       "22  10.1101/2021.12.13.21267729      36  medRxiv   \n",
       "23    10.1101/2021.12.13.472419      35  bioRxiv   \n",
       "24    10.1101/2021.12.09.472009      34  bioRxiv   \n",
       "25    10.1101/2021.12.12.472330      34  bioRxiv   \n",
       "26  10.1101/2021.12.13.21267756      32  medRxiv   \n",
       "27    10.1101/2021.12.15.472746      31  bioRxiv   \n",
       "28    10.1101/2021.12.15.472866      31  bioRxiv   \n",
       "29    10.1101/2021.12.16.473016      30  bioRxiv   \n",
       "30    10.1101/2021.12.13.472438      30  bioRxiv   \n",
       "31    10.1101/2021.12.13.472426      30  bioRxiv   \n",
       "32    10.1101/2021.12.13.472377      29  bioRxiv   \n",
       "33    10.1101/2021.12.16.473018      28  bioRxiv   \n",
       "34  10.1101/2021.12.15.21267834      28  medRxiv   \n",
       "35    10.1101/2021.12.14.472599      28  bioRxiv   \n",
       "36    10.1101/2021.12.14.472499      27  bioRxiv   \n",
       "37    10.1101/2021.12.15.472793      27  bioRxiv   \n",
       "38    10.1101/2021.12.14.472556      26  bioRxiv   \n",
       "39    10.1101/2021.12.14.472574      26  bioRxiv   \n",
       "40    10.1101/2021.12.13.472358      24  bioRxiv   \n",
       "41    10.1101/2021.12.15.472838      24  bioRxiv   \n",
       "42    10.1101/2021.12.15.472619      24  bioRxiv   \n",
       "43    10.1101/2021.12.13.472475      23  bioRxiv   \n",
       "44  10.1101/2021.12.10.21267582      23  medRxiv   \n",
       "45    10.1101/2021.12.14.472698      21  bioRxiv   \n",
       "46    10.1101/2021.12.13.472416      20  bioRxiv   \n",
       "47    10.1101/2021.12.17.473116      20  bioRxiv   \n",
       "48    10.1101/2021.12.14.472674      20  bioRxiv   \n",
       "49    10.1101/2021.12.12.472255      19  bioRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0                              Immunology   True   \n",
       "1   Infectious Diseases (except HIV/AIDS)   True   \n",
       "2                              Biophysics  False   \n",
       "3                                Genomics  False   \n",
       "4   Infectious Diseases (except HIV/AIDS)   True   \n",
       "5                               Pathology   True   \n",
       "6                            Neuroscience  False   \n",
       "7                            Epidemiology   True   \n",
       "8                              Biophysics   True   \n",
       "9                            Microbiology  False   \n",
       "10                           Biochemistry   True   \n",
       "11                           Microbiology   True   \n",
       "12                                Ecology  False   \n",
       "13                           Microbiology   True   \n",
       "14                           Neuroscience  False   \n",
       "15                              Neurology  False   \n",
       "16                           Neuroscience  False   \n",
       "17                           Microbiology   True   \n",
       "18                           Biochemistry  False   \n",
       "19     Psychiatry and Clinical Psychology   True   \n",
       "20                         Bioinformatics  False   \n",
       "21                      Molecular Biology   True   \n",
       "22  Infectious Diseases (except HIV/AIDS)   True   \n",
       "23                         Bioengineering  False   \n",
       "24                           Neuroscience  False   \n",
       "25                          Plant Biology  False   \n",
       "26           Genetic and Genomic Medicine  False   \n",
       "27                           Neuroscience  False   \n",
       "28                      Molecular Biology  False   \n",
       "29                           Cell Biology  False   \n",
       "30                   Evolutionary Biology  False   \n",
       "31                               Genomics  False   \n",
       "32                           Neuroscience  False   \n",
       "33                         Bioinformatics  False   \n",
       "34                 Allergy and Immunology   True   \n",
       "35                      Molecular Biology  False   \n",
       "36                          Plant Biology  False   \n",
       "37                      Molecular Biology  False   \n",
       "38          Animal Behavior and Cognition  False   \n",
       "39                         Cancer Biology  False   \n",
       "40                               Genetics  False   \n",
       "41                             Immunology   True   \n",
       "42                      Molecular Biology   True   \n",
       "43                           Neuroscience  False   \n",
       "44  Infectious Diseases (except HIV/AIDS)   True   \n",
       "45                   Evolutionary Biology  False   \n",
       "46                   Evolutionary Biology  False   \n",
       "47                           Microbiology  False   \n",
       "48            Pharmacology and Toxicology  False   \n",
       "49                           Microbiology   True   \n",
       "\n",
       "                                                title  \\\n",
       "0   Considerable escape of SARS-CoV-2 variant Omic...   \n",
       "1   mRNA-based COVID-19 vaccine boosters induce ne...   \n",
       "2   Dynamics of CTCF and cohesin mediated chromati...   \n",
       "3   Giant Starship elements mobilize accessory gen...   \n",
       "4   Human serum from SARS-CoV-2 vaccinated and COV...   \n",
       "5   SARS-CoV-2 Spike protein activates TMEM16F-med...   \n",
       "6   Assembly of recombinant tau into filaments ide...   \n",
       "7   Global epidemiology of SARS-CoV-2 infection: a...   \n",
       "8   Simulation of the omicron variant of SARS-CoV-...   \n",
       "9   Linear plasmids in Klebsiella and other Entero...   \n",
       "10        Amyloidogenesis of SARS-CoV-2 Spike Protein   \n",
       "11  SARS-CoV-2 Spike triggers barrier dysfunction ...   \n",
       "12  Areas Important for Ecological Connectivity Th...   \n",
       "13  Improved neutralization of the SARS-CoV-2 Omic...   \n",
       "14  Characterizing the Behavioral and Neuroendocri...   \n",
       "15  Artificial intelligence for diagnosis and prog...   \n",
       "16  A reusable benchmark of brain-age prediction f...   \n",
       "17  Striking Antibody Evasion Manifested by the Om...   \n",
       "18  The ParB clamp docks onto Smc for DNA loading ...   \n",
       "19  The impact of the COVID-19 school closure on a...   \n",
       "20  EraSOR: Erase Sample Overlap in polygenic scor...   \n",
       "21  Compared with SARS-CoV2 wild types spike prot...   \n",
       "22  Interdependencies between cellular and humoral...   \n",
       "23   Generative Language Modeling for Antibody Design   \n",
       "24  A somatosensory computation that unifies limbs...   \n",
       "25  Yield is negatively correlated with nucleotide...   \n",
       "26  A spectrum of recessiveness among Mendelian di...   \n",
       "27  Astrocytes in the dorsal vagal complex are not...   \n",
       "28  Understanding the source of METTL3-independent...   \n",
       "29  Centromere identity is dependent on nuclear sp...   \n",
       "30  Artificial selection of communities drives the...   \n",
       "31  Absolute Scaling of Single-Cell Transcriptomes...   \n",
       "32  An intrinsic endothelial dysfunction causes ce...   \n",
       "33  vRhyme enables binning of viral genomes from m...   \n",
       "34  Comparison of the Immunogenicity of five COVID...   \n",
       "35  Sir2-domain associated short prokaryotic Argon...   \n",
       "36  The structural repertoire of Fusarium oxysporu...   \n",
       "37  Dynamic epistasis analysis reveals how chromat...   \n",
       "38  Simulated visual hallucinations in virtual rea...   \n",
       "39  Combined SHP2 and ERK inhibition for the treat...   \n",
       "40  Lessons from the meiotic recombination landsca...   \n",
       "41  A third vaccination with a single T cell epito...   \n",
       "42  Transcriptomic responses of the human kidney t...   \n",
       "43  Non-rapid eye movement sleep and wake neurophy...   \n",
       "44  Echinacea purpurea for the Long-term Preventio...   \n",
       "45  Evolution of stickleback spines through indepe...   \n",
       "46  Ecology drives the degree of convergence in th...   \n",
       "47  Intra-colony channel morphology in Escherichia...   \n",
       "48  Qualifying a human Liver-Chip for predictive t...   \n",
       "49  Broad antiviral effects of Echinacea purpurea ...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Delphine Planas'}, {'name': 'Nell S...   \n",
       "1   [{'name': 'Wilfredo F. Garcia-Beltran'}, {'nam...   \n",
       "2   [{'name': 'Michele Gabriele'}, {'name': 'Hugo ...   \n",
       "3   [{'name': 'Emile Gluck-Thaler'}, {'name': 'Tim...   \n",
       "4   [{'name': 'Maren Schubert'}, {'name': 'Federic...   \n",
       "5   [{'name': 'Ambra Cappelletto'}, {'name': 'Harr...   \n",
       "6   [{'name': 'Sofia Lvestam'}, {'name': 'Fujiet ...   \n",
       "7   [{'name': 'Isabel Bergeri'}, {'name': 'Mairead...   \n",
       "8   [{'name': 'M. Zaki Jawaid'}, {'name': 'A. Baid...   \n",
       "9   [{'name': 'Jane Hawkey'}, {'name': 'Hugh Cotti...   \n",
       "10  [{'name': 'Sofie Nystrom'}, {'name': 'Per Hamm...   \n",
       "11  [{'name': 'Scott B. Biering'}, {'name': 'Franc...   \n",
       "12  [{'name': 'Richard Pither'}, {'name': \"Paul O'...   \n",
       "13  [{'name': 'Kerri Basile'}, {'name': 'Rebecca J...   \n",
       "14  [{'name': 'Dalia Murra'}, {'name': 'Kathryn L....   \n",
       "15  [{'name': 'R Borchert'}, {'name': 'T Azevedo'}...   \n",
       "16  [{'name': 'Denis A Engemann'}, {'name': 'Apoll...   \n",
       "17  [{'name': 'Lihong Liu'}, {'name': 'Sho Iketani...   \n",
       "18  [{'name': 'Florian P. Bock'}, {'name': 'Anna A...   \n",
       "19  [{'name': 'Helena Svaleryd'}, {'name': 'Evelin...   \n",
       "20  [{'name': 'Shing Wan Choi'}, {'name': 'Timothy...   \n",
       "21  [{'name': 'Michael O. Glocker'}, {'name': 'Kwa...   \n",
       "22  [{'name': 'Moritz M. Hollstein'}, {'name': 'Le...   \n",
       "23  [{'name': 'Richard W. Shuai'}, {'name': 'Jeffr...   \n",
       "24  [{'name': 'Luke E. Miller'}, {'name': 'Ccile ...   \n",
       "25  [{'name': 'Philipp E. Bayer'}, {'name': 'Haife...   \n",
       "26  [{'name': 'Alison R. Barton'}, {'name': 'Marga...   \n",
       "27  [{'name': 'Alastair J MacDonald'}, {'name': 'K...   \n",
       "28  [{'name': 'Hui Xian Poh'}, {'name': 'Aashiq H ...   \n",
       "29  [{'name': 'Weifang Wu'}, {'name': 'Toni McHugh...   \n",
       "30  [{'name': 'Jules Fraboul'}, {'name': 'Giulio B...   \n",
       "31  [{'name': 'Yun-Kyo Kim'}, {'name': 'Miguel Ram...   \n",
       "32  [{'name': 'Sophie Quick'}, {'name': 'Tessa V. ...   \n",
       "33  [{'name': 'Kristopher Kieft'}, {'name': 'Alyss...   \n",
       "34  [{'name': 'Chandima Jeewandara'}, {'name': 'In...   \n",
       "35  [{'name': 'Mindaugas Zaremba'}, {'name': 'Dona...   \n",
       "36  [{'name': 'Daniel S. Yu'}, {'name': 'Megan A. ...   \n",
       "37  [{'name': 'Ineke Brouwer'}, {'name': 'Emma Ker...   \n",
       "38  [{'name': 'Clara Rastelli'}, {'name': 'Antonin...   \n",
       "39  [{'name': 'Katrin J. Ciecielski'}, {'name': 'A...   \n",
       "40  [{'name': 'Fabien Dutreux'}, {'name': 'Abhishe...   \n",
       "41  [{'name': 'Iris N. Pardieck'}, {'name': 'Esme ...   \n",
       "42  [{'name': 'Christian Hinze'}, {'name': 'Christ...   \n",
       "43  [{'name': 'Nataliia Kozhemiako'}, {'name': 'Ju...   \n",
       "44  [{'name': 'Emil Kolev'}, {'name': 'Lilyana Mir...   \n",
       "45  [{'name': 'Julia I. Wucherpfennig'}, {'name': ...   \n",
       "46  [{'name': 'Michael Tobler'}, {'name': 'Ryan Gr...   \n",
       "47  [{'name': 'Beatrice Bottura'}, {'name': 'Liam ...   \n",
       "48  [{'name': 'Lorna Ewart'}, {'name': 'Athanasia ...   \n",
       "49  [{'name': 'Selvarani Vimalanathan'}, {'name': ...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>The SARS-CoV-2 Omicron variant was first id...  2021-12-15  \n",
       "1   <p>Recent surveillance has revealed the emerge...  2021-12-14  \n",
       "2   <p>Animal genomes are folded into loops and to...  2021-12-13  \n",
       "3   <p>Accessory genes are variably present among ...  2021-12-14  \n",
       "4   <p>The COVID-19 pandemic is caused by the beta...  2021-12-13  \n",
       "5   <sec><title>Background</title><p>Thrombosis of...  2021-12-15  \n",
       "6   <p>Abundant filamentous inclusions of tau are ...  2021-12-17  \n",
       "7   <sec><title>Background</title><p>COVID-19 case...  2021-12-15  \n",
       "8   <p>The recent emergence of the omicron variant...  2021-12-15  \n",
       "9   <p>Linear plasmids are extrachromosomal DNA th...  2021-12-15  \n",
       "10  <p>SARS-CoV-2 infection is associated with a s...  2021-12-17  \n",
       "11  <p>Severe COVID-19 is associated with epitheli...  2021-12-13  \n",
       "12  <p>Governments around the world have acknowled...  2021-12-16  \n",
       "13  <p>In late November 2021, the World Health Org...  2021-12-13  \n",
       "14  <p>Evaluating and coping with stressful social...  2021-12-14  \n",
       "15  <sec><title>Introduction</title><p>Recent deve...  2021-12-14  \n",
       "16  <p>Population-level modeling can define quanti...  2021-12-16  \n",
       "17  <p>The Omicron (B.1.1.529) variant of SARS-CoV...  2021-12-15  \n",
       "18  <p>Chromosomes readily unlink from one another...  2021-12-15  \n",
       "19  <sec><title>Background</title><p>School closur...  2021-12-14  \n",
       "20  <sec><title>Background</title><p>Polygenic ris...  2021-12-13  \n",
       "21  <p>Our study focuses on free energy calculatio...  2021-12-15  \n",
       "22  <sec><title>Background</title><p>Homologous an...  2021-12-14  \n",
       "23  <p>Successful development of monoclonal antibo...  2021-12-14  \n",
       "24  <p>It is often claimed that tools are embodied...  2021-12-13  \n",
       "25  <p>The availability of increasing quantities o...  2021-12-13  \n",
       "26  <p>Recent work has found increasing evidence o...  2021-12-14  \n",
       "27  <p>The dorsal vagal complex (DVC) is a brainst...  2021-12-16  \n",
       "28  <p><italic>N</italic><sup>6</sup>-methyladenos...  2021-12-16  \n",
       "29  <p>The establishment of centromere-specific CE...  2021-12-16  \n",
       "30  <p>Species-rich communities, such as the micro...  2021-12-14  \n",
       "31  <p>Hypertranscription facilitates biosynthetic...  2021-12-13  \n",
       "32  <p>Small Vessel Disease (SVD) is the leading c...  2021-12-14  \n",
       "33  <p>Genome binning has been essential for chara...  2021-12-18  \n",
       "34  <p>We assessed antibody responses 3 months pos...  2021-12-15  \n",
       "35  <p>Argonaute (Ago) proteins are found in all t...  2021-12-14  \n",
       "36  <p>Plant pathogens secrete proteins, known as ...  2021-12-15  \n",
       "37  <p>Transcriptional bursting has been linked to...  2021-12-15  \n",
       "38  <p>Historically, psychedelic drugs are known t...  2021-12-15  \n",
       "39  <p>Mutant KRAS is present in over 90% of pancr...  2021-12-15  \n",
       "40  <p>Meiotic recombination has been deeply chara...  2021-12-13  \n",
       "41  <p>Understanding the mechanisms and impact of ...  2021-12-18  \n",
       "42  <p>Background: Acute kidney injury (AKI) occur...  2021-12-16  \n",
       "43  <p>Motivated by the potential of objective neu...  2021-12-16  \n",
       "44  <sec><title>Introduction</title><p>SARS-CoV-2 ...  2021-12-14  \n",
       "45  <p>Understanding the genetic mechanisms leadin...  2021-12-15  \n",
       "46  <p>Convergent evolution, where independent lin...  2021-12-14  \n",
       "47  <p>Nutrient-transporting channels are found th...  2021-12-18  \n",
       "48  <p>Human organ-on-a-chip (Organ-Chip) technolo...  2021-12-16  \n",
       "49  <sec><title>Background</title><p>SARS-CoV-2 va...  2021-12-13  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "271 preprints (including 57 covering SARS-CoV-2, 221 from bioRxiv and 50 from medRxiv) published in the last 7 days before 2021-12-20 had been tweeted at least 3 times (maximum 646)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.9"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
