{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "26919 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=21ee8ccf-00e9-4109-816e-c91cf9848a98&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=79859332-466d-4e53-bd66-48495e51ee0d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c6db1142-4267-481e-ba41-3d060702dbf8&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=63e68712-166f-4ae6-b09c-bfcb6065b278&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=adbf374c-08ec-4b03-9be2-b9b15ada702c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7a5fcfcd-8312-4fe4-a0a3-53d977d1a64e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d0c71639-c41f-43e1-a2e4-34988d73cc10&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=76de2562-235e-4f77-a271-8097d218ee8b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a13ef075-ba13-4593-bcfe-aa52b0c551a4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2c1e8c0b-75ad-461c-8898-2a85f5a23588&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=08abf5e0-b996-4a22-acee-f8c55c1cb53b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=ae332989-87e8-4dc4-be18-89a1cc1fcb09&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=11fee6c5-7e78-4594-a8d1-be1980c22804&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=be35ccf0-f354-4a75-b099-ab5c2646dd40&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=26270c2d-3c9f-4857-94d6-5cf8f396e5e6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=0608b357-0762-403f-8e3e-089631842eab&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=728133f7-fb1e-4e41-b1fe-cbeb0fa236ed&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=69bf37d8-2388-4eeb-9e34-6ab370f3e04a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0018.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=54cd05f1-6a68-4a89-aa50-6f30329948df&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0019.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=eaeecab6-6af7-45b0-8a97-ae442f763aad&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0020.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=ab1dc7de-31d7-43ec-b451-47cb96dd46cc&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0021.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=788a31c0-8c0b-4381-a062-c52d6ae0ab6e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0022.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4fcc718b-1506-48fa-8823-577eba201d1f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0023.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=317e73c5-35d1-451f-a0cf-48277ec2df4a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0024.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e363cf62-287b-4546-9022-6c49e4f176f6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0025.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=04aa15fb-6b57-4aae-9de8-5413ab5a4ec2&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-12-27&until-occurred-date=2022-01-03\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0026.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-10-18.csv\n",
      "failed to load preprint_tweets_2021-10-25.csv\n",
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-12-20.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load preprint_tweets_2021-12-27.csv\n",
      "failed to load preprint_tweets_2021-10-23.csv\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-11-01.csv\n",
      "failed to load preprint_tweets_2021-11-15.csv\n",
      "failed to load preprint_tweets_2021-11-29.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-11-08.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load preprint_tweets_2021-11-22.csv\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-12-06.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-12-13.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.12.20.21267966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.18.21268018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.21268436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21265796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256931...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.07.20180448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.18.20021881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21267146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472585...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21263608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471446...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.21266068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21268007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474149...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.21267114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21266377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21268008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21267961...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.471245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474382...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21268179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474304...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.472683...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21253914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20216630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.20244145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.21268134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.474183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275939.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.471045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.470764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.01.20144030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.21268540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.474246...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.472081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.21267985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.21268378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473191...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.21265574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21266035...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.471005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.21268373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.23.352831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474462...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.21268385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469842...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.30.21267048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474357...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.24.436523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265968...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473899...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.473725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.473670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.21268103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21266055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265397...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.471325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.21268538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.21268442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.446289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.472248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.436228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.474189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.21268159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21265753...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20087965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.27.20045427...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.471934...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21268338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.01.20241653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.31.21268575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.30.21267047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.474085...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.21268070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.474262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.25.20162107...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.456256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21267350...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.465626...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.09.032995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.19.158717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.05.935387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266775...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.15.275891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.21251022...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.21267432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.474025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.11.872564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.471185...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.21267194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.18.942938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265216...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.026559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452118...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.470392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.30.424757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.11.987073...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.02.019075...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473890...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.19.043851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.104356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.24.111831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.27.063552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.433931...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/480418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.21260250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265155...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464043...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.02.019711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.28.066316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.031138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.10.079236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.21260556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20200600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.28.20079582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.20232850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.31.474679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261118...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.31.016915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265197...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261623...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.21268171...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.21265567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.16.993683...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.02.930917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.29.926022...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.21256281...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473928...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.472410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.01.892539...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.21.885863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.20.884171...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.11.872937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/111674...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/078378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.444523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257486...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.08.20021311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.20248612...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.425973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.21267898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.322388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.445746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.09.940510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.21.954966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21257974...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.15.098137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.05.188862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.07.191197...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.23.111971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.25.220806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474558...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.20103440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.231852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.09.142257...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.205922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.20113324...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.21267504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474445...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.30.181438...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.21257899...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266824...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263412...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.938563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.21251034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.06.961805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.10.942656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264593...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.21259561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.426404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.973685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.08.139733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.973412...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.29.226357...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474454...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.20.21250141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.21249118...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.21249214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.07.938621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.221598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.07.21266031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.20162420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265860...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20120857...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.25.20161984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.18.20024653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.03.974923...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.444366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.07.938381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.20143941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.438778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.184986...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.14.876391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.12.873844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.21253264...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264257...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.28.21254516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.25.20139725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21266268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.21.436342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.26.888503...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.08.193839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.439437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.08.20149120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.05.936047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.18.209528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.07.20170142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264585...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.15.148353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21268285...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21268244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.29.924332...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.15.20154245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.19.911990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.17.910786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.17.20017897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.14.905075...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.07.897900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.22.20232959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.31.363481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.03.20242412...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.110577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.10.20229369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.27.20068346...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263119...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.21262859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.02.20087544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.21263168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.01.20241570...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21262892...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.25.115675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21263855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.26.062158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.26.062927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.473672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.20232561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.20.20198499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.127589...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.111401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264534...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.471057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.21265395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.28.270561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.29.20203646...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.09.20227843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474397...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.473349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.29.318766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21262871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423273...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.18.992784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.126839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21262896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.11.245720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.116715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.23.003509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.20243337...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.474252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.474179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.28.20046235...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.25.008078...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21260111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.29.20203653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21262017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.20.052027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.21264136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21261771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/022459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256683...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265618...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21264979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21265033...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21253487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.471237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264512...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.21267300...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.21264033...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.442980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.21264455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.21261731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21260506...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.470882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263694...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.21259341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21258830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.21263342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264698...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259306...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265263...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.468910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.21265709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.21265203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.21263768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.473367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.21267295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.21256639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.21257205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.473393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21265985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258127...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.470978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.469760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.21260633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/008326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.21268505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.20246710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.428622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.23.057547...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20201848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/795997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.09.20126490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.390583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.303990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.20248150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.01.474684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21250526...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.10.292284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.24.005223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20115485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473923...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.09.21251280...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.20239954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.26.20236489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.21.105551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.28.20240176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21252634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.20155556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.20211755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.24.21250413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.26.20080820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.22.349985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.198283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.473589...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.05.187542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.426390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.474205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.12.874230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.17.20214304...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275981.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.14.20212613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.20167932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.474302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.25.171918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.21268413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21251982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21268319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.09.083907...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20088260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.218768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.21265088...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.21264908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.463740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.442916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.427519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264054...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.471999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.17.208199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463346...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.473377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.470946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.25.20238592...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.21265575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.302067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.29.470324...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.20236968...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.15.20232090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.21265693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.472549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.27.20202481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.17.252007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267793...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265094...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.12.21.473774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.442270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.21.21253968...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/117994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.10.034777...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.23.449658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.18.21267628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.01.320507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.384594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474119...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.17.20156265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.09.417709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263700...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260003...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20161042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.29.20235218...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.28.458030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264078...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.423933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.21268206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.27.450067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.452858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.30.20248890...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.31.474618...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21268289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.21252528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21250657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.11.21251166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.21251526...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.434227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.21266704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473676...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21252086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.29.065136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.13.039503...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.30.470679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.440580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/412056...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.25.20079590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.29.470447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.27.20081893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.17.952663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.31.21250871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.27.920371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.20.162701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.468249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.21250840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.474186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.13.435247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.14.21253532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.428137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.446922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.21264450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21266341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.21266627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.25.354241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/509315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.21266727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20217042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/578542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.474232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466336...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.20.955955...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.10.20210070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456894...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275827.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275453.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.21266875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.466521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.13.20211524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.13.947655...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.12.977918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.09.032375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.225102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264626...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21250296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.08.20050559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.474396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.471467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.21268553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267454...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.184507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.21267386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.26.20182824...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.14.20062463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.471847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.03.20167395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474412...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.29.227223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472674...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.07.20092353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.093195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.111005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.21266601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.21252427...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.451222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.20248402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.21268102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474099...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.430171...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445506...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.20237354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21257742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473797...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.29.399923...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.21268269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.472804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.410969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.411389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.451812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.451839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21257075...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.444295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.444389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.444363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.02.441992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.403188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.21268161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.21268226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21249256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.12.430975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.20248932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.437032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.363820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.435052...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.04.20226126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259558...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.473372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473503...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.21254472...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.12.28.21268436', 'tweets': 382, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Vaccine effectiveness against hospital admission in South African health care workers who received a homologous booster of Ad26.COV2 during an Omicron COVID19 wave: Preliminary Results of the Sisonke 2 Study.', 'authors': \"[{'name': 'Glenda E Gray'}, {'name': 'Shirley COLLIE'}, {'name': 'Nigel Garrett'}, {'name': 'Ameena Goga'}, {'name': 'Jared Champion'}, {'name': 'Matt Zylstra'}, {'name': 'Tarylee Reddy'}, {'name': 'Nonhlanhla Yende'}, {'name': 'Ishen Seocharan'}, {'name': 'Azwi Takalani'}, {'name': 'Ian Sanne'}, {'name': 'Fatima Mayat'}, {'name': 'Jackline Odhiambo'}, {'name': 'Lesley Bamford'}, {'name': 'Harry Moultrie'}, {'name': 'Lara Fairall'}, {'name': 'Linda-Gail Bekker'}]\", 'abstract': '<p>Following the results of the ENSEMBLE 2 study, which demonstrated improved vaccine efficacy of a two-dose regimen of Ad26.COV.2 vaccine given 2 months apart, we expanded the Sisonke study which had provided single dose Ad26.COV.2 vaccine to almost 500 000 health care workers (HCW) in South Africa to include a booster dose of the Ad26.COV.2. Sisonke 2 enrolled 227 310 HCW from the 8 November to the 17 December 2021. Enrolment commenced before the onset of the Omicron driven fourth wave in South Africa affording us an opportunity to evaluate early VE in preventing hospital admissions of a homologous boost of the Ad26.COV.2 vaccine given 6-9 months after the initial vaccination in HCW.\\nWe estimated vaccine effectiveness (VE) of the Ad26.COV2.S vaccine booster in 69 092 HCW as compared to unvaccinated individuals enrolled in the same managed care organization using a test negative design. We compared VE against COVID19 admission for omicron during the period 15 November to 20 December 2021. After adjusting for confounders, we observed that VE for hospitalisation increased over time since booster dose, from 63% (95%CI 31-81%); to 84% (95% CI 67-92%) and then 85% (95% CI: 54-95%), 0-13 days, 14-27 days, and 1-2 months post-boost.\\nWe provide the first evidence of the effectiveness of a homologous Ad26.COV.2 vaccine boost given 6-9 months after the initial single vaccination series during a period of omicron variant circulation. This data is important given the increased reliance on the Ad26.COV.2 vaccine in Africa.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.25.474149', 'tweets': 71, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Expanded ACE2 dependencies of diverse SARS-like coronavirus receptor binding domains', 'authors': \"[{'name': 'Sarah M Roelle'}, {'name': 'Nidhi Shukla'}, {'name': 'Anh T Pham'}, {'name': 'Anna M Bruchez'}, {'name': 'Kenneth A Matreyek'}]\", 'abstract': '<p>Viral spillover from animal reservoirs can trigger public health crises and cripple the world economy. Knowing which viruses are primed for zoonotic transmission can focus surveillance efforts and mitigation strategies for future pandemics. Successful engagement of receptor protein orthologs is necessary during cross-species transmission. The clade 1 sarbecoviruses including SARS-CoV and SARS-CoV-2 enter cells via engagement of ACE2, while the receptor for clade 2 and clade 3 remains largely uncharacterized. We developed a mixed cell pseudotyped virus infection assay to determine whether various clade 2 and 3 sarbecovirus spike proteins can enter HEK 293T cells expressing human or Rhinolophus horseshoe bat ACE2 proteins. The receptor binding domains from BtKY72 and Khosta-2 used human ACE2 for entry, while BtKY72 and Khosta-1 exhibited widespread use of diverse rhinolophid ACE2s. A lysine at ACE2 position 31 appeared to be a major determinant of the inability of these RBDs to use a certain ACE2 sequence. The ACE2 protein from R. alcyone engaged all known clade 3 and clade 1 receptor binding domains. We observed little use of Rhinolophus ACE2 orthologs by the clade 2 viruses, supporting the likely use of a separate, unknown receptor. Our results suggest that clade 3 sarbecoviruses from Africa and Europe use Rhinolophus ACE2 for entry, and their spike proteins appear primed to contribute to zoonosis under the right conditions.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.29.474455', 'tweets': 62, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Nerve-mediated amputation-induced stem cell activation primes distant appendages for future regeneration events in axolotl', 'authors': \"[{'name': 'Duygu Payzin-Dogru'}, {'name': 'Sarah E. Wilson'}, {'name': 'Steven J. Blair'}, {'name': 'Burcu Erdogan'}, {'name': 'Shifa Hossain'}, {'name': 'Louis V. Cammarata'}, {'name': 'Juan C. Velazquez Matos'}, {'name': 'Alan Y. Wong'}, {'name': 'Julia Losner'}, {'name': 'Sangwon Min'}, {'name': 'Hani Singer'}, {'name': 'Jessica L. Whited'}]\", 'abstract': '<p>Animals exhibit extreme diversity in regenerative ability. This likely reflects different, lineage-specific selective pressures in their evolutionary histories, but how specific molecular features of regenerative programs help solve species-specific challenges has not been examined in detail.  Here we discover that, in the highly-regenerative axolotl salamander, a conserved, body-wide stem cell activation response triggered in response to limb removal primes undisturbed limbs for regeneration upon subsequent amputation. This response should be particularly useful to salamanders, which frequently lose limbs in response to cannibalism. We further demonstrate the body-wide response requires both peripheral nervous system input at these distant sites and mTOR signaling. We defined gene expression changes within the nerves and nearby tissues, harboring responsive stem cells, leading to identification of candidate genetic pathways influencing distant stem cell activation following amputation. Functional experimentation confirmed a requirement for adrenergic signaling in amputation-induced activation of distant stem cells. These findings reveal a direct link between systemic cellular activation responses to local tissue damage and overall regenerative ability. Similar systemic activation responses to tissue removal have been observed in animals with widely differing regenerative abilities (e.g., planaria to mice), suggesting that it is the responses downstream of these signals, likely sculpted by differing evolutionary pressures, that ultimately distinguish regenerators from non-regenerators.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474457', 'tweets': 59, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Effects of periodic bottlenecks on the dynamics of adaptive evolution in microbial populations', 'authors': \"[{'name': 'Minako Izutsu'}, {'name': 'Devin M. Lake'}, {'name': 'Zachary W. D. Matson'}, {'name': 'Jack P. Dodson'}, {'name': 'Richard E. Lenski'}]\", 'abstract': '<p>Population bottlenecks are common in nature, and they can impact the rate of adaptation in evolving populations. On the one hand, each bottleneck reduces the genetic variation that fuels adaptation. On the other hand, fewer founders can undergo more generations and leave more descendants in a resource-limited environment, which allows surviving beneficial mutations to spread more quickly. Here we investigate the impact of repeated bottlenecks on the dynamics of adaptation in experimental populations of <italic>Escherichia coli</italic>. We propagated 48 populations under four dilution regimes (2-, 8-, 100-, and 1000-fold), all reaching the same final size each day, for 150 days. A simple model in which adaptation is limited by the supply rate of beneficial mutations predicts that fitness gains should be maximized with ~8-fold dilutions. The model also assumes that selection acts only on the overall growth rate and is otherwise identical across dilution regimes. However, we found that selection in the 2-fold regime was qualitatively different from the other treatments. Moreover, we observed earlier and greater fitness gains in the populations subjected to 100- and 1000-fold dilutions than in those that evolved in the 8 fold regime. We also ran simulations using parameters estimated independently from a long-term experiment using the same ancestral strain and environment. The simulations produced dynamics similar to our empirical results under these regimes, and they indicate that the simple model fails owing to the assumption that the supply of beneficial mutations limits adaptation.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.474555', 'tweets': 55, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'A nanopore interface for high bandwidth DNA computing', 'authors': \"[{'name': 'Karen Zhang'}, {'name': 'Yuan-Jyue Chen'}, {'name': 'Kathryn Doroschak'}, {'name': 'Karin Strauss'}, {'name': 'Luis Ceze'}, {'name': 'Georg Seelig'}, {'name': 'Jeff Nivala'}]\", 'abstract': \"<p>DNA has emerged as a powerful substrate for programming information processing machines at the nanoscale. Among the DNA computing primitives used today, DNA strand displacement (DSD) is arguably the most popular, with DSD-based circuit applications ranging from disease diagnostics to molecular artificial neural networks. The outputs of DSD circuits are generally read using fluorescence spectroscopy. However, due to the spectral overlap of typical small-molecule fluorescent reporters, the number of unique outputs that can be detected in parallel is limited, requiring complex optical setups or spatial isolation of reactions to make output bandwidths scalable. Here, we present a multiplexable sequencing-free readout method that enables real-time, kinetic measurement of DSD circuit activity through highly parallel, direct detection of barcoded output strands using nanopore sensor array technology (Oxford Nanopore Technologies' MinION device). We show that engineered reporter probes can be detected and classified with high accuracy at the single-molecule level directly from raw nanopore signals using deep learning. We then demonstrate this method's utility in multiplexed detection of clinically relevant microRNA sequences. These results increase DSD output bandwidth by an order of magnitude over what is possible with fluorescence spectroscopy, laying the foundations for a new paradigm in DNA circuit readout and programmable multiplexed molecular diagnostics using portable nanopore devices.</p>\", 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474489', 'tweets': 50, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'High density single-unit human cortical recordings using the Neuropixels probe', 'authors': \"[{'name': 'Jason E Chung'}, {'name': 'Kristin K Sellers'}, {'name': 'Matthew K Leonard'}, {'name': 'Laura Gwilliams'}, {'name': 'Duo Xu'}, {'name': 'Maximilian Dougherty'}, {'name': 'Viktor Kharazia'}, {'name': 'Marleen Welkenhuysen'}, {'name': 'Barundeb Dutta'}, {'name': 'Edward F Chang'}]\", 'abstract': '<p>A fundamental unit of neural computation is the action potential. While significant advances have been made in the ability to sample action potentials of large numbers of individual neurons in animal models, translation of these methodologies to humans has been lacking due to clinical time constraints, electrical noise in the operating room, and reliability of the methodology. Here we present a reliable method for intraoperative recording of dozens of neurons in humans using the Neuropixels probe, yielding up to ~100 simultaneously-recorded single-units (n=596 across 11 recordings in 8 participants). Most single-units were active within 1 minute of reaching target depth, compatible with clinical time constraints. Cell pairs active close in time were spatially closer in most recordings, demonstrating the power to resolve complex cortical dynamics. Altogether, this approach provides access to population single-unit activity across the depth of human neocortex at scales previously only accessible in animal models.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.28.474382', 'tweets': 46, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Precise measurement of nanoscopic septin ring structures in deep learning-assisted quantitative superresolution microscopy', 'authors': \"[{'name': 'Amin Zehtabian'}, {'name': 'Paul Markus Müller'}, {'name': 'Maximilian Goisser'}, {'name': 'Leon Obendorf'}, {'name': 'Lea Jänisch'}, {'name': 'Nadja Hümpfer'}, {'name': 'Jakob Rentsch'}, {'name': 'Helge Ewers'}]\", 'abstract': '<p>The combination of image analysis and fluorescence superresolution microscopy methods allows for unprecedented insight into the organization of macromolecular assemblies in cells. Advances in deep learning-based object recognition enables the automated processing of large amounts of data, resulting in high accuracy through averaging. However, while the analysis of highly symmetric structures of constant size allows for a resolution approaching the dimensions of structural biology, deep learning methods are prone to different forms of bias. A biased recognition of structures may prohibit the development of readouts for processes that involve significant changes in size or shape of amorphous macromolecular complexes. What is required to overcome this problem is a detailed investigation of potential sources of bias and the rigorous testing of trained models using real or simulated data covering a wide dynamic range of possible results. Here we combine single molecule localization-based superresolution microscopy of septin ring structures with the training of several different deep learning models for a quantitative investigation of bias resulting from different training approaches and finally quantitative changes in septin ring structures. We find that trade-off exists between measurement accuracy and the dynamic range of recognized phenotypes. Using our trained models, we furthermore find that septin ring size can be explained by the number of subunits they are assembled from alone. Our work provides a new experimental system for the investigation of septin polymerization.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.23.21268179', 'tweets': 44, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': False, 'title': 'An online atlas of human plasma metabolite signatures of gut microbiome composition', 'authors': \"[{'name': 'Koen F. Dekkers'}, {'name': 'Sergi Sayols-Baixeras'}, {'name': 'Gabriel Baldanzi'}, {'name': 'Christoph Nowak'}, {'name': 'Ulf Hammar'}, {'name': 'Diem Nguyen'}, {'name': 'Georgios Varotsis'}, {'name': 'Louise Brunkwall'}, {'name': 'Nynne Nielsen'}, {'name': 'Aron C. Eklund'}, {'name': 'Jacob Bak Holm'}, {'name': 'H. Bjørn Nielsen'}, {'name': 'Filip Ottosson'}, {'name': 'Yi-Ting Lin'}, {'name': 'Shafqat Ahmad'}, {'name': 'Lars Lind'}, {'name': 'Johan Sundström'}, {'name': 'Gunnar Engström'}, {'name': 'J. Gustav Smith'}, {'name': 'Johan Ärnlöv'}, {'name': 'Marju Orho-Melander'}, {'name': 'Tove Fall'}]\", 'abstract': '<p>The human gut microbiota produces a variety of small compounds, some of which enter the bloodstream and impact host health. Conversely, various exogenous nutritional and pharmaceutical compounds affect the gut microbiome composition before entering circulation. Characterization of the gut microbiota—host plasma metabolite interactions is an important step towards understanding the effects of the gut microbiota on human health. However, studies involving large and deeply phenotyped cohorts that would reveal such meaningful interactions are scarce. Here, we used deep metagenomic sequencing and ultra-high-performance liquid chromatography linked to mass spectrometry for detailed characterization of the fecal microbiota and plasma metabolome, respectively, of 8,584 participants invited at age 50 to 64 of the Swedish CArdioPulmonary bioImage Study (SCAPIS). After adjusting for multiple comparisons, we identified 1,008 associations between species alpha diversity and plasma metabolites, and 318,944 associations between specific gut metagenomic species and plasma metabolites. The gut microbiota explained up to 50% of the variance of individual plasma metabolites (mean of 4.7%). We present all results as the searchable association atlas \"GUTSY\" as a rich resource for mining associations, and exemplify the potential of the atlas by presenting novel associations between oral medication and the gut microbiome, and microbiota species strongly associated with levels of the uremic toxin p-cresol sulfate. The association atlas can be used as the basis for targeted studies of perturbation of specific bacteria and for identification of candidate plasma biomarkers of gut flora composition.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.28.474304', 'tweets': 40, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Techniques for visualizing fibroblast-vessel interactions in the developing and adult CNS', 'authors': \"[{'name': 'Hannah E Jones'}, {'name': 'Kelsey A Abrams'}, {'name': 'Julie A Siegenthaler'}]\", 'abstract': '<p>Fibroblasts are found associated with blood vessels in various locations across the CNS: in the meninges, the choroid plexus, and in the parenchyma within perivascular spaces. CNS fibroblasts have been characterized using transcriptional profiling and a Col1a1-GFP mouse line used to identify CNS fibroblasts in vivo. However, current methods for visualizing CNS fibroblasts are lacking and, in particular, prevent adequate assessment of fibroblast-vessel interactions. Here, we describe methods for whole mount visualization of meningeal and choroid plexus fibroblasts, and optical tissue clearing methods for visualization of parenchymal vessel-associated fibroblasts. Importantly, these techniques can be combined with immunohistochemistry methods for labeling different cell types in the meninges and blood vasculature as well as EdU-based cell proliferation assays. These methods are ideal for visualization of vessel-fibroblast interactions in these CNS structures and provide significant improvement over traditional sectioning and staining methods. We expect these methods will advance studies of CNS fibroblast development and functions in homeostasis, injury, and disease.</p>', 'posted': '2021-12-28'}, {'doi': '10.1101/2021.12.29.474385', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Single-nuclei isoform RNA sequencing reveals combination patterns of transcript elements across human brain cell types', 'authors': \"[{'name': 'Simon A Hardwick'}, {'name': 'Wen Hu'}, {'name': 'Anoushka Joglekar'}, {'name': 'Li Fan'}, {'name': 'Paul G Collier'}, {'name': 'Careen Foord'}, {'name': 'Jennifer Balacco'}, {'name': 'Natan Belchikov'}, {'name': 'Julien Jarroux'}, {'name': 'Andrey D Prjibelski'}, {'name': 'Alla Mikheenko'}, {'name': 'Wenjie Luo'}, {'name': 'Teresa A Milner'}, {'name': 'Lishomwa Ndhlovu'}, {'name': 'John Q Trojanowski'}, {'name': 'Virginia MY Lee'}, {'name': 'Olivier Fedrigo'}, {'name': 'Dora Tombacz'}, {'name': 'Margaret Elizabeth Ross'}, {'name': 'Erich Jarvis'}, {'name': 'Zsolt Boldogkoi'}, {'name': 'Li Gan'}, {'name': 'Hagen U Tilgner'}]\", 'abstract': \"<p>Single-nuclei RNA-Seq is being widely employed to investigate cell types, especially of human brain and other frozen samples. In contrast to single-cell approaches, however, the majority of single-nuclei RNA counts originate from partially processed RNA leading to intronic cDNAs, thus hindering the investigation of complete isoforms. Here, using microfluidics, PCR-based artifact removal, target enrichment, and long-read sequencing, we developed single-nuclei isoform RNA-sequencing ('SnISOr-Seq'), and applied it to the analysis of human adult frontal cortex samples. We found that exons associated with autism exhibit coordinated and more cell-type specific inclusion than exons associated with schizophrenia or ALS. We discovered two distinct modes of combination patterns: first, those distinguishing cell types in the human brain. These are enriched in combinations of TSS-exon, exon-polyA site, and distant (non-adjacent) exon pairs. Second, those with all isoform combinations found within one neural cell type, which are enriched in adjacent exon pairs. Furthermore, adjacent exon pairs are predominantly mutually associated, while distant pairs are frequently mutually exclusive. Finally, we observed that human-specific exons are as tightly coordinated as conserved exons, pointing to an efficient evolutionary mechanism underpinning coordination. SnISOr-Seq opens the door to single-nuclei long-read isoform analysis in the human brain, and in any frozen, archived or hard-to-dissociate sample.</p>\", 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.474605', 'tweets': 31, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Roles for Mitochondrial Complex I subunits in regulating synaptic transmission and growth', 'authors': \"[{'name': 'Bhagaban Mallik'}, {'name': 'C. Andrew Frank'}]\", 'abstract': '<p>To identify conserved components of synapse function that are also associated with human diseases, we conducted a genetic screen. We used the Drosophila melanogaster neuromuscular junction (NMJ) as a model. We employed RNA interference (RNAi) on selected targets and assayed synapse function by electrophysiology. We focused our screen on genetic factors known to be conserved from human neurological or muscle functions (321 total RNAi lines screened). Knockdown of a particular Mitochondrial Complex I (MCI) subunit gene (ND-20L) lowered levels of NMJ neurotransmission. Due to the severity of the phenotype, we studied MCI function further. Knockdown of core MCI subunits concurrently in neurons and muscle led to impaired neurotransmission. Further, pharmacology targeting MCI phenocopied the impaired neurotransmission phenotype. Finally, MCI subunit knockdowns led to profound cytological defects, including reduced NMJ growth and altered NMJ morphology. Mitochondria are essential for cellular bioenergetics and produce ATP through oxidative phosphorylation. Five multi-protein complexes achieve this task, and MCI is the largest. Impaired Mitochondrial Complex I subunits in humans are associated with disorders such as Parkinsons disease, Leigh syndrome, and cardiomyopathy. Together, our data present an analysis of Complex I in the context of synapse function and plasticity. We speculate that in the context of human MCI dysfunction, similar neuronal and synaptic defects could contribute to pathogenesis.</p>', 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.28.474323', 'tweets': 30, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'A versatile active learning workflow for optimization of genetic and metabolic networks', 'authors': \"[{'name': 'Amir Pandi'}, {'name': 'Christoph Diehl'}, {'name': 'Ali Yazdizadeh Kharrazi'}, {'name': 'Léon Faure'}, {'name': 'Scott A. Scholz'}, {'name': 'Maren Nattermann'}, {'name': 'David Adam'}, {'name': 'Nils Chapin'}, {'name': 'Yeganeh Foroughijabbari'}, {'name': 'Charles Moritz'}, {'name': 'Nicole Paczia'}, {'name': 'Niña Socorro Cortina'}, {'name': 'Jean-Loup Faulon'}, {'name': 'Tobias J. Erb'}]\", 'abstract': '<p>The study, engineering and application of biological networks require practical and efficient approaches. Current optimization efforts of these systems are often limited by wet lab labor and cost, as well as the lack of convenient, easily adoptable computational tools. Aimed at democratization and standardization, we describe METIS, a modular and versatile active machine learning workflow with a simple online interface for the optimization of biological target functions with minimal experimental datasets. We demonstrate our workflow for various applications, from simple to complex gene circuits and metabolic networks, including several cell-free transcription and translation systems, a <italic>LacI</italic>-based multi-level controller and a 27-variable synthetic CO<sub>2</sub>-fixation cycle (CETCH cycle). Using METIS, we could improve above systems between one and two orders of magnitude compared to their original setup with minimal experimental efforts. For the CETCH cycle, we explored the combinatorial space of ~10<sup>25</sup> conditions with only 1,000 experiments to yield the most efficient CO<sub>2</sub>-fixation cascade described to date. Beyond optimization, our workflow also quantifies the relative importance of individual factors to the performance of a system. This allows to identify so far unknown interactions and bottlenecks in complex systems, which paves the way for their hypothesis-driven improvement, which we demonstrate for the <italic>LacI</italic> multi-level controller that we were able to improve by 34-fold after having identified resource competition as limiting factor. Overall, our workflow opens the way for convenient optimization and prototyping of genetic and metabolic networks with customizable adjustments according to user experience, experimental setup, and laboratory facilities.</p>', 'posted': '2021-12-28'}, {'doi': '10.1101/2021.12.30.21268540', 'tweets': 29, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Magnitude and breadth of neutralizing antibody responses elicited by SARS-CoV-2 infection or vaccination', 'authors': \"[{'name': 'Benjamin L. Sievers'}, {'name': 'Saborni Chakraborty'}, {'name': 'Yong Xue'}, {'name': 'Terri Gelbart'}, {'name': 'Joseph C. Gonzalez'}, {'name': 'Arianna G. Cassidy'}, {'name': 'Yarden Golan'}, {'name': 'Mary Prahl'}, {'name': 'Stephanie L. Gaw'}, {'name': 'Prabhu S Arunachalam'}, {'name': 'Catherine A Blish'}, {'name': 'Scott D. Boyd'}, {'name': 'Mark M Davis'}, {'name': 'Prasanna Jagannathan'}, {'name': 'Kari C. Nadeau'}, {'name': 'Bali Pulendran'}, {'name': 'Upinder Singh'}, {'name': 'Richard H. Scheuermann'}, {'name': 'Matthew H. Frieman'}, {'name': 'Sanjay Vashee'}, {'name': 'Taia T. Wang'}, {'name': 'Gene S Tan'}]\", 'abstract': '<p>Multiple SARS-CoV-2 variants that possess mutations associated with increased transmission and antibody escape have arisen over the course of the current pandemic. While the current vaccines have largely been effective against past variants, the number of mutations found on the Omicron (B.1.529) spike appear to diminish the efficacy of pre-existing immunity. Using pseudoparticles expressing the spike of several SARS-CoV-2 variants, we evaluated the magnitude and breadth of the neutralizing antibody response over time in naturally infected and in mRNA-vaccinated individuals. We observed that while boosting increases the magnitude of the antibody response to wildtype (D614), Beta, Delta and Omicron variants, the Omicron variant was the most resistant to neutralization. We further observed that vaccinated healthy adults had robust and broad antibody responses while responses were relatively reduced in vaccinated pregnant women, underscoring the importance of learning how to maximize mRNA vaccine responses in pregnant populations. Findings from this study show substantial heterogeneity in the magnitude and breadth of responses after infection and mRNA vaccination and may support the addition of more conserved viral antigens to existing SARS-CoV-2 vaccines.</p>', 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.27.474246', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Species-specific mitochondria dynamics and metabolism regulate the timing of neuronal development', 'authors': \"[{'name': 'Ryohei Iwata'}, {'name': 'Pierre Casimir'}, {'name': 'Emir Erkol'}, {'name': 'Leïla Boubakar'}, {'name': 'Mélanie Planque'}, {'name': 'Martyna Ditkowska'}, {'name': 'Katlijn Vints'}, {'name': 'Suresh Poovathingal'}, {'name': 'Vaiva Gaspariunaite'}, {'name': 'Matthew Bird'}, {'name': 'Nikky Corthout'}, {'name': 'Pieter Vermeersch'}, {'name': 'Kristofer Davie'}, {'name': 'Natalia V. Gounko'}, {'name': 'Stein Aerts'}, {'name': 'Bart Ghesquière'}, {'name': 'Sarah-Maria Fendt'}, {'name': 'Pierre Vanderhaeghen'}]\", 'abstract': '<p>The evolution of species involves changes in the timeline of key developmental programs. Among these, neuronal development is considerably prolonged in the human cerebral cortex compared with other mammals, leading to brain neoteny. Here we explore whether mitochondria influence the species-specific properties of cortical neuron maturation. By comparing human and mouse cortical neuronal maturation at high temporal and cell resolution, we found a slower pattern of mitochondria development in human cortical neurons compared with the mouse, together with lower mitochondria metabolic activity, particularly oxidative phosphorylation. Stimulation of mitochondria metabolism in human neurons resulted in accelerated maturation, leading to excitable and complex cells weeks ahead of time. Our data identify mitochondria as important regulators of the pace of neuronal development underlying human-specific features of brain evolution.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.24.21268378', 'tweets': 23, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Preserved Omicron Spike specific antibody binding and Fc-recognition across COVID-19 vaccine platforms', 'authors': \"[{'name': 'Yannic Bartsch'}, {'name': 'Xin Tong'}, {'name': 'Jaweon Kang'}, {'name': 'María José Avendaño'}, {'name': 'Eileen F. Serrano'}, {'name': 'Tamara García-Salum'}, {'name': 'Catalina Pardo-Roa'}, {'name': 'Arnoldo Riquelme'}, {'name': 'Rafael A. Medina'}, {'name': 'Galit Alter'}]\", 'abstract': '<p>Despite the dramatic spread of Omicron globally, even among highly vaccinated populations, death rates have not increased concomitantly. These data argue that alternative immune mechanisms, beyond neutralization, may continue to confer protection against severe disease. Beyond their ability to bind and block infection, antibodies contribute to control and clearance of multiple infections via their ability to direct antiviral immunity via Fc-effector mechanisms. Thus, here we probed the ability of vaccine induced antibodies, across three COVID-19 vaccines, to drive Fc-effector activity against Omicron. Despite the significant loss of IgM, IgA and IgG binding to the Omicron Receptor Binding Domain (RBD) across BNT162b2, mRNA-1273, and CoronaVac vaccines, stable isotype binding was observed across all of these vaccines to the Omicron Spike. Compromised RBD binding IgG was accompanied by a significant loss of cross RBD-specific antibody Fcγ-receptor binding by the CoronaVac vaccine, but preservation of RBD-specific FcγR2a and Fcγ3a binding across the mRNA vaccines. Conversely, Spike-specific antibodies exhibited persistent binding to Fcγ-receptors, across all three vaccines, albeit higher binding was observed with the mRNA vaccines, marked by a selective preservation of FcγR2a and Fcγ3a binding antibodies. Thus, despite the significant to near complete loss of Omicron neutralization across several vaccine platforms against Omicron, vaccine induced Spike-specific antibodies continue to recognize the virus and recruit Fc-receptors pointing to a persistent capacity for extra-neutralizing antibodies to contribute Omicron disease attenuation.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.30.474462', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'The evolution and biological correlates of hand preferences in anthropoid primates', 'authors': \"[{'name': 'Kai R Caspar'}, {'name': 'Fabian Pallasdies'}, {'name': 'Larissa Mader'}, {'name': 'Heitor Sartorelli'}, {'name': 'Sabine Begall'}]\", 'abstract': '<p>The evolution of human right-handedness has been intensively debated for decades. Manual lateralization patterns in non-human primates have the potential to elucidate evolutionary determinants of human handedness. However, restricted species samples and inconsistent methodologies are limiting comparative phylogenetic studies. By combining original data with published literature reports, we assembled data on hand preferences for standardized object manipulation in 1,806 individuals from 38 species of anthropoid primates, including monkeys, apes, and humans. Based on that, we employ quantitative phylogenetic methods to test prevalent hypotheses on the roles of ecology, brain size and tool use in primate handedness evolution. We confirm that human right-handedness represents an unparalleled extreme among anthropoids and found taxa displaying significant population-level handedness to be notably rare. Species-level direction of manual lateralization was largely uniform among non-human primates and neither correlated with phylogeny nor with any of the selected biological predictors. In contrast, we recovered highly variable patterns of hand preference strength, which show signatures of both ecology and phylogeny. In particular, terrestrial primates tend to display weaker hand preferences than arboreal species. These results challenge popular ideas on primate handedness evolution, especially the postural origins hypothesis. Furthermore, they point to a potential adaptive benefit of disparate lateralization strength in primates, a measure of hand preference that has often been overlooked in the past. Finally, our data show that human lateralization patterns do not align with trends found among other anthropoids, suggesting that unique selective pressures gave rise to the unusual hand preferences displayed by our species.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.24.21268385', 'tweets': 20, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Schizophrenia-associated somatic copy number variants from 12,834 cases reveal contribution to risk and recurrent, isoform-specific NRXN1 disruptions', 'authors': '[{\\'name\\': \\'Eduardo A Maury\\'}, {\\'name\\': \\'Maxwell A Sherman\\'}, {\\'name\\': \\'Giulio Genovese\\'}, {\\'name\\': \\'Thomas G. Gilgenast\\'}, {\\'name\\': \\'Prashanth Rajarajan\\'}, {\\'name\\': \\'Erin Flaherty\\'}, {\\'name\\': \\'Schahram Akbarian\\'}, {\\'name\\': \\'Andrew Chess\\'}, {\\'name\\': \\'Steven A. McCarroll\\'}, {\\'name\\': \\'Po-Ru Loh\\'}, {\\'name\\': \\'Jennifer E. Philips-Cremins\\'}, {\\'name\\': \\'Kristen J. Brennand\\'}, {\\'name\\': \\'James T.R . Walters\\'}, {\\'name\\': \"Michael O\\'Donovan\"}, {\\'name\\': \\'Patrick Sullivan\\'}, {\\'name\\': \\'Jonathan Sebat\\'}, {\\'name\\': \\'Eunjung A. Lee\\'}, {\\'name\\': \\'Christopher A. Walsh\\'}, {\\'name\\': \\' \\'}, {\\'name\\': \\' \\'}]', 'abstract': \"<p>While inherited and de novo copy number variants (CNV) have been implicated in the genetic architecture of schizophrenia (SCZ), the contribution of somatic CNVs (sCNVs), present in some but not all cells of the body, remains unknown. Here we explore the role of sCNVs in SCZ by analyzing blood-derived genotype arrays from 12,834 SCZ cases and 11,648 controls. sCNVs were more common in cases (0.91%) than in controls (0.51%, p = 2.68e-4). We observed recurrent somatic deletions of exons 1-5 of the NRXN1 gene in 5 SCZ cases. Allele-specific Hi-C maps revealed ectopic, allele-specific loops forming between a potential novel cryptic promoter and non-coding cis regulatory elements upon deletions in the 5' region of NRXN1. We also observed recurrent intragenic deletions of ABCB11, a gene associated with anti-psychotic response, in 5 treatment-resistant SCZ cases. Taken together our results indicate an important role of sCNVs to SCZ risk and treatment-responsiveness.</p>\", 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.28.474357', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'A Musculoskeletal Model of the Hand and Wrist Capable of Simulating Functional Tasks', 'authors': \"[{'name': 'Daniel Clinton McFarland'}, {'name': 'Benjamin I Binder-Markey'}, {'name': 'Jennifer A Nichols'}, {'name': 'Sarah J Wohlman'}, {'name': 'Marije de Bruin'}, {'name': 'Wendy M Murray'}]\", 'abstract': \"<p>Objective: The purpose of this work was to develop an open-source musculoskeletal model of the hand and wrist and to evaluate its performance during simulations of functional tasks. Methods: The musculoskeletal model was developed by adapting and expanding upon existing musculoskeletal models. An optimal control theory framework that combines forward-dynamics simulations with a simulated-annealing optimization was used to simulate maximum grip and pinch force. Active and passive hand opening were simulated to evaluate coordinated kinematic hand movements. Results: The model's maximum grip force production matched experimental measures of grip force, force distribution amongst the digits, and displayed sensitivity to wrist flexion. Simulated lateral pinch strength fell within variability of in vivo palmar pinch strength data. Additionally, predicted activation for 7 of 8 muscles fell within variability of EMG data during palmar pinch. The active and passive hand opening simulations predicted reasonable activations and demonstrated passive motion mimicking tenodesis, respectively. Conclusion: This work advances simulation capabilities of hand and wrist models and provides a foundation for future work to build upon. Significance: This is the first open-source musculoskeletal model of the hand and wrist to be implemented during both functional kinetic and kinematic tasks. We provide a novel simulation framework to predict maximal grip and pinch force which can be used to evaluate how potential surgical and rehabilitation interventions influence these functional outcomes while requiring minimal experimental data.</p>\", 'posted': '2021-12-28'}, {'doi': '10.1101/2021.12.28.474366', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The mitochondrial calcium uniporter regulates mitochondrial mass and dendritic localization in distinct hippocampal circuits', 'authors': \"[{'name': 'Katy E Pannoni'}, {'name': 'Daniela Gil'}, {'name': 'Logan Campbell'}, {'name': 'Shannon Farris'}]\", 'abstract': '<p>CA2 is an understudied subregion of the hippocampus that is critical for social memory. Previous studies identified multiple components of the mitochondrial calcium uniporter (MCU) complex as selectively enriched in CA2, however the functional significance of this enrichment remains unclear. The MCU complex regulates calcium entry into mitochondria, which in turn regulates mitochondrial transport and localization to active synapses. We found that MCU is strikingly enriched in CA2 distal apical dendrites, precisely where CA2 neurons receive entorhinal cortical input carrying social information. Further, MCU-enriched mitochondria in CA2 distal dendrites are larger compared to mitochondria in CA2 proximal apical dendrites and neighboring CA1 apical dendrites. Genetic knockdown of MCU in CA2 resulted in smaller mitochondria in CA2 distal dendrites, indicating that MCU expression plays a role in regulating mitochondrial mass in CA2. MCU overexpression in neighboring CA1 led to larger mitochondria preferentially in proximal dendrites compared to distal dendrites and GFP controls. Our findings demonstrate that mitochondria are molecularly and structurally diverse across hippocampal cell types and circuits, and that MCU expression cell-autonomously regulates mitochondrial mass, but layer-specific dendritic localization depends on cell type. Our data support the idea that CA2 mitochondria are functionally distinct from CA1 mitochondria, which may confer unique synaptic and circuit properties underlying CA2 function in social memory.</p>', 'posted': '2021-12-28'}, {'doi': '10.1101/2021.12.29.473670', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Reconstitution of the S. aureus agr quorum sensing pathway reveals a direct role for the integral membrane protease MroQ in pheromone biosynthesis', 'authors': \"[{'name': 'Aishan Zhao'}, {'name': 'Steven Parker Bodine'}, {'name': 'Qian Xie'}, {'name': 'Boyuan Wang'}, {'name': 'Geeta Ram'}, {'name': 'Richard P Novick'}, {'name': 'Tom W Muir'}]\", 'abstract': '<p>In Staphylococcus aureus, virulence is under the control of a quorum sensing (QS) circuit encoded in the accessory gene regulator (agr) genomic locus. Key to this pathogenic behavior is the production and signaling activity of a secreted pheromone, the autoinducing peptide (AIP), generated following the ribosomal synthesis and post-translational modification of a precursor polypeptide, AgrD, through two discrete cleavage steps. The integral membrane protease AgrB is known to catalyze the first processing event, generating the AIP biosynthetic intermediate, AgrD (1-32) thiolactone. However, the identity of the second protease in this biosynthetic pathway, which removes an N-terminal leader sequence, has remained ambiguous. Here, we show that MroQ, an integral membrane protease recently implicated in the agr response, is directly involved in AIP production. Genetic complementation and biochemical experiments reveal that MroQ proteolytic activity is required for AIP biosynthesis in agr specifiy groups -I and -II, but not group-III. Notably, as part of this effort, the biosynthesis and AIP-sensing arms of the QS circuit were reconstituted together in vitro. Our experiments also reveal the molecular features guiding MroQ cleavage activity, a critical factor in defining agr specificity group identity. Collectively, our study adds to the molecular understanding of the agr response and Staphylococcus aureus virulence.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.21268538', 'tweets': 15, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Protective effect of BNT162b2 vaccination on aerobic capacity following mild to moderate SARS-CoV-2 infection: a cross sectional study, Israel, March-December 2021', 'authors': \"[{'name': 'Yair Blumberg'}, {'name': 'Michael Edelstein'}, {'name': 'Kamal Abu Jabal'}, {'name': 'Ron Golan'}, {'name': 'Yuval Perets'}, {'name': 'Musa Saad'}, {'name': 'Tatyana Levinas'}, {'name': 'Saleem Dabbah'}, {'name': 'Zeev Israeli'}, {'name': 'Salah Yacoubi'}, {'name': 'Alaa Abu Raya'}, {'name': 'Anat Amital'}, {'name': 'Majdi Halabi'}]\", 'abstract': \"<p>There is increasing evidence that patients who were infected with SARS-CoV-2 may experience adverse health outcomes months after the acute infection has resolved including reduction in aerobic capacity and fatigue. In this study, we compared aerobic capacity and exercise performance of 28 unvaccinated participants to 15 vaccinated ones who performed a symptom limited cardio-pulmonary exercise test (CPET) after acute COVID-19. We identified a significant difference in aerobic capacity between vaccinated and unvaccinated individuals, with a lower V'O2 peak percentage of predicted in the unvaccinated group. In addition, the unvaccinated group had a reduction in the peak-exercise heart rate and lower ventilation values. Our results suggest objective limitations to exercise capacity in the months following acute COVID19 illness, mitigated by vaccination</p>\", 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.27.21268442', 'tweets': 15, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Brain atrophy in prodromal synucleinopathy is shaped by structural connectivity and gene expression', 'authors': \"[{'name': 'Shady Rahayel'}, {'name': 'Christina Tremblay'}, {'name': 'Andrew Vo'}, {'name': 'Ying-Qiu Zheng'}, {'name': 'Stéphane Lehéricy'}, {'name': 'Isabelle Arnulf'}, {'name': 'Marie Vidailhet'}, {'name': 'Jean-Christophe Corvol'}, {'name': 'Jean-François Gagnon'}, {'name': 'Ronald B Postuma'}, {'name': 'Jacques Montplaisir'}, {'name': 'Simon Lewis'}, {'name': 'Elie Matar'}, {'name': 'Kaylena Ehgoetz Martens'}, {'name': 'Per Borghammer'}, {'name': 'Karoline Knudsen'}, {'name': 'Allan Hansen'}, {'name': 'Oury Monchi'}, {'name': 'Bratislav Misic'}, {'name': 'Alain Dagher'}, {'name': ' '}]\", 'abstract': \"<p>Isolated REM sleep behaviour disorder (iRBD) is a synucleinopathy characterized by abnormal behaviours and vocalizations during REM sleep. Most iRBD patients develop dementia with Lewy bodies, Parkinson's disease, or multiple system atrophy over time. Patients with iRBD exhibit brain atrophy patterns that are reminiscent of those observed in overt synucleinopathies. However, the mechanisms linking brain atrophy to the underlying alpha-synuclein pathophysiology are poorly understood. Our objective was to investigate how the prion-like and regional vulnerability hypotheses of alpha-synuclein might explain brain atrophy in iRBD.\\n\\nUsing a multicentric cohort of 182 polysomnography-confirmed iRBD patients who underwent T1-weighted MRI, we performed vertex-based cortical surface and deformation-based morphometry analyses to quantify brain atrophy in patients (67.8 years, 84% men) and 261 healthy controls (66.2 years, 75%) and investigated the morphological correlates of motor and cognitive functioning in iRBD. Next, we applied the agent-based Susceptible-Infected-Removed model (i.e., a computational model that simulates in silico the spread of pathologic alpha-synuclein based on structural connectivity and gene expression) and tested if it recreated atrophy in iRBD by statistically comparing simulated regional brain atrophy to the atrophy observed in patients. The impact of SNCA and GBA gene expression and brain connectivity was then evaluated by comparing the model fit to the one obtained in null models where either gene expression or connectivity was randomized.\\n\\nThe results showed that iRBD patients present with cortical thinning and tissue deformation, which correlated with motor and cognitive functioning. Next, we found that the atrophy simulated based on brain connectivity and gene expression recreated cortical thinning (r=0.51, p=0.0007) and tissue deformation (r=0.52, p=0.0005) in patients, and that the connectome's architecture along with SNCA and GBA gene expression contributed to shaping atrophy in iRBD. We further demonstrated that the full agent-based model performed better than network measures or gene expression alone in recreating the atrophy pattern in iRBD.\\n\\nIn summary, atrophy in iRBD is extensive, correlates with motor and cognitive functioning, and can be recreated using the dynamics of agent-based modelling, structural connectivity, and gene expression. These findings support the concepts that both prion-like spread and regional susceptibility account for the atrophy observed in prodromal synucleinopathies. Therefore, the agent-based Susceptible-Infected-Removed model may be a useful tool for testing hypotheses underlying neurodegenerative diseases and new therapies aimed at slowing or stopping the spread of alpha-synuclein pathology.</p>\", 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474477', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Optogenetic inhibition of the dorsal hippocampus CA3 region during early-stage cocaine-memory reconsolidation disrupts subsequent context-induced cocaine seeking in rats', 'authors': \"[{'name': 'Shuyi Qi'}, {'name': 'Shi Min Tan'}, {'name': 'Rong Wang'}, {'name': 'Jessica A. Higginbotham'}, {'name': 'Jobe L. Ritchie'}, {'name': 'Christopher K. Ibarra'}, {'name': 'Amy A. Arguello'}, {'name': 'Robert J. Christian'}, {'name': 'Rita A. Fuchs'}]\", 'abstract': '<p>The dorsal hippocampus (DH) is key to the long-term maintenance of cocaine memories following retrieval-induced memory destabilization; even though, it is not the site of protein synthesis-dependent memory reconsolidation. Here, we took advantage of the temporal and spatial specificity of an optogenetic manipulation to examine the role of the cornu ammonis 3 subregion of the DH (dCA3) in early-stage cocaine-memory reconsolidation. Male Sprague-Dawley rats expressing eNpHR3.0 in the DH were trained to self-administer cocaine in a distinct context and underwent extinction training in a different context. Rats then received a 15-min memory-reactivation session, to destabilize cocaine memories and trigger reconsolidation, or remained in their home cages (no-reactivation controls). Optogenetic inhibition of the dCA3 for 1 h immediately, but not 1 h, after memory reactivation resulted in cocaine-memory impairment as indicated by reduction in drug-seeking behavior selectively in the cocaine-paired context 3 d later, at test, relative to responding in no-inhibition, no-reactivation, and no-eNpHR3.0 controls. Cocaine-memory impairment was associated with reduced c-Fos expression, an index of neuronal activation, in the dCA3 stratum lucidum (SL) and stratum pyramidale (SP) at test. Based on these observations and extant literature, we postulate that recurrent circuits in the SP are activated during early-stage memory reconsolidation to maintain labile cocaine memories prior to protein synthesis-dependent restabilization in another brain region, such as the basolateral amygdala. Furthermore, SL and SP interneurons may enhance memory reconsolidation by limiting synaptic noise in the SP and also contribute to recall as elements of the updated cocaine engram or retrieval links.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.28.474379', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'FastCAT accelerates absolute quantification of proteins by using multiple short non-purified chimeric standards', 'authors': \"[{'name': 'Ignacy Rzagalinski'}, {'name': 'Aliona Bogdanova'}, {'name': 'Bharath Kumar Raghuraman'}, {'name': 'Eric R. Geertsma'}, {'name': 'Lena Hersemann'}, {'name': 'Tjalf Ziemssen'}, {'name': 'Andrej Shevchenko'}]\", 'abstract': '<p>Absolute (molar) quantification of proteins provides the analytical rationale for system-level modelling of diverse molecular mechanisms. FastCAT method employs multiple short (&lt;50 kDa) stable-isotope labeled chimeric proteins (CPs) composed of concatenated quantotypic (Q-) peptides representing the quantified proteins. Each CP also comprises scrambled sequences of reference (R-) peptides that relate its abundance to a single protein standard (BSA). FastCAT not only alleviates the need in purifying CP or using SDS-PAGE, but also improves the accuracy, precision and dynamic range of the absolute quantifications by grouping Q-peptides according to the expected abundance of target proteins. We benchmarked FastCAT against the reference method of MS Western and tested it in the direct molar quantifications of neurological markers in human cerebrospinal fluid at the low ng/mL level.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.31.21268575', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'COVID-19 severity prediction based on patient risk factors and number of vaccines received', 'authors': \"[{'name': 'Ariel Israel'}, {'name': 'Alejandro A. Schäffer'}, {'name': 'Eugene Merzon'}, {'name': 'Ilan Green'}, {'name': 'Eli Magen'}, {'name': 'Avivit Golan-Cohen'}, {'name': 'Shlomo Vinker'}, {'name': 'Eytan Ruppin'}]\", 'abstract': '<p>Background\\nVaccines are highly effective in preventing severe disease and death from COVID-19, and new medications that can reduce severity of disease have been approved. However, many countries are facing limited supply of vaccine doses and medications. A model estimating the probabilities for hospitalization and mortality according to individual risk factors and vaccine doses received could help prioritize vaccination and yet scarce medications to maximize lives saved and reduce the burden on hospitalization facilities.\\nMethods\\nElectronic health records from 101,034 individuals infected with SARS-CoV-2, since the beginning of the pandemic and until November 30, 2021 were extracted from a national healthcare organization in Israel. Logistic regression models were built to estimate the risk for subsequent hospitalization and death based on the number of BNT162b2 mRNA vaccine doses received and few major risk factors (age, sex, body mass index, hemoglobin A1C, kidney function, and presence of hypertension, pulmonary disease or malignancy).\\nResults\\nThe models built predict the outcome of newly infected individuals with remarkable accuracy: area under the curve was 0.889 for predicting hospitalization, and 0.967 for predicting mortality. Even when a breakthrough infection occurs, having received three vaccination doses significantly reduces the risk of hospitalization by 66% (OR=0.336) and of death by 78% (OR=0.220).\\nConclusions\\nThe models enable rapid identification of individuals at high risk for hospitalization and death when infected. These patients can be prioritized to receive booster vaccination and the yet scarce medications. A calculator based on these models is made publicly available on http://covidest.web.app</p>', 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.26.474085', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'SARS-CoV-2 Omicron-B.1.1.529 Variant leads to less severe disease than Pango B and Delta variants strains in a mouse model of severe COVID-19', 'authors': \"[{'name': 'Eleanor G Bentley'}, {'name': 'Adam Kirby'}, {'name': 'Parul Sharma'}, {'name': 'Anja Kipar'}, {'name': 'Daniele F Mega'}, {'name': 'Chloe Bramwell'}, {'name': 'Rebekah Penrice-Randal'}, {'name': 'Tessa Prince'}, {'name': 'Jonathan C Brown'}, {'name': 'Jie Zhou'}, {'name': 'Gavin R Screaton'}, {'name': 'Wendy S Barclay'}, {'name': 'Andrew Owen'}, {'name': 'Julian Alexander Hiscox'}, {'name': 'James P Stewart'}]\", 'abstract': '<p>COVID-19 is a spectrum of clinical symptoms in humans caused by infection with SARS-CoV-2. The B.1.1.529 Omicron variant is rapidly emerging and has been designated a Variant of Concern (VOC). The variant is highly transmissible and partially or fully evades a spectrum of neutralising antibodies due to a high number of substitutions in the spike glycoprotein. A major question is the relative severity of disease caused by the Omicron variant compared with previous and currently circulating variants of SARS-CoV-2. To address this, a mouse model of infection that recapitulates severe disease in humans, K18-hACE2 mice, were infected with either a Pango B, Delta or Omicron variant of SARS-CoV-2 and their relative pathogenesis compared. In contrast to mice infected with Pango B and Delta variant viruses, those infected with the Omicron variant had less severe clinical signs (weight loss), showed recovery and had a lower virus load in both the lower and upper respiratory tract. This is also reflected by less extensive inflammatory processes in the lungs. Although T cell epitopes may be conserved, the antigenic diversity of Omicron from previous variants would suggest that a change in vaccine may be required to mitigate against the higher transmissibility and global disease burden. However, the lead time to develop such a response may be too late to mitigate the spread and effects of Omicron. These animal model data suggest the clinical consequences of infection with the Omicron variant may be less severe but the higher transmissibility could still place huge burden upon healthcare systems even if a lower proportion of infected patients are hospitalised.</p>', 'posted': '2021-12-28'}, {'doi': '10.1101/2021.12.24.474110', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Reduced infectivity but increased immune escape of the new SARS-CoV-2 variant of concern Omicron', 'authors': \"[{'name': 'Jie Hu'}, {'name': 'Pai Peng'}, {'name': 'Kang Wu'}, {'name': 'Quan-xin Long'}, {'name': 'Juan Chen'}, {'name': 'Kai Wang'}, {'name': 'Ni Tang'}, {'name': 'Ai-long Huang'}]\", 'abstract': '<p>A new detected SARS-CoV-2 variant Omicron (B.1.1.529) had reported from more than 80 countries. In the past few weeks, a new wave of infection driven by Omicron is in progress. Omicron Spike (S) protein pseudotyped virus was used to determine the effect of S mutations on its capacity of infectivity and immune evasion. Our results showed the lower entry efficiency and less cleavage ability of Omicron than D614G variant. Pseudotype-based neutralizing assay was performed to analyze neutralizing antibodies elicited by previously infection or the RBD-based protein subunit vaccine ZF2001 against the Omicron variant. Sera sampled at around one month after symptom onset from 12 convalescents who were previously infected by SARS-CoV-2 original strain shows a more than 20-fold decrease of neutralizing activity against Omicron variant, when compared to D614G variant. Among 12 individuals vaccinated by RBD subunit vaccine, 58.3% (7/12) sera sampled at 15-60 days after 3rd-dose vaccination did not neutralize Omicron. Geometric mean titers (GMTs, 50% inhibitory dose [ID50]) of these sera against Omicron were 9.4-fold lower than against D614G. These results suggested a higher risk of Omicron breakthrough infections and reduced efficiency of the protective immunity elicited by existing vaccines. There are important implications about the modification and optimization of the current epidemic prevention and control including vaccine strategies and therapeutic antibodies against Omicron variant.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.29.474474', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'A NEUROGENIC GENE EXPRESSION SIGNATURE SUPPORTS HUMAN THERMOGENIC ADIPOSE TISSUE DEVELOPMENT IN VIVO.', 'authors': \"[{'name': 'Javier Solivan-Rivera'}, {'name': 'Zinger Yang Loureiro'}, {'name': 'Tiffany DeSouza'}, {'name': 'Anand Desai'}, {'name': 'Qin Yang'}, {'name': 'Raziel Rojas-Rodriguez'}, {'name': 'Pantos Skritakis'}, {'name': 'Shannon Joyce'}, {'name': 'Denise Zhong'}, {'name': 'Tammy Nguyen'}, {'name': 'Silvia Corvera'}]\", 'abstract': '<p>Human beige/brite thermogenic adipose tissue exerts beneficial metabolic effects and may be harnessed to improve metabolic health. To uncover mechanisms by which thermogenic adipose tissue is generated and maintained we developed a species-hybrid model in which human mesenchymal progenitor cells are induced in vitro to differentiate into white or thermogenic adipocytes and are then implanted into immuno-compromised mice. Upon implantation, thermogenic adipocytes form a more densely vascularized and innervated adipose tissue compared to non-thermogenic adipocytes. Mouse endothelial and stem/progenitor cells recruited by implanted human thermogenic adipocytes are also qualitatively different, with differentially expressed genes mapping predominantly to circadian rhythm pathways. We trace the formation of this enhanced neurovascular architecture to higher expression of a distinct set of genes directly associated with neurogenesis (THBS4, TNC, NTRK3 and SPARCL1), and to lower expression of genes associated with neurotransmitter degradation (MAOA, ACHE) by adipocytes in the developed tissue. Further analysis reveals that MAOA is abundant in human adipocytes but absent in mouse adipocytes, revealing species-specific mechanisms of neurotransmitter tone regulation. In summary, our work discovers specific neurogenic genes associated with development and maintenance of human thermogenic adipose tissue,  reveals species-specific mechanisms of control of neurotransmitter tone, and suggests that targeting adipocyte MAOA may be a strategy for enhancing thermogenic adipose tissue activity in humans.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.27.474262', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Uncovering a neural circuit controlling adult quiescent neural stem cell activation in the subventricular zone', 'authors': \"[{'name': 'Moawiah Naffaa'}, {'name': 'Rehan Khan'}, {'name': 'Chay T Kuo'}, {'name': 'Henry H Yin'}]\", 'abstract': '<p>The maintenance and differentiation of the adult neural stem cells (NSCs) in the subventricular zone (SVZ) are controlled by cell-intrinsic molecular pathways that interact with extrinsic signaling cues. How neurogenesis in the SVZ is regulated by neural circuit activity remains poorly understood. Here we identified a novel neural circuit that regulates the state of lateral ventricular wall (LV) NSCs. Our results demonstrate that direct glutamatergic inputs from the frontal cortex, as well as local inhibitory interneurons, control the activity of subependymal cholinergic neurons. In vivo optogenetic and chemogenetic stimulation of defined neuronal populations within this circuit were sufficient to control LV NSC proliferation and SVZ neurogenesis. Moreover, acetylcholine (ACh), which activates M1 muscarinic ACh receptors, triggers the activation of quiescent NSCs. These findings shed light on neural activity-dependent regulation of postnatal and adult LV NSCs activation and SVZ neurogenesis.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.24.474138', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Homologous or Heterologous Booster of Inactivated Vaccine Reduces SARS-CoV-2 Omicron Variant Escape from Neutralizing Antibodies', 'authors': \"[{'name': 'Xun Wang'}, {'name': 'Xiaoyu Zhao'}, {'name': 'Jieyu Song'}, {'name': 'Jing Wu'}, {'name': 'Yuqi Zhu'}, {'name': 'Minghui Li'}, {'name': 'Yuchen Cui'}, {'name': 'Yanjia Chen'}, {'name': 'Lulu Yang'}, {'name': 'Jun Liu'}, {'name': 'Huanzhang Zhu'}, {'name': 'Shibo Jiang'}, {'name': 'Pengfei Wang'}]\", 'abstract': '<p>The massive and rapid transmission of SARS-CoV-2 has led to the emergence of several viral variants of concern (VOCs), with the most recent one, B.1.1.529 (Omicron), which accumulated a large number of spike mutations, raising the specter that this newly identified variant may escape from the currently available vaccines and therapeutic antibodies. Using VSV-based pseudovirus, we found that Omicron variant is markedly resistant to neutralization of sera form convalescents or individuals vaccinated by two doses of inactivated whole-virion vaccines (BBIBP-CorV). However, a homologous inactivated vaccine booster or a heterologous booster with protein subunit vaccine (ZF2001) significantly increased neutralization titers to both WT and Omicron variant. Moreover, at day 14 post the third dose, neutralizing antibody titer reduction for Omicron was less than that for convalescents or individuals who had only two doses of the vaccine, indicating that a homologous or heterologous booster can reduce the Omicron escape from neutralizing. In addition, we tested a panel of 17 SARS-CoV-2 monoclonal antibodies (mAbs). Omicron resists 7 of 8 authorized/approved mAbs, as well as most of the other mAbs targeting distinct epitopes on RBD and NTD. Taken together, our results suggest the urgency to push forward the booster vaccination to combat the emerging SARS-CoV-2 variants.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.30.474557', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Properties and proximity proteomics of synaptopodin provide insight into the molecular organization of the spine apparatus of dendritic spines', 'authors': \"[{'name': 'Hanieh Falahati'}, {'name': 'Yumei Wu'}, {'name': 'Vanessa Feuerer'}, {'name': 'Pietro De Camilli'}]\", 'abstract': '<p>The spine apparatus is a specialization of the neuronal ER in dendritic spines consisting of stacks of interconnected cisterns separated by a dense matrix. Synaptopodin, a specific actin binding protein of the spine apparatus, is essential for its formation, but the underlying mechanisms remain unknown. We show that synaptopodin, when expressed in fibroblasts, forms actin-rich structures with connections to the ER, and that an ER-tethered synaptopodin assembles into liquid condensates. We also identified protein neighbors of synaptopodin in spines by in vivo proximity biotinylation. We validated a small subset of such proteins and showed that they co-assemble with synaptopodin in living cells.  One of them is Pdlim7, an actin binding protein not previously identified in spines, and we show its precise colocalization with synaptopodin. We suggest that the matrix of the spine apparatus has the property of a liquid protein condensate generated by a multiplicity of low affinity interactions.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.27.474025', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Integrated cytometry with machine learning applied to high-content imaging of human kidney tissue for in-situ cell classification and neighborhood analysis', 'authors': \"[{'name': 'Seth Winfree'}, {'name': 'Andrew T McNutt'}, {'name': 'Suraj Khochare'}, {'name': 'Tyler J Borgard'}, {'name': 'Daria Barwinska'}, {'name': 'Angela R Sabo'}, {'name': 'Michael J Ferkowicz'}, {'name': 'James C Williams'}, {'name': 'James E Lingeman'}, {'name': 'Connor J Gulbronson'}, {'name': 'Katherine J Kelly'}, {'name': 'Timothy A Sutton'}, {'name': 'Pierre C Dagher'}, {'name': 'Michael T Eadon'}, {'name': 'Kenneth W Dunn'}, {'name': 'Tarek M El-Achkar'}]\", 'abstract': \"<p>The human kidney is a complex organ with various cell types that are intricately organized to perform key physiological functions and maintain homeostasis. New imaging modalities such as mesoscale and highly multiplexed fluorescence microscopy are increasingly applied to human kidney tissue to create single cell resolution datasets that are both spatially large and multi-dimensional. These single cell resolution high-content imaging datasets have a great potential to uncover the complex spatial organization and cellular make-up of the human kidney. Tissue cytometry is a novel approach used for quantitative analysis of imaging data, but the scale and complexity of such datasets pose unique challenges for processing and analysis. We have developed the Volumetric Tissue Exploration and Analysis (VTEA) software, a unique tool that integrates image processing, segmentation and interactive cytometry analysis into a single framework on desktop computers. Supported by an extensible and open-source framework, VTEA's integrated pipeline now includes enhanced analytical tools, such as machine learning, data visualization, and neighborhood analyses for hyperdimensional large-scale imaging datasets. These novel capabilities enable the analysis of mesoscale two and three-dimensional multiplexed human kidney imaging datasets (such as CODEX and 3D confocal multiplexed fluorescence imaging). We demonstrate the utility of this approach in identifying cell subtypes in the kidney based on labels, spatial association and their microenvironment or neighborhood membership. VTEA provides integrated and intuitive approach to decipher the cellular and spatial complexity of the human kidney and complement other transcriptomics and epigenetic efforts to define the landscape of kidney cell types.</p>\", 'posted': '2021-12-28'}, {'doi': '10.1101/2021.12.29.474488', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Structural insights into the ion selectivity of the MgtE channel for Mg2+ over Ca2+', 'authors': \"[{'name': 'Xinyu Teng'}, {'name': 'Danqi Sheng'}, {'name': 'Jin Wang'}, {'name': 'Ye Yu'}, {'name': 'Motoyuki Hattori'}]\", 'abstract': '<p>MgtE is a Mg2+-selective ion channel whose orthologs are widely distributed from prokaryotes to eukaryotes, including humans, and play an important role in the maintenance of cellular Mg2+ homeostasis. Previous functional analyses showed that MgtE transports divalent cations with high selectivity for Mg2+ over Ca2+. Whereas the high-resolution structure determination of the MgtE transmembrane (TM) domain in complex with Mg2+ ions revealed a Mg2+ recognition mechanism of MgtE, the previous Ca2+-bound structure of the MgtE TM domain was determined only at moderate resolution (3.2 angstrom resolution), which was insufficient to visualize the water molecules coordinated to Ca2+ ions. Thus, the structural basis of the ion selectivity of MgtE for Mg2+ over Ca2+ has remained unclear. Here, we showed that the metal-binding site of the MgtE TM domain binds to Mg2+ ~500-fold more strongly than Ca2+. We then determined the crystal structure of the MgtE TM domain in complex with Ca2+ ions at a higher resolution (2.5 angstrom resolution), allowing us to reveal hexahydrated Ca2+, which is similarly observed in the previously determined Mg2+-bound structure but with extended metal-oxygen bond lengths. Our structural, biochemical, and computational analyses provide mechanistic insights into the ion selectivity of MgtE for Mg2+ over Ca2+.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.31.474679', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'PlasmidMaker: a Versatile, Automated, and High Throughput End-to-End Platform for Plasmid Construction', 'authors': \"[{'name': 'Behnam Enghiad'}, {'name': 'Pu Xue'}, {'name': 'Nilmani Singh'}, {'name': 'Aashutosh Girish Boob'}, {'name': 'Chengyou Shi'}, {'name': 'Vassily Andrew Petrov'}, {'name': 'Roy Liu'}, {'name': 'Siddhartha Suryanarayana Peri'}, {'name': 'Stephan Thomas Lane'}, {'name': 'Emily Danielle Gaither'}, {'name': 'Huimin Zhao'}]\", 'abstract': '<p>Plasmids are used extensively in basic and applied biology. However, design and construction of plasmids, specifically the ones carrying complex genetic information, remains one of the most time-consuming, labor-intensive, and rate-limiting steps in performing sophisticated biological experiments. Here, we report the development of a versatile, robust, automated end-to-end platform named PlasmidMaker that allows error-free construction of plasmids with virtually any sequences in a high-throughput manner. This platform consists of a most versatile DNA assembly method using Pyrococcus furiosus Argonaute (PfAgo)-based artificial restriction enzymes, a user-friendly frontend for plasmid design, and a backend that streamlines the workflow and integration with a robotic system. As a proof of concept, we used this platform to generate 101 plasmids from six different species ranging from 5 to 18 kb in size from up to 11 DNA fragments within 3 days. PlasmidMaker should greatly expand the potential of synthetic biology.</p>', 'posted': '2022-01-02'}, {'doi': '10.1101/2021.12.30.474558', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'BDNF-induced BDNF release mediates presynaptic LTP and is regulated by cannabinoids', 'authors': \"[{'name': 'Coralie Berthoux'}, {'name': 'Kaoutsar Nasrallah'}, {'name': 'Pablo E Castillo'}]\", 'abstract': '<p>Although brain-derived neurotrophic factor (BDNF) and its effector, Tropomyosin receptor kinase B (TrkB), are implicated in activity-dependent synaptic plasticity, the precise underlying mechanisms remain unclear. In the dentate gyrus, a hippocampal input region that expresses uniquely high levels of BDNF, repetitive activation of mossy cells (MCs) induces a presynaptic, BDNF/TrkB-dependent form of LTP at MC to granule cell (GC) synapses. Here, we report that activity-induced BDNF release from MC axons in mice elicits postsynaptic BDNF release in a TrkB- and calcium-dependent manner, and that BDNF-induced BDNF release is essential for LTP induction. Suppression of BDNF release by tonic and phasic activity of presynaptic type-1 cannabinoid receptors dampened LTP, revealing an unprecedented role of these receptors in controlling neuropeptide release. Lastly, BDNF-mediated MC-GC LTP can be elicited in vivo. Thus, BDNF-induced BDNF release emerges as a mechanism for activity-dependent LTP that may contribute to dentate gyrus-dependent learning, epilepsy, and neuropsychiatric disorders.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474445', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Nipah virus detection at bat roosts following spillover events in Bangladesh, 2012-2019', 'authors': \"[{'name': 'Clifton D McKee'}, {'name': 'Ausraful Islam'}, {'name': 'Mohammed Ziaur Rahman'}, {'name': 'Salah Uddin Khan'}, {'name': 'Mahmudur Rahman'}, {'name': 'Syed M Satter'}, {'name': 'Ariful Islam'}, {'name': 'Claude Kwe Yinda'}, {'name': 'Jonathan H Epstein'}, {'name': 'Peter Daszak'}, {'name': 'Vincent J Munster'}, {'name': 'Peter J Hudson'}, {'name': 'Raina K Plowright'}, {'name': 'Stephen P Luby'}, {'name': 'Emily S Gurley'}]\", 'abstract': '<p>Knowledge of the dynamics and genetic diversity of Nipah virus circulating in bats and at the human-animal interface is limited by current sampling efforts, which produce few detections of viral RNA. We report on a series of investigations at bat roosts identified near human Nipah cases in Bangladesh between 2012 and 2019. Pooled bat urine samples were collected from 23 roosts; seven roosts (30%) had at least one sample with Nipah RNA detected from the first visit. In subsequent visits to these seven roosts, RNA was detected in bat urine up to 52 days after the presumed exposure of the human case, although the probability of detection declined rapidly with time. These results suggest that rapidly deployed investigations of Nipah virus shedding from bat roosts near human cases could increase the success of viral sequencing compared to background surveillance and enhance our understanding of Nipah virus ecology and evolution.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474460', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Protein interaction analysis of Plasmodium falciparum circumsporozoite protein variants with human immunoproteins explains RTS,S vaccine efficacy in Ghana', 'authors': \"[{'name': 'Cheikh Cambel Dieng'}, {'name': 'Colby T Ford'}, {'name': 'Anita T Lerch'}, {'name': 'Jennifer Huynh'}, {'name': 'Kovidh Vegesna'}, {'name': 'Jun-tao Guo'}, {'name': 'Daniel A Janies'}, {'name': 'Linda E. Amoah'}, {'name': 'Yaw Afrane'}, {'name': 'Eugenia Lo'}]\", 'abstract': \"<p>The world's first malaria vaccine RTS,S provides only partial protection against Plasmodium falciparum infections. The explanation for such low efficacy is unclear. This study examined the associations of parasite genetic variations with binding affinity to human immunological proteins including human leukocyte antigen (HLA) and T cell receptors (TCR) involved in RTS,S-induced immune responses. Multiplicity of infections was determined by amplicon deep sequencing of merozoite surface protein 1 (PfMSP1). Genetic variations in the C-terminal of circumsporozoite protein (PfMSP1) gene were examined across 88 samples of P. falciparum collected from high and low transmission settings of Ghana. Binding interactions of PfMSP1 variants and HLA/TCR were analyzed using NetChop} and HADDOCK predictions. Anti-CSP IgG levels were measured by ELISA in a subset of 10 samples. High polyclonality was detected among P. falciparum infections. A total 27 CSP haplotypes were detected among samples. A significant correlation was detected between the CSP and MSP multiplicity of infection (MOI). No clear clustering of haplotypes was observed by geographic regions. The number of genetic differences in PfCSP between 3D7 and non-3D7 variants does not influence binding interactions to HLA/T cells nor anti-CSP IgG levels. Nevertheless, PfCSP peptide length significantly affects its molecular weight and binding affinity to the HLA. The presence of multiple non-3D7 strains among P. falciparum infections in Ghana impact the effectiveness of RTS,S. Longer PfCSP peptides may elicit a stronger immune response and should be considered in future version RTS,S. The molecular mechanisms of RTS,S cell-mediated immune responses related to longer CSP peptides warrants further investigations.</p>\", 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474454', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Longitudinal analysis of invariant natural killer T cell activation reveals a cMAF-associated transcriptional state of NKT10 cells', 'authors': \"[{'name': 'Lydia Lynch'}, {'name': 'Harry Kane'}, {'name': 'Nelson M LaMarche'}, {'name': 'Áine Ní Scannail'}, {'name': 'Michael P. Brenner'}]\", 'abstract': '<p>Innate T cells, including CD1d-restricted invariant natural killer T (iNKT) cells, are characterized by their rapid activation in response to nonpeptide antigens, such as lipids. While the transcriptional profiles of naive, effector and memory adaptive T cells have been well studied, less is known about transcriptional regulation of different iNKT cell activation states. Here, using single cell RNA-sequencing, we performed longitudinal profiling of activated iNKT cells, generating a transcriptomic atlas of iNKT cell activation states. We found that transcriptional signatures of activation are highly conserved among heterogeneous iNKT cell populations, including NKT1, NKT2 and NKT17 subsets, and human iNKT cells. Strikingly, we found that regulatory iNKT cells, such as adipose iNKT cells, undergo blunted activation, and display constitutive enrichment of memory-like cMAF<sup>+</sup> and KLRG1<sup>+</sup> populations. Moreover, we identify a conserved cMAF-associated transcriptional network among NKT10 cells, providing novel insights into the biology of regulatory and antigen experienced iNKT cells.</p>', 'posted': '2021-12-31'}, {'doi': '10.1101/2021.12.23.21268244', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Effectiveness of casirivimab and imdevimab, and sotrovimab during Delta variant surge: a prospective cohort study and comparative effectiveness randomized trial', 'authors': \"[{'name': 'David T. Huang'}, {'name': 'Erin K. McCreary'}, {'name': 'J. Ryan Bariola'}, {'name': 'Tami E. Minnier'}, {'name': 'Richard J. Wadas'}, {'name': 'Judith A. Shovel'}, {'name': 'Debbie Albin'}, {'name': 'Oscar C. Marroquin'}, {'name': 'Kevin E. Kip'}, {'name': 'Kevin Collins'}, {'name': 'Mark Schmidhofer'}, {'name': 'Mary Kay Wisniewski'}, {'name': 'David A. Nace'}, {'name': 'Colleen Sullivan'}, {'name': 'Meredith Axe'}, {'name': 'Russell Meyers'}, {'name': 'Alexandra Weissman'}, {'name': 'William Garrard'}, {'name': 'Octavia M. Peck-Palmer'}, {'name': 'Alan Wells'}, {'name': 'Robert D. Bart'}, {'name': 'Anne Yang'}, {'name': 'Lindsay R. Berry'}, {'name': 'Scott Berry'}, {'name': 'Amy M. Crawford'}, {'name': 'Anna McGlothlin'}, {'name': 'Tina Khadem'}, {'name': 'Kelsey Linstrum'}, {'name': 'Stephanie K. Montgomery'}, {'name': 'Daniel Ricketts'}, {'name': 'Jason N. Kennedy'}, {'name': 'Caroline J. Pidro'}, {'name': 'Rachel L. Zapf'}, {'name': 'Paula L. Kip'}, {'name': 'Ghady Haidar'}, {'name': 'Graham M. Snyder'}, {'name': 'Bryan J. McVerry'}, {'name': 'Donald M. Yealy'}, {'name': 'Derek C. Angus'}, {'name': 'Christopher W. Seymour'}]\", 'abstract': '<sec><title>IMPORTANCE</title><p>The effectiveness of monoclonal antibodies (mAbs), casirivimab and imdevimab, and sotrovimab, for patients with mild to moderate COVID-19 from the Delta variant is unknown.</p></sec><sec><title>OBJECTIVE</title><p>To evaluate the effectiveness of mAbs for the Delta variant compared to no treatment, and the comparative effectiveness between mAbs.</p></sec><sec><title>DESIGN, SETTING, AND PARTICIPANTS</title><p>Two parallel studies among patients who met Emergency Use Authorization criteria for mAbs from July 14, 2021 to September 29, 2021: i.) prospective observational cohort study comparing mAb treatment to no mAb treatment and, ii.) Bayesian adaptive randomized trial comparing the effectiveness of casirivimab-imdevimab versus sotrovimab. In the observational study, we compared eligible patients who received mAb at an outpatient infusion center at UPMC, to nontreated patients with a positive SARS-CoV-2 test. In the comparative effectiveness trial, we randomly allocated casirivimab-imdevimab or sotrovimab to patients presenting to infusion centers and emergency departments, per system therapeutic interchange policy.</p></sec><sec><title>EXPOSURE</title><p>Intravenous mAb per their EUA criteria.</p></sec><sec><title>MAIN OUTCOMES AND MEASURES</title><p>For the observational study, risk ratio estimates for hospitalization or death by 28 days were compared between mAb treatment to no mAb treatment using propensity matched models. For the comparative effectiveness trial, the primary outcome was hospital-free days (days alive and free of hospital) within 28 days, where patients who died were assigned -1 day) in a Bayesian cumulative logistic model, adjusted for treatment location, age, sex, and time. Inferiority was defined as a 99% posterior probability of an odds ratio &lt;1. Equivalence was defined as a 95% posterior probability that the odds ratio is within a given bound.</p></sec><sec><title>RESULTS</title><p>Among 3,558 patients receiving mAb, the mean age was 54 (SD 18 years), 1,511 (43%) were treated in an infusion center, and 450 (13%) were hospitalized or died by day 28. In propensity matched models, mAb treatment was associated with reduced risk of hospitalization or death compared to no treatment (risk ratio (RR)=0.40, 95% CI: 0.28–0.57). Both casirivimab and imdevimab (RR=0.31, 95% CI: 0.20–0.50), and sotrovimab (RR=0.60, 95% CI: 0.37–1.00) reduced hospitalization or death compared to no mAb treatment. Among patients allocated randomly to casirivimab and imdevimab (n=2,454) or sotrovimab (n=1,104), the median hospital-free days were 28 (IQR 28–28) for both groups, 28-day mortality was 0.5% (n=12) and 0.6% (n=7), and hospitalization by day 28 was 12% (n=291) and 12% (n=140), respectively. Compared to casirivimab and imdevimab, the median adjusted odds ratio for hospital-free days was 0.88 (95% credible interval, 0.70–1.11) for sotrovimab. This odds ratio yielded 86% probability of inferiority of sotrovimab versus casirivimab and imdevimab, and 79% probability of equivalence.</p></sec><sec><title>CONCLUSIONS AND RELEVANCE</title><p>In non-hospitalized patients with mild to moderate COVID-19 due to the Delta variant, casirivimab and imdevimab and sotrovimab were both associated with a reduced risk of hospitalization or death. The comparative effectiveness of mAbs appeared similar, though prespecified criteria for statistical inferiority or equivalence were not met.</p></sec><sec><title>TRIAL REGISTRATION</title><p><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://ClinicalTrials.gov\">ClinicalTrials.gov</ext-link>: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04790786\">NCT04790786</ext-link></p></sec><sec><title>Key Points</title><sec><title>Question</title><p>In non-hospitalized patients with mild to moderate COVID-19 due to the Delta variant, what is the effectiveness of monoclonal antibodies (mAb) compared to no treatment, and what is the comparative effectiveness between mAb?</p></sec><sec><title>Findings</title><p>Among 3,069 patients, mAb treatment (casirivimab and imdevimab or sotrovimab) was associated with reduced risk of hospitalization or death by 28 days compared to no treatment (risk ratio=0.40, 95% CI: 0.28–0.57). In a Bayesian randomized comparative effectiveness trial of casirivimab and imdevimab vs. sotrovimab in 3,558 patients, the median hospital–free days were 28 days for both groups. Compared to casirivimab-imdevimab, the median adjusted odds ratio for hospital-free days was 0.88 (95% credible interval, 0.70–1.11) for sotrovimab, an 86% probability of inferiority of sotrovimab versus casirivimab and imdevimab, and 79% probability of equivalence.</p></sec><sec><title>Meaning</title><p>In non-hospitalized patients with mild to moderate COVID-19 due to the Delta variant, casirivimab and imdevimab and sotrovimab were associated with reduced risk of hospitalization or death compared to no treatment. The comparative effectiveness of mAbs appeared similar, though prespecified criteria for statistical inferiority or equivalence were not met.</p></sec></sec>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.27.473672', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Effector-mediated subversion of proteasome activator (PA)28αβ enhances lysosomal targeting of Legionella pneumophila within cytokine-activated macrophages', 'authors': \"[{'name': 'Tshegofatso Ngwaga'}, {'name': 'Deepika Chauhan'}, {'name': 'Abigail G Salberg'}, {'name': 'Stephanie R Shames'}]\", 'abstract': \"<p>Legionella pneumophila causes Legionnaires' Disease via replication within host macrophages using an arsenal of hundreds of translocated virulence factors termed effector proteins. Effectors are critical for intracellular replication but can also enhance pathogen clearance in mammalian hosts via effector-triggered immunity. The effector LegC4 confers a fitness disadvantage on L. pneumophila within mouse models of Legionnaires' Disease and uniquely potentiates the antimicrobial activity of macrophages activated with either tumor necrosis factor (TNF) or interferon (IFN)-γ. Here, we investigated the mechanism of LegC4 function. We found that LegC4 binds proteasome activator (PA)28α, a subunit of the PA28αβ (11S) proteasome regulator, and that the LegC4 restriction phenotype is abolished within PA28αβ-deficient macrophages. PA28αβ facilitates ubiquitin-independent proteasomal degradation of oxidant-damaged proteins. Impaired proteasome activity results in compensatory upregulation of lysosomal degradation pathways to relieve oxidative proteotoxic stress. We found that LegC4 impairs the resolution of oxidative proteotoxic stress and enhances phagolysosomal fusion with the Legionella-containing vacuole. PA28αβ has been traditionally associated with antigen presentation and adaptive immunity; however, our data support a model whereby suppression of PA28αβ by LegC4 impairs resolution of oxidative proteotoxic stress, which culminates in the lysosomal killing of L. pneumophila within activated macrophages. This work provides a solid foundation on which to evaluate induced proteasome regulators as mediators of cell-autonomous immunity.</p>\", 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.28.474397', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Comparative analysis of Lysine and Arginine biosynthesis pathway in Deinococcus genomes', 'authors': \"[{'name': 'Sankar Mahesh'}, {'name': 'Deepa Sethi'}, {'name': 'Richa Priyadarshini'}, {'name': 'Ragothaman M Yennamalli'}]\", 'abstract': '<p>The members of the Deinococcaceae family have the ability to survive extreme environmental conditions. Deinococcus species have a complex cell envelope composed of L-ornithine containing peptidoglycan. Anabolism of L-ornithine is intrinsically linked to L-lysine and L-arginine biosynthetic pathways. To understand these two pathways, we analyzed the L-lysine and L-arginine pathways using 23 Deinococcus genomes, including D. indicus. We used BLAST-P based ortholog identification using D. radiodurans genes as the query. We identified some BLAST-P hits that shared the same functional annotation. We analyzed three (class I aminotransferase, acetyl-lysine deacetylase, and acetyl glutamate/acetyl aminoadipate kinase) from L-lysine biosynthesis pathway and three (bifunctional ornithine acetyltransferase or N-acetyl glutamate synthase protein, nitric oxide synthase-like protein, and Acetyl-lysine deacetylase) from L-arginine biosynthesis pathway. Two proteins showed certain structural variations. Specifically, [LysW]-lysine hydrolase protein sequence and structure level changes indicated changes in oligomeric conformation, which could likely be a result of divergent evolution. And, bifunctional ornithine acetyltransferase or N-acetyl glutamate synthase had its active site pocket positions shifted at the structural level and we hypothesize that it may not perform at the optimal level. Thus, we were able to compare and contrast different Deinococcus species indicating some genes occurring because of divergent evolution.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.27.474252', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Dichotomous Feedback: A Signal Sequestration-based Feedback Mechanism for Biocontroller Design', 'authors': \"[{'name': 'Aivar Sootla'}, {'name': 'Nicolas Delalez'}, {'name': 'Emmanouil Alexis'}, {'name': 'Arthur Norman'}, {'name': 'Harrison Steel'}, {'name': 'George Wadhams'}, {'name': 'Antonis Papachristodoulou'}]\", 'abstract': '<p>We introduce a new design framework for implementing negative feedback regulation in Synthetic Biology, which we term \"dichotomous feedback\". Our approach is different from current methods, in that it sequesters existing fluxes in the process to be controlled, and in this way takes advantage of the process\\'s architecture to design the control law. This signal sequestration mechanism appears in many natural biological systems and can potentially be easier to realise than \\'molecular sequestration\\' and other comparison motifs that are nowadays common in biomolecular feedback control design. The loop is closed by linking the strength of signal sequestration to the process output. Our feedback regulation mechanism is motivated by two-component signalling systems, where we introduce a second response regulator competing with the natural response regulator thus sequestering kinase activity. Here, dichotomous feedback is established by increasing the concentration of the second response regulator as the level of the output of the natural process increases. Extensive analysis demonstrates how this type of feedback shapes the signal response, attenuates intrinsic noise while increasing robustness and reducing crosstalk.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.29.21268505', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Head-to-head comparison of nasal and nasopharyngeal sampling using SARS-CoV-2 rapid antigen testing in Lesotho', 'authors': \"[{'name': 'Niklaus Daniel Labhardt'}, {'name': 'Lucia Gonzalez Fernandez'}, {'name': 'Bulemba Katende'}, {'name': 'Josephine Muhairwe'}, {'name': 'Moniek Bresser'}, {'name': 'Alain Amstutz'}, {'name': 'Tracy R Glass'}, {'name': 'Morten Ruhwald'}, {'name': 'Jilian A Sacks'}, {'name': 'Camille Escadafal'}, {'name': 'Mathabo Mareka'}, {'name': 'Mooko Sekhele Mookho'}, {'name': 'Margaretha Daniel de Vos'}, {'name': 'Klaus Reither'}]\", 'abstract': '<p>Objectives\\nTo assess the real-world diagnostic performance of nasal and nasopharyngeal swabs for SD Biosensor STANDARD Q COVID-19 Antigen Rapid Diagnostic Test (Ag-RDT).\\nMethods\\nIndividuals ≥5 years with COVID-19 compatible symptoms or history of exposure to SARS-CoV-2 presenting at hospitals in Lesotho received two nasopharyngeal and one nasal swab. Ag-RDT from nasal and nasopharyngeal swabs were performed as point-of-care on site, the second nasopharyngeal swab used for polymerase chain reaction (PCR) as the reference standard. \\nResults \\nOut of 2198 participants enrolled, 2131 had a valid PCR result (61% female, median age 41 years, 8% children), 84.5% were symptomatic. Overall PCR positivity rate was 5.8%. The sensitivity for nasopharyngeal, nasal, and combined nasal and nasopharyngeal Ag-RDT result was 70.2% (95%CI: 61.3-78.0), 67.3% (57.3-76.3) and 74.4% (65.5-82.0), respectively. The respective specificity was 97.9% (97.1-98.4), 97.9% (97.2-98.5) and 97.5% (96.7-98.2). For both sampling modalities, sensitivity was higher in participants with symptom duration ≤ 3days versus ≤ 7days. Agreement between nasal and nasopharyngeal Ag-RDT was 99.4%.\\nConclusions\\nThe STANDARD Q Ag-RDT showed high specificity. Sensitivity was, however, below the WHO recommended minimum requirement of ≥ 80%. The high agreement between nasal and nasopharyngeal sampling suggests that for Ag-RDT nasal sampling is a good alternative to nasopharyngeal sampling.</p>', 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.29.474400', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Extreme value analysis of intracellular cargo transport by motor proteins', 'authors': \"[{'name': 'Takuma Naoi'}, {'name': 'Yuki Kagawa'}, {'name': 'Kimiko Nagino'}, {'name': 'Shinsuke Niwa'}, {'name': 'Kumiko Hayashi'}]\", 'abstract': '<p>In the long axon of a neuron, cargo transport between the cell body and terminal synaptic region are mainly supported by the motor proteins kinesin and dynein, which are nano-sized drivers. Synaptic materials packed as cargos are anterogradely transported to the synaptic region by kinesin, whereas materials accumulated at the axon terminals are returned to the cell body by dynein. Extreme value analysis, typically used for disaster prevention in our society, was applied to analyze the velocity of kinesin and dynein nanosized drivers to disclose their physical properties in living cells.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2022.01.01.474684', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Concerted expansion and contraction of immune receptor gene repertoires in plant genomes', 'authors': \"[{'name': 'Bruno Pok Man Ngou'}, {'name': 'Robert Heal'}, {'name': 'Michele Wyler'}, {'name': 'Marc W Schmid'}, {'name': 'Jonathan DG Jones'}]\", 'abstract': '<p>Recent reports suggest that cell-surface and intracellular immune receptors function synergistically to activate robust defence against pathogens, but whether or not they co-evolve is unclear. Here we determined the copy numbers of cell-surface and intracellular immune receptors in 208 species. Surprisingly, these receptor gene families contract and/or expand together in plant genomes, suggesting the mutual potentiation of immunity initiated by cell-surface and intracellular receptors is reflected in the concerted co-evolution of the size of their repertoires across plant species.</p>', 'posted': '2022-01-02'}, {'doi': '10.1101/2021.12.26.473589', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Biological Aging of CNS-Resident Cells Alters the Clinical Course and Immunopathology of Autoimmune Demyelinating Disease', 'authors': \"[{'name': 'Jeffrey R. Atkinson'}, {'name': 'Andrew D. Jerome'}, {'name': 'Andrew R. Sas'}, {'name': 'Ashley Munie'}, {'name': 'W. David Arnold'}, {'name': 'Benjamin M. Segal'}]\", 'abstract': '<p>Biological aging is the strongest factor associated with the clinical phenotype of multiple sclerosis (MS). Relapsing remitting MS (RRMS) typically presents in the third or fourth decade, while the mean age of presentation of progressive MS (pMS) is 45 years old. Here we show that experimental autoimmune encephalomyelitis (EAE), induced by the adoptive transfer of encephalitogenic CD4+ Th17 cells, is more severe, and less like to remit, in middle-aged compared with young adult mice. Donor T cells and neutrophils are more abundant, while B cells are relatively sparse, in central nervous system (CNS) infiltrates of the older mice. Experiments with reciprocal bone marrow chimeras demonstrate that radio-resistant, non-hematopoietic cells play a dominant role in shaping age-related features of the neuroinflammatory response, as well as the clinical course, during EAE. Reminiscent of pMS, EAE in middle-aged adoptive transfer recipients is characterized by widespread microglial activation. Microglia from older mice express a distinctive transcriptomic profile, suggestive of enhanced chemokine synthesis and antigen presentation. Collectively, our findings suggest that drugs that suppress microglial activation, and acquisition or expression of aging-associated properties, may be beneficial in the treatment of progressive forms of inflammatory demyelinating disease.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.29.474495', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'DENA: training an authentic neural network model using Nanopore sequencing data of Arabidopsis transcripts for detection and quantification of N6-methyladenosine on RNA', 'authors': \"[{'name': 'Xuan Li'}, {'name': 'Hang Qin'}, {'name': 'Liang Ou'}, {'name': 'Jian Gao'}, {'name': 'Longxian Chen'}, {'name': 'Pei Hao'}, {'name': 'Jiawei Wang'}]\", 'abstract': '<p>Models developed using Nanopore direct RNA sequencing data from in vitro synthetic RNA with all adenosine replaced by N6-methyladenosine (m6A), are likely distorted due to superimposed signals from saturated m6A residues. Here, we develop a neural network, DENA, for m6A quantification using the sequencing data of in vivo transcripts from Arabidopsis. DENA identifies 90% of miCLIP-detected m6A sites in Arabidopsis, and obtains modification rates in human consistent to those found by SCARLET, demonstrating its robustness across species. We sequence the transcriptome of two additional m6A-deficient Arabidopsis, mtb and fip37-4, using Nanopore and evaluate their single-nucleotide m6A profiles using DENA.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474482', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Rheology of crossbridge ensembles', 'authors': \"[{'name': 'Khoi D Nguyen'}, {'name': 'Madhusudhan Venkadesan'}]\", 'abstract': \"<p>How skeletal muscle responds to mechanical perturbations, its rheology, is important for animal movement control. The molecular machinery of myosin II-actin crossbridge cycling is a crucial part of muscle's rheological properties, and multiple models have been proposed for this mechanochemical process. But current understanding of the scale-connection from individual molecular motors to ensemble rheology is limited. Here we present computational and mathematical analyses of several different hypotheses of crossbridge dynamics, from 2-state to 5-state myosin II motor models, and show that an ensemble of actomyosin crossbridges exhibits surprisingly simple rheological behavior in all cases. The ensemble rheology is captured by a sum of at most three linear viscoelastic sub-processes, and as few as one for some crossbridge models. This simplification lends itself to computationally efficient phenomenological muscle models with experimentally measurable parameters, while still remaining grounded in crossbridge theory. However, the collapse of the ensemble behavior to few linear sub-processes identifies major limitations of crossbridge models that cannot be resolved by adding complexity to the crossbridge cycle and point to the roles of inter-crossbridge interactions and non-crossbridge elements.</p>\", 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.27.474302', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Perceptual uncertainty alternates top-down and bottom-up fronto-temporal network signaling during response inhibition', 'authors': \"[{'name': 'Kaho Tsumura'}, {'name': 'Reiko Shintaki'}, {'name': 'Masaki Takeda'}, {'name': 'Junichi Chikazoe'}, {'name': 'Kiyoshi Nakahara'}, {'name': 'Koji Jimura'}]\", 'abstract': '<p>Response inhibition is a primary executive control function that allows the withholding of inappropriate responses, and requires appropriate perception of the external environment to achieve a behavioral goal. It remains unclear, however, how response inhibition is achieved when goal-relevant information involves perceptual uncertainty. Twenty-six human participants of both sexes performed a go/no-go task where visually presented random-dot motion stimuli involved perceptual uncertainties. The right inferior frontal cortex (rIFC) was involved in response inhibition, and the middle temporal (MT) region showed greater activity when dot motions involved less uncertainty. A neocortical temporal region in the superior temporal sulcus (STS) specifically showed greater activity during response inhibition in more perceptually certain trials. In this STS region, activity was greater when response inhibition was successful than when it failed. Directional effective connectivity analysis revealed that in more coherent trials, the MT and STS regions showed enhanced connectivity to the rIFC, whereas in less coherent trials, the signal direction was reversed. These results suggest that a reversible fronto-temporal functional network guides response inhibition under perceptual uncertainty, and in this network, perceptual information in the MT is converted to control information in the rIFC via STS, enabling achievement of response inhibition.</p>', 'posted': '2021-12-28'}, {'doi': '10.1101/2021.12.28.21268413', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Increased persuadability and credulity in people with corpus callosum dysgenesis', 'authors': \"[{'name': 'Linda J Richards'}, {'name': 'Joseph Barnby'}, {'name': 'Ryan Dean'}, {'name': 'Henry Burgess'}, {'name': 'Jeffrey Kim'}, {'name': 'Alessa Teunisse'}, {'name': 'Lisa Mackenzie'}, {'name': 'Gail Robinson'}, {'name': 'Peter Dayan'}]\", 'abstract': '<p>Corpus callosum dysgenesis is one of the most common congenital neurological malformations. Despite being a clear and identifiable structural alteration of the brains white matter connectivity, the impact of corpus callosum dysgenesis on cognition and behavior has remained unclear. Here we build upon past clinical observations in the literature to define the clinical phenotype of corpus callosum dysgenesis better using unadjusted and adjusted group differences compared with a neurotypical sample on a range of social and cognitive measures that have been previously reported to be impacted by a corpus callosum dysgenesis diagnosis. Those with a diagnosis of corpus callosum dysgenesis (n = 22) demonstrated significantly higher persuadability, credulity, and insensitivity to social trickery than neurotypical (n = 86) participants, after controlling for age, sex, education, autistic-like traits, social intelligence, and general cognition. To explore this further, machine learning, utilizing a set neurotypical sample for training the normative covariance structure of our psychometric variables, was used to test whether these dimensions possessed the capability to discriminate between a test-set of neurotypical and corpus callosum dysgenesis participants. We found that participants with a diagnosis of corpus callosum dysgenesis were best classed within dimension space along the same axis as persuadability, credulity, and insensitivity to social trickery after controlling for age and sex, with Leave-One-Out-Cross-Validation across 250 training-set permutations providing a mean accuracy of 71.7 percent. These results have wide-reaching implications for a) the characterization of corpus callosum dysgenesis, and b) the role of the corpus callosum in social inference.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.29.474483', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Retrovirus-derived acquired genes, RTL5 and RTL6, are novel constituents of the innate immune system in the eutherian brain.', 'authors': \"[{'name': 'Masahito Irie'}, {'name': 'Johbu Itoh'}, {'name': 'Ayumi Matsuzawa'}, {'name': 'Masahito Ikawa'}, {'name': 'Toru Suzuki'}, {'name': 'Yuichi Hiraoka'}, {'name': 'Fumitoshi Ishino'}, {'name': 'Tomoko Kaneko-Ishino'}]\", 'abstract': '<p>Retrotransposon Gag-like 5 (RTL5, also known as sushi-ichi-related retrotransposon homolog 8 (SIRH8)) and RTL6 (aka SIRH3) are eutherian-specific genes presumably derived from a retrovirus and phylogenetically related to each other. RTL5 encodes a strongly acidic protein while RTL6 encodes an extremely basic protein, and the former is well conserved and the latter extremely well conserved among the eutherians, indicating their unique and critically important roles as acquired genes. Here we report that RTL5 and RTL6 are microglial genes playing roles in the front line of brain innate immune responses against distinct pathogens. Venus and mCherry knock-in mice exhibited expression of RTL5-mCherry and RTL6-Venus fusion proteins in microglia and as extracellular granules in the central nervus system (CNS), and displayed a rapid response to pathogens such as lipopolysaccharide (LPS), double-stranded (ds) RNA analog and non-methylated CpG DNA. These proteins trapped pathogens in microglia in a variety of RTL-pathogen complexes depending on the pathogens. These results demonstrate that RTL5 and RTL6 exert functional effects against different hazardous substances cooperatively and/or independently to protect the developing and/or mature brain. This provides the first evidence that retrovirus-derived genes play a role in the innate immune system of the eutherian brain.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.474505', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Information encoded in volumes and areas of dendritic spines is nearly maximal across mammalian brains', 'authors': \"[{'name': 'Jan Karbowski'}, {'name': 'Paulina Urban'}]\", 'abstract': '<p>Long-term information associated with neuronal memory resides in dendritic spines. However, spines can have a limited size due to metabolic and neuroanatomical constraints, which should effectively limit the amount of encoded information in excitatory synapses. This study investigates how much information can be stored in the sizes of dendritic spines, and whether is it optimal in any sense? It is shown here, using empirical data for several mammalian brains across different regions and physiological conditions, that dendritic spines nearly maximize entropy contained in their volumes and surface areas for a given mean size. This result is essentially independent of the type of a fitting distribution to size data, as both short- and heavy-tailed distributions yield similar nearly 100 % information efficiency in the majority of cases, although heavy-tailed distributions slightly better fit the data. On average, the highest information is contained in spine volume, and the lowest in spine length or spine head diameter. Depending on a species and brain region, a typical spine can encode between 6.1 and 10.8 bits of information in its volume, and 3.1-8.1 bits in its surface area. Our results suggest a universality of entropy maximization in spine volumes and areas, which can be a new principle of memory storing in synapses.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.474511', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A systematic characterization of intrinsically formed microglia-like cells during retinal organoid differentiation.', 'authors': \"[{'name': 'Katarina Bartalska'}, {'name': 'Verena Hübschmann'}, {'name': 'Medina Korkut-Demirbaş'}, {'name': 'Ryan John Abat Cubero'}, {'name': 'Alessandro Venturino'}, {'name': 'Karl Rössler'}, {'name': 'Thomas Czech'}, {'name': 'Sandra Siegert'}]\", 'abstract': '<p>Brain organoids differentiated from human induced pluripotent stem cells provide a unique opportunity to investigate the development, organization and connectivity of neurons in a complex cellular environment. However, organoids usually lack microglia, brain-resident immune cells which are both present in the early human embryonic brain and participate in neuronal circuit development. Here, we find that microglia innately develop in unguided retinal organoid differentiation between week 3 and 4 in 2.5D culture and appear later in floating, non-pigmented, 3D-cystic compartments. We enriched for cystic structures using a low-dosed BMP4 application and performed mass spectrometry, thus defining the protein composition of microglia-containing compartments. We found that cystic compartments expressed both mesenchymal and epithelial markers with microglia enriched in the mesenchymal region. Interestingly, microglia-like cells started to express the border-associated macrophage marker CD163. The preferential localization of human microglia to a mesenchymal compartment provides insight into the behavior and migration of microglia. The model will ultimately allow detailed study of these enigmatic cells and how they enter and distribute within the human brain.</p>', 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.30.474546', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Bioprinting the Tumor Microenvironment with an Upgraded Consumer Stereolithographic 3D Printer', 'authors': \"[{'name': 'Louise Breideband'}, {'name': 'Kaja Nicole Wächtershäuser'}, {'name': 'Levin Hafa'}, {'name': 'Konstantin Wieland'}, {'name': 'Achilleas Frangakis'}, {'name': 'Ernst H.K. Stelzer'}, {'name': 'Francesco Pampaloni'}]\", 'abstract': '<p>A widespread application of three-dimensional (3D) bioprinting in basic and translational research requires the accessibility to affordable printers able to produce physiologically relevant tissue models. To facilitate the use of bioprinting as a standard technique in biology, an open-source device based on a consumer-grade 3D stereolithographic (SL) printer was developed. This SL bioprinter can produce complex constructs that preserve cell viability and recapitulate the physiology of tissues. The detailed documentation of the modifications apported to the printer as well as a throughout performance analysis allow for a straightforward adoption of the device in other labs and its customization for specific applications. Given the low cost, several modified bioprinters could be simultaneously operated for a highly parallelized tissue production. To showcase the capability of the bioprinter, we produced constructs consisting of patient-derived cholangiocarcinoma organoids encapsulated in a gelatin methacrylate (GelMA)/polyethylene glycol diacrylate (PEGDA) hydrogel. A thorough characterization of different GelMA/PEGDA ratios revealed that the mechanical properties of the bioprinted tumor model can be accurately fine-tuned to mimic a specific tumor micro-environment. Immunofluorescence and gene expression analyses of tumor markers confirmed that the bioprinted synthetic hydrogel provides a flexible and adequate replacement of animal-derived reconstituted extracellular matrix.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.28.474388', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Interleukin 4 controls the role of macrophages in pulmonary metastatic tumor cell seeding and growth', 'authors': \"[{'name': 'Carolina A Rodriguez-Tirado'}, {'name': 'David Entenberg'}, {'name': 'Jiufeng Li'}, {'name': 'Bin-Zhi Qian'}, {'name': 'John S Condeelis'}, {'name': 'Jeffrey W Pollard'}]\", 'abstract': '<p>Metastasis is the systemic manifestation of cancer and the main cause of death from breast cancer. In mouse models of lung metastases, recruitment of classical monocytes from blood to the lung and their differentiation to metastasis-associated macrophages (MAMs) facilitate cancer cell extravasation, survival, and growth. Ablation of MAMs or their monocytic progenitors inhibits metastasis. We hypothesized that factors controlling macrophage polarization modulate tumor cell extravasation in the lung. We evaluated whether signaling by Th1 or Th2 cytokines in macrophages affected trans-endothelial migration of tumor cells in vitro. Interferon γ and LPS inhibited macrophage-dependent tumor cell extravasation while the Th2 cytokine interleukin-4 (IL4) enhanced this process. We demonstrated that IL4 receptor (IL4rα) null mice develop fewer and smaller lung metastasis. Adoptive transfer of wild type monocytes to IL4rα deficient mice rescued this phenotype. IL4 signaling in macrophages controls the expression of the chemokine receptor CXCR2, necessary for IL4-mediated tumor cell extravasation in vitro. Furthermore, IL4 signaling in macrophages transcriptionally regulates several other genes already causally associated with lung metastasis including CCL2, CSF1, CCR1, HGF and FLT1. The central role for IL4 signaling in MAMs was confirmed by high-resolution intravital imaging of the lung in mice at the time of metastatic seeding, which showed reduced physical interaction between tumor cells and IL4rα-deficient macrophages. This interaction enhances tumor cell survival. These data indicate that IL4 signaling in monocytes and macrophages is key during seeding and growth of breast metastasis in the lung as it regulates pro-tumoral paracrine signaling between cancer cells and macrophages.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.29.474478', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'First high-quality genome assembly data of sago palm (Metroxylon sagu Rottboll)', 'authors': \"[{'name': 'LEONARD WHYE KIT LIM'}, {'name': 'Melinda Mei Lin Lau'}, {'name': 'Hung-Hui Chung'}, {'name': 'Hasnain Hussain'}, {'name': 'HAN MING GAN'}]\", 'abstract': '<p>The sago palm (Metroxylon sagu Rottboll) is a all-rounder palm, it is both a tropical halophytic starch-producing palm as well as an ornamental plant. Recently, a genome survey was conducted on this palm using Illumina sequencing platform but the BUSCO genome completeness is very low (21.5%) and most of them (~78%) are either fragmented or missing. Thus, in this study, the sago palm genome completeness was further improved with the utilization of the Nanopore sequencing platform that produced longer reads. A hybrid genome assembly was conducted and the outcome was a much complete sago palm genome with BUSCO completeness achieved at as high as 97.9% with only ~2% of them either fragmented or missing. The estimated genome size of the sago palm is 509,812,790 bp in this study. A sum of 33,242 protein-coding genes were revealed from the sago palm genome and around 96.39% of them had been functionally annotated. An investigation on the carbohydrate metabolism KEGG pathways also unearthed that starch synthesis was one of the major sago palm activities. These data are indispensable for future molecular evolutionary and genome-wide association studies.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474479', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Equivalence of the Transition Heat Capacities of Proteins and DNA', 'authors': \"[{'name': 'Matthew W Eskew'}, {'name': 'Albert S Benight'}]\", 'abstract': '<p>It has been reported for many globular proteins that the native heat capacity at 25°C, per gram, is the same. This has been interpreted to indicate that heat capacity is a fundamental property of native proteins that provides important information on molecular structure and stability. Heat capacities for both proteins and DNA has been suggested to be related to universal effects of hydration/solvation on native structures. Here we report on results from thermal denaturation analysis of two well-known proteins, human serum albumin and lysozyme, and a short DNA hairpin. The transition heat capacities at the Tm for the three molecules were quantitatively evaluated by differential scanning calorimetry. When normalized per gram rather than per mol the transition heat capacities were found to be precisely equivalent. This observation for the transition heat capacities of the proteins is consistent with previous reports. However, an identical transition heat capacity for DNA has not been reported and was unexpected. Further analysis of the collected data suggested a mass dependence of hydration effects on thermal denaturation that is preserved at the individual protein amino acid and DNA base levels. Equivalence of transition heat capacities suggests the possibility of a universal role of hydration effects on the thermal stability of both proteins and DNA.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.28.474375', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Introducing a New Bond-Forming Activity in an Archaeal DNA Polymerase by Structure-Guided Enzyme Redesign', 'authors': \"[{'name': 'Tushar Aggarwal'}, {'name': 'William A Hansen'}, {'name': 'Jonathan Hong'}, {'name': 'Abir Ganguly'}, {'name': 'Darrin M York'}, {'name': 'Sagar D Khare'}, {'name': 'Enver C Izgu'}]\", 'abstract': '<p>DNA polymerases have evolved to feature a highly conserved activity across the tree of life: formation of, without exception, phosphodiester linkages that create the repeating sugar-phosphate backbone of DNA. Can this linkage selectivity observed in nature be overcome by design to produce non-natural nucleic acids? Here, we report that structure-guided redesign of an archaeal DNA polymerase (9°N) enables a new polymerase activity that is undetectable in the wild type enzyme: catalyzing the formation of N3′→P5′ phosphoramidate linkages in the presence of 3′-amino-2′,3′-dideoxynucleoside 5′-triphosphate (3′-NH<sub>2</sub>-ddNTP) building blocks. Replacing a highly conserved metal-binding aspartate in the 9°N active site (Asp-404) with asparagine was key to the emergence of this unnatural enzyme activity. Molecular dynamics simulations provided insights into how a single substitution could enhance the productive positioning of the 3′-amino nucleophile in the active site. Further remodeling of the protein-nucleic acid interface with substitutions in the finger subdomain led to a quadruple-mutant variant (9°N-NRQS) that incorporated 3′-NH<sub>2</sub>-ddNTPs into a 3′-amino-primer on various DNA templates. This work presents the first example of an active-site substitution of a metal-binding residue that leads to a novel activity in a DNA polymerase, and sheds light on the molecular basis of substrate fidelity and latent promiscuity in enzymes.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.25.21268206', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Topological data analysis identifies distinct biomarker phenotypes during the inflammatory phase of COVID-19', 'authors': \"[{'name': 'Paul W Blair'}, {'name': 'Joost Brandsma'}, {'name': 'Josh G. Chenoweth'}, {'name': 'Stephanie A. Richard'}, {'name': 'Nusrat J. Epsi'}, {'name': 'Rittal Mehta'}, {'name': 'Deborah A. Striegel'}, {'name': 'Emily G. Clemens'}, {'name': 'David A. Lindholm'}, {'name': 'Ryan C. Maves'}, {'name': 'Derek T. Larson'}, {'name': 'Katrin Mende'}, {'name': 'Rhonda E. Colombo'}, {'name': 'Anuradha Ganesan'}, {'name': 'Tahaniyat Lalani'}, {'name': 'Christopher J. Colombo'}, {'name': 'Allison A. Malloy'}, {'name': 'Andrew L. Snow'}, {'name': 'Kevin L. Schully'}, {'name': 'Charlotte Lanteri'}, {'name': 'Mark P. Simons'}, {'name': 'John S. Dumler'}, {'name': 'David Tribble'}, {'name': 'Timothy Burgess'}, {'name': 'Simon Pollett'}, {'name': 'Brian K. Agan'}, {'name': 'Danielle V. Clark'}, {'name': ' '}]\", 'abstract': '<p>OBJECTIVES: The relationships between baseline clinical phenotypes and the cytokine milieu of the peak inflammatory phase of coronavirus 2019 (COVID-19) are not yet well understood. We used Topological Data Analysis (TDA), a dimensionality reduction technique to identify patterns of inflammation associated with COVID-19 severity and clinical characteristics.\\nDESIGN: Exploratory analysis from a multi-center prospective cohort study.\\nSETTING: Eight military hospitals across the United States between April 2020 and January 2021.\\nPATIENTS: Adult (≥18 years of age) SARS-CoV-2 positive inpatient and outpatient participants were enrolled with plasma samples selected from the putative inflammatory phase of COVID-19, defined as 15-28 days post symptom onset. \\nINTERVENTIONS: None.\\nMEASUREMENTS AND MAIN RESULTS: Concentrations of 12 inflammatory protein biomarkers were measured using a broad dynamic range immunoassay. TDA identified 3 distinct inflammatory protein expression clusters. Peak severity (outpatient, hospitalized, ICU admission or death), Charlson Comorbidity Index (CCI), and body mass index (BMI) were evaluated with logistic regression for associations with each cluster. The study population (n=129, 33.3% female, median 41.3 years of age) included 77 outpatient, 31 inpatient, 16 ICU-level, and 5 fatal cases. Three distinct clusters were found that differed by peak disease severity (p &lt;0.001), age (p &lt;0.001), BMI (p&lt;0.001), and CCI (p=0.001). \\nCONCLUSIONS: Exploratory clustering methods can stratify heterogeneous patient populations and identify distinct inflammation patterns associated with comorbid disease, obesity, and severe illness due to COVID-19.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.31.474618', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Apoplastic CBM1-interacting proteins bind conserved carbohydrate binding module 1 motifs in fungal hydrolases to counter pathogen invasion', 'authors': \"[{'name': 'Takumi Takeda'}, {'name': 'Machiko Takahashi'}, {'name': 'Motoki Shimizu'}, {'name': 'Yu Sugihara'}, {'name': 'Hiromasa Saitoh'}, {'name': 'Koki Fujisaki'}, {'name': 'Kazuya Ishikawa'}, {'name': 'Hiroe Utsushi'}, {'name': 'Eiko Kanzaki'}, {'name': 'Yuichi Sakamoto'}, {'name': 'Akira Abe'}, {'name': 'Ryohei Terauchi'}]\", 'abstract': '<p>When infecting plants, fungal pathogens secrete cell wall degrading enzymes (CWDEs) that break down cellulose and hemicellulose, the primary components of plant cell walls. Some fungal CWDEs contain a unique domain, named the carbohydrate binding module (CBM), that facilitates their access to polysaccharides. However, little is known about how plants counteract pathogen degradation of their cell walls. Here, we show that the rice cysteine-rich repeat secretion protein OsCBMIP binds to and inhibits xylanase MoCel10A of the blast fungus pathogen Magnaporthe oryzae, interfering with its access to the rice cell wall and degradation of rice xylan. We found binding of OsCBMIP to various CBM1-containing enzymes, suggesting it has a general role in inhibiting the catalytic activities of fungal enzymes. OsCBMIP is localized to the apoplast, and its expression is strongly induced in leaves infected with M. oryzae. Remarkably, knockdown of OsCBMIP reduced rice defense against M. oryzae, demonstrating that inhibition of CBM1-containing fungal enzymes by OsCBMIP is crucial for rice defense. We also identified additional CBMIP-related proteins from Arabidopsis thaliana and Setaria italica, indicating that a wide range of plants counteract pathogens through this mechanism.</p>', 'posted': '2022-01-02'}, {'doi': '10.1101/2021.12.30.474596', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Oxytocin modulates social brain network correlations in resting and task state', 'authors': \"[{'name': 'Qingyuan Wu'}, {'name': 'Qi Huang'}, {'name': 'Chao Liu'}, {'name': 'Haiyan Wu'}]\", 'abstract': \"<p>Oxytocin (OT) is a neuropeptide that modulates social behaviors and the social brain. The effects of OT on the social brain can be tracked by assessing the neural activity in the resting and task states, providing a system-level framework for characterizing state-based functional relationships of its distinct effect. Here, we contribute to this framework by examining how OT modulates social brain network correlations during the resting and task states using fMRI. Firstly, we investigated network activation, followed by analyzing the relationship between networks and individual differences measured by the Positive and Negative Affect Schedule and the Big-Five scales. Subsequently, we evaluated functional connectivity in both states. Finally, the relationship between networks across the states was represented by the predictive power of networks in the resting state for task-evoked activity. The difference in predicted accuracy between subjects displayed individual variations in this relationship. Our results showed decreased  dorsal default mode network (DDMN) for OT group in the resting state. Additionally, only in the OT group, the activity of the DDMN in the resting state had the largest predictive power for task-evoked activation of the precuneus network (PN). The results also demonstrated OT reduced individual variation of PN, specifically, the difference of accuracy between predicting a subject's own and others' PN task activation. These findings suggest a distributed but modulatory effect of OT on the association between resting brain networks and task-dependent brain networks, showing increased DDMN to PN connectivity after OT administration, which may support OT-induced distributed processing during task performance.</p>\", 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.30.474527', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Gut mélange à trois: fluctuating selection modulated by microbiota, host immune system, and antibiotics', 'authors': \"[{'name': 'Hugo Condessa Barreto'}, {'name': 'Beatriz Abreu'}, {'name': 'Isabel Gordo'}]\", 'abstract': \"<p>Iron is critical in host-microbe interactions, and its availability is under tight regulation in the mammalian gut. Antibiotics and inflammation are known to perturb iron availability in the gut, which could subsequently alter host-microbe interactions. Here, we show that an adaptive allele of <italic>iscR</italic>, encoding a major regulator of iron homeostasis of <italic>Escherichia coli</italic>, is under fluctuating selection in the mouse gut. <italic>In vivo</italic> competitions in immune-competent, immune-compromised, and germ-free mice reveal that the selective pressure on an <italic>iscR</italic> mutant <italic>E. coli</italic> is modulated by the presence of antibiotics, other members of the microbiota, and the immune system. <italic>In vitro</italic> assays show that iron availability is an important mediator of the <italic>iscR</italic> allele fitness benefits or costs.  We identify Lipocalin-2, a host's innate immune system protein that prevents bacterial iron acquisition, as a major host mechanism underlying fluctuating selection of the <italic>iscR allele</italic>. Our results provide a remarkable example of strong fluctuating selection acting on bacterial iron regulation in the mammalian gut.</p>\", 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.474508', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Sequestration of histidine kinases by non-cognate response regulators establishes a threshold level of stimulation for bacterial two-component signaling', 'authors': \"[{'name': 'Gaurav D Sankhe'}, {'name': 'Rubesh Raja'}, {'name': 'Narendra M Dixit'}, {'name': 'Deepak Kumar Saini'}]\", 'abstract': '<p>Two-component signaling systems (TCSs) in bacteria are often positively auto-regulated, where the histidine kinase (HK) and response regulator (RR) proteins comprising a TCS are expressed downstream of the signal they transduce. This auto-regulation improves the sensitivity of the TCS to stimuli and amplifies adaptive responses. The downside, however, is that the TCS may mount disproportionately large responses to weak or fleeting signals. How bacteria prevent such disproportionate responses is not known. Here, we show that sequestration of phosphorylated HKs by non-cognate RRs serves as a design to prevent such disproportionate responses. Using TCSs of M. tuberculosis as model systems, we found that with every one of the five HKs we studied, there was at least one non-cognate RR with higher affinity than that of the cognate RR for the HK. Phosphorylated HKs would thus preferentially bind the non-cognate RRs, suppressing signal transduction through the cognate pathways, which we demonstrated in vitro. Using mathematical modeling of TCS signaling in vivo, we predicted that this sequestration would introduce a threshold level of stimulation for a significant response, preventing responses to signals below this threshold. Finally, we showed in vivo using tunable expression systems in M. bovis that upregulation of a higher affinity non-cognate RR substantially suppressed the output from the cognate TCS pathway, presenting strong evidence of sequestration by non-cognate RRs as a design to regulate TCS signaling. Blocking this sequestration may be a novel intervention strategy, as it would compromise bacterial fitness by letting it respond unnecessarily to signals.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.474514', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Kinetics Of Interferon-λ And Receptor Expression In Response To In Vitro Respiratory Viral Infection', 'authors': \"[{'name': 'Alexey Lozhkov'}, {'name': 'Nikita Yolshin'}, {'name': 'Irina Baranovskaya'}, {'name': 'Marina Plotnikova'}, {'name': 'Mariia Sergeeva'}, {'name': 'Natalia Gyulikhandanova'}, {'name': 'Sergey Klotchenko'}, {'name': 'Andrey Vasin'}]\", 'abstract': '<p>The major protective immune response against viruses is production of type I and III interferons (IFNs). IFNs induce the expression of hundreds of IFN-stimulated genes (ISGs) that block viral replication and further viral spread. The ability of respiratory viruses to suppress induction of IFN-mediated antiviral defenses in infected epithelial cells may be a factor contributing to the particular pathogenicity of several strains. In this report, we analyzed expression of IFNs and some ISGs in an alveolar epithelial cell subtype (A549) in response to infection with: influenza A viruses (A/California/07/09pdm (H1N1), A/Texas/50/12 (H3N2)); influenza B virus (B/Phuket/3073/13); adenovirus type 5 and 6; or respiratory syncytial virus (strain A2). IFNL and ISGs expression significantly increased in response to infection with all RNA viruses 24 hpi. Nevertheless, only IBV led to early increase in IFNL and ISGs mRNA level. IBV and H1N1 infection led to elevated proinflammatory cytokine production. We speculate that augmented IFN-α, IFN-β, IL-6 levels negatively correlate to SOCS1 expression. Importantly, we showed a decrease in IFNLR1 mRNA in case of IBV infection that implies the existence of negative ISGs expression regulation at IFNλR level. It could be either a specific feature of IBV or a consequence of early IFNL expression.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474498', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Macrophages break interneuromast cell quiescence by intervening the inhibition of Schwann cells in zebrafish lateral line', 'authors': \"[{'name': 'Meng-Ju Lin'}, {'name': 'Chia-Ming Lee'}, {'name': 'Wei-Lin Hsu'}, {'name': 'Bi-Chang Chen'}, {'name': 'Shyh-Jye Lee'}]\", 'abstract': '<p>In the zebrafish lateral line system, interneuromast cells (INCs) between neuromasts are normally kept quiescent by underlying Schwann cells (SWCs). Upon severe injuries that cause the complete loss of an entire NM, INCs can occasionally differentiate into NMs but how they escape from the inhibition by SWCs is still unclear. Using a genetic/chemical method to specifically ablate a neuromast, we found a small portion of larvae can regenerate a new neuromast, but the regeneration was hindered by inhibiting macrophages. We also demonstrated that the inhibition of macrophage can reduce the percentage of tail fin-amputated larvae to regenerate a new NM. By in toto imaging, we further discovered heterogeneities in macrophage behavior and distribution along lateral line. We witnessed the crawling of macrophages in between injured lateral line and SWCs during regeneration and also in between the second primordium and the first mature lateral line during development. It implies that macrophages may physically separate and alleviate the inhibition from pLLn and SWCs to break the quiescence of INCs during regeneration and development in the zebrafish lateral line.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.27.474232', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Discovery of a class of giant virus relatives displaying unusual functional traits and prevalent within plankton: the Mirusviricetes', 'authors': \"[{'name': 'Morgan Gaia'}, {'name': 'Lingjie Meng'}, {'name': 'Eric Pelletier'}, {'name': 'Patrick Forterre'}, {'name': 'Chiara Vanni'}, {'name': 'Antonio Fernendez-Guerra'}, {'name': 'Olivier Jaillon'}, {'name': 'Patrick Wincker'}, {'name': 'Hiroyuki Ogata'}, {'name': 'Tom O. Delmont'}]\", 'abstract': '<p>Large and giant DNA viruses of the phylum Nucleocytoviricota have a profound influence on the ecology and evolution of planktonic eukaryotes. Recently, various Nucleocytoviricota genomes have been characterized from environmental metagenomes based on the occurrence of hallmark genes identified from cultures. However, lineages diverging from the culture genomics functional principles have been overlooked thus far. Here, we developed a phylogeny-guided genome-resolved metagenomic framework using a single hallmark gene as compass, a subunit of DNA-dependent RNA polymerase encoded by most Nucleocytoviricota. We applied this method to large metagenomic data sets from the surface of five oceans and two seas and characterized 697 non-redundant Nucleocytoviricota genomes up to 1.45 Mbp in length. This database expands the known diversity of the class Megaviricetes and revealed two additional putative classes we named Proculviricetes and Mirusviricetes. Critically, the diverse and prevalent Mirusviricetes population genomes seemingly lack several hallmark genes, in particular those related to viral particle morphogenesis. Instead, they share various genes of known (e.g., TATA-binding proteins, histones, proteases and viral rhodopsins) and unknown functions rarely detected if not entirely missing in all other characterized Nucleocytoviricota lineages. Phylogenomics, comparative genomics, functional trends and the signal among planktonic cellular size fractions point to Mirusviricetes being a major, functionally divergent class of large DNA viruses that actively infect eukaryotes in the sunlit ocean using an enigmatic functional life style. Finally, we built a comprehensive marine genomic database for Nucleocytoviricota by combining multiple environmental surveys that might contribute to future endeavors exploring the ecology and evolution of plankton.</p>', 'posted': '2021-12-27'}, {'doi': '10.1101/2021.12.28.474260', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Simple and cost-effective SNP genotyping method for discriminating subpopulations of the fish pathogen, Nocardia seriolae', 'authors': \"[{'name': 'Cuong T. Le'}, {'name': 'Erin P. Price'}, {'name': 'Derek S. Sarovich'}, {'name': 'Thu T.A. Nguyen'}, {'name': 'Hung Vu-Khac'}, {'name': 'Ipek Kurtböke'}, {'name': 'Wayne Knibb'}, {'name': 'Shih-Chu Chen'}, {'name': 'Mohammad Katouli'}]\", 'abstract': '<p>Nocardia seriolae has caused significant fish losses in Asia and the Americas in recent decades, including in Vietnam, which has witnessed devastating economic and social impacts due to this bacterial pathogen. Surveillance strategies are urgently needed to mitigate N. seriolae dissemination in Vietnamese aquaculture and mariculture industries. Whole-genome sequencing (WGS) offers the highest level of resolution to discriminate closely related strains and to determine their putative origin and transmission routes. However, WGS is impractical for epidemiological investigations and pathogen surveillance due to its time-consuming and costly nature, putting this technology out-of-reach for many industry end-users. To overcome this issue, we targeted two previously characterised, phylogenetically informative single-nucleotide polymorphisms (SNPs) in N. seriolae that accurately distinguish: i) Vietnamese from non-Vietnamese strains, and ii) the two Vietnamese subclades. Using the mismatch amplification mutation assay (MAMA) format, we developed assays that genotype strains based on differences in amplicon melting temperature (melt-MAMA) and size (agarose-MAMA). Our MAMA assays accurately genotyped strains both from culture and fish tissues at low cost, using either real-time (~AUD$1/per sample) or conventional (~AUD$0.50/per sample) PCR instrumentation. Our novel assays provide a rapid, reproducible, and cost-effective tool for routine genotyping of this pathogen, allowing faster identification and treatment of nocardiosis-effected permit fish within Vietnamese aquaculture/mariculture facilities, an essential step in mitigating N. seriolae-associated losses.</p>', 'posted': '2021-12-28'}, {'doi': '10.1101/2021.12.30.474515', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Exercise and the brain: a mechanical model for pulsation on flow of cerebrospinal fluid', 'authors': \"[{'name': 'Michael George Hale'}, {'name': 'Jonathan Ainsley Coles'}]\", 'abstract': '<p>Exchange of molecules between cerebrospinal fluid (CSF) and brain cells contributes to brain function and protection from dementia, but the route by which CSF is brought close enough to the neural tissue to be exchanged by extracellular diffusion is not clear. Exogenous molecules injected into CSF are carried along channels outside arteries and reach the basement lamina that surrounds the dense capillary network.  Transport of solutes by diffusion along the basement lamina, a gel of macromolcules about 100 nm thick, would be too slow; bulk flow in a static geometry would require unphysiologically high pressures. However, it is known that the pulsation of blood aids transport of CSF, and we hypothesized that this is because the pulsation intermittently squeezes the pericapillary lamina. In a primitive mimicry, we have tested whether intermittent squeezing increases flow through an agar gel. In all but one of 216 tests, pulsation caused a reversible increase, sometimes by a factor of 100 or more. The enhancement was greatest for frequencies 5-11 Hz  and, over the tested range of pressure heads (20 - 50 cmH2O), was greatest for the lowest pressure. The results suggest one reason why exercise slows the aging of the brain.</p>', 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.28.474396', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Differential regulation of prelimbic and thalamic transmission to the basolateral amygdala by acetylcholine receptors', 'authors': \"[{'name': 'Sarah C. Tryon'}, {'name': 'Joshua X. Bratsch-Prince'}, {'name': 'James W. Warren'}, {'name': 'Grace C. Jones'}, {'name': 'Alexander J. McDonald'}, {'name': 'David D. Mott'}]\", 'abstract': '<p>The amygdalar anterior basolateral nucleus (BLa) plays a vital role in emotional behaviors. This region receives dense cholinergic projections from basal forebrain which are critical in regulating neuronal activity and synaptic transmission. Cholinergic signaling in BLa is thought to occur through both a slow mode of volume transmission as well as a rapid, phasic mode. However, the relative effect of each mode of signaling in BLa is not understood. Here, we used electrophysiology and optogenetics in mouse brain slices to compare regulation of afferent input from cortex and thalamus to the BLa by these two modes of transmission. Phasic ACh release evoked by single pulse stimulation of cholinergic terminals had a biphasic effect on glutamatergic transmission at cortical input, producing rapid nicotinic receptor-mediated facilitation followed by slower muscarinic receptor (mAChR)-mediated depression. In contrast, tonic elevation of ACh through application of the cholinesterase inhibitor physostigmine suppressed glutamatergic transmission at cortical inputs through mAChRs only. This suppression was not observed at thalamic inputs to BLa. In agreement with this pathway-specificity, the mAChR agonist, muscarine more potently suppressed transmission at inputs from prelimbic cortex (PL) than thalamus. Muscarinic inhibition at PL input was dependent on presynaptic M4 mAChRs, while at thalamic input it depended upon M3 mAChR-mediated stimulation of retrograde endocannabinoid signaling. Muscarinic inhibition at both pathways was frequency-dependent, allowing only high frequency activity to pass. These findings demonstrate complex cholinergic regulation of afferent input to BLa that depends upon the mode of ACh release and is both pathway specific and frequency dependent.</p>', 'posted': '2021-12-31'}, {'doi': '10.1101/2021.12.30.471467', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Engineered chimeras unveil swappable modular features of fatty acid and polyketide synthase acyl carrier proteins', 'authors': \"[{'name': 'Yae In Cho'}, {'name': 'Claire L Armstrong'}, {'name': 'Ariana Sulpizio'}, {'name': 'Kofi K Acheampong'}, {'name': 'Kameron N Banks'}, {'name': 'Oishi Bardhan'}, {'name': 'Sydney J Churchill'}, {'name': 'Annie E Connolly-Sporing'}, {'name': 'Callie E.W. Crawford'}, {'name': 'Peter L Cruz Parrilla'}, {'name': 'Sarah M Curtis'}, {'name': 'Lauren M De La Ossa'}, {'name': 'Samuel C Epstein'}, {'name': 'Clara J Farrehi'}, {'name': 'Grayson S Hamrick'}, {'name': 'William J Hillegas'}, {'name': 'Austin Kang'}, {'name': 'Olivia C Laxton'}, {'name': 'Joie Ling'}, {'name': 'Sara M Matsumura'}, {'name': 'Victoria M Marino'}, {'name': 'Shahla H Mukhtar'}, {'name': 'Neel J Shah'}, {'name': 'Casey H Londergan'}, {'name': 'Clyde A Daly'}, {'name': 'Bashkim Kokona'}, {'name': 'Louise K Charkoudian'}]\", 'abstract': '<p>The strategic redesign of microbial biosynthetic pathways is a compelling route to access molecules of diverse structure and function in a potentially environmentally sustainable fashion. The promise of this approach hinges on an improved understanding of acyl carrier proteins (ACPs), which serve as central hubs in biosynthetic pathways. These small, flexible proteins mediate the transport of molecular building blocks and intermediates to enzymatic partners that extend and tailor the growing natural products. Past combinatorial biosynthesis efforts have failed due to incompatible ACP-enzyme pairings. Herein we report the design of chimeric ACPs with features of the actinorhodin polyketide synthase ACP (ACT) and of the E. coli fatty acid synthase (FAS) ACP (AcpP). We evaluate the ability of the chimeric ACPs to interact with the E. coli FAS ketosynthase FabF, which represents an interaction essential to building the carbon backbone of the synthase molecular output. Given that AcpP interacts with FabF but ACT does not, we sought to exchange modular features of ACT with AcpP to confer functionality with FabF. The interactions of chimeric ACPs with FabF were interrogated using sedimentation velocity experiments, surface plasmon resonance analyses, mechanism-based crosslinking assays, and molecular dynamics simulations. Results suggest that the residues guiding AcpP-FabF compatibility and ACT-FabF incompatibility may reside in the loop I, α-helix II region. These findings can inform the development of strategic secondary element swaps that expand the enzyme compatibility of ACPs across systems and therefore represent a critical step towards the strategic engineering of unnatural natural products.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.21268553', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Performance of COVIDSeq and Swift normalase amplicon SARS-CoV-2 panels for SARS-CoV-2 Genomes Sequencing: Practical Guide and Combining FASTQ Strategy', 'authors': \"[{'name': 'Hosoon Choi'}, {'name': 'Munok Hwang'}, {'name': 'Dhammika Navarathna'}, {'name': 'Jing Xu'}, {'name': 'Janell Lukey'}, {'name': 'Chetan Jinadatha'}]\", 'abstract': '<p>The whole genomic sequencing (WGS) of SARS-CoV-2 has been performed extensively and is playing a crucial role in fighting against COVID-19 pandemic. Obtaining sufficient WGS data from clinical samples is often challenging especially from the samples with low viral load. We evaluated two SARS-CoV-2 sequencing protocols for their efficiency/accuracy and limitations. Sequence coverage of &gt;95% was obtained by Swift normalase amplicon SARS-CoV-2 panels (SNAP) protocol for all the samples with Ct ≤ 35 and by COVIDSeq protocol for 97% of samples with Ct ≤ 30. Sample RNA quantitation obtained using digital PCR provided more precise cutoff values. The quantitative digital PCR cutoff values for obtaining 95% coverage are 10.5 copies/μL for SNAP protocol and 147 copies/μL for COVIDSeq protocol. Combining FASTQ files obtained from 2 protocols improved the outcome of sequence analysis by compensating for missing amplicon regions. This process resulted in an increase of sequencing coverage and lineage call precision.</p>', 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.30.474528', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'The effects of protein crowders on small molecule drug diffusion', 'authors': \"[{'name': 'Debabrata Dey'}, {'name': 'Ariane Nunes-Alves'}, {'name': 'Rebecca C. Wade'}, {'name': 'Gideon Schreiber'}]\", 'abstract': '<p>Crowded environments affect the pharmacokinetics of drug molecules. Here, we investigate how three macromolecular protein crowders, bovine serum albumin, hen egg-white lysozyme and myoglobin, influence the translational diffusion rates and interactions of four low molecular-weight drugs, fluorescein, doxorubicin, glycogen synthase kinase-3 inhibitor SB216763 and quinacrine. Using Fluorescence Recovery After Photo-bleaching in Line mode (Line FRAP), Brownian dynamics simulations and molecular docking, we find that the diffusive behavior of the small molecules is highly affected by self-aggregation, interactions with the proteins, and surface adhesion. Fluorescein diffusion is decreased by protein crowders due to their interactions. On the other hand, for doxorubicin, the presence of protein crowders increases diffusion by reducing surface interactions. SB216763 shows a third scenario, where BSA, but not myoglobin or lysozyme, reduces self-aggregation, resulting in faster diffusion. Quinacrine was the only small molecule whose diffusion was not affected by the presence of protein crowders. The mechanistic insights gained here into the effects of interactions of small molecules with proteins and surfaces on the translational diffusion of small molecules can assist in optimizing the design of compounds for higher mobility and lower occlusion in complex macromolecular environments.</p>', 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.30.474544', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'pixOL: pixel-wise dipole-spread function engineering for simultaneously measuring the 3D orientation and 3D localization of dipole-like emitters', 'authors': \"[{'name': 'Tingting Wu'}, {'name': 'Jin Lu'}, {'name': 'Matthew D. Lew'}]\", 'abstract': \"<p>Interactions between biomolecules are characterized by both where they occur and how they are organized, e.g., the alignment of lipid molecules to form a membrane. However, spatial and angular information are mixed within the image of a fluorescent molecule-the microscopy's dipole spread function (DSF). We demonstrate the pixOL algorithm for simultaneously optimizing all pixels within a phase mask to produce an engineered Green's tensor-the dipole extension of point-spread function engineering. The pixOL DSF achieves optimal precision for measuring simultaneously the 3D orientation and 3D location of a single molecule, i.e., 1.14 degree orientation, 0.24 sr wobble angle, 8.17 nm lateral localization, and 12.21 nm axial localization precisions over an 800-nm depth range using 2500 detected photons. The pixOL microscope accurately and precisely resolves the 3D positions and 3D orientations of Nile red within a spherical supported lipid bilayer, resolving both membrane defects and differences in cholesterol concentration, in 6 dimensions.</p>\", 'posted': '2021-12-30'}, {'doi': '10.1101/2021.12.29.474412', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Biomechanical and biochemical assessment of YB-1 expression in melanoma cells', 'authors': \"[{'name': 'Anna Magdalena Cykowska'}, {'name': 'Ulf Krister Hofmann'}, {'name': 'Aadhya Tiwari'}, {'name': 'Corinna Kosnopfel'}, {'name': 'Rosa Riester'}, {'name': 'Marina Danalache'}]\", 'abstract': '<p>Malignant melanoma is the most lethal form of skin cancer; its incidence has increased over the last five decades. Y-box binding protein 1 (YB-1) plays a prominent role in mediating metastatic behavior by promoting epithelial-to-mesenchymal transition (EMT) processes. Migratory melanoma cells exhibit two major phenotypes: elongated mesenchymal or rounded amoeboid. The actomyosin cytoskeleton is key in both phenotypes, but intermediate filaments also undergo a significant rearrangement process, switching from cytokeratin-rich to vimentin and nestin-rich network. In this study, we aimed to investigate to what extent YB-1 impacts the biomechanical (cell stiffness) and biochemical aspects of melanoma cells and their cytoskeleton. To this end, we subjected A375 YB-1 knock-out and parental cells to atomic force microscopy investigations (stiffness determination), immunolabelling, and proteome analysis. We found that YB-1 expressing cells were significantly stiffer compared to the corresponding YB-1 knock-out cell line. Proteomic analysis revealed that expression of YB-1 results in a strong co-expression of nestin, vimentin, fascin-1, and septin-9. In the YB-1 knock-out nestin was completely depleted, but zyxin was strongly upregulated. Collectively, our results showed that YB-1 knock-out acquires some characteristics of mesenchymal phenotype but lacks important markers of malignancy and invasiveness such as nestin or vimentin. We posit that there is an association of YB-1 expression with an amoeboid phenotype, which would explain the increased migratory capacity.</p>', 'posted': '2021-12-29'}, {'doi': '10.1101/2021.12.30.474582', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'NETISCE: A Network-Based Tool for Cell Fate Reprogramming', 'authors': \"[{'name': 'Lauren Marazzi'}, {'name': 'Milan Shah'}, {'name': 'Shreedula Balakrishnan'}, {'name': 'Ananya Patil'}, {'name': 'Paola Vera-Licona'}]\", 'abstract': '<p>The search for effective therapeutic targets in fields like regenerative medicine and cancer research has generated interest in cell fate reprogramming. This cellular reprogramming paradigm can drive cells to a desired target state from any initial state. However, methods for identifying reprogramming targets remain limited for biological systems that lack large sets of experimental data or a dynamical characterization. We present NETISCE, a novel computational tool for identifying cell fate reprogramming targets in static networks. NETISCE identifies reprogramming targets through the innovative use of control theory within a dynamical systems framework. Through validations in studies of cell fate reprogramming from developmental, stem cell, and cancer biology, we show that NETISCE can predict previously identified cell fate reprogramming targets and identify potentially novel combinations of targets. NETISCE extends cell fate reprogramming studies to larger-scale biological networks without the need for full model parameterization and can be implemented by experimental and computational biologists to identify parts of a biological system that are relevant for the desired reprogramming task.</p>', 'posted': '2022-01-01'}, {'doi': '10.1101/2021.12.22.21268161', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Primary Care Research', 'covid': False, 'title': 'Predicting risk of pancreatic cancer in individuals with new-onset type-2 diabetes in primary care: protocol for the development and validation of a clinical prediction model (QPancreasD)', 'authors': \"[{'name': 'Pui San Tan'}, {'name': 'Ashley Kieran Clift'}, {'name': 'Weiqi Liao'}, {'name': 'Martina Patone'}, {'name': 'Carol Coupland'}, {'name': 'Rachael Bashford-Rogers'}, {'name': 'Shivan Sivakumar'}, {'name': 'David Clifton'}, {'name': 'Stephen P Pereira'}, {'name': 'Julia Hippisley-Cox'}]\", 'abstract': '<sec><title>Background</title><p>Pancreatic cancer continues to have an extremely poor prognosis in part due to late diagnosis. 25% of pancreatic cancer patients have a prior diagnosis of diabetes, and hence identifying individuals at risk of pancreatic cancer in those with recently diagnosed type 2 diabetes may be a useful opportunity to identify candidates for screening and early detection. In this study, we will comparatively evaluate regression and machine learning-based clinical prediction models for estimating individual risk of developing pancreatic cancer two years after type 2 diabetes diagnosis.</p></sec><sec><title>Methods</title><p>In the development dataset, we will include adults aged 30-84 years with incident type-2 diabetes registered with QResearch primary care database. Patients will be followed up from type-2 diabetes diagnosis to first diagnosis of pancreatic cancer as recorded in any one of primary care records, hospital episode statistics, cancer registry data, or death records. Cox-proportional hazards models will be used to develop a risk prediction model for estimating individual risk of developing pancreatic cancer during up to 2 years of follow-up. We will perform variable selection using a combination of clinical and statistical significance approach i.e. HR &lt;0.9 or &gt;1.1 and p&lt;0.01. Linear predictors and baseline survivor function at 2 years will be used to compute absolute risk predictions.</p><p>Internal-external cross-validation (IECV) framework across geographical regions within England will be used to assess performance and pooled using random effects meta-analysis using: (i) model fit in terms of variation explained by the model Royston &amp; Sauerbrei’s R2D, (ii) calibration slope and calibration-in-the-large, and (iii) discrimination measured in terms of Harrell’s C and Royston &amp; Sauerbrei’s D-statistic.</p><p>Further, we will evaluate machine learning (ML) approaches for the clinical prediction model using neural networks (NN) and XGBoost. The model predictors and performance of these will be compared with the results of those derived from the regression-based strategy.</p></sec><sec><title>Discussion</title><p>The proposed study will develop and validate a novel risk prediction model to aid early diagnosis of pancreatic cancer in patients with new-onset diabetes in primary care. With an enhanced decision-risk tool for use at point-of care by general practitioners to assess pancreatic cancer risk, it may improve decision-making so that at-risk patients are rapidly prioritised to aid early diagnosis of pancreatic cancer in patients with newly diagnosed diabetes.</p></sec>', 'posted': '2021-12-27'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.12.28.21268436</td>\n",
       "      <td>382</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Vaccine effectiveness against hospital admissi...</td>\n",
       "      <td>[{'name': 'Glenda E Gray'}, {'name': 'Shirley ...</td>\n",
       "      <td>&lt;p&gt;Following the results of the ENSEMBLE 2 stu...</td>\n",
       "      <td>2021-12-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.12.25.474149</td>\n",
       "      <td>71</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Expanded ACE2 dependencies of diverse SARS-lik...</td>\n",
       "      <td>[{'name': 'Sarah M Roelle'}, {'name': 'Nidhi S...</td>\n",
       "      <td>&lt;p&gt;Viral spillover from animal reservoirs can ...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.12.29.474455</td>\n",
       "      <td>62</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Nerve-mediated amputation-induced stem cell ac...</td>\n",
       "      <td>[{'name': 'Duygu Payzin-Dogru'}, {'name': 'Sar...</td>\n",
       "      <td>&lt;p&gt;Animals exhibit extreme diversity in regene...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.12.29.474457</td>\n",
       "      <td>59</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Effects of periodic bottlenecks on the dynamic...</td>\n",
       "      <td>[{'name': 'Minako Izutsu'}, {'name': 'Devin M....</td>\n",
       "      <td>&lt;p&gt;Population bottlenecks are common in nature...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.12.30.474555</td>\n",
       "      <td>55</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioengineering</td>\n",
       "      <td>False</td>\n",
       "      <td>A nanopore interface for high bandwidth DNA co...</td>\n",
       "      <td>[{'name': 'Karen Zhang'}, {'name': 'Yuan-Jyue ...</td>\n",
       "      <td>&lt;p&gt;DNA has emerged as a powerful substrate for...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.12.29.474489</td>\n",
       "      <td>50</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>High density single-unit human cortical record...</td>\n",
       "      <td>[{'name': 'Jason E Chung'}, {'name': 'Kristin ...</td>\n",
       "      <td>&lt;p&gt;A fundamental unit of neural computation is...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.12.28.474382</td>\n",
       "      <td>46</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Precise measurement of nanoscopic septin ring ...</td>\n",
       "      <td>[{'name': 'Amin Zehtabian'}, {'name': 'Paul Ma...</td>\n",
       "      <td>&lt;p&gt;The combination of image analysis and fluor...</td>\n",
       "      <td>2021-12-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.12.23.21268179</td>\n",
       "      <td>44</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>False</td>\n",
       "      <td>An online atlas of human plasma metabolite sig...</td>\n",
       "      <td>[{'name': 'Koen F. Dekkers'}, {'name': 'Sergi ...</td>\n",
       "      <td>&lt;p&gt;The human gut microbiota produces a variety...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.12.28.474304</td>\n",
       "      <td>40</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Techniques for visualizing fibroblast-vessel i...</td>\n",
       "      <td>[{'name': 'Hannah E Jones'}, {'name': 'Kelsey ...</td>\n",
       "      <td>&lt;p&gt;Fibroblasts are found associated with blood...</td>\n",
       "      <td>2021-12-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.12.29.474385</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Single-nuclei isoform RNA sequencing reveals c...</td>\n",
       "      <td>[{'name': 'Simon A Hardwick'}, {'name': 'Wen H...</td>\n",
       "      <td>&lt;p&gt;Single-nuclei RNA-Seq is being widely emplo...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.12.30.474605</td>\n",
       "      <td>31</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Roles for Mitochondrial Complex I subunits in ...</td>\n",
       "      <td>[{'name': 'Bhagaban Mallik'}, {'name': 'C. And...</td>\n",
       "      <td>&lt;p&gt;To identify conserved components of synapse...</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.12.28.474323</td>\n",
       "      <td>30</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A versatile active learning workflow for optim...</td>\n",
       "      <td>[{'name': 'Amir Pandi'}, {'name': 'Christoph D...</td>\n",
       "      <td>&lt;p&gt;The study, engineering and application of b...</td>\n",
       "      <td>2021-12-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.12.30.21268540</td>\n",
       "      <td>29</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Magnitude and breadth of neutralizing antibody...</td>\n",
       "      <td>[{'name': 'Benjamin L. Sievers'}, {'name': 'Sa...</td>\n",
       "      <td>&lt;p&gt;Multiple SARS-CoV-2 variants that possess m...</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.12.27.474246</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Species-specific mitochondria dynamics and met...</td>\n",
       "      <td>[{'name': 'Ryohei Iwata'}, {'name': 'Pierre Ca...</td>\n",
       "      <td>&lt;p&gt;The evolution of species involves changes i...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.12.24.21268378</td>\n",
       "      <td>23</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Preserved Omicron Spike specific antibody bind...</td>\n",
       "      <td>[{'name': 'Yannic Bartsch'}, {'name': 'Xin Ton...</td>\n",
       "      <td>&lt;p&gt;Despite the dramatic spread of Omicron glob...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.12.30.474462</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Animal Behavior and Cognition</td>\n",
       "      <td>False</td>\n",
       "      <td>The evolution and biological correlates of han...</td>\n",
       "      <td>[{'name': 'Kai R Caspar'}, {'name': 'Fabian Pa...</td>\n",
       "      <td>&lt;p&gt;The evolution of human right-handedness has...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.12.24.21268385</td>\n",
       "      <td>20</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Schizophrenia-associated somatic copy number v...</td>\n",
       "      <td>[{'name': 'Eduardo A Maury'}, {'name': 'Maxwel...</td>\n",
       "      <td>&lt;p&gt;While inherited and de novo copy number var...</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.12.28.474357</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioengineering</td>\n",
       "      <td>False</td>\n",
       "      <td>A Musculoskeletal Model of the Hand and Wrist ...</td>\n",
       "      <td>[{'name': 'Daniel Clinton McFarland'}, {'name'...</td>\n",
       "      <td>&lt;p&gt;Objective: The purpose of this work was to ...</td>\n",
       "      <td>2021-12-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.12.28.474366</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>The mitochondrial calcium uniporter regulates ...</td>\n",
       "      <td>[{'name': 'Katy E Pannoni'}, {'name': 'Daniela...</td>\n",
       "      <td>&lt;p&gt;CA2 is an understudied subregion of the hip...</td>\n",
       "      <td>2021-12-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.12.29.473670</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Reconstitution of the S. aureus agr quorum sen...</td>\n",
       "      <td>[{'name': 'Aishan Zhao'}, {'name': 'Steven Par...</td>\n",
       "      <td>&lt;p&gt;In Staphylococcus aureus, virulence is unde...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.12.30.21268538</td>\n",
       "      <td>15</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Protective effect of BNT162b2 vaccination on a...</td>\n",
       "      <td>[{'name': 'Yair Blumberg'}, {'name': 'Michael ...</td>\n",
       "      <td>&lt;p&gt;There is increasing evidence that patients ...</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.12.27.21268442</td>\n",
       "      <td>15</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>False</td>\n",
       "      <td>Brain atrophy in prodromal synucleinopathy is ...</td>\n",
       "      <td>[{'name': 'Shady Rahayel'}, {'name': 'Christin...</td>\n",
       "      <td>&lt;p&gt;Isolated REM sleep behaviour disorder (iRBD...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.12.29.474477</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Optogenetic inhibition of the dorsal hippocamp...</td>\n",
       "      <td>[{'name': 'Shuyi Qi'}, {'name': 'Shi Min Tan'}...</td>\n",
       "      <td>&lt;p&gt;The dorsal hippocampus (DH) is key to the l...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.12.28.474379</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>FastCAT accelerates absolute quantification of...</td>\n",
       "      <td>[{'name': 'Ignacy Rzagalinski'}, {'name': 'Ali...</td>\n",
       "      <td>&lt;p&gt;Absolute (molar) quantification of proteins...</td>\n",
       "      <td>2021-12-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.12.31.21268575</td>\n",
       "      <td>12</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>COVID-19 severity prediction based on patient ...</td>\n",
       "      <td>[{'name': 'Ariel Israel'}, {'name': 'Alejandro...</td>\n",
       "      <td>&lt;p&gt;Background\\nVaccines are highly effective i...</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.12.26.474085</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 Omicron-B.1.1.529 Variant leads to ...</td>\n",
       "      <td>[{'name': 'Eleanor G Bentley'}, {'name': 'Adam...</td>\n",
       "      <td>&lt;p&gt;COVID-19 is a spectrum of clinical symptoms...</td>\n",
       "      <td>2021-12-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.12.24.474110</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Reduced infectivity but increased immune escap...</td>\n",
       "      <td>[{'name': 'Jie Hu'}, {'name': 'Pai Peng'}, {'n...</td>\n",
       "      <td>&lt;p&gt;A new detected SARS-CoV-2 variant Omicron (...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.12.29.474474</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A NEUROGENIC GENE EXPRESSION SIGNATURE SUPPORT...</td>\n",
       "      <td>[{'name': 'Javier Solivan-Rivera'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;Human beige/brite thermogenic adipose tissu...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.12.27.474262</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Uncovering a neural circuit controlling adult ...</td>\n",
       "      <td>[{'name': 'Moawiah Naffaa'}, {'name': 'Rehan K...</td>\n",
       "      <td>&lt;p&gt;The maintenance and differentiation of the ...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.12.24.474138</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Homologous or Heterologous Booster of Inactiva...</td>\n",
       "      <td>[{'name': 'Xun Wang'}, {'name': 'Xiaoyu Zhao'}...</td>\n",
       "      <td>&lt;p&gt;The massive and rapid transmission of SARS-...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.12.30.474557</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Properties and proximity proteomics of synapto...</td>\n",
       "      <td>[{'name': 'Hanieh Falahati'}, {'name': 'Yumei ...</td>\n",
       "      <td>&lt;p&gt;The spine apparatus is a specialization of ...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.12.27.474025</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Physiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Integrated cytometry with machine learning app...</td>\n",
       "      <td>[{'name': 'Seth Winfree'}, {'name': 'Andrew T ...</td>\n",
       "      <td>&lt;p&gt;The human kidney is a complex organ with va...</td>\n",
       "      <td>2021-12-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.12.29.474488</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Structural insights into the ion selectivity o...</td>\n",
       "      <td>[{'name': 'Xinyu Teng'}, {'name': 'Danqi Sheng...</td>\n",
       "      <td>&lt;p&gt;MgtE is a Mg2+-selective ion channel whose ...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.12.31.474679</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>PlasmidMaker: a Versatile, Automated, and High...</td>\n",
       "      <td>[{'name': 'Behnam Enghiad'}, {'name': 'Pu Xue'...</td>\n",
       "      <td>&lt;p&gt;Plasmids are used extensively in basic and ...</td>\n",
       "      <td>2022-01-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.12.30.474558</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>BDNF-induced BDNF release mediates presynaptic...</td>\n",
       "      <td>[{'name': 'Coralie Berthoux'}, {'name': 'Kaout...</td>\n",
       "      <td>&lt;p&gt;Although brain-derived neurotrophic factor ...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.12.29.474445</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Nipah virus detection at bat roosts following ...</td>\n",
       "      <td>[{'name': 'Clifton D McKee'}, {'name': 'Ausraf...</td>\n",
       "      <td>&lt;p&gt;Knowledge of the dynamics and genetic diver...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.12.29.474460</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Protein interaction analysis of Plasmodium fal...</td>\n",
       "      <td>[{'name': 'Cheikh Cambel Dieng'}, {'name': 'Co...</td>\n",
       "      <td>&lt;p&gt;The world's first malaria vaccine RTS,S pro...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.12.29.474454</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Longitudinal analysis of invariant natural kil...</td>\n",
       "      <td>[{'name': 'Lydia Lynch'}, {'name': 'Harry Kane...</td>\n",
       "      <td>&lt;p&gt;Innate T cells, including CD1d-restricted i...</td>\n",
       "      <td>2021-12-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.12.23.21268244</td>\n",
       "      <td>7</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Effectiveness of casirivimab and imdevimab, an...</td>\n",
       "      <td>[{'name': 'David T. Huang'}, {'name': 'Erin K....</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;IMPORTANCE&lt;/title&gt;&lt;p&gt;The effective...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.12.27.473672</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Effector-mediated subversion of proteasome act...</td>\n",
       "      <td>[{'name': 'Tshegofatso Ngwaga'}, {'name': 'Dee...</td>\n",
       "      <td>&lt;p&gt;Legionella pneumophila causes Legionnaires'...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.12.28.474397</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Comparative analysis of Lysine and Arginine bi...</td>\n",
       "      <td>[{'name': 'Sankar Mahesh'}, {'name': 'Deepa Se...</td>\n",
       "      <td>&lt;p&gt;The members of the Deinococcaceae family ha...</td>\n",
       "      <td>2021-12-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.12.27.474252</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Dichotomous Feedback: A Signal Sequestration-b...</td>\n",
       "      <td>[{'name': 'Aivar Sootla'}, {'name': 'Nicolas D...</td>\n",
       "      <td>&lt;p&gt;We introduce a new design framework for imp...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.12.29.21268505</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Head-to-head comparison of nasal and nasophary...</td>\n",
       "      <td>[{'name': 'Niklaus Daniel Labhardt'}, {'name':...</td>\n",
       "      <td>&lt;p&gt;Objectives\\nTo assess the real-world diagno...</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.12.29.474400</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Extreme value analysis of intracellular cargo ...</td>\n",
       "      <td>[{'name': 'Takuma Naoi'}, {'name': 'Yuki Kagaw...</td>\n",
       "      <td>&lt;p&gt;In the long axon of a neuron, cargo transpo...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2022.01.01.474684</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Concerted expansion and contraction of immune ...</td>\n",
       "      <td>[{'name': 'Bruno Pok Man Ngou'}, {'name': 'Rob...</td>\n",
       "      <td>&lt;p&gt;Recent reports suggest that cell-surface an...</td>\n",
       "      <td>2022-01-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.12.26.473589</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Biological Aging of CNS-Resident Cells Alters ...</td>\n",
       "      <td>[{'name': 'Jeffrey R. Atkinson'}, {'name': 'An...</td>\n",
       "      <td>&lt;p&gt;Biological aging is the strongest factor as...</td>\n",
       "      <td>2021-12-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.12.29.474495</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>DENA: training an authentic neural network mod...</td>\n",
       "      <td>[{'name': 'Xuan Li'}, {'name': 'Hang Qin'}, {'...</td>\n",
       "      <td>&lt;p&gt;Models developed using Nanopore direct RNA ...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.12.29.474482</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Rheology of crossbridge ensembles</td>\n",
       "      <td>[{'name': 'Khoi D Nguyen'}, {'name': 'Madhusud...</td>\n",
       "      <td>&lt;p&gt;How skeletal muscle responds to mechanical ...</td>\n",
       "      <td>2021-12-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.12.27.474302</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Perceptual uncertainty alternates top-down and...</td>\n",
       "      <td>[{'name': 'Kaho Tsumura'}, {'name': 'Reiko Shi...</td>\n",
       "      <td>&lt;p&gt;Response inhibition is a primary executive ...</td>\n",
       "      <td>2021-12-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.12.28.21268413</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>False</td>\n",
       "      <td>Increased persuadability and credulity in peop...</td>\n",
       "      <td>[{'name': 'Linda J Richards'}, {'name': 'Josep...</td>\n",
       "      <td>&lt;p&gt;Corpus callosum dysgenesis is one of the mo...</td>\n",
       "      <td>2021-12-29</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2021.12.28.21268436     382  medRxiv   \n",
       "1     10.1101/2021.12.25.474149      71  bioRxiv   \n",
       "2     10.1101/2021.12.29.474455      62  bioRxiv   \n",
       "3     10.1101/2021.12.29.474457      59  bioRxiv   \n",
       "4     10.1101/2021.12.30.474555      55  bioRxiv   \n",
       "5     10.1101/2021.12.29.474489      50  bioRxiv   \n",
       "6     10.1101/2021.12.28.474382      46  bioRxiv   \n",
       "7   10.1101/2021.12.23.21268179      44  medRxiv   \n",
       "8     10.1101/2021.12.28.474304      40  bioRxiv   \n",
       "9     10.1101/2021.12.29.474385      37  bioRxiv   \n",
       "10    10.1101/2021.12.30.474605      31  bioRxiv   \n",
       "11    10.1101/2021.12.28.474323      30  bioRxiv   \n",
       "12  10.1101/2021.12.30.21268540      29  medRxiv   \n",
       "13    10.1101/2021.12.27.474246      27  bioRxiv   \n",
       "14  10.1101/2021.12.24.21268378      23  medRxiv   \n",
       "15    10.1101/2021.12.30.474462      21  bioRxiv   \n",
       "16  10.1101/2021.12.24.21268385      20  medRxiv   \n",
       "17    10.1101/2021.12.28.474357      20  bioRxiv   \n",
       "18    10.1101/2021.12.28.474366      19  bioRxiv   \n",
       "19    10.1101/2021.12.29.473670      17  bioRxiv   \n",
       "20  10.1101/2021.12.30.21268538      15  medRxiv   \n",
       "21  10.1101/2021.12.27.21268442      15  medRxiv   \n",
       "22    10.1101/2021.12.29.474477      12  bioRxiv   \n",
       "23    10.1101/2021.12.28.474379      12  bioRxiv   \n",
       "24  10.1101/2021.12.31.21268575      12  medRxiv   \n",
       "25    10.1101/2021.12.26.474085      12  bioRxiv   \n",
       "26    10.1101/2021.12.24.474110      11  bioRxiv   \n",
       "27    10.1101/2021.12.29.474474      11  bioRxiv   \n",
       "28    10.1101/2021.12.27.474262      11  bioRxiv   \n",
       "29    10.1101/2021.12.24.474138      11  bioRxiv   \n",
       "30    10.1101/2021.12.30.474557      11  bioRxiv   \n",
       "31    10.1101/2021.12.27.474025      10  bioRxiv   \n",
       "32    10.1101/2021.12.29.474488      10  bioRxiv   \n",
       "33    10.1101/2021.12.31.474679       8  bioRxiv   \n",
       "34    10.1101/2021.12.30.474558       8  bioRxiv   \n",
       "35    10.1101/2021.12.29.474445       8  bioRxiv   \n",
       "36    10.1101/2021.12.29.474460       7  bioRxiv   \n",
       "37    10.1101/2021.12.29.474454       7  bioRxiv   \n",
       "38  10.1101/2021.12.23.21268244       7  medRxiv   \n",
       "39    10.1101/2021.12.27.473672       7  bioRxiv   \n",
       "40    10.1101/2021.12.28.474397       7  bioRxiv   \n",
       "41    10.1101/2021.12.27.474252       7  bioRxiv   \n",
       "42  10.1101/2021.12.29.21268505       6  medRxiv   \n",
       "43    10.1101/2021.12.29.474400       6  bioRxiv   \n",
       "44    10.1101/2022.01.01.474684       6  bioRxiv   \n",
       "45    10.1101/2021.12.26.473589       6  bioRxiv   \n",
       "46    10.1101/2021.12.29.474495       6  bioRxiv   \n",
       "47    10.1101/2021.12.29.474482       6  bioRxiv   \n",
       "48    10.1101/2021.12.27.474302       6  bioRxiv   \n",
       "49  10.1101/2021.12.28.21268413       6  medRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0   Infectious Diseases (except HIV/AIDS)   True   \n",
       "1                            Microbiology   True   \n",
       "2                   Developmental Biology  False   \n",
       "3                    Evolutionary Biology  False   \n",
       "4                          Bioengineering  False   \n",
       "5                            Neuroscience  False   \n",
       "6                            Cell Biology  False   \n",
       "7                            Epidemiology  False   \n",
       "8                            Neuroscience  False   \n",
       "9                                Genomics  False   \n",
       "10                           Neuroscience  False   \n",
       "11                      Synthetic Biology  False   \n",
       "12  Infectious Diseases (except HIV/AIDS)   True   \n",
       "13                           Neuroscience  False   \n",
       "14  Infectious Diseases (except HIV/AIDS)   True   \n",
       "15          Animal Behavior and Cognition  False   \n",
       "16           Genetic and Genomic Medicine  False   \n",
       "17                         Bioengineering  False   \n",
       "18                           Neuroscience  False   \n",
       "19                           Microbiology  False   \n",
       "20                           Epidemiology   True   \n",
       "21                              Neurology  False   \n",
       "22                           Neuroscience  False   \n",
       "23                           Biochemistry  False   \n",
       "24  Infectious Diseases (except HIV/AIDS)   True   \n",
       "25                           Microbiology   True   \n",
       "26                           Microbiology   True   \n",
       "27                  Developmental Biology  False   \n",
       "28                           Neuroscience  False   \n",
       "29                           Microbiology   True   \n",
       "30                           Neuroscience  False   \n",
       "31                             Physiology  False   \n",
       "32                             Biophysics  False   \n",
       "33                      Synthetic Biology  False   \n",
       "34                           Neuroscience  False   \n",
       "35                                Ecology  False   \n",
       "36                         Bioinformatics  False   \n",
       "37                             Immunology  False   \n",
       "38  Infectious Diseases (except HIV/AIDS)   True   \n",
       "39                           Microbiology  False   \n",
       "40                         Bioinformatics  False   \n",
       "41                      Synthetic Biology  False   \n",
       "42  Infectious Diseases (except HIV/AIDS)   True   \n",
       "43                             Biophysics  False   \n",
       "44                          Plant Biology  False   \n",
       "45                           Neuroscience  False   \n",
       "46                         Bioinformatics  False   \n",
       "47                             Biophysics  False   \n",
       "48                           Neuroscience  False   \n",
       "49                              Neurology  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Vaccine effectiveness against hospital admissi...   \n",
       "1   Expanded ACE2 dependencies of diverse SARS-lik...   \n",
       "2   Nerve-mediated amputation-induced stem cell ac...   \n",
       "3   Effects of periodic bottlenecks on the dynamic...   \n",
       "4   A nanopore interface for high bandwidth DNA co...   \n",
       "5   High density single-unit human cortical record...   \n",
       "6   Precise measurement of nanoscopic septin ring ...   \n",
       "7   An online atlas of human plasma metabolite sig...   \n",
       "8   Techniques for visualizing fibroblast-vessel i...   \n",
       "9   Single-nuclei isoform RNA sequencing reveals c...   \n",
       "10  Roles for Mitochondrial Complex I subunits in ...   \n",
       "11  A versatile active learning workflow for optim...   \n",
       "12  Magnitude and breadth of neutralizing antibody...   \n",
       "13  Species-specific mitochondria dynamics and met...   \n",
       "14  Preserved Omicron Spike specific antibody bind...   \n",
       "15  The evolution and biological correlates of han...   \n",
       "16  Schizophrenia-associated somatic copy number v...   \n",
       "17  A Musculoskeletal Model of the Hand and Wrist ...   \n",
       "18  The mitochondrial calcium uniporter regulates ...   \n",
       "19  Reconstitution of the S. aureus agr quorum sen...   \n",
       "20  Protective effect of BNT162b2 vaccination on a...   \n",
       "21  Brain atrophy in prodromal synucleinopathy is ...   \n",
       "22  Optogenetic inhibition of the dorsal hippocamp...   \n",
       "23  FastCAT accelerates absolute quantification of...   \n",
       "24  COVID-19 severity prediction based on patient ...   \n",
       "25  SARS-CoV-2 Omicron-B.1.1.529 Variant leads to ...   \n",
       "26  Reduced infectivity but increased immune escap...   \n",
       "27  A NEUROGENIC GENE EXPRESSION SIGNATURE SUPPORT...   \n",
       "28  Uncovering a neural circuit controlling adult ...   \n",
       "29  Homologous or Heterologous Booster of Inactiva...   \n",
       "30  Properties and proximity proteomics of synapto...   \n",
       "31  Integrated cytometry with machine learning app...   \n",
       "32  Structural insights into the ion selectivity o...   \n",
       "33  PlasmidMaker: a Versatile, Automated, and High...   \n",
       "34  BDNF-induced BDNF release mediates presynaptic...   \n",
       "35  Nipah virus detection at bat roosts following ...   \n",
       "36  Protein interaction analysis of Plasmodium fal...   \n",
       "37  Longitudinal analysis of invariant natural kil...   \n",
       "38  Effectiveness of casirivimab and imdevimab, an...   \n",
       "39  Effector-mediated subversion of proteasome act...   \n",
       "40  Comparative analysis of Lysine and Arginine bi...   \n",
       "41  Dichotomous Feedback: A Signal Sequestration-b...   \n",
       "42  Head-to-head comparison of nasal and nasophary...   \n",
       "43  Extreme value analysis of intracellular cargo ...   \n",
       "44  Concerted expansion and contraction of immune ...   \n",
       "45  Biological Aging of CNS-Resident Cells Alters ...   \n",
       "46  DENA: training an authentic neural network mod...   \n",
       "47                  Rheology of crossbridge ensembles   \n",
       "48  Perceptual uncertainty alternates top-down and...   \n",
       "49  Increased persuadability and credulity in peop...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Glenda E Gray'}, {'name': 'Shirley ...   \n",
       "1   [{'name': 'Sarah M Roelle'}, {'name': 'Nidhi S...   \n",
       "2   [{'name': 'Duygu Payzin-Dogru'}, {'name': 'Sar...   \n",
       "3   [{'name': 'Minako Izutsu'}, {'name': 'Devin M....   \n",
       "4   [{'name': 'Karen Zhang'}, {'name': 'Yuan-Jyue ...   \n",
       "5   [{'name': 'Jason E Chung'}, {'name': 'Kristin ...   \n",
       "6   [{'name': 'Amin Zehtabian'}, {'name': 'Paul Ma...   \n",
       "7   [{'name': 'Koen F. Dekkers'}, {'name': 'Sergi ...   \n",
       "8   [{'name': 'Hannah E Jones'}, {'name': 'Kelsey ...   \n",
       "9   [{'name': 'Simon A Hardwick'}, {'name': 'Wen H...   \n",
       "10  [{'name': 'Bhagaban Mallik'}, {'name': 'C. And...   \n",
       "11  [{'name': 'Amir Pandi'}, {'name': 'Christoph D...   \n",
       "12  [{'name': 'Benjamin L. Sievers'}, {'name': 'Sa...   \n",
       "13  [{'name': 'Ryohei Iwata'}, {'name': 'Pierre Ca...   \n",
       "14  [{'name': 'Yannic Bartsch'}, {'name': 'Xin Ton...   \n",
       "15  [{'name': 'Kai R Caspar'}, {'name': 'Fabian Pa...   \n",
       "16  [{'name': 'Eduardo A Maury'}, {'name': 'Maxwel...   \n",
       "17  [{'name': 'Daniel Clinton McFarland'}, {'name'...   \n",
       "18  [{'name': 'Katy E Pannoni'}, {'name': 'Daniela...   \n",
       "19  [{'name': 'Aishan Zhao'}, {'name': 'Steven Par...   \n",
       "20  [{'name': 'Yair Blumberg'}, {'name': 'Michael ...   \n",
       "21  [{'name': 'Shady Rahayel'}, {'name': 'Christin...   \n",
       "22  [{'name': 'Shuyi Qi'}, {'name': 'Shi Min Tan'}...   \n",
       "23  [{'name': 'Ignacy Rzagalinski'}, {'name': 'Ali...   \n",
       "24  [{'name': 'Ariel Israel'}, {'name': 'Alejandro...   \n",
       "25  [{'name': 'Eleanor G Bentley'}, {'name': 'Adam...   \n",
       "26  [{'name': 'Jie Hu'}, {'name': 'Pai Peng'}, {'n...   \n",
       "27  [{'name': 'Javier Solivan-Rivera'}, {'name': '...   \n",
       "28  [{'name': 'Moawiah Naffaa'}, {'name': 'Rehan K...   \n",
       "29  [{'name': 'Xun Wang'}, {'name': 'Xiaoyu Zhao'}...   \n",
       "30  [{'name': 'Hanieh Falahati'}, {'name': 'Yumei ...   \n",
       "31  [{'name': 'Seth Winfree'}, {'name': 'Andrew T ...   \n",
       "32  [{'name': 'Xinyu Teng'}, {'name': 'Danqi Sheng...   \n",
       "33  [{'name': 'Behnam Enghiad'}, {'name': 'Pu Xue'...   \n",
       "34  [{'name': 'Coralie Berthoux'}, {'name': 'Kaout...   \n",
       "35  [{'name': 'Clifton D McKee'}, {'name': 'Ausraf...   \n",
       "36  [{'name': 'Cheikh Cambel Dieng'}, {'name': 'Co...   \n",
       "37  [{'name': 'Lydia Lynch'}, {'name': 'Harry Kane...   \n",
       "38  [{'name': 'David T. Huang'}, {'name': 'Erin K....   \n",
       "39  [{'name': 'Tshegofatso Ngwaga'}, {'name': 'Dee...   \n",
       "40  [{'name': 'Sankar Mahesh'}, {'name': 'Deepa Se...   \n",
       "41  [{'name': 'Aivar Sootla'}, {'name': 'Nicolas D...   \n",
       "42  [{'name': 'Niklaus Daniel Labhardt'}, {'name':...   \n",
       "43  [{'name': 'Takuma Naoi'}, {'name': 'Yuki Kagaw...   \n",
       "44  [{'name': 'Bruno Pok Man Ngou'}, {'name': 'Rob...   \n",
       "45  [{'name': 'Jeffrey R. Atkinson'}, {'name': 'An...   \n",
       "46  [{'name': 'Xuan Li'}, {'name': 'Hang Qin'}, {'...   \n",
       "47  [{'name': 'Khoi D Nguyen'}, {'name': 'Madhusud...   \n",
       "48  [{'name': 'Kaho Tsumura'}, {'name': 'Reiko Shi...   \n",
       "49  [{'name': 'Linda J Richards'}, {'name': 'Josep...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>Following the results of the ENSEMBLE 2 stu...  2021-12-29  \n",
       "1   <p>Viral spillover from animal reservoirs can ...  2021-12-27  \n",
       "2   <p>Animals exhibit extreme diversity in regene...  2021-12-30  \n",
       "3   <p>Population bottlenecks are common in nature...  2021-12-30  \n",
       "4   <p>DNA has emerged as a powerful substrate for...  2021-12-30  \n",
       "5   <p>A fundamental unit of neural computation is...  2021-12-30  \n",
       "6   <p>The combination of image analysis and fluor...  2021-12-29  \n",
       "7   <p>The human gut microbiota produces a variety...  2021-12-30  \n",
       "8   <p>Fibroblasts are found associated with blood...  2021-12-28  \n",
       "9   <p>Single-nuclei RNA-Seq is being widely emplo...  2021-12-30  \n",
       "10  <p>To identify conserved components of synapse...  2022-01-01  \n",
       "11  <p>The study, engineering and application of b...  2021-12-28  \n",
       "12  <p>Multiple SARS-CoV-2 variants that possess m...  2022-01-01  \n",
       "13  <p>The evolution of species involves changes i...  2021-12-27  \n",
       "14  <p>Despite the dramatic spread of Omicron glob...  2021-12-27  \n",
       "15  <p>The evolution of human right-handedness has...  2021-12-30  \n",
       "16  <p>While inherited and de novo copy number var...  2022-01-01  \n",
       "17  <p>Objective: The purpose of this work was to ...  2021-12-28  \n",
       "18  <p>CA2 is an understudied subregion of the hip...  2021-12-28  \n",
       "19  <p>In Staphylococcus aureus, virulence is unde...  2021-12-30  \n",
       "20  <p>There is increasing evidence that patients ...  2022-01-01  \n",
       "21  <p>Isolated REM sleep behaviour disorder (iRBD...  2021-12-30  \n",
       "22  <p>The dorsal hippocampus (DH) is key to the l...  2021-12-30  \n",
       "23  <p>Absolute (molar) quantification of proteins...  2021-12-29  \n",
       "24  <p>Background\\nVaccines are highly effective i...  2022-01-01  \n",
       "25  <p>COVID-19 is a spectrum of clinical symptoms...  2021-12-28  \n",
       "26  <p>A new detected SARS-CoV-2 variant Omicron (...  2021-12-27  \n",
       "27  <p>Human beige/brite thermogenic adipose tissu...  2021-12-30  \n",
       "28  <p>The maintenance and differentiation of the ...  2021-12-27  \n",
       "29  <p>The massive and rapid transmission of SARS-...  2021-12-27  \n",
       "30  <p>The spine apparatus is a specialization of ...  2021-12-30  \n",
       "31  <p>The human kidney is a complex organ with va...  2021-12-28  \n",
       "32  <p>MgtE is a Mg2+-selective ion channel whose ...  2021-12-30  \n",
       "33  <p>Plasmids are used extensively in basic and ...  2022-01-02  \n",
       "34  <p>Although brain-derived neurotrophic factor ...  2021-12-30  \n",
       "35  <p>Knowledge of the dynamics and genetic diver...  2021-12-30  \n",
       "36  <p>The world's first malaria vaccine RTS,S pro...  2021-12-30  \n",
       "37  <p>Innate T cells, including CD1d-restricted i...  2021-12-31  \n",
       "38  <sec><title>IMPORTANCE</title><p>The effective...  2021-12-27  \n",
       "39  <p>Legionella pneumophila causes Legionnaires'...  2021-12-27  \n",
       "40  <p>The members of the Deinococcaceae family ha...  2021-12-29  \n",
       "41  <p>We introduce a new design framework for imp...  2021-12-27  \n",
       "42  <p>Objectives\\nTo assess the real-world diagno...  2022-01-01  \n",
       "43  <p>In the long axon of a neuron, cargo transpo...  2021-12-30  \n",
       "44  <p>Recent reports suggest that cell-surface an...  2022-01-02  \n",
       "45  <p>Biological aging is the strongest factor as...  2021-12-27  \n",
       "46  <p>Models developed using Nanopore direct RNA ...  2021-12-30  \n",
       "47  <p>How skeletal muscle responds to mechanical ...  2021-12-30  \n",
       "48  <p>Response inhibition is a primary executive ...  2021-12-28  \n",
       "49  <p>Corpus callosum dysgenesis is one of the mo...  2021-12-29  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "76 preprints (including 13 covering SARS-CoV-2, 62 from bioRxiv and 14 from medRxiv) published in the last 7 days before 2022-01-03 had been tweeted at least 3 times (maximum 382)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.9"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
