{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28505 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7ed9e3d1-fc1f-447a-b7ca-6fde4577d334&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=94591669-2c17-400f-b198-4d65812e8395&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a784bbf9-827a-49c9-bd39-819ec85627f4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2d53e2d1-4844-419d-94f0-6fdb2c2e27c8&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=131253b9-17f8-4752-9638-b27e4067191f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=55de930c-bd36-4b69-8719-26618877c56e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e15d3b12-239e-470c-96a4-6329a2737d1e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2fef2c41-2af3-4e08-9491-d7a38cdfe362&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=921b9ea4-b784-41e5-8ec6-7fbe00342d87&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=55fdab98-8cf7-4310-9cdc-906faf3e9f3a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e91c1d45-6959-4f41-9149-c3c6c9276f9a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e1a7e0d3-9e32-45fe-9e22-790d6559c177&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=1335d2f0-b359-44a0-a31f-4552d1feab4e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=0dc54c33-ef03-4416-8845-17b8a660abd7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4042d80a-df73-4234-9848-f837a37b09f3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a4454ee5-d1d6-4956-bf51-6544cccd2bef&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=bcdb8c9a-ffc7-4d96-aaa6-50a5cf46a9c6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9e869c16-33af-4bd2-ba10-e47da6b82d07&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0018.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2c622f22-0c05-45e1-9e04-edaca5df70c8&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0019.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=dda904a3-f207-4155-b679-6158f5c2e4d9&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0020.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=32e5e61e-d21f-4c05-88ff-cc5592998aa3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0021.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5ebbc14f-f89d-422a-acef-90ebea570f9a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0022.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2ebecd92-d112-48f1-bd9f-6583b60b3e33&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0023.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7109f37e-a564-4095-a457-c190a086a2e6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0024.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=948b0090-f464-42ab-8e9a-a46ff19d0329&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0025.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d55058db-82e6-4345-be78-dbd2437747b2&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0026.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=30a13a05-419c-4161-ba15-6e70a82bc172&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0027.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a3de2446-b9df-4e81-817b-9a67535b2781&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2022-01-11&until-occurred-date=2022-01-18\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0028.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-10-18.csv\n",
      "failed to load preprint_tweets_2021-10-25.csv\n",
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-12-20.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load preprint_tweets_2021-12-27.csv\n",
      "failed to load preprint_tweets_2021-10-23.csv\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-11-01.csv\n",
      "failed to load preprint_tweets_2021-11-15.csv\n",
      "failed to load preprint_tweets_2022-01-18.csv\n",
      "failed to load preprint_tweets_2021-11-29.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2022-01-03.csv\n",
      "failed to load .tweets.json.icloud\n",
      "failed to load preprint_tweets_2021-11-08.csv\n",
      "failed to load preprint_tweets_2022-01-10.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load preprint_tweets_2021-11-22.csv\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-12-06.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-12-13.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.12.20.21267966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.05.22268800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.22269010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.18.21268018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475305...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21265796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.31.474653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.18.20021881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21263608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.20.106401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.18.101717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472585...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/142554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.08.22268954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.09.22268936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.21266068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475793...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476251...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.21268130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267162...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.21268373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.470764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476171...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476155...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475641...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.30.21267048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.08.475498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475860...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.21.21253968...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.21258895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.21268586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.205534...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21268008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21268007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22269288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.21267203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471446...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.21267633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476191...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.475546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.22268755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.21267114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21266377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21267146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22268989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475870...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.475871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.21268382...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.09.475560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.465626...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.474936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.21268560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21267853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.21268021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.14.095430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.02.474743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265968...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475947...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.475930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.01.20144030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476279...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.22269008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.427194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.471045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475842...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.20156018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.475944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473118...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.21265038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.29.424619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21252379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.24.436523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.475507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.21257505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.09.475567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.474735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.05.474142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476216...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.21267432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.475527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473899...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22268736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.22269234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.12.20151316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.472081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.471005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.441366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.13.435275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.21268436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20043919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.21268134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22269328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.21268460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.471325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.21267985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.471245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476162...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.388819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21265753...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476125...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.21268159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266824...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475257...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468476...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.22269318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476347...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.22.469588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475836...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476238...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.444556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.474085...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.472683...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475153...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.09.475576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.475240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.475329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.03.474825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.28.470226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475336...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473623...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.25.469987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.22269176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.425971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.08.475501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476178...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.473633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.22269229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22269287...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.21268103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.22269035...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475638...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475665...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.05.475139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.366062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21267350...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.17.156752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275533.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468547...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.05.475103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22269295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.28.21268391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.21267961...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.04.21267300...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.05.475140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.429898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265397...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.17.255166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.09.20245175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472383...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.22268948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473890...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.17.473223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.475903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.09.475544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.475352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22269323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.20.884171...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.475985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21261243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.22268812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.22269022...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465190...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21266055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.475405...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.29.21267006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.26.400630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475238...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.05.187542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473454...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.472668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475251...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.21267898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20087965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.04.367730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.21267417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.05.21263141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.475850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476444...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.05.475072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.474160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.21267198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475221...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.17.476578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476060...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.21267194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.442916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22269108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.444774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.21268171...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.11.21267660...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.24.252296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.03.474708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.474522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.475824...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.21268254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476201...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.22268799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.05.475028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.21256639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.05.475094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.475214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453263...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.26.474183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.463740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.473016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259218...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.22.914952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/154724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/110197...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.21268070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275981.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21268319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/689851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.470882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.09.471910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.29.20085449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.17.475672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475465...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.21268285...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.447441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.21251526...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.422529...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.25.472549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476151...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.15.275891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.22268729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.20248579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476280...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22269064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.428114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.22269000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.22269137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.474044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.21268270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.433266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.17.476647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.429164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.01.20241653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.21258272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.21251264...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.12.472315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.462725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.21264461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.23.21266775...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.16.472981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.21252091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.226035.117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/148296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265945...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.20.427409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/169045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452118...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452280...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.423933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.21266779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.448703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476185...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.472838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/117143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.21268536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21266457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.05.471237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.473393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.442293...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465197...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.453231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22269182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.21257857...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.24.474119...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.29.442018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.475960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.20162420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464280...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476360...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.21266350...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.283036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.451353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256931...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.20.436243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.20226894...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.21267352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.22269024...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.435979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.02.471004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.384594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265216...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460099...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21266112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.08.475494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.473349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476336...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.473367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473928...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476462...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.437048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.22269373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21267155...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.30.474527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.475321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.02.454829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464753...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.27.474232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458273...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.06.475101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.475006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.03.22268599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.07.22268907...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.03.474812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.29.474488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.474416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.04.474988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.473972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.426000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.29.470378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.17.476579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.30.21267047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.472114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.21267115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.27.063552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.21267132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.01.470818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.29.399923...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.395574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.426861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.22.21250304...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21266419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.09.032995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.21267471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.22269267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.29.123497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.24.469537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.21251745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.475748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.12.430975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.12.21251659...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.475833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.21250670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.21250840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.21250957...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.371526...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21266857...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/507590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.093195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.471455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.14.095414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.01.20144253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.07.470392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.19.158717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.08.471717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.14.150490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473558...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.18.210161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.21267240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.03.470962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476281...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.476286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.22268980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.20209114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.10.21267372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.475644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.06.21266524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.20.259937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.21.473663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.14.476310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.16.476512...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.07.20056424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.13.22269272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.09.940510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.22269075...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21266069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.447249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.21258049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/191288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.07.467645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.21252528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.442232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/159111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.451077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.451222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/053033...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.19.470191...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/098061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/116962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.10.475716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.21259544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/111393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.473296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.23.21258377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.23.471999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475884...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.15.476461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475974...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439793...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.09.20033217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.475264...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.21266229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476046...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.22.473872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.21266627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.20.473457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.15.21267849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.11.475900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.441302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.13.472469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.12.14.21267713...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2022.01.12.476135...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2022.01.10.22269010', 'tweets': 2384, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Infectious viral load in unvaccinated and vaccinated patients infected with SARS-CoV-2 WT, Delta and Omicron', 'authors': \"[{'name': 'Olha Puhach'}, {'name': 'Kenneth Adea'}, {'name': 'Nicolas Hulo'}, {'name': 'Pascale Sattonnet'}, {'name': 'Camille Genecand'}, {'name': 'Anne Iten'}, {'name': 'Frédérique Jacquérioz Bausch'}, {'name': 'Laurent Kaiser'}, {'name': 'Pauline Vetter'}, {'name': 'Isabella Eckerle'}, {'name': 'Benjamin Meyer'}]\", 'abstract': '<sec><title>Background</title><p>Viral load (VL) is one determinant of secondary transmission of SARS-CoV-2. Emergence of variants of concerns (VOC) Alpha and Delta was ascribed, at least partly, to higher VL. Furthermore, with parts of the population vaccinated, knowledge on VL in vaccine breakthrough infections is crucial. As RNA VL is only a weak proxy for infectiousness, studies on infectious virus presence by cell culture isolation are of importance.</p></sec><sec><title>Methods</title><p>We assessed nasopharyngeal swabs of COVID-19 patients for quantitative infectious viral titres (IVT) by focus-forming assay and compared to overall virus isolation success and RNA genome copies. We assessed infectious viral titres during the first 5 symptomatic days in a total of 384 patients: unvaccinated individuals infected with pre-VOC SARS-CoV-2 (n= 118) or Delta (n= 127) and vaccine breakthrough infections with Delta (n= 121) or Omicron (n=18).</p></sec><sec><title>Findings</title><p>Correlation between RNA copy number and IVT was low for all groups. No correlation between IVTs and age or sex was seen. We observed higher RNA genome copies in pre-VOC SARS-CoV-2 compared to Delta, but significantly higher IVTs in Delta infected individuals. In vaccinated vs. unvaccinated Delta infected individuals, RNA genome copies were comparable but vaccinated individuals have significantly lower IVTs, and cleared virus faster. Vaccinated individuals with Omicron infection had comparable IVTs to Delta breakthrough infections.</p></sec><sec><title>Interpretation</title><p>Quantitative IVTs can give detailed insights into virus shedding kinetics. Vaccination was associated with lower infectious titres and faster clearance for Delta, showing that vaccination would also lower transmission risk. Omicron vaccine breakthrough infections did not show elevated IVTs compared to Delta, suggesting that other mechanisms than increase VL contribute to the high infectiousness of Omicron.</p></sec><sec><title>Funding</title><p>This work was supported by the Swiss National Science Foundation 196644, 196383, NRP (National Research Program) 78 Covid-19 Grant 198412, the Fondation Ancrage Bienfaisance du Groupe Pictet and the Fondation Privée des Hôpitaux Universitaires de Genève.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.12.476031', 'tweets': 368, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'The SARS-CoV-2 Omicron (B.1.1.529) variant exhibits altered pathogenicity, transmissibility, and fitness in the golden Syrian hamster model', 'authors': \"[{'name': 'Shuofeng Yuan'}, {'name': 'Zi-Wei Ye'}, {'name': 'Ronghui Liang'}, {'name': 'Kaiming Tang'}, {'name': 'Anna Jinxia Zhang'}, {'name': 'Gang Lu'}, {'name': 'Chon Phin Ong'}, {'name': 'Vincent Kwok-Man Poon'}, {'name': 'Chris Chung-Sing Chan'}, {'name': 'Bobo Wing-Yee Mok'}, {'name': 'Zhenzhi Qin'}, {'name': 'Yubin Xie'}, {'name': 'Haoran Sun'}, {'name': 'Jessica Oi-Ling Tsang'}, {'name': 'Terrence Tsz-Tai Yuen'}, {'name': 'Kenn Ka-Heng Chik'}, {'name': 'Chris Chun-Yiu Chan'}, {'name': 'Jian-Piao Cai'}, {'name': 'Cuiting Luo'}, {'name': 'Lu Lu'}, {'name': 'Cyril Chik-Yan Yip'}, {'name': 'Hin Chu'}, {'name': 'Kelvin Kai-Wang To'}, {'name': 'Honglin Chen'}, {'name': 'Dong-Yan Jin'}, {'name': 'Kwok-Yung Yuen'}, {'name': 'Jasper Fuk-Woo Chan'}]\", 'abstract': '<p>The newly emerging SARS-CoV-2 Omicron (B.1.1.529) variant first identified in South Africa in November 2021 is characterized by an unusual number of amino acid mutations in its spike that renders existing vaccines and therapeutic monoclonal antibodies dramatically less effective. The <italic>in vivo</italic> pathogenicity, transmissibility, and fitness of this new Variant of Concerns are unknown. We investigated these virological attributes of the Omicron variant in comparison with those of the currently dominant Delta (B.1.617.2) variant in the golden Syrian hamster COVID-19 model. Omicron-infected hamsters developed significantly less body weight losses, clinical scores, respiratory tract viral burdens, cytokine/chemokine dysregulation, and tissue damages than Delta-infected hamsters. The Omicron and Delta variant were both highly transmissible (100% vs 100%) via contact transmission. Importantly, the Omicron variant consistently demonstrated about 10-20% higher transmissibility than the already-highly transmissible Delta variant in repeated non-contact transmission studies (overall: 30/36 vs 24/36, 83.3% vs 66.7%). The Delta variant displayed higher fitness advantage than the Omicron variant without selection pressure in both <italic>in vitro</italic> and <italic>in vivo</italic> competition models. However, this scenario drastically changed once immune selection pressure with neutralizing antibodies active against the Delta variant but poorly active against the Omicron variant were introduced, with the Omicron variant significantly outcompeting the Delta variant. Taken together, our findings demonstrated that while the Omicron variant is less pathogenic than the Delta variant, it is highly transmissible and can outcompete the Delta variant under immune selection pressure. Next-generation vaccines and antivirals effective against this new VOC are urgently needed.</p><sec><title>One Sentence Summary</title><p>The novel SARS-CoV-2 Omicron variant, though less pathogenic, is highly transmissible and outcompetes the Delta variant under immune selection pressure in the golden Syrian hamster COVID-19 model.</p></sec>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.14.476225', 'tweets': 104, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Single-cell multi-omics of human clonal hematopoiesis reveals that DNMT3A R882 mutations perturb early progenitor states through selective hypomethylation', 'authors': \"[{'name': 'Anna S. Nam'}, {'name': 'Neville Dusaj'}, {'name': 'Franco Izzo'}, {'name': 'Rekha Murali'}, {'name': 'Robert M. Myers'}, {'name': 'Tarek Mouhieddine'}, {'name': 'Jesus Sotelo'}, {'name': 'Salima Benbarche'}, {'name': 'Michael Waarts'}, {'name': 'Federico Gaiti'}, {'name': 'Sabrin Tahri'}, {'name': 'Ross Levine'}, {'name': 'Omar Abdel-Wahab'}, {'name': 'Lucy A. Godley'}, {'name': 'Ronan Chaligne'}, {'name': 'Irene M Ghobrial'}, {'name': 'Dan Landau'}]\", 'abstract': '<p>Somatic mutations in cancer genes have been ubiquitously detected in clonal expansions across healthy human tissue, including in clonal hematopoiesis. However, mutated and wildtype cells are morphologically and phenotypically similar, limiting the ability to link genotypes with cellular phenotypes. To overcome this limitation, we leveraged multi-modality single-cell sequencing, capturing the mutation with transcriptomes and methylomes in stem and progenitors from individuals with <italic>DNMT3A</italic> R882 mutated clonal hematopoiesis. <italic>DNMT3A</italic> mutations resulted in myeloid over lymphoid bias, and in expansion of immature myeloid progenitors primed toward megakaryocytic-erythroid fate. We observed dysregulated expression of lineage and leukemia stem cell markers. DNMT3A R882 led to preferential hypomethylation of polycomb repressive complex 2 targets and a specific sequence motif. Notably, the hypomethylation motif is enriched in binding motifs of key hematopoietic transcription factors, serving as a potential mechanistic link between <italic>DNMT3A</italic> R882 mutations and aberrant transcriptional phenotypes. Thus, single-cell multi-omics pave the road to defining the downstream consequences of mutations that drive human clonal mosaicism.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.11.475838', 'tweets': 93, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Lightweight compositional analysis of metagenomes with FracMinHash and minimum metagenome covers', 'authors': \"[{'name': 'Luiz Carlos Irber'}, {'name': 'Phillip T Brooks'}, {'name': 'Taylor E Reiter'}, {'name': 'N Tessa Pierce-Ward'}, {'name': 'Mahmudur Rahman Hera'}, {'name': 'David Koslicki'}, {'name': 'C. Titus Brown'}]\", 'abstract': '<p>The identification of reference genomes and taxonomic labels from metagenome data underlies many microbiome studies. Here we describe two algorithms for compositional analysis of metagenome sequencing data. We first investigate the FracMinHash sketching technique, a derivative of modulo hash that supports Jaccard containment estimation between sets of different sizes. We implement FracMinHash in the sourmash software, evaluate its accuracy, and demonstrate large-scale containment searches of metagenomes using 700,000 microbial reference genomes. We next frame shotgun metagenome compositional analysis as the problem of finding a minimum collection of reference genomes that \"cover\" the known k-mers in a metagenome, a minimum set cover problem. We implement a greedy approximate solution using FracMinHash sketches, and evaluate its accuracy for taxonomic assignment using a CAMI community benchmark. Finally, we show that the minimum metagenome cover can be used to guide the selection of reference genomes for read mapping. sourmash is available as open source software under the BSD 3-Clause license at github.com/dib-lab/sourmash/.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475793', 'tweets': 88, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Global monitoring of soil animal communities using a common methodology', 'authors': \"[{'name': 'Anton M. Potapov'}, {'name': 'Xin Sun'}, {'name': 'Maria J. I. Briones'}, {'name': 'George G. Brown'}, {'name': 'Erin K. Cameron'}, {'name': 'Jérôme Cortet'}, {'name': 'Nico Eisenhauer'}, {'name': 'Saori Fujii'}, {'name': 'Stefan Geisen'}, {'name': 'Konstantin B. Gongalsky'}, {'name': 'Carlos Guerra'}, {'name': 'I.Tanya Handa'}, {'name': 'Charlene Janion-Scheepers'}, {'name': 'Zoë Lindo'}, {'name': 'Jérôme Mathieu'}, {'name': 'Maka Murvanidze'}, {'name': 'Uffe N Nielsen'}, {'name': 'Stefan Scheu'}, {'name': 'Olaf Schmidt'}, {'name': 'Clement Schneider'}, {'name': 'Julia Seeber'}, {'name': 'Jiri Tuma'}, {'name': 'Alexei V. Tiunov'}, {'name': 'Andrey S. Zaitsev'}, {'name': 'Diana H. Wall'}]\", 'abstract': '<p>Here we introduce the Soil BON Foodweb Team, a cross-continental collaborative network that aims to monitor soil animal communities and food webs using consistent methodology at a global scale. Soil animals support vital soil processes via soil structure modification, direct consumption of dead organic matter, and interactions with microbial and plant communities. Soil animal effects on ecosystem functions have been demonstrated by correlative analyses as well as in laboratory and field experiments, but these studies typically focus on selected animal groups or species at one or few sites with limited variation in environmental conditions. The lack of comprehensive harmonised large-scale soil animal community data including microfauna, mesofauna, and macrofauna, in conjunction with related soil functions, limits our understanding of biological interactions in soil communities and how these interactions affect ecosystem functioning. To provide such data, the Soil BON Foodweb Team invites researchers worldwide to use a common methodology to address six long-term goals: (1) to collect globally representative harmonised data on soil micro-, meso-, and macrofauna communities; (2) to describe key environmental drivers of soil animal communities and food webs; (3) to assess the efficiency of conservation approaches for the protection of soil animal communities; (4) to describe soil food webs and their association with soil functioning globally; (5) to establish a global research network for soil biodiversity monitoring and collaborative projects in related topics; (6) to reinforce local collaboration networks and expertise and support capacity building for soil animal research around the world. In this paper, we describe the vision of the global research network and the common sampling protocol to assess soil animal communities and advocate for the use of standard methodologies across observational and experimental soil animal studies. We will use this protocol to conduct soil animal assessments and reconstruct soil food webs on the sites included in the global soil biodiversity monitoring network, Soil BON, allowing us to assess linkages among soil biodiversity, vegetation, soil physico-chemical properties, and ecosystem functions. In the present paper, we call for researchers especially from countries and ecoregions that remain underrepresented in the majority of soil biodiversity assessments to join us. Together we will be able to provide science-based evidence to support soil biodiversity conservation and functioning of terrestrial ecosystems.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476251', 'tweets': 84, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Mixing genome annotation methods in a comparative analysis inflates the apparent number of lineage-specific genes', 'authors': \"[{'name': 'Caroline M. Weisman'}, {'name': 'Andrew M. Murray'}, {'name': 'Sean R Eddy'}]\", 'abstract': \"<p>Comparisons of genomes of different species are used to identify lineage-specific genes, those genes that appear unique to one species or clade. Lineage-specific genes are often thought to represent genetic novelty that underlies unique adaptations. Identification of these genes depends not only on genome sequences, but also on inferred gene annotations. Comparative analyses typically use available genomes that have been annotated using different methods, increasing the risk that orthologous DNA sequences may be erroneously annotated as a gene in one species but not another, appearing lineage-specific as a result. To evaluate the impact of such 'annotation heterogeneity', we identified four clades of species with sequenced genomes with more than one publicly available gene annotation, allowing us to compare the number of lineage-specific genes inferred when differing annotation methods are used to those resulting when annotation method is uniform across the clade. In these case studies, annotation heterogeneity increases the apparent number of lineage-specific genes by up to 15-fold, suggesting that annotation heterogeneity is a substantial source of potential artifact.</p>\", 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.11.475254', 'tweets': 76, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Identifying and correcting repeat-calling errors in nanopore sequencing of telomeres', 'authors': \"[{'name': 'Kar-Tong Tan'}, {'name': 'Michael K. Slevin'}, {'name': 'Matthew Meyerson'}, {'name': 'Heng Li'}]\", 'abstract': '<p>Nanopore long-read genome sequencing is emerging as a potential approach for the study of genomes including long repetitive elements like telomeres. Here, we report extensive basecalling induced errors at telomere repeats across nanopore datasets, sequencing platforms, basecallers, and basecalling models. We found that telomeres which are represented by (TTAGGG)<sub>n</sub> and (CCCTAA)<sub>n</sub> repeats in many organisms were frequently miscalled (~40-50% of reads) as (TTAAAA)<sub>n</sub>, or as (CTTCTT)<sub>n</sub> and (CCCTGG)<sub>n</sub> repeats respectively in a strand-specific manner during nanopore sequencing. We showed that this miscalling is likely caused by the high similarity of current profiles between telomeric repeats and these repeat artefacts, leading to mis-assignment of electrical current profiles during basecalling. We further demonstrated that tuning of nanopore basecalling models, and selective application of the tuned models to telomeric reads led to improved recovery and analysis of telomeric regions, with little detected negative impact on basecalling of other genomic regions. Our study thus highlights the importance of verifying nanopore basecalls in long, repetitive, and poorly defined regions of the genome, and showcases how such artefacts in regions like telomeres can potentially be resolved by improvements in nanopore basecalling models.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475719', 'tweets': 74, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Learning in reverse: Dopamine errors drive excitatory and inhibitory components of backward conditioning in an outcome-specific manner', 'authors': \"[{'name': 'Benjamin M. Seitz'}, {'name': 'Ivy B. Hoang'}, {'name': 'Aaron P. Blaisdell'}, {'name': 'Melissa J. Sharpe'}]\", 'abstract': '<p>For over two decades, midbrain dopamine was considered synonymous with the prediction error in temporal-difference reinforcement learning. Central to this proposal is the notion that reward-predictive stimuli become endowed with the scalar value of predicted rewards. When these cues are subsequently encountered, their predictive value is compared to the value of the actual reward received allowing for the calculation of prediction errors. Phasic firing of dopamine neurons was proposed to reflect this computation, facilitating the backpropagation of value from the predicted reward to the reward-predictive stimulus, thus reducing future prediction errors. There are two critical assumptions of this proposal: 1) that dopamine errors can only facilitate learning about scalar value and not more complex features of predicted rewards, and 2) that the dopamine signal can only be involved in anticipatory learning in which cues or actions precede rewards. Recent work has challenged the first assumption, demonstrating that phasic dopamine signals across species are involved in learning about more complex features of the predicted outcomes, in a manner that transcends this value computation. Here, we tested the validity of the second assumption. Specifically, we examined whether phasic midbrain dopamine activity would be necessary for backward conditioning—when a neutral cue reliably <italic>follows</italic> a rewarding outcome. Using a specific Pavlovian-to-Instrumental Transfer (PIT) procedure, we show rats learn both excitatory and inhibitory components of a backward association, and that this association entails knowledge of the specific identity of the reward and cue. We demonstrate that brief optogenetic inhibition of VTA<sub>DA</sub> neurons timed to the transition between the reward and cue, reduces both of these components of backward conditioning. These findings suggest VTA<sub>DA</sub> neurons are capable of facilitating associations between contiguously occurring events, regardless of the content of those events. We conclude that these data are in line with suggestions that the VTA<sub>DA</sub> error acts as a universal teaching signal. This may provide insight into why dopamine function has been implicated in a myriad of psychological disorders that are characterized by very distinct reinforcement-learning deficits.</p><sec><title>Graphical Abstract</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"475719v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475728', 'tweets': 73, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Predicting patient treatment response and resistance via single-cell transcriptomics of their tumors', 'authors': \"[{'name': 'Sanju Sinha'}, {'name': 'Rahulsimham Vegesna'}, {'name': 'Saugato Rahman Dhruba'}, {'name': 'Wei Wu'}, {'name': 'D. Lucas Kerr'}, {'name': 'Oleg V. Stroganov'}, {'name': 'Ivan Grishagin'}, {'name': 'Kenneth D. Aldape'}, {'name': 'Collin M. Blakely'}, {'name': 'Peng Jiang'}, {'name': 'Craig J. Thomas'}, {'name': 'Trever G. Bivona'}, {'name': 'Alejandro A. Schäffer'}, {'name': 'Eytan Ruppin'}]\", 'abstract': '<p>Tailoring the best treatments to cancer patients is an important open challenge. Here, we build a precision oncology data science and software framework for <underline>PER</underline>sonalized single-<underline>C</underline>ell <underline>E</underline>xpression-based <underline>P</underline>lanning for <underline>T</underline>reatments <underline>In On</underline>cology (PERCEPTION). Our approach capitalizes on recently published matched bulk and single-cell transcriptome profiles of large-scale cell-line drug screens to build treatment response models from patients’ single-cell (SC) tumor transcriptomics. First, we show that PERCEPTION successfully predicts the response to monotherapy and combination treatments in screens performed in cancer and patient-tumor-derived primary cells based on SC-expression profiles. Second, it successfully stratifies responders to combination therapy based on the patients’ tumor’s SC-expression in two very recent multiple myeloma and breast cancer clinical trials. Thirdly, it captures the development of clinical resistance to five standard tyrosine kinase inhibitors using tumor SC-expression profiles obtained during treatment in a lung cancer patients’ cohort. Notably, PERCEPTION outperforms state-of-the-art bulk expression-based predictors in all three clinical cohorts. In sum, this study provides a first-of-its-kind conceptual and computational method that is predictive of response to therapy in patients, based on the clonal SC gene expression of their tumors.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476171', 'tweets': 65, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'A high-resolution comparative atlas across 74 fish genomes illuminates teleost evolution after whole-genome duplication', 'authors': \"[{'name': 'Elise Parey'}, {'name': 'Alexandra Louis'}, {'name': 'Jérôme Monfort'}, {'name': 'Yann Guiguen'}, {'name': 'Hugues Roest Crollius'}, {'name': 'Camille Berthelot'}]\", 'abstract': '<p>Teleost fish are one of the most species-rich and diverse clades amongst vertebrates, which makes them an outstanding model group for evolutionary, ecological and functional genomics. Yet, despite a growing number of sequence reference genomes, large-scale comparative analysis remains challenging in teleosts due to the specifics of their genomic organization. As legacy of a whole genome duplication dated 320 million years ago, a large fraction of teleost genomes remain in duplicate paralogous copies. This ancestral polyploidy confounds the detailed identification of orthologous genomic regions across teleost species. Here, we combine tailored gene phylogeny methodology together with the state-of-the art ancestral karyotype reconstruction to establish the first high resolution comparative atlas of paleopolyploid regions across 74 teleost fish genomes. We show that this atlas represents a unique, robust and reliable resource for fish genomics. We then use the comparative atlas to study the tetraploidization and rediploidization mechanisms that affected the ancestor of teleosts. Although the polyploid history of teleost genomes appears complex, we uncover that meiotic recombination persisted between duplicated chromosomes for over 60 million years after polyploidization, suggesting that the teleost ancestor was an autotetraploid.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.11.475815', 'tweets': 65, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Panoramic visual statistics shape retina-wide organization of receptive fields', 'authors': \"[{'name': 'Divyansh Gupta'}, {'name': 'Wiktor Młynarski'}, {'name': 'Olga Symonova'}, {'name': 'Jan Svatoň'}, {'name': 'Maximilian Joesch'}]\", 'abstract': '<p>Visual systems have adapted to the structure of natural stimuli. In the retina, center-surround receptive fields (RFs) of retinal ganglion cells (RGCs) appear to efficiently encode natural sensory signals<sup>1–4</sup>. Conventionally, it has been assumed that natural scenes are isotropic and homogeneous; thus, the RF properties are expected to be uniform across the visual field. However, natural scene statistics such as luminance and contrast are not uniform and vary significantly across elevation<sup>5,6</sup>. Here, by combining theory and novel experimental approaches, we demonstrate that this inhomogeneity is exploited by RGC RFs across the entire retina to increase the coding efficiency. We formulated three predictions derived from the efficient coding theory: (i) optimal RFs should strengthen their surround from the dimmer ground to the brighter sky, (ii) RFs should simultaneously decrease their center size and (iii) RFs centered at the horizon should have a marked surround asymmetry due to a stark contrast drop-off. To test these predictions, we developed a new method to image high-resolution RFs of thousands of RGCs in individual retinas. We found that the RF properties match theoretical predictions, and consistently change their shape from dorsal to the ventral retina, with a distinct shift in the RF surround at the horizon. These effects are observed across RGC subtypes, which were thought to represent visual space homogeneously, indicating that functional retinal streams share common adaptations to visual scenes. Our work shows that RFs of mouse RGCs exploit the non-uniform, panoramic structure of natural scenes at a previously unappreciated scale, to increase coding efficiency.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476155', 'tweets': 64, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Transatlantic spread of highly pathogenic avian influenza H5N1 by wild birds from Europe to North America in 2021', 'authors': \"[{'name': 'V. Caliendo'}, {'name': 'N. S. Lewis'}, {'name': 'A. Pohlmann'}, {'name': 'S.R. Baillie'}, {'name': 'A.C. Banyard'}, {'name': 'M. Beer'}, {'name': 'I.H. Brown'}, {'name': 'R.A.M. Fouchier'}, {'name': 'R.D.E. Hansen'}, {'name': 'T.K. Lameris'}, {'name': 'A.S. Lang'}, {'name': 'S. Laurendeau'}, {'name': 'O. Lung'}, {'name': 'G. Robertson'}, {'name': 'H. van der Jeugd'}, {'name': 'T.N. Alkie'}, {'name': 'K. Thorup'}, {'name': 'M.L. van Toor'}, {'name': 'J. Waldenström'}, {'name': 'C. Yason'}, {'name': 'T. Kuiken'}, {'name': 'Y. Berhane'}]\", 'abstract': '<p>Highly pathogenic avian influenza (HPAI) viruses of the A/Goose/Guangdong/1/1996 lineage (GsGd), which threaten the health of poultry, wildlife and humans, are spreading across Asia, Europe and Africa, but are currently absent from Oceania and the Americas. In December 2021, H5N1 HPAI viruses were detected in poultry and a free-living gull in St. John’s, Newfoundland and Labrador, Canada. Phylogenetic analysis showed that these viruses were most closely related to HPAI GsGd viruses circulating in northwestern Europe in spring 2021. Analysis of wild bird migration suggested that these viruses may have been carried across the Atlantic via Iceland, Greenland/Arctic or pelagic routes. The here documented incursion of HPAI GsGd viruses into North America raises concern for further virus spread across the Americas by wild bird migration.</p><sec><title>One-Sentence Summary</title><p>Detection of H5N1 highly pathogenic avian influenza in Canada raises concern for spread in the Americas by migratory birds.</p></sec>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.10.475641', 'tweets': 63, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The genetic structure of a wild wheat population has remained associated with microhabitats over 36 years', 'authors': \"[{'name': 'Tal Dahan-Meir'}, {'name': 'Thomas James Ellis'}, {'name': 'Fabrizio Mafessoni'}, {'name': 'Hanan Sela'}, {'name': 'Jacob Manisterski'}, {'name': 'Naomi Avivi-Ragolsky'}, {'name': 'Amir Raz'}, {'name': 'Moshe Feldman'}, {'name': 'Yehoshua Anikster'}, {'name': 'Magnus Nordborg'}, {'name': 'Avraham A. Levy'}]\", 'abstract': '<p>Wild progenitors of major crops can help us understand domestication, and may also provide the genetic resources needed for ensuring food security in the face of climate change. We examined the genetic structure of a wild emmer wheat population, sampled over 36 years while both temperature and CO<sub>2</sub> concentration increased significantly. The genotypes of 832 individuals revealed high genetic diversity over scales of tens of meters and were clustered spatially into ecological microhabitats. This pattern was remarkably stable over time. Simulations indicate that neutral processes alone are unlikely to fully explain the spatial and temporal stability of the population. These results are consistent with a role for local adaptation in shaping the fine-scale structure of plant populations, which is relevant for <italic>in-situ</italic> conservation strategies of biodiversity in nature.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475860', 'tweets': 57, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Two modes of evolution shape bacterial strain diversity in the gut for thousands of generations', 'authors': \"[{'name': 'N. Frazão'}, {'name': 'A. Konrad'}, {'name': 'D. Güleresi'}, {'name': 'M. Lässig'}, {'name': 'I. Gordo'}]\", 'abstract': '<p>How and at what pace bacteria evolve when colonizing healthy hosts remains unclear. Here, by monitoring evolution for more than six thousand generations in the mouse gut, we show that the successful colonization of an invader <italic>Escherichia coli</italic> depends on the diversity of the existing microbiota and the presence of a closely related strain. Following colonization, two modes of evolution were observed: one in which diversifying selection leads to long-term coexistence of ecotypes and a second in which directional selection propels selective sweeps. These modes can be quantitatively distinguished by the statistics of mutation trajectories. In our experiments, diversifying selection was marked by the emergence of metabolic mutations, and directional selection by acquisition of prophages, which bring their own benefits and costs. In both modes, we observed parallel evolution, with mutation accumulation rates comparable to those typically observed <italic>in vitro</italic> on similar time scales. Our results show that gut environments can rapidly generate diversifying selection and ecotype formation.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476419', 'tweets': 56, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Population genomic evidence of selection on structural variants in a natural hybrid zone', 'authors': \"[{'name': 'Linyi Zhang'}, {'name': 'Samridhi Chaturvedi'}, {'name': 'Chris Nice'}, {'name': 'Lauren Lucas'}, {'name': 'Zachariah Gompert'}]\", 'abstract': '<p>Structural variants (SVs) can promote speciation by directly causing reproductive isolation or by suppressing recombination across large genomic regions. Whereas examples of each mechanism have been documented, systematic tests of the role of SVs in speciation are lacking. Here, we take advantage of long-read (Oxford nanopore) whole-genome sequencing and a hybrid zone between two Lycaeides butterfly taxa (L. melissa and Jackson Hole Lycaeides) to comprehensively evaluate genome-wide patterns of introgression for SVs and relate these patterns to hypotheses about speciation. We found &gt;100,000 SVs segregating within or between the two hybridizing species. SVs and SNPs exhibited similar levels of genetic differentiation between species, with the exception of inversions, which were more differentiated. We detected credible variation in patterns of introgression among SV loci in the hybrid zone, with 562 of 1419 ancestry-informative SVs exhibiting genomic clines that deviating from null expectations based on genome-average ancestry. Overall, hybrids exhibited a directional shift towards Jackson Hole Lycaeides ancestry at SV loci, consistent with the hypothesis that these loci experienced more selection on average then SNP loci. Surprisingly, we found that deletions, rather than inversions, showed the highest skew towards excess introgression from Jackson Hole Lycaeides. Excess Jackson Hole Lycaeides ancestry in hybrids was also especially pronounced for Z-linked SVs and inversions containing many genes. In conclusion, our results show that SVs are ubiquitous and suggest that SVs in general, but especially deletions, might contribute disproportionately to hybrid fitness and thus (partial) reproductive isolation.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.15.476471', 'tweets': 55, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'The cytoplasmic incompatibility Cif proteins from prophage WO modify sperm genome integrity', 'authors': \"[{'name': 'Rupinder Kaur'}, {'name': 'Brittany A. Leigh'}, {'name': 'Isabella T. Ritchie'}, {'name': 'Seth R. Bordenstein'}]\", 'abstract': '<p>Inherited microorganisms can selfishly manipulate host reproduction to drive through populations. In Drosophila melanogaster, germline expression of the native prophage WO proteins CifA and CifB cause cytoplasmic incompatibility (CI) in which sperms fertilize uninfected embryos that suffer catastrophic mitotic defects and lethality; however in infected females, CifA rescues the embryonic lethality and thus imparts a fitness advantage to Wolbachia. Despite widespread relevance to sex determination, evolution, and vector control, the mechanisms underlying when and how CI impairs male reproduction remain unknown and a topic of debate. Here we use cytochemical, microscopic, and transgenic assays in D. melanogaster to demonstrate that CifA and CifB proteins of wMel localize to nuclear DNA throughout the process of spermatogenesis. Cif proteins cause abnormal histone retention in elongating spermatids and protamine deficiency in mature sperms of CI-causing males. Protamine-deficient sperms travel to the female reproductive tract together with Cif proteins. In female ovaries, CifA localizes to germ cell nuclei and overlaps with Wolbachia in the nurse cell cytoplasm and the oocyte, however Cifs are not present in late-stage oocytes and the embryo. Moreover, CI and rescue are contingent upon a newly annotated CifA bipartite nuclear localization sequence. Our results reveal a previously unrecognized phenomena in which prophage proteins invade animal gametic nuclei and modify the histone-protamine transition of spermatogenesis.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.476329', 'tweets': 53, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Meiosis in the human pathogen Aspergillus fumigatus has the highest known number of crossovers', 'authors': \"[{'name': 'Ben Auxier'}, {'name': 'Frank Becker'}, {'name': 'Reindert Nijland'}, {'name': 'Alfons J.M. Debets'}, {'name': 'Joost van den Heuvel'}, {'name': 'Eveline Snelders'}]\", 'abstract': '<p>Evidence from both population genetics and a laboratory sexual cycle indicate that sex is common in the fungus Aspergillus fumigatus. However, the impact of sexual reproduction has remained unclear. Here, we show that meiosis in A. fumigatus involves the highest known recombination rate, producing ~29 crossovers per chromosome. This represents the highest known crossover rate for any Eukaryotic species. We validate this recombination rate by mapping resistance to acriflavine, a common genetic marker. We further show that this recombination rate can produce the commonly encountered TR34/L98H azole-resistant cyp51A haplotype in each sexual event, facilitating its rapid and global spread. Understanding the consequences of this unparalleled crossover rate will not only enrich our genetic understanding of this emergent human pathogen, but of meiosis in general.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.476228', 'tweets': 52, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Critical assessment of pan-genomics of metagenome-assembled genomes', 'authors': \"[{'name': 'Tang Li'}, {'name': 'Yanbin Yin'}]\", 'abstract': '<p>Background: Large scale metagenome assembly and binning to generate metagenome-assembled genomes (MAGs) has become possible in the past five years. As a result, millions of MAGs have been produced and increasingly included in pan-genomics workflow. However, pan-genome analyses of MAGs may suffer from the known issues with MAGs: fragmentation, incompleteness, and contamination, due to mis-assembly and mis-binning. Here, we conducted a critical assessment of including MAGs in pan-genome analysis, by comparing pan-genome analysis results of complete bacterial genomes and simulated MAGs. Results: We found that incompleteness led to more significant core gene loss than fragmentation. Contamination had little effect on core genome size but had major influence on accessory genomes. The core gene loss remained when using different pan-genome analysis tools and when using a mixture of MAGs and complete genomes. Importantly, the core gene loss was partially alleviated by lowering the core gene threshold and using gene prediction algorithms that consider fragmented genes, but to a less degree when incompleteness was higher than 5%. The core gene loss also led to incorrect pan-genome functional predictions and inaccurate phylogenetic trees. Conclusions: We conclude that lowering core gene threshold and predicting genes in metagenome mode (as Anvio does with Prodigal) are necessary in pan-genome analysis of MAGs to alleviate the accuracy loss. Better quality control of MAGs and development of new pan-genome analysis tools specifically designed for MAGs are needed in future studies.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.14.22269288', 'tweets': 46, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Characterizing features of outbreak duration for novel SARS-CoV-2 variants of concern', 'authors': \"[{'name': 'Alex D. Washburne'}, {'name': 'Nathaniel Hupert'}, {'name': 'Nicole Kogan'}, {'name': 'William Hanage'}, {'name': 'Mauricio Santillana'}]\", 'abstract': '<p>Characterizing the dynamics of epidemic trajectories is critical to understanding the potential impacts of emerging outbreaks and to designing appropriate mitigation strategies. As the COVID-19 pandemic evolves, however, the emergence of SARS-CoV-2 variants of concern has complicated our ability to assess in real-time the potential effects of imminent outbreaks, such as those presently caused by the Omicron variant. Here, we report that SARS-CoV-2 outbreaks across regions exhibit strain-specific times from onset to peak, specifically for Delta and Omicron variants. Our findings may facilitate real-time identification of peak medical demand and may help fine-tune ongoing and future outbreak mitigation deployment efforts.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.21267633', 'tweets': 44, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'National-scale surveillance of emerging SARS-CoV-2 variants in wastewater', 'authors': \"[{'name': 'Fabian Amman'}, {'name': 'Rudolf Markt'}, {'name': 'Lukas Endler'}, {'name': 'Sebastian Hupfauf'}, {'name': 'Benedikt Agerer'}, {'name': 'Anna Schedl'}, {'name': 'Lukas Richter'}, {'name': 'Melanie Zechmeister'}, {'name': 'Martin Bicher'}, {'name': 'Georg Heiler'}, {'name': 'Petr Triska'}, {'name': 'Matthew Thornton'}, {'name': 'Thomas Penz'}, {'name': 'Martin Senekowitsch'}, {'name': 'Jan Laine'}, {'name': 'Zsofia Keszei'}, {'name': 'Beatrice Daleiden'}, {'name': 'Martin Steinlechner'}, {'name': 'Harald Niederstaetter'}, {'name': 'Christoph Scheffknecht'}, {'name': 'Gunther Vogl'}, {'name': 'Guenther Weichlinger'}, {'name': 'Andreas Wagner'}, {'name': 'Katarzyna Slipko'}, {'name': 'Amandine Masseron'}, {'name': 'Elena Radu'}, {'name': 'Franz Allerberger'}, {'name': 'Niki Popper'}, {'name': 'Christoph Bock'}, {'name': 'Daniela Schmid'}, {'name': 'Herbert Oberacher'}, {'name': 'Norbert Kreuzinger'}, {'name': 'Heribert Insam'}, {'name': 'Andreas Bergthaler'}]\", 'abstract': \"<p>SARS-CoV-2 surveillance is crucial to identify variants with altered epidemiological properties. Wastewater-based epidemiology (WBE) provides an unbiased and complementary approach to sequencing individual cases. Yet, national WBE surveillance programs have not been widely implemented and data analyses remain challenging. We deep-sequenced 2,093 wastewater samples representing 95 municipal catchments, covering &gt;57% of Austria's population, from December 2020 to September 2021. Our Variant Quantification in Sewage pipeline designed for Robustness (VaQuERo) enabled us to deduce variant abundance from complex wastewater samples and delineate the spatiotemporal dynamics of the dominant Alpha and Delta variants as well as regional clusters of other variants of concern. These results were cross validated by epidemiological records of &gt;130,000 individual cases. Finally, we provide a framework to predict emerging variants de novo and infer variant-specific reproduction numbers from wastewater. This study demonstrates the power of national-scale WBE to support public health and promises particular value for countries without dense individual monitoring.</p>\", 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476191', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'cGAS recruitment to micronuclei is dictated by pre-existing nuclear chromatin status', 'authors': \"[{'name': 'Kate M. MacDonald'}, {'name': 'Shirony Nicholson-Puthenveedu'}, {'name': 'Maha M. Tageldein'}, {'name': 'Cheryl Arrowsmith'}, {'name': 'Shane M. Harding'}]\", 'abstract': '<p>Micronuclei (MN) are aberrant cytosolic compartments containing broken genomic fragments or whole lagging chromosomes. MN envelopes irreversibly rupture, allowing the viral receptor cGAS to localize to MN and initiate an inflammatory signalling cascade. Here, we demonstrate that MN envelope rupture is not sufficient for cGAS localization. Unlike MN that arise following ionizing radiation (IR), ruptured MN generated from acute transcription stressors DRB or siSRSF1 are refractory to cGAS localization. Recruitment of cGAS to MN is blocked by inhibiting the histone methyltransferase DOT1L prior to IR exposure, demonstrating that cGAS recruitment to MN is dictated by nuclear chromatin organization at the time of DNA damage. Loss of cGAS+ MN, caused either by acute transcription stressors or by preventing DOT1L-deposited histone methylation, corresponded to significantly decreased cGAS-dependent inflammatory signalling. These results implicate nuclear chromatin organization in micronuclear composition and activity, influencing the ability of damage-induced MN to retain cytosolic proteins upon rupture.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.14.475546', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Identification and Age-dependent Increase of Platelet Biased Human Hematopoietic Stem Cells', 'authors': \"[{'name': 'Merve Aksöz'}, {'name': 'Grigore-Aristide Gafencu'}, {'name': 'Bilyana Stoilova'}, {'name': 'Mario Buono'}, {'name': 'Yiran Meng'}, {'name': 'Niels Asger Jakobsen'}, {'name': 'Marlen Metzner'}, {'name': 'Sally-Ann Clark'}, {'name': 'Ryan Beveridge'}, {'name': 'Supat Thongjuea'}, {'name': 'Paresh Vyas'}, {'name': 'Claus Nerlov'}]\", 'abstract': '<p>Hematopoietic stem cells (HSC) reconstitute multi-lineage human hematopoiesis after clinical bone marrow transplantation and are the cells-of-origin of hematological malignancies. Though HSC provide multi-lineage engraftment, individual murine HSCs are lineage-biased and contribute unequally to blood cell lineages. Now, by combining xenografting of molecularly barcoded adult human bone marrow (BM) HSCs and high-throughput single cell RNA sequencing we demonstrate that human individual BM HSCs are also functionally and transcriptionally lineage biased. Specifically, we identify platelet-biased and multi-lineage human HSCs. Quantitative comparison of transcriptomes from single HSCs from young, and aged, BM show that both the proportion of platelet-biased HSCs, and their level of transcriptional platelet priming, increases with age. Therefore, platelet-biased HSCs, as well as their increased prevalence and elevated transcriptional platelet priming during ageing, are conserved between human and murine hematopoiesis.</p><sec><title>One-Sentence Summary</title><p><italic>In vivo</italic> barcoding and single cell RNA sequencing identifies platelet-biased human bone marrow HSCs.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475922', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': True, 'title': 'Structural and functional impact by SARS-CoV-2 Omicron spike mutations', 'authors': \"[{'name': 'Jun Zhang'}, {'name': 'Yongfei Cai'}, {'name': 'Christy L. Lavine'}, {'name': 'Hanqin Peng'}, {'name': 'Haisun Zhu'}, {'name': 'Krishna Anand'}, {'name': 'Pei Tong'}, {'name': 'Avneesh Gautam'}, {'name': 'Megan L. Mayer'}, {'name': 'Sophia Rits-Volloch'}, {'name': 'Shaowei Wang'}, {'name': 'Piotr Sliz'}, {'name': 'Duane R. Wesemann'}, {'name': 'Wei Yang'}, {'name': 'Michael S. Seaman'}, {'name': 'Jianming Lu'}, {'name': 'Tianshu Xiao'}, {'name': 'Bing Chen'}]\", 'abstract': '<p>The Omicron variant of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), bearing an unusually high number of mutations, has become a dominant strain in many countries within several weeks. We report here structural, functional and antigenic properties of its full-length spike (S) protein with a native sequence in comparison with those of previously prevalent variants. Omicron S requires a substantially higher level of host receptor ACE2 for efficient membrane fusion than other variants, possibly explaining its unexpected cellular tropism. Mutations not only remodel the antigenic structure of the N-terminal domain of the S protein, but also alter the surface of the receptor-binding domain in a way not seen in other variants, consistent with its remarkable resistance to neutralizing antibodies. These results suggest that Omicron S has acquired an extraordinary ability to evade host immunity by excessive mutations, which also compromise its fusogenic capability.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475327', 'tweets': 38, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Systemic infection of SARS-CoV-2 in free ranging Leopard (Panthera pardus fusca) in India', 'authors': \"[{'name': 'Sonalika Mahajan'}, {'name': 'Mathesh Karikalan'}, {'name': 'Vishal Chander'}, {'name': 'Abhijit M. Pawde'}, {'name': 'G Saikumar'}, {'name': 'M Semmaran'}, {'name': 'P Sree Lakshmi'}, {'name': 'Megha Sharma'}, {'name': 'Sukdeb Nandi'}, {'name': 'Karam Pal Singh'}, {'name': 'Vivek Kumar Gupta'}, {'name': 'Raj Kumar Singh'}, {'name': 'Gaurav Kumar Sharma'}]\", 'abstract': '<p>We report patho-morphological and virological characterization of SARS-CoV-2 in naturally infected, free ranging Indian Leopard (<italic>Panthera pardus fusca</italic>). Whole genome sequence analysis confirmed infection of Delta variant of SARS-CoV-2, possibly spill over from humans, but the case was detected when infection level had dropped significantly in human population. This report underlines the need for intensive screening of wild animals for keeping track of the virus evolution and development of carrier status of SARS-CoV-2 among wildlife species.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475735', 'tweets': 38, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'GoPeaks: Histone Modification Peak Calling for CUT&Tag', 'authors': \"[{'name': 'William M Yashar'}, {'name': 'Garth Kong'}, {'name': 'Jake VanCampen'}, {'name': 'Brittany M Smith'}, {'name': 'Daniel J Coleman'}, {'name': 'Lucia Carbone'}, {'name': 'Galip Gürkan Yardimci'}, {'name': 'Julia E Maxson'}, {'name': 'Theodore P Braun'}]\", 'abstract': '<p>Genome-wide mapping of the histone modification landscape is critical to understanding tran-scriptional regulation. Cleavage Under Targets and Tagmentation (CUT&amp;Tag) is a new method for profiling the localization of covalent histone modifications, offering improved sensitivity and decreased cost compared with Chromatin Immunoprecipitation Sequencing (ChIP-seq). Here, we present GoPeaks, a peak calling method specifically designed for histone modification CUT&amp;Tag data. GoPeaks implements a Binomial distribution and stringent read count cut-off to nominate candidate genomic regions. We compared the performance of GoPeaks against commonly used peak calling algorithms to detect H3K4me3, H3K4me1, and H3K27Ac peaks from CUT&amp;Tag data. These histone modifications display a range of peak profiles and are frequently used in epigenetic studies. We found GoPeaks robustly detects genome-wide histone modifications and, notably, identifies H3K27Ac with improved sensitivity compared to other standard peak calling algorithms.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.22268989', 'tweets': 38, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': False, 'title': 'Network Constraints on Longitudinal Grey Matter Changes in First Episode Psychosis', 'authors': \"[{'name': 'Sidhant Chopra'}, {'name': 'Stuart Oldham'}, {'name': 'Ashlea Segal'}, {'name': 'Alexander Holmes'}, {'name': 'Kristina Sabaroedin'}, {'name': 'Edwina R. Orchard'}, {'name': 'Shona M. Francey'}, {'name': 'Brian O’Donoghue'}, {'name': 'Vanessa Cropley'}, {'name': 'Barnaby Nelson'}, {'name': 'Jessica Graham'}, {'name': 'Lara Baldwin'}, {'name': 'Jeggan Tiego'}, {'name': 'Hok Pan Yuen'}, {'name': 'Kelly Allott'}, {'name': 'Mario Alvarez-Jimenez'}, {'name': 'Susy Harrigan'}, {'name': 'Christos Pantelis'}, {'name': 'Stephen J Wood'}, {'name': 'Mark Bellgrove'}, {'name': 'Patrick McGorry'}, {'name': 'Alex Fornito'}]\", 'abstract': '<sec><title>Background</title><p>Different regions of the brain’s grey matter are connected by a complex structural network of white matter fibres, which are responsible for the propagation of action potentials and the transport of trophic and other molecules. In neurodegenerative disease, these connections constrain the way in which grey matter volume loss progresses. Here, we investigated whether connectome architecture also shapes the spatial pattern of longitudinal grey matter volume changes attributable to illness and antipsychotic medication in first episode psychosis (FEP).</p></sec><sec><title>Methods</title><p>We conducted a triple-blind randomised placebo-control MRI study where 62 young adults with first episode psychosis received either an atypical antipsychotic or placebo over 6-months. A healthy control group was also recruited. Anatomical MRI scans were acquired at baseline, 3-months and 12-months. Deformation-based morphometry was used to estimate illness-related and antipsychotic-related grey matter volume changes over time. Representative functional and structural brain connectivity patterns were derived from an independent healthy control group using resting-state functional MRI and diffusion-weighted imaging. We used neighbourhood deformation models to predict the extent of brain change in a given area by the changes observed in areas to which it is either structurally connected or functionally coupled.</p></sec><sec><title>Results</title><p>At baseline, we found that empirical illness-related regional volume differences were strongly correlated with predicted differences using a model constrained by structural connectivity weights (r = .541; p &lt; .001). At 3-months and 12-months, we also found a strong correlation between longitudinal regional illness-related (r &gt; .516; p &lt; .001) and antipsychotic-related volume change (r &gt; .591; p &lt; .001) with volumetric changes in structurally connected areas. These correlations were significantly greater than those observed across various null models accounting for lower-order spatial and network properties of the data. Associations between empirical and predicted volume change estimates were much lower for models that only considered binary structural connectivity (all r &lt; .376), or which were constrained by inter-regional functional coupling (all r &lt; .436). Finally, we found that potential epicentres of volume change emerged posteriorly early in the illness and shifted to the prefrontal cortex by later illness stages.</p></sec><sec><title>Conclusion</title><p>Psychosis- and antipsychotic-related grey matter volume changes are strongly shaped by anatomical brain connectivity. This result is consistent with findings in other neurological disorders and implies that such connections may constrain pathological processes causing brain dysfunction in FEP.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475690', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Atlasing white matter and grey matter joint contributions to resting-state networks in the human brain', 'authors': \"[{'name': 'Victor Nozais'}, {'name': 'Stephanie J Forkel'}, {'name': 'Laurent Petit'}, {'name': 'Michel Thiebaut de Schotten'}, {'name': 'Marc Joliot'}]\", 'abstract': '<p>Over the past two decades, the study of resting-state functional magnetic resonance imaging (<italic>f</italic>MRI) has revealed the existence of multiple brain areas displaying synchronous functional blood oxygen level-dependent signals (BOLD) — resting-state networks (RSNs). The variation in functional connectivity between the different areas of a resting-state network or between multiple networks, have been extensively studied and linked to cognitive states and pathologies. However, the white matter connections supporting each network remain only partially described. In this work, we developed a data-driven method to systematically map the white and grey matter contributing to resting-state networks. Using the Human Connectome Project, we generated an atlas of 30 resting-state networks, each with two maps: white matter and grey matter. By integrating structural and functional neuroimaging data, this method builds an atlas that unlocks the joint anatomical exploration of white and grey matter to resting-state networks. The method also allows highlighting the overlap between networks, which revealed that most (89%) of the brain’s white matter is shared amongst multiple networks, with 16% shared by at least 7 resting-state networks. These overlaps, especially the existence of regions shared by numerous networks, suggest that white matter lesions in these areas might strongly impact the correlations and the communication within resting-state networks. We provide an open-source software to explore the joint contribution of white and grey matter to RSNs and facilitate the study of the impact of white matter damage on RSNs.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475870', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Debiasing FracMinHash and deriving confidence intervals for mutation rates across a wide range of evolutionary distances', 'authors': \"[{'name': 'Mahmudur Rahman Hera'}, {'name': 'N. Tessa Pierce-Ward'}, {'name': 'David Koslicki'}]\", 'abstract': '<p>Sketching methods offer computational biologists scalable techniques to analyze data sets that continue to grow in size. MinHash is one such technique that has enjoyed recent broad application. However, traditional MinHash has previously been shown to perform poorly when applied to sets of very dissimilar sizes. <italic>FracMinHash</italic> was recently introduced as a modification of MinHash to compensate for this lack of performance when set sizes differ. While experimental evidence has been encouraging, FracMinHash has not yet been analyzed from a theoretical perspective. In this paper, we perform such an analysis and prove that while FracMinHash is not unbiased, this bias is easily corrected. Next, we detail how a simple mutation model interacts with FracMinHash and are able to derive confidence intervals for evolutionary mutation distances between pairs of sequences as well as hypothesis tests for FracMinHash. We find that FracMinHash estimates the containment of a genome in a large metagenome more accurately and more precisely when compared to traditional MinHash, and the confidence interval performs significantly better in estimating mutation distances. A python-based implementation of the theorems we derive is freely available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/KoslickiLab/mutation-rate-ci-calculator\">https://github.com/KoslickiLab/mutation-rate-ci-calculator</ext-link>. The results presented in this paper can be reproduced using the code at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/KoslickiLab/ScaledMinHash-reproducibles\">https://github.com/KoslickiLab/ScaledMinHash-reproducibles</ext-link>.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.22269017', 'tweets': 36, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Healthcare workers’ views on mandatory SARS-CoV-2 vaccination in the United Kingdom: findings from the UK-REACH prospective longitudinal cohort study', 'authors': \"[{'name': 'Katherine Woolf'}, {'name': 'Mayuri Gogoi'}, {'name': 'Christopher A Martin'}, {'name': 'Padmasayee Papineni'}, {'name': 'Susie Lagrata'}, {'name': 'Laura B Nellums'}, {'name': 'I Chris McManus'}, {'name': 'Anna L Guyatt'}, {'name': 'Carl Melbourne'}, {'name': 'Luke Bryant'}, {'name': 'Amit Gupta'}, {'name': 'Catherine John'}, {'name': 'Sue Carr'}, {'name': 'Martin D Tobin'}, {'name': 'Sandra Simpson'}, {'name': 'Bindu Gregary'}, {'name': 'Avinash Aujayeb'}, {'name': 'Stephen Zingwe'}, {'name': 'Rubina Reza'}, {'name': 'Laura J Gray'}, {'name': 'Kamlesh Khunti'}, {'name': 'Manish Pareek'}]\", 'abstract': '<sec><title>Background</title><p>Several countries now have mandatory SARS-CoV-2/COVID-19 vaccination for healthcare workers (HCWs) or the general population. HCWs’ views on this are largely unknown.</p></sec><sec><title>Methods</title><p>We administered an online questionnaire to 17891 United Kingdom (UK) HCWs in Spring 2021 as part of the United Kingdom Research study into Ethnicity And COVID-19 outcomes in Healthcare workers (UK-REACH) nationwide prospective cohort study. We categorised responses to a free-text question “What should society do if people don’t get vaccinated against COVID-19?” using content analysis. We collapsed categories into a binary variable: favours mandatory vaccination or not and used logistic regression to calculate its demographic predictors, and occupational, health and attitudinal predictors adjusted for demographics.</p></sec><sec><title>Findings</title><p>Of 5633 questionnaire respondents, 3235 answered the freetext question; 18% (n=578) of those favoured mandatory vaccination but the most frequent suggestion was education (32%, n=1047). Older HCWs, HCWs vaccinated against influenza (OR 1.48; 95%CI 1.10 – 1.99, vs none) and with more positive vaccination attitudes generally (OR 1.10; 95%CI 1.06 – 1.14) were more likely to favour mandatory vaccination (OR 1.26; 95%CI 1.17 – 1.37, per decade increase), whereas female HCWs (OR= 0.80, 95%CI 0.65 – 0.99, vs male), Black HCWs (OR= 0.48, 95%CI 0.26 – 0.87, vs White), those hesitant about COVID-19 vaccination (OR= 0.56; 95%CI 0.43 – 0.71, vs not hesitant), in an Allied Health Profession (OR 0.67; 95%CI 0.51 – 0.88, vs Medical), or who trusted their organisation (OR 0.78; 95%CI 0.63 – 0.96) were less likely to.</p></sec><sec><title>Interpretation</title><p>Only one in six of the HCWs in this large, diverse, UK-wide sample favoured mandatory vaccination. Building trust, educating and supporting HCWs who are hesitant about vaccination may be more acceptable, effective and equitable.</p></sec><sec><title>Funding</title><p>MRC-UK Research and Innovation grant (MR/V027549/1) and the Department of Health and Social Care via the National Institute for Health Research.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.14.475871', 'tweets': 36, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'An ancient truncated duplication of the anti-Mullerian hormone receptor type 2 gene is a potential conserved master sex determinant in the Pangasiidae catfish family', 'authors': \"[{'name': 'Ming Wen'}, {'name': 'Qiaowei Pan'}, {'name': 'Elodie Jouanno'}, {'name': 'Jerome Montfort'}, {'name': 'Margot Zahm'}, {'name': 'Cedric Cabau'}, {'name': 'Christophe Klopp'}, {'name': 'Carole Iampietro'}, {'name': 'Celine Roques'}, {'name': 'Olivier Bouchez'}, {'name': 'Adrien Castinel'}, {'name': 'Cecile Donnadieu'}, {'name': 'Hugues Parrinello'}, {'name': 'Charles Poncet'}, {'name': 'Elodie Belmonte'}, {'name': 'Veronique Gautier'}, {'name': 'Jean-Christophe Avarre'}, {'name': 'Remi Dugue'}, {'name': 'Rudhy Gustiano'}, {'name': 'Tran Thi thuy Ha'}, {'name': 'Marc Campet'}, {'name': 'Kednapat Sriphairoj'}, {'name': 'Josiane Ribolli'}, {'name': 'Fernanda L. deAlmeida'}, {'name': 'Thomas Desvignes'}, {'name': 'John H. Postlethwait'}, {'name': 'Christabel FloiBucao'}, {'name': 'Marc Robinson-Rechavi'}, {'name': 'Julien Bobe'}, {'name': 'Amaury Herpin'}, {'name': 'Yann Guiguen'}]\", 'abstract': '<p>The evolution of sex determination (SD) mechanisms in teleost fishes is amazingly dynamic, as reflected by the variety of different master sex-determining genes identified, even sometimes among closely related species. Pangasiids are a group of economically important catfishes in many South-Asian countries, but little is known about their sex determination system. Here, we generated novel genomic resources for 12 Pangasiid species and provided a first characterization of their SD system. Based on an Oxford Nanopore long-read chromosome-scale high quality genome assembly of the striped catfish Pangasianodon hypophthalmus, we identified a duplication of the anti-Mullerian hormone receptor type II gene (amhr2), which was further characterized as being sex-linked in males and expressed only in testicular samples. These first results point to a male-specific duplication on the Y chromosome (amhr2by) of the autosomal amhr2a. Sequence annotation revealed that the P. hypophthalmus Amhr2by is truncated in its N-terminal domain, lacking the cysteine-rich extracellular part of the receptor that is crucial for ligand binding, suggesting a potential route for its neofunctionalization. Short-read genome sequencing and reference-guided assembly of 11 additional Pangasiid species, along with sex-linkage studies, revealed that this truncated amhr2by duplication is also conserved as a male-specific gene in many Pangasiids. Reconstructions of the amhr2 phylogeny suggested that amhr2by arose from an ancient duplication / insertion event at the root of the Siluroidei radiation that is dated around 100 million years ago. Altogether these results bring multiple lines of evidence supporting that amhr2by is an ancient and conserved master sex-determining gene in Pangasiid catfishes, a finding that highlights the recurrent usage of the transforming growth factor β pathway in teleost sex determination and brings another empirical case towards the understanding of the dynamics or stability of sex determination systems.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.12.476080', 'tweets': 33, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'A split ribozyme that links detection of a native RNA to orthogonal protein outputs', 'authors': \"[{'name': 'Lauren Gambill'}, {'name': 'August Staubus'}, {'name': 'Andrea Ameruoso'}, {'name': 'James Chappell'}]\", 'abstract': '<p>Individual RNA remains a challenging signal to synthetically transduce into different types of cellular information. Here, we describe Ribozyme-ENabled Detection of RNA (RENDR), a plug-and-play strategy that uses cellular transcripts to template the assembly of split ribozymes, triggering splicing reactions that generate orthogonal protein outputs. To identify split ribozymes that require templating for splicing, we used laboratory evolution to evaluate the activities of different split variants of the <italic>Tetrahymena thermophila</italic> ribozyme. The best design delivered a 93-fold dynamic range of splicing with RENDR controlling fluorescent protein production in response to an RNA input. We resolved a thermodynamic model to guide RENDR design, showed how input signals can be transduced into diverse visual, chemical, and regulatory outputs, and used RENDR to detect an antibiotic resistance phenotype in bacteria. This work shows how transcriptional signals can be monitored <italic>in situ</italic> using RNA synthetic biology and converted into different types of biochemical information.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475790', 'tweets': 32, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The phylogenomic landscape of the genus Serratia', 'authors': \"[{'name': 'David J. Williams'}, {'name': 'Patrick A. D. Grimont'}, {'name': 'Adrián Cazares'}, {'name': 'Francine Grimont'}, {'name': 'Elisabeth Ageron'}, {'name': 'Kerry A. Pettigrew'}, {'name': 'Daniel Cazares'}, {'name': 'Elisabeth Njamkepo'}, {'name': 'François-Xavier Weill'}, {'name': 'Eva Heinz'}, {'name': 'Matthew T. G. Holden'}, {'name': 'Nicholas R. Thomson'}, {'name': 'Sarah J. Coulthurst'}]\", 'abstract': '<p>The genus <italic>Serratia</italic> has been studied for over a century and includes clinically-important and diverse environmental members. Despite this, there is a paucity of genomic information across the genus and a robust whole genome-based phylogenetic framework is lacking. Here, we have assembled and analysed a representative set of 664 genomes from across the genus, including 215 historic isolates originally used in defining the genus. Phylogenomic analysis of the genus reveals a clearly-defined population structure which displays deep divisions and aligns with ecological niche, as well as striking congruence between historical biochemical phenotyping data and contemporary genomics data. We show that <italic>Serratia</italic> is a diverse genus which displays striking plasticity and ability to adapt to its environment, including a highly-varied portfolio of plasmids, and provide evidence of different patterns of gene flow across the genus. This work provides an essential platform for understanding the emergence of clinical and other lineages of <italic>Serratia</italic>.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.474936', 'tweets': 32, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'The megabase-scale crossover landscape is independent of sequence divergence', 'authors': \"[{'name': 'Qichao Lian'}, {'name': 'Victor Solier'}, {'name': 'Birgit Walkemeier'}, {'name': 'Bruno Huettel'}, {'name': 'Korbinian Schneeberger'}, {'name': 'Raphael Mercier'}]\", 'abstract': '<p>Meiotic recombination frequency varies along chromosomes and strongly correlates with sequence divergence. However, the causality underlying this correlation is unclear. To untangle the relationship between recombination landscapes and polymorphisms, we characterized the genome-wide recombination landscape in the absence of polymorphisms, using <italic>Arabidopsis thaliana</italic> homozygous inbred lines in which a few hundred genetic markers were introduced through mutagenesis. We found that megabase-scale recombination landscapes in inbred lines are strikingly similar to the recombination landscapes in hybrids, with the sole exception of heterozygous large rearrangements where recombination is prevented locally. In addition, we found that the megabase-scale recombination landscape can be accurately predicted by chromatin features. Our results show that polymorphisms are not causal for the shape of the megabase-scale recombination landscape, rather, favor alternative models in which recombination and chromatin shape sequence divergence across the genome.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475536', 'tweets': 32, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'In vivo imaging of calcium dynamics in zebrafish hepatocytes', 'authors': \"[{'name': 'Macarena Pozo-Morales'}, {'name': 'Inés Garteizgogeascoa'}, {'name': 'Camille Perazzolo'}, {'name': 'Sumeet Pal Singh'}]\", 'abstract': '<p>Hepatocytes were the first cell-type for which oscillations of cytoplasmic calcium levels in response to hormones were described. Since then, investigation of calcium dynamics in liver explants and culture has greatly increased our understanding of calcium signaling. A bottleneck, however, exists in observing calcium dynamics in a non-invasive manner due to the optical inaccessibility of the mammalian liver. Here we take advantage of the transparency of the zebrafish larvae to develop a setup that allows <italic>in vivo</italic> imaging of calcium flux in zebrafish hepatocytes at cellular resolution. Using this, we provide quantitative assessment of intracellular calcium dynamics during multiple contexts, including growth, feeding, ethanol-induced stress and cell ablation. Specifically, we show that synchronized calcium oscillations are present <italic>in vivo</italic>, which are lost upon starvation. Feeding recommences calcium waves in the liver, but in a spatially restricted manner. Further, ethanol treatment as well as cell ablation induces calcium flux, but with different dynamics. The former causes asynchronous calcium oscillations, while the latter leads to a single calcium spike. Overall, we demonstrate the presence of oscillations, waves and spikes <italic>in vivo</italic>. Thus, our study introduces a platform for observing diverse calcium dynamics while maintaining the native environment of the liver, which will help investigations into the dissection of molecular mechanisms supporting the intra- and intercellular calcium signaling in the liver.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.476002', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'In vivo single-cell profiling of lncRNAs during Ebola virus infection', 'authors': \"[{'name': 'Luisa Santus'}, {'name': 'Raquel García-Pérez'}, {'name': 'Maria Sopena-Rios'}, {'name': 'Aaron E Lin'}, {'name': 'Gordon C Adams'}, {'name': 'Kayla G Barnes'}, {'name': 'Katherine J Siddle'}, {'name': 'Shirlee Wohl'}, {'name': 'Ferran Reverter'}, {'name': 'John L Rinn'}, {'name': 'Richard S Bennett'}, {'name': 'Lisa E Hensley'}, {'name': 'Pardis C Sabeti'}, {'name': 'Marta Melé'}]\", 'abstract': '<p>Long non-coding RNAs (lncRNAs) are pivotal mediators of systemic immune response to viral infection, yet most studies concerning their expression and functions upon immune stimulation are limited to <italic>in vitro</italic> bulk cell populations. This strongly constrains our understanding of how lncRNA expression varies at single-cell resolution, and how their cell-type specific immune regulatory roles may differ compared to protein-coding genes. Here, we perform the first in-depth characterization of lncRNA expression variation at single-cell resolution during Ebola virus (EBOV) infection <italic>in vivo</italic>. Using bulk RNA-sequencing from 119 samples and 12 tissue types, we significantly expand the current macaque lncRNA annotation. We then profile lncRNA expression variation in immune circulating single-cells during EBOV infection and find that lncRNAs’ expression in fewer cells is a major differentiating factor from their protein-coding gene counterparts. Upon EBOV infection, lncRNAs present dynamic and mostly cell-type specific changes in their expression profiles especially in monocytes, the main cell type targeted by EBOV. Such changes are associated with gene regulatory modules related to important innate immune responses such as interferon response and purine metabolism. Within infected cells, several lncRNAs have positively and negatively correlated expression with viral load, suggesting that expression of some of these lncRNAs might be directly hijacked by EBOV to attack host cells. This study provides novel insights into the roles that lncRNAs play in the host response to acute viral infection and paves the way for future lncRNA studies at single-cell resolution.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.13.476254', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Caenorhabditis nematodes colonize ephemeral resource patches in neotropical forests.', 'authors': \"[{'name': 'Solomon A Sloat'}, {'name': 'Luke M Noble'}, {'name': 'Annalise B Paaby'}, {'name': 'Max Bernstein'}, {'name': 'Audrey Chang'}, {'name': 'Taniya Kaur'}, {'name': 'John Yuen'}, {'name': 'Sophia C. Tintori'}, {'name': 'Jacqueline L. Jackson'}, {'name': 'Arielle Martel'}, {'name': 'Jose A. Salome Correa'}, {'name': 'Lewis Stevens'}, {'name': 'Mark Blaxter'}, {'name': 'Matthew V. Rockman'}]\", 'abstract': \"<p>Factors shaping the distribution and abundance of species include life-history traits, population structure, and stochastic colonization-extinction dynamics. Field studies of model species groups help reveal the roles of these factors. Species of <italic>Caenorhabditis</italic> nematodes are highly divergent at the sequence level but exhibit highly conserved morphological uniformity, and many of these species live in sympatry on microbe-rich patches of rotten material. Here, we use field experiments and large-scale opportunistic collections to investigate species composition, abundance, and colonization efficiency of <italic>Caenorhabditis</italic> in two of the world's best studied lowland tropical field sites: Barro Colorado Island in Panamá and La Selva in Sarapiquí, Costa Rica. We observed seven species of <italic>Caenorhabditis</italic>, four of them known only from these collections. While these localities contain species from many parts of the phylogeny, both localities were dominated by globally distributed androdiecious species. We found that <italic>Caenorhabditis</italic> were able to colonize baits accessible only by phoresy, preferring to colonize baits making direct contact with the ground. We estimate founder numbers per colonization event to be low.</p>\", 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.11.475947', 'tweets': 28, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': True, 'title': 'SARS-CoV-2 drives NLRP3 inflammasome activation in human microglia through spike-ACE2 receptor interaction', 'authors': \"[{'name': 'Eduardo Albornoz'}, {'name': 'Alberto A Amarilla'}, {'name': 'Naphak Modhiran'}, {'name': 'Sandra Parker'}, {'name': 'Xaria X. Li'}, {'name': 'Danushka K. Wijesundara'}, {'name': 'Adriana Pliego Zamora'}, {'name': 'Christopher LD McMillan'}, {'name': 'Benjamin Liang'}, {'name': 'Nias Y.G. Peng'}, {'name': 'Julian D.J. Sng'}, {'name': 'Fatema Tuj Saima'}, {'name': 'Devina Paramitha'}, {'name': 'Rhys Parry'}, {'name': 'Michael S. Avumegah'}, {'name': 'Ariel Isaacs'}, {'name': 'Martin Lo'}, {'name': 'Zaray Miranda-Chacon'}, {'name': 'Daniella Bradshaw'}, {'name': 'Constanza Salinas-Rebolledo'}, {'name': 'Niwanthi W. Rajapakse'}, {'name': 'Trent Munro'}, {'name': 'Alejandro Rojas-Fernandez'}, {'name': 'Paul R. Young'}, {'name': 'Katryn J Stacey'}, {'name': 'Alexander A. Khromykh'}, {'name': 'Keith J. Chappell'}, {'name': 'Daniel Watterson'}, {'name': 'Trent M. Woodruff'}]\", 'abstract': '<p>Coronavirus disease-2019 (COVID-19) is primarily a respiratory disease, however, an increasing number of reports indicate that SARS-CoV-2 infection can also cause severe neurological manifestations, including precipitating cases of probable Parkinson’s disease. As microglial NLRP3 inflammasome activation is a major driver of neurodegeneration, here we interrogated whether SARS-CoV-2 can promote microglial NLRP3 inflammasome activation utilising a model of human monocyte-derived microglia. We identified that SARS-CoV-2 isolates can bind and enter microglia, triggering inflammasome activation in the absence of viral replication. Mechanistically, microglial NLRP3 could be both primed and activated with SARS-CoV-2 spike glycoprotein in a NF-κB and ACE2-dependent manner. Notably, virus- and spike protein-mediated inflammasome activation in microglia was significantly enhanced in the presence of α-synuclein fibrils, which was entirely ablated by NLRP3-inhibition. These results support a possible mechanism of microglia activation by SARS-CoV-2, which could explain the increased vulnerability to developing neurological symptoms akin to Parkinson’s disease in certain COVID-19 infected individuals, and a potential therapeutic avenue for intervention.</p><sec><title>SIGNIFICANCE STATEMENT</title><p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) principally affects the lungs, however there is evidence that the virus can also reach the brain and lead to chronic neurological symptoms. In this study, we examined the interaction SARS-CoV-2 with brain immune cells, by using an ex-vivo model of human monocyte-derived microglia. We identified robust activation of the innate immune sensor complex, NLRP3 inflammasome, in cells exposed to SARS-CoV-2. This was dependent on spike protein-ACE2 receptor interaction and was potentiated in the presence of α-synuclein. We therefore identify a possible mechanism for SARS-CoV-2 and increased vulnerability to developing neurological dysfunction. These findings support a potential therapeutic avenue for treatment of SARS-CoV-2 driven neurological manifestations, through use of NLRP3 inflammasome or ACE2 inhibitors.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.475930', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Cross-linking of the Endolysosomal System Reveals Flotillin Structures and Putative Cargo', 'authors': \"[{'name': 'Jasjot Singh'}, {'name': 'Hadeer Elhabashy'}, {'name': 'Pathma Muthukottiappan'}, {'name': 'Markus Stepath'}, {'name': 'Martin Eisenacher'}, {'name': 'Oliver Kohlbacher'}, {'name': 'Volkmar Gieselmann'}, {'name': 'Dominic Winter'}]\", 'abstract': '<p>Lysosomes are well-established as the main cellular organelles for the degradation of macromolecules and emerging as regulatory centers of metabolism. They are of crucial importance for cellular homeostasis, which is exemplified by a plethora of disorders related to alterations in lysosomal function. In this context, protein complexes play a decisive role, regulating not only metabolic lysosomal processes, but also lysosome biogenesis, transport, and interaction with other organelles. Using cross-linking mass spectrometry, we analyzed lysosomes and early endosomes. Based on the identification of 5,376 cross-links, we investigated protein-protein interactions and structures of lysosome- and endosome-related proteins. In particular, we present evidence for a tetrameric assembly of the lysosomal hydrolase PPT1 and heterodimeric/- multimeric structures of FLOT1/FLOT2 at lysosomes and early endosomes. For FLOT1-/FLOT2- positive early endosomes, we identified &gt;300 proteins presenting putative cargo, and confirm the latrophilin family of adhesion G protein-coupled receptors as substrates for flotillin-dependent endocytosis.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476279', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Benchmarking software to predict antibiotic resistance phenotypes in shotgun metagenomes using simulated data', 'authors': \"[{'name': 'Emily F Wissel'}, {'name': 'Brooke M Talbot'}, {'name': 'Bjorn A Johnson'}, {'name': 'Robert A Petit'}, {'name': 'Vicki Hertzberg'}, {'name': 'Anne Dunlop'}, {'name': 'Timothy D Read'}]\", 'abstract': '<p>The use of shotgun metagenomics for AMR detection is appealing because data can be generated from clinical samples with minimal processing. Detecting antimicrobial resistance (AMR) in clinical genomic data is an important epidemiological task, yet a complex bioinformatic process. Many software tools exist to detect AMR genes, but they have mostly been tested in their detection of genotypic resistance in individual bacterial strains. It is important to understand how well these bioinformatic tools detect AMR genes in shotgun metagenomic data. We developed a software pipeline, hAMRoaster (https://github.com/ewissel/hAMRoaster), for assessing accuracy of prediction of antibiotic resistance phenotypes. For evaluation purposes, we simulated a short read (Illumina) shotgun metagenomics community of eight bacterial pathogens with extensive antibiotic susceptibility testing profiles. We benchmarked nine open source bioinformatics tools for detecting AMR genes that 1) were conda or Docker installable, 2) had been actively maintained, 3) had an open source license, and 4) took FASTA or FASTQ files as input. Several metrics were calculated for each tool including sensitivity, specificity, and F1 at three coverage levels. This study revealed that tools were highly variable in sensitivity (0.25 - 0.99) and specificity (0.2 - 1) in detection of resistance in our synthetic FASTQ files despite similar databases and methods implemented. Tools performed similarly at all coverage levels (5x, 50x, 100x). Cohen’s kappa revealed low agreement across tools.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.10.22269008', 'tweets': 26, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Hydroxychloroquine/Chloroquine for the Treatment of Hospitalized Patients with COVID-19: An Individual Participant Data Meta-Analysis', 'authors': \"[{'name': 'Leon Di Stefano'}, {'name': 'Elizabeth L. Ogburn'}, {'name': 'Malathi Ram'}, {'name': 'Daniel O. Scharfstein'}, {'name': 'Tianjing Li'}, {'name': 'Preeti Khanal'}, {'name': 'Sheriza N. Baksh'}, {'name': 'Nichol McBee'}, {'name': 'Joshua Gruber'}, {'name': 'Marianne R. Gildea'}, {'name': 'Neil A. Goldenberg'}, {'name': 'Yussef Bennani'}, {'name': 'Samuel M. Brown'}, {'name': 'Whitney R. Buckel'}, {'name': 'Meredith E. Clement'}, {'name': 'Mark J. Mulligan'}, {'name': 'Jane A. O’Halloran'}, {'name': 'Adriana M. Rauseo'}, {'name': 'Wesley H. Self'}, {'name': 'Matthew W. Semler'}, {'name': 'Todd Seto'}, {'name': 'Jason E. Stout'}, {'name': 'Robert J. Ulrich'}, {'name': 'Jennifer Victory'}, {'name': 'Barbara E. Bierer'}, {'name': 'Daniel F. Hanley'}, {'name': 'Daniel Freilich'}, {'name': ' '}]\", 'abstract': '<sec><title>Importance</title><p>Results from observational studies and randomized clinical trials (RCTs) have led to the consensus that hydroxychloroquine (HCQ) and chloroquine (CQ) are not effective for COVID-19 prevention or treatment. Pooling individual participant data (IPD), including unanalyzed data from trials terminated early, enables further investigation of the efficacy and safety of HCQ/CQ.</p></sec><sec><title>Objective</title><p>To assess efficacy of HCQ/CQ in patients hospitalized with COVID-19, both overall and in prespecified subgroups.</p></sec><sec><title>Data Sources</title><p><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://ClinicalTrials.gov\">ClinicalTrials.gov</ext-link> was searched multiple times in May-June 2020. Principal investigators of US-based RCTs evaluating HCQ/CQ in hospitalized COVID-19 patients were invited to collaborate in this IPD meta-analysis.</p></sec><sec><title>Study Selection</title><p>RCTs in which: (1) HCQ/CQ was a treatment arm; (2) patient informed consent and/or individual study IRB approval allowed for data sharing; (3) principal investigators/their institutions signed a data use agreement for the present study; and (4) the outcomes defined in this study were recorded or could be extrapolated.</p></sec><sec><title>Data Extraction and Synthesis</title><p>Wherever possible, harmonized de-identified data were collected via a common template spreadsheet sent to each principal investigator, then shared via a secure online data sharing platform to create a pooled data set. When this was not possible, individual study data were harmonized and merged manually. Data were analyzed by fitting a prespecified Bayesian ordinal regression model and standardizing the resulting predictions.</p></sec><sec><title>Main Outcome(s) and Measure(s)</title><p>7-point ordinal scale, measured between day 28 and 35 post-enrollment.</p></sec><sec><title>Results</title><p>Eight of 19 trials met eligibility criteria and agreed to participate. Patient-level data were available from 770 participants (412 HCQ/CQ vs 358 control). Baseline characteristics were similar between groups. We found no evidence of a difference in ordinal scores between days 28 and 35 post-enrollment in the pooled patient population (odds ratio, 0.97; 95% credible interval, 0.76-1.24; higher favors HCQ/CQ), and no convincing evidence of meaningful treatment effect heterogeneity among prespecified subgroups. Adverse event and serious adverse event rates were numerically higher with HCQ/CQ vs control (0.39 vs 0.29 and 0.13 vs 0.09 per patient, respectively).</p></sec><sec><title>Conclusions and Relevance</title><p>The findings of this IPD meta-analysis reinforce those of individual RCTs that HCQ/CQ is not efficacious for treatment of COVID-19 in hospitalized patients.</p></sec><sec><title>Key Points</title><sec><title>Question</title><p>Does hydroxychloroquine/chloroquine benefit hospitalized patients with COVID-19, overall or within prespecified subgroups?</p></sec><sec><title>Findings</title><p>In this individual participant data meta-analysis of 770 hospitalized COVID-19 patients across 8 clinical trials, we did not find evidence of a benefit of hydroxychloroquine/chloroquine, measured by a 7-point ordinal COVID-19 severity score at 28-35 days post-enrollment, in the pooled study population. We also found no substantial treatment effect heterogeneity among prespecified patient subgroups.</p></sec><sec><title>Meaning</title><p>This study supports the consensus that hydroxychloroquine/chloroquine should not be used to treat hospitalized patients with COVID-19.</p></sec></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475830', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Loss of plastid developmental genes coincides with a reversion to monoplastidy in hornworts', 'authors': \"[{'name': 'Alexander I. MacLeod'}, {'name': 'Parth K. Raval'}, {'name': 'Simon Stockhorst'}, {'name': 'Michael R. Knopp'}, {'name': 'Eftychios Frangedakis'}, {'name': 'Sven B. Gould'}]\", 'abstract': '<p>The first plastid evolved from an endosymbiotic cyanobacterium in the common ancestor of the Archaeplastida. The transformative steps from cyanobacterium to organelle included the transfer of control over developmental processes; a necessity for the host to orchestrate, for example, the fission of the organelle. The plastids of almost all embryophytes divide independent from nuclear division, leading to cells housing multiple plastids. Hornworts, however, are monoplastidic (or near-monoplastidic) and their photosynthetic organelles are a curious exception among embryophytes for reasons such as the occasional presence of pyrenoids. Here we screened genomic and transcriptomic data of eleven hornworts for components of plastid developmental pathways. We find intriguing differences among hornworts and specifically highlight that pathway components involved in regulating plastid development and biogenesis were differentially lost in this group of bryophytes. In combination with ancestral state reconstruction, our data suggest that hornworts have reverted back to a monoplastidic phenotype due to the combined loss of two plastid division-associated genes: ARC3 and FtsZ2.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.476034', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'CCPLS reveals cell-type-specific spatial dependence of transcriptomes in single cells', 'authors': \"[{'name': 'Takaho Tsuchiya'}, {'name': 'Hiroki Hori'}, {'name': 'Haruka Ozaki'}]\", 'abstract': '<sec><title>Motivation</title><p>Cell-cell communications regulate internal cellular states of the cell, e.g., gene expression and cell functions, and play pivotal roles in normal development and disease states. Furthermore, single-cell RNA sequencing methods have revealed cell-to-cell expression variability of highly variable genes (HVGs), which is also crucial. Nevertheless, the regulation on cell-to-cell expression variability of HVGs via cell-cell communications is still unexplored. The recent advent of spatial transcriptome measurement methods has linked gene expression profiles to the spatial context of single cells, which has provided opportunities to reveal those regulations. The existing computational methods extract genes with expression levels that are influenced by neighboring cell types based on the spatial transcriptome data. However, limitations remain in the quantitativeness and interpretability: it neither focuses on HVGs, considers cooperation of neighboring cell types, nor quantifies the degree of regulation with each neighboring cell type.</p></sec><sec><title>Results</title><p>Here, we propose CCPLS (Cell-Cell communications analysis by Partial Least Square regression modeling), which isa statistical framework for identifying cell-cell communications as the effects of multiple neighboring cell types on cell-to-cell expression variability of HVGs, based on the spatial transcriptome data. For each cell type, CCPLS performs PLS regression modeling and reports coefficients as the quantitative index of the cell-cell communications. Evaluation using simulated data showed our method accurately estimated the effects of multiple neighboring cell types on HVGs. Furthermore, by applying CCPLS to the two real datasets, we demonstrate CCPLS can be used to extract biologically interpretable insights from the inferred cell-cell communications.</p></sec><sec><title>Availability</title><p>The R package is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/bioinfo-tsukuba/CCPLS\">https://github.com/bioinfo-tsukuba/CCPLS</ext-link>. The data are available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/bioinfo-tsukuba/CCPLS_paper\">https://github.com/bioinfo-tsukuba/CCPLS_paper</ext-link>.</p></sec><sec><title>Contact</title><p><email>haruka.ozaki@md.tsukuba.ac.jp</email></p></sec><sec><title>Supplementary information</title><p>Supplementary data are available at Bioinformatics online.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475721', 'tweets': 25, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Sex differences in pain-related behaviors and clinical progression of disease in mouse models of visceral pain', 'authors': \"[{'name': 'Adela M. Francis-Malave'}, {'name': 'Santiago Martinez Gonzalez'}, {'name': 'Caren Pichardo'}, {'name': 'Torri D. Wilson'}, {'name': 'Luis G. Rivera'}, {'name': 'Lauren R. Brinster'}, {'name': 'Yarimar Carrasquillo'}]\", 'abstract': '<p>Previous studies have reported sex differences in irritable bowel syndrome (IBS) and inflammatory bowel disease (IBD) patients, including differences in visceral pain perception. Despite this, sex differences in behavioral manifestations of visceral pain and underlying pathology of the gastrointestinal tract have been largely understudied in preclinical research. In this study, we evaluated potential sex differences in spontaneous visceral nociceptive responses, referred abdominal hypersensitivity, disease progression and bowel pathology in mouse models of acute and persistent colon inflammation. Our experiments show that females exhibit more visceral nociceptive responses and referred abdominal hypersensitivity than males in the context of acute but not persistent colon inflammation. We further demonstrate that, following acute and persistent colon inflammation, visceral pain-related behavioral responses in females and males are distinct, with increases in licking of the abdomen only observed in females and increases in abdominal contractions only seen in males. During persistent colon inflammation, males exhibit worse disease progression than females, which is manifested as worse physical appearance and higher weight loss. However, no measurable sex differences were observed in persistent inflammation-induced bowel pathology, stool consistency or fecal blood. Overall, our findings demonstrate that visceral pain-related behaviors and disease progression in the context of acute and persistent colon inflammation are sex-dependent, highlighting the importance of considering sex as a biological variable in future mechanistic studies of visceral pain as well as in the development of diagnostics and therapeutic options for chronic gastrointestinal diseases.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475842', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A new class of protein sensor links spirochete pleomorphism, persistence, and chemotaxis', 'authors': \"[{'name': 'A.R. Muok'}, {'name': 'K. Kurniyati'}, {'name': 'D.R. Ortega'}, {'name': 'F.A. Olsthoorn'}, {'name': 'A. Sidi Mabrouk'}, {'name': 'C. Li'}, {'name': 'A. Briegel'}]\", 'abstract': '<p>Pathogenic spirochetes can alter their morphologies and behaviors to infect and survive within their hosts. Previous reports demonstrate that the formation of so-called ‘round bodies’ and biofilms, and chemotaxis are involved in spirochete pathogenesis. Here, in the spirochete <italic>Treponema denticola</italic>, we report a direct link between these cellular states that involves a new class of protein sensor (CheWS) with hitherto unclear function. Using cryo-EM methods, protein modeling, bioinformatics, genetics methods, and behavioral assays we demonstrate that spirochetes regulate these behaviors in response to the small molecule s-adenosylmethionine (SAM) via a SAM sensor that is anchored to chemotaxis arrays. CheWS influences chemotaxis, biofilm and round body formation under nonstressed conditions by a novel sporulation-like mechanism. Taken together, we establish an improved model for round body formation, we discovered a direct link between this SAM sensor and changes in cellular states, as well as characterized a new sensor class involved in chemotaxis.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.12.475944', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Epigenetic dysregulation from chromosomal transit in micronuclei', 'authors': \"[{'name': 'Albert Agustinus'}, {'name': 'Ramya Raviram'}, {'name': 'Bhargavi Dameracharla'}, {'name': 'Jens Luebeck'}, {'name': 'Stephanie Stransky'}, {'name': 'Lorenzo Scipioni'}, {'name': 'Robert M. Myers'}, {'name': 'Melody Di Bona'}, {'name': 'Mercedes Duran'}, {'name': 'Britta Weigelt'}, {'name': 'Shira Yomtoubian'}, {'name': 'Eléonore Toufektchan'}, {'name': 'Paul S. Mischel'}, {'name': 'Vivek Mittal'}, {'name': 'Sohrab Shah'}, {'name': 'John Maciejowski'}, {'name': 'Enrico Gratton'}, {'name': 'Peter Ly'}, {'name': 'Mathieu F. Bakhoum'}, {'name': 'Dan Landau'}, {'name': 'Vineet Bafna'}, {'name': 'Simone Sidoli'}, {'name': 'Yael David'}, {'name': 'Samuel F. Bakhoum'}]\", 'abstract': '<p>Chromosomal instability (CIN) and epigenetic alterations are characteristics of advanced and metastatic cancers [1-4], yet whether they are mechanistically linked is unknown. Here we show that missegregation of mitotic chromosomes, their sequestration in micronuclei [5, 6], and subsequent micronuclear envelope rupture [7] profoundly disrupt normal histone post-translational modifications (PTMs), a phenomenon conserved across humans and mice as well as cancer and non-transformed cells. Some of the changes to histone PTMs occur due to micronuclear envelope rupture whereas others are inherited from mitotic abnormalities prior to micronucleus formation. Using orthogonal techniques, we show that micronuclei exhibit extensive differences in chromatin accessibility with a strong positional bias between promoters and distal or intergenic regions. Finally, we show that inducing CIN engenders widespread epigenetic dysregulation and that chromosomes which transit in micronuclei experience durable abnormalities in their accessibility long after they have been reincorporated into the primary nucleus. Thus, in addition to genomic copy number alterations, CIN can serve as a vehicle for epigenetic reprogramming and heterogeneity in cancer.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476226', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'The minimizer Jaccard estimator is biased and inconsistent', 'authors': \"[{'name': 'Mahdi Belbasi'}, {'name': 'Antonio Blanca'}, {'name': 'Robert S. Harris'}, {'name': 'David Koslicki'}, {'name': 'Paul Medvedev'}]\", 'abstract': '<p>Sketching is now widely used in bioinformatics to reduce data size and increase data processing speed. Sketching approaches entice with improved scalability but also carry the danger of decreased accuracy and added bias. In this paper, we investigate the minimizer sketch and its use to estimate the Jaccard similarity between two sequences. We show that the minimizer Jaccard estimator is biased and inconsistent, which means that the expected difference (i.e., the bias) between the estimator and the true value is not zero, even in the limit as the lengths of the sequences grow. We derive an analytical formula for the bias as a function of how the shared k-mers are laid out along the sequences. We show both theoretically and empirically that there are families of sequences where the bias can be substantial (e.g. the true Jaccard can be more than double the estimate). Finally, we demonstrate that this bias affects the accuracy of the widely used mashmap read mapping tool.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.11.22269097', 'tweets': 21, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'High-cited favorable studies for COVID-19 treatments ineffective in large trials', 'authors': \"[{'name': 'John P.A. Ioannidis'}]\", 'abstract': '<p>Importance. COVID-19 has resulted in massive production, publication and wide dissemination of clinical studies trying to identify effective treatments. However, several widely touted treatments failed to show effectiveness in large well-done randomized controlled trials (RCTs). \\nObjective. To evaluate for COVID-19 treatments that showed no benefits in subsequent large RCTs how many of their most-cited clinical studies had declared favorable results for these interventions. \\nMethods. Scopus (last update December 23, 2021) identified articles on lopinavir-ritonavir, hydroxycholoroquine/azithromycin, remdesivir, convalescent plasma, colchicine or interferon (index interventions) that represented clinical trials and that had received &gt;150 citations. Their conclusions were assessed and correlated with study design features. The ten most recent citations for the most-cited article on each index intervention were examined on whether they were critical to the highly-cited study. Altmetric scores were also obtained. \\nFindings. 40 articles of clinical studies on these index interventions had received &gt;150 citations (7 exceeded 1,000 citations). 20/40 (50%) had favorable conclusions and 4 were equivocal. Highly-cited articles with favorable conclusions were rarely RCTs while those without favorable conclusions were mostly RCTs (3/20 vs 15/20, p=0.0003). Only 1 RCT with favorable conclusions had sample size &gt;160. Citation counts correlated strongly with Altmetric scores, in particular news items. Only 9 (15%) of 60 recent citations to the most highly-cited studies with favorable or equivocal conclusions were critical to the highly-cited study. \\nConclusion. Many clinical studies with favorable conclusions for largely ineffective COVID-19 treatments are uncritically heavily cited and disseminated. Early observational studies and small randomized trials may cause spurious claims of effectiveness that get perpetuated.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.12.475507', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Glucocorticoid Receptor-Regulated Enhancers Play a Central Role in the Gene Regulatory Networks Underlying Drug Addiction', 'authors': \"[{'name': 'Sascha H. Duttke'}, {'name': 'Patricia Montilla-Perez'}, {'name': 'Max W. Chang'}, {'name': 'Hairi Li'}, {'name': 'Hao Chen'}, {'name': 'Lieselot L. G. Carrette'}, {'name': 'Giordano de Guglielmo'}, {'name': 'Olivier George'}, {'name': 'Abraham A. Palmer'}, {'name': 'Christopher Benner'}, {'name': 'Francesca Telese'}]\", 'abstract': '<p>Substance abuse and addiction represent a significant public health problem that impacts multiple dimensions of society, including healthcare, the economy, and the workforce. In 2021, over 100,000 drug overdose deaths were reported in the US, with an alarming increase in fatalities related to opioids and psychostimulants. Understanding the fundamental gene regulatory mechanisms underlying addiction and related behaviors could facilitate more effective treatments. To explore how repeated drug exposure alters gene regulatory networks in the brain, we combined capped small (cs)RNA-seq, which accurately captures nascent-like initiating transcripts from total RNA, with Hi-C and single nuclei (sn)ATAC-seq. We profiled initiating transcripts in two addiction-related brain regions, the prefrontal cortex (PFC) and the nucleus accumbens (NAc), from rats that were never exposed to drugs or were subjected to prolonged abstinence after oxycodone or cocaine intravenous self-administration (IVSA). Interrogating over 100,000 active transcription start regions (TSRs) revealed that most TSRs had hallmarks of bonafide enhancers and highlighted the KLF/SP1, RFX, and AP1 transcription factors families as central to establishing brain-specific gene regulatory programs. Analysis of rats with addiction-like behaviors versus controls identified addiction-associated repression of transcription at regulatory enhancers recognized by nuclear receptor subfamily 3 group C (NR3C) factors, which include glucocorticoid receptors. Cell-type deconvolution analysis using snATAC-seq uncovered a potential role of glial cells in driving the gene regulatory programs associated with addiction-related phenotypes. These findings highlight the power of advanced transcriptomics methods to provide insight into how addiction perturbs gene regulatory programs in the brain.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475847', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'High-fructose feeding suppresses cold-stimulated brown adipose tissue glucose uptake in young men independently of changes in thermogenesis and the gut microbiome', 'authors': \"[{'name': 'Gabriel Richard'}, {'name': 'Denis P. Blondin'}, {'name': 'Saad A. Syed'}, {'name': 'Laura Rossi'}, {'name': 'Michelle E. Fontes'}, {'name': 'Mélanie Fortin'}, {'name': 'Serge Phoenix'}, {'name': 'Frédérique Frisch'}, {'name': 'Stéphanie Dubreuil'}, {'name': 'Brigitte Guérin'}, {'name': 'Éric E. Turcotte'}, {'name': 'Martin Lepage'}, {'name': 'Michael G. Surette'}, {'name': 'Jonathan D. Schertzer'}, {'name': 'Gregory R. Steinberg'}, {'name': 'Katherine M. Morrison'}, {'name': 'André C. Carpentier'}]\", 'abstract': '<p>Diets rich in added sugars, especially high in fructose, are associated with metabolic diseases such as insulin resistance, and non-alcoholic fatty liver disease. Studies have shown a link between these pathologies and changes in the microbiome and its metabolites. Given the reported associations in animal models between the microbiome and brown or beige adipose tissue (BAT) function, and the alterations in the microbiome induced by high glucose or high fructose diets, we investigated the potential causal link between high glucose or fructose diets and BAT dysfunction in humans. We show that BAT glucose uptake, but not thermogenesis, is impaired by a high fructose but not high glucose diet, in the absence of changes in body mass, the gastrointestinal microbiome, and faecal short-chain fatty acids. We conclude that BAT metabolic dysfunction occurs independently from changes in gut microbiome composition, and earlier than other pathophysiological abnormalities associated with insulin resistance and dyslipidemia during fructose overconsumption in humans.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475889', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': True, 'title': 'Favipiravir, umifenovir and camostat mesylate: a comparative study against SARS-CoV-2', 'authors': \"[{'name': 'Mehnmet Altay Unal'}, {'name': 'Omur Besbinar'}, {'name': 'Hasan Nazir'}, {'name': 'Gokce Yagmur Summak'}, {'name': 'Fatma Bayrakdar'}, {'name': 'Lucia Gemma Delogu'}, {'name': 'Tambay Taskin'}, {'name': 'Sibel Aysil Ozkan'}, {'name': 'Kamil Can Akcali'}, {'name': 'Acelya Yilmazer'}]\", 'abstract': '<p>Since the first cases the coronavirus disease caused by SARS-CoV-2 (COVID-19) reported in December 2019, worldwide continuous efforts have been placed both for the prevention and treatment of this infectious disease. As new variants of the virus emerge, the need for an effective antiviral treatment continues. The concept of preventing SARS-CoV-2 on both pre-entry and post-entry stages has not been much studied. Therefore, we compared the antiviral activities of three antiviral drugs which have been currently used in the clinic. In silico docking analyses and in vitro viral infection in Vero E6 cells were performed to delineate their antiviral effectivity when used alone or in combination. Both in silico and in vitro results suggest that the combinatorial treatment by favipiravir and umifenovir or camostat mesylate has more antiviral activity against SARS-CoV-2 rather than single drug treatment. These results suggest that inhibiting both viral entry and viral replication at the same time is much more effective for the antiviral treatment of SARS-CoV-2.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.15.474735', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Comprehensive assessment of functional effects of commonly used sweeteners on ex vivo human gut microbiome', 'authors': \"[{'name': 'Zhongzhi Sun'}, {'name': 'Wenju Wang'}, {'name': 'Leyuan Li'}, {'name': 'Xu Zhang'}, {'name': 'Zhibin Ning'}, {'name': 'Janice Mayne'}, {'name': 'Krystal Walker'}, {'name': 'Alain Stintzi'}, {'name': 'Daniel Figeys'}]\", 'abstract': '<p>The gut microbiome composition and function are associated with health and diseases. Sweeten-ers are widely used food additives, although many studies using animal models have linked sweetener consumption to gut microbial changes and health issues. Whether sweeteners directly change the human gut microbiome functionality remains largely unknown. In this study, we sys-tematically investigated the responses of five human gut microbiomes to 21 common sweeteners, using an approach combining high-throughput ex vivo microbiome culturing and metaproteomics to quantify functional changes in different taxa. Hierarchical clustering based on metaproteomic responses of individual microbiomes resulted in two clusters. The first cluster was composed of non-caloric artificial sweeteners (NAS) and two sugar alcohols with shorter carbon backbones (4-5 carbon atoms), and the second cluster was composed of sugar alcohols with longer carbon backbones. The metaproteomic functional responses of the second cluster were similar to the prebiotic fructooligosaccharides and kestose, indicating that these sugar alcohol-type sweeteners have potential prebiotic functions. This study provides a comprehensive evaluation of the direct effects of commonly used sweeteners on the functions of the human gut microbiome using a func-tional  metaproteomics approach, improving our understanding of the roles of sweeteners on mi-crobiome-associated human health and disease issues.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.11.475331', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Foveal vision predictively sensitizes to defining features of eye movement targets', 'authors': \"[{'name': 'Lisa M. Kroell'}, {'name': 'Martin Rolfs'}]\", 'abstract': '<p>Despite the fovea’s singular importance for active human vision, the impact of large eye movements on foveal processing remains elusive. Building on findings from passive fixation tasks, we hypothesized that during the preparation of rapid eye movements (saccades), foveal processing anticipates soon-to-be fixated visual features. Using a dynamic large-field noise paradigm, we indeed demonstrate that sensitivity for defining features of a saccade target is enhanced in the pre-saccadic center of gaze. Enhancement manifested in higher Hit Rates for foveal probes with target-congruent orientation, and a sensitization to incidental, target-like orientation information in foveally presented noise. Enhancement was spatially confined to the center of gaze and its immediate vicinity. We suggest a crucial contribution of foveal processing to trans-saccadic visual continuity which has previously been overlooked: Foveal processing of saccade targets commences before the movement is executed and thereby enables a seamless transition once the center of gaze reaches the target.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476216', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The RNA polymerase gene specificity factor σ54 is required for homogeneous non-planktonic growth of uropathogenic Escherichia coli', 'authors': \"[{'name': 'Amy Switzer'}, {'name': 'Lynn Burchell'}, {'name': 'Panagiotis Mitsidis'}, {'name': 'Sivaramesh Wigneshweraraj'}]\", 'abstract': '<p>The canonical function of a bacterial sigma (σ) factor is to determine the gene specificity of the RNA polymerase (RNAP). In several diverse bacterial species, the σ<sup>54</sup> factor uniquely confers distinct functional and regulatory properties on the RNAP. A hallmark feature of the σ<sup>54</sup>-RNAP is the obligatory requirement for an activator ATPase to allow transcription initiation. The genes that rely upon σ<sup>54</sup> for their transcription have a wide range of different functions suggesting that the repertoire of functions performed by genes, directly or indirectly affected by σ<sup>54</sup>, is not yet exhaustive. By comparing the non-planktonic growth properties of prototypical enteropathogenic, uropathogenic and non-pathogenic <italic>Escherichia coli</italic> strains devoid of σ<sup>54</sup>, we uncovered σ<sup>54</sup> as a determinant of homogenous non-planktonic growth specifically in the uropathogenic strain. Notably, bacteria devoid of individual activator ATPases of the σ<sup>54</sup>-RNAP do not phenocopy the σ<sup>54</sup> mutant strain. It seems that σ<sup>54</sup>’s role as a determinant of homogenous non-planktonic growth represents a putative non-canonical function of σ<sup>54</sup> in regulating genetic information flow.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.16.475527', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Cryo-EM reconstruction of the human 40S ribosomal subunit at 2.15 Å resolution', 'authors': \"[{'name': 'Simone Pellegrino'}, {'name': 'Kyle C Dent'}, {'name': 'Tobias Spikes'}, {'name': 'Alan J Warren'}]\", 'abstract': '<p>The chemical modification of ribosomal RNA and proteins is critical for ribosome assembly, for protein synthesis and may drive ribosome specialization in development and disease. However, the inability to accurately visualize these modifications has limited mechanistic understanding of the role of these modifications in ribosome function. Here we report the 2.15 Å resolution cryo-EM reconstruction of the human 40S ribosomal subunit. We directly visualize post-transcriptional modifications within the 18S rRNA and post-translational modifications at the N-termini of two ribosomal proteins. Additionally, we interpret the solvation shells in the core regions of the 40S ribosomal subunit and reveal how potassium and magnesium ions establish both universally conserved and eukaryote-specific coordination to promote the stabilization and folding of key ribosomal elements. This work provides unprecedented structural details for the human 40S ribosomal subunit that will serve as an important reference for unraveling the functional role of ribosomal RNA modifications.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.10.475722', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'UV-C light completely blocks highly contagious Delta SARS-CoV-2 aerosol transmission in hamsters', 'authors': \"[{'name': 'Robert J. Fischer'}, {'name': 'Julia R. Port'}, {'name': 'Myndi G. Holbrook'}, {'name': 'Kwe Claude Yinda'}, {'name': 'Martin Creusen'}, {'name': 'Jeroen ter Stege'}, {'name': 'Marc de Samber'}, {'name': 'Vincent J. Munster'}]\", 'abstract': '<p>Behavioral and medical control measures are not effective in containing the spread of SARS-CoV-2. Here we report on the effectiveness of a preemptive environmental strategy using UV-C light to prevent airborne transmission of the virus in a hamster model and show that UV-C exposure completely prevents airborne transmission between individuals</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.476120', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'An antibody targeting the N-terminal domain of SARS-CoV-2 disrupts the spike trimer', 'authors': \"[{'name': 'Naveenchandra Suryadevara'}, {'name': 'Andrea R. Shiakolas'}, {'name': 'Laura A. VanBlargan'}, {'name': 'Elad Binshtein'}, {'name': 'Rita E. Chen'}, {'name': 'James Brett Case'}, {'name': 'Kevin J. Kramer'}, {'name': 'Erica Armstrong'}, {'name': 'Luke Myers'}, {'name': 'Andrew Trivette'}, {'name': 'Christopher Gainza'}, {'name': 'Rachel S. Nargi'}, {'name': 'Christopher N. Selverian'}, {'name': 'Edgar Davidson'}, {'name': 'Benjamin J. Doranz'}, {'name': 'Summer M. Diaz'}, {'name': 'Laura S. Handal'}, {'name': 'Robert H. Carnahan'}, {'name': 'Michael S. Diamond'}, {'name': 'Ivelin S. Georgiev'}, {'name': 'James E. Crowe'}]\", 'abstract': '<p>The protective human antibody response to the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus focuses on the spike (S) protein which decorates the virion surface and mediates cell binding and entry. Most SARS-CoV-2 protective antibodies target the receptor- binding domain or a single dominant epitope (‘supersite’) on the N terminal domain (NTD). Here, using the single B cell technology LIBRA-seq, we isolated a large panel of NTD-reactive and SARS-CoV-2 neutralizing antibodies from an individual who had recovered from COVID-19. We found that neutralizing antibodies to the NTD supersite commonly are encoded by the <italic>IGHV1-24</italic> gene, forming a genetic cluster that represents a public B cell clonotype. However, we also discovered a rare human antibody, COV2-3434, that recognizes a site of vulnerability on the SARS-CoV-2 S protein in the trimer interface and possesses a distinct class of functional activity. COV2-3434 disrupted the integrity of S protein trimers, inhibited cell-to-cell spread of virus in culture, and conferred protection in human ACE2 transgenic mice against SARS-CoV-2 challenge. This study provides insight about antibody targeting of the S protein trimer interface region, suggesting this region may be a site of virus vulnerability.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.11.22268736', 'tweets': 17, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'COVID-19 vaccination breakthrough infections in a real-world setting: Using community reporters to evaluate vaccine effectiveness', 'authors': \"[{'name': 'Matthew W Reynolds'}, {'name': 'Yiqiong Xie'}, {'name': 'Kendall B Knuth'}, {'name': 'Christina D Mack'}, {'name': 'Emma Brinkley'}, {'name': 'Stephen Toovey'}, {'name': 'Nancy A Dreyer'}]\", 'abstract': '<sec><title>Background</title><p>COVID-19 has highlighted the need for new methods of pharmacovigilance. Here we use community volunteers to obtain systematic information on vaccine effectiveness and the nature and severity of breakthrough infections.</p></sec><sec><title>Methods</title><p>Between December 15, 2020 to September 16, 2021, 10,412 unpaid community-based participants reported the following information to an on-line registry: COVID-19 test results, vaccination (Pfizer, Moderna, or Johnson &amp; Johnson), COVID-19 symptoms and perceived severity using a 4-point scale. COVID-19 infections were described for those who were 1) fully vaccinated, 2) partially vaccinated (received first of two dose vaccines or were &lt;14 days post-final dose), or 3) unvaccinated.</p></sec><sec><title>Results</title><p>Of 8,554 who were vaccinated, COVID-19 infections were reported by 74 (1.0%) of those who were fully vaccinated and 198 (2.3%) of those who were partially vaccinated. Among the 74 participants who reported a breakthrough infection after full vaccination, the median time to reported positive test result was 104.5 days (Interquartile range: 77-135 days), with no difference among vaccine manufactures. One quarter (25.7%) of breakthrough infections in the fully vaccinated cases were asymptomatic. More than 97% of fully vaccinated participants reported no moderate/severe symptoms compared to 89.3% of the unvaccinated cases; and only 1.4% of fully vaccinated participants reported experiencing at least 3 moderate to severe symptoms compared to 7.8% in the unvaccinated.</p></sec><sec><title>Conclusion</title><p>Person-generated health data, also referred to as patient-reported outcomes, is a useful resource for quantifying breakthrough infections and their severity, showing here that fully vaccinated participants report no or very mild COVID-19 symptoms.</p></sec><sec><title>Trial registration</title><p><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://Clinicaltrials.gov\">Clinicaltrials.gov</ext-link><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04368065\">NCT04368065</ext-link>, EU PAS Register EUPAS36240</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.13.22269234', 'tweets': 17, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Increased airborne transmission of COVID-19 with new variants. Implications for health policies.', 'authors': \"[{'name': 'Bertrand R Rowe'}, {'name': 'Andre Canosa'}, {'name': 'Amina Meslem'}, {'name': 'Frantz Rowe'}]\", 'abstract': '<p>The aim of the present paper is to highlight that new variants, either of higher viral load such as delta or higher contagiousness like omicron, lead to an even higher airborne transmission than historical strains. We first review the question of the route of contamination and of the dose following exposure, and the importance of the counting unit for pathogens, itself linked to the dose-response law. Using the counting unit of Wells, i.e. the quantum of contagium, we develop the conservation equation of quanta which allows deriving the value of the quantum concentration at steady state for a well-mixed room. With the choice of an exponential response function, this leads to the famous Wells-Riley equation. The analysis points out the importance of a number of parameters such as the time of exposure, the quantum production rate, mask wearing and the incidence rate in the population in order to evaluate the risk. The link with the monitoring concentration of carbon dioxide is made and used for a risk analysis of a variety of situations for which we have made concrete CO2 time monitoring. The main conclusion of these observations is that the present norms of ventilation, already insufficient, are not respected, especially in a variety of public premises, leading to high risk of contamination. Finally, we insist that public health policy in the field of airborne transmission should be based on a multi parameter analysis, considering the whole complexity of dose evaluation.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.13.476148', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Identification of candidate virulence loci in Striga hermonthica, a devastating parasite of African cereal crops', 'authors': \"[{'name': 'Suo Qiu'}, {'name': 'James M. Bradley'}, {'name': 'Peijun Zhang'}, {'name': 'Roy Chaudhuri'}, {'name': 'Mark Blaxter'}, {'name': 'Roger K. Butlin'}, {'name': 'Julie Diane Scholes'}]\", 'abstract': '<p>Parasites have evolved suites of proteins, Virulence Factors (VFs), that are delivered into host plants to facilitate colonization. Whilst VFs mediating plant-microbe and plant-nematode interactions have been characterised extensively, less is known about VFs mediating parasitic plant interactions with their hosts. Striga hermonthica is an obligate, root-parasitic plant capable of parasitizing multiple cereal hosts in sub-Saharan Africa, causing devastating losses in yields. An understanding of the molecular nature and allelic variation of VFs in S. hermonthica is essential for breeding durable resistance and delaying the evolution of parasite virulence. To address this issue, we assembled a genome for Striga hermonthica and identified candidate VFs by combining in silico prediction of secreted proteins with pooled sequencing of parasites growing on a susceptible and a strongly resistant rice host. Consistent with predictions for parasites, like S. hermonthica, that can interact with multiple hosts, we identified multiple loci, potentially with a wide range of functions, implicated in overcoming host resistance. Extremely different allele frequencies were observed at 152 non-secreted and 38 putatively secreted VFs between S. hermonthica parasitising the resistant and susceptible rice varieties. Our candidate, secreted VFs encompassed functions such as host cell wall modification, protease inhibitors, oxidoreductase and kinase activities, as well as several with unknown functions. Consistent with maintenance of variation at virulence loci by balancing selection the candidate loci had significantly higher Tajima’s D on average than the genomic background.\\xa0Our results show that diverse strategies are used by\\xa0  S. hermonthica  \\xa0to overcome different layers of host resistance.\\xa0Understanding the maintenance of variation at virulence loci by balancing selection will be critical to managing the evolution of virulence as a part of a sustainable control strategy.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.13.476230', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': '3D-imaging reveals conserved cerebrospinal fluid drainage via meningeal lymphatic vasculature in mice and humans', 'authors': \"[{'name': 'Laurent Jacob'}, {'name': 'Jose de Brito Neto'}, {'name': 'Stephanie Lenck'}, {'name': 'Celine Corcy'}, {'name': 'Farhat Benbelkacem'}, {'name': 'Luiz Henrique Medeiros Geraldo'}, {'name': 'Yunling Xu'}, {'name': 'Jean-Mickael Thomas'}, {'name': 'Marie-Renee El Kamouh'}, {'name': 'Marie-Claude Potier'}, {'name': 'Stephane Haik'}, {'name': 'Stephane Lehericy'}, {'name': 'Anne Eichmann'}, {'name': 'Jean-Leon Thomas'}]\", 'abstract': '<p>Meningeal lymphatic vessels (MLVs) contribute to waste product elimination and immune surveillance in brain tissues. MLVs were identified in the dorsal and caudo-basal regions of the dura mater, where they ensure the clearance of cerebrospinal fluid (CSF). Whether MLVs exist in the complex anterior part of the murine and human skull, and how they connect with the glymphatic system and extracranial lymphatic vasculature remained unclear. Here, we generated three-dimensional (3D) maps of MLV drainage by light-sheet fluorescence microscopy (LSFM) imaging of mouse whole-head preparations following fluorescent OVA-A555 tracer injections into the CSF. In humans, we performed real-time magnetic resonance vessel wall imaging (MR-VWI) after systemic gadobutrol injections. We observed a conserved 3D-anatomy of MLVs in mice and humans, and we discovered an extended anterior network around the dural cavernous sinus including multiple capillary beds and exit routes through the foramina of emissary veins. MR-VWI may provide a diagnostic tool for patients with CSF drainage defects and neurological diseases.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.12.476129', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'The blowfly  Chrysomya latifrons  inhabits fragmented rainforests, but lacks genetic diversity and population structure', 'authors': \"[{'name': 'Nathan J Butterworth'}, {'name': 'James F Wallman'}, {'name': 'Nikolas P Johnston'}, {'name': 'Blake M Dawson'}, {'name': 'Angela McGaughran'}]\", 'abstract': '<p>Climate change and deforestation are causing rainforests to become increasingly fragmented, placing them at heightened risk of biodiversity loss. Invertebrates constitute the greatest proportion of this biodiversity, yet we lack basic knowledge of their population structure and ecology. It is not currently feasible to assess the population structure of every invertebrate species, so there is a compelling need to identify indicator species that are broadly indicative of habitat-level patterns and processes. Blowflies are an ideal candidate, because they are widespread, abundant, and can be easily collected within rainforests. Here, we present the first study of the blowfly <italic> Chrysomya latifrons </italic>, which is endemic to the rainforests of New South Wales, Australia. We genotyped 188 flies from 15 isolated rainforests and found low overall genetic diversity and a complete lack of genetic structure between populations, suggesting the presence of a single large panmictic population along 1,000 km of the Australian east coast. This highlights that: (1) <italic> Ch. latifrons </italic> inhabits every rainforest in NSW and undoubtedly plays an important role in these ecosystems, but low genetic diversity may cause it to struggle to adapt to a changing climate; (2) strongly dispersing insects have the capacity to migrate between isolated rainforests, likely carrying pollen, parasites, phoronts, and pathogens with them to form crucial trophic networks; and (3) there is an urgent need for similar studies on poorly dispersing rainforest insects, as these may be the most fragmented and at highest risk of local extinction.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475823', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Pathological relevance of post-translationally modified alpha-synuclein (pSer87, pSer129, nTyr39) in idiopathic Parkinson’s disease and Multiple System Atrophy', 'authors': \"[{'name': 'Berkiye Sonustun'}, {'name': 'Firat M Altay'}, {'name': 'Catherine Strand'}, {'name': 'Geshanthi Hondhamuni'}, {'name': 'Thomas T Warner'}, {'name': 'Hilal A Lashuel'}, {'name': 'Rina Bandopadhyay'}]\", 'abstract': '<p>Aggregated alpha-synuclein (α-synuclein) is the main component of Lewy bodies (LBs), Lewy neurites (LNs), and glial cytoplasmic inclusions (GCIs), which are pathological hallmarks of idiopathic Parkinson’s disease (IPD) and multiple system atrophy (MSA), respectively. Initiating factors that culminate in forming LBs/LNs/GCIs remain elusive. Several species of α-synuclein exist, including phosphorylated and nitrated forms. It is unclear which α-synuclein post-translational modifications (PTMs) appear within aggregates throughout disease pathology. Herein we aimed to establish the predominant α–synuclein PTMs in post-mortem IPD and MSA pathology using immunohistochemistry. We examined the patterns of three α-synuclein PTMs (pS87, pS129, nY39) simultaneously in pathology-affected regions of 15 PD, 5 MSA, 6 neurologically normal controls. All antibodies recognized LBs, LNs, and GCIs, albeit to a variable extent. pS129 α-synuclein antibody was particularly immunopositive for LNs and synaptic dot-like structures followed by nY39 α-synuclein antibody. GCIs, neuronal inclusions, and small threads were positive for nY39 α-synuclein in MSA. Quantification of the LB scores revealed that pS129 α-synuclein was the dominant and earliest α-synuclein PTM followed by nY39 α-synuclein, while lower amounts of pSer87 α-synuclein appeared later in disease progression in PD. These results may have implications for novel biomarker and therapeutic developments.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475875', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'The RNA-Binding Protein SAM68 regulates cardiomyocyte differentiation by enhancing Gata4 translation', 'authors': \"[{'name': 'Alessandro Dasti'}, {'name': 'Maria Carla Antonelli'}, {'name': 'Magdalena Arnal Segura'}, {'name': 'Alexandros Armaos'}, {'name': 'Sarah Bonnin'}, {'name': 'Thomas Graf'}, {'name': 'Maria Paola Paronetto'}, {'name': 'Tian V Tian'}, {'name': 'Elias Bechara'}, {'name': 'Gian Gaetano Tartaglia'}]\", 'abstract': '<p>The signal transduction and activation of RNA (STAR) family is composed of RNA-binding proteins (RBPs) that play a central role in mammalian development. Nonetheless, the functions and modes of action that STAR proteins have in lineage specification are still poorly understood. Here, we characterized the role of STAR proteins SAM68 and QUAKING (QKI) in pluripotency and differentiation by performing their depletion through CRISPR-Cas9 in mouse embryonic stem cells (mESCs). Combining RNA-sequencing, ribosome profiling and advanced computational predictions, we found that both SAM68 and QKI regulate the mESCs self-renewal and are indispensable for cardiomyocyte differentiation. At the molecular level, we discovered that SAM68 and QKI antagonistically control the expression of cardiogenic factors. Our calculations indicated that SAM68, unlike QKI, binds the cardiogenic-specific transcription factor Gata4 in a region spanning nucleotides 500 to 1000 of the mRNA corresponding to part of the 5’ untranslated region and the first exon. We validated the predictions by electrophoretic mobility shift assay and RNA immunoprecipitation showing that SAM68 controls the translation of Gata4 during mESCs differentiation towards the cardiomyocyte lineage.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.14.476352', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'A semi-supervised Bayesian mixture modelling approach for joint batch correction and classification', 'authors': \"[{'name': 'Stephen Coleman'}, {'name': 'Xaquin Castro Dopico'}, {'name': 'Gunilla B. Karlsson Hedestam'}, {'name': 'Paul D.W. Kirk'}, {'name': 'Chris Wallace'}]\", 'abstract': '<p>Systematic differences between batches of samples present significant challenges when analysing biological data. Such <italic>batch effects</italic> are well-studied and are liable to occur in any setting where multiple batches are assayed. Many existing methods for accounting for these have focused on high-dimensional data such as RNA-seq and have assumptions that reflect this. Here we focus on batch-correction in low-dimensional classification problems. We propose a semi-supervised Bayesian generative classifier based on mixture models that jointly predicts class labels and models batch effects. Our model allows observations to be probabilistically assigned to classes in a way that incorporates uncertainty arising from batch effects. We explore two choices for the within-class densities: the multivariate normal and the multivariate <italic>t</italic>. A simulation study demonstrates that our method performs well compared to popular off-the-shelf machine learning methods and is also quick; performing 15,000 iterations on a dataset of 500 samples with 2 measurements each in 7.3 seconds for the MVN mixture model and 11.9 seconds for the MVT mixture model. We apply our model to two datasets generated using the enzyme-linked immunosorbent assay (ELISA), a spectrophotometric assay often used to screen for antibodies. The examples we consider were collected in 2020 and measure seropositivity for SARS-CoV-2. We use our model to estimate seroprevalence in the populations studied. We implement the models in C++ using a Metropolis-within-Gibbs algorithm; this is available in the R package at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/stcolema/BatchMixtureModel\">https://github.com/stcolema/BatchMixtureModel</ext-link>. Scripts to recreate our analysis are at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/stcolema/BatchClassifierPaper\">https://github.com/stcolema/BatchClassifierPaper</ext-link>.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476326', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Transposable elements contribute to the spatiotemporal microRNA landscape in human brain development', 'authors': \"[{'name': 'Christopher J. Playfoot'}, {'name': 'Shaoline Sheppard'}, {'name': 'Evarist Planet'}, {'name': 'Didier Trono'}]\", 'abstract': '<p>Transposable elements (TEs) contribute to the evolution of gene regulatory networks and are dynamically expressed throughout human brain development and disease. One gene regulatory mechanism influenced by TEs is the miRNA system of post-transcriptional control. miRNA sequences frequently overlap TE loci and this miRNA expression landscape is crucial for control of gene expression in adult brain and different cellular contexts. Despite this, a thorough investigation of the spatiotemporal expression of TE-embedded miRNAs in human brain development is lacking. Here, we identify a spatiotemporally dynamic TE-embedded miRNA expression landscape between childhood and adolescent stages of human brain development. These miRNAs sometimes arise from two apposed TEs of the same subfamily, such as for L2 or MIR elements, but in the majority of cases stem from solo TEs. They give rise to <italic>in silico</italic> predicted high-confidence pre-miRNA hairpin structures, likely represent functional miRNAs and have predicted genic targets associated with neurogenesis. TE-embedded miRNA expression is distinct in the cerebellum when compared to other brain regions, as has previously been described for gene and TE expression. Furthermore, we detect expression of previously non-annotated TE-embedded miRNAs throughout human brain development, suggestive of a previously undetected miRNA control network. Together, as with non-TE-embedded miRNAs, TE-embedded sequences give rise to spatiotemporally dynamic miRNA expression networks, the implications of which for human brain development constitute extensive avenues of future experimental research. To facilitate interactive exploration of these spatiotemporal miRNA expression dynamics, we provide the “Brain miRTExplorer” web application freely accessible for the community.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.22269328', 'tweets': 15, 'archive': 'medRxiv', 'subject-area': 'Endocrinology (including Diabetes Mellitus and Metabolic Disease)', 'covid': False, 'title': 'Estimating body fat distribution - a driver of cardiometabolic health - from silhouette images', 'authors': \"[{'name': 'Marcus D.R. Klarqvist'}, {'name': 'Saaket Agrawal'}, {'name': 'Nathaniel Diamant'}, {'name': 'Patrick T. Ellinor'}, {'name': 'Anthony Philippakis'}, {'name': 'Kenney Ng'}, {'name': 'Puneet Batra'}, {'name': 'Amit V. Khera'}]\", 'abstract': \"<p>Background: Inter-individual variation in fat distribution is increasingly recognized as clinically important but is not routinely assessed in clinical practice because quantification requires medical imaging.\\n\\nObjectives: We hypothesized that a deep learning model trained on an individual's body shape outline - or silhouette - would enable accurate estimation of specific fat depots, including visceral (VAT), abdominal subcutaneous (ASAT), and gluteofemoral (GFAT) adipose tissue volumes, and VAT/ASAT ratio. We additionally set out to study whether silhouette-estimated VAT/ASAT ratio may stratify risk of cardiometabolic diseases independent of body mass index (BMI) and waist circumference.\\n\\nMethods: Two-dimensional coronal and sagittal silhouettes were constructed from whole-body magnetic resonance images in 40,032 participants of the UK Biobank and used to train a convolutional neural network to predict VAT, ASAT, and GFAT volumes, and VAT/ASAT ratio. Logistic and Cox regressions were used to determine the independent association of silhouette-predicted VAT/ASAT ratio with type 2 diabetes and coronary artery disease.\\n\\nResults: Mean age of the study participants was 65 years and 51% were female. A deep learning model trained on silhouettes enabled accurate estimation of VAT, ASAT, and GFAT volumes (R2: 0.88, 0.93, and 0.93, respectively), outperforming a comparator model combining anthropometric and bioimpedance measures (ΔR2 = 0.05-0.13). Next, we studied VAT/ASAT ratio, a nearly BMI- and waist circumference-independent marker of unhealthy fat distribution. While the comparator model poorly predicted VAT/ASAT ratio (R2: 0.17-0.26), a silhouette-based model enabled significant improvement (R2: 0.50-0.55). Silhouette-predicted VAT/ASAT ratio was associated with increased prevalence of type 2 diabetes and coronary artery disease.\\n\\nConclusions: Body silhouette images can estimate important measures of fat distribution, laying the scientific foundation for population-based assessment.</p>\", 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.10.475696', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Using syncmers improves long-read mapping', 'authors': \"[{'name': 'Abhinav Dutta'}, {'name': 'David Pellow'}, {'name': 'Ron Shamir'}]\", 'abstract': '<p>As sequencing datasets keep growing larger, time and memory efficiency of read mapping are becoming more critical. Many clever algorithms and data structures were used to develop mapping tools for next generation sequencing, and in the last few years also for third generation long reads. A key idea in mapping algorithms is to sketch sequences with their minimizers. Recently, syncmers were introduced as an alternative sketching method that is more robust to mutations and sequencing errors.</p><p>Here we introduce parameterized syncmer schemes, and provide a theoretical analysis for multi-parameter schemes. By combining these schemes with downsampling or minimizers we can achieve any desired compression and window guarantee. We introduced syncmer schemes into the popular minimap2 and Winnowmap2 mappers. In tests on simulated and real long read data from a variety of genomes, the syncmer-based algorithms reduced unmapped reads by 20-60% at high compression while using less memory. The advantage of syncmer-based mapping was even more pronounced at lower sequence identity. At sequence identity of 65-75% and medium compression, syncmer mappers had 50-60% fewer unmapped reads, and ∼ 10% fewer of the reads that did map were incorrectly mapped. We conclude that syncmer schemes improve mapping under higher error and mutation rates. This situation happens, for example, when the high error rate of long reads is compounded by a high mutation rate in a cancer tumor, or due to differences between strains of viruses or bacteria.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.22269005', 'tweets': 15, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': False, 'title': 'Identifying the potential role of insomnia on multimorbidity: A Mendelian randomization phenome-wide association study in UK Biobank', 'authors': \"[{'name': 'Mark J Gibson'}, {'name': 'Deborah A Lawlor'}, {'name': 'Louise AC Millard'}]\", 'abstract': '<p>Objectives: To identify the breadth of potential causal effects of insomnia on health outcomes and hence its possible role in multimorbidity.\\nDesign: Mendelian randomisation (MR) Phenome-wide association study (MR-PheWAS) with two-sample Mendelian randomisation follow-up.\\nSetting: Individual data from UK Biobank and summary data from a number of genome-wide association studies.\\nParticipants: 336,975 unrelated white-British UK Biobank participants.\\nExposures: Standardised genetic risk of insomnia for the MR-PheWAS and genetically predicted insomnia for the two-sample MR follow-up, with insomnia instrumented by a genetic risk score (GRS) created from 129 single-nucleotide polymorphisms (SNPs).\\nMain outcomes measures: 11,409 outcomes from UK Biobank extracted and processed by an automated pipeline (PHESANT). Potential causal effects (i.e., those passing a Bonferroni-corrected significance threshold) were followed up with two-sample MR in MR-Base, where possible.\\nResults: 437 potential causal effects of insomnia were observed for a number of traits, including anxiety, stress, depression, mania, addiction, pain, body composition, immune, respiratory, endocrine, dental, musculoskeletal, cardiovascular and reproductive traits, as well as socioeconomic and behavioural traits. We were able to undertake two-sample MR for 71 of these 437 and found evidence of causal effects (with directionally concordant effect estimates across all analyses) for 25 of these. These included, for example, risk of anxiety disorders (OR=1.55 [95% confidence interval (CI): 1.30, 1.86] per category increase in insomnia), diseases of the oesophagus/stomach/duodenum (OR=1.32 [95% CI: 1.14, 1.53]) and spondylosis (OR=1.57 [95% CI: 1.22, 2.01]). \\nConclusion: Insomnia potentially causes a wide range of adverse health outcomes and behaviours. This has implications for developing interventions to prevent and treat a number of diseases in order to reduce multimorbidity and associated polypharmacy.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476312', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Probabilistic modeling of ambient noise in single-cell omics data', 'authors': \"[{'name': 'Caibin Sheng'}, {'name': 'Rui Lopes'}, {'name': 'Gang Li'}, {'name': 'Sven Schuierer'}, {'name': 'Annick Waldt'}, {'name': 'Rachel Cuttat'}, {'name': 'Slavica Dimitrieva'}, {'name': 'Audrey Kauffmann'}, {'name': 'Eric Durand'}, {'name': 'Giorgio G. Galli'}, {'name': 'Guglielmo Roma'}, {'name': 'Antoine de Weck'}]\", 'abstract': '<p>Droplet-based single-cell omics, including single-cell RNA sequencing (scRNAseq), single cell CRISPR perturbations (e.g., CROP-seq) and single-cell protein and transcriptomic profiling (e.g., CITE-seq) hold great promise for comprehensive cell profiling and genetic screening at the single cell resolution, yet these technologies suffer from substantial noise, among which ambient signals present in the cell suspension may be the predominant source. Current efforts to address this issue are highly specific to a certain technology, while a universal model to describe the noise across these technologies may reveal this common source thereby improving the denoising accuracy. To this end, we explicitly examined these unexpected signals and observed a predictable pattern in multiple datasets across different technologies. Based on the finding, we developed single cell Ambient Remover (scAR) which uses probabilistic deep learning to deconvolute the observed signals into native and ambient composition. scAR provides an efficient and universal solution to count denoising for multiple types of single-cell omics data, including single cell CRISPR screens, CITE-seq and scRNAseq. It will facilitate the application of single-cell omics technologies.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476255', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Location bias contributes to functionally selective responses of biased CXCR3 agonists', 'authors': \"[{'name': 'Dylan Scott Eiger'}, {'name': 'Noelia Boldizsar'}, {'name': 'Christopher Cole Honeycutt'}, {'name': 'Julia Gardner'}, {'name': 'Stephen Kirchner'}, {'name': 'Chloe Hicks'}, {'name': 'Issac Choi'}, {'name': 'Uyen Pham'}, {'name': 'Kevin Zheng'}, {'name': 'Anmol Warman'}, {'name': 'Jeffrey Smith'}, {'name': 'Jennifer Zhang'}, {'name': 'Sudarshan Rajagopal'}]\", 'abstract': '<p>Some G protein-coupled receptor (GPCR) ligands act as biased agonists which preferentially activate specific signaling transducers over others. Although GPCRs are primarily found at the plasma membrane, GPCRs can traffic to and signal from many subcellular compartments. Here, we determine that differential subcellular signaling contributes to the biased signaling generated by three endogenous ligands of the chemokine GPCR CXCR3. The signaling profile of CXCR3 changed as it trafficked from the plasma membrane to endosomes in a ligand-specific manner. Endosomal signaling was critical for biased activation of G proteins, β-arrestins, and ERK1/2. In CD8+ T cells, the chemokines promoted unique transcriptional responses predicted to regulate inflammatory pathways. In a mouse model of contact hypersensitivity, β-arrestin-biased CXCR3-mediated inflammation was dependent on receptor internalization. Our work demonstrates that differential subcellular signaling is critical to the overall biased response observed at CXCR3, which has important implications for drugs targeting chemokine receptors and other GPCRs.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476162', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Involvement of prelimbic cortex neurons and related circuits in the acquisition of cooperative learning by pairs of rats', 'authors': \"[{'name': 'A.R. Conde-Moro'}, {'name': 'F. Rocha-Almeida'}, {'name': 'E. Gebara'}, {'name': 'J.M. Delgado-García'}, {'name': 'C. Sandi'}, {'name': 'A. Gruart'}]\", 'abstract': '<p>Social behaviors such as cooperation are crucial for mammals. A deeper knowledge of the neuronal mechanisms underlying cooperation can be beneficial for people suffering from pathologies with impaired social behavior. Our aim was to study the brain activity when two animals synchronize their behavior to obtain a mutual reinforcement. In a previous work, we showed that the activity of the prelimbic cortex (PrL) was enhanced during cooperation in rats, especially in the ones leading most cooperative trials (<italic>leader</italic> rats). In this study, we investigated the specific cell type/s in the PrL contributing to cooperative behaviors. To this end, we collected rats’ brains at key moments of the learning process to analyze the levels of c-FOS expression in the main cellular groups of the PrL (glutamatergic cells containing D1 and D2 receptors and interneurons). <italic>Leader</italic> rats showed increased c-FOS activity in cells expressing D1 receptors during cooperation. In addition, we analyzed the levels of anxiety, dominance, and locomotor behavior, finding that <italic>leader</italic> rats are in general less anxious and less dominant than <italic>followers</italic>. We also recorded local field potentials (LFPs) from the PrL, the nucleus accumbens septi (NAc), and the basolateral amygdala (BLA). Spectral analysis showed that delta activity in PrL and NAc increased when rats cooperated, while BLA activity in delta and theta bands decreased considerably during cooperation. The PrL and NAc also increased their connectivity in the high theta band during cooperation. Thus, the present work identifies the specific PrL cell types engaged in this behavior, as well as its connectivity with subcortical brain regions (BLA, NAc) during cooperation.</p><sec><title>Significance Statement</title><p>Brain mechanisms underlying cooperative behaviors remain unknown. The present study identified specific neuronal types from the PrL cortex engaged in the acquisition of a cooperative task, as well as their connectivity with subcortical projection sites, such as the NAc and the BLA during cooperation. Rats leading the cooperation trials (designated <italic>leaders</italic>) presented an increased activation of D1-containing neurons in the PrL during cooperation. The PrL and NAc electrical activity increased when rats were cooperating, while the BLA activity increased before cooperation. The PrL and NAc showed increased functional connectivity at the moment of cooperation on the platform, whereas during the individual phase the highest connectivity was found before the animals climbed individually onto the platform.</p></sec>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.12.476125', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'MS1Connect: a mass spectrometry run similarity measure', 'authors': \"[{'name': 'Andy Lin'}, {'name': 'Brooke L. Deatherage Kaiser'}, {'name': 'Janine R. Hutchison'}, {'name': 'Jeffrey A. Bilmes'}, {'name': 'William Stafford Noble'}]\", 'abstract': '<p>Interpretation of newly acquired mass spectrometry data can be improved by identifying, from an online repos- itory, previous mass spectrometry runs that resemble the new data. However, this retrieval task requires comput- ing the similarity between an arbitrary pair of mass spectrometry runs. This is particularly challenging for runs acquired using different experimental protocols. We propose a method, MS1Connect, that calculates the simi- larity between a pair of runs by examining only the intact peptide (MS1) scans, and we show evidence that the MS1Connect score is accurate. Specifically, we show that MS1Connect outperforms several baseline methods on the task of predicting the species from which a given proteomics sample originated. In addition, we show that MS1Connect scores are highly correlated with similarities computed from fragment (MS2) scans, even though this data is not used by MS1Connect.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.10.475733', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'cdon and boc affect trunk neural crest cell migration through a non-cell autonomous reduction of hedgehog signaling in zebrafish slow-twitch muscle', 'authors': \"[{'name': 'Ezra Lencer'}, {'name': 'Rytis Prekeris'}, {'name': 'Kristin Artinger'}]\", 'abstract': '<p>The immunoglobin superfamily members <italic>cdon</italic> and <italic>boc</italic> are transmembrane proteins implicated in regulating hedgehog signaling during vertebrate development. Recent work showing roles for these genes in axon guidance and neural crest cell migration further suggest that <italic>cdon</italic>/<italic>boc</italic> may play additional functions in regulating directed cell movements during development. Here we use novel and existing mutants to investigate a role for <italic>cdon</italic> and <italic>boc</italic> in zebrafish neural crest cell migration. We find that single <italic>cdon</italic> or <italic>boc</italic> mutant embryos exhibit normal neural crest phenotypes, but that neural crest migration is strikingly disrupted in double <italic>cdon</italic>/<italic>boc</italic> mutant embryos. We further show that this neural crest migration phenotype is associated with defects to the differentiation of slow-twitch muscle cells, and that this slow-twitch muscle phenotype is a consequence of reduced hedgehog signaling in mutant fish. While neural crest migratory ability is not affected in double mutant embryos, neural crest directionality is severely affected. These data suggest that neural crest migration defects are likely to be secondary to defects in slow-twitch muscle differentiation. Combined, our data add to a growing literature showing that <italic>cdon</italic> and <italic>boc</italic> act synergistically to promote hedgehog signaling during vertebrate development, and provide a foundation for using zebrafish to further study the function of these hedgehog receptor paralogs.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476242', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Local-scale virome depiction supports significant differences between Aedes aegypti and Aedes albopictus', 'authors': \"[{'name': 'Arley Calle-Tobón'}, {'name': 'Juliana Pérez-Pérez'}, {'name': 'Nicolás Forero-Pineda'}, {'name': 'Omar Triana Chávez'}, {'name': 'Winston Rojas-Montoya'}, {'name': 'Guillermo Rúa-Uribe'}, {'name': 'Andrés Gómez-Palacio'}]\", 'abstract': '<p><italic>Aedes</italic> spp. comprise the primary group of mosquitoes that transmit arboviruses such as dengue, Zika, and chikungunya viruses to humans, and thus these insects pose a significant burden on public health worldwide. Advancements in next-generation sequencing and metagenomics have expanded our knowledge on the richness of RNA viruses harbored by arthropods such as <italic>Ae. aegypti</italic> and <italic>Ae. albopictus</italic>; increasing evidence suggests that vectorial competence can be modified by the microbiome (comprising both bacteriome and virome) of mosquitoes present in endemic zones. Using an RNA-seq-based metataxonomic approach, this study determined the virome structure of field-caught <italic>Ae. aegypti</italic> and <italic>Ae. albopictus</italic> mosquitoes in Medellín, Colombia, a municipality with a high incidence of mosquito-transmitted arboviruses. The two species are sympatric, but their core viromes differed considerably in richness, diversity, and abundance; the viromes were dominated by a few viruses. BLAST searches of assembled contigs suggested that at least 17 virus species (16 of which are insect-specific viruses [ISVs]) infect the <italic>Ae. aegypti</italic> population. Dengue virus 3 was detected in one sample. In <italic>Ae. albopictus</italic>, up to 11 ISVs and one plant virus were detected. Therefore, the virome composition was species-specific. The bacterial endosymbiont <italic>Wolbachia</italic> was identified in all <italic>Ae. albopictus</italic> samples and in some <italic>Ae. aegypti</italic> samples collected after 2017. The presence of <italic>Wolbachi</italic>a sp. in <italic>Ae. aegypti</italic> was not related to significant changes in the richness, diversity, or abundance of this mosquito’s virome, although it was related to an increase in the abundance of Aedes aegypti To virus 2 (unclassified). The mitochondrial diversity of these mosquitoes suggested that the <italic>Ae. aegypti</italic> population underwent a change that started in the second half of 2017, which coincides with the release of <italic>Wolbachia</italic>-infected mosquitoes in Medellín, indicating that the population of <italic>w</italic>Mel-infected mosquitoes has expanded. However, additional studies are required on the dispersal speed and intergenerational stability of <italic>w</italic>Mel in Medellín and nearby areas as well as on the introgression of genetic variants in the native mosquito population.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.14.476307', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Metabolic basis for the evolution of a common pathogenic Pseudomonas aeruginosa variant', 'authors': \"[{'name': 'Dallas L. Mould'}, {'name': 'Mirjana Stevanovic'}, {'name': 'Alix Ashare'}, {'name': 'Daniel Schultz'}, {'name': 'Deborah A Hogan'}]\", 'abstract': '<p>Microbes frequently evolve in reproducible ways. Here, we show that differences in specific metabolic regulation explain the frequent presence of <italic>lasR</italic> loss-of-function mutations in the bacterial pathogen <italic>Pseudomonas aeruginosa.</italic> While LasR contributes to virulence, <italic>lasR</italic> mutants have been associated with more severe disease. A model based on the intrinsic growth kinetics for a wild type strain and its LasR<sup>-</sup> derivative, in combination with an experimental evolution based genetic screen and further genetics analyses, indicated that differences in metabolism were sufficient to explain the rise of these common mutant types. The evolution of LasR<sup>-</sup> lineages in laboratory and clinical isolates depended on activity of the two-component system CbrAB, which modulates substrate prioritization through the catabolite repression control pathway. LasR<sup>-</sup> lineages frequently arise in cystic fibrosis lung infections and their detection correlates with disease severity. Our analysis of bronchoalveolar lavage fluid metabolomes identified compounds that negatively correlate with lung function, and we show that these compounds support enhanced growth of LasR<sup>-</sup> cells in a CbrB-controlled manner. We propose that <italic>in vivo</italic> metabolomes are a major driver of pathogen evolution, which may influence the progression of disease and its treatment.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.12.476089', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Systematic analysis of human antibody response to ebolavirus glycoprotein reveals high prevalence of neutralizing public clonotypes', 'authors': \"[{'name': 'Elaine C. Chen'}, {'name': 'Pavlo Gilchuk'}, {'name': 'Seth J. Zost'}, {'name': 'Philipp A. Ilinykh'}, {'name': 'Elad Binshtein'}, {'name': 'Kai Huang'}, {'name': 'Luke Myers'}, {'name': 'Stefano Bonissone'}, {'name': 'Samuel Day'}, {'name': 'Chandrahaas R. Kona'}, {'name': 'Andrew Trivette'}, {'name': 'Joseph X. Reidy'}, {'name': 'Rachel E. Sutton'}, {'name': 'Christopher Gainza'}, {'name': 'Summer Monroig'}, {'name': 'Edgar Davidson'}, {'name': 'Erica Ollmann Saphire'}, {'name': 'Benjamin J. Doranz'}, {'name': 'Natalie Castellana'}, {'name': 'Alexander Bukreyev'}, {'name': 'Robert H. Carnahan'}, {'name': 'James E. Crowe'}]\", 'abstract': '<p>Understanding the human antibody response to emerging viral pathogens is key to epidemic preparedness. As the size of the B cell response to a pathogenic virus protective antigen is undefined, we performed deep paired heavy and light chain sequencing in EBOV-GP specific memory B cells, allowing analysis of the ebolavirus-specific antibody repertoire both genetically and functionally. This approach facilitated investigation of the molecular and genetic basis for evolution of cross-reactive antibodies by elucidating germline-encoded properties of antibodies to EBOV and identification of the overlap between antibodies in the memory B-cell and serum repertoire. We identified 73 public clonotypes to EBOV, 20% of which encoded antibodies with neutralization activity and capacity to protect <italic>in vivo</italic>. This comprehensive analysis of the public and private antibody repertoire provides insight into the molecular basis of the humoral immune response to EBOV-GP, which informs vaccine design of new vaccines and improved therapeutics.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.11.475913', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Background selection theory overestimates effective population size for high mutation rates', 'authors': \"[{'name': 'Joseph Matheson'}, {'name': 'Joanna Masel'}]\", 'abstract': '<p>Simple models from the neutral theory of molecular evolution are claimed to be flexible enough to incorporate the complex effects of background selection against linked deleterious mutations. Complexities are collapsed into an “effective” population size that specifies neutral genetic diversity. To achieve this, current background selection theory assumes linkage equilibrium among deleterious variants. Data do not support this assumption, nor do theoretical considerations when the genome-wide deleterious mutation is realistically high. We simulate genomes evolving under background selection, allowing the emergence of linkage disequilibria. With realistically high deleterious mutation rates, neutral diversity is much lower than predicted from previous analytical theory.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.476021', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'A bacterial secretosome for regulated envelope biogenesis and quality control', 'authors': \"[{'name': 'Daniel W. Watkins'}, {'name': 'Ian Collinson'}]\", 'abstract': '<p>As the first line of defence against antibiotics, the Gram-negative bacterial envelope and its biogenesis are of considerable interest to the microbiological and biomedical communities. All bacterial proteins are synthesised in the cytosol, so inner- and outer-membrane proteins, and periplasmic residents have to be transported to their final destinations <italic>via</italic> specialised protein machinery. The Sec translocon, a ubiquitous integral inner-membrane (IM) complex, is key to this process as the major gateway for protein transit from the cytosol to the cell envelope; this can be achieved during their translation, or afterwards. Proteins need to be directed to the inner-membrane (usually co-translational), otherwise SecA utilises ATP and the proton-motive-force (PMF) to drive proteins across the membrane post-translationally. These proteins are then picked up by chaperones for folding in the periplasm or delivered to the β-barrel assembly machinery (BAM) for incorporation into the outer-membrane. The core heterotrimeric SecYEG-complex forms the hub for an extensive network of interactions that regulate protein delivery and quality control. Here, we conduct a biochemical exploration of this ‘secretosome’ – a very large, versatile and inter-changeable assembly with the Sec-translocon at its core; featuring interactions that facilitate secretion (SecDF), inner- and outer-membrane protein insertion (respectively, YidC and BAM), protein folding and quality control (<italic>e</italic>.<italic>g</italic>. PpiD, YfgM and FtsH). We propose the dynamic interplay amongst these and other factors act to ensure efficient whole envelope biogenesis, regulated to accommodate the requirements of cell elongation and division. This organisation would be essential for cell wall biogenesis and remodelling and thus its perturbation would be a good strategy for the development of anti-microbials.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.15.22269318', 'tweets': 13, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Implementation of a Rapid RT-LAMP Saliva-based SARS-CoV-2 Testing Program in the Workplace', 'authors': \"[{'name': 'Bradley W.M. Cook'}, {'name': 'Kaitlyn Kobasa'}, {'name': 'Marielou Tamayo'}, {'name': 'Natasha Theriault'}, {'name': 'Diane J.R. Gordon Pappas'}, {'name': 'Steven S. Theriault'}]\", 'abstract': '<p>Rising SARS-CoV-2 cases, testing delays and the risk of pre-symptomatic and asymptomatic transmission provided the impetus for an in-house rapid testing pro-gram. Employees and their household contacts were encouraged to self-collect saliva samples which were pooled for routine testing using an established colorimetric reverse transcription loop-mediated isothermal amplification (RT-LAMP) assay. In brief, individual or a maximum of four saliva samples were pooled, heat-inactivated to render microorganisms, especially SARS-CoV-2, non-infectious prior to being added to RT-LAMP assay tubes containing either human sample control gene, RNase P or a region of the SARS-CoV-2 gene, ORF1ab. During the second wave of SARS-CoV-2 infections in November 2020, two samples from an employee and a member of their household tested positive via RT-LAMP within two days of each other. A delayed clinical qRT-PCR test confirmation of both individuals 5 days later underscores the power of routine rapid testing with within-the-hour turnaround times. Workplace rapid testing programs using RT-LAMP are flexible in their design, have a reduced cost compared to qRT-PCR, may involve non-invasive self-saliva collection for increased safety for the testing personnel, and can be performed with minimal training.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.14.476291', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'CtIP-dependent nascent RNA expression flanking DNA breaks guides the choice of DNA repair pathway', 'authors': \"[{'name': 'Daniel Gomez-Cabello'}, {'name': 'Georgios Pappas'}, {'name': 'Diana Aguilar-Morante'}, {'name': 'Christoffel Dinant'}, {'name': 'Jiri Bartek'}]\", 'abstract': '<p>The RNA world is changing our views about sensing and resolution of DNA damage. Here, we developed single-molecule DNA/RNA analysis approaches to visualize how nascent RNA facilitates the repair of DNA double-strand breaks (DSBs). RNA polymerase II (RNAPII) is crucial for DSB resolution in human cells. DSB-flanking, RNAPII-generated nascent RNA forms RNA:DNA hybrids, guiding the upstream DNA repair steps towards favouring the error-free Homologous Recombination (HR) pathway over Non-Homologous End Joining. Specific RNAPII inhibitor, THZ1, impairs recruitment of essential HR proteins to DSBs, implicating nascent RNA in DNA end resection, initiation and execution of HR repair. We further propose that resection factor CtIP interacts with and re-activates RNAPII when paused by the RNA:DNA hybrids, collectively promoting faithful repair of chromosome breaks to maintain genomic integrity.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475888', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'H2A.Z deposition by SWR1C involves multiple ATP-dependent steps', 'authors': \"[{'name': 'Jiayi Fan'}, {'name': 'Andrew T. Moreno'}, {'name': 'Alexander S. Baier'}, {'name': 'Joseph J. Loparo'}, {'name': 'Craig L. Peterson'}]\", 'abstract': '<p>The histone variant H2A.Z is a conserved feature of nucleosomes flanking protein-coding genes. Deposition of H2A.Z requires ATP-dependent replacement of nucleosomal H2A by a chromatin remodeler related to the multi-subunit enzyme, yeast SWR1C. How these enzymes use ATP to promote this nucleosome editing reaction remains unclear. Here we use single-molecule and ensemble methodologies to identify three ATP-dependent phases in the H2A.Z deposition reaction. Real-time analysis of single nucleosome remodeling events reveals an initial, priming step that occurs after ATP addition that likely involves transient DNA unwrapping from the nucleosome. Priming is followed by rapid loss of histone H2A, which is subsequently released from the H2A.Z nucleosomal product. Surprisingly, the rates of both priming and the release of the H2A/H2B dimer are sensitive to ATP concentration. This complex reaction pathway provides multiple opportunities to regulate the timely and accurate deposition of H2A.Z at key genomic locations.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.14.476347', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evolution of assortative mating following selective introgression of pigmentation genes between two Drosophila species', 'authors': \"[{'name': 'Jean R David'}, {'name': 'Erina A Ferreira'}, {'name': 'Laure Jabaud'}, {'name': 'David Ogereau'}, {'name': 'Héloïse Bastide'}, {'name': 'Amir Yassin'}]\", 'abstract': '<p>Adaptive introgression is ubiquitous in animals but experimental support for its role in driving speciation remains scarce. In the absence of conscious selection, admixed laboratory strains of Drosophila asymmetrically and progressively lose alleles from one parental species and reproductive isolation against the predominant parent ceases after 10 generations. Here, we selectively introgressed during one year light pigmentation genes of D. santomea into the genome of its dark sibling D. yakuba, and vice versa. We found that the pace of phenotypic change differed between the species and the sexes, and identified through genome sequencing common as well as distinct introgressed loci in each species. Mating assays showed that assortative mating between introgressed flies and both parental species persisted even after four years (~ 60 generations) from the end of the selection. Those results indicate that selective introgression of as low as 0.5% of the genome can beget morphologically-distinct and reproductively-isolated strains, two prerequisites for the delimitation of new species. Our findings hence represent a significant step towards understanding the genome-wide dynamics of speciation-through-introgression.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.13.476159', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'In vitro Kinase-to-Phosphosite database (iKiP-DB) predicts kinase activity in phosphoproteomic datasets', 'authors': \"[{'name': 'Tommaso Mari'}, {'name': 'Kristin Moesbauer'}, {'name': 'Emanuel Wyler'}, {'name': 'Markus Landthaler'}, {'name': 'Christian Drosten'}, {'name': 'Matthias Selbach'}]\", 'abstract': '<p>Phosphoproteomics routinely quantifies changes in the levels of thousands of phosphorylation sites, but functional analysis of such data remains a major challenge. While databases like PhosphoSitePlus contain information about many phosphorylation sites, the vast majority of known sites are not assigned to any protein kinase. Assigning changes in the phosphoproteome to the activity of individual kinases therefore remains a key challenge.. A recent large-scale study systematically identified in vitro substrates for most human protein kinases. Here, we reprocessed and filtered these data to generate an in vitro Kinase-to-Phosphosite database (iKiP-DB). We show that iKiP-DB can accurately predict changes in kinase activity in published phosphoproteomic datasets for both well-studied and poorly characterized kinases. We apply iKiP-DB to a newly generated phosphoproteomic analysis of SARS-CoV-2 infected human lung epithelial cells and provide evidence for coronavirus-induced changes in host cell kinase activity. In summary, we show that iKiP-DB is widely applicable to facilitate the functional analysis of phosphoproteomic datasets.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476205', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'BitterMatch: Recommendation systems for matching molecules with bitter taste receptors', 'authors': \"[{'name': 'Eitan Margulis'}, {'name': 'Yuli Slavutsky'}, {'name': 'Tatjana Lang'}, {'name': 'Maik Behrens'}, {'name': 'Yuval Benjamini'}, {'name': 'Masha Y Niv'}]\", 'abstract': '<p>Bitterness is an aversive cue elicited by thousands of chemically diverse compounds.  Bitter taste may prevent consumption of foods and jeopardize drug compliance. The G protein-coupled receptors for bitter taste, TAS2Rs, have species-dependent number of subtypes and varying expression levels in extraoral tissues. Molecular recognition by TAS2R subtypes is physiologically important, and presents a challenging case study for ligand-receptor matchmaking. Inspired by hybrid recommendation systems, we developed a new set of similarity features, and created the BitterMatch algorithm that predicts associations of ligands to receptors with ~80% precision at ~50% recall. Associations for several compounds were tested in-vitro, resulting in 80% precision and 42% recall. The encouraging performance was achieved by including receptor properties and integrating experimentally determined ligand-receptor associations with chemical ligand-to-ligand similarities. BitterMatch can predict off-targets for bitter drugs, identify novel ligands and guide flavor design. Inclusion of neighbor-informed similarities improves as experimental data mounts, and provides a generalizable framework for molecule-biotarget matching.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.11.475836', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'ProtFinder: finding subcellular locations of proteins using protein interaction networks', 'authors': \"[{'name': 'Aayush Grover'}, {'name': 'Laurent Gatto'}]\", 'abstract': '<p>Protein subcellular localization prediction plays a crucial role in improving our understanding of different diseases and consequently assists in building drug targeting and drug development pipelines. Proteins are known to co-exist at multiple subcellular locations which make the task of prediction extremely challenging. A protein interaction network is a graph that captures interactions between different proteins. It is safe to assume that if two proteins are interacting, they must share some subcellular locations. With this regard, we propose ProtFinder – the first deep learning-based model that exclusively relies on protein interaction networks to predict the multiple subcellular locations of proteins. We also integrate biological priors like the cellular component of Gene Ontology to make ProtFinder a more biology-aware intelligent system. ProtFinder is trained and tested using the STRING and BioPlex databases whereas the annotations of proteins are obtained from the Human Protein Atlas. Our model obtained an AUC-ROC score of 90.00% and an MCC score of 83.42% on a held-out set of proteins. We also apply ProtFinder to annotate proteins that currently do not have confident location annotations. We observe that ProtFinder is able to confirm some of these unreliable location annotations, while in some cases complementing the existing databases with novel location annotations. The source code for ProtFinder is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/UCLouvain-CBIO/ProtFinder\">https://github.com/UCLouvain-CBIO/ProtFinder</ext-link>.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476238', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Bat Red Blood Cells express Nucleic Acid Sensing Receptors and bind RNA and DNA', 'authors': \"[{'name': 'LK Metthew Lam'}, {'name': 'Jane Dobkin'}, {'name': 'Kaitlyn A. Eckart'}, {'name': 'Ian Gereg'}, {'name': 'Andrew DiSalvo'}, {'name': 'Amber Nolder'}, {'name': 'Eman Anis'}, {'name': 'Julie C. Ellis'}, {'name': 'Greg Turner'}, {'name': 'Nilam S. Mangalmurti'}]\", 'abstract': '<p>Red blood cells (RBCs) demonstrate immunomodulatory capabilities through the expression of nucleic acid sensors. However, little is known about bat RBCs, and no studies have examined the immune function of bat erythrocytes. Here we show that bat RBCs express the nucleic acid-sensing Toll-like receptors TLR7 and TLR9 and bind the nucleic acid ligands, single-stranded RNA, and CpG DNA. Collectively, these data suggest that, like human RBCs, bat erythrocytes possess immune function and may be reservoirs for nucleic acids. These findings provide unique insight into bat immunity and may uncover potential mechanisms by which virulent pathogens in humans are concealed in bats.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.12.476097', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Myeloid-biased HSC require Semaphorin 4A from the bone marrow niche for self-renewal under stress and life-long persistence', 'authors': \"[{'name': 'Dorsa Toghani'}, {'name': 'Sharon Zeng'}, {'name': 'Elmir Mahammadov'}, {'name': 'Edie I. Crosse'}, {'name': 'Negar Seyedhassantehrani'}, {'name': 'Christian Burns'}, {'name': 'David Gravano'}, {'name': 'Stefan Radtke'}, {'name': 'Hans-Peter Kiem'}, {'name': 'Sonia Rodriguez'}, {'name': 'Nadia Carlesso'}, {'name': 'Amogh Pradeep'}, {'name': 'Nicola K. Wilson'}, {'name': 'Sarah J. Kinston'}, {'name': 'Berthold Göttgens'}, {'name': 'Claus Nerlov'}, {'name': 'Eric Pietras'}, {'name': 'Marion Mesnieres'}, {'name': 'Christa Maes'}, {'name': 'Atsushi Kumanogoh'}, {'name': 'Thomas Worzfeld'}, {'name': 'Peter Kharchenko'}, {'name': 'David T. Scadden'}, {'name': 'Antonio Scialdone'}, {'name': 'Joel A Spencer'}, {'name': 'Lev Silberstein'}]\", 'abstract': '<p>Tissue stem cells are hierarchically organized. Those that are most primitive serve as key drivers of regenerative response but the signals that selectively preserve their functional integrity are largely unknown. Here, we identify a secreted factor, Semaphorin 4A (Sema4A), as a specific regulator of myeloid-biased hematopoietic stem cells (myHSC), which are positioned at the top of the HSC hierarchy. Lack of Sema4A leads to exaggerated myHSC (but not downstream “balanced” HSC) proliferation after acute inflammatory stress, indicating that Sema4A enforces myHSC quiescence. Strikingly, aged Sema4A knock-out myHSC expand but almost completely lose reconstitution capacity. The effect of Sema4A is non cell-autonomous, since upon transplantation into Sema4A-deficient environment, wild-type myHSC excessively proliferate but fail to engraft long-term. Sema4A constrains inflammatory signaling in myHSC and acts via a surface receptor Plexin-D1. Our data support a model whereby the most primitive tissue stem cells critically rely on a dedicated signal from the niche for self-renewal and life-long persistence.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475153', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Evaluation of methods incorporating biological function and GWAS summary statistics to accelerate discovery', 'authors': \"[{'name': 'Amy Moore'}, {'name': 'Jesse Marks'}, {'name': 'Bryan C. Quach'}, {'name': 'Yuelong Guo'}, {'name': 'Laura J. Bierut'}, {'name': 'Nathan C. Gaddis'}, {'name': 'Dana B. Hancock'}, {'name': 'Grier P. Page'}, {'name': 'Eric O. Johnson'}]\", 'abstract': '<p>Where sufficiently large genome-wide association study (GWAS) samples are not currently available or feasible, methods that leverage increasing knowledge of the biological function of variants may illuminate discoveries without increasing sample size. We comprehensively evaluated 18 functional weighting methods for identifying novel associations. We assessed the performance of these methods using published results from multiple GWAS waves across each of five complex traits. Although no method achieved both high sensitivity and positive predictive value (PPV) for any trait, a subset of methods utilizing pleiotropy and expression quantitative trait loci nominated variants with high PPV (&gt;75%) for multiple traits. Application of functionally weighting methods to enhance GWAS power for locus discovery is unlikely to circumvent the need for larger sample sizes in truly underpowered GWAS, but these results suggest that applying functional weighting to GWAS can accurately nominate additional novel loci from available samples for follow-up studies.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.09.475576', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Frequent co-regulation of splicing and polyadenylation by RNA-binding proteins inferred with MAPP', 'authors': \"[{'name': 'Maciej Bak'}, {'name': 'Erik van Nimwegen'}, {'name': 'Ralf Schmidt'}, {'name': 'Mihaela Zavolan'}, {'name': 'Andreas J. Gruber'}]\", 'abstract': '<p>Maturation of eukaryotic pre-mRNAs via splicing, 3’ end cleavage and polyadenylation is modulated across cell types and conditions by a variety of RNA-binding proteins (RBPs). Although over 1’500 proteins are associated with RNAs in human cells, their binding motifs, targets and functions still remain to be elucidated, especially in the complex environment of human tissues and in the context of diseases. To overcome the lack of methods for systematic and automated detection of sequence motif-guided changes in pre-mRNA processing based on RNA sequencing (RNA-seq) data we have developed MAPP (Motif Activity on Pre-mRNA Processing). We demonstrate MAPP’s functionality by applying it to RNA-seq data from 284 RBP knock-down experiments in the ENCODE project, from which MAPP not only infers position-dependent impact profiles of known regulators, but also reveals RBPs that modulate both the inclusion of cassette exons and the poly(A) site choice. Among these, the Polypyrimidine Tract Binding Protein 1 (PTBP1) has a similar activity in glioblastoma samples. This highlights the ability of MAPP to unveil global regulators of mRNA processing under physiological and pathological conditions.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.13.476166', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Growth orientations, rather than heterogeneous growth rates, dominate jaw joint morphogenesis in the larval zebrafish', 'authors': \"[{'name': 'Josepha Godivier'}, {'name': 'Elizabeth A. Lawrence'}, {'name': 'Mengdi Wang'}, {'name': 'Chrissy L. Hammond'}, {'name': 'Niamh C. Nowlan'}]\", 'abstract': '<p>In early limb embryogenesis, synovial joints acquire specific shapes which determine joint motion and function. The process by which the opposing cartilaginous joint surfaces are moulded into reciprocal and interlocking shapes, called joint morphogenesis, is one of the least understood aspect of joint formation and the cell-level dynamics underlying it are yet to be unravelled. In this research, we quantified key cellular dynamics involved in growth and morphogenesis of the zebrafish jaw joint and synthesised them in a predictive computational simulation of joint development. Cells in larval zebrafish jaw joints labelled with cartilage markers were tracked over a forty-eight hour time window using confocal imaging. Changes in distance and angle between adjacent cell centroids resulting from cell rearrangement, volume expansion and extracellular matrix (ECM) deposition were measured and used to calculate the rate and direction of local tissue deformations. We observed spatially and temporally heterogeneous growth patterns with marked anisotropy over the developmental period assessed. There was notably elevated growth at the level of the retroarticular process of the Meckel’s cartilage, a feature known to undergo pronounced shape changes during zebrafish development. Analysis of cell dynamics indicated a dominant role for cell volume expansion in growth, with minor influences from ECM volume increases and cell intercalation. Cell proliferation in the joint was minimal over the timeframe of interest. Synthesising the dynamic cell data into a finite element model of jaw joint development resulted in accurate shape predictions. Our biofidelic computational simulation demonstrated that zebrafish jaw joint growth can be reasonably approximated based on cell positional information over time, where cell positional information derives mainly from cell orientation and cell volume expansion. By modifying the input parameters of the simulation, we were able to assess the relative contributions of heterogeneous growth rates and of growth orientation. The use of uniform rather than heterogeneous growth rates only minorly impacted the shape predictions whereas isotropic growth fields resulted in altered shape predictions. The simulation results suggest that growth anisotropy is the dominant influence on joint growth and morphogenesis. This study addresses the gap of the cellular processes underlying joint morphogenesis, with implications for understanding the aetiology of developmental joint disorders such as developmental dysplasia of the hip and arthrogryposis.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.12.475240', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'GR13-type plasmids in Acinetobacter potentiate the accumulation and horizontal transfer of diverse accessory genes', 'authors': \"[{'name': 'Robert A. Moran'}, {'name': 'Haiyang Liu'}, {'name': 'Emma L. Doughty'}, {'name': 'Xiaoting Hua'}, {'name': 'Elizabeth A. Cummins'}, {'name': 'Tomas Liveikis'}, {'name': 'Alan McNally'}, {'name': 'Zhihui Zhou'}, {'name': 'Willem van Schaik'}, {'name': 'Yunsong Yu'}]\", 'abstract': '<p>Carbapenem resistance and other antibiotic resistance genes (ARGs) can be found in plasmids in <italic>Acinetobacter</italic>, but many plasmid types in this genus have not been well-characterised. Here we describe the distribution, diversity and evolutionary capacity of <italic>rep</italic> group 13 (GR13) plasmids that are found in <italic>Acinetobacter</italic> species from diverse environments. Our investigation was prompted by the discovery of two GR13 plasmids in <italic>A. baumannii</italic> isolated in an intensive care unit (ICU). The plasmids harbour distinct accessory genes: pDETAB5 contains <italic>bla</italic><sub>NDM-1</sub> and genes that confer resistance to four further antibiotic classes, while pDETAB13 carries putative alcohol tolerance determinants. Both plasmids contain multiple <italic>dif</italic> modules, which are flanked by p<italic>dif</italic> sites recognised by XerC/XerD tyrosine recombinases. The ARG-containing <italic>dif</italic> modules in pDETAB5 are almost identical to those found in pDETAB2, a GR34 plasmid from an unrelated <italic>A. baumannii</italic> isolated in the same ICU a month prior. Examination of a further 41 complete, publicly available plasmid sequences revealed that the GR13 pangenome consists of just four core but 1086 accessory genes, 123 in the shell and 1063 in the cloud, reflecting substantial capacity for diversification. The GR13 core genome includes genes for replication and partitioning, and for a putative tyrosine recombinase. Accessory segments encode proteins with diverse putative functions, including for metabolism, antibiotic/heavy metal/alcohol tolerance, restriction-modification, an anti-phage system and multiple toxin-antitoxin systems. The movement of <italic>dif</italic> modules and actions of insertion sequences play an important role in generating diversity in GR13 plasmids. Discrete GR13 plasmid lineages are internationally disseminated and found in multiple <italic>Acinetobacter</italic> species, which suggests they are important platforms for the accumulation, horizontal transmission and persistence of accessory genes in this genus.</p><sec><title>Impact statement</title><p><italic>Acinetobacter</italic> species are particularly well-adapted for persistence in hospital environments where they pose a life-threatening infection risk to the most clinically-vulnerable patients. Plasmids with the potential to transfer multiple antibiotic resistance determinants between <italic>Acinetobacter</italic> species are therefore concerning, but most are not well-characterised. This work sheds further light on the poorly-understood mobile gene pool associated with <italic>Acinetobacter</italic>. We show here that GR13 plasmids carry a small set of core genes but have access to a highly diverse set of accessory segments that might provide fitness advantages under certain conditions. The complex evolutionary dynamics of GR13 plasmids appear to be driven by the exchange of <italic>dif</italic> modules and by the actions of a diverse population of insertion sequences. The novel <italic>dif</italic> modules characterised here emphasise the broader importance of these elements to the dissemination of accessory genes in <italic>Acinetobacter</italic>. This study has improved our understanding of the diversity and distribution of <italic>dif</italic> modules, plasmids that carry them, and how both disseminate in the continuum of <italic>Acinetobacter</italic> populations that link hospitals and the wider environment.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.475329', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Comprehensive preclinical evaluation of human-derived anti-poly-GA antibodies in cellular and animal models of C9ORF72 disease', 'authors': \"[{'name': 'Melanie Jambeau'}, {'name': 'Kevin D. Meyer'}, {'name': 'Marian Hruska-Plochan'}, {'name': 'Ricardos Tabet'}, {'name': 'Chao-Zong Lee'}, {'name': 'Ananya Ray-Soni'}, {'name': 'Corey Aguilar'}, {'name': 'Kitty Savage'}, {'name': 'Nibha Mishra'}, {'name': 'Nicole Cavegn'}, {'name': 'Petra Borter'}, {'name': 'Chun-Chia Lin'}, {'name': 'Karen Jansen-West'}, {'name': 'Jay Jiang'}, {'name': 'Fernande Freyermuth'}, {'name': 'Nan Li'}, {'name': 'Pierre De Rossi'}, {'name': 'Manuela Perez-Berlanga'}, {'name': 'Xin Jiang'}, {'name': 'Lilian M. Daughrity'}, {'name': 'Joao Pereira'}, {'name': 'Sarav Narayanan'}, {'name': 'Yuanzheng Gu'}, {'name': 'Shekhar Dhokai'}, {'name': 'Isin Dalkilic-Liddle'}, {'name': 'Zuzanna Maniecka'}, {'name': 'Julien Weber'}, {'name': 'Michael Workman'}, {'name': 'Melissa McAlonis-Downes'}, {'name': 'Eugene Berezovski'}, {'name': 'Yong-Jie Zhang'}, {'name': 'James Berry'}, {'name': 'Brian J. Wainger'}, {'name': 'Mark W. Kankel'}, {'name': 'Mia Rushe'}, {'name': 'Christoph Hock'}, {'name': 'Roger M. Nitsch'}, {'name': 'Don W. Cleveland'}, {'name': 'Leonard Petrucelli'}, {'name': 'Tania Gendron'}, {'name': 'Fabio Montrasio'}, {'name': 'Jan Grimm'}, {'name': 'Magdalini Polymenidou'}, {'name': 'Clotilde Lagier-Tourenne'}]\", 'abstract': '<p>Hexanucleotide G4C2 repeat expansions in the C9ORF72 gene are the most common genetic cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). Dipeptide repeat proteins (DPRs) generated by translation of repeat-containing RNAs show toxic effects in vivo as well as in vitro and are key targets for therapeutic intervention. We generated human antibodies that bind DPRs with high affinity and specificity. Anti-GA antibodies engaged extra- and intracellular poly-GA and reduced aggregate formation in a poly-GA over-expressing human cell line. However, antibody treatment in human neuronal cultures synthesizing exogenous poly-GA resulted in the formation of large extracellular immune complexes and did not affect accumulation of intracellular poly-GA aggregates. Treatment with antibodies was also shown to directly alter the morphological and biochemical properties of poly-GA and to shift poly-GA/antibody complexes to more rapidly sedimenting ones. These alterations were not observed with poly-GP and have important implications for accurate measurement of poly-GA levels including the need to evaluate all centrifugation fractions and disrupt the interaction between treatment antibodies and poly-GA by denaturation. Targeting poly-GA and poly-GP in two mouse models expressing G4C2 repeats by systemic antibody delivery for up to 16 months was well-tolerated and led to measurable brain penetration of antibodies. Long term treatment with anti-GA antibodies produced improvement in an open field movement test in aged C9ORF72450 mice. However, chronic administration of anti-GA antibodies in AAV-(G4C2)149 mice was associated with increased levels of poly-GA detected by immunoassay and did not significantly reduce poly-GA aggregates or alleviate disease progression in this model.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.11.475911', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Evolution of a cis-acting SNP that controls Type VI Secretion in Vibrio cholerae', 'authors': \"[{'name': 'Siu Lung Ng'}, {'name': 'Sophia Kammann'}, {'name': 'Gabi Steinbach'}, {'name': 'Tobias Hoffmann'}, {'name': 'Peter J Yunker'}, {'name': 'Brian K. Hammer'}]\", 'abstract': '<p>Mutations in regulatory mechanisms that control gene expression contribute to phenotypic diversity and thus facilitate the adaptation of microbes to new niches. Regulatory architecture is often inferred from transcription factor identification and genome analysis using purely computational approaches. However, there are few examples of phenotypic divergence that arise from the rewiring of bacterial regulatory circuity by mutations in intergenic regions, because locating regulatory elements within regions of DNA that do not code for protein requires genomic and experimental data. We identify a single <italic>cis</italic>-acting single nucleotide polymorphism (SNP) dramatically alters control of the type VI secretion system (T6), a common weapon for inter-bacterial competition. Tight T6 regulatory control is necessary for adaptation of the waterborne pathogen <italic>Vibrio cholerae</italic> to <italic>in vivo</italic> conditions within the human gut, which we show can be altered by this single non-coding SNP that results in constitutive expression <italic>in vitro</italic>. Our results support a model of pathogen evolution through <italic>cis</italic>-regulatory mutation and preexisting, active transcription factors, thus conferring different fitness advantages to tightly regulated strains inside a human host and unfettered strains adapted to environmental niches.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476237', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'RefZ and Noc act synthetically to prevent aberrant divisions during Bacillus subtilis sporulation', 'authors': \"[{'name': 'Allyssa K. Miller'}, {'name': 'Jennifer K Herman'}]\", 'abstract': '<p>During sporulation, Bacillus subtilis undergoes an atypical cell division that requires overriding mechanisms which protect chromosomes from damage and ensure inheritance by daughter cells. Instead of assembling between segregated chromosomes at midcell, the FtsZ-ring (Z-ring) coalesces polarly, directing division over one chromosome. The DNA-binding protein RefZ facilitates the timely assembly of polar Z-rings and partially defines the region of chromosome initially captured in the forespore. RefZ binds to motifs (RBMs) located proximal to the origin of replication (oriC). Although refZ and the RBMs are conserved across the Bacillus genus, a refZ deletion mutant sporulates with wildtype efficiency, so the functional significance of RefZ during sporulation remains unclear. To further investigate RefZ function, we performed a candidate-based screen for synthetic sporulation defects by combining ∆refZ with deletions of genes previously implicated in FtsZ regulation and/or chromosome capture. Combining ∆refZ with deletions of ezrA, sepF, parA, or minD did not detectably affect sporulation. In contrast, a ∆refZ ∆noc mutant exhibited a sporulation defect, revealing a genetic interaction between RefZ and Noc. Using reporters of sporulation progression, we determined the ∆refZ ∆noc mutant exhibited sporulation delays after Spo0A activation but prior to late sporulation, with a subset of cells failing to divide polarly or activate the first forespore-specific sigma factor, SigF. The ∆refZ ∆noc mutant also exhibited extensive dysregulation of cell division, producing cells with extra, misplaced, or otherwise aberrant septa. Our results reveal a previously unknown epistatic relationship that suggests refZ and noc contribute synthetically to regulating cell division and supporting spore development.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476313', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Utilizing citizen science to model the distribution of Aedes aegypti ([Diptera]: [Culicidae]) in West Africa', 'authors': \"[{'name': 'Elizabeth A. Freeman'}, {'name': 'Elizabeth J. Carlton'}, {'name': 'Sara Paull'}, {'name': 'Samuel Dadzie'}, {'name': 'Andrea Buchwald'}]\", 'abstract': '<p>In a rapidly urbanizing region such as West Africa, <italic>Aedes</italic> mosquitoes pose an emerging threat of infectious disease that is compounded by limited vector surveillance. Citizen science has been proposed as a way to fill surveillance gaps by training local residents to collect and share information on disease vectors. Increasing citizen science efforts can begin to bridge the gaps in our current knowledge of <italic>Aedes</italic> distribution while engaging locals with mosquito control and public health efforts. Understanding the distribution of disease vectors in West Africa can inform researchers and public health officials on where to conduct disease surveillance and focus public health interventions. We aimed to compare citizen science data to published literature observations of <italic>Aedes</italic> mosquitoes and to quantify how incorporating citizen science changes our understanding of <italic>Aedes</italic> mosquito distribution in West Africa. We utilized citizen science data collected through NASA’s GLOBE Observer mobile phone application and data from a previously published literature review on <italic>Aedes</italic> mosquito distribution to examine the contribution of citizen science to understanding the distribution of <italic>Ae. aegypti</italic> in West Africa using Maximum Entropy modeling. Combining citizen science and literature-derived observations improved the fit of the model compared to models created by each data source alone, but did not alleviate location bias within the models, likely due to lack of widespread observations. Understanding <italic>Ae. aegypti</italic> distribution will require greater investment in <italic>Aedes</italic> mosquito surveillance in the region, and citizen science should be utilized as a tool in this mission to increase the reach of surveillance.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.22269176', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'ExPRSweb - An Online Repository with Polygenic Risk Scores for Common Health-related Exposures', 'authors': \"[{'name': 'Ying Ma'}, {'name': 'Snehal Patil'}, {'name': 'Xiang Zhou'}, {'name': 'Bhramar Mukherjee'}, {'name': 'Lars G. Fritsche'}]\", 'abstract': '<p>Complex traits are influenced by genetic risk factors, lifestyle, and environmental variables, so called exposures. Some exposures, e.g., smoking or lipid levels, have common genetic modifiers identified in genome-wide association studies. Since measurements are often unfeasible, Exposure Polygenic Risk Scores (ExPRSs) offer an alternative to study the influence of exposures on various phenotypes. Here, we collected publicly available summary statistics for 28 exposures and applied four common PRS methods to generate ExPRSs in two large biobanks, the Michigan Genomics Initiative and the UK Biobank. We established ExPRS for 27 exposures and demonstrated their applicability in phenome-wide association studies and as predictors for common chronic conditions. Especially, the addition of multiple ExPRSs showed, for several chronic conditions, an improvement compared prediction models that only included traditional, disease-focused PRSs. To facilitate follow-up studies, we share all ExPRS constructs and generated results via an online repository called ExPRSweb.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475864', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Inferring tumor-specific cancer dependencies through integrating ex-vivo drug response assays and drug-protein profiling', 'authors': \"[{'name': 'Alina Batzilla'}, {'name': 'Junyan Lu'}, {'name': 'Jarno Kivioja'}, {'name': 'Kerstin Putzker'}, {'name': 'Joe Lewis'}, {'name': 'Thorsten Zenz'}, {'name': 'Wolfgang Huber'}]\", 'abstract': '<p>The development of cancer therapies may be improved by the discovery of tumor-specific molecular dependencies. The requisite tools include genetic and chemical perturbations, each with its strengths and limitations. Drug perturbations can be readily applied to primary cancer samples at a large scale, but mechanistic understanding of hits and further pharmaceutical development is often complicated by the fact that a small compound has a range of affinities to multiple proteins.</p><p>To computationally infer the molecular dependencies of individual cancers from their ex-vivo drug sensitivity profiles, we developed a mathematical model that deconvolutes these data using measurements of protein-drug affinity profiles.</p><p>Our method, DepInfeR, correctly identified known dependencies, including EGFR dependence in Her2+ breast cancer cell line, FLT3 dependence in AML tumors with FLT3-ITD mutations and the differential dependencies on the B-cell receptor pathway in two major subtypes of chronic lymphocytic leukemia (CLL). Furthermore, our method uncovered new subgroup-specific dependencies, including a previously unreported dependence of high-risk CLL on Checkpoint kinase 1 (CHEK1). The method also produced a more accurate map of the molecular dependencies in a heterogeneous set of 117 CLL samples.</p><p>The ability to deconvolute polypharmacological phenotypes into underlying causal molecular dependencies should increase the utility of high-throughput drug response assays for functional precision oncology.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.08.475501', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'OptiDiff: structural variation detection from single optical mapping reads', 'authors': \"[{'name': 'Mehmet Akdel'}, {'name': 'Dick de Ridder'}]\", 'abstract': '<p>Detecting structural variation (SV) in eukaryotic genomes is of broad interest due to its often dramatic phenotypic effects, but remains a major, costly challenge based on DNA sequencing data. A cost-effective alternative in detecting large-scale SV has become available with advances in optical mapping technology. However, the algorithmic approaches to identifying SVs from optical mapping data are limited. Here, we propose a novel, open-source SV detection tool, OptiDiff, which employs a single molecule based approach to detect and classify homozygous and heterozygous SVs at coverages as low as 20x, showing better performance than the state of the art.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.13.476178', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Building programmable multicompartment artificial cells incorporating remotely activated protein channels using microfluidics and acoustic levitation', 'authors': \"[{'name': 'Jin Li'}, {'name': 'William David Jamieson'}, {'name': 'Pantelitsa Dimitriou'}, {'name': 'Wen Xu'}, {'name': 'Paul Rohde'}, {'name': 'Boris Martinac'}, {'name': 'Matthew AB Baker'}, {'name': 'Bruce Drinkwater'}, {'name': 'Oliver Castell'}, {'name': 'David Barrow'}]\", 'abstract': '<p>Intracellular compartments are functional units that support the metabolic processes within living cells, through spatiotemporal regulation of chemical reactions and biological processes. Consequently, as a step forward in the bottom-up creation of artificial cells, building analogous intracellular architectures is essential for the expansion of cell-mimicking functionality. Herein, we report the development of a droplet laboratory platform to engineer customised complex emulsion droplets as a multicompartment artificial cell chassis, using multiphase microfluidics and acoustic levitation. Such levitated constructs provide free-standing, dynamic, definable droplet networks for the encapsulation and organisation of chemical species. Equally, they can be remotely operated with pneumatic, heating, and magnetic elements for post-processing, including the incorporation of membrane proteins; alpha-hemolysin; and large-conductance mechanosensitive channel (MscL) and their activation. The assembly of droplet networks is three-dimensionally patterned with fluidic inputs configurations determining droplet contents and connectivity. Whilst acoustic manipulation can be harnessed to reconfigure the droplet network in situ. In addition, a mechanosensitive channel, MscL, can be repeatedly activated and deactivated in the levitated artificial cell by the application of acoustic and magnetic fields to modulate membrane tension on demand. This offers possibilities beyond one-time chemically mediated activation to provide repeated, non-contact control of membrane protein function. Collectively, this will expand our capability to program and operate increasingly sophisticated artificial cells as life-like materials.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.22269069', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Hydroxychloroquine/Chloroquine in COVID-19 With Focus on Hospitalized Patients - A Systematic Review', 'authors': \"[{'name': 'Daniel Freilich'}, {'name': 'Jennifer Victory'}, {'name': 'Anne Gadomski'}]\", 'abstract': '<p>Background\\nIn the beginning of the COVID-19 pandemic, many hospitalized patients received empiric hydroxychloroquine/chloroquine (HC/CQ). Although some retrospective-observational trials suggested potential benefit, all subsequent randomized clinical trials (RCTs) failed to show benefit and use generally ceased. Herein, we summarize key studies that clinicians advising patients on HC/CQ efficacy:safety calculus in hospitalized COVID-19 patients would want to know about in a practical one-stop-shopping source.\\n\\nMethods\\nPubmed and Google were searched on November 4, 2021. Search words included: COVID-19, hydroxychloroquine, chloroquine, in vitro, animal studies, clinical trials, and meta-analyses. Studies were assessed for import and included if considered impactful for benefit:risk assessment.\\n\\nResults\\nThese searches led to inclusion of 12 in vitro and animal reports; 12 retrospective-observational trials, 19 interventional clinical trials (17 RCTs, 1 single-arm, 1 controlled but unblinded), and 51 meta-analyses in hospitalized patients.\\nInconsistent efficacy was seen in vitro and in animal studies for coronaviruses and nil in SARS-CoV-2 animal models specifically. Most retrospective-observational studies in hospitalized COVID-19 patients found no efficacy; QT prolongation and increased adverse events and mortality were reported in some. All RCTs and almost all meta-analyses provided robust data showing no benefit in overall populations and subgroups, yet concerning safety issues in many.\\n\\nConclusions\\nHC/CQ have inconsistent anti-coronavirus efficacy in vitro and in animal models, and no convincing efficacy yet substantial safety issues in the overwhelming majority of retrospective-observational trials, RCTs, and meta-analyses in hospitalized COVID-19 patients. HC/CQ should not be prescribed for hospitalized COVID-19 patients outside of clinical trials.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.473633', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Hebbian activity-dependent plasticity in white matter', 'authors': \"[{'name': 'Alberto Lazari'}, {'name': 'Piergiorgio Salvan'}, {'name': 'Michiel Cottaar'}, {'name': 'Daniel Papp'}, {'name': 'Matthew Rushworth'}, {'name': 'Heidi Johansen-Berg'}]\", 'abstract': '<p>Synaptic plasticity is required for learning and follows Hebb’s Rule, the com-putational principle underpinning associative learning. In recent years, a complementary type of brain plasticity has been identified in myelinated axons, which make up the majority of brain’s white matter. Like synaptic plasticity, myelin plasticity is required for learning, but it is unclear whether it is Hebbian or whether it follows different rules. Here, we provide evidence that white matter plasticity operates following Hebb’s Rule in humans. Across two experiments, we find that co-stimulating cortical areas to induce Hebbian plasticity leads to relative increases in cortical excitability and associated increases in a myelin marker within the stimulated fiber bundle. We conclude that Hebbian plasticity extends beyond synaptic changes, and can be observed in human white matter fibers.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.22269229', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Endocrinology (including Diabetes Mellitus and Metabolic Disease)', 'covid': False, 'title': 'The impact of depression diagnosis on diabetes and lifetime hyperglycaemia', 'authors': \"[{'name': 'Saskia P Hagenaars'}, {'name': 'Alexandra C Gillett'}, {'name': 'Francesco Casanova'}, {'name': 'Katherine G Young'}, {'name': 'Harry Green'}, {'name': 'Cathryn M Lewis'}, {'name': 'Jess Tyrrell'}]\", 'abstract': '<sec><title>Aims</title><p>The aim of this study was to evaluate longitudinal associations between the mean and variability of HbA1c levels in individuals with type 2 diabetes (T2D) and major depressive disorder (MDD).</p></sec><sec><title>Methods</title><p>Individuals with T2D from the UK Biobank with linked primary care records were analysed. An HbA1c measurement within +/- 6-months of T2D diagnosis was taken as baseline, with subsequent HbA1c measurements used as the outcome in generalised least squares regression to evaluate longitudinal associations with a three-level MDD diagnosis variable (MDD controls, pre-T2D MDD cases and post-T2D MDD cases).</p></sec><sec><title>Results</title><p>Using 7,968 T2D individuals, we show that MDD has utility in explaining mean HbA1c levels (p=6.53E-08). This is attributable to MDD diagnosis interacting with baseline T2D medication (p=3.36E-04) and baseline HbA1c (p=2.66E-05), but not with time-when all else is equal, the temporal trend in expected HbA1c did not differ by MDD diagnosis. However, joint consideration with baseline T2D medication showed that each additional medication prescribed was associated with a +4 mmol/mol (2.5%) increase in expected HbA1c across follow up for post-T2D MDD cases, relative to pre-T2D MDD cases and MDD controls. Furthermore, variability in HbA1c increased across time for post-T2D MDD cases but decreased for MDD controls and pre-T2D MDD cases.</p></sec><sec><title>Conclusions</title><p>These findings suggest closer monitoring of individuals with both T2D and MDD is essential to improve their diabetic control, particularly for those who develop MDD after T2D diagnosis.</p></sec><sec><title>Novelty statement</title><sec><title>What is already known?</title><p>Comorbid T2D and MDD is associated with poorer diabetic control and worse prognosis.</p></sec><sec><title>What this study has found?</title><p>We demonstrate a strong complex relationship between MDD and diabetic control, influenced by diabetic medication and baseline HbA1c levels. We showed that individuals who develop MDD after their T2D diagnosis have greater variability in HbA1c levels over time.</p></sec><sec><title>What are the implications of the study?</title><p>This study shows the importance of closer monitoring of HbA1c in individuals with both T2D and MDD, particularly those who develop MDD after diabetes, to improve diabetic control and reduce complications associated comorbid T2D and MDD.</p></sec></sec>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.14.22269287', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': False, 'title': 'Pramipexole Enhances Reward Learning by Preserving Value Estimates', 'authors': \"[{'name': 'Don Chamith Halahakoon'}, {'name': 'Alexander Kaltenboeck'}, {'name': 'Marieke Martens'}, {'name': 'John G. Geddes'}, {'name': 'Catherine J. Harmer'}, {'name': 'Philip Cowen'}, {'name': 'Michael Browning'}]\", 'abstract': '<sec><title>Background</title><p>Dopamine D2-like receptor agonists show promise as treatments for depression. They are thought to act by altering how individuals learn from rewarding experiences. However, the nature of these reward learning alterations, and the mechanisms by which they are produced is not clear. Reinforcement learning accounts describe three distinct processes that may produce similar changes in reward learning behaviour; increased reward sensitivity, increased inverse decision temperature and decreased value decay. As these processes produce equivalent effects on behaviour, arbitrating between them requires measurement of how expectations and prediction errors are altered. In the present study, we characterised the behavioural effects of a sustained 2-week course of the D2/3/4 receptor agonist pramipexole on reward learning and used fMRI measures of expectation and prediction error to assess which of these three mechanistic processes were responsible for the behavioural effects.</p></sec><sec><title>Methods</title><p>40 healthy volunteers (Age: 18-43, 50% female) were randomly allocated to receive either two weeks of pramipexole (titrated to 1mg/day) or placebo in a double-blind, between subject design. Participants completed a probabilistic instrumental learning task, in which stimuli were associated with either rewards or losses, before the pharmacological intervention and twice between days 12-15 of the intervention (once with and once without fMRI). Both asymptotic choice accuracy, and a reinforcement learning model, were used to assess reward learning.</p></sec><sec><title>Results</title><p>Behaviourally, pramipexole specifically increased choice accuracy in the reward condition, with no effect in the loss condition. Pramipexole increased the BOLD response in the orbital frontal cortex during the expectation of win trials but decreased the BOLD response to reward prediction errors in the ventromedial prefrontal cortex. This pattern of results indicates that pramipexole enhances choice accuracy by reducing the decay of estimated values during reward learning.</p></sec><sec><title>Conclusions</title><p>The D2-like receptor agonist pramipexole enhances reward learning by preserving learned values. This is a plausible candidate mechanism for pramipexole’s observed anti-depressant effect.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.12.476116', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Impaired spatial memory in adult vitamin D deficient BALB/c mice is associated with reductions in spine density, nitric oxide, and neural nitric oxide synthase in the hippocampus', 'authors': \"[{'name': 'Md. Mamun Al-Amin'}, {'name': 'Robert K. P. Sullivan'}, {'name': 'Suzy. Alexander'}, {'name': 'David A. Carter'}, {'name': 'DanaKai. Bradford'}, {'name': 'Thomas H. J. Burne'}]\", 'abstract': '<p>Vitamin D deficiency is prevalent in adults and is associated with cognitive impairment. However, the mechanism by which adult vitamin D (AVD) deficiency affects cognitive function remains unclear. We examined spatial memory impairment in AVD-deficient BALB/c mice and its underlying mechanism by measuring spine density, long term potentiation (LTP), nitric oxide (NO), neuronal nitric oxide synthase (nNOS) and endothelial NOS (eNOS) in the hippocampus. Adult male BALB/c mice were fed a control or vitamin D deficient diet for 20 weeks. Spatial memory performance was measured using an active place avoidance (APA) task, where AVD-deficient mice had reduced latency entering the shock zone compared to controls. We characterised hippocampal spine morphology in the CA1 and dentate gyrus (DG) and made electrophysiological recordings in the hippocampus of behaviourally naïve mice to measure LTP. We next measured NO, as well as glutathione, lipid peroxidation and oxidation of protein products and quantified hippocampal immunoreactivity for nNOS and eNOS. Spine morphology analysis revealed a significant reduction in the number of mushroom spines in the CA1 dendrites but not in the DG. There was no effect of diet on LTP. However, hippocampal NO levels were depleted whereas other oxidation markers were unaltered by AVD deficiency. We also showed a reduced nNOS, but not eNOS, immunoreactivity. Finally, vitamin D supplementation for 10 weeks to AVD-deficient mice restored nNOS immunoreactivity to that seen in in control mice. Our results suggest that lower levels of NO, reduced nNOS immunostaining contribute to hippocampal-dependent spatial learning deficits in AVD-deficient mice.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476248', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'OrganoID: a versatile deep learning platform for organoid image analysis', 'authors': \"[{'name': 'Jonathan M Matthews'}, {'name': 'Brooke Schuster'}, {'name': 'Sara Saheb Kashaf'}, {'name': 'Ping Liu'}, {'name': 'Mustafa Bilgic'}, {'name': 'Andrey Rzhetsky'}, {'name': 'Savas Tay'}]\", 'abstract': '<p>Organoids are three-dimensional in vitro tissue models that closely represent the native heterogeneity, microanatomy, and functionality of an organ or diseased tissue. Analysis of organoid morphology, growth, and drug response is challenging due to the diversity in shape and size of organoids, movement through focal planes, and limited options for live-cell staining. Here, we present OrganoID, an open-source image analysis platform that automatically recognizes, labels, and tracks single organoids in brightfield and phase-contrast microscopy. The platform identifies organoid morphology pixel by pixel without the need for fluorescence or transgenic labeling and accurately analyzes a wide range of organoid types in time-lapse microscopy experiments. OrganoID uses a modified u-net neural network with minimal feature depth to encourage model generalization and allow fast execution. The network was trained on images of human pancreatic cancer organoids and was validated on images from pancreatic, lung, colon, and adenoid cystic carcinoma organoids with a mean intersection-over-union of 0.76. OrganoID measurements of organoid count and individual area concurred with manual measurements at 96% and 95% agreement respectively. Tracking accuracy remained above 89% over the duration of a four-day validation experiment. Automated single-organoid morphology analysis of a dose-response experiment identified significantly different organoid circularity after exposure to different concentrations of gemcitabine. The OrganoID platform enables straightforward, detailed, and accurate analysis of organoid images to accelerate the use of organoids as physiologically relevant models in high-throughput research.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.10.22269035', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Oncology', 'covid': False, 'title': 'Whole genome analysis reveals the genomic complexity in metastatic cutaneous squamous cell carcinoma', 'authors': \"[{'name': 'Amarinder S. Thind'}, {'name': 'Bruce Ashford'}, {'name': 'Dario Strbenac'}, {'name': 'Ruta Gupta'}, {'name': 'Jonathan R Clark'}, {'name': 'N. Gopalakrishna Iyer'}, {'name': 'Jenny Mitchell'}, {'name': 'Jenny Lee'}, {'name': 'Simon A Mueller'}, {'name': 'Elahe Minaei'}, {'name': 'Jay Perry'}, {'name': 'Marie Ranson'}]\", 'abstract': '<p>Metastatic cutaneous squamous cell carcinoma (cSCC) is associated with a high risk of recurrence and poor prognosis. There is limited published data exploring whole genome sequencing (WGS). The aim of this project was to provide the first comprehensive genomic understanding of the state of metastatic cSCC. In this study, we used WGS on matched tumor and blood DNA to detect somatic genetic alterations from 25 patients with regional metastases of head and neck cSCC. Our computational analyses interrogate clinical impacts of these genetic alterations on metastatic cSCC across the cohort for both the coding and non-coding genome.</p><p>In the non-coding genome, 3’UTR regions of <italic>EVC</italic> (48%), <italic>PPP1R1A</italic> (48%) and <italic>LUM</italic> (16%) were significantly functionally altered (Q-value &lt; 0.05). Further, significant functional alterations are observed in the tumor suppressing lncRNA LINC01003 (68% of specimens, Q-value: 0.0158). In addition, significant recurrent copy number loss in tumor suppressor genes <italic>KANSL1</italic> and <italic>PTPRD</italic> and gain in <italic>CALR, CCND1</italic> and <italic>FGF3</italic> was observed for coding regions. SNV’s driver analyses predicted <italic>TP53, CDKN2A</italic>, as potential drivers of the metastasis cSCC (using 3 different tools). Indel signature analysis highlight dominance of ID signature 13 followed by ID8 &amp; ID9. Interestingly, ID 9 has previously been shown to have no association with skin melanoma, unlike ID 13 and 8, suggesting some point of difference between these two skin-based diseases. The overall landscape of variation in metastatic cSCC is dominated by cell cycle and DNA repair disruption.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.475638', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A striatal plasticity that supports the long-term preservation of motor function in Parkinsonian mice', 'authors': \"[{'name': 'Joe C. Brague'}, {'name': 'Rebecca P. Seal'}]\", 'abstract': '<p>Motor deficits of Parkinson’s disease (PD) such as rigidity, bradykinesia and akinesia result from a progressive loss of nigrostriatal dopamine neurons. No therapies exist that slow their degeneration and the most effective treatments for the motor symptoms: L-dopa -the precursor to dopamine, and deep brain stimulation can produce dyskinesias and are highly invasive, respectively. Hence, alternative strategies targeted to slow the progression or delay the onset of motor symptoms are still highly sought. Here we report the identification of a long-term striatal plasticity mechanism that delays for several months, the onset of motor deficits in a mouse PD model. Specifically, we show that a one-week transient daily elevation of midbrain dopamine neuron activity during depletion preserves the connectivity of direct but not indirect pathway projection neurons. The findings are consistent with the balance theory of striatal output pathways and suggest a novel approach for treating the motor symptoms of PD.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475665', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Aedes aegypti and Aedes albopictus abundance, landscape coverage and spectral indices effects in a subtropical city of Argentina', 'authors': \"[{'name': 'Mía Elisa Martín'}, {'name': 'Ana Carolina Alonso'}, {'name': 'Janinna Faraone'}, {'name': 'Marina Stein'}, {'name': 'Elizabet Lilia Estallo'}]\", 'abstract': '<p>The presence, abundance and distribution of <italic>Aedes</italic> (<italic>Stegomyia</italic>) <italic>aegypti</italic> (Linnaeus 1762) and <italic>Aedes</italic> (<italic>Stegomyia</italic>) <italic>albopictus</italic> (Skuse 1894) could be conditioned by different data obtained from satellite remote sensors. In this paper, we aim to estimate the effect of landscape coverage and spectral indices on the abundance of <italic>Ae. aegypti</italic> and <italic>Ae. albopictus</italic> from the use of satellite remote sensors in Eldorado, Misiones, Argentina. Larvae of <italic>Aedes aegypti</italic> and <italic>Ae. albopictus</italic> were collected monthly from June 2016 to April 2018, in four outdoor environments: tire repair shops, cemeteries, family dwellings, and an urban natural park. The proportion of each land cover class was determined by Sentinel-2 image classification. Furthermore spectral indices were calculated. Generalized Linear Mixed Models were developed to analyze the possible effects of landscape coverage and vegetation indices on the abundance of mosquitoes. The model’s results showed the abundance of <italic>Ae. aegypti</italic> was better modeled by the minimum values of the NDVI index, the maximum values of the NDBI index and the interaction between both variables. In contrast, the abundance of <italic>Ae. albopictus</italic> has to be better explained by the model that includes the variables bare soil, low vegetation and the interaction between both variables.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475844', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Influence of data sampling methods on the representation of neural spiking activity in vivo', 'authors': \"[{'name': 'Meike E. van der Heijden'}, {'name': 'Amanda M. Brown'}, {'name': 'Roy V. Sillitoe'}]\", 'abstract': '<p><italic>In vivo</italic> single-unit recordings distinguish the basal spiking properties of neurons in different experimental settings and disease states. Here, we examined over 300 spike trains recorded from Purkinje cells and cerebellar nuclei neurons to test whether data sampling approaches influence the extraction of rich descriptors of firing properties. Our analyses included neurons recorded in awake and anesthetized control mice, as well as disease models of ataxia, dystonia, and tremor. We find that recording duration circumscribes overall representations of firing rate and pattern. Notably, shorter recording durations skew estimates for global firing rate variability towards lower values. We also find that only some populations of neurons in the same mouse are more similar to each other than to neurons recorded in different mice. These data reveal that recording duration and approach are primary considerations when interpreting task-independent single-neuron firing properties. If not accounted for, group differences may be concealed or exaggerated.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476072', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': \"Integrative in situ mapping of single-cell transcriptional states and tissue histopathology in an Alzheimer's disease model\", 'authors': \"[{'name': 'Hu Zeng'}, {'name': 'Jiahao Huang'}, {'name': 'Haowen Zhou'}, {'name': 'William J. Meilandt'}, {'name': 'Borislav Dejanovic'}, {'name': 'Yiming Zhou'}, {'name': 'Christopher J. Bohlen'}, {'name': 'Seung-Hye Lee'}, {'name': 'Jingyi Ren'}, {'name': 'Albert Liu'}, {'name': 'Hao Sheng'}, {'name': 'Jia Liu'}, {'name': 'Morgan Sheng'}, {'name': 'Xiao Wang'}]\", 'abstract': \"<p>Amyloid-β plaques and neurofibrillary tau tangles are the neuropathologic hallmarks of Alzheimer's disease (AD), but the spatiotemporal cellular responses and molecular mechanisms underlying AD pathophysiology remain poorly understood. Here we introduce STARmap PLUS to simultaneously map single-cell transcriptional states and disease marker proteins in brain tissues of AD mouse models at subcellular resolution (200 nm). This high-resolution spatial transcriptomics map revealed a core-shell structure where disease-associated microglia (DAM) closely contact amyloid-β plaques, whereas disease-associated astrocytes (DAA) and oligodendrocyte precursor cells (OPC) are enriched in the outer shells surrounding the plaque-DAM complex. Hyperphosphorylated tau emerged mainly in excitatory neurons in the CA1 region accompanied by the infiltration of oligodendrocyte subtypes into the axon bundles of hippocampal alveus. The integrative STARmap PLUS method bridges single-cell gene expression profiles with tissue histopathology at subcellular resolution, providing an unprecedented roadmap to pinpoint the molecular and cellular mechanisms of AD pathology and neurodegeneration.</p>\", 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.22269295', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Primary Care Research', 'covid': True, 'title': 'Simple risk scores to predict hospitalization or death in outpatients with COVID-19', 'authors': \"[{'name': 'Mark H. Ebell'}, {'name': 'Roya Hamadani'}, {'name': 'Autumn Kieber-Emmons'}]\", 'abstract': '<sec><title>Importance</title><p>Outpatient physicians need guidance to support their clinical decisions regarding management of patients with COVID-19, in particular whether to hospitalize a patient and if managed as an outpatient, how closely to follow them.</p></sec><sec><title>Objective</title><p>To develop and prospectively validate a clinical prediction rule to predict the likelihood of hospitalization for outpatients with COVID-19 that does not require laboratory testing or imaging.</p></sec><sec><title>Design</title><p>Derivation and temporal validation of a clinical prediction rule, and prospective validation of two externally derived clinical prediction rules.</p></sec><sec><title>Setting</title><p>Primary and Express care clinics in a Pennsylvania health system.</p></sec><sec><title>Participants</title><p>Patients 12 years and older presenting to outpatient clinics who had a positive polymerase chain reaction test for COVID-19.</p></sec><sec><title>Main outcomes and measures</title><p>Classification accuracy (percentage in each risk group hospitalized) and area under the receiver operating characteristic curve (AUC).</p></sec><sec><title>Results</title><p>Overall, 7.4% of outpatients in the early derivation cohort (5843 patients presenting before 3/1/21) and 5.5% in the late validation cohort (3806 patients presenting 3/1/21 or later) were ultimately hospitalized. We developed and temporally validated three risk scores that all included age, dyspnea, and the presence of comorbidities, adding respiratory rate for the second score and oxygen saturation for the third. All had very good overall accuracy (AUC 0.77 to 0.78) and classified over half of patients in the validation cohort as very low risk with a 1.7% or lower likelihood of hospitalization. Two externally derived risk scores identified more low risk patients, but with a higher overall risk of hospitalization (2.8%).</p></sec><sec><title>Conclusions and relevance</title><p>Simple risk scores applicable to outpatient and telehealth settings can identify patients with very low (1.6% to 1.7%), low (5.2% to 5.9%), moderate (14.7% to 15.6%), and high risk (32.0% to 34.2%) of hospitalization. The Lehigh Outpatient COVID Hospitalization (LOCH) risk score is available online as a free app: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://ebell-projects.shinyapps.io/LehighRiskScore/\">https://ebell-projects.shinyapps.io/LehighRiskScore/</ext-link>.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476375', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Identification of a transcriptional signature found in multiple models of ASD and related disorders', 'authors': '[{\\'name\\': \\'Samuel Thudium\\'}, {\\'name\\': \\'Katherine C Palozola\\'}, {\\'name\\': \"Eloise L\\'Her\"}, {\\'name\\': \\'Erica Korb\\'}]', 'abstract': '<p>Epigenetic regulation plays a critical role in many neurodevelopmental disorders, including Autism Spectrum Disorder (ASD). In particular, many such disorders are the result of mutations in genes that encode chromatin modifying proteins. However, while these disorders share many features, it is unclear whether they also share gene expression disruptions resulting from the aberrant regulation of chromatin. We examined 5 chromatin modifiers that are all linked to ASD despite their different roles in regulating chromatin. Specifically, we depleted Ash1L, Chd8, Crebbp, Ehmt1, and Nsd1 in parallel in a highly controlled neuronal culture system. We then identified sets of shared genes, or transcriptional signatures, that are differentially expressed following loss of multiple ASD-linked chromatin modifiers. We examined the functions of genes within the transcriptional signatures and found an enrichment in many neurotransmitter transport genes and activity-dependent genes. In addition, these genes are enriched for specific chromatin features such as bivalent domains that allow for highly dynamic regulation of gene expression. The downregulated transcriptional signature is also observed within multiple mouse models of neurodevelopmental disorders that result in ASD, but not those only associated with intellectual disability. Finally, the downregulated transcriptional signature can distinguish between neurons generated from iPSCs derived from healthy donors and idiopathic ASD patients through RNA-deconvolution, demonstrating that this gene set is relevant to the human disorder. This work identifies a transcriptional signature that is found within many neurodevelopmental syndromes, helping to elucidate the link between epigenetic regulation and the underlying cellular mechanisms that result in ASD.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.14.476345', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Griottes: a generalist tool for network generation from segmented tissue images', 'authors': \"[{'name': 'Gustave Ronteix'}, {'name': 'Valentin Bonnet'}, {'name': 'Sebastien Sart'}, {'name': 'Jeremie Sobel'}, {'name': 'Elric Esposito'}, {'name': 'Charles N. Baroud'}]\", 'abstract': '<p>Microscopy techniques and image segmentation algorithms have improved dramatically this decade, leading to an ever increasing amount of biological images and a greater reliance on imaging to investigate biological questions. This has created a need for methods to extract the relevant information on the behaviors of cells and their interactions, while reducing the amount of computing power required to organize this information. This task can be performed by using a network representation in which the cells and their properties are encoded in the nodes, while the neighborhood interactions are encoded by the links. Here we introduce <monospace>Griottes</monospace>, an open-source tool to build the “network twin” of 2D and 3D tissues from segmented microscopy images. We show how the library can provide a wide range of biologically relevant metrics on individual cells and their neighborhoods, with the objective of providing multi-scale biological insights. The library’s capacities are demonstrated on different image and data types. This library is provided as an open-source tool that can be integrated into common image analysis workflows to increase their capacities.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475882', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'EnteroBase: Hierarchical clustering of 100,000s of bacterial genomes into species/sub-species and populations', 'authors': \"[{'name': 'Mark Achtman'}, {'name': 'Zhemin Zhou'}, {'name': 'Jane Charlesworth'}, {'name': 'Laura Baxter'}]\", 'abstract': '<p>The definition of bacterial species is traditionally a taxonomic issue while defining bacterial populations is done with population genetics. These assignments are species specific, and depend on the practitioner. Legacy multilocus sequence typing is commonly used to identify sequence types (STs) and clusters (ST Complexes). However, these approaches are not adequate for the millions of genomic sequences from bacterial pathogens that have been generated since 2012. EnteroBase (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://enterobase.warwick.ac.uk\">http://enterobase.warwick.ac.uk</ext-link>) automatically clusters core genome MLST alleles into hierarchical clusters (HierCC) after assembling annotated draft genomes from short read sequences. HierCC clusters span core sequence diversity from the species level down to individual transmission chains. Here we evaluate HierCC’s ability to correctly assign 100,000s of genomes to the species/subspecies and population levels for <italic>Salmonella, Clostridoides, Yersinia, Vibrio</italic> and <italic>Streptococcus</italic>. HierCC assignments were more consistent with maximum-likelihood super-trees of core SNPs or presence/absence of accessory genes than classical taxonomic assignments or 95% ANI. However, neither HierCC nor ANI were uniformly consistent with classical taxonomy of <italic>Streptococcus.</italic> HierCC was also consistent with legacy eBGs/ST Complexes in <italic>Salmonella</italic> or <italic>Escherichia</italic> and revealed differences in vertical inheritance of O serogroups. Thus, EnteroBase HierCC supports the automated identification of and assignment to species/subspecies and populations for multiple genera.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475895', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Brain responses in aggression-prone individuals: A systematic review and meta-analysis of functional magnetic resonance imaging (fMRI) studies of anger- and aggression-eliciting tasks', 'authors': \"[{'name': 'Maja Nikolic'}, {'name': 'Patrizia Pezzoli'}, {'name': 'Natalia Jaworska'}, {'name': 'Michael C. Seto'}]\", 'abstract': '<sec><title>Background</title><p>While reactive aggression (in response to a perceived threat or provocation) is part of humans’ adaptive behavioral repertoire, it can violate social and legal norms. Understanding brain function in individuals with high levels of reactive aggression as they process anger- and aggression-eliciting stimuli is critical for refining interventions. Three neurobiological models of reactive aggression–the limbic hyperactivity, prefrontal hypoactivity, and dysregulated limbic-prefrontal connectivity models–have been proposed. However, these models are based on neuroimaging studies involving mainly healthy individuals, leaving it unclear which model best describes brain function in aggression-prone individuals.</p></sec><sec><title>Methods</title><p>We conducted a systematic literature search (PubMed and Psycinfo) and Multilevel Kernel Density meta-analysis (MKDA) of nine functional magnetic resonance imaging (fMRI) studies of brain responses to tasks putatively eliciting anger and aggression in aggression-prone individuals alone, and relative to healthy controls.</p></sec><sec><title>Results</title><p>Aggression-prone individuals exhibited greater activity during reactive aggression relative to baseline in the superior temporal gyrus and in regions comprising the cognitive control and default mode networks (right posterior cingulate cortex, precentral gyrus, precuneus, right inferior frontal gyrus). Compared to healthy controls, aggression-prone individuals exhibited increased activity in limbic regions (left hippocampus, left amygdala, left parahippocampal gyrus) and temporal regions (superior, middle, inferior temporal gyrus), and reduced activity in occipital regions (left occipital cortex, left calcarine cortex).</p></sec><sec><title>Conclusions</title><p>These findings lend support to the limbic hyperactivity model and further indicate altered temporal and occipital activity in anger- and aggression-eliciting situations that involve face and speech processing.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476259', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Three Reagents for in-Solution Enrichment of Ancient Human DNA at More than a Million SNPs', 'authors': \"[{'name': 'Nadin Rohland'}, {'name': 'Swapan Mallick'}, {'name': 'Matthew Mah'}, {'name': 'Robert M Maier'}, {'name': 'Nick J Patterson'}, {'name': 'David Reich'}]\", 'abstract': '<p>In-solution enrichment for hundreds of thousands of single nucleotide polymorphisms (SNPs) has been the source of &gt;70% of all genome-scale ancient human DNA data published to date. This approach has made it possible to generate data for one to two orders of magnitude lower cost than random shotgun sequencing, making it economical to study ancient samples with low proportions of human DNA, and increasing the rate of conversion of sampled remains into working data thereby facilitating ethical stewardship of human remains. So far, nearly all ancient DNA data obtained using in-solution enrichment has been generated using a set of bait sequences targeting about 1.24 million SNPs (the 1240k reagent). These sequences were published in 2015, but synthesis of the reagent has been cost-effective for only a few laboratories. In 2021, two companies made available reagents that target the same core set of SNPs along with supplementary content. Here, we test the properties of the three reagents on a common set of 27 ancient DNA libraries across a range of richness of DNA content and percentages of human molecules. All three reagents are highly effective at enriching many hundreds of thousands of SNPs. For all three reagents and a wide range of conditions, one round of enrichment produces data that is as useful as two rounds when tens of millions of sequences are read out as is typical for such experiments. In our testing, the Twist Ancient DNA reagent produces the highest coverages, greatest uniformity on targeted positions, and almost no bias toward enriching one allele more than another relative to shotgun sequencing. Allelic bias in 1240k enrichment has made it challenging to carry out joint analysis of these data with shotgun data, creating a situation where the ancient DNA community has been publishing two important bodes of data that cannot easily be co-analyzed by population genetic methods. To address this challenge, we introduce a subset of hundreds of thousands of SNPs for which 1240k data can be effectively co-analyzed with all other major data types.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.22268948', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Health Informatics', 'covid': True, 'title': 'Algorithmic Fairness and Bias Mitigation for Clinical Machine Learning: Insights from Rapid COVID-19 Diagnosis by Adversarial Learning', 'authors': \"[{'name': 'Jenny Yang'}, {'name': 'Andrew AS Soltan'}, {'name': 'Yang Yang'}, {'name': 'David A Clifton'}]\", 'abstract': '<p>Machine learning is becoming increasingly promi-\\nnent in healthcare. Although its benefits are clear, growing attention is being given to how machine learning may exacerbate existing biases and disparities. In this study, we introduce an\\nadversarial training framework that is capable of mitigating biases that may have been acquired through data collection or magnified during model development. For example, if one class is\\nover-presented or errors/inconsistencies in practice are reflected in the training data, then a model can be biased by these. To evaluate our adversarial training framework, we used the statistical definition of equalized odds. We evaluated our model for the task of rapidly predicting COVID-19 for patients presenting to\\nhospital emergency departments, and aimed to mitigate regional (hospital) and ethnic biases present. We trained our framework on a large, real-world COVID-19 dataset and demonstrated that\\nadversarial training demonstrably improves outcome fairness (with respect to equalized odds), while still achieving clinically-effective\\nscreening performances (NPV&gt;0.98). We compared our method to the benchmark set by related previous work, and performed prospective and external validation on four independent hospital\\ncohorts. Our method can be generalized to any outcomes, models, and definitions of fairness.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475781', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Consideration of genetic variation and evolutionary history in future conservation of Indian one-horned rhinoceros (Rhinoceros unicornis)', 'authors': \"[{'name': 'Tista Ghosh'}, {'name': 'Shrewshree Kumar'}, {'name': 'Kirtika Sharma'}, {'name': 'Parikshit Kakati'}, {'name': 'Amit Sharma'}, {'name': 'Samrat Mondol'}]\", 'abstract': '<p>The extant members of the Eurasian rhino species have experienced severe population and range declines through a combination of natural and anthropogenic factors since Pleistocene. The one-horned rhino is the only Asian species recovered from such strong population decline but most of their fragmented populations in India and Nepal are reaching carrying capacity. Implementation of any future reintroduction-based conservation efforts would greatly benefit from currently unavailable detailed genetic assessments and evolutionary history of these populations. We sequenced wild one-horned rhino mitogenome from all the extant populations (n=16 individuals) for the first time, identified the polymorphic sites and assessed genetic variation (2531bp mtDNA, n=111 individuals) across India. Results showed 30 unique rhino haplotypes distributed as three distinct genetic clades (F<sub>st</sub> value 0.68-1) corresponding to the states of Assam (n=28 haplotypes), West Bengal and Uttar Pradesh (both monomorphic). Phylogenetic analyses suggest earlier coalescence of Assam (∼0.5 Mya) followed by parallel divergence of West Bengal and Uttar Pradesh/Nepal (∼0.06-0.05Mya), supported by the paleobiogeographic history of the Indian subcontinent. Combined together, we propose recognising three ‘Evolutionary Significant Units (ESUs)’ of Indian rhino. As recent assessments suggest further genetic isolations of Indian rhinos at local scales, future management efforts should focus on identifying genetically variable founder animals and consider periodic supplementation events while planning future rhino reintroduction programs in India. Such well-informed, multidisciplinary approach is the only way to ensure evolutionary, ecological and demographic stability of the species across its range.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476355', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The C terminus of the mycobacterium ESX-1 secretion system substrate ESAT-6 is required for phagosomal membrane damage and virulence', 'authors': \"[{'name': 'Morwan M Osman'}, {'name': 'Jonthan K Shanahan'}, {'name': 'Frances Chu'}, {'name': 'Kevin Takaki'}, {'name': 'Malte Pinckert'}, {'name': 'Antonio J Pagan'}, {'name': 'Roland Brosch'}, {'name': 'William Conrad'}, {'name': 'Lalita Ramakrishnan'}]\", 'abstract': '<p>Mycobacterium tuberculosis and its close relative Mycobacterium marinum infect macrophages and induce the formation of granulomas, organized macrophage-rich immune aggregates. These mycobacterial pathogens can accelerate and co-opt granuloma formation for their benefit, using the specialized secretion system ESX-1, a key virulence determinant. ESX-1-mediated virulence is attributed to the damage it causes to the membranes of macrophage phagosomal compartments, within which the bacteria reside. This phagosomal damage, in turn, has been attributed to the membranolytic activity of ESAT-6, the major secreted substrate of ESX-1. However, mutations that perturb ESAT-6 membranolytic activity often result in global impairment of ESX-1 secretion. This has precluded an understanding of the causal and mechanistic relationships between ESAT-6 membranolysis and ESX-1-mediated virulence. Here, we identify two conserved residues in the unstructured C-terminal tail of ESAT-6 required for phagosomal damage, granuloma formation and virulence. Importantly, these ESAT-6 mutants have near-normal levels of secretion, far higher than the minimal threshold we establish is needed for ESX-1-mediated virulence early in infection. Unexpectedly, these loss-of-function ESAT-6 mutants retain the ability to lyse acidified liposomes. Thus, ESAT-6 virulence functions in vivo can be uncoupled from this in vitro surrogate assay. These uncoupling mutants highlight an enigmatic functional domain of ESAT-6 and provide key tools to investigate the mechanism of phagosomal damage and virulence.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.476163', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Multi-omic data helps improve prediction of personalised tumor suppressors and oncogenes', 'authors': \"[{'name': 'Malvika Sudhakar'}, {'name': 'Raghunathan Rengaswamy'}, {'name': 'Karthik Raman'}]\", 'abstract': '<p>The progression of tumorigenesis starts with a few mutational and structural driver events in the cell. Various cohort-based computational tools exist to identify driver genes but require a large number of samples to produce reliable results. Many studies use different methods to identify driver mutations/genes from mutations that have no impact on tumour progression; however, a small fraction of patients show no mutational events in any known driver genes.  Current unsupervised methods map somatic and expression data onto a network to identify the perturbation in the network. Our method is the first machine learning model to classify genes as tumour suppressor gene (TSG), oncogene (OG) or neutral, thus assigning the functional impact of the gene in the patient. In this study, we develop a multi-omic approach, PIVOT (Personalised Identification of driVer OGs and TSGs), to train on experimentally or computationally validated mutational and structural driver events. Given the lack of any gold standards for the identification of personalised driver genes, we label the data using four strategies and, based on classification metrics, show gene-based labelling strategies perform best. We build different models using SNV, RNA, and multi-omic features to be used based on the data available. Our models trained on multi-omic data improved predictions compared to mutation and expression data, achieving an accuracy &gt;0.99  for BRCA, LUAD and COAD datasets. We show network and expression-based features contribute the most to PIVOT. Our predictions on BRCA, COAD and LUAD cancer types reveal commonly altered genes such as TP53, and PIK3CA, which are predicted drivers for multiple cancer types. Along with known driver genes, our models also identify new driver genes such as PRKCA, SOX9 and PSMD4. Our multi-omic model labels both CNV and mutations with a more considerable contribution by CNV alterations. While predicting labels for genes mutated in multiple samples, we also label rare driver events occurring in as few as one sample. We also identify genes with dual roles within the same cancer type. Overall, PIVOT labels personalised driver genes as TSGs and OGs and also identifies rare driver genes. PIVOT is available at https://github.com/RamanLab/PIVOT.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476325', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'DeepGOZero: Improving protein function prediction from sequence and zero-shot learning based on ontology axioms', 'authors': \"[{'name': 'Maxat Kulmanov'}, {'name': 'Robert Hoehndorf'}]\", 'abstract': '<p>Motivation: Protein functions are often described using the Gene Ontology (GO) which is an ontology consisting of over 50,000 classes and a large set of formal axioms. Predicting the functions of proteins is one of the key challenges in computational biology and a variety of machine learning methods have been developed for this purpose. However, these methods usually require significant amount of training data and cannot make predictions for GO classes which have only few or no experimental annotations. \\nResults: We developed DeepGOZero, a machine learning model which improves predictions for functions with no or only a small number of annotations. To achieve this goal, we rely on a model-theoretic approach for learning ontology embeddings and combine it with neural networks for protein function prediction. DeepGOZero can exploit formal axioms in the GO to make zero-shot predictions, i.e., predict protein functions even if not a single protein in the training phase was associated with that function. Furthermore, the zero-shot prediction method employed by DeepGOZero is generic and can be applied whenever associations with ontology classes need to be predicted.\\nAvailability: http://github.com/bio-ontology-research-group/deepgozero</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.12.476124', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'BlenderPhotonics – a versatile environment for 3-D complex bio-tissue modeling and light transport simulations based on Blender', 'authors': \"[{'name': 'Yuxuang Zhang'}, {'name': 'Qianqian Fang'}]\", 'abstract': '<sec><title>Significance</title><p>Rapid advances in biophotonics techniques require quantitative, model-based computational approaches to obtain functional and structural information from increasingly complex and multi-scaled anatomies. The lack of efficient tools to accurately model tissue structures and subsequently perform quantitative multi-physics modeling greatly impedes the clinical translation of these modalities.</p></sec><sec><title>Aim</title><p>While the mesh-based Monte Carlo (MMC) method expands our capabilities in simulating complex tissues by using tetrahedral meshes, the generation of such domains often requires specialized meshing tools such as Iso2Mesh. Creating a simplified and intuitive interface for tissue anatomical modeling and optical simulations is essential towards making these advanced modeling techniques broadly accessible to the user community.</p></sec><sec><title>Approach</title><p>We responded to the above challenge by combining the powerful, open-source 3-D modeling software, Blender, with state-of-the-art 3-D mesh generation and MC simulation tools, utilizing the interactive graphical user interface (GUI) in Blender as the front-end to allow users to create complex tissue mesh models, and subsequently launch MMC light simulations.</p></sec><sec><title>Results</title><p>We have developed a Python-based Blender add-on – BlenderPhotonics – to interface with Iso2Mesh and MMC, allowing users to create, configure and refine complex simulation domains and run hardware-accelerated 3-D light simulations with only a few clicks. In this tutorial, we provide a comprehensive introduction to this new tool and walk readers through 5 examples, ranging from simple shapes to sophisticated realistic tissue models.</p></sec><sec><title>Conclusion</title><p>BlenderPhotonics is user-friendly and open-source, leveraging the vastly rich ecosystem of Blender. It wraps advanced modeling capabilities within an easy-to-use and interactive interface. The latest software can be downloaded at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://mcx.space/bp\">http://mcx.space/bp</ext-link>.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.475903', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The interactome of Cryptococcus neoformans Rmt5 reveals multiple regulatory points in fungal cell biology and pathogenesis', 'authors': \"[{'name': 'Murat C. Kalem'}, {'name': 'Harini Subbiah'}, {'name': 'Shichen Shen'}, {'name': 'Runpu Chen'}, {'name': 'Luke Terry'}, {'name': 'Yijun Sun'}, {'name': 'Jun Qu'}, {'name': 'John C. Panepinto'}]\", 'abstract': '<p>Protein arginine methylation is a key post-translational modification in eukaryotes that modulates core cellular processes, including translation, morphology, transcription, and RNA fate. However, this has not been explored in <italic>Cryptococcus neoformans</italic>, a human-pathogenic basidiomycetous encapsulated fungus. We characterized the five protein arginine methyltransferases in <italic>C. neoformans</italic> and highlight Rmt5 as critical regulator of cryptococcal morphology and virulence. An <italic>rmt5</italic>Δ mutant was defective in thermotolerance, had a remodeled cell wall, and exhibited enhanced growth in an elevated carbon dioxide atmosphere and in chemically induced hypoxia. We revealed that Rmt5 interacts with post-transcriptional gene regulators, such as RNA-binding proteins and translation factors. Further investigation of the <italic>rmt5</italic>Δ mutant showed that Rmt5 is critical for the homeostasis of eIF2α and its phosphorylation state following 3-amino-1,2,4-triazole-induced ribosome stalling. RNA sequencing of one <italic>rmt5</italic>Δ clone revealed stable chromosome 9 aneuploidy that was ameliorated by complementation but did not impact the <italic>rmt5</italic>Δ phenotype. As a result of these diverse interactions and functions, loss of <italic>RMT5</italic> enhanced phagocytosis by murine macrophages and attenuated disease progression in mice. Taken together, our findings link arginine methylation to critical cryptococcal cellular processes that impact pathogenesis, including post-transcriptional gene regulation by RNA-binding proteins.</p><sec><title>Significance</title><p>The fungal pathogen <italic>Cryptococcus neoformans</italic> is a huge threat for people living with immune deficits, especially HIV/AIDS. Its virulence potential is dependent on virulence factors, stress adaptation, and thermotolerance. Post-transcriptional gene regulation is important for these pathogenic processes, but the mechanisms that govern post-transcriptional regulator function are unexplored. Protein arginine methylation is a major modification of post-transcriptional regulators that has not been investigated in pathogenic fungi. Here we investigated the role of arginine methylation by arginine methyltransferases on the biology and virulence of <italic>C. neoformans</italic>. Phenotypic characterization of deletion mutants revealed pleiotropic functions for RMTs in this pathogen. Further investigation of the Rmt5 interactome using proximity-dependent biotinylation revealed interactions with RNA binding proteins and translation factors, thereby impacting virulence-associated processes.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.15.475352', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structural Insights into Sphingosine-1-phosphate Receptor Activation', 'authors': \"[{'name': 'Leiye Yu'}, {'name': 'Licong He'}, {'name': 'Bing Gan'}, {'name': 'Rujuan Ti'}, {'name': 'Qingjie Xiao'}, {'name': 'Hongli Hu'}, {'name': 'Lizhe Zhu'}, {'name': 'Sheng Wang'}, {'name': 'Ruobing Ren'}]\", 'abstract': '<p>As a critical sphingolipid metabolite, sphingosine-1-phosphate (S1P) plays an essential role in immune and vascular systems. There are five S1P receptors, designated as S1PR1-5, encoded in the human genome, and their activities are governed by endogenous S1P, lipid-like S1P mimics, or non-lipid-like therapeutic molecules. Among S1PRs, S1PR1 stands out due to its non-redundant functions, such as the egress of T and B cells from the thymus and secondary lymphoid tissues, making it a potential therapeutic target. However, the structural basis of S1PR1 activation and regulation by various agonists remains unclear. Here we reported four atomic resolution cryo-EM structures of Gi-coupled human S1PR1 complexes: bound to endogenous agonist d18:1 S1P, benchmark lipid-like S1P mimic phosphorylated Fingolimod ((S)-FTY720-P), or non-lipid-like therapeutic molecule CBP-307 in two binding modes. Our results revealed the similarities and differences of activation of S1PR1 through distinct ligands binding to the amphiphilic orthosteric pocket. We also proposed a two-step \"shallow to deep\" transition process of CBP-307 for S1PR1 activation. Both binding modes of CBP-307 could activate S1PR1, but from shallow to deep transition may trigger the rotation of the N-terminal helix of Gαi and further stabilize the complex by increasing the Gαi interaction with the cell membrane. We combine with extensive biochemical analysis and molecular dynamic simulations to suggest key steps of S1P binding and receptor activation. The above results decipher the common feature of the S1PR1 agonist recognition and activation mechanism and will firmly promote the development of therapeutics targeting S1P receptors.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.22269323', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Cerebrospinal fluid neurofilament light chain differentiates behavioural variant frontotemporal dementia progressors from phenocopy non-progressors', 'authors': \"[{'name': 'Dhamidhu Eratne'}, {'name': 'Michael Keem'}, {'name': 'Courtney Lewis'}, {'name': 'Matthew Kang'}, {'name': 'Mark Walterfang'}, {'name': 'Samantha M Loi'}, {'name': 'Wendy Kelso'}, {'name': 'Claire Cadwallader'}, {'name': 'Samuel F Berkovic'}, {'name': 'Qiao-Xin Li'}, {'name': 'Colin L Masters'}, {'name': 'Steven Collins'}, {'name': 'Alexander Santillo'}, {'name': 'Dennis Velakoulis'}, {'name': ' '}]\", 'abstract': '<p>Background: \\nDistinguishing behavioural variant frontotemporal dementia (bvFTD) from non-neurodegenerative non-progressor, phenocopy mimics of frontal lobe dysfunction, can be one of the most challenging clinical dilemmas. A biomarker of neuronal injury, neurofilament light chain (NfL), could reduce misdiagnosis and delay.\\n\\nMethods: \\nCerebrospinal fluid (CSF) NfL, amyloid beta 1-42 (AB42), total and phosphorylated tau (T-tau, P-tau) levels were examined in patients with an initial diagnosis of bvFTD. Based on follow up information, patients were categorised as Progressors. Non-Progressors were subtyped in to Phenocopy Non-Progressors (non-neurological/neurodegenerative final diagnosis), and Static Non-Progressors (static deficits, not fully explained by non-neurological/neurodegenerative causes).\\n\\nResults: \\nForty-three patients were included: 20 Progressors, 23 Non-Progressors (15 Phenocopy, 8 Static), 20 controls. NfL concentrations were lower in Non-Progressors (Non-Progressors Mean, M=554pg/mL, 95%CI:[461, 675], Phenocopy Non-Progressors M=459pg/mL, 95%CI:[385, 539], Static Non-Progressors M=730pg/mL, 95%CI:[516, 940]), compared to bvFTD Progressors (M=2397pg/mL, 95%CI:[1607, 3332]).  NfL distinguished Progressors from Non-Progressors with the highest accuracy (area under the curve 0.92, 90%/87% sensitivity/specificity, 86%/91% positive/negative predictive value, 88% accuracy). Static Non-Progressors tended to have higher T-tau and P-tau levels compared to Phenocopy Non-Progressors.  \\n\\nConclusion: \\nThis study demonstrated strong diagnostic utility of CSF NfL to distinguish bvFTD from phenocopy non-progressor variants, at baseline, with high accuracy, in a real-world clinical setting. This has important clinical implications, to improve outcomes for patients and clinicians facing this challenging clinical dilemma, as well as for healthcare services, and clinical trials. Further research is required to investigate heterogeneity within the non-progressor group and potential diagnostic algorithms, and prospective studies are underway assessing plasma NfL</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.12.475985', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Role of Yap1 in adult neural stem cell activation', 'authors': \"[{'name': 'Wenqiang Fan'}, {'name': 'Jerónimo Jurado-Arjona'}, {'name': 'Gregorio Alanis-Lobato'}, {'name': 'Sophie Péron'}, {'name': 'Christian Berger'}, {'name': 'Miguel A. Andrade-Navarro'}, {'name': 'Sven Falk'}, {'name': 'Benedikt Berninger'}]\", 'abstract': '<p>Most adult hippocampal neural stem cells (NSCs) remain quiescent with only a minor portion undergoing active proliferation and neurogenesis. The molecular mechanisms that trigger eventually the transition from quiescence to activation are still poorly understood. Here, we found the activity of the transcriptional activator Yap1 to be enriched in active NSCs. Genetic deletion of Yap1 led to a significant reduction in the relative proportion of active NSCs supporting a physiological role of Yap1 in regulating the transition from quiescence to activation. Overexpression of wild type Yap1 in adult NSCs did not induce NSC activation suggesting tight upstream control mechanisms, but overexpression of a gain-of-function mutant (Yap1-5SA) elicited cell cycle entry in NSCs and hilar astrocytes. Consistent with a role of Yap1 in NSC activation, single cell RNA sequencing revealed the partial induction of an activated NSC gene expression program. Yet, Yap1-5SA expression also induced Taz and other key components of the Yap/Taz regulon previously identified in glioblastoma stem cell-like cells. Consequently, dysregulated Yap1 activity led to repression of hippocampal neurogenesis, promoting aberrant differentiation instead.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.11.22269065', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'The incidence and mortality of COVID-19 related TB infection in Sub-Saharan Africa: A systematic review and meta-analysis', 'authors': \"[{'name': 'Jacques L Tamuzi'}, {'name': 'Gomer Lulendo'}, {'name': 'Patrick Mbuesse'}]\", 'abstract': '<sec><title>Background</title><p>Coronavirus disease 2019 (COVID-19) is also associated with other co-morbidities among with previous and current pulmonary tuberculosis (PTB). PTB is a risk factor for COVID-19, both in terms of severity and mortality, regardless of human immunodeficiency virus (HIV) status. However, there is less information available on COVID-19 associated with PTB in point of view incidence and mortality rates in sub-Saharan Africa (SSA) as a high burden TB region. This systematic review served to provide data synthesis of available evidence on COVID-19/PTB incidence and case fatality rates, and mortality rate found in clinical and post-mortem COVID-19/PTB diagnostics in SSA.</p></sec><sec><title>Methods</title><p>We conducted a systematic electronic search in the PubMed, Medline, Google Scholar, Medrxix and COVID-19 Global literature on coronavirus disease databases for studies including COVID-19 associated with PTB in sub-Saharan Africa. The main outcomes were the proportion of people with COVID-19 associated to current /or previous PTB and the case fatality associated to COVID-19/PTB. The combination method was based on methodological similarities in the included random effect model studies using Prometa 3 software. We further undertook sensitivity analysis and meta-regression.</p></sec><sec><title>Results</title><p>From the 548 references extracted by the literature search, 25 studies were selected and included in the meta-analysis with a total of 191, 250 COVID-19 infected patients and 11, 452 COVID-19 deaths. The pooled COVID-19/PTB incidence was 2% [1%-3%] and mortality of 10% [4%-20%]. The pooled estimates for case fatality rate among COVID-19/PTB were 6% [3%-11%] for clinical PTB diagnostic and 26% [14%-48%] for post-mortem PTB diagnostic. Meta-regression model including the effect sizes and cumulative COVID-19 cases (P= 0.032), HIV prevalence (P= 0.041) and TB incidence (P= 0.002) to explained high heterogeneity between studies.</p></sec><sec><title>Conclusion</title><p>As a summary, the incidence of TB associated with COVID-19 and case fatality rates are higher in SSA. However, COVID-19 associated to TB may be underreported in the studies conducted in SSA as the post-mortem TB diagnostic was higher. Large-scale cohort studies that adequately clear tool on previous and/or current TB diagnostic tools are required to confirmed COVID-19/TB incidence and case fatality in SSA.</p></sec><sec><title>Review registration</title><p>PROSPERO (CRD42021233387)</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.22268812', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Independent morphological variables correlate with Aging, Mild Cognitive Impairment, and Alzheimer’s Disease', 'authors': \"[{'name': 'Fernanda Hansen Pacheco de Moraes'}, {'name': 'Felipe Sudo'}, {'name': 'Marina Monteiro Carneiro'}, {'name': 'Bruno R. P. de Melo'}, {'name': 'Paulo Mattos'}, {'name': 'Bruno Mota'}, {'name': 'Fernanda Tovar-Moll'}]\", 'abstract': '<p>This manuscript presents a study with recruited volunteers that comprehends three sorts of events present in Alzheimer’s Disease (AD) evolution (structural, biochemical, and cognitive) to propose an update in neurodegeneration biomarkers for AD. The novel variables, K, I, and S, suggested based on physics properties and empirical evidence, are defined by power-law relations between cortical thickness, exposed and total area, and natural descriptors of brain morphology. Our central hypothesis is that variable K, almost constant in healthy human subjects, is a better discriminator of a diseased brain than the current morphological biomarker, Cortical Thickness, due to its aggregated information. We extracted morphological features from 3T MRI T1w images of 123 elderly subjects: 77 Healthy Cognitive Unimpaired Controls (CTL), 33 Mild Cognitive Impairment (MCI) patients, and 13 Alzheimer’s Disease (AD) patients. Moreover, Cerebrospinal Fluid (CSF) biomarkers and clinical data scores were correlated with K, intending to characterize health and disease in the cortex with morphological criteria and cognitive-behavioral profiles. K distinguishes Alzheimer’s Disease, Mild Cognitive Impairment, and Healthy Cognitive Unimpaired Controls globally and locally with reasonable accuracy (CTL-AD, 0.82; CTL-MCI, 0.58). Correlations were found between global and local K associated with clinical behavioral data (executive function and memory assessments) and CSF biomarkers (t-Tau, A<italic>β</italic>-40, and A<italic>β</italic>-42). The results suggest that the cortical folding component, K, is a premature discriminator of healthy aging, Mild Cognitive Impairment, and Alzheimer’s Disease, with significant differences within diagnostics. Despite the non-concomitant events, we found correlations between brain structural degeneration (K), cognitive tasks, and biochemical markers.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.13.22269022', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'A national survey of early treatment seeking behavior among those with incident SARS-CoV-2 infection', 'authors': \"[{'name': 'Noah Kojima'}, {'name': 'Matthew Brobeck'}, {'name': 'Vladimir Slepnev'}, {'name': 'Jeffrey D Klausner'}]\", 'abstract': '<p>Background: Despite effective means to treat SARS-CoV-2 infection, the early treatment seeking behavior of those newly diagnosed with infection is not clear.\\nMethods: We surveyed users of a national SARS-CoV-2 testing company to assess the frequency and correlates of early treatment seeking behavior for a positive test result.\\nWe recruited adults (18 years or older) who had tested positive for SARS-CoV-2 by PCR at a large clinical laboratory. To be eligible, individuals had to have a positive test result within 7 days of enrollment. Surveys were anonymous and voluntary. We collected data on demographic characteristics, general health care access and utilization, awareness of treatment for COVID-19, treatment seeking behavior, and treatments received. Descriptive statistics and odds ratios (OR) with 95% confidence intervals (95% CI) were calculated on StataSE.\\nResults: Participants were surveyed from 3-7 January 2022: among the 15,991 who viewed a survey request, 7,647 individuals were eligible and provided responses. The median age of a respondent was 42 years (interquartile range: 32 to 54), 68.9% of respondents were women, and respondents represented 33 different states, districts, and territories. Among respondents, 23.1% reported they had sought treatment or medical advice for their current COVID-19 diagnosis. Of those who were very aware of treatment for COVID-19, 31.0% sought treatment versus 16.7% who were unaware (p-value&lt; 0.001). The odds of treatment seeking behavior were higher for those that were contacted by a medical professional after their diagnosis (OR: 4.57 [95% CI: 3.89 to 5.37]), those with a primary doctor (OR: 2.94 [95% CI: 2.52 to 3.43]), those who self-measured their oxygen saturation (OR: 2.53 [95% CI: 2.25 to 2.84]), and those over 65 years of age (OR: 2.36 [95% CI: 2.02 to 2.76]). There was no difference in those seeking treatment based on heritage, ethnicity, prior COVID-19 diagnosis, state political affiliation, or vaccination status. The odds of seeking treatment were lower among men (OR: 0.88 [95% CI: 0.78 to 0.99]) and those without insurance (OR: 0.62 [95% CI: 0.52 to 0.72]). The most common treatment locations were clinics and most common treatments were Vitamin C, Vitamin D, Zinc, Tylenol, and NSAIDs.\\nConclusion: More public outreach is needed to raise awareness of the benefits of treatment for COVID-19. We found that people who were more aware about treatment for COVID-19 were more likely to seek medical advice or therapy. Efforts to increase awareness might increase early treatment for SARS-CoV-2 infection. Increased outreach with treatment facilitation from medical professionals and/or public health staff to those with newly detected SARS-CoV-2 infections, particularly among those at higher-risk of complications, might also be helpful.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475938', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Generation and characterisation of P. falciparum parasites with a G358S mutation in the PfATP4 Na+ pump and clinically relevant levels of resistance to some PfATP4 inhibitors', 'authors': \"[{'name': 'Deyun Qiu'}, {'name': 'Jinxin V. Pei'}, {'name': 'James E. O. Rosling'}, {'name': 'Dongdi Li'}, {'name': 'Yi Xue'}, {'name': 'Jocelyn Sietsma Penington'}, {'name': 'Krittikorn Kümpornsin'}, {'name': 'Yi Tong Vincent Aw'}, {'name': 'Jessica Yi Han Aw'}, {'name': 'Heath Hasemer'}, {'name': 'Adelaide S. M. Dennis'}, {'name': 'Melanie C. Ridgway'}, {'name': 'Anthony T. Papenfuss'}, {'name': 'Marcus C. S. Lee'}, {'name': 'Giel G. van Dooren'}, {'name': 'Kiaran Kirk'}, {'name': 'Adele M. Lehane'}]\", 'abstract': '<p>Small-molecule inhibitors of PfATP4, a <italic>Plasmodium falciparum</italic> protein that is believed to pump Na<sup>+</sup> out of the parasite while importing H<sup>+</sup>, are on track to become much-needed new antimalarial drugs. The spiroindolone cipargamin is poised to become the first PfATP4 inhibitor to reach the field, having performed strongly in Phase 1 and 2 clinical trials. Previous attempts to generate cipargamin-resistant parasites in the laboratory have yielded parasites with reduced susceptibility to the drug; however, the highest 50% inhibitory concentration reported to date is 24 nM. Here, we show that <italic>P. falciparum</italic> parasites can acquire a clinically-significant level of resistance to cipargamin that enables them to withstand micromolar concentrations of the drug. Independent experiments to generate high-level cipargamin resistance using different protocols and strains led to the same change each time – a G358S mutation in PfATP4. Parasites with this mutation showed high-level resistance not only to cipargamin, but also to the dihydroisoquinolone (+)-SJ733. However, for certain other (less clinically advanced) PfATP4-associated compounds the G358S mutation in PfATP4 conferred only moderate resistance or no resistance. The G358S mutation in PfATP4 did not affect parasite susceptibility to antimalarials that do not target PfATP4. The G358S mutation in PfATP4, and the equivalent mutation in the <italic>Toxoplasma gondii</italic> ATP4 homologue (G419S), decreased the sensitivity of the Na<sup>+</sup>-ATPase activity of ATP4 to inhibition by cipargamin and (+)-SJ733, and decreased the sensitivity of parasites expressing these ATP4 mutations to disruption of parasite Na<sup>+</sup> regulation by cipargamin- and (+)-SJ733. The G358S mutation in PfATP4 reduced the affinity of the protein for Na<sup>+</sup> and was associated with an increase in the parasite’s resting cytosolic Na<sup>+</sup> concentration; however, no significant defect in parasite growth rate was observed. Our findings suggest that codon 358 in <italic>pfatp4</italic> should be monitored closely in the field as a molecular marker for cipargamin resistance, and that PfATP4 inhibitors in clinical development should be tested for their activity against PfATP4<sup>G358S</sup> parasites.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475810', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'EagleImp: Fast and Accurate Genome-wide Phasing and Imputation in a Single Tool', 'authors': \"[{'name': 'Lars Wienbrandt'}, {'name': 'David Ellinghaus'}]\", 'abstract': '<sec><title>Background</title><p>Reference-based phasing and genotype imputation algorithms have been developed with sublinear theoretical runtime behaviour, but runtimes are still high in practice when large genome-wide reference datasets are used.</p></sec><sec><title>Methods</title><p>We developed <italic>EagleImp</italic>, a software with algorithmic and technical improvements and new features for accurate and accelerated phasing and imputation in a single tool.</p></sec><sec><title>Results</title><p>We compared accuracy and runtime of <italic>EagleImp</italic> with <italic>Eagle2, PBWT</italic> and prominent imputation servers using whole-genome sequencing data from the 1000 Genomes Project, the Haplotype Reference Consortium and simulated data with more than 1 million reference genomes. <italic>EagleImp</italic> is 2 to 10 times faster (depending on the single or multiprocessor configuration selected) than <italic>Eagle2/PBWT</italic>, with the same or better phasing and imputation quality in all tested scenarios. For common variants investigated in typical GWAS studies, <italic>EagleImp</italic> provides same or higher imputation accuracy than the <italic>Sanger Imputation Service, Michigan Imputation Server</italic> and the newly developed <italic>TOPMed Imputation Server</italic>, despite larger (not publicly available) reference panels. It has many new features, including automated chromosome splitting and memory management at runtime to avoid job aborts, fast reading and writing of large files, and various user-configurable algorithm and output options.</p></sec><sec><title>Conclusions</title><p>Due to the technical optimisations, <italic>EagleImp</italic> can perform fast and accurate reference-based phasing and imputation for future very large reference panels with more than 1 million genomes. <italic>EagleImp</italic> is freely available for download from <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/ikmb/eagleimp\">https://github.com/ikmb/eagleimp</ext-link>.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.475405', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Characterizing the conformational free-energy landscape of RNA using single-molecule field-effect transistors', 'authors': \"[{'name': 'Sukjin S. Jang'}, {'name': 'Sarah Dubnik'}, {'name': 'Jason Hon'}, {'name': 'Colin Nuckolls'}, {'name': 'Ruben L. Gonzalez'}]\", 'abstract': '<p>We have developed and used high-time-resolution, single-molecule field-effect transistors (smFETs) to characterize the conformational free-energy landscape of RNA stem-loops. Stem-loops are some of the most common RNA structural motifs and serve as building blocks for the formation of more complex RNA structures. Given their prevalence and integral role in RNA folding, the kinetics of stem-loop (un)folding has been extensively characterized using both experimental and computational approaches. Interestingly, these studies have reported vastly disparate timescales of (un)folding, which has been recently interpreted as evidence that (un)folding of even simple stem-loops occurs on a highly rugged conformational energy landscape. Because smFETs do not rely on fluorophore reporters of conformation or on the application of mechanical (un)folding forces, they provide a unique and complementary approach that has allowed us to directly monitor tens of thousands of (un)folding events of individual stem-loops at a 200 μs time resolution. Our results show that under our experimental conditions, stem-loops fold and unfold over a 1-200 ms timescale during which they transition between ensembles of unfolded and folded conformations, the latter of which is composed of at least two sub-populations. The 1-200 ms timescale of (un)folding we observe here indicates that smFETs report on complete (un)folding trajectories in which relatively extended unfolded conformations of the RNA spend long periods of time wandering the free-energy landscape before sampling one of several misfolded conformations or, alternatively, the natively folded conformation. Our findings demonstrate how the combination of single-molecule sensitivity and high time resolution makes smFETs unique and powerful tools for characterizing the conformational free-energy landscape of RNA and highlight the extremely rugged landscape on which even the simplest RNA structural elements fold.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475901', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': True, 'title': 'Effect of cannabidiol on apoptosis and cellular interferon and interferon-stimulated gene responses to the SARS-CoV-2 genes ORF8, ORF10 and M protein', 'authors': \"[{'name': 'Maria Fernanda Fernandes'}, {'name': 'John Zewen Chan'}, {'name': 'Chia Chun Joey Hung'}, {'name': 'Michelle Victoria Tomczewski'}, {'name': 'Robin Elaine Duncan'}]\", 'abstract': '<sec><title>Aims</title><p>To study effects on cellular innate immune responses to novel genes <italic>ORF8</italic> and <italic>ORF10,</italic> and the more conserved <italic>Membrane protein</italic> (<italic>M protein</italic>) from the Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) that causes COVID-19, either alone, or in combination with cannabidiol (CBD).</p></sec><sec><title>Main Methods</title><p>HEK293 cells were transfected with a control plasmid, or plasmids expressing <italic>ORF8</italic>, <italic>ORF10</italic>, or <italic>M protein</italic>, and assayed for cell number and markers of apoptosis at 24 h, and expression of interferon and interferon-stimulated genes at 14 h.</p></sec><sec><title>Key findings</title><p>A significant reduction in cell number, and increase in early and late apoptosis, was found after 24 h in cells where expression of viral genes was combined with 1-2 μM CBD treatment, but not in control-transfected cells treated with CBD, or in cells expressing viral genes but treated only with vehicle. CBD (2 μM) augmented expression of <italic>IFNγ</italic>, <italic>IFNλ1</italic> and <italic>IFNλ2/3</italic>, as well as the 2’-5’-oligoadenylate synthetase (OAS) family members <italic>OAS1</italic>, <italic>OAS2</italic>, <italic>OAS3</italic>, and <italic>OASL,</italic> in cells expressing <italic>ORF8</italic>, <italic>ORF10</italic>, and <italic>M protein</italic>. CBD also augmented expression of these genes in control cells not expressing viral genes, without enhancing apoptosis.</p></sec><sec><title>Significance</title><p>Our results demonstrate a poor ability of HEK293 cells to respond to SARS-CoV-2 genes alone, but suggest an augmented innate anti-viral response to these genes in the presence of CBD. Furthermore, our results indicate that CBD may prime components of the innate immune system, increasing readiness to respond to viral infection without activating apoptosis, and therefore could be studied for potential in prophylaxis.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476070', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Mitochondrial Network Configuration Influences Sarcomere and Myosin Filament Structure in Striated Muscles', 'authors': \"[{'name': 'Prasanna Katti'}, {'name': 'Alexander S. Hall'}, {'name': 'Peter T. Ajayi'}, {'name': 'Yuho Kim'}, {'name': 'T. Bradley Willingham'}, {'name': 'Christopher K. E. Bleck'}, {'name': 'Han Wen'}, {'name': 'Brian Glancy'}]\", 'abstract': '<p>Sustained muscle contraction occurs through interactions between actin and myosin filaments within sarcomeres and requires a constant supply of adenosine triphosphate (ATP) from nearby mitochondria. However, it remains unclear how different physical configurations between sarcomeres and mitochondria alter the energetic support for contractile function. Here, we show that sarcomere cross-sectional area (CSA) varies along its length in a cell type-dependent manner where the reduction in Z-disk CSA relative to the sarcomere center is closely coordinated with mitochondrial network configuration in flies, mice, and humans. Further, we find myosin filaments near the sarcomere periphery are curved relative to interior filaments with greater curvature for filaments near mitochondria compared to sarcoplasmic reticulum. Finally, we demonstrate smaller myosin filament lattice spacing at filament ends than filament centers in a cell type-dependent manner. These data suggest both sarcomere structure and myofilament interactions are influenced by the location and orientation of mitochondria within muscle cells.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476158', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The erythrocyte membrane properties of beta thalassaemia heterozygotes and their consequences for Plasmodium falciparum invasion', 'authors': \"[{'name': 'Viola Introini'}, {'name': 'Alejandro Marin-Menendez'}, {'name': 'Guilherme Nettesheim'}, {'name': 'Yen-Chun Lin'}, {'name': 'Silvia N. Kariuki'}, {'name': 'Adrian L Smith'}, {'name': 'Letitia Jean'}, {'name': 'John N. Brewin'}, {'name': 'David C. Rees'}, {'name': 'Pietro Cicuta'}, {'name': 'Julian C. Rayner'}, {'name': 'Bridget S. Penman'}]\", 'abstract': '<p>Malaria parasites such as <italic>Plasmodium falciparum</italic> have exerted formidable selective pressures on the human genome. Of the human genetic variants associated with malaria protection, beta thalassaemia (a haemoglobinopathy) was the earliest to be associated with malaria prevalence. However, the malaria protective properties of beta thalassaemic erythrocytes remain unclear. Here we studied the mechanics and surface protein expression of beta thalassaemia heterozygous erythrocytes, measured their susceptibility to <italic>P. falciparum</italic> invasion, and calculated the energy required for merozoites to invade them. We found invasion-relevant differences in beta thalassaemic cells versus matched controls, specifically: elevated membrane tension, reduced bending modulus, and higher levels of expression of the major invasion receptor basigin. However, these differences acted in opposition to each other with respect to their likely impact on invasion, and overall we did not observe beta thalassaemic cells to have lower <italic>P. falciparum</italic> invasion efficiency for any of the strains tested.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476239', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Expressing the pro-apoptotic Reaper protein via insertion into the structural open reading frame of Sindbis virus reduces the ability to infect Aedes aegypti mosquitoes', 'authors': \"[{'name': 'Alexis Carpenter'}, {'name': 'Rollie J. Clem'}]\", 'abstract': '<p>Arboviruses continue to threaten a significant portion of the human population, and a better understanding is needed of the determinants of successful arbovirus infection of arthropod vectors. Avoiding apoptosis has been shown to be one such determinant. Previous work showed that a Sindbis virus (SINV) construct called MRE/rpr that expresses the pro-apoptotic protein Reaper via a duplicated subgenomic promoter had a reduced ability to orally infect <italic>Aedes aegypti</italic> mosquitoes at 3 days post-blood meal (PBM), but this difference diminished over time as virus variants containing deletions in the inserted <italic>reaper</italic> gene rapidly predominated. The goal of this study was to generate a SINV construct that more stably expressed Reaper, in order to further clarify the effect of midgut apoptosis on disseminated infection in <italic>Ae. aegypti</italic>. We did this by inserting <italic>reaper</italic> as an in-frame fusion into the structural open reading frame (ORF) of SINV. This construct, MRE/rprORF, successfully expressed Reaper, replicated similarly to MRE/rpr in cell lines, and induced apoptosis in cultured cells and in mosquito midgut tissue. Mosquitoes that fed on blood containing MRE/rprORF developed less midgut and disseminated infection when compared to MRE/rpr or a control virus up to at least 7 days PBM, when less than 50% of mosquitoes that ingested MRE/rprORF had detectable disseminated infection, compared with around 80% or more of mosquitoes fed with MRE/rpr or control virus. However, virus titer in mosquitoes infected with MRE/rprORF was not significantly different from control virus, suggesting that induction of apoptosis by expression of Reaper by this method can reduce infection prevalence, but if infection is established then apoptosis induced by this method has limited ability to continue to suppress replication.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.16.476490', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Outer membrane protein A (OmpA) deficient Salmonella Typhimurium displays enhanced susceptibility towards β-lactam antibiotics: third-generation cephalosporins (ceftazidime) and carbapenems (meropenem)', 'authors': \"[{'name': 'Atish Roy Chowdhury'}, {'name': 'Debapriya Mukherjee'}, {'name': 'Ashish Kumar Singh'}, {'name': 'Dipshikha Chakravortty'}]\", 'abstract': '<p>The invasive non-typhoidal serovar of Salmonella enterica, namely Salmonella Typhimurium ST313, causes bloodstream infection in sub-Saharan Africa. Like other bacterial pathogens, the development of antimicrobial resistance is a severe problem in curing non-typhoidal Salmonella infection. In this work, we have investigated the role of four prominent outer membrane porins of S. Typhimurium, namely OmpA, OmpC, OmpD, and OmpF, in resistance against broad-spectrum β-lactam antibiotics- ceftazidime and meropenem. We found that deleting OmpA from Salmonella makes the bacteria susceptible to β-lactam drugs. The MIC for both the antibiotics reduced significantly for STM ΔompA compared to the wild-type and the ompA complemented strains. Despite the presence of antibiotics, the uninterrupted growth of STM ΔompC, ΔompD, and ΔompF endorsed the dispensability of these three porins in antimicrobial resistance. The β-lactam antibiotics caused massive depolarization in the outer membrane of the bacteria in the absence of OmpA. We have proved that none of the extracellular loops but the complete structure of perfectly folded OmpA is required by the bacteria for developing antimicrobial resistance. Our data revealed that STM ΔompA consumed more antibiotics than the wild-type and the complemented strain, resulting in severe damage of the bacterial outer membrane and subsequent killing of the pathogen by antibiotic-mediated oxidative stress. Upon deleting ompA, the steady decrease in the relative proportion of antibiotic-resistant persisters and the clearance of the STM ΔompA from the liver and spleen of C57BL/6 mice upon treatment with ceftazidime proved the role of OmpA in rendering protection against β-lactam antibiotics.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.476289', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Asymmetric Pendrin Homodimer Reveals its Molecular Mechanism as Anion Exchanger', 'authors': \"[{'name': 'Qianying Liu'}, {'name': 'Xiang Zhang'}, {'name': 'Hui Huang'}, {'name': 'Yuxin Chen'}, {'name': 'Fang Wang'}, {'name': 'Aihua Hao'}, {'name': 'Wuqiang Zhan'}, {'name': 'Qiyu Mao'}, {'name': 'Yuxia Hu'}, {'name': 'Lin Han'}, {'name': 'Yifang Sun'}, {'name': 'Meng Zhang'}, {'name': 'Zhimin Liu'}, {'name': 'Genglin Li'}, {'name': 'Weijia Zhang'}, {'name': 'Yilai Shu'}, {'name': 'Lei Sun'}, {'name': 'Zhenguo Chen'}]\", 'abstract': '<p>Pendrin SLC26A4 is an anion exchanger expressed in apical membranes of selected epithelia. Pendrin ablation causes Pendred syndrome, a genetic disorder disease associated with sensorineural hearing loss, hypothyroid goiter, and reduced blood pressure. However, its molecular structure has remained unknown limiting our understanding. Here, we determined the structures of mouse pendrin with symmetric and characteristically asymmetric homodimer conformations by cryo-electron microscopy. The asymmetric homodimer consists of an inward-facing protomer and an intermediate-state protomer, representing the coincident uptake and secretion process, and exhibits the unique state of pendrin as an electroneutral exchanger. This previously unrevealed conformation, together with other conformations we captured, provides an inverted alternate-access mechanism for anion exchange. Furthermore, our structural and functional data disclosed the properties of anion exchange cleft and interpreted the important pathogenetic mutations. These investigations shed light on the pendrin exchange mechanism and extend our structure-guided understanding of pathogenetic mutations.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.476390', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Integrative analysis of relative abundance data and presence-absence data of the microbiome using the LDM', 'authors': \"[{'name': 'Zhengyi Zhu'}, {'name': 'Glen A. Satten'}, {'name': 'Yi-Juan Hu'}]\", 'abstract': '<sec><title>Summary</title><p>We previously developed LDM for testing hypotheses about the microbiome that performs the test at both the community level and the individual taxon level. LDM can be applied to relative abundance data and presence-absence data separately, which work well when associated taxa are abundant and rare, respectively. Here we propose an omnibus test based on LDM that allows simultaneous consideration of data at different scales, thus offering optimal power across scenarios with different association mechanisms. The omnibus test is available for the wide range of data types and analyses that are supported by LDM.</p></sec><sec><title>Availability and Implementation</title><p>The omnibus test has been added to the R package LDM, which is available on GitHub at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/yijuanhu/LDM\">https://github.com/yijuanhu/LDM</ext-link>.</p></sec><sec><title>Contact</title><p><email>yijuan.hu@emory.edu</email></p></sec><sec><title>Supplementary information</title><p>Supplementary data are available at Bioinformatics online.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.475850', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'coveR: An R package for processing Digital Cover Photography images to retrieve forest canopy attributes', 'authors': \"[{'name': 'Francesco Chianucci'}, {'name': 'Carlotta Ferrara'}, {'name': 'Nicola Puletti'}]\", 'abstract': '<p>Digital Cover Photography (DCP) is an increasingly popular tool for estimating canopy cover and leaf area index (LAI). However, existing solutions to process canopy images are predominantly tailored for fisheye photography, whereas open-access tools for DCP are lacking. We developed an R package (coveR) to support the whole processing of DCP images in an automated, fast and reproducible way. The package functions, which are designed for step-by-step single-image analysis, can be performed  sequentially in a pipeline and also allow simple implementation of batch-processing bunches of images. A case study is presented to demonstrate the reliability of canopy attributes derived from coveR in pure beech (Fagus sylvatica L.) stands with variable canopy density and structure. Estimates of gap fraction and effective LAI from DCP were validated against reference measurements obtained from terrestrial laser scanning. By providing a simple, transparent and flexible image processing procedure, coveR supported the use of DCP for routine measurements and monitoring of forest canopy attributes.  This, combined with the implementability of DCP in many devices, including smartphones, micro-cameras, and remote trail cameras, can greatly expand the accessibility of the method also to non-experts.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.12.476128', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Harmonized-Multinational qEEG Norms (HarMNqEEG)', 'authors': \"[{'name': 'Min Li'}, {'name': 'Ying Wang'}, {'name': 'Carlos Lopez-Naranjo'}, {'name': 'Aini Ismafairus Abd Hamid'}, {'name': 'Alan C. Evans'}, {'name': 'Alexander N. Savostyanov'}, {'name': 'Ana Calzada-Reyes'}, {'name': 'Ariosky Areces-Gonzalez'}, {'name': 'Arno Villringer'}, {'name': 'Carlos A. Tobon-Quintero'}, {'name': 'Daysi Garcia-Agustin'}, {'name': 'Deirel Paz-Linares'}, {'name': 'Dezhong Yao'}, {'name': 'Li Dong'}, {'name': 'Eduardo Aubert-Vazquez'}, {'name': 'Faruque Reza'}, {'name': 'Hazim Omar'}, {'name': 'Jafri Malin Abdullah'}, {'name': 'Janina R. Galler'}, {'name': 'John F. Ochoa-Gomez'}, {'name': 'Leslie S. Prichep'}, {'name': 'Lidice Galan-Garcia'}, {'name': 'Lilia Morales-Chacon'}, {'name': 'Mitchell J. Valdes-Sosa'}, {'name': 'Marius Tröndle'}, {'name': 'Mohd Faizal Bin Mohd Zulkifly'}, {'name': 'Muhammad Riddha Bin Abdul Rahman'}, {'name': 'Natalya S. Milakhina'}, {'name': 'Nicolas Langer'}, {'name': 'Pavel Rudych'}, {'name': 'Shiang Hu'}, {'name': 'Thomas Koenig'}, {'name': 'Trinidad A. Virues-Alba'}, {'name': 'Xu Lei'}, {'name': 'Maria L. Bringas-Vega'}, {'name': 'Jorge F. Bosch-Bayard'}, {'name': 'Pedro Antonio Valdes-Sosa'}]\", 'abstract': '<p>This paper extends our frequency domain quantitative electroencephalography (qEEG) methods pursuing higher sensitivity to detect Brain Developmental Disorders. Prior qEEG work lacked integration of cross-spectral information omitting important functional connectivity descriptors. Lack of geographical diversity precluded accounting for site-specific variance, increasing qEEG nuisance variance. We ameliorate these weaknesses. i) Create lifespan Hermitian Riemannian multinational qEEG norms for cross-spectral tensors. These norms result from the HarMNqEEG project fostered by the Global Brain Consortium. We calculate the norms with data from 9 countries, 12 devices, and 14 studies, including 1564 subjects. Instead of raw data, only anonymized metadata and EEG cross-spectral tensors were shared. After visual and automatic quality control developmental equations for the mean and standard deviation of qEEG traditional and Hermitian Riemannian descriptive parameters were calculated using additive mixed-effects models. We demonstrate qEEG “batch effects” and provide methods to calculate harmonized z-scores. ii) We also show that the multinational harmonized Hermitian Riemannian norms produce z-scores with increased diagnostic accuracy to predict brain dysfunction at school-age produced by malnutrition only in the first year of life. We provide data and software for constructing norms. iii) We offer open code and data to calculate different individual z-scores from the HarMNqEEG dataset. These results contribute to developing bias-free, low-cost neuroimaging technologies applicable in various health settings.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>We create lifespan Hermitian Riemannian qEEG norms for cross-spectral tensors.</p></list-item><list-item><p>The norms are based on 9 countries, 12 devices, and 14 studies, with 1564 subjects.</p></list-item><list-item><p>We demonstrate qEEG “batch effects”, providing harmonization methods to remove them.</p></list-item><list-item><p>Multinational harmonized z-scores increase diagnostic accuracy of brain dysfunction.</p></list-item><list-item><p>Data and software are available for norm and individual z-scores calculation.</p></list-item></list></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.15.476444', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Broad-spectrum mono- and combinational drug therapies for global viral pandemic preparedness', 'authors': \"[{'name': 'Aleksandr Ianevski'}, {'name': 'Rouan Yao'}, {'name': 'Ronja Meyer Simonsen'}, {'name': 'Vegard Myhre'}, {'name': 'Erlend Ravlo'}, {'name': 'Gerda Dominyka Kaynova'}, {'name': 'Eva Zusinaite'}, {'name': 'Judith M White'}, {'name': 'Qiuwei Pan'}, {'name': 'Stephen J Polyak'}, {'name': 'Valentyn Oksenych'}, {'name': 'Marc P. Windisch'}, {'name': 'Egle Lastauskiene'}, {'name': 'Astra Vitkauskiene'}, {'name': 'Algimantas Matukevicius'}, {'name': 'Tanel Tenson'}, {'name': 'Magnar Bjoras'}, {'name': 'Denis E Kainov'}]\", 'abstract': '<p>Broadly effective antiviral therapies must be developed to be ready for clinical trials, which should begin soon after the emergence of new life-threatening viruses. Here, we pave the way towards this goal by analyzing conserved druggable virus-host interactions, mechanisms of action and immunomodulatory properties of broad-spectrum antivirals (BSAs), routes of BSA delivery, and BSA interactions with other antivirals. Based on the analysis we developed scoring systems, which allowed us to predict novel BSAs and BSA-containing drug combinations (BCCs). Thus, we have developed a new strategy to broaden the spectrum of BSA indications and predict novel mono- and combinational therapies that can help better prepare for imminent future viral outbreaks.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.476136', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'A preliminary description of the ecological characteristics of wild waterbird Japanese encephalitis virus hosts in high risk landscapes in India', 'authors': \"[{'name': 'Michael Walsh'}, {'name': 'Amrita Pattanaik'}, {'name': 'Navya Vyas'}, {'name': 'Deepak Saxena'}, {'name': 'Cameron Webb'}, {'name': 'Shailendra Sawleshwarkar'}, {'name': 'Chiranjay Mukhopadhyay'}]\", 'abstract': '<p>Wild reservoirs of Japanese encephalitis virus are under-studied globally, which presents critical knowledge gaps for JEV infection ecology despite decades of received wisdom regarding this high-impact mosquito-borne virus. As a result, ardeid birds, generally understood to be the primary reservoirs for JEV, as well as other waterbirds occupying landscapes at high risk for spillover to humans, are frequently ignored by current surveillance mechanisms and infrastructure. This is particularly true in India, which experiences a high annual burden of human outbreaks. Incorporating wild reservoirs into surveillance of human and livestock populations is therefore essential but will first require a data-driven approach to target individual host species. The current study sought to define a preliminary ecological profile of JEV hosts based on 1) species ecological traits, and 2) species presence and abundance adjusted for the biotic constraints of sympatry. Optimal host species tended to be generalists and demonstrate regionally-increasing populations. While ardeid bird species richness, abundance, and relative abundance did demonstrate the strongest and most consistent associations with the distribution of human JEV outbreaks, this study also identified several individual species among two other bird families in these landscapes, the Anatidae and the Rallidae, which also exhibited an optimal host profile and were strongly associated with the distribution of outbreaks. The findings from this work provide the first data-driven evidence base to inform wildlife sampling for the monitoring of JEV circulation in outbreak hotspots in India and thus identify good preliminary targets for the development of One Health wildlife JEV surveillance.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.476223', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'ATP6V1B2 and IFI27 and their intrinsic functional genomic characteristics associated with SARS-CoV-2', 'authors': \"[{'name': 'Zhengjun Zhang'}]\", 'abstract': '<p>Genes functionally associated with SARS-CoV-2 and genes functionally related to COVID-19 disease can be different, whose distinction will become the first essential step for successfully fighting against the COVID-19 pandemic. Unfortunately, this first step has not been completed in all biological and medical research. Using a newly developed max-competing logistic classifier, two genes, ATP6V1B2 and IFI27, stand out to be critical in transcriptional response to SARS-CoV-2 with differential expressions derived from NP/OP swab PCR. This finding is evidenced by combining these two genes with one another gene in predicting disease status to achieve better-indicating power than existing classifiers with the same number of genes. In addition, combining these two genes with three other genes to form a five-gene classifier outperforms existing classifiers with ten or more genes. With their exceptional predicting power, these two genes can be critical in fighting against the COVID-19 pandemic as a new focus and direction. Comparing the functional effects of these genes with a  five-gene classifier with 100% accuracy identified and tested from blood samples in the literature, genes and their transcriptional response and functional effects to SARS-CoV-2 and genes and their functional signature patterns to COVID-19 antibody are significantly different, which can be interpreted as the former is the point of a phenomenon, and the latter is the essence of the disease. Such significant findings can help explore the causal and pathological clue between SARS-CoV-2 and COVID-19 disease and fight against the disease with more targeted vaccines, antiviral drugs, and therapies.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.10.475627', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Head Removal Enhances Planarian Electrotaxis', 'authors': \"[{'name': 'Ziad Sabry'}, {'name': 'Rui Wang'}, {'name': 'Aryo Jahromi'}, {'name': 'Christina Rabeler'}, {'name': 'William B. Kristan'}, {'name': 'Eva-Maria S. Collins'}]\", 'abstract': '<p>Sensing electric fields is an ability that certain animal species utilize for communication, hunting, and spatial orientation. Freshwater planarians move toward the cathode in a static electric field (cathodic electrotaxis). First described by Raymond Pearl more than a century ago, planarian electrotaxis has received little attention and the underlying mechanisms and evolutionary significance remain unknown. We developed an apparatus and scoring metrics for automated quantitative and mechanistic studies of planarian behavior upon exposure to a static electric field. Using this automated setup, we characterized electrotaxis in the planarian <italic>Dugesia japonica</italic> and found that this species responds to voltage instead of to current, in contrast to results from previous studies using other species. Because longer planarians exhibited more robust electrotaxis than shorter planarians, we hypothesized that signals from the head impede cathodic electrotaxis. To test this hypothesis, we took advantage of the regenerative abilities of planarians and compared electrotaxis in head and tail fragments of various lengths. We found that tail and trunk fragments electrotaxed while head fragments did not, regardless of size. However, we could restore cathodic electrotaxis in head fragments via decapitation, demonstrating that the presence of the head impaired cathodic electrotaxis. This result is in stark contrast to other stimulated behaviors such as phototaxis, thermotaxis or chemotaxis, which are weaker or absent in headless fragments. Thus, electrotaxis may be an important ability of headless planarian fragments to support survival prior to head regeneration.</p><sec><title>Summary statement</title><p>We present a new method for quantitative studies of planarian electrotaxis and show that <italic>Dugesia japonica</italic> move toward the cathode. This behavior is enhanced by removal of the head.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.17.476578', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Meta-analysis of the black soldier fly (Hermetia illucens) microbiota based on 16S rRNA gene amplicon sequencing', 'authors': \"[{'name': 'Freek Ijdema'}, {'name': 'Jeroen De Smet'}, {'name': 'Sam Crauwels'}, {'name': 'Bart Lievens'}, {'name': 'Leen Van Campenhout'}]\", 'abstract': '<p>Black soldier fly larvae (BSFL) belong to the most widely reared insect species as an alternative protein source at industrial scale. Bacteria in the larval gut can provide benefits for the animal, though some bacteria can also be pathogenic for the insect. Accurate characterization of the BSFL microbiota is important for the production of BSFL in terms of yield and microbiological safety. In this study, 16S ribosomal RNA gene sequence data sets from 11 studies were re-analysed to gain better insights in the BSFL gut microbiota, potential factors that influence their composition, and differences between the gut and the whole larvae microbiota. A core gut microbiota was found consisting of members of Enterococcus, Klebsiella, Morganella, Providencia, and Scrofimicrobium. Further, the factors Study, Age and Feed significantly affected the microbiota gut composition. When compared to whole larvae, a significantly lower number of observed zero-radius Operational Taxonomic Units and a lower diversity was found for gut samples, suggesting that the larvae harboured additional microbes on their cuticle or in the insect body. Universal choices in insect sample type, primer selection and bio-informatics can strengthen future meta-analyses and improve our understanding of the BSFL gut microbiota towards the optimization of insect production.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.16.476524', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'A chromosome-scale genome assembly of the false clownfish, Amphiprion ocellaris', 'authors': \"[{'name': 'Taewoo Ryu'}, {'name': 'Marcela Herrera'}, {'name': 'Billy Moore'}, {'name': 'Michael Izumiyama'}, {'name': 'Erina Kawai'}, {'name': 'Vincent Laudet'}, {'name': 'Timothy Ravasi'}]\", 'abstract': '<p>Background: The false clownfish Amphiprion ocellaris is a popular fish species and an emerging model organism for studying the ecology, evolution, adaptation, and developmental biology of reef fishes. Despite this, high-quality genomic resources for this species are scarce, hindering advanced genomic analyses. Leveraging the power of PacBio long-read sequencing and Hi-C chromosome conformation capture techniques, we constructed a high-quality chromosome-scale genome assembly for the clownfish A. ocellaris. Results: The initial genome assembly comprised of 1,551 contigs of 861.42 Mb, with an N50 of 863.85 kb. Hi-C scaffolding of the genome resulted in 24 chromosomes containing 856.61 Mb. The genome was annotated with 26,797 protein-coding genes and had 96.62 % completeness of conserved actinopterygian genes, making this genome the most complete and high quality among published anemonefish genomes. Transcriptomic analysis identified tissue-specific gene expression patterns, with the brain and optic lobe having the largest number of expressed genes. Further, comparative genomic analysis revealed 91 genome elements conserved only in A. ocellaris and its sister species Amphiprion percula, and not in other anemonefish species. These elements are close to genes that are involved in various nervous system functions and exhibited distinct expression patterns in brain tissue, potentially highlighting the genetic toolkits involved in lineage-specific divergence and behaviors of the clownfish branch. Conclusions: Overall, our study provides the highest quality A. ocellaris genome assembly and annotation to date, whilst also providing a valuable resource for understanding the ecology and evolution of reef fishes.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.12.476060', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Approximated Gene Expression Trajectories (AGETs) for Gene Regulatory Network Inference on Cell Tracks', 'authors': \"[{'name': 'Kay Spiess'}, {'name': 'Timothy Fulton'}, {'name': 'Seogwon Hwang'}, {'name': 'Kane Toh'}, {'name': 'Dillan Saunders'}, {'name': 'Brooks Paige'}, {'name': 'Benjamin Steventon'}, {'name': 'Berta Verd'}]\", 'abstract': '<p>The study of pattern formation has benefited from reverse-engineering gene regulatory network (GRN) structure from spatio-temporal quantitative gene expression data. Traditional approaches omit tissue morphogenesis, hence focusing on systems where the timescales of pattern formation and morphogenesis can be separated. In such systems, pattern forms as an emergent property of the underlying GRN. This is not the case in many animal patterning systems, where patterning and morphogenesis are simultaneous. To address pattern formation in these systems we need to adapt our methodologies to explicitly accommodate cell movements and tissue shape changes. In this work we present a novel framework to reverse-engineer GRNs underlying pattern formation in tissues experiencing morphogenetic changes and cell rearrangements. By combination of quantitative data from live and fixed embryos we approximate gene expression trajectories (AGETs) in single cells and use a subset to reverse-engineer candidate GRNs using a Markov Chain Monte Carlo approach. GRN fit is assessed by simulating on cell tracks (live-modelling) and comparing the output to quantitative data-sets. This framework outputs candidate GRNs that recapitulate pattern formation at the level of the tissue and the single cell. To our knowledge, this inference methodology is the first to integrate cell movements and gene expression data, making it possible to reverse-engineer GRNs patterning tissues undergoing morphogenetic changes.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475832', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Divergence and introgression among the virilis group of Drosophila', 'authors': \"[{'name': 'Leeban H. Yusuf'}, {'name': 'Venera Tyukmaeva'}, {'name': 'Anneli Hoikkala'}, {'name': 'Michael G. Ritchie'}]\", 'abstract': '<p>Speciation with gene flow is now widely regarded as common. However, the frequency of introgression between recently diverged species and the evolutionary consequences of gene flow are still poorly understood. The <italic>virilis</italic> group of <italic>Drosophila</italic> contains around a dozen species that are geographically widespread and show varying levels of pre-zygotic and post-zygotic isolation. Here, we utilize de novo genome assemblies and whole-genome sequencing data to resolve phylogenetic relationships and describe patterns of introgression and divergence across the group. We suggest that the <italic>virilis</italic> group consists of three, rather than the traditional two, subgroups. We found evidence of pervasive phylogenetic discordance caused by ancient introgression events between distant lineages within the group, and much more recent gene flow between closely-related species. When assessing patterns of genome-wide divergence in species pairs across the group, we found no consistent genomic evidence of a disproportionate role for the X chromosome. Some genes undergoing rapid sequence divergence across the group were involved in chemical communication and may be related to the evolution of sexual isolation. We suggest that gene flow between closely-related species has potentially had an impact on lineage-specific adaptation and the evolution of reproductive barriers. Our results show how ancient and recent introgression confuse phylogenetic reconstruction, and suggest that shared variation can facilitate adaptation and speciation.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.22269108', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Rehabilitation Medicine and Physical Therapy', 'covid': False, 'title': 'Novel Application of a Force Sensor during Sit-to-Stands to Measure Dynamic Cerebral Autoregulation Onset', 'authors': \"[{'name': 'Alicen A Whitaker'}, {'name': 'Eric D. Vidoni'}, {'name': 'Stacey E. Aaron'}, {'name': 'Adam G. Rouse'}, {'name': 'Sandra A Billinger'}]\", 'abstract': '<p>Purpose: Current sit-to-stand methods measuring dynamic cerebral autoregulation (dCA) do not capture the precise onset of the time delay (TD) response. Reduced sit-to-stand reactions in older adults and individuals post-stroke could inadvertently introduce variability, error, and imprecise timing. We applied a force sensor during a sit-to-stand task to more accurately determine how TD before dCA onset may be altered. \\nMethods: Middle cerebral artery blood velocity (MCAv) and mean arterial pressure (MAP) were measured during two sit-to-stands separated by 15 minutes. Recordings started with participants sitting on a force-sensitive resistor for 60 seconds, then asked to stand for two minutes. Upon standing, the force sensor voltage immediately dropped and marked the exact moment of arise-and-off (AO). Time from AO until an increase in cerebrovascular conductance (CVC = MCAv/MAP) was calculated as TD. \\nResults: We tested the sensor in 4 healthy young adults, 2 older adults, and 2 individuals post-stroke. Healthy young adults stood quickly and the force sensor detected a small change in TD  compared to classically estimated AO, from verbal command to stand. When compared to the estimated AO, older adults had a delayed measured AO and TD decreased  up to ~50% while individuals post-stroke had an early AO and TD increased up to ~14%.\\nConclusion: The transition reaction speed during the sit to stand has the potential to influence dCA metrics. As observed in the older adults and participants with stroke, this response may drastically vary and influence TD.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.16.476488', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'ISSAAC-seq enables sensitive and flexible multimodal profiling of chromatin accessibility and gene expression in single cells', 'authors': \"[{'name': 'Wei Xu'}, {'name': 'Weilong Yang'}, {'name': 'Yunlong Zhang'}, {'name': 'Yawen Chen'}, {'name': 'Qian Zhang'}, {'name': 'Xuefei Wang'}, {'name': 'Kun Song'}, {'name': 'Wenfei Jin'}, {'name': 'Xi Chen'}]\", 'abstract': '<p>Joint profiling of chromatin accessibility and gene expression from the same single cell/nucleus provides critical information about cell types in a tissue and cell states during a dynamic process. These emerging multi-omics techniques help the investigation of cell-type resolved gene regulatory mechanisms. However, many methods are currently limited by low sensitivity, low throughput or complex workflow. Here, we developed in situ SHERRY after ATAC-seq (ISSAAC-seq), a highly sensitive and flexible single cell multi-omics method to interrogate chromatin accessibility and gene expression from the same single nucleus. We demonstrated that ISSAAC-seq is sensitive and provides high quality data with orders of magnitude more features than existing methods. Using the joint profiles from over 10,000 nuclei from the mouse cerebral cortex, we uncovered major and rare cell types and cell-type specific regulatory elements and identified heterogeneity at the chromatin level within established cell types defined by gene expression. Finally, we revealed distinct dynamics and relationships of gene expression and chromatin accessibility during an oligodendrocyte maturation trajectory.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.16.476501', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Escherichia coli response to subinhibitory concentration of Colistin: Insights from study of membrane dynamics and morphology.', 'authors': \"[{'name': 'Ilanila Ilangumaran Ponmalar'}, {'name': 'Jitendriya Swain'}, {'name': 'Jaydeep Kumar Basu'}]\", 'abstract': '<p>Prevalence of wide spread bacterial infections bring forth a critical need in understanding the molecular mechanisms of the antibiotics as well as the bacterial response to those antibiotics. Improper usage of antibiotics, which can be in sub-lethal concentrations is one among the multiple reasons for acquiring antibiotic resistance which makes it vital to understand the bacterial response towards sub-lethal concentrations of antibiotics. In this work, we have used colistin, a well-known membrane active antibiotic used to treat severe bacterial infections and explored the impact of its subminimum inhibitory concentration (MIC) on the lipid membrane dynamics and morphological changes of E. coli. Upon investigation of live cell membrane properties such as lipid dynamics using fluorescence correlation spectroscopy, we observed that colistin disrupts the lipid membrane at sub-MIC by altering the lipid diffusivity. Interestingly, filamentationlike cell elongation was observed upon colistin treatment which led to further exploration of surface morphology with the help of atomic force spectroscopy. The changes in the surface roughness upon colistin treatment provides additional insight on the colistin-membrane interaction corroborating with the altered lipid diffusion. Although altered lipid dynamics could be attributed to an outcome of lipid rearrangement due to direct disruption by antibiotic molecules on the membrane or an indirect consequence of disruptions in lipid biosynthetic pathways, we were able to ascertain that altered bacterial membrane dynamics is due to direct disruptions. Our results provide a broad overview on the consequence of the cyclic polypeptide, colistin on membrane specific lipid dynamics and morphology of a live Gram-negative bacterial cell.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.13.476214', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'The assembled Banana dihaploid mitochondrial genome is compact with a high number of gene copies', 'authors': \"[{'name': 'Shruthy Priya Prakash'}, {'name': 'Vaidheki Chandrasekar'}, {'name': 'Selvi Subramanian'}, {'name': 'Rahamatthunnisha Ummar'}]\", 'abstract': '<p>Banana being a major food crop all around the world, attracts various research interests in crop improvement. In banana, complete genome sequences of <italic>Musa accuminata</italic> and <italic>Musa balbisiana</italic> are available. However, the mitochondrial genome is not sequenced or assembled. Mitochondrial (mt) genes play an important role in flower and seed development and in Cytoplasmic Male Sterility. Unraveling banana mt genome architecture will be a foundation for understanding inheritance of traits and their evolution. In this study, the complete banana mt genome is assembled from the whole genome sequence data of the <italic>Musa acuminata subsp. malaccensis</italic> DH-Pahang. The mt genome sequence acquired by this approach was 409574 bp and it contains, 54 genes coding for 25 respiratory complex proteins 15 ribosomal proteins, 12 tRNA genes and two ribosomal RNA gene. Except atpB, rps11 and rps19 other genes are in multiple copies. The copy number is 12 in tRNA genes. In addition, nearly 25% tandem repeats are also present in it. These mt proteins are identical to the mt proteins present in the other members of AA genome and share 98% sequence similarity with <italic>M. balbisiana.</italic> The C to U RNA editing is profoundly higher (87 vs 13%) in transcripts of <italic>M. balbisiana</italic> (BB) compared to <italic>M. accuminata</italic> (AA). The banana AA mitochondrial genome is tightly packed with 233 genes, with less rearrangements and just 5.3% chloroplast DNA in it. The maintenance of high copy number of functional mt genes suggest that they have a crucial role in the evolution of banana.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.10.475759', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Multi-cancer classification; an analysis of neural network complexity', 'authors': \"[{'name': 'James W. Webber'}, {'name': 'Kevin Elias'}]\", 'abstract': '<sec><title>Background</title><p>Cancer identification is generally framed as binary classification, normally discrimination of a control group from a single cancer group. However, such models lack any cancer-specific information, as they are only trained on one cancer type. The models fail to account for competing cancer risks. For example, an ostensibly healthy individual may have any number of different cancer types, and a tumor may originate from one of several primary sites. Pan-cancer evaluation requires a model trained on multiple cancer types, and controls, simultaneously, so that a physician can be directed to the correct area of the body for further testing.</p></sec><sec><title>Methods</title><p>We introduce novel neural network models to address multi-cancer classification problems across several data types commonly applied in cancer prediction, including circulating miRNA expression, protein, and mRNA. In particular, we present an analysis of neural network depth and complexity, and investigate how this relates to classification performance. Comparisons of our models with state-of-the-art neural networks from the literature are also presented.</p></sec><sec><title>Results</title><p>Our analysis evidences that shallow, feed-forward neural net architectures offer greater performance when compared to more complex deep feed-forward, Convolutional Neural Network (CNN), and Graph CNN (GCNN) architectures considered in the literature.</p></sec><sec><title>Conclusion</title><p>The results show that multiple cancers and controls can be classified accurately using the proposed models, across a range of expression technologies in cancer prediction.</p></sec><sec><title>Impact</title><p>This study addresses the important problem of pan-cancer classification, which is often overlooked in the literature. The promising results highlight the urgency for further research.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.14.476154', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Plasma cell maintenance and antibody secretion are under the control of Sec22b-mediated regulation of organelle dynamics', 'authors': \"[{'name': 'Amelie Bonaud'}, {'name': 'Laetitia Gargowitsch'}, {'name': 'Simon Gilbert'}, {'name': 'Elanchezhian Rajan'}, {'name': 'Pablo Canales-Herrerias'}, {'name': 'Daniel Stockholm'}, {'name': 'Nabila Rahman'}, {'name': 'Mark Collins'}, {'name': 'Danika L Hill'}, {'name': 'Andres Alloatti'}, {'name': 'Nagham Alouche'}, {'name': 'Stephanie Balor'}, {'name': 'Vanessa Soldan'}, {'name': 'Daniel Gillet'}, {'name': 'Julien Barbier'}, {'name': 'Francoise Bachelerie'}, {'name': 'Kenneth G. C. Smith'}, {'name': 'Pierre Bruhns'}, {'name': 'Sebastian Amigorena'}, {'name': 'Karl Balabanian'}, {'name': 'Michelle A Linterman'}, {'name': 'Andrew Peden'}, {'name': 'Marion Espeli'}]\", 'abstract': '<p>Despite the essential role of plasma cells in health and disease, the cellular mechanisms controlling their survival and secretory capacity are still poorly understood. Here, we identified the SNARE Sec22b as a unique and critical regulator of plasma cell maintenance and function. In absence of Sec22b, plasma cells were barely detectable and serum antibody titres were dramatically reduced. Accordingly, Sec22b deficient mice fail to mount a protective immune response. At the mechanistic level, we demonstrated that Sec22b is indispensable for efficient antibody secretion but also for plasma cell fitness through the regulation of the morphology of the endoplasmic reticulum and mitochondria. Altogether, our results unveil a critical role for Sec22b-mediated regulation of plasma cell biology through the control of organelle dynamics.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.10.475717', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Astrogliosis mapping in individual brains using multidimensional MRI', 'authors': \"[{'name': 'Dan Benjamini'}, {'name': 'David S Priemer'}, {'name': 'Daniel P Perl'}, {'name': 'David L Brody'}, {'name': 'Peter J Basser'}]\", 'abstract': '<p>There are currently no noninvasive imaging methods available for astrogliosis mapping in the central nervous system despite its essential role in the response to injury, disease, and infection. We have developed a machine learning-based multidimensional MRI framework that provides a signature of astrogliosis, distinguishing it from normative brain at the individual level. We investigated <italic>ex vivo</italic> cortical tissue specimen derived from subjects who sustained blast induced injuries, which resulted in scar-border forming astrogliosis without being accompanied by other types of neuropathology. By performing a combined postmortem radiology and histopathology correlation study we found that astrogliosis induces microstructural changes that are robustly detected using our framework, resulting in MRI neuropathology maps that are significantly and strongly correlated with co-registered histological images of increased glial fibrillary a cidic protein deposition. The demonstrated high spatial sensitivity in detecting reactive astrocytes at the individual level has great potential to significantly impact neuroimaging studies in diseases, injury, repair, and aging.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.474522', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Ocimum sanctum, OscWRKY1, regulates phenylpropanoid pathway genes and promotes resistance to pathogen infection in Arabidopsis', 'authors': \"[{'name': 'Ashutosh Joshi'}, {'name': 'Gajendra Singh Jeena'}, {'name': ' Shikha'}, {'name': 'Ravi Kumar'}, {'name': 'Alok Pandey'}, {'name': 'Rakesh Kumar Shukla'}]\", 'abstract': '<p>WRKY transcription factor (TF) family regulates various developmental and physiological functions in plants. <italic>PAL</italic> genes encode enzymes which are involved in plant defense responses, but the direct regulation of <italic>PAL</italic> genes and phenylpropanoid pathway through WRKY TF’s is not well characterized. In the present study, we have characterized an <italic>OscWRKY1</italic> gene from <italic>O. sanctum</italic> which shows induced expression after methyl jasmonate (MeJA), salicylic acid (SA), and wounding. Recombinant OscWRKY1 protein binds to the W-box <italic>cis</italic>-element TTGAC[C/T] and activates the reporter gene in yeast. Overexpression of <italic>OscWRKY1</italic> enhances Arabidopsis resistance towards <italic>Pseudomonas syringae pv. tomato Pst</italic> DC3000. Upstream activator sequences of <italic>PAL</italic> and <italic>C4H</italic> have identified the conserved W-box <italic>cis</italic>-element (TTGACC) in both <italic>O. sanctum</italic> and Arabidopsis. OscWRKY1 was found to interact with W-box <italic>cis</italic>-element present in the <italic>PAL</italic> and <italic>C4H</italic> promoters. Silencing of <italic>OscWRKY1</italic> using VIGS resulted in reduced expression of <italic>PAL, C4H, COMT, F5H</italic> and <italic>4CL</italic> transcripts. <italic>OscWRKY1</italic> silenced plants exhibit reduced PAL activity, whereas, the overexpression lines of <italic>OscWRKY1</italic> in Arabidopsis exhibit increased PAL activity. These results revealed that <italic>OscWRKY1</italic> positively regulates the phenylpropanoid pathway genes and enhances the resistance against bacterial pathogen in Arabidopsis.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.14.476392', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Phase Separation of Shell Protein and Enzyme: An Insight into the Biogenesis of a Prokaryotic Metabolosome', 'authors': \"[{'name': 'Gaurav Kumar'}, {'name': 'Sharmistha Sinha'}]\", 'abstract': '<p>Bacterial microcompartments are substrate specific metabolic modules that are conditionally expressed in certain bacterial species. These all protein structures have size in the range of 100-150 nm and are formed by the self-assembly of thousands of protein subunits, all encoded by genes belonging to a single operon. The operon contains genes that encode for both enzymes and shell proteins. The shell proteins self-assemble to form the outer coat of the compartment and enzymes are encapsulated within. A perplexing question in MCP biology is to understand the mechanism which governs the formation of these small yet complex assemblages of proteins. In this work we use 1,2-propanediol utilization microcompartments (PduMCP) as a paradigm to identify the factors that drive the self-assembly of MCP proteins. We find that a major shell protein PduBB tend to self-assemble under macromolecular crowded environment and suitable ionic strength. Microscopic visualization and biophysical studies reveal phase separation to be the principle mechanism behind the self-association of shell protein in the presence of salts and macromolecular crowding. The shell protein PduBB interacts with the enzyme diol-dehydratase PduCDE and co-assemble into phase separated liquid droplets. The co-assembly of PduCDE and PduBB results in the enhancement of catalytic activity of the enzyme. A combination of spectroscopic and biochemical techniques shows the relevance of divalent cation Mg2+ in providing stability to intact PduMCP in vivo. Together our results suggest a combination of protein-protein interactions and phase separation guiding the self-assembly of Pdu shell protein and enzyme in solution phase.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.475824', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Functional resilience of mutually repressing motifs embedded in larger networks', 'authors': \"[{'name': 'Pradyumna Harlapur'}, {'name': 'Atchuta Srinivas Duddu'}, {'name': 'Kishore Hari'}, {'name': 'Mohit Kumar Jolly'}]\", 'abstract': '<p>Elucidating the design principles of regulatory networks driving cellular decision-making has important implications in understanding cell differentiation and guiding the design of synthetic circuits. Mutually repressing feedback loops between ‘master regulators’ of cell-fates can exhibit multistable dynamics, thus enabling multiple “single-positive” phenotypes: (high A, low B) and (low A, high B) for a toggle switch, and (high A, low B, low C), (low A, high B, low C) and (low A, low B, high C) for a toggle triad. However, the dynamics of these two network motifs has been interrogated in isolation <italic>in silico</italic>, but <italic>in vitro</italic> and <italic>in vivo</italic>, they often operate while embedded in larger regulatory networks. Here, we embed these network motifs in complex larger networks of varying sizes and connectivity and identify conditions under which these motifs maintain their canonical dynamical behavior, thus identifying hallmarks of their functional resilience. We show that the in-degree of a motif - defined as the number of incoming edges onto a motif - determines its functional properties. For a smaller in-degree, the functional traits for both these motifs (bimodality, pairwise correlation coefficient(s), and the frequency of “single-positive” phenotypes) are largely conserved, but increasing the in-degree can lead to a divergence from their stand-alone behaviors. These observations offer insights into design principles of biological networks containing these network motifs, as well as help devise optimal strategies for integration of these motifs into larger synthetic networks.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476429', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'EPSTEIN-BARR VIRUS LMP1 ENHANCES LEVELS OF MICROVESICLE-ASSOCIATED PD-L1', 'authors': \"[{'name': 'Monica Abou Harb'}, {'name': 'Li Sun'}, {'name': 'David G. Meckes'}]\", 'abstract': '<p>Extracellular vesicles (EVs) circulate throughout the body and carry cargo that can be conferred to proximal or distant cells, making them major delivery vehicles for cellular communication. Epstein-Barr virus (EBV) infected cells release EVs that contain viral proteins such as the major viral oncogene, latent membrane protein 1 (LMP1). LMP1 has been shown to regulate the cellular gene expression of programmed cell death protein 1 ligand (PD-L1). PD-L1, a protein that suppresses the immune system by binding to PD-1, (a receptor found on cytotoxic T cells). PD-L1 has been recently found to be packaged into small EVs contributing to immune evasion of lung cancer cells. Recent studies establish that MVs are shed in very large amounts by tumor cells, and that elevated levels of MVs correlate to disease metastasis and cancers being more aggressive. Here, we demonstrate PD-L1 enrichment in MVs released from nasopharyngeal carcinoma cells and an important function of EBV LMP1 in regulating PD-L1 levels in MVs. These PD-L1+ MVs containing LMP1 likely contribute to the immunosuppressive microenvironment found in EBV-associated cancers.</p><sec><title>Importance</title><p>Accumulating evidence over the past decade supports that viruses utilize EVs and associated pathways to incorporate viral products to evade eliciting an immune response, while concurrently enabling viral spread or persistence within the host. Considering that viral proteins confer very strong antigenic peptides that will be recognized by T cells, the regulation of the PD-1 pathway by the overexpression of MV-associated PD-L1 may be a strong immune evasion tactic utilized by viruses. The discovery that EBV LMP1 increases PD-L1 microvesicle secretion, identifies a new therapeutic target in immune blockade therapy. We expect that our findings will begin to clarify the mechanism of LMP1-mediated enhanced packaging of PD-L1 into MVs and may produce more specific targets to treat EBV-associated cancers. Consequently, identifying whether a disease is of viral origin through predictive MV biomarkers could further allow for more targeted therapies.</p></sec>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.476201', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'The role of LHCBM1 in non-photochemical quenching in Chlamydomonas reinhardtii', 'authors': \"[{'name': 'Xin Liu'}, {'name': 'Wojciech J Nawrocki'}, {'name': 'Roberta Croce'}]\", 'abstract': '<p>Non-photochemical quenching (NPQ) is the process that protects photosynthetic organisms from photodamage by dissipating the energy absorbed in excess as heat. In the model green alga Chlamydomonas reinhardtii, NPQ was abolished in the knock-out mutants of the pigment-protein complexes LHCSR3 and LHCBM1. However, while LHCSR3 was shown to be a pH sensor and switching to a quenched conformation at low pH, the role of LHCBM1 in NPQ has not been elucidated yet. In this work, we combine biochemical and physiological measurements to study short-term high light acclimation of npq5, the mutant lacking LHCBM1. We show that while in low light in the absence of this complex, the antenna size of PSII is smaller than in its presence, this effect is marginal in high light, implying that a reduction of the antenna is not responsible for the low NPQ. We also show that the mutant expresses LHCSR3 at the WT level in high light, indicating that the absence of this complex is also not the reason. Finally, NPQ remains low in the mutant even when the pH is artificially lowered to values that can switch LHCSR3 to the quenched conformation. It is concluded that both LHCSR3 and LHCBM1 need to be present for the induction of NPQ and that LHCBM1 is the interacting partner of LHCSR3. This interaction can either enhance the quenching capacity of LHCSR3 or connect this complex with the PSII supercomplex.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475910', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'HIV and FIV glycoproteins increase cellular tau pathology via cGMP-dependent kinase II activation', 'authors': \"[{'name': 'Matheus F. Sathler'}, {'name': 'Michael J. Doolittle'}, {'name': 'James A. Cockrell'}, {'name': 'India R. Nadalin'}, {'name': 'Franz Hofmann'}, {'name': 'Sue VandeWoude'}, {'name': 'Seonil Kim'}]\", 'abstract': '<p>As the development of combination antiretroviral therapy (cART) against human immunodeficiency virus (HIV) drastically improves the lifespan of individuals with HIV, many are now entering the prime age when Alzheimer’s disease (AD)-like symptoms begin to manifest. Hyperphosphorylated tau, a known AD pathological characteristic, has been prematurely increased in the brains of HIV-infected patients as early as in their 30s and is increased with age. This thus suggests that HIV infection may lead to accelerated AD phenotypes. However, whether HIV infection causes AD to develop more quickly in the brain is not yet fully determined. Interestingly, we have previously revealed that viral glycoproteins, HIV gp120 and feline immunodeficiency virus (FIV) gp95, induce neuronal hyperexcitation via cGMP-dependent kinase II (cGKII) activation in cultured hippocampal neurons. Here, we use cultured mouse cortical neurons to demonstrate that HIV gp120 and FIV gp95 are sufficient to increase cellular tau pathology, including intracellular tau hyperphosphorylation and tau release to the extracellular space. We further reveal that viral glycoprotein-induced cellular tau pathology requires cGKII activation. Together, HIV infection likely accelerates AD-related tau pathology via cGKII activation.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.22269106', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'COVID-19 policies in practice and their direct and indirect impacts in Northern California jails', 'authors': \"[{'name': 'Yiran E. Liu'}, {'name': 'Christopher LeBoa'}, {'name': 'Marcela Rodriguez'}, {'name': 'Beruk Sherif'}, {'name': 'Chrisele Trinidad'}, {'name': 'Michael del Rosario'}, {'name': 'Sophie Allen'}, {'name': 'Christine Clifford'}, {'name': 'Jennifer Redding'}, {'name': 'Wei-ting Chen'}, {'name': 'Lisa G. Rosas'}, {'name': 'Carlos Morales'}, {'name': 'Alexander Chyorny'}, {'name': 'Jason R. Andrews'}]\", 'abstract': '<sec><title>Background</title><p>Although the increased risk of COVID-19 in carceral facilities is well documented, little is known about the practical barriers to infection control and indirect impacts of pandemic policies in these settings. Evidence in jails is especially scarce.</p></sec><sec><title>Methods</title><p>Between July 8, 2020 and April 30, 2021 we performed SARS-CoV-2 serology testing and administered a questionnaire among residents and staff in four Northern California jails. We analyzed seroprevalence in conjunction with demographic factors and survey responses of self-perceived COVID-19 risk, recent illness, COVID-19 test results, and symptom reporting behaviors. We additionally assessed COVID-19 policies in practice and evaluated their impacts on court dates, mental health, and routine health care. We engaged stakeholder representatives, including incarcerated individuals and their advocates, to guide study design, conduct, and interpretation.</p></sec><sec><title>Results</title><p>We enrolled 788 incarcerated individuals and 380 staff across four county jails. Most seropositive individuals had not previously tested positive for COVID-19, despite many suspecting prior infection. Among incarcerated participants, we identified deficient access to face masks and prevalent symptom underreporting associated with fears of isolation and perceptions of medical neglect in jail. Incarcerated participants also reported substantial hindrances to court cases and reductions in routine health care due to COVID-19. Incarcerated individuals and staff both cited worsened mental health due to COVID-19, which for incarcerated individuals was largely attributable to further isolation from loved ones and other pandemic restrictions on recreation and programming.</p></sec><sec><title>Conclusion</title><p>Perceptions of inadequate protection from COVID-19 were pervasive among incarcerated individuals. Simultaneously, restrictive measures compounded poor mental health and fostered fears of isolation that undermined effective infection control. Custody officials should work to systematically improve provision of masks, understand and mitigate fears and mistrust, and take proactive steps to minimize the detrimental impacts of restrictive policies on residents’ mental health and well-being.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476371', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Mechanisms of A-type lamin targeting to nuclear ruptures are disrupted in LMNA- and BANF1-associated progerias', 'authors': \"[{'name': 'Rhiannon M. Sears'}, {'name': 'Kyle J. Roux'}]\", 'abstract': '<p>Mutations in the genes <italic>LMNA</italic> and <italic>BANF1</italic> can lead to accelerated aging syndromes called progeria. The protein products of these genes, A-type lamins and BAF, respectively, are nuclear envelope (NE) proteins that interact and participate in various cellular processes, including nuclear envelope rupture and repair. BAF localizes to sites of nuclear rupture and recruits NE-repair machinery, including the LEM-domain proteins, ESCRT-III complex, A-type lamins, and membranes. Here, we show that it is a mobile, nucleoplasmic population of A-type lamins that is rapidly recruited to ruptures in a BAF-dependent manner via BAF’s association with the Ig-like β fold domain of A-type lamins. These initially mobile lamins become progressively stabilized at the site of rupture. Farnesylated prelamin A and lamin B1 fail to localize to nuclear ruptures, unless that farnesylation is inhibited. Progeria-associated <italic>LMNA</italic> mutations inhibit the recruitment affected A-type lamin to nuclear ruptures, due to either permanent farnesylation or inhibition of BAF binding. A progeria-associated BAF mutant targets to nuclear ruptures but is unable to recruit A-type lamins. Together, these data reveal the mechanisms that determine how lamins respond to nuclear ruptures and how progeric mutations of <italic>LMNA</italic> and <italic>BANF1</italic> impair recruitment of A-type lamins to nuclear ruptures.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.15.476453', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Sugar shock: Probing Streptococcus pyogenes metabolism through bioluminescence imaging', 'authors': \"[{'name': 'Richard W Davis'}, {'name': 'Charlotte Muse'}, {'name': 'Heather Eggleston'}, {'name': 'Micaila Hill'}, {'name': 'Peter Panizzi'}]\", 'abstract': '<p>Streptococcus pyogenes (S. pyogenes) can thrive in its host during an infection, and, as a result, it must be able to respond to external stimuli and available carbon sources. The pre-clinical use of engineered pathogens capable of constitutive light production may provide real-time information on microbial-specific metabolic processes. Here we mapped the central metabolism of a luxABCDE-modified S. pyogenes Xen20 (Strep. Xen20) to its de novo synthesis of luciferase substrates as assessed by the rate of light production in response to different environmental triggers. Previous characterization predicted that the lux operon was under the myo-inositol iolE promotor. Here we show that supplementation with myo-inositol generated increased Xen20 luminescence. Surprisingly, when supplemented with infection-relevant carbon sources, such as glucose or glycine, light production was diminished. This was presumably due to the scavenging of pyruvate by L-lactate dehydrogenase (LDH). Inhibition of LDH by its inhibitor, oxamate, partially restored luminescent signal in the presence of glucose, presumably by allowing the resulting pyruvate to proceed to acetyl-coenzyme A (CoA). This phenomenon appeared specific to the lactic acid bacterial metabolism as glucose or glycine did not reduce signal in an analogous luxABCDE-modified Gram-positive pathogen, Staph. Xen29. The Strep. Xen20 cells produced light in a concentration-dependent manner, inversely related to the amount of glucose present. Taken together, our measures of microbial response could provide new information regarding the responsiveness of S. pyogenes metabolism to acute changes in its local environments and cellular health.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.15.475214', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Modulating CRISPR-Cas genome editing using guide-complementary DNA oligonucleotides', 'authors': \"[{'name': 'Thomas Swartjes'}, {'name': 'Peng Shang'}, {'name': 'Dennis van den Berg'}, {'name': 'Tim A. Kunne'}, {'name': 'Niels Geijsen'}, {'name': 'Stan J.J. Brouns'}, {'name': 'John van der Oost'}, {'name': 'Raymond Staals'}, {'name': 'Richard A. Notebaart'}]\", 'abstract': '<p>CRISPR-Cas has revolutionized genome editing and has a great potential for applications, such as correcting human genetic disorders. To increase the safety of genome editing applications, CRISPR-Cas may benefit from strict control over Cas enzyme activity. Previously, anti-CRISPR proteins and designed oligonucleotides have been proposed to modulate CRISPR-Cas activity. Here we report on the potential of guide-complementary DNA oligonucleotides as controlled inhibitors of Cas9 ribonucleoprotein complexes. First, we show that DNA oligonucleotides down-regulate Cas9 activity in human cells, reducing both on and off-target cleavage. We then used <italic>in vitro</italic> assays to better understand how inhibition is achieved and under which conditions. Two factors were found to be important for robust inhibition: the length of the complementary region, and the presence of a PAM-loop on the inhibitor. We conclude that DNA oligonucleotides can be used to effectively inhibit Cas9 activity both <italic>ex vivo</italic> and <italic>in vitro</italic>.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.11.475878', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Hydrogen production in an aerobic environment by Escherichia coli K-12', 'authors': \"[{'name': 'George D. Metcalfe'}, {'name': 'Frank Sargent'}, {'name': 'Michael Hippler'}]\", 'abstract': '<p><italic>Escherichia coli</italic> (<italic>E. coli</italic>) is a facultative anaerobe that can grow in a variety of environmental conditions. In the complete absence of O<sub>2</sub>, <italic>E. coli</italic> can perform a mixed-acid fermentation that contains within it an elaborate metabolism of formic acid. In this study, we use cavity-enhanced Raman spectroscopy (CERS), FTIR, liquid Raman spectroscopy, isotopic labelling, and molecular genetics to make advances in the understanding of bacterial formate and H<sub>2</sub> metabolism. It is shown that, under anaerobic conditions, formic acid is generated endogenously, excreted briefly from the cell, and then taken up again to be disproportionated to H<sub>2</sub> and CO<sub>2</sub> by formate hydrogenlyase (FHL-1). However, exogenously added D-labelled formate behaves quite differently from the endogenous formate and is taken up immediately, independently, and possibly by a different mechanism, by the cell and converted to H<sub>2</sub> and CO<sub>2</sub>. Our data support an anion-proton symport model for formic acid transport. In addition, when <italic>E. coli</italic> was grown in a microaerobic environment it was possible to analyse aspects of formate and O<sub>2</sub> respiration occurring alongside anaerobic metabolism. While cells growing under microaerobic conditions generated endogenous formic acid, no H<sub>2</sub> was produced. However, addition of exogenous formate at the outset of cell growth did induce FHL-1 biosynthesis and resulted in formate-dependent H<sub>2</sub> production in the presence of O<sub>2</sub>.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.14.476338', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'White Matter Brain Structure Predicts Language Performance and Learning Success', 'authors': \"[{'name': 'Stella M. Sánchez'}, {'name': 'Helmut Schmidt'}, {'name': 'Guillermo Gallardo'}, {'name': 'Alfred Anwander'}, {'name': 'Jens Brauer'}, {'name': 'Angela D. Friederici'}, {'name': 'Thomas R. Knösche'}]\", 'abstract': '<p>Individual differences in the ability to deal with language have long been discussed. The neural basis of these, however, is yet unknown. Here we investigated the relationship between long-range white matter connectivity of the brain, as revealed by diffusion tractography, and the ability to process syntactically complex sentences in the participants’ native language as well as the improvement thereof by multi-day training. We identified specific network motifs that indeed related white matter tractography to individual language processing performance. First, for two such motifs, one in the left and one in the right hemisphere, their individual prevalence significantly predicted the individual language performance suggesting a predisposition for the individual ability to process syntactically complex sentences, which manifests itself in the white matter brain structure. Both motifs comprise a number of cortical regions, but seem to be dominated by areas known for the involvement in working memory rather than the classical language network itself. Second, we identified another left hemispheric network motif, whose change of prevalence over the training period significantly correlated with the individual change in performance, thus reflecting training induced white matter plasticity. This motif comprises diverse cortical areas including regions known for their involvement in language processing, working memory and motor functions. The present findings suggest that individual differences in language processing and learning can be explained, in part, by individual differences in the brain’s white matter structure. Brain structure may be a crucial factor to be considered when discussing variations in human cognitive performance, more generally.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.475941', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Hemagglutinin stability determines influenza A virus susceptibility to a broad-spectrum fusion inhibitor Arbidol', 'authors': \"[{'name': 'Li Zhenyu'}, {'name': 'Li Tian'}, {'name': 'Liu Meisui'}, {'name': 'Ivanovic Tijana'}]\", 'abstract': '<p>Understanding mechanisms of resistance to antiviral inhibitors can reveal nuanced features of targeted viral mechanisms and, in turn, lead to improved strategies for inhibitor design. Arbidol is a broad-spectrum antiviral which binds to and prevents the fusion-associated conformational changes in the trimeric influenza hemagglutinin (HA). The rate-limiting step during HA-mediated membrane fusion is the release of the hydrophobic fusion peptides from a conserved pocket on HA. Here, we investigated how destabilizing or stabilizing mutations in or near the fusion peptide affect viral sensitivity to Arbidol. The degree of sensitivity was proportional to the extent of fusion peptide stability on the pre-fusion HA: stabilized mutants were more sensitive, and destabilized ones resistant to Arbidol. Single-virion membrane fusion experiments for representative Wild Type and mutant viruses demonstrated that resistance is a direct consequence of fusion-peptide destabilization not dependent on reduced Arbidol binding to HA at neutral pH. Our results support the model whereby the probability of individual HAs extending to engage the target membrane is determined by the composite of two critical forces: a ‘tug’ on the fusion peptide by the extension of the central coiled-coil on HA, and the key interactions stabilizing fusion peptide in the pre-fusion pocket. Arbidol increases the free-energy penalty for coiled-coil extension, but destabilizing mutations decrease the free-energy cost for fusion peptide release, accounting for the observed resistance. Our findings have broad implications for fusion-inhibitor design, viral mechanisms of resistance, and our basic understanding of HA-mediated membrane fusion.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475853', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Cd59 and inflammation orchestrate Schwann cell development', 'authors': \"[{'name': 'Ashtyn T. Wiltbank'}, {'name': 'Emma R. Steinson'}, {'name': 'Stacey J. Criswell'}, {'name': 'Melanie Piller'}, {'name': 'Sarah Kucenas'}]\", 'abstract': '<p>Efficient neurotransmission is essential for organism survival and is enhanced by myelination. However, the genes that regulate myelin and myelinating glial cell development have not been fully characterized. Data from our lab and others demonstrates that <italic>cd59</italic>, which encodes for a small GPI-anchored glycoprotein, is highly expressed in developing zebrafish, rodent, and human oligodendrocytes (OLs) and Schwann cells (SCs), and that patients with CD59 dysfunction develop neurological dysfunction during early childhood. Yet, the function of CD59 in the developing nervous system is currently undefined. In this study, we demonstrate that <italic>cd59</italic> is expressed in a subset of developing SCs. Using <italic>cd59</italic> mutant zebrafish, we show that developing SCs proliferate excessively, which leads to reduced myelin volume, altered myelin ultrastructure, and perturbed node of Ranvier assembly. Finally, we demonstrate that complement activity is elevated in <italic>cd59</italic> mutants and that inhibiting inflammation restores SC proliferation, myelin volume, and nodes of Ranvier to wildtype levels. Together, this work identifies Cd59 and developmental inflammation as key players in myelinating glial cell development, highlighting the collaboration between glia and the innate immune system to ensure normal neural development.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476145', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Increased insula activity precedes the formation of subjective Gestalt', 'authors': \"[{'name': 'Marilena Wilding'}, {'name': 'Christof Koerner'}, {'name': 'Anja Ischebeck'}, {'name': 'Natalia Zaretskaya'}]\", 'abstract': '<p>The constructive nature of human perception sometimes leads us to perceiving rather complex impressions from simple sensory input. Bistable stimuli give us a rare opportunity to study the neural mechanisms behind this process. Such stimuli can be visually interpreted as simple or as more complex on the basis of the same sensory input. Previous studies demonstrated increased activity in the superior parietal cortex when participants perceived an illusory Gestalt impression compared to a simpler interpretation of individual elements. Here we tested whether activity related to the illusory Gestalt can be detected not only during, but also prior to it, by examining the slow fluctuations of resting-state-fMRI activity before the stimulus onset. We presented 31 participants with a bistable motion stimulus, which can be perceived either as four moving dot pairs (local) or two moving illusory squares (global). This allowed us to isolate the specific neural mechanisms that accompany the experience of an illusion under matched sensory input. fMRI was used to measure brain activity in a sparse event-related design. We observed stronger IPS and putamen responses to the stimulus when participants perceived the global interpretation compared to local, confirming the previously reported role of these areas in perceptual grouping. Most importantly, we also observed that the global stimulus interpretation was preceded by an increased activity of the bilateral dorsal insula, which is known to process saliency and gate information for conscious access. Our data suggest an important role of the dorsal insula in shaping an internally generated illusory Gestalt percept.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.10.475595', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'mTORC2 Drives ZFX-mediated Ganglioside Biosynthesis to Promote Breast Cancer Progression', 'authors': \"[{'name': 'Kajal Rajput'}, {'name': 'Mohd. Nafees Ansari'}, {'name': 'Somesh K. Jha'}, {'name': 'Nihal Medatwal'}, {'name': 'Pankaj Sharma'}, {'name': 'Sudeshna Datta'}, {'name': 'Animesh Kar'}, {'name': 'Trishna Pani'}, {'name': 'Kaushavi Cholke'}, {'name': 'Kajal Rana'}, {'name': 'Ali Khan'}, {'name': 'Geetashree Mukherjee'}, {'name': 'SVS Deo'}, {'name': 'Jyothi S Prabhu'}, {'name': 'Arnab Mukhopadhyay'}, {'name': 'Avinash Bajaj'}, {'name': 'Ujjaini Dasgupta'}]\", 'abstract': '<p>Sphingolipid and ganglioside metabolic pathways are crucial components of cell signalling, having established roles in tumor cell proliferation, invasion, and migration. However, regulatory mechanisms controlling sphingolipid and ganglioside synthesis in mammalian cells is less known. Here, we show that RICTOR, the regulatory subunit of mTORC2, regulates the synthesis of sphingolipids and gangliosides in Luminal breast cancer-specific MCF-7 cells through transcriptional and epigenetic mechanisms. RICTOR regulates glucosylceramide levels by modulating the expression of UDP-Glucose Ceramide Glucosyl transferase (UGCG). We identify Zinc Finger protein X-linked (ZFX) as a RICTOR-responsive transcription factor whose recruitment to the <italic>UGCG</italic> promoter is regulated by DNA methyltransferases and histone demethylase (KDM5A) that are known AKT substrates. We further demonstrate that RICTOR regulates the synthesis of GD3 gangliosides through ZFX and UGCG, and triggers the activation of the EGFR signalling pathway, thereby promoting tumor growth. In line with our findings in cell culture and mice models, we observe an elevated expression of RICTOR, ZFX, and UGCG in Indian Luminal breast cancer patient samples, and in TCGA and METABRIC datasets. Together, we establish a key regulatory circuit, RICTOR-AKT-ZFX-UGCG-Ganglioside-EGFR-AKT, and elucidate its contribution to breast cancer progression.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475685', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Multi-GBS: A massively multiplexed GBS-based protocol optimized for large, repetitive conifer genomes', 'authors': \"[{'name': 'Miguel Vallebueno-Estrada'}, {'name': 'Sonja Steindl'}, {'name': 'Vasilina Akulova'}, {'name': 'Julia Riefler'}, {'name': 'Lucyna Slusarz'}, {'name': 'Alexis Arizpe'}, {'name': 'Kelly Swarts'}]\", 'abstract': '<p>Reduced representation library approaches are still a valuable tool for breeding and population and ecological genomics, even with impressive increases in sequencing capacity in recent years. Unfortunately, current approaches only allow for multiplexing up to 384 samples. To take advantage of increased sequencing capacity, we present Multi-GBS, a massively multiplexable extension to Genotyping-by-Sequencing that is also optimized for large conifer genomes. In Norway Spruce, a highly repetitive 20Gbp diploid genome with high population genetic variation, we call over a million variants in 32 genotypes from three populations, two natural forest in the Alps and Bohemian Alps, and a managed population from southeastern Austria using the existing TASSEL GBSv2 pipeline. Metric MDS analysis of replicated genotypes shows that technical bias in resulting genotype calling is minimal and that populations cluster in biologically meaningful ways.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.475671', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Intracellular flow cytometry staining of antibody-secreting cells using phycoerythrin-conjugated antibodies: pitfalls and solutions', 'authors': \"[{'name': 'Patrick Renner'}, {'name': 'Michael Crone'}, {'name': 'Matthew Kornas'}, {'name': 'KimAnh T. Pioli'}, {'name': 'Peter D. Pioli'}]\", 'abstract': '<p>Antibody-secreting cells are terminally differentiated B cells that play a critical role in humoral immunity through immunoglobulin secretion along with possessing the potential to be long-lived. It is now appreciated that antibody-secreting cells regulate multiple aspects of biology through the secretion of various cytokines. In this regard, intracellular flow cytometry is a key tool used to assess the presence of intracellular proteins such as cytokines and transcription factors. Here, we showed that the use of phycoerythrin-containing antibody conjugates led to a false interpretation of antibody-secreting cell intracellular protein expression compared to other cell types. This was mainly due to the inappropriate retention of these antibodies specifically within antibody-secreting cells. Furthermore, we demonstrated how to reduce this retention which allowed for a more accurate comparison of intracellular protein expression between antibody-secreting cells and other cell types such as B lymphocytes. Using this methodology, our data revealed that spleen antibody-secreting cells expressed Toll-like receptor 7 as well as the pro-form of the inflammatory cytokine interleukin-1β.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.475658', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Sensitive protein detection using site-specifically oligonucleotide-conjugated nanobodies', 'authors': \"[{'name': 'Rasel A. Al-Amin'}, {'name': 'Phathutshedzo M. Muthelo'}, {'name': 'Eldar Abdurakhmanov'}, {'name': 'Cécile Vincke'}, {'name': 'Serge Muyldermans'}, {'name': 'U. Helena Danielson'}, {'name': 'Ulf Landegren'}]\", 'abstract': '<p>High-quality affinity probes are critical for sensitive and specific protein detection, in particular to detect protein biomarkers at early phases of disease development. Clonal affinity reagents can offer advantages over the commonly used polyclonal antibodies (pAbs) in terms of reproducibility and standardization of such assays. In particular, clonal reagents offer opportunities for site-directed attachment of exactly one modification per affinity reagent at a site designed not to interfere with target binding to help standardize assays. The proximity extension assays (PEA) is a widely used protein assay where pairs of protein-binding reagents are modified with oligonucleotides (oligos), so that their proximal binding to a target protein generates a reporter DNA strand for DNA-assisted readout. The assays have been used for high-throughput multiplexed protein detection of up to a few thousand different proteins in one or a few microliters of plasma. Here we explore nanobodies (Nb) as an alternative to polyclonal antibodies pAbs as affinity reagents for PEA. We describe an efficient site-specific approach for preparing high-quality oligo-conjugated Nb probes via Sortase A (SrtA) enzyme coupling. The procedure allows convenient removal of unconjugated affinity reagents after conjugation. The purified high-grade Nb probes were used in PEA and the reactions provided an efficient means to select optimal pairs of binding reagents from a group of affinity reagents. We demonstrate that Nb-based PEA for interleukin-6 (IL6) detection can augment assay performance, compared to the use of pAb probes. We identify and validate Nb combinations capable of binding in pairs without competition for IL6 antigen detection by PEA.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.15.476463', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Genome assembly, structural variants, and genetic differentiation between Lake Whitefish young species pairs (Coregonus sp.) with long and short reads', 'authors': \"[{'name': 'Claire M&eacuterot'}, {'name': 'Kristina S R Stenl&oslashkk'}, {'name': 'Clare Venney'}, {'name': 'Martin Laporte'}, {'name': 'Michel Moser'}, {'name': 'Eric Normandeau'}, {'name': 'Mariann &Agravernyasi'}, {'name': 'Matthew Kent'}, {'name': 'Cl&eacutement Rougeux'}, {'name': 'Jullien M Flynn'}, {'name': 'Sigbjorn Lien'}, {'name': 'Louis Bernatchez'}]\", 'abstract': '<p>The parallel evolution of nascent pairs of ecologically differentiated species offers an opportunity to get a better glimpse at the genetic architecture of speciation. Of particular interest is our recent ability to consider a wider range of genomic variants, not only single-nucleotide polymorphisms (SNPs), thanks to long-read sequencing technology. We can now identify structural variants (SVs) like insertions, deletions, and other structural rearrangements, allowing further insights into the genetic architecture of speciation and how different variants are involved in species differentiation. Here, we investigated genomic patterns of differentiation between sympatric species pairs (Dwarf and Normal) belonging to the Lake Whitefish (Coregonus clupeaformis) species complex. We assembled the first reference genomes for both Dwarf and Normal Lake Whitefish, annotated the transposable elements, and analysed the genome in the light of related coregonid species. Next, we used a combination of long-read and short-read sequencing to characterize SVs and genotype them at population-scale using genome-graph approaches, showing that SVs cover five times more of the genome than SNPs. We then integrated both SNPs and SVs to investigate the genetic architecture of species differentiation in two different lakes and highlighted an excess of shared outliers of differentiation. In particular, a large fraction of SVs differentiating the two species was driven by transposable elements (TEs), suggesting that TE accumulation during a period of allopatry predating secondary contact may have been a key process in the speciation of the Dwarf and Normal Whitefish. Altogether, our results suggest that SVs play an important role in speciation and that by combining second and third generation sequencing we now have the ability to integrate SVs into speciation genomics.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.15.476464', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'MeShClust v3.0: High-quality clustering of DNA sequences using the mean shift algorithm and alignment-free identity scores', 'authors': \"[{'name': 'Hani Z Girgis'}]\", 'abstract': '<p>Background: Tools for accurately clustering biological sequences are among the most important tools in computational biology. Two pioneering tools for clustering sequences are CD-HIT and UCLUST, both of which are fast and consume reasonable amounts of memory; however, there is a big room for improvement in terms of cluster quality. Motivated by this opportunity for improving cluster quality, we applied the mean shift algorithm in MeShClust v1.0. The mean shift algorithm is an instance of unsupervised learning. Its strong theoretical foundation guarantees the convergence to the true cluster centers. Our implementation of the mean shift algorithm in MeShClust v1.0 was a step forward; however, it was not the original algorithm. In this work, we make progress toward applying the original algorithm while utilizing alignment-free identity scores in a new tool: MeShClust v3.0. Results: We evaluated CD-HIT, MeShClust v1.0, MeShClust v3.0, and UCLUST on 22 synthetic sets and five real sets. These data sets were designed or selected for testing the tools in terms of scalability and different similarity levels among sequences comprising clusters. On the synthetic data sets, MeShClust v3.0 outperformed the related tools on all sets in terms of cluster quality. On two real data sets obtained from human microbiome and maize transposons, MeShClust v3.0 outperformed the related tools by wide margins, achieving 55%-300% improvement in cluster quality. On another set that includes degenerate viral sequences, MeShClust v3.0 came third. On two bacterial sets, MeShClust v3.0 was the only applicable tool because of the long sequences in these sets. MeShClust v3.0 requires more time and memory than the related tools; almost all personal computers at the time of this writing can accommodate such requirements. MeShClust v3.0 can estimate an important parameter that controls cluster membership with high accuracy. Conclusions: These results demonstrate the high quality of clusters produced by MeShClust v3.0 and its ability to apply the mean shift algorithm to large data sets and long sequences. Because clustering tools are utilized in many studies, providing high-quality clusters will help with deriving accurate biological knowledge.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.11.475834', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'House sparrows do not exhibit a preference for the scent of potential partners with different MHC-I allele numbers and genetic distances', 'authors': \"[{'name': 'Luisa Amo'}, {'name': 'Guillermo Amo de Paz'}, {'name': 'Johanna Kabbert'}, {'name': 'Annie Machordom'}]\", 'abstract': '<p>MHC genes play a fundamental role in immune recognition of pathogens and parasites. Therefore, females may increase offspring heterozygosity and genetic diversity by selecting MHC genetically compatible or heterozygous males. In birds, several studies suggest that MHC genes play a role in mate choice, and recent evidence suggest that olfaction may play a role in such discrimination. Previous studies indicated that house sparrow females with low allelic diversity prefer males with higher diversity in MHC-I alleles. Here, we directly explored whether both house sparrow females and males could estimate by scent the number in MHC amino acid and functional variants as well as the level of MHC-I similarity or dissimilarity of potential partners. Our results show that neither females nor males exhibit a preference related to the number of MHC-I amino acid variants or functional variants or in relation to MHC amino acid or functional similarity of potential partners, suggesting that MHC-I is not detected through olfaction. Further studies are needed to understand the mechanisms responsible for MHC-I based mate discrimination in birds.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475787', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Impact of Ghost Introgression on Coalescent-based Species Tree Inference and Estimation of Divergence Time', 'authors': \"[{'name': 'Xiao-Xu Pang'}, {'name': 'Da-Yong Zhang'}]\", 'abstract': '<p>The species studied in any evolutionary investigation generally constitute a very small proportion of all the species currently existing or that have gone extinct. It is therefore likely that introgression, which is widespread across the tree of life, involves “ghosts,” i.e., unsampled, unknown, or extinct lineages. However, the impact of ghost introgression on estimations of species trees has been rarely studied and is thus poorly understood. In this study, we use mathematical analysis and simulations to examine the robustness of species tree methods based on a multispecies coalescent model under gene flow sourcing from an extant or ghost lineage. We found that very low levels of extant or ghost introgression can result in anomalous gene trees (AGTs) on three-taxon rooted trees if accompanied by strong incomplete lineage sorting (ILS). In contrast, even massive introgression, with more than half of the recipient genome descending from the donor lineage, may not necessarily lead to AGTs. In cases involving an ingroup lineage (defined as one that diverged no earlier than the most basal species under investigation) acting as the donor of introgression, the time of root divergence among the investigated species was either underestimated or remained unaffected, but for the cases of outgroup ghost lineages acting as donors, the divergence time was generally overestimated. Under many conditions of ingroup introgression, the stronger the ILS was, the higher was the accuracy of estimating the time of root divergence, although the topology of the species tree is more prone to be biased by the effect of introgression.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475946', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Abscisic Acid Increases Hydrogen Peroxide in Multiple Subcellular Compartments to Drive Stomatal Closure', 'authors': \"[{'name': 'Anthony E. Postiglione'}, {'name': 'Gloria K. Muday'}]\", 'abstract': '<p>Stomatal closure regulates transpiration and gas exchange in response to environmental cues. Drought upregulates ABA signaling, which elevates levels of reactive oxygen species (ROS). However, the subcellular location and identity of these ROS has received limited study. We found that in guard cells, ABA increased fluorescence of the general redox sensor, dichlorofluorescein (DCF), in distinct subcellular locations including chloroplasts, cytosol, nuclei, and cytosolic puncta. These changes were lost in an ABA-insensitive quintuple receptor mutant and accentuated in an ABA-hypersensitive mutant. ABA induced ROS accumulation in these subcellular compartments was lost in mutants with defects in genes encoding H<sub>2</sub>O<sub>2</sub> elevating respiratory burst oxidase homolog (RBOH) enzymes and guard cells treated with the RBOH inhibitor VAS2870, while exogenous H<sub>2</sub>O<sub>2</sub> treatment is sufficient to close guard cells. The H<sub>2</sub>O<sub>2</sub>-selective probe, peroxy orange 1, also showed ABA-dependent increases in chloroplasts and cytosolic puncta. Using the more sensitive genetically-encoded H<sub>2</sub>O<sub>2</sub> reporter roGFP-Orp1, we also detected significant H<sub>2</sub>O<sub>2</sub> increases in the cytosol and nucleus. These cytosolic puncta accumulate ROS after ABA treatment and show colocalization with Mitotracker and with a mitochondrial targeted mt-roGFP2-Orp1, which also reports ABA-increased H<sub>2</sub>O<sub>2</sub> in mitochondria. These results indicate that elevated H<sub>2</sub>O<sub>2</sub> after ABA treatment in these subcellular compartments is necessary and sufficient to drive stomatal closure.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.16.476537', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Using strain-resolved analysis to identify contamination in metagenomics data', 'authors': \"[{'name': 'Yue Clare Lou'}, {'name': 'Jordan Hoff'}, {'name': 'Matthew R. Olm'}, {'name': 'Jacob West-Roberts'}, {'name': 'Spencer Diamond'}, {'name': 'Brian A. Firek'}, {'name': 'Michael J. Morowitz'}, {'name': 'Jillian F. Banfield'}]\", 'abstract': '<p>Metagenomics analysis can be negatively impacted by DNA contamination. While external sources of contamination such as DNA extraction kits have been widely reported and investigated, contamination originating within the study itself remains underreported. Here we applied high-resolution strain-resolved analyses to identify contamination in two large-scale clinical metagenomics datasets. By mapping strain sharing to DNA extraction plates, we identified well-to-well contamination in both negative controls and biological samples in one dataset. Such contamination is more likely to occur among samples that are on the same or adjacent columns or rows of the extraction plate than samples that are far apart. Our strain-resolved workflow also reveals the presence of externally derived contamination, primarily in the other dataset. Overall in both datasets, contamination is more significant in samples with lower biomass. Our work demonstrates that genome-resolved strain tracking, with its essentially genome-wide nucleotide-level resolution, can be used to detect contamination in sequencing-based microbiome studies. Our results underscore the value of strain-specific methods to detect contamination and the critical importance of looking for contamination beyond negative and positive controls.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.17.475672', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A deep learning pipeline for segmentation of Proteus mirabilis colony patterns', 'authors': \"[{'name': 'Anjali Doshi'}, {'name': 'Marian Shaw'}, {'name': 'Ruxandra Tonea'}, {'name': 'Rosalía Minyety'}, {'name': 'Soonhee Moon'}, {'name': 'Andrew Laine'}, {'name': 'Jia Guo'}, {'name': 'Tal Danino'}]\", 'abstract': '<p>The motility mechanisms of microorganisms are critical virulence factors, enabling their spread and survival during infection. Motility is frequently characterized by qualitative analysis of macroscopic colonies, yet the standard quantification method has mainly been limited to manual measurement. Recent studies have applied deep learning for classification and segmentation of specific microbial species in microscopic images, but less work has focused on macroscopic colony analysis. Here, we advance computational tools for analyzing colonies of Proteus mirabilis, a bacterium that produces a macroscopic bullseye-like pattern via periodic swarming, a process implicated in its virulence. We present a dual-task pipeline for segmenting (1) the macroscopic colony including faint outer swarm rings, and (2) internal ring boundaries, unique features of oscillatory swarming. Our convolutional neural network for patch-based colony segmentation and U-Net with a VGG-11 encoder for ring boundary segmentation achieved test Dice scores of 93.28% and 83.24%, respectively. The predicted masks at times improved on the ground truths from our automated annotation algorithms. We demonstrate how application of our pipeline to a typical swarming assay enables ease of colony analysis and precise measurements of more complex pattern features than those which have been historically quantified.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.11.475465', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Drug Contraindications in Comorbid Diseases: a Protein Interactome Perspective', 'authors': \"[{'name': 'Kalyani B Karunakaran'}, {'name': 'Madhavi Ganapathiraju'}, {'name': 'Sanjeev Jain'}, {'name': 'Samir Brahmachari'}, {'name': 'Narayanaswamy Balakrishnan'}]\", 'abstract': '<p>Adverse drug reactions (ADRs) are leading causes of death and drug withdrawals and frequently co-occur with comorbidities. However, systematic studies on the effects of drugs in comorbidities are lacking. Drug interactions with the cellular protein-protein interaction (PPI) network give rise to ADRs. We selected 6 comorbid disease pairs, identified the drugs used in the treatment of the individual diseases A and B — 44 drugs in anxiety and depression, 128 in asthma and hypertension, 48 in chronic obstructive pulmonary disease and heart failure, 58 in type 2 diabetes and obesity, 58 in Parkinson′s disease and schizophrenia, and 84 in rheumatoid arthritis and osteoporosis — and categorized them based on whether they aggravate the comorbid condition. We constructed drug target networks (DTNs) and examined their enrichment among genes in disease A/B PPI networks, expressed across 53 tissues and involved in ≈1000 pathways. To pinpoint the biological features characterizing the DTNs, we performed principal component analysis and computed the Euclidean distance between DTN component scores and feature loading values. DTNs of disease A drugs not contraindicated in B were affiliated with proteins common to A/B networks or uniquely found in the B network, similarly regulated common pathways, and disease-B specific pathways and tissues. DTNs of disease A drugs contraindicated in B were affiliated with common proteins or those uniquely found in the A network, differentially regulated common pathways, and disease A-specific pathways and tissues. Hence, DTN enrichment in pathways, tissues, and PPI networks of comorbid diseases will help identify drugs contraindications in comorbidities.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.11.22269101', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Racial/ethnic disparities in exposure to COVID-19, susceptibility to COVID-19 and access to health care - findings from a U.S. national cohort', 'authors': \"[{'name': 'McKaylee Robertson'}, {'name': 'Meghana Shamsunder'}, {'name': 'Ellen Brazier'}, {'name': 'Mekhala Mantravadi'}, {'name': 'Madhura S Rane'}, {'name': 'Drew A Westmoreland'}, {'name': 'Angela Parcesepe'}, {'name': 'Rebecca Zimba'}, {'name': 'Andrew R Maroko'}, {'name': 'Sarah Kulkarni'}, {'name': 'Christian Grov'}, {'name': 'Denis Nash'}]\", 'abstract': '<p>We examined the influence of racial/ethnic differences in socioeconomic position on COVID-19 seroconversion and hospitalization within a community-based prospective cohort enrolled in March 2020 and followed through October 2021 (N=6740). The ability to social distance as a measure of exposure to COVID-19, susceptibility to COVID-19 complications, and access to healthcare varied by race/ethnicity with non-white participants having more exposure risk and more difficulty with healthcare access than white participants. Participants with more (versus less) exposure had greater odds of seroconversion (aOR:1.64, 95% Confidence Interval [CI] 1.18-2.29). Participants with more susceptibility and more barriers to healthcare had greater odds of hospitalization (respective aOR:2.36; 1.90-2.96 and 2.31; 1.69-2.68). Race/ethnicity positively modified the association between susceptibility and hospitalization (aORnon-White:2.79, 2.06-3.78). Findings may explain the disproportionate burden of COVID-19 infections and complications among Hispanic and non-Hispanic Black persons. Primary and secondary prevention efforts should address disparities in exposure, COVID-19 vaccination, and treatment.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476327', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'CARD9 in Neutrophils Protects from Colitis and Controls Mitochondrial Metabolism and Cell Survival', 'authors': \"[{'name': 'Camille Danne'}, {'name': 'Chloe Michaudel'}, {'name': 'Jurate Skerniskyte'}, {'name': 'Julien Planchais'}, {'name': 'Aurelie Magniez'}, {'name': 'Allison Agus'}, {'name': 'Marie-Laure Michel'}, {'name': 'Bruno Lamas'}, {'name': 'Gregory Da-Costa'}, {'name': 'Madeleine Spatz'}, {'name': 'Cyriane Oeuvray'}, {'name': 'Chloe Galbert'}, {'name': 'Maxime Poirier'}, {'name': 'Yazhou Wang'}, {'name': 'Alexia Lapiere'}, {'name': 'Nathalie Rolhion'}, {'name': 'Tatiana Ledent'}, {'name': 'Cedric Pionneau'}, {'name': 'Solenne Chardonnet'}, {'name': 'Floriant Bellvert'}, {'name': 'Edern Cahoreau'}, {'name': 'Amandine Rocher'}, {'name': 'Rafael J Arguello'}, {'name': 'Carole Peyssonnaux'}, {'name': 'Sabine Louis'}, {'name': 'Mathias Lavie-Richard'}, {'name': 'Philippe Langella'}, {'name': 'Jamel El-Benna'}, {'name': 'Benoit Marteyn'}, {'name': 'Harry Sokol'}]\", 'abstract': '<p>Objectives: Inflammatory bowel disease (IBD) results from a combination of genetic predisposition, dysbiosis of the gut microbiota and environmental factors, leading to alterations in the gastrointestinal immune response and chronic inflammation. Caspase recruitment domain 9 (Card9), one of the IBD susceptibility genes, has been shown to protect against intestinal inflammation and fungal infection. However, the cell types and mechanisms involved in the CARD9 protective role against inflammation remain unknown. Design: We used dextran sulfate sodium (DSS)-induced and adoptive transfer colitis models in total and conditional CARD9 knock-out mice to uncover which cell types play a role in the CARD9 protective phenotype. The impact of Card9 deletion on neutrophil function was assessed by an in vivo model of fungal infection and various functional assays, including endpoint dilution assay, apoptosis assay by flow cytometry, proteomics and real time bioenergetic profile analysis (Seahorse). Results: Lymphocytes are not intrinsically involved in the CARD9 protective role against colitis. CARD9 expression in neutrophils, but not in epithelial or CD11c+ cells, protects against DSS-induced colitis. In the absence of CARD9, mitochondrial dysfunction in neutrophils leads to their premature death through apoptosis, especially in oxidative environment. The decrease of fonctional neutrophils in tissues could explain the impaired containment of fungi and increased susceptibility to intestinal inflammation. Conclusion: These results provide new insight into the role of CARD9 in neutrophil mitochondrial function and its involvement in intestinal inflammation, paving the way for new therapeutic strategies targeting neutrophils.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.10.475630', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Enhancement of central norepinephrinergic neurotransmission contributes to inhibition of seizure-induced respiratory arrest by targeting the beta-1 adrenergic receptor (β1-AR) locating in the cardiomyocytes in the DBA/1 mouse SUDEP model', 'authors': \"[{'name': 'XiTing Lian'}, {'name': 'Qian Yu'}, {'name': 'HaiXiang Ma'}, {'name': 'LeYuan Gu'}, {'name': 'Qing Xu'}, {'name': 'HongHai Zhang'}]\", 'abstract': '<p>Sudden unexpected death of epilepsy (SUDEP) is the key cause of of death in patients with epilepsy. Due to the complicated pathogenesis of SUDEP, however, the exact mechanism of SUDEP remains elusive. Currently, although it is recognized that the seizure-induced respiratory arrest (S-IRA) may be a main cause for SUDEP, other factors resulting in SUDEP can not be excluded e.g arrhythmias. Our previous findings indicated that the incidence of seizure-induced respiratory arrest S-IRA and SUDEP evoked by acoustic stimulation or pentetrazol (PTZ) injection was significantly reduced by atomoxetine, a norepinephrine reuptake inhibitor (NRI), suggesting that noradrenergic neurotransmission modulates S-IRA and SUDEP. Given that norepinephrine acts on the central and peripheral target to modulate respiratory and circulation function by targeting adrenergic receptor α and beta (a-AR and β-AR) and the arrhythmias can be contributed to SUDEP. Meanwhile, to further test whether cardiac factors are implicated in S-IRA and SUDEP, we choose esmolol hydrochloride, a selective antagonist of beta-1 adrenergic receptor (β1-AR) to test it in our models. Our findings demonstrated that the lower incidence of S-IRA and SUDEP evoked by acoustic stimulation or PTZ in DBA/1 mice by administration with atomoxetine was significantly reversed by intraperitoneal (IP) of esmolol hydrochloride. Importantly, the data of electrocardiogram (ECG) showed that the cardiac arrhythmia evoked by acoustic stimulation including the ventricular tachycardia, ventricular premature beat and atrioventricular block and administration of atomoxetine significantly reduced theses arrhythmias and the incidence of S-IRA and SUDEP in our models. Thus, the dysfunction of respiratory and circulation may be implicated in the pathogenesis of S-IRA and SUDEP hand in hand and enhancing central norepinephrinergic neurotransmission contributes to inhibition of seizure-induced respiratory arrest by targeting β1-AR locating in the cardiomyocytes. Our findings will show a new light on decoding the pathogenesis of SUDEP.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.13.476151', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Site-specific adaptation mechanisms of an oral pathobiont in the oral and gut mucosae', 'authors': \"[{'name': 'Yijie Guo'}, {'name': 'Sho Kitamoto'}, {'name': 'Gustavo Caballero-Flores'}, {'name': 'Daisuke Watanabe'}, {'name': 'Kohei Sugihara'}, {'name': 'Gabriel Núñez'}, {'name': 'Christopher J. Alteri'}, {'name': 'Naohiro Inohara'}, {'name': 'Nobuhiko Kamada'}]\", 'abstract': '<p>Periodontal inflammation leads to oral dysbiosis with the expansion of oral pathobionts. Besides the pathogenic role of oral pathobionts during periodontal inflammation, studies have revealed that oral pathobionts contribute to diseases in distant organs beyond the oral mucosa. For example, the oral pathobiont <italic>Klebsiella aerogenes</italic>, which accumulates in the oral mucosa during periodontitis in mice, can exacerbate colitis when it ectopically colonizes the gastrointestinal tract. However, the precise mechanisms by which oral pathobionts establish their colonization in extra-oral mucosal sites remains incompletely understood. We performed high-throughput <italic>in vivo</italic> genetic screening to identify fitness genes required for the adaptation of the oral pathobiont <italic>K. aerogenes</italic> to different mucosal sites – the oral and gut mucosae – at the steady state and during inflammation. In addition, the global transcriptome of <italic>K. aerogenes</italic> in different environments was analyzed. We determined that <italic>K. aerogenes</italic> employs genes related to iron acquisition and chaperone usher pili, which are encoded on a newly identified genomic locus named “locus of colonization in the inflamed gut” (LIG), for adaptation in the gut mucosa, particularly during inflammation. In contrast, the LIG virulence factors are not required for <italic>K. aerogenes</italic> to adapt to the oral mucosa. Thus, oral pathobionts likely exploit distinct adaptation mechanisms in their ectopically colonized intestinal niche compared to their original niche.</p><sec><title>Author Summary</title><p>The symbiotic bacterial community evolves uniquely at each body site. However, under certain circumstances, symbionts may disseminate far from their original niche and colonize a site ectopically. It has been reported that oral resident bacteria associated with periodontal disease, namely oral pathobionts, contribute to gastrointestinal diseases, such as inflammatory bowel disease and colorectal cancer, through ectopic colonization of the gut. However, the molecular mechanisms by which oral pathobionts adapt to the gut environment remain largely unexplored. In this study, using a model oral pathobiont <italic>Klebsiella aerogenes</italic>, we screened genes essential for the colonization of the pathobiont at different mucosal sites. We discovered that the oral pathobiont uses distinct virulence mechanisms at two different mucosal sites – the oral and gut mucosae. Understanding the strategies of bacteria localized in ectopic organs may lead to the development of strategies to prevent disease caused by the dissemination of pathogenic oral bacteria.</p></sec>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.13.476204', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Covariance predicts conserved protein residue interactions important to the emergence and continued evolution of SARS-CoV-2 as a human pathogen', 'authors': \"[{'name': 'William P. Robins'}, {'name': 'John J. Mekalanos'}]\", 'abstract': '<p>SARS-CoV-2 is one of three recognized coronaviruses (CoVs) that have caused epidemics or pandemics in the 21st century and that likely emerged from animal reservoirs. Differences in nucleotide and protein sequence composition within related β-coronaviruses are often used to better understand CoV evolution, host adaptation, and their emergence as human pathogens. Here we report the comprehensive analysis of amino acid residue changes that have occurred in lineage B β-coronaviruses (sarbecoviruses) that show covariance with each other. This analysis revealed patterns of covariance within conserved viral proteins that potentially define conserved interactions within and between core proteins encoded by SARS-CoV-2 related β-coranaviruses. We identified not only individual pairs but also networks of amino acid residues that exhibited statistically high frequencies of covariance with each other using an independent pair model followed by a tandem model approach. Using 149 different CoV genomes that vary in their relatedness, we identified networks of unique combinations of alleles that can be incrementally traced genome by genome within different phylogenic lineages. Remarkably, covariant residues and their respective regions most abundantly represented are implicated in the emergence of SARS-CoV-2 and are also enriched in dominant SARS-CoV-2 variants.</p><sec><title>Significance Statement</title><p>There currently are enormous international efforts to better understand the emergence of SARS-CoV-2 as a human pathogen and its persistence in the ongoing pandemic in various mutated forms. Most studies have focused on identifying presumptive gain-of-fitness mutations in core viral proteins. The contributions of residues unique to SARS-CoV-2 or those conserved in other β-coronaviruses are not yet understood. Also, the absence of an identified ancestral virus to SARS-CoV-2 prevents a continuous comparison-driven analysis within a lineage of other potential bat-, civet-, and human-adapted viruses such as SARS-CoV that may have similarly contributed to the emergence of SAR-CoV-2 through viral evolution and recombination. Here we identified unique amino acid residues that are found to be variable between distinct β-coronaviruses but are conserved in covariant pairs or networks of covariant residues within or between viral proteins. The relationships revealed are likely preserved in the evolutionary record because these covariant residues impose selective pressure on each other through direct or indirect interactions. The covariant residues we identified could play key roles in the emergence of SARS-CoV-2 as a human pathogen and its continued evolution in more recently emerged variants of clinical interest.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476280', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Bulk and spatially resolved extracellular metabolomics of free-living nitrogen fixation', 'authors': \"[{'name': 'Darian Smercina'}, {'name': 'Young-Mo Kim'}, {'name': 'Mary Lipton'}, {'name': 'Dusan Velickovic'}, {'name': 'Kirsten Hofmockel'}]\", 'abstract': '<p>Soil microorganisms drive ecosystem function, but challenges of scale between microbe and ecosystem hinder our ability to accurately quantify and predictively model the soil microbe-ecosystem function relationship. Quantifying this relationship necessitates studies that systematically characterize multi-omics of soil microorganisms and their activity across sampling scales from spatially resolved to bulk measures, and structural complexity, from liquid pure culture to in situ. To address this need, we cultured two diazotrophic bacteria in liquid and solid media, with and without nitrogen (N) to quantify differences in extracellular metabolites associated with nitrogen fixation under increasing environmental structural complexity. We also quantified extracellular metabolites across sampling scales including bulk sampling via GC-MS analysis and spatially resolved analysis via MALDI mass spectrometry imaging. We found extracellular production of inorganic and organic N during free-living nitrogen fixation activity, highlighting a key mechanism of terrestrial N contributions from this process. Additionally, our results emphasize the need to consider the structural complexity of the environment and spatial scale when quantifying microbial activity. We found differences in metabolite profiles between culture conditions, supporting previous work indicating environmental structure influences microbial function, and across scales, underscoring the need to quantify microbial scale conditions to accurately interpret microbial function.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.22269064', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Evaluation and modelling of the performance of an automated SARS-CoV-2 antigen assay according to sample type, target population and epidemic trends', 'authors': \"[{'name': 'Nicolas Yin'}, {'name': 'Cyril Debuysschere'}, {'name': 'Valery Daubie'}, {'name': 'Marc Hildebrand'}, {'name': 'Charlotte Martin'}, {'name': 'Sonja Curac'}, {'name': 'Fanny Ponthieux'}, {'name': 'Marie-Christine Payen'}, {'name': 'Olivier Vandenberg'}, {'name': 'Marie Hallin'}]\", 'abstract': '<p>The Lumipulse® G SARS-CoV-2 Ag assay performance was evaluated on prospectively collected saliva and nasopharyngeal swabs (NPS) of recently ill in- and outpatients and according to the estimated viral load. Performances were calculated using RT-PCR positive NPS from patients with symptoms ≤ 7 days and RT-PCR negative NPS as gold standard. In addition, non-selected positive NPS were analyzed to assess the performances on various viral loads. This assay yielded a sensitivity of 93.1% on NPS and 71.4% on saliva for recently ill patients. For NPS with a viral load &gt; 103 RNA copies/mL, sensitivity was 96.4%. A model established on our daily routine showed fluctuations of the performances depending on the epidemic trends but an overall good negative predictive value. Lumipulse® G SARS-CoV-2 assay yielded good performance for an automated antigen detection assay on NPS. Using it for the detection of recently ill patient or to screen high-risk patients could be an interesting alternative to the more expensive RT-PCR.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476283', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Pathology', 'covid': False, 'title': 'Ehrlichia SLiM ligand mimetic activates Notch signaling in human monocytes', 'authors': \"[{'name': 'LaNisha L. Patterson'}, {'name': 'Thangam Sudha Velayutham'}, {'name': 'Caitlan D. Byerly'}, {'name': 'Duc Cuong Bui'}, {'name': 'Jignesh Patel'}, {'name': 'Veljko Veljkovic'}, {'name': 'Slobodan Paessler'}, {'name': 'Jere W. McBride'}]\", 'abstract': '<p><italic>Ehrlichia chaffeensis</italic> evades innate host defenses by reprogramming the mononuclear phagocyte through mechanisms that involve exploitation of multiple evolutionarily conserved cellular signaling pathways including Notch. This immune evasion strategy is directed in part by tandem repeat protein (TRP) effectors. Specifically, the TRP120 effector activates and regulates Notch signaling through interactions with the Notch receptor and the negative regulator, F-Box and WD repeat domain-containing 7 (FBW7). However, the specific molecular interactions and motifs required for E. chaffeensis TRP120-Notch receptor interaction and activation have not been defined. To investigate the molecular basis of TRP120 Notch activation, we compared TRP120 with endogenous canonical/non-canonical Notch ligands and identified a short region of sequence homology within the tandem repeat (TR) domain. TRP120 was predicted to share biological function with Notch ligands, and a function-associated sequence in the TR domain was identified. To investigate TRP120-Notch receptor interactions, colocalization between TRP120 and endogenous Notch-1 was observed. Moreover, direct interactions between full length TRP120, the TRP120 TR domain containing the putative Notch ligand sequence, and the Notch receptor LBR were demonstrated. To molecularly define the TRP120 Notch activation motif, peptide mapping was used to identify an 11-amino acid short linear motif (SLiM) located within the TRP120 TR that activated Notch signaling and downstream gene expression. Peptide mutants of the Notch SLiM or anti-Notch SLiM antibody reduced or eliminated Notch activation and NICD nuclear translocation. This investigation reveals a novel molecularly defined pathogen encoded Notch SLiM mimetic that activates Notch signaling consistent with endogenous ligands.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.12.476098', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Maternally transferred monoclonal antibodies protect neonatal mice from herpes simplex virus-induced mortality and morbidity', 'authors': \"[{'name': 'Iara M. Backes'}, {'name': 'Brook K. Byrd'}, {'name': 'Chaya D. Patel'}, {'name': 'Sean A. Taylor'}, {'name': 'Callaghan R. Garland'}, {'name': 'Scott W. MacDonald'}, {'name': 'Alejandro B. Balazs'}, {'name': 'Scott C. Davis'}, {'name': 'Margaret E. Ackerman'}, {'name': 'David A. Leib'}]\", 'abstract': '<p>Neonatal herpes simplex virus (HSV) infections often result in significant mortality and neurological morbidity despite antiviral drug therapy. Maternally-transferred HSV-specific antibodies reduce the risk of clinically-overt neonatal HSV (nHSV), but this observation has not been translationally applied. Using a neonatal mouse model, we tested the hypothesis that passive transfer of HSV-specific human monoclonal antibodies (mAbs) can prevent mortality and morbidity associated with nHSV. The mAbs were expressed <italic>in vivo</italic> by vectored immunoprophylaxis, or administered <italic>in vivo</italic> following recombinant expression <italic>in vitro</italic>. Through these maternally-derived routes or through direct administration to pups, diverse mAbs to HSV glycoprotein D protected against neonatal HSV-1 and HSV-2 infection. Using <italic>in vivo</italic> bioluminescent imaging, both pre- and post-exposure mAb treatment significantly reduced viral load. Administration of mAb also reduced nHSV-induced behavioral morbidity, as measured by anxiety-like behavior. Together these studies support the notion that HSV-specific mAb-based therapies may prevent or improve HSV infection outcomes in neonates.</p><sec><title>Graphical Abstract</title><p><fig id=\"ufig1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"476098v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></p><p>Different antibody sources were used to maternally-transfer or directly administer HSV-specific mAbs to mouse pups. Neonatal mice were challenged with wild type or bioluminescent virus before or after mAb acquisition. Following infection, pups were assessed for survival, virus-induced bioluminescence and anxiety-like behavior as a measure of neurological morbidity. Efficacy was time and mAb dependent. Notably, all HSV-specific mAbs prevented nHSV-associated mortality.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.22269000', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Obstetrics and Gynecology', 'covid': False, 'title': 'Development of a screening tool for pelvic floor dysfunction in female athletes: protocol of a Delphi consensus', 'authors': \"[{'name': 'Silvia Giagio'}, {'name': 'Andrea Turolla'}, {'name': 'Tiziano Innocenti'}, {'name': 'Stefano Salvioli'}, {'name': 'Giulia Gava'}, {'name': 'Paolo Pillastrini'}, {'name': 'Marco Vecchiato'}, {'name': 'Jodie Dakic'}, {'name': 'Helena Catherine Frawley'}, {'name': 'Kari Bo'}]\", 'abstract': \"<p>Background/aim: Several epidemiological studies have found a high prevalence of Pelvic Floor Dysfunction (PFD) among female athletes. However, according to several authors, these data could even be underestimated, both in research and clinical practice. Screening for potential PFD is often delayed and risk factors are not often evaluated. As a consequence, withdrawal from sport, negative influence on performance, worsening symptoms and unrecognized diagnosis may occur. \\nThe aim of our research is to develop a screening tool for pelvic floor dysfunction in female athletes useful for clinicians (musculoskeletal/sport physiotherapists, sports medicine physicians, team physicians) to guide referral to a PFD expert (e.g. pelvic floor/women's health physiotherapist, gynecologist, uro-gynecologist, urologist).\\nMethods: A 2-round modified Delphi study will be conducted to ascertain expert opinion on which combination of variables and risk factors should be included in the screening tool. \\nConclusion: The implementation of the present screening tool into clinical practice may facilitate the referral to a PFD expert for further assessment of the pelvic floor and therefore, to identify potential dysfunction and, eventually, the related treatment pathway.</p>\", 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476241', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Enumeration of Citrus endophytic bacterial communities based on illumine metagenomics technique', 'authors': \"[{'name': 'Sehrish Mushtaq'}, {'name': 'Muhammad Shafiq'}, {'name': 'Tehseen Ashraf'}, {'name': 'Muhammad Saleem Haider'}, {'name': 'Sagheer Atta'}]\", 'abstract': '<p>Citrus is a valuable crop in Pakistan because it is rich in vitamin C and antioxidants. Huanglongbing (HLB) has an influence on citrus production around the world caused by a bacterium <italic>“Candidatus liberibacter asmticus”</italic> (CLas), africanus and americanus. The structure and diversity of bacterial species in various ecosystems can be quickly examined using NGS. This approach is considerably quicker and more precise than outdated methods. Healthy or citrus greening infected leaf samples of Grapefruit, <italic>Citrus aurantifolia</italic>, and <italic>Citrus reticulata</italic> Blanco was used for diversity analysis. In this study high throughput, NGS technique was used to access the population of both cultivable and non-cultivable bacterial endophytes from citrus leaves, by using PCR amplicons of 16S rDNA sequences (V5–V7 regions) with Illumina Hi seq. As a result, a total number of 68,722 sequences were produced from the test samples. According to the NGS-based diversity classification, the most common genera of exploited bacterial endophytes were Proteobacteria, Firmicutes, Bacteroides, Cyanobacteria, and Actinobacteria. <italic>Citrus aurantifolia</italic> and <italic>Citrus paradisi</italic> showed almost equal diversity, whereas <italic>Citrus reticulata</italic> Blanco had a higher proportion of Proteobacteria and Cyanobacteria in their leaves. To determine alpha diversity (AD), additional data was analyzed using statistical indices such as Shannon, Chao1, and Simpson. According to the inverse Simpson diversity index, the abundance of the microbial population in six different citrus samples was 0.48, 0.567, and 0.163, respectively. The metagenomics of microbiota in plant tissues was successfully recorded by NGS technology, which can help us learn more about the interactions between plants and microbes. This research is the first step toward a better understanding of 16SrRNA-based metagenomics from citrus in Pakistan using Illumina (Hi seq) Technology.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.14.476308', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'MIA: An Open Source Standalone Deep Learning Application for Microscopic Image Analysis', 'authors': \"[{'name': 'Nils Körber'}]\", 'abstract': '<p>In recent years the amount of data generated by imaging techniques has grown rapidly along with increasing computational power and the development of deep learning algorithms. To address the need for powerful automated image analysis tools for a broad range of applications in the biomedical sciences, we present the Microscopic Image Analyzer (MIA). MIA combines a graphical user interface that obviates the need for programming skills with state-of-the-art deep learning algorithms for segmentation, object detection, and classification. It runs as a standalone, platform-independent application and is compatible with commonly used open source software packages. The software provides a unified interface for easy image labeling, model training and inference. Furthermore the software was evaluated in a public competition and performed among the top three for all tested data sets. The source code is available on <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/MIAnalyzer/MIA\">https://github.com/MIAnalyzer/MIA</ext-link>.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.12.22269137', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Cell-type-specific transcriptome architecture underlying the establishment and exacerbation of systemic lupus erythematosus', 'authors': \"[{'name': 'Masahiro Nakano'}, {'name': 'Mineto Ota'}, {'name': 'Yusuke Takeshima'}, {'name': 'Yukiko Iwasaki'}, {'name': 'Hiroaki Hatano'}, {'name': 'Yasuo Nagafuchi'}, {'name': 'Takahiro Itamiya'}, {'name': 'Junko Maeda'}, {'name': 'Ryochi Yoshida'}, {'name': 'Saeko Yamada'}, {'name': 'Aya Nishiwaki'}, {'name': 'Haruka Takahashi'}, {'name': 'Hideyuki Takahashi'}, {'name': 'Yuko Akutsu'}, {'name': 'Takeshi Kusuda'}, {'name': 'Hiroyuki Suetsugu'}, {'name': 'Lu Liu'}, {'name': 'Kwangwoo Kim'}, {'name': 'Xianyong Yin'}, {'name': 'So-Young Bang'}, {'name': 'Yong Cui'}, {'name': 'Hye-Soon Lee'}, {'name': 'Hirofumi Shoda'}, {'name': 'Xuejun Zhang'}, {'name': 'Sang-Cheol Bae'}, {'name': 'Chikashi Terao'}, {'name': 'Kazuhiko Yamamoto'}, {'name': 'Tomohisa Okamura'}, {'name': 'Kazuyoshi Ishigaki'}, {'name': 'Keishi Fujio'}]\", 'abstract': '<p>Systemic lupus erythematosus (SLE) is a complex and heterogeneous autoimmune disease involving multiple immune cells. A major hurdle to the elucidation of SLE pathogenesis is our limited understanding of dysregulated gene expression linked to various clinical statuses with a high cellular resolution. Here, we conducted a large-scale transcriptome study with 6,386 RNA sequencing data covering 27 immune cell types from 159 SLE and 89 healthy donors. We first profiled two distinct cell-type-specific transcriptomic signatures: disease-state and disease-activity signatures, reflecting disease establishment and exacerbation, respectively. We next identified candidate biological processes unique to each signature. This study suggested the clinical value of disease-activity signatures, which were associated with organ involvement and responses to therapeutic agents such as belimumab. However, disease-activity signatures were less enriched around SLE risk variants than disease-state signatures, suggesting that the genetic studies to date may not well capture clinically vital biology in SLE. Together, we identified comprehensive gene signatures of SLE, which will provide essential foundations for future genomic, genetic, and clinical studies.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.13.476146', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Overexpression of miR5505 enhanced drought and salt resistance in rice (Orayza sativa)', 'authors': \"[{'name': 'Yuanwei Fan'}, {'name': 'Jiankun Xie'}, {'name': 'Fantao Zhang'}]\", 'abstract': '<p>Rice is one of the most important crops in the world and half of the world population consumes it as their staple food. The abiotic stresses caused by drought, salt and other stresses have severely impacted rice production. MicroRNAs (miRNAs) are a type of small non-coding RNAs which widely reported as gene regulators, suppressing genes expression by degradation mRNA or translation inhibition. Previously, high-throughput sequencing has found a conserved miRNA miR5505 responding to drought stress in Dongxiang wild rice (DXWR). Several other studies also revealed that miR5505 was involved in rice stress responses. We further studied the effect of miRNA in drought and salt tolerance by overexpression it in rice. 2 in 18 successfully transformed transgenic lines with higher miR5505 expression were selected and then drought and salt resistance ability were evaluated. Both transgenic lines showed stronger drought and salt tolerance than wild-type (WT). Putative targets of miR5505 were identified by psRNATarget and several of them were found stress-related. RNA-seq found 1,980 differentially expressed genes (DEGs) in transgenic lines. Among them, 978 genes were down-regulated. Three genes were predicted by psRNATarget and two of them might be stress-related. We also found various environmental stress <italic>cis</italic>-acting elements in upstream of miR5505 promoter through Software PlantCARE. In all, we improved rice drought and salt tolerance by overexpressing miR5505, and the generated putative targets and <italic>cis</italic>-acting elements also suggested miR5505 might play important roles in the regulation of drought and salt responses.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.13.21268270', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Health Economics', 'covid': True, 'title': 'Health and Economic Consequences of Universal Paid Sick Leave Policies During the COVID-19 Pandemic', 'authors': \"[{'name': 'David Naimark'}, {'name': 'Juan David Rios'}, {'name': 'Sharmistha Mishra'}, {'name': 'Beate Sander'}, {'name': 'Petros Pechlivanoglou'}]\", 'abstract': '<sec><title>Importance</title><p>Universal paid sick-leave (PSL) policies have been implemented in jurisdictions to mitigate the spread of SARS-CoV-2. However empirical data regarding health and economic consequences of PSL policies is scarce.</p></sec><sec><title>Objective</title><p>To estimate effects of a universal PSL policy in Ontario, Canada’s most populous province.</p></sec><sec><title>Design</title><p>An agent-based model (ABM) to simulate SARS-CoV-2 transmission informed by data from Statistics Canada, health administrative sources, and from the literature.</p></sec><sec><title>Setting</title><p>Ontario from January 1<sup>st</sup> to May 1<sup>st</sup>, 2021.</p></sec><sec><title>Participants</title><p>A synthetic population (1 million) with occupation and household characteristics representative of Ontario residents (14.5 million).</p></sec><sec><title>Exposure</title><p>A base case of existing employer-based PSL alone versus the addition of a 3-or 10-day universal PSL policy to facilitate testing and self-isolation among workers infected with SARS-CoV-2 themselves or because of infected household members.</p></sec><sec><title>Main Outcome(s) and Measure(s)</title><p>Number of SARS-CoV-2 infections and COVID-19 hospitalizations, worker productivity, lost wages, and presenteeism (going to a workplace while infected).</p></sec><sec><title>Results</title><p>If a 3- and 10-day universal PSL were implemented over the 4-month study period, then compared with the base-case, the PSL policies were estimated to reduce cumulative SARS-CoV-2 cases by 85,531 (95% credible interval, CrI -2,484; 195,318) and 215,302 (81,500; 413,742), COVID-19 hospital admissions by 1,307 (-201; 3,205) and 3,352 (1,223; 6,528), numbers of workers forgoing wages by 558 (-327;1,608) and 7,406 (6,764; 8,072), and numbers of workers engaged in presenteeism by 24,499 (216; 54,170) and 279,863 (262,696; 295,449). Hours of productivity loss were estimated to be 10,854,379 (10,212,304; 11,465,635) in the base case, 17,446,525 (15,934,321; 18,854,683) in the 3-day scenario, and 26,127,165 (20,047,239; 29,875,161) in the 10-day scenario. Lost wages were $5,256,316 ($4,077,280; $6,804,983) and $12,610,962 ($11,463,128; $13,724,664) lower in the 3 day and 10 day scenarios respectively, relative to the base case.</p></sec><sec><title>Conclusions and Relevance</title><p>Expanded access to PSL is estimated to reduce total numbers of COVID-19 cases, reduce presenteeism of workers with SARS-CoV-2 at workplaces, and mitigate wage loss experienced by workers.</p></sec><sec><title>Competing interests</title><p>The authors have no competing interests relevant to this article to disclose.</p></sec><sec><title>Funding</title><p>Supported by COVID-19 Rapid Research Funding (C-291-2431272-SANDER). This research was further supported, in part, by a Canada Research Chair in Economics of Infectious Diseases held by Beate Sander (CRC-950-232429). The study sponsor had no role in the design, collection, analysis, interpretation of the data, manuscript preparation or the decision to submit for publication.</p></sec><sec><title>Author Contributions</title><p>Conceptualization: PP, JDR, BS, DN</p><p>Data Curation: PP, JDR, BS, DN</p><p>Formal Analysis: PP, JDR, DN</p><p>Methodology: PP, JDR, BS, DN</p><p>Supervision: PP, DN, BS</p><p>Validation: PP, JDR, BS, DN</p><p>First Draft: PP, JDR, BS, DN</p><p>Review and Edit</p><p>PP, JDR, BS, DN</p></sec><sec><title>Key points</title><sec><title>Question</title><p>What could be the health and economic consequence of more generous paid sick leave policies in the context of the COVID-19 pandemic?</p></sec><sec><title>Findings</title><p>More generous policies are estimated to reduce SARS-CoV-2 infections (and thus COVID-19 hospitalizations), lost wages and presence of individuals with infection at workplaces.</p></sec><sec><title>Meaning</title><p>More generous paid sick leave can be a valuable addition to other COVID-19 public health interventions.</p></sec></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.10.475617', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Pervasive conditional selection of driver mutations and modular epistasis networks in cancer', 'authors': \"[{'name': 'Jaime Iranzo'}, {'name': 'George Gruenhagen'}, {'name': 'Jorge Calle-Espinosa'}, {'name': 'Eugene V. Koonin'}]\", 'abstract': '<p>Cancer driver mutations often display mutual exclusion or co-occurrence, underscoring the key role of epistasis in carcinogenesis. However, estimating the magnitude of epistatic interactions and their quantitative effect on tumor evolution remains a challenge. We developed a method to quantify <italic>COnditional SELection on the Excess of Nonsynonymous Substitutions</italic> (<italic>Coselens</italic>) in cancer genes. <italic>Coselens</italic> infers the number of drivers per gene in different partitions of a cancer genomics dataset using covariance-based mutation models and determines whether coding mutations in a gene affect selection for drivers in any other gene. Using <italic>Coselens</italic>, we identified 296 conditionally selected gene pairs across 16 cancer types in the TCGA dataset. Conditional selection accounts for 25-50% of driver substitutions in tumors with &gt;2 drivers. Conditionally co-selected genes form modular networks, whose structures challenge the traditional interpretation of within-pathway mutual exclusivity and across-pathway synergy, suggesting a more complex scenario, where gene-specific across-pathway interactions shape differentiated cancer subtypes.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.17.476647', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genome sequence data reveal at least two distinct incursions of the tropical race 4 (TR4) variant of Fusarium wilt into South America', 'authors': \"[{'name': 'Paula H. Reyes-Herrera'}, {'name': 'Eliana Torres-Bedoya'}, {'name': 'Diana Lopez-Alvarez'}, {'name': 'Diana Burbano-David'}, {'name': 'Sandra L. Carmona'}, {'name': 'Daniel Patrick Bebber'}, {'name': 'David John Studholme'}, {'name': 'Monica Betancourt'}, {'name': 'Mauricio Soto-Suarez'}]\", 'abstract': '<p>The global banana industry is threatened by one of the most devastating diseases: Fusarium wilt (FWB). FWB is caused by the soil-borne fungus Fusarium oxysporum f. sp. cubense (Foc), which almost annihilated the banana production in the late 1950s. A new strain of Foc, known as tropical race 4 (TR4), attacks a wide range of banana varieties including Cavendish clones which are the source of 99% of banana exports. In 2019, Foc TR4 was reported in Colombia, and more recently (2021) in Peru. In this study, we sequenced three fungal isolates identified as Foc TR4 from La Guajira (Colombia) and compared them against 19 whole-genome sequences of Foc TR4 publicly available, including four genome sequences recently released from Peru. To understand the genetic relatedness of the Colombian Foc TR4 isolates and those from Peru, we conducted a phylogenetic analysis based on a genome-wide set of single nucleotide polymorphisms (SNPs). Additionally, we compared the genomes of the 22 available Foc TR4 isolates looking for the presence-absence of gene polymorphisms and genomic regions. Our results reveal that (i) the Colombian and Peruvian isolates are genetically distant, which could be better explained by independent incursions of the pathogen to the continent, and (ii) there is a high correspondence between the genetic relatedness and geographic origin of Foc TR4. The profile of present/absent genes and the distribution of missing genomic regions showed a high correspondence to the clades recovered in the phylogenetic analysis, supporting the results obtained by SNP-based phylogeny.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.13.476211', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Enhancement of RecET-mediated in vivo linear DNA assembly by a xonA mutation', 'authors': \"[{'name': 'James A. Sawitzke'}, {'name': 'Nina Costantino'}, {'name': 'Ellen Hutchinson'}, {'name': 'Lynn C. Thomason'}, {'name': 'Donald L. Court'}]\", 'abstract': '<p>Assembly of intact, replicating plasmids from linear DNA fragments introduced into bacterial cells, i.e. <italic>in vivo</italic> cloning, is a facile genetic engineering technology that avoids many of the problems associated with standard <italic>in vitro</italic> cloning. Here we report characterization of various parameters of <italic>in vivo</italic> linear DNA assembly mediated by either the RecET recombination system or the bacteriophage λ Red recombination system. As previously observed, RecET is superior to Red for this reaction when the terminal homology is 50 bases. Deletion of the <italic>E. coli xonA</italic> gene, encoding Exonuclease I, a 3’→5’ single-strand DNA exonuclease, substantially improves the efficiency of <italic>in vivo</italic> linear DNA assembly for both systems. Deletion of ExoI function allowed robust RecET assembly of six DNA segments to create a functional plasmid. The linear DNAs are joined accurately with very few errors. This discovery provides a significant improvement to previously reported <italic>in vivo</italic> linear DNA assembly technologies.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.10.475553', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Detection of bifenthrin, bifenazate, etoxazole resistance in Tetranychus urticae collected from mint fields and hop yards using targeted sequencing and TaqMan approaches', 'authors': \"[{'name': 'Silas Shumate'}, {'name': 'Maggie Haylett'}, {'name': 'Brenda Nelson'}, {'name': 'Nicole Young'}, {'name': 'Kurt Lamour'}, {'name': 'Doug Walsh'}, {'name': 'Benjamin Bradford'}, {'name': 'Justin Clements'}]\", 'abstract': '<p><italic>Tetranychus urticae</italic> (Koch) is an economically important pest of many agricultural commodities in the Pacific Northwest. Multiple miticides are currently registered for control including abamectin, bifenazate, bifenthrin, and extoxazole. However, populations of <italic>Tetranychus urticae</italic> have developed miticide resistance through multiple mechanisms, in many different growing regions. Producers of agricultural commodities where <italic>Tetranychus urticae</italic> infestations are problematic rely on integrated pest management tools to determine optimal control methods. Within this species multiple single nucleotide polymorphisms have been documented in different genes which are associated with miticide resistance phenotypes. The detection of these mutations through TaqMan qPCR has been suggested as a practical, quick, and reliable tool to inform agricultural producers of miticide resistance phenotypes present within their fields and have potential utility for making appropriate miticide application and integrated pest management decisions. Within this investigation we examined the use of a TaqMan qPCR-based approach to determine miticide resistance genotypes in field-collected populations of <italic>Tetranychus urticae</italic> from mint fields and hop yards in the Pacific Northwest of the United States and confirmed the results with a multiplex targeted sequencing. The results suggest the TaqMan approach accurately genotypes <italic>Tetranychus urticae</italic> populations collected from agricultural fields. The interpretation of the results, however, provide additional challenges for integrated pest management practitioners, including making miticide application recommendations where populations of <italic>Tetranychus urticae</italic> are a mix of resistant and wildtype individuals.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475737', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Corrective feedback control of competing neural network with entire connections', 'authors': \"[{'name': 'Uramogi Wang'}]\", 'abstract': '<p>Continuous persist activity of the competitive network is related to many functions, such as working memory, oculomotor integrator and decision making. Many competition models with mutual inhibition structures achieve activity maintenance via positive feedback, which requires meticulous fine tuning of the network parameters strictly. Negative derivative feedback, according to recent research, might represent a novel mechanism for sustaining neural activity that is more resistant to multiple neural perturbations than positive feedback. Many classic models with only mutual inhibition structure are not capable of providing negative derivative feedback because double-inhibition acts as a positive feedback loop, and lack of negative feedback loop that is indispensable for negative derivative feedback. Here in the proposal, we aim to derive a new competition network with negative derivative feedback. The network is made up of two symmetric pairs of EI populations that the four population are completely connected. We conclude that the negative derivative occurs in two circumstances, in which one the activity of the two sides is synchronous but push-pull-like in the other, as well as the switch of two conditions in mathematical analysis and numerical simulation.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475724', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'DNA methylation meta-analysis confirms the division of the genome into two functional groups', 'authors': \"[{'name': 'Lev Salnikov'}, {'name': 'Saveli Goldberg'}, {'name': 'Parvathy Sukumaran'}, {'name': 'Eugene Pinsky'}]\", 'abstract': '<p>Based on a meta-analysis of human genome methylation data, we tested a theoretical model in which aging is explained by the redistribution of limited resources in cells between two main tasks of the organism: its self-sustenance based on the function of the housekeeping gene group (HG) and functional differentiation, provided by the (IntG) integrative gene group. A meta-analysis of methylation of 100 genes, 50 in the HG group and 50 in IntG, showed significant differences (p&lt;0.0001) between our groups in the level of absolute methylation values of genes bodies and its promoters. We showed a reliable decrease of absolute methylation values in IntG with rising age in contrast to HG, where this level remained constant. The one-sided decrease in methylation in the IntG group is indirectly confirmed by the dispersion data analysis, which also decreased in the genes of this group. The imbalance between HG and IntG in methylation levels suggests that this IntG-shift is a side effect of the ontogenesis grownup program and the main cause of aging. The theoretical model of functional genome division also suggests the leading role of slow dividing and post mitotic cells in triggering and implementing the aging process.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.10.475598', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Noise source importance in linear stochastic models of biological systems that grow, shrink, wander, or persist', 'authors': \"[{'name': 'Alexander Strang'}, {'name': 'William Huffmyer'}, {'name': 'Hilary Rollins'}, {'name': 'Karen C. Abbott'}, {'name': 'Peter J. Thomas'}]\", 'abstract': '<p>While noise is an important factor in biology, biological processes often involve multiple noise sources, whose relative importance can be unclear. Here we develop tools that quantify the importance of noise sources in a network based on their contributions to variability in a quantity of interest. We generalize the edge importance measures proposed by Schmidt and Thomas [1] for first-order reaction networks whose steady-state variance is a linear combination of variance produced by each directed edge. We show that the same additive property extends to a general family of stochastic processes subject to a set of linearity assumptions, whether in discrete or continuous state or time. Our analysis applies to both expanding and contracting populations, as well as populations obeying a martingale (“wandering”) at long times. We show that the original Schmidt-Thomas edge importance measure is a special case of our more general measure, and is recovered when the model satisfies a conservation constraint (“persists”). In the growing and wandering cases we show that the choice of observables (measurements) used to monitor the process does not influence which noise sources are important at long times. In contrast, in the shrinking or persisting case, which noise sources are important depends on what is measured. We also generalize our measures to admit models with affine moment update equations, which admit additional limiting scenarios, and arise naturally after linearization. We illustrate our results using examples from cell biology and ecology: (i) a model for the dynamics of the inositol trisphospate receptor, (ii) a model for an endangered population of white-tailed eagles, and (iii) a model for wood frog dispersal.</p><sec><title>Author summary</title><p>Biological processes are frequently subject to an ensemble of independent noise sources. Noise sources produce fluctuations that propagate through the system, driving fluctuations in quantities of interest such as population size or ion channel configuration. We introduce a measure that quantifies how much variability each noise source contributes to any given quantity of interest. Using these methods, we identify which binding events contribute significantly to fluctuations in the state of a molecular signalling channel, which life history events contribute the most variability to an eagle population before and after a successful conservation effort rescued the population from the brink of extinction, and which dispersal events, at what times, matter most to variability in the recolonization of a series of ponds by wood frogs after a drought.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.22269032', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': False, 'title': 'Spatial clustering in vaccination hesitancy: the role of social influence and social selection', 'authors': \"[{'name': 'Lucila Gisele Alvarez Zuzek'}, {'name': 'Casey M Zipfel'}, {'name': 'Shweta Bansal'}]\", 'abstract': '<p>The phenomenon of vaccine hesitancy behavior has gained ground over the last three decades, jeopardizing the maintenance of herd immunity. This behavior tends to cluster spatially, creating pockets of unprotected sub-populations that can be hotspots for outbreak emergence. What remains less understood are the social mechanisms that can give rise to spatial clustering in vaccination behavior, particularly at the landscape scale. We focus on the presence of spatial clustering, and aim to mechanistically understand how different social processes can give rise to this phenomenon. In particular, we propose two hypotheses to explain the presence of spatial clustering: (i) social selection, in which vaccine-hesitant individuals share socio-demographic traits, and clustering of these traits generates spatial clustering in vaccine hesitancy; and (ii) social influence, in which hesitant behavior is contagious and spreads through neighboring societies, leading to hesitant clusters. Adopting a theoretical spatial network approach, we explore the role of these two processes in generating patterns of spatial clustering in vaccination behaviors under a range of spatial structures. We find that both processes are independently capable of generating spatial clustering, and the more spatially structured the social dynamics in a society are, the higher spatial clustering in vaccine-hesitant behavior it realizes. Together, we demonstrate that these processes result in unique spatial configurations of hesitant clusters, and we validate our models on both processes with fine-grain empirical data on vaccine hesitancy, social determinants, and social connectivity in the US. Finally, we propose, and evaluate the effectiveness of, two novel intervention strategies to diminish hesitant behavior. Our generative modeling approach informed by unique empirical data provides insights on the role of complex social processes in driving spatial heterogeneity in vaccine hesitancy.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475769', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Efficient inactivation of African swine fever virus by a highly complexed iodine combined with compound organic acids', 'authors': \"[{'name': 'Mengnan Qi'}, {'name': 'Li Pan'}, {'name': 'Ying Gao'}, {'name': 'Miao Li'}, {'name': 'Yanjin Wang'}, {'name': 'Lian-Feng Li'}, {'name': 'Chen Ji'}, {'name': 'Yuan Sun'}, {'name': 'Hua-Ji Qiu'}]\", 'abstract': '<p>African swine fever (ASF) is a highly contagious disease with high morbidity and mortality caused by African swine fever virus (ASFV). Cleaning and disinfection remain one of the most effective biosecurity measures to prevent and control the spread of ASFV. In this study, we evaluated the inactivation effects of highly complexed iodine (HPCI) combined with compound organic acids (COAs) against ASFV under different conditions. The results showed that the inactivation rates of the disinfectants on the reporter ASFV increased in dose- and time-dependent manners, the best inactivation effects were obtained when the compatibility ratio of HPCI and COAs was 5:1 at 25°C. Furthermore, there were no significant differences by comparing the efficacy of HPCI combined with COAs (HPCI+COAs) in inactivating wild-type ASFV and the reporter ASFV (<italic>P</italic> &gt; 0.05). ASFV of 10<sup>4.0</sup> TCID<sub>50</sub>/mL was completely inactivated by 0.13% HPCI (0.0065% effective iodine), 0.06% COAs or 0.13% HPCI+COAs (approximately 0.0054% effective iodine), respectively, while 10<sup>6.0</sup> TCID<sub>50</sub>/mL ASFV was completely inactivated by 1.00% HPCI (0.05% effective iodine), 0.50% COAs or 1.00% HPCI+COAs (0.042% effective iodine), respectively. Therefore, HPCI+COAs had synergistic effects to inactivate ASFV. This study demonstrated that HPCI+COAs could rapidly and efficiently inactivate ASFV and represent an effective compound disinfectant for the control of ASF.</p><sec><title>IMPORTANCE</title><p>African swine fever (ASF) is a highly contagious disease with high morbidity and mortality caused by African swine fever virus (ASFV). Due to the lack of commercial vaccines and treatment available for ASF, effective disinfectants and the proper use of them are extremely essential to inactivate ASFV. The significance of this research is in searching for an ideal disinfectant that not only has the advantages of low toxicity and non-pollution but also can inactivate ASFV rapidly and efficiently. In this study, we proved that HPCI+COAs not only exhibited low cytotoxicity, but also could completely inactivate ASFV within 5 min at 4°C, 25°C and 37°C. In addition, HPCI+COAs had synergistic effects on inactivated ASFV. Thus, HPCI +COAs could be used as an effective disinfectant for the control of ASF.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.13.476185', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Forebrain glucocorticoid receptor overexpression alters behavioral encoding of hippocampal CA1 pyramidal cells in mice', 'authors': \"[{'name': 'Joanna L Spencer-Segal'}, {'name': 'Swapnil Gavade'}, {'name': 'Qiang Wei'}, {'name': 'Colin Johnston'}, {'name': 'Savannah Kounelis'}, {'name': 'Klaudia Laborc'}, {'name': 'Huda Akil'}]\", 'abstract': '<p>Stress hormone signaling via the glucocorticoid receptor (GR) modulates vulnerability to stress-related disorders, but whether GR influences how the brain encodes contextual experience is unknown. Mice with lifelong GR overexpression in forebrain glutamatergic neurons (GRov) show increased sensitivity to environmental stimuli. This phenotype is developmentally programmed and associated with profound changes in hippocampal gene expression. We hypothesized that GR overexpression influences hippocampal encoding of experiences. To test our hypothesis, we performed in vivo microendoscopic calcium imaging of 1359 dorsal CA1 pyramidal cells in freely behaving male and female WT and GRov mice during exploration of a novel open field. We compared calcium amplitude and event rate as well as sensitivity to center location and mobility between genotypes. GRov neurons exhibited higher average calcium activity than WT neurons in the novel open field. While most neurons showed sensitivity to center location and/or mobility, GRov neurons were more likely to be sensitive to center location and less likely to be sensitive to mobility, as compared to WT neurons. More than one-third of behavior-selective GRov neurons were uniquely sensitive to location without mobility sensitivity; these uniquely center-sensitive neurons were rare in WT. We conclude that dorsal CA1 pyramidal cells in GRov mice show increased activity in a novel environment and preferentially encode emotionally salient behavior. This heightened sensitivity to a novel environment and preferential encoding of emotionally salient elements of experience could underlie differential stress vulnerability in humans with increased glucocorticoid sensitivity.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.10.475751', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Type I IFN promotes pathogenic inflammatory monocyte maturation during H5N1 infection', 'authors': \"[{'name': 'Slim Fourati'}, {'name': 'David Jimenez-Morales'}, {'name': 'Judd Hultquist'}, {'name': 'Max W. Chang'}, {'name': 'Christopher Benner'}, {'name': 'Nevan Krogan'}, {'name': 'Lars Pache'}, {'name': 'Sumit Chanda'}, {'name': 'Rafick-Pierre Sekaly'}, {'name': 'Adolfo García-Sastre'}, {'name': 'Melissa B. Uccellini'}]\", 'abstract': '<p>Ly6C<sup>hi</sup> inflammatory monocytes show high IFN responses, and contribute to both protective and pathogenic functions following influenza virus infection. In order to understand the significance of IFN responses in this subset, we examined monocytes during infection with a lethal H5N1 virus that induces high levels of IFN and a low-pathogenicity H1N1 virus that induces low levels of IFN. We show that H5N1 infection results in early recruitment of high numbers of Ly6C<sup>hi</sup> monocytes and induction of chemokines and <italic>Ifnb1</italic>. Using unbiased transcriptomic and proteomic approaches, we also find that monocytes are significantly enriched during H5N1 infection and are associated with chemokine and IFN signatures in mice, and with severity of symptoms after influenza virus infection in humans. Recruited Ly6C<sup>hi</sup> monocytes subsequently become infected in the lung, produce IFN-β, and mature into FasL<sup>+</sup> monocyte-derived cells (FasL<sup>+</sup>MCs) expressing dendritic cell markers. Both <italic>Ccr2<sup>-/-</sup></italic> and <italic>Fasl<sup>gld</sup></italic> mice are protected from lethal infection, indicating that monocytes contribute to pathogenesis. Global loss of type I and type III IFN signaling in <italic>Stat2<sup>-/-</sup></italic> mice results in loss of monocyte recruitment, likely reflecting a requirement for IFN-dependent chemokine induction. Here we show that IFN is not directly required for monocyte recruitment on an IFN-sufficient background, but is required for maturation to FasL<sup>+</sup>MCs. Loss of IFN signaling skews to a Ly6C<sup>lo</sup> phenotype associated with tissue repair, suggesting that IFN signaling in monocytes is a critical determinant of influenza virus pathogenesis.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475678', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Machine learning for exploring microbial inter-kingdom associations in Cystic Fibrosis and Bronchiectasis', 'authors': \"[{'name': 'L. Cuthbertson'}, {'name': 'J. Ish-Horowicz'}, {'name': 'I.C. Felton'}, {'name': 'P. James'}, {'name': 'E. Turek'}, {'name': 'M.J. Cox'}, {'name': 'M.R. Loebinger'}, {'name': 'N.J. Simmonds'}, {'name': 'S. Filippi'}, {'name': 'M.F. Moffatt'}, {'name': 'W.O.C. Cookson'}]\", 'abstract': '<sec><title>Background</title><p>Cystic fibrosis (CF) and non-CF bronchiectasis (BX) are lung diseases characterised by severe chronic infections. Fungal and bacterial components of infection are both recognized. Recent molecular investigation of sputum from patients with CF and BX has revealed a complex mycobiome. However, little is known about how fungal and bacterial organisms interact or whether the interactions impact on disease outcomes.</p></sec><sec><title>Methods</title><p>Quantitative PCR and next generation sequencing of ITS2 and 16S rRNA gene was carried out on 107 patients with CF and BX and defined clinical fungal infection status. Fungal and bacterial communities were explored using supervised and unsupervised machine learning to understand associations between fungal and bacterial communities and their relationship to disease.</p></sec><sec><title>Results</title><p>Fungal and bacterial communities both had significantly higher biomass and lower diversity in CF compared to BX patients. Random forest modelling demonstrated that the fungal and bacterial communities were distinct between CF and BX patients. Within the CF group, bacterial communities contained no predictive signal for fungal disease status. Neither bacterial nor fungal community composition were predictive of the presence of CF pulmonary exacerbation (CFPE). Intra-kingdom correlations were far stronger than those between the two kingdoms. Dirichlet mixture components analysis identified two distinct clusters of bacteria related to the relative abundance of <italic>Pseudomonas</italic>. Fungal community composition contained no predictive signal for bacterial clusters.</p></sec><sec><title>Conclusions</title><p>Clear changes in diversity were observed between patients with different clinical disease status. Although our results demonstrate that bacterial community composition differs in the presence of fungal disease, no direct relationship between bacterial and fungal OTUs was found.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.475705', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Single-nucleus RNA-seq of normal-appearing brain regions in relapsing-remitting vs. secondary progressive multiple sclerosis', 'authors': \"[{'name': 'Yasuyuki Kihara'}, {'name': 'Yunjiao Zhu'}, {'name': 'Deepa Jonnalagadda'}, {'name': 'William Romanow'}, {'name': 'Carter Palmer'}, {'name': 'Benjamin Siddoway'}, {'name': 'Richard Rivera'}, {'name': 'Ranjan Dutta'}, {'name': 'Bruce D. Trapp'}, {'name': 'Jerold Chun'}]\", 'abstract': '<p>Multiple sclerosis (MS) is an immune-mediated demyelinating disease that alters central nervous system (CNS) functions. Relapsing-remitting MS (RRMS) is the most common form, which can transform into secondary-progressive MS (SPMS) that is associated with progressive neurodegeneration. Single-nucleus RNA sequencing (snRNA-seq) of MS lesions identified disease-related transcriptomic alterations; however, their relationship to non-lesioned MS brain regions has not been reported and which could identify prodromal or other disease susceptibility signatures. Here, snRNA-seq was used to generate high-quality RRMS vs. SPMS datasets of 33,197 nuclei from 8 normal-appearing MS brains, which revealed divergent cell type-specific changes. Notably, SPMS brains downregulated astrocytic sphingosine kinases (<italic>SPHK1/2</italic>) – the enzymes required to phosphorylate and activate the MS drug, fingolimod. This reduction was modeled with astrocyte-specific <italic>Sphk1/2</italic> null mice in which fingolimod lost activity, supporting functionality of observed transcriptomic changes. These data provide an initial resource for studies of single cells from non-lesioned RRMS and SPMS brains.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.476011', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Scientific Communication and Education', 'covid': False, 'title': 'No One Is Leaving This Time: Appraisal of Awareness and Knowledge of Port Health Officers on International Health Regulation IHR (2005)', 'authors': \"[{'name': 'Yusuf Babatunde Adiama'}, {'name': 'Solomon Olayinka Adewoye'}, {'name': 'Opasola Afolabi Olaniyi'}, {'name': 'Habeeb Modupe Lateefat'}, {'name': 'Abdullahi Ahmed'}, {'name': 'Morufu Olalekan Raimi'}]\", 'abstract': '<sec><title>Background</title><p>Historically, ships have played an important role in transmitting infectious diseases around the world. The spread of cholera pandemics in the 19<sup>th</sup> century was thought to be linked to trade routes and facilitated by merchant shipping. The international maritime traffic of people and goods has often contributed to the spread of pathogens affecting public health.</p></sec><sec><title>Objectives</title><p>To assess level of awareness and knowledge of international Health regulation (IHR 2005) content among port health officer</p></sec><sec><title>Methods</title><p>The study design was descriptive cross-sectional evaluation, questionnaires were used to capture the respondents’ knowledge, awareness and sanitary condition of ship in accordance with (IHR 2005)</p></sec><sec><title>Results</title><p>On awareness and knowledge, Majority of the respondent (77.1 %) demonstrate good awareness of the IHR (2005), while 22.9% had not and some even testified of hearing the said document for the first time. Despite the fact that majority of respondent were aware but only 24.6% of them can actually demonstrate good knowledge of IHR (2005) and its intent to protect and prevent spread of disease along the international route.</p></sec><sec><title>Conclusion</title><p>There is need to improve the knowledge of port health officers by expand training and guidance on application of the IHR’s to frontline officer at point of entries. Also ensure more thorough inspection and avoid influence of ship agent during inspection of ship.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.15.476372', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structural Basis for the Calmodulin-Mediated Activation of eEF-2K', 'authors': \"[{'name': 'Andrea Piserchio'}, {'name': 'Eta A Isiroho'}, {'name': 'Kimberly Long'}, {'name': 'Amanda L Bohanon'}, {'name': 'Eric A Kumar'}, {'name': 'Nathan Will'}, {'name': 'David Jeruzalmi'}, {'name': 'Kevin N Dalby'}, {'name': 'Ranajeet Ghose'}]\", 'abstract': '<p>Translation is a highly energy consumptive process tightly regulated for optimal protein quality and adaptation to energy and nutrient availability. A key facilitator of this process is the α-kinase eEF-2K that specifically phosphorylates the GTP-dependent translocase eEF-2, thereby reducing its affinity for the ribosome and suppressing the elongation phase of protein synthesis. eEF-2K activation requires calmodulin binding and auto-phosphorylation at the primary stimulatory site, T348. Biochemical studies have predicted that calmodulin activates eEF-2K through a unique allosteric process mechanistically distinct from other calmodulin-dependent kinases. Here we resolve the atomic details of this mechanism through a 2.3 Å crystal structure of the heterodimeric complex of calmodulin with the functional core of eEF-2K (eEF-2K<sub>TR</sub>). This structure, which represents the activated T348-phosphorylated state of eEF-2K<sub>TR</sub>, highlights how through an intimate association with the calmodulin C-lobe, the kinase creates a spine that extends from its N-terminal calmodulin-targeting motif through a conserved regulatory element to its active site. Modification of key spine residues has deleterious functional consequences.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.476144', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Bifidobacterium longum attenuates ovariectomy-induced bone loss via modulating the Immunoporotic “Breg-Treg-Th17” cell axis', 'authors': \"[{'name': 'Leena Sapra'}, {'name': 'Niti Shokeen'}, {'name': 'Konica Gupta'}, {'name': 'Chaman Saini'}, {'name': 'Asha Bhardwaj'}, {'name': 'Mary Mathew'}, {'name': 'Pradyumna K. Mishra'}, {'name': 'Naibedya Chattopadhyay'}, {'name': 'Bhupendra Verma'}, {'name': 'Rupesh K. Srivastava'}]\", 'abstract': '<p>Discoveries in the last few years have emphasized the existence of an enormous breadth of communication between osteo-immune system. These discoveries fuel novel approaches for the treatment of several bone-pathologies including osteoporosis, an inflammatory bone anomaly affecting more than 500 million people globally. <italic>Bifidobacterium longum</italic> (BL) is preferred probiotic of choice due to its varied immunomodulatory potential in alleviating various inflammatory diseases. Here, we evaluate the effect of BL in ovariectomy (ovx)-induced post-menopausal osteoporotic mice model. Our in vitro findings reveal that BL suppresses the differentiation and functional activity of RANKL-induced osteoclastogenesis in both mouse bone marrow cells and human PBMCs. Our in vivo data clearly establish that BL exhibits osteoprotective potential via modulating the “immunoporotic” Breg-Treg-Th17 cell-axis. Furthermore, µCT and bone mechanical strength data support that BL supplementation significantly enhanced bone mass and strength, and improved microarchitecture in ovx mice. Remarkably, alteration in frequencies of CD19<sup>+</sup>CD1d<sup>hi</sup>CD5<sup>+</sup> Bregs, CD4<sup>+</sup>Foxp3<sup>+</sup>IL-10<sup>+</sup> Tregs, and CD4<sup>+</sup>Rorγt<sup>+</sup>IL-17<sup>+</sup> Th17 immune cells in distinct lymphoid organs along with serum-cytokine data (enhanced anti-osteoclastogenic cytokines IFN-γ and IL-10 and reduced osteoclastogenic-cytokines IL-6, IL-17, and TNF-α) strongly support the immunomodulatory potential of BL. Altogether our findings establish a novel osteo-protective and immunoporotic potential of BL in augmenting bone health under osteoporotic conditions.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.12.476037', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Impaired functional connectivity of the hippocampus in murine models of NMDA-receptor antibody associated pathology', 'authors': \"[{'name': 'Joseph Kuchling'}, {'name': 'Betty Jurek'}, {'name': 'Mariya Kents'}, {'name': 'Jakob Kreye'}, {'name': 'Christian Geis'}, {'name': 'Jonathan Wickel'}, {'name': 'Susanne Mueller'}, {'name': 'Stefan Paul Koch'}, {'name': 'Philipp Boehm-Sturm'}, {'name': 'Harald Prüss'}, {'name': 'Carsten Finke'}]\", 'abstract': '<sec><title>Introduction</title><p>While decreased hippocampal connectivity and disruption of functional networks are established MRI features in human anti-N-methyl-D-aspartate receptor (NMDAR) encephalitis, the underlying pathophysiology for brain network alterations remains poorly understood. Application of patient-derived monoclonal antibodies against the NR1 subunit of the NMDAR allows for the investigation of potential functional connectivity alterations in experimental murine NMDAR antibody disease models.</p></sec><sec><title>Objective</title><p>To explore functional connectivity changes in NR1 antibody mouse models using resting-state functional MRI (rs-fMRI).</p></sec><sec><title>Methods</title><p>Adult C57BL/6J mice (n=10) were intrathecally injected with a recombinant human NR1 antibody over 14 days and then studied using rs-fMRI at 7 Tesla. In addition, a newly established mouse model with <italic>in utero</italic> exposure to a human recombinant NR1 antibody characterized by a neurodevelopmental disorder (NR1-offspring) was investigated with rs-fMRI at the age of 8 weeks (n=15) and 10 months (n=14). Mice exposed to isotype-matched control antibodies served as controls. Independent component analysis (ICA) and dual regression analysis were performed to compare functional connectivity between NMDAR antibody mouse models and control mice.</p></sec><sec><title>Results</title><p>Adult NR1-antibody injected mice showed significantly impaired functional connectivity within the dentate gyrus of the left hippocampus in comparison to controls, resembling impaired hippocampal functional connectivity patterns observed in human patients with NMDAR encephalitis. Similarly, analyses showed significantly reduced functional connectivity in the dentate gyrus in NR1-offspring compared after 8 weeks, and impaired connectivity in the dentate gyrus and CA3 hippocampal subregion in NR1-offspring at the age of 10 months.</p></sec><sec><title>Conclusion</title><p>Functional connectivity changes within the hippocampus resulting from both direct application and <italic>in utero</italic> exposure to NMDAR antibodies can be modeled in experimental murine systems. With this translational approach, we successfully reproduced functional MRI alterations previously observed in human NMDAR encephalitis patients. Future experimental studies will identify the detailed mechanisms that cause functional network alterations and may eventually allow for non-invasive monitoring of disease activity and therapeutic effects in autoimmune encephalitis.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.22269182', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Nephrology', 'covid': True, 'title': 'Immunosuppression impaired the immunogenicity of inactivated SARS-CoV-2 vaccine in non-dialysis kidney disease patients', 'authors': \"[{'name': 'Yuemiao Zhang'}, {'name': 'Xingzi Liu'}, {'name': 'Miaomiao Lin'}, {'name': 'Jincan Zan'}, {'name': 'Yitong Hu'}, {'name': 'Xiangqiu Wang'}, {'name': 'Wenqi Wu'}, {'name': 'Taicheng Zhou'}, {'name': 'Hong Zhang'}, {'name': 'Jicheng Lv'}, {'name': 'Li Yang'}, {'name': 'Zijie Zhang'}]\", 'abstract': '<p>Patients with chronic kidney disease (CKD) are at higher risk for coronavirus disease 2019 (COVID-19)-related morbidity and mortality. However, a significant portion of CKD patients showed hesitation toward vaccination in telephone survey of our center. Yet no serial data available on humoral response in patients with CKD, especially those on immunosuppression. We conducted a pilot, prospective study to survey the safety and humoral response to inactivated SARS-CoV-2 vaccine in CKD patients receiving a 2-dose immunization of inactivated SARS-CoV-2 vaccine. We found the neutralizing antibody titers in CKD patients was significantly lower than that in healthy controls, hypertension patients, and diabetes patients. Notably, immunosuppressive medication rather than eGFR levels or disease types  showed effect on the reduction of immunogenicity. Interestingly, a third dose significantly boosted neutralizing antibody in CKD patients while immunosuppressants impeded the boosting effects. In conclusion, our data demonstrates that CKD patients, even for those on immunosuppression treatment, can benefit from a third vaccination boost by improving their humoral immunity.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476140', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Histone chaperones exhibit conserved functionality in nucleosome remodeling', 'authors': \"[{'name': 'P. Buzón'}, {'name': 'A. Velázquez-Cruz'}, {'name': 'K. González-Arzola'}, {'name': 'A. Díaz-Quintana'}, {'name': 'I. Díaz-Moreno'}, {'name': 'W.H. Roos'}]\", 'abstract': '<p>Chromatin homeostasis mediates some of the most fundamental processes in the eukaryotic cell. In this regard, histone chaperones have emerged as major regulatory factors during DNA replication, repair, and transcription. However, the dynamic nature of these processes has severely impeded their characterization at the molecular level. Here we apply single-molecule probing by fluorescence optical tweezers to follow histone chaperone dynamics in real-time. The molecular action of SET/template-activating factor-Iβ and nucleophosmin 1—representing the two most common histone chaperone folds—were examined using both nucleosomes and isolated core histones. We show that these chaperones present binding specificity for partially dismantled nucleosomes and are able to recognize and disrupt non-native histone-DNA interactions. Furthermore, we reveal that cytochrome <italic>c</italic> inhibition of histone chaperones is coupled to chaperone accumulation on DNA-bound histones. Our single-molecule approach shows that despite the drastically different structures of these chaperones, they present conserved modes of action mediating nucleosome remodeling.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.12.475960', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'BECN1 F121A mutation increases autophagic flux in aged mice and improves aging phenotypes in an organ-dependent manner', 'authors': \"[{'name': 'Salwa Sebti'}, {'name': 'Zhongju Zou'}, {'name': 'Michael U. Shiloh'}]\", 'abstract': '<p>Autophagy is necessary for lifespan extension in multiple model organisms and autophagy dysfunction impacts age-related phenotypes and diseases. Introduction of an F121A mutation into the essential autophagy protein BECN1 constitutively increases basal autophagy in young mice and reduces cardiac and renal age-related changes in longer-lived <italic>Becn1<sup>F121A</sup></italic> mutant mice. However, both autophagic and lysosomal activity have been described to decline with age. Thus, whether autophagic flux is maintained during aging and whether it is enhanced in <italic>Becn1<sup>F121A</sup></italic> mice is unknown. Here we demonstrate that old wild type mice maintained functional autophagic flux in heart, kidney and skeletal muscle but not liver, and old <italic>Becn1<sup>F121A</sup></italic> mice had increased autophagic flux in those same organs compared to wild type. In parallel, <italic>Becn1<sup>F121A</sup></italic> mice were not protected against age-associated hepatic phenotypes but demonstrated reduced skeletal muscle fiber atrophy. These findings identify an organ-specific role for the ability of autophagy to impact organ aging phenotypes.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476360', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Partial sex linkage and linkage disequilibrium on the guppy sex chromosome', 'authors': \"[{'name': 'Suo Qiu'}, {'name': 'Lenxob Yong'}, {'name': 'Alstair Wilson'}, {'name': 'Darrren P Croft'}, {'name': 'Chay Graham'}, {'name': 'Deborah Charlesworth'}]\", 'abstract': '<p>The guppy Y chromosome has been considered a model system for the evolution of suppressed recombination between sex chromosomes, and it has been proposed that complete sex-linkage has evolved across about 3 Mb surrounding the sex-determining locus of this fish, followed by recombination suppression across a further 7 Mb of the 23 Mb XY pair, forming younger evolutionary strata. Sequences of the guppy genome show that Y is very similar to the X chromosome, making it important to understand which parts of the Y are completely non-recombining, and whether there is indeed a large completely non-recombining region. Here, we describe new evidence that supports a different interpretation of the data that suggested the presence of such a region. We analysed PoolSeq data in samples from multiple natural populations from Trinidad. This yields evidence for linkage disequilibrium (LD) between sequence variants and the sex-determining locus. Downstream populations have higher diversity than upstream ones (which display the expected signs of bottlenecks). The associations we observe conform to predictions for a genome region with infrequent recombination that carries one or more sexually antagonistic polymorphisms. They also suggest the region in which the sex-determining locus must be located. However, no consistently male-specific variants were found, supporting the suggestion that any completely sex-linked region may be very small.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.10.475628', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Optimal experimental designs for estimating genetic and non-genetic effects underlying infectious disease transmission', 'authors': \"[{'name': 'Christopher Pooley'}, {'name': 'Glenn Marion'}, {'name': 'Stephen Bishop'}, {'name': 'Andrea Doeschl-Wilson'}]\", 'abstract': '<sec><title>Background</title><p>Infectious disease spread in populations is controlled by individuals’ susceptibility (propensity to acquire infection), infectivity (propensity to pass on infection to others) and recoverability (propensity to recover/die). Estimating the effects of genetic risk factors on these host epidemiological traits can help reduce disease spread through genetic control strategies. However, the effects of previously identified ‘disease resistance SNPs’ on these epidemiological traits are usually unknown. Recent advances in computational statistics make it now possible to estimate the effects of single nucleotide polymorphisms (SNPs) on these traits from longitudinal epidemic data (<italic>e</italic>.<italic>g</italic>. infection and/or recovery times of individuals or diagnostic test results). However, little is known how to optimally design disease transmission experiments or field studies to maximise the precision at which pleiotropic SNP effects estimates for susceptibility, infectivity and recoverability can be estimated.</p></sec><sec><title>Results</title><p>We develop and validate analytical expressions for the precision of SNP effects estimates on the three host traits assuming a disease transmission experiment with one or more non-interacting contact groups. Maximising these leads to three distinct ‘experimental’ designs, each specifying a different set of ideal SNP genotype compositions across groups: a) appropriate for a single contact-group, b) a multi-group design termed “pure”, and c) a multi-group design termed “mixed”, where ‘pure’ and ‘mixed’ refer to contact groups consisting of individuals with the same or different SNP genotypes, respectively. Precision estimates for susceptibility and recoverability were found to be less sensitive to the experimental design than infectivity. Data from multiple groups were found more informative about infectivity effects than from a single group containing the same number of individuals. Whilst the analytical expressions suggest that the multi-group pure and mixed designs estimate SNP effects with similar precision, the mixed design is preferable because it uses information from naturally occurring infections rather than those artificially induced. The same optimal design principles apply to estimating other categorical fixed effects, such as vaccinations status, helping to more effectively quantify their epidemiological impact.</p><p>An online <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://theiteam.github.io/SIRE-PC.html\">software tool</ext-link><italic>SIRE-PC</italic> has been developed which calculates the precision of estimated substitution and dominance effects of a single SNP (or vaccine status) associated with all three traits depending on experimental design parameters.</p></sec><sec><title>Conclusions</title><p>The developed methodology and software tool can be used to aid the design of disease transmission experiments for estimating the effect of individual SNPs and other categorical variables underlying host susceptibility, infectivity and recoverability.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.14.476381', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Mitochondrial and redox modifications in early stages of Huntington disease', 'authors': \"[{'name': 'Carla Lopes'}, {'name': 'Ildete Luisa Ferreira'}, {'name': 'Carina Maranga'}, {'name': 'Margarida Beatriz'}, {'name': 'Sandra Mota'}, {'name': 'Jose Sereno'}, {'name': 'Joao Castelhano'}, {'name': 'Antero Abrunhosa'}, {'name': 'Francisco Oliveira'}, {'name': 'Maura De Rosa'}, {'name': 'Michael R. Hayden'}, {'name': 'Mario Laco'}, {'name': 'Cristina Januario'}, {'name': 'Miguel Castelo Branco'}, {'name': 'Ana Cristina Rego'}]\", 'abstract': '<p>Defects in mitochondrial function and mitochondrial-related redox deregulation have been attributed to Huntington disease (HD), a genetic neurodegenerative disorder largely affecting the striatum. However, whether these changes occur in early stages of the disease and can be detected in vivo is still unclear. Thus, in the present study, we analyzed changes in mitochondrial function and overreduced states associated with production of reactive oxygen species (ROS) at early stages and along disease progression in vivo in the brain by positron emission tomography (PET) and in skin fibroblasts of premanifest/early and manifest HD patients, and in YAC128 transgenic mouse brain (striatum and cortex) at early-symptomatic (3 month-old, mo) and symptomatic (6 to 12 mo) stages. In vivo human and mouse brain PET imaging was assessed using [64Cu]-ATSM; analysis of oxygen consumption rates was assessed by Seahorse analysis, hydrogen peroxide levels were determined using fluorescent probes and mitochondrial morphology by transmission electron microscopy in human skin fibroblasts and mouse striatal and cortical isolated mitochondria. Premanifest and prodromal HD carriers exhibited enhanced whole-brain (with exception of caudate) [64Cu]-ATSM labelling, correlating with CAG repeat number, concomitantly with enhanced basal and maximal respiration, proton (H+) leak and increased hydrogen peroxide levels, the later progressing to advanced HD stage, in human fibroblasts. Mitochondria from fibroblasts of premanifest HD carriers also showed reduced roundness, while higher number of mitochondrial DNA copies correlated with maximal respiratory capacity. In vivo animal PET analysis showed increased accumulation of [64Cu]-ATSM in YAC128 mouse striatum. Pre/early-symptomatic YAC128 mouse striatal, but not cortical, isolated mitochondria exhibited a rise in basal and maximal mitochondrial respiration and in ATP production along with increased complex II and III activities, enhanced mitochondrial hydrogen peroxide and roundness, as revealed by brain ultrastructure analysis, further presenting defects in Ca2+ handling, supporting increased striatal susceptibility in the YAC128 mouse model. Data demonstrate both human and mouse mitochondrial overactivity and altered morphology at early HD stages, facilitating redox unbalance, the latter extending over late disease stages.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.14.476215', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'B cells discriminate HIV-1 Envelope protein affinities by sensing antigen binding association rates', 'authors': \"[{'name': 'Md. Alamgir Hossain'}, {'name': 'Kara Anasti'}, {'name': 'Brian Watts'}, {'name': 'Kenneth Cronin'}, {'name': 'Advaiti Pai Kane'}, {'name': 'Robert J Edwards'}, {'name': 'David Easterhoff'}, {'name': 'Jinsong Zhang'}, {'name': 'Wes Rountree'}, {'name': 'Yaneth Ortiz'}, {'name': 'Laurent Verkoczy'}, {'name': 'Michael Reth'}, {'name': 'S. Munir Alam'}]\", 'abstract': '<p>HIV-1 Envelope (Env) proteins designed to induce neutralizing antibody responses allow study of the role of affinities (equilibrium dissociation constant, KD) and kinetic rates (association/dissociation rates) on B cell antigen recognition. It is unclear whether affinity discrimination during B cell activation is based solely on Env protein binding KD, and whether B cells discriminate between proteins of similar affinities but that bind with different kinetic rates. Here we used a panel of Env proteins and Ramos B cell lines expressing IgM BCRs with specificity for CD4 binding-site broadly neutralizing (bnAb) or a precursor antibody to study the role of antigen binding kinetic rates on both early (proximal/distal signaling) and late events (BCR/antigen internalization) in B cell activation. Our results support a kinetic model for B cell activation in which Env protein affinity discrimination is based not on overall KD, but on sensing of association rate and a threshold antigen-BCR half-life.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.476421', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Gene regulatory evolution in cold-adapted fly populations neutralizes plasticity and may undermine genetic canalization', 'authors': \"[{'name': 'Yuheng Huang'}, {'name': 'Justin Lack'}, {'name': 'Grant Hoppel'}, {'name': 'John E Pool'}]\", 'abstract': '<p>The relationships between adaptive evolution, phenotypic plasticity, and canalization remain incompletely understood. Theoretical and empirical studies have made conflicting arguments on whether adaptive evolution may enhance or oppose the plastic response. Gene regulatory traits offer excellent potential to study the relationship between plasticity and adaptation, and they can now be studied at the transcriptomic level. Here we take advantage of three closely-related pairs of natural populations of Drosophila melanogaster from contrasting thermal environments that reflect three separate instances of cold tolerance evolution. We measure the transcriptome-wide plasticity in gene expression levels and alternative splicing (intron usage) between warm and cold laboratory environments. We find that suspected adaptive changes in both gene expression and alternative splicing tend to neutralize the ancestral plastic response. Further, we investigate the hypothesis that adaptive evolution can lead to decanalization of selected gene regulatory traits. We find strong evidence that suspected adaptive gene expression (but not splicing) changes in cold-adapted populations are more vulnerable to the genetic perturbation of inbreeding than putatively neutral changes. We find some evidence that these patterns may reflect a loss of genetic canalization accompanying adaptation, although other processes including hitchhiking recessive deleterious variants may contribute as well. Our findings augment our understanding of genetic and environmental effects on gene regulation in the context of adaptive evolution.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.11.475854', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Acetazolamide modulates intracranial pressure directly by its action on the cerebrospinal fluid secretion apparatus', 'authors': \"[{'name': 'Dagne Barbuskaite'}, {'name': 'Eva K. Oernbo'}, {'name': 'Jonathan H. Wardman'}, {'name': 'Trine L. Toft-Bertelsen'}, {'name': 'Eller Conti'}, {'name': 'Søren N. Andreassen'}, {'name': 'Niklas J. Gerkau'}, {'name': 'Christine R. Rose'}, {'name': 'Nanna MacAulay'}]\", 'abstract': '<p>Elevated intracranial pressure (ICP) is observed in many neurological pathologies, e.g. hydrocephalus and stroke. This condition is routinely relieved with neurosurgical approaches, since effective and targeted pharmacological tools are still lacking. The carbonic anhydrase inhibitor, acetazolamide (AZE), may be employed to treat elevated ICP. However, its effectiveness is questioned, its location of action unresolved, and its tolerability low. Here, we employed <italic>in vivo</italic> and <italic>ex vivo</italic> approaches to reveal the efficacy and mode of action of AZE in the rat brain. The drug effectively reduced the ICP, irrespective of the mode of drug administration and level of anaesthesia. The effect occurred via a direct action on the choroid plexus and an associated decrease in cerebrospinal fluid secretion, and not indirectly via the systemic action of AZE on renal and vascular processes. Upon a single administration, the reduced ICP endured for approximately 10 h post-AZE delivery with no long-term changes of brain water content or choroidal transporter expression. However, a persistent reduction of ICP was secured with repeated AZE administrations throughout the day. Future specific targeting of choroidal carbonic anhydrases may limit the systemic side effects, and therefore enhance the treatment tolerability and effectiveness in select patient groups experiencing elevated ICP.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475967', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Unifying Gene Expression Complexity and Cellular Operational Efficiency: Application of the Locality and Caching Principles via a Cell-to-Computer Analogy', 'authors': \"[{'name': 'Wen Jiang'}, {'name': 'Denis Feliers'}, {'name': 'W. Jim Zheng'}, {'name': 'Fangyuan Zhang'}, {'name': 'Degeng Wang'}]\", 'abstract': '<p>Gene expression is time-consuming, and the delay from transcription activation to produced proteins is sequentially longer from bacteria to yeast and to humans. How human cells bypass the delay and attain operational efficiency, <italic>i</italic>.<italic>e</italic>., quick proteomic response to signals, is not well understood. The computer has endured the same system latency issue due to much slower information retrieval (hard drive (HD) to memory and to CPU) than CPU execution, and mitigated it via efficient memory management, namely, the spatiotemporal locality principles that control specialized user functions and the permanent caching of core system functions – the operating system (OS) kernel. Thus, in this study, we unified gene expression and HD-memory-CPU information flow as instances of the Shannon information theory, both supporting the respective system operations and consisting of three components: information storage, the execution/decoding step, and the channel for the dynamic storage-to-execution information flow; the gene expression machinery and their regulators, and the OS kernel, were deemed as the respective channels. This abstraction prompted a multi-omic comparative analysis, generating experimental evidence that transcriptome regulation shares the computer memory management principles. First, the temporal locality principle explains the mRNA stabilization-by-translation regulatory mechanism and controls specialized cellular functions. Second, the caching principle explains cytoplasmic mRNA sequestration and the defiance of the locality principle by highly sequestered mRNAs. Third, strikingly, in both systems, the caching principle controls the information channels; similar to permanent caching of OS kernel, basic translation/transcription machinery and their regulators are the top most sequestered mRNAs. Summarily, the locality and the caching principles differentially regulate specialized functions and core system functions, respectively, integrating the complexity of transcriptome regulation with cellular operational latency mitigation.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.10.475621', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Ecosystem size-induced environmental fluctuations affect the temporal dynamics of community assembly mechanisms', 'authors': \"[{'name': 'Raven L. Bier'}, {'name': 'Máté Vass'}, {'name': 'Anna J. Székely'}, {'name': 'Silke Langenheder'}]\", 'abstract': '<p>Understanding processes that determine community membership and abundance is important for many fields from theoretical community ecology to conservation. However, spatial community studies are often conducted only at a single timepoint despite the known influence of temporal variability on community assembly processes. Here we used a spatiotemporal study to determine how environmental fluctuation differences induced by mesocosm volumes (larger volumes were more stable) influence assembly processes of aquatic bacterial metacommunities along a press disturbance gradient. By combining path analysis and network approaches, we found mesocosm size categories had distinct relative influences of assembly process and environmental factors that determined spatiotemporal bacterial community composition, including dispersal and species sorting by conductivity. These processes depended on, but were not affected proportionately by, mesocosm size. Low fluctuation, large mesocosms primarily developed through the interplay of species sorting that became more important over time and transient priority effects as evidenced by more time-delayed associations. High fluctuation, small mesocosms had regular disruptions to species sorting and greater importance of ecological drift and dispersal limitation indicated by lower richness and higher taxa replacement. Together, these results emphasize that environmental fluctuations influence ecosystems over time and its impacts are modified by biotic properties intrinsic to ecosystem size.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475795', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Optimized cell culture conditions promote ex-vivo manipulation and expansion of primitive hematopoietic stem cells for therapeutic gene editing', 'authors': \"[{'name': 'Rajeev Rai'}, {'name': 'Winston Vetharoy'}, {'name': 'Asma Naseem'}, {'name': 'Zohar Steinberg'}, {'name': 'Adrian J Thrasher'}, {'name': 'Giorgia Santilli'}, {'name': 'Alessia Cavazza'}]\", 'abstract': '<p>During the last few years, gene editing has emerged as a powerful tool for the therapeutic correction of monogenic diseases. CRISPR/Cas9 applied to hematopoietic stem and progenitor cells (HSPCs) has shown great promise in proof-of-principle preclinical studies to treat haematological disorders, and clinical trials using these tools are now underway. Nonetheless, there remain important challenges that need to be addressed, such as the efficiency of targeting primitive, long-term repopulating HSPCs and expand them <italic>in vitro</italic> for clinical purposes. Here we have tested the effect exerted by different culture media compositions on the ability of HSPCs to proliferate and undergo homology directed repair-mediated knock-in of a reporter gene, while preserving their stemness features during <italic>ex-vivo</italic> culture. We tested different combinations of compounds and demonstrated that by supplementing the culture media with inhibitors of histone deacetylases, and/or by fine-tuning its cytokine composition it is possible to achieve high levels of gene targeting in long-term repopulating HSPCs both <italic>in vitro</italic> and <italic>in vivo</italic>, with a beneficial balance between preservation of stemness and cell expansion, thus allowing to obtain a significant amount of edited, primitive HSPCs compared to established, state-of-the-art culture conditions. Overall, the implantation of this optimized ex vivo HSPC culture protocol will improve the efficacy, feasibility and applicability of gene editing and will likely provide one step further to unlock the full therapeutic potential of such powerful technology.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.22269024', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Orthopedics', 'covid': False, 'title': 'IMU-derived kinematics detect gait differences with age or knee osteoarthritis but differ from marker-derived inverse kinematics', 'authors': \"[{'name': 'Jocelyn F. Hafer'}, {'name': 'Julien A. Mihy'}, {'name': 'Andrew Hunt'}, {'name': 'Ronald F. Zernicke'}, {'name': 'Russell T. Johnson'}]\", 'abstract': '<p>Common in-lab, marker-based gait analyses may not represent daily, real-world gait. Real-world gait analyses may be feasible using inertial measurement units (IMUs), especially with recent advancements in open-source methods (e.g., OpenSense). Before using OpenSense to study real-world gait, we must determine whether these methods: (1) estimate joint kinematics similarly to traditional marker-based motion capture (MoCap) and (2) differentiate groups with clinically different gait mechanics.</p><p>Healthy young and older adults and older adults with knee osteoarthritis completed this study. We captured MoCap and IMU data during overground walking at participants’ self-selected and faster speeds. MoCap and IMU kinematics were computed with appropriate OpenSim workflows. We tested whether sagittal kinematics differed between MoCap- and IMU-derived data, whether tools detected between-group differences similarly, and whether kinematics differed between tools by speed.</p><p>MoCap data showed more flexion than IMU data (hip: 0-47 and 65-100% stride, knee: 0-38 and 58-91% stride, ankle: 18-100% stride). Group kinematics differed at the hip (young extension &gt; knee osteoarthritis at 30-47% stride) and ankle (young plantar flexion &gt; older healthy at 62-65% stride). Group-by-tool interactions occurred at the hip (61-63% stride). Significant tool-by-speed interactions were found, with hip and knee flexion increasing more for MoCap than IMU data with speed (hip: 12-15% stride, knee: 60-63% stride).</p><p>While MoCap- and IMU-derived kinematics differed, our results suggested that the tools similarly detected clinically meaningful differences in gait. Results of the current study suggest that IMU-derived kinematics with OpenSense may enable the valid and reliable evaluation of gait in real-world, unobserved settings.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475766', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Machine Learning-Based Enzyme Engineering of PETase for Improved Efficiency in Degrading Non-Biodegradable Plastic', 'authors': \"[{'name': 'Arjun Gupta'}, {'name': 'Sangeeta Agrawal'}]\", 'abstract': '<p>Globally, nearly a million plastic bottles are produced every minute (1). These non-biodegradable plastic products are composed of Polyethylene terephthalate (PET). In 2016, researchers discovered PETase, an enzyme from the bacteria <italic>Ideonella sakaiensis</italic> which breaks down PET and nonbiodegradable plastic. However, PETase has low efficiency at high temperatures. In this project, we optimized the rate of PET degradation by PETase by designing new mutant enzymes which could break down PET much faster than PETase, which is currently the gold standard. We used machine learning (ML) guided directed evolution to modify the PETase enzyme to have a higher optimal temperature (Topt), which would allow the enzyme to degrade PET more efficiently.</p><p>First, we trained three machine learning models to predict Topt with high performance, including Logistic Regression, Linear Regression and Random Forest. We then used Random Forest to perform ML-guided directed evolution. Our algorithm generated hundreds of mutants of PETase and screened them using Random Forest to select mutants with the highest Topt, and then used the top mutants as the enzyme being mutated.</p><p>After 1000 iterations, we produced a new mutant of PETase with Topt of 71.38°C. We also produced a new mutant enzyme after 29 iterations with Topt of 61.3°C. To ensure these mutant enzymes would remain stable, we predicted their melting temperatures using an external predictor and found the 29-iteration mutant had improved thermostability over PETase.Our research is significant because using our approach and algorithm, scientists can optimize additional enzymes for improved efficiency.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475723', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Impaired OTUD7A-dependent Ankyrin regulation mediates neuronal dysfunction in mouse and human models of the 15q13.3 microdeletion syndrome', 'authors': \"[{'name': 'Brianna K. Unda'}, {'name': 'Leon Chalil'}, {'name': 'Sehyoun Yoon'}, {'name': 'Savannah Kilpatrick'}, {'name': 'Sansi Xing'}, {'name': 'Nadeem Murtaza'}, {'name': 'Anran Cheng'}, {'name': 'Alexandria Afonso'}, {'name': 'Elizabeth McCready'}, {'name': 'Gabriel Ronen'}, {'name': 'Jennifer Howe'}, {'name': 'Aurélie Caye-Eude'}, {'name': 'Alain Verloes'}, {'name': 'Brad W. Doble'}, {'name': 'Laurence Faivre'}, {'name': 'Antonio Vitobello'}, {'name': 'Stephen W Scherer'}, {'name': 'Yu Lu'}, {'name': 'Peter Penzes'}, {'name': 'Karun K. Singh'}]\", 'abstract': '<p>Copy number variations (CNV) are associated with psychiatric and neurodevelopmental disorders (NDDs), and most, including the recurrent 15q13.3 microdeletion disorder, have unknown disease mechanisms. We used a heterozygous 15q13.3 microdeletion mouse model and patient iPSC-derived neurons to reveal developmental defects in neuronal maturation and network activity. To identify the underlying molecular dysfunction, we developed a neuron-specific proximity-labeling proteomics (BioID2) pipeline, combined with patient mutations, to target the 15q13.3 CNV genetic driver <italic>OTUD7A</italic>. <italic>OTUD7A</italic> is an emerging independent NDD risk gene with no known function in the brain, but has putative deubiquitinase (DUB) function. The OTUD7A protein-protein interaction (PPI) network revealed interactions with synaptic, axonal, and cytoskeletal proteins and was enriched for known ASD and epilepsy risk genes. The interactions between OTUD7A and the NDD risk genes Ankyrin-G (<italic>Ank3</italic>) and Ankyrin-B (<italic>Ank2</italic>) were disrupted by an epilepsy-associated <italic>OTUD7A</italic> L233F variant. Further investigation of Ankyrin-G in mouse and human 15q13.3 microdeletion and OTUD7A<italic><sup>L233F/L233F</sup></italic> models revealed protein instability, increased polyubiquitination, and decreased levels in the axon initial segment (AIS), while structured illumination microscopy identified reduced Ankyrin-G nanodomains in dendritic spines. Functional analysis of human 15q13.3 microdeletion and OTUD7A<italic><sup>L233F/L233F</sup></italic> models revealed shared and distinct impairments to axonal growth and intrinsic excitability. Importantly, restoring OTUD7A or Ankyrin-G expression in 15q13.3 microdeletion neurons led to a reversal of abnormalities. These data reveal a critical OTUD7A-Ankyrin pathway in neuronal development, which is impaired in the 15q13.3 microdeletion syndrome, leading to neuronal dysfunction. Further, our study highlights the utility of targeting CNV genes using cell-type specific proteomics to identify shared and unexplored disease mechanisms across NDDs.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.14.476336', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Limb Specific Failure of Proliferation and Translation in the Mesenchyme Leads to Skeletal Defects in Diamond Blackfan Anemia', 'authors': \"[{'name': 'Jimmy Hom'}, {'name': 'Theodoros Karnavas'}, {'name': 'Emily Hartman'}, {'name': 'Julien Papoin'}, {'name': 'Yuefeng Tang'}, {'name': 'Brian M. Dulmovits'}, {'name': 'Mushran Khan'}, {'name': 'Hiren Patel'}, {'name': 'Jedediah Bondy'}, {'name': 'Morris Edelman'}, {'name': 'Renaud Touraine'}, {'name': 'Geneviève Chanoz-Poulard'}, {'name': 'Gregory Ottenberg'}, {'name': 'Robert Maynard'}, {'name': 'Douglas J. Adams'}, {'name': 'Raymond F. Robledo'}, {'name': 'Daniel A Grande'}, {'name': 'Philippe Marambaud'}, {'name': 'Betsy J Barnes'}, {'name': 'Sébastien Durand'}, {'name': 'Anupama Narla'}, {'name': 'Steven Ellis'}, {'name': 'Leonard I. Zon'}, {'name': 'Luanne L. Peters'}, {'name': 'Lydie Da Costa'}, {'name': 'Jeffrey M. Lipton'}, {'name': 'Cheryl L. Ackert-Bicknell'}, {'name': 'Lionel Blanc'}]\", 'abstract': '<p>Ribosomopathies are a class of disorders caused by defects in the structure or function of the ribosome and characterized by tissue-specific abnormalities. Diamond Blackfan anemia (DBA) arises from different mutations, predominantly in genes encoding ribosomal proteins (RPs). Apart from the anemia, skeletal defects are among the most common anomalies observed in patients with DBA, but they are virtually restricted to radial ray and other upper limb defects. What leads to these site-specific skeletal defects in DBA remains a mystery. Using a novel mouse model for RP haploinsufficiency, we observed specific, differential defects of the limbs. Using complementary <italic>in vitro</italic> and <italic>in vivo</italic> approaches, we demonstrate that reduced WNT signaling and subsequent increased β-catenin degradation in concert with increased expression of p53 contribute to mesenchymal lineage failure. We observed differential defects in the proliferation and differentiation of mesenchymal stem cells (MSCs) from the forelimb versus the hind limbs of the RP haploinsufficient mice that persisted after birth and were partially rescued by allelic reduction of <italic>Trp53</italic>. These defects are associated with a global decrease in protein translation in RP haploinsufficient MSCs, with the effect more pronounced in cells isolated from the forelimbs. Together these results demonstrate translational differences inherent to the MSC, explaining the site-specific skeletal defects observed in DBA.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.15.476462', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'The coupling between healthspan and lifespan in Caenorhabditis depends on complex interactions between compound intervention and genetic background', 'authors': \"[{'name': 'Stephen A Banse'}, {'name': 'E Grace Jackson'}, {'name': 'Christine A Sedore'}, {'name': 'Brian Onken'}, {'name': 'David Hall'}, {'name': 'Anna L Coleman-Hulbert'}, {'name': 'Phu Huynh'}, {'name': 'Theo Garrett'}, {'name': 'Erik Johnson'}, {'name': 'Girish Harinath'}, {'name': 'Delaney Inman'}, {'name': 'Suzhen Guo'}, {'name': 'Mackenzie Morshead'}, {'name': 'Jian Xue'}, {'name': 'Ron Falkowski'}, {'name': 'Esteban Chen'}, {'name': 'Christopher Herrera'}, {'name': 'Allie J Kirsch'}, {'name': 'Viviana I Perez'}, {'name': 'Max Guo'}, {'name': 'Gordon J Lithgow'}, {'name': 'Monica Driscoll'}, {'name': 'Patrick C Phillips'}]\", 'abstract': '<p>Aging is characterized by declining health that results in decreased neuromuscular function and cellular resilience. The relationship between lifespan and health, and the influence of genetic background on that relationship, has important implications in the development of anti-aging interventions. Here we combined survival under thermal and oxidative stress with swimming performance, to evaluate health effects across a nematode genetic diversity panel for three compounds previously studied in the <italic>Caenorhabditis</italic> Intervention Testing Program - NP1, propyl gallate, and resveratrol. We show that oxidative stress resistance and thermotolerance vary with compound intervention, genetic background, and age. The effects of tested compounds on swimming locomotion, in contrast, are largely species-specific. Additionally, thermotolerance, but not oxidative stress or swimming ability, correlates with lifespan. Our results demonstrate the importance of assessing health and lifespan across genetic backgrounds in the effort to identify reproducible aging interventions.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.14.476349', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'The RNA-binding landscape of HAX1 protein indicates its involvement in ribosome biogenesis and translation.', 'authors': \"[{'name': 'Anna Balcarak'}, {'name': 'Ewelina Macech-Klicka'}, {'name': 'Maciej Wakula'}, {'name': 'Rafal Tomecki'}, {'name': 'Krzysztof Goryca'}, {'name': 'Malgorzata Rydzanicz'}, {'name': 'Mateusz Chmielarczyk'}, {'name': 'Marta Wisniewska'}, {'name': 'Filip Lyczek'}, {'name': 'Aleksandra Helwak'}, {'name': 'David Tollervey'}, {'name': 'Grzegorz Kudla'}, {'name': 'Ewa Grzybowska'}]\", 'abstract': '<p>HAX1 is a human protein with no known homologues or structural domains, mutations in which cause severe congenital neutropenia through mechanisms that are poorly understood. Previous studies reported RNA-binding capacity of HAX1, but the role of this binding in physiology and pathology remains unexplained. Here we report transcriptome-wide characterization of HAX1 RNA targets using RIP-seq and CRAC, indicating that HAX1 binds transcripts involved in ribosome biogenesis and rRNA processing. Using CRISPR knockouts we find that RNA targets of HAX1 partially overlap with transcripts downregulated in HAX1 KO, implying a role in mRNA stabilization. Gene ontology analysis demonstrated that genes differentially expressed in HAX1 KO (including genes involved in ribosome biogenesis and translation) are also enriched in a subset of genes whose expression correlates with HAX1 expression in four analyzed neoplasms. Functional connection to ribosome biogenesis was also demonstrated by gradient sedimentation ribosome profiles, which revealed differences in the small subunit:monosome ratio in HAX1 WT/KO. We speculate that changes in HAX1 expression may be important for the etiology of HAX1-linked diseases through dysregulation of translation.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.12.476066', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Epac1 regulates cellular SUMOylation by promoting the formation of SUMO-activating nuclear condensates', 'authors': \"[{'name': 'Wenli Yang'}, {'name': 'William G. Robichaux'}, {'name': 'Fang C. Mei'}, {'name': 'Wei Lin'}, {'name': 'Li Li'}, {'name': 'Sheng Pan'}, {'name': 'Mark A. White'}, {'name': 'Yuan Chen'}, {'name': 'Xiaodong Cheng'}]\", 'abstract': '<p>Protein SUMOylation plays an essential role in maintaining cellular homeostasis when cells are under stress. However, precisely how SUMOylation is regulated, and a molecular mechanism linking cellular stress to SUMOylation remains elusive. Herein, we report that cAMP, a major stress-response second messenger, acts through Epac1 as a regulator of cellular SUMOylation. The Epac1-associated proteome is highly enriched with components of the SUMOylation pathway. Activation of Epac1 by intracellular cAMP triggers phase separation and the formation of nuclear condensates containing Epac1 and general components of the SUMOylation machinery to promote cellular SUMOylation. Furthermore, genetic knockout of Epac1 obliterates oxidized low-density lipoprotein induced cellular SUMOylation in macrophages, leading to suppression of foam cell formation. These results provide a direct nexus connecting two major cellular stress responses to define a molecular mechanism in which cAMP regulates the dynamics of cellular condensates to modulate protein SUMOylation.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475831', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'De novo protein structure prediction by incremental inter-residue geometries prediction and model quality assessment using deep learning', 'authors': \"[{'name': 'Jun Liu'}, {'name': 'Guang-Xing He'}, {'name': 'Kai-Long Zhao'}, {'name': 'Gui-Jun Zhang'}]\", 'abstract': '<sec><title>Motivation</title><p>The successful application of deep learning has promoted progress in protein model quality assessment. How to use model quality assessment to further improve the accuracy of protein structure prediction, especially not reliant on the existing templates, is helpful for unraveling the folding mechanism. Here, we investigate whether model quality assessment can be introduced into structure prediction to form a closed-loop feedback, and iteratively improve the accuracy of de novo protein structure prediction.</p></sec><sec><title>Results</title><p>In this study, we propose a de novo protein structure prediction method called RocketX. In RocketX, a feedback mechanism is constructed through the geometric constraint prediction network GeomNet, the structural simulation module, and the model quality evaluation network EmaNet. In GeomNet, the co-evolutionary features extracted from MSA that search from the sequence databases are sent to an improved residual neural network to predict the inter-residue geometric constraints. The structure model is folded based on the predicted geometric constraints. In EmaNet, the 1D and 2D features are extracted from the folded model and sent to the deep residual neural network to estimate the inter-residue distance deviation and per-residue lDDT of the model, which will be fed back to GeomNet as dynamic features to correct the geometries prediction and progressively improve model accuracy. RocketX is tested on 483 benchmark proteins and 20 FM targets of CASP14. Experimental results show that the closed-loop feedback mechanism significantly contributes to the performance of RocketX, and the prediction accuracy of RocketX outperforms that of the state-of-the-art methods trRosetta (without templates) and RaptorX. In addition, the blind test results on CAMEO show that although no template is used, the prediction accuracy of RocketX on medium and hard targets is comparable to the advanced methods that integrate templates.</p></sec><sec><title>Availability</title><p>The RocketX web server are freely available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://zhanglab-bioinf.com/RocketX\">http://zhanglab-bioinf.com/RocketX</ext-link>.</p></sec><sec><title>Contact</title><p><email>zgj@zjut.edu.cn</email></p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475656', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Microstructural Impairments in a Topologically Distinct Prefrontal-Habenular Connection in Cocaine Addiction', 'authors': \"[{'name': 'Sarah G. King'}, {'name': 'Pierre-Olivier Gaudreault'}, {'name': 'Pias Malaker'}, {'name': 'Joo-won Kim'}, {'name': 'Nelly Alia-Klein'}, {'name': 'Junqian Xu'}, {'name': 'Rita Z. Goldstein'}]\", 'abstract': '<p>Drug addiction is characterized by neuroadaptations in mesocorticolimbic networks regulating reward and inhibitory control. The habenula (Hb) is central to adaptive reward and aversion-driven behaviors, serving as a hub connecting emotion/cognitive processing regions including the prefrontal cortex (PFC). However, its role in human drug addiction has not been fully explored. Using diffusion tractography, we detailed PFC structural connectivity with three regions, namely the Hb, ventral tegmental area (VTA), and anterior thalamus (AT), and quantified the tract-specific microstructural integrity using diffusion tensor imaging within the anterior limb of the internal capsule (ALIC) in healthy and cocaine-addicted individuals. White matter microstructure in cocaine-addicted individuals was uniquely impaired in PFC-Hb projections in the ALIC, distinguishable from adjacent PFC-VTA and PFC-AT projections, with more pronounced abnormalities in short-term abstinence. These findings extend preclinical evidence of PFC-Hb circuit impairments in addiction and contextualize the plausible existence of a similar PFC-Hb connection in the human brain.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.16.22269373', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'The statistical analysis of daily data associated with different parameters of the New Coronavirus COVID-19 pandemic in Georgia and their monthly interval prediction from September 1, 2021 to December 31, 2021', 'authors': \"[{'name': 'Avtandil G. Amiranashvili'}, {'name': 'Ketevan R. Khazaradze'}, {'name': 'Nino D. Japaridze'}]\", 'abstract': \"<p>The lockdown introduced in Georgia on November 28, 2020 contributed to positive trends in the spread of COVID-19 until February - the first half of March 2021. Then, in April-May 2021, the epidemiological situation worsened significantly, and from June to the end of December COVID - situation in Georgia was very difficult. \\n\\tIn this work results of the next statistical analysis of the daily data associated with New Coronavirus COVID-19 infection of confirmed (C), recovered (R), deaths (D) and infection rate (I) cases of the population of Georgia in the period from September 01, 2021 to December 31, 2021 are presented. It also presents the results of the analysis of monthly forecasting of the values of C, D and I. As earlier, the information was regularly sent to the National Center for Disease Control &amp; Public Health of Georgia and posted on the Facebook page https://www.facebook.com/Avtandil1948/.\\nThe analysis of data is carried out with the use of the standard statistical analysis methods of random events and methods of mathematical statistics for the non-accidental time-series of observations. In particular, the following results were obtained.\\nGeorgia's ranking in the world for Covid-19 monthly mean values of infection and deaths cases in investigation period (per 1 million population) was determined. Among 157 countries with population ≥ 1 million inhabitants in October 2021 Georgia was in the 4 place on new infection cases, and in September - in the 1 place on death. Georgia took the best place in terms of confirmed cases of diseases (thirteenth) in December, and in mortality (fifth) - in October.\\nA comparison between the daily mortality from Covid-19 in Georgia from September 01, 2021 to December 31, 2021with the average daily mortality rate in 2015-2019 shows, that the largest share value of D from mean death in 2015-2019 was 76.8 % (September 03, 2021), the smallest 18.7 % (November 10, 2021). \\nAs in previous work [9,10] the statistical analysis of the daily and decade data associated with coronavirus COVID-19 pandemic of confirmed, recovered, deaths cases and infection rate of the population of Georgia are carried out. Maximum daily values of investigation parameters are following: C = 6024 (November 3, 2021), R = 6017 (November 15, 2021), D = 86 (September 3, 2021), I = 12.04 % (November 24, 2021). Maximum mean decade values of investigation parameters are following: C = 4757 (1 Decade of November 2021), R = 4427 (3 Decade of November 2021), D = 76 (2 Decade of November 2021), I = 10.55 % (1 Decade of November 2021).\\nIt was found that as in spring and summer 2021 [9,10], from September to December 2021 the regression equations for the time variability of the daily values of C, R, D and I have the form of a tenth order polynomial.\\nMean values of speed of change of confirmed -V(C), recovered - V(R), deaths - V(D) and infection rate V(I) coronavirus-related cases in different decades of months for the indicated period of time were determined.  Maximum mean decade values of investigation parameters are following: V(C) = +139 cases/day (1 Decade of October 2021), V(R) = +124 cases/day (3 Decade of October 2021), V(D) = +1.7 cases/day (3 Decade of October 2021), V(I) = + 0.20 %/ day (1 decades of October 2021).\\nCross-correlations analysis between confirmed COVID-19 cases with recovered and deaths cases shows, that from September 1, 2021 to November 30, 2021 the maximum effect of recovery is observed on 12 and 14 days after infection (CR=0.77 and 0.78 respectively), and deaths - after 7, 9, 11, 13 and 14 days (0.70≤CR≤0.72); from October 1, 2021 to December 31, 2021 -  the maximum effect of recovery is observed on 14 days after infection (RC=0.71), and deaths - after 9 days (CR=0.43). In Georgia from September 1, 2021 to November 30, 2021 the duration of the impact of the delta variant of the coronavirus on people (recovery, mortality) could be up to 28 and 35 days respectively; from October 1, 2021 to December 31, 2021 - up to 21 and 29 days respectively.\\n\\tComparison of daily real and calculated monthly predictions data of C, D and I in Georgia are carried out. It was found that in investigation period of time daily and mean monthly real values of C, D and I practically fall into the 67% - 99.99% confidence interval of these predicted values.\\t\\n\\tTraditionally, the comparison of data about C and D in Georgia (GEO) with similar data in Armenia (ARM), Azerbaijan (AZE), Russia (RUS), Turkey (TUR) and in the World (WRL) is also carried out.</p>\", 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.10.475604', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Local interactions between steady-state visually evoked potentials at nearby flickering frequencies', 'authors': \"[{'name': 'Kumari Liza'}, {'name': 'Supratim Ray'}]\", 'abstract': '<p>Steady-state visually evoked potentials (SSVEP) are widely used to index top-down cognitive processing in human electroencephalogram (EEG) studies. Typically, two stimuli flickering at different temporal frequencies (TFs) are presented, each producing a distinct response in the EEG at its flicker frequency. However, how SSVEP responses in EEG are modulated in the presence of a competing flickering stimulus just due to sensory interactions is not well understood. We have previously shown in local field potentials (LFP) recorded from awake monkeys that when two overlapping full screen gratings are counter-phased at different TFs, there is an asymmetric SSVEP response suppression, with greater suppression from lower TFs, which further depends on the relative orientations of the gratings (stronger suppression and asymmetry for parallel compared to orthogonal gratings). Here, we first confirmed these effects in both male and female human EEG recordings. Then, we mapped the response suppression of one stimulus (target) by a competing stimulus (mask) over a much wider range than the previous study. Surprisingly, we found that the suppression was not stronger at low frequencies in general, but systematically varied depending on the target TF, indicating local interactions between the two competing stimuli. These results were confirmed in both human EEG and monkey LFP and electrocorticogram (ECoG) data. Our results show that sensory interactions between multiple SSVEPs are more complex than shown previously and are influenced by both local and global factors, underscoring the need to cautiously interpret the results of studies involving SSVEP paradigms.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.475608', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Short periods of bipolar anodal TDCS induce no instantaneous dose-dependent increase in cerebral blood flow in the targeted human motor cortex', 'authors': \"[{'name': 'Marie Louise Liu'}, {'name': 'Anke Ninija Karabanov'}, {'name': 'Marjolein Piek'}, {'name': 'Esben Thade Petersen'}, {'name': 'Axel Thielscher'}, {'name': 'Hartwig Roman Siebner'}]\", 'abstract': '<sec><title>Background</title><p>Anodal transcranial direct current stimulation (aTDCS) of primary motor hand area (M1-HAND) can enhance corticomotor excitability. Yet, it is still unknown which current intensity produces the strongest effect on regional neural activity. Magnetic resonance imaging (MRI) combined with pseudo-continuous Arterial Spin Labeling (pc-ASL MRI) can map regional cortical blood flow (rCBF) and may thus be useful to probe the relationship between current intensity and neural response at the individual level.</p></sec><sec><title>Objective</title><p>Here we employed pc-ASL MRI to map acute rCBF changes during short-duration aTDCS of left M1-HAND. Using the rCBF response as a proxy for regional neuronal activity, we investigated if short-duration aTDCS produces an instantaneous dose-dependent rCBF increase in the targeted M1-HAND that may be useful for individual dosing.</p></sec><sec><title>Methods</title><p>Nine healthy right-handed participants received 30 seconds of aTDCS at 0.5, 1.0, 1.5, and 2.0 mA with the anode placed over left M1-HAND and cathode over the right supraorbital region. Concurrent pc-ASL MRI at 3 T probed TDCS-related rCBF changes in the targeted M1-HAND. Movement-induced rCBF changes were also assessed.</p></sec><sec><title>Results</title><p>Apart from a subtle increase in rCBF at 0.5 mA, short-duration aTDCS did not modulate rCBF in the M1-HAND relative to no-stimulation periods. None of the participants showed a dose-dependent increase in rCBF during aTDCS, even after accounting for individual differences in TDCS-induced electrical field strength. In contrast, finger movements led to robust activation of left M1-HAND before and after aTDCS.</p></sec><sec><title>Conclusion</title><p>Short-duration bipolar aTDCS does not produce instantaneous dose-dependent rCBF increases in the targeted M1-HAND at conventional intensity ranges.</p><p>Therefore, the regional hemodynamic response profile to short-duration aTDCS may not be suited to inform individual dosing of TDCS intensity.</p></sec><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Arterial spin labeling (ASL) MRI probed regional cerebral blood flow (rCBF) during anodal TDCS.</p></list-item><list-item><p>Short-duration anodal TDCS was applied to left motor hand area (M1-HAND) at 0.5, 1.0, 1.5 and 2.0 mA.</p></list-item><list-item><p>Anodal TDCS produced no instantaneous dose-dependent rCBF increase in left M1-HAND.</p></list-item><list-item><p>Apart from a subtle increase at 0.5 mA, rCBF was not modified in M1-HAND during anodal TDCS.</p></list-item></list></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.07.22268907', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Health Informatics', 'covid': False, 'title': 'A Computational Framework for Identifying Age Risks in Drug-Adverse Event Pairs', 'authors': \"[{'name': 'Zhizhen Zhao'}, {'name': 'Ruoqi Liu'}, {'name': 'Lei Wang'}, {'name': 'Lang Li'}, {'name': 'Chi Song'}, {'name': 'Ping Zhang'}]\", 'abstract': '<p>The identification of associations between drugs and adverse drug events (ADEs) is crucial for drug safety surveil-lance. An increasing number of studies have revealed that children and seniors are susceptible to ADEs at the population level. However, the comprehensive explorations of age risks in drug-ADE pairs are still limited. The FDA Adverse Event Reporting System (FAERS) provides individual case reports, which can be used for quantifying different age risks. In this study, we developed a statistical computational framework to detect age group of patients who are susceptible to some ADEs after taking specific drugs. We adopted different Chi-squared tests and conducted disproportionality analysis to detect drug-ADE pairs with age differences. We analyzed 4,580,113 drug-ADE pairs in FAERS (2004 to 2018Q3) and identified 2,523 pairs with the highest age risk. Furthermore, we conducted a case study on statin-induced ADE in children and youth. The code and results are available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/Zhizhen-Zhao/Age-Risk-Identification\">https://github.com/Zhizhen-Zhao/Age-Risk-Identification</ext-link></p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.10.475450', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'FTY720 requires vitamin B12-TCN2-CD320 signaling in astrocytes to reduce disease in an animal model of multiple sclerosis', 'authors': \"[{'name': 'Deepa Jonnalagadda'}, {'name': 'Yasuyuki Kihara'}, {'name': 'Aran Groves'}, {'name': 'Manisha Ray'}, {'name': 'Arjun Saha'}, {'name': 'Hyeon-Cheol Lee-Okada'}, {'name': 'Tomomi Furihata'}, {'name': 'Takehiko Yokomizo'}, {'name': 'Edward V. Quadros'}, {'name': 'Richard Rivera'}, {'name': 'Jerold Chun'}]\", 'abstract': '<p>FTY720 (fingolimod) is a sphingosine 1-phosphate (S1P) receptor modulator and sphingosine analogue approved for multiple sclerosis (MS) therapy, which can functionally antagonize the S1P receptor, S1P<sub>1</sub>. Vitamin B<sub>12</sub> (B<sub>12</sub>) deficiency produces neurological manifestations resembling MS. Here, we report a new mechanism where FTY720 suppresses neuroinflammation by regulating B<sub>12</sub> metabolic pathways. Nuclear RNA-seq of c-Fos-activated astrocytes (called <italic>ieAstrocytes</italic>) from experimental autoimmune encephalomyelitis (EAE) spinal cords identified up-regulation of CD320, a transcobalamin 2 (TCN2)-B<sub>12</sub> receptor, by S1P<sub>1</sub> inhibition. CD320 was reduced in MS plaques. Deficiency of CD320 or dietary B<sub>12</sub> worsened EAE and eliminated FTY720’s efficacy, while concomitantly down-regulating type I interferon signaling. TCN2 functioned as a chaperone for FTY720 and sphingosine, which induced astrocytic CD320 internalization. An accompanying paper identified a requirement for astrocyte sphingosine kinases in FTY720 efficacy and its altered expression in MS brains, molecularly linking MS and B<sub>12</sub> deficiency that can be accessed by sphingolipid/fingolimod metabolic pathways.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.474416', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Collagen XVIII promotes breast cancer through EGFR/ErbB signaling and its ablation improves the efficacy of ErbB-targeting inhibitors', 'authors': \"[{'name': 'Raman Devarajan'}, {'name': 'Hellevi Peltoketo'}, {'name': 'Valerio Izzi'}, {'name': 'Heli Ruotsalainen'}, {'name': 'Saila Kauppila'}, {'name': 'Marja-Riitta Väisänen'}, {'name': 'Gunilla Rask'}, {'name': 'Guillermo Martínez-Nieto'}, {'name': 'Sanna-Maria Karppinen'}, {'name': 'Timo Väisänen'}, {'name': 'Inderjeet Kaur'}, {'name': 'Jussi Koivunen'}, {'name': 'Takako Sasaki'}, {'name': 'Robert Winqvist'}, {'name': 'Fredrik Wärnberg'}, {'name': 'Malin Sund'}, {'name': 'Taina Pihlajaniemi'}, {'name': 'Ritva Heljasvaara'}]\", 'abstract': '<p>The tumor extracellular matrix (ECM) is a critical regulator of cancer progression and metastasis, significantly affecting the treatment response. Expression of collagen XVIII (ColXVIII), a ubiquitous component of basement membranes, is induced in many solid tumors, but its involvement in tumorigenesis has remained elusive. We show here that ColXVIII is markedly upregulated in human breast cancer (BC) cells and is closely associated with a poor prognosis in high-grade BC, especially in human epidermal growth factor receptor 2 (HER2)-positive and basal/triple-negative cases. We identified a novel mechanism of action for ColXVIII as a modulator of epidermal growth factor receptor (EGFR/ErbB) signaling and show that it forms a complex with EGFR, HER2 and α6 integrin to promote cancer cell proliferation in a pathway involving its N-terminal portion and the MAPK/ERK1/2 and PI3K/Akt cascades. <italic>In vivo</italic> studies with <italic>Col18a1</italic> mouse models crossed with the MMTV-PyMT mammary carcinogenesis model showed that the short ColXVIII isoform promotes BC growth and metastasis in a tumor cell-autonomous manner. Moreover, the number of mammary cancer stem cells was significantly reduced in both mouse and human cell models upon ColXVIII inhibition. Finally, ablation of ColXVIII in human BC cells and the MMTV-PyMT model substantially improved the efficacy of certain EGFR/ERbB-targeting therapies, even abolishing resistance to EGFR/ErbB inhibitors in some cell lines. In summary, a new function is revealed for ColXVIII in sustaining the stemness properties of BC cells, and tumor progression and metastasis through EGFR/ErbB signaling, suggesting that targeting ColXVIII in the tumor milieu may have significant therapeutic potential.</p><sec><title>One Sentence Summary</title><p>Collagen XVIII is upregulated in breast cancer and promotes mammary carcinogenesis through EGFR/ErbB signaling and by sustaining cancer stem cells, so that its targeting improves the efficacy of ErbB-targeted therapies.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.10.473972', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Accurate sex classification from neural responses to sexual stimuli', 'authors': \"[{'name': 'Vesa Putkinen'}, {'name': 'Sanaz Nazari-Farsani'}, {'name': 'Tomi Karjalainen'}, {'name': 'Severi Santavirta'}, {'name': 'Matthew Hudson'}, {'name': 'Kerttu Seppälä'}, {'name': 'Lihua Sun'}, {'name': 'Henry K. Karlsson'}, {'name': 'Jussi Hirvonen'}, {'name': 'Lauri Nummenmaa'}]\", 'abstract': '<p>Sex differences in brain activity evoked by sexual stimuli remain elusive despite robust evidence for stronger enjoyment of and interest towards sexual stimuli in men than in women. To test whether visual sexual stimuli evoke different brain activity patterns in men and women, we measured haemodynamic brain activity induced by visual sexual stimuli in two experiments in 91 subjects (46 males). In one experiment, the subjects viewed sexual and non-sexual film clips and dynamic annotations for nudity in the clips was used to predict their hemodynamic activity. In the second experiment, the subjects viewed sexual and non-sexual pictures in an event-related design. Males showed stronger activation than females in the visual and prefrontal cortices and dorsal attention network in both experiments. Furthermore, using multivariate pattern classification we could accurately predict the sex of the subject on the basis of the brain activity elicited by the sexual stimuli. The classification generalized across the experiments indicating that the sex differences were consistent. Eye tracking data obtained from an independent sample of subjects (N = 110) showed that men looked longer than women at the chest area of the nude female actors in the film clips. These results indicate that visual sexual stimuli evoke discernible brain activity patterns in men and women which may reflect stronger attentional engagement with sexual stimuli in men than women.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.12.476006', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Factor VII activating protease (FSAP) inhibits the outcome of ischemic stroke in mouse models', 'authors': \"[{'name': 'Jeong Yeon Kim'}, {'name': 'Dipankar Manna'}, {'name': 'Trygve B. Leergaard'}, {'name': 'Sandip M. Kanse'}]\", 'abstract': '<p>Factor VII activating protease (FSAP) is a circulating serine protease, and individuals with the Marburg I (MI) single nucleotide polymorphism (SNP), which results in an inactive enzyme, have an increased risk of stroke. The outcome of ischemic stroke is more marked in FSAP-deficient mice compared to their wild-type (WT) counterparts. Plasma FSAP levels are raised in patients as well as mice after stroke. <italic>In vitro</italic>, FSAP promotes fibrinolysis by cleavage of fibrinogen, activates protease-activated receptors and decreases the cellular cytotoxicity of histones. Since these are desirable properties in stroke treatment, we tested the effect of recombinant serine protease domain of FSAP (FSAP-SPD) on ischemic stroke in mice. A combination of tissue plasminogen activator (tPA) and FSAP-SPD enhanced clot lysis, improved microvascular perfusion and neurological outcome and reduced infarct volumes in a mouse model of thromboembolic stroke. In the tail bleeding model FSAP-SPD treatment provoked a faster clotting time indicating that it has a pro-coagulant effect that is described before. FSAP-SPD improved stroke outcome and diminished the negative effects of co-treatment with tPA in the transient middle cerebral artery occlusion model. The inactive MI-isoform of FSAP did not have any effects in either model. In mice with FSAP deficiency there were minor differences in the outcomes of stroke but the treatment with FSAP-SPD was equally effective. Thus, FSAP represents a promising novel therapeutic strategy in the treatment of ischemic stroke that requires further evaluation.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.13.476227', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Transverse and axial resolution of femtosecond laser ablation', 'authors': \"[{'name': 'Yao L. Wang'}, {'name': 'Noa W. F. Grooms'}, {'name': 'Samuel H Chung'}]\", 'abstract': '<p>Femtosecond lasers are capable of precise ablation that produce surgical dissections <italic>in vivo</italic>. The transverse and axial resolution of the laser damage inside the bulk are important parameters of ablation. The transverse resolution is routinely quantified, but the axial resolution is more difficult to measure and is less commonly performed. In some <italic>in vivo</italic> samples, fine dissections can also be difficult to visualize, but <italic>in vitro</italic> samples may allow clear imaging. Using a 1040-nm, 400-fs pulsed laser, we performed ablation inside agarose and glass, producing clear and persistent damage spots. Near the ablation threshold of both media, we found that the axial resolution is similar to the transverse resolution. We also ablated neuron cell bodies and fibers in <italic>C. elegans</italic> and demonstrate submicrometer resolution in both the transverse and axial directions, consistent with our results in agarose and glass. Using simple yet rigorous methods, we define the resolution of laser ablation in transparent media along all directions.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.476164', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Fiber-parenchyma trade-off underlies changes in tropical forest structure and xylem architecture across a soil water gradient', 'authors': \"[{'name': 'Jehová Lourenço'}, {'name': 'Paulo Roberto de Lima Bittencourt'}, {'name': 'Brian J. Enquist'}, {'name': 'Camilla Rozindo Dias Milanez'}, {'name': 'Georg von Arx'}, {'name': 'Kiyomi Morino'}, {'name': 'Luciana Dias Thomaz'}, {'name': 'Lucy Rowland'}]\", 'abstract': '<p><list list-type=\"bullet\"><list-item><p>Wood anatomical traits can underpin tropical forest structural and functional changes across soil water gradients and therefore could improve our mechanistic understanding of how plants adapt to environmental change.</p></list-item><list-item><p>We assessed how the variation in the forest maximum height (Hmax), stem diameter, and wood density (WD) is associated with variation in xylem traits (area of fibers and parenchyma, conductive area [CondA, sum of all vessels lumens], vessel lumen area [VLA], vessel density [VD], and vessel wall reinforcement [VWR]) across 42 plots of a Brazilian Atlantic Forest habitat that span strong soil water gradients.</p></list-item><list-item><p>We found that in wetter communities, greater height and lower WD were associated with greater parenchyma area (capacitance), and lower fibers, VD, VWR. Contrastingly, in drier communities, lower height was associated with higher fiber area (xylem reinforcement), WD, VD, and VWR, while parenchyma area and vessels are reduced.</p></list-item><list-item><p>Tree communities vary from conservative resource-use and structurally dependent hydraulic safety (Fibers) to acquisitive resource-use and capacitance dependent hydraulic safety (parenchyma). Such a fiber-parenchyma trade-off (FPT) underlies the variation in tree height across a soil water gradient. Wood anatomy is fundamental to understanding and predicting the impacts of environmental change on forest structure.</p></list-item></list></p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.17.476579', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'piggyBac-system-mediated genomic integration of large variant libraries for deep mutational scanning in mammalian cells', 'authors': \"[{'name': 'Yanjie Zhao'}, {'name': 'Yifan Li'}, {'name': 'Yi Wang'}, {'name': 'Weijun Su'}, {'name': 'Shuai Li'}]\", 'abstract': '<p>There is a need of introducing large variant libraries into mammalian cells to facilitate deep mutational scanning (DMS). In this study, we develop an approach to integrate in-vitro ligation dsDNA variant library into cultured mammalian cell genome via the piggyBac transposon system. T4 DNA ligase was employed to produce dsDNA ligation products containing variant library sequence flanked by piggyBac inverted terminal repeat sequences (ITRs) at both ends. The ligation products were then cotransfected with a piggyBac transposase expression vector into mammalian cells for library integration. The clone formation assay results showed that about 29,000 resistance clones were generated from a single well of 6-well-plate transfected HEK293T cells. As a proof of principle of using the strategy for DMS, a GFP chromophore random variant library was integrated into the HEK293T cell genome. By FACS selection, we enriched GFP-positive cells. The next-generation sequencing (NGS) results showed that the GFP-chromophore amino acid sequences were restored after selection.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.13.476188', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Environmental factors shape the culturable bacterial diversity in deep-sea water and surface sediments from the Mariana Trench', 'authors': \"[{'name': 'Yuewei Ma'}, {'name': 'Wenmian Ding'}, {'name': 'Yuepeng Wang'}, {'name': 'Ping Chen'}, {'name': 'Hui Zhou'}, {'name': 'Xuan Li'}, {'name': 'Yanyan Huang'}, {'name': 'Peng Nan'}]\", 'abstract': '<p>Hailed as “The Fourth Pole”, the Mariana Trench is the deepest part of the ocean. The microbial diversity in it is extremely complicated, which might be caused by the unique environmental factors such as high salinity, low temperature, high hydrostatic pressure, and limited nutrition. Based on 4 seawater samples and 4 sediment samples obtained from the Mariana Trench, we isolated and fostered the microorganism clones with kinds of culture mediums and high-throughput culturing. By using the molecular identification methods based on PCR of 16S rDNA and ITS gene, 1266 bacterial strains in total were isolated and identified, which affiliated to 7 classes, 16 orders, 25 families and 36 genera in four phyla : Proteobacteria, Firmicutes, Actinobacteria and Bacteroidetes. Strains in genera <italic>Halomonas, Pseudoaltermonas</italic> were the dominant bacteria isolated from the samples. With Mantel tests on the sample-environmental parameter matrix, the sample-environmental organic matter diversity matrix and the sample-microbial diversity matrix, we concluded that the environmental parameters and the organic matters in the condition can shape the culturable bacterial diversity in deep-sea water and surface sediments from the Mariana Trench.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.16.476546', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Conserved Untranslated Regions of Multipartite Viruses: Natural Markers of Novel Viral Genomic Components and Tags of Viral Evolution', 'authors': \"[{'name': 'Song Zhang'}, {'name': 'Caixia Yang'}, {'name': 'Jiaxing Wu'}, {'name': 'Yuanjian Qiu'}, {'name': 'Zhiyou Xuan'}, {'name': 'Liu Yang'}, {'name': 'Ruiling Liao'}, {'name': 'Xiaofei Liang'}, {'name': 'Haodong Yu'}, {'name': 'Fang Ren'}, {'name': 'Yafeng Dong'}, {'name': 'Xiaoying Xie'}, {'name': 'Yanhong Han'}, {'name': 'Di Wu'}, {'name': 'Pedro Luis Ramos-González'}, {'name': 'Juliana Freitas-Astúa'}, {'name': 'Changyong Zhou'}, {'name': 'Mengji Cao'}]\", 'abstract': '<p>Viruses with split genomes are categorized as being either segmented or multipartite according to whether their genomic segments occur in single or different virions. Some complexity will exist, in that inherited “core” vital segments viruses may renew the others once host and environmental alterations keep driving viral evolution. Despite this uncertainty, empirical observations have been made across the split genomes in the untranslated regions (UTRs) on the short or long stretches of conserved or identical sequences. In this study, we describe a methodology that combines RNA and small RNA sequencing, conventional BLASTx, and iterative BLASTn of UTRs to detect viral genomic components even if they encode orphan genes (ORFans). Within the phylum Kitrinoviricota, novel putative multipartite viruses and viral genomic components were annotated using data obtained from our sampling or publicly available sources. The novel viruses, as extensions or intermediate nodes, enriched the information of the evolutionary networks. Furthermore, the diversity of novel genomic components emphasized the evolutionary roles of reassortment and recombination, as well as genetic deletion, strongly supporting the genomic complexity. These data also suggest insufficient knowledge of these genomic components for categorizing some extant viral taxa. The relative conservation of UTRs at the genome level may explain the relationships between monopartite and multipartite viruses and how the multipartite viruses can have a life strategy involving multiple host cells.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.10.475639', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Maternal transmission as a symbiont sieve, and the absence of lactation in male mammals', 'authors': \"[{'name': 'George W. A. Constable'}, {'name': 'Brennen Fagan'}, {'name': 'Richard Law'}]\", 'abstract': '<p>Gut microbiomes of humans carry a complex symbiotic assemblage of microorganisms. As in all mammals, the special mode of feeding newborn infants through milk from the mammary gland enhances the opportunity for vertical transmission of elements of this microbiome. This has potential benefits, but it also brings with it some hazards for the host. We point out here that vertical transmission from both parents would allow host populations to be invaded by elements that are deleterious. In contrast, vertical transmission, when restricted to one parent, acts as a sieve preventing the spread of such elements. Because some transmission from mother to infant is unavoidable in placental mammals, uniparental transmission of symbionts, if it were to be selected, would require some separation of the father from the newborn infant. This paper therefore puts forward the hypothesis that the asymmetry in roles of mother and father, together with the hazards that come with biparental transmission, generate a selection pressure against male lactation in humans, and in mammals in general.</p>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.16.476543', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'An efficient simplified method for the generation of corneal epithelial cells from human pluripotent stem cells', 'authors': \"[{'name': 'Rodi Abdalkader'}, {'name': 'Ken-ichiro Kamei'}]\", 'abstract': '<p>Corneal epithelial cells derived from human pluripotent stem cells (hPSCs) are an important cell source for preclinical models to test ophthalmic drugs. However, current differentiation protocols lack instructions regarding optimal culturing conditions, which hinders the quality of cells and limits scale-up. Here, we introduce a simplified small molecule-based corneal induction method (SSM-CI) to generate corneal epithelial cells from hPSCs. SSM-CI provides the advantage of minimizing cell culturing time using two defined culturing media containing TGFβ, and Wnt/β-catenin pathway inhibitors, and bFGF growth factor over 25 days. Compared to the conventional human corneal epithelial cell line (HCE-T) and human primary corneal epithelial cells (hPCEpCs), corneal epithelial cells generated by SSM-CI are well-differentiated and express relevant maturation markers, including PAX6 and CK12. RNA-seq analysis indicated the faithful differentiation of hPSCs into corneal epithelia, with significant upregulation of corneal progenitor and adult corneal epithelial phenotypes. Furthermore, despite the initial inhibition of TGF-β and Wnt/β-catenin, upregulation of these pathway-related transcripts was observed in the later stages, indicating their necessity in the generation of mature corneal epithelial cells. Moreover, we observed a shift in gene signatures associated with the metabolic characteristics of mature corneal epithelial cells, involving a decrease in glycolysis- and an increase in fatty acid oxidation. This was also attributed to the overexpression of metabolic enzymes and transporter-related transcripts responsible for fatty acid metabolism. Thus, SSM-CI provides a comprehensive method for the generation of functional corneal epithelial cells for use in preclinical models.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.13.22269267', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Anesthesia', 'covid': False, 'title': 'Rapid detection of single nucleotide polymorphisms using the MinION nanopore sequencer: a feasibility study for perioperative precision medicine', 'authors': \"[{'name': 'Yoshiteru Tabata'}, {'name': 'Yoshiyuki Matsuo'}, {'name': 'Yosuke Fujii'}, {'name': 'Atsufumi Ohta'}, {'name': 'Kiichi Hirota'}]\", 'abstract': '<p>Introduction: Precision medicine is a phrase used to describe personalized medical care tailored to specific patients based on their clinical presentation and genetic makeup. However, despite the fact that several single nucleotide polymorphisms (SNPs) have been reported to be associated with increased susceptibility to particular anesthetic agents and the occurrence of perioperative complications, genomic profiling and thus precision medicine has not been widely applied in perioperative management.\\nMethods: We validated six SNP loci known to affect perioperative outcomes in Japanese patients using genomic DNA from saliva specimens and nanopore sequencing of each SNP loci to facilitate allele frequency calculations and then compared the nanopore results to those produced using the conventional dideoxy sequencing method.\\nResults: Nanopore sequencing reads clustered into the expected genotypes in both homozygous and heterozygous cases. In addition, the nanopore sequencing results were consistent with those obtained using conventional dideoxy sequencing and the workflow provided reliable allele frequency estimation, with a total analysis time of less than 4 h.\\nConclusion: Thus, our results suggest that nanopore sequencing may be a promising and versatile tool for SNP genotyping, allowing for rapid and feasible risk prediction of perioperative outcomes.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.475748', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Cell lineage specification during development of the anterior lateral plate mesoderm and forelimb field', 'authors': \"[{'name': 'Axel H Newton'}, {'name': 'Sarah M Williams'}, {'name': 'Andrew T Major'}, {'name': 'Craig A Smith'}]\", 'abstract': '<p>The lateral plate mesoderm (LPM) is a transient embryonic tissue that gives rise to a diverse range of mature cell types, including the cardiovascular system, the urogenital system, endoskeleton of the limbs, and mesenchyme of the gut. While the genetic processes that drive development of these tissues are well defined, the early cell fate choices underlying LPM development and specification are poorly understood. In this study, we utilize single-cell transcriptomics to define cell lineage specification during development of the anterior LPM and the forelimb field in the chicken embryo. We identify the molecular pathways directing differentiation of the aLPM towards a somatic or splanchnic cell fate, and subsequent emergence of the forelimb mesenchyme. We establish the first transcriptional atlas of progenitor, transitional and mature cell types throughout the early forelimb field and uncover the global signalling pathways which are active during LPM differentiation and forelimb initiation. Specification of the somatic and splanchnic LPM from undifferentiated mesoderm utilizes distinct signalling pathways and involves shared repression of early mesodermal markers, followed by activation of lineage-specific gene modules. We identify rapid activation of the transcription factor <italic>TWIST1</italic> in the somatic LPM preceding activation of known limb initiation genes, such as <italic>TBX</italic>5, which plays a likely role in epithelial-to-mesenchyme transition of the limb bud mesenchyme. Furthermore, development of the somatic LPM and limb is dependent on ectodermal BMP signalling, where BMP antagonism reduces expression of key somatic LPM and limb genes to inhibit formation of the limb bud mesenchyme. Together, these findings provide new insights into molecular mechanisms that drive fate cell choices during specification of the aLPM and forelimb initiation.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.475833', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Prom1 and Notch regulate ciliary length and dynamics in multiciliated cells of the airway epithelium', 'authors': \"[{'name': 'Carlos F. H. Serra'}, {'name': 'Helu Liu'}, {'name': 'Jun Qian'}, {'name': 'Munemasa Mori'}, {'name': 'Jining Lu'}, {'name': 'Wellington V. Cardoso'}]\", 'abstract': '<p>Differences in ciliary morphology and dynamics among multiciliated cells of the respiratory tract have been well reported and known to contribute to efficient mucociliary clearance. Nevertheless, little is known about how phenotypic differences among multiciliated cells are established in the mammalian lung. Here we show that Prominin-1 (Prom1), a transmembrane protein widely used as stem cell and tumor-initiating marker, is crucial to this process. During airway differentiation, Prom1 becomes restricted to multiciliated cells, where it is expressed at distinct levels along the proximal-distal axis of the airways and in the adult airway epithelium in vitro. We found that Prom1 is induced by Notch in post-specified multiciliated cells and that Notch inactivation abolishes the gradients of Prom1 in the developing airways and in differentiating organotypic cultures. Prom1 was not required for multicilia formation and when inactivated resulted in longer cilia, which remained functional but beating at a lower frequency. Disruption of Notch resulted in opposite effects and suggested that Notch fine-tunes Prom1 levels to regulate the multiciliated cell phenotype and generate diversity among these cells in the respiratory tract. By controlling these features, this mechanism contributes to the innate defense of the lung against environmental agents and prevent pulmonary disease.</p><sec><title>Significance Statement</title><p>Multiciliated cells are integral components of the epithelia from a variety of organs. In the respiratory tract they are crucial for mucociliary clearance, a first line of defense against environmental agents and microorganisms. Regional differences in ciliary morphology and dynamics of multiciliated cells have been well described. However, little is known about the events generating phenotypical and functional differences among these cells in airways. Here we provide evidence of a novel mechanism in post-specified multiciliated progenitors whereby local Notch and Prom1 regulate ciliary length and ciliary beating to generate morphological and functional diversity among the multiciliated cells. The findings provide insights into the impact of these signals in maintaining the integrity and function of the airway epithelium, preventing pulmonary disease.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476044', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Automated bioprocess feedback operation in a high throughput facility via the integration of a mobile robotic lab assistant', 'authors': \"[{'name': 'Lucas Kaspersetz'}, {'name': 'Saskia Waldburger'}, {'name': 'M.-Therese Schermeyer'}, {'name': 'Sebastian L. Riedel'}, {'name': 'Sebastian Groß'}, {'name': 'Peter Neubauer'}, {'name': 'M.-Nicolas Cruz-Bournazou'}]\", 'abstract': '<p>Biotechnological processes development is challenging due to the sheer variety of process parameters. For efficient upstream development parallel cultivation systems have proven to reduce costs and associated timelines successfully, while offering excellent process control. However, the degree of automation of such small scale systems is comparably low and necessary sample analysis requires manual steps. Although the subsequent analysis can be performed in a high-throughput manner, the integration of analytic devices remains challenging. Especially, when cultivation and analysis laboratories are spatially separated. Mobile robots offer a potential solution, but the implementation in research laboratories is not widely adopted. Our approach demonstrates the integration of a small scale cultivation system into a liquid handling station for an automated sample procedure. The samples are transferred via a mobile robotic lab assistant and subsequently analysed by a high-throughput analyzer. The process data is stored in a centralized database. The mobile robotic workflow guarantees a flexible solution for device integration and facilitates automation. Restrictions regarding spatial separation of devices are circumvented, enabling a modular platform throughout different laboratories. The presented cultivation platform is evaluated based on industrial relevant <italic>E. coli</italic> BW25113 high cell density fed-batch cultivation. Here its suitability for accelerating bioprocess development is proven. The necessary magnesium addition for reaching high cell densities in mineral salt medium is automated via a feedback operation loop. The feedback operation loop demonstrates the possibility for advanced control options. This study sets the foundation for a fully integrated facility with different cultivation scales sharing the same data infrastructure, where the mobile robotic lab assistant physically connects the devices.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476359', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Kynurenine metabolism is altered in mdx mice: a potential muscle to brain connection', 'authors': \"[{'name': 'Emily N Copeland'}, {'name': 'Kennedy C Whitley'}, {'name': 'Colton JF Watson'}, {'name': 'Bradley J Baranowski'}, {'name': 'Nigel Kurgan'}, {'name': 'Adam J MacNeil'}, {'name': 'Rebecca EK MacPherson'}, {'name': 'Val A. Fajardo'}, {'name': 'David J Allison'}]\", 'abstract': '<p>Regular exercise can direct muscle kynurenine (KYN) metabolism toward the neuroprotective branch of the kynurenine pathway thereby limiting the accumulation of neurotoxic metabolites in the brain and contributing to mental resilience. While the effect of regular exercise has been studied, the effect of muscle disease on KYN metabolism has not yet been investigated. Previous work has highlighted anxiety-like behaviors in approximately 25% of patients with DMD, possibly due to altered KYN metabolism. Here, we characterized KYN metabolism in mdx mouse models of Duchenne muscular dystrophy (DMD). Young (8-10 week old) DBA/2J (D2) mdx mice, but not age-matched C57BL/10 (C57) mdx mice, had lower levels of circulating KYNA and KYNA:KYN ratio compared with their respective wild-type (WT) controls. Moreover, only D2 mdx mice displayed signs of anxiety-like behaviour, spending more time in the corners of their cages during a novel object recognition test when compared with WT. Along with this, we found that muscles from D2 mdx mice had less peroxisome proliferator-activated receptor-gamma coactivator 1-alpha and kynurenine amino transferase-1 enzyme content as well as elevated expression of inflammatory cytokines compared with WT muscles. Thus, our pilot work shows that KYN metabolism is altered in D2 mdx mice, with a potential contribution from altered muscle health.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.476385', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Phenology effects on productivity and hatching-asynchrony of American kestrels (Falco sparverius) across a continent', 'authors': \"[{'name': 'Kathleen R. Callery'}, {'name': 'Sarah E. Schulwitz'}, {'name': 'Anjolene R. Hunt'}, {'name': 'Jason M. Winiarski'}, {'name': 'Christopher J. W. McClure'}, {'name': 'Richard A. Fischer'}, {'name': 'Julie A. Heath'}]\", 'abstract': '<p>Climate-driven advances in spring can result in phenological mismatch between brood rearing and prey availability and consequently cause decreased productivity in birds. How consequences of mismatch vary across species’ ranges, and how individual behavior can mitigate mismatch effects is less studied. We quantified the relationship between phenological mismatch, productivity, and behavioral adaptations of American kestrels (<italic>Falco sparverius</italic>) across their breeding range in the United States and southern Canada. We obtained phenology and productivity data using nest observations from long term nest box monitoring, remote trail cameras, and community-scientist based programs. We collected data on parental incubation behavior and hatch asynchrony using trail cameras in nest boxes. Kestrels that laid eggs after the start of spring had higher rates of nest failure and fewer nestlings than earlier nesters, and effects of mismatch on productivity were most severe in the Northeast. In contrast, kestrels in the Southwest experienced a more gradual decline in productivity with seasonal mismatch. We attribute the effect of location to the growing season and temporal nesting windows (duration of nesting season). Specifically, resource availability in the Northeast is narrow and highly peaked during the breeding season, potentially resulting in shorter nesting windows. Conversely, resource curves may be more prolonged and dampened in the Southwest, and growing seasons are becoming longer with climate change, potentially resulting in longer nesting windows. We found that the onset of male incubation was negatively associated with lay date. Males from breeding pairs that laid eggs after the start of spring began incubation sooner than males from breeding pairs that laid before the start of spring. Early-onset male incubation was positively associated with hatching asynchrony, creating increased age variation in developing young. In sum, we demonstrate that American kestrels are vulnerable to phenological mismatch, and that this vulnerability varies across space. Northeastern populations could be more vulnerable to mismatch consequences, which may be one factor contributing to declines of kestrels in this region. Also, we demonstrate early onset of incubation as a potential adaptive behavior to advance average hatch date and spread out offspring demands, but it is unknown how impactful this will be in mitigating the fitness consequences of phenology mismatch.</p><sec><title>Highlights</title><p>Climate-driven phenological mismatch is a growing conservation concern.</p><p>We studied phenology and productivity of an avian predator across North America.</p><p>American kestrels nesting after the start of spring had lower productivity.</p><p>Later nesting kestrels altered incubation to create hatching asynchrony.</p><p>Mismatch effects were strongest in the Northeast and may contribute to population declines.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476414', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Ancestral sequence reconstructions evaluated by extant sequence cross-validation', 'authors': \"[{'name': 'Michael Sennett'}, {'name': 'Douglas Theobald'}]\", 'abstract': '<p>Ancestral sequence reconstruction (ASR) has become widely used to analyze the properties of ancient biomolecules and to elucidate the mechanisms of molecular evolution.  By recapitulating the structural, mechanistic, and functional changes of proteins during their evolution, ASR has been able to address many fundamental and challenging evolutionary questions where more traditional methods have failed.  Despite the tangible successes of ASR, the accuracy of its reconstructions is currently unknown, because it is generally impossible to compare resurrected proteins to the true ancient ancestors that are now extinct. Which evolutionary models are the best for ASR?  How accurate are the resulting inferences? Here we answer these questions by applying cross-validation (CV) to sets of aligned extant sequences. To assess the adequacy of a chosen evolutionary model for predicting extant sequence data, our column-wise CV method iteratively cross-validates each column in an alignment. Unlike other phylogenetic model selection criteria, this method does not require bias correction and does not make  restrictive assumptions commonly violated by phylogenetic data. We find that column-wise CV generally provides a more conservative criterion than the AIC by preferring less complex models. To validate ASR methods, we also apply cross-validation to each sequence in an alignment by reconstructing the extant sequences using ASR methodology, a method we term extant sequence reconstruction (ESR). We can thus quantify the accuracy of ASR methodology by comparing ESR reconstructions to the corresponding true sequences. We find that a common measure of the quality of a reconstructed sequence, the average probability of the sequence, is indeed a good estimate of the fraction of the sequence that is correct when the evolutionary model is accurate or overparameterized. However, the average probability is a poor measure for comparing reconstructions, because more accurate phylogenetic models typically result in reconstructions with lower average probabilities.  In contrast, the entropy of the reconstructed distribution is a reliable indicator of the quality of a reconstruction, as the entropy provides an accurate estimate of the log-probability of the true sequence. Both column-wise CV and ESR are useful methods to validate evolutionary models used for ASR and can be applied in practice to any phylogenetic analysis of real biological sequences.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.476417', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Dickeya colocasiae sp. nov. isolated from wetland taro, Colocasia esculentum', 'authors': \"[{'name': 'Gamze Boluk'}, {'name': 'Shefali Dobhal'}, {'name': 'Dario Arizala'}, {'name': 'Anne M. Alvarez'}, {'name': 'Mohammad Arif'}]\", 'abstract': '<p>Bacterial pathogens identified as Dickeya sp. have recently been associated with a corm rot of wetland taro on Oahu, Hawaii, but the species designation of these strains was unclear.  A Gram-negative, pectinolytic bacterial strain PL65T isolated from an infected taro corm was subjected to polyphasic analysis to determine its genomic and phenotypic characteristics. Multi-locus sequence analyses (MLSA) based on five housekeeping genes (dnaA, gapA, gyrB, atpD, and purA) revealed that Dickeya zeae and D. oryzae, were the closest relatives. Phylogenetic analysis based on 463 core gene sequences clearly showed two potentially new species within Dickeya oryzae. In silico DNA-DNA hybridization value of strain PL65T with 12 Type strains of Dickeya species was &lt;68%. Average nucleotide identity (ANI) analysis revealed that PL65T was at the margin of the species delineation cut-off values with a 96% ANI value. The metabolic profile of strain PL65T using BIOLOG differentiated it from the type strains of all other known species of Dickeya. Based on the results of genome-to-genome comparisons and phenotypic data presented in this report, we propose establishment of a new species, Dickeya colocasiae sp. nov.   with strain PL65T as the type strain (ICMP 24361T).</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.14.476424', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'FRET monitoring of transcription factor activities in living bacteria', 'authors': \"[{'name': 'Pengchao Wang'}, {'name': 'Guangming Zhang'}, {'name': 'Zeling Xu'}, {'name': 'Zhe Chen'}, {'name': 'Xiaohong Liu'}, {'name': 'Chenyin Wang'}, {'name': 'Chaogu Zheng'}, {'name': 'Jiangyun Wang'}, {'name': 'Hongmin Zhang'}, {'name': 'Aixin Yan'}]\", 'abstract': '<p>Bacteria adapt to the constantly changing environments largely by transcriptional regulation through the activities of various transcription factors (TFs). However, techniques that monitor the in situ TF-promoter interactions in living bacteria are lacking. Herein, we developed a whole-cell TF-promoter binding assay based on the intermolecular Förster resonance energy transfer (FRET) between a fluorescent unnatural amino acid CouA which is genetically encoded into defined sites in TFs and the live cell fluorescent nucleic acid stain SYTO 9. We show that this new FRET pair monitors the intricate TF-promoter interactions elicited by various types of signal transduction systems with specificity and sensitivity. Furthermore, the assay is applicable to identify novel modulators of the regulatory systems of interest and monitor TF activities in bacteria colonized in <italic>C. elegans</italic>. In conclusion, we established a tractable and sensitive TF-promoter binding assay in living bacteria which not only complements currently available approaches for DNA-protein interactions but also provides novel opportunities for functional annotation of bacterial signal transduction systems and studies of the bacteria-host interface.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.15.476378', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Activation of the dorsal septum increases alcohol consumption in male C57BL/6J mice.', 'authors': '[{\\'name\\': \\'Harold L Haun\\'}, {\\'name\\': \"Shannon L D\\'Ambrosio\"}, {\\'name\\': \\'Thomas L Kash\\'}]', 'abstract': '<p>Binge drinking is a common pattern of excessive alcohol consumption associated with Alcohol Use Disorder (AUD) and unraveling the neurocircuitry that promotes this type of drinking is critical to the development of novel therapeutic interventions. The septal region was once a focal point of alcohol research yet has seen limited study over the last decade in relation to binge drinking. Numerous studies point to involvement of the dorsal septum (dSep) in excessive drinking and withdrawal, but few studies have manipulated this region in the context of binge drinking behavior. The present experiments were primarily designed to determine the effect of chemogenetic manipulation of the dSep on binge-like alcohol drinking in male and female C57BL/6J mice. Mice received bilateral infusion of AAVs harboring hM4Di, hM3Dq, or mCherry into the dSep and subjects were challenged with systemic administration of clozapine-N-oxide (CNO; 3 mg/kg) or vehicle (saline; 0.9%) in the context of binge-like alcohol consumption, locomotor activity, and sucrose drinking. CNO-mediated activation (hM3Dq) of the dSep resulted in a significant increase in binge-like alcohol consumption, locomotor activity, and sucrose intake in male mice. DSep activation promoted sucrose drinking in female mice, but alcohol intake and locomotor activity were unaffected. Conversely, silencing (hM4Di) of the dSep modestly decreased locomotor activity in males and did not influence alcohol or sucrose intake in either sex. Lastly, CNO was without effect in mCherry-expressing control groups. These data support a role for the dSep in promoting excessive, binge-like drinking behavior in a sex-dependent fashion and suggests a broad role for the region in the modulation of locomotor activity as well as general appetitive behavior such as sucrose drinking.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.15.476391', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Microbiomes of hadal fishes contain similar taxa, obligate symbionts, and known piezophiles across trench habitats', 'authors': \"[{'name': 'Jessica M Blanton'}, {'name': 'Logan M Peoples'}, {'name': 'Mackenzie E Gerringer'}, {'name': 'Caroline M Iacuniello'}, {'name': 'Natalya D Gallo'}, {'name': 'Thomas D Linley'}, {'name': 'Alan J Jamieson'}, {'name': 'Jeffrey C Drazen'}, {'name': 'Douglas H Bartlett'}, {'name': 'Eric E Allen'}]\", 'abstract': '<p>Hadal snailfishes are the deepest-living fishes in the ocean, inhabiting trenches from depths of ~6,000 to 8,000 m. While the microbial communities in trench environments have begun to be characterized, the microbes associated with hadal megafauna remain relatively unknown. Here, we describe the gut microbiomes of two hadal snailfishes, <italic>Pseudoliparis swirei</italic> (Mariana Trench) and <italic>Notoliparis kermadecensis</italic> (Kermadec Trench) using 16S rRNA gene amplicon sequencing. We contextualize these microbiomes with comparisons to the abyssal macrourid <italic>Coryphaenoides yaquinae</italic> and the continental shelf-dwelling snailfish <italic>Careproctus melanurus</italic>. The microbial communities of the hadal snailfishes were distinct from their shallower counterparts and were dominated by the same sequences related to the Mycoplasmataceae and Desulfovibrionaceae. These shared taxa indicate that symbiont lineages may have remained similar to the ancestral symbiont since their geographic separation or that they are dispersed between geographically distant trenches and subsequently colonize specific hosts. The abyssal and hadal fishes contained sequences related to known, cultured piezophiles, microbes that grow optimally under high hydrostatic pressure, including <italic>Psychromonas</italic>, <italic>Moritella</italic>, and <italic>Shewanella</italic>. These taxa are adept at colonizing nutrient-rich environments present in the deep ocean, such as on particles and in the guts of hosts, and we hypothesize they could make a dietary contribution to deep-sea fishes by degrading chitin and producing fatty acids. We characterize the gut microbiota within some of the deepest fishes to provide new insight into the diversity and distribution of host-associated microbial taxa and the potential of these animals, and the microbes they harbor, for understanding adaptation to deep-sea habitats.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.15.476457', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'User-guided local and global copy-number segmentation for tumor sequencing data', 'authors': \"[{'name': 'Zubair Lalani'}, {'name': 'Gillian Chu'}, {'name': 'Silas Hsu'}, {'name': 'Simone Zaccaria'}, {'name': 'Mohammed El-Kebir'}]\", 'abstract': '<p>Motivation: Copy-number aberrations (CNA) are genetic alterations that amplify or delete the number of copies of large genomic segments. Although they are ubiquitous in cancer and subsequently a critical area of current cancer research, CNA identification from DNA sequencing data is challenging because it requires partitioning of the genome into complex segments that may not be contiguous. Existing segmentation algorithms address these challenges either by leveraging the local information among neighboring genomic regions, or by globally grouping genomic regions that are affected by similar CNAs across the entire genome. However, both approaches have limitations: overclustering in the case of local segmentation, or the omission of clusters corresponding to focal CNAs in the case of global segmentation. Importantly, inaccurate segmentation will lead to inaccurate identification of important CNAs. Results: We introduce CNAViz, a web-based tool that enables the user to simultaneously perform local and global segmentation, thus overcoming the limitations of each approach. Using simulated data, we demonstrate that by several metrics, CNAViz yields more accurate segmentations relative to existing local and global segmentation methods. Moreover, we analyze six bulk DNA sequencing samples from three breast cancer patients. By validating with parallel single-cell DNA sequencing data from the same samples, we show that with CNAViz we produce more accurate segmentation which improves accuracy in downstream copy-number calling.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.15.476460', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Vexitoxins: a novel class of conotoxin-like venom peptides from predatory gastropods of the genus Vexillum', 'authors': \"[{'name': 'Ksenia G Kuznetsova'}, {'name': 'Sofia S Zvonareva'}, {'name': 'Rustam Ziganshin'}, {'name': 'Elena S Mekhova'}, {'name': 'Polina Yu Dgebuadze'}, {'name': 'Yen T.H. Dinh'}, {'name': 'Thanh H.T. Nguyen'}, {'name': 'Sergey A Moshkovskii'}, {'name': 'Alexander E Fedosov'}]\", 'abstract': '<p>Venoms of predatory marine cone snails (the family Conidae, order Neogastropoda) are intensely studied because of the broad range of biomedical applications of the neuropeptides that they contain, conotoxins. Meanwhile anatomy in some other neogastropod lineages strongly suggests that they have evolved similar venoms independently of cone snails, nevertheless their venom composition remains unstudied. Here we focus on the most diversified of these lineages, the genus Vexillum (the family Costellariidae). We have generated comprehensive multi-specimen, multi-tissue RNA-Seq data sets for three Vexillum species, and supported our findings in two species by proteomic profiling. We show that venoms of Vexillum are dominated by highly diversified short cysteine-rich peptides that in many aspects are very similar to conotoxins. Vexitoxins possess the same precursor organization, display overlapping cysteine frameworks and share several common post-translational modifications with conotoxins. Some vexitoxins show detectable sequence similarity to conotoxins, and are predicted to adopt similar domain conformations, including a pharmacologically relevant inhibitory cysteine-know motif (ICK). The tubular gL of Vexillum is a notably more recent evolutionary novelty than the conoidean venom gland. Thus, we hypothesize lower divergence between the toxin genes, and their somatic counterparts compared to that in conotoxins, and we find support for this hypothesis in the molecular evolution of the vexitoxin cluster V027. We use this example to discuss how future studies on vexitoxins can inform origin and evolution of conotoxins, and how they may help addressing standing questions in venom evolution.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.14.476318', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Glucose-stimulated calcium dynamics in beta cells from C57BL/6J, C57BL/6N, and NMRI mice: A systematic comparison of activation, activity, and deactivation properties in tissue slices', 'authors': \"[{'name': 'Viljem Pohorec'}, {'name': 'Lidija Krizancic Bombek'}, {'name': 'Masa Skelin Klemen'}, {'name': 'Jurij Dolensek'}, {'name': 'Andraz Stozer'}]\", 'abstract': '<p>Although mice are a very instrumental model in islet beta cell research, possible phenotypic differences between strains and substrains are largely neglected in the scientific community. In this study, we show important phenotypic differences in beta cell responses to glucose between NMRI, C57BL/6J, and C57BL/6N mice, i.e., the three most commonly used strains. High-resolution multicellular confocal imaging of beta cells in acute pancreas tissue slices was used to measure and quantitatively compare the calcium dynamics in response to a wide range of glucose concentrations. Strain- and substrain-specific features were found in all three phases of beta cell responses to glucose: a shift in the dose-response curve characterizing the delay to activation and deactivation in response to stimulus onset and termination, respectively, and distinct concentration-encoding principles during the plateau phase in terms of frequency, duration, and active time changes with increasing glucose concentrations. Our results underline the significance of carefully choosing and reporting the strain to enable comparison and increase reproducibility, emphasize the importance of analyzing a number of different beta cell physiological parameters characterizing the response to glucose, and provide a valuable standard for future studies on beta cell calcium dynamics in health and disease.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476270', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Graphics processing unit-accelerated Monte Carlo simulation of polarized light in complex three-dimensional media', 'authors': \"[{'name': 'Shijie Yan'}, {'name': 'Steven L Jacques'}, {'name': 'Jessica C. Ramella-Roman'}, {'name': 'Qianqian Fang'}]\", 'abstract': '<p>Significance: Monte Carlo (MC) methods have been applied for studying interactions between polarized light and biological tissues, but most existing MC codes supporting polarization modeling can only simulate homogeneous or multi-layered domains, resulting in approximations when handling realistic tissue structures. Aim: Over the past decade, the speed of MC simulations has seen dramatic improvement with massively-parallel computing techniques. Developing hardware-accelerated MC simulation algorithms that can accurately model polarized light inside 3-D heterogeneous tissues can greatly expand the utility of polarization in biophotonics applications. Approach: Here we report a highly efficient polarized MC algorithm capable of modeling arbitrarily complex media defined over a voxelated domain. Each voxel of the domain can be associated with spherical scatters of various radii and densities. The Stokes vector of each simulated photon packet is updated through photon propagation, creating spatially resolved polarization measurements over the detectors or domain surface. Results: We have implemented this algorithm in our widely disseminated MC simulator, Monte Carlo eXtreme (MCX). It is validated by comparing with a reference CPU-based simulator in both homogeneous and layered domains, showing excellent agreement and a 931-fold speedup. Conclusion: The polarization-enabled MCX (pMCX) offers biophotonics community an efficient tool to explore polarized light in bio-tissues, and is freely available at http://mcx.space/.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.13.476281', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The bladder microbiome, metabolome, cytokines, and phenotypes in patients with systemic lupus erythematosus', 'authors': \"[{'name': 'Fengping Liu'}, {'name': 'Jingjie Du'}, {'name': 'Qixiao Zhai'}, {'name': 'Jialin Hu'}, {'name': 'Aaron W. Miller'}, {'name': 'Tianli Ren'}, {'name': 'Yangkun Feng'}, {'name': 'Peng Jiang'}, {'name': 'Lei Hu'}, {'name': 'Jiayi Sheng'}, {'name': 'Chaoqun Gu'}, {'name': 'Ren Yan'}, {'name': 'Longxian Lv'}, {'name': 'Alan J. Wolfe'}, {'name': 'Ninghan Feng'}]\", 'abstract': '<sec><title>Background and aims</title><p>Emerging studies reveal a unique bacterial community in the human bladder, with alteration of composition associated to disease states. Systemic lupus erythematosus (SLE) is a complex autoimmune disease that is characterized by frequent impairment of the kidney. Here, we explored the bladder microbiome, metabolome, and cytokine profiles in SLE patients, as well as correlations between microbiome and metabolome, cytokines, and disease profiles.</p></sec><sec><title>Methods and materials</title><p>We recruited a cohort of 50 SLE patients and 50 individually matched asymptomatic controls. We used transurethral catheterization to collect urine samples, 16S rRNA gene sequencing to profile bladder microbiomes, and LC-MS/MS to perform untargeted metabolomic profiling.</p></sec><sec><title>Results</title><p>Compared to controls, SLE patients possessed a unique bladder microbial community and increased alpha diversity. These differences were accompanied by differences in urinary metabolomes, cytokines, and patients’ disease profiles. The SLE-enriched genera, including <italic>Bacteroides</italic>, were positively correlated with several SLE-enriched metabolites, including olopatadine. The SLE-depleted genera, such as <italic>Pseudomonas</italic>, were negatively correlated to SLE-depleted cytokines, including IL-8. Alteration of the bladder microbiome was associated with disease profile. For example, the genera <italic>Megamonas</italic> and <italic>Phocaeicola</italic> were negatively correlated with serum complement C3, and <italic>Streptococcus</italic> was positively correlated with IgG.</p></sec><sec><title>Conclusions</title><p>Our present study reveals associations between the bladder microbiome and the urinary metabolome, cytokines, and disease phenotypes. Our results could help identify biomarkers for SLE.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.13.476286', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Transcriptomic analysis reveals a role for the nervous system in regulating growth and development of Fasciola hepatica juveniles', 'authors': \"[{'name': 'Emily Robb'}, {'name': 'Erin McCammick'}, {'name': 'Duncan Wells'}, {'name': 'Paul McVeigh'}, {'name': 'Erica Gardiner'}, {'name': 'Rebecca Armstrong'}, {'name': 'Paul McCusker'}, {'name': 'Angela Mousley'}, {'name': 'Nathan Clarke'}, {'name': 'Nikki Marks'}, {'name': 'Aaron G. Maule'}]\", 'abstract': '<p><italic>Fasciola</italic> spp. liver fluke have significant impacts in veterinary and human medicine. The absence of a vaccine and increasing anthelmintic resistance threaten sustainable control and underscore the need for novel flukicides. Functional genomic approaches underpinned by <italic>in vitro</italic> culture of juvenile <italic>Fasciola hepatica</italic> facilitate control target validation in the most pathogenic life stage. Comparative transcriptomics of <italic>in vitro</italic> and <italic>in vivo</italic> maintained 21 day old <italic>F. hepatica</italic> finds that 86% of genes are expressed at similar levels across maintenance treatments suggesting commonality in core biological functioning within these juveniles. Phenotypic comparisons revealed higher cell proliferation and growth rates in the <italic>in vivo</italic> juveniles compared to their <italic>in vitro</italic> counterparts. These phenotypic differences were consistent with the upregulation of neoblast-like stem cell and cell-cycle associated genes in <italic>in vivo</italic> maintained worms. The more rapid growth/development of <italic>in vivo</italic> juveniles was further evidenced by a switch in cathepsin protease expression profiles, dominated by cathepsin B in <italic>in vitro</italic> juveniles and then by cathepsin L in <italic>in vivo</italic> juveniles. Coincident with more rapid growth/development was the marked downregulation of both classical and peptidergic neuronal signalling components in <italic>in vivo</italic> maintained juveniles, supporting a role for the nervous system in regulating liver fluke growth and development. Differences in the miRNA complements of <italic>in vivo</italic> and <italic>in vitro</italic> juveniles identified 31 differentially expressed miRNAs, notably <italic>fhe-let-7a-5p</italic>, <italic>fhe-mir-124-3p</italic> and, miRNAs predicted to target Wnt-signalling, supporting a key role for miRNAs in driving the growth/developmental differences in the <italic>in vitro</italic> and <italic>in vivo</italic> maintained juvenile liver fluke. Widespread differences in the expression of neuronal genes in juvenile fluke grown <italic>in vitro</italic> and <italic>in vivo</italic> expose significant interplay between neuronal signalling and the rate of growth/development, encouraging consideration of neuronal targets in efforts to dysregulate growth/development for parasite control.</p><sec><title>Author Summary</title><p>Parasitic worms are notoriously difficult to study outside of a host organism. However, recent developments in culture methods for <italic>Fasciola hepatica</italic> liver fluke juveniles support growth and development of these parasites in the laboratory (<italic>in vitro</italic>) towards adult parasites. Having the ability to grow pathogenic juvenile stages <italic>in vitro</italic> enables functional studies to validate potential drug and vaccine targets. However, comparison of <italic>in vitro</italic> grown juveniles to juveniles retrieved from infected hosts (<italic>in vivo</italic>) shows considerable size differences suggesting at least some differences in biology that could undermine the relevance of data generated from <italic>in vitro</italic> maintained parasites. This study examines gene expression differences between <italic>in vitro</italic> and <italic>in vivo</italic> maintained <italic>F. hepatica</italic> juveniles via transcriptomic analysis to identify similarities and differences in their biology which may help explain differences in the rate of growth and development. 86% of genes were shown to be expressed at similar levels across treatment groups suggesting a high level of biological similarity between <italic>in vitro and in vivo</italic> juveniles. However, the genes that are expressed differently between these juveniles will help improve current culture methods and provide a new group of potential drug targets that impact on juvenile growth and development.</p></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.22268980', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'HAT-field: a very cheap, robust and quantitative point-of-care serological test for Covid-19.', 'authors': \"[{'name': 'Etienne Joly'}, {'name': 'Agnes Maurel Ribes'}]\", 'abstract': '<p>We have recently described a very simple and cheap serological test called HAT to detect antibodies directed against the RBD of the SARS-Cov-2 virus. HAT is based on hemagglutination, triggered by a single reagent (IH4-RBD) comprised of the viral RBD domain fused to a nanobody specific for glycophorin, which is expressed at very high levels at the surface of human red blood cells (RBCs). \\nOne of the main initial goals of this study was to devise a test protocol that would be sensitive and reliable, yet require no specialized laboratory equipment such as adjustable pipets, so that it could be performed in the most remote corners of the world by people with minimal levels of training. Because antibody levels against the viral RBD have been found to correlate closely with sero-neutralisation titers, and thus with protection against reinfection, it has become obvious during the course of this study that making this test reliably quantitative would be a further significant advantage. \\nWe have found that, in PBN, a buffer which contains BSA and sodium azide, IH4-RBD is stable for over 6 months at room temperature, and that PBN also improves HAT performance compared to using straight PBS. We also show that performing HAT at either 4°C, room temperature or 37°C has minimal influence on the results, and that quantitative evaluation of the levels of antibodies directed against the SARS-CoV-2 RBD can be achieved in a single step using titration of the IH4-RBD reagent.\\nThe HAT-field protocol described here requires only very simple disposable equipment and a few microliters of whole blood, such as can be obtained by finger prick. Because it is based on a single soluble reagent, the test can be adapted very simply and rapidly to detect antibodies against variants of the SARS-CoV-2, or conceivably against different pathogens.\\nHAT-field appears well suited to provide quantitative assessments of the serological protection of populations as well as individuals, and given its very low cost, the stability of the IH4-RBD reagent in the adapted buffer, and the simplicity of the procedure, could be deployed pretty much anywhere, including in the poorest countries and the most remote corners of the globe.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.475644', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The C-terminal domain of SEC-10 is fundamental for exocyst function, Spitzenkörper organization and cell morphogenesis in Neurospora crassa', 'authors': \"[{'name': 'Alfredo Figueroa-Meléndez'}, {'name': 'Leonora Martínez-Núñez'}, {'name': 'Adriana M. Rico-Ramírez'}, {'name': 'Juan M. Martínez-Andrade'}, {'name': 'Mary Munson'}, {'name': 'Meritxell Riquelme'}]\", 'abstract': '<p>The exocyst is a conserved multimeric complex that participates in the final steps of the secretion of vesicles. In the filamentous fungus <italic>Neurospora crassa,</italic> the exocyst is crucial for polar growth, morphology, and the organization of the Spitzenkörper (Spk), the apical body where secretory vesicles accumulate before being delivered to the plasma membrane. In the highly polarized cells of <italic>N. crassa,</italic> the exocyst subunits SEC-3, SEC-5, SEC-6, SEC-8, and SEC-15 were previously found localized at the plasma membrane of the cells’ apices, while EXO-70 and EXO-84 occupied the frontal outer layer of the Spk, occupied by vesicles. The localization of SEC-10 had remained so far elusive. In this work, SEC-10 was tagged with the green fluorescent protein (GFP) either at its N- or C-terminus and found localized at the plasma membrane of growing hyphal tips, similar to what was previously observed for some exocyst subunits. While expression of an N-terminally tagged version of SEC-10 at its native locus was fully viable, expression of a C-terminally tagged version at its native locus resulted in severe hyphal growth and polarity defects. Additionally, a <italic>sec-10</italic> knockout mutant in a heterokaryotic state (with genetically different nuclei) was viable but showed a strongly aberrant phenotype, confirming that this subunit is essential to maintain hyphal morphogenesis. Transmission electron microscopy analysis revealed the lack of a Spk in the SEC-10-GFP strain, suggesting a critical role of the exocyst in the vesicular organization at the Spk. Mass spectrometry analysis revealed fewer peptides of exocyst subunits interacting with SEC-10-GFP than with GFP-SEC-10, suggesting an essential role of the C-terminus of SEC-10 in exocyst assembly and/or stability. Altogether, our data suggest that an unobstructed C-terminus of SEC-10 is indispensable for the exocyst complex function and that a GFP tag could be blocking important subunit-subunit interactions.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.16.476525', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'RT-qPCR as a screening platform for mutational and small molecule impacts on structural stability of RNA tertiary structures', 'authors': \"[{'name': 'Martina Zafferani'}, {'name': 'Dhanasheel Muralidharan'}, {'name': 'Amanda E Hargrove'}]\", 'abstract': \"<p>The exponential increase in the discovery and characterization of RNA tertiary structures has highlighted their active role in a variety of human disease, yet often their interactome and specific function remain unknown. Small molecules offer opportunities to both decode these cellular roles and develop therapeutics, yet there are few examples of small molecules that target biologically relevant RNA tertiary structures. While RNA triple helices are a particular attractive target, discovery of triple helix modulators has been hindered by the lack of correlation between small molecule affinity and effect on structural modulation, thereby limiting the utility of affinity-based screening as a primary filtering method. To address this challenge, we developed a high-throughput RT-qPCR screening platform that reports on the effect of mutations and additives, such as small molecules, on the structuredness of triple helices. Using the 3'-end of the oncogenic non-coding RNA MALAT1 as an example, we demonstrated the applicability of both a two-step and a one-pot method to assess the impact of mutations and small molecules on the stability of the triple helix. Employment of a functional high-throughput assay as a primary screen will significantly expedite the discovery of probes that modulate RNA triple helices structural landscape and, consequently, help gain insight into the roles of these pervasive structures.</p>\", 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.14.476296', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'A Four-Step Enzymatic Cascade for Efficient Production of L- Phenylglycine from Biobased L-Phenylalanine', 'authors': \"[{'name': 'Yuling Zhu'}, {'name': 'Jifeng Yuan'}]\", 'abstract': '<p>Enantiopure amino acids are of particular interest in the agrochemical and pharmaceutical industries. Here, we reported a multi-enzyme cascade for efficient production of L-phenylglycine (L-Phg) from biobased L-phenylalanine (L-Phe). We first attempted to engineer <italic>Escherichia coli</italic> for expressing L-amino acid deaminase (LAAD) from <italic>Proteus mirabilis</italic>, hydroxymandelate synthase (HmaS) from <italic>Amycolatopsis orientalis</italic>, (<italic>S</italic>)-mandelate dehydrogenase (SMDH) from <italic>Pseudomonas putida</italic>, the endogenous aminotransferase (AT) encoded by <italic>ilvE</italic> and L-glutamate dehydrogenase (GluDH) from <italic>E. coli</italic>. However, 10 mM L-Phe only afforded the synthesis of 7.21 ± 0.15 mM L-Phg. The accumulation of benzoylformic acid suggested that the transamination step might be rate-limiting. We next used leucine dehydrogenase (LeuDH) from <italic>Bacillus cereus</italic> to bypass the use of L-glutamate as amine donor, and 40 mM L-Phe gave 39.97 ± 3.84 mM (6.04 ± 0.58 g/L) L-Phg, reaching 99.9% conversion. In summary, this work demonstrated a concise four-step enzymatic cascade for the L-Phg synthesis from biobased L-Phe, with a potential for future industrial applications.</p><sec><title>Graphical abstract</title><p>a concise four-step enzymatic cascade for the L-phenylglycine synthesis from biobased L-phenylalanine was devised. 40 mM L-phenylalanine afforded the synthesis of 39.97 ± 3.84 mM (6.04 ± 0.58 g/L) L-phenylglycine, reaching 99.9% conversion.</p><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"476296v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.14.476303', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Host related factors determine co-occurrence patterns between pathogenic bacteria, protozoa, and helminths in populations of the multimammate mouse, Mastomys natalensis', 'authors': \"[{'name': 'joachim Marien'}, {'name': 'Bram Vanden Broecke'}, {'name': 'Pamela June Tafompa'}, {'name': 'Lisse Bernaerts'}, {'name': 'Alexis Ribas Salvador'}, {'name': 'Ladslaus L. Mnyone'}, {'name': 'Loth Mulungu'}, {'name': 'Herwig Leirs'}]\", 'abstract': '<p>Advances in experimental and theoretical work increasingly suggest that parasite interactions within a single host can affect the spread and severity of wildlife diseases. Yet empirical data to support predicted co-infection patterns are limited due to the practical challenges of gathering convincing data from animal populations and the stochastic nature of parasite transmission. Here, we investigated co-infection patterns between micro- (bacteria and protozoa) and macroparasites (gastrointestinal helminths) in natural populations of the multimammate mouse (Mastomys natalensis). Fieldwork was performed in Morogoro (Tanzania), where we trapped 211 individual M. natalensis and tested their behavior using a modified open-field arena. All animals were checked on the presence of helminths in their gastrointestinal tract, three bacteria (Anaplasma, Bartonella, and Borrelia) and two protozoan genera (Piroplasma and Hepatozoon). Besides the presence of eight different helminth genera (reported earlier), we found that 21% of M. natalensis were positive for Anaplasma, 13% for Bartonella, and 2% for Hepatozoon species. Hierarchical modelling of species communities was used to investigate the effect of the different host-related factors on these parasites infection probability and community structure. Our results show that the infection probability of Anaplasma and Bartonella was higher in adults than juveniles. We also observed that females and less explorative individuals had a higher infection probability with Bartonella. We found limited support for within-host interactions between micro-and macroparasites, as only animals infected with Bartonella were significantly more likely to be infected with Protospirura, Trichuris, and Trichostrongylidae helminths.</p>', 'posted': '2022-01-15'}, {'doi': '10.1101/2022.01.14.476310', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Non-target effects of ten essential oils on the egg-parasitoid Trichogramma evanescens', 'authors': \"[{'name': 'Louise van Oudenhove'}, {'name': 'Aurelie Cazier'}, {'name': 'Marine Fillaud'}, {'name': 'Anne-Violette Lavoir'}, {'name': 'Hicham Fatnassi'}, {'name': 'Guy Perez'}, {'name': 'Vincent Calcagno'}]\", 'abstract': \"<p>Essential oils (EOs) are increasingly used as biopesticides due to their insecticidal potential. This study addresses their non-target effects on a biological control agent: the egg parasitoid <italic>Trichogramma evanescens</italic>. In particular, we tested whether EOs affected parasitoid fitness either directly, by decreasing pre-imaginal survival, or indirectly, by disrupting parasitoids' orientation abilities. The effect of Anise, Fennel, Sweet orange,\\nBasil, Coriander, Oregano, Peppermint, Mugwort, Rosemary and Thyme EOs were studied on five strains of <italic>T. evanescens</italic>. Specific experimental setups were developed, and data obtained from image analysis were interpreted with phenomenological models fitted with Bayesian inference. Results highlight the fumigant toxicity of EOs on parasitoid development. Anise, Fennel, Basil, Coriander, Oregano, Peppermint and Thyme EOs are particularly toxic and drastically reduce the emergence rate of <italic>T. evanescens</italic>. Most EOs also affect parasitoid behavior: (i) Basil, Coriander, Oregano, Peppermint, Mugwort and Thyme EOs are highly repellent for naive female parasitoids; (ii) Anise and Fennel EOs can have repellent to attractive effects depending on strains; and (iii) Sweet orange, Oregano and Rosemary EOs have no detectable impact on orientation behavior. This study shows that EOs fumigation have non-target effects on egg parasitoids. This highlights the need to cautiously precise the deployment framework of biopesticides in an agroecological perspective.</p>\", 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.16.476512', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Blocking Kir6.2 channels with SpTx1 potentiates glucose-stimulated insulin secretion from murine pancreatic β cells and lowers blood glucose in diabetic mice', 'authors': \"[{'name': 'Zhe Lu'}, {'name': 'Yajamana Ramu'}, {'name': 'Jayden Yamakaze'}, {'name': 'Yufeng Zhou'}, {'name': 'Toshinori Hoshi'}]\", 'abstract': '<p>ATP-sensitive K<sup>+</sup> (K<sub>ATP</sub>) channels in pancreatic β cells comprise pore-forming subunits (Kir6.2) and modulatory sulfonylurea receptor subunits (SUR1). The ATP sensitivity of these channels enables them to couple metabolic state to insulin secretion in β cells. Antidiabetic sulfonylureas such as glibenclamide target SUR1 and indirectly suppress Kir6.2 activity. Glibenclamide acts as both primary and secondary secretagogues to trigger insulin secretion and potentiate glucose-stimulated insulin secretion, respectively. We tested whether blocking Kir6.2 itself causes the same effects as glibenclamide, and found that the Kir6.2 pore-blocker SpTx1 acts as a strong secondary, but not a primary, secretagogue. SpTx1 triggered a transient rise of plasma insulin and lowered the elevated blood glucose of diabetic mice over-expressing Kir6.2 but did not affect those of non-diabetic mice. This proof-of-concept study suggests that blocking Kir6.2 may serve as an effective treatment for diabetes and other diseases stemming from Kir6.2 hyperactivity that cannot be suppressed with sulfonylureas.</p>', 'posted': '2022-01-17'}, {'doi': '10.1101/2022.01.13.22269272', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Cardiovascular Medicine', 'covid': False, 'title': 'Association of Retinal Age Gap with Arterial Stiffness and Incident Cardiovascular Disease', 'authors': \"[{'name': 'Zhuoting Zhu'}, {'name': 'Yifan Chen'}, {'name': 'Wei Wang'}, {'name': 'Yueye Wang'}, {'name': 'Wenyi Hu'}, {'name': 'Xianwen Shang'}, {'name': 'Huan Liao'}, {'name': 'Danli Shi'}, {'name': 'Yu Huang'}, {'name': 'Jason Ha'}, {'name': 'Zachary Tan'}, {'name': 'Katerina Kiburg'}, {'name': 'Xueli Zhang'}, {'name': 'Shulin Tang'}, {'name': 'Honghua Yu'}, {'name': 'Xiaohong Yang'}, {'name': 'Mingguang He'}]\", 'abstract': '<p>Background: Retinal parameters could reflect systemic vascular changes. With the advances of deep learning technology, we have recently developed an algorithm to predict retinal age based on fundus images, which could be a novel biomarker for ageing and mortality.\\nObjective: To investigate associations of retinal age gap with arterial stiffness index (ASI) and incident cardiovascular disease (CVD). \\nMethods: A deep learning (DL) model was trained based on 19,200 fundus images of 11,052 participants without any past medical history at baseline to predict the retinal age. Retinal age gap (retinal age predicted minus chronological age) was generated for the remaining 35,917 participants. Regression models were used to assess the association between retinal age gap and ASI. Cox proportional hazards regression models and restricted cubic splines were used to explore the association between retinal age gap and incident CVD. \\nResults: We found each one-year increase in retinal age gap was associated with increased ASI (β=0.002, 95% confidence interval [CI]: 0.001-0.003, P&lt;0.001). After a median follow-up of 5.83 years (interquartile range [IQR]: 5.73-5.97), 675 (2.00%) developed CVD. In the fully adjusted model, each one-year increase in retinal age gap was associated with a 3% increase in the risk of incident CVD (hazard ratio [HR]=1.03, 95% CI: 1.01-1.06, P=0.012). In the restricted cubic splines analysis, the risk of incident CVD increased significantly when retinal age gap reached 1.21 (HR=1.05; 95% CI, 1.00-1.10; P-overall &lt;0.0001; P-nonlinear=0.0681).\\nConclusion: We found that retinal age gap was significantly associated with ASI and incident CVD events, supporting the potential of this novel biomarker in identifying individuals at high risk of future CVD events.</p>', 'posted': '2022-01-14'}, {'doi': '10.1101/2022.01.11.22269068', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'The impact of post-hospital remote monitoring of COVID-19 patients using pulse oximetry: a national observational study using hospital activity data', 'authors': \"[{'name': 'Theo Georghiou'}, {'name': 'Chris Sherlaw-Johnson'}, {'name': 'Efthalia Massou'}, {'name': 'Stephen Morris'}, {'name': 'Nadia E Crellin'}, {'name': 'Lauren Herlitz'}, {'name': 'Manbinder S Sidhu'}, {'name': 'Sonila M Tomini'}, {'name': 'Cecilia Vindrola-Padros'}, {'name': 'Holly Walton'}, {'name': 'Naomi J Fulop'}]\", 'abstract': '<sec><title>Background</title><p>There was a national roll out of ‘COVID Virtual Wards’ (CVW) during England’s second COVID-19 wave (Autumn 2020 – Spring 2021). These services used remote pulse oximetry monitoring for COVID-19 patients following discharge from hospital. A key aim was to enable rapid detection of patient deterioration. It was anticipated that the services would support early discharge and avoid readmissions, reducing pressure on beds. This study is an evaluation of the impact of the CVW services on hospital activity.</p></sec><sec><title>Methods</title><p>Using retrospective patient-level hospital admissions data, we built multivariate models to analyse the relationship between the implementation of CVW services and hospital activity outcomes: length of COVID-19 related stays and subsequent COVID-19 readmissions within 28 days. We used data from more than 98% of recorded COVID-19 hospital stays in England, where the patient was discharged alive between mid-August 2020 and late February 2021.</p></sec><sec><title>Findings</title><p>We found a longer length of stay for COVID-19 patients discharged from hospitals where a CVW was available, when compared to patients discharged from hospitals where there was no CVW (adjusted IRR 1·05, 95% CI 1·01 to 1·09). We found no evidence of a relationship between the availability of CVW and subsequent rates of readmission for COVID-19 (adjusted OR 0·95, 95% CI 0·89 to 1·02).</p></sec><sec><title>Interpretation</title><p>We found no evidence of early discharges or reduced readmissions associated with the roll out of COVID Virtual Wards across England. Our analysis made pragmatic use of national-scale hospital data, but it is possible that a lack of specific data (for example, on which patients were enrolled) may have meant that true impacts, especially at a local level, were not ultimately discernible.</p></sec><sec><title>Funding</title><p>This is independent research funded by the National Institute for Health Research, Health Services &amp; Delivery Research programme and NHSEI.</p></sec><sec><title>Research in context</title><sec><title>Evidence before this study</title><p>Post-hospital virtual wards have been found to have a positive impact on patient outcomes when focussed on patients with specific diseases, for example those with heart disease. There has been less evidence of impact for more heterogenous groups of patients. While these services have been rolled out at scale in England, there has been little evidence thus far that post-hospital virtual wards (using pulse oximetry monitoring) have helped to reduce the length of stay of hospitalised COVID-19 patients, or rates of subsequent readmissions for COVID-19.</p></sec><sec><title>Added value of this study</title><p>This national-scale study provides evidence that the rollout of post-hospital discharge virtual ward services for COVID-19 patients in England did not reduce lengths of stay in hospital, or rates of readmission.</p></sec><sec><title>Implications of all the available evidence</title><p>While there is currently an absence of evidence of positive impacts for COVID-19 patients discharged to a virtual ward, our study emphasises the need for quality data to be collected as part of future service implementation.</p></sec></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.22269075', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Respiratory Medicine', 'covid': False, 'title': 'Genome-wide association study of chronic sputum production implicates loci involved in mucus production and infection', 'authors': \"[{'name': 'R Packer'}, {'name': 'N Shrine'}, {'name': 'R Hall'}, {'name': 'C Melbourne'}, {'name': 'R Thompson'}, {'name': 'AT Williams'}, {'name': 'ML Paynton'}, {'name': 'J Davitte'}, {'name': 'E Hessel'}, {'name': 'D Michalovich'}, {'name': 'JC Betts'}, {'name': 'I Sayers'}, {'name': 'A Yeo'}, {'name': 'IP Hall'}, {'name': 'MD Tobin'}, {'name': 'LV Wain'}]\", 'abstract': '<sec><title>Background</title><p>Chronic sputum production impacts on quality of life and is a feature of many respiratory diseases. Identification of the genetic variants associated with chronic sputum production in a disease agnostic sample could improve understanding of its causes and identify new molecular targets for treatment.</p></sec><sec><title>Methods</title><p>We conducted a genome-wide association study (GWAS) of chronic sputum production in UK Biobank. Signals meeting genome-wide significance (P&lt;5×10<sup>−8</sup>) were fine-mapped and putative causal genes identified by gene expression analysis. GWAS of respiratory traits were interrogated to identify whether the signals were driven by existing respiratory disease amongst the cases and variants were further investigated for wider pleiotropic effects using phenome-wide association studies (PheWAS).</p></sec><sec><title>Findings</title><p>From a GWAS of 9,714 cases and 48,471 controls, we identified six novel genome-wide significant signals for chronic sputum production including the Human Leukocyte Antigen (HLA) locus, chromosome 11 mucin locus (containing <italic>MUC2, MUC5AC</italic> and <italic>MUC5B</italic>) and the <italic>FUT2</italic> locus. The mucin locus signal had previously been reported for association with moderate-to-severe asthma. The HLA signal was fine-mapped to an amino-acid change of threonine to arginine (frequency 36.8%) in HLA-DRB1 (HLA-<italic>DRB1</italic>*03:147). The signal near <italic>FUT2</italic> was associated with expression of several genes including <italic>FUT2</italic>, for which the direction of effect was tissue dependent. Our PheWAS identified a wide range of associations.</p></sec><sec><title>Interpretation</title><p>Novel signals at the <italic>FUT2</italic> and mucin loci highlight mucin flucosylation as a driver of chronic sputum production even in the absence of diagnosed respiratory disease and provide genetic support for this pathway as a target for therapeutic intervention.</p></sec>', 'posted': '2022-01-11'}, {'doi': '10.1101/2022.01.11.475794', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Teasing apart the joint effect of demography and natural selection in the birth of a contact zone', 'authors': \"[{'name': 'Lili Li'}, {'name': 'Pascal Milesi'}, {'name': 'Mathieu Tiret'}, {'name': 'Jun Chen'}, {'name': 'Janek Sendrowski'}, {'name': 'John Baison'}, {'name': 'Zhiqiang Chen'}, {'name': 'Linghua Zhou'}, {'name': 'Bo Karlsson'}, {'name': 'Mats Berlin'}, {'name': 'Johan Westin'}, {'name': 'Rosario Garcia-Gil'}, {'name': 'Harry Wu'}, {'name': 'Martin Lascoux'}]\", 'abstract': '<p>Vast population movements induced by recurrent climatic cycles have shaped the genetic structure of plant species. This is especially true in Scandinavia that was repeatedly glaciated. During glacial periods trees were confined to refugia, south and east of the ice sheet, from which they recolonized Scandinavia as the ice melted away. This multi-pronged recolonization led to large contact zones in most species. We leverage large genomic data from 5000 trees to reconstruct the demographic history of Norway spruce (<italic>Picea abies</italic>) and test for the presence of natural selection during the recolonization process and the establishment of the contact zone. Sweden is today made up of two large genetic clusters, a southern one originating from the Baltics and a Northern one originating from Northern Russia. The contact zone delineating these two clusters closely matches the limit between two major climatic regions. This suggests that natural selection contributed to the establishment and the maintenance of the contact zone. To test this hypothesis we first used Approximate Bayesian Computation; an Isolation-with migration model with genome-wide linked selection fits the data better than a purely neutral one. Secondly, we identified loci characterized by both extreme allele frequency differences between geographic regions and association to the variables defining the climatic zones. These loci, many of which are related to phenology, form clusters present on all linkage groups. Altogether, the current genetic structure reflects the joint effect of climatic cycles, recolonization and selection on the establishment of strong local adaptation and contact zones.</p><sec><title>Significance Statement</title><p>Understanding how past climatic events, human actions and evolutionary forces contributed to the present distribution of genetic diversity is crucial to predict their reaction to the current climate crisis. Vast distribution shifts induced by past environmental changes, local ecological processes, natural selection and human transfers contributed to the current distribution of Norway spruce across Northern Europe. Genome-wide polymorphisms from thousands of individuals show that Scandinavia was recolonized after the Last Glacial from both south and north. This two-pronged recolonization established a contact zone between two genetic clusters that matches the limit between two major climate zones. The contact zone is shaped and maintained by natural selection on a large number of loci that form blocks of co-adapted loci spread genome-wide.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475845', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Differentiation of a CD4+/CD8αβ+ Double Positive T Cell Population From The CD8 Pool Is Sufficient To Mediate Graft-vs-Host Disease but not Graft-vs-Leukemia Effects', 'authors': \"[{'name': 'Nicholas J. Hess'}, {'name': 'David P. Turicek'}, {'name': 'Kalyan Nadiminti'}, {'name': 'Amy Hudson'}, {'name': 'Peiman Hematti'}, {'name': 'Jenny E. Gumperz'}, {'name': 'Christian M. Capitini'}]\", 'abstract': '<p>Acute graft-vs-host disease (aGVHD) and tumor relapse remain the primary complications following allogeneic hematopoietic stem cell transplantation (allo-HSCT) for malignant blood disorders. While post-transplant cyclophosphamide has reduced the overall prevalence and severity of aGVHD, relapse rates remain a concern. Thus, there remains a need to identify the specific human T cell subsets mediating GVHD pathology versus graft-versus-leukemia (GVL) effects. In xenogeneic transplantation studies using primary human cells from a variety of donors and tissue sources, we observed the development of a mature CD4<sup>+</sup>/CD8αβ<sup>+</sup> double positive T cell (DPT) population in mice succumbing to lethal aGVHD but not in mice that failed to develop aGVHD. The presence of DPT, irrespective of graft source, was predictive of lethal GVHD as early as one week after xenogeneic transplantation. DPT co-express the master transcription factors of the CD8 and CD4 lineages, RUNX3 and THPOK respectively, and produce both cytotoxic and modulatory cytokines. To identify the origin of DPT, we transplanted isolated human CD4 or CD8 T cells, which in turn revealed that DPT only arise from the CD8 pool. Interestingly, re-transplantation of sorted CD8 T cells from GVHD mice did not reveal a second wave of DPT differentiation. Re-transplantation of flow-sorted DPT, CD8 or CD4 T cells from GVHD mice revealed that DPT are sufficient to mediate GVHD pathology but not GVL effects versus B-cell acute lymphoblastic leukemia. Lastly, we confirmed the presence and correlation of DPT with aGVHD pathology in a small cohort of allo-HSCT patients that developed grade 2-4 aGVHD in our clinic. Further understanding of DPT differentiation and pathology may lead to targeted prophylaxis and/or treatment regimens for aGVHD and potentially other human chronic inflammatory diseases.</p><sec><title>Key Points</title><list list-type=\"order\"><list-item><p>Human DPT cell differentiation is a predictive metric of xenogeneic GVHD lethality</p></list-item><list-item><p>The origin of DPT are CD8 T cells that gain THPOK expression and CD4 lineage effector functions</p></list-item><list-item><p>DPT cells are sufficient to mediate GVHD pathology but not GVL effects</p></list-item></list></sec><sec><title>Visual Abstract</title><fig id=\"ufig1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"475845v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.10.475716', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'How many bits of information can be transferred between residues in a protein and how fast?', 'authors': \"[{'name': 'Aysima Hacisuleyman'}, {'name': 'Burak Erman'}]\", 'abstract': '<p>Time resolved Raman and infrared spectroscopy experiments show the basic features of information transfer between residues in proteins. Here, we present the theoretical basis of information transfer using a simple elastic net model and recently developed entropy transfer concept in proteins. Mutual information between two residues is a measure of communication in proteins which shows the maximum amount of information that may be transferred between two residues. However, it does not explain the actual amount of transfer nor the transfer rate of information between residues. For this, dynamic equations of the system are needed. We used the Schreiber theory of information transfer and the Gaussian network Model of proteins, together with the solution of the Langevin equation, to quantify allosteric information transfer. Results of the model are in perfect agreement with ultraviolet resonance Raman measurements. Analysis of the allosteric protein Human NAD-dependent isocitrate dehydrogenase shows that a multitude of paths contribute collectively to information transfer. While the peak values of information transferred are small relative to information content of residues, considering the estimated transfer rates, which are in the order of megabits per second, sustained transfer during the activity time-span of proteins may be significant.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475884', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Identification of Mfn2-S249 as a Phosphoregulatory Switch of Mitochondrial Fusion Dynamics', 'authors': \"[{'name': 'Sanjay Kumar'}, {'name': 'Aaron Ramonett'}, {'name': 'Tasmia Ahmed'}, {'name': 'Eun-A Kwak'}, {'name': 'Paola Cruz Flores'}, {'name': 'Hannah R. Ortiz'}, {'name': 'Paul R. Langlais'}, {'name': 'Karthikeyan Mythreye'}, {'name': 'Nam Y. Lee'}]\", 'abstract': '<p>Mitochondrial remodeling is a fundamental process underlying cellular respiration and metabolism. Here we report TAK1 as a direct regulator of mitochondrial fusion. TAK1 is activated by a variety of mitogenic factors, cytokines and environmental stimuli, which we find induces rapid fragmentation through Mfn2 inactivation. TAK1 phosphorylates Mfn2 at Ser249, which inhibits the binding of GTP required for Mfn trans-dimerization and mitochondrial membrane fusion. Accordingly, expression of Mfn2-S249 phosphomimetics (Mfn2-E/D) constitutively promote fission whereas alanine mutant (Mfn2-A) yields hyperfused mitochondria and increased bioenergetics in cells. In mice, Mfn2-E knock-in yields embryonic lethality in homozygotes whereas heterozygotes are viable but exhibit increased visceral fat accumulation despite normal body weight and cognitive/motor functions compared to wildtype and Mfn2-A mice. Mature white adipocytes isolated from mutant mice reveal cell-autonomous TAK1-related effects on mitochondrial remodeling and lipid metabolism. These results identify Mfn2-S249 as a dynamic phosphoregulatory switch of mitochondrial fusion during development and energy homeostasis.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.15.476461', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Culture and genomic analysis of Symbiopectobacterium purcellii, gen.nov. sp. nov., isolated from the leafhopper Empoasca decipiens', 'authors': \"[{'name': 'Pol Nadal-Jimenez'}, {'name': 'Stefanos Siozios'}, {'name': 'Nigel Halliday'}, {'name': 'Miguel Camara'}, {'name': 'Greg D.D. Hurst'}]\", 'abstract': '<p>Bacterial endosymbionts are found in multiple arthropod species, where they play crucial roles as nutritional symbionts, defensive symbionts or reproductive parasites. Recent work has highlighted a new clade of heritable microbes within the gammaproteobacteria that enter into both obligate and facultative symbioses, with an obligately required unculturable symbiont recently given the name Cand. Symbiopectobacterium. In this study, we describe a culturable rod shaped non-flagellated bacterial symbiont from this clade isolated from the leafhopper Empoasca decipiens. The symbiont is related to the transovarially-transmitted \\'BEV\\' bacterium that was first isolated from the leafhopper Euscelidius variegatus by Alexander Purcell, and we therefore name the symbiont Symbiopectobacterium purcellii sp. nov. gen. nov. We further report the closed genome sequence for S. purcellii. The genome is atypical for a heritable microbe, being large in size, without profound AT bias and with little evidence of pseudogenization. The genome is predicted to encode Type II, III and VI secretion systems and associated effectors and a non-ribosomal peptide synthase array likely to produce bioactive small molecules. Predicted metabolism is more complete than for other symbionts in the Symbiopectobacterium clade, and the microbe is predicted to synthesize a range of B vitamins. However, Biolog plate analysis indicate metabolism is depauperate compared to the sister clade, represented by Pectobacterium carotovorum. A quorum-sensing pathway related to that of Pectobacterium spp. (containing an overlapping expI-expR1 pair in opposite directions and a \"solo\" expR2) is evidenced, and LC-MS/MS analysis reveals the presence of 3-hydroxy-C10-HSL as the sole N-acylhomoserine lactone (AHL) in our strain. This AHL profile is profoundly divergent from that of other Erwinia and Pectobacterium spp., which produce mostly 3-oxo-C6- and 3-oxo-C8-HSL and could aid group identification. Thus, this microbe denotes one that has lost certain pathways associated with a saprophytic lifestyle but represents an important baseline against which to compare other members of the genus Symbiopectobacterium that show more profound integration into host biology.</p>', 'posted': '2022-01-16'}, {'doi': '10.1101/2022.01.11.475896', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evolution under juvenile malnutrition impacts adult metabolism and fitness', 'authors': \"[{'name': 'Berra Erkosar'}, {'name': 'Cindy Dupuis'}, {'name': 'Fanny Cavigliasso'}, {'name': 'Loriane Savary'}, {'name': 'Laurent Kremmer'}, {'name': 'Hector Gallart-Ayala'}, {'name': 'Julijana Ivanisevic'}, {'name': 'Tadeusz J. Kawecki'}]\", 'abstract': '<p>Juveniles are often first to suffer from nutrient shortage, and juvenile undernutrition is likely an important force of natural selection shaping animal physiology, with consequences potentially extending into adulthood. We combined RNAseq, targeted metabolomics and genomics to study the consequences of experimental evolution under juvenile undernutrition for metabolism of reproductively active adult females of <italic>Drosophila melanogaster</italic>. Compared to six Control populations maintained on standard diet, six Selected populations evolved for over 230 generations on a nutrient-poor larval diet showed major changes in adult gene expression and metabolite abundance. In particular, Selected flies were relatively deficient in essential amino acids and purine nucleotides, but showed overabundance of several non-essential amino-acids involved in purine synthesis and overexpression of multiple enzymes catalyzing this pathway. Selected flies also accumulated medium-chain acylcarnitines suggestive of congestion in beta-oxidation, possibly linked to deficiency of electron transporters. Some aspects of the metabolic profile of Selected flies resembled that of flies subject to starvation. Furthermore, differences between Selected and Control populations in adult gene expression were in general positively correlated with differences in larval expression, consistent with pleiotropy in gene regulation between the life stages. Finally, Selected flies were less fit in terms of fecundity than Controls even when both were raised under the conditions under which the Selected populations evolved. These results suggest that evolutionary adaptation to juvenile undernutrition has large pleiotropic consequences for adult metabolism, and that they are costly rather than adaptive for adult fitness.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475974', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A Longitudinal Study of Dominant E. coli Lineages and Antimicrobial Resistance in the Gut of Children Living in an Upper Middle-income Country', 'authors': \"[{'name': 'Diana Calderón'}, {'name': 'Paúl A. Cárdenas'}, {'name': 'Belen Prado'}, {'name': 'Jay P. Graham'}, {'name': 'Gabriel Trueba'}]\", 'abstract': \"<p>The gastrointestinal tract constitutes a complex and diverse ecosystem. <italic>Escherichia coli</italic> is one of the most frequently studied and characterized species in the gut ecosystem, nevertheless, there has been little research to determine their diversity and population dynamics in the intestines of children over time. In this prospective study, a fresh fecal sample was obtained from children longitudinally over one year (30 fecal samples at sampling period 1 and 22 fecal samples at sampling periods 2 and 3). From each stool sample, five <italic>E. coli</italic> colonies were randomly selected (n = 405 <italic>E. coli</italic> isolates total) in order to characterize the genotype and phenotypic antimicrobial resistance patterns. We found that all numerically dominant <italic>E. coli</italic> lineages in children’s intestines were transient colonizers, and antimicrobial resistance phenotypes of these strains varied significantly over time without any apparent selective force. Whole-genome sequencing of 3 isolates belonging to ST131 found in one child during the sampling period I and II indicated that isolates were three different ST 131 clones that carried extended-spectrum β-lactamase (ESBL) genes.</p><sec><title>IMPORTANCE</title><p>The length of residency and numeric dominance of antimicrobial-resistant <italic>E. coli</italic> may affect the extent to which an isolate contributes to the dissemination of antimicrobial resistance in a community. We studied the persistence of numerically dominant and antimicrobial-resistant lineages of <italic>E. coli</italic> in the human intestine. We found that <italic>E. coli</italic> lineages in the children's gut change considerably and rapidly over time. This study suggests that some phenotypic resistance patterns may result from the random distribution of genes in <italic>E. coli</italic> populations over time and may not be associated with differential exposure to antimicrobials.</p></sec>\", 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.475264', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': True, 'title': 'Hypercapnia limits β-catenin-mediated alveolar type 2 cell progenitor function by altering Wnt production from adjacent fibroblasts', 'authors': \"[{'name': 'Laura A. Dada'}, {'name': 'Lynn C. Welch'}, {'name': 'Natalia D. Magnani'}, {'name': 'Ziyou Ren'}, {'name': 'Patricia L. Brazee'}, {'name': 'Diego Celli'}, {'name': 'Annette S. Flozak'}, {'name': 'Anthea Weng'}, {'name': 'Mariana Maciel-Herrerias'}, {'name': 'Vitalii Kryvenko'}, {'name': 'István Vadász'}, {'name': 'Constance E Runyan'}, {'name': 'Hiam Abdala-Valencia'}, {'name': 'Masahiko Shigemura'}, {'name': 'S. Marina Casalino-Matsuda'}, {'name': 'Alexander V. Misharin'}, {'name': 'G.R. Scott Budinger'}, {'name': 'Cara J. Gottardi'}, {'name': 'Jacob I. Sznajder'}]\", 'abstract': '<p>Persistent symptoms and radiographic abnormalities suggestive of failed lung repair are among the most common symptoms in patients with COVID-19 after hospital discharge. In mechanically ventilated patients with ARDS secondary to SARS-CoV-2 pneumonia, low tidal volumes to reduce ventilator-induced lung injury necessarily elevate blood CO<sub>2</sub> levels, often leading to hypercapnia. The role of hypercapnia on lung repair after injury is not completely understood. Here, we show that hypercapnia limits β-catenin signaling in alveolar type 2 (AT2) cells, leading to reduced proliferative capacity. Hypercapnia alters expression of major Wnts in PDGFRa+-fibroblasts from those maintaining AT2 progenitor activity and towards those that antagonize β-catenin signaling and limit progenitor function. Activation of β-catenin signaling in AT2 cells, rescues the inhibition AT2 proliferation induced by hypercapnia. Inhibition of AT2 proliferation in hypercapnic patients may contribute to impaired lung repair after injury, preventing sealing of the epithelial barrier, increasing lung flooding, ventilator dependency and mortality.</p>', 'posted': '2022-01-13'}, {'doi': '10.1101/2022.01.12.476061', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'A survey on antimicrobial resistance genes of frequently used bacteria in kefir and yoghurt', 'authors': \"[{'name': 'Maura Fiona Judge'}, {'name': 'Adrienn Gréta Tóth'}, {'name': 'Sára Ágnes Nagy'}, {'name': 'Márton Papp'}, {'name': 'Norbert Solymosi'}]\", 'abstract': '<p>Antimicrobial resistance (AMR) is one of the foremost threats facing the treatment of infectious diseases worldwide. Recent studies have highlighted the potential for antimicrobial resistance genes (ARGs) in fermented foods to contribute to AMR via horizontal gene transfer (HGT). The focus of our study was investigating the ARG content (resistome) and mobility potential of the ARGs (mobilome) of bacterial strains commonly used in probiotic products, namely yoghurt and kefir. We performed metagenomic analyses on freely available data sets (n=584) originating from various kefir and yoghurt strains using next generation sequencing (NGS) in order to gain an insight into the ARG diversity, frequency and mobility. Our study shows that kefir and yoghurt products carry diverse and significant amounts of ARGs and that these genes may often be associated with iMGEs or plasmids, conferring mobility. Certain bacteria species such as <italic>Bifidobacterium animalis</italic> and <italic>Streptococcus thermophilus</italic> were found to have higher ARG content. Overall, our results support the hypothesis that ARGs are present in fermented foods, namely yoghurt and kefir, and have the potential to contribute to AMR.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.476046', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Optimal protein production by a synthetic microbial consortium: Coexistence, distribution of labor, and syntrophy', 'authors': \"[{'name': 'Carlos Martínez'}, {'name': 'Eugenio Cinquemani'}, {'name': 'Hidde de Jong'}, {'name': 'Jean-Luc Gouzé'}]\", 'abstract': '<p>The bacterium <italic>E. coli</italic> is widely used to produce recombinant proteins such as growth hormone and insulin. One inconvenience with <italic>E. coli</italic> cultures is the secretion of acetate through overflow metabolism. Acetate inhibits cell growth and represents a carbon diversion, which results in several negative effects on protein production. One way to over-come this problem is the use of a synthetic consortium of two different <italic>E. coli</italic> strains, one producing recombinant proteins and one reducing the acetate concentration. In this paper, we study a chemostat model of such a synthetic community where both strains are allowed to produce recombinant proteins. We give necessary and sufficient conditions for the existence of a coexistence equilibrium and show that it is unique. Based on this equilibrium, we define a multi-objective optimization problem for the maximization of two important bioprocess performance metrics, process yield and productivity. Solving numerically this problem, we find the best available trade-offs between the metrics. Under optimal operation of the mixed community, both strains must produce the protein of interest, and not only one (distribution instead of division of labor). Moreover, in this regime acetate secretion by one strain is necessary for the survival of the other (syntrophy). The results thus illustrate how complex multi-level dynamics shape the optimal production of recombinant proteins by synthetic microbial consortia.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.476048', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Cell adhesion and immune response, two main functions altered in the transcriptome of seasonally regressed testes of two mammalian species', 'authors': \"[{'name': 'Francisca M. Real'}, {'name': 'Miguel Lao-Pérez'}, {'name': 'Miguel Burgos'}, {'name': 'Stefan Mundlos'}, {'name': 'Darío G. Lupiáñez'}, {'name': 'Rafael Jiménez'}, {'name': 'Francisco J. Barrionuevo'}]\", 'abstract': '<p>In species with seasonal breeding, male specimens undergo substantial testicular regression during the non-breeding period of the year. However, the molecular mechanisms that control this biological process are largely unknown. Here, we report a transcriptomic analysis on the Iberian mole, <italic>Talpa occidentalis</italic>, in which the desquamation of live, non-apoptotic germ cells is the major cellular event responsible for testis regression. By comparing testes at different reproductive states (active, regressing and inactive), we demonstrate that the molecular pathways controlling the cell adhesion function in the seminiferous epithelium, such as the MAPK, ERK and TGF-β signalling, are altered during the regression process. In addition, inactive testes display a global upregulation of genes associated with immune response, indicating a selective loss of the “immune privilege” that normally operates in sexually active testes. Interspecies comparative analyses using analogous data from the Mediterranean pine vole, a rodent species where testis regression is controlled by halting meiosis entry, revealed a common gene expression signature in the regressed testes of these two evolutionary distant species. Our study advances in the knowledge of the molecular mechanisms associated to gonadal seasonal breeding, highlighting the existence of a conserved transcriptional program of testis involution across mammalian clades.</p><sec><title>Research Highlights</title><p>By comparing the trascriptomes of the testes from males of the iberian mole, <italic>Talpa occidentalis</italic> (order Eulipotyphla), captured at different stages of the seasonal breeding cycle of this species, we show that two main functions are altered during seasonal testis regression: cell adhesion and immune response. The fact that the same functions alre also altered in the Mediterranean pine vole, <italic>Microtus duodecimcostatus</italic> (order Rodentia), evidences the existence of a conserved transcriptional program of testis regression across mammalian clades.</p></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475908', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Machine-learning from Pseudomonas putida Transcriptomes Reveals Its Transcriptional Regulatory Network', 'authors': \"[{'name': 'Hyun Gyu Lim'}, {'name': 'Kevin Rychel'}, {'name': 'Anand V. Sastry'}, {'name': 'Joshua Mueller'}, {'name': 'Wei Niu'}, {'name': 'Adam M. Feist'}, {'name': 'Bernhard O. Palsson'}]\", 'abstract': '<p>Bacterial gene expression is orchestrated by numerous transcription factors (TFs). Elucidating how gene expression is regulated is fundamental to understanding bacterial physiology and engineering it for practical use. In this study, a machine-learning approach was applied to uncover the genome-scale transcriptional regulatory network (TRN) in <italic>Pseudomonas putida</italic>, an important organism for bioproduction. We performed independent component analysis of a compendium of 321 high-quality gene expression profiles, which were previously published or newly generated in this study. We identified 84 groups of independently modulated genes (iModulons) that explain 75.7% of the total variance in the compendium. With these iModulons, we (i) expand our understanding of the regulatory functions of 39 iModulon associated TFs (e.g., HexR, Zur) by systematic comparison with 1,993 previously reported TF-gene interactions; (ii) outline transcriptional changes after the transition from the exponential growth to stationary phases; (iii) capture group of genes required for utilizing diverse carbon sources and increased stationary response with slower growth rates; (iv) unveil multiple evolutionary strategies of transcriptome reallocation to achieve fast growth rates; and (v) define an osmotic stimulon, which includes the Type VI secretion system, as coordination of multiple iModulon activity changes. Taken together, this study provides the first quantitative genome-scale TRN for <italic>P. putida</italic> and a basis for a comprehensive understanding of its complex transcriptome changes in a variety of physiological states.</p>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.11.475900', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Reciprocal epigenetic Sox2 regulation by SMAD1-SMAD3 is critical for anoikis resistance and metastasis in cancer', 'authors': \"[{'name': 'Zainab Shonibare'}, {'name': 'Mehri Monavarian'}, {'name': 'Kathleen O’ Connell'}, {'name': 'Diego Altomare'}, {'name': 'Abigail Shelton'}, {'name': 'Shubham Mehta'}, {'name': 'Renata Jaskula-Sztul'}, {'name': 'Rebecca Phaeton'}, {'name': 'Mark D. Starr'}, {'name': 'Regina Whitaker'}, {'name': 'Andrew Berchuck'}, {'name': 'Andrew B Nixon'}, {'name': 'Rebecca Arend'}, {'name': 'Nam Y Lee'}, {'name': 'C. Ryan Miller'}, {'name': 'Nadine Hempel'}, {'name': 'Karthikeyan Mythreye'}]\", 'abstract': '<p>Growth factors in the tumor environment are key regulators of cell survival and metastasis. Here we reveal, dichotomy between TGF-β superfamily growth factors BMP and TGF-β/activin, and their downstream SMAD effectors. Gene expression profiling uncovered Sox2 as a key signaling node regulated in an opposing manner by anoikis-promoting BMP2, 4 and 9, and anoikis-suppressing TGF-β and activin A. We find that Sox2 repression by BMPs robustly inhibits intraperitoneal tumor burden and increases survival in multiple ovarian cancer models. Repression of Sox2 is driven by SMAD1 dependent histone H3K27me3 recruitment and DNA methylation at <italic>SOX2’s</italic> promoter. Conversely, TGF-β and activin A promote Sox2 expression, and anoikis resistance by SMAD3 mediated histone H3K4me3 recruitment. We find that balancing Sox2 levels is critical for anoikis, as transcriptomics reveals regulation of key cell death pathways. Moreover, BMP-driven SMAD1 signaling can override TGF-β and activin’s effect on Sox2, which has clinical significance due to the high levels of TGF-β we find in ovarian cancer patients. Together, our findings identify Sox2 as a contextual and contrastingly regulated key node, downstream of TGF-β superfamily members controlling anoikis and metastasis in ovarian cancers.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Sox2 is a key node for anoikis resistance in cancer</p></list-item><list-item><p>Sox2 is differentially regulated by TGF-β/activin and BMPs in broad cancers</p></list-item><list-item><p>BMP9 is a robust metastasis suppressor by lowering Sox2</p></list-item><list-item><p>Sox2 regulation is contextual, epigenetic and at the transcriptional level</p></list-item></list></sec>', 'posted': '2022-01-12'}, {'doi': '10.1101/2022.01.12.476135', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Optineurin mediates delivery of membrane from transferrin receptor-positive endosomes to autophagosomes', 'authors': \"[{'name': 'Megha Bansal'}, {'name': 'Kapil Sirohi'}, {'name': 'Shivranjani C Moharir'}, {'name': 'Ghanshyam Swarup'}]\", 'abstract': '<p>Autophagy is a conserved quality control mechanism that removes damaged proteins, organelles and invading bacteria through lysosome-mediated degradation. During autophagy several organelles including endoplasmic reticulum, mitochondria, plasma membrane and endosomes contribute membrane for autophagosome formation. However, the mechanisms and proteins involved in membrane delivery to autophagosomes are not clear. Optineurin (OPTN), a cytoplasmic adaptor protein, is involved in promoting maturation of phagophores into autophagosomes; it is also involved in regulating endocytic trafficking and recycling of transferrin receptor (TFRC). Here, we have examined the role of optineurin in the delivery of membrane from TFRC-positive endosomes to autophagosomes. Only a small fraction of autophagosomes was positive for TFRC, indicating that TFRC-positive endosomes could contribute membrane to a subset of autophagosomes. The percentage of TFRC-positive autophagosomes was reduced in Optineurin knockout mouse embryonic fibroblasts (<italic>Optn<sup>−/−</sup></italic> MEFs) in comparison with normal MEFs. Upon over-expression of optineurin, the percentage of TFRC-positive autophagosomes was increased in <italic>Optn<sup>−/−</sup></italic> MEFs. Unlike wild-type optineurin, a disease-associated mutant, E478G, defective in ubiquitin binding, was not able to enhance formation of TFRC-positive autophagosomes in <italic>Optn<sup>−/−</sup></italic> MEFs. TFRC degradation mediated by autophagy was decreased in optineurin deficient cells. Our results suggest that optineurin mediates delivery of TFRC and perhaps associated membrane from TFRC-positive endosomes to autophagosomes, and this may contribute to autophagosome formation.</p>', 'posted': '2022-01-13'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2022.01.10.22269010</td>\n",
       "      <td>2384</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Infectious viral load in unvaccinated and vacc...</td>\n",
       "      <td>[{'name': 'Olha Puhach'}, {'name': 'Kenneth Ad...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Viral load (V...</td>\n",
       "      <td>2022-01-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2022.01.12.476031</td>\n",
       "      <td>368</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>The SARS-CoV-2 Omicron (B.1.1.529) variant exh...</td>\n",
       "      <td>[{'name': 'Shuofeng Yuan'}, {'name': 'Zi-Wei Y...</td>\n",
       "      <td>&lt;p&gt;The newly emerging SARS-CoV-2 Omicron (B.1....</td>\n",
       "      <td>2022-01-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2022.01.14.476225</td>\n",
       "      <td>104</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Single-cell multi-omics of human clonal hemato...</td>\n",
       "      <td>[{'name': 'Anna S. Nam'}, {'name': 'Neville Du...</td>\n",
       "      <td>&lt;p&gt;Somatic mutations in cancer genes have been...</td>\n",
       "      <td>2022-01-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2022.01.11.475838</td>\n",
       "      <td>93</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Lightweight compositional analysis of metageno...</td>\n",
       "      <td>[{'name': 'Luiz Carlos Irber'}, {'name': 'Phil...</td>\n",
       "      <td>&lt;p&gt;The identification of reference genomes and...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2022.01.11.475793</td>\n",
       "      <td>88</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Global monitoring of soil animal communities u...</td>\n",
       "      <td>[{'name': 'Anton M. Potapov'}, {'name': 'Xin S...</td>\n",
       "      <td>&lt;p&gt;Here we introduce the Soil BON Foodweb Team...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2022.01.13.476251</td>\n",
       "      <td>84</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Mixing genome annotation methods in a comparat...</td>\n",
       "      <td>[{'name': 'Caroline M. Weisman'}, {'name': 'An...</td>\n",
       "      <td>&lt;p&gt;Comparisons of genomes of different species...</td>\n",
       "      <td>2022-01-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2022.01.11.475254</td>\n",
       "      <td>76</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Identifying and correcting repeat-calling erro...</td>\n",
       "      <td>[{'name': 'Kar-Tong Tan'}, {'name': 'Michael K...</td>\n",
       "      <td>&lt;p&gt;Nanopore long-read genome sequencing is eme...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2022.01.10.475719</td>\n",
       "      <td>74</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Learning in reverse: Dopamine errors drive exc...</td>\n",
       "      <td>[{'name': 'Benjamin M. Seitz'}, {'name': 'Ivy ...</td>\n",
       "      <td>&lt;p&gt;For over two decades, midbrain dopamine was...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2022.01.11.475728</td>\n",
       "      <td>73</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Predicting patient treatment response and resi...</td>\n",
       "      <td>[{'name': 'Sanju Sinha'}, {'name': 'Rahulsimha...</td>\n",
       "      <td>&lt;p&gt;Tailoring the best treatments to cancer pat...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2022.01.13.476171</td>\n",
       "      <td>65</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>A high-resolution comparative atlas across 74 ...</td>\n",
       "      <td>[{'name': 'Elise Parey'}, {'name': 'Alexandra ...</td>\n",
       "      <td>&lt;p&gt;Teleost fish are one of the most species-ri...</td>\n",
       "      <td>2022-01-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2022.01.11.475815</td>\n",
       "      <td>65</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Panoramic visual statistics shape retina-wide ...</td>\n",
       "      <td>[{'name': 'Divyansh Gupta'}, {'name': 'Wiktor ...</td>\n",
       "      <td>&lt;p&gt;Visual systems have adapted to the structur...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2022.01.13.476155</td>\n",
       "      <td>64</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Transatlantic spread of highly pathogenic avia...</td>\n",
       "      <td>[{'name': 'V. Caliendo'}, {'name': 'N. S. Lewi...</td>\n",
       "      <td>&lt;p&gt;Highly pathogenic avian influenza (HPAI) vi...</td>\n",
       "      <td>2022-01-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2022.01.10.475641</td>\n",
       "      <td>63</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The genetic structure of a wild wheat populati...</td>\n",
       "      <td>[{'name': 'Tal Dahan-Meir'}, {'name': 'Thomas ...</td>\n",
       "      <td>&lt;p&gt;Wild progenitors of major crops can help us...</td>\n",
       "      <td>2022-01-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2022.01.11.475860</td>\n",
       "      <td>57</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Two modes of evolution shape bacterial strain ...</td>\n",
       "      <td>[{'name': 'N. Frazão'}, {'name': 'A. Konrad'},...</td>\n",
       "      <td>&lt;p&gt;How and at what pace bacteria evolve when c...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2022.01.14.476419</td>\n",
       "      <td>56</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Population genomic evidence of selection on st...</td>\n",
       "      <td>[{'name': 'Linyi Zhang'}, {'name': 'Samridhi C...</td>\n",
       "      <td>&lt;p&gt;Structural variants (SVs) can promote speci...</td>\n",
       "      <td>2022-01-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2022.01.15.476471</td>\n",
       "      <td>55</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The cytoplasmic incompatibility Cif proteins f...</td>\n",
       "      <td>[{'name': 'Rupinder Kaur'}, {'name': 'Brittany...</td>\n",
       "      <td>&lt;p&gt;Inherited microorganisms can selfishly mani...</td>\n",
       "      <td>2022-01-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2022.01.14.476329</td>\n",
       "      <td>53</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Meiosis in the human pathogen Aspergillus fumi...</td>\n",
       "      <td>[{'name': 'Ben Auxier'}, {'name': 'Frank Becke...</td>\n",
       "      <td>&lt;p&gt;Evidence from both population genetics and ...</td>\n",
       "      <td>2022-01-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2022.01.13.476228</td>\n",
       "      <td>52</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Critical assessment of pan-genomics of metagen...</td>\n",
       "      <td>[{'name': 'Tang Li'}, {'name': 'Yanbin Yin'}]</td>\n",
       "      <td>&lt;p&gt;Background: Large scale metagenome assembly...</td>\n",
       "      <td>2022-01-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2022.01.14.22269288</td>\n",
       "      <td>46</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Characterizing features of outbreak duration f...</td>\n",
       "      <td>[{'name': 'Alex D. Washburne'}, {'name': 'Nath...</td>\n",
       "      <td>&lt;p&gt;Characterizing the dynamics of epidemic tra...</td>\n",
       "      <td>2022-01-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2022.01.14.21267633</td>\n",
       "      <td>44</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>National-scale surveillance of emerging SARS-C...</td>\n",
       "      <td>[{'name': 'Fabian Amman'}, {'name': 'Rudolf Ma...</td>\n",
       "      <td>&lt;p&gt;SARS-CoV-2 surveillance is crucial to ident...</td>\n",
       "      <td>2022-01-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2022.01.13.476191</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>cGAS recruitment to micronuclei is dictated by...</td>\n",
       "      <td>[{'name': 'Kate M. MacDonald'}, {'name': 'Shir...</td>\n",
       "      <td>&lt;p&gt;Micronuclei (MN) are aberrant cytosolic com...</td>\n",
       "      <td>2022-01-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2022.01.14.475546</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Identification and Age-dependent Increase of P...</td>\n",
       "      <td>[{'name': 'Merve Aksöz'}, {'name': 'Grigore-Ar...</td>\n",
       "      <td>&lt;p&gt;Hematopoietic stem cells (HSC) reconstitute...</td>\n",
       "      <td>2022-01-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2022.01.11.475922</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>True</td>\n",
       "      <td>Structural and functional impact by SARS-CoV-2...</td>\n",
       "      <td>[{'name': 'Jun Zhang'}, {'name': 'Yongfei Cai'...</td>\n",
       "      <td>&lt;p&gt;The Omicron variant of severe acute respira...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2022.01.11.475327</td>\n",
       "      <td>38</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Systemic infection of SARS-CoV-2 in free rangi...</td>\n",
       "      <td>[{'name': 'Sonalika Mahajan'}, {'name': 'Mathe...</td>\n",
       "      <td>&lt;p&gt;We report patho-morphological and virologic...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2022.01.10.475735</td>\n",
       "      <td>38</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>GoPeaks: Histone Modification Peak Calling for...</td>\n",
       "      <td>[{'name': 'William M Yashar'}, {'name': 'Garth...</td>\n",
       "      <td>&lt;p&gt;Genome-wide mapping of the histone modifica...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2022.01.11.22268989</td>\n",
       "      <td>38</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Psychiatry and Clinical Psychology</td>\n",
       "      <td>False</td>\n",
       "      <td>Network Constraints on Longitudinal Grey Matte...</td>\n",
       "      <td>[{'name': 'Sidhant Chopra'}, {'name': 'Stuart ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Different reg...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2022.01.10.475690</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Atlasing white matter and grey matter joint co...</td>\n",
       "      <td>[{'name': 'Victor Nozais'}, {'name': 'Stephani...</td>\n",
       "      <td>&lt;p&gt;Over the past two decades, the study of res...</td>\n",
       "      <td>2022-01-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2022.01.11.475870</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Debiasing FracMinHash and deriving confidence ...</td>\n",
       "      <td>[{'name': 'Mahmudur Rahman Hera'}, {'name': 'N...</td>\n",
       "      <td>&lt;p&gt;Sketching methods offer computational biolo...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2022.01.11.22269017</td>\n",
       "      <td>36</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Healthcare workers’ views on mandatory SARS-Co...</td>\n",
       "      <td>[{'name': 'Katherine Woolf'}, {'name': 'Mayuri...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Several count...</td>\n",
       "      <td>2022-01-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2022.01.14.475871</td>\n",
       "      <td>36</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>An ancient truncated duplication of the anti-M...</td>\n",
       "      <td>[{'name': 'Ming Wen'}, {'name': 'Qiaowei Pan'}...</td>\n",
       "      <td>&lt;p&gt;The evolution of sex determination (SD) mec...</td>\n",
       "      <td>2022-01-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2022.01.12.476080</td>\n",
       "      <td>33</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A split ribozyme that links detection of a nat...</td>\n",
       "      <td>[{'name': 'Lauren Gambill'}, {'name': 'August ...</td>\n",
       "      <td>&lt;p&gt;Individual RNA remains a challenging signal...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2022.01.11.475790</td>\n",
       "      <td>32</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>The phylogenomic landscape of the genus Serratia</td>\n",
       "      <td>[{'name': 'David J. Williams'}, {'name': 'Patr...</td>\n",
       "      <td>&lt;p&gt;The genus &lt;italic&gt;Serratia&lt;/italic&gt; has bee...</td>\n",
       "      <td>2022-01-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2022.01.10.474936</td>\n",
       "      <td>32</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>The megabase-scale crossover landscape is inde...</td>\n",
       "      <td>[{'name': 'Qichao Lian'}, {'name': 'Victor Sol...</td>\n",
       "      <td>&lt;p&gt;Meiotic recombination frequency varies alon...</td>\n",
       "      <td>2022-01-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2022.01.11.475536</td>\n",
       "      <td>32</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>In vivo imaging of calcium dynamics in zebrafi...</td>\n",
       "      <td>[{'name': 'Macarena Pozo-Morales'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;Hepatocytes were the first cell-type for wh...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2022.01.12.476002</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>In vivo single-cell profiling of lncRNAs durin...</td>\n",
       "      <td>[{'name': 'Luisa Santus'}, {'name': 'Raquel Ga...</td>\n",
       "      <td>&lt;p&gt;Long non-coding RNAs (lncRNAs) are pivotal ...</td>\n",
       "      <td>2022-01-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2022.01.13.476254</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Caenorhabditis nematodes colonize ephemeral re...</td>\n",
       "      <td>[{'name': 'Solomon A Sloat'}, {'name': 'Luke M...</td>\n",
       "      <td>&lt;p&gt;Factors shaping the distribution and abunda...</td>\n",
       "      <td>2022-01-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2022.01.11.475947</td>\n",
       "      <td>28</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 drives NLRP3 inflammasome activatio...</td>\n",
       "      <td>[{'name': 'Eduardo Albornoz'}, {'name': 'Alber...</td>\n",
       "      <td>&lt;p&gt;Coronavirus disease-2019 (COVID-19) is prim...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2022.01.12.475930</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Cross-linking of the Endolysosomal System Reve...</td>\n",
       "      <td>[{'name': 'Jasjot Singh'}, {'name': 'Hadeer El...</td>\n",
       "      <td>&lt;p&gt;Lysosomes are well-established as the main ...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2022.01.13.476279</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Benchmarking software to predict antibiotic re...</td>\n",
       "      <td>[{'name': 'Emily F Wissel'}, {'name': 'Brooke ...</td>\n",
       "      <td>&lt;p&gt;The use of shotgun metagenomics for AMR det...</td>\n",
       "      <td>2022-01-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2022.01.10.22269008</td>\n",
       "      <td>26</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Hydroxychloroquine/Chloroquine for the Treatme...</td>\n",
       "      <td>[{'name': 'Leon Di Stefano'}, {'name': 'Elizab...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Importance&lt;/title&gt;&lt;p&gt;Results from ...</td>\n",
       "      <td>2022-01-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2022.01.11.475830</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Loss of plastid developmental genes coincides ...</td>\n",
       "      <td>[{'name': 'Alexander I. MacLeod'}, {'name': 'P...</td>\n",
       "      <td>&lt;p&gt;The first plastid evolved from an endosymbi...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2022.01.12.476034</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>CCPLS reveals cell-type-specific spatial depen...</td>\n",
       "      <td>[{'name': 'Takaho Tsuchiya'}, {'name': 'Hiroki...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Motivation&lt;/title&gt;&lt;p&gt;Cell-cell com...</td>\n",
       "      <td>2022-01-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2022.01.11.475721</td>\n",
       "      <td>25</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Sex differences in pain-related behaviors and ...</td>\n",
       "      <td>[{'name': 'Adela M. Francis-Malave'}, {'name':...</td>\n",
       "      <td>&lt;p&gt;Previous studies have reported sex differen...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2022.01.11.475842</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>A new class of protein sensor links spirochete...</td>\n",
       "      <td>[{'name': 'A.R. Muok'}, {'name': 'K. Kurniyati...</td>\n",
       "      <td>&lt;p&gt;Pathogenic spirochetes can alter their morp...</td>\n",
       "      <td>2022-01-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2022.01.12.475944</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Epigenetic dysregulation from chromosomal tran...</td>\n",
       "      <td>[{'name': 'Albert Agustinus'}, {'name': 'Ramya...</td>\n",
       "      <td>&lt;p&gt;Chromosomal instability (CIN) and epigeneti...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2022.01.14.476226</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>The minimizer Jaccard estimator is biased and ...</td>\n",
       "      <td>[{'name': 'Mahdi Belbasi'}, {'name': 'Antonio ...</td>\n",
       "      <td>&lt;p&gt;Sketching is now widely used in bioinformat...</td>\n",
       "      <td>2022-01-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2022.01.11.22269097</td>\n",
       "      <td>21</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>High-cited favorable studies for COVID-19 trea...</td>\n",
       "      <td>[{'name': 'John P.A. Ioannidis'}]</td>\n",
       "      <td>&lt;p&gt;Importance. COVID-19 has resulted in massiv...</td>\n",
       "      <td>2022-01-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2022.01.12.475507</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Glucocorticoid Receptor-Regulated Enhancers Pl...</td>\n",
       "      <td>[{'name': 'Sascha H. Duttke'}, {'name': 'Patri...</td>\n",
       "      <td>&lt;p&gt;Substance abuse and addiction represent a s...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2022.01.11.475847</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Physiology</td>\n",
       "      <td>False</td>\n",
       "      <td>High-fructose feeding suppresses cold-stimulat...</td>\n",
       "      <td>[{'name': 'Gabriel Richard'}, {'name': 'Denis ...</td>\n",
       "      <td>&lt;p&gt;Diets rich in added sugars, especially high...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2022.01.11.475889</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pharmacology and Toxicology</td>\n",
       "      <td>True</td>\n",
       "      <td>Favipiravir, umifenovir and camostat mesylate:...</td>\n",
       "      <td>[{'name': 'Mehnmet Altay Unal'}, {'name': 'Omu...</td>\n",
       "      <td>&lt;p&gt;Since the first cases the coronavirus disea...</td>\n",
       "      <td>2022-01-12</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2022.01.10.22269010    2384  medRxiv   \n",
       "1     10.1101/2022.01.12.476031     368  bioRxiv   \n",
       "2     10.1101/2022.01.14.476225     104  bioRxiv   \n",
       "3     10.1101/2022.01.11.475838      93  bioRxiv   \n",
       "4     10.1101/2022.01.11.475793      88  bioRxiv   \n",
       "5     10.1101/2022.01.13.476251      84  bioRxiv   \n",
       "6     10.1101/2022.01.11.475254      76  bioRxiv   \n",
       "7     10.1101/2022.01.10.475719      74  bioRxiv   \n",
       "8     10.1101/2022.01.11.475728      73  bioRxiv   \n",
       "9     10.1101/2022.01.13.476171      65  bioRxiv   \n",
       "10    10.1101/2022.01.11.475815      65  bioRxiv   \n",
       "11    10.1101/2022.01.13.476155      64  bioRxiv   \n",
       "12    10.1101/2022.01.10.475641      63  bioRxiv   \n",
       "13    10.1101/2022.01.11.475860      57  bioRxiv   \n",
       "14    10.1101/2022.01.14.476419      56  bioRxiv   \n",
       "15    10.1101/2022.01.15.476471      55  bioRxiv   \n",
       "16    10.1101/2022.01.14.476329      53  bioRxiv   \n",
       "17    10.1101/2022.01.13.476228      52  bioRxiv   \n",
       "18  10.1101/2022.01.14.22269288      46  medRxiv   \n",
       "19  10.1101/2022.01.14.21267633      44  medRxiv   \n",
       "20    10.1101/2022.01.13.476191      41  bioRxiv   \n",
       "21    10.1101/2022.01.14.475546      41  bioRxiv   \n",
       "22    10.1101/2022.01.11.475922      41  bioRxiv   \n",
       "23    10.1101/2022.01.11.475327      38  bioRxiv   \n",
       "24    10.1101/2022.01.10.475735      38  bioRxiv   \n",
       "25  10.1101/2022.01.11.22268989      38  medRxiv   \n",
       "26    10.1101/2022.01.10.475690      37  bioRxiv   \n",
       "27    10.1101/2022.01.11.475870      37  bioRxiv   \n",
       "28  10.1101/2022.01.11.22269017      36  medRxiv   \n",
       "29    10.1101/2022.01.14.475871      36  bioRxiv   \n",
       "30    10.1101/2022.01.12.476080      33  bioRxiv   \n",
       "31    10.1101/2022.01.11.475790      32  bioRxiv   \n",
       "32    10.1101/2022.01.10.474936      32  bioRxiv   \n",
       "33    10.1101/2022.01.11.475536      32  bioRxiv   \n",
       "34    10.1101/2022.01.12.476002      29  bioRxiv   \n",
       "35    10.1101/2022.01.13.476254      29  bioRxiv   \n",
       "36    10.1101/2022.01.11.475947      28  bioRxiv   \n",
       "37    10.1101/2022.01.12.475930      27  bioRxiv   \n",
       "38    10.1101/2022.01.13.476279      26  bioRxiv   \n",
       "39  10.1101/2022.01.10.22269008      26  medRxiv   \n",
       "40    10.1101/2022.01.11.475830      26  bioRxiv   \n",
       "41    10.1101/2022.01.12.476034      26  bioRxiv   \n",
       "42    10.1101/2022.01.11.475721      25  bioRxiv   \n",
       "43    10.1101/2022.01.11.475842      24  bioRxiv   \n",
       "44    10.1101/2022.01.12.475944      23  bioRxiv   \n",
       "45    10.1101/2022.01.14.476226      22  bioRxiv   \n",
       "46  10.1101/2022.01.11.22269097      21  medRxiv   \n",
       "47    10.1101/2022.01.12.475507      21  bioRxiv   \n",
       "48    10.1101/2022.01.11.475847      21  bioRxiv   \n",
       "49    10.1101/2022.01.11.475889      20  bioRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0   Infectious Diseases (except HIV/AIDS)   True   \n",
       "1                            Microbiology   True   \n",
       "2                          Cancer Biology  False   \n",
       "3                          Bioinformatics  False   \n",
       "4                                 Ecology  False   \n",
       "5                                Genomics  False   \n",
       "6                          Bioinformatics  False   \n",
       "7                            Neuroscience  False   \n",
       "8                          Bioinformatics  False   \n",
       "9                                Genomics  False   \n",
       "10                           Neuroscience  False   \n",
       "11                           Microbiology  False   \n",
       "12                   Evolutionary Biology  False   \n",
       "13                   Evolutionary Biology  False   \n",
       "14                               Genomics  False   \n",
       "15                           Cell Biology  False   \n",
       "16                               Genetics  False   \n",
       "17                         Bioinformatics  False   \n",
       "18                           Epidemiology   True   \n",
       "19                           Epidemiology   True   \n",
       "20                           Cell Biology  False   \n",
       "21                           Cell Biology  False   \n",
       "22                           Biochemistry   True   \n",
       "23                           Microbiology   True   \n",
       "24                         Bioinformatics  False   \n",
       "25     Psychiatry and Clinical Psychology  False   \n",
       "26                           Neuroscience  False   \n",
       "27                         Bioinformatics  False   \n",
       "28  Infectious Diseases (except HIV/AIDS)   True   \n",
       "29                               Genomics  False   \n",
       "30                      Synthetic Biology  False   \n",
       "31                           Microbiology  False   \n",
       "32                               Genetics  False   \n",
       "33                  Developmental Biology  False   \n",
       "34                               Genomics  False   \n",
       "35                                Ecology  False   \n",
       "36                           Neuroscience   True   \n",
       "37                           Biochemistry  False   \n",
       "38                         Bioinformatics  False   \n",
       "39  Infectious Diseases (except HIV/AIDS)   True   \n",
       "40                   Evolutionary Biology  False   \n",
       "41                         Bioinformatics  False   \n",
       "42                           Neuroscience  False   \n",
       "43                           Microbiology  False   \n",
       "44                         Cancer Biology  False   \n",
       "45                         Bioinformatics  False   \n",
       "46                           Epidemiology   True   \n",
       "47                           Neuroscience  False   \n",
       "48                             Physiology  False   \n",
       "49            Pharmacology and Toxicology   True   \n",
       "\n",
       "                                                title  \\\n",
       "0   Infectious viral load in unvaccinated and vacc...   \n",
       "1   The SARS-CoV-2 Omicron (B.1.1.529) variant exh...   \n",
       "2   Single-cell multi-omics of human clonal hemato...   \n",
       "3   Lightweight compositional analysis of metageno...   \n",
       "4   Global monitoring of soil animal communities u...   \n",
       "5   Mixing genome annotation methods in a comparat...   \n",
       "6   Identifying and correcting repeat-calling erro...   \n",
       "7   Learning in reverse: Dopamine errors drive exc...   \n",
       "8   Predicting patient treatment response and resi...   \n",
       "9   A high-resolution comparative atlas across 74 ...   \n",
       "10  Panoramic visual statistics shape retina-wide ...   \n",
       "11  Transatlantic spread of highly pathogenic avia...   \n",
       "12  The genetic structure of a wild wheat populati...   \n",
       "13  Two modes of evolution shape bacterial strain ...   \n",
       "14  Population genomic evidence of selection on st...   \n",
       "15  The cytoplasmic incompatibility Cif proteins f...   \n",
       "16  Meiosis in the human pathogen Aspergillus fumi...   \n",
       "17  Critical assessment of pan-genomics of metagen...   \n",
       "18  Characterizing features of outbreak duration f...   \n",
       "19  National-scale surveillance of emerging SARS-C...   \n",
       "20  cGAS recruitment to micronuclei is dictated by...   \n",
       "21  Identification and Age-dependent Increase of P...   \n",
       "22  Structural and functional impact by SARS-CoV-2...   \n",
       "23  Systemic infection of SARS-CoV-2 in free rangi...   \n",
       "24  GoPeaks: Histone Modification Peak Calling for...   \n",
       "25  Network Constraints on Longitudinal Grey Matte...   \n",
       "26  Atlasing white matter and grey matter joint co...   \n",
       "27  Debiasing FracMinHash and deriving confidence ...   \n",
       "28  Healthcare workers’ views on mandatory SARS-Co...   \n",
       "29  An ancient truncated duplication of the anti-M...   \n",
       "30  A split ribozyme that links detection of a nat...   \n",
       "31   The phylogenomic landscape of the genus Serratia   \n",
       "32  The megabase-scale crossover landscape is inde...   \n",
       "33  In vivo imaging of calcium dynamics in zebrafi...   \n",
       "34  In vivo single-cell profiling of lncRNAs durin...   \n",
       "35  Caenorhabditis nematodes colonize ephemeral re...   \n",
       "36  SARS-CoV-2 drives NLRP3 inflammasome activatio...   \n",
       "37  Cross-linking of the Endolysosomal System Reve...   \n",
       "38  Benchmarking software to predict antibiotic re...   \n",
       "39  Hydroxychloroquine/Chloroquine for the Treatme...   \n",
       "40  Loss of plastid developmental genes coincides ...   \n",
       "41  CCPLS reveals cell-type-specific spatial depen...   \n",
       "42  Sex differences in pain-related behaviors and ...   \n",
       "43  A new class of protein sensor links spirochete...   \n",
       "44  Epigenetic dysregulation from chromosomal tran...   \n",
       "45  The minimizer Jaccard estimator is biased and ...   \n",
       "46  High-cited favorable studies for COVID-19 trea...   \n",
       "47  Glucocorticoid Receptor-Regulated Enhancers Pl...   \n",
       "48  High-fructose feeding suppresses cold-stimulat...   \n",
       "49  Favipiravir, umifenovir and camostat mesylate:...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Olha Puhach'}, {'name': 'Kenneth Ad...   \n",
       "1   [{'name': 'Shuofeng Yuan'}, {'name': 'Zi-Wei Y...   \n",
       "2   [{'name': 'Anna S. Nam'}, {'name': 'Neville Du...   \n",
       "3   [{'name': 'Luiz Carlos Irber'}, {'name': 'Phil...   \n",
       "4   [{'name': 'Anton M. Potapov'}, {'name': 'Xin S...   \n",
       "5   [{'name': 'Caroline M. Weisman'}, {'name': 'An...   \n",
       "6   [{'name': 'Kar-Tong Tan'}, {'name': 'Michael K...   \n",
       "7   [{'name': 'Benjamin M. Seitz'}, {'name': 'Ivy ...   \n",
       "8   [{'name': 'Sanju Sinha'}, {'name': 'Rahulsimha...   \n",
       "9   [{'name': 'Elise Parey'}, {'name': 'Alexandra ...   \n",
       "10  [{'name': 'Divyansh Gupta'}, {'name': 'Wiktor ...   \n",
       "11  [{'name': 'V. Caliendo'}, {'name': 'N. S. Lewi...   \n",
       "12  [{'name': 'Tal Dahan-Meir'}, {'name': 'Thomas ...   \n",
       "13  [{'name': 'N. Frazão'}, {'name': 'A. Konrad'},...   \n",
       "14  [{'name': 'Linyi Zhang'}, {'name': 'Samridhi C...   \n",
       "15  [{'name': 'Rupinder Kaur'}, {'name': 'Brittany...   \n",
       "16  [{'name': 'Ben Auxier'}, {'name': 'Frank Becke...   \n",
       "17      [{'name': 'Tang Li'}, {'name': 'Yanbin Yin'}]   \n",
       "18  [{'name': 'Alex D. Washburne'}, {'name': 'Nath...   \n",
       "19  [{'name': 'Fabian Amman'}, {'name': 'Rudolf Ma...   \n",
       "20  [{'name': 'Kate M. MacDonald'}, {'name': 'Shir...   \n",
       "21  [{'name': 'Merve Aksöz'}, {'name': 'Grigore-Ar...   \n",
       "22  [{'name': 'Jun Zhang'}, {'name': 'Yongfei Cai'...   \n",
       "23  [{'name': 'Sonalika Mahajan'}, {'name': 'Mathe...   \n",
       "24  [{'name': 'William M Yashar'}, {'name': 'Garth...   \n",
       "25  [{'name': 'Sidhant Chopra'}, {'name': 'Stuart ...   \n",
       "26  [{'name': 'Victor Nozais'}, {'name': 'Stephani...   \n",
       "27  [{'name': 'Mahmudur Rahman Hera'}, {'name': 'N...   \n",
       "28  [{'name': 'Katherine Woolf'}, {'name': 'Mayuri...   \n",
       "29  [{'name': 'Ming Wen'}, {'name': 'Qiaowei Pan'}...   \n",
       "30  [{'name': 'Lauren Gambill'}, {'name': 'August ...   \n",
       "31  [{'name': 'David J. Williams'}, {'name': 'Patr...   \n",
       "32  [{'name': 'Qichao Lian'}, {'name': 'Victor Sol...   \n",
       "33  [{'name': 'Macarena Pozo-Morales'}, {'name': '...   \n",
       "34  [{'name': 'Luisa Santus'}, {'name': 'Raquel Ga...   \n",
       "35  [{'name': 'Solomon A Sloat'}, {'name': 'Luke M...   \n",
       "36  [{'name': 'Eduardo Albornoz'}, {'name': 'Alber...   \n",
       "37  [{'name': 'Jasjot Singh'}, {'name': 'Hadeer El...   \n",
       "38  [{'name': 'Emily F Wissel'}, {'name': 'Brooke ...   \n",
       "39  [{'name': 'Leon Di Stefano'}, {'name': 'Elizab...   \n",
       "40  [{'name': 'Alexander I. MacLeod'}, {'name': 'P...   \n",
       "41  [{'name': 'Takaho Tsuchiya'}, {'name': 'Hiroki...   \n",
       "42  [{'name': 'Adela M. Francis-Malave'}, {'name':...   \n",
       "43  [{'name': 'A.R. Muok'}, {'name': 'K. Kurniyati...   \n",
       "44  [{'name': 'Albert Agustinus'}, {'name': 'Ramya...   \n",
       "45  [{'name': 'Mahdi Belbasi'}, {'name': 'Antonio ...   \n",
       "46                  [{'name': 'John P.A. Ioannidis'}]   \n",
       "47  [{'name': 'Sascha H. Duttke'}, {'name': 'Patri...   \n",
       "48  [{'name': 'Gabriel Richard'}, {'name': 'Denis ...   \n",
       "49  [{'name': 'Mehnmet Altay Unal'}, {'name': 'Omu...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <sec><title>Background</title><p>Viral load (V...  2022-01-11  \n",
       "1   <p>The newly emerging SARS-CoV-2 Omicron (B.1....  2022-01-13  \n",
       "2   <p>Somatic mutations in cancer genes have been...  2022-01-16  \n",
       "3   <p>The identification of reference genomes and...  2022-01-12  \n",
       "4   <p>Here we introduce the Soil BON Foodweb Team...  2022-01-12  \n",
       "5   <p>Comparisons of genomes of different species...  2022-01-15  \n",
       "6   <p>Nanopore long-read genome sequencing is eme...  2022-01-12  \n",
       "7   <p>For over two decades, midbrain dopamine was...  2022-01-12  \n",
       "8   <p>Tailoring the best treatments to cancer pat...  2022-01-12  \n",
       "9   <p>Teleost fish are one of the most species-ri...  2022-01-13  \n",
       "10  <p>Visual systems have adapted to the structur...  2022-01-12  \n",
       "11  <p>Highly pathogenic avian influenza (HPAI) vi...  2022-01-13  \n",
       "12  <p>Wild progenitors of major crops can help us...  2022-01-11  \n",
       "13  <p>How and at what pace bacteria evolve when c...  2022-01-12  \n",
       "14  <p>Structural variants (SVs) can promote speci...  2022-01-15  \n",
       "15  <p>Inherited microorganisms can selfishly mani...  2022-01-16  \n",
       "16  <p>Evidence from both population genetics and ...  2022-01-15  \n",
       "17  <p>Background: Large scale metagenome assembly...  2022-01-15  \n",
       "18  <p>Characterizing the dynamics of epidemic tra...  2022-01-14  \n",
       "19  <p>SARS-CoV-2 surveillance is crucial to ident...  2022-01-14  \n",
       "20  <p>Micronuclei (MN) are aberrant cytosolic com...  2022-01-13  \n",
       "21  <p>Hematopoietic stem cells (HSC) reconstitute...  2022-01-14  \n",
       "22  <p>The Omicron variant of severe acute respira...  2022-01-12  \n",
       "23  <p>We report patho-morphological and virologic...  2022-01-12  \n",
       "24  <p>Genome-wide mapping of the histone modifica...  2022-01-12  \n",
       "25  <sec><title>Background</title><p>Different reg...  2022-01-12  \n",
       "26  <p>Over the past two decades, the study of res...  2022-01-11  \n",
       "27  <p>Sketching methods offer computational biolo...  2022-01-12  \n",
       "28  <sec><title>Background</title><p>Several count...  2022-01-11  \n",
       "29  <p>The evolution of sex determination (SD) mec...  2022-01-15  \n",
       "30  <p>Individual RNA remains a challenging signal...  2022-01-12  \n",
       "31  <p>The genus <italic>Serratia</italic> has bee...  2022-01-11  \n",
       "32  <p>Meiotic recombination frequency varies alon...  2022-01-11  \n",
       "33  <p>Hepatocytes were the first cell-type for wh...  2022-01-12  \n",
       "34  <p>Long non-coding RNAs (lncRNAs) are pivotal ...  2022-01-13  \n",
       "35  <p>Factors shaping the distribution and abunda...  2022-01-15  \n",
       "36  <p>Coronavirus disease-2019 (COVID-19) is prim...  2022-01-12  \n",
       "37  <p>Lysosomes are well-established as the main ...  2022-01-12  \n",
       "38  <p>The use of shotgun metagenomics for AMR det...  2022-01-15  \n",
       "39  <sec><title>Importance</title><p>Results from ...  2022-01-11  \n",
       "40  <p>The first plastid evolved from an endosymbi...  2022-01-12  \n",
       "41  <sec><title>Motivation</title><p>Cell-cell com...  2022-01-14  \n",
       "42  <p>Previous studies have reported sex differen...  2022-01-12  \n",
       "43  <p>Pathogenic spirochetes can alter their morp...  2022-01-11  \n",
       "44  <p>Chromosomal instability (CIN) and epigeneti...  2022-01-12  \n",
       "45  <p>Sketching is now widely used in bioinformat...  2022-01-17  \n",
       "46  <p>Importance. COVID-19 has resulted in massiv...  2022-01-14  \n",
       "47  <p>Substance abuse and addiction represent a s...  2022-01-12  \n",
       "48  <p>Diets rich in added sugars, especially high...  2022-01-12  \n",
       "49  <p>Since the first cases the coronavirus disea...  2022-01-12  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "292 preprints (including 35 covering SARS-CoV-2, 252 from bioRxiv and 40 from medRxiv) published in the last 7 days before 2022-01-18 had been tweeted at least 3 times (maximum 2384)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.10"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
