{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15635 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=02d82d0a-8a8d-45ff-9c0b-38a52b301f45&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f4af1ed3-c115-4dbd-91b8-b6066671c27a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9eac33a8-3bfd-422f-96fb-b211c9b1b502&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b1e095ee-18cb-4acc-ab9a-abe588396215&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=3c9b1679-b114-424d-a8b7-b0bda7d2d663&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d3469c47-81c6-4a74-988c-774fbabb4189&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=755703ec-89ec-4182-a934-8e7e8e1994e8&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2739358e-8de5-46ac-b652-44ece3402e4d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7cffc1cb-5f72-4ff8-a6ca-70ce975a0086&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=6c842545-67c7-4434-bf65-e37793c0b075&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4a8dd7d4-dccb-4be1-a1fb-73d56b03307d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9bab361e-5d4b-4839-896f-cde6eda47f47&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=0428538d-2d6a-4dfc-a8bd-324bb8349209&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=35b68b09-3b40-406c-952b-908c59168c0c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=234c5c26-bfb2-453d-a733-8b7433b9218f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-23&until-occurred-date=2021-08-30\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21261912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.225102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.20231399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.20172288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.341479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.28.20029272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/135582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457153...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.04.451053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.11.20229708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433127...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.17.20156034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.455988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21262021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.23.449503...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.03.075069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.439272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457694...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21253964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21252086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.10.376814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.21257129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.438781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.25.398172...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.08.12.455048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275407.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.20205633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.380741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.428136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.24.436752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.27.922062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.332023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.445128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.444774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.261586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.213132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.424421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21261814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.07.20084319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.445019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.20248932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.13.039917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.20171413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257738...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.449134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457251...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.21251670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/571380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21259864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.21.20236018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.454022...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/715482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.425021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.451001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.434040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.20157792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/596718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.11.20062372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.15.150912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.18.159434...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.452439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.03.20117994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.444019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.437224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21257809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20159228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/613141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.13.247304...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.07.425768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.214148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.383083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450190...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20075838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.14.21251715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.14.21251650...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21261000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.06.20147199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21251066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.11.21251562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.441313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20223891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.15.20065995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.10.036418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.434453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451547...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.22.309229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.424746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/809988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.13.944777...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.01.322164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.451089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.447233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.31.426979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.14.150490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20124461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.433940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20118877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.30.125237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.455365...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.21253051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/292581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.20109231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.21255081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.27.20161976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.31.276386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.438087...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.437635...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.225854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21254000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255438...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.244459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.21257879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.21260732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.093195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.447418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.05.078659...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.27.20081893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.451480...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.449914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21260074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.04.20122481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.440647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.07.20145979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/037762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21257080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.11.378133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.434227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262170...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.09.374710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.20247205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.425825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457043...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.27.20220905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.09.374165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.427855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21249554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.415687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.21249998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.20209114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430513...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.422352...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.08.20.457153', 'tweets': 56, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Loops, TADs, Compartments, and Territories are Elastic and Robust to Dramatic Nuclear Volume Swelling', 'authors': \"[{'name': 'Jacob T. Sanders'}, {'name': 'Rosela Golloshi'}, {'name': 'Peyton H. Terry'}, {'name': 'Darrian G. Nash'}, {'name': 'Yang Xu'}, {'name': 'Job Dekker'}, {'name': 'Rachel Patton McCord'}]\", 'abstract': '<p>Layers of genome organization are becoming increasingly better characterized, but less is known about how these structures respond to perturbation or shape changes. Low-salt swelling of isolated chromatin fibers or nuclei has been used for decades to investigate the structural properties of chromatin. But, visible changes in chromatin appearance have not been linked to known building blocks of genome structure or features along the genome sequence. We combine low-salt swelling of isolated nuclei with genome-wide chromosome conformation capture (Hi-C) and imaging approaches to probe the effects of chromatin extension genome-wide. Photoconverted patterns on nuclei during expansion and contraction indicate that global genome structure is preserved after dramatic nuclear volume swelling, suggesting a highly elastic chromosome topology. Hi-C experiments before, during, and after nuclear swelling show changes in average contact probabilities at short length scales, reflecting the extension of the local chromatin fiber. But, surprisingly, during this large increase in nuclear volume, there is a striking maintenance of loops, TADs, active and inactive compartments, and chromosome territories. Subtle differences after expansion are observed, suggesting that the local chromatin state, protein interactions, and location in the nucleus can affect how strongly a given structure is maintained under stress. From these observations, we propose that genome topology is robust to extension of the chromatin fiber and isotropic shape change, and that this elasticity may be beneficial in physiological circumstances of changes in nuclear size and volume.</p>', 'posted': '2021-08-23'}, {'doi': '10.1101/2021.08.25.457692', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'The Lambda variant of SARS-CoV-2 has a better chance than the Delta variant to escape vaccines', 'authors': \"[{'name': 'Haolin Liu'}, {'name': 'Pengcheng Wei'}, {'name': 'Qianqian Zhang'}, {'name': 'Katja Aviszus'}, {'name': 'Jared Linderberger'}, {'name': 'John Yang'}, {'name': 'Junfeng Liu'}, {'name': 'Zhongzhou Chen'}, {'name': 'Hassan Waheed'}, {'name': 'Lyndon Reynoso'}, {'name': 'Gregory P. Downey'}, {'name': 'Stephen K. Frankel'}, {'name': 'John Kappler'}, {'name': 'Philippa Marrack'}, {'name': 'Gongyi Zhang'}]\", 'abstract': '<p>The newly emerging variants of SARS-CoV-2 from India (Delta variant) and South America (Lambda variant) have led to a higher infection rate of either vaccinated or unvaccinated people. We found that sera from Pfizer-BioNTech vaccine remain high reactivity toward the receptor binding domain (RBD) of Delta variant while it drops dramatically toward that of Lambda variant. Interestingly, the overall titer of antibodies of Pfizer-BioNTech vaccinated individuals drops 3-fold after 6 months, which could be one of major reasons for breakthrough infections, emphasizing the importance of potential third boost shot. While a therapeutic antibody, Bamlanivimab, decreases binding affinity to Delta variant by ~20 fold, it fully lost binding to Lambda variant. Structural modeling of complexes of RBD with human receptor, Angiotensin Converting Enzyme 2 (ACE2), and Bamlanivimab suggest the potential basis of the change of binding. The data suggest possible danger and a potential surge of Lambda variant in near future.</p>', 'posted': '2021-08-26'}, {'doi': '10.1101/2021.08.25.457694', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'SOX transcription factors direct TCF-independent WNT/beta-catenin transcription', 'authors': \"[{'name': 'Shreyasi Mukherjee'}, {'name': 'David M. Luedeke'}, {'name': 'Leslie Brown'}, {'name': 'Aaron M. Zorn'}]\", 'abstract': '<p>WNT/-catenin signaling regulates gene expression across numerous biological contexts including development, stem cell homeostasis and tissue regeneration, and dysregulation of this pathway has been implicated in many diseases including cancer. One fundamental question is how distinct WNT target genes are activated in a context-specific manner, given the dogma that most, if not all, WNT/-catenin responsive transcription is mediated by TCF/LEF transcription factors (TFs) that have similar DNA-binding specificities. Here we show that the SOX family of TFs direct lineage-specific WNT/-catenin responsive transcription during the differentiation of human pluripotent stem cells (hPSCs) into definitive endoderm (DE) and neuromesodermal progenitors (NMPs). Using time-resolved multi-omics analyses, we show that -catenin association with chromatin is highly dynamic, colocalizing with distinct TCFs and/or SOX TFs at distinct stages of differentiation, indicating both cooperative and competitive modes of genomic interactions. We demonstrate that SOX17 and SOX2 are required to recruit -catenin to hundreds of lineage-specific WNT-responsive enhancers, many of which are not occupied by TCFs. At a subset of these TCF-independent enhancers, SOX TFs are required to both establish a permissive chromatin landscape and recruit a WNT-enhanceosome complex that includes -catenin, BCL9, PYGO and transcriptional coactivators to direct SOX/-catenin-dependent transcription. Given that SOX TFs are expressed in almost every cell type, these results have broad mechanistic implications for the specificity of WNT responses across many developmental and disease contexts.</p>', 'posted': '2021-08-26'}, {'doi': '10.1101/2021.08.24.457563', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The Evolution of Nest Site Specialization and its Macroevolutionary Consequences in Passerine Birds', 'authors': \"[{'name': 'Rosana Zenil-Ferguson'}, {'name': 'Jay P. McEntee'}, {'name': 'J. Gordon Burleigh'}, {'name': 'Rene A. Duckworth'}]\", 'abstract': '<p>A long-standing hypothesis in evolutionary biology is that the evolution of resource specialization can lead to an evolutionary dead end, where specialists have low diversification rates and limited ability to evolve into generalists. However, in recent years, advances in comparative methods investigating trait-based differences associated with diversification have enabled more robust tests of this idea and have found mixed support. Here we test the evolutionary dead end hypothesis by estimating net diversification rate differences associated with nest site specialization among 3,224 species of passerine birds. In particular, we test whether the adoption of hole-nesting, a nest site specialization that decreases predation, results in reduced diversification rates relative to nesting outside of holes. Further, we examine whether evolutionary transitions to the specialist hole-nesting state have been more frequent than transitions out of hole-nesting. Using diversification models that accounted for background rate heterogeneity and different extinction rate scenarios, we found that hole-nesting specialization was not associated with diversification rate differences. Furthermore, contrary to the assumption that specialists rarely evolve into generalists, we found that transitions out of hole-nesting occur more frequently than transitions into hole-nesting. These results suggest that interspecific competition may limit adoption of hole-nesting, but that such competition does not result in limited diversification of hole-nesters. In conjunction with other recent studies using robust comparative methods, our results add to growing evidence that evolutionary dead ends are not a typical outcome of resource specialization.</p>', 'posted': '2021-08-25'}, {'doi': '10.1101/2021.08.20.21261814', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Comprehensive genetic analysis of the human lipidome identifies novel loci controlling lipid homeostasis with links to coronary artery disease', 'authors': \"[{'name': 'Gemma Cadby'}, {'name': 'Corey Giles'}, {'name': 'Phillip E Melton'}, {'name': 'Kevin Huynh'}, {'name': 'Natalie A Mellett'}, {'name': 'Thy Duong'}, {'name': 'Anh Nguyen'}, {'name': 'Michelle Cinel'}, {'name': 'Alex Smith'}, {'name': 'Gavriel Olshansky'}, {'name': 'Tingting Wang'}, {'name': 'Marta Brozynska'}, {'name': 'Mike Inouye'}, {'name': 'Nina S McCarthy'}, {'name': 'Amir Ariff'}, {'name': 'Joseph Hung'}, {'name': 'Jennie Hui'}, {'name': 'John Beilby'}, {'name': 'Marie-Pierre Dub'}, {'name': 'Gerald F Watts'}, {'name': 'Sonia Shah'}, {'name': 'Naomi R Wray'}, {'name': 'Wei Ling Florence Lim'}, {'name': 'Pratishtha Chatterjee'}, {'name': 'Ian Martins'}, {'name': 'Simon M Laws'}, {'name': 'Tenielle Porter'}, {'name': 'Michael Vacher'}, {'name': 'Ashley I Bush'}, {'name': 'Christopher C Rowe'}, {'name': 'Victor L Villemagne'}, {'name': 'David Ames'}, {'name': 'Colin L Masters'}, {'name': 'Kevin Taddei'}, {'name': 'Matthias Arnold'}, {'name': 'Gabi Kastenmller'}, {'name': 'Kwangsik Nho'}, {'name': 'Andrew J Saykin'}, {'name': 'Xianlin Han'}, {'name': 'Rima Kaddurah-Daouk'}, {'name': 'Ralph N Martins'}, {'name': 'John Blangero'}, {'name': 'Peter J Meikle'}, {'name': 'Eric K Moses'}]\", 'abstract': '<p>We integrated lipidomics and genomics to unravel the genetic architecture of lipid metabolism and identify genetic variants associated with lipid species that are putatively in the mechanistic pathway to coronary artery disease (CAD). We quantified 596 lipid species in serum from 4,492 phenotyped individuals from the Busselton Health Study. In our discovery GWAS we identified 667 independent loci associations with these lipid species (479 novel), followed by meta-analysis and validation in two independent cohorts. Lipid endophenotypes (134) identified for CAD were associated with variation at 186 genomic loci. Associations between independent lipid-loci with coronary atherosclerosis were assessed in 456,000 individuals from the UK Biobank. Of the 53 lipid-loci that showed evidence of association (P&lt;110<sup>3</sup>), 43 loci were associated with at least one of the 134 lipid endophenotypes. The findings of this study illustrate the value of integrative biology to investigate the genetics and lipid metabolism in the aetiology of atherosclerosis and CAD, with implications for other complex diseases.</p>', 'posted': '2021-08-25'}, {'doi': '10.1101/2021.08.23.457408', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Deep immune profiling of the maternal-fetal interface with mild SARS-CoV-2 infection', 'authors': \"[{'name': 'Suhas Sureshchandra'}, {'name': 'Michael Z Zulu'}, {'name': 'Brianna Doratt'}, {'name': 'Allen Jankeel'}, {'name': 'Delia Tifrea'}, {'name': 'Robert Edwards'}, {'name': 'Monica Rincon'}, {'name': 'Nicole E. Marshall'}, {'name': 'Ilhem Messaoudi'}]\", 'abstract': '<p>Pregnant women are an at-risk group for severe COVID-19, though the majority experience mild/asymptomatic disease. Although severe COVID-19 has been shown to be associated with immune activation at the maternal-fetal interface even in the absence of active viral replication, the immune response to asymptomatic/mild COVID-19 remains unknown. Here, we assessed immunological adaptations in both blood and term decidua from 9 SARS-exposed pregnant women with asymptomatic/mild disease and 15 pregnant SARS-naive women. In addition to selective loss of tissue-resident decidual macrophages, we report attenuation of antigen presentation and type I IFN signaling but upregulation of inflammatory cytokines and chemokines in blood monocyte derived decidual macrophages. On the other hand, infection was associated with remodeling of the T cell compartment with increased frequencies of activated CD69+ tissue-resident T cells and decreased abundance of Tregs. Interestingly, frequencies of cytotoxic CD4 and CD8 T cells increased only in the blood, while CD8 effector memory T cells were expanded in the decidua. In contrast to decidual macrophages, signatures of type I IFN signaling were increased in decidual T cells. Finally, T cell receptor diversity was significantly reduced with infection in both compartments, albeit to a much greater extent in the blood. The resulting aberrant immune activation in the placenta, even with asymptomatic disease may alter the exquisitely sensitive developing fetal immune system, leading to long-term adverse outcomes for offspring.</p>', 'posted': '2021-08-23'}, {'doi': '10.1101/2021.08.22.457251', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Modeling movie-evoked human brain activity using motion-energy and space-time vision transformer features', 'authors': \"[{'name': 'Shinji Nishimoto'}]\", 'abstract': '<p>In this paper, the process of building a model for predicting human brain activity under video viewing conditions was described as a part of an entry into the Algonauts Project 2021 Challenge. The model was designed to predict brain activity measured using functional MRI (fMRI) by weighted linear summations of the spatiotemporal visual features that appear in the video stimuli (video features). Two types of video features were used: (1) motion-energy features designed based on neurophysiological findings, and (2) features derived from a space-time vision transformer (TimeSformer). To utilize the features of various video domains, the features of the TimeSformer models pre-trained using several different movie sets were combined. Through these model building and validation processes, results showed that there is a certain correspondence between the hierarchical representation of the TimeSformer model and the hierarchical representation of the visual system in the brain. The motion-energy features are effective in predicting brain activity in the early visual areas, while TimeSformer-derived features are effective in higher-order visual areas, and a hybrid model that uses motion energy and TimeSformer features is effective for predicting whole brain activity.</p>', 'posted': '2021-08-23'}, {'doi': '10.1101/2021.08.24.457556', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Interactions between a mechanosensitive channel and cell wall integrity signaling influence pollen germination in Arabidopsis thaliana', 'authors': \"[{'name': 'Yanbing Wang'}, {'name': 'Joshua Coomey'}, {'name': 'Kari Miller'}, {'name': 'Gregory S. Jensen'}, {'name': 'Elizabeth S. Haswell'}]\", 'abstract': '<p>Cells employ multiple systems to maintain cellular integrity, including mechanosensitive (MS) ion channels and the cell wall integrity (CWI) pathway. Here, we use pollen as a model system to ask how these different mechanisms are interconnected at the cellular level. MscS-Like (MSL)8 is an MS channel required to protect <italic>Arabidopsis thaliana</italic> pollen from osmotic challenges during <italic>in vitro</italic> rehydration, germination and tube growth. New CRISPR/Cas9 and artificial microRNA-generated <italic>msl8</italic> alleles produced unexpected pollen phenotypes, including the ability to germinate a tube after bursting, dramatic defects in cell wall structure and disorganized callose deposition at the germination site. We document complex genetic interactions between <italic>MSL8</italic> and two previously established components of the CWI pathway, <italic>MARIS</italic>, and <italic>ANXUR1/2</italic>. Overexpression of <italic>MARIS<sup>R240C</sup>-FP</italic> suppressed the bursting, germination, and callose deposition phenotypes of <italic>msl8</italic> mutant pollen. Null <italic>msl8</italic> alleles suppressed the internalized callose structures observed in <italic>MARIS<sup>R240C</sup>-FP</italic> lines. Similarly, <italic>MSL8-YFP</italic> overexpression suppressed bursting in the <italic>anxur1/2</italic> mutant background, while <italic>anxur1/2</italic> alleles reduced the strong rings of callose around ungerminated pollen grains in <italic>MSL8-YFP</italic> over-expressors. These data show that MS ion channels modulate callose deposition in pollen and provides evidence that cell wall and membrane surveillance systems coordinate in a complex manner to maintain cell integrity.</p>', 'posted': '2021-08-25'}, {'doi': '10.1101/2021.08.25.457601', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Plant Polymerase IV sensitizes chromatin through histone modifications to preclude spread of silencing into protein-coding domains', 'authors': \"[{'name': 'Vivek Hari Sundar G'}, {'name': 'Chenna Swetha'}, {'name': 'Debjani Basu'}, {'name': 'Kannan Pachamuthu'}, {'name': 'Tania Chakraborty'}, {'name': 'Rebecca A. Mosher'}, {'name': 'P. V. Shivaprasad'}]\", 'abstract': '<p>Heterochromatin is the predominant architectural feature of genomes that ensures genomic stability across eukaryotes. It mostly functions in restricting expression of repeats, mobile elements such as transposons and other regions. The establishment, maintenance and spreading of heterochromatin requires several factors including chromatin modifiers. However, how exactly heterochromatin formation is avoided in protein-coding domains is poorly understood. Here we show that a plant specific paralogue of RNA polymerase (pol) II, named pol IV, is involved in avoidance of facultative heterochromatic marks in protein coding genes, in addition to silencing the repeats and transposons forming constitutive heterochromatin. In its absence, H3K27 trimethylation mark intrudes the protein coding genes, more profoundly in genes embedded with repeats. In a subset of genes that lack the compensatory silencing, spurious transcriptional activity results in small(s)RNA production leading to post-transcriptional gene silencing. We show that such effects are significantly pronounced in rice, a plant with larger genome with distributed heterochromatin when compared to Arabidopsis. Our results indicate the surprising division of labour among plant-specific polymerases, not just in establishing effective silencing via small RNAs and epigenetics, but also in influencing chromatin boundaries.</p><sec><title>Abstract Figure</title><fig id=\"ufig1\" position=\"anchor\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"457601v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec><sec><title>One-line summary</title><p>RNA polymerase IV maintains chromatin states to influence regulation of protein coding genes.</p></sec>', 'posted': '2021-08-25'}, {'doi': '10.1101/2021.08.22.457032', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'A multiscale spatiotemporal model explains succession in the early infant gut microbiota as a switch from aerobic to anaerobic metabolism', 'authors': \"[{'name': 'David M. Versluis'}, {'name': 'Ruud Schoemaker'}, {'name': 'Ellen Looijesteijn'}, {'name': 'Danil Muysken'}, {'name': 'Prescilla V. Jeurink'}, {'name': 'Marcel Paques'}, {'name': 'Jan M. W. Geurts'}, {'name': 'Roeland M. H. Merks'}]\", 'abstract': '<p>The human intestinal microbiome starts to form immediately after birth, and can greatly influence the health of the infant. During the first days facultative anaerobic species generally dominate, followed by a dominance of strictly anaerobic species, particularly <italic>Bifidobacterium</italic> species. An early transition to <italic>Bifidobacterium</italic> is associated with health benefits. To study the mechanisms of this transition and its hypothesised relation to oxygen, we introduce a multiscale mathematical model that considers metabolism, spatial bacterial population dynamics and resource sharing. Based on publicly available metabolic network data, the model predicts that differences in oxygen availability explain some of the observed individual variation in succession to anaerobic species. The model also predicts that anaerobic <italic>Bifidobacterium</italic> species become dominant through metabolizing lactose with a suboptimal yield, but a higher anaerobic growth rate than its competitors. The current work is the first step towards a more comprehensive understanding of the formation of a steady state adult colonic microbiota.</p>', 'posted': '2021-08-23'}, {'doi': '10.1101/2021.08.22.457102', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Tracheal tube fusion in Drosophila involves release of exosomes from multivesicular bodies', 'authors': \"[{'name': 'Carolina Camelo'}, {'name': 'Anna Krte'}, {'name': 'Thea Jacobs'}, {'name': 'Peter Robin Hiesinger'}, {'name': 'Stefan Luschnig'}]\", 'abstract': '<p>Fusion of endothelial or epithelial tubes is essential for the development of organs like the vertebrate vasculature or the insect tracheal system, but the mechanisms underlying the formation of tubular connections (anastomoses) are not well understood. Tracheal tube fusion in <italic>Drosophila</italic> is mediated by tip cells that transform into lumenized toroids to connect adjacent tubes. This process depends on the Munc13-4 orthologue Staccato (Stac), which localizes to tip-cell-specific lysosome-related organelles (LROs) that display features of multivesicular bodies (MVBs). We show that the tracheal lumen contains membranous extracellular vesicles (EVs), a subset of which carries Stac/Munc13-4 and is associated with tracheal anastomosis sites. The presence of LROs and luminal Stac-EVs depends on the tip-cell-specific GTPase Arl3, suggesting that Stac-EVs derive from fusion of MVBs with the luminal membrane in tip cells during anastomosis formation. The GTPases Rab27 and Rab35 cooperate downstream of Arl3 to promote Stac-MVB formation and tube fusion. We propose that Stac-MVBs act as membrane reservoirs that facilitate lumen fusion in tip cells, in a process regulated by Arl3, Rab27, Rab35, and Stac/Munc13-4.</p>', 'posted': '2021-08-23'}, {'doi': '10.1101/2021.08.26.457786', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Pseudomonas aeruginosa differentiates substrate rigidity using retraction of type IV pili', 'authors': \"[{'name': 'Matthias D. Koch'}, {'name': 'Endao Han'}, {'name': 'Joshua W. Shaevitz'}, {'name': 'Zemer Gitai'}]\", 'abstract': '<p>The ability of eukaryotic cells to differentiate substrate stiffness is fundamental for many processes such as the development of stem cells into mature tissue. Here, we establish that bacteria feel their microenvironment in a similar manner. We show that <italic>Pseudomonas aeruginosa</italic> actively probes and measures substrate stiffness using type IV pili (TFP). The activity of the major virulence factor regulator Vfr is peaked with stiffness in a physiologically important range between 0.1 kPa (mucus) and 1000 kPa (cartilage). The local concentration of PilA at the base of dynamic TFP changes during extension and retraction in a surface dependent manner due to slow PilA diffusion in the cell membrane. Traction force measurements reveal that TFP retraction deforms even stiff substrates. Modeling of the measured substrate deformation and optical tweezers experiments suggest that TFP adhere at the tip only. Informed by these experimental results, we developed a model that describes substrate stiffness dependent dynamics of the polar PilA concentration which are quantitatively consistent with the transcriptional response to stiffness. Manipulating the ATPase activity of the TFP motors changes the TFP extension and retraction velocities and consequently the PilA concentration dynamics in a manner that is predictive of the experimental stiffness response. This work points to the use of a competition between PilA diffusion and TFP extension-retraction as a molecular shear rheometer. Our results highlight that stiffness sensing is a conserved property between the kingdoms of life.</p>', 'posted': '2021-08-26'}, {'doi': '10.1101/2021.08.24.457481', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A yeast model of the ALS protein Matrin3 uncovers Hsp90 and its co-chaperone Sti1 as modifiers of misfolding and toxicity', 'authors': \"[{'name': 'Sonja E. Di Gregorio'}, {'name': 'Mohammad Esmaeili'}, {'name': 'Ahmed Salem'}, {'name': 'Martin L. Duennwald'}]\", 'abstract': '<p>The MATR3 gene encoding the protein Matrin3 is implicated in the pathogenesis of the neurodegenerative disease amyotrophic lateral sclerosis (ALS). Matrin3 forms neuronal cytoplasmic and nuclear inclusions in ALS-affected neurons. Additionally, 13 heterozygous missense mutations in MATR3 are identified in ALS patients. To further explore Matrin3 misfolding and toxicity, we established and characterized a yeast model. We demonstrate that wild type Matrin3 and the ALS-associated variant F115C are toxic and form inclusions in yeast. Our further characterization uncovers substantial modification of Matrin3 toxicity and inclusion formation by Hsp90 and its co-chaperones, specifically Sti1. Thus, our study demonstrates how specific branches of cellular protein quality control regulate the misfolding and toxicity of Matrin3.</p><sec><title>Summary Statement</title><p>We established and characterized a yeast model expressing human Matrin3, a protein implicated in the pathogenesis of amyotrophic lateral sclerosis (ALS). Using this yeast model and mammalian neuronal cells, we showed that Matrin3 mislocalizes and forms inclusions, is cytotoxic, and increases sensitivity to cellular stress. We also uncovered that Hsp90 and particularly its co-chaperone Sti1 alter Matrin3 toxicity.</p></sec>', 'posted': '2021-08-24'}, {'doi': '10.1101/2021.08.21.457222', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Reconstructing the biogeography of a hunter-gatherer planet using machine-learning', 'authors': \"[{'name': 'Marcus J. Hamilton'}, {'name': 'Robert S. Walker'}, {'name': 'Briggs Buchanan'}, {'name': 'Damian E. Blasi'}, {'name': 'Claire L. Bowern'}]\", 'abstract': '<p>Estimating the total human population size (i.e., abundance) of the preagricultural planet is important for setting the baseline expectations for human-environment interactions if all energy and material requirements to support growth, maintenance, and well-being were foraged from local environments. However, demographic parameters and biogeographic distributions do not preserve directly in the archaeological record. Rather than attempting to estimate human abundance at some specific time in the past, a principled approach to making inferences at this scale is to ask what the human demography and biogeography of a hypothetical planet Earth would look like if populated by ethnographic hunter-gatherer societies. Given ethnographic hunter-gatherer societies likely include the largest, densest, and most complex foraging societies to have existed, we suggest population inferences drawn from this sample provide an upper bound to demographic estimates in prehistory. Our goal in this paper is to produce principled estimates of hunter-gatherer abundance, diversity, and biogeography. To do this we trained an extreme gradient boosting algorithm (XGBoost) to learn ethnographic hunter-gatherer population densities from a large matrix of climatic, environmental, and geographic data. We used the predictions generated by this model to reconstruct the hunter-gatherer biogeography of the rest of the planet. We find the human abundance of this world to be 6.12 million with an ethnolinguistic diversity of 8,3302,770 populations, most of whom would have lived near coasts and in the tropics.</p><sec><title>Significance Statement</title><p>Understanding the abundance of humans on planet Earth prior to the development of agriculture and the industrialized world is essential to understanding human population growth. However, the problem is that these features of human populations in the past are unknown and so must be estimated from data. We developed a machine learning approach that uses ethnographic and environmental data to reconstruct the demography and biogeography of planet Earth if populated by hunter-gatherers. Such a world would house about 6 million people divided into about 8,330 populations with a particular concentration in the tropics and along coasts.</p></sec>', 'posted': '2021-08-24'}, {'doi': '10.1101/2021.08.22.457271', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Vapor cannabis exposure generationally affects male reproductive functions in mice', 'authors': \"[{'name': 'Mingxin Shi'}, {'name': 'Esther M. Langholt'}, {'name': 'Logan C. Butler'}, {'name': 'Madeleine E. Harvey'}, {'name': 'Emma C. Wheeler'}, {'name': 'Liang Zhao'}, {'name': 'James A. MacLean'}, {'name': 'Yeongseok Oh'}, {'name': 'Emily Sabrowsky'}, {'name': 'Sue Yu'}, {'name': 'Shane Watson'}, {'name': 'Jon F. Davis'}, {'name': 'Kanako Hayashi'}]\", 'abstract': '<p>This study was performed to examine whether vapor exposure to cannabis plant matter negatively impacts male reproductive functions and testis development in mice. Adult CD-1 male mice (F0) were exposed to air (control) or 200 mg of vaporized cannabis plant matter 3x/day over a 10 day period. Subsequently, F0 males were bred with drug nave CD-1 females to generate F1 males, and F1 offspring were used to generate F2 males. Cannabis vapor exposure decreased sperm count and/or motility in F0 and F1 males and disrupted the progression of germ cell development, as morphometric analyses exhibited an abnormal distribution of the stages of spermatogenesis in F0 males. Although plasma levels of testosterone were not affected by cannabis exposure in any ages or generations of males, dysregulated steroidogenic enzymes, <italic>Cyp11a1</italic> and <italic>Cyp19a1</italic>, were observed in F0 testis. In the neonatal testis from F1 males, while apoptosis was not altered, DNA damage and DNMT1, but not DNMT3A and DNMT3B, were increased in germ cells following cannabis exposure. In contrast, the alterations of DNA damage and DNMT1 expression were not observed in F2 neonatal males. These results suggest that cannabis vapor exposure generationally affects male reproductive functions, probably due to disruption of spermatogenesis in the developing testis.</p><sec><title>GRAPHICAL ABSTRACT</title><fig id=\"ufig1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"457271v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec><sec><title>HIGHLIGHTS</title><list list-type=\"bullet\"><list-item><p>Cannabis vapor exposure to adult males impairs sperm counts, motility and spermatogenesis.</p></list-item><list-item><p>Cannabis vapor exposure to F0 males causes reduced sperm counts in F1, but not in F2 males.</p></list-item><list-item><p>Plasma testosterone levels are not affected in F0, F1 and F2 males by cannabis exposure.</p></list-item><list-item><p>Cannabis increases DNA damage and the expression of DNA methyltransferase (DNMT) 1 in the F1 neonatal testis.</p></list-item></list></sec>', 'posted': '2021-08-24'}, {'doi': '10.1101/2021.08.23.457415', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Scientific Communication and Education', 'covid': False, 'title': 'Time Evolution of the Stroke Symptom-Herb Networks Based on TCM Prescriptions', 'authors': \"[{'name': 'Xing Fu'}, {'name': 'Qianqian Li'}, {'name': 'Feng Yang'}, {'name': 'Ranran Zhou'}, {'name': 'Ran Li'}, {'name': 'Jianchen Hou'}, {'name': 'Xiaohua Tao'}]\", 'abstract': '<p>Traditional Chinese Medicine (TCM) has its origins in distant antiquity and has piled up over a long time with much knowledge about diseases, especially stroke. Different combinations of symptom variables yield different combinations of herbs to form a myriad of prescriptions, and they have undergone repeated confirmation and are worthy objects of excavation and analysis. Herbal studies on stroke have developed from genomics to transcriptomics, proteomics and metabolomics, yet more thought is needed on putting time in a wider perspective of the symptoms and herbs for stroke in Chinese medicine.</p><p>Due to this, we studied the dynamic structure of TCM prescriptions on stroke, using over 270 TCM prescription books containing 2231 prescriptions related to stroke recorded from 341 to 2000 CE. We labeled the functions of the prescriptions with the symptoms based on subject terms in MESH <italic>Neurologic Manifestations</italic>, then standardized the herbs in the prescriptions, and finally connected the co-occurring symptoms and herbs in the prescriptions to build an undirected complex network. The Stroke Symptom-Herb Networks (SSHNs) can be seen from its network characteristics that it is not a random network and has small-world characteristics. It has experienced two peaks in its nearly 1700-year history, during the Song dynasty, the Ming and Qing dynasties. From 600 years onwards, the core herb cluster has been initially formed. The comparison of sub-network similarities allowed us to identify several symptoms with similar herb clusters.</p><p>We divided the community based on modularity, and by analyzing the community evolution, we found a more fixed historical evolutionary trend with <italic>Hemiplegia</italic> and <italic>Sialorrhea</italic> nodes and their associated symptom and herb nodes. In the time series analysis, we found many symptom-herb combinations that were consistently closely related to historic time depends on assessing the similarity between the symptoms and the herbs. The complex network provides a distinctive perspective for understanding the symptom-herb relationships embedded in TCM prescriptions in remote antiquity.</p>', 'posted': '2021-08-23'}, {'doi': '10.1101/2021.08.24.457491', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'TREM2 regulates purinergic receptor-mediated calcium signaling and motility in human iPSC-derived microglia', 'authors': \"[{'name': 'Amit Jairaman'}, {'name': 'Amanda McQuade'}, {'name': 'You Jung Kang'}, {'name': 'Alberto Granzotto'}, {'name': 'Jean Paul Chadarevian'}, {'name': 'Sunil Gandhi'}, {'name': 'Ian Parker'}, {'name': 'Ian Smith'}, {'name': 'Shivashankar Othy'}, {'name': 'Hansang Cho'}, {'name': 'Mathew Blurton-Jones'}, {'name': 'Michael Cahalan'}]\", 'abstract': '<p>The membrane protein TREM2 (Triggering Receptor Expressed on Myeloid cells 2) regulates key microglial functions including phagocytosis and chemotaxis. Loss-of-function variants of TREM2 are associated with increased risk of Alzheimers disease (AD). Because abnormalities in Ca<sup>2+</sup> signaling have been observed in several AD models, we investigated TREM2 regulation of Ca<sup>2+</sup> signaling via genetic deletion in human induced pluripotent stem cell-derived microglia (iPSC- microglia). We found that iPSC-microglia lacking TREM2 (TREM2 KO) show exaggerated Ca<sup>2+</sup> signals in response to purinergic agonists such as ADP that shape microglial injury responses. This ADP hypersensitivity, driven by increased expression of P2Y12 and P2Y13 receptors, results in sustained Ca<sup>2+</sup> influx and alters cell motility and process extension in TREM2 KO microglia. In a Ca<sup>2+</sup> clamp assay using iPSC-microglia expressing the genetically encoded Ca<sup>2+</sup> probe, Salsa6f, we found that cytosolic Ca<sup>2+</sup> tunes motility to a greater extent in TREM2 KO microglia. Despite showing greater overall displacement, TREM2 KO microglia exhibit reduced directional chemotaxis along ADP gradients. Accordingly, the chemotactic defect in TREM2 KO microglia was rescued by reducing cytosolic Ca<sup>2+</sup> using a P2Y12 receptor antagonist. Our results show that loss of TREM2 confers a defect in microglial Ca<sup>2+</sup> response to purinergic signals, suggesting a window of Ca<sup>2+</sup> signaling for optimal microglial motility.</p>', 'posted': '2021-08-24'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.08.20.457153</td>\n",
       "      <td>56</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Loops, TADs, Compartments, and Territories are...</td>\n",
       "      <td>[{'name': 'Jacob T. Sanders'}, {'name': 'Rosel...</td>\n",
       "      <td>&lt;p&gt;Layers of genome organization are becoming ...</td>\n",
       "      <td>2021-08-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.08.25.457692</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>The Lambda variant of SARS-CoV-2 has a better ...</td>\n",
       "      <td>[{'name': 'Haolin Liu'}, {'name': 'Pengcheng W...</td>\n",
       "      <td>&lt;p&gt;The newly emerging variants of SARS-CoV-2 f...</td>\n",
       "      <td>2021-08-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.08.25.457694</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>SOX transcription factors direct TCF-independe...</td>\n",
       "      <td>[{'name': 'Shreyasi Mukherjee'}, {'name': 'Dav...</td>\n",
       "      <td>&lt;p&gt;WNT/-catenin signaling regulates gene expr...</td>\n",
       "      <td>2021-08-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.08.24.457563</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The Evolution of Nest Site Specialization and ...</td>\n",
       "      <td>[{'name': 'Rosana Zenil-Ferguson'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;A long-standing hypothesis in evolutionary ...</td>\n",
       "      <td>2021-08-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.08.20.21261814</td>\n",
       "      <td>9</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Comprehensive genetic analysis of the human li...</td>\n",
       "      <td>[{'name': 'Gemma Cadby'}, {'name': 'Corey Gile...</td>\n",
       "      <td>&lt;p&gt;We integrated lipidomics and genomics to un...</td>\n",
       "      <td>2021-08-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.08.23.457408</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Deep immune profiling of the maternal-fetal in...</td>\n",
       "      <td>[{'name': 'Suhas Sureshchandra'}, {'name': 'Mi...</td>\n",
       "      <td>&lt;p&gt;Pregnant women are an at-risk group for sev...</td>\n",
       "      <td>2021-08-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.08.22.457251</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Modeling movie-evoked human brain activity usi...</td>\n",
       "      <td>[{'name': 'Shinji Nishimoto'}]</td>\n",
       "      <td>&lt;p&gt;In this paper, the process of building a mo...</td>\n",
       "      <td>2021-08-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.08.24.457556</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Interactions between a mechanosensitive channe...</td>\n",
       "      <td>[{'name': 'Yanbing Wang'}, {'name': 'Joshua Co...</td>\n",
       "      <td>&lt;p&gt;Cells employ multiple systems to maintain c...</td>\n",
       "      <td>2021-08-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.08.25.457601</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Plant Polymerase IV sensitizes chromatin throu...</td>\n",
       "      <td>[{'name': 'Vivek Hari Sundar G'}, {'name': 'Ch...</td>\n",
       "      <td>&lt;p&gt;Heterochromatin is the predominant architec...</td>\n",
       "      <td>2021-08-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.08.22.457032</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A multiscale spatiotemporal model explains suc...</td>\n",
       "      <td>[{'name': 'David M. Versluis'}, {'name': 'Ruud...</td>\n",
       "      <td>&lt;p&gt;The human intestinal microbiome starts to f...</td>\n",
       "      <td>2021-08-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.08.22.457102</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Tracheal tube fusion in Drosophila involves re...</td>\n",
       "      <td>[{'name': 'Carolina Camelo'}, {'name': 'Anna K...</td>\n",
       "      <td>&lt;p&gt;Fusion of endothelial or epithelial tubes i...</td>\n",
       "      <td>2021-08-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.08.26.457786</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Pseudomonas aeruginosa differentiates substrat...</td>\n",
       "      <td>[{'name': 'Matthias D. Koch'}, {'name': 'Endao...</td>\n",
       "      <td>&lt;p&gt;The ability of eukaryotic cells to differen...</td>\n",
       "      <td>2021-08-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.08.24.457481</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A yeast model of the ALS protein Matrin3 uncov...</td>\n",
       "      <td>[{'name': 'Sonja E. Di Gregorio'}, {'name': 'M...</td>\n",
       "      <td>&lt;p&gt;The MATR3 gene encoding the protein Matrin3...</td>\n",
       "      <td>2021-08-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.08.21.457222</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Reconstructing the biogeography of a hunter-ga...</td>\n",
       "      <td>[{'name': 'Marcus J. Hamilton'}, {'name': 'Rob...</td>\n",
       "      <td>&lt;p&gt;Estimating the total human population size ...</td>\n",
       "      <td>2021-08-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.08.22.457271</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pharmacology and Toxicology</td>\n",
       "      <td>False</td>\n",
       "      <td>Vapor cannabis exposure generationally affects...</td>\n",
       "      <td>[{'name': 'Mingxin Shi'}, {'name': 'Esther M. ...</td>\n",
       "      <td>&lt;p&gt;This study was performed to examine whether...</td>\n",
       "      <td>2021-08-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.08.23.457415</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Scientific Communication and Education</td>\n",
       "      <td>False</td>\n",
       "      <td>Time Evolution of the Stroke Symptom-Herb Netw...</td>\n",
       "      <td>[{'name': 'Xing Fu'}, {'name': 'Qianqian Li'},...</td>\n",
       "      <td>&lt;p&gt;Traditional Chinese Medicine (TCM) has its ...</td>\n",
       "      <td>2021-08-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.08.24.457491</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>TREM2 regulates purinergic receptor-mediated c...</td>\n",
       "      <td>[{'name': 'Amit Jairaman'}, {'name': 'Amanda M...</td>\n",
       "      <td>&lt;p&gt;The membrane protein TREM2 (Triggering Rece...</td>\n",
       "      <td>2021-08-24</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.08.20.457153      56  bioRxiv   \n",
       "1     10.1101/2021.08.25.457692      24  bioRxiv   \n",
       "2     10.1101/2021.08.25.457694      20  bioRxiv   \n",
       "3     10.1101/2021.08.24.457563       9  bioRxiv   \n",
       "4   10.1101/2021.08.20.21261814       9  medRxiv   \n",
       "5     10.1101/2021.08.23.457408       9  bioRxiv   \n",
       "6     10.1101/2021.08.22.457251       8  bioRxiv   \n",
       "7     10.1101/2021.08.24.457556       7  bioRxiv   \n",
       "8     10.1101/2021.08.25.457601       7  bioRxiv   \n",
       "9     10.1101/2021.08.22.457032       7  bioRxiv   \n",
       "10    10.1101/2021.08.22.457102       5  bioRxiv   \n",
       "11    10.1101/2021.08.26.457786       5  bioRxiv   \n",
       "12    10.1101/2021.08.24.457481       4  bioRxiv   \n",
       "13    10.1101/2021.08.21.457222       4  bioRxiv   \n",
       "14    10.1101/2021.08.22.457271       3  bioRxiv   \n",
       "15    10.1101/2021.08.23.457415       3  bioRxiv   \n",
       "16    10.1101/2021.08.24.457491       3  bioRxiv   \n",
       "\n",
       "                              subject-area  covid  \\\n",
       "0                                 Genomics  False   \n",
       "1                               Immunology   True   \n",
       "2                    Developmental Biology  False   \n",
       "3                     Evolutionary Biology  False   \n",
       "4             Genetic and Genomic Medicine  False   \n",
       "5                               Immunology   True   \n",
       "6                             Neuroscience  False   \n",
       "7                            Plant Biology  False   \n",
       "8                        Molecular Biology  False   \n",
       "9                          Systems Biology  False   \n",
       "10                   Developmental Biology  False   \n",
       "11                            Microbiology  False   \n",
       "12                       Molecular Biology  False   \n",
       "13                                 Ecology  False   \n",
       "14             Pharmacology and Toxicology  False   \n",
       "15  Scientific Communication and Education  False   \n",
       "16                            Neuroscience  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Loops, TADs, Compartments, and Territories are...   \n",
       "1   The Lambda variant of SARS-CoV-2 has a better ...   \n",
       "2   SOX transcription factors direct TCF-independe...   \n",
       "3   The Evolution of Nest Site Specialization and ...   \n",
       "4   Comprehensive genetic analysis of the human li...   \n",
       "5   Deep immune profiling of the maternal-fetal in...   \n",
       "6   Modeling movie-evoked human brain activity usi...   \n",
       "7   Interactions between a mechanosensitive channe...   \n",
       "8   Plant Polymerase IV sensitizes chromatin throu...   \n",
       "9   A multiscale spatiotemporal model explains suc...   \n",
       "10  Tracheal tube fusion in Drosophila involves re...   \n",
       "11  Pseudomonas aeruginosa differentiates substrat...   \n",
       "12  A yeast model of the ALS protein Matrin3 uncov...   \n",
       "13  Reconstructing the biogeography of a hunter-ga...   \n",
       "14  Vapor cannabis exposure generationally affects...   \n",
       "15  Time Evolution of the Stroke Symptom-Herb Netw...   \n",
       "16  TREM2 regulates purinergic receptor-mediated c...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Jacob T. Sanders'}, {'name': 'Rosel...   \n",
       "1   [{'name': 'Haolin Liu'}, {'name': 'Pengcheng W...   \n",
       "2   [{'name': 'Shreyasi Mukherjee'}, {'name': 'Dav...   \n",
       "3   [{'name': 'Rosana Zenil-Ferguson'}, {'name': '...   \n",
       "4   [{'name': 'Gemma Cadby'}, {'name': 'Corey Gile...   \n",
       "5   [{'name': 'Suhas Sureshchandra'}, {'name': 'Mi...   \n",
       "6                      [{'name': 'Shinji Nishimoto'}]   \n",
       "7   [{'name': 'Yanbing Wang'}, {'name': 'Joshua Co...   \n",
       "8   [{'name': 'Vivek Hari Sundar G'}, {'name': 'Ch...   \n",
       "9   [{'name': 'David M. Versluis'}, {'name': 'Ruud...   \n",
       "10  [{'name': 'Carolina Camelo'}, {'name': 'Anna K...   \n",
       "11  [{'name': 'Matthias D. Koch'}, {'name': 'Endao...   \n",
       "12  [{'name': 'Sonja E. Di Gregorio'}, {'name': 'M...   \n",
       "13  [{'name': 'Marcus J. Hamilton'}, {'name': 'Rob...   \n",
       "14  [{'name': 'Mingxin Shi'}, {'name': 'Esther M. ...   \n",
       "15  [{'name': 'Xing Fu'}, {'name': 'Qianqian Li'},...   \n",
       "16  [{'name': 'Amit Jairaman'}, {'name': 'Amanda M...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>Layers of genome organization are becoming ...  2021-08-23  \n",
       "1   <p>The newly emerging variants of SARS-CoV-2 f...  2021-08-26  \n",
       "2   <p>WNT/-catenin signaling regulates gene expr...  2021-08-26  \n",
       "3   <p>A long-standing hypothesis in evolutionary ...  2021-08-25  \n",
       "4   <p>We integrated lipidomics and genomics to un...  2021-08-25  \n",
       "5   <p>Pregnant women are an at-risk group for sev...  2021-08-23  \n",
       "6   <p>In this paper, the process of building a mo...  2021-08-23  \n",
       "7   <p>Cells employ multiple systems to maintain c...  2021-08-25  \n",
       "8   <p>Heterochromatin is the predominant architec...  2021-08-25  \n",
       "9   <p>The human intestinal microbiome starts to f...  2021-08-23  \n",
       "10  <p>Fusion of endothelial or epithelial tubes i...  2021-08-23  \n",
       "11  <p>The ability of eukaryotic cells to differen...  2021-08-26  \n",
       "12  <p>The MATR3 gene encoding the protein Matrin3...  2021-08-24  \n",
       "13  <p>Estimating the total human population size ...  2021-08-24  \n",
       "14  <p>This study was performed to examine whether...  2021-08-24  \n",
       "15  <p>Traditional Chinese Medicine (TCM) has its ...  2021-08-23  \n",
       "16  <p>The membrane protein TREM2 (Triggering Rece...  2021-08-24  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "17 preprints (including 2 covering SARS-CoV-2, 16 from bioRxiv and 1 from medRxiv) published in the last 7 days before 2021-08-30 had been tweeted at least 3 times (maximum 56)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.6"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
