{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "12792 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=df405233-f398-4063-b960-2ec661523c4f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f58d12c0-1a53-41bb-b5cf-d5c1f732030f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c31666a9-3fa5-41e7-aa37-d36eea7f577e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=3c6353e2-3e20-4ff5-b077-2d06016db99a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5275acc2-c3f2-4a30-ab6a-a80fd6dcfa96&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=88b34e35-2b58-4481-82ee-d9dba2b63317&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f6a280d1-ccd8-4847-865c-f03f6555422f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=02b7a1df-eefb-4175-a2a3-55a491918aa5&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e238516b-56b8-4ebb-b659-9ad64bb30f02&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b3fa064b-e579-4a51-87cd-132d5d97c087&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=94aacefa-c6ac-4137-8d8d-8adde7b8926f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5a5e3ffe-4633-40ee-ae85-411310d6dea5&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-13&until-occurred-date=2021-09-20\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.16.20023754...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20159228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21261912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.335083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.455288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.09.10.459728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.20172288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.09.373258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459669...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.21249563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.21251670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.09.416438...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.437441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20124461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.424510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.21263331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.437510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.443470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.20167205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.12.20230292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/135582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.437224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.455988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261929...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.441398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.438511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.23.353045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.07.20145979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.19.431931...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.439272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.422737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21261000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459178...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435343...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.409151...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.21263146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.380741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.07.21249419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.22.258459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.442971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.442687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.28.20029272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.03.075069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.447951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.09.196444...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455713...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.449152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.261586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.434390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.432780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460445...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.447255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.21263189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.20.20108183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20213546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.14.20062463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.451567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454155...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.459529...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.21.392639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.21256049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.13.039917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.428136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.444467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.459918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.213132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.419044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.28.316794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.20248932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.21249603...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.21253183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.21261496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460322...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.11.439311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.441157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452462...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.23.263665...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/571380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.460651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/440321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.21261544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.458031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442775...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20132639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.21255454...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.21263026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.444292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.425825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454085...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.444494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.18.20070417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.434040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/715482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.07.030650...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.460001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.17.21260679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.17.452576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.458941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460257...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268169.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.455550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.440978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460235...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.21.108316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262945...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.21251942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.21253542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.460530...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.21255094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.459937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.11.20192997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.332023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263383...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.04.20225573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.459767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.383083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.340588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.19.20214494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.437240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.430920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448347...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.15.20065995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.10.036418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432170...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.29.20041962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275607.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443549...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.04.30.442096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20223891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.25.21252363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/782557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452022...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/758540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.403477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.21261083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.460688...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460151...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20151423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.434223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.424565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.454011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423191...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.21256274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258335...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/292581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/318329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.451001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.14.435285...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.433940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.443762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255841...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.440438...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21254000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/037762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.444019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.28.176560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260961...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.25.20079251...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460213...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21259050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.225854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.20244145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.20245977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.415687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.449914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.11.421693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.20.050237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20068528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.04.20122481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.447990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.21249236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.458711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.372276...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/238527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.13.20211359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.093195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.10.20097543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.10.20191619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21258780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20216630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.23.352443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.27.20220905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.459983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.29.20081174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/523712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.21249998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.11.430703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.21251652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.431090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.27.011361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.23.20039446...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.21251899...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.21258079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21257993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.06.20147199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.455487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.25.21252415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.21251526...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.214148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.20.21250152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.20167932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.22.427830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21249554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.21262633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.21258272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.31.426979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.454052...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.21251112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.14.150490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.21251134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460073...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.09.14.460138', 'tweets': 39, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structure of a bacterial Rhs effector exported by the type VI secretion system', 'authors': \"[{'name': 'Patrick Gnther'}, {'name': 'Dennis Quentin'}, {'name': 'Shehryar Ahmad'}, {'name': 'Kartik Sachar'}, {'name': 'Christos Gatsogiannis'}, {'name': 'John C. Whitney'}, {'name': 'Stefan Raunser'}]\", 'abstract': '<p>The type VI <underline>s</underline>ecretion <underline>s</underline>ystem (T6SS) is a widespread protein export apparatus found in Gram-negative bacteria. The majority of T6SSs deliver toxic effector proteins into competitor bacteria. Yet, the structure, function, and activation of many of these effectors remains poorly understood. Here, we present the structures of the T6SS effector RhsA from <italic>Pseudomonas protegens</italic> and its cognate T6SS spike protein, VgrG1, at 3.3  resolution. The structures reveal that the <underline>r</underline>earrangement <underline>h</underline>ot<underline>s</underline>pot (Rhs) repeats of RhsA assemble into a closed anticlockwise -barrel spiral similar to that found in bacterial insecticidal Tc toxins and in metazoan teneurin proteins. We find that the C-terminal toxin domain of RhsA is autoproteolytically cleaved but remains inside the Rhs cocoon where, with the exception of three ordered structural elements, most of the toxin is disordered. The N-terminal plug domain is unique to T6SS Rhs proteins and resembles a champagne cork that seals the Rhs cocoon at one end while also mediating interactions with VgrG1. Interestingly, this domain is also autoproteolytically cleaved inside the cocoon but remains associated with it. We propose that mechanical force is required to remove the cleaved part of the plug, resulting in the release of the toxin domain as it is delivered into a susceptible bacterial cell by the T6SS.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.13.460111', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Therapeutic efficacy of an oral nucleoside analog of remdesivir against SARS-CoV-2 pathogenesis in mice.', 'authors': \"[{'name': 'Alexandra Schafer'}, {'name': 'David R Martinez'}, {'name': 'John J Won'}, {'name': 'Fernanado R Moreira'}, {'name': 'Ariane J Brown'}, {'name': 'Kendra L Gully'}, {'name': 'Rao Kalla'}, {'name': 'Kwon Chun'}, {'name': 'Venice Du Pont'}, {'name': 'Darius Babusis'}, {'name': 'Jennifer Tang'}, {'name': 'Eisuke Murakami'}, {'name': 'Raju Subramanian'}, {'name': 'Kimberly T Barrett'}, {'name': 'Blake J. Bleier'}, {'name': 'Roy Bannister'}, {'name': 'Joy Y. Feng'}, {'name': 'John P. Bilello'}, {'name': 'Tomas Cihlar'}, {'name': 'Richard L. Mackman'}, {'name': 'Stephanie A. Montgomery'}, {'name': 'Ralph S. Baric'}, {'name': 'Timothy P. Sheahan'}]\", 'abstract': '<p>The COVID-19 pandemic remains uncontrolled despite the rapid rollout of safe and effective SARS-CoV-2 vaccines, underscoring the need to develop highly effective antivirals. In the setting of waning immunity from infection and vaccination, breakthrough infections are becoming increasingly common and treatment options remain limited. Additionally, the emergence of SARS-CoV-2 variants of concern with their potential to escape therapeutic monoclonal antibodies emphasizes the need to develop second-generation oral antivirals targeting highly conserved viral proteins that can be rapidly deployed to outpatients. Here, we demonstrate the in vitro antiviral activity and in vivo therapeutic efficacy of GS-621763, an orally bioavailable prodrug of GS-441524, the parental nucleoside of remdesivir, which targets the highly conserved RNA-dependent RNA polymerase. GS-621763 exhibited significant antiviral activity in lung cell lines and two different human primary lung cell culture systems. The dose-proportional pharmacokinetic profile observed after oral administration of GS-621763 translated to dose-dependent antiviral activity in mice infected with SARS-CoV-2. Therapeutic GS-621763 significantly reduced viral load, lung pathology, and improved pulmonary function in COVID-19 mouse model. A direct comparison of GS-621763 with molnupiravir, an oral nucleoside analog antiviral currently in human clinical trial, proved both drugs to be similarly efficacious. These data demonstrate that therapy with oral prodrugs of remdesivir can significantly improve outcomes in SARS-CoV-2 infected mice. Thus, GS-621763 supports the exploration of GS-441524 oral prodrugs for the treatment of COVID-19 in humans.</p>', 'posted': '2021-09-13'}, {'doi': '10.1101/2021.09.09.21263331', 'tweets': 34, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Second wave of the Covid-19 pandemic in Delhi, India: high seroprevalence not a deterrent?', 'authors': \"[{'name': 'Nandini Sharma'}, {'name': 'Pragya Sharma'}, {'name': 'Saurav Basu'}, {'name': 'Ritika Bakshi'}, {'name': 'Ekta Gupta'}, {'name': 'Reshu Agarwal'}, {'name': 'Kumar Dushyant'}, {'name': 'Nutan Mundeja'}, {'name': 'Zeasley Marak'}, {'name': 'Sanjay Singh'}, {'name': 'Gautam Kumar Singh'}, {'name': 'Ruchir Rustagi'}, {'name': 'S K Sarin'}]\", 'abstract': '<sec><title>Background</title><p>We report the findings of a large follow-up community-based serosurvey and correlating it with the COVID-19 test-positivity rate and the case load observed during the peak of the second wave of the Covid-19 pandemic in Delhi, India.</p></sec><sec><title>Methods</title><p>Individuals of age 5 years were recruited from 274 wards of the state (population  19.6 million) during January 11 to January 22 2021. A total of 100 participants each were included from all the wards for a net sample size of 28,000. A multi-stage sampling technique was applied for selection of participants for the household serosurvey. Anti SARS CoV-2 IgG antibodies were detected by using the VITROS assay (90% Sn, 100% Sp).</p></sec><sec><title>Results</title><p>Antibody positivity was observed in 14,298 (50.76%) of the 28,169 samples. The age, sex and district population weighted seroprevalence of the IgG SARS-CoV-2 was 50.52% (95% C.I. 49.94-51.10) and after adjustment for assay characteristics was 56.13% (95% C.I. 55.49-56.77). On adjusted analysis, participants aged 50 years, of female gender, housewives, having ever lived in containment zones, urban slum dwellers, and diabetes or hypertensive patients had significantly higher odds of SARS-CoV-2 antibody positivity.</p><p>The peak infection rate and the test positivity rate since October 2020 were initially observed in mid-November 2020 with a subsequent steep declining trend, followed by a period of persistently low case burden lasting until the first week of March 2021. This was followed by a steady increase followed by an exponential surge in infections from April 2021 onwards culminating in the second wave of the pandemic.</p></sec><sec><title>Conclusions</title><p>The presence of infection induced immunity from SARS-CoV-2 even in more than one in two people can be ineffective in protecting the population.</p></sec>', 'posted': '2021-09-13'}, {'doi': '10.1101/2021.09.13.460168', 'tweets': 32, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Random Forest Factorization Reveals Latent Structure in Single Cell RNA Sequencing Data', 'authors': \"[{'name': 'Boris M. Brenerman'}, {'name': 'Benjamin D. Shapiro'}, {'name': 'Michael C. Schatz'}, {'name': 'Alexis Battle'}]\", 'abstract': '<p>Single-cell RNA sequencing data contain patterns of correlation that are poorly captured by techniques that rely on linear estimation or assumptions of Gaussian behavior. We apply random forest regression to scRNAseq data from mouse brains, which identifies the co-regulation of genes within specific cellular contexts. By analyzing the estimators of the random forest, we identify several novel candidate gene regulatory networks and compare these networks in aged and young mice. We demonstrate that cell populations have cell-type specific phenotypes of aging that are not detected by other methods, including the collapse of differentiating oligodendrocytes but not precursors or mature oligodendrocytes.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.01.21262607', 'tweets': 31, 'archive': 'medRxiv', 'subject-area': 'Health Informatics', 'covid': False, 'title': 'ADataViewer: Exploring Semantically Harmonized Alzheimers Disease Cohort Datasets', 'authors': \"[{'name': 'Yasamin Salimi'}, {'name': 'Daniel Domingo-Fernndz'}, {'name': 'Carlos Bobis-lvarez'}, {'name': 'Martin Hofmann-Apitius'}, {'name': 'Colin Birkenbihl'}, {'name': ' '}, {'name': ' '}, {'name': ' '}, {'name': ' '}, {'name': ' '}]\", 'abstract': '<sec><title>INTRODUCTION</title><p>Currently, AD cohort datasets are difficult to find, lack across-cohort interoperability, and the content of the shared datasets often only becomes clear to third-party researchers once data access has been granted.</p></sec><sec><title>METHODS</title><p>We accessed and systematically investigated the content of 20 major AD cohort datasets on data-level. A medical professional and a data specialist manually curated and semantically harmonized the acquired datasets. We developed a platform that facilitates data exploration.</p></sec><sec><title>RESULTS</title><p>We present ADataViewer, an interactive platform that facilitates the exploration of 20 cohort datasets with respect to longitudinal follow-up, demographics, ethnoracial diversity, measured modalities, and statistical properties of individual variables. Additionally, we publish a variable mapping catalog harmonizing 1,196 variables across the 20 cohorts. The platform is available under <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://adata.scai.fraunhofer.de/\">https://adata.scai.fraunhofer.de/</ext-link>.</p></sec><sec><title>DISCUSSION</title><p>ADataViewer supports robust data-driven research by transparently displaying cohort dataset content and suggesting datasets suited for discovery and validation studies based on selected variables of interest.</p></sec>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.13.21263499', 'tweets': 30, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': False, 'title': 'The natural history of TB disease-a synthesis of data to quantify progression and regression across the spectrum', 'authors': \"[{'name': 'Alexandra S Richards'}, {'name': 'Bianca Sossen'}, {'name': 'Jon C Emery'}, {'name': 'Katherine C Horton'}, {'name': 'Torben Heinsohn'}, {'name': 'Beatrice Frascella'}, {'name': 'Federica Balzarini'}, {'name': 'Aurea Oradini-Alacreu'}, {'name': 'Brit Hacker'}, {'name': 'Anna Odone'}, {'name': 'Nicky McCreesh'}, {'name': 'Alison D Grant'}, {'name': 'Katharina Kranzer'}, {'name': 'Frank Cobelens'}, {'name': 'Hanif Esmail'}, {'name': 'Rein MGJ Houben'}]\", 'abstract': '<sec><title>Background</title><p>Prevalence surveys have found a substantial burden of subclinical (asymptomatic but infectious) TB, from which individuals can progress, regress or even persist in a chronic disease state. We aimed to quantify these pathways across the spectrum of TB disease.</p></sec><sec><title>Methods</title><p>We created deterministic framework of TB disease with progression and regression between three states of pulmonary TB disease: minimal (non-infectious), subclinical, and clinical (symptomatic and infectious) disease. We estimated ranges for each parameter by considering all data from a systematic review in a Bayesian framework, enabling quantitative estimation of TB disease pathways.</p></sec><sec><title>Findings</title><p>Twenty-four studies contributed data from 6030 individuals. Results suggested that, after five years, 24.7%(95% uncertainty interval, UI, 21.3%-28.6%) of individuals with prevalent subclinical disease at baseline had either progressed to clinical disease or died from TB, whereas 16.1%(95%UI, 13.8%-18.5%) had recovered after regressing to minimal disease. Over the course of five years 30% (95%UI, 27.2%-32.6%) of the subclinial cohort never developed symptoms. For those with clinical disease at baseline, 39%(95%UI, 35.8%-41.9%) and 10.3%(95%UI, 8.5%-12.4%) had died or recovered from TB, with the remainder in, or undulating between, the three disease states. The ten-year mortality of people with untreated prevalent infectious disease was 38%.</p></sec><sec><title>Interpretation</title><p>Our results show that for people with subclinical disease, classic clinical disease is neither inevitable nor an irreversible outcome. As such, reliance on symptom-based screening means a large proportion of people with infectious disease may never be detected.</p></sec><sec><title>Funding</title><p>TB Modelling and Analysis Consortium and European Research Council</p></sec><sec><title>Research in Context</title><sec><title>Evidence before this study</title><p>In recent years the existence of a spectrum of TB disease has been re-accepted. The classic paradigm of disease is one active state of symptomatic presentation with bacteriologically positive sputum, now referred to as clinical disease. Within the spectrum, a subclinical phase (where people do not report symptoms but have bacteriologically positive sputum) has been widely accepted, due to prevalence surveys using chest radiography screening in addition to symptom screening. On average these prevalence surveys have found around 50% of people with prevalent infectious TB had subclinical disease. There is also another state of minimal disease, or non-infectious disease, that is the earliest point on the disease spectrum after progression from infection. The likelihood or speed of natural progression, regression, or persistence of individuals across this spectrum remains unknown. As a consequence, the ability to accurately predict the impact of interventions has been limited. As individuals with bacteriologically-positive TB now receive treatment, contemporary data to inform the required transitions is highly limited. However, a large number of cohorts of patients were described in the pre-chemotherapy era. Until now, these data have not been synthesised to inform parameters to describe the natural history of TB disease.</p></sec><sec><title>Added value of this study</title><p>We synthesised data from historical and contemporary literature to explore the expected trajectories of individuals across the spectrum of TB disease. We considered a cohort of people with prevalent bacteriologically positive disease, with a 50/50 split of people with subclinical and clinical disease at baseline. We found that within five years, 13.3% of people recover from TB, with no chance of progressing to active disease without reinfection. However, we also find that 26.3% are still spending time infectious at the end of the five years. Our estimates for 10 year mortality and duration of symptoms before treatment aligned with the known and accepted values.</p><p>We also show that regression from subclinical disease results in a large reservoir of people with minimal disease, from which they can permanently recover, but can also progress again to subclinical disease. The undulating pathways that lead to regression and progression mean that 30% (27.2%-32.6%) of individuals with prevalent subclinical disease do not experience symptoms over the course of five years. This shows that clinical disease is neither a rapid, nor inevitable outcome of subclinical disease.</p></sec><sec><title>Implications of the available evidence</title><p>With these data-driven estimates of parameters, informed projections of the relative value of addressing minimal, subclinical, or clinical disease can now be provided. Given the known reservoir of prevalent subclinical disease and its contribution to transmission, efforts to diagnose and treat people with earlier stages of TB are likely to have a larger impact than strategies targeting clinical disease, particularly on individuals who never would have progressed to clinical disease.</p></sec></sec>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.15.460550', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Intact reading ability in spite of a spatially distributed visual word form area in an individual born without the left superior temporal lobe', 'authors': \"[{'name': 'Jin Li'}, {'name': 'Evelina Fedorenko'}, {'name': 'Zeynep Saygin'}]\", 'abstract': '<p>The visual word form area (VWFA) is an experience-dependent brain region in the left ventral temporal cortex of literate adults that responds selectively to visual words. Why does it emerge in this stereotyped location? Past research has shown that the VWFA is preferentially connected to the left-lateralized frontotemporal language network. However, it remains unclear whether the presence of a typical language network and its connections with ventral temporal cortex (VTC) are critical for the VWFAs emergence, and whether alternative functional architectures may support reading ability. We explored these questions in an individual (EG) born without the left superior temporal lobe but exhibiting normal reading ability. Using fMRI, we recorded brain activation to visual words, objects, faces, and scrambled words in EG and neurotypical controls. We did not observe word selectivity either in EGs right homotope of the VWFA (rVWFA)the most expected location given that EGs language network is right-lateralizedor in her spared left VWFA (lVWFA), in the presence of typical face selectivity in both the right and left fusiform face area (rFFA, lFFA). Interestingly, multivariate pattern analyses revealed voxels in EGs rVWFA and lVWFA that showed 1) higher within- than between-category correlations for words (e.g., Words-Words&gt;Words-Faces), and 2) higher within-category correlations for words than other categories (e.g., Words-Words&gt;Faces-Faces). These results suggest that a typical left-hemisphere language network may be necessary for the emergence of focal word selectivity within ventral temporal cortex, and that orthographic processing may depend on a distributed neural code, which appears capable of supporting reading ability.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.15.460516', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Long-term Motor Learning in the Wild with High Volume Video Game Data', 'authors': \"[{'name': 'Jennifer B Listman'}, {'name': 'Jonathan Tsay'}, {'name': 'Hyosub E Kim'}, {'name': 'Wayne E Mackey'}, {'name': 'David J Heeger'}]\", 'abstract': '<p>Motor learning occurs over long periods of practice during which motor acuity (the ability to execute actions more accurately, precisely, and within a shorter amount of time) improves. Laboratory-based motor learning studies are typically limited to a small number of participants and a time frame of minutes to several hours per participant. Thus, there is a need to assess the generalizability of theories and findings from lab-based motor learning studies on much larger samples across longer time scales. In addition, laboratory-based studies of motor learning use relatively simple motor tasks which participants are unlikely to be intrinsically motivated to learn, limiting the interpretation of their findings in more ecologically valid settings. We studied the acquisition and longitudinal refinement of a complex sensorimotor skill embodied in a first-person shooter video game scenario, with a large sample size (N = 7174 participants, 682,564 repeats of the 60 sec game) over a period of months. Participants voluntarily practiced the gaming scenario for as much as several hours per day up to 100 days. We found improvement in performance accuracy (quantified as hit rate) was modest over time but motor acuity (quantified as hits per second) improved considerably, with 40-60% retention from one day to the next. We observed steady improvements in motor acuity across multiple days of video game practice, unlike most motor learning tasks studied in the lab that hit a performance ceiling rather quickly. Learning rate was a nonlinear function of baseline performance level, amount of daily practice, and to a lesser extent, number of days between practice sessions. In addition, we found that the benefit of additional practice on any given day was non-monotonic; the greatest improvements in motor acuity were evident with about an hour of practice and 90% of the learning benefit was achieved by practicing 30 minutes per day. Taken together, these results provide a proof-of-concept in studying motor skill acquisition outside the confines of the traditional laboratory and provide new insights into how a complex motor skill is acquired in an ecologically valid setting and refined across much longer time scales than typically explored.</p>', 'posted': '2021-09-17'}, {'doi': '10.1101/2021.09.14.460331', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Defective endosome-TGN retrograde transport promotes NLRP3 inflammasome activation', 'authors': \"[{'name': 'Zhirong Zhang'}, {'name': 'Li Ran'}, {'name': 'Rossella Venditti'}, {'name': 'Zengzhen Liu'}, {'name': 'Annette Schrmann'}, {'name': 'Juan S. Bonifacino'}, {'name': 'Maria Antonietta De Matteis'}, {'name': 'Romeo Ricci'}]\", 'abstract': '<p>Inflammasome complexes are pivotal in the innate immune response to pathogens and other danger signals<sup>14</sup>. The NLRP3 inflammasome is activated in response to a broad variety of cellular stressors. Most of the stimuli act in a potassium efflux-dependent manner but a primary and converging sensing mechanism by the NLRP3 receptor initiating inflammasome assembly remains ill-defined. Here we show that NLRP3 activators disrupt endosome-TGN retrograde transport (ETRT) and lead to localization of NLRP3 to endosomal vesicles. Genetic and pharmacologic perturbation of ETRT leads to accumulation of phosphoinositol-4-phosphate (PI4P) in endosomes to which NLRP3 is recruited. Disruption of ETRT potentiates NLRP3 inflammasome activation in murine and human macrophages <italic>in vitro</italic>. Mice with defects in ETRT in the myeloid compartment are more susceptible to LPS-induced sepsis showing enhanced mortality and IL-1 serum levels as compared to control animals. Our study thus uncovers that changes in endocytic trafficking mediate NLRP3-dependent inflammatory responses.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.14.460408', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': True, 'title': 'Synthetic lethality-based prediction of anti-SARS-CoV-2 targets', 'authors': \"[{'name': 'Lipika R. Pal'}, {'name': 'Kuoyuan Cheng'}, {'name': 'Nishanth Ulhas Nair'}, {'name': 'Laura Martin-Sancho'}, {'name': 'Sanju Sinha'}, {'name': 'Yuan Pu'}, {'name': 'Laura Riva'}, {'name': 'Xin Yin'}, {'name': 'Fiorella Schischlik'}, {'name': 'Joo Sang Lee'}, {'name': 'Sumit K. Chanda'}, {'name': 'Eytan Ruppin'}]\", 'abstract': '<p>Novel strategies are needed to identify drug targets and treatments for the COVID-19 pandemic. The altered gene expression of virus-infected host cells provides an opportunity to specifically inhibit viral propagation via targeting the synthetic lethal (SL) partners of such altered host genes. Pursuing this antiviral strategy, here we comprehensively analyzed multiple <italic>in vitro</italic> and <italic>in vivo</italic> bulk and single-cell RNA-sequencing datasets of SARS-CoV-2 infection to predict clinically relevant candidate antiviral targets that are SL with altered host genes. The predicted SL-based targets are highly enriched for infected cell inhibiting genes reported in four SARS-CoV-2 CRISPR-Cas9 genome-wide genetic screens. Integrating our predictions with the results of these screens, we further selected a focused subset of 26 genes that we experimentally tested in a targeted siRNA screen using human Caco-2 cells. Notably, as predicted, knocking down these targets reduced viral replication and cell viability only under the infected condition without harming non-infected cells. Our results are made publicly available, to facilitate their in vivo testing and further validation.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.14.460206', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Network visualisation of synthetic biology designs', 'authors': \"[{'name': 'Matthew Crowther'}, {'name': 'Anil Wipat'}, {'name': 'ngel Goi-Moreno'}]\", 'abstract': '<label>1</label><title>ABSTRACT</title><p>Visualising the complex information captured by synthetic biology designs is still a major challenge. The popular glyph approach where each genetic part is displayed on a linear sequence allows researchers to generate diagrams and visualise abstract designs [2], but only represents a single, static representation that results in visualisation that is not specific to the requirements of a user resulting in a one-size-fits-all visualisation. We developed a network visualisation technique that automatically turns all design information into a graph, displaying otherwise hidden data. The structure of the resulting graphs can be dynamically adjusted according to specific visualisation requirements, such as highlighting proteins, interactions or hierarchy. Since biological systems have an inherent affinity with network visualization [6], we advocate for adopting this approach to standardise and automate the representation of complex information.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.14.460283', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Multiplexed profiling of kinase interactomes quantifies cellular network plasticity', 'authors': \"[{'name': 'Martin Golkowski'}, {'name': 'Andrea Lius'}, {'name': 'Tanmay Sapre'}, {'name': 'Ho-Tak Lau'}, {'name': 'Taylor Moreno'}, {'name': 'Dustin J. Maly'}, {'name': 'Shao-En Ong'}]\", 'abstract': '<p>Cells utilize protein-protein interaction (PPI) networks to receive, transduce, and respond to stimuli. Interaction network rewiring drives devastating diseases like cancers, making PPIs attractive targets for pharmacological intervention. Kinases are druggable nodes in PPI networks but high-throughput proteomics approaches to quantify disease-associated kinome PPI rewiring are lacking. We introduce kinobead competition and correlation analysis (Ki-CCA), a chemoproteomics approach to simultaneously map hundreds of endogenous kinase PPIs. We identified 2,305 PPIs of 300 kinases across 18 diverse cancer lines, quantifying the high plasticity of interaction networks between cancer types, signaling, and phenotypic states; this database of dynamic kinome PPIs provides deep insights into cancer cell signaling. We discovered an AAK1 complex promoting epithelial-mesenchymal transition and drug resistance, and depleting its components sensitized cells to targeted therapy. Ki-CCA enables rapid and highly multiplexed mapping of kinome PPIs in native cell and tissue lysates, without epitope tagged baits, protein labeling, or antibodies.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.11.21263146', 'tweets': 14, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': False, 'title': 'Causal effect of C-reactive protein and vitamin D on human cerebral anatomy observed among genetically correlated biomarkers in blood', 'authors': \"[{'name': 'Dylan J. Kiltschewskij'}, {'name': 'William R. Reay'}, {'name': 'Murray J. Cairns'}]\", 'abstract': '<p>Psychiatric disorders such as schizophrenia are commonly associated with structural brain alterations affecting the cortex, which frequently vary with clinically relevant factors including antipsychotic treatment, duration of illness and age of onset. While the underlying variables mediating these structural changes are poorly understood, recent genetic evidence suggests circulating metabolites and other biochemical traits play a causal role in a number of psychiatric disorders which could be mediated by changes in the cerebral cortex. In the current study, we leveraged publicly available genome-wide association study (GWAS) data to explore shared genetic architecture and evidence for causal relationships between a panel of 50 biochemical traits and measures of cortical thickness and surface area at both the global and regional levels. Linkage disequilibrium score regression identified a total of 20 significant and 156 suggestive genetically correlated biochemical-cortical trait pairings, of which six exhibited strong evidence for causality in a latent causal variable model. Interestingly, a negative causal relationship was identified between a unit increase in serum C-reactive protein levels and thickness of the lingual and lateral occipital regions that was also supported by Mendelian randomisation, while circulating vitamin D (25-hydroxyvitamin D) levels exhibited a positive causal effect on temporal pole thickness. Taken together, our findings suggest a subset of biochemical traits exhibit shared genetic architecture and potentially causal relationships with cortical thickness in functionally distinct regions, which may contribute to alteration of cortical structure in psychiatric disorders.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.14.460226', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Enhanced Specificity Mutations Perturb Allosteric Signaling in CRISPR-Cas9', 'authors': \"[{'name': 'ukasz Nierzwicki'}, {'name': 'Kyle W. East'}, {'name': 'Uriel N. Morzan'}, {'name': 'Pablo R. Arantes'}, {'name': 'Victor S. Batista'}, {'name': 'George P. Lisi'}, {'name': 'Giulia Palermo'}]\", 'abstract': '<p>CRISPR-Cas9 is a molecular tool with transformative genome editing capabilities. At the molecular level, an intricate allosteric signaling is critical for DNA cleavage, but its role in the specificity enhancement of the Cas9 endonuclease is poorly understood. Here, solution NMR is combined with multi-microsecond molecular dynamics and graph theory-derived models to probe the allosteric role of key enhancement specificity mutations. We show that the mutations responsible for increasing the specificity of Cas9 alter the allosteric structure of the catalytic HNH domain, impacting the signal transmission from the DNA recognition region to the catalytic sites for cleavage. Specifically, the K855A mutation strongly disrupts the HNH domain allosteric structure, exerting the highest perturbation on the signaling transfer, while K810A and K848A result in more moderate effects on the allosteric intercommunication. This differential perturbation of the allosteric signaling reflects the different capabilities of the single mutants to increase Cas9 specificity, with the mutation achieving the highest specificity also strongly perturbing the signaling transfer. These outcomes reveal that the allosteric regulation is critical for the specificity enhancement of the Cas9 enzyme, and are valuable to harness the signaling network to improve the systems specificity.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.13.460076', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Elicitation of potent SARS-CoV-2 neutralizing antibody responses through immunization using a versatile adenovirus-inspired multimerization platform', 'authors': \"[{'name': 'C Chevillard'}, {'name': 'A Amen'}, {'name': 'S Besson'}, {'name': 'D Hannani'}, {'name': 'I Bally'}, {'name': 'V Dettling'}, {'name': 'E Gout'}, {'name': 'CJ Moreau'}, {'name': 'M Buisson'}, {'name': 'S Gallet'}, {'name': 'D Fenel'}, {'name': 'E Vassal-Stermann'}, {'name': 'G Schoehn'}, {'name': 'P Poignard'}, {'name': 'MC Dagher'}, {'name': 'P Fender'}]\", 'abstract': '<p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic has shown that vaccine preparedness is critical to anticipate a fast response to emergent pathogens with high infectivity. To rapidly reach herd immunity, an affordable, easy to store and versatile vaccine platform is thus desirable. We previously designed a non-infectious adenovirus-inspired nanoparticle (ADDomer), and in the present work, we efficiently decorated this original vaccine platform with glycosylated receptor binding domain (RBD) of SARS-CoV-2. Cryo-Electron Microscopy structure revealed that up to 60 copies of this antigenic domain were bound on a single ADDomer particle with the symmetrical arrangements of a dodecahedron. Mouse immunization with the RBD decorated particles showed as early as the first immunization a significant anti-coronavirus humoral response, which was boosted after a second immunization. Neutralization assays with spike pseudo-typed-virus demonstrated the elicitation of strong neutralization titers. Remarkably, the existence of pre-existing immunity against adenoviral-derived particles enhanced the humoral response against SARS-CoV-2. This plug and play vaccine platform revisits the way of using adenovirus to combat emergent pathogens while potentially taking advantage of the adenovirus pre-immunity.</p>', 'posted': '2021-09-13'}, {'doi': '10.1101/2021.09.15.460484', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Hippocampus maintains a flexible and coherent map under reward flavor-landmark cue conflict.', 'authors': \"[{'name': 'Indrajith R Nair'}, {'name': 'Dipanjan Roy'}]\", 'abstract': '<p>Animals predominantly use salient visual cues (landmarks) for efficient navigation over other sensory modalities. When the relative position of the visual cues is altered, the hippocampal population exhibits heterogeneous responses and constructs context-specific spatial maps. Another critical factor that can strongly modulate spatial representation is the presence of reward. Reward features can drive behavior and are known to bias spatial attention. However, it is unclear whether reward flavors are used for spatial reference in the presence of distal cues and how the hippocampus population dynamics changes when the association between reward flavors and distal cues is altered. We investigated these questions by recording place cells from the CA1 while the rats ran in an environment with the conflicting association between reward flavors and distal cues. We report that the hippocampal place cells coherently and dynamically bind to reward flavors or distal cues across sessions, but not simultaneously suggesting the use of a single spatial map. We found that place cells maintained their spatial offset in the cue conflict conditions, thus showing a robust spatial coupling featuring an attractor-like property in the CA1. When the textures were added on the track, the coherency of the CA1 is degraded, as the hippocampus showed a heterogeneous response and weak spatial coupling of co-recorded cells suggesting a break away from the attractor network. These results indicate that reward flavors alone may be used for spatial reference but may not cause sufficient input difference to create context-specific spatial maps in the CA1.</p>', 'posted': '2021-09-17'}, {'doi': '10.1101/2021.09.14.460211', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': True, 'title': 'Mitoxantrone dihydrochloride, an FDA approved drug, binds with SARS-CoV-2 NSP1 C-terminal', 'authors': \"[{'name': 'Prateek Kumar'}, {'name': 'Taniya Bhardwaj'}, {'name': 'Rajanish Giri'}]\", 'abstract': '<p>One of the major virulence factors of SARS-CoV-2, NSP1, is a vital drug target due to its role in host immune evasion through multiple pathways. NSP1 protein is associated with inhibiting host mRNA translation by binding to the small subunit of ribosome through its C-terminal region. Previously, we have shown the structural dynamics of NSP1 C-terminal region (NSP1-CTR) in different physiological environments. So, it would be very interesting to investigate the druggable compounds that could bind with NSP1-CTR. Here, in this article, we have performed the different spectroscopic technique-based binding assays of an anticancer drug Mitoxantrone dihydrochloride (MTX) against the NSP1-CTR. We have also performed molecular docking followed by computational simulations with two different forcefields up to one microsecond. Overall, our results have suggested good binding between NSP1-CTR and MTX and may have implications in developing therapeutic strategies targeting NSP1 protein of SARS-CoV-2.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.15.460445', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Physical geography, isolation by distance and environmental variables shape genomic variation of wild barley (Hordeum vulgare L. ssp. spontaneum) in the Southern Levant', 'authors': \"[{'name': 'Che-Wei Chang'}, {'name': 'Eyal Fridman'}, {'name': 'Martin Mascher'}, {'name': 'Axel Himmelbach'}, {'name': 'Karl Schmid'}]\", 'abstract': '<p>Determining the extent of genetic variation that reflects local adaptation in crop wild relatives is of interest to discovering useful genetic diversity for plant breeding. We investigated the association of genomic variation with geographical and environmental factors in wild barley (<italic>Hordeum vulgare L. ssp. spontaneum</italic>) populations of the Southern Levant using genotyping-by-sequencing (GBS) of 244 accessions of the Barley1K+ collection. Inference of population structure resulted in four genetic clusters that corresponded to eco-geographical habitats and a significant association of lower gene flow rates with geographical barriers, e.g. the Judaean Mountains and the Sea of Galilee. Redundancy analysis (RDA) revealed that spatial autocorrelation explained 45% and environmental variables explained 15% of total genomic variation. Only 4.5% of genomic variation was exclusively attributed to environmental variation if the component confounded with spatial autocorrelation was excluded. A synthetic environmental variable combining latitude, solar radiation, and accumulated precipitation explained the highest proportion of genomic variation (3.9%). After correcting for population structure, soil water capacity was the most important environmental variable explaining 1.18% of genomic variation. Genome scans with outlier analysis and genome-environment association studies were conducted to identify signatures of adaptation. RDA and outlier methods jointly detected selection signatures in the pericentromeric regions of chromosome 3H, 4H, and 5H, but they mostly disappeared after correction for population structure. In conclusion, adaptation to the highly diverse environments of the Southern Levant over short geographical ranges has a small effect on the genomic diversity of wild barley highlighting the importance of non-selective forces in genetic differentiation.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.14.460110', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Synthetic amyloid beta does not induce a robust transcriptional response in innate immune cell culture systems', 'authors': \"[{'name': 'I.Y. Quiroga'}, {'name': 'A.E. Cruikshank'}, {'name': 'K. S. M. Reed'}, {'name': 'M.L. Bond'}, {'name': 'B.A. Evangelista'}, {'name': 'J.H. Tseng'}, {'name': 'J. V. Ragusa'}, {'name': 'R. B. Meeker'}, {'name': 'H. Won'}, {'name': 'S. Cohen'}, {'name': 'T.J. Cohen'}, {'name': 'D.H. Phanstiel'}]\", 'abstract': '<p>Alzheimers disease (AD) is a progressive neurodegenerative disease that impacts nearly 400 million people worldwide. The accumulation of amyloid beta (A) in the brain has historically been associated with AD, and recent evidence suggests that neuroinflammation plays a central role in its origin and progression. These observations have given rise to the theory that A is the primary trigger of AD, and induces proinflammatory activation of immune brain cells (i.e. microglia), which culminates in neuronal damage and cognitive decline. In order to test this hypothesis, many in vitro systems have been established to study A-mediated activation of innate immune cells. Nevertheless, the transcriptional resemblance of these models to the microglia in the AD brain has never been comprehensively studied on a genome-wide scale. To address this, we used bulk RNA-seq to assess the transcriptional differences between in vitro cell types used to model neuroinflammation in AD, including several established, primary and iPSC-derived immune cell lines (macrophages, microglia and astrocytes) and their similarities to primary cells in the AD brain. We then analyzed the transcriptional response of these innate immune cells to synthetic A. We found that human induced pluripotent stem cell (hIPSC)-derived microglia (IMGL) are the in vitro cell model that best resembles primary microglia. Surprisingly, synthetic A does not trigger a robust transcriptional response in any of the cellular models analyzed, despite testing a wide variety of A formulations, concentrations, and treatment conditions. Finally, we found that bacterial LPS and INF activate microglia and induce transcriptional changes similar to those observed in disease associated microglia present in the AD brain, suggesting the potential suitability of this model to study AD-related neuroinflammation.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.15.460093', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Structural Basis of Substrate Recognition by the Mitochondrial ADP/ATP Transporter', 'authors': \"[{'name': 'Shihao Yao'}, {'name': 'Qiuzi Yi'}, {'name': 'Boyuan Ma'}, {'name': 'Xiaoting Mao'}, {'name': 'Ye Chen'}, {'name': 'Min-Xin Guan'}, {'name': 'Xiaohui Cang'}]\", 'abstract': '<p>Specific import of ADP and export of ATP by ADP/ATP carrier (AAC) across the inner mitochondrial membrane are crucial for sustainable energy supply in all eukaryotes. However, mechanism for highly specific substrate recognition in the dynamic transport process remains largely elusive. Here, unguided MD simulations of 22 microseconds in total reveal that AAC in ground c-state uses the second basic patch (K91K95R187), tyrosine ladder (Y186Y190Y194), F191 and N115 in the upper region of the cavity to specifically recognize ADP and confer selectivity for ADP over ATP. Mutations of these residues in yeast AAC2 reduce ADP transport across the <italic>L. lactis</italic> membrane and induce defects in OXPHOS and ATP production in yeast. Sequence analyses also suggest that AAC and other adenine nucleotide transporters use the upper region of the cavity, rather than the central binding site to discriminate their substrates. Identification of the new site unveils the unusually high substrate specificity of AAC, and together with central binding site support early biochemical findings about existence of two substrate binding sites. Our results imply that using different sites for substrate recognition and conformational transition could be a smart strategy for transporters to cope with substrate recognition problem in the highly dynamic transport process.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.15.459215', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': True, 'title': 'Signatures of adaptive evolution during human to mink SARS CoV2 cross-species transmission inform estimates of the COVID19 pandemic timing', 'authors': \"[{'name': 'Jui-Hung Tai'}, {'name': 'Shu-Miaw Chaw'}, {'name': 'Hsiao-Yu Sun'}, {'name': 'Yi-Cheng Tseng'}, {'name': 'Guanghao Li'}, {'name': 'Sui-Yuan Chang'}, {'name': 'Shiou-Hwei Yeh'}, {'name': 'Pei-Jer Chen'}, {'name': 'Hurng-Yi Wang'}]\", 'abstract': '<p>One of the unique features of SARS-CoV-2 is that it mainly evolved neutrally or under purifying selection during the early pandemic. This contrasts with the preceding epidemics of the closely related SARS-CoV and MERS-CoV, both of which evolved adaptively. It is possible that the SARS-CoV-2 exhibits a unique or adaptive feature which deviates from other coronaviruses. Alternatively, the virus may have been cryptically circulating in humans for a sufficient time to have acquired adaptive changes for efficient transmission before the onset of the current pandemic. In order to test the above scenarios, we analyzed the SARS-CoV-2 sequences from minks (<italic>Neovision vision</italic>) and parenteral human strains. In the early phase of the mink epidemic (April to May 2020), nonsynonymous to synonymous mutation ratios per site within the spike protein was 2.93, indicating a selection process favoring adaptive amino acid changes. In addition, mutations within this protein concentrated within its receptor binding domain and receptor binding motif. Positive selection also left a trace on linked neutral variation. An excess of high frequency derived variants produced by genetic hitchhiking was found during middle (June to July 2020) and early late (August to September 2020) phases of the mink epidemic, but quickly diminished in October and November 2020. Strong positive selection found in SARS-CoV-2 from minks implies that the virus may be not unique in super-adapting to a wide range of new hosts. The mink study suggests that SARS-CoV-2 already went through adaptive evolution in humans, and likely been circulating in humans at least six months before the first case found in Wuhan, China. We also discuss circumstances under which the virus can be well-adapted to its host but fail to induce an outbreak.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.13.459529', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Germline Missense Variants in CDC20 Result in Aberrant Mitotic Progression and Familial Cancer', 'authors': \"[{'name': 'Owen J. Chen'}, {'name': 'Ester Castellsagu'}, {'name': 'Mohamed Moustafa-Kamal'}, {'name': 'Javad Nadaf'}, {'name': 'Barbara Rivera'}, {'name': 'Somayyeh Fahiminiya'}, {'name': 'Yilin Wang'}, {'name': 'Isabelle Gamache'}, {'name': 'Caterina Pacifico'}, {'name': 'Lai Jiang'}, {'name': 'Jian Carrot-Zhang'}, {'name': 'Leora Witkowski'}, {'name': 'Albert M. Berghuis'}, {'name': 'Stefan Schoenberger'}, {'name': 'Dominik Schneider'}, {'name': 'Susanne Bens'}, {'name': 'Reiner Siebert'}, {'name': 'Colin J. R. Stewart'}, {'name': 'Ziguo Zhang'}, {'name': 'William C. Chao'}, {'name': 'Celia M.T. Greenwood'}, {'name': 'David Barford'}, {'name': 'Marc Tischkowitz'}, {'name': 'Jacek Majewski'}, {'name': 'William D. Foulkes'}, {'name': 'Jose G. Teodoro'}]\", 'abstract': '<p>CDC20 is a co-activator of the anaphase promoting complex/cyclosome (APC/C) and is essential for mitotic progression. APC/C<sup>CDC20</sup> is inhibited by the spindle assembly checkpoint (SAC), which prevents premature separation of sister chromatids and aneuploidy in daughter cells. Although overexpression of <italic>CDC20</italic> is common in many cancers, oncogenic mutations have never been identified in humans. Using whole exome sequencing, we identified heterozygous missense <italic>CDC20</italic> variants (L151R and N331K) that segregate with cancer in two families. Characterization of these mutants showed they retain APC/C activation activity but show reduced binding to BUBR1, a component of the SAC. Expression of L151R and N331K promoted mitotic slippage in HeLa cells and primary skin fibroblasts derived from carriers. CRISPR/Cas9 was used to generate mice carrying N331K. Homozygous mice carrying N331K were non-viable, however, heterozygotes displayed accelerated oncogenicity in Myc-driven cancers. These findings highlight an unappreciated role for <italic>CDC20</italic> variants as tumor promoting genes in humans.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.13.459918', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Impaired KDM2B-mediated PRC1 recruitment to chromatin causes neural stem cell senescence and ASD/ID-like behavioral deficits', 'authors': \"[{'name': 'Yuen Gao'}, {'name': 'Natalia Duque-Wilckens'}, {'name': 'Mohammad B Aljazi'}, {'name': 'Adam J Moeser'}, {'name': 'George I Mias'}, {'name': 'Alfred J Robison'}, {'name': 'Yi Zhang'}, {'name': 'Jin He'}]\", 'abstract': '<p>Autism spectrum disorder (ASD) and intellectual disability (ID) are neurodevelopmental diseases associated with various genetic mutations. Recent clinical studies report that chromosomal 12q24.31 microdeletions are associated with human ASD/ID. However, the causality and underlying mechanisms linking 12q24.31 microdeletions to ASD/ID pathogenesis remain undetermined. Here we show <italic>Kdm2b</italic>, one of the genes located in chromosomal 12q24.31, plays a critical role in maintaining neural stem cells (NSCs) in the developing mouse brain. Loss of the CxxC-ZF domain of KDM2B impairs its function in recruiting Polycomb repressive complex 1 (PRC1) to chromatin, resulting in de-repression of genes involved in cell apoptosis, cell cycle arrest, NSC premature senescence, and leading to the loss of NSC populations in the brain. Importantly, the <italic>Kdm2b</italic> mutation is sufficient to induce ASD/ID-like social and memory deficits in adult mice. Thus, our study reveals a critical role of an epigenetic factor KDM2B in normal brain development, a causality between the <italic>Kdm2b</italic> mutation and genesis of ASD/ID-like phenotypes in mice, and potential molecular mechanisms linking the function of KDM2B-PRC1 in transcriptional regulation and NSC senescence to the12q24.31 microdeletion-associated ASD/ID.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.13.460150', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Gsx2 but not Gsx1 is necessary for early forebrain patterning and long-term survival in zebrafish', 'authors': \"[{'name': 'RA Coltogirone'}, {'name': 'EI Sherfinski'}, {'name': 'ZA Dobler'}, {'name': 'SN Peterson'}, {'name': 'AR Andlinger'}, {'name': 'LC Fadel'}, {'name': 'RL Patrick'}, {'name': 'SA Bergeron'}]\", 'abstract': '<p>Central nervous system (CNS) development is regulated by regionally expressed transcription factors that impart initial cell identity, connectivity, and function to neural circuits through complex molecular genetic cascades. <italic>genomic screen homeobox 1</italic> and <italic>2</italic> (<italic>gsx1</italic> and <italic>gsx2</italic>) encode homeobox transcription factors expressed in the developing CNS in multiple vertebrates examined to date. However, we have limited knowledge of the expression of these transcription factors and the gene networks that they regulate across developing brain regions in zebrafish. The objective of this study was to comprehensively examine <italic>gsx1</italic> and <italic>gsx2</italic> expression throughout neurodevelopment and characterize <italic>gsx1</italic> and <italic>gsx2</italic> mutants to study the essential roles of these closely related transcription factors. Using RT-PCR, whole-mount <italic>in situ</italic> hybridization (WISH), and fluorescence <italic>in situ</italic> hybridization, we examine <italic>gsx1</italic> and <italic>gsx2</italic> expression from early embryonic to late larval stages. <italic>gsx1</italic> is expressed initially in the hindbrain and diencephalon and later in the optic tectum, pretectum, and cerebellar plate. Comparatively, <italic>gsx2</italic> is expressed in the early telencephalon and later in the pallium and olfactory bulb. <italic>gsx1</italic> and <italic>gsx2</italic> are regionally co-expressed in the hypothalamus, preoptic area, and hindbrain, however rarely co-localize in the same cells. To identify forebrain target genes, we utilize mutants made with Transcription activator-like effector nucleases (TALEN). <italic>gsx1</italic> mutant zebrafish exhibit stunted growth, however, they survive through adulthood and are fertile. <italic>gsx2</italic> mutant zebrafish experience swim bladder inflation failure that prevents survival past larval stage. Using WISH and RT-qPCR we demonstrate altered expression of genes including, <italic>distal-less homeobox</italic> genes and <italic>forkhead box</italic> gene <italic>foxp2</italic>. This work provides novel tools with which other target genes and functions of Gsx1 and Gsx2 can be characterized across the CNS to better understand the unique and overlapping roles of these highly conserved transcription factors.</p>', 'posted': '2021-09-13'}, {'doi': '10.1101/2021.09.14.460297', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Mitochondrial ATP production is required for endothelial cell control of vascular tone', 'authors': \"[{'name': 'Calum Wilson'}, {'name': 'Matthew D. Lee'}, {'name': 'Charlotte Buckley'}, {'name': 'Xun Zhang'}, {'name': 'John G. McCarron'}]\", 'abstract': '<p>Arteries and veins are lined by non-proliferating endothelial cells that play a critical role in regulating blood flow. Endothelial cells also regulate tissue perfusion, metabolite exchange, and thrombosis. It is thought that endothelial cells rely on ATP generated via glycolysis to fuel each of these energy-demanding processes. However, endothelial metabolism has mainly been studied in the context of proliferative cells in angiogenesis, and little is known about energy production in endothelial cells within the fully-formed vascular wall. Using intact arteries isolated from rats and mice, we show that inhibiting mitochondrial oxidative phosphorylation disrupts endothelial control of vascular tone. The role for endothelial cell energy production is independent of species, sex, or vascular bed. Basal, mechanically-activated, and agonist-evoked calcium activity in intact artery endothelial cells are each prevented by inhibiting mitochondrial ATP synthesis. This effect is mimicked by blocking the transport of pyruvate, the master fuel for mitochondrial energy production, through the mitochondrial pyruvate carrier. These data show that mitochondrial ATP is necessary for calcium-dependent, nitric oxide mediated endothelial control of vascular tone, and identifies the critical role of endothelial mitochondrial energy production in fueling perfused blood vessel function.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.14.460322', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Variance of allele balance calculated from low coverage sequencing data infers departure from a diploid state', 'authors': \"[{'name': 'Kyle Fletcher'}, {'name': 'Rongkui Han'}, {'name': 'Diederik Smilde'}, {'name': 'Richard Michelmore'}]\", 'abstract': '<sec><title>Motivation</title><p>Polyploidy and heterokaryosis are common and consequential genetic phenomena that increase the number of haplotypes in an organism and complicate whole-genome sequence analysis. Allele balance has been used to infer polyploidy and heterokaryosis in diverse organisms using read sets sequenced to greater than 50x whole-genome coverage. However, Sequencing to adequate depth is costly if applied to multiple individuals or large genomes.</p></sec><sec><title>Results</title><p>We developed VCFvariance.pl to utilize the variance of allele balance to infer polyploidy and/or heterokaryosis at low sequence coverage. This analysis requires as little as 10x whole-genome coverage and reduces the allele balance profile down to a single value, which can be used to determine if an individual has two or more haplotypes. This approach was validated on simulated, synthetic, and authentic read sets from an oomycete, fungus, and plant. The approach was deployed to ascertain the genome status of multiple isolates of <italic>Bremia lactucae</italic> and <italic>Phytophthora infestans</italic>.</p></sec><sec><title>Availability and implementation</title><p>VCFvariance.pl is a Perl script available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/kfletcher88/VCFvariance\">https://github.com/kfletcher88/VCFvariance</ext-link>.</p></sec>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.14.460311', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Elevated phosphorylation of EGFR in NSCLC due to mutations in PTPRH', 'authors': \"[{'name': 'Matthew R. Swiatnicki'}, {'name': 'Jonathan P. Rennhack'}, {'name': 'Daniel P. Hollern'}, {'name': 'Ashlee V. Perry'}, {'name': 'Rachel Kubiak'}, {'name': 'Sarai M. Riveria Riveria'}, {'name': 'Sandra OReilly'}, {'name': 'Eran R. Andrechek'}]\", 'abstract': '<p>The role of EGFR in lung cancer is well described with numerous activating mutations that result in phosphorylation and tyrosine kinase inhibitors that target EGFR. While the role of the EGFR kinase in non-small cell lung cancer (NSCLC) is appreciated, control of EGFR signaling pathways through dephosphorylation by phosphatases is not as clear. In recent work we identified mutations in Protein Tyrosine Phosphatase Receptor Type H (<italic>Ptprh</italic>, also known as SAP-1) as being associated with elevated phosphorylation of EGFR in a mouse model of breast cancer. We have examined a series of tumors from this mouse model, revealing conserved V483M <italic>Ptprh</italic> mutations within the FVB background, but a series of varied mutations in other backgrounds. Despite the varied <italic>Ptprh</italic> mutations in other background strains, matched primary and metastatic tumors largely shared mutational profiles. Profiling the downstream events of <italic>Ptprh</italic> mutant tumors revealed AKT activation, suggesting a key target of PTPRH was EGFR tyrosine 1197. Given the role of EGFR in lung cancer, we explored TCGA data which revealed that a subset of <italic>PTPRH</italic> mutant tumors shared gene expression profiles with <italic>EGFR</italic> mutant tumors, but that <italic>EGFR</italic> mutations and <italic>PTPRH</italic> mutations were mutually exclusive. Generation of a PTPRH knockout NSCLC cell line resulted in Y1197 phosphorylation of EGFR, and a rescue with expression of wild type PTPRH returned EGFR phosphorylation to parental line values while a rescue with a D986A catalytically dead mutant PTPRH did not, demonstrating that PTPRH targets EGFR. As expected with active EGFR, the knockout of PTPRH was associated with increased growth rate. Moreover, a dose response curve illustrated that two human NSCLC lines that had naturally occurring <italic>PTPRH</italic> mutations responded to EGFR tyrosine kinase inhibition. Injection of one of the NSCLC human lines into mice resulted in tumors, and Osimertinib treatment resulted in a reduction of tumor volume relative to vehicle controls. Consistent with prior literature from breast cancer, <italic>PTPRH</italic> mutation resulted in nuclear pEGFR as seen in immunohistochemistry, suggesting that there may also be a role for EGFR as a transcriptional co-factor. Other roles for PTPRH were explored through a receptor tyrosine kinase array, noting elevated phosphorylation of FGFR1. Knockout of PTPRH in NSCLC cell lines resulted in elevated phosphorylated FGFR1 relative to controls, indicating that PTPRH has a number of targets that may be aberrantly activated in NSCLC with mutations in PTPRH. Together these data suggest that mutations in PTPRH in NSCLC may result in clinically actionable alterations using existing therapies.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.16.460651', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Unlabeled salivary gland organoids have distinct Raman signatures following FGF2-induced proacinar cell differentiation', 'authors': \"[{'name': 'Kate Tubbesing'}, {'name': 'Nicholas Moskwa'}, {'name': 'Ting Chean Khoo'}, {'name': 'Deirdre A. Nelson'}, {'name': 'Anna Sharikova'}, {'name': 'Melinda Larsen'}, {'name': 'Alexander Khmaladze'}]\", 'abstract': '<p>Organoids are self-organized three-dimensional (3D) tissue cultures that model the structure and function of organs to provide insights into signaling during organ formation and have translational applications in disease modeling and assessing drug responses. Due to their heterogeneity, there is a need for non-destructive methods to identify the differentiation state, or the phenotype, of organoids. As organoids often contain complex mixtures of basement membrane and/or extracellular matrix proteins, which are often highly auto-fluorescent, it typically makes low-resolution Raman measurements a challenge. We developed Raman confocal micro-spectroscopy methods to avoid and minimize the matrix signal and define specific Raman signatures for growth factor-differentiated and non-differentiated organoids. In complex, branched salivary gland organoids derived from mouse embryonic epithelial and stromal cells embedded within the laminin-rich basement membrane matrix, Matrigel, we identified specific Raman spectral signatures for organoids in different differentiation states. We report that either comparison of spectral signatures or multivariate SVD analysis can be used to distinguish between organoids treated with FGF2, organoids treated with EGF, and non-treated controls. Raman spectral signatures can be used to non-invasively distinguish between different phenotypes in the 3D context of unlabeled organoids.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>FGF2-dependent proacinar cell differentiation in salivary organoids have unique Raman signatures detected with a novel confocal-based Raman imaging approach.</p></list-item><list-item><p>These signatures can be used in unlabeled salivary organoids to monitor proacinar cell differentiation.</p></list-item><list-item><p>Confocal-based Raman imaging may be applicable to monitoring differentiation state of other types of organoids.</p></list-item></list></sec>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.18.460647', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Diet-derived metabolites and mucus link the gut microbiome to fever after cytotoxic cancer treatment', 'authors': \"[{'name': 'Zaker Schwabkey'}, {'name': 'Diana Wiesnoski'}, {'name': 'Chia-Chi Chang'}, {'name': 'Wen-Bin Tsai'}, {'name': 'Dung Pham'}, {'name': 'Saira S Ahmed'}, {'name': 'Tomo Hayase'}, {'name': 'Miriam R Ortega Turrubiates'}, {'name': 'Rawan K El-Himri'}, {'name': 'Christopher A Sanchez'}, {'name': 'Eiko Hayase'}, {'name': 'Annette C Frenk Oquendo'}, {'name': 'Takahiko Miyama'}, {'name': 'Taylor M Halsey'}, {'name': 'Brooke E Heckel'}, {'name': 'Alexandria N Brown'}, {'name': 'Yimei Jin'}, {'name': 'Philip L Lorenzi'}, {'name': 'Marc O Warmoes'}, {'name': 'Lin Tan'}, {'name': 'Alton Swennes'}, {'name': 'Vanessa B Jensen'}, {'name': 'Christine B Peterson'}, {'name': 'Kim-Anh Do'}, {'name': 'Liangliang Zhang'}, {'name': 'Yushu Shi'}, {'name': 'Yinghong Wang'}, {'name': 'Jessica R Galloway-Pena'}, {'name': 'Pablo C Okhuysen'}, {'name': 'Carrie Daniel-MacDougall'}, {'name': 'Yusuke Shono'}, {'name': 'Marina Burgos da Silva'}, {'name': 'Jonathan Peled'}, {'name': 'Marcel RM van den Brink'}, {'name': 'Nadim J Ajami'}, {'name': 'Jennifer Wargo'}, {'name': 'Gabriela Rondon'}, {'name': 'Samer A Srour'}, {'name': 'Rohtesh S Mehta'}, {'name': 'Amin M Alousi'}, {'name': 'Elizabeth J Shpall'}, {'name': 'Richard E Champlin'}, {'name': 'Samuel A Shelburne'}, {'name': 'Jeffrey J Molldrem'}, {'name': 'Mohamed A Jamal'}, {'name': 'Jennifer L Karmouch'}, {'name': 'Robert R Jenq'}]\", 'abstract': '<p>Fever in the setting of low white blood cell counts, also known as neutropenic fever, is considered an oncologic emergency. Only some 30% of patients with neutropenic fever have an identifiable source of infection, such as a bloodstream bacterial infection of intestinal origin. Whether other nonpathogenic intestinal bacteria can contribute to neutropenic fever is not known. In a cohort of 119 patients undergoing hematopoietic cell transplantation, a treatment with a high risk for neutropenic fever, the fecal microbiome was examined at onset of neutropenia. Over the next 4 days, 63 patients (53%) developed a fever, which was associated with increased Akkermansia muciniphila, a species of intestinal commensal bacteria with mucus-degrading capabilities (p=0.006, corrected for multiple comparisons). In mouse models, two cytotoxic therapies, irradiation and melphalan, each also produced an expansion of fecal A. muciniphila as well as thinning of the colonic mucus layer. Direct irradiation of fecal bacteria, however, did not lead to expansion of A. muciniphila, suggesting an indirect effect via the host. Irradiated mice displayed reduced oral food intake, and dietary restriction of unirradiated mice was sufficient to produce an expansion of A. muciniphila and thinning of the colonic mucus layer. Treatment of diet-restricted mice with several narrow-spectrum antibiotics demonstrated that azithromycin, which depleted intestinal A. muciniphila, led to preservation of the colonic mucus layer. Diet-restricted mice developed an increase in colonic luminal pH and reductions in acetate, propionate, and butyrate. Treatment of A. muciniphila in vitro with lower pH and increased propionate led to delayed growth and prevented utilization of mucin. Dietary restriction and low propionate also produced large changes in A. muciniphila gene expression, including upregulation of L-fucose isomerase, a member of glycosyl hydrolase family 109, and a member of the Idh/MocA family of oxidoreductases, which may play roles in mucin glycan utilization. Supplementing the drinking water of diet-restricted mice with propionate led to preservation of the colonic mucus layer. In the setting of irradiation, we found that by 6 days following treatment mice developed signs of systemic infection manifesting as hypothermia. Upon examination of colon tissues, we found that irradiated mice had elevated levels of inflammatory cytokines including IL-1b, CCL2, CCL7, IL-22, CXCL1, and CXCL10. Treatment with azithromycin or propionate lessened the severity of hypothermia, preserved the mucus layer, and mitigated elevations in inflammatory cytokines in the colon. In summary, we have found that clinical neutropenic fever is associated with increased intestinal abundance of mucolytic bacteria, and further experiments in mice have identified a pathway linking impaired diet and loss of bacterial metabolites as important mediators of this process.</p>', 'posted': '2021-09-18'}, {'doi': '10.1101/2021.09.09.21263026', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'The clinically extremely vulnerable to COVID: Identification and changes in healthcare while self-isolating (shielding) during the coronavirus pandemic', 'authors': \"[{'name': 'Jessica E. Butler'}, {'name': 'Mintu Nath'}, {'name': 'Dimitra Blana'}, {'name': 'William P. Ball'}, {'name': 'Nicola Beech'}, {'name': 'Corri Black'}, {'name': 'Graham Osler'}, {'name': 'Sebastien Peytrignet'}, {'name': 'Katie Wilde'}, {'name': 'Artur Wozniak'}, {'name': 'Simon Sawhney'}]\", 'abstract': '<sec><title>Background</title><p>In March 2020, the government of Scotland identified people deemed clinically extremely vulnerable to COVID due to their pre-existing health conditions. These people were advised to strictly self-isolate (shield) at the start of the pandemic, except for necessary healthcare. We examined who was identified as clinically extremely vulnerable, how their healthcare changed during isolation, and whether this process exacerbated healthcare inequalities.</p></sec><sec><title>Methods</title><p>We linked those on the shielding register in NHS Grampian, a health authority in Scotland, to healthcare records from 2015-2020. We described the source of identification, demographics, and clinical history of the cohort. We measured changes in out-patient, in-patient, and emergency healthcare during isolation in the shielding population and compared to the general non-shielding population.</p></sec><sec><title>Results</title><p>The register included 16,092 people (3% of the population), clinically vulnerable primarily due to a respiratory disease, immunosuppression, or cancer. Among them, 42% were not identified by national healthcare record screening but added <italic>ad hoc</italic>, with these additions including more children and fewer economically-deprived.</p><p>During isolation, all forms of healthcare use decreased (25%-46%), with larger decreases in scheduled care than in emergency care. However, people shielding had better maintained scheduled care compared to the non-shielding general population: out-patient visits decreased 35% vs 49%; in-patient visits decreased 46% vs 81%. Notably, there was substantial variation in whose scheduled care was maintained during isolation: younger people and those with cancer had significantly higher visit rates, but there was no difference between sexes or socioeconomic levels.</p></sec><sec><title>Conclusions</title><p>Healthcare changed dramatically for the clinically extremely vulnerable population during the pandemic. The increased reliance on emergency care while isolating indicates that continuity of care for existing conditions was not optimal. However, compared to the general population, there was success in maintaining scheduled care, particularly in young people and those with cancer. We suggest that integrating demographic and primary care data would improve identification of the clinically vulnerable and could aid prioritising their care.</p></sec>', 'posted': '2021-09-13'}, {'doi': '10.1101/2021.09.14.460362', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Testing for heterogeneous rates of discrete character evolution on phylogenies', 'authors': \"[{'name': 'Liam J. Revell'}, {'name': 'Klaus P. Schliep'}, {'name': 'D. Luke Mahler'}, {'name': 'Travis Ingram'}]\", 'abstract': '<p>Many hypotheses in the field of phylogenetic comparative biology involve specific changes in the rate or process of trait evolution. We present a method designed to test whether the rate of evolution of a discrete character has changed in one or more clades, lineages, or time periods. This method differs from other related approaches (such as the covarion model) in that the regimes in which the rate or process is postulated to have changed are specified a priori by the user, rather than inferred from the data. Similarly, it differs from methods designed to model a correlation between two binary traits in that the regimes mapped onto the tree are fixed. We apply our method to investigate the rate of dewlap color and/or caudal vertebra number evolution in Caribbean and mainland clades of the diverse lizard genus <italic>Anolis</italic>. We find little evidence to support any difference between mainland and island evolution in either character. We also examine the statistical properties of the method more generally and show that it has acceptable type I error, parameter estimation, and power. Finally, we discuss the relationship of our method to existing models of heterogeneity in the rate of discrete character evolution on phylogenies.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.12.460001', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'DeepREAL: A Deep Learning Powered Multi-scale Modeling Framework Towards Predicting Out-of-distribution Receptor Activity of Ligand Binding', 'authors': \"[{'name': 'Tian Cai'}, {'name': 'Kyra Alyssa Abbu'}, {'name': 'Yang Liu'}, {'name': 'Lei Xie'}]\", 'abstract': '<p>Drug discovery has witnessed intensive exploration of the problem of drug-target physical interactions over two decades, however, a strong drug binding affinity to a single target often fails to translate into desired clinical outcomes. A critical knowledge gap needs to be filled for correlating drug-target interactions with phenotypic responses: predicting the receptor activities or function selectivity upon the ligand binding (i.e., agonist vs. antagonist) on a genome-scale and for novel chemicals. Two major obstacles compound the difficulty on this direction: known data of receptor activity is far too scarce to train a robust model in light of genome-scale applications, and real-world applications need to deploy a model on data from various shifted distributions. To address these challenges, we have developed an end-to-end deep learning framework, DeepREAL, for multi-scale modeling of genome-wide receptor activities of ligand binding. DeepREAL utilizes self-supervised learning on tens of millions of protein sequences and pre-trained binary interaction classification to solve the data distribution shift and data scarcity problems. Extensive benchmark studies that simulate real-world scenarios demonstrate that DeepREAL achieves state-of-the-art performance in out-of-distribution settings.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.14.460257', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Chronic sodium bromide treatment relieves autistic-like behavioral deficits in three mouse models of autism', 'authors': \"[{'name': 'Ccile Derieux'}, {'name': 'Audrey Laut'}, {'name': 'Agathe Brugoux'}, {'name': 'Dborah Jacaz'}, {'name': 'Jean-Philippe Pin'}, {'name': 'Julie Kniazeff'}, {'name': 'Julie Le Merrer'}, {'name': 'Jerome AJ Becker'}]\", 'abstract': '<p>Autism Spectrum Disorders (ASD) are neurodevelopmental disorders whose diagnosis relies on deficient social interaction and communication together with repetitive behavior. To date, no pharmacological treatment has been approved that ameliorates social behavior in patients with ASD. Based on the excitation/inhibition imbalance theory of autism, we hypothesized that bromide ions, long used as an antiepileptic medication, could relieve core symptoms of ASD. We evaluated the effects of chronic sodium bromide (NaBr) administration on autistic-like symptoms in three genetic mouse models of autism: <italic>Oprm1<sup>-/-</sup></italic> , <italic>Fmr1<sup>-/-</sup></italic> and <italic>Shank3<sup>ex13-16-/-</sup></italic> mice. We showed that chronic NaBr treatment relieved autistic-like behaviors in these three models. In <italic>Oprm1<sup>-/-</sup></italic> mice, these beneficial effects were superior to those of chronic bumetanide administration. At transcriptional level, chronic NaBr in <italic>Oprm1</italic> null mice was associated with increased expression of genes coding for chloride ions transporters, GABAA receptor subunits, oxytocin and mGlu4 receptor. Lastly, we uncovered synergistic alleviating effects of chronic NaBr and a positive allosteric modulator (PAM) of mGlu4 receptor on autistic-like behavior in <italic>Oprm1<sup>-/-</sup></italic> mice. We evidenced in heterologous cells that bromide ions behave as PAMs of mGlu4, providing a molecular mechanism for such synergy. Our data reveal the therapeutic potential of bromide ions, alone or in combination with a PAM of mGlu4 receptor, for the treatment of ASDs.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.14.460235', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Paramagnetic spin labeling of a bacterial DnaB helicase for solid-state NMR', 'authors': \"[{'name': 'Johannes Zehnder'}, {'name': 'Riccardo Cadalbert'}, {'name': 'Maxim Yulikov'}, {'name': 'Georg Knze'}, {'name': 'Thomas Wiegand'}]\", 'abstract': '<p>Labeling of biomolecules with a paramagnetic probe for nuclear magnetic resonance (NMR) spectroscopy enables determining long-range distance restraints, which are otherwise not accessible by classically used dipolar coupling-based NMR approaches. Distance restraints derived from paramagnetic relaxation enhancements (PREs) can facilitate the structure determination of large proteins and protein complexes. We herein present the site-directed labeling of the large oligomeric bacterial DnaB helicase from <italic>Helicobacter pylori</italic> with cysteine-reactive maleimide tags carrying either a nitroxide radical or a lanthanide ion. The success of the labeling reaction was followed by quantitative continuous-wave electron paramagnetic resonance (EPR) experiments performed on the nitroxide-labeled protein. PREs were extracted site-specifically from 2D and 3D solid-state NMR spectra. A good agreement with predicted PRE values, derived by computational modeling of nitroxide and Gd<sup>3+</sup> tags in the low-resolution DnaB crystal structure, was found. Comparison of experimental PREs and model-predicted spin label-nucleus distances indicated that the size of the blind sphere around the paramagnetic center, in which NMR resonances are not detected, is slightly larger for Gd<sup>3+</sup> (~14 ) than for nitroxide (~11 ) in <sup>13</sup>C-detected 2D spectra of DnaB. We also present Gd<sup>3+</sup>-Gd<sup>3+</sup> dipolar electron-electron resonance EPR experiments on DnaB supporting the conclusion that DnaB was present as a hexameric assembly.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.16.460530', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genome-wide identification of genes important for growth of Dickeya dadantii and D. dianthicola in potato (Solanum tuberosum) tubers', 'authors': \"[{'name': 'Tyler C. Helmann'}, {'name': 'Melanie J. Filiatrault'}, {'name': 'Paul V. Stodghill'}]\", 'abstract': '<p><italic>Dickeya</italic> species are causal agents of soft rot diseases in many economically important crops, including soft rot disease of potato (<italic>Solanum tuberosum</italic>). Using random barcode transposon-site sequencing (RB-TnSeq), we generated genome-wide mutant fitness profiles of <italic>Dickeya dadantii</italic> 3937, <italic>Dickeya dianthicola</italic> ME23, and <italic>Dickeya dianthicola</italic> 67-19 isolates collected after passage through several <italic>in vitro</italic> and <italic>in vivo</italic> conditions. Tubers from the potato cultivars Atlantic, Dark Red Norland, and Upstate Abundance provided highly similar conditions for bacterial growth. Using the homolog detection software PyParanoid, we matched fitness values for orthologous genes in the three bacterial strains. Direct comparison of fitness among the strains highlighted shared and variable traits important for growth. Bacterial growth in minimal medium required many metabolic traits that were also essential for competitive growth <italic>in planta</italic>, such as amino acid, carbohydrate, and nucleotide biosynthesis. Growth in tubers specifically required the pectin degradation gene <italic>kduD</italic>. Disruption in three putative DNA-binding proteins had strain-specific effects on competitive fitness in tubers. Though the Soft Rot <italic>Pectobacteriaceae</italic> can cause disease with little host specificity, it remains to be seen the extent to which strain-level variation impacts virulence.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.13.459937', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Amelioration of L-DOPA-induced dyskinesia with vitamin D3 in Parkinsonian mice model', 'authors': \"[{'name': 'Adedamola Aminat Bayo-Olugbami'}, {'name': 'Abdulrazaq Bidemi Nafiu'}, {'name': 'Abdulbasit Amin'}, {'name': 'Olalekan Michael Ogundele'}, {'name': 'Charles C. Lee'}, {'name': 'Bamidele Victor Owoyele'}]\", 'abstract': '<p>L-DOPA Induced Dyskinesia (LID) is associated with prolonged L-DOPA therapy. Vitamin-D receptor modulation improves motor-cognitive deficit in experimental LID Parkinsonism. Therefore, in this study, we investigated the mechanism underlying the anti-dyskinetic potential of Vitamin D3 (VD<sub>3</sub>). Dyskinesia was induced by chronic L-DOPA administration in 6-OHDA lesioned male C57BL6 mice. The experimental groups (Dyskinesia, Dyskinesia/VD<sub>3</sub>, and Dyskinesia/Amantadine) and controls were challenged with L-DOPA to determine the abnormal involuntary movements (AIMs) score during 14 days of VD<sub>3</sub> (30 mg/kg) or Amantadine (40 mg/kg) treatment. Global behavioral Axial, Limb &amp; Orolingual (ALO) AIMs were scored for 1 min at every 20 mins interval, over a duration of 100 mins on days 1,3,7,11 and 14 of treatment. Thereafter, brain samples were collected and processed for immunoblotting to assess striatal expression of tyrosine hydroxylase (TH), monoamine oxidase (MAO), cathecol-o-methyl transferase (COMT), dopamine decarboxylase (DDC), CD11b, BAX, P47phox, and IL-1. VD<sub>3</sub> significantly attenuated ALO AIMs only on days 11 &amp; 14, with maximal reduction of 32.7% compared with dyskinetic mice but had no effect on days 1, 3 &amp; 7, while amantadine decreased AIMs all through days 1 to 14 with maximal reduction of 64.5%. TH and MAO-B expression were not significantly different across the groups. DDC was significantly suppressed in dyskinetic mice <italic>vs</italic> control (p&lt;0.001) but remained unchanged in VD<sub>3</sub> mice <italic>vs</italic> dyskinetic mice. COMT was upregulated in the dyskinetic group <italic>vs</italic> control (p&lt;0.01) and attenuated in VD<sub>3</sub> mice (p&lt;0.05) compared to the dyskinetic group. Interestingly, VD<sub>3</sub> inhibited significantly (p&lt;0.01) oxidative stress (p47phox), apoptosis (BAX), inflammation (IL-1), and microglial activation (CD11b) in dyskinetic mice. Overall, we find that the anti-dyskinetic effects of VD<sub>3</sub> is associated with modulation of striatal oxidative stress, microglial responses, inflammation, and apoptotic signaling.</p><sec><title>Impact statement</title><p>There are evidences showing that VD<sub>3</sub> supplementation improves motor disorders, including Parkinsons disease. We hypothesized that VD<sub>3</sub> could improve LID, an abnormal involuntary movement that results from prolonged L-DOPA therapy in the management of PD. We have demonstrated the novel anti-dyskinetic effect of VD<sub>3</sub> and associated mechanistic factors in a mouse model of L-DOPA Induced Dyskinesia (LID), which identifies promising targets for dyskinesia therapy.</p></sec>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.10.21263383', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Social-economic drivers overwhelm climate in underlying the COVID-19 early growth rate', 'authors': \"[{'name': 'Zhenghua Liu'}]\", 'abstract': '<p>Identifying the drivers underlying the spatial occurrence and spreading rate of COVID-19 can provide valuable information for their preventions and controls. Here, we examine how socio-economic and climate drivers affect the early growth rates of COVID-19 in China and the other countries, the former of which have consistently stricter quarantine during early epidemic and thus are used to enquire the influences of human interventions on trainsimissions. We find that the early growth rates of COVID-19 are higher in China than the other countries, which is consistent with previous reports. The global spread is mainly driven by the socio-economic factors such as GDP per capita, human movement and population density rather than climate. Among socio-economic factors, GDP per capita is most important showing negative relationships in China, while positive in the other countries. However, the predicability of early growth rates by socio-economic and climate variables is at least 1.6 times higher in Chinas provinces than the other countries, which is further supported by metapopulation network model. These findings collectively indicate that the stochasticity of transimission processes decrease upon strict quarantine measures such as travel restrictions.</p><sec><title>Key findings</title><list list-type=\"order\"><list-item><p>GDP per capita is most important in driving the spread of COVID-19, which shows negative relationships within China, while positive in the other countries.</p></list-item><list-item><p>Socio-economic and climate factors are key in driving the early growth rate of COVID-19, while the former is more important.</p></list-item><list-item><p>Socio-economic and climate features explain more variations of early growth rates in China due to the decreased stochasticity of transimission processes upon strict quarantine measures.</p></list-item></list></sec>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.12.459767', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Drought stress increases the expression of barley defence genes with negative consequences for infesting cereal aphids', 'authors': \"[{'name': 'Daniel J. Leybourne'}, {'name': 'Tracy A Valentine'}, {'name': 'Kirsty Binnie'}, {'name': 'Anna Taylor'}, {'name': 'Alison J Karley'}, {'name': 'Jorunn IB Bos'}]\", 'abstract': '<p>Crops are exposed to myriad abiotic and biotic stressors with negative consequences. Two stressors that are expected to increase under climate change are drought and infestation with herbivorous insects, including important aphid species. Expanding our understanding of the impact drought has on the plant-aphid relationship will become increasingly important under future climate scenarios. Here we use a previously characterised plant-aphid system comprising a susceptible variety of barley, a wild relative of barley with partial-aphid resistance, and the bird cherry-oat aphid to examine the drought-plant-aphid relationship. We show that drought has a negative effect on plant physiology and aphid fitness and provide evidence to suggest that plant resistance influences aphid responses to drought stress, with the expression of aphid detoxification genes increasing under drought when feeding on the susceptible plant but decreasing on the partially-resistant plant. Furthermore, we show that the expression of thionin genes, plant defensive compounds that contribute aphid resistance, increase ten-fold in susceptible plants exposed to drought stress but remain at constant levels in the partially-resistant plant, suggesting they play an important role in modulating aphid populations. This study highlights the role of plant defensive processes in mediating the interactions between the environment, plants, and herbivorous insects.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.13.21263371', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Spread of SARS-CoV-2 Delta variant infections bearing the S:E484Q and S:T95I mutations in July and August 2021 in France', 'authors': \"[{'name': 'Laura Verdurme'}, {'name': 'Gonch Danesh'}, {'name': 'Sabine Trombert-Paolantoni'}, {'name': 'Mircea T. Sofonea'}, {'name': 'Valrie Noel'}, {'name': 'Vincent Foulongne'}, {'name': 'Brigitte Monts'}, {'name': 'Edouard Tuaillon'}, {'name': 'Stphanie Haim-Boukobza'}, {'name': 'Bndicte Roquebert'}, {'name': 'Samuel Alizon'}]\", 'abstract': '<p>Analysing 92,598 variant screening tests performed on SARS-CoV-2 positive samples collected in France between 1 July and 31 August 2021 shows an increase of Kappa-like infections. Full genome sequencing reveals that these correspond to Delta variants bearing the S:E484Q mutation. Most of these sequences belong to a phylogenetic cluster and also bear the S:T95I mutation. Further monitoring is needed to determine if this trend is driven by undocumented superspreading events or an early signal of future viral evolutionary dynamics.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.14.460215', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Paleontology', 'covid': False, 'title': 'Dinosaur bonebed amber from an original swamp forest soil', 'authors': \"[{'name': 'Sergio lvarez-Parra'}, {'name': 'Ricardo Prez-de la Fuente'}, {'name': 'Enrique Pealver'}, {'name': 'Eduardo Barrn'}, {'name': 'Luis Alcal'}, {'name': 'Jordi Prez-Cano'}, {'name': 'Carles Martn-Closas'}, {'name': 'Khaled Trabelsi'}, {'name': 'Nieves Melndez'}, {'name': 'Rafael Lpez Del Valle'}, {'name': 'Rafael P Lozano'}, {'name': 'David Peris'}, {'name': 'Ana Rodrigo'}, {'name': 'Vctor Sarto i Monteys'}, {'name': 'Carlos A Bueno-Cebollada'}, {'name': 'Csar Menor-Salvn'}, {'name': 'Marc Philippe'}, {'name': 'Alba Snchez-Garca'}, {'name': 'Constanza Pea-Kairath'}, {'name': 'Antonio Arillo'}, {'name': 'Eduardo Esplez'}, {'name': 'Luis Mampel'}, {'name': 'Xavier Delcls'}]\", 'abstract': '<p>Dinosaur bonebeds with amber content, yet scarce, offer a superior wealth and quality of data on ancient terrestrial ecosystems. However, the preserved palaeodiversity and/or taphonomic characteristics of these exceptional localities had hitherto limited their palaeobiological potential. Here we describe the amber from the Lower Cretaceous dinosaur bonebed of Ario (Teruel, Spain) using a multidisciplinary approach. Amber is found in both a root layer with amber strictly in situ and a litter layer namely composed of aerial pieces unusually rich in bioinclusions, encompassing 11 insect orders, arachnids, and a few plant and vertebrate remains, including a feather. Additional palaeontological data  charophytes, palynomorphs, ostracods are provided. Ario arguably represents the most prolific and palaeobiologically diverse locality in which fossiliferous amber and a dinosaur bonebed have been found in association, and the only one known where the vast majority of the palaeontological assemblage suffered no or low-grade pre-burial transport. That has enabled unlocking unprecedentedly complete and reliable palaeoecological data out of two complementary windows of preservation from the same site.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.15.460086', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Little hope for the polyploid endemic Pyrenean Larkspur (Delphinium montanum): evidences from population genomics and Ecological Niche Modelling', 'authors': \"[{'name': 'Pascaline Salvado'}, {'name': 'Pere Aymerich Boixader'}, {'name': 'Josep Parera'}, {'name': 'Albert Vila Bonfill'}, {'name': 'Maria Martin'}, {'name': 'Cline Quelennec'}, {'name': 'Jean-Marc Lewin'}, {'name': 'Valrie Hinoux'}, {'name': 'Joris A. M. Bertrand'}]\", 'abstract': '<p>Species endemic to restricted geographical ranges represent a particular conservation issue, be it for their heritage interest. In a context of global change, this is particularly the case for plants which belong to high-mountain ecosystems and, because of their ecological requirements, are doomed to survive or disappear on their sky islands. The Pyrenean Larkspur (<italic>Delphinium montanum</italic>, Ranunculaceae) is endemic to the Eastern part of the Pyrenees (France and Spain). It is now only observable at a dozen of localities and some populations show signs of decline, such as a recurrent lack of flowering. Implementing population genomic approach (e.g. RAD-seq like) is particularly useful to understand genomic patterns of diversity and differentiation in order to provide recommendations in term of conservation. However, it remains challenging for species such as <italic>D. montanum</italic> that are autotetraploid with a large genome size (1C-value &gt; 10 pg) as most methods currently available were developed for diploid species. A Bayesian framework able to call genotypes with uncertainty allowed us to assess genetic diversity and population structure in this system. Our results show evidence for inbreeding (mean <italic>G</italic><sub>IS</sub> = 0.361) within all the populations and substantial population structure (mean <italic>G</italic><sub>ST</sub> = 0.403) at the metapopulation level. In addition to a lack of connectivity between populations, spatial projections of Ecological Niche Modelling analyses under different climatic scenarios predict a dramatic decrease of suitable habitat for <italic>D. montanum</italic> in the future. Based on these results, we discuss the relevance and feasibility of different conservation measures.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.16.460688', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Comparative transcriptomics of seven Impatiens species', 'authors': \"[{'name': 'Mria urinov'}, {'name': 'tepn Stoes'}, {'name': 'Tom Dostlek'}, {'name': 'Andrea Jaroov'}, {'name': 'Zuzana Mnzbergov'}]\", 'abstract': '<p><italic>Impatiens</italic> is a genus containing more than 1000 species. Thanks to its size, it is a unique system for studying species diversification in natural populations. This study focused on the characterization of novel transcriptomes from seven <italic>Impatiens</italic> species originating from Nepal. Leave transcriptome of <italic>Impatines balsamina</italic> L.<italic>, I. racemosa</italic> DC.<italic>, I. bicornuta</italic> Wall<italic>, I. falcifer</italic> Hook<italic>, I. devendrae</italic> Pusalkar, <italic>I. scullyi</italic> Hook and <italic>I. scabrida</italic> DC were sequenced and compared. Reads were <italic>de novo</italic> assembled and aligned to 92 035-226 081 contigs. We identified 14 728 orthology groups shared among all the species and 3 020 which were unique to a single species. In single species, 2536-3009 orthology groups were under selection from which 767 were common for all species. Six of the seven investigated species shared 77% of gene families with <italic>I. bicornuta</italic> being the most distinct species. Specific gene families involved in response to different environmental stimuli were closely described. <italic>Impatiens bicornuta</italic> selection profile shared selection on zing finger protein structures and flowering regulation and stress response proteins with the other investigated species. Overall, the study showed substantial similarity in patterns of selections on transcribed genes across the species suggesting similar evolutionary pressures. This suggests that the species group may have evolved via adaptive radiation.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.13.460151', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Allosteric modulation of the adenosine A2A receptor by cholesterol', 'authors': \"[{'name': 'Shuya Kate Huang'}, {'name': 'Omar Almurad'}, {'name': 'Reizel J. Pejana'}, {'name': 'Zachary A. Morrison'}, {'name': 'Aditya Pandey'}, {'name': 'Louis-Philippe Picard'}, {'name': 'Mark Nitz'}, {'name': 'Adnan Sljoka'}, {'name': 'R. Scott Prosser'}]\", 'abstract': '<p>Cholesterol is a major component of the cell membrane and commonly regulates membrane protein function. Here, we investigate how cholesterol modulates the conformational equilibria and signaling of the adenosine A<sub>2A</sub> receptor (A<sub>2A</sub>R) in reconstituted phospholipid bilayers. GTP hydrolysis assays show that cholesterol is a weak positive allosteric modulator of A<sub>2A</sub>R, as seen through enhanced basal signaling and a small decrease in agonist EC<sub>50</sub>. Fluorine nuclear magnetic resonance (<sup>19</sup>F NMR) spectroscopy suggests that this enhancement arises from an increase in the receptors active state populations and stronger G protein coupling. <sup>19</sup>F NMR of fluorinated cholesterol analogs reveals transient and non-specific interactions with A<sub>2A</sub>R, indicating a lack of high-affinity binding sites or direct allosteric modulation. This is confirmed by computational analysis which suggests that cholesterol contacts confer a weak and possibly negative allosteric effect. The combined results suggest that the observed cholesterol allostery in A<sub>2A</sub>R is likely a result of indirect membrane effects through cholesterol-mediated changes in membrane properties, as shown by membrane fluidity measurements and high-pressure NMR.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.13.460193', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genomic and physiological characterization of Novosphingobium terrae sp. nov., an alphaproteobacterium isolated from Cerrado soil containing a megasized chromid', 'authors': \"[{'name': 'Aline Belmok'}, {'name': 'Felipe Marques de Almeida'}, {'name': 'Rodrigo Theodoro Rocha'}, {'name': 'Carla Simone Vizzotto'}, {'name': 'Marcos Rogrio Ttola'}, {'name': 'Marcelo Henrique Soller Ramada'}, {'name': 'Ricardo Henrique Krger'}, {'name': 'Cynthia Maria Kyaw'}, {'name': 'Georgios J. Pappas'}]\", 'abstract': '<p>A novel bacterial strain, designated GeG2<sup>T</sup>, was isolated from soils of native Cerrado, a highly biodiverse savanna-like Brazilian biome. 16S rRNA gene sequence analysis of strain GeG2<sup>T</sup> revealed high sequence identity (100%) to the alphaproteobacterium <italic>Novosphingobium rosa</italic>, however, comparisons with <italic>N. rosa</italic> DSM7285<sup>T</sup> showed several distinctive features, prompting a full characterization of the new strain in terms of growth, morphology, biochemistry and, ultimately, its genome. GeG2<sup>T</sup> cells were Gram-stain negative bacilli, facultatively anaerobic, motile, positive for catalase and oxidase activities and for starch hydrolysis. Strain GeG2<sup>T</sup> presented planktonic-sessile dimorphism and cell aggregates surrounded by extracellular matrix and nanometric spherical structures were observed in liquid cultures, suggesting the production of exopolysaccharides (EPS) and outer membrane vesicles (OMVs). Whole genome assembly revealed four circular replicons: a 4.1 Mb chromosome, a 2.7 Mb extrachromosomal megareplicon and two plasmids (212.7 and 68.6 kb). The megareplicon contains few core genes and plasmid-type replication/maintenance systems, consistent with its classification as a chromid. Genome annotation shows a vast repertoire of carbohydrate active enzymes and genes involved in the degradation of aromatic compounds, highlighting the biotechnological potential of the new isolate obtained from Cerrado soils, especially regarding EPS production and biodegradation of recalcitrant compounds. Chemotaxonomic features, including polar lipid and fatty acid profiles, as well as physiological, molecular and whole genome comparisons showed significant differences between strain GeG2<sup>T</sup> and a <italic>N. rosa</italic>, clearly indicating that it represents a novel species, for which the name <italic>Novosphingobium terrae</italic> is proposed. The type strain is GeG2<sup>T</sup> (=CBMAI 2313T =CBAS 753T).</p><sec><title>IMPORTANCE</title><p><italic>Novosphingobium</italic> is an alphaproteobacterial genus presenting diverse physiological profiles and broad biotechnological applications. However, many aspects regarding the biology of this important bacterial group remain elusive. A novel <italic>Novosphingobium</italic> strain was isolated from soils of Cerrado, an important Brazilian biome. Despite 100% 16S rRNA gene identity with <italic>Novosphingobium rosa</italic>, polyphasic characterizations, including physiological, chemotaxonomic, and whole genome- based analyses revealed significant differences between GeG2<sup>T</sup> and N. rosa DSM7285<sup>T</sup>, reinforcing resolution limitations in phylogenetic analysis based solely on 16S RNA and highlighting the importance of employing different approaches for the description of bacterial species. Using short and long read sequencing approaches, a high-quality fully resolved genome assembly was generated and one of the largest chromids reported to date was identified. A comprehensive characterization of environmental isolates allows us to better elucidate the diversity and biology of members of this bacterial group with potential biotechnological importance, guiding future bioprospecting efforts and genomic studies.</p></sec>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.14.460204', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Controls on the isotopic composition of microbial methane', 'authors': \"[{'name': 'Jonathan Gropp'}, {'name': 'Qusheng Jin'}, {'name': 'Itay Halevy'}]\", 'abstract': '<p>Microbial methane production (methanogenesis) is responsible for more than half of the annual emission of this major greenhouse gas to the atmosphere. Though the stable isotopic composition of methane is often used to characterize its sources and sinks, empirical descriptions of the isotopic signature of methanogenesis currently limit such attempts. We developed a biochemical-isotopic model of methanogenesis by CO<sub>2</sub> reduction, which predicts carbon and hydrogen isotopic fractionations, and clumped isotopologue distributions, as functions of the cells environment. We mechanistically explain multiple-isotopic patterns in laboratory and natural settings and show that such patterns constrain the in-situ energetics of methanogenesis. Combining our model with environmental data, we infer that in almost all marine environments and gas deposits, energy-limited methanogenesis operates close to chemical and isotopic equilibrium.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.14.460205', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Forkhead transcription factor FKH-8 is a master regulator of primary cilia in C. elegans', 'authors': \"[{'name': 'Rebeca Brocal-Ruiz'}, {'name': 'Ainara Esteve-Serrano'}, {'name': 'Carlos Mora-Martinez'}, {'name': 'Juan Tena'}, {'name': 'Nuria Flames'}]\", 'abstract': '<p>Cilia, either motile or non-motile (a.k.a primary or sensory), are complex evolutionary conserved eukaryotic structures composed of hundreds of proteins required for their assembly, structure and function that are collectively known as the ciliome. Ciliome mutations underlie a group of pleiotropic genetic diseases known as ciliopathies. Proper cilium function requires the tight coregulation of ciliome gene transcription, which is only fragmentarily understood. RFX transcription factors (TF) have an evolutionarily conserved role in the direct activation of ciliome genes both in motile and non-motile cilia cell types. In vertebrates, FoxJ1 and FoxN4 Forkhead (FKH) TFs work with RFX in the direct activation of ciliome genes, exclusively in motile cilia cell-types. No additional TFs have been described to act together with RFX in primary cilia cell-types in any organism. Here we describe FKH-8, a FKH TF, as master regulator of the primary ciliome in <italic>Caenorhabditis elegans</italic>. <italic>fkh-8</italic> is expressed in all ciliated neurons in C. <italic>elegans</italic>, binds the regulatory regions of ciliome genes, regulates ciliome gene expression, cilium morphology and a wide range of behaviours mediated by sensory cilia. Importantly, we find FKH-8 function can be replaced by mouse FOXJ1 and FOXN4 but not by members of other mouse FKH subfamilies. In conclusion, our results show that RFX and FKH TF families act as master regulators of ciliogenesis also in sensory ciliated cell types and suggest that this regulatory logic could be an ancient trait predating functional cilia sub-specialization.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.11.21256274', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Health Policy', 'covid': False, 'title': 'The Morbidity Costs of Air Pollution through the Lens of Health Spending in China', 'authors': \"[{'name': 'Xin Zhang'}, {'name': 'Xun Zhang'}, {'name': 'Xi Chen'}, {'name': 'Yuehua Liu'}, {'name': 'Xintong Zhao'}]\", 'abstract': '<p>This study offers one of the first causal evidence on the morbidity costs of fine particulates (PM2.5) for all age cohorts in a developing country, using individual-level healthcare spending data from the basic medical insurance program in Wuhan, China. Our instrumental variable (IV) approach uses thermal inversion to address potential endogeneity in PM2.5 concentrations and shows that PM2.5 imposes a significant impact on medical expenditures. The IV estimate suggests that a 10 g/m<sup>3</sup> reduction in monthly average PM2.5 leads to a 2.79% decrease in the value of health spending and a 0.70% decline in the number of transactions in pharmacies and health facilities. The effect is more salient for males, children, and older adults. Moreover, our estimates provide a lower bound of peoples willingness-to-pay, which amounts to CNY 51.85 (or USD 8.38) per capita per year for a 10 g/m<sup>3</sup> reduction in PM2.5.</p><sec><title>JEL Codes</title><p>Q51, Q53, I11, I31</p></sec><sec><title>Conflict of interest</title><p>The authors declare that they have no conflict of interest.</p></sec>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.17.460803', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'CUX2 mutations in temporal lobe epilepsy; increased kainate susceptibility and excitatory input to hippocampus in deficient mice', 'authors': \"[{'name': 'Toshimitsu Suzuki'}, {'name': 'Tetsuya Tatsukawa'}, {'name': 'Genki Sudo'}, {'name': 'Caroline Delandre'}, {'name': 'Yun Jin Pai'}, {'name': 'Hiroyuki Miyamoto'}, {'name': 'Matthieu Raveau'}, {'name': 'Atsushi Shimohata'}, {'name': 'Iori Ohmori'}, {'name': 'Shin-ichiro Hamano'}, {'name': 'Kazuhiro Haginoya'}, {'name': 'Mitsugu Uematsu'}, {'name': 'Yukitoshi Takahashi'}, {'name': 'Masafumi Morimoto'}, {'name': 'Shinji Fujimoto'}, {'name': 'Hitoshi Osaka'}, {'name': 'Hirokazu Oguni'}, {'name': 'Makiko Osawa'}, {'name': 'Atsushi Ishii'}, {'name': 'Shinichi Hirose'}, {'name': 'Sunao Kaneko'}, {'name': 'Yushi Inoue'}, {'name': 'Adrian Walton Moore'}, {'name': 'Kazuhiro Yamakawa'}]\", 'abstract': '<p><italic>CUX2</italic> gene encodes a transcription factor that controls neuronal proliferation, dendrite branching and synapse formation, locating at the epilepsy-associated chromosomal region 12q24 that we previously identified by a genome-wide association study (GWAS) in Japanese population. A <italic>CUX2</italic> recurrent <italic>de novo</italic> variant p.E590K has been described in patients with rare epileptic encephalopathies and the gene is a candidate for the locus, however the mutation may not be enough to generate the genome-wide significance in the GWAS and whether <italic>CUX2</italic> variants appear in other types of epilepsies and physiopathological mechanisms are remained to be investigated. Here in this study, we conducted targeted sequencings of <italic>CUX2</italic>, a paralog <italic>CUX1</italic> and its short isoform <italic>CASP</italic> harboring a unique C-terminus on 271 Japanese patients with a variety of epilepsies, and found that multiple <italic>CUX2</italic> missense variants, other than the p.E590K, and some <italic>CASP</italic> variants including a deletion, predominantly appeared in patients with temporal lobe epilepsy (TLE). Human cell culture and fly dendritic arborization analyses revealed loss-of- function properties for the <italic>CUX2</italic> variants. <italic>Cux2</italic>- and <italic>Casp</italic>-specific knockout mice both showed high susceptibility to kainate, increased excitatory cell number in the entorhinal cortex, and significant enhancement in glutamatergic synaptic transmission to the hippocampus. CASP and CUX2 proteins physiologically bound to each other and co-expressed in excitatory neurons in brain regions including the entorhinal cortex. These results suggest that <italic>CUX2</italic> and <italic>CASP</italic> variants contribute to the TLE pathology through a facilitation of excitatory synaptic transmission from entorhinal cortex to hippocampus.</p>', 'posted': '2021-09-18'}, {'doi': '10.1101/2021.09.13.460067', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Mitochondrial dynamics regulate genome stability via control of caspase-dependent DNA damage', 'authors': \"[{'name': 'Kai Cao'}, {'name': 'Joel S Riley'}, {'name': 'Catherine Cloix'}, {'name': 'Yassmin Elmasry'}, {'name': 'Gabriel Ichim'}, {'name': 'Kirsteen J Campbell'}, {'name': 'Stephen WG Tait'}]\", 'abstract': '<p>Mitochondrial dysfunction is interconnected with cancer. Nevertheless, how defective mitochondria promote cancer is poorly understood. We find that mitochondrial dysfunction promotes DNA damage under conditions of increased apoptotic priming. Underlying this process, we reveal a key role for mitochondrial dynamics in the regulation of DNA damage and genome instability. The ability of mitochondrial dynamics to regulate oncogenic DNA damage centres upon the control of minority MOMP, a process that enables non-lethal caspase activation leading to DNA damage. Mitochondrial fusion suppresses minority MOMP, and its associated DNA damage, by enabling homogenous mitochondrial expression of anti-apoptotic BCL-2 proteins. Finally, we find that mitochondrial dysfunction inhibits pro-apoptotic BAX retrotranslocation, causing BAX mitochondrial localization thereby promoting minority MOMP. Unexpectedly, these data reveal oncogenic effects of mitochondrial dysfunction that are mediated via mitochondrial dynamics and caspase-dependent DNA damage.</p>', 'posted': '2021-09-13'}, {'doi': '10.1101/2021.09.13.460068', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'The Paradox of the Plankton: Coexistence of Structured Microbial Communities', 'authors': \"[{'name': 'Alberto Scarampi'}]\", 'abstract': '<p>In the framework of resource-competition models, it has been argued that the number of species stably coexisting in an ecosystem cannot exceed the number of shared resources. However, plankton seems to be an exception of this so-called competitive-exclusion principle. In planktic ecosystems, a large number of different species stably coexist in an environment with limited resources. This contradiction between theoretical expectations and empirical observations is often referred to as The Paradox of the Plankton. This project aims to investigate biophysical models that can account for the large biodiversity observed in real ecosystems in order to resolve this paradox. A model is proposed that combines classical resource competition models, metabolic trade-offs and stochastic ecosystem assembly. Simulations of the model match empirical observations, while relaxing some unrealistic assumptions from previous models.</p><p><bold>Paradox:</bold> from Greek <italic>para</italic>: distinct from, and <italic>doxa</italic>: opinion. Sainsbury (1995) defines a paradox as an apparently unacceptable conclusion derived by apparently acceptable reasoning from apparently acceptable premises. Paradoxes are useful research tools as they suggest logical inconsistencies. In order to spot the flaw, the validity of all the premises has to be carefully assessed.</p><p><bold>Plankton:</bold> refers to the collection of organisms that spend part or all of their lives in suspension in water (Reynolds 2006). Plankton, or <italic>plankters</italic>, are organisms that have velocities significantly smaller than oceanic currents and thus are considered to travel with the water parcel they occupy (Lombard et al. 2019). <italic>Phytoplankters</italic> refer to the members of the plankton that perform photosynthesis.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.13.460064', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'High interaction valency ensures cohesion and persistence of a microtubule +TIP body at the plus-end of a single specialized microtubule in yeast', 'authors': \"[{'name': 'Sandro M. Meier'}, {'name': 'Ana-Maria Farcas'}, {'name': 'Anil Kumar'}, {'name': 'Mahdiye Ijavi'}, {'name': 'Robert T. Bill'}, {'name': 'Jrg Stelling'}, {'name': 'Eric Dufresne'}, {'name': 'Michel O. Steinmetz'}, {'name': 'Yves Barral'}]\", 'abstract': '<p>Microtubule plus-end tracking proteins (+TIPs) control microtubule specialization and are as such essential notably during eukaryotic cell division. Here, we investigated interactions and functions of the budding yeast Kar9 network consisting of the core +TIPs components Kar9 (functional homologue of APC, MACF, and SLAIN), Bim1 (orthologue of EB1), and Bik1 (orthologue of CLIP-170). Our data indicate that a redundant, multivalent web of interactions links the three +TIPs together to form a Kar9 body at the tip of a single cytoplasmic microtubule. They further suggest that this body is a liquid-like condensate, allowing it to persist on both growing and shrinking microtubule tips, and functions as a mechanical coupling device between microtubules and actin cables during mitosis. Our study underlines the power of dissecting the web of low-affinity interactions driving liquid-liquid phase separation of proteins in order to demonstrate the importance and establish the functional roles of condensation processes in vivo.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.14.460213', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'The value of genotype-specific reference for transcriptome analyses', 'authors': \"[{'name': 'Wenbin Guo'}, {'name': 'Max Coulter'}, {'name': 'Robbie Waugh'}, {'name': 'Runxuan Zhang'}]\", 'abstract': '<p>High quality transcriptome assembly using short reads from RNA-seq data still heavily relies upon reference-based approaches, of which the primary step is to align RNA-seq reads to a single reference genome of haploid sequence. However, it is increasingly apparent that while different genotypes within a species share core genes, they also contain variable numbers of specific genes that are only present a subset of individuals. Using a common reference may thus lead to a loss of genotype-specific information in the assembled transcript dataset and the generation of erroneous, incomplete or misleading transcriptomics analysis results. With the recent development of pan-genome information in many species, it is important that we understand the limitations of single genotype references for transcriptomics analysis. In this study, we quantitively evaluated the advantages of using genotype-specific reference genomes for transcriptome assembly and analysis using cultivated barley as a model. We mapped barley cultivar Barke RNA-seq reads to the Barke genome and to the cultivar Morex genome (common barley genome reference) to construct a genotype specific Reference Transcript Dataset (sRTD) and a common Reference Transcript Datasets (cRTD), respectively. We compared the two RTDs according to their transcript diversity, transcript sequence and structure similarity and the accuracy they provided for transcript quantification and differential expression analysis. Our evaluation shows that the sRTD has a significantly higher diversity of transcripts and alternative splicing events. Despite using a high-quality reference genome for assembly of the cRTD, we miss ca. 40% transcripts present in the sRTD and cRTD only has ca. 70% true assemblies. We found that the sRTD is more accurate for transcript quantification as well as differential expression and differential alternative splicing analysis. However, gene level quantification and comparative expression analysis are less affected by the source RTD, which indicates that analysing transcriptomic data at the gene level may be a reasonable compromise when a high-quality genotype-specific reference is not available.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.14.460260', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Community diversity determines the evolution of synthetic bacterial communities under artificial selection', 'authors': \"[{'name': 'Tiffany Raynaud'}, {'name': 'Marion Devers'}, {'name': 'Aym Spor'}, {'name': 'Manuel Blouin'}]\", 'abstract': '<p>Artificial selection can be conducted at the community level in the laboratory through a differential propagation of the communities according to their level of expression of a targeted function (i.e. community phenotype). Working with communities instead of individuals as selection units brings in additional sources of variation in the considered phenotype that can arise through changes in community structure and influence the outcome of the artificial selection. These sources of variation could even be increased by manipulating species diversity. In this study, we wanted to assess the effect of manipulating initial community richness on artificial selection efficiency, defined as the change in the targeted function over time as compared to a control treatment without artificial selection. We applied artificial selection for a high productivity on synthetic bacterial communities varying for their initial richness level (from one to 16 strains). Our results showed that, overall, the communities that were artificially selected were 16% more productive than the control communities. Community richness positively influenced community productivity and metabolic capacities and was a strong determinant of the dynamics of community evolution. Our results suggested that community richness could influence artificial selection efficiency but a convergence of the community composition might have limited the effect of diversity on artificial selection efficiency. We propose that applying artificial selection on communities varying for their diversity could allow to find communities differing for their level of expression of a function but also for their responsiveness to artificial selection, provided that their initial composition is different enough.</p>', 'posted': '2021-09-14'}, {'doi': '10.1101/2021.09.14.460266', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Investigation of a derived adverse outcome pathway (AOP) network for endocrine-mediated perturbations', 'authors': \"[{'name': 'Janani Ravichandran'}, {'name': 'Bagavathy Shanmugam Karthikeyan'}, {'name': 'Areejit Samal'}]\", 'abstract': '<p>An adverse outcome pathway (AOP) is a compact representation of the available mechanistic information on observed adverse effects upon environmental exposure. Sharing of events across individual AOPs has led to the emergence of AOP networks. Since AOP networks are expected to be functional units of toxicity prediction, there is current interest in their development tailored to specific research question or regulatory problem. To this end, we have developed a detailed workflow to construct a comprehensive endocrine-specific AOP (ED-AOP) network. Connectivity analysis of the ED-AOP network comprising 48 AOPs reveals 7 connected components and 12 isolated AOPs. Subsequently, we apply standard network measures to perform an in-depth analysis of the two largest connected components of the ED-AOP network. Notably, the graph-theoretic analyses led to the identification of important events including points of convergence or divergence in the ED-AOP network. Detailed analysis of the largest component in the ED-AOP network gives insights on the systems-level perturbations caused by endocrine disruption, emergent paths, and stressor-event associations. In sum, the derived ED-AOP network can be used to address the current knowledge gaps in the existing regulatory framework and aid in better risk assessment of environmental chemicals.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.13.460053', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Pathology', 'covid': False, 'title': 'Dysregulation of the Tweak/Fn14 pathway in skeletal muscle of spinal muscular atrophy mice', 'authors': \"[{'name': 'Katharina E. Meijboom'}, {'name': 'Emily McFall'}, {'name': 'Daniel Anthony'}, {'name': 'Benjamin Edwards'}, {'name': 'Sabrina Kubinski'}, {'name': 'Gareth Hazell'}, {'name': 'Nina Ahlskog'}, {'name': 'Peter Claus'}, {'name': 'Kay E. Davies'}, {'name': 'Rashmi Kothary'}, {'name': 'Matthew J.A. Wood'}, {'name': 'Melissa Bowerman'}]\", 'abstract': '<p>Spinal muscular atrophy (SMA) is a childhood neuromuscular disorder caused by depletion of the survival motor neuron (SMN) protein. SMA is characterized by the selective death of spinal cord motor neurons, leading to progressive muscle wasting. Loss of skeletal muscle in SMA is a combination of denervation-induced muscle atrophy and intrinsic muscle pathologies. Elucidation of the pathways involved is essential to identify the key molecules that contribute to and sustain muscle pathology. The tumor necrosis factor-like weak inducer of apoptosis (TWEAK)/TNF receptor superfamily member fibroblast growth factor inducible 14 (Fn14) pathway has been shown to play a critical role in the regulation of denervation-induced muscle atrophy as well as muscle proliferation, differentiation and metabolism in adults. However, it is not clear whether this pathway would be important in highly dynamic and developing muscle. We thus investigated the potential role of the TWEAK/Fn14 pathway in SMA muscle pathology, using the severe Taiwanese <italic>Smn<sup>-/-</sup>;SMN2</italic> and the less severe <italic>Smn<sup>2B/-</sup></italic> SMA mice, which undergo a progressive neuromuscular decline in the first three post-natal weeks. Here, we report significantly dysregulated expression of the TWEAK/Fn14 pathway during disease progression in skeletal muscle of the two SMA mouse models. In addition, siRNA-mediated <italic>Smn</italic> knockdown in C2C12 myoblasts suggests a genetic interaction between Smn and the TWEAK/Fn14 pathway. Further analyses of SMA, <italic>Tweak<sup>-/-</sup></italic> and <italic>Fn14<sup>-/-</sup></italic> mice revealed dysregulated myopathy, myogenesis and glucose metabolism pathways as a common skeletal muscle feature, and providing further evidence in support of a relationship between the TWEAK/Fn14 pathway and Smn. Finally, a pharmacological intervention (Fc-TWEAK) to upregulate the activity of the TWEAK/Fn14 pathway improved disease phenotypes in the two SMA mouse models. Our study provides novel mechanistic insights into the molecular players that contribute to muscle pathology in SMA and into the role of the TWEAK/Fn14 pathway in developing muscle.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.13.460049', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Convex-PLR  Revisiting affinity predictions and virtual screening using physics-informed machine learning', 'authors': \"[{'name': 'Maria Kadukova'}, {'name': 'Vladimir Chupin'}, {'name': 'Sergei Grudinin'}]\", 'abstract': '<p>Virtual screening is an essential part of the modern drug design pipeline, which significantly accelerates the discovery of new drug candidates. Structure-based virtual screening involves ligand conformational sampling, which is often followed by re-scoring of docking poses. A great variety of scoring functions have been designed for this purpose. The advent of structural and affinity databases and the progress in machine-learning methods have recently boosted scoring function performance. Nonetheless, the most successful scoring functions are typically designed for specific tasks or systems. All-purpose scoring functions still perform poorly on the virtual screening tests, compared to precision with which they are able to predict co-crystal binding poses. Another limitation is the low interpretability of the heuristics being used.</p><p>We analyzed scoring functions performance in the CASF benchmarks and discovered that the vast majority of them have a strong bias towards predicting larger binding interfaces. This motivated us to develop a physical model with additional entropic terms with the aim of penalizing such a preference. We parameterized the new model using affinity and structural data, solving a classification problem followed by regression. The new model, called Convex-PL<sup><italic>R</italic></sup>, demonstrated high-quality results on multiple tests and a substantial improvement over its predecessor Convex-PL. Convex-PL<sup><italic>R</italic></sup> can be used for molecular docking together with VinaCPL, our version of AutoDock Vina, with Convex-PL integrated as a scoring function. Convex-PL<sup><italic>R</italic></sup>, Convex-PL, and VinaCPL are available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://team.inria.fr/nano-d/convex-pl/\">https://team.inria.fr/nano-d/convex-pl/</ext-link>.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.13.460013', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Detecting Phytophthora cinnamomi associated with dieback disease on Carya cathayensis using loop-mediated isothermal amplification', 'authors': \"[{'name': 'Xiaoqing Tong'}, {'name': 'Jiayi Wu'}, {'name': 'Li Mei'}, {'name': 'Yongjun Wang'}]\", 'abstract': '<p>Chinese hickory (<italic>Carya cathayensis</italic> Sarg.) is an economically and ecologically important nut plant in China. Dieback and basal stem necrosis have been observed in the plants since 2016, and its recent spread has significantly affected plant growth and nut production. Therefore, a survey was conducted to evaluate the disease incidence at five sites in Linan County, China. The highest incidence was recorded at the Tuankou site at up to 11.39% in 2019. The oomycete, <italic>Phytophthora cinnamomi</italic>, was isolated from symptomatic plant tissue and plantation soil using baiting and selective media-based detection methods and identified. Artificial infection with the representative <italic>P. cinnamomi</italic> ST402 isolate produced vertically elongated discolorations in the outer xylem and necrotic symptoms in <italic>C. cathayensis</italic> seedlings in a greenhouse trial. Molecular detections based on loop-mediated isothermal amplification (LAMP) specific to <italic>P. cinnamomi</italic> ST402 were conducted. Result indicated that LAMP detection showed a high coherence level with the baiting assays for <italic>P. cinnamomi</italic> detection in the field. This study provides the evidence of existence of high-pathogenic <italic>P. cinnamomi</italic> in the <italic>C. cathayensis</italic> plantation soil in China and the insights into a convenient tool developed for conducting field monitoring of this aggressive pathogen.</p>', 'posted': '2021-09-13'}, {'doi': '10.1101/2021.09.12.459983', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Transcutaneous auricular vagus nerve stimulation increases eye-gaze on salient facial features and oxytocin release', 'authors': \"[{'name': 'Siyu Zhu'}, {'name': 'Yanan Qing'}, {'name': 'Yingying Zhang'}, {'name': 'Xiaolu Zhang'}, {'name': 'Fangyuan Ding'}, {'name': 'Rong Zhang'}, {'name': 'Shuxia Yao'}, {'name': 'Keith Kendrick'}, {'name': 'Weihua Zhao'}]\", 'abstract': '<sec><title>Background</title><p>Transcutaneous auricular vagus nerve stimulation (taVNS) is a non-invasive neuromodulation technique with promising therapeutic potential in the context of epilepsy, pain, and depression and which may also have beneficial effects on social cognition. However, the underlying mechanisms of taVNS are unclear and evidence regarding its role in social cognition improvement is limited.</p></sec><sec><title>Objective</title><p>In order to investigate the impact of taVNS on social cognition we have studied its effects on gaze towards emotional faces using an eye-tracking task and also on release of the neuropeptide oxytocin which plays a key role in influencing social cognition and motivation.</p></sec><sec><title>Methods</title><p>A total of fifty-four subjects were enrolled in a sham-controlled, participant-blind crossover experiment, consisting of two treatment sessions, separated by one week. In one session participants received 30-min taVNS (tragus), and in the other, they received 30-min sham (earlobe) stimulation with the treatment order counterbalanced across participants. Gaze duration towards the faces and facial features (eyes, nose, and mouth) were measured together with resting pupil size. Additionally, saliva samples were taken for the measurement of oxytocin concentrations by enzyme-linked immunoassay.</p></sec><sec><title>Results</title><p>Saliva oxytocin concentrations increased significantly after taVNS compared to sham stimulation, while resting pupil size did not. In addition, taVNS increased fixation time on the nose region irrespective of face emotion, and this was positively correlated with increased saliva oxytocin concentrations.</p></sec><sec><title>Conclusion</title><p>Our findings suggest that taVNS biases visual attention towards socially salient facial features across different emotions and this is associated with its effects on increasing endogenous oxytocin release.</p></sec>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.07.21263194', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'COVID-19 convalescent plasma and randomized clinical trials: rebuilding confidence by explaining failures and finding signals of efficacy', 'authors': \"[{'name': 'Daniele Focosi'}, {'name': 'Massimo Franchini'}, {'name': 'Liise-anne Pirofski'}, {'name': 'Thierry Burnouf'}, {'name': 'Nigel Paneth'}, {'name': 'Michael J. Joyner'}, {'name': 'Arturo Casadevall'}]\", 'abstract': '<p>Convalescent plasma (CP) recurs as a frontline treatment in epidemics because it is available as soon as there are survivors. The COVID-19 pandemic represented the first large-scale opportunity to shed light into mechanisms of action, safety and efficacy of convalescent plasma using modern evidence-based medicine approaches. Studies ranging from observational case series to randomized controlled trials (RCT) have reported highly variable efficacy results for COVID-19 CP (CCP), resulting in more doubt than certainty. Reasons for CCP success and failure may be hidden in study details, which are usually difficult to explain to physicians and the public but provide fertile ground for designing next-generation studies. In this paper we analyzed variables associated with efficacy such as clinical settings, disease severity, CCP SARS-CoV-2 antibody levels and function, dose, timing of administration (variously defined as time from onset of symptoms, molecular diagnosis, diagnosis of pneumonia, or hospitalization, or by serostatus), outcomes (defined as hospitalization, requirement for ventilation, clinical improvement or mortality), CCP provenance and time for collection, and criteria for efficacy. Focusing only on the results from the 23 available RCT we noted that these were more likely to show signals of efficacy, including reductions in mortality, if the plasma neutralizing titer was  160 and the time to randomization was  9 days, consistent with passive antibody therapy efficacy requiring dosing with sufficient antibody. The fact that most studies revealed signals of efficacy despite variability in CCP and its use suggest robust therapeutic effects that become apparent despite the data noise.</p>', 'posted': '2021-09-15'}, {'doi': '10.1101/2021.09.14.460366', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'The polyol pathway is an evolutionarily conserved system for sensing glucose uptake', 'authors': \"[{'name': 'Hiroko Sano'}, {'name': 'Akira Nakamura'}, {'name': 'Mariko Yamane'}, {'name': 'Hitoshi Niwa'}, {'name': 'Takashi Nishimura'}, {'name': 'Kimi Araki'}, {'name': 'Kazumasa Takemoto'}, {'name': 'Kei-ichiro Ishiguro'}, {'name': 'Hiroki Aoki'}, {'name': 'Masayasu Kojima'}]\", 'abstract': '<p>Cells must adjust the expression levels of metabolic enzymes in response to fluctuating nutrient supply. For glucose, such metabolic remodeling is highly dependent on a master transcription factor ChREBP/MondoA. However, it remains elusive how glucose fluctuations are sensed by ChREBP/MondoA despite the stability of major glycolytic pathways. Here we show that in both flies and mice, ChREBP/MondoA activation in response to glucose ingestion depends on an evolutionarily conserved glucose-metabolizing pathway: the polyol pathway. The polyol pathway converts glucose to fructose via sorbitol. It has been believed that this pathway is almost silent, and its activation in hyperglycemic conditions has deleterious effects on human health. We show that the polyol pathway is required for the glucose-induced nuclear translocation of Mondo, a <italic>Drosophila</italic> homologue of ChREBP/MondoA, which directs gene expression for organismal growth and metabolism. Likewise, inhibition of the polyol pathway in mice impairs ChREBPs nuclear localization and reduces glucose tolerance. We propose that the polyol pathway is an evolutionarily conserved sensing system for the glucose uptake that allows metabolic remodeling.</p>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.17.460664', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Genetically diverse mouse models of SARS-CoV-2 infection model clinical variation and cytokine responses in COVID-19', 'authors': \"[{'name': 'Shelly Robertson'}, {'name': 'Olivia Bedard'}, {'name': 'Kristin McNally'}, {'name': 'Matthew Lewis'}, {'name': 'Chad Clancy'}, {'name': 'Carl Shaia'}, {'name': 'Rebecca Broeckel'}, {'name': 'Abhilash Chiramel'}, {'name': 'Gail Sturdevant'}, {'name': 'Elvira Forte'}, {'name': 'Christoph Preuss'}, {'name': 'Candice Baker'}, {'name': 'Daniel Sturdevant'}, {'name': 'Craig Martens'}, {'name': 'Steven Holland'}, {'name': 'Nadia Rosenthal'}, {'name': 'Sonja Best'}]\", 'abstract': '<p>Host genetics are a significant determinant of coronavirus disease 2019 (COVID-19). Animal models that reflect genetic diversity and a range of clinical outcomes observed in human populations are needed to understand mechanisms of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection dynamics and disease. Here, we report a mouse panel comprising the diverse genetic backgrounds of the Collaborative Cross (CC) founder strains crossed to C57BL/6J mice expressing the K18-hACE2 transgene3 that enables infection by SARS-CoV-2. Infection of CCxK18-hACE2 F1 progeny resulted in a spectrum of weight loss, survival, viral replication kinetics, histopathology, and cytokine profiles, some of which were sex-specific. Importantly, survival was closely associated with early type I interferon expression and a phased proinflammatory response distinct from mice with severe disease. Thus, dynamics of inflammatory responses observed in COVID-19 can be modeled in diverse mice that provide a genetically tractable platform for understanding antiviral immunity and evaluating countermeasures.</p>', 'posted': '2021-09-18'}, {'doi': '10.1101/2021.09.15.460555', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Ecdysteroid kinase-like (EcKL) paralogs confer developmental tolerance to caffeine in Drosophila melanogaster', 'authors': \"[{'name': 'Jack L. Scanlan'}, {'name': 'Paul Battlay'}, {'name': 'Charles Robin'}]\", 'abstract': '<p>A unique aspect of metabolic detoxification in insects compared to other animals is the presence of xenobiotic phosphorylation, about which little is currently understood. Our previous work raised the hypothesis that members of the taxonomically restricted ecdysteroid kinase-like (EcKL) gene family encode the enzymes responsible for xenobiotic phosphorylation in the model insect <italic>Drosophila melanogaster</italic> (Diptera: Ephydroidea)however, candidate detoxification genes identified in the EcKL family have yet to be functionally validated. Here, we test the hypothesis that EcKL genes in the rapidly evolving Dro5 clade are involved in the detoxification of plant and fungal toxins in <italic>D. melanogaster</italic>. The mining and reanalysis of existing data indicated multiple Dro5 genes are transcriptionally induced by the plant alkaloid caffeine and that adult caffeine susceptibility is associated with a novel naturally occurring indel in <italic>CG31370</italic> (Dro5-8) in the Drosophila Genetic Reference Panel (DGRP). CRISPR-Cas9 mutagenesis of five Dro5 EcKLs substantially decreased developmental tolerance of caffeine, while individual overexpression of two of these genes<italic>CG31300</italic> (Dro5-1) and <italic>CG13659</italic> (Dro5-7)in detoxification-related tissues increased developmental tolerance. In addition, we found Dro5 loss-of-function animals also have decreased developmental tolerance of the fungal secondary metabolite kojic acid. Taken together, this work provides the first compelling functional evidence that EcKLs encode detoxification enzymes and suggests that EcKLs in the Dro5 clade are involved in the metabolism of multiple ecologically relevant toxins in <italic>D. melanogaster</italic>. We also propose a biochemical hypothesis for EcKL involvement in caffeine detoxification and highlight the many unknown aspects of caffeine metabolism in <italic>D. melanogaster</italic> and other insects.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Phosphorylation is an under-characterised detoxification reaction in insects</p></list-item><list-item><p>Dro5 EcKL genes are good detoxification candidate genes in <italic>Drosophila melanogaster</italic></p></list-item><list-item><p>Knockout and misexpression of some Dro5 genes modulated tolerance to caffeine</p></list-item><list-item><p>Dro5 genes may also confer tolerance to the fungal toxin kojic acid</p></list-item><list-item><p>Caffeine tolerance could be adaptive for <italic>Drosophila</italic> associating with <italic>Citrus</italic> fruits</p></list-item></list></sec>', 'posted': '2021-09-16'}, {'doi': '10.1101/2021.09.13.460073', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Addition of a carboxy terminal tail to the normally tailless gonadotropin-releasing hormone receptor impairs fertility in female mice', 'authors': \"[{'name': 'Chirine Toufaily'}, {'name': 'Jrme Fortin'}, {'name': 'Carlos A. I. Alonso'}, {'name': 'Evelyn Lapointe'}, {'name': 'Xiang Zhou'}, {'name': 'Yorgui Santiago-Andres'}, {'name': 'Yeu-Farn Lin'}, {'name': 'Yimming Cui'}, {'name': 'Ying Wang'}, {'name': 'Dominic Devost'}, {'name': 'Ferdinand Roelfsema'}, {'name': 'Frederik Steyn'}, {'name': 'Aylin C. Hanyaloglu'}, {'name': 'Terence E. Hbert'}, {'name': 'Tatiana Fiordelisio'}, {'name': 'Derek Boerboom'}, {'name': 'Daniel J. Bernard'}]\", 'abstract': '<p>Gonadotropin-releasing hormone (GnRH) is the primary neuropeptide controlling reproduction in vertebrates. GnRH stimulates follicle-stimulating hormone (FSH) and luteinizing hormone (LH) synthesis via a G protein-coupled receptor, GnRHR, in the pituitary gland. In mammals, GnRHR lacks a C-terminal cytosolic tail (Ctail) and does not exhibit homologous desensitization. This might be an evolutionary adaptation that enables LH surge generation and ovulation. To test this idea, we fused the chicken GnRHR Ctail to the endogenous murine GnRHR in a transgenic model. The LH surge was blunted, but not blocked in these mice. In contrast, they showed reductions in FSH production, ovarian follicle development, and fertility. Addition of the Ctail altered the nature of agonist-induced calcium signaling required for normal FSH production. The loss of the GnRHR Ctail during mammalian evolution is unlikely to have conferred a selective advantage by enabling the LH surge. The adaptive significance of this specialization remains to be determined.</p>', 'posted': '2021-09-16'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.09.14.460138</td>\n",
       "      <td>39</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Structure of a bacterial Rhs effector exported...</td>\n",
       "      <td>[{'name': 'Patrick Gnther'}, {'name': 'Dennis...</td>\n",
       "      <td>&lt;p&gt;The type VI &lt;underline&gt;s&lt;/underline&gt;ecretio...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.09.13.460111</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Therapeutic efficacy of an oral nucleoside ana...</td>\n",
       "      <td>[{'name': 'Alexandra Schafer'}, {'name': 'Davi...</td>\n",
       "      <td>&lt;p&gt;The COVID-19 pandemic remains uncontrolled ...</td>\n",
       "      <td>2021-09-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.09.09.21263331</td>\n",
       "      <td>34</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Second wave of the Covid-19 pandemic in Delhi,...</td>\n",
       "      <td>[{'name': 'Nandini Sharma'}, {'name': 'Pragya ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;We report the...</td>\n",
       "      <td>2021-09-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.09.13.460168</td>\n",
       "      <td>32</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Random Forest Factorization Reveals Latent Str...</td>\n",
       "      <td>[{'name': 'Boris M. Brenerman'}, {'name': 'Ben...</td>\n",
       "      <td>&lt;p&gt;Single-cell RNA sequencing data contain pat...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.09.01.21262607</td>\n",
       "      <td>31</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Health Informatics</td>\n",
       "      <td>False</td>\n",
       "      <td>ADataViewer: Exploring Semantically Harmonized...</td>\n",
       "      <td>[{'name': 'Yasamin Salimi'}, {'name': 'Daniel ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;INTRODUCTION&lt;/title&gt;&lt;p&gt;Currently, ...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.09.13.21263499</td>\n",
       "      <td>30</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>False</td>\n",
       "      <td>The natural history of TB disease-a synthesis ...</td>\n",
       "      <td>[{'name': 'Alexandra S Richards'}, {'name': 'B...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Prevalence su...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.09.15.460550</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Intact reading ability in spite of a spatially...</td>\n",
       "      <td>[{'name': 'Jin Li'}, {'name': 'Evelina Fedoren...</td>\n",
       "      <td>&lt;p&gt;The visual word form area (VWFA) is an expe...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.09.15.460516</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Long-term Motor Learning in the Wild with High...</td>\n",
       "      <td>[{'name': 'Jennifer B Listman'}, {'name': 'Jon...</td>\n",
       "      <td>&lt;p&gt;Motor learning occurs over long periods of ...</td>\n",
       "      <td>2021-09-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.09.14.460331</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Defective endosome-TGN retrograde transport pr...</td>\n",
       "      <td>[{'name': 'Zhirong Zhang'}, {'name': 'Li Ran'}...</td>\n",
       "      <td>&lt;p&gt;Inflammasome complexes are pivotal in the i...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.09.14.460408</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>Synthetic lethality-based prediction of anti-S...</td>\n",
       "      <td>[{'name': 'Lipika R. Pal'}, {'name': 'Kuoyuan ...</td>\n",
       "      <td>&lt;p&gt;Novel strategies are needed to identify dru...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.09.14.460206</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Network visualisation of synthetic biology des...</td>\n",
       "      <td>[{'name': 'Matthew Crowther'}, {'name': 'Anil ...</td>\n",
       "      <td>&lt;label&gt;1&lt;/label&gt;&lt;title&gt;ABSTRACT&lt;/title&gt;&lt;p&gt;Visu...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.09.14.460283</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Multiplexed profiling of kinase interactomes q...</td>\n",
       "      <td>[{'name': 'Martin Golkowski'}, {'name': 'Andre...</td>\n",
       "      <td>&lt;p&gt;Cells utilize protein-protein interaction (...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.09.11.21263146</td>\n",
       "      <td>14</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Psychiatry and Clinical Psychology</td>\n",
       "      <td>False</td>\n",
       "      <td>Causal effect of C-reactive protein and vitami...</td>\n",
       "      <td>[{'name': 'Dylan J. Kiltschewskij'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Psychiatric disorders such as schizophrenia...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.09.14.460226</td>\n",
       "      <td>13</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Enhanced Specificity Mutations Perturb Alloste...</td>\n",
       "      <td>[{'name': 'ukasz Nierzwicki'}, {'name': 'Kyle...</td>\n",
       "      <td>&lt;p&gt;CRISPR-Cas9 is a molecular tool with transf...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.09.13.460076</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Elicitation of potent SARS-CoV-2 neutralizing ...</td>\n",
       "      <td>[{'name': 'C Chevillard'}, {'name': 'A Amen'},...</td>\n",
       "      <td>&lt;p&gt;The severe acute respiratory syndrome coron...</td>\n",
       "      <td>2021-09-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.09.15.460484</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Hippocampus maintains a flexible and coherent ...</td>\n",
       "      <td>[{'name': 'Indrajith R Nair'}, {'name': 'Dipan...</td>\n",
       "      <td>&lt;p&gt;Animals predominantly use salient visual cu...</td>\n",
       "      <td>2021-09-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.09.14.460211</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>True</td>\n",
       "      <td>Mitoxantrone dihydrochloride, an FDA approved ...</td>\n",
       "      <td>[{'name': 'Prateek Kumar'}, {'name': 'Taniya B...</td>\n",
       "      <td>&lt;p&gt;One of the major virulence factors of SARS-...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.09.15.460445</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Physical geography, isolation by distance and ...</td>\n",
       "      <td>[{'name': 'Che-Wei Chang'}, {'name': 'Eyal Fri...</td>\n",
       "      <td>&lt;p&gt;Determining the extent of genetic variation...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.09.14.460110</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Synthetic amyloid beta does not induce a robus...</td>\n",
       "      <td>[{'name': 'I.Y. Quiroga'}, {'name': 'A.E. Crui...</td>\n",
       "      <td>&lt;p&gt;Alzheimers disease (AD) is a progressive n...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.09.15.460093</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Structural Basis of Substrate Recognition by t...</td>\n",
       "      <td>[{'name': 'Shihao Yao'}, {'name': 'Qiuzi Yi'},...</td>\n",
       "      <td>&lt;p&gt;Specific import of ADP and export of ATP by...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.09.15.459215</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>Signatures of adaptive evolution during human ...</td>\n",
       "      <td>[{'name': 'Jui-Hung Tai'}, {'name': 'Shu-Miaw ...</td>\n",
       "      <td>&lt;p&gt;One of the unique features of SARS-CoV-2 is...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.09.13.459529</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Germline Missense Variants in CDC20 Result in ...</td>\n",
       "      <td>[{'name': 'Owen J. Chen'}, {'name': 'Ester Cas...</td>\n",
       "      <td>&lt;p&gt;CDC20 is a co-activator of the anaphase pro...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.09.13.459918</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Impaired KDM2B-mediated PRC1 recruitment to ch...</td>\n",
       "      <td>[{'name': 'Yuen Gao'}, {'name': 'Natalia Duque...</td>\n",
       "      <td>&lt;p&gt;Autism spectrum disorder (ASD) and intellec...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.09.13.460150</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Gsx2 but not Gsx1 is necessary for early foreb...</td>\n",
       "      <td>[{'name': 'RA Coltogirone'}, {'name': 'EI Sher...</td>\n",
       "      <td>&lt;p&gt;Central nervous system (CNS) development is...</td>\n",
       "      <td>2021-09-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.09.14.460297</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Physiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Mitochondrial ATP production is required for e...</td>\n",
       "      <td>[{'name': 'Calum Wilson'}, {'name': 'Matthew D...</td>\n",
       "      <td>&lt;p&gt;Arteries and veins are lined by non-prolife...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.09.14.460322</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Variance of allele balance calculated from low...</td>\n",
       "      <td>[{'name': 'Kyle Fletcher'}, {'name': 'Rongkui ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Motivation&lt;/title&gt;&lt;p&gt;Polyploidy an...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.09.14.460311</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Elevated phosphorylation of EGFR in NSCLC due ...</td>\n",
       "      <td>[{'name': 'Matthew R. Swiatnicki'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;The role of EGFR in lung cancer is well des...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.09.16.460651</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Unlabeled salivary gland organoids have distin...</td>\n",
       "      <td>[{'name': 'Kate Tubbesing'}, {'name': 'Nichola...</td>\n",
       "      <td>&lt;p&gt;Organoids are self-organized three-dimensio...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.09.18.460647</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Diet-derived metabolites and mucus link the gu...</td>\n",
       "      <td>[{'name': 'Zaker Schwabkey'}, {'name': 'Diana ...</td>\n",
       "      <td>&lt;p&gt;Fever in the setting of low white blood cel...</td>\n",
       "      <td>2021-09-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.09.09.21263026</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>The clinically extremely vulnerable to COVID: ...</td>\n",
       "      <td>[{'name': 'Jessica E. Butler'}, {'name': 'Mint...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;In March 2020...</td>\n",
       "      <td>2021-09-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.09.14.460362</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Testing for heterogeneous rates of discrete ch...</td>\n",
       "      <td>[{'name': 'Liam J. Revell'}, {'name': 'Klaus P...</td>\n",
       "      <td>&lt;p&gt;Many hypotheses in the field of phylogeneti...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.09.12.460001</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>DeepREAL: A Deep Learning Powered Multi-scale ...</td>\n",
       "      <td>[{'name': 'Tian Cai'}, {'name': 'Kyra Alyssa A...</td>\n",
       "      <td>&lt;p&gt;Drug discovery has witnessed intensive expl...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.09.14.460257</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Chronic sodium bromide treatment relieves auti...</td>\n",
       "      <td>[{'name': 'Ccile Derieux'}, {'name': 'Audrey ...</td>\n",
       "      <td>&lt;p&gt;Autism Spectrum Disorders (ASD) are neurode...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.09.14.460235</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Paramagnetic spin labeling of a bacterial DnaB...</td>\n",
       "      <td>[{'name': 'Johannes Zehnder'}, {'name': 'Ricca...</td>\n",
       "      <td>&lt;p&gt;Labeling of biomolecules with a paramagneti...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.09.16.460530</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Genome-wide identification of genes important ...</td>\n",
       "      <td>[{'name': 'Tyler C. Helmann'}, {'name': 'Melan...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Dickeya&lt;/italic&gt; species are causal...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.09.13.459937</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Amelioration of L-DOPA-induced dyskinesia with...</td>\n",
       "      <td>[{'name': 'Adedamola Aminat Bayo-Olugbami'}, {...</td>\n",
       "      <td>&lt;p&gt;L-DOPA Induced Dyskinesia (LID) is associat...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.09.10.21263383</td>\n",
       "      <td>5</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Social-economic drivers overwhelm climate in u...</td>\n",
       "      <td>[{'name': 'Zhenghua Liu'}]</td>\n",
       "      <td>&lt;p&gt;Identifying the drivers underlying the spat...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.09.12.459767</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Drought stress increases the expression of bar...</td>\n",
       "      <td>[{'name': 'Daniel J. Leybourne'}, {'name': 'Tr...</td>\n",
       "      <td>&lt;p&gt;Crops are exposed to myriad abiotic and bio...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.09.13.21263371</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Spread of SARS-CoV-2 Delta variant infections ...</td>\n",
       "      <td>[{'name': 'Laura Verdurme'}, {'name': 'Gonch ...</td>\n",
       "      <td>&lt;p&gt;Analysing 92,598 variant screening tests pe...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.09.14.460215</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Paleontology</td>\n",
       "      <td>False</td>\n",
       "      <td>Dinosaur bonebed amber from an original swamp ...</td>\n",
       "      <td>[{'name': 'Sergio lvarez-Parra'}, {'name': 'R...</td>\n",
       "      <td>&lt;p&gt;Dinosaur bonebeds with amber content, yet s...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.09.15.460086</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Little hope for the polyploid endemic Pyrenean...</td>\n",
       "      <td>[{'name': 'Pascaline Salvado'}, {'name': 'Pere...</td>\n",
       "      <td>&lt;p&gt;Species endemic to restricted geographical ...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.09.16.460688</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Comparative transcriptomics of seven Impatiens...</td>\n",
       "      <td>[{'name': 'Mria urinov'}, {'name': 'tepn ...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Impatiens&lt;/italic&gt; is a genus conta...</td>\n",
       "      <td>2021-09-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.09.13.460151</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Allosteric modulation of the adenosine A2A rec...</td>\n",
       "      <td>[{'name': 'Shuya Kate Huang'}, {'name': 'Omar ...</td>\n",
       "      <td>&lt;p&gt;Cholesterol is a major component of the cel...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.09.13.460193</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Genomic and physiological characterization of ...</td>\n",
       "      <td>[{'name': 'Aline Belmok'}, {'name': 'Felipe Ma...</td>\n",
       "      <td>&lt;p&gt;A novel bacterial strain, designated GeG2&lt;s...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.09.14.460204</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Controls on the isotopic composition of microb...</td>\n",
       "      <td>[{'name': 'Jonathan Gropp'}, {'name': 'Qusheng...</td>\n",
       "      <td>&lt;p&gt;Microbial methane production (methanogenesi...</td>\n",
       "      <td>2021-09-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.09.14.460205</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Forkhead transcription factor FKH-8 is a maste...</td>\n",
       "      <td>[{'name': 'Rebeca Brocal-Ruiz'}, {'name': 'Ain...</td>\n",
       "      <td>&lt;p&gt;Cilia, either motile or non-motile (a.k.a p...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.09.11.21256274</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Health Policy</td>\n",
       "      <td>False</td>\n",
       "      <td>The Morbidity Costs of Air Pollution through t...</td>\n",
       "      <td>[{'name': 'Xin Zhang'}, {'name': 'Xun Zhang'},...</td>\n",
       "      <td>&lt;p&gt;This study offers one of the first causal e...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.09.17.460803</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>CUX2 mutations in temporal lobe epilepsy; incr...</td>\n",
       "      <td>[{'name': 'Toshimitsu Suzuki'}, {'name': 'Tets...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;CUX2&lt;/italic&gt; gene encodes a transc...</td>\n",
       "      <td>2021-09-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.09.13.460067</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Mitochondrial dynamics regulate genome stabili...</td>\n",
       "      <td>[{'name': 'Kai Cao'}, {'name': 'Joel S Riley'}...</td>\n",
       "      <td>&lt;p&gt;Mitochondrial dysfunction is interconnected...</td>\n",
       "      <td>2021-09-13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.09.13.460068</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The Paradox of the Plankton: Coexistence of St...</td>\n",
       "      <td>[{'name': 'Alberto Scarampi'}]</td>\n",
       "      <td>&lt;p&gt;In the framework of resource-competition mo...</td>\n",
       "      <td>2021-09-15</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.09.14.460138      39  bioRxiv   \n",
       "1     10.1101/2021.09.13.460111      37  bioRxiv   \n",
       "2   10.1101/2021.09.09.21263331      34  medRxiv   \n",
       "3     10.1101/2021.09.13.460168      32  bioRxiv   \n",
       "4   10.1101/2021.09.01.21262607      31  medRxiv   \n",
       "5   10.1101/2021.09.13.21263499      30  medRxiv   \n",
       "6     10.1101/2021.09.15.460550      23  bioRxiv   \n",
       "7     10.1101/2021.09.15.460516      21  bioRxiv   \n",
       "8     10.1101/2021.09.14.460331      16  bioRxiv   \n",
       "9     10.1101/2021.09.14.460408      15  bioRxiv   \n",
       "10    10.1101/2021.09.14.460206      15  bioRxiv   \n",
       "11    10.1101/2021.09.14.460283      14  bioRxiv   \n",
       "12  10.1101/2021.09.11.21263146      14  medRxiv   \n",
       "13    10.1101/2021.09.14.460226      13  bioRxiv   \n",
       "14    10.1101/2021.09.13.460076      12  bioRxiv   \n",
       "15    10.1101/2021.09.15.460484      12  bioRxiv   \n",
       "16    10.1101/2021.09.14.460211      11  bioRxiv   \n",
       "17    10.1101/2021.09.15.460445       9  bioRxiv   \n",
       "18    10.1101/2021.09.14.460110       9  bioRxiv   \n",
       "19    10.1101/2021.09.15.460093       9  bioRxiv   \n",
       "20    10.1101/2021.09.15.459215       9  bioRxiv   \n",
       "21    10.1101/2021.09.13.459529       8  bioRxiv   \n",
       "22    10.1101/2021.09.13.459918       8  bioRxiv   \n",
       "23    10.1101/2021.09.13.460150       8  bioRxiv   \n",
       "24    10.1101/2021.09.14.460297       7  bioRxiv   \n",
       "25    10.1101/2021.09.14.460322       7  bioRxiv   \n",
       "26    10.1101/2021.09.14.460311       7  bioRxiv   \n",
       "27    10.1101/2021.09.16.460651       7  bioRxiv   \n",
       "28    10.1101/2021.09.18.460647       6  bioRxiv   \n",
       "29  10.1101/2021.09.09.21263026       6  medRxiv   \n",
       "30    10.1101/2021.09.14.460362       6  bioRxiv   \n",
       "31    10.1101/2021.09.12.460001       6  bioRxiv   \n",
       "32    10.1101/2021.09.14.460257       5  bioRxiv   \n",
       "33    10.1101/2021.09.14.460235       5  bioRxiv   \n",
       "34    10.1101/2021.09.16.460530       5  bioRxiv   \n",
       "35    10.1101/2021.09.13.459937       5  bioRxiv   \n",
       "36  10.1101/2021.09.10.21263383       5  medRxiv   \n",
       "37    10.1101/2021.09.12.459767       5  bioRxiv   \n",
       "38  10.1101/2021.09.13.21263371       4  medRxiv   \n",
       "39    10.1101/2021.09.14.460215       4  bioRxiv   \n",
       "40    10.1101/2021.09.15.460086       4  bioRxiv   \n",
       "41    10.1101/2021.09.16.460688       4  bioRxiv   \n",
       "42    10.1101/2021.09.13.460151       4  bioRxiv   \n",
       "43    10.1101/2021.09.13.460193       4  bioRxiv   \n",
       "44    10.1101/2021.09.14.460204       4  bioRxiv   \n",
       "45    10.1101/2021.09.14.460205       4  bioRxiv   \n",
       "46  10.1101/2021.09.11.21256274       4  medRxiv   \n",
       "47    10.1101/2021.09.17.460803       4  bioRxiv   \n",
       "48    10.1101/2021.09.13.460067       4  bioRxiv   \n",
       "49    10.1101/2021.09.13.460068       4  bioRxiv   \n",
       "\n",
       "                          subject-area  covid  \\\n",
       "0                         Biochemistry  False   \n",
       "1                         Microbiology   True   \n",
       "2                         Epidemiology   True   \n",
       "3                       Bioinformatics  False   \n",
       "4                   Health Informatics  False   \n",
       "5                         Epidemiology  False   \n",
       "6                         Neuroscience  False   \n",
       "7                         Neuroscience  False   \n",
       "8                           Immunology  False   \n",
       "9                      Systems Biology   True   \n",
       "10                   Synthetic Biology  False   \n",
       "11                        Biochemistry  False   \n",
       "12  Psychiatry and Clinical Psychology  False   \n",
       "13                          Biophysics  False   \n",
       "14                        Microbiology   True   \n",
       "15                        Neuroscience  False   \n",
       "16                          Biophysics   True   \n",
       "17                Evolutionary Biology  False   \n",
       "18                        Neuroscience  False   \n",
       "19                          Biophysics  False   \n",
       "20                Evolutionary Biology   True   \n",
       "21                      Cancer Biology  False   \n",
       "22                        Neuroscience  False   \n",
       "23               Developmental Biology  False   \n",
       "24                          Physiology  False   \n",
       "25                      Bioinformatics  False   \n",
       "26                      Cancer Biology  False   \n",
       "27                          Biophysics  False   \n",
       "28                        Microbiology  False   \n",
       "29                        Epidemiology   True   \n",
       "30                Evolutionary Biology  False   \n",
       "31                      Bioinformatics  False   \n",
       "32                        Neuroscience  False   \n",
       "33                          Biophysics  False   \n",
       "34                        Microbiology  False   \n",
       "35                        Neuroscience  False   \n",
       "36                        Epidemiology   True   \n",
       "37                       Plant Biology  False   \n",
       "38                        Epidemiology   True   \n",
       "39                        Paleontology  False   \n",
       "40                Evolutionary Biology  False   \n",
       "41                       Plant Biology  False   \n",
       "42                          Biophysics  False   \n",
       "43                        Microbiology  False   \n",
       "44                        Microbiology  False   \n",
       "45               Developmental Biology  False   \n",
       "46                       Health Policy  False   \n",
       "47                        Neuroscience  False   \n",
       "48                        Cell Biology  False   \n",
       "49                     Systems Biology  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Structure of a bacterial Rhs effector exported...   \n",
       "1   Therapeutic efficacy of an oral nucleoside ana...   \n",
       "2   Second wave of the Covid-19 pandemic in Delhi,...   \n",
       "3   Random Forest Factorization Reveals Latent Str...   \n",
       "4   ADataViewer: Exploring Semantically Harmonized...   \n",
       "5   The natural history of TB disease-a synthesis ...   \n",
       "6   Intact reading ability in spite of a spatially...   \n",
       "7   Long-term Motor Learning in the Wild with High...   \n",
       "8   Defective endosome-TGN retrograde transport pr...   \n",
       "9   Synthetic lethality-based prediction of anti-S...   \n",
       "10  Network visualisation of synthetic biology des...   \n",
       "11  Multiplexed profiling of kinase interactomes q...   \n",
       "12  Causal effect of C-reactive protein and vitami...   \n",
       "13  Enhanced Specificity Mutations Perturb Alloste...   \n",
       "14  Elicitation of potent SARS-CoV-2 neutralizing ...   \n",
       "15  Hippocampus maintains a flexible and coherent ...   \n",
       "16  Mitoxantrone dihydrochloride, an FDA approved ...   \n",
       "17  Physical geography, isolation by distance and ...   \n",
       "18  Synthetic amyloid beta does not induce a robus...   \n",
       "19  Structural Basis of Substrate Recognition by t...   \n",
       "20  Signatures of adaptive evolution during human ...   \n",
       "21  Germline Missense Variants in CDC20 Result in ...   \n",
       "22  Impaired KDM2B-mediated PRC1 recruitment to ch...   \n",
       "23  Gsx2 but not Gsx1 is necessary for early foreb...   \n",
       "24  Mitochondrial ATP production is required for e...   \n",
       "25  Variance of allele balance calculated from low...   \n",
       "26  Elevated phosphorylation of EGFR in NSCLC due ...   \n",
       "27  Unlabeled salivary gland organoids have distin...   \n",
       "28  Diet-derived metabolites and mucus link the gu...   \n",
       "29  The clinically extremely vulnerable to COVID: ...   \n",
       "30  Testing for heterogeneous rates of discrete ch...   \n",
       "31  DeepREAL: A Deep Learning Powered Multi-scale ...   \n",
       "32  Chronic sodium bromide treatment relieves auti...   \n",
       "33  Paramagnetic spin labeling of a bacterial DnaB...   \n",
       "34  Genome-wide identification of genes important ...   \n",
       "35  Amelioration of L-DOPA-induced dyskinesia with...   \n",
       "36  Social-economic drivers overwhelm climate in u...   \n",
       "37  Drought stress increases the expression of bar...   \n",
       "38  Spread of SARS-CoV-2 Delta variant infections ...   \n",
       "39  Dinosaur bonebed amber from an original swamp ...   \n",
       "40  Little hope for the polyploid endemic Pyrenean...   \n",
       "41  Comparative transcriptomics of seven Impatiens...   \n",
       "42  Allosteric modulation of the adenosine A2A rec...   \n",
       "43  Genomic and physiological characterization of ...   \n",
       "44  Controls on the isotopic composition of microb...   \n",
       "45  Forkhead transcription factor FKH-8 is a maste...   \n",
       "46  The Morbidity Costs of Air Pollution through t...   \n",
       "47  CUX2 mutations in temporal lobe epilepsy; incr...   \n",
       "48  Mitochondrial dynamics regulate genome stabili...   \n",
       "49  The Paradox of the Plankton: Coexistence of St...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Patrick Gnther'}, {'name': 'Dennis...   \n",
       "1   [{'name': 'Alexandra Schafer'}, {'name': 'Davi...   \n",
       "2   [{'name': 'Nandini Sharma'}, {'name': 'Pragya ...   \n",
       "3   [{'name': 'Boris M. Brenerman'}, {'name': 'Ben...   \n",
       "4   [{'name': 'Yasamin Salimi'}, {'name': 'Daniel ...   \n",
       "5   [{'name': 'Alexandra S Richards'}, {'name': 'B...   \n",
       "6   [{'name': 'Jin Li'}, {'name': 'Evelina Fedoren...   \n",
       "7   [{'name': 'Jennifer B Listman'}, {'name': 'Jon...   \n",
       "8   [{'name': 'Zhirong Zhang'}, {'name': 'Li Ran'}...   \n",
       "9   [{'name': 'Lipika R. Pal'}, {'name': 'Kuoyuan ...   \n",
       "10  [{'name': 'Matthew Crowther'}, {'name': 'Anil ...   \n",
       "11  [{'name': 'Martin Golkowski'}, {'name': 'Andre...   \n",
       "12  [{'name': 'Dylan J. Kiltschewskij'}, {'name': ...   \n",
       "13  [{'name': 'ukasz Nierzwicki'}, {'name': 'Kyle...   \n",
       "14  [{'name': 'C Chevillard'}, {'name': 'A Amen'},...   \n",
       "15  [{'name': 'Indrajith R Nair'}, {'name': 'Dipan...   \n",
       "16  [{'name': 'Prateek Kumar'}, {'name': 'Taniya B...   \n",
       "17  [{'name': 'Che-Wei Chang'}, {'name': 'Eyal Fri...   \n",
       "18  [{'name': 'I.Y. Quiroga'}, {'name': 'A.E. Crui...   \n",
       "19  [{'name': 'Shihao Yao'}, {'name': 'Qiuzi Yi'},...   \n",
       "20  [{'name': 'Jui-Hung Tai'}, {'name': 'Shu-Miaw ...   \n",
       "21  [{'name': 'Owen J. Chen'}, {'name': 'Ester Cas...   \n",
       "22  [{'name': 'Yuen Gao'}, {'name': 'Natalia Duque...   \n",
       "23  [{'name': 'RA Coltogirone'}, {'name': 'EI Sher...   \n",
       "24  [{'name': 'Calum Wilson'}, {'name': 'Matthew D...   \n",
       "25  [{'name': 'Kyle Fletcher'}, {'name': 'Rongkui ...   \n",
       "26  [{'name': 'Matthew R. Swiatnicki'}, {'name': '...   \n",
       "27  [{'name': 'Kate Tubbesing'}, {'name': 'Nichola...   \n",
       "28  [{'name': 'Zaker Schwabkey'}, {'name': 'Diana ...   \n",
       "29  [{'name': 'Jessica E. Butler'}, {'name': 'Mint...   \n",
       "30  [{'name': 'Liam J. Revell'}, {'name': 'Klaus P...   \n",
       "31  [{'name': 'Tian Cai'}, {'name': 'Kyra Alyssa A...   \n",
       "32  [{'name': 'Ccile Derieux'}, {'name': 'Audrey ...   \n",
       "33  [{'name': 'Johannes Zehnder'}, {'name': 'Ricca...   \n",
       "34  [{'name': 'Tyler C. Helmann'}, {'name': 'Melan...   \n",
       "35  [{'name': 'Adedamola Aminat Bayo-Olugbami'}, {...   \n",
       "36                         [{'name': 'Zhenghua Liu'}]   \n",
       "37  [{'name': 'Daniel J. Leybourne'}, {'name': 'Tr...   \n",
       "38  [{'name': 'Laura Verdurme'}, {'name': 'Gonch ...   \n",
       "39  [{'name': 'Sergio lvarez-Parra'}, {'name': 'R...   \n",
       "40  [{'name': 'Pascaline Salvado'}, {'name': 'Pere...   \n",
       "41  [{'name': 'Mria urinov'}, {'name': 'tepn ...   \n",
       "42  [{'name': 'Shuya Kate Huang'}, {'name': 'Omar ...   \n",
       "43  [{'name': 'Aline Belmok'}, {'name': 'Felipe Ma...   \n",
       "44  [{'name': 'Jonathan Gropp'}, {'name': 'Qusheng...   \n",
       "45  [{'name': 'Rebeca Brocal-Ruiz'}, {'name': 'Ain...   \n",
       "46  [{'name': 'Xin Zhang'}, {'name': 'Xun Zhang'},...   \n",
       "47  [{'name': 'Toshimitsu Suzuki'}, {'name': 'Tets...   \n",
       "48  [{'name': 'Kai Cao'}, {'name': 'Joel S Riley'}...   \n",
       "49                     [{'name': 'Alberto Scarampi'}]   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>The type VI <underline>s</underline>ecretio...  2021-09-14  \n",
       "1   <p>The COVID-19 pandemic remains uncontrolled ...  2021-09-13  \n",
       "2   <sec><title>Background</title><p>We report the...  2021-09-13  \n",
       "3   <p>Single-cell RNA sequencing data contain pat...  2021-09-14  \n",
       "4   <sec><title>INTRODUCTION</title><p>Currently, ...  2021-09-14  \n",
       "5   <sec><title>Background</title><p>Prevalence su...  2021-09-16  \n",
       "6   <p>The visual word form area (VWFA) is an expe...  2021-09-16  \n",
       "7   <p>Motor learning occurs over long periods of ...  2021-09-17  \n",
       "8   <p>Inflammasome complexes are pivotal in the i...  2021-09-16  \n",
       "9   <p>Novel strategies are needed to identify dru...  2021-09-15  \n",
       "10  <label>1</label><title>ABSTRACT</title><p>Visu...  2021-09-14  \n",
       "11  <p>Cells utilize protein-protein interaction (...  2021-09-15  \n",
       "12  <p>Psychiatric disorders such as schizophrenia...  2021-09-15  \n",
       "13  <p>CRISPR-Cas9 is a molecular tool with transf...  2021-09-14  \n",
       "14  <p>The severe acute respiratory syndrome coron...  2021-09-13  \n",
       "15  <p>Animals predominantly use salient visual cu...  2021-09-17  \n",
       "16  <p>One of the major virulence factors of SARS-...  2021-09-14  \n",
       "17  <p>Determining the extent of genetic variation...  2021-09-16  \n",
       "18  <p>Alzheimers disease (AD) is a progressive n...  2021-09-15  \n",
       "19  <p>Specific import of ADP and export of ATP by...  2021-09-15  \n",
       "20  <p>One of the unique features of SARS-CoV-2 is...  2021-09-16  \n",
       "21  <p>CDC20 is a co-activator of the anaphase pro...  2021-09-14  \n",
       "22  <p>Autism spectrum disorder (ASD) and intellec...  2021-09-15  \n",
       "23  <p>Central nervous system (CNS) development is...  2021-09-13  \n",
       "24  <p>Arteries and veins are lined by non-prolife...  2021-09-16  \n",
       "25  <sec><title>Motivation</title><p>Polyploidy an...  2021-09-14  \n",
       "26  <p>The role of EGFR in lung cancer is well des...  2021-09-14  \n",
       "27  <p>Organoids are self-organized three-dimensio...  2021-09-16  \n",
       "28  <p>Fever in the setting of low white blood cel...  2021-09-18  \n",
       "29  <sec><title>Background</title><p>In March 2020...  2021-09-13  \n",
       "30  <p>Many hypotheses in the field of phylogeneti...  2021-09-16  \n",
       "31  <p>Drug discovery has witnessed intensive expl...  2021-09-15  \n",
       "32  <p>Autism Spectrum Disorders (ASD) are neurode...  2021-09-16  \n",
       "33  <p>Labeling of biomolecules with a paramagneti...  2021-09-14  \n",
       "34  <p><italic>Dickeya</italic> species are causal...  2021-09-16  \n",
       "35  <p>L-DOPA Induced Dyskinesia (LID) is associat...  2021-09-15  \n",
       "36  <p>Identifying the drivers underlying the spat...  2021-09-15  \n",
       "37  <p>Crops are exposed to myriad abiotic and bio...  2021-09-15  \n",
       "38  <p>Analysing 92,598 variant screening tests pe...  2021-09-16  \n",
       "39  <p>Dinosaur bonebeds with amber content, yet s...  2021-09-16  \n",
       "40  <p>Species endemic to restricted geographical ...  2021-09-16  \n",
       "41  <p><italic>Impatiens</italic> is a genus conta...  2021-09-16  \n",
       "42  <p>Cholesterol is a major component of the cel...  2021-09-14  \n",
       "43  <p>A novel bacterial strain, designated GeG2<s...  2021-09-15  \n",
       "44  <p>Microbial methane production (methanogenesi...  2021-09-14  \n",
       "45  <p>Cilia, either motile or non-motile (a.k.a p...  2021-09-15  \n",
       "46  <p>This study offers one of the first causal e...  2021-09-15  \n",
       "47  <p><italic>CUX2</italic> gene encodes a transc...  2021-09-18  \n",
       "48  <p>Mitochondrial dysfunction is interconnected...  2021-09-13  \n",
       "49  <p>In the framework of resource-competition mo...  2021-09-15  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "63 preprints (including 11 covering SARS-CoV-2, 54 from bioRxiv and 9 from medRxiv) published in the last 7 days before 2021-09-20 had been tweeted at least 3 times (maximum 39)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
