{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "13300 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=16c3dcc4-1a9d-45c6-8b3d-b9541a8e005a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=dd28598a-ae9c-4306-bcb7-069e34a99f3c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=fdc677aa-6692-4e14-a987-28d7c7df2f51&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=6f07fd18-a3d6-4b66-8f77-bd7ac26ac2c5&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=795582ef-2df6-4a29-a211-a95d3a99d4ca&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=169c58f9-6e4f-435b-8e4d-16acd592d3f6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7fdb3277-5b84-4977-89ad-1c66b93180a1&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=858bdd81-426b-4df4-8f62-e43776c00ad7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9e2cd948-0f9a-4afc-9225-30ae45ce3477&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2f21ad3d-4a89-42ff-ba38-56e41a2fb373&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=8e046611-7792-4731-b043-00a46f6c164f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a52c158b-cdf4-483a-90ac-0560331ceb47&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=63bb5194-8237-4f4b-9a70-4154a5229499&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-01&until-occurred-date=2021-11-08\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-10-18.csv\n",
      "failed to load preprint_tweets_2021-10-25.csv\n",
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load preprint_tweets_2021-10-23.csv\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-11-01.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.09.20143164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.21261741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20222778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.12.149203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/530097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.440722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.434193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.382143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467435...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.465965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.466924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.21262114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.441537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.465409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.420448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275569.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.466369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273300.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.446360...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.466897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466434...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460480...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/240317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467534...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.465610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.28.923532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.423933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466512...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.464583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.428294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466884...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.459575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466024...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.21265382...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.09.20184143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.444558...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.185181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466953...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467109...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.20.106575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.25.266619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.466508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21257910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21258385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.07.083444...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.367649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.27.437353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.06.467543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458411...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.413609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.465169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466411...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.466530...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.463794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21261811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.06.467542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265094...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.11.02.466666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466347...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275989.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.466696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.21265424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.412098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.260109...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467434...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.466744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.443778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.25.20026369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261955...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.462913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.163816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.14.20062463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452365...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.11.244996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.24.21253992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.03.131573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467060...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20213546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466589...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467513...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.03.20167395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20131243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467344...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.11.439329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265945...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20201848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/833400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.444847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457547...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.21254952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260836...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.213074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.465996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.17.452128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.465122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.413229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.451245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466659...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.11.451967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.28.21262763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21263507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.438428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.462574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466454...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.466970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.20142703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.11.439311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466346...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.11.378133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.441323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.20172288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467149...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/765685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.457921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.21.885863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275868.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262382...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.21257899...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.20.052019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.01.406223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466695...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.437652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.465895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.21263548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.415505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.08.372995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.365932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.08.371179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.20204693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467506...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456961...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20088260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.06.467539...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275308.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.271908.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.186684.114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/865931...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457357...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/238527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20124461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459904...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452700...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467033...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.07.20145979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467191...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.206680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.440682...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.21265662...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.441482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.425896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.21255937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.465104...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.26.424423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.21249603...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467332...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.438084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.451635...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.23.20200006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.459925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21264623...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.22.427830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.21250505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.460970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.20086439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.21256049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.235549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261039...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467127...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.439788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.14.21251715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.174383...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.466494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.02.280040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.30.272237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.455272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.21265430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.28.176024...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.225854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467100...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.28.458013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/768663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/743294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.389031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.385401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21258418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/592725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.17.20037689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435343...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435700...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.19.423614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275648.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.434387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.461806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268581.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465870...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445986...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.07.467592...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.462716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.06.467582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.21265729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.465885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.23.449550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.459429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.435034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20087965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.11.902726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268037.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.439509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.21254026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.438521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047860...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273631.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461681...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467322...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467445...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.465983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.23.056309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.21261496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467190...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.21262409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.439482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.043224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.463867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467119...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.21.21251403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.11.20129007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265476...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.23.917211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.21254494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467339...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.05.21262643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/276717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.11.934471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.27.968354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.17.20133504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.447957...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466907...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.371609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.460601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258640...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.20231746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.426266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466939...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21256632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.13.20211813...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.19.20214494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.21249563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265826...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422826...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466837...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.20248876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257337...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.444292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.462738...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.17.386995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.388108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.09.443096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.465785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.463467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.399154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.466906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466526...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.28.272534...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.02.20186734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466444...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.28.270306...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.20202267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.29.293274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21260184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467039...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.466767...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.11.05.467528', 'tweets': 307, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Large-scale discovery of recombinases for integrating DNA into the human genome', 'authors': \"[{'name': 'Matthew G Durrant'}, {'name': 'Alison Fanton'}, {'name': 'Josh Tycko'}, {'name': 'Michaela Hinks'}, {'name': 'Sita Chandrasekaran'}, {'name': 'Nicholas T Perry'}, {'name': 'Julia Schaepe'}, {'name': 'Peter P Du'}, {'name': 'Lacramioara Bintu'}, {'name': 'Michael C Bassik'}, {'name': 'Ami S Bhatt'}, {'name': 'Patrick D. Hsu'}]\", 'abstract': '<p>Recent microbial genome sequencing efforts have revealed a vast reservoir of mobile genetic elements containing integrases that could be useful genome engineering tools. Large serine recombinases (LSRs), such as Bxb1 and PhiC31, are bacteriophage-encoded integrases that can facilitate the insertion of phage DNA into bacterial genomes. However, only a few LSRs have been previously characterized and they have limited efficiency in human cells. Here, we developed a systematic computational discovery workflow that searches across the bacterial tree of life to expand the diversity of known LSRs and their cognate DNA attachment sites by &gt;100-fold. We validated this approach via experimental characterization of LSRs, leading to three classes of LSRs distinguished from one another by their efficiency and specificity. We identify landing pad LSRs that efficiently integrate into native attachment sites in a human cell context, human genome-targeting LSRs with computationally predictable pseudosites, and multi-targeting LSRs that can unidirectionally integrate cargos with similar efficiency and superior specificity to commonly used transposases. LSRs from each category were functionally characterized in human cells, overall achieving up to 7-fold higher plasmid recombination than Bxb1 and genome insertion efficiencies of 40-70% with cargo sizes over 7 kb. Overall, we establish a paradigm for the large-scale discovery of microbial recombinases directly from sequencing data and the reconstruction of their target sites. This strategy provided a rich resource of over 60 experimentally characterized LSRs that can function in human cells and thousands of additional candidates for large-payload genome editing without double-stranded DNA breaks.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.10.31.21265703', 'tweets': 176, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Efficacy and safety of SOBERANA 02, a COVID-19 conjugate vaccine in heterologous three-dose combination', 'authors': \"[{'name': 'Maria Eugenia Toledo-Romani'}, {'name': 'Mayra Garcia-Carmenate'}, {'name': 'Carmen Valenzuela Silva'}, {'name': 'Waldemar Baldoquin-Rodriguez'}, {'name': 'Marisel Martinez Perez'}, {'name': 'Meiby C Rodriguez Gonzalez'}, {'name': 'Beatriz Paredes Moreno'}, {'name': 'Ivis Mendoza Hernandez'}, {'name': 'Raul Gonzalez-Mujica Romero'}, {'name': 'Oscar Samon Tabio'}, {'name': 'Pablo Velazco Villares'}, {'name': 'Juan Pablo Bacallao Castillo'}, {'name': 'Ernesto Licea Martin'}, {'name': 'Misladys Rodriguez Ortega,'}, {'name': 'Nuris Liem Herrera Marrero'}, {'name': 'Esperanza Caballero Gonzalez'}, {'name': 'Liudmila Ibelin Egues Torres'}, {'name': 'Reinaldo Duarte Gonzalez'}, {'name': 'Serguey Garcia Blanco'}, {'name': 'Suzette Perez Cabrera'}, {'name': 'Santos Huete Ferreira'}, {'name': 'Kirenia Idalmis Idalmis Cisnero'}, {'name': 'Omaida Fonte Galindo'}, {'name': 'Dania Melia Perez'}, {'name': 'Ivonne Rojas Remedios'}, {'name': 'Sonsire Fernandez Castillo'}, {'name': 'Yanet Climent Ruiz'}, {'name': 'Yury Valdes Balbin'}, {'name': 'Dagmar Garcia Rivera'}, {'name': 'vicente Verez-Bencomo'}]\", 'abstract': '<p>Background: SOBERANA 02 is a COVID19 conjugate vaccine (recombinant RBD\\nconjugated to tetanus toxoid). Phase 1 and 2 clinical trials demonstrated its high\\nimmunogenicity, promoting neutralizing IgG and specific Tcell response. A third dose, of\\nSOBERANA Plus (RBDdimer), further increased the specific anti RBD neutralizing\\nantibodies.\\nMethods: In a randomized, double-blind, placebo controlled, phase 3 trial\\n(https://rpcec.sld.cu/trials/RPCEC00000354-En) we randomly assigned 44 031 participants,\\naged 19 to 80 years to three groups to receive a) two doses, SOBERANA 02, or b) two doses,\\nSOBERANA 02 and a third dose, SOBERANA Plus, or c) placebo. Study endpoints are\\nvaccine efficacy (VE) evaluated through confirmed symptomatic COVID-19 and safety.\\nDuring the trial, the SARS CoV-2 isolates in Havana were 74.0 BETA;, shifting gradually to\\n100% DELTA; .\\nResults: Two doses of SOBERANA 02 protected against symptomatic COVID-19: 43\\ncases in the two-dose group (14 371) vs. 155 in the placebo group (14 403), VE 71.0%,\\nadjusted (CI 95%, 58.9 to 79.1). The heterologous three-dose combination increased the\\nprotection: 15 cases in the vaccine groups (13 833) vs. 155 in the placebo group (14 303),\\nVE 92.4%, adjusted (CI 95%, 86.9 to 95.6%). For the two-dose schedule, VE against severe\\nCOVID-19 was 63.0% and for death, 59.0%; for the heterologous three-dose schedule, VE\\nwas 100% in both cases.\\nConclusions: This is the first phase-3 study of a three-dose, heterologous COVID-19\\nvaccine. Two doses of SOBERANA 02 were safe and attained efficacy of 71.0% in the\\nadult population 19-80 y/o; incorporating SOBERANA Plus increased efficacy from 71.0\\n% to 92.4%.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.467152', 'tweets': 146, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'DeepBacs: Bacterial image analysis using open-source deep learning approaches', 'authors': \"[{'name': 'Christoph Spahn'}, {'name': 'Romain F. Laine'}, {'name': 'Pedro Matos Pereira'}, {'name': 'Estibaliz Gómez-de-Mariscal'}, {'name': 'Lucas von Chamier'}, {'name': 'Mia Conduit'}, {'name': 'Mariana Gomes de Pinho'}, {'name': 'Guillaume Jacquemet'}, {'name': 'Séamus Holden'}, {'name': 'Mike Heilemann'}, {'name': 'Ricardo Henriques'}]\", 'abstract': '<p>Deep Learning (DL) is rapidly changing the field of microscopy, allowing for efficient analysis of complex data while often out-performing classical algorithms. This revolution has led to a significant effort to create user-friendly tools allowing biomedical researchers with little background in computer sciences to use this technology effectively. Thus far, these approaches have mainly focused on analysing microscopy images from eukaryotic samples and are still underused in microbiology. In this work, we demonstrate how to use a range of state-of-the-art artificial neural-networks particularly suited for the analysis of bacterial microscopy images, using our recently developed ZeroCostDL4Mic platform. We showcase different DL approaches for segmenting bright field and fluorescence images of different bacterial species, use object detection to classify different growth stages in time-lapse imaging data, and carry out DL-assisted phenotypic profiling of antibiotic-treated cells. To also demonstrate the DL capacity to enhance low-phototoxicity live-cell microscopy, we showcase how image denoising can allow researchers to attain high-fidelity data in faster and longer imaging. Finally, artificial labelling of cell membranes and predictions of super-resolution images allow for accurate mapping of cell shape and intracellular targets. To aid in the training of novice users, we provide a purposefully-built database of training and testing data, enabling bacteriologists to quickly explore how to analyse their data through DL. We hope this lays a fertile ground for the efficient application of DL in microbiology and fosters the creation of novel tools for bacterial cell biology and antibiotic research.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466936', 'tweets': 113, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Bright, fluorogenic and photostable avidity probes for RNA imaging', 'authors': \"[{'name': 'Bastian Bühler'}, {'name': 'Anja Benderoth'}, {'name': 'Daniel Englert'}, {'name': 'Franziska Grün'}, {'name': 'Janin Schokolowski'}, {'name': 'Andres Jäschke'}, {'name': 'Murat Sunbul'}]\", 'abstract': '<p>Fluorescent light-up aptamers (FLAPs) emerged as valuable tools to visualize RNA, but are mostly limited by poor brightness, low photostability and high fluorescence background. In this study, we combine bivalent (silicon) rhodamine fluorophores with dimeric FLAPs to yield bright and photostable complexes with low picomolar dissociation constants. Our avidity-based approach resulted in extreme binding strength and high fluorogenicity, enabling single mRNA tracking in living cells.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.02.466983', 'tweets': 95, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Structure Determination of Inactive-State GPCRs with a Universal Nanobody', 'authors': \"[{'name': 'Michael J. Robertson'}, {'name': 'Feng He'}, {'name': 'Justin G. Meyerowitz'}, {'name': 'Alpay B. Seven'}, {'name': 'Ouliana Panova'}, {'name': 'Maria-Claudia Peroto'}, {'name': 'Tao Che'}, {'name': 'Georgios Skiniotis'}]\", 'abstract': '<p>Cryogenic electron microscopy (cryo-EM) has widened the field of structure-based drug discovery by allowing for routine determination of membrane protein structures previously intractable. However, despite representing one of the largest classes of therapeutic targets, most inactive-state G protein-coupled receptors (GPCRs) have remained inaccessible for cryo-EM because their small size and membrane-embedded nature impedes projection alignment for high-resolution map reconstructions. Here we demonstrate that a camelid single-chain antibody (nanobody) recognizing a grafted intracellular loop can be used to obtain cryo-EM structures of different inactive-state GPCRs at resolutions comparable or better than those obtained by X-ray crystallography. Using this approach, we obtained the structure of human neurotensin 1 receptor (NTSR1) bound to antagonist SR48692, of μ-opioid receptor (MOR) bound to the clinical antagonist alvimopan, as well as the structure of the previously uncharacterized somatostatin receptor 2 (SSTR2) in the apo state; each of these structures yields novel insights into ligand binding and specificity. We expect this rapid, straightforward approach to facilitate the broad structural exploration of GPCR inactive states without the need for extensive engineering and crystallization.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466765', 'tweets': 88, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Global patterns and rates of habitat transitions across the eukaryotic tree of life', 'authors': \"[{'name': 'Mahwash Jamy'}, {'name': 'Charlie Biwer'}, {'name': 'Daniel Vaulot'}, {'name': 'Aleix Obiol'}, {'name': 'Homgmei Jing'}, {'name': 'Sari Peura'}, {'name': 'Ramon Massana'}, {'name': 'Fabien Burki'}]\", 'abstract': '<p>The successful colonisation of new habitats has played a fundamental role during the evolution of life. Salinity is one of the strongest barriers for organisms to cross, which has resulted in the evolution of distinct marine and terrestrial (including both freshwater and soil) communities. Although microbes represent by far the vast majority of eukaryote diversity, the role of the salt barrier in shaping the diversity across the eukaryotic tree is poorly known. Traditional views suggest rare and ancient marine-terrestrial transitions, but this view is being challenged by the discovery of several recently transitioned lineages. Here, we investigate habitat evolution across the tree of eukaryotes using a unique set of taxon-rich environmental phylogenies inferred from a combination of long-read and short-read metabarcoding data spanning the ribosomal DNA operon. Our results show that overall marine and terrestrial microbial communities are phylogenetically distinct, but transitions have occurred in both directions in almost all major eukaryotic lineages, with at least 350 transition events detected. Some groups have experienced relatively high rates of transitions, most notably fungi for which crossing the salt barrier has most likely been an important aspect of their successful diversification. At the deepest phylogenetic levels, ancestral habitat reconstruction analyses suggest that eukaryotes may have first evolved in non-saline habitats, and that the two largest known eukaryotic assemblages (TSAR and Amorphea) arose in different habitats. Overall, our findings indicate that crossing the salt barrier has played an important role in eukaryotic evolution by providing new ecological niches to fill.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.467131', 'tweets': 68, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'A molecular network of conserved factors keeps ribosomes dormant in the egg', 'authors': \"[{'name': 'Friederike Leesch'}, {'name': 'Laura Lorenzo-Orts'}, {'name': 'Carina Pribitzer'}, {'name': 'Irina Grishkovskaya'}, {'name': 'Manuel Matzinger'}, {'name': 'Elisabeth Roitinger'}, {'name': 'Katarina Belačić'}, {'name': 'Susanne Kandolf'}, {'name': 'Tzi-Yang Lin'}, {'name': 'Karl Mechtler'}, {'name': 'Anton Meinhart'}, {'name': 'David Haselbach'}, {'name': 'Andrea Pauli'}]\", 'abstract': '<p>Ribosomes are produced in large quantities during oogenesis and stored in the egg. However, the egg and early embryo are translationally repressed. Using mass-spectrometry and cryo-EM analyses of ribosomes isolated from zebrafish and <italic>Xenopus</italic> eggs and embryos, we provide molecular evidence that ribosomes transition from a dormant to an active state during the first hours of embryogenesis. Dormant ribosomes are associated with four conserved factors that form two modules and occupy functionally important sites of the ribosome: a Habp4-eEF2 module that stabilizes ribosome levels and a Dap1b/Dapl1-eIF5a module that represses translation. Dap1b/Dapl1 is a newly discovered translational inhibitor that stably inserts into the polypeptide exit tunnel. Thus, a developmentally programmed, conserved ribosome state plays a key role in ribosome storage and translational repression in the egg.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.04.467077', 'tweets': 68, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'The oral protease inhibitor (PF-07321332) protects Syrian hamsters against infection with SARS-CoV-2 variants of concern', 'authors': \"[{'name': 'Rana Abdelnabi'}, {'name': 'Caroline Shi-Yan Foo'}, {'name': 'Dirk Jochmans'}, {'name': 'Laura Vangeel'}, {'name': 'Steven De Jonghe'}, {'name': 'Patrick Augustijns'}, {'name': 'Raf Mols'}, {'name': 'Birgit Weynand'}, {'name': 'Thanaporn Wattanakul'}, {'name': 'Richard Hoglund'}, {'name': 'Joel Tarning'}, {'name': 'Charles Mowbray'}, {'name': 'Peter Sjo'}, {'name': 'Fanny Escudie'}, {'name': 'Ivan Scandale'}, {'name': 'Eric Chatelain'}, {'name': 'Johan Neyts'}]\", 'abstract': '<p>There is an urgent need for potent and selective antivirals against SARS-CoV-2. Pfizer developed PF-07321332 (PF-332), a potent inhibitor of the main viral protease (Mpro, 3CLpro) that can be dosed orally; the compound is in clinical development. We demonstrate that PF-332 exerts equipotent in vitro activity against the four SARS-CoV-2 variants of concerns (VoC) and can completely arrest replication of the alpha variant in primary human airway epithelial cells grown at the air-liquid interface. Treatment of Syrian hamsters with PF-332 (250 mg/kg, twice daily) completely protected the animals against intranasal infection with the beta (B.1.351) and delta (B.1.617.2) SARS-CoV-2 variants. Moreover, treatment of SARS-CoV-2 (B.1.617.2) infected animals with PF-332 completely prevented transmission to untreated co-housed sentinels. The trough drug concentration at this efficacious dose were above the in vitro efficacious concentrations.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.02.466915', 'tweets': 58, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'RBBP6 activates the pre-mRNA 3’-end processing machinery in humans', 'authors': \"[{'name': 'Vytaute Boreikaite'}, {'name': 'Thomas Elliott'}, {'name': 'Jason Chin'}, {'name': 'Lori A Passmore'}]\", 'abstract': '<p>3’-end processing of most human mRNAs is carried out by the cleavage and polyadenylation specificity factor (CPSF; CPF in yeast). Endonucleolytic cleavage of the nascent pre-mRNA defines the 3’-end of the mature transcript, which is important for mRNA localization, translation and stability. Cleavage must therefore be tightly regulated. Here, we reconstitute specific and efficient 3’-endonuclease activity of human CPSF with purified proteins. This requires the sevensubunit CPSF as well as three additional protein factors: cleavage stimulatory factor (CStF), cleavage factor IIm (CFIIm) and, importantly, the multi-domain protein RBBP6. Unlike its yeast homologue Mpe1, which is a stable subunit of CPF, RBBP6 does not copurify with CPSF and is recruited in an RNA-dependent manner. Sequence and mutational analyses suggest that RBBP6 interacts with the WDR33 and CPSF73 subunits of CPSF. Thus, it is likely that the role of RBBP6 is conserved from yeast to human. Overall, our data are consistent with CPSF endonuclease activation and site-specific pre-mRNA cleavage being highly controlled to maintain fidelity in RNA processing.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.03.467079', 'tweets': 53, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Parent-of-origin effects in the UK Biobank', 'authors': \"[{'name': 'Robin J Hofmeister'}, {'name': 'Simone Rubinacci'}, {'name': 'Diogo M Ribeiro'}, {'name': 'Zoltan Kutalik'}, {'name': 'Alfonso Buil'}, {'name': 'Olivier Delaneau'}]\", 'abstract': '<p>Identical genetic variations can have different phenotypic effects depending on their parent of origin (PofO). Yet, studies focussing on PofO effects have been largely limited in terms of sample size due to the need of parental genomes or known genealogies. Here, we used a novel probabilistic approach to infer PofO of individual alleles in the UK Biobank that does not require parental genomes nor prior knowledge of genealogy. Our model uses Identity-By-Descent (IBD) sharing with second- and third-degree relatives to assign alleles to parental groups and leverages chromosome X data in males to distinguish maternal from paternal groups. When combined with robust haplotype inference and haploid imputation, this allowed us to infer the PofO at 5.4 million variants genome-wide for 26,393 UK Biobank individuals. We used this large dataset to systematically screen 59 biomarkers and 38 anthropomorphic phenotypes for PofO effects and discovered 101 significant associations, demonstrating that this type of effects contributes to the genetics of complex traits. Notably, we retrieved well known PofO effects, such as the MEG3/DLK1 locus on platelet count, and we discovered many new ones at loci often unsuspected of being imprinted and, in some cases, previously thought to harbour additive associations.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.03.467097', 'tweets': 44, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A drug repurposing screen identifies altiratinib as a selective inhibitor of a key regulatory splicing kinase and a potential therapeutic for toxoplasmosis and malaria', 'authors': \"[{'name': 'Christopher Swale'}, {'name': 'Valeria Bellini'}, {'name': 'Matthew W. Bowler'}, {'name': 'Nardella Flore'}, {'name': 'Marie-Pierre Brenier-Pinchart'}, {'name': 'Dominique Cannella'}, {'name': 'Lucid Belmudes'}, {'name': 'Caroline Mas'}, {'name': 'Yohann Couté'}, {'name': 'Fabrice Laurent'}, {'name': 'Artur Scherf'}, {'name': 'Alexandre Bougdour'}, {'name': 'Mohamed-Ali Hakimi'}]\", 'abstract': '<p>The apicomplexa comprise a large phylum of single-celled, obligate intracellular protozoa that infect humans and animals and cause severe parasitic diseases. Available therapeutics against these devastating diseases are limited by suboptimal efficacy and frequent side effects, as well as the emergence and spread of resistance. Here, we use a drug repurposing strategy and identify altiratinib, a compound originally developed to treat glioblastoma, as a promising drug candidate with broad spectrum activity against apicomplexans. Altiratinib is parasiticidal and blocks the development of intracellular zoites in the nanomolar range and with a high selectivity index. We have identified <italic>Tg</italic>PRP4K of <italic>T. gondii</italic> as the primary target of altiratinib by genetic target deconvolution, highlighting key residues within the kinase catalytic site that, when mutated, confer resistance to the drug. We have further elucidated the molecular basis of the inhibitory mechanism and species selectivity of altiratinib for <italic>Tg</italic>PRP4K as well as for its <italic>P. falciparum</italic> counterpart <italic>Pf</italic>CLK3. Our data also point to structural features critical for binding of the other <italic>Pf</italic>CLK3 inhibitor, TCMDC-135051. Consistent with the role of this kinase family in splicing in a broad spectrum of eukaryotes, we have shown that altiratinib causes global disruption of splicing, primarily through intron retention in both <italic>T. gondii</italic> and <italic>P. falciparum</italic>. Thus, our data establish parasitic PRP4K/CLK3 as a promising pan-apicomplexan target whose repertoire of inhibitors can be expanded by the addition of altiratinib.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466728', 'tweets': 43, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Haldane’s cost of selection imposes a mild constraint on adaptation, with a high proportion of deaths in A. thaliana being selective', 'authors': \"[{'name': 'Joseph Matheson'}, {'name': 'Moises Exposito-Alonso'}, {'name': 'Joanna Masel'}]\", 'abstract': '<p>Haldane’s Dilemma refers to the concern that the need for many “selective deaths” to complete a substitution creates a speed limit to adaptation. However, discussion of this concern has been marked by confusion over which features of substitutions produce limits to the speed of adaptation, what those limits are, and the consequences of violating speed limits. The term ‘substitution load’ has been particularly unhelpful in this regard. Here we distinguish different lines of reasoning that lead to speed limits, including one line of reasoning which has not yet been fully addressed. We then apply these lines of reasoning to a dataset measuring survival and fecundity of 517 different genotypes of <italic>Arabidopsis thaliana</italic> grown in eight different environmental conditions. We estimate highly permissive limits to the speed of adaptation in all environmental conditions. We also estimate that much higher proportions of deaths contribute to adaptation than were anticipated during historical discussions of speed limits.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.05.467435', 'tweets': 43, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Ancient genomes from the last three millennia support multiple human dispersals into Wallacea', 'authors': '[{\\'name\\': \\'Sandra Oliveira\\'}, {\\'name\\': \\'Kathrin Nägele\\'}, {\\'name\\': \\'Selina Carlhoff\\'}, {\\'name\\': \\'Irina Pugach\\'}, {\\'name\\': \\'Toetik Koesbardiati\\'}, {\\'name\\': \\'Alexander Hübner\\'}, {\\'name\\': \\'Matthias Meyer\\'}, {\\'name\\': \\'Adhi Agus Oktaviana\\'}, {\\'name\\': \\'Masami Takenaka\\'}, {\\'name\\': \\'Chiaki Katagiri\\'}, {\\'name\\': \\'Delta Bayu Murti\\'}, {\\'name\\': \\'Rizky Sugianto Putri\\'}, {\\'name\\': \\' Mahirta\\'}, {\\'name\\': \\'Thomas Higham\\'}, {\\'name\\': \\'Charles F W Higham\\'}, {\\'name\\': \"Sue O\\'Connor\"}, {\\'name\\': \\'Stuart Hawkins\\'}, {\\'name\\': \\'Rebecca Kinaston\\'}, {\\'name\\': \\'Peter Bellwood\\'}, {\\'name\\': \\'Rintaro Ono\\'}, {\\'name\\': \\'Adam Powell\\'}, {\\'name\\': \\'Johannes Krause\\'}, {\\'name\\': \\'Cosimo Posth\\'}, {\\'name\\': \\'Mark Stoneking\\'}]', 'abstract': \"<p>Previous research indicates that the human genetic diversity found in Wallacea - islands in present-day Eastern Indonesia and Timor-Leste that were never part of the Sunda or Sahul continental shelves - has been shaped by complex interactions between migrating Austronesian farmers and indigenous hunter-gatherer communities. Here, we provide new insights into this region's demographic history based on genome-wide data from 16 ancient individuals (2600-250 yrs BP) from islands of the North Moluccas, Sulawesi, and East Nusa Tenggara. While the ancestry of individuals from the northern islands fit earlier views of contact between groups related to the Austronesian expansion and the first colonization of Sahul, the ancestry of individuals from the southern islands revealed additional contributions from Mainland Southeast Asia, which seems to predate the Austronesian admixture in the region. Admixture time estimates for the oldest individuals of Wallacea are closer to archaeological estimates for the Austronesian arrival into the region than are admixture time estimates for present-day groups. The decreasing trend in admixture times exhibited by younger individuals supports a scenario of multiple or continuous admixture involving Papuan- and Asian-related groups. Our results clarify previously debated times of admixture and suggest that the Neolithic dispersals into Island Southeast Asia are associated with the spread of multiple genetic ancestries.</p>\", 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.01.466817', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'A powerful and versatile new fixation protocol for immunohistology and in situ hybridization that preserves delicate tissues in planaria', 'authors': \"[{'name': 'Carlos Guerrero-Hernández'}, {'name': 'Viraj Doddihal'}, {'name': 'Frederick G. Mann'}, {'name': 'Alejandro Sánchez Alvarado'}]\", 'abstract': '<p>Whole-mount <italic>in situ</italic> hybridization (WISH) is a powerful and widely used technique to visualize the expression pattern of genes in different biological systems. Here we describe a new protocol for ISH and immunostaining in the planarian <italic>Schmidtea mediterranea</italic>. The new Nitric Acid/Formic Acid (NAFA) protocol is compatible with both assays and prevents degradation of the epidermis or blastema. Instead of proteinase K digestion, formic acid treatment is used to permeabilize tissues and preserve antigen epitopes. We show that the NAFA protocol successfully permits development of chromogenic and fluorescent signals <italic>in situ</italic>, while preserving the anatomy of the animal. Further, the immunostaining of different proteins was compatible with the NAFA protocol following fluorescent <italic>in situ</italic> hybridization. Finally, we demonstrate with high resolution confocal imaging that the regeneration blastema is preserved when using the new method. This new NAFA protocol will be a valuable technique to study the process of wounding response and regeneration.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.466924', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Organization and composition of apicomplexan kinetochores reveal plasticity in chromosome segregation across parasite modes of division', 'authors': \"[{'name': 'Lorenzo Brusini'}, {'name': 'Nicolas Dos Santos Pacheco'}, {'name': 'Dominique Soldati-Favre'}, {'name': 'Mathieu Brochet'}]\", 'abstract': '<p>Kinetochores are multiprotein assemblies directing mitotic spindle attachment and chromosome segregation. In apicomplexan parasites, most known kinetochore components and associated regulators are apparently missing, suggesting a minimal structure with limited control over chromosome segregation. In this study, we use interactomics combined with deep homology searches to identify six divergent eukaryotic components, in addition to a set of eight apicomplexan kinetochore proteins (AKiTs) that bear no detectable sequence similarity to known proteins. The nanoscale organization of the apicomplexan kinetochore includes four subdomains, each displaying different evolutionary rates across the phylum. Functional analyses confirm AKiTs are essential for mitosis and reveal architectures parallel to biorientation at metaphase. Furthermore, we identify a homolog of MAD1 at the apicomplexan kinetochore, suggesting conserved spindle assembly checkpoint signaling. Finally, we show unexpected plasticity in kinetochore composition and segregation throughout the parasite lifecycle, indicating diverse requirements to maintain fidelity of chromosome segregation across apicomplexan modes of division.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.465409', 'tweets': 35, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Structural insights into human excitatory amino acid transporter EAAT2', 'authors': \"[{'name': 'Takafumi Kato'}, {'name': 'Tsukasa Kusakizako'}, {'name': 'Chunhuan Jin'}, {'name': 'LiLi Quan'}, {'name': 'Ryuichi Ohgaki'}, {'name': 'Suguru Okuda'}, {'name': 'Kan Kobayashi'}, {'name': 'Keitaro Yamashita'}, {'name': 'Tomohiro Nishizawa'}, {'name': 'Yoshikatsu Kanai'}, {'name': 'Osamu Nureki'}]\", 'abstract': '<p>Glutamate is a pivotal excitatory neurotransmitter in mammalian brains, but excessive glutamate causes numerous neural disorders. Almost all extracellular glutamate is retrieved by the glial transporter, Excitatory Amino Acid Transporter 2 (EAAT2), belonging to the SLC1A family. However, in some cancers, EAAT2 expression is enhanced and causes resistance to therapies by metabolic disturbance. Despite its crucial roles, the detailed structural information about EAAT2 has not been available. Here, we report cryo-EM structures of human EAAT2 in substrate-free and selective inhibitor WAY213613-bound states. EAAT2 forms a trimer, with each protomer consisting of transport and scaffold domains. Along with a glutamate-binding site, the transport domain possesses a cavity, that could be disrupted during the transport cycle. WAY213613 occupies both the glutamate-binding site and cavity of EAAT2 to interfere with its alternating access, where the sensitivity is defined by the inner environment of the cavity. This is the first characterization of molecular features of EAAT2 and the selective inhibition mechanism, underlying structure-based drug design for EAAT2.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.467002', 'tweets': 34, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Detection of velogenic avian paramyxoviruses in rock doves in New York City, New York', 'authors': \"[{'name': 'Isabel Francisco'}, {'name': 'Shatoni Bailey'}, {'name': 'Teresa Bautista'}, {'name': 'Djenabou Diallo'}, {'name': 'Jesus Gonzalez'}, {'name': 'Joel Gonzalez'}, {'name': 'Ericka Kirkpatrick Roubidoux'}, {'name': 'Paul K. Ajayi'}, {'name': 'Randy A. Albrecht'}, {'name': 'Rita McMahon'}, {'name': 'Florian Krammer'}, {'name': 'Christine Marizzi'}]\", 'abstract': '<p>Avian paramyxovirus 1 (APMV-1), also known as Newcastle disease virus (NDV), causes severe and economically important disease in poultry around the globe. Although a limited amount of APMV-1 strains in urban areas have been characterized, the role of the urban wild bird population as an APMV-1 reservoir is unclear. Since urban birds may have an important role for long-term circulation of the virus, fecal and swab samples were collected by community scientists from wild birds in New York City (NYC), New York, United States. These samples were screened for APMV-1 and genotypically characterized by sequencing of the complete genome. A total of 885 samples were collected from NYC parks and from a local wildlife rehabilitation clinic from October 2020 through June 2021. Eight birds (1.1 %) screened positive for the APMV-1 nucleoprotein gene by conventional reverse transcription polymerase chain reaction (RT-PCR), and two live viruses were isolated via egg culture. The F protein cleavage site, an indicator of pathogenicity, was present in the two samples fully sequenced by next generation sequencing, and positioned <sup>112</sup>R R K K R F<sup>117</sup>. Phylogenetic analysis of the F gene coding sequence classified both isolates into genotype VI, a diverse and predominant genotype responsible for NDV outbreaks in pigeon and dove species worldwide.</p><sec><title>Importance</title><p>Here we describe the first large-scale effort to screen for APMV-1 in New York City’s wild bird population as part of the New York City Virus Hunters program, a community science initiative. We have characterized two isolates of NDV, with phylogenetic analyses suggesting diversity in established and circulating strains of pigeon paramyxoviruses. Our isolates are also domestic reference strains for future APMV-1 vaccine developments. Future surveillance in this region may contribute to our understanding of NDV’s evolution and genetic diversity, as well as inform poultry husbandry and vaccination practices in New York State.</p></sec>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.10.29.466453', 'tweets': 33, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Invariant neural subspaces maintained by feedback modulation', 'authors': \"[{'name': 'Laura Bella Naumann'}, {'name': 'Joram Keijser'}, {'name': 'Henning Sprekeler'}]\", 'abstract': '<p>Sensory systems reliably process incoming stimuli in spite of changes in context. Most recent models accredit this context invariance to an extraction of increasingly complex sensory features in hierarchical feedforward networks. Here, we study how context-invariant representations can be established by feedback rather than feedforward processing. We show that feedforward neural networks modulated by feedback can dynamically generate invariant sensory representations. The required feedback can be implemented as a slow and spatially diffuse gain modulation. The invariance is not present on the level of individual neurons, but emerges only on the population level. Mechanistically, the feedback modulation dynamically reorients the manifold of neural activity and thereby maintains an invariant neural subspace in spite of contextual variations. Our results highlight the importance of population-level analyses for understanding the role of feedback in flexible sensory processing.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.466369', 'tweets': 31, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structure of mammalian V-ATPase with the TLDc domain protein mEAK7 bound', 'authors': \"[{'name': 'Yong Zi Tan'}, {'name': 'Yazan M. Abbas'}, {'name': 'Jing Ze Wu'}, {'name': 'Di Wu'}, {'name': 'Geoffrey G. Hesketh'}, {'name': 'Stephanie A. Bueler'}, {'name': 'Anne-Claude Gingras'}, {'name': 'Carol V. Robinson'}, {'name': 'Sergio Grinstein'}, {'name': 'John L. Rubinstein'}]\", 'abstract': '<p>V-ATPases are rotary proton pumps that serve as signaling hubs, with numerous proposed binding partners in cells. We used cryoEM to detect endogenous proteins that associate with V-ATPase from porcine kidney. A super-stoichiometric copy of subunit C was found in ∼3% of complexes, while an additional ∼1.6% of complexes bound mEAK7, a protein with proposed roles in dauer formation in nematodes and mTOR signaling in mammals. High-resolution cryoEM of porcine kidney V-ATPase with recombinant mEAK7 shows that mEAK7’s TLDc domain, which is found in other proteins proposed to bind V-ATPase, interacts with V-ATPase’s stator while its C-terminal α helix binds V-ATPase’s rotor. This crosslink would be expected to inhibit rotary catalysis. However, exogenous mEAK7 does not inhibit purified V-ATPase activity and mEAK7 overexpression in cells does not alter lysosomal or phagosomal pH. Instead, cryoEM suggests that interaction of mEAK7 with V-ATPase is disrupted by ATP-induced rotation of the rotor. Together, these results reveal how TLDc domains bind V-ATPases and suggest that V-ATPase binding proteins can form labile interactions that are sensitive to the enzyme’s activity.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.05.467437', 'tweets': 30, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'PHOSPHATE STARVATION RESPONSE enables arbuscular mycorrhiza symbiosis', 'authors': \"[{'name': 'Debatosh Das'}, {'name': 'Michael Paries'}, {'name': 'Karen Hobecker'}, {'name': 'Michael Gigl'}, {'name': 'Corinna Dawid'}, {'name': 'Hon-Ming Lam'}, {'name': 'Jianhua Zhang'}, {'name': 'Moxian Chen'}, {'name': 'Caroline Gutjahr'}]\", 'abstract': '<p>Arbuscular mycorrhiza (AM) is a widespread symbiosis between roots of the majority of land plants and Glomeromycotina fungi. AM is important for ecosystem health and functioning as the fungi critically support plant performance by providing essential mineral nutrients, particularly the poorly accessible phosphate, in exchange for organic carbon. AM fungi colonize the inside of roots and this is promoted at low but inhibited at high plant phosphate status, while the mechanistic basis for this phosphate-dependence remained obscure. Here we demonstrate that a major transcriptional regulator of phosphate starvation responses in rice PHOSPHATE STARVATION RESPONSE 2 (PHR2) regulates AM. Root colonization of phr2 mutants is drastically reduced, and PHR2 is required for root colonization, mycorrhizal phosphate uptake, and yield increase in field soil. PHR2 promotes AM by targeting genes required for pre-contact signaling, root colonization, and AM function. Thus, this important symbiosis is directly wired to the PHR2-controlled plant phosphate starvation response.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.04.466897', 'tweets': 30, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genome-wide association analyses of individual differences in quantitatively assessed reading- and language-related skills in up to 34,000 people', 'authors': \"[{'name': 'Else Eising'}, {'name': 'Nazanin Mirza-Schreiber'}, {'name': 'Eveline L de Zeeuw'}, {'name': 'Carol A Wang'}, {'name': 'Dongnhu T Truong'}, {'name': 'Andrea G Allegrini'}, {'name': 'Chin Yang Shapland'}, {'name': 'Gu Zhu'}, {'name': 'Karen G Wigg'}, {'name': 'Margot Gerritse'}, {'name': 'Barbara Molz'}, {'name': 'Gokberk Alagoz'}, {'name': 'Alessandro Gialluisi'}, {'name': 'Filippo Abbondanza'}, {'name': 'Kaili Rimfeld'}, {'name': 'Marjolein MJ Van Donkelaar'}, {'name': 'Zhijie Liao'}, {'name': 'Philip R Jansen'}, {'name': 'Till F M Andlauer'}, {'name': 'Timothy C Bates'}, {'name': 'Manon Bernard'}, {'name': 'Kirsten Blokland'}, {'name': 'Anders D Borglum'}, {'name': 'Thomas Bourgeron'}, {'name': 'Daniel Brandeis'}, {'name': 'Fabiola Ceroni'}, {'name': 'Philip S Dale'}, {'name': 'Peter F. de Jong'}, {'name': 'John C DeFries'}, {'name': 'Ditte Demontis'}, {'name': 'Yu Feng'}, {'name': 'Scott D. Gordon'}, {'name': 'Sharon L Guger'}, {'name': 'Marianna E Hayiou-Thomas'}, {'name': 'Juan A. Hernandez-Cabrera'}, {'name': 'Jouke- Jan Hottenga'}, {'name': 'Charles Hulme'}, {'name': 'Elizabeth N Kerr'}, {'name': 'Tanner Koomar'}, {'name': 'Karin Landerl'}, {'name': 'Maureen W Lovett'}, {'name': 'Heikki Lyytinen'}, {'name': 'Nicholas G. Martin'}, {'name': 'Angela Martinelli'}, {'name': 'Urs Maurer'}, {'name': 'Jacob J Michaelson'}, {'name': 'Kristina Moll'}, {'name': 'Antony P. Monaco'}, {'name': 'Angela T. Morgan'}, {'name': 'Markus M. Noethen'}, {'name': 'Zdenka Pausova'}, {'name': 'Craig E Pennell'}, {'name': 'Bruce Pennington'}, {'name': 'Kaitlyn M Price'}, {'name': 'Veera M. Rajagopal'}, {'name': 'Franck Ramus'}, {'name': 'Louis Richer'}, {'name': 'Nuala H Simpson'}, {'name': 'Shelley Smith'}, {'name': 'Margaret J. Snowling'}, {'name': 'John Stein'}, {'name': 'Lisa J Strug'}, {'name': 'Joel B Talcott'}, {'name': 'Henning Tiemeier'}, {'name': 'Marc MP van der Schroeff'}, {'name': 'Ellen Verhoef'}, {'name': 'Kate E Watkins'}, {'name': 'Margaret Wilkinson'}, {'name': 'Margaret J. Wright'}, {'name': 'Cathy L Barr'}, {'name': 'Dorret I Boomsma'}, {'name': 'Manuel Carreiras'}, {'name': 'Marie-Christine J Franken'}, {'name': 'Jeffrey R Gruen'}, {'name': 'Michelle Luciano'}, {'name': 'Bertram Mueller-Myhsok'}, {'name': 'Dianne F Newbury'}, {'name': 'Richard K Olson'}, {'name': 'Silvia Paracchini'}, {'name': 'Tomas Paus'}, {'name': 'Robert Plomin'}, {'name': 'Sheena Reilly'}, {'name': 'Gerd Schulte-Koerne'}, {'name': 'James B Tomblin'}, {'name': 'Elsje van Bergen'}, {'name': 'Andrew JO Whitehouse'}, {'name': 'Erik G. Willcutt'}, {'name': 'Beate St Pourcain'}, {'name': 'Clyde Francks'}, {'name': 'Simon E. Fisher'}]\", 'abstract': '<p>The use of spoken and written language is a capacity that is unique to humans. Individual differences in reading- and language-related skills are influenced by genetic variation, with twin-based heritability estimates of 30-80%, depending on the trait. The relevant genetic architecture is complex, heterogeneous, and multifactorial, and yet to be investigated with well-powered studies. Here, we present a multicohort genome-wide association study (GWAS) of five traits assessed individually using psychometric measures: word reading, nonword reading, spelling, phoneme awareness, and nonword repetition, with total sample sizes ranging from 13,633 to 33,959 participants aged 5-26 years (12,411 to 27,180 for those with European ancestry, defined by principal component analyses). We identified a genome-wide significant association with word reading (rs11208009, p=1.098 x 10-8) independent of known loci associated with intelligence or educational attainment. All five reading-/language-related traits had robust SNP-heritability estimates (0.13-0.26), and genetic correlations between them were modest to high. Using genomic structural equation modelling, we found evidence for a shared genetic factor explaining the majority of variation in word and nonword reading, spelling, and phoneme awareness, which only partially overlapped with genetic variation contributing to nonword repetition, intelligence and educational attainment. A multivariate GWAS was performed to jointly analyse word and nonword reading, spelling, and phoneme awareness, maximizing power for follow-up investigation. Genetic correlation analysis of multivariate GWAS results with neuroimaging traits identified association with cortical surface area of the banks of the left superior temporal sulcus, a brain region with known links to processing of spoken and written language. Analysis of evolutionary annotations on the lineage that led to modern humans showed enriched heritability in regions depleted of Neanderthal variants. Together, these results provide new avenues for deciphering the biological underpinnings of these uniquely human traits.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.10.29.466434', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Multilevel neural gradients reflect transdiagnostic effects of major psychiatric conditions on cortical morphology', 'authors': \"[{'name': 'Bo-yong Park'}, {'name': 'Valeria Kebets'}, {'name': 'Sara Larivière'}, {'name': 'Meike D. Hettwer'}, {'name': 'Casey Paquola'}, {'name': 'Daan van Rooij'}, {'name': 'Jan Buitelaar'}, {'name': 'Barbara Franke'}, {'name': 'Martine Hoogman'}, {'name': 'Lianne Schmaal'}, {'name': 'Dick J. Veltman'}, {'name': 'Odile van den Heuvel'}, {'name': 'Dan J. Stein'}, {'name': 'Ole A. Andreassen'}, {'name': 'Christopher R. K. Ching'}, {'name': 'Jessica Turner'}, {'name': 'Theo G. M. van Erp'}, {'name': 'Alan C. Evans'}, {'name': 'Alain Dagher'}, {'name': 'Sophia I. Thomopoulos'}, {'name': 'Paul M. Thompson'}, {'name': 'Sofie L. Valk'}, {'name': 'Matthias Kirschner'}, {'name': 'Boris C. Bernhardt'}]\", 'abstract': '<p>It is increasingly recognized that multiple psychiatric conditions are underpinned by shared neural pathways, affecting similar brain systems. Here, we assessed <italic>i)</italic> shared dimensions of alterations in cortical morphology across six major psychiatric conditions (autism spectrum disorder, attention deficit/hyperactivity disorder, major depression, obsessive-compulsive disorder, bipolar disorder, schizophrenia) and <italic>ii)</italic> carried out a multiscale neural contextualization, by cross-referencing shared anomalies against cortical myeloarchitecture and cytoarchitecture, as well as connectome and neurotransmitter organization. Pooling disease-related effects on MRI-based cortical thickness measures across six ENIGMA working groups, including a total of 28,546 participants (12,876 patients and 15,670 controls), we computed a shared disease dimension on cortical morphology using principal component analysis that described a sensory-fugal pattern with paralimbic regions showing the most consistent abnormalities across conditions. The shared disease dimension was closely related to cortical gradients of microstructure and intrinsic connectivity, as well as neurotransmitter systems, specifically serotonin and dopamine. Our findings embed the shared effects of major psychiatric conditions on brain structure in multiple scales of brain organization and may provide novel insights into neural mechanisms into transdiagnostic vulnerability.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.05.467534', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Kaptive 2.0: updated capsule and LPS locus typing for the Klebsiella pneumoniae species complex', 'authors': \"[{'name': 'Margaret M.C. Lam'}, {'name': 'Ryan R. Wick'}, {'name': 'Kathryn E. Holt'}, {'name': 'Kelly L Wyres'}]\", 'abstract': '<p>The outer polysaccharide capsule and lipopolysaccharide antigens are key targets for novel control strategies targeting Klebsiella pneumoniae and related taxa from the K. pneumoniae species complex (KpSC), including vaccines, phage and monoclonal antibody therapies. Given the importance and growing interest in these highly diverse surface antigens, we had previously developed Kaptive, a tool for rapidly identifying and typing capsule (K) and outer lipopolysaccharide (O) loci from whole genome sequence data. Here, we report two significant updates, now freely available in Kaptive 2.0 (github.com/katholt/kaptive); i) the addition of 16 novel K locus sequences to the K locus reference database following an extensive search of &gt;17,000 KpSC genomes; and ii) enhanced O locus typing to enable prediction of the clinically relevant O2 antigen (sub)types, for which the genetic determinants have been recently described. We applied Kaptive 2.0 to a curated dataset of &gt;12,000 public KpSC genomes to explore for the first time the distribution of predicted O (sub)types across species, sampling niches and clones, which highlighted key differences in the distributions that warrant further investigation. As the uptake of genomic surveillance approaches continues to expand globally, the application of Kaptive 2.0 will generate novel insights essential for the design of effective KpSC control strategies.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.05.467388', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'A temporally resolved, multiplex molecular recorder based on sequential genome editing', 'authors': \"[{'name': 'Junhong Choi'}, {'name': 'Wei Chen'}, {'name': 'Anna Minkina'}, {'name': 'Florence M Chardon'}, {'name': 'Chase C Suiter'}, {'name': 'Samuel G Regalado'}, {'name': 'Silvia Domcke'}, {'name': 'Nobuhiko Hamazaki'}, {'name': 'Choli Lee'}, {'name': 'Beth Martin'}, {'name': 'Riza M Daza'}, {'name': 'Jay Shendure'}]\", 'abstract': '<p>DNA is naturally well-suited to serve as a digital medium for in vivo molecular recording. However, DNA-based memory devices described to date are constrained in terms of the number of distinct signals that can be concurrently recorded and/or by a failure to capture the precise order of recorded events. Here we describe DNA Ticker Tape, a general system for in vivo molecular recording that largely overcomes these limitations. Blank DNA Ticker Tape consists of a tandem array of partial CRISPR-Cas9 target sites, with all but the first site truncated at their 5\\' ends, and therefore inactive. Signals of interest are coupled to the expression of specific prime editing guide RNAs. Editing events are insertional, and record the identity of the guide RNA mediating the insertion while also shifting the position of the \"write head\" by one unit along the tandem array, i.e. sequential genome editing. In this proof-of-concept of DNA Ticker Tape, we demonstrate the recording and decoding of complex event histories or short text messages; evaluate the performance of dozens of orthogonal tapes; and construct \"long tape\" potentially capable of recording the order of as many as 20 serial events. Finally, we demonstrate how DNA Ticker Tape simplifies the decoding of cell lineage histories.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.10.29.466512', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Sex and Estrous Cycle in Memory for Sequences of Events in Rats', 'authors': \"[{'name': 'M. Jayachandran'}, {'name': 'P. Langius'}, {'name': 'F. Pazos Rego'}, {'name': 'R. P. Vertes'}, {'name': 'T. A. Allen'}]\", 'abstract': '<p>The ability to remember sequences of events is fundamental to episodic memory. While rodent studies have examined sex and estrous cycle in episodic-like spatial memory tasks, little is known about these biological variables in memory for sequences of events that depend on representations of temporal context. We investigated the role of sex and estrous cycle in rats during all training and testing stages of a cross-species validated sequence memory task (Jayachandran et al., 2019). Rats were trained on a task composed of two sequences, each with four unique odors delivered on opposite ends of a linear track. Training occurred in six successive stages starting with learning to poke in a nose port for ≥1.2s; eventually demonstrating sequence memory by holding their nose in the port for ≥1s for in-sequence odors and &lt;1s for out-of-sequence odors in order to receive a water reward. Performance was analyzed across sex and estrous cycle (proestrus, estrus, metestrus, and diestrus), the latter being determined by the cellular composition of a daily vaginal lavage. We found no evidence of sex differences in asymptotic sequence memory performance, similar to published data in humans performing the analogous task (Reeders et al., 2021). Likewise, we found no differences in performance across the estrous cycle. One minor difference was that female rats tended to have slightly longer poke times, while males had slightly more short poke times but this did not affect their decisions. These results suggest sex and estrous cycle are not major factors in sequence memory capacities.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.02.464583', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Marker-free coselection for successive rounds of prime editing in human cells', 'authors': \"[{'name': 'Sébastien Levesque'}, {'name': 'Diana Mayorga'}, {'name': 'Jean-Philippe Fiset'}, {'name': 'Claudia Goupil'}, {'name': 'Alexis Duringer'}, {'name': 'Andréanne Loiselle'}, {'name': 'Eva Bouchard'}, {'name': 'Daniel Agudelo'}, {'name': 'Yannick Doyon'}]\", 'abstract': '<p>Prime editing enables the introduction of precise point mutations, small insertions, or short deletions without requiring donor DNA templates. However, editing efficiency remains a key challenge in a broad range of cell types. We designed a robust coselection strategy to perform successive rounds of genetic changes in human cells using CRISPR-Cas nucleases and prime editors. Through coediting, the recovery of cells modified at a target gene of interest is coupled to selection for dominant alleles of the ubiquitous and essential sodium/potassium pump (Na<sup>+</sup>/K<sup>+</sup> ATPase). The method improves prime editing efficiency 1.5–10-fold in K562 and HeLa cells, reaching up to 83% of desired alleles in cell populations, for previously optimized prime editing guide RNAs (pegRNAs). Using pegRNAs designed <italic>de novo</italic>, we readily engineered homozygous cell lines with cancer-associated or drug resistant <italic>MTOR</italic> mutations displaying altered mTORC1 signaling. Our approach streamlines the production of cell lines with multiple genetic modifications to create cellular models for biological research and lays the foundation for the development of cell-type specific coselection strategies.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.01.466790', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Programmable large DNA deletion, replacement, integration, and inversion with twin prime editing and site-specific recombinases', 'authors': \"[{'name': 'Andrew V. Anzalone'}, {'name': 'Xin D. Gao'}, {'name': 'Christopher J. Podracky'}, {'name': 'Andrew T. Nelson'}, {'name': 'Luke W. Koblan'}, {'name': 'Aditya Raguram'}, {'name': 'Jonathan M. Levy'}, {'name': 'Jaron A. M. Mercer'}, {'name': 'David R. Liu'}]\", 'abstract': '<p>The targeted deletion, replacement, integration, or inversion of DNA sequences at specified locations in the genome could be used to study or treat many human genetic diseases. Here, we describe twin prime editing (twinPE), a method for the programmable replacement or excision of DNA sequence at endogenous human genomic sites without requiring double-strand DNA breaks. TwinPE uses a prime editor (PE) protein and two prime editing guide RNAs (pegRNAs) that template the synthesis of complementary DNA flaps on opposing strands of genomic DNA, resulting in the replacement of endogenous DNA sequence between the PE-induced nick sites with pegRNA-encoded sequences. We show that twinPE in human cells can perform precise deletions of at least 780 bp and precise replacements of genomic DNA sequence with new sequences of at least 108 bp. By combining single or multiplexed twinPE with site-specific serine recombinases either in separate steps or in a single step, we demonstrate targeted integration of gene-sized DNA plasmids (&gt;5,000 bp) into safe-harbor loci including <italic>AAVS1</italic>, <italic>CCR5,</italic> and <italic>ALB</italic> in human cells. To our knowledge, these results represent the first RNA-programmable insertion of gene-sized DNA sequences into targeted genomic sites of unmodified human cells without requiring double-strand breaks or homology-directed repair. Twin PE combined with recombinases also mediated a 40,167-bp inversion at <italic>IDS</italic> that corrects a common Hunter syndrome allele. TwinPE expands the capabilities of precision gene editing without requiring double-strand DNA breaks and synergizes with other tools to enable the correction or complementation of large or complex pathogenic alleles in human cells.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.02.466884', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Retinal OFF ganglion cells allow detection of quantal shadows at starlight', 'authors': \"[{'name': 'Johan Westö'}, {'name': 'Nataliia Martyniuk'}, {'name': 'Sanna Koskela'}, {'name': 'Tuomas Turunen'}, {'name': 'Santtu Pentikäinen'}, {'name': 'Petri Ala-Laurila'}]\", 'abstract': '<p>Perception of light in darkness requires no more than a handful of photons and this remarkable behavioral performance can be directly linked to a particular retinal circuit – the retinal ON pathway. However, the neural limits to shadow detection in very dim light have remained unresolved. Here, we unravel the neural mechanisms that determine the sensitivity of mice to the dimmest light decrements by measuring signals from the most sensitive ON and OFF retinal ganglion cell types and by correlating their signals with visually guided behavior. We show that mice can detect shadows when only a few photon absorptions are missing among thousands of rods. Behavioral detection of “quantal” shadows relies only on the retinal OFF pathway and is limited by noise and losses of single-photon signals in retinal processing. Thus, in the dim-light regime, light increments and decrements are encoded separately via the ON and OFF retinal pathways, respectively.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466363', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Distinct representation of cue-outcome association by D1 and D2 neurons in the olfactory striatum', 'authors': \"[{'name': 'Nuné Martiros'}, {'name': 'Spencer E. Kim'}, {'name': 'Vikrant Kapoor'}, {'name': 'Venkatesh N. Murthy'}]\", 'abstract': '<p>Positive and negative associations acquired through olfactory experience are thought to be especially strong and long-lasting. The conserved direct olfactory sensory input to the ventral striatal olfactory tubercle (OT) and its convergence with dense dopaminergic input to the OT could underlie this privileged form of associative memory, but how this process occurs is not well understood. We imaged the activity of the two canonical types of striatal neurons, expressing D1 or D2 type dopamine receptors, in the OT at cellular resolution while mice learned odor-outcome associations ranging from aversive to rewarding. D1 and D2 neurons both responded to rewarding and aversive odors. D1 neurons in the OT robustly and bidirectionally represented odor valence, responding similarly to odors predicting similar outcomes regardless of odor identity. This valence representation persisted even in the absence of an instrumental response to the odors and in the absence of the outcomes, indicating a true transformation of odor sensory information by D1 OT neurons. In contrast, D2 neuronal representation of the odor-outcome associations was weaker, contingent on an instrumental response by the mouse, and D2 neurons were more selective for odor identity than valence. Stimulus valence coding in the OT was not modality invariant, with separate sets of D1 neurons responding to odors and sounds predicting the same outcomes, suggesting that integration of multimodal valence information happens downstream of the OT. Our results point to distinct representation of identity and valence of odor stimuli by D1 and D2 neurons in the OT.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.02.466953', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Population genomics analysis with RAD, reprised: STACKS 2', 'authors': \"[{'name': 'Angel G Rivera-Colon'}, {'name': 'Julian M Catchen'}]\", 'abstract': '<p>Restriction enzymes have been one of the primary tools in the population genetics toolkit for fifty years, being coupled with each new generation of technology to provide a more detailed view into the genetics of natural populations. Restriction site-Associated DNA protocols, which joined enzymes with short-read sequencing technology have democratized the field of population genomics, providing a means to assay the underlying alleles in scores of populations. More than ten years on, the technique has been widely applied across the tree of life and served as the basis for many different analysis techniques. Here, we provide a detailed protocol to conduct a RAD analysis from experimental design to de novo analysis -- including parameter optimization -- as well as reference-based analysis, all in STACKS version 2, which is designed to work with paired-end reads to assemble RAD loci up to 1000 nucleotides in length. The protocol focuses on major points of friction in the molecular approaches and downstream analysis, with special attention given to validating experimental analyses. Finally, the protocol provides several points of departure for further analysis.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.467474', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Aggregation controlled by condensate rheology', 'authors': \"[{'name': 'Wolfram Poenisch'}, {'name': 'Thomas C.T. Michaels'}, {'name': 'Christoph A. Weber'}]\", 'abstract': '<p>Condensates in living cells can exhibit a complex rheology including viscoelastic and glassy behaviour. This rheological behavior of condensates was suggested to regulate polymerisation of cytoskeletal filaments and aggregation of amyloid fibrils. Here, we theoretically investigate how the rheological properties of condensates can control the formation of linear aggregates. To this end, we propose a kinetic theory for the aggregate size distribution and the exchange of aggregates between inside and outside of condensates. The rheology of condensates is accounted for by aggregate mobilities that depend on aggregate size. We show that condensate rheology can control whether either aggregates of all sizes or dominantly small aggregates are exchanged between condensate inside and outside on the time-scale of aggregation. As a result, the ratio of aggregate numbers inside to outside of condensates differs significantly. Strikingly, we also find that weak variations in the rheological properties of condensates can lead to a switch-like change of the number of aggregates. These results suggest a possible physical mechanism for how living cells could control linear aggregation in a switch-like fashion through variations in condensate rheology.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.04.467378', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': True, 'title': 'A high-throughput, automated, cell-free expression and screening platform for antibody discovery', 'authors': \"[{'name': 'Andrew C Hunt'}, {'name': 'Bastian Vogeli'}, {'name': 'Weston K. Kightlinger'}, {'name': 'Danielle J. Yoesep'}, {'name': 'Antje Kruger'}, {'name': 'Michael C. Jewett'}]\", 'abstract': '<p>Antibody discovery is bottlenecked by the individual expression and evaluation of antigen- specific hits. Here, we address this gap by developing an automated workflow combining cell-free DNA template generation, protein synthesis, and high-throughput binding measurements of antibody fragments in a process that takes hours rather than weeks. We apply this workflow to 119 published SARS-CoV-2 neutralizing antibodies and demonstrate rapid identification of the most potent antibody candidates.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.467109', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Reciprocal zebrafish-medaka hybrids reveal maternal control of zygotic genome activation timing', 'authors': \"[{'name': 'Krista R. Gert'}, {'name': 'Luis Enrique Cabrera Quio'}, {'name': 'Maria Novatchkova'}, {'name': 'Yixuan Guo'}, {'name': 'Bradley R. Cairns'}, {'name': 'Andrea Pauli'}]\", 'abstract': '<p>After fertilization, the sperm and egg contribute unequally to the newly formed zygote. While the sperm contributes mainly paternal DNA, the egg provides both maternal DNA and the bulk of the future embryonic cytoplasm. Most embryonic processes (like the onset of zygotic transcription) depend on maternally-provided cytoplasmic components, and it is largely unclear whether paternal components besides the centrosome play a role in the regulation of early embryogenesis. Here we report a reciprocal zebrafish-medaka hybrid system as a powerful tool to investigate paternal vs. maternal influence during early development. By combining expression of zebrafish Bouncer on the medaka egg with artificial egg activation, we demonstrate the <italic>in vitro</italic> generation of paternal zebrafish x maternal medaka (reripes) hybrids. These hybrids complement the previously established paternal medaka x maternal zebrafish (latio) hybrids (Herberg et al., 2018). As proof of concept, we investigated maternal vs. paternal control of zygotic genome activation (ZGA) timing using this reciprocal hybrid system. RNA-seq analysis of the purebred fish species and hybrids revealed that the onset of ZGA is primarily governed by the egg. Overall, our study establishes the reciprocal zebrafish-medaka hybrid system as a versatile tool to dissect parental control mechanisms during early development.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.04.467308', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'SARS-CoV-2 infection in free-ranging white-tailed deer (Odocoileus virginianus)', 'authors': \"[{'name': 'Vanessa L Hale'}, {'name': 'Patricia M Dennis'}, {'name': 'Dillon S McBride'}, {'name': 'Jacqueline M Nolting'}, {'name': 'Christopher Madden'}, {'name': 'Devra Huey'}, {'name': 'Margot Ehrlich'}, {'name': 'Jennifer Grieser'}, {'name': 'Jenessa A Winston'}, {'name': 'Dusty Lombardi'}, {'name': 'Stormy Gibson'}, {'name': 'Linda Saif'}, {'name': 'Mary Lea Killian'}, {'name': 'Kristina Lantz'}, {'name': 'Rachel M Tell'}, {'name': 'Mia Kim Torchetti'}, {'name': 'Suelee Robbe-Austerman'}, {'name': 'Martha I Nelson'}, {'name': 'Seth A Faith'}, {'name': 'Andrew Bowman'}]\", 'abstract': '<p>Human-to-animal spillover of SARS-CoV-2 virus has occurred in a wide range of animals, but thus far, the establishment of a new natural animal reservoir has not been detected. Here, we detected SARS-CoV-2 virus using rRT-PCR in 129 out of 360 (35.8%) free-ranging white-tailed deer (Odocoileus virginianus) from northeast Ohio (USA) sampled between January-March 2021. Deer in 6 locations were infected with at least 3 lineages of SARS-CoV-2 (B.1.2, B.1.596, B.1.582). The B.1.2 viruses, dominant in Ohio at the time, spilled over multiple times into deer populations in different locations. Deer-to-deer transmission may have occurred in three locations. The establishment of a natural reservoir of SARS-CoV-2 in white-tailed deer could facilitate divergent evolutionary trajectories and future spillback to humans, further complicating long-term COVID-19 control strategies.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.02.466829', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Single molecule tracking of bacterial cell surface cytochromes reveals dynamics that impact long-distance electron transport', 'authors': \"[{'name': 'Grace W. Chong'}, {'name': 'Sahand Pirbadian'}, {'name': 'Yunke Zhao'}, {'name': 'Lori A. Zacharoff'}, {'name': 'Fabien Pinaud'}, {'name': 'Mohamed Y. El-Naggar'}]\", 'abstract': '<p>Using a series of multiheme cytochromes, the metal-reducing bacterium <italic>Shewanella oneidensis</italic> MR-1 can perform extracellular electron transfer (EET) to respire redox-active surfaces, including minerals and electrodes outside the cell. While the role of multiheme cytochromes in transporting electrons across the cell wall is well established, these cytochromes were also recently found to facilitate long-distance (micrometer-scale) redox conduction along outer membranes and across multiple cells bridging electrodes. Recent studies proposed that long-distance conduction arises from the interplay of electron hopping and cytochrome diffusion, which allows collisions and electron exchange between cytochromes along membranes. However, the diffusive dynamics of the multiheme cytochromes have never been observed or quantified <italic>in vivo</italic>, making it difficult to assess their hypothesized contribution to the collision-exchange mechanism. Here we use quantum dot labeling, total internal reflection fluorescence microscopy, and single-particle tracking to quantify the lateral diffusive dynamics of the outer membrane-associated decaheme cytochromes MtrC and OmcA, two key components of EET in <italic>S. oneidensis</italic>. We observe confined diffusion behavior for both quantum dot-labeled MtrC and OmcA along cell surfaces (diffusion coefficients <italic>D</italic><sub>MtrC</sub> = 0.0192 ± 0.0018 μm<sup>2</sup>/s, <italic>D</italic><sub>OmcA</sub> = 0.0125 ± 0.0024 μm<sup>2</sup>/s) and the membrane extensions thought to function as bacterial nanowires. We find that these dynamics can trace a path for electron transport via overlap of cytochrome trajectories, consistent with the long-distance conduction mechanism. The measured dynamics inform kinetic Monte Carlo simulations that combine direct electron hopping and redox molecule diffusion, revealing significant electron transport rates along cells and membrane nanowires.</p><sec><title>Significance</title><p>Multiheme cytochromes in <italic>Shewanella oneidensis</italic> MR-1 transport electrons across the cell wall, in a process called extracellular electron transfer. These electron conduits can also enable electron transport along and between cells. While the underlying mechanism is thought to involve a combination of electron hopping and lateral diffusion of cytochromes along membranes, these diffusive dynamics have never been observed <italic>in vivo</italic>. Here, we observe the mobility of quantum dot-labeled cytochromes on living cell surfaces and membrane nanowires, quantify their diffusion with single-particle tracking techniques, and simulate the contribution of these dynamics to electron transport. This work reveals the impact of redox molecule dynamics on bacterial electron transport, with implications for understanding and harnessing this process in the environment and bioelectronics.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.467034', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'NASP maintains histone H3–H4 homeostasis through two distinct H3 binding modes', 'authors': \"[{'name': 'Hongyu Bao'}, {'name': 'Massimo Carraro'}, {'name': 'Valentin Flury'}, {'name': 'Yanhong Liu'}, {'name': 'Min Luo'}, {'name': 'Liu Chen'}, {'name': 'Anja Groth'}, {'name': 'Hongda Huang'}]\", 'abstract': '<p>Histone chaperones regulate all aspects of histone metabolism. NASP is a major histone chaperone for H3–H4 dimers critical for preventing histone degradation. Here, we identify two distinct histone binding modes of NASP and reveal how they cooperate to ensure histone H3–H4 supply. We determine the structures of a sNASP dimer, a complex of sNASP with an H3 α3 peptide, and the sNASP–H3–H4–ASF1b co-chaperone complex. This captures distinct functionalities of NASP and identifies two distinct binding modes involving the H3 α3 helix and the H3 αN region, respectively. Functional studies demonstrate the H3 αN-interaction represents the major binding mode of NASP in cells and shielding of the H3 αN region by NASP is essential in maintaining the H3–H4 histone soluble pool. In conclusion, our studies uncover the molecular basis of NASP as a major H3–H4 chaperone in guarding histone homeostasis.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.21265843', 'tweets': 21, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'A RCT of a third dose CoronaVac or BNT162b2 vaccine in adults with two doses of CoronaVac', 'authors': \"[{'name': 'Chris Ka Pun Mok'}, {'name': 'Samuel M.S. Cheng'}, {'name': 'Chunke Chen'}, {'name': 'Karen Yiu'}, {'name': 'Tat-On Chan'}, {'name': 'Kiu Cheung Lai'}, {'name': 'Kwun Cheung Ling'}, {'name': 'Yuanxin Sun'}, {'name': 'Lun Lai Ho'}, {'name': 'Malik Peiris'}, {'name': 'David S Hui'}]\", 'abstract': '<sec><title>Background</title><p>Poor immunogenicity and antibody waning were found in vaccinees of CoronaVac. There is lack of randomized controlled trial (RCT) data to compare the immunogenicity and safety of schedules using homologous and heterologous vaccine as a booster dose.</p></sec><sec><title>Methods</title><p>We randomly assigned adults who had received 2 doses of CoronaVac with low antibody response to receive an additional booster dose of either BNT162b2 or CoronaVac. The local and systemic adverse reactions were recorded. Levels of SARS-CoV-2 neutralizing and spike binding antibody in plasma were measured.</p></sec><sec><title>Findings</title><p>At one month after the third dose of vaccine, BNT162b2 vaccines elicited significantly higher surrogate virus neutralizing test (sVNT), spike receptor binding, spike N terminal domain binding, spike S2 domain binding levels than CoronaVac. More participants from the BNT162b2 group reported injection site pain and swelling as well as fatigue and muscle pain than those who received CoronaVac as the third dose. The mean results of the sVNT against the wild type, beta, gamma and delta variants in the BNT162b2 boosted group was 96.83%, 92.29%, 92.51% and 95.33% respectively which were significantly higher than the CoronaVac boosted group (Wild type: 57.75%; Beta: 38.79 %; Gamma: 32.22%; Delta: 48.87%)</p></sec><sec><title>Conclusion</title><p>Our RCT study shows that BNT162b2 booster dose for those people who poorly responded to the previous vaccination of CoronaVac is significantly more immunogenic than a CoronaVac booster. BNT162b2 also elicits higher levels of SARS-CoV-2 specific neutralizing antibodies to different variants of concern. The adverse reactions were only mild and short-lived.</p></sec><sec><title>At a Glance Commentary</title><sec><title>Scientific Knowledge on the Subject</title><p>Poor immunogenicity and antibody waning were found in vaccinees of CoronaVac. There is lack of randomized controlled trial (RCT) data to compare the immunogenicity and safety of schedules using homologous and heterologous vaccine as a booster dose.</p></sec><sec><title>What This Study Adds to the Field</title><p>Our RCT study shows that BNT162b2 booster dose for those people who poorly responded to the previous vaccination of CoronaVac is significantly more immunogenic than a CoronaVac booster. The adverse reactions were only mild and short-lived.</p></sec></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.05.467477', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Age-induced P-bodies become detrimental and shorten the lifespan of yeast', 'authors': \"[{'name': 'Joonhyuk Choi'}, {'name': 'Shuhao Wang'}, {'name': 'Yang Li'}, {'name': 'Nan Hao'}, {'name': 'Brian M Zid'}]\", 'abstract': '<p>Aging is an irreversible process characterized by a progressive loss of homeostasis in cells, which often manifests as protein aggregates. Recently, it has been speculated that aggregates of RNA-binding proteins (RBPs) may go through pathological transitions during aging and drive the progression of age-associated neurodegenerative diseases. Using Saccharomyces cerevisiae as a model system of aging, we find that P-bodies - an RBP granule that is formed and can be beneficial for cell growth during stress conditions - naturally form during aging without any external stresses and an increase in P-body intensity is negatively correlated with the future lifespan of yeast cells. When mother cells transfer age-induced P-bodies to daughter cells, the mother cells extend lifespan, while the daughter cells grow poorly, suggesting that these age-induced P-bodies may be directly pathological. Furthermore, we find that suppressing acidification of the cytosol during aging slows down the increase in the intensity of P-body foci and extends lifespan. Our data suggest that acidification of the cytosol may facilitate the pathological transition of RBP granules during aging.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.01.466478', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Cooperative electrolyte-PEG interactions drive the signal amplification in a solid-state nanopore', 'authors': \"[{'name': 'Chalmers C. Chau'}, {'name': 'Fabio Marcuccio'}, {'name': 'Dimitrios Soulias'}, {'name': 'Martin A. Edwards'}, {'name': 'Sheena E. Radford'}, {'name': 'Eric W. Hewitt'}, {'name': 'Paolo Actis'}]\", 'abstract': '<p>Nanopore systems have emerged as a leading platform for the analysis of biomolecular complexes with single molecule resolution. However, the analysis of several analytes like short nucleic acids or proteins with nanopores represents a sensitivity challenge, because their translocation lead to small signals difficult to distinguish from the noise. Here, we report a simple method to enhance the signal to noise ratio in nanopore experiments by a simple modification of the solution used in nanopore sensing. The addition of poly-ethylene glycol (PEG) and the careful selection of the supporting electrolyte leads to large signal enhancement. We observed that the translocation dynamics are in good agreement with an established method that uses the lattice energy of an electrolyte to approximate the affinity of an ion to PEG. We identified CsBr as the optimal supporting electrolyte to complement PEG to enable the analysis of dsDNA at 500 kHz bandwidth, and the detection of dsDNA as short as 75 bp.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.10.29.466473', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Multi-attribute Glycan Identification and FDR Control for Glycoproteomics', 'authors': \"[{'name': 'Daniel A. Polasky'}, {'name': 'Daniel J. Geiszler'}, {'name': 'Fengchao Yu'}, {'name': 'Alexey I. Nesvizhskii'}]\", 'abstract': '<p>Rapidly improving methods for glycoproteomics have enabled increasingly large-scale analyses of complex glycopeptide samples, but annotating the resulting mass spectrometry data with high confidence remains a major bottleneck. We recently introduced a fast and sensitive glycoproteomics search method in our MSFragger search engine, which reports glycopeptides as a combination of a peptide sequence and the mass of the attached glycan. In samples with complex glycosylation patterns, converting this mass to a specific glycan composition is not straightforward, however, as many glycans have similar or identical masses. Here, we have developed a new method for determining the glycan composition of N-linked glycopeptides fragmented by collision or hybrid activation that uses multiple sources of information from the spectrum, including observed glycan B- (oxonium) and Y-type ions and mass and precursor monoisotopic selection errors to discriminate between possible glycan candidates. Combined with false discovery rate estimation for the glycan assignment, we show this method is capable of specifically and sensitively identifying glycans in complex glycopeptide analyses and effectively controls the rate of false glycan assignments. The new method has been incorporated into the PTM-Shepherd modification analysis tool to work directly with the MSFragger glyco search in the FragPipe graphical user interface, providing a complete computational pipeline for annotation of N-glycopeptide spectra with FDR control of both peptide and glycan components that is both sensitive and robust against false identifications.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.10.29.21265555', 'tweets': 18, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'IFI27 transcription is an early predictor for COVID-19 outcomes; a multi-cohort observational study', 'authors': \"[{'name': 'Maryam Shojaei'}, {'name': 'Amir Shamshirian'}, {'name': 'James Monkman'}, {'name': 'Laura Grice'}, {'name': 'Minh Tran'}, {'name': 'Chin Wee Tan'}, {'name': 'Gustavo Rodrigues Rossi'}, {'name': 'Timothy R. McCulloch'}, {'name': 'Marek Nalos'}, {'name': 'Keng Yih Chew'}, {'name': 'Yanshan Zhu'}, {'name': 'Yao Xia'}, {'name': 'Timothy J. Wells'}, {'name': 'Alexandra Cristina Senegaglia'}, {'name': 'Carmen Lúcia Kuniyoshi Rebelatto'}, {'name': 'Claudio Luciano Franck'}, {'name': 'Anna Flavia Ribeiro dos Santos'}, {'name': 'Lucia de Noronha'}, {'name': 'Sepideh Motamen'}, {'name': 'Reza Valadan'}, {'name': 'Omolbanin Amjadi'}, {'name': 'Rajan Gogna'}, {'name': 'Esha Madan'}, {'name': 'Reza Alizadeh-Navaei'}, {'name': 'Liliana Lamperti'}, {'name': 'Felipe Zuñiga'}, {'name': 'Estefania Nova-Lamperti'}, {'name': 'Gonzalo Labarca'}, {'name': 'Ben Knippenberg'}, {'name': 'Velma Herwanto'}, {'name': 'Ya Wang'}, {'name': 'Amy Phu'}, {'name': 'Tracy Chew'}, {'name': 'Timothy Kwan'}, {'name': 'Karan Kim'}, {'name': 'Sally Teoh'}, {'name': 'Tiana M Pelaia'}, {'name': 'Win Sen Kuan'}, {'name': 'Yvette Jee'}, {'name': 'Jon Iredell'}, {'name': 'Ken O’Byrne'}, {'name': 'John F. Fraser'}, {'name': 'Melissa J. Davis'}, {'name': 'Gabrielle Belz'}, {'name': 'Majid Warkiani'}, {'name': 'Carlos Salomon Gallo'}, {'name': 'Fernando Souza-Fonseca-Guimaraes'}, {'name': 'Quan Nguyen'}, {'name': 'Anthony Mclean'}, {'name': 'Arutha Kulasinghe'}, {'name': 'Kirsty R. Short'}, {'name': 'Benjamin Tang'}]\", 'abstract': '<sec><title>Background</title><p>Robust biomarkers that predict disease outcomes amongst COVID-19 patients are necessary for both patient triage and resource prioritisation. Numerous candidate biomarkers have been proposed for COVID-19. However, at present, there is no consensus on the best diagnostic approach to predict outcomes in infected patients. Moreover, it is not clear whether such tools would apply to other potentially pandemic pathogens and therefore of use as stockpile for future pandemic preparedness.</p></sec><sec><title>Methods</title><p>We conducted a multi-cohort observational study to investigate the biology and the prognostic role of interferon alpha-inducible protein 27 (<italic>IFI27</italic>) in COVID-19 patients.</p></sec><sec><title>Findings</title><p>We show that <italic>IFI27</italic> is expressed in the respiratory tract of COVID-19 patients and elevated <italic>IFI27</italic> expression is associated with the presence of a high viral load. We further demonstrate that systemic host response, as measured by blood <italic>IFI27</italic> expression, is associated with COVID-19 severity. For clinical outcome prediction (e.g. respiratory failure), <italic>IFI27</italic> expression displays a high positive (0.83) and negative (0.95) predictive value, outperforming all other known predictors of COVID-19 severity. Furthermore, <italic>IFI27</italic> is upregulated in the blood of infected patients in response to other respiratory viruses. For example, in the pandemic H1N1/09 swine influenza virus infection, <italic>IFI27-</italic>like genes were highly upregulated in the blood samples of severely infected patients.</p></sec><sec><title>Interpretation</title><p>These data suggest that prognostic biomarkers targeting the family of <italic>IFI27</italic> genes could potentially supplement conventional diagnostic tools in future virus pandemics, independent of whether such pandemics are caused by a coronavirus, an influenza virus or another as yet-to-be discovered respiratory virus.</p></sec><sec><title>Research in context</title><sec><title>Evidence before this study</title><p>We searched the scientific literature using PubMed to identify studies that used the <italic>IFI27</italic> biomarker to predict outcomes in COVID-19 patients. We used the search terms “<italic>IFI27</italic>”, “COVID-19, “gene expression” and “outcome prediction”. We did not identify any study that investigated the role of <italic>IFI27</italic> biomarker in outcome prediction. Although ten studies were identified using the general terms of “gene expression” and “COVID-19”, <italic>IFI27</italic> was only mentioned in passing as one of the identified genes. All these studies addressed the broader question of the host response to COVID-19; none focused solely on using <italic>IFI27</italic> to improve the risk stratification of infected patients in a pandemic.</p></sec><sec><title>Added value of this study</title><p>Here, we present the findings of a multi-cohort study of the <italic>IFI27</italic> biomarker in COVID-19 patients. Our findings show that the host response, as reflected by blood <italic>IFI27</italic> gene expression, accurately predicts COVID-19 disease progression (positive and negative predictive values; 0.83 and 0.95, respectively), outperforming age, comorbidity, C-reactive protein and all other known risk factors. The strong association of <italic>IFI27</italic> with disease severity occurs not only in SARS-CoV-2 infection, but also in other respiratory viruses with pandemic potential, such as the influenza virus. These findings suggest that host response biomarkers, such as <italic>IFI27</italic>, could help identify high-risk COVID-19 patients - those who are more likely to develop infection complications - and therefore may help improve patient triage in a pandemic.</p></sec><sec><title>Implications of all the available evidence</title><p>This is the first systemic study of the clinical role of <italic>IFI27</italic> in the current COVID-19 pandemic and its possible future application in other respiratory virus pandemics. The findings not only could help improve the current management of COVID-19 patients but may also improve future pandemic preparedness.</p></sec></sec>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.06.467543', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Malaria transmission relies on concavin-mediated maintenance of Plasmodium sporozoite cell shape', 'authors': \"[{'name': 'Jessica Kehrer'}, {'name': 'Pauline Formaglio'}, {'name': 'Julianne Mendi Muthinja'}, {'name': 'Sebastian Weber'}, {'name': 'Danny Baltissen'}, {'name': 'Christopher Lance'}, {'name': 'Johanna Ripp'}, {'name': 'Janessa Grech'}, {'name': 'Markus Meissner'}, {'name': 'Charlotta Funaya'}, {'name': 'Rogerio Amino'}, {'name': 'Friedrich Frischknecht'}]\", 'abstract': '<p>During transmission of malaria-causing parasites from mosquitoes to mammals, <italic>Plasmodium</italic> sporozoites migrate rapidly in the skin to search for a blood vessel. The high migratory speed and narrow passages taken by the parasites suggest considerable strain on the sporozoites to maintain their shape. Here we report on a newly identified protein, concavin, that is important for maintenance of the sporozoite shape inside salivary glands of mosquitoes and during migration in the skin. Concavin-GFP localized at the cytoplasmic periphery of sporozoites and <italic>concavin(-)</italic> sporozoites progressively rounded up upon entry of salivary glands. These rounded <italic>concavin(-)</italic> sporozoites failed to pass through the narrow salivary ducts and were hence rarely ejected by mosquitoes. However, normally shaped <italic>concavin(-)</italic> sporozoites could be transmitted and migrated in the skin or skin like environments. Strikingly, motile <italic>concavin(-)</italic> sporozoites could disintegrate while migrating through narrow strictures in the skin leading to parasite arrest or death and decreased transmission efficiency. We suggest that concavin contributes to cell shape maintenance by riveting the plasma membrane to the subtending inner membrane complex.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.10.30.466530', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Consequences of endogenous and virally-induced hyperphosphorylated tau on behavior and cognition in a rat model of Alzheimer’s disease', 'authors': \"[{'name': 'Michael A. Kelberman'}, {'name': 'Claire R. Anderson'}, {'name': 'Eli Chlan'}, {'name': 'Jacki M. Rorabaugh'}, {'name': 'Katharine E. McCann'}, {'name': 'David Weinshenker'}]\", 'abstract': '<sec><title>Background</title><p>The locus coeruleus (LC) is one of the earliest brain regions to accumulate hyperphosphorylated tau, but a lack of animal models that recapitulate this pathology has hampered our ability to understand its contributions to the pathophysiology of Alzheimer’s disease (AD).</p></sec><sec><title>Objective</title><p>We previously reported that TgF344-AD rats, which overexpress mutant human amyloid precursor protein and presenilin-1, accumulate early endogenous hyperphosphorylated tau in the LC. Here, we used TgF344-AD rats and a wild-type (WT) human tau virus to interrogate the effects of endogenous hyperphosphorylated rat tau and human tau in the LC on AD-related neuropathology and behavior.</p></sec><sec><title>Methods</title><p>Two-month old TgF344-AD and WT rats received bilateral LC infusions of WT human tau or mCherry control virus driven by the noradrenergic-specific PRSx8 promoter. Rats were subsequently assessed at 6 and 12 months for arousal (sleep latency), anxiety-like behavior (open field, elevated plus maze, novelty-suppressed feeding), passive coping (forced swim task), and learning and memory (Morris water maze and fear conditioning). Hippocampal microglia, astrocyte, and AD pathology were evaluated using immunohistochemistry.</p></sec><sec><title>Results</title><p>In general, the effects of age were more pronounced than genotype or treatment; older rats displayed greater hippocampal pathology, took longer to fall asleep, had reduced locomotor activity, floated more, and had impaired cognition compared to younger animals. TgF344-AD rats showed increased anxiety-like behavior and impaired learning and memory. The tau virus had negligible influence on most measures.</p></sec><sec><title>Conclusion</title><p>Effects of hyperphosphorylated tau on AD-like neuropathology and behavioral symptoms were subtle. Further investigation of different forms of tau is warranted.</p></sec>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.05.467461', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Design, construction and optimization of a synthetic RNA polymerase operon in Escherichia coli.', 'authors': \"[{'name': 'Joep Houkes'}, {'name': 'Lorenzo Olivi'}, {'name': 'Zacharie Paquet'}, {'name': 'Nico J Claassens'}, {'name': 'John van der Oost'}]\", 'abstract': '<p>Prokaryotic genes encoding functionally related proteins are often clustered in operons. The compact structure of operons allows for co-transcription of the genes, and for co-translation of the polycistronic messenger RNA to the corresponding proteins. This leads to reduced regulatory complexity and enhanced gene expression efficiency, and as such to an overall metabolic benefit for the protein production process in bacteria and archaea. Interestingly, the genes encoding the subunits of one of the most conserved and ubiquitous protein complexes, the RNA polymerase, are not clustered in a single operon. Rather, its genes are scattered in all known prokaryotic genomes, generally integrated in different ribosomal operons. To analyze the impact of this genetic organization on the fitness of Escherichia coli, we constructed a bacterial artificial chromosome harboring the genes encoding the RNA polymerase complex in a single operon. Subsequent deletion of the native chromosomal genes led to a reduced growth on minimal medium. However, by using adaptive laboratory evolution the growth rate was restored to wild-type level. Hence, we show that a highly conserved genetic organization of core genes in a bacterium can be reorganized by a combination of design, construction and optimization, yielding a well-functioning synthetic genetic architecture.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.06.467542', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Shaping the size of a neuronal lineage: the role of Imp and Syp RBPs in the precise elimination of neurons by apoptosis.', 'authors': \"[{'name': 'Jonathan ENRIQUEZ'}, {'name': 'Wenyue GUAN'}, {'name': 'Mathilde BOUCHET'}, {'name': 'Aurelien DARMAS'}]\", 'abstract': '<p>Neuronal stem cells produce a finite and stereotyped number of neuronal progenies. This process must be finely regulated during development and adult stages to ensure proper brain function.  In Drosophila, stem cells, called Neuroblasts, produce an invariant number of neurons. Two RNA binding proteins, Imp and Syp, play a central role in controlling the speed of division and the end of the proliferative phase of individual NBs, two parameters that influences the final number of neurons produced. Here, we have discovered a novel function for Imp and Syp, where both RBPs shape the number of neurons produced by a stem cell by controlling program cell death (PCD) in immature neurons. By studying a neuroblast lineage, called Lin A/15, which produces motoneurons (MNs) and glia, we have demonstrated that Lin A/15 stem cell spends 40% of its time producing immature MNs which are eliminated by apoptosis. We have revealed that only the first born MNs (Imp +) survive while the last born MNs (Imp- Syp+) are eliminated by apoptosis. Both RBPs play a central role in neuronal survival, Imp promotes neuronal survival while Syp promotes cell death in immature motoneurons. Interestingly their opposite temporal gradient in Lin A/15 stem cell also determines the end of Lin A/15 stem cell neurogenesis by PCD. Both RNA binding proteins are conserved in vertebrates and seem to play a central role in the number of neurons produce during development. The Drosophila model and its genetic tools offer a unique chance to decipher their function in neural stem cell versus immature neurons.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.02.466666', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Montage electron tomography of vitrified specimens', 'authors': \"[{'name': 'Ariana Peck'}, {'name': 'Stephen D. Carter'}, {'name': 'Huanghao Mai'}, {'name': 'Songye Chen'}, {'name': 'Alister Burt'}, {'name': 'Grant J. Jensen'}]\", 'abstract': '<p>Cryo-electron tomography provides detailed views of macromolecules <italic>in situ</italic>. However, imaging a large field of view to provide more cellular context requires reducing magnification during data collection, which in turn restricts the resolution. To circumvent this trade-off between field of view and resolution, we have developed a montage data collection scheme that uniformly distributes the dose throughout the specimen. In this approach, sets of slightly overlapping circular tiles are collected at high magnification and stitched to form a composite projection image at each tilt angle. These montage tilt-series are then reconstructed into massive tomograms with a small pixel size but a large field of view. For proof-of-principle, we applied this method to the thin edge of HeLa cells. Thon rings to better than 15 Å were observed in the montaged tilt-series, and diverse cellular features were evident in the resulting tomograms. These results indicate that the additional dose required by this technique is not prohibitive to performing structural analysis to intermediate resolution across a large field of view. We anticipate that montage tomography will prove particularly useful for lamellae, increase the likelihood of imaging rare cellular events, and facilitate visual proteomics.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467004', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Discordance between different bioinformatic methods for identifying resistance genes from short-read genomic data, with a focus on Escherichia coli', 'authors': \"[{'name': 'Timothy J Davies'}, {'name': 'Jeremy Swan'}, {'name': 'Anna E Sheppard'}, {'name': 'Hayleah Pickford'}, {'name': 'Samuel Lipworth'}, {'name': 'Manal AbuOun'}, {'name': 'Matthew Ellington'}, {'name': 'Philip W Fowler'}, {'name': 'Susan Hopkins'}, {'name': 'Katie L Hopkins'}, {'name': 'Derrick W Crook'}, {'name': 'Tim EA Peto'}, {'name': 'Muna F Anjum'}, {'name': 'A Sarah Walker'}, {'name': 'Nicole Stoesser'}]\", 'abstract': '<label>2.</label><title>Abstract</title><p>Several bioinformatics genotyping algorithms are now commonly used to characterise antimicrobial resistance (AMR) gene profiles in whole genome sequencing (WGS) data, with a view to understanding AMR epidemiology and developing resistance prediction workflows using WGS in clinical settings. Accurately evaluating AMR in Enterobacterales, particularly <italic>Escherichia coli</italic>, is of major importance, because this is a common pathogen. However, robust comparisons of different genotyping approaches on relevant simulated and large real-life WGS datasets are lacking. Here, we used both simulated datasets and a large set of real <italic>E. coli</italic> WGS data (n=1818 isolates) to systematically investigate genotyping methods in greater detail.</p><p>Simulated constructs and real sequences were processed using four different bioinformatic programs (ABRicate, ARIBA, KmerResistance, and SRST2, run with the ResFinder database) and their outputs compared. For simulations tests where 3,092 AMR gene variants were inserted into random sequence constructs, KmerResistance was correct for all 3,092 simulations, ABRicate for 3,082 (99.7%), ARIBA for 2,927 (94.7%) and SRST2 for 2,120 (68.6%). For simulations tests where two closely related gene variants were inserted into random sequence constructs, ABRicate identified the correct alleles in 11,382/46,279 (25%) of simulations, ARIBA in 2494/46,279 (5%), SRST in 2539/46,279 (5%) and KmerResistance in 38,826/46,279 (84%). In real data, across all methods, 1392/1818 (76%) isolates had discrepant allele calls for at least one gene.</p><p>Our evaluations revealed poor performance in scenarios that would be expected to be challenging (e.g. identification of AMR genes at &lt;10x coverage, discriminating between closely related AMR gene sequences), but also identified systematic sequence classification (i.e. naming) errors even in straightforward circumstances, which contributed to 1081/3092 (35%) errors in our most simple simulations and at least 2530/4321 (59%) discrepancies in real data. Further, many of the remaining discrepancies were likely “artefactual” with reporting cut-off differences accounting for at least 1430/4321 (33%) discrepants. Comparing outputs generated by running multiple algorithms on the same dataset can help identify and resolve these artefacts, but ideally new and more robust genotyping algorithms are needed.</p><sec><label>3.</label><title>Impact statement</title><p>Whole-genome sequencing is widely used for studying the epidemiology of antimicrobial resistance (AMR) genes in bacteria; however, there is some concern that outputs are highly dependent on the bioinformatics methods used. This work evaluates these concerns in detail by comparing four different, commonly used AMR gene typing methods using large simulated and real datasets. The results highlight performance issues for most methods in at least one of several simulated and real-life scenarios. However most discrepancies between methods were due to differential labelling of the same sequences related to the assumptions made regarding the underlying structure of the reference resistance gene database (i.e. that resistance genes can be easily classified in well-defined groups). This study represents a major advance in quantifying and evaluating the nature of discrepancies between outputs of different AMR typing algorithms, with relevance for historic and future work using these algorithms. Some of the discrepancies can be resolved by choosing methods with fewer assumptions about the reference AMR gene database and manually resolving outputs generated using multiple programs. However, ideally new and better methods are needed.</p></sec><sec><label>1.5</label><title>Repositories</title><p>Sequencing data are available at the following NCBI BioProject accession number: PRJNA540750.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466347', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Dorsal striatum coding for the timely execution of action sequences', 'authors': \"[{'name': 'María Cecilia Martinez'}, {'name': 'Camila Lidia Zold'}, {'name': 'Mario Gustavo Murer'}, {'name': 'Mariano Andrés Belluscio'}]\", 'abstract': '<p>The automatic initiation of actions can be highly functional. But occasionally these actions cannot be withheld and are released at inappropriate times, impulsively. Striatal activity has been shown to participate in the timing of action sequence initiation and it has been linked to impulsivity. Using a self- initiated task, we trained adult rats to withhold a rewarded action sequence until a waiting time interval has elapsed. By analyzing neuronal activity we show that the striatal response preceding the initiation of the learned sequence is strongly modulated by the time subjects wait before eliciting the sequence. Interestingly, the modulation is steeper in adolescent rats, which show a strong prevalence of impulsive responses compared to adults. We hypothesize this anticipatory striatal activity reflects the animals’ subjective reward expectation, based on the elapsed waiting time, while its steeper waiting modulation in adolescence reflects age-related differences in temporal discounting, internal urgency states or explore- exploit balance.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466988', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Plasticity in Ligand Recognition at Somatostatin Receptors', 'authors': \"[{'name': 'Michael J. Robertson'}, {'name': 'Justin G. Meyerowitz'}, {'name': 'Ouliana Panova'}, {'name': 'Kenneth Borrelli'}, {'name': 'Georgios Skiniotis'}]\", 'abstract': '<p>Somatostatin is a signaling peptide that plays a pivotal and wide-ranging role in physiologic processes relating to metabolism and growth through its actions at somatostatin receptors (SSTRs). Somatostatin receptors, particularly somatostatin receptor 2, are key drug targets for neuroendocrine neoplasms, with several synthetic peptide agonists currently in use. Here, we present the cryogenic electron microscopy structures of SSTR2 in the active G-protein complex with either the endogenous ligand SST14 or the FDA-approved drug octreotide. Complemented by biochemical assays and molecular dynamics simulations, these structures reveal key details of ligand recognition, receptor activation, and subtype-selectivity at somatostatin receptors. We find that SSTR ligand recognition is highly diverse, as demonstrated by ligand-induced conformational changes in ECL2, substantial sequence divergence across subtypes in extracellular regions, and loss of ligand binding upon several structurally homologous substitutions between subtypes. Despite this complexity, were are able to rationalize several discrete sources of SSTR subtype selectivity and identify an additional key interaction for SSTR2/3/5 specific binding. These results shed light on the basis of ligand recognition by somatostatin receptors and provide valuable insights for structure-based drug discovery at these important targets.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.466696', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'A paradox of parasite resistance: Disease-driven trophic cascades increase the cost of resistance, selecting for lower resistance with parasites than without them', 'authors': \"[{'name': 'Jason Cosens Walsman'}, {'name': 'Alexander T Strauss'}, {'name': 'Jessica Hite'}, {'name': 'Marta S Shocket'}, {'name': 'Spencer R Hall'}]\", 'abstract': \"<p>Most evolutionary theory predicts that, during epidemics, hosts will evolve higher resistance to parasites that kill them. Here, we provide an alternative to that typical expectation, with an explanation centered on resource feedbacks. When resistance is costly, hosts evolve decreasing resistance without parasites, as expected. But with parasites, hosts can evolve lower resistance than they would in the absence of parasites. This outcome arises in an eco-evolutionary model when four conditions are met: first, resistance has a fecundity cost (here, via decreased foraging/exposure rate); second, resources increase during epidemics via trophic cascades; third, increased resources magnify the benefit of maintaining a fast foraging rate, thereby magnifying the cost of evolving a slower foraging/exposure rate (i.e., resistance); fourth, that amplification of the cost outweighs the benefit of resistance. When these conditions are met, hosts evolve lower resistance than without parasites. This phenomenon was previously observed in a mesocosm experiment with fungal parasites, zooplankton hosts, and algal resources. Re-analyzing this experiment produced evidence for our model's mechanism. Thus, both model and experiment indicate that, via resource feedbacks, parasites can counterintuitively select against resistance.</p>\", 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.01.466834', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Glycan-masking spike antigen in NTD and RBD elicits broadly neutralizing antibodies against SARS-CoV-2 variants', 'authors': \"[{'name': 'Wei-Shuo Lin'}, {'name': 'I-Chen Chen'}, {'name': 'Hui-Chen Chen'}, {'name': 'Yi-Chien Lee'}, {'name': 'Suh-Chin Wu'}]\", 'abstract': '<p>Glycan-masking the vaccine antigen by mutating the undesired antigenic sites with an additional N-linked glycosylation motif can refocus B-cell responses to desired/undesired epitopes, without affecting the antigen’s overall-folded structure. This study examine the impact of glycan-masking mutants of the N-terminal domain (NTD) and receptor-binding domain (RBD) of SARS-CoV-2, and found that the antigenic design of the S protein increases the neutralizing antibody titers against the Wuhan-Hu-1 ancestral strain and the recently emerged SARS-CoV-2 variants Alpha (B.1.1.7), Beta (B.1.351), and Delta (B.1.617.2). Our results demonstrated that the use of glycan-masking Ad-S-R158N/Y160T in the NTD elicited a 2.8-fold, 6.5-fold, and 4.6-fold increase in the IC-50 NT titer against the Alpha (B.1.1.7), Beta (B.1.351) and Delta (B.1.617.2) variants, respectively. Glycan-masking of Ad-S-D428N in the RBD resulted in a 3.0-fold and 2.0-fold increase in the IC50 neutralization titer against the Alpha (B.1.1.7) and Beta (B.1.351) variants, respectively. The use of glycan-masking in Ad-S-R158N/Y160T and Ad-S-D428N antigen design may help develop universal COVID-19 vaccines against current and future emerging SARS-CoV-2 variants.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.05.467349', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Sexual reproduction contributes to the evolution of resistance breaking isolates of the spinach pathogen Peronospora effusa', 'authors': \"[{'name': 'Petros Skiadas'}, {'name': 'Joel Klein'}, {'name': 'Thomas Quiroz Monnens'}, {'name': 'Joyce Elberse'}, {'name': 'Ronnie de Jonge'}, {'name': 'Guido Van den Ackerveken'}, {'name': 'Michael F. Seidl'}]\", 'abstract': \"<p>Peronospora effusa causes downy mildew, the economically most important disease of cultivated spinach worldwide. To date, 19 P. effusa races have been denominated based on their capacity to break spinach resistances, but their genetic diversity and the evolutionary processes that contribute to race emergence are unknown. Here, we performed the first systematic analysis of P. effusa races showing that those emerge by both asexual and sexual reproduction. Specifically, we studied the diversity of 26 P. effusa isolates from 16 denominated races based on mitochondrial and nuclear comparative genomics. Mitochondrial genomes based on long-read sequencing coupled with diversity assessment based on short-read sequencing uncovered two mitochondrial haplogroups, each with distinct genome organization. Nuclear genome-wide comparisons of the 26 isolates revealed that ten isolates from six races could clearly be divided into three asexually evolving groups, in concordance with their mitochondrial phylogeny. The remaining isolates showed signals of reticulated evolution and discordance between nuclear and mitochondrial phylogenies, suggesting that these evolved through sexual reproduction. Increased understanding of this pathogen's reproductive modes will provide the framework for future studies into the molecular mechanisms underlying race emergence and into the P. effusa-spinach interaction, thus assisting in sustainable production of spinach through knowledge-driven resistance breeding.</p>\", 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.467086', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'RNA splicing analysis using heterogeneous and large RNA-seq datasets', 'authors': \"[{'name': 'Jorge Vaquero-Garcia'}, {'name': 'Joseph K. Aicher'}, {'name': 'Paul Jewell'}, {'name': 'Matthew R. Gazzara'}, {'name': 'Caleb M. Radens'}, {'name': 'Anupama Jha'}, {'name': 'Christopher J. Green'}, {'name': 'Scott S. Norton'}, {'name': 'Nicholas F. Lahens'}, {'name': 'Gregory R. Grant'}, {'name': 'Yoseph Barash'}]\", 'abstract': '<p>The ubiquity of RNA-seq has led to many methods that use RNA-seq data to analyze variations in RNA splicing. However, available methods are not well suited for handling heterogeneous and large datasets. Such datasets scale to thousands of samples across dozens of experimental conditions, exhibit increased variability compared to biological replicates, and involve thousands of unannotated splice variants resulting in increased transcriptome complexity. We describe here a suite of algorithms and tools implemented in the MAJIQ v2 package to address challenges in detection, quantification, and visualization of splicing variations from such datasets. Using both large scale synthetic data and GTEx v8 as benchmark datasets, we demonstrate that the approaches in MAJIQ v2 outperform existing methods. We then apply MAJIQ v2 package to analyze differential splicing across 2,335 samples from 13 brain subregions, demonstrating its ability to offer new insights into brain subregion-specific splicing regulation.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.467106', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Cas9-Assisted Biological Containment of a Genetically Engineered Human Commensal Bacterium and Genetic Elements', 'authors': \"[{'name': 'Naoki Hayashi'}, {'name': 'Yong Lai'}, {'name': 'Mark Mimee'}, {'name': 'Timothy K. Lu'}]\", 'abstract': '<p>Sophisticated gene circuits built by synthetic biology can enable bacteria to sense their environment and respond predictably. Biosensing bacteria can potentially probe the human gut microbiome to prevent, diagnose, or treat disease. To provide robust biocontainment for engineered bacteria, we devised a Cas9-assisted auxotrophic biocontainment system combining thymidine auxotrophy, an Engineered Riboregulator (ER) for controlled gene expression, and a CRISPR Device (CD). The CD prevents the engineered bacteria from acquiring <italic>thyA</italic> via horizontal gene transfer, which would disrupt the biocontainment system, and inhibits the spread of genetic elements by killing bacteria harboring the gene cassette. This system tunably controlled gene expression in the human gut commensal bacterium <italic>Bacteroides thetaiotaomicron</italic>, prevented escape from thymidine auxotrophy, and blocked transgene dissemination for at least 10 days. These capabilities were validated <italic>in vitro</italic> and <italic>in vivo.</italic> This biocontainment system exemplifies a powerful strategy for bringing genetically engineered microorganisms safely into biomedicine.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.05.467434', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Multiplex genomic recording of enhancer and signal transduction activity in mammalian cells', 'authors': \"[{'name': 'Wei Chen'}, {'name': 'Junhong Choi'}, {'name': 'Jenny F. Nathans'}, {'name': 'Vikram Agarwal'}, {'name': 'Beth Martin'}, {'name': 'Eva Nichols'}, {'name': 'Anh Leith'}, {'name': 'Choli Lee'}, {'name': 'Jay Shendure'}]\", 'abstract': '<p>Measurements of gene expression and signal transduction activity are conventionally performed with methods that require either the destruction or live imaging of a biological sample within the timeframe of interest. Here we demonstrate an alternative paradigm, termed ENGRAM (ENhancer-driven Genomic Recording of transcriptional Activity in Multiplex), in which the activity and dynamics of multiple transcriptional reporters are stably recorded to DNA. ENGRAM is based on the prime editing-mediated insertion of signal- or enhancer-specific barcodes to a genomically encoded recording unit. We show how this strategy can be used to concurrently genomically record the relative activity of at least hundreds of enhancers with high fidelity, sensitivity and reproducibility. Leveraging synthetic enhancers that are responsive to specific signal transduction pathways, we further demonstrate time- and concentration-dependent genomic recording of Wnt, NF-κB, and Tet-On activity. Finally, by coupling ENGRAM to sequential genome editing, we show how serially occurring molecular events can potentially be ordered. Looking forward, we envision that multiplex, ENGRAM-based recording of the strength, duration and order of enhancer and signal transduction activities has broad potential for application in functional genomics, developmental biology and neuroscience.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.10.31.21265711', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Cross-border transmissions of delta substrain AY.29 during Olympic and Paralympic Games', 'authors': \"[{'name': 'Takahiko Koyama'}, {'name': 'Reitaro Tokumasu'}, {'name': 'Kotoe Katayama'}, {'name': 'Ayumu Saito'}, {'name': 'Michiharu Kudo'}, {'name': 'Seiya Imoto'}]\", 'abstract': '<p>Tokyo Olympic and Paralympic Games, postponed for COVID-19 pandemic, were finally held in summer of 2021. Just before the games, alpha variant was being replaced with more contagious delta variant (B.1.617.2). AY.4 substrain AY.29, which harbors two additional characteristic mutations of 5239C&gt;T (NSP3 Y840Y) and 5514T&gt;C (NSP3 V932A), emerged in Japan and became the dominant strain in Tokyo by the time of the Olympic Games. As of October 18, 98 AY.29 samples are identified in 16 countries outside of Japan. Phylogenetic analysis and ancestral searches identified 46 distinct introductions of AY.29 strains into those 16 countries. United States has 44 samples with 10 distinct introductions, and United Kingdom has 13 distinct AY.29 strains introduced in 16 samples. Other countries or regions with multiple introductions of AY.29 are Canada, Germany, South Korea, and Hong Kong while Italy, France, Spain, Sweden, Belgium, Peru, Australia, New Zealand, and Indonesia have only one distinct strain introduced. There exists no unambiguous evidence that Olympic and Paralympic Games induced cross-border transmission of the delta substrain AY.29. Since most of unvaccinated countries are also under sampled for genome analysis with longer lead time for data sharing, it will take longer to capture the whole picture of cross-border transmissions of AY.29.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.466744', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Live-cell chromosome dynamics in Arabidopsis thaliana reveals increased chromatin mobility in response to DNA damage', 'authors': \"[{'name': 'Anis Meschichi'}, {'name': 'Svenja Reeck'}, {'name': 'Adrien Sicard'}, {'name': 'Frédéric Pontvianne'}, {'name': 'Stefanie Rosa'}]\", 'abstract': '<p>Double-strand breaks (DSBs) are a particularly deleterious type of DNA damage potentially leading to translocations and genome instability. Homologous recombination (HR) is a conservative repair pathway in which intact homologous sequences are used as a template for repair. How damaged DNA molecules search for homologous sequences in the crowded space of the cell nucleus is, however, still poorly understood. Here, we measured global chromosome and DSB site mobility, in <italic>Arabidopsis thaliana</italic>, using <italic>lacO</italic>/LacI tagged lines and two GFP-tagged HR regulators, RAD51 and RAD54. We observed an increase in chromatin mobility upon the induction of DNA damage, specifically at the S/G2 phases of the cell cycle. Importantly, this increase in mobility was lost in <italic>sog1-1</italic> mutant, a central transcription factor of the DNA damage response (DDR), indicating that repair mechanisms actively regulate chromatin mobility upon DNA damage. Interestingly, we observed that DSB sites show remarkably high mobility levels at the early HR stage. Subsequently, a drastic decrease of DSB mobility is observed, which seems to be associated to the relocation of DSBs to the nucleus periphery. Altogether, our data suggest that changes in chromatin mobility are triggered in response to DNA damage, and that this may act as a mechanism to enhance the physical search within the nuclear space to locate a homologous template during homology-directed DNA repair.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.467017', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'RAP1 regulates TIP60 function during fate transition between 2 cell-like and pluripotent states', 'authors': \"[{'name': 'Raymond Mario Barry'}, {'name': 'Olivia Sacco'}, {'name': 'Amel Mameri'}, {'name': 'Martin Stojaspal'}, {'name': 'William Kartsonis'}, {'name': 'Pooja Shah'}, {'name': 'Pablo De Ioannes'}, {'name': 'Ctirad Hofr'}, {'name': 'Jacques Côté'}, {'name': 'Agnel Sfeir'}]\", 'abstract': '<p>In mammals, the conserved telomere binding protein RAP1 serves a diverse set of non- telomeric functions including activation of the NF-kB signaling pathway, maintenance of metabolic function <italic>in vivo,</italic> and transcriptional regulation. Here, we uncover the mechanism by which RAP1 modulates gene expression. Using a separation-of-function allele, we show that RAP1 transcriptional regulation is independent of TRF2-mediated binding to telomeres and does not involve direct binding to genomic loci. Instead, RAP1 interacts with the TIP60/p400 complex and modulates its histone acetyltransferase activity. Notably, we show that deletion of RAP1 in mouse embryonic stem cells increases the fraction of 2-cell-like cells. Specifically, RAP1 enhances the repressive activity of Tip60/p400 across a subset of 2-cell-stage genes, including <italic>Zscan4</italic> and the endogenous retrovirus MERVL. Preferential upregulation of genes proximal to MERVL elements in Rap1 deficient settings implicate these endogenous retroviral elements in the de- repression of proximal genes. Altogether, our study reveals an unprecedented link between RAP1 and TIP60/p400 complex in the regulation of totipotency.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466863', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'mRNA Vaccines Induce Rapid Antibody Responses in Mice', 'authors': \"[{'name': 'Makda S. Gebre'}, {'name': 'Susanne Rauch'}, {'name': 'Nicole Roth'}, {'name': 'Janina Gergen'}, {'name': 'Jingyou Yu'}, {'name': 'Xiaowen Liu'}, {'name': 'Andrew C. Cole'}, {'name': 'Stefan O. Mueller'}, {'name': 'Benjamin Petsch'}, {'name': 'Dan H. Barouch'}]\", 'abstract': '<p>mRNA vaccines can be developed and produced quickly, making them attractive for immediate outbreak responses. Furthermore, clinical trials have demonstrated rapid protection following mRNA vaccination. We sought to investigate how quickly mRNA vaccines elicit antibody responses compared to other vaccine modalities. We first examined immune kinetics of mRNA and DNA vaccines expressing SARS-CoV-2 spike in mice. We observed rapid induction of antigen-specific binding and neutralizing antibodies by day 5 following mRNA, but not DNA, immunization. The mRNA vaccine also induced increased levels of IL-5, IL-6 and MCP-1. We then evaluated immune kinetics of an HIV-1 mRNA vaccine in comparison to DNA, protein, and rhesus adenovirus 52 (RhAd52) vaccines with the same HIV-1 envelope antigen in mice. Induction of envelope-specific antibodies was observed by day 5 following mRNA vaccination, whereas antibodies were detected by day 7-14 following DNA, protein, and RhAd52 vaccination. Eliciting rapid humoral immunity may be an advantageous property of mRNA vaccines for controlling infectious disease outbreaks.</p><sec><title>IMPORTANCE</title><p>mRNA vaccines can be developed and produced in record time. Here we demonstrate induction of rapid antibody responses by mRNA vaccines encoding two different viral antigens by day 5 following immunization in mice. The rapid immune kinetics of mRNA vaccines can be an advantageous property that makes them well suited for rapid control of infectious disease outbreaks.</p></sec>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.03.467112', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Time-dependent regulation of cytokine production by RNA binding proteins defines T cell effector function', 'authors': \"[{'name': 'Branka Popović'}, {'name': 'Aurélie Guislain'}, {'name': 'Sander Engels'}, {'name': 'Benoît P. Nicolet'}, {'name': 'Anouk P. Jurgens'}, {'name': 'Natali Paravinja'}, {'name': 'Julian J. Freen-van Heeren'}, {'name': 'Floris P.J. van Alphen'}, {'name': 'Maartje van den Biggelaar'}, {'name': 'Fiamma Salerno'}, {'name': 'Monika C. Wolkers'}]\", 'abstract': '<p>Effective T cell responses against target cells require controlled production of the key pro-inflammatory cytokines IFN-γ, TNF and IL-2. Post-transcriptional events determine the magnitude and duration of cytokine production in T cells, a process that is largely regulated by RNA binding proteins (RBPs). Here we studied the identity and mode of action of RBPs interacting with cytokine mRNAs. With an RNA aptamer-based capture assay from human T cell lysates, we identified &gt;130 RBPs interacting with the full length 3’untranslated regions of <italic>IFNG</italic>, <italic>TNF</italic> and <italic>IL2.</italic> The RBP landscape altered upon T cell activation. Furthermore, RBPs display temporal activity profiles to control cytokine production. Whereas HuR promotes early cytokine production, the peak production levels and response duration is controlled by ZFP36L1, ATXN2L and ZC3HAV1. Importantly, ZFP36L1 deletion boosts T cell responses against tumors <italic>in vivo,</italic> revealing the potential of the RBP map to identify critical modulators of T cell responses.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.21265876', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Investigating the relationship between interventions, contact patterns, and SARS-CoV-2 transmissibility', 'authors': \"[{'name': 'Filippo Trentini'}, {'name': 'Adriana Manna'}, {'name': 'Nicoletta Balbo'}, {'name': 'Valentina Marziano'}, {'name': 'Giorgio Guzzetta'}, {'name': 'Stefano Merler'}, {'name': 'Marco Ajelli'}, {'name': 'Piero Poletti'}, {'name': 'Alessia Melegaro'}]\", 'abstract': '<sec><title>Background</title><p>After a rapid upsurge of COVID-19 cases in Italy during the fall of 2020, the government introduced a three-tiered restriction system aimed at increasing physical distancing. The Ministry of Health, after periodic epidemiological risk assessments, assigned a tier to each of the 21 Italian regions and autonomous provinces (AP). It is still unclear to what extent these different measures altered mixing patterns and how quickly the population adapted their social interactions to continuous changes in restrictions.</p></sec><sec><title>Methods and findings</title><p>We conducted a survey between July 2020 and March 2021 to monitor changes in social contact patterns among individuals in the metropolitan city of Milan, Italy, which was hardly hit by the second wave of COVID-19 pandemic. The number of contacts during periods characterized by different levels of restrictions was analyzed through negative binomial regression models and age-specific contact matrices were estimated under the different tiers. Relying on the empirically estimated mixing patterns, we quantified relative changes in SARS-CoV-2 transmission potential associated with the different tiers.</p><p>As tighter restrictions were implemented during the fall of 2020, a progressive reduction in the mean number of contacts recorded by study participants was observed: from 16.4% under mild restrictions (yellow tier), to 45.6% under strong restrictions (red tier). Higher restrictions levels were also found to increase the relative contribution of contacts occurring within the household. The SARS-CoV-2 reproduction number was estimated to decrease by 18.7% (95%CI: 4.6-30.8), 33.4% (95%CI: 22.7-43.2), and 50.2% (95%CI: 40.9-57.7) under the yellow, orange, and red tiers, respectively.</p></sec><sec><title>Conclusions</title><p>Our results give an important quantification of the expected contribution of different restriction levels in shaping social contacts and decreasing the transmission potential of SARS-CoV-2. These estimates can find an operational use in anticipating the effect that the implementation of these tiered restriction can have on SARS-CoV-2 reproduction number under an evolving epidemiological situation.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.05.467329', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Cooperativity mediates drug resistance and metabolism in Plasmodium falciparum malaria parasites', 'authors': \"[{'name': 'Andrew J Jezewski'}, {'name': 'Ann M Guggisberg'}, {'name': 'Dana M Hodge'}, {'name': 'Naomi Ghebremichael'}, {'name': 'Lisa K. McLellan'}, {'name': 'Audrey Ragan Odom John'}]\", 'abstract': '<p>Efforts to control the global malaria health crisis are undermined by antimalarial resistance. Identifying mechanisms of resistance will uncover the underlying biology of the <italic>Plasmodium falciparum</italic> malaria parasites that allow evasion of our most promising therapeutics and may reveal new drug targets. We utilized fosmidomycin (FSM) as a chemical inhibitor of plastidial isoprenoid biosynthesis through the methylerythritol phosphate (MEP) pathway. We have thus identified an unusual metabolic regulation scheme in the malaria parasite through the essential glycolytic enzyme, glyceraldehyde 3-phosphate dehydrogenase (GAPDH). Two parallel genetic screens converged on independent but functionally analogous resistance alleles in GAPDH. Metabolic profiling of FSM-resistant gapdh mutant parasites indicates that neither of these mutations disrupt overall glycolytic output. While FSM-resistant GAPDH variant proteins are catalytically active, they have reduced assembly into the homotetrameric state favored by wild-type GAPDH. Disrupted oligomerization of FSM-resistant GAPDH variant proteins is accompanied by altered enzymatic cooperativity and reduced susceptibility to inhibition by free heme. Together, our data identifies a new genetic biomarker of FSM-resistance and reveals the central role of GAPDH cooperativity in MEP pathway control and antimalarial sensitivity.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.04.467340', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'A comparative analysis of current phasing and imputation software', 'authors': \"[{'name': 'Puya G Yazdi'}, {'name': 'Adriano De Marino'}, {'name': 'Abdallah Amr Mahmoud'}, {'name': 'Madhuchanda Bose'}, {'name': 'Karatuğ Ozan Bircan'}, {'name': 'Andrew Terpolovsky'}, {'name': 'Varuna Bamunusinghe'}, {'name': 'Umar Khan'}, {'name': 'Biljana Novković'}]\", 'abstract': '<p>Whole-genome data has become significantly more accessible over the last two decades. This can largely be attributed to both reduced sequencing costs and imputation models which make it possible to obtain nearly whole-genome data from less expensive genotyping methods, such as microarray chips. Although there are many different approaches to imputation, the Hidden Markov Model remains the most widely used. In this study, we compared the latest versions of the most popular Hidden Markov Model based tools for phasing and imputation: Beagle 5.2, Eagle 2.4.1, Shapeit 4, Impute 5 and Minimac 4. We benchmarked them on three input datasets with three levels of chip density. We assessed each imputation software on the basis of accuracy, speed and memory usage, and showed how the choice of imputation accuracy metric can result in different interpretations. The highest average concordance rate was achieved by Beagle 5.2, followed by Impute 5 and Minimac 4, using a reference-based approach during phasing and the highest density chip. IQS and R  2  metrics revealed that IMPUTE5 obtained better results for low frequency markers, while Beagle 5.2 remained more accurate for common markers (MAF&gt;5%). Computational load as measured by run time was lower for Beagle 5.2 than Impute 5 and Minimac 4, while Minimac utilized the least memory of the imputation tools we compared. ShapeIT 4, used the least memory of the phasing tools examined, even with the highest density chip. Finally, we determined the combination of phasing software, imputation software, and reference panel, best suited for different situations and analysis needs and created an automated pipeline that provides a way for users to create customized chips designed to optimize their imputation results.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.04.467355', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Toward optimal fingerprint indexing for large scale genomics', 'authors': \"[{'name': 'Clement Agret'}, {'name': 'Bastien Cazaux'}, {'name': 'Antoine Limasset'}]\", 'abstract': '<p>Motivation: To keep up with the scale of genomic databases, several methods rely on local sensitive hashing methods to efficiently find potential matches within large genome collections. Existing solutions rely on Minhash or Hyperloglog fingerprints and require reading the whole index to perform a query. Such solutions can not be considered scalable with the growing amount of documents to index. Results: We present NIQKI, a novel structure using well-designed fingerprints that lead to theoretical and practical query time improvements, outperforming state-of-the-art by orders of magnitude. Our contribution is threefold. First, we generalize the concept of Hyperminhash fingerprints in (h,m)-HMH fingerprints that can be tuned to present the lowest false positive rate given the expected sub-sampling applied. Second, we provide a structure able to index any kind of fingerprints based on inverted indexes that provide optimal queries, namely linear with the size of the output. Third, we implemented these approaches in a tool dubbed NIQKI that can index and calculate pairwise distances for over one million bacterial genomes from GenBank in a matter of days on a small cluster. We show that our approach can be orders of magnitude faster than state-of-the-art with comparable precision. We believe that this approach can lead to tremendous improvement allowing fast query, scaling on extensive genomic databases. Availability and implementation: We wrote the NIQKI index as an open-source C++ library under the AGPL3 license available at https://github.com/Malfoy/ NIQKI. It is designed as a user-friendly tool and comes along with usage sample</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.21265881', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'Fidaxomicin as first line: What will it cost in the USA and Canada?', 'authors': \"[{'name': 'Devangi Patel'}, {'name': 'Julien Senecal'}, {'name': 'Brad Spellberg'}, {'name': 'Andrew M. Morris'}, {'name': 'Lynora Saxinger'}, {'name': 'Brent W. Footer'}, {'name': 'Emily G. McDonald'}, {'name': 'Todd C. Lee'}]\", 'abstract': '<sec><title>Importance</title><p>Recent changes in the Infectious Diseases and Healthcare Epidemiology Societies of America (IDSA-SHEA) guidelines for managing <italic>Clostridioides difficile</italic> infections (CDI) have placed fidaxomicin as first-line treatment for CDI.</p></sec><sec><title>Objective</title><p>To estimate the net cost of first line fidaxomicin as compared to vancomycin in the American and Canadian healthcare systems and to estimate the price points at which fidaxomicin would become cost saving.</p></sec><sec><title>Data sources</title><p>We identified all randomized controlled trials comparing fidaxomicin with vancomycin through the 2021 IDSA-SHEA guideline update. Medication costs were obtained from wholesale prices (US) and the Quebec drug formulary (Canada). The average cost of a CDI recurrence was established through two systematic reviews using PubMed.</p></sec><sec><title>Study selection</title><p>For fidaxomicin efficacy, we included double-blind and placebo-controlled trials. For the systematic review of recurrence costs, studies were included if they were primary research articles, had a cost-analysis of CDI, included cases of recurrent CDI, and were calculated with cost parameters from American or Canadian healthcare systems. Studies were excluded if the population was solely pediatric or hospitalized.</p></sec><sec><title>Data extraction and Synthesis</title><p>For the efficacy meta-analysis, data was pooled using a random effects model. For the costs review, literature screening was performed by 2 independent reviewers. The mean cost across identified studies was adjusted to 2021 dollars.</p></sec><sec><title>Main Outcomes and Measures</title><p>The primary outcome of the meta-analysis was CDI recurrence at day 40. The primary outcome of the systematic review was the average cost of a CDI recurrence in the American and Canadian healthcare systems. The objective was to estimate the net cost per recurrence prevented and the price point below which fidaxomicin would be cost saving.</p></sec><sec><title>Results</title><p>At current drug pricing, the estimated additional cost of a 10-day course of fidaxomicin compared to vancomycin in order to prevent one recurrence was $46,178USD (95%CI $36,942-$69,267) and $13,760CAD (95%CI $11,008-$20,640), respectively. The estimated mean systemic cost of a CDI recurrence was $14,506USD and $8,588CAD, respectively. When priced below $1550USD and $800CAD, fidaxomicin was likely to become cost saving.</p></sec><sec><title>Conclusions and Relevance</title><p>The increased drug expenditure on fidaxomicin will not be offset through recurrence prevention unless fidaxomicin price is re-negotiated.</p></sec><sec><title>Key Points</title><sec><title>Question</title><p>At what price point does new guideline recommendations for changing from vancomycin to fidaxomicin for first line <italic>C. difficile</italic> infection (CDI) therapy become cost saving?</p></sec><sec><title>Findings</title><p>For a 10-day course, we estimated that compared to vancomycin, treatment with fidaxomicin costs an additional $46,178USD to prevent one CDI recurrence and that treating one recurrence costs $14,506USD. The current cost of fidaxomicin is only partially offset through prevention of recurrences. Upfront therapy with fidaxomicin would be cost saving if the price were reduced below $1550USD per treatment.</p></sec><sec><title>Meaning</title><p>New guideline updates could add billions in additional drug costs for CDI treatment.</p></sec></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.21265863', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Downsizing of contact tracing during COVID-19 vaccine roll-out', 'authors': \"[{'name': 'Maria M. Martignoni'}, {'name': 'Josh Renault'}, {'name': 'Joseph Baafi'}, {'name': 'Amy Hurford'}]\", 'abstract': '<p>Contact tracing is a key component of successful management of COVID-19. Contacts of infected individuals are asked to quarantine, which can significantly slow down (or prevent) community spread. Contact tracing is particularly effective when infections are detected quickly (e.g., through rapid testing), when contacts are traced with high probability, when the initial number of cases is low, and when social distancing and border restrictions are in place. However, the magnitude of the individual contribution of these factors in reducing epidemic spread and the impact of vaccination in determining contact tracing outputs is not fully understood. We present a delayed differential equation model to investigate how vaccine roll-out and the relaxation of social distancing requirements affect contact tracing practises. We provide an analytical criteria to determine the minimal contact tracing efficiency (defined as the the probability of identifying and quarantining contacts of symptomatic individuals) required to keep an outbreak under control, with respect to the contact rate and vaccination status of the population. Additionally, we consider how delays in outbreak detection and increased case importation rates affect the number of contacts to be traced daily. We show that in vaccinated communities a lower contact tracing efficiency is required to avoid uncontrolled epidemic spread, and delayed outbreak detection and relaxation of border restrictions do not lead to a significantly higher risk of overwhelming contact tracing. We find that investing in testing programs, rather than increasing the contact tracing capacity, has a larger impact in determining whether an outbreak will be controllable. This is because early detection activates contact tracing, which will slow, and eventually reverse exponential growth, while the contact tracing capacity is a threshold that will easily become overwhelmed if exponential growth is not curbed. Finally, we evaluate quarantine effectiveness during vaccine roll-out, by considering the proportion of people that will develop an infection while in isolation in relation to the vaccination status of the population and for different viral variants. We show that quarantine effectiveness decreases with increasing proportion of fully vaccinated individuals, and increases in the presence of more transmissible variants. These results suggest that a cost-effective approach during vaccine roll-out is to establish different quarantine rules for vaccinated and unvaccinated individuals, where rules should depend on viral trans-missibility. Altogether, our study provides quantitative information for contact tracing downsizing during vaccine roll-out, to guide COVID-19 exit strategies.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.05.467060', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'De novo assembly of 20 chickens reveals the undetectable phenomenon for thousands of core genes on sub-telomeric regions', 'authors': \"[{'name': 'Ming Li'}, {'name': 'Congjiao Sun'}, {'name': 'Naiyi Xu'}, {'name': 'Peipei Bian'}, {'name': 'Xiaomeng Tian'}, {'name': 'Xihong Wang'}, {'name': 'Yuzhe Wang'}, {'name': 'Xinzheng Jia'}, {'name': 'Rasmus Heller'}, {'name': 'Mingshan Wang'}, {'name': 'Fei Wang'}, {'name': 'Xuelei Dai'}, {'name': 'Rongsong Luo'}, {'name': 'Yingwei Guo'}, {'name': 'Xiangnan Wang'}, {'name': 'Peng Yang'}, {'name': 'Shunjin Zhang'}, {'name': 'Xiaochang Li'}, {'name': 'Chaoliang Wen'}, {'name': 'Fangren Lan'}, {'name': 'AMAM Z SIDDIKI'}, {'name': 'Chatmongkon Suwannapoom'}, {'name': 'Xin Zhao'}, {'name': 'Qinghua Nie'}, {'name': 'Xiaoxiang Hu'}, {'name': 'Yu Jiang'}, {'name': 'Ning Yang'}]\", 'abstract': '<p>The gene numbers and evolutionary rates of birds were assumed to be much lower than that of mammals, which in sharp contrast to the huge species number and morphological diversity of birds. It is very necessary to construct a complete avian genome and analyze its evolution.We constructed a chicken pan-genome from 20 de novo genome assemblies with high sequencing depth, newly identified 1,335 protein-coding genes and 3,011 long noncoding RNAs. The majority of these novel genes were detected across most individuals of the examined transcriptomes but were accidentally measured in each of the DNA sequencing data regardless of Illumina or PacBio technology. Furthermore, different from previous pan-genome models, most of these novel genes were overrepresented on chromosomal sub-telomeric regions, surrounded with extremely high proportions of tandem repeats, and strongly blocked DNA sequencing. These hidden genes were proved to be shared by all chicken genomes, included many housekeeping genes, and enriched in immune pathways. Comparative genomics revealed the novel genes had three-fold elevated substitution rates than known ones, updating the evolutionary rates of birds. Our study provides a framework for constructing a better chicken genome, which will contribute towards the understanding of avian evolution and improvement of poultry breeding.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.02.466951', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': True, 'title': 'Drug targeting Nsp1-ribosomal complex shows antiviral activity against SARS-CoV-2', 'authors': \"[{'name': 'Mohammad Afsar'}, {'name': 'Rohan Narayan'}, {'name': 'Md Noor Akhtar'}, {'name': 'Huma Rahil'}, {'name': 'Sandeep M Eswarappa'}, {'name': 'Shashank Tripathi'}, {'name': 'Tanweer Hussain'}]\", 'abstract': '<p>The SARS-Cov-2 non-structural protein 1 (Nsp1) contains an N-terminal domain and C-terminal helices connected by a short linker region. The C-terminal helices of Nsp1 (Nsp1-C-ter) from SARS-Cov-2 bind in the mRNA entry channel of the 40S ribosomal subunit and block the entry of mRNAs thereby shutting down host protein synthesis. Nsp1 suppresses the host immune function and is vital for viral replication. Hence, Nsp1 appears to be an attractive target for therapeutics. In this study, we have <italic>in silico</italic> screened Food and Drug Administration (FDA)-approved drugs against Nsp1-C-ter and find that montelukast sodium hydrate binds to Nsp1-C-ter with a binding affinity (K<sub>D</sub>) of 10.8±0.2 μM <italic>in vitro</italic> and forms a stable complex with it in simulation runs with a binding energy of −76.71±8.95 kJ/mol. The drug also rescues the inhibitory effect of Nsp1 in host protein synthesis as demonstrated by the expression of firefly luciferase reporter gene in cells. Importantly, montelukast sodium hydrate demonstrates antiviral activity against SARS-CoV-2 with reduced viral replication in HEK cells expressing ACE2 and Vero-E6 cells. We therefore propose montelukast sodium hydrate may help in combatting SARS-CoV-2 infection.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466589', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Loop-extrusion and polymer phase-separation can co-exist at the single-molecule level to shape chromatin folding', 'authors': \"[{'name': 'Mattia Conte'}, {'name': 'Ehsan Irani'}, {'name': 'Andrea M. Chiariello'}, {'name': 'Alex Abraham'}, {'name': 'Simona Bianco'}, {'name': 'Andrea Esposito'}, {'name': 'Mario Nicodemi'}]\", 'abstract': '<p>Loop-extrusion and phase-separation have been proposed as mechanisms that shape chromosome large-scale spatial organization. It is unclear, however, how they perform relative to each other in explaining chromatin architecture data and whether they compete or co-exist at the single-molecule level. Here, we compare models of polymer physics based on loop-extrusion and phase-separation, as well as models where both mechanisms act simultaneously in a single molecule, against multiplexed FISH data available in human loci in IMR90 and HCT116 cells. We find that the different models recapitulate bulk Hi-C and average microscopy data. Single-molecule chromatin conformations are also well captured, especially by phase-separation based models that better reflect the experimentally reported segregation in globules of the considered genomic loci and their cell-to-cell structural variability. Such a variability is consistent with two main concurrent causes: single-cell epigenetic heterogeneity and an intrinsic thermodynamic conformational degeneracy of folding. Overall, the model combining loop-extrusion and polymer phase-separation provides a very good description of the data, particularly higher-order contacts, showing that the two mechanisms can co-exist in shaping chromatin architecture in single cells.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.02.21265668', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Black and Native Overdose Mortality Overtook that of White Individuals During the COVID-19 Pandemic', 'authors': \"[{'name': 'Joseph Friedman'}, {'name': 'Helena Hansen'}]\", 'abstract': '<p>Drug overdose mortality rates have increased sharply during the COVID-19 pandemic. In recent years, overdose death rates were rising most rapidly among racial/ethnic minority communities. The pandemic has disproportionately affected communities of color in a wide swath of health, social, and economic outcomes. Careful attention is therefore warranted to trends in overdose mortality by race/ethnicity during COVID-19. We calculated total drug overdose death rates per 100,000 population by race/ethnicity for the 1999-2020 time period. We find that Black overdose mortality overtook that of White individuals in 2020 for the first time since 1999. Between 2019 and 2020 Black individuals had the largest percent increase in overdose mortality, of 48.8%, compared to 26.3% among White individuals. In 2020, Black overdose death rates rose to 36.8 per 100,000, representing 16.3% higher than the rate for White individuals for the same period. American Indian and Alaska Native (AI/AN) individuals experienced the highest rate of overdose mortality in 2020, of 41.4 per 100,000, representing 30.8% higher than the rate among White individuals. Our findings suggest that drug overdose mortality is increasingly becoming a racial justice issue in the United States and appears to have been exacerbated by the COVID-19 pandemic. Providing individuals with a safer supply of drugs, closing gaps in access to MOUD and harm reductions services, and ending routine incarceration of individuals with substance use disorders represent urgently needed, evidence-based strategies that can be employed to reduce rising inequalities in overdose.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.05.467513', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Single-Cell Multiomics Defines Tolerogenic Extrathymic Aire-Expressing Populations with Unique Homology to Thymic Epithelium', 'authors': \"[{'name': 'Jiaxi Wang'}, {'name': 'Caleb A Lareau'}, {'name': 'Jhoanne L Bautista'}, {'name': 'Alexander R Gupta'}, {'name': 'Katalin Sandor'}, {'name': 'Joe Germino'}, {'name': 'Yajie Yin'}, {'name': 'Matthew P Arvedson'}, {'name': 'Gabriella C Reeder'}, {'name': 'Nathan T Cramer'}, {'name': 'Fang Xie'}, {'name': 'Vasilis Ntranos'}, {'name': 'Ansuman T Satpathy'}, {'name': 'Mark S Anderson'}, {'name': 'James M Gardner'}]\", 'abstract': '<p>The Autoimmune Regulator (Aire) gene, well defined for its role in medullary thymic epithelial cells (mTECs) and immune self-tolerance, is also expressed in extrathymic Aire-expressing cells (eTACs) in the secondary lymphoid organs. eTACs have been shown to be hematopoietic antigen presenting cells (APCs) and potent inducers of immune tolerance. However, the precise identity and function of these cells remain unclear. Here, we use high-dimensional single-cell multiomics and functional approaches to define eTACs at the transcriptional, genomic, and proteomic level. We find that eTACs consist of two similar cell types: CCR7+ Aire-expressing migratory dendritic cells (AmDCs) and a unique Aire-hi population co-expressing Aire and RAR-related orphan receptor gamma-t (RORγt). The latter, which have significant transcriptional and genomic homology to migratory dendritic cells (migDCs) and mTECs, we term Janus cells (JCs). All eTACs, and JCs in particular, have a highly accessible chromatin structure and high levels of broad gene expression, including tissue-specific antigens, as well as remarkable transcriptional and genomic homology to thymic medullary epithelium. As in the thymus, Aire expression in eTACs is also dependent on RANK-RANK-ligand interactions. Furthermore, lineage-tracing shows that JCs are not precursors to the majority of AmDCs. Finally, self-antigen expression by eTACs is sufficient to mediate negative selection of T cells escaping thymic selection and can prevent autoimmune diabetes in non-obese diabetic mice. This transcriptional, genomic, and functional symmetry between a hematopoietic Aire-expressing population in the periphery and an epithelial Aire-expressing population in the thymus suggests that a core biological program may influence self-tolerance and self-representation across the spectrum of immune development.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.04.467241', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Intracellular development and impact of a eukaryotic parasite on its zombified microalgal host in the marine plankton', 'authors': \"[{'name': 'Johan Decelle'}, {'name': 'Ehsan Kayal'}, {'name': 'Estelle Bigeard'}, {'name': 'Benoit Gallet'}, {'name': 'Jeremy Bougoure'}, {'name': 'Peta Clode'}, {'name': 'Nicole Schieber'}, {'name': 'Rachel Templin'}, {'name': 'Elisabeth Hegenberger'}, {'name': 'Gerard Prensier'}, {'name': 'Fabien Chevalier'}, {'name': 'Yannick Schwab'}, {'name': 'Laure Guillou'}]\", 'abstract': '<p>Parasites are widespread and diverse in the oceanic plankton, and many of them infect single-celled algae for survival. How these parasites develop and scavenge energy within the host and whether the cellular organization and metabolism of the host is altered remain open questions. Combining quantitative structural and chemical imaging with time-resolved transcriptomics, we unveil dramatic morphological and metabolic changes of the parasite Amoebophrya (Syndiniales) during intracellular infection (e.g. 200-fold increase of mitochondrion volume), particularly following digestion of nutrient-rich host chromosomes. Some of these changes are also found in the apicomplexan parasites (e.g. sequential acristate and cristate mitochondrion, switch from glycolysis to TCA), thus underlining key evolutionary-conserved mechanisms. In the algal host, energy-producing organelles (chloroplast) remain intact during most of the infection, but sugar reserves diminish while lipid droplets increase. Thus, rapid infection of the host nucleus could be a zombifying strategy to digest nutrient-rich chromosomes and escape cytoplasmic defense while benefiting from the maintained C-energy production of the host cell.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.10.31.21265717', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'A multi-ethnic polygenic risk score is associated with hypertension prevalence and progression throughout adulthood', 'authors': \"[{'name': 'Nuzulul Kurniansyah'}, {'name': 'Matthew O Goodman'}, {'name': 'Tanika Kelly'}, {'name': 'Tali Elfassi'}, {'name': 'Kerri L Wiggins'}, {'name': 'Joshua C Bis'}, {'name': 'Xiuqing Guo'}, {'name': 'Walter Palmas'}, {'name': 'Kent D Taylor'}, {'name': 'Henry J Lin'}, {'name': 'Jeffrey Haessler'}, {'name': 'Yan Gao'}, {'name': 'Daichi Shimbo'}, {'name': 'Jennifer A Smith'}, {'name': 'Bing Yu'}, {'name': 'Elena Feofanova'}, {'name': 'Roelof Smit'}, {'name': 'Zhe Wang'}, {'name': 'Shih-Jen Hwang'}, {'name': 'Simin Liu'}, {'name': 'Sylvia Wassertheil-Smoller'}, {'name': 'JoAnn E Manson'}, {'name': 'Donald M Lloyd-Jones'}, {'name': 'Stephen S Rich'}, {'name': 'Ruth JF Loos'}, {'name': 'Susan Redline'}, {'name': 'Adolfo Correa'}, {'name': 'Charles Kooperberg'}, {'name': 'Myriam Fornage'}, {'name': 'Robert C Kaplan'}, {'name': 'Bruce M Psaty'}, {'name': 'Jerome I Rotter'}, {'name': 'Donna K Arnett'}, {'name': 'Alanna C Morrison'}, {'name': 'Nora Franceschini'}, {'name': 'Daniel Levy'}, {'name': 'Tamar Sofer'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>We used summary statistics from previously-published GWAS of systolic and diastolic BP and of hypertension to construct Polygenic Risk Scores (PRS) to predict hypertension across diverse populations.</p></sec><sec><title>Methods</title><p>We used 10,314 participants of diverse ancestry from BioMe to train trait-specific PRS. We implemented a novel approach to select one of multiple potential PRS based on the same GWAS, by optimizing the coefficient of variation across estimated PRS effect sizes in independent subsets of the training dataset. We combined the 3 selected trait-specific PRS as their unweighted sum, called “PRSsum”. We evaluated PRS associations in an independent dataset of 39,035 individuals from eight cohort studies, to select the final, multi-ethnic, HTN-PRS. We estimated its association with prevalent and incident hypertension 4-6 years later. We studied hypertension development within HTN-PRS strata in a longitudinal, six-visit, longitudinal dataset of 3,087 self-identified Black and White participants from the CARDIA study. Finally, we evaluated the HTN-PRS association with clinical outcomes in 40,201 individuals from the MGB Biobank.</p></sec><sec><title>Results</title><p>Compared to other race/ethnic backgrounds, African-Americans had higher average values of the HTN-PRS. The HTN-PRS was associated with prevalent hypertension (OR=2.10, 95% CI [1.99, 2.21], per one standard deviation (SD) of the PRS) across all participants, and in each race/ethnic background, with heterogeneity by background (p-value &lt; 1.0×10<sup>-4</sup>). The lowest estimated effect size was in African Americans (OR=1.53, 95% CI [1.38, 1.69]). The HTN-PRS was associated with new onset hypertension among individuals with normal (respectively, elevated) BP at baseline: OR=1.71, 95% CI [1.55, 1.91] (OR=1.48, 95% CI [1.27, 1.71]). Association was further observed in age-stratified analysis. In CARDIA, Black participants with high HTN-PRS percentiles developed hypertension earlier than White participants with high HTN-PRS percentiles. The HTN-PRS was significantly associated with increased risk of coronary artery disease (OR=1.12), ischemic stroke (OR=1.15), type 2 diabetes (OR=1.19), and chronic kidney disease (OR=1.12), in the MGB Biobank.</p></sec><sec><title>Conclusions</title><p>The multi-ethnic HTN-PRS is associated with both prevalent and incident hypertension at 4-6 years of follow up across adulthood and is associated with clinical outcomes.</p></sec>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.01.466818', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Behavioral context affects social signal representations within single primate prefrontal cortex neurons', 'authors': \"[{'name': 'Vladimir Jovanovic'}, {'name': 'Adam R. Fishbein'}, {'name': 'Lisa de la Mothe'}, {'name': 'Kuo-Fen Lee'}, {'name': 'Cory T. Miller'}]\", 'abstract': '<p>We tested whether social signal processing in more traditional, head-restrained contexts is representative of the putative natural analog – social communication – by comparing responses to vocalizations within individual neurons in marmoset prefrontal cortex (PFC) across a series of behavioral contexts ranging from traditional to naturalistic. Although vocalization responsive neurons were evident in all contexts, cross-context consistency was notably limited. A response to these social signals when subjects were head-restrained was not predictive of a comparable neural response to the identical vocalizations during natural communication, even within the same neuron. Neural activity at the population level followed a similar pattern, as PFC activity could be reliably decoded for the context in which vocalizations were heard. This suggests that neural representations of social signals in primate PFC are not static, but highly flexible and likely reflect how nuances of the dynamic behavioral contexts affect the perception of these signals and what they communicate.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.04.467344', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': True, 'title': 'Brilacidin, a COVID-19 Drug Candidate, demonstrates broad-spectrum antiviral activity against human coronaviruses OC43, 229E and NL63 through targeting both the virus and the host cell', 'authors': \"[{'name': 'Yanmei Hu'}, {'name': 'Hyunil Jo'}, {'name': 'William DeGrado'}, {'name': 'Jun Wang'}]\", 'abstract': '<p>Brilacidin, a mimetic of host defense peptides (HDPs), is currently in phase 2 clinical trial as an antibiotic drug candidate. A recent study reported that brilacidin has antiviral activity against SARS-CoV-2 by inactivating the virus. In this work, we discovered an additional mechanism of action of brilacidin by targeting heparan sulfate proteoglycans (HSPGs) on host cell surface. Brilacidin, but not acetyl brilacidin, inhibits the entry of SARS-CoV-2 pseudovirus into multiple cell lines, and heparin, a HSPG mimetic, abolishes the inhibitory activity of brilacidin on SARS-CoV-2 pseudovirus cell entry. In addition, we found that brilacidin has broad-spectrum antiviral activity against multiple human coronaviruses (HCoVs) including HCoV-229E, HCoV-OC43, and HCoV-NL63. Mechanistic studies revealed that brilacidin has a dual antiviral mechanism of action including virucidal activity and binding to coronavirus attachment factor HSPGs on host cell surface. Brilacidin partially loses its antiviral activity when heparin was included in the cell cultures, supporting the host-targeting mechanism. Drug combination therapy showed that brilacidin has a strong synergistic effect with remdesivir against HCoV-OC43 in cell culture. Taken together, this study provides appealing findings for the translational potential of brilacidin as a broad-spectrum antiviral for coronaviruses including SARS-CoV-2.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.05.467377', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Discrete-to-analog signal conversion in human pluripotent stem cells', 'authors': \"[{'name': 'Laura Prochazka'}, {'name': 'Peter W Zandstra'}, {'name': 'Yale S Michaels'}, {'name': 'Charles Lau'}, {'name': 'Mona Siu'}, {'name': 'Ting Yin'}, {'name': 'Diana Wu'}, {'name': 'Esther Jang'}, {'name': 'Ross D Jones'}, {'name': 'Mercedes Vazquez-Cantu'}, {'name': 'Penney M Gilbert'}, {'name': 'Himanshu Kaul'}, {'name': 'Yaakov Benenson'}]\", 'abstract': '<p>During development, state transitions are coordinated through changes in the identity of molecular regulators in a cell state- and dose specific manner. The ability to rationally engineer such functions in human pluripotent stem cells (hPSC) will enable numerous applications in regenerative medicine. Herein we report the generation of synthetic gene circuits that can detect a discrete cell state, and upon state detection, produce fine-tuned effector proteins in a programmable manner. Effectively, these gene circuits convert a discrete (digital-like) cell state into an analog signal by merging AND-like logic integration of endogenous miRNAs (classifiers) with a miRNA-mediated output fine-tuning technology (miSFITs). Using an automated miRNA identification and model-guided circuit optimization approach, we were able to produce robust cell state specific and graded output production in undifferentiated hPSC. We further finely controlled the levels of endogenous BMP4 secretion, which allowed us to document the effect of endogenous factor secretion in comparison to exogenous factor addition on early tissue development using the hPSC-derived gastruloid system.  Our work provides the first demonstration of a discrete-to-analog signal conversion circuit operating in living hPSC, and a platform for customized cell state-specific control of desired physiological factors, laying the foundation for programming cell compositions in hPSC-derived tissues and beyond.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.02.466881', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Demographic history shapes genomic variation in an intracellular parasite with a wide geographic distribution', 'authors': \"[{'name': 'Pascal Angst'}, {'name': 'Dieter Ebert'}, {'name': 'Peter D. Fields'}]\", 'abstract': '<p>Analyzing variation in a species’ genomic diversity can provide insights into its historical demography, biogeography and population structure, and thus, its ecology and evolution. Although such studies are rarely undertaken for parasites, they can be highly revealing because of the parasite’s coevolutionary relationships with hosts. Modes of reproduction and transmission are thought to be strong determinants of genomic diversity for parasites and vary widely among microsporidia (fungal-related intracellular parasites), which are known to have high intraspecific genetic diversity and interspecific variation in genome architecture. Here we explore genomic variation in the microsporidium <italic>Hamiltosporidium</italic>, a parasite of the freshwater crustacean <italic>Daphnia magna</italic>, looking especially at which factors contribute to nucleotide variation. Genomic samples from 18 Eurasian populations and a new, long-read based reference genome were used to determine the roles that reproduction mode, transmission mode and geography play in determining population structure and demographic history. We demonstrate two main <italic>H. tvaerminnensis</italic> lineages and a pattern of isolation-by-distance, but note an absence of congruence between these two parasite lineages and the two Eurasian host lineages. We suggest a comparatively recent parasite spread through Northern Eurasian host populations after a change from vertical to mixed-mode transmission and the loss of sexual reproduction. While gaining knowledge about the ecology and evolution of this focal parasite, we also identify common features that shape variation in genomic diversity for many parasites, e.g., distinct modes of reproduction and the intertwining of host–parasite demographies.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.467029', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Analysis of sex-specific lipid metabolism in P. falciparum gametocytes points to importance of sphingomyelin for gametocytogenesis', 'authors': \"[{'name': 'Melanie C. Ridgway'}, {'name': 'Daniela Cihalova'}, {'name': 'Simon H.J. Brown'}, {'name': 'Phuong Tran'}, {'name': 'Todd W. Mitchell'}, {'name': 'Alexander G. Maier'}]\", 'abstract': '<p>Male and female <italic>Plasmodium falciparum</italic> gametocytes are the parasite lifecycle stage responsible for transmission of malaria from the human host to mosquito vector. Not only are gametocytes able to survive in radically different host environments, but they are also precursors for male and female gametes that reproduce sexually soon after ingestion by the mosquito. Here we investigate the sex-specific lipid metabolism of gametocytes within their host red blood cell and poised for ingestion by the mosquito vector and subsequent sexual reproduction.</p><p>Comparison of the male and female lipidome identifies cholesteryl esters and dihydrosphingomyelin enrichment in female gametocytes. Chemical inhibition of each of these lipid types in mature gametocytes suggests dihydrosphingomyelin synthesis but not cholesteryl ester synthesis is important for sex-specific gametocyte viability. Genetic disruption of each of the two sphingomyelin synthase gene points towards sphingomyelin synthesis contributing to gametocytogenesis.</p><p>This study shows that gametocytes are not only distinct from asexual stages, but that the lipid composition is also vastly different between male and female gametocytes, reflecting the different cellular roles these stages play. Together our results highlight the sex-specific nature of gametocyte lipid metabolism that has the potential to be targeted to block malaria transmission.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.04.21265945', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Inhaled Corticosteroids for Outpatients with Covid-19: A Meta-Analysis', 'authors': \"[{'name': 'Todd C Lee'}, {'name': 'Emilie Bortolussi-Courval'}, {'name': 'Sara Belga'}, {'name': 'Nick Daneman'}, {'name': 'Adrienne K Chan'}, {'name': 'Ryan Hanula'}, {'name': 'Nicole Ezer'}, {'name': 'Emily G McDonald'}]\", 'abstract': '<p>The role of inhaled corticosteroids for outpatient COVID-19 is evolving. We meta-analyzed reported clinical trials and estimated probability of any effect and number needed to treat of 50 or 20 for symptom resolution by day 14 [100%, 99.8%, 93.1%] and hospitalization [88.6%, 72.7%, 26.3%] respectively.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.02.466935', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Multiple hybridization events punctuate the evolutionary trajectory of Malassezia furfur', 'authors': \"[{'name': 'Bart Theelen'}, {'name': 'Veronica Mixao'}, {'name': 'Giuseppe Ianiri'}, {'name': 'Joleen P.Z. Goh'}, {'name': 'Jan Dijksterhuis'}, {'name': 'Joe Heitman'}, {'name': 'Thomas L. Dawson'}, {'name': 'Toni Gabaldon'}, {'name': 'Teun Boekhout'}]\", 'abstract': '<p>Malassezia species are important fungal skin commensals and are part of the normal microbiota of humans and other animals. However, under certain circumstances these fungi can also display a pathogenic behaviour. For example, Malassezia furfur is a common commensal of human skin, and yet is often responsible for skin disorders but also systemic infections. Comparative genomics analysis of M. furfur revealed that some isolates have a hybrid origin, similar to several other recently described hybrid fungal pathogens. Because hybrid species exhibit genomic plasticity that can impact phenotypes, we sought to elucidate the genomic evolution and phenotypic characteristics of M. furfur hybrids in comparison to their parental lineages. To this end, we performed a comparative genomics analysis between hybrid strains and their presumptive parental lineages, and assessed phenotypic characteristics. Our results provide evidence that at least two distinct hybridization events occurred between the same parental lineages, and that the parental strains may have originally been hybrids themselves. Analysis of the mating-type locus reveals that M. furfur has a pseudobipolar mating system, and provides evidence that hybridization involved sexual liaisons resulting in cell-cell fusion leading to a diploid/aneuploid state. This study provides new insights into the evolutionary trajectory of M. furfur and contributes valuable genomic resources for future pathogenicity studies.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.467490', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Bacterial chemotaxis to saccharides is governed by a trade-off between sensing and uptake', 'authors': \"[{'name': 'Noele Norris'}, {'name': 'Uria Alcolombri'}, {'name': 'Johannes M Keegstra'}, {'name': 'Yutaka Yawata'}, {'name': 'Filippo Menolascina'}, {'name': 'Emilio Frazzoli'}, {'name': 'Naomi M Levine'}, {'name': 'Vicente I Fernandez'}, {'name': 'Roman Stocker'}]\", 'abstract': '<p>To swim up gradients of nutrients, E. coli senses nutrient concentrations within its periplasm. For small nutrient molecules, periplasmic concentrations typically match extracellular concentrations. However, this is not necessarily the case for saccharides, such as maltose, which is transported into the periplasm via a specific porin. Previous observations have shown that under various conditions E. coli limits maltoporin abundance so that, for extracellular micromolar concentrations of maltose, there are predicted to be only nanomolar concentrations of free maltose in the periplasm. Thus, in the micromolar regime, the total uptake of maltose from the external environment into the cytoplasm is limited not by the abundance of cytoplasmic transport proteins but by the abundance of maltoporins. Here we present results from experiments and modeling showing that this porin-limited transport enables E. coli to sense micromolar gradients of maltose despite having a high-affinity ABC transport system that is saturated at these micromolar levels. We used microfluidic assays to study chemotaxis of E. coli in various gradients of maltose and methyl-aspartate and leveraged our experimental observations to develop a mechanistic transport-and-sensing chemotaxis model. Incorporating this model into agent-based simulations, we discover a trade-off between uptake and sensing: although high-affinity transport enables higher uptake rates at low nutrient concentrations, it severely limits dynamic sensing range. We thus propose that E. coli may limit periplasmic uptake to increase its chemotactic sensitivity, enabling it to use maltose as an environmental cue.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.04.467250', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Genome-wide association study of cerebellar volume', 'authors': \"[{'name': 'Elleke Tissink'}, {'name': 'Siemon C de Lange'}, {'name': 'Jeanne E Savage'}, {'name': 'Douglas P Wightman'}, {'name': 'Kristen Kelly'}, {'name': 'Mats Nagel'}, {'name': 'Martijn P van den Heuvel'}, {'name': 'Danielle Posthuma'}]\", 'abstract': \"<p>Cerebellar volume is highly heritable and associated with neurodevelopmental and neurodegenerative disorders. Understanding the genetic architecture of cerebellar volume may improve our insight into these disorders. This study aims to investigate the convergence of cerebellar volume genetic associations in close detail. A genome-wide associations study for cerebellar volume was performed in a sample of 27,486 individuals from UK Biobank, resulting in 29 genome-wide significant loci and a SNP heritability of 39.82%. We pinpoint variants that have effects on amino acid sequence or cerebellar gene-expression. Additionally, 85 genome-wide significant genes were detected and tested for convergence onto biological pathways, cerebellar cell types or developmental stages. Local genetic correlations between cerebellar volume and neurodevelopmental and neurodegenerative disorders reveal shared loci with Parkinson's disease, Alzheimer's disease and schizophrenia. These results provide insights into the heritable mechanisms that contribute to developing a brain structure important for cognitive functioning and mental health.</p>\", 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.04.467226', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Disease-linked mutations trigger exposure of a protein quality control degron in the DHFR protein', 'authors': \"[{'name': 'Caroline Kampmeyer'}, {'name': 'Sven Larsen-Ledet'}, {'name': 'Morten Rose Wagnkilde'}, {'name': 'Mathias Michelsen'}, {'name': 'Henriette K. M. Iversen'}, {'name': 'Sofie V. Nielsen'}, {'name': 'Søren Lindemose'}, {'name': 'Alberto Caregnato'}, {'name': 'Tommer Ravid'}, {'name': 'Amelie Stein'}, {'name': 'Kaare Teilum'}, {'name': 'Kresten Lindorff-Larsen'}, {'name': 'Rasmus Hartmann-Petersen'}]\", 'abstract': '<p>Degrons are short stretches of amino acids or structural motifs that are embedded in proteins. They mediate recognition by E3 ubiquitin-protein ligases and thus confer protein degradation via the ubiquitin-proteasome system. Well-described degrons include the N-degrons, destruction boxes, and the PIP degrons, which mediate the controlled degradation of various proteins including signaling components and cell cycle regulators. In comparison, the so-called protein quality control (PQC) degrons that mediate the degradation of structurally destabilized or misfolded proteins are not well described. Here, we show that disease-linked DHFR missense variants are structurally destabilized and chaperone-dependent proteasome targets. We systematically mapped regions within DHFR to assess those that act as cytosolic PQC degrons in yeast cells. Two regions, DHFR-Deg13-36 (here Deg1) and DHFR-Deg61-84 (here Deg2), act as degrons and conferred degradation to unrelated fusion partners. The proteasomal turnover of Deg2 was dependent on the molecular chaperone Hsp70. Structural analyses by NMR and hydrogen/deuterium exchange revealed that Deg2 is buried in wild-type DHFR, but becomes transiently exposed in the disease-linked missense variants.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466973', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Fendrr synergizes with Wnt signalling to regulate fibrosis related genes during lung development', 'authors': \"[{'name': 'Tamer Ali'}, {'name': 'Sandra Rogala'}, {'name': 'Maria-Theodora Melissari'}, {'name': 'Sandra Währisch'}, {'name': 'Bernhard G Herrmann'}, {'name': 'Phillip Grote'}]\", 'abstract': '<p>Long non-coding RNAs are a very versatile class of molecules that can have important roles in regulating a cells function, including regulating other genes on the transcriptional level. One of these mechanisms is that RNA can directly interact with DNA thereby recruiting additional components such as proteins to these sites via a RNA:dsDNA triplex formation. We genetically deleted the triplex forming sequence (<italic>FendrrBox</italic>) from the lncRNA <italic>Fendrr</italic> in mice and find that this <italic>FendrrBox</italic> is partially required for <italic>Fendrr</italic> function <italic>in vivo</italic>. We find that the loss of the triplex forming site in developing lungs causes a dysregulation of gene programs, associated with lung fibrosis. A set of these genes contain a triplex site directly at their promoter and are expressed in fibroblasts. We find that <italic>Fendrr</italic> with the Wnt signaling pathway regulates these genes, implicating that <italic>Fendrr</italic> synergizes with Wnt signaling in lung fibrosis.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466992', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Epithelial-mesenchymal transition is the main driver of intrinsic metabolism in cancer cell lines', 'authors': \"[{'name': 'Sarah Cherkaoui'}, {'name': 'Stephan Durot'}, {'name': 'Jenna Bradley'}, {'name': 'Susan Critchlow'}, {'name': 'Sebastien Dubuis'}, {'name': 'Mauro Miguel Masiero'}, {'name': 'Rebekka Wegmann'}, {'name': 'Berend Snijder'}, {'name': 'Alaa Othman'}, {'name': 'Claus Bendtsen'}, {'name': 'Nicola Zamboni'}]\", 'abstract': '<p>A fundamental feature of cancer cells is genomic heterogeneity. It is a main driver of phenotypic differences, including the response to drugs, and therefore a key factor in therapy selection. Motivated by the increasing role attributed to metabolic reprogramming in tumor development, we wondered how genomic heterogeneity affects metabolic phenotype. To this end, we profiled the intracellular metabolome of 180 cancer cell lines grown in similar conditions to exclude environmental factors. For each cell line, we estimate activity for 49 pathways across the whole metabolic network. Upon clustering of activity data, we found a convergence into only two major metabolic types. These were further characterized by 13C-flux analysis, lipidomics, and analysis of sensitivity to perturbations. These experiments revealed differences in lipid, mitochondrial, and carbohydrate metabolism between the two major types. Finally, a thorough integration of our metabolic data with multiple omics data revealed a strong association with markers of epithelial-mesenchymal transition (EMT). Our analysis indicates that in absence of variations imposed by the microenvironment, the metabolism of cancer cell lines falls into only two major classes despite genetic heterogeneity.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466900', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Bovine pangenome reveals trait-associated structural variation from diverse assembly inputs', 'authors': \"[{'name': 'Alexander S. Leonard'}, {'name': 'Danang Crysnanto'}, {'name': 'Zih-Hua Fang'}, {'name': 'Michael P Heaton'}, {'name': 'Brian L. Vander Ley'}, {'name': 'Carolina Herrera'}, {'name': 'Heinrich Bollwein'}, {'name': 'Derek M. Bickhart'}, {'name': 'Kristen L. Kuhn'}, {'name': 'Timothy PL. Smith'}, {'name': 'Benjamin D. Rosen'}, {'name': 'Hubert Pausch'}]\", 'abstract': '<p>Advantages of pangenomes over linear reference assemblies for genome research have recently been established. However, potential effects of sequence platform and assembly approach, or of combining assemblies created by different approaches, on pangenome construction have not been investigated. Ten haplotype-resolved assemblies of three bovine trios representing increasing levels of heterozygosity were generated that each demonstrate a substantial improvement in contiguity and accuracy over the current <italic>Bos taurus</italic> reference genome, with more telomere and centromere content and an average 2.5x increase in NG50 and 11x decrease in base errors. Downsampling analysis demonstrated that diploid coverage as low as 20x for HiFi or 60x for ONT was sufficient to produce two haplotype-resolved assemblies meeting the standards set by the Vertebrate Genome Project. The assemblies were integrated into structural variant-based pangenomes that demonstrated significant consensus regardless of sequence platform, assembler algorithm, or coverage. Inspecting pangenome topologies identified approximately 900 structural variants overlapping with coding sequences; this approach revealed variants affecting <italic>QRICH2</italic>, <italic>PRDM9</italic>, <italic>HSPA1A</italic>, <italic>TAS2R46</italic> and <italic>GC</italic> that have potential to affect phenotype.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466889', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Maize AFP1 confers antifungal activity by inhibiting chitin deacetylases from a broad range of fungi', 'authors': \"[{'name': 'Lay-Sun Ma'}, {'name': 'Wei-Lun Tsai'}, {'name': 'Raviraj M. Kalunke'}, {'name': 'Meng-Yun Xu'}, {'name': 'Yu-Han Lin'}, {'name': 'Florensia Ariani Damei'}, {'name': 'Hui-Chun Lee'}]\", 'abstract': '<p>Adapted plant pathogenic fungi deacetylate chitin to chitosan to avoid host perception and disarm the chitin-triggered plant immunity. Whether plants have evolved factors to counteract this fungal evasion mechanism in the plant-pathogen interface remains obscure. Here, we decipher the underlying mechanism of maize cysteine-rich receptor-like secreted proteins (CRRSPs)-AFP1, which exhibits mannose-binding dependent antifungal activity. AFP1 initials the action by binding to specific sites on the surface of yeast-like cells, filaments, and germinated spores of the biotrophic fungi <italic>Ustilago maydis</italic>. This could result in fungal cell growth and cell budding inhibition, delaying spore germination and subsequently reducing fungal viability in a mannose-binding dependence manner. The antifungal activity of AFP1 is conferred by its interaction with the PMT-dependent mannosylated chitin deacetylases (CDAs) and interfering with the conversion of chitin. Our finding that AFP1 targets CDAs from pathogenic fungi and nonpathogenic budding yeast suggests a potential application of the CRRSP in combating fungal diseases and reducing threats posed by the fungal kingdom.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.03.21265819', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Effectiveness of COVID-19 Vaccines and Post-vaccination SARS-COV 2 Infection, Hospitalization, and Mortality: a Systematic Review and Meta-analysis of Observational Studies', 'authors': \"[{'name': 'Kazem Rahmani'}, {'name': 'Rasoul Shavaleh'}, {'name': 'Mahtab Forouhi'}, {'name': 'Hamideh Feiz Disfani'}, {'name': 'Mostafa Kamandi'}, {'name': 'Aram Asareh Zadegan Dezfuli'}, {'name': 'Rozita Khatamian Oskooi'}, {'name': 'Molood Foogerdi'}, {'name': 'Moslem Soltani'}]\", 'abstract': '<sec><title>Introduction &amp; Objective</title><p>Vaccination is one of the most important and effective ways of preventing infectious diseases, and has recently been used in the COVID-19 epidemic and pandemic. The present meta-analysis study aimed to evaluate the effectiveness of COVID-19 vaccines in reducing the incidence of infection, hospitalization, and mortality in observational studies.</p></sec><sec><title>Materials and Methods</title><p>A systematic search was performed independently in Scopus, PubMed, ProQuest, and Google Scholar electronic databases as well as Preprint servers using the keywords under study. The heterogeneity of the studies was assessed using <italic>I</italic><sup><italic>2</italic></sup><italic>and</italic> χ<sup><italic>2</italic></sup> statistics, according to which the I<sup>2</sup> of &gt; 50% and P -value &lt;0.1 was reported as heterogeneity of the studies. In addition, the Pooled Vaccine Effectiveness (PVE) obtained from the studies was calculated by converting (1-Pooled estimate × 100%) based on the type of outcome.</p></sec><sec><title>Results</title><p>A total of 54 records were included in this meta-analysis. The rate of PVE against SARS-COV 2 infection was about 71% (OR = 0.29, 95% CI: 0.23-0.36) in the first dose and 87% (OR = 0.13, 95% CI: 0.08-0.21) in the second, and the highest effectiveness in the first and second doses was that of BNT162b2 mRNA and combined studies. The PVE versus COVID-19-associated hospitalization was 73% (OR = 0.27, 95% CI: 0.18-0.41) in the first dose and 89% (OR = 0.11, 95% CI: 0.07-0.17) in the second. mRNA-1273 and combined studies in the first dose and ChAdOx1 and mRNA-1273 in the second dose had the highest effectiveness. Regarding the COVID-19-related mortality, PVE was about 28% (HR = 0.39, 95% CI: 0.23-0.45) in the first dose and 89% (HR = 0.11, 95% CI: 0.03-0.43) in the second.</p></sec><sec><title>Conclusion</title><p>The evidence obtained from this study showed that the effectiveness of BNT162b2 mRNA, mRNA-1273, and ChAdOx1 in the first and second doses, and even combined studies were associated with increased effectiveness against SARS-COV2 infection, hospitalization, and death from COVID-19. In addition, considering that the second dose was significantly more efficient than the first one, a booster dose injection could be effective in high-risk individuals. On the other hand, it was important to observe other prevention considerations in the first days after taking the first dose.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466926', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Learning from ingroup experiences changes intergroup impressions', 'authors': \"[{'name': 'Yuqing Zhou'}, {'name': 'Björn Lindström'}, {'name': 'Alexander Soutschek'}, {'name': 'Pyungwon Kang'}, {'name': 'Philippe N. Tobler'}, {'name': 'Grit Hein'}]\", 'abstract': '<p>Living in multicultural societies, humans form impressions towards individuals of their own social groups (ingroup members) and of different social groups (outgroup members). Some psychological theories predict that intergroup impressions are mainly shaped by experiences with outgroup individuals (“outgroup focused theories”), while other theories predict that ingroup experiences play a dominant role (“ingroup focused theories”). Here we test predictions from these two psychological theories by estimating how intergroup impressions are dynamically shaped when people learn from both ingroup and outgroup experiences. Participants expected to receive painful shocks but were saved from pain by different ingroup or outgroup members in 75% of all trials. We measured neural responses using functional magnetic resonance imaging (fMRI), and participants rated their social closeness as well as impressions towards the ingroup and the outgroup. Behavioral results showed an initial ingroup bias in impression ratings which was significantly reduced over the course of learning. Computational learning models revealed that these changes in intergroup impressions were predicted by the weight given to ingroup prediction errors. The weight of the ingroup prediction error and its effect on intergroup impression change was stronger the more individuals identified with their ingroup. On the neural level, the left inferior parietal lobule (IPL) encoded more negative prediction errors for the ingroup compared to the outgroup. Moreover, stronger weight for ingroup prediction errors was related to increased neural coupling between the left IPL and the anterior insula (AI). This coupling further predicted learning-related changes in intergroup impressions. Together, our work provides computational and neural evidence for “ingroup focused theories”, highlighting the importance of ingroup experiences in shaping social impressions in intergroup settings.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466760', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Natural variation in the maternal and zygotic mRNA complements of the early embryo in Drosophila melanogaster', 'authors': \"[{'name': 'Anna A. Feitzinger'}, {'name': 'Anthony Le'}, {'name': 'Ammon Thompson'}, {'name': 'Mehnoor Haseeb'}, {'name': 'Mohan K. Murugesan'}, {'name': 'Austin M. Tang'}, {'name': 'Susan E. Lott'}]\", 'abstract': '<p>Maternal gene products supplied to the egg during oogenesis drive the earliest events of development in all metazoans. After the initial stages of embryogenesis, maternal transcripts are degraded as zygotic transcription is activated, this is known as the maternal to zygotic transition (MZT). Altering the abundances of maternally deposited factors in the laboratory can have a dramatic effect on development, adult phenotypes and ultimately fitness. Zygotic transcription activation is a tightly regulated process, where the zygotic genome takes over control of development from the maternal genome, and is required for the viability of the organism. Recently, it has been shown that the expression of maternal and zygotic transcripts have evolved in the Drosophila genus over the course of 50 million years of evolution. However, the extent of natural variation of maternal and zygotic transcripts within a species has yet to be determined. We asked how the maternal and zygotic pools of mRNA vary within and between populations of <italic>D. melanogaster</italic>. In order to maximize sampling of genetic diversity, African lines of <italic>D. melanogaster</italic> originating from Zambia as well as DGRP lines originating from North America were chosen for transcriptomic analysis. Single embryo RNA-seq was performed at a stage before and a stage after zygotic genome activation in order to determine which transcripts are maternally deposited and which are zygotically expressed within and between these populations. Differential gene expression analysis has been used to quantify quantitative changes in RNA levels within populations as well as fixed expression differences between populations at both stages. Generally, we find that maternal transcripts are more highly conserved, and zygotic transcripts evolve at a higher rate. We find that there is more within population variation in transcript abundance than between populations and that expression variation is highest post-MZT between African lines. Determining the natural variation of gene expression surrounding the MZT in natural populations of <italic>D. melanogaster</italic> gives insight into the extent of how a tightly regulated process may vary within a species, the extent of developmental constraint at both stages and on both the maternal and zygotic genomes, and reveals expression changes allowing this species to adapt as it spread across the world.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.10.31.21265627', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Predictors of COVID testing among Australian youth: Insights from the Longitudinal Study of Australian Children', 'authors': \"[{'name': 'Md Irteja Islam'}, {'name': 'Verity Chadwick'}, {'name': 'Alexandra Martiniuk'}]\", 'abstract': '<sec><title>Background</title><p>Testing has played a crucial role in reducing the spread of COVID. Although COVID symptoms tend to be less severe in children and adolescents, a key concern is young people’s role in the transmission of the virus given their highly social lifestyles. In this study, we aimed to identify the predictors associated with COVID testing in Australian youth using data from the Longitudinal Study of Australian Children (LSAC).</p></sec><sec><title>Methods</title><p>We used the latest wave 9C1 of the LSAC, where data were collected from 16–21-year-old Australians via an online survey between October and December 2021. In total, 2291 Australian youths responded to the questions about COVID testing and COVID symptom severity. Data was stratified by living with/without parents, and bivariate and logistic regression analyses examined predictor variables (age, sex, country of birth, remoteness, education level, employment, relationship status, number of household members, living with parents, receiving the COVID financial supplement from government and index of relative socio-economic advantage and disadvantage) and their distributions over the outcome variable COVID testing.</p></sec><sec><title>Results</title><p>Youths aged 16-17 were more likely to live at home than youths aged 20-21 years. The strongest predictor of COVID testing was living in major cities (regardless of living with or without parents). Changed household composition was significantly associated with COVID testing among the youths living in the parental home. While among the respondents living without their parents, living with multiple household members and low or no cohesion among household members was associated with higher rates of COVID testing.</p></sec><sec><title>Conclusion</title><p>Our study revealed young people have been very good at getting tested for COVID. To further incentivise testing in this age group, we should consider providing this age group with continued financial and social support while awaiting the outcome of the test and during any isolation.</p></sec><sec><title>Strengths and limitations of this study</title><list list-type=\"bullet\"><list-item><p>Large national cohort of young people strengthened the findings of the study and allowing us to examine the factors associated with COVID testing for the first time in Australia.</p></list-item><list-item><p>A broad-based assessment of potential predictors of COVID testing, including sociodemographic and coronavirus specific factor.</p></list-item><list-item><p>Cross-sectional observational design limits causal inference.</p></list-item><list-item><p>Self-reported information about COVID testing can be subject to recall as well as social desirability bias.</p></list-item></list></sec>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.01.466856', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Population replacement gene drive characteristics for malaria elimination in a range of seasonal transmission settings: a modeling study', 'authors': \"[{'name': 'Shirley Leung'}, {'name': 'Nikolai Windblicher'}, {'name': 'Edward Wenger'}, {'name': 'Caitlin Bever'}, {'name': 'Prashanth Selvaraj'}]\", 'abstract': '<p>Genetically engineering mosquitoes is a promising new vector control strategy to reinvigorate the fight against malaria in Sub-Saharan Africa. Using an agent-based model of malaria transmission with vector genetics, we examine the impacts of releasing population-replacement gene drive mosquitoes on malaria transmission and quantify the gene drive system parameters required to achieve local elimination within a spatially-resolved, seasonal Sahelian setting. We evaluate the performance of two different gene drive systems: “classic” and “integral.” Various transmission regimes (low, moderate, and high - corresponding to annual entomological inoculation rates of 10, 30, and 80 infectious bites per person) and other simultaneous interventions, including deployment of insecticide-treated nets (ITNs) and passive healthcare seeking, are also simulated. Local elimination probabilities decreased with pre-existing population target site resistance frequency, increased with transmission-blocking effectiveness of the introduced antiparasitic gene and drive efficiency, and were context dependent with respect to fitness costs associated with the introduced gene. Of the four parameters, transmission-blocking effectiveness may be the most important to focus on for improvements to future gene drive strains because a single release of classic gene drive mosquitoes is likely to locally eliminate malaria in low to moderate transmission settings only when transmission-blocking effectiveness is very high (above <sup>~</sup>80-90%). However, simultaneously deploying ITNs and releasing integral rather than classic gene drive mosquitoes significantly boosts elimination probabilities, such that elimination remains highly likely in low to moderate transmission regimes down to transmission-blocking effectiveness values as low as <sup>~</sup>50% and in high transmission regimes with transmission-blocking effectiveness values above <sup>~</sup>80-90%. Thus, a single release of currently achievable population replacement gene drive mosquitoes, in combination with traditional forms of vector control, can likely locally eliminate malaria in low to moderate transmission regimes within the Sahel. In a high transmission regime, higher levels of transmission-blocking effectiveness than are currently available may be required.</p><sec><title>Author summary</title><p>Malaria remains a significant health burden in Sub-Saharan Africa. The mass deployment of insecticide-treated nets and antimalarial drugs have drastically reduced malaria incidence, but insecticide and drug resistance threaten to stall these efforts. The genetic engineering of mosquito populations is a promising new vector control strategy to reinvigorate the fight against malaria. Releases of engineered gene drive mosquitoes that can spread introduced antimalarial genes quickly throughout the mosquito population may be a particularly effective new method for reducing malaria transmission. Important questions arise, however, about how well these gene drive systems must work in order to deliver substantial reductions in transmission. Here we use a spatial model of individual humans and vectors to simulate the effects of releasing gene drive mosquitoes with antimalarial properties on malaria transmission in a Sahelian setting. We quantify the gene drive system parameters required to achieve local elimination and find that when deployed in combination with traditional forms of vector control, a single release of gene drive mosquitoes with realistically achievable characteristics is highly likely to locally eliminate malaria in low to moderate transmission regimes. In a high transmission regime, improved strains of gene drive mosquitoes may be required. In all settings, releasing gene drive mosquitoes with antimalarial properties helps create a window of opportunity during which malaria prevalence is suppressed and other interventions can be ramped up to achieve elimination, even when a single gene drive mosquito release by itself cannot.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.467135', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Prefrontal-amygdalar oscillations related to social interaction behavior in mice', 'authors': \"[{'name': 'Nahoko Kuga'}, {'name': 'Reimi Abe'}, {'name': 'Kotomi Takano'}, {'name': 'Yuji Ikegaya'}, {'name': 'Takuya Sasaki'}]\", 'abstract': '<p>The medial prefrontal cortex and amygdala are involved in the regulation of social behavior and associated with psychiatric diseases but their detailed neurophysiological mechanisms at a network level remain unclear. We recorded local field potentials (LFPs) from the dorsal medial PFC (dmPFC) and basolateral amygdala (BLA) while mice engaged on social behavior. We found that in wild-type mice, both the dmPFC and BLA increased 4–7 Hz oscillation power and decreased 30–60 Hz power when they needed to attend to another target mouse. In mouse models with reduced social interactions, dmPFC 4–7 Hz power further increased especially when they exhibited social avoidance behavior. In contrast, dmPFC and BLA decreased 4–7 Hz power when wild-type mice socially approached a target mouse. Frequency-specific optogenetic manipulations of replicating social approach-related LFP patterns restored social interaction behavior in socially deficient mice. These results demonstrate a neurophysiological substrate of the prefrontal cortex and amygdala related to social behavior and provide a unified pathophysiological understanding of neuronal population dynamics underlying social behavioral deficits.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466756', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Comparative genomics among three cyst nematode species reveals distinct evolutionary histories among effector families and an irregular distribution of effector-associated promoter motifs', 'authors': \"[{'name': 'Joris J.M. van Steenbrugge'}, {'name': 'Sven van den Elsen'}, {'name': 'Martijn Holterman'}, {'name': 'Jose L. Lozano-Torres'}, {'name': 'Vera Putker'}, {'name': 'Peter Thorpe'}, {'name': 'Aska Goverse'}, {'name': 'Mark G. Sterken'}, {'name': 'Geert Smant'}, {'name': 'Johannes Helder'}]\", 'abstract': '<p>Potato cyst nematodes (PCNs), an umbrella term used for two species, <italic>Globodera pallida</italic> and <italic>G. rostochiensis</italic>, belong worldwide to the most harmful pathogens of potato. Pathotype-specific host plant resistances are an essential handle for PCN control. However, the poor delineation of <italic>G. pallida</italic> pathotypes hampers the efficient use of available host plant resistances. Long-read sequencing technology allowed us to generate a new reference genome of <italic>G. pallida</italic> population D383 and, as compared to the current reference, the new genome assembly is 42 times less fragmented. For comparison of diversification patterns of six effector families between <italic>G. pallida</italic> and <italic>G. rostochiensis</italic>, an additional reference genome was generated for an outgroup, the beet cyst nematode <italic>Heterodera schachtii</italic> (IRS population). Large evolutionary contrasts in effector family topologies were observed. While VAPs diversified before the split between the three cyst nematode species, the families GLAND5 and GLAND13 only expanded in PCN after their separation from the genus <italic>Heterodera</italic>. Although DNA motifs in the promoter regions thought to be involved in the orchestration of effector expression (‘DOG boxes’) were present in all three cyst nematode species, their presence is not a necessity for dorsal gland-produced effectors. Notably, DOG box dosage was only loosely correlated with expression level of individual effector variants. Comparison of the <italic>G. pallida</italic> genome with those of two other cyst nematodes underlined the fundamental differences in evolutionary history between effector families. Re-sequencing of PCN populations with deviant virulence characteristics will allow for the linking of these characteristics with the composition of the effector repertoire as well as for the mapping of PCN diversification patterns resulting from extreme anthropogenic range expansion.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466751', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Penalized generalized estimating equations for relative risk regression with applications to brain lesion data', 'authors': \"[{'name': 'Petya Kindalova'}, {'name': 'Michele Veldsman'}, {'name': 'Thomas E. Nichols'}, {'name': 'Ioannis Kosmidis'}]\", 'abstract': '<p>Motivated by a brain lesion application, we introduce penalized generalized estimating equations for relative risk regression for modelling correlated binary data. Brain lesions can have varying incidence across the brain and result in both rare and high incidence outcomes. As a result, odds ratios estimated from generalized estimating equations with logistic regression structures are not necessarily directly interpretable as relative risks. On the other hand, use of log-link regression structures with the binomial variance function may lead to estimation instabilities when event probabilities are close to 1. To circumvent such issues, we use generalized estimating equations with log-link regression structures with identity variance function and unknown dispersion parameter. Even in this setting, parameter estimates can be infinite, which we address by penalizing the generalized estimating functions with the gradient of the Jeffreys prior.</p><p>Our findings from extensive simulation studies show significant improvement over the standard log-link generalized estimating equations by providing finite estimates and achieving convergence when boundary estimates occur. The real data application on UK Biobank brain lesion maps further reveals the instabilities of the standard log-link generalized estimating equations for a large-scale data set and demonstrates the clear interpretation of relative risk in clinical applications.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466971', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': True, 'title': 'Protein arginylation is regulated during SARS-CoV-2 infection', 'authors': \"[{'name': 'Janaina Macedo-da-Silva'}, {'name': 'Livia Rosa-Fernandes'}, {'name': 'Vinicius de Moraes Gomes'}, {'name': 'Veronica Feijoli Santiago'}, {'name': 'Catarina Maria Stanischesk Molnar'}, {'name': 'Bruno R Barboza'}, {'name': 'Edmarcia Elisa de Sousa'}, {'name': 'Edison Luiz Durigon'}, {'name': 'Claudio Romero Farias Marinho'}, {'name': 'Carsten Wrenger'}, {'name': 'Suely Kazue Nagahashi Marie'}, {'name': 'Giuseppe Palmisano'}]\", 'abstract': '<p>In 2019, the world witnessed the onset of an unprecedented pandemic. In September 2021, the infection by SARS-CoV-2 had already been responsible for the death of more than 4 million people worldwide. Recently, we and other groups discovered that SARS-CoV-2 infection induces ER-stress and activation of unfolded protein response (UPR) pathway. The degradation of misfolded/unfolded proteins is an essential element of proteostasis and occurs mainly in lysosomes or proteasomes. The N-terminal arginylation of proteins is characterized as an inducer of ubiquitination and proteasomal degradation by the N-end rule pathway. Here we present, for the first time, data on the role of arginylation during SARS-CoV-2 infection. We studied the modulation of protein arginylation in Vero CCL-81 and Calu-3 cells infected after 2h, 6h, 12h, 24h, and 48h. A reanalysis of in vivo and in vitro public omics data combined with immunoblotting was performed to measure the levels of ATE1 and arginylated proteins. This regulation is seen specifically during infections by coronaviruses. We demonstrate that during SARS-CoV-2 infection there is an increase in the expression of the ATE1 enzyme associated with regulated levels of specific arginylated proteins. On the other hand, infected macrophages showed no ATE1 regulation. An important finding revealed that modulation of the N-end rule pathway differs between different types of infected cells. We also confirmed the potential of tannic acid to reduce viral load, and furthermore, to modulate ATE1 levels during infection. In addition, the arginylation inhibitor merbromin (MER) is also capable of both reducing viral load and reducing ATE1 levels. Taken together, these data show the importance of arginylation during the progression of SARS-CoV-2 infection and open the door for future studies that may unravel the role of ATE1 and its inhibitors in pathogen infection.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466921', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'A life cycle alteration can correct molting defects in Caenorhabditis elegans', 'authors': \"[{'name': 'Shaonil Binti'}, {'name': 'Rosa V. Melinda'}, {'name': 'Braveen B. Joseph'}, {'name': 'Phil Edeen'}, {'name': 'Sam D. Miller'}, {'name': 'David S. Fay'}]\", 'abstract': '<p>Molting is a widespread feature in the development of many invertebrates, including nematodes and arthropods. In <italic>Caenorhabditis elegans</italic>, the highly conserved protein kinases NEKL-2/NEK8/9 and NEKL-3/NEK6/7 (NEKLs) promote molting through their involvement in the uptake and intracellular trafficking of epidermal cargos. We found that the relative requirements for NEKL-2 and NEKL-3 differed at different life-cycle stages and under different environmental conditions. Most notably, the transition from the second to the third larval stage (L2→L3 molt) required a higher level of NEKL function than during several other life stages or when animals had experienced starvation at the L1 stage. Specifically, larvae that entered the pre-dauer L2d stage could escape molting defects when transiting to the (non-dauer) L3 stage. Consistent with this, mutations that promote entry into L2d suppressed <italic>nekl</italic>-associated molting defects, whereas mutations that inhibit L2d entry reduced starvation-mediated suppression. We further showed that loss or reduction of NEKL functions led to defects in the transcription of cyclically expressed molting genes, many of which are under the control of systemic steroid hormone regulation. Moreover, the timing and severity of these transcriptional defects correlated closely with the strength of <italic>nekl</italic> alleles and with their stage of arrest. Interestingly, transit through L2d rescued <italic>nekl</italic>-associated expression defects in suppressed worms, providing an example of how life-cycle decisions can impact subsequent developmental events. Given that NEKLs are implicated in the uptake of sterols by the epidermis, we propose that loss of NEKLs leads to a physiological reduction in steroid-hormone signaling and consequent defects in the transcription of genes required for molting.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466376', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Riding into the future on a fly: toward a better understanding of phoresy and avian lice evolution (Phthiraptera) by screening bird carcasses for phoretic lice on hippoboscid flies (Diptera)', 'authors': \"[{'name': 'Leshon Lee'}, {'name': 'David J.X. Tan'}, {'name': 'Jozef Oboňa'}, {'name': 'Daniel R. Gustafsson'}, {'name': 'Ang Yuchen'}, {'name': 'Rudolf Meier'}]\", 'abstract': '<p>Many phoretic relationships between insects are understudied because of taxonomic impediments. We here illustrate for avian lice riding on hippoboscid flies how new natural history data on phoretic relationships can be acquired quickly with NGS barcoding. Most avian lice are host-specific, but some can arrive on new hosts by riding hippoboscid flies that feed on bird blood. Our summary of the literature yielded 254 published records which we here show to belong to two large and 13 small interaction networks for birds, flies, and lice. In order to generate new records, we then developed a new protocol based on screening bird carcasses sourced by citizen scientists. The inspection of 131 carcasses from Singapore lead to the first record of a <italic>Guimaraesiella</italic> louse species riding on <italic>Ornithoica momiyamai</italic> flies collected from a pitta carcass. Phoresy may explain why this louse species is now known from three phylogenetically disparate hosts (<italic>Pitta moluccensis</italic>, <italic>Ficedula zanthopygia</italic>; <italic>Pardaliparus elegans</italic>). A second new case of phoresy enhances a large interaction network dominated by <italic>Ornithophila metallica</italic>, a cosmopolitan and polyphagous hippoboscid fly species. Overall, we argue that many other two- and three-way phoretic relationships between arthropods (e.g., mites, pseudoscorpiones, beetles, flies) can be resolved using cost-effective large-scale NGS barcoding, which can be used to pre-sort specimens for taxonomic revision and thus partially overcome some taxonomic impediments.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466768', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Coherent theta oscillations in the cerebellum and supplementary motor area mediate visuomotor adaptation', 'authors': \"[{'name': 'Elinor Tzvi'}, {'name': 'Leila Gajiyeva'}, {'name': 'Laura Bindel'}, {'name': 'Gesa Hartwigsen'}, {'name': 'Joseph Classen'}]\", 'abstract': '<p>The cerebellum and its interaction with cortical areas play a key role in our ability to flexibly adapt a motor program in response to sensory input. Current knowledge about specific neural mechanisms underlying the process of visuomotor adaptation is however lacking. Using a novel placement of EEG electrodes to record electric activity from the cerebellum, we studied local cerebellar activity, as well as its coupling with neocortical activity to obtain direct neurophysiological markers of visuomotor adaptation in humans. We found increased theta (4-8Hz) power in “cerebellar” as well as cortical electrodes, when subjects first encountered a visual perturbation. Theta power decreased as subjects adapted to the perturbation, and rebounded when the perturbation was suddenly removed. This effect was observed in two distinct locations: a cerebellar cluster and a central cluster, which were localized in left cerebellar crus I (lCB) and right supplementary motor area (rSMA) using linear constrained minimum variance beamforming. Importantly, we found that better adaptation was associated with increased theta power in left cerebellar electrodes and a right sensorimotor cortex electrode. Finally, increased rSMA –&gt; lCB connectivity was significantly decreased with adaptation. These results demonstrate that: (1) cerebellar theta power is markedly modulated over the course of visuomotor adaptation and (2) theta oscillations could serve as a key mechanism for communication within a cortico-cerebellar loop.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.10.31.21265672', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Respiratory Medicine', 'covid': True, 'title': 'Chest X-ray Severity and its Association with Outcomes in Patients with COVID-19 Presenting to the Emergency Department', 'authors': \"[{'name': 'Daniel Kotok'}, {'name': 'Jose Rivera Robles'}, {'name': 'Christine Girard'}, {'name': 'Shruti Shettigar'}, {'name': 'Allen Lavina'}, {'name': 'Samantha Gillenwater'}, {'name': 'Andrew Kim'}, {'name': 'Anas Hadeh'}]\", 'abstract': '<sec><title>Background</title><p>Severity of radiographic abnormalities on chest X-ray (CXR) in patients with COVID-19 has been shown to be associated with worse outcomes, but studies are limited by different scoring systems, sample size, patient age and study duration. Data regarding the longitudinal evolution of radiographic abnormalities and its association with outcomes is scarce. We sought to evaluate these questions using a well-validated scoring system (the Radiographic Assessment of Lung Edema [RALE] score) using data over 6 months from a large, multi-hospital healthcare system.</p></sec><sec><title>Methods</title><p>We collected clinical and demographic data and quantified radiographic edema on CXRs obtained in the emergency department (ED) as well as on days 1-2 and 3-5 (in those admitted) in patients with a nasopharyngeal swab positive for SARS-CoV-2 PCR visiting the ED for COVID-19-related complaints between March and September 2020. We examined the association of baseline and longitudinal evolution of radiographic edema with severity of hypoxemia and clinical outcomes.</p></sec><sec><title>Results</title><p>870 patients were included (median age 53.6, 50.8% female). Inter-rate agreement for RALE scores was excellent (ICC = 0.84, 95% CI 0.82 - 0.87, p &lt; 0.0001). RALE scores correlated with hypoxemia as quantified by SpO2-FiO2 ratio (r = -0.42, p &lt; 0.001). Admitted patients had higher RALE scores than those discharged (6 [2, 11] vs 0 [0, 3], p &lt; 0.001). An increase of RALE score of 4 or more was associated with worse 30-day survival (p &lt; 0.01). Larger increases in the RALE score were associated with worse survival.</p></sec><sec><title>Conclusions</title><p>The RALE score is reproducible and easily implementable in adult patients presenting to the ED with COVID-19. Its association with physiologic parameters and outcomes at baseline and longitudinally makes it a readily available tool for prognostication and early ICU triage, particularly in patients with worsening radiographic edema.</p></sec>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.466970', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'PGC-1α in the myofibers regulates the balance between myogenic and adipogenic progenitors affecting muscle regeneration', 'authors': \"[{'name': 'Marc Beltrà'}, {'name': 'Fabrizio Pin'}, {'name': 'Domiziana Costamagna'}, {'name': 'Robin Duelen'}, {'name': 'Alessandra Renzini'}, {'name': 'Riccardo Ballarò'}, {'name': 'Lorena Garcia-Castillo'}, {'name': 'Ambra Iannuzzi'}, {'name': 'Viviana Moresi'}, {'name': 'Dario Coletti'}, {'name': 'Maurilio Sampaolesi'}, {'name': 'Fabio Penna'}, {'name': 'Paola Costelli'}]\", 'abstract': '<p>Skeletal muscle repair is accomplished by satellite cells (MuSC) in cooperation with interstitial stromal cells (ISCs). So far, the relationship between the function of these cells and the metabolic state of myofibers remains unclear. The present study reports alterations in the proportion of both MuSCs and adipogenesis regulators (Aregs) induced by overexpression of peroxisome proliferator-activated receptor gamma coactivator 1-alpha (PGC-1α) in the myofibers (MCK-PGC-1α mice). Although PGC-1α–driven increase of MuSCs does not accelerate muscle regeneration, myogenic progenitors isolated from MCK-PGC-1α mice and transplanted into intact and regenerating muscles are more prone to fuse with recipient myofibers than those derived from WT donors. Moreover, both young and aged MCK-PGC-1α animals show reduced perilipin-positive areas when challenged with an adipogenic stimulus, demonstrating low propensity to accumulate adipocytes within the muscle. These results provide new insights on the role played by PGC-1α in promoting myogenesis and hindering adipogenesis in the skeletal muscle.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.21265478', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Obstetrics and Gynecology', 'covid': True, 'title': 'Waning of the Humoral Response to SARS-CoV-2 in Pregnancy is Variant-Dependent', 'authors': \"[{'name': 'Romina Plitman Mayo'}, {'name': 'Tal Raz'}, {'name': 'Bar Ben David'}, {'name': 'Gila Meir'}, {'name': 'Haim Barr'}, {'name': 'Leonardo J. Solmesky'}, {'name': 'Rony Chen'}, {'name': 'Ana Idelson'}, {'name': 'Lucilla Zorzetti'}, {'name': 'Rinat Gabbay-Benziv'}, {'name': 'Yuval Jaffe Moshkovich'}, {'name': 'Tal Biron-Shental'}, {'name': 'Gil Shechter-Maor'}, {'name': 'Hen Yitzhak Sela'}, {'name': 'Itamar Glick'}, {'name': 'Hedi Benyamini Raischer'}, {'name': 'Raed Salim'}, {'name': 'Yariv Yogev'}, {'name': 'Ofer Beharier'}, {'name': 'Debra Goldman-Wohl'}, {'name': 'Ariel Many'}, {'name': 'Michal Kovo'}, {'name': 'Simcha Yagel'}, {'name': 'Michal Neeman'}]\", 'abstract': '<sec><title>Importance</title><p><bold>The</bold> SARS-CoV-2 alpha variant posed increased risk for COVID-19 complications in pregnant women. However, its impact on the maternal humoral response and placental IgG transport remains unclear.</p></sec><sec><title>Objective</title><p>To characterize the maternal humoral waning and neonate immunity acquired during the 3<sup>rd</sup> COVID-19 wave in Israel, dominated by the Alpha variant, as compared to earlier Wildtype infections and humoral response to vaccination across gestation.</p></sec><sec><title>Design</title><p>Maternal and fetal blood serum were collected at delivery since April 2020 from parturients. Sera IgG and IgM titers were measured using the Milliplex MAP SARS-CoV-2 Antigen Panel supplemented with additional HA-coupled microspheres.</p></sec><sec><title>Setting</title><p>A nationwide multicenter cohort study on SARS-CoV-2 infections and vaccination during pregnancy.</p></sec><sec><title>Participants</title><p>Expectant women presenting for delivery were recruited at 8 medical centers across Israel and assigned to 3 primary groups: SARS-CoV-2 positive (<italic>n</italic> = 157) and fully vaccinated during pregnancy (<italic>n</italic> = 125), and unvaccinated noninfected controls matched to the infected group by BMI, maternal age, comorbidities and gestational age (<italic>n =</italic> 212). Eligibility criteria included pregnant women without active COVID-19 disease, age ≥18 years and willingness to provide informed consent.</p></sec><sec><title>Main Outcome(s) and Measure(s)</title><p>Pregnant women’s humoral response is dependent on the SARS-CoV-2 strain.</p></sec><sec><title>Results</title><p>The humoral response to infection as detected at birth, showed a gradual and significant decline as the interval between infection/vaccination and delivery increased. Significantly faster decay of antibody titers was found for infections occurring during the 3<sup>rd</sup> wave compared to earlier infections/vaccination. Cord blood IgG antigens levels correlated with maternal IgG. However, cord IgG-HA variance significantly differed in SARS-CoV2 infections as compared to the other groups. No sexual dimorphism in IgG transfer was observed. Lastly, high fetal IgM response to SARS-CoV-2 was detected in 17 neonates, all showing elevated IgM to N suggesting exposure to SARS-Cov-2 antigens.</p></sec><sec><title>Conclusions and Relevance</title><p>Infections occurring during the 3<sup>rd</sup> wave induced a faster decline in humoral response when compared to Wildtype infections or mRNA BNT162b2 vaccination during pregnancy, consistent with a shift in disease etiology and severity induced by the Alpha variant. Vaccination policies in previously infected pregnant women should consider the timing of exposure along pregnancy as well as the risk of infection to specific variants of concern.</p></sec><sec><title>Key Points</title><sec><title>Question</title><p>What is the difference in the maternal-fetal humoral response between Alpha variant and SARS-CoV-2 Wildtype infections?</p></sec><sec><title>Findings</title><p>In this nationwide multicenter study including 494 pregnant women, the maternal humoral response to Alpha variant infection was weaker and shorter when compared to Wildtype infections. Placental transport compensated for the maternal waning of immunity. Fetal sex did not affect humoral response.</p></sec><sec><title>Meaning</title><p>Vaccination policies should be adjusted to account for the timing of infection and the SARS-CoV-2 variant.</p></sec></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.04.467289', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'ProDCoNN-server: a web server for protein sequence prediction and design from a three-dimensional structure', 'authors': \"[{'name': 'Yuan Zhang'}, {'name': 'Arunima Mandal'}, {'name': 'Kevin Cui'}, {'name': 'Xiuwen Liu'}, {'name': 'Jinfeng Zhang'}]\", 'abstract': '<p>We present ProDCoNN-server, a web server for protein sequence design and prediction from a given protein structure. The server is based on a previously developed deep learning model for protein design, ProDCoNN, which achieved state-of-the-art performance when tested on large numbers of test proteins and benchmark datasets. The prediction is very fast compared with other protein sequence prediction servers - it takes only a few minutes for a query protein on average. Two models could be selected for different purposes: BBO for full sequence prediction, extendable for multiple sequence generation, and BBS for single position prediction with the type of other residues known. ProDCoNN-server outputs the predicted sequence and the probability matrix for each amino acid at each predicted residue. The probability matrix can also be visualized as a sequence logos figure (BBO) or probability distribution plot (BBS). The server is available at: https://prodconn.stat.fsu.edu/.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.02.466878', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Reconstitution of ATP-dependent 3’-cleavage of mammalian pre-mRNA reveals a central role of RBBP6', 'authors': \"[{'name': 'Moritz Schmidt'}, {'name': 'Florian Kluge'}, {'name': 'Felix Sandmeir'}, {'name': 'Peter Schäfer'}, {'name': 'Christian Tüting'}, {'name': 'Christian Ihling'}, {'name': 'Uwe Kühn'}, {'name': 'Elena Conti'}, {'name': 'Elmar Wahle'}]\", 'abstract': '<p>The 3’ ends of almost all eukaryotic mRNAs are generated in an essential two-step processing reaction, endonucleolytic cleavage of an extended precursor followed by the addition of a poly(A) tail. By reconstituting the reaction from overproduced and purified proteins, we provide a minimal list of fourteen polypeptides essential and two stimulatory for RNA cleavage. In a reaction depending on the polyadenylation signal AAUAAA, the reconstituted system cleaves pre-mRNA at a single preferred site corresponding to the one used <italic>in vivo</italic>. Among the proteins, Cleavage Factor I stimulates cleavage but is not essential, consistent with its prominent role in alternative polyadenylation. RBBP6 is required, structural data showing it to contact and presumably activate the endonuclease CPSF73 through its DWNN domain. The C-terminal domain of RNA polymerase II is dispensable. ATP, but not its hydrolysis, supports RNA cleavage by binding to the hClp1 subunit of cleavage factor II with submicromolar affinity.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.10.29.466459', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Finger representations in primary somatosensory cortex are modulated during vibrotactile working memory', 'authors': \"[{'name': 'Finn Rabe'}, {'name': 'Sanne Kikkert'}, {'name': 'Nicole Wenderoth'}]\", 'abstract': '<p>It is well-established that vibrotactile stimulations elicit Blood-oxygen-level-dependent (BOLD) responses in somatotopically organized brain regions. Whether these somatotopic maps are modulated by working memory (WM) is still unknown. In our WM experiment, participants had to compare frequencies that were separated by a delay period. Vibrotactile stimuli were sequentially applied to either their right index or little finger. Using functional MRI, we investigated whether vibrotactile WM modulated neural activity in primary somatosensory (S1), an area that is known to contain individual finger representations. Our mass-univariate results revealed the well-described network of brain regions involved in WM. Interestingly, our mass-univariate results did not demonstrate S1 to be part of this network. However, when we parametrically modulated the time-binned regressors in our GLM we found that the delay activity in S1 and secondary somatosensory cortex (S2) was reflected in a U-shaped manner. Using multi-voxel pattern analysis (MVPA), an analysis technique that is more sensitive to subtle activity differences, we found finger-specific patterns of activation in the S1 hand area during the WM delay period. These results indicate that processes underlying WM modulate finger-specific representations during our discrimination task.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.02.466972', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Creatine utilization as a sole nitrogen source in Pseudomonas putida KT2440 is transcriptionally regulated by CahR', 'authors': \"[{'name': 'Lauren A. Hinkel'}, {'name': 'Graham G. Willsey'}, {'name': 'Sean M. Lenahan'}, {'name': 'Korin Eckstrom'}, {'name': 'Kristin C. Schutz'}, {'name': 'Matthew J. Wargo'}]\", 'abstract': '<p>Glutamine amidotransferase-1 domain-containing AraC-family transcriptional regulators (GATRs) are present in the genomes of many bacteria, including all <italic>Pseudomonas</italic> species. The involvement of several characterized GATRs in amine-containing compound metabolism has been determined, but the full scope of GATR ligands and regulatory networks are still unknown. Here, we characterize <italic>Pseudomonas putida’</italic>s detection of the animal-derived amine compound, creatine, a compound particularly enriched in muscle and ciliated cells by a creatine-specific GATR, PP_3665, here named CahR (<underline>C</underline>reatine <underline>a</underline>mido<underline>h</underline>ydrolase <underline>R</underline>egulator). <italic>cahR</italic> is necessary for transcription of the gene encoding creatinase (<italic>PP_3667/creA</italic>) in the presence of creatine and is critical for <italic>P. putida</italic>’s ability to utilize creatine as a sole source of nitrogen. The CahR/creatine regulon is small and electrophoretic mobility shift demonstrates strong and specific CahR binding only at the <italic>creA</italic> promoter, supporting the conclusion that much of the regulon is dependent on downstream metabolites. Phylogenetic analysis of creA orthologs associated with cahR orthologs highlights a strain distribution and organization supporting likely horizontal gene transfer, particularly evident within the genus <italic>Acinetobacter</italic>. This study identifies and characterizes the GATR that transcriptionally controls <italic>P. putida</italic> metabolism of creatine, broadening the scope of known GATR ligands and suggesting GATR diversification during evolution of metabolism for aliphatic nitrogen compounds.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.10.31.466697', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Microbiome restructuring: dominant coral bacterium Endozoicomonas species display differential adaptive capabilities to environmental changes', 'authors': \"[{'name': 'Kshitij Tandon'}, {'name': 'Yu-Jing Chiou'}, {'name': 'Sheng-Ping Yu'}, {'name': 'Hernyi Justin Hsieh'}, {'name': 'Chih-Ying Lu'}, {'name': 'Ming-Tsung Hsu'}, {'name': 'Pei-Wen Chiang'}, {'name': 'Hsing-Ju Chen'}, {'name': 'Naohisa Wada'}, {'name': 'Sen-Lin Tang'}]\", 'abstract': '<p>Bacteria in the coral microbiome play a crucial role in determining coral health and fitness, and the coral host often restructures its microbiome composition in response to external factors. An important but often neglected factor determining this microbiome restructuring is the capacity of microbiome members to adapt to a new environment. To address this issue, we examined how the microbiome structure of <italic>Acropora muricata</italic> corals changed over 9 months following a reciprocal transplant experiment. Using a combination of metabarcoding, genomics, and comparative genomics approaches, we found that coral colonies separated by a small distance harbored different dominant <italic>Endozoicomonas related phylotypes belonging to two different</italic> species, including a novel species, <italic>Candidatus</italic> Endozoicomonas penghunesis 4G, whose chromosome level (complete) genome was also sequenced in this study. Furthermore, the two dominant <italic>Endozoicomonas</italic> species showed varied adaptation capabilities when coral colonies were transplanted in a new environment. The differential adaptation capabilities of dominant members of the microbiome can a) provide distinct advantages to coral hosts when subjected to changing environmental conditions and b) have positive implications for future reefs.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.467149', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Genomes contain relics of a triplet code connecting the origins of primordial RNA synthesis to the origins of genetically coded protein synthesis', 'authors': \"[{'name': 'Geoffrey H. Siwo'}]\", 'abstract': '<p>Life on earth relies on three types of information polymers-DNA, RNA and proteins. In all organisms and viruses, these molecules are synthesized by the copying of pre-existing templates. A triplet-based code known as the genetic code guides the synthesis of proteins by complex enzymatic machines that decode genetic information in RNA sequences. The origin of the genetic code is one of the most fundamental questions in biology. In this study, computational analysis of about 5,000 species level metagenomes using techniques for the analysis of human language suggests that the genomes of extant organisms contain relics of a distinct triplet code that potentially predates the genetic code. This code defines the relationship between adjacent triplets in DNA/RNA sequences, whereby these triplets predominantly differ by a single base. Furthermore, adjacent triplets encode amino acids that are thought to have emerged around the same period in the earth’s early history. The results suggest that the order of triplets in primordial RNA sequences was associated with the availability of specific amino acids, perhaps due to a coupling of a triplet-based primordial RNA synthesis mechanism to a primitive mechanism of peptide bond formation. Together, this coupling could have given rise to early nucleic acid sequences and a system for encoding amino acid sequences in RNA, i.e. the genetic code. Thus, the central role of triplets in biology potentially extends to the primordial world, contributing to both the origins of genomes and the origins of genetically coded protein synthesis.</p><sec><title>Significance</title><p>One of the most intriguing discoveries in biology is that the order of amino acids in each protein is determined by the order of nucleotides (commonly represented by the letters A, U, G, C) in a biological molecule known as RNA. The genetic code serves as a dictionary that maps each of the 64 triplets ‘words’ in RNA to the 20 amino acids, thereby specifying how information encoded in RNA is decoded into sequences of amino acids (i.e., proteins). The deciphering of the genetic code was one of the greatest discoveries of the 20th century (1968 Nobel Prize in Medicine and Physiology) and is central to modern molecular biology. Yet, how it came to be that the order of triplets in RNA encodes the sequence of the protein synthesized remains one of the most important enigmas of biology. Paradoxically, in all life forms proteins cannot be synthesized without RNA and RNA itself cannot also be synthesized without proteins, presenting a chicken and egg dilemma. By analyzing thousands of microbial genomes using approaches drawn from the field of natural language processing, this study finds that the order of triplets across genomes contains relics of an ancient triplet code, distinct from but closely connected to the genetic code. Unlike the genetic code which specifies the relationship between information in RNA and the sequence of proteins, this ancient code describes the relationship between adjacent triplets in extant genome sequences, whereby such triplets are often different from each other by a single letter. Triplets that are closely related by this ancient code encode amino acids that are thought to have emerged around the same period in the earth’s early history. In other words, a fossil record of the chronological order of appearance of amino acids on early earth appears written in genome sequences. This potentially demonstrates that the process by which RNA sequences were synthesized in the primordial world relied on triplets and was coupled to amino acids available at the time. Hence, the connections between primordial RNA synthesis and a primitive mechanism for linking amino acids to form peptides could have enabled one type of molecule (RNA) to code for the other (protein), facilitating the emergence of the genetic code.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.467059', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'The Biosynthetic Costs of Amino Acids at the Base of the Food Chain Determine Their Use in Higher-order Consumer Genomes', 'authors': \"[{'name': 'Javier Gómez Ortega'}, {'name': 'Sonika Tyagi'}, {'name': 'Christen Mirth'}, {'name': 'Matthew Piper'}]\", 'abstract': \"<p>Dietary nutrient composition is essential for shaping important fitness traits and behaviours. Many organisms are protein limited and for Drosophila melanogaster, this limitation manifests at the level of the single most limiting essential Amino Acid (AA) in the diet. The identity of this AA and its effects on female fecundity is readily predictable by a procedure called exome matching in which the sum of AAs encoded by a consumer's exome is used to predict the relative proportion of AAs required in its diet. However, the exome matching calculation does not weight AA contributions to the overall profile by protein size or expression. Here we update the exome matching calculation to include these weightings. Surprisingly, although nearly half of the transcriptome is differentially expressed when comparing male and female flies, we found that creating transcriptome-weighted exome matched diets for each sex did not enhance their fecundity over that supported by exome matching alone. These data indicate that while organisms may require different amounts of dietary protein across conditions, the relative proportion of the constituent AAs remains constant. Interestingly, we also found remarkable conservation of exome matched AA profiles across taxa and that the composition of these profiles could be explained by the metabolic costs of microbial AA synthesis. Thus, it appears that bioenergetic constraints amongst autotrophs shape the relative proportion of AAs that are available across trophic levels and that that this constrains biomass composition.</p>\", 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.467391', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Organizing Principles of Astrocytic Nanoarchitecture in the Mouse Cerebral Cortex', 'authors': \"[{'name': 'Christopher K Salmon'}, {'name': 'Tabish A Syed'}, {'name': 'J Benjamin Kacerovsky'}, {'name': 'Nensi Alivodej'}, {'name': 'Alexandra L Schober'}, {'name': 'Michael T Pratte'}, {'name': 'Michael P Rosen'}, {'name': 'Miranda Green'}, {'name': 'Adario Chirgwin-Dasgupta'}, {'name': 'Hojatollah Vali'}, {'name': 'Craig A Mandato'}, {'name': 'Kaleem Siddiqi'}, {'name': 'Keith K Murai'}]\", 'abstract': '<p>Astrocytes have complex roles in central nervous system (CNS) health and disease. Underlying these roles is an elaborate architecture based on frequent, extremely fine, but seemingly haphazard branches, as well as prominent features including tripartite synaptic complexes and perivascular endfeet. While broad categories of structures in astrocytes are known, the fundamental building blocks that compose them and their organizing principles have yet to be adequately defined. This is largely due to the absence of high-resolution datasets that can reveal nanoscopic features of astrocytes (i.e. 10-20nm diameter in x, y, and z) and a lack of computational approaches that can effectively interrogate astrocyte shape, organization, and nanoarchitecture. Here, we produced and analyzed multiple, high-resolution datasets of layer 2/3 mouse somatosensory cortex using focused ion beam scanning electron microscopy (8nm intervals) and computer vision approaches to provide a principled, quantitative analysis of astrocytic nanoarchitecture. A decomposition of astrocytes into fundamental \"parts\" led to the discovery of unique structural components, recurring structural motifs, and assembly of parts into an organized hierarchy. New relationships were also discerned between astrocytic processes and other CNS microanatomy including mitochondria, tripartite synapses, and cerebrovasculature. By deploying computational resources to quantitatively understand the organizing principles and nanoarchitecture of astrocytes, this study reveals the specialized anatomical adaptations of these complex cells within the CNS.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.01.466695', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Antiviral activity of Pacific oyster (Crassostrea gigas) hemolymph against a human coronavirus', 'authors': \"[{'name': 'Rebecca L. Pedler'}, {'name': 'James O. Harris'}, {'name': 'Peter G. Speck'}]\", 'abstract': '<p>Coronaviruses can cause severe respiratory infections in humans. This study aimed to assess the antiviral activity of Pacific oyster (<italic>Crassostrea gigas</italic>) hemolymph against a human coronavirus, HCoV-229E. An eight-fold reduction in infectivity of HCoV-229E on Huh-7 cells was observed in the presence of 10% <italic>C. gigas</italic> hemolymph. Antiviral activity of <italic>C. gigas</italic> hemolymph positively correlated with its concentration and appears to be active during an intracellular stage of HCoV-229E infection.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.05.467536', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Construction of lux-based promoter-reporter platforms in Mycobacterium bovis BCG for screening of drug repurposing small-molecule compounds as new anti-tuberculosis drugs', 'authors': \"[{'name': 'Li Zhu'}, {'name': 'Annie Wing-tung Lee'}, {'name': 'Kelvin Ka-Lok WU'}, {'name': 'Peng GAO'}, {'name': 'Kingsley King-Gee Tam'}, {'name': 'Rahim Rajwani'}, {'name': 'Galata Chala CHABURTE'}, {'name': 'Timothy Ting-Leung NG'}, {'name': 'Chloe Toi Mei Chan'}, {'name': 'Hiu-Yin Lao'}, {'name': 'Wing-Cheong Yam'}, {'name': 'Richard Yi-Tsun Kao'}, {'name': 'Gilman Kit Hang Siu'}]\", 'abstract': '<p>The emergence of multidrug-resistant strains and hyper-virulent strains of Mycobacterium tuberculosis are big therapeutic challenges for tuberculosis (TB) control. Repurposing bioactive small-molecule compounds has recently become a new therapeutic approach against TB. This study aimed to construct a rapid screening system to identify novel anti-TB agents from a library of small-molecule compounds. In this study, a total of 320 small-molecule compounds were used to screen for their ability to suppress the expression of a key virulence gene, phoP, of M. tuberculosis complex using luminescence (lux)-based promoter-reporter platforms. The minimum inhibitory and bactericidal concentrations on drug-resistant M. tuberculosis and cytotoxicity to human macrophage were determined. RNA-sequencing (RNA-seq) was conducted to determine the drug mechanisms of the selected compounds as novel antibiotics or anti-virulent agents against the M. tuberculosis complex. Six compounds displayed bactericidal activity against M. bovis BCG, in which Ebselen demonstrated the lowest cytotoxicity to macrophage and was considered as a potential antibiotic for TB. Another ten compounds did not inhibit the in vitro growth of the M. tuberculosis complex but down-regulated the expression of phoP specifically. Of them, ST-193 and ST-193 (hydrochloride) showed low cytotoxicity and could dysregulate the entire phoP-associated gene network, and thus identified as potential anti-virulence agents for M. tuberculosis. This study provides a rapid screening platform coupled with a systematic validation and eventually suggested one potential antibiotic and two anti-virulence agents for M. tuberculosis infections.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.02.467032', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'KinasePhos 3.0: Redesign and Expansion of the Prediction on Kinase-specific Phosphorylation Sites', 'authors': \"[{'name': 'Renfei Ma'}, {'name': 'Shangfu Li'}, {'name': 'Wenshuo Li'}, {'name': 'Lantian Yao'}, {'name': 'Hsien-Da Huang'}, {'name': 'Tzong-Yi Lee'}]\", 'abstract': '<p>The purpose of this work is to enhance KinasePhos, a machine-learning-based kinase-specific phosphorylation site prediction tool. Experimentally verified kinase-specific phosphorylation data were collected from PhosphoSitePlus, UniProt, GPS 5.0, and Phospho.ELM. In total, 41,421 experimentally verified kinase-specific phosphorylation sites were identified. A total of 1380 unique kinases were identified, including 753 with existing classification information from KinBase and the remaining 627 annotated by building a phylogenetic tree. Based on this kinase classification, a total of 771 predictive models were built at the individual, family, and group levels, using at least 15 experimentally verified substrate sites in positive training datasets. The improved models were observed to be more effective than other prediction tools. For example, the prediction of sites phosphorylated by the Akt, CKT, and PKA families had accuracies of 94.5%, 92.5%, and 90.0%, respectively. The average prediction accuracy for all 771 models was 87.2%. For enhancing interpretability, the Shapley additive explanations (SHAP) method was employed to assess feature importance. The web interface of KinasePhos 3.0 has been redesigned with the goal of providing comprehensive annotations of kinase-specific phosphorylation sites on multiple proteins. Additionally, considering the large scale of phosphoproteomic data, a downloadable prediction tool is available at https://awi.cuhk.edu.cn/KinasePhos/index.html or https://github.com/tom-209/KinasePhos-3.0-executable-file.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.467040', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Discrimination of species within the Enterobacter cloacae complex using MALDI-TOF Mass Spectrometry and Fourier-Transform Infrared Spectroscopy coupled with Machine Learning tools', 'authors': \"[{'name': 'Ana Candela'}, {'name': 'Miriam Mateos'}, {'name': 'Alicia Gómez-Asenjo'}, {'name': 'Manuel J. Arroyo'}, {'name': 'Marta Hernandez-García'}, {'name': 'Rosa del Campo'}, {'name': 'Emilia Cercenado'}, {'name': 'Gema Méndez'}, {'name': 'Luis Mancera'}, {'name': 'Juan de Dios Caballero'}, {'name': 'Laura Martínez-García'}, {'name': 'Desirée Gijón'}, {'name': 'María Isabel Morosini'}, {'name': 'Patricia Ruiz-Garbajosa'}, {'name': 'Rafael Cantón'}, {'name': 'Patricia Muñoz'}, {'name': 'David Rodríguez-Temporal'}, {'name': 'Belén Rodríguez-Sánchez'}]\", 'abstract': '<p><italic>Enterobacter cloacae</italic> complex (ECC) encompasses heterogenic genetic clusters of species that have been associated with nosocomial outbreaks. These species may host different acquired antimicrobial susceptibility patterns and their identification is challenging. DNA-based techniques are laborious and require specific equipment. MALDI-TOF MS has showed low accuracy for the discrimination of ECC species. The aim of this study is to develop machine learning predictive models using MALDI-TOF MS and new diagnostic technologies like Fourier-Transform Infrared Spectroscopy (FTIR-S) for species-level identification of these species.</p><p>A total of 163 ECC clinical isolates were included in the study: 47 for the predictive model development and internal validation and 126 for external validation. All spectra obtained by MALDI-TOF MS and FTIR-S were processed using Clover MS Data Analysis software. Two models were created: Model A for differentiate six ECC species and Model B for <italic>E. hormaechei</italic> subspecies.</p><p>For MALDI-TOF MS spectra, Model A identified correctly 96.0% of isolates using Random Forest (RF) algorithm, and Model B identified 94.1% using Support Vector Machine (SVM). Regarding FTIR-S, Model A identified 73.0% of isolates by RF and Model B 72.5%. Two new predictive models were created for FTIR-S: Model C for discrimination of <italic>E. hormaechei</italic> from non-<italic>E. hormaechei</italic> (87.3% identification, RF) and Model D for differentiation among non-<italic>E. hormaechei</italic> species (62.7% identification, RF).</p><p>MALDI-TOF MS combined with machine learning tools could be a rapid and accurate method for species-level identification within the ECC. FTIR-S differentiated general groups of the ECC although discrimination of non-<italic>E. hormaechei</italic> species was poor.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466849', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Sex significantly impacts the function of major depression-linked variants in vivo', 'authors': \"[{'name': 'Bernard Mulvey'}, {'name': 'Din Selmanovic'}, {'name': 'Joseph D. Dougherty'}]\", 'abstract': '<p>Genome-wide association studies have discovered blocks of common variants—likely transcriptional-regulatory—associated with major depressive disorder (MDD), though the functional subset and their biological impacts remain unknown. Likewise, why depression occurs in females more frequently than males is unclear. We therefore tested the hypothesis that risk-associated functional variants interact with sex and produce greater impact in female brains. We developed methods to directly measure regulatory variant activity and sex interactions using massively parallel reporter assays (MPRAs) in the mouse brain <italic>in vivo</italic>, in a cell type-specific manner. We measured activity of &gt;1,000 variants from &gt;30 MDD loci, identifying extensive sex-by-allele effects in mature hippocampal neurons and suggesting sex-differentiated impacts of genetic risk may underlie sex bias in disease. Unbiased informatics approaches indicated that functional MDD variants recurrently disrupt sex hormone receptor binding sequences. We confirmed this with MPRAs in neonatal brains, comparing brains undergoing the masculinizing hormone surge to hormonally-quiescent juveniles. Our study provides novel insights into the influence of age, biological sex, and cell type on regulatory-variant function, and provides a framework for <italic>in vivo</italic> parallel assays to functionally define interactions between organismal variables like sex and regulatory variation.</p><sec><title>One-Sentence Summary</title><p>Massively parallel assays <italic>in vivo</italic> identified extensive functional and sex-interacting common variants in depression risk loci.</p></sec>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.21265839', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Effects of non-invasive brain stimulation in dystonia: a systematic review and meta-analysis', 'authors': \"[{'name': 'Jordan Morrison-Ham'}, {'name': 'Gillian M. Clark'}, {'name': 'Elizabeth G. Ellis'}, {'name': 'Andris Cerins'}, {'name': 'Juho Joutsa'}, {'name': 'Peter G. Enticott'}, {'name': 'Daniel T. Corp'}]\", 'abstract': '<sec><title>Background</title><p>Deep brain stimulation is a highly effective treatment of dystonia, but is invasive and associated with risks, such as intraoperative bleeding and infections. Previous research has used non-invasive brain stimulation (NIBS) in an attempt to alleviate symptoms of dystonia. The results of these studies, however, have been variable, leaving efficacy unclear. This study aimed to evaluate the effects of NIBS on symptoms of dystonia and determine whether methodological characteristics are associated with variability in effect size.</p></sec><sec><title>Methods</title><p>Embase and MEDLINE Complete databases were searched for articles using any type of NIBS as an intervention in dystonia patients, with changes in dystonia symptoms the primary outcome of interest.</p></sec><sec><title>Results</title><p>Meta-analysis of 26 studies demonstrated a small effect size for NIBS in reducing symptoms of dystonia (random-effects Hedges’ g = 0.21, p = .002). Differences in the type of NIBS, type of dystonia, and brain region stimulated had a significant effect on dystonia symptoms. Meta-regression revealed that 10 sessions of active stimulation, and the application of concurrent motor training programs resulted in significantly larger mean effect sizes.</p></sec><sec><title>Conclusion</title><p>NIBS has yielded small improvements to dystonic symptoms, but effect sizes depended on methodological characteristics, with more sessions of stimulation producing a larger response. Future research should further investigate the application of NIBS parallel to motor training, in addition to providing a greater quantity of sessions, to help define optimal parameters for NIBS protocols in dystonia.</p></sec><sec><title>Registration</title><p>PROSPERO 2020, CRD42020175944.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466905', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Bacterial colonisation dynamics of household plastics in a coastal environment', 'authors': \"[{'name': 'Luke Lear'}, {'name': 'Daniel Padfield'}, {'name': 'Tirion Dowsett'}, {'name': 'Maia Jones'}, {'name': 'Suzanne Kay'}, {'name': 'Alex Hayward'}, {'name': 'Michiel Vos'}]\", 'abstract': '<p>Accumulation of plastics in the marine environment has widespread detrimental consequences for ecosystems and wildlife. Marine plastics are rapidly colonised by a wide diversity of bacteria, including human pathogens, posing potential risks to human health. Here, we investigate the effect of polymer type, residence time and estuarine location on bacterial colonisation of common household plastics, including pathogenic bacteria. To do so, we submerged five main household plastic types: low-density PE (LDPE), high-density PE (HDPE), polypropylene (PP), polyvinyl chloride (PVC) and polyethylene terephthalate (PET) at an estuarine site in Cornwall (U.K.) and tracked bacterial colonisation dynamics. Using both culture-dependent and culture-independent approaches, we found that bacteria rapidly colonised plastics irrespective of polymer type. While biofilm community composition changed with colonisation time, no difference was observed between polymer types. Likewise, the presence of pathogenic bacteria, quantified using the insect model Galleria mellonella, increased over a five-week period, with no consistent differences observed between polymer types. Pathogens isolated from plastic biofilms using Galleria enrichment included Serratia and Enterococcus species and harboured a wide range of antimicrobial resistance genes. Our findings show that plastics in coastal waters are rapidly colonised by a wide diversity of bacteria, including known human pathogens, independent of polymer type.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466922', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Pseudomonas zeiradicis from corn and Pseudomonas soyae from soybean, two new Pseudomonas species and endophytes from agricultural crops', 'authors': \"[{'name': 'Jacqueline Lemaire'}, {'name': 'Sarah Seaton'}, {'name': 'Patrik Inderbitzin'}, {'name': 'Martha E. Trujillo'}]\", 'abstract': '<p>Two novel <italic>Pseudomonas</italic> species associated with healthy plants and other habitats are described from the United States. They are <italic>Pseudomonas zeiradicis</italic> strain PI116 from corn in Missouri, compost from Massachusetts, urban soil from Iowa and water of Lake Erie; and <italic>Pseudomonas soyae</italic> strain JL117 from soybean in Indiana and Wisconsin, and soil in Wyoming. No plant pathogenic strains are known for any of the novel species based on genome comparisons to assemblies in GenBank.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.05.467506', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Structure and Function of the Retina of Low-density Lipoprotein Receptor-related Protein 5 (Lrp5)-deficient Rats', 'authors': \"[{'name': 'John L. Ubels'}, {'name': 'Cheng-Mao Lin'}, {'name': 'David A. Antonetti'}, {'name': 'Monica Diaz-Coranguez'}, {'name': 'Cassandra R. Diegel'}, {'name': 'Bart O. Williams'}]\", 'abstract': '<p>Loss-of-function mutations in the Wnt co-receptor, low-density lipoprotein receptor-related protein 5 (LRP5), result in familial exudative vitreoretinopathy (FEVR), osteoporosis-pseudoglioma syndrome (OPPG), and Norrie disease. CRISPR/Cas9 gene editing was used to produce rat strains deficient in Lrp5. The purpose of this study was to validate this rat model for studies of hypovascular, exudative retinopathies. The retinal vasculature of wildtype and Lrp5 knockout rats was stained with Giffonia simplifolia isolectin B4 and imaged by fluorescence microscopy. Effects on retinal structure were investigated by histology. The integrity of the blood-retina barrier was analyzed by staining for claudin-5 and measurement of permeability to Evans blue dye. Retinas were imaged by fundus photography and SD-OCT, and electroretinograms were recorded. Lrp5 gene deletion led to sparse superficial retinal capillaries and loss of the deep and intermediate plexuses. Autofluorescent exudates were observed, correlated with absence of claudin-5 expression in superficial vessels and increased Evans blue permeability. OCT images show pathology similar to OCT of humans with FEVR, and retinal thickness is reduced by 50% compared to wild-type rats. Histology and OCT reveal that photoreceptor and outer plexiform layers are absent. The retina failed to demonstrate an ERG response. CRISPR/Cas9 gene-editing produced a predictable rat Lrp5 knockout model with extensive defects in the retinal vascular and neural structure and function. This rat model should be useful for studies of exudative retinal vascular diseases involving the Wnt and norrin pathways.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.05.467472', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Discovery of a carbonic anhydrase-Rubisco supercomplex within the alpha-carboxysome', 'authors': \"[{'name': 'Cecilia Blikstad'}, {'name': 'Eli J Dugan'}, {'name': 'Thomas G Laughlin'}, {'name': 'Mira D Liu'}, {'name': 'Sophie R Shoemaker'}, {'name': 'Jonathan P Remis'}, {'name': 'David F Savage'}]\", 'abstract': '<p>Carboxysomes are proteinaceous organelles that encapsulate key enzymes of CO2 fixation, Rubisco and carbonic anhydrase, and are the centerpiece of the bacterial CO2 concentrating mechanism (CCM). In the CCM, actively accumulated cytosolic bicarbonate diffuses into the carboxysome and is converted to CO2 by carbonic anhydrase, producing a high CO2 concentration near Rubisco and ensuring efficient carboxylation. Self-assembly of the α-carboxysome is orchestrated by the intrinsically disordered scaffolding protein, CsoS2, which interacts with both Rubisco and carboxysomal shell proteins, but it is unknown how CsoSCA, the carbonic anhydrase, is incorporated into the α-carboxysome. Here, we present the structural basis of carbonic anhydrase encapsulation into α-carboxysomes from Halothiobacillus neapolitanus. We find that CsoSCA interacts directly with Rubisco via an intrinsically disordered N-terminal domain. A 1.98 Å single-particle cryo-electron microscopy structure of Rubisco in complex with this peptide reveals that CsoSCA binding is predominantly mediated by a network of hydrogen bonds. CsoSCAs binding site overlaps with that of CsoS2 but the two proteins utilize substantially different motifs and modes of binding, revealing a plasticity of the Rubisco binding site. Our results advance the understanding of biogenesis of carboxysomes and highlights the importance of Rubisco, not only as an enzyme, but also as a hub protein central for assembling supercomplexes.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.05.467356', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'A large-scale genome-based survey of acidophilic Bacteria suggests that genome streamlining is an adaption for life at low pH', 'authors': \"[{'name': 'Diego Nahuel Cortez'}, {'name': 'Gonzalo Andres Neira'}, {'name': 'Carolina Mabel Gonzalez'}, {'name': 'Eva Marilyn Vergara'}, {'name': 'David S Holmes'}]\", 'abstract': \"<p>Genome streamlining theory suggests that reduction of microbial genome size optimizes energy utilization in stressful environments. Although this hypothesis has been explored in several cases of low nutrient (oligotrophic) and high temperature environments, little work has been carried out on microorganisms from low pH environments and what has been reported is inconclusive. In this study, we performed a large-scale comparative genomics investigation of more than 260 bacterial high-quality genome sequences of acidophiles, together with genomes of their closest phylogenetic relatives that live at circum-neutral pH. A statistically supported correlation is reported between reduction of genome size and decreasing pH that we demonstrate is due to gene loss and reduced gene sizes. This trend is independent from other genome size constraints such as temperature and G+C content. Genome streamlining in the evolution of acidophilic Bacteria is thus supported by our results. Analyses of predicted COG categories and subcellular location predictions indicate that acidophiles have a lower representation of genes encoding extra-cellular proteins, signal transduction mechanisms and proteins with unknown function, but are enriched in inner membrane proteins, chaperones, basic metabolism, and core cellular functions. Contrary to other reports for genome streamlining, there was no significant change in paralog frequencies across pH. However, a detailed analysis of COG categories revealed a higher proportion of genes in acidophiles in the following categories: 'Replication and repair', 'Amino acid transport' and 'Intracellular trafficking'. This study brings increasing clarity regarding genomic adaptations of acidophiles to life at low pH while putting elements such as the reduction of average gene size under the spotlight of streamlining theory.</p>\", 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.04.467353', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'SHIP164 is a Chorein Motif Containing Lipid Transport Protein that Controls Membrane Dynamics and Traffic at the Endosome-Golgi Interface.', 'authors': \"[{'name': 'Michael G Hanna'}, {'name': 'Patreece Suen'}, {'name': 'Yumei Wu'}, {'name': 'Karin M Reinisch'}, {'name': 'Pietro De Camilli'}]\", 'abstract': '<p>Cellular membranes differ in protein and lipid composition as well as in the protein-lipid ratio. Thus, progression of membranous organelles along traffic routes requires mechanisms to control bilayer lipid chemistry and their abundance relative to proteins. The recent structural and functional characterization of VPS13-family proteins has suggested a mechanism through which lipids can be transferred in bulk from one membrane to another at membrane contact sites, and thus independently of vesicular traffic. Here we show that SHIP164 (UHRF1BP1L) shares structural and lipid transfer properties with these proteins and is localized on a subpopulation of vesicle clusters in the early endocytic pathway whose membrane cargo includes the cation-independent mannose-6-phosphate receptor (MPR) and ATG9.  Loss of SHIP164 disrupts retrograde traffic of these organelles to the Golgi complex.  Our findings raise the possibility that bulk transfer of lipids to endocytic membranes may play a role in their traffic.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.467485', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Probabilistic Genotyping of Single Cell Replicates from Complex DNA Mixtures Recovers Higher Contributor LRs than Standard Analysis', 'authors': \"[{'name': 'Kaitlin Huffman'}, {'name': 'Erin Hanson'}, {'name': 'Jack Ballantyne'}]\", 'abstract': '<p>DNA mixtures are a common source of crime scene evidence and are often one of the more difficult sources of biological evidence to interpret. With the implementation of probabilistic genotyping (PG), mixture analysis has been revolutionized allowing previously unresolvable mixed profiles to be analyzed and probative genotype information from contributors to be recovered. However, due to allele overlap, artifacts, or low-level minor contributors, genotype information loss inevitably occurs. In order to reduce the potential loss of significant DNA information from donors in complex mixtures, an alternative approach is to physically separate individual cells from mixtures prior to performing DNA typing thus obtaining single source profiles from contributors. In the present work, a simplified micro-manipulation technique combined with enhanced single-cell DNA typing was used to collect one or few cells, referred to as direct single-cell subsampling (DSCS). Using this approach, single and 2-cell subsamples were collected from 2-6 person mixtures. Single-cell subsamples resulted in single source DNA profiles while the 2-cell subsamples returned either single source DNA profiles or new mini-mixtures that are less complex than the original mixture due to the presence of fewer contributors.  PG (STRmixTM) was implemented, after appropriate validation, to analyze the original bulk mixtures, single source cell subsamples, and the 2-cell mini mixture subsamples from the original 2-6-person mixtures. PG further allowed replicate analysis to be employed which, in many instances, resulted in a significant gain of genotype information such that the returned donor likelihood ratios (LRs) were comparable to that seen in their single source reference profiles (i.e., the reciprocal of their random match probabilities). In every mixture, the DSCS approach gave improved results for each donor compared to standard bulk mixture analysis. With the 5- and 6- person complex mixtures, DSCS recovered highly probative LRs (&gt; 1020) from donors that had returned non-probative LRs (&lt;103) by standard methods.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.06.467539', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Whole-genome sequencing of Chinese native goat offers biological insights into cashmere fiber formation', 'authors': \"[{'name': 'Hu Han'}, {'name': 'Man-Man Yang'}, {'name': 'Dan Jiang'}, {'name': 'Xing-Ju Zhang'}, {'name': 'Qiang Wei'}, {'name': 'Tao Chen'}, {'name': 'Qi-Ju Wang'}, {'name': 'Cheng-Ye Yang'}, {'name': 'Bater Wulan'}, {'name': 'Ting-Ting Zhang'}, {'name': 'Geng Gang'}, {'name': 'Dala Mengke'}, {'name': 'Bin LI'}, {'name': 'Wei-Dong Deng'}, {'name': 'Ze-Pu Miao'}, {'name': 'Ran Wang'}, {'name': 'Qing-Feng Zhang'}, {'name': 'Lin Li'}, {'name': 'Sheng-Yu CHao'}, {'name': 'Ming Fang'}, {'name': 'Yong Li'}]\", 'abstract': '<p>Cashmere evolved naturally in the goat, and almost all breeds of goat can produce more or less cashmere fibers. However, the genetic alterations underlying cashmere trait selection are still unclear.We sequenced 120 Chinese native goat including two cashmere goat breeds (Ujumain, Chaidamu) and six ordinary goat breeds (Jining Gray, Matou, Guizhou Black, Jintang Black, Yunnan Black Bone, Chengdu Brown). The genome-wide selective sweep of cashmere goat and ordinary goat revealed a novel set of candidate genes as well as pathways, such as Nuclear factor kappa-B and Wnt Signaling pathways. Of them, the LHX2 gene regulating hair follicle development, was evident from the strongest selection signal when comparing the Uhumqin cashmere goat and ordinary goat. Interestingly, we identified a 582bp deletion at 367 kb upstream of LHX2 with higher frequency in cashmere goats and their ancient relatives. This mutation probably rises along the breeding procedures, and is putatively responsible for cashmere production and diameter, as revealed by association studies. Luciferase assay shows that the deletion, which acts as an insulator, restrains the expression of LHX2 by interfering its upstream enhancers.Our study discovers a novel insulator of the LHX2 involved in regulating cashmere production and diameter, which would be beneficial to understanding hair follicle development and regeneration. Our findings also provide new insights into the genetic formation of cashmere, and facilitate subsequent molecular breeding for cashmere goat improvement.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.03.467033', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'A multi-species co-occurrence index to avoid type II errors in null model testing', 'authors': \"[{'name': 'Vitalis K. Lagat'}, {'name': 'Guillaume Latombe'}, {'name': 'Cang Hui'}]\", 'abstract': '<p>Community structure is determined by the interplay among different processes, including biotic interactions, abiotic filtering and dispersal. Their effects can be detected by comparing observed patterns of co-occurrence between different species (e.g. C-score and the natural metric) to patterns generated by null models based on permutations of species-by-site matrices under constraints on row or column sums. These comparisons enable us to detect significant signals of species association or dissociation, from which the type of biotic interactions between species (e.g. facilitative or antagonistic) can be inferred. Commonly used patterns are based on the levels of co-occurrence between randomly paired species. The level of co-occurrence for three or more species is rarely considered, ignoring the potential existence of functional guilds or motifs composed of multiple species within the community. Null model tests that do not consider multi-species co-occurrence could therefore generate false negatives (Type II error) in detecting non-random forces at play that would only be apparent for such guilds. Here, we propose a multi-species co-occurrence index (hereafter, joint occupancy) that measures the number of sites jointly occupied by multiple species simultaneously, of which the pairwise metric of co-occurrence is a special case. Using this joint occupancy index along with standard permutation algorithms for null model testing, we illustrate nine archetypes of multi-species co-occurrence and explore how frequent they are in the seminal database of 289 species-by-site community matrices published by Atmar and Patterson in 1995. We show that null model testing using pairwise co-occurrence metrics could indeed lead to severe Type II errors in one specific archetype, accounting for 2.4% of the tested community matrices.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.21265869', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Phase-locked transcranial electrical brain stimulation for tremor suppression in dystonic tremor syndromes', 'authors': \"[{'name': 'Freek Nieuwhof'}, {'name': 'Ivan Toni'}, {'name': 'Arthur W.G. Buijink'}, {'name': 'Anne-Fleur van Rootselaar'}, {'name': 'Bart P.C. van de Warrenburg'}, {'name': 'Rick C. Helmich'}]\", 'abstract': '<sec><title>Background</title><p>Tremor is a common and burdensome symptom in patients with dystonia, which is clinically heterogeneous and often resistant to treatment. The pathophysiology is suggested to involve abnormal activity in the cerebellum and motor cortex, but the causal role of these brain regions remains to be established. Transcranial alternating current stimulaton (TACS) can suppress rhytmic cerebral activity in other tremor disorders when phase-locked to the ongoing arm tremor, but the effect on dystonic tremor syndromes is unknown.</p></sec><sec><title>Objective/Hypothesis</title><p>We aimed to establish the causal role of the cerebellum and motor cortex in dystonic tremor syndromes, and explore the therapeutic efficacy of phase-locked TACS.</p></sec><sec><title>Methods</title><p>We applied phase-locked TACS over the ipsilateral cerebellum (N=14) and contralateral motor cortex (N=17) in dystonic tremor syndrome patients, while patients assumed a tremor-evoking posture. We measured tremor power using accelerometery during 30s stimulation periods at 10 different phase-lags (36-degrees increments) between tremor and TACS for each target. Post-hoc, TACS-effects were related to a key clinical feature: the jerkiness (regularity) of tremor.</p></sec><sec><title>Results</title><p>Cerebellar TACS modulated tremor amplitude in a phase-dependent manner, such that tremor amplitude was suppressed or enhanced at opposite sides of the phase-cycle. This effect was specific for patients with non-jerky (sinusoidal) tremor (n=10), but absent in patients with jerky (irregular) tremor (n=4). Phase-locked stimulation over the motor cortex did not modulate tremor amplitude.</p></sec><sec><title>Conclusions</title><p>This study indicates that the cerebellum plays a causal role in the generation of (non-jerky) dystonic tremor syndrome. Our findings suggest pathophysiologic heterogeneity between patients with dystonic tremor syndrome, which mirrors clinical variability.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.21265807', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Use cases for COVID-19 screening and surveillance with rapid antigen-detecting tests: a systematic review', 'authors': \"[{'name': 'Apoorva Anand'}, {'name': 'Jacob Bigio'}, {'name': 'Emily MacLean'}, {'name': 'Talya Underwood'}, {'name': 'Nitika Pant Pai'}, {'name': 'Sergio Carmona'}, {'name': 'Samuel G. Schumacher'}, {'name': 'Amy Toporowski'}]\", 'abstract': '<sec><title>Introduction</title><p>Testing is critical to controlling the COVID-19 pandemic. Antigen-detecting rapid diagnostic tests (Ag-RDTs) that can be used at the point of care have the potential to increase access to COVID-19 testing, particularly in settings with limited laboratory capacity. This systematic review synthesized literature on specific use cases and performance of Ag-RDTs for detecting SARS-CoV-2, for the first comprehensive assessment of Ag-RDT use in real-world settings.</p></sec><sec><title>Methods</title><p>We searched three databases (PubMed, EMBASE and medRxiv) up to 12 April 2021 for publications on Ag-RDT use for large-scale screening, irrespective of symptoms, and surveillance of COVID-19, excluding studies of only presumptive COVID-19 patients. We tabulated data on the study setting, populations, type of test, diagnostic performance and operational findings. We assessed risk of bias using QUADAS-2 and an adapted tool for prevalence studies.</p></sec><sec><title>Results</title><p>From 4313 citations, 39 studies conducted in asymptomatic and symptomatic adults were included. Study sample sizes varied from 40 to &gt;5 million. Of 39 studies, 37 (94.9%) investigated lateral flow Ag-RDTs and two (5.1%) investigated multiplex sandwich chemiluminescent enzyme immunoassay Ag-RDTs. Six categories of testing (screening/surveillance) initiatives were identified: mass screening (n=13), targeted screening (n=11), healthcare entry testing (n=6), at-home testing (n=4), surveillance (n=4) and prevalence survey (n=1). Across studies, Ag-RDT sensitivity varied from 40% to 100%. Ag-RDTs were noted as convenient, easy-to-use and low cost, with a rapid turnaround time and high user acceptability. Risk of bias was generally low or unclear across the studies.</p></sec><sec><title>Conclusion</title><p>This systematic review demonstrates the use of Ag-RDTs across a wide range of real-world settings for screening and surveillance of COVID-19 in both symptomatic and asymptomatic individuals. Ag-RDTs were overall found to be easy-to-use, low cost and rapid tools, when consideration is given to their implementation and interpretation. The review was funded by FIND, the global alliance for diagnostics.</p></sec><sec><title>SUMMARY</title><sec><title>What is already known?</title><list list-type=\"bullet\"><list-item><p>Antigen-detecting rapid diagnostic tests (Ag-RDTs) have the potential to substantially improve access to timely testing for COVID-19 and are being deployed in a variety of settings around the world</p></list-item><list-item><p>While studies have investigated the diagnostic accuracy of Ag-RDTs, less is known about how and in what settings Ag-RDTs are being used around the world and their performance in these different settings</p></list-item></list></sec><sec><title>What are the new findings?</title><list list-type=\"bullet\"><list-item><p>Ag-RDTs are being used in a diverse range of real-world settings for mass screening and surveillance of COVID-19 among symptomatic and asymptomatic individuals</p></list-item><list-item><p>The sensitivity of Ag-RDTs is variable - ranging from 40% to 100% - and in some cases low compared with RT-PCR, meaning that the value of testing with Ag-RDTs needs to be carefully evaluated for each use case taking into account factors such as the prevalence of COVID-19 in the population, the consequences of false positive or false negative results, and whether confirmatory testing of positive or negative Ag-RDT results with RT-PCR is required</p></list-item><list-item><p>Nevertheless, Ag-RDTs are generally reported as being easy to use and low cost, with a rapid turnaround time that enables timely identification of cases and subsequent interventions to prevent onward transmission of COVID-19</p></list-item></list></sec><sec><title>What do the new findings imply?</title><list list-type=\"bullet\"><list-item><p>The evidence indicates that Ag-RDTs can be effectively deployed across a broad range of settings when consideration is given to how they are implemented and interpreted</p></list-item><list-item><p>The development of more detailed, evidence-based testing policies for Ag-RDTs will be important to help countries implement effective testing programmes and make the best use of Ag-RDTs as part of the COVID-19 testing toolkit</p></list-item></list></sec></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.21265803', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Cerebello-thalamic activity drives an abnormal motor network into dystonic tremor', 'authors': \"[{'name': 'Freek Nieuwhof'}, {'name': 'Ivan Toni'}, {'name': 'Michiel F. Dirkx'}, {'name': 'Cecile Gallea'}, {'name': 'Marie Vidailhet'}, {'name': 'Arthur W.G. Buijink'}, {'name': 'Anne-Fleur van Rootselaar'}, {'name': 'Bart P.C. van de Warrenburg'}, {'name': 'Rick C. Helmich'}]\", 'abstract': '<p>Dystonic tremor syndromes are highly burdensome and treatment is often inadequate. This is partly due to poor understanding of the underlying pathophysiology. Several lines of research suggest involvement of the cerebello-thalamo-cortical circuit and the basal ganglia in dystonic tremor syndromes, but their role is unclear. Here we aimed to investigate the contribution of the cerebello-thalamo-cortical circuit and the basal ganglia to the pathophysiology of dystonic tremor syndrome, by directly linking tremor fluctuations to cerebral activity during scanning. In 27 patients with dystonic tremor syndrome (dystonic tremor: <italic>n</italic>=23; tremor associated with dystonia: <italic>n</italic>=4), we used concurrent accelerometery and functional MRI during a posture holding task that evoked tremor, alternated with rest. Using multiple regression analyses, we separated tremor-related activity from brain activity related to (voluntary) posture holding. Using dynamic causal modelling, we tested for altered effective connectivity between tremor-related brain regions as a function of tremor amplitude fluctuations. Finally, we compared grey matter volume between patients (<italic>n</italic>=27) and matched controls (<italic>n</italic>=27).</p><p>We found tremor-related activity in sensorimotor regions of the bilateral cerebellum, contralateral ventral intermediate (VIM) and ventro-oralis posterior nuclei (VOp) of the thalamus, contralateral primary motor cortex (hand area), contralateral pallidum, and the bilateral frontal cortex (laterality with respect to the tremor). Grey matter volume was increased in patients compared to controls in the portion of contralateral thalamus also showing tremor-related activity, as well as in bilateral medial and left lateral primary motor cortex, where no tremor-related activity was present. Effective connectivity analyses showed that inter-regional coupling in the cerebello-thalamic pathway, as well as the thalamic self-connection, were strengthened as a function of increasing tremor power.</p><p>These findings indicate that the pathophysiology of dystonic tremor syndromes involves functional and structural changes in the cerebello-thalamo-cortical circuit and pallidum. Deficient input from the cerebellum towards the thalamo-cortical circuit, together with hypertrophy of the thalamus, may play a key role in the generation of dystonic tremor syndrome.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.467115', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Blueprint for Phasing and Assembling the Genomes of Heterozygous Polyploids: Application to the Octoploid Genome of Strawberry', 'authors': \"[{'name': 'Michael A Hardigan'}, {'name': 'Mitchell J Feldmann'}, {'name': 'Dominique DA Pincot'}, {'name': 'Randi A Famula'}, {'name': 'Michaela V Vachev'}, {'name': 'Mary A Madera'}, {'name': 'Philipp J Zerbe'}, {'name': 'Kristin Mars'}, {'name': 'Paul Peluso'}, {'name': 'David Rank'}, {'name': 'Shujun Ou'}, {'name': 'Christopher A Saski'}, {'name': 'Charlotte B Acharya'}, {'name': 'Glenn S Cole'}, {'name': 'Alan E Yocca'}, {'name': 'Adrian E Platts'}, {'name': 'Patrick P Edger'}, {'name': 'Steven J Knapp'}]\", 'abstract': \"<p>The challenge of allelic diversity for assembling haplotypes is exemplified in polyploid genomes containing homoeologous chromosomes of identical ancestry, and significant homologous variation within their ancestral subgenomes. Cultivated strawberry (Fragaria x ananassa) and its wild progenitors are outbred octoploids (2n = 8x = 56) in which up to eight homologous and homoeologous alleles are preserved. This introduces significant risk of haplotype collapse, switching, and chimeric fusions during assembly. Using third generation HiFi sequences from PacBio, we assembled the genome of the day-neutral octoploid F. x ananassa hybrid 'Royal Royce' from the University of California. Our goal was to produce subgenome- and haplotype-resolved assemblies of all 56 chromosomes, accurately reconstructing the parental haploid chromosome complements.  Previous work has demonstrated that partitioning sequences by parental phase supports direct assembly of haplotypes in heterozygous diploid species. We leveraged the accuracy of HiFi sequence data with pedigree-informed sequencing to partition long read sequences by phase, and reduce the downstream risk of subgenomic chimeras during assembly. We were able to utilize an octoploid strawberry recombination breakpoint map containing 3.6 M variants to identify and break chimeric junctions, and perform scaffolding of the phase-1 and phase-2 octoploid assemblies. The N50 contiguity of the phase-1 and phase-2 assemblies prior to scaffolding and gap-filling was 11 Mb. The final haploid assembly represented seven of 28 chromosomes in a single contiguous sequence, and averaged fewer than three gaps per pseudomolecule. Additionally, we re-annotated the octoploid genome to produce a custom F. x ananassa repeat library and improved set of gene models based on IsoSeq transcript data and an expansive RNA-seq expression atlas. Here we present 'FaRR1', a gold-standard reference genome of F. x ananassa cultivar 'Royal Royce' to assist future genomic research and molecular breeding of allo-octoploid strawberry.</p>\", 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467176', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'A microRNA cluster downstream of the selector gene Fezf2 coordinates fate specification with dendritic branching in cortical neurons', 'authors': \"[{'name': 'Asha Iyer'}, {'name': 'Verl B Siththanandan'}, {'name': 'Victoria Lu'}, {'name': 'Ramesh Nair'}, {'name': 'Lee O. Vaasjo'}, {'name': 'Maria J Galazo'}, {'name': 'Suzanne Tharin'}]\", 'abstract': '<p>In the cerebral cortex, cortical projection neurons comprise classes of neurons project to distant regions of the central nervous system. These neurons develop from the same progenitor pool, but they acquire strikingly different inputs and outputs to underpin strikingly different functions. The question of how corticospinal projection neurons - involved in motor function and implicated in paralysis - and callosal projection neurons - involved in cognitive function and implicated in autism - develop represents a fundamental and clinically important question in neurodevelopment. A network of transcription factors, including the selector gene Fezf2, is central to specifying cortical projection neuron fates. Gene regulation up- and down-stream of these transcription factors, however, is not well understood, particularly as it relates to the development of the major inputs to cortical projection neurons. Here we show that the miR-193b~365 microRNA cluster downstream of Fezf2 cooperatively represses the signaling molecule Mapk8, and impacts dendritic branching of cortical projection neurons.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.04.467016', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Repeats Mimic Immunostimulatory Viral Features Across a Vast Evolutionary Landscape', 'authors': \"[{'name': 'Petr Šulc'}, {'name': 'Alexander Solovyov'}, {'name': 'Sajid A Marhon'}, {'name': 'Siyu Sun'}, {'name': 'John A LaCava'}, {'name': 'Omar I. Abdel-Wahab'}, {'name': 'Nicolas Vabret'}, {'name': 'Daniel De Carvalho'}, {'name': 'Rémi Monasson'}, {'name': 'Simona Cocco'}, {'name': 'Benjamin D. Greenbaum'}]\", 'abstract': '<p>An emerging hallmark across many human diseases – such as cancer, autoimmune and neurodegenerative disorders – is the aberrant transcription of typically silenced repetitive elements. Once transcribed they can mimic pathogen-associated molecular patterns and bind pattern recognition receptors, thereby engaging the innate immune system and triggering inflammation in a process known as viral mimicry. Yet how to quantify pathogen mimicry, and the degree to which it is shaped by natural selection, remains a gap in our understanding of both genome evolution and the immunological basis of disease. Here we propose a theoretical framework that combines recent biological observations with statistical physics and population genetics to quantify the selective forces on virus-like features generated by repeats and integrate these forces into predictive evolutionary models. We establish that many repeat families have evolutionarily maintained specific classes of viral mimicry. We show that for HSATII and intact LINE-1 selective forces maintain CpG motifs, while for a set of SINE and LINE elements the formation of long double-stranded RNA is more prevalent than expected from a neutral evolutionary model. We validate our models by showing predicted immunostimulatory inverted SINE elements bind the MDA5 receptor under conditions of epigenetic dysregulation and that they are disproportionately present during intron retention when RNA splicing is pharmacologically inhibited. We conclude viral mimicry is a general evolutionary mechanism whereby genomes co-opt features generated by repetitive sequences to trigger the immune system, acting as a quality control system to flag genome dysregulation. We demonstrate these evolutionary principles can be learned and applied to predictive models. Our work therefore serves as a resource to identify repeats with candidate immunostimulatory features and leverage them therapeutically.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.04.21265918', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': True, 'title': 'Longitudinal Changes of Cardiac and Aortic Imaging Phenotypes Following COVID-19 in the UK Biobank Cohort', 'authors': \"[{'name': 'Wenjia Bai'}, {'name': 'Betty Raman'}, {'name': 'Steffen E Peterson'}, {'name': 'Stefan Neubauer'}, {'name': 'Zahra Raisi-Estabragh'}, {'name': 'Nay Aung'}, {'name': 'Nicholas C Harvey'}, {'name': 'Naomi Allen'}, {'name': 'Rory Collins'}, {'name': 'Paul M Matthews'}]\", 'abstract': '<p>Case studies conducted after recovery from acute infection with SARS-CoV-2 have frequently identified abnormalities on CMR imaging, suggesting the possibility that SARS-CoV-2 infection commonly leads to cardiac pathology.  However, these observations have not been able to distinguish between associations that reflect pre-existing cardiac abnormalities (that might confer a greater likelihood of more severe infection) from those that arise as consequences of infection.  To address this question, UK Biobank volunteers (n=1285; 54.5% women; mean age at baseline, 59.8 years old; 96.3% white) who attended an imaging assessment including cardiac magnetic resonance (CMR) before the start of the COVID-19 pandemic were invited to attend a second imaging assessment in 2021.  Cases with evidence of previous SARS-CoV-2 infection were identified through linkage to PCR-testing or other medical records, or a positive antibody lateral flow test; n=640 in data available on 22 Sep 2021) and were matched to controls with no evidence of previous infection (n=645). The majority of these infections were milder and did not involve hospitalisation.  Measures of cardiac and aortic structure and function were derived from the CMR images obtained on the cases before and after SARS-CoV-2 infection from images for the controls obtained over the same time interval using a previously validated, automated algorithm.  Cases and controls had similar cardiac and aortic imaging phenotypes at their first imaging assessment.  Changes between CMR imaging measures in cases before and after infection were not significantly different from those in the matched control group. Additional adjustment for comorbidities made no material difference to the results.  While these results are preliminary and limited to imaging metrics derived from automated analyses, they do not suggest clinically significant persistent cardiac pathology in the UK Biobank population after generally milder (non-hospitalised) SARS-CoV-2 infection.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.467191', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'miR-409-3p represses Cited2 at the evolutionary emergence of the callosal and corticospinal projections', 'authors': \"[{'name': 'Nikolaus R Wagner'}, {'name': 'Ashis Sinha'}, {'name': 'Verl B Siththanandan'}, {'name': 'Angelica N Kowalchuk'}, {'name': 'Jessica MacDonald'}, {'name': 'Suzanne Tharin'}]\", 'abstract': '<p>Callosal projection neurons are a broad population of interhemispheric projection neurons that extend an axon across the corpus callosum to connect the two cerebral hemispheres. The corticospinal tract, comprised of the axons of corticospinal projection neurons, is unique to mammals, and its full extension to the lumbar segments that control walking is, like the corpus callosum, unique to placental mammals. The emergence of these two distinct axonal tracts is thought to underpin the evolutionary expansion of complex motor and cognitive abilities. The molecular mechanisms regulating the divergence of corticospinal and callosal projection neurons are incompletely understood. Our recent work identifies a genomic cluster of microRNAs (12qF1/Mirg) unique to placental mammals. These clustered miRNAs are specifically expressed by corticospinal vs. callosal projection neurons during the molecular refinement of corticospinal vs. callosal projection neuron fate (1). One of these, miR-409-3p, can convert layer V callosal into corticospinal projection neurons, acting in part through repression of the callosal-expressed transcriptional regulator Lmo4. This conversion is partial, however, suggesting that miR-409-3p represses multiple callosal projection neuron control genes in order to specify corticospinal projection neurons. One potential additional target of miR-409-3p repression is the callosal-expressed transcriptional co-activator Cited2. Cited2 interacts genetically with Lmo4, and Lmo4 can partially functionally compensate for Cited2 in thymus development(2). Further, Cited2 and Lmo4 function as opposing molecular controls over specific areal identity within superficial layer callosal projection neurons of the somatosensory and motor cortices, respectively (3). Cited2 is highly expressed by callosal, relative to corticospinal, projection neurons from the earliest stages of neurogenesis. Cited2 is necessary for the expansion of intermediate progenitor cells (IPCs) in the subventricular zone (SVZ), and the resulting generation of superficial layer callosal projection neurons. Here we show that miR-409-3p and Cited2 interact in IPCs and in corticospinal vs. deep layer callosal projection neuron development. miR-409-3p represses the Cited2 3UTR in luciferase assays. Mirg, which encodes miR-409-3p, and Cited2, are reciprocally expressed in IPCs at e15.5 by qPCR.  Furthermore, miR-409-3p gain-of-function results in a phenocopy of established Cited2 loss-of-function in IPCs. Later on, miR-409-3p and Cited2 exert opposing effects on the adoption of corticospinal vs. callosal projection neuron subtype identity. Taken together, our work suggests that miR-409-3p, and possibly other 12qF1 miRNAs, represses Cited2 in IPCs to limit their proliferation, and in developing corticospinal and deep layer callosal projection neurons to favor corticospinal fate.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467126', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Towards routine employment of computational tools for antimicrobial resistance determination via high-throughput sequencing', 'authors': \"[{'name': 'Simone Marini'}, {'name': 'Rodrigo A. Mora'}, {'name': 'Christina Boucher'}, {'name': 'Noelle Noyes'}, {'name': 'Mattia Prosperi'}]\", 'abstract': '<p>Antimicrobial resistance (AMR) is a growing threat to public health and farming at large. Without appropriate interventions, it can lead to millions of deaths per year and substantial economic loss worldwide. In clinical and veterinary practice, a timely characterization of the antibiotic susceptibility profile of bacterial infections is a crucial step in optimizing treatment. Fast turnaround of AMR testing is also needed in food safety and infection control surveillance (e.g., contamination of healthcare or long-term nursing facilities). High-throughput sequencing is a promising option for clinical point-of-care and ecological surveillance, opening the opportunity to develop genotyping-based AMR determination as a possibly faster alternative to phenotypic testing. In the present work, we compare the performance of state-of-the-art methods for detection of AMR from high-throughput sequencing data in healthcare settings. We consider five complementary computational approaches –alignment (AMRPlusPlus), deep learning (DeepARG), <italic>k</italic>-mer genomic signatures (KARGA, Res-Finder), and hidden Markov models (Meta-MARC). We use an extensive collection of clinical studies never employed for model training. To do so, we assemble data from multiple, independent AMR high-throughput sequencing experiments collected in a variety of hospital settings, comprising of 585 isolates with a available AMR resistance profiles determined by phenotypic tests across nine antibiotic classes. We show how the pre-diction landscape of AMR classifiers is highly heterogeneous, with balanced accuracy varying from 0.4 to 0.92. Although some algorithms—ResFinder, KARGA, and AMRPlusPlus– exhibit overall better balanced accuracy than others, the high per-AMR-class variance and related findings suggest that: (1) all algorithms might be subject to sampling bias present both in data repositories used for training and experimental/clinical settings; and (2) a portion of clinical samples might contain uncharacterized AMR genes that the algorithms—mostly trained on known AMR genes—fail to generalize upon. These results lead us to formulate practical advice for software configuration and application, as well as give suggestions for future study design to further develop AMR prediction tools from proof-of-concept to bedside.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466806', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Decision-Making in Rats is Sensitive to Rare and Extreme Events: the Black Swan Avoidance', 'authors': \"[{'name': 'Mickaël Degoulet'}, {'name': 'Louis-Matis Willem'}, {'name': 'Christelle Baunez'}, {'name': 'Stéphane Luchini'}, {'name': 'Patrick A. Pintus'}]\", 'abstract': '<p>Most studies assessing decision-making under uncertainty use events with probabilities that are above 10-20 %. Here, to study decision-making in radical uncertainty conditions, Degoulet, Willem, Baunez, Luchini and Pintus provide a novel experimental design that aims at measuring the extent to which rats are sensitive - and how they respond - to extremely rare (below 1% of probability) but extreme events in a four-armed bandit task. Gains (sugar pellets) and losses (time-out punishments) are such that large - but rare - values materialize or not depending on the option chosen. The results show that all rats diversify their choices across options. However, most rats exhibit sensitivity to rare and extreme events despite their sparse occurrence, by combining more often options with extreme gains (Jackpots) and/or avoidance of extreme losses (Black Swans). In general, most rats’ choices feature one-sided sensitivity in favor of trying more often to avoid extreme losses than to seek extreme gains - that is, they feature Black Swan Avoidance.</p><sec><title>Highlights</title><list list-type=\"simple\"><list-item><label>-</label><p>A novel experiment is proposed to measure whether rats are sensitive - and how they respond - to radical uncertainty materialized by extremely rare and extreme events in a four-armed bandit task;</p></list-item><list-item><label>-</label><p>Most rats exhibit sensitivity to rare and extreme events;</p></list-item><list-item><label>-</label><p>Most rats opt more often to avoid rare and extreme losses than to seek rare and extreme gains: they feature Black Swan avoidance.</p></list-item></list></sec>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.10.30.21265662', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Intensive Care and Critical Care Medicine', 'covid': True, 'title': 'Heterogenous pattern of the inflammasome-related markers in the critically ill COVID-19 patients at the ICU admission', 'authors': \"[{'name': 'Barbara Adamik'}, {'name': 'Magdalena Ambrożek-Latecka'}, {'name': 'Barbara Dragan'}, {'name': 'Aldona Jeznach'}, {'name': 'Jakub Śmiechowicz'}, {'name': 'Waldemar Goździk'}, {'name': 'Tomasz Skirecki'}]\", 'abstract': '<sec><title>Purpose</title><p>Development of targeted biological therapies for COVID-19 requires reliable biomarkers that could help indicate the responding patients. Hyperactivation of the inflammasome by SARS-CoV2 virus is hypothesized to contribute to severe course of the COVID-19 disease. Therefore, we aimed to evaluate the prognostic value of several inflammasome-related cytokines and proteins at the admission to the intensive care unit (ICU).</p></sec><sec><title>Patients and methods</title><p>Plasma samples were obtained from 45 critically ill COVID-19 patients and from10 patients with severe traumatic brain injury (TBI) at the admission to the ICU. The concentration of IL-1<italic>α</italic>, IL-1<italic>β</italic>, IL-18, IL-1RA, galectin-1, ASC, LDH, ferritin, and gasdermin D were analyzed. A novel cell-free caspase-1 plasma assay was developed by inhibitor-based immunoprecipitation followed by Western Blot. Demographic and clinical characteristics were recorded.</p></sec><sec><title>Results</title><p>The inflammasome-related biomarkers were in similar concentration in COVID-19 and TBI patients except for galectin-1 being lower in the former. None of the tested markers were related to the outcome, length of stay or development of secondary infections. Patients with SOFA score of &gt;9 at admission who were at high risk of death had significantly higher galectin-1 but lower IL-1RA in comparison to low-risk patients. Weak but significant correlations were observed between: IL-1<italic>α</italic> and platelets, IL-1<italic>β</italic> and platelets, ferritin and INR. Activated caspase-1 p35 was detectable in 12/22 COVID-19 patients but in none of the TBI patients. Its presence was related with higher fibrinogen and lower D-dimers. It was also significantly higher in patients with SOFA&gt;9.</p></sec><sec><title>Conclusion</title><p>Our results indicate that the activation of the inflammasome in critically ill COVID-19 patients is a heterogenous process and is not directly related with outcome. Therefore, potential interventions aimed at the inflammasome pathway in this group of patients may be of limited effectiveness and should be biomarker-guided.</p></sec>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.01.466686', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'A low-resource reliable pipeline to democratize multi-modal connectome estimation and analysis', 'authors': \"[{'name': 'Ross Lawrence'}, {'name': 'Alex Loftus'}, {'name': 'Gregory Kiar'}, {'name': 'Eric W. Bridgeford'}, {'name': 'William Gray Roncal'}, {'name': 'Vikram Chandrashekhar'}, {'name': 'Disa Mhembere'}, {'name': 'Sephira Ryman'}, {'name': 'Xi-Nian Zuo'}, {'name': 'Daniel S. Margulies'}, {'name': 'R. Cameron Craddock'}, {'name': 'Carey E. Priebe'}, {'name': 'Rex Jung'}, {'name': 'Vince D. Calhoun'}, {'name': 'Brian Caffo'}, {'name': 'Randal Burns'}, {'name': 'Michael P. Milham'}, {'name': 'Joshua T. Vogelstein'}, {'name': ' '}]\", 'abstract': '<p>Connectomics—the study of brain networks—provides a unique and valuable opportunity to study the brain. However, research in human connectomics, accomplished via Magnetic Resonance Imaging (MRI), is a resource-intensive practice: typical analysis routines require impactful decision making and significant computational capabilities. Mitigating these issues requires the development of low-resource, easy to use, and flexible pipelines which can be applied across data with variable collection parameters. In response to these challenges, we have developed the MRI to Graphs (<monospace>m2g</monospace>) pipeline. <monospace>m2g</monospace> leverages functional and diffusion datasets to estimate connectomes reliably. To illustrate, <monospace>m2g</monospace> was used to process MRI data from 35 different studies (≈6,000 scans) from 15 sites without any manual intervention or parameter tuning. Every single scan yielded an estimated connectome that followed established properties, such as stronger ipsilateral than contralateral connections in structural connectomes, and stronger homotopic than heterotopic correlations in functional connectomes. Moreover, the connectomes generated by <monospace>m2g</monospace> are more similar within individuals than between them, suggesting that <monospace>m2g</monospace> preserves biological variability. <monospace>m2g</monospace> is portable, and can run on a single CPU with 16 GB of RAM in less than a couple hours, or be deployed on the cloud using its docker container. All code is available on <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://neurodata.io/mri/\">https://neurodata.io/mri/</ext-link>.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466736', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Optimality of extracellular enzyme production and activity in dynamic flux balance modeling', 'authors': \"[{'name': 'Michael Quintin'}, {'name': 'Ilija Dukovski'}, {'name': 'Jennifer Bhatnagar'}, {'name': 'Daniel Segrè'}]\", 'abstract': '<p>In microbial communities, many vital metabolic functions, including the degradation of cellulose, proteins and other complex macromolecules, are carried out by costly, extracellularly secreted enzymes. While significant effort has been dedicated to analyzing genome-scale metabolic networks for individual microbes and communities, little is known about the interplay between global allocation of metabolic resources in the cell and extracellular enzyme secretion and activity. Here we introduce a method for modeling the secretion and catalytic functions of extracellular enzymes using dynamic flux balance analysis. This new addition, implemented within COMETS (Computation Of Microbial Ecosystems in Time and Space), simulates the costly production and secretion of enzymes and their diffusion and activity throughout the environment, independent of the producing organism. After tuning our model based on data for a <italic>Saccharomyces cerevisiae</italic> strain engineered to produce exogenous cellulases, we explored the dynamics of the system at different cellulose concentrations and enzyme production rates. We found that there are distinct rates of constitutive enzyme secretion which maximize either growth rate or biomass yield. These optimal rates are strongly dependent on enzyme kinetic properties and environmental conditions, including the amount of cellulose substrate available. Our framework will facilitate the development of more realistic simulations of microbial community dynamics within environments rich in complex macromolecules, with applications in the study of soil and plant-associated ecosystems, and other natural and engineered microbiomes.</p><sec><title>Importance</title><p>Many organisms - including soil, marine and human-associated bacteria and fungi - perform part of their metabolic functions outside of the boundary of the cell, through the secretion of extracellular enzymes that can diffuse and facilitate reactions independently of the organism that produced them. In order to better understand and predict microbial ecosystems, it would be helpful to create mathematical models incorporating these extracellular reactions within simulations of metabolism at the whole-cell level. In this paper we demonstrate the implementation of such a methodology and apply it to study a cellulase-secreting yeast. This work will be useful for a number of microbial ecology applications, including modeling of microbiome dynamics, engineering of bioproducts (e.g. biofuels) from plant biomass through synthetic communities or modified organisms, and testing of basic ecological hypotheses about the balance between cost and benefits of the production of common goods in microbial communities.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.10.28.466361', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A retrograde mechanism coordinates memory allocation across brain regions', 'authors': \"[{'name': 'Ayal Lavi'}, {'name': 'Megha Sehgal'}, {'name': 'Fardad Sisan'}, {'name': 'Anna Okabe'}, {'name': 'Donara Ter-Mkrtchyan'}, {'name': 'Alcino J Silva'}]\", 'abstract': '<p>Memories engage ensembles of neurons across different brain regions within a memory system. However, it is unclear whether the allocation of a memory to these ensembles is coordinated across brain regions. To address this question, we used CREB expression to bias memory allocation in one brain region, and rabies retrograde tracing to test memory allocation in connected presynaptic neurons in the other brain regions. We find that biasing allocation of CTA memory in the basolateral amygdala (BLA) also biases memory allocation in presynaptic neurons of the insular cortex (IC). By manipulating the allocation of CTA memory to specific neurons in both BLA and IC, we found that we increased their connectivity and enhanced CTA memory performance. These results – which are corroborated by mathematical simulations, and by studies with auditory fear conditioning – demonstrate that a retrograde mechanism coordinates the allocation of memories across different brain regions.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.10.31.466680', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Vertex protein PduN tunes encapsulated pathway performance by dictating bacterial metabolosome morphology', 'authors': \"[{'name': 'Carolyn E. Mills'}, {'name': 'Curt Waltmann'}, {'name': 'Andre G. Archer'}, {'name': 'Nolan W. Kennedy'}, {'name': 'Charlotte H. Abrahamson'}, {'name': 'Alexander D. Jackson'}, {'name': 'Eric W. Roth'}, {'name': 'Sasha Shirman'}, {'name': 'Michael C. Jewett'}, {'name': 'Niall M. Mangan'}, {'name': 'Monica Olvera de la Cruz'}, {'name': 'Danielle Tullman-Ercek'}]\", 'abstract': '<p>Engineering subcellular organization in microbes shows great promise in addressing bottlenecks in metabolic engineering efforts; however, rules guiding selection of an organization strategy or platform are lacking. Here, we study compartment morphology as a factor in mediating encapsulated pathway performance. Using the 1,2-propanediol utilization microcompartment (Pdu MCP) system from <italic>Salmonella enterica</italic> serovar Typhimurium LT2, we find that we can shift the morphology of this protein nanoreactor from polyhedral to tubular by removing vertex protein PduN. Analysis of the metabolic function between these Pdu microtubes (MTs) shows that they provide a diffusional barrier capable of shielding the cytosol from a toxic pathway intermediate, similar to native MCPs. However, kinetic modeling suggests that the different surface area to volume ratios of MCP and MT structures alters encapsulated pathway performance. Finally, we report a microscopy-based assay that permits rapid assessment of Pdu MT formation to enable future engineering efforts on these structures.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.04.467332', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Functional characterization of mutants deficient in N-terminal phosphorylation of the CF1 subunit beta', 'authors': \"[{'name': 'Ralph Bock'}, {'name': 'Deserah D Strand'}, {'name': 'Daniel Karcher'}, {'name': 'Stephanie Ruf'}, {'name': 'Anne Schadach'}, {'name': 'Mark Aurel Schöttler'}, {'name': 'Omar Sandoval-Ibáñez'}]\", 'abstract': '<p>Understanding the regulation of photosynthetic light harvesting and electron transfer is of great importance to efforts to improve the ability of the electron transport chain to supply downstream metabolism. The central regulator of the electron transport chain is the ATP synthase, the molecular motor that harnesses the chemiosmotic potential generated from proton coupled electron transport to synthesize ATP. The ATP synthase is regulated both thermodynamically and post-translationally, with proposed phosphorylation sites on multiple subunits. In this study we focused on two N-terminal serines on the catalytic subunit beta, previously proposed to be important for dark inactivation of the complex to avoid ATP hydrolysis at night. Here we show that there is no clear role for phosphorylation in the dark inactivation of ATP synthase. Instead, mutation of one of the two phosphorylated serine residues to aspartate strongly decreased ATP synthase abundance. We propose that the loss of N-terminal phosphorylation of ATP beta may be involved in proper ATP synthase accumulation during complex assembly.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.03.21265861', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'The impact of the initial and 2nd national COVID-19 lockdown on mental health in young people with and without pre-existing depressive symptoms', 'authors': \"[{'name': 'Andrea Joensen'}, {'name': 'Stine Danielsen'}, {'name': 'Per Kragh Andersen'}, {'name': 'Jonathan Groot'}, {'name': 'Katrine Strandberg-Larsen'}]\", 'abstract': '<sec><title>Background</title><p>The evidence on mental well-being and loneliness among young people during the initial lockdown is mixed, and little is known about the long-lasting impact of the sequential lockdowns. We examine changes in young people’s mental health from before to during the initial and second more prolonged lockdown, and whether women and those with pre-existing depressive symptoms were disproportionally impacted.</p></sec><sec><title>Methods</title><p>Participants reported on mental health indicators in an ongoing 18-year data collection in the Danish National Birth Cohort and in a COVID-19 survey, including 8 data points: 7 in the initial lockdown, and 1 year post. Changes in quality of life (QoL), mental well-being, and loneliness were estimated with random effect linear regressions on longitudinal data (N=32,985), and linear regressions on repeated cross-sections (N=28,579).</p></sec><sec><title>Findings</title><p>Interim deterioration in mental well-being and loneliness was observed during the initial lockdown, and only in those without pre-existing depressive symptoms. During the second lockdown, a modest deterioration was again observed for mental well-being and loneliness. QoL likewise only declined among those without pre-existing symptoms, where women showed a greater decline than men. QoL did not normalise during the initial lockdown and remained at lower levels during the second lockdown. These findings were not replicated in the repeated cross-sections.</p></sec><sec><title>Interpretation</title><p>Except for an interim decrease in mental health during lockdown, and only in those without pre-existing depressive symptoms, this study’s findings do not suggest a substantial detrimental impact of the lockdowns. Potential methodological differences in-between studies are a possible explanation for the mixed evidence.</p></sec><sec><title>Funding</title><p>The Velux Foundation</p></sec><sec><title>Research in context</title><sec><title>Evidence before this study</title><p>We searched PubMed, PsycINFO, MedrXiv, and PsyArXiv with the terms (“Mental*” OR “Psychological*” OR “Emotional*”) AND (“Youth” OR “Young Adult*”) AND (“COVID*” OR “Coronavirus” OR “Lockdown*”) for articles published in English between January 1<sup>st</sup> 2020 and October 1<sup>st</sup> 2021. Included studies varied in terms of quality of data used but overall studies reported a detrimental impact of the lockdowns on young people’s mental health. However, the evidence on mental well-being and loneliness has shown to be inconsistent and with signs of resilience. Young people, women, and those with a pre-existing mental disorder have been identified as vulnerable subgroups, but only a few studies investigating mental health in individuals with a pre-existing mental disorder included a pre-lockdown measurement. The included studies also demonstrated that there is a gap in the evidence in understanding how mental health changed week by week, as well as the long-term impact over the course of the lockdowns.</p></sec><sec><title>Added value of this study</title><p>With longitudinal data, this study shows an interim impact of the initial and second lockdown on mental health during the COVID-19 pandemic in young individuals without pre-existing symptoms in Denmark. Since commencement of the initial lockdown, the levels of mental health returned to before levels, but one year after the initial lockdown, the levels were still lower than before lockdown in young people without pre-existing depressive symptoms. Young individuals with pre-existing depressive symptoms did not experience more detrimental impact of the lockdown, but rather indication of resilience or even improvements in mental health were observed. A disproportional impact of the lockdown on women compared to men was only observed for QoL, as women without pre-exiting depressive symptoms experienced a greater decline in QoL than men without pre-existing depressive symptoms. However, findings based on the repeated cross-sectional data did not show similar interim impact – but instead no – or clinically irrelevant impact. Thus, taken together our findings do not suggest a substantial lasting impact of the lockdowns on mental health among young individuals.</p></sec><sec><title>Implications of all available evidence</title><p>A great majority of earlier studies suggest that the lockdowns due to the COVID-19 pandemic have had substantial detrimental impact on mental health, and that women and those with a pre-existing mental disorder constitute vulnerable subgroups. However, these studies vary considerably in terms of method applied. Our findings emphasise the importance of the use of different data setups, as well as methodology applied for the investigation of mental health. More studies based on high-quality data used in different settings are needed to fully understand the impact of the lockdowns on young people’s mental health, including potential disproportional impact on vulnerable subgroups.</p></sec></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.04.467325', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Glioma synapses recruit mechanisms of adaptive plasticity', 'authors': \"[{'name': 'Kathryn R Taylor'}, {'name': 'Tara Barron'}, {'name': 'Helena Zhang'}, {'name': 'Alexa C Hui'}, {'name': 'Griffin G Hartmann'}, {'name': 'Lijun Ni'}, {'name': 'Humsa S Venkatesh'}, {'name': 'Peter Du'}, {'name': 'Rebecca Mancusi'}, {'name': 'Belgin Yalcin'}, {'name': 'Isabelle J Chau'}, {'name': 'Anitha Ponnuswami'}, {'name': 'Razina Aziz-Bose'}, {'name': 'Michelle Monje'}]\", 'abstract': '<p>The nervous system plays an increasingly appreciated role in the regulation of cancer. In malignant gliomas, neuronal activity drives tumor progression not only through paracrine signaling factors such as neuroligin-3 and brain-derived neurotrophic factor (BDNF), but also through electrophysiologically functional neuron-to-glioma synapses. Malignant synapses are mediated by calcium-permeable AMPA (α-amino-3-hydroxy-5-methyl-4-isoxazole propionic acid) receptors in both pediatric and adult high-grade gliomas, and consequent depolarization of the glioma cell membrane drives tumor proliferation. The nervous system exhibits plasticity of both synaptic connectivity and synaptic strength, contributing to neural circuit form and functions. In health, one factor that promotes plasticity of synaptic connectivity and strength is activity-regulated secretion of the neurotrophin BDNF. Here, we show that malignant synapses exhibit similar plasticity regulated by BDNF-TrkB (tropomyosin receptor kinase B) signaling. Signaling through the receptor TrkB, BDNF promotes AMPA receptor trafficking to the glioma cell membrane, resulting in increased amplitude of glutamate-evoked currents in the malignant cells. This potentiation of malignant synaptic strength shares mechanistic features with the long-term potentiation (LTP) that is thought to contribute to memory and learning in the healthy brain. BDNF-TrkB signaling also regulates the number of neuron-to-glioma synapses. Abrogation of activity-regulated BDNF secretion from the brain microenvironment or loss of TrkB in human glioma cells exerts growth inhibitory effects in vivo and in neuron:glioma co-cultures that cannot be explained by classical growth factor signaling alone. Blocking TrkB genetically or pharmacologically abrogates these effects of BDNF on glioma synapses and substantially prolongs survival in xenograft models of pediatric glioblastoma and diffuse intrinsic pontine glioma (DIPG). Taken together, these findings indicate that BDNF-TrkB signaling promotes malignant synaptic plasticity and augments tumor progression.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.21265877', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'REACT-1 round 15 interim report: High and rising prevalence of SARS-CoV-2 infection in England from end of September 2021 followed by a fall in late October 2021', 'authors': \"[{'name': 'Marc Chadeau-Hyam'}, {'name': 'Oliver Eales'}, {'name': 'Barbara Bodinier'}, {'name': 'Haowei Wang'}, {'name': 'David Haw'}, {'name': 'Matthew Whitaker'}, {'name': 'Caroline E. Walters'}, {'name': 'Christina Atchison'}, {'name': 'Peter J. Diggle'}, {'name': 'Andrew J. Page'}, {'name': 'Deborah Ashby'}, {'name': 'Wendy Barclay'}, {'name': 'Graham Taylor'}, {'name': 'Graham Cooke'}, {'name': 'Helen Ward'}, {'name': 'Ara Darzi'}, {'name': 'Christl A. Donnelly'}, {'name': 'Paul Elliott'}]\", 'abstract': '<sec><title>Background</title><p>The third wave of COVID-19 in England coincided with the rapid spread of the Delta variant of SARS-CoV-2 from the end of May 2021. Case incidence data from the national testing programme (Pillar 2) in England may be affected by changes in testing behaviour and other biases. Community surveys may provide important contextual information to inform policy and the public health response.</p></sec><sec><title>Methods</title><p>We estimated patterns of community prevalence of SARS-CoV-2 infection in England using RT-PCR swab-positivity, demographic and other risk factor data from round 15 (interim) of the REal-time Assessment of Community Transmission-1 (REACT-1) study (round 15a, carried out from 19 to 29 October 2021). We compared these findings with those from round 14 (9 to 27 September 2021).</p></sec><sec><title>Results</title><p>During mid- to late-October 2021 (round 15a) weighted prevalence was 1.72% (1.61%, 1.84%) compared to 0.83% (0.76%, 0.89%) in September 2021 (round 14). The overall reproduction number (R) from round 14 to round 15a was 1.12 (1.11, 1.14) with increases in prevalence over this period (September to October) across age groups and regions except Yorkshire and The Humber. However, within round 15a (mid- to late-October) there was evidence of a fall in prevalence with R of 0.76 (0.65, 0.88). The highest weighted prevalence was observed among children aged 5 to 12 years at 5.85% (5.10%, 6.70%) and 13 to 17 years at 5.75% (5.02%, 6.57%). At regional level, there was an almost four-fold increase in weighted prevalence in South West from round 14 at 0.59% (0.43%,0.80%) to round 15a at 2.18% (1.84%, 2.58%), with highest smoothed prevalence at subregional level also found in South West in round 15a. Age, sex, key worker status, and presence of children in the home jointly contributed to the risk of swab-positivity. Among the 126 sequenced positive swabs obtained up until 23 October, all were Delta variant; 13 (10.3%) were identified as the AY.4.2 sub-lineage.</p></sec><sec><title>Discussion</title><p>We observed the highest overall prevalence of swab-positivity seen in the REACT-1 study in England to date in round 15a (October 2021), with a two-fold rise in swab-positivity from round 14 (September 2021). Despite evidence of a fall in prevalence from mid- to late-October 2021, prevalence remains high, particularly in school-aged children, with evidence also of higher prevalence in households with one or more children. Thus, vaccination of children aged 12 and over remains a high priority (with possible extension to children aged 5-12) to help reduce within-household transmission and disruptions to education, as well as among adults, to lessen the risk of serious disease among those infected.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.04.467269', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'The Conformational Plasticity of the Selectivity Filter Methionines Controls the In-Cell Cu(I) Uptake through the CTR1 transporter', 'authors': \"[{'name': 'Pavel Janoš'}, {'name': 'Jana Aupič'}, {'name': 'Sharon Ruthstein'}, {'name': 'Alessandra Magistrato'}]\", 'abstract': '<p>Copper is a trace element vital to many cellular functions. Yet its abnormal levels are toxic to cells, provoking a variety of severe diseases. The high affinity Copper Transporter 1 (CTR1), being the main in-cell copper (Cu(I)) entry route, tightly regulates its cellular uptake via a still elusive mechanism. Here, all-atoms simulations unlock the molecular terms of Cu(I) transport in eukaryotes disclosing that the two Methionine triads, forming the selectivity filter, play an unprecedented dual role both enabling selective Cu(I) transport and regulating its uptake-rate thanks to an intimate coupling between the conformational plasticity of their bulky side chains and the number of bound Cu(I) ions. Namely, the Met residues act as a gate reducing the Cu(I) import-rate when two ions simultaneously bind to CTR1. This may represent an elegant autoregulatory mechanism through which CTR1 protects the cells from excessively high, and hence toxic, in-cell Cu(I) levels. Overall, these outcomes resolve fundamental questions in CTR1 biology and open new windows of opportunity to tackle diseases associated with an imbalanced copper uptake.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.21265569', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Evaluation of patients treated by telemedicine in the COVID-19 pandemic by a private clinic in Sao Paulo, Brazil: A non-randomized clinical trial preliminary study', 'authors': \"[{'name': 'Michelle Chechter'}, {'name': 'Gustavo Maximiliano Dutra da Silva'}, {'name': 'Thomas Gabriel Miklos'}, {'name': 'Marta Maria Kemp'}, {'name': 'Nilzio Antonio da Silva'}, {'name': 'Gabriel Lober Lober'}, {'name': 'Marcela Ferreira Tavares Zanut'}, {'name': 'Rute Alves Pereira e Costa'}, {'name': 'Aline Pinheiro dos Santos Cortada'}, {'name': 'Luciana de Nazare Lima da Cruz'}, {'name': 'Paulo Macio Porto de Melo'}, {'name': 'Bruno Campello de Souza'}, {'name': 'Morton Aaron Scheinberg'}]\", 'abstract': '<p>Introduction: As a result of the coronavirus disease 2019 (COVID-19) pandemic the year 2020 brought major changes on the delivery of health care and face to face physician patient communication was significantly reduced and the practice of remote telehealth care using computer technology is assuming a standard of care particularly with COVID-19 patients with attempts to reduce viral spread.\\nObjective: To describe the clinical practice experience using telemedicine towards COVID-19 and the respective clinical outcomes.\\nMethods: We performed a pilot open-label non-randomized controlled clinical trial. The patients were divided into four groups according severity of symptoms: (1) asymptomatic (2) mild symptoms (3) moderate symptoms and (4) severe symptoms and were followed up for five days counted from the beginning of the symptoms. A drug intervention was performed in group 3 for which the protocol followed as suggested by the International Pulmonology Societys consensus for adults with moderate symptoms: first day (attack phase) hydroxychloroquine sulfate 400 mg 12/12h second to fifth day (maintenance phase) 200 mg (half pill) 12/12h. The medication was associated with azithromycin 500mg once a day for five days. For children with moderate symptoms were used: hydroxychloroquine sulfate 6.5 mg/kg/dose every 12 hours in the first day and 3.25 mg/kg/dose every 12 hours from day 2 to 5. The therapeutic response was telemonitored. Group 4 patients were directly oriented to seek hospital care. During the use of the drugs, the patients were telemonitored daily.\\nResults: One hundred eighty-seven patients were seen with mean age of 37,6 years (about 15,6). The most frequent symptom was cough (57,6%) followed by malaise (60,3%) fever (41,1%) headache (56,0%) muscle pain (51,1%). Of all the patients that sought telemedicine service in our center 23% were asymptomatic despite contact with people with probable diagnostic of COVID-19 29,4% reported mild symptoms 43,9% moderate symptoms and 3,7% severe symptoms. It was possible to observe in patients treated their symptoms of COVID-19 (group 3) with hydroxychloroquine and azithromycin for five days presented statistically better improvement of the symptoms when compared to those that did not follow the protocol (p = 0.039). Three patients were hospitalized and discharged after recovery.\\nConclusions: Our study showed that patients with COVID-19 who had delivery of health care through telemedicine initiated in early stages of the disease presented satisfactory clinical response, reducing the need of face-to-face consultations and hospitalizations. Our results indicate that the use of telemedicine with diagnosis and drug treatment protocols is a safe and effective strategy to reduce overload of health services and the exposure of healthcare providers and the general population to infected patients in a pandemic situation.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.04.467338', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Direct addition of poly-lysine or poly-ethylenimine to the medium, a simple alternative to plate pre-coating', 'authors': \"[{'name': 'Alexander Faussner'}, {'name': 'Matthias Manfred Deininger'}, {'name': 'Chrisitan Weber'}, {'name': 'Sabine Steffens'}]\", 'abstract': '<p>For most cell culture experiments, it is indispensable that the cells are firmly anchored to the culture plates, tolerating several rinsing steps, and withstanding shear forces or temperature changes without detaching. For semi-adherent cells such as the very common HEK 293 cells, this could so far be obtained only by time-consuming plate pre-coating with cationic polymer solutions. We report here, that i) pre-coating with the cheaper poly-ethylenimine (PEI) works as well as the commonly used poly-D-lysine (PDL), but more importantly and novel ii) that simple direct addition of either PEI (1.5 µg/ml) or PDL (2 µg/ml) to the cell culture medium results in strongly anchored HEK 293 cells, indistinguishable from ones seeded on pre-coated plates. Therefore, the replacement of plate pre-coating by direct addition of either PEI or PDL gives comparable excellent results, but is highly labour-, time-, and cost-efficient. Interestingly, additional experiments in this context showed that strong cell attachment requires only cationic polymers but not fetal calf serum added to the medium. Fetal calf serum is, however, of course required for further maintenance and growth of the cells.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466989', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Validating the wearable MUSE headset for EEG spectral analysis and Frontal Alpha Asymmetry', 'authors': \"[{'name': 'Cedric Cannard'}, {'name': 'Helane Wahbeh'}, {'name': 'Arnaud Delorme'}]\", 'abstract': '<p>EEG power spectral density (PSD), the individual alpha frequency (IAF) and the frontal alpha asymmetry (FAA) are all EEG spectral measures that have been widely used to evaluate cognitive and attentional processes in experimental and clinical settings, and that can be used for real-world applications (e.g., remote EEG monitoring, brain-computer interfaces, neurofeedback, neuromodulation, etc.). Potential applications remain limited by the high cost, low mobility, and long preparation times associated with high-density EEG recording systems. Low-density wearable systems address these issues and can increase access to larger and diversified samples. The present study tested whether a low-cost, 4-channel wearable EEG system (the MUSE) could be used to quickly measure continuous EEG data, yielding similar frequency components compared to research a grade EEG system (the 64-channel BIOSEMI Active Two). We compare the spectral measures from MUSE EEG data referenced to mastoids to those from BIOSEMI EEG data with two different references for validation. A minimal amount of data was deliberately collected to test the feasibility for real-world applications (EEG setup and data collection being completed in under 5 min). We show that the MUSE can be used to examine power spectral density (PSD) in all frequency bands, the individual alpha frequency (IAF; i.e., peak alpha frequency and alpha center of gravity), and frontal alpha asymmetry. Furthermore, we observed satisfying internal consistency reliability in alpha power and asymmetry measures recorded with the MUSE. Estimating asymmetry on PAF and CoG frequencies did not yield significant advantages relative to the traditional method (whole alpha band). These findings should advance human neurophysiological monitoring using wearable neurotechnologies in large participant samples and increase the feasibility of their implementation in real-world settings.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.10.31.466690', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'A prior exposure to Serratia marcescens or xenobiotics primes Drosophila enterocytes against a recurring cytoplasmic purge', 'authors': \"[{'name': 'Simone Terreri'}, {'name': 'Bechara Sina-Rahme'}, {'name': 'Inês Pais'}, {'name': 'Catherine Socha'}, {'name': 'Matthieu Lestradet'}, {'name': 'Miriam Yamba'}, {'name': 'Stefanie Schirmeier'}, {'name': 'Kwang-Zin Lee'}, {'name': 'Dominique Ferrandon'}]\", 'abstract': '<p>The cytoplasmic extrusion of enterocytes is a fast response to an exposure to pore-forming toxin (PFT)-producing bacteria whereby their apical cytoplasm is extruded into the intestinal lumen. As a result of this purge, the intestinal epithelium becomes thin prior to a subsequent recovery. We report here that the ingestion of ethanol or caffeine induces a similar response, which suggests that a common purging process is triggered by bacterial toxins and abiotic toxicants. We also delineate an additional mechanism that is initiated by these stimuli that we refer to as priming. The initial exposure of the intestinal epithelium to either PFT or xenobiotics protects enterocytes against a further round of purging upon a second bacterial infection. Priming prevents the epithelium from being persistently thin in the context of chronic intestinal infections. We have identified the upper part of the p38b MAPK pathway as well as the homeobox-containing transcription factors E5/EMS as being required for priming and not for the regrowth of enterocytes after the cytoplasmic purge. Unexpectedly, the priming process appears to function cell-nonautonomously. Our findings suggest that the cytoplasmic purge extrusion has been selected because it constitutes a fast reaction to accidental exposure to bacterial toxins or toxicants.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.10.31.21265714', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Azithromycin consumption during the COVID-19 pandemic in Croatia, 2020', 'authors': \"[{'name': 'Nikolina Bogdanić'}, {'name': 'Loris Močibob'}, {'name': 'Toni Vidović'}, {'name': 'Ana Soldo'}, {'name': 'Josip Begovac'}]\", 'abstract': '<sec><title>Background</title><p>During the initial phase of the COVID-19 pandemic, there was great enthusiasm for the use of azithromycin with or without hydroxychloroquine.</p></sec><sec><title>Objectives</title><p>We analyzed azithromycin consumption in Croatia in 2020 and compared this to the period 2017–2019.</p></sec><sec><title>Methods</title><p>Azithromycin consumption was evaluated using the IQVIA Adriatic d.o.o. database which collects data on azithromycin distribution from wholesale pharmacies to hospital and non-hospital pharmacies in Croatia. We analyzed data for the period from January 2017 to December 2020. Azithromycin distribution was measured as days of therapy (DOT) and reported as per 1000 inhabitants or per 1000 inhabitant-days.</p></sec><sec><title>Results</title><p>In the period 2017–2020, total azithromycin DOT in Croatia increased in 2017, 2018, 2019, and 2020 (1.76, 1.91, 1.91 and 2.01/1000 inhabitant-days, respectively). Non-hospital pharmacies received 2.18 times and hospital pharmacies 4.39 times more DOT units/1000 inhabitants of azithromycin in March 2020 compared to the average distribution rate in March 2017–2019. During the peak of the COVID-19 epidemic (November and December 2020) azithromycin distribution increased considerably in hospital (3.62 and 3.19 times, respectively) and non-hospital pharmacies (1.93 and 1.84 times, respectively) compared to the average consumption in the same months in 2017–2019.</p></sec><sec><title>Conclusions</title><p>Our data showed increased azithromycin distribution in the period 2017 – 2020 which indicates azithromycin overuse. Preliminary information on COVID-19 treatments with a desire to offer and try what is available even in the absence of strong scientific evidence may have influenced practices of antimicrobial prescriptions.</p></sec>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.21265744', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': False, 'title': 'Metabolome-defined obesity and the risk of future diabetes and mortality', 'authors': \"[{'name': 'Filip Ottosson'}, {'name': 'Einar Smith'}, {'name': 'Ulrika Ericson'}, {'name': 'Salvatore Di Somma'}, {'name': 'Paola Antonini'}, {'name': 'Peter M Nilsson'}, {'name': 'Céline Fernandez'}, {'name': 'Olle Melander'}]\", 'abstract': '<sec><title>Background</title><p>Obesity is a key risk factor for type 2 diabetes, however, up to 20% of patients are normal weight. Our aim was to identify metabolite patterns reproducibly predictive of BMI, and subsequently to test if lean individuals who carry an obese metabolome are at hidden high risk of obesity related diseases, such as diabetes.</p></sec><sec><title>Methods</title><p>We measured 109 metabolites in fasted plasma samples of 7663 individuals from two Swedish and one Italian population-based cohort. Ridge regression models were used to predict BMI using the plasma metabolites. Individuals with a predicted BMI either more than 5 kg/m<sup>2</sup> higher (overestimated) or lower (underestimated) than their actual BMI were characterized as outliers and further investigated for obesity related risk factors and future risk of diabetes and mortality.</p></sec><sec><title>Results</title><p>The plasma metabolome could predict BMI in all cohorts (r<sup>2</sup> = 0.48, 0.26 and 0.19). The overestimated group had a BMI similar to individuals correctly predicted as normal weight, similar waist circumference, were not more likely to change weight over time but had a 2 times higher risk of future diabetes and an 80 % increased risk of all-cause mortality. These associations remained after adjustments for obesity-related risk factors and lifestyle parameters.</p></sec><sec><title>Conclusions</title><p>We found that lean individuals with an obese metabolome, have an increased risk for diabetes and all-cause mortality compared to lean individuals with a healthy metabolome. Metabolomics may be used to identify hidden high-risk individuals, in order to initiate lifestyle and pharmacological interventions.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.21265791', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Cohort study of Covid-19 vaccine effectiveness among healthcare workers in Finland, December 2020 - October 2021', 'authors': \"[{'name': 'Eero Poukka'}, {'name': 'Ulrike Baum'}, {'name': 'Arto A Palmu'}, {'name': 'Toni O Lehtonen'}, {'name': 'Heini Salo'}, {'name': 'Hanna Nohynek'}, {'name': 'Tuija Leino'}]\", 'abstract': '<p>Recently, Covid-19 vaccine effectiveness has decreased especially against mild disease due to emergence of the Delta variant and waning protection. In this register-based study among healthcare workers in Finland, the vaccine effectiveness of two-dose mRNA vaccine series against SARS-CoV-2 infection decreased from 82% (95% CI 79-85%) 14-90 days after vaccination to 53% (43-62%) after 6 months. Similar trend was observed for other series. Waning was not observed against Covid-19 hospitalization. These results facilitate decision-making of booster doses for healthcare workers.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467007', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Increasing the throughput of sensitive proteomics by plexDIA', 'authors': \"[{'name': 'Jason Derks'}, {'name': 'Andrew Leduc'}, {'name': 'R. Gray Huffman'}, {'name': 'Harrison Specht'}, {'name': 'Markus Ralser'}, {'name': 'Vadim Demichev'}, {'name': 'Nikolai Slavov'}]\", 'abstract': '<p>Current mass-spectrometry methods enable high-throughput proteomics of large sample amounts, but proteomics of low sample amounts remains limited in depth and throughput. We aimed to increase the throughput of high-sensitivity proteomics while achieving high proteome coverage and quantitative accuracy. We developed a general experimental and computational framework, plexDIA, for simultaneously multiplexing the analysis of both peptides and samples. Multiplexed analysis with plexDIA increases throughput multiplicatively with the number of labels without reducing proteome coverage or quantitative accuracy. Specifically, plexDIA using 3-plex nonisobaric mass tags enables quantifying 3-fold more protein ratios among nanogram-level samples. Using 1 hour active gradients and first-generation Q Exactive, plexDIA quantified about 8,000 proteins in each sample of labeled 3-plex sets. Furthermore, plexDIA increases the consistency of protein quantification, resulting in over 2-fold reduction of missing data across samples. We applied plexDIA to quantify proteome dynamics during the cell division cycle in cells isolated based on their DNA content. The high sensitivity and accuracy of plexDIA detected many classical cell cycle proteins and discovered new ones. These results establish a general framework for increasing the throughput of highly sensitive and quantitative protein analysis.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467105', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Transcription factor induction of vascular blood stem cell niches in vivo', 'authors': \"[{'name': 'Elliott J. Hagedorn'}, {'name': 'Julie R. Perlin'}, {'name': 'Rebecca J. Freeman'}, {'name': 'Samuel J. Wattrus'}, {'name': 'Tianxiao Han'}, {'name': 'Clara Mao'}, {'name': 'Ji Wook Kim'}, {'name': 'Inés Fernández-Maestre'}, {'name': 'Madeleine L. Daily'}, {'name': 'Christopher D’Amato'}, {'name': 'Michael J. Fairchild'}, {'name': 'Raquel Riquelme'}, {'name': 'Brian Li'}, {'name': 'Dana A.V.E. Ragoonanan'}, {'name': 'Khaliun Enkhbayar'}, {'name': 'Emily L. Henault'}, {'name': 'Helen G. Wang'}, {'name': 'Shelby E. Redfield'}, {'name': 'Samantha H. Collins'}, {'name': 'Asher Lichtig'}, {'name': 'Song Yang'}, {'name': 'Yi Zhou'}, {'name': 'Balvir Kunar'}, {'name': 'Jesus Maria Gomez-Salinero'}, {'name': 'Thanh T. Dinh'}, {'name': 'Junliang Pan'}, {'name': 'Karoline Holler'}, {'name': 'Henry A. Feldman'}, {'name': 'Eugene C. Butcher'}, {'name': 'Alexander van Oudenaarden'}, {'name': 'Shahin Rafii'}, {'name': 'J. Philipp Junker'}, {'name': 'Leonard I. Zon'}]\", 'abstract': '<p>The hematopoietic niche is a supportive microenvironment comprised of distinct cell types, including specialized vascular endothelial cells that directly interact with hematopoietic stem and progenitor cells (HSPCs). The molecular factors that specify niche endothelial cells and orchestrate HSPC homeostasis remain largely unknown. Using multi-dimensional gene expression and chromatin accessibility analyses, we define a conserved gene expression signature and <italic>cis</italic>-regulatory landscape unique to sinusoidal endothelial cells in the HSPC niche. Using enhancer mutagenesis and transcription factor overexpression, we elucidate a transcriptional code involving members of the Ets, Sox and Nuclear Hormone Receptor families that is sufficient to induce ectopic niche endothelial cells that associate with mesenchymal stromal cells and support the recruitment, maintenance and division of HSPCs <italic>in vivo</italic>. These studies set forth an approach for generating synthetic HSPC niches, <italic>in vitro</italic> or <italic>in vivo</italic>, and for effective therapies to modulate the endogenous niche.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467111', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Estimating the Designability of Protein Structures', 'authors': \"[{'name': 'Feng Pan'}, {'name': 'Yuan Zhang'}, {'name': 'Xiuwen Liu'}, {'name': 'Jinfeng Zhang'}]\", 'abstract': '<p>The total number of amino acid sequences that can fold to a target protein structure, known as \"designability\", is a fundamental property of proteins that contributes to their structure and function robustness. The highly designable structures always have higher thermodynamic stability, mutational stability, fast folding, regular secondary structures, and tertiary symmetries. Although it has been studied on lattice models for very short chains by exhaustive enumeration, it remains a challenge to estimate the designable quantitatively for real proteins. In this study, we designed a new deep neural network model that samples protein sequences given a backbone structure using sequential Monte Carlo method. The sampled sequences with proper weights were used to estimate the designability of several real proteins. The designed sequences were also tested using the latest AlphaFold2 and RoseTTAFold to confirm their foldabilities. We report this as the first study to estimate the designability of real proteins.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467116', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Detection of the HIV-1 accessory proteins Nef and Vpu by flow cytometry represents a new tool to study their functional interplay within a single infected CD4+ T cell', 'authors': \"[{'name': 'Jérémie Prévost'}, {'name': 'Jonathan Richard'}, {'name': 'Romain Gasser'}, {'name': 'Halima Medjahed'}, {'name': 'Frank Kirchhoff'}, {'name': 'Beatrice H. Hahn'}, {'name': 'John C. Kappes'}, {'name': 'Christina Ochsenbauer'}, {'name': 'Ralf Duerr'}, {'name': 'Andrés Finzi'}]\", 'abstract': '<p>The HIV-1 Nef and Vpu accessory proteins are known to protect infected cells from antibody-dependent cellular cytotoxicity (ADCC) responses by limiting exposure of CD4-induced (CD4i) envelope (Env) epitopes at the cell surface. Although both proteins target the host receptor CD4 for degradation, the extent of their functional redundancy is unknown. Here, we developed an intracellular staining technique that permits the intracellular detection of both Nef and Vpu in primary CD4+ T cells by flow cytometry. Using this method, we show that the combined expression of Nef and Vpu predicts the susceptibility of HIV-1-infected primary CD4+ T cells to ADCC by HIV+ plasma. We also show that Vpu cannot compensate for the absence of Nef, thus providing an explanation for why some infectious molecular clones that carry a LucR reporter gene upstream of Nef render infected cells more susceptible to ADCC responses. Our method thus represents a new tool to dissect the biological activity of Nef and Vpu in the context of other host and viral proteins within single infected CD4+ T cells.</p><sec><title>IMPORTANCE</title><p>HIV-1 Nef and Vpu exert several biological functions that are important for viral immune evasion, release and replication. Here, we developed a new method allowing simultaneous detection of these accessory proteins in their native form together with some of their cellular substrates. This allowed us to show that Vpu cannot compensate the lack of a functional Nef, which has implication for studies that use Nef-defective viruses to study ADCC responses.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.467127', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Adar1 deletion causes degeneration of exocrine pancreas via Mavs-dependent interferon signaling', 'authors': \"[{'name': 'Dhwani N. Rupani'}, {'name': 'Robert W. Cowan'}, {'name': 'Fredrik I. Thege'}, {'name': 'Vidhi Chandra'}, {'name': 'Sonja M. Wörmann'}, {'name': 'Hajar Rajaei'}, {'name': 'Prerna Malaney'}, {'name': 'Olivereen Le Roux'}, {'name': 'Sara L. Manning'}, {'name': 'Jack Hashem'}, {'name': 'Jennifer Bailey-Lundberg'}, {'name': 'Florencia McAllister'}, {'name': 'Andrew D. Rhim'}]\", 'abstract': '<p>Adenosine deaminase acting on RNA 1 (ADAR1) is an RNA-binding protein that deaminates adenosine(A) to inosine(I). A-to-I editing alters post-transcriptional RNA processing making ADAR1 a critical regulator of gene expression. Consequently, Adar1 has been implicated in organogenesis. To determine the role of Adar1 in pancreatic development and homeostasis, we specifically deleted <italic>Adar1</italic> from the murine pancreas (<italic>Ptf1a<sup>Cre/+</sup>; Adar1<sup>Fl/Fl</sup></italic>). The resulting mice had stunted growth likely due to malabsorption associated with exocrine pancreas insufficiency. Analyses of pancreases revealed ductal expansion, heightened interferon-stimulated gene expression and an increased influx of immune cells. In addition, we observed an increased prevalence of CD4<sup>+</sup> T and natural killer cells in their splenic tissue. These results indicate an association between loss of pancreatic Adar1 with dysregulation of systemic immunity. Concurrent deletion of <italic>Adar1</italic> and <italic>Mavs</italic>, a signaling protein implicated in the innate immune pathway rescued the degenerative phenotype and resulted in normal pancreatic development. Taken together, our work suggests that the primary function of Adar1 in the pancreas is to prevent aberrant activation of the Mavs-mediated innate immune pathway, thereby maintaining pancreatic homeostasis.</p><sec><title>Summary statement</title><p>This work defines the role of Adar1 in pancreatic development and homeostasis.</p></sec>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467147', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Chigno/CG11180 and SUMO are Chinmo-Interacting Proteins with a Role in Drosophila Testes Stem Cells', 'authors': \"[{'name': 'Leanna Rinehart'}, {'name': 'Wendy E. Stewart'}, {'name': 'Natalie Luffman'}, {'name': 'Matthew Wawersik'}, {'name': 'Oliver Kerscher'}]\", 'abstract': '<p>Maintenance of sexual identity on the cellular level ensures the proper function of sexually dimorphic genes expressed in the brain and gonads. Disruption of genes that regulate sex maintenance alters the cellular structure of these tissues and leads to infertility and diseases, such as diabetes, obesity, and gonadal cancers. Sex maintenance in the testis of <italic>Drosophila melanogaster</italic> depends on the previously identified gene <italic>chinmo</italic> (Chronologically inappropriate morphogenesis). Chinmo’s effect on testis differentiation has been investigated in detail, but there is still much to be elucidated about its structure, function, and interactions with other proteins. Using a two-hybrid screen, we find that Chinmo interacts with itself, the small ubiquitin-like modifier SUMO, the novel protein CG11180, and four other proteins (CG4318, Ova (Ovaries absent), Taf3 (TBP-associated factor 3), and CG18269). Since both Chinmo and CG11180 contain sumoylation sites and SUMO-interacting motifs (SIMs), we analyzed their interaction in more detail. Using site-directed mutagenesis of a unique SIM in CG11180, we demonstrate that Chinmo’s interaction with CG11180 is SUMOdependent. Furthermore, to assess the functional relevance of both SUMO and CG11180, we performed RNAi-mediated knockdown of both proteins in somatic cells of the <italic>Drosophila</italic> testis. Using this approach, we find that CG11180 and SUMO are required in somatic cells of adult testes, and that reduction of either protein causes formation of germ cell tumors. Overall, our work indicates that SUMO functionally links Chinmo and CG11180 in somatic cells of the adult <italic>Drosophila</italic> testis. Consistent with the CG11180 knockdown phenotype in male testes, and to underscore its connection to Chinmo, we propose the name Childless Gambino (Chigno) for CG11180.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467186', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Stenoparib, an inhibitor of cellular poly (ADP-ribose) polymerases (PARPs), blocks in vitro replication of SARS-CoV-2 variants', 'authors': \"[{'name': 'Katherine E Zarn'}, {'name': 'Sierra A Jaramillo'}, {'name': 'Anthony R Zapata'}, {'name': 'Nathan E Stone'}, {'name': 'Ashley N Jones'}, {'name': 'Haley E Nunnally'}, {'name': 'Erik W Settles'}, {'name': 'Ken Ng'}, {'name': 'Paul S Keim'}, {'name': 'Steen Knudsen'}, {'name': 'Patricia M Nuijten'}, {'name': 'Aloys SI Tijsma'}, {'name': 'Christopher T French'}]\", 'abstract': '<p>We recently published a preliminary assessment of the activity of a poly (ADP-ribose) polymerase (PARP) inhibitor, stenoparib, also known as 2X-121, which inhibits viral replication by affecting pathways of the host. Stenoparib is an inhibitor of mammalian poly (ADP-ribose) polymerases (PARPs). Here we show that stenoparib effectively inhibits additional SARS-CoV-2 variants, including an additional wild-type strain (Germany/BavPat1/2020), and the variants <italic>alpha</italic> (B.1.1.7), <italic>beta</italic> (B.1.351) and <italic>gamma</italic> (P.1) <italic>in vitro</italic>, with 50% effective concentration (EC<sub>50</sub>) estimates of 4.1 μM, 8.5 μM, 24.2 μM and 13.6 μM, respectively. A second study focusing on a combination of 10 μM stenoparib and 0.5 µM remdesivir resulted in over 90% inhibition of the <italic>alpha</italic> (B.1.1.7) variant, which is substantially greater than what was achieved with stenoparib or remdesivir alone at these concentrations.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.466494', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Towards a Theory of Evolution as Multilevel Learning', 'authors': \"[{'name': 'Vitaly Vanchurin'}, {'name': 'Yuri I. Wolf'}, {'name': 'Mikhail Katsnelson'}, {'name': 'Eugene V. Koonin'}]\", 'abstract': '<p>We apply the theory of learning to physically renormalizable systems in an attempt to develop a theory of biological evolution, including the origin of life, as multilevel learning. We formulate seven fundamental principles of evolution that appear to be necessary and sufficient to render a universe observable and show that they entail the major features of biological evolution, including replication and natural selection. These principles also follow naturally from the theory of learning. We formulate the theory of evolution using the mathematical framework of neural networks, which provides for detailed analysis of evolutionary phenomena. To demonstrate the potential of the proposed theoretical framework, we derive a generalized version of the Central Dogma of molecular biology by analyzing the flow of information during learning (back-propagation) and predicting (forward-propagation) the environment by evolving organisms. The more complex evolutionary phenomena, such as major transitions in evolution, in particular, the origin of life, have to be analyzed in the thermodynamic limit, which is described in detail in the accompanying paper.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.04.467244', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Mapping single-cell transcriptomes to copy number evolutionary trees', 'authors': \"[{'name': 'Pedro F Ferreira'}, {'name': 'Jack Kuipers'}, {'name': 'Niko Beerenwinkel'}]\", 'abstract': '<p>Cancer arises and evolves by the accumulation of somatic mutations that provide a selective advantage. The interplay of mutations and their functional consequences shape the evolutionary dynamics of tumors and contribute to different clinical outcomes. In the absence of scalable methods to jointly assay genomic and transcriptomic profiles of the same individual cell, the two data modalities are usually measured separately and need to be integrated computationally. Here, we introduce SCATrEx, a statistical model to map single-cell gene expression data onto the evolutionary history of copy number alterations of the tumor. SCATrEx jointly assigns cancer cells assayed with scRNA-seq to copy number profiles arranged in a copy number aberration tree and augments the tree with clone-specific clusters. Our simulations show that SCATrEx improves over both state-of-the-art unsupervised clustering methods and cell-to-clone assignment methods. In an application to real data, we observe that SCATrEx finds inter-clone and intra-clone gene expression heterogeneity not detectable using other integration methods. SCATrEx will allow for a better understanding of tumor evolution by jointly analysing the genomic and transcriptomic changes that drive it.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.02.466980', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Aryl hydrocarbon receptor (AhR) activation by 2,3,7,8-tetrachlorodibenzo-p-dioxin (TCDD) dose-dependently shifts the gut microbiome consistent with the progression of steatosis to steatohepatitis with fibrosis', 'authors': \"[{'name': 'Russell R. Fling'}, {'name': 'Tim R. Zacharewski'}]\", 'abstract': '<p>Gut dysbiosis with disrupted enterohepatic bile acid metabolism is commonly associated with non-alcoholic fatty liver disease (NAFLD) and recapitulated in a NAFLD-phenotype elicited by 2,3,7,8-tetrachlorodibenzo-p-dioxin (TCDD) in mice. TCDD induces hepatic fat accumulation and increases levels of secondary bile acids including taurolithocholic acid and deoxycholic acid, microbial modified bile acids involved in host bile acid regulation signaling pathways. To investigate the effects of TCDD on the gut microbiota, cecum contents of male C57BL/6 mice orally gavaged with sesame oil vehicle or 0.3, 3, or 30 µg/kg TCDD were examined using shotgun metagenomic sequencing. Taxonomic analysis identified dose-dependent increases in Lactobacillus species (i.e., <italic>Lactobacillus reuteri</italic>). Increased species were also associated with dose-dependent increases in bile salt hydrolase sequences, responsible for deconjugation reactions in secondary bile acid metabolism. Increased <italic>L. reuteri</italic> levels were further associated with mevalonate-dependent isopentenyl diphosphate (IPP) biosynthesis and menaquinone biosynthesis genes. Analysis of gut microbiomes from cirrhosis patients identified increased abundance of these pathways as identified in the mouse cecum metagenomic analysis. These results extend the association of lactobacilli with the AhR/intestinal axis in NAFLD progression and highlight the similarities between TCDD-elicited phenotypes in mice to human NAFLD.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.04.467225', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Pathology', 'covid': False, 'title': 'Spread of Xanthomonas vasicola pv. musacearum within banana mats: implications for Xanthomonas wilt management', 'authors': \"[{'name': 'Walter Ocimati'}, {'name': 'Anthony F Tazuba'}, {'name': 'Guy Blomme'}]\", 'abstract': '<p>Xanthomonas wilt (XW) of banana caused by Xanthomonas vasicola pv. musacearum (Xvm) does not spread to all plants physically interconnected through the rhizome when one or a few are diseased. However, the factors behind this incomplete systemic spread of Xvm are not fully known yet could inform XW management. This study explored the effect of Xvm inoculum amounts, number and size of suckers, sucker positioning on mother plant corms and other mother plant corm attributes on sucker colonization. A shorter (p &lt;0.05) incubation period (17.9 vs 21.1 days) and higher (p&lt;.001) cumulative number of symptomatic leaves (5.2 vs 1.6 leaves) was observed when all (high inoculum) compared to two leaves (low inoculum) were inoculated. Xvm was recovered in corms at 29 days post inoculation (dpi) in both treatments with no differences (p &gt;0.05) in proportions of corms with Xvm between the treatments. However, Xvm was recovered earlier and at a higher frequency in suckers when all leaves were inoculated. Lower Xvm recoveries occurred in the lower corm sections to which most suckers were attached relative to the middle and upper corm sections. Xvm incidence in corms increased with the number of attached maiden suckers, and the dpi while it declined with increasing mother plant and corm height. Thus, Xvm spread within mats is influenced by the amount of inoculum and the physiological stage (e.g., height) of the plant and attached suckers. The position of suckers, predominantly at the bottom of corms also protects them from infection. Measures that reduce Xvm inoculum build-up in mats are thus crucial for minimizing within mat XW spread.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.467014', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Age-dependent increased sag current in human pyramidal neurons dampens baseline cortical activity', 'authors': \"[{'name': 'Alexandre Guet-McCreight'}, {'name': 'Homeira Moradi Chameh'}, {'name': 'Sara Mahallati'}, {'name': 'Margaret Wishart'}, {'name': 'Shreejoy J Tripathy'}, {'name': 'Taufik A Valiante'}, {'name': 'Etay Hay'}]\", 'abstract': '<p>Aging involves various neurobiological changes, although their effect on brain function in humans remains poorly understood. The growing availability of human neuronal and circuit data provides opportunities for uncovering age-dependent changes of brain networks and for constraining models to predict consequences on brain activity. Here we found increased sag current in human layer 5 pyramidal neurons from older subjects, and captured this effect in biophysical models of younger and older pyramidal neurons. We used these models to simulate detailed layer 5 microcircuits and found lower baseline firing in older pyramidal neuron microcircuits, with minimal effect on response. We then validated the predicted reduced baseline firing using extracellular multi-electrode recordings from human brain slices of different ages. Our results thus report changes in human pyramidal neuron input integration properties that can sufficiently account for age-dependent decreases in cortical resting state activity and may underpin a clinical relevance in aging.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467045', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Trim41 is essential for preventing X chromosome chaotic synapsis in male mice', 'authors': \"[{'name': 'Seiya Oura'}, {'name': 'Toshiaki Hino'}, {'name': 'Takashi Satoh'}, {'name': 'Taichi Noda'}, {'name': 'Takayuki Koyano'}, {'name': 'Ayako Isotani'}, {'name': 'Makoto Matsuyama'}, {'name': 'Shizuo Akira'}, {'name': 'Kei-ichiro Ishiguro'}, {'name': 'Masahito Ikawa'}]\", 'abstract': '<p>Meiosis is a hallmark event in germ cell development that accompanies sequential chromosome events executed by numerous molecules. Therefore, characterization of these factors is one of the best strategies to clarify the mechanism of meiosis. Here, we report tripartite motif-containing 41 (TRIM41), a ubiquitin ligase E3, as an essential factor for proper meiotic progression and fertility in male mice. <italic>Trim41</italic> KO spermatocytes exhibited synaptonemal complex protein 3 (SYCP3) overloading, especially on the X chromosome, showing extensive self-synapsis of X chromosome and non-homologous synapsis between the X chromosome and autosomes. Furthermore, the mutant mice lacking the RING domain of TRIM41, required for the ubiquitin ligase E3 activity, phenocopied <italic>Trim41</italic> KO mice. We then examined the behavior of mutant TRIM41 (ΔRING-TRIM41) and found that ΔRING-TRIM41 accumulated on the chromosome axes with overloaded SYCP3. This result showed that TRIM41 exerts the function on the chromosome axes. In summary, our study revealed that <italic>Trim41</italic> is essential for preventing SYCP3 overloading and chaotic synapsis of the X chromosome, suggesting a TRIM41-mediated mechanism for regulating unsyapsed axes during male meiotic progression.</p><sec><title>Summary statement</title><p><italic>Trim41</italic>-disruption caused abnormal synapsis configuration of the X chromosome and complete infertility in male mice. Thus, TRIM41 prevents the sex chromosome from chaotic synapsis.</p></sec>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467051', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Restricting a single amino acid cross-protects D. melanogaster from nicotine poisoning through mTORC1 and GCN2 signalling', 'authors': \"[{'name': 'Tahlia L. Fulton'}, {'name': 'Christen K. Mirth'}, {'name': 'Matthew D. W. Piper'}]\", 'abstract': '<p>Dietary interventions that restrict protein intake have repeatedly been shown to offer beneficial health outcomes to the consumer. Benefits such as increased stress tolerance can be observed in response to restricting individual amino acids, thus mimicking dietary protein restriction. Here, we sought to further understand the relationship between dietary amino acids and stress tolerance using <italic>Drosophila melanogaster</italic>. Utilising a chemically defined medium for <italic>Drosophila</italic>, we found that transiently restricting adult flies of a single essential amino acid generally protects against a lethal dose of the naturally occurring insecticide, nicotine. This protection was conferred during the pre-treatment window, was specific for individual amino acids and depended on the identity of the focal amino acid, as well as the duration and intensity of its restriction. For instance, complete isoleucine deprivation for 7 days maximised its protective effect - increasing survival during nicotine exposure by 100%. However, a dose of 25% threonine was required to maximise its protective effect (53% enhanced survival). To understand the molecular basis of these effects, we modified the signalling of two cellular sensors of amino acids, GCN2 (General control non-derepressible) and mTORC1 (mechanistic Target of Rapamycin Complex 1) in combination with amino acid restriction. We found that GCN2 was necessary for diets to protect against nicotine, whereas suppression of mTORC1 was sufficient to induce nicotine resistance. This finding implies that amino acid restriction acts via amino acid signalling to cross-protect against seemingly unrelated stressors. Altogether, our study offers new insights into the physiological responses to restriction of individual amino acids that confer stress tolerance. This has broad potential for application in animal and human health.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.10.30.21265430', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Automated Interpretable Discovery of Heterogeneous Treatment Effectiveness: A Covid-19 Case Study', 'authors': \"[{'name': 'Benjamin J. Lengerich'}, {'name': 'Mark E. Nunally'}, {'name': 'Yin Aphinyanaphongs'}, {'name': 'Rich Caruana'}]\", 'abstract': '<p>Testing multiple treatments for heterogeneous (varying) effectiveness with respect to many underlying risk factors requires many pairwise tests; we would like to instead automatically discover and visualize patient archetypes and predictors of treatment effectiveness using multitask machine learning. In this paper, we present a method to estimate these heterogeneous treatment effects with an interpretable hierarchical framework that uses additive models to visualize expected treatment benefits as a function of patient factors (identifying personalized treatment benefits) and concurrent treatments (identifying combinatorial treatment benefits). This method achieves state-of-the-art predictive power for Covid-19 in-hospital mortality and interpretable identification of heterogeneous treatment benefits. We first validate this method on the large public MIMIC-IV dataset of ICU patients to test recovery of heterogeneous treatment effects. Next we apply this method to a proprietary dataset of over 3000 patients hospitalized for Covid-19, and find evidence of heterogeneous treatment effectiveness predicted largely by indicators of inflammation and throm-bosis risk: patients with few indicators of thrombosis risk benefit most from treatments against inflammation, while patients with few indicators of inflammation risk benefit most from treatments against thrombosis. This approach provides an automated methodology to discover heterogeneous and individualized effectiveness of treatments.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.10.29.466479', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Cycles of transcription and local translation consolidate Arc protein at dendritic sites', 'authors': \"[{'name': 'Sulagna Das'}, {'name': 'Pablo J. Lituma'}, {'name': 'Pablo E. Castillo'}, {'name': 'Robert H. Singer'}]\", 'abstract': '<p>Maintenance of long-term memory requires transcription and translation of activity-regulated genes. Many of these are immediate early genes (IEGs) with short-lived mRNAs and proteins, often decaying rapidly after stimulation. It remains unknown how an IEG with rapid mRNA and protein turnover can impact long-lasting changes at the synapses. Here, we imaged the long-term transcription and translation dynamics of an IEG, <italic>Arc</italic>, important for memory consolidation, after stimulation in individual neurons. We demonstrated that the gene underwent reactivation in the same neuron without an additional stimulus, often at the same allele. The cycling of <italic>Arc</italic> transcription required protein synthesis. Indeed, phases of <italic>Arc</italic> translation were coordinated with the transcription cycles. <italic>Arc</italic> mRNAs from later cycles preferentially localized at dendritic sites marked by previous Arc protein, thereby creating hubs of local protein enrichment, potentially maintained by cycles of dendritic translation. These results identify novel characteristics of an IEG and provide insights into how unstable proteins may be sustained over the long-time scale of memory consolidation.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.467100', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Microniches in biofilm depth are hot-spots for antibiotic resistance acquisition in response to in situ stress', 'authors': \"[{'name': 'Linda Tlili'}, {'name': 'Marie-Cécile Ploy'}, {'name': 'Sandra Da Re'}]\", 'abstract': '<p>Class 1 integrons play a major role in antibiotic resistance dissemination among Gram-negative bacteria. They are genetic platforms able to capture, exchange and express antibiotic resistance gene cassettes. The integron integrase, whose expression is regulated by the bacterial SOS response, is the key element of the integron catalyzing insertion/excision/shuffling of gene cassettes. We previously demonstrated that the basal level of integrase expression and in consequence, its activity, is increased <italic>via</italic> the starvation-induced stringent response in the biofilm population. However, biofilms are heterogeneous environments where bacteria are under various physiological states. Here we thus analyzed at the bacterial level, the SOS response and integrase expression within the biofilm, using confocal microscopy and flow cytometry. We showed that in the absence of exogenous stress, only a small number of bacteria (~ 1%) located in the depth of the biofilm induce the SOS-response leading to a high level of integrase expression, through both a stringent response-dependent and -independent manner. Our results thus indicate that few bacteria located in microniches of the biofilm depth undergo sufficient endogenous stress to promote the acquisition of antibiotic resistance, forming a reservoir of bacteria ready to rapidly resist antibiotic treatments.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466436', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evolution of warfare by resource raiding favors polymorphism in belligerence and bravery', 'authors': \"[{'name': 'Charles Mullon'}, {'name': 'Laurent Lehmann'}]\", 'abstract': '<p>From protists to primates, intergroup aggression and warfare over resources has been observed in several taxa whose populations typically consist of groups connected by limited genetic mixing. Here, we model the co-evolution between four traits relevant to this setting: (i) investment into common-pool resource production within groups (“helping”); (ii) proclivity to raid other groups to appropriate their resources (“bel-ligerence”); and investments into (iii) defense and (iv) offense of group contests (“defensive and offensive bravery”). We show that when traits co-evolve, the population often experiences disruptive selection favouring two morphs: “Hawks”, who express high levels of both belligerence and offensive bravery; and “Doves”, who express neither. This social polymorphism involves further among-traits associations when the fitness costs of helping and bravery interact. In particular if helping is antagonistic with both forms of bravery, co-evolution leads to the coexistence of individuals that either: (i) do not participate into common-pool resource production but only in its defense and appropriation (“Scrounger Hawks”); or (ii) only invest into common pool resource production (“Producer Doves”). Provided groups are not randomly mixed, these findings are robust to several modelling assumptions. This suggests that inter-group aggression is a potent mechanism in favoring within-group social diversity and behavioural syndromes.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466879', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Biological Factors and Self-Perception of Stress Predict Human Freeze-Like Responses in the Context of Self-Defence Training and Personal Experience with Violence', 'authors': \"[{'name': 'Peter Lenart'}, {'name': 'Michal Vit'}, {'name': 'Klara Mareckova'}, {'name': 'Jan Novak'}, {'name': 'Filip Zlamal'}, {'name': 'Michal Mikl'}, {'name': 'Zdenko Reguli'}, {'name': 'Martin Bugala'}, {'name': 'Jitka Cihounkova'}, {'name': 'Pavel Precechtel'}, {'name': 'Vojtech Malcik'}, {'name': 'Tomas Vojtisek'}, {'name': 'Jan Kucera'}, {'name': 'Jana Fialova Kucerova'}, {'name': 'Veronika Hajnova'}, {'name': 'Marie Tomandlova'}, {'name': 'Radek Sip'}, {'name': 'Lucie Rackova'}, {'name': 'Marketa Grulichova'}, {'name': 'Josef Tomandl'}, {'name': 'Milan Brazdil'}, {'name': 'Julie Bienertova-Vasku'}]\", 'abstract': '<p>Many animals react to threatening stimuli such as a predator attacks by freezing. However, little experimental research investigated freeze response in humans. Here, we have employed practices commonly used in self-defence training to create two unique scenarios simulating armed physical attacks. Sixty healthy men volunteers divided into three groups of twenty (untrained, trained but unexperienced, trained and experienced) underwent these scenarios accompanied by measurement of biochemical, physiological, and psychological markers of stress. All participants also underwent an fMRI session during which they observed neutral and negative images from the International Affective Picture System (IAPS). Our results show that scenarios simulating physical attacks can induce a freeze-like response in men. In addition, we demonstrate that while electrodermal activity (EDA), subjective stress perception, and brain activity in fMRI predict freeze-like response in men, their effect on freeze-like response is entirely dependent on the level of training and experience of a given individual.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466885', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Operon formation by insertion sequence IS3 in Escherichia coli', 'authors': \"[{'name': 'Yuki Kanai'}, {'name': 'Saburo Tsuru'}, {'name': 'Chikara Furusawa'}]\", 'abstract': '<p>Operons are a hallmark of the genomic and regulatory architecture of prokaryotes. However, the mechanism by which two genes placed far apart gradually come close and form operons remains to be elucidated. Here, we propose a new model of the origin of operons: Mobile genetic elements called insertion sequences can facilitate the formation of operons by consecutive insertion-deletion-excision reactions. This mechanism barely leaves traces of insertion sequences and is difficult to detect in evolution in nature. We performed, to the best of our knowledge, the first experimental demonstration of operon formation, as a proof of concept. The insertion sequence IS<italic>3</italic> and the insertion sequence excision enhancer are genes found in a broad range of bacterial species. We introduced these genes into insertion sequence-less <italic>Escherichia coli</italic> and found that, supporting our hypothesis, the activity of the two genes altered the expression of genes surrounding IS<italic>3</italic>, closed a 2.7 kilobase pair gap between a pair of genes, and formed new operons. This study shows how insertion sequences can facilitate the rapid formation of operons through locally increasing the structural mutation rates and highlights how coevolution with mobile elements may shape the organization of prokaryotic genomes and gene regulation.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466809', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Yersinia pseudotuberculosis doxycycline tolerance strategies include modulating expression of genes involved in cell permeability and tRNA modifications', 'authors': \"[{'name': 'Hector S. Alvarez-Manzo'}, {'name': 'Robert K. Davidson'}, {'name': 'Jasper Van Cauwelaert de Wyels'}, {'name': 'Katherine L. Cotten'}, {'name': 'Benjamin Nguyen'}, {'name': 'Zeyu Zhu'}, {'name': 'Jon Anthony'}, {'name': 'Tim van Opijnen'}, {'name': 'Kimberly M. Davis'}]\", 'abstract': '<p>Antibiotic tolerance is typically associated with a phenotypic change within a bacterial population, resulting in a transient decrease in antibiotic susceptibility that can contribute to treatment failure and recurrent infections. Although tolerant cells may emerge prior to treatment, the stress of prolonged antibiotic exposure can also promote tolerance. Here, we sought to determine how <italic>Yersinia pseudotuberculosis</italic> responds to doxycycline exposure, to then verify if these gene expression changes could promote doxycycline tolerance in culture and in our mouse model of infection. Only four genes were differentially regulated in response to a physiologically-relevant dose of doxycycline: <italic>osmB</italic> and <italic>ompF</italic> were upregulated<italic>, tusB</italic> and <italic>cnfy</italic> were downregulated; differential expression also occurred during doxycycline treatment in the mouse. <italic>ompF, tusB</italic> and <italic>cnfy</italic> were also differentially regulated in response to chloramphenicol, indicating these could be general responses to ribosomal inhibition. <italic>cnfy</italic> has previously been associated with persistence and was not a major focus here. We found deletion of the OmpF porin resulted in increased antibiotic accumulation, suggesting expression may promote diffusion of doxycycline out of the cell, while OsmB lipoprotein had a minor impact on antibiotic permeability. Overexpression of <italic>tusB</italic> significantly impaired bacterial survival in culture and in the mouse, suggesting that tRNA modification by <italic>tusB</italic>, and the resulting impacts on translational machinery, may play an important role in promoting tolerance. We believe this is the first observation of bactericidal activity of doxycycline, which was revealed by reversing <italic>tusB</italic> downregulation.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.02.21265767', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Determinants of pre-vaccination antibody responses to SARS-CoV-2: a population-based longitudinal study (COVIDENCE UK)', 'authors': \"[{'name': 'Mohammad Talaei'}, {'name': 'Sian Faustini'}, {'name': 'Hayley Holt'}, {'name': 'David A. Jolliffe'}, {'name': 'Giulia Vivaldi'}, {'name': 'Matthew Greenig'}, {'name': 'Natalia Perdek'}, {'name': 'Sheena Maltby'}, {'name': 'Jane Symons'}, {'name': 'Gwyneth A Davies'}, {'name': 'Ronan A Lyons'}, {'name': 'Christopher J Griffiths'}, {'name': 'Frank Kee'}, {'name': 'Aziz Sheikh'}, {'name': 'Alex G Richter'}, {'name': 'Seif O Shaheen'}, {'name': 'Adrian R Martineau'}]\", 'abstract': '<sec><title>Background</title><p>Prospective population-based studies investigating multiple determinants of pre-vaccination antibody responses to SARS-CoV-2 are lacking.</p></sec><sec><title>Methods</title><p>We did a prospective population-based study in SARS-CoV-2 vaccine-naive UK adults between May 1 and Nov 2, 2020. Information on 88 potential risk factors was obtained through online questionnaires, and combined IgG/IgA/IgM responses to SARS-CoV-2 spike glycoprotein were determined in dried blood spots. We used logistic and linear regression to estimate adjusted odds ratios (aORs) and adjusted geometric mean ratios (aGMRs) for potential determinants of SARS-CoV-2 seropositivity (all participants) and antibody titres (seropositive participants only), respectively.</p></sec><sec><title>Results</title><p>1696 (15.2%) of 11,130 participants were seropositive. Factors independently associated with increased risk included frontline health/care occupation (aOR 1.86, 95% CI 1.49–2.33), international travel (1.22, 1.08–1.37), BMI &gt;30 <italic>vs</italic> &lt;25 kg/m<sup>2</sup> (1.22, 1.05–1.42), Asian/Asian British <italic>vs</italic> White ethnicity (1.65, 1.10–2.47), and alcohol consumption ≥15 <italic>vs</italic> 0 units/week (1.26, 1.06–1.49). Light physical exercise associated with decreased risk (0.80, 0.69–0.93, for ≥10 <italic>vs</italic> 0–4 h/week). Higher titres associated with frontline health/care occupation (aGMR 1.26, 95% CI 1.13–1.41), international travel (1.10, 1.04–1.16), BMI &gt;30 <italic>vs</italic> &lt;25 kg/m<sup>2</sup> (1.09, 1.01–1.17), and Asian/Asian British <italic>vs</italic> White ethnicity (1.23, 1.03–1.46); these associations were not substantially attenuated by adjustment for disease severity.</p></sec><sec><title>Conclusions</title><p>Higher alcohol consumption and reduced physical exercise represent new modifiable risk factors for SARS-CoV-2 infection. Recognised associations between Asian/Asian British ethnic origin and obesity and increased risk of SARS-CoV-2 seropositivity were independent of other sociodemographic, clinical, or behavioural factors investigated.</p></sec><sec><title>Funding</title><p>Barts Charity, Health Data Research UK.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.21265831', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Multiplex Solid-Phase RPA Coupled CRISPR-Based Visual Detection of SARS-CoV-2', 'authors': \"[{'name': 'Xiaochen Qin'}, {'name': 'Yuyuan Zhou'}, {'name': 'Ratul Paul'}, {'name': 'Yue Wu'}, {'name': 'Yaling Liu'}]\", 'abstract': '<p>COVID-19 has challenged the world’s public health and led to over 4.5 million deaths. A rapid, sensitive, and cost-effective point-of-care virus detection device is crucial to the control and surveillance of the contagious severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) pandemic. Here we demonstrate a solid phase isothermal recombinase polymerase amplification coupled CRISPR-based (spRPA-CRISPR) assay for on-chip multiplexed, sensitive and visual COVID-19 DNA detection. By targeting the SARS-CoV-2 structure protein encoded genomes, two specific genes were simultaneously detected with the control sample without cross-interaction with other sequences. The endpoint signal can be directly visualized for rapid detection of COVID-19. The amplified target sequences were immobilized on the one-pot device surface and detected using the mixed Cas12a-crRNA collateral cleavage of reporter released fluorescent signal when specific genes were recognized. The system was tested with samples of a broad range of concentrations (20 to 2×10<sup>5</sup> copies) and showed analytical sensitivity down to 20 copies per reaction. Furthermore, a low-cost LED UV flashlight (∼$12) was used to provide a visible SARS-CoV-2 detection signal of the spRPA-CRISPR assay which could be purchased online easily. Thus, our platform provides a sensitive and easy-to-read multiplexed gene detection method with the capacity to specifically identify low concentration genes. Similar CRISPR biosensor chips can support a broad range of applications such as HPV DNA detection, influenza SARS-CoV-2 multiplex detection, and other infectious disease testing assays.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>A commercially available UV flashlight excited CRISPR-Cas12a based visual sensor combined with solid-phase RPA</p></list-item><list-item><p>The developed assay could detect SARS-CoV-2 down to 20 copies with no signal crosswalk</p></list-item><list-item><p>This multiplexed on-chip virus detection system would provide great insight for detecting and differentiating SARS-CoV-2 from other viral infections with a single sample simultaneously.</p></list-item></list></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.07.467592', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Construction of a bioluminescence-based assay for bitter taste receptors (TAS2Rs)', 'authors': \"[{'name': 'Shi Min Tan'}, {'name': 'Wei-Guang Seetoh'}]\", 'abstract': '<p>In humans, a family of 25 bitter taste receptors (TAS2Rs) mediates bitter taste perception. A common approach to characterize bitter causative agents involves expressing TAS2Rs and the appropriate signal transducers in heterologous cell systems, and monitoring changes in the intracellular free calcium levels upon ligand exposure using a fluorescence-based modality. However, a fluorescence-based assay typically suffers from a low signal window, and is susceptible to interference by autofluorescence, therefore limiting its application to screening of plant or food extracts, which are likely to contain autofluorescent compounds. Here, we report the development and validation of a bioluminescence-based intracellular calcium release assay for TAS2Rs that has a better assay performance than a fluorescence-based assay. Furthermore, the bioluminescence-based assay enables the evaluation of TAS2R agonists within an autofluorescent matrix, highlighting its potential utility in the assessment of the bitterness-inducing properties of plant or food fractions by the food industry. Additionally, further improvement to the bioluminescence-based assay for some TAS2Rs was achieved by altering their N-terminal signal sequences, leading to signal window enhancement. Altogether, the bioluminescence-based TAS2R assay can be used to perform functional studies of TAS2Rs, evaluate TAS2R-modulating properties of autofluorescent samples, and facilitate the discovery of compounds that can function as promising bitter taste modulators.</p>', 'posted': '2021-11-07'}, {'doi': '10.1101/2021.11.06.467582', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'specificity\\t: an R package for generalized specificity analysis, with example analyses of Hawaiian foliar endophytic Fungi, human gut microbiome bacteria, and Antarctic glacier bacteria.', 'authors': \"[{'name': 'John L Darcy'}, {'name': 'Anthony S Amend'}, {'name': 'Sean O I Swift'}, {'name': 'Pacifica S Sommers'}, {'name': 'Catherine A Lozupone'}]\", 'abstract': '<p>Understanding the factors that influence microbes\\' environmental distributions is important for determining drivers of microbial community composition. Species distributions are governed by parameters that influence their dispersal or survival. These include environmental variables like temperature and pH, and higher-dimensional variables like geographic distance and host species phylogenies. In microbial ecology, \"specificity\" is often described in the context of symbiotic or host parasitic interactions, but specificity can be more broadly used to describe the extent to which a species occupies a narrower range of an environmental variable than expected by chance. Using a standardization we describe here, Rao\\'s (1982, 2010) Quadratic Entropy can be conveniently applied to calculate specificity of a focal species to many different environmental variables, including 1-dimensional variables (vectors), dissimilarity matrices, phylogenies, and ontologies. We present our R package \"<italic>specificity</italic>\" for performing the above analyses, and apply it to four real-life microbial datasets to demonstrate its application: fungi living within the leaves of native Hawaiian plants, bacteria from the human gut microbiome, bacteria living within Antarctic glacier ice, and the Earth Microbiome Project data set. Finally, we present an interactive visualization and data exploration tool for these analyses, available as a companion R-package called \"<italic>specificity.shiny</italic>\".</p>', 'posted': '2021-11-07'}, {'doi': '10.1101/2021.11.02.466942', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Validation analysis of EMDB entries', 'authors': \"[{'name': 'Zhe Wang'}, {'name': 'Ardan Patwardhan'}, {'name': 'Gerard J Kleywegt'}]\", 'abstract': '<p>The Electron Microscopy Data Bank (EMDB) is the central archive of the electron cryo-microscopy (cryo-EM) community for storing and disseminating volume maps and tomograms. With input from the community, EMDB has developed new resources for validation of cryo-EM structures, focussing on the quality of the volume data alone and that of the fit of any models, themselves archived in the Protein Data Bank (PDB), to the volume data. Based on recommendations from community experts, the validation resources are developed in a three-tiered system. Tier 1 covers an extensive and evolving set of validation metrics, including tried and tested as well as more experimental ones, which are calculated for all EMDB entries and presented in the Validation Analysis (VA) web resource. This system is particularly useful for cryo-EM experts, both to validate individual structures and to assess the utility of new validation metrics. Tier 2 comprises a subset of the validation metrics covered by the VA resource that have been subjected to extensive testing and are considered to be useful for specialists as well as non-specialists. These metrics are presented on the entry-specific web pages for the entire archive on the EMDB website. As more experience is gained with the metrics included in the VA resource, it is expected that consensus will emerge in the community regarding a subset that is suitable for inclusion in the tier 2 system. Tier 3, finally, consists of the validation reports and servers that are produced by the Worldwide Protein Data Bank (wwPDB) Consortium. Successful metrics from tier 2 will be proposed for inclusion in the wwPDB validation pipeline and reports. We describe the details of the new resource, with an emphasis on the tier 1 system. The output of all three tiers is publicly available, either through the EMDB website (tiers 1 and 2) or through the wwPDB ftp sites (tier 3), although the content of all three will evolve over time (fastest for tier 1 and slowest for tier 3). It is our hope that these validation resources will help the cryo-EM community to get a better understanding of the quality, and the best ways to assess the quality of cryo-EM structures in EMDB and PDB.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.467511', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Natural variation in root exudation of GABA and DIMBOA impacts the maize root endosphere and rhizosphere microbiomes', 'authors': \"[{'name': 'Peng Wang'}, {'name': 'Lucas Dantes Lopez'}, {'name': 'Sophie Alvarez'}, {'name': 'Martha Lopez-Guerrero'}, {'name': 'Karin van Dijk'}, {'name': 'Jean-Jack Riethoven'}, {'name': 'Daniel Schachtman'}]\", 'abstract': '<p>Root exudates are important for shaping root-associated microbiomes. However, studies on a wider range of metabolites in exudates are required for a comprehensive understanding about their influence on microbial communities. We identified maize inbred lines that differ in exudate concentrations of DIMBOA and GABA using a semi-hydroponic system. These lines were grown in the field to determine the changes in microbial diversity and gene expression due to varying concentrations of DIMBOA and GABA in exudates using 16S rRNA amplicon sequencing and metatranscriptomics. Results showed individual and interaction effects of DIMBOA and GABA on the rhizosphere and root endosphere beta-diversity, most strongly at the V10 growth stage. The main bacterial families affected by both compounds were Ktedonobacteraceae and Xanthomonadaceae. Higher concentrations of DIMBOA in exudates affected the rhizosphere metatranscriptome, enriching for KEGG pathways associated with plant disease. This study validated the use of natural variation within plant species as a powerful approach for understanding the role of root exudates on microbiome selection. We also showed that a semi-hydroponic system can be used to identify maize genotypes that differ in GABA and DIMBOA exudate concentrations under field conditions. The impact of GABA exudation on root-associated microbiomes was shown for the first time.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.01.21265729', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Quantifying the effects of non-pharmaceutical and pharmaceutical interventions against COVID-19 epidemic in the Republic of Korea: Mathematical model-based approach considering age groups and the Delta variant', 'authors': \"[{'name': 'Youngsuk Ko'}, {'name': 'Victoria May P. Mendoza'}, {'name': 'Yubin Seo'}, {'name': 'Jacob Lee'}, {'name': 'Yeonju Kim'}, {'name': 'Donghyok Kwon'}, {'name': 'Eunok Jung'}]\", 'abstract': '<sec><title>Background</title><p>Early vaccination efforts and non-pharmaceutical interventions were insufficient to prevent a surge of coronavirus disease 2019 (COVID-19) cases triggered by the Delta variant. This study aims to understand how vaccination and variants contribute to the spread of COVID-19 so that appropriate measures are implemented.</p></sec><sec><title>Methods</title><p>A compartment model that includes age, vaccination, and infection with the Delta or non-Delta variants was developed. We estimated the transmission rates using maximum likelihood estimation and phase-dependent reduction effect of non-pharmaceutical interventions (NPIs) according to government policies from 26 February to 8 October 2021. We extended our model simulation until 31 December considering the initiation of eased NPIs. Furthermore, we also performed simulations to examine the effect of NPIs, arrival timing of Delta variant, and speed of vaccine administration.</p></sec><sec><title>Results</title><p>The estimated transmission rate matrices show distinct pattern, with the transmission rates of younger age groups (0 39 years) much larger than non-Delta. Social distancing (SD) level 2 and SD4 in Korea were associated with transmission reduction factors of 0.64 to 0.69 and 0.70 to 0.78, respectively. The easing of NPIs to a level comparable to SD2 should be initiated not earlier than 16 October to keep the number of severe cases below the capacity of Korea’s healthcare system. Simulation results also showed that a surge prompted by the spread of the Delta variant can be prevented if the number of people vaccinated daily was larger.</p></sec><sec><title>Conclusions</title><p>Simulations showed that the timing of easing and intensity of NPIs, vaccination speed, and screening measures are key factors in preventing another epidemic wave.</p></sec><sec><label>2</label><title>Key Messages</title><list list-type=\"bullet\"><list-item><p>Maximum likelihood estimation can be utilized to determine the transmission rates of the Delta and non-Delta variants.</p></list-item><list-item><p>The phase-dependent NPIs implemented by the Korean government were effectively quantified in the modelling study.</p></list-item><list-item><p>Even with fast vaccination, resurgence of cases is still possible if NPIs are eased too early or screening measures are relaxed.</p></list-item><list-item><p>The model can be used as a guide for policy makers on deciding appropriate SD level that considers not only disease control, but also the socio-economic impact of maintaining strict measures.</p></list-item></list></sec>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.01.465885', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Generation and Culture of Cardiac Microtissues in a Microfluidic Chip with a Reversible Open Top Enables Electrical Pacing, Dynamic Drug Dosing and Endothelial Cell Co-Culture', 'authors': \"[{'name': 'Aisen Vivas'}, {'name': 'Camilo IJspeert'}, {'name': 'Jesper Yue Pan'}, {'name': 'Kim Vermeul'}, {'name': 'Albert van den Berg'}, {'name': 'Robert Passier'}, {'name': 'Stephan Sylvest Keller'}, {'name': 'Andries D. van der Meer'}]\", 'abstract': '<p>Cardiovascular disease morbidity has increased worldwide in recent years while drug development has been affected by failures in clinical trials and lack of physiologically relevant models. Organs-on-chips and human pluripotent stem cell technologies aid to overcome some of the limitations in cardiac <italic>in vitro</italic> models. Here, a bi-compartmental, monolithic heart-on-chip device that facilitates porous membrane integration in a single fabrication step is presented. Moreover, the device includes open-top compartments that allow facile co-culture of human pluripotent stem cell-derived cardiomyocytes and human adult cardiac fibroblast into geometrically defined cardiac microtissues. The device can be reversibly closed with a glass seal or a lid with fully customized 3D-printed pyrolytic carbon electrodes allowing electrical stimulation of cardiac microtissues. A subjacent microfluidic channel allowed localized and dynamic drug administration to the cardiac microtissues, as demonstrated by a chronotropic response to isoprenaline. Moreover, the microfluidic channel could also be populated with human induced pluripotent stem-derived endothelial cells allowing co-culture of heterotypic cardiac cells in one device. Overall, this study demonstrates a unique heart-on-chip model that systematically integrates the structure and electromechanical microenvironment of cardiac tissues in a device that enables active perfusion and dynamic drug dosing. Advances in the engineering of human heart-on-chip models represent an important step towards making organ-on-a-chip technology a routine aspect of preclinical cardiac drug development.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466919', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Annexin A4 senses membrane curvature in a density dependent manner', 'authors': \"[{'name': 'Christoffer D. Florentsen'}, {'name': 'Joshua A. Daniels'}, {'name': 'Guillermo Moreno-Pescador'}, {'name': 'Iliriana Qoqaj'}, {'name': 'Jesper Nylandsted'}, {'name': 'Poul Martin Bendix'}]\", 'abstract': '<p>Annexins (ANXs) are a family of peripheral membrane binding proteins which play a vital role in the maintenance and function of cellular membranes. These proteins are known to be part of the plasma membrane repair machinery where they are known to bind to negatively charged lipids in the cell membrane in a calcium dependent manner. The shape of the plasma membrane is known to be a regulator of protein density and thereby affects several biological functions of the cell such as exo-and endocytosis, cell motility, immune responses and also membrane repair. Membrane deformation and curvature sensing by proteins is a well described phenomenon which can assist in recruitment of specific proteins to certain regions in the cell and facilitate membrane bending. Following ruptures in the plasma membrane, calcium influx assist in the association of ANXs with the membrane around the ruptured area. Due to the expected increase in curvature at the damaged membrane site, it has been suggested that both membrane curvature and Ca<sup>2+</sup> participate in the recruitment of ANXs. We have investigated the curvature sensing of ANXA4 in giant unilamellar vesicles (GUVs) by pulling high curvature membrane tethers from the vesicle surface using optical tweezers showing that ANXA4 recruitment increases with higher membrane curvature. We also describe an assay for determining protein density on the plasma membrane by utilizing ANXA4’s property as a calcium dependent membrane binding protein. This new assay allows us to investigate the effect of protein density on curvature sensing.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.05.467370', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Efficient transformation and genome editing of watermelon assisted by genes that encode developmental regulators', 'authors': \"[{'name': 'Wenbo Pan'}, {'name': 'Zhentao Cheng'}, {'name': 'Zhiguo Han'}, {'name': 'Hong Yang'}, {'name': 'Wanggen Zhang'}, {'name': 'Huawei Zhang'}]\", 'abstract': '<p>The Cucurbitaceae contains multiple species of important food plants. But most of them are difficult to be genetically transformed. Watermelon is one of the most important fruit species of Cucurbitaceae, and it is a model horticulture crops. Its low transgenic efficiency is the major bottleneck in functional genome research and genome editing-based breeding. Here, with the help of genes that encode developmental regulators (DRs), in particular AtGRF5, and an appropriate Agrobacterium strain (GV3101), we have significantly increased the transformation efficiency of watermelon to about 25%, which is a 40-fold increase compared with a traditional vector. AtGRF5-mediated transformation increased the number of transformable watermelon genotypes without causing obvious side effects. Moreover, when applying this strategy to CRISPR/Cas9-mediated genome editing, clpds knockout mutants were generated in the T0 generation. Our results show that AtGRF5 is a powerful and robust tool to effectively create transgenic plants or knockout mutations in watermelon. Similar strategies using DRs might also be able to overcome the transformation barriers in many other Cucurbitaceae species.</p>', 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.04.467322', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'AlphaFold Models Illuminate Half of Dark Human Proteins', 'authors': \"[{'name': 'Jessica L Binder'}, {'name': 'Joel Berendzen'}, {'name': 'Amy O Stevens'}, {'name': 'Yi He'}, {'name': 'Jian Wang'}, {'name': 'Nikolay V Dokholyan'}, {'name': 'Tudor I Oprea'}]\", 'abstract': '<p>We investigate the use of confidence scores to predict the accuracy of a given AlphaFold model for drug discovery. Predicted accuracy is improved by eliminating confidence scores below 80, due to effects of disorder. 95% of models corresponding to a set of recent crystal structures are accurate at the fold level. Conformational discordance in the training set has a more significant effect on accuracy than sequence divergence. We propose criteria for models and residues that are possibly useful for virtual screening, by which AlphaFold provides models for half of understudied (dark) human proteins and two-thirds of residues in those models.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.467160', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Metabolic heritage mapping: heterogenous pools of cytoplasmic nucleotide sugars are selectively utilized by various glycosyltransferases', 'authors': \"[{'name': 'Paulina Sosicka'}, {'name': 'Bobby G. Ng'}, {'name': 'Lauren E. Pepi'}, {'name': 'Asif Shajahan'}, {'name': 'Maurice Wong'}, {'name': 'David A. Scott'}, {'name': 'Kenjiroo Matsumoto'}, {'name': 'Zhi-Jie Xia'}, {'name': 'Carlito B. Lebrilla'}, {'name': 'Robert S. Haltiwanger'}, {'name': 'Parastoo Azadi'}, {'name': 'Hudson H. Freeze'}]\", 'abstract': '<p>Biosynthesis of macromolecules requires precursors such as sugars or amino acids, originating from exogenous/dietary sources, reutilization/salvage of degraded molecules or <italic>de novo</italic> synthesis. Since these sources are assumed to contribute to one homogenous pool, their individual contributions are often overlooked. Protein glycosylation uses monosaccharides from all the above sources to produce nucleotide sugars required to assemble hundreds of distinct glycans. Here we demonstrate that cells identify the origin/heritage of the monosaccharide, fucose, for glycosylation. We measured the contribution of GDP-fucose from each of these sources for glycan synthesis and found that different fucosyltransferases, individual glycoproteins, and linkage-specific fucose residues identify and select different GDP-fucose pools dependent on their heritage. This supports the hypothesis that GDP-fucose exists in multiple, distinct pools, not as a single homogenous pool. The selection is tightly regulated since the overall pool size remains constant. We present novel perspectives on monosaccharide metabolism, which may have general applicability.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.467445', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Mitochondrial DNA variants segregate during human preimplantation development into genetically different cell lineages that are maintained postnatally', 'authors': \"[{'name': 'Joke Mertens'}, {'name': 'Marius Regin'}, {'name': 'Neelke De Munck'}, {'name': 'Edouard Couvreu de Deckersberg'}, {'name': 'Florence Belva'}, {'name': 'Karen D Sermon'}, {'name': 'Herman Tournaye'}, {'name': 'Christophe Blockeel'}, {'name': 'Hilde Van de Velde'}, {'name': 'Claudia Spits'}]\", 'abstract': '<p>Humans present remarkable mitochondrial DNA (mtDNA) variant mosaicism, not only across tissues but even across individual cells within one person. The timing of the first appearance of this mosaicism has not yet been established. In this study, we hypothesized it occurs during preimplantation development. To investigate this, we deep-sequenced the mtDNA of 254 oocytes from 85 donors, 158 single blastomeres of 25 day-3 embryos, 17 inner cell mass and trophectoderm samples of 7 day-5 blastocysts, 142 bulk DNA and 68 single cells of different adult tissues. We found that day-3 preimplantation embryos already present blastomeres that carry variants unique to that cell, showing that the first events of mtDNA mosaicism happen very early in human development. We classified the mtDNA variants based on their recurrence or uniqueness across sibling oocytes and embryos, and between single cells and samples from the same embryos or adult individuals. Variants that recurred across samples had higher heteroplasmic loads and more frequently resulted in synonymous changes or were located in non-coding regions than variants that were unique to one oocyte or single embryonic cell. These differences were maintained through developmental stages, suggesting that the mtDNA mosaicism arising in preimplantation development is maintained into adulthood. Further, the results support a model in which close clustering of mitochondria carrying specific mtDNA variants in the ooplasm leads to asymmetric distribution of these mitochondria throughout the cell divisions of the preimplantation embryo, resulting in the appearance of the first form of mtDNA mosaicism in human development.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.467166', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Two-speed genome expansion drives the evolution of pathogenicity in animal fungal pathogens', 'authors': \"[{'name': 'Theresa Wacker'}, {'name': 'Nicolas Helmstetter'}, {'name': 'Duncan Wilson'}, {'name': 'Matthew C. Fisher'}, {'name': 'David J. Studholme'}, {'name': 'Rhys A. Farrer'}]\", 'abstract': '<p>The origins of virulence in amphibian-infecting chytrids <italic>Batrachochytrium dendrobatidis</italic> (<italic>Bd</italic>) and <italic>Batrachochytrium salamandrivorans</italic> (<italic>Bsal)</italic> are largely unknown. Here, we use deep nanopore sequencing of <italic>Bsal</italic> and comparative genomics against 21 high-quality genome assemblies that span the fungal Chytridiomycota. <italic>Bsal</italic> has the most repeat-rich genome, comprising 40.9% repetitive elements, which has expanded to more than 3X the length of its conspecific <italic>Bd</italic>. M36 metalloprotease virulence factors are highly expanded in <italic>Bsal</italic> and 53% of the 177 unique genes are flanked by transposable elements, suggesting repeat-driven expansion. The largest M36 sub-family are mostly (84%) flanked upstream by a novel LINE element, a repeat superfamily implicated with gene copy number variations. We find that <italic>Bsal</italic> has a highly compartmentalized genome architecture, with virulence factors enriched in gene-sparse/repeat-rich compartments, while core conserved genes occur in gene-rich/repeat-poor compartments. This is a hallmark of two-speed genome evolution. Furthermore, genes with signatures of positive selection in <italic>Bd</italic> are enriched in repeat-rich regions, suggesting they are a cradle for chytrid pathogenicity evolution, and <italic>Bd</italic> also has a two-speed genome. This is the first evidence of two-speed genomes in any animal pathogen, and sheds new light on the evolution of fungal pathogens of vertebrates driving global declines and extinctions.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467190', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Detection of pseudouridine modifications and type I/II hypermodifications in human mRNAs using direct, long-read sequencing', 'authors': \"[{'name': 'Sepideh Tavakoli'}, {'name': 'Mohammad Nabizadehmashhadtoroghi'}, {'name': 'Amr Makhamreh'}, {'name': 'Howard Gamper'}, {'name': 'Neda Rezapour'}, {'name': 'Ya-MIng Hou'}, {'name': 'Meni Wanunu'}, {'name': 'Sara H Rouhanifard'}]\", 'abstract': '<p>Enzyme-mediated chemical modifications to mRNAs have the potential to fine-tune gene expression in response to environmental stimuli. Notably, pseudouridine-modified mRNAs are more resistant to RNase-mediated degradation, more responsive to cellular stress, and have the potential to modulate immunogenicity and enhance translation in vivo. However, the precise biological functions of pseudouridine modification on mRNAs remain unclear due to the lack of sensitive and accurate tools for mapping. We developed a semi-quantitative method for mapping pseudouridylated sites with high confidence directly on mammalian mRNA transcripts via direct RNA, long-read nanopore sequencing. By analysis of a modification-free transcriptome, we demonstrate that the depth of coverage and intrinsic errors associated with specific k-mer sequences are critical parameters for accurate base-calling. We adjust these parameters for high-confidence U-to-C base-calling errors that occur at pseudouridylated sites, which are benchmarked against sites that were identified previously by biochemical methods. We also uncovered new pseudouridylated sites, many of which fall on genes that encode RNA binding proteins and on uridine-rich k-mers. Sites identified by U-to-C base calling error were verified using 1000mer synthetic RNA controls bearing a single pseudouridine in the center position, demonstrating that 1. the U-to-C base-calling error occurs at the site of pseudouridylation, and 2. the basecalling error is systematically under-calling the pseudouridylated sites. High-occupancy sites with &gt;40% U-to-C basecalling error are classified as sites of hyper modification type I, whereas genes with more than one site of pseudouridylation are classified as having type II hyper modification which is confirmed by single-molecule analysis. We report the discovery of mRNAs with up to 7 unique sites of pseudouridine modification. Here we establish an innovative pipeline for direct identification, quantification, and detection of pseudouridine modifications and type I/II hypermodifications on native RNA molecules using long-read sequencing without resorting to RNA amplification, chemical reactions on RNA, enzyme-based replication, or DNA sequencing steps.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.05.463867', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Reconstructing the Origins of a Neuropeptide Signaling System Using the Accelerated Evolution of Biodiverse Cone Snail Venoms', 'authors': \"[{'name': 'Thomas Lund Koch'}, {'name': 'Iris Bea L Ramiro'}, {'name': 'Paula Flórez-Salcedo'}, {'name': 'Ebbe Engholm'}, {'name': 'Knud Jørgen Jensen'}, {'name': 'Kevin Chase'}, {'name': 'Baldomero M Olivera'}, {'name': 'Walden Emil Bjørn-Yoshimoto'}, {'name': 'Helena Safavi-Hemami'}]\", 'abstract': \"<p>Somatostatin and its related peptides (SSRPs) form a family of hormones with diverse physiological roles. The ubiquitous presence of SSRPs in vertebrates and several invertebrate deuterostomes suggests an ancient origin of the SSRP signaling system. However, the existence of SSRP genes outside of deuterostomes has not been established and the evolutionary history of this signaling system remains poorly understood. Our recent discovery of SSRP-like toxins (consomatins) in venomous marine cone snails (Conus) suggested the presence of a homologous signaling system in mollusks and potentially other protostomes. Here we identify the molluscan SSRP-like signaling gene that gave rise to the consomatin family. Following recruitment into venom, consomatin genes experience strong positive selection and repeated gene duplications resulting in the formation of a hyper-diverse family of venom peptides. Intriguingly, the largest number of consomatins was found in worm-hunting species (&gt; 400 sequences), indicating a homologous system in annelids, another large protostome phylum. Comprehensive sequence mining enabled the identification of orthologous SSRP-like sequences (and their corresponding orphan receptor) in annelids and several other protostome phyla. These results establish the existence of SSRP-like sequences in many major branches of bilaterians, including xenacoelomorphs, a phylum believed to have emerged before the divergence of protostomes and deuterostomes, ~ 600 My ago. Finally, having a large set of predator-prey SSRP sequences available, we show that while the cone snail's signaling SSRP-like genes are under purifying selection, in striking contrast, the consomatin genes experience rapid directional selection to target receptors in a changing mix of prey.</p>\", 'posted': '2021-11-06'}, {'doi': '10.1101/2021.11.04.467119', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Distinct roles for CDK-Mediator in controlling Polycomb-dependent chromosomal interactions and priming genes for induction', 'authors': \"[{'name': 'Emilia Dimitrova'}, {'name': 'Angelika Feldmann'}, {'name': 'Robin H van der Weide'}, {'name': 'Koen D Flach'}, {'name': 'Anna Lastuvkova'}, {'name': 'Elzo de Wit'}, {'name': 'Robert J Klose'}]\", 'abstract': '<p>Precise control of gene expression underpins normal development. This relies on mechanisms that enable communication between gene promoters and other regulatory elements. In embryonic stem cells (ESCs), the CDK-Mediator (CDK-MED) complex has been reported to physically link gene regulatory elements to enable gene expression and also prime genes for induction during differentiation. Here we discover that CDK-MED contributes little to 3D genome organisation in ESCs, but has a specific and essential role in controlling interactions between inactive gene regulatory elements bound by Polycomb repressive complexes (PRCs).  These interactions are established by the canonical PRC1 (cPRC1) complex but rely on CDK-MED, which facilitates binding of cPRC1 to its target sites.  Importantly, through separation of function experiments, we reveal that this collaboration between CDK-MED and cPRC1 in creating long-range interactions does not function to prime genes for induction during differentiation. Instead, we discover that priming relies on an interaction-independent mechanism whereby the CDK module supports core Mediator engagement with gene promoters to enable gene activation.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.10.28.466340', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Analysis of cellular and synaptic mechanisms behind spontaneous cortical activity in vitro: Insights from optimization of spiking neuronal network models', 'authors': \"[{'name': 'Jugoslava Aćimović'}, {'name': 'Tuomo Mäki-Marttunen'}, {'name': 'Heidi Teppola'}, {'name': 'Marja-Leena Linne'}]\", 'abstract': '<p>Spontaneous network bursts, the intervals of intense network-wide activity interleaved with longer periods of sparse activity, are a hallmark phenomenon observed in cortical networks at postnatal developmental stages. Generation, propagation and termination of network bursts depend on a combination of synaptic, cellular and network mechanisms; however, the interplay between these mechanisms is not fully understood. We study this interplay <italic>in silico,</italic> using a new data-driven framework for generating spiking neuronal networks fitted to the microelectrode array recordings. We recorded the network bursting activity from rat postnatal cortical networks under several pharmacological conditions. In each condition, the function of specific excitatory and inhibitory synaptic receptors was reduced in order to examine their impact on global network dynamics. The obtained data was used to develop two complementary model fitting protocols for automatic model generation. These protocols allowed us to disentangle systematically the modeled cellular and synaptic mechanisms that affect the observed network bursts. We confirmed that the change in excitatory and inhibitory synaptic transmission <italic>in silico</italic>, consistent with pharmacological conditions, can account for the changes in network bursts relative to the control data. Reproducing the exact recorded network bursts statistics required adapting both the synaptic transmission and the cellular excitability separately for each pharmacological condition. Our results bring new understanding of the complex interplay between cellular, synaptic and network mechanisms supporting the burst dynamics. While here we focused on analysis of <italic>in vitro</italic> data, our approach can be applied <italic>ex vivo</italic> and <italic>in vivo</italic> given that the appropriate experimental data is available.</p><sec><title>New &amp; Noteworthy</title><p>We studied the role of synaptic mechanisms in shaping the neural population activity by proposing a new method to combine experimental data and data-driven computational modeling based on spiking neuronal networks. We analyze a dataset recorded from postnatal rat cortical cultures <italic>in vitro</italic> under the pharmacological influence of excitatory and inhibitory synaptic receptor antagonists. Our computational model identifies neurobiological mechanisms necessary to reproduce the changes in population activity seen across pharmacological conditions.</p></sec>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.04.467237', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Dynamic spreading of chromatin-mediated gene silencing and reactivation between neighboring genes in single cells', 'authors': \"[{'name': 'Sarah Lensch'}, {'name': 'Michael H. Herschl'}, {'name': 'Connor H. Ludwig'}, {'name': 'Joydeb Sinha'}, {'name': 'Michaela M. Hinks'}, {'name': 'Adi Mukund'}, {'name': 'Taihei Fujimori'}, {'name': 'Lacramioara Bintu'}]\", 'abstract': '<p>In mammalian cells genes that are in close proximity are coupled transcriptionally: silencing or activating one gene can affect its neighbors. Understanding these dynamics is important for natural processes, such as heterochromatin spreading during development and aging, and when designing synthetic gene regulation. Here, we systematically dissect this process in single cells by recruiting and releasing repressive chromatin regulators at dual-gene synthetic reporters, and measuring how fast gene silencing and reactivation spread as a function of intergenic distance and configuration of insulator elements. We find that silencing by KRAB, associated with histone methylation, spreads between two genes within hours, with a time delay that increases with distance. This fast KRAB-mediated spreading is not blocked by the classical cHS4 insulators. Silencing by histone deacetylase HDAC4 of the upstream gene can also lead to downstream gene silencing, but with a days-long delay that does not change with distance. This slower silencing can sometimes be stopped by insulators. Gene reactivation of neighboring genes is also coupled, with strong promoters and insulators determining the order of reactivation. We propose a new model of multi-gene regulation, where both gene silencing and gene reactivation can act at a distance, allowing for coordinated dynamics via chromatin regulator recruitment.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.04.467339', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Chromosome splitting of Plasmodium berghei using the CRISPR/Cas9 system.', 'authors': \"[{'name': 'Daniel Addo-Gyan'}, {'name': 'Haruka Matsushita'}, {'name': 'Enya Sora'}, {'name': 'Tsubasa Nishi'}, {'name': 'Naoaki Shinzawa'}, {'name': 'Masao Yuda'}, {'name': 'Shiroh Iwanaga'}]\", 'abstract': '<p>Spatial arrangement of chromosomes is responsible for gene expression in  Plasmodium  parasites. However, methods for rearranging chromosomes have not been established, which makes it difficult to investigate its role in detail. Here, we report a method for splitting chromosome in rodent malaria parasite by CRISPR/Cas9 system using fragments in which a telomere and a centromere were incorporated. The resultant split chromosomes segregated accurately into daughter parasites by the centromere. In addition, elongation of  de novo  telomeres were observed, indicating its proper function. Furthermore, chromosome splitting had no effect on development of parasites. Splitting of the chromosome is expected to alter its spatial arrangement, and our method will thus be useful for investigating its biological role related with gene expression.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.04.467271', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'decoupleR: Ensemble of computational methods to infer biological activities from omics data', 'authors': \"[{'name': 'Pau Badia-i-Mompel'}, {'name': 'Jesús Vélez'}, {'name': 'Jana Braunger'}, {'name': 'Celina Geiss'}, {'name': 'Daniel Dimitrov'}, {'name': 'Sophia Müller-Dott'}, {'name': 'Petr Taus'}, {'name': 'Aurelien Dugourd'}, {'name': 'Christian Haydar Holland'}, {'name': 'Ricardo Omar Ramírez Flores'}, {'name': 'Julio Saez-Rodriguez'}]\", 'abstract': '<p>Summary: Many methods allow us to extract biological activities from omics data using information from prior knowledge resources, reducing the dimensionality for increased statistical power and better interpretability. Here, we present decoupleR, a Bioconductor package containing computational methods to extract these activities within a unified framework. decoupleR allows us to flexibly run any method with a given resource, including methods that leverage mode of regulation and weights of interactions. Using decoupleR, we evaluated the performance of methods on transcriptomic and phospho-proteomic perturbation experiments. Our findings suggest that simple linear models and the consensus score across methods perform better than other methods at predicting perturbed regulators.\\nAvailability and Implementation: decoupleR is open source available in Bioconductor (https://www.bioconductor.org/packages/release/bioc/html/decoupleR.html). The code to reproduce the results is in Github (https://github.com/saezlab/decoupleR_manuscript) and the data in Zenodo (https://zenodo.org/record/5645208).</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.04.21265926', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Whole Exome Sequencing in dense families suggests genetic pleiotropy amongst Mendelian and complex neuropsychiatric syndromes', 'authors': \"[{'name': 'Suhas Ganesh'}, {'name': 'Alekhya Vemula'}, {'name': 'Samsiddhi Bhattacharjee'}, {'name': 'Kezia Mathew'}, {'name': 'Dhruva Ithal'}, {'name': 'Karthick Navin'}, {'name': 'Ravi Kumar Nadella'}, {'name': 'Biju Viswanath'}, {'name': 'Sanjeev Jain'}, {'name': 'Meera Purushottam'}, {'name': ' '}]\", 'abstract': '<p>Introduction: Whole Exome Sequencing (WES) studies have provided important insights into the genetic architecture of neuropsychiatric syndromes identifying rare and novel variants in the protein-coding sequence of the genome that impact function. Variants and genes that are central to the shared biology of these clinical syndromes may be identified by WES in families with multiple affected individuals with serious mental illnesses (F-SMI). Methods: We performed WES in 250 individuals (affected = 186, family-control = 64) from 100 families, each with 2 or more members with SMI, and 60 unrelated population-controls. Within pedigree prioritization employed criteria of 1. rarity (Minor Allele Frequency &lt;0.1%, GnomAD South-Asian sample, 15308 exomes); 2. functional consequence (Loss of Function or missense deleterious in 4/5 in silico predictions). 3. sharing by 3 or more affected members within a family. Across the sample, gene-set-wide case-control association analysis was performed with Sequence Kernel Association Test, accounting for kinship. Results: In 17 families with 3 or more exome samples, we identified 79 rare predicted deleterious variants in 79 unique genes shared by 3 or more affected members and absent in 60 unrelated controls. Twenty (25.32%) genes were implicated in monogenic neurodevelopmental syndromes in Online Mendelian Inheritance in Man representing a statistically significant overrepresentation (Fishers Exact test OR = 2.47, p = 0.001). In gene-set wise SKAT, statistically significant association was noted for genes related to synaptic function (SKAT-p = 0.017). Discussion: In F-SMI based WES study, we identify private, rare, protein altering variants in genes previously implicated in monogenic Mendelian neuropsychiatric syndromes; suggesting pleotropic influences in neurodevelopment between complex and monogenic syndromes.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.04.21265937', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Estimation of the basic reproduction number of COVID-19 from the incubation period distribution', 'authors': \"[{'name': 'Lasko Basnarkov'}, {'name': 'Igor Tomovski'}, {'name': 'Florin Avram'}]\", 'abstract': '<p>Background\\n\\n\\nThe estimates of future course of spreading of the SARS-CoV-2 virus are frequently based on Markovian models in which the transitions between the compartments are exponentially distributed. Specifically, the basic reproduction number R0 is also determined from formulae where it is related to the parameters of such models. The observations show that the start of infectivity of an individual appears nearly at the same time as the onset of symptoms, while the distribution of the incubation period is not an exponential.\\n\\nMethods\\n\\nWe propose a method for estimation of R0 for COVID-19 based on the empirical incubation period distribution and assumed very short infectivity period that lasts only few days around the onset of symptoms. It is tested on daily new cases in six major countries in Europe, in the first wave of epidemic in spring, 2020. \\n\\nResults\\n\\nThe calculations show that even if the infectivity starts two days before the onset of symptoms and stops immediately when they appear, the value of R0 is larger than that from the classical, Markovian approach. For more realistic cases, when only individuals with mild symptoms spread the virus for few days after onset of symptoms, the respective values are even larger.\\n\\nConclusions\\n\\nThe calculations of R0 and other characteristics of spreading of COVID-19 based on the classical, Markovian approaches should be taken very cautiously. Instead, non-Markovian models with general distribution functions of transition between compartments should be considered as more appropriate.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466907', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Single-cell epigenetic analysis reveals principles of chromatin states in H3.3-K27M gliomas', 'authors': \"[{'name': 'Nofar Harpaz'}, {'name': 'Tamir Mittelman'}, {'name': 'Olga Beresh'}, {'name': 'Tomer-Meir Salame'}, {'name': 'Noa Furth'}, {'name': 'Guy Ron'}, {'name': 'Efrat Shema'}]\", 'abstract': '<p>Cancer cells are highly heterogeneous, both at the transcriptional level and in their epigenetic state. Methods to study epigenetic heterogeneity are limited in their throughput and the information obtained per cell. Here, we adapted Cytometry by Time of Flight (CyTOF) to analyze a wide panel of histone modifications and chromatin regulators in primary tumor-derived lines of Diffused Intrinsic Pontine Glioma (DIPG). DIPG is a lethal pediatric brain cancer, driven by a mutation in histone H3 leading to substitution of lysine 27 with methionine (H3-K27M mutation). We identified two epigenetically distinct subpopulations in DIGP, reflecting inherent heterogeneity in the expression levels of the mutant histone. These two epigenetic subpopulations are robust across tumor lines derived from different patients and show differential proliferation capacity as well as expression of stem-cell and differentiation markers. Moreover, we demonstrate the use of this single-cell high-dimensional data to elucidate potential interactions between histone modifications and epigenetic alterations during the cell-cycle. Our work establishes new concepts for the analysis of epigenetic heterogeneity and cross-talk in cancer that could be applied to diverse biological systems.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466896', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Influence of Response Criterion on Nociceptive Detection Thresholds and Evoked Potentials', 'authors': \"[{'name': 'Boudewijn van den Berg'}, {'name': 'L. Vanwinsen'}, {'name': 'G. Pezzali'}, {'name': 'Jan R. Buitenweg'}]\", 'abstract': '<p>Pain scientists and clinicians search for objective measures of altered nociceptive processing to study and stratify chronic pain patients. Nociceptive processing can be studied by observing a combination of nociceptive detection thresholds and evoked potentials. However, it is unknown whether the nociceptive detection threshold measured using a Go-/No-Go (GN) procedure can be biased by a response criterion. In this study, we compared nociceptive detection thresholds, psychometric slopes and central evoked potentials obtained during a GN procedure with those obtained during a 2-interval forced choice (2IFC) procedure to determine 1) if the nociceptive detection threshold during a GN procedure is biased by a criterion and 2) to determine if nociceptive evoked potentials observed in response to stimuli around the detection threshold are biased by a criterion. We found that the detection threshold can be higher when assessed using a GN procedure in comparison with the 2IFC procedure. The average P2 component in the central evoked potential showed on-off behavior with respect to stimulus detection and increased proportionally with the detection probability during a GN procedure. These data suggest that nociceptive detection thresholds estimated using a GN procedure are subject to a response criterion.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466852', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Comprehensive identification of fetal cis-regulatory elements in the human genome by single-cell multi-omics analysis', 'authors': \"[{'name': 'Hao Yu'}, {'name': 'Na Ai'}, {'name': 'Ping Peng'}, {'name': 'Yuwen Ke'}, {'name': 'Xuepeng Chen'}, {'name': 'Yun Li'}, {'name': 'Ting Zhao'}, {'name': 'Shan Jiang'}, {'name': 'Jiang Liu'}, {'name': 'Lan Jiang'}]\", 'abstract': '<p>The regulatory programs driving early organogenesis in human is complex and still poorly understood. We performed parallel profiling of gene expression and chromatin accessibility to 28 human fetal tissue samples representing 14 organs in the first trimester. Collectively, we have generated 415,793 single-cell profiles. By integration analysis of transcriptome and chromatin accessibility, we detected 225 distinct cell types and 848,475 candidate accessible cis-regulatory elements (aCREs). By linking regulatory elements to their putative target genes, we identified not only 108,699 enhancers, but also 23,392 silencers elements. We uncovered thousands of genes regulated by both enhancers and silencers in an organ or cell-type-specific manner. Furthermore, our unique approach revealed a substantial proportion of distal DNA elements are transcribed CREs (tCREs), which show both open chromatin signal and transcription initiation activity of non-coding transcript. The landscape of fetal cis-regulatory elements facilitates the interpretation of the genetic variant of complex disease and infer the cell type of origin for cancer. Overall, our data provide a comprehensive map of the fetal cis-regulatory elements at single-cell resolution and a valuable resource for future study of human development and disease.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466525', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Concerted cellular responses to type I interferon propel memory impairment associated with amyloid β plaques', 'authors': \"[{'name': 'Ethan R. Roy'}, {'name': 'Gabriel Chiu'}, {'name': 'Sanming Li'}, {'name': 'Nicholas E. Propson'}, {'name': 'Hui Zheng'}, {'name': 'Wei Cao'}]\", 'abstract': '<p>Despite well-documented maladaptive neuroinflammation in Alzheimer’s disease (AD), the principal signal that drives memory and cognitive impairment remains elusive. Here, we reveal robust, age-dependent cellular reactions to type I interferon (IFN), an innate immune cytokine aberrantly elicited by β amyloid plaques, and examine their role in cognition and neuropathology relevant to AD in a murine amyloidosis model. Long-term blockade of IFN receptor rescued both memory and synaptic deficits, and also resulted in reduced microgliosis, inflammation, and neuritic pathology. Interestingly, microglia-specific IFN receptor ablation attenuated the loss of post-synaptic terminals, whereas IFN signaling in neural cells contributed to pre-synaptic alteration and plaque accumulation. Intriguingly, IFN pathway activation displayed a strong inverse correlation with cognitive performance, promoting selective synapse engulfment by microglia rather than amyloid plaques. Overall, IFN signaling represents a critical module within the neuroinflammatory network of AD and prompts a concerted cellular state that is detrimental to memory and cognition.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.466939', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'DDK inhibition disrupts replication leading to mitotic catastrophe in Ewing sarcoma', 'authors': \"[{'name': 'Jeffrey C Martin'}, {'name': 'Ajay Gupta'}, {'name': 'Tamara J Hagoel'}, {'name': 'Lingqiu Gao'}, {'name': 'Miranda L. Lynch'}, {'name': 'Anna Woloszynska'}, {'name': 'Thomas Melendy'}, {'name': 'Jeremy Kane'}, {'name': 'Joseph Kuechle'}, {'name': 'Joyce Ohm'}]\", 'abstract': '<p>Ewing sarcoma is the second most common bone malignancy in children and adolescents. Patients with upfront metastatic or recurrent disease have poor outcomes with 5-year survival rates of &lt;30%. CDC7, also known as DDK (DBF4-dependent kinase), is a serine-threonine kinase that, in coordination with its activation subunit ASK (or DBF4), is involved in a diverse array of cellular functions including the regulation of DNA replication initiation and activation of the replication stress response. Due to DDK diverse roles during replication, coupled with an increased level of genomic instability and R-loop-mediated replication stress within Ewing sarcoma cells, we hypothesized that Ewing sarcoma cells would be particularly vulnerable to DDK inhibitors. Here, we show that treatment with two selective DDK inhibitors, TAK-931 and XL413, results in apoptosis and a significant reduction in cell viability in EWS-FLI1-harboring Ewing sarcoma cell lines. We show that low dose DDK inhibition in Ewing sarcoma cells causes an accumulation of cells in late-S phase with a reduced replication capacity. There is also evidence of premature mitotic entry indicating an inability to properly complete DNA replication in a timely manner upon DDK inhibition. Also, there is a significant increase in the formation of micronuclei and other aberrant mitotic structures upon DDK inhibition in Ewing sarcoma cells indicating a failure to properly progress through S-phase followed by improper mitotic entry/progression, resulting in mitotic catastrophe. Interestingly, we observed minimal signs of mitotic accumulation, despite clear evidence of replication and mitotic stress, suggesting a failure to properly enforce the mitotic checkpoint. Together, these results suggest that Ewing sarcoma cells rely on the activity of DDK to maintain cell viability and suggest that DDK inhibition may prove to be a viable therapeutic strategy for patients with Ewing sarcoma.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.10.31.466677', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Multiple spillovers and onward transmission of SARS-Cov-2 in free-living and captive White-tailed deer', 'authors': \"[{'name': 'Suresh V Kuchipudi'}, {'name': 'Meera Surendran-Nair'}, {'name': 'Rachel M Ruden'}, {'name': 'Michele Yon'}, {'name': 'Ruth H Nissly'}, {'name': 'Rahul K Nelli'}, {'name': 'Lingling Li'}, {'name': 'Bhushan M Jayarao'}, {'name': 'Kurt Vandegrift'}, {'name': 'Costas D Maranas'}, {'name': 'Nicole Levine'}, {'name': 'Katriina Willgert'}, {'name': 'Andrew J.K Conlan'}, {'name': 'Randall J Olsen'}, {'name': 'James Davis'}, {'name': 'James M. Musser'}, {'name': 'Peter J Hudson'}, {'name': 'Vivek Kapur'}]\", 'abstract': '<p>Many animal species are susceptible to SARS-CoV-2 and could potentially act as reservoirs, yet transmission of the virus in non-human free-living animals has not been documented. White-tailed deer (Odocoileus virginianus), the predominant cervid in North America, are susceptible to SARS-CoV-2 infection, and experimentally infected fawns can transmit the virus. To test the hypothesis that SARS-CoV-2 may be circulating in deer, we tested 283 retropharyngeal lymph node (RPLN) samples collected from 151 free-living and 132 captive deer in Iowa from April 2020 through December of 2020 for the presence of SARS-CoV-2 RNA.  Ninety-four of the 283 deer (33.2%; 95% CI: 28, 38.9) samples were positive for SARS-CoV-2 RNA as assessed by RT-PCR. Notably, between November 23, 2020 and January 10, 2021, 80 of 97 (82.5%; 95% CI 73.7, 88.8) RPLN samples had detectable SARS-CoV-2 RNA by RT-PCR. Whole genome sequencing of the 94 positive RPLN samples identified 12 SARS-CoV-2 lineages, with B.1.2 (n = 51; 54.5%), and B.1.311 (n = 19; 20%) accounting for ~75% of all samples. The geographic distribution and nesting of clusters of deer and human lineages strongly suggest multiple zooanthroponotic spillover events and deer-to-deer transmission. The discovery of sylvatic and enzootic SARS-CoV-2 transmission in deer has important implications for the ecology and long-term persistence, as well as the potential for spillover to other animals and spillback into humans.  These findings highlight an urgent need for a robust and proactive One Health approach to obtaining a better understanding of the ecology and evolution of SARS-CoV-2.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.02.467006', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'SVJAM: Joint Analysis of Structural Variants Using Linked Read Sequencing Data', 'authors': \"[{'name': 'Mustafa Hakan Gunturkun'}, {'name': 'Flavia Villani'}, {'name': 'Vincenza Colonna'}, {'name': 'David Ashbrook'}, {'name': 'Robert W Williams'}, {'name': 'Hao Chen'}]\", 'abstract': '<p>Linked-read whole genome sequencing methods, such as the 10x Chromium, attach a unique molecular barcode to each high molecular weight DNA molecule. The samples are then sequenced using short-read technology. During analysis, sequence reads sharing the same barcode are aligned to adjacent genomic locations. The pattern of barcode sharing between genomic regions allows the discovery of large structural variants (SVs) in the range of 1 Kb to a few Mb. Most SV calling methods for these data, such as LongRanger, analyze one sample at a time and often produces inconsistent results for the same genomic location across multiple samples. We developed a method, SVJAM, for joint calling of SVs, using data from 152 members of the BXD family of recombinant inbred strains of mice. Our method first collects candidate SV regions from single sample analysis, such as those produced by LongRanger. We then retrieve barcode overlapping data from all samples for each region. These data are organized as a high dimensional matrix. The dimension of this matrix is then reduced using principal component analysis. Samples projected onto a two dimensional space formed by the first two principal components forms two or three clusters based on their genotype, representing the reference, alternative, or heterozygotic alleles. We developed a novel distance measure for hierarchical clustering and rotating the axes to find the optimal clustering results. We also developed an algorithm to decide whether the pattern of sample distribution is best fitted with one, two, or three genotypes. For each sample, we calculate its membership score for each genotype. We compared results produced by SVJAM with LongRanger and few methods that rely on PacBio or Oxford Nanopore data. In a comparison of SVJAM with SV detected using long-read sequencing data for the DBA/2J strain, we found that our results recovered many SVs missed by LongRanger. We also found many SVs called by LongRanger were assigned with an incorrect SV type. Our algorithm also consistently identified heterozygotic regions.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466993', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Laser-driven Wireless Deep Brain Stimulation using Temporal Interference and Organic Electrolytic Photocapacitors', 'authors': \"[{'name': 'Florian MISSEY'}, {'name': 'Mary Jocelyn DONAHUE'}, {'name': 'Pascal WEBER'}, {'name': 'Ibrahima NGOM'}, {'name': 'Emma ACERBO'}, {'name': 'Boris BOTZANOWSKI'}, {'name': 'Ludovico MIGLIACCIO'}, {'name': 'Viktor Jirsa'}, {'name': 'Eric Daniel GLOWACKI'}, {'name': 'Adam Williamson'}]\", 'abstract': '<p>Deep brain stimulation (DBS) is a technique commonly used both in clinical and fundamental neurosciences. Classically, brain stimulation requires an implanted and wired electrode system to deliver stimulation directly to the target area. Although techniques such as temporal interference (TI) can provide stimulation at depth without involving any implanted electrodes, these methods still rely on a wired apparatus which limits free movement. Herein we report organic photocapacitors as untethered light-driven electrodes which convert deep-red light into electric current. Pairs of these ultrathin devices can be driven using lasers at two different frequencies to deliver stimulation at depth via temporally interfering fields. We validate this concept of laser TI stimulation using numerical modeling, ex vivo tests with phantom samples, and finally in vivo tests. Wireless organic photocapacitors are placed on the cortex and elicit stimulation in the hippocampus, while not delivering off-target stimulation in the cortex. This laser-driven wireless TI evoked a neuronal response at depth that is comparable to control experiments induced with deep brain stimulation protocols using implanted electrodes. Our work shows that a combination of these two techniques, temporal interference and organic electrolytic photocapacitors, provides a reliable way to target brain structures requiring neither deeply implanted electrodes nor tethered stimulator devices. The laser TI protocol demonstrated here address two of the most important drawbacks in the field of deep brain stimulation and thus holds potential to solve many issues in freely-moving animal experiments or for clinical chronic therapy application.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466952', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Dynamic Flux Balance Analysis Game', 'authors': \"[{'name': 'Garud Iyengar'}, {'name': 'Mitchell Perry'}]\", 'abstract': '<p>Flux balance analysis (FBA) for microbial communities often assumes a global objective function that all species cooperatively maximize in addition to maximizing their own growth. Combining communityFBA with dynamic FBA to understand the time course and steady states of communities typically entails discretizing time and solving a community FBA model at each time point, a time-intensive process. We propose a dynamic community FBA model where species compete for metabolites to grow off of without needing to  cooperate to maximize a community-level objective. An efficient method for computing steady state community compositions is provided, as well as methods for determining the stability of a steady state community to perturbations in biomass and invasion by species outside the community. The model and methods are applied to a model of four E. Coli. mutants with elements of competition (for shared metabolites) and cooperation (via mutants being auxotrophic for metabolites exported by other mutants).</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466949', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Time-of-day influences on skill acquisition and consolidation after physical and mental practices', 'authors': \"[{'name': 'Charlene TRUONG'}, {'name': 'Pauline M HILT'}, {'name': 'Fatma BOUGUILA'}, {'name': 'Marco BOVE'}, {'name': 'Florent LEBON'}, {'name': 'Charalambos PAPAXANTHIS'}, {'name': 'Celia RUFFINO'}]\", 'abstract': '<p>Time-of-day influences both physical and mental performance. Its impact on motor learning is, however, not well established yet. Here, using a finger tapping-task, we investigated the time-of-day effect on skill acquisition (i.e., immediately after a physical or mental practice session) and consolidation (i.e., 24 hours later). Two groups (one physical and one mental) were trained in the morning (10 a.m.) and two others (one physical and one mental) in the afternoon (3 p.m.). We found an enhancement of motor skill following both types of practice, whatever the time of the day, with a better acquisition for the physical than the mental group. Interestingly, there was a better consolidation for both groups when the training session was scheduled in the afternoon. Overall, our results indicate that the time-of-day positively influences motor skill consolidation and thus must be considered to optimize training protocols in sport and clinical domains to potentiate motor (re)learning.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466940', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Towards a general framework for modeling large-scale biophysical neuronal networks: a full-scale computational model of the rat dentate gyrus', 'authors': \"[{'name': 'Ivan Georgiev Raikov'}, {'name': 'Aaron D Milstein'}, {'name': 'Prannath Moolchand'}, {'name': 'Gergely G Szabo'}, {'name': 'Calvin J Schneider'}, {'name': 'Darian H Hadjiabadi'}, {'name': 'Alexandra Pierri Chatzikalymniou'}, {'name': 'Ivan Soltesz'}]\", 'abstract': '<p>Large-scale computational models of the brain are necessary to accurately represent anatomical and functional variability in neuronal biophysics across brain regions and also to capture and study local and global interactions between neuronal populations on a behaviorally-relevant temporal scale. We present the methodology behind and an initial implementation of a novel open-source computational framework for construction, simulation, and analysis of models consisting of millions of neurons on high-performance computing systems, based on the NEURON and CoreNEURON simulators. This framework includes an HDF5-based data format for storing morphological, synaptic, and connectivity information of large neuronal network models, and an accompanying open source software library that provides efficient, scalable parallel storage and MPI-based data movement capabilities. We outline our approaches for constructing detailed large-scale biophysical models with topographical connectivity and input stimuli, and present simulation results obtained with a full-scale model of the dentate gyrus constructed with our framework.  The model generates sparse and spatially selective population activity that fits well with in-vivo experimental data. Moreover, our approach is fully general and can be applied to modeling other regions of the hippocampal formation in order to rapidly evaluate specific hypotheses about large-scale neural architectural features.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466821', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A deep exon cryptic splice site promotes aberrant intron retention in a von Willebrand disease patient', 'authors': \"[{'name': 'John G. Conboy'}]\", 'abstract': '<p>A translationally silent single nucleotide mutation, in exon 44 of the von Willebrand factor (VWF) gene, is associated with inefficient removal of intron 44 in a von Willebrand disease (VWD) patient. This intron retention (IR) event was previously attributed to altered secondary structure that sequesters the normal splice donor site. We propose an alternative mechanism: that the mutation introduces a cryptic splice donor site that interferes with function of the annotated site to favor IR. We evaluated both models using minigene splicing reporters engineered to vary in secondary structure and/or cryptic splice site content. Analysis of reporter splicing efficiency in transfected K562 cells suggested that the mutation-generated internal splice site was sufficient to induce substantial IR. Mutations predicted to vary secondary structure at the annotated site had modest effects on IR, and also shifted the balance of residual splicing between the cryptic site and annotated site, supporting competition between the sites. Further studies demonstrated that introduction of cryptic splice donor motifs at other positions in E44 did not promote IR, indicating that interference with the annotated site is context-dependent. We conclude that mutant deep exon splice sites can interfere with proper splicing by inducing IR.</p>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.02.21265826', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'The basic reproduction number of COVID-19 across Africa', 'authors': \"[{'name': 'Sarafa A. Iyaniwura'}, {'name': 'Musa Rabiu'}, {'name': 'Jummy F. David'}, {'name': 'Jude D. Kong'}]\", 'abstract': '<p>The pandemic of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) took the world by surprise. Following the first outbreak of COVID-19 in December 2019, several models have been developed to study and understand its transmission dynamics. Although the spread of COVID-19 is being slowed down by vaccination and other interventions, there is still a need to have a clear understanding of the evolution of the pandemic across countries, states and communities. To this end, there is a need to have a clearer picture of the initial spread of the disease in different regions. In this project, we used a simple SEIR model and a Bayesian inference framework to estimate the basic reproduction number of COVID-19 across Africa. Our estimates vary between 1.98 (Sudan) and 9.66 (Mauritius), with a median of 3.67 (90% CrI: 3.31 - 4.12). The estimates provided in this paper will help to inform COVID-19 modeling in the respective countries/regions.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466865', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Low selectivity index of ivermectin and macrocyclic lactones on SARS-CoV2 replication in vitro argues against their therapeutic use for COVID-19', 'authors': \"[{'name': 'Christine Chable-Bessia'}, {'name': 'Charlotte Boullé'}, {'name': 'Aymeric Neyret'}, {'name': 'Jitandrya Swain'}, {'name': 'Mathilde Hénaut'}, {'name': 'Peggy Merida'}, {'name': 'Nathalie Gros'}, {'name': 'Alain Makinson'}, {'name': 'Sébastien Lyonnais'}, {'name': 'Cédric B. Chesnais'}, {'name': 'Delphine Muriaux'}]\", 'abstract': '<p>There are very limited antiviral therapeutic options for coronavirus infections, therefore global drug re-purposing efforts are paramount to identify available compounds that could provide clinical benefits to patients with COVID-19. Ivermectin was first approved for human use as an endectocide in the 1980s. It remains one of the most important global health medicines in history and has recently been shown to exert <italic>in vitro</italic> activity against SARS-CoV-2. However, the macrocyclic lactone family of compounds has not previously been evaluated for activity against SARS-CoV-2. The present study aims at comparing their anti-viral activity in relevant pulmonary cell lines in vitro. Here, <italic>in vitro</italic> antiviral activity of the avermectins (ivermectin and selamectin) and milbemycins (moxidectin and milbemycin oxime) were assessed against a clinical isolate from a CHU Montpellier patient infected with SARS-CoV-2 in 2020. Ivermectin demonstrated anti-SARS-CoV-2 activity in vitro in human pulmonary cells in comparison to VeroE6 (with EC<sub>50</sub> of 1-3 μM). Similarly, the other macrocyclic lactones moxidectin, milbemycin oxime and selamectin reduced SARS-CoV-2 replication in vitro (with EC<sub>50</sub> of 2-5 μM). Immunofluorescence assays with ivermectin and moxidectin showed a reduction in the number of infected and polynuclear cells suggesting a drug action on viral cell fusion. However, cellular toxicity of the avermectins and milbemycins during infection showed a very low selectivity index &lt;10 for all compounds. In conclusion, none of these agents appears suitable for human use for its anti-SARS-CoV-2 activity <italic>per se</italic>, due to low selectivity index. This is discussed in regards to recent clinical COVID studies on ivermectin.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466861', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A critical role for E2-p53 interaction during the HPV16 life cycle', 'authors': \"[{'name': 'Christian T. Fontan'}, {'name': 'Claire D. James'}, {'name': 'Molly L. Bristol'}, {'name': 'Apurva T. Prabhakar'}, {'name': 'Raymonde Otoa'}, {'name': 'Xu Wang'}, {'name': 'Elmira Karimi'}, {'name': 'Pavithra Rajagopalan'}, {'name': 'Devraj Basu'}, {'name': 'Iain M. Morgan'}]\", 'abstract': '<p>Human papillomaviruses (HPV) are causative agents in ano-genital and oral cancers; HPV16 is the most prevalent type detected in human cancers. The HPV16 E6 protein targets p53 for proteasomal degradation to facilitate proliferation of the HPV16 infected cell. However, in HPV16 immortalized cells E6 is predominantly spliced (E6*) and unable to degrade p53. Here we demonstrate that human foreskin keratinocytes immortalized by HPV16 (HFK+HPV16), and HPV16 positive oropharyngeal cancers, retain significant expression of p53. In addition, p53 levels can be increased in HPV16+ head and neck cancer cell lines following treatment with cisplatin. Introduction of full-length E6 into HFK+HPV16 resulted in attenuation of cellular growth (in hTERT immortalized HFK, E6 expression promoted enhanced proliferation). An understudied interaction is that between E2 and p53 and we investigated whether this was important for the viral life cycle. We generated mutant genomes with E2 unable to interact with p53 resulting in profound phenotypes in primary HFK. The mutant induced hyper-proliferation, but an ultimate arrest of cell growth; β-galactosidase staining demonstrated increased senescence, and COMET assays showed increased DNA damage compared with HFK+HPV16 wild type cells. There was failure of the viral life cycle in organotypic rafts with the mutant HFK resulting in premature differentiation and reduced proliferation. The results indicate that the E2-p53 interaction is critical during the HPV16 life cycle, and that disruption of this interaction has anti-viral potential. We discuss potential mechanisms to explain these phenotypes.</p><sec><title>Importance</title><p>Human papillomaviruses are causative agents in around 5% of all cancers. There are currently no antivirals available to combat these infections and cancers, therefore it remains a priority to enhance our understanding of the HPV life cycle. Here we demonstrate that an interaction between the viral replication/transcription/segregation factor E2 and the tumor suppressor p53 is critical for the HPV16 life cycle. HPV16 immortalized cells retain significant expression of p53, and the critical role for the E2-p53 interaction demonstrates why this is the case. If the E2-p53 interaction is disrupted then HPV16 immortalized cells fail to proliferate, have enhanced DNA damage and senescence, and there is premature differentiation during the viral life cycle. Results suggest that targeting the E2-p53 interaction would have therapeutic benefits, potentially attenuating the spread of HPV16.</p></sec>', 'posted': '2021-11-02'}, {'doi': '10.1101/2021.11.01.466837', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Comparison of Oxidative and Hypoxic Stress Responsive Genes from Meta-Analysis of Public Transcriptomes', 'authors': \"[{'name': 'Takayuki Suzuki'}, {'name': 'Yoko Ono'}, {'name': 'Hidemasa Bono'}]\", 'abstract': '<p>Analysis of RNA-sequencing (RNA-seq) data is an effective means to analyze the gene expression levels under specific conditions and discover new biological knowledge. More than 74000 ex-perimental series with RNA-seq have been stored in public databases as of October 20, 2021. Since this huge amount of expression data accumulated from past studies is a promising source of new biological insights, we focused on a meta-analysis of 1783 runs of RNA-seq data under the conditions of two types of stresses: oxidative stress (OS) and hypoxia. The collected RNA-seq data of OS were organized as the OS dataset to retrieve and analyze differentially expressed genes (DEGs). The OS-induced DEGs were compared with the hypoxia-induced DEGs retrieved from a previous study. The results from the meta-analysis of OS transcriptomes revealed two genes, CRIP1 and CRIP3, which were particularly downregulated, suggesting a relationship be-tween OS and zinc homeostasis. The comparison between meta-analysis of OS and hypoxia showed that several genes were differentially expressed under both stress conditions, and it was inferred that the downregulation of cell cycle-related genes is a mutual biological process in both OS and hypoxia.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466776', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Coiled Coil Crosslinked Alginate Hydrogels Dampen Macrophage-Driven Inflammation', 'authors': '[{\\'name\\': \\'Zain Clapacs\\'}, {\\'name\\': \"Conor L O\\'Neill\"}, {\\'name\\': \\'Paresh Shrimali\\'}, {\\'name\\': \\'Giriraj Lokhande\\'}, {\\'name\\': \\'Megan Files\\'}, {\\'name\\': \\'Darren D Kim\\'}, {\\'name\\': \\'Akhilesh Gaharwar\\'}, {\\'name\\': \\'Jai S Rudra\\'}]', 'abstract': '<p>Alginate hydrogels are widely used for tissue engineering and regenerative medicine due to their excellent biocompatibility. A facile and commonly used strategy to crosslink alginate is the addition of Ca2+ that leads to hydrogelation. However, extracellular Ca2+ is a secondary messenger in activating inflammasome pathways following physical injury or pathogenic insult leading to persistent inflammation and scaffold rejection. Here we present graft copolymers of charge complementary heterodimeric coiled coil (CC) peptides and alginate that undergo supramolecular self-assembly to form Ca2+ free alginate hydrogels. The formation of heterodimeric CCs was confirmed using circular dichroism spectroscopy and scanning electron microscopy revealed a significant difference in pore size between Ca2+ and CC crosslinked gels. The resulting hydrogels were self-supporting and display shear-thinning and shear-recovery properties. In response to lipopolysaccharide (LPS) stimulation, peritoneal macrophages and bone marrow derived dendritic cells cultured in the CC crosslinked gels exhibited a 10-fold  reduction in secretion of the proinflammatory cytokine IL-1β compared to Ca2+ crosslinked gels. A similar respose was also observed in vivo upon peritoneal delivery of Ca2+ or CC crosslinked gels. Analysis of peritoneal lavage showed that macrophages in mice injected with Ca2+ crosslinked gels display a more inflammatory phenotype compared to macrophages from mice injected with CC crosslinked gels. These results suggest that CC peptides by virtue of their tunable sequence-structure-function relationship and mild gelation conditions are promising alternative crosslinkers for alginate and other biopolymer scaffolds used in tissue engineering.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466789', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Prioritising Autoimmunity Risk Variants for Functional Analyses by Fine-Mapping Mutations Under Natural Selection', 'authors': \"[{'name': 'Vasili Pankratov'}, {'name': 'Milyausha Yunusbaeva'}, {'name': 'Sergei Ryakhovsky'}, {'name': 'Maksym Zarodniuk'}, {'name': 'Bayazit Yunusbayev'}, {'name': ' '}]\", 'abstract': '<p>Pathogens imposed selective pressure on humans and shaped genetic variation in immunity genes. This can also be true for a fraction of causal variants implicated in chronic inflammatory disorders. Hence, locating adaptive mutations among candidate variants for these disorders can be a promising way to prioritize and decipher their functional response to microbial stimuli and contribution to pathogenesis. This idea has been discussed for decades, but challenges in locating adaptive SNPs hindered its application in practice. Our study addresses this issue and shows that a fraction of candidate variants for inflammatory conditions evolved under moderate and weak selection regimes (sweeps), and such variants are mappable. Using a novel powerful local-tree-based methodology, we show that 204 out of 593 risk loci for 21 autoimmune disorders contain at least one candidate SNP with strong evidence of selection. More importantly, in 28% of cases, these candidates for causal variants colocalize with SNPs under natural selection that we fine-mapped in this study. Causal SNPs under selection represent promising targets for functional experiments. Such experiments will help decipher molecular events triggered by infectious agents, a likely early event in autoimmunity. Finally, we show that a large fraction (60%) of candidate variants are either hitchhikers or linked with the selected mutation. Our findings, thus, support both hitchhiking and natural selection models, with the latter having important practical implications in medicine.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466801', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Estimated limits of organism-specific training for epitope prediction', 'authors': \"[{'name': 'Jodie Ashford'}, {'name': 'Felipe Campelo'}]\", 'abstract': '<p>Background: The identification of linear B-cell epitopes remains an important task in the development of vaccines, therapeutic antibodies and several diagnostic tests. Machine learning predictors are trained to flag potential epitope candidates for experimental validation and currently, most predictors are trained as generalist models using large, heterogeneous data sets. Recently, organism-specific training has been shown to improve prediction performance for data-rich organisms. Unfortunately, for most organisms, large volumes of validated epitope data are not yet available. This article investigates the limits of organism-specific training for epitope prediction. It explores the validity of organism-specific training for data-poor organisms by examining how the size of the training data set affects prediction performance. It also compares the performance of organism-specific training under simulated data-poor conditions to that of models trained using traditional large heterogeneous and hybrid data sets.\\nResults: This work shows how models trained on small organism-specific data sets can outperform  similar models trained on (potentially much larger) heterogeneous and mixed data sets. The results reported indicate that as few as 20 labelled peptides from a given pathogen can be sufficient to generate models that outperform widely-used predictors from the literature, which are trained on heterogeneous data. Models trained using more than about 100 to 150 organism-specific peptides perform consistently better than most generalist models across a wide variety of performance measures, and in some cases can even approach the performance of organism-specific models trained on considerably larger data sets.\\nConclusions: Organism-specific training improves linear B-cell epitope prediction performance even in situations when only small training sets are available, which opens new possibilities for the development of bespoke, high-performance predictive models when studying data-poor organisms such as emerging or neglected pathogens.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.462738', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Fungal biodiversity in Arctic paleoecosystems assessed by metabarcoding of lake sedimentary ancient DNA', 'authors': \"[{'name': 'Peter A. Seeber'}, {'name': 'Barbara von Hippel'}, {'name': 'Havard Kauserud'}, {'name': 'Ulrike Loeber'}, {'name': 'Kathleen Stoof-Leichsenring'}, {'name': 'Ulrike Herzschuh'}, {'name': 'Laura Epp'}]\", 'abstract': '<p>Fungi are crucial organisms in most ecosystems as they exert ecological key functions and are closely associated with land plants. Fungal community changes may therefore help reveal biodiversity changes in past ecosystems. Lake sediments contain DNA of organisms in the catchment area, which allows reconstructing past biodiversity by using metabarcoding of ancient sedimentary DNA. We developed a novel PCR primer combination for fungal metabarcoding targeting a short amplicon to account for length bias of amplification due to ancient DNA degradation. In-silico PCRs showed higher diversity using this primer combination than using previously established fungal metabarcoding primers. We analyzed existing data from sediment cores from four artic and one boreal lake in Siberia. These cores had been stored for 2-22 years and examined degradation effects of ancient DNA and storage time-related bias in fungal communities. Amplicon size differed between fungal divisions, however, we observed no significant effect of sample age on amplicon length and GC content, suggesting robust results. We also found no indication of post-coring fungal growth during storage distorting ancient fungal communities. Terrestrial soil fungi, including mycorrhizal fungi and saprotrophs, were predominant in all lakes, which supports the use of lake sedimentary ancient DNA for reconstructing terrestrial communities.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.466675', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Initial approach to the Conservation Genetics of the Guatemalan Beaded Lizard (Heloderma charlesbogerti): A route to genetic rescue', 'authors': \"[{'name': 'Sergio González'}, {'name': 'Thomas Schrei'}, {'name': 'Brad Lock'}]\", 'abstract': '<p>In this study, samples from 33 Guatemalan Beaded Lizard (<italic>Heloderma charlesbogerti</italic>) were analyzed for genetic diversity. Twenty-three samples were obtained from wild individuals from two separate population areas, and 10 samples were obtained from captive individuals. Because the seasonally dry tropical forest habitat sampled for this study, is degraded and fragmented, it was hypothesized that beaded lizard populations were small and isolated and would be subject to genetic erosion and an elevated extinction risk. To test this hypothesis, eight microsatellite markers were employed to analyze 22 individual samples from the population of Cabañas, Zacapa, a single individual from the eastern-most population and 10 captive individuals of unknown origin. An average of three alleles per maker was reported for the Cabañas population, evidencing a low genetic diversity. In addition, a recent bottleneck event was detected and an effective population size of 19.6 was estimated. Demographic reconstruction using a Bayesian approach was inconclusive possibly due to a small dataset and shallow coalescence trees obtained with the generated data. No clear structuring pattern was detected for the Cabañas population and most samples from individuals in captivity were found to have similar alleles to the ones from Cabañas. Population designation is challenging without the genotyping of every wild population, but unique alleles were found in captive individuals of unknown origin that could suggest that different genotypes might exist within other, less studied, wild populations. Low genetic diversity, and a small effective population size represent a risk for the Cabañas population facing the threats of isolation, habitat loss and climate change. These findings suggest that genetic management of the Cabañas population might be utilized to avoid high rates of inbreeding and subsequent inbreeding depression.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.465785', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Arp2/3 complex activity is necessary for mouse ESC differentiation, times formative pluripotency, and enables lineage specification', 'authors': \"[{'name': 'Francesca M. Aloisio'}, {'name': 'Diane L. Barber'}]\", 'abstract': '<p>Mouse embryonic stem cells (mESCs), a model for differentiation into primed epiblast-like cells (EpiLCs), have revealed transcriptional and epigenetic control of early embryonic development. The control and significance of morphological changes, however, remain less defined. We show marked changes in morphology and actin architectures during differentiation that depend on Arp2/3 complex but not formin activity. Inhibiting Arp2/3 complex activity pharmacologically or genetically does not block exit from naive pluripotency but attenuates increases in EpiLC markers. We find that inhibiting Arp2/3 complex activity delays formative pluripotency and causes globally defective lineage specification as indicated by RNA-sequencing, with significant effects on TBX3-depedendent transcriptional programs. We also identify two previously unreported indicators of mESC differentiation; MRTF and FHL2, which have inverse Arp2/3 complex-dependent nuclear translocation. Our findings on Arp2/3 complex activity in differentiation and the established role of formins in EMT indicate that these two actin nucleators regulate distinct modes of epithelial plasticity.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Arp2/3 complex activity is necessary for morphology changes during differentiation</p></list-item><list-item><p>Arp2/3 complex activity regulates transcriptional markers of differentiation</p></list-item><list-item><p>Inhibiting Arp2/3 complex activity delays entry into formative pluripotency</p></list-item><list-item><p>Arp2/3 complex activity-dependent shuttling of FHL2 and MRTF occurs in mESCs</p></list-item></list></sec>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.02.463467', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Comprehensive O-GlcNAc glycoproteomics on NOTCH1 EGF repeats refined sequons for O-GlcNAcylation and uncovered unique Lewis X epitopes in mammals', 'authors': \"[{'name': 'Yohei Tsukamoto'}, {'name': 'Mitsutaka Ogawa'}, {'name': 'Kentarou Yogi'}, {'name': 'Hideyuki Takeuchi'}, {'name': 'Tetsuya Okajima'}]\", 'abstract': '<p>The O-GlcNAc modification of Notch receptors regulates Notch ligand interactions in a manner distinct from other forms of O-glycans on epidermal growth factor-like (EGF) repeats of Notch receptors. Although many proteins, besides Notch receptors, are expected to be O-GlcNAcylated by EGF domain-specific O-GlcNAc transferase (EOGT), only a small number of proteins have been reported to be modified <italic>in vivo</italic>, and elongated O-GlcNAc glycans have not been extensively explored. To extend our view of the specificity and variety of the glycan modification, we conducted a comprehensive analysis of O-GlcNAc glycans on NOTCH1 in mammals. Mass spectrometric analysis of NOTCH1 fragments expressed in HEK293T cells revealed that several EGF domains with putative O-GlcNAcylation sites were hardly modified with O-GlcNAc. Although amino acid residues before the modification site are preferentially occupied with aromatic residues, Phe and Tyr are preferable to Trp for the apparent modification with O-GlcNAc. Furthermore, a minor form of fucosylated O-GlcNAc glycans was detected in a subset of EGF domains. Fucosylation of O-GlcNAc glycans was enhanced by <italic>FUT1, FUT2</italic>, or <italic>FUT9</italic> expression. The FUT9-dependent Lewis X epitope was confirmed by immunoblotting using an anti-Lewis X antibody. As expected from the similarity in the glycan structures, the Lexis X antigen was detected on O-fucose glycans. Notably, the Lewis X structure on O-glycans was identified in endogenous NOTCH1 isolated from MCF7 cells. Our results refined the putative consensus sequence for the EOGT-dependent extracellular O-GlcNAc modification in mammals and revealed the structural diversity of functional Notch O-glycans.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Comprehensive analysis of O-GlcNAc glycan on NOTCH1 refined O-GlcNAcylation sequons</p></list-item><list-item><p><italic>FUT1, FUT2</italic>, and <italic>FUT9</italic> modify O-GlcNAc glycans in the selective EGF domains of NOTCH1</p></list-item><list-item><p>The Lewis X epitope can be formed on both O-GlcNAc and O-fucose glycans on NOTCH1</p></list-item></list></sec><sec><title>In Brief</title><p>This study performed comprehensive mass spectrometric analyses of O-GlcNAc glycans in the NOTCH1 extracellular region in cultured mammalian cells. The results indicated that EOGT-dependent O-GlcNAcylation occurs on a limited number of EGF repeats harboring putative O-GlcNAcylation sites, refining the sequons for extracellular O-GlcNAc modification in mammals. Moreover, novel structures with fucosylated O-GlcNAc glycans were identified, and a similar structure was detectable on O-fucose glycans, extending the view of structural diversity of functional Notch O-glycans.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.01.466692', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Caste, sex, and parasitism influence brain plasticity in a social wasp', 'authors': \"[{'name': 'Kristine M. Gandia'}, {'name': 'Federico Cappa'}, {'name': 'David Baracchi'}, {'name': 'Mark. E. Hauber'}, {'name': 'Laura Beani'}, {'name': 'Floria M. K. Uy'}]\", 'abstract': '<p>Phenotypic plasticity is the capacity of a single genotype to exhibit different phenotypes, and can be an adaptive response to specific environmental and social conditions. Social insects are particularly well-suited to study plasticity, because the division of labor amongst females and the different life histories of males and females are associated with specific sensory needs. Here, we take advantage of the social wasp <italic>Polistes dominula</italic> to explore if brain plasticity is influenced by caste and sex, and the exploitation by the parasite <italic>Xenos vesparum</italic>. Within sexes, males had proportionally larger optic lobes, while females, regardless of caste, had larger antennal lobes, which is consistent with sensory needs of sex-specific life histories. Within castes, reproductive females had larger calyces, as predicted by their sensory needs for extensive within-colony interactions and forming winter aggregations, than workers who spend more time foraging for nest material and prey. Surprisingly, parasites had different effects on female and male hosts. Female workers were castrated and behaviorally manipulated by female or male parasites, but only showed moderate differences in relative allocation of different brain tissue compared to non-parasitized workers. In contrast, the testes and behavior of parasitized males were essentially unaffected, but they had smaller brains and greater relative volume of most sensory brain regions than non-parasitized males. Our results are consistent with caste and sex mediating brain plasticity in <italic>P. dominula</italic> and that the parasite’s manipulation can also drive differential allocation of brain regions depending on host sex.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.466906', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Anatomically-based skeleton kinetics and pose estimation in freely-moving rodents', 'authors': \"[{'name': 'Arne Monsees'}, {'name': 'Kay-Michael Voit'}, {'name': 'Damian J Wallace'}, {'name': 'Juergen Sawinski'}, {'name': 'Edyta Leks'}, {'name': 'Klaus Scheffler'}, {'name': 'Jakob H Macke'}, {'name': 'Jason ND Kerr'}]\", 'abstract': '<p>Forming a complete picture of the relationship between neural activity and body kinetics requires quantification of skeletal joint biomechanics during behavior. However, without detailed knowledge of the underlying skeletal motion, inferring joint kinetics from surface tracking approaches is difficult, especially for animals where the relationship between surface anatomy and skeleton changes during motion. Here we developed a videography-based method enabling detailed three-dimensional kinetic quantification of an anatomically defined skeleton in untethered freely-behaving animals. This skeleton-based model has been constrained by anatomical principles and joint motion limits and provided skeletal pose estimates for a range of rodent sizes, even when limbs were occluded. Model-inferred joint kinetics for both gait and gap-crossing behaviors were verified by direct measurement of limb placement, showing that complex decision making behaviors can be accurately reconstructed at the level of skeletal kinetics using our anatomically constrained model.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.10.29.466526', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Serial invasions can disrupt the time course of ecosystem recovery', 'authors': \"[{'name': 'Vadim A. Karatayev'}, {'name': 'Lars G. Rudstam'}, {'name': 'Alexander Y. Karatayev'}, {'name': 'Lyubov E. Burlakova'}, {'name': 'Boris V. Adamovich'}, {'name': 'Hanna A. Zhukava'}, {'name': 'Kristen T. Holeck'}, {'name': 'Amy Lee Hetherington'}, {'name': 'James R. Jackson'}, {'name': 'Christopher W. Hotaling'}, {'name': 'Tatyana V. Zhukova'}, {'name': 'Tamara M. Mikheyeva'}, {'name': 'Raisa Z. Kovalevskaya'}, {'name': 'Oleg A. Makarevich'}, {'name': 'Darya V. Kruk'}]\", 'abstract': '<p>The impacts of species invasions can subside or amplify over time as ecosystems “adapt” or additional invaders arrive. These long-term changes provide important insights into ecosystem dynamics. Yet studies of long-term dynamics are rare and often confound species impacts with coincident environmental change. We synthesize many-decade time-series across ecosystems to resolve shared changes in seven key features following invasion by quagga and zebra mussels, two widespread congeners that re-engineer and increasingly co-invade freshwaters. Six polymictic shallow lakes with long-term data sets reveal remarkably similar trends, with the strongest ecosystem impacts occurring within 5-10 years of zebra mussel invasion. Surprisingly, plankton communities then exhibited a partial, significant recovery. This recovery was absent, and impacts of initial invasion amplified, in lakes where quagga mussels outcompeted zebra mussels and more completely depleted phytoplankton. Thus, invasion impacts subside over time but can amplify with serial introductions of competing, even closely similar, taxa.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.467076', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Trehalose-6-phosphate links singlet oxygen-induced signalling with metabolic signalling in Chlamydomonas reinhardtii', 'authors': \"[{'name': 'Waeil Al Youssef'}, {'name': 'Regina Feil'}, {'name': 'Maureen Saint-Sorny'}, {'name': 'Xenie Johnson'}, {'name': 'John E. Lunn'}, {'name': 'Bernhard Grimm'}, {'name': 'Pawel Brzezowski'}]\", 'abstract': '<p>Singlet oxygen (<sup>1</sup>O<sub>2</sub>) induces retrograde signalling in chloroplasts. Using a novel mutant screen, we identified a mutation in the <italic>TREHALOSE-6-PHOSPHATE PHOSPHATASE 1</italic> (<italic>T6PP1</italic>) gene that results in accumulation of trehalose 6-phosphate, a reprogramming of cell metabolism, and impairment of <sup>1</sup>O<sub>2</sub>-induced retrograde signalling in <italic>Chlamydomonas reinhardtii</italic>. From transcriptomic analysis and metabolite profiling, we conclude that accumulation or deficiency of certain metabolites directly affect <sup>1</sup>O<sub>2</sub>-signalling. <sup>1</sup>O<sub>2</sub>-inducible <italic>GLUTATHIONE PEROXIDASE 5</italic> (<italic>GPX5</italic>) gene expression is suppressed by increased content of fumarate, an intermediate in the tricarboxylic acid cycle (TCA cycle) in mitochondria and dicarboxylate metabolism in the cytosol, while it is promoted by another TCA cycle intermediate, aconitate. Furthermore, genes encoding known essential components of chloroplast-to-nucleus <sup>1</sup>O<sub>2</sub>-signalling show decreased transcript levels in a <italic>t6pp1</italic> mutant, which can be rescued by exogenous application of aconitate. We demonstrate that chloroplast retrograde signalling involving <sup>1</sup>O<sub>2</sub> depends on mitochondrial and cytosolic processes and that the metabolic status of the cell determines the response to <sup>1</sup>O<sub>2</sub>.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.01.466780', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'A system for multiplexed selection of aptamers with exquisite specificity without counter-selection', 'authors': \"[{'name': 'Alex M. Yoshikawa'}, {'name': 'Leighton Wan'}, {'name': 'Liwei Zheng'}, {'name': 'Michael Eisenstein'}, {'name': 'H. Tom Soh'}]\", 'abstract': '<p>Aptamers have proven to be valuable tools for the detection of small molecules due to their remarkable ability to specifically discriminate between structurally similar molecules. Most aptamer selection efforts have relied on counter-selection to eliminate aptamers that exhibit unwanted cross-reactivity to interferents or structurally similar relatives to the target of interest. However, because the affinity and specificity characteristics of an aptamer library are fundamentally unknowable <italic>a priori</italic>, it is not possible to determine the optimal counter-selection parameters. As a result, counter-selection experiments require trial-and-error approaches that are inherently inefficient and may not result in aptamers with the best combination of affinity and specificity. In this work, we describe a high-throughput screening process for generating high-specificity aptamers to multiple targets in parallel, while also eliminating the need for counter-selection. We employ a platform based on a modified benchtop sequencer to conduct a massively-parallel aptamer screening process that enables the selection of highly-specific aptamers against multiple structurally similar molecules in a single experiment, without any counter-selection. As a demonstration, we have selected aptamers with high affinity and exquisite specificity for three structurally similar kynurenine metabolites that differ by a single hydroxyl group in a single selection experiment. This process can easily be adapted to other small-molecule analytes, and should greatly accelerate the development of aptamer reagents that achieve exquisite specificity for their target analytes.</p><sec><title>Significance statement</title><p>Aptamers offer the exciting potential to discriminate between structurally similar small molecules. However, generating such highly specific aptamers has been proven challenging using the conventional process of counter-selection. In this work, we describe a high-throughput screening platform that can characterize the specificity of millions of aptamers towards a group of structurally related molecules in a single experiment and generate exquisitely specific aptamers without any counter-selection. As exemplars, we generated aptamers with high affinity and specificity towards three structurally related kynurenine metabolites using our platform. Our platform can be readily adapted to other small molecule targets and should therefore accelerate the development of aptamer reagents with exquisite specificity.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.467069', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Biofabrication of development-inspired scaffolds for regeneration of the annulus fibrosus macro- and microarchitecture', 'authors': \"[{'name': 'Nadine Kluser'}, {'name': 'Christoph Sprecher'}, {'name': 'Gion Ursin Alig'}, {'name': 'Sonja Haeckel'}, {'name': 'Christoph E Albers'}, {'name': 'Mauro Alini'}, {'name': 'Sibylle Grad'}, {'name': 'David Eglin'}, {'name': 'Andrea J Vernengo'}]\", 'abstract': '<p>Annulus fibrosus (AF) tissue engineering is a promising strategy for repairing the degenerated intervertebral disc (IVD) and a research area that could benefit from improved tissue models to drive translation. AF tissue is composed of concentric layers of aligned collagen bundles arranged in an angle-ply pattern, an architecture which is challenging to recapitulate with current scaffold design strategies. In response to this need, we developed a strategy to print 3D scaffolds that induce cell and tissue organization into oriented patterns mimicking the AF. Polycaprolactone (PCL) was printed in an angle-ply macroarchitecture possessing microscale aligned topographical cues. The topography was achieved by extrusion through custom-designed printer nozzles which were either round or possessing circumferential sinusoidal peaks. Whereas the round nozzle produced extruded filaments with a slight uniaxial texture, patterned nozzles with peak heights of 60 or 120 μm produced grooves, 10.87 ± 3.09 μm or 17.77 ± 4.91 μm wide, respectively. Bone marrow derived mesenchymal stem cells (BM-MSCs) cultured on the scaffolds for four weeks exhibited similar degrees of alignment within ± 10 ° of the printing direction and upregulation of outer AF markers (COL1, COL12, SFRP, MKX, MCAM, SCX and TAGLN), with no statistically significant differences as a function of topography. Interestingly, the grooves generated by the patterned nozzles induced longitudinal end-to-end alignment of cells, capturing the arrangement of cells during fibrillogenesis. In contrast, topography produced from the round nozzle induced a continuous web of elongated cells without end-to-end alignment. Extracellular collagen I, decorin and fibromodulin were detected in patterns closely following cellular organization. Taken together, we present a single-step biofabrication strategy to induce anisotropic cellular alignments in x-, y-, and z-space, with potential application as an in vitro model for studying AF tissue morphogenesis and growth.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467049', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Meta-Analysis of cortical inhibitory interneurons markers landscape and their performances in scRNA-seq studies.', 'authors': \"[{'name': 'Lorenzo Martini'}, {'name': 'Roberta Bardini'}, {'name': 'Stefano Di Carlo'}]\", 'abstract': '<p>The mammalian cortex contains a great variety of neuronal cells. In particular, GABAergic interneurons, which play a major role in neuronal circuit function, exhibit an extraordinary diversity of cell types. In this regard, single-cell RNA-seq analysis is crucial to study cellular heterogeneity. To identify and analyze rare cell types, it is necessary to reliably label cells through known markers. In this way, all the related studies are dependent on the quality of the employed marker genes. Therefore, in this work, we investigate how a set of chosen inhibitory interneurons markers perform. The gene set consists of both immunohistochemistry-derived genes and single-cell RNA-seq taxonomy ones. We employed various human and mouse datasets of the brain cortex, consequently processed with the Monocle3 pipeline. We defined  metrics based on the relations between unsupervised cluster results and the marker expression. Specifically, we calculated the specificity, the fraction of cells expressing, and some metrics derived from decision tree analysis like entropy gain and impurity reduction. The results highlighted the strong reliability of some markers but also the low quality of others. More interestingly, though, a correlation emerges between the general performances of the genes set and the experimental quality of the datasets. Therefore, the proposed method allows evaluating the quality of a dataset in relation to its reliability regarding the inhibitory interneurons cellular heterogeneity study.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467146', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Intact Drosophila Whole Brain Cellular Quantitation reveals Sexual Dimorphism', 'authors': \"[{'name': 'Wei Jiao'}, {'name': 'Gard Spreemann'}, {'name': 'Evelyne Ruchti'}, {'name': 'Soumya Banerjee'}, {'name': 'Ying Shi'}, {'name': 'R. Steven Stowers'}, {'name': 'Kathryn Hess'}, {'name': 'Brian D. McCabe'}]\", 'abstract': '<p>Establishing with precision the quantity and identity of the cell types of the brain is a prerequisite for a detailed compendium of gene and protein expression in the central nervous system. Currently however, strict quantitation of cell numbers has been achieved only for the nervous system of C.elegans.   Here we describe the development of a synergistic pipeline of molecular genetic, imaging, and computational technologies designed to allow high-throughput, precise quantitation with cellular resolution of reporters of gene expression in intact whole tissues with complex cellular constitutions such as the brain.  We have deployed the approach to determine with exactitude the number of functional neurons and glia in the entire intact Drosophila larval brain, revealing fewer neurons and many more glial cells than previously estimated.   Moreover, we discover an unexpected divergence between the sexes at this juvenile developmental stage, with female brains having significantly more neurons than males.  Topological analysis of our data establishes that this sexual dimorphism extends to deeper features of brain organisation.   Our methodology enables robust and accurate quantification of the number and positioning of cells within intact organs, facilitating sophisticated analysis of cellular identity, diversity, and expression characteristics.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467145', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'A phosphoinositide and RAB switch controls early macropinocytosis', 'authors': \"[{'name': 'Hélène Spangenberg'}, {'name': 'Marte Sneeggen'}, {'name': 'Maria Mateo Tortola'}, {'name': 'Camila Valenzuela'}, {'name': 'Yuen-Yan Chang'}, {'name': 'Harald Stenmark'}, {'name': 'Camilla Raiborg'}, {'name': 'Kay Oliver Schink'}]\", 'abstract': '<p>Macropinocytosis is a non-selective endocytic process by which cells take up large amounts of extracellular fluids into giant vesicles known as macropinosomes. This mechanism is used by immune cells to sample the surroundings for antigens and can be exploited by cancer cells for nutrient uptake. What determines the fate of macropinosomes after they have been internalized is largely unknown. Here we investigate the role of the phosphatidylinositol 3-kinase VPS34/PIK3C3 and its product phosphatidylinositol 3-phosphate (PtdIns3P) in macropinosome fate determination. Inhibition of VPS34 led to a decrease in macropinosome survival and fluid phase uptake as well as preventing recruitment of early endosomal factors, including the small GTPase RAB5 and its effectors, to the forming macropinosomes. Instead, forming macropinosomes under VPS34 inhibition accumulated regulators of endocytic recycling, including RAB8A, RAB10, RAB11A, and PtdIns4P, which led to fusion of macropinosomes with the plasma membrane.</p><p>Whereas RAB5 was critical for macropinosome formation, macropinosome fusion with the plasma membrane depended on RAB8A. Thus, macropinosome maturation is regulated by a PtdIns3P-controlled switch that balances macropinosome fate between the default, endolysosomal maturation and an alternative, secretory route.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.467132', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Synergy of color and motion vision for detecting approaching objects in Drosophila', 'authors': \"[{'name': 'Kit D Longden'}, {'name': 'Edward M Rogers'}, {'name': 'Aljoscha Nern'}, {'name': 'Heather Dionne'}, {'name': 'Michael B Reiser'}]\", 'abstract': '<p>Color and motion are used by many species to identify salient moving objects. They are processed largely independently, but color contributes to motion processing in humans, for example, enabling moving colored objects to be detected when their luminance matches the background. Here, we demonstrate an unexpected, additional contribution of color to motion vision in Drosophila. We show that behavioral ON-motion responses are more sensitive to UV than for OFF-motion, and we identify cellular pathways connecting UV-sensitive R7 photoreceptors to ON and OFF-motion-sensitive T4 and T5 cells, using neurogenetics and calcium imaging. Remarkably, the synergy of color and motion vision enhances the detection of approaching UV discs, but not green discs with the same chromatic contrast, and we show how this generalizes for visual systems with ON and OFF pathways. Our results provide a computational and circuit basis for how color enhances motion vision to favor the detection of saliently colored objects.</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.10.29.466444', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Improving mathematical models of cancer by including resistance to therapy: a study in non-small cell lung cancer', 'authors': \"[{'name': 'Virginia Ardévol Martinez'}, {'name': 'Narmin Ghaffari Laleh'}, {'name': 'Monica Salvioli'}, {'name': 'Frank Thuijsman'}, {'name': 'Joel S. Brown'}, {'name': 'Rachel Cavill'}, {'name': 'Jakob Nikolas Kather'}, {'name': 'Kateřina Staňková'}]\", 'abstract': '<p>In this paper, a large dataset of 590 Non-Small Cell Lung Patients treated with either chemotherapy or immunotherapy was used to determine whether a game-theoretic model including both evolution of therapy resistance and cost of resistance provides a better fit than classical mathematical models of population growth (exponential, logistic, classic Bertalanffy, general Bertalanffy, Gompertz, general Gompertz). This is the first time a large clinical patient cohort (as opposed to only in-vitro data) has been used to apply a game-theoretic cancer model. The game-theoretic model provides a better fit to the tumor dynamics of the 590 Non-Small Cell Lung Cancer patients than any of the non-evolutionary population growth models. This is not simply due to having more parameters in the game-theoretic model. The game-theoretic model is able to fit accurately patients whose tumor burden exhibit a U-shaped trajectory over time. We then demonstrate how this game-theoretic model provides predictions of tumor growth based on just a few initial measurements. Assuming that treatment-specific parameters define the treatment impact completely, we then explore alternative treatment protocols and their impact on the tumor growth. As such, the model can be used to suggest patient-specific optimal treatment regimens with the goal of minimizing final tumor burden. Therapeutic protocols based on game-theoretic modeling can predict tumor growth, and improve patient outcome. The model invites evolutionary therapies that anticipate and steer the evolution of therapy resistance.</p>', 'posted': '2021-11-01'}, {'doi': '10.1101/2021.11.03.21265854', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': False, 'title': 'WOMEN’S BIRTH PREPAREDNESS AND COMPLICATION READINESS IN NIGERIA: A SYSTEMATIC REVIEW AND META-ANALYSIS', 'authors': \"[{'name': 'Margaret O. Akinwaare'}, {'name': 'Abimbola Oluwatosin'}, {'name': 'Olakekan Uthman'}, {'name': 'Elizabeth Ike'}]\", 'abstract': '<sec><title>Introduction</title><p>Globally, efforts are being made to reduce the menace of maternal death in order to achieve the sustainable development goal. Maternal death has been associated with inadequate birth preparedness and complication readiness especially in the low and middle-income countries. Therefore, this review assessed birth preparedness and complication readiness in Nigeria.</p></sec><sec><title>Methods</title><p>A systematic review and meta-analysis of published research articles on birth preparedness and complication readiness in Nigeria was done using PubMed, EMBASE and MEDLINE databases. All published articles from inception to November, 2018 were included in the review. A total of 8913 published articles were identified from electronic search, a total of 4440 studies were included in this review, while only 12 articles met the inclusion criteria and were included in the meta-analysis.</p></sec><sec><title>Results</title><p>The pooled prevalence of ‘Good BPCR’ for all studies yielded an estimate of 58.7% (95% CI 43.9 to 72.7%). The <italic>I</italic><sup><italic>2</italic></sup> statistic was 98%, indicating statistically significant heterogeneity among the studies. The percentage of women with good birth preparedness and complication readiness increases with increasing year of publication, such that women tended to be more aware good birth preparedness and complication readiness in recent years. More than half of the women had knowledge of obstetric danger signs (52.0%, 95% CI 39.5 to 64.4%, 10 studies), arranged for transportation (59.5%, 95% CI 36.2 to 80.7, 11 studies) or saved money (63.4%, 95% CI 44.7 to 80.2%, 11 studies) as part of the BPCR.</p></sec><sec><title>Conclusion</title><p>Women in Nigeria are better prepared for birth preparedness and complication readiness in recent years. Therefore, interventions to promote more adequate birth preparedness and complication readiness among women is recommended.</p></sec><sec><title>PROSPERO registration number</title><p>The study protocol was registered with PROSPERO number, CRD42019123220</p></sec><sec><title>Key questions</title><sec><title>What is already known?</title><p>Globally, birth preparedness and complication readiness is a concept which has been proven to be effective at increasing skilled birth attendance. It is a strategy for reducing maternal death especially in middle and lo-income countries.</p></sec><sec><title>What are the new findings?</title><list list-type=\"bullet\"><list-item><p>More pregnant women and recently delivered mothers in the Southern part of Nigeria have a better birth preparedness and complication readiness compared to their counterpart in the Northern part of the country.</p></list-item><list-item><p>Number of pregnant women and recently delivered mothers with good birth preparedness and complication readiness increases with increasing year of publication.</p></list-item></list></sec><sec><title>What do the new findings imply?</title><list list-type=\"bullet\"><list-item><p>This review provides data about women’s birth preparedness and complication readiness across different regions of Nigeria. Having a poorer BPCR in the Northern region of the country could be responsible for higher maternal death in the region.</p></list-item><list-item><p>This could be used to plan interventions to improve birth preparedness and complication readiness in different regions of the country.</p></list-item></list></sec><sec><title>Data availability statement</title><p>No data are available.</p></sec></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.21265685', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Hypothyroidism does not lead to worse prognosis in COVID-19: findings from the Brazilian COVID-19 registry', 'authors': \"[{'name': 'Daniella Nunes Pereira'}, {'name': 'Leticia Ferreira Gontijo Silveira'}, {'name': 'Milena Maria Moreira Guimarães'}, {'name': 'Carísi Anne Polanczyk'}, {'name': 'Aline Gabrielle Sousa Nunes'}, {'name': 'André Soares de Moura Costa'}, {'name': 'Barbara Lopes Farace'}, {'name': 'Christiane Corrêa Rodrigues Cimini'}, {'name': 'Cíntia Alcantara de Carvalho'}, {'name': 'Daniela Ponce'}, {'name': 'Eliane Würdig Roesch'}, {'name': 'Euler Roberto Fernandes Manenti'}, {'name': 'Fernanda Barbosa Lucas'}, {'name': 'Fernanda d’Athayde Rodrigues'}, {'name': 'Fernando Anschau'}, {'name': 'Fernando Graça Aranha'}, {'name': 'Frederico Bartolazzi'}, {'name': 'Giovanna Grunewald Vietta'}, {'name': 'Guilherme Fagundes Nascimento'}, {'name': 'Helena Duani'}, {'name': 'Heloisa Reniers Vianna'}, {'name': 'Henrique Cerqueira Guimarães'}, {'name': 'Jamille Hemétrio Salles Martins Costa'}, {'name': 'Joanna d’Arc Lyra Batista'}, {'name': 'Joice Coutinho de Alvarenga'}, {'name': 'José Miguel Chatkin'}, {'name': 'Júlia Drumond Parreiras de Morais'}, {'name': 'Juliana Machado-Rugolo'}, {'name': 'Karen Brasil Ruschel'}, {'name': 'Lílian Santos Pinheiro'}, {'name': 'Luanna Silva Monteiro Menezes'}, {'name': 'Luciana Siuves Ferreira Couto'}, {'name': 'Luciane Kopittke'}, {'name': 'Luís César de Castro'}, {'name': 'Luiz Antônio Nasi'}, {'name': 'Máderson Alvares de Souza Cabral'}, {'name': 'Maiara Anschau Floriani'}, {'name': 'Maíra Dias Souza'}, {'name': 'Marcelo Carneiro'}, {'name': 'Maria Aparecida Camargos Bicalho'}, {'name': 'Mariana Frizzo de Godoy'}, {'name': 'Matheus Carvalho Alves Nogueira'}, {'name': 'Milton Henriques Guimarães Júnior'}, {'name': 'Natália da Cunha Severino Sampaio'}, {'name': 'Neimy Ramos de Oliveira'}, {'name': 'Pedro Ledic Assaf'}, {'name': 'Renan Goulart Finger'}, {'name': 'Roberta Xavier Campos'}, {'name': 'Rochele Mosmann Menezes'}, {'name': 'Saionara Cristina Francisco'}, {'name': 'Samuel Penchel Alvarenga'}, {'name': 'Silvana Mangeon Mereilles Guimarães'}, {'name': 'Silvia Ferreira Araújo'}, {'name': 'Talita Fischer Oliveira'}, {'name': 'Thulio Henrique Oliveira Diniz'}, {'name': 'Yuri Carlotto Ramires'}, {'name': 'Evelin Paola de Almeida Cenci'}, {'name': 'Thainara Conceição de Oliveira'}, {'name': 'Alexandre Vargas Schwarzbold'}, {'name': 'Patricia Klarmann Ziegelmann'}, {'name': 'Roberta Pozza'}, {'name': 'Magda Carvalho Pires'}, {'name': 'Milena Soriano Marcolino'}]\", 'abstract': '<sec><title>Background</title><p>It is not clear whether previous thyroid diseases influence the course and outcomes of COVID-19. The study aims to compare clinical characteristics and outcomes of COVID-19 patients with and without hypothyroidism.</p></sec><sec><title>Methods</title><p>The study is a part of a multicentric cohort of patients with confirmed COVID-19 diagnosis, including data collected from 37 hospitals. Matching for age, sex, number of comorbidities and hospital was performed to select the patients without hypothyroidism for the paired analysis.</p></sec><sec><title>Results</title><p>From 7,762 COVID-19 patients, 526 had previously diagnosed hypothyroidism (50%) and 526 were selected as matched controls. The median age was 70 (interquartile range 59.0-80.0) years-old and 68.3% were females. The prevalence of underlying comorbidities were similar between groups, except for coronary and chronic kidney diseases, that had a higher prevalence in the hypothyroidism group (9.7% vs. 5.7%, p=0.015 and 9.9% vs. 4.8%, p=0.001, respectively). At hospital presentation, patients with hypothyroidism had a lower frequency of respiratory rate &gt; 24 breaths per minute (36.1% vs 42.0%; p=0.050) and need of mechanical ventilation (4.0% vs 7.4%; p=0.016). D-dimer levels were slightly lower in hypothyroid patients (2.3 times higher than the reference value vs 2.9 times higher; p=0.037). In-hospital management was similar between groups, but hospital length-of-stay (8 vs 9 days; p=0.029) and mechanical ventilation requirement (25.4% vs. 33.1%; p=0.006) were lower for patients with hypothyroidism. There was a trend of lower in-hospital mortality in patients with hypothyroidism (22.1% vs. 27.0%; p=0.062).</p></sec><sec><title>Conclusion</title><p>In this large Brazilian COVID-19 Registry, patients with hypothyroidism had a lower requirement of mechanical ventilation, and showed a trend of lower in-hospital mortality. Therefore, hypothyroidism does not seem to be associated with a worse prognosis, and should not be considered among the comorbidities that indicate a risk factor for COVID-19 severity.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.03.21265077', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Oncology', 'covid': False, 'title': 'Predicting Melanoma Staging using Targeted RNA Sequencing data using Machine Learning.', 'authors': \"[{'name': 'Fahad Shabbir Ahmed'}, {'name': 'Furqan Bin Irfan'}]\", 'abstract': '<p>The aim of this study is to use machine learning to predict tumor staging and metastasis in melanoma with differentially expressed genes. Machine has been used in different clinical setting to predict different outcomes. However, it has not been used to look at predicting the diagnostic aspect of tumor staging. We used the TCGA RNA-Sequencing data on melanomas to predict tumor staging nodal and/or metastasis using deep neural networks (DNN) and random forest classifier (RF). Results: We were able to predict tumor staging (lower vs higher stage, i.e. Tis / T1 / T2 vs T3 and higher), nodal metastasis and combined nodal or distant metastasis in patients with melanomas with high accuracies. However, we need to further validate these results.</p>', 'posted': '2021-11-05'}, {'doi': '10.1101/2021.11.03.21260184', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Allergy and Immunology', 'covid': True, 'title': 'Whole-Blood DNA Methylation Analysis Reveals Respiratory Environmental Traits Involved in COVID-19 Severity Following SARS-CoV-2 Infection', 'authors': \"[{'name': 'Guillermo Barturen'}, {'name': 'Elena Carnero-Montoro'}, {'name': 'Manuel Martínez-Bueno'}, {'name': 'Silvia Rojo-Rello'}, {'name': 'Beatriz Sobrino'}, {'name': 'Clara Alcántara-Domínguez'}, {'name': 'David Bernardo'}, {'name': 'Marta E. Alarcón-Riquelme'}]\", 'abstract': '<p>SARS-CoV-2 causes a severe inflammatory syndrome (COVID-19) leading, in many cases, to bilateral pneumonia, severe dyspnea and in ∼5% of these, death. DNA methylation is known to play an important role in the regulation of the immune processes behind COVID-19 progression, however it has not been studied in depth, yet. In this study, we aim to evaluate the implication of DNA methylation in COVID-19 progression by means of a genome-wide DNA methylation analysis combined with DNA genotyping.</p><p>The results reveal the existence of epigenomic regulation of functional pathways associated with COVID-19 progression and mediated by genetic loci. We found an environmental trait-related signature that discriminates mild from severe cases, and regulates IL-6 expression via the transcription factor CEBP. The analyses suggest that an interaction between environmental contribution, genetics and epigenetics might be playing a role in triggering the cytokine storm described in the most severe cases.</p>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.467039', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Inhibition of BET family proteins suppresses African swine fever virus infection', 'authors': \"[{'name': 'Yaru Zhao'}, {'name': 'Qingli Niu'}, {'name': 'Saixia Yang'}, {'name': 'Jifei Yang'}, {'name': 'Zhonghui Zhang'}, {'name': 'Shuxian Geng'}, {'name': 'Jie Fan'}, {'name': 'Zhijie Liu'}, {'name': 'Guiquan Guan'}, {'name': 'Zhiqing Liu'}, {'name': 'Jia Zhou'}, {'name': 'Haitao Hu'}, {'name': 'Jianxun Luo'}, {'name': 'Hong Yin'}]\", 'abstract': '<p>African swine fever (ASF), an acute, severe, highly contagious disease caused by African swine fever virus (ASFV) infection in domestic pigs and boars, has a mortality rate of up to 100%. Because effective vaccines and treatments for ASF are lacking, effective control of the spread of ASF remains a great challenge for the pig industry. Host epigenetic regulation is essential for the viral gene transcription. Bromodomain and extraterminal (BET) family proteins, including BRD2, BRD3, BRD4, and BRDT, are epigenetic “readers” critical for gene transcription regulation. Among these proteins, BRD4 recognizes acetylated histones via its two bromodomains (BD1 and BD2) and recruits transcription factors, thereby playing a pivotal role in transcriptional regulation and chromatin remodeling during viral infection. However, how BET/BRD4 regulates ASFV replication and gene transcription is unknown. Here, we randomly selected 12 representative BET family inhibitors and compared their effects on ASFV infection in pig’s primary alveolar macrophages (PAMs). They were found to inhibit viral infection by interfering with the different stages of viral life cycle (attachment, internalization, desencapsidation and formation of viral factories). The four most effective inhibitors (ARV-825, ZL0580, I-BET-762 and PLX51107) were selected for further antiviral activity analysis. These BET/BRD4 inhibitors dose-dependently decreased the ASFV titer, viral RNA transcription and protein production in PAMs. Collectively,our study reported novel activity of BET/BRD4 inhibitors in inducing suppression of ASFV infection, providing insights into role of BET/BRD4 in epigenetic regulation of ASFV and potential new strategies for ASF prevention and control.</p><sec><title>IMPORTANCE</title><p>Since the continuing spread of the ASFV in the world, and lack of commercial vaccines, the development of improved control strategies including antiviral drugs are urgently needed. BRD4 is an important epigenetic factor and has been commonly used for drug development for tumor treatment. Furthermore, the latest research showed that BET/BRD4 inhibition could suppress replication of virus. In this study, we first showed the inhibitory effect of agents targeting BET/BRD4 on ASFV infection with no significant host cytotoxicity. Then, we found 4 BET/BRD4 inhibitors which can inhibit ASFV replication, RNA transcription and protein synthesis. Finally, we analyzed 4 inhibitors’ biological effect on BRD4 according to the structure of BRD4, and docking analysis of BET-762, PLX51107, ARV-825 and ZL0580 binding to BD1 and BD2 domains of BRD4 was performed. Our findings support the hypothesis that BET/BRD4 can be considered as attractive host targets in antiviral drug discovery against ASFV.</p></sec>', 'posted': '2021-11-03'}, {'doi': '10.1101/2021.11.02.467030', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'SPCS: A Spatial and Pattern Combined Smoothing Method of Spatial Transcriptomic Expression', 'authors': \"[{'name': 'Yusong Liu'}, {'name': 'Tongxin Wang'}, {'name': 'Ben Duggan'}, {'name': 'Kun Huang'}, {'name': 'Jie Zhang'}, {'name': 'Xiufen Ye'}, {'name': 'Travis S. Johnson'}]\", 'abstract': '<p>The recently developed spatial transcriptomics (ST) technique has made it possible to view spatial transcriptional heterogeneity in a high throughput manner. It is based on highly multiplexed sequence analysis and uses barcodes to split the sequenced reads into respective tissue locations. However, this type of sequencing technique suffers from high noise and drop-out events in the data, which makes smoothing a necessary step before performing downstream analysis. Traditional smoothing methods used in the similar single cell RNA sequencing (scRNA-seq) data are one-factor methods that can only utilize associations in transcriptome space. Since they do not account for associations in the Euclidean space, i.e. tissue location distances on the ST slide, these one-factor methods cannot take full advantage of all the knowledge in ST data. In this study, we present a novel two-factor smoothing technique, Spatial and Pattern Combined Smoothing (SPCS), that employs k-nearest neighbor technique to utilize associations from transcriptome and Euclidean space from the ST data. By performing SPCS on 10 ST slides from pancreatic ductal adenocarcinoma (PDAC), smoothed ST slides have better separability, partition accuracy, and biological interpretability than the ones smoothed by pre-existing one-factor smoothing methods. Source code of SPCS is provided in Github (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/Usos/SPCS\">https://github.com/Usos/SPCS</ext-link>).</p>', 'posted': '2021-11-04'}, {'doi': '10.1101/2021.11.03.466767', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'SorCS2 dynamically interacts with TrkB and GluN2B to control neurotransmission and Huntington’s disease progression', 'authors': \"[{'name': 'Alena Salašová'}, {'name': 'Niels Sanderhoff Degn'}, {'name': 'Mikhail Paveliev'}, {'name': 'Niels Kjærgaard Madsen'}, {'name': 'Saray López Benito'}, {'name': 'Plinio Casarotto'}, {'name': 'Peter Lund Ovesen'}, {'name': 'Benedicte Vestergaard'}, {'name': 'Andreea Cornelia Udrea'}, {'name': 'Lilian Kisiswa'}, {'name': 'Lucie Woloszczuková'}, {'name': 'Islam Faress'}, {'name': 'Sadegh Nabavi'}, {'name': 'Eero Castrén'}, {'name': 'Juan Carlos Arévalo'}, {'name': 'Mai Marie Holm'}, {'name': 'Mads Fuglsang Kjølby'}, {'name': 'Ulrik Bølcho'}, {'name': 'Anders Nykjaer'}]\", 'abstract': '<p>Huntington’s disease (HD) is a fatal neurodegenerative disorder characterized by progressive motor dysfunction and loss of medium spiny neurons (MSNs) in dorsal striatum. Brain-derived neurotrophic factor (BDNF) sustains functionality and integrity of MSNs, and thus reduced BDNF signaling is integral to the disease. Here we show that SorCS2 is expressed in MSNs with reduced expression in R6/1 HD model, and that SorCS2 deficiency exacerbates the disease progression in R6/1 mice. Furthermore, we find that SorCS2 binds TrkB and the NMDA receptor subunit GluN2B, which is required to control neurotransmission in corticostriatal synapses. While BDNF stimulates SorCS2-TrkB complex formation to enable TrkB signaling, it disengages SorCS2 from GluN2B, leading to enrichment of the subunit at postsynaptic densities. Consequently, long-term potentiation (LTP) is abolished in SorCS2 deficient mice, despite increased striatal TrkB and unaltered BDNF expression. In contrast, the addition of exogenous BDNF rescues the phenotype. Finally, GluN2B, but not GluN2A, currents are also severely impaired in the SorCS2 KO mice. To conclude, we uncovered that SorCS2 dynamically targets TrkB and GluN2B to orchestrate BDNF-dependent plasticity in MSNs of dorsal striatum. We propose that SorCS2 deficiency impairs MSN function thereby increasing neuronal vulnerability and accelerating the motor deficits in Huntington’s disease.</p>', 'posted': '2021-11-03'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.11.05.467528</td>\n",
       "      <td>307</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioengineering</td>\n",
       "      <td>False</td>\n",
       "      <td>Large-scale discovery of recombinases for inte...</td>\n",
       "      <td>[{'name': 'Matthew G Durrant'}, {'name': 'Alis...</td>\n",
       "      <td>&lt;p&gt;Recent microbial genome sequencing efforts ...</td>\n",
       "      <td>2021-11-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.10.31.21265703</td>\n",
       "      <td>176</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Efficacy and safety of SOBERANA 02, a COVID-19...</td>\n",
       "      <td>[{'name': 'Maria Eugenia Toledo-Romani'}, {'na...</td>\n",
       "      <td>&lt;p&gt;Background: SOBERANA 02 is a COVID19 conjug...</td>\n",
       "      <td>2021-11-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.11.03.467152</td>\n",
       "      <td>146</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>DeepBacs: Bacterial image analysis using open-...</td>\n",
       "      <td>[{'name': 'Christoph Spahn'}, {'name': 'Romain...</td>\n",
       "      <td>&lt;p&gt;Deep Learning (DL) is rapidly changing the ...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.11.02.466936</td>\n",
       "      <td>113</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Bright, fluorogenic and photostable avidity pr...</td>\n",
       "      <td>[{'name': 'Bastian Bühler'}, {'name': 'Anja Be...</td>\n",
       "      <td>&lt;p&gt;Fluorescent light-up aptamers (FLAPs) emerg...</td>\n",
       "      <td>2021-11-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.11.02.466983</td>\n",
       "      <td>95</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Structure Determination of Inactive-State GPCR...</td>\n",
       "      <td>[{'name': 'Michael J. Robertson'}, {'name': 'F...</td>\n",
       "      <td>&lt;p&gt;Cryogenic electron microscopy (cryo-EM) has...</td>\n",
       "      <td>2021-11-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.11.01.466765</td>\n",
       "      <td>88</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Global patterns and rates of habitat transitio...</td>\n",
       "      <td>[{'name': 'Mahwash Jamy'}, {'name': 'Charlie B...</td>\n",
       "      <td>&lt;p&gt;The successful colonisation of new habitats...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.11.03.467131</td>\n",
       "      <td>68</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>A molecular network of conserved factors keeps...</td>\n",
       "      <td>[{'name': 'Friederike Leesch'}, {'name': 'Laur...</td>\n",
       "      <td>&lt;p&gt;Ribosomes are produced in large quantities ...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.11.04.467077</td>\n",
       "      <td>68</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>The oral protease inhibitor (PF-07321332) prot...</td>\n",
       "      <td>[{'name': 'Rana Abdelnabi'}, {'name': 'Carolin...</td>\n",
       "      <td>&lt;p&gt;There is an urgent need for potent and sele...</td>\n",
       "      <td>2021-11-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.11.02.466915</td>\n",
       "      <td>58</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>RBBP6 activates the pre-mRNA 3’-end processing...</td>\n",
       "      <td>[{'name': 'Vytaute Boreikaite'}, {'name': 'Tho...</td>\n",
       "      <td>&lt;p&gt;3’-end processing of most human mRNAs is ca...</td>\n",
       "      <td>2021-11-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.11.03.467079</td>\n",
       "      <td>53</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Parent-of-origin effects in the UK Biobank</td>\n",
       "      <td>[{'name': 'Robin J Hofmeister'}, {'name': 'Sim...</td>\n",
       "      <td>&lt;p&gt;Identical genetic variations can have diffe...</td>\n",
       "      <td>2021-11-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.11.03.467097</td>\n",
       "      <td>44</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>A drug repurposing screen identifies altiratin...</td>\n",
       "      <td>[{'name': 'Christopher Swale'}, {'name': 'Vale...</td>\n",
       "      <td>&lt;p&gt;The apicomplexa comprise a large phylum of ...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.11.01.466728</td>\n",
       "      <td>43</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Haldane’s cost of selection imposes a mild con...</td>\n",
       "      <td>[{'name': 'Joseph Matheson'}, {'name': 'Moises...</td>\n",
       "      <td>&lt;p&gt;Haldane’s Dilemma refers to the concern tha...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.11.05.467435</td>\n",
       "      <td>43</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Ancient genomes from the last three millennia ...</td>\n",
       "      <td>[{'name': 'Sandra Oliveira'}, {'name': 'Kathri...</td>\n",
       "      <td>&lt;p&gt;Previous research indicates that the human ...</td>\n",
       "      <td>2021-11-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.11.01.466817</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A powerful and versatile new fixation protocol...</td>\n",
       "      <td>[{'name': 'Carlos Guerrero-Hernández'}, {'name...</td>\n",
       "      <td>&lt;p&gt;Whole-mount &lt;italic&gt;in situ&lt;/italic&gt; hybrid...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.11.03.466924</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Organization and composition of apicomplexan k...</td>\n",
       "      <td>[{'name': 'Lorenzo Brusini'}, {'name': 'Nicola...</td>\n",
       "      <td>&lt;p&gt;Kinetochores are multiprotein assemblies di...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.11.03.465409</td>\n",
       "      <td>35</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Structural insights into human excitatory amin...</td>\n",
       "      <td>[{'name': 'Takafumi Kato'}, {'name': 'Tsukasa ...</td>\n",
       "      <td>&lt;p&gt;Glutamate is a pivotal excitatory neurotran...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.11.02.467002</td>\n",
       "      <td>34</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Detection of velogenic avian paramyxoviruses i...</td>\n",
       "      <td>[{'name': 'Isabel Francisco'}, {'name': 'Shato...</td>\n",
       "      <td>&lt;p&gt;Avian paramyxovirus 1 (APMV-1), also known ...</td>\n",
       "      <td>2021-11-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.10.29.466453</td>\n",
       "      <td>33</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Invariant neural subspaces maintained by feedb...</td>\n",
       "      <td>[{'name': 'Laura Bella Naumann'}, {'name': 'Jo...</td>\n",
       "      <td>&lt;p&gt;Sensory systems reliably process incoming s...</td>\n",
       "      <td>2021-11-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.11.03.466369</td>\n",
       "      <td>31</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Structure of mammalian V-ATPase with the TLDc ...</td>\n",
       "      <td>[{'name': 'Yong Zi Tan'}, {'name': 'Yazan M. A...</td>\n",
       "      <td>&lt;p&gt;V-ATPases are rotary proton pumps that serv...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.11.05.467437</td>\n",
       "      <td>30</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>PHOSPHATE STARVATION RESPONSE enables arbuscul...</td>\n",
       "      <td>[{'name': 'Debatosh Das'}, {'name': 'Michael P...</td>\n",
       "      <td>&lt;p&gt;Arbuscular mycorrhiza (AM) is a widespread ...</td>\n",
       "      <td>2021-11-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.11.04.466897</td>\n",
       "      <td>30</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Genome-wide association analyses of individual...</td>\n",
       "      <td>[{'name': 'Else Eising'}, {'name': 'Nazanin Mi...</td>\n",
       "      <td>&lt;p&gt;The use of spoken and written language is a...</td>\n",
       "      <td>2021-11-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.10.29.466434</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Multilevel neural gradients reflect transdiagn...</td>\n",
       "      <td>[{'name': 'Bo-yong Park'}, {'name': 'Valeria K...</td>\n",
       "      <td>&lt;p&gt;It is increasingly recognized that multiple...</td>\n",
       "      <td>2021-11-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.11.05.467534</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Kaptive 2.0: updated capsule and LPS locus typ...</td>\n",
       "      <td>[{'name': 'Margaret M.C. Lam'}, {'name': 'Ryan...</td>\n",
       "      <td>&lt;p&gt;The outer polysaccharide capsule and lipopo...</td>\n",
       "      <td>2021-11-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.11.05.467388</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>A temporally resolved, multiplex molecular rec...</td>\n",
       "      <td>[{'name': 'Junhong Choi'}, {'name': 'Wei Chen'...</td>\n",
       "      <td>&lt;p&gt;DNA is naturally well-suited to serve as a ...</td>\n",
       "      <td>2021-11-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.10.29.466512</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Animal Behavior and Cognition</td>\n",
       "      <td>False</td>\n",
       "      <td>Sex and Estrous Cycle in Memory for Sequences ...</td>\n",
       "      <td>[{'name': 'M. Jayachandran'}, {'name': 'P. Lan...</td>\n",
       "      <td>&lt;p&gt;The ability to remember sequences of events...</td>\n",
       "      <td>2021-11-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.11.02.464583</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Marker-free coselection for successive rounds ...</td>\n",
       "      <td>[{'name': 'Sébastien Levesque'}, {'name': 'Dia...</td>\n",
       "      <td>&lt;p&gt;Prime editing enables the introduction of p...</td>\n",
       "      <td>2021-11-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.11.01.466790</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Programmable large DNA deletion, replacement, ...</td>\n",
       "      <td>[{'name': 'Andrew V. Anzalone'}, {'name': 'Xin...</td>\n",
       "      <td>&lt;p&gt;The targeted deletion, replacement, integra...</td>\n",
       "      <td>2021-11-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.11.02.466884</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Retinal OFF ganglion cells allow detection of ...</td>\n",
       "      <td>[{'name': 'Johan Westö'}, {'name': 'Nataliia M...</td>\n",
       "      <td>&lt;p&gt;Perception of light in darkness requires no...</td>\n",
       "      <td>2021-11-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.11.01.466363</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Distinct representation of cue-outcome associa...</td>\n",
       "      <td>[{'name': 'Nuné Martiros'}, {'name': 'Spencer ...</td>\n",
       "      <td>&lt;p&gt;Positive and negative associations acquired...</td>\n",
       "      <td>2021-11-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.11.02.466953</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Population genomics analysis with RAD, reprise...</td>\n",
       "      <td>[{'name': 'Angel G Rivera-Colon'}, {'name': 'J...</td>\n",
       "      <td>&lt;p&gt;Restriction enzymes have been one of the pr...</td>\n",
       "      <td>2021-11-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.11.05.467474</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Aggregation controlled by condensate rheology</td>\n",
       "      <td>[{'name': 'Wolfram Poenisch'}, {'name': 'Thoma...</td>\n",
       "      <td>&lt;p&gt;Condensates in living cells can exhibit a c...</td>\n",
       "      <td>2021-11-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.11.04.467378</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>A high-throughput, automated, cell-free expres...</td>\n",
       "      <td>[{'name': 'Andrew C Hunt'}, {'name': 'Bastian ...</td>\n",
       "      <td>&lt;p&gt;Antibody discovery is bottlenecked by the i...</td>\n",
       "      <td>2021-11-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.11.03.467109</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Reciprocal zebrafish-medaka hybrids reveal mat...</td>\n",
       "      <td>[{'name': 'Krista R. Gert'}, {'name': 'Luis En...</td>\n",
       "      <td>&lt;p&gt;After fertilization, the sperm and egg cont...</td>\n",
       "      <td>2021-11-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.11.04.467308</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 infection in free-ranging white-tai...</td>\n",
       "      <td>[{'name': 'Vanessa L Hale'}, {'name': 'Patrici...</td>\n",
       "      <td>&lt;p&gt;Human-to-animal spillover of SARS-CoV-2 vir...</td>\n",
       "      <td>2021-11-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.11.02.466829</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Single molecule tracking of bacterial cell sur...</td>\n",
       "      <td>[{'name': 'Grace W. Chong'}, {'name': 'Sahand ...</td>\n",
       "      <td>&lt;p&gt;Using a series of multiheme cytochromes, th...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.11.02.467034</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>NASP maintains histone H3–H4 homeostasis throu...</td>\n",
       "      <td>[{'name': 'Hongyu Bao'}, {'name': 'Massimo Car...</td>\n",
       "      <td>&lt;p&gt;Histone chaperones regulate all aspects of ...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.11.02.21265843</td>\n",
       "      <td>21</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>A RCT of a third dose CoronaVac or BNT162b2 va...</td>\n",
       "      <td>[{'name': 'Chris Ka Pun Mok'}, {'name': 'Samue...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Poor immunoge...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.11.05.467477</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Age-induced P-bodies become detrimental and sh...</td>\n",
       "      <td>[{'name': 'Joonhyuk Choi'}, {'name': 'Shuhao W...</td>\n",
       "      <td>&lt;p&gt;Aging is an irreversible process characteri...</td>\n",
       "      <td>2021-11-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.11.01.466478</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Cooperative electrolyte-PEG interactions drive...</td>\n",
       "      <td>[{'name': 'Chalmers C. Chau'}, {'name': 'Fabio...</td>\n",
       "      <td>&lt;p&gt;Nanopore systems have emerged as a leading ...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.10.29.466473</td>\n",
       "      <td>18</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Multi-attribute Glycan Identification and FDR ...</td>\n",
       "      <td>[{'name': 'Daniel A. Polasky'}, {'name': 'Dani...</td>\n",
       "      <td>&lt;p&gt;Rapidly improving methods for glycoproteomi...</td>\n",
       "      <td>2021-11-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.10.29.21265555</td>\n",
       "      <td>18</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>IFI27 transcription is an early predictor for ...</td>\n",
       "      <td>[{'name': 'Maryam Shojaei'}, {'name': 'Amir Sh...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Robust biomar...</td>\n",
       "      <td>2021-11-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.11.06.467543</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Malaria transmission relies on concavin-mediat...</td>\n",
       "      <td>[{'name': 'Jessica Kehrer'}, {'name': 'Pauline...</td>\n",
       "      <td>&lt;p&gt;During transmission of malaria-causing para...</td>\n",
       "      <td>2021-11-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.10.30.466530</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Consequences of endogenous and virally-induced...</td>\n",
       "      <td>[{'name': 'Michael A. Kelberman'}, {'name': 'C...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The locus coe...</td>\n",
       "      <td>2021-11-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.11.05.467461</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Design, construction and optimization of a syn...</td>\n",
       "      <td>[{'name': 'Joep Houkes'}, {'name': 'Lorenzo Ol...</td>\n",
       "      <td>&lt;p&gt;Prokaryotic genes encoding functionally rel...</td>\n",
       "      <td>2021-11-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.11.06.467542</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Shaping the size of a neuronal lineage: the ro...</td>\n",
       "      <td>[{'name': 'Jonathan ENRIQUEZ'}, {'name': 'Weny...</td>\n",
       "      <td>&lt;p&gt;Neuronal stem cells produce a finite and st...</td>\n",
       "      <td>2021-11-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.11.02.466666</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Montage electron tomography of vitrified speci...</td>\n",
       "      <td>[{'name': 'Ariana Peck'}, {'name': 'Stephen D....</td>\n",
       "      <td>&lt;p&gt;Cryo-electron tomography provides detailed ...</td>\n",
       "      <td>2021-11-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.11.03.467004</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Discordance between different bioinformatic me...</td>\n",
       "      <td>[{'name': 'Timothy J Davies'}, {'name': 'Jerem...</td>\n",
       "      <td>&lt;label&gt;2.&lt;/label&gt;&lt;title&gt;Abstract&lt;/title&gt;&lt;p&gt;Sev...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.11.01.466347</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Dorsal striatum coding for the timely executio...</td>\n",
       "      <td>[{'name': 'María Cecilia Martinez'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;The automatic initiation of actions can be ...</td>\n",
       "      <td>2021-11-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.11.02.466988</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Plasticity in Ligand Recognition at Somatostat...</td>\n",
       "      <td>[{'name': 'Michael J. Robertson'}, {'name': 'J...</td>\n",
       "      <td>&lt;p&gt;Somatostatin is a signaling peptide that pl...</td>\n",
       "      <td>2021-11-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.11.05.466696</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>A paradox of parasite resistance: Disease-driv...</td>\n",
       "      <td>[{'name': 'Jason Cosens Walsman'}, {'name': 'A...</td>\n",
       "      <td>&lt;p&gt;Most evolutionary theory predicts that, dur...</td>\n",
       "      <td>2021-11-06</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.11.05.467528     307  bioRxiv   \n",
       "1   10.1101/2021.10.31.21265703     176  medRxiv   \n",
       "2     10.1101/2021.11.03.467152     146  bioRxiv   \n",
       "3     10.1101/2021.11.02.466936     113  bioRxiv   \n",
       "4     10.1101/2021.11.02.466983      95  bioRxiv   \n",
       "5     10.1101/2021.11.01.466765      88  bioRxiv   \n",
       "6     10.1101/2021.11.03.467131      68  bioRxiv   \n",
       "7     10.1101/2021.11.04.467077      68  bioRxiv   \n",
       "8     10.1101/2021.11.02.466915      58  bioRxiv   \n",
       "9     10.1101/2021.11.03.467079      53  bioRxiv   \n",
       "10    10.1101/2021.11.03.467097      44  bioRxiv   \n",
       "11    10.1101/2021.11.01.466728      43  bioRxiv   \n",
       "12    10.1101/2021.11.05.467435      43  bioRxiv   \n",
       "13    10.1101/2021.11.01.466817      41  bioRxiv   \n",
       "14    10.1101/2021.11.03.466924      37  bioRxiv   \n",
       "15    10.1101/2021.11.03.465409      35  bioRxiv   \n",
       "16    10.1101/2021.11.02.467002      34  bioRxiv   \n",
       "17    10.1101/2021.10.29.466453      33  bioRxiv   \n",
       "18    10.1101/2021.11.03.466369      31  bioRxiv   \n",
       "19    10.1101/2021.11.05.467437      30  bioRxiv   \n",
       "20    10.1101/2021.11.04.466897      30  bioRxiv   \n",
       "21    10.1101/2021.10.29.466434      29  bioRxiv   \n",
       "22    10.1101/2021.11.05.467534      29  bioRxiv   \n",
       "23    10.1101/2021.11.05.467388      27  bioRxiv   \n",
       "24    10.1101/2021.10.29.466512      26  bioRxiv   \n",
       "25    10.1101/2021.11.02.464583      26  bioRxiv   \n",
       "26    10.1101/2021.11.01.466790      26  bioRxiv   \n",
       "27    10.1101/2021.11.02.466884      26  bioRxiv   \n",
       "28    10.1101/2021.11.01.466363      23  bioRxiv   \n",
       "29    10.1101/2021.11.02.466953      23  bioRxiv   \n",
       "30    10.1101/2021.11.05.467474      23  bioRxiv   \n",
       "31    10.1101/2021.11.04.467378      23  bioRxiv   \n",
       "32    10.1101/2021.11.03.467109      23  bioRxiv   \n",
       "33    10.1101/2021.11.04.467308      22  bioRxiv   \n",
       "34    10.1101/2021.11.02.466829      22  bioRxiv   \n",
       "35    10.1101/2021.11.02.467034      21  bioRxiv   \n",
       "36  10.1101/2021.11.02.21265843      21  medRxiv   \n",
       "37    10.1101/2021.11.05.467477      19  bioRxiv   \n",
       "38    10.1101/2021.11.01.466478      19  bioRxiv   \n",
       "39    10.1101/2021.10.29.466473      18  bioRxiv   \n",
       "40  10.1101/2021.10.29.21265555      18  medRxiv   \n",
       "41    10.1101/2021.11.06.467543      17  bioRxiv   \n",
       "42    10.1101/2021.10.30.466530      16  bioRxiv   \n",
       "43    10.1101/2021.11.05.467461      16  bioRxiv   \n",
       "44    10.1101/2021.11.06.467542      15  bioRxiv   \n",
       "45    10.1101/2021.11.02.466666      15  bioRxiv   \n",
       "46    10.1101/2021.11.03.467004      15  bioRxiv   \n",
       "47    10.1101/2021.11.01.466347      15  bioRxiv   \n",
       "48    10.1101/2021.11.02.466988      15  bioRxiv   \n",
       "49    10.1101/2021.11.05.466696      14  bioRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0                          Bioengineering  False   \n",
       "1   Infectious Diseases (except HIV/AIDS)   True   \n",
       "2                            Microbiology  False   \n",
       "3                       Molecular Biology  False   \n",
       "4                              Biophysics  False   \n",
       "5                    Evolutionary Biology  False   \n",
       "6                            Biochemistry  False   \n",
       "7                            Microbiology   True   \n",
       "8                       Molecular Biology  False   \n",
       "9                                Genomics  False   \n",
       "10                           Microbiology  False   \n",
       "11                   Evolutionary Biology  False   \n",
       "12                               Genomics  False   \n",
       "13                  Developmental Biology  False   \n",
       "14                           Microbiology  False   \n",
       "15                      Molecular Biology  False   \n",
       "16                           Microbiology  False   \n",
       "17                           Neuroscience  False   \n",
       "18                           Biochemistry  False   \n",
       "19                          Plant Biology  False   \n",
       "20                               Genomics  False   \n",
       "21                           Neuroscience  False   \n",
       "22                               Genomics  False   \n",
       "23                               Genomics  False   \n",
       "24          Animal Behavior and Cognition  False   \n",
       "25                      Molecular Biology  False   \n",
       "26                               Genomics  False   \n",
       "27                           Neuroscience  False   \n",
       "28                           Neuroscience  False   \n",
       "29                         Bioinformatics  False   \n",
       "30                             Biophysics  False   \n",
       "31                      Synthetic Biology   True   \n",
       "32                  Developmental Biology  False   \n",
       "33                      Molecular Biology   True   \n",
       "34                           Microbiology  False   \n",
       "35                           Biochemistry  False   \n",
       "36  Infectious Diseases (except HIV/AIDS)   True   \n",
       "37                           Cell Biology  False   \n",
       "38                             Biophysics  False   \n",
       "39                         Bioinformatics  False   \n",
       "40  Infectious Diseases (except HIV/AIDS)   True   \n",
       "41                           Microbiology  False   \n",
       "42                           Neuroscience  False   \n",
       "43                      Synthetic Biology  False   \n",
       "44                  Developmental Biology  False   \n",
       "45                             Biophysics  False   \n",
       "46                           Microbiology  False   \n",
       "47                           Neuroscience  False   \n",
       "48                             Biophysics  False   \n",
       "49                                Ecology  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Large-scale discovery of recombinases for inte...   \n",
       "1   Efficacy and safety of SOBERANA 02, a COVID-19...   \n",
       "2   DeepBacs: Bacterial image analysis using open-...   \n",
       "3   Bright, fluorogenic and photostable avidity pr...   \n",
       "4   Structure Determination of Inactive-State GPCR...   \n",
       "5   Global patterns and rates of habitat transitio...   \n",
       "6   A molecular network of conserved factors keeps...   \n",
       "7   The oral protease inhibitor (PF-07321332) prot...   \n",
       "8   RBBP6 activates the pre-mRNA 3’-end processing...   \n",
       "9          Parent-of-origin effects in the UK Biobank   \n",
       "10  A drug repurposing screen identifies altiratin...   \n",
       "11  Haldane’s cost of selection imposes a mild con...   \n",
       "12  Ancient genomes from the last three millennia ...   \n",
       "13  A powerful and versatile new fixation protocol...   \n",
       "14  Organization and composition of apicomplexan k...   \n",
       "15  Structural insights into human excitatory amin...   \n",
       "16  Detection of velogenic avian paramyxoviruses i...   \n",
       "17  Invariant neural subspaces maintained by feedb...   \n",
       "18  Structure of mammalian V-ATPase with the TLDc ...   \n",
       "19  PHOSPHATE STARVATION RESPONSE enables arbuscul...   \n",
       "20  Genome-wide association analyses of individual...   \n",
       "21  Multilevel neural gradients reflect transdiagn...   \n",
       "22  Kaptive 2.0: updated capsule and LPS locus typ...   \n",
       "23  A temporally resolved, multiplex molecular rec...   \n",
       "24  Sex and Estrous Cycle in Memory for Sequences ...   \n",
       "25  Marker-free coselection for successive rounds ...   \n",
       "26  Programmable large DNA deletion, replacement, ...   \n",
       "27  Retinal OFF ganglion cells allow detection of ...   \n",
       "28  Distinct representation of cue-outcome associa...   \n",
       "29  Population genomics analysis with RAD, reprise...   \n",
       "30      Aggregation controlled by condensate rheology   \n",
       "31  A high-throughput, automated, cell-free expres...   \n",
       "32  Reciprocal zebrafish-medaka hybrids reveal mat...   \n",
       "33  SARS-CoV-2 infection in free-ranging white-tai...   \n",
       "34  Single molecule tracking of bacterial cell sur...   \n",
       "35  NASP maintains histone H3–H4 homeostasis throu...   \n",
       "36  A RCT of a third dose CoronaVac or BNT162b2 va...   \n",
       "37  Age-induced P-bodies become detrimental and sh...   \n",
       "38  Cooperative electrolyte-PEG interactions drive...   \n",
       "39  Multi-attribute Glycan Identification and FDR ...   \n",
       "40  IFI27 transcription is an early predictor for ...   \n",
       "41  Malaria transmission relies on concavin-mediat...   \n",
       "42  Consequences of endogenous and virally-induced...   \n",
       "43  Design, construction and optimization of a syn...   \n",
       "44  Shaping the size of a neuronal lineage: the ro...   \n",
       "45  Montage electron tomography of vitrified speci...   \n",
       "46  Discordance between different bioinformatic me...   \n",
       "47  Dorsal striatum coding for the timely executio...   \n",
       "48  Plasticity in Ligand Recognition at Somatostat...   \n",
       "49  A paradox of parasite resistance: Disease-driv...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Matthew G Durrant'}, {'name': 'Alis...   \n",
       "1   [{'name': 'Maria Eugenia Toledo-Romani'}, {'na...   \n",
       "2   [{'name': 'Christoph Spahn'}, {'name': 'Romain...   \n",
       "3   [{'name': 'Bastian Bühler'}, {'name': 'Anja Be...   \n",
       "4   [{'name': 'Michael J. Robertson'}, {'name': 'F...   \n",
       "5   [{'name': 'Mahwash Jamy'}, {'name': 'Charlie B...   \n",
       "6   [{'name': 'Friederike Leesch'}, {'name': 'Laur...   \n",
       "7   [{'name': 'Rana Abdelnabi'}, {'name': 'Carolin...   \n",
       "8   [{'name': 'Vytaute Boreikaite'}, {'name': 'Tho...   \n",
       "9   [{'name': 'Robin J Hofmeister'}, {'name': 'Sim...   \n",
       "10  [{'name': 'Christopher Swale'}, {'name': 'Vale...   \n",
       "11  [{'name': 'Joseph Matheson'}, {'name': 'Moises...   \n",
       "12  [{'name': 'Sandra Oliveira'}, {'name': 'Kathri...   \n",
       "13  [{'name': 'Carlos Guerrero-Hernández'}, {'name...   \n",
       "14  [{'name': 'Lorenzo Brusini'}, {'name': 'Nicola...   \n",
       "15  [{'name': 'Takafumi Kato'}, {'name': 'Tsukasa ...   \n",
       "16  [{'name': 'Isabel Francisco'}, {'name': 'Shato...   \n",
       "17  [{'name': 'Laura Bella Naumann'}, {'name': 'Jo...   \n",
       "18  [{'name': 'Yong Zi Tan'}, {'name': 'Yazan M. A...   \n",
       "19  [{'name': 'Debatosh Das'}, {'name': 'Michael P...   \n",
       "20  [{'name': 'Else Eising'}, {'name': 'Nazanin Mi...   \n",
       "21  [{'name': 'Bo-yong Park'}, {'name': 'Valeria K...   \n",
       "22  [{'name': 'Margaret M.C. Lam'}, {'name': 'Ryan...   \n",
       "23  [{'name': 'Junhong Choi'}, {'name': 'Wei Chen'...   \n",
       "24  [{'name': 'M. Jayachandran'}, {'name': 'P. Lan...   \n",
       "25  [{'name': 'Sébastien Levesque'}, {'name': 'Dia...   \n",
       "26  [{'name': 'Andrew V. Anzalone'}, {'name': 'Xin...   \n",
       "27  [{'name': 'Johan Westö'}, {'name': 'Nataliia M...   \n",
       "28  [{'name': 'Nuné Martiros'}, {'name': 'Spencer ...   \n",
       "29  [{'name': 'Angel G Rivera-Colon'}, {'name': 'J...   \n",
       "30  [{'name': 'Wolfram Poenisch'}, {'name': 'Thoma...   \n",
       "31  [{'name': 'Andrew C Hunt'}, {'name': 'Bastian ...   \n",
       "32  [{'name': 'Krista R. Gert'}, {'name': 'Luis En...   \n",
       "33  [{'name': 'Vanessa L Hale'}, {'name': 'Patrici...   \n",
       "34  [{'name': 'Grace W. Chong'}, {'name': 'Sahand ...   \n",
       "35  [{'name': 'Hongyu Bao'}, {'name': 'Massimo Car...   \n",
       "36  [{'name': 'Chris Ka Pun Mok'}, {'name': 'Samue...   \n",
       "37  [{'name': 'Joonhyuk Choi'}, {'name': 'Shuhao W...   \n",
       "38  [{'name': 'Chalmers C. Chau'}, {'name': 'Fabio...   \n",
       "39  [{'name': 'Daniel A. Polasky'}, {'name': 'Dani...   \n",
       "40  [{'name': 'Maryam Shojaei'}, {'name': 'Amir Sh...   \n",
       "41  [{'name': 'Jessica Kehrer'}, {'name': 'Pauline...   \n",
       "42  [{'name': 'Michael A. Kelberman'}, {'name': 'C...   \n",
       "43  [{'name': 'Joep Houkes'}, {'name': 'Lorenzo Ol...   \n",
       "44  [{'name': 'Jonathan ENRIQUEZ'}, {'name': 'Weny...   \n",
       "45  [{'name': 'Ariana Peck'}, {'name': 'Stephen D....   \n",
       "46  [{'name': 'Timothy J Davies'}, {'name': 'Jerem...   \n",
       "47  [{'name': 'María Cecilia Martinez'}, {'name': ...   \n",
       "48  [{'name': 'Michael J. Robertson'}, {'name': 'J...   \n",
       "49  [{'name': 'Jason Cosens Walsman'}, {'name': 'A...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>Recent microbial genome sequencing efforts ...  2021-11-06  \n",
       "1   <p>Background: SOBERANA 02 is a COVID19 conjug...  2021-11-01  \n",
       "2   <p>Deep Learning (DL) is rapidly changing the ...  2021-11-03  \n",
       "3   <p>Fluorescent light-up aptamers (FLAPs) emerg...  2021-11-02  \n",
       "4   <p>Cryogenic electron microscopy (cryo-EM) has...  2021-11-04  \n",
       "5   <p>The successful colonisation of new habitats...  2021-11-03  \n",
       "6   <p>Ribosomes are produced in large quantities ...  2021-11-03  \n",
       "7   <p>There is an urgent need for potent and sele...  2021-11-05  \n",
       "8   <p>3’-end processing of most human mRNAs is ca...  2021-11-02  \n",
       "9   <p>Identical genetic variations can have diffe...  2021-11-06  \n",
       "10  <p>The apicomplexa comprise a large phylum of ...  2021-11-03  \n",
       "11  <p>Haldane’s Dilemma refers to the concern tha...  2021-11-03  \n",
       "12  <p>Previous research indicates that the human ...  2021-11-05  \n",
       "13  <p>Whole-mount <italic>in situ</italic> hybrid...  2021-11-03  \n",
       "14  <p>Kinetochores are multiprotein assemblies di...  2021-11-03  \n",
       "15  <p>Glutamate is a pivotal excitatory neurotran...  2021-11-03  \n",
       "16  <p>Avian paramyxovirus 1 (APMV-1), also known ...  2021-11-02  \n",
       "17  <p>Sensory systems reliably process incoming s...  2021-11-01  \n",
       "18  <p>V-ATPases are rotary proton pumps that serv...  2021-11-03  \n",
       "19  <p>Arbuscular mycorrhiza (AM) is a widespread ...  2021-11-06  \n",
       "20  <p>The use of spoken and written language is a...  2021-11-04  \n",
       "21  <p>It is increasingly recognized that multiple...  2021-11-02  \n",
       "22  <p>The outer polysaccharide capsule and lipopo...  2021-11-06  \n",
       "23  <p>DNA is naturally well-suited to serve as a ...  2021-11-05  \n",
       "24  <p>The ability to remember sequences of events...  2021-11-01  \n",
       "25  <p>Prime editing enables the introduction of p...  2021-11-02  \n",
       "26  <p>The targeted deletion, replacement, integra...  2021-11-02  \n",
       "27  <p>Perception of light in darkness requires no...  2021-11-04  \n",
       "28  <p>Positive and negative associations acquired...  2021-11-02  \n",
       "29  <p>Restriction enzymes have been one of the pr...  2021-11-04  \n",
       "30  <p>Condensates in living cells can exhibit a c...  2021-11-06  \n",
       "31  <p>Antibody discovery is bottlenecked by the i...  2021-11-05  \n",
       "32  <p>After fertilization, the sperm and egg cont...  2021-11-04  \n",
       "33  <p>Human-to-animal spillover of SARS-CoV-2 vir...  2021-11-05  \n",
       "34  <p>Using a series of multiheme cytochromes, th...  2021-11-03  \n",
       "35  <p>Histone chaperones regulate all aspects of ...  2021-11-03  \n",
       "36  <sec><title>Background</title><p>Poor immunoge...  2021-11-03  \n",
       "37  <p>Aging is an irreversible process characteri...  2021-11-05  \n",
       "38  <p>Nanopore systems have emerged as a leading ...  2021-11-03  \n",
       "39  <p>Rapidly improving methods for glycoproteomi...  2021-11-01  \n",
       "40  <sec><title>Background</title><p>Robust biomar...  2021-11-01  \n",
       "41  <p>During transmission of malaria-causing para...  2021-11-06  \n",
       "42  <sec><title>Background</title><p>The locus coe...  2021-11-02  \n",
       "43  <p>Prokaryotic genes encoding functionally rel...  2021-11-05  \n",
       "44  <p>Neuronal stem cells produce a finite and st...  2021-11-06  \n",
       "45  <p>Cryo-electron tomography provides detailed ...  2021-11-04  \n",
       "46  <label>2.</label><title>Abstract</title><p>Sev...  2021-11-03  \n",
       "47  <p>The automatic initiation of actions can be ...  2021-11-03  \n",
       "48  <p>Somatostatin is a signaling peptide that pl...  2021-11-04  \n",
       "49  <p>Most evolutionary theory predicts that, dur...  2021-11-06  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "237 preprints (including 40 covering SARS-CoV-2, 200 from bioRxiv and 37 from medRxiv) published in the last 7 days before 2021-11-08 had been tweeted at least 3 times (maximum 307)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.8"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
