{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "17155 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=47bd5be6-ca90-456a-9107-2a7248c34fbf&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5d0f01df-5d15-45a3-a0b7-47a219e31e12&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=bcb4a202-6fa0-4dc2-b837-d87ef1ad94a3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7dffc976-6db7-40d9-94db-1c97fa644d90&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9f7349fa-4f8c-4027-9451-4993fdca30be&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=8c43baea-1c9a-467c-921e-bd984d1725cd&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=3e42f15d-a8d3-41df-b38b-8fc3eb21cf0f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=73a8e046-5c74-479d-b664-b77e29ff5258&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=66cae185-b530-4d96-8d54-d719b492a04b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=0da8418d-6327-4c4e-9494-31ec2a9fff02&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=561323ad-7cd8-468e-9191-2b4961fbc103&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9b407a06-1ba1-4596-ab0b-792d8d930887&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=cc945af3-abf7-45db-8aa9-c2ebf43a32ff&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=432bf3ba-488f-42cb-9a21-d82ba7a01506&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=affae5ca-ba93-4b86-a8d3-f4f2ea7a606a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c20ca55f-b7c3-4c20-b5e6-0f9f0313b690&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=29183369-458a-497e-86bc-d47281d528d8&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-18&until-occurred-date=2021-10-25\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-10-18.csv\n",
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load preprint_tweets_2021-10-23.csv\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21264623...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.21256814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265094...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464465...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.339317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/530097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.382143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.462913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20087965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.09.20184143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.412098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.420448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/240317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275648.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275569.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273300.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.29.293274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.21249563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.28.923532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464073...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.457995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.440736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.459575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.432725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.18.210161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465155...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275989.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.21264267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.415505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21262852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458713...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.436603...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.426522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.367649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.439271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.413609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20198796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21258385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.21263139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.437513...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.11.244996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.20209114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.17.952663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.324368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.450922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.20222661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.27.175455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.27.175067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464654...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20213546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.27.437361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.458981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463761...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.460212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.02.424917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.464887...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.10.21.465288...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.09.23.21263924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.163816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.429110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.451231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21251504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.463719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.02.232710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.447954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.434203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.11.451967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.28.20029272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.21260732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265231...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.10.21.465371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.413229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.442903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.206680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.12.430990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.03.20167395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.04.451074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.463767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.21263026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.21264893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.21257899...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.458083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465099...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.20142703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.439549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.07.191197...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275503.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.21255913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.436212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.21252386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.21253805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.18.158964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.21261741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.328948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21256946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.385401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/570580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/600106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/458992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.09.287508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.464688...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256753...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453060...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/749341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/676312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.271288.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275223.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454669...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21262468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.461806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.20204693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.458035...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.25.169946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.01.424252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.465025...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.09.24.461616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435343...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.464965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.463833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.454052...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.08.372995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.21265037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.08.23.457426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.04.20053058...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.462574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464285...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.22.20025791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268581.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.450999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.365932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.459925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.26.424423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.447279...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.21250756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.463730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.454108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.464824...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443052...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.21263548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/481655...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275308.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.21262262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461293...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442149...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.01.277152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.21250840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462466...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.11.439311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21261243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.09.443331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455750...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453874...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.21258272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.450005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21258418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.441522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.460971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20131243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.319814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.424510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.20086439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.30.272237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/407627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.01.20136317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465087...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.03.20145797...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21264783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.02.20186734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.21250420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.235549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265209...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.09.22.21263942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265213...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.09.22.21263935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.21265096...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.10.19.21265221...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.459429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.20127357...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265279...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.17.20037689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.21254026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.03.21254866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.31.928630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.438122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.03.932046...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.462716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21253914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.13.947366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264099...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.389031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.387720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/768663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.463729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.460970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.03.20051649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.29.402743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20222778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21262551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.463784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.21258079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.17.386995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.14.297424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448344...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.384743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.17.301804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.447951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.04.20225557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461278...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.341958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21256632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/780627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/656165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21258047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/028522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275430.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/19007096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.21.914044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268037.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.451701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/372193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.174888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.465067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.23.449647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.460811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21264255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263963...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.464802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/282848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/276717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/231209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.20109231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465238...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.20233155...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.438706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.14.21251704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.458655...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.21263783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.21250706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.21250769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.21250946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460382...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.428837...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.21.21251403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.21254578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.439482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.438404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21254672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.21256049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461107...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.20245993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.399154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.20248876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.395079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461491...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.10.19.21265208', 'tweets': 1705, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Airborne transmission of SARS-CoV-2 over distances greater than two metres: a rapid systematic review', 'authors': \"[{'name': 'Jennifer C Palmer'}, {'name': 'Daphne Duval'}, {'name': 'Isobel Tudge'}, {'name': 'Jason Kwasi Sarfo-Annin'}, {'name': 'Nicola Pearce-Smith'}, {'name': 'Emer OConnell'}, {'name': 'Allan Bennett'}, {'name': 'Rachel Clark'}]\", 'abstract': '<sec><title>Objective</title><p>To evaluate the potential for long-distance (over two metres) airborne transmission of SARS-CoV-2 in indoor community settings and investigate factors which may impact this transmission.</p></sec><sec><title>Design</title><p>Systematic review and narrative synthesis.</p></sec><sec><title>Data source</title><p>MEDLINE, Embase, medRxiv, Arxiv and WHO COVID-19 Research Database for studies published from 27 July 2020 to 21 April 2021; existing relevant rapid systematic reviews for studies published between 1 January to 27 July 2020.</p></sec><sec><title>Eligibility criteria for study selection</title><p>Observational studies that included a thorough epidemiological assessment of routes of transmission and which reported on the likelihood of airborne transmission of SARS-CoV-2 at a distance greater than two metres in indoor community settings.</p></sec><sec><title>Data extraction and synthesis</title><p>Data extraction was completed by one reviewer and independently checked by a second reviewer. Primary outcomes were COVID-19 infections via airborne transmission over distances greater than two metres and any factors that may have modified transmission risk. Included studies were rated using a quality criteria checklist (QCC) for primary research and certainty of key outcomes was determined using GRADE. Narrative synthesis was themed by setting.</p></sec><sec><title>Results</title><p>Of the 3,780 articles screened for inclusion, 15 publications reporting on 13 epidemiological investigations were included (three high, six medium and four low quality). Airborne transmission at distances greater than two metres was likely to have occurred for some or all transmission events in 12 studies and was unclear in one study (GRADE: very low certainty). In all studies, one or more factors plausibly increased the likelihood of long-distance airborne transmission occurring, particularly insufficient air replacement (GRADE: very low certainty), recirculating air flow (GRADE: very low certainty) and singing (GRADE: very low certainty). In nine studies, the primary cases were reported as being asymptomatic, presymptomatic or around symptom onset at the time of transmission.</p></sec><sec><title>Conclusion</title><p>This rapid systematic review found evidence of long-distance airborne transmission of SARS-CoV-2 in indoor community settings and identified factors that likely contributed to this transmission in all included studies. These results strengthen the need for adequate mitigation measures in indoor community settings, particularly adequate ventilation with fresh air, and caution required with the use of recirculating air flow systems.</p></sec><sec><title>Systematic review registration</title><p>PROSPERO CRD42021236762</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.18.21264623', 'tweets': 419, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'SARS-CoV-2 variants associated with vaccine breakthrough in the Delaware Valley through summer 2021', 'authors': \"[{'name': 'Andrew D. Marques'}, {'name': 'Scott Sherrill-Mix'}, {'name': 'John Everett'}, {'name': 'Shantan Reddy'}, {'name': 'Pascha Hokama'}, {'name': 'Aoife M. Roche'}, {'name': 'Young Hwang'}, {'name': 'Abigail Glascock'}, {'name': 'Samantha A. Whiteside'}, {'name': 'Jevon Graham-Wooten'}, {'name': 'Layla A. Khatib'}, {'name': 'Ayannah S. Fitzgerald'}, {'name': 'Ahmed M. Moustafa'}, {'name': 'Colleen Bianco'}, {'name': 'Swetha Rajagopal'}, {'name': 'Jenna Helton'}, {'name': 'Regan Deming'}, {'name': 'Lidiya Denu'}, {'name': 'Azad Ahmed'}, {'name': 'Eimear Kitt'}, {'name': 'Susan E. Coffin'}, {'name': 'Claire Newbern'}, {'name': 'Josh Chang Mell'}, {'name': 'Paul J. Planet'}, {'name': 'Nitika Badjatia'}, {'name': 'Bonnie Richards'}, {'name': 'Zi-Xuan Wang'}, {'name': 'Carolyn C. Cannuscio'}, {'name': 'Katherine M. Strelau'}, {'name': 'Anne Jaskowiak-Barr'}, {'name': 'Leigh Cressman'}, {'name': 'Sean Loughrey'}, {'name': 'Arupa Ganguly'}, {'name': 'Michael D. Feldman'}, {'name': 'Ronald G. Collman'}, {'name': 'Kyle G. Rodino'}, {'name': 'Brendan J. Kelly'}, {'name': 'Frederic D. Bushman'}]\", 'abstract': '<p>The severe acute respiratory coronavirus-2 (SARS-CoV-2) is the cause of the global outbreak of COVID-19. Evidence suggests that the virus is evolving to allow efficient spread through the human population, including vaccinated individuals. Here we report a study of viral variants from surveillance of the Delaware Valley, including the city of Philadelphia, and variants infecting vaccinated subjects. We sequenced and analyzed complete viral genomes from 2621 surveillance samples from March 2020 to September 2021 and compared them to genome sequences from 159 vaccine breakthroughs. In the early spring of 2020, all detected variants were of the B.1 and closely related lineages. A mixture of lineages followed, notably including B.1.243 followed by B.1.1.7 (alpha), with other lineages present at lower levels. Later isolations were dominated by B.1.617.2 (delta) and other delta lineages; delta was the exclusive variant present by the last time sampled. To investigate whether any variants appeared preferentially in vaccine breakthroughs, we devised a model based on Bayesian autoregressive moving average logistic multinomial regression to allow rigorous comparison. This revealed that B.1.617.2 (delta) showed three-fold enrichment in vaccine breakthrough cases (odds ratio of 3; 95% credible interval 0.89-11). Viral point substitutions could also be associated with vaccine breakthroughs, notably the N501Y substitution found in the alpha, beta and gamma variants (odds ratio 2.04; 95% credible interval of 1.25-3.18). This study thus provides a detailed picture of viral evolution in the Delaware Valley and a geographically matched analysis of vaccine breakthroughs; it also introduces a rigorous statistical approach to interrogating enrichment of viral variants.</p><sec><title>Importance</title><p>SARS-CoV-2 vaccination is highly effective at reducing viral infection, hospitalization and death. However, vaccine breakthrough infections have been widely observed, raising the question of whether particular viral variants or viral mutations are associated with breakthrough. Here we report analysis of 2621 surveillance isolates from xsxpeople diagnosed with COVID-19 in the Delaware Valley in South Eastern Pennsylvania, allowing rigorous comparison to 159 vaccine breakthrough case specimens. Our best estimate is a three-fold enrichment for some lineages of delta among breakthroughs, and enrichment of a notable spike substitution, N501Y. We introduce statistical methods that should be widely useful for evaluating vaccine breakthroughs and other viral phenotypes.</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.22.465012', 'tweets': 96, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Hippocampal spatio-temporal cognitive maps adaptively guide reward generalization', 'authors': \"[{'name': 'Mona M Garvert'}, {'name': 'Tankred Saanum'}, {'name': 'Eric Schulz'}, {'name': 'Nicolas W Schuck'}, {'name': 'Christian F. Doeller'}]\", 'abstract': \"<p>The brain forms cognitive maps of relational knowledge, an organizing principle thought to underlie our ability to generalize and make inferences. However, how can a relevant map be selected in situations where a stimulus is embedded in multiple relational structures? Here, we find that both spatial and temporal cognitive maps influence generalization in a choice task, where spatial location determines reward magnitude. Mirroring behavior, the hippocampus not only builds a map of spatial relationships but also encodes temporal distances. As the task progresses, participants' choices become more influenced by spatial relationships, reflected in a strengthening of the spatial and a weakening of the temporal map. This change is driven by orbitofrontal cortex, which represents the evidence that an observed outcome is generated from the spatial rather than the temporal map and updates hippocampal representations accordingly. Taken together, this demonstrates how hippocampal cognitive maps are used and updated flexibly for inference.</p>\", 'posted': '2021-10-23'}, {'doi': '10.1101/2021.10.18.21265113', 'tweets': 90, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Immune Responses in Fully Vaccinated Individuals Following Breakthrough Infection with the SARS-CoV-2 Delta Variant in Provincetown, Massachusetts', 'authors': \"[{'name': 'Ai-ris Y. Collier'}, {'name': 'Catherine M. Brown'}, {'name': 'Katherine Mcmahan'}, {'name': 'Jingyou Yu'}, {'name': 'Jinyan Liu'}, {'name': 'Catherine Jacob-Dolan'}, {'name': 'Abishek Chandrashekar'}, {'name': 'Dylan Tierney'}, {'name': 'Jessica L. Ansel'}, {'name': 'Marjorie Rowe'}, {'name': 'Daniel Sellers'}, {'name': 'Kunza Ahmad'}, {'name': 'Ricardo Aguayo'}, {'name': 'Tochi Anioke'}, {'name': 'Sarah Gardner'}, {'name': 'Mazuba Siamatu'}, {'name': 'Lorraine Bermudez Rivera'}, {'name': 'Michele R. Hacker'}, {'name': 'Lawrence C. Madoff'}, {'name': 'Dan H. Barouch'}]\", 'abstract': '<sec><title>Background</title><p>A cluster of over a thousand infections with the SARS-CoV-2 delta variant was identified in a predominantly fully vaccinated population in Provincetown, Massachusetts in July 2021. Immune responses in breakthrough infections with the SARS-CoV-2 delta variant remain to be defined.</p></sec><sec><title>Methods</title><p>Humoral and cellular immune responses were assessed in 35 vaccinated individuals who were tested for SARS-CoV-2 in the Massachusetts Department of Public Health outbreak investigation.</p></sec><sec><title>Results</title><p>Vaccinated individuals who tested positive for SARS-CoV-2 demonstrated substantially higher antibody responses than vaccinated individuals who tested negative for SARS-CoV-2, including 28-fold higher binding antibody titers and 34-fold higher neutralizing antibody titers against the SARS-CoV-2 delta variant. Vaccinated individuals who tested positive also showed 4.4-fold higher Spike-specific CD8+ T cell responses against the SARS-CoV-2 delta variant than vaccinated individuals who tested negative.</p></sec><sec><title>Conclusions</title><p>Fully vaccinated individuals developed robust anamnestic antibody and T cell responses following infection with the SARS-CoV-2 delta variant. These data suggest important immunologic benefits of vaccination in the context of breakthrough infections.</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.21.465374', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Neural knowledge assembly in humans and deep networks', 'authors': \"[{'name': 'Stephanie M Nelli'}, {'name': 'Lukas Braun'}, {'name': 'Tsvetomira Dumbalska'}, {'name': 'Andrew Saxe'}, {'name': 'Christopher Summerfield'}]\", 'abstract': '<p>Human understanding of the world can change rapidly when new information comes to light, such as when a plot twist occurs in a work of fiction. This flexible knowledge assembly requires few-shot reorganisation of neural codes for relations among objects and events. However, existing computational theories are largely silent about how this could occur. Here, participants learned a transitive ordering among novel objects within two distinct contexts, before exposure to new knowledge revealing how the contexts were linked. BOLD signals in dorsal frontoparietal cortical areas revealed that objects were rapidly and dramatically rearranged on the neural manifold after minimal exposure to the linking information. We then adapt stochastic online gradient descent to permit similar rapid knowledge assembly in a neural network model.</p>', 'posted': '2021-10-23'}, {'doi': '10.1101/2021.10.17.21265101', 'tweets': 33, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Time-Varying Effectiveness of Three Covid-19 Vaccines in Puerto Rico', 'authors': \"[{'name': 'Mnica M. Robles Fontn'}, {'name': 'Elvis G. Nieves'}, {'name': 'Iris Cardona Gerena'}, {'name': 'Rafael A. Irizarry'}]\", 'abstract': '<sec><title>Background</title><p>As of October 1, 2021 2,217,547 individuals were fully vaccinated against COVID-19 in Puerto Rico. Since the vaccination process commenced on December 15, 2020 111,052 laboratory-confirmed SARS-CoV-2 infections have been reported. These data permitted us to quantify the benefits of the immunization campaign and to compare effectiveness of the mRNA-1273 (Moderna), BNT162b2 (Pfizer), and Ad26.COV2.S (J&amp;J) vaccines.</p></sec><sec><title>Methods</title><p>Department of Health databases holding vaccination status, SARS-CoV-2 test results, and COVID-19 hospitalizations and deaths were integrated. We fit a statistical model that adjusted for time-varying incidence rates and age to estimate vaccine effectiveness and hospitalization and death relative risks. Code and data are provided here: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/rafalab/vax-eff-pr\">https://github.com/rafalab/vax-eff-pr</ext-link>.</p></sec><sec><title>Results</title><p>At the peak of their protection, mRNA-1273, BNT162b2, and Ad26.COV2.S had an effectiveness of 90% (88%-91%), 87% (85%-89%), and 58% (51%-65%), respectively. After four months, effectiveness waned to about 70%, 60%, and 30%. We found no evidence that effectiveness was different after the Delta variant became dominant. For those infected, the vaccines provided further protection against hospitalization and deaths across all age groups. All vaccines had a lower effectiveness for those over 85 years, with a larger decrease for the Ad26.COV2.S vaccine. Overall, thousands of hospitalizations and deaths were avoided thanks to the vaccines.</p></sec><sec><title>Conclusions</title><p>The mRNA-1273 and BNT162b2 vaccines were highly effective across all age groups. They were still effective after four months although the protection waned. The Ad26.COV2.S vaccine was effective but to a lesser degree, especially for older age groups.</p></sec>', 'posted': '2021-10-18'}, {'doi': '10.1101/2021.10.21.465323', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Defining cellular complexity in human autosomal dominant polycystic kidney disease by multimodal single cell analysis', 'authors': \"[{'name': 'Yoshiharu Muto'}, {'name': 'Eryn E. Dixon'}, {'name': 'Yasuhiro Yoshimura'}, {'name': 'Haojia Wu'}, {'name': 'Kohei Omachi'}, {'name': 'Andrew J. King'}, {'name': 'Eric N. Olson'}, {'name': 'Marvin G. Gunawan'}, {'name': 'Jay J. Kuo'}, {'name': 'Jennifer Cox'}, {'name': 'Jeffrey H. Miner'}, {'name': 'Stephen L. Seliger'}, {'name': 'Owen M. Woodward'}, {'name': 'Paul A. Welling'}, {'name': 'Terry J. Watnick'}, {'name': 'Benjamin D. Humphreys'}]\", 'abstract': '<p>Autosomal dominant polycystic kidney disease (ADPKD) is the leading genetic cause of end stage renal disease and is characterized by the formation and progressive expansion of kidney cysts. Most ADPKD cases arise from mutations in either the <italic>PKD1</italic> or <italic>PKD2</italic> gene but the precise downstream signaling pathways driving cyst growth are not well understood, and relatively few studies investigate human cystic kidney due to sample scarcity. In order to better understand the cell types and states driving human ADPKD progression, we analyzed eight ADPKD and five healthy human kidney samples, generating a single cell multiomic atlas consisting of ~100,000 single nucleus transcriptomes and ~50,000 single nucleus epigenomes. The integrated datasets identified 11 primary cell clusters including most epithelial cell types as well as large endothelial and fibroblast cell clusters. Proximal tubular cells from ADPKD kidneys expressed a failed repair transcriptomic signature characterized by profibrotic and proinflammatory transcripts. We identified the G protein-coupled receptor GPRC5A as specifically upregulated in cyst lining cells derived from collecting duct. The principal cell subpopulation enriched for GPRC5A expression also exhibited increased transcription factor binding motif availability for NF-B, TEAD, CREB and retinoic acid receptor families and we identified and validated a distal enhancer regulating GPRC5A expression containing these transcription factor binding motifs. This study establishes the single cell transcriptomic and epigenomic landscape of ADPKD, revealing previously unrecognized cellular heterogeneity.</p>', 'posted': '2021-10-22'}, {'doi': '10.1101/2021.10.20.465093', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Inherited MUTYH mutations cause elevated somatic mutation rates and distinctive mutational signatures in normal human cells', 'authors': \"[{'name': 'Philip S. Robinson'}, {'name': 'Laura E. Thomas'}, {'name': 'Federico Abascal'}, {'name': 'Hyunchul Jung'}, {'name': 'Luke M.R. Harvey'}, {'name': 'Hannah D. West'}, {'name': 'Sigurgeir Olafsson'}, {'name': 'Bernard C. H. Lee'}, {'name': 'Tim H.H. Coorens'}, {'name': 'Henry Lee-Six'}, {'name': 'Laura Butlin'}, {'name': 'Nicola Lander'}, {'name': 'Mathijs A. Sanders'}, {'name': 'Stefanie V. Lensing'}, {'name': 'Simon J.A. Buczacki'}, {'name': 'Rogier ten Hoppen'}, {'name': 'Nicholas Coleman'}, {'name': 'Roxanne Brunton-Sim'}, {'name': 'Simon Rushbrook'}, {'name': 'Kourosh Saeb-Parsy'}, {'name': 'Fiona Lalloo'}, {'name': 'Peter J. Campbell'}, {'name': 'Iigo Martincorena'}, {'name': 'Julian R. Sampson'}, {'name': 'Michael R. Stratton'}]\", 'abstract': '<p>Cellular DNA damage caused by reactive oxygen species is repaired by the base excision repair (BER) pathway which includes the DNA glycosylase MUTYH. Inherited biallelic <italic>MUTYH</italic> mutations cause predisposition to colorectal adenomas and carcinoma. However, the mechanistic progression from germline <italic>MUTYH</italic> mutations to <bold>M</bold>UTYH-<bold>A</bold>ssociated <bold>P</bold>olyposis (MAP) is incompletely understood. Here, we sequenced normal tissue DNAs from 10 individuals with MAP. Somatic base substitution mutation rates in intestinal epithelial cells were elevated 2 to 5-fold in all individuals, except for one showing a 33-fold increase, and were also increased in other tissues. The increased mutation burdens were of multiple mutational signatures characterised by C&gt;A changes. Different mutation rates and signatures between individuals were likely due to different <italic>MUTYH</italic> mutations or additional inherited mutations in other BER pathway genes. The elevated base substitution rate in normal cells likely accounts for the predisposition to neoplasia in MAP. Despite ubiquitously elevated mutation rates, individuals with MAP do not display overt evidence of premature ageing. Thus, accumulation of somatic mutations may not be sufficient to cause the global organismal functional decline of ageing.</p><p><bold>Summary</bold></p>', 'posted': '2021-10-21'}, {'doi': '10.1101/2021.10.22.465484', 'tweets': 25, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Genomic insights into variation in thermotolerance between hybridizing swordtail fishes', 'authors': \"[{'name': 'Cheyenne Payne'}, {'name': 'Stephen Bovio'}, {'name': 'Dan Powell'}, {'name': 'Theresa Gunn'}, {'name': 'Shreya Banerjee'}, {'name': 'Victoria Grant'}, {'name': 'Gil Rosenthal'}, {'name': 'Molly Schumer'}]\", 'abstract': '<p>Understanding how organisms adapt to changing environments is a core focus of research in evolutionary biology. One common mechanism is adaptive introgression, which has received increasing attention as a potential route to rapid adaptation in populations struggling in the face of ecological change, particularly global climate change. However, hybridization can also result in deleterious genetic interactions that may limit the benefits of adaptive introgression. Here, we used a combination of genome-wide quantitative trait locus mapping and differential gene expression analyses between the swordtail fish species Xiphophorus malinche and X. birchmanni to study the consequences of hybridization on thermotolerance. While these two species are adapted to different thermal environments, we document a complicated architecture of thermotolerance in hybrids. We identify a region of the genome that contributes to reduced thermotolerance in individuals heterozygous for X. malinche and X. birchmanni ancestry, as well as widespread misexpression in hybrids of genes that respond to thermal stress in the parental species, particularly in the circadian clock pathway. We also show that a previously mapped hybrid incompatibility between X. malinche and X. birchmanni contributes to reduced thermotolerance in hybrids. Together, our results highlight the challenges of understanding the impact of hybridization on complex ecological traits and its potential impact on adaptive introgression.</p>', 'posted': '2021-10-24'}, {'doi': '10.1101/2021.10.20.465155', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Human milk oligosaccharides reduce murine group B Streptococcus vaginal colonization with minimal impact on the vaginal microbiota', 'authors': \"[{'name': 'Marlyd E Mejia'}, {'name': 'Samantha Ottinger'}, {'name': 'Alison Vrbanac'}, {'name': 'Priyanka Babu'}, {'name': 'Jacob Zulk'}, {'name': 'David Moorshead'}, {'name': 'Lars Bode'}, {'name': 'Victor Nizet'}, {'name': 'Kathryn Patras'}]\", 'abstract': '<p>Group B Streptococcus (GBS) colonizes the vaginal mucosa of a significant percentage of healthy women and is a leading cause of neonatal bacterial infections. Currently, pregnant women are screened in the last month of pregnancy and GBS-positive women are given antibiotics during parturition to prevent bacterial transmission to the neonate. Recently, human milk oligosaccharides (HMOs) isolated from breastmilk were found to inhibit GBS growth and biofilm formation in vitro, and women that make certain HMOs are less likely to be vaginally colonized with GBS. Using in vitro human vaginal epithelial cells and a murine vaginal colonization model, we tested the impact of HMO treatment on GBS burdens and the composition of the endogenous microbiota by 16S rRNA amplicon sequencing. HMO treatment reduced GBS vaginal burdens in vivo with minimal alterations to the vaginal microbiota. HMOs displayed potent inhibitory activity against GBS in vitro, but HMO pretreatment did not alter adherence of GBS or the probiotic Lactobacillus rhamnosus to human vaginal epithelial cells. Additionally, disruption of a putative GBS glycosyltransferase (san_0913) rendered the bacterium largely resistant to HMO inhibition in vitro and in vivo but did not compromise its adherence, colonization, or biofilm formation in the absence of HMOs. We conclude that HMOs are a promising therapeutic bioactive to limit GBS vaginal colonization with minimal impacts on the vaginal microenvironment.</p>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.21.465386', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Variations in cell-surface ACE2 levels alter direct binding of SARS-CoV-2 Spike protein and viral infectivity: Implications for measuring Spike protein interactions with animal ACE2 orthologs', 'authors': \"[{'name': 'Soheila Kazemi'}, {'name': 'Alberto Domingo Lpez-Muoz'}, {'name': 'Jaroslav Holl'}, {'name': 'Ling Jin'}, {'name': 'Jonathan W. Yewdell'}, {'name': 'Brian P. Dolan'}]\", 'abstract': '<p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the causative agent of COVID-19, the most severe pandemic in a century. The virus gains access to host cells when the viral Spike protein (S-protein) binds to the host cell-surface receptor angiotensin-converting enzyme 2 (ACE2). Studies have attempted to understand SARS-CoV-2 S-protein interaction with vertebrate orthologs of ACE2 by expressing ACE2 orthologs in mammalian cells and measuring viral infection or S- protein binding. Often these cells only transiently express ACE2 proteins and levels of ACE2 at the cell surface are not quantified. Here, we describe a cell-based assay that uses stably transfected cells expressing ACE2 proteins in a bi-cistronic vector with an easy to quantify reporter protein to normalize ACE2 expression. We found that both binding of the S-protein receptor-binding domain (RBD) and infection with a SARS-CoV-2 pseudovirus is proportional to the amount of human ACE2 expressed at the cell surface, which can be inferred by quantifying the level of reporter protein, Thy1.1. We also compared different ACE2 orthologs which were expressed in stably transfected cells expressing equivalent levels of Thy1.1. When ranked for either viral infectivity or RBD binding, mouse ACE2 had a weak to undetectable affinity for S-protein while human ACE2 was the highest level detected and feline ACE2 had an intermediate phenotype. The generation of stably transfected cells whose ACE2 level can be normalized for cross-ortholog comparisons allows us to create a reusable cellular library useful for measuring emerging SARS-CoV-2 variants ability to potentially infect different animals.</p>', 'posted': '2021-10-23'}, {'doi': '10.1101/2021.10.21.465363', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'A Null Allele of the Pol IV Second Subunit is Viable in Oryza sativa', 'authors': \"[{'name': 'Tania Chakraborty'}, {'name': 'Joshua T. Trujillo'}, {'name': 'Timmy Kendall'}, {'name': 'Rebecca A. Mosher'}]\", 'abstract': '<p>All eukaryotes possess three DNA-dependent RNA polymerases, Pols I-III, while land plants possess two additional polymerases, Pol IV and Pol V. Derived through duplication of Pol II subunits, Pol IV produces 24-nt siRNAs that interact with Pol V transcripts to target <italic>de novo</italic> DNA methylation and silence transcription of transposons. Members of the grass family encode additional duplicated subunits of Pol IV and V, raising questions regarding the function of each paralog. In this study, we identify a null allele of the putative Pol IV second subunit, <italic>NRPD2</italic>, and demonstrate that NRPD2 is the sole subunit functioning with NRPD1 in small RNA production and CHH methylation in leaves. Homozygous <italic>nrpd2</italic> mutants have neither gametophytic defects, nor embryo lethality, although adult plants are dwarf and sterile.</p>', 'posted': '2021-10-22'}, {'doi': '10.1101/2021.10.19.21265226', 'tweets': 13, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Wastewater and surface monitoring to detect COVID-19 in elementary school settings: The Safer at School Early Alert project', 'authors': \"[{'name': 'Rebecca K Fielding-Miller'}, {'name': 'Smruthi Karthikeyan'}, {'name': 'Tommi Gaines'}, {'name': 'Richard S Garfein'}, {'name': 'Rodolfo A Salido'}, {'name': 'Victor Cantu'}, {'name': 'Laura Kohn'}, {'name': 'Fitri C Wijaya'}, {'name': 'Marlene Flores'}, {'name': 'Vinton Omaleki'}, {'name': 'Araz Majnoonian'}, {'name': 'Natasha K Martin'}, {'name': 'Dahl Austin'}, {'name': 'Louise C Laurent'}, {'name': 'Patricia Gonzalez-Zuniga'}, {'name': 'Megan Nguyen'}, {'name': 'Anh V Vo'}, {'name': 'Tina T Le'}, {'name': 'Dawn Duong'}, {'name': 'Ashkan Hassani'}, {'name': 'Samantha Tweeten'}, {'name': 'Kristen Jepsen'}, {'name': 'Benjamin Henson'}, {'name': 'Abbas Hakim'}, {'name': 'Amanda Birmingham'}, {'name': 'Adam M Mark'}, {'name': 'Chanond A Nasamran'}, {'name': 'Sara B Rosenthal'}, {'name': 'Niema Moshiri'}, {'name': 'Kathleen M Fisch'}, {'name': 'Greg Humphrey'}, {'name': 'Sawyer Farmer'}, {'name': 'Helena Tubb'}, {'name': 'Tommy Valles'}, {'name': 'Justin Morris'}, {'name': 'Jaeyoung Kang'}, {'name': 'Behnam Khaleghi'}, {'name': 'Colin Young'}, {'name': 'Ameen D Akel'}, {'name': 'Sean Eilert'}, {'name': 'Justin Eno'}, {'name': 'Ken Curewitz'}, {'name': 'Tajana Rosing'}, {'name': 'Rob Knight'}, {'name': ' '}]\", 'abstract': \"<p>Schools are high-risk settings for SARS-CoV-2 transmission, but necessary for children's educational and social-emotional wellbeing. While wastewater monitoring has been implemented to mitigate outbreak risk in universities and residential settings, its effectiveness in community K-12 sites is unknown. We implemented a wastewater and surface monitoring system to detect SARS-CoV-2 in nine elementary schools in San Diego County. Ninety-three percent of identified cases were associated with either a positive wastewater or surface sample; 67% were associated with a positive wastewater sample, and 40% were associated with a positive surface sample. The techniques we utilized allowed for near-complete genomic sequencing of wastewater and surface samples. Passive environmental surveillance can complement approaches that require individual consent, particularly in communities with limited access and/or high rates of testing hesitancy.</p>\", 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.20.21265245', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Comparative effectiveness of allocation strategies of COVID-19 vaccines and antivirals against emerging SARS-CoV-2 variants of concern in East Asia and Pacific region', 'authors': \"[{'name': 'Kathy Leung'}, {'name': 'Mark Jit'}, {'name': 'Gabriel M Leung'}, {'name': 'Joseph T Wu'}]\", 'abstract': '<sec><title>Background</title><p>We aimed to evaluate the impact of various allocation strategies of COVID-19 vaccines and antiviral such that the pandemic exit strategy could be tailored to risks and preferences of jurisdictions in the East Asia and Pacific region (EAP) to improve its efficiency and effectiveness.</p></sec><sec><title>Methods</title><p>Vaccine efficacies were estimated from the titre distributions of 50% plaque reduction neutralization test (PRNT50), assuming that PRNT50 titres of primary vaccination decreased by 2-10 folds due to antibody waning and emergence of VOCs, and an additional dose of vaccine would increase PRNT50 titres by 3- or 9-fold. We then used an existing SARS-CoV-2 transmission model to assess the outcomes of vaccine allocation strategies with and without the use of antivirals for symptomatic patients in Japan, Hong Kong and Vietnam.</p></sec><sec><title>Findings</title><p>Increasing primary vaccination coverage was the most important contributing factor in reducing the total and peak number of COVID-19 hospitalizations, especially when population vaccine coverage or vaccine uptake among older adults was low. Providing antivirals to 50% of symptomatic infections only further reduced total and peak hospitalizations by 10-13%. The effectiveness of an additional dose of vaccine was highly dependent on the immune escape potential of VOCs and antibody waning, but less dependent on the boosting efficacy of the additional dose.</p></sec><sec><title>Interpretation</title><p>Increasing primary vaccination coverage should be prioritised in the design of allocation strategies of COVID-19 vaccines and antivirals in the EAP region. Heterologous vaccination with any available vaccine as the additional dose could be considered when planning pandemic exit strategies tailored to the circumstances of EAP jurisdictions.</p></sec><sec><title>Funding</title><p>Health and Medical Research Fund, General Research Fund, AIR@InnoHK</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.22.465468', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': True, 'title': 'PermaPhosSer: autonomous synthesis of functional, permanently phosphorylated proteins', 'authors': \"[{'name': 'Phillip Zhu'}, {'name': 'Rachel Franklin'}, {'name': 'Amber Vogel'}, {'name': 'Stanislau Stanisheuski'}, {'name': 'Patrick Reardon'}, {'name': 'Joseph S Beckman'}, {'name': 'P. Andrew Karplus'}, {'name': 'Ryan A Mehl'}, {'name': 'Richard B Cooley'}]\", 'abstract': '<p>Installing stable, functional mimics of phosphorylated amino acids into proteins offers a powerful strategy to study protein regulation. Previously, a genetic code expansion (GCE) system was developed to translationally install non-hydrolyzable phosphoserine (nhpSer), with the -oxygen replaced with carbon, but it has seen limited usage. Here, we achieve a 40-fold improvement in this system by engineering into <italic>Escherichia coli</italic> a biosynthetic pathway that produces nhpSer from the central metabolite phosphoenolpyruvate. Using this \"PermaPhos<sup>Ser</sup>\" system  an autonomous 21-amino acid <italic>E. coli</italic> expression system for incorporating nhpSer into target proteins  we show that nhpSer faithfully mimics the effects of phosphoserine in three stringent test cases: promoting 14-3-3/client complexation, disrupting 14-3-3 dimers, and activating GSK3- phosphorylation of the SARS-CoV-2 nucleocapsid protein. This facile access to nhpSer containing proteins should allow nhpSer to replace Asp and Glu as the go-to pSer phosphomimetic for proteins produced in <italic>E. coli</italic>.</p>', 'posted': '2021-10-23'}, {'doi': '10.1101/2021.10.19.21265062', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Mutations on non-structural proteins of SARS-CoV-2 are possibly responsible for adverse clinical outcomes in a real-life practice', 'authors': \"[{'name': 'Takaya Ichikawa'}, {'name': 'Shiho Torii'}, {'name': 'Hikoyu Suzuki'}, {'name': 'Akio Takada'}, {'name': 'Satoshi Suzuki'}, {'name': 'Masahide Nakajima'}, {'name': 'Akihito Tampo'}, {'name': 'Yasutaka Kakinoki'}]\", 'abstract': '<p>Among a single COVID-19 cluster population from the end of March through April 2021 in Asahikawa, we experienced the cases in which patients manifested severe clinical symptoms compared to patients who were infected before that. A hundred three patients (age range: 65 to 89 years old) enrolled in this study were divided into two groups, group A: the patients infected from November 2020 to March 2021, and group B: the patients in this cluster population. The mortality rates were 6.1% in group A and 16.2% in group B (OR: 2.97, 95%CI: 0.65-15.38). For the severity of disease, the patients in group B reached to the clinical state which requires higher oxygen flow rate more quickly (mild; p=0.892, moderate; p=0.117, severe; p=0.029). Whole viral genome sequences revealed five non-synonymous mutations by comparison of the isolates with each group. Of these, four were on NSPs including nsp3, 6 and 15, and one was on S protein located near the C-terminus, suggesting that mutations on NSPs could be responsible for adverse clinical outcomes in COVID-19 patients.</p>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.20.21265239', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Gastroenterology', 'covid': True, 'title': 'Immunogenicity of BNT162b2 Vaccine in Patients with Inflammatory Bowel Disease on Infliximab Combination Therapy: A Multicenter Prospective Study', 'authors': \"[{'name': 'Mohammad Shehab'}, {'name': 'Mohamed Abu-Farha'}, {'name': 'Fatema Alrashed'}, {'name': 'Ahmad Alfadhli'}, {'name': 'Khazna AlOtaibi'}, {'name': 'Abdulla AlSahli'}, {'name': 'Thangavel Alphonse Thanaraj'}, {'name': 'Arshad Channanath'}, {'name': 'Hamad Ali'}, {'name': 'Jehad Abubaker'}, {'name': 'Fahd Al-Mulla'}]\", 'abstract': '<sec><title>Background</title><p>Vaccination is a promising strategy to protect vulnerable groups like inflammatory bowel disease (IBD) patients against COVID-19 and associated severe outcomes. COVID-19 Vaccines clinical trials excluded IBD patients taking infliximab with azathioprine or 6-mercaptopurine (infliximab combination). Therefore, we sought to evaluate serologic responses to COVID-19 vaccination with the mRNA vaccine, BNT162b2 in IBD patients receiving infliximab combination therapy compared to healthy participants.</p></sec><sec><title>Methods</title><p>This is a multicenter prospective study. IBD patients were recruited at the time of attendance at infusion center between August 1<sup>st</sup>, 2021, and September 15<sup>th</sup>, 2021. Our primary outcome was the concentrations of SARS-CoV-2 antibodies 4-10 weeks after vaccination with two doses of BNT162b2 vaccine in IBD patients taking infliximab combination therapy (study group) compared to healthy participants group (control group). Both study and healthy participants groups were matched for age, sex and time-since-last-vaccine-dose using optimal pair matching method.</p></sec><sec><title>Results</title><p>In total 116 participants were recruited in the study, 58 patients in the study group and 58 in the control group. Median (IQR) IgG concentrations were lower in the study group [99 BAU/mL (40, 177)] than the control group [139 BAU/mL (120, 188)], following vaccination (p = 0.0032). Neutralizing antibodies was also lower in the study group compared to the control group [64% (23, 94) vs 91% (85, 94), p &lt;0.001]. The median IgA levels in the study group was also significantly lower when compared to the control group [6 U/ml (3, 34) vs 13 U/ml (7, 30), p =0.0097]. In the study group, the percentage of patients who achieved positive IgG, neutralizing antibody and IgA levels were 81%, 75% and 40% respectively. In the control group, all participants (100%) had positive IgG and neutralizing antibody levels while 62% had positive IgA levels.</p></sec><sec><title>Conclusion</title><p>In patients with IBD receiving infliximab combination therapy, IgG, IgA and neutralizing antibody levels after BNT162b2 vaccine were lower compared to healthy participants. However, most patients treated with infliximab combination therapy seroconverted after two doses of the vaccine.</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.21.465371', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Real-Time Structure Search and Structure Classification for AlphaFold Protein Models', 'authors': \"[{'name': 'Tunde Aderinwale'}, {'name': 'Vijay Bharadwaj'}, {'name': 'Charles Christoffer'}, {'name': 'Genki Terashi'}, {'name': 'Zicong Zhang'}, {'name': 'Rhashidedin Jahandideh'}, {'name': 'Yuki Kagaya'}, {'name': 'Daisuke Kihara'}]\", 'abstract': '<p>AlphaFold2 showed a substantial improvement in the accuracy of protein structure prediction. Following the release of the software, whole-proteome protein structure predictions by AlphaFold2 for 21 organisms were made publicly available. Here, we developed the infrastructure, 3D-AF-Surfer, to enable real-time structure-based search for the AlphaFold2 models by combining molecular surface representation with 3D Zernike descriptors and deep neural networks.</p>', 'posted': '2021-10-23'}, {'doi': '10.1101/2021.10.18.21265130', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Survey of Direct and Indirect Effects of COVID-19 on Eyes and the Common Ocular Manifestations', 'authors': \"[{'name': 'Dianeh Rabi'}, {'name': 'Razan Rabi'}, {'name': 'Arkan Jarrar'}, {'name': 'Sarah Mharma'}, {'name': 'Aya Jaradat'}, {'name': 'Shatha Bzoor'}]\", 'abstract': '<sec><title>Purpose</title><p>Ocular manifestations were reported in many recent observations that studied either the effect of COVID-19 directly on eyes or of face mask use. Hence, this study aimed to investigate the effect of COVID-19 on the eyes and make a clear comparison of its direct and indirect effect from face mask-wearing.</p></sec><sec><title>Methods</title><p>This was a cross-sectional study of both written and web-based questionnaires, distributed among a group of COVID-19 patients and a matched control group, the questionnaire consisted of common demographic data, COVID-19 infection history and its symptoms, focusing on ocular symptoms and the presence of conditions related to or cause eye symptoms. As well as the use of face masks that were assessed in terms of the complained ocular manifestation</p></sec><sec><title>Results</title><p>Of 618 participants, 252 had COVID-19 and 366 never had COVID-19. Ocular manifestation among COVID-19 incidence was 44%, significantly higher than non-infected participants incidence (35.8%), adjusted odds ratio, 95% confidence interval (AOR, 95%CI); 1.45 (1.02-2.06)). Eye discharges (p-value = 0.033) and photosensitivity (p-value = 0.003) were noted more commonly among COVID-19 participants compared to healthy control. When comparing long periods of face mask use with each ocular symptom; dry eye based on OSDI, forging body sensation, eye pain and eye discharges, were found significantly common among extended periods of face mask use.</p></sec><sec><title>Conclusion</title><p>COVID-19 pandemic affected eyes, both directly from the virus or from its preventive measure of face mask use.</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.23.465463', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'The mechano-sensitive ion channel Piezo mediates Rho activation and actin stress fibre formation in Drosophila nephrocytes', 'authors': \"[{'name': 'Sybille Koehler'}, {'name': 'Barry Denholm'}]\", 'abstract': '<p>Mechanotransduction is an important process of sensing physical forces in the environment of organisms, tissues and cells and transducing them into a biochemical response. Due to their position on the glomerular capillaries, podocytes are exposed to near-constant biomechanical force, which can fluctuate widely. These include shear stress and hydrostatic pressure. A pathological increase in these forces can induce morphological change to podocytes, their detachment from the glomerular basement membrane and subsequent loss into the primary urine. The ability to sense and respond to variations in mechanical force would be beneficial to a cell exposed to these conditions. It is likely podocytes have such mechanisms, however their identity are unknown. Here we investigated the hypothesis that the mechanotransducer Piezo is involved in a mechanotransduction pathway in Drosophila nephrocytes, the podocyte homologue in the fly. We find Piezo is expressed in nephrocytes and localizes to the nephrocyte diaphragm. The Piezo agonist YODA, which stimulates channel opening in the absence of mechanical force, leads to a significant increase in intracellular Ca++ upon shear stress in the nephrocyte. This leads to activation of Rho1, delineating a putative Piezo mechanotransductive pathway in these cells. Loss of function analysis revealed minor defects in nephrocyte filtration function. In contrast, we show that elevated Piezo levels resulted in constantly oscillating Ca++ signals even in the absence of shear stress, increased active Rho1 and accumulation of actin stress fibers, culminating in a severe nephrocyte filtration phenotype, suggesting that pathway hyperactivity is detrimental. We asked if this phenotype could be reversed by blocking Piezo activity pharmacologically using the tarantula toxin GsMTx4. Treatment with GsMTx4 brought levels of activated Rho1 into the normal range. This work delineates a mechanotransductive pathway in nephrocytes involving Piezo, Ca++, Rho1 and the actin-cytoskeleton, and suggest this is part of a mechanism by which nephrocytes sense and adapt to changes in mechanical force.</p>', 'posted': '2021-10-23'}, {'doi': '10.1101/2021.10.20.465099', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'The superior salinity tolerance of wheat cultivar Shanrong No. 3 cannot be attributed to elevated Ta-sro1 poly(ADP-ribose) polymerase activity', 'authors': \"[{'name': 'Sarah Vogt'}, {'name': 'Karla Feijs'}, {'name': 'Sebastian Hosch'}, {'name': 'Raffaella De Masi'}, {'name': 'Ruth Lintermann'}, {'name': 'Bernhard Loll'}, {'name': 'Lennart Wirthmueller'}]\", 'abstract': '<p>Saline soils limit the production of important staple crops such as wheat, particularly in arid and semiarid regions. Salt tolerance is a multi-gene trait and this complicates breeding of wheat varieties that deliver high yields under saline soil conditions. Notably, the elevated salinity tolerance of wheat cultivar Shanrong No. 3 (SR3) has been linked to a specific proteoform of the wheat SIMILAR TO RCD1 ONE (SRO1) protein that was created in an asymmetric genome hybridization with tall wheat grass. The two amino acid polymorphisms of the Ta-sro1 proteoform enhance the poly(ADP-ribose) polymerase (PARP) activity of the protein suggesting that altered poly-ADP-ribosylation of unknown substrate proteins or nucleic acids underlie the elevated salinity tolerance of cultivar SR3. To elucidate the molecular basis for the elevated PARP activity of the Ta-sro1 proteoform we solved a crystal structure of the catalytic PARP domain. Surprisingly, the structure revealed that the postulated binding site for the co-substrate NAD<sup>+</sup> substantially differs from the structurally conserved NAD<sup>+</sup> binding sites of canonical PARP enzymes. Consistently, we find that Ta-sro1 does not bind NAD<sup>+</sup> and lacks ADP-ribosyltransferase activity. Therefore, although the structure revealed that one of the polymorphic amino acids is located close to the proposed active site, the elevated salinity tolerance of cultivar SR3 cannot be explained by altered ADP-ribosyltransferase activity of Ta-sro1.</p>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.22.465449', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Quantitative fragmentomics allow affinity mapping of interactomes', 'authors': \"[{'name': 'Gergo Gogl'}, {'name': 'Boglarka Zambo'}, {'name': 'Camille Kostmann'}, {'name': 'Alexandra Cousido-Siah'}, {'name': 'Bastien Morlet'}, {'name': 'Fabien Durbesson'}, {'name': 'Luc Negroni'}, {'name': 'Pascal Eberling'}, {'name': 'Pau Jane'}, {'name': 'Yves Nomine'}, {'name': 'Andras Zeke'}, {'name': 'Soren Ostergaard'}, {'name': 'Elodie Monsellier'}, {'name': 'Renaud Vincentelli'}, {'name': 'Gilles Trave'}]\", 'abstract': '<p>Human protein networks have been widely explored but most binding affinities remain unknown, hindering quantitative interactome-function studies. Yet interactomes rely on minimal interacting fragments displaying quantifiable affinities. Here we measured the affinities of 65,000 interactions involving PDZ domains and their target PDZ-binding motifs (PBM) within a human interactome region particularly relevant for viral infection and cancer. We calculate interactomic distances, identify hot spots for viral interference, generate binding profiles and specificity logos, and explain selected cases by crystallographic studies. Mass spectrometry experiments on cell extracts and literature surveys show that quantitative fragmentomics effectively complement protein interactomics by providing affinities and completeness of coverage, putting a full human interactome affinity survey within realistic reach. Finally, we show that interactome hijacking by the viral PBM of human papillomavirus (HPV) E6 oncoprotein deeply impacts the host cell proteome way beyond immediate E6 binders, illustrating the complex system-wide relationship between interactome and function.</p>', 'posted': '2021-10-24'}, {'doi': '10.1101/2021.10.19.21265222', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Monoclonal Antibody Treatment of Breakthrough COVID-19 in Fully Vaccinated Individuals with High-Risk Comorbidities', 'authors': \"[{'name': 'Dennis M. Bierle'}, {'name': 'Ravindra Ganesh'}, {'name': 'Sidna Tulledge-Scheitel'}, {'name': 'Sara N. Hanson'}, {'name': 'Lori L. Arndt'}, {'name': 'Caroline G. Wilker'}, {'name': 'Raymund R. Razonable'}]\", 'abstract': '<p>Breakthrough COVID-19 may occur in fully vaccinated persons. In this cohort of 1395 persons (mean age, 54.3 years; 60% female; median body mass index, 30.7) who developed breakthrough COVID-19, there were 107 (7.7%) who required hospitalization by day 28. Hospitalization was significantly associated with the number of medical comorbidities. Anti-spike monoclonal antibody treatment was significantly associated with a lower risk of hospitalization (Odds Ratio: 0.227; 95% confidence interval, 0.128 - 0.403; p&lt;0.001). The number needed to treat to prevent one hospitalization was 225 among the lowest-risk patient group compared to 4 among the groups with highest numbers of medical comorbidity.</p><sec><title>summary</title><p>Breakthrough COVID-19 may occur among vaccinated individuals with a high number of medical comorbidities, especially during the surge of SARS-CoV-2 Delta variant. Treatment with anti-spike monoclonal antibody was associated with significantly lower rates of hospitalization and oxygen supplementation.</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.19.464965', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Characterising the vegetation-rainfall relationship in the Northeast Himalaya, India', 'authors': \"[{'name': 'Bidyut Sarania'}, {'name': 'Vishwesha Guttal'}, {'name': 'Krishnapriya Tamma'}]\", 'abstract': '<p>Ecosystems are complex systems and are characterised by positive and negative feedbacks between the abiotic and biotic components. The response of an ecosystem to its environment can be determined by examining state diagrams, which are plots of the state variable as a function of the environmental driver. For instance, tree cover as a function of rainfall is widely used to characterise vegetation patterns. Previous studies have shown that tree cover shows bimodal distributions for intermediate rainfall regimes in Africa and South America. In this study, we construct a vegetation state diagram by plotting vegetation cover as a function of mean annual rainfall for Northeast India, which is part of the Eastern Himalaya and the Indo-Burma biodiversity hotspot. We use remotely sensed satellite data of Enhanced Vegetation Index (EVI) as a proxy for vegetation cover. We obtain Mean Annual Precipitation (MAP) from the CHIRPS data (Climate Hazards Group InfraRed Precipitation with Station data). We find that EVI increases monotonically as a function of MAP in the range 1000-2000 mm, after which it plateaus. The 1000 to 2000 mm MAP corresponds to the vegetation transitional zone (1200-3700 m), whereas &gt;2000 MAP region covers the greater extent of the tropical forest (&lt;1200 m) of NEI. In other words, we find no evidence for bimodality in tree cover or vegetation states at coarser scales in North Eastern India. Our characterisation of the state diagram for vegetation in northeast India is important to understand response to ongoing change in rainfall patterns. Keywords: spatial ecology, remotely sensed data, Enhanced Vegetation Index, State diagram, northeast India</p>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.20.465206', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Regulatory variants active in iPSC-derived pancreatic progenitor cells are associated with Type 2 Diabetes in adults', 'authors': '[{\\'name\\': \\'Jennifer P. Nguyen\\'}, {\\'name\\': \"Agnieszka D\\'Antonio-Chronowska\"}, {\\'name\\': \\'Kyohei Fujita\\'}, {\\'name\\': \\'Bianca M. Salgado\\'}, {\\'name\\': \\'Hiroko Matsui\\'}, {\\'name\\': \\'Timothy D. Arthur\\'}, {\\'name\\': \\'Margaret KR Donovan\\'}, {\\'name\\': \"Matteo D\\'Antonio\"}, {\\'name\\': \\'Kelly A Frazer\\'}, {\\'name\\': \\' \\'}]', 'abstract': '<p>Pancreatic progenitor cells (PPC) are an early developmental multipotent cell type that give rise to mature endocrine, exocrine, and ductal cells. To investigate the extent to which regulatory variants active in PPC contribute to pancreatic complex traits and disease in the adult, we derived PPC from induced pluripotent stem cells (iPSCs) of nine unrelated individuals and generated single cell profiles of chromatin accessibility (snATAC-seq) and transcriptome (scRNA-seq). While iPSC-PPC differentiation was asynchronous and included cell types from early to late developmental stages, we found that the predominant cell type consisted of NKX6-1+ progenitors. Genetic characterization using snATAC-seq identified 86,261 regulatory variants that either displayed chromatin allelic bias and/or were predicted to affect active transcription factor (TF) binding sites. Integration of these regulatory variants with 380 fine-mapped type 2 diabetes (T2D) risk loci identified regulatory variants in 209 of these loci that are functional in iPSC-PPC, either by affecting transcription factor binding or through association with allelic effects on chromatin accessibility. The PPC active regulatory variants in 65 of these loci showed strong evidence of causally underlying the association with T2D. Our study shows that studying the functional associations of regulatory variation in iPSC-PPC enables the identification and characterization of causal SNPs for adult Type 2 Diabetes.</p>', 'posted': '2021-10-21'}, {'doi': '10.1101/2021.10.21.465269', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Systematic analysis of intrinsic enhancer-promoter compatibility in the mouse genome', 'authors': \"[{'name': 'Miguel Martinez-Ara'}, {'name': 'Federico Comoglio'}, {'name': 'Joris van Arensbergen'}, {'name': 'Bas van Steensel'}]\", 'abstract': '<p>Gene expression is in part controlled by cis-regulatory elements (CREs) such as enhancers and repressive elements. Anecdotal evidence has indicated that a CRE and a promoter need to be biochemically compatible for promoter regulation to occur, but this compatibility has remained poorly characterised in mammalian cells. We used high-throughput combinatorial reporter assays to test thousands of CRE  promoter pairs from three Mb-sized genomic regions in mouse cells. This revealed that CREs vary substantially in their promoter compatibility, ranging from striking specificity for a single promoter to quantitative differences in activation across a broad set of promoters. More than half of the tested CREs exhibit significant promoter selectivity. Housekeeping promoters tend to have similar CRE preferences, but other promoters exhibit a wide diversity of compatibilities. Higher-order TF motif combinations may account for compatibility. CREpromoter selectivity does not correlate with looping interactions in the native genomic context, suggesting that chromatin folding and compatibility are two orthogonal mechanisms that confer specificity to gene regulation.</p>', 'posted': '2021-10-22'}, {'doi': '10.1101/2021.10.14.21264762', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Longitudinal and comparative analysis of humoral response upon COVID-19 vaccination', 'authors': \"[{'name': 'Salvador Romero-Pinedo'}, {'name': 'Marina Quesada'}, {'name': 'Stela lvarez-Fernndez'}, {'name': 'Asuncin Olmo'}, {'name': 'David Abia'}, {'name': 'Balbino Alarcn'}, {'name': 'Pilar Delgado'}]\", 'abstract': '<p>The emergence of COVID-19 has led to a worldwide challenge for the rapid development of vaccines. Several types of safe and effective vaccines have been available in a time frame never seen before. Comparative studies to know the extent of protection and the immune response elicited by the different vaccines are of outstanding utility. Here, as a correlate for protection, we perform a comparative study of the humoral response to three vaccines, ChAdOx1 (Oxford-AstraZeneca), mRNA-1273 (Moderna), and BNT162b2 (Pfizer-BioNTech) by applying a flow cytometry-based highly sensitive method that we had previously developed. We have found that mRNA vaccines (mRNA-1273 and BNT162b2) induce a stronger humoral response that lasts for at least 6 months after vaccination. We also show that only one dose of BNT162b2 is enough to achieve the maximum response in seropositive pre-vaccination donors.</p>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.18.464824', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Transcriptional variability accelerates pre-leukemia by cell diversification and perturbation of protein synthesis', 'authors': \"[{'name': 'Shikha Gupta'}, {'name': 'Oliver M. Dovey'}, {'name': 'Ana Filipa Domingues'}, {'name': 'Oliwia W. Cyran'}, {'name': 'Caitlin M. Cash'}, {'name': 'George Giotopoulos'}, {'name': 'Justyna Rak'}, {'name': 'Jonathan Cooper'}, {'name': 'Malgorzata Gozdecka'}, {'name': 'Ryan J. Asby'}, {'name': 'Noor Al-Jabery'}, {'name': 'Victor Hernandez-Hernandez'}, {'name': 'Sudhakaran Prabakaran'}, {'name': 'Brian J. Huntly'}, {'name': 'George S. Vassiliou'}, {'name': 'Cristina Pina'}]\", 'abstract': '<p>Transcriptional variability facilitates stochastic cell diversification and can in turn underpin adaptation to stress or injury. We hypothesize that it may analogously facilitate progression of pre-malignancy to cancer. To investigate this, we initiated pre-leukemia in mouse cells with enhanced transcriptional variability due to conditional disruption of the histone lysine acetyltransferase gene <italic>Kat2a</italic>. By combining single-cell RNA-sequencing of pre-leukemia with functional analysis of transformation, we show that <italic>Kat2a</italic> loss results in global variegation of cell identity and accumulation of pre-leukemic cells. Leukemia progression is subsequently facilitated by destabilization of ribosome biogenesis and protein synthesis, which confer a transient transformation advantage. The contribution of transcriptional variability to early cancer evolution reflects a generic role in promoting cell fate transitions, which, in the case of well-adapted malignancies, contrastingly differentiates and depletes cancer stem cells. In other words, transcriptional variability confers forward momentum to cell fate systems, with differential multi-stage impact throughout cancer evolution.</p><sec><title>One-sentence summary</title><p>Loss of Kat2a enhances transcriptional variability of ribosome biosynthetic programs and transiently accelerates pre-leukemia</p></sec>', 'posted': '2021-10-18'}, {'doi': '10.1101/2021.10.22.465514', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'E-Cannula reveals anatomical diversity in sharp-wave ripples as a driver for the recruitment of distinct hippocampal assemblies', 'authors': \"[{'name': 'Xin Liu'}, {'name': 'Satoshi Terada'}, {'name': 'Jeonghoon Kim'}, {'name': 'Yichen Lu'}, {'name': 'Mehrdad Ramezani'}, {'name': 'Andres D Grosmark'}, {'name': 'Attila Losonczy'}, {'name': 'Duygu Kuzum'}]\", 'abstract': '<p>The hippocampus plays a critical role in spatial navigation and episodic memory. However, research on in vivo hippocampal activity dynamics has mostly relied on single modalities such as electrical recordings or optical imaging, with respectively limited spatial and temporal resolution. This technical difficulty greatly impedes multi-level investigations into network state-related changes in cellular activity. To overcome these limitations, we developed the E-Cannula integrating fully transparent graphene microelectrodes with imaging-cannula. The E-Cannula enables the simultaneous electrical recording and two-photon calcium imaging from the exact same population of neurons across an anatomically extended region of the mouse hippocampal CA1 stably across several days. These large-scale simultaneous optical and electrical recordings showed that local hippocampal sharp wave ripples (SWRs) are associated with synchronous calcium events involving large neural populations in CA1. We show that SWRs exhibit spatiotemporal wave patterns along multiple axes in 2D space with different spatial extents (local or global) and temporal propagation modes (stationary or travelling). Notably, distinct SWR wave patterns were associated with, and decoded from, the selective recruitment of orthogonal CA1 cell assemblies. These results suggest that the diversity in the anatomical progression of SWRs may serve as a mechanism for the selective activation of the unique hippocampal cell assemblies extensively implicated in the encoding of distinct memories. Through these results we demonstrate the utility of the E-Cannula as a versatile neurotechnology with the potential for future integration with other optical components such as green lenses, fibers or prisms enabling the multi-modal investigation of cross-time scale population-level neural dynamics across brain regions.</p>', 'posted': '2021-10-24'}, {'doi': '10.1101/2021.10.19.21265230', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Psychological, social and financial impact of COVID-19 on culturally and linguistically diverse communities: a cross-sectional Australian study', 'authors': \"[{'name': 'Danielle M Muscat'}, {'name': 'Julie Ayre'}, {'name': 'Olivia Mac'}, {'name': 'Carys Batcup'}, {'name': 'Erin Cvejic'}, {'name': 'Kristen Pickles'}, {'name': 'Hankiz Dolan'}, {'name': 'Carissa Bonner'}, {'name': 'Dana Mouwad'}, {'name': 'Dipti Zachariah'}, {'name': 'Una Turalic'}, {'name': 'Yvonne Santalucia'}, {'name': 'Tingting Chen'}, {'name': 'Gordana Vasic'}, {'name': 'Kirsten J McCaffery'}]\", 'abstract': '<p>Objective: This study aimed to explore the psychological, social, and financial impacts of COVID-19 on culturally and linguistically diverse communities in Australia.\\n\\nDesign: Cross-sectional survey informed by the Framework for Culturally Competent Health Research conducted between March and July, 2021.\\n\\nSetting: Participants were recruited from Greater Western Sydney, New South Wales, Australia.\\n\\nParticipants: 708 community members who speak a language other than English at home participated (mean age: 45.4years [range 18 to 91]; 88% [n=622] born outside of Australia).\\n\\nOutcome measures: Fifteen items regarding impacts of COVID-19, adapted from validated scales, previous surveys or co-designed in partnership with Multicultural Health and interpreter service staff. Logistic regression models (using post-stratification weighted frequencies) identified factors associated with psychological, social, and financial impacts. Surveys were available in English or translated (11 languages). \\n\\nResults: Even prior to the COVID-19 outbreak in Sydney, 25% of the sample reported feeling nervous or stressed most/all of the time and 22% felt lonely or alone most/all of the time. One quarter of participants reported negative impacts on their spousal relationships as a result of COVID-19 and most parents reported that their children were less active (64%), had more screen time (63%), and were finding school harder (45%). Mean financial burden was 2.9/5 (95%CI=2.8 to 2.9). Regression analyses consistently showed distinct impact patterns for different language groups and more negative outcomes for those with comorbidities. \\n\\nConclusion: Culturally and linguistically diverse communities experience significant psychological, social and financial impacts of COVID-19, with distinct impact patterns across language groups. A whole-of-government approach with policy and sustainable infrastructure is needed to co-design innovative, tailored and culturally-safe COVID-19 support packages.</p>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.21.465321', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'PI4P and BLOC-1 remodel endosomal membranes into tubules', 'authors': \"[{'name': 'Riddhi Atul Jani'}, {'name': 'Aurlie Di Cicco'}, {'name': 'Tal Keren-Kaplan'}, {'name': 'Silvia Vale-Costa'}, {'name': 'Daniel Hamaoui'}, {'name': 'Ilse Hurbain'}, {'name': 'Feng-Ching Tsai'}, {'name': 'Mathilde Dimarco'}, {'name': 'Anne-Sophie Mac'}, {'name': 'Yueyao Zhu'}, {'name': 'Maria Joo Amorim'}, {'name': 'Patricia Bassereau'}, {'name': 'Juan S. Bonifacino'}, {'name': 'Agathe Subtil'}, {'name': 'Michael S. Marks'}, {'name': 'Daniel Lvy'}, {'name': 'Graa Raposo'}, {'name': 'Cdric Delevoye'}]\", 'abstract': '<p>Intracellular trafficking is mediated by transport carriers that originate by membrane remodeling from donor organelles. Tubular carriers play major roles in the flux of membrane lipids and proteins to acceptor organelles. However, how lipids and proteins impose a tubular geometry on the carriers is incompletely understood. By exploiting imaging approaches at different scales on cells and in vitro membrane systems, we show that phosphatidylinositol-4-phosphate (PI4P) and biogenesis of lysosome-related organelles complex 1 (BLOC-1) govern the formation, stability and functions of recycling endosomal tubules. Endosomal PI4P production by type II PI4-kinases is needed to form nascent curved tubules through binding of BLOC-1 that stabilize and elongate them. Membrane remodeling by the PI4P/ BLOC-1 module functions not only in the recycling of endosomal cargoes, but also in the lifecycles of intracellular pathogens such as <italic>Chlamydia</italic> bacteria and influenza virus. This study demonstrates how a phospholipid and a protein complex coordinate as a minimal machinery to remodel cellular membranes into functional tubes.</p>', 'posted': '2021-10-21'}, {'doi': '10.1101/2021.10.20.465103', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Highly efficient generation of canker resistant sweet orange enabled by an improved CRISPR/Cas9 system', 'authors': \"[{'name': 'Xiaoen Huang'}, {'name': 'Nian Wang'}]\", 'abstract': '<p>Sweet orange (<italic>Citrus sinensis</italic>) is the most economically important species for the citrus industry. However, it is susceptible to many diseases including citrus bacterial canker caused by <italic>Xanthomonas citri</italic> subsp. citri (<italic>Xcc</italic>) that triggers devastating effects on citrus production. Conventional breeding has not met the challenge to improve disease resistance of sweet orange due to the long juvenility and other limitations. CRISPR-mediated genome editing has shown promising potentials for genetic improvements of plants. Generation of biallelic/homozygous mutants remains difficult for sweet orange due to low transformation rate, existence of heterozygous alleles for target genes and low biallelic editing efficacy using the CRISPR technology. Here, we report improvements in the CRISPR/Cas9 system for citrus gene editing. Based on the improvements we made previously (dicot codon optimized Cas9, tRNA for multiplexing, a modified sgRNA scaffold with high efficiency, CsU6 to drive sgRNA expression), we further improved our CRISPR/Cas9 system by choosing superior promoters (CmYLCV or CsUbi promoter) to drive Cas9 and optimizing culture temperature. This system was able to generate a biallelic mutation rate of up to 89% for Carrizo citrange and 79% for Hamlin sweet orange. Consequently, this system was used to generate canker resistant Hamlin sweet orange by mutating the effector binding element (EBE) of canker susceptibility gene <italic>CsLOB1</italic>, which is required for causing canker symptoms by <italic>Xcc</italic>. Six biallelic Hamlin sweet orange mutant lines in the EBE were generated. The biallelic mutants are resistant to <italic>Xcc</italic>. Biallelic mutation of the EBE region abolishes the induction of <italic>CsLOB1</italic> by <italic>Xcc</italic>. This study represents a significant improvement in sweet orange gene editing efficacy and generating disease resistant varieties via CRISPR-mediated genome editing. This improvement in citrus genome editing makes genetic studies and manipulations of sweet orange more feasible.</p>', 'posted': '2021-10-21'}, {'doi': '10.1101/2021.10.22.465537', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Amplicon sequencing of single-copy protein-coding genes reveals accurate diversity for sequence-discrete microbiome populations', 'authors': \"[{'name': 'Chengfeng Yang'}, {'name': 'Qinzhi Su'}, {'name': 'Min Tang'}, {'name': 'Shiqi Luo'}, {'name': 'Hao Zheng'}, {'name': 'Xue Zhang'}, {'name': 'Xin Zhou'}]\", 'abstract': '<p>An in-depth understanding of microbial function and the division of ecological niches requires accurate delineation and identification of microbes at a fine taxonomic resolution. Microbial phylotypes are typically defined using a 97% small subunit (16S) rRNA threshold. However, increasing evidence has demonstrated the ubiquitous presence of taxonomic units of distinct functions within phylotypes. These so-called sequence-discrete populations (SDPs) have used to be mainly delineated by disjunct sequence similarity at the whole-genome level. However, gene markers that could accurately identify and quantify SDPs are lacking in microbial community studies. Here we developed a pipeline to screen single-copy protein-coding genes that could accurately characterize SDP diversity via amplicon sequencing of microbial communities. Fifteen candidate marker genes were evaluated using three criteria (extent of sequence divergence, phylogenetic accuracy, and conservation of primer regions) and the selected genes were subject to test the efficiency in differentiating SDPs within Gilliamella, a core honeybee gut microbial phylotype, as a proof-of-concept. The results showed that the 16S V4 region failed to report accurate SDP diversities due to low taxonomic resolution and changing copy numbers. In contrast, the single-copy genes recommended by our pipeline were able to successfully quantify Gilliamella SDPs for both mock samples and honeybee guts, with results highly consistent with those of metagenomics. The pipeline developed in this study is expected to identify single-copy protein coding genes capable of accurately quantifying diverse bacterial communities at the SDP level.</p>', 'posted': '2021-10-23'}, {'doi': '10.1101/2021.10.23.465551', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Retraction ATPase motors are promiscuous among type IVa pilus systems', 'authors': \"[{'name': 'Evan Couser'}, {'name': 'Jennifer L Chlebek'}, {'name': 'Ankur B Dalia'}]\", 'abstract': \"<p>Bacterial surface appendages called type IVa pili (T4aP) promote diverse activities including DNA uptake, twitching motility, and virulence. These activities rely on the ability of T4aP to dynamically extend and retract from the cell surface. Dynamic extension relies on a motor ATPase commonly called PilB. Most T4aP also rely on specific motor ATPases, commonly called PilT and PilU, to dynamically and forcefully retract. Here, we systematically assess whether motor ATPases from 4 distinct T4aP could functionally complement Vibrio cholerae mutants that lacked their endogenous motors. We found that the retraction ATPases PilT and PilU are highly promiscuous and promote retraction of the V. cholerae competence T4aP despite a high degree of sequence divergence. In contrast, orthologous extension ATPases were not able to mediate extension of the V. cholerae competence T4aP despite a similar degree of sequence divergence. Also, we show that one of the PilT orthologs characterized does not support PilU-dependent retraction and we attributed this loss of activity to the 3' end of the gene, which suggests that the C-terminus of PilT plays an important role in promoting PilU-dependent retraction. Together, our data suggest that retraction ATPases have maintained a high degree of promiscuity for promoting retraction of diverse T4aP, while extension ATPases have evolved to become highly specific for their cognate systems.</p>\", 'posted': '2021-10-23'}, {'doi': '10.1101/2021.10.13.21264693', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Modeling brings additional insights into the kinetics of SARS-CoV-2 neutralizing antibody', 'authors': \"[{'name': 'Prague Mlanie'}, {'name': 'Quentin Clairon'}, {'name': 'Jrmie Guedj'}, {'name': 'Rodolphe Thibaut'}]\", 'abstract': '<p>Following the paper by Seow et al. published in Nature Microbiology, we reanalyzed the publicly available data using dynamical models of humoral response. The main conclusion is that, from available data, we can demonstrate that the decline of neutralizing antibodies (as measured with ID50) is biphasic, which is compatible with two types of antibody secreting cells (short lived and long lived). We found that lower bound of half life of the long-lived antibody secreting cells is 450 days. Moreover, our model predicts that the neutralizing antibody response could be more durable than suggested (up to 129 days for individuals with no requirement of supplemental oxygen and up to 175 days for others). A result which adds insight on the longevity of immune response.</p>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.20.465087', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'A Network-based Deep Learning Framework Catalyzes GWAS and Multi-Omics Findings to Biology and Drug Repurposing for Alzheimers Disease', 'authors': \"[{'name': 'Jielin Xu'}, {'name': 'Yuan Hou'}, {'name': 'Yadi Zhou'}, {'name': 'Ming Hu'}, {'name': 'Feixiong Cheng'}]\", 'abstract': '<p>Human genome sequencing studies have identified numerous loci associated with complex diseases, including Alzheimers disease (AD). Translating human genetic findings (i.e., genome-wide association studies [GWAS]) to pathobiology and therapeutic discovery, however, remains a major challenge. To address this critical problem, we present a <underline><bold>net</bold></underline>work <underline><bold>t</bold></underline>opology-based deep learning framework to identify disease-<underline><bold>a</bold></underline>ssociated <underline><bold>g</bold></underline>enes (NETTAG). NETTAG is capable of integrating multigenomics data along with the protein-protein interactome to infer putative risk genes and drug targets impacted by GWAS loci. Specifically, we leverage non-coding GWAS loci effects on expression quantitative trait loci (eQTLs), histone-QTLs, and transcription factor binding-QTLs, enhancers and CpG islands, promoter regions, open chromatin, and promoter flanking regions. The key premises of NETTAG are that the disease risk genes exhibit distinct functional characteristics compared to non-risk genes and therefore can be distinguished by their aggregated genomic features under the human protein interactome. Applying NETTAG to the latest AD GWAS data, we identified 156 putative AD-risk genes (i.e., <italic>APOE</italic>, <italic>BIN1</italic>, <italic>GSK3B</italic>, <italic>MARK4</italic>, and <italic>PICALM</italic>). We showed that predicted risk genes are: 1) significantly enriched in AD-related pathobiological pathways, 2) more likely to be differentially expressed regarding transcriptome and proteome of AD brains, and 3) enriched in druggable targets with approved medicines (i.e., choline and ibudilast). In summary, our findings suggest that understanding of human pathobiology and therapeutic development could benefit from a network-based deep learning methodology that utilizes GWAS findings under the multimodal genomic analyses.</p>', 'posted': '2021-10-21'}, {'doi': '10.1101/2021.10.18.21264783', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'SARS-CoV-2 variant transmission in a community-health population (Mexico City, Mexico)', 'authors': \"[{'name': 'Wenjuan Zhang'}, {'name': 'Marcela Martinez'}, {'name': 'Brian D Davis'}, {'name': 'Stephanie S Chen'}, {'name': 'Jorge Sincuir Martinez'}, {'name': 'Clara Corona'}, {'name': 'Guadalupe Diaz'}, {'name': 'Elias Makhoul'}, {'name': 'Saleh Heneidi'}, {'name': 'Jorge Goldberg'}, {'name': 'Jasmine T Plummer'}, {'name': 'Eric Vail'}]\", 'abstract': '<p>The SARS-CoV-2 variant, B.1.1.519, arose in North and Central America, circulating primarily in Mexico. We demonstrate that this variant peaked during the second wave of COVID-19 in Mexico City in the spring of 2021. This variant is likely more infectious, attributed to mutation in the RBD of the spike protein T478K also seen in the alpha variant (B.1.1.7). However the time dynamics of the spread of this variant drastically changed upon the introduction of delta (B.1.617.2) to the country in which we observe a shift from 0% in May 2021 to 55% delta in the span of one month. Since the delta variant has dominantly spread across the globe, we investigated the increasing frequency of the Mexico variant, B.1.1.519, in the public community within Mexico City. Once present, the delta variant was 78% of the Mexico City catchment in July 2021, a time which marked the commencement of Mexicos third wave. Our data supports the growing concern that the delta variant is closely associated with the massive infection spread of the VOC in Central and South America. While the T478K mutation, also seen in the alpha variant, has evidence for increased transmissibility, these data suggest that the delta variant shows overall increased fitness seeing as it outcompeted the B.1.1519 this Mexico community.</p>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.20.21265279', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': True, 'title': 'Factors affecting the mental health of pregnant women using UK maternity services during the COVID-19 pandemic: A qualitative interview study', 'authors': \"[{'name': 'A R McKinlay'}, {'name': 'D Fancourt'}, {'name': 'A Burton'}]\", 'abstract': '<sec><title>Background</title><p>People using maternity services in the United Kingdom (UK) have faced significant changes brought on by the COVID-19 pandemic and social distancing regulations. We focused on the experiences of pregnant women using UK maternity services during the pandemic and the impact of social distancing rules on their mental health and wellbeing.</p></sec><sec><title>Methods</title><p>We conducted 23 qualitative semi-structured interviews from June 2020 to August 2021, with women from across the UK who experienced a pregnancy during the pandemic. Nineteen women in the study carried their pregnancy to term and four women experienced a miscarriage during the pandemic. Interviews took place remotely over video or telephone call, discussing topics such as mental health during pregnancy and use of UK maternity services. We used reflexive thematic analysis to analyse interview transcripts.</p></sec><sec><title>Results</title><p>We generated six higher order themes: (1) Some pregnancy discomforts alleviated by social distancing measures, (2) The importance of relationships that support coping and adjustment, (3) Missed pregnancy and parenthood experiences, (4) The mental health consequences of birth partner and visitor restrictions, (5) Maternity services under pressure, and (6) Lack of connection with staff. Many participants felt a sense of loss over a pregnancy experience that differed so remarkably to what they had expected because of the pandemic. Supportive relationships were important to help cope with pregnancy and pandemic-related changes; but feelings of isolation were compounded for some participants because opportunities to build social connections through face-to-face parent groups were unavailable. Participants also described feeling alone due to restrictions on partners being present when accessing UK maternity services.</p></sec><sec><title>Conclusions</title><p>Our findings highlight some of the changes that may have affected pregnant womens mental health during the COVID-19 pandemic. Reduced social support and being unable to have a partner or support person present during maternity service use were the greatest concerns reported by women in this study, as this absence removed a protective buffer in times of uncertainty and distress. This suggests that the availability of a birth partner or support person must be prioritised wherever possible to protect the mental health of women experiencing pregnancy and miscarriage in times of pandemics.</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.13.21264946', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Intensive Care and Critical Care Medicine', 'covid': True, 'title': 'Clinical observation of high-flow nasal cannula (HFNC) with non-rebreather mask (NRM) use on severe or critically ill COVID-19 diabetic patients', 'authors': \"[{'name': 'AFM Tareq Bhuiyan'}, {'name': 'Sudipta Deb Nath'}, {'name': 'Shuva Das'}, {'name': 'Moumita Das'}, {'name': 'Rajdeep Biswas'}, {'name': 'Moinul Ahsan'}, {'name': 'Md. Iftekher-E-Alam Ziad'}, {'name': 'Fahmida Khatun Padma'}, {'name': 'Rana Dey'}, {'name': 'AKM Shamsul Alam'}, {'name': 'Farial Hoque Zehan'}, {'name': 'Ayan Saha'}]\", 'abstract': \"<p>Objective: Diabetes prevalence is a vital factor in COVID-19's clinical prognosis. This study aimed to investigate and compare the efficacy of High-flow Nasal Cannula (HFNC) with/without non-rebreather mask (NRM) on critical COVID-19 patients with/without diabetes. \\nMaterials and methods: For analysis and comparison, epidemiological, biochemical, and clinical data were collected from 240 HFNCNRM treated severe and critical COVID-19 ICU patients (diabetic = 136; non-diabetic = 104) of five hospitals in Chattogram, Bangladesh. \\nResults and Discussion: 59.1% patients with fever had diabetes (p=0.012). ICU stay was longer for diabetic patients (9.065.70) than non-diabetic ones (7.415.11) (p=0.020). Majority of hypertensive patients were diabetic (68.3%; p&lt;0.001). Most of the diabetic patients (70.4%; p&lt;0.005) had elevated creatinine levels. The partial pressure of oxygen after HFNC (only) was significantly (p=0.031) higher in non-diabetic patients (69.3023.56) than diabetic ones (61.5014.49). Diabetic (62.6413.05) and non-diabetic patients (59.4013.22) had similar partial pressure of oxygen from HFNC+NRM. Majority of the diabetic patients who required HFNC+NRM had elevated RBS (73.8%; p=0.001) and creatinine (75.7%; p=0.009). Factors affecting the HFNC only treated patients were fever and impaired glucose tolerance. Besides, increased plasma glucose level, age, and hypertension affected the HFNC + NRM treated diabetic patients.\\nConclusion: The results of this study imply that oxygen supply with HFNC+NRM may be beneficial for the elderly/hypertensive diabetic patients with COVID-19 associated AHRF; and that IGT and increased blood glucose levels could be determinants for COVID-19 severity. However, further experiments to substantiate these claims are required on a larger sample and among different clinical cohorts.</p>\", 'posted': '2021-10-18'}, {'doi': '10.1101/2021.10.19.465067', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Phospho-regulation accommodates Type III secretion and assembly of a tether of ER-Chlamydia inclusion membrane contact sites', 'authors': \"[{'name': 'Rebecca L. Murray'}, {'name': 'Rachel J. Ende'}, {'name': 'Samantha K. DSpain'}, {'name': 'Isabelle Derr'}]\", 'abstract': '<p>Membrane contact sites (MCS) are crucial for non-vesicular trafficking-based inter-organelle communication. ER-organelle tethering occurs in part through the interaction of the ER resident protein VAP with FFAT-motif containing proteins. FFAT motifs are characterized by a seven amino acidic core surrounded by acid tracks. We have previously shown that the human intracellular bacterial pathogen <italic>Chlamydia trachomatis</italic> establishes MCS between its vacuole (the inclusion) and the ER through expression of a bacterial tether, IncV, displaying molecular mimicry of eukaryotic FFAT motif cores. Here, we show that multiple layers of host cell kinase-mediated phosphorylation events govern the assembly of the IncV-VAP tethering complex. CK2-mediated phosphorylation of a C-terminal region of IncV enables IncV hyperphosphorylation of a phospho- FFAT motif core and serine-rich tracts immediately upstream of IncV FFAT motif cores. Phosphorylatable serine tracts, rather than genetically-encoded acidic tracts, accommodate Type III-mediated translocation of IncV to the inclusion membrane, while achieving full mimicry of FFAT motifs. Thus, regulatory components and post-translational modifications are integral to MCS biology, and intracellular pathogens such as <italic>C. trachomatis</italic> have evolved complex molecular mimicry of these eukaryotic features.</p>', 'posted': '2021-10-21'}, {'doi': '10.1101/2021.10.19.21264255', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Gastroenterology', 'covid': False, 'title': 'Metabolic Dysfunction Related Liver Disease as a Risk Factor for Cancer', 'authors': \"[{'name': 'Alasdair Taylor'}, {'name': 'Moneeza K Siddiqui'}, {'name': 'Philip Ambery'}, {'name': 'Javier Armisen'}, {'name': 'Benjamin G. Challis'}, {'name': 'Carolina Haefliger'}, {'name': 'Ewan R. Pearson'}, {'name': 'Alex S.F. Doney'}, {'name': 'John F. Dillon'}, {'name': 'Colin N.A. Palmer'}]\", 'abstract': '<p>The aim of this study was to investigate the association with obesity, diabetes and related liver dysfunction and the incidence of cancer.</p><p>This study was conducted with health record data available from the National Health Service in Tayside and Fife. GoDARTS, SHARE and Tayside and Fife diabetics, three Scottish cohorts of 13,695, 62,438, and 16,312 patients respectively were analysed in this study. Participants in GoDARTS were a volunteer sample, with half having T2DM. SHARE were a volunteer sample. Tayside and Fife diabetics was a population level cohort. Metabolic dysfunction-related liver disease (MDLD) was defined using ALT measurements, and individuals with alternative causes of liver disease (alcohol abuse, viruses etc) were excluded from the analysis. Other indicators of liver disease were analysed including the Fatty Liver Index, Fibrosis Score(FIB-4) and hospital admissions for NASH. The main outcomes were cancer incidence and cancer death.</p><p>MDLD associated with increased cancer incidence with a hazard ratio of 1.31 in a cox proportional hazards model adjusted for sex, type 2 diabetes, BMI, and smoking status (95% CI = 1.27  1.35, p &lt; 0.0001). This was replicated in two further cohorts, and similar associations with cancer incidence were found for Fatty Liver Index (FLI), FIB-4 and NASH. Homozygous carriers of the common NAFLD risk variant PNPLA3 rs738409 had increased risk of cancer. (HR = 1.27 (1.02-1.58), p = 3.110<sup>2</sup>). BMI was not independently associated with cancer incidence when MDLD was included as a covariate. MDLD was associated with increased risk of cancer death (HR = 1.40, 95% CI =1.33 - 1.47, p &lt; 0.0001).</p><p>MDLD, FLI, FIB-4 and NASH associated with increased risk of cancer incidence and death. Further, we found evidence of a causal association between NAFLD and cancer using the established causal risk allele of PNPLA3 as a genetic instrument. NAFLD may be a major component of the relationship between obesity and cancer incidence.</p><sec><title>Lay Summary</title><p>We found that individuals with metabolic dysfunction-related liver disease (MDLD) have higher overall cancer risk than healthy individuals, as well as increased risk of specific cancers such as colon, breast and lung. We also show that when MDLD is accounted for, obesity does not significantly increase overall cancer risk.</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.18.464802', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Microglia remodel the synaptic signaling required for context-dependent cognitive performance', 'authors': \"[{'name': 'Renato Socodato'}, {'name': 'Tiago O. Almeida'}, {'name': 'Camila C. Portugal'}, {'name': 'Evelyn C. S. Santos'}, {'name': 'Joana Tedim-Moreira'}, {'name': 'Teresa Canedo'}, {'name': 'Filipa I. Baptista'}, {'name': 'Ana Magalhes'}, {'name': 'Antnio F. Ambrsio'}, {'name': 'Cord Brakebusch'}, {'name': 'Boris Rubinstein'}, {'name': 'Irina Moreira'}, {'name': 'Teresa Summavielle'}, {'name': 'Ins Mendes Pinto'}, {'name': 'Joo B. Relvas'}]\", 'abstract': '<p>Microglia modulate synaptic activity, essential for context-dependent cognitive performance, allowing organism-level adaptations to different environmental scenarios. Yet, the microglial molecular drivers required for synaptic remodeling related to cognitive performance remain largely elusive. Here, combining conditional gene targeting, single-cell live imaging, RNA-seq, high-throughput proteomics, systems biology, and animal behavior, we mapped a molecular nexus between microglia and synapses that instructs cognitive performance. Specifically, we found that microglia use the RhoGTPase Rac1 as a relay switch to sense the brain microenvironment and drive synaptic remodeling required for experience-dependent sociability and learning related to memory. Targeting this microglial relay modifies context-dependent cognitive performance.</p>', 'posted': '2021-10-19'}, {'doi': '10.1101/2021.10.18.21265057', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Impact of routine asymptomatic screening on COVID-19 incidence in a highly vaccinated university population', 'authors': \"[{'name': 'Rebeca C. Falco'}, {'name': 'Michael Otterstatter'}, {'name': 'May A. Ahmed'}, {'name': 'Michelle Spencer'}, {'name': 'Sarafa Iyaniwura'}, {'name': 'Naveed Z. Janjua'}, {'name': 'Geoff McKee'}, {'name': 'Michael A. Irvine'}]\", 'abstract': '<sec><title>Background</title><p>With the return of in-person classes, an understanding of COVID-19 transmission in vaccinated university campuses is essential. Given the context of high anticipated vaccination rates and other measures, there are outstanding questions of the potential impact of campus-based asymptomatic screening.</p></sec><sec><title>Methods</title><p>We estimated the expected number of cases and hospitalizations in one semester using rates derived for British Columbia (BC), Canada up to September 15<sup>th</sup>, 2021 and age-standardizing to a University population. To estimate the expected number of secondary cases averted due to routine tests of unvaccinated individuals in a BC post-secondary institution, we used a probabilistic model based on the incidence, vaccination effectiveness, vaccination coverage and <italic>R</italic><sub>0</sub>. We examined multiple scenarios of vaccine coverage, screening frequency, and pre-vaccination <italic>R</italic><sub>0</sub>.</p></sec><sec><title>Results</title><p>For one 12 week semester, the expected number of cases is 67 per 50,000 for 80% vaccination coverage and 37 per 50,000 for 95% vaccination coverage. Screening of the unvaccinated population averts an expected 6-16 cases per 50,000 at 80% decreasing to 1-2 averted cases per 50,000 at 95% vaccination coverage for weekly to daily screening. Further scenarios can be explored using a web-based application.</p></sec><sec><title>Interpretation</title><p>Routine screening of unvaccinated individuals may be of limited benefit if vaccination coverage is 80% or greater within a university setting.</p></sec>', 'posted': '2021-10-20'}, {'doi': '10.1101/2021.10.20.465108', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'hyperTRIBER: a flexible R package for the analysis of differential RNA editing', 'authors': \"[{'name': 'Sarah Rennie'}, {'name': 'Daniel Heidar Magnusson'}, {'name': 'Robin Andersson'}]\", 'abstract': '<p>RNA editing by ADAR (adenosine deaminase acting on RNA) is gaining an increased interest in the field of post-transcriptional regulation. Fused to an RNA-binding protein (RBP) of interest, the catalytic activity of ADAR results in A-to-I RNA edits, whose identification will determine RBP-bound RNA transcripts. However, the computational tools available for their identification and differential RNA editing statistical analysis are limited or too specialised for general-purpose usage. Here we present <monospace>hyperTRIBER</monospace>, a flexible suite of tools, wrapped into a convenient R package, for the detection of differential RNA editing. <monospace>hyperTRIBER</monospace> is applicable to complex scenarios and experimental designs, and provides a robust statistical framework allowing for the control for coverage of reads at a given base, the total expression level and other co-variates. We demonstrate the capabilities of our approach on HyperTRIBE RNA-seq data for the detection of bound RNAs by the <italic>N6</italic>-methyladenosine (m<sup>6</sup>A) reader protein ECT2 in Arabidopsis roots. We show that <monospace>hyperTRIBER</monospace> finds edits with a high statistical power, even where editing proportions and RNA transcript expression levels are low, together demonstrating its usability and versatility for analysing differential RNA editing.</p>', 'posted': '2021-10-21'}, {'doi': '10.1101/2021.10.21.465238', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Automated and parallelized spike collision tests to identify spike signal projections', 'authors': \"[{'name': 'Keita Mitani'}, {'name': 'Masanori Kawabata'}, {'name': 'Yoshikazu Isomura'}, {'name': 'Yutaka Sakai'}]\", 'abstract': '<p>The spike collision test is a highly reliable technique to identify axonal projection of a neuron recorded electrophysiologically for investigating functional spike information among brain areas. It is potentially applicable to more neuronal projections by combining multi-channel recording with optogenetic stimulation. Yet, it remains inefficient and laborious because an experimenter must visually select spikes in every channel and manually repeat spike collision tests for each neuron serially. Here, we established a novel technique to automatically perform spike collision tests for all channels in parallel (Multi-Linc analysis), employing two distinct protocols implemented in a multi-channel real-time processing system. The rat cortical neurons identified with this technique displayed physiological spike features consistent with excitatory projection neurons. Their antidromic spikes were similar in shape but slightly larger in amplitude compared with spontaneous spikes. In addition, we demonstrated simultaneous identification of reciprocal or bifurcating projections among cortical areas. Thus, our Multi-Linc analysis will be a powerful research approach to elucidate interareal spike communication.</p>', 'posted': '2021-10-22'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.10.19.21265208</td>\n",
       "      <td>1705</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Airborne transmission of SARS-CoV-2 over dista...</td>\n",
       "      <td>[{'name': 'Jennifer C Palmer'}, {'name': 'Daph...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Objective&lt;/title&gt;&lt;p&gt;To evaluate th...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.10.18.21264623</td>\n",
       "      <td>419</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 variants associated with vaccine br...</td>\n",
       "      <td>[{'name': 'Andrew D. Marques'}, {'name': 'Scot...</td>\n",
       "      <td>&lt;p&gt;The severe acute respiratory coronavirus-2 ...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.10.22.465012</td>\n",
       "      <td>96</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Hippocampal spatio-temporal cognitive maps ada...</td>\n",
       "      <td>[{'name': 'Mona M Garvert'}, {'name': 'Tankred...</td>\n",
       "      <td>&lt;p&gt;The brain forms cognitive maps of relationa...</td>\n",
       "      <td>2021-10-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.10.18.21265113</td>\n",
       "      <td>90</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Immune Responses in Fully Vaccinated Individua...</td>\n",
       "      <td>[{'name': 'Ai-ris Y. Collier'}, {'name': 'Cath...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;A cluster of ...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.10.21.465374</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Neural knowledge assembly in humans and deep n...</td>\n",
       "      <td>[{'name': 'Stephanie M Nelli'}, {'name': 'Luka...</td>\n",
       "      <td>&lt;p&gt;Human understanding of the world can change...</td>\n",
       "      <td>2021-10-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.10.17.21265101</td>\n",
       "      <td>33</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Time-Varying Effectiveness of Three Covid-19 V...</td>\n",
       "      <td>[{'name': 'Mnica M. Robles Fontn'}, {'name':...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;As of October...</td>\n",
       "      <td>2021-10-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.10.21.465323</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Defining cellular complexity in human autosoma...</td>\n",
       "      <td>[{'name': 'Yoshiharu Muto'}, {'name': 'Eryn E....</td>\n",
       "      <td>&lt;p&gt;Autosomal dominant polycystic kidney diseas...</td>\n",
       "      <td>2021-10-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.10.20.465093</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Inherited MUTYH mutations cause elevated somat...</td>\n",
       "      <td>[{'name': 'Philip S. Robinson'}, {'name': 'Lau...</td>\n",
       "      <td>&lt;p&gt;Cellular DNA damage caused by reactive oxyg...</td>\n",
       "      <td>2021-10-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.10.22.465484</td>\n",
       "      <td>25</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Genomic insights into variation in thermotoler...</td>\n",
       "      <td>[{'name': 'Cheyenne Payne'}, {'name': 'Stephen...</td>\n",
       "      <td>&lt;p&gt;Understanding how organisms adapt to changi...</td>\n",
       "      <td>2021-10-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.10.20.465155</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Human milk oligosaccharides reduce murine grou...</td>\n",
       "      <td>[{'name': 'Marlyd E Mejia'}, {'name': 'Samanth...</td>\n",
       "      <td>&lt;p&gt;Group B Streptococcus (GBS) colonizes the v...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.10.21.465386</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Variations in cell-surface ACE2 levels alter d...</td>\n",
       "      <td>[{'name': 'Soheila Kazemi'}, {'name': 'Alberto...</td>\n",
       "      <td>&lt;p&gt;The severe acute respiratory syndrome coron...</td>\n",
       "      <td>2021-10-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.10.21.465363</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A Null Allele of the Pol IV Second Subunit is ...</td>\n",
       "      <td>[{'name': 'Tania Chakraborty'}, {'name': 'Josh...</td>\n",
       "      <td>&lt;p&gt;All eukaryotes possess three DNA-dependent ...</td>\n",
       "      <td>2021-10-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.10.19.21265226</td>\n",
       "      <td>13</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>Wastewater and surface monitoring to detect CO...</td>\n",
       "      <td>[{'name': 'Rebecca K Fielding-Miller'}, {'name...</td>\n",
       "      <td>&lt;p&gt;Schools are high-risk settings for SARS-CoV...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.10.20.21265245</td>\n",
       "      <td>11</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Comparative effectiveness of allocation strate...</td>\n",
       "      <td>[{'name': 'Kathy Leung'}, {'name': 'Mark Jit'}...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;We aimed to e...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.10.22.465468</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>PermaPhosSer: autonomous synthesis of function...</td>\n",
       "      <td>[{'name': 'Phillip Zhu'}, {'name': 'Rachel Fra...</td>\n",
       "      <td>&lt;p&gt;Installing stable, functional mimics of pho...</td>\n",
       "      <td>2021-10-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.10.19.21265062</td>\n",
       "      <td>9</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Mutations on non-structural proteins of SARS-C...</td>\n",
       "      <td>[{'name': 'Takaya Ichikawa'}, {'name': 'Shiho ...</td>\n",
       "      <td>&lt;p&gt;Among a single COVID-19 cluster population ...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.10.20.21265239</td>\n",
       "      <td>9</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Gastroenterology</td>\n",
       "      <td>True</td>\n",
       "      <td>Immunogenicity of BNT162b2 Vaccine in Patients...</td>\n",
       "      <td>[{'name': 'Mohammad Shehab'}, {'name': 'Mohame...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Vaccination i...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.10.21.465371</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Real-Time Structure Search and Structure Class...</td>\n",
       "      <td>[{'name': 'Tunde Aderinwale'}, {'name': 'Vijay...</td>\n",
       "      <td>&lt;p&gt;AlphaFold2 showed a substantial improvement...</td>\n",
       "      <td>2021-10-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.10.18.21265130</td>\n",
       "      <td>8</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Survey of Direct and Indirect Effects of COVID...</td>\n",
       "      <td>[{'name': 'Dianeh Rabi'}, {'name': 'Razan Rabi...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Purpose&lt;/title&gt;&lt;p&gt;Ocular manifesta...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.10.23.465463</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The mechano-sensitive ion channel Piezo mediat...</td>\n",
       "      <td>[{'name': 'Sybille Koehler'}, {'name': 'Barry ...</td>\n",
       "      <td>&lt;p&gt;Mechanotransduction is an important process...</td>\n",
       "      <td>2021-10-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.10.20.465099</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The superior salinity tolerance of wheat culti...</td>\n",
       "      <td>[{'name': 'Sarah Vogt'}, {'name': 'Karla Feijs...</td>\n",
       "      <td>&lt;p&gt;Saline soils limit the production of import...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.10.22.465449</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Quantitative fragmentomics allow affinity mapp...</td>\n",
       "      <td>[{'name': 'Gergo Gogl'}, {'name': 'Boglarka Za...</td>\n",
       "      <td>&lt;p&gt;Human protein networks have been widely exp...</td>\n",
       "      <td>2021-10-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.10.19.21265222</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Monoclonal Antibody Treatment of Breakthrough ...</td>\n",
       "      <td>[{'name': 'Dennis M. Bierle'}, {'name': 'Ravin...</td>\n",
       "      <td>&lt;p&gt;Breakthrough COVID-19 may occur in fully va...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.10.19.464965</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Characterising the vegetation-rainfall relatio...</td>\n",
       "      <td>[{'name': 'Bidyut Sarania'}, {'name': 'Vishwes...</td>\n",
       "      <td>&lt;p&gt;Ecosystems are complex systems and are char...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.10.20.465206</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Regulatory variants active in iPSC-derived pan...</td>\n",
       "      <td>[{'name': 'Jennifer P. Nguyen'}, {'name': \"Agn...</td>\n",
       "      <td>&lt;p&gt;Pancreatic progenitor cells (PPC) are an ea...</td>\n",
       "      <td>2021-10-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.10.21.465269</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Systematic analysis of intrinsic enhancer-prom...</td>\n",
       "      <td>[{'name': 'Miguel Martinez-Ara'}, {'name': 'Fe...</td>\n",
       "      <td>&lt;p&gt;Gene expression is in part controlled by ci...</td>\n",
       "      <td>2021-10-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.10.14.21264762</td>\n",
       "      <td>5</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Longitudinal and comparative analysis of humor...</td>\n",
       "      <td>[{'name': 'Salvador Romero-Pinedo'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;The emergence of COVID-19 has led to a worl...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.10.18.464824</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Transcriptional variability accelerates pre-le...</td>\n",
       "      <td>[{'name': 'Shikha Gupta'}, {'name': 'Oliver M....</td>\n",
       "      <td>&lt;p&gt;Transcriptional variability facilitates sto...</td>\n",
       "      <td>2021-10-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.10.22.465514</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>E-Cannula reveals anatomical diversity in shar...</td>\n",
       "      <td>[{'name': 'Xin Liu'}, {'name': 'Satoshi Terada...</td>\n",
       "      <td>&lt;p&gt;The hippocampus plays a critical role in sp...</td>\n",
       "      <td>2021-10-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.10.19.21265230</td>\n",
       "      <td>5</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>Psychological, social and financial impact of ...</td>\n",
       "      <td>[{'name': 'Danielle M Muscat'}, {'name': 'Juli...</td>\n",
       "      <td>&lt;p&gt;Objective: This study aimed to explore the ...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.10.21.465321</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>PI4P and BLOC-1 remodel endosomal membranes in...</td>\n",
       "      <td>[{'name': 'Riddhi Atul Jani'}, {'name': 'Aurl...</td>\n",
       "      <td>&lt;p&gt;Intracellular trafficking is mediated by tr...</td>\n",
       "      <td>2021-10-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.10.20.465103</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Highly efficient generation of canker resistan...</td>\n",
       "      <td>[{'name': 'Xiaoen Huang'}, {'name': 'Nian Wang'}]</td>\n",
       "      <td>&lt;p&gt;Sweet orange (&lt;italic&gt;Citrus sinensis&lt;/ital...</td>\n",
       "      <td>2021-10-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.10.22.465537</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Amplicon sequencing of single-copy protein-cod...</td>\n",
       "      <td>[{'name': 'Chengfeng Yang'}, {'name': 'Qinzhi ...</td>\n",
       "      <td>&lt;p&gt;An in-depth understanding of microbial func...</td>\n",
       "      <td>2021-10-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.10.23.465551</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Retraction ATPase motors are promiscuous among...</td>\n",
       "      <td>[{'name': 'Evan Couser'}, {'name': 'Jennifer L...</td>\n",
       "      <td>&lt;p&gt;Bacterial surface appendages called type IV...</td>\n",
       "      <td>2021-10-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.10.13.21264693</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Modeling brings additional insights into the k...</td>\n",
       "      <td>[{'name': 'Prague Mlanie'}, {'name': 'Quentin...</td>\n",
       "      <td>&lt;p&gt;Following the paper by Seow et al. publishe...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.10.20.465087</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>A Network-based Deep Learning Framework Cataly...</td>\n",
       "      <td>[{'name': 'Jielin Xu'}, {'name': 'Yuan Hou'}, ...</td>\n",
       "      <td>&lt;p&gt;Human genome sequencing studies have identi...</td>\n",
       "      <td>2021-10-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.10.18.21264783</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 variant transmission in a community...</td>\n",
       "      <td>[{'name': 'Wenjuan Zhang'}, {'name': 'Marcela ...</td>\n",
       "      <td>&lt;p&gt;The SARS-CoV-2 variant, B.1.1.519, arose in...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.10.20.21265279</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Psychiatry and Clinical Psychology</td>\n",
       "      <td>True</td>\n",
       "      <td>Factors affecting the mental health of pregnan...</td>\n",
       "      <td>[{'name': 'A R McKinlay'}, {'name': 'D Fancour...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;People using ...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.10.13.21264946</td>\n",
       "      <td>3</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Intensive Care and Critical Care Medicine</td>\n",
       "      <td>True</td>\n",
       "      <td>Clinical observation of high-flow nasal cannul...</td>\n",
       "      <td>[{'name': 'AFM Tareq Bhuiyan'}, {'name': 'Sudi...</td>\n",
       "      <td>&lt;p&gt;Objective: Diabetes prevalence is a vital f...</td>\n",
       "      <td>2021-10-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.10.19.465067</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Phospho-regulation accommodates Type III secre...</td>\n",
       "      <td>[{'name': 'Rebecca L. Murray'}, {'name': 'Rach...</td>\n",
       "      <td>&lt;p&gt;Membrane contact sites (MCS) are crucial fo...</td>\n",
       "      <td>2021-10-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.10.19.21264255</td>\n",
       "      <td>3</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Gastroenterology</td>\n",
       "      <td>False</td>\n",
       "      <td>Metabolic Dysfunction Related Liver Disease as...</td>\n",
       "      <td>[{'name': 'Alasdair Taylor'}, {'name': 'Moneez...</td>\n",
       "      <td>&lt;p&gt;The aim of this study was to investigate th...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.10.18.464802</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Microglia remodel the synaptic signaling requi...</td>\n",
       "      <td>[{'name': 'Renato Socodato'}, {'name': 'Tiago ...</td>\n",
       "      <td>&lt;p&gt;Microglia modulate synaptic activity, essen...</td>\n",
       "      <td>2021-10-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.10.18.21265057</td>\n",
       "      <td>3</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Impact of routine asymptomatic screening on CO...</td>\n",
       "      <td>[{'name': 'Rebeca C. Falco'}, {'name': 'Micha...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;With the retu...</td>\n",
       "      <td>2021-10-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.10.20.465108</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>hyperTRIBER: a flexible R package for the anal...</td>\n",
       "      <td>[{'name': 'Sarah Rennie'}, {'name': 'Daniel He...</td>\n",
       "      <td>&lt;p&gt;RNA editing by ADAR (adenosine deaminase ac...</td>\n",
       "      <td>2021-10-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.10.21.465238</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Automated and parallelized spike collision tes...</td>\n",
       "      <td>[{'name': 'Keita Mitani'}, {'name': 'Masanori ...</td>\n",
       "      <td>&lt;p&gt;The spike collision test is a highly reliab...</td>\n",
       "      <td>2021-10-22</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2021.10.19.21265208    1705  medRxiv   \n",
       "1   10.1101/2021.10.18.21264623     419  medRxiv   \n",
       "2     10.1101/2021.10.22.465012      96  bioRxiv   \n",
       "3   10.1101/2021.10.18.21265113      90  medRxiv   \n",
       "4     10.1101/2021.10.21.465374      41  bioRxiv   \n",
       "5   10.1101/2021.10.17.21265101      33  medRxiv   \n",
       "6     10.1101/2021.10.21.465323      26  bioRxiv   \n",
       "7     10.1101/2021.10.20.465093      26  bioRxiv   \n",
       "8     10.1101/2021.10.22.465484      25  bioRxiv   \n",
       "9     10.1101/2021.10.20.465155      22  bioRxiv   \n",
       "10    10.1101/2021.10.21.465386      21  bioRxiv   \n",
       "11    10.1101/2021.10.21.465363      14  bioRxiv   \n",
       "12  10.1101/2021.10.19.21265226      13  medRxiv   \n",
       "13  10.1101/2021.10.20.21265245      11  medRxiv   \n",
       "14    10.1101/2021.10.22.465468      10  bioRxiv   \n",
       "15  10.1101/2021.10.19.21265062       9  medRxiv   \n",
       "16  10.1101/2021.10.20.21265239       9  medRxiv   \n",
       "17    10.1101/2021.10.21.465371       8  bioRxiv   \n",
       "18  10.1101/2021.10.18.21265130       8  medRxiv   \n",
       "19    10.1101/2021.10.23.465463       8  bioRxiv   \n",
       "20    10.1101/2021.10.20.465099       7  bioRxiv   \n",
       "21    10.1101/2021.10.22.465449       7  bioRxiv   \n",
       "22  10.1101/2021.10.19.21265222       6  medRxiv   \n",
       "23    10.1101/2021.10.19.464965       5  bioRxiv   \n",
       "24    10.1101/2021.10.20.465206       5  bioRxiv   \n",
       "25    10.1101/2021.10.21.465269       5  bioRxiv   \n",
       "26  10.1101/2021.10.14.21264762       5  medRxiv   \n",
       "27    10.1101/2021.10.18.464824       5  bioRxiv   \n",
       "28    10.1101/2021.10.22.465514       5  bioRxiv   \n",
       "29  10.1101/2021.10.19.21265230       5  medRxiv   \n",
       "30    10.1101/2021.10.21.465321       5  bioRxiv   \n",
       "31    10.1101/2021.10.20.465103       4  bioRxiv   \n",
       "32    10.1101/2021.10.22.465537       4  bioRxiv   \n",
       "33    10.1101/2021.10.23.465551       4  bioRxiv   \n",
       "34  10.1101/2021.10.13.21264693       4  medRxiv   \n",
       "35    10.1101/2021.10.20.465087       4  bioRxiv   \n",
       "36  10.1101/2021.10.18.21264783       4  medRxiv   \n",
       "37  10.1101/2021.10.20.21265279       4  medRxiv   \n",
       "38  10.1101/2021.10.13.21264946       3  medRxiv   \n",
       "39    10.1101/2021.10.19.465067       3  bioRxiv   \n",
       "40  10.1101/2021.10.19.21264255       3  medRxiv   \n",
       "41    10.1101/2021.10.18.464802       3  bioRxiv   \n",
       "42  10.1101/2021.10.18.21265057       3  medRxiv   \n",
       "43    10.1101/2021.10.20.465108       3  bioRxiv   \n",
       "44    10.1101/2021.10.21.465238       3  bioRxiv   \n",
       "\n",
       "                                 subject-area  covid  \\\n",
       "0       Infectious Diseases (except HIV/AIDS)   True   \n",
       "1       Infectious Diseases (except HIV/AIDS)   True   \n",
       "2                                Neuroscience  False   \n",
       "3       Infectious Diseases (except HIV/AIDS)   True   \n",
       "4                                Neuroscience  False   \n",
       "5                                Epidemiology   True   \n",
       "6                                    Genomics  False   \n",
       "7                                    Genomics  False   \n",
       "8                        Evolutionary Biology  False   \n",
       "9                                Microbiology  False   \n",
       "10                               Microbiology   True   \n",
       "11                              Plant Biology  False   \n",
       "12                   Public and Global Health   True   \n",
       "13                               Epidemiology   True   \n",
       "14                          Synthetic Biology   True   \n",
       "15      Infectious Diseases (except HIV/AIDS)   True   \n",
       "16                           Gastroenterology   True   \n",
       "17                             Bioinformatics  False   \n",
       "18      Infectious Diseases (except HIV/AIDS)   True   \n",
       "19                               Cell Biology  False   \n",
       "20                              Plant Biology  False   \n",
       "21                            Systems Biology  False   \n",
       "22      Infectious Diseases (except HIV/AIDS)   True   \n",
       "23                                    Ecology  False   \n",
       "24                                   Genetics  False   \n",
       "25                                   Genetics  False   \n",
       "26      Infectious Diseases (except HIV/AIDS)   True   \n",
       "27                             Cancer Biology  False   \n",
       "28                               Neuroscience  False   \n",
       "29                   Public and Global Health   True   \n",
       "30                               Cell Biology  False   \n",
       "31                              Plant Biology  False   \n",
       "32                               Microbiology  False   \n",
       "33                               Microbiology  False   \n",
       "34      Infectious Diseases (except HIV/AIDS)   True   \n",
       "35                             Bioinformatics  False   \n",
       "36      Infectious Diseases (except HIV/AIDS)   True   \n",
       "37         Psychiatry and Clinical Psychology   True   \n",
       "38  Intensive Care and Critical Care Medicine   True   \n",
       "39                               Microbiology  False   \n",
       "40                           Gastroenterology  False   \n",
       "41                               Neuroscience  False   \n",
       "42                               Epidemiology   True   \n",
       "43                             Bioinformatics  False   \n",
       "44                               Neuroscience  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Airborne transmission of SARS-CoV-2 over dista...   \n",
       "1   SARS-CoV-2 variants associated with vaccine br...   \n",
       "2   Hippocampal spatio-temporal cognitive maps ada...   \n",
       "3   Immune Responses in Fully Vaccinated Individua...   \n",
       "4   Neural knowledge assembly in humans and deep n...   \n",
       "5   Time-Varying Effectiveness of Three Covid-19 V...   \n",
       "6   Defining cellular complexity in human autosoma...   \n",
       "7   Inherited MUTYH mutations cause elevated somat...   \n",
       "8   Genomic insights into variation in thermotoler...   \n",
       "9   Human milk oligosaccharides reduce murine grou...   \n",
       "10  Variations in cell-surface ACE2 levels alter d...   \n",
       "11  A Null Allele of the Pol IV Second Subunit is ...   \n",
       "12  Wastewater and surface monitoring to detect CO...   \n",
       "13  Comparative effectiveness of allocation strate...   \n",
       "14  PermaPhosSer: autonomous synthesis of function...   \n",
       "15  Mutations on non-structural proteins of SARS-C...   \n",
       "16  Immunogenicity of BNT162b2 Vaccine in Patients...   \n",
       "17  Real-Time Structure Search and Structure Class...   \n",
       "18  Survey of Direct and Indirect Effects of COVID...   \n",
       "19  The mechano-sensitive ion channel Piezo mediat...   \n",
       "20  The superior salinity tolerance of wheat culti...   \n",
       "21  Quantitative fragmentomics allow affinity mapp...   \n",
       "22  Monoclonal Antibody Treatment of Breakthrough ...   \n",
       "23  Characterising the vegetation-rainfall relatio...   \n",
       "24  Regulatory variants active in iPSC-derived pan...   \n",
       "25  Systematic analysis of intrinsic enhancer-prom...   \n",
       "26  Longitudinal and comparative analysis of humor...   \n",
       "27  Transcriptional variability accelerates pre-le...   \n",
       "28  E-Cannula reveals anatomical diversity in shar...   \n",
       "29  Psychological, social and financial impact of ...   \n",
       "30  PI4P and BLOC-1 remodel endosomal membranes in...   \n",
       "31  Highly efficient generation of canker resistan...   \n",
       "32  Amplicon sequencing of single-copy protein-cod...   \n",
       "33  Retraction ATPase motors are promiscuous among...   \n",
       "34  Modeling brings additional insights into the k...   \n",
       "35  A Network-based Deep Learning Framework Cataly...   \n",
       "36  SARS-CoV-2 variant transmission in a community...   \n",
       "37  Factors affecting the mental health of pregnan...   \n",
       "38  Clinical observation of high-flow nasal cannul...   \n",
       "39  Phospho-regulation accommodates Type III secre...   \n",
       "40  Metabolic Dysfunction Related Liver Disease as...   \n",
       "41  Microglia remodel the synaptic signaling requi...   \n",
       "42  Impact of routine asymptomatic screening on CO...   \n",
       "43  hyperTRIBER: a flexible R package for the anal...   \n",
       "44  Automated and parallelized spike collision tes...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Jennifer C Palmer'}, {'name': 'Daph...   \n",
       "1   [{'name': 'Andrew D. Marques'}, {'name': 'Scot...   \n",
       "2   [{'name': 'Mona M Garvert'}, {'name': 'Tankred...   \n",
       "3   [{'name': 'Ai-ris Y. Collier'}, {'name': 'Cath...   \n",
       "4   [{'name': 'Stephanie M Nelli'}, {'name': 'Luka...   \n",
       "5   [{'name': 'Mnica M. Robles Fontn'}, {'name':...   \n",
       "6   [{'name': 'Yoshiharu Muto'}, {'name': 'Eryn E....   \n",
       "7   [{'name': 'Philip S. Robinson'}, {'name': 'Lau...   \n",
       "8   [{'name': 'Cheyenne Payne'}, {'name': 'Stephen...   \n",
       "9   [{'name': 'Marlyd E Mejia'}, {'name': 'Samanth...   \n",
       "10  [{'name': 'Soheila Kazemi'}, {'name': 'Alberto...   \n",
       "11  [{'name': 'Tania Chakraborty'}, {'name': 'Josh...   \n",
       "12  [{'name': 'Rebecca K Fielding-Miller'}, {'name...   \n",
       "13  [{'name': 'Kathy Leung'}, {'name': 'Mark Jit'}...   \n",
       "14  [{'name': 'Phillip Zhu'}, {'name': 'Rachel Fra...   \n",
       "15  [{'name': 'Takaya Ichikawa'}, {'name': 'Shiho ...   \n",
       "16  [{'name': 'Mohammad Shehab'}, {'name': 'Mohame...   \n",
       "17  [{'name': 'Tunde Aderinwale'}, {'name': 'Vijay...   \n",
       "18  [{'name': 'Dianeh Rabi'}, {'name': 'Razan Rabi...   \n",
       "19  [{'name': 'Sybille Koehler'}, {'name': 'Barry ...   \n",
       "20  [{'name': 'Sarah Vogt'}, {'name': 'Karla Feijs...   \n",
       "21  [{'name': 'Gergo Gogl'}, {'name': 'Boglarka Za...   \n",
       "22  [{'name': 'Dennis M. Bierle'}, {'name': 'Ravin...   \n",
       "23  [{'name': 'Bidyut Sarania'}, {'name': 'Vishwes...   \n",
       "24  [{'name': 'Jennifer P. Nguyen'}, {'name': \"Agn...   \n",
       "25  [{'name': 'Miguel Martinez-Ara'}, {'name': 'Fe...   \n",
       "26  [{'name': 'Salvador Romero-Pinedo'}, {'name': ...   \n",
       "27  [{'name': 'Shikha Gupta'}, {'name': 'Oliver M....   \n",
       "28  [{'name': 'Xin Liu'}, {'name': 'Satoshi Terada...   \n",
       "29  [{'name': 'Danielle M Muscat'}, {'name': 'Juli...   \n",
       "30  [{'name': 'Riddhi Atul Jani'}, {'name': 'Aurl...   \n",
       "31  [{'name': 'Xiaoen Huang'}, {'name': 'Nian Wang'}]   \n",
       "32  [{'name': 'Chengfeng Yang'}, {'name': 'Qinzhi ...   \n",
       "33  [{'name': 'Evan Couser'}, {'name': 'Jennifer L...   \n",
       "34  [{'name': 'Prague Mlanie'}, {'name': 'Quentin...   \n",
       "35  [{'name': 'Jielin Xu'}, {'name': 'Yuan Hou'}, ...   \n",
       "36  [{'name': 'Wenjuan Zhang'}, {'name': 'Marcela ...   \n",
       "37  [{'name': 'A R McKinlay'}, {'name': 'D Fancour...   \n",
       "38  [{'name': 'AFM Tareq Bhuiyan'}, {'name': 'Sudi...   \n",
       "39  [{'name': 'Rebecca L. Murray'}, {'name': 'Rach...   \n",
       "40  [{'name': 'Alasdair Taylor'}, {'name': 'Moneez...   \n",
       "41  [{'name': 'Renato Socodato'}, {'name': 'Tiago ...   \n",
       "42  [{'name': 'Rebeca C. Falco'}, {'name': 'Micha...   \n",
       "43  [{'name': 'Sarah Rennie'}, {'name': 'Daniel He...   \n",
       "44  [{'name': 'Keita Mitani'}, {'name': 'Masanori ...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <sec><title>Objective</title><p>To evaluate th...  2021-10-20  \n",
       "1   <p>The severe acute respiratory coronavirus-2 ...  2021-10-20  \n",
       "2   <p>The brain forms cognitive maps of relationa...  2021-10-23  \n",
       "3   <sec><title>Background</title><p>A cluster of ...  2021-10-20  \n",
       "4   <p>Human understanding of the world can change...  2021-10-23  \n",
       "5   <sec><title>Background</title><p>As of October...  2021-10-18  \n",
       "6   <p>Autosomal dominant polycystic kidney diseas...  2021-10-22  \n",
       "7   <p>Cellular DNA damage caused by reactive oxyg...  2021-10-21  \n",
       "8   <p>Understanding how organisms adapt to changi...  2021-10-24  \n",
       "9   <p>Group B Streptococcus (GBS) colonizes the v...  2021-10-20  \n",
       "10  <p>The severe acute respiratory syndrome coron...  2021-10-23  \n",
       "11  <p>All eukaryotes possess three DNA-dependent ...  2021-10-22  \n",
       "12  <p>Schools are high-risk settings for SARS-CoV...  2021-10-20  \n",
       "13  <sec><title>Background</title><p>We aimed to e...  2021-10-20  \n",
       "14  <p>Installing stable, functional mimics of pho...  2021-10-23  \n",
       "15  <p>Among a single COVID-19 cluster population ...  2021-10-20  \n",
       "16  <sec><title>Background</title><p>Vaccination i...  2021-10-20  \n",
       "17  <p>AlphaFold2 showed a substantial improvement...  2021-10-23  \n",
       "18  <sec><title>Purpose</title><p>Ocular manifesta...  2021-10-20  \n",
       "19  <p>Mechanotransduction is an important process...  2021-10-23  \n",
       "20  <p>Saline soils limit the production of import...  2021-10-20  \n",
       "21  <p>Human protein networks have been widely exp...  2021-10-24  \n",
       "22  <p>Breakthrough COVID-19 may occur in fully va...  2021-10-20  \n",
       "23  <p>Ecosystems are complex systems and are char...  2021-10-20  \n",
       "24  <p>Pancreatic progenitor cells (PPC) are an ea...  2021-10-21  \n",
       "25  <p>Gene expression is in part controlled by ci...  2021-10-22  \n",
       "26  <p>The emergence of COVID-19 has led to a worl...  2021-10-20  \n",
       "27  <p>Transcriptional variability facilitates sto...  2021-10-18  \n",
       "28  <p>The hippocampus plays a critical role in sp...  2021-10-24  \n",
       "29  <p>Objective: This study aimed to explore the ...  2021-10-20  \n",
       "30  <p>Intracellular trafficking is mediated by tr...  2021-10-21  \n",
       "31  <p>Sweet orange (<italic>Citrus sinensis</ital...  2021-10-21  \n",
       "32  <p>An in-depth understanding of microbial func...  2021-10-23  \n",
       "33  <p>Bacterial surface appendages called type IV...  2021-10-23  \n",
       "34  <p>Following the paper by Seow et al. publishe...  2021-10-20  \n",
       "35  <p>Human genome sequencing studies have identi...  2021-10-21  \n",
       "36  <p>The SARS-CoV-2 variant, B.1.1.519, arose in...  2021-10-20  \n",
       "37  <sec><title>Background</title><p>People using ...  2021-10-20  \n",
       "38  <p>Objective: Diabetes prevalence is a vital f...  2021-10-18  \n",
       "39  <p>Membrane contact sites (MCS) are crucial fo...  2021-10-21  \n",
       "40  <p>The aim of this study was to investigate th...  2021-10-20  \n",
       "41  <p>Microglia modulate synaptic activity, essen...  2021-10-19  \n",
       "42  <sec><title>Background</title><p>With the retu...  2021-10-20  \n",
       "43  <p>RNA editing by ADAR (adenosine deaminase ac...  2021-10-21  \n",
       "44  <p>The spike collision test is a highly reliab...  2021-10-22  "
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "45 preprints (including 19 covering SARS-CoV-2, 27 from bioRxiv and 18 from medRxiv) published in the last 7 days before 2021-10-25 had been tweeted at least 3 times (maximum 1705)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.8"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
