{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&obj-id.prefix=10.1101&source=twitter&rows=0&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(email = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 0,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "13139 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=35281021-ffaf-460f-89bd-4db8232e7ef7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=02017bdf-575c-45e6-9233-8690717a4b2a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=78b76350-95f3-4462-873c-0c36105df1fc&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=e1e3d1aa-9e0c-48f6-af0c-658064698263&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=2045d8ca-f5fe-4c2b-b0d8-f50ee4bcf93b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=377a5a42-ca1f-44f2-aae1-a62cd502a39b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=965d0881-ac5f-47a4-8b6f-c4e7be5b20ed&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=1d8901b5-c801-4a53-9188-b927a97844ed&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=0cf2b3b2-4ad4-4de3-99f5-63c8a7ed265b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=d0652a5d-c333-4ae5-9460-4989c5499c2c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=b9e31378-56d1-4009-ac61-943dc3923c6e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=751d66b3-98b6-466c-a43a-6c3b1cf060bf&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=de008e9c-a55f-4383-8f5a-380862cd1034&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-19&until-occurred-date=2021-07-26\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(email = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258122...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.451321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.02.28.21250921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.451701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21256661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.21257766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450803...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.23.453070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.27.401232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.429735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.31.274639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.434227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.435971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453027...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.453013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.429147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.21259318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.30.070094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/053009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260362...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.19.452967...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.12.452063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448190...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.347740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.341958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.446059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.20167932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21253964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.21255913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.236703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.444885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.441297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.452530...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.17.440282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.21256193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.383604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.270371.120...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.19.452378...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.05.20.21257461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21257602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.338160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275303.121...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.01.28.428673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.21.392639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.453258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.12.430858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21256456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.21259977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.445318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.266429.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453077...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.24.453184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.444823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.21257971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451278...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.20109231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/807560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.21260079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.05.446432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450676...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.05.14.444165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260171...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.343210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20234369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.27.20045427...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452886...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448912...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.06.09.447621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21258785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.434760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.17.20061440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.424528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.21251664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.28.970103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256982...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433056...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.20248159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452875...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.23.453610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20094193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.21252597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.31.275362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453512...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.01.322487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453011...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2020.11.22.392217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.447297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.19.20248484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.427855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.115832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.07.414334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.29.069344...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.12.380386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.26.919985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21252728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.21256067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.446289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257738...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.453008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.26.965202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443493...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.23.453341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.21257017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452932...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.05.21.21257595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.451212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.366807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.20209114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.20243949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.28.359356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.431385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.21258447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.430328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453458...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.13.435259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.21254128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.447244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.21254024...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.20245993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.438792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.20202614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.31.928630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.21250670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21256113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.09.20148429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.371963...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.451608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441683...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.441101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.438959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.222158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.451090...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.21257257...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.05.17.21257092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.06.28.21258780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.21257505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.21257844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.116111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.10.20021675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.257352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453478...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.02.18.431871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21252086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.224584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.27.21254483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21249256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.23.164004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.116533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.09.430440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.28.254672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.20.427467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.426448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.17.254607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.03.437906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.440514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.12.20151084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21253727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.20248301...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442770...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/794305...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450588...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2020.04.06.028712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.15.383323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215343...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261041...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.20235010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.17.452800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.20202267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.21258886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.03.075069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.274563.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.04.325373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453026...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.06.14.448402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.22.20026500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.437067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21254000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.225102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.30.228916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453339...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.436382...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.191684.115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.274282.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.429765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.25.265546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.21251572...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.24.265702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.21251946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.260851.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.25.21252415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.241414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/741355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/710624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.434601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.11.20145086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21251509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.26.966580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.21260251...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.443989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.164442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20131243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.05.27.21257909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.30.125856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.13.448261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.446775...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.13.21258859...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2020.06.30.177071...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452052...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21256007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.09.20033217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.452818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.440215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20043919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.440856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047365...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.441133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452612...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.017624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.03.003178...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.01.20144030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275560.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.406751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.388819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.366716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.426718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20195925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.27.350553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.07.20245274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.22.427834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.20222661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.340588...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.07.19.21260767', 'tweets': 542, 'archive': 'medRxiv', 'subject-area': 'Pediatrics', 'covid': True, 'title': 'Children with SARS-CoV-2 in the National COVID Cohort Collaborative (N3C)', 'authors': \"[{'name': 'Blake Martin'}, {'name': 'Peter E. DeWitt'}, {'name': 'Seth Russell'}, {'name': 'Adit Anand'}, {'name': 'Katie R. Bradwell'}, {'name': 'Carolyn Bremer'}, {'name': 'Davera Gabriel'}, {'name': 'Andrew T. Girvin'}, {'name': 'Janos G. Hajagos'}, {'name': 'Julie A. McMurry'}, {'name': 'Andrew J. Neumann'}, {'name': 'Emily R. Pfaff'}, {'name': 'Anita Walden'}, {'name': 'Jacob T. Wooldridge'}, {'name': 'Yun Jae Yoo'}, {'name': 'Joel Saltz'}, {'name': 'Ken R. Gersing'}, {'name': 'Christopher G. Chute'}, {'name': 'Melissa A. Haendel'}, {'name': 'Richard Moffitt'}, {'name': 'Tellen D. Bennett'}]\", 'abstract': '<sec><title>Importance</title><p>SARS-CoV-2</p></sec><sec><title>Objective</title><p>To determine the characteristics, changes over time, outcomes, and severity risk factors of SARS-CoV-2 affected children within the National COVID Cohort Collaborative (N3C)</p></sec><sec><title>Design</title><p>Prospective cohort study of patient encounters with end dates before May 27th, 2021.</p></sec><sec><title>Setting</title><p>45 N3C institutions</p></sec><sec><title>Participants</title><p>Children &lt;19-years-old at initial SARS-CoV-2 testing</p></sec><sec><title>Main Outcomes and Measures</title><p>Case incidence and severity over time, demographic and comorbidity severity risk factors, vital sign and laboratory trajectories, clinical outcomes, and acute COVID-19 vs MIS-C contrasts for children infected with SARS-CoV-2.</p></sec><sec><title>Results</title><p>728,047 children in the N3C were tested for SARS-CoV-2; of these, 91,865 (12.6%) were positive. Among the 5,213 (6%) hospitalized children, 685 (13%) met criteria for severe disease: mechanical ventilation (7%), vasopressor/inotropic support (7%), ECMO (0.6%), or death/discharge to hospice (1.1%). Male gender, African American race, older age, and several pediatric complex chronic condition (PCCC) subcategories were associated with higher clinical severity (p<italic></italic>0.05). Vital signs (all p0.002) and many laboratory tests from the first day of hospitalization were predictive of peak disease severity. Children with severe (vs moderate) disease were more likely to receive antimicrobials (71% vs 32%, p&lt;0.001) and immunomodulatory medications (53% vs 16%, p&lt;0.001).</p><p>Compared to those with acute COVID-19, children with MIS-C were more likely to be male, Black/African American, 1-to-12-years-old, and less likely to have asthma, diabetes, or a PCCC (p<italic>&lt;</italic>0.04). MIS-C cases demonstrated a more inflammatory laboratory profile and more severe clinical phenotype with higher rates of invasive ventilation (12% vs 6%) and need for vasoactive-inotropic support (31% vs 6%) compared to acute COVID-19 cases, respectively (p&lt;0.03).</p></sec><sec><title>Conclusions</title><p>In the largest U.S. SARS-CoV-2-positive pediatric cohort to date, we observed differences in demographics, pre-existing comorbidities, and initial vital sign and laboratory test values between severity subgroups. Taken together, these results suggest that early identification of children likely to progress to severe disease could be achieved using readily available data elements from the day of admission. Further work is needed to translate this knowledge into improved outcomes.</p></sec>', 'posted': '2021-07-22'}, {'doi': '10.1101/2021.07.15.21260561', 'tweets': 405, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Viral Load of SARS-CoV-2 in Respiratory Aerosols Emitted by COVID-19 Patients while Breathing, Talking, and Singing', 'authors': \"[{'name': 'Kristen K. Coleman'}, {'name': 'Douglas Jie Wen Tay'}, {'name': 'Kai Sen Tan'}, {'name': 'Sean Wei Xiang Ong'}, {'name': 'Than The Son'}, {'name': 'Ming Hui Koh'}, {'name': 'Yi Qing Chin'}, {'name': 'Haziq Nasir'}, {'name': 'Tze Minn Mak'}, {'name': 'Justin Jang Hann Chu'}, {'name': 'Donald K. Milton'}, {'name': 'Vincent T. K. Chow'}, {'name': 'Paul Anantharajah Tambyah'}, {'name': 'Mark Chen'}, {'name': 'Tham Kwok Wai'}]\", 'abstract': '<sec><title>Background</title><p>Multiple SARS-CoV-2 superspreading events suggest that aerosols play an important role in driving the COVID-19 pandemic. However, the detailed roles of coarse (&gt;5m) and fine (5m) respiratory aerosols produced when breathing, talking, and singing are not well-understood.</p></sec><sec><title>Methods</title><p>Using a G-II exhaled breath collector, we measured viral RNA in coarse and fine respiratory aerosols emitted by COVID-19 patients during 30 minutes of breathing, 15 minutes of talking, and 15 minutes of singing.</p></sec><sec><title>Results</title><p>Among the 22 study participants, 13 (59%) emitted detectable levels of SARS-CoV-2 RNA in respiratory aerosols, including 3 asymptomatic patients and 1 presymptomatic patient. Viral loads ranged from 635,821 N gene copies per expiratory activity per patient. Patients earlier in illness were more likely to emit detectable RNA, and loads differed significantly between breathing, talking, and singing. The largest proportion of SARS-CoV-2 RNA copies was emitted by singing (53%), followed by talking (41%) and breathing (6%). Overall, fine aerosols constituted 85% of the viral load detected in our study. Virus cultures were negative.</p></sec><sec><title>Conclusions</title><p>Fine aerosols produced by talking and singing contain more SARS-CoV-2 copies than coarse aerosols and may play a significant role in the transmission of SARS-CoV-2. Exposure to fine aerosols should be mitigated, especially in indoor environments where airborne transmission of SARS-CoV-2 is likely to occur. Isolating viable SARS-CoV-2 from respiratory aerosol samples remains challenging, and whether this can be more easily accomplished for emerging SARS-CoV-2 variants is an important enquiry for future studies.</p></sec><sec><title>Key Points</title><p>We sampled respiratory aerosols emitted by COVID-19 patients and discovered that fine aerosols (5m) generated during talking and singing contain more SARS-CoV-2 copies than coarse aerosols (&gt;5m) and may play a significant role in the transmission of SARS-CoV-2.</p></sec>', 'posted': '2021-07-19'}, {'doi': '10.1101/2021.07.21.451321', 'tweets': 219, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Unlike Chloroquine, mefloquine inhibits SARS-CoV-2 infection in physiologically relevant cells and does not induce viral variants', 'authors': \"[{'name': 'Carolina Q. Sacramento'}, {'name': 'Natalia Fintelman-Rodrigues'}, {'name': 'Suelen S. G. Dias'}, {'name': 'Jairo R. Temerozo'}, {'name': 'Aline de Paula D. Da Silva'}, {'name': 'Carine S. da Silva'}, {'name': 'Andr C. Ferreira'}, {'name': 'Mayara Mattos'}, {'name': 'Vinicius C. Soares'}, {'name': 'Filipe Pereira-Dutra'}, {'name': 'Milene D. Miranda'}, {'name': 'Debora F. Barreto-Vieira'}, {'name': 'Marcos Alexandre N. da Silva'}, {'name': 'Suzana S. Santos'}, {'name': 'Mateo Torres'}, {'name': 'Rajith K R Rajoli'}, {'name': 'Alberto Paccanaro'}, {'name': 'Andrew Owen'}, {'name': 'Dumith Chequer Bou-Habib'}, {'name': 'Patrcia T. Bozza'}, {'name': 'Thiago Moreno L. Souza'}]\", 'abstract': '<p>Repositioning of clinical approved drugs could represent the fastest way to identify therapeutic options during public health emergencies, the majority of drugs explored for repurposing as antivirals for 2019 coronavirus disease (COVID-19) have failed to demonstrate clinical benefit. Without specific antivirals, the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic continues to cause major global mortality. Antimalarial drugs, such as chloroquine (CQ)/hydroxychloroquine (HCQ) and mefloquine have emerged as potential anti-SARS-CoV-2 antivirals. CQ/HCQ entered the Solidarity and RECOVERY clinical trials against COVID-19 and showed lack of efficacy. Importantly, mefloquine is not a 4-aminoquinoline like CQ and HCQ and has been previously repurposed for other respiratory diseases. Unlike the 4-aminoquinolines that accumulate in the high pH of intracellular lysosomes of the lung, the high respiratory tract penetration of mefloquine is driven by its high lipophilicity. While CQ and HCQ exhibit activity in Vero E6 cells, their activity is obviated in TMPRSS2-expressing cells, such as Calu-3 cells, which more accurately recapitulate in vivo entry mechanisms for SARS-CoV-2. Accordingly, here we report the anti-SARS-CoV-2 activity of mefloquine in Calu-3 type II pneumocytes and primary human monocytes. Mefloquine inhibited SARS-CoV-2 replication in Calu-3 cells with low cytotoxicity and EC<sub>50</sub> and EC<sub>90</sub> values of 1.2 and 5.3 M, respectively. In addition, mefloquine reduced up to 68% the SARS-CoV-2 RNA levels in infected monocytes, reducing viral-induced inflammation. Mefloquine blocked early steps of the SARS-CoV-2 replicative cycle and was less prone than CQ to induce drug-associated viral mutations and synergized with RNA polymerase inhibitor. The pharmacological parameters of mefloquine are consistent with its plasma exposure in humans and its tissue-to-plasma predicted coefficient points that this drug may accumulate in the lungs. These data indicate that mefloquine could represent an orally available clinically approved drug option against COVID-19 and should not be neglected on the basis of the failure of CQ and HCQ.</p>', 'posted': '2021-07-21'}, {'doi': '10.1101/2021.07.19.452771', 'tweets': 50, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Comparison of Neutralizing Antibody Titers Elicited by mRNA and Adenoviral Vector Vaccine against SARS-CoV-2 Variants', 'authors': \"[{'name': 'Takuya Tada'}, {'name': 'Hao Zhou'}, {'name': 'Marie I. Samanovic'}, {'name': 'Belinda M. Dcosta'}, {'name': 'Amber Cornelius'}, {'name': 'Mark J. Mulligan'}, {'name': 'Nathaniel R. Landau'}]\", 'abstract': '<p>The increasing prevalence of SARS-CoV-2 variants has raised concerns regarding possible decreases in vaccine efficacy. Here, neutralizing antibody titers elicited by mRNA-based and an adenoviral vector-based vaccine against variant pseudotyped viruses were compared. BNT162b2 and mRNA-1273-elicited antibodies showed modest neutralization resistance against Beta, Delta, Delta plus and Lambda variants whereas Ad26.COV2.S-elicited antibodies from a significant fraction of vaccinated individuals were of low neutralizing titer (IC<sub>50</sub> &lt;50). The data underscore the importance of surveillance for breakthrough infections that result in severe COVID-19 and suggest the benefit of a second immunization following Ad26.COV2.S to increase protection against the variants.</p>', 'posted': '2021-07-19'}, {'doi': '10.1101/2021.07.23.453070', 'tweets': 49, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Regulation of sedimentation rate shapes the evolution of multicellularity in a unicellular relative of animals', 'authors': \"[{'name': 'Omaya Dudin'}, {'name': 'Sbastien Wielgoss'}, {'name': 'Aaron M. New'}, {'name': 'Iaki Ruiz-Trillo'}]\", 'abstract': '<p>Significant increases in sedimentation rate accompany the evolution of multicellularity. These increases should lead to rapid changes in ecological distribution, thereby affecting the costs and benefits of multicellularity and its likelihood to evolve. However, how genetic and cellular traits which control this process, their likelihood of emergence over evolutionary timescales, and the variation in these traits as multicellularity evolves, are still poorly understood. Here, using isolates of the ichthyosporean <italic>Sphaeroforma</italic> genus - close unicellular relatives of animals with brief transient multicellular life stages - we demonstrate that sedimentation rate is a highly variable and evolvable trait affected by at least two distinct physical mechanisms. We first find a dramatic &gt;300x variation in sedimentation rate for different <italic>Sphaeroforma</italic> species, mainly driven by size and density during the unicellular-to-multicellular life cycle transition. Using experimental evolution with sedimentation rate as a focal trait, we readily obtained fast settling <italic>S. arctica</italic> isolates. Quantitative microscopy showed that increased sedimentation rates most often arose by incomplete cellular separation after cell division, leading to clonal clumping multicellular variants with increased size and density. Additionally, density increases arose by an acceleration of the nuclear doubling time relative to cell size. Similar size- and density-affecting phenotypes were observed in four additional species from the <italic>Sphaeroforma</italic> genus, suggesting variation in these traits might be widespread in the marine habitat. By sequencing evolved isolates, we identified mutations in regulators of cytokinesis, plasma membrane remodelling, and chromatin condensation that may contribute to both clump formation and the increase in the nuclear number-to-volume ratio. Taken together, this study illustrates how extensive cellular control of density and size drive sedimentation rate variation, likely shaping the evolution of multicellularity.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453521', 'tweets': 35, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Probiotic consortia improve anti-viral immunity to SARS-CoV-2 in Ferrets', 'authors': \"[{'name': 'J Lehtinen Markus'}, {'name': 'Kumar Ritesh'}, {'name': 'Zabel Bryan'}, {'name': 'M Mkel Sanna'}, {'name': 'Nedveck Derek'}, {'name': 'Tang Peipei'}, {'name': 'Latvala Sinikka'}, {'name': 'Guery Sebastien'}, {'name': 'R Budinoff Charles'}]\", 'abstract': '<p>Probiotics have been suggested as one solution to counter detrimental health effects by SARS-CoV-2, however, data so far is scarce. We tested the effect of two probiotic consortia, OL-1 and OL-2, against SARS-CoV-2 in ferrets and assessed their effect on cytokine production and transcriptome in a human monocyte-derived macrophage (Mf) and dendritic cell (DC) model.</p><p>The results showed that the consortia significantly reduced the viral load, modulated immune response, and regulated viral receptor expression in ferrets compared to placebo. In human Mf and DC model, OL-1 and OL-2 induced cytokine production and genes related to SARS-CoV-2 anti-viral immunity.</p><p>The study results indicate that probiotic stimulation of the ferret immune system leads to improved anti-viral immunity against SARS-COV-2 and that critical genes and cytokines for anti-SARS-CoV-2 immunity are stimulated in human immune cells <italic>in vitro</italic>. The effect of the consortia against SARS-CoV-2 warrants further investigations in human clinical trials.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453573', 'tweets': 34, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Strong and opponent contributions of dorsomedial striatal pathways to behavior depends on cognitive demands and task strategy', 'authors': \"[{'name': 'Scott Bolkan'}, {'name': 'Iris Stone'}, {'name': 'Lucas Pinto'}, {'name': 'Zoe Ashwood'}, {'name': 'Jorge Garcia'}, {'name': 'Alison Herman'}, {'name': 'Priyanka Singh'}, {'name': 'Akhil Bandi'}, {'name': 'Julia Cox'}, {'name': 'Christopher Zimmerman'}, {'name': 'Jounhong Cho'}, {'name': 'Ben Engelhard'}, {'name': 'Sue Koay'}, {'name': 'Jonathan Pillow'}, {'name': 'Ilana Witten'}]\", 'abstract': \"<p>A classic view of the striatum holds that activity in direct and indirect pathways oppositely modulates motor output. Whether this involves direct control of movement, or reflects a cognitive process underlying movement, has remained unresolved. Here we find that strong, opponent control of behavior by the two pathways of the dorsomedial striatum (DMS) depends on a task's cognitive demands. Furthermore, a latent state model (a hidden markov model with generalized linear model observations) reveals that - even within a single task - the contribution of the two pathways to behavior is state-dependent. Specifically, the two pathways have large contributions in one of two states associated with a strategy of evidence accumulation, compared to a state associated with a strategy of repeating previous choices. Thus, both the cognitive demands imposed by a task, as well as the strategy that mice pursue within a task, determine whether DMS pathways provide strong and opponent control of behavior.</p>\", 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.19.453013', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Phylogenomic analyses of echinoid diversification prompt a re-evaluation of their fossil record', 'authors': \"[{'name': 'Nicolas Mongiardino Koch'}, {'name': 'Jeffrey R Thompson'}, {'name': 'Avery S Hatch'}, {'name': 'Marina F McCowin'}, {'name': 'Frances Armstrong'}, {'name': 'Simon E Coppard'}, {'name': 'Felipe Aguilera'}, {'name': 'Omri Bronstein'}, {'name': 'Andreas Kroh'}, {'name': 'Rich Mooi'}, {'name': 'Greg W Rouse'}]\", 'abstract': '<p>Echinoids are key components of modern marine ecosystems. Despite a remarkable fossil record, the emergence of their crown group is documented by few specimens of unclear affinities, rendering much of their early history uncertain. The origin of sand dollars, one of its most distinctive clades, is also unclear due to an unstable phylogenetic context and discrepancies between molecular divergence times and fossil evidence. We employ seventeen novel genomes and transcriptomes to build a phylogenomic dataset with a near-complete sampling of major lineages. With it, we revise the phylogeny and divergence times of echinoids, and place their history within the broader context of echinoderm evolution. We also introduce the concept of a chronospace - a multidimensional representation of node ages - and use it to explore the effects of using alternative gene samples, models of molecular evolution, and clock priors. We find the choice of clock model to have the strongest impact on divergence times, while the use of site-heterogeneous models shows little effects. The choice of loci shows an intermediate impact, affecting mostly deep Paleozoic nodes, for which clock-like genes recover dates more congruent with fossil evidence. Our results reveal that crown group echinoids originated in the Permian and diversified rapidly in the Triassic, despite the relative lack of fossil evidence for this early diversification. We also clarify the relationships among sand dollars and their close relatives, showing that the genus Apatopygus represents a relict lineage with a deep Jurassic origin. Surprisingly, the origin of sand dollars is confidently dated to the Cretaceous, implying ghost ranges spanning approximately 50 million years, a remarkable discrepancy with their rich fossil record.</p>', 'posted': '2021-07-20'}, {'doi': '10.1101/2021.07.19.452969', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'A Bayesian model selection approach to mediation analysis', 'authors': \"[{'name': 'Wesley L Crouse'}, {'name': 'Gregory R Keele'}, {'name': 'Madeleine S Gastonguay'}, {'name': 'Gary A Churchill'}, {'name': 'William Valdar'}]\", 'abstract': '<p>Mediation analysis is a powerful tool for discovery of causal relationships. We describe a Bayesian model selection approach to mediation analysis that is implemented in our bmediatR software. Using simulations, we show that bmediatR performs as well or better than established methods including the Sobel test, while allowing greater flexibility in both model specification and in the types of inference that are possible. We applied bmediatR to genetic data from mice and human cell lines to demonstrate its ability to derive biologically meaningful findings. The Bayesian model selection framework is extensible to support a wide variety of mediation models.</p>', 'posted': '2021-07-20'}, {'doi': '10.1101/2021.07.22.21260852', 'tweets': 19, 'archive': 'medRxiv', 'subject-area': 'Transplantation', 'covid': True, 'title': 'Comparison of infected and vaccinated transplant recipients highlights the role of Tfh and neutralizing IgG in COVID-19 protection', 'authors': \"[{'name': 'Xavier Charmetant'}, {'name': 'Maxime ESPI'}, {'name': 'Ilies Benotmane'}, {'name': 'Francoise Heibel'}, {'name': 'Fanny Buron'}, {'name': 'Gabriela Gautier-Vargas'}, {'name': 'Marion Delafosse'}, {'name': 'Peggy Perrin'}, {'name': 'Alice Koenig'}, {'name': 'Noelle Cognard'}, {'name': 'Charlene Levi'}, {'name': 'Floriane Gallais'}, {'name': 'Louis Maniere'}, {'name': 'Paola Rossolillo'}, {'name': 'Eric Soulier'}, {'name': 'Florian Pierre'}, {'name': 'Anne Ovize'}, {'name': 'Emmanuel Morelon'}, {'name': 'Thierry Defrance'}, {'name': 'Samira Fafi-Kremer'}, {'name': 'Sophie Caillard'}, {'name': 'Olivier Thaunat'}]\", 'abstract': '<p>Transplant recipients, which receive therapeutic immunosuppression to prevent graft rejection, are characterized by high COVID-19-related mortality and defective response to vaccines. Having observed that previous infection by SARS-CoV-2 but not the standard 2 doses scheme of vaccination, provided complete protection against COVID-19 to transplant recipients, we undertook this translational study to compare the cellular and humoral immune responses of these 2 groups of patients. Neutralizing anti-Receptor Binding Domain (RBD) IgG were identified as the critical immune effectors associated with protection. Generation of anti-RBD IgG was dependent upon spike-specific T follicular helper (Tfh) CD4+ T cells, which acted as limiting checkpoint. Tfh generation was impeded by high dose mycophenolate mofetil in non-responders to vaccine but not in infected patients, suggesting that increasing immunogenicity of vaccine could improve response rate to mRNA vaccine. This theory was validated in two independent prospective cohorts, in which administration of a 3rd dose of vaccine resulted in the generation of anti-RBD IgG in half of non-responders to 2 doses.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.20.453126', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Differences in the dynamics of the tandem-SH2 modules of the Syk and ZAP-70 tyrosine kinases', 'authors': \"[{'name': 'Helen T Hobbs'}, {'name': 'Neel H Shah'}, {'name': 'Jean M Badroos'}, {'name': 'Christine L Gee'}, {'name': 'Susan Marqusee'}, {'name': 'John Kuriyan'}]\", 'abstract': '<p>The catalytic activity of Syk-family tyrosine kinases is regulated by a tandem-SH2 module (tSH2 module). In the autoinhibited state, this module adopts a conformation which stabilizes an inactive conformation of the kinase domain. The binding of the tSH2 module to doubly-phosphorylated tyrosine-containing motifs necessitates a conformational change, thereby relieving kinase inhibition and promoting activation. We determined the crystal structure of the isolated tSH2 module of Syk and find, in contrast to ZAP-70, that its conformation more closely resembles that of the peptide-bound state, rather than the autoinhibited state. Hydrogen-deuterium exchange by mass spectrometry, as well as molecular dynamics simulations, reveal that the dynamics of the tSH2 modules of Syk and ZAP-70 differ, with most of these differences occurring in the C-terminal SH2 domain. Our data suggest that the conformational landscapes of the tSH2 modules in Syk and ZAP-70 have been tuned differently, such that the auto-inhibited conformation of the Syk tSH2 module is less stable. This feature of Syk likely contributes to its ability to more readily escape autoinhibition when compared to ZAP-70, consistent with tighter control of downstream signaling pathways in T cells.</p>', 'posted': '2021-07-20'}, {'doi': '10.1101/2021.07.21.453258', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Retrosplenial cortex is necessary for spatial and non-spatial latent learning in mice', 'authors': \"[{'name': 'Ana Carolina Bottura de Barros'}, {'name': 'Liad J. Baruchin'}, {'name': 'Marios C. Panayi'}, {'name': 'Nils Nyberg'}, {'name': 'Veronika Samborska'}, {'name': 'Mitchell T. Mealing'}, {'name': 'Thomas Akam'}, {'name': 'Jeehyun Kwag'}, {'name': 'David M. Bannerman'}, {'name': 'Michael M. Kohl'}]\", 'abstract': '<p>Latent learning occurs when associations are formed between stimuli in the absence of explicit reinforcement. Traditionally, latent learning in rodents has been associated with the creation internal models of space. However, increasing evidence points to roles of internal models also in non-spatial decision making. Whether the same brain structures and processes support the creation of spatially-anchored or non-spatial internal models via latent learning, is an open question. To address this question, we developed a novel operant box task that allows to test spatial and non-spatial versions of a flavour-based sensory preconditioning paradigm. We probed the role of the retrosplenial cortex, a brain area associated with spatial cognition and subjective value representation, in this task using precise, closed-loop optogenetic silencing during different task phases. We show that the retrosplenial cortex is necessary for both spatial and non-spatial latent learning in mice. We further demonstrate that the requirement of retrosplenial cortex is limited to the preconditioning phase of the task. Our results provide insight into the specific role of the retrosplenial cortex in latent learning, demonstrate that latent learning plays a general part in the creation of internal models, independent of spatial anchors, and provide a novel avenue for studying model-based decision making.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.22.21260551', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Willingness to receive vaccination against COVID-19: results from a large nationally representative Australian population survey', 'authors': \"[{'name': 'Sharon R Davis'}, {'name': 'Rosario D Ampon'}, {'name': 'Leanne D Poulos'}, {'name': 'Guy B Marks'}, {'name': 'Brett Gregory Toelle'}, {'name': 'Helen K Reddel'}]\", 'abstract': '<p>We surveyed 10,024 Australians regarding COVID-19 vaccine willingness. Overall, 59.9% indicated yes, 13.9% no and 26.3% unsure/dont know. Vaccine willingness was higher in males, and increased with increasing education and socioeconomic advantage. Results contrast with earlier, smaller Australian surveys regarding vaccination willingness and confirm the need for targeted vaccination information.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.18.21259977', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Altered functional connectome hierarchy with gene expression signatures in newly-diagnosed focal epilepsy', 'authors': \"[{'name': 'Christophe E de Bzenac'}, {'name': 'Lorenzo Caciagli'}, {'name': 'Batil K Alonazi'}, {'name': 'Boris C Bernhardt'}, {'name': 'Anthony G Marson'}, {'name': 'Simon S Keller'}]\", 'abstract': '<sec><title>Objective</title><p>Neuroimaging research is providing insights into epilepsy as a disorder of brain connectivity linked to functional impairments which may have an identifiable genetic component. This case-control study aims to identify imbalances in a functional connectome dimension spanning from unimodal to transmodal networks and explore the potential genetic basis of such alterations in patients with newly diagnosed focal epilepsy (NDfE).</p></sec><sec><title>Methods</title><p>We used gradient-based analysis of resting-sate fMRI data comparing cortical gradient maps in patients with NDfE (n = 27) to age and sex-matched controls (n = 36). Using a brain-wide gene expression dataset, gene combinations associated with altered brain regions were then entered into an enrichment analysis.</p></sec><sec><title>Results</title><p>We found an increased differentiation of connectivity profiles between unimodal and transmodal networks in NDfE, which was particularly pronounced in the patients with persistent seizures at 12-months follow-up (n=10). Differences corresponded to gradient score reductions in a visual network and increases in limbic and default mode systems which subserve higher-level cognition. Cortical difference maps were spatially correlated with regional expression of a weighted gene combination. These genes were enriched for disease and ontology terms and pathways previously associated with epilepsy and seizure susceptibility.</p></sec><sec><title>Interpretations</title><p>Large-scale functional hierarchy may be altered from in focal epilepsy from diagnosis and correlate with response to treatment. Combining functional neuroimaging and transcriptional data analysis may provide a framework for understanding the wide-ranging impairments associated with the disorder and mechanistic insight into how gene processes may drive alterations in brain function mediating the genetic risk of epilepsy.</p></sec>', 'posted': '2021-07-22'}, {'doi': '10.1101/2021.07.24.453184', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The dual nature of bacteriophage: growth-dependent predation and generalised transduction of antimicrobial resistance', 'authors': \"[{'name': 'Quentin J Leclerc'}, {'name': 'Jacob Wildfire'}, {'name': 'Arya Gupta'}, {'name': 'Jodi A Lindsay'}, {'name': 'Gwenan M Knight'}]\", 'abstract': '<p>Bacteriophage (\"phage\") are both predators and evolutionary drivers for bacteria, notably contributing to the spread of antimicrobial resistance (AMR) genes by generalised transduction. Our current understanding of the dual nature of this relationship is limited. We used an interdisciplinary approach to quantify how these interacting dynamics can lead to the evolution of multi-drug resistant bacteria. We co-cultured two strains of Methicillin-resistant <italic>Staphylococcus aureus</italic>, each harbouring a different antibiotic resistance gene, with 80 generalized transducing phage. After a growth phase of 8h, bacteria and phage surprisingly coexisted at a stable equilibrium in our culture, the level of which was dependent on the starting concentration of phage. We detected double-resistant bacteria as early as 7h, indicating that transduction of AMR genes had occurred. We developed multiple mathematical models of the bacteria and phage relationship, and found that phage-bacteria dynamics were best captured by a model in which the phage burst size decreases as the bacteria population reaches stationary phase, and where phage predation is frequency-dependent. We estimated that one in every 10<sup>8</sup> new phage generated was a transducing phage carrying an AMR gene, and that double-resistant bacteria were always predominantly generated by transduction rather than by growth. Our results suggest a fundamental shift in how we understand and model phage-bacteria dynamics. Although rates of generalised transduction may seem insignificant, they are sufficient to consistently lead to the evolution of multi-drug resistant bacteria. Currently, the potential of phage to contribute to the growing burden of AMR is likely underestimated.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.22.21258785', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Gastroenterology', 'covid': True, 'title': 'Proton pump inhibitor use is not associated with severe COVID-19 related outcomes: A propensity score weighted analysis of a national veteran cohort', 'authors': \"[{'name': 'Shailja C. Shah'}, {'name': 'Alese Halvorson'}, {'name': 'Brandon McBay'}, {'name': 'Chad Dorn'}, {'name': 'Otis Wilson'}, {'name': 'Sony Tuteja'}, {'name': 'Kyong-Mi Chang'}, {'name': 'Ayako Suzuki'}, {'name': 'Christine Hunt'}, {'name': 'Richard Hauger'}, {'name': 'Kelly Cho'}, {'name': 'Edward Siew'}, {'name': 'Michael Matheny'}, {'name': 'Adriana Hung'}, {'name': 'Robert Greevy'}, {'name': 'Christianne Roumie'}]\", 'abstract': '<p>Background and Aims: Low pH deactivates most pathogens, including coronaviruses. Proton pump inhibitors (PPIs) are potent gastric acid suppressing medications. Whether PPI use vs non-use is associated with severe Coronavirus disease-2019 (COVID-19) outcomes remains uncertain. We aimed to compare severe COVID-19 outcomes between current outpatient PPI users and non-users.\\n\\nMethods: We conducted a retrospective propensity score-weighted analysis of a national cohort of US veterans with established care who tested positive for severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) through January 9, 2021, and who had 60 days of follow-up. The positive test date was the index date. Current outpatient PPI use up to and including the index date (primary exposure) was compared to non-use, defined as no PPI prescription fill in the 365 days prior to the index date. The primary outcome was a composite of use of mechanical ventilation or death within 60 days. Weighted logistic regression models evaluated severe COVID-19 outcomes between current PPI users vs non-users.\\n\\nResults: Of 97,674 Veterans with SARS-CoV-2 testing, 14,958 tested positive (6262 [41.9%] current PPI users, 8696 [58.1%] non-users) and comprised the analytic cohort. After weighting, all covariates were well-balanced. In the weighted cohort, there was no difference in the primary composite outcome (8.2% vs 8.0%; OR 1.03, 95% CI 0.91-1.16), secondary composite outcome, nor individual component outcomes between current PPI users and non-users. There was no significant interaction between age and PPI use on outcomes.\\n\\nConclusion: Among patients with SARS-CoV-2 infection, current PPI use vs non-use is not associated with severe COVID-19 outcomes.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.23.453610', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Adapted protocol for Saccharibacteria co-cultivation: two new members join the club of Candidate Phyla radiation', 'authors': \"[{'name': 'Ahmad Ibrahim'}, {'name': 'Mohamad Maatouk'}, {'name': 'Andreamiharimamy Rajaonison'}, {'name': 'Rita Zgheib'}, {'name': 'Gabriel Haddad'}, {'name': 'Jacques Yaacoub Bou Khalil'}, {'name': 'Didier Raoult'}, {'name': 'Fadi Bittar'}]\", 'abstract': '<p>The growing application of metagenomics to different ecological and microbiome niches in recent years has enhanced our knowledge of global microbial biodiversity. Among these abundant and widespread microbes, Candidate Phyla Radiation or CPR have been recognised as representing a large proportion of the microbial kingdom (&gt; 26%). CPR are characterised by their obligate symbiotic or exo-parasitic activity with other microbial hosts, mainly bacteria. Currently, isolating CPR is still considered challenging for microbiologists. The idea of this study was to develop an adapted protocol for the co-culture of CPR with a suitable bacterial host. Based on various sputa, we tried to purify CPR (Saccharibacteria members) and to cultivate them with pure hosts. This protocol was monitored by real-time PCR quantification using a specific system for Saccharibacteria designed in this study, as well as by electron microscopy and sequencing. We succeeded in co-culturing and sequencing a complete genome of two new Saccharibacteria species: Candidatus Minimicrobia naudis and Candidatus Minimicrobia vallesae. In addition, we noticed a decrease in the Ct number of Saccharibacteria, and a significant multiplication through their physical association with Schaalia odontolytica strains in the enriched medium that we developed. This work may help bridge gaps in the genomic database by providing new CPR members and, in the future, their currently unknown characteristics may be revealed.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.23.453553', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Community health and human-animal contacts on the edges of Bwindi Impenetrable National Park, Uganda', 'authors': \"[{'name': 'Renata L. Muylaert'}, {'name': 'Ben Davidson'}, {'name': 'Alex Ngabirano'}, {'name': 'Gladys Kalema-Zikusoka'}, {'name': 'Hayley MacGregor'}, {'name': 'James O. Lloyd-Smith'}, {'name': 'Ahmed Fayaz'}, {'name': 'Matthew A. Knox'}, {'name': 'David T. S. Hayman'}]\", 'abstract': '<p>Cross-species transmission of pathogens is intimately linked to human and environmental health. With limited healthcare and challenging living conditions, people living in poverty may be particularly susceptible to endemic and emerging diseases. Similarly, wildlife is impacted by human influences, including pathogen sharing, especially for species in close contact with people and domesticated animals. Here we investigate human and animal contacts and human health in a community living around the Bwindi Impenetrable National Park (BINP), Uganda. We used contact and health survey data to identify opportunities for cross-species pathogen transmission, focusing mostly on people and the endangered mountain gorilla. We conducted a survey with background questions and self-reported diaries to investigate 100 participants health, such as symptoms and behaviours, and contact patterns, including direct contacts and sightings over a week. Contacts were revealed through networks, including humans, domestic, peri-domestic, and wild animals for 1) network of contacts seen in the week of background questionnaire completion, 2) network of contacts seen during the diary week. Participants frequently felt unwell during the study, reporting from one to 10 disease symptoms at different intensity levels (maximum of seven symptoms in one day), with severe symptoms comprising 6.4% of the diary records and tiredness and headaches the most common symptoms. Besides human-human contacts, direct contacts with livestock and peri-domestic animals were the most common. Wildlife contacts were the rarest, including one direct contact with gorilla with a concerning timeline of reported symptoms. The contact networks were moderately connected and revealing a preference in contacts within the same species or taxon and within their groups. Despite sightings of wildlife being much more common than touching, one participant declared direct contact with a mountain gorilla during the week. Gorillas were seen very close to six animal taxa (including themselves) considering all interaction types, mostly seen closer to other gorillas, but also people and domestic animals. Our findings reveal a local human population with recurrent symptoms of illness in a location with intense exposure to factors that can increase pathogen transmission, such as direct contact with domestic and wild animals and proximity among animal species. Despite significant biases and study limitations, the information generated here can guide future studies, such as models for disease spread and One Health interventions.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453512', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'In vitro and in vivo analysis of microvesicle-mediated metastasis using a bright, red-shifted bioluminescent reporter protein of extracellular vesicles', 'authors': \"[{'name': 'Ahmed A Zarea'}, {'name': 'Gloria I Perez'}, {'name': 'David Broadbent'}, {'name': 'Benedikt Dolgikh'}, {'name': 'Matthew P Bernard'}, {'name': 'Alicia Withrow'}, {'name': 'Amelia McGill'}, {'name': 'Victoria Toomajian'}, {'name': 'Lukose K Thampy'}, {'name': 'Jack Harkema'}, {'name': 'Joel R Walker'}, {'name': 'Thomas A Kirkland'}, {'name': 'Michael H Bachmann'}, {'name': 'Jens Schmidt'}, {'name': 'Masamitsu Kanada'}]\", 'abstract': '<p>Cancer cells produce heterogeneous extracellular vesicles (EVs) as mediators of intercellular communication. Our study focused on a novel method to image EV subtypes and their biodistribution in vivo. Regardless of injection routes, we established that reporter EVs isolated from murine mammary carcinoma cells expressing PalmReNL, which utilizes bioluminescence resonance energy transfer (BRET), localized to the lungs. This new EV reporter allowed highly sensitive EV tracking in vitro and in vivo and enabled us to begin studies to understand the commonalities and functional differences of the EV subtypes. We demonstrated the early appearance of metastatic foci in the lungs of mammary tumor-bearing mice following multiple injections of the microvesicle (MV)-enriched fraction derived from mammary carcinoma cells. In addition, the results we present here show that tumor cell-derived MVs act on distant tissues through upregulating LC3 expression within the lung.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.22.453456', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Fast transcriptional activation of developmental signalling pathways during wound healing of the calcareous sponge Sycon ciliatum', 'authors': \"[{'name': 'Cneyt Caglar'}, {'name': 'Alexander Ereskovsky'}, {'name': 'Mary Laplante'}, {'name': 'Daria Tokina'}, {'name': 'Sven Leininger'}, {'name': 'Ilya Borisenko'}, {'name': 'Genevieve Aisbett'}, {'name': 'Di Pan'}, {'name': 'Marcin Adamski'}, {'name': 'Maja Adamska'}]\", 'abstract': '<sec><title>Background</title><p>The ability to regenerate lost or damaged body parts is an ancient animal characteristic with a wide yet variable distribution across all phyla. Sponges, likely the sister group to all other animals, have remarkable regenerative abilities including whole body regeneration and re-development from dissociated cells. The calcareous sponge <italic>Sycon ciliatum</italic> has been subject to various regeneration studies since the beginning of the last century. However, the early steps of wound healing of <italic>S. ciliatum</italic> have not been addressed from the molecular perspective.</p></sec><sec><title>Results</title><p>In this study, we combined electron microscopy with gene expression analysis to investigate wound healing after transverse sectioning of <italic>S. ciliatum</italic>. Microscopic analysis revealed massive transdifferentiation and collective migration behaviour of choanocytes and pinacocytes early upon injury (6-12h) as the main mechanisms for quick closure of the wound surface. RNA-sequencing identified upregulation of components of the conserved metazoan Wnt and TGF signalling pathways within 3h, preceding morphologically detectable wound healing events. De novo upregulation after a decline in expression coincides with morphologically visible polarity establishment. Moreover, by integrating the new wound healing data set with previously published data derived from intact sponge, we demonstrate similarity between gene activity during early wound healing and osculum maintenance. Whole mount <italic>in situ</italic> hybridisation of the TGF signalling pathway ligand SciTGFU and signal transducer SciSmadRa show that the early activation of both is initially encompassing a large area surrounding the cut surface with gradual restriction to the edge of the forming regenerative membrane as wound healing progresses. While SciTGFU transcripts are localised to exo- and endopinacocytes, SciSmadRa expression appears across all cell types. Using an EdU cell proliferation assay, we found that a global increase in cell proliferation is not visible before 12h into wound healing. Hence, the initial stages to cover the injury site including cell transdifferentiation and migration seem to be executed by cells remaining after injury. Gene expression clustering coupled with GO term enrichment analysis confirmed that expression of genes involved in processes related to cell proliferation, DNA repair as well as apoptotic processes at 3 and 6h of wound healing was not upregulated. On the other hand, genes associated with positive regulation of transcription, signal transduction, actin filament and chromatin organisation, as well as the Wnt signalling pathway are upregulated at early wound healing stages.</p></sec><sec><title>Conclusion</title><p>We have analysed wound healing in the calcareous sponge <italic>Sycon ciliatum</italic> using microscopic and genomic methods. This study highlights a remarkable mechanism of interplay between cell transdifferentiation and collective migration we hypothesise to be regulated by conserved metazoan developmental pathways and numerous taxonomically restricted genes. Expression of these genes in regenerating and intact sponges sheds light on the long-standing question whether embryonic developmental pathways are redeployed in regeneration.</p></sec>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.19.453008', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': \"Don't think, feel: basal ganglia-cortical connectivity underlies self-regulation of brain oscillations in humans\", 'authors': \"[{'name': 'Kazumi Kasahara'}, {'name': 'Charles Sayo DaSalla'}, {'name': 'Manabu Honda'}, {'name': 'Takashi Hanakawa'}]\", 'abstract': '<p>Brain-computer interfaces (BCIs) provide an artificial link by which the brain can directly interact with the environment. To achieve fine BCI control, participants must modulate the patterns of the cortical oscillations generated from the motor and somatosensory cortices. However, it remains unclear how humans regulate cortical oscillations, the controllability of which substantially varies across individuals. Here, we performed simultaneous electroencephalography (to assess BCI control) and functional magnetic resonance imaging (to measure brain activity) in healthy participants. Self-regulation of cortical oscillations induced activity in the basal ganglia-cortical network (BgCN) and the neurofeedback control network (NfCN). Successful self-regulation correlated with striatal activity in the BgCN, through which patterns of cortical oscillations were likely modulated. Moreover, BgCN and NfCN connectivity correlated with strong and weak self-regulation, respectively. The findings indicate that the BgCN is important for self-regulation, the understanding of which should help advance BCI technology.</p>', 'posted': '2021-07-20'}, {'doi': '10.1101/2021.07.23.453341', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Trait-trait relationships and functional tradeoffs vary with genome size in prokaryotes', 'authors': \"[{'name': 'Sara Beier'}, {'name': 'Johannes Werner'}, {'name': 'Thierry Bouvier'}, {'name': 'Nicolas Mouquet'}, {'name': 'Cyrille Violle'}]\", 'abstract': '<p>We report genomic traits that have been associated with the life history of prokaryotes and highlight conflicting findings concerning earlier observed trait correlations and tradeoffs. In order to address possible explanations for these contradictions we examined trait-trait variations of 11 genomic traits from  17,000 sequenced genomes. The studied trait-trait variations suggested: (i) the predominance of two resistance and resilience-related orthogonal axes, (ii) an overlap between a resilience axis and an axis of resource usage efficiency. These findings imply that resistance associated traits of prokaryotes are globally decoupled from resilience and resource use efficiencies associated traits. However, further inspection of pairwise scatterplots showed that resistance and resilience traits tended to be positively related for genomes up to roughly five million base pairs and negatively for larger genomes. This in turn precludes a globally consistent assignment of prokaryote genomic traits to the competitor - stress-tolerator -ruderal (CSR) schema that sorts species depending on their location along disturbance and productivity gradients into three ecological strategies and may serve as an explanation for conflicting findings from earlier studies. All reviewed genomic traits featured significant phylogenetic signals and we propose that our trait table can be applied to extrapolate genomic traits from taxonomic marker genes. This will enable to empirically evaluate the assembly of these genomic traits in prokaryotic communities from different habitats and under different productivity and disturbance scenarios as predicted via the resistance-resilience framework formulated here.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.23.453533', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Decoding the Neural Signatures of Valence and Arousal From Portable EEG Headset', 'authors': \"[{'name': 'Nikhil Garg'}, {'name': 'Rohit Garg'}, {'name': 'NS Parrivesh'}, {'name': 'Apoorv Anand'}, {'name': 'V.A.S. Abhinav'}, {'name': 'Veeky Baths'}]\", 'abstract': '<p>This paper focuses on classifying emotions on the valence-arousal plane using various feature extraction, feature selection and machine learning techniques. Emotion classification using EEG data and machine learning techniques has been on the rise in the recent past. We evaluate different feature extraction techniques, feature selection techniques and propose the optimal set of features and electrodes for emotion recognition. The images from the OASIS image dataset were used for eliciting the valence and arousal emotions, and the EEG data was recorded using the Emotiv Epoc X mobile EEG headset. The analysis is additionally carried out on publicly available datasets: DEAP and DREAMER. We propose a novel feature ranking technique and incremental learning approach to analyze the dependence of performance on the number of participants. Leave-one-subject-out cross-validation was carried out to identify subject bias in emotion elicitation patterns. The importance of different electrode locations was calculated, which could be used for designing a headset for emotion recognition. Our study achieved root mean square errors (RMSE) of less than 0.75 on DREAMER, 1.76 on DEAP, and 2.39 on our dataset.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453507', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Toxin breakdown does not preclude the potential for defensive toxin use in a fruit fly', 'authors': \"[{'name': 'Tyler E. Douglas'}, {'name': 'Sofia G. Beskid'}, {'name': 'Callie E. Gernand'}, {'name': 'Brianna E. Nirtaut'}, {'name': 'Kristen E. Tamsil'}, {'name': 'Richard W. Fitch'}, {'name': 'Rebecca D. Tarvin'}]\", 'abstract': '<p>Animals that ingest toxins can themselves become toxic or unpalatable to predators and parasites. Because most animals rapidly eliminate toxins to survive toxin ingestion, it is unclear how species transition from susceptibility and toxin elimination to tolerance and accumulation as chemical defense emerges. Studies of chemical defense have generally focused on species that display active toxin sequestration and target-site insensitivity mutations that permit survival without necessitating toxin metabolism. Here we investigate whether animals that presumably rely on toxin elimination for survival can also utilize ingested toxins for defense. We use the A4 and A3 <italic>Drosophila melanogaster</italic> fly strains from the Drosophila Synthetic Population Resource (DSPR), which respectively possess elevated and reduced metabolic resistance to nicotine. We find that ingesting nicotine increased the survival of A4 but not of A3 flies against <italic>Leptopilina heterotoma</italic> wasp parasitism. Further, we find that despite possessing enhanced toxin clearance mechanisms, A4 flies accrued more nicotine than A3 individuals. Our results suggest that enhanced metabolic detoxification can allow for greater toxin intake by offsetting the cost of toxin ingestion. Passive toxin accumulation that accompanies increased toxin intake may underlie the early origins of chemical defense.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.22.453458', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Epithelial-mesenchymal plasticity determines estrogen receptor positive (ER+) breast cancer dormancy and reacquisition of an epithelial state drives awakening', 'authors': \"[{'name': 'Aouad Patrik'}, {'name': 'Zhang Yueyun'}, {'name': 'Celine Stibolt'}, {'name': 'Mani Sendurai'}, {'name': 'Georgios Sflomos'}, {'name': 'Cathrin Brisken'}]\", 'abstract': '<p>Estrogen receptor -positive (ER+) breast cancers (BCs) represent more than 70% of all breast cancers and pose a particular clinical challenge because they recur up to decades after initial diagnosis and treatment. The mechanisms governing tumor cell dormancy and latent disease remain elusive due to a lack of adequate models. Here, we compare tumor progression of ER+ and triple-negative (TN) BC subtypes with a clinically relevant mouse intraductal  xenografting approach (MIND). Both ER+ and TN BC cells disseminate already during the in situstage. However, TN disseminated tumor cells (DTCs) proliferate at the same rate as cells at the primary site and give rise to macro-metastases. ER+ DTCs have low proliferative indices, form only micro-metastases and lose epithelial characteristics. Expression of CDH1 is decreased whereas the mesenchymal marker VIM and the transcription factors, ZEB1/ZEB2, which control epithelial-mesenchymal plasticity (EMP) are increased. EMP is not detected earlier during ER+ BC development and not required for invasion or metastasis. In vivo, forced transition to the epithelial state through ectopic E-cadherin expression overcomes dormancy with increased growth of lung metastases. We conclude that EMP is essential for the generation of a dormant cell state and the development of latent disease. Targeting exit from EMP is of therapeutic potential.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.453319', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Pectin remodeling belongs to a homeostatic system and triggers transcriptomic and hormonal modulations', 'authors': \"[{'name': 'Franois Jobert'}, {'name': 'Stphanie Gunin'}, {'name': 'Aline Voxeur'}, {'name': 'Kieran J. D. Lee'}, {'name': 'Sophie Bouton'}, {'name': 'Fabien Snchal'}, {'name': 'Ludivine Hocq'}, {'name': 'Galle Mongelard'}, {'name': 'Herv Demailly'}, {'name': 'Petra Amakorov'}, {'name': 'Miroslav Strnad'}, {'name': 'Samantha Vernhettes'}, {'name': 'Gregory Mouille'}, {'name': 'Serge Pilard'}, {'name': 'J. Paul Knox'}, {'name': 'Ondej Novk'}, {'name': 'Jrme Pelloux'}, {'name': 'Laurent Gutierrez'}]\", 'abstract': '<p>Pectins occur in primary cell walls and consist of multiblock co-polymers among which homogalacturonan (HG) is the simplest and most abundant form. Methylesterification patterns of HG are tuned by pectin methylesterases (PMEs), the activities of which are controlled by specific inhibitors (PMEIs). By impacting cell wall mechanical properties, PME-mediated regulation of HG methylesterification plays a major role in several developmental processes, including seed germination and dark-grown hypocotyl elongation. Arabidopsis <italic>PME36</italic> is preferentially expressed during the late stage of seed development and, using the knock-out mutant <italic>pme36-1</italic>, we show here that PME36 is required to implement the characteristic pattern of de-methylesterified pectin in the mature seed. Surprisingly, while this pattern is strongly impaired in <italic>pme36-1</italic> mature seed, no phenotypical effect is observed in the mutant during seed germination and dark-grown hypocotyl elongation, suggesting the existence of a compensatory mechanism overcoming the defect in pectin de-methylesterification. By analyzing hormone contents and gene expression, a strong, dynamic, and long-lasting physiological disorder is revealed in the mutant. These results suggest the existence of complex connections between pectin remodeling, transcriptomic regulations and hormonal homeostasis, modulating several physiological parameters to ensure the maintenance of a normal seed-to-seedling developmental program in <italic>pme36-1</italic>. Considered for a long time as an end-point passive effector mainly involved in modification of cell wall mechanics, the role of pectin methylesterification needs to be reconsidered as a modulator acting upstream of diverse regulatory pathways involved in plant development.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453605', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Identifying cell-state associated alternative splicing events and their co-regulation', 'authors': \"[{'name': 'Carlos F Buen Abad Najar'}, {'name': 'Prakruthi Burra'}, {'name': 'Nir Yosef'}, {'name': 'Liana F Lareau'}]\", 'abstract': \"<p>Alternative splicing shapes the transcriptome and contributes to each cell's unique identity, but single-cell RNA sequencing has struggled to capture the impact of alternative splicing. We previously showed that low recovery of mRNAs from single cells led to erroneous conclusions about the cell-to-cell variability of alternative splicing. Here, we present a method, Psix, to confidently identify splicing that changes across a landscape of single cells, using a probabilistic model that is robust against the data limitations of scRNA-seq. Its autocorrelation-inspired approach finds patterns of alternative splicing that correspond to patterns of cell identity, such as cell type or developmental stage, without the need for explicit cell clustering, labeling, or trajectory inference. Applying Psix to data that follow the trajectory of mouse brain development, we identify exons whose alternative splicing patterns cluster into modules of co-regulation. We show that the exons in these modules are enriched for binding by distinct neuronal splicing factors, and that their changes in splicing correspond to changes in expression of these splicing factors. Thus, Psix reveals cell-type-dependent splicing patterns and the wiring of the splicing regulatory networks that control them. Our new method will enable scRNA-seq analysis to go beyond transcription to understand the roles of post-transcriptional regulation in determining cell identity.</p>\", 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.23.453525', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Zoology', 'covid': False, 'title': 'Insights on the historical biogeography of Philippine native pigs and its relationship with Continental domestic and wild boars', 'authors': \"[{'name': 'John King N. Layos'}, {'name': 'Ronel N. Geromo'}, {'name': 'Dinah M. Espina'}, {'name': 'Masahide Nishibori'}]\", 'abstract': '<p>The Philippine archipelago was believed to have never been connected to the Asian continent even during the severe Quaternary sea-level drops. As a result, the history of pig dispersal in the Philippines remains controversial and must have some anthropogenic origin associated with some human migration events. In this study, the context of origin, dispersal, and the level of genetic introgression in Philippine native pigs were deduced using mitochondrial DNA D-loop analysis altogether with domestic pigs and wild boars corresponding to their geographic origin. Results revealed a considerable genetic diversity (0.9000.016), and a widespread Asian pig-ancestry (94.60%) were revealed in the phylogenetic analysis with admixed European pig-origin (5.10%) harboring various fractions of ancestry from Berkshire and Landrace. The close genetic connection between the continental wild boars and domestic pigs present in the Philippine pigs corroborates our hypothesis of a genetic signal that could potentially be associated with the recently reported multiple waves of human migrations to the Philippines during the last 50,000 years. The high frequency of haplotypes (54.08%) that collapsed in the D7 haplogroup represent an interesting challenge as its distribution does not coincide with the hypothesized migratory route of the Neolithic Austronesian-speaking populations. We detected the first Pacific Clade signature and ubiquitously distributed D2 haplotypes which postulate the legitimate dispersal of pigs associated with the multiple waves of human migrations involving the Philippines. The multimodal mismatch and neutrality test statistics both Fus F<italic>s</italic> and Tajimas D correlates the long stationary period of effective population size revealed in the Bayesian skyline plot. While the sudden decrease in population was consistent with the pronounced population bottleneck of Asian and European pigs during the interglacial periods of the Pleistocene.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453515', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Low dose/dose rate  irradiation protects Drosophila melanogaster chromosomes from double strand breaks and telomere fusions by modulating the expression of Loquacious.', 'authors': \"[{'name': 'Antonella Porrazzo'}, {'name': 'Francesca Cipressa'}, {'name': 'Alex De Gregorio'}, {'name': 'Cristiano De Pitta'}, {'name': 'Gabriele Sales'}, {'name': 'Patrizia Morciano'}, {'name': 'Giuseppe Esposito'}, {'name': 'Antonella Tabocchini'}, {'name': 'Giovanni Cenci'}]\", 'abstract': '<p>It is still continuously debated whether the low-dose/dose-rate (LDDR) of ionizing radiation represents a hazard for humans. Model organisms, such as fruit flies, are considered valuable systems to reveal insights into this issue. We found that, in wild-type Drosophila melanogaster larval neuroblasts, the frequency of Chromosome Breaks (CBs), induced by acute -radiation, is significantly reduced when flies are previously exposed to a protracted dose of 0.4 Gy delivered at a dose rate of 2.5 mGy/h. This indicates that this exposure, which is associated with an increased expression of DNA damage response proteins, induces a radioadaptive response (RAR) that protects Drosophila from extensive DNA damage. Interestingly, the same exposure reduces the frequency of telomere fusions (TFs) from Drosophila telomere capping mutants suggesting that the LDDR can generally promote a protective response on chromatin sites that are recognized as DNA breaks. Deep RNA sequencing revealed that RAR is associated with a modulation of genes involved in RNA processing and the reduction of expression of Loquacious D (Loqs-RD) gene that encodes a well-conserved dsRNA binding protein required for esiRNAs biogenesis. Remarkably, loss of Loqs mimics the LDDR-mediated chromosome protection as it decreases the IR-induced CBs and TF frequency. Thus, our work identifies Loqs as a key factor in mediating the cellular response to LDDR and reveals that the study of the molecular bases underlying RAR is instrumental for unraveling novel and conserved epigenetic routes that have evolved to preserve genome integrity following robust insults to DNA.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.23.21261042', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Robust antibody levels in both diabetic and non-diabetic individuals after BNT162b2 mRNA COVID-19 vaccination', 'authors': \"[{'name': 'Hamad Ali'}, {'name': 'Abdulmohsen Al-Terki'}, {'name': 'Sardar Sindhu'}, {'name': 'Barrak Alahmad'}, {'name': 'Maha Hammad'}, {'name': 'Salman Al-Sabah'}, {'name': 'Mohammad Alghounaim'}, {'name': 'Mohammad H. Jamal'}, {'name': 'Ali Aldei'}, {'name': 'Mohammad J. Mairza'}, {'name': 'Maitham Husain'}, {'name': 'Sriraman Devarajan'}, {'name': 'Rasheed Ahmad'}, {'name': 'Preethi Cherian'}, {'name': 'Irina Alkhairi'}, {'name': 'Abdullah Alkandari'}, {'name': 'Jehad Abubaker'}, {'name': 'Mohamed Abu-farha'}, {'name': 'Fahd Al-Mulla'}]\", 'abstract': '<p>The emergence of effective vaccines for COVID-19 has been welcomed by the world with great optimism. Given their increased susceptibility to COVID-19, the question arises whether individuals with type-2 diabetes mellitus (T2DM) and other metabolic conditions can respond effectively to the mRNA-based vaccine. We aimed to evaluate the levels of anti-SARS-CoV-2 IgG and neutralizing antibodies in people with T2DM and/or other metabolic risk factors (hypertension and obesity) compared to those without. This study included 262 people that took two doses of BNT162b2 (Pfizer-BioNTech) mRNA vaccine. Both T2DM and non-diabetic individuals had a robust response to vaccination as demonstrated by their high antibody titers. However, both SARS-CoV-2 IgG and neutralizing antibodies titers were lower in people with T2DM. Their levels were 15449.1 vs. 13859.4BAU/mL for IgG and 87.111.6 vs. 79.719.5% for neutralizing antibodies in individuals without diabetes compared to those with T2DM, respectively. In a multiple linear regression adjusted for individual characteristics, comorbidities, previous COVID-19 infection and duration since second vaccine dose, diabetics had 13.86 BAU/ml (95%CI: -27.08 to -0.64BAU/ml, p=0.041) less IgG antibodies and 4.42% (95%CI: -8.53 to -0.32%, p=0.036) less neutralizing antibodies than non-diabetics. Hypertension and obesity did not show significant changes in antibody titers. Taken together, both type-2 diabetic and non-diabetic individuals elicited strong immune responses to SARS-CoV-2 BNT162b2 mRNA vaccine; nonetheless, lower levels were seen in people with diabetes. Continuous monitoring of the antibody levels might be a good indicator to guide personalized needs for further booster shots to maintain adaptive immunity.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261041', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'The impact of large mobile air purifiers on aerosol concentration in classrooms and the reduction of airborne transmission of SARS-CoV-2', 'authors': \"[{'name': 'Finn F. Duill'}, {'name': 'Florian Schulz'}, {'name': 'Aman Jain'}, {'name': 'Leve Krieger'}, {'name': 'Berend van Wachem'}, {'name': 'Frank Beyrau'}]\", 'abstract': '<p>In the wake of the SARS-CoV-2 pandemic, an increased risk of infection by virus-containing aerosols indoors is assumed. Especially in schools, the duration of stay is long and the number of people in the rooms is large, increasing the risk of infection. This problem particularly affects schools without pre-installed ventilation systems that are equipped with filters and/or operate with fresh air. Here, the aerosol concentration is reduced by natural ventilation. In this context, we are investigating the effect of large mobile air purifiers (AP) with HEPA filters on particle concentration and their suitability for classroom use in a primary school in Germany. The three tested APs differ significantly in their air outlet characteristics. Measurements of the number of particles, the particle size distribution, and the CO2 concentration were carried out in the classroom with students (April/May 2021) and with an aerosol generator without students. In this regard, the use of APs leads to a substantial reduction in aerosol particles. At the same time, the three APs are found to have differences in their particle decay rate, noise level, and flow velocity. In addition to the measurements, the effect of various influencing parameters on the potential inhaled particle dose was investigated using a calculation model. The parameters considered include the duration of stay, particle concentration in exhaled air, respiratory flow rate, virus lifetime, ventilation interval, ventilation efficiency, AP volumetric flow, as well as room size. Based on the resulting effect diagrams, significant recommendations can be derived for reducing the risk of infection from virus-laden aerosols. Finally, the measurements were compared to computational fluid dynamics (CFD) modeling, as such tools can aid the optimal placement and configuration of APs and can be used to study the effect of the spread of aerosols from a source in the classroom.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453597', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Generalizing Bayesian phylogenetics to infer shared evolutionary events', 'authors': \"[{'name': 'Jamie R Oaks'}, {'name': 'Perry L Wood'}]\", 'abstract': '<p>Many processes of biological diversification can affect multiple evolutionary lineages. Examples include multiple members of a gene family diverging when a region of a chromosome is duplicated, multiple viral strains diverging at a \"super-spreading\" event, and a geological event fragmenting whole communities of species. It is difficult to test for patterns predicted by such processes, because all phylogenetic methods assume that lineages diverge independently. We introduce a general Bayesian framework to relax the assumption of independent divergences during phylogenetic inference, and test for patterns predicted by processes of diversification that affect multiple evolutionary lineages. Using simulations, we find our new method accurately infers shared divergence events when they occur, and performs as well as current methods when divergnces are independent. We apply our new approach to genomic data from two genera of geckos from across the Philippines to test if past changes to the islands\\' landscape caused bursts of speciation. Unlike previous analyses restricted to only pairs of populations, we find evidence for patterns of shared divergences. By generalizing the space of phylogenetic trees in a way that is independent from the likelihood model, our approach opens many avenues for future research.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453632', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'PTEN mutant NSCLC require ATM to suppress pro-apoptotic signalling and evade radiotherapy', 'authors': \"[{'name': 'Thomas Fischer'}, {'name': 'Oliver Hartmann'}, {'name': 'Michaela Reissland'}, {'name': 'Cristian Prieto-Garcia'}, {'name': 'Kevin Klann'}, {'name': 'Christina Schuelein-Voelk'}, {'name': 'Buelent Polat'}, {'name': 'Elena Gebhard-Hartmann'}, {'name': 'Mathias Rosenfeldt'}, {'name': 'Christian Muench'}, {'name': 'Michael Flentje'}, {'name': 'Markus Elmar Diefenbacher'}]\", 'abstract': '<p>Background: Despite advances in treatment of patients with non-small cell lung cancer, carriers of certain genetic alterations are prone to failure. One such factor frequently mutated, is the tumor suppressor PTEN. These tumors are supposed to be more resistant to radiation, chemo- and immunotherapy. Methods: Using CRISPR genome editing, we deleted PTEN in a human tracheal stem cell-like cell line as well generated primary murine NSCLC, proficient or deficient for Pten, in vivo. These models were used to verify the impact of PTEN loss in vitro and in vivo by immunohistochemical staining, western blot and RNA-Sequencing. Radiation sensitivity was assessed by colony formation and growth assays. To elucidate putative treatment options, identified via the molecular characterisation, PTEN pro- and deficient cells were treated with PI3K/mTOR/DNA-PK-inhibitor PI-103 or the ATM-inhibitors KU-60019 und AZD 1390. Changes in radiation sensitivity were assessed by colony-formation assay, FACS, western-blot, phospho-proteomic mass spectrometry and ex vivo lung slice cultures. Results: We demonstrate that loss of PTEN led to altered expression of transcriptional programs which directly regulate therapy resistance, resulting in establishment of radiation resistance. While PTEN-deficient tumor cells were not dependent on DNA PK for IR resistance nor activated ATR during IR, they showed a significant dependence for the DNA damage kinase ATM. Pharmacologic inhibition of ATM, via KU-60019 and AZD1390 at non-toxic doses, restored and even synergized with IR in PTEN-deficient human and murine NSCLC cells as well in a multicellular organotypic ex vivo tumor model. Conclusion: PTEN tumors are addicted to ATM to detect and repair radiation induced DNA damage. This creates an exploitable bottleneck. At least in cellulo and ex vivo we show that low concentration of ATM inhibitor is able to synergise with IR to treat PTEN-deficient tumors in genetically well-defined IR resistant lung cancer models.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.17.452800', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Causal inference using deep-learning variable selection identifies and incorporates direct and indirect causalities in complex biological systems', 'authors': \"[{'name': 'Zhenjiang Fan'}, {'name': 'Kate F Kernan'}, {'name': 'Panayiotis V Benos'}, {'name': 'Scott W Canna'}, {'name': 'Joseph A Carcillo'}, {'name': 'Soyeon Kim'}, {'name': 'Hyun Jung Park'}]\", 'abstract': '<p>In complex diseases, causal structure learning across biological variables is critical to identify modifiable triggers or potential therapeutic agents. A limitation of existing causal learning methods is that they cannot identify indirect causal relations, those that would interact through latent mediating variables. We developed the first computational method that identifies both direct and indirect causalities, causal inference using deep-learning variable-selection (causalDeepVASE). To accurately identify indirect causalities and incorporate them with direct causalities, causalDeepVASE develops a deep neural network approach and extends a flexible causal inference method. In simulated and biological data of various contexts, causalDeepVASE outperforms existing methods in identifying expected or validated causal relations. Further, causalDeepVASE facilitates a systematic understanding of complex diseases. For example, causalDeepVASE uniquely identified a possible causal relation between IFN and creatinine suggested in a polymicrobial sepsis model. In future biomedical studies, causalDeepVASE can facilitate the identification of driver genes and therapeutic agents.</p>', 'posted': '2021-07-19'}, {'doi': '10.1101/2021.07.22.453339', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Segmenting biological specimens from photos to understand the evolution of UV plumage in passerine birds', 'authors': \"[{'name': 'Yichen He'}, {'name': 'Christopher R Cooney'}, {'name': 'Zoe K Varley'}, {'name': 'Lara O Nouri'}, {'name': 'Christopher J. A. Moody'}, {'name': 'Michael D Jardine'}, {'name': 'Steve Maddock'}, {'name': 'Gavin H Thomas'}]\", 'abstract': '<p>Ultraviolet (UV) colouration is thought to be an important signalling mechanism in many bird species, yet broad insights regarding the prevalence of UV plumage colouration and the factors promoting its evolution are currently lacking. Here, we develop a novel image segmentation pipeline based on deep learning that considerably outperforms classical (i.e. non-deep learning) segmentation methods, and use this to extract accurate information on whole-body plumage colouration from photographs of &gt;24,000 museum specimens covering &gt;4,500 species of passerine birds. Our results demonstrate that UV reflectance, particularly as a component of other colours, is widespread across the passerine radiation but is strongly phylogenetically conserved. We also find clear evidence in support of the role of light environment in promoting the evolution of UV plumage colouration, and a weak trend towards higher UV plumage reflectance among bird species with ultraviolet rather than violet-sensitive visual systems. Overall, our study provides important broad-scale insight into an enigmatic component of avian colouration, as well as demonstrating that deep learning has considerable promise for allowing new data to be bought to bare on long-standing questions in ecology and evolution.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261034', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Orthopedics', 'covid': False, 'title': 'Running related biomechanical risk factors for overuse injuries in distance runners: A systematic review considering injury specificity and the potentials for future research', 'authors': \"[{'name': 'Steffen Willwacher'}, {'name': 'Markus Kurz'}, {'name': 'Johanna Robbin'}, {'name': 'Matthias Thelen'}, {'name': 'Joseph Hamill'}, {'name': 'Luke Kelly'}]\", 'abstract': '<p>Objective\\nTo identify and evaluate the evidence of the most relevant running-related risk factors (RRRFs) for running-related overuse injuries (ROIs) and to suggest future research directions.\\n\\nDesign\\nSystematic review considering prospective and retrospective studies. (PROSPERO_ID: 236832)\\n\\nData sources\\nPubmed. Connected Papers. The search was performed in February 2021. \\n\\nEligibility criteria\\nEnglish language. Studies on participants whose primary sport is running addressing the risk for the seven most common ROIs and at least one kinematic, kinetic (including pressure measurements), or electromyographic RRRF. An RRRF needed to be identified in at least one prospective or two retrospective studies. \\n\\nResults \\nSixty-two articles fulfilled our eligibility criteria. Levels of evidence for specific ROIs ranged from conflicting to moderate evidence. Running populations and methods applied varied considerably between studies. While some RRRFs appeared for several ROIs, most RRRFs were specific for a particular ROI. The biomechanical measurements performed in many studies would have allowed for consideration of many more RRRFs than have been reported, highlighting a potential for more effective data usage in the future.\\n\\nConclusion\\nThis study offers a comprehensive overview of RRRFs for the most common ROIs, which might serve as a starting point to develop ROI-specific risk profiles of individual runners. Future work should use macroscopic (big data) approaches involving long-term data collections in the real world and microscopic approaches involving precise stress calculations using recent developments in biomechanical modelling. However, consensus on data collection standards (including the quantification of workload and stress tolerance variables and the reporting of injuries) is warranted.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.21260946', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Pediatrics', 'covid': False, 'title': \"Association between prenatal alcohol exposure and children's facial shape. A prospective population-based cohort study\", 'authors': \"[{'name': 'Xianjing Liu'}, {'name': 'Manfred Kayser'}, {'name': 'Steven A. Kushner'}, {'name': 'Henning Tiemeier'}, {'name': 'Fernando Rivadeneira'}, {'name': 'Vincent W. V. Jaddoe'}, {'name': 'Wiro Niessen'}, {'name': 'Eppo B. Wolvius'}, {'name': 'Gennady V. Roshchupkin'}]\", 'abstract': '<p>IMPORTANCE: Children exposed to a high level of prenatal alcohol exposure (PAE) are more likely to develop fetal alcohol spectrum disorder with adverse phenotypes on their faces. However, it is still poorly understood, which level of PAE is associated with facial manifestation and if such associations persist during childhood. \\nOBJECTIVE: To examine the association between PAE and children facial phenotype in a prospective multi-ethnic population-based study.\\nDESIGN, SETTING, AND PARTICIPANTS: This study was based on the Generation R Study, a prospective cohort from fetal life onwards with maternal and offspring data. Children had a 3D facial image taken at ages 9 (n=3160) and 13 years (n=2492). \\nEXPOSURES: We defined 6 levels of PAE based on the frequency and dose of alcohol consumption, and defined three tiers based on the timing of alcohol exposure of the unborn child.\\nMAIN OUTCOMES AND MEASURES:  For image analysis, we used 3D graph convolutional networks for non-linear dimensionality reduction, which compressed the high-dimensional images into 200 endophenotypes representing facial morphology. These facial endophenotypes were used as dependent variables in a linear regression analysis to search for associations with PAE. Finally, to detect specific facial components associated with PAE, we mapped statistically significant endophenotypes back to the 3D facial shape. We generated heatmaps to display the facial changes associated with PAE. PAE prediction based on facial shape was also performed; the prediction accuracy was estimated by area under the receiver operating characteristic curve (AUC).\\nRESULTS: A significant association between PAE and facial shape was found in the 9-year-old children, at all levels of alcohol exposure: the higher the level of exposure, the stronger the association. Moreover, PAE before and during pregnancy was associated with facial shape. The most common detected facial phenotypes included turned-up nose tip, shortened nose, turned-out jaw and turned-in lower-eyelid-related regions. For the 13-year-old children, the associations were weaker and the AUCs lower than those of the 9-years-old children.\\nCONCLUSIONS AND RELEVANCE: PAE before and during pregnancy, even at low level, is associated with the facial shape of children, and these associations become weaker as children grow older.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.22.453424', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Mechanisms of phosphatidylserine influence on viral production: a computational model of Ebola virus matrix protein assembly', 'authors': \"[{'name': 'Xiao Liu'}, {'name': 'Ethan J Pappas'}, {'name': 'Monica L Husby'}, {'name': 'Robert V Stahelin'}, {'name': 'Elsje Pienaar'}]\", 'abstract': '<p>Ebola virus (EBOV) infections continue to pose a global public health threat, with high mortality rates and sporadic outbreaks in Central and Western Africa. A quantitative understanding of the key processes driving EBOV assembly and budding could provide valuable insights to inform drug development. Here we used a computational model to evaluate EBOV matrix assembly. Our model focused on the assembly kinetics of VP40, the matrix protein in EBOV, and its interaction with phosphatidylserine (PS) in the host cell membrane. Human cells transfected with VP40-expressing plasmids are capable of producing virus-like particles (VLPs) that closely resemble EBOV virions. We used data from this in vitro VP40 system to calibrate our computational model. PS levels in the host cell membrane had been shown to affect VP40 dynamics as well as VLP production through recruiting VP40 dimers to plasma membrane inner leaflet. Our computational results indicated that PS may have direct influence on VP40 filament growth and affect multiple steps in the assembly and budding of VP40 VLPs. We also proposed that the assembly of VP40 filaments may follow the nucleation-elongation theory where initialization and oligomerization of VP40 are two separate and distinct steps in the assembly process. This work illustrated how computational and experimental approaches can be combined to allow for additional analysis and hypothesis generation. Our findings advanced understanding of the molecular process of EBOV assembly and budding processes and may help the development of new EBOV treatments targeting VP40 matrix assembly.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.23.453379', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'AlphaPept, a modern and open framework for MS-based proteomics', 'authors': \"[{'name': 'Maximilian T Strauss'}, {'name': 'Isabell Bludau'}, {'name': 'Wen-Feng Zeng'}, {'name': 'Eugenia Voytik'}, {'name': 'Constantin Ammar'}, {'name': 'Julia Schessner'}, {'name': 'Rajesh Ilango'}, {'name': 'Michelle Gill'}, {'name': 'Florian Meier'}, {'name': 'Sander Willems'}, {'name': 'Matthias Mann'}]\", 'abstract': '<p>In common with other omics technologies, mass spectrometry (MS)-based proteomics produces ever-increasing amounts of raw data, making their efficient analysis a principal challenge. There is a plethora of different computational tools that process the raw MS data and derive peptide and protein identification and quantification. During the last decade, there has been dramatic progress in computer science and software engineering, including collaboration tools that have transformed research and industry. To leverage these advances, we developed AlphaPept, a Python-based open-source framework for efficient processing of large high-resolution MS data sets. Using Numba for just-in-time machine code compilation on CPU and GPU, we achieve hundred-fold speed improvements while maintaining clear syntax and rapid development speed. AlphaPept uses the Python scientific stack of highly optimized packages, reducing the code base to domain-specific tasks while providing access to the latest advances in machine learning. We provide an easy on-ramp for community validation and contributions through the concept of literate programming, implemented in Jupyter Notebooks of the different modules. A framework for continuous integration, testing, and benchmarking enforces solid software engineering principles. Large datasets can rapidly be processed as shown by the analysis of hundreds of cellular proteomes in minutes per file, many-fold faster than the data acquisiton. The AlphaPept framework can be used to build automated processing pipelines using efficient HDF5 based file formats, web-serving functionality and compatibility with downstream analysis tools. Easy access for end-users is provided by one-click installation of the graphical user interface, for advanced users via a modular Python library, and for developers via a fully open GitHub repository.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453634', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Crowding-induced phase separation of nuclear transport receptors in FG nucleoporin assemblies', 'authors': \"[{'name': 'Luke K Davis'}, {'name': 'Ian J Ford'}, {'name': 'Bart W Hoogenboom'}]\", 'abstract': '<p>The rapid (&lt; 1~ms) transport of biological material to and from the cell nucleus is regulated by the nuclear pore complex (NPC). At the core of the NPC is a permeability barrier consisting of intrinsically disordered Phe-Gly (FG) nucleoporins (FG Nups). Various types of nuclear transport receptors (NTRs) facilitate transport by partitioning in the FG Nup assembly, overcoming the barrier by their affinity to the FG Nups, and comprise a significant fraction of proteins in the NPC barrier. In previous work, we revealed that the experimental binding of the NTRs NTF2 and -- the larger -- Imp to different planar assemblies of FG Nups follows a universal physical law defined by negative cooperativity, which was further validated by a minimal physical model that treated the FG Nups as flexible homopolymers and the NTRs as uniformly cohesive spheres Zahn et al. (2016). Here, we build upon our original study by first parametrizing our model to experimental data, and next to predict the effects of crowding by different types of NTRs. We show how varying the amounts of one type of NTR modulates how the other NTR penetrates the FG Nup assembly. Notably, at similar and physiologically relevant NTR concentrations, our model predicts demixed phases of NTF2 and Imp within the FG Nup assembly. The functional implication of NTR phase separation is that NPCs may sustain separate transport pathways that are determined by inter-NTR competition.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453477', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Generative Adversarial Network augmented the gut microbiome-based health index by profoundly improved discrimination power', 'authors': \"[{'name': 'Yuxue Li'}, {'name': 'Gang Xie'}, {'name': 'Yuguo Zha'}, {'name': 'Kang Ning'}]\", 'abstract': '<sec><title>Summary</title><p>Gut microbiome-based health index (GMHI) has been applied with success, while the discrimination powers of GMHI varied for different diseases, limiting its utility on a broad spectrum of diseases. In this work, a Generative Adversarial Network (GAN) model is proposed to improve the discrimination power of GMHI. Built based on the batch corrected data through GAN (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/HUST-NingKang-Lab/GAN-GMHI\">https://github.com/HUST-NingKang-Lab/GAN-GMHI</ext-link>), GAN-GMHI has largely reduced the batch effects, and profoundly improved the performance for distinguishing healthy individuals and different diseases. GAN-GMHI has provided results to support the strong association of gut microbiome and diseases, and indicated a more accurate venue towards microbiome-based disease monitoring.</p></sec><sec><title>Availability and implementation</title><p>GAN-GMHI is publicly available on GitHub: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/HUST-NingKang-Lab/GAN-GMHI\">https://github.com/HUST-NingKang-Lab/GAN-GMHI</ext-link>.</p></sec><sec><title>Contact</title><p><email>ningkang@hust.edu.cn</email></p></sec><sec><title>Supplementary information</title><p>Supplementary data are available at Bioinformatics online.</p></sec>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.21.21251509', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'Incidence of Lyme disease in the United Kingdom and association with fatigue: a population-based, historical cohort study', 'authors': \"[{'name': 'Florence Brellier'}, {'name': 'Mar Pujades-Rodriguez'}, {'name': 'Emma Powell'}, {'name': 'Kathleen Mudie'}, {'name': 'Eliana Lacerda'}, {'name': 'Luis Nacul'}, {'name': 'Kevin Wing'}]\", 'abstract': '<sec><title>Objectives</title><p>To evaluate incidence rates of Lyme disease in the UK and to investigate a possible association with subsequent fatigue</p></sec><sec><title>Design</title><p>Population-based historical cohort study with a comparator cohort matched by age, sex, and general practice</p></sec><sec><title>Setting</title><p>Patients treated in UK general practices contributing to IQVIA Medical Research Data</p></sec><sec><title>Participants</title><p>2,130 patients with a first diagnosis of Lyme disease between 2000 and 2018, and 8,510 randomly-sampled matched comparators, followed-up for a median time of 3 years and 8 months.</p></sec><sec><title>Main outcome measures</title><p>Time from Lyme disease diagnosis to consultation for any fatigue-related symptoms or diagnosis and for myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS). Adjusted hazard ratios (HRs) were estimated from Cox models.</p></sec><sec><title>Results</title><p>Average incidence rate for Lyme disease across the UK was 5.18 per 100,000 py between 2000 and 2018, increasing from 2.55 in 2000 to 9.33 in 2018. In total 929 events of any types of fatigue were observed, i.e. an incidence rate of 307.90 per 10,000 py in the Lyme cohort (282 events) and 165.60 in the comparator cohort (647 events). Effect of Lyme disease on any subsequent fatigue varied by index season with highest adjusted HRs in autumn [3.14 (95%CI: 1.92 to 5.13)] and winter [2.23 (1.21 to 4.11)]. Incidence rates of ME/CFS were 11.16 per 10,000 py in Lyme patients (12 events) and 1.20 in comparators (5 events), corresponding to an adjusted HR of 16.95 (5.17 to 55.60). Effect on any types of fatigue and ME/CFS was attenuated 6 months after diagnosis but still clearly visible.</p></sec><sec><title>Conclusions</title><p>UK primary care records provided strong evidence that Lyme disease was associated with acute and chronic fatigue. Albeit weaker, these effects persisted beyond 6 months, suggesting that patients and healthcare providers should remain alert to fatigue symptoms months to years following Lyme disease diagnosis.</p></sec><sec><title>Key messages box</title><sec><title>What is already known on this topic</title><p>Incidence rates of Lyme disease in the UK are increasing but estimations vary according to data sources used.</p><p>Reports investigating the association between Lyme disease and long-term fatigue are contradictory.</p></sec><sec><title>What this study adds</title><p>Average incidence rate for Lyme disease across the UK was estimated at 5.18 per 100,000 py between 2000 and 2018, and followed an increasing trend.</p><p>In patients with Lyme disease, a 2- and 3-fold increase in any subsequent fatigue was observed in winter and autumn, respectively, and a 16-fold increase in ME/CFS (all seasons combined), compared to a non-Lyme cohort matched by sex, age, and general practice.</p></sec></sec>', 'posted': '2021-07-24'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.07.19.21260767</td>\n",
       "      <td>542</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Pediatrics</td>\n",
       "      <td>True</td>\n",
       "      <td>Children with SARS-CoV-2 in the National COVID...</td>\n",
       "      <td>[{'name': 'Blake Martin'}, {'name': 'Peter E. ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Importance&lt;/title&gt;&lt;p&gt;SARS-CoV-2&lt;/p...</td>\n",
       "      <td>2021-07-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.07.15.21260561</td>\n",
       "      <td>405</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Viral Load of SARS-CoV-2 in Respiratory Aeroso...</td>\n",
       "      <td>[{'name': 'Kristen K. Coleman'}, {'name': 'Dou...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Multiple SARS...</td>\n",
       "      <td>2021-07-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.07.21.451321</td>\n",
       "      <td>219</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Unlike Chloroquine, mefloquine inhibits SARS-C...</td>\n",
       "      <td>[{'name': 'Carolina Q. Sacramento'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Repositioning of clinical approved drugs co...</td>\n",
       "      <td>2021-07-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.07.19.452771</td>\n",
       "      <td>50</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Comparison of Neutralizing Antibody Titers Eli...</td>\n",
       "      <td>[{'name': 'Takuya Tada'}, {'name': 'Hao Zhou'}...</td>\n",
       "      <td>&lt;p&gt;The increasing prevalence of SARS-CoV-2 var...</td>\n",
       "      <td>2021-07-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.07.23.453070</td>\n",
       "      <td>49</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Regulation of sedimentation rate shapes the ev...</td>\n",
       "      <td>[{'name': 'Omaya Dudin'}, {'name': 'Sbastien ...</td>\n",
       "      <td>&lt;p&gt;Significant increases in sedimentation rate...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.07.23.453521</td>\n",
       "      <td>35</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Probiotic consortia improve anti-viral immunit...</td>\n",
       "      <td>[{'name': 'J Lehtinen Markus'}, {'name': 'Kuma...</td>\n",
       "      <td>&lt;p&gt;Probiotics have been suggested as one solut...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.07.23.453573</td>\n",
       "      <td>34</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Strong and opponent contributions of dorsomedi...</td>\n",
       "      <td>[{'name': 'Scott Bolkan'}, {'name': 'Iris Ston...</td>\n",
       "      <td>&lt;p&gt;A classic view of the striatum holds that a...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.07.19.453013</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Phylogenomic analyses of echinoid diversificat...</td>\n",
       "      <td>[{'name': 'Nicolas Mongiardino Koch'}, {'name'...</td>\n",
       "      <td>&lt;p&gt;Echinoids are key components of modern mari...</td>\n",
       "      <td>2021-07-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.07.19.452969</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>A Bayesian model selection approach to mediati...</td>\n",
       "      <td>[{'name': 'Wesley L Crouse'}, {'name': 'Gregor...</td>\n",
       "      <td>&lt;p&gt;Mediation analysis is a powerful tool for d...</td>\n",
       "      <td>2021-07-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.07.22.21260852</td>\n",
       "      <td>19</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Transplantation</td>\n",
       "      <td>True</td>\n",
       "      <td>Comparison of infected and vaccinated transpla...</td>\n",
       "      <td>[{'name': 'Xavier Charmetant'}, {'name': 'Maxi...</td>\n",
       "      <td>&lt;p&gt;Transplant recipients, which receive therap...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.07.20.453126</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Differences in the dynamics of the tandem-SH2 ...</td>\n",
       "      <td>[{'name': 'Helen T Hobbs'}, {'name': 'Neel H S...</td>\n",
       "      <td>&lt;p&gt;The catalytic activity of Syk-family tyrosi...</td>\n",
       "      <td>2021-07-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.07.21.453258</td>\n",
       "      <td>13</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Retrosplenial cortex is necessary for spatial ...</td>\n",
       "      <td>[{'name': 'Ana Carolina Bottura de Barros'}, {...</td>\n",
       "      <td>&lt;p&gt;Latent learning occurs when associations ar...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.07.22.21260551</td>\n",
       "      <td>12</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Willingness to receive vaccination against COV...</td>\n",
       "      <td>[{'name': 'Sharon R Davis'}, {'name': 'Rosario...</td>\n",
       "      <td>&lt;p&gt;We surveyed 10,024 Australians regarding CO...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.07.18.21259977</td>\n",
       "      <td>12</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>False</td>\n",
       "      <td>Altered functional connectome hierarchy with g...</td>\n",
       "      <td>[{'name': 'Christophe E de Bzenac'}, {'name':...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Objective&lt;/title&gt;&lt;p&gt;Neuroimaging r...</td>\n",
       "      <td>2021-07-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.07.24.453184</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>The dual nature of bacteriophage: growth-depen...</td>\n",
       "      <td>[{'name': 'Quentin J Leclerc'}, {'name': 'Jaco...</td>\n",
       "      <td>&lt;p&gt;Bacteriophage (\"phage\") are both predators ...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.07.22.21258785</td>\n",
       "      <td>9</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Gastroenterology</td>\n",
       "      <td>True</td>\n",
       "      <td>Proton pump inhibitor use is not associated wi...</td>\n",
       "      <td>[{'name': 'Shailja C. Shah'}, {'name': 'Alese ...</td>\n",
       "      <td>&lt;p&gt;Background and Aims: Low pH deactivates mos...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.07.23.453610</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Adapted protocol for Saccharibacteria co-culti...</td>\n",
       "      <td>[{'name': 'Ahmad Ibrahim'}, {'name': 'Mohamad ...</td>\n",
       "      <td>&lt;p&gt;The growing application of metagenomics to ...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.07.23.453553</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Community health and human-animal contacts on ...</td>\n",
       "      <td>[{'name': 'Renata L. Muylaert'}, {'name': 'Ben...</td>\n",
       "      <td>&lt;p&gt;Cross-species transmission of pathogens is ...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.07.23.453512</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>In vitro and in vivo analysis of microvesicle-...</td>\n",
       "      <td>[{'name': 'Ahmed A Zarea'}, {'name': 'Gloria I...</td>\n",
       "      <td>&lt;p&gt;Cancer cells produce heterogeneous extracel...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.07.22.453456</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Fast transcriptional activation of development...</td>\n",
       "      <td>[{'name': 'Cneyt Caglar'}, {'name': 'Alexande...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The ability t...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.07.19.453008</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Don't think, feel: basal ganglia-cortical conn...</td>\n",
       "      <td>[{'name': 'Kazumi Kasahara'}, {'name': 'Charle...</td>\n",
       "      <td>&lt;p&gt;Brain-computer interfaces (BCIs) provide an...</td>\n",
       "      <td>2021-07-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.07.23.453341</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Trait-trait relationships and functional trade...</td>\n",
       "      <td>[{'name': 'Sara Beier'}, {'name': 'Johannes We...</td>\n",
       "      <td>&lt;p&gt;We report genomic traits that have been ass...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.07.23.453533</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Animal Behavior and Cognition</td>\n",
       "      <td>False</td>\n",
       "      <td>Decoding the Neural Signatures of Valence and ...</td>\n",
       "      <td>[{'name': 'Nikhil Garg'}, {'name': 'Rohit Garg...</td>\n",
       "      <td>&lt;p&gt;This paper focuses on classifying emotions ...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.07.23.453507</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Toxin breakdown does not preclude the potentia...</td>\n",
       "      <td>[{'name': 'Tyler E. Douglas'}, {'name': 'Sofia...</td>\n",
       "      <td>&lt;p&gt;Animals that ingest toxins can themselves b...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.07.22.453458</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Epithelial-mesenchymal plasticity determines e...</td>\n",
       "      <td>[{'name': 'Aouad Patrik'}, {'name': 'Zhang Yue...</td>\n",
       "      <td>&lt;p&gt;Estrogen receptor -positive (ER+) breast c...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.07.22.453319</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Pectin remodeling belongs to a homeostatic sys...</td>\n",
       "      <td>[{'name': 'Franois Jobert'}, {'name': 'Stpha...</td>\n",
       "      <td>&lt;p&gt;Pectins occur in primary cell walls and con...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.07.23.453605</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Identifying cell-state associated alternative ...</td>\n",
       "      <td>[{'name': 'Carlos F Buen Abad Najar'}, {'name'...</td>\n",
       "      <td>&lt;p&gt;Alternative splicing shapes the transcripto...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.07.23.453525</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Zoology</td>\n",
       "      <td>False</td>\n",
       "      <td>Insights on the historical biogeography of Phi...</td>\n",
       "      <td>[{'name': 'John King N. Layos'}, {'name': 'Ron...</td>\n",
       "      <td>&lt;p&gt;The Philippine archipelago was believed to ...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.07.23.453515</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Low dose/dose rate  irradiation protects Dros...</td>\n",
       "      <td>[{'name': 'Antonella Porrazzo'}, {'name': 'Fra...</td>\n",
       "      <td>&lt;p&gt;It is still continuously debated whether th...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.07.23.21261042</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Robust antibody levels in both diabetic and no...</td>\n",
       "      <td>[{'name': 'Hamad Ali'}, {'name': 'Abdulmohsen ...</td>\n",
       "      <td>&lt;p&gt;The emergence of effective vaccines for COV...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.07.23.21261041</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>The impact of large mobile air purifiers on ae...</td>\n",
       "      <td>[{'name': 'Finn F. Duill'}, {'name': 'Florian ...</td>\n",
       "      <td>&lt;p&gt;In the wake of the SARS-CoV-2 pandemic, an ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.07.23.453597</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Generalizing Bayesian phylogenetics to infer s...</td>\n",
       "      <td>[{'name': 'Jamie R Oaks'}, {'name': 'Perry L W...</td>\n",
       "      <td>&lt;p&gt;Many processes of biological diversificatio...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.07.24.453632</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>PTEN mutant NSCLC require ATM to suppress pro-...</td>\n",
       "      <td>[{'name': 'Thomas Fischer'}, {'name': 'Oliver ...</td>\n",
       "      <td>&lt;p&gt;Background: Despite advances in treatment o...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.07.17.452800</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Causal inference using deep-learning variable ...</td>\n",
       "      <td>[{'name': 'Zhenjiang Fan'}, {'name': 'Kate F K...</td>\n",
       "      <td>&lt;p&gt;In complex diseases, causal structure learn...</td>\n",
       "      <td>2021-07-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.07.22.453339</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Segmenting biological specimens from photos to...</td>\n",
       "      <td>[{'name': 'Yichen He'}, {'name': 'Christopher ...</td>\n",
       "      <td>&lt;p&gt;Ultraviolet (UV) colouration is thought to ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.07.23.21261034</td>\n",
       "      <td>3</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Orthopedics</td>\n",
       "      <td>False</td>\n",
       "      <td>Running related biomechanical risk factors for...</td>\n",
       "      <td>[{'name': 'Steffen Willwacher'}, {'name': 'Mar...</td>\n",
       "      <td>&lt;p&gt;Objective\\nTo identify and evaluate the evi...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.07.22.21260946</td>\n",
       "      <td>3</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Pediatrics</td>\n",
       "      <td>False</td>\n",
       "      <td>Association between prenatal alcohol exposure ...</td>\n",
       "      <td>[{'name': 'Xianjing Liu'}, {'name': 'Manfred K...</td>\n",
       "      <td>&lt;p&gt;IMPORTANCE: Children exposed to a high leve...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.07.22.453424</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Mechanisms of phosphatidylserine influence on ...</td>\n",
       "      <td>[{'name': 'Xiao Liu'}, {'name': 'Ethan J Pappa...</td>\n",
       "      <td>&lt;p&gt;Ebola virus (EBOV) infections continue to p...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.07.23.453379</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>AlphaPept, a modern and open framework for MS-...</td>\n",
       "      <td>[{'name': 'Maximilian T Strauss'}, {'name': 'I...</td>\n",
       "      <td>&lt;p&gt;In common with other omics technologies, ma...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.07.24.453634</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Crowding-induced phase separation of nuclear t...</td>\n",
       "      <td>[{'name': 'Luke K Davis'}, {'name': 'Ian J For...</td>\n",
       "      <td>&lt;p&gt;The rapid (&amp;lt; 1~ms) transport of biologic...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.07.23.453477</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Generative Adversarial Network augmented the g...</td>\n",
       "      <td>[{'name': 'Yuxue Li'}, {'name': 'Gang Xie'}, {...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Summary&lt;/title&gt;&lt;p&gt;Gut microbiome-b...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.07.21.21251509</td>\n",
       "      <td>3</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>False</td>\n",
       "      <td>Incidence of Lyme disease in the United Kingdo...</td>\n",
       "      <td>[{'name': 'Florence Brellier'}, {'name': 'Mar ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Objectives&lt;/title&gt;&lt;p&gt;To evaluate i...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2021.07.19.21260767     542  medRxiv   \n",
       "1   10.1101/2021.07.15.21260561     405  medRxiv   \n",
       "2     10.1101/2021.07.21.451321     219  bioRxiv   \n",
       "3     10.1101/2021.07.19.452771      50  bioRxiv   \n",
       "4     10.1101/2021.07.23.453070      49  bioRxiv   \n",
       "5     10.1101/2021.07.23.453521      35  bioRxiv   \n",
       "6     10.1101/2021.07.23.453573      34  bioRxiv   \n",
       "7     10.1101/2021.07.19.453013      27  bioRxiv   \n",
       "8     10.1101/2021.07.19.452969      20  bioRxiv   \n",
       "9   10.1101/2021.07.22.21260852      19  medRxiv   \n",
       "10    10.1101/2021.07.20.453126      16  bioRxiv   \n",
       "11    10.1101/2021.07.21.453258      13  bioRxiv   \n",
       "12  10.1101/2021.07.22.21260551      12  medRxiv   \n",
       "13  10.1101/2021.07.18.21259977      12  medRxiv   \n",
       "14    10.1101/2021.07.24.453184      11  bioRxiv   \n",
       "15  10.1101/2021.07.22.21258785       9  medRxiv   \n",
       "16    10.1101/2021.07.23.453610       9  bioRxiv   \n",
       "17    10.1101/2021.07.23.453553       8  bioRxiv   \n",
       "18    10.1101/2021.07.23.453512       8  bioRxiv   \n",
       "19    10.1101/2021.07.22.453456       7  bioRxiv   \n",
       "20    10.1101/2021.07.19.453008       7  bioRxiv   \n",
       "21    10.1101/2021.07.23.453341       7  bioRxiv   \n",
       "22    10.1101/2021.07.23.453533       6  bioRxiv   \n",
       "23    10.1101/2021.07.23.453507       6  bioRxiv   \n",
       "24    10.1101/2021.07.22.453458       6  bioRxiv   \n",
       "25    10.1101/2021.07.22.453319       6  bioRxiv   \n",
       "26    10.1101/2021.07.23.453605       5  bioRxiv   \n",
       "27    10.1101/2021.07.23.453525       5  bioRxiv   \n",
       "28    10.1101/2021.07.23.453515       5  bioRxiv   \n",
       "29  10.1101/2021.07.23.21261042       4  medRxiv   \n",
       "30  10.1101/2021.07.23.21261041       4  medRxiv   \n",
       "31    10.1101/2021.07.23.453597       4  bioRxiv   \n",
       "32    10.1101/2021.07.24.453632       4  bioRxiv   \n",
       "33    10.1101/2021.07.17.452800       4  bioRxiv   \n",
       "34    10.1101/2021.07.22.453339       3  bioRxiv   \n",
       "35  10.1101/2021.07.23.21261034       3  medRxiv   \n",
       "36  10.1101/2021.07.22.21260946       3  medRxiv   \n",
       "37    10.1101/2021.07.22.453424       3  bioRxiv   \n",
       "38    10.1101/2021.07.23.453379       3  bioRxiv   \n",
       "39    10.1101/2021.07.24.453634       3  bioRxiv   \n",
       "40    10.1101/2021.07.23.453477       3  bioRxiv   \n",
       "41  10.1101/2021.07.21.21251509       3  medRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0                              Pediatrics   True   \n",
       "1   Infectious Diseases (except HIV/AIDS)   True   \n",
       "2                            Microbiology   True   \n",
       "3                              Immunology   True   \n",
       "4                    Evolutionary Biology  False   \n",
       "5                              Immunology   True   \n",
       "6                            Neuroscience  False   \n",
       "7                    Evolutionary Biology  False   \n",
       "8                                Genetics  False   \n",
       "9                         Transplantation   True   \n",
       "10                           Biochemistry  False   \n",
       "11                           Neuroscience  False   \n",
       "12  Infectious Diseases (except HIV/AIDS)   True   \n",
       "13                              Neurology  False   \n",
       "14                           Microbiology  False   \n",
       "15                       Gastroenterology   True   \n",
       "16                           Microbiology  False   \n",
       "17                                Ecology  False   \n",
       "18                         Cancer Biology  False   \n",
       "19                  Developmental Biology  False   \n",
       "20                           Neuroscience  False   \n",
       "21                                Ecology  False   \n",
       "22          Animal Behavior and Cognition  False   \n",
       "23                   Evolutionary Biology  False   \n",
       "24                         Cancer Biology  False   \n",
       "25                          Plant Biology  False   \n",
       "26                               Genomics  False   \n",
       "27                                Zoology  False   \n",
       "28                               Genetics  False   \n",
       "29  Infectious Diseases (except HIV/AIDS)   True   \n",
       "30               Public and Global Health   True   \n",
       "31                   Evolutionary Biology  False   \n",
       "32                         Cancer Biology  False   \n",
       "33                         Bioinformatics  False   \n",
       "34                   Evolutionary Biology  False   \n",
       "35                            Orthopedics  False   \n",
       "36                             Pediatrics  False   \n",
       "37                        Systems Biology  False   \n",
       "38                         Bioinformatics  False   \n",
       "39                             Biophysics  False   \n",
       "40                         Bioinformatics  False   \n",
       "41  Infectious Diseases (except HIV/AIDS)  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Children with SARS-CoV-2 in the National COVID...   \n",
       "1   Viral Load of SARS-CoV-2 in Respiratory Aeroso...   \n",
       "2   Unlike Chloroquine, mefloquine inhibits SARS-C...   \n",
       "3   Comparison of Neutralizing Antibody Titers Eli...   \n",
       "4   Regulation of sedimentation rate shapes the ev...   \n",
       "5   Probiotic consortia improve anti-viral immunit...   \n",
       "6   Strong and opponent contributions of dorsomedi...   \n",
       "7   Phylogenomic analyses of echinoid diversificat...   \n",
       "8   A Bayesian model selection approach to mediati...   \n",
       "9   Comparison of infected and vaccinated transpla...   \n",
       "10  Differences in the dynamics of the tandem-SH2 ...   \n",
       "11  Retrosplenial cortex is necessary for spatial ...   \n",
       "12  Willingness to receive vaccination against COV...   \n",
       "13  Altered functional connectome hierarchy with g...   \n",
       "14  The dual nature of bacteriophage: growth-depen...   \n",
       "15  Proton pump inhibitor use is not associated wi...   \n",
       "16  Adapted protocol for Saccharibacteria co-culti...   \n",
       "17  Community health and human-animal contacts on ...   \n",
       "18  In vitro and in vivo analysis of microvesicle-...   \n",
       "19  Fast transcriptional activation of development...   \n",
       "20  Don't think, feel: basal ganglia-cortical conn...   \n",
       "21  Trait-trait relationships and functional trade...   \n",
       "22  Decoding the Neural Signatures of Valence and ...   \n",
       "23  Toxin breakdown does not preclude the potentia...   \n",
       "24  Epithelial-mesenchymal plasticity determines e...   \n",
       "25  Pectin remodeling belongs to a homeostatic sys...   \n",
       "26  Identifying cell-state associated alternative ...   \n",
       "27  Insights on the historical biogeography of Phi...   \n",
       "28  Low dose/dose rate  irradiation protects Dros...   \n",
       "29  Robust antibody levels in both diabetic and no...   \n",
       "30  The impact of large mobile air purifiers on ae...   \n",
       "31  Generalizing Bayesian phylogenetics to infer s...   \n",
       "32  PTEN mutant NSCLC require ATM to suppress pro-...   \n",
       "33  Causal inference using deep-learning variable ...   \n",
       "34  Segmenting biological specimens from photos to...   \n",
       "35  Running related biomechanical risk factors for...   \n",
       "36  Association between prenatal alcohol exposure ...   \n",
       "37  Mechanisms of phosphatidylserine influence on ...   \n",
       "38  AlphaPept, a modern and open framework for MS-...   \n",
       "39  Crowding-induced phase separation of nuclear t...   \n",
       "40  Generative Adversarial Network augmented the g...   \n",
       "41  Incidence of Lyme disease in the United Kingdo...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Blake Martin'}, {'name': 'Peter E. ...   \n",
       "1   [{'name': 'Kristen K. Coleman'}, {'name': 'Dou...   \n",
       "2   [{'name': 'Carolina Q. Sacramento'}, {'name': ...   \n",
       "3   [{'name': 'Takuya Tada'}, {'name': 'Hao Zhou'}...   \n",
       "4   [{'name': 'Omaya Dudin'}, {'name': 'Sbastien ...   \n",
       "5   [{'name': 'J Lehtinen Markus'}, {'name': 'Kuma...   \n",
       "6   [{'name': 'Scott Bolkan'}, {'name': 'Iris Ston...   \n",
       "7   [{'name': 'Nicolas Mongiardino Koch'}, {'name'...   \n",
       "8   [{'name': 'Wesley L Crouse'}, {'name': 'Gregor...   \n",
       "9   [{'name': 'Xavier Charmetant'}, {'name': 'Maxi...   \n",
       "10  [{'name': 'Helen T Hobbs'}, {'name': 'Neel H S...   \n",
       "11  [{'name': 'Ana Carolina Bottura de Barros'}, {...   \n",
       "12  [{'name': 'Sharon R Davis'}, {'name': 'Rosario...   \n",
       "13  [{'name': 'Christophe E de Bzenac'}, {'name':...   \n",
       "14  [{'name': 'Quentin J Leclerc'}, {'name': 'Jaco...   \n",
       "15  [{'name': 'Shailja C. Shah'}, {'name': 'Alese ...   \n",
       "16  [{'name': 'Ahmad Ibrahim'}, {'name': 'Mohamad ...   \n",
       "17  [{'name': 'Renata L. Muylaert'}, {'name': 'Ben...   \n",
       "18  [{'name': 'Ahmed A Zarea'}, {'name': 'Gloria I...   \n",
       "19  [{'name': 'Cneyt Caglar'}, {'name': 'Alexande...   \n",
       "20  [{'name': 'Kazumi Kasahara'}, {'name': 'Charle...   \n",
       "21  [{'name': 'Sara Beier'}, {'name': 'Johannes We...   \n",
       "22  [{'name': 'Nikhil Garg'}, {'name': 'Rohit Garg...   \n",
       "23  [{'name': 'Tyler E. Douglas'}, {'name': 'Sofia...   \n",
       "24  [{'name': 'Aouad Patrik'}, {'name': 'Zhang Yue...   \n",
       "25  [{'name': 'Franois Jobert'}, {'name': 'Stpha...   \n",
       "26  [{'name': 'Carlos F Buen Abad Najar'}, {'name'...   \n",
       "27  [{'name': 'John King N. Layos'}, {'name': 'Ron...   \n",
       "28  [{'name': 'Antonella Porrazzo'}, {'name': 'Fra...   \n",
       "29  [{'name': 'Hamad Ali'}, {'name': 'Abdulmohsen ...   \n",
       "30  [{'name': 'Finn F. Duill'}, {'name': 'Florian ...   \n",
       "31  [{'name': 'Jamie R Oaks'}, {'name': 'Perry L W...   \n",
       "32  [{'name': 'Thomas Fischer'}, {'name': 'Oliver ...   \n",
       "33  [{'name': 'Zhenjiang Fan'}, {'name': 'Kate F K...   \n",
       "34  [{'name': 'Yichen He'}, {'name': 'Christopher ...   \n",
       "35  [{'name': 'Steffen Willwacher'}, {'name': 'Mar...   \n",
       "36  [{'name': 'Xianjing Liu'}, {'name': 'Manfred K...   \n",
       "37  [{'name': 'Xiao Liu'}, {'name': 'Ethan J Pappa...   \n",
       "38  [{'name': 'Maximilian T Strauss'}, {'name': 'I...   \n",
       "39  [{'name': 'Luke K Davis'}, {'name': 'Ian J For...   \n",
       "40  [{'name': 'Yuxue Li'}, {'name': 'Gang Xie'}, {...   \n",
       "41  [{'name': 'Florence Brellier'}, {'name': 'Mar ...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <sec><title>Importance</title><p>SARS-CoV-2</p...  2021-07-22  \n",
       "1   <sec><title>Background</title><p>Multiple SARS...  2021-07-19  \n",
       "2   <p>Repositioning of clinical approved drugs co...  2021-07-21  \n",
       "3   <p>The increasing prevalence of SARS-CoV-2 var...  2021-07-19  \n",
       "4   <p>Significant increases in sedimentation rate...  2021-07-23  \n",
       "5   <p>Probiotics have been suggested as one solut...  2021-07-23  \n",
       "6   <p>A classic view of the striatum holds that a...  2021-07-25  \n",
       "7   <p>Echinoids are key components of modern mari...  2021-07-20  \n",
       "8   <p>Mediation analysis is a powerful tool for d...  2021-07-20  \n",
       "9   <p>Transplant recipients, which receive therap...  2021-07-24  \n",
       "10  <p>The catalytic activity of Syk-family tyrosi...  2021-07-20  \n",
       "11  <p>Latent learning occurs when associations ar...  2021-07-23  \n",
       "12  <p>We surveyed 10,024 Australians regarding CO...  2021-07-25  \n",
       "13  <sec><title>Objective</title><p>Neuroimaging r...  2021-07-22  \n",
       "14  <p>Bacteriophage (\"phage\") are both predators ...  2021-07-24  \n",
       "15  <p>Background and Aims: Low pH deactivates mos...  2021-07-25  \n",
       "16  <p>The growing application of metagenomics to ...  2021-07-25  \n",
       "17  <p>Cross-species transmission of pathogens is ...  2021-07-23  \n",
       "18  <p>Cancer cells produce heterogeneous extracel...  2021-07-25  \n",
       "19  <sec><title>Background</title><p>The ability t...  2021-07-23  \n",
       "20  <p>Brain-computer interfaces (BCIs) provide an...  2021-07-20  \n",
       "21  <p>We report genomic traits that have been ass...  2021-07-24  \n",
       "22  <p>This paper focuses on classifying emotions ...  2021-07-23  \n",
       "23  <p>Animals that ingest toxins can themselves b...  2021-07-24  \n",
       "24  <p>Estrogen receptor -positive (ER+) breast c...  2021-07-26  \n",
       "25  <p>Pectins occur in primary cell walls and con...  2021-07-23  \n",
       "26  <p>Alternative splicing shapes the transcripto...  2021-07-24  \n",
       "27  <p>The Philippine archipelago was believed to ...  2021-07-23  \n",
       "28  <p>It is still continuously debated whether th...  2021-07-25  \n",
       "29  <p>The emergence of effective vaccines for COV...  2021-07-26  \n",
       "30  <p>In the wake of the SARS-CoV-2 pandemic, an ...  2021-07-26  \n",
       "31  <p>Many processes of biological diversificatio...  2021-07-26  \n",
       "32  <p>Background: Despite advances in treatment o...  2021-07-25  \n",
       "33  <p>In complex diseases, causal structure learn...  2021-07-19  \n",
       "34  <p>Ultraviolet (UV) colouration is thought to ...  2021-07-26  \n",
       "35  <p>Objective\\nTo identify and evaluate the evi...  2021-07-26  \n",
       "36  <p>IMPORTANCE: Children exposed to a high leve...  2021-07-25  \n",
       "37  <p>Ebola virus (EBOV) infections continue to p...  2021-07-25  \n",
       "38  <p>In common with other omics technologies, ma...  2021-07-26  \n",
       "39  <p>The rapid (&lt; 1~ms) transport of biologic...  2021-07-26  \n",
       "40  <sec><title>Summary</title><p>Gut microbiome-b...  2021-07-23  \n",
       "41  <sec><title>Objectives</title><p>To evaluate i...  2021-07-24  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "42 preprints (including 10 covering SARS-CoV-2, 31 from bioRxiv and 11 from medRxiv) published in the last 7 days before 2021-07-26 had been tweeted at least 3 times (maximum 542)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.5"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
