{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "26130 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f19c203a-cd32-4b9d-b461-de9708549e4e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=1fd9b4c5-bc42-4db1-81fd-46de0a1a481d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=41b02dc6-6053-4172-8a54-068b1bfa3228&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=8720100a-4594-426d-852a-afcd619c4e8e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b0de68f0-24eb-4cdc-a3ca-4bd1e30d2662&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=fd06d878-416b-4441-acdd-f84ca1bcabd7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=3b79d3ad-f5a2-434b-91da-e8defe377699&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d2781540-dad0-4908-8b62-3926eb4f91f9&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=98c37e33-a545-4b19-a773-f445b369905f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=31bed33d-6f75-4c2b-9c2a-d4efb28bf9a6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4f9a855f-4856-4d3f-a5bc-f10a08a0ef8d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b7bd60e8-1234-4c4b-83a5-734932caf72b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f3bef284-96c8-4cba-a133-77e837f7d287&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e15ba5e4-47f9-47fa-b51f-faee28394982&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c817b0f4-2796-4c90-a39a-3a2ed4680f57&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2881f3cc-bff3-4949-a6fd-ad5097f796fc&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=797ec83f-db7d-4be8-8a45-e40d778ad0c0&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=50477cd3-7b7f-40cb-b9fd-82e231aecca7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0018.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=6928cc58-c14c-4d30-b91b-c2dc8de933c9&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0019.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=3309d3b1-a2d7-42e4-ae55-e8c06c800f7f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0020.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=144323fb-4871-4b23-9792-9c7df8b2414c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0021.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=769beeae-bbbe-4bdd-a090-883bf7682d25&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0022.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=646bae0f-d51d-47f9-804a-d4f484f4bcae&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0023.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=54cd7018-5e32-4202-8fb7-d093b06aaebe&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0024.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b6017e96-9cbc-4a5b-b3d7-98ac8afd417b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0025.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b98b732e-35a6-4911-aef3-1160eabd2419&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-16&until-occurred-date=2021-08-23\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0026.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.20172288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21261912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.20231399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.434841...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.21255036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.444042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455263...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.03.075069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.25.397729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.455988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.11.246801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.20.20178772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/135582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456813...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.31.928630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.10.377481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.438781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.439272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/571380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456185...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.371419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.442102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.06.412304...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.13.039917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.261586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.20209114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.447124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.30.125237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.20.423657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.03.429226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21254000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21260074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.10.20097543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456438...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/571083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.20.21254022...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.434040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.380741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.455609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.21261093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.435000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.213132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.06.413591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.21251670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.439638...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456127...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258188...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20118877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.332023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.452439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275368.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.437175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.02.021519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.15.296228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275607.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456221...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261934...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.20248932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.428136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20159707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/616565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.28.316786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.297366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.21253051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.434227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.426782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/428318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.27.20018929...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.362236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.21254965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.29.441881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.21257017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.05.20088757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.20248134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456039...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262170...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.02.424917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/715482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.20248159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457043...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20216630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.441323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.22.21250282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.383083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.434223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.27.175067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.14.21252289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.21249998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257344...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.21250420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.21253331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.21250916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.09.893560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/596718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.30.437506...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.425825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.14.296814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.437224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.438475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.05.284604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.29.21256294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.093195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.27.20161976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.214148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.14.20153981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.441307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20075838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20234369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.434300...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.20245993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.21259544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.06.028712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.455687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.448554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21260038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.21258886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257738...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.431212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.441313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.442666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.21256243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21254060...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434529...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.451001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.03.438300...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.443516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.21250494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.426720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.18.159780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21260542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20223891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.415687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.06.413542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21262006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.22.20199802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261928...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455939...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/292581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457033...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.30.069922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/847160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/750497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.15.20065995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/799270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.10.036418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452435...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.03.20029983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.29.21255961...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21255013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.444019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.453827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.09.290502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451547...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.28.316265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.20.884908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20209742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/124263...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.350116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.27.20220905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.433940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.14.150490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20065920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.456516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.440862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.21257879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.04.20122481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/037762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.29.20117184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.25.111690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.11.20061473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.444275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.06.20147199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.194282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456413...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.23.20039446...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.16.993683...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260153...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.425021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.21258689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.431313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.19.21251232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.424510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.25.398511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21250921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.21.20236018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261738...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21249554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.24.428004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.451021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.453013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20224659...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.20227215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.436173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.455838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.21259133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449264...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260072...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.08.19.456982', 'tweets': 812, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Somatic mutation rates scale with lifespan across mammals', 'authors': \"[{'name': 'Alex Cagan'}, {'name': 'Adrian Baez-Ortega'}, {'name': 'Natalia Brzozowska'}, {'name': 'Federico Abascal'}, {'name': 'Tim H. H. Coorens'}, {'name': 'Mathijs A. Sanders'}, {'name': 'Andrew R. J. Lawson'}, {'name': 'Luke M. R. Harvey'}, {'name': 'Shriram G. Bhosle'}, {'name': 'David Jones'}, {'name': 'Raul E. Alcantara'}, {'name': 'Timothy M. Butler'}, {'name': 'Yvette Hooks'}, {'name': 'Kirsty Roberts'}, {'name': 'Elizabeth Anderson'}, {'name': 'Edmund Flach'}, {'name': 'Simon Spiro'}, {'name': 'Inez Januszczak'}, {'name': 'Ethan Wrigglesworth'}, {'name': 'Matthew W. Perkins'}, {'name': 'Robert Deaville'}, {'name': 'Megan Druce'}, {'name': 'Ruzhica Bogeska'}, {'name': 'Michael D. Milsom'}, {'name': 'Bjrn Neumann'}, {'name': 'Frank Gorman'}, {'name': 'Fernando Constantino-Casas'}, {'name': 'Laura Peachey'}, {'name': 'Diana Bochynska'}, {'name': 'Ewan St. John Smith'}, {'name': 'Moritz Gerstung'}, {'name': 'Peter J. Campbell'}, {'name': 'Elizabeth P. Murchison'}, {'name': 'Michael R. Stratton'}, {'name': 'Iigo Martincorena'}]\", 'abstract': '<p>The rates and patterns of somatic mutation in normal tissues are largely unknown outside of humans. Comparative analyses can shed light on the diversity of mutagenesis across species and on long-standing hypotheses regarding the evolution of somatic mutation rates and their role in cancer and ageing. Here, we used whole-genome sequencing of 208 intestinal crypts from 56 individuals to study the landscape of somatic mutation across 16 mammalian species. We found somatic mutagenesis to be dominated by seemingly endogenous mutational processes in all species, including 5-methylcytosine deamination and oxidative damage. With some differences, mutational signatures in other species resembled those described in humans, although the relative contribution of each signature varied across species. Remarkably, the somatic mutation rate per year varied greatly across species and exhibited a strong inverse relationship with species lifespan, with no other life-history trait studied displaying a comparable association. Despite widely different life histories among the species surveyed, including ~30-fold variation in lifespan and ~40,000-fold variation in body mass, the somatic mutation burden at the end of lifespan varied only by a factor of ~3. These data unveil common mutational processes across mammals and suggest that somatic mutation rates are evolutionarily constrained and may be a determinant of lifespan.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.16.21261912', 'tweets': 190, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Genomic profiles of Covishield and Covaxin breakthrough SARS-CoV-2 strains from Odisha, India', 'authors': \"[{'name': 'Arup Ghosh'}, {'name': 'Safal Walia'}, {'name': 'Roma Rattan'}, {'name': 'Amol Kanampalliwar'}, {'name': 'Atimukta Jha'}, {'name': 'Shifu Aggarwal'}, {'name': 'Sana Fatma'}, {'name': 'Niyati Das'}, {'name': 'Nirupama Chayani'}, {'name': 'Punit Prasad'}, {'name': 'Sunil K. Raghav'}, {'name': 'Ajay Parida'}]\", 'abstract': '<p>Vaccine breakthrough infections pose a vast challenge in the eradication of the COVID pandemic situation. Emerging SARS-CoV-2 variants of concern infecting the immunized individuals indicate an ongoing battle between host immunity and natural selection of the pathogen. Our report sheds light on the prominent SARS-CoV-2 variations observed in the isolates from AZD1222/Covishield and BBV152/Covaxin vaccinated subjects.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.15.21262067', 'tweets': 102, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Significant reduction in humoral immunity among healthcare workers and nursing home residents 6 months after COVID-19 BNT162b2 mRNA vaccination', 'authors': \"[{'name': 'David H. Canaday'}, {'name': 'Oladayo A. Oyebanji'}, {'name': 'Debbie Keresztesy'}, {'name': 'Michael Payne'}, {'name': 'Dennis Wilk'}, {'name': 'Lenore Carias'}, {'name': 'Htin Aung'}, {'name': 'Kerri St. Denis'}, {'name': 'Evan C. Lam'}, {'name': 'Christopher F. Rowley'}, {'name': 'Sarah D. Berry'}, {'name': 'Cheryl M. Cameron'}, {'name': 'Mark J. Cameron'}, {'name': 'Brigid Wilson'}, {'name': 'Alejandro B. Balazs'}, {'name': 'Christopher L. King'}, {'name': 'Stefan Gravenstein'}]\", 'abstract': '<p>High COVID-19 mortality among nursing home (NH) residents led to their prioritization for SARS-CoV-2 vaccination; most NH residents received BNT162b2 mRNA vaccination under the Emergency Use Authorization due to first to market and its availability. With NH residents poor initial vaccine response, the rise of NH breakthrough infections and outbreaks, characterization of the durability of immunity to inform public health policy on the need for boosting is needed. We report on humoral immunity from 2 weeks to 6-months post-vaccination in 120 NH residents and 92 ambulatory healthcare worker controls with and without pre-vaccination SARS-CoV-2 infection. Anti-spike and anti-receptor binding domain (RBD) IgG, and serum neutralization titers, were assessed using a bead-based ELISA method and pseudovirus neutralization assay. Anti-spike, anti-RBD and neutralization levels dropped more than 80% over 6 months time in all groups irrespective of prior SARS-CoV-2 infection. At 6 months post-vaccine, 72% of the infection-naive NH residents had neutralization titers at or below the lower limit of detection compared to 16% at 2 weeks after full vaccination. These data demonstrate a significant reduction in levels of antibody in all groups. In particular, those infection-naive NH residents had lower initial post-vaccination humoral immunity immediately and exhibited the greatest declines 6 months later. Healthcare workers, given their younger age and relative good-health, achieved higher initial antibody levels and better maintained them, yet also experienced significant declines in humoral immunity. Based on the rapid spread of the delta variant and reports of vaccine breakthrough in NH and among younger community populations, boosting NH residents may be warranted.</p>', 'posted': '2021-08-18'}, {'doi': '10.1101/2021.08.16.456551', 'tweets': 59, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Tuning levels of low-complexity domain interactions to modulate endogenous oncogenic transcription', 'authors': \"[{'name': 'Shasha Chong'}, {'name': 'Thomas G.W. Graham'}, {'name': 'Claire Dugast-Darzacq'}, {'name': 'Gina M. Dailey'}, {'name': 'Xavier Darzacq'}, {'name': 'Robert Tjian'}]\", 'abstract': '<p>Gene activation by mammalian transcription factors (TFs) requires dynamic, multivalent, and selective interactions of their intrinsically disordered low-complexity domains (LCDs), but how such interactions mediate transcription remains unclear. It has been proposed that extensive LCD-LCD interactions culminating in liquid-liquid phase separation (LLPS) of TFs is the dominant mechanism underlying transactivation. Here, we investigated how tuning the amount and localization of LCD-LCD interactions <italic>in vivo</italic> affects transcription of endogenous human genes. Quantitative single-cell and single-molecule imaging reveals that the oncogenic TF EWS/FLI1 requires a finely tuned range of LCD-LCD interactions to efficiently activate target genes. Modest or more dramatic increases in LCD-LCD interactions toward putative LLPS repress EWS/FLI1-driven transcription in patient cells. Likewise, ectopically creating LCD-LCD interactions to sequester EWS/FLI1 into a bona fide LLPS compartment, the nucleolus, inhibits EWS/FLI1-driven transcription and oncogenic transformation. Our findings reveal fundamental principles underlying LCD-mediated transcription and suggest mislocalizing specific LCD-LCD interactions as a novel therapeutic strategy for targeting disease-causing TFs.</p>', 'posted': '2021-08-17'}, {'doi': '10.1101/2021.08.20.457082', 'tweets': 44, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A nitrite-oxidizing bacterium constitutively consumes atmospheric hydrogen', 'authors': \"[{'name': 'Pok Man Leung'}, {'name': 'Anne Daebeler'}, {'name': 'Eleonora Chiri'}, {'name': 'Paul R. F. Cordero'}, {'name': 'Iresha Hanchapola'}, {'name': 'David L. Gillett'}, {'name': 'Ralf B Schittenhelm'}, {'name': 'Holger Daims'}, {'name': 'Chris Greening'}]\", 'abstract': '<p>Chemolithoautotrophic nitrite-oxidizing bacteria (NOB) of the genus Nitrospira contribute to nitrification in diverse natural environments and engineered systems. Nitrospira are thought to be well-adapted to substrate limitation owing to their high affinity for nitrite and capacity to use alternative energy sources. Here, we demonstrate that the canonical nitrite oxidizer Nitrospira moscoviensis oxidizes hydrogen (H2) below atmospheric levels using a high-affinity group 2a nickel-iron hydrogenase [Km(app) = 32 nM]. Atmospheric H2 oxidation occurred under both nitrite-replete and nitrite-deplete conditions, suggesting low-potential electrons derived from H2 oxidation promote nitrite-dependent growth and enable survival during nitrite limitation. Proteomic analyses confirmed the hydrogenase was abundant under both conditions and indicated extensive metabolic changes occur to reduce energy expenditure and growth under nitrite-deplete conditions. Respirometry analysis indicates the hydrogenase and nitrite oxidoreductase are bona fide components of the aerobic respiratory chain of N. moscoviensis, though they transfer electrons to distinct electron carriers in accord with the contrasting redox potentials of their substrates. Collectively, this study suggests atmospheric H2 oxidation enhances the growth and survival of NOB in amid variability of nitrite supply. These findings also extend the phenomenon of atmospheric H2 oxidation to a seventh phylum (Nitrospirota) and reveal unexpected new links between the global hydrogen and nitrogen cycles.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.20.457057', 'tweets': 42, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'MultiVI: deep generative model for the integration of multi-modal data', 'authors': \"[{'name': 'Tal Ashuach'}, {'name': 'Mariano I Gabitto'}, {'name': 'Michael I Jordan'}, {'name': 'Nir Yosef'}]\", 'abstract': '<p>The ability to jointly profile the transcriptional and chromatin landscape of single-cells has emerged as a powerful technique to identify cellular populations and shed light on their regulation of gene expression. Current computational methods analyze jointly profiled (paired) or individual data modalities (unpaired), but do not offer a principled method to analyze both paired and unpaired samples jointly. Here we present MultiVI, a probabilistic framework that leverages deep neural networks to jointly analyze scRNA, scATAC and multiomic (scRNA + scATAC) data. MultiVI creates an informative low-dimensional latent space that accurately reflects both chromatin and transcriptional properties of the cells even when one of the modalities is missing. MultiVI accounts for technical effects in both scRNA and scATAC-seq while correcting for batch effects in both data modalities. We use public datasets to demonstrate that MultiVI is stable, easy to use, and outperforms current approaches for the joint analysis of paired and unpaired data. MultiVI is available as an open source package, implemented in the scvi-tools framework: https://docs.scvi-tools.org</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.17.21262195', 'tweets': 41, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'COVID-19 Incidence and Hospitalization Rates are Inversely Related to Vaccination Coverage Among the 112 Most Populous Counties in the United States', 'authors': \"[{'name': 'Jeffrey E Harris'}]\", 'abstract': '<p>We tested whether COVID-19 incidence and hospitalization rates were inversely related to vaccination coverage among the 112 most populous counties in the United States, each with a population exceeding 600,000, and together with a combined total population of 147 million persons. We measured vaccination coverage as the percent of the total population fully vaccinated as of July 15, 2021, with the exception of 11 Texas counties, where the cutoff date was July 14, 2021. We measured COVID-19 incidence as the number of confirmed cases per 100,000 population during the 14-day period ending August 12, 2021. We measured hospitalization rates as the number of confirmed COVID-19 admissions per 100,000 population during the same 14-day period. COVID-19 incidence was significantly higher among counties in the lower half of the distribution of vaccination coverage (incidence 543.8 per 100,000 among 56 counties with mean coverage 42.61%) than among counties in the upper half of the distribution of coverage (incidence 280.7 per 100,000 among 56 counties with mean coverage 57.37%, p &lt; 0.0001). Hospital admissions were also significantly higher among counties in the lower half of the distribution (55.37 per 100,000) than in the upper half of the distribution (20.48 per 100,000, p &lt; 0.0001). In log-linear regression models, a 10-percentage-point increase in vaccination coverage was associated with a 28.3% decrease in COVID-19 incidence (95% confidence interval, 16.8 - 39.7%), a 44.9 percent decrease in the rate of COVID-19 hospitalization (95% CI, 28.8 - 61.0%), and a 16.6% decrease in COVID-19 hospitalizations per 100 cases (95% CI, 8.4 - 24.8%). Higher vaccination coverage is associated not only with significantly lower COVID-19 incidence, but also significantly less severe cases of the disease.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.20.457079', 'tweets': 40, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Humans and mice fluctuate between external and internal modes of sensory processing', 'authors': \"[{'name': 'Veith Andreas Weilnhammer'}, {'name': 'Heiner Stuke'}, {'name': 'Anna-Lena Eckert'}, {'name': 'Kai Standvoss'}, {'name': 'Philipp Sterzer'}]\", 'abstract': '<p>Perception cycles through periods of enhanced and reduced sensitivity to external information. Here, we asked whether such infra-slow oscillations arise as a noise-related epiphenomenon of limited processing capacity or, alternatively, represent a structured mechanism of perceptual inference. Using two large-scale datasets, we found that humans and mice waver between alternating intervals of externally- and internally-oriented modes of sensory analysis. During external mode, perception was more sensitive to external sensory information, whereas internal mode was characterized by enhanced biases toward perceptual history. Computational modeling indicated that dynamic changes in mode are governed by two interlinked factors: (i), the integration of subsequent stimuli over time and, (ii), infra-slow anti-phase oscillations in the perceptual impact of external sensory information versus internal predictions that are provided by perceptual history. Between-mode fluctuations may benefit perception by enabling the generation of stable representations of the environment despite an ongoing stream of noisy sensory inputs.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.21.457186', 'tweets': 40, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evidence of long-term purging of mutation load in killer whale genomes', 'authors': \"[{'name': 'Andrew D Foote'}, {'name': 'Thomas P Gilbert'}, {'name': 'Shyam Gopalakrishnan'}, {'name': 'Marie Louis'}, {'name': 'Michael Martin'}, {'name': 'Phillip A. Morin'}, {'name': 'Milaja Nykanen'}, {'name': 'Camilla Scharff-Olsen'}, {'name': 'Laurent A. Excoffier'}]\", 'abstract': '<p>Genomes of high latitude killer whales harbour signatures of post-glacial founding and expansion. Here, we investigate whether reduced efficacy of selection increased mutation load in founder populations, or whether recessive deleterious mutations exposed to selection in homozygous genotypes were purged. Comparing the accumulation of synonymous and non-synonymous mutations across pairs of globally sampled genomes reveals that the most significant outliers are high latitude North Atlantic genomes, which have accumulated significantly fewer non-synonymous mutations than all other populations. Comparisons with the genome of a 7.5-Kyr-old North Atlantic killer whale, inferred to be closely related to the population directly ancestral to present-day Icelandic and Norwegian populations, calibrates the timing of the action of selection on non-synonymous mutations predominantly to the mid-late Holocene. Non-synonymous mutations purged in modern Norwegian killer whale genomes are found as globally shared standing variation in heterozygote genotypes more often than expected, suggesting overdominance. Taken together, our findings are consistent with purging of recessive non-synonymous mutations exposed to selection in founder-associated homozygous genotypes.</p>', 'posted': '2021-08-21'}, {'doi': '10.1101/2021.08.16.456499', 'tweets': 36, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'scFlow: A Scalable and Reproducible Analysis Pipeline for Single-Cell RNA Sequencing Data', 'authors': \"[{'name': 'Combiz Khozoie'}, {'name': 'Nurun Fancy'}, {'name': 'Mahdi M. Marjaneh'}, {'name': 'Alan E. Murphy'}, {'name': 'Paul M. Matthews'}, {'name': 'Nathan Skene'}]\", 'abstract': '<p>Advances in single-cell RNA-sequencing technology over the last decade have enabled exponential increases in throughput: datasets with over a million cells are becoming commonplace. The burgeoning scale of data generation, combined with the proliferation of alternative analysis methods, led us to develop the scFlow toolkit and the nf-core/scflow pipeline for reproducible, efficient, and scalable analyses of single-cell and single-nuclei RNA-sequencing data. The scFlow toolkit provides a higher level of abstraction on top of popular single-cell packages within an R ecosystem, while the nf-core/scflow Nextflow pipeline is built within the nf-core framework to enable compute infrastructure-independent deployment across all institutions and research facilities. Here we present our flexible pipeline, which leverages the advantages of containerization and the potential of Cloud computing for easy orchestration and scaling of the analysis of large case/control datasets by even non-expert users. We demonstrate the functionality of the analysis pipeline from sparse-matrix quality control through to insight discovery with examples of analysis of four recently published public datasets and describe the extensibility of scFlow as a modular, open-source tool for single-cell and single nuclei bioinformatic analyses.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.17.456740', 'tweets': 33, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Accuracy of multiple sequence alignment methods in the reconstruction of transposable element families', 'authors': \"[{'name': 'Robert Hubley'}, {'name': 'Travis J. Wheeler'}, {'name': 'Arian F.A. Smit'}]\", 'abstract': '<p>The construction of a high-quality multiple sequence alignment (MSA) from copies of a transposable element (TE) is a critical step in the characterization of a new TE family. Most studies of MSA accuracy have been conducted on protein or RNA sequence families where structural features and strong signals of selection may assist with alignment. Less attention has been given to the quality of sequence alignments involving neutrally evolving DNA sequences such as those resulting from TE replication. Such alignments play an important role in understanding and representing TE family history. Transposable element sequences are challenging to align due to their wide divergence ranges, fragmentation, and predominantly-neutral mutation patterns. To gain insight into the effects of these properties on MSA accuracy, we developed a simulator of TE sequence evolution, and used it to generate a benchmark with which we evaluated the MSA predictions produced by several popular aligners, along with Refiner, a method we developed in the context of our RepeatModeler software. We find that MAFFT and Refiner generally outperform other aligners for low to medium divergence simulated sequences, while Refiner is uniquely effective when tasked with aligning high-divergent and fragmented instances of a family. As a result, consensus sequences derived from Refiner-based MSAs are more similar to the true consensus.</p>', 'posted': '2021-08-18'}, {'doi': '10.1101/2021.08.16.456545', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Learning predictive cognitive maps with spiking neurons during behaviour and replays', 'authors': \"[{'name': 'Jacopo Bono'}, {'name': 'Sara Zannone'}, {'name': 'Victor Pedrosa'}, {'name': 'Claudia Clopath'}]\", 'abstract': '<p>We describe a framework where a biologically plausible spiking neural network mimicking hippocampal layers learns a cognitive map known as the successor representation. We show analytically how, on the algorithmic level, the learning follows the TD(<italic></italic>) algorithm, which emerges from the underlying spike-timing dependent plasticity rule. We then analyze the implications of this framework, uncovering how behavioural activity and experience replays can play complementary roles when learning the representation of the environment, how we can learn relations over behavioural timescales with synaptic plasticity acting on the range of milliseconds, and how the learned representation can be flexibly encoded by allowing state-dependent delay discounting through neuromodulation and altered firing rates.</p>', 'posted': '2021-08-17'}, {'doi': '10.1101/2021.08.19.456935', 'tweets': 28, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Characterising the Photoplethysmography Pulse Waveform for Use in Human Neuroscience: The Hybrid Excess and Decay (HED) Model', 'authors': \"[{'name': 'Simon Williamson'}, {'name': 'Lucie Daniel-Watanabe'}, {'name': 'Johanna Finnemann'}, {'name': 'Craig Powell'}, {'name': 'Adam Teed'}, {'name': 'Micah Allen'}, {'name': 'Martin Paulus'}, {'name': 'Sahib S. Khalsa'}, {'name': 'Paul C. Fletcher'}]\", 'abstract': '<p>Photoplethysmography (PPG) offers a widely-used, convenient and non-invasive approach to monitoring basic indices of cardiovascular function such as heart rate and blood oxygenation. However, while the pulse waveform, generated by PPG comprises features that are shaped by physiological and psychological factors, it is frequently overlooked in analyses of such data. We suggest that studies could be enriched by exploiting the possibilities afforded by a systematic analysis of PPG waveforms. To do this we initially require a robust and automated means of characterising it, thereby allowing us to examine variations across individuals and between different physiological and psychological contexts. We present a psychophysiologically-relevant model, the Hybrid Excess and Decay (HED) Model, which characterises pulse wave morphology in terms of three underlying pressure waves and a decay function. We show that these parameters capture PPG data with a high degree of precision and, moreover, are sensitive to specific, physiologically-relevant changes within individuals. We present the theoretical and practical basis for the model and demonstrate its performance when applied to a pharmacological dataset of 105 participants receiving intravenous administrations of the sympathomimetic drug isoproterenol (Isoprenaline). We conclude by discussing the possible value in using the HED model to complement standard measures of PPG outputs.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.18.456813', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Prefrontal cortex neural compensation during an operant devaluation task', 'authors': \"[{'name': 'Hayley Fisher'}, {'name': 'Hongyu Lin'}, {'name': 'Jensen May'}, {'name': 'Caitlin McLean'}, {'name': 'Charles L. Pickens'}]\", 'abstract': '<p>Deficits in goal-directed action are reported in multiple neuropsychiatric conditions, including schizophrenia. However, dysfunction is not always apparent in early stages of schizophrenia, possibly due to neural compensation. We designed a novel devaluation task in which goal-directed action could be guided by stimulus-outcome (S-O) [presumably orbitofrontal cortex (OFC)-mediated] or response-outcome (R-O) associations [presumably prelimbic cortex (PL)-mediated]. We previously found suggestive evidence that OFC and PL could compensate for each other in this task, and we more directly assessed this potential compensation here. In Experiment 1, rats received OFC, PL, combined OFC+PL, or sham lesions and then completed our devaluation task. The OFC+PL lesion group exhibited impaired devaluation. In Experiment 2, rats received cholera-toxin-b (CTb) into OFC and either neurotoxic or sham PL lesions. Rats were then sacrificed on the last training day to double-label for Arc and CTb. We found increased Arc+CTb in mediodorsal thalamus (MD) and increased Arc+ neurons in OFC when PL was lesioned, suggesting that PL lesions lead to a compensatory increased activation of the MDOFC circuit. Our results suggest that our devaluation task can model neural compensation between OFC and PL and this compensation may be regulated by MD.</p><sec><title>Significance Statement</title><p>To detect compensatory responses, behavioral models that use different strategies must be developed to determine if the strategies shift when a brain area or circuit is incapacitated. Neural compensation is commonly observed in human research but only a few models of neural compensation exist, and few identify compensation within the prefrontal cortex. This research is among the first to show neural compensation between prefrontal cortex regions and implicate a thalamocortical circuit in modulating this compensation. Not only will this model provide a way to behaviorally identify subtle neurological shifts, it can also elucidate basic neurological mechanisms that mediate how circuits interact with each other and how dysfunction in one circuit can affect connectivity in other brain areas.</p></sec>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.19.456368', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Signatures of adaptive evolution in platyrrhine primate genomes', 'authors': \"[{'name': 'Hazel Byrne'}, {'name': 'Timothy H Webster'}, {'name': 'Sarah F Brosnan'}, {'name': 'Patricia Izar'}, {'name': 'Jessica W Lynch'}]\", 'abstract': '<p>The family Cebidae (capuchin and squirrel monkeys) form a remarkable platyrrhine clade exhibiting among the largest primate encephalisation quotients. Each cebid lineage is characterised by notable lineage-specific traits, with capuchins showing striking similarities to Hominidae including high sensorimotor intelligence with tool use, advanced cognitive abilities, and behavioural flexibility. Here, we take a comparative genomics approach, analysing five cebid branches including successive lineages, to infer a stepwise timeline for cebid adaptive evolution. We uncover candidate targets of selection across various periods of cebid evolution that may underlie the emergence of lineage-specific traits. Our analyses highlight shifting and sustained selective pressures on genes related to brain development, longevity, reproduction, and morphology, including evidence for cumulative and diversifying neurobiological adaptations over cebid evolutionary history. In addition to generating a new, high-quality reference genome assembly for robust capuchins, our results lend to a better understanding of the adaptive diversification of this distinctive primate clade.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.16.456345', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Brain transcriptomic profiling reveals common alterations across neurodegenerative and psychiatric disorders', 'authors': '[{\\'name\\': \\'Iman Sadeghi\\'}, {\\'name\\': \\'Juand Domingo Gispert\\'}, {\\'name\\': \\'Emilio Palumbo\\'}, {\\'name\\': \\'Manuel Munoz-Aguirre\\'}, {\\'name\\': \\'Valentin Wucher\\'}, {\\'name\\': \"Valeria D\\'Argenio\"}, {\\'name\\': \\'Gabriel Santpere\\'}, {\\'name\\': \\'Arcadi Navarro\\'}, {\\'name\\': \\'Roderic Guigo\\'}, {\\'name\\': \\'Natalia Vilor-Tejedor\\'}]', 'abstract': \"<p>Neurodegenerative and neuropsychiatric disorders (ND-NPs) are multifactorial, polygenic and complex behavioral phenotypes caused by brain abnormalities. Large-scale collaborative efforts have tried to identify the genetic architecture of these conditions. However, specific and shared underlying molecular pathobiology of brain illnesses is not clear. Here, we examine transcriptome-wide characterization of eight conditions, using a total of 2633 post-mortem brain samples from patients with Alzheimer's disease (AD), Parkinson's disease (PD), Progressive Supranuclear Palsy (PSP), Pathological Aging (PA), Autism Spectrum Disorder (ASD), Schizophrenia (Scz), Major Depressive Disorder (MDD), and Bipolar Disorder (BP)in comparison with 2,078 brain samples from matched control subjects. Similar transcriptome alterations were observed between NDs and NPs with the top correlations obtained between Scz-BP, ASD-PD, AD-PD, and Scz-ASD. Region-specific comparisons also revealed shared transcriptome alterations in frontal and temporal lobes across NPs and NDs. Co-expression network analysis identified coordinated dysregulations of cell-type-specific modules across NDs and NPs. This study provides a transcriptomic framework to understand the molecular alterations of NPs and NDs through their shared- and specific gene expression in the brain.</p>\", 'posted': '2021-08-16'}, {'doi': '10.1101/2021.08.18.456832', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'The genome of stress tolerant crop wild relative Paspalum vaginatum leads to increased biomass productivity in the crop Zea mays', 'authors': \"[{'name': 'Guangchao Sun'}, {'name': 'Nishikant Wase'}, {'name': 'Shengqiang Shu'}, {'name': 'Jerry Jenkins'}, {'name': 'Bangjun Zhou'}, {'name': 'Cindy Chen'}, {'name': 'Laura Sandor'}, {'name': 'Chris Plott'}, {'name': 'Yuko Yoshinga'}, {'name': 'Christopher Daum'}, {'name': 'Peng Qi'}, {'name': 'Kerrie Barry'}, {'name': 'Anna Lipzen'}, {'name': 'Luke Berry'}, {'name': 'Thomas Gottilla'}, {'name': 'Ashley Foltz'}, {'name': 'Huihui Yu'}, {'name': 'Ronan OMalley'}, {'name': 'Chi Zhang'}, {'name': 'Katrien M. Devos'}, {'name': 'Brandi Sigmon'}, {'name': 'Bin Yu'}, {'name': 'Toshihiro Obata'}, {'name': 'Jeremy Schmutz'}, {'name': 'James C. Schnable'}]\", 'abstract': '<p>A number of crop wild relatives can tolerate extreme stressed to a degree outside the range observed in their domesticated relatives. However, it is unclear whether or how the molecular mechanisms employed by these species can be translated to domesticated crops. Paspalum <italic>Paspalum vaginatum</italic> is a self-incompatible and multiply stress-tolerant wild relative of maize and sorghum. Here we describe the sequencing and pseudomolecule level assembly of a vegetatively propagated accession of <italic>P. vaginatum</italic>. Phylogenetic analysis based on 6,151 single-copy syntenic orthologous conserved in 6 related grass species placed paspalum as an outgroup of the maize-sorghum clade demonstrating paspalum as their closest sequenced wild relative. In parallel metabolic experiments, paspalum, but neither maize nor sorghum, exhibited significant increases in trehalose when grown under nutrient-deficit conditions. Inducing trehalose accumulation in maize, imitating the metabolic phenotype of paspalum, resulting in autophagy dependent increases in biomass accumulation.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.16.456500', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'SeqDistK: a Novel Tool for Alignment-free Phylogenetic Analysis', 'authors': \"[{'name': 'Xuemei Liu'}, {'name': 'Wen Li'}, {'name': 'Guanda Huang'}, {'name': 'Tianlai Huang'}, {'name': 'Qingang Xiong'}, {'name': 'Wen Chen'}, {'name': 'Li C. Xia'}]\", 'abstract': '<p>Algorithms for constructing phylogenetic trees are fundamental to study the evolution of viruses, bacteria, and other microbes. Established multiple alignment-based algorithms are inefficient for large scale metagenomic sequence data because of their high requirement of inter-sequence correlation and high computational complexity. In this paper, we present SeqDistK, a novel tool for alignment-free phylogenetic analysis. SeqDistK computes the dissimilarity matrix for phylogenetic analysis, incorporating seven k-mer based dissimilarity measures, namely d2, d2S, d2star, Euclidean, Manhattan, CVTree, and Chebyshev. Based on these dissimilarities, SeqDistK constructs phylogenetic tree using the Unweighted Pair Group Method with Arithmetic Mean algorithm. Using a golden standard dataset of 16S rRNA and its associated phylogenetic tree, we compared SeqDistK to Muscle  a multi sequence aligner. We found SeqDistK was not only 38 times faster than Muscle in computational efficiency but also more accurate. SeqDistK achieved the smallest symmetric difference between the inferred and ground truth trees with a range between 13 to 18, while that of Muscle was 62. When measures d2, d2star, d2S, Euclidean, and k-mer size k=5 were used, SeqDistK consistently inferred phylogenetic tree almost identical to the ground truth tree. We also performed clustering of 16S rRNA sequences using SeqDistK and found the clustering was highly consistent with known biological taxonomy. Among all the measures, d2S (k=5, M=2) showed the best accuracy as it correctly clustered and classified all sample sequences. In summary, SeqDistK is a novel, fast and accurate alignment-free tool for large-scale phylogenetic analysis. SeqDistK software is freely available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/htczero/SeqDistK\">https://github.com/htczero/SeqDistK</ext-link>.</p>', 'posted': '2021-08-17'}, {'doi': '10.1101/2021.08.17.456620', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Immunopeptidomic analysis of influenza A virus infected human tissues identifies internal proteins as a rich source of HLA ligands', 'authors': \"[{'name': 'Ben Nicholas'}, {'name': 'Alistair Bailey'}, {'name': 'Karl J Staples'}, {'name': 'Tom Wilkinson'}, {'name': 'Tim Elliott'}, {'name': 'Paul Skipp'}]\", 'abstract': '<p>CD8+ and CD4+ T cells provide cell-mediated cross-protection against multiple influenza strains by recognising epitopes bound as peptides to human leukocyte antigen (HLA) class I and -II molecules respectively. Two challenges in identifying the immunodominant epitopes needed to generate a universal T cell influenza vaccine are: A lack of cell models susceptible to influenza infection which present population-prevalent HLA allotypes, and an absence of a reliable in-vitro method of identifying class II HLA peptides. Here we present a mass spectrometry-based proteomics strategy for identifying viral peptides derived from the A/H3N2/X31 and A/H3N2/Wisconsin/67/2005 strains of influenza. We compared the HLA-I and -II immunopeptidomes presented by ex-vivo influenza challenged human lung tissues. We then compared these with directly infected immortalised macrophage-like cell line (THP1) and primary dendritic cells fed apoptotic influenza-infected respiratory epithelial cells. In each of the three experimental conditions we identified novel influenza class I and II HLA peptides with motifs specific for the host allotype. Ex-vivo infected lung tissues yielded few class-II HLA peptides despite significant numbers of alveolar macrophages, including directly infected ones, present within the tissues. THP1 cells presented HLA-I viral peptides derived predominantly from internal proteins. Primary dendritic cells presented predominantly viral envelope-derived HLA class II peptides following phagocytosis of apoptotic infected cells. The most frequent viral source protein for HLA-I and -II was matrix 1 protein (M1). This work confirms that internal influenza proteins, particularly M1, are a rich source of CD4+ and CD8+ T cell epitopes. Moreover, we demonstrate the utility of two ex-vivo fully human infection models which enable direct HLA-I and -II immunopeptide identification without significant viral tropism limitations. Application of this epitope discovery strategy in a clinical setting will provide more certainty in rational vaccine design against influenza and other emergent viruses.</p><sec><title>Author Summary</title><p>Influenza infections present a significant global health challenge. High rates of mutation require reformulation of vaccines annually. Vaccines are designed to induce antibody responses to the surface proteins of the influenza virus, but the contribution of T cells to overall immunity is unclear. Here, we used several totally human laboratory models to show how the viral proteins are presented to the T cells to induce immunity. We found that CD8 T cells, which kill infected cells, and CD4 T cells which support the CD8 T cells as well as the antibody-producing B cells, mainly see proteins from inside the viral particle, not the surface ones which are targeted by antibodies. These internal viral proteins are more similar between different viral strains than the surface proteins, and therefore suggest that vaccines designed to induce T cell responses could be better protective if they target internal viral proteins.</p></sec>', 'posted': '2021-08-17'}, {'doi': '10.1101/2021.08.16.456482', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Particle Uptake Driven Phagocytosis in Macrophages and Neutrophils Enhances Bacterial Clearance', 'authors': \"[{'name': 'Preeti Sharma'}, {'name': 'Anjali Vijaykumar'}, {'name': 'Jayashree Vijaya Raghavan'}, {'name': 'Supriya Rajendra Rananaware'}, {'name': 'Alakesh Alakesh'}, {'name': 'Janhavi Bodele'}, {'name': 'Junaid Ur Rehman'}, {'name': 'Shivani Shukla'}, {'name': 'Virta Wagde'}, {'name': 'Savitha Nadig'}, {'name': 'Sveta Chakrabarti'}, {'name': 'Sandhya S. Visweswariah'}, {'name': 'Dipankar Nandi'}, {'name': 'Balasubramanian Gopal'}, {'name': 'Siddharth Jhunjhunwala'}]\", 'abstract': '<p>Humans are exposed to numerous synthetic foreign particulates in the form of environmental pollutants and diagnostic or therapeutic agents. Specialized immune cells (phagocytes) clear these particulates by phagocytosing and attempting to degrade them. The process of recognition and internalization of the particulates may trigger changes in the function of phagocytes. Some of these changes, especially the ability of a particle-loaded phagocyte to take up and neutralize pathogens, remains poorly studied. Herein, we demonstrate that the uptake of non-stimulatory cargo-free particles enhances the phagocytic ability of monocytes, macrophages and neutrophils. The enhancement in phagocytic ability was independent of particle properties, such as size or the base material constituting the particle. Additionally, we show that the increased phagocytosis was not a result of cellular activation or cellular heterogeneity but was driven by changes in cell membrane fluidity and cellular compliance. A consequence of the enhanced phagocytic activity was that particulate-laden immune cells neutralize <italic>E. coli</italic> faster in culture. Moreover, when administered in mice as a prophylactic, particulates enable faster clearance of <italic>E. coli</italic> and <italic>S. epidermidis</italic>. Together, we demonstrate that the process of uptake induces cellular changes that favor additional phagocytic events. This study provides insights into using non-stimulatory cargo-free particles to engineer immune cell functions for applications involving faster clearance of phagocytosable particulates.</p>', 'posted': '2021-08-16'}, {'doi': '10.1101/2021.08.20.457167', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Evolving a generalist biosensor for bicyclic monoterpenes', 'authors': '[{\\'name\\': \"Simon d\\'Oelsnitz\"}, {\\'name\\': \\'Vylan Nguyen\\'}, {\\'name\\': \\'Hal S Alper\\'}, {\\'name\\': \\'Andrew D Ellington\\'}]', 'abstract': \"<p>Prokaryotic transcription factors can be repurposed as analytical and synthetic tools for precise chemical measurement and regulation. Monoterpenes encompass a broad chemical family that are commercially valuable as flavors, cosmetics, and fragrances, but have proven difficult to measure, especially in cells. Herein, we develop genetically-encoded, generalist monoterpene biosensors by using directed evolution to expand the effector specificity of the camphor-responsive TetR-family regulator CamR from Pseudomonas putida. Using a novel negative selection coupled with a high-throughput positive screen (Seamless Enrichment of Ligand-Inducible Sensors, SELIS), we evolve CamR biosensors that can recognize four distinct monoterpenes: borneol, fenchol, eucalyptol, and camphene. Different evolutionary trajectories surprisingly yielded common mutations, emphasizing the utility of CamR as a platform for creating generalist biosensors. Systematic promoter optimization driving the reporter increased the system's signal-to-noise ratio to 150-fold. These sensors can serve as a starting point for the high-throughput screening and dynamic regulation of bicyclic monoterpene production strains.</p>\", 'posted': '2021-08-21'}, {'doi': '10.1101/2021.08.18.456823', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Divergent transcriptional and transforming properties of PAX3-FOXO1 and PAX7-FOXO1 paralogs', 'authors': \"[{'name': 'Line Manceau'}, {'name': 'Julien Richard Albert'}, {'name': 'Pier-Luigi Lollini'}, {'name': 'Maxim V. C. Greenberg'}, {'name': 'Pascale Gilardi-Hebenstreit'}, {'name': 'Vanessa Ribes'}]\", 'abstract': '<p>The hallmarks of the alveolar subclass of rhabdomyosarcoma are chromosomal translocations that generate chimeric PAX3-FOXO1 or PAX7-FOXO1 transcription factors. Both PAX-FOXO1s result in related cell transformation in animal models, but both mutations are associated with distinct pathological manifestations in patients. To assess the mechanisms underlying these differences, we generated isogenic fibroblast lines expressing either PAX-FOXO1 paralog. Mapping of their genomic recruitment using CUT&amp;Tag revealed that the two chimeric proteins have distinct DNA binding preferences. In addition, PAX7-FOXO1 causes stronger <italic>de novo</italic> transactivation of its bound regions than PAX3-FOXO1, resulting in greater transcriptomic dynamics involving genes regulating cell shape and cycle. Consistently, PAX3-FOXO1 accentuates fibroblast cellular traits associated with contractility and surface adhesion and limits entry into M phase. In contrast, PAX7-FOXO1 drives cells to adopt an amoeboid shape, reduces entry into S phase, and causes more genomic instabilities. Altogether, our results argue that the diversity of rhabdomyosarcoma manifestation arises, in part, from the divergence between the transcriptional activities of PAX3-FOXO1 and PAX7-FOXO1. Furthermore, the identified pronounced deleterious effects of PAX7-FOXO1 provide an explanation for the low frequency of the translocation generating this factor in patients with rhabdomyosarcoma.</p>', 'posted': '2021-08-18'}, {'doi': '10.1101/2021.08.16.456438', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Genetic analysis of early phenology in lentil identifies distinct loci controlling component traits', 'authors': \"[{'name': 'Vinodan Rajandran'}, {'name': 'Raul Ortega'}, {'name': 'Jacqueline K. Vander Schoor'}, {'name': 'Jakob B. Butler'}, {'name': 'Jules S. Freeman'}, {'name': 'Valerie F.G. Hecht'}, {'name': 'Willie Erskine'}, {'name': 'Ian C. Murfet'}, {'name': 'Kirstin E. Bett'}, {'name': 'James L. Weller'}]\", 'abstract': '<p>Reproductive phenology is well known to be a key feature of crop adaptation to diverse ecogeographic variation and management practices. Lentil is one of the founder pulse crops of middle-eastern Neolithic agriculture, and the modern-day domesticated lentil germplasm is generally considered to form three broad adaptation groups: Mediterranean, South Asian and northern temperate, which correspond approximately to the major global production environments. Understanding the molecular basis of these adaptations is crucial to maximise efficiency of breeding programs. Here, we use a QTL approach to dissect the earliness that is characteristic of the South Asian <italic>pilosae</italic> ecotype, and that suits it to the typically short winter cropping season. We identified two loci, <italic>DTF6a</italic> and <italic>DTF6b</italic>, at which dominant alleles confer early flowering. We show that, although these loci can interact in an additive manner, <italic>DTF6a</italic> alone is sufficient to confer early flowering even in extremely short photoperiods. Comparisons with closely related legume species confirmed the presence of a conserved cluster of three <italic>FT</italic> orthologs among potential candidate genes in the region, and expression analysis in near-isogenic material showed that the early <italic>dtf6a</italic> allele is associated with a strong derepression of the <italic>FTa1</italic> gene in particular. Analysis of sequence variation revealed the presence of a 7.4 kb deletion in the <italic>FTa1</italic>-<italic>FTa2</italic> intergenic region in the <italic>pilosae</italic> parent, and a wide survey of over 400 accessions with diverse origin showed that the <italic>dtf6a</italic> allele is dominant in South Asia material. Collectively, these results contribute to understanding the molecular basis of global adaptation in lentil, and further emphasize the importance of this conserved genomic region for adaptation in temperate legumes generally.</p>', 'posted': '2021-08-16'}, {'doi': '10.1101/2021.08.20.457146', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'SARS-CoV-2 hijacks p38/MAPK11 to promote viral protein translation', 'authors': \"[{'name': 'Christina A Higgins'}, {'name': 'Benjamin E Nilsson-Payant'}, {'name': 'Andrew Kurland'}, {'name': 'Prithy Adhikary'}, {'name': 'Ilona Golynker'}, {'name': 'Oded Danziger'}, {'name': 'Maryline Panis'}, {'name': 'Brad R Rosenberg'}, {'name': 'Benjamin R tenOever'}, {'name': 'Jeffrey R Johnson'}]\", 'abstract': '<p>SARS-CoV-2, the causative agent of the COVID-19 pandemic, drastically modifies the cells that it infects. One such effect is the activation of the host p38 mitogen-activated protein kinase (MAPK) pathway, which plays a major role in inflammation pathways that are dysregulated in severe COVID-19 cases. Inhibition of p38/MAPK activity in SARS-CoV-2-infected cells reduces both cytokine production and viral replication. Here, we applied a systems biology approach to better understand interactions between the p38/MAPK pathway and SARS-CoV-2 in human lung epithelial cells. We found several components of the p38/MAPK pathway positively and negatively impact SARS-CoV-2 infection and that p38 is a required host factor for SARS-CoV-2 that acts by promoting viral protein translation in a manner that prevents innate immune sensing. Furthermore, we combined chemical and genetic perturbations of p38 with quantitative phosphoproteomics to identify novel, putative p38 substrates in an unbiased manner, with broad relevance beyond SARS-CoV-2 biology.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.19.457007', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Methods for recording and imaging spreading depolarizations and seizure-like events in mouse hippocampal slices', 'authors': \"[{'name': 'Yi-Ling Lu'}, {'name': 'Helen E. Scharfman'}]\", 'abstract': '<p>Spreading depolarization (SD) is a sudden and synchronized depolarization of principal cells followed by depression of activity, which slowly propagates across brain regions like cortex or hippocampus. SD is considered to be mechanistically relevant to migraine, epilepsy, and traumatic brain injury. Interestingly, research into SD typically uses SD triggered immediately after a focal stimulus. Here we optimize an in vitro experimental model allowing us to record SD without focal stimulation. This method uses electrophysiological recordings and intrinsic optical imaging in slices. The method is also relatively easy and inexpensive. Acute hippocampal slices from mice or rats were prepared and used for extracellular and whole-cell recordings. Recordings were made in a submerged-style chamber with flow of artificial cerebrospinal fluid (aCSF) above and below the slices. Flow was fast (&gt; 5ml/min), and temperature was 32C. As soon as slices were placed in the chamber, aCSF containing 0 mM Mg<sup>2+</sup> and 5 mM K<sup>+</sup> (0 Mg<sup>2+</sup>/5 K<sup>+</sup> aCSF) was used. Two major types of activity were observed: SD and seizure-like events (SLEs). Both occurred after many minutes of recording. Although both mouse and rat slices showed SLEs, only mouse slices developed SD and did so in the first hour of 0 Mg<sup>2+</sup>/5 K<sup>+</sup> aCSF exposure. Intrinsic optical imaging showed that most SDs initiated in CA3 and could propagate into CA1 and dentate gyrus. In dentate gyrus, SD propagated in two separate waves: (1) into the hilus and (2) into granule cell and molecular layers simultaneously. This in vitro model can be used to better understand the mechanisms and relationship between SD and SLEs. It could also be useful in preclinical drug screening.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.16.456464', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'RSC-mediated nucleosome positioning and GRFs form barriers in promoters to limit divergent noncoding transcription', 'authors': \"[{'name': 'Andrew Wu'}, {'name': 'Claudia Vivori'}, {'name': 'Harshil Patel'}, {'name': 'Theodora Sideri'}, {'name': 'Folkert J. van Werven'}]\", 'abstract': '<p>The directionality of gene promoters - the ratio of protein-coding over divergent noncoding transcription - is highly variable and regulated. How promoter directionality is controlled remains poorly understood. We show that the chromatin remodelling complex RSC and general regulatory factors (GRFs) dictate promoter directionality by attenuating divergent transcription. Depletion of RSC increased divergent noncoding transcription and decreased protein-coding transcription at promoters with strong directionality. Consistent with RSCs role in regulating chromatin, RSC depletion impacts nucleosome occupancy upstream of the nucleosome depleted region where divergent transcription initiates, suggesting that nucleosome positioning at the 5 border of gene promoters physically blocks the recruitment of the transcription machinery and inhibits initiation of divergent transcription. Highly directional promoters were also enriched for the binding of GRFs such as Reb1 and Abf1. Furthermore, ectopic targeting of divergent transcription initiation sites with GRFs or the dCas9 protein can suppress divergent transcription. Our data suggest that RSC-mediated nucleosome positioning and GRFs play a pervasive role in repressing divergent transcription. We propose that any DNA binding factor, when stably associated with cryptic transcription start sites, can form a barrier for repressing divergent transcription. Our study provides an explanation as to why certain promoters are more directional than others.</p>', 'posted': '2021-08-16'}, {'doi': '10.1101/2021.08.18.456372', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Identification and comparative analysis of long non-coding RNAs in the brain of fire ant queens in two different reproductive states', 'authors': \"[{'name': 'Cheng-Hung Tsai'}, {'name': 'Tzu-Chieh Lin'}, {'name': 'Yi-Hsien Chang'}, {'name': 'Huai-Kuang Tsai'}, {'name': 'Jia-Hsin Huang'}]\", 'abstract': '<sec><title>Background</title><p>Many long non-coding RNAs (lncRNAs) have been extensively identified in many higher eukaryotic species. The function of lncRNAs has been reported to play important roles in diverse biological processes, including developmental regulation and behavioral plasticity. However, there are no reports of systematic characterization of long non-coding RNAs in the fire ant <italic>Solenopsis invicta</italic>.</p></sec><sec><title>Results</title><p>In this study, we performed a genome-wide analysis of lncRNAs in the brains of <italic>S. invicta</italic> from RNA-seq. In total, 1,393 novel lncRNA transcripts were identified in the fire ant. In contrast to the annotated lncRNA transcripts having at least two exons, novel lncRNAs are monoexonic transcripts with a shorter length. Besides, the transcriptome from virgin alate and dealate mated queens were analyzed and compared. The results showed 295 differentially expressed mRNA genes (DEGs) and 65 differentially expressed lncRNA genes (DELs) between virgin and mated queens, of which 17 lncRNAs were highly expressed in the virgin alates and 47 lncRNAs were highly expressed in the mated dealates. By identifying the DEL:DEG pairs with high association in their expression (Spearmans |<italic>rho</italic>| &gt; 0.8 and <italic>p</italic>-value &lt; 0.01), many DELs were co-regulated with DEGs after mating. Furthermore, several remarkable lncRNAs (<italic>MSTRG</italic>.<italic>6523, MSTRG</italic>.<italic>588</italic>, and <italic>nc909</italic>) that were found to associate with particular coding genes may play important roles in the regulation of brain gene expression in reproductive transition in fire ants.</p></sec><sec><title>Conclusion</title><p>This study provides the first genome-wide identification of <italic>S. invicta</italic> lncRNAs in the brains in different reproductive states and will contribute to a fuller understanding of the transcriptional regulation underpinning reproductive changes.</p></sec>', 'posted': '2021-08-18'}, {'doi': '10.1101/2021.08.17.21262177', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Intensive Care and Critical Care Medicine', 'covid': True, 'title': 'Do Covid-19 patients needing ICU admission have worse 6 months follow up outcomes when compared with hospitalized non-ICU patients? A prospective cohort study', 'authors': \"[{'name': 'Suleyman Yildirim'}, {'name': 'Seher Susam'}, {'name': 'Pinar Cimen'}, {'name': 'Sena Yapicioglu'}, {'name': 'Onur Sunecli'}, {'name': 'Ozlem Ediboglu'}, {'name': 'Cenk Kirakli'}]\", 'abstract': '<p>Introduction \\nStudies focus on pathogenesis, clinical manifestations, and complications during the early phase of the coronavirus disease-19 (COVID-19). Long-term outcomes of COVID-19 patients who discharge intensive care unit (ICU) are unclear. \\nObjectives\\nWe investigated the effect of COVID-19 on lung structure, pulmonary functional, exercise capacity and quality of life in patients discharge from ICU and medical ward.\\nMethods\\nA prospective single-centre study conducted in PCR confirmed COVID-19 patients who has been discharged from University of Health Sciences, Dr. Suat Seren Chest Disease and Thoracic Surgery Teaching and Research Hospital between 15 January and 5 March 2021. Patients who followed up for more than 48 hours in ICU and more than 72 hours in medical ward were included the study. Computed tomography scores, pulmonary functional tests (PFT), 6-min walking distance and health related quality of life by SF-36 were compared between ICU and medical ward patients at 6 months after discharge.\\nResults\\t\\n\\tSeventy patients were included final analyses and 31 of them discharged from ICU. ICU patients had higher CT scores than non-ICU patients at admission (17 vs 11) and follow up visit (6 vs 0). Two-three of ICU patients had at least one abnormal finding at control CT. Advanced age (OR 1.08, 95% CI 1.02-1.15) and higher CT score at admission (OR 1.13, 95% CI 1.01-1.27) were risk factors for having radiological abnormalities at control CT.  \\nConclusion\\n\\tA number of COVID-19 survivors especially with severe disease could not fully recover after 6 months of hospital discharge.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.18.456871', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'RRNPP_detector: a tool to detect RRNPP quorum sensing systems in chromosomes, plasmids and phages of gram-positive bacteria', 'authors': \"[{'name': 'Charles Bernard'}, {'name': 'Yanyan Li'}, {'name': 'Eric Bapteste'}, {'name': 'Philippe Lopez'}]\", 'abstract': '<p>Gram-positive bacteria (e.g. Firmicutes) and their mobile genetic elements (plasmids, bacteriophages) encode peptide-based quorum sensing systems (QSSs) that regulate behavioral transitions in a density-dependent manner. In their simplest form, termed RRNPP, these QSSs are composed of two adjacent genes: a communication propeptide and its cognate intracellular receptor. Despite the prime importance of RRNPP QSSs in the regulation of key biological pathways such as virulence, sporulation or biofilm formation in bacteria, conjugation in plasmids or lysogeny in temperate bacteriophages, no tools exist to predict their presence in target genomes/mobilomes. Here, we introduce RRNPP_detector, a software to predict RRNPP QSSs in chromosomes, plasmids and bacteriophages of gram-positive bacteria, available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/TeamAIRE/RRNPP_detector\">https://github.com/TeamAIRE/RRNPP_detector</ext-link>. RRNPP_detector does not rely on homology searches but on a signature of multiple criteria, which are common between distinct families of experimentally-validated RRNPP QSSs. Because this signature is generic while specific to the canonical mechanism of RRNPP quorum sensing, it enables the discovery of novel RRNPP QSSs and thus of novel languages of biocommunication. Applying RRNPP_detector against complete genomes of viruses and Firmicutes available on the NCBI, we report a potential 7.5-fold expansion of RRNPP QSS diversity, alternative secretion-modes for certain candidate QSS propeptides, bilingual bacteriophages and plasmids, as well as predicted chromosomal and plasmidic Biosynthetic-Gene-Clusters regulated by QSSs.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.21.457182', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Decoding cellular deformation from pseudo-simultaneously observed Rho GTPase activities', 'authors': \"[{'name': 'Katsuyuki Kunida'}, {'name': 'Nobuhiro Takagi'}, {'name': 'Kazushi Ikeda'}, {'name': 'Takeshi Nakamura'}, {'name': 'Yuichi Sakumura'}]\", 'abstract': '<p>The inability to simultaneously observe all of the important Rho GTPases (Cdc42, Rac1, and RhoA) has prevented us from obtaining evidence of their coordinated regulation during cell deformation. Here, we propose Motion-Triggered Average (MTA), an algorithm that converts individually observed GTPases into pseudo-simultaneous observations. Using the time series obtained by MTA and mathematical model, we succeeded for the first time in decoding the cell edge velocity from the three GTPase activities to provide clear numerical evidence for coordinated cell edge regulation by the three GTPases. We found that the characteristics of the obtained activities were consistent with those of previous studies, and that GTPase activities and their derivatives were involved in edge regulation. Our approach provides an effective strategy for using single-molecule observations to elucidate problems hampered by the lack of simultaneous observations.</p>', 'posted': '2021-08-21'}, {'doi': '10.1101/2021.08.20.457124', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Spacer2PAM: A computational framework for identification of functional PAM sequences for endogenous CRISPR systems', 'authors': \"[{'name': 'Grant A Rybnicky'}, {'name': 'Nicholas A Fackler'}, {'name': 'Ashty S Karim'}, {'name': 'Michael Kpke'}, {'name': 'Michael C Jewett'}]\", 'abstract': '<p>RNA-guided nucleases from clustered regularly interspaced short palindromic repeats (CRISPR) systems expand opportunities for precise, targeted genome modification. Endogenous CRISPR systems in many bacteria and archaea are particularly attractive to circumvent expression, functionality, and unintended activity hurdles posed by heterologous CRISPR effectors. However, each CRISPR system recognizes a unique set of PAM sequences, which requires extensive screening of randomized DNA libraries. This challenge makes it difficult to develop endogenous CRISPR systems, especially in organisms that are slow-growing or have transformation idiosyncrasies. To address this limitation, we present Spacer2PAM, an easy-to-use, easy-to-interpret R package built to identify potential PAM sequences for any CRISPR system given its corresponding CRISPR array as input. Spacer2PAM can be used in Quick mode to generate a single PAM prediction that is likely to be functional or in Comprehensive mode to inform targeted, unpooled PAM libraries small enough to screen in difficult to transform organisms. We demonstrate Spacer2PAM by predicting PAM sequences for industrially relevant organisms and experimentally identifying seven PAM sequences that mediate interference from the Spacer2PAM-predicted PAM library for the type I-B CRISPR system from Clostridium autoethanogenum. We anticipate that Spacer2PAM will facilitate the use of endogenous CRISPR systems for industrial biotechnology and synthetic biology.</p>', 'posted': '2021-08-21'}, {'doi': '10.1101/2021.08.19.457008', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Tools and applications for integrative analysis of DNA methylation in social insects', 'authors': \"[{'name': 'Claire Morandin'}, {'name': 'Volker P. Brendel'}]\", 'abstract': '<p>DNA methylation is a common epigenetic signaling tool and an important biological process which is widely studied in a large array of species. The presence, level, and function of DNA methylation varies greatly across species. In insects, DNA methylation systems are reduced, and methylation rates are often low. Low methylation levels probed by whole genome bisulfite sequencing require great care with respect to data quality control and intepretation. Here we introduce BWASP/R, a reproducible, scalable workflow, that allows efficient, scalable, and entirely reproducible analyses of raw DNA methylation sequencing data. Consistent application of quality control filters and analysis parameters provides fair comparisons among different studies and an integrated view of all experiments on one species. We describe the capabilities of the <monospace>BWASP/R</monospace> workflow by re-analyzing several publicly available social insect WGBS datasets, comprising 70 samples and cumulatively 147 replicates from four different species. We show that the CpG methylome comprises only about 1.5% of CpG sites in the honeybee genome and that the cumulative data are consistent with genetic signatures of site accessibility and physiological control of methylation levels.</p><sec><title>Significance Statement</title><p>DNA methylation in the honeybee genome occurs occurs almost entirely at CpG sites. Methylation rates are small compared to rates in mammalian or plant genomes. De novo analysis of all published honeybee methylation studies and statistical modeling suggests that the CpG methylome consists of about only 300,000 sites. The development of a fully reproducible, scalable, portable workflow allows for easy accessible updates of integrative views of all current experiments. The integrated results for the honeybee are consistent with genetic determination of methylation site accessibility by yet uncharacterized sequence features and physiological control of methylation levels at those sites.</p></sec>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.19.21262314', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Intensive Care and Critical Care Medicine', 'covid': True, 'title': 'Inhaled nitric oxide use in COVID19-induced hypoxemic respiratory failure.', 'authors': \"[{'name': 'Abhishek R. Giri'}, {'name': 'Siva Naga S. Yarrarapu'}, {'name': 'Nirmaljot Kaur'}, {'name': 'Alexander P. Hochwald'}, {'name': 'Julia Crook'}, {'name': 'Scott Helgeson'}, {'name': 'Michael F. Harrison'}, {'name': 'Neal Patel'}, {'name': 'Pramod K. Guru'}, {'name': 'Philip Lowman'}, {'name': 'Pablo Moreno-Franco'}, {'name': 'Augustine Lee'}, {'name': 'Devang K. Sanghavi'}]\", 'abstract': '<p>Introduction: Nitric Oxide (NO) is an endogenous vasodilator that is synthesized by the vascular endothelium.  Due to its vasodilatory effect and short half-life, the use of NO as an exogenous inhaled medication (iNO) to target the pulmonary vasculature, in conditions with increased pulmonary vascular resistance, has been studied. \\nThe use of iNO in patients with ARDS secondary to COVID-19 has therapeutic importance in improving oxygenation. It also has potential anti-viral, anti-inflammatory, and anti-thrombotic properties.\\nHerein, we want to share our experience of use of iNO in hypoxemic respiratory failure secondary to COVID 19 pneumonia. We hypothesized that iNO may be beneficial at preventing intubation, decreasing invasive mechanical ventilation duration, and consequently improve outcomes including hospital mortality.\\n\\nMethods: This is a descriptive hypothesis generating study of patients admitted for COVID-19 pneumonia who received iNO for hypoxemic respiratory failure, at a single tertiary care center. We collected information on patient demographics, co-morbidities, iNO treatment, need for intubation, arterial blood gas analysis, laboratory values, hospital length of stay, and mortality. Patients were divided into two groups based on the timing of iNO administration: group 1 - pre-intubation (i.e. iNO started at least 1 day prior to endotracheal intubation, if any) and group 2 - post-intubation (i.e. iNO started on the same day as or after endotracheal intubation and mechanical ventilation). \\n\\nResult: A total of 45 (group 1, n=26 [57.8%] vs group2, n=19 [42.2%]) COVID 19 patients who had iNO use. The mean time from hospital admission to iNO administration(days) in group 1 was 2.1 (1.8, SD) vs 4.2 (5.9, SD) in group 2. The mean hospital length of stay from the beginning of iNO treatment until discharge or death was 18.3 vs 26.2 days, with 8 deaths (30.8%) vs 9 deaths (47.4%) in group 1 vs group 2, respectively. \\nDiscussion:\\nOur study is unable to demonstrate comparably outcomes benefit of iNO. Although there was a trend towards decreased  need for  invasive mechanical ventilation in group 1[Only 11 (42.3 %) patients were intubated out of 26 who received iNO early after hospital admission (2.3 days)], no statistical significance could be achieved because of small sample size. \\nOur study demonstrated that iNO administration pre-intubation did not appear harmful and appears to be safe, complementary to HFNC,  signalling the domain where systematic investigation is required to confirm or not the potential for iNO to improve patient outcomes in the management of COVID  19-induced hypoxemic respiratory failure.\\nConclusion:\\nThis study showcases the potential benefit of early pre-intubation use of iNO in COVID patients with hypoxemic respiratory failure. This study could conclusively form the basis for a prospective trial and could have a tremendous impact in improving patient outcomes.</p>', 'posted': '2021-08-21'}, {'doi': '10.1101/2021.08.20.457106', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Trade-driven trapping dampens the biodiversity benefits of forest restoration in Southeast Asia', 'authors': \"[{'name': 'H S Sathya Chandra Sagar'}, {'name': 'James J Gilroy'}, {'name': 'Tom Swinfield'}, {'name': 'Ding Li Yong'}, {'name': 'Elva Gemita'}, {'name': 'Novriyanti Novriyanti'}, {'name': 'David C Lee'}, {'name': 'Muhammad Nazri Janra'}, {'name': 'Andrew Balmford'}, {'name': 'Fangyuan Hua'}]\", 'abstract': \"<p>Tropical forest restoration stands to deliver important conservation gains in lowland Southeast Asia, which has suffered some of the world's highest rates of forest degradation and loss. This promise, however, may be undermined by defaunation driven by ubiquitous wildlife trapping in the region, particularly for forest birds that are part of the multi-million-dollar pet trade. To date, quantification of the impacts of trade-driven trapping on rates of biodiversity recovery from forest restoration has been limited. Here, we use a unique long-term survey dataset to ask how trade-driven trapping may interfere with the expected recovery of avian community under forest restoration, at a flagship ecosystem restoration site in the lowland rainforests of Sumatra, Indonesia. We show that tropical forest restoration is associated with the increases in the abundance of 88% of bird species over time. However, impacts of trapping within more accessible areas of the forest meant that this recovery was dampened for 74% of bird species, relative to levels expected as a result of the magnitude of forest recovery observed. Most species (80%) showed increasingly positive relationships between abundance and site remoteness over the period, a pattern that was found for both species targeted for the pet trade (85% of species) and those trapped opportunistically or as 'bycatch' (78% of species). We emphasize the urgency of tackling the emerging threat of pet trade to Southeast Asia's avian diversity, not least to ensure the effectiveness of efforts towards forest restoration.</p>\", 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.22.457247', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Beyond dimension reduction: Stable electric fields emerge from and allow representational drift', 'authors': \"[{'name': 'Dimitris Pinotsis'}, {'name': 'Earl K Miller'}]\", 'abstract': \"<p>It is known that the exact neurons maintaining a given memory (the neural ensemble) change from trial to trial. This raises the question of how the brain achieves stability in the face of this representational drift.  Here, we demonstrate that this stability emerges at the level of the electric fields that arise from neural activity.  The electric fields, in turn, can act as 'guard rails' that funnel higher dimensional variable neural activity along stable lower dimensional routes. We show that electric fields carry information about working memory content.   We obtained the latent space associated with each memory. We then confirmed the stability of the electric field by mapping the latent space to different cortical patches (that comprise a neural ensemble) and reconstructing information flow between patches.   Stable electric fields can allow latent states to be transferred between brain areas, in accord with modern engram theory.</p>\", 'posted': '2021-08-22'}, {'doi': '10.1101/2021.08.17.21262170', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'The Rapid Assessment of Aggregated Wastewater Samples for Genomic Surveillance of SARS-CoV-2 on a City-Wide Scale', 'authors': \"[{'name': 'Eric Christian Rouchka'}, {'name': 'Julia H. Chariker'}, {'name': 'Kumar Saurabh'}, {'name': 'Sabine Waigel'}, {'name': 'Wolfgang Zacharias'}, {'name': 'Mei Zhang'}, {'name': 'Daymond Talley'}, {'name': 'Ian Santisteban'}, {'name': 'Madeline Puccio'}, {'name': 'Sarah Moyer'}, {'name': 'Rochelle Holm'}, {'name': 'Ray Yeager'}, {'name': 'Kevin J Sokoloski'}, {'name': 'Joshua Fuqua'}, {'name': 'Aruni Bhatnagar'}, {'name': 'Ted Smith'}]\", 'abstract': '<p>Throughout the course of the ongoing SARS-CoV-2 pandemic there has been a need for approaches that enable rapid monitoring of public health using an unbiased and minimally invasive means. A major way this has been accomplished is through the regular assessment of wastewater samples by qRT-PCR to detect the prevalence of viral nucleic acid with respect to time and location. Further expansion of SARS-CoV-2 wastewater monitoring efforts to include the detection of variants of interest / concern through next-generation sequencing have enhanced the understanding of the SARS-CoV-2 outbreak. In this report we detail the results of a collaborative effort between public health and metropolitan wastewater management authorities and the University of Louisville to monitor the SARS-CoV-2 pandemic through the monitoring of aggregate wastewater samples over a period of 28 weeks. Our data indicates that wastewater monitoring of water quality treatment centers and smaller neighborhood-scale catchment areas is a viable means by which the prevalence and genetic variation of SARS-CoV-2 within a metropolitan community of approximately one million individuals may be monitored. Importantly, these efforts confirm that regional emergence and spread of variants of interest / concern may be detected as readily in aggregate wastewater samples as compared to the individual wastewater sheds.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.19.456973', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': True, 'title': 'Modeling Coronavirus Spike Protein Dynamics: Implications for Immunogenicity and Immune Escape', 'authors': \"[{'name': 'G. Kunkel'}, {'name': 'M. Madani'}, {'name': 'S. J. White'}, {'name': 'P. H. Verardi'}, {'name': 'A. Tarakanova'}]\", 'abstract': '<p>The ongoing COVID-19 pandemic is a global public health emergency requiring urgent development of efficacious vaccines. While concentrated research efforts are underway to develop antibody-based vaccines that would neutralize SARS-CoV-2, and several first-generation vaccine candidates are currently in Phase III clinical trials or have received emergency use authorization, it is forecasted that COVID-19 will become an endemic disease requiring second-generation vaccines. The SARS-CoV-2 surface Spike (S) glycoprotein represents a prime target for vaccine development because antibodies that block viral attachment and entry, i.e. neutralizing antibodies, bind almost exclusively to the receptor binding domain (RBD). Here, we develop computational models for a large subset of S proteins associated with SARS-CoV-2, implemented through coarse-grained elastic network models and normal mode analysis. We then analyze local protein domain dynamics of the S protein systems and their thermal stability to characterize structural and dynamical variability among them. These results are compared against existing experimental data, and used to elucidate the impact and mechanisms of SARS-CoV-2 S protein mutations and their associated antibody binding behavior. We construct a SARS-CoV-2 antigenic map and offer predictions about the neutralization capabilities of antibody and S mutant combinations based on protein dynamic signatures. We then compare SARS-CoV-2 S protein dynamics to SARS-CoV and MERS-CoV S proteins to investigate differing antibody binding and cellular fusion mechanisms that may explain the high transmissibility of SARS-CoV-2. The outbreaks associated with SARS-CoV, MERS-CoV, and SARS-CoV-2 over the last two decades suggest that the threat presented by coronaviruses is ever-changing and long-term. Our results provide insights into the dynamics-driven mechanisms of immunogenicity associated with coronavirus S proteins, and present a new approach to characterize and screen potential mutant candidates for immunogen design, as well as to characterize emerging natural variants that may escape vaccine-induced antibody responses.</p><sec><title>STATEMENT OF SIGNIFICANCE</title><p>We present novel dynamic mechanisms of coronavirus S proteins that encode antibody binding and cellular fusion properties. These mechanisms may offer an explanation for the widespread nature of SARS-CoV-2 and more limited spread of SARS-CoV and MERS-CoV. A comprehensive computational characterization of SARS-CoV-2 S protein structures and dynamics provides insights into structural and thermal stability associated with a variety of S protein mutants. These findings allow us to make recommendations about the future mutant design of SARS-CoV-2 S protein variants that are optimized to elicit neutralizing antibodies, resist structural rearrangements that aid cellular fusion, and are thermally stabilized. The integrated computational approach can be applied to optimize vaccine immunogen design and predict escape of vaccine-induced antibody responses by SARS-CoV-2 variants.</p></sec>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.18.456644', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Intracellular Salmonella Paratyphi A is motile and differs in the expression of flagella-chemotaxis, SPI-1 and carbon utilization pathways in comparison to Intracellular S. Typhimurium', 'authors': \"[{'name': 'Helit Cohen'}, {'name': 'Claire Hoede'}, {'name': 'Felix Scharte'}, {'name': 'Charles Coluzzi'}, {'name': 'Emiliano Cohen'}, {'name': 'Inna Shomer'}, {'name': 'Ludovic Mallet'}, {'name': 'Remy Felix Serre'}, {'name': 'Thomas Schiex'}, {'name': 'Isabelle Virlogeux-Payant'}, {'name': 'Guntram Grassl'}, {'name': 'Michael Hensel'}, {'name': 'Helene Chiapello'}, {'name': 'Ohad Gal-Mor'}]\", 'abstract': '<p>Although Salmonella Typhimurium (STM) and Salmonella Paratyphi A (SPA) belong to the same phylogenetic species, share large portion of their genome and express many common virulence factors, they differ vastly in their host specificity, the immune response they elicit, and the clinical manifestations they cause. In this work, we compared for the first time their intracellular trascriptomic architecture and cellular phenotypes during epithelial cell infection. While transcription induction of many metal transport systems, purines, biotin, PhoPQ and SPI-2 regulons was common in both intracellular SPA and STM, we identified 234 differentially expressed genes that showed distinct expression patterns in intracellular SPA vs. STM. Surprisingly, clear expression differences were found in SPI-1, motility and chemotaxis, and carbon (mainly citrate, galactonate and ethanolamine) utilization pathways, indicating that these pathways are regulated and possibly function differently, during their intracellular phase. Moreover, we show that the induction of flagella genes by intracellular SPA leads to cytosolic motility, a conserved trait specific to SPA. To the best of our knowledge, this is the first report of a flagellum-dependent intracellular motility of any Salmonella serovar in living host cells. Importantly, we demonstrate that the elevated expression of SPI-1 and motility genes by intracellular SPA results in increased invasiveness of SPA, following exit from host cells. We propose that such changes prime SPA towards new cycles of host cell infection and contribute to the ability of SPA to disseminate beyond the intestinal lamina propria of the human host, during enteric fever.</p>', 'posted': '2021-08-18'}, {'doi': '10.1101/2021.08.20.457043', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Constitutive signal bias mediated by the human GHRHR splice variant 1', 'authors': \"[{'name': 'Zhaotong Cong'}, {'name': 'Fulai Zhou'}, {'name': 'Chao Zhang'}, {'name': 'Xinyu Zou'}, {'name': 'Huibing Zhang'}, {'name': 'Yuzhe Wang'}, {'name': 'Qingtong Zhou'}, {'name': 'Xiaoqing Cai'}, {'name': 'Qiaofeng Liu'}, {'name': 'Jie Li'}, {'name': 'Lijun Shao'}, {'name': 'Chunyou Mao'}, {'name': 'Xi Wang'}, {'name': 'Jihong Wu'}, {'name': 'Tian Xia'}, {'name': 'Lihua Zhao'}, {'name': 'Hualiang Jiang'}, {'name': 'Yan Zhang'}, {'name': 'H. Eric Xu'}, {'name': 'Xi Cheng'}, {'name': 'Dehua Yang'}, {'name': 'Ming-Wei Wang'}]\", 'abstract': '<p>Alternative splicing of G protein-coupled receptors has been observed, but their functions are largely unknown. Here, we report that a splice variant (SV1) of the human growth hormone releasing hormone receptor (GHRHR) is capable of transducing biased signal. Differing only at the receptor N terminus, GHRHR predominantly activates G<sub>s</sub> while SV1 selectively couples to -arrestins. Based on the cryo-electron microscopy structures of SV1 in the apo state or in complex with the G<sub>s</sub> protein, molecular dynamics simulations reveal that the N termini of GHRHR and SV1 differentiate the downstream signaling pathways, G<sub>s</sub> vs. -arrestins. Suggested by mutagenesis and functional studies, it appears that GHRH-elicited signal bias towards -arrestin recruitment is constitutively mediated by SV1. The level of SV1 expression in prostate cancer cells is also positively correlated with ERK1/2 phosphorylation but negatively correlated with cAMP response. Our findings imply that constitutive signal bias may be a mechanism that ensures cancer cell proliferation.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.20.457086', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Dysfunction of grey matter NG2 glial cells affects neuronal plasticity and behavior', 'authors': \"[{'name': 'Aline Timmermann'}, {'name': 'Ronald Jabs'}, {'name': 'Anne Boehlen'}, {'name': 'Catia Domingos'}, {'name': 'Magdalena Skubal'}, {'name': 'Wenhui Huang'}, {'name': 'Frank Kirchhoff'}, {'name': 'Christian Henneberger'}, {'name': 'Andras Bilkei-Gorzo'}, {'name': 'Gerald Seifert'}, {'name': 'Christian Werner Steinhauser'}]\", 'abstract': '<p>NG2 glia represent a distinct type of macroglial cells in the CNS and are unique among glia because they receive synaptic input from neurons. They are abundantly present in white and grey matter. While the majority of white matter NG2 glia differentiates into oligodendrocytes, the physiological impact of grey matter NG2 glia and their synaptic input are ill defined yet. Here we asked whether dysfunctional NG2 glia affect neuronal signaling and behavior. We generated mice with inducible deletion of the K<sup>+</sup> channel Kir4.1 in NG2 glia and performed comparative electrophysiological, immunohistochemical, molecular and behavioral analyses. Focussing on the hippocampus, we found that loss of the Kir4.1 potentiated synaptic depolarizations of NG2 glia and enhanced the expression of myelin basic protein. Notably, while mice with targeted deletion of the K<sup>+</sup> channel in NG2 glia showed impaired long term potentiation at CA3-CA1 synapses, they demonstrated improved spatial memory as revealed by testing new object location recognition. Our data demonstrate that proper NG2 glia function is critical for normal brain function and behavior.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.20.457062', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Quantification of macromolecular biomass composition for constraint-based metabolic modeling', 'authors': \"[{'name': 'Vetle Simensen'}, {'name': 'Christian Schulz'}, {'name': 'Emil Karlsen'}, {'name': 'Signe Brtelund'}, {'name': 'Idun Burgos'}, {'name': 'Lilja Brekke Thorfinnsdottir'}, {'name': 'Laura Garca-Calvo'}, {'name': 'Per Bruheim'}, {'name': 'Eivind Almaas'}]\", 'abstract': '<p>Genome-scale metabolic models (GEMs) are mathematical representations of metabolism that allow for <italic>in silico</italic> simulation of metabolic phenotypes and capabilities. A prerequisite for these predictions is an accurate representation of the biomolecular composition of the cell necessary for replication and growth, implemented in GEMs as the so-called biomass objective function (BOF). The BOF contains the metabolic precursors required for synthesis of the cellular macro- and micromolecular constituents (e.g. protein, RNA, DNA), and its composition is highly dependent on the particular organism, strain, and growth condition. Despite its critical role, the BOF is rarely constructed using specific measurements of the modeled organism, drawing the validity of this approach into question. Thus, there is a need to establish robust and reliable protocols for experimental condition-specific biomass determination. Here, we address this challenge by presenting a general pipeline for biomass quantification, evaluating its performance on <italic>Escherichia coli</italic> K-12 MG1655 sampled during balanced exponential growth under controlled conditions in a batch-fermentor set-up. We significantly improve both the coverage and molecular resolution compared to previously published workflows, quantifying 91.6% of the biomass. Our measurements display great correspondence with previously reported measurements, and we were also able to detect subtle characteristics specific to the particular <italic>E. coli</italic> strain. Using the modified <italic>E. coli</italic> GEM <italic>i</italic>ML1515a, we compare the feasible flux ranges of our experimentally determined BOF with the original BOF, finding that the changes in BOF coefficients considerably affect the attainable fluxes at the genome-scale.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.16.21262116', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'To share or not to share  What the general public thinks about global COVID-19 vaccine distribution', 'authors': \"[{'name': 'Matthias Klumpp'}, {'name': 'Ida G. Monfared'}, {'name': 'Sebastian Vollmer'}]\", 'abstract': '<p>We conducted representative surveys among the adult population in Germany (n=1,003) and the United States (n=1,000) with discrete choice experiments and analytical hierarchy process items to elicit opinions about global COVID-19 vaccine distribution. Results unveil a high consent towards utilitarian or egalitarian distribution principles, prioritizing countries with weak health care systems and high numbers of current cases or deaths. This implies that the public in Germany and the United States has very well understood the global nature of the pandemic and how important sharing of resources and vaccines really is.</p><sec><title>One-Sentence Summary</title><p>The people in Germany and the United States favor utilitarian or egalitarian distribution principles for global COVID-19 vaccine distribution.</p></sec>', 'posted': '2021-08-18'}, {'doi': '10.1101/2021.08.19.457018', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Cellulose synthase-like D (CSLD) proteins move in the plasma membrane and their targeting to cell tips, but not cell plates, depends on the actin cytoskeleton', 'authors': \"[{'name': 'Shu-Zon Wu'}, {'name': 'Arielle M. Chaves'}, {'name': 'Rongrong Li'}, {'name': 'Magdalena Bezanilla'}, {'name': 'Alison W. Roberts'}]\", 'abstract': '<p>Cellulose Synthase-Like D (CSLD) proteins are implicated in cell wall remodeling during tip growth and cell division in plants, and are known to generate -1,4-glucan. It is unknown whether they form complexes and move in the plasma membrane like members of the Cellulose Synthase (CESA) family. We used the genetically tractable moss <italic>Physcomitrium patens</italic>, which has a filamentous protonemal stage that undergoes both tip growth and cell division and is amenable to high resolution live cell imaging, to investigate CSLD function and intracellular trafficking. <italic>CSLD2</italic> and <italic>CSLD6</italic> are highly expressed in gametophores and are redundantly required for gametophore cellular patterning. Live cell imaging revealed that CSLD6 is also expressed in protonemata where it moves in the plasma membrane and localizes to cell plates and cell tips. Notably, delivery to the apical plasma membrane, but not the cell plate, depends on actin. By comparing the behavior of endogenously tagged CSLD6 and CESA10, we discovered that CSLD6 movements in the plasma membrane were significantly faster, shorter in duration and less linear than CESA10 movements and were insensitive to the cellulose synthesis inhibitor isoxaben. These data suggest that CSLD6 and CESA10 function within different structures and may thus produce structurally distinct cellulose microfibrils.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.16.456538', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Application of components analysis in dynamic thermal breast imaging to identify pathophysiologic mechanisms of heat transfer', 'authors': \"[{'name': 'Meir Gershenson'}, {'name': 'Jonathan Gershenson'}]\", 'abstract': '<sec><title>Significance</title><p>Of the most compelling unsolved issues in the paradigm to create success in the field of breast cancer infrared imaging is localization of direct internal heat of the tumor. The contribution of differential heat production related to metabolism versus perfusion is not understood. Previous work until now has not shown progress beyond identifying veins which are fed by the hot cancer. Employing signal analysis techniques, we probe important questions which may lead to further understanding pathophysiology of heat transfer occurring in the setting of malignancy.</p></sec><sec><title>Aim</title><p>When using thermal imaging to detect breast cancer, the dominant heat signature is that of indirect heat transported in gradient away from the tumor location. Unprocessed images strikingly display vasculature which acts to direct excess heat superficially towards the skin surface before dissipating. In current clinical use, interpretation of thermogram images considers abnormal vascular patterns and overall temperature as indicators of disease. The goal of this work is to present a processing method for dynamic external stimulus thermogram images to isolate and separate the indirect vascular heat while revealing the desired direct heat from the tumor.</p></sec><sec><title>Approach</title><p>In dynamic thermal imaging of the breast, a timed series of images are taken following application of external temperature stimulus (most often cooled air). While the tumor heat response is thought to be independent of the external stimulus, the secondary heat of the veins is known to be affected by vasomodulation. The recorded data is analyzed using independent component analysis (ICA) and principal component analysis (PCA) methods. ICA separates the image sequence into new independent images having a common characteristic time behavior. Resulting individual components are analyzed for correspondence to the presence or lack of vasomodulation.</p></sec><sec><title>Results</title><p>Using the Brazilian visual lab mastology data set containing dynamic thermograms, applying components analysis resulted in three corresponding images: 1. Minimum change as a function of applied temperature or time (suggests correlation with the cancer generated heat), 2. Moderate temperature dependence (suggests correlation with veins affected by vasomodulation) and 3. Complex time behavior (suggests correlation with heat absorption due to high tumor perfusion). All components appear clear and distinct.</p></sec><sec><title>Conclusions</title><p>Applying signal processing methods to the dynamic infrared data, we found three distinct components with correspondence to understood physiologic processes. The two cases shown are self-evident of the capability of the method but are lacking supporting ground truth that is unavailable with such a limited data set. Validation of this proposed paradigm and studying furthering clinical applications has potential to create significant achievement for IR modality in diagnostic imaging.</p></sec>', 'posted': '2021-08-17'}, {'doi': '10.1101/2021.08.19.456809', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Optimized detection of somatic allelic imbalances specific for homologous recombination deficiency improves the prediction of clinical outcomes in ovarian cancer', 'authors': \"[{'name': 'Fernando Perez-Villatoro'}, {'name': 'Jaana Oikkonen'}, {'name': 'Julia Casado'}, {'name': 'Anastasiya Chernenko'}, {'name': 'Doga C. Gulhan'}, {'name': 'Manuela Tumiati'}, {'name': 'Sakari Hietanen'}, {'name': 'Johanna Hynninen'}, {'name': 'Ulla-Maija Haltia'}, {'name': 'Hannele Laivuori'}, {'name': 'Panagiotis Konstantinopoulos'}, {'name': 'Sampsa Hautaniemi'}, {'name': 'Liisa Kauppi'}, {'name': 'Anniina Frkkil'}]\", 'abstract': '<p>Specific patterns of genomic allelic imbalances (AIs) have been associated with Homologous recombination DNA-repair deficiency (HRD). We performed a systematic pan-cancer characterization of AIs across tumor types, revealing unique patterns in ovarian cancer. Using machine learning on a multi-omics dataset, we generated an optimized algorithm to detect HRD in ovarian cancer (ovaHRDscar). ovaHRDscar improved the prediction of clinical outcomes in three independent validation cohorts (PCAWG, HERCULES, TERVA). Characterization of 98 spatiotemporally distinct tumor samples indicated ovary/adnex as the preferred site to assess HRD. In conclusion, ovaHRDscar improves the detection of HRD in ovarian cancer with the premise to improve patient selection for HR-targeted therapies.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.18.456812', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Improved radiation expression profiling by sequential application of sensitive and specific gene signatures', 'authors': \"[{'name': 'Eliseos J Mucaki'}, {'name': 'Ben C Shirley'}, {'name': 'Peter K Rogan'}]\", 'abstract': '<p>Purpose: Combinations of expressed genes can discriminate radiation-exposed from normal control blood samples by machine learning based signatures (with 8 to 20% misclassification rates). These signatures can quantify therapeutically-relevant as well as accidental radiation exposures. The prodromal symptoms of Acute Radiation Syndrome (ARS) overlap some viral infections. We recently showed that these radiation signatures produced unexpected false positive misclassification of influenza and dengue infected samples. The present study investigates these and other confounders, and then mitigates their effects on signature accuracy.\\nMethods: This study investigated recall by previous and novel radiation signatures independently derived from multiple Gene Expression Omnibus datasets on common and rare non-malignant blood disorders and blood-borne infections (thromboembolism, S. aureus bacteremia, malaria, sickle cell disease, polycythemia vera, and aplastic anemia). Normalized expression levels of signature genes are used as input to machine learning-based classifiers to predict radiation exposure in other hematological conditions. \\nResults: Except for aplastic anemia, these confounders modify the normal baseline expression values, leading to false-positive misclassification of radiation exposures in 8 to 54% of individuals. Shared changes were predominantly in DNA damage response and apoptosis-related gene transcripts in radiation and confounding hematological conditions compromise the utility of these signatures for radiation assessment. These confounding conditions are known to induce neutrophil extracellular traps, initiated by chromatin decondensation, fragmentation and often, programmed cell death. Ribovirus infections are proposed to deplete RNA binding proteins, inducing R-loops in chromatin which collide with replication forks resulting in DNA damage and apoptosis. To mitigate the effects of confounders, we evaluated predicted radiation positive samples with novel gene expression signatures derived from radiation-responsive transcripts of secreted blood plasma proteins whose expression levels are unperturbed in these confounding conditions. \\nConclusions: This approach identifies and eliminates misclassified samples with underlying hematological or infectious conditions, leaving only samples with true radiation exposures. Diagnostic accuracy is significantly improved by selecting genes that maximize both sensitivity and specificity in the appropriate tissue using combinations of the best signatures for each of these classes of signatures.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.18.456772', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Multiple parasitism promotes facultative host acceptance of cuckoo eggs and rejection of cuckoo chicks', 'authors': \"[{'name': 'Hee-Jin Noh'}, {'name': 'Ros Gloag'}, {'name': 'Naomi E. Langmore'}]\", 'abstract': '<p>Many hosts of brood parasitic cuckoos reject foreign eggs from the nest. Yet where nests commonly receive more than one cuckoo egg, hosts might benefit by instead accepting parasite eggs. This is because cuckoos remove an egg from the nest before adding their own, and keeping cuckoo eggs in the nest reduces the odds that further host eggs are removed by subsequent cuckoos. This clutch dilution effect has been proposed as a precondition for the evolution of cuckoo nestling eviction by hosts, but no previous studies have tested this in a host that rejects cuckoo nestlings. We tested the clutch dilution hypothesis in large-billed gerygones (<italic>Gerygone magnirostris</italic>), which are multiply parasitized by little bronze-cuckoos (<italic>Chalcites minutillus</italic>). Gerygones evict cuckoo nestlings but accept cuckoo eggs. Consistent with multiple parasitism favouring egg acceptance, we found gerygone egg survival was higher under scenarios of cuckoo egg acceptance than rejection. Yet gerygones were also flexible in their egg acceptance, with 35% abandoning cuckoo-egg-only clutches. This novel demonstration of adaptive clutch dilution suggests that multiple parasitism can favour a facultative response to brood parasite eggs, whereby hosts accept or reject parasite eggs depending on clutch composition.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.17.456507', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'A micro-evolutionary change in target binding sites as key determinant of Ultrabithorax function in Drosophila', 'authors': \"[{'name': 'Soumen Khan'}, {'name': 'Saurabh J. Pradhan'}, {'name': 'Guillaume Giraud'}, {'name': 'Franoise Bleicher'}, {'name': 'Rachel Paul'}, {'name': 'Samir Merabet'}, {'name': 'LS Shashidhara'}]\", 'abstract': '<p>All Hox proteins are known to recognize, in vitro, similar DNA-binding sites containing a TAAT core sequence. This poor DNA-binding specificity is in sharp contrast with their specific functions in vivo. Here we report a new binding motif with TAAAT core sequence to which the Hox protein Ultrabithorax (Ubx) binds with higher affinity and specificity. Using transgenic and luciferase assays, we show that this new motif is critical for Ubx-mediated regulation of a target gene in <italic>Drosophila melanogaster</italic>. Interestingly, this new motif with TAAAT core sequences is not associated with the targets of Ubx in the honeybee, <italic>Apis mellifera</italic>, wherein hindwings are nearly identical to the forewings. We show that introduction of TAAAT motif in the place of TAAT motif is sufficient to bring an enhancer of a wing-promoting gene of <italic>A. mellifera</italic> under the regulation of Ubx. Our results, thus, suggest that binding motifs with a TAAAT core sequence may help identify functionally relevant direct targets of Ubx in <italic>D. melanogaster</italic> and the emergence of these binding sites may be crucial for Hox-mediated morphological changes during insect evolution.</p>', 'posted': '2021-08-17'}, {'doi': '10.1101/2021.08.17.21262183', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'The association between immunosuppressants use and COVID-19 adverse outcome: National COVID-19 cohort in South Korea', 'authors': \"[{'name': 'Hyun Joon Shin'}, {'name': 'Ronald Chow'}, {'name': 'Hyerim Noh'}, {'name': 'Jongseong Lee'}, {'name': 'Jihui Lee'}, {'name': 'Young-Geun Choi'}]\", 'abstract': '<p>Background: There is uncertainty of the effect of immunosuppression, including corticosteroids, before COVID-19 infection on COVID-19 outcomes. The aim of this study was to investigate the relationship between prehospitalization immunosuppressants use (exposure), and COVID-19 patient outcomes. \\n\\nMethods: We conducted a population-based retrospective cohort study using a nationwide healthcare claims database of South Korea as of May 15, 2020. Confirmed COVID-19 infection in hospitalized individuals aged 40 years or older were included for analysis. We defined exposure variable by using inpatient and outpatient prescription records of immunosuppressants from the database. Our primary outcome was a composite endpoint of all-cause death, intensive care unit (ICU) admission, and mechanical ventilation use. Inverse probability of treatment weighting (IPTW)-adjusted logistic regression analyses were used, to estimate odds ratio (OR) and 95% confidence intervals, comparing immunosuppressants users and non-users.\\n \\nFindings: We identified 4,349 patients, for which 1,356 were immunosuppressants users and 2,903 were non-users. Patients who used immunosuppressants were at increased odds of the primary outcome of all-cause death, ICU admission and mechanical ventilation use (IPTW OR 1.32; 95% CI: 1.06 - 1.63). Patients who used corticosteroids were at increased odds of the primary outcome (IPTW OR 1.33; 95% CI: 1.07 - 1.64). \\n\\nInterpretation: We support the latest guidelines from the CDC, that people on immunosuppressants are at high risk of severe COVID-19 and immunocompromised people may need booster COVID-19 vaccinations.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.16.456439', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'SUMO E2/E3 Enzyme Co-Recognize Substrates and Confers High Substrates Specificity', 'authors': \"[{'name': 'Zhehao Xiong'}, {'name': 'Ling Jiang'}, {'name': 'Yan Liu'}, {'name': 'Jun Li'}, {'name': 'Jiayu Liao'}]\", 'abstract': '<p>Although various technologies can determine protein-protein interaction affinity, Kd, current approaches require at least one interacting protein to be purified. Here, we present a development of high-throughput approach to determine protein interaction affinity for un-purified interacting proteins using quantitative FRET assay. We developed this approach first using SUMO E2 conjugating enzyme, Ubc9, and SUMO substrate, RanGap1c. The interaction affinities from two purified proteins and two un-purified proteins in the presence of BSA, bacterial extracts or two mixtures are all in excellent agreement with that obtained from the SPR measurement. We then applied this approach to determine the interaction affinities of SUMO E3 PIAS1 and Ubc9 with a SUMO substrate, influenza virus protein, NS1, and, for the first time, the SUMO E3 ligase-substrate interaction affinity is determined, which enables us to provide a kinetics explanation for the two-enzyme substrate recognition mode. In general, our studies allow high-throughput determination of protein interacting affinity without any purification, and both the approach and scientific discoveries can be applied to proteins that are difficult to be expressed in general.</p>', 'posted': '2021-08-16'}, {'doi': '10.1101/2021.08.19.456943', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Multi view based imaging genetics analysis on Parkinson disease', 'authors': \"[{'name': 'Guglielmo Cerri'}, {'name': 'Manuel Tognon'}, {'name': 'Simone Avesani'}, {'name': 'Neil Oxtoby'}, {'name': 'Andre Altmann'}, {'name': 'Rosalba Giugno'}]\", 'abstract': \"<p>Longitudinal studies integrating imaging and genetic data have recently become widespread among bioinformatics researchers. Combining such heterogeneous data allows a better understanding of complex diseases origins and causes. Through a multi-view based workflow proposal, we show the common steps and tools used in imaging genetics analysis, interpolating genotyping, neuroimaging and transcriptomic data. We describe the advantages of existing methods to analyze heterogeneous datasets,  using Parkinson's Disease (PD) as a case study. Parkinson's disease is associated with both genetic and neuroimaging factors, however such imaging genetics associations are at an early investigation stage. Therefore it is desirable to have a  free and open source workflow that integrates different analysis flows  in order to recover potential genetic biomarkers in PD, as in other complex diseases.</p>\", 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.20.457033', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'GinJinn2: Object detection and segmentation for ecology and evolution', 'authors': \"[{'name': 'Tankred Ott'}, {'name': 'Ulrich Lautenschlager'}]\", 'abstract': '<p>Proper collection and preparation of empirical data still represent one of the most important, but also expensive steps in ecological and evolutionary/systematic research. Modern machine learning approaches, however, have the potential to automate a variety of tasks, which until recently could only be performed manually. Unfortunately, the application of such methods by researchers outside the field is hampered by technical difficulties, some of which, we believe, can be avoided. Here, we present GinJinn2, a user-friendly toolbox for deep learning-based object detection and instance segmentation on image data. Besides providing a convenient command-line interface to existing software libraries, it comprises several additional tools for data handling, pre- and postprocessing, and building advanced analysis pipelines. We demonstrate the application of GinJinn2 for biological purposes using four exemplary analyses, namely the evaluation of seed mixtures, detection of insects on glue traps, segmentation of stomata, and extraction of leaf silhouettes from herbarium specimens. GinJinn2 will enable users with a primary background in biology to apply deep learning-based methods for object detection and segmentation in order to automate feature extraction from image data.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.19.456998', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structures of the Human SPAK and OSR1 Conserved C-Terminal (CCT) Domains', 'authors': \"[{'name': 'Karen T. Elvers'}, {'name': 'Magdalena Lipka-Lloyd'}, {'name': 'Rebecca C. Trueman'}, {'name': 'Benjamin D. Bax'}, {'name': 'Youcef Mehellou'}]\", 'abstract': '<p>STE20/SPS1-related proline/alanine-rich kinase (SPAK) and Oxidative Stress Responsive 1 (OSR1) kinase are two serine/threonine protein kinase that regulate the function of ion co-transporters through phosphorylation. The highly conserved C-terminal (CCT) domains of SPAK and OSR1 bind to RFx[V/I] peptide sequences from their upstream With No Lysine Kinases (WNKs), facilitating their activation via phosphorylation. Thus, the inhibition of SPAK and OSR1 binding, via their CCT domains, to WNK kinases is a plausible strategy for inhibiting SPAK and OSR1 kinases. To facilitate structure-guided drug design of such inhibitors, we expressed and purified human SPAK and OSR1 CCT domains and solved their crystal structures. We also employed a biophysical strategy and determined the affinity of SPAK and OSR1 CCT domains to an 18-mer peptide derived from WNK4. Together, the crystal structures and affinity data reported herein provide a robust platform to facilitate the design of CCT domain specific small molecule inhibitors of SPAK-activation by WNK kinases, potentially leading to new improved treatments for hypertension and ischemic stroke.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.20.456516', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Spatial and temporal expression of PORCN is highly dynamic in the developing mouse cochlea.', 'authors': \"[{'name': 'Brianna L Oliver'}, {'name': 'Caryl A Young'}, {'name': 'Vidhya Munnamalai'}]\", 'abstract': '<p>The mammalian organ of Corti is a highly specialized sensory organ of the cochlea with fine-grained pattern that is essential for auditory function. Previous studies show that the Wnt pathway regulates proliferation, promotes medial compartment formation in the cochlea, differentiation of the mechanosensory hair cells and axon guidance of Type II afferent neurons. WNT ligand expressions are highly dynamic throughout development but are insufficient to explain the roles of the Wnt pathway. We address a potential way for how WNTs specify the medial compartment by characterizing the expression of Porcupine (PORCN), an O-acyltransferase that palmitoylates WNT ligands for secretion. We show PORCN expression across embryonic ages (E)12.5 - E14.5, E16.5, and postnatal day (P)1.  Our results showed enriched PORCN in the medial domains during early stages of development, indicating that WNTs have a stronger influence on patterning of the medial compartment. PORCN was rapidly downregulated after E14.5, following the onset of sensory cell differentiation; residual expression remained in some hair cells and supporting cells. On E14.5 and E16.5, we also examined the spatial expression of Gsk3, an inhibitor of canonical Wnt signaling to determine its potential role in radial patterning of the cochlea. Gsk3 was broadly expressed across the radial axis of the epithelium; therefore, unlikely to control WNT-mediated medial specification. In conclusion, the spatial expression of PORCN enriches WNT secretion from the medial domains of the cochlea to influence the specification of cell fates in the medial sensory domain.</p>', 'posted': '2021-08-20'}, {'doi': '10.1101/2021.08.18.456889', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Does plant diversity determine the diversity of the rhizosphere microbial community?', 'authors': \"[{'name': 'Aleksei Zverev'}, {'name': 'Arina Kichko'}, {'name': 'Vasiliy Shapkin'}, {'name': 'Aleksandr Pinaev'}, {'name': 'Nikolay Provorov'}, {'name': 'Evgeny Andronov'}]\", 'abstract': '<p>The rhizosphere community represents an ecological interface between plant and soil, providing the plant with a number of advantages. Close connection and mutual influence in this communication allow to talk about the self-adjusting plant-rhizosphere community system, which should be be studied in connection. Diversity estimation is one of the ways of describing both bacterial and plant communities. Based on the literature, there are two assumptions of how the diversity of plant communities related to the diversity of bacterial communities: 1) an increase in the species richness of plants leads to an increase in the number of available micro-niches, and increasing of microbial diversity, 2) an increase in the species richness of plants is accompanied by the predominant development of bacteria from highly productive specific taxa and decreasing in the diversity of microorganisms. Experimental studies show controversial results.</p><p>We analyzed field sites (rye crop field and two fallow sites), using DNA isolation of both the plant root mass (followed by sequencing of the ITS1 region) and rhizosphere microorganisms (followed by sequencing of the 16s rDNA V4 region). This allowed us to 1) accurately determine the abundance and taxonomic position of plant communities; 2) extract information about both plant and microbial communities from the same sample.</p><p>There was no correlation between alpha-diversity indices of plants and rhizosphere communities. Alpha-diversity connection should be explored in similar plant communities, such as synusia. We hypothesize, that the significant differences in plant abundances lead to significant changes in exudation profiles, and the loss of diversity connection. The beta-diversity between rhizosphere communities and plant communities is highly correlated, in particular in terms of the abundance of taxa. This can be explained by a potential correlation (as reported in the literature) or by the presence of statistical artifacts.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.18.456900', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A bacterial genome and culture collection of gut microbial in weanling piglet', 'authors': \"[{'name': 'Bo Dong'}, {'name': 'Xiaoqian Lin'}, {'name': 'Xiaohuan Jing'}, {'name': 'Tuanyuan Hu'}, {'name': 'Jianwei Zhou'}, {'name': 'Jianwei Chen'}, {'name': 'Liang Xiao'}, {'name': 'Bo Wang'}, {'name': 'Zhuang Chen'}, {'name': 'Jing Liu'}, {'name': 'Yiyin Hu'}, {'name': 'Guilin Liu'}, {'name': 'Shanshan Liu'}, {'name': 'Junnian Liu'}, {'name': 'Wenkang Wei'}, {'name': 'Yuanqiang Zou'}]\", 'abstract': '<p>The microbiota hosted in the pig gastrointestinal tract are important for productivity of livestock. However, the individual species and functional repertoires that make up the pig gut microbiome remain largely undefined. Here we comprehensively investigated the genomes and functions of the piglet gut microbiome using culture-based and metagenomics approaches. A collection included 266 cultured genomes and 482 metagenome-assembled genomes (MAGs) that were clustered to 428 species across 10 phyla was established. Among these clustered species, 333 genomes represent potential new species. Less matches between cultured genomes and MAGs revealed a substantial bias for the acquisition of reference genomes by the two strategies. Glycoside hydrolases was the dominant category of carbohydrate-active enzymes. 445 secondary metabolite biosynthetic genes were predicted from 292 genomes with bacteriocin being the most. Pan genome analysis of <italic>Limosilactobacillus reuteri</italic> uncover the biosynthesis of reuterin was strain-specific and the production was experimentally determined. These genomic resources will enable a comprehensive characterization of the microbiome composition and function of pig gut.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.16.456579', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Cyclin-G-associated kinase GAK/Aux orchestrates glial autophagy via Atg9 phosphorylation in Parkinsons disease', 'authors': \"[{'name': 'Shiping Zhang'}, {'name': 'Linfang Wang'}, {'name': 'Shuanglong Yi'}, {'name': 'Shuhua Li'}, {'name': 'Honglei Wang'}, {'name': 'Li Song'}, {'name': 'Jiayao Ou'}, {'name': 'Min Zhang'}, {'name': 'Mengxiao Wang'}, {'name': 'Yuchen Zheng'}, {'name': 'Ruiqi Wang'}, {'name': 'Kai Yang'}, {'name': 'Tong Liu'}, {'name': 'Wei Yan'}, {'name': 'Chang Liu'}, {'name': 'Margaret S. Ho'}]\", 'abstract': '<p>Glia serve as double-edged swords to modulate neuropathology in Parkinsons disease (PD), but how they react opposingly to be beneficial or detrimental under pathological conditions, like promote or eliminate -Synuclein inclusions, remain elusive. Here we present evidence that the PD risk factor Cyclin G-associated kinase (GAK)/dAuxilin (Aux) is a new component in glial autophagy. Lack of GAK/Aux promotes autophagic induction, disrupts lysosome acidification, and blocks glial clearance of substrates. Aux regulates Atg9 trafficking to autophagosomes and lysosomes for autophagosome biogenesis and autolysosome maturation, respectively, via Atg9 phosphorylation at newly identified residues T62 and T69. GAK/Aux-mediated glial autophagy is required for -Syn degradation and contributes to a broad spectrum of parkinsonian symptoms in <italic>Drosophila</italic> and mice. Our findings indicate that glial autophagy is a critical component in PD progression and identify a new autophagy factor functioning in a tissue-specific manner; targeting glial autophagy is potentially a therapeutic solution for PD.</p>', 'posted': '2021-08-19'}, {'doi': '10.1101/2021.08.16.456505', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The cytokine receptor Fn14 regulates neuronal transcription during development and brain function in the adult', 'authors': \"[{'name': 'Austin Ferro'}, {'name': 'Uma Vrudhula'}, {'name': 'Yohan S.S. Auguste'}, {'name': 'Lucas Cheadle'}]\", 'abstract': '<p>Cytokine signaling between microglia and neurons is required for synapse elimination during brain development, but the mechanisms by which neurons respond to microglia-derived cytokines remain to be fully defined. Here, we demonstrate that microglia are necessary for sensory experience-dependent synapse elimination in the dorsal lateral geniculate nucleus (dLGN) of the mouse, a process previously shown to be mediated by the microglia-derived cytokine TWEAK and its neuronal receptor Fn14. Single-nucleus RNA-sequencing in mice lacking Fn14 or TWEAK at the height of experience-dependent refinement revealed that TWEAK-Fn14 signaling coordinates robust programs of gene expression in excitatory thalamocortical neurons of the dLGN. Gene targets of TWEAK and Fn14 are enriched for regulators of synapse and chromatin remodeling, suggesting that TWEAK-Fn14 signaling coordinates neuronal transcription to promote circuit refinement and epigenomic maturation in response to experience. We further find that Fn14 expression is not restricted to the dLGN but extends to other brain regions as well, both during development and in the adult. Consistent with the expression of Fn14 outside of the visual system, Fn14 knockout mice show significant impairments in multiple tests of memory task proficiency. At baseline, loss of Fn14 does not affect macroscopic neural activity measured by electroencephalogram recordings <italic>in vivo</italic>. However, mice lacking Fn14 displayed worse seizure outcomes and were less likely to survive when seizures were pharmacologically induced. Thus, TWEAK-Fn14 signaling coordinates neuronal transcription during development and contributes to cognitive function in the adult. Taken together, these data reveal that molecular pathways associated with inflammation in the periphery can coordinate with sensory experience to shape neuronal transcription and connectivity across the lifespan.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Microglia regulate sensory-dependent refinement at the retinogeniculate synapse</p></list-item><list-item><p>TWEAK-Fn14 signaling induces the expression of neuronal genes involved in synaptic and chromatin remodeling</p></list-item><list-item><p>Fn14 signaling is dispensable for learning but required for memory in multiple tasks</p></list-item><list-item><p>Loss of Fn14 exacerbates seizure susceptibility and the severity of seizure outcomes</p></list-item></list></sec>', 'posted': '2021-08-16'}, {'doi': '10.1101/2021.08.15.456413', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Multiple Channel Electrocardiogram QRS Detection by Temporal Pattern Search', 'authors': \"[{'name': 'Bruce Hopenfeld'}]\", 'abstract': '<p>A highly constrained temporal pattern search (TEPS) based multiple channel heartbeat detector is described. TEPS generates sequences of peaks and statistically scores them according to: 1) peak time coherence across channels; 2) peak prominence; 3) temporal regularity; and 4) number of skipped beats. TEPS was tested on 31 records of three channel capacitive electrode data from the UnoViS automobile database. TEPS showed a sensitivity (SE) of 91.3% and a false discovery rate (FDR) of 3.0% compared to an SE and FDR of 75.3% and 65.0% respectively for a conventional single channel detector (OSEA) applied separately to the three channels. The peak matching window was 30ms. The percentage of 5 second segments with average heart rates within 5 beats/minute of reference was also measured. In 6 of the 31 records, TEPS percentage was at least 30% greater than OSEAs. TEPS was also applied to synthetic data comprising a known signal corrupted with calibrated amounts of noise. At a fixed SE of 85%, increasing the number of channels from one to two resulted in an improvement of approximately 5dB in noise resistance, while increasing the number of channels from two to four resulted in an improvement of approximately 3dB in noise resistance. The quantification of noise resistance as a function of the number of channels could help guide the development of wearable electrocardiogram monitors.</p>', 'posted': '2021-08-16'}, {'doi': '10.1101/2021.08.19.456946', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Shaping coral traits: plasticity more than filtering', 'authors': \"[{'name': 'Viviana Brambilla'}, {'name': 'Miguel Barbosa'}, {'name': 'Inga Dehnert'}, {'name': 'Joshua Madin'}, {'name': 'Davide Maggioni'}, {'name': 'Clare Peddie'}, {'name': 'Maria Dornelas'}]\", 'abstract': '<p>The structure of ecosystems is usually determined by the shape of the organisms that build it, commonly known as ecosystem engineers. Understanding to what extent plasticity and environmental filtering determine variation in ecosystem engineer physical structure is necessary to predict how ecosystem structure may change. Here, we explored coral survival and the plasticity of morphological traits that are critical for habitat provision in coral reefs. We conducted a reciprocal clonal transplant experiment in which branching corals from the genus <italic>Porites</italic> and <italic>Acropora</italic> were moved to and from a deep and a shallow site within a lagoon in the Maldives. Survival and trait analyses showed that transplant destination consistently induced the strongest changes, particularly among <italic>Acropora</italic> spp. The origin of the corals only marginally affected some of the traits. We also detected variation in the way individuals from the same species and site differentiate their shape, showing that traits linked to habitat provision are phenotypically plastic. The results suggest coral phenotypic plasticity plays a stronger role than environmental filtering, in determining zonation of coral morphologies, and consequently the habitats they provide for other taxa.</p>', 'posted': '2021-08-19'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.08.19.456982</td>\n",
       "      <td>812</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Somatic mutation rates scale with lifespan acr...</td>\n",
       "      <td>[{'name': 'Alex Cagan'}, {'name': 'Adrian Baez...</td>\n",
       "      <td>&lt;p&gt;The rates and patterns of somatic mutation ...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.08.16.21261912</td>\n",
       "      <td>190</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Genomic profiles of Covishield and Covaxin bre...</td>\n",
       "      <td>[{'name': 'Arup Ghosh'}, {'name': 'Safal Walia...</td>\n",
       "      <td>&lt;p&gt;Vaccine breakthrough infections pose a vast...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.08.15.21262067</td>\n",
       "      <td>102</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Significant reduction in humoral immunity amon...</td>\n",
       "      <td>[{'name': 'David H. Canaday'}, {'name': 'Olada...</td>\n",
       "      <td>&lt;p&gt;High COVID-19 mortality among nursing home ...</td>\n",
       "      <td>2021-08-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.08.16.456551</td>\n",
       "      <td>59</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Tuning levels of low-complexity domain interac...</td>\n",
       "      <td>[{'name': 'Shasha Chong'}, {'name': 'Thomas G....</td>\n",
       "      <td>&lt;p&gt;Gene activation by mammalian transcription ...</td>\n",
       "      <td>2021-08-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.08.20.457082</td>\n",
       "      <td>44</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>A nitrite-oxidizing bacterium constitutively c...</td>\n",
       "      <td>[{'name': 'Pok Man Leung'}, {'name': 'Anne Dae...</td>\n",
       "      <td>&lt;p&gt;Chemolithoautotrophic nitrite-oxidizing bac...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.08.20.457057</td>\n",
       "      <td>42</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>MultiVI: deep generative model for the integra...</td>\n",
       "      <td>[{'name': 'Tal Ashuach'}, {'name': 'Mariano I ...</td>\n",
       "      <td>&lt;p&gt;The ability to jointly profile the transcri...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.08.17.21262195</td>\n",
       "      <td>41</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>COVID-19 Incidence and Hospitalization Rates a...</td>\n",
       "      <td>[{'name': 'Jeffrey E Harris'}]</td>\n",
       "      <td>&lt;p&gt;We tested whether COVID-19 incidence and ho...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.08.20.457079</td>\n",
       "      <td>40</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Humans and mice fluctuate between external and...</td>\n",
       "      <td>[{'name': 'Veith Andreas Weilnhammer'}, {'name...</td>\n",
       "      <td>&lt;p&gt;Perception cycles through periods of enhanc...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.08.21.457186</td>\n",
       "      <td>40</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Evidence of long-term purging of mutation load...</td>\n",
       "      <td>[{'name': 'Andrew D Foote'}, {'name': 'Thomas ...</td>\n",
       "      <td>&lt;p&gt;Genomes of high latitude killer whales harb...</td>\n",
       "      <td>2021-08-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.08.16.456499</td>\n",
       "      <td>36</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>scFlow: A Scalable and Reproducible Analysis P...</td>\n",
       "      <td>[{'name': 'Combiz Khozoie'}, {'name': 'Nurun F...</td>\n",
       "      <td>&lt;p&gt;Advances in single-cell RNA-sequencing tech...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.08.17.456740</td>\n",
       "      <td>33</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Accuracy of multiple sequence alignment method...</td>\n",
       "      <td>[{'name': 'Robert Hubley'}, {'name': 'Travis J...</td>\n",
       "      <td>&lt;p&gt;The construction of a high-quality multiple...</td>\n",
       "      <td>2021-08-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.08.16.456545</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Learning predictive cognitive maps with spikin...</td>\n",
       "      <td>[{'name': 'Jacopo Bono'}, {'name': 'Sara Zanno...</td>\n",
       "      <td>&lt;p&gt;We describe a framework where a biologicall...</td>\n",
       "      <td>2021-08-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.08.19.456935</td>\n",
       "      <td>28</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Characterising the Photoplethysmography Pulse ...</td>\n",
       "      <td>[{'name': 'Simon Williamson'}, {'name': 'Lucie...</td>\n",
       "      <td>&lt;p&gt;Photoplethysmography (PPG) offers a widely-...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.08.18.456813</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Prefrontal cortex neural compensation during a...</td>\n",
       "      <td>[{'name': 'Hayley Fisher'}, {'name': 'Hongyu L...</td>\n",
       "      <td>&lt;p&gt;Deficits in goal-directed action are report...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.08.19.456368</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Signatures of adaptive evolution in platyrrhin...</td>\n",
       "      <td>[{'name': 'Hazel Byrne'}, {'name': 'Timothy H ...</td>\n",
       "      <td>&lt;p&gt;The family Cebidae (capuchin and squirrel m...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.08.16.456345</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Brain transcriptomic profiling reveals common ...</td>\n",
       "      <td>[{'name': 'Iman Sadeghi'}, {'name': 'Juand Dom...</td>\n",
       "      <td>&lt;p&gt;Neurodegenerative and neuropsychiatric diso...</td>\n",
       "      <td>2021-08-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.08.18.456832</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The genome of stress tolerant crop wild relati...</td>\n",
       "      <td>[{'name': 'Guangchao Sun'}, {'name': 'Nishikan...</td>\n",
       "      <td>&lt;p&gt;A number of crop wild relatives can tolerat...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.08.16.456500</td>\n",
       "      <td>18</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>SeqDistK: a Novel Tool for Alignment-free Phyl...</td>\n",
       "      <td>[{'name': 'Xuemei Liu'}, {'name': 'Wen Li'}, {...</td>\n",
       "      <td>&lt;p&gt;Algorithms for constructing phylogenetic tr...</td>\n",
       "      <td>2021-08-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.08.17.456620</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Immunopeptidomic analysis of influenza A virus...</td>\n",
       "      <td>[{'name': 'Ben Nicholas'}, {'name': 'Alistair ...</td>\n",
       "      <td>&lt;p&gt;CD8+ and CD4+ T cells provide cell-mediated...</td>\n",
       "      <td>2021-08-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.08.16.456482</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Particle Uptake Driven Phagocytosis in Macroph...</td>\n",
       "      <td>[{'name': 'Preeti Sharma'}, {'name': 'Anjali V...</td>\n",
       "      <td>&lt;p&gt;Humans are exposed to numerous synthetic fo...</td>\n",
       "      <td>2021-08-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.08.20.457167</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Evolving a generalist biosensor for bicyclic m...</td>\n",
       "      <td>[{'name': \"Simon d'Oelsnitz\"}, {'name': 'Vylan...</td>\n",
       "      <td>&lt;p&gt;Prokaryotic transcription factors can be re...</td>\n",
       "      <td>2021-08-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.08.18.456823</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Divergent transcriptional and transforming pro...</td>\n",
       "      <td>[{'name': 'Line Manceau'}, {'name': 'Julien Ri...</td>\n",
       "      <td>&lt;p&gt;The hallmarks of the alveolar subclass of r...</td>\n",
       "      <td>2021-08-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.08.16.456438</td>\n",
       "      <td>13</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Genetic analysis of early phenology in lentil ...</td>\n",
       "      <td>[{'name': 'Vinodan Rajandran'}, {'name': 'Raul...</td>\n",
       "      <td>&lt;p&gt;Reproductive phenology is well known to be ...</td>\n",
       "      <td>2021-08-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.08.20.457146</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 hijacks p38/MAPK11 to promote vira...</td>\n",
       "      <td>[{'name': 'Christina A Higgins'}, {'name': 'Be...</td>\n",
       "      <td>&lt;p&gt;SARS-CoV-2, the causative agent of the COVI...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.08.19.457007</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Methods for recording and imaging spreading de...</td>\n",
       "      <td>[{'name': 'Yi-Ling Lu'}, {'name': 'Helen E. Sc...</td>\n",
       "      <td>&lt;p&gt;Spreading depolarization (SD) is a sudden a...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.08.16.456464</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>RSC-mediated nucleosome positioning and GRFs f...</td>\n",
       "      <td>[{'name': 'Andrew Wu'}, {'name': 'Claudia Vivo...</td>\n",
       "      <td>&lt;p&gt;The directionality of gene promoters - the ...</td>\n",
       "      <td>2021-08-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.08.18.456372</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Identification and comparative analysis of lon...</td>\n",
       "      <td>[{'name': 'Cheng-Hung Tsai'}, {'name': 'Tzu-Ch...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Many long non...</td>\n",
       "      <td>2021-08-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.08.17.21262177</td>\n",
       "      <td>8</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Intensive Care and Critical Care Medicine</td>\n",
       "      <td>True</td>\n",
       "      <td>Do Covid-19 patients needing ICU admission hav...</td>\n",
       "      <td>[{'name': 'Suleyman Yildirim'}, {'name': 'Sehe...</td>\n",
       "      <td>&lt;p&gt;Introduction \\nStudies focus on pathogenesi...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.08.18.456871</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>RRNPP_detector: a tool to detect RRNPP quorum ...</td>\n",
       "      <td>[{'name': 'Charles Bernard'}, {'name': 'Yanyan...</td>\n",
       "      <td>&lt;p&gt;Gram-positive bacteria (e.g. Firmicutes) an...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.08.21.457182</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Decoding cellular deformation from pseudo-simu...</td>\n",
       "      <td>[{'name': 'Katsuyuki Kunida'}, {'name': 'Nobuh...</td>\n",
       "      <td>&lt;p&gt;The inability to simultaneously observe all...</td>\n",
       "      <td>2021-08-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.08.20.457124</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Spacer2PAM: A computational framework for iden...</td>\n",
       "      <td>[{'name': 'Grant A Rybnicky'}, {'name': 'Nicho...</td>\n",
       "      <td>&lt;p&gt;RNA-guided nucleases from clustered regular...</td>\n",
       "      <td>2021-08-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.08.19.457008</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Tools and applications for integrative analysi...</td>\n",
       "      <td>[{'name': 'Claire Morandin'}, {'name': 'Volker...</td>\n",
       "      <td>&lt;p&gt;DNA methylation is a common epigenetic sign...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.08.19.21262314</td>\n",
       "      <td>7</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Intensive Care and Critical Care Medicine</td>\n",
       "      <td>True</td>\n",
       "      <td>Inhaled nitric oxide use in COVID19-induced hy...</td>\n",
       "      <td>[{'name': 'Abhishek R. Giri'}, {'name': 'Siva ...</td>\n",
       "      <td>&lt;p&gt;Introduction: Nitric Oxide (NO) is an endog...</td>\n",
       "      <td>2021-08-21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.08.20.457106</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Trade-driven trapping dampens the biodiversity...</td>\n",
       "      <td>[{'name': 'H S Sathya Chandra Sagar'}, {'name'...</td>\n",
       "      <td>&lt;p&gt;Tropical forest restoration stands to deliv...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.08.22.457247</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Beyond dimension reduction: Stable electric fi...</td>\n",
       "      <td>[{'name': 'Dimitris Pinotsis'}, {'name': 'Earl...</td>\n",
       "      <td>&lt;p&gt;It is known that the exact neurons maintain...</td>\n",
       "      <td>2021-08-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.08.17.21262170</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>The Rapid Assessment of Aggregated Wastewater ...</td>\n",
       "      <td>[{'name': 'Eric Christian Rouchka'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Throughout the course of the ongoing SARS-C...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.08.19.456973</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>True</td>\n",
       "      <td>Modeling Coronavirus Spike Protein Dynamics: I...</td>\n",
       "      <td>[{'name': 'G. Kunkel'}, {'name': 'M. Madani'},...</td>\n",
       "      <td>&lt;p&gt;The ongoing COVID-19 pandemic is a global p...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.08.18.456644</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Intracellular Salmonella Paratyphi A is motile...</td>\n",
       "      <td>[{'name': 'Helit Cohen'}, {'name': 'Claire Hoe...</td>\n",
       "      <td>&lt;p&gt;Although Salmonella Typhimurium (STM) and S...</td>\n",
       "      <td>2021-08-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.08.20.457043</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Constitutive signal bias mediated by the human...</td>\n",
       "      <td>[{'name': 'Zhaotong Cong'}, {'name': 'Fulai Zh...</td>\n",
       "      <td>&lt;p&gt;Alternative splicing of G protein-coupled r...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.08.20.457086</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Dysfunction of grey matter NG2 glial cells aff...</td>\n",
       "      <td>[{'name': 'Aline Timmermann'}, {'name': 'Ronal...</td>\n",
       "      <td>&lt;p&gt;NG2 glia represent a distinct type of macro...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.08.20.457062</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Quantification of macromolecular biomass compo...</td>\n",
       "      <td>[{'name': 'Vetle Simensen'}, {'name': 'Christi...</td>\n",
       "      <td>&lt;p&gt;Genome-scale metabolic models (GEMs) are ma...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.08.16.21262116</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>To share or not to share  What the general pu...</td>\n",
       "      <td>[{'name': 'Matthias Klumpp'}, {'name': 'Ida G....</td>\n",
       "      <td>&lt;p&gt;We conducted representative surveys among t...</td>\n",
       "      <td>2021-08-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.08.19.457018</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Cellulose synthase-like D (CSLD) proteins move...</td>\n",
       "      <td>[{'name': 'Shu-Zon Wu'}, {'name': 'Arielle M. ...</td>\n",
       "      <td>&lt;p&gt;Cellulose Synthase-Like D (CSLD) proteins a...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.08.16.456538</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Application of components analysis in dynamic ...</td>\n",
       "      <td>[{'name': 'Meir Gershenson'}, {'name': 'Jonath...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Significance&lt;/title&gt;&lt;p&gt;Of the most...</td>\n",
       "      <td>2021-08-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.08.19.456809</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Optimized detection of somatic allelic imbalan...</td>\n",
       "      <td>[{'name': 'Fernando Perez-Villatoro'}, {'name'...</td>\n",
       "      <td>&lt;p&gt;Specific patterns of genomic allelic imbala...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.08.18.456812</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Improved radiation expression profiling by seq...</td>\n",
       "      <td>[{'name': 'Eliseos J Mucaki'}, {'name': 'Ben C...</td>\n",
       "      <td>&lt;p&gt;Purpose: Combinations of expressed genes ca...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.08.18.456772</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Animal Behavior and Cognition</td>\n",
       "      <td>False</td>\n",
       "      <td>Multiple parasitism promotes facultative host ...</td>\n",
       "      <td>[{'name': 'Hee-Jin Noh'}, {'name': 'Ros Gloag'...</td>\n",
       "      <td>&lt;p&gt;Many hosts of brood parasitic cuckoos rejec...</td>\n",
       "      <td>2021-08-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.08.17.456507</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A micro-evolutionary change in target binding ...</td>\n",
       "      <td>[{'name': 'Soumen Khan'}, {'name': 'Saurabh J....</td>\n",
       "      <td>&lt;p&gt;All Hox proteins are known to recognize, in...</td>\n",
       "      <td>2021-08-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.08.17.21262183</td>\n",
       "      <td>4</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>The association between immunosuppressants use...</td>\n",
       "      <td>[{'name': 'Hyun Joon Shin'}, {'name': 'Ronald ...</td>\n",
       "      <td>&lt;p&gt;Background: There is uncertainty of the eff...</td>\n",
       "      <td>2021-08-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.08.16.456439</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>SUMO E2/E3 Enzyme Co-Recognize Substrates and ...</td>\n",
       "      <td>[{'name': 'Zhehao Xiong'}, {'name': 'Ling Jian...</td>\n",
       "      <td>&lt;p&gt;Although various technologies can determine...</td>\n",
       "      <td>2021-08-16</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.08.19.456982     812  bioRxiv   \n",
       "1   10.1101/2021.08.16.21261912     190  medRxiv   \n",
       "2   10.1101/2021.08.15.21262067     102  medRxiv   \n",
       "3     10.1101/2021.08.16.456551      59  bioRxiv   \n",
       "4     10.1101/2021.08.20.457082      44  bioRxiv   \n",
       "5     10.1101/2021.08.20.457057      42  bioRxiv   \n",
       "6   10.1101/2021.08.17.21262195      41  medRxiv   \n",
       "7     10.1101/2021.08.20.457079      40  bioRxiv   \n",
       "8     10.1101/2021.08.21.457186      40  bioRxiv   \n",
       "9     10.1101/2021.08.16.456499      36  bioRxiv   \n",
       "10    10.1101/2021.08.17.456740      33  bioRxiv   \n",
       "11    10.1101/2021.08.16.456545      29  bioRxiv   \n",
       "12    10.1101/2021.08.19.456935      28  bioRxiv   \n",
       "13    10.1101/2021.08.18.456813      23  bioRxiv   \n",
       "14    10.1101/2021.08.19.456368      22  bioRxiv   \n",
       "15    10.1101/2021.08.16.456345      21  bioRxiv   \n",
       "16    10.1101/2021.08.18.456832      19  bioRxiv   \n",
       "17    10.1101/2021.08.16.456500      18  bioRxiv   \n",
       "18    10.1101/2021.08.17.456620      16  bioRxiv   \n",
       "19    10.1101/2021.08.16.456482      15  bioRxiv   \n",
       "20    10.1101/2021.08.20.457167      14  bioRxiv   \n",
       "21    10.1101/2021.08.18.456823      14  bioRxiv   \n",
       "22    10.1101/2021.08.16.456438      13  bioRxiv   \n",
       "23    10.1101/2021.08.20.457146      12  bioRxiv   \n",
       "24    10.1101/2021.08.19.457007      10  bioRxiv   \n",
       "25    10.1101/2021.08.16.456464      10  bioRxiv   \n",
       "26    10.1101/2021.08.18.456372       8  bioRxiv   \n",
       "27  10.1101/2021.08.17.21262177       8  medRxiv   \n",
       "28    10.1101/2021.08.18.456871       8  bioRxiv   \n",
       "29    10.1101/2021.08.21.457182       8  bioRxiv   \n",
       "30    10.1101/2021.08.20.457124       7  bioRxiv   \n",
       "31    10.1101/2021.08.19.457008       7  bioRxiv   \n",
       "32  10.1101/2021.08.19.21262314       7  medRxiv   \n",
       "33    10.1101/2021.08.20.457106       7  bioRxiv   \n",
       "34    10.1101/2021.08.22.457247       6  bioRxiv   \n",
       "35  10.1101/2021.08.17.21262170       6  medRxiv   \n",
       "36    10.1101/2021.08.19.456973       6  bioRxiv   \n",
       "37    10.1101/2021.08.18.456644       6  bioRxiv   \n",
       "38    10.1101/2021.08.20.457043       6  bioRxiv   \n",
       "39    10.1101/2021.08.20.457086       6  bioRxiv   \n",
       "40    10.1101/2021.08.20.457062       6  bioRxiv   \n",
       "41  10.1101/2021.08.16.21262116       6  medRxiv   \n",
       "42    10.1101/2021.08.19.457018       5  bioRxiv   \n",
       "43    10.1101/2021.08.16.456538       5  bioRxiv   \n",
       "44    10.1101/2021.08.19.456809       4  bioRxiv   \n",
       "45    10.1101/2021.08.18.456812       4  bioRxiv   \n",
       "46    10.1101/2021.08.18.456772       4  bioRxiv   \n",
       "47    10.1101/2021.08.17.456507       4  bioRxiv   \n",
       "48  10.1101/2021.08.17.21262183       4  medRxiv   \n",
       "49    10.1101/2021.08.16.456439       4  bioRxiv   \n",
       "\n",
       "                                 subject-area  covid  \\\n",
       "0                                    Genomics  False   \n",
       "1       Infectious Diseases (except HIV/AIDS)   True   \n",
       "2       Infectious Diseases (except HIV/AIDS)   True   \n",
       "3                           Molecular Biology  False   \n",
       "4                                Microbiology  False   \n",
       "5                              Bioinformatics  False   \n",
       "6                                Epidemiology   True   \n",
       "7                                Neuroscience  False   \n",
       "8                        Evolutionary Biology  False   \n",
       "9                              Bioinformatics  False   \n",
       "10                                   Genomics  False   \n",
       "11                               Neuroscience  False   \n",
       "12                               Neuroscience  False   \n",
       "13                               Neuroscience  False   \n",
       "14                                   Genomics  False   \n",
       "15                               Neuroscience  False   \n",
       "16                              Plant Biology  False   \n",
       "17                             Bioinformatics  False   \n",
       "18                                 Immunology  False   \n",
       "19                                 Immunology  False   \n",
       "20                          Synthetic Biology  False   \n",
       "21                             Cancer Biology  False   \n",
       "22                                   Genetics  False   \n",
       "23                               Microbiology   True   \n",
       "24                               Neuroscience  False   \n",
       "25                          Molecular Biology  False   \n",
       "26                                   Genomics  False   \n",
       "27  Intensive Care and Critical Care Medicine   True   \n",
       "28                             Bioinformatics  False   \n",
       "29                            Systems Biology  False   \n",
       "30                          Synthetic Biology  False   \n",
       "31                             Bioinformatics  False   \n",
       "32  Intensive Care and Critical Care Medicine   True   \n",
       "33                                    Ecology  False   \n",
       "34                               Neuroscience  False   \n",
       "35                               Epidemiology   True   \n",
       "36                                 Biophysics   True   \n",
       "37                               Microbiology  False   \n",
       "38                          Molecular Biology  False   \n",
       "39                               Neuroscience  False   \n",
       "40                            Systems Biology  False   \n",
       "41                   Public and Global Health   True   \n",
       "42                              Plant Biology  False   \n",
       "43                                 Biophysics  False   \n",
       "44                             Cancer Biology  False   \n",
       "45                            Systems Biology  False   \n",
       "46              Animal Behavior and Cognition  False   \n",
       "47                      Developmental Biology  False   \n",
       "48      Infectious Diseases (except HIV/AIDS)   True   \n",
       "49                               Biochemistry  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Somatic mutation rates scale with lifespan acr...   \n",
       "1   Genomic profiles of Covishield and Covaxin bre...   \n",
       "2   Significant reduction in humoral immunity amon...   \n",
       "3   Tuning levels of low-complexity domain interac...   \n",
       "4   A nitrite-oxidizing bacterium constitutively c...   \n",
       "5   MultiVI: deep generative model for the integra...   \n",
       "6   COVID-19 Incidence and Hospitalization Rates a...   \n",
       "7   Humans and mice fluctuate between external and...   \n",
       "8   Evidence of long-term purging of mutation load...   \n",
       "9   scFlow: A Scalable and Reproducible Analysis P...   \n",
       "10  Accuracy of multiple sequence alignment method...   \n",
       "11  Learning predictive cognitive maps with spikin...   \n",
       "12  Characterising the Photoplethysmography Pulse ...   \n",
       "13  Prefrontal cortex neural compensation during a...   \n",
       "14  Signatures of adaptive evolution in platyrrhin...   \n",
       "15  Brain transcriptomic profiling reveals common ...   \n",
       "16  The genome of stress tolerant crop wild relati...   \n",
       "17  SeqDistK: a Novel Tool for Alignment-free Phyl...   \n",
       "18  Immunopeptidomic analysis of influenza A virus...   \n",
       "19  Particle Uptake Driven Phagocytosis in Macroph...   \n",
       "20  Evolving a generalist biosensor for bicyclic m...   \n",
       "21  Divergent transcriptional and transforming pro...   \n",
       "22  Genetic analysis of early phenology in lentil ...   \n",
       "23  SARS-CoV-2 hijacks p38/MAPK11 to promote vira...   \n",
       "24  Methods for recording and imaging spreading de...   \n",
       "25  RSC-mediated nucleosome positioning and GRFs f...   \n",
       "26  Identification and comparative analysis of lon...   \n",
       "27  Do Covid-19 patients needing ICU admission hav...   \n",
       "28  RRNPP_detector: a tool to detect RRNPP quorum ...   \n",
       "29  Decoding cellular deformation from pseudo-simu...   \n",
       "30  Spacer2PAM: A computational framework for iden...   \n",
       "31  Tools and applications for integrative analysi...   \n",
       "32  Inhaled nitric oxide use in COVID19-induced hy...   \n",
       "33  Trade-driven trapping dampens the biodiversity...   \n",
       "34  Beyond dimension reduction: Stable electric fi...   \n",
       "35  The Rapid Assessment of Aggregated Wastewater ...   \n",
       "36  Modeling Coronavirus Spike Protein Dynamics: I...   \n",
       "37  Intracellular Salmonella Paratyphi A is motile...   \n",
       "38  Constitutive signal bias mediated by the human...   \n",
       "39  Dysfunction of grey matter NG2 glial cells aff...   \n",
       "40  Quantification of macromolecular biomass compo...   \n",
       "41  To share or not to share  What the general pu...   \n",
       "42  Cellulose synthase-like D (CSLD) proteins move...   \n",
       "43  Application of components analysis in dynamic ...   \n",
       "44  Optimized detection of somatic allelic imbalan...   \n",
       "45  Improved radiation expression profiling by seq...   \n",
       "46  Multiple parasitism promotes facultative host ...   \n",
       "47  A micro-evolutionary change in target binding ...   \n",
       "48  The association between immunosuppressants use...   \n",
       "49  SUMO E2/E3 Enzyme Co-Recognize Substrates and ...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Alex Cagan'}, {'name': 'Adrian Baez...   \n",
       "1   [{'name': 'Arup Ghosh'}, {'name': 'Safal Walia...   \n",
       "2   [{'name': 'David H. Canaday'}, {'name': 'Olada...   \n",
       "3   [{'name': 'Shasha Chong'}, {'name': 'Thomas G....   \n",
       "4   [{'name': 'Pok Man Leung'}, {'name': 'Anne Dae...   \n",
       "5   [{'name': 'Tal Ashuach'}, {'name': 'Mariano I ...   \n",
       "6                      [{'name': 'Jeffrey E Harris'}]   \n",
       "7   [{'name': 'Veith Andreas Weilnhammer'}, {'name...   \n",
       "8   [{'name': 'Andrew D Foote'}, {'name': 'Thomas ...   \n",
       "9   [{'name': 'Combiz Khozoie'}, {'name': 'Nurun F...   \n",
       "10  [{'name': 'Robert Hubley'}, {'name': 'Travis J...   \n",
       "11  [{'name': 'Jacopo Bono'}, {'name': 'Sara Zanno...   \n",
       "12  [{'name': 'Simon Williamson'}, {'name': 'Lucie...   \n",
       "13  [{'name': 'Hayley Fisher'}, {'name': 'Hongyu L...   \n",
       "14  [{'name': 'Hazel Byrne'}, {'name': 'Timothy H ...   \n",
       "15  [{'name': 'Iman Sadeghi'}, {'name': 'Juand Dom...   \n",
       "16  [{'name': 'Guangchao Sun'}, {'name': 'Nishikan...   \n",
       "17  [{'name': 'Xuemei Liu'}, {'name': 'Wen Li'}, {...   \n",
       "18  [{'name': 'Ben Nicholas'}, {'name': 'Alistair ...   \n",
       "19  [{'name': 'Preeti Sharma'}, {'name': 'Anjali V...   \n",
       "20  [{'name': \"Simon d'Oelsnitz\"}, {'name': 'Vylan...   \n",
       "21  [{'name': 'Line Manceau'}, {'name': 'Julien Ri...   \n",
       "22  [{'name': 'Vinodan Rajandran'}, {'name': 'Raul...   \n",
       "23  [{'name': 'Christina A Higgins'}, {'name': 'Be...   \n",
       "24  [{'name': 'Yi-Ling Lu'}, {'name': 'Helen E. Sc...   \n",
       "25  [{'name': 'Andrew Wu'}, {'name': 'Claudia Vivo...   \n",
       "26  [{'name': 'Cheng-Hung Tsai'}, {'name': 'Tzu-Ch...   \n",
       "27  [{'name': 'Suleyman Yildirim'}, {'name': 'Sehe...   \n",
       "28  [{'name': 'Charles Bernard'}, {'name': 'Yanyan...   \n",
       "29  [{'name': 'Katsuyuki Kunida'}, {'name': 'Nobuh...   \n",
       "30  [{'name': 'Grant A Rybnicky'}, {'name': 'Nicho...   \n",
       "31  [{'name': 'Claire Morandin'}, {'name': 'Volker...   \n",
       "32  [{'name': 'Abhishek R. Giri'}, {'name': 'Siva ...   \n",
       "33  [{'name': 'H S Sathya Chandra Sagar'}, {'name'...   \n",
       "34  [{'name': 'Dimitris Pinotsis'}, {'name': 'Earl...   \n",
       "35  [{'name': 'Eric Christian Rouchka'}, {'name': ...   \n",
       "36  [{'name': 'G. Kunkel'}, {'name': 'M. Madani'},...   \n",
       "37  [{'name': 'Helit Cohen'}, {'name': 'Claire Hoe...   \n",
       "38  [{'name': 'Zhaotong Cong'}, {'name': 'Fulai Zh...   \n",
       "39  [{'name': 'Aline Timmermann'}, {'name': 'Ronal...   \n",
       "40  [{'name': 'Vetle Simensen'}, {'name': 'Christi...   \n",
       "41  [{'name': 'Matthias Klumpp'}, {'name': 'Ida G....   \n",
       "42  [{'name': 'Shu-Zon Wu'}, {'name': 'Arielle M. ...   \n",
       "43  [{'name': 'Meir Gershenson'}, {'name': 'Jonath...   \n",
       "44  [{'name': 'Fernando Perez-Villatoro'}, {'name'...   \n",
       "45  [{'name': 'Eliseos J Mucaki'}, {'name': 'Ben C...   \n",
       "46  [{'name': 'Hee-Jin Noh'}, {'name': 'Ros Gloag'...   \n",
       "47  [{'name': 'Soumen Khan'}, {'name': 'Saurabh J....   \n",
       "48  [{'name': 'Hyun Joon Shin'}, {'name': 'Ronald ...   \n",
       "49  [{'name': 'Zhehao Xiong'}, {'name': 'Ling Jian...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>The rates and patterns of somatic mutation ...  2021-08-19  \n",
       "1   <p>Vaccine breakthrough infections pose a vast...  2021-08-20  \n",
       "2   <p>High COVID-19 mortality among nursing home ...  2021-08-18  \n",
       "3   <p>Gene activation by mammalian transcription ...  2021-08-17  \n",
       "4   <p>Chemolithoautotrophic nitrite-oxidizing bac...  2021-08-20  \n",
       "5   <p>The ability to jointly profile the transcri...  2021-08-20  \n",
       "6   <p>We tested whether COVID-19 incidence and ho...  2021-08-20  \n",
       "7   <p>Perception cycles through periods of enhanc...  2021-08-20  \n",
       "8   <p>Genomes of high latitude killer whales harb...  2021-08-21  \n",
       "9   <p>Advances in single-cell RNA-sequencing tech...  2021-08-19  \n",
       "10  <p>The construction of a high-quality multiple...  2021-08-18  \n",
       "11  <p>We describe a framework where a biologicall...  2021-08-17  \n",
       "12  <p>Photoplethysmography (PPG) offers a widely-...  2021-08-19  \n",
       "13  <p>Deficits in goal-directed action are report...  2021-08-19  \n",
       "14  <p>The family Cebidae (capuchin and squirrel m...  2021-08-20  \n",
       "15  <p>Neurodegenerative and neuropsychiatric diso...  2021-08-16  \n",
       "16  <p>A number of crop wild relatives can tolerat...  2021-08-19  \n",
       "17  <p>Algorithms for constructing phylogenetic tr...  2021-08-17  \n",
       "18  <p>CD8+ and CD4+ T cells provide cell-mediated...  2021-08-17  \n",
       "19  <p>Humans are exposed to numerous synthetic fo...  2021-08-16  \n",
       "20  <p>Prokaryotic transcription factors can be re...  2021-08-21  \n",
       "21  <p>The hallmarks of the alveolar subclass of r...  2021-08-18  \n",
       "22  <p>Reproductive phenology is well known to be ...  2021-08-16  \n",
       "23  <p>SARS-CoV-2, the causative agent of the COVI...  2021-08-20  \n",
       "24  <p>Spreading depolarization (SD) is a sudden a...  2021-08-19  \n",
       "25  <p>The directionality of gene promoters - the ...  2021-08-16  \n",
       "26  <sec><title>Background</title><p>Many long non...  2021-08-18  \n",
       "27  <p>Introduction \\nStudies focus on pathogenesi...  2021-08-20  \n",
       "28  <p>Gram-positive bacteria (e.g. Firmicutes) an...  2021-08-19  \n",
       "29  <p>The inability to simultaneously observe all...  2021-08-21  \n",
       "30  <p>RNA-guided nucleases from clustered regular...  2021-08-21  \n",
       "31  <p>DNA methylation is a common epigenetic sign...  2021-08-19  \n",
       "32  <p>Introduction: Nitric Oxide (NO) is an endog...  2021-08-21  \n",
       "33  <p>Tropical forest restoration stands to deliv...  2021-08-20  \n",
       "34  <p>It is known that the exact neurons maintain...  2021-08-22  \n",
       "35  <p>Throughout the course of the ongoing SARS-C...  2021-08-20  \n",
       "36  <p>The ongoing COVID-19 pandemic is a global p...  2021-08-19  \n",
       "37  <p>Although Salmonella Typhimurium (STM) and S...  2021-08-18  \n",
       "38  <p>Alternative splicing of G protein-coupled r...  2021-08-20  \n",
       "39  <p>NG2 glia represent a distinct type of macro...  2021-08-20  \n",
       "40  <p>Genome-scale metabolic models (GEMs) are ma...  2021-08-20  \n",
       "41  <p>We conducted representative surveys among t...  2021-08-18  \n",
       "42  <p>Cellulose Synthase-Like D (CSLD) proteins a...  2021-08-19  \n",
       "43  <sec><title>Significance</title><p>Of the most...  2021-08-17  \n",
       "44  <p>Specific patterns of genomic allelic imbala...  2021-08-19  \n",
       "45  <p>Purpose: Combinations of expressed genes ca...  2021-08-19  \n",
       "46  <p>Many hosts of brood parasitic cuckoos rejec...  2021-08-19  \n",
       "47  <p>All Hox proteins are known to recognize, in...  2021-08-17  \n",
       "48  <p>Background: There is uncertainty of the eff...  2021-08-20  \n",
       "49  <p>Although various technologies can determine...  2021-08-16  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "60 preprints (including 10 covering SARS-CoV-2, 52 from bioRxiv and 8 from medRxiv) published in the last 7 days before 2021-08-23 had been tweeted at least 3 times (maximum 812)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.5"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
