{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&obj-id.prefix=10.1101&source=twitter&rows=0&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(email = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 0,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15954 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=c778b3d2-1b13-47d7-8287-8e0a5102902b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=3c5841c6-4402-4e71-b0a7-4a9f9fa20b71&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=8685d638-5cb0-4832-831f-538bf199c663&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=a36bbe9a-4d2d-4dd2-9e57-110850dcd7e1&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=94041289-5445-4655-8b0c-9065d03ca88d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=65f15744-3732-4890-ac41-1d36ac846071&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=8c41bafe-9b0f-4242-bbd0-5f9bf19fd4b6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=6df34529-630e-4f5f-b7b0-c47e45162dab&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=43be25e9-db73-41f6-94e8-752c017bbefb&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=b6fb38d0-a433-4ca2-9ef3-20a4b1dff4d4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=a4b29616-595c-4e62-b43c-621fe2108d48&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=12c7aa2c-e28f-428b-9e46-2faee62a112e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=f5823781-a509-4fd2-8125-1746e0a9d845&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=9d387e13-5fd5-4071-9235-421d1bbec089&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=e47c583b-ab76-431d-b5e2-3dfd9965013b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-22&until-occurred-date=2021-07-29\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(email = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.451321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.446289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.434169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.26.453748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21256661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.21257766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.27.401232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261041...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.429735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453458...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.434227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21254060...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.21261093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.355651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.453258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.31.928630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.21259318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/658930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.440404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.388819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.20109231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.347740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.17.440282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/794305...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21253964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.28.065714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.443368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21250080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.270371.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.429147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.426881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.21257017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.03.003178...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.30.424837...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.03.282186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.20249026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275303.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.266429.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453674...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.445318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20234369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21250921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.21259977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.30.070094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.444885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20159251...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.372524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.05.446432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.452477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.20167932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257738...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.389544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260171...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21257602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.21255913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.24.962316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.26.965202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.27.20045427...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275188.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.21252597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/710624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/507590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256982...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.16.21260079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.20248159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21258785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.22.392217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453512...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.12.380386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.20243949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.19.20248484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.427855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.21251664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.265058.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.447341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21256456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/550640...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.29.069344...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20131243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.31.275362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.21256067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.448271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21255146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443493...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.02.17.431620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.366807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453750...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.383604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259929...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.451701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.26.919985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.395079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.07.20245274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453293...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.017624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.447244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.28.359356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.21257844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.20245993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.438792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261046...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.01.20144030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453650...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.22.262527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.20202614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.21254024...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21260947...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433056...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.116111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21258780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.405670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.21257505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.19.21252062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.437754...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.441101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441683...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.440514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.21258886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.451608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.20245266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.447066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454213...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20177857...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.454001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.09.20148429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.17.20069716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.17.20061440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.257352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/596718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.20202267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.236703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20088260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275560.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.452189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.01.20135194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.452934...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.06.028712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.432992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.320515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.23.164004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.20.427467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.450983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.324145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450676...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.17.254607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451278...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.10.20021675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.21253051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.435634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.435971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.274563.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.439231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.14.095414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.395905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.21258447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442770...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.21256242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.09.430440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.447104...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.363861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.03.075069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.22.20026500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453662...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.26.966580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.21258488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.191684.115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.21260259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452052...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.31.275644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.24.265702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21252086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.21251946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.431212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.20235010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21254092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453688...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.434601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.164442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.11.20128520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.443998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.440988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.13.448261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.09.442808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.13.21258859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.04.076067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047365...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.241414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21253973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.226043...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.389916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.437067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.12.20151084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.03.437766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21256113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.02.20080036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.274282.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.446775...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21251509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.366716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.360842...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.25.20248853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.23.350348...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.429765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/197608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.21258332...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.423113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.20222661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453339...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.03.20243352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260986...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261039...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.15.383323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.320721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.20205633...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.07.19.21260767', 'tweets': 625, 'archive': 'medRxiv', 'subject-area': 'Pediatrics', 'covid': True, 'title': 'Children with SARS-CoV-2 in the National COVID Cohort Collaborative (N3C)', 'authors': \"[{'name': 'Blake Martin'}, {'name': 'Peter E. DeWitt'}, {'name': 'Seth Russell'}, {'name': 'Adit Anand'}, {'name': 'Katie R. Bradwell'}, {'name': 'Carolyn Bremer'}, {'name': 'Davera Gabriel'}, {'name': 'Andrew T. Girvin'}, {'name': 'Janos G. Hajagos'}, {'name': 'Julie A. McMurry'}, {'name': 'Andrew J. Neumann'}, {'name': 'Emily R. Pfaff'}, {'name': 'Anita Walden'}, {'name': 'Jacob T. Wooldridge'}, {'name': 'Yun Jae Yoo'}, {'name': 'Joel Saltz'}, {'name': 'Ken R. Gersing'}, {'name': 'Christopher G. Chute'}, {'name': 'Melissa A. Haendel'}, {'name': 'Richard Moffitt'}, {'name': 'Tellen D. Bennett'}]\", 'abstract': '<sec><title>Importance</title><p>SARS-CoV-2</p></sec><sec><title>Objective</title><p>To determine the characteristics, changes over time, outcomes, and severity risk factors of SARS-CoV-2 affected children within the National COVID Cohort Collaborative (N3C)</p></sec><sec><title>Design</title><p>Prospective cohort study of patient encounters with end dates before May 27th, 2021.</p></sec><sec><title>Setting</title><p>45 N3C institutions</p></sec><sec><title>Participants</title><p>Children &lt;19-years-old at initial SARS-CoV-2 testing</p></sec><sec><title>Main Outcomes and Measures</title><p>Case incidence and severity over time, demographic and comorbidity severity risk factors, vital sign and laboratory trajectories, clinical outcomes, and acute COVID-19 vs MIS-C contrasts for children infected with SARS-CoV-2.</p></sec><sec><title>Results</title><p>728,047 children in the N3C were tested for SARS-CoV-2; of these, 91,865 (12.6%) were positive. Among the 5,213 (6%) hospitalized children, 685 (13%) met criteria for severe disease: mechanical ventilation (7%), vasopressor/inotropic support (7%), ECMO (0.6%), or death/discharge to hospice (1.1%). Male gender, African American race, older age, and several pediatric complex chronic condition (PCCC) subcategories were associated with higher clinical severity (p<italic>≤</italic>0.05). Vital signs (all p≤0.002) and many laboratory tests from the first day of hospitalization were predictive of peak disease severity. Children with severe (vs moderate) disease were more likely to receive antimicrobials (71% vs 32%, p&lt;0.001) and immunomodulatory medications (53% vs 16%, p&lt;0.001).</p><p>Compared to those with acute COVID-19, children with MIS-C were more likely to be male, Black/African American, 1-to-12-years-old, and less likely to have asthma, diabetes, or a PCCC (p<italic>&lt;</italic>0.04). MIS-C cases demonstrated a more inflammatory laboratory profile and more severe clinical phenotype with higher rates of invasive ventilation (12% vs 6%) and need for vasoactive-inotropic support (31% vs 6%) compared to acute COVID-19 cases, respectively (p&lt;0.03).</p></sec><sec><title>Conclusions</title><p>In the largest U.S. SARS-CoV-2-positive pediatric cohort to date, we observed differences in demographics, pre-existing comorbidities, and initial vital sign and laboratory test values between severity subgroups. Taken together, these results suggest that early identification of children likely to progress to severe disease could be achieved using readily available data elements from the day of admission. Further work is needed to translate this knowledge into improved outcomes.</p></sec>', 'posted': '2021-07-22'}, {'doi': '10.1101/2021.07.23.21261030', 'tweets': 232, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Breakthrough Symptomatic COVID-19 Infections Leading to Long Covid: Report from Long Covid Facebook Group Poll', 'authors': \"[{'name': 'Daisy Massey'}, {'name': 'Diana Berrent'}, {'name': 'Harlan Krumholz'}]\", 'abstract': '<p>Vaccines have been shown to be extremely effective in preventing COVID-19 hospitalizations and deaths. However, a question remains whether vaccine breakthrough cases can still lead to Post-Acute Sequelae of SARS-CoV-2 (PASC), also known as Long Covid. To address this question, the Survivor Corps group, a grassroots COVID-19 organization focused on patient support and research, posted a poll to its 169,900 members that asked about breakthrough cases, Long Covid, and hospitalizations. 1,949 people who self-report being fully vaccinated have responded to date. While robust data are needed in a larger, unbiased sample to extrapolate rates to the population, we analyzed the results of this public poll to determine what people were reporting regarding Long Covid after breakthrough infection and to prompt discussion of how breakthrough cases are measured. The poll was posted in the Survivor Corps Facebook group (∼169,900 members). Of the 1,949 participants who responded to the poll, 44 reported a symptomatic breakthrough case and 24 of those reported that the case led to symptoms of Long Covid. 1 of these 24 cases was reported to have led to hospitalization in addition to Long Covid.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453748', 'tweets': 74, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Identification of a new family of “megaphages” that are abundant in the marine environment', 'authors': \"[{'name': 'Slawomir Michniewski'}, {'name': 'Branko Rihtman'}, {'name': 'Ryan Cook'}, {'name': 'Michael A. Jones'}, {'name': 'William H Wilson'}, {'name': 'David J Scanlan'}, {'name': 'Andrew Millard'}]\", 'abstract': '<p>Megaphages – bacteriophages harbouring extremely large genomes – have recently been found to be ubiquitous, being described from a variety of microbiomes ranging from the animal gut to soil and freshwater systems. However, no complete marine megaphage has been identified to date. Here, using both short and long read sequencing, we assembled &gt;900 high-quality draft viral genomes from water in the English Channel. One of these genomes included a novel megaphage, Mar_Mega_1 at &gt;650 Kb, making it one of the largest phage genomes assembled to date. Utilising phylogenetic and network approaches, we found this phage represents a new family of bacteriophages. Genomic analysis showed Mar_Mega_1 shares relatively few homologues with its closest relatives, but, as with other mega-phages Mar_Mega_1 contained a variety of auxiliary metabolic genes responsible for carbon metabolism and nucleotide biosynthesis, including isocitrate dehydrogenase [NADP] and nicotinamide-nucleotide amidohydrolase [PncC] which have not previously been identified in megaphages. The results of this study indicate that phages containing extremely large genomes can be found in abundance in the marine environment and augment host metabolism by mechanisms not previously described.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453070', 'tweets': 60, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Regulation of sedimentation rate shapes the evolution of multicellularity in a unicellular relative of animals', 'authors': \"[{'name': 'Omaya Dudin'}, {'name': 'Sébastien Wielgoss'}, {'name': 'Aaron M. New'}, {'name': 'Iñaki Ruiz-Trillo'}]\", 'abstract': '<p>Significant increases in sedimentation rate accompany the evolution of multicellularity. These increases should lead to rapid changes in ecological distribution, thereby affecting the costs and benefits of multicellularity and its likelihood to evolve. However, how genetic and cellular traits which control this process, their likelihood of emergence over evolutionary timescales, and the variation in these traits as multicellularity evolves, are still poorly understood. Here, using isolates of the ichthyosporean <italic>Sphaeroforma</italic> genus - close unicellular relatives of animals with brief transient multicellular life stages - we demonstrate that sedimentation rate is a highly variable and evolvable trait affected by at least two distinct physical mechanisms. We first find a dramatic &gt;300x variation in sedimentation rate for different <italic>Sphaeroforma</italic> species, mainly driven by size and density during the unicellular-to-multicellular life cycle transition. Using experimental evolution with sedimentation rate as a focal trait, we readily obtained fast settling <italic>S. arctica</italic> isolates. Quantitative microscopy showed that increased sedimentation rates most often arose by incomplete cellular separation after cell division, leading to clonal “clumping” multicellular variants with increased size and density. Additionally, density increases arose by an acceleration of the nuclear doubling time relative to cell size. Similar size- and density-affecting phenotypes were observed in four additional species from the <italic>Sphaeroforma</italic> genus, suggesting variation in these traits might be widespread in the marine habitat. By sequencing evolved isolates, we identified mutations in regulators of cytokinesis, plasma membrane remodelling, and chromatin condensation that may contribute to both clump formation and the increase in the nuclear number-to-volume ratio. Taken together, this study illustrates how extensive cellular control of density and size drive sedimentation rate variation, likely shaping the evolution of multicellularity.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453379', 'tweets': 57, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'AlphaPept, a modern and open framework for MS-based proteomics', 'authors': \"[{'name': 'Maximilian T. Strauss'}, {'name': 'Isabell Bludau'}, {'name': 'Wen-Feng Zeng'}, {'name': 'Eugenia Voytik'}, {'name': 'Constantin Ammar'}, {'name': 'Julia Schessner'}, {'name': 'Rajesh Ilango'}, {'name': 'Michelle Gill'}, {'name': 'Florian Meier'}, {'name': 'Sander Willems'}, {'name': 'Matthias Mann'}]\", 'abstract': '<p>In common with other omics technologies, mass spectrometry (MS)-based proteomics produces ever-increasing amounts of raw data, making their efficient analysis a principal challenge. There is a plethora of different computational tools that process the raw MS data and derive peptide and protein identification and quantification. During the last decade, there has been dramatic progress in computer science and software engineering, including collaboration tools that have transformed research and industry. To leverage these advances, we developed AlphaPept, a Python-based open-source framework for efficient processing of large high-resolution MS data sets. Using Numba for just-in-time machine code compilation on CPU and GPU, we achieve hundred-fold speed improvements while maintaining clear syntax and rapid development speed. AlphaPept uses the Python scientific stack of highly optimized packages, reducing the code base to domain-specific tasks while providing access to the latest advances in machine learning. We provide an easy on-ramp for community validation and contributions through the concept of literate programming, implemented in Jupyter Notebooks of the different modules. A framework for continuous integration, testing, and benchmarking enforces solid software engineering principles. Large datasets can rapidly be processed as shown by the analysis of hundreds of cellular proteomes in minutes per file, many-fold faster than the data acquisiton. The AlphaPept framework can be used to build automated processing pipelines using efficient HDF5 based file formats, web-serving functionality and compatibility with downstream analysis tools. Easy access for end-users is provided by one-click installation of the graphical user interface, for advanced users via a modular Python library, and for developers via a fully open GitHub repository.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453492', 'tweets': 52, 'archive': 'bioRxiv', 'subject-area': 'Scientific Communication and Education', 'covid': False, 'title': 'Delineating Medical Education: Bibliometric Research Approach(es)', 'authors': \"[{'name': 'Lauren A. Maggio'}, {'name': 'Anton Ninkov'}, {'name': 'Jason R. Frank'}, {'name': 'Joseph A. Costello'}, {'name': 'Anthony R. Artino'}]\", 'abstract': '<sec><title>Background</title><p>The field of medical education remains poorly delineated such that there is no broad consensus of the articles and journals that comprise “the field.” This lack of consensus has implications for conducting bibliometric studies and other research designs (e.g., systematic reviews); it also challenges the field to compare citation scores in the field and across others and for an individual to identify themselves as “a medical education researcher.” Other fields have utilized bibliometric field delineation, which is the assigning of articles or journals to a certain field in an effort to define that field.</p></sec><sec><title>Process</title><p>In this <italic>Research Approach</italic>, three bibliometric field delineation approaches -- information retrieval, core journals, and journal co-citation -- are introduced. For each approach, the authors describe their attempt to apply it in the medical education context and identify related strengths and weaknesses. Based on co-citation, the authors propose the Medical Education Journal List 24 (MEJ-24), as a starting point for delineating medical education and invite the community to collaborate on improving and potentially expanding this list.</p></sec><sec><title>Pearls</title><p>As a research approach, field delineation is complicated, and there is no clear best way to delineate the field of medical education. However, recent advances in information and computer science provide potentially more fruitful approaches to deal with the complexity of the field. When considering these emerging approaches, researchers should consider collaborating with bibliometricians.</p><p>Bibliometric approaches rely on available metadata for articles and journals, which necessitates that researchers examine the metadata prior to analysis to understand its strengths and weaknesses, and to assess how this might affect their data interpretation. While using bibliometric approaches for field delineation is valuable, it is important to remember that these techniques are only as good as the research team’s interpretation of the data, which suggests that an expanded research approach is needed to better delineate medical education, an approach that includes active discussion within the medical education community.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453605', 'tweets': 51, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Identifying cell-state associated alternative splicing events and their co-regulation', 'authors': \"[{'name': 'Carlos F. Buen Abad Najar'}, {'name': 'Prakruthi Burra'}, {'name': 'Nir Yosef'}, {'name': 'Liana F. Lareau'}]\", 'abstract': '<p>Alternative splicing shapes the transcriptome and contributes to each cell’s unique identity, but single-cell RNA sequencing has struggled to capture the impact of alternative splicing. We previously showed that low recovery of mRNAs from single cells led to erroneous conclusions about the cell-to-cell variability of alternative splicing (1). Here, we present a method, Psix, to confidently identify splicing that changes across a landscape of single cells, using a probabilistic model that is robust against the data limitations of scRNA-seq. Its autocorrelation-inspired approach finds patterns of alternative splicing that correspond to patterns of cell identity, such as cell type or developmental stage, without the need for explicit cell clustering, labeling, or trajectory inference. Applying Psix to data that follow the trajectory of mouse brain development, we identify exons whose alternative splicing patterns cluster into modules of co-regulation. We show that the exons in these modules are enriched for binding by distinct neuronal splicing factors, and that their changes in splicing correspond to changes in expression of these splicing factors. Thus, Psix reveals cell-type-dependent splicing patterns and the wiring of the splicing regulatory networks that control them. Our new method will enable scRNA-seq analysis to go beyond transcription to understand the roles of post-transcriptional regulation in determining cell identity.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.24.21261040', 'tweets': 47, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Novel risk factors for Coronavirus disease-associated mucormycosis (CAM): a case control study during the outbreak in India', 'authors': \"[{'name': 'Umang Arora'}, {'name': 'Megha Priyadarshi'}, {'name': 'Varidh Katiyar'}, {'name': 'Manish Soneja'}, {'name': 'Prerna Garg'}, {'name': 'Ishan Gupta'}, {'name': 'Vishwesh Bharadiya'}, {'name': 'Parul Berry'}, {'name': 'Tamoghna Ghosh'}, {'name': 'Lajjaben Patel'}, {'name': 'Radhika Sarda'}, {'name': 'Shreya Garg'}, {'name': 'Shubham Agarwal'}, {'name': 'Veronica Arora'}, {'name': 'Aishwarya Ramprasad'}, {'name': 'Amit Kumar'}, {'name': 'Rohit Kumar Garg'}, {'name': 'Parul Kodan'}, {'name': 'Neeraj Nischal'}, {'name': 'Gagandeep Singh'}, {'name': 'Pankaj Jorwal'}, {'name': 'Arvind Kumar'}, {'name': 'Upendra Baitha'}, {'name': 'Ved Prakash Meena'}, {'name': 'Animesh Ray'}, {'name': 'Prayas Sethi'}, {'name': 'Immaculata Xess'}, {'name': 'Naval Vikram'}, {'name': 'Sanjeev Sinha'}, {'name': 'Ashutosh Biswas'}, {'name': 'Alok Thakar'}, {'name': 'Sushma Bhatnagar'}, {'name': 'Anjan Trikha'}, {'name': 'Naveet Wig'}]\", 'abstract': '<sec><title>Background</title><p>The epidemiology of the Coronavirus-disease associated mucormycosis (CAM) syndemic is poorly elucidated. We aimed to identify risk factors that may explain the burden of cases and help develop preventive strategies.</p></sec><sec><title>Methods</title><p>We performed a case-control study comparing cases diagnosed with CAM and those who had recovered from COVID-19 without developing mucormycosis (controls). Information on comorbidities, glycemic control, and practices related to COVID-19 prevention and treatment was recorded.</p></sec><sec><title>Results</title><p>352 patients (152 cases and 200 controls) diagnosed with COVID-19 during April-May 2021 were included. In the CAM group, symptoms of mucormycosis began a mean 18.9 (SD 9.1) days after onset of COVID-19, and predominantly rhino-sinus and orbital involvement was present. All, but one, CAM cases carried conventional risk factors of diabetes and steroid use. On multivariable regression, increased odds of CAM were associated with the presence of diabetes (adjusted OR 3.5, 95%CI 1.1-11), use of systemic steroids (aOR 7.7,95% CI 2.4-24.7), prolonged use of cloth and surgical masks (vs no mask, aOR 6.9, 95%CI 1.5-33.1), and repeated nasopharyngeal swab testing during the COVID-19 illness (aOR 1.6,95% CI 1.2-2.2). Zinc therapy, probably due to its utility in immune function, was found to be protective (aOR 0.05, 95%CI 0.01-0.19). Notably, the requirement of oxygen supplementation or hospitalization did not affect the risk of CAM.</p></sec><sec><title>Conclusion</title><p>Judicious use of steroids and stringent glycemic control are vital to preventing mucormycosis. Use of clean masks, preference for N95 masks if available, and minimizing swab testing after the diagnosis of COVID-19 may further reduce the incidence of CAM.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261041', 'tweets': 44, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'The impact of large mobile air purifiers on aerosol concentration in classrooms and the reduction of airborne transmission of SARS-CoV-2', 'authors': \"[{'name': 'F. F. Duill'}, {'name': 'F. Schulz'}, {'name': 'A. Jain'}, {'name': 'L. Krieger'}, {'name': 'B. van Wachem'}, {'name': 'F. Beyrau'}]\", 'abstract': '<p>In the wake of the SARS-CoV-2 pandemic, an increased risk of infection by virus-containing aerosols indoors is assumed. Especially in schools, the duration of stay is long and the number of people in the rooms is large, increasing the risk of infection. This problem particularly affects schools without pre-installed ventilation systems that are equipped with filters and/or operate with fresh air. Here, the aerosol concentration is reduced by natural ventilation. In this context, we are investigating the effect of large mobile air purifiers (AP) with HEPA filters on particle concentration and their suitability for classroom use in a primary school in Germany. The three tested APs differ significantly in their air outlet characteristics. Measurements of the number of particles, the particle size distribution, and the CO<sub>2</sub> concentration were carried out in the classroom with students (April/May 2021) and with an aerosol generator without students. In this regard, the use of APs leads to a substantial reduction in aerosol particles. At the same time, the three APs are found to have differences in their particle decay rate, noise level, and flow velocity. In addition to the measurements, the effect of various influencing parameters on the potential inhaled particle dose was investigated using a calculation model. The parameters considered include the duration of stay, particle concentration in exhaled air, respiratory flow rate, virus lifetime, ventilation interval, ventilation efficiency, AP volumetric flow, as well as room size. Based on the resulting effect diagrams, significant recommendations can be derived for reducing the risk of infection from virus-laden aerosols. Finally, the measurements were compared to computational fluid dynamics (CFD) modeling, as such tools can aid the optimal placement and configuration of APs and can be used to study the effect of the spread of aerosols from a source in the classroom.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453478', 'tweets': 44, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Mechanosensitivity of nucleocytoplasmic transport', 'authors': \"[{'name': 'Ion Andreu'}, {'name': 'Ignasi Granero-Moya'}, {'name': 'Nimesh R. Chahare'}, {'name': 'Kessem Clein'}, {'name': 'Marc Molina Jordàn'}, {'name': 'Amy E. M. Beedle'}, {'name': 'Alberto Elosegui-Artola'}, {'name': 'Xavier Trepat'}, {'name': 'Barak Raveh'}, {'name': 'Pere Roca-Cusachs'}]\", 'abstract': '<p>Mechanical force controls fundamental cellular processes in health and disease, and increasing evidence shows that the nucleus both experiences and senses applied forces. Here we show that nuclear forces differentially control both passive and facilitated nucleocytoplasmic transport, setting the rules for the mechanosensitivity of shuttling proteins. We demonstrate that nuclear force increases permeability across nuclear pore complexes, with a dependence on molecular weight that is stronger for passive than facilitated diffusion. Due to this differential effect, force leads to the translocation into or out of the nucleus of cargoes within a given range of molecular weight and affinity for nuclear transport receptors. Further, we show that the mechanosensitivity of several transcriptional regulators can be both explained by this mechanism, and engineered exogenously by introducing appropriate nuclear localization signals. Our work sets a novel framework to understand mechanically induced signalling, with potential general applicability across signalling pathways and pathophysiological scenarios.</p><sec><title>One sentence summary</title><p>Force application to the nucleus leads to nuclear accumulation of proteins by differentially affecting passive versus facilitated nucleocytoplasmic transport.</p></sec>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.23.453573', 'tweets': 43, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Strong and opponent contributions of dorsomedial striatal pathways to behavior depends on cognitive demands and task strategy', 'authors': \"[{'name': 'Scott S. Bolkan'}, {'name': 'Iris R. Stone'}, {'name': 'Lucas Pinto'}, {'name': 'Zoe C. Ashwood'}, {'name': 'Jorge M. Iravedra Garcia'}, {'name': 'Alison L. Herman'}, {'name': 'Priyanka Singh'}, {'name': 'Akhil Bandi'}, {'name': 'Julia Cox'}, {'name': 'Christopher A. Zimmerman'}, {'name': 'Jounhong Ryan Cho'}, {'name': 'Ben Engelhard'}, {'name': 'Sue A. Koay'}, {'name': 'Jonathan W. Pillow'}, {'name': 'Ilana B. Witten'}]\", 'abstract': '<p>A classic view of the striatum holds that activity in direct and indirect pathways oppositely modulates motor output. Whether this involves direct control of movement, or reflects a cognitive process underlying movement, has remained unresolved. Here we find that strong, opponent control of behavior by the two pathways of the dorsomedial striatum (DMS) depends on a task’s cognitive demands. Furthermore, a latent state model (a hidden markov model with generalized linear model observations) reveals that—even within a single task—the contribution of the two pathways to behavior is state-dependent. Specifically, the two pathways have large contributions in one of two states associated with a strategy of evidence accumulation, compared to a state associated with a strategy of repeating previous choices. Thus, both the cognitive demands imposed by a task, as well as the strategy that mice pursue within a task, determine whether DMS pathways provide strong and opponent control of behavior.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.22.453458', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Epithelial-mesenchymal plasticity determines estrogen receptor positive (ER+) breast cancer dormancy and reacquisition of an epithelial state drives awakening', 'authors': \"[{'name': 'Patrick Aouad'}, {'name': 'Yueyun Zhang'}, {'name': 'Céline Stibolt'}, {'name': 'Sendurai A. Mani'}, {'name': 'George Sflomos'}, {'name': 'Cathrin Brisken'}]\", 'abstract': '<p>Estrogen receptor α-positive (ER+) breast cancers (BCs) represent more than 70% of all breast cancers and pose a particular clinical challenge because they recur up to decades after initial diagnosis and treatment. The mechanisms governing tumor cell dormancy and latent disease remain elusive due to a lack of adequate models. Here, we compare tumor progression of ER+ and triple-negative (TN) BC subtypes with a clinically relevant mouse intraductal xenografting approach (MIND). Both ER+ and TN BC cells disseminate already during the <italic>in situ</italic>stage. However, TN disseminated tumor cells (DTCs) proliferate at the same rate as cells at the primary site and give rise to macro-metastases. ER+ DTCs have low proliferative indices, form only micro-metastases and lose epithelial characteristics. Expression of <italic>CDH1</italic> is decreased whereas the mesenchymal marker <italic>VIM</italic> and the transcription factors, <italic>ZEB1</italic>/<italic>ZEB2,</italic> which control epithelial-mesenchymal plasticity (EMP) are increased. EMP is not detected earlier during ER+ BC development and not required for invasion or metastasis. <italic>In vivo</italic>, forced transition to the epithelial state through ectopic E-cadherin expression overcomes dormancy with increased growth of lung metastases. We conclude that EMP is essential for the generation of a dormant cell state and the development of latent disease. Targeting exit from EMP is of therapeutic potential.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453521', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Probiotic consortia improve anti-viral immunity to SARS-CoV-2 in Ferrets', 'authors': \"[{'name': 'J Lehtinen Markus'}, {'name': 'Kumar Ritesh'}, {'name': 'Zabel Bryan'}, {'name': 'M Mäkelä Sanna'}, {'name': 'Nedveck Derek'}, {'name': 'Tang Peipei'}, {'name': 'Latvala Sinikka'}, {'name': 'Guery Sebastien'}, {'name': 'R Budinoff Charles'}]\", 'abstract': '<p>Probiotics have been suggested as one solution to counter detrimental health effects by SARS-CoV-2, however, data so far is scarce. We tested the effect of two probiotic consortia, OL-1 and OL-2, against SARS-CoV-2 in ferrets and assessed their effect on cytokine production and transcriptome in a human monocyte-derived macrophage (Mf) and dendritic cell (DC) model.</p><p>The results showed that the consortia significantly reduced the viral load, modulated immune response, and regulated viral receptor expression in ferrets compared to placebo. In human Mf and DC model, OL-1 and OL-2 induced cytokine production and genes related to SARS-CoV-2 anti-viral immunity.</p><p>The study results indicate that probiotic stimulation of the ferret immune system leads to improved anti-viral immunity against SARS-COV-2 and that critical genes and cytokines for anti-SARS-CoV-2 immunity are stimulated in human immune cells <italic>in vitro</italic>. The effect of the consortia against SARS-CoV-2 warrants further investigations in human clinical trials.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.26.453760', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A bacterial dynamin-like protein confers a novel phage resistance strategy on the population level in Bacillus subtilis', 'authors': \"[{'name': 'Lijun Guo'}, {'name': 'Laura Sattler'}, {'name': 'Peter Graumann'}, {'name': 'Marc Bramkamp'}]\", 'abstract': '<p><italic>Bacillus subtilis</italic> DynA is a member of the dynamin superfamily, involved in membrane remodeling processes. DynA was shown to catalyze full membrane fusion and it plays a role in membrane surveillance against antibiotics. We show here that DynA also provides a novel resistance mechanism against phage infection. Cells lacking DynA are efficiently lysed after phage infection and virus replication. DynA does not prevent phage infection and replication in individual cells, but significantly delays host cell lysis, thereby slowing down the release of phage progeny from the host cells. During the process, DynA forms large, almost immobile clusters on the cell membrane that seem to support membrane integrity. Single molecule tracking revealed a shift of freely diffusive molecules within the cytosol towards extended, confined motion at the cell membrane following phage induction. Thus, the bacterial dynamins are the first anti-phage system reported to delay host cell lysis and the last line of defense of a multilayered antiviral defense. DynA is therefore providing protective effects on the population, but not on single cell level.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.27.453996', 'tweets': 33, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Disruption of the TCA cycle reveals an ATF4-dependent integration of redox and amino acid metabolism', 'authors': \"[{'name': 'Dylan Gerard Ryan'}, {'name': 'Ming Yang'}, {'name': 'Hiran A Prag'}, {'name': 'Giovanny Rodriguez Blanco'}, {'name': 'Efterpi Nikitopoulou'}, {'name': 'Marc Segarra-Mondejar'}, {'name': 'Christopher A Powell'}, {'name': 'Tim Young'}, {'name': 'Nils Burger'}, {'name': 'Jan Lj Miljkovic'}, {'name': 'Michal Minczuk'}, {'name': 'Michael P Murphy'}, {'name': 'Alexander Von Kriegsheim'}, {'name': 'Christian Frezza'}]\", 'abstract': '<p>The Tricarboxylic Acid Cycle (TCA) cycle is arguably the most critical metabolic cycle in physiology and exists as an essential interface coordinating cellular metabolism, bioenergetics, and redox homeostasis. Despite decades of research, a comprehensive investigation into the consequences of TCA cycle dysfunction remains elusive. Here, we targeted two TCA cycle enzymes, fumarate hydratase (FH) and succinate dehydrogenase (SDH), and combined metabolomics, transcriptomics, and proteomics analyses to fully appraise the consequences of TCA cycle inhibition (TCAi) in kidney epithelial cells. Our comparative approach shows that TCAi elicits a convergent rewiring of redox and amino acid metabolism dependent on the activation of ATF4 and the integrated stress response (ISR). Furthermore, we also uncover a divergent metabolic response, whereby acute FHi, but not SDHi, can maintain asparagine levels via reductive carboxylation and maintenance of cytosolic aspartate synthesis. Our work highlights an important interplay between the TCA cycle, redox biology and amino acid homeostasis.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.25.21261093', 'tweets': 31, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Protection afforded by the BNT162b2 and mRNA-1273 COVID-19 vaccines in fully vaccinated cohorts with and without prior infection', 'authors': \"[{'name': 'Laith J. Abu-Raddad'}, {'name': 'Hiam Chemaitelly'}, {'name': 'Houssein H. Ayoub'}, {'name': 'Hadi M. Yassine'}, {'name': 'Fatiha M. Benslimane'}, {'name': 'Hebah A. Al Khatib'}, {'name': 'Patrick Tang'}, {'name': 'Mohammad R. Hasan'}, {'name': 'Peter Coyle'}, {'name': 'Zaina Al Kanaani'}, {'name': 'Einas Al Kuwari'}, {'name': 'Andrew Jeremijenko'}, {'name': 'Anvar Hassan Kaleeckal'}, {'name': 'Ali Nizar Latif'}, {'name': 'Riyazuddin Mohammad Shaik'}, {'name': 'Hanan F. Abdul Rahim'}, {'name': 'Gheyath K. Nasrallah'}, {'name': 'Mohamed Ghaith Al Kuwari'}, {'name': 'Adeel A. Butt'}, {'name': 'Hamad Eid Al Romaihi'}, {'name': 'Mohamed H. Al-Thani'}, {'name': 'Abdullatif Al Khal'}, {'name': 'Roberto Bertollini'}]\", 'abstract': '<p>Effect of prior SARS-CoV-2 infection on vaccine protection remains poorly understood. Here, we investigated whether persons vaccinated after a prior infection have better protection against future infection than those vaccinated without prior infection. Effect of prior infection was assessed in Qatar’s population, where the Alpha (B.1.1.7) and Beta (B.1.351) variants dominate incidence, using two national retrospective, matched-cohort studies, one for the BNT162b2 (Pfizer-BioNTech) vaccine, and one for the mRNA-1273 (Moderna) vaccine. Incidence rates of infection among BNT162b2-vaccinated persons, with and without prior infection, were estimated, respectively, at 1.66 (95% CI: 1.26-2.18) and 11.02 (95% CI: 9.90-12.26) per 10,000 person-weeks. The incidence rate ratio was 0.15 (95% CI: 0.11-0.20). Analogous incidence rates among mRNA-1273-vaccinated persons were estimated at 1.55 (95% CI: 0.86-2.80) and 1.83 (95% CI: 1.07-3.16) per 10,000 person-weeks. The incidence rate ratio was 0.85 (95% CI: 0.34-2.05). Prior infection enhanced protection of those BNT162b2-vaccinated, but not those mRNA-1273-vaccinated. These findings may have implications for dosing, interval between doses, and potential need for booster vaccination.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.21.453258', 'tweets': 30, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Retrosplenial cortex is necessary for spatial and non-spatial latent learning in mice', 'authors': \"[{'name': 'Ana Carolina Bottura de Barros'}, {'name': 'Liad J. Baruchin'}, {'name': 'Marios C. Panayi'}, {'name': 'Nils Nyberg'}, {'name': 'Veronika Samborska'}, {'name': 'Mitchell T. Mealing'}, {'name': 'Thomas Akam'}, {'name': 'Jeehyun Kwag'}, {'name': 'David M. Bannerman'}, {'name': 'Michael M. Kohl'}]\", 'abstract': '<p>Latent learning occurs when associations are formed between stimuli in the absence of explicit reinforcement. Traditionally, latent learning in rodents has been associated with the creation internal models of space. However, increasing evidence points to roles of internal models also in non-spatial decision making. Whether the same brain structures and processes support the creation of spatially-anchored or non-spatial internal models via latent learning, is an open question. To address this question, we developed a novel operant box task that allows to test spatial and non-spatial versions of a flavour-based sensory preconditioning paradigm. We probed the role of the retrosplenial cortex, a brain area associated with spatial cognition and subjective value representation, in this task using precise, closed-loop optogenetic silencing during different task phases. We show that the retrosplenial cortex is necessary for both spatial and non-spatial latent learning in mice. We further demonstrate that the requirement of retrosplenial cortex is limited to the preconditioning phase of the task. Our results provide insight into the specific role of the retrosplenial cortex in latent learning, demonstrate that latent learning plays a general part in the creation of internal models, independent of spatial anchors, and provide a novel avenue for studying model-based decision making.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.27.453460', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Extracellular mechanical forces drive endocardial cell volume decrease during cardiac valve morphogenesis', 'authors': \"[{'name': 'Julien Vermot'}, {'name': 'Helene Vignes'}, {'name': 'Christina Vagena-Pantoula'}, {'name': 'Mangal Prakash'}, {'name': 'Caren Norden'}, {'name': 'Florian Jug'}]\", 'abstract': '<p>Organ morphogenesis involves dynamic changes of tissue properties at the cellular scale. In addition, cells need to adapt to their mechanical environment through mechanosensitive pathways. How mechanical cues influence cell behaviors during morphogenesis, however, remains poorly understood. Here we studied the influence of mechanical forces during the formation of the atrioventricular canal (AVC) where cardiac valves develop. We show that in zebrafish the AVC forms within a zone of tissue convergence between the atrium and the ventricle which is associated with increased activation of the actomyosin meshwork and endocardial cell orientation changes. We demonstrate that tissue convergence occurs with a major reduction of endocardial cell volume triggered by mechanical forces and the mechanosensitive channels TRPP2/TRPV4. In addition, we show that the extracellular matrix component hyaluronic acid controls cell volume changes. Together, our data suggest that cell volume change is a key cellular feature activated by mechanical forces during cardiovascular morphogenesis. This work further unravels how mechanical forces and\\nextracellular matrix can influence tissue remodeling in developing organs.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.26.453816', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Synaptic Mechanisms of Top-Down Control by The Auditory Cortico-Collicular Pathway', 'authors': \"[{'name': 'Hannah M. Oberle'}, {'name': 'Alex Ford'}, {'name': 'Jordyn Czarny'}, {'name': 'Pierre F. Apostolides'}]\", 'abstract': '<p>Corticofugal projections to evolutionarily ancient, sub-cortical structures are ubiquitous across mammalian sensory systems. These “descending” pathways enable the neocortex to control ascending sensory representations in a predictive or feedback manner, but the underlying cellular mechanisms are poorly understood. Here we combine optogenetic approaches with <italic>in vivo</italic> and <italic>in vitro</italic> patch-clamp electrophysiology to study the projection from auditory cortex to the inferior colliculus (IC), a major descending auditory pathway that controls IC neuron feature selectivity, plasticity and auditory perceptual learning. Although individual auditory cortico-collicular synapses were generally weak, IC neurons often integrated inputs from multiple corticofugal axons that generated reliable, tonic depolarizations even during prolonged presynaptic activity. Latency measurements <italic>in vivo</italic> showed that descending signals reach the IC within 30 ms of sound onset, which in IC neurons corresponded to the peak of synaptic depolarizations evoked by short sounds. Activating ascending and descending pathways at latencies expected <italic>in vivo</italic> caused a NMDA receptor dependent, supra-linear EPSP summation, indicating that descending signals can non-linearly amplify IC neurons’ moment-to-moment acoustic responses. Our results shed light upon the synaptic bases of descending sensory control, and imply that heterosynaptic cooperativity contributes to the auditory cortico-collicular pathway’s role in plasticity and perceptual learning.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453747', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'ABlooper: Fast accurate antibody CDR loop structure prediction with accuracy estimation', 'authors': \"[{'name': 'Brennan Abanades'}, {'name': 'Guy Georges'}, {'name': 'Alexander Bujotzek'}, {'name': 'Charlotte M. Deane'}]\", 'abstract': '<p>Antibodies are a key component of the immune system and have been extensively used as biotherapeutics. Accurate knowledge of their structure is central to understanding their function. The key area for antigen binding and the main area of structural variation in antibodies are concentrated in their six complementarity determining regions (CDRs), with the most variable being the CDR-H3 loop. The sequence and structure variability of CDR-H3 make it particularly challenging to model. Recently, deep learning methods have offered a step change in our ability to predict protein structures. In this work we present ABlooper, an end-to-end equivariant deep-learning based CDR loop structure prediction tool. ABlooper predicts the structure of CDR loops with high accuracy and provides a confidence estimate for each of its predictions. On the models of the Rosetta Antibody Benchmark, ABlooper makes predictions with an average H3 RMSD of 2.45Å, which drops to 2.02Å when considering only its 76% most confident predictions.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.21261130', 'tweets': 22, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Vaccine effectiveness when combining the ChAdOx1 vaccine as the first dose with an mRNA COVID-19 vaccine as the second dose', 'authors': \"[{'name': 'Mie Agermose Gram'}, {'name': 'Hanne-Dorthe Emborg'}, {'name': 'Ida Rask Moustsen-Helms'}, {'name': 'Jens Nielsen'}, {'name': 'Anne Katrine Bj&oslashrkholt S&oslashrensen'}, {'name': 'Palle Valentiner-Branth'}, {'name': 'Astrid Blicher Schelde'}, {'name': 'Katrine Finderup Nielsen'}]\", 'abstract': '<p>Background: The recommendations in several countries to stop using the ChAdOx1 vaccine has led to vaccine programs combining different vaccine types, which necessitates new knowledge on vaccine effectiveness (VE). In this study, we aimed to estimate the VE when combining the ChAdOx1 vaccine as the first dose and an mRNA vaccine as the second dose. \\nMethods: This nationwide population-based cohort study estimated VE against SARS-CoV-2 infection, all-cause and COVID-19 related hospitalization and death after receiving the ChAdOx1 vaccine as the first dose followed by an mRNA vaccine as the second dose. VE estimates were obtained using a Cox regression with calendar time as underlying time and adjusted for sex, age, comorbidity, heritage and hospital admission. Information on all individuals was extracted and linked from high-quality national registries. \\nResults: A total of 5,542,079 individuals were included in the analyses (97.6% of the total Danish population). A total of 144,360 were vaccinated with the ChAdOx1 vaccine as the first dose and of these 136,551 individuals received an mRNA vaccine as the second dose. A total of 1,691,464 person-years and 83,034 cases of SARS-CoV-2 infection were included. The VE against SARS-CoV-2 infection when combining the ChAdOx1 and an mRNA vaccine was 88% (95% confidence interval (CI): 83; 92) 14 days after the second dose and onwards. There were no COVID-19 related hospitalizations and deaths among the individuals vaccinated with the combination of the ChAdOx1 and an mRNA vaccine during the study period. \\nConclusion: In conclusion, this study found a reduction in the risk of SARS-CoV-2 infection when combining the ChAdOx1 and an mRNA vaccine, compared with unvaccinated individuals. This is similar to the VE of two doses of an mRNA vaccine. Longer follow-up time is needed to confirm vaccine induced protection against severe events, such as COVID-19 related hospitalization and death.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.22.21260852', 'tweets': 20, 'archive': 'medRxiv', 'subject-area': 'Transplantation', 'covid': True, 'title': 'Comparison of infected and vaccinated transplant recipients highlights the role of Tfh and neutralizing IgG in COVID-19 protection', 'authors': \"[{'name': 'Xavier Charmetant'}, {'name': 'Maxime ESPI'}, {'name': 'Ilies Benotmane'}, {'name': 'Francoise Heibel'}, {'name': 'Fanny Buron'}, {'name': 'Gabriela Gautier-Vargas'}, {'name': 'Marion Delafosse'}, {'name': 'Peggy Perrin'}, {'name': 'Alice Koenig'}, {'name': 'Noelle Cognard'}, {'name': 'Charlene Levi'}, {'name': 'Floriane Gallais'}, {'name': 'Louis Maniere'}, {'name': 'Paola Rossolillo'}, {'name': 'Eric Soulier'}, {'name': 'Florian Pierre'}, {'name': 'Anne Ovize'}, {'name': 'Emmanuel Morelon'}, {'name': 'Thierry Defrance'}, {'name': 'Samira Fafi-Kremer'}, {'name': 'Sophie Caillard'}, {'name': 'Olivier Thaunat'}]\", 'abstract': '<p>Transplant recipients, which receive therapeutic immunosuppression to prevent graft rejection, are characterized by high COVID-19-related mortality and defective response to vaccines. Having observed that previous infection by SARS-CoV-2 but not the standard 2 doses scheme of vaccination, provided complete protection against COVID-19 to transplant recipients, we undertook this translational study to compare the cellular and humoral immune responses of these 2 groups of patients. Neutralizing anti-Receptor Binding Domain (RBD) IgG were identified as the critical immune effectors associated with protection. Generation of anti-RBD IgG was dependent upon spike-specific T follicular helper (Tfh) CD4+ T cells, which acted as limiting checkpoint. Tfh generation was impeded by high dose mycophenolate mofetil in non-responders to vaccine but not in infected patients, suggesting that increasing immunogenicity of vaccine could improve response rate to mRNA vaccine. This theory was validated in two independent prospective cohorts, in which administration of a 3rd dose of vaccine resulted in the generation of anti-RBD IgG in half of non-responders to 2 doses.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.25.453647', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Oncogenic c-Myc induces replication stress by increasing cohesins chromatin occupancy', 'authors': \"[{'name': 'Silvia Peripolli'}, {'name': 'Tanya Singh'}, {'name': 'Harshil Patel'}, {'name': 'Leticia Meneguello'}, {'name': 'Koshiro Kiso'}, {'name': 'Peter Thorpe'}, {'name': 'Cosetta Bertoli'}, {'name': 'Robertus A.M. de Bruin'}]\", 'abstract': '<p>Oncogene-induced replication stress is a major driver of genomic instability in cancer cells, with a central role in both cancer initiation and progression (1). Despite its critical role in cancer development, the mechanisms that lay at the basis of oncogene-induced replication stress remains poorly understood. Here, we investigate the mechanism of c-Myc-induced replication stress. Our data shows that c-Myc induces replication stress by increasing the amount of cohesins bound to chromatin in the G1 phase of the cell cycle. This is independent of previously suggested mechanisms involving deregulation of replication initiation and transcriptional interference. Restoring the amount of chromatin-bound cohesins to control levels, or preventing the accumulation of cohesins at CTCF sites, in cells experiencing oncogenic c-Myc activity prevents replication stress. Increased cohesins chromatin occupancy correlates with a c-Myc-dependent increase in the levels of the cohesion loader Mau2. Preventing c-Myc-induced increase in Mau2 reduces oncogene-induced replication stress. Together our data support a novel mechanism for oncogene-induced replication stress. Since c-Myc activation is a crucial event in many human cancers (2), identifying the mechanisms through which this oncogene promotes replication stress provides critical insights into cancer biology.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453184', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The dual nature of bacteriophage: growth-dependent predation and generalised transduction of antimicrobial resistance', 'authors': \"[{'name': 'Quentin J Leclerc'}, {'name': 'Jacob Wildfire'}, {'name': 'Arya Gupta'}, {'name': 'Jodi A Lindsay'}, {'name': 'Gwenan M Knight'}]\", 'abstract': '<p>Bacteriophage (“phage”) are both predators and evolutionary drivers for bacteria, notably contributing to the spread of antimicrobial resistance (AMR) genes by generalised transduction. Our current understanding of the dual nature of this relationship is limited. We used an interdisciplinary approach to quantify how these interacting dynamics can lead to the evolution of multi-drug resistant bacteria. We co-cultured two strains of Methicillin-resistant <italic>Staphylococcus aureus</italic>, each harbouring a different antibiotic resistance gene, with 80α generalized transducing phage. After a growth phase of 8h, bacteria and phage surprisingly coexisted at a stable equilibrium in our culture, the level of which was dependent on the starting concentration of phage. We detected double-resistant bacteria as early as 7h, indicating that transduction of AMR genes had occurred. We developed multiple mathematical models of the bacteria and phage relationship, and found that phage-bacteria dynamics were best captured by a model in which the phage burst size decreases as the bacteria population reaches stationary phase, and where phage predation is frequency-dependent. We estimated that one in every 10<sup>8</sup> new phage generated was a transducing phage carrying an AMR gene, and that double-resistant bacteria were always predominantly generated by transduction rather than by growth. Our results suggest a fundamental shift in how we understand and model phage-bacteria dynamics. Although rates of generalised transduction may seem insignificant, they are sufficient to consistently lead to the evolution of multi-drug resistant bacteria. Currently, the potential of phage to contribute to the growing burden of AMR is likely underestimated.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.24.453616', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'A yeast lysosomal biogenesis map uncovers the cargo spectrum of lysosomal protein targeting pathways', 'authors': \"[{'name': 'Sebastian Eising'}, {'name': 'Bianca M. Esch'}, {'name': 'Mike Wälte'}, {'name': 'Prado Vargas Duarte'}, {'name': 'Stefan Walter'}, {'name': 'Christian Ungermann'}, {'name': 'Maria Bohnert'}, {'name': 'Florian Fröhlich'}]\", 'abstract': '<p>The lysosome is the major catabolic organelle and a key metabolic signaling center of the cell. Mutations in lysosomal proteins can have catastrophic effects, causing neurodegeneration, cancer, and age-related diseases. The vacuole is the lysosomal analog of <italic>Saccharomyces cerevisiae</italic> that harbors many conserved proteins. Vacuolar proteins reach their destination via the endosomal vacuolar protein sorting (VPS) pathway, via the alkaline phosphatase (ALP or AP-3) pathway, and via the cytosol-to-vacuole transport (CVT) pathway. While these pathways have been extensively studied, a systematic understanding of the cargo spectrum of each pathway is completely lacking. Here we combine quantitative proteomics of purified vacuoles with mutant analyses to generate the lysosomal biogenesis map. This dataset harbors information on the cargo-receptor relationship of virtually all vacuolar proteins. We map binding motifs of Vps10 and the AP-3 complex and identify a novel cargo of the CVT pathway under nutrient-rich conditions. Our data uncover how organelle purification and quantitative proteomics can uncover fundamental insights into organelle biogenesis.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.25.453678', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Reward-dependent selection of feedback gains impacts rapid motor decisions', 'authors': \"[{'name': 'Antoine De Comite'}, {'name': 'Frédéric Crevecoeur'}, {'name': 'Philippe Lefèvre'}]\", 'abstract': '<p>Expected reward is known to affect planning strategies through modulation of movement vigor. Strikingly, although current theories suggest that movement planning consists in selecting a goal-directed control policy, the influence of reward on feedback control strategies remains unknown. Here we investigated this question in three human reaching experiments. First, we varied the explicit reward associated with the goal target and found an overall increase in movement vigor for higher reward targets, highlighted by larger velocities, feedback responses to external loads, and background muscle activity. Then, assuming that larger feedback gains were used to reject perturbations, we sought to investigate whether this effect hindered online decisions to switch to a new target in the presence of multiple successful goals. We indeed observed idiosyncratic switching strategies dependent on both target rewards and movement vigor, such that the more vigorous movements were less likely to switch to a new goal following perturbations. To gain further insight into a causal influence of movement vigor on rapid motor decisions, we demonstrated that biasing the baseline activity and reflex gains by means of a background load evoked a larger proportion of target switches in the direction opposite to the background load associated with lower muscle activity. Our results highlight the competition between movement vigor and flexibility to switch target during movement.</p><sec><title>Significance statement</title><p>Humans can modulate their movement vigor based on the expected reward. However, a potential influence of reward on feedback control has not been documented. Here we investigated reaching control strategies in different contexts associated with explicit rewards for one or multiple goals, while exposed to external perturbations. We report two strategies: reward could either invigorate feedback gains, or promote flexible switches between goals. The engagement of peripheral circuits in the modulation of feedback gains was confirmed by the application of a background load that biased feedback vigor directionally, which evoked differences in switching behavior in the opposite direction. We conclude that feedback vigor and flexible changes in goal are two competing mechanisms to be selected when one interacts with a dynamic environment.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453637', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'LAMP2A regulates the loading of proteins into exosomes', 'authors': \"[{'name': 'João Vasco Ferreira'}, {'name': 'Ana da Rosa Soares'}, {'name': 'José Ramalho'}, {'name': 'Catarina Máximo Carvalho'}, {'name': 'Maria Helena Cardoso'}, {'name': 'Petra Pintado'}, {'name': 'Ana Sofia Carvalho'}, {'name': 'Hans Christian Beck'}, {'name': 'Rune Matthiesen'}, {'name': 'Mónica Zuzarte'}, {'name': 'Henrique Girão'}, {'name': 'Guillaume van Niel'}, {'name': 'Paulo Pereira'}]\", 'abstract': '<p>Exosomes are extracellular vesicles of endosomal origin released by virtually all cell types across metazoans. Exosomes are active vehicles of intercellular communication and can transfer lipids, RNAs and proteins between different cells, tissues or organs. However, the mechanisms that regulate the selective loading of cytosolic proteins into these vesicles are still largely unknow. Here we describe a mechanism whereby proteins containing a pentapeptide sequence, biochemically related to the KFERQ-motif, are loaded into a subpopulation of exosomes in a process that is dependent on the membrane protein LAMP2A. Moreover, this mechanism is independent of the ESCRT machinery components TSG101 and VPS4b and dependent on HSC70, CD63, Alix, Syntenin-1, Rab31 and ceramides. The transcription factor and master regulator of hypoxia HIF1A is loaded into exosomes by this mechanism to transport hypoxia signaling to normoxic cells. Additionally, by tagging fluorescent proteins with KFERQ-like sequences we were able to follow inter-organ transfer of exosomes in zebrafish larvae. Our findings identify LAMP2A as a key component in exosome biogenesis while opening new avenues for exosome engineering by allowing the loading of bioactive proteins by tagging them with KFERQ-like motifs.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453673', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Multivariate mining of an alpaca immune repertoire identifies potent cross-neutralising SARS-CoV-2 nanobodies', 'authors': \"[{'name': 'Leo Hanke'}, {'name': 'Daniel J. Sheward'}, {'name': 'Alec Pankow'}, {'name': 'Laura Perez Vidakovics'}, {'name': 'Vivien Karl'}, {'name': 'Changil Kim'}, {'name': 'Egon Urgard'}, {'name': 'Natalie L. Smith'}, {'name': 'Juan Astorga-Wells'}, {'name': 'Simon Ekström'}, {'name': 'Jonathan M. Coquet'}, {'name': 'Gerald M. McInerney'}, {'name': 'Ben Murrell'}]\", 'abstract': '<p>Conventional approaches to isolate and characterize nanobodies are laborious and cumbersome. Here we combine phage display, multivariate enrichment, and novel sequence analysis techniques to annotate an entire nanobody repertoire from an immunized alpaca. We combine this approach with a streamlined screening strategy to identify numerous anti-SARS-CoV-2 nanobodies, and use neutralization assays and Hydrogen/Deuterium exchange coupled to mass spectrometry (HDX-MS) epitope mapping to characterize their potency and specificity. Epitope mapping revealed that the binding site is a key determinant of neutralization potency, rather than affinity alone. The most potent nanobodies bind to the receptor binding motif of the RBD, directly preventing interaction with the host cell receptor ACE2, and we identify two exceptionally potent members of this category (with monomeric IC50s around 13 and 16 ng/ml). Other nanobodies bind to a more conserved epitope on the side of the RBD, and are able to potently neutralize the SARS-CoV-2 founder virus (42 ng/ml), the beta variant (B.1.351/501Y.V2) (35 ng/ml), and also cross-neutralize the more distantly related SARS-CoV-1 (0.46 μg/ml). The approach presented here is well suited for the screening of phage libraries to identify functional nanobodies for various biomedical and biochemical applications.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453634', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Crowding-induced phase separation of nuclear transport receptors in FG nucleoporin assemblies', 'authors': \"[{'name': 'Luke K. Davis'}, {'name': 'Ian J. Ford'}, {'name': 'Bart W. Hoogenboom'}]\", 'abstract': '<p>The rapid (&lt; 1 ms) transport of biological material to and from the cell nucleus is regulated by the nuclear pore complex (NPC). At the core of the NPC is a permeability barrier consisting of intrinsically disordered Phe-Gly (FG) nucleoporins (FG Nups). Various types of nuclear transport receptors (NTRs) facilitate transport by partitioning in the FG Nup assembly, overcoming the barrier by their affinity to the FG Nups, and comprise a significant fraction of proteins in the NPC barrier. In previous work, we revealed that the experimental binding of the NTRs NTF2 and – the larger – Imp<italic>β</italic> to different planar assemblies of FG Nups follows a universal physical law defined by negative cooperativity, which was further validated by a minimal physical model that treated the FG Nups as flexible homopolymers and the NTRs as uniformly cohesive spheres <bold><italic>Zahn et al. (2016</italic></bold>). Here, we build upon our original study by first parametrizing our model to experimental data, and next to predict the effects of crowding by different types of NTRs. We show how varying the amounts of one type of NTR modulates how the other NTR penetrates the FG Nup assembly. Notably, at similar and physiologically relevant NTR concentrations, our model predicts demixed phases of NTF2 and Imp<italic>β</italic> within the FG Nup assembly. The functional implication of NTR phase separation is that NPCs may sustain separate transport pathways that are determined by inter-NTR competition.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.453442', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Leveraging gene co-regulation to identify gene sets enriched for disease heritability', 'authors': \"[{'name': 'Katherine M. Siewert-Rocks'}, {'name': 'Samuel S. Kim'}, {'name': 'Douglas W. Yao'}, {'name': 'Huwenbo Shi'}, {'name': 'Alkes L. Price'}]\", 'abstract': '<p>Identifying gene sets that are associated to disease can provide valuable biological knowledge, but a fundamental challenge of gene set analyses of GWAS data is linking disease-associated SNPs to genes. Transcriptome-wide association studies (TWAS) can be used to detect associations between the genetically predicted expression of a gene and disease risk, thus implicating candidate disease genes. However, causal disease genes at TWAS-associated loci generally remain unknown due to gene co-regulation, which leads to correlations across genes in predicted expression. We developed a new method, gene co-regulation score (GCSC) regression, to identify gene sets that are enriched for disease heritability explained by the predicted expression of causal disease genes in the gene set. GCSC regresses TWAS chi-square statistics on gene co-regulation scores reflecting correlations in predicted gene expression; GCSC determines that a gene set is enriched for disease heritability if genes with high co-regulation to the gene set have higher TWAS chi-square statistics than genes with low co-regulation to the gene set, beyond what is expected based on co-regulation to all genes. We verified via simulations that GCSC is well-calibrated, and well-powered to identify gene sets that are enriched for disease heritability explained by predicted expression. We applied GCSC to gene expression data from GTEx (48 tissues) and GWAS summary statistics for 43 independent diseases and complex traits (average <italic>N</italic> =344K), analyzing a broad set of biological pathways and specifically expressed gene sets. We identified many enriched gene sets, recapitulating known biology. For Alzheimer’s disease, we detected evidence of an immune basis, and specifically a role for antigen presentation, in analyses of both biological pathways and specifically expressed gene sets. Our results highlight the advantages of leveraging gene co-regulation within the TWAS framework to identify gene sets associated to disease.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.21260551', 'tweets': 14, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Willingness to receive vaccination against COVID-19: results from a large nationally representative Australian population survey', 'authors': \"[{'name': 'Sharon R Davis'}, {'name': 'Rosario D Ampon'}, {'name': 'Leanne M Poulos'}, {'name': 'Guy B Marks'}, {'name': 'Brett G Toelle'}, {'name': 'Helen K Reddel'}]\", 'abstract': '<p>We surveyed 10,024 Australians regarding COVID-19 vaccine willingness. Overall, 59.9% indicated yes, 13.9% no and 26.3% unsure/don’t know. Vaccine willingness was higher in males, and increased with increasing education and socioeconomic advantage. Results contrast with earlier, smaller Australian surveys regarding vaccination willingness and confirm the need for targeted vaccination information.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.25.453674', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Role of non-specific interactions in the phase-separation and maturation of macromolecules', 'authors': \"[{'name': 'Rakesh Krishnan'}, {'name': 'Srivastav Ranganathan'}, {'name': 'Samir K Maji'}, {'name': 'Ranjith Padinhateeri'}]\", 'abstract': '<p>Phase separation of biomolecules could be mediated by both specific and non-specific interactions. How the interplay between non-specific and specific interactions along with polymer entropy influences phase separation is an open question. We address this question by simulating self-associating molecules as polymer chains with a short core stretch that forms the specifically interacting functional interface, and longer non-core regions that participate in non-specific/promiscuous interactions. Our results show that the interplay of specific (strength, <italic>ϵ</italic><sub>sp</sub>) and non-specific interactions (strength, <italic>ϵ</italic><sub>ns</sub>) could result in phase separation of polymers and its transition to solid-like aggregates (mature state). In the absence of <italic>ϵ</italic><sub>ns</sub>, the polymer chains do not dwell long enough in the vicinity of each other to undergo phase separation and transition into a mature state. On the other hand, in the limit of strong <italic>ϵ</italic><sub>ns</sub>, the assemblies cannot transition into the mature state because they get kinetically trapped, suggesting an optimal range of interactions favoring mature multimers. In the scenario where only a fraction (N<sub>frac</sub>) of the non-core regions participate in attractive interactions, we find that a slight modifications to either <italic>ϵ</italic><sub>ns</sub> or N<sub>frac</sub> can result in a dramatically altered self-assembled states. Using a combination of heterogeneous and homogeneous mix of polymers, we establish how this interplay between interaction energies dictates the propensity of biomolecules to find the correct binding partner at dilute concentrations in crowded environments.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453610', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Adapted protocol for Saccharibacteria co-cultivation: two new members join the club of Candidate Phyla radiation', 'authors': \"[{'name': 'Ahmad Ibrahim'}, {'name': 'Mohamad Maatouk'}, {'name': 'Andriamiharimamy Rajaonison'}, {'name': 'Rita Zgheib'}, {'name': 'Gabriel Haddad'}, {'name': 'Jacques Bou-Khalil'}, {'name': 'Didier Raoult'}, {'name': 'Fadi Bittar'}]\", 'abstract': '<p>The growing application of metagenomics to different ecological and microbiome niches in recent years has enhanced our knowledge of global microbial biodiversity. Among these abundant and widespread microbes, Candidate Phyla Radiation or CPR have been recognised as representing a large proportion of the microbial kingdom (&gt; 26%). CPR are characterised by their obligate symbiotic or exo-parasitic activity with other microbial hosts, mainly bacteria. Currently, isolating CPR is still considered challenging for microbiologists. The idea of this study was to develop an adapted protocol for the co-culture of CPR with a suitable bacterial host. Based on various sputa, we tried to purify CPR (<italic>Saccharibacteria</italic> members) and to cultivate them with pure hosts. This protocol was monitored by real-time PCR quantification using a specific system for <italic>Saccharibacteria</italic> designed in this study, as well as by electron microscopy and sequencing. We succeeded in co-culturing and sequencing a complete genome of two new <italic>Saccharibacteria</italic> species: <italic>Candidatus</italic> Minimicrobia naudis and <italic>Candidatus</italic> Minimicrobia vallesae. In addition, we noticed a decrease in the Ct number of <italic>Saccharibacteria</italic>, and a significant multiplication through their physical association with <italic>Schaalia odontolytica</italic> strains in the enriched medium that we developed. This work may help bridge gaps in the genomic database by providing new CPR members and, in the future, their currently unknown characteristics may be revealed.</p><sec><title>IMPORTANCE</title><p>In this study, the first real-time PCR system has been developed. This technique is able to quantify specifically <italic>Saccharibacteria</italic> members in any sample of interest in order to investigate their prevalence. In addition, another easy, specific and sensitive protocol has been developed to maintain the viability of <italic>Saccharibacteria</italic> cells in an enriched medium with their bacterial host. The use of this protocol subsequently facilitates studying the phenotypic characteristics of CPR and their physical interactions with bacterial species, as well as the sequencing of new genomes to improve the current database.</p></sec>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.18.21259977', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Neurology', 'covid': False, 'title': 'Altered functional connectome hierarchy with gene expression signatures in newly-diagnosed focal epilepsy', 'authors': \"[{'name': 'Christophe E de Bézenac'}, {'name': 'Lorenzo Caciagli'}, {'name': 'Batil K Alonazi'}, {'name': 'Boris C Bernhardt'}, {'name': 'Anthony G Marson'}, {'name': 'Simon S Keller'}]\", 'abstract': '<sec><title>Objective</title><p>Neuroimaging research is providing insights into epilepsy as a disorder of brain connectivity linked to functional impairments which may have an identifiable genetic component. This case-control study aims to identify imbalances in a functional connectome dimension spanning from unimodal to transmodal networks and explore the potential genetic basis of such alterations in patients with newly diagnosed focal epilepsy (NDfE).</p></sec><sec><title>Methods</title><p>We used gradient-based analysis of resting-sate fMRI data comparing cortical gradient maps in patients with NDfE (n = 27) to age and sex-matched controls (n = 36). Using a brain-wide gene expression dataset, gene combinations associated with altered brain regions were then entered into an enrichment analysis.</p></sec><sec><title>Results</title><p>We found an increased differentiation of connectivity profiles between unimodal and transmodal networks in NDfE, which was particularly pronounced in the patients with persistent seizures at 12-months follow-up (n=10). Differences corresponded to gradient score reductions in a visual network and increases in limbic and default mode systems which subserve higher-level cognition. Cortical difference maps were spatially correlated with regional expression of a weighted gene combination. These genes were enriched for disease and ontology terms and pathways previously associated with epilepsy and seizure susceptibility.</p></sec><sec><title>Interpretations</title><p>Large-scale functional hierarchy may be altered from in focal epilepsy from diagnosis and correlate with response to treatment. Combining functional neuroimaging and transcriptional data analysis may provide a framework for understanding the wide-ranging impairments associated with the disorder and mechanistic insight into how gene processes may drive alterations in brain function mediating the genetic risk of epilepsy.</p></sec>', 'posted': '2021-07-22'}, {'doi': '10.1101/2021.07.25.453574', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Bats host the most virulent—but not the most dangerous—zoonotic viruses', 'authors': \"[{'name': 'Sarah Guth'}, {'name': 'Nardus Mollentze'}, {'name': 'Katia Renault'}, {'name': 'Daniel G. Streicker'}, {'name': 'Elisa Visher'}, {'name': 'Mike Boots'}, {'name': 'Cara E. Brook'}]\", 'abstract': '<p>Identifying virus characteristics associated with the largest public health impacts on human populations is critical to informing zoonotic risk assessments and surveillance strategies. Efforts to assess “zoonotic risk” often use trait-based analyses to identify which viral and reservoir host groups are most likely to source zoonoses but have not fully addressed how and why the impacts of zoonotic viruses vary in terms of disease severity (‘virulence’), capacity to spread within human populations (‘transmissibility’), or total human mortality (‘death burden’). We analyzed trends in human case fatality rates, transmission capacities, and total death burdens across a comprehensive dataset of mammalian and avian zoonotic viruses. Bats harbor the most virulent zoonotic viruses even when compared to birds, which alongside bats, have been hypothesized to be “special” zoonotic reservoirs due to molecular adaptations that support the physiology of flight. Reservoir host groups more closely related to humans—in particular, Primates—harbor less virulent, but more highly transmissible viruses. Importantly, disproportionately high human death burden, arguably the most important metric of zoonotic risk, is not associated with any animal reservoir, including bats. Our data demonstrate that mechanisms driving death burdens are diverse and often contradict trait-based predictions. Ultimately, total human mortality is dependent on context-specific epidemiological dynamics, which are shaped by a combination of viral traits and conditions in the animal host population and across and beyond the human-animal interface. Understanding the conditions that predict high zoonotic burden in humans will require longitudinal studies of epidemiological dynamics in wildlife and human populations.</p><sec><title>Significance statement</title><p>The clear need to mitigate zoonotic risk has fueled increased viral discovery in specific reservoir host taxa. We show that a combination of viral and reservoir traits can predict zoonotic virus virulence and transmissibility in humans, supporting the hypothesis that bats harbor exceptionally virulent zoonoses. However, pandemic prevention requires thinking beyond zoonotic capacity, virulence, and transmissibility to consider collective ‘burden’ on human health. For this, viral discovery targeting specific reservoirs may be inefficient as death burden correlates with viral, not reservoir, traits, and depends on context-specific epidemiological dynamics across and beyond the human-animal interface. These findings suggest that longitudinal studies of viral dynamics in reservoir and spillover host populations may offer the most effective strategy for mitigating zoonotic risk.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453666', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Identification of a divergent basal body assembly protein involved in land plant spermatogenesis', 'authors': \"[{'name': 'Shizuka Koshimizu'}, {'name': 'Naoki Minamino'}, {'name': 'Tomoaki Nishiyama'}, {'name': 'Emiko Yoro'}, {'name': 'Kazuo Ebine'}, {'name': 'Keiko Sakakibara'}, {'name': 'Takashi Ueda'}, {'name': 'Kentaro Yano'}]\", 'abstract': '<p>Oogamy is a form of sexual reproduction and evolved independently in animals, fungi, and plants. In streptophyte plants, Charophyceae, Coleochaetophyceae, bryophytes, lycophytes, ferns (monilophytes), and some gymnosperms (Cycads and Ginkgo) utilize spermatozoids as the male gamete. Plant spermatozoids commonly possess characteristic structures such as the spline, which consists of a microtubule array, the multilayered structure (MLS) in which the uppermost layer is continuum of the spline, and multiple flagella. However, the molecular mechanisms underpinning plant spermatogenesis remain to be elucidated. To identify the genes involved in plant spermatogenesis, we performed computational analyses and successfully found deeply divergent BLD10s by combining multiple methods and omics-data. We then validated the functions of candidate genes in the liverwort <italic>Marchantia polymorpha</italic> and the moss <italic>Physcomitrium patens</italic> and found that MpBLD10 and PpBLD10 are required for normal basal body and flagella formation. Mp<italic>bld10</italic> mutants exhibited defects in remodeling of the cytoplasm and nucleus during spermatozoid formation, thus MpBLD10 should be involved in chromatin reorganization and elimination of the cytoplasm during spermiogenesis. Streptophyte BLD10s are orthologous to BLD10/CEP135 family proteins, which function in basal body assembly, but we found that BLD10s evolved especially fast in land plants and MpBLD10 might obtain additional functions in spermatozoid formation through the fast molecular evolution. This study provides a successful example of combinatorial study from evolutionary and molecular genetic perspectives that elucidated a function of the key protein of the basal body formation that fast evolved in land plants.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.452477', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Trial-wise exposure to visual appetitive cues increases physiological arousal but not temporal discounting', 'authors': \"[{'name': 'Kilian Knauth'}, {'name': 'Jan Peters'}]\", 'abstract': '<p>Humans and many animals devalue future rewards as a function of time (temporal discounting). Increased discounting has been linked to various psychiatric conditions, including substance-use-disorders, behavioral addictions and obesity. Despite its high intra-individual stability, temporal discounting is partly under contextual control. One prominent manipulation that has been linked to increases in discounting is the exposure to highly arousing appetitive cues. However, results from trial-wise cue exposure studies appear highly mixed, and changes in physiological arousal were not adequately controlled. Here we tested the effects of appetitive (erotic), aversive and neutral visual cues on temporal discounting in thirty-five healthy male participants. The contribution of single-trial physiological arousal was assessed using comprehensive monitoring of autonomic activity (pupil size, heart rate, electrodermal activity). Physiological arousal was elevated following aversive and in particular erotic cues. In contrast to our pre-registered hypothesis, if anything, we observed <italic>decreased</italic> temporal discounting following erotic cue exposure. Aversive cues tended to increase decision noise. Computational modeling revealed that trial-wise arousal only accounted for minor variance over and above aversive and erotic condition effects, arguing against a general effect of physiological arousal on temporal discounting.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453708', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The effective family size of immigrant founders predicts their long-term demographic outcome: from Quebec settlers to their 20th-century descendants', 'authors': \"[{'name': 'Tommy Harding'}, {'name': 'Emmanuel Milot'}, {'name': 'Helene Vezina'}, {'name': 'Damian Labuda'}]\", 'abstract': '<p>Human evolution involves population splits, size fluctuations, founder effects, and admixture. Population history reconstruction based on genetic diversity data routinely relies on simple demographic models while projecting the past. Yet, no specific demographic assumptions are needed to understand the genetic structure of the founder population of Quebec. Because genealogy and genetics are intimately related, we used descending genealogies of this population to pursue the fate of its founder lineages. Maternal and paternal lines reflect the transmission of mtDNA and the Y-chromosome, respectively. We followed their transmission in real-time, from the 17th century down to its 20th-century population. We counted the number of married children of immigrants (i.e., their effective family size, EFS), estimated the proportion of successful immigrants in terms of their survival ratio, and assessed net growth rates and extinction. Likewise, we evaluated the same parameters for their Quebec-born descendants. The survival ratio of the first immigrants was the highest and declined over time in association with the decreasing immigrants EFS. Parents with high EFS left plentiful married progeny, putting EFS as the most important variable determining the parental demographic success throughout time for generations ahead. The 17th and 18th-century immigrants bear the most remarkable demographic and genetic impact on the 20th-century population of Quebec. Lessons learned from Quebec genealogies can teach us about the consequences of founder effects and migrations through real peoples history. The effective family size of immigrant founders predicts their long-term demographic outcome.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.25.453671', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'MetaBinner: a high-performance and stand-alone ensemble binning method to recover individual genomes from complex microbial communities', 'authors': \"[{'name': 'Ziye Wang'}, {'name': 'Pingqin Huang'}, {'name': 'Ronghui You'}, {'name': 'Fengzhu Sun'}, {'name': 'Shanfeng Zhu'}]\", 'abstract': '<p>Binning is an essential procedure during metagenomic data analysis. However, the available individual binning methods usually do not simultaneously fully use different features or biological information. Furthermore, it is challenging to integrate multiple binning results efficiently and effectively. Therefore, we developed an ensemble binner, MetaBinner, which generates component results with multiple types of features and utilizes single-copy gene (SCG) information for k-means initialization. It then utilizes a two-step ensemble strategy based on SCGs to integrate the component results. Extensive experimental results over three large-scale simulated datasets and one real-world dataset demonstrate that MetaBinner outperforms other state-of-the-art individual binners and ensemble binners. MetaBinner is freely available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/ziyewang/MetaBinner\">https://github.com/ziyewang/MetaBinner</ext-link>.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261042', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Robust antibody levels in both diabetic and non-diabetic individuals after BNT162b2 mRNA COVID-19 vaccination', 'authors': \"[{'name': 'Hamad Ali'}, {'name': 'Abdelmohsen Al-Terki'}, {'name': 'Sardar Sindhu'}, {'name': 'Barrak Alahmad'}, {'name': 'Maha Hammad'}, {'name': 'Salman Al-Sabah'}, {'name': 'Mohammad Alghounaim'}, {'name': 'Mohammad H. Jamal'}, {'name': 'Ali Aldei'}, {'name': 'Mohammad J. Mairza'}, {'name': 'Maitham Husain'}, {'name': 'Sriraman Deverajan'}, {'name': 'Rasheed Ahmad'}, {'name': 'Preethi Cherian'}, {'name': 'Irina Alkhairi'}, {'name': 'Abdullah Alkandari'}, {'name': 'Jehad Abubaker'}, {'name': 'Mohamed Abu-Farha'}, {'name': 'Fahd Al-Mulla'}]\", 'abstract': '<p>The emergence of effective vaccines for COVID-19 has been welcomed by the world with great optimism. Given their increased susceptibility to COVID-19, the question arises whether individuals with type-2 diabetes mellitus (T2DM) and other metabolic conditions can respond effectively to the mRNA-based vaccine. We aimed to evaluate the levels of anti-SARS-CoV-2 IgG and neutralizing antibodies in people with T2DM and/or other metabolic risk factors (hypertension and obesity) compared to those without. This study included 262 people that took two doses of BNT162b2 (Pfizer–BioNTech) mRNA vaccine. Both T2DM and non-diabetic individuals had a robust response to vaccination as demonstrated by their high antibody titers. However, both SARS-CoV-2 IgG and neutralizing antibodies titers were lower in people with T2DM. Their levels were 154±49.1 vs. 138±59.4BAU/mL for IgG and 87.1±11.6 vs. 79.7±19.5% for neutralizing antibodies in individuals without diabetes compared to those with T2DM, respectively. In a multiple linear regression adjusted for individual characteristics, comorbidities, previous COVID-19 infection and duration since second vaccine dose, diabetics had 13.86 BAU/ml (95%CI: −27.08 to −0.64BAU/ml, p=0.041) less IgG antibodies and 4.42% (95%CI: −8.53 to −0.32%, p=0.036) less neutralizing antibodies than non-diabetics. Hypertension and obesity did not show significant changes in antibody titers. Taken together, both type-2 diabetic and non-diabetic individuals elicited strong immune responses to SARS-CoV-2 BNT162b2 mRNA vaccine; nonetheless, lower levels were seen in people with diabetes. Continuous monitoring of the antibody levels might be a good indicator to guide personalized needs for further booster shots to maintain adaptive immunity.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453744', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Global protein responses of multi-drug resistant plasmid containing Escherichia coli to ampicillin, cefotaxime, imipenem and ciprofloxacin', 'authors': \"[{'name': 'Anatte Margalit'}, {'name': 'James C. Carolan'}, {'name': 'Fiona Walsh'}]\", 'abstract': '<p>Antimicrobial resistance (AMR) and multi-drug resistance (MDR) in pathogenic bacteria are frequently mediated by plasmids. However, plasmids do not exist in isolation but rather require the bacterial host interaction in order to produce the AMR phenotype. This study aimed to utilise mass spectrometry-based proteomics to reveal the plasmid and chromosomally derived protein profile of <italic>Escherichia coli</italic> under antimicrobial stress. This was achieved by comparing the proteomes of <italic>E. coli</italic> containing the MDR pEK499 plasmid, under ampicillin, cefotaxime, imipenem or ciprofloxacin stress with the proteomes of these bacteria grown in the absence of antimicrobial. Our analysis identified statistically significant differentially abundant proteins common to groups exposed to the β-lactam antimicrobials but not ciprofloxacin, indicating a β-lactam stress response to exposure from this class of drugs, irrespective of β-lactam resistance or susceptibility. These include ecotin and free methionine-R-sulfoxide reductase. These data also identified distinct differences in the cellular response to each β-lactam. Data arising from comparisons of the proteomes of ciprofloxacin-treated <italic>E. coli</italic> and controls detected an increase in the relative abundance of proteins associated with ribosomes, translation, the TCA-cycle and several proteins associated with detoxification and a decrease in the relative abundances of proteins associated with stress response, including oxidative stress. We identified changes in proteins associated with persister formation in the presence of ciprofloxacin but not the β-lactams. The plasmid proteome differed across each treatment and did not follow the pattern of antimicrobial – AMR protein associations. For example, a relative increase in the amount of blaCTX-M-15 in the presence of cefotaxime and ciprofloxacin but not the other β-lactams, suggesting regulation of the blaCTX-M-15 protein production. The proteomic data from the this study provided novel insights into the proteins produced from the chromosome and plasmid under different antimicrobial stresses. These data also identified novel proteins not previously associated with AMR or antimicrobials responses in pathogens, which may well represent potential targets of AMR inhibition.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453807', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'The benefits of merging two tracking approaches: passive acoustic telemetry combined with active acoustic tracking by transects provide new insights into riverine migration by salmonid smolts', 'authors': \"[{'name': 'Louise Chavarie'}, {'name': 'Hannele M. Honkanen'}, {'name': 'Matthew Newton'}, {'name': 'Jessie M. Lilly'}, {'name': 'Hannah R. Greetham'}, {'name': 'Colin E. Adams'}]\", 'abstract': '<p>The process of smolting is a critical phase in the life-cycle of anadromous salmonids and it has been associated with substantial rates of mortality. Survival during freshwater and marine migration is known to have population level effects, thus an understanding of the patterns of mortality has the potential to yield important insights into population bottlenecks. Despite important advancements in tracking techniques, the specifics of mortality events in anadromous salmonids during their initial migration to sea remains somewhat elusive. Here, we develop a framework combining spatial and temporal detections of smolt riverine migration from two tracking techniques, which enable inferences to be made about mortality locations, causes, and rates. We embed this framework into a fine-scale behaviour study of migration and social structure. In this study, we demonstrate that during their initial riverine transitional phase, smolts were particularly vulnerable to predators. Specifically, avian predation appeared to be the main cause of mortality (42%), although piscine predation events were not trivial (14%). Our results suggested some direct and indirect tagging-induced mortality (e.g., through increased predation vulnerability), which highlights the importance of determining tagging mortality in a telemetry study to ensure adequate interpretation of migration success. There is evidence that predation induced selection on smolt morphology. Unsuccessful river salmon migrants with a phenotype comprising a shorter head and jaw and smaller eye had a higher probability of mortality in the later parts of riverine migration where avian and aquatic predation mortality dominated. In contrast, mortality earlier in river migration was independent of phenotype, most likely a result of tagging effects. Successfully river migrants were found to be interacting with each other, often in a pair or as a trio, indicating that a few individuals of Atlantic salmon and sea trout formed notable intra- and interspecific social associations. However, the heterogeneity of the social associations of successful migrants did not support the assumption that social network features attributed any clear specific benefits of reduced predation risk. Overall, by estimating migration loss and its variability, our study framework should help to guide management actions to mitigate the widespread population declines these species are currently facing.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453733', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'AGO1 regulates major satellite transcripts and H3K9me3 distribution at pericentromeric regions in mESCs', 'authors': \"[{'name': 'Madlen Müller'}, {'name': 'Tara Fäh'}, {'name': 'Moritz Schäfer'}, {'name': 'Janina Luitz'}, {'name': 'Patrick Stalder'}, {'name': 'Rajika Arora'}, {'name': 'Richard Patryk Ngondo'}, {'name': 'Constance Ciaudo'}]\", 'abstract': '<p>In the past years, several studies reported nuclear roles for the Argonaute (AGO) proteins associating them with transcriptional activation or repression, alternative splicing and, chromatin organization. However, as most of these experiments have been conducted in human cancer cell lines, the nuclear functions of the AGO proteins in mouse early embryonic development still remains elusive. In this study, we investigated possible nuclear functions of the AGO proteins in mouse Embryonic Stem Cells (mESCs). By biochemical assays, we observed that AGO1 and AGO2 are present in a small fraction in the nucleus and even less on chromatin in mESCs. To profile the nuclear interactome of the AGO proteins, we performed immunoprecipitation followed by Mass Spectrometry and identified three novel nuclear interactors for AGO1, namely DNMT3a, HP1<italic>α</italic>, and ATRX. These interactors are well-known proteins involved in the establishment and maintenance of heterochromatin at pericentromeric regions. Indeed, upon depletion of <italic>Ago1,</italic> we observed a specific redistribution of the heterochromatin protein HP1<italic>α</italic> and the repressive histone mark H3K9me3, away from pericentromeric regions. Furthermore, these regions are characterized by AT-rich tandem repeats known as major satellite sequences. We demonstrated that major satellite transcripts are strongly upregulated in <italic>Ago1_KO</italic> mESCs. Interestingly, this phenotype was not caused by the loss of genome integrity at pericentromeres, as these could still form normally in <italic>Ago1</italic>_KO mESCs. Lastly, we showed that specific microRNAs loaded in AGO1, regulate the expression of the major satellite transcripts. Overall, our results demonstrate for the first time a novel role for AGO1 in regulating major satellite transcripts and localization of HP1<italic>α</italic> and H3K9me3 at pericentromeres in mESCs.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453597', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Generalizing Bayesian phylogenetics to infer shared evolutionary events', 'authors': \"[{'name': 'Jamie R. Oaks'}, {'name': 'Perry L. Wood'}]\", 'abstract': '<p>Many processes of biological diversification can affect multiple evolutionary lineages. Examples include multiple members of a gene family diverging when a region of a chromosome is duplicated, multiple viral strains diverging at a “super-spreading” event, and a geological event fragmenting whole communities of species. It is difficult to test for patterns predicted by such processes, because all phylogenetic methods assume that lineages diverge independently. We introduce a general Bayesian framework to relax the assumption of independent divergences during phylogenetic inference, and test for patterns predicted by processes of diversification that affect multiple evolutionary lineages. Using simulations, we find our new method accurately infers shared divergence events when they occur, and performs as well as current methods when divergnces are independent. We apply our new approach to genomic data from two genera of geckos from across the Philippines to test if past changes to the islands’ landscape caused bursts of speciation. Unlike previous analyses restricted to only pairs of populations, we find evidence for patterns of shared divergences. By generalizing the space of phylogenetic trees in a way that is independent from the likelihood model, our approach opens many avenues for future research.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453355', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Inhibition of striatal SEZ6 by miR-3594-5p is a drug-specific marker for late-stage heroin intake escalation', 'authors': \"[{'name': 'Magalie Lenoir'}, {'name': 'Isabella Bondi'}, {'name': 'Loïc Clemenceau'}, {'name': 'Isabelle Nondier'}, {'name': 'Margaux Ballé'}, {'name': 'Sébastien Jacques'}, {'name': 'Angéline Duché'}, {'name': 'Corinne Canestrelli'}, {'name': 'Séverine Martin-Lannerée'}, {'name': 'Sophie Mouillet-Richard'}, {'name': 'Jenny M. Gunnersen'}, {'name': 'Serge H. Ahmed'}, {'name': 'Nicolas Marie'}, {'name': 'Florence Noble'}]\", 'abstract': '<p>Escalation of drug use is a hallmark stage in the transition to addiction and uncovering its underlying brain molecular mechanisms constitutes a considerable challenge. Here, we report in rats with extended access to heroin for self-administration that miR-3594-5p was upregulated in the dorsal striatum at late, but not early, stages during escalation when excessive heroin intake plateaued. Striatal miR-3594-5p bound directly to the 3’UTR region of <italic>Sez6</italic> transcript and inhibited its expression, thereby decreasing the mature form of the translated SEZ6 protein. This miR-3594-5p/<italic>Sez6</italic> interaction was specific to heroin, as it was not observed with cocaine, and correlated with the severity of heroin intake escalation. Our findings reveal that miRNA alterations during escalation of drug self-administration are spatially and temporally regulated and drug-specific.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453341', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Trait-trait relationships and functional tradeoffs vary with genome size in prokaryotes', 'authors': \"[{'name': 'Sara Beier'}, {'name': 'Johannes Werner'}, {'name': 'Thierry Bouvier'}, {'name': 'Nicolas Mouquet'}, {'name': 'Cyrille Violle'}]\", 'abstract': '<p>We report genomic traits that have been associated with the life history of prokaryotes and highlight conflicting findings concerning earlier observed trait correlations and tradeoffs. In order to address possible explanations for these contradictions we examined trait-trait variations of 11 genomic traits from ∼ 17,000 sequenced genomes. The studied trait-trait variations suggested: (i) the predominance of two resistance and resilience-related orthogonal axes, (ii) an overlap between a resilience axis and an axis of resource usage efficiency. These findings imply that resistance associated traits of prokaryotes are globally decoupled from resilience and resource use efficiencies associated traits. However, further inspection of pairwise scatterplots showed that resistance and resilience traits tended to be positively related for genomes up to roughly five million base pairs and negatively for larger genomes. This in turn precludes a globally consistent assignment of prokaryote genomic traits to the competitor - stress-tolerator -ruderal (CSR) schema that sorts species depending on their location along disturbance and productivity gradients into three ecological strategies and may serve as an explanation for conflicting findings from earlier studies. All reviewed genomic traits featured significant phylogenetic signals and we propose that our trait table can be applied to extrapolate genomic traits from taxonomic marker genes. This will enable to empirically evaluate the assembly of these genomic traits in prokaryotic communities from different habitats and under different productivity and disturbance scenarios as predicted via the resistance-resilience framework formulated here.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.25.453699', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Supra-physiological levels of gibberellins/DELLAs modify the root cell size/number and the root architecture in root tips of A. thaliana seedlings. Connections to the root hair patterning and abundance', 'authors': \"[{'name': 'Iva McCarthy-Suárez'}]\", 'abstract': '<p>A previous study (McCarthy-Suárez, 2021) showed that growing <italic>A. thaliana</italic> seedlings for 5 days under excessive levels of gibberellins (GAs)/DELLAs altered the arrangement, shape and frequency of root hairs in root tips. Because no changes in the distribution or number of root hairs occurred when the gai-1 (gibberellin-insensitive-1) DELLA was over-expressed at the root epidermis, it was concluded that the GAs/DELLAs might regulate the root hair patterning and abundance in <italic>A. thaliana</italic> seedlings by acting from the root sub-epidermal tissues. In the present study, microscopy analyses showed that excessive levels of GAs/DELLAs also modified the size and number of root tip cells in <italic>A. thaliana</italic> seedlings. While excessive DELLAs shortened and widened the root epidermal, cortical, endodermal and pericycle cells, excessive GAs, excepting the epidermal cells, generally narrowed them. However, no changes of root cell size occurred when gai-1 was over-expressed at the root epidermis. In addition, high levels of DELLAs often induced extra cells at the root epidermis, cortex, endodermis and pericycle, whereas high levels of GAs sometimes induced extra cells at the root cortex and pericycle. On the other hand, excessive levels of DELLAs enhanced the outgrowth of lateral roots in root tips, unlike excessive levels of GAs. Thus, the results of this study suggest that supra-physiological levels of GAs/DELLAs might modify the size/number of root tip cells by acting from the root sub-epidermal tissues. This, in turn, might impact on the patterning and abundance of root hairs and on the root architecture.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.21258785', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Gastroenterology', 'covid': True, 'title': 'Proton pump inhibitor use is not associated with severe COVID-19 related outcomes: A propensity score weighted analysis of a national veteran cohort', 'authors': \"[{'name': 'Shailja C. Shah'}, {'name': 'Alese Halvorson'}, {'name': 'Brandon McBay'}, {'name': 'Chad Dorn'}, {'name': 'Otis Wilson'}, {'name': 'Sony Tuteja'}, {'name': 'Kyong-Mi Chang'}, {'name': 'Ayako Suzuki'}, {'name': 'Christine Hunt'}, {'name': 'Richard Hauger'}, {'name': 'Kelly Cho'}, {'name': 'Edward Siew'}, {'name': 'Michael Matheny'}, {'name': 'Adriana Hung'}, {'name': 'Robert Greevy'}, {'name': 'Christianne Roumie'}]\", 'abstract': '<p>Background and Aims: Low pH deactivates most pathogens, including coronaviruses. Proton pump inhibitors (PPIs) are potent gastric acid suppressing medications. Whether PPI use vs non-use is associated with severe Coronavirus disease-2019 (COVID-19) outcomes remains uncertain. We aimed to compare severe COVID-19 outcomes between current outpatient PPI users and non-users.\\n\\nMethods: We conducted a retrospective propensity score-weighted analysis of a national cohort of US veterans with established care who tested positive for severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) through January 9, 2021, and who had 60 days of follow-up. The positive test date was the index date. Current outpatient PPI use up to and including the index date (primary exposure) was compared to non-use, defined as no PPI prescription fill in the 365 days prior to the index date. The primary outcome was a composite of use of mechanical ventilation or death within 60 days. Weighted logistic regression models evaluated severe COVID-19 outcomes between current PPI users vs non-users.\\n\\nResults: Of 97,674 Veterans with SARS-CoV-2 testing, 14,958 tested positive (6262 [41.9%] current PPI users, 8696 [58.1%] non-users) and comprised the analytic cohort. After weighting, all covariates were well-balanced. In the weighted cohort, there was no difference in the primary composite outcome (8.2% vs 8.0%; OR 1.03, 95% CI 0.91-1.16), secondary composite outcome, nor individual component outcomes between current PPI users and non-users. There was no significant interaction between age and PPI use on outcomes.\\n\\nConclusion: Among patients with SARS-CoV-2 infection, current PPI use vs non-use is not associated with severe COVID-19 outcomes.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.24.453449', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Heat-inactivated modified vaccinia virus Ankara boosts Th1-biased cellular and humoral immune responses as a vaccine adjuvant by activating the STING-mediated cytosolic DNA-sensing pathway', 'authors': \"[{'name': 'Ning Yang'}, {'name': 'Aitor Garzia'}, {'name': 'Cindy Meyer'}, {'name': 'Thomas Tuschl'}, {'name': 'Taha Merghoub'}, {'name': 'Jedd D. Wolchok'}, {'name': 'Liang Deng'}]\", 'abstract': '<sec><title>Background</title><p>Protein or peptide-based subunit vaccines are promising platforms for combating human cancers and infectious diseases. However, one primary concern regarding subunit vaccines is the relatively weak immune responses induced by proteins or peptides. Therefore, developing novel and effective vaccine adjuvants is critical for the success of subunit vaccines. Modified vaccinia virus (MVA) is a safe and effective vaccine against smallpox and monkeypox. In this study, we explored the potential of heat-inactivated MVA (heat-iMVA) as a novel vaccine adjuvant.</p></sec><sec><title>Methods</title><p>We co-administered heat-iMVA with a model antigen, chicken ovalbumin (OVA), either intramuscularly or subcutaneously twice, two weeks apart, and analyzed anti-OVA specific CD8<sup>+</sup> and CD4<sup>+</sup> T cells in the spleens and skin draining lymph nodes (dLNs) and serum anti-OVA IgG1 and IgG2c antibodies. We also compared the adjuvanticity of heat-iMVA with several known vaccine adjuvants, including complete Freund’s adjuvant (CFA) and AddaVax, an MF59-like preclinical grade nano-emulsion. In addition, we tested whether co-administration of heat-iMVA plus tumor neoantigen peptides or irradiated tumor cells improves antitumor efficacy in a B16-F10 therapeutic vaccination model. Using Stimulator of Interferon Genes (STING) or Batf3-deficient mice, we evaluated the contribution of the STING pathway and Batf3-dependent CD103<sup>+</sup>/CD8α DCs in heat-iMVA-induced immunity.</p></sec><sec><title>Results</title><p>Co-administration of protein- or peptide-based immunogens with heat-iMVA dramatically enhances Th1-biased cellular and humoral immune responses. This adjuvant effect of heat-iMVA is dependent on the STING-mediated cytosolic DNA-sensing pathway, and the antigen-specific CD8<sup>+</sup> T cell response requires Batf3-dependent CD103<sup>+</sup>/CD8α<sup>+</sup> dendritic cells (DCs). Heat-iMVA infection of bone marrow-derived DCs (BMDCs) promoted antigen cross-presentation, whereas live MVA infection did not. RNA-seq analyses revealed that heat-iMVA is a more potent activator of the STING pathway than live MVA. Additionally, combining tumor neoantigen peptides or irradiated tumor cells with heat-iMVA delayed tumor growth and extended the median survival in B16-F10 therapeutic vaccination models.</p></sec><sec><title>Conclusions</title><p>Heat-iMVA induces type I interferon (IFN) production and antigen cross-presentation via a STING-dependent mechanism in DCs. Co-administration of heat-iMVA with peptide antigen generates strong Th1-biased cellular and humoral immunity. Collectively, our results demonstrate that heat-iMVA is a safe and potent vaccine adjuvant.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453553', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Community health and human-animal contacts on the edges of Bwindi Impenetrable National Park, Uganda', 'authors': \"[{'name': 'Renata L. Muylaert'}, {'name': 'Ben Davidson'}, {'name': 'Alex Ngabirano'}, {'name': 'Gladys Kalema-Zikusoka'}, {'name': 'Hayley MacGregor'}, {'name': 'James O. Lloyd-Smith'}, {'name': 'Ahmed Fayaz'}, {'name': 'Matthew A. Knox'}, {'name': 'David T. S. Hayman'}]\", 'abstract': '<p>Cross-species transmission of pathogens is intimately linked to human and environmental health. With limited healthcare and challenging living conditions, people living in poverty may be particularly susceptible to endemic and emerging diseases. Similarly, wildlife is impacted by human influences, including pathogen sharing, especially for species in close contact with people and domesticated animals. Here we investigate human and animal contacts and human health in a community living around the Bwindi Impenetrable National Park (BINP), Uganda. We used contact and health survey data to identify opportunities for cross-species pathogen transmission, focusing mostly on people and the endangered mountain gorilla. We conducted a survey with background questions and self-reported diaries to investigate 100 participants’ health, such as symptoms and behaviours, and contact patterns, including direct contacts and sightings over a week. Contacts were revealed through networks, including humans, domestic, peri-domestic, and wild animals for 1) network of contacts seen in the week of background questionnaire completion, 2) network of contacts seen during the diary week. Participants frequently felt unwell during the study, reporting from one to 10 disease symptoms at different intensity levels (maximum of seven symptoms in one day), with severe symptoms comprising 6.4% of the diary records and tiredness and headaches the most common symptoms. Besides human-human contacts, direct contacts with livestock and peri-domestic animals were the most common. Wildlife contacts were the rarest, including one direct contact with gorilla with a concerning timeline of reported symptoms. The contact networks were moderately connected and revealing a preference in contacts within the same species or taxon and within their groups. Despite sightings of wildlife being much more common than touching, one participant declared direct contact with a mountain gorilla during the week. Gorillas were seen very close to six animal taxa (including themselves) considering all interaction types, mostly seen closer to other gorillas, but also people and domestic animals. Our findings reveal a local human population with recurrent symptoms of illness in a location with intense exposure to factors that can increase pathogen transmission, such as direct contact with domestic and wild animals and proximity among animal species. Despite significant biases and study limitations, the information generated here can guide future studies, such as models for disease spread and One Health interventions.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453512', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'In vitro and in vivo analysis of microvesicle-mediated metastasis using a bright, red-shifted bioluminescent reporter protein of extracellular vesicles', 'authors': \"[{'name': 'Ahmed A. Zarea'}, {'name': 'Gloria I. Perez'}, {'name': 'David Broadbent'}, {'name': 'Benedikt Dolgikh'}, {'name': 'Matthew P. Bernard'}, {'name': 'Alicia Withrow'}, {'name': 'Amelia McGill'}, {'name': 'Victoria Toomajian'}, {'name': 'Lukose K. Thampy'}, {'name': 'Jack Harkema'}, {'name': 'Joel R. Walker'}, {'name': 'Thormas A. Kirkland'}, {'name': 'Michael H. Bachmann'}, {'name': 'Jens Schmidt'}, {'name': 'Masamitsu Kanada'}]\", 'abstract': '<p>Cancer cells produce heterogeneous extracellular vesicles (EVs) as mediators of intercellular communication. Our study focused on a novel method to image EV subtypes and their biodistribution <italic>in vivo</italic>. Regardless of injection routes, we established that reporter EVs isolated from murine mammary carcinoma cells expressing PalmReNL, which utilizes bioluminescence resonance energy transfer (BRET), localized to the lungs. This new EV reporter allowed highly sensitive EV tracking <italic>in vitro</italic> and <italic>in vivo</italic> and enabled us to begin studies to understand the commonalities and functional differences of the EV subtypes. We demonstrated the early appearance of metastatic foci in the lungs of mammary tumor-bearing mice following multiple injections of the microvesicle (MV)-enriched fraction derived from mammary carcinoma cells. In addition, the results we present here show that tumor cell-derived MVs act on distant tissues through upregulating LC3 expression within the lung.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.22.453029', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'CD8+ T cell signature in acute SARS-CoV-2 infection identifies memory precursors', 'authors': \"[{'name': 'Sarah Adamo'}, {'name': 'Jan Michler'}, {'name': 'Yves Zurbuchen'}, {'name': 'Carlo Cervia'}, {'name': 'Patrick Taeschler'}, {'name': 'Miro E. Raeber'}, {'name': 'Simona Baghai Sain'}, {'name': 'Jakob Nilsson'}, {'name': 'Andreas E. Moor'}, {'name': 'Onur Boyman'}]\", 'abstract': '<p>Immunological memory is a hallmark of adaptive immunity and facilitates an accelerated and enhanced immune response upon re-infection with the same pathogen<sup>1,2</sup>. Since the outbreak of the ongoing coronavirus disease 19 (COVID-19) pandemic, a key question has focused on whether severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-specific T cells stimulated during acute infection give rise to long-lived memory T cells<sup>3</sup>. Using spectral flow cytometry combined with cellular indexing of transcriptomes and T cell receptor (TCR) sequencing we longitudinally characterize individual SARS-CoV-2-specific CD8<sup>+</sup> T cells of COVID-19 patients from acute infection to one year into recovery and find a distinct signature identifying long-lived memory CD8<sup>+</sup> T cells. SARS-CoV-2-specific memory CD8<sup>+</sup> T cells persisting one year after acute infection re-express CD45RA and interleukin-7 receptor α (CD127), upregulate T cell factor-1 (TCF1), and maintain low CCR7, thus resembling CD45RA<sup>+</sup> effector-memory T (T<sub>EMRA</sub>) cells. Tracking individual clones of SARS-CoV-2-specific CD8<sup>+</sup> T cells, we reveal that an interferon signature marks clones giving rise to long-lived cells, whereas prolonged proliferation and mammalian target of rapamycin (mTOR) signaling are associated with clone contraction and disappearance. Collectively, we identify a transcriptional signature differentiating short-from long-lived memory CD8<sup>+</sup> T cells following an acute virus infection in humans.</p>', 'posted': '2021-07-22'}, {'doi': '10.1101/2021.07.26.453739', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Connectivity between the hippocampus and default mode network during the relief – but not elicitation – of curiosity supports curiosity-enhanced memory enhancements', 'authors': \"[{'name': 'Charlotte Murphy'}, {'name': 'Charan Ranganath'}, {'name': 'Matthias J. Gruber'}]\", 'abstract': '<p>Consistent with the idea that curiosity enhances information seeking, it has been shown that activity within both the dopaminergic circuit and hippocampus supports curiosity-enhanced learning. However, the role of whole-brain mechanisms involved in cognitive control (fronto-parietal network; FPN) and memory integration (default mode network; DMN) that might underpin curiosity states and their effects on memory remain elusive. We hypothesised that the FPN and DMN should distinguish between high- and low-curiosity conditions and be recruited more heavily for later remembered information associated with high-curiosity. Here, we used functional magnetic resonance imaging whilst participants completed a trivia paradigm, in which we presented trivia questions associated with high- and low-curiosity, followed by the associated answer. After a short delay, we tested memory for trivia answers. We adopted a network-based parcellation of the brain into subnetworks of the FPN and DMN to examine how neural activity within, and functional connectivity between, each subnetwork predicts curiosity-enhanced memory. Across elicitation and relief of curiosity, we found focal recruitment of FPNA and widespread recruitment of DMN subnetworks in support of curiosity and curiosity-enhanced memory. Most importantly, during the elicitation of curiosity, functional subcortical connectivity and across cortical networks, but not subcortical-cortical coupling, correlated with curiosity-enhanced memory. However, during the relief of curiosity, coupling between subcortical regions and DMNA emerged in support of curiosity-enhanced memory. Taken together, our results provide the first evidence about how neuromodulatory mechanisms via the hippocampal-dopaminergic circuit trigger states of curiosity and thereby communicate to higher-order cortical regions to facilitate curiosity-enhanced memory.</p><sec><title>Significant statement</title><p>Does neural activity within, and functional connectivity between, the dopaminergic-hippocampal network, fronto-parietal network (FPN), and default mode network (DMN) underpin curiosity states and their effects on memory? Here, we show how the dopaminergic system together with the hippocampus interact specifically with subnetwork DMNA potentially reflecting how subcortical regions support the enhancement of memory intergration of semantic information associated with curiosity. As DMNA (the core DMN subnetwork) was also functionally coupled with the whole DMN network and the semantic control network (FPNA), these findings provide a plausible neuromodulatory mechanism through which hippocampal-dopaminergic input triggers curiosity and then communicates to higher-order brain regions via DMNA to facilitate curiosity-enhanced memory.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453751', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Quantitative Profiling of Adaptation to Cyclin E Overproduction', 'authors': \"[{'name': 'Juanita C. Limas'}, {'name': 'Amiee N. Littlejohn'}, {'name': 'Amy M. House'}, {'name': 'Katarzyna M. Kedziora'}, {'name': 'Dalia Fleifel'}, {'name': 'Brandon L. Mouery'}, {'name': 'Andrea Walens'}, {'name': 'Maria M. Aleman'}, {'name': 'Daniel Dominguez'}, {'name': 'Jeanette Gowen Cook'}]\", 'abstract': '<p>Cyclin E/CDK2 drives cell cycle progression from G1 to S phase. Cyclin E overproduction is toxic to mammalian cells, although the gene encoding cyclin E (<italic>CCNE1</italic>) is overexpressed in some cancers. It is not yet understood how cancer cells tolerate high levels of cyclin E. To address this question, we extensively characterized non-transformed epithelial cells subjected to chronic cyclin E overproduction. Cells overproducing human cyclin E briefly experienced truncated G1 phases, then consistently endured a transient period of DNA replication origin underlicensing, replication stress, and severely impaired proliferation. Individual cells displayed substantial intercellular heterogeneity in both cell cycle dynamics and CDK activity. Each of these phenotypes improved rapidly despite maintaining high cyclin E-associated activity. Transcriptome analysis revealed that adapted cells downregulated a cohort of G1-regulated genes. These cells also shared at least one unique change also found in breast tumors that overproduce cyclin E, expression of the cancer/testis antigen <italic>HORMAD1</italic>. Withdrawing cyclin E induction partially reversed the intermediate licensing phenotype of adapted cells indicating that adaptation is at least partly independent of genetic alterations. This study provides evidence that mammalian cyclin E/CDK inhibits origin licensing by an indirect mechanism through premature S phase onset. It serves as an example of specific oncogene adaptation that can identify key molecular changes during tumorigenesis.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453271', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Understanding associative false memories in aging using multivariate analyses', 'authors': \"[{'name': 'Nancy A. Dennis'}, {'name': 'Amy A. Overman'}, {'name': 'Catherine M. Carpenter'}, {'name': 'Courtney R. Gerver'}]\", 'abstract': '<p>Associative memory declines in aging arise, in part, from age-related increases in false memories to recombined lures. Studies have shown that there is a benefit to associative memory if the configural context of associative pairs is maintained from encoding to retrieval. The current study aimed to examine whether this benefit of contextual congruency is reduced in aging, and whether the neural similarity of memory representations between targets and lures underlies age- related increases in false memories. Behaviorally, both age groups benefited from target pairs presented in a visual format that was congruent with how the pair was learned. While no age difference was observed in hits, the typical age-related increase in false alarms was found. Congruent with behavioral results, neither the relationship between target-related patterns of neural activity across memory phases (as measured by ERS) nor the discriminability of target classification as a function of condition at retrieval (as measured by MVPA). However, with regard to false memories, older adults exhibited overall lower pattern similarity for hits and FAs compared to hits and CRs (as measured by RSA). Additionally, Hit-FA RSA correlated with age- related increases in associative FAs across visual, frontal, and parietal cortices. Results suggest that while neural processes supporting associative memory retrieval are dependent on configural congruency between encoding and retrieval, there is no difference as to how congruency affects these processes in aging. Additionally, similarity of target and lure processing may reflect reduced diagnosticity of information processing in aging.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.453319', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Pectin remodeling belongs to a homeostatic system and triggers transcriptomic and hormonal modulations', 'authors': \"[{'name': 'François Jobert'}, {'name': 'Stéphanie Guénin'}, {'name': 'Aline Voxeur'}, {'name': 'Kieran J. D. Lee'}, {'name': 'Sophie Bouton'}, {'name': 'Fabien Sénéchal'}, {'name': 'Ludivine Hocq'}, {'name': 'Gaëlle Mongelard'}, {'name': 'Hervé Demailly'}, {'name': 'Petra Amakorová'}, {'name': 'Miroslav Strnad'}, {'name': 'Samantha Vernhettes'}, {'name': 'Gregory Mouille'}, {'name': 'Serge Pilard'}, {'name': 'J. Paul Knox'}, {'name': 'Ondřej Novák'}, {'name': 'Jérôme Pelloux'}, {'name': 'Laurent Gutierrez'}]\", 'abstract': '<p>Pectins occur in primary cell walls and consist of multiblock co-polymers among which homogalacturonan (HG) is the simplest and most abundant form. Methylesterification patterns of HG are tuned by pectin methylesterases (PMEs), the activities of which are controlled by specific inhibitors (PMEIs). By impacting cell wall mechanical properties, PME-mediated regulation of HG methylesterification plays a major role in several developmental processes, including seed germination and dark-grown hypocotyl elongation. Arabidopsis <italic>PME36</italic> is preferentially expressed during the late stage of seed development and, using the knock-out mutant <italic>pme36-1</italic>, we show here that PME36 is required to implement the characteristic pattern of de-methylesterified pectin in the mature seed. Surprisingly, while this pattern is strongly impaired in <italic>pme36-1</italic> mature seed, no phenotypical effect is observed in the mutant during seed germination and dark-grown hypocotyl elongation, suggesting the existence of a compensatory mechanism overcoming the defect in pectin de-methylesterification. By analyzing hormone contents and gene expression, a strong, dynamic, and long-lasting physiological disorder is revealed in the mutant. These results suggest the existence of complex connections between pectin remodeling, transcriptomic regulations and hormonal homeostasis, modulating several physiological parameters to ensure the maintenance of a normal seed-to-seedling developmental program in <italic>pme36-1</italic>. Considered for a long time as an end-point passive effector mainly involved in modification of cell wall mechanics, the role of pectin methylesterification needs to be reconsidered as a modulator acting upstream of diverse regulatory pathways involved in plant development.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.24.453649', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Important Docking study of Structure of Plasmodium falciparum thioredoxin reductase-thioredoxin complex: the case of potent drug Telatinib, a small molecule angiogenesis inhibitor', 'authors': \"[{'name': 'Ivan Vito Ferrari'}]\", 'abstract': '<sec><title>Background</title><p>Over the last decades, malaria parasites have been rapidly developing resistance against antimalarial drugs, which underlines the need for novel drug targets. Thioredoxin reductase (TrxR) is crucially involved in redox homeostasis and essential for Plasmodium falciparum. In this communication, we report first time important Docking study by in Silico approach, using AutoDock Vina. After a selective analysis of over 300 drugs, processed with Pyrx (a Virtual Screening software into the active site of protein (ID PDB 4J56 Thioredoxin reductase 2 Chain A), we noticed excellent value of Binding Energy of Telatinib estimated by Pyrx software. These results are comparable to the crystallized ligand FAD (FLAVIN-ADENINE DINUCLEOTIDE) completed in the above-mentioned protein. Indeed, from the results of Autodock Vina, Telatinib an inhibitor of tyrosine kinases, has excellent a Binding affinity value, ca. −12 kcal/mol.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453750', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'EMDA: A Python package for Electron Microscopy Data Analysis', 'authors': \"[{'name': 'Rangana Warshamanage'}, {'name': 'Keitaro Yamashita'}, {'name': 'Garib N. Murshudov'}]\", 'abstract': '<p>An open-source Python library EMDA for cryo-EM map and model manipulation is presented with a specific focus on validation. The architecture of the library and its programmable and command-line interfaces are presented. The use of several functionalities in the library is presented through several examples. The utility of local correlation as a metric for identifying model errors and unmodeled regions in maps, and how it is used as a metric of map-model validation is demonstrated. The mapping of local correlation to individual atoms, and its use to draw insights on local signal variations are discussed. EMDA’s likelihood-based map overlay is demonstrated by carrying out a superposition of two domains in two related structures. The overlay is carried out globally to bring both maps into the same coordinate frame and then locally to estimate the relative movement of domains. Finally, the map magnification refinement in EMDA is presented with an example to highlight the importance of correcting map magnification in structural comparison studies.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.21261065', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'COVID-Q: validation of the first COVID-19 questionnaire based on patient-rated symptom gravity', 'authors': \"[{'name': 'G Spinato'}, {'name': 'C Fabbris'}, {'name': 'F Conte'}, {'name': 'A Menegaldo'}, {'name': 'L Franz'}, {'name': 'P Gaudioso'}, {'name': 'F Cinetto'}, {'name': 'C Agostini'}, {'name': 'G Costantini'}, {'name': 'P Boscolo-Rizzo'}]\", 'abstract': '<sec><title>Objectives</title><p>The aim of the present study was to develop and validate the CoronaVirus Disease – 2019 (COVID-19) Questionnaire (COVID-Q), a novel symptom questionnaire specific for COVID-19 patients, to provide a comprehensive evaluation which may be helpful for physicians.. A secondary goal of the present study was to evaluate the performance of the COVID-Q in identifying subjects at higher risk of being tested positive for COVID-19.</p></sec><sec><title>Material and methods</title><p>Consecutive non-hospitalized adults who underwent nasopharyngeal and throat swab for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) detection at Treviso Hospital in March 2020, were enrolled. Subjects were divided into positive (cases) and negative (controls) in equal number. All of them gave consent and answered the COVID-Q. Patients not able to answer the COVID-Q due to clinical conditions were excluded.</p><p>Parallel Analysis and Principal Component Analysis were used to identify clusters of items measuring the same dimension. The Item Response Theory (IRT)-based analyses evaluated the functioning of item categories, the presence of clusters of local dependence among items, item fit within the model and model fit to the data.</p></sec><sec><title>Results</title><p>Answers obtained from 230 COVID-19 cases (113 males, and 117 females; mean age 55 years, range 20-99 years) and 230 controls (61 males, and 169 females; mean age 46 years, range 21-89) were analyzed. Parallel analysis led to the extraction of six components, which corresponded to as many clinical presentation patterns: asthenia, influenza-like symptoms, ear and nose symptoms, breathing issues, throat symptoms, and anosmia/ageusia. The final IRT models retained 27 items as significant for symptom assessment. The total score on the questionnaire was significantly associated with positivity to the molecular SARS-CoV-2 test: subjects with multiple symptoms were significantly more likely to be affected by COVID-19 (p &lt; .001). Older age and male gender also represented risk factors. Presence of breathing issues and anosmia/ageusia were significantly related to positivity to SARS-CoV-2 (p &lt; 0.001). None of the examined comorbidities had a significant association with COVID-19 diagnosis.</p></sec><sec><title>Conclusion</title><p>According to the analyses, COVID-Q could be validated since the aspects it evaluated were overall significantly related to SARS-CoV-2 infection. The application of the novel COVID-Q to everyday clinical practice may help identifying subjects who are likely to be affected by COVID-19 and address them to a nasopharyngeal swab in order to achieve an early diagnosis.</p></sec><sec><title>What is already known about this topic?</title><p>COVID-19 symptoms are widely known. Lots of studies have been published regarding self-administered questionnaires in order to characterize and know as much as possible regarding this disease. By the way, no specific questionnaires have been validated, yet, and there is no consensus regarding this topic.</p></sec><sec><title>What does this article add?</title><p>This paper shows the COVID-Q, a novel symptom questionnaire specific for COVID-19 patients. The aim is to provide a comprehensive evaluation that may be helpful to clinicians in order to suspect SARS-CoV-2 infection or not.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.453456', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Fast transcriptional activation of developmental signalling pathways during wound healing of the calcareous sponge Sycon ciliatum', 'authors': \"[{'name': 'Cüneyt Caglar'}, {'name': 'Alexander Ereskovsky'}, {'name': 'Mary Laplante'}, {'name': 'Daria Tokina'}, {'name': 'Sven Leininger'}, {'name': 'Ilya Borisenko'}, {'name': 'Genevieve Aisbett'}, {'name': 'Di Pan'}, {'name': 'Marcin Adamski'}, {'name': 'Maja Adamska'}]\", 'abstract': '<sec><title>Background</title><p>The ability to regenerate lost or damaged body parts is an ancient animal characteristic with a wide yet variable distribution across all phyla. Sponges, likely the sister group to all other animals, have remarkable regenerative abilities including whole body regeneration and re-development from dissociated cells. The calcareous sponge <italic>Sycon ciliatum</italic> has been subject to various regeneration studies since the beginning of the last century. However, the early steps of wound healing of <italic>S. ciliatum</italic> have not been addressed from the molecular perspective.</p></sec><sec><title>Results</title><p>In this study, we combined electron microscopy with gene expression analysis to investigate wound healing after transverse sectioning of <italic>S. ciliatum</italic>. Microscopic analysis revealed massive transdifferentiation and collective migration behaviour of choanocytes and pinacocytes early upon injury (6-12h) as the main mechanisms for quick closure of the wound surface. RNA-sequencing identified upregulation of components of the conserved metazoan Wnt and TGFβ signalling pathways within 3h, preceding morphologically detectable wound healing events. De novo upregulation after a decline in expression coincides with morphologically visible polarity establishment. Moreover, by integrating the new wound healing data set with previously published data derived from intact sponge, we demonstrate similarity between gene activity during early wound healing and osculum maintenance. Whole mount <italic>in situ</italic> hybridisation of the TGFβ signalling pathway ligand SciTGFβU and signal transducer SciSmadRa show that the early activation of both is initially encompassing a large area surrounding the cut surface with gradual restriction to the edge of the forming regenerative membrane as wound healing progresses. While SciTGFβU transcripts are localised to exo- and endopinacocytes, SciSmadRa expression appears across all cell types. Using an EdU cell proliferation assay, we found that a global increase in cell proliferation is not visible before 12h into wound healing. Hence, the initial stages to cover the injury site including cell transdifferentiation and migration seem to be executed by cells remaining after injury. Gene expression clustering coupled with GO term enrichment analysis confirmed that expression of genes involved in processes related to cell proliferation, DNA repair as well as apoptotic processes at 3 and 6h of wound healing was not upregulated. On the other hand, genes associated with positive regulation of transcription, signal transduction, actin filament and chromatin organisation, as well as the Wnt signalling pathway are upregulated at early wound healing stages.</p></sec><sec><title>Conclusion</title><p>We have analysed wound healing in the calcareous sponge <italic>Sycon ciliatum</italic> using microscopic and genomic methods. This study highlights a remarkable mechanism of interplay between cell transdifferentiation and collective migration we hypothesise to be regulated by conserved metazoan developmental pathways and numerous taxonomically restricted genes. Expression of these genes in regenerating and intact sponges sheds light on the long-standing question whether embryonic developmental pathways are redeployed in regeneration.</p></sec>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453507', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Toxin breakdown does not preclude the potential for defensive toxin use in a fruit fly', 'authors': \"[{'name': 'Tyler E. Douglas'}, {'name': 'Sofia G. Beskid'}, {'name': 'Callie E. Gernand'}, {'name': 'Brianna E. Nirtaut'}, {'name': 'Kristen E. Tamsil'}, {'name': 'Richard W. Fitch'}, {'name': 'Rebecca D. Tarvin'}]\", 'abstract': '<p>Animals that ingest toxins can themselves become toxic or unpalatable to predators and parasites. Because most animals rapidly eliminate toxins to survive toxin ingestion, it is unclear how species transition from susceptibility and toxin elimination to tolerance and accumulation as chemical defense emerges. Studies of chemical defense have generally focused on species that display active toxin sequestration and target-site insensitivity mutations that permit survival without necessitating toxin metabolism. Here we investigate whether animals that presumably rely on toxin elimination for survival can also utilize ingested toxins for defense. We use the A4 and A3 <italic>Drosophila melanogaster</italic> fly strains from the Drosophila Synthetic Population Resource (DSPR), which respectively possess elevated and reduced metabolic resistance to nicotine. We find that ingesting nicotine increased the survival of A4 but not of A3 flies against <italic>Leptopilina heterotoma</italic> wasp parasitism. Further, we find that despite possessing enhanced toxin clearance mechanisms, A4 flies accrued more nicotine than A3 individuals. Our results suggest that enhanced metabolic detoxification can allow for greater toxin intake by offsetting the cost of toxin ingestion. Passive toxin accumulation that accompanies increased toxin intake may underlie the early origins of chemical defense.</p>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.25.453293', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Age- and sex-related topological organisation of human brain functional networks and their relationship to cognition', 'authors': \"[{'name': 'Heidi Foo'}, {'name': 'Anbupalam Thalamuthu'}, {'name': 'Jiyang Jiang'}, {'name': 'Forrest Koch'}, {'name': 'Karen A. Mather'}, {'name': 'Wei Wen'}, {'name': 'Perminder S. Sachdev'}]\", 'abstract': '<sec><title>BACKGROUND</title><p>Age and sex associated with changes in functional brain network topology and cognition in large population of older adults have been poorly understood. We explored this question further by examining differences in 11 resting-state graph theory measures with respect to age, sex, and their relationships with cognitive performance in 17,127 UK Biobank participants (mean=62.83±7.41 years).</p></sec><sec><title>METHODS</title><p>Brain connectivity toolbox was used to derive the graph theory measures that assessed network integration, segregation, and strength. Multiple linear regression was performed the relationship between age, sex, cognition, and network measures. Subsequently, multivariate analysis was done to further examine the joint effect of the network measures on cognitive functions.</p></sec><sec><title>RESULTS</title><p>Age was associated with an overall decrease in the effectiveness of network communication (i.e. integration) and loss of functional specialisation (i.e. segregation) of specific brain regions. Sex differences were also observed, with women showing more efficient networks which were less segregated than in men (FDR adjusted <italic>p</italic>&lt;.05). Age-related changes were also more apparent in men than women, which suggests that men may be more vulnerable to cognitive decline with age. Interestingly, while network segregation and strength of limbic network were only nominally associated with cognitive performance, the network measures collectively were significantly associated with cognition (FDR adjusted <italic>p</italic>≤.002). This may imply that individual measures may be inadequate to capture much of the variance in neural activity or its output and need further refinement.</p></sec><sec><title>CONCLUSION</title><p>The complexity of the functional brain organisation may be shaped by an individual’s age and sex, which ultimately may influence cognitive performance of older adults. Age and sex stratification may be used to inform clinical neuroscience research to identify older adults at risk of cognitive dysfunction.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453533', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Decoding the Neural Signatures of Valence and Arousal From Portable EEG Headset', 'authors': \"[{'name': 'Nikhil Garg'}, {'name': 'Rohit Garg'}, {'name': 'NS Parrivesh'}, {'name': 'Apoorv Anand'}, {'name': 'V.A.S. Abhinav'}, {'name': 'Veeky Baths'}]\", 'abstract': '<p>This paper focuses on classifying emotions on the valence-arousal plane using various feature extraction, feature selection and machine learning techniques. Emotion classification using EEG data and machine learning techniques has been on the rise in the recent past. We evaluate different feature extraction techniques, feature selection techniques and propose the optimal set of features and electrodes for emotion recognition. The images from the OASIS image dataset were used for eliciting the valence and arousal emotions, and the EEG data was recorded using the Emotiv Epoc X mobile EEG headset. The analysis is additionally carried out on publicly available datasets: DEAP and DREAMER. We propose a novel feature ranking technique and incremental learning approach to analyze the dependence of performance on the number of participants. Leave-one-subject-out cross-validation was carried out to identify subject bias in emotion elicitation patterns. The importance of different electrode locations was calculated, which could be used for designing a headset for emotion recognition. Our study achieved root mean square errors (RMSE) of less than 0.75 on DREAMER, 1.76 on DEAP, and 2.39 on our dataset.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.21261025', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Cardiovascular Medicine', 'covid': True, 'title': 'Clinical echocardiography does not indicate cardiac dysfunction in critically ill Covid-19 patients', 'authors': \"[{'name': 'Henrik Isackson'}, {'name': 'Anders Larsson'}, {'name': 'Miklos Lipcsey'}, {'name': 'Robert Frithiof'}, {'name': 'Frank A. Flachskampf'}, {'name': 'Michael Hultström'}]\", 'abstract': '<sec><title>Aims</title><p>We aimed to investigate the acute effects of severe SARS-CoV-2 on myocardial function.</p></sec><sec><title>Methods and Results</title><p>This is an observational study generated from the first 79 patients admitted to intensive care in Uppsala due to respiratory failure with SARS-CoV-2 infection, during the first wave in 2020, included in the PRONMED study. From this group 34 underwent echocardiographic examination of which 25 were included in the study, and compared to 44 non-echo patients. Demographic analysis compared standard parameters and previous morbidities between the echo and non-echo group. Standard echocardiographic parameters were analysed indicating a reduced left ventricular function as assessed by global longitudinal strain and very discrete increases in wall thickness in the group as a whole. A group comparison between the outcomes survival and death was carried out. Right sided dimensions and functional parameters did not indicate major strain. An increased maximum tricuspid valve regurgitation velocity indicating increased pulmonary arterial pressure was significantly associated with death, but failed to maintain significance when corrected for multiple comparison. Biochemical cardiac markers and D-dimer correlated to initiation of echocardiography and mortality. Tricuspid regurgitation maximum velocity was positively correlated with maximum troponin I.</p></sec><sec><title>Conclusion</title><p>These results suggests that there is no clear negative effect on cardiac function in critical SARS-CoV-2 infection. There are indications that pulmonary pressure elevation carries a negative predictive outcome suggesting pulmonary disease as the major driver of mortality. Cardiac biomarkers as well as D-dimer carry a predictive outcome value.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.21261046', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Respiratory Viral Sequencing Panel identifies SARS-CoV-2 variants, transmission and other co-circulating viruses in Georgia, USA: A Diagnostic and Epidemiologic Tool for Mass Surveillance in COVID-19 Pandemic', 'authors': \"[{'name': 'Nikhil S Sahajpal'}, {'name': 'Ashis K Mondal'}, {'name': 'Allan Njau'}, {'name': 'Zachary Petty'}, {'name': 'Jiani Chen'}, {'name': 'Sudha Ananth'}, {'name': 'Pankaj Ahluwalia'}, {'name': 'Colin Williams'}, {'name': 'Ted M Ross'}, {'name': 'Alka Chaubey'}, {'name': 'Grace DeSantis'}, {'name': 'Gary P. Schroth'}, {'name': 'Justin Bahl'}, {'name': 'Ravindra Kolhe'}]\", 'abstract': '<sec><title>Background</title><p>In the current phase of COVID-19 pandemic, we are facing two serious public health challenges that include deficits in SARS-CoV-2 variant monitoring, and neglect of other co-circulating respiratory viruses. Additionally, accurate assessment of the evolution, extent and dynamics of the outbreak are required to understand the transmission of the virus amongst seemingly unrelated cases and provide critical epidemiological information. To address these challenges, we evaluated a new high-throughput next-generation sequencing (NGS), respiratory viral panel (RVP) that includes 40 viral pathogens with the aim of analyzing viral subtypes, mutational variants of SARS-CoV-2, model to understand the spread of the virus in the state of Georgia, USA, and to assess other circulating viruses in the same population.</p></sec><sec><title>Methods</title><p>This study evaluated a total of 522 samples that included 483 patient samples and 42 synthetic positive control material. The performance metrics were calculated for both clinical and reference control samples by comparing detection results with the RT-PCR assay. The limit of detection (LoD) studies were conducted as per the FDA guidelines. Inference and visualization of the phylogeny of the SARS-CoV-2 sequences were performed through the Nextstrain Command-Line Interface (CLI) tool, utilizing the associated augur and auspice toolkits.</p></sec><sec><title>Results</title><p>The performance metrics calculated using both the clinical samples and the reference controls revealed a PPA, NPA and accuracy of 95.98%, 85.96% and 94.4%, respectively. The LoD was determined to be 10 copies/ml with all 25 replicates detected across two different runs. The clade for pangolin lineage B that contains certain distant variants, including P4715L in ORF1ab, Q57H in ORF 3a and, S84L in ORF8 covarying with the D614G spike protein mutation were the most prevalent, early in the pandemic, in Georgia, USA. In our analysis, isolates from the same county formed paraphyletic groups, which indicated virus transmission between counties.</p></sec><sec><title>Conclusion</title><p>The study demonstrates the clinical and public health utility of the NGS-RVP to identify novel variants that can provide actionable information to prevent or mitigate emerging viral threats, models that provide insights into viral transmission patterns and predict transmission/ resurgence of regional outbreaks and provide critical information on co-circulating respiratory viruses that might be independent factors contributing to the global disease burden.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453650', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'A network-based approach reveals the dysregulated transcriptional regulation in non-alcohol fatty liver disease', 'authors': \"[{'name': 'Hong Yang'}, {'name': 'Muhammad Arif'}, {'name': 'Meng Yuan'}, {'name': 'Xiangyu Li'}, {'name': 'Ko Eun Shong'}, {'name': 'Hasan Turkez'}, {'name': 'Jens Nielsen'}, {'name': 'Mathias Uhlén'}, {'name': 'Jan Borén'}, {'name': 'Zhang Cheng'}, {'name': 'Adil Mardinoglu'}]\", 'abstract': '<p>Non-alcohol-related fatty liver disease (NAFLD) is a leading cause of chronic liver disease worldwide. We performed network analysis to investigate the dysregulated biological processes in the disease progression and revealed the molecular mechanism underlying NAFLD. Based on network analysis, we identified a highly conserved disease-associated gene module across three different NAFLD cohorts and highlighted the predominant role of key transcriptional regulators associated with lipid and cholesterol metabolism. In addition, we revealed the detailed metabolic differences between heterogenous NAFLD patients through integrative systems analysis of transcriptomic data and liver-specific genome-scale metabolic model. Furthermore, we identified transcription factors (TFs), including SREBF2, HNF4A, SREBF1, YY1 and KLF13, showing regulation of hepatic expression of genes in the NAFLD-associated modules and validated the TFs using data generated from a mouse NAFLD model. In conclusion, our integrative analysis facilitated our understanding of the regulatory mechanism of these perturbed TFs and associated biological processes.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.21260947', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Respiratory Medicine', 'covid': False, 'title': 'Prospective validation of host transcriptomic biomarkers for pulmonary tuberculosis by real-time PCR', 'authors': \"[{'name': 'Simon C Mendelsohn'}, {'name': 'Stanley Kimbung Mbandi'}, {'name': 'Andrew Fiore-Gartland'}, {'name': 'Adam Penn-Nicholson'}, {'name': 'Munyaradzi Musvosvi'}, {'name': 'Humphrey Mulenga'}, {'name': 'Michelle Fisher'}, {'name': 'Katie Hadley'}, {'name': 'Mzwandile Erasmus'}, {'name': 'Onke Nombida'}, {'name': 'Michele Tameris'}, {'name': 'Gerhard Walzl'}, {'name': 'Kogielium Naidoo'}, {'name': 'Gavin Chruchyard'}, {'name': 'Mark Hatherill'}, {'name': 'Thomas Scriba'}]\", 'abstract': '<p>We tested performance of host-blood transcriptomic tuberculosis (TB) signatures for active case-finding. Among 20,207 HIV-uninfected and 963 HIV-infected adults screened; 2,923 and 861 were enrolled from five South African communities. Eight signatures were measured by microfluidic RT-qPCR and participants were microbiologically-investigated for pulmonary TB at baseline, and actively surveilled for incident disease through 15 months. Diagnostic AUCs for 61 HIV-uninfected (weighted-prevalence 1.1%) and 10 HIV-infected (prevalence 1.2%) prevalent TB cases for the 8 signatures were 0.63-0.79 and 0.65-0.88, respectively. Thereafter, 24 HIV-uninfected and 9 HIV-infected participants progressed to incident TB (1.1 and 1.0 per 100 person-years, respectively). Prognostic AUCs through 15-months follow-up were 0.49-0.66 and 0.54-0.81, respectively. Prognostic performance for incident TB occurring within 6-12 months in HIV-negative participants was higher for all signatures. None of the signatures met WHO Target Product Profile criteria for a triage test to diagnose asymptomatic TB; most signatures met the criteria for symptomatic TB. Prognostic accuracy of most signatures for incident TB within six months of testing met the criteria for an incipient TB test.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.26.453736', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Towards an integrative multi-omics workflow', 'authors': \"[{'name': 'Florian Jeanneret'}, {'name': 'Stéphane Gazut'}]\", 'abstract': '<p>The advent of high-throughput techniques has greatly enhanced biological discovery. Last years, analysis of multi-omics data has taken the front seat to improve physiological understanding. Handling functional enrichment results from various biological data raises practical questions.</p><p>We propose an integrative workflow to better interpret biological process insights in a multi-omics approach applied to breast cancer data from The Cancer Genome Atlas (TCGA) related to Invasive Ductal Carcinoma (IDC) and Invasive Lobular Carcinoma (ILC). Pathway enrichment by Over Representation Analysis (ORA) and Gene Set Enrichment Analysis (GSEA) has been conducted with both features information from differential expression analysis or selected features from multi-block sPLS-DA methods. Then, comprehensive comparisons of enrichment results have been carried out by looking at classical enrichment analysis, probabilities pooling by Stouffer’s Z scores method and pathways clustering in biological themes.</p><p>Our work shows that ORA enrichment with selected sPLS-DA features and pathways probabilities pooling by Stouffer’s method lead to enrichment maps highly associated to physiological knowledge of IDC or ILC phenotypes, better than ORA and GSEA with differential expression driven features.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453727', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Miniature inverted-repeat transposable elements drive rapid microRNA diversification in angiosperms', 'authors': \"[{'name': 'Zhonglong Guo'}, {'name': 'Zheng Kuang'}, {'name': 'Yihan Tao'}, {'name': 'Haotian Wang'}, {'name': 'Miaomiao Wan'}, {'name': 'Chen Hao'}, {'name': 'Fei Shen'}, {'name': 'Xiaozeng Yang'}, {'name': 'Lei Li'}]\", 'abstract': '<p>MicroRNAs (miRNAs) are rapidly evolving endogenous small RNAs programing organism function and behavior. Although models for miRNA origination have been proposed based on sporadic cases, the genomic mechanisms driving swift diversification of the miRNA repertoires in plants remain elusive. Here, by comprehensively analyzing 20 phylogenetically representative plant species, we identified miniature inverted-repeat transposable elements (MITEs) as the predominant genomic sources for de novo miRNAs in angiosperms. Our data illustrated a transposition-transcription process whereby properly sized MITEs transposed into active genic regions could be converted into new miRNAs, termed MITE-miRNAs, in as few as 20 generations. We showed that this molecular domestication mechanism leads to a possible evolutionary arms race between the MITEs and the host genomes that rapidly and continuously changes the miRNA repertoires. We found that the MITE-miRNAs are selected for targeting genes associated with plant adaptation and habitat expansion, thereby constituting a genomic innovation potentially underlying angiosperm megadiversity.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261038', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Epidemics of chikungunya, Zika, and COVID-19 reveal bias in case-based mapping', 'authors': \"[{'name': 'Fausto Andres Bustos Carrillo'}, {'name': 'Brenda Lopez Mercado'}, {'name': 'Jairo Carey Monterrey'}, {'name': 'Damaris Collado'}, {'name': 'Saira Saborio'}, {'name': 'Tatiana Miranda'}, {'name': 'Carlos Barilla'}, {'name': 'Sergio Ojeda'}, {'name': 'Nery Sanchez'}, {'name': 'Miguel Plazaola'}, {'name': 'Harold Suazo Laguna'}, {'name': 'Douglas Elizondo'}, {'name': 'Sonia Arguello'}, {'name': 'Anna M. Gajewski'}, {'name': 'Hannah E. Maier'}, {'name': 'Krista Latta'}, {'name': 'Bradley Carlson'}, {'name': 'Josefina Coloma'}, {'name': 'Leah Katzelnick'}, {'name': 'Hugh Sturrock'}, {'name': 'Angel Balmaseda'}, {'name': 'Guillermina Kuan'}, {'name': 'Aubree Gordon'}, {'name': 'Eva Harris'}]\", 'abstract': '<p>Explosive epidemics of chikungunya, Zika, and COVID-19 have recently occurred worldwide, all of which featured large proportions of subclinical infections. Spatial studies of infectious disease epidemics typically use symptomatic infections (cases) to estimate incidence rates (cases/total population), often misinterpreting them as infection risks (infections/total population) or disease risks (cases/infected population). We examined these three measures in a pediatric cohort (N≈3,000) over two chikungunya epidemics and one Zika epidemic and in a household cohort (N=1,793) over one COVID-19 epidemic in Nicaragua. Across different analyses and all epidemics, case incidence rates considerably underestimated both risk-based measures. Spatial infection risk differed from spatial disease risk, and typical case-only approaches precluded a full understanding of the spatial seroprevalence patterns. For epidemics of pathogens that cause many subclinical infections, relying on case-only datasets and misinterpreting incidence rates, as is common, results in substantial bias, a general finding applicable to many pathogens of high human concern.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453525', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Zoology', 'covid': False, 'title': 'Insights on the historical biogeography of Philippine native pigs and its relationship with Continental domestic and wild boars', 'authors': \"[{'name': 'John King N. Layos'}, {'name': 'Ronel N. Geromo'}, {'name': 'Dinah M. Espina'}, {'name': 'Masahide Nishibori'}]\", 'abstract': '<p>The Philippine archipelago was believed to have never been connected to the Asian continent even during the severe Quaternary sea-level drops. As a result, the history of pig dispersal in the Philippines remains controversial and must have some anthropogenic origin associated with some human migration events. In this study, the context of origin, dispersal, and the level of genetic introgression in Philippine native pigs were deduced using mitochondrial DNA D-loop analysis altogether with domestic pigs and wild boars corresponding to their geographic origin. Results revealed a considerable genetic diversity (0.900±0.016), and a widespread Asian pig-ancestry (94.60%) were revealed in the phylogenetic analysis with admixed European pig-origin (5.10%) harboring various fractions of ancestry from Berkshire and Landrace. The close genetic connection between the continental wild boars and domestic pigs present in the Philippine pigs corroborates our hypothesis of a genetic signal that could potentially be associated with the recently reported multiple waves of human migrations to the Philippines during the last 50,000 years. The high frequency of haplotypes (54.08%) that collapsed in the D7 haplogroup represent an interesting challenge as its distribution does not coincide with the hypothesized migratory route of the Neolithic Austronesian-speaking populations. We detected the first Pacific Clade signature and ubiquitously distributed D2 haplotypes which postulate the legitimate dispersal of pigs associated with the multiple waves of human migrations involving the Philippines. The multimodal mismatch and neutrality test statistics both Fu’s F<italic>s</italic> and Tajima’s D correlates the long stationary period of effective population size revealed in the Bayesian skyline plot. While the sudden decrease in population was consistent with the pronounced population bottleneck of Asian and European pigs during the interglacial periods of the Pleistocene.</p>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.28.454213', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Mitochondria dysfunction in Charcot Marie Tooth 2B Peripheral Sensory Neuropathy', 'authors': \"[{'name': 'Yingli Gu'}, {'name': 'Flora Guerra'}, {'name': 'Mingzheng Hu'}, {'name': 'Alexander Pope'}, {'name': 'Kijung Sung'}, {'name': 'Wanlin Yang'}, {'name': 'Simone Jetha'}, {'name': 'Thomas A. Shoff'}, {'name': 'Tessanya Gunatilake'}, {'name': 'Owen Dahlkamp'}, {'name': 'Linda Zhixia Shi'}, {'name': 'Fiore Manganelli'}, {'name': 'Maria Nolano'}, {'name': 'Yue Zhou'}, {'name': 'Jianqing Ding'}, {'name': 'Cecilia Bucci'}, {'name': 'Chengbiao Wu'}]\", 'abstract': '<p>Recent evidence has uncovered an important role of Rab7 in regulating mitochondrial morphology and function. Missense mutation(s) of Rab7 underlies the pathogenesis of Charcot Marie Tooth 2B (CMT2B) peripheral neuropathy. Herein, we investigated how mitochondrial morphology and function were impacted by the CMT2B associated Rab7V162M mutation in fibroblasts from human CMT2B patients as well as in a knockin mouse model. In contrast to recently published results from studies of using heterologous overexpression systems, our results have demonstrated significant mitochondrial fragmentation in fibroblasts of both human CMT2B patients and CMT2B mouse embryonic fibroblasts (MEFs). Furthermore, we have shown that mitochondria were fragmented and axonal mitochondrial movement was dysregulated in primary cultured E18 dorsal root ganglion (DRG) sensory neurons, but not in E18 hippocampal and cortical primary neurons. We also show that inhibitors to either the mitochondrial fission protein Drp1 or to the nucleotide binding to Rab7 normalized the mitochondrial deficits in both MEFs and E18 cultured DRG neurons. Our study has revealed, for the first time, that expression of CMT2B Rab7 mutation at physiological level enhances Drp1 activity to promote mitochondrial fission, that may potentially underlie selective vulnerability of peripheral sensory neurons in CMT2B pathogenesis.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.25.453710', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A comprehensive collection of transcriptome data to support life cycle analysis of the poplar rust fungus Melampsora larici-populina', 'authors': \"[{'name': 'Pamela Guerillot'}, {'name': 'Emmanuelle Morin'}, {'name': 'Annegret Kohler'}, {'name': 'Clémentine Louet'}, {'name': 'Sébastien Duplessis'}]\", 'abstract': '<p><italic>Melampsora larici-populina</italic> is an obligate biotrophic plant pathogen responsible for the poplar rust disease. This fungus belongs to the taxonomical order Pucciniales and exhibits a complex heteroecious and macrocyclic life cycle, i.e. it has the capacity to infect two unrelated host plants, larch and poplar, and to form five distinct spore types through the year. The <italic>M. larici-populina</italic> genome has been sequenced and annotated in 2011 and since, different transcriptomic analyses were conducted at different stages with oligoarrays and later on with RNA-Seq covering most of its life cycle. Here, we collected published transcriptome data for the poplar rust fungus, transposed them for the version 2 of the genome now available at the Joint Genome Institute Mycocosm website and performed normalization of different oligoarray datasets on one hand, and of RNA-Seq on the other hand. We report a comprehensive normalized dataset for this fungus, made available for the community, which allows life cycle transcriptomics analysis.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453290', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PCRMS: a database of predicted cis-regulatory modules and constituent transcription factor binding sites in genomes', 'authors': \"[{'name': 'Pengyu Ni'}, {'name': 'Zhengchang Su'}]\", 'abstract': '<p>More accurate and more complete predictions of <italic>cis</italic>-regulatory modules (CRMs) and constituent transcriptional factor (TF) binding sites (TFBSs) in genomes can facilitate characterizing functions of regulatory sequences. Here, we developed a database PCRMS (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://cci-bioinfo.uncc.edu\">https://cci-bioinfo.uncc.edu</ext-link>) that stores highly accurate and unprecedentedly complete maps of predicted CRMs and TFBSs in the human and mouse genomes. The web interface allows the user to browse CRMs and TFBSs in an organism, find the closest CRMs to a gene, search CRMs around a gene, and find all TFBSs of a TF. PCRMS can be a useful resource for the research community to characterize regulatory genomes.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453632', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'PTEN mutant NSCLC require ATM to suppress pro-apoptotic signalling and evade radiotherapy', 'authors': \"[{'name': 'Thomas Fischer'}, {'name': 'Oliver Hartmann'}, {'name': 'Michaela Reissland'}, {'name': 'Cristian Prieto-Garcia'}, {'name': 'Kevin Klann'}, {'name': 'Christina Schülein-Völk'}, {'name': 'Bülent Polat'}, {'name': 'Elena Gerhard-Hartmann'}, {'name': 'Mathias Rosenfeldt'}, {'name': 'Christian Münch'}, {'name': 'Michael Flentje'}, {'name': 'Markus E. Diefenbacher'}]\", 'abstract': '<sec><title>Background</title><p>Despite advances in treatment of patients with non-small cell lung cancer, carriers of certain genetic alterations are prone to failure. One such factor frequently mutated, is the tumor suppressor PTEN. These tumors are supposed to be more resistant to radiation, chemo- and immunotherapy.</p></sec><sec><title>Methods</title><p>Using CRISPR genome editing, we deleted PTEN in a human tracheal stem cell-like cell line as well generated primary murine NSCLC, proficient or deficient for <italic>Pten</italic>, <italic>in vivo</italic>. These models were used to verify the impact of PTEN loss <italic>in vitro</italic> and <italic>in vivo</italic> by immunohistochemical staining, western blot and RNA-Sequencing. Radiation sensitivity was assessed by colony formation and growth assays. To elucidate putative treatment options, identified via the molecular characterisation, PTEN pro- and deficient cells were treated with PI3K/mTOR/DNA-PK-inhibitor PI-103 or the ATM-inhibitors KU-60019 und AZD 1390. Changes in radiation sensitivity were assessed by colony-formation assay, FACS, western-blot, phospho-proteomic mass spectrometry and <italic>ex vivo</italic> lung slice cultures.</p></sec><sec><title>Results</title><p>We demonstrate that loss of PTEN led to altered expression of transcriptional programs which directly regulate therapy resistance, resulting in establishment of radiation resistance. While PTEN-deficient tumor cells were not dependent on DNA-PK for IR resistance nor activated ATR during IR, they showed a significant dependence for the DNA damage kinase ATM. Pharmacologic inhibition of ATM, via KU-60019 and AZD1390 at non-toxic doses, restored and even synergized with IR in PTEN-deficient human and murine NSCLC cells as well in a multicellular organotypic <italic>ex vivo</italic> tumor model.</p></sec><sec><title>Conclusion</title><p>PTEN tumors are addicted to ATM to detect and repair radiation induced DNA damage. This creates an exploitable bottleneck. At least <italic>in cellulo</italic> and <italic>ex vivo</italic> we show that low concentration of ATM inhibitor is able to synergise with IR to treat PTEN-deficient tumors in genetically well-defined IR resistant lung cancer models.</p></sec>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.27.454001', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Endorsement and Phylogenetic Analysis of some Fabaceae Plants based on DNA Barcoding gene MatK', 'authors': \"[{'name': 'Nader R. Abdelsalam'}, {'name': 'Mohamed E. Hasan'}, {'name': 'Samar M.A. Rabie'}, {'name': 'Houssam El-Din M.F. El-wakeel'}, {'name': 'Amera F. Zaitoun'}, {'name': 'Rehab Y. Ghareeb'}, {'name': 'Aly Z. Abdelsalam'}, {'name': 'Hesham M. Aly'}, {'name': 'Amira A. Ibrahim'}, {'name': 'Alaa A. Hemeida'}]\", 'abstract': '<p>DNA barcodes have been considered as a tool to facilitate species identification based on their simplicity and high-level accuracy compression to the complexity and subjective biases linked to morphological identification of taxa. MaturaseK gene “  MatK”  of the chloroplast is very crucial in the plant system which is involved in the group II intron splicing. The main objective of this current study is determining the relative utility of the “  MatK”  chloroplast gene for barcoding in fifteen legume trees by both single region and multiregional approaches. The chloroplast “  MatK”  gene sequences were submitted to GenBank and accession numbers (GenBank: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC602060\">LC602060</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC602154\">LC602154</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC602263\">LC602263</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603347\">LC603347</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603655\">LC603655</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603845\">LC603845</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603846\">LC603846</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603847\">LC603847</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC604717\">LC604717</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC604718\">LC604718</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC605994\">LC605994</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC604799\">LC604799</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC605995\">LC605995</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC606468\">LC606468</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC606469\">LC606469</ext-link>) were obtained with sequence length ranging from 730 to 1545 nucleotides. These DNA sequences were aligned with database sequence using PROMALS server  ,  Clustal Omega server and Bioedit program. Also,\\xa0 the maximum likelihood and neighbor-joining algorithms for phylogenetic reconstruction using the MEGA-X program were employed. Overall, these results indicated that  the phylogenetic tree  analysis and the evolutionary distances of an individual dataset of each species were agreed with a phylogenetic tree of all each other consisting of two clades, the first clade comprising  (Enterolobium contortisiliquum, Albizia lebbek), Acacia saligna  ,  Leucaena leucocephala,  Dichrostachys Cinerea, (Delonix regia, Parkinsonia aculeata), (Senna surattensis, Cassia fistula, Cassia javanica)  and  Schotia brachypetala  were more closely to each other, respectively. The remaining four species of  Erythrina humeana, (Sophora secundiflora, Dalbergia Sissoo, Tipuana Tipu)  constituted the second clade. Therefore,  MatK  gene is considered promising a candidate for DNA barcoding in plant family Fabaceae and providing a clear relationship between the families. Moreover, their sequences could be successfully utilized in single nucleotide polymorphism (SNP) or part of the sequence as DNA fragment analysis utilizing polymerase chain reaction (PCR) in plant systematic.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.23.453582', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Taxonomic repositioning of twelve Xanthomonas campestris, seven Xanthomonas axonopodis and one Pseudomonas cissicola reference pathovars to Xanthomonas citri', 'authors': \"[{'name': 'Kanika Bansal'}, {'name': 'Sanjeet Kumar'}, {'name': 'Prabhu B. Patil'}]\", 'abstract': '<p>Here on the basis of phylo-taxonogenomics criteria we present amended description of 20 pathovars to <italic>Xanthomonas citri</italic> majority (18/20) of which were first reported from India. 7/20 are currently classified as <italic>X. axonopodis</italic>, 12/20 as <italic>X. campestris</italic> and 1/20 as <italic>Pseudomonas cissicola</italic>. Here, we have generated genome sequence data for 4 pathovars and genomes of remaining 16 were used from the published data. Genome based investigation reveals that all these pathovars belong to <italic>X. citri</italic> and not to <italic>X. axonopodis</italic> or <italic>X. campestris</italic> as previously reported. Present proposal is to aid in resolving the taxonomic confusion of the <italic>X. ctiri</italic> pathovars and prevent future usage of invalid names.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453552', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Emotional states as distinct configurations of functional brain networks', 'authors': \"[{'name': 'Rotem Dan'}, {'name': 'Marta Weinstock'}, {'name': 'Gadi Goelman'}]\", 'abstract': '<p>The conceptualization of emotional states as patterns of interactions between large-scale brain networks has recently gained support. Yet, few studies have directly examined the brain’s network structure during emotional experiences. Here, we investigated the brain’s functional network organization during experiences of sadness, amusement, and neutral states elicited by movies, in addition to a resting-state. We tested the effects of the experienced emotion on individual variability in the brain’s functional connectome. Next, for each state, we defined a modular organization of the brain and quantified its segregation and integration. Our results show that emotional states increase the similarity between and within individuals in the brain’s functional connectome. Second, in the brain’s modular organization, sadness, relative to amusement, was associated with higher integration and increased connectivity of cognitive control networks: the salience and fronto-parietal networks. Modular metrics of brain segregation and integration were further associated with reported emotional valence. Last, in both the functional connectome and emotional report, a higher similarity was found among women. Our results suggest that the experience of emotion is linked to a reconfiguration of whole-brain distributed, not emotion-specific, functional brain networks and that the topological structure carries information about the subjective emotional experience.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.27.453123', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Electrophysiological measures of visual suppression', 'authors': \"[{'name': 'Daniel H Baker'}, {'name': 'Greta Vilidaite'}, {'name': 'Alex R Wade'}]\", 'abstract': '<p>In the early visual system, suppression occurs between neurons representing different stimulus properties. This includes features such as orientation (cross-orientation suppression), eye-of-origin (interocular suppression) and spatial location (surround suppression), which are thought to involve distinct anatomical pathways. We asked if these separate routes to suppression can be differentiated by their pattern of gain control on the contrast response function measured in human participants using steady-state electroencephalography. Changes in contrast gain shift the contrast response function laterally, whereas changes in response gain scale the function vertically. We used a Bayesian hierarchical model to summarise the evidence for each type of gain control. A computational meta-analysis of 16 previous studies found the most evidence for contrast gain effects with overlaid masks, but no clear evidence favouring either response gain or contrast gain for other mask types. We then conducted two new experiments, comparing suppression from four mask types (monocular and dichoptic overlay masks, and aligned and orthogonal surround masks) on responses to sine wave grating patches flickering at 5Hz. At the occipital pole, there was strong evidence for contrast gain effects in all four mask types at the first harmonic frequency (5Hz). Suppression generally became stronger at more lateral electrode sites, but there was little evidence of response gain effects. At the second harmonic frequency (10Hz) suppression was stronger overall, and involved both contrast and response gain effects. Although suppression from different mask types involves distinct anatomical pathways, gain control processes appear to serve a common purpose, which we suggest might be to suppress less reliable inputs.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.23.453227', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Disentangling key species interactions in diverse and heterogeneous communities: A Bayesian sparse modeling approach', 'authors': \"[{'name': 'Christopher P. Weiss-Lehman'}, {'name': 'Chhaya M. Werner'}, {'name': 'Catherine H. Bowler'}, {'name': 'Lauren M. Hallett'}, {'name': 'Margaret M. Mayfield'}, {'name': 'Oscar Godoy'}, {'name': 'Lina Aoyama'}, {'name': 'György Barabás'}, {'name': 'Chengjin Chu'}, {'name': 'Emma Ladouceur'}, {'name': 'Loralee Larios'}, {'name': 'Lauren G. Shoemaker'}]\", 'abstract': '<label>1</label><title>Abstract</title><p>Modeling species interactions in diverse communities traditionally requires a prohibitively large number of species-interaction coefficients, especially when considering environmental dependence of parameters. We implemented Bayesian variable selection via sparsity-inducing priors on non-linear species abundance models to determine which species-interactions should be retained and which can be represented as an average heterospecific interaction term, reducing the number of model parameters. We evaluated model performance using simulated communities, computing out-of-sample predictive accuracy and parameter recovery across different input sample sizes. We applied our method to a diverse empirical community, allowing us to disentangle the direct role of environmental gradients on species’ intrinsic growth rates from indirect effects via competitive interactions. We also identified a few neighboring species from the diverse community that had non-generic interactions with our focal species. This sparse modeling approach facilitates exploration of species-interactions in diverse communities while maintaining a manageable number of parameters.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453515', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Low dose/dose rate γ irradiation protects Drosophila melanogaster chromosomes from double strand breaks and telomere fusions by modulating the expression of Loquacious', 'authors': \"[{'name': 'A. Porrazzo'}, {'name': 'F. Cipressa'}, {'name': 'A. De Gregorio'}, {'name': 'C De Pittà'}, {'name': 'G. Sales'}, {'name': 'P. Morciano'}, {'name': 'G Esposito'}, {'name': 'M.A Tabocchini'}, {'name': 'G. Cenci'}]\", 'abstract': '<p>It is still continuously debated whether the low-dose/dose-rate (LDDR) of ionizing radiation represents a hazard for humans. Model organisms, such as fruit flies, are considered valuable systems to reveal insights into this issue. We found that, in wild-type <italic>Drosophila melanogaster</italic> larval neuroblasts, the frequency of Chromosome Breaks (CBs), induced by acute γ-radiation, is significantly reduced when flies are previously exposed to a protracted dose of 0.4 Gy delivered at a dose rate of 2.5 mGy/h. This indicates that this exposure, which is associated with an increased expression of DNA damage response proteins, induces a radioadaptive response (RAR) that protects Drosophila from extensive DNA damage. Interestingly, the same exposure reduces the frequency of telomere fusions (TFs) from Drosophila telomere capping mutants suggesting that the LDDR can generally promote a protective response on chromatin sites that are recognized as DNA breaks. Deep RNA sequencing revealed that RAR is associated with a modulation of genes involved in RNA processing and the reduction of expression of <italic>Loquacious D (Loqs-RD)</italic> gene that encodes a well-conserved dsRNA binding protein required for esiRNAs biogenesis. Remarkably, loss of <italic>Loqs</italic> mimics the LDDR-mediated chromosome protection as it decreases the IR-induced CBs and TF frequency. Thus, our work identifies Loqs as a key factor in mediating the cellular response to LDDR and reveals that the study of the molecular bases underlying RAR is instrumental for unraveling novel and conserved epigenetic routes that have evolved to preserve genome integrity following robust insults to DNA.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.25.453693', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Hoslundia opposita and other Nigerian plants inhibit sickle hemoglobin polymerization and prevent erythrocyte sickling: a virtual screening-guided identification of bioactive plants.', 'authors': \"[{'name': 'Eric O. Olori'}, {'name': 'Olujide Oludayo Olubiyi'}, {'name': 'Chinedum Peace Babalola'}]\", 'abstract': '<p>In sickle cell disease, a hereditary hemoglobinopathy, clinically observed disease presentations are the endpoint of a point mutation involving the substitution of glutamic acid with valine at the position 6 of the beta globin chain. With about 4.4 million people globally being affected, and another 43 million people bearing the sickle cell trait, several research efforts have been made to discover new and affordable treatment and possibly cure for the disease. Africa is endowed with a large flora population and traditional healers and citizens have over time depended on the use of herbs in folkloric medicine for different ailments including sickle cell disease (SCD). Such native knowledge has often formed the basis for different research exploration into the anti-sickling activities of selected African plants. These plants that have been so far investigated for their anti-sickling properties represent about 0.05 % of the 45,000 plant species enriching the flora landscape in Sub-Saharan Africa. Some of these have yielded potent anti-sickling profiles. In the current work we seek to achieve a more extensive search of the African plant diversity with anti-sickling properties: for this we have adopted a hybrid computational-cum-experimental protocol that employed computer-aided drug design (CADD) means for identifying plants with at least one constituent capable of interacting with the sickle hemoglobin, followed by extractive procedures and anti-sickling experiments for validating the predictions. Over two thousand (or 2,000) African natural products, representing over 200 plant species, were first virtually screened against the crystal structure of the dimerized human sickle hemoglobin. The natural products with the best computed sickle hemoglobin interaction energetics were found to belong to five plant species including Catharanthus roseus, Rauvolfia vomitoria, Hoslundia opposita, Lantana camara and Euphorbia hirta. The leaves of these plants were each collected subsequently and subjected to standard processing and extraction procedures. Using both HbSS polymerization inhibition and sickling reversal tests significant reductions in polymerization of erythrocyte hemolysate of the HbSS genotype were observed with the methanolic extracts of the plants, as well as sickling reversal levels of up to 68.50 % (H. opposita) was observed.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453762', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Effect of stimulation time on the expression of human macrophage polarization markers', 'authors': \"[{'name': 'Duygu Unuvar Purcu'}, {'name': 'Asli Korkmaz'}, {'name': 'Sinem Gunalp'}, {'name': 'Derya Goksu Helvaci'}, {'name': 'Yonca Erdal'}, {'name': 'Yavuz Dogan'}, {'name': 'Gerhard Wingender'}, {'name': 'Duygu Sag'}]\", 'abstract': '<p>Macrophages are highly plastic cells that can polarize into functionally distinct subsets <italic>in vivo</italic> and <italic>in vitro</italic> in response to environmental signals. The development of protocols to model macrophage polarization <italic>in vitro</italic> greatly contributes to our understanding of macrophage biology. Macrophages are divided into two main groups: Pro-inflammatory M1 macrophages (classically activated) and anti-inflammatory M2 macrophages (alternatively activated), based on several key surface markers and the production of inflammatory mediators. However, the expression of these common macrophage polarization markers is greatly affected by the stimulation time used. Unfortunately, there is no consensus yet regarding the optimal stimulation times for particular macrophage polarization markers in <italic>in vitro</italic> experiments. This situation is problematic, (i) as analysing a particular marker at a suboptimal time point can lead to false-negative results, and (ii) as it clearly impedes the comparison of different studies. Using human monocyte-derived macrophages (MDMs) <italic>in vitro</italic>, we analysed how the expression of the main polarization markers for M1 (CD64, CD86, CXCL9, CXCL10, HLA-DR, IDO1, IL1β, IL12p70, TNF), M2a (CD200R, CD206, CCL17, CCL22, IL-10, TGM2), and M2c (CD163, IL-10, TGFβ) macrophages changes over time at mRNA and protein levels. Our data establish the most appropriate stimulation time for the analysis of the expression of human macrophage polarization markers <italic>in vitro</italic>. Providing such a reference guide will likely facilitate the investigation of macrophage polarization and its reproducibility.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.452189', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'The first crested duck genome reveals clues to genetic compensation and crest cushion formation', 'authors': \"[{'name': 'Guobin Chang'}, {'name': 'Xiaoya Yuan'}, {'name': 'Qixin Guo'}, {'name': 'Hao Bai'}, {'name': 'Xiaofang Cao'}, {'name': 'Meng Liu'}, {'name': 'Zhixiu Wang'}, {'name': 'Bichun Li'}, {'name': 'Shasha Wang'}, {'name': 'Yong Jiang'}, {'name': 'Zhiquan Wang'}, {'name': 'Yang Zhang'}, {'name': 'Qi Xu'}, {'name': 'Qianqian Song'}, {'name': 'Rui Pan'}, {'name': 'Shenghan Zheng'}, {'name': 'Lingling Qiu'}, {'name': 'Tiantian Gu'}, {'name': 'Xinsheng Wu'}, {'name': 'Yulin Bi'}, {'name': 'Zhengfeng Cao'}, {'name': 'Yu Zhang'}, {'name': 'Yang Chen'}, {'name': 'Hong Li'}, {'name': 'Jianfeng Liu'}, {'name': 'Wangcheng Dai'}, {'name': 'Guohong Chen'}]\", 'abstract': '<p>The Chinese crested (CC) duck is a unique indigenous waterfowl breed with a phenotypic crest trait that affects its high survival rate. Therefore, the CC duck is an ideal model to investigate the genetic compensation response to maintain genetic stability. In the present study, we first generated a chromosome-level genome of CC ducks. Comparative genomics revealed genes related to tissue repair, immune function, and tumors were under strong positive selection, which suggested that these adaptive changes might enhance cancer resistance and immune response to maintain the genetic stability of CC ducks. We sub-assembled a Chinese spot-billed duck genome and detected genome-assembled structure variants among three ducks. Functional analysis revealed that a large number of structural variants were related to the immune system, which strongly suggests the occurrence of genetic compensation in the anti-tumor and immune systems to further support the survival of CC ducks. Moreover, we confirmed that the CC duck originated from the mallard ducks. Finally, we revealed the physiological and genetic basis of crest traits and identified a causative mutation in <italic>TAS2R40</italic> that leads to crest formation. Overall, the findings of this study provide new insights into the role of genetic compensation in adaptive evolution.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.452934', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Modelling time evolution of in blood alcohol concentration', 'authors': \"[{'name': 'Antoni Oliver Gelabert'}]\", 'abstract': '<p>A theoretical model based on mass conservation is proposed to predict blood alcohol concentration (BAC) over time. The proposed model takes as input the body parameters, the data timing for each drink, as well as his alcoholic strength and ingested amount. Three test cases with variations are presented and discussed. The results given are obtained by solving numerically the equation proposed. The solution predicts a well located alcohol peak and a reasonable time of alcohol body cleaning. This model represents a first order approach to a complex problem. Similar scenarios, like, for instance, drug concentrations can be modelled by using the same concept. The proposed model has been implemented on easy-to-use program COGORZA, which source code is available online at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/tonibois/Cogorza\">https://github.com/tonibois/Cogorza</ext-link>.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453568', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Global, Low Amplitude Cortical State Predicts Response Outcomes in a Selective Detection Task', 'authors': \"[{'name': 'Krista Marrero'}, {'name': 'Krithiga Aruljothi'}, {'name': 'Behzad Zareian'}, {'name': 'Chengchun Gao'}, {'name': 'Zhaoran Zhang'}, {'name': 'Edward Zagha'}]\", 'abstract': '<p>Spontaneous neuronal activity strongly impacts stimulus encoding and behavioral responses. We sought to determine the effects of neocortical prestimulus activity on stimulus detection. We trained mice in a selective whisker detection task, in which they learned to respond (lick) to target stimuli in one whisker field and ignore distractor stimuli in the contralateral whisker field. During expert task performance, we used widefield Ca<sup>2+</sup> imaging to assess prestimulus and post-stimulus neuronal activity broadly across frontal and parietal cortices. We found that lower prestimulus activity correlated with enhanced stimulus detection: lower prestimulus activity predicted response versus no response outcomes and faster reaction times. The activity predictive of trial outcome was distributed through dorsal neocortex, rather than being restricted to whisker or licking regions. Using principal component analysis, we demonstrate that response trials are associated with a distinct and less variable prestimulus neuronal subspace. For single units, prestimulus choice probability was weak yet distributed broadly, with lower than chance choice probability correlating with stronger sensory and motor encoding. These findings support a low amplitude, low variability, optimal prestimulus cortical state for stimulus detection that presents globally and predicts response outcomes for both target and distractor stimuli.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.27.453915', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'HyperChIP for identifying hypervariable signals across ChIP/ATAC-seq samples', 'authors': \"[{'name': 'Haojie Chen'}, {'name': 'Shiqi Tu'}, {'name': 'Chongze Yuan'}, {'name': 'Feng Tian'}, {'name': 'Yijing Zhang'}, {'name': 'Yihua Sun'}, {'name': 'Zhen Shao'}]\", 'abstract': '<p>With the reduction in sequencing costs, studies become prevalent that profile the chromatin landscape for tens or even hundreds of human individuals by using ChIP/ATAC-seq techniques. Identifying genomic regions with hypervariable ChIP/ATAC-seq signals across given samples is essential for such studies. In particular, the hypervariable regions (HVRs) across tumors from different patients indicate their heterogeneity and can contribute to revealing potential cancer subtypes and the associated epigenetic markers. We present HyperChIP as the first complete statistical tool for the task. HyperChIP uses scaled variances that account for the mean-variance dependence to rank genomic regions, and it increases the statistical power by diminishing the influence of true HVRs on model fitting. Applying it to a large pan-cancer ATAC-seq data set, we found that the identified HVRs not only provided a solid basis to uncover the underlying similarity structure among the involved tumor samples, but also led to the identification of transcription factors pertaining to the similarity structure when coupled with a motif-scanning analysis.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.24.453642', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Molecular Learning of a Soft-Disks Fluid', 'authors': \"[{'name': 'Luca Zammataro'}]\", 'abstract': '<p>This work is based on the Equivalence between Molecular Dynamics and Neural Network. It provides learning proofs in a Lennard-Jones (LJ) fluid, presented as a network of particles having non-bonded interactions. I describe the fluid’s learning as the property of an order that emerges as an adaptation in establishing equilibrium with energy and thermal conservation. The experimental section demonstrates the fluid can be trained with logic-gates patterns. The work goes beyond Molecular Computing’s application, explaining how this model uses its intrinsic minimizing properties in learning and predicting outputs. Finally, it gives hints for a theory on real chemistry’s computational universality.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.21260946', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Pediatrics', 'covid': False, 'title': \"Association between prenatal alcohol exposure and children's facial shape. A prospective population-based cohort study\", 'authors': \"[{'name': 'Xianjing Liu'}, {'name': 'Manfred Kayser'}, {'name': 'Steven A. Kushner'}, {'name': 'Henning Tiemeier'}, {'name': 'Fernando Rivadeneira'}, {'name': 'Vincent W. V. Jaddoe'}, {'name': 'Wiro Niessen'}, {'name': 'Eppo B. Wolvius'}, {'name': 'Gennady V. Roshchupkin'}]\", 'abstract': '<p>IMPORTANCE: Children exposed to a high level of prenatal alcohol exposure (PAE) are more likely to develop fetal alcohol spectrum disorder with adverse phenotypes on their faces. However, it is still poorly understood, which level of PAE is associated with facial manifestation and if such associations persist during childhood. \\nOBJECTIVE: To examine the association between PAE and children facial phenotype in a prospective multi-ethnic population-based study.\\nDESIGN, SETTING, AND PARTICIPANTS: This study was based on the Generation R Study, a prospective cohort from fetal life onwards with maternal and offspring data. Children had a 3D facial image taken at ages 9 (n=3160) and 13 years (n=2492). \\nEXPOSURES: We defined 6 levels of PAE based on the frequency and dose of alcohol consumption, and defined three tiers based on the timing of alcohol exposure of the unborn child.\\nMAIN OUTCOMES AND MEASURES:  For image analysis, we used 3D graph convolutional networks for non-linear dimensionality reduction, which compressed the high-dimensional images into 200 endophenotypes representing facial morphology. These facial endophenotypes were used as dependent variables in a linear regression analysis to search for associations with PAE. Finally, to detect specific facial components associated with PAE, we mapped statistically significant endophenotypes back to the 3D facial shape. We generated heatmaps to display the facial changes associated with PAE. PAE prediction based on facial shape was also performed; the prediction accuracy was estimated by area under the receiver operating characteristic curve (AUC).\\nRESULTS: A significant association between PAE and facial shape was found in the 9-year-old children, at all levels of alcohol exposure: the higher the level of exposure, the stronger the association. Moreover, PAE before and during pregnancy was associated with facial shape. The most common detected facial phenotypes included turned-up nose tip, shortened nose, turned-out jaw and turned-in lower-eyelid-related regions. For the 13-year-old children, the associations were weaker and the AUCs lower than those of the 9-years-old children.\\nCONCLUSIONS AND RELEVANCE: PAE before and during pregnancy, even at low level, is associated with the facial shape of children, and these associations become weaker as children grow older.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.24.453662', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Population colonization history influences behavioral responses of European starlings in personality tests', 'authors': \"[{'name': 'Alexandra Rodriguez'}, {'name': 'Martine Hausberger'}, {'name': 'Patricia Le Quilliec'}, {'name': 'Philippe Clergeau'}, {'name': 'Laurence Henry'}]\", 'abstract': '<p>To understand the processes involved in biological invasions, the genetic, morphological, physiological and behavioral characteristics of invasive populations need to be understood.</p><p>Many invasive species have been reported to be flying species. In birds, both invaders and migrants encounter novel situations, therefore one could expect that both groups might react similarly to novelty.</p><p>Here we analyzed the behavioral responses of individuals from three populations of European starling <italic>Sturnus vulgaris</italic>: a population settled for centuries in a rural region, a population that recently colonized an urban area, and a population of winter migrant birds. We conducted a social isolation test, a novel environment test, a novel food test and a novel object test to explore their reactions towards novelty. We identified and characterized different behavioral profiles for each test.</p><p>The group of migratory adults appeared to be less anxious in social isolation than the group of urban young. Urban and migrant groups entered the novel environment sooner than rural birds. Shy, bold and intermediate individuals were observed in all three groups when presented with novel food. Finally, the proportion of shy individuals which did not touch the novel object was higher than the proportion of bold individuals in the rural group. Our study emphasizes that neophilia or boldness present in migrant and invasive populations may facilitate the occupation of novel habitats. Our analysis also suggests that mixed reactions of neophobia ensure behavioral flexibility in a gregarious invasive species.</p><sec><title>Significant statement</title><p>In this paper, we show that an invasive species like European starling, <italic>Sturnus vulgaris</italic>, presents an important flexibility in neophobia and in reaction towards social isolation. These variations depend on the settlement history of populationseven when the birds had been wild-caught as nestlings and hand-raised in standard conditions. This is significant because it highlights possible scenarios of colonization processes.</p><p>We believe that this manuscript is appropriate for publication by Behavioral Ecology and Sociobiology because it places individuals’ behavior in the core mechanisms of an ecological phenomenom as biological invasions. Our manuscript enlarges the paradigms related to the ways of coping with novelty in animals.</p><p>This manuscript has not been published and is not under consideration for publication elsewhere.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453685', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'SegmA: Residue Segmentation of cryo-EM density maps', 'authors': \"[{'name': 'Mark Rozanov'}, {'name': 'Haim J. Wolfson'}]\", 'abstract': '<label>1</label><title>Abstract</title><p>The cryo-EM resolution revolution enables the development of algorithms for direct de-novo modelling of protein structures from given cryo-EM density maps. Deep Learning tools have been applied to locate structure patterns, such as rotamers, secondary structures and C<italic>α</italic> atoms. We present a deep neural network (nicknamed SegmA) for the residue type segmentation of a cryo-EM density map. The network labels voxels in a cryo-EM map by the residue type (amino acid type or nucleic acid) of the sampled macromolecular structure. It also provides a visual representation of the density map by coloring the different types of voxels by their assgned colors. SegmA’s algorithm combines a rotation equivariant group convolutional network with a traditional U-net network in a cascade. In addition SegmA estimates the labeling accuracy and reports only labels assigned with high confidence. At resolution of 3<italic>Å</italic> SegmAs accuracy is 80% for nucleotides. Amino acids which can be seen by eye, such as LEU, ARG and PHE, are detected by Segma with about 70% accuracy.</p><p>A web server of the application is under development at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://dev.dcsh7cbr3o89e.amplifyapp.com\">https://dev.dcsh7cbr3o89e.amplifyapp.com</ext-link>. The SegmA open code is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/Mark-Rozanov/SegmA_3A/tree/master\">https://github.com/Mark-Rozanov/SegmA_3A/tree/master</ext-link></p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453636', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'IRI-CCE: A Platform for Evaluating CRISPR-based Genome Editing Tools and Its Components', 'authors': \"[{'name': 'Rahul Mahadev Shelake'}, {'name': 'Dibyajyoti Pramanik'}, {'name': 'Jae-Yean Kim'}]\", 'abstract': '<p>Rapid assessment of Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-based tools and validation of cloned CRISPR-plasmid vectors are critical aspects for the successful application of genome editing (GE) in different organisms, including plants. Also, the selection of active guide RNAs (gRNAs) is a determining factor to achieve efficient GE at the targeted locus. Incidentally, plasmid vectors capable of ectopic expression of Cas enzyme and sgRNAs in bacteria will facilitate the quick screening and reliability of cloned plasmids and the functionality of designed gRNAs. We report a platform for in vivo rapid investigation of CRISPR components in <italic>Escherichia coli</italic> (IRI-CCE), which is compatible with plant-transforming binary vectors comprising plant promoters/terminators. Besides, IRI-CCE analyses reveal distinct features of cytidine (PmCDA1, evoCDA1, APOBEC3A) and adenine (ABE8e) base editors. Finally, we show the IRI-CCE platform as a reliable and rapid method for screening functional gRNAs to achieve successful GE outcomes. Base editor-based CRISPR components expressed by promoters of different strengths led to the establishment of IRI-CCE platform for three major applications: optimization of novel CRISPR tools, validation of cloned CRISPR plasmids, and to know gRNA functionality.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453664', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Habituation, task solving and memorization may facilitate biological invasions: the Starling example', 'authors': \"[{'name': 'Alexandra Rodriguez'}, {'name': 'Martine Hausberger'}, {'name': 'Philippe Clergeau'}, {'name': 'Laurence Henry'}]\", 'abstract': '<p>Invasions ecology deals more and more with behavioural characteristics of invasive species. Particularly, research have focused on the personality of invaders and on their way of coping with novelty in new habitats. Traits of neophobia may limit individuals in their exploration of novel objects or the consumption of novel foods, they may stop the access to valuable ressources. Actually, in novel environments like cities, food can be unreachable in throwaway dishes with lids or hidden in the garbage or even close to frightening objects. Animals may either left the place and waste these resources, or they can express low neophobia from the beginning and manipulate the objects to reach food. They may also habituate progressively to the context and use the ressources.</p><p>Here we analyzed the behavioural responses of individuals from three populations of European starling <italic>Sturnus vulgaris</italic>: a population anciently settled in a rural region, a population that has recently colonized a urban area and a population of wintering migrant birds.</p><p>We used a series of tests in order to explore if individuals would habituate to a novel object and if they could remember it eight months later. We explored if individuals would be less neophobic when confronted to two novel objects successively and we tested them in a learning task involving a novel object and an attractive food. Our results show that <italic>Sturnus vulgaris</italic> habituates rapidly to novel objects and that categorization facilitates neophobia lost when confronted to two different novel objects. Initially young birds appeared to be more skilled than adults in the learning task. Individuals from this species seem to be able to remember an object durably. We suggest that habituation, task solving and memorization are three mechanisms enhancing biological invasions.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453688', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Overexpression of the gene encoding neurosecretory protein GL precursor prevents excessive fat accumulation in the adipose tissue of mice fed a long-term high-fat diet', 'authors': \"[{'name': 'Keisuke Fukumura'}, {'name': 'Yuki Narimatsu'}, {'name': 'Shogo Moriwaki'}, {'name': 'Eiko Iwakoshi-Ukena'}, {'name': 'Megumi Furumitsu'}, {'name': 'Kazuyoshi Ukena'}]\", 'abstract': '<p>We previously identified a novel small hypothalamic protein, neurosecretory protein GL (NPGL), which induces feeding behavior and fat accumulation in rodents depending on their diet. In the present study, we explored the effects of NPGL on feeding behavior and energy metabolism in mice placed on a long-term high-fat diet with 60% calories from fat (HFD 60). Overexpression of the NPGL precursor gene <italic>(Npgl</italic>) over 18 weeks increased food intake and body mass. The weekly body mass gain of <italic>Npgl</italic>-overexpressing mice was higher than that of controls until 7 weeks from induction of overexpression, after which it ceased to be so. Oral glucose tolerance tests showed that <italic>Npgl</italic> overexpression maintained glucose tolerance and increased blood insulin levels, and intraperitoneal insulin tolerance tests showed that it maintained insulin sensitivity. At the experimental endpoint, <italic>Npgl</italic> overexpression was associated with increased mass of the perirenal white adipose tissue (WAT) and decreased mass of the epididymal WAT (eWAT), resulting in little effect on the total WAT mass. These results suggest that under long-term HFD 60 feeding, <italic>Npgl</italic> overexpression may play a role in avoiding metabolic disturbance both by accelerating energy storage and by suppressing excess fat accumulation in certain tissues, such as the eWAT.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453821', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PEGR: a flexible management platform for reproducible epigenomic and genomic research', 'authors': \"[{'name': 'Danying Shao'}, {'name': 'Gretta D. Kellogg'}, {'name': 'Ali Nematbakhsh'}, {'name': 'Prashant Kuntala'}, {'name': 'Shaun Mahony'}, {'name': 'B. Franklin Pugh'}, {'name': 'William KM Lai'}]\", 'abstract': '<p>Reproducibility is a significant challenge in (epi)genomic research due to the complexity of experiments composed of traditional biochemistry and informatics. Recent advances have exacerbated this challenge as high-throughput sequencing data is generated at an unprecedented pace. Here we report on our development of a <underline>P</underline>latform for <underline>E</underline>pi-<underline>G</underline>enomic <underline>R</underline>esearch (PEGR), a web-based project management platform that tracks and quality controls experiments from conception to publication-ready figures, compatible with multiple assays and bioinformatic pipelines. It supports rigor and reproducibility for biochemists working at the wet bench, while continuing to fully support reproducibility and reliability for bioinformaticians through integration with the Galaxy platform.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453858', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Crosstalk with keratinocytes causes GNAQ oncogene specificity in melanoma', 'authors': \"[{'name': 'Oscar Urtatiz'}, {'name': 'Amanda Haage'}, {'name': 'Guy Tanentzapf'}, {'name': 'Catherine D. Van Raamsdonk'}]\", 'abstract': '<p>Different melanoma subtypes exhibit specific and non-overlapping sets of oncogene and tumor suppressor mutations, despite a common cell of origin in melanocytes. For example, activation of the Gα<sub>q/11</sub> signaling pathway is a characteristic initiating event in primary melanomas that arise in the dermis, uveal tract or central nervous system. It is rare in melanomas arising in the epidermis. Here, we present evidence that in the mouse, crosstalk with the epidermal microenvironment actively impairs the survival of melanocytes expressing the GNAQ<sup>Q209L</sup> oncogene, providing a new model for oncogene specificity in cancer. The presence of epidermal cells inhibited cell division and fragmented dendrites of melanocytes expressing GNAQ<sup>Q209L</sup> in culture, while they promoted the growth of normal melanocytes. Differential gene expression analysis of FACS sorted epidermal melanocytes showed that cells expressing GNAQ<sup>Q209L</sup> exhibit an oxidative stress and apoptosis signature previously linked to vitiligo. Furthermore, PLCB4, the direct downstream effector of Gα<sub>q/11</sub> signaling, is frequently mutated in cutaneous melanoma alongside P53 and NF1. Our results suggest that a deficiency of PLCB4 promotes cutaneous melanomagenesis by reducing GNAQ driven signaling. Hence, our studies reveal the flip side of the GNAQ/PLCB4 signaling pathway, which was hitherto unsuspected. In the future, understanding how epidermal crosstalk restrains the GNAQ<sup>Q209L</sup> oncogene could suggest novel melanoma therapies.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453903', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': \"3D Convolutional Neural Networks for Classification of Alzheimer's and Parkinson's Disease with T1-Weighted Brain MRI\", 'authors': \"[{'name': 'Nikhil J. Dhinagar'}, {'name': 'Sophia I. Thomopoulos'}, {'name': 'Conor Owens-Walton'}, {'name': 'Dimitris Stripelis'}, {'name': 'Jose-Luis Ambite'}, {'name': 'Greg Ver Steeg'}, {'name': 'Daniel Weintraub'}, {'name': 'Philip Cook'}, {'name': 'Corey McMillan'}, {'name': 'Paul M. Thompson'}]\", 'abstract': \"<p>Parkinson's disease (PD) and Alzheimer's disease (AD) are progressive neurodegenerative disorders that affect millions of people worldwide. In this work, we propose a deep learning approach to classify these diseases based on 3D T1-weighted brain MRI. We analyzed several datasets including the Parkinson's Progression Markers Initiative (PPMI), an independent dataset from the University of Pennsylvania School of Medicine (UPenn), the Alzheimer's Disease Neuroimaging Initiative (ADNI), and the Open Access Series of Imaging Studies (OASIS) dataset. PPMI and ADNI were partitioned to train (70%), validate (20%), and test (10%) a 3D convolutional neural network (CNN) for PD and AD classification. The UPenn and OASIS datasets were used as independent test sets to evaluate the model performance during inference. We also implemented a random forest classifier as a baseline model by extracting key radiomics features from the same T1-weighted MRI scans. The proposed 3D CNN model was trained from scratch for the classification tasks. For AD classification, the 3D CNN model achieved an ROC-AUC of 0.878 on the ADNI test set and an average ROC-AUC of 0.789 on the OASIS dataset. For PD classification, the proposed 3D CNN model achieved an ROC-AUC of 0.667 on the PPMI test set and an average ROC-AUC of 0.743 on the UPenn dataset. We also found that model performance was largely maintained when using only 25% of the training dataset. The 3D CNN outperformed the random forest classifier for both the PD and AD tasks. The 3D CNN also generalized better on unseen MRI data from different imaging centers. Our results show that the proposed 3D CNN model was less prone to overfitting for AD than for PD classification. This approach shows promise for screening of PD and AD patients using only T1-weighted brain MRI, which is relatively widely available. This model with additional validation could also be used to help differentiate between challenging cases of AD and PD when they present with similarly subtle motor and non-motor symptoms.</p>\", 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.25.453696', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Artificial Intelligence Resolves Kinetic Pathways of Magnesium Binding to RNA', 'authors': \"[{'name': 'Jan Neumann'}, {'name': 'Nadine Schwierz'}]\", 'abstract': '<p>Magnesium is an indispensable cofactor in countless vital processes. In order to understand its functional role, the characterization of the binding pathways to biomolecules such as RNA is crucial. Despite the importance, a molecular description is still lacking since the transition from the water-mediated outer-sphere to the direct inner-sphere conformation is on the millisecond timescale and therefore out of reach for conventional simulation techniques. To fill this gap, we use transition path sampling to resolve the binding pathways and to elucidate the role of the solvent in the reaction. The results reveal that the molecular void provoked by the leaving phosphate oxygen of the RNA is immediately filled by an entering water molecule. In addition, water molecules from the first and second hydration shell couple to the concerted exchange. To capture the intimate solute-solvent coupling, we perform a committor analysis as basis for a machine learning algorithm that derives the optimal deep learning model from thousands of scanned architectures using hyperparameter tuning. The results reveal that the properly optimized deep network architecture recognizes the important solvent structures, extracts the relevant information and predicts the commitment probability with high accuracy. Our results provide a quantitative description of solute-solvent coupling which is ubiquitous for kosmotropic ions and governs a large variety of biochemical reactions in aqueous solutions.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453697', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Specialized subsets of innate-like T cells and dendritic cells protect from lethal pneumococcal infection in the lung', 'authors': \"[{'name': 'Mallory Paynich Murray'}, {'name': 'Catherine M. Crosby'}, {'name': 'Paola Marcovecchio'}, {'name': 'Nadine Hartmann'}, {'name': 'Shilpi Chandra'}, {'name': 'Meng Zhao'}, {'name': 'Archana Khurana'}, {'name': 'Sonja P. Zahner'}, {'name': 'Björn E. Clausen'}, {'name': 'Fadie T. Coleman'}, {'name': 'Joseph P. Mizgerd'}, {'name': 'Zbigniew Mikulski'}, {'name': 'Mitchell Kronenberg'}]\", 'abstract': '<p>Innate-like T cells, including invariant natural killer T (iNKT) cells, mucosal-associated invariant T (MAIT) cells and γδ T cells, are present in various barrier tissues, including the lung. They carry out protective responses during infections, but the mechanisms for protection are not completely understood. Here, we investigated their roles during pulmonary infection with <italic>Streptococcus pneumoniae</italic>. Following infection, innate-like T cells rapidly increased in lung tissue, in part through recruitment, but TCR activation and cytokine production occurred mostly in IL-17-producing NKT17 and γδ T cells. NKT17 cells were preferentially located outside the vasculature prior to infection, as were CD103<sup>+</sup> dendritic cells (cDC1), which were important both for antigen presentation to NKT17 cells and γδ T cell activation. Whereas IL-17A-producing γδ T cells also were numerous, GM-CSF was exclusive to NKT17 cells and contributed to iNKT cell-mediated protection. These studies demonstrate how particular cellular interactions and responses of functional subsets of innate-like T cells contribute to protection from pathogenic lung infection.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453707', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Opposing roles of blood-borne monocytes and tissue-resident macrophages in limbal stem cell damage after ocular injury', 'authors': \"[{'name': 'Chengxin Zhou'}, {'name': 'Fengyang Lei'}, {'name': 'Mirja Mittermaier'}, {'name': 'Bruce Ksander'}, {'name': 'Reza Dana'}, {'name': 'Claes H. Dohlman'}, {'name': 'Demetrios G. Vavvas'}, {'name': 'James Chodosh'}, {'name': 'Eleftherios I. Paschalis'}]\", 'abstract': '<p>Limbal stem cell (LSC) deficiency is a frequent and severe complication after chemical injury to the eye. Previous studies have assumed that this is mediated by direct chemical injury. Here we show that LSC loss can occur even without direct chemical injury to LSCs. Elevation of anterior chamber (AC) pH causes acute uveal stress, release of inflammatory cytokines at the basal limbal tissue, and subsequent LSC death. In particular, peripherally-derived CCR2<sup>+</sup>CX3CR1<sup>−-</sup>monocytes cause upregulation of tumor necrosis factor alpha (TNF-α) at the limbus, causing LSC loss. In contrast to peripherally-derived monocytes, CX3CR1<sup>+</sup>CCR2<sup>−-</sup>tissue-resident macrophages are protective. Pharmacological depletion of tissue-resident macrophages prior to injury exacerbates LSC loss and increases LSC vulnerability to apoptosis following exposure to TNF-α, despite reduced CCR2<sup>+</sup>cell infiltration into the tissue. Consistently, repopulation of the cornea by new tissue-resident macrophages after depletion reverse LSC susceptibility to inflammatory cell death. This findings have important implication in LSC protection after chemical injury.</p><sec><title>Significance</title><p>Limbal stem cell (LSC) loss after chemical burn is a significant clinical problem that has long been thought to be the result of direct chemical injury. This study demonstrates the existence of competing inflammatory mechanism responsible for LSC death, mediated by infiltrating peripheral monocytes that antagonize and overcome the protective role of tissue-resident macrophages. Thus, enhancing the role of tissue-resident macrophages while suppressing that of infiltrating monocytes could be a novel therapeutic strategy for LSC survival.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453746', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'EXPRESSION OF A SECRETABLE, CELL-PENETRATING CDKL5 PROTEIN ENHANCES THE EFFICACY OF AAV VECTOR-MEDIATED GENE THERAPY FOR CDKL5 DEFICIENCY DISORDER', 'authors': \"[{'name': 'Giorgio Medici'}, {'name': 'Marianna Tassinari'}, {'name': 'Giuseppe Galvani'}, {'name': 'Stefano Bastianini'}, {'name': 'Laura Gennaccaro'}, {'name': 'Manuela Loi'}, {'name': 'Claudia Fuchs'}, {'name': 'Nicola Mottolese'}, {'name': 'Sara Alvente'}, {'name': 'Chiara Berteotti'}, {'name': 'Helen Rappe Baggett'}, {'name': 'Giovanna Zoccoli'}, {'name': 'Maurizio Giustetto'}, {'name': 'Alysson Muotri'}, {'name': 'Hiroyuki Nakai'}, {'name': 'Stefania Trazzi'}, {'name': 'Elisabetta Ciani'}]\", 'abstract': '<p>No therapy is currently available for CDKL5 (cyclin-dependent kinase-like 5) deficiency disorder (CDD), a severe neurodevelopmental disorder caused by mutations in the CDKL5 gene. Although delivery of a wild-type copy of the mutated gene to cells represents the most curative approach for a monogenic disease, proof-of-concept studies highlight significant efficacy caveats for brain gene therapy. Herein, we used a secretable TATk-CDKL5 protein to enhance the efficiency of a gene therapy for CDD. We found that, although AAVPHP.B_Igk-TATk-CDKL5 and AAVPHP.B_CDKL5 vectors had similar brain infection efficiency, the AAVPHP.B_Igk-TATk-CDKL5 vector led to a higher CDKL5 protein replacement due to secretion and transduction of the TATk-CDKL5 protein into the neighboring cells. Importantly, <italic>Cdkl5</italic> KO mice treated with the AAVPHP.B_Igk-TATk-CDKL5 vector showed a behavioral and neuroanatomical improvement in comparison with vehicle-treated <italic>Cdkl5</italic> KO mice or <italic>Cdkl5</italic> KO mice treated with the AAVPHP.B_CDKL5 vector, indicating that a gene therapy based on a secretable recombinant TATk-CDKL5 protein is more effective at compensating <italic>Cdkl5</italic>-null brain defects than gene therapy based on the expression of the native CDKL5.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.21.21251509', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'Incidence of Lyme disease in the United Kingdom and association with fatigue: a population-based, historical cohort study', 'authors': \"[{'name': 'Florence Brellier'}, {'name': 'Mar Pujades-Rodriguez'}, {'name': 'Emma Powell'}, {'name': 'Kathleen Mudie'}, {'name': 'Eliana Lacerda'}, {'name': 'Luis Nacul'}, {'name': 'Kevin Wing'}]\", 'abstract': '<sec><title>Objectives</title><p>To evaluate incidence rates of Lyme disease in the UK and to investigate a possible association with subsequent fatigue</p></sec><sec><title>Design</title><p>Population-based historical cohort study with a comparator cohort matched by age, sex, and general practice</p></sec><sec><title>Setting</title><p>Patients treated in UK general practices contributing to IQVIA Medical Research Data</p></sec><sec><title>Participants</title><p>2,130 patients with a first diagnosis of Lyme disease between 2000 and 2018, and 8,510 randomly-sampled matched comparators, followed-up for a median time of 3 years and 8 months.</p></sec><sec><title>Main outcome measures</title><p>Time from Lyme disease diagnosis to consultation for any fatigue-related symptoms or diagnosis and for myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS). Adjusted hazard ratios (HRs) were estimated from Cox models.</p></sec><sec><title>Results</title><p>Average incidence rate for Lyme disease across the UK was 5.18 per 100,000 py between 2000 and 2018, increasing from 2.55 in 2000 to 9.33 in 2018. In total 929 events of any types of fatigue were observed, i.e. an incidence rate of 307.90 per 10,000 py in the Lyme cohort (282 events) and 165.60 in the comparator cohort (647 events). Effect of Lyme disease on any subsequent fatigue varied by index season with highest adjusted HRs in autumn [3.14 (95%CI: 1.92 to 5.13)] and winter [2.23 (1.21 to 4.11)]. Incidence rates of ME/CFS were 11.16 per 10,000 py in Lyme patients (12 events) and 1.20 in comparators (5 events), corresponding to an adjusted HR of 16.95 (5.17 to 55.60). Effect on any types of fatigue and ME/CFS was attenuated 6 months after diagnosis but still clearly visible.</p></sec><sec><title>Conclusions</title><p>UK primary care records provided strong evidence that Lyme disease was associated with acute and chronic fatigue. Albeit weaker, these effects persisted beyond 6 months, suggesting that patients and healthcare providers should remain alert to fatigue symptoms months to years following Lyme disease diagnosis.</p></sec><sec><title>Key messages box</title><sec><title>What is already known on this topic</title><p>Incidence rates of Lyme disease in the UK are increasing but estimations vary according to data sources used.</p><p>Reports investigating the association between Lyme disease and long-term fatigue are contradictory.</p></sec><sec><title>What this study adds</title><p>Average incidence rate for Lyme disease across the UK was estimated at 5.18 per 100,000 py between 2000 and 2018, and followed an increasing trend.</p><p>In patients with Lyme disease, a 2- and 3-fold increase in any subsequent fatigue was observed in winter and autumn, respectively, and a 16-fold increase in ME/CFS (all seasons combined), compared to a non-Lyme cohort matched by sex, age, and general practice.</p></sec></sec>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.22.453424', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Mechanisms of phosphatidylserine influence on viral production: a computational model of Ebola virus matrix protein assembly', 'authors': \"[{'name': 'Xiao Liu'}, {'name': 'Ethan J. Pappas'}, {'name': 'Monica L. Husby'}, {'name': 'Robert V. Stahelin'}, {'name': 'Elsje Pienaar'}]\", 'abstract': '<p>Ebola virus (EBOV) infections continue to pose a global public health threat, with high mortality rates and sporadic outbreaks in Central and Western Africa. A quantitative understanding of the key processes driving EBOV assembly and budding could provide valuable insights to inform drug development. Here we used a computational model to evaluate EBOV matrix assembly. Our model focused on the assembly kinetics of VP40, the matrix protein in EBOV, and its interaction with phosphatidylserine (PS) in the host cell membrane. Human cells transfected with VP40-expressing plasmids are capable of producing virus-like particles (VLPs) that closely resemble EBOV virions. We used data from this in vitro VP40 system to calibrate our computational model. PS levels in the host cell membrane had been shown to affect VP40 dynamics as well as VLP production through recruiting VP40 dimers to plasma membrane inner leaflet. Our computational results indicated that PS may have direct influence on VP40 filament growth and affect multiple steps in the assembly and budding of VP40 VLPs. We also proposed that the assembly of VP40 filaments may follow the nucleation-elongation theory where initialization and oligomerization of VP40 are two separate and distinct steps in the assembly process. This work illustrated how computational and experimental approaches can be combined to allow for additional analysis and hypothesis generation. Our findings advanced understanding of the molecular process of EBOV assembly and budding processes and may help the development of new EBOV treatments targeting VP40 matrix assembly.</p>', 'posted': '2021-07-25'}, {'doi': '10.1101/2021.07.23.453477', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Generative Adversarial Network augmented the gut microbiome-based health index by profoundly improved discrimination power', 'authors': \"[{'name': 'Yuxue Li'}, {'name': 'Gang Xie'}, {'name': 'Yuguo Zha'}, {'name': 'Kang Ning'}]\", 'abstract': '<sec><title>Summary</title><p>Gut microbiome-based health index (GMHI) has been applied with success, while the discrimination powers of GMHI varied for different diseases, limiting its utility on a broad spectrum of diseases. In this work, a Generative Adversarial Network (GAN) model is proposed to improve the discrimination power of GMHI. Built based on the batch corrected data through GAN (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/HUST-NingKang-Lab/GAN-GMHI\">https://github.com/HUST-NingKang-Lab/GAN-GMHI</ext-link>), GAN-GMHI has largely reduced the batch effects, and profoundly improved the performance for distinguishing healthy individuals and different diseases. GAN-GMHI has provided results to support the strong association of gut microbiome and diseases, and indicated a more accurate venue towards microbiome-based disease monitoring.</p></sec><sec><title>Availability and implementation</title><p>GAN-GMHI is publicly available on GitHub: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/HUST-NingKang-Lab/GAN-GMHI\">https://github.com/HUST-NingKang-Lab/GAN-GMHI</ext-link>.</p></sec><sec><title>Contact</title><p><email>ningkang@hust.edu.cn</email></p></sec><sec><title>Supplementary information</title><p>Supplementary data are available at Bioinformatics online.</p></sec>', 'posted': '2021-07-23'}, {'doi': '10.1101/2021.07.23.453588', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Accelerated epigenetic aging and inflammatory/immunological profile (ipAGE) in patients with chronic kidney disease', 'authors': \"[{'name': 'Igor Yusipov'}, {'name': 'Elena Kondakova'}, {'name': 'Alena Kalyakulina'}, {'name': 'Mikhail Krivonosov'}, {'name': 'Nadezhda Lobanova'}, {'name': 'Maria Giulia Bacalini'}, {'name': 'Claudio Franceschi'}, {'name': 'Maria Vedunova'}, {'name': 'Mikhail Ivanchenko'}]\", 'abstract': '<p>Chronic kidney disease (CKD) is defined by reduced estimated glomerular filtration rate (eGFR). This failure can be related to a phenotype of accelerated aging. In this work we considered 76 subjects with end-stage renal disease (ESRD) and 83 healthy controls. We evaluated two measures that can be informative of the rate of aging, i.e. whole blood DNA methylation using the Illumina Infinium EPIC array and plasma levels of a selection of inflammatory/immunological proteins using Multiplex Immunoassays. We demonstrated accelerated aging in terms of the most common epigenetic age estimators in CKD patients. We developed a new predictor of age based on inflammatory/immunological profile (ipAGE) and confirmed age acceleration in CKD patients. Finally, we evaluated the relationship between epigenetic age predictors and ipAGE and further identified the inflammatory/immunological biomarkers differentially expressed between cases and controls. In summary, our data show an accelerated aging phenotype in CKD patients sustained by inflammatory processes.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.453339', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Segmenting biological specimens from photos to understand the evolution of UV plumage in passerine birds', 'authors': \"[{'name': 'Yichen He'}, {'name': 'Christopher R. Cooney'}, {'name': 'Zoë K. Varley'}, {'name': 'Lara O. Nouri'}, {'name': 'Christopher J. A. Moody'}, {'name': 'Michael D. Jardine'}, {'name': 'Steve Maddock'}, {'name': 'Gavin H. Thomas'}]\", 'abstract': '<p>Ultraviolet (UV) colouration is thought to be an important signalling mechanism in many bird species, yet broad insights regarding the prevalence of UV plumage colouration and the factors promoting its evolution are currently lacking. Here, we develop a novel image segmentation pipeline based on deep learning that considerably outperforms classical (i.e. non-deep learning) segmentation methods, and use this to extract accurate information on whole-body plumage colouration from photographs of &gt;24,000 museum specimens covering &gt;4,500 species of passerine birds. Our results demonstrate that UV reflectance, particularly as a component of other colours, is widespread across the passerine radiation but is strongly phylogenetically conserved. We also find clear evidence in support of the role of light environment in promoting the evolution of UV plumage colouration, and a weak trend towards higher UV plumage reflectance among bird species with ultraviolet rather than violet-sensitive visual systems. Overall, our study provides important broad-scale insight into an enigmatic component of avian colouration, as well as demonstrating that deep learning has considerable promise for allowing new data to be bought to bear on long-standing questions in ecology and evolution.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21260986', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': False, 'title': 'A plasma metabolite score of three eicosanoids predicts incident type 2 diabetes – a prospective study in three independent cohorts', 'authors': \"[{'name': 'Karolina Tuomisto'}, {'name': 'Joonatan Palmu'}, {'name': 'Tao Long'}, {'name': 'Jeramie D. Watrous'}, {'name': 'Kysha Mercader'}, {'name': 'Kim A. Lagerborg'}, {'name': 'Allen Andres'}, {'name': 'Marko Salmi'}, {'name': 'Sirpa Jalkanen'}, {'name': 'Ramachandran S. Vasan'}, {'name': 'Michael Inouye'}, {'name': 'Aki S. Havulinna'}, {'name': 'Pekka Jousilahti'}, {'name': 'Teemu J. Niiranen'}, {'name': 'Susan Cheng'}, {'name': 'Mohit Jain'}, {'name': 'Veikko Salomaa'}]\", 'abstract': '<sec><title>Aim</title><p>Peptide markers of inflammation have been associated with the development of type 2 diabetes. The role of upstream, lipid-derived mediators of inflammation such as eicosanoids, remains less clear. The aim was to examine whether eicosanoids are associated with incident type 2 diabetes.</p></sec><sec><title>Methods</title><p>In the FINRISK 2002 study, a population-based sample of Finnish men and women aged 25-74 years, we used directed, non-targeted liquid chromatography – mass spectrometry to identify 545 eicosanoids and related oxylipins in the participants’ plasma samples (n=8,292). We used multivariable-adjusted Cox regression to examine associations between eicosanoids and incident type 2 diabetes. The findings were replicated in the Framingham Heart Study (FHS, n=2,886) and DILGOM 2007 (n=3,905). Together, these three cohorts had 1070 cases of incident type 2 diabetes.</p></sec><sec><title>Results</title><p>76 eicosanoids were associated individually with incident type 2 diabetes. We identified three eicosanoids independently associated with incident type 2 diabetes using stepwise Cox regression with forward selection and a Bonferroni-corrected inclusion threshold. A three-eicosanoid risk score produced a hazard ratio (HR) of 1.56 (95% confidence interval 1.41-1.72) per one standard deviation (SD) increment for risk of incident diabetes. The HR for comparing the top quartile to the lowest was 2.80 (2.53-3.07). Meta-analysis of the three cohorts yielded a pooled HR per SD of 1.31 (1.05-1.56).</p></sec><sec><title>Conclusion</title><p>Plasma eicosanoid profiles predict incident type 2 diabetes and the clearest signals replicate in three independent cohorts. Our findings give new information on the biology underlying type 2 diabetes and suggest opportunities for early identification of people at risk.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453511', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PatchPerPixMatch for Automated 3d Search of Neuronal Morphologies in Light Microscopy', 'authors': \"[{'name': 'Lisa Mais'}, {'name': 'Peter Hirsch'}, {'name': 'Claire Managan'}, {'name': 'Kaiyu Wang'}, {'name': 'Konrad Rokicki'}, {'name': 'Robert R. Svirskas'}, {'name': 'Barry J. Dickson'}, {'name': 'Wyatt Korff'}, {'name': 'Gerald M. Rubin'}, {'name': 'Gudrun Ihrke'}, {'name': 'Geoffrey W. Meissner'}, {'name': 'Dagmar Kainmueller'}]\", 'abstract': '<p>Studies of individual neurons in the Drosophila nervous system are facilitated by transgenic lines that sparsely and repeatably label respective neurons of interest. Sparsity can be enhanced by means of intersectional approaches like the split-GAL4 system, which labels the positive intersection of the expression patterns of two (denser) GAL4 lines. To this end, two GAL4 lines have to be identified as labelling a neuron of interest. Current approaches to tackling this task include visual inspection, as well as automated search in 2d projection images, of single cell multi-color flip-out (MCFO) acquisitions of GAL4 expression patterns. There is to date no automated method available that performs full 3d search in MCFO imagery of GAL4 lines, nor one that leverages automated reconstructions of the labelled neuron morphologies. To close this gap, we propose PatchPerPixMatch, a fully automated approach for finding a given neuron morphology in MCFO acquisitions of Gen1 GAL4 lines. PatchPerPixMatch performs automated instance segmentation of MCFO acquisitions, and subsequently searches for a target neuron morphology by minimizing an objective that aims at covering the target with a set of well-fitting segmentation fragments. Patch-PerPixMatch is computationally efficient albeit being full 3d, while also highly robust to inaccuracies in the automated neuron instance segmentation. We are releasing PatchPerPixMatch search results for ~30,000 neuron morphologies from the Drosophila <italic>hemibrain</italic> in ~20,000 MCFO acquisitions of ~3,500 Gen1 GAL4 lines.</p><sec><title>Code</title><p><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/Kainmueller-Lab/PatchPerPixMatch\">https://github.com/Kainmueller-Lab/PatchPerPixMatch</ext-link></p></sec><sec><title>Results</title><p><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://pppm.janelia.org\">https://pppm.janelia.org</ext-link></p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261034', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Orthopedics', 'covid': False, 'title': 'Running related biomechanical risk factors for overuse injuries in distance runners: A systematic review considering injury specificity and the potentials for future research', 'authors': \"[{'name': 'Steffen Willwacher'}, {'name': 'Markus Kurz'}, {'name': 'Johanna Robbin'}, {'name': 'Matthias Thelen'}, {'name': 'Joseph Hamill'}, {'name': 'Luke Kelly'}, {'name': 'Patrick Mai'}]\", 'abstract': '<sec><title>Objective</title><p>To identify and evaluate the evidence of the most relevant running-related risk factors (RRRFs) for running-related overuse injuries (ROIs) and to suggest future research directions.</p></sec><sec><title>Design</title><p>Systematic review considering prospective and retrospective studies. (PROSPERO_ID: 236832)</p></sec><sec><title>Data sources</title><p>Pubmed. Connected Papers. The search was performed in February 2021.</p></sec><sec><title>Eligibility criteria</title><p>English language. Studies on participants whose primary sport is running addressing the risk for the seven most common ROIs and at least one kinematic, kinetic (including pressure measurements), or electromyographic RRRF. An RRRF needed to be identified in at least one prospective or two retrospective studies.</p></sec><sec><title>Results</title><p>Sixty-two articles fulfilled our eligibility criteria. Levels of evidence for specific ROIs ranged from conflicting to moderate evidence. Running populations and methods applied varied considerably between studies. While some RRRFs appeared for several ROIs, most RRRFs were specific for a particular ROI. The biomechanical measurements performed in many studies would have allowed for consideration of many more RRRFs than have been reported, highlighting a potential for more effective data usage in the future.</p></sec><sec><title>Conclusion</title><p>This study offers a comprehensive overview of RRRFs for the most common ROIs, which might serve as a starting point to develop ROI-specific risk profiles of individual runners. Future work should use macroscopic (big data) approaches involving long-term data collections in the real world and microscopic approaches involving precise stress calculations using recent developments in biomechanical modelling. However, consensus on data collection standards (including the quantification of workload and stress tolerance variables and the reporting of injuries) is warranted.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.21260972', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'The mystery of COVID-19 reinfections: A global systematic review and meta-analysis of 577 cases', 'authors': \"[{'name': 'Rubaid Azhar Dhillon'}, {'name': 'Mohammad Aadil Qamar'}, {'name': 'Omar Irfan'}, {'name': 'Jaleed Ahmed Gilani'}, {'name': 'Usama Waqar'}, {'name': 'Mir Ibrahim Sajid'}, {'name': 'Syed Faisal Mahmood'}]\", 'abstract': '<sec><title>Background</title><p>As the COVID-19 pandemic rages on, reports on disparities in vaccine roll out alongside reinfection and reactivation from previously recovered cases have been emerging. With newer waves and variants of COVID-19, we conducted a systematic review to assess the determinants and disease spectrum of COVID-19 reinfection.</p></sec><sec><title>Methods</title><p>A comprehensive search covering relevant databases was conducted for observational studies reporting Polymerase Chain Reaction (PCR) confirmed infection and reinfection cases. Quality assessment tool developed by the National Institute of Health (NIH) for assessment of case series was used. Meta-analyses were performed using RevMan 5.3 for pooled proportions of findings in first infection and reinfection with 95% confidence interval (CI).</p></sec><sec><title>Results</title><p>Eighty-one studies reporting 577 cases were included from 22 countries. The mean age of patients was 46.2±18.9 years with males accounting for 45.8% of the study population while 179 (31.0%) cases of comorbidities were reported. The average time duration between first infection and reinfection was 63.6±48.9 days. During first infection and reinfection, fever was the most common symptom (41.4% and 36.4%, respectively) whilst anti-viral therapy was the most common treatment regimen administered (44.5% and 43.0%, respectively). Overall, comparable odds of symptomatic presentation and management were reported in the two infections. However, a higher Intensive Care Unit (ICU) admission rate was observed in reinfection compared to first infection (10 vs 3). Ten deaths were reported with 565 patients fully recovering. Respiratory failure was the most common cause of death (7/10 deaths). Seventy-two studies were determined to be of good quality whilst nine studies were of fair quality.</p></sec><sec><title>Conclusion</title><p>As the first global-scale systematic review of its kind, our findings support immunization practices given increased ICU admissions and mortality in reinfections. Our cohort serves as a guide for clinicians and authorities for devising an optimal strategy for controlling the pandemic.</p></sec>', 'posted': '2021-07-24'}, {'doi': '10.1101/2021.07.24.21261076', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Health Informatics', 'covid': True, 'title': 'The U.S. COVID-19 Trends and Impact Survey, 2020-2021: Continuous real-time measurement of COVID-19 symptoms, risks, protective behaviors, testing and vaccination', 'authors': \"[{'name': 'Joshua A. Salomon'}, {'name': 'Alex Reinhart'}, {'name': 'Alyssa Bilinski'}, {'name': 'Eu Jing Chua'}, {'name': 'Wichada La Motte-Kerr'}, {'name': 'Minttu M. Rönn'}, {'name': 'Marissa Reitsma'}, {'name': 'Katherine Ann Morris'}, {'name': 'Sarah LaRocca'}, {'name': 'Tamar Farag'}, {'name': 'Frauke Kreuter'}, {'name': 'Roni Rosenfeld'}, {'name': 'Ryan J. Tibshirani'}]\", 'abstract': '<p>The U.S. COVID-19 Trends and Impact Survey (CTIS) is a large, cross-sectional, Internet-based survey that has operated continuously since April 6, 2020. By inviting a random sample of Facebook active users each day, CTIS collects information about COVID-19 symptoms, risks, mitigating behaviors, mental health, testing, vaccination, and other key priorities. The large scale of the survey – over 20 million responses in its first year of operation – allows tracking of trends over short timescales and allows comparisons at fine demographic and geographic detail. The survey has been repeatedly revised to respond to emerging public health priorities. In this paper, we describe the survey methods and content and give examples of CTIS results that illuminate key patterns and trends and help answer high-priority policy questions relevant to the COVID-19 epidemic and response. These results demonstrate how large online surveys can provide continuous, real-time indicators of important outcomes that are not subject to public health reporting delays and backlogs. The CTIS offers high value as a supplement to official reporting data by supplying essential information about behaviors, attitudes toward policy and preventive measures, economic impacts, and other topics not reported in public health surveillance systems.</p><sec><title>Significance statement</title><p>The U.S. COVID-19 Trends and Impact Survey (CTIS) has operated continuously since April 6, 2020, collecting over 20 million responses. The largest public health survey ever conducted in the United States, CTIS was designed to facilitate detailed demographic and geographic analyses, track trends over time, and accommodate rapid response to emerging priorities. Using examples of CTIS results illuminating trends in symptoms, risks, mitigating behaviors, testing and vaccination in relation to evolving high-priority policy questions over 12 months of the pandemic, we illustrate the value of online surveys for tracking patterns and trends in COVID outcomes as an adjunct to official reporting, and showcase unique insights that would not be visible through traditional public health reporting.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261039', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Geographical concentration of COVID-19 cases by social determinants of health in 16 large metropolitan areas in Canada – a cross-sectional study', 'authors': \"[{'name': 'Yiqing Xia'}, {'name': 'Huiting Ma'}, {'name': 'Gary Moloney'}, {'name': 'Héctor A. Velásquez García'}, {'name': 'Monica Sirski'}, {'name': 'Naveed Z. Janjua'}, {'name': 'David Vickers'}, {'name': 'Tyler Williamson'}, {'name': 'Alan Katz'}, {'name': 'Kristy Yu'}, {'name': 'Rafal Kustra'}, {'name': 'David L Buckeridge'}, {'name': 'Marc Brisson'}, {'name': 'Stefan D Baral'}, {'name': 'Sharmistha Mishra'}, {'name': 'Mathieu Maheu-Giroux'}]\", 'abstract': '<sec><title>Background</title><p>There is a growing recognition that strategies to reduce SARS-CoV-2 transmission should be responsive to local transmission dynamics. Studies have revealed inequalities along social determinants of health, but little investigation was conducted surrounding geographic concentration within cities. We quantified social determinants of geographic concentration of COVID-19 cases across sixteen census metropolitan areas (CMA) in four Canadian provinces.</p></sec><sec><title>Methods</title><p>We used surveillance data on confirmed COVID-19 cases at the level of dissemination area. Gini (co-Gini) coefficients were calculated by CMA based on the proportion of the population in ranks of diagnosed cases and each social determinant using census data (income, education, visible minority, recent immigration, suitable housing, and essential workers) and the corresponding share of cases. Heterogeneity was visualized using Lorenz (concentration) curves.</p></sec><sec><title>Results</title><p>Geographic concentration was observed in all CMAs (half of the cumulative cases were concentrated among 21-35% of each city’s population): with the greatest geographic heterogeneity in Ontario CMAs (Gini coefficients, 0.32-0.47), followed by British Columbia (0.23-0.36), Manitoba (0.32), and Québec (0.28-0.37). Cases were disproportionately concentrated in areas with lower income, education attainment, and suitable housing; and higher proportion of visible minorities, recent immigrants, and essential workers. Although a consistent feature across CMAs was concentration by proportion visible minorities, the magnitude of concentration by social determinants varied across CMAs.</p></sec><sec><title>Interpretation</title><p>The feature of geographical concentration of COVID-19 cases was consistent across CMAs, but the pattern by social determinants varied. Geographically-prioritized allocation of resources and services should be tailored to the local drivers of inequalities in transmission in response to SARS-CoV-2’s resurgence.</p></sec>', 'posted': '2021-07-26'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.07.19.21260767</td>\n",
       "      <td>625</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Pediatrics</td>\n",
       "      <td>True</td>\n",
       "      <td>Children with SARS-CoV-2 in the National COVID...</td>\n",
       "      <td>[{'name': 'Blake Martin'}, {'name': 'Peter E. ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Importance&lt;/title&gt;&lt;p&gt;SARS-CoV-2&lt;/p...</td>\n",
       "      <td>2021-07-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.07.23.21261030</td>\n",
       "      <td>232</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Breakthrough Symptomatic COVID-19 Infections L...</td>\n",
       "      <td>[{'name': 'Daisy Massey'}, {'name': 'Diana Ber...</td>\n",
       "      <td>&lt;p&gt;Vaccines have been shown to be extremely ef...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.07.26.453748</td>\n",
       "      <td>74</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Identification of a new family of “megaphages”...</td>\n",
       "      <td>[{'name': 'Slawomir Michniewski'}, {'name': 'B...</td>\n",
       "      <td>&lt;p&gt;Megaphages – bacteriophages harbouring extr...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.07.23.453070</td>\n",
       "      <td>60</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Regulation of sedimentation rate shapes the ev...</td>\n",
       "      <td>[{'name': 'Omaya Dudin'}, {'name': 'Sébastien ...</td>\n",
       "      <td>&lt;p&gt;Significant increases in sedimentation rate...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.07.23.453379</td>\n",
       "      <td>57</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>AlphaPept, a modern and open framework for MS-...</td>\n",
       "      <td>[{'name': 'Maximilian T. Strauss'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;In common with other omics technologies, ma...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.07.23.453492</td>\n",
       "      <td>52</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Scientific Communication and Education</td>\n",
       "      <td>False</td>\n",
       "      <td>Delineating Medical Education: Bibliometric Re...</td>\n",
       "      <td>[{'name': 'Lauren A. Maggio'}, {'name': 'Anton...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The field of ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.07.23.453605</td>\n",
       "      <td>51</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Identifying cell-state associated alternative ...</td>\n",
       "      <td>[{'name': 'Carlos F. Buen Abad Najar'}, {'name...</td>\n",
       "      <td>&lt;p&gt;Alternative splicing shapes the transcripto...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.07.24.21261040</td>\n",
       "      <td>47</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Novel risk factors for Coronavirus disease-ass...</td>\n",
       "      <td>[{'name': 'Umang Arora'}, {'name': 'Megha Priy...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The epidemiol...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.07.23.21261041</td>\n",
       "      <td>44</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>The impact of large mobile air purifiers on ae...</td>\n",
       "      <td>[{'name': 'F. F. Duill'}, {'name': 'F. Schulz'...</td>\n",
       "      <td>&lt;p&gt;In the wake of the SARS-CoV-2 pandemic, an ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.07.23.453478</td>\n",
       "      <td>44</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Mechanosensitivity of nucleocytoplasmic transport</td>\n",
       "      <td>[{'name': 'Ion Andreu'}, {'name': 'Ignasi Gran...</td>\n",
       "      <td>&lt;p&gt;Mechanical force controls fundamental cellu...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.07.23.453573</td>\n",
       "      <td>43</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Strong and opponent contributions of dorsomedi...</td>\n",
       "      <td>[{'name': 'Scott S. Bolkan'}, {'name': 'Iris R...</td>\n",
       "      <td>&lt;p&gt;A classic view of the striatum holds that a...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.07.22.453458</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Epithelial-mesenchymal plasticity determines e...</td>\n",
       "      <td>[{'name': 'Patrick Aouad'}, {'name': 'Yueyun Z...</td>\n",
       "      <td>&lt;p&gt;Estrogen receptor α-positive (ER+) breast c...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.07.23.453521</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Probiotic consortia improve anti-viral immunit...</td>\n",
       "      <td>[{'name': 'J Lehtinen Markus'}, {'name': 'Kuma...</td>\n",
       "      <td>&lt;p&gt;Probiotics have been suggested as one solut...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.07.26.453760</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>A bacterial dynamin-like protein confers a nov...</td>\n",
       "      <td>[{'name': 'Lijun Guo'}, {'name': 'Laura Sattle...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Bacillus subtilis&lt;/italic&gt; DynA is ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.07.27.453996</td>\n",
       "      <td>33</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Disruption of the TCA cycle reveals an ATF4-de...</td>\n",
       "      <td>[{'name': 'Dylan Gerard Ryan'}, {'name': 'Ming...</td>\n",
       "      <td>&lt;p&gt;The Tricarboxylic Acid Cycle (TCA) cycle is...</td>\n",
       "      <td>2021-07-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.07.25.21261093</td>\n",
       "      <td>31</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Protection afforded by the BNT162b2 and mRNA-1...</td>\n",
       "      <td>[{'name': 'Laith J. Abu-Raddad'}, {'name': 'Hi...</td>\n",
       "      <td>&lt;p&gt;Effect of prior SARS-CoV-2 infection on vac...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.07.21.453258</td>\n",
       "      <td>30</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Retrosplenial cortex is necessary for spatial ...</td>\n",
       "      <td>[{'name': 'Ana Carolina Bottura de Barros'}, {...</td>\n",
       "      <td>&lt;p&gt;Latent learning occurs when associations ar...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.07.27.453460</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Extracellular mechanical forces drive endocard...</td>\n",
       "      <td>[{'name': 'Julien Vermot'}, {'name': 'Helene V...</td>\n",
       "      <td>&lt;p&gt;Organ morphogenesis involves dynamic change...</td>\n",
       "      <td>2021-07-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.07.26.453816</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Synaptic Mechanisms of Top-Down Control by The...</td>\n",
       "      <td>[{'name': 'Hannah M. Oberle'}, {'name': 'Alex ...</td>\n",
       "      <td>&lt;p&gt;Corticofugal projections to evolutionarily ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.07.26.453747</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>ABlooper: Fast accurate antibody CDR loop stru...</td>\n",
       "      <td>[{'name': 'Brennan Abanades'}, {'name': 'Guy G...</td>\n",
       "      <td>&lt;p&gt;Antibodies are a key component of the immun...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.07.26.21261130</td>\n",
       "      <td>22</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Vaccine effectiveness when combining the ChAdO...</td>\n",
       "      <td>[{'name': 'Mie Agermose Gram'}, {'name': 'Hann...</td>\n",
       "      <td>&lt;p&gt;Background: The recommendations in several ...</td>\n",
       "      <td>2021-07-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.07.22.21260852</td>\n",
       "      <td>20</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Transplantation</td>\n",
       "      <td>True</td>\n",
       "      <td>Comparison of infected and vaccinated transpla...</td>\n",
       "      <td>[{'name': 'Xavier Charmetant'}, {'name': 'Maxi...</td>\n",
       "      <td>&lt;p&gt;Transplant recipients, which receive therap...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.07.25.453647</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Oncogenic c-Myc induces replication stress by ...</td>\n",
       "      <td>[{'name': 'Silvia Peripolli'}, {'name': 'Tanya...</td>\n",
       "      <td>&lt;p&gt;Oncogene-induced replication stress is a ma...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.07.24.453184</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>The dual nature of bacteriophage: growth-depen...</td>\n",
       "      <td>[{'name': 'Quentin J Leclerc'}, {'name': 'Jaco...</td>\n",
       "      <td>&lt;p&gt;Bacteriophage (“phage”) are both predators ...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.07.24.453616</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A yeast lysosomal biogenesis map uncovers the ...</td>\n",
       "      <td>[{'name': 'Sebastian Eising'}, {'name': 'Bianc...</td>\n",
       "      <td>&lt;p&gt;The lysosome is the major catabolic organel...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.07.25.453678</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Reward-dependent selection of feedback gains i...</td>\n",
       "      <td>[{'name': 'Antoine De Comite'}, {'name': 'Fréd...</td>\n",
       "      <td>&lt;p&gt;Expected reward is known to affect planning...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.07.26.453637</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>LAMP2A regulates the loading of proteins into ...</td>\n",
       "      <td>[{'name': 'João Vasco Ferreira'}, {'name': 'An...</td>\n",
       "      <td>&lt;p&gt;Exosomes are extracellular vesicles of endo...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.07.25.453673</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Multivariate mining of an alpaca immune repert...</td>\n",
       "      <td>[{'name': 'Leo Hanke'}, {'name': 'Daniel J. Sh...</td>\n",
       "      <td>&lt;p&gt;Conventional approaches to isolate and char...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.07.24.453634</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Crowding-induced phase separation of nuclear t...</td>\n",
       "      <td>[{'name': 'Luke K. Davis'}, {'name': 'Ian J. F...</td>\n",
       "      <td>&lt;p&gt;The rapid (&amp;lt; 1 ms) transport of biologic...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.07.22.453442</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Leveraging gene co-regulation to identify gene...</td>\n",
       "      <td>[{'name': 'Katherine M. Siewert-Rocks'}, {'nam...</td>\n",
       "      <td>&lt;p&gt;Identifying gene sets that are associated t...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.07.22.21260551</td>\n",
       "      <td>14</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Willingness to receive vaccination against COV...</td>\n",
       "      <td>[{'name': 'Sharon R Davis'}, {'name': 'Rosario...</td>\n",
       "      <td>&lt;p&gt;We surveyed 10,024 Australians regarding CO...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.07.25.453674</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Role of non-specific interactions in the phase...</td>\n",
       "      <td>[{'name': 'Rakesh Krishnan'}, {'name': 'Srivas...</td>\n",
       "      <td>&lt;p&gt;Phase separation of biomolecules could be m...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.07.23.453610</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Adapted protocol for Saccharibacteria co-culti...</td>\n",
       "      <td>[{'name': 'Ahmad Ibrahim'}, {'name': 'Mohamad ...</td>\n",
       "      <td>&lt;p&gt;The growing application of metagenomics to ...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.07.18.21259977</td>\n",
       "      <td>12</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>False</td>\n",
       "      <td>Altered functional connectome hierarchy with g...</td>\n",
       "      <td>[{'name': 'Christophe E de Bézenac'}, {'name':...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Objective&lt;/title&gt;&lt;p&gt;Neuroimaging r...</td>\n",
       "      <td>2021-07-22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.07.25.453574</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Bats host the most virulent—but not the most d...</td>\n",
       "      <td>[{'name': 'Sarah Guth'}, {'name': 'Nardus Moll...</td>\n",
       "      <td>&lt;p&gt;Identifying virus characteristics associate...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.07.25.453666</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Identification of a divergent basal body assem...</td>\n",
       "      <td>[{'name': 'Shizuka Koshimizu'}, {'name': 'Naok...</td>\n",
       "      <td>&lt;p&gt;Oogamy is a form of sexual reproduction and...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.07.24.452477</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Trial-wise exposure to visual appetitive cues ...</td>\n",
       "      <td>[{'name': 'Kilian Knauth'}, {'name': 'Jan Pete...</td>\n",
       "      <td>&lt;p&gt;Humans and many animals devalue future rewa...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.07.25.453708</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The effective family size of immigrant founder...</td>\n",
       "      <td>[{'name': 'Tommy Harding'}, {'name': 'Emmanuel...</td>\n",
       "      <td>&lt;p&gt;Human evolution involves population splits,...</td>\n",
       "      <td>2021-07-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.07.25.453671</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>MetaBinner: a high-performance and stand-alone...</td>\n",
       "      <td>[{'name': 'Ziye Wang'}, {'name': 'Pingqin Huan...</td>\n",
       "      <td>&lt;p&gt;Binning is an essential procedure during me...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.07.23.21261042</td>\n",
       "      <td>10</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Robust antibody levels in both diabetic and no...</td>\n",
       "      <td>[{'name': 'Hamad Ali'}, {'name': 'Abdelmohsen ...</td>\n",
       "      <td>&lt;p&gt;The emergence of effective vaccines for COV...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.07.26.453744</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Global protein responses of multi-drug resista...</td>\n",
       "      <td>[{'name': 'Anatte Margalit'}, {'name': 'James ...</td>\n",
       "      <td>&lt;p&gt;Antimicrobial resistance (AMR) and multi-dr...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.07.26.453807</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>The benefits of merging two tracking approache...</td>\n",
       "      <td>[{'name': 'Louise Chavarie'}, {'name': 'Hannel...</td>\n",
       "      <td>&lt;p&gt;The process of smolting is a critical phase...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.07.26.453733</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>AGO1 regulates major satellite transcripts and...</td>\n",
       "      <td>[{'name': 'Madlen Müller'}, {'name': 'Tara Fäh...</td>\n",
       "      <td>&lt;p&gt;In the past years, several studies reported...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.07.23.453597</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Generalizing Bayesian phylogenetics to infer s...</td>\n",
       "      <td>[{'name': 'Jamie R. Oaks'}, {'name': 'Perry L....</td>\n",
       "      <td>&lt;p&gt;Many processes of biological diversificatio...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.07.26.453355</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Inhibition of striatal SEZ6 by miR-3594-5p is ...</td>\n",
       "      <td>[{'name': 'Magalie Lenoir'}, {'name': 'Isabell...</td>\n",
       "      <td>&lt;p&gt;Escalation of drug use is a hallmark stage ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.07.23.453341</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Trait-trait relationships and functional trade...</td>\n",
       "      <td>[{'name': 'Sara Beier'}, {'name': 'Johannes We...</td>\n",
       "      <td>&lt;p&gt;We report genomic traits that have been ass...</td>\n",
       "      <td>2021-07-24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.07.25.453699</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Supra-physiological levels of gibberellins/DEL...</td>\n",
       "      <td>[{'name': 'Iva McCarthy-Suárez'}]</td>\n",
       "      <td>&lt;p&gt;A previous study (McCarthy-Suárez, 2021) sh...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.07.22.21258785</td>\n",
       "      <td>8</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Gastroenterology</td>\n",
       "      <td>True</td>\n",
       "      <td>Proton pump inhibitor use is not associated wi...</td>\n",
       "      <td>[{'name': 'Shailja C. Shah'}, {'name': 'Alese ...</td>\n",
       "      <td>&lt;p&gt;Background and Aims: Low pH deactivates mos...</td>\n",
       "      <td>2021-07-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.07.24.453449</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Heat-inactivated modified vaccinia virus Ankar...</td>\n",
       "      <td>[{'name': 'Ning Yang'}, {'name': 'Aitor Garzia...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Protein or pe...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.07.23.453553</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Community health and human-animal contacts on ...</td>\n",
       "      <td>[{'name': 'Renata L. Muylaert'}, {'name': 'Ben...</td>\n",
       "      <td>&lt;p&gt;Cross-species transmission of pathogens is ...</td>\n",
       "      <td>2021-07-23</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2021.07.19.21260767     625  medRxiv   \n",
       "1   10.1101/2021.07.23.21261030     232  medRxiv   \n",
       "2     10.1101/2021.07.26.453748      74  bioRxiv   \n",
       "3     10.1101/2021.07.23.453070      60  bioRxiv   \n",
       "4     10.1101/2021.07.23.453379      57  bioRxiv   \n",
       "5     10.1101/2021.07.23.453492      52  bioRxiv   \n",
       "6     10.1101/2021.07.23.453605      51  bioRxiv   \n",
       "7   10.1101/2021.07.24.21261040      47  medRxiv   \n",
       "8   10.1101/2021.07.23.21261041      44  medRxiv   \n",
       "9     10.1101/2021.07.23.453478      44  bioRxiv   \n",
       "10    10.1101/2021.07.23.453573      43  bioRxiv   \n",
       "11    10.1101/2021.07.22.453458      41  bioRxiv   \n",
       "12    10.1101/2021.07.23.453521      37  bioRxiv   \n",
       "13    10.1101/2021.07.26.453760      37  bioRxiv   \n",
       "14    10.1101/2021.07.27.453996      33  bioRxiv   \n",
       "15  10.1101/2021.07.25.21261093      31  medRxiv   \n",
       "16    10.1101/2021.07.21.453258      30  bioRxiv   \n",
       "17    10.1101/2021.07.27.453460      29  bioRxiv   \n",
       "18    10.1101/2021.07.26.453816      23  bioRxiv   \n",
       "19    10.1101/2021.07.26.453747      23  bioRxiv   \n",
       "20  10.1101/2021.07.26.21261130      22  medRxiv   \n",
       "21  10.1101/2021.07.22.21260852      20  medRxiv   \n",
       "22    10.1101/2021.07.25.453647      20  bioRxiv   \n",
       "23    10.1101/2021.07.24.453184      20  bioRxiv   \n",
       "24    10.1101/2021.07.24.453616      19  bioRxiv   \n",
       "25    10.1101/2021.07.25.453678      16  bioRxiv   \n",
       "26    10.1101/2021.07.26.453637      15  bioRxiv   \n",
       "27    10.1101/2021.07.25.453673      15  bioRxiv   \n",
       "28    10.1101/2021.07.24.453634      14  bioRxiv   \n",
       "29    10.1101/2021.07.22.453442      14  bioRxiv   \n",
       "30  10.1101/2021.07.22.21260551      14  medRxiv   \n",
       "31    10.1101/2021.07.25.453674      12  bioRxiv   \n",
       "32    10.1101/2021.07.23.453610      12  bioRxiv   \n",
       "33  10.1101/2021.07.18.21259977      12  medRxiv   \n",
       "34    10.1101/2021.07.25.453574      12  bioRxiv   \n",
       "35    10.1101/2021.07.25.453666      11  bioRxiv   \n",
       "36    10.1101/2021.07.24.452477      11  bioRxiv   \n",
       "37    10.1101/2021.07.25.453708      11  bioRxiv   \n",
       "38    10.1101/2021.07.25.453671      10  bioRxiv   \n",
       "39  10.1101/2021.07.23.21261042      10  medRxiv   \n",
       "40    10.1101/2021.07.26.453744       9  bioRxiv   \n",
       "41    10.1101/2021.07.26.453807       9  bioRxiv   \n",
       "42    10.1101/2021.07.26.453733       9  bioRxiv   \n",
       "43    10.1101/2021.07.23.453597       9  bioRxiv   \n",
       "44    10.1101/2021.07.26.453355       9  bioRxiv   \n",
       "45    10.1101/2021.07.23.453341       9  bioRxiv   \n",
       "46    10.1101/2021.07.25.453699       9  bioRxiv   \n",
       "47  10.1101/2021.07.22.21258785       8  medRxiv   \n",
       "48    10.1101/2021.07.24.453449       8  bioRxiv   \n",
       "49    10.1101/2021.07.23.453553       8  bioRxiv   \n",
       "\n",
       "                              subject-area  covid  \\\n",
       "0                               Pediatrics   True   \n",
       "1                             Epidemiology   True   \n",
       "2                             Microbiology  False   \n",
       "3                     Evolutionary Biology  False   \n",
       "4                           Bioinformatics  False   \n",
       "5   Scientific Communication and Education  False   \n",
       "6                                 Genomics  False   \n",
       "7                             Epidemiology   True   \n",
       "8                 Public and Global Health   True   \n",
       "9                               Biophysics  False   \n",
       "10                            Neuroscience  False   \n",
       "11                          Cancer Biology  False   \n",
       "12                              Immunology   True   \n",
       "13                            Microbiology  False   \n",
       "14                            Biochemistry  False   \n",
       "15                            Epidemiology   True   \n",
       "16                            Neuroscience  False   \n",
       "17                   Developmental Biology  False   \n",
       "18                            Neuroscience  False   \n",
       "19                          Bioinformatics  False   \n",
       "20                            Epidemiology   True   \n",
       "21                         Transplantation   True   \n",
       "22                          Cancer Biology  False   \n",
       "23                            Microbiology  False   \n",
       "24                            Cell Biology  False   \n",
       "25                            Neuroscience  False   \n",
       "26                            Cell Biology  False   \n",
       "27                              Immunology   True   \n",
       "28                              Biophysics  False   \n",
       "29                                Genetics  False   \n",
       "30   Infectious Diseases (except HIV/AIDS)   True   \n",
       "31                              Biophysics  False   \n",
       "32                            Microbiology  False   \n",
       "33                               Neurology  False   \n",
       "34                                 Ecology  False   \n",
       "35                    Evolutionary Biology  False   \n",
       "36                            Neuroscience  False   \n",
       "37                    Evolutionary Biology  False   \n",
       "38                          Bioinformatics  False   \n",
       "39   Infectious Diseases (except HIV/AIDS)   True   \n",
       "40                            Microbiology  False   \n",
       "41                                 Ecology  False   \n",
       "42                            Cell Biology  False   \n",
       "43                    Evolutionary Biology  False   \n",
       "44                            Neuroscience  False   \n",
       "45                                 Ecology  False   \n",
       "46                           Plant Biology  False   \n",
       "47                        Gastroenterology   True   \n",
       "48                              Immunology  False   \n",
       "49                                 Ecology  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Children with SARS-CoV-2 in the National COVID...   \n",
       "1   Breakthrough Symptomatic COVID-19 Infections L...   \n",
       "2   Identification of a new family of “megaphages”...   \n",
       "3   Regulation of sedimentation rate shapes the ev...   \n",
       "4   AlphaPept, a modern and open framework for MS-...   \n",
       "5   Delineating Medical Education: Bibliometric Re...   \n",
       "6   Identifying cell-state associated alternative ...   \n",
       "7   Novel risk factors for Coronavirus disease-ass...   \n",
       "8   The impact of large mobile air purifiers on ae...   \n",
       "9   Mechanosensitivity of nucleocytoplasmic transport   \n",
       "10  Strong and opponent contributions of dorsomedi...   \n",
       "11  Epithelial-mesenchymal plasticity determines e...   \n",
       "12  Probiotic consortia improve anti-viral immunit...   \n",
       "13  A bacterial dynamin-like protein confers a nov...   \n",
       "14  Disruption of the TCA cycle reveals an ATF4-de...   \n",
       "15  Protection afforded by the BNT162b2 and mRNA-1...   \n",
       "16  Retrosplenial cortex is necessary for spatial ...   \n",
       "17  Extracellular mechanical forces drive endocard...   \n",
       "18  Synaptic Mechanisms of Top-Down Control by The...   \n",
       "19  ABlooper: Fast accurate antibody CDR loop stru...   \n",
       "20  Vaccine effectiveness when combining the ChAdO...   \n",
       "21  Comparison of infected and vaccinated transpla...   \n",
       "22  Oncogenic c-Myc induces replication stress by ...   \n",
       "23  The dual nature of bacteriophage: growth-depen...   \n",
       "24  A yeast lysosomal biogenesis map uncovers the ...   \n",
       "25  Reward-dependent selection of feedback gains i...   \n",
       "26  LAMP2A regulates the loading of proteins into ...   \n",
       "27  Multivariate mining of an alpaca immune repert...   \n",
       "28  Crowding-induced phase separation of nuclear t...   \n",
       "29  Leveraging gene co-regulation to identify gene...   \n",
       "30  Willingness to receive vaccination against COV...   \n",
       "31  Role of non-specific interactions in the phase...   \n",
       "32  Adapted protocol for Saccharibacteria co-culti...   \n",
       "33  Altered functional connectome hierarchy with g...   \n",
       "34  Bats host the most virulent—but not the most d...   \n",
       "35  Identification of a divergent basal body assem...   \n",
       "36  Trial-wise exposure to visual appetitive cues ...   \n",
       "37  The effective family size of immigrant founder...   \n",
       "38  MetaBinner: a high-performance and stand-alone...   \n",
       "39  Robust antibody levels in both diabetic and no...   \n",
       "40  Global protein responses of multi-drug resista...   \n",
       "41  The benefits of merging two tracking approache...   \n",
       "42  AGO1 regulates major satellite transcripts and...   \n",
       "43  Generalizing Bayesian phylogenetics to infer s...   \n",
       "44  Inhibition of striatal SEZ6 by miR-3594-5p is ...   \n",
       "45  Trait-trait relationships and functional trade...   \n",
       "46  Supra-physiological levels of gibberellins/DEL...   \n",
       "47  Proton pump inhibitor use is not associated wi...   \n",
       "48  Heat-inactivated modified vaccinia virus Ankar...   \n",
       "49  Community health and human-animal contacts on ...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Blake Martin'}, {'name': 'Peter E. ...   \n",
       "1   [{'name': 'Daisy Massey'}, {'name': 'Diana Ber...   \n",
       "2   [{'name': 'Slawomir Michniewski'}, {'name': 'B...   \n",
       "3   [{'name': 'Omaya Dudin'}, {'name': 'Sébastien ...   \n",
       "4   [{'name': 'Maximilian T. Strauss'}, {'name': '...   \n",
       "5   [{'name': 'Lauren A. Maggio'}, {'name': 'Anton...   \n",
       "6   [{'name': 'Carlos F. Buen Abad Najar'}, {'name...   \n",
       "7   [{'name': 'Umang Arora'}, {'name': 'Megha Priy...   \n",
       "8   [{'name': 'F. F. Duill'}, {'name': 'F. Schulz'...   \n",
       "9   [{'name': 'Ion Andreu'}, {'name': 'Ignasi Gran...   \n",
       "10  [{'name': 'Scott S. Bolkan'}, {'name': 'Iris R...   \n",
       "11  [{'name': 'Patrick Aouad'}, {'name': 'Yueyun Z...   \n",
       "12  [{'name': 'J Lehtinen Markus'}, {'name': 'Kuma...   \n",
       "13  [{'name': 'Lijun Guo'}, {'name': 'Laura Sattle...   \n",
       "14  [{'name': 'Dylan Gerard Ryan'}, {'name': 'Ming...   \n",
       "15  [{'name': 'Laith J. Abu-Raddad'}, {'name': 'Hi...   \n",
       "16  [{'name': 'Ana Carolina Bottura de Barros'}, {...   \n",
       "17  [{'name': 'Julien Vermot'}, {'name': 'Helene V...   \n",
       "18  [{'name': 'Hannah M. Oberle'}, {'name': 'Alex ...   \n",
       "19  [{'name': 'Brennan Abanades'}, {'name': 'Guy G...   \n",
       "20  [{'name': 'Mie Agermose Gram'}, {'name': 'Hann...   \n",
       "21  [{'name': 'Xavier Charmetant'}, {'name': 'Maxi...   \n",
       "22  [{'name': 'Silvia Peripolli'}, {'name': 'Tanya...   \n",
       "23  [{'name': 'Quentin J Leclerc'}, {'name': 'Jaco...   \n",
       "24  [{'name': 'Sebastian Eising'}, {'name': 'Bianc...   \n",
       "25  [{'name': 'Antoine De Comite'}, {'name': 'Fréd...   \n",
       "26  [{'name': 'João Vasco Ferreira'}, {'name': 'An...   \n",
       "27  [{'name': 'Leo Hanke'}, {'name': 'Daniel J. Sh...   \n",
       "28  [{'name': 'Luke K. Davis'}, {'name': 'Ian J. F...   \n",
       "29  [{'name': 'Katherine M. Siewert-Rocks'}, {'nam...   \n",
       "30  [{'name': 'Sharon R Davis'}, {'name': 'Rosario...   \n",
       "31  [{'name': 'Rakesh Krishnan'}, {'name': 'Srivas...   \n",
       "32  [{'name': 'Ahmad Ibrahim'}, {'name': 'Mohamad ...   \n",
       "33  [{'name': 'Christophe E de Bézenac'}, {'name':...   \n",
       "34  [{'name': 'Sarah Guth'}, {'name': 'Nardus Moll...   \n",
       "35  [{'name': 'Shizuka Koshimizu'}, {'name': 'Naok...   \n",
       "36  [{'name': 'Kilian Knauth'}, {'name': 'Jan Pete...   \n",
       "37  [{'name': 'Tommy Harding'}, {'name': 'Emmanuel...   \n",
       "38  [{'name': 'Ziye Wang'}, {'name': 'Pingqin Huan...   \n",
       "39  [{'name': 'Hamad Ali'}, {'name': 'Abdelmohsen ...   \n",
       "40  [{'name': 'Anatte Margalit'}, {'name': 'James ...   \n",
       "41  [{'name': 'Louise Chavarie'}, {'name': 'Hannel...   \n",
       "42  [{'name': 'Madlen Müller'}, {'name': 'Tara Fäh...   \n",
       "43  [{'name': 'Jamie R. Oaks'}, {'name': 'Perry L....   \n",
       "44  [{'name': 'Magalie Lenoir'}, {'name': 'Isabell...   \n",
       "45  [{'name': 'Sara Beier'}, {'name': 'Johannes We...   \n",
       "46                  [{'name': 'Iva McCarthy-Suárez'}]   \n",
       "47  [{'name': 'Shailja C. Shah'}, {'name': 'Alese ...   \n",
       "48  [{'name': 'Ning Yang'}, {'name': 'Aitor Garzia...   \n",
       "49  [{'name': 'Renata L. Muylaert'}, {'name': 'Ben...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <sec><title>Importance</title><p>SARS-CoV-2</p...  2021-07-22  \n",
       "1   <p>Vaccines have been shown to be extremely ef...  2021-07-26  \n",
       "2   <p>Megaphages – bacteriophages harbouring extr...  2021-07-26  \n",
       "3   <p>Significant increases in sedimentation rate...  2021-07-23  \n",
       "4   <p>In common with other omics technologies, ma...  2021-07-26  \n",
       "5   <sec><title>Background</title><p>The field of ...  2021-07-26  \n",
       "6   <p>Alternative splicing shapes the transcripto...  2021-07-24  \n",
       "7   <sec><title>Background</title><p>The epidemiol...  2021-07-26  \n",
       "8   <p>In the wake of the SARS-CoV-2 pandemic, an ...  2021-07-26  \n",
       "9   <p>Mechanical force controls fundamental cellu...  2021-07-24  \n",
       "10  <p>A classic view of the striatum holds that a...  2021-07-25  \n",
       "11  <p>Estrogen receptor α-positive (ER+) breast c...  2021-07-26  \n",
       "12  <p>Probiotics have been suggested as one solut...  2021-07-23  \n",
       "13  <p><italic>Bacillus subtilis</italic> DynA is ...  2021-07-26  \n",
       "14  <p>The Tricarboxylic Acid Cycle (TCA) cycle is...  2021-07-27  \n",
       "15  <p>Effect of prior SARS-CoV-2 infection on vac...  2021-07-26  \n",
       "16  <p>Latent learning occurs when associations ar...  2021-07-23  \n",
       "17  <p>Organ morphogenesis involves dynamic change...  2021-07-27  \n",
       "18  <p>Corticofugal projections to evolutionarily ...  2021-07-26  \n",
       "19  <p>Antibodies are a key component of the immun...  2021-07-26  \n",
       "20  <p>Background: The recommendations in several ...  2021-07-28  \n",
       "21  <p>Transplant recipients, which receive therap...  2021-07-24  \n",
       "22  <p>Oncogene-induced replication stress is a ma...  2021-07-26  \n",
       "23  <p>Bacteriophage (“phage”) are both predators ...  2021-07-24  \n",
       "24  <p>The lysosome is the major catabolic organel...  2021-07-25  \n",
       "25  <p>Expected reward is known to affect planning...  2021-07-26  \n",
       "26  <p>Exosomes are extracellular vesicles of endo...  2021-07-26  \n",
       "27  <p>Conventional approaches to isolate and char...  2021-07-26  \n",
       "28  <p>The rapid (&lt; 1 ms) transport of biologic...  2021-07-26  \n",
       "29  <p>Identifying gene sets that are associated t...  2021-07-26  \n",
       "30  <p>We surveyed 10,024 Australians regarding CO...  2021-07-25  \n",
       "31  <p>Phase separation of biomolecules could be m...  2021-07-26  \n",
       "32  <p>The growing application of metagenomics to ...  2021-07-25  \n",
       "33  <sec><title>Objective</title><p>Neuroimaging r...  2021-07-22  \n",
       "34  <p>Identifying virus characteristics associate...  2021-07-26  \n",
       "35  <p>Oogamy is a form of sexual reproduction and...  2021-07-26  \n",
       "36  <p>Humans and many animals devalue future rewa...  2021-07-26  \n",
       "37  <p>Human evolution involves population splits,...  2021-07-27  \n",
       "38  <p>Binning is an essential procedure during me...  2021-07-26  \n",
       "39  <p>The emergence of effective vaccines for COV...  2021-07-26  \n",
       "40  <p>Antimicrobial resistance (AMR) and multi-dr...  2021-07-26  \n",
       "41  <p>The process of smolting is a critical phase...  2021-07-26  \n",
       "42  <p>In the past years, several studies reported...  2021-07-26  \n",
       "43  <p>Many processes of biological diversificatio...  2021-07-26  \n",
       "44  <p>Escalation of drug use is a hallmark stage ...  2021-07-26  \n",
       "45  <p>We report genomic traits that have been ass...  2021-07-24  \n",
       "46  <p>A previous study (McCarthy-Suárez, 2021) sh...  2021-07-26  \n",
       "47  <p>Background and Aims: Low pH deactivates mos...  2021-07-25  \n",
       "48  <sec><title>Background</title><p>Protein or pe...  2021-07-26  \n",
       "49  <p>Cross-species transmission of pathogens is ...  2021-07-23  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "112 preprints (including 20 covering SARS-CoV-2, 89 from bioRxiv and 23 from medRxiv) published in the last 7 days before 2021-07-29 had been tweeted at least 3 times (maximum 625)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.5"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
