{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "19733 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=40325a6e-33a1-40f9-860f-8d68ed677db3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c09f462f-8a74-4321-91fd-2114f28fb574&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=fb16870a-d71e-4592-ad1e-376015e23f38&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5da06bd6-700d-43d0-9c80-4ba0ba91505d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7f1a5d75-df0b-4f42-a9e5-277b8e580c57&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=30193eb1-efea-4358-9eaa-31b268a5f252&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2aa17415-f6e8-4551-8511-ee1604144143&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=814f3e4a-ecb2-47fa-b681-2694542cd8d7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=583ca4b5-42fa-45ef-80e8-e6a7bfef261b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f885c6c0-a44a-4143-b7d5-c66f61b94843&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=12f162ab-5f7f-4cb6-a943-9b36d8547a12&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=859c7978-8216-480c-912e-298f4cc49aad&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=e23df112-f1ac-4531-afa4-e3ad53b60c71&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=341295a4-040f-4e7b-a8a2-52bdbfb04eb0&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=1b387b7f-2a06-492c-9df4-837f2344567a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=16acb960-84d5-4bd7-878c-da62bda61b4c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=790578d8-be5b-43b8-8b3a-57b1bed6f9c8&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=42ff4e9e-07fd-45bf-ba61-eea5277b889c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0018.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9148ac43-f3ca-4952-8231-f17269de5cb2&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-11-15&until-occurred-date=2021-11-22\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0019.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-10-18.csv\n",
      "failed to load preprint_tweets_2021-10-25.csv\n",
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load preprint_tweets_2021-10-23.csv\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-11-01.csv\n",
      "failed to load preprint_tweets_2021-11-15.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-11-08.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.21264626...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469078...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469088...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468777...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.465477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468476...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468676...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21265758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.468202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468761...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468512...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.468258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21266187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.10.986059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.420448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275752.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.06.981365...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273300.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/428318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.463794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.27.175067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/240317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468770...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.21265383...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.468884...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.21266552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.467545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.28.923532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275323.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.09.467796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21266245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468719...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275569.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.06.467573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.459575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.467908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467405...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469020...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468694...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.21265959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21263608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.367649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.413609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468056...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21253893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.10.20021675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21258385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468709...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468963...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468669...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.448737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.09.21265082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.467523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.31.928630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.451328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468503...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.11.21249258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467793...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460480...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/710624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.264705.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21253075...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21265571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.468228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468749...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468904...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468603...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.466891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.21264567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468929...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.449462...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469412...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.163816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.465712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266221...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.440511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469104...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/530097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.05.20088757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20213546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468372...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.443815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.465905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.21266408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.21265767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468953...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275989.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469306...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/238527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.382143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.467162...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442486...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.08.139964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469056...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266322...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21266178...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.457527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.452858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.11.451967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.21261741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.413229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.28.065714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468572...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.225854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21266249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.467006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469119...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.20156018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.09.467870...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21266064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.467798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.379750...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21266262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.426317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.436616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.28.21262763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446238...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.441495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.436351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.21266115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.20086439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.21266289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266330...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21266325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469435...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468024...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.439071...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.09.467923...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469099...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428071...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.21266074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469149...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275325.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.21250840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468928...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.467047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/623959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.03.21265876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.174888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.25.394544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.433258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454246...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.430068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468957...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.20.20108183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.467373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.13.20174078...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.17.20069716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469022...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468558...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441655...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.945956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.467680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.261115.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.21255913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.26.424423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275308.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.462574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435518...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.444465...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21263008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469035...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468986...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.20142703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.21266456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467773...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.09.467989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.459429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275503.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.21265779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.09.467878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21258418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.24.167353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.461806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.31.116061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.459925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.467517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.467977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465343...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.462716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/801654...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.460970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469458...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.462501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.465336...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.02.21261379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.467353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21266481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.467812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469324...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21266286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21253685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21266338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.21266362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.433782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.235549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.467961...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.21255690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.20204693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.389031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.444999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446191...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.13.468422...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.21265797...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.20.20178533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.30.272237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.29.318881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.21266571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.21266277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.467495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.438036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469109...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.11.20229708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469073...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20198796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.468912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.21.436284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469075...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459618...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.01.466797...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.17.386995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.388819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.19.20054262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.21250420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/760819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.21265703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/768663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461293...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.385401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.444531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.20248876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.21266374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.464426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.464095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.19.423614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.464715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.335083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.464515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.440450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.02.21249146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.441312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.21263548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.29.20248953...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.21266532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.21264595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21256632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.24.20042937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.464338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.14.468520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.21249603...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.15.468665...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.460746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.08.982637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.443021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264191...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21265988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/462853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/276717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469337...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.466718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.21266051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.153361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.03.20167395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.11.21266119...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.24.20133249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.02.454825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.05.21265656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.12.468394...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.433891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.14.21251715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.454924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.12.21258794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.21258569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.389957...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450899...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.18.469049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.21260277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.09.467917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.21266197...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.469067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.451243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.09.467972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468841...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.469222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.21.21251403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.21.469446...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.21.20236018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.10.468074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.08.467712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21255526...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.16.468878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.20.468818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.19.469265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.04.21265945...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.17.468905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.11.02.466829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.11.13.468472', 'tweets': 502, 'archive': 'bioRxiv', 'subject-area': 'Pathology', 'covid': True, 'title': 'A live attenuated influenza virus-vectored intranasal COVID-19 vaccine provides rapid, prolonged, and broad protection against SARS-CoV-2 infection', 'authors': \"[{'name': 'Junyu Chen'}, {'name': 'Pui Wang'}, {'name': 'Lunzhi Yuan'}, {'name': 'Liang Zhang'}, {'name': 'Limin Zhang'}, {'name': 'Hui Zhao'}, {'name': 'Congjie Chen'}, {'name': 'Yaode Chen'}, {'name': 'Jinle Han'}, {'name': 'Jizong Jia'}, {'name': 'Zhen Lu'}, {'name': 'Junping Hong'}, {'name': 'Liqiang Chen'}, {'name': 'Changfa Fan'}, {'name': 'Zicen Lu'}, {'name': 'Qian Wang'}, {'name': 'Rirong Chen'}, {'name': 'Minping Cai'}, {'name': 'Ruoyao Qi'}, {'name': 'Xijing Wang'}, {'name': 'Jian Ma'}, {'name': 'Min Zhou'}, {'name': 'Huan Yu'}, {'name': 'Chunlan Zhuang'}, {'name': 'Xiaohui Liu'}, {'name': 'Qiangyuan Han'}, {'name': 'Guosong Wang'}, {'name': 'Yingying Su'}, {'name': 'Quan Yuan'}, {'name': 'Tong Cheng'}, {'name': 'Ting Wu'}, {'name': 'Xiangzhong Ye'}, {'name': 'Changgui Li'}, {'name': 'Tianying Zhang'}, {'name': 'Jun Zhang'}, {'name': 'Huachen Zhu'}, {'name': 'Yixin Chen'}, {'name': 'Honglin Chen'}, {'name': 'Ningshao Xia'}]\", 'abstract': '<p>Remarkable progress has been made in developing intramuscular vaccines against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2); however, they are limited with respect to eliciting local immunity in the respiratory tract, which is the primary infection site for SARS-CoV-2. To overcome the limitations of intramuscular vaccines, we constructed a nasal vaccine candidate based on an influenza vector by inserting a gene encoding the receptor-binding domain (RBD) of the spike protein of SARS-CoV-2, named CA4-dNS1-nCoV-RBD (dNS1-RBD). A preclinical study showed that in hamsters challenged 1 day and 7 days after single-dose vaccination or 6 months after booster vaccination, dNS1-RBD largely mitigated lung pathology, with no loss of body weight, caused by either the prototype-like strain or beta variant of SARS-CoV-2. Lasted data showed that the animals could be well protected against beta variant challenge 9 months after vaccination. Notably, the weight loss and lung pathological changes of hamsters could still be significantly reduced when the hamster was vaccinated 24 h after challenge. Moreover, such cellular immunity is relatively unimpaired for the most concerning SARS-CoV-2 variants. The protective immune mechanism of dNS1-RBD could be attributed to the innate immune response in the nasal epithelium, local RBD-specific T cell response in the lung, and RBD-specific IgA and IgG response. Thus, this study demonstrates that the intranasally delivered dNS1-RBD vaccine candidate may offer an important addition to fight against the ongoing COVID-19 pandemic, compensating limitations of current intramuscular vaccines, particularly at the start of an outbreak.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469078', 'tweets': 145, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Metformin Suppresses SARS-CoV-2 in Cell Culture', 'authors': \"[{'name': 'Dixit Tandel'}, {'name': 'Haripriya Parthasarathy'}, {'name': 'Krishnan Harinivas Harshan'}]\", 'abstract': '<p>People with diabetes are reported to have a higher risk of experiencing severe COVID-19 complications. Metformin, a first-line medication for type 2 diabetes, has antiviral properties. Some studies have indicated its prognostic potential in COVID-19. Here, we report that metformin significantly inhibits SARS-CoV-2 growth in cell culture models. SARS-CoV-2 infection of gut epithelial cell line, Caco2, resulted in higher phosphorylation of AMPK. Metformin reduced viral titers in the infected cells by nearly 99%, and by about 90% when cells were treated prior to infection. Metformin pre-treatment resulted in further phosphorylation of AMPK and caused a ten-fold reduction of viral titers indicating its potential in preventing naive infections. Confirming the positive impact of AMPK activation, another AMPK activator AICAR substantially inhibited of viral titers and, AMPK inhibitor Compound C, augmented it considerably. Metformin treatment post-SARS-CoV-2 infection resulted in nearly hundred-fold reduction of viral titers, indicating that the antiviral potency of the drug is far higher in infected cells, while still being able to reduce fresh infection. Metformin displayed SARS-CoV-2 TCID50 and TCID90 at 3.5 and 8.9 mM, respectively. In conclusion, our study demonstrates that metformin is very effective in limiting the replication of SARS-CoV-2 in cell culture and thus possibly could offer double benefits to diabetic COVID-19 patients by lowering both blood glucose levels and viral load.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469088', 'tweets': 117, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structural basis for Cas9 off-target activity', 'authors': \"[{'name': 'Martin Pacesa'}, {'name': 'Chun-Han Lin'}, {'name': 'Antoine Cléry'}, {'name': 'Katja Bargsten'}, {'name': 'Matthew J. Irby'}, {'name': 'Frédéric H.T. Allain'}, {'name': 'Peter Cameron'}, {'name': 'Paul D. Donohoue'}, {'name': 'Martin Jinek'}]\", 'abstract': '<p>The target DNA specificity of the CRISPR-associated genome editor nuclease Cas9 is determined by complementarity to a 20-nucleotide segment in its guide RNA. However, Cas9 can bind and cleave partially complementary off-target sequences, which raises safety concerns for its use in clinical applications. Here we report crystallographic structures of Cas9 bound to <italic>bona fide</italic> off-target substrates, revealing that off-target binding is enabled by a range of non-canonical base pairing interactions and preservation of base stacking within the guide–off-target heteroduplex. Off-target sites containing single-nucleotide deletions relative to the guide RNA are accommodated by base skipping rather than RNA bulge formation. Additionally, PAM-distal mismatches result in duplex unpairing and induce a conformational change of the Cas9 REC lobe that perturbs its conformational activation. Together, these insights provide a structural rationale for the off-target activity of Cas9 and contribute to the improved rational design of guide RNAs and off-target prediction algorithms.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.19.469228', 'tweets': 107, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Pre-metazoan origin of neuropeptide signalling', 'authors': \"[{'name': 'Luis Alfonso Yanez-Guerra'}, {'name': 'Daniel Thiel'}, {'name': 'Gaspar Jekely'}]\", 'abstract': '<p>Neuropeptides are a diverse class of signalling molecules in metazoans. They occur in all animals with a nervous system and also in neuron-less placozoans. However, their origin has remained unclear because no neuropeptide shows deep homology across lineages and none have been found in sponges. Here, we identify two neuropeptide precursors, phoenixin and nesfatin, with broad evolutionary conservation. By database searches, sequence alignments and gene-structure comparisons we show that both precursors are present in bilaterians, cnidarians, ctenophores and sponges. We also found phoenixin and a secreted nesfatin precursor homolog in the choanoflagellate Salpingoeca rosetta. Phoenixin in particular, is highly conserved, including its cleavage sites, suggesting that prohormone processing occurs also in choanoflagellates. In addition, based on phyletic patterns and negative pharmacological assays we question the originally proposed GPR-173 (SREB3) as a phoenixin receptor. Our findings indicate that signalling by secreted neuropeptide homologs has pre-metazoan origins and thus evolved before neurons.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.16.468777', 'tweets': 99, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'The Delta variant of SARS-CoV-2 maintains high sensitivity to interferons in human lung cells', 'authors': \"[{'name': 'Rayhane Nchioua'}, {'name': 'Annika Schundner'}, {'name': 'Susanne Klute'}, {'name': 'Sabrina Noettger'}, {'name': 'Fabian Zech'}, {'name': 'Lennart Koepke'}, {'name': 'Alexander Graf'}, {'name': 'Stefan Krebs'}, {'name': 'Helmut Blum'}, {'name': 'Dorota Kmiec'}, {'name': 'Manfred Frick'}, {'name': 'Frank Kirchhoff'}, {'name': 'Konstantin M.J. Sparrer'}]\", 'abstract': '<p>Interferons are a major part of the anti-viral innate defense system. Successful pathogens, including the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), need to overcome these defenses to establish an infection. Early induction of interferons (IFNs) protects against severe coronavirus disease 2019 (COVID-19). In line with this, SARS-CoV-2 is inhibited by IFNs <italic>in vitro</italic>, and IFN-based therapies against COVID-19 are investigated in clinical trials. However, SARS-CoV-2 continues to adapt to the human population resulting in the emergence of variants characterized by increased transmission fitness and/or decreased sensitivity to preventive or therapeutic measures. It has been suggested that the efficient spread of these so-called “Variants of Concern” (VOCs) may also involve reduced sensitivity to IFNs. Here, we examined whether the four current VOCs (Alpha, Beta, Gamma and Delta) differ in replication efficiency or IFN sensitivity from an early isolate of SARS-CoV-2. All viruses replicated in a human lung cell line and in iPSC-derived alveolar type II cells (iAT2). The Delta variant showed accelerated replication kinetics and higher infectious virus production compared to the early 2020 isolate. Replication of all SARS-CoV-2 VOCs was reduced in the presence of exogenous type I, II and III IFNs. On average, the Alpha variant was the least susceptible to IFNs and the Alpha, Beta and Gamma variants show increased resistance against type III IFN. Although the Delta variant has outcompeted all other variants in humans it remained as sensitive to IFNs as an early 2020 SARS-CoV-2 isolate. This suggests that increased replication fitness rather than IFN resistance may be a reason for its dominance. Our results may help to understand changes in innate immune susceptibility of VOCs, and inform clinical trials exploring IFN-based COVID-19 therapies.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.15.468391', 'tweets': 97, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'An atypical NLR protein modulates the NRC immune receptor network', 'authors': \"[{'name': 'Hiroaki Adachi'}, {'name': 'Toshiyuki Sakai'}, {'name': 'Adeline Harant'}, {'name': 'Cian Duggan'}, {'name': 'Tolga O Bozkurt'}, {'name': 'Chih-hang Wu'}, {'name': 'Sophien Kamoun'}]\", 'abstract': '<p>The NRC immune receptor network has evolved in asterid plants from a pair of linked genes into a genetically dispersed and phylogenetically structured network of sensor and helper NLR (nucleotide-binding domain and leucine-rich repeat-containing) proteins. In some species, such as the model plant <italic>Nicotiana benthamiana</italic> and other Solanaceae, the NRC network forms up to half of the NLRome, and NRCs are scattered throughout the genome in gene clusters of varying complexities. Here, we describe NRCX, an atypical, but essential member of the NRC family that lacks canonical features of these NLR helper proteins, such as a functional N-terminal MADA motif and the capacity to trigger autoimmunity. In contrast to other NRCs, systemic gene silencing of <italic>NRCX</italic> markedly impairs plant growth resulting in a dwarf phenotype. Remarkably, dwarfism of <italic>NRCX</italic> silenced plants is partially dependent on NRCX paralogs NRC2 and NRC3, but not NRC4. Despite its negative impact on plant growth when silenced systemically, transient RNA interference of <italic>NRCX</italic> in mature <italic>N. benthamiana</italic> leaves doesn’t result in visible cell death phenotypes. However, alteration of <italic>NRCX</italic> expression modulates the hypersensitive response mediated by NRC2 and NRC3 in a manner consistent with a negative role for NRCX in the NRC network. We conclude that NRCX is an atypical member of the NRC network that has evolved to contribute to the homeostasis of this genetically unlinked NLR network.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.14.21266295', 'tweets': 93, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Estimation of R0 for the spread of SARS-CoV-2 in Germany from Excess Mortality', 'authors': \"[{'name': 'Juan Pablo Prada'}, {'name': 'Luca Estelle Maag'}, {'name': 'Laura Siegmund'}, {'name': 'Elena Bencurova'}, {'name': 'Liang Chunguang'}, {'name': 'Eleni Koutsilieri'}, {'name': 'Thomas Dandekar'}, {'name': 'Carsten Scheller'}]\", 'abstract': '<p>For SARS-CoV-2, R0 calculations report usually 2-3, biased by PCR testing increases. Covid-19-induced excess mortality is less biased.</p><p>We used data from Robert Koch Institute on Covid incidence, deaths, and PCR tests and excess mortality to determine early, policy-free R0 estimates with a serial interval of 4.7 days.</p><p>The PCR-based R0 value was 2.56 (95% CI 2.52-2.60) for Covid-19 cases and 2.03 (95%CI 1.96-2.10) for Covid-19-related deaths. As the number of PCR tests increased, R0 values were corrected accordingly, yielding 1.86 for Covid-19 cases and 1.47 for Covid-19 deaths, excess deaths were 1.34 (95% CI 1.32-1.37).</p><p>R0 is much lower than previously thought. This fits the observed seasonal pattern of infection across Europe in 2020-2021, including emergence of more contagious escape variants such as delta.</p><sec><title>One-Sentence Summary</title><p>Excess mortality reveals infection speed in Covid-19 is surprisingly low with seasonal infection patterns and escape variants.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.17.465477', 'tweets': 84, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'kTMP: A New Non-invasive Magnetic Induction Method to Modulate Cortical Excitability', 'authors': \"[{'name': 'Ludovica Labruna'}, {'name': 'Christina Merrick'}, {'name': 'Ben Inglis'}, {'name': 'Richard Ivry'}, {'name': 'Daniel Sheltraw'}]\", 'abstract': '<p>We report a novel subthreshold non-invasive brain stimulation approach that we refer to as kilohertz transcranial magnetic perturbation, or kTMP. kTMP is a magnetic induction method that delivers kHz-frequency cortical E-fields and, through amplitude modulation of the kHz carrier frequency, may mimic E-fields at physiological frequencies. To evaluate the efficacy of kTMP, we used suprathreshold TMS to elicit motor-evoked potentials (MEP) in a peripheral muscle, comparing the amplitude of the MEPs before and after kTMP stimulation. In Experiment 1, we used non-modulated kTMP with an E-field amplitude of 2 V/m over motor cortex. Ten minutes of kTMP stimulation resulted in an increase in cortical excitability in a frequency-specific manner. We replicated this effect in Experiment 2 and found that amplitude-modulation at 20 Hz produced an additional boost in cortical excitability. The only percept associated with kTMP is a faint auditory tone, making kTMP ideal for double-blind experimentation.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469090', 'tweets': 82, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Mechanics of cell integration in vivo', 'authors': \"[{'name': 'Guilherme B. Ventura'}, {'name': 'Aboutaleb Amiri'}, {'name': 'Raghavan Thiagarajan'}, {'name': 'Mari Tolonen'}, {'name': 'Amin Doostmohammadi'}, {'name': 'Jakub Sedzinski'}]\", 'abstract': '<p>During embryonic development, regeneration and homeostasis, cells have to physically integrate into their target tissues, where they ultimately execute their function. Despite a significant body of research on how mechanical forces instruct cellular behaviors within the plane of an epithelium, very little is known about the mechanical interplay at the interface between migrating cells and their surrounding tissue, which has its own dynamics, architecture and identity. Here, using quantitative in vivo imaging and molecular perturbations, together with a theoretical model, we reveal that multiciliated cell (MCC) precursors in the Xenopus embryo form dynamic filopodia that pull at the vertices of the overlying epithelial sheet to probe their stiffness and identify the preferred positions for their integration into the tissue. Moreover, we report a novel function for a structural component of vertices, the lipolysis-stimulated lipoprotein receptor (LSR), in filopodia dynamics and show its critical role in cell intercalation. Remarkably, we find that pulling forces equip the MCCs to remodel the epithelial junctions of the neighboring tissue, enabling them to generate a permissive environment for their integration. Our findings reveal the intricate physical crosstalk at the cell-tissue interface and uncover previously unknown functions for mechanical forces in orchestrating cell integration.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.16.468622', 'tweets': 80, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'The evolution of genomic, transcriptomic, and single-cell protein markers of metastatic upper tract urothelial carcinoma', 'authors': \"[{'name': 'Kentaro Ohara'}, {'name': 'Andrè Figueiredo Rendeiro'}, {'name': 'Bhavneet Bhinder'}, {'name': 'Kenneth Wha Eng'}, {'name': 'Hiranmayi Ravichandran'}, {'name': 'David Pisapia'}, {'name': 'Aram Vosoughi'}, {'name': 'Evan Fernandez'}, {'name': 'Kyrillus Shohdy'}, {'name': 'Jyothi Manohar'}, {'name': 'Shaham Beg'}, {'name': 'David Wilkes'}, {'name': 'Brian Robinson'}, {'name': 'Francesca Khani'}, {'name': 'Rohan Bareja'}, {'name': 'Scott Tagawa'}, {'name': 'Andrea Sboner'}, {'name': 'Olivier Elemento'}, {'name': 'Bishoy Morris Faltas'}, {'name': 'Juan Miguel Mosquera'}]\", 'abstract': '<p>The molecular characteristics of metastatic upper tract urothelial carcinoma (UTUC) are unknown. The genomic and transcriptomic differences between primary and metastatic UTUC is not well described either. We combined whole-exome sequencing, RNA-sequencing, and Imaging Mass Cytometry<sup>TM</sup> (IMC<sup>TM</sup>) of 44 tumor samples from 28 patients with high-grade primary and metastatic UTUC. IMC enables spatially resolved single-cell analyses to examine the evolution of cancer cell, immune cell, and stromal cell markers using mass cytometry with lanthanide metal-conjugated antibodies. We discovered that actionable genomic alterations are frequently discordant between primary and metastatic UTUC tumors in the same patient. In contrast, molecular subtype membership and immune depletion signature were stable across primary and matched metastatic UTUC. Molecular and immune subtypes were consistent between bulk RNA-sequencing and mass cytometry of protein markers from 340,798 single-cells. Molecular subtyping at the single cell level was highly conserved between primary and metastatic UTUC tumors within the same patient.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468999', 'tweets': 79, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Reversible bacteriophage resistance by shedding the bacterial cell wall', 'authors': \"[{'name': 'Veronique Ongenae'}, {'name': 'Adam Sidi Mabrouk'}, {'name': 'Marjolein Crooijmans'}, {'name': 'Daniel Rozen'}, {'name': 'Ariane Briegel'}, {'name': 'Dennis Claessen'}]\", 'abstract': '<p>Phages are highly abundant in the environment and a major threat for bacteria. Therefore, bacteria have evolved sophisticated defense systems to withstand phage attacks. Here, we describe a previously unknown mechanism by which mono- and diderm bacteria survive infection with diverse lytic phages. Phage exposure leads to a rapid and near complete conversion of walled cells to a cell wall-deficient state, which remain viable in osmoprotective conditions and can revert to the walled state. While shedding the cell wall dramatically reduces the number of progeny phages produced by the host, it does not always preclude phage infection. Altogether, these results show that the formation of cell wall-deficient cells prevents complete eradication of the bacterial population and suggest that cell wall-deficiency may limit the efficacy of phage therapy, especially in highly osmotic environments or when used together with antibiotics that target the cell wall.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.15.468686', 'tweets': 71, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Evaluation of methods for the inference of ancestral recombination graphs', 'authors': \"[{'name': 'Débora Y. C. Brandt'}, {'name': 'Xinzhu Wei'}, {'name': 'Yun Deng'}, {'name': 'Andrew H. Vaughn'}, {'name': 'Rasmus Nielsen'}]\", 'abstract': '<p>The ancestral recombination graph (ARG) is a structure that describes the joint genealogies of sampled DNA sequences along the genome. Recent computational methods have made impressive progress towards scalably estimating whole-genome genealogies. In addition to inferring the ARG, some of these methods can also provide ARGs sampled from a defined posterior distribution. Obtaining good samples of ARGs is crucial for quantifying statistical uncertainty and for estimating population genetic parameters such as effective population size, mutation rate, and allele age. Here, we use simulations to benchmark three popular ARG inference programs: ARGweaver, Relate, and tsdate. We use neutral coalescent simulations to 1) compare the true coalescence times to the inferred times at each locus; 2) compare the distribution of coalescence times across all loci to the expected exponential distribution; 3) evaluate whether the sampled coalescence times have the properties expected of a valid posterior distribution. We find that inferred coalescence times at each locus are more accurate in ARGweaver and Relate than in tsdate. However, all three methods tend to overestimate small coalescence times and underestimate large ones. Lastly, the posterior distribution of ARGweaver is closer to the expected posterior distribution than Relate’s, but this higher accuracy comes at a substantial trade-off in scalability. The best choice of method will depend on the number and length of input sequences and on the goal of downstream analyses, and we provide guidelines for the best practices.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.18.469106', 'tweets': 71, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Transgenic tools for targeted chromosome rearrangements allow construction of balancer chromosomes in non-melanogaster Drosophila species', 'authors': \"[{'name': 'David L. Stern'}]\", 'abstract': '<p>Perhaps the most valuable single set of resources for genetic studies of <italic>Drosophila melanogaster</italic> is the collection of multiply-inverted chromosomes commonly known as balancer chromosomes. Balancers prevent the recovery of recombination exchange products within genomic regions included in inversions and allow perpetual maintenance of deleterious alleles in living stocks and the execution of complex genetic crosses. Balancer chromosomes have been generated traditionally by exposing animals to ionizing radiation and screening for altered chromosome structure or for unusual marker segregation patterns. These approaches are tedious and unpredictable, and have failed to produce the desired products in some species. Here I describe transgenic tools that allow targeted chromosome rearrangements in <italic>Drosophila</italic> species. The key new resources are engineered reporter genes containing introns with yeast recombination sites and enhancers that drive fluorescent reporter genes in multiple body regions. These tools were used to generate a doubly-inverted chromosome 3R in <italic>D. simulans</italic> that serves as an effective balancer chromosome.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.15.468676', 'tweets': 70, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Built on sand: the shaky foundations of simulating single-cell RNA sequencing data', 'authors': \"[{'name': 'Helena L. Crowell'}, {'name': 'Sarah X. Morillo Leonardo'}, {'name': 'Charlotte Soneson'}, {'name': 'Mark D. Robinson'}]\", 'abstract': '<p>With the emergence of hundreds of single-cell RNA-sequencing (scRNA-seq) datasets, the number of computational tools to analyse aspects of the generated data has grown rapidly. As a result, there is a recurring need to demonstrate whether newly developed methods are truly performant – on their own as well as in comparison to existing tools. Benchmark studies aim to consolidate the space of available methods for a given task, and often use simulated data that provide a ground truth for evaluations. Thus, demanding a high quality standard for synthetically generated data is critical to make simulation study results credible and transferable to real data.</p><p>Here, we evaluated methods for synthetic scRNA-seq data generation in their ability to mimic experimental data. Besides comparing gene- and cell-level quality control summaries in both one- and two-dimensional settings, we further quantified these at the batch- and cluster-level. Secondly, we investigate the effect of simulators on clustering and batch correction method comparisons, and, thirdly, which and to what extent quality control summaries can capture reference-simulation similarity.</p><p>Our results suggest that most simulators are unable to accommodate complex designs without introducing artificial effects; they yield over-optimistic performance of integration, and potentially unreliable ranking of clustering methods; and, it is generally unknown which summaries are important to ensure effective simulation-based method comparisons.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.14.21265758', 'tweets': 54, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Seroprevalence in Tamil Nadu through India’s two COVID waves: Evidence on antibody decline following infection and vaccination', 'authors': \"[{'name': 'T.S. Selvavinayagam'}, {'name': 'A. Somasundaram'}, {'name': 'Jerard Maria Selvam'}, {'name': 'Sabareesh Ramachandran'}, {'name': 'P. Sampath'}, {'name': 'V. Vijayalakshmi'}, {'name': 'C. Ajith Brabhu Kumar'}, {'name': 'Sudharshini Subramaniam'}, {'name': 'S Raju'}, {'name': 'R Avudaiselvi'}, {'name': 'V. Prakash'}, {'name': 'N. Yogananth'}, {'name': 'Gurunathan Subramanian'}, {'name': 'A Roshini'}, {'name': 'D.N. Dhiliban'}, {'name': 'Sofia Imad'}, {'name': 'Vaidehi Tandel'}, {'name': 'Rajeswari Parasa'}, {'name': 'Stuti Sachdeva'}, {'name': 'Anup Malani'}]\", 'abstract': '<p>Three rounds of population-representative serological studies through India’s two COVID waves (round 1, 19 October-30 November 2020; round 2, 7-30 April 2021; and round 3, 28 June-7 July7, 2021) were conducted at the district-level in Tamil Nadu state (population 72 million). State-level seroprevalence in rounds 1, 2 and 3 were 31.5%, 22.9%, and 67.1%. Estimated seroprevalence implies that at least 22.6 and 48.1 million persons were infected by the 30 November 2020 and 7 July 2021. There was substantial variation across districts in the state in each round. Seroprevalence ranged from 11.1 to 49.8% (round 1), 7.9 to 50.3% (round 2), and 37.8 to 84% (round 3). Seroprevalence in urban areas was higher than in rural areas (35.7 v. 25.7% in round 1, 74.8% v. 64.1% in round 3). Females had similar seroprevalence to males (30.8 v. 30.2% in round 1, 67.5 v. 65.5% in round 3). While working age populations (age 40-49: 31.6%) had significantly higher seroprevalence than the youth (age 18-29: 30.4%) or elderly (age 70+: 26.5%) in round 1, only the gap between working age (age 40-49: 66.7%) and elderly (age 70+: 59.6%) remained significant in round 3. Seroprevalence was greater among those who were vaccinated for COVID (25.7% v. 20.9% in round 2, 80.0% v. 62.3% in round 3). While the decline in seroprevalence from round 1 to 2 suggests antibody decline after natural infection, we do not find significant decline in antibodies among those receiving at least 1 dose of vaccine between rounds 2 and 3.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.19.468202', 'tweets': 53, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Smart soils to observe hidden rhizosphere processes', 'authors': \"[{'name': 'Daniel Patko'}, {'name': 'Qizhi Yang'}, {'name': 'Yangminghao Liu'}, {'name': 'Panagiotis Falireas'}, {'name': 'Benoit Briou'}, {'name': 'Bhausaheb V Tawade'}, {'name': 'Timothy S George'}, {'name': 'Tim J Daniell'}, {'name': 'Michael P MacDonald'}, {'name': 'Vincent Ladmiral'}, {'name': 'Bruno Ameduri'}, {'name': 'Lionel Xavier Dupuy'}]\", 'abstract': '<p>Agriculture must reduce green-house gas emission and pollution, produce safer and healthier food, closer to home, reducing waste whilst delivering more diverse diets to a growing world population. Soils could enable this transformation, but unfortunately, they have a hugely complex and opaque structure and studies of its myriad of mechanisms are difficult. Here, the fabrication of smart soils for the screening of below-ground bio-processes is demonstrated. Particles were generated from fluoropolymer waste with functionalisation converting them into sensors able to report on key chemical dynamics. Tailored functionalization was obtained by radical terpolymerisation to improve growth conditions and sensing capabilities. The study demonstrates the potential for the development of accelerated genetic or agrochemical screens and could pave the way for controlled indoor soil bio-production systems.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468761', 'tweets': 50, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Microglia do not restrict SARS-CoV-2 replication following infection of the central nervous system of K18-hACE2 transgenic mice', 'authors': \"[{'name': 'Gema M. Olivarria'}, {'name': 'Yuting Cheng'}, {'name': 'Susana Furman'}, {'name': 'Collin Pachow'}, {'name': 'Lindsay A. Hohsfield'}, {'name': 'Charlene Smith-Geater'}, {'name': 'Ricardo Miramontes'}, {'name': 'Jie Wu'}, {'name': 'Mara S. Burns'}, {'name': 'Kate I. Tsourmas'}, {'name': 'Jennifer Stocksdale'}, {'name': 'Cynthia Manlapaz'}, {'name': 'William H. Yong'}, {'name': 'John Teijaro'}, {'name': 'Robert Edwards'}, {'name': 'Kim N. Green'}, {'name': 'Leslie M. Thompson'}, {'name': 'Thomas E. Lane'}]\", 'abstract': '<p>Unlike SARS-CoV-1 and MERS-CoV, infection with SARS-CoV-2, the viral pathogen responsible for COVID-19, is often associated with neurologic symptoms that range from mild to severe, yet increasing evidence argues the virus does not exhibit extensive neuroinvasive properties. We demonstrate SARS-CoV-2 can infect and replicate in human iPSC-derived neurons and that infection shows limited anti-viral and inflammatory responses but increased activation of EIF2 signaling following infection as determined by RNA sequencing. Intranasal infection of K18 human ACE2 transgenic mice (K18-hACE2) with SARS-CoV-2 resulted in lung pathology associated with viral replication and immune cell infiltration. In addition, ∼50% of infected mice exhibited CNS infection characterized by wide-spread viral replication in neurons accompanied by increased expression of chemokine (<italic>Cxcl9, Cxcl10, Ccl2, Ccl5</italic> and <italic>Ccl19</italic>) and cytokine (<italic>Ifn-λ</italic> and <italic>Tnf-α</italic>) transcripts associated with microgliosis and a neuroinflammatory response consisting primarily of monocytes/macrophages. Microglia depletion via administration of colony-stimulating factor 1 receptor inhibitor, PLX5622, in SARS-CoV-2 infected mice did not affect survival or viral replication but did result in dampened expression of proinflammatory cytokine/chemokine transcripts and a reduction in monocyte/macrophage infiltration. These results argue that microglia are dispensable in terms of controlling SARS-CoV-2 replication in in the K18-hACE2 model but do contribute to an inflammatory response through expression of pro-inflammatory genes. Collectively, these findings contribute to previous work demonstrating the ability of SARS-CoV-2 to infect neurons as well as emphasizing the potential use of the K18-hACE2 model to study immunological and neuropathological aspects related to SARS-CoV-2-induced neurologic disease.</p><sec><title>Importance</title><p>Understanding the immunological mechanisms contributing to both host defense and disease following viral infection of the CNS is of critical importance given the increasing number of viruses that are capable of infecting and replicating within the nervous system. With this in mind, the present study was undertaken to evaluate the role of microglia in aiding in host defense following experimental infection of the central nervous system (CNS) of K18-hACE2 with SARS-CoV-2, the causative agent of COVID-19. Neurologic symptoms that range in severity are common in COVID-19 patients and understanding immune responses that contribute to restricting neurologic disease can provide important insight into better understanding consequences associated with SARS-CoV-2 infection of the CNS.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.15.468729', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'NAD precursors cycle between host tissues and the gut microbiome', 'authors': \"[{'name': 'Karthikeyani Chellappa'}, {'name': 'Melanie R. McReynolds'}, {'name': 'Wenyun Lu'}, {'name': 'Xianfeng Zeng'}, {'name': 'Mikhail Makarov'}, {'name': 'Faisal Hayat'}, {'name': 'Sarmistha Mukherjee'}, {'name': 'Yashaswini R. Bhat'}, {'name': 'Siddharth R. Lingala'}, {'name': 'Rafaella T. Shima'}, {'name': 'Hélène C. Descamps'}, {'name': 'Timothy Cox'}, {'name': 'Lixin Ji'}, {'name': 'Connor Jankowski'}, {'name': 'Qingwei Chu'}, {'name': 'Shawn M. Davidson'}, {'name': 'Christoph A. Thaiss'}, {'name': 'Marie E. Migaud'}, {'name': 'Joshua D. Rabinowtiz'}, {'name': 'Joseph A. Baur'}]\", 'abstract': '<p>Nicotinamide adenine dinucleotide (NAD) is an essential redox cofactor in both mammals and microbes. Here we use isotope tracing to investigate the precursors supporting NAD synthesis in the gut microbiome. We find that preferred dietary NAD precursors are absorbed in the proximal part of the gastrointestinal tract and not available to microbes in the distal gut. Instead, circulating host nicotinamide enters the gut lumen and supports gut microbiome NAD synthesis. In addition, the microbiome converts nicotinamide, originating from the host circulation, into nicotinic acid. Host tissues uptake and utilize this microbiome-derived nicotinic acid for NAD synthesis, maintaining circulating nicotinic acid levels even in the absence of dietary consumption. Moreover, the main route from oral nicotinamide riboside, a widely used nutraceutical, to host NAD is via conversion into nicotinic acid by the gut microbiome. Thus, NAD precursors cycle between the host and gut microbiome to maintain NAD homeostasis.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.14.21266328', 'tweets': 41, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Human genetic evidence supports MAP3K15 inhibition as a therapeutic strategy for diabetes', 'authors': \"[{'name': 'Abhishek Nag'}, {'name': 'Ryan S. Dhindsa'}, {'name': 'Andrew R. Harper'}, {'name': 'Dimitrios Vitsios'}, {'name': 'Andrea Ahnmark'}, {'name': 'Bilada Bilican'}, {'name': 'Katja Madeyski-Bengtson'}, {'name': 'Bader Zarrouki'}, {'name': 'Quanli Wang'}, {'name': 'Katherine Smith'}, {'name': 'Dave Smith'}, {'name': 'Benjamin Challis'}, {'name': 'Dirk S. Paul'}, {'name': 'Mohammad Bohlooly-Y'}, {'name': 'Mike Snowden'}, {'name': 'David Baker'}, {'name': 'Regina Fritsche-Danielson'}, {'name': 'Menelas N. Pangalos'}, {'name': 'Slavé Petrovski'}]\", 'abstract': '<p>Diabetes mellitus is a chronic health condition that can result in significant end-organ complications and is estimated to impact at least 8.5% of the global adult population. Here, we performed gene-level collapsing analysis on exome sequences from 454,796 multi-ancestry UK Biobank participants to detect genetic associations with diabetes. Rare non-synonymous variants in <italic>GCK, GIGYF1, HNF1A</italic>, and <italic>HNF4A</italic> were significantly associated (<italic>P</italic>&lt;1×10<sup>-8</sup>) with increased risk of diabetes, whereas rare non-synonymous variants in <italic>MAP3K15</italic> were significantly associated with reduced risk of diabetes. Recessive carriers of rare non-synonymous variants in the X chromosome gene <italic>MAP3K15</italic> had a 30% reduced risk of diabetes (OR=0.70, 95% CI: [0.62,0.79], <italic>P</italic>=5.7×10<sup>-10</sup>), along with reduced blood glucose (beta=-0.13, 95% CI: [-0.15,-0.10], <italic>P</italic>=5.5×10<sup>-18</sup>) and reduced glycosylated haemoglobin levels (beta=-0.14, 95% CI: [-0.16,-0.11], <italic>P</italic>=1.1×10<sup>-24</sup>). Hemizygous males carrying protein-truncating variants (PTVs) in <italic>MAP3K15</italic> demonstrated a 40% reduced risk of diabetes (OR=0.60, 95% CI: [0.45,0.81], <italic>P=</italic>0.0007). These findings were independently replicated in FinnGen, with a <italic>MAP3K15</italic> PTV associating with decreased risk of both type 1 diabetes (T1DM) and type 2 diabetes (T2DM) (p&lt;0.05). The effect of <italic>MAP3K15</italic> loss on diabetes was independent of body mass index, suggesting its protective effect is unlikely to be mediated via the insulin resistance pathway. Tissue expression profile of <italic>MAP3K15</italic> indicates a possible involvement of pancreatic islet cell or stress response pathways. No safety concerns were identified among heterozygous or recessive <italic>MAP3K15</italic> PTV carriers across over 15,719 studied endpoints in the UK Biobank. Human population genetic evidence supports <italic>MAP3K15</italic> inhibition as a novel therapeutic target for diabetes.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.21266187', 'tweets': 41, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Controlling SARS-CoV-2 in schools using repetitive testing strategies', 'authors': \"[{'name': 'Andrea Torneri'}, {'name': 'Lander Willem'}, {'name': 'Vittoria Colizza'}, {'name': 'Cécile Kremer'}, {'name': 'Christelle Meuris'}, {'name': 'Gilles Darcis'}, {'name': 'Niel Hens'}, {'name': 'Pieter Libin'}]\", 'abstract': '<p>SARS-CoV-2 remains a worldwide emergency. While vaccines have been approved and are widely administered, these are only available to adults and adolescents in Europe. Therefore, in order to mitigate the spread of more transmissible SARS-CoV-2 variants among children, the use of non-pharmaceutical interventions is still warranted. We investigate the impact of different testing strategies on the SARS-CoV-2 infection dynamics in a primary school environment, using an individual-based modelling approach. Specifically, we consider three testing strategies: 1) <italic>symptomatic isolation</italic>, where we test symptomatic individuals and isolate them when they test positive, 2) <italic>reactive screening</italic>, where a class is screened once one symptomatic individual was identified, and 3) <italic>repetitive screening</italic>, where the school in its entirety is screened on regular time intervals. Through this analysis, we demonstrate that repetitive testing strategies can significantly reduce the attack rate in schools, contrary to a reactive screening approach. Furthermore, we investigate the impact of these testing strategies on the average number of school days lost per child.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.17.469009', 'tweets': 39, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Not so fast: Limited validity of deep convolutional neural networks as in silico models for human naturalistic face processing', 'authors': \"[{'name': 'Guo Jiahui'}, {'name': 'Ma Feilong'}, {'name': 'Matteo Visconti di Oleggio Castello'}, {'name': 'Samuel A Nastase'}, {'name': 'James V Haxby'}, {'name': 'Maria Ida Gobbini'}]\", 'abstract': '<p>Deep convolutional neural networks (DCNNs) trained for face identification can rival and even exceed human-level performance. The relationships between internal representations learned by DCNNs and those of the primate face processing system are not well understood, especially in naturalistic settings. We developed the largest naturalistic dynamic face stimulus set in human neuroimaging research (700+ naturalistic video clips of unfamiliar faces) and used representational similarity analysis to investigate how well the representations learned by high-performing DCNNs match human brain representations across the entire distributed face processing system. DCNN representational geometries were strikingly consistent across diverse architectures and captured meaningful variance among faces. Similarly, representational geometries throughout the human face network were highly consistent across subjects. Nonetheless, correlations between DCNN and neural representations were very weak overall—DCNNs captured 3% of variance in the neural representational geometries at best. Intermediate DCNN layers better matched visual and face-selective cortices than the final fully-connected layers. Behavioral ratings of face similarity were highly correlated with intermediate layers of DCNNs, but also failed to capture representational geometry in the human brain. Our results suggest that the correspondence between intermediate DCNN layers and neural representations of naturalistic human face processing is weak at best, and diverges even further in the later fully-connected layers. This poor correspondence can be attributed, at least in part, to the dynamic and cognitive information that plays an essential role in human face processing but is not modeled by DCNNs. These mismatches indicate that current DCNNs have limited validity as <italic>in silico</italic> models of dynamic, naturalistic face processing in humans.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.468552', 'tweets': 35, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Proteolysis Targeting Chimeras With Reduced Off-targets', 'authors': \"[{'name': 'Tuan M Nguyen'}, {'name': 'Arghya Deb'}, {'name': 'Praveen Kokkonda'}, {'name': 'Vedagopuram Sreekanth'}, {'name': 'Praveen K Tiwari'}, {'name': 'Veronika M Shoba'}, {'name': 'Sophia Lai'}, {'name': 'Santosh K Chaudhary'}, {'name': 'Jaron A.M. Mercer'}, {'name': 'Max Jan'}, {'name': 'David R Liu'}, {'name': 'Benjamin L Ebert'}, {'name': 'Amit Choudhary'}]\", 'abstract': '<p>Proteolysis Targeting Chimeras (PROTACs), a class of heterobifunctional molecules that recruit target proteins to E3 ligases, have gained traction for targeted protein degradation. However, pomalidomide, a widely used E3 ligase recruiter in PROTACs, can independently degrade other targets, such as zinc-finger (ZF) proteins, that hold key functions in normal development and disease progression. This off-target degradation of pomalidomide-based PROTACs raises concerns about their therapeutic applicability and long-term side effects. Therefore, there is a crucial need to develop rules for PROTAC design that minimize off-target degradation. In this study, we developed a high-throughput platform that interrogates the off-target degradation of ZF domains and discovered, using this platform, that PROTACs with the current design paradigm induce significant degradation of several ZF proteins. To identify new rules for PROTAC design, we generated a rationalized library of pomalidomide analogs with distinct exit vector modifications on the C4 and C5 positions of the phthalimide ring and profiled their propensities for ZF protein degradation. We found that modifications on the C5 position with nucleophilic aromatic substitution (S<sub>N</sub>Ar) reduce off-target ZF degradation. We applied our newfound design principles on a previously developed ALK oncoprotein-targeting PROTAC and generated PROTACs with enhanced potency and minimal off-target degradation. We envision the reported off-target profiling platform and pomalidomide analogs will find utility in the design of specific PROTACs.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.14.468535', 'tweets': 35, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Rapid reconstruction of neural circuits using tissue expansion and lattice light sheet microscopy', 'authors': \"[{'name': 'Joshua L. Lillvis'}, {'name': 'Hideo Otsuna'}, {'name': 'Xiaoyu Ding'}, {'name': 'Igor Pisarev'}, {'name': 'Takashi Kawase'}, {'name': 'Jennifer Colonell'}, {'name': 'Konrad Rokicki'}, {'name': 'Cristian Goina'}, {'name': 'Ruixuan Gao'}, {'name': 'Amy Hu'}, {'name': 'Kaiyu Wang'}, {'name': 'John Bogovic'}, {'name': 'Daniel E. Milkie'}, {'name': 'Linus Meienberg'}, {'name': 'Edward S. Boyden'}, {'name': 'Stephan Saalfeld'}, {'name': 'Paul W. Tillberg'}, {'name': 'Barry J. Dickson'}]\", 'abstract': '<p>Electron microscopy (EM) allows for the reconstruction of dense neuronal connectomes but suffers from low throughput, limiting its application to small numbers of reference specimens. We developed a protocol and analysis pipeline using tissue expansion and lattice light-sheet microscopy (ExLLSM) to rapidly reconstruct selected circuits across many samples with single synapse resolution and molecular contrast. We validate this approach in <italic>Drosophila</italic>, demonstrating that it yields synaptic counts similar to those obtained by EM, can be used to compare counts across sex and experience, and to correlate structural connectivity with functional connectivity. This approach fills a critical methodological gap in studying variability in the structure and function of neural circuits across individuals within and between species.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.17.469005', 'tweets': 33, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Robust temporal map of human in vitro myelopoiesis using single-cell genomics', 'authors': \"[{'name': 'Clara Alsinet'}, {'name': 'Maria Primo'}, {'name': 'Valentina Lorenzi'}, {'name': 'Andrew J Knights'}, {'name': 'Carmen Sancho-Serra'}, {'name': 'Jong-Eun Park'}, {'name': 'Beata S Wyspianska'}, {'name': 'David F Tough'}, {'name': 'Damiana Alvarez'}, {'name': 'Daniel J Gaffney'}, {'name': 'Roser Vento-Tormo'}]\", 'abstract': '<p>Myeloid cells have a central role in homeostasis and tissue defence. Characterising the current in vitro protocols of myelopoiesis is imperative for their use in research and immunotherapy as well as for understanding the early stages of myeloid differentiation in humans. Here, we profiled the transcriptome of more than 400k cells and generated a robust molecular map of the differentiation of human induced pluripotent stem cells (iPSC) into macrophages. By integrating our in vitro datasets with in vivo single-cell developmental atlases, we found that in vitro macrophage differentiation recapitulates features of in vivo yolk sac hematopoiesis, which happens prior to the appearance of definitive hematopoietic stem cells (HSC). During in vitro myelopoiesis, a wide range of myeloid cells are generated, including erythrocytes, mast cells and monocytes, suggesting that, during early human development, the HSC-independent immune wave gives rise to multiple myeloid cell lineages. We leveraged this model to characterize the transition of hemogenic endothelium into myeloid cells, uncovering poorly described myeloid progenitors and regulatory programs. Taking advantage of the variety of myeloid cells produced, we developed a new protocol to produce type 2 conventional dendritic cells (cDC2) in vitro. We found that the underlying regulatory networks coding for myeloid identity are conserved in vivo and in vitro. Using genetic engineering techniques, we validated the effects of key transcription factors important for cDC2 and macrophage identity and ontogeny. This roadmap of early myeloid differentiation will serve as an important resource for investigating the initial stages of hematopoiesis, which are largely unexplored in humans, and will open up new therapeutic opportunities.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468770', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Comparative genomics on cultivated and uncultivated, freshwater and marine Candidatus Manganitrophaceae species implies their worldwide reach in manganese chemolithoautotrophy', 'authors': \"[{'name': 'Hang Yu'}, {'name': 'Grayson L. Chadwick'}, {'name': 'Usha F. Lingappa'}, {'name': 'Jared R. Leadbetter'}]\", 'abstract': '<p>Chemolithoautotrophic manganese oxidation has long been theorized, but only recently demonstrated in a bacterial co-culture. The majority member of the co-culture, <italic>Candidatus</italic> Manganitrophus noduliformans, is a distinct but not yet isolated lineage in the phylum <italic>Nitrospirota</italic> (<italic>Nitrospirae</italic>). Here, we established two additional MnCO<sub>3</sub>-oxidizing cultures using inocula from Santa Barbara (USA) and Boetsap (South Africa). Both cultures were dominated by strains of a new species, designated <italic>Candidatus</italic> Manganitrophus morganii. The next abundant members differed in the available cultures, suggesting that while <italic>Ca</italic>. Manganitrophus species have not been isolated in pure culture, they may not require a specific syntrophic relationship with another species. Phylogeny of cultivated <italic>Ca</italic>. Manganitrophus and related metagenome-assembled genomes revealed a coherent taxonomic family, <italic>Candidatus</italic> Manganitrophaceae, from both freshwater and marine environments and distributed globally. Comparative genomic analyses support this family being Mn(II)-oxidizing chemolithoautotrophs. Among the 895 shared genes were a subset of those hypothesized for Mn(II) oxidation (Cyc2 and PCC_1) and oxygen reduction (TO_1 and TO_2) that could facilitate Mn(II) lithotrophy. An unusual, plausibly reverse Complex 1 containing 2 additional pumping subunits was also shared by the family, as were genes for the reverse TCA carbon fixation cycle, which could enable Mn(II) autotrophy. All members of the family lacked genes for nitrification found in <italic>Nitrospira</italic> species. The results suggest that <italic>Ca</italic>. Manganitrophaceae share a core set of candidate genes for the newly discovered manganese dependent chemolithoautotrophic lifestyle, and likely have a broad, global distribution.</p><sec><title>Importance</title><p>Manganese (Mn) is an abundant redox-active metal that cycled in many of Earth’s biomes. While diverse bacteria and archaea have been demonstrated to respire Mn(III/IV), only recently have bacteria been implicated in Mn(II) oxidation dependent growth. Here, two new Mn(II)-oxidizing enrichment cultures originated from two continents and hemispheres were examined. By comparing the community composition of the enrichments and performing phylogenomic analysis on the abundant <italic>Nitrospirota</italic> therein, new insights are gleaned on cell interactions, taxonomy, and machineries that may underlie Mn(II)-based lithotrophy and autotrophy.</p></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.18.469135', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Automated assembly scaffolding elevates a new tomato system for high-throughput genome editing', 'authors': \"[{'name': 'Michael Alonge'}, {'name': 'Ludivine Lebeigle'}, {'name': 'Melanie Kirsche'}, {'name': 'Sergey Aganezov'}, {'name': 'Xingang Wang'}, {'name': 'Zachary B. Lippman'}, {'name': 'Michael C. Schatz'}, {'name': 'Sebastian Soyk'}]\", 'abstract': '<p>Advancing crop genomics requires efficient genetic systems enabled by high-quality personalized genome assemblies. Here, we introduce RagTag, a toolset for automating assembly scaffolding and patching, and we establish chromosome-scale reference genomes for the widely used tomato genotype M82 along with Sweet-100, a rapid-cycling genotype that we developed to accelerate functional genomics and genome editing. This work outlines strategies to rapidly expand genetic systems and genomic resources in other plant species.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.19.21265383', 'tweets': 29, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'GWAS defines pathogenic signaling pathways and prioritizes drug targets for IgA nephropathy', 'authors': \"[{'name': 'Krzysztof Kiryluk'}, {'name': 'Elena Sanchez-Rodriguez'}, {'name': 'Xu-jie Zhou'}, {'name': 'Francesca Zanoni'}, {'name': 'Lili Liu'}, {'name': 'Nikol Mladkova'}, {'name': 'Atlas Khan'}, {'name': 'Maddalena Marasa'}, {'name': 'Jun-Ying Zhang'}, {'name': 'Olivia Balderes'}, {'name': 'Simone Sanna-Cherchi'}, {'name': 'Andrew S. Bomback'}, {'name': 'Pietro A. Canetta'}, {'name': 'Gerald B. Appel'}, {'name': 'Jai Radhakrishnan'}, {'name': 'Hernan Trimarchi'}, {'name': 'Ben Sprangers'}, {'name': 'Daniel C. Cattran'}, {'name': 'Heather Reich'}, {'name': 'York Pei'}, {'name': 'Pietro Ravani'}, {'name': 'Kresimir Galesic'}, {'name': 'Dita Maixnerova'}, {'name': 'Vladimir Tesar'}, {'name': 'Benedicte Stengel'}, {'name': 'Marie Metzger'}, {'name': 'Guillaume Canaud'}, {'name': 'Nicolas Maillard'}, {'name': 'Francois Berthoux'}, {'name': 'Laureline Berthelot'}, {'name': 'Evangeline Pillebout'}, {'name': 'Renato Monteiro'}, {'name': 'Raoul Nelson'}, {'name': 'Robert Wyatt'}, {'name': 'William Smoyer'}, {'name': 'John Mahan'}, {'name': 'Al-Akash Samhar'}, {'name': 'Guillermo Hidalgo'}, {'name': 'Alejandro Quiroga'}, {'name': 'Patricia Weng'}, {'name': 'Raji Sreedharan'}, {'name': 'David Selewski'}, {'name': 'Keefe Davis'}, {'name': 'Mahmoud Kallash'}, {'name': 'Tetyana L. Vasylyeva'}, {'name': 'Michelle Rheault'}, {'name': 'Aftab Chishti'}, {'name': 'Daniel Ranch'}, {'name': 'Scott E. Wenderfer'}, {'name': 'Dmitry Samsonov'}, {'name': 'Donna J. Claes'}, {'name': 'Akchurin Oleh'}, {'name': 'Dimitrios Goumenos'}, {'name': 'Maria Stangou'}, {'name': 'Judit Nagy'}, {'name': 'Tibor Kovacs'}, {'name': 'Enrico Fiaccadori'}, {'name': 'Antonio Amoroso'}, {'name': 'Cristina Barlassina'}, {'name': 'Daniele Cusi'}, {'name': 'Lucia Del Vecchio'}, {'name': 'Giovanni-Giorgio Battaglia'}, {'name': 'Monica Bodria'}, {'name': 'Emanuela Boer'}, {'name': 'Luisa Bono'}, {'name': 'Giuliano Boscutti'}, {'name': 'Gianluca Caridi'}, {'name': 'Francesca Lugani'}, {'name': 'GianMarco Ghiggeri'}, {'name': 'Rosanna Coppo'}, {'name': 'Licia Peruzzi'}, {'name': 'Vittoria Esposito'}, {'name': 'Ciro Esposito'}, {'name': 'Sandro Feriozzi'}, {'name': 'Rosaria Polci'}, {'name': 'Giovanni Frasca'}, {'name': 'Marco Galliani'}, {'name': 'Maurizio Garozzo'}, {'name': 'Adele Mitrotti'}, {'name': 'Loreto Gesualdo'}, {'name': 'Simona Granata'}, {'name': 'Gianluigi Zaza'}, {'name': 'Francesco Londrino'}, {'name': 'Riccardo Magistroni'}, {'name': 'Isabella Pisani'}, {'name': 'Andrea Magnano'}, {'name': 'Carmelita Marcantoni'}, {'name': 'Piergiorgio Messa'}, {'name': 'Renzo Mignani'}, {'name': 'Antonello Pani'}, {'name': 'Claudio Ponticelli'}, {'name': 'Dario Roccatello'}, {'name': 'Maurizio Salvadori'}, {'name': 'Erica Salvi'}, {'name': 'Domenico Santoro'}, {'name': 'Guido Gembillo'}, {'name': 'Silvana Savoldi'}, {'name': 'Donatella Spotti'}, {'name': 'Pasquale Zamboli'}, {'name': 'Claudia Izzi'}, {'name': 'Federico Alberici'}, {'name': 'Elisa Delbarba'}, {'name': 'Michal Florczak'}, {'name': 'Natalia Krata'}, {'name': 'Krzysztof Mucha'}, {'name': 'Leszek Paczek'}, {'name': 'Stanisaw Niemczyk'}, {'name': 'Barbara Moszczuk'}, {'name': 'Magorzata Panczyk-Tomaszewska'}, {'name': 'Malgorzata Mizerska-Wasiak'}, {'name': 'Agnieszka Perkowska-Ptasinska'}, {'name': 'Teresa Baczkowska'}, {'name': 'Magdalena Durlik'}, {'name': 'Krzysztof Pawlaczyk'}, {'name': 'Przemyslaw Sikora'}, {'name': 'Marcin Zaniew'}, {'name': 'Dorota Kaminska'}, {'name': 'Magdalena Krajewska'}, {'name': 'Izabella Kuzmiuk-Glembin'}, {'name': 'Zbigniew Heleniak'}, {'name': 'Barbara Bullo-Piontecka'}, {'name': 'Tomasz Liberek'}, {'name': 'Alicja Dbska-Slizien'}, {'name': 'Tomasz Hryszko'}, {'name': 'Anna Materna-Kiryluk'}, {'name': 'Monika Miklaszewska'}, {'name': 'Maria Szczepanska'}, {'name': 'Katarzyna Dyga'}, {'name': 'Edyta Machura'}, {'name': 'Katarzyna Siniewicz-Luzenczyk'}, {'name': 'Monika Pawlak-Bratkowska'}, {'name': 'Marcin Tkaczyk'}, {'name': 'Dariusz Runowski'}, {'name': 'Norbert Kwella'}, {'name': 'Dorota Drozdz'}, {'name': 'Ireneusz Habura'}, {'name': 'Florian Kronenberg'}, {'name': 'Larisa Prikhodina'}, {'name': 'David van Heel'}, {'name': 'Bertrand Fontaine'}, {'name': 'Chris Cotsapas'}, {'name': 'Cisca Wijmenga'}, {'name': 'Andre Franke'}, {'name': 'Vito Annese'}, {'name': 'Peter K. Gregersen'}, {'name': 'Sreeja Parameswaran'}, {'name': 'Matthew Weirauch'}, {'name': 'Leah Kottyan'}, {'name': 'John B. Harley'}, {'name': 'Hitoshi Suzuki'}, {'name': 'Ichiei Narita'}, {'name': 'Shin Goto'}, {'name': 'Hajeong Lee'}, {'name': 'Dong-Ki Kim'}, {'name': 'Yon Su Kim'}, {'name': 'Jin-Ho Park'}, {'name': 'BeLong Cho'}, {'name': 'Murim Choi'}, {'name': 'Ans Van Wijk'}, {'name': 'Ana Huerta'}, {'name': 'Elisabet Ars'}, {'name': 'Jose Ballarin'}, {'name': 'Sigrid Lundberg'}, {'name': 'Bruno Vogt'}, {'name': 'Laila-Yasmin Mani'}, {'name': 'Yasar Caliskan'}, {'name': 'Jonathan Barratt'}, {'name': 'Thilini Abeygunaratne'}, {'name': 'Philip A. Kalra'}, {'name': 'Daniel P. Gale'}, {'name': 'Ulf Panzer'}, {'name': 'Thomas Rauen'}, {'name': 'Jurgen Floege'}, {'name': 'Pascal Schlosser'}, {'name': 'Arif B. Ekici'}, {'name': 'Kai-Uwe Eckardt'}, {'name': 'Nan Chen'}, {'name': 'Jingyuan Xie'}, {'name': 'Richard P. Lifton'}, {'name': 'Ruth J.F. Loos'}, {'name': 'Eimear E. Kenny'}, {'name': 'Iuliana Ionita-Laza'}, {'name': 'Anna Kottgen'}, {'name': 'Bruce Julian'}, {'name': 'Jan Novak'}, {'name': 'Francesco Scolari'}, {'name': 'Hong Zhang'}, {'name': 'Ali G. Gharavi'}]\", 'abstract': '<p>IgA nephropathy (IgAN) is a progressive form of kidney disease defined by glomerular deposition of IgA. We performed a genome-wide association study involving 10,146 kidney biopsy-diagnosed IgAN cases and 28,751 matched controls across 17 international cohorts. We defined 30 independent genome-wide significant risk loci jointly explaining 11% of disease risk. A total of 16 loci were novel, including TNFSF4, REL, CD28, CXCL8/PF4V1, LY86, LYN, ANXA3, TNFSF8/15, REEP3, ZMIZ1, RELA, ETS1, IGH, IRF8, TNFRSF13B and FCAR. The SNP-based heritability of IgAN was estimated at 23%. We observed a positive genetic correlation between IgAN and total serum IgA levels, allergy, tonsillectomy, and several infections, and a negative correlation with inflammatory bowel disease. All significant non-HLA loci shared with serum IgA levels had a concordant effect on the risk of IgAN. Moreover, IgAN loci were globally enriched in gene orthologs causing abnormal IgA levels when genetically manipulated in mice. The explained heritability was enriched in the regulatory elements of cells from the immune and hematopoietic systems and intestinal mucosa, providing support for the pathogenic role of extra-renal tissues. The polygenic risk of IgAN was associated with early disease onset, increased lifetime risk of kidney failure, as well as hematuria and several other traits in a phenome-wide association study of 590,515 individuals. In the comprehensive functional annotation analysis of candidate causal genes across genome-wide significant loci, we observed the convergence of biological candidates on a common set of inflammatory signaling pathways and cytokine ligand-receptor pairs, prioritizing potential new drug targets.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.15.468712', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structures of the Cyanobacterial Phycobilisome', 'authors': \"[{'name': 'Paul V. Sauer'}, {'name': 'Maria Agustina Dominguez-Martin'}, {'name': 'Henning Kirst'}, {'name': 'Markus Sutter'}, {'name': 'David Bina'}, {'name': 'Basil J. Greber'}, {'name': 'Eva Nogales'}, {'name': 'Tomáš Polívka'}, {'name': 'Cheryl A. Kerfeld'}]\", 'abstract': '<p>The phycobilisome is an elaborate antenna that is responsible for light-harvesting in cyanobacteria and red-algae. This large macromolecular complex captures incident sunlight and transfers the energy via a network of pigment molecules called bilins to the photosynthetic reaction centers. The phycobilisome of the model organism Synechocystis PCC 6803 consists of a core to which six rods are attached but its detailed molecular architecture and regulation in response to environmental conditions is not well understood. Here we present cryo-electron microscopy structures of the 6.2 MDa phycobilisome from Synechocystis PCC 6803 resolved at 2.1 Å (rods) to 2.7 Å (core), revealing three distinct conformations, two previously unknown. We found that two of the rods are mobile and can switch conformation within the complex, revealing a layer of regulation not described previously. In addition, we found a novel linker protein in the structure, that may represent a long-sought subunit that tethers the phycobilisome to the thylakoid membrane. Finally, we show how excitation energy is transferred within the phycobilisome and correlate our structures with known spectroscopic properties. Together, our results provide detailed insights into the biophysical underpinnings of cyanobacterial light harvesting and lay the foundation for bioengineering of future phycobilisome variants and artificial light harvesting systems.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.20.468884', 'tweets': 28, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'A glycerol shunt functions as a glucose excess security valve in pancreatic β-cells', 'authors': \"[{'name': 'Anfal Al-Mass'}, {'name': 'Pegah Poursharifi'}, {'name': 'Marie-Line Peyot'}, {'name': 'Roxane Lussier'}, {'name': 'Emily J Levens'}, {'name': 'Julian Guida'}, {'name': 'Yves Mugabo'}, {'name': 'Elite Possik'}, {'name': 'Rasheed Ahmad'}, {'name': 'Fahd Al-Mulla'}, {'name': 'Rob Sladek'}, {'name': 'S.R.Murthy Madiraju'}, {'name': 'Marc Prentki'}]\", 'abstract': '<p>The recently identified glycerol-3-phosphate (Gro3P) phosphatase (G3PP) in mammalian cells, encoded by the PGP gene, was shown to regulate intermediary metabolism by hydrolyzing Gro3P and to control glucose-stimulated insulin secretion (GSIS) in β-cells, in vitro. We now examined in inducible β-cell specific G3PP-KO (BKO) mice, the role of G3PP in the control of insulin secretion in vivo, β-cell function and glucotoxicity. BKO mice, compared to MCre controls, showed increased body weight, adiposity, fed insulinemia, GSIS, reduced plasma triglycerides and mildly altered glucose tolerance. Isolated BKO mouse islets at high (16.7 mM) but not low or intermediate glucose (3-8 mM) showed elevated GSIS, Gro3P, metabolites reflecting β-cell activation, O2 consumption, ATP production and reduced glycerol release. BKO islets chronically exposed to elevated glucose showed increased apoptosis, reduced insulin content and expression of Pdx-1, MafA and Ins-2 genes. As G3PP channels glucose carbons towards glycerol formation and release, the results demonstrate that β-cell are endowed with a glycerol shunt acting as a glucose excess security valve. We propose that the glycerol shunt plays a role in glucodetoxification, the prevention of insulin hypersecretion, acts as a defense against excess body weight gain and contributes to β-cell mass preservation in the face of hyperglycemia.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.19.21266552', 'tweets': 27, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Low neutralizing antibody titers against the Mu variant of SARS-CoV-2 in BNT162b2 vaccinated individuals', 'authors': \"[{'name': 'Diego A. Álvarez-Díaz'}, {'name': 'Ana Luisa Muñoz'}, {'name': 'Pilar Tavera-Rodríguez'}, {'name': 'María T. Herrera-Sepúlveda'}, {'name': 'Hector Alejandro Ruiz-Moreno'}, {'name': 'Katherine Laiton-Donato'}, {'name': 'Carlos Franco-Muñoz'}, {'name': 'Dioselina Pelaez-Carvajal'}, {'name': 'Diego Cuellar'}, {'name': 'Alejandra M. Muñoz-Ramirez'}, {'name': 'Marisol Galindo'}, {'name': 'Edgar J. Arias-Ramirez'}, {'name': 'Marcela Mercado-Reyes'}]\", 'abstract': '<sec><title>Background</title><p>Global surveillance programs for the virus that causes COVID-19 are showing the emergence of variants with mutations in the Spike protein, including the Mu variant, recently declared a Variant of Interest (VOI) by the World Health Organization. Genomic and laboratory surveillance is important in these types of variants because they may be more infectious or less susceptible to antiviral treatments and vaccine-induced antibodies.</p></sec><sec><title>Objectives</title><p>To evaluate the sensitivity of the Mu variant (B.1.621) to neutralizing antibodies induced by the BNT162b2 vaccine.</p></sec><sec><title>Study design</title><p>Three of the most predominant SARS-CoV-2 variants in Colombia during the epidemiological peaks of 2021 were isolated. Microneutralization assays were performed by incubating 120 TCDI<sub>50</sub> of each SARS-CoV-2 isolate with five 2-fold serial dilutions of sera from 14 BNT162b2 vaccinated volunteers. The MN<sub>50</sub> titer was calculated by the Reed-Muench formula</p></sec><sec><title>Results</title><p>The three isolated variants were Mu, a Variant of Interest (VOI), Gamma, a variant of concern (VOC), and B.1.111 that lacks genetic markers associated with greater virulence. At the end of August, the Mu and Gamma variants were widely distributed in Colombia. Mu was predominant (49%), followed by Gamma (25%). In contrast, B.1.111 became almost undetectable. The evaluation of neutralizing antibodies suggests that patients vaccinated with BNT162-2 generate neutralizing antibody titers against the Mu variant at significantly lower concentrations relative to B.1.111 and Gamma.</p></sec><sec><title>Conclusions</title><p>This study shows the importance of continuing with surveillance programs of emerging variants as well as the need to evaluate the neutralizing antibody response induced by other vaccines circulating in the country against Mu and other variants with high epidemiological impact.</p></sec><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Mu and Gamma variants represented 49% and 25% of cases in Colombia by August 2021.</p></list-item><list-item><p>Increased proportion of SARS-COV-2 cases were mostly associated with Mu variant, despite being detected simultaneously with the VOC Gamma</p></list-item><list-item><p>The Mu variant remarkably escapes from neutralizing antibodies elicited by the BNT162b2-vaccine</p></list-item><list-item><p>Laboratory studies of neutralizing antibodies are useful to determine the efficacy of SARS-CoV-2 vaccines against VOC and VOI.</p></list-item></list></sec>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.467545', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Slow fluctuations in ongoing brain activity decrease in amplitude with ageing yet their impact on task-related evoked responses is dissociable from behaviour', 'authors': \"[{'name': 'Maria J. Ribeiro'}, {'name': 'Miguel Castelo-Branco'}]\", 'abstract': '<p>In humans, ageing is characterized by decreased brain signal variability and increased behavioural variability. To understand how reduced brain variability segregates with increased behavioural variability, we investigated the association between reaction time variability, evoked brain responses and ongoing brain signal dynamics, in young (N = 36) and older adults (N = 39). We studied the electroencephalogram (EEG) and pupil size fluctuations to characterize the cortical and arousal responses elicited by a cued go/no-go task. Evoked responses were strongly modulated by slow (&lt; 2 Hz) fluctuations of the ongoing signals, which presented reduced power in the older participants. Although variability of the evoked responses was lower in the older participants, once we adjusted for the effect of the ongoing signal fluctuations, evoked responses were equally variable in both groups. Moreover, the modulation of the evoked responses caused by the ongoing signal fluctuations had no impact on reaction time, thereby explaining why although ongoing brain signal variability is decreased in older individuals, behavioural variability is not. Finally, we showed that adjusting for the effect of the ongoing signal was critical to unmask the link between neural responses and behaviour.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.14.468560', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Shade suppresses wound-induced leaf repositioning through a mechanism involving PHYTOCHROME KINASE SUBSTRATE (PKS) genes', 'authors': \"[{'name': 'Anne-Sophie Fiorucci'}, {'name': 'Olivier Michaud'}, {'name': 'Emanuel Schmid-Siegert'}, {'name': 'Martine Trevisan'}, {'name': 'Laure Allenbach Petrolati'}, {'name': 'Yetkin Çaka Ince'}, {'name': 'Christian Fankhauser'}]\", 'abstract': '<p>Shaded plants challenged with herbivores or pathogens prioritize growth over defense. However, most experiments have focused on the effect of shading light cues on defense responses. To investigate the potential interaction between shade-avoidance and wounding-induced Jasmonate (JA)-mediated signaling on leaf growth and movement, we used repetitive mechanical wounding of leaf blades to mimic herbivore attacks. Phenotyping experiments with combined treatments on <italic>Arabidopsis thaliana</italic> rosettes revealed that shade strongly inhibits the wound effect on leaf elevation. By contrast, petiole length is reduced by wounding both in the sun and in the shade. Thus, the relationship between the shade and wounding/JA pathways varies depending on the physiological response, implying that leaf growth and movement can be uncoupled. Using RNA-sequencing, we identified genes with expression patterns matching the hyponastic response (opposite regulation by both stimuli, interaction between treatments with shade dominating the wound signal). Among them were genes from the <italic>PKS</italic> (Phytochrome Kinase Substrate) family, which was previously studied for its role in phototropism and leaf positioning. Interestingly, we observed reduced shade suppression of the wounding effect in <italic>pks2pks4</italic> double mutants while a <italic>PKS4</italic> overexpressing line showed constitutively elevated leaves and was less sensitive to wounding. Our results indicate a trait-specific interrelationship between shade and wounding cues on Arabidopsis leaf growth and positioning. Moreover, we identify <italic>PKS</italic> genes as integrators of external cues in the control of leaf hyponasty further emphasizing the role of these genes in aerial organ positioning.</p><sec><title>Author summary</title><p>Plants face different types of stressful situations without the ability to relocate to favorable environments. For example, increasing plant density reduces access to sunlight as plants start to shade each other. Foliar shading represents a stress that many plants cope with by changing their morphology. This includes elongation of stem-like structures and repositioning of leaves to favor access to unfiltered sunlight. Plants also defend themselves against various pathogens including herbivores. Defense mechanisms include the production of deterrent chemical and morphological adaptations such as stunted growth and downwards leaf repositioning. Here we studied the morphological response of plants when simultaneously facing shade and herbivore stress. When facing both stresses petiole growth was intermediate between the shade-enhanced and wound-repressed response. In contrast, the shade cue overrides the wounding cue leading to a similar upwards leaf repositioning in the combined treatments or in the response to shade alone. Using gene expression analyses and genetics we identified two members of the Phytochrome Kinase Substrate family as playing a signal integration role when plants simultaneously faced both stresses. This contributes to our understanding of the mechanisms underlying plant morphological adaptations when facing multiple stresses.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.14.468537', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Viral E Protein Neutralizes BET Protein-Mediated Post-Entry Antagonism of SARS-CoV-2', 'authors': \"[{'name': 'Irene P. Chen'}, {'name': 'James E. Longbotham'}, {'name': 'Sarah McMahon'}, {'name': 'Rahul K. Suryawanshi'}, {'name': 'Jared Carlson-Stevermer'}, {'name': 'Meghna Gupta'}, {'name': 'Meng Yao Zhang'}, {'name': 'Frank W. Soveg'}, {'name': 'Jennifer M. Hayashi'}, {'name': 'Taha Y. Taha'}, {'name': 'Victor L. Lam'}, {'name': 'Yang Li'}, {'name': 'Zanlin Yu'}, {'name': 'Erron W. Titus'}, {'name': 'Amy Diallo'}, {'name': 'Jennifer Oki'}, {'name': 'Kevin Holden'}, {'name': 'Nevan Krogan'}, {'name': 'Danica Galonić Fujimori'}, {'name': 'Melanie Ott'}, {'name': ' '}]\", 'abstract': '<p>Inhibitors of Bromodomain and Extra-terminal domain (BET) proteins are possible anti-SARS-CoV-2 prophylactics as they downregulate angiotensin-converting enzyme 2 (ACE2). Here, we show that BET proteins should not be inactivated therapeutically as they are critical antiviral factors at the post-entry level. Knockouts of BRD3 or BRD4 in cells overexpressing ACE2 exacerbate SARS-CoV-2 infection; the same is observed when cells with endogenous ACE2 expression are treated with BET inhibitors during infection, and not before. Viral replication and mortality are also enhanced in BET inhibitor-treated mice overexpressing ACE2. BET inactivation suppresses interferon production induced by SARS-CoV-2, a process phenocopied by the envelope (E) protein previously identified as a possible “histone mimetic.” E protein, in an acetylated form, directly binds the second bromodomain of BRD4. Our data support a model where SARS-CoV-2 E protein evolved to antagonize interferon responses via BET protein inhibition; this neutralization should not be further enhanced with BET inhibitor treatment.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468735', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Modulation of Cutaneous Responses in the Cuneate Nucleus of Macaques During Active Movement', 'authors': \"[{'name': 'Qinpu He'}, {'name': 'Christopher S. Versteeg'}, {'name': 'Aneesha K. Suresh'}, {'name': 'Lee E. Miller'}, {'name': 'Sliman J. Bensmaia'}]\", 'abstract': '<p>To achieve stable and precise movement execution, the sensorimotor system integrates exafferent sensory signals originating from interactions with the external world and reafferent signals caused by our own movements. This barrage of sensory information is regulated such that behaviorally relevant signals are boosted at the expense of irrelevant ones. For example, sensitivity to touch is reduced during movement – when cutaneous signals caused by skin stretch are expected and uninteresting – a phenomenon reflected in a decreased cutaneous responsiveness in thalamus and cortex. Some evidence suggests that movement gating of touch may originate from the cuneate nucleus (CN), the first recipient of signals from tactile nerve fibers along the dorsal columns medial lemniscal pathway. To test this possibility, we intermittently delivered mechanical pulses to the receptive fields (RFs) of identified cutaneous CN neurons as monkeys performed a reach-to-grasp task. As predicted, we found that the cutaneous responses of individual CN neurons were reduced during movement. However, this movement gating of cutaneous signals was observed for CN neurons with RFs on the arm but not those with RFs on the hand. We conclude that sensory gating occurs in the first processing stage along the somatosensory neuraxis and sculpts incoming signals according to their task relevance.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.469032', 'tweets': 25, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Anatomical connectivity along the anterior-posterior axis of the human hippocampus: new insights using quantitative fibre-tracking.', 'authors': '[{\\'name\\': \\'Marshall Axel Dalton\\'}, {\\'name\\': \"Arkiev D\\'Souza\"}, {\\'name\\': \\'Jinglei Lv\\'}, {\\'name\\': \\'Fernando Calamante\\'}]', 'abstract': '<p>The hippocampus supports multiple cognitive functions including episodic memory. Recent work has highlighted functional differences along the anterior-posterior axis of the human hippocampus but the neuroanatomical underpinnings of these differences remain unclear. We leveraged track-density imaging to systematically examine anatomical connectivity between the cortical mantle and the anterior-posterior axis of the in-vivo human hippocampus. We first identified the most highly connected cortical areas and detailed the degree to which they preferentially connect along the anterior-posterior axis of the hippocampus. Then, using a tractography pipeline specifically tailored to measure the location and density of streamline endpoints within the hippocampus, we characterised where, within the hippocampus, these cortical areas preferentially connect. Our results were striking in showing that different parts of the hippocampus preferentially connect with distinct cortical areas. Furthermore, we provide evidence that both gradients and circumscribed areas of dense extrinsic anatomical connectivity exist within the human hippocampus. These findings inform conceptual debates in the field by unveiling how specific regions along the anterior-posterior axis of the hippocampus are associated with different cortical inputs/outputs. Overall, our results represent a major advance in our ability to map the anatomical connectivity of the human hippocampus in-vivo and inform our understanding of the neural architecture of hippocampal dependent memory systems in the human brain. This detailed characterization of how specific portions of the hippocampus anatomically connect with cortical brain regions may promote a better understanding of its role in cognition and we emphasize the importance of considering the hippocampus as a heterogeneous structure.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468719', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structure of the Quenched Cyanobacterial OCP-Phycobilisome Complex', 'authors': \"[{'name': 'Maria Agustina Dominguez-Martin'}, {'name': 'Paul V. Sauer'}, {'name': 'Markus Sutter'}, {'name': 'Henning Kirst'}, {'name': 'David Bina'}, {'name': 'Basil J. Greber'}, {'name': 'Eva Nogales'}, {'name': 'Tomáš Polívka'}, {'name': 'Cheryl A. Kerfeld'}]\", 'abstract': '<p>Photoprotection is an essential mechanism in photosynthetic organisms to balance the harvesting of light energy against the risks of photodamage. In cyanobacteria, photoprotective non-photochemical quenching relies on the interaction between a photoreceptor, the Orange Carotenoid Protein (OCP), and the antenna, the phycobilisome (PBS). Here we report the first structure of the OCP-PBS complex at 2.7 Å overall resolution obtained by cryo-electron microscopy. The structure shows that the 6.2 MDa PBS is quenched by four 34 kDa OCP organized as two dimers. The complex also reveals that the structure of the active form of the OCP is drastically different than its resting, non-quenching form, with an ∼60 Å displacement of its regulatory domain. These results provide a high-resolution blueprint of the structural basis of the protective quenching of excess excitation energy that enables cyanobacteria to harvest light energy and fix CO<sub>2</sub> across environmentally diverse and dynamic surface of our planet.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469072', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'scCODE: an R package for personalized differentially expressed gene detection on single-cell RNA-sequencing data', 'authors': \"[{'name': 'jiawei Zou'}, {'name': 'miaochen Wang'}, {'name': 'zhen Zhang'}, {'name': 'zheqi Liu'}, {'name': 'xiaobin Zhang'}, {'name': 'rong Hua'}, {'name': 'ke Chen'}, {'name': 'xin Zou'}, {'name': 'Jie Hao'}]\", 'abstract': \"<p>Differential expression (DE) gene detection in single-cell RNA-seq (scRNA-seq) data is a key step to understand the biological question investigated. We find that DE methods together with gene filtering have profound impact on DE gene identification, and different datasets will benefit from personalized DE gene detection strategies. Existing tools don't take gene filtering into consideration, and couldn't evaluate DE performance on real datasets without prior knowledge of true results. Based on two new metrics, we propose scCODE (single cell Consensus Optimization of Differentially Expressed gene detection), an R package to automatically optimize DE gene detection for each experimental scRNA-seq dataset.</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.468666', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PURC v2.0: a program for improved sequence inference for polyploid phylogenetics and other manifestations of the multiple-copy problem', 'authors': \"[{'name': 'Peter W Schafran'}, {'name': 'Fay-Wei W Li'}, {'name': 'Carl Rothfels'}]\", 'abstract': '<p>Inferring the true biological sequences from  amplicon mixtures remains a difficult bioinformatic problem. The traditional approach is to cluster  sequencing reads by similarity thresholds and treat the consensus sequence of each cluster as an \"operational taxonomic unit\" (OTU). Recently, this approach has been improved upon by model-based methods that correct PCR and sequencing errors in  order to infer \"amplicon sequence variants\"  (ASVs). To date, ASV approaches have been used primarily in metagenomics, but they are also useful for identifying allelic or paralogous variants and  for determining homeologs in polyploid organisms. To facilitate the usage of ASV methods among polyploidy researchers, we incorporated ASV inference alongside OTU clustering in PURC v2.0, a major update to PURC (Pipeline for Untangling Reticulate Complexes). In addition to preserving original PURC functions, PURC v2.0 allows users to process PacBio CCS/HiFi reads through DADA2 to generate and annotate ASVs for multiplexed data, with outputs including separate alignments for each locus ready for phylogenetic inference. In addition, PURC v2.0 features faster demultiplexing than the original version and has been updated to be compatible with Python 3. In this chapter we present results indicating that PURC v2.0 (using the ASV approach) is more likely to infer the correct biological sequences in comparison to the earlier OTU-based PURC, and describe how to prepare sequencing data, run PURC v2.0 under several different modes, and interpret the output. We  expect that PURC v2.0 will provide biologists with  a method for generating multi-locus \"moderate data\" datasets that are large enough to be phylogenetically informative and small enough for manual curation.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468613', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Live imaging of the co-translational recruitment of XBP1 mRNA to the ER and its processing by diffuse, non-polarized IRE1α', 'authors': \"[{'name': 'Silvia Gómez-Puerta'}, {'name': 'Roberto Ferrero'}, {'name': 'Tobias Hochstoeger'}, {'name': 'Ivan Zubiri'}, {'name': 'Jeffrey A. Chao'}, {'name': 'Tomás Aragón'}, {'name': 'Franka Voigt'}]\", 'abstract': '<p>Endoplasmic reticulum (ER) to nucleus homeostatic signalling, known as the unfolded protein response (UPR), relies on the non-canonical splicing of XBP1 mRNA. The molecular switch that initiates splicing is the oligomerization of the ER stress sensor and UPR endonuclease IRE1α. While IRE1α can form large clusters that have been proposed to function as XBP1 processing centers on the ER, the actual oligomeric state of active IRE1α complexes as well as the targeting mechanism that recruits XBP1 to IRE1α oligomers, remain unknown.</p><p>Here, we used a single molecule imaging approach to directly monitor the recruitment of individual XBP1 transcripts to the ER surface. We confirmed that stable ER association of unspliced XBP1 mRNA is established through HR2-dependent targeting and relies on active translation. In addition, we show that IRE1α-catalyzed splicing mobilizes XBP1 mRNA from the ER membrane in response to ER stress. Surprisingly, we find that XBP1 transcripts are not recruited into large IRE1α clusters, which only assemble upon overexpression of fluorescently-tagged IRE1α during ER stress. Our findings support a model where ribosome-engaged, ER-poised XBP1 mRNA is processed by functional IRE1α assemblies that are homogenously distributed throughout the ER membrane.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468621', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Mobilome-driven segregation of the resistome in biological wastewater treatment', 'authors': \"[{'name': 'Laura de Nies'}, {'name': 'Susheel Bhanu Busi'}, {'name': 'Benoit Josef Kunath'}, {'name': 'Patrick May'}, {'name': 'Paul Wilmes'}]\", 'abstract': '<p>Biological wastewater treatment plants (BWWTP) are considered to be hotspots of evolution and subsequent spread of antimicrobial resistance (AMR). Mobile genetic elements (MGEs) promote the mobilization and dissemination of antimicrobial resistance genes (ARGs) and are thereby critical mediators of AMR within the BWWTP microbial community. At present, it is unclear whether specific AMR categories are differentially disseminated via bacteriophages (phages) or plasmids. To understand the segregation of AMR in relation to MGEs, we analyzed meta-omic (metagenomic, metatranscriptomic and metaproteomic) data systematically collected over 1.5 years from a BWWTP. Our results showed a core group of fifteen AMR categories which were found across all timepoints. Some of these AMR categories were disseminated exclusively (bacitracin) or primarily (aminoglycoside, MLS, sulfonamide) via plasmids or phages (fosfomycin and peptide), whereas others were disseminated equally by both MGEs. Subsequent expression- and protein-level analyses further demonstrated that aminoglycoside, bacitracin and sulfonamide resistance genes were expressed more by plasmids, in contrast to fosfomycin and peptide AMR expression by phages, thereby validating our genomic findings. Longitudinal assessment further underlined these findings whereby the log2-fold changes of aminoglycoside, bacitracin and sulfonamide resistance genes were increased in plasmids, while fosfomycin and peptide resistance showed similar trends in phages. In the analyzed communities, the dominant taxon <italic>Candidatus</italic> Microthrix parvicella was a major contributor to several AMR categories whereby its plasmids primarily mediated aminoglycoside resistance. Importantly, we also found AMR associated with ESKAPEE pathogens within the BWWTP, for which MGEs also contributed differentially to the dissemination of ARGs. Collectively our findings pave the way towards understanding the segmentation of AMR within MGEs, thereby shedding new light on resistome populations and their mediators, essential elements that are of immediate relevance to human health.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469161', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Indirect effects of parental conflict on conspecific offspring development', 'authors': \"[{'name': 'Jenn M Coughlan'}]\", 'abstract': '<p>Hybrid Seed Inviability (HSI) is a common barrier in angiosperms. Recent work suggests that the rapid evolution of HSI may, in part, be due to conflict between maternal and paternal optima for resource allocation to developing offspring (i.e. parental conflict). However, parental conflict requires that paternally-derived resource acquiring alleles impose a maternal cost. I test this requirement using three closely related species in the Mimulus guttatus species complex that exhibit significant HSI and differ in their inferred histories of parental conflict. I show that the presence of hybrid seeds significantly affects conspecific seed size for almost all crosses, such that conspecific seeds are smaller after developing with hybrids from fathers with a stronger history of conflict, and larger after developing with hybrids from fathers with a weaker history of conflict. This work demonstrates a cost of paternally-derived alleles, and also has implications for species fitness in secondary contact.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.15.468614', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Wolbachia endosymbionts in two Anopheles species indicates independent acquisitions and lack of prophage elements', 'authors': \"[{'name': 'Shannon Quek'}, {'name': 'Louise Cerdeira'}, {'name': 'Claire L Jeffries'}, {'name': 'Sean Tomlinson'}, {'name': 'Thomas Walker'}, {'name': 'Grant L. Hughes'}, {'name': 'Eva Heinz'}]\", 'abstract': '<p><italic>Wolbachia</italic> is a genus of obligate bacterial endosymbionts that infect a diverse range of arthropod species as well as filarial nematodes, with its single described species, <italic>Wolbachia pipientis,</italic> divided into several ‘supergroups’ based on multilocus sequence typing. <italic>Wolbachia</italic> strains in mosquitoes have been shown to inhibit the transmission of human pathogens including <italic>Plasmodium</italic> malaria parasites and arboviruses. Despite their large host range, <italic>Wolbachia</italic> strains within the major malaria vectors of the <italic>Anopheles (A.) gambiae</italic> and <italic>A. funestus</italic> complexes appear at low density based solely on PCR-based methods. Questions have been raised as to whether this represents a true endosymbiotic relationship. However, recent definitive evidence for two distinct, high-density strains of supergroup B <italic>Wolbachia</italic> within <italic>A. demeilloni</italic> and <italic>A. moucheti</italic> has opened exciting possibilities to explore naturally occurring <italic>Wolbachia</italic> endosymbionts in <italic>Anopheles</italic> for biocontrol strategies to block <italic>Plasmodium</italic> transmission. Here we utilise genomic analyses to demonstrate that both <italic>Wolbachia</italic> strains have retained all key metabolic and transport pathways despite their smaller genome size. We further confirm the presence of cytoplasmic incompatibility factor genes, despite noticeably few prophage regions. Additionally, phylogenetic analysis indicates that these <italic>Wolbachia</italic> strains may have been introduced into these two <italic>Anopheles</italic> species via horizontal transmission events, and unlikely to be by ancestral acquisition and subsequent loss events in the <italic>Anopheles gambiae</italic> species complex. These are the first <italic>Wolbachia</italic> genomes that enable us to study the relationship between natural strains <italic>Plasmodium</italic> malaria parasites and their <italic>Anopheline</italic> hosts.</p><sec><title>Impact statement</title><p><italic>Wolbachia</italic> naturally infects a wide range of arthropod species, including insect vectors of human pathogens, where they may play a role in inhibiting their replication. These bacteria have been commonly found within <italic>Aedes (Ae.) albopictus</italic> and <italic>Culex pipiens</italic> mosquitoes but have been noticeably absent in the <italic>Anopheles</italic> mosquito genera, which includes all species responsible for malaria transmission. Recent PCR-based methods have suggested the potential for natural <italic>Wolbachia</italic> strains within the <italic>A. gambiae</italic> species complex, which includes major malaria vector species including <italic>A. gambiae</italic> s.s., <italic>A. coluzzii</italic> and <italic>A. arabiensis</italic>. We recently reported the presence of stable <italic>Wolbachia</italic> strains naturally occurring within two different <italic>Anopheles</italic> species (<italic>A. demeilloni</italic> and <italic>A. moucheti)</italic>. In this study, we perform comparative genomic analysis of these two <italic>Wolbachia</italic> genomes against each other and published <italic>Wolbachia</italic> strains. The current assemblies are some of the smallest sequenced <italic>Wolbachia</italic> strains of insects, although their metabolic pathway repertoire is comparable to other strains. Interestingly, prophage fragments were identified within only one of the two strains. The findings of this study will be of significant interest to researchers investigating <italic>Wolbachia</italic> as a potential malaria biocontrol strategy, giving greater insight into the evolution and diversity of this obligate intracellular endosymbiont.</p></sec><sec><title>Data summary</title><p>Sequence data generated and used for this analysis are available in the National Centre for Biotechnology Information Sequence Read Archive (NCBI SRA bioproject number PRJNA642000). The two assembled <italic>Wolbachia</italic> genomes are available with genome accession numbers GCA_018491735.2 and GCA_018491625.2. Additional <italic>Wolbachia</italic> genomes used for comparative analysis are described in the supplementary material.</p><p>The authors confirm all supporting data, code and protocols have been provided within the article or through supplementary data files. Additional supplementary data files used to generate several figures can be found at: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://figshare.com/projects/Wolbachia_endosymbionts_in_two_Anopheles_species_indicates_independent_acquisitions_and_lack_of_prophage_elements/126533\">https://figshare.com/projects/Wolbachia_endosymbionts_in_two_Anopheles_species_indicates_independent_acquisitions_and_lack_of_prophage_elements/126533</ext-link></p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.16.468548', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Reconstitution of Spiroplasma swimming by expressing two bacterial actins in synthetic minimal bacterium', 'authors': \"[{'name': 'Hana Kiyama'}, {'name': 'Shigeyuki Kakizawa'}, {'name': 'Yuya Sasajima'}, {'name': 'Yuhei O Tahara'}, {'name': 'Makoto Miyata'}]\", 'abstract': '<p>Motility is one of the most important features of life, but its evolutionary origin is still unknown. Here, we focus on <italic>Spiroplasma</italic>, commensal or parasitic bacteria. They swim by the helicity switching of a ribbon-like cytoskeleton composed of six proteins each evolved from a nucleosidase and a bacterial actin called MreB. We expressed these proteins in a synthetic minimal bacterium, JCVI-syn3B whose genome was computer-designed and chemically synthesized. The synthetic bacterium showed swimming motility with the features common with <italic>Spiroplasma</italic> swimming. Moreover, some combinations of two proteins showed helical cell shape and swimming, suggesting that the swimming was originated from differentiation and coupling of the bacterial actin.</p><sec><title>One-Sentence Summary</title><p>Expression of two bacterial actins gave cell helicity and swimming to a synthetic minimal cell.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.12.468374', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Full protection against all four SARS-CoV-2 variants of concern (VOC) in hamsters requires revision of spike antigen used for vaccination', 'authors': \"[{'name': 'Sapna Sharma'}, {'name': 'Thomas Vercruysse'}, {'name': 'Lorena Sanchez-Felipe'}, {'name': 'Winnie Kerstens'}, {'name': 'Rana Abdelnabi'}, {'name': 'Caroline Foo'}, {'name': 'Viktor Lemmens'}, {'name': 'Dominique Van Looveren'}, {'name': 'Piet Maes'}, {'name': 'Guy Baele'}, {'name': 'Birgit Weynand'}, {'name': 'Philippe Lemey'}, {'name': 'Johan Neyts'}, {'name': 'Hendrik Jan Thibaut'}, {'name': 'Kai Dallmeier'}]\", 'abstract': '<p>Current licensed COVID-19 vaccines are based on antigen sequences of initial SARS-CoV-2 isolates that emerged in 2019. By mid 2021 these historical virus strains have been completely replaced by new cosmopolitan SARS-CoV-2 lineages. The ongoing pandemic has been further driven by emerging variants of concern (VOC) Alpha, Beta, Gamma and, lately predominant, Delta. These are characterized by an increased transmissibility and possible escape from naturally acquired or vaccine-induced immunity. We here show, using a YF17D-vectored first-generation COVID-19 vaccine (Sanchez-Felipe et al., 2021) and a stringent hamster challenge model (Abdelnabi et al., 2021) that the immunity elicited by a prototypic spike antigen is insufficient to provide optimal protection against the Beta VoC, urging for an antigenic update. We therefore designed an updated second-generation vaccine candidate that carries the sequence of a spike antigen that includes crucial epitopes from multiple VOCs. This vaccine candidate yielded a marked change in target antigen spectrum covered as demonstrated by (<italic>i</italic>) antigenic cartography and (<italic>ii</italic>) full protection against infection and virus-induced disease caused by any of the four VOCs (Alpha, Beta, Gamma and Delta) used for challenge. This more universal COVID-19 vaccine candidate also efficiently blocked direct transmission of VOC Delta from vaccinated infected hamsters to non-vaccinated sentinels under prolonged co-housing conditions. In conclusion, our data suggest that current first-generation COVID-19 vaccines need to be adapted to cover emerging sequence diversity of VOC to preserve vaccine efficacy and to contain virus spread at the community level.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.16.467908', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Tuning spherical cells into kinking helices in wall-less bacteria', 'authors': \"[{'name': 'Carole Lartigue'}, {'name': 'Bastien Lambert'}, {'name': 'Fabien Rideau'}, {'name': 'Marion Decossas'}, {'name': 'Mélanie Hillion'}, {'name': 'Jean-Paul Douliez'}, {'name': 'Julie Hardouin'}, {'name': 'Olivier Lambert'}, {'name': 'Alain Blanchard'}, {'name': 'Laure Béven'}]\", 'abstract': '<p>In bacteria, cell shape is determined and maintained through a complex interplay between the peptidoglycan cell wall and cytoplasmic filaments made of polymerized MreB. <italic>Spiroplasma</italic> species, members of the Mollicutes class, challenge this general understanding because they are characterized by a helical cell shape and motility without a cell wall. This specificity is thought to rely on five MreB isoforms and a specific fibril protein. In this study, combinations of these five MreBs and of the fibril from <italic>Spiroplasma citri</italic> were expressed in another Mollicutes, <italic>Mycoplasma capricolum. Mycoplasma</italic> cells that were initially pleomorphic, mostly spherical, turned into helices when MreBs and fibrils were expressed in this heterologous host. The fibril protein was essential neither for helicity nor for cell movements. The isoform MreB5 had a special role as it was sufficient to confer helicity and motility to the mycoplasma cells. Cryo-electron microscopy confirmed the association of MreBs and fibril-based cytoskeleton with the plasma membrane, suggesting a direct effect on the membrane curvature. Finally, the heterologous expression of these proteins, MreBs and fibril, made it possible to reproduce the kink-like motility of spiroplasmas without providing the ability of cell movement in liquid broth. We suggest that other <italic>Spiroplasma</italic> components, not yet identified, are required for swimming, a hypothesis that could be evaluated in future studies using the same model.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.17.469020', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Effects of crude oil on juvenile threespine stickleback somatic and immune development', 'authors': \"[{'name': 'Kelly S Ireland'}, {'name': 'Kathryn Milligan-Myhre'}]\", 'abstract': '<p>Aquatic oil spills have resounding effects on surrounding ecosystems, and thus significant resources are committed to oil spill responses to remove the oil from the environment as quickly as possible. Oil has immunotoxic effects and may be particularly harmful to larval and juvenile fish as it can cause a number of developmental defects and stunt growth. In spite of significant efforts to clean oil, it is unclear whether larval and juvenile fish can recover from the effects of oil and no work has been done on the effect crude oil has on developing threespine stickleback (Gasterosteus aculeatus) fish. Threespine stickleback are a ubiquitous sentinel species in the northern hemisphere and are an important food source for many larger, economically valuable fish. As fish with fully marine, anadromous, and freshwater populations, stickleback are exposed to oil in a variety of aquatic environments. We hypothesized that oil exposure would suppress both growth and immunity of developing stickleback, but that fish health could be recovered by removal of the crude oil. Fish were exposed to Alaska North Slope crude oil and then were moved to water without crude oil for two weeks (depuration).  Measurements of growth and immunity were taken before and after the depuration. We found that crude oil effected different developmental pathways independently, significantly impacting some but not others. This is the first study to examine the effect crude oil has on early stages of stickleback development, and that stickleback fish are unable to recover from exposure after being transferred to clean water for two-weeks, suggesting larval/juvenile stickleback exposed to crude oil need longer than two-weeks to recover if they are able to recover at all.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.19.469182', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'TK216 targets microtubules in Ewing sarcoma cells', 'authors': \"[{'name': 'Juan Manuel Povedano Selfa'}, {'name': 'Vicky Li'}, {'name': 'Katherine E. Lake'}, {'name': 'Xin Bai'}, {'name': 'Rameshu Rallabandi'}, {'name': 'Jiwoong Kim'}, {'name': 'Yang Xie'}, {'name': 'Jef K. De Brabander'}, {'name': 'David G. McFadden'}]\", 'abstract': '<p>Ewing sarcoma (EWS) is a pediatric malignancy driven by the EWSR1-FLI1 fusion protein formed by the chromosomal translocation t(11;22).  The small molecule TK216 was developed as a first-in-class direct EWSR1-FLI1 inhibitor and is in phase II clinical trials in combination with vincristine for EWS patients. However, TK216 exhibits anti-cancer activity against cancer cell lines and xenografts that do not express EWSR1-FLI1, and the mechanism underlying cytotoxicity remains unresolved. We apply a forward genetics screening platform utilizing engineered hypermutation in EWS cell lines and identify recurrent mutations in TUBA1B, encoding α-tubulin, that prove sufficient to drive resistance to TK216.  Using reconstituted microtubule (MT) polymerization in vitro and cell-based chemical probe competition assays, we demonstrate that TK216 acts as an MT destabilizing agent. This work defines the mechanism of cytotoxicity of TK216, explains the synergy observed with vincristine, and calls for a reexamination of ongoing clinical trials with TK216.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468615', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Using OPM-MEG in contrasting magnetic environments', 'authors': \"[{'name': 'Ryan M. Hill'}, {'name': 'Jasen Devasagayam'}, {'name': 'Niall Holmes'}, {'name': 'Elena Boto'}, {'name': 'Vishal Shah'}, {'name': 'James Osborne'}, {'name': 'Kristina Safar'}, {'name': 'Frank Worcester'}, {'name': 'Christopher Mariani'}, {'name': 'Eliot Dawson'}, {'name': 'David Woolger'}, {'name': 'Richard Bowtell'}, {'name': 'Margot J. Taylor'}, {'name': 'Matthew J. Brookes'}]\", 'abstract': '<p>Magnetoencephalography (MEG) has been revolutionised in recent years by optically pumped magnetometers (OPMs). “OPM-MEG” offers higher sensitivity, better spatial resolution and lower cost than conventional instrumentation based on superconducting quantum interference devices (SQUIDS). Moreover, OPMs offer the possibility of motion robustness and lifespan compliance, dramatically expanding the range of MEG applications. However, OPM-MEG remains nascent technology; it places stringent requirements on magnetic shielding, and whilst a number of viable systems exist, most are custom made and there have been no cross-site investigations showing the reliability of data. In this paper, we undertake the first cross-site OPM-MEG comparison, using near identical commercial systems scanning the same participant. The two sites are deliberately contrasting, with different magnetic environments: a “green field” campus university site with an OPM-optimised shielded room (low interference) and a city centre hospital site with a “standard” (non-optimised) MSR (high interference). We show that despite a 25-fold difference in background field, and a 30-fold difference in low frequency interference, using dynamic field control and software-based suppression of interference we can generate comparable noise floors at both sites. In human data recorded during a visuo-motor task and a face processing paradigm, we were able to generate similar data, with source localisation showing that brain regions could be pinpointed with just ~10 mm spatial discrepancy and temporal correlations of &gt; 80%. Overall, our study demonstrates that “plug- and-play” OPM-MEG systems exist and can be sited even in challenging magnetic environments.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.18.469186', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'TCR-BERT: learning the grammar of T-cell receptors for flexible antigen- binding analyses', 'authors': \"[{'name': 'Kevin E. Wu'}, {'name': 'Kathryn E. Yost'}, {'name': 'Bence Daniel'}, {'name': 'Julia A Belk'}, {'name': 'Yu Xia'}, {'name': 'Takeshi Egawa'}, {'name': 'Ansuman Satpathy'}, {'name': 'Howard Chang'}, {'name': 'James Zou'}]\", 'abstract': \"<p>The T-cell receptor (TCR) allows T-cells to recognize and respond to antigens presented by infected and diseased cells. However, due to TCRs' staggering diversity and the complex binding dynamics underlying TCR antigen recognition, it is challenging to predict which antigens a given TCR may bind to. Here, we present TCR-BERT, a deep learning model that applies self-supervised transfer learning to this problem. TCR-BERT leverages unlabeled TCR sequences to learn a general, versatile representation of TCR sequences, enabling numerous downstream applications. We demonstrate that TCR-BERT can be used to build state-of-the-art TCR-antigen binding predictors with improved generalizability compared to prior methods. TCR-BERT simultaneously facilitates clustering sequences likely to share antigen specificities. It also facilitates computational approaches to challenging, unsolved problems such as designing novel TCR sequences with engineered binding affinities. Importantly, TCR-BERT enables all these advances by focusing on residues with known biological significance. TCR-BERT can be a useful tool for T-cell scientists, enabling greater understanding and more diverse applications, and provides a conceptual framework for leveraging unlabeled data to improve machine learning on biological sequences.</p>\", 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.15.468694', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structure and functionality of a multimeric human COQ7:COQ9 complex', 'authors': \"[{'name': 'Mateusz Manicki'}, {'name': 'Halil Aydin'}, {'name': 'Luciano A. Abriata'}, {'name': 'Katherine A. Overmyer'}, {'name': 'Rachel M. Guerra'}, {'name': 'Joshua J. Coon'}, {'name': 'Matteo Dal Peraro'}, {'name': 'Adam Frost'}, {'name': 'David J. Pagliarini'}]\", 'abstract': '<p>Coenzyme Q (CoQ, ubiquinone) is a redox-active lipid essential for core metabolic pathways and antioxidant defense. CoQ is synthesized upon the mitochondrial inner membrane by an ill-defined ‘complex Q’ metabolon. Here we present a structure and functional analyses of a substrate- and NADH-bound oligomeric complex comprised of two complex Q subunits: the hydroxylase COQ7, which performs the penultimate step in CoQ biosynthesis, and the prenyl lipid-binding protein COQ9. We reveal that COQ7 adopts a modified ferritin-like fold with an extended hydrophobic access channel whose substrate binding capacity is enhanced by COQ9. Using molecular dynamics simulations, we further show that two COQ7:COQ9 heterodimers form a curved tetramer that deforms the membrane, potentially opening a pathway for CoQ intermediates to translocate from within the bilayer to the proteins’ lipid-binding sites. Two such tetramers assemble into a soluble octamer, closed like a capsid, with lipids captured within. Together, these observations indicate that COQ7 and COQ9 cooperate to access hydrophobic precursors and coordinate subsequent synthesis steps toward producing mature CoQ.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.468982', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Activating silent glycolysis bypasses in Escherichia coli', 'authors': \"[{'name': 'Camillo Iacometti'}, {'name': 'Katharina Marx'}, {'name': 'Maria Hönick'}, {'name': 'Viktoria Biletskaia'}, {'name': 'Helena Schulz-Mirbach'}, {'name': 'Ari Satanowski'}, {'name': 'Beau Dronsella'}, {'name': 'Valérie A. Delmas'}, {'name': 'Anne Berger'}, {'name': 'Ivan Dubois'}, {'name': 'Madeleine Bouzon'}, {'name': 'Volker Döring'}, {'name': 'Elad Noor'}, {'name': 'Arren Bar-Even'}, {'name': 'Steffen N. Lindner'}]\", 'abstract': '<p>All living organisms share similar reactions within their central metabolism to provide precursors for all essential building blocks and reducing power. To identify whether alternative metabolic routes of glycolysis can operate in <italic>E. coli</italic>, we complementarily employed <italic>in silico</italic> design, rational engineering, and adaptive laboratory evolution. First, we used a genome-scale model and identified two potential pathways within the metabolic network of this organism replacing canonical Embden-Meyerhof-Parnas (EMP) glycolysis to convert phosphosugars into organic acids. One of these glycolytic routes proceeds via methylglyoxal, the other via serine biosynthesis and degradation. Then, we implemented both pathways in <italic>E. coli</italic> strains harboring defective EMP glycolysis. Surprisingly, the pathway via methylglyoxal immediately operated in a triosephosphate isomerase deletion strain cultivated on glycerol. By contrast, in a phosphoglycerate kinase deletion strain, the overexpression of methylglyoxal synthase was necessary for implementing a functional methylglyoxal pathway. Furthermore, we engineered the ‘serine shunt’ which converts 3-phosphoglycerate via serine biosynthesis and degradation to pyruvate, bypassing an enolase deletion. Finally, to explore which of these alternatives would emerge by natural selection we performed an adaptive laboratory evolution study using an enolase deletion strain. The evolved mutants were shown to use the serine shunt. Our study reveals the flexible repurposing of metabolic pathways to create new metabolite links and rewire central metabolism.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.17.468975', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Recognition and deubiquitination of free 40S for translational reset by Otu2', 'authors': \"[{'name': 'Ken Ikeuchi'}, {'name': 'Nives Ivic'}, {'name': 'Jingdong Cheng'}, {'name': 'Robert Buschauer'}, {'name': 'Yoshitaka Matsuo'}, {'name': 'Thomas Fröhlich'}, {'name': 'Otto Berninghausen'}, {'name': 'Toshifumi Inada'}, {'name': 'Thomas Becker'}, {'name': 'Roland Beckmann'}]\", 'abstract': '<p>In actively translating 80S ribosomes the ribosomal protein eS7 of the 40S subunit is monoubiquitinated by the E3 ligase Not4<sup>1,2</sup> and deubiquitinated by the deubiquitination enzyme Otu2 upon ribosomal subunit recycling<sup>3</sup>. Despite its importance for general efficiency of translation the exact role and structural basis for this specific translational reset are only poorly understood. Here we present biochemical and structural data showing that Otu2 can engage the recycled 40S subunit together with the recycling factors ABCE1 and Tma64 immediately after 60S dissociation for mRNA recycling, and that it dissociates before 48S initiation complex formation. A combined structural analysis of Otu2 and Otu2-40S complexes by X-ray crystallography, AlphaFold2 prediction<sup>4</sup> and cryo-EM revealed how Otu2 can specifically be recruited to the 40S, but not to the 80S ribosome, for removal of the eS7-bound ubiquitin moiety. Here, interactions of the largely helical N-terminal domain of Otu2 to sites that are masked and therefore inaccessible in the 80S ribosome are of crucial importance. Collectively, we provide the structural basis for the Otu2 driven deubiquitination step providing a first mechanistic understanding of this translational reset step during ribosome recycling/(re)initiation.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.17.21263608', 'tweets': 18, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Reduced Incidence of Long-COVID Symptoms Related to Administration of COVID-19 Vaccines Both Before COVID-19 Diagnosis and Up to 12 Weeks After', 'authors': \"[{'name': 'Michael A. Simon'}, {'name': 'Ryan D. Luginbuhl'}, {'name': 'Richard Parker'}]\", 'abstract': '<p>Both clinical trials and studies leveraging real-world data have repeatedly confirmed the three COVID-19 vaccines authorized for use by the Food and Drug Administration are safe and effective at preventing infection, hospitalization, and death due to COVID-19 and a recent observational study of self-reported symptoms provides support that vaccination may also reduce the probability of developing long-COVID. As part of a federated research study with the COVID-19 Patient Recovery Alliance, Arcadia.io performed a retrospective analysis of the medical history of 240,648 COVID-19-infected persons to identity factors influencing the development and progression of long-COVID. This analysis revealed that patients who received at least one dose of any of the three COVID vaccines prior to their diagnosis with COVID-19 were 7-10 times less likely to report two or more long-COVID symptoms compared to unvaccinated patients. Furthermore, unvaccinated patients who received their first COVID-19 vaccination within four weeks of SARS-CoV-2 infection were 4-6 times less likely to report multiple long-COVID symptoms, and those who received their first dose 4-8 weeks after diagnosis were 3 times less likely to report multiple long-COVID symptoms compared to those who remained unvaccinated. This relationship supports the hypothesis that COVID-19 vaccination is protective against long-COVID and that effect persists even if vaccination occurs up to 12 weeks after COVID-19 diagnosis. A critical objective of this study was hypothesis generation, and the authors intend to perform further studies to substantiate the findings and encourage other researchers to as well.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.14.21266294', 'tweets': 18, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Inactivated virus vaccine BBV152/Covaxin elicits robust cellular immune memory to SARS-CoV-2 and variants of concern', 'authors': \"[{'name': 'Rajesh Vikkurthi'}, {'name': 'Asgar Ansari'}, {'name': 'Anupama R Pai'}, {'name': 'Someshwar Nath Jha'}, {'name': 'Shilpa Sachan'}, {'name': 'Suvechchha Pandit'}, {'name': 'Bhushan Nikam'}, {'name': 'Anurag Kalia'}, {'name': 'Bimal Prasad Jit'}, {'name': 'Hilal Ahmad Parray'}, {'name': 'Savita Singh'}, {'name': 'Pallavi Kshetrapal'}, {'name': 'Nitya Wadhwa'}, {'name': 'Tripti Shrivastava'}, {'name': 'Poonam Coshic'}, {'name': 'Suresh Kumar'}, {'name': 'Pragya Sharma'}, {'name': 'Nandini Sharma'}, {'name': 'Juhi Taneja'}, {'name': 'Anil K Pandey'}, {'name': 'Ashok Sharma'}, {'name': 'Ramachandran Thiruvengadam'}, {'name': 'Alba Grifoni'}, {'name': 'Shinjini Bhatnagar'}, {'name': 'Daniela Weiskopf'}, {'name': 'Alessandro Sette'}, {'name': 'Pramod Kumar Garg'}, {'name': 'Nimesh Gupta'}]\", 'abstract': '<p>The characteristics of immune memory established in response to inactivated SARS-CoV-2 vaccines remains unclear. We determined the magnitude, quality and persistence of cellular and humoral memory responses up to 6 months after vaccination with BBV152/Covaxin. Here, we show that the quantity of vaccine-induced spike- and nucleoprotein-antibodies is comparable to that following natural infection and the antibodies are detectable up to 6 months. The RBD-specific antibodies decline in the range of 3 to 10-fold against the SARS-CoV-2 variants in the order of alpha (B.1.1.7) &gt; delta (B.1.617.2) &gt; beta (B.1.351), with no observed impact of gamma (P.1) and kappa (B.1.617.1) variant. We found that the vaccine induces memory B cells, similar to natural infection, which are impacted by virus variants in the same order as antibodies. The vaccine further induced antigen-specific functionally potent multi-cytokine expressing CD4<sup>+</sup> T cells in ∼85% of the subjects, targeting spike and nucleoprotein of SARS-CoV-2. Marginal ∼1.3 fold-reduction was observed in vaccine-induced CD4<sup>+</sup> T cells against the beta variant, with no significant impact of the alpha and the delta variants. The antigen-specific CD4<sup>+</sup> T cells were populated in the central memory compartment and persisted up to 6 months of vaccination. Importantly the vaccine generated Tfh cells that are endowed with B cell help potential, similar to the Tfh cells induced after natural infection. Altogether, these findings establish that the inactivated virus vaccine BBV152 induces robust immune memory to SARS-CoV-2 and variants of concern, which persist for at least 6 months after vaccination. This study provides insight into the attributes of BBV152-elicited immune memory, and has implication for future vaccine development, guidance for use of inactivated virus vaccine, and booster immunization.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.17.468248', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Broadly-neutralizing antibodies that bind to the influenza hemagglutinin stalk domain enhance the effectiveness of neuraminidase inhibitors via Fc-mediated effector functions', 'authors': '[{\\'name\\': \\'Ali Zhang\\'}, {\\'name\\': \\'Hanu Chaudhari\\'}, {\\'name\\': \\'Yonathan Agung\\'}, {\\'name\\': \"Michael D\\'Agostino\"}, {\\'name\\': \\'Jann Ang\\'}, {\\'name\\': \\'Matthew S Miller\\'}]', 'abstract': \"<p>The conserved hemagglutinin stalk domain is an attractive target for broadly effective antibody-based therapeutics and next generation universal influenza vaccines. Protection provided by hemagglutinin stalk binding antibodies is principally mediated through activation of immune effector cells. Titers of stalk-binding antibodies are highly variable on an individual level, and tend to increase with age as a result of increasing exposures to influenza virus. In our study, we show that stalk-binding antibodies cooperate with neuraminidase inhibitors to protect against influenza virus infection in an Fc-dependent manner. These data suggest that the effectiveness of neuraminidase inhibitors is likely influenced by an individual's titers of stalk-binding antibodies, and that neuraminidase inhibitors may enhance the effectiveness of future stalk-binding monoclonal antibody-based treatments.</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468866', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Brain state and cortical layer-specific mechanisms underlying perception at threshold', 'authors': \"[{'name': 'Mitchell P. Morton'}, {'name': 'Sachira Denagamage'}, {'name': 'Isabel J. Blume'}, {'name': 'John H. Reynolds'}, {'name': 'Monika P. Jadi'}, {'name': 'Anirvan S. Nandy'}]\", 'abstract': '<p>Identical stimuli can be perceived or go unnoticed across successive presentations, producing divergent behavioral readouts despite similarities in sensory input. We hypothesized that fluctuations in neurophysiological states in the sensory neocortex, which could alter cortical processing at the level of neural subpopulations, underlies this perceptual variability. We analyzed cortical layer-specific electrophysiological activity in visual area V4 during a cued attention task. We find that hit trials are characterized by a larger pupil diameter and lower incidence of microsaccades, indicative of a behavioral state with increased arousal and perceptual stability. Target stimuli presented at perceptual threshold evoke elevated multi-unit activity in V4 neurons in hit trials compared to miss trials, across all cortical layers. Putative excitatory and inhibitory neurons are strongly positively modulated in the input (IV) and deep (V &amp; VI) layers of the cortex during hit trials. Excitatory neurons in the superficial cortical layers exhibit lower variability in hit trials. Deep layer neurons are less phase-locked to low frequency rhythms in hits. Hits are also characterized by greater interlaminar coherence between the superficial and deep layers in the pre-stimulus period, and a complementary pattern between the input layer and both the superficial and deep layers in the stimulus-evoked period. Taken together, these results indicate that a state of elevated levels of arousal and perceptual stability allow enhanced processing of sensory stimuli, which contributes to hits at perceptual threshold.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.17.469047', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Awake ripples enhance emotional memory encoding in the human brain', 'authors': \"[{'name': 'Haoxin Zhang'}, {'name': 'Ivan Skelin'}, {'name': 'Shiting Ma'}, {'name': 'Michelle Paff'}, {'name': 'Michael A Yassa'}, {'name': 'Robert T Knight'}, {'name': 'Jack J Lin'}]\", 'abstract': '<p>Intracranial recordings from the human amygdala and the hippocampus during an emotional memory encoding and discrimination task reveal increased awake sharp-wave/ripples (aSWR) after encoding of emotional compared to neutral stimuli. Further, post-encoding aSWR-locked memory reinstatement in the amygdala and the hippocampus was predictive of later memory discrimination. These findings provide electrophysiological evidence that post-encoding aSWRs enhance memory for emotional events.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468709', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Recent zoonotic spillover and tropism shift of a Canine Coronavirus is associated with relaxed selection and putative loss of function in NTD subdomain of spike protein', 'authors': \"[{'name': 'Jordan D. Zehr'}, {'name': 'Sergei L. Kosakovsky Pond'}, {'name': 'Darren P. Martin'}, {'name': 'Kristina Ceres'}, {'name': 'Gary R. Whittaker'}, {'name': 'Laura B. Goodman'}, {'name': 'Michael J. Stanhope'}]\", 'abstract': '<p>A recent study reported the occurrence of Canine Coronavirus (CCoV) in nasopharyngeal swabs from a small number of patients hospitalized with pneumonia during a 2017-18 period in Sarawak, Malaysia. Because the genome sequence for one of these isolates is available, we conducted comparative evolutionary analyses of the spike gene of this strain (CCoV-HuPn-2018), with other available <italic>Alphacoronavirus</italic> 1 spike sequences. The most N-terminus subdomain (0-domain) of the CCoV-HuPn-2018 spike protein has sequence similarity to Transmissible Gastroenteritis Virus (TGEV) and CCoV2b strains, but not to other members of the type II Alphacoronaviruses (i.e., CCoV2a and Feline CoV2-FCoV2). This 0-domain in CCoV-HuPn-2018 has evidence for relaxed selection pressure, an increased rate of molecular evolution, and a number of unique amino acid substitutions relative to CCoV2b and TGEV sequences. A region of the 0-domain determined to be key to sialic acid binding and pathogenesis in TGEV had clear differences in amino acid sequences in CCoV-HuPn-2018 relative to both CCoV2b (enteric) and TGEV (enteric and respiratory). The 0-domain of CCoV-HuPn-2018 also had several sites inferred to be under positive diversifying selection, including sites within the signal peptide. Downstream of the 0-domain, FCoV2 shared sequence similarity to the CCoV2b and TGEV sequences, with analyses of this larger alignment identifying positively selected sites in the putative Receptor Binding Domain (RBD) and Connector Domain (CD). Recombination analyses strongly implicated a particular FCoV2 strain in the recombinant history of CCoV-HuPn-2018 with molecular divergence times estimated at around 60 years ago. We hypothesize that CCoV-HuPn-2018 had an enteric origin, but that it has lost that particular tropism, because of mutations in the sialic acid binding region of the spike 0-domain. As selection pressure on this region was reduced, the virus evolved a respiratory tropism, analogous to other <italic>Alphacoronavirus</italic> 1, such as Porcine Respiratory Coronavirus (PRCV), that have lost this region entirely. We also suggest that signals of positive selection in the signal peptide as well as other changes in the 0-domain of CCoV-HuPn-2018 could represent an adaptive role in this new host and that this could be in part due to the different spatial distribution of the N-linked glycan repertoire for this strain.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.18.469017', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'ZARP: An automated workflow for processing of RNA-seq data', 'authors': \"[{'name': 'Maria Katsantoni'}, {'name': 'Foivos Gypas'}, {'name': 'Christina J. Herrmann'}, {'name': 'Dominik Burri'}, {'name': 'Maciej Bak'}, {'name': 'Paula Iborra'}, {'name': 'Krish Agarwal'}, {'name': 'Meric Ataman'}, {'name': 'Anastasiya Boersch'}, {'name': 'Mihaela Zavolan'}, {'name': 'Alexander Kanitz'}]\", 'abstract': '<p>RNA sequencing (RNA-seq) is a crucial technique for many scientific studies and multiple models, and software packages have been developed for the processing and analysis of such data. Given the plethora of available tools, choosing the most appropriate ones is a time-consuming process that requires an in-depth understanding of the data, as well as of the principles and parameters of each tool. In addition, packages designed for individual tasks are developed in different programming languages and have dependencies of various degrees of complexity, which renders their installation and execution challenging for users with limited computational expertise. The use of workflow languages and execution engines with support for virtualization and encapsulation options such as containers and Conda environments facilitates these tasks considerably. Computational workflows defined in those languages can be reliably shared with the scientific community, enhancing reusability, while improving reproducibility of results by making individual analysis steps more transparent. Here we present ZARP, a general purpose RNA-seq analysis workflow which builds on state-of-the-art software in the field to facilitate the analysis of RNA-seq data sets. ZARP is developed in the Snakemake workflow language using best software development practices. It can run locally or in a cluster environment, generating extensive reports not only of the data but also of the options utilized. It is built using modern technologies with the ultimate goal to reduce the hands-on time for bioinformaticians and non-expert users. ZARP is available under a permissive Open Source license and open to contributions by the scientific community.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468996', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Phylogenomic analyses using genomes and transcriptomes do not resolve relationships among major clades in Phrymaceae', 'authors': \"[{'name': 'Diego F. Morales-Briones'}, {'name': 'Nan Lin'}, {'name': 'Eileen Y. Huang'}, {'name': 'Dena L. Grossenbacher'}, {'name': 'James M. Sobel'}, {'name': 'Caroline D. Gilmore'}, {'name': 'David C. Tank'}, {'name': 'Ya Yang'}]\", 'abstract': '<p>Premise of the study: Phylogenomic datasets using genomes and transcriptomes provide rich opportunities beyond resolving bifurcating phylogenetic relationships. Monkeyflower (Phrymaceae) is a model system for evolutionary ecology. However, it lacks a well-supported phylogeny for a stable taxonomy and for macroevolutionary comparisons.\\nMethods: We sampled 24 genomes and transcriptomes in Phrymaceae and closely related families, including eight newly sequenced transcriptomes. We reconstructed the phylogeny using IQ-TREE and ASTRAL, evaluated gene tree discordance using PhyParts, Quartet Sampling, and cloudogram, and carried out phylogenetic network analyses using PhyloNet and HyDe. We searched for whole genome duplication (WGD) events using chromosome numbers, synonymous distance, and gene duplication events. \\nKey results: Most gene trees support the monophyly of Phrymaceae and each of its tribes. Most gene trees also support the tribe Mimuleae being sister to Phrymeae + Diplaceae + Leucocarpeae, with extensive gene tree discordance among the latter three. Despite the discordance, polyphyly of Mimulus s.l. is strongly supported, and no particular reticulation event among the Phrymaceae tribes is well supported. Reticulation likely occurred among Erythranthe bicolor and close relatives. No ancient WGD event was detected in Phrymaceae. Instead, small-scale duplications are among potential drivers of macroevolutionary diversification of Phrymaceae.\\nConclusions: We show that analysis of reticulate evolution is sensitive to taxon sampling and methods used. We also demonstrate that genome-scale data do not always fully \"resolve\" phylogenetic relationships. They present rich opportunities to investigate reticulate evolution, and gene and genome evolution involved in lineage diversification and adaptation.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468963', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Epoxidized graphene grid for high-throughput high-resolution cryoEM structural analysis', 'authors': \"[{'name': 'Junso Fujita'}, {'name': 'Fumiaki Makino'}, {'name': 'Haruyasu Asahara'}, {'name': 'Maiko Moriguchi'}, {'name': 'Shota Kumano'}, {'name': 'Itsuki Anzai'}, {'name': 'Jun-ichi Kishikawa'}, {'name': 'Yoshiharu Matsuura'}, {'name': 'Takayuki Kato'}, {'name': 'Keiichi Namba'}, {'name': 'Tsuyoshi Inoue'}]\", 'abstract': '<p>Many specimens suffer from low particle density and/or preferred orientation in cryoEM specimen grid preparation, making data collection and structure determination time consuming. We developed an epoxidized graphene grid (EG-grid) that effectively immobilizes protein particles by applying an oxidation reaction using photoactivated ClO<sub>2</sub><sup>•</sup> and further chemical modification. The particle density and orientation distribution are both dramatically improved, having enabled us to reconstruct the density map of GroEL and glyceraldehyde 3-phosphate dehydrogenase (GAPDH), at 1.99 and 2.16 Å resolution from only 504 and 241 micrographs, respectively. A low concentration sample solution of 0.1 mg ml<sup>−1</sup> was sufficient to reconstruct a 3.10 Å resolution density map of SARS-CoV-2 spike protein from 1,163 micrographs. The density maps of V<sub>1</sub>-ATPase, β-galactosidase, and apoferritin were also reconstructed at 3.03, 1.81, and 1.29 Å resolution, respectively. These results indicate that the EG-grid will be a powerful tool for high-throughput cryoEM data collection to accelerate high-resolution structural analysis of biological macromolecules.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.16.468906', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Dithiothreitol causes toxicity in C. elegans by modulating the methionine-homocysteine cycle', 'authors': \"[{'name': 'G Gokul'}, {'name': 'Jogender Singh'}]\", 'abstract': '<p>The redox reagent dithiothreitol (DTT) causes stress in the endoplasmic reticulum (ER) by disrupting its oxidative protein folding environment, which results in the accumulation and misfolding of the newly synthesized proteins. DTT may potentially impact cellular physiology by ER-independent mechanisms; however, such mechanisms remain poorly characterized. Using the nematode model <italic>Caenorhabditis elegans</italic>, here we show that DTT toxicity is modulated by the bacterial diet. Specifically, the dietary component vitamin B12 alleviates DTT toxicity in a methionine synthase-dependent manner. Using a forward genetic screen, we identify that loss-of-function of <italic>R08E5</italic>.<italic>3</italic>, an S-adenosylmethionine (SAM)-dependent methyltransferase, imparts resistance to DTT. DTT upregulates <italic>R08E5</italic>.<italic>3</italic> expression and modulates the activity of the methionine-homocysteine cycle. Employing genetic studies, we show that DTT toxicity is a result of the depletion of SAM. Finally, we show that a functional IRE-1/XBP-1 unfolded protein response pathway is required to counteract toxicity at high, but not low, DTT concentrations.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.15.468669', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Spatially regulated protease activity in lymph nodes renders B cell follicles a sanctuary for retention of intact antigens', 'authors': \"[{'name': 'Aereas Aung'}, {'name': 'Ang Cui'}, {'name': 'Ava P. Soleimany'}, {'name': 'Maurice Bukenya'}, {'name': 'Heya Lee'}, {'name': 'Christopher A. Cottrell'}, {'name': 'Murillo Silva'}, {'name': 'Jesse D. Kirkpatrick'}, {'name': 'Parastoo Amlashi'}, {'name': 'Tanaka Remba'}, {'name': 'Shuhao Xiao'}, {'name': 'Leah Michelle Froehle'}, {'name': 'Wuhbet Abraham'}, {'name': 'Josetta Adams'}, {'name': 'Heikyung Suh'}, {'name': 'Phillip Huyett'}, {'name': 'Douglas S. Kwon'}, {'name': 'Nir Hacohen'}, {'name': 'William R. Schief'}, {'name': 'Sangeeta N. Bhatia'}, {'name': 'Darrell J. Irvine'}]\", 'abstract': '<p>The structural integrity of vaccine antigens is critical, because antigen breakdown <italic>in vivo</italic> could eliminate neutralizing epitopes and create competing B cell responses against irrelevant breakdown products. Using FRET imaging and imaging zymography, we found that protease activity and antigen breakdown are spatially heterogeneous in lymph nodes. Following protein immunization, antigens are rapidly degraded in the subcapsular sinus, paracortex, and interfollicular regions of the tissue. By contrast, the follicles and follicular dendritic cell (FDC) networks exhibit low protease activity and antigen degradation rates. Immunization regimens targeting antigen rapidly to FDCs led to germinal centers (GCs) where responses to intact antigen were highly dominant, while traditional bolus immunizations led to weaker GC responses where more GC B cells bound to breakdown products than intact antigen. Thus, spatially-compartmentalized antigen proteolysis impacts humoral immunity and can be exploited to enhance vaccine-induced production of antibody responses against key pathogen structural epitopes.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468745', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Emergence of order in condensates composed of multi-valent, multi-domain proteins', 'authors': \"[{'name': 'Srivastav Ranganathan'}, {'name': 'Eugene Shakhnovich'}]\", 'abstract': '<p>Many RNA-binding proteins (RBPs) that assemble into membraneless organelles, have a common architecture including disordered prion-like domain (PLD) and folded RNA-binding domain (RBD). An enrichment of PLD within the condensed phase gives rise to formation, on longer time scales, amyloid-like fibrils (aging). In this study, we employ coarse-grained Langevin dynamics simulations to explore the physical basis for the structural diversity in condensed phases of multi-domain RBPs. We discovered a highly cooperative first order transition between disordered (liquid-like) structures and an ordered (solid-like) phase whereby chains of PLD organize in fibrils with high nematic orientational order. Cooperativity of this liquid-solid transition makes fibril formation highly malleable to mutations or post-translational modifications. An interplay between homo-domain (PLD-PLD) and hetero-domain (PLD-RBD) interactions results in variety of structures with distinct spatial architectures. Our results provide a mechanistic understanding of how multi-domain RBPs could form assemblies with distinct structural and, potentially, material properties.</p><sec><title>Significance Statement</title><p>Assembly of proteins and nucleic acids into dense, liquid-like pockets is associated with several key functions including stress response, gene-regulation, DNA-repair and RNA processing. Several RNA binding proteins such as FUS are known to form liquid-like condensates that progressively harden into more dynamically, solid-like structures, a phenomenon that gets accelerated by disease mutations. In this study, we discover the mechanistic origins of this transition and show that small mutational or posttranslational modifications could result in sharp disorder-order transitions that could characterize accelerated liquid-solid transition in disease mutants.</p></sec>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.17.468942', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'IFITM dependency of SARS-CoV-2 variants of concern', 'authors': \"[{'name': 'Rayhane Nchioua'}, {'name': 'Annika Schundner'}, {'name': 'Dorota Kmiec'}, {'name': 'Caterina Prelli Bozzo'}, {'name': 'Fabian Zech'}, {'name': 'Lennart Koepke'}, {'name': 'Manfred Frick'}, {'name': 'Konstantin M. J. Sparrer'}, {'name': 'Frank Kirchhoff'}]\", 'abstract': '<p>We have recently shown that a SARS-CoV-2 strain isolated in the Netherlands in February 2020 (NL-02-2020) hijacks interferon-induced transmembrane proteins, especially IFITM2, as entry cofactors for efficient infection. Here, we examined whether SARS-CoV-2 “variants of concern” (VOCs), including the currently dominating delta variant, maintained the dependency on IFITMs for efficient replication. Depletion of IFITM2 reduced viral RNA production from 31- (B.1.1.7) to 755-fold (P.1). In comparison, silencing of IFITM1 had little effect, while knock-down of IFITM3 resulted in an intermediate phenotype. Strikingly, silencing of IFITM2 generally reduced infectious virus production in Calu-3 cells to near background levels. An antibody directed against the N-terminus of IFITM2 inhibited SARS-CoV-2 VOC replication in iPSC-derived alveolar epithelial type II cells. In conclusion, endogenously expressed IFITM proteins (especially IFITM2) are important cofactors for replication of genuine SARS-CoV-2 VOCs, including the Delta variant.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.16.468831', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Building an RNA-based Toggle Switch using Inhibitory RNA Aptamers', 'authors': \"[{'name': 'Alicia Climent-Catala'}, {'name': 'Thomas E. Ouldridge'}, {'name': 'Guy-Bart V. Stan'}, {'name': 'Wooli Bae'}]\", 'abstract': '<p>Synthetic RNA systems offer unique advantages such as faster response, increased specificity, and programmability compared to conventional protein-based networks. Here, we demonstrate an in-vitro RNA-based toggle switch using RNA aptamers capable of inhibiting the transcriptional activity of T7 or SP6 RNA polymerases. The activities of both polymerases are monitored simultaneously by using Broccoli and Malachite green light-up aptamer systems. In our toggle switch, a T7 promoter drives the expression of SP6 inhibitory aptamers, and an SP6 promoter expresses T7 inhibitory aptamers. We show that the two distinct states originating from the mutual inhibition of aptamers can be toggled by adding DNA sequences to sequester the RNA inhibitory aptamers. Finally, we assessed our RNA-based toggle switch in cell-like conditions by introducing controlled degradation of RNAs using a mix of RNases. Our results demonstrate that the RNA-based toggle switch could be used as a control element for nucleic acid networks in synthetic biology applications.</p><sec><title>Graphical TOC Entry</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"468831v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.18.469141', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Glacier-fed stream biofilms harbour diverse resistomes and biosynthetic gene clusters', 'authors': \"[{'name': 'Susheel Bhanu Busi'}, {'name': 'Laura de Nies'}, {'name': 'Paraskevi Pramateftaki'}, {'name': 'Massimo Bourquin'}, {'name': 'Leïla Ezzat'}, {'name': 'Tyler J. Kohler'}, {'name': 'Stilianos Fodelianakis'}, {'name': 'Grégoire Michoud'}, {'name': 'Hannes Peter'}, {'name': 'Michail Styllas'}, {'name': 'Matteo Tolosano'}, {'name': 'Vincent De Staercke'}, {'name': 'Martina Schön'}, {'name': 'Valentina Galata'}, {'name': 'Tom Battin'}, {'name': 'Paul Wilmes'}]\", 'abstract': '<sec><title>Background</title><p>Antimicrobial resistance (AMR) is a universal phenomenon whose origins lay in natural ecological interactions such as competition within niches, within and between micro- to higher-order organisms. However, the ecological and evolutionary processes shaping AMR need to be better understood in view of better antimicrobial stewardship. Resolving antibiotic biosynthetic pathways, including biosynthetic gene clusters (BGCs), and corresponding antimicrobial resistance genes (ARGs) may therefore help in understanding the inherent mechanisms. However, to study these phenomena, it is crucial to examine the origins of AMR in pristine environments with limited anthropogenic influences. In this context, epilithic biofilms residing in glacier-fed streams (GFSs) are an excellent model system to study diverse, intra- and inter-domain, ecological crosstalk.</p></sec><sec><title>Results</title><p>We assessed the resistomes of epilithic biofilms from GFSs across the Southern Alps (New Zealand) and the Caucasus (Russia) and observed that both bacteria and eukaryotes encoded twenty-nine distinct AMR categories. Of these, beta-lactam, aminoglycoside, and multidrug resistance were both abundant and taxonomically distributed in most of the bacterial and eukaryotic phyla. AMR-encoding phyla included Bacteroidota and Proteobacteria among the bacteria, alongside Ochrophyta (algae) among the eukaryotes. Additionally, BGCs involved in the production of antibacterial compounds were identified across all phyla in the epilithic biofilms. Furthermore, we found that several bacterial genera (<italic>Flavobacterium, Polaromonas</italic>, etc.) including representatives of the superphylum Patescibacteria encode both ARGs and BGCs within close proximity of each other, thereby demonstrating their capacity to simultaneously influence and compete within the microbial community.</p></sec><sec><title>Conclusions</title><p>Our findings highlight the presence and abundance of AMR in epilithic biofilms within GFSs. Additionally, we identify their role in the complex intra- and inter-domain competition and the underlying mechanisms influencing microbial survival in GFS epilithic biofilms. We demonstrate that eukaryotes may serve as AMR reservoirs owing to their potential for encoding ARGs. We also find that the taxonomic affiliation of the AMR and the BGCs are congruent. Importantly, our findings allow for understanding how naturally occurring BGCs and AMR contribute to the epilithic biofilms mode of life in GFSs. Importantly, these observations may be generalizable and potentially extended to other environments which may be more or less impacted by human activity.</p></sec>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.19.469253', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Improved resolution of recalcitrant nodes in the animal phylogeny through the analysis of genome gene content and morphology', 'authors': \"[{'name': 'Ksenia Juravel'}, {'name': 'Luis Porras'}, {'name': 'Sebastian Hoehna'}, {'name': 'Davide Pisani'}, {'name': 'Gert Wörheide'}]\", 'abstract': '<p>An accurate phylogeny of animals is needed to clarify their evolution, ecology, and impact on shaping the biosphere. Although multi-gene alignments of up to several hundred thousand amino acids are nowadays routinely used to test hypotheses of animal relationships, some nodes towards the root of the animal phylogeny are proving hard to resolve. While the relationships of the non-bilaterian lineages, primarily sponges (Porifera) and comb jellies (Ctenophora), have received much attention since more than a decade, controversies about the phylogenetic position of the worm-like bilaterian lineage Xenacoelomorpha and the monophyly of the \"Superphylum\" Deuterostomia have more recently emerged. Here we independently analyse novel genome gene content and morphological datasets to assess patterns of phylogenetic congruence with previous amino-acid derived phylogenetic hypotheses. Using statistical hypothesis testing, we show that both our datasets very strongly support sponges as the sister group of all the other animals, Xenoacoelomorpha as the sister group of the other Bilateria, and largely support monophyletic Deuterostomia. Based on these results, we conclude that the last common animal ancestor may have been a simple, filter-feeding organism without a nervous system and muscles, while the last common ancestor of Bilateria might have been a small, acoelomate-like worm without a through gut.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468763', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Disruption of vacuolin microdomains in the host Dictyostelium discoideum increases resistance to Mycobacterium marinum-induced membrane damage and infection', 'authors': \"[{'name': 'Cristina Bosmani'}, {'name': 'Angélique Perret'}, {'name': 'Florence Leuba'}, {'name': 'Aurélie Guého'}, {'name': 'Nabil Hanna'}, {'name': 'Thierry Soldati'}]\", 'abstract': '<p><italic>Mycobacterium tuberculosis</italic> (Mtb), the causative agent of tuberculosis, manipulates the host phagosome maturation pathway to replicate intracellularly. <italic>Mycobacterium marinum</italic>, a closely-related species, and <italic>Dictyostelium discoideum</italic>, a social amoeba and alternative phagocytic host, have been used as models to study host-pathogen interactions occurring during mycobacterial infections. Vacuolins, functional homologues of the mammalian flotillins, organize membrane microdomains and play a role in vesicular trafficking. Various pathogens have been reported to manipulate their membrane association and function. During infection of <italic>D</italic>. <italic>discoideum</italic> with <italic>M</italic>. <italic>marinum</italic>, Vacuolin C was specifically and highly induced and all three vacuolin isoforms were enriched at the mycobacteria-containing-vacuole (MCV). In addition, absence of vacuolins reduced escape from the MCV and conferred resistance to <italic>M</italic>. <italic>marinum</italic> infection. Moreover, ESAT-6, the membrane-disrupting virulence factor of <italic>M</italic>. <italic>marinum</italic>, was less associated with membranes when vacuolins were absent. Together, these results suggest that vacuolins are important host factors that are manipulated by mycobacteria to inflict membrane damage and escape from their compartment.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.17.468714', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'NMR metabolomics of symbioses between bacterial vaginosis associated bacteria', 'authors': \"[{'name': 'Victoria Horrocks'}, {'name': 'Charlotte K. Hind'}, {'name': 'Matthew E. Wand'}, {'name': 'Joel Chan'}, {'name': 'Jade C. Hopkins'}, {'name': 'Georgina L. Houston'}, {'name': 'Rachel M. Tribe'}, {'name': 'J. Mark Sutton'}, {'name': 'A. James Mason'}]\", 'abstract': '<p>Bacterial vaginosis (BV) is a dysbiosis of the vaginal microbiome, characterised by low levels of lacto-bacilli and overgrowth of a diverse group of bacteria, and associated with higher risk of a variety of infections, surgical complications, cancer and spontaneous preterm birth (PTB). Despite the lack of a consistently applicable aetiology, <italic>Prevotella</italic> spp. are often associated with both BV and PTB and <italic>P. bivia</italic> has known symbiotic relationships with both <italic>Peptostreptococcus anaerobius</italic> and <italic>Gardnerella vaginalis</italic>. Higher risk of PTB can also be predicted by a composite of metabolites linked to bacterial metabolism but their specific bacterial source remains poorly understood. Here we characterise diversity of metabolic strategies among BV associated bacteria and lactobacilli and the symbiotic metabolic relationships between <italic>P. bivia</italic> and its partners and show how these influence the availability of metabolites associated with BV/PTB and/or pro- or anti-inflammatory immune responses. We confirm a commensal relationship between <italic>Pe. anaerobius</italic> and <italic>P. bivia</italic>, refining its mechanism; <italic>P. bivia</italic> supplies tyrosine, phenylalanine, methionine, uracil and proline, the last of which leads to a substantial increase in overall acetate production. In contrast, our data indicate the relationship between <italic>P. bivia</italic> and <italic>G. vaginalis</italic> strains, with sequence variant G2, is mutualistic with outcome dependent on the metabolic strategy of the <italic>G. vaginalis</italic> strain. Seven <italic>G. vaginalis</italic> strains could be separated according to whether they performed mixed acid fermentation (MAF) or bifid shunt (BS). In co-culture, <italic>P. bivia</italic> supplies all <italic>G. vaginalis</italic> strains with uracil and received substantial amounts of asparagine in return. Acetate production, which is lower in BS strains, then matched that of MAF strains while production of aspartate increased for the latter. Taken together, our data show how knowledge of inter- and intra-species metabolic diversity and the effects of symbiosis may refine our understanding of the mechanism and approach to risk prediction in BV and/or PTB.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.17.468725', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Trait Association and Prediction Through Integrative K-mer Analysis', 'authors': \"[{'name': 'Cheng He'}, {'name': 'Jacob D Washburn'}, {'name': 'Yangfan Hao'}, {'name': 'Zhiwu Zhang'}, {'name': 'Jinliang Yang'}, {'name': 'Sanzhen Liu'}]\", 'abstract': '<p>Genome-wide association study (GWAS) with single nucleotide polymorphisms (SNPs) has been widely used to explore genetic controls of phenotypic traits. Here we employed an GWAS approach using k-mers, short substrings from sequencing reads. Using maize cob and kernel color traits, we demonstrated that k-mer GWAS can effectively identify associated k-mers. Co-expression analysis of kernel color k-mers and pathway genes directly found k-mers from causal genes. Analyzing complex traits of kernel oil and leaf angle resulted in k-mers from both known and candidate genes. Evolution analysis revealed most k-mers positively correlated with kernel oil were strongly selected against in maize populations, while most k-mers for upright leaf angle were positively selected. In addition, phenotypic prediction of kernel oil, leaf angle, and flowering time using k-mer data showed at least a similarly high prediction accuracy to the standard SNP-based method. Collectively, our results demonstrated the bridging role of k-mers for data integration and functional gene discovery.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469145', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Cryo soft X-ray tomography to explore Escherichia coli nucleoid remodelling by Hfq master regulator', 'authors': \"[{'name': 'Antoine Cossa'}, {'name': 'Sylvain Trépout'}, {'name': 'Frank Wien'}, {'name': 'Etienne Le Brun'}, {'name': 'Florian Turbant'}, {'name': 'Eva Pereiro'}, {'name': 'Véronique Arluison'}]\", 'abstract': '<p>Bacterial chromosomic DNA is packed within a membrane-less structure, the nucleoid, thanks to proteins called Nucleoid Associated Proteins (NAPs). The NAP composition of the nucleoid varies during the bacterial life cycle and is growth phase-dependent. Among these NAPs, Hfq is one of the most intriguing as it plays both direct and indirect roles on DNA structure. Indeed, Hfq is best known to mediate post-transcriptional regulation by using small noncoding RNA (sRNA). Although Hfq presence in the nucleoid has been demonstrated for years, its precise role is still unclear. Recently, it has been shown <italic>in vitro</italic> that Hfq belongs to the bridging family of NAPs. Its bridging mechanism relies on the formation of the amyloid-like structure of Hfq C-terminal region. Here, using cryo soft X-ray tomography imaging of native unlabelled cells and using a semi-automatic analysis and segmentation procedure, we show that Hfq significantly remodels the <italic>Escherichia coli</italic> nucleoid, especially during the stationary growth phase. Hfq influences both nucleoid volume and absorbance. Hfq cumulates direct effects and indirect effects due to sRNA-based regulation of other NAPs. Taken together, our findings reveal a new role for this protein in nucleoid remodelling that may serve in response to stress conditions and in adapting to changing environments. This implies that Hfq regulates nucleoid compaction directly via its interaction with DNA, but also at the post-transcriptional level via its interaction with RNA.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.19.469341', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Scoping the Line Up: A Comparison of Biomonitoring Methodologies for Surf Zone Fish Communities', 'authors': \"[{'name': 'Zachary Gold'}, {'name': 'McKenzie Q. Koch'}, {'name': 'Nicholas K. Schooler'}, {'name': 'Kyle A. Emery'}, {'name': 'Jennifer E. Dugan'}, {'name': 'Robert J. Miller'}, {'name': 'Henry M. Page'}, {'name': 'Donna M. Schroeder'}, {'name': 'David M. Hubbard'}, {'name': 'Jessica R. Madden'}, {'name': 'Stephen G. Whitaker'}, {'name': 'Paul H. Barber'}]\", 'abstract': '<p>Surf zones are highly dynamic marine ecosystems that are subject to increasing anthropogenic and climatic pressures, posing multiple challenges for biomonitoring. Traditional methods such as seines and hook and line surveys are often labor intensive, taxonomically biased, and can be physically hazardous. Emerging techniques, such as baited remote underwater video (BRUV) and environmental DNA (eDNA) are promising nondestructive tools for assessing marine biodiversity in surf zones of sandy beaches. Here we compare the relative performance of beach seines, BRUV, and eDNA in characterizing community composition of bony (teleost) and cartilaginous (elasmobranch) fishes of surf zones at 18 open coast sandy beaches in southern California. Seine and BRUV surveys captured overlapping, but distinct fish communities with 50% (18/36) of detected species shared. BRUV surveys more frequently detected larger species (e.g. sharks and rays) while seines more frequently detected one of the most abundant species, barred surfperch (  Amphistichus argenteus  ). In contrast, eDNA metabarcoding captured 83.3% (30/36) of all fishes observed in seine and BRUV surveys plus 59 additional species, including 13 that frequent surf zone habitats. eDNA approaches showed significantly higher sensitivity than seine and BRUV methods and more consistently detected 29 of the 30 (96.7%) jointly observed species across beaches. The six species detected by BRUV/seines, but not eDNA either lacked reference sequences, were only resolved at higher taxonomic ranks (e.g.  Embiotocidae  surfperches), or were detected below occupancy thresholds. Low site-species overlap between methods limited comparisons of richness and abundance estimates, highlighting the challenge of comparing biomonitoring approaches. Despite potential for improvement, results overall demonstrate that eDNA can provide a cost-effective tool for long-term surf zone monitoring that complements data from seine and BRUV surveys, allowing more comprehensive surveys of vertebrate diversity in surf zone habitats.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468657', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Learning a new class of multisensory associations: High-density electrophysiological mapping of the temporal course of audio-visual object processing', 'authors': \"[{'name': 'Tiziana Vercillo'}, {'name': 'Edward G. Freedman'}, {'name': 'Joshua B. Ewen'}, {'name': 'Sophie Molholm'}, {'name': 'John J. Foxe'}]\", 'abstract': '<p>Multisensory objects that are frequently encountered in the natural environment lead to strong associations across a distributed sensory cortical network, with the end result experience of a unitary percept. Remarkably little is known, however, about the cortical processes sub-serving multisensory object formation and recognition. To advance our understanding in this important domain, the present study investigated the brain processes involved in learning and identification of novel visual-auditory objects. Specifically, we introduce and test a rudimentary three-stage model of multisensory object-formation and processing. Thirty adults were remotely trained for a week to recognize a novel class of multisensory objects (3D shapes paired to complex sounds), and high-density event related potentials (ERPs) were recorded to the corresponding unisensory (shapes or sounds only) and multisensory (shapes and sounds) stimuli, before and after intensive training. We identified three major stages of multisensory processing: 1) an early, multisensory, automatic effect (&lt;100 ms) in occipital areas, related to the detection of simultaneous audiovisual signals and not related to multisensory learning 2) an intermediate object-processing stage (100-200 ms) in occipital and parietal areas, sensitive to the learned multisensory associations and 3) a late multisensory processing stage (&gt;250 ms) that appears to be involved in both object recognition and possibly memory consolidation. Results from this study provide support for multiple stages of multisensory object learning and recognition that are subserved by an extended network of cortical areas.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469103', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Evasion of Toll-like Receptor Recognition by Escherichia coli is mediated via Population Level Regulation of Flagellin Production', 'authors': \"[{'name': 'M Lanz'}, {'name': 'C Birchall'}, {'name': 'L Drage'}, {'name': 'D Picton'}, {'name': 'C Mowbray'}, {'name': 'Q Alsenani'}, {'name': 'A. Tan'}, {'name': 'A Ali'}, {'name': 'C Harding'}, {'name': 'R Pickard'}, {'name': 'J Hall'}, {'name': 'PD Aldridge'}]\", 'abstract': '<p>Uropathogenic <italic>Escherichia coli</italic> (UPEC) is a major cause of urinary tract infections. Analysis of the innate immune response in immortalised urothelial cells suggests that the bacterial flagellar subunit, flagellin, is key in inducing host defences. A panel of 39 clinical uro-associated <italic>Escherichia coli</italic> isolates recovered from either asymptomatic bacteruria (ASB), cystitis or pyelonephritis patients, were characterised for motility and their ability to induce an innate response in urothelial cells stably transfected with a NFκB luciferase reporter. Twenty-four isolates (60%) were identified as motile with strains recovered from cystitis patients exhibiting a bipolar motility distribution pattern (P &lt; 0.005) and associated with a 2-5 fold increase in NFκB signalling. Although two isolates were associated with swarm sizes of &gt;7 cm and NFκB activities of &gt;30 fold (P = 0.029), data overall suggested bacterial motility and the NFκB signalling response were not directly correlated. To explore whether the signalling response reflected antigenic variation flagellin was purified from 11 different isolates and the urothelial cell challenges repeated. Purified flagellin filaments generated comparable (30.4±1.8 to 46.1±2.5 fold, P = NS) NFκB signalling responses, irrespective of either the source of the isolate or H-serotype. These data argued against any variability between isolates being related to flagellin itself. To determine the roles, if any, of flagellar abundance in inducing these responses flagellar hook numbers of a range of cystitis and ABU isolates were quantified using a plasmid encoded flagellar hook gene <italic>flgEA240C</italic>. Foci data suggested isolates were averaging between 1 and 2 flagella per cell, while only 10 to 60% each isolates population exhibited foci. These data suggested selective pressures exist in the urinary tract that allow uro-associated <italic>E. coli</italic> strains to maintain motility exploiting population heterogeneity to prevent host TLR5 recognition.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.15.468584', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Ly6Chi monocytes are metabolically reprogrammed in the blood during inflammatory stimulation allowing for macrophage lineage commitment', 'authors': \"[{'name': 'Gareth S.D. Purvis'}, {'name': 'Eileen McNeill'}, {'name': 'Benjamin Wright'}, {'name': 'Santiago Revale'}, {'name': 'Helen Lockstone'}, {'name': 'Keith M. Channon'}, {'name': 'David R. Greaves'}]\", 'abstract': '<p>Acute inflammation is a rapid and dynamic process involving the recruitment and activation of multiple cell types in a co-ordinated and precise manner. Using cell tracking, linage tracing and single cell transcriptomics we investigated the origin and transcriptional reprogramming of monocytes and macrophages in acute inflammation.</p><p>Monocyte trafficking and adoptive transfer experiments revealed that monocytes undergo rapid phenotypic change as they exit the blood and give rise to monocyte-derived macrophages that persist during the resolution of inflammation. Single cell transcriptomics revealed significant heterogeneity within the surface marker defined CD11b<sup>+</sup>Ly6G<sup>-</sup>Ly6C<sup>hi</sup> monocyte population within the blood and at the site of inflammation. Lineage trajectory analysis revealed that Ly6C<sup>hi</sup> monocytes in the blood are re-programmed into a defined differentiation pathway following inflammatory stimulus. We show that two major transcriptional reprogramming events occur during the initial 6 h of Ly6C<sup>hi</sup> monocyte mobilisation, one in the blood priming monocytes for migration and a second at the site of inflammation. Pathway analysis revealed an important role for oxidative phosphorylation (OxPhos) during both these reprogramming events in a subset of M2-like cells. Experimentally we also demonstrate that OxPhos is essential for murine and human monocyte chemotaxis. These new findings opening up the possibility that altering monocyte metabolic capacity towards OxPhos could facilitate enhanced macrophage M2-like polarisation to aid inflammation resolution and tissue repair.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469079', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Multitrophic higher-order interactions modulate species persistence', 'authors': \"[{'name': 'Lisa Buche'}, {'name': 'Ignasi Bartomeus'}, {'name': 'Oscar Godoy'}]\", 'abstract': '<p>There is growing recognition that interactions between species pairs are modified in a multispecies context by the density of a third species. However, how these higher-order interactions (HOIs) affect species persistence remains poorly understood. To explore the effect of HOIs steaming from multiple trophic layers on plant persistence, we experimentally built a mutualistic system containing three plants and three pollinators species with two contrasting network structures. For both structures, we first estimated the statistically supported HOIs on plant species, in addition to the pairwise interactions among plants and  plant-pollinators. Following a structuralist approach, we then assessed the effects of the supported HOIs on the persistence probability of each of the three competing plant species and their combinations. HOIs produced substantial effects on the strength and sign of per capita interactions between plant species to such an extent that predictions of species persistence differ from a non-HOIs scenario. Changes in network structure due to removing a plant-pollinator link further modulated the species persistence probabilities by reorganizing per capita interaction strengths of both pairwise interactions and HOIs. Our study provides empirical evidence of the joint importance of HOIs and network structure for determining the probability of species to persist within diverse communities.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468757', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Impacts of Taxon-Sampling Schemes on Bayesian Molecular Dating under the Unresolved Fossilized Birth-Death Process', 'authors': \"[{'name': 'Arong Luo'}, {'name': 'Chi Zhang'}, {'name': 'Qing-Song Zhou'}, {'name': 'Simon Y.W. Ho'}, {'name': 'Chao-Dong Zhu'}]\", 'abstract': '<p>Evolutionary timescales can be estimated using a combination of genetic data and fossil evidence based on the molecular clock. Bayesian phylogenetic methods such as tip dating and total-evidence dating provide a powerful framework for inferring evolutionary timescales, but the most widely used priors for tree topologies and node times often assume that present-day taxa have been sampled randomly or exhaustively. In practice, taxon sampling is often carried out so as to include representatives of major lineages, such as orders or families. We examined the impacts of these diversified sampling schemes on Bayesian molecular dating under the unresolved fossilized birth-death (FBD) process, in which fossil taxa are topologically constrained but their exact placements are not inferred. We used synthetic data generated by simulation of nucleotide sequence evolution, fossil occurrences, and diversified taxon sampling. Our analyses show that increasing sampling density does not substantially improve divergence-time estimates under benign conditions. However, when the tree topologies were fixed to those used for simulation or when evolutionary rates varied among lineages, the performance of Bayesian tip dating improves with sampling density. By exploring three situations of model mismatches, we find that including all relevant fossils without pruning off those inappropriate for the FBD process can lead to underestimation of divergence times. Our reanalysis of a eutherian mammal data set confirms some of the findings from our simulation study, and reveals the complexity of diversified taxon sampling in phylogenomic data sets. In highlighting the interplay of taxon-sampling density and other factors, the results of our study have useful implications for Bayesian molecular dating in the era of phylogenomics.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468633', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': True, 'title': 'Tunneling nanotubes provide a novel route for SARS-CoV-2 spreading between permissive cells and to non-permissive neuronal cells', 'authors': \"[{'name': 'Anna Pepe'}, {'name': 'Stefano Pietropaoli'}, {'name': 'Matthijn Vos'}, {'name': 'Giovanna Barba-Spaeth'}, {'name': 'Chiara Zurzolo'}]\", 'abstract': '<p>SARS-CoV-2 entry into host cells is mediated by the binding of its spike glycoprotein to the angiotensin-converting enzyme 2 (ACE2) receptor, highly expressed in several organs, but very low in the brain. The mechanism through which SARS-CoV-2 infects neurons is not understood. Tunneling nanotubes (TNTs), actin-based intercellular conduits that connect distant cells, allow the transfer of cargos, including viruses. Here, we explored the neuroinvasive potential of SARS-CoV-2 and whether TNTs are involved in its spreading between cells <italic>in vitro</italic>. We report that neuronal cells, not permissive to SARS-CoV-2 through an exocytosis/endocytosis dependent pathway, can be infected when co-cultured with permissive infected epithelial cells. SARS-CoV-2 induces TNTs formation between permissive cells and exploits this route to spread to uninfected permissive cells in co-culture. Correlative Cryo-electron tomography reveals that SARS-CoV-2 is associated with the plasma membrane of TNTs formed between permissive cells and virus-like vesicular structures are inside TNTs established both between permissive cells and between permissive and non-permissive cells. Our data highlight a potential novel mechanism of SARS-CoV-2 spreading which could serve as route to invade non-permissive cells and potentiate infection in permissive cells.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.16.468554', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Reversible regulation of Cas12a activities by AcrVA5-mediated acetylation and CobB-mediated deacetylation', 'authors': \"[{'name': 'Xiaoman Kang'}, {'name': 'Lei Yin'}, {'name': 'Songkuan Zhuang'}, {'name': 'Tianshuai Hu'}, {'name': 'Zhile Wu'}, {'name': 'Guoping Zhao'}, {'name': 'Yijian Chen'}, {'name': 'Yong Xu'}, {'name': 'Jin Wang'}]\", 'abstract': '<p>The clustered regularly interspaced short palindromic repeat (CRISPR)/CRISPR associated (Cas) system protects bacteria and archaea from the invasion of foreign genetic elements. To cope with the host CRISPR systems, phages have evolved many strategies, including the anti-CRISPR (Acr) proteins, to inactivate the Cas nucleases. Recently, it has been reported that the type V-A Cas12a effector can be acetylated and inactivated by AcrVA5, which is a GNAT-family acetyltransferase. However, it is unclear whether the host has any coping strategies to reactivate the defense system. Here we show that the AcrVA5-acetylated Cas12a can be deacetylated by bacterial deacetylase CobB, reactivating Cas12a for both in vitro cleavage of target DNA sequences and in vivo protection of the host from invasion of foreign nucleic acids. Therefore, this study not only shows the reversible regulation of Cas12a activities by post-translational modification but also reveals CobB as a secondary safeguard to bacterial CRISPR defense systems. In addition, we demonstrate that AcrVA5 is a wide-spectrum acetyltransferase, acetylating a large number of target proteins besides Cas12a, and the AcrVA5-acetylated targets can also be deacetylated by CobB.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.16.468835', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Plasmodium-encoded murine IL-6 impairs liver stage infection and elicits long-lasting sterilizing immunity', 'authors': \"[{'name': 'Selma Belhimeur'}, {'name': 'Sylvie Briquet'}, {'name': 'Roger Peronet'}, {'name': 'Jennifer Pham'}, {'name': 'Pierre-Henri Commere'}, {'name': 'Rogerio Amino'}, {'name': 'Artur Scherf'}, {'name': 'Olivier Silvie'}, {'name': 'Salaheddine Mecheri'}]\", 'abstract': '<p><italic>Plasmodium</italic> sporozoites inoculated by <italic>Anopheles</italic> mosquitoes into the skin of the mammalian host migrate to the liver before infecting hepatocytes. Previous work demonstrated that early production of IL-6 in the liver was found to be detrimental for the parasite growth, leading to the acquisition of a long-lasting immune protection. Considering IL-6 as a critical pro-inflammatory signal, we explored a novel approach whereby the parasite itself encodes for the murine IL-6 gene. We generated transgenic <italic>P</italic>.<italic>berghei</italic> parasites that express murine IL-6 during liver stage development. Though IL-6 transgenic sporozoites develop into exo-erythrocytic forms in cultured hepatocytes <italic>in vitro</italic>, these parasites were not capable of inducing a blood stage infection in mice. Furthermore, immunization of mice with transgenic IL-6 sporozoites elicited a long-lasting CD8<sup>+</sup> T cell-mediated protective immunity against a subsequent infectious sporozoite challenge. Collectively, this study demonstrates that parasite-encoded IL-6 impairs <italic>Plasmodium</italic> infection at the liver stage, forming the basis of a novel suicide vaccine strategy to elicit protective antimalarial immunity.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.11.468228', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Structural and functional characterization of NEMO cleavage by SARS-CoV-2 3CLpro', 'authors': \"[{'name': 'Mikhail Ali Hameedi'}, {'name': 'Erica T. Prates'}, {'name': 'Michael R. Garvin'}, {'name': 'Irimpan Mathews'}, {'name': 'B Kirtley Amos'}, {'name': 'Omar Demerdash'}, {'name': 'Mark Bechthold'}, {'name': 'Mamta Iyer'}, {'name': 'Simin Rahighi'}, {'name': 'Daniel W. Kneller'}, {'name': 'Andrey Kovalevsky'}, {'name': 'Stephan Irle'}, {'name': 'Van-Quan Vuong'}, {'name': 'Julie C. Mitchell'}, {'name': 'Audrey Labbe'}, {'name': 'Stephanie Galanie'}, {'name': 'Soichi Wakatsuki'}, {'name': 'Daniel Jacobson'}]\", 'abstract': '<p>In addition to its essential role in viral polyprotein processing, the SARS-CoV-2 3C-like (3CLpro) protease can cleave human immune signaling proteins, like NF-κB Essential Modulator (NEMO) and deregulate the host immune response. Here, <italic>in vitro</italic> assays show that SARS-CoV-2 3CLpro cleaves NEMO with fine-tuned efficiency. Analysis of the 2.14 Å resolution crystal structure of 3CLpro C145S bound to NEMO<sub>226-235</sub> reveals subsites that tolerate a range of viral and host substrates through main chain hydrogen bonds while also enforcing specificity using side chain hydrogen bonds and hydrophobic contacts. Machine learning- and physics-based computational methods predict that variation in key binding residues of 3CLpro- NEMO helps explain the high fitness of SARS-CoV-2 in humans. We posit that cleavage of NEMO is an important piece of information to be accounted for in the pathology of COVID-19.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469113', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'STRling: a k-mer counting approach that detects short tandem repeat expansions at known and novel loci', 'authors': \"[{'name': 'Harriet Dashnow'}, {'name': 'Brent S. Pedersen'}, {'name': 'Laurel Hiatt'}, {'name': 'Joe Brown'}, {'name': 'Sarah J. Beecroft'}, {'name': 'Gianina Ravenscroft'}, {'name': 'Amy J. LaCroix'}, {'name': 'Phillipa Lamont'}, {'name': 'Richard H. Roxburgh'}, {'name': 'Miriam J. Rodrigues'}, {'name': 'Mark Davis'}, {'name': 'Heather C. Mefford'}, {'name': 'Nigel G. Laing'}, {'name': 'Aaron R. Quinlan'}]\", 'abstract': '<p>Expansions of short tandem repeats (STRs) cause dozens of rare Mendelian diseases. However, STR expansions, especially those arising from repeats not present in the reference genome, are challenging to detect from short-read sequencing data. Such \"novel\" STRs include new repeat units occurring at known STR loci, or entirely new STR loci where the sequence is absent from the reference genome. A primary cause of difficulty detecting STR expansions is that reads arising from STR expansions are frequently mismapped or unmapped. To address this challenge, we have developed STRling, a new STR detection algorithm that counts k-mers (short DNA sequences of length k) in DNA sequencing reads, to efficiently recover reads that inform the presence and size of STR expansions. As a result, STRling can call expansions at both known and novel STR loci. STRling has a sensitivity of 83% for 14 known STR disease loci, including the novel STRs that cause CANVAS and DBQD2. It is the first method to resolve the position of novel STR expansions to base pair accuracy. Such accuracy is essential to interpreting the consequence of each expansion. STRling has an estimated 0.078 false discovery rate for known pathogenic loci in unaffected individuals and a 0.20 false discovery rate for genome-wide loci in unaffected individuals when using variants called from long-read data as truth. STRling is fast, scalable on cloud computing, open-source, and freely available at https://github.com/quinlan-lab/STRling.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.15.468749', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Affinity Scores: An Individual-centric Fingerprinting Framework for Neuropsychiatric Disorders', 'authors': \"[{'name': 'Cassandra M J Wannan'}, {'name': 'Christos Pantelis'}, {'name': 'Antonia Merritt'}, {'name': 'Bruce Tonge'}, {'name': 'Warda T Syeda'}]\", 'abstract': '<p>Background: Population-centric frameworks of biomarker identification for psychiatric disorders focus primarily on comparing averages between groups and assume that diagnostic groups are (1) mutually-exclusive, and (2) homogeneous. There is a paucity of individual-centric approaches capable of identifying individual-specific fingerprints across multiple domains. To address this, we propose a novel framework, combining a range of biopsychosocial markers, including brain structure, cognition, and clinical markers, into higher-level fingerprints, capable of capturing intra-illness heterogeneity and inter-illness overlap. Methods: A multivariate framework was implemented to identify individualised patterns of brain structure, cognition and clinical markers based on affinity to other participants in the database. First, individual-level affinity scores defined a neighbourhood for each participant across each measure based on variable-specific hop sizes. Next, diagnostic verification and classification algorithms were implemented based on multivariate affinity score profiles. To perform affinity-based classification, data were divided into training and test samples, and 5-fold nested cross-validation was performed on the training data. Affinity-based classification was compared to weighted K-nearest neighbours (KNN) classification. K-means clustering was used to create clusters based on multivariate affinity score profiles. The framework was applied to the Australian Schizophrenia Research Bank (ASRB) dataset. Results: Individualised affinity scores provided a fingerprint of brain structure, cognition, and clinical markers, which described the affinity of an individual to the representative groups in the dataset Diagnostic verification capability was moderate to high depending on the choice of multivariate affinity metric. Affinity score-based classification achieved a high degree of accuracy in the training, nested cross-validation and prediction steps, and outperformed KNN classification in the training and test datasets. Conclusion: Affinity scores demonstrate utility in two keys ways: (1) Early and accurate diagnosis of neuropsychiatric disorders, whereby an individual can be grouped within a diagnostic category/ies that best matches their fingerprint, and (2) identification of biopsychosocial factors that most strongly characterise individuals/disorders, and which may be most amenable to intervention.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468904', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'VIVID: a web application for variant interpretation and visualisation in multidimensional analyses', 'authors': \"[{'name': 'Swapnil Tichkule'}, {'name': 'Yoochan Myung'}, {'name': 'Myo T Naung'}, {'name': 'Brendan RE Ansell'}, {'name': 'Andrew Guy'}, {'name': 'Namrata Srivastava'}, {'name': 'Somya Mehra'}, {'name': 'Simone M Caccio'}, {'name': 'Ivo Mueller'}, {'name': 'Alyssa E Barry'}, {'name': 'Cock van Oosterhout'}, {'name': 'Bernard Pope'}, {'name': 'David B Ascher'}, {'name': 'Aaron Jex'}]\", 'abstract': '<p>Large-scale comparative genomics- and population genetic studies generate enormous amounts of polymorphism data in the form of DNA variants. Ultimately, the goal of many of these studies is to associate genetic variants to phenotypes or fitness. We introduce VIVID, an interactive, user-friendly web application that integrates a wide range of approaches for encoding genotypic to phenotypic information in any organism or disease, from an individual or population, in three-dimensional (3D) space. It allows mutation mapping and annotation, calculation of interactions and conservation scores, prediction of harmful effects, analysis of diversity and selection, and 3-dimensional (3D) visualisation of genotypic information encoded in Variant Call Format (VCF) on AlphaFold2 protein models. VIVID enables the rapid assessment of genes of interest in the study of adaptive evolution and the genetic load, and it helps prioritising targets for experimental validation. We demonstrate the utility of VIVID by exploring the evolutionary genetics of the parasitic protist Plasmodium falciparum, revealing geographic variation in the signature of balancing selection in potential targets of functional antibodies.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.469027', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Monophyly of Diverse Bigyromonadea and their Impact on Phylogenomic Relationships Within Stramenopiles', 'authors': \"[{'name': 'Anna Cho'}, {'name': 'Denis V. Tikhonenkov'}, {'name': 'Elisabeth Hehenberger'}, {'name': 'Anna Karnkowska'}, {'name': 'Patrick J. Keeling'}]\", 'abstract': '<p>Stramenopiles are a diverse but relatively well-studied eukaryotic supergroup with considerable genomic information available (Sibbald and Archibald, 2017). Nevertheless, the relationships between major stramenopile subgroups remain unresolved, in part due to a lack of data from small nanoflagellates that make up a lot of the genetic diversity of the group. This is most obvious in Bigyromonadea, which is one of four major stramenopile subgroups but represented by a single transcriptome. To examine the diversity of Bigyromonadea and how the lack of data affects the tree, we generated transcriptomes from seven novel bigyromonada species described in this study: Develocauda condao, Develocanicus komovi, Develocanicus vyazemskyi, Cubaremonas variflagellatum, Pirsonia chemainus, Feodosia pseudopoda, and Koktebelia satura. Both maximum likelihood and Bayesian phylogenomic trees based on a 247 gene-matrix recovered a monophyletic Bigyromonadea that includes two diverse subgroups, Developea and Pirsoniales, that were not previously related based on single gene trees. Maximum likelihood analyses show Bigyromonadea related to oomycetes, whereas Bayesian analyses and topology testing were inconclusive. We observed similarities between the novel bigyromonad species and motile zoospores of oomycetes in morphology and the ability to self-aggregate. Rare formation of pseudopods and fused cells were also observed, traits that are also found in members of labyrinthulomycetes, another osmotrophic stramenopiles. Furthermore, we report the first case of eukaryovory in the flagellated stages of Pirsoniales. These analyses reveal new diversity of Bigyromonadea, and altogether suggest their monophyly with oomycetes, collectively known as Pseudofungi, is the most likely topology of the stramenopile tree.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468603', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Estimating the Mosquito Density in Guangzhou City, China', 'authors': \"[{'name': 'Meili Li'}, {'name': 'Xian Zhang'}, {'name': 'Junling Ma'}]\", 'abstract': '<p>Mosquito is a vector of many diseases. Predicting the trend of mosquito density is important for early warning and control of mosquito diseases. In this paper, we fit a discrete time mosquito model developed by Gong et al. in 2011, which considers the immature and adult stages, and weather dependent model parameters, to the Breteau Index and Bite Index data for Aedes aegypti in Guangzhou city, China in 2014, as well as the weather data for average temperature, precipitation, evaporation and daylight for the same period. We estimated the model parameters using the Markov Chain Monte-Carlo (MCMC) method. We find that many parameters are not identifiable. We revise and simplify the model so that the parameters of our new model are identifiable. Our results indicate that the model predicted mosquito prevalence agrees well with data. We then use the fitted parameter values against the Breteau Index and Bite Index data for Guangzhou city in 2017 and 2018, and show that the estimated parameter values are applicable for other seasons.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.17.469063', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Bacterial competition systems share a domain required for inner membrane transport of the nuclease bacteriocin pyocin G', 'authors': \"[{'name': 'Iva Atanaskovic'}, {'name': 'Connor Sharp'}, {'name': 'Cara Press'}, {'name': 'Renata Kaminska'}, {'name': 'Colin Kleanthous'}]\", 'abstract': '<p>Bacteria exploit a variety of attack strategies to gain dominance within ecological niches. Prominent amongst these are contact-dependent inhibition (CDI), type VI secretion (T6SS) and bacteriocins. The cytotoxic endpoint of these systems is often the delivery of a nuclease to the cytosol. How such nucleases translocate across the cytoplasmic membrane of Gram-negative bacteria is unknown. Here, we identify a small, conserved, 15-kDa domain, which we refer to as the inner membrane translocation (IMT) domain that is common to T6SS and bacteriocins and linked to nuclease effector domains. Through fluorescence microscopy assays using intact and spheroplasted cells, we demonstrate that the IMT domain of the <italic>Pseudomonas aeruginosa</italic> specific bacteriocin pyocin G (PyoG) is required for import of the toxin nuclease domain to the cytoplasm. We also show that translocation of PyoG into the cytosol is dependent on inner membrane proteins FtsH, a AAA+ATPase/protease, and TonB1, the latter more typically associated with transport of bacteriocins across the outer membrane. Our study reveals that the IMT domain directs the cytotoxic nuclease of PyoG to cross the cytoplasmic membrane and, more broadly, has been adapted for the transport of other toxic nucleases delivered into Gram-negative bacteria by both contact dependent- and contact-independent means.</p><sec><title>Importance</title><p>Nuclease bacteriocins are potential antimicrobials for the treatment of antibiotic resistant bacterial infections. While the mechanism of outer membrane translocation is beginning to be understood, the mechanism of inner membrane transport is not known. This study uses PyoG as a model nuclease bacteriocin and defines a conserved domain which is essential for inner membrane translocation and which is widespread in other bacterial competition systems. Additionally, the presented data links two membrane proteins, FtsH and TonB1, with inner membrane translocation of PyoG. These findings point to the general importance of this domain to the cellular uptake mechanisms of nucleases delivered by otherwise diverse and distinct bacterial competition systems. The work is also of importance for the design of new protein antibiotics.</p></sec>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.15.468710', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Cryo-EM of the yeast VO complex reveals distinct binding sites for macrolide V-ATPase inhibitors', 'authors': \"[{'name': 'Kristine A. Keon'}, {'name': 'Samir Benlekbir'}, {'name': 'John L. Rubinstein'}]\", 'abstract': '<p>Vacuolar-type adenosine triphosphatases (V-ATPases) are proton pumps found in almost all eukaryotic cells. These enzymes consist of a soluble catalytic V<sub>1</sub> region that hydrolyzes ATP and a membrane-embedded V<sub>O</sub> region responsible for proton translocation. V-ATPase activity leads to acidification of endosomes, phagosomes, lysosomes, secretory vesicles, and the trans-Golgi network, with extracellular acidification occurring in some specialized cells. Small molecule inhibitors of V-ATPase have played a crucial role in elucidating numerous aspects of cell biology by blocking acidification of intracellular compartments, while therapeutic use of V-ATPase inhibitors has been proposed for treatment of cancer, osteoporosis, and some infections. Here, we determine structures of the isolated V<sub>O</sub> complex from <italic>Saccharomyces cerevisiae</italic> bound to two well-known macrolide inhibitors: bafilomycin A1 and archazolid A. The structures reveal different binding sites for the inhibitors on the surface of the proton-carrying c ring, with only a small amount of overlap between the two sites. Binding of both inhibitors is mediated primarily through van der Waals interactions in shallow pockets and suggests that the inhibitors block rotation of the ring. Together, these structures indicate the existence of a large chemical space available for V-ATPase inhibitors that block acidification by binding the c ring.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.17.468962', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Membrane adaptation in the hyperthermophilic archaeon Pyrococcus furiosus relies upon a novel strategy involving glycerol monoalkyl glycerol tetraether lipids', 'authors': \"[{'name': 'Maxime Tourte'}, {'name': 'Philippe Schaeffer'}, {'name': 'Vincent Grossi'}, {'name': 'Philippe M. Oger'}]\", 'abstract': '<p>Microbes preserve membrane functionality under fluctuating environmental conditions by modulating their membrane lipid composition. Although several studies have documented membrane adaptations in Archaea, the influence of most biotic and abiotic factors on archaeal lipid compositions remains underexplored. Here, we studied the influence of temperature, pH, salinity, the presence/absence of elemental sulfur, the carbon source, and the genetic background on the core lipid composition of the hyperthermophilic neutrophilic marine archaeon <italic>Pyrococcus furiosus.</italic> Every growth parameter tested affected the core lipid composition to some extent, the carbon source and the genetic background having the greatest influence. Surprisingly, <italic>P. furiosus</italic> appeared to only marginally rely on the two major responses implemented by Archaea, i.e., the regulation of the ratio of diether to tetraether lipids and that of the number of cyclopentane rings in tetraethers. Instead, this species increased the ratio of glycerol monoalkyl glycerol tetraethers (GMGT, aka. H-shaped tetraethers) to glycerol dialkyl glycerol tetrathers (GDGT) in response to decreasing temperature and pH and increasing salinity, thus providing for the first time evidence of adaptive functions for GMGT. Besides <italic>P. furiosus</italic>, numerous other species synthesize significant proportions of GMGT, which suggests that this unprecedented adaptive strategy might be common in Archaea.</p><sec><title>Significance statement</title><p>We describe here the membrane adaptive strategies the hyperthermophilic, neutrophilic, and marine model archaeon <italic>Pyrococcus furiosus</italic> implements in response to one of the largest sets of environmental stressors tested to date, including temperature, pH, salinity, presence/absence of elemental sulfur, carbon source, and genetic background. In contrast to the other archaea investigated so far, which response mainly involves the modulation of their diether/tetraether ratio and/or of their average number of cyclopentane rings, <italic>P. furiosus</italic> regulates its monoalkyl (so called H-shaped) to dialkyl tetraether ratio. Our study thus provides for the first time evidence of adaptive functions of archaeal monoalkyl tetraethers towards low temperature and pH and high salinity.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.16.468838', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'A genome in flux: homoeologous exchanges, subgenome dominance, and gene dosage balance constraints in resynthesized allopolyploid Brassica napus', 'authors': \"[{'name': 'Kevin A. Bird'}, {'name': 'J. Chris Pires'}, {'name': 'Robert VanBuren'}, {'name': 'Zhiyong Xiong'}, {'name': 'Patrick P. Edger'}]\", 'abstract': '<p>Allopolyploidy involves the hybridization of two evolutionary diverged species and the doubling of genomic material. Frequently, allopolyploids exhibit genomic rearrangements that recombine, duplicate, or delete homoeologous regions of the newly formed genome. While decades of investigation have focused on how genome duplication leads to systematic differences in the retention and expression of duplicate genes, the impact of genomic rearrangements on genome evolution has received less attention. We used genomic and transcriptomic data for six independently resynthesized, isogenic <italic>Brassica napus</italic> lines in the first, fifth, and tenth generation to identify genomic rearrangements and assess their impact on gene expression dynamics related to subgenome dominance and gene dosage constraint. We find that dosage constraints on the gene expression response to polyploidy begin to loosen within the first ten generations of evolution and systematically differ between dominant and non-dominant subgenomes. We also show that genomic rearrangements can bias estimation of homoeolog expression bias, but fail to fully obscure which subgenome is dominantly expressed. Finally, we demonstrate that dosage-sensitive genes exhibit the same kind of coordinated response to homoeologous exchange as they do for genome duplication, suggesting constraint on dosage balance also acts on these changes to gene dosage.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468929', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'Predicting SARS-CoV-2 epitope-specific TCR recognition using pre-trained protein embeddings', 'authors': \"[{'name': 'Youngmahn Han'}, {'name': 'Aeri Lee'}]\", 'abstract': '<p>The COVID-19 pandemic is ongoing because of the high transmission rate and the emergence of SARS-CoV-2 variants. The <bold>P272L</bold> mutation in SARS-Cov-2 <bold>S</bold>-protein is known to be highly relevant to the viral escape associated with the second pandemic wave in Europe. Epitope-specific T-cell receptor (TCR) recognition is a key factor in determining the T-cell immunogenicity of a SARS-CoV-2 epitope. Although several data-driven methods for predicting epitope-specific TCR recognition have been proposed, they remain challenging owing to the enormous diversity of TCRs and the lack of available training data. Self-supervised transfer learning has recently been demonstrated to be powerful for extracting useful information from unlabeled protein sequences and increasing the predictive performance of the fine-tuned models in downstream tasks.</p><p>Here, we present a predictive model based on Bidirectional Encoder Representations from Transformers (BERT), employing self-supervised transfer learning, to predict SARS-CoV-2 T-cell epitope-specific TCR recognition. The fine-tuned model showed notably high predictive performance for independent evaluation using the SARS-CoV-2 epitope-specific TCR CDR3β sequence datasets. In particular, we found the proline at position 4 corresponding to the <bold>P272L</bold> mutation in the SARS-CoV-2 <bold>S</bold>-protein<sub>269-277</sub> epitope (<bold>YLQPRTFLL</bold>) may contribute substantially to TCR recognition of the epitope through interpreting the output attention weights of our model.</p><p>We anticipate that our findings will provide new directions for constructing a reliable data-driven model to predict the immunogenic T-cell epitopes using limited training data and help accelerate the development of an effective vaccine in response to SARS-CoV-2 variants.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.20.469401', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The Contributions of Eye Gaze Fixations and Target-Lure Similarity to Behavioral and fMRI Indices of Pattern Separation and Pattern Completion', 'authors': \"[{'name': 'Daniel K Bjornn'}, {'name': 'Julie Van'}, {'name': 'Brock Kirwan'}]\", 'abstract': '<p>Pattern separation and pattern completion are generally studied in humans using mnemonic discrimination tasks such as the Mnemonic Similarity Task (MST) where participants identify similar lures and repeated items from a series of images. Failures to correctly discriminate lures are thought to reflect a failure of pattern separation and a propensity toward pattern completion. Recent research has challenged this perspective, suggesting that poor encoding rather than pattern completion accounts for the occurrence of false alarm responses to similar lures. In two experiments, participants completed a continuous recognition task version of the MST while eye movement (Experiment 1 and 2) and fMRI data (Experiment 2) were collected. While we replicated the result that fixation counts at study predicted accuracy on lure trials, we found that target-lure similarity was a much stronger predictor of accuracy on lure trials across both experiments. Lastly, we found that fMRI activation changes in the hippocampus were significantly correlated with the number of fixations at study for correct but not incorrect mnemonic discrimination judgments when controlling for target-lure similarity. Our findings indicate that while eye movements during encoding predict subsequent hippocampal activation changes, mnemonic discrimination performance is better described by pattern separation and pattern completion processes that are influenced by target-lure similarity than simply poor encoding.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.20.469412', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Artificial Intelligence uncovers carcinogenic human metabolites', 'authors': \"[{'name': 'Aayushi Mittal'}, {'name': 'Vishakha Gautam'}, {'name': 'Roshan S'}, {'name': 'Sakshi Arora'}, {'name': 'Sheetanshu Saproo'}, {'name': 'Ria Gupta'}, {'name': 'Sanjay Kumar Mohanty'}, {'name': 'Prakriti Garg'}, {'name': 'Anmol Aggarwal'}, {'name': 'Padmasini R'}, {'name': 'Nilesh Kumar Dixit'}, {'name': 'Vijay Pal Singh'}, {'name': 'Anurag Mehta'}, {'name': 'Juhi Tayal'}, {'name': 'Srivatsava Naidu'}, {'name': 'Debarka Sengupta'}, {'name': 'Gaurav Ahuja'}]\", 'abstract': '<p>The genome of a eukaryotic cell is often vulnerable to both intrinsic and extrinsic threats due to its constant exposure to a myriad of heterogeneous chemical compounds. Despite the availability of innate DNA damage repair pathways, some genomic lesions trigger cells for malignant transformation. Accurate prediction of carcinogens is an ever-challenging task due to the limited information about bonafide (non)carcinogens. This, in turn, constrains the generalisability of such models. We developed a novel ensemble classifier (Metabokiller) that accurately recognizes carcinogens by quantitatively assessing their chemical composition as well as potential to induce proliferation, oxidative stress, genotoxicity, alterations in epigenetic signatures, and activation of anti-apoptotic pathways, therefore obviates the need for bonafide (non)carcinogens for training model. Concomitant with the carcinogenicity prediction, it also reveals the contribution of the aforementioned biochemical processes in carcinogenicity, thereby making the proposed approach highly interpretable. Metabokiller outwits existing best practice methods for the carcinogenicity prediction task. We used Metabokiller to decode the cellular endogenous metabolic threats by screening a large pool of human metabolites and identified putative metabolites that could potentially trigger malignancy in normal cells. To cross-validate our predictions, we performed an array of functional assays and genome-wide transcriptome analysis on two Metabokiller-flagged, and previously uncharacterized human metabolites by using Saccharomyces cerevisiae as a model organism and observed larger synergy with the prediction probabilities. Finally, the carcinogenicity potential of these metabolites was evaluated using a malignancy transformation assay on human cells.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.16.468908', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The mycobacterial mutasome: composition and recruitment in live cells', 'authors': \"[{'name': 'Sophia Gessner'}, {'name': 'Zela Martin'}, {'name': 'Michael A. Reiche'}, {'name': 'Joana A. Santos'}, {'name': 'Neeraj Dhar'}, {'name': 'Ryan Dinkele'}, {'name': 'Timothy De Wet'}, {'name': 'Atondaho Ramudzuli'}, {'name': 'Saber Anoosheh'}, {'name': 'Dirk M. Lang'}, {'name': 'Jesse Aaron'}, {'name': 'Teng-Leong Chew'}, {'name': 'Jennifer Herrmann'}, {'name': 'Rolf Müller'}, {'name': 'John D. McKinney'}, {'name': 'Roger Woodgate'}, {'name': 'Valerie Mizrahi'}, {'name': 'Meindert H. Lamers'}, {'name': 'Digby F. Warner'}]\", 'abstract': '<p>A DNA damage-inducible mutagenic gene cassette has been implicated in the emergence of drug resistance in <italic>Mycobacterium tuberculosis</italic> during anti-tuberculosis (TB) chemotherapy. However, the molecular composition and operation of the encoded “mycobacterial mutasome” – minimally comprising DnaE2 polymerase and ImuA′ and ImuB accessory proteins – remain elusive. Following exposure of mycobacteria to DNA damaging agents, we observe that DnaE2 and ImuB co-localize with the DNA polymerase III β subunit (β clamp) in distinct intracellular foci. Notably, genetic inactivation of the mutasome in an <italic>imuB</italic><sup>AAAAGG</sup> mutant containing a disrupted β clamp-binding motif abolishes ImuB-β clamp focus formation, a phenotype recapitulated pharmacologically by treating bacilli with griselimycin and in biochemical assays in which this β clamp-binding antibiotic collapses pre-formed ImuB-β clamp complexes. These observations establish the essentiality of the ImuB-β clamp interaction for mutagenic DNA repair in mycobacteria, identifying the mutasome as target for adjunctive therapeutics designed to protect anti-TB drugs against emerging resistance.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.14.21266221', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': False, 'title': 'Naloxone Buyers Club: Overlooked Critical Public Health Infrastructure for Preventing Overdose Deaths', 'authors': \"[{'name': 'Maya Doe-Simkins'}, {'name': 'Eliza Jane Wheeler'}, {'name': 'Mary C. Figgatt'}, {'name': 'T. Stephen Jones'}, {'name': 'Alice Bell'}, {'name': 'Peter J. Davidson'}, {'name': 'Nabarun Dasgupta'}]\", 'abstract': '<sec><title>Background</title><p>Community-based naloxone distribution is an evidence-based pillar of overdose prevention. Since 2012, the naloxone Buyers Club facilitated purchase of low-cost naloxone by harm reduction and syringe service programs, the primary conduits for reaching people who use drugs. This innovative purchasing and mutual aid network has not been previously described.</p></sec><sec><title>Methods</title><p>We analyzed transactional records of naloxone orders (2017-2020, n=965), a survey of current Buyers Club members (2020, n=104), and mutual aid requests (2021, n=86).</p></sec><sec><title>Results</title><p>Between 2017 and 2020, annual orders for naloxone increased 2.6-fold. 114 unique harm reduction programs from 40 states placed orders for 3,714,110 vials of 0.4 mg/mL generic naloxone through the Buyers Club. States with most orders were: Arizona (600,000 vials), Illinois (576,800), Minnesota (347,450), California (317,200), North Carolina (315,040). Among programs that ordered naloxone in 2020, 52% (n=32) received no federal funding and ordered half as much as funded programs. During the 2021 shortage, mutual aid redistribution was common, with 80% participating as either a donor or recipient. Among 59 mutual aid requestors, 59% (n=35) were willing to accept expired naloxone; the clear preference was for generic injectable naloxone, 95% (n=56).</p></sec><sec><title>Conclusions</title><p>The naloxone Buyers Club is a critical element of overdose prevention infrastructure. Yet, barriers from corporate compliance officers and federal prescription-only status impede access. These barriers can be reduced by FDA removing the prescription requirement for naloxone and government funding for harm reduction programs.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.19.469233', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'A 1-bp deletion in bovine QRICH2 causes low sperm count and immotile sperm with multiple morphological abnormalities', 'authors': \"[{'name': 'Maya Hiltpold'}, {'name': 'Fredi Janett'}, {'name': 'Xena Marie Mapel'}, {'name': 'Naveen Kumar Kadri'}, {'name': 'Zih-Hua Fang'}, {'name': 'Hermann Schwarzenbacher'}, {'name': 'Franz Seefried'}, {'name': 'Mirjam Spengeler'}, {'name': 'Ulrich Witschi'}, {'name': 'Hubert Pausch'}]\", 'abstract': '<p>Background: Semen quality and male fertility are monitored in artificial insemination bulls to ensure high insemination success rates. Only ejaculates that fulfill minimum quality requirements are processed and eventually used for artificial inseminations. We examined 70,990 ejaculates from 1343 Brown Swiss bulls to identify bulls from which all ejaculates were rejected due to low semen quality. This procedure identified a bull that produced twelve ejaculates with an aberrantly low number of sperm (0.2±0.2 x 10<sup>9</sup> sperm per ml) which were mostly immotile due to multiple morphological abnormalities.\\nResults: The genome of the bull was sequenced at 12-fold coverage to investigate a suspected genetic cause. Comparing the sequence variant genotypes of the bull with those from 397 fertile bulls revealed a 1-bp deletion in the coding sequence of <italic>QRICH2</italic> encoding glutamine rich 2 as a compelling candidate causal variant. The 1-bp deletion causes a frameshift in translation and induces a premature termination codon (ENSBTAP00000018337.1:p.Cys1644AlafsTer52). The analysis of testis transcriptomes from 76 bulls showed that the transcript with the premature termination codon is subjected to nonsense-mediated mRNA decay. The 1-bp deletion resides on a 675 kb haplotype spanning 181 SNPs from the Illumina BovineHD Bead chip. The haplotype segregates at a frequency of 5% in the Brown Swiss cattle population. This analysis also identified another bull that carried the 1-bp deletion in the homozygous state. Semen analyses from the second bull confirmed low sperm concentration and immotile sperm with multiple morphological abnormalities primarily affecting the sperm flagellum and, to a lesser extent, the sperm head.\\nConclusions: A recessive loss-of-function allele of bovine <italic>QRICH2</italic> likely causes low sperm concentration and immotile sperm with multiple morphological abnormalities. Routine sperm analyses unambiguously identify homozygous bulls. A direct gene test can be implemented to monitor the frequency of the undesired allele in cattle populations.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468767', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Antenna modification leads to enhanced nitrogenase activity in a high light tolerant cyanobacterium', 'authors': \"[{'name': 'Anindita Bandyopadhyay'}, {'name': 'Zi Ye'}, {'name': 'Zuzana Benedikty'}, {'name': 'Martin Trtilek'}, {'name': 'Himadri B. Pakrasi'}]\", 'abstract': '<p>Biological nitrogen fixation is an energy intensive process that contributes significantly towards supporting life on this planet. Among nitrogen-fixing organisms, cyanobacteria remain unrivaled in their ability to fuel the energetically expensive nitrogenase reaction with photosynthetically harnessed solar energy. In heterocystous cyanobacteria light-driven, photosystem I (PSI)-mediated ATP synthesis plays a key role in propelling the nitrogenase reaction. Efficient light transfer to the photosystems rely on phycobilisomes (PBS), the major antenna protein complexes. PBS undergo degradation as a natural response to nitrogen starvation. Upon nitrogen availability, these proteins are resynthesized back to normal levels in vegetative cells, but their occurrence and function in heterocysts remains inconclusive. <italic>Anabaena</italic> 33047 is a heterocystous cyanobacterium that thrives under high light, harbors higher amounts of PBS in its heterocysts and fixes nitrogen at higher rates compared to other heterocystous cyanobacteria. To assess the relationship between PBS in heterocysts and nitrogenase function, we engineered a strain that retains high amounts of the antenna proteins in its heterocysts. Intriguingly, under high light intensities the engineered strain exhibited unusually high rates of nitrogenase activity compared to the wild type. Spectroscopic analysis revealed altered PSI kinetics in the mutant, with increased cyclic electron flow around PSI, a route that contributes to ATP generation and nitrogenase activity in heterocysts. Retaining higher levels of PBS in heterocysts appears to be an effective strategy to enhance nitrogenase function in cyanobacteria that are equipped with the machinery to operate under high light intensities.</p><sec><title>Importance</title><p>The function of phycobilisomes, the large antenna protein complexes in heterocysts has long been debated. This study provides direct evidence of the involvement of these proteins in supporting nitrogenase activity in <italic>Anabaena</italic> 33047, a heterocystous cyanobacterium that has affinity for very high light intensities. This strain was previously known to be recalcitrant to genetic manipulation and hence despite its many appealing traits, remained largely unexplored. We developed a genetic modification system for this strain and generated a Δ<italic>nblA</italic> mutant that exhibited resistance to phycobilisome degradation upon nitrogen starvation. Physiological characterization of the strain indicated that PBS degradation is not essential for acclimation to nitrogen deficiency and retention of PBS is advantageous for nitrogenase function.</p></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.19.469224', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Cleavage of PGAM5 by the intramembrane protease PARL is governed by transmembrane helix dynamics and oligomeric state', 'authors': \"[{'name': 'Varena Siebert'}, {'name': 'Mara Silber'}, {'name': 'Elena Heuten'}, {'name': 'Claudia Muhle-Goll'}, {'name': 'Marius K. Lemberg'}]\", 'abstract': '<p>The intramembrane protease PARL is a crucial mitochondrial safeguard by cleaving the mitophagy regulators PINK1 and PGAM5. PGAM5 substrate determinates have not been rigorously investigated and it is unclear how uncoupling the mitochondrial membrane potential regulates its processing inversely to PINK1. Here we show that in PGAM5 several hydrophilic residues distant from the cleavage site serve as key determinant for PARL-catalyzed cleavage. NMR analysis indicates that a short N-terminal amphipathic helix, followed by a kink and a C-terminal helix harboring the scissile peptide bond, is key for a productive interaction with PARL. In difference to PINK1, PGAM5 is stably inserted into the inner mitochondrial membrane until uncoupling the membrane potential triggers its disassembly into monomers that are vulnerable to PARL-catalyzed processing. We suggest a model in which PGAM5 is a slowly processed substrate with PARL-catalyzed cleavage that is influenced by multiple hierarchical substrate features including a membrane-potential-dependent oligomeric switch.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.19.469195', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'A Time-calibrated Firefly (Coleoptera: Lampyridae) Phylogeny: Using Genomic Data for Divergence Time Estimation', 'authors': \"[{'name': 'Sebastian Hoehna'}, {'name': 'Sarah E Lower'}, {'name': 'Pablo Duchen'}, {'name': 'Ana Catalan'}]\", 'abstract': '<p>Fireflies (Coleoptera: Lampyridae) consist of over 2,000 described extant species. A well-resolved phylogeny of fireflies is important for the study of their bioluminescence, evolution, and conservation. We used a recently published anchored hybrid enrichment dataset (AHE; 436 loci for 88 Lampyridae species and 10 outgroup species) and state-of-the-art statistical methods (the fossilized birth-death-range process implemented in a Bayesian framework) to estimate a time-calibrated phylogeny of Lampyridae. Unfortunately, estimating calibrated phylogenies using AHE and the latest and most robust time-calibration strategies is not possible because of computational constraints. As a solution, we subset the full dataset and applied three different strategies: using the most complete loci, the most homogeneous loci, and the loci with the highest accuracy to infer the well established Photinus clade. The estimated topology using the three data subsets agreed on almost all major clades and only showed minor discordance with less supported nodes. The estimated divergence times overlapped for all nodes that are shared between the topologies. Thus, divergence time estimation is robust as long as the topology inference is robust and any well selected data subset suffices. Additionally, we observed an unexpected amount of gene tree discordance between the 436 AHE loci. Our assessment of model adequacy showed that standard phylogenetic substitution models are not adequate for any of the 436 AHE loci which is likely to bias phylogenetic inferences. We performed a simulation study to explore the impact of (a) incomplete lineage sorting, (b) uniformly distributed and systematic missing data, and (c) systematic bias in the position of highly variable and conserved sites. For our simulated data, we observed less gene tree variation and hence the empirically observed amount of gene tree discordance for the AHE dataset is unexpected.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.15.468702', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Effector loss drives adaptation of Pseudomonas syringae pv. actinidiae to Actinidia arguta', 'authors': \"[{'name': 'Lauren M. Hemara'}, {'name': 'Jay Jayaraman'}, {'name': 'Paul W. Sutherland'}, {'name': 'Mirco Montefiori'}, {'name': 'Saadiah Arshed'}, {'name': 'Abhishek Chatterjee'}, {'name': 'Ronan Chen'}, {'name': 'Mark Andersen'}, {'name': 'Carl H. Mesarich'}, {'name': 'Otto van der Linden'}, {'name': 'Magan M. Schipper'}, {'name': 'Joel L. Vanneste'}, {'name': 'Cyril Brendolise'}, {'name': 'Matthew D. Templeton'}]\", 'abstract': '<p>A pandemic isolate of <italic>Pseudomonas syringae</italic> pv. <italic>actinidiae</italic> biovar 3 (Psa3) has devastated kiwifruit orchards growing cultivars of <italic>Actinidia chinensis</italic>. In contrast, <italic>A. arguta</italic> (kiwiberry) is resistant to Psa3. This resistance is mediated via effector-triggered immunity, as demonstrated by induction of the hypersensitive response in infected <italic>A. arguta</italic> leaves, observed by microscopy and quantified by ion-leakage assays. Isolates of Psa3 that cause disease in <italic>A. arguta</italic> have been isolated and analyzed, revealing a 49 kb deletion in the exchangeable effector locus (EEL). This natural EEL-mutant isolate and strains with synthetic knockouts of the EEL were more virulent in <italic>A. arguta</italic> plantlets than wild-type Psa3. Screening of a complete library of Psa3 effector knockout strains identified increased growth <italic>in planta</italic> for knockouts of four effectors – AvrRpm1a, HopF1c, HopZ5a, and the EEL effector HopAW1a – suggesting a resistance response in <italic>A. arguta</italic>. Hypersensitive response (HR) assays indicate that three of these effectors trigger a host species-specific HR. A Psa3 strain with all four effectors knocked out escaped host recognition, but a cumulative increase in bacterial pathogenicity and virulence was not observed. These avirulence effectors can be used in turn to identify the first cognate resistance genes in <italic>Actinidia</italic> for breeding durable resistance into future kiwifruit cultivars.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469146', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Profile of the in silico secretome of the palm dieback pathogen, a fungus that puts natural oases at risk', 'authors': \"[{'name': 'Maryam Rafiqi'}, {'name': 'Lukas Jelonek'}, {'name': 'Aliou M. Diouf'}, {'name': 'AbdouLahat Mbaye'}, {'name': 'Martijn Rep'}, {'name': 'Alhousseine Diarra'}]\", 'abstract': '<p>Understanding biotic changes that occur alongside climate change constitute a research priority of global significance. Here, we address a plant pathogen that poses a serious threat to life on natural oases, where climate change is already taking a toll and severely impacting human subsistence. <italic>Fusarium oxysporum</italic> f. sp. <italic>albedinis</italic> is a pathogen that causes dieback disease on date palms, a tree that provides several critical ecosystem services in natural oases; and consequently, of major importance in this vulnerable habitat. Here, we assess the current state of global pathogen spread, we annotate the genome of a sequenced pathogen strain isolated from the native range and we analyse its <italic>in silico</italic> secretome. The palm dieback pathogen secretes a large arsenal of effector candidates including a variety of toxins, a distinguished profile of secreted in xylem proteins (SIX) as well as an expanded protein family with an N-terminal conserved motif [SG]PC[KR]P that could be involved in interactions with host membranes. Using agrobiodiversity as a strategy to decrease pathogen infectivity, while providing short term resilient solutions, seems to be widely overcome by the pathogen. Hence, the urgent need for future mechanistic research on the palm dieback disease and a better understanding of pathogen genetic diversity.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.18.469104', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Effortless integration of probabilistic visual input', 'authors': \"[{'name': 'Andrey Chetverikov'}, {'name': 'Árni Kristjánsson'}]\", 'abstract': '<p>Prominent theories of perception suggest that the brain builds probabilistic models of the world, assessing the statistics of the visual input to inform this construction. However, the evidence for this idea is often based on simple impoverished stimuli, and the results have often been discarded as an illusion reflecting simple \"summary statistics\" of visual inputs. Here we show that the visual system represents probabilistic distributions of complex heterogeneous stimuli. Importantly, we show how these statistical representations are integrated with representations of other features and bound to locations, and can therefore serve as building blocks for object and scene processing. We uncover the organization of these representations at different spatial scales by showing how expectations for incoming features are biased by neighboring locations. We also show that there is not only a bias, but also a skew in the representations, arguing against accounts positing that probabilistic representations are discarded in favor of simplified summary statistics (e.g., mean and variance). In sum, our results reveal detailed probabilistic encoding of stimulus distributions, representations that are bound with other features and to particular locations.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468283', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Mutagenic distinction between the receptor-binding and fusion subunits of the SARS-CoV-2 spike glycoprotein', 'authors': \"[{'name': 'Robert Penner'}]\", 'abstract': '<p>We observe that a residue R of the spike glycoprotein of SARS-CoV-2 which has mutated in one or more of the current Variants of Concern or Interest and under Monitoring rarely participates in a backbone hydrogen bond if R lies in the S<sub>1</sub> subunit and usually participates in one if R lies in the S<sub>2</sub> subunit. A possible explanation for this based upon free energy is explored as a potentially general principle in the mutagenesis of viral glycoproteins. This observation could help target future vaccine cargos for the evolving coronavirus as well as more generally.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.16.465905', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Aging and interferon gamma response drive the phenotype of neutrophils in the inflamed joint', 'authors': \"[{'name': 'Ricardo Grieshaber-Bouyer'}, {'name': 'Tarik Exner'}, {'name': 'Nicolaj S. Hackert'}, {'name': 'Felix A. Radtke'}, {'name': 'Scott A. Jelinsky'}, {'name': 'Olha Halyabar'}, {'name': 'Alexandra Wactor'}, {'name': 'Elham Karimizadeh'}, {'name': 'Jorge Schettini'}, {'name': 'A. Helena Jonsson'}, {'name': 'Deepak A. Rao'}, {'name': 'Lauren A. Henderson'}, {'name': 'Carsten Müller-Tidow'}, {'name': 'Hanns-Martin Lorenz'}, {'name': 'Guido Wabnitz'}, {'name': 'James A. Lederer'}, {'name': 'Angela Hadjipanayis'}, {'name': 'Peter A. Nigrovic'}]\", 'abstract': '<sec><title>Objectives</title><p>Neutrophils are typically the most abundant leukocyte in arthritic synovial fluid. We sought to understand changes that occur in neutrophils as they migrate from blood to joint.</p></sec><sec><title>Methods</title><p>We performed RNA sequencing of neutrophils from healthy human blood, arthritic blood, and arthritic synovial fluid, comparing transcriptional signatures with those from murine K/BxN serum transfer arthritis. We employed mass cytometry to quantify protein expression and sought to reproduce the synovial fluid phenotype <italic>ex vivo</italic> in cultured healthy blood neutrophils.</p></sec><sec><title>Results</title><p>Blood neutrophils from healthy donors and patients with active arthritis exhibited largely similar transcriptional signatures. By contrast, synovial fluid neutrophils exhibited more than 1,600 differentially expressed genes. Gene signatures identified a prominent response to interferon gamma (IFNγ), as well as to tumor necrosis factor, interleukin 6, and hypoxia, in both humans and mice. Mass cytometry also found healthy and arthritic donor blood neutrophils largely indistinguishable but revealed a range of neutrophil phenotypes in synovial fluid defined by downregulation of CXCR1 and upregulation of FcγRI, HLA-DR, PD-L1, ICAM-1 and CXCR4. Reproduction of key elements of this signature in cultured blood neutrophils required both IFNγ and prolonged culture.</p></sec><sec><title>Conclusions</title><p>Circulating neutrophils from arthritis patients resemble those from healthy controls, but joint fluid cells exhibit a network of changes, conserved across species, that implicate IFNγ response and aging as complementary drivers of the synovial neutrophil phenotype.</p></sec><sec><title>KEY MESSAGES</title><sec><title>What is already known about this subject?</title><list list-type=\"bullet\"><list-item><p>Neutrophils are central in the effector phase of inflammatory arthritis but their phenotypic heterogeneity in inflamed synovial fluid is poorly understood.</p></list-item></list></sec><sec><title>What does this study add?</title><list list-type=\"bullet\"><list-item><p>RNA-seq and mass cytometry identify a hallmark phenotype of neutrophils in synovial fluid consisting of upregulated ICAM-1, HLA-DR, PD-L1, Fc receptors and CXCR4.</p></list-item><list-item><p>Transcriptomics highlight an IFNγ response signature conserved across humans and mice.</p></list-item><list-item><p>In vitro experiments implicate aging and IFNγ as complementary factors orchestrating the synovial fluid neutrophil phenotype.</p></list-item></list></sec><sec><title>How might this impact on clinical practice or future developments?</title><list list-type=\"bullet\"><list-item><p>Understanding the specific features of neutrophils in the arthritic joint may disclose opportunities for safe therapeutic targeting of this lineage.</p></list-item></list></sec></sec>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.16.21266408', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'HLA-A*03:01 is associated with increased risk of fever, chills, and more severe reaction to Pfizer-BioNTech COVID-19 vaccination', 'authors': \"[{'name': 'Alexandre Bolze'}, {'name': 'Iva Neveux'}, {'name': 'Kelly M Schiabor Barrett'}, {'name': 'Simon White'}, {'name': 'Magnus Isaksson'}, {'name': 'Shaun Dabe'}, {'name': 'William Lee'}, {'name': 'Joseph J Grzymski'}, {'name': 'Nicole L Washington'}, {'name': 'Elizabeth T Cirulli'}]\", 'abstract': '<p>COVID-19 vaccines are safe and highly effective, but some individuals experience unpleasant reactions to vaccination. As the majority of adults in the US have received a COVID-19 vaccine this year, there is an unprecedented opportunity to study the genetics of reactions to vaccination via surveys of individuals who are already part of genetic research studies. Here, we have queried 17,440 participants in the Helix DNA Discovery Project and Healthy Nevada Project about their reactions to COVID-19 vaccination. Our GWAS identifies an association between severe difficulties with daily routine after vaccination and HLA-A*03:01. This association was statistically significant only for those who received the Pfizer-BioNTech vaccine (BNT162b2; p=4.70E-11), but showed a trending association in those who received the Moderna vaccine (mRNA-1273; p=0.005) despite similar sample sizes for study. In Pfizer-BioNTech recipients, HLA-A*03:01 was associated with a two-fold increase in risk of severe vaccine reactions. The effect was consistent across ages, sexes, and whether the person had previously had a COVID-19 infection. The reactions experienced by HLA-A*03:01 carriers were driven by associations with chills, fever, fatigue, and in general feeling unwell.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.20.469375', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Balancing selection and the crossing of fitness valleys in structured populations: diversification in the gametophytic self-incompatibility system', 'authors': \"[{'name': 'Roman Stetsenko'}, {'name': 'Thomas Brom'}, {'name': 'Vincent Castric'}, {'name': 'Sylvain Billiard'}]\", 'abstract': '<p>The self-incompatibility locus (S-locus) of flowering plants displays a striking allelic diversity. How such a diversity has emerged remains unclear. In this paper, we performed numerical simulations in a finite island population genetics model to investigate how population subdivision affects the diversification process at a S-locus, given that the two-genes architecture typical of S-loci involves the crossing of a fitness valley. We show that population structure increases the number of self-incompatibility haplotypes (S-haplotypes) maintained in the whole metapopulation, but at the same time also slightly reduces the parameter range allowing for their diversification. This increase is partly due to a reinforcement of the diversification and replacement dynamics of S-haplotypes within and among demes. We also show that the two-genes architecture leads to a higher diversity compared with a simpler genetic architecture where new S-haplotypes appear in a single mutation step. We conclude that population structure helps explain the large allelic diversity at the self-incompatibility locus. Overall, our results suggest that population subdivision can act in two opposite directions: it makes easier S-haplotypes diversification but increases the risk that the SI system is lost.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.15.468130', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Genome editing of Capsaspora owczarzaki suggests an ancestral function of the Hippo signaling effector YAP/TAZ/Yorkie in cytoskeletal dynamics but not proliferation', 'authors': \"[{'name': 'Jonathan E Phillips'}, {'name': 'Maribel Santos'}, {'name': 'Mohammed Kanchwala'}, {'name': 'Chao Xing'}, {'name': 'Duojia Pan'}]\", 'abstract': '<p>Many genes that function in animal development are present in the close unicellular relatives of animals, but little is known regarding the premetazoan function of these genes. Here, we develop techniques for genetic manipulation in the filasterean <italic>Capsaspora owczarzaki</italic> and use these tools to characterize the <italic>Capsaspora</italic> ortholog of the Hippo signaling nuclear effector YAP/TAZ/Yorkie (coYki). In contrast to its potent oncogene activity in metazoans, we show that coYki is dispensable for cell proliferation but regulates cytoskeletal dynamics and the morphology of multicellular aggregates in <italic>Capsaspora</italic>. Our results suggest an ancestral role for the Hippo pathway in cytoskeletal regulation, which was later co-opted to regulate cell proliferation in animals.</p><sec><title>One Sentence Summary</title><p>Gene disruption in a unicellular holozoan suggests an ancestral role for YAP/TAZ/Yorkie in cytoskeletal regulation and cell aggregation but not cell proliferation.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.16.468852', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Structure of the rabies virus glycoprotein trimer bound to a pre-fusion specific neutralizing antibody', 'authors': \"[{'name': 'Heather M. Callaway'}, {'name': 'Dawid Zyla'}, {'name': 'Kathryn M. Hastie'}, {'name': 'Ruben Diaz Avalos'}, {'name': 'Alyssa Agarwal'}, {'name': 'Hervé Bourhy'}, {'name': 'Davide Corti'}, {'name': 'Erica Ollmann Saphire'}]\", 'abstract': '<p>Rabies infection is nearly 100% lethal if untreated and kills over 50,000 people annually, many of them children. Existing rabies vaccines target the rabies virus glycoprotein (RABV-G) but generate short-lived immune responses, likely because the protein is heterogeneous under physiological conditions. Here, we report the 3.39Å cryo-EM structure of trimeric, pre-fusion RABV-G complexed with RVA122, a potently neutralizing human antibody. RVA122 binds to a quaternary epitope at the top of RABV-G, bridging domains and stabilizing RABV-G protomers in a prefusion state. RABV-G trimerization involves side-to-side interactions between the central α-helix and adjacent loops, rather than contacts between central helices, and interactions among the fusion loops at the glycoprotein base. These results provide a basis to develop improved rabies vaccines based on RABV-G stabilized in the prefusion conformation.</p><sec><title>One sentence summary</title><p>We report the structure and trimeric interface of pre-fusion rabies virus glycoprotein bound to the neutralizing antibody RVA122.</p></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.17.468953', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'An assessment of the evidence for antibacterial activity of stinging nettle (Urtica dioica) extracts', 'authors': \"[{'name': 'Freya Harrison'}, {'name': 'Jessica Furner-Pardoe'}, {'name': 'Erin Connelly'}]\", 'abstract': '<p>Stinging nettles (<italic>Urtica</italic> spp.) have been used in a diverse range of traditional and historical medicines from around the world for the treatment of skin diseases, wounds, urinary disorders, respiratory diseases, bone and joint pain, anaemia and other circulatory problems, as well as in cosmetic preparations for skin and haircare. As part of an interdisciplinary exploration of nettle-based remedies, we performed a systematic review of published evidence for the antimicrobial activity of <italic>Urtica</italic> spp. extracts against bacteria and fungi that commonly cause skin, soft tissue and respiratory infections. We focussed on studies in which minimum inhibitory concentration (MIC) assays of <italic>U. dioica</italic> were conducted on the common bacterial opportunistic pathogens <italic>Escherichia coli</italic>, <italic>Pseudomonas aeruginosa</italic>, <italic>Klebsiella pneumoniae</italic> and <italic>Staphylococcus aureus</italic>. No studies used fresh leaves (all were dried prior to use), and no studies prepared nettles in weak acid (corresponding to vinegar) or in fats/oils, which are common combinations in historical and traditional preparations. We addressed this gap by conducting new antibacterial tests of extracts of fresh <italic>U. dioica</italic> leaves prepared in vinegar, butter or olive oil against <italic>P. aeruginosa</italic> and <italic>S. aureus</italic>. Our systematic review and additional experimental data leads us to conclude that there is no strong evidence for nettles containing molecules with clinically useful antimicrobial activity. It seems most likely that the utility of nettles in traditional topical preparations for wounds may simply be as a “safe” absorbent medium for keeping antibacterial (vinegar) or emollient (oils) ingredients at the treatment site.</p><sec><title>Data Summary</title><p>All data relating to the systematic review are supplied in the Datasets and Supplementary tables contained in the Supporting Data file.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.19.469306', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Enhancer and super-enhancer landscape in polycystic kidney disease', 'authors': \"[{'name': 'Ronak Lakhia'}, {'name': 'Abheepsa Mishra'}, {'name': 'Laurence Biggers'}, {'name': 'Venakt Malladi'}, {'name': 'Patricia Cobo-Stark'}, {'name': 'Sachin Hajarnis'}, {'name': 'Vishal Patel'}]\", 'abstract': '<p>Widespread aberrant gene expression is pathological hallmark of polycystic kidney disease (PKD). Numerous pathogenic signaling cascades, including c-Myc, Fos, and Jun are transactivated. However, the underlying epigenetic regulators are poorly defined. Here we show that H3K27ac, a histone modification that marks active enhancers, is elevated in mouse and human ADPKD samples. Using comparative H3K27ac ChIP-Seq analysis, we mapped &gt;16000 active intronic and intergenic enhancer elements in Pkd1-mutant mouse kidneys. We find that the cystic kidney epigenetic landscape resembles that of a developing kidney, and &gt;90% of upregulated genes in Pkd1-mutant kidneys are co-housed with activated enhancers in the same topologically associated domains. Furthermore, we identify an evolutionarily-conserved enhancer cluster downstream of the c-Myc gene and super-enhancers flanking both Jun and Fos loci in mouse and human ADPKD models. Deleting these regulatory elements reduces c-Myc, Jun, or Fos abundance and suppresses proliferation and 3D cyst growth of Pkd1-mutant cells. Finally, inhibiting glycolysis and glutaminolysis or activating Ppara in Pkd1-mutant cells lowers global H3K27ac levels and on c-Myc enhancers. Thus, our work suggests that epigenetic rewiring mediates the transcriptomic dysregulation in PKD, and the regulatory elements can be targeted to slow cyst growth.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468903', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Orthogonal control of gene expression in plants using synthetic promoters and CRISPR-based transcription factors', 'authors': \"[{'name': 'Shaunak Kar'}, {'name': 'Yogendra Bordiya'}, {'name': 'Nestor Rodriguez'}, {'name': 'Junghyun Kim'}, {'name': 'Elizabeth C. Gardner'}, {'name': 'Jimmy Gollihar'}, {'name': 'Sibum Sung'}, {'name': 'Andrew D. Ellington'}]\", 'abstract': '<sec><title>Background</title><p>The construction and application of synthetic genetic circuits is frequently improved if gene expression can be orthogonally controlled, relative to the host. In plants, orthogonality can be achieved via the use of CRISPR-based transcription factors that are programmed to act on natural or synthetic promoters. The construction of complex gene circuits can require multiple, orthogonal regulatory interactions, and this in turn requires that the full programmability of CRISPR elements be adapted to non-natural and non-standard promoters that have few constraints on their design. Therefore, we have developed synthetic promoter elements in which regions upstream of the minimal 35S CaMV promoter are designed from scratch to interact via programmed gRNAs with dCas9 fusions that allow activation of gene expression.</p></sec><sec><title>Results</title><p>A panel of three, mutually orthogonal promoters that can be acted on by artificial gRNAs bound by CRISPR regulators were designed. Guide RNA expression targeting these promoters was in turn controlled by either Pol III (U6) or ethylene-inducible Pol II promoters, implementing for the first time a fully artificial Orthogonal Control System (OCS). Following demonstration of the complete orthogonality of the designs, the OCS was tied to cellular metabolism by putting gRNA expression under the control of an endogenous plant signaling molecule, ethylene. The ability to form complex circuitry was demonstrated via the ethylene-driven, ratiometric expression of fluorescent proteins in single plants.</p></sec><sec><title>Conclusions</title><p>The design of synthetic promoters is highly generalizable to large tracts of sequence space, allowing Orthogonal Control Systems of increasing complexity to potentially be generated at will. The ability to tie in several different basal features of plant molecular biology (Pol II and Pol III promoters, ethylene regulation) to the OCS demonstrates multiple opportunities for engineering at the system level. Moreover, given the fungibility of the core 35S CaMV promoter elements, the derived synthetic promoters can potentially be utilized across a variety of plant species.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.16.468834', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': True, 'title': 'Global Mutational Sweep of SARS-CoV-2: from Chaos to Order', 'authors': \"[{'name': 'Xin Wang'}, {'name': 'Mingda Hu'}, {'name': 'Yuan Jin'}, {'name': 'Boqian Wang'}, {'name': 'Yunxiang Zhao'}, {'name': 'Long Liang'}, {'name': 'Junjie Yue'}, {'name': 'Hongguang Ren'}]\", 'abstract': '<p>Analysis of large-scale genome sequences demonstrates the mutation of SARS-CoV-2 has been undergoing significant sweeps. Driven by emerging variants, global sweeps are accelerated and purified over time. This may prolong the pandemic with repeating epidemics, presenting challenges to the control and prevention of SARS-CoV-2.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.19.469307', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Constructing threat probability, fear behaviour, and aversive prediction error in the brainstem.', 'authors': \"[{'name': 'Jasmin Strickland'}, {'name': 'Michael McDannald'}]\", 'abstract': '<p>When faced with potential threat we must estimate its probability, respond advantageously, and leverage experience to update future estimates. Threat estimates are the proposed domain of the forebrain, while behaviour is elicited by the brainstem. Yet, the brainstem is also a source of prediction error, a learning signal to acquire and update threat estimates. Neuropixels probes allowed us to record single-unit activity across a 21-region brainstem axis during probabilistic fear discrimination. Against a backdrop of widespread threat probability and behaviour signaling, a dorsally-based brainstem network rapidly signaled threat probability. Remapping of neuronal function following shock outcome gave rise to brainstem networks signaling prediction error on multiple timescales. The results reveal construction of threat probability, behaviour, and prediction error along a single brainstem axis.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.17.468978', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': \"Identification of antibiotic resistance proteins via MiCId's augmented workflow. A mass spectrometry-based proteomics approach\", 'authors': \"[{'name': 'Gelio Alves'}, {'name': 'Aleksey Y Ogurtsov'}, {'name': 'Roger Karlsson'}, {'name': 'Daniel Jaen-Luchoro'}, {'name': 'Beatriz Pineiro-Iglesias'}, {'name': 'Francisco Salva-Serra'}, {'name': 'Bjorn Andersson'}, {'name': 'Edward Moore'}, {'name': 'Yi-Kuo Yu'}]\", 'abstract': \"<p>Fast and accurate identifications of pathogenic bacteria along with their associated antibiotic resistance proteins are  of paramount importance for patient treatments and public health. While mass spectrometry has become an important, technique for diagnostics of infectious disease, there is a need for mass spectrometry workflows offering this capability. To meet this need, we have augmented the previously published Microorganism Classification and Identification (MiCId) workflow for this capability. To evaluate the performance of the newly augmented MiCId workflow, we have used  MS/MS datafiles from samples of 10 antibiotic resistance bacterial strains belonging to three different species: Escherichia coli, Klebsiella pneumoniae, and Pseudomonas aeruginosa. The evaluation results show that MiCId's workflow has a sensitivity value around 85% (with a lower bound at about 72%) and a precision greater than 95% in the identification of antibiotic resistance  proteins. Using MS/MS datasets from samples of two bacterial clonal isolates, one being antibiotic-sensitive while the other (obtained from the same patient at different times) being multidrug-resistant, we applied MiCId's workflow to investigate possible mechanisms of antibiotic resistance in these pathogenic bacteria; the results showed that MiCId's conclusions are in agreement with the published study. Furthermore, we show that MiCId's workflow is fast. It provides microorganismal identifications, protein identifications, sample biomass estimates, and antibiotic resistance protein identifications in 6-17 minutes per MS/MS sample using computing resources that are available in most desktop and laptop computers, making it a highly portable workflow. This study demonstrated that MiCId's workflow is fast, portable, and with high sensitivity and high precision, making it a valuable tool for rapid identifications of bacteria as well as detection of their antibiotic resistance  proteins. The new version of MiCId (v.07.01.2021) is freely available for download at https://www.ncbi.nlm.nih.gov/CBBresearch/Yu/downloads.html.</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468769', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Effects of Chronic Intermittent Hypoxia and Chronic Sleep Fragmentation on Gut Microbiome, Serum Metabolome, Liver and Adipose Tissue Morphology', 'authors': \"[{'name': 'Fan Wang'}, {'name': 'Juanjuan Zou'}, {'name': 'Huajun Xu'}, {'name': 'Weijun Huang'}, {'name': 'Xiaoman Zhang'}, {'name': 'Zhicheng Wei'}, {'name': 'Xinyi Li'}, {'name': 'Yupu Liu'}, {'name': 'Jianyin Zou'}, {'name': 'Feng Liu'}, {'name': 'Huaming Zhu'}, {'name': 'Hongliang Yi'}, {'name': 'Jian Guan'}, {'name': 'Shankai Yin'}]\", 'abstract': '<p>Chronic intermittent hypoxia (CIH) and chronic sleep fragmentation (CSF) are two cardinal pathological features of obstructive sleep apnea (OSA). Dietary obesity is a crucial risk intermediator for OSA and metabolic disorders. Gut microbiota affect hepatic and adipose tissue morphology under conditions of CIH or CSF through downstream metabolites. However, the exact relationship is unclear. Herein, chow and high-fat diet (HFD)-fed mice were subjected to CIH or CSF for 10 weeks each and compared to normoxia (NM) or normal sleep (NS) controls. 16S rRNA amplicon sequencing, untargeted liquid chromatography-tandem mass spectrometry, and histological assessment of liver and adipose tissues were used to investigate the correlations between the microbiome, metabolome, and lipid metabolism under CIH or CSF condition. Our results demonstrated that CIH and CSF regulate the abundance of intestinal microbes (such as <italic>Akkermansia mucinphila</italic>, <italic>Clostridium spp.</italic>, <italic>Lactococcus spp.</italic>, <italic>and Bifidobacterium spp</italic>.) and functional metabolites, such as tryptophan, free fatty acids, branched amino acids, and bile acids, which influence adipose tissue and hepatic lipid metabolism, and the level of lipid deposition in tissues and peripheral blood. In conclusion, CIH and CSF adversely affect fecal microbiota composition and function, and host metabolism; these findings provide new insight into the independent and synergistic effects of CIH, CSF, and HFD on lipid disorders.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.18.469134', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Distinct neurocognitive bases for social trait judgments of faces in autism spectrum disorder', 'authors': \"[{'name': 'Hongbo Yu'}, {'name': 'Runnan Cao'}, {'name': 'Chujun Lin'}, {'name': 'Shuo Wang'}]\", 'abstract': '<p>Autism spectrum disorder (ASD) is characterized by difficulties in social processes, interactions, and communication. Yet, the neurocognitive bases underlying these difficulties are unclear. Here, we triangulated the trans-diagnostic approach to personality, social trait judgments of faces, and neurophysiology to investigate (1) the relative position of autistic traits in a comprehensive social-affective personality space and (2) the distinct associations between the social-affective personality dimensions and social trait judgment from faces in individuals with ASD and neurotypical individuals. We collected personality and facial judgment data from a large sample of online participants (N = 89 self-identified ASD; N = 308 neurotypical controls). Factor analysis with 33 sub-scales of 10 social-affective personality questionnaires identified a 4-dimensional personality space. This analysis revealed that ASD and control participants did not differ significantly along the personality dimensions of empathy and prosociality, antisociality, or social agreeableness. However, the associations between these dimensions and judgments of facial trustworthiness and warmth differed across groups. Neurophysiological data also indicated that ASD and control participants might rely on distinct neuronal representations for judging trustworthiness and warmth from faces. These results suggest that the atypical association between social-affective personality and social trait judgment from faces may contribute to the social and affective difficulties associated with ASD.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.15.468656', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'A nuclear cAMP microdomain suppresses tumor growth by Hippo pathway inactivation', 'authors': \"[{'name': 'Marek M. Drozdz'}, {'name': 'Ashley S. Doane'}, {'name': 'Rached Alkallas'}, {'name': 'Garrett Desman'}, {'name': 'Rohan Bareja'}, {'name': 'Michael Reilly'}, {'name': 'Jakyung Bang'}, {'name': 'Maftuna Yusupova'}, {'name': 'Jaewon You'}, {'name': 'Jenny Z. Wang'}, {'name': 'Akansha Verma'}, {'name': 'Kelsey Aguirre'}, {'name': 'Elsbeth Kane'}, {'name': 'Ian R. Watson'}, {'name': 'Olivier Elemento'}, {'name': 'Elena Piskounova'}, {'name': 'Taha Merghoub'}, {'name': 'Jonathan H. Zippin'}]\", 'abstract': '<p>cAMP signaling pathways are critical for both oncogenesis and tumor suppression. cAMP signaling is localized to multiple spatially distinct microdomains, but the role of cAMP microdomains in cancer cell biology is poorly understood. We developed a tunable genetic system that allows us to activate cAMP signaling in specific microdomains. We uncovered a previously unappreciated nuclear cAMP microdomain that functionally activates a tumor suppressive pathway in a broad range of cancers by inhibiting YAP, a key effector protein of the Hippo pathway, inside the nucleus. We show that nuclear cAMP induces a LATS-dependent pathway leading to phosphorylation of nuclear YAP solely at serine 397, export of YAP from the nucleus, without YAP protein degradation. Thus, nuclear cAMP inhibition of nuclear YAP is distinct from other known mechanisms of Hippo regulation. Pharmacologic targeting of specific cAMP microdomains remains an untapped therapeutic approach for cancer, and since Hippo pathway deregulation can lead to oncogenesis and chemotherapeutic resistance, drugs directed at the nuclear cAMP microdomain may provide new avenues for the treatment of cancer.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.17.468872', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'A3D Database: Structure-based Protein Aggregation Predictions for the Human Proteome', 'authors': \"[{'name': 'Aleksandra Elzbieta Badaczewska-Dawid'}, {'name': 'Javier Garcia-Pardo'}, {'name': 'Aleksander Kuriata'}, {'name': 'Jordi Pujols'}, {'name': 'Salvador Ventura'}, {'name': 'Sebastian Kmiecik'}]\", 'abstract': '<p>Motivation: Protein aggregation is associated with highly debilitating human disorders and constitutes a major bottleneck for producing therapeutic proteins. Our knowledge of the human protein structures repertoire has dramatically increased with the recent development of the AlphaFold (AF) deep-learning method. This structural information can be used to understand better protein aggregation properties and the rational design of protein solubility. This article uses the Aggrescan3D (A3D) tool to compute the structure-based aggregation predictions for the human proteome and make the predictions available in a database form. Results: Here, we present the A3D Database, in which we analyze the AF-predicted human protein structures (for over 17 thousand non-membrane proteins) in terms of their aggregation properties using the A3D tool. Each entry of the A3D Database provides a detailed analysis of the structure-based aggregation propensity computed with A3D. The A3D Database implements simple but useful graphical tools for visualizing and interpreting protein structure datasets. We discuss case studies illustrating how the database could be used to analyze physiologically relevant proteins. Furthermore, the database enables testing the influence of user-selected mutations on protein solubility and stability, all integrated into a user-friendly interface. Availability and implementation: A3D Database is freely available at: http://biocomp.chem.uw.edu.pl/A3D2/hproteome</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469114', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Double mutant of chymotrypsin inhibitor 2 stabilized through increased conformational entropy', 'authors': \"[{'name': 'Yulian Gavrilov'}, {'name': 'Felix Kümmerer'}, {'name': 'Simone Orioli'}, {'name': 'Andreas Prestel'}, {'name': 'Kresten Lindorff-Larsen'}, {'name': 'Kaare Teilum'}]\", 'abstract': '<p>The conformational heterogeneity of a folded protein can affect both its function but also stability and folding. We recently discovered and characterized a stabilized double mutant (L49I/I57V) of the protein CI2 and showed that state-of-the-art prediction methods could not predict the increased stability relative to the wild-type protein. Here we have examined whether changed native state dynamics, and resulting entropy changes, can explain the stability changes in the double mutant protein, as well as the two single mutant forms. We have combined NMR relaxation measurements of the ps-ns dynamics of amide groups in the backbone and the methyl groups in the side-chains with molecular dynamics simulations to quantify the native state dynamics. The NMR experiments reveal that the mutations have different effects on the conformational flexibility of CI2: A reduction in conformational dynamics (and entropy) of the native state of L49I variant correlates with its decreased stability, while increased dynamics of the I57V and L49I/I57V variants correlates with their increased stability. These findings suggest that explicitly accounting for changes in native state entropy might be needed to improve the predictions of the effect of mutations on protein stability.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469115', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Characterization of plasma circulating small extracellular vesicles in patients with metastatic solid tumors and newly diagnosed brain metastasis', 'authors': \"[{'name': 'Alberto Carretero-González'}, {'name': 'Sara Sánchez-Redondo'}, {'name': 'Luis Manso Sánchez'}, {'name': 'Eva Ciruelos Gil'}, {'name': 'Daniel Castellano'}, {'name': 'Marta Hergueta-Redondo'}, {'name': 'Guillermo de Velasco'}, {'name': 'Héctor Peinado'}]\", 'abstract': '<sec><title>Purpose</title><p>Nearly 40% of the advanced cancer patients will present brain metastases during the course of their disease, with a 2-year life expectancy of less than 10%. Immune system impairment, including the modulation of both STAT3 and PD-L1, is one of the hallmarks of brain metastases. Liquid biopsy could offer several advantages in brain metastases management, such as the possibility of non-invasive dynamic monitoring. Extracellular vesicles (EVs) have been recently proposed as novel biomarkers especially useful in liquid biopsy due to their secretion in biofluids and their role in cell communication during tumor progression.</p></sec><sec><title>Materials and Methods</title><p>The main aim of this work was to characterize the size and protein cargo of plasma circulating EVs in patients with solid tumors and their correlation with newly diagnosed brain metastases, in addition to their association with other relevant clinical variables.</p></sec><sec><title>Results</title><p>We analyzed circulating EVs in the plasma of 123 patients: 42 patients with brain metastases, 50 without brain metastases and 31 healthy controls. Patients with newly diagnosed brain metastases had a lower number of circulating EVs in the plasma and a higher protein concentration in small EVs (sEVs) compared to patients without brain metastases and healthy controls. Interestingly, melanoma patients with brain metastases presented decreased STAT3 activation and increased PD-L1 levels in circulating sEVs compared to patients without central nervous system metastases.</p></sec><sec><title>Conclusions</title><p>Decreased STAT3 activation and increased PD-L1 in plasma circulating sEVs identify melanoma patients with brain metastasis.</p></sec><sec><title>Statement of translational relevance</title><p>Brain metastases are critical for outcomes and quality of life in almost 50% of oncological patients, generally associated with a poor short-term prognosis. Early or preventive diagnosis of this complication represents an unmet need. There is a necessity of discovering new biomarkers that could aid to predict disease outcome.</p><p>In this study, we analyzed plasma circulating extracellular vesicles (EVs) from a cohort of 92 patients with different solid tumors (lung, breast, kidney cancer and melanoma) and found that newly diagnosed patients with brain metastases presented lower number of circulating particles and a higher protein concentration in small extracellular vesicles (sEVs) compared to patients without brain metastases and healthy controls. Out of all groups analyzed, melanoma patients with brain metastases presented decreased STAT3 activation and increased PD-L1 levels in circulating sEVs compared to patients without central nervous system metastases.</p><p>The data presented in this work suggest that circulating sEVs may represent the immunosuppressive status of newly diagnosed brain metastases characterized by the reduced phospho-STAT3 (pSTAT3) and increased PD-L1, although the origin of these molecules found in circulating sEVs remains to be uncovered.</p></sec>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468653', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Deep embedding and alignment of protein sequences', 'authors': \"[{'name': 'Felipe Llinares-López'}, {'name': 'Quentin Berthet'}, {'name': 'Mathieu Blondel'}, {'name': 'Olivier Teboul'}, {'name': 'Jean-Philippe Vert'}]\", 'abstract': '<p>Protein sequence alignment is a key component of most bioinformatics pipelines to study the structures and functions of proteins. Aligning highly divergent sequences remains, however, a difficult task that current algorithms often fail to perform accurately, leaving many proteins or open reading frames poorly annotated. Here, we leverage recent advances in deep learning for language modelling and differentiable programming to propose DEDAL, a flexible model to align protein sequences and detect homologs. DEDAL is a machine learning-based model that learns to align sequences by observing large datasets of raw protein sequences and of correct alignments. Once trained, we show that DEDAL improves by up to two- or three-fold the alignment correctness over existing methods on remote homologs, and better discriminates remote homologs from evolutionarily unrelated sequences, paving the way to improvements on many downstream tasks relying on sequence alignment in structural and functional genomics.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.17.469056', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A novel growth and isolation medium for exoelectrogenic bacteria', 'authors': \"[{'name': 'Zumaira Nazeer'}, {'name': 'Eustace Y. Fernando'}]\", 'abstract': '<p>A microbiological isolation and growth medium that can effectively discriminate electrochemically active exoelectrogenic bacteria from other non-exoelectrogenic bacteria, is currently unavailable. In this study, we developed a novel chromogenic growth and isolation solid medium based on MnO<sub>2</sub> that can selectively allow the growth of exoelectrogenic bacteria and change the medium colour in the process. Known exoelectrogenic bacteria such as <italic>Shewanella oneidensis</italic> MR1 and other such bacteria from functional microbial fuel cell (MFC) anodes were capable of growing and changing colour in the novel growth medium. On the contrary, non-exoelectrogenic bacteria such as <italic>Escherichia coli</italic> ATCC 25922 were incapable of growing and inducing a colour change in the novel medium. Further biochemical characterisation of these isolated exoelectrogenic bacteria by Raman micro-spectroscopy demonstrated that these bacteria over express cytochrome proteins that are vital in extracellular electron transfer events. This medium is a convenient method to isolate exoelectrogenic bacteria from complex environmental samples.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.14.468550', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Ca2+-inactivation of the mammalian ryanodine receptor type 1 in a lipidic environment revealed by cryo-EM', 'authors': \"[{'name': 'Ashok R. Nayak'}, {'name': 'Montserrat Samsó'}]\", 'abstract': '<p>Activation of the intracellular Ca<sup>2+</sup> channel ryanodine receptor (RyR) triggers a cytosolic Ca<sup>2+</sup> surge, while elevated cytosolic Ca<sup>2+</sup> inhibits the channel in a negative feedback mechanism. Cryo-EM carried out under partially inactivating Ca<sup>2+</sup> conditions revealed two conformations of RyR1, an open state and an inactivated state, resolved at 4.0 and 3.3 Å resolution, respectively. RyR1s were embedded in nanodiscs with two lipids resolved at each inter-subunit crevice. Ca<sup>2+</sup> binding to the high affinity site engages the central (CD) and C-terminal domains (CTD) into a quasi-rigid unit, which separates the S6 four-helix bundle and opens the channel. Further out-of-plane rotation of the quasi-rigid unit pushes S6 towards the central axis, closing (inactivating) the channel. The inactivated conformation is characterized by a downward conformation of the cytoplasmic assembly, a tightly-knit subunit interface contributed by a fully occupied and partially remodeled Ca<sup>2+</sup> activation site, and two salt bridges between the EF hand domain and the S2-S3 loop of the neighboring subunit validated by naturally-occurring diseasecausing mutations. Ca<sup>2+</sup> also bound to ATP, mediating a tighter interaction between S6 and CTD. Our study suggests that the <italic>closed-inactivated</italic> is a distinctive state of the RyR1 and its transition to the <italic>closed-activable</italic> state is not a simple reverse of the Ca<sup>2+</sup> mediated activation pathway.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468561', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'pGlycoQuant with a deep residual network for precise and minuscule-missing-value quantitative glycoproteomics enabling the functional exploration of site-specific glycosylation', 'authors': \"[{'name': 'Weiqian Cao'}, {'name': 'Siyuan Kong'}, {'name': 'Wenfeng Zeng'}, {'name': 'Pengyun Gong'}, {'name': 'Biyun Jiang'}, {'name': 'Xinhang Hou'}, {'name': 'Yang Zhang'}, {'name': 'Huanhuan Zhao'}, {'name': 'Mingqi Liu'}, {'name': 'Xihua Qiao'}, {'name': 'Mengxi Wu'}, {'name': 'Guoquan Yan'}, {'name': 'Chao Liu'}, {'name': 'Pengyuan Yang'}]\", 'abstract': '<p>Interpreting large-scale glycoproteomic data for intact glycopeptide identification has been tremendously advanced by software tools. However, software tools for quantitative analysis of intact glycopeptides remain lagging behind, which greatly hinders exploring the differential expression and functions of site-specific glycosylation in organisms. Here, we report pGlycoQuant, a generic software tool for accurate and convenient quantitative intact glycopeptide analysis, supporting both primary and tandem mass spectrometry quantitation for multiple quantitative strategies. pGlycoQuant enables intact glycopeptide quantitation with very low missing values via a deep residual network, thus greatly expanding the quantitative function of several powerful search engines, currently including pGlyco 2.0, pGlyco3, Byonic and MSFragger-Glyco. The pGlycoQuant-based site-specific N-glycoproteomic study conducted here quantifies 6435 intact N-glycopeptides in three hepatocellular carcinoma cell lines with different metastatic potentials and, together with in vitro molecular biology experiments, illustrates core fucosylation at site 979 of the L1 cell adhesion molecule (L1CAM) as a potential regulator of HCC metastasis. pGlycoQuant is freely available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/expellir-arma/pGlycoQuant/releases/\">https://github.com/expellir-arma/pGlycoQuant/releases/</ext-link>. We have demonstrated pGlycoQuant to be a powerful tool for the quantitative analysis of site-specific glycosylation and the exploration of potential glycosylation-related biomarker candidates, and we expect further applications in glycoproteomic studies.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468597', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Protein Engineering with A Glycosylation Circuit Enables Improved Enzyme Characteristics', 'authors': \"[{'name': 'Eray Ulaş Bozkurt'}, {'name': 'İrem Niran Çağıl'}, {'name': 'Ebru Şahin Kehribar'}, {'name': 'Musa Efe Işılak'}, {'name': 'Urartu Özgür Şafak Şeker'}]\", 'abstract': '<p>Protein glycosylation is one of the most crucial and common post-translational modifications. It plays a fate-determining role and can alter many properties of proteins, making it an interesting for many biotechnology applications. The discovery of bacterial glycosylation mechanisms, opened a new perspective and transfer of <italic>C</italic>.<italic>jejuni</italic> N-linked glycosylation into laboratory work-horse <italic>E. coli</italic> increased research pace in the field exponentially. It has been previously showed that utilizing <italic>N-Linked Glycosylation</italic>, certain recombinant proteins have been furnished with improved features, such as stability and solubility. In this study, we utilized <italic>N-linked Glycosylation</italic> to glycosylate alkaline phosphatase (ALP) enzyme in <italic>E. coli</italic> and investigate the effects of glycosylation on an enzyme. Considering the glycosylation mechanism is highly dependent on the acceptor protein, ALP constructs carrying glycosylation tag at different locations of the gene has been created and glycosylation rates have been calculated. The most glycosylated construct has been selected for comparison with the native enzyme. We investigated the performance of glycosylated ALP in terms of its thermostability, proteolytic stability, tolerance to suboptimal pH and under denaturing conditions. Studies showed that glycosylated ALP performed remarkably better at optimal and harsh conditions Therefore, <italic>N-linked Glycosylation</italic> mechanism can be employed for enzyme engineering purposes and is a useful tool for industrial applications that require enzymatic activity.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.14.21266322', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Should we Mitigate or Suppress the next Pandemic? Time-Horizons and Costs shape optimal Social Distancing Strategies', 'authors': \"[{'name': 'Sarah Nowak'}, {'name': 'Pedro Nascimento de Lima'}, {'name': 'Raffaele Vardavas'}]\", 'abstract': '<p>The COVID-19 pandemic has called for swift action from local governments, which have instated Nonpharmaceutical Interventions (NPIs) to curb the spread of SARS-Cov-2. The quick and decisive decision to save lives through blunt instruments has raised questions about the conditions under which decision-makers should employ mitigation or suppression strategies to tackle the COVID-19 pandemic. More broadly, there are still debates over which set of strategies should be adopted to control different pandemics, and the lessons learned for SARS-Cov-2 may not apply to a new pathogen. While curbing SARS-Cov-2 required blunt instruments, it is unclear whether a less-transmissible and less-deadly emerging pathogen would justify the same response. This paper illuminates this question using a parsimonious transmission model by formulating the social distancing lives vs. livelihoods dilemma as a boundary value problem. In this setup, society balances the costs and benefits of social distancing contingent on the costs of reducing transmission relative to the burden imposed by the disease. To the best of our knowledge, our approach is distinct in the sense that strategies emerge from the problem structure rather than being imposed <italic>a priori</italic>. We find that the relative time-horizon of the pandemic (i.e., the time it takes to develop effective vaccines and treatments) and the relative cost of social distancing influence the choice of the optimal policy. Unsurprisingly, we find that the appropriate policy response depends on these two factors. We discuss the conditions under which each policy archetype (suppression vs. mitigation) appears to be the most appropriate.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.16.468804', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Whole-brain fluorescence-MRI coregistration for precise anatomical mapping of virus infection', 'authors': \"[{'name': 'Nunya Chotiwan'}, {'name': 'Stefanie M. A. Willekens'}, {'name': 'Erin Schexnaydre'}, {'name': 'Max Hahn'}, {'name': 'Federico Morini'}, {'name': 'Jianguo Zhang'}, {'name': 'Emma Nilsson'}, {'name': 'Lars-Anders Carlson'}, {'name': 'Ulf Ahlgren'}, {'name': 'Daniel Marcellino'}, {'name': 'Anna K. Överby'}]\", 'abstract': '<p>Neurotropic virus infections cause tremendous disease burden. Methods visualizing infection in the whole brain remain unavailable which greatly impedes understanding of viral neurotropism and pathogenesis. We devised an approach to visualize the distribution of neurotropic virus infection in whole mouse brain <italic>ex vivo</italic>. Optical projection tomography (OPT) signal was coregistered with a unique magnetic resonance imaging (MRI) brain template, enabling precise anatomical mapping of viral distribution, and the effect of type I interferon on distribution of infection was analyzed. Guided by OPT-MR, we show that Langat virus specifically targets sensory brain systems and the lack of type I interferon response results in an anatomical shift in infection patterns in the brain. We confirm this regional tropism, observed with whole brain OPT-MRI, by confocal and electron microscopy to provide unprecedented insight into viral neurotropism. This approach can be applied to any fluorescently labeled target in the brain.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.12.21266178', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'The challenge of SARS-CoV-2 environmental monitoring in schools using floors and portable HEPA filtration units: Fresh or relic RNA?', 'authors': \"[{'name': 'Rogelio Zuniga-Montanez'}, {'name': 'David A. Coil'}, {'name': 'Jonathan A. Eisen'}, {'name': 'Randi Pechacek'}, {'name': 'Roque G. Guerrero'}, {'name': 'Minji Kim'}, {'name': 'Karen Shapiro'}, {'name': 'Heather N. Bischel'}]\", 'abstract': '<p>Testing surfaces in school classrooms for the presence of SARS-CoV-2, the virus that causes COVID-19, can provide public-health information that complements clinical testing. We monitored the presence of SARS-CoV-2 RNA in five schools (96 classrooms) in Davis, California (USA) by collecting weekly surface-swab samples from classroom floors and/or portable high-efficiency particulate air (HEPA) units. Twenty-two surfaces tested positive, with qPCR cycle threshold (Ct) values ranging from 36.07–38.01. Intermittent repeated positives in a single room were observed for both floor and HEPA filter samples for up to 52 days, even following regular cleaning and HEPA filter replacement after a positive result. We compared the two environmental sampling strategies by testing one floor and two HEPA filter samples in 57 classrooms at Schools D and E. HEPA filter sampling yielded 3.02% and 0.41% positivity rates per filter sample collected for Schools D and E, respectively, while floor sampling yielded 0.48% and 0% positivity rates. Our results indicate that HEPA filter swabs are more sensitive than floor swabs at detecting SARS-CoV-2 RNA in interior spaces. During the study, all schools were offered weekly free COVID-19 clinical testing. On-site clinical testing was offered in Schools D and E, and upticks in testing participation were observed following a confirmed positive environmental sample. However, no confirmed COVID-19 cases were identified among students associated with classrooms yielding positive environmental samples. The positive samples detected in this study appeared to reflect relic viral RNA from individuals infected before the monitoring program started and/or RNA transported into classrooms via fomites. The high-Ct positive results from environmental swabs further suggest the absence of active infections. Additional research is needed to differentiate between fresh and relic SARS-CoV-2 RNA in environmental samples and to determine what types of results should trigger interventions.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.19.469338', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'RIF1-ASF1-mediated high-order chromatin structure safeguards genome integrity', 'authors': \"[{'name': 'Dongyi Xu'}, {'name': 'Sumin Feng'}, {'name': 'Sai Ma'}, {'name': 'Kejiao Li'}, {'name': 'Shengxian Gao'}, {'name': 'Shaokai Ning'}, {'name': 'Jinfeng Shang'}, {'name': 'Ruiyuan Guo'}, {'name': 'Qing Li'}, {'name': 'Rong Guo'}, {'name': 'Britny Blumenfeld'}, {'name': 'Itamar Simon'}]\", 'abstract': '<p>The 53BP1-RIF1 pathway antagonizes resection of DNA broken ends and confers PARP inhibitor sensitivity on BRCA1-mutated tumors. However, it is unclear how this pathway suppresses initiation of resection. Here, we identify ASF1 as a partner of RIF1 via an interacting manner similar to its interactions with histone chaperones CAF-1 and HIRA. ASF1 is recruited to distal chromatin flanking DNA breaks by 53BP1-RIF1 and promotes non-homologous end joining (NHEJ) using its histone chaperone activity. Epistasis analysis shows that ASF1 acts in the same NHEJ pathway as RIF1, but via a parallel pathway with the shieldin complex, which suppresses resection after initiation. Moreover, defects in end resection and homologous recombination (HR) in BRCA1-deficient cells are largely suppressed by ASF1 deficiency. Mechanistically, ASF1 compacts adjacent chromatin by heterochromatinization to protect broken DNA ends from BRCA1-mediated resection. Taken together, our findings identified a RIF1-ASF1 histone chaperone complex that promotes changes in high-order chromatin structure to stimulate the NHEJ pathway for DSB repair.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469014', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Mitochondrial dysfunction promotes alternative gasdermin D-mediated inflammatory cell death and susceptibility to infection', 'authors': \"[{'name': 'Chi G Weindel'}, {'name': 'Xiao Zhao'}, {'name': 'Eduardo Martinez'}, {'name': 'Samantha L Bell'}, {'name': 'Krystal J Vail'}, {'name': 'Aja K Coleman'}, {'name': 'Jordan J VanPortfliet'}, {'name': 'Baoyu Zhao'}, {'name': 'Cory Mabry'}, {'name': 'Pingwei Li'}, {'name': 'Phillip West'}, {'name': 'Jason Karpac'}, {'name': 'Kristin L Patrick'}, {'name': 'Robert O Watson'}]\", 'abstract': '<p>Human mutations in mitochondrial-associated genes are associated with inflammatory diseases and susceptibility to infection. However, their mechanistic contributions to immune outcomes remain ill-defined.  We discovered that the disease-associated gain-of-function allele <italic>Lrrk2<sup>G2019S</sup></italic> (leucine-rich repeat kinase 2) promotes mitochondrial hyper-fission, depolarization, and oxidative stress in macrophages. In the presence of <italic>Lrrk2<sup>G2019S</sup></italic>-dependent mitochondrial perturbations, AIM2 inflammasome activation promotes more cell death but not more pyroptotic IL-1β release. Instead, inflammasome activation in <italic>Lrrk2<sup>G2019S</sup></italic> macrophages triggers gasdermin D (GSDMD)-mediated mitochondrial pores, driving up ROS-mediated RIPK1/RIPK3/MLKL dependent necroptosis. Consequently, infection of <italic>Lrrk2<sup>G2019S</sup></italic> mice with <italic>Mycobacterium tuberculosis</italic> elicits hyperinflammation and immunopathology via enhanced neutrophil infiltration. By uncovering that GSDMD promotes non-pyroptotic cell death in <italic>Lrrk2<sup>G2019S</sup></italic> macrophages, our findings demonstrate that altered mitochondrial function can reprogram cell death modalities to elicit distinct immune outcomes. This provides mechanistic insights into why mutations in LRRK2 are associated with susceptibility to chronic inflammatory and infectious diseases.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469000', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'The distinct effects of MEK and GSK3 inhibition upon the methylome and transcriptome of mouse embryonic stem cells', 'authors': '[{\\'name\\': \\'Julia Spindel\\'}, {\\'name\\': \\'Christel Krueger\\'}, {\\'name\\': \\'Felix Krueger\\'}, {\\'name\\': \\'Evangelia K Papachristou\\'}, {\\'name\\': \\'Kamal Kishore\\'}, {\\'name\\': \"Clive S D\\'Santos\"}, {\\'name\\': \\'Wolf Reik\\'}]', 'abstract': \"<p>Mouse embryonic stem cells (mESCs) were first cultured <italic>in vitro</italic> in serum-containing medium with leukaemia inhibitory factor, in which they exhibit heterogeneous expression of both pluripotency and some early differentiation markers. Over the last decade, however, it has become commonplace to grow mESCs with inhibitors of MEK and GSK3 signalling, which together elicit a more homogeneously 'naive' state of pluripotency. Whilst 2i/L-cultured mESCs have been shown to be globally hypomethylated, a comprehensive understanding of the distinct effects of these signalling inhibitors upon the DNA methylome is still lacking. Here we carried out whole genome bisulphite and RNA sequencing of mESCs grown with MEK or GSK3 inhibition alone, including different time points and concentrations of MEK inhibitor treatment. This demonstrated that MEK inhibition causes a dose-dependent impairment of maintenance methylation via loss of UHRF1 protein, as well as rapid impairment of <italic>de novo</italic> methylation. In contrast, GSK3 inhibition triggers impairment of <italic>de novo</italic> methylation alone, and consequent hypomethylation is enriched at enhancers with a 2i/L-specific chromatin signature and coincides with upregulation of nearby genes. Our study provides detailed insights into the epigenetic and transcriptional impacts of inhibiting MEK or GSK3 signalling in mouse pluripotent cells.</p>\", 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.17.468809', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'DeepProjection: Rapid and structure-specific projections of tissue sheets embedded in 3D microscopy stacks using deep learning', 'authors': \"[{'name': 'Daniel Haertter'}, {'name': 'Xiaolei Wang'}, {'name': 'Stephanie M Fogerson'}, {'name': 'Nitya Ramkumar'}, {'name': 'Janice M Crawford'}, {'name': 'Kenneth D Poss'}, {'name': 'Stefano Di Talia'}, {'name': 'Daniel P Kiehart'}, {'name': 'Christoph F Schmidt'}]\", 'abstract': '<p>The efficient extraction of local high-resolution content from massive amounts of imaging data remains a serious and unsolved problem in studies of complex biological tissues. Here we present DeepProjection, a trainable projection algorithm based on deep learning. This algorithm rapidly and robustly extracts image content contained in curved manifolds from time-lapse recorded 3D image stacks by binary masking of background content, stack by stack. The masks calculated for a given movie, when predicted, e.g., on fluorescent cell boundaries on one channel, can subsequently be applied to project other fluorescent channels from the same manifold. We apply DeepProjection to follow the dynamic movements of 2D-tissue sheets in embryonic development. We show that we can selectively project the amnioserosa cell sheet during dorsal closure in Drosophila melanogaster embryos and the periderm layer in the elongating zebrafish embryo while masking highly fluorescent out-of-plane artifacts.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468867', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Evaluating the contribution of aestivation to the persistence of malaria mosquitoes through the Sahelian dry season using stable isotopes', 'authors': \"[{'name': 'Roy Faiman'}, {'name': 'Alpha SEYDOU YARO'}, {'name': 'Adama Dao'}, {'name': 'Zana L Sanogo'}, {'name': 'Moussa Diallo'}, {'name': 'Djibril Samake'}, {'name': 'Ousmane Yossi'}, {'name': 'Laura R Veru'}, {'name': 'Leland C Graber'}, {'name': 'Abigail R Conte'}, {'name': 'Cedric Kouam'}, {'name': 'Benjamin J Krajacich'}, {'name': 'Tovi Lehmann'}]\", 'abstract': '<p>Data suggests the malaria vector Anopheles coluzzii persists in the Sahel by dry-season aestivation though evidence is scant. We have marked Anopheles mosquitoes using deuterium (2H) to assess the contribution of aestivation to persistence of mosquitoes through the seven-month dry season.  If local aestivation is the only way A. coluzzii persists, the frequency of marked mosquitoes should remain stable throughout, whereas finding no marked mosquitoes would be evidence against aestivation. Larval sites were spiked with 2H at the end of the 2017 wet season in two Sahelian villages in Mali. We monitored 2H-enriched populations until the onset of rains.  By the end of the enrichment period, 33% of A. coluzzii mosquitoes were clearly marked. Expectedly, 2H levels in marked mosquitoes degraded over time, resulting in a partial overlap of the marked and non-marked 2H distributions. We utilized three methods to estimate the fraction of marked mosquitoes in the population. Seven months after enrichment, 7% of the population had 2H values above the highest pre-enrichment value. An excess of 21% exceeded the 3rd quartile of the pre-enrichment population. A finite mixed population model showed 2.5% represented a subpopulation of marked mosquitoes with elevated 2H, compatible with our predictions. We provide evidence that aestivation is a major persistence mechanism of A. coluzzii in the Sahel, contributing at least 20% of the adults at the onset of rains, suggesting A. coluzzii utilizes multiple persistence strategies enabling its populations rapid buildup, facilitating subsequent malaria resurgence. These may complicate vector control and malaria elimination campaigns.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468727', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'A Cryptic K48 Ubiquitin Chain Binding Site on UCH37 is Required for its Role in Proteasomal Degradation', 'authors': \"[{'name': 'Jiale Du'}, {'name': 'Sandor Babik'}, {'name': 'Yanfeng Li'}, {'name': 'Kirandeep K. Deol'}, {'name': 'Stephen J. Eyles'}, {'name': 'Jasna Fejzo'}, {'name': 'Marco Tonelli'}, {'name': 'Eric R. Strieter'}]\", 'abstract': '<p>Degradation by the 26S proteasome is an intricately regulated process fine-tuned by the precise nature of ubiquitin modifications attached to a protein substrate. By debranching ubiquitin chains composed of K48 linkages, the proteasome-associated ubiquitin C-terminal hydrolase UCHL5/UCH37 serves as a positive regulator of protein degradation. How UCH37 achieves specificity for K48 chains is unclear. Here, we use a combination of hydrogen-deuterium mass spectrometry, chemical crosslinking, small-angle X-ray scattering, NMR, molecular docking, and targeted mutagenesis to uncover a cryptic K48 ubiquitin chain specific binding site on the opposite face of UCH37 relative to the canonical S1 ubiquitin-binding site. Biochemical assays demonstrate the K48 chain-specific binding site is required for chain debranching and proteasome-mediated degradation of proteins modified with branched chains. Using quantitative proteomics, translation shutoff experiments, and linkage-specific affinity tools, we then identify specific proteins whose degradation depends on the debranching activity of UCH37. Our findings suggest that UCH37 and potentially other DUBs could use more than one S1 site to perform different biochemical functions.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.10.21266196', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Oncology', 'covid': False, 'title': 'Development of an accessible gene expression bioinformatics pipeline to study driver mutations of colorectal cancer', 'authors': \"[{'name': 'Lisa van den Driest'}, {'name': 'Caroline H Johnson'}, {'name': 'Nicholas JW Rattray'}, {'name': 'Zahra Rattray'}]\", 'abstract': '<p>Colorectal cancer (CRC) is a global cause of cancer-related mortality driven by genetic and environmental factors which influence therapeutic outcomes. The emergence of next-generation sequencing technologies enables the rapid and extensive collection and curation of genetic data for each cancer type into clinical gene expression biobanks.</p><p>In this study we used a combination of bioinformatics tools to investigate the expression patterns and prognostic significance of two genes, adenomatous polyposis coli (<italic>APC</italic>) and B-Raf proto-oncogene (<italic>BRAF</italic>), that are commonly dysregulated in colon cancer. Subsequently, we investigated the pathways and biomolecular effectors implicated in <italic>APC</italic> and <italic>BRAF</italic> function.</p><p>Our results show mutation types, frequency, anatomical location and differential expression patterns for <italic>APC</italic> and <italic>BRAF</italic> between colorectal tumour and matched healthy tissue. The prognostic values of <italic>APC</italic> and <italic>BRAF</italic> was investigated as a function of expression level in CRC and other cancer types.</p><p>In the era of precision medicine and with significant advancements in biobanking and data curation, there is significant scope to use existing clinical datasets for evaluating the role of mutational drivers in carcinogenesis. This offers the potential for studying combinations of less well-known genes and the discovery of novel biomarkers or studying the association between various effector proteins and pathways.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468632', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Nieman-Pick Type C2 proteins in Aedes aegypti: Their Structure-Function relationships and Expression in Uninfected versus Virus-infected Mosquitos', 'authors': \"[{'name': 'Prathigna Jaishankar Thambi'}, {'name': 'Cassandra M. Modahl'}, {'name': 'R. Manjunatha Kini'}]\", 'abstract': '<p><italic>Aedes aegypti</italic> is a major vector that transmits arboviruses through the saliva injected into the host. Salivary proteins help in uninterrupted blood intake and enhance the transmission of pathogens. We studied Nieman-Pick Type C2 (NPC2) proteins, a superfamily of saliva proteins that play important role in arbovirus infections. In vertebrates, a single conserved gene encodes for the NPC2 protein that functions in cholesterol trafficking. Arthropods, in contrast, have several genes that encode for divergent NPC2 proteins. We compared the sequences of 20 <italic>A. aegypti</italic> NPC2 proteins to the cholesterol-binding residues of human and bovine, and fatty acid-binding residues of ant NPC2 proteins. We identified four and one mosquito NPC2 proteins as potential sterol- and fatty acid-binding proteins, respectively. From the published data, we analysed the expression of NPC2 genes in various tissues and their differential expression in midgut and salivary gland post-arbovirus infections. NPC2 genes are downregulated rather than upregulated in virus-infected tissues. Interestingly, AAEL012064 is the only gene that is downregulated in both the midgut and salivary gland in all virus infections. In addition, AAEL001650 is downregulated in the salivary gland infected with CHIKV, DENV2 or ZIKV. This gene in the midgut is downregulated infected with DENV1 but upregulated with DENV2. We studied the variation in <italic>cis</italic> elements in the promoter regions of two groups of closely related NPC2 genes and the expression of relevant transcription factors (TFs). In the midgut infected with DENV1 or DENV2, six TFs (CRE-BP1, AP1, c-Jun, c-Fos, Odd, and NF-kB) appear to play an opposing role in the expression of AAEL006854. Two TFs (RXR-beta/alpha and USF) have a potential role in the downregulation of AAEL009556 in CHIKV-infected midgut and salivary gland. Similarly, two TFs (COUP and Ftz) may have a key role in the downregulation of AAEL009555 and AAEL009556 in DENV2-infected salivary gland.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468572', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Hypoxia classifier for transcriptome datasets', 'authors': \"[{'name': 'Laura Puente-Santamaria'}, {'name': 'Lucia Sanchez-Gonzalez'}, {'name': 'Ricardo Ramos-Ruiz'}, {'name': 'Luis del Peso'}]\", 'abstract': '<p>Molecular gene signatures are useful tools to characterize the physiological state of cell populations according to their gene expression profiles. However, most molecular gene signatures have been developed under a very limited set of conditions and cell types, and are often restricted to a set of gene identities linked to an event or biological process, therefore making necessary to develop and test additional procedures for its application to new data.</p><p>Focusing on the transcriptional response to hypoxia, we aimed to generate widely applicable classifiers capable of detecting hypoxic samples while maintaining transparency and ease of use and interpretation. Here we describe several tree-based classifiers sourced from the results of a meta-analysis of 69 differential expression datasets which included 425 individual RNA-seq experiments from 33 different human cell types exposed to different degrees of hypoxia (0.1-5%O<sub>2</sub>) for a time spanning between 2 and 48h.</p><p>These decision trees include both the identities of genes key in the response to hypoxia and defined quantitative boundaries, allowing for the classification of individual samples without needing a control or normoxic reference. Despite their simplicity and ease of use, these classifiers achieve over 95% accuracy in cross validation and over 80% accuracy when applied to additional challenging datasets. Moreover, the explicit structure of the trees allowed for the identification of relevant biological features in cases where prediction was not accurate. Finally, we demonstrate that the classifiers can be applied to spatial gene expression data to identify hypoxic regions within histological sections. Although we have focused on the identification of hypoxia, this method can be applied to detect activation of other processes or cellular states.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>Description of a method to generate simple tree-based gene signatures that combine gene identity and expression.</p></list-item><list-item><p>Tree-based gene signatures can be easily applied for accurate absolute classification of samples and regions in spatial gene expression data.</p></list-item><list-item><p>The explicit structure of the classifiers permits the intuitive interpretation of classification allowing for novel biological insights.</p></list-item></list></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.12.21266249', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Early and rapid identification of COVID-19 patients with neutralizing type I-interferon auto-antibodies by an easily implementable algorithm', 'authors': \"[{'name': 'Bengisu Akbil'}, {'name': 'Tim Meyer'}, {'name': 'Paula Stubbemann'}, {'name': 'Charlotte Thibeault'}, {'name': 'Olga Staudacher'}, {'name': 'Daniela Niemeyer'}, {'name': 'Jenny Jansen'}, {'name': 'Barbara Mühlemann'}, {'name': 'Jan Doehn'}, {'name': 'Christoph Tabeling'}, {'name': 'Christian Nusshag'}, {'name': 'Cédric Hirzel'}, {'name': 'David Sökler Sanchez'}, {'name': 'Alexandra Nieters'}, {'name': 'Achim Lother'}, {'name': 'Daniel Duerschmied'}, {'name': 'Nils Schallner'}, {'name': 'Jan Nikolaus Lieberum'}, {'name': 'Dietrich August'}, {'name': 'Siegbert Rieg'}, {'name': 'Valeria Falcone'}, {'name': 'Hartmut Hengel'}, {'name': 'Uwe Kölsch'}, {'name': 'Nadine Unterwalder'}, {'name': 'Ralf-Harto Hübner'}, {'name': 'Terry C. Jones'}, {'name': 'Norbert Suttorp'}, {'name': 'Christian Drosten'}, {'name': 'Klaus Warnatz'}, {'name': 'Thibaud Spinetti'}, {'name': 'Joerg C. Schefold'}, {'name': 'Thomas Dörner'}, {'name': 'Leif Sander'}, {'name': 'Victor M. Corman'}, {'name': 'Uta Merle'}, {'name': 'Florian Kurth'}, {'name': 'Horst von Bernuth'}, {'name': 'Christian Meisel'}, {'name': 'Christine Goffinet'}, {'name': ' '}]\", 'abstract': '<sec><title>Purpose</title><p>Six-19% of critically ill COVID-19 patients display circulating auto-antibodies against type I interferons (IFN-AABs). Here, we establish a clinically applicable strategy for early identification of IFN-AAB-positive patients for potential subsequent clinical interventions.</p></sec><sec><title>Methods</title><p>We analysed sera of 430 COVID-19 patients with severe and critical disease from four hospitals for presence of IFN-AABs by ELISA. Binding specificity and neutralizing activity were evaluated via competition assay and virus-infection-based neutralization assay. We defined clinical parameters associated with IFN-AAB positivity. In a subgroup of critically ill patients, we analyzed effects of therapeutic plasma exchange (TPE) on the levels of IFN-AABs, SARS-CoV-2 antibodies and clinical outcome.</p></sec><sec><title>Results</title><p>The prevalence of neutralizing AABs to IFN-α and IFN-ω in COVID-19 patients was 4.2% (18/430), while being undetectable in an uninfected control cohort. Neutralizing IFN-AABs were detectable exclusively in critically affected, predominantly male (83%) patients (7.6% IFN-α and 4.6% IFN-ω in 207 patients with critical COVID-19). IFN-AABs were present early post-symptom onset and at the peak of disease. Fever and oxygen requirement at hospital admission co-presented with neutralizing IFN-AAB positivity. IFN-AABs were associated with higher mortality (92.3% versus 19.1 % in patients without IFN-AABs). TPE reduced levels of IFN-AABs in three of five patients and may increase survival of IFN-AAB-positive patients compared to those not undergoing TPE.</p></sec><sec><title>Conclusion</title><p>IFN-AABs may serve as early biomarker for development of severe COVID-19. We propose to implement routine screening of hospitalized COVID-19 patients according to our algorithm for rapid identification of patients with IFN-AABs who most likely benefit from specific therapies.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468433', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Cryptococcus neoformans capsule regrowth experiments reveal dynamics of enlargement and architecture', 'authors': \"[{'name': 'Maggie P. Wear'}, {'name': 'Ella Jacobs'}, {'name': 'Siqing Wang'}, {'name': 'Scott McConnell'}, {'name': 'Anthony Bowen'}, {'name': 'Camilla Strother'}, {'name': 'Radames J.B. Cordero'}, {'name': 'Arturo Casadevall'}]\", 'abstract': '<p>The polysaccharide capsule of fungal pathogen <italic>Cryptococcus neoformans</italic> is a critical virulence factor that has historically evaded characterization. Polysaccharides remain attached to the cell as capsular polysaccharide (CPS) or are shed into the surroundings in the form of exopolysaccharide (EPS). While a great deal of study has been done examining the properties of EPS, far less is known about CPS. In this work, we detail the development of new physical and enzymatic methods for the isolation of CPS which can be used to explore the architecture of the capsule and removed capsular material. Sonication and glucanex digestion yield soluble CPS preparations, while French Press and modified glucanex digestion plus vortexing remove the capsule and cell wall producing polysaccharide aggregates that we call ‘capsule ghosts.’ The existence of capsule ghosts implies an inherent organization that allows it to exist independent of the cell wall surface. As sonication and glucanex digestion were noncytotoxic, it was possible to observe the cryptococcal cells rebuilding their capsule, revealing new insights into capsule architecture and synthesis consistent with a model in which the capsule is assembled from smaller polymers, which are then assemble into larger ones.</p><sec><title>Importance</title><p>Characterization of the cryptococcal polysaccharide capsule relies on methods of isolation for its <italic>in vitro</italic> study. This study demonstrates that the capsule is susceptible to physical and enzymatic removal. The application of new methods yields insights into the anatomy, modular nature, and architecture of the capsule with both soluble CPS preparations and ‘capsule ghosts.’ Together these insights inform on a long-standing debate modeling capsular assembly wherein our data shows that the capsule is assembled by smaller polymers added distally rather than by proximal addition or by polymers spanning the entire capsule radius.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469119', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'An insight into new glycotherapeutic in glial inflammation: Understanding the role of glycosylation from acute to chronic phase of inflammation', 'authors': \"[{'name': 'Vaibhav Patil'}, {'name': 'Raghvendra Bohara'}, {'name': 'Carla Winter'}, {'name': 'Michelle Kilcoyne'}, {'name': 'Siobhan McMahon'}, {'name': 'Abhay Pandit'}]\", 'abstract': \"<p>Glycosylation plays a critical role during inflammation and glial scar formation upon spinal cord injury (SCI) disease progression. Astrocytes and microglia are involved in this cascade to modulate the inflammation and tissue remodelling from acute to chronic phases. Therefore, understating the glycan changes in these glial cells is paramount. Herein a lectin microarray was undertaken using a cytokine-driven inflammatory MGC model, revealing considerable differential glycosylation from the acute to the chronic phase in a cytokine-combination generated inflamed MGC model. It was found that several N- and O-linked glycans associated with glia during SCI were differentially regulated. Pearson's correlation hierarchical clustering showed that groups were separated into several clusters, illustrating the heterogenicity among the control, cytokine combination, and LPS treated groups and the day on which treatment was given. Control and LPS treatments were observed to be in dense clusters. This was further confirmed with lectin immunostaining in which GalNAc, GlcNAc, mannose, fucose and sialic acid-binding residues were detected in astrocytes and microglia. However, this modification (upregulation of sialic acid expression) was inhibited by the sialyltransferase inhibitor which indeed modulates the mitochondrial functions. The present study is the first functional investigation of glycosylation modulation in a MGC (MGC) model which elucidates the role of the glycome in neuroinflammation and identified potential therapeutic targets for future glycol- therapeutics in neuroinflammation</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.14.21266309', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Safety and immunogenicity of anti-SARS CoV-2 vaccine SOBERANA 02 in homologous or heterologous scheme', 'authors': \"[{'name': 'Maria Eugenia-Toledo-Romani'}, {'name': 'Leslihana Verdecia-Sánchez'}, {'name': 'Meybis Rodriguez-González'}, {'name': 'Laura Rodríguez-Noda'}, {'name': 'Carmen Valenzuela-Silva'}, {'name': 'Beatriz Paredes-Moreno'}, {'name': 'Belinda Sanchez-Ramirez'}, {'name': 'Rocmira Perez-Nicado'}, {'name': 'Raul González-Mugica'}, {'name': 'Tays Hernández-Garcia'}, {'name': 'Gretchen Bergado-Baez'}, {'name': 'Franciscary Pi-Estopiñán'}, {'name': 'Otto Cruz-Sui'}, {'name': 'Anitza Fraga-Quintero'}, {'name': 'Majela García-Montero'}, {'name': 'Ariel Palenzuela-Diaz'}, {'name': 'Gerardo Baro-Roman'}, {'name': 'Ivis Mendosa-Hernández'}, {'name': 'Sonsire Fernandez-Castillo'}, {'name': 'Yanet Climent-Ruiz'}, {'name': 'Darielys Santana-Mederos'}, {'name': 'Ubel Ramírez Gonzalez'}, {'name': 'Yanelda García-Vega'}, {'name': 'Beatriz Perez-Massón'}, {'name': ' Guang-Wu-Chen'}, {'name': 'Tammy Boggiano-Ayo'}, {'name': 'Eduardo Ojito-Magaz'}, {'name': 'Daniel G. Rivera'}, {'name': 'Yury Valdés-Balbín'}, {'name': 'Dagmar García-Rivera'}, {'name': 'Vicente Vérez-Bencomo'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>SOBERANA 02 is a COVID-19 conjugate vaccine candidate based on SARS-CoV-2 recombinant RBD conjugated to tetanus toxoid. SOBERANA Plus antigen is dimeric-RBD. Here we report safety, reactogenicity and immunogenicity from phase I and IIa clinical trials using two-doses SOBERANA 02 (homologous protocol) and three-doses (homologous) or heterologous (with SOBERANA Plus) protocols.</p></sec><sec><title>Method</title><p>We performed an open-label, monocentric, sequential and adaptive phase I for evaluating safety, reactogenicity and exploring immunogenicity of SOBERANA 02 in two formulations (15 and 25 μg) in 40 subjects, 19–59 years old. Phase IIa was open-label including 100 volunteers 19–80 years, receiving two doses of SOBERANA 02-25 μg. In both trials, half of volunteers received a third dose of SOBERANA 02, half received a heterologous dose of SOBERANA Plus-50 μg. Primary outcomes were safety and reactogenicity. The secondary outcome was vaccine immunogenicity evaluated by anti-RBD IgG ELISA, molecular neutralization test of RBD:hACE2 interaction, live-virus neutralization test and specific T-cells response.</p></sec><sec><title>Results</title><p>The most frequent AE was local pain, other AEs had frequencies ≤ 5%. No serious related AEs were reported. Phase IIa confirmed the safety results in 60–80 years subjects. In phase-I SOBERANA 02-25µg elicited higher immune response than SOBERANA 02-15 µg; in consequence, the higher dose progressed to phase IIa. Phase IIa results confirmed the immunogenicity of SOBERANA 02-25 μg even in 60–80 age range. Two doses of SOBERANA02-25 μg elicited an immune response similar to that of the Cuban Convalescent Serum Panel; it was higher after both the homologous and heterologous third doses; the heterologous scheme showing a higher immunological response.</p></sec><sec><title>Conclusions</title><p>SOBERANA 02 was safe and immunogenic in persons aged 19–80 years, eliciting neutralizing antibodies and specific T cell response. Highest immune responses were obtained in the heterologous three doses protocol. Trial registry: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://rpcec.sld.cu/trials/RPCEC00000340\">https://rpcec.sld.cu/trials/RPCEC00000340</ext-link> and <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://rpcec.sld.cu/trials/RPCEC00000347\">https://rpcec.sld.cu/trials/RPCEC00000347</ext-link></p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469101', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Essential role for an isoform of Escherichia coli translation initiation factor IF2 in repair of two-ended DNA double-strand breaks', 'authors': \"[{'name': 'Jillella Mallikarjun'}, {'name': 'J Gowrishankar'}]\", 'abstract': '<p>In <italic>Escherichia coli</italic>, three isoforms of the essential translation initiation factor IF2 (IF2-1, IF2-2, and IF2-3) are generated from separate in-frame initiation codons in <italic>infB</italic>. The isoforms have earlier been suggested to additionally participate in DNA damage repair and replication restart. It is also known that the proteins RecA and RecBCD are needed for repair of DNA double-strand breaks (DSBs) in E. <italic>coli</italic>. Here we show that strains lacking IF2-1 are profoundly sensitive to two-ended DSBs in DNA generated by radiomimetic agents phleomycin or bleomycin, or by endonuclease I-SceI. However, these strains remained tolerant to other DSB-generating genotoxic agents or perturbations to which <italic>recA</italic> and <italic>recBC</italic> mutants remained sensitive, such as to mitomycin C, type-2 DNA topoisomerase inhibitors, or DSB caused by palindrome cleavage behind a replication fork. Data from genome-wide copy number analyses following I-SceI cleavage at a single chromosomal locus suggested that, in a strain lacking IF2-1, the magnitude of break induced replication through replication restart mechanisms is largely preserved but the extent of DNA resection around the DSB site is reduced. We propose that in absence of IF2-1 it is the annealing of a RecA nucleoprotein filament to its homologous target that is weakened, which in turn leads to a specific failure in assembly of <italic>Ter-to-oriC</italic> directed replisomes needed for consummation of two-ended DSB repair.</p><sec><title>Importance</title><p>Double-strand breaks (DSBs) in DNA are major threats to genome integrity. In <italic>Escherichia coli</italic>, DSBs are repaired by Rec- and RecBCD-mediated homologous recombination (HR). This study demonstrates a critical role for an isoform (IF2-1) of the translation initiation factor IF2 in the repair of two-ended DSBs in <italic>E. coli</italic> (that can be generated by ionizing radiation, certain DNA-damaging chemicals, or endonuclease action). It is proposed that IF2-1 acts to facilitate the function of RecA in the annealing between a pair of DNA molecules during HR.</p></sec>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.17.21266262', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Endocrinology (including Diabetes Mellitus and Metabolic Disease)', 'covid': False, 'title': 'Understanding the interrelationships of type-2 diabetes and hypertension with brain and cognitive health: a UK Biobank study', 'authors': \"[{'name': 'Danielle Newby'}, {'name': 'Victoria Garfield'}]\", 'abstract': '<sec><title>Aims/hypothesis</title><p>Diabetes and hypertension are associated with poorer cognitive and brain health. Less is known about comorbid diabetes and hypertension and: cognitive and brain health in mid-life. We hypothesised that individuals with both diabetes and hypertension have worse cognitive and brain health.</p></sec><sec><title>Methods</title><p>We used the UK Biobank cohort, a population-based study which recruited 500,000 individuals, aged 40-69 years. Type-2 diabetes was assigned using self-report, HbA1c and clinical data, while hypertension was defined based on self-report. Our outcomes included a breadth of brain structural magnetic resonance imaging (MRI) parameters and cognitive function tests in a maximum of 38918 individuals. We tested associations in a cross sectional design firstly between those with diabetes (n = 2043) versus those without (reference category, n = 36875) then we tested associations between comorbid diabetes/hypertension (reference category, n=1283) and our outcomes by comparing this group with those with only diabetes (n=760), hypertension (n=9649) and neither disease (n=27226). Our analytical approach comprised linear regression models, with adjustment for a range of demographic and health factors.</p></sec><sec><title>Results</title><p>Those with diabetes had worse overall brain health, as indexed by multiple neuroimaging parameters, with the exception of gFA (white matter integrity) and the amygdala. The largest difference was observed in the pallidum (β=0.179, 95%CI=0.137;0.220). Individuals with diabetes had poorer performance on certain cognitive tests, with the largest difference observed in the symbol digit substitution test (β=0.132, 95%CI=0.079;0.187). Compared to individuals with comorbid diabetes and hypertension, those with only hypertension had better brain health overall, with the largest difference observed in the pallidum (β=0.189, 95%CI=0.241;0.137), while those with only diabetes differed in total grey volume (β=0.150, 95%CI=0.122;0.179). Compared with individuals who had comorbid diabetes and hypertension, those with only diabetes performed distinctly better on the verbal and numeric reasoning task (β=0.129, 95%CI=0.077;0.261), whereas those with only hypertension performed better on the symbol digit substitution task (β=0.117, 95%CI=0.048;0.186)</p></sec><sec><title>Conclusions/interpretation</title><p>Individuals with comorbid diabetes and hypertension have worse brain and cognitive health compared to those with only one of these diseases. These findings potentially suggest that prevention of both diabetes and hypertension may delay changes in brain structure, as well as cognitive decline and dementia diagnosis.</p></sec><sec><title>Research in context</title><sec><title>What is already known about this subject?</title><list list-type=\"bullet\"><list-item><p>Diabetes and hypertension are both independently associated with poorer brain health, cognition and increase in dementia risk</p></list-item><list-item><p>50% of adults with diabetes also have hypertension</p></list-item><list-item><p>Comorbid diabetes and hypertension are associated with excess dementia risk</p></list-item></list></sec><sec><title>What is the key question?</title><list list-type=\"bullet\"><list-item><p>Do individuals with both diabetes and hypertension have worse cognitive and brain health?</p></list-item></list></sec><sec><title>What are the new findings?</title><list list-type=\"bullet\"><list-item><p>Individuals with diabetes have poorer brain health and cognitive performance using a breadth of neuroimaging measures and cognitive tests</p></list-item><list-item><p>Those with both diabetes and hypertension have worse brain health and cognitive performance (particularly processing speed) compared to those with only hypertension</p></list-item></list></sec><sec><title>How might this impact on clinical practice in the foreseeable future?</title><list list-type=\"bullet\"><list-item><p>Prevention of both diabetes and hypertension may help delay changes in brain structure and cognitive decline as well as reduce future dementia risk.</p></list-item></list></sec></sec>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468920', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'A general model of the dynamics of genes with dual σ factor preference', 'authors': \"[{'name': 'Ines S. C. Baptista'}, {'name': 'Vinodh Kandavalli'}, {'name': 'Vatsala Chauhan'}, {'name': 'Mohammed Bahurudeen'}, {'name': 'Bilena Almeida'}, {'name': 'Cristina Palma'}, {'name': 'Suchintak Dash'}, {'name': 'Andre S. Ribeiro'}]\", 'abstract': '<p><italic>Escherichia coli</italic> uses the ability of σ factors to recognize specific DNA sequences in order to quickly control large gene cohorts. While most genes respond to only one σ factor, approximately 5% have dual σ factor preference. The ones in significant numbers are ‘σ<sup>70+38</sup> genes’, responsive to σ<sup>70</sup>, which controls housekeeping genes, as well as to σ<sup>38</sup>, which activates genes during stationary growth and stresses. We show that σ<sup>70+38</sup> genes are almost as upregulated in stationary growth as genes responsive to σ<sup>38</sup> alone. Also, their response strengths to σ<sup>38</sup> are predictable from their promoter sequences. Next, we propose a sequence- and σ<sup>38</sup> level-dependent, analytical model of σ<sup>70+38</sup> genes applicable in the exponential, stationary, and in the transition period between the two growth phases. Finally, we propose a general model, applicable to other σ factors as well. This model can guide the design of synthetic circuits with sequence-dependent sensitivity and plasticity to transitions between the exponential and stationary growth phases.</p><sec><title>Author Summary</title><p>Present challenges in Synthetic Biology include the design of genetic circuits that are robust to growth phase transitions and whose responsiveness is sequence-dependent, and, thus predictable prior to design. We present and validate an empirical-based, sequence-dependent analytical model of <italic>E. coli</italic> genes with dual responsiveness to the regulators σ<sup>70</sup> and σ<sup>38</sup>. These genes, supported by our sequence-dependent model, could become building blocks for synthetic genetic circuits functional in both the exponential and the stationary growth phases.</p></sec>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468780', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Getting close to nature – Plasmodium knowlesi reference genome sequences from contemporary clinical isolates', 'authors': \"[{'name': 'Damilola R. Oresegun'}, {'name': 'Peter Thorpe'}, {'name': 'Ernest Diez Benavente'}, {'name': 'Susana Campino'}, {'name': 'Fauzi Muh'}, {'name': 'Robert Moon'}, {'name': 'Taane G. Clark'}, {'name': 'Janet Cox-Singh'}]\", 'abstract': '<p><italic>Plasmodium knowlesi,</italic> a malaria parasite of old-world macaque monkeys, is used extensively to model <italic>Plasmodium</italic> biology. Recently <italic>P. knowlesi</italic> was found in the human population of Southeast Asia, particularly Malaysia. <italic>P. knowlesi</italic> causes un-complicated to severe and fatal malaria in the human host with features in common with the more prevalent and virulent malaria caused by <italic>Plasmodium falciparum</italic>.</p><p>As such <italic>P. knowlesi</italic> presents a unique opportunity to inform an experimental model for malaria with clinical data from same-species human infections.</p><p>Experimental lines of <italic>P. knowlesi</italic> represent well characterised genetically static parasites and to maximise their utility as a backdrop for understanding malaria pathophysiology, genetically diverse contemporary clinical isolates, essentially wild-type, require comparable characterization.</p><p>The Oxford Nanopore PCR-free long-read sequencing platform was used to sequence <italic>P. knowlesi</italic> parasites from archived clinical samples. The sequencing platform and assembly pipeline was designed to facilitate capturing data on important multiple gene families, including the <italic>P. knowlesi schizont-infected cell agglutination</italic> (<italic>SICA</italic>) <italic>var</italic> genes and the <italic>Knowlesi-Interspersed Repeats</italic> (<italic>KIR</italic>) genes.</p><p>The <italic>SICAvar</italic> and <italic>KIR</italic> gene families code for antigenically variant proteins that have been difficult to resolve and characterise. Analyses presented here suggest that the family members have arisen through a process of gene duplication, selection pressure and variation. Highly evolving genes tend to be located proximal to genetic elements that drive change rather than regions that support core gene conservation. For example, the virulence-associated <italic>P. falciparum</italic> erythrocyte membrane protein (<italic>PfEMP1</italic>) gene family members are restricted to relatively unstable sub-telomeric regions. In contrast the <italic>SICAvar</italic> and <italic>KIR</italic> genes are located throughout the genome but as the study presented here shows, they occupy otherwise gene-sparse chromosomal locations.</p><p>The novel methods presented here offer the malaria research community new tools to generate comprehensive genome sequence data from small clinical samples and renewed insight into these complex real-world parasites.</p><sec><title>Author summary</title><p>Malaria is a potentially severe disease caused by parasite species within genus Plasmodium. Even though the number of cases is in decline there were over 200 million reported cases of malaria in 2019 that resulted in &gt;400,000 deaths. Despite huge research efforts we still do not understand precisely how malaria makes some individuals very ill and by extension how to successfully augment and manage severe disease.</p><p>Here we developed a novel method to generate comprehensive robust genome sequences from the malaria parasite <italic>Plasmodium knowlesi</italic> collected from clinical samples.</p><p>We propose to use the method and initial data generated here to begin to build a resource to identify disease associated genetic traits of <italic>P. knowlesi</italic> taken from patient’s samples. In addition to the methodology, what further sets this work apart is the unique opportunity to utilize same-species experimental <italic>P. knowlesi</italic> parasites to discover a potential role for particular parasite traits in the differential disease progression we observe in patients with <italic>P. knowlesi</italic> malaria.</p><p>While we developed the methods to study severe malaria, they are affordable and accessible, and offer the wider malaria research community the means to add context and insight into real-world malaria parasites.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.17.468921', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Structural comparative modeling of multi-domain ΔF508 CFTR', 'authors': \"[{'name': 'Eli Fritz McDonald'}, {'name': 'Hope Woods'}, {'name': 'Shannon Smith'}, {'name': 'Minsoo Kim'}, {'name': 'Clara T. Schoeder'}, {'name': 'Lars Plate'}, {'name': 'Jens Meiler'}]\", 'abstract': '<p>Cystic Fibrosis (CF) is a common genetic disease caused by mutations in the Cystic Fibrosis Transmembrane Conductance Regulator (CFTR), an epithelial anion channel expressed in several vital organs. Absence of functional CFTR results in imbalanced osmotic equilibrium and subsequent mucus build up in the lungs - which increases the risk of infection and eventually causes death. CFTR is an ATP binding cassette (ABC) transporter composed of two transmembrane domains (TMDs), two nucleotide binding domains (NBDs), and an unstructured regulatory domain. The most prevalent patient mutation is the deletion of F508 (ΔF508), making ΔF508 CFTR the primary target for current FDA approved CF therapies. However, no experimental multi-domain ΔF508 CFTR structure has been determined and few studies have modeled ΔF508 using multi-domain WT CFTR structures. Here, we used cryo-EM density data and Rosetta comparative modeling (RosettaCM) to compare a ΔF508 model with published experimental data on CFTR NBD1 thermodynamics. We then apply this modeling method to generate multi-domain WT and ΔF508 CFTR structural models. These models demonstrate the destabilizing effects of ΔF508 on NBD1 and the NBD1/TMD interface in both the closed and open conformation of CFTR. Furthermore, we modeled ΔF508/R1070W and ΔF508 bound to a the CFTR corrector VX-809. Our models reveal the stabilizing effects of R1070W and VX-809 on multi-domain models of ΔF508 CFTR and pave the way for rational design of additional drugs that target ΔF508 CFTR for treatment of CF.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.21266115', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Epithelial RIG-I inflammasome activation suppresses antiviral immunity and promotes inflammatory responses in virus-induced asthma exacerbations and COVID-19', 'authors': \"[{'name': 'U Radzikowska'}, {'name': 'A Eljaszewicz'}, {'name': 'G Tan'}, {'name': 'N Stocker'}, {'name': 'A Heider'}, {'name': 'P Westermann'}, {'name': 'S Steiner'}, {'name': 'A Dreher'}, {'name': 'P Wawrzyniak'}, {'name': 'B Rückert'}, {'name': 'J Rodriguez-Coira'}, {'name': 'D Zhakparov'}, {'name': 'M Huang'}, {'name': 'B Jakiela'}, {'name': 'M Sanak'}, {'name': 'M Moniuszko'}, {'name': 'L O’Mahony'}, {'name': 'T Kebadze'}, {'name': 'DJ Jackson'}, {'name': 'MR Edwards'}, {'name': 'V Thiel'}, {'name': 'SL Johnston'}, {'name': 'CA Akdis'}, {'name': 'M Sokolowska'}]\", 'abstract': '<p>Rhinoviruses (RV) and inhaled allergens, such as house dust mite (HDM) are the major agents responsible for asthma onset, its life-threatening exacerbations and progression to severe disease. The role of severe acute respiratory syndrome coronavirus (SARS-CoV-2) in exacerbations of asthma or the influence of preexisting viral or allergic airway inflammation on the development of coronavirus disease 2019 (COVID-19) is largely unknown. To address this, we compared molecular mechanisms of HDM, RV and SARS-CoV-2 interactions in experimental RV infection in patients with asthma and healthy individuals. RV infection was sensed via retinoic acid-inducible gene I (RIG-I) helicase, but not via NLR family pyrin domain containing 3 (NLRP3), which led to subsequent apoptosis-associated speck like protein containing a caspase recruitment domain (ASC) recruitment, oligomerization and RIG-I inflammasome activation. This phenomenon was augmented in bronchial epithelium in patients with asthma, especially upon pre-exposure to HDM, which itself induced a priming step, pro-IL-1<italic>β</italic> release and early inhibition of RIG-I/TANK binding kinase 1/I<italic>κ</italic>B kinase <italic>ε</italic> /type I/III interferons (RIG-I/TBK1/IKK<italic>ε</italic>/IFN-I/III) responses. Excessive activation of RIG-I inflammasomes was partially responsible for the alteration and persistence of type I/III IFN responses, prolonged viral clearance and unresolved inflammation in asthma. RV/HDM-induced sustained IFN I/III responses initially restricted SARS- CoV-2 replication in epithelium of patients with asthma, but even this limited infection with SARS-CoV-2 augmented RIG-I inflammasome activation. Timely inhibition of the epithelial RIG-I inflammasome and reduction of IL-1<italic>β</italic> signaling may lead to more efficient viral clearance and lower the burden of RV and SARS- CoV-2 infection.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.13.21266289', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Health Policy', 'covid': True, 'title': 'AI/ML Models to Aid in the Diagnosis of COVID-19 Illness from Forced Cough Vocalizations: Good Machine Learning Practice and Good Clinical Practices from Concept to Consumer for AI/ML Software Devices', 'authors': \"[{'name': 'K. Kelley'}, {'name': 'M. Kelley'}, {'name': 'S. C. Kelley'}, {'name': 'A.A. Sakara'}, {'name': 'M.A. Ramirez'}]\", 'abstract': '<p>From a comprehensive and systematic search of the relevant literature on signal data signature (SDS)-based artificial intelligence/machine learning (AI/ML) systems designed to aid in the diagnosis of COVID-19 illness, we identified the highest quality articles with statistically significant data sets for a head-to-head comparison to our own model in development. Further comparisons were made to the recently released “Good Machine Learning Practice (GMLP) for Medical Device Development: Guiding Principles” and, in conclusions, we proposed supplemental principles aimed at bringing AI/ML technologies in closer alignment GMLP and Good Clinical Practices (GCP).</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.468628', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'A non-transcriptional function of Yap orchestrates the DNA replication program', 'authors': \"[{'name': 'Rodrigo Meléndez García'}, {'name': 'Olivier Haccard'}, {'name': 'Albert Chesneau'}, {'name': 'Hemalatha Narassimprakash'}, {'name': 'Jerome E Roger'}, {'name': 'Muriel Perron'}, {'name': 'Kathrin Marheineke'}, {'name': 'Odile Bronchain'}]\", 'abstract': '<p>In multicellular eukaryotic organisms, the initiation of DNA replication occurs asynchronously throughout S-phase according to a regulated replication timing program. Here, using <italic>Xenopus</italic> egg extracts, we showed that Yap (Yes-associated protein 1), a downstream effector of the Hippo signaling pathway, is required for the control of DNA replication dynamics. We found that Yap is recruited to chromatin at the start of DNA replication and that Yap depletion accelerates DNA replication dynamics by increasing the number of activated replication origins. Furthermore, we identified Rif1, a major regulator of the DNA replication timing program, as a novel Yap binding protein. In <italic>Xenopus</italic> embryos, using a Trim-Away approach during cleavage stages devoid of transcription, we found that both Yap and Rif1 depletion trigger an acceleration of cell divisions, suggesting a shorter S-phase by alterations of the replication program. Finally, our data show that Rif1 knockdown leads to defects in the partitioning of early versus late replication foci in retinal stem cells, as we previously showed for Yap. Altogether, our findings unveil a non-transcriptional role for Yap in regulating replication dynamics. We propose that Yap and Rif1 function as breaks to control the DNA replication program in early embryos and post-embryonic stem cells.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.14.21266330', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'SARS-CoV-2 Mortality Surveillance among Community Deaths brought to University Teaching Hospital Mortuary in Lusaka, Zambia, 2020', 'authors': \"[{'name': 'Amos Hamukale'}, {'name': 'Jonas Z. Hines'}, {'name': 'Nyambe Sinyange'}, {'name': 'Sombo Fwoloshi'}, {'name': 'Warren Malambo'}, {'name': 'Suilanji Sivile'}, {'name': 'Stephen Chanda'}, {'name': 'Luchenga Adam Mucheleng’anga'}, {'name': 'Nkomba Kayeyi'}, {'name': 'Cordelia Maria Himwaze'}, {'name': 'Aaron Shibemba'}, {'name': 'Tally Leigh'}, {'name': 'Mazyanga L. Mazaba'}, {'name': 'Nathan Kapata'}, {'name': 'Paul Zulu'}, {'name': 'Khozya Zyambo'}, {'name': 'Francis Mupeta'}, {'name': 'Simon Agolory'}, {'name': 'Lloyd B. Mulenga'}, {'name': 'Kennedy Malama'}, {'name': 'Muzala Kapina'}]\", 'abstract': '<sec><title>Introduction</title><p>During March-December 2020, Zambia recorded 20,725 confirmed COVID-19 cases, with the first wave peaking between July and August. Of the 388 COVID-19-related deaths occurring nationwide, most occurred in the community. We report findings from COVID-19 mortality surveillance among community deaths brought to the University Teaching Hospital (UTH) mortuary in Lusaka.</p></sec><sec><title>Methods</title><p>In Zambia, when a person dies in the community, and is brought into a health facility mortuary, they are recorded as ‘brought in dead’ (BID). The UTH mortuary accepts persons BID for Lusaka District, the most populated district in Zambia. We analyzed data for persons BID at UTH during 2020. We analyzed two data sources: weekly SARS-CoV-2 test results for persons BID and monthly all-cause mortality numbers among persons BID. For all-cause mortality among persons BID, monthly deaths during 2020 that were above the upper bound of the 95% confidence interval for the historic mean (2017-2019) were considered significant. Spearman’s rank test was used to correlate the overall percent positivity in Zambia with all-cause mortality and SARS-CoV-2 testing among persons BID at UTH mortuary.</p></sec><sec><title>Results</title><p>During 2020, 7,756 persons were BID at UTH (monthly range 556-810). SARS-CoV-2 testing began in April 2020, and through December 3,131 (51.9%) of 6,022 persons BID were tested. Of these, 212 (6.8%) were SARS-CoV-2 positive with weekly percent test positivity ranging from 0-32%, with the highest positivity occurring during July 2020. There were 1,139 excess persons BID from all causes at UTH mortuary in 2020 compared to the 2017-2019 mean. The monthly number of persons BID from all causes was above the upper bound of the 95% confidence interval during June-September and December.</p></sec><sec><title>Conclusion</title><p>Increases in all-cause mortality and SARS-CoV-2 test positivity among persons BID at UTH mortuary corresponded with the first peak of the COVID-19 epidemic in June and August 2020, indicating possible increased mortality related to the COVID-19 epidemic in Zambia. Combining all-cause mortality and SARS-CoV-2 testing for persons BID provides useful information about the severity of the epidemic in Lusaka and should be implemented throughout Zambia.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469199', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Three orphan histidine kinases inhibit Clostridioides difficile sporulation', 'authors': \"[{'name': 'Adrianne N. Edwards'}, {'name': 'Daniela Wetzel'}, {'name': 'Michael A. DiCandia'}, {'name': 'Shonna M. McBride'}]\", 'abstract': '<p>The ability of the anaerobic gastrointestinal pathogen, <italic>Clostridioides difficile</italic>, to survive outside the host relies on the formation of dormant endospores. Spore formation is contingent on the activation of a conserved transcription factor, Spo0A, by phosphorylation. Multiple kinases and phosphatases regulate Spo0A activity in other spore-forming organisms; however, these factors are not well conserved in <italic>C. difficile</italic>. Previously, we discovered that deletion of a conserved phosphotransfer protein, CD1492, increases sporulation, indicating that CD1492 inhibits <italic>C. difficile</italic> spore formation. In this study, we investigate the functions of additional conserved orphan phosphotransfer proteins, CD2492, CD1579, and CD1949 which are hypothesized to regulate Spo0A phosphorylation. Disruption of the conserved phosphotransfer protein, CD2492, also increased sporulation frequency, similarly to the <italic>CD1492</italic> mutant, and in contrast to a previous study. A <italic>CD1492 CD2492</italic> mutant phenocopied the sporulation and gene expression patterns of the single mutants, suggesting that these proteins function in the same genetic pathway to repress sporulation. Deletion of the conserved CD1579 phosphotransfer protein also variably increased sporulation frequency; however, knockdown of <italic>CD1949</italic> expression did not influence sporulation. We provide evidence that CD1492, CD2492 and CD1579 function as phosphatases, as mutation of the conserved histidine residue for phosphate transfer abolished CD2492 function, and expression of the <italic>CD1492</italic> or <italic>CD2492</italic> histidine site-directed mutants or the wild-type <italic>CD1579</italic> allele in a parent strain resulted in a dominant negative hypersporulation phenotype. Altogether, at least three phosphotransfer proteins, CD1492, CD2492 and CD1579 (herein, PtpA, PtpB and PtpC) repress <italic>C. difficile</italic> sporulation initiation by regulating activity of Spo0A.</p><sec><title>IMPORTANCE</title><p>The formation of inactive spores is critical for the long-term survival of the gastrointestinal pathogen <italic>Clostridioides difficile</italic>. The onset of sporulation is controlled by the master regulator of sporulation, Spo0A, which is activated by phosphorylation. Multiple kinases and phosphatases control Spo0A phosphorylation; however, this regulatory pathway is not defined in <italic>C. difficile</italic>. We show that two conserved phosphotransfer proteins, CD1492 (PtpA) and CD2492 (PtpB), function in the same regulatory pathway to repress sporulation by preventing Spo0A phosphorylation. We show that another conserved phosphotransfer protein, CD1579 (PtpC), also represses sporulation, and we eliminate the possibility that a fourth orphan histidine kinase protein, CD1949, impacts <italic>C. difficile</italic> sporulation. These results support the idea that <italic>C. difficile</italic> inhibits sporulation initiation through multiple phosphatases.</p></sec>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.14.21266325', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Higher Vaccination Rate Predicts Reduction in SARS-CoV-2 Transmission across the United States', 'authors': \"[{'name': 'Jacky Au'}]\", 'abstract': '<p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) began proliferating widely throughout the world in late 2019/early 2020, creating a global pandemic and health crisis. Although vaccines became available to the public approximately one year after the onset of the pandemic, there still remains much hesitancy surrounding vaccination even two years into the pandemic. One key concern comes from reports of breakthrough infections among the vaccinated that show comparable levels of peak viral load as the unvaccinated, calling into question the ability of vaccines to slow or prevent transmission. Therefore young, healthy individuals who are at low risk of serious complications themselves have little incentive to receive a vaccine that they are not convinced will protect others around them. To address this important concern, this article analyzes COVID-19 incidence in the United States as a function of each state’s vaccination rate. Results show that states with higher percentages of fully vaccinated individuals report fewer new cases among the remaining unvaccinated population. These data add to accumulating evidence that COVID-19 vaccinations can indeed slow the spread of SARS-CoV-2, and are an important tool in society’s arsenal to put this pandemic behind us.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468600', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Marmoset Model for Mycobacterium avium Complex Pulmonary Disease', 'authors': \"[{'name': 'Jay I Peters'}, {'name': 'Diego Jose Maselli'}, {'name': 'Mandeep Mangat'}, {'name': 'Jacqueline J. Coalson'}, {'name': 'Ceci Hinojosa'}, {'name': 'Luis Giavedoni'}, {'name': 'Barbara A. Brown-Elliott'}, {'name': 'Edward D. Chan'}, {'name': 'David E. Griffith'}]\", 'abstract': '<sec><title>Rationale</title><p><italic>Mycobacterium avium</italic> complex, is the most common nontuberculous mycobacterial respiratory pathogen in humans. Disease mechanisms are poorly understood due to the absence of a reliable animal model for <italic>M. avium</italic> complex pulmonary disease.</p></sec><sec><title>Objective</title><p>Assess the susceptibility, immunologic and histopathologic responses of the common marmoset (<italic>Callithrix jacchus</italic>) to <italic>M. avium</italic> complex pulmonary infection.</p></sec><sec><title>Methods</title><p>7 adult female marmosets underwent endobronchial inoculation with 10<sup>8</sup> colony-forming units <italic>of M. intracellulare</italic> and were monitored for 30 or 60 days. Prior to infection, chest radiograph and serum cytokines were assessed; serum cytokines were also monitored weekly for 30 days. At sacrifice 30 days (3 animals) or 60 days (4 animals) after infection, chest radiograph, serum and bronchoalveolar lavage cytokines, histopathology, and cultures of the bronchoalveolar lavage, lungs, liver, and kidney were analyzed.</p></sec><sec><title>Measurements and Main Results</title><p>Five of seven animals (two at 30 days and three at 60 days of infection) had positive lung cultures for <italic>M. intracellulare</italic>. Extra-pulmonary cultures were positive in three animals. All animals appeared healthy throughout the study. All five animals with positive lung cultures had radiographic changes consistent with pneumonitis. At 30 days, those with <italic>M. intracellulare</italic> lung infection showed granulomatous inflammation while at 60 days there was less inflammatory change, but bronchiectasis was noted. The cytokine response in the bronchoalveolar lavage fluid was uniformly greater in the animals with positive <italic>M. intracellulare</italic> cultures than those without a productive infection with greater levels at 30-days compared to 60-days. Similarly, serum cytokines were more elevated in the animals that had positive <italic>M. intracellulare</italic> cultures compared to those without a productive infection, peaking 14-21 days after inoculation.</p></sec><sec><title>Conclusion</title><p>Endobronchial instillation of <italic>M. intracellulare</italic> resulted in pulmonary mycobacterial infection in marmosets with a differential immune response, radiographic and histopathologic abnormalities, and an indolent course consistent with <italic>M. avium</italic> complex lung infection in humans.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.21.469435', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Biomechanics of aging and osteoarthritic human knee ligaments', 'authors': \"[{'name': 'Abby E Peters'}, {'name': 'Brendan Geraghty'}, {'name': 'Karl T Bates'}, {'name': 'Riaz Akhtar'}, {'name': 'Rosti Readioff'}, {'name': 'Eithne Comerford'}]\", 'abstract': '<p>Background. Ligaments work to stabilize the human knee joint and prevent excessive movement. Whilst ligaments are known to decline in structure and function with aging, there has been no systematic effort to study changes in gross mechanical properties in the four major human knee ligaments due to osteoarthritis (OA). This study aims to collate material properties for the anterior (ACL) and posterior (PCL) cruciate ligaments, medial (MCL) and lateral (LCL) collateral ligaments. Our cadaveric samples come from a diverse demographic from which the effects of aging and OA on bone and cartilage material properties have already been quantified. Therefore, by combining our previous bone and cartilage data with the new ligament data from this study we are facilitating subject-specific whole-joint modelling studies.\\n\\nMethods. The demographics of the collected cadaveric knee joints were diverse with age range between 31 to 88 years old, and OA International Cartilage Repair Society grade 0 to 4. Twelve cadaveric human knee joints were dissected, and bone-ligament-bone specimens were extracted for mechanical loading to failure. Ligament material properties were determined from the load-extension curves, namely: linear and ultimate (failure) stress and strain, secant modulus, tangent modulus, and stiffness.\\n\\nResults. There were significant negative correlations between age and ACL linear force (p=0.01), stress (p=0.03) and extension (p=0.05), ACL failure force (p=0.02), stress (p=0.02) and extension (p=0.02), PCL secant (p=0.02) and tangent (p=0.02) modulus, and LCL stiffness (p=0.05). Significant negative correlations were also found between OA grades and ACL linear force (p=0.05), stress (p=0.02), extension (p=0.01) and strain (p=0.03), and LCL failure stress (p=0.05). However, changes in age or OA grade did not show a statistically significant correlation with the MCL tensile parameters. Trends showed that almost all the tensile parameters of the ACL and PCLs decreased with increasing age and progression of OA. Due to small sample size, the combined effect of age and presence of OA could not be statistically derived.\\n\\nConclusions. This research is the first to correlate changes in tensile properties of the four major human knee ligaments to aging and OA. The current ligament study when combined with our previous findings on bone and cartilage for the same twelve knee cadavers, supports conceptualization of OA as a whole-joint disease that impairs the integrity of many peri-articular tissues within the knee. The subject-specific data pool consisting of the material properties of the four major knee ligaments, subchondral and trabecular bones and articular cartilage will aid reconstruction and graft replacements and advance knee joint finite element models, whilst knowledge of aged or diseased mechanics may direct future therapeutic interventions.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.20.469368', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Heat activation and inactivation of bacterial spores. Is there an overlap?', 'authors': \"[{'name': 'Juan Wen'}, {'name': 'Jan P.P.M. Smelt'}, {'name': 'Norbert O.E. Vischer'}, {'name': 'Arend D Vos'}, {'name': 'Peter Setlow'}, {'name': 'Stanley Brul'}]\", 'abstract': '<p>Heat activation at a sublethal temperature is widely applied to promote Bacillus species spore germination. This treatment also has potential to be employed in food processing to eliminate undesired bacterial spores by enhancing their germination, and then inactivating the less heat resistant germinated spores at a milder temperature. However, incorrect heat treatment could also generate heat damage in spores, and lead to more heterogeneous spore germination. Here, the heat activation and heat damage profile of Bacillus subtilis spores was determined by testing spore germination and outgrowth at both population and single spore levels. The heat treatments used were 40-80 degrees Celcius, and for 0-300 min. The results were as follows. 1) Heat activation at 40-70 degrees Celcius promoted L-valine and L-asparagine-glucose-fructose-potassium (AGFK) induced germination in a time dependent manner. 2) The optimal heat activation temperatures for AGFK and L-valine germination via the GerB plus GerK or GerA germinant receptors were 65 and 50-65 degrees Celcius, respectively. 3) Heat inactivation of dormant spores appeared at 70 degrees Celcius, and the heat damage of molecules essential for germination and growth began at 70 and 65 degrees Celcius, respectively. 4) Heat treatment at 75 degrees Celcius resulted in both activation of germination and damage to the germination apparatus, and 80 degrees Celcius treatment caused more pronounced heat damage. 5) For the spores that should withstand adverse environmental temperatures in nature, heat activation seems functional for a subsequent optimal germination process, while heat damage affected both germination and outgrowth.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.18.469099', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Multiple adaptive solutions to face climatic constraints: novel insights in the debate over the role of convergence in local adaptation', 'authors': \"[{'name': 'Badr Benjelloun'}, {'name': 'Kevin Leempoel'}, {'name': 'Frédéric Boyer'}, {'name': 'Sylvie Stucki'}, {'name': 'Ian Streeter'}, {'name': 'Pablo Orozco-terWengel'}, {'name': 'Florian J. Alberto'}, {'name': 'Bertrand Servin'}, {'name': 'Filippo Biscarini'}, {'name': 'Adriana Alberti'}, {'name': 'Stefan Engelen'}, {'name': 'Alessandra Stella'}, {'name': 'Licia Colli'}, {'name': 'Eric Coissac'}, {'name': 'Michael W. Bruford'}, {'name': 'Paolo Ajmone-Marsan'}, {'name': 'Riccardo Negrini'}, {'name': 'Laura Clarke'}, {'name': 'Paul Flicek'}, {'name': 'Abdelkader Chikhi'}, {'name': 'Stéphane Joost'}, {'name': 'Pierre Taberlet'}, {'name': 'François Pompanon'}]\", 'abstract': '<p>The extent to which genomic convergence shapes locally adapted phenotypes in different species remains a fundamental question in evolutionary biology. To bring new insights to this debate we set up a framework which aimed to compare the adaptive trajectories of two domesticated mammal species co-distributed in diversified landscapes. We sequenced the genomes of 160 sheep and 161 goats extensively managed along environmental gradients, including temperature, rainfall, seasonality and altitude, to identify genes and biological processes shaping local adaptation. Allele frequencies at adaptive loci were rarely found to vary gradually along environmental gradients, but rather displayed a discontinuous shift at the extremities of environmental clines. Of the more than 430 adaptive genes identified, only 6 were orthologous between sheep and goats and those responded differently to environmental pressures, suggesting different adaptive mechanisms in these two closely related species. Such diversity of adaptive pathways may result from a high number of biological functions involved in adaptation to multiple eco-climatic gradients, and provides more arguments for the role of contingency and stochasticity in adaptation rather than repeatability.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469120', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Genotyping strategies for detecting CRISPR mutations in polyploid species: a case study-based approach in hexaploid wheat', 'authors': \"[{'name': 'Ajay Gupta'}, {'name': 'Wanlong Li'}]\", 'abstract': '<p>As a versatile tool for genome engineering, CRISPR-Cas9 has been revolutionizing the field of molecular biology, biotechnology, and crop improvement. By precisely targeting pre-selected genomic sites, CRISPR-Cas9 primarily induces insertions or deletions (indels) of variable size. Despite the significant advance in the technology per se, detecting these indels is the major and difficult part of the CRISPR program in polyploid species, like wheat, with relatively low mutation rates. A plethora of methods are available for detecting mutations, but no method is perfect for all mutation types. In this case study, we demonstrated a new, protocol for capturing length polymorphism from small indels using a nested PCR approach. This new method is tractable, efficient, and cost-effective in detecting and genotyping indels &gt;3-bp. We also discussed the major genotyping platforms used in our wheat CRISPR projects, such as mismatch cleavage assay, restriction enzyme assay, ribonucleoprotein assay, and Sanger sequencing, for their advantages and pitfalls in wheat CRISPR mutation detection.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.18.469149', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Heat stress leads to rapid lipid remodelling and transcriptional adaptations in Nicotiana tabacum pollen tubes', 'authors': \"[{'name': 'Hannah E Krawczyk'}, {'name': 'Alexander Helmut Rotsch'}, {'name': 'Cornelia Herrfurth'}, {'name': 'Patricia Scholz'}, {'name': 'Orr Shomroni'}, {'name': 'Gabriela Salinas-Riester'}, {'name': 'Ivo Feussner'}, {'name': 'Till Ischebeck'}]\", 'abstract': '<p>After reaching the stigma, pollen grains germinate and form a pollen tube that transports the sperm cells to the ovule. Due to selection pressure between pollen tubes, they likely evolved mechanisms to quickly adapt to temperature changes to sustain an elongation at the highest possible rate. We investigated these adaptions in Nicotiana tabacum pollen tubes grown in vitro under 22 °C and 37 °C by a multi-omic approach including lipidomic, metabolomic and transcriptomic analysis. Both glycerophospholipids and galactoglycerolipids increased in saturated acyl chains under heat stress while triacylglycerols changed less in respect to desaturation but showed higher levels. Free sterol composition was altered, and sterol ester levels decreased. The levels of sterylglycosides and several sphingolipid classes and species were augmented. Most amino acids increased during heat stress, including the non-codogenic amino acids γ-amino butyrate and pipecolate. Furthermore, the sugars sedoheptulose and sucrose showed higher levels. Also the transcriptome underwent pronounced changes with 1,570 of 24,013 genes being differentially up- and 813 being downregulated. Transcripts coding for heat shock proteins and many transcriptional regulators were most strongly upregulated, but also transcripts that have so far not been linked to heat stress. Transcripts involved in triacylglycerol synthesis were increased, while the modulation of acyl chain desaturation seemed not to be transcriptionally controlled indicating other means of regulation.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.18.468950', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Differential replay for reward and punishment paths predicts approach and avoidance', 'authors': \"[{'name': 'Jessica McFadyen'}, {'name': 'Yunzhe Liu'}, {'name': 'Raymond J Dolan'}]\", 'abstract': '<p>Planning is thought to involve neural replay, where states relevant to a task goal are rapidly reactivated in sequence. It remains unclear if, during planning, replay of a path relates to an actual prospective choice. Here, using magnetoencephalography (MEG), we studied participants while they planned to either approach or avoid an uncertain environment that contained paths leading to reward and to punishment. We show significant planning-related forward sequential replay with state-to-state transitions in the range of 20 to 90 ms. Replay of rewarding paths was boosted prior to a decision to avoid, and attenuated prior to a decision to approach. Crucially, a trial-by-trial bias in replaying punishing paths predicted an irrational choice to approach when a prospective environment was more risky, an effect that was particularly marked in more anxious participants. The findings reveal a coupling between the content of forwards replay and rational choice behaviour, such that replay prioritises an online representation of potential reward.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469142', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'TWIST1 methylation by SETD6 selectively antagonizes LINC-PINT expression in Glioblastoma multiforme', 'authors': \"[{'name': 'Lee Admoni-Elisha'}, {'name': 'Michal Feldman'}, {'name': 'Tzofit Elbaz'}, {'name': 'Anand Chopra'}, {'name': 'Guy Shapira'}, {'name': 'Christopher J Fry'}, {'name': 'Noam Shomron'}, {'name': 'Kyle Biggar'}, {'name': 'Dan Levy'}]\", 'abstract': '<p>Glioblastoma multiforme (GBM) is the most common and aggressive malignant brain tumor among adults, which is characterized by high invasion, migration and proliferation abilities. One important process that contributes to the invasiveness of GBM is the epithelial to mesenchymal transition (EMT). EMT is regulated by a set of defined transcription factors which tightly regulate this process, among them is the basic helix-loop-helix family member, TWIST1. Here we show that TWIST1 is methylated on lysine-33 at chromatin by SETD6, a methyltransferase with expression levels correlating with poor survival in GBM patients. RNA-seq analysis in U251 GBM cells suggested that both SETD6 and TWIST1 regulate cell adhesion and migration processes. We further show that TWIST1 methylation attenuates the expression of the long-non-coding RNA, LINC-PINT, thereby suppressing EMT in GBM. Mechanistically, TWIST1 methylation represses the transcription of LINC-PINT by increasing the occupancy of EZH2 and the catalysis of the repressive H3K27me3 mark at the LINC-PINT locus. Under un-methylated conditions, TWIST1 dissociates from the LINC-PINT locus, allowing the expression of LINC-PINT which leads to increased cell adhesion and decreased cell migration. Together, our findings unravel a new mechanistic dimension for selective expression of LINC-PINT mediated by TWIST1 methylation.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.16.468893', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': True, 'title': 'BSG/CD147 and ACE2 receptors facilitate SARS-CoV-2 infection of human iPS cell-derived kidney podocytes', 'authors': \"[{'name': 'Titilola D. Kalejaiye'}, {'name': 'Rohan Bhattacharya'}, {'name': 'Morgan A. Burt'}, {'name': 'Tatianna Travieso'}, {'name': 'Arinze E. Okafor'}, {'name': 'Xingrui Mou'}, {'name': 'Maria Blasi'}, {'name': 'Samira Musah'}]\", 'abstract': '<sec><title>Background</title><p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes the Coronavirus disease 2019 (COVID-19), which was declared a pandemic by the World Health Organization (WHO) in March 2020. The disease has caused more than 5.1 million deaths worldwide. While cells in the respiratory system are frequently the initial target for SARS-CoV-2, clinical studies suggest that COVID-19 can become a multi-organ disease in the most severe cases. Still, the direct affinity of SARS-CoV-2 for cells in other organs such as the kidneys, which are often affected in severe COVID-19, remains poorly understood.</p></sec><sec><title>Method</title><p>In this study, we employed a human induced pluripotent stem (iPS) cell-derived model to investigate the affinity of SARS-CoV-2 for kidney glomerular podocytes. We studied uptake of the live SARS-CoV-2 virus as well as pseudotyped viral particles by human iPS cell derived podocytes using qPCR, western blot, and immunofluorescence. Global gene expression and qPCR analyses revealed that human iPS cell-derived podocytes express many host factor genes (including ACE2, BSG/CD147, PLS3, ACTR3, DOCK7, TMPRSS2, CTSL CD209, and CD33) associated with SARS-CoV-2 binding and viral processing.</p></sec><sec><title>Result</title><p>Infection of podocytes with live SARS-CoV-2 or spike-pseudotyped lentiviral particles revealed viral uptake by the cells at low Multiplicity of Infection (MOI of 0.01) as confirmed by RNA quantification and immunofluorescence studies. Our results also indicate that direct infection of human iPS cell-derived podocytes by SARS-CoV-2 virus can cause cell death and podocyte foot process retraction, a hallmark of podocytopathies and progressive glomerular diseases including collapsing glomerulopathy observed in patients with severe COVID-19 disease. Additionally, antibody blocking experiments identified BSG/CD147 and ACE2 receptors as key mediators of spike binding activity in human iPS cell-derived podocytes.</p></sec><sec><title>Conclusion</title><p>These results show that SARS-CoV-2 can infect kidney glomerular podocytes <italic>in vitro</italic>. These results also show that the uptake of SARS-CoV-2 by kidney podocytes occurs via multiple binding interactions and partners, which may underlie the high affinity of SARS-CoV-2 for kidney tissues. This stem cell-derived model is potentially useful for kidney-specific antiviral drug screening and mechanistic studies of COVID-19 organotropism.</p></sec><sec><title>Significant statement</title><p>Many patients with COVID19 disease exhibit multiorgan complications, suggesting that SARS-CoV-2 infection can extend beyond the respiratory system. Acute kidney injury is a common COVID-19 complication contributing to increased morbidity and mortality. Still, SARS-Cov-2 affinity for specialized kidney cells remain less clear. By leveraging our protocol for stem cell differentiation, we show that SARS-CoV-2 can directly infect kidney glomerular podocytes by using multiple Spike-binding proteins including ACE2 and BSG/CD147. Our results also indicate that infection by SARS-CoV-2 virus can cause podocyte cell death and foot process effacement, a hallmark of podocytopathies including collapsing glomerulopathy observed in patients with severe COVID-19 disease. This stem cell-derived model is potentially useful for kidney-specific antiviral drug screening and mechanistic studies of COVID-19 organotropism.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.17.468820', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'The relationship between land cover and microbial community composition in European lakes', 'authors': \"[{'name': 'Theodor Sperlea'}, {'name': 'Jan Philip Schenk'}, {'name': 'Hagen Dreßler'}, {'name': 'Daniela Beisser'}, {'name': 'Georges Hattab'}, {'name': 'Jens Boenigk'}, {'name': 'Dominik Heider'}]\", 'abstract': '<p>Microbes such as bacteria, archaea, and protists are essential for element cycling and ecosystem functioning, but many questions central to the understanding of the role of microbes in ecology are still open. Here, we analyze the relationship between lake microbiomes and the land cover surrounding the lakes. By applying machine learning methods, we quantify the covariance between land cover categories and the microbial community composition recorded in the largest amplicon sequencing dataset of European lakes available to date. We identify microbial bioindicators for these land cover categories. Combining land cover and physico-chemical bioindicators identified from the same amplicon sequencing dataset, we develop two novel similarity metrics that facilitate insights into the ecology of the lake microbiome. We show that the bioindicator network, i.e., the graph linking OTUs indicative of the same environmental parameters, corresponds to microbial co-occurrence patterns. Taken together, we demonstrate the strength of machine learning approaches to identify correlations between microbial diversity and environmental factors, potentially opening new approaches to integrate environmental molecular diversity into monitoring and water quality assessments.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.13.468488', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Single-sample proteome enrichment enables missing protein recovery and phenotype association', 'authors': \"[{'name': 'Bertrand Jern Han Wong'}, {'name': 'Weijia Kong'}, {'name': 'Wilson Wen Bin Goh'}]\", 'abstract': '<p>Proteomic studies characterize the protein composition of complex biological samples. Despite recent developments in mass spectrometry instrumentation and computational tools, low proteome coverage remains a challenge. To address this, we present Proteome Support Vector Enrichment (PROSE), a fast, scalable, and effective pipeline for scoring protein identifications based on gene co-expression matrices. Using a simple set of observed proteins as input, PROSE gauges the relative importance of proteins in the phenotype. The resultant enrichment scores are interpretable and stable, corresponding well to the source phenotype, thus enabling reproducible recovery of missing proteins. We further demonstrate its utility via reanalysis of the Cancer Cell Line Encyclopedia (CCLE) proteomic data, with prediction of oncogenic dependencies and identification of well-defined regulatory modules. PROSE is available as a user-friendly Python module from <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/bwbio/PROSE\">https://github.com/bwbio/PROSE</ext-link>.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.468928', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Deciphering a hexameric protein complex with Angstrom optical resolution', 'authors': \"[{'name': 'Hisham Mazal'}, {'name': 'Franz-Ferdinand Wieser'}, {'name': 'Vahid Sandoghdar'}]\", 'abstract': '<p>Cryogenic optical localization in three dimensions (COLD) was recently shown to resolve up to four binding sites on a single protein. However, because COLD relies on intensity fluctuations that result from the blinking behavior of fluorophores, it is limited to cases, where individual emitters show different brightness. This significantly lowers the measurement yield. To extend the number of resolved sites as well as the measurement yield, we employ partial labeling and combine it with polarization encoding in order to identify single fluorophores during their stochastic blinking. We then use a particle classification scheme to identify and resolve heterogenous subsets and combine them to reconstruct the three-dimensional arrangement of large molecular complexes. We showcase this method (polarCOLD) by resolving the trimer arrangement of proliferating cell nuclear antigen (PCNA) and the hexamer geometry of Caseinolytic Peptidase B (ClpB) of Thermus thermophilus in its quaternary structure, both with Angstrom resolution. The combination of polarCOLD and single-particle cryogenic electron microscopy (cryoEM) promises to provide crucial insight into intrinsic, environmental and dynamic heterogeneities of biomolecular structures. Furthermore, our approach is fully compatible with fluorescent protein labeling and can, thus, be used in a wide range of studies in cell and membrane biology.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.03.467047', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'The evolution of color naming reflects pressure for efficiency: Evidence from the recent past', 'authors': \"[{'name': 'Noga Zaslavsky'}, {'name': 'Karee Garvin'}, {'name': 'Charles Kemp'}, {'name': 'Naftali Tishby'}, {'name': 'Terry Regier'}]\", 'abstract': '<p>It has been proposed that semantic systems evolve under pressure for efficiency. This hypothesis has so far been supported largely indirectly, by synchronic cross-language comparison, rather than directly by diachronic data. Here, we directly test this hypothesis in the domain of color naming, by analyzing recent diachronic data from Nafaanra, a language of Ghana and Côte d’Ivoire, and comparing it with quantitative predictions derived from the mathematical theory of efficient data compression. We show that color naming in Nafaanra has changed over the past four decades while remaining near-optimally efficient, and that this outcome would be unlikely under a random drift process that maintains structured color categories without pressure for efficiency. To our knowledge, this finding provides the first direct evidence that color naming evolves under pressure for efficiency, supporting the hypothesis that efficiency shapes the evolution of the lexicon.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.15.468720', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Susceptibility of sheep to experimental co-infection with the ancestral lineage of SARS-CoV-2 and its alpha variant', 'authors': \"[{'name': 'Natasha N. Gaudreault'}, {'name': 'Konner Cool'}, {'name': 'Jessie D. Trujillo'}, {'name': 'Igor Morozov'}, {'name': 'David A. Meekins'}, {'name': 'Chester McDowell'}, {'name': 'Dashzeveg Bold'}, {'name': 'Mariano Carossino'}, {'name': 'Velmurugan Balaraman'}, {'name': 'Dana Mitzel'}, {'name': 'Taeyong Kwon'}, {'name': 'Daniel W. Madden'}, {'name': 'Bianca Libanori Artiaga'}, {'name': 'Roman M. Pogranichniy'}, {'name': 'Gleyder Roman-Sosa'}, {'name': 'William C. Wilson'}, {'name': 'Udeni B. R. Balasuriya'}, {'name': 'Adolfo García-Sastre'}, {'name': 'Juergen A. Richt'}]\", 'abstract': '<p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is responsible for a global pandemic that has had significant impacts on human health and economies worldwide. SARS-CoV-2 is highly transmissible and the cause of coronavirus disease 2019 (COVID-19) in humans. A wide range of animal species have also been shown to be susceptible to SARS-CoV-2 infection by experimental and/or natural infections. Domestic and large cats, mink, ferrets, hamsters, deer mice, white-tailed deer, and non-human primates have been shown to be highly susceptible, whereas other species such as mice, dogs, pigs, and cattle appear to be refractory to infection or have very limited susceptibility. Sheep (<italic>Ovis aries</italic>) are a commonly farmed domestic ruminant that have not previously been thoroughly investigated for their susceptibility to SARS-CoV-2. Therefore, we performed <italic>in vitro</italic> and <italic>in vivo</italic> studies which consisted of infection of ruminant-derived cell cultures and experimental challenge of sheep to investigate their susceptibility to SARS-CoV-2. Our results showed that sheep-derived cell cultures support SARS-CoV-2 replication. Furthermore, experimental challenge of sheep demonstrated limited infection with viral RNA shed in nasal and oral swabs primarily at 1-day post challenge (DPC), and also detected in the respiratory tract and lymphoid tissues at 4 and 8 DPC. Sero-reactivity was also observed in some of the principal infected sheep but not the contact sentinels, indicating that transmission to co-mingled naïve sheep was not highly efficient; however, viral RNA was detected in some of the respiratory tract tissues of sentinel animals at 21 DPC. Furthermore, we used challenge inoculum consisting of a mixture of two SARS-CoV-2 isolates, representatives of the ancestral lineage A and the B.1.1.7-like alpha variant of concern (VOC), to study competition of the two virus strains. Our results indicate that sheep show low susceptibility to SARS-CoV-2 infection, and that the alpha VOC outcompeted the ancestral lineage A strain.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.16.468875', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Birnaviridae virus factories show features of liquid-liquid phase separation, and are distinct from paracrystalline arrays of virions observed by electron microscopy', 'authors': \"[{'name': 'Vishwanatha R. A. P. Reddy'}, {'name': 'Elle A. Campbell'}, {'name': 'Joanna Wells'}, {'name': 'Jennifer Simpson'}, {'name': 'Salik Nazki'}, {'name': 'Philippa C. Hawes'}, {'name': 'Andrew J. Broadbent'}]\", 'abstract': '<p>To gain more information about the nature of <italic>Birnaviridae</italic> virus factories (VFs), we used a recombinant infectious bursal disease virus (IBDV) expressing split-GFP11 tagged to the polymerase (VP1) that we have previously shown is a marker for VFs in infected cells expressing GFP1-10. We found that VFs co-localized with 5-ethynyl uridine in the presence of actinomycin D, confirming they were the site of <italic>de novo</italic> RNA synthesis, and VFs were visible in infected cells that were fixed and permeabilized with digitonin, demonstrating that they were not membrane bound. Fluorescence recovery after photobleaching (FRAP) a region of interest within the VFs occurred rapidly, recovering from approximately 25% to 87% the original intensity over 146 seconds, and VFs were dissolved by 1,6-hexanediol treatment, demonstrating they showed properties consistent with liquid-liquid phase separation. There was a lower co-localization of the VF GFP signal with the capsid protein VP2 (Manders’ coefficient (MC) 0.6), compared to VP3 (MC, 0.9), which prompted us to investigate the VF ultrastructure by transmission electron microscopy (TEM). In infected cells, paracrystalline arrays (PAs) of virions were observed in the cytoplasm, as well as discrete electron dense regions. Using correlative light and electron microscopy (CLEM), we observed that the electron dense regions correlated with the GFP signal of the VFs, which were distinct from the PAs. In summary, <italic>Birnaviridae</italic> VFs are sites of <italic>de novo</italic> RNA synthesis, are not bound by a membrane, show properties consistent with liquid-liquid phase separation, and are distinct from the PAs observed by TEM.</p><sec><title>Importance</title><p>Members of the <italic>Birnaviridae</italic> infect birds, fish and insects, and are responsible for diseases of significant economic importance to the poultry industry and aquaculture. Despite their importance, how they replicate in cells remains poorly understood. Here, we show that the <italic>Birnaviridae</italic> virus factories are not membrane bound, demonstrate properties consistent with liquid-liquid phase separation, and are distinct from the paracrystalline arrays of virions observed by transmission electron microscopy, enhancing our fundamental knowledge of virus replication that could be used to develop strategies to control disease, or optimize their therapeutic application.</p></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.20.469315', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Conformational Changes in the Negative Arm of the Circadian Clock Correlate with Dynamic Interactomes Involved in Post-transcriptionally Regulated Processes', 'authors': \"[{'name': 'Jacqueline F Pelham'}, {'name': 'Alexander E Mosier'}, {'name': 'Samuel C Altshuler'}, {'name': 'Christopher L Kirchhoff'}, {'name': 'William B Fall'}, {'name': 'Lisa S Baik'}, {'name': 'Joanna C Chiu'}, {'name': 'Jennifer Marie Hurley'}]\", 'abstract': '<p>The circadian clock employs a transcriptional/translational negative feedback loop (TTFL) to anticipate environmental changes due to the Earth′s diurnal cycle, with regulation of organismal physiology believed to stem from temporal transcriptional activation by the positive arm. However, up to 80% of oscillating proteins do not have rhythmic mRNA, establishing circadian post-transcriptional regulation through unknown mechanisms. Given the pervasive conservation of the intrinsically disordered nature of negative-arm clock proteins, we hypothesized that post-transcriptional regulation may stem from conformational shifts in negative-arm proteins that time vacillations in the constituents of negative-arm macromolecular complexes to time cellular physiology. Our investigation of the negative arm clock protein in <italic>Neurospora crassa</italic>, FREQUENCY (FRQ), demonstrated temporal conformational fluidity correlated with daily changes in physiologically diverse macromolecular complex components. A parallel investigation of the macromolecular complexes centered around <italic>Drosophila melanogaster</italic> PERIOD (dPER) and human PERIOD (hPER2) found a similar number and physiological diversity of interacting partners in higher eukaryotes. Short linear motifs (SLiMs) associated with the interactors localized to disordered and phosphorylated regions on the PERs and FRQ, with disordered interactors oscillating in the macromolecular complexes over circadian time. This oscillation correlated with oscillations in post-transcriptionally regulated proteins, suggesting the negative arm may tune cellular physiology and proteostasis post-transcriptionally via vacillations in the circadian negative-arm macromolecular protein complexes.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.18.469045', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Ubiquitous mRNA decay fragments in E. coli redefine the functional transcriptome', 'authors': \"[{'name': 'Lydia Herzel'}, {'name': 'Julian A. Stanley'}, {'name': 'James C. Taggart'}, {'name': 'Gene-Wei Li'}]\", 'abstract': '<p>Bacterial mRNAs have short life cycles, in which transcription is rapidly followed by translation and degradation within seconds to minutes. The resulting diversity of mRNAs impacts their functionality but has remained unresolved. Here we quantitatively map the 3’ status of cellular RNAs in <italic>Escherichia coli</italic> during steady-state growth and report a large fraction of molecules (median&gt;60%) that are fragments of canonical full-length mRNAs. The majority of RNA fragments are decay intermediates following endonuclease cleavage by RNase E and yet-unknown nucleases, whereas nascent RNAs contribute to a smaller fraction. Despite the prevalence of decay intermediates in total RNA, they are underrepresented in the pool of ribosome-associated transcripts and can thus distort quantifications for the abundance of full-length, functional mRNAs. The large heterogeneity within mRNA molecules <italic>in vivo</italic> highlights the importance in discerning functional transcripts and provides a lens for studying the dynamic life cycle of mRNAs.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.18.468957', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'TIS transformer: Re-annotation of the human proteome using deep learning', 'authors': \"[{'name': 'Jim Clauwaert'}, {'name': 'Zahra McVey'}, {'name': 'Ramneek Gupta'}, {'name': 'Gerben Menschaert'}]\", 'abstract': \"<p>The precise detection of translation initiation sites is essential for proteome delineation. In turn, the accurate mapping of the proteome is fundamental in advancing our understanding of biological systems and cellular mechanisms. We propose TIS Transformer, a deep learning model for the determination of translation start sites, based on information embedded in processed transcript nucleotide sequences. Through the application of deep learning techniques first designed for natural language processing tasks, we have developed an approach that achieves state-of-the-art performances on the prediction of translation initiation sites. TIS Transformer utilizes the FAVOR+ algorithm for attention calculation, enabling processing of full transcript sequences by the model. Analysis of input importance revealed TIS Transformer's ability to detect key features of translation, such as translation stop sites and reading frames. Furthermore, we demonstrate TIS Transformer's ability to detect multiple peptides on a transcript, and peptides encoded by short Open Reading Frames (sORFs), either alongside a canonical coding sequence or in long non-coding RNAs. Using a cross-validation scheme, we apply TIS Transformer to re-annotate the full human transcriptome.</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468122', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The output of interneurons in the primary visual cortex is best reflected by pre-synaptic activity, not somatic activity', 'authors': \"[{'name': 'Rozan Vroman'}, {'name': 'Lawrie S McKay'}]\", 'abstract': '<p>Recent advances in 2-photon calcium-imaging in awake mice have made it possible to study the effect of different behavioural states on cortical circuitry. Many studies assume that somatic activity can be used as a measure for neuronal output. We set out to test the validity of this assumption by comparing somatic activity with the pre-synaptic activity of VIP (Vasoactive intestinal peptide)- and SST (Somatostatin)-positive interneurons in layer 2/3 of the primary visual cortex (V1). We used mice expressing genetically encoded calcium indicators in VIP/SST-interneurons across the whole cell (VIP/SST:GCaMP6f) or confined to pre-synapses (VIP/SST:SyGCaMP5). Mice were exposed to a full-field visual stimulation protocol consisting of 60-second-long presentations of moving Gabor gratings (0.04 cpd, 2 Hz) alternated by 30 seconds of grey screen. During imaging, mice were placed on an air-suspended Styrofoam ball, allowing them to run voluntarily. We compared neural activity during three 4-second time-windows: Before visual stimulation (−4 to 0 sec), during the initial onset (1 to 5 sec) and at the end of the stimulation (56 to 60 sec.). These were further compared while the mice were stationary and while they were voluntarily locomoting. Unlike VIP-somas, VIP-pre-synapses showed strong suppressive responses to the visual stimulus. Furthermore, VIP-somas were positively correlated with locomotion, whereas in VIP-synapses we observed a split between positive and negative correlations. In addition, a similar but weaker distinction was found between SST-somas and pre-synapses. The excitatory effect of locomotion in VIP-somas increased over the course of the visual stimulus but this property was only shared with the positively correlated VIP-pre-synapses. The remaining negatively correlated pre-synapses showed no relation to the overall activity of the Soma. Our results suggest that when making statements about the involvement of interneurons in V1 layer 2/3 circuitry it is crucial to measure from synaptic terminals as well as from somas.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.467373', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Integrating transcriptomes to investigate genes associated with adaptation to aquatic environments, and assess phylogenetic conflict and whole-genome duplications in Alismatales', 'authors': \"[{'name': 'Ling-Yun Chen'}, {'name': 'Bei Lu'}, {'name': 'Diego F. Morales-Briones'}, {'name': 'Michael L. Moody'}, {'name': 'Fan Liu'}, {'name': 'Guang-Wan Hu'}, {'name': 'Chien-Hsun Huang'}, {'name': 'Jin-Ming Chen'}, {'name': 'Qing-Feng Wang'}]\", 'abstract': '<p>Land plants first evolved from freshwater algae, and flowering plants returned to water as early as the Cretaceous and multiple times beyond. Alismatales is the largest clade of aquatic angiosperms including all marine angiosperms, as well as terrestrial plants. We used Alismatales to explore plant adaptation to aquatic environments by including 95 samples (89 Alismatales species) covering four genomes and 91 transcriptomes (59 generated in this study). To provide a basis for investigating adaptation, we assessed phylogenetic conflict and whole-genome duplication (WGD) events in Alismatales. We recovered a relationship for the three main clades in Alismatales as ((Tofieldiaceae, Araceae), core Alismatids). There is phylogenetic conflict among the backbone of the three main clades that could be due to incomplete lineage sorting and introgression. We identified 18 putative WGD events. One of them had occurred at the most recent common ancestor of core Alismatids, and four occurred at seagrass lineages. Other events are distributed in terrestrial, emergent, and submersed life-forms and seagrasses across Alismatales. We also found that lineage and life-form were each important for different evolutionary patterns for the genes related to freshwater/marine adaptation. For example, some light or ethylene-related genes were lost in the seagrass Zosteraceae, but present in other seagrasses and freshwater species. Stomata-related genes were lost in both submersed freshwater species and seagrasses. Nicotianamine synthase genes, which are important in iron intake, expanded in both submersed freshwater species and seagrasses. Our results advance the understanding of the adaptation to aquatic environments, phylogeny, and whole-genome duplication of Alismatales.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.12.468309', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Repressing PTBP1 is incapable to convert reactive astrocytes to dopaminergic neurons in a mouse model of Parkinson’s disease', 'authors': \"[{'name': 'Weizhao Chen'}, {'name': 'Qiongping Zheng'}, {'name': 'Qiaoying Huang'}, {'name': 'Shanshan Ma'}, {'name': 'Mingtao Li'}]\", 'abstract': '<p>Lineage reprograming of resident glia cells to induced dopaminergic neurons (iDAns) holds attractive prospect for cell-replacement therapy of Parkinson’s disease (PD). Recently, whether repressing polypyrimidine tract binding protein 1 (PTBP1) could truly achieve efficient astrocyte-to-iDAn conversion in substantia nigra and striatum aroused widespread controversy. Although reporter positive iDAns were observed by two groups after delivering adeno-associated virus (AAV) expressing a reporter with shRNA or Crispr-CasRx to repress astroglial PTBP1, the possibility of AAV leaking into endogenous DAns could not be excluded without using a reliable lineage tracing method. By adopting stringent lineage tracing strategy, two other studies showed that neither knockdown nor genetic deletion of quiescent astroglial PTBP1 fails to obtain iDAns under physiological condition. However, the role of reactive astrocyte might be underestimated since upon brain injury, reactive astrocyte could acquire certain stem cell hallmarks which may facilitate the lineage conversion process. Therefore, whether reactive astrocytes could be genuinely converted to DAns after PTBP1 repression in a PD model needs further validation. In this study, we used <italic>Aldh1l1-CreER</italic><sup><italic>T2</italic></sup>-mediated specific astrocyte-lineage tracing method to investigate whether reactive astrocytes could be converted to DAns in the 6-OHDA PD model. However, we found that no astrocyte-originated DAn was generated after effective knockdown of astroglial PTBP1 either in the substantia nigra or in the striatum, while AAV “leakage” to nearby neurons was observed. Our results further confirmed that repressing PTBP1 is unable to convert astrocytes to DAns no matter in physiological or PD-related pathological conditions.</p><sec><title>Graphic abstract</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"468309v2_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>AAV-sh<italic>Ptbp1</italic> rapidly and efficiently induces viral-reporter-labeled DAns in mouse brain under physiological condition</p></list-item><list-item><p>Viral-reporter-positive DAns are not originated from PTBP1 repressed and lineage traced reactive astrocytes in a mouse PD model</p></list-item></list></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.16.468802', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'NUDT18 catalyzes hydrolysis of active metabolites of the antivirals Remdesivir and Ribavirin', 'authors': \"[{'name': 'Ann-Sofie Jemth'}, {'name': 'Emma Rose Scaletti'}, {'name': 'Evert Homan'}, {'name': 'Pål Stenmark'}, {'name': 'Thomas Helleday'}, {'name': 'Maurice Michel'}]\", 'abstract': '<p>Remdesivir (GS-5734) has gained considerable interest due to its activity against replication of SARS-CoV2. Remdesivir is a broad-spectrum antiviral prodrug that is hydrolyzed intracellularly and phosphorylated by cellular kinases to its active triphosphate form (Remdesivir-TP). Here we tested Remdesivir-TP as a substrate for a panel of human hydrolases and found that NUDIX hydrolase 18 (NUDT18) catalyzes the hydrolysis of Remdesivir-TP. NUDT18 is expressed in respiratory epithelial cells suggesting that NUDT18 may limit the antiviral efficacy of Remdesivir by decreasing the intracellular concentration of its active metabolite at its intended site of action. The k<sub>cat</sub> of NUDT18 for Remdesivir-TP was determined to 2.6 s<sup>-1</sup> and the K<sub>m</sub> value was 156 μM, suggesting that NUDT18 catalyzed hydrolysis occurs in cells. In addition, we found that the triphosphate of the antiviral Ribavirin, with broad-spectrum activity against several RNA and DNA viruses, was hydrolyzed by NUDT18, albeit with a lower efficiency compared to Remdesivir-TP. NUDT18 activity was also tested with the triphosphates of the antivirals Sofosbuvir and Aciclovir for which low activity, in comparison to activities with Remdesivir-TP and Ribavirin-TP, was detected. These results suggest that NUDT18 can act as a cellular sanitizer and may influence the antiviral efficacy of Remdesivir and Ribavirin.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.17.469022', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'The natural adaptation and human selection history of African sheep genomes', 'authors': \"[{'name': 'Abulgasim M Ahbara'}, {'name': 'Christelle Robert'}, {'name': 'Adebabay Kebede'}, {'name': 'Ayele Abebe'}, {'name': 'Suliman Latairish'}, {'name': 'Mukhtar Omar Agoub'}, {'name': 'Hassan H. Musa'}, {'name': 'Pam Wiener'}, {'name': 'Emily Clark'}, {'name': 'Olivier Hanotte'}, {'name': 'Joram M. Mwacharo'}]\", 'abstract': '<p>African sheep manifest diverse but distinct physio-anatomical traits which are the outcomes of natural- and human-driven selection. Here, we generated 34.8 million variants from 150 indigenous African sheep genomes sequenced at an average depth of ∼54x for 130 samples (Ethiopia, Libya) and ∼10x for 20 samples (Sudan), representing sheep from diverse environments, tail morphology and post-Neolithic introductions to Africa. Phylogenetic and model-based admixture analysis provided evidence of four genetic groups that correspond to altitudinal geographic origins and tail morphotypes. Comparative genomic analysis identified targets of selection spanning conserved haplotype structures overlapping genes and gene families relating to hypoxia responses, caudal vertebrae and tail skeleton length, ear morphology, and tail fat-depot structure. Our findings provide novel insights underpinning variation and response to human selection and environmental adaptation, and possible pleiotropic gene interactions in indigenous African sheep genomes, which guaranteed the successful establishment of the species on the continent.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468558', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Bacteriophage cocktails displayed protective effects against soft rot-causing Pectobacterium sp.', 'authors': \"[{'name': 'Aeron Jade S. Parena'}, {'name': 'Benji Brayan I. Silva'}, {'name': 'Rae Mark L. Mercado'}, {'name': 'Adelbert Adrian A. Sendon'}, {'name': 'Freddiewebb B. Signabon'}, {'name': 'Johnny F. Balidion'}, {'name': 'Jaymee R. Encabo'}]\", 'abstract': '<p>Soft rot caused by <italic>Pectobacterium</italic> spp. is responsible for significant losses in vegetable production worldwide. Methods for the effective control of this disease are limited and are primarily based on good agricultural practices. The use of phages as biocontrol agents appears to be a promising alternative to combat phytopathogens. In this study, we investigated the efficacy of lytic phages against soft rot caused by <italic>Pectobacterium carotovorum</italic> subsp. <italic>carotovorum</italic>. Designated as PPc_A3, PPc_D1, and PPc_J3, three bacteriophage isolates, which were recovered from symptomatic tissues and environmental samples, were observed to effectively lyse <italic>P. carotovorum</italic> subsp. <italic>carotovorum</italic>. PPc_A3 belongs to the <italic>Podoviridae</italic> family, while phages PPc_D1 and PPc_J3 belong to the <italic>Myoviridae</italic> family based on the morphological features of the virions as observed using transmission electron microscopy. The optimal multiplicity of infection (MOI) differed greatly among the three phages. All survived incubations at 30°C, 40°C and 50°C and at pH ranging from 3.0 to 9.0, but were all inactivated at 60°C and at pH 12. Both monophage and cocktail preparations were effective in inhibiting the growth of <italic>P. carotovorum</italic> subsp. <italic>carotovorum</italic> in <italic>in vitro</italic> challenge tests. In the semi-<italic>in planta</italic> assays, monophage treatments resulted in significant reduction of tissue maceration in potato slices, while treatment with cocktail preparations completely inhibited the development of soft rot disease. Overall, these results demonstrate the efficacy of cocktail formulations of phages PPc_A3, PPc_D1, and PPc_J3 for the biocontrol of soft rot disease caused by <italic>P. carotovorum</italic> subsp. <italic>carotovorum</italic>.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.467680', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Loss of soluble guanylyl cyclase in platelets contributes to atherosclerotic plaque formation and vascular inflammation', 'authors': \"[{'name': 'Carina Mauersberger'}, {'name': 'Hendrik B. Sager'}, {'name': 'Jana Wobst'}, {'name': 'Tan An Dang'}, {'name': 'Laura Lambrecht'}, {'name': 'Simon Koplev'}, {'name': 'Marlène Stroth'}, {'name': 'Noomen Bettaga'}, {'name': 'Jens Schlossmann'}, {'name': 'Frank Wunder'}, {'name': 'Andreas Friebe'}, {'name': 'Johan L.M. Björkegren'}, {'name': 'Lisa Dietz'}, {'name': 'Peter Sandner'}, {'name': 'Oliver Soehnlein'}, {'name': 'Heribert Schunkert'}, {'name': 'Thorsten Kessler'}]\", 'abstract': '<sec><title>Aim</title><p>The role of platelets in atherosclerosis remains incompletely understood. Variants in genes encoding the soluble guanylyl cyclase (sGC) in platelets are associated with coronary artery disease (CAD) risk. Here we sought to investigate the contribution of platelet sGC to atherosclerosis and the therapeutic potential of targeting sGC in atherosclerosis.</p></sec><sec><title>Methods and Results</title><p>We genetically deleted sGC in platelets of atherosclerosis-prone <italic>Ldlr</italic><sup>-/-</sup> mice. By intravital fluorescence microscopy such Pf4-<italic>Cre</italic><sup>+</sup><italic>Gucy1b1</italic><sup>flox</sup>/<sup>flox</sup><italic>Ldlr</italic><sup>-/-</sup> mice displayed enhanced leukocyte adhesion to atherosclerotic plaques in comparison with their litter mates. Moreover, histological and flow cytometry analyses revealed more numerous inflammatory leukocytes and larger plaque sizes in aortic tissue of <italic>Ldlr</italic><sup>-/-</sup> mice lacking sGC in platelets. <italic>In vitro</italic>, supernatant from activated platelets lacking sGC promoted leukocyte adhesion to endothelial cells (EC) via enhanced EC activation. Using cytokine profiling, we identified reduced angiopoietin-1 release by Pf4-<italic>Cre</italic><sup>+</sup><italic>Gucy1b1</italic><sup>flox</sup>/<sup>flox</sup> and human <italic>GUCY1A1</italic> risk allele carrier platelets to be responsible for enhanced activation of EC and subsequent leukocyte adhesion. Pharmacological sGC stimulation increased platelet angiopoietin-1 release <italic>in vitro</italic> and reduced recruitment of adoptively transferred leukocytes in <italic>Ldlr</italic><sup>-/-</sup> mice fed a Western diet. Pharmacological sGC stimulation further reduced atherosclerotic plaque formation and vascular inflammation.</p></sec><sec><title>Conclusion</title><p>Loss of sGC in platelets contributes to atherosclerotic plaque formation via reduced release of the soluble factor angiopoietin-1 and, subsequently, enhanced leukocyte recruitment. Pharmacological sGC stimulation might represent a novel therapeutic strategy to prevent and treat CAD.</p></sec><sec><title>Translational perspective</title><p>Reduced platelet soluble guanylyl cyclase activity contributes to atherosclerotic plaque formation and vascular inflammation. Stimulators of the soluble guanylyl cyclase, an emerging class of drugs already used in pulmonary hypertension and heart failure, are able to reduce atherosclerosis and inflammation in this preclinical model. Together with evidence from human genetics, our findings suggest a promising role of soluble guanylyl cyclase stimulation to prevent coronary artery disease.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468679', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Essential functions of MLL1 and MLL2 in retinal development and cone cell maintenance', 'authors': \"[{'name': 'Chi Sun'}, {'name': 'Xiaodong Zhang'}, {'name': 'Philip A. Ruzycki'}, {'name': 'Shiming Chen'}]\", 'abstract': '<p>MLL1 (KMT2A) and MLL2 (KMT2B) are homologous members of the mixed-lineage leukemia (MLL) family of histone methyltransferases involved in epigenomic transcriptional regulation. Their sequence variants have been associated with neurological and psychological disorders, but little is known about their roles and mechanism of action in CNS development. Using mouse retina as a model, we previously reported MLL1’s role in retinal neurogenesis and horizontal cell maintenance. Here we determine roles of MLL2 and MLL1/MLL2 together in retinal development using conditional knockout (CKO) mice. Deleting <italic>Mll2</italic> from Chx10<italic>+</italic> retinal progenitors resulted in a similar phenotype as <italic>Mll1 CKO</italic>, but removal of both alleles produced much more severe deficits than each single <italic>CKO</italic>: 1-month <italic>double CKO</italic> mutants displayed null light responses in electroretinogram; thin retinal layers, including shorter photoreceptor outer segments with impaired phototransduction gene expression; and reduced numbers of M-cones, horizontal and amacrine neurons, followed by fast retinal degeneration. Despite moderately reduced progenitor cell proliferation at P0, the neurogenic capacity was largely maintained in <italic>double CKO</italic> mutants. However, upregulated apoptosis and reactive gliosis were detected during postnatal retinal development. Finally, the removal of both MLLs in fated rods produced a normal phenotype, but the CKO in M-cones impaired M-cone function and survival, indicating both cell non-autonomous and autonomous mechanisms. Altogether, our results suggest that MLL1/MLL2 play redundant roles in maintaining specific retinal neurons after cell fate specification and are essential for establishing functional neural networks.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468678', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'A cell wall-associated gene network shapes leaf boundary domains', 'authors': \"[{'name': 'Nathalie Bouré'}, {'name': 'Alexis Peaucelle'}, {'name': 'Magali Goussot'}, {'name': 'Bernard Adroher'}, {'name': 'Ludivine Soubigou-Taconnat'}, {'name': 'Eric Biot'}, {'name': 'Zakia Tariq'}, {'name': 'Marie-Laure Martin-Magniette'}, {'name': 'Patrick Laufs'}, {'name': 'Nicolas Arnaud'}]\", 'abstract': '<p>Boundary domains delimit and organize organ growth throughout plant development almost relentlessly building plant architecture and morphogenesis. Boundary domains display reduced growth and orchestrate development of adjacent tissues in a non-cell autonomous manner. How these two functions are achieved remains elusive despite the identification of several boundary-specific genes. Here, we show using morphometrics at the organ and cellular levels that leaf boundary domain development requires SPINDLY (SPY), an O-fucosyltransferase, to act as cell growth repressor. Further we show that SPY acts redundantly with the CUP-SHAPED COTYLEDON transcription factors (CUC2 and CUC3), which are major determinants of boundaries development. Accordingly at the molecular level, CUC2 and SPY repress a common set of genes involved in cell wall loosening providing a molecular framework for the growth repression associated with boundary domains. Atomic force microscopy (AFM) confirmed that young leaf boundary domain cells have stiffer cell walls than marginal outgrowth. This differential cell wall stiffness was reduced in <italic>spy</italic> mutant. Taken together our data reveal a concealed <italic>CUC2</italic> cell wall associated gene network linking tissue patterning with cell growth and mechanics.</p><sec><title>Summary statement</title><p>Decreased cell-wall loosening gene expression contributes to the coordination of cell growth and mechanics with tissue patterning thus driving boundary development.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.16.468742', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Tunable Transcription Factor Library for Robust Quantification of Gene Expression Dynamics in E. coli', 'authors': \"[{'name': 'Vinuselvi Parisutham'}, {'name': 'Shivani Chhabra'}, {'name': 'Md Zulfikar Ali'}, {'name': 'Robert C. Brewster'}]\", 'abstract': '<p>Predicting the quantitative regulatory function of a TF based on factors such as binding sequence, binding location and promoter type is not possible. The interconnected nature of gene networks and the difficulty in tuning individual TF concentrations makes the isolated study of TF function challenging. Here we present a library of <italic>E. coli</italic> strains designed to allow for precise control of the concentration of individual TFs enabling the study of the role of TF concentration on physiology and regulation. We demonstrate the usefulness of this resource by measuring the regulatory function of the zinc responsive TF, ZntR and the paralogous TF pair, GalR/GalS. For ZntR, we find that zinc alters ZntR regulatory function in a way that enables activation of the regulated gene to be robust with respect to ZntR concentration. For GalR and GalS, we are able to demonstrate that these parlogous TFs have fundamentally distinct regulatory roles beyond differences in binding affinity.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468966', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'A new lineage of non-photosynthetic green algae with extreme organellar genomes', 'authors': \"[{'name': 'Tomáš Pánek'}, {'name': 'Dovilė Barcytė'}, {'name': 'Sebastian C. Treitli'}, {'name': 'Kristína Záhonová'}, {'name': 'Martin Sokol'}, {'name': 'Tereza Ševčíková'}, {'name': 'Eliška Zadrobílková'}, {'name': 'Karin Jaške'}, {'name': 'Naoji Yubuki'}, {'name': 'Ivan Čepička'}, {'name': 'Marek Eliáš'}]\", 'abstract': '<p>Background: The plastid genomes of the green algal order Chlamydomonadales tend to expand their non-coding regions, but this phenomenon is poorly understood. Here we shed new light on organellar genome evolution in Chlamydomonadales by studying a previously unknown non-photosynthetic lineage. We established cultures of two new Polytoma-like flagellates, defined their basic characteristics and phylogenetic position, and obtained complete organellar genome sequences and a transcriptome assembly for one of them.\\nResults: We discovered a novel deeply diverged chlamydomonadalean lineage that has no close photosynthetic relatives and represents an independent case of photosynthesis loss. To accommodate these organisms, we establish a new genus, Leontynka, with two species L. pallida and L. elongata distinguished by both morphological and molecular characteristics. Notable features of the colourless plastid of L. pallida deduced from the plastid genome (plastome) sequence and transcriptome assembly include the retention of ATP synthase, thylakoid-associated proteins, carotenoid biosynthesis pathway, and plastoquinone-based electron transport chain, the latter two modules having an obvious functional link to the eyespot present in Leontynka. Most strikingly, the L. pallida plastome with its ~362 kbp is by far the largest among non-photosynthetic eukaryotes investigated to date. Instead of a high gene content, its size reflects extreme proliferation of sequence repeats. These are present also in coding sequences, with one repeat type found in exons of 11 out of 34 protein-coding genes and up to 36 copies per gene, affecting thus the encoded proteins. The mitochondrial genome of L. pallida is likewise exceptionally large, with its &gt;104 kbp surpassed only by the mitogenome of Haematococcus lacustris among all members of Chlamydomonadales studied so far. It is also bloated with repeats, yet completely different from those in the L. pallida plastome, which contrasts with the situation in H. lacustris where both organellar genomes have accumulated related repeats. Furthermore, the L. pallida mitogenome exhibits an extremely high GC content in both coding and non-coding regions and, strikingly, a high number of predicted G-quadruplexes.\\nConclusions: With the unprecedented combination of plastid and mitochondrial genome characteristics, Leontynka pushes the frontiers of organellar genome diversity and becomes an interesting model for studying organellar genome evolution.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.469035', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The influence of somatosensory and auditory evoked potentials on concurrent transcranial-magnetic stimulation electroencephalography recordings', 'authors': \"[{'name': 'Nahian S Chowdhury'}, {'name': 'Nigel C Rogasch'}, {'name': 'Alan Chiang'}, {'name': 'Samantha K Millard'}, {'name': 'Patrick Skippen'}, {'name': 'Wei-Ju Chang'}, {'name': 'Katarzyna Bilska'}, {'name': 'Emily Si'}, {'name': 'David A Seminowicz'}, {'name': 'Siobhan M Schabrun'}]\", 'abstract': '<p>Background: Transcranial magnetic stimulation (TMS) evoked potentials (TEPs) can be used to index cortical excitability. However, it remains unclear to what extent TEPs reflect somatosensory and auditory-evoked potentials which arise from the scalp sensation and click of the TMS coil, as opposed to transcranial stimulation of cortical circuits. Objectives: The present study had two aims; a) to determine the extent to which sensory potentials contaminate TEPs using a spatially matched sham condition, and b) to determine whether sensory potentials reflect auditory or somatosensory potentials alone, or a combination of the two. Methods: Twenty healthy participants received active or sham stimulation, with the latter consisting of the click of a sham coil combined with scalp electrical stimulation. Earplugs/headphones were used to suppress the TMS click noise. Two additional control conditions i) electrical stimulation alone and ii) auditory stimulation alone were included in a subset of 13 participants. Results: Signals from active and sham stimulation were correlated in spatial and temporal domains, especially &gt;70ms post-stimulation. Relative to auditory or electrical stimulation alone, combined (sham) stimulation resulted in a) larger evoked responses b) stronger correlations with active stimulation and c) a signal that could not be explained by the linear sum of electrical and auditory stimulation alone. Conclusions: Sensory potentials can confound data interpretations of TEPs at timepoints &gt;70ms post-TMS, while earlier timepoints appear reflective of cortical excitability. Furthermore, contamination of TEPs cannot be explained by auditory or somatosensory potentials alone, but instead reflects a non-linear interaction between both sources. Future studies may benefit from controlling for sensory contamination using sham conditions that are spatially matched to active TMS, and which consist of combined auditory and somatosensory stimulation.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.468986', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Morpholino-mediated knockdown of the brain mineralocorticoid receptor affects glucocorticoid signaling and neuroplasticity in wild ocellated wrasse (Symphodus ocellatus)', 'authors': \"[{'name': 'Bridget M. Nugent'}, {'name': 'Kelly A. Stiver'}, {'name': 'Jiawei Han'}, {'name': 'Holly K Kindsvater'}, {'name': 'Susan E. Marsh-Rollo'}, {'name': 'Hans A. Hofmann'}, {'name': 'Suzanne H. Alonzo'}]\", 'abstract': '<p>Uncovering the genetic, physiological, and developmental mechanisms underlying phenotypic variation is necessary for understanding how genetic and genomic variation shape phenotypic variation and for discovering possible targets of selection. Although the neural and endocrine mechanisms underlying social behavior are evolutionarily ancient, we lack an understanding of the proximate causes and evolutionary consequences of variation in these mechanisms. Here, we examine in the natural environment the behavioral, neuromolecular, and fitness consequences of a morpholino-mediated knockdown of the mineralocorticoid receptor (MR) in the brain of nesting males of the ocellated wrasse, Symphodus ocellatus, a species with male alternative reproductive tactics. Even though MR knockdown did not significantly change male behavior directly, this experimental manipulation strongly altered glucocorticoid signaling and neuroplasticity in the preoptic area, the putative hippocampus homolog, and the putative basolateral amygdala homolog. We also found that individual variation in stress axis gene expression and neuroplasticity is strongly associated with variation in male behavior and fitness-related traits. The brain region-specific effects of MR knockdown on phenotypic integration in the wild reported here suggest specific neuroendocrine and neuroplasticity pathways that may be targets of selection.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.21266456', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': False, 'title': 'Interventions in sexual and reproductive health services addressing violence against women in low- and middle- income countries: a mixed-methods systematic review', 'authors': \"[{'name': 'Natalia V Lewis'}, {'name': 'Muzrif Munas'}, {'name': 'Manuela Colombini'}, {'name': 'Ana Flavia d’Oliveira'}, {'name': 'Stephanie Pereira'}, {'name': 'Satya Shrestha'}, {'name': 'Thilini Rajapakse'}, {'name': 'Amira Shaheen'}, {'name': 'Poonam Rishal'}, {'name': 'Abdulsalam Alkaiyat'}, {'name': 'Alison Richards'}, {'name': 'Claudia M García-Moreno'}, {'name': 'Gene S Feder'}, {'name': 'Loraine J Bacchus'}]\", 'abstract': '<sec><title>Objectives</title><p>To synthesise evidence on the effectiveness, cost-effectiveness, and barriers to responding to violence against women (VAW) in sexual and reproductive health (SRH) services in low- and middle-income countries (LMICs).</p></sec><sec><title>Design</title><p>Mixed-methods systematic review.</p></sec><sec><title>Data sources</title><p>Medline, Embase, Psycinfo, Cochrane, Cinahl, IMEMR, Web of Science, Popline, Lilacs, WHO RHL, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://ClinicalTrials.gov\">ClinicalTrials.gov</ext-link>, Google, Google Scholar, websites of key organisations through December 2019.</p></sec><sec><title>Eligibility criteria</title><p>Studies of any design that evaluated VAW interventions in SRH services in LMICs.</p></sec><sec><title>Data extraction and synthesis</title><p>Concurrent narrative quantitative and thematic qualitative syntheses, integration through line of argument and mapping onto a logic model. Two reviewers extracted data and appraised quality.</p></sec><sec><title>Results</title><p>26 studies of varied interventions using heterogeneous outcomes. Of ten interventions that strengthened health systems capacity to respond to VAW during routine SRH consultation, three reported no harm and reduction in some types of violence. Of nine interventions that strengthened health systems and communities’ capacity to respond to VAW, three reported conflicting effects on re-exposure to some types of VAW and mixed effect on SRH. The interventions increased identification of VAW but had no effect on the provision (75-100%) and uptake (0.6-53%) of referrals to VAW services. Of seven psychosocial interventions in addition to SRH consultation that strengthened women’s readiness to address VAW, four reduced re-exposure to some types of VAW and improved health. Factors that disrupted the pathway to better outcomes included accepting attitudes towards VAW, fear of consequences, and limited readiness of the society, health systems, and individuals. No study evaluated cost-effectiveness.</p></sec><sec><title>Conclusions</title><p>Some VAW interventions in SRH services reduced re-exposure to some types of VAW and improved some health outcomes in single studies. Future interventions should strengthen capacity to address VAW across health systems, communities, and individual women. First-line support should be better tailored to women’s needs and expectations.</p><p>PROSPERO protocol CRD42019137167.</p></sec>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.18.468847', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Common variation in EDN1 regulatory regions highlights the role of PPARγ as a key regulator of Endothelin in vitro', 'authors': \"[{'name': 'Mauro Lago-Docampo'}, {'name': 'Carlos Solarat'}, {'name': 'Luis Mendez-Martinez'}, {'name': 'Adolfo Baloira'}, {'name': 'Diana Valverde'}]\", 'abstract': \"<p>Pulmonary Arterial Hypertension (PAH) is a rare disease caused by the obliteration of the pulmonary arterioles, increasing pulmonary vascular resistance and eventually causing right heart failure. Endothelin-1 is a vasoconstrictor peptide whose levels are indicators of disease progression and its pathway is one of the commonest targeted by current treatments.\\nWe sequenced the EDN1 untranslated regions of a small subset of PAH patients, predicted the effect in silico, and used a luciferase assay with the different genotypes to analyze its influence on gene expression. Finally, we used siRNAs against the major transcription factors predicted for these regions (PPARγ, KLF4, and VDR) to assess Endothelin-1 expression in cell culture and validate the binding sites.\\nFirst, we detected a SNP in the 5'UTR (rs397751713) and another in the 3'regulatory region (rs2859338) that increased luciferase activity in vitro depending on their genotype. We determined in silico that KLF4/PPARγ could be binding in the rs397751713 and VDR in rs2859338. By using siRNAs and luciferase, we determined that PPARγ binds differentially in rs397751713. PPARγ and VDR Knock-Down, increased EDN1 mRNA levels and Endothelin-1 production in PAECs, while PPARγ and KLF4 Knock-Down increased the Endothelin-1 production in HeLa. \\nIn conclusion, common variants in EDN1 regulatory regions could alter Endothelin-1 levels. We were able to validate that PPARγ binds in rs397751713 and is key to regulate Endothelin-1. Also, KLF4 and VDR regulate Endothelin-1 production in a cell-dependent manner, but for VDR this interaction does not happen by binding directly to the regions we studied.</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.20.469378', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Organization and dynamics of the SpoVAEa protein, and its surrounding inner membrane lipids upon germination of Bacillus subtilis spores', 'authors': \"[{'name': 'Juan Wen'}, {'name': 'Norbert O.E. Vischer'}, {'name': 'Arend D Vos'}, {'name': 'Erik M.M. Manders'}, {'name': 'Peter Setlow'}, {'name': 'Stanley Brul'}]\", 'abstract': \"<p>The SpoVA proteins make up a channel in the inner membrane (IM) of B. subtilis spore. This channel responds to signals from activated germinant receptors (GRs), and allows release of Ca2+-DPA from the spore core during germination. In the current work, we studied the location and dynamics of SpoVAEa in dormant spores. Notably, the SpoVAEa-SGFP2 proteins were present in a single spot in spores, similar to the complex formed by all GRs. However, while the GR's spot remains in one location, the SpoVAEa-SGFP2 spot in the IM moved randomly with high frequency. The dynamics of the SpoVAEa-SGFP2 and its surrounding IM region as stained by fluorescent dyes were also tracked during spore germination, as the dormant spore IM appeared to have an immobile germination related functional microdomain. This microdomain disappeared around the time of appearance of a germinated spore, the loss of fluorescence of the IM by fluorescent dyes, as well as the appearance of SpoVAEa-SGFP2 peak fluorescent intensity occurred in parallel. These observed events were highly related to the rapid phase darkening, which is considered as the Ca2+DPA rapid release. We also tested the response of SpoVAEa and the IM to thermal treatments at 40-80 degrees Celcius. Heat treatment triggered an increase of green autofluorescence, which is speculated to be due to coat protein denaturation, and 80 degrees Celcius treatments induce the appearance of phase-grey-like spores. These spores presumably have a similar intracellular physical state as the phase grey spores detected in the germination but lack the functional proteins for further germination events.</p>\", 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.17.469050', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Guide-directed DNA cleavage by a prokaryotic Argonaute protein induces chromosome recombination in Escherichia coli', 'authors': \"[{'name': 'Shan Huang'}, {'name': 'Kaihang Wang'}, {'name': 'Stephen L. Mayo'}]\", 'abstract': '<p>Emerging evidence supports the argument that some prokaryotic argonautes (pAgos) serve as a defensive system against invasion of viruses and plasmids through guide DNAs (gDNAs) directed DNA cleavage. This DNA-guided DNA interference motivates research to induce genomic mutations via pAgo mediated cleavage. Here we demonstrate that CbAgo, a pAgo from <italic>Clostridium butyricum</italic>, is able to induce chromosomal recombination between direct repeat sequences via its gDNA-directed cleavage in <italic>Escherichia coli</italic> chromosome. We also show that CbAgo targeting can assist Lambda-Red recombineering in RecA-deficient strain. Our study reveals that cleavage by CbAgo in <italic>E. coli</italic> chromosome can be mutagenic and suggests its broader application in genetic manipulation.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.21.469381', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Stress deceleration theory: chronic adolescent stress exposure results in decelerated neurobehavioral maturation', 'authors': \"[{'name': 'Kshitij Jadhav'}, {'name': 'Alexander Hoffman'}, {'name': 'Morgane Burgisser'}, {'name': 'Clara Rosetti'}, {'name': 'Lea Aeschlimann'}, {'name': 'Carl R Lupica'}, {'name': 'Benjamin Boutrel'}]\", 'abstract': '<p>Normative development in adolescence indicates that the prefrontal cortex is still under development thereby unable to exert efficient top-down inhibitory control on subcortical regions such as the basolateral amygdala and the nucleus accumbens. This imbalance in the developmental trajectory between cortical and subcortical regions is implicated in expression of the prototypical impulsive, compulsive, reward seeking and risk-taking adolescent behavior. Here we demonstrate that a chronic mild unpredictable stress procedure during adolescence in male Wistar rats arrests the normal behavioral maturation such that they continue to express adolescent-like impulsive, hyperactive, and compulsive behaviors into late adulthood. This arrest in behavioral maturation is associated with the hypoexcitability of prelimbic cortex (PLC) pyramidal neurons and reduced PLC-mediated synaptic glutamatergic control of BLA and nucleus accumbens core (NAcC) neurons that lasts late into adulthood. At the same time stress exposure in adolescence results in the hyperexcitability of the BLA pyramidal neurons sending stronger glutamatergic projections to the NAcC. Chemogenetic reversal of the PLC hypoexcitability decreased compulsivity and improved the expression of goal-directed behavior in rats exposed to stress during adolescence, suggesting a causal role for PLC hypoexcitability in this stress-induced arrested behavioral development.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.16.468913', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genome-wide identification of novel sRNAs in Streptococcus mutans', 'authors': \"[{'name': 'Madeline C Krieger'}, {'name': 'Justin Merritt'}, {'name': 'Rahul Raghavan'}]\", 'abstract': '<p><italic>Streptococcus mutans</italic> is a major pathobiont involved in the development of dental caries. Its ability to utilize numerous sugars and to effectively respond to environmental stress promotes <italic>S. mutans</italic> proliferation in oral biofilms. Because of their quick action and low energetic cost, non-coding small RNAs (sRNAs) represent an ideal mode of gene regulation in stress response networks, yet their roles in oral pathogens have remained largely unexplored. We identified 15 novel sRNAs in <italic>S. mutans</italic> and show that they respond to four stress-inducing conditions commonly encountered by the pathogen in human mouth: sugar-phosphate stress, hydrogen peroxide exposure, high temperature, and low pH. To better understand the role of sRNAs in <italic>S. mutans</italic>, we further explored the function of the novel sRNA, SmsR4. Our data demonstrate that SmsR4 regulates the EIIA component of the sorbitol phosphotransferase system, which transports and phosphorylates the sugar alcohol sorbitol. The fine-tuning of EIIA availability by SmsR4 likely promotes <italic>S. mutans</italic> growth while using sorbitol as the main carbon source. Our work lays a foundation for understanding the role of sRNAs in regulating gene expression in stress response networks in <italic>S. mutans</italic> and highlights the importance of the underexplored phenomenon of posttranscriptional gene regulation in oral bacteria.</p><sec><title>IMPORTANCE</title><p>Small RNAs (sRNAs) are important gene regulators in bacteria, but the identities and functions of sRNAs in <italic>Streptococcus mutans</italic>, the principal bacterium involved in the formation of dental caries, are unknown. In this study, we identified 15 putative sRNAs in <italic>S. mutans</italic> and show that they respond to four common stress-inducing conditions present in human mouth: sugar-phosphate stress, hydrogen peroxide exposure, high temperature, and low pH. We further show that the novel sRNA SmsR4 likely modulates sorbitol transport into the cell by regulating SMU_313 mRNA, which encodes the EIIA subunit of the sorbitol phosphotransferase system. Gaining a better understanding of sRNA-based gene regulation may provide new opportunities to develop specific inhibitors of <italic>S. mutans</italic> growth, thereby improving oral health.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.18.467517', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Semi-supervised single-cell cross-modality translation using Polarbear', 'authors': \"[{'name': 'Ran Zhang'}, {'name': 'Laetitia Meng-Papaxanthos'}, {'name': 'Jean-Philippe Vert'}, {'name': 'William Stafford Noble'}]\", 'abstract': '<p>The emergence of single-cell co-assays enables us to learn to translate between single-cell modalities, potentially offering valuable insights from datasets where only one modality is available. However, the sparsity of single-cell measurements and the limited number of cells measured in typical co-assay datasets impedes the power of cross-modality translation. Here, we propose Polarbear, a semi-supervised translation framework to predict cross-modality profiles that is trained using a combination of co-assay data and traditional \"single-assay\" data. Polarbear uses single-assay and co-assay data to train an autoencoder for each modality and then uses just the co-assay data to train a translator between the embedded representations learned by the autoencoders. With this approach, Polarbear is able to translate between modalities with improved accuracy relative to state-of-the-art translation techniques. As an added benefit of the training procedure, we show that Polarbear also produces a matching of cells across modalities.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.467977', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': \"Towards a catalog of pome tree architecture genes: the draft 'd'Anjou' genome (Pyrus communis L.)\", 'authors': \"[{'name': 'Huiting Zhang'}, {'name': 'Eric K Wafula'}, {'name': 'Jon Eilers'}, {'name': 'Alex Harkess'}, {'name': 'Paula E Ralph'}, {'name': 'Prakash Raj Timilsena'}, {'name': 'Claude dePamphilis'}, {'name': 'Jessica Waite'}, {'name': 'Loren A Honaas'}]\", 'abstract': \"<p>The rapid development of sequencing technologies has led to a deeper understanding of horticultural plant genomes. However, experimental evidence connecting genes to important agronomic traits is still lacking in most non-model organisms. For instance, the genetic mechanisms underlying plant architecture are poorly understood in pome fruit trees, creating a major hurdle in developing new cultivars with desirable architecture, such as dwarfing rootstocks in European pear (Pyrus communis). Further, the quality and content of genomes vary widely. Therefore, it can be challenging to curate a list of genes with high-confidence gene models across reference genomes. This is often an important first step towards identifying key genetic factors for important traits. Here we present a draft genome of P. communis 'd'Anjou' and an improved assembly of the latest P. communis 'Bartlett' genome. To study gene families involved in tree architecture in European pear and other rosaceous species, we developed a workflow using a collection of bioinformatic tools towards curation of gene families of interest across genomes. This lays the groundwork for future functional studies in pear tree architecture. Importantly, our workflow can be easily adopted for other plant genomes and gene families of interest.</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.468631', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Paleontology', 'covid': False, 'title': 'High diversity of pimpline parasitoid wasps (Hymenoptera, Ichneumonidae, Pimplinae) from the lowermost Eocene Fur Formation', 'authors': \"[{'name': 'Seraina Klopfstein'}]\", 'abstract': '<p>With an estimated 100,000 extant species, Darwin wasps (Ichneumonidae) are more specious than all vertebrates together. However, only 288 fossil species have been described to date, with hundreds more awaiting formal description in palaeontological collections. One of the largest gaps in our knowledge concerns the ~12 million years between the K-PG mass extinction and the late Ypresian, from which only two species have been formally described, including Pimpla stigmatica Henriksen from the Danish Fur Formation (~55 Ma). Ie here redescribe and reclassify this species in the genus Epitheronia Gupta, n. comb., and describe nine new species from this fossil locality that are consistent with a placement in Pimplinae: Crusopimpla collina n. sp., C. elongata n. sp., C. minuta n. sp., C. rettigi n. sp., C. violina n. sp., Theronia? furensis n. sp.,  T. nigriscutum n. sp., Xanthopimpla ciboisae n. sp., and X. crescendae n. sp. The diagnosis of the genus Crusopimpla Kopylov, Spasojevic &amp; Klopfstein is amend in the light of the new species. By comparing the preserved colouration between and within specimens of different species, we draw conclusions about the taxonomic usefulness of colour patterns observed in Fur Formation ichneumonids. The number of described species of pimpline parasitoid wasps from Fur is very high when compared to any other fossil deposit, but low with respect to numbers of extant species. Further study and excavation of Fur ichneumonids will certainly reveal an even higher diversity.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.469021', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Hidden temporal structure of the ongoing task impacts detection strategy and is reflected in pupillary dynamics', 'authors': \"[{'name': 'Jennifer Lawlor'}, {'name': 'Agnes Zagala'}, {'name': 'Sara Jamali'}, {'name': 'Yves Boubenec'}]\", 'abstract': \"<p>Estimating temporal regularities in incoming sensory inputs supports optimal decisions in noisy environments. In particular, inferred temporal structure can ease the detection of likely target events. Here we postulated that timely urgency signals can adapt subjects' decision-making to the ongoing task temporal structure, possibly through neuromodulatory tone. To test this hypothesis, we used an auditory change detection task in which targets followed a block-based temporal contingency, unbeknownst to participants. False alarm occurrences were driven by the distribution of target timings, indicating that participants adapted their behavior to the ongoing temporal structure. Task-evoked pupillary responses were larger for blocks with earliest target timings, and correlated with individual subjects' behavioral adaptation. Individual pupil responses matched an urgency signal extracted from a decision model fitted to behavior. This work demonstrates that internal temporal expectation can be tracked through pupillary dynamics, suggesting a role of neuromodulatory systems in context-dependent modulation of decision variable dynamics.</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468917', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Resistance profiling of Aspergillus fumigatus to olorofim indicates absence of intrinsic resistance and unveils the molecular mechanisms of acquired olorofim resistance', 'authors': \"[{'name': 'Jochem B. Buil'}, {'name': 'Jason D. Oliver'}, {'name': 'Derek Law'}, {'name': 'Tim Baltussen'}, {'name': 'Jan Zoll'}, {'name': 'Margriet W.J. Hokken'}, {'name': 'Marlou Tehupeiory-Kooreman'}, {'name': 'Willem J.G. Melchers'}, {'name': 'Mike Birch'}, {'name': 'Paul E. Verweij'}]\", 'abstract': '<p>Olorofim (F901318) is a new antifungal currently under clinical development that shows both in vitro and in vivo activity against a number of filamentous fungi including <italic>Aspergillus fumigatus</italic>. In this study we screened <italic>A. fumigatus</italic> isolates for intrinsic olorofim-resistant <italic>A. fumigatus</italic> and evaluated the ability of <italic>A. fumigatus</italic> to acquire an olorofim-resistant phenotype. No intrinsic resistance was found in 975 clinical <italic>A. fumigatus</italic> isolates. However, we found that isolates with increased olorofim MICs (&gt; 8 mg/L) could be selected using a high number of conidia and olorofim exposure under laboratory conditions. Assessment of the frequency of acquired olorofim resistance development of <italic>A. fumigatus</italic> was shown to be higher than for voriconazole but lower than for itraconazole. Sequencing the <italic>PyrE</italic> gene of isogenic isolates with olorofim MICs of &gt;8 mg/L identified various amino acid substitutions with a hotspot at locus G119. Olorofim was shown to have reduced affinity to mutated target protein dihydroorotate dehydrogenase (DHODH) and the effect of these mutations were proven by introducing the mutations directly in <italic>A. fumigatus</italic>. We then investigated whether G119 mutations were associated with a fitness cost in <italic>A. fumigatus.</italic> These experiments showed a small but significant reduction in growth rate for strains with a G119V substitution, while strains with a G119C substitution did not exhibit a reduction in growth rate. These in vitro findings were confirmed in an in vivo pathogenicity model.</p><sec><title>Importance</title><p>Olorofim represents an important new treatment option for patients with difficult to treat invasive fungal infections, including triazole-resistant <italic>A. fumigatus</italic> infection. Our study provides insights into one resistance mechanism and the potential dynamics of olorofim resistance, which will help to prevent and manage resistance selection. Such insights are critical to antifungal stewardship and to safeguard its prolonged use in clinical practice.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.21.469458', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Zinc finger protein Zfp335 controls early T cell development and survival through beta-selection-dependent and -independent mechanisms', 'authors': \"[{'name': 'Xin Wang'}, {'name': 'Anjun Jiao'}, {'name': 'Yanhong Su'}, {'name': 'Wenhua Li'}, {'name': 'Biao Yang'}, {'name': 'Renyi Ding'}, {'name': 'Cangang Zhang'}, {'name': 'Haiyan Liu'}, {'name': 'Xiaofeng Yang'}, {'name': 'Chenming Sun'}, {'name': 'Lina Sun'}, {'name': 'Baojun Zhang'}]\", 'abstract': '<p>T cell development in the thymus undergoes the process of differentiation, selective proliferation and survival from CD4<sup>-</sup>CD8<sup>-</sup> double negative (DN) stage to CD4<sup>+</sup>CD8<sup>+</sup> double positive (DP) stage prior to the formation of CD4<sup>+</sup> helper and CD8<sup>+</sup> cytolytic T cells ready for circulation. Each developmental stage is tightly regulated by sequentially-operating molecular networks, of which only limited numbers of transcription regulators have been deciphered. Here we identified Zfp335 transcription factor as a new player in the regulatory network controlling thymocyte development. We demonstrate that Zfp335 intrinsically controls DN to DP transition, as T cell-specific deficiency in Zfp335 leads to a substantial accumulation of DN3 along with reduction of DP, CD4<sup>+</sup> and CD8<sup>+</sup> thymocytes. This developmental blockade at DN stage results from the impaired intracellular TCRβ expression as well as increased susceptibility to apoptosis in thymocytes. Transcriptomic and ChIP-seq analyses revealed a direct regulation of transcription factors Bcl6 and Rorc by Zfp335. Importantly, enhanced expression of TCRβ and Bcl6/RorγT restores the developmental defect during DN3 to DN4 transition and improves thymocytes survival, respectively. These findings identify a critical role of Zfp335 in controlling T cell development by maintaining intracellular TCRβ expression-mediated β-selection and independently activating cell survival signaling.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.17.21266481', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Effects of trust, risk perception, and health behavior on COVID-19 disease burden: Evidence from a multi-state US survey', 'authors': \"[{'name': 'Benjamin J. Ridenhour'}, {'name': 'Dilshani Sarathchandra'}, {'name': 'Erich Seamon'}, {'name': 'Helen Brown'}, {'name': 'Fok-Yan Leung'}, {'name': 'Maureen Johnson-Leon'}, {'name': 'Mohamed Megheib'}, {'name': 'Craig R. Miller'}, {'name': 'Jennifer Johnson-Leung'}]\", 'abstract': '<p>Early public health strategies to prevent the spread of COVID-19 in the United States relied on non-pharmaceutical interventions (NPIs) as vaccines and therapeutic treatments were not yet available. Implementation of NPIs, primarily social distancing and mask wearing, varied widely between communities within the US due to variable government mandates, as well as differences in attitudes and opinions. To understand the interplay of trust, risk perception, behavioral intention, and disease burden, we developed a survey instrument to study attitudes concerning COVID-19 and pandemic behavioral change in three states: Idaho, Texas, and Vermont. We designed our survey (<italic>n</italic> = 1034) to detect whether these relationships were significantly different in rural populations. The best fitting structural equation models show that trust indirectly affects protective pandemic behaviors via health and economic risk perception. We explore two different variations of this social cognitive model: the first assumes behavioral intention affects future disease burden while the second assumes that observed disease burden affects behavioral intention. In our models we include several exogenous variables to control for demographic and geographic effects. Notably, political ideology is the only exogenous variable which significantly affects all aspects of the social cognitive model (trust, risk perception, and behavioral intention). While there is a direct negative effect associated with rurality on disease burden, likely due to the protective effect of low population density in the early pandemic waves, we found a marginally significant, positive, indirect effect of rurality on disease burden via decreased trust (<italic>p</italic> = 0.095). This trust deficit creates additional vulnerabilities to COVID-19 in rural communities which also have reduced healthcare capacity. Increasing trust by methods such as in-group messaging could potentially remove some of the disparities inferred by our models and increase NPI effectiveness.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468914', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Identification of shared bacterial strains in the vaginal microbiota of reproductive-age mothers and daughters using genome-resolved metagenomics', 'authors': \"[{'name': 'M. France'}, {'name': 'S. Brown'}, {'name': 'A. Rompalo'}, {'name': 'R. M. Brotman'}, {'name': 'J. Ravel'}]\", 'abstract': '<p>It has been suggested that the human microbiome might be vertically transmitted from mother to offspring and that early colonizers may play a critical role in development of the immune system. Studies have shown limited support for the vertical transmission of the intestinal microbiota but the derivation of the vaginal microbiota remains largely unknown. Although the vaginal microbiota of children and reproductive age cis women differ in composition, the vaginal microbiota could be vertically transmitted. To determine whether there was any support for this hypothesis, we examined the vaginal microbiota of daughter-mother pairs from the Baltimore metropolitan area (ages 14-27, 32-51; n=39). We assessed whether the daughter’s microbiota was similar in composition to their mother’s using metataxonomics. Permutation tests revealed that while some pairs did have similar vaginal microbiota, the degree of similarity did not exceed that expected by chance. Genome-resolved metagenomics was used to identify shared bacterial strains in a subset of the families (n=22). We found a small number of bacterial strains that were shared between mother-daughter pairs but identified more shared strains between individuals from different families, indicating that vaginal bacteria may display biogeographic patterns. Earlier-in-life studies are needed to demonstrate vertical transmission of the vaginal microbiota.</p><sec><title>Importance</title><p>Early colonizers of our microbiota are theorized to play an important role in the development of our immune system, yet we know little about how these communities are established. Vertical transmission from mother to offspring at the time of birth is theorized to be a major source of early colonizers but limited evidence supporting this process has only been shown for the intestinal tract microbiota. The provenance of the vaginal microbiota is largely unknown, although some have posited it is similarly vertically transmitted. We examined the vaginal microbiota of mother-daughter pairs and found limited evidence in support of this hypothesis. However, our analysis also revealed putative biogeographic patterns in the distribution of the strains which comprise the vaginal microbiota. Our results give insight into the role of vertical transmission for the vaginal microbiota and motivate future studies on the biogeography of these bacteria.</p></sec>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.18.469068', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Epigenetic machinery is functionally conserved in cephalopods', 'authors': \"[{'name': 'Filippo Macchi'}, {'name': 'Eric Edsinger'}, {'name': 'Kirsten C. Sadler'}]\", 'abstract': '<p>Epigenetic regulatory mechanisms are divergent across the animal kingdom, yet little is known about the epigenome in non-model organisms. Unique features of cephalopods make them attractive for investigating behavioral, sensory, developmental and regenerative processes, but using molecular approaches in such studies is hindered by the lack of knowledge about genome organization and gene regulation in these animals. We combined bioinformatic and molecular analysis of <italic>Octopus bimaculoides</italic> to identify gene expression signatures for 12 adult tissues and a hatchling, and investigate the presence and pattern of DNA methylation and histone methylation marks across tissues. This revealed a dynamic gene expression profile encoding several epigenetic regulators, including DNA methylation maintenance factors that were highly conserved and functional in cephalopods, as shown by detection of 5-methyl-cytosine in multiple tissues of octopus, squid and bobtail squid. WGBS of octopus brain and RRBS from a hatchling revealed that less than 10% of CpGs are methylated, highlighting a non-random distribution in the genome of all tissues, with enrichment in the bodies of a subset of 14,000 genes and absence from transposons. Each DNA methylation pattern encompassed genes with distinct functions and, strikingly, many of these genes showed similar expression levels across tissues. In contrast to the static pattern of DNA methylation, the histone marks H3K27me3, H3K9me3 and H3K4me3 were detected at different levels in diverse cephalopod tissues. This suggests the methylome and histone code cooperate to regulate tissue specific gene expression in a way that may be unique to cephalopods.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.14.468555', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Genome-wide H3K9 Acetylation Level Increases with Age-Dependent Senescence of Flag Leaf in Rice (Oryza sativa)', 'authors': \"[{'name': 'Yu Zhang'}, {'name': 'Yanyun Li'}, {'name': 'Yuanyuan Zhang'}, {'name': 'Zeyu Zhang'}, {'name': 'Deyu Zhang'}, {'name': 'Xiaonan Wang'}, {'name': 'Binfan Lai'}, {'name': 'Dandan Huang'}, {'name': 'Lianfeng Gu'}, {'name': 'Yakun Xie'}, {'name': 'Ying Miao'}]\", 'abstract': '<p>Flag leaf senescence is an important biological process that drives the remobilization of nutrients to the growing organs of rice. Leaf senescence is controlled by genetic information via gene expression and epigenetic modification, but the precise mechanism is as of yet unclear. Here, we analyzed genome-wide acetylated lysine residue 9 of histone H3 (H3K9ac) enrichment by chromatin immunoprecipitation-sequencing (ChIP-seq) and examined its association with transcriptomes by RNA-seq during flag leaf aging in rice (<italic>Oryza sativa</italic>). We found that genome-wide H3K9 acetylation levels increased with age-dependent senescence in rice flag leaf, and there was a positive correlation between the density and breadth of H3K9ac and gene expression and transcript elongation. A set of 1,249 up-regulated, differentially expressed genes (DEGs) and 996 down-regulated DEGs showing a strong relationship between temporal changes in gene expression and gain/loss of H3K9ac was observed during rice flag leaf aging. We produced a landscape of H3K9 acetylation-modified gene expression targets that includes known senescence-associated genes, metabolism-related genes, as well as miRNA biosynthesis-related genes. Our findings reveal a complex regulatory network of metabolism- and senescence-related pathways mediated by H3K9ac and also elucidate patterns of H3K9ac-mediated regulation of gene expression during flag leaf aging in rice.</p><sec><title>Significance statement</title><p>Genome-wide H3K9 acetylation levels increased with age-dependent senescence in rice flag leaf, and positively correlation the density and breadth of H3K9ac with transcript elongation and expression. Identified numerous H3K9 acetylation-modified gene expression targets reveal a complex regulatory network and metabolism-mediated senescence network that are associated with H3K9ac during leaf aging in rice.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.19.469324', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'The sperm protein Spaca6 is essential for fertilization in zebrafish', 'authors': \"[{'name': 'Mirjam Isabel Binner'}, {'name': 'Anna Kogan'}, {'name': 'Karin Panser'}, {'name': 'Alexander Schleiffer'}, {'name': 'Victoria Eugenia Deneke'}, {'name': 'Andrea Pauli'}]\", 'abstract': '<p>Fertilization is a key process in all sexually reproducing species, yet the molecular mechanisms that underlie this event remain unclear. To date, only a few proteins have been shown to be essential for sperm-egg binding and fusion in mice, and only some are conserved across vertebrates. One of these conserved, testis-expressed factors is SPACA6, yet its function has not been investigated outside of mammals. Here we show that zebrafish spaca6 encodes for a sperm membrane protein which is essential for fertilization. Zebrafish spaca6 knockout males are sterile. Furthermore, Spaca6-deficient sperm have normal morphology, are motile, and can approach the egg, but fail to bind to the egg and therefore cannot complete fertilization. Interestingly, sperm lacking Spaca6 have decreased levels of another essential and conserved sperm fertility factor, Dcst2, revealing a previously unknown dependence of Dcst2 expression on Spaca6. Together, our results show that zebrafish Spaca6 regulates Dcst2 levels and is required for binding between the sperm membrane and the oolemma. This is in contrast to murine SPACA6, which was reported not to be required for sperm-egg membrane binding but necessary for fusion. These findings demonstrate that Spaca6 is essential for zebrafish fertilization and is a conserved sperm factor in vertebrate reproduction.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.12.21266286', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'CAN A MACHINE LEARNING ALGORITHM IDENTIFY SARS-COV-2 VARIANTS BASED ON CONVENTIONAL rRT-PCR? PROOF OF CONCEPT', 'authors': \"[{'name': 'Jorge Cabrera Alvargonzález'}, {'name': 'Ana Larrañaga Janeiro'}, {'name': 'Sonia Pérez Castro'}, {'name': 'Javier Martínez Torres'}, {'name': 'Lucía Martínez Lamas'}, {'name': 'Carlos Daviña Nuñez'}, {'name': 'Víctor Del Campo-Pérez'}, {'name': 'Silvia Suarez Luque'}, {'name': 'Benito Regueiro García'}, {'name': 'Jacobo Porteiro Fresco'}]\", 'abstract': '<label>1</label><title>ABSTRACT</title><p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been and remains one of the major challenges humanity has faced thus far. Over the past few months, large amounts of information have been collected that are only now beginning to be assimilated. In the present work, the existence of residual information in the massive numbers of rRT-PCRs that tested positive out of the almost half a million tests that were performed during the pandemic is investigated. This residual information is believed to be highly related to a pattern in the number of cycles that are necessary to detect positive samples as such. Thus, a database of more than 20,000 positive samples was collected, and two supervised classification algorithms (a support vector machine and a neural network) were trained to temporally locate each sample based solely and exclusively on the number of cycles determined in the rRT-PCR of each individual. Finally, the results obtained from the classification show how the appearance of each wave is coincident with the surge of each of the variants present in the region of Galicia (Spain) during the development of the SARS-CoV-2 pandemic and clearly identified with the classification algorithm.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.21266338', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Comparative analysis of antibody production by mRNA 1273, AZD 1222 and BBIBP-CorV on elderly people suffering from different co-morbidity in Bangladesh', 'authors': \"[{'name': 'Ashraful Hoque'}, {'name': 'Md Marufur Rahman'}, {'name': 'Shahnewaz Parvez'}, {'name': 'Hossain Imam'}, {'name': 'Nurun Nahar'}, {'name': 'Forhad Uddin Hasan Chowdhury'}]\", 'abstract': '<sec><title>Background</title><p>As the pandemic spread so quickly all over the world, the scientist did not get the right time to cover all age group populations for a vaccine trial. The elderly population is usually vulnerable for COVID-19 which was proven by different research work and focus was to save this group of people was the prime concern for every country of the world. Though vaccines that got emergency use authorization have proven their efficacy which one is better for elderly people suffering from different co-morbid conditions is still not established. In this study, we want to evaluate the antibody production by mRNA 1273, AZD 1222, and BBIBP-CorV in elderly people with different comorbidity.</p></sec><sec><title>Method</title><p>We include 40 people in each group who have at least one comorbid condition and the total sample size was 120. The sample was taken from them before vaccination and 14 days after the second dose. Adverse event following immunization was recorded if any. Antibody measurement was done by ELISA method by using DiaSino SARS-CoV-2 S1 RBD IgG Quant.</p></sec><sec><title>Result</title><p>Among 120 participants with an equal number of participants in each of the vaccine groups all of them were aged between 60-72 years, of whom 65% were males and 35 % were females. Anti S1 RBD IgG was detected among all the participants from each vaccine group after 14 days of taking their 2nd dose. A non-parametric multiple comparison test (Kruskal-Wallis test) of Anti S1 RBD IgG levels among three vaccine groups revealed significant differences (P-value &lt;0.05) between groups. The IgG level was almost twice in the mRNA-1273 group (mean 577.1± 44.33 AU/ml) compared to AZD1222(mean 308.5 ±37.91 AU/ml) and BBIBP-CorV group.</p></sec><sec><title>Conclusion</title><p>From this small sample size, we predicate that mRNA 1273 produce much higher anti-S1 RBD IgG than the other two vaccines. Every vaccine is safe and effective whose is approved by WHO. Calculative use of the vaccine may produce an outcome for the future as we are still way behind in the proper amount of vaccine production.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.21266362', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': False, 'title': 'Memory perfectionism is associated with persistent memory complaints after concussion', 'authors': \"[{'name': 'Edwina L. Picon'}, {'name': 'Evgenia Todorova'}, {'name': 'Daniela J. Palombo'}, {'name': 'David L. Perez'}, {'name': 'Andrew Howard'}, {'name': 'Noah D. Silverberg'}]\", 'abstract': '<sec><title>Objective</title><p>The etiology of persistent memory complaints after concussion is poorly understood. Memory perfectionism (highly valuing memory ability and intolerance of minor memory lapses) may help explain why some people report persistent subjective memory problems in the absence of corresponding objective memory impairment. The present study investigated the relationship between memory perfectionism and persistent memory complaints after concussion.</p></sec><sec><title>Methods</title><p>Adults (N=77; 61% women) with persistent symptoms following concussion were recruited from outpatient specialty clinics. Participants completed the National Institutes of Health Toolbox Cognition Battery, Test of Memory Malingering-Trial 1, and questionnaires measuring memory perfectionism (Memory in Adulthood-Achievement subscale), forgetfulness and other post-concussion symptoms (Rivermead Postconcussion Symptoms Questionnaire; RPQ), and depression (Patient Health Questionnaire-2) at M=17.8 weeks post-injury. Patients with vs. without severe memory complaints (based on the RPQ) were compared.</p></sec><sec><title>Results</title><p>Memory perfectionism was associated with severe memory complaint, after controlling for objective memory ability, overall cognitive ability, and depression (95% confidence interval for odds ratio = 1.11 to 1.40). Sensitivity analyses showed that this relationship did not depend on use of specific objective memory tests nor on inclusion of participants who failed performance validity testing. In a control comparison to test the specificity of identified relationships, memory perfectionism was not associated with severe fatigue (95% confidence interval for odds ratio = 0.91 to 1.07).</p></sec><sec><title>Discussion</title><p>Memory perfectionism may predispose people to experience persistent memory symptoms and/or contribute to their perpetuation after concussion, with potential relevance to the spectrum of functional cognitive disorders more broadly.</p></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.18.469202', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Replication protein Rep provides selective advantage to viruses in the presence of CRISPR-Cas immunity', 'authors': \"[{'name': 'Weijia Zhang'}, {'name': 'Yuvaraj Bhoobalan-Chitty'}, {'name': 'Xichuan Zhai'}, {'name': 'Yan Hui'}, {'name': 'Lars Hestbjerg Hansen'}, {'name': 'Ling Deng'}, {'name': 'Xu Peng'}]\", 'abstract': '<p>Prokaryotic viruses express anti-CRISPR (Acr) proteins to inhibit the host adaptive immune system, CRISPR-Cas. While the virus infection biology was shown to be strongly dependent on the relative strengths of the host CRISPR-Cas and viral Acrs, little is known about the role of the core processes of viral life cycle (replication, packaging etc) in defence/anti-defence arms race. Here, we demonstrate the selective advantage provided by a replication initiator, Rep, in the context of CRISPR-Acr interactions. First, we developed a two-host based CRISPR-Cas genome editing tool for the deletion of highly conserved and thus potentially important viral genes. Using this strategy, we deleted a highly conserved Rep-coding gene, <italic>gp16</italic>, from the genome of <italic>Sulfolobus islandicus</italic> rod-shaped virus 2 (SIRV2). The knockout mutant (Δ<italic>gp16</italic>) produced around 4 fold less virus in a CRISPR-null host, suggesting that Rep is the major contributor to replication initiation in Rudiviridae. Indeed, DNA sequencing revealed Rep-dependent replication initiation from the viral genome termini, in addition to Rep-independent replication initiation from non-terminal sites. Intriguingly, the lack of Rep showed a profound effect on virus propagation in a host carrying CRISPR-Cas immunity. Accordingly, the co-infecting parental virus (<italic>rep</italic>-containing) outcompeted the Δ<italic>gp16</italic> mutant much more quickly in CRISPR-containing host than in CRISPR-null host, demonstrating a selective advantage provided by Rep in the presence of host CRISPR-Cas immunity. Despite the non-essentiality, <italic>rep</italic> is carried by all known members of Rudiviridae, which is likely an evolutionary outcome driven by the ubiquitous presence of CRISPR-Cas in Sulfolobales.</p><sec><title>Importance</title><p>CRISPR-Cas and anti-CRISPR proteins are accessary to prokaryotes and their viruses respectively. To date, research has been focused on their diversity, molecular mechanisms and application in genome editing. How CRISPR-Acr arms race influence the evolution of viral core genes involved in the basic virus life cycle remained a gap of knowledge so far. This study provides the first evidence that CRISPR-Acr arms race poses a selection pressure on the efficiency of viral genome replication, forcing viruses to evolve highly productive replication machineries..</p></sec>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468806', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Trajectory of rich club properties in structural brain networks', 'authors': \"[{'name': 'Levin Riedel'}, {'name': 'Martijn P van den Heuvel'}, {'name': 'Sebastian Markett'}]\", 'abstract': '<p>Many organizational principles of structural brain networks are established before birth and undergo considerable developmental changes afterwards. These include the topologically central hub regions and a densely connected rich club. While several studies have mapped developmental trajectories of brain connectivity and brain network organization across childhood and adolescence, comparatively little is known about subsequent development over the course of the lifespan. Here, we present a cross-sectional analysis of structural brain network development in N = 8,066 participants aged 5 to 80 years. Across all brain regions, structural connectivity strength followed an ′inverted-U′-shaped trajectory with vertex in the early 30s. Connectivity strength of hub regions showed a similar trajectory and the identity of hub regions remained stable across all age groups. While connectivity strength declined with advancing age, the organization of hub regions into a rich club did not only remain intact but became more pronounced, presumingly through a selected sparing of relevant connections from age-related connectivity loss. The stability of rich club organization in the face of overall age-related decline is consistent with a ′first come, last served′ model of neurodevelopment, where the first principles to develop are the last to decline with age. Rich club organization has been shown to be highly beneficial for communicability and higher cognition. A resilient rich club might thus be protective of a functional loss in late adulthood and represent a neural reserve to sustain cognitive functioning in the aging brain.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.467961', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Extending and using anatomical vocabularies in the Stimulating Peripheral Activity to Relieve Conditions (SPARC) project', 'authors': \"[{'name': 'Monique C. Surles - Zeigler'}, {'name': 'Troy Sincomb'}, {'name': 'Thomas H. Gillespie'}, {'name': 'Bernard de Bono'}, {'name': 'Jacqueline Bresnahan'}, {'name': 'Gary M. Mawe'}, {'name': 'Jeffrey S. Grethe'}, {'name': 'Susan Tappan'}, {'name': 'Maci Heal'}, {'name': 'Maryann E. Martone'}]\", 'abstract': '<p>The Stimulating Peripheral Activity to Relieve Conditions (SPARC) program is a US National Institutes of Health-funded effort to improve our understanding of the neural circuitry of the autonomic nervous system in support of bioelectronic medicine. As part of this effort, the SPARC project is generating multi-species, multimodal data, models, simulations, and anatomical maps supported by a comprehensive knowledge base of autonomic circuitry. To facilitate the organization of and integration across multi-faceted SPARC data and models, SPARC is implementing the FAIR data principles to ensure that all SPARC products are findable, accessible, interoperable, and reusable. We are therefore annotating and describing all products with a common FAIR vocabulary. The SPARC Vocabulary is built from a set of community ontologies covering major domains relevant to SPARC, including anatomy, physiology, experimental techniques, and molecules. The SPARC Vocabulary is incorporated into tools researchers use to segment and annotate their data, facilitating the application of these ontologies for annotation of research data. However, since investigators perform deep annotations on experimental data, not all terms and relationships are available in community ontologies. We therefore implemented a term management and vocabulary extension pipeline where SPARC researchers may extend the SPARC Vocabulary using InterLex, an online vocabulary management system. To ensure the quality of contributed terms, we have set up a curated term request and review pipeline specifically for anatomical terms involving expert review. Accepted terms are added to the SPARC Vocabulary and, when appropriate, contributed back to community ontologies to enhance autonomic nervous system coverage. Here, we provide an overview of the SPARC Vocabulary, the infrastructure and process for implementing the term management and review pipeline. In an analysis of &gt; 300 anatomical contributed terms, the majority represented composite terms that necessitated combining terms within and across existing ontologies. Although these terms are not good candidates for community ontologies, they can be linked to structures contained within these ontologies. We conclude that the term request pipeline serves as a useful adjunct to community ontologies for annotating experimental data and increases the FAIRness of SPARC data.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468399', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'The effect of sequencing and assembly on the inference of horizontal gene transfer on chromosomal and plasmid phylogenies', 'authors': \"[{'name': 'Jana S. Huisman'}, {'name': 'Timothy G. Vaughan'}, {'name': 'Adrian Egli'}, {'name': 'Sarah Tschudin-Sutter'}, {'name': 'Tanja Stadler'}, {'name': 'Sebastian Bonhoeffer'}]\", 'abstract': '<p>The spread of antibiotic resistance genes on plasmids is a threat to human and animal health. Phylogenies of bacteria and their plasmids contain clues regarding the frequency of plasmid transfer events, as well as the co-evolution of plasmids and their hosts. However, whole genome sequencing data from diverse ecological or clinical bacterial samples is rarely used to study plasmid phylogenies and resistance gene transfer. This is partially due to the difficulty to extract plasmids from short-read sequencing data. Here, we use both short- and long-read sequencing data of 24 clinical extended-spectrum <italic>β</italic>-lactamase producing <italic>Escherichia coli</italic> to estimate chromosomal and plasmid phylogenies. We compare the impact of different sequencing and assembly methodologies on these phylogenies and on the inference of horizontal gene transfer. We find chromosomal phylogenies can be estimated robustly with all methods, whereas plasmid phylogenies have more variable topology and branch lengths across the methods used. Specifically, hybrid methods that use long reads to resolve short-read assemblies (HybridSPAdes and Unicycler) perform better than those that started from long-reads during assembly graph generation (Canu). In contrast, the inference of plasmid and antibiotic resistance gene transfer using a parsimony-based criterion is mostly robust to the choice of sequencing and assembly method.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.15.468502', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structure and mechanism of oxalate transporter OxlT in an oxalate-degrading bacterium in the gut microbiota', 'authors': \"[{'name': 'Titouan Jaunet-Lahary'}, {'name': 'Tatsuro Shimamura'}, {'name': 'Masahiro Hayashi'}, {'name': 'Norimichi Nomura'}, {'name': 'Kouta Hirasawa'}, {'name': 'Tetsuya Shimizu'}, {'name': 'Masao Yamashita'}, {'name': 'Keiichi Kojima'}, {'name': 'Yuki Sudo'}, {'name': 'Takashi Tamura'}, {'name': 'Hiroko Iwanari'}, {'name': 'Takao Hamakubo'}, {'name': 'So Iwata'}, {'name': 'Kei-ichi Okazaki'}, {'name': 'Teruhisa Hirai'}, {'name': 'Atsuko Yamashita'}]\", 'abstract': '<p><italic>Oxalobacter formigenes</italic> is an oxalate-degrading bacterium in the gut microbiota that absorbs food-derived oxalate to use this as a carbon and energy source and thereby helps reduce the risk of kidney stone formation of the host animals <sup>1–4</sup>. The bacterial oxalate transporter OxlT uptakes oxalate from the gut to bacterial cells and excrete formate as a degradation product, with a strict discrimination from other carboxylates that serve as nutrients <sup>5–7</sup>. Nevertheless, the underlying mechanism remains unclear. Here, we present crystal structures of oxalate-bound and ligand-free OxlT in two different conformations, occluded and outward-facing states. The oxalate binding site contains two basic residues that form salt bridges with a dicarboxylate substrate while preventing the conformational switch to the occluded state without an acidic substrate, a ‘disallowed’ state for an antiporter <sup>8, 9</sup>. The occluded ligand-binding pocket can accommodate oxalate but not larger dicarboxylates, such as metabolic intermediates. The permeation pathways from the binding pocket are completely blocked by extensive interdomain hydrophobic and ionic interactions. Nevertheless, a molecular dynamics simulation showed that a flip of a single side chain neighbouring the substrate is sufficient to trigger the gate opening. The OxlT structure indicates the underlying metabolic interactions enabling favourable symbiosis at a molecular level.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.19.469353', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Exploration of the antimicrobial synergy between selected natural substances on Streptococcus mutans to identify candidates for the control of dental caries', 'authors': \"[{'name': 'Alisha Evangeline Prince'}, {'name': 'David J McDonald'}, {'name': 'Soumya Roy'}]\", 'abstract': '<p>Dental caries is caused by dental plaque, a community of micro-organisms embedded in an extracellular polymer matrix as a biofilm on the tooth surface. Natural products that are widely available could be used as an alternative or adjunctive anti-caries therapy. Sometimes, when two products are used together, they yield a more powerful antimicrobial effect than the anticipated additive effect. These synergistic combinations are often better treatment options because individual agents may not have sufficient antimicrobial action to be effective when used alone. Cranberries contain phenolic compounds like proanthocyanidins (PAC) that disrupt biofilm formation. Manuka honey has high concentrations of the agent methylglyoxal, which is cariostatic. Because these agents have varied modes of antimicrobial action, they show potential for possible synergistic effects when paired. Various cranberry extracts were tested pairwise with manuka honey or methylglyoxal by well-diffusion assays and 96-well checkerboard assays in the presence of Streptococcus mutans to test for synergy. Synergy was demonstrated in two of the cranberry extracts paired with manuka honey. The synergistic combinations found in this research thus can be considered as candidates for the formulation of a dentifrice that could be used to inhibit the formation of dental plaque and thereby avoid the development of caries.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.18.469179', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'evSeq: Cost-Effective Amplicon Sequencing of Every Variant in a Protein Library', 'authors': \"[{'name': 'Bruce J. Wittmann'}, {'name': 'Kadina E. Johnston'}, {'name': 'Patrick J. Almhjell'}, {'name': 'Frances H. Arnold'}]\", 'abstract': '<p>Widespread availability of protein sequence-fitness data would revolutionize both our biochemical understanding of proteins and our ability to engineer them. Unfortunately, even though thousands of protein variants are generated and evaluated for fitness during a typical protein engineering campaign, most are never sequenced, leaving a wealth of potential sequence-fitness information untapped. This largely stems from the fact that sequencing is unnecessary for many protein engineering strategies; the added cost and effort of sequencing is thus unjustified. Here, we present every variant sequencing (evSeq), an efficient protocol for sequencing a variable region within every variant gene produced during a protein engineering campaign at a cost of cents per variant. Execution of evSeq is simple, requires no sequencing experience to perform, relies only on resources and services typically available to biology labs, and slots neatly into existing protein engineering workflows. Analysis of evSeq data is likewise made simple by its accompanying software (found at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/fhalab/evSeq\">github.com/fhalab/evSeq</ext-link>, documentation at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://fhalab.github.io/evSeq\">fhalab.github.io/evSeq</ext-link>), which can be run on a personal laptop and was designed to be accessible to users with no computational experience. Low-cost and easy to use, evSeq makes collection of extensive protein variant sequence-fitness data practical.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468783', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Identification of diverse viruses associated with grasshoppers unveils phylogenetic congruence between hosts and viruses', 'authors': \"[{'name': 'Yao Xu'}, {'name': 'Jingyi Jiang'}, {'name': 'Xiaoju Lin'}, {'name': 'Wangpeng Shi'}, {'name': 'Chuan Cao'}]\", 'abstract': '<p>Locusts and grasshoppers are one of the most dangerous agricultural pests. Environmentally benign microbial pesticides are increasingly desirable for controlling locust outbreaks in fragile ecosystems. Here we use metagenomic sequencing to profile the rich viral communities in 34 grasshopper species and report 322 viruses, including 202 novel species. Most of the identified viruses are related to other insect viruses and some are targeted by antiviral RNAi pathway, indicating they infect grasshoppers. Some plant/fungi/vertebrate associated viruses are also abundant in our samples. Our analysis of relationships between host and virus phylogenies suggests that the composition of viromes is closely allied with host evolution, and there is significant phylogenetic relatedness between grasshoppers and viruses from <italic>Lispiviridae, Partitiviridae, Orthomyxoviridae, Virgaviridae</italic> and <italic>Flaviviridae</italic>. Overall, this study is a thorough exploration of viruses in grasshoppers and provide an essential evolutionary and ecological context for host-virus interaction in Acridoidea.</p><sec><title>Author Summary</title><p>Locusts are the most destructive migratory pest in the world and continue to cause massive damages that endanger food security and threaten millions of people in 21<sup>st</sup> century. While chemical pesticides are still heavily relied on, biological pesticides developed from natural pathogens offer a reliable, less harmful alternative for controlling locust outbreaks in fragile ecosystems. Unfortunately, little is known about natural pathogens infecting this pest. In this study, we profile the viral communities in 34 grasshopper species include some major swarming species. While we identified as many as 202 novel viral species associated with grasshoppers, some of them are of potential to be developed as biocontrol agents. Our analysis of relatedness of phylogenies of grasshoppers and associated viruses helps to shed light on the eco-evolutionary interactions between insects and viruses. This work provides a valuable dataset of both academic and applied interest.</p></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.19.469250', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Preclinical and Toxicology Studies of BRD5529, a Selective Inhibitor of CARD9', 'authors': \"[{'name': 'Theodore Kottom'}, {'name': 'Kyle Schaefbauer'}, {'name': 'Eva M Carmona'}, {'name': 'Joanne E Yi'}, {'name': 'Andrew Limper'}]\", 'abstract': '<p>Background: Exuberant inflammation during Pneumocystis pneumonia leads to lung injury. CARD9 is a central mediator of inflammatory signaling mediated by C-type lectin receptors. CARD9 inhibitor BRD5529 has been shown to be an effective in vitro inhibitor of Pneumocystis Beta-glucan-induced proinflammatory signaling and downstream TNF-alpha production, suggesting its viability as a candidate for preliminary drug testing as an anti-inflammatory agent in the rodent Pneumocystis pneumonia model (PCP).\\nMethods: To assess for potential toxicity, mice were injected intraperitoneally (IP) daily either with vehicle or BRD5529 at 0.1 mg/kg or 1.0 mg/kg for two weeks. Mouse weights were taken daily. At day 14, mice were euthanized, weighed, and analyzed by flexiVent for lung stiffness. Lungs, liver, and kidney were then harvested for H&amp;E staining and pathology scoring. Lung samples were further analyzed for proinflammatory cytokines via ELISA and extracellular matrix generation via quantitative PCR (q-PCR). Blood collection postmortem was performed for blood chemistry analysis.\\nResults: BRD5529 at both doses of IP administration resulted in no significant changes in daily or final weight gain. Analysis of lung stiffness by flexiVent showed no significant differences between the control or treated groups. Furthermore, ELISA results for proinflammatory IL-1 Beta, IL-6, and TNF-alpha showed no major differences in the respective groups. qPCR analysis of extracellular matrix transcripts collagen type I, alpha 1 (Col1a1) and fibronectin (Fn) were statically similar as well in the treated and control groups. Examination and pathology scoring of H&amp;E slides from lung, liver, and kidney from the each of the mice in all groups and subsequent pathology scoring showed no significant change. Blood chemistry analysis revealed similar, non-significant patterns.\\nConclusions: BRD5529 in our initial general safety and toxicology assessments displayed no inherent safety concerns in the analyzed parameters. These data support broader in vivo testing of the inhibitor as a timed adjunct therapy to the deleterious proinflammatory host immune response often associated with anti-Pneumocystis therapy.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.14.21265797', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Curating, collecting, and cataloguing global COVID-19 datasets for the aim of predicting personalized risk', 'authors': \"[{'name': 'Sepehr Golriz Khatami'}, {'name': 'Maria Francesca Russo'}, {'name': 'Daniel Domingo-Fernández'}, {'name': 'Andrea Zaliani'}, {'name': 'Sarah Mubeen'}, {'name': 'Yojana Gadiya'}, {'name': 'Astghik Sargsyan'}, {'name': 'Reagon Karki'}, {'name': 'Stephan Gebel'}, {'name': 'Ram Kumar Ruppa Surulinathan'}, {'name': 'Vanessa Lage-Rupprecht'}, {'name': 'Saulius Archipovas'}, {'name': 'Geltrude Mingrone'}, {'name': 'Marc Jacobs'}, {'name': 'Carsten Claussen'}, {'name': 'Martin Hofmann-Apitius'}, {'name': 'Alpha Tom Kodamullil'}, {'name': ' '}]\", 'abstract': '<p>The COVID-19 data catalogue is a repository that provides a landscape view of COVID-19 studies and datasets as a putative source to enable researchers to develop personalized COVID-19 predictive risk models. The COVID-19 data catalogue currently contains over 400 studies and their relevant information collected from a wide range of global sources such as global initiatives, clinical trial repositories, publications and data repositories. Further, the curated content stored in this data catalogue is complemented by a web application, providing visualizations of these studies, including their references, relevant information such as measured variables, and the geographical locations of where these studies were performed. This resource is one of the first to capture, organize and store studies, datasets and metadata in the area of COVID-19 in a comprehensive repository. We are convinced that our work will facilitate future research and development of personalized predictive risk models of COVID-19.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.16.468784', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'The Nerve Growth Factor IB-like Receptor Nurr1 (NR4A2) Recruits CoREST Transcription Repressor Complexes to Silence HIV Following Proviral Reactivation in Microglial Cells', 'authors': \"[{'name': 'Fengchun Ye'}, {'name': 'David Alvarez-Carbonell'}, {'name': 'Kien Nguyen'}, {'name': 'Saba Valadkhan'}, {'name': 'Konstantin Leskov'}, {'name': 'Yoelvis Garcia-Mesa'}, {'name': 'Sheetal Sreeram'}, {'name': 'Jonathan Karn'}]\", 'abstract': '<p>Human immune deficiency virus (HIV) infection of microglial cells in the brain leads to chronic neuroinflammation, which is antecedent to the development of HIV-associated neurocognitive disorders (HAND) in the majority of patients. Productively HIV infected microglia release multiple neurotoxins including proinflammatory cytokines and HIV proteins such as envelope glycoprotein (gp120) and transactivator of transcription (Tat). However, powerful counteracting silencing mechanisms in microglial cells result in the rapid shutdown of HIV expression to limit neuronal damage. Here we investigated whether the Nerve Growth Factor IB-like nuclear receptor Nurr1 (NR4A2), which is a repressor of inflammation in the brain, acts to directly restrict HIV expression. HIV silencing was substantially enhanced by Nurr1 agonists in both immortalized human microglial cells (hµglia) and induced pluripotent stem cells (iPSC)-derived human microglial cells (iMG). Overexpression of Nurr1 led to viral suppression, whereas by contrast, knock down (KD) of endogenous Nurr1 blocked HIV silencing. Chromatin immunoprecipitation (ChIP) assays showed that Nurr1 mediates recruitment of the CoREST/HDAC1/G9a/EZH2 transcription repressor complex to HIV promoter resulting in epigenetic silencing of active HIV. Transcriptomic studies demonstrated that in addition to repressing HIV transcription, Nurr1 also downregulated numerous cellular genes involved in inflammation, cell cycle, and metabolism, thus promoting HIV latency and microglial homoeostasis. Thus, Nurr1 plays a pivotal role in modulating the cycles of proviral reactivation by cytokines and potentiating the proviral transcriptional shutdown. These data highlight the therapeutic potential of Nurr1 agonists for inducing HIV silencing and microglial homeostasis and amelioration of the neuroinflammation associated with HAND.</p><sec><title>AUTHOR SUMMARY</title><p>HIV enters the brain almost immediately after infection where it infects perivascular macrophages, microglia and, to a less extent, astrocytes. In previous work using an immortalized human microglial cell model, we observed that integrated HIV constantly underwent cycles of reactivation and subsequent silencing. In the present study, we found that the Nurr1 nuclear receptor is a key mediator of HIV silencing. The functional activation of Nurr1 by specific agonists, or the over expression of Nurr1, resulted in rapid silencing of activated HIV in microglial cells. Global gene expression analysis confirmed that Nurr1 not only repressed HIV expression but also regulated numerous genes involved in microglial homeostasis and inflammation. Thus, Nurr1 is pivotal for HIV silencing and repression of inflammation in the brain and is a promising therapeutic target for treatment of HAND.</p></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.20.21266571', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Respiratory Medicine', 'covid': False, 'title': 'Turning green: the impact of changing to more eco-friendly respiratory healthcare. A carbon and cost analysis of Dutch prescription data.', 'authors': \"[{'name': 'Pieter ten Have'}, {'name': 'Peter van Hal'}, {'name': 'Iris Wichers'}, {'name': 'Johan Kooistra'}, {'name': 'Paul Hagedoorn'}, {'name': 'Niels Chavannes'}, {'name': 'Evelyn Brakema'}, {'name': 'Pauline de Heer'}, {'name': 'Hans C Ossebaard'}]\", 'abstract': '<p><italic>Objectives</italic> \\nDry powder inhalers (DPIs) have a substantially lower global warming potential than pressurized metered-dose inhalers (MDIs). To help mitigate climate change, we assessed the potential reduction in CO2-equivalents when replacing MDIs by DPIs in the Netherlands, and estimated the associated cost.\\n \\n<italic>Design</italic> \\nWe performed a four-step analysis based on data from two national databases of two independent governmental bodies (Dutch National Healthcare Institute and the Dutch Healthcare Authority). First, we calculated the number of patients with Chronic Obstructive Pulmonary Disease (COPD) and asthma that were using inhalation medication (2020). Second, we calculated the number and total of daily defined doses of MDIs, DPIs, and soft mist inhalers and the number of spacers per patients, dispensed by non-hospital based pharmacies  in 2020. Third, we estimated the potential reduction in CO2-equivalents (eq.) if all eligible patients (≥7 years old; COPD with ≤exacerbation per year) would switch from using MDIs to using DPIs as eco-friendly alternatives. Fourth, we performed a cost-effectiveness analysis.\\n\\n<italic>Results</italic> \\nIn 2020, 1.4 million patients used inhalers for COPD or asthma treatment. A total of 460 million defined daily doses (DDDs) from inhalers were dispensed, of which, after the exclusion of nebulisers, 50.4% were from MDIs. We estimated that this use could be reduced by 70% leading to annual reduction in emissions of  77 - 84  million kg CO2eq. saving at best EUR 49.8 million annually.\\n\\n<italic>Conclusions</italic>\\nIn the Netherlands, substitution of MDIs to DPIs for eligible patients is theoretically safe and in accordance with medical guidelines, while reducing emissions by 80 million kg.CO2eq. on average and saving at best EUR 49.8 million per year. This study confirms the potential climate and economic benefit of delivering eco-friendlier respiratory care.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.15.468661', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Investigating Carboxysome Morphology Dynamics with a Rotationally Invariant Variational Autoencoder', 'authors': \"[{'name': 'Miguel Fuentes-Cabrera'}, {'name': 'Jonathan K. Sakkos'}, {'name': 'Daniel C. Ducat'}, {'name': 'Maxim Ziatdinov'}]\", 'abstract': '<p>Carboxysomes are a class of bacterial microcompartments that form proteinaceous organelles within the cytoplasm of cyanobacteria and play a central role in photosynthetic metabolism by defining a cellular microenvironment permissive to <italic>CO</italic><sub>2</sub> fixation. Critical aspects of the assembly of the carboxysomes remain relatively unknown, especially with regard to the dynamics of this microcompartment. We have recently expressed an exogenous protease as a way of gaining control over endogenous protein levels, including carboxysomal components, in the model cyanobacterium <italic>Synechococcous elongatus</italic> PCC 7942. By utilizing this system, proteins that compose the carboxysome can be tuned in real-time as a method to examine carboxysome dynamics. Yet, analysis of subtle changes in carboxysome morphology with microscopy remains a low-throughput and subjective process. Here we use deep learning techniques, specifically a Rotationally Invariant Variational Autoencoder (rVAE), to analyze the fluorescence microscopy images and quantitatively evaluate how carboxysome shell remodelling impacts trends in the morphology of the microcompartment over time. We find that rVAEs are able to assist in the quantitative evaluation of changes in carboxysome location, shape, and size over time. We propose that rVAEs may be a useful tool to accelerate the analysis of carboxysome assembly and dynamics in response to genetic or environmental perturbation, and may be more generally useful to probe regulatory processes involving a broader array of bacterial microcompartments.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469102', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Single nucleus RNA-seq in the hippocampus of a Down syndrome mouse model reveals new key players in memory', 'authors': \"[{'name': 'Cesar Sierra'}, {'name': 'Ilario De Toma'}, {'name': 'Mara Dierssen'}]\", 'abstract': '<p>Down syndrome (DS) is the most common genetic cause of intellectual disability. Even though great advances in the last decades have allowed better delineation of its pathogenetic mechanisms, its cellular and molecular bases are still poorly understood. To evaluate the consequences of chromosome aneuploidy on the hippocampus, we analyzed single-nucleus transcriptional profiles of the DS mouse model Ts65Dn. Our data revealed abnormal cell composition specifically of the Ts65Dn dentate gyrus and of specific subtypes of interneurons, without major changes in CA1 or CA3. We found that trisomy results in a highly cell-type specific global alteration of the transcriptome and detected previously undefined differentially expressed genes in specific neuronal populations. We identified the long-non-coding gene Snhg11 to be specifically downregulated in the trisomic dentate gyrus and provide evidence for its involvement in hippocampal-dependent cognitive phenotypes, possibly contributed by impaired adult neurogenesis.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.18.469109', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Adapting to novel environments together: evolutionary and ecological correlates of the bacterial microbiome of the world’s largest cavefish diversification', 'authors': \"[{'name': 'Shipeng Zhou'}, {'name': 'Amrapali Prithvisingh Rajput'}, {'name': 'Yewei Liu'}, {'name': 'Tingru Mao'}, {'name': 'Jian Yang'}, {'name': 'Jayampathi Herath'}, {'name': 'Madhava Meegaskumbura'}]\", 'abstract': '<p>The symbiosis between a host and its microbiome is essential for host fitness, and this association is a consequence of the host’s physiology and habitat. <italic>Sinocyclocheilus</italic>, the largest cavefish diversification of the world, an emerging multi-species model system for evolutionary novelty, provides an excellent opportunity for examining correlates of host evolutionary history, habitat, and gut-microbial community diversity. From the diversification-scale patterns of habitat occupation, major phylogenetic clades (A–D), geographic distribution, and knowledge from captive-maintained <italic>Sinocyclocheilus</italic> populations, we hypothesize habitat to be the major determinant of microbiome diversity, with phylogeny playing a lesser role. For this, we subject environmental water samples and fecal samples (representative of gut-microbiome) from 24 <italic>Sinocyclocheilus</italic> species, both from the wild and after being in captivity for six months, to bacterial 16S rRNA gene profiling using Illumina sequencing. We see significant differences in the gut microbiota structure of <italic>Sinocyclocheilus,</italic> reflective of the three habitat types; gut microbiomes too, were influenced by host-related factors. There is no significant association between the gut microbiomes and host phylogeny. However, there is some microbiome related structure at clade level, with the most geographically distant clades (A and D) being the most distinct, and two geographically overlapping clades (B and C) being similar. Microbes inhabiting water were not a cause for significant differences in fish-gut microbiota, but water quality parameters was. Transferring from wild to captivity, the fish microbiomes changed significantly and became homogenized, signifying adaptability and highlighting the importance of environmental factors (habitat) in microbiome community assembly. The core microbiome of this group closely resembled that of other teleost fishes. Our results suggest that divergent selection giving rise to evolutionary novelties also includes the microbiome of these fishes, which provides a functional advantage for life in the resource-depleted cave environment.</p><sec><title>SIGNIFICANCE STATEMENT</title><p>The largest diversification of cavefishes of the world, <italic>Sinocyclocheilus</italic>, not only show that habitat, and phylogenetic clade is important in determining their gut microbiome, but also that they reach a common microbiome in captivity irrespective of their phylogenetic position, region of origin and habitat, indicating that they are adaptable in the context of microbe related changes in their environment.</p></sec>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.18.469098', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': True, 'title': 'Mechanistic insights into global suppressors of protein folding defects', 'authors': \"[{'name': 'Gopinath Chattopadhyay'}, {'name': 'Jayantika Bhowmick'}, {'name': 'Kavyashree Manjunath'}, {'name': 'Shahbaz Ahmed'}, {'name': 'Parveen Goyal'}, {'name': 'Raghavan Varadarajan'}]\", 'abstract': '<p>Most amino acid substitutions in a protein either lead to partial loss of function or are near neutral. Several studies have shown the existence of second-site mutations that can rescue defects caused by diverse loss of function mutations. Such global suppressor mutations are key drivers of protein evolution. However, the mechanisms responsible for such suppression remain poorly understood. To address this, we characterized multiple suppressor mutations both in isolation and in combination with inactive mutants. We examined five global suppressors of the bacterial toxin CcdB, the known M182T global suppressor of TEM-1 β-lactamase, the N239Y global suppressor of p53-DBD and three suppressors of the SARS-CoV-2 spike Receptor Binding Domain. The suppressors both alone, and in conjunction with inactive mutants, stabilise the protein both thermodynamically and kinetically in-vitro, predominantly through acceleration of the refolding rate parameters. When coupled to inactive mutants they promote increased in-vivo solubilities as well as regain-of-function phenotypes. Our study also demonstrates that the global suppressor approach can be used to consistently stabilise wild-type proteins, including for downstream translational applications.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468684', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Engraftment of allotransplantated tumour cells in adult rag2 mutant Xenopus tropicalis', 'authors': \"[{'name': 'Dieter Tulkens'}, {'name': 'Dionysia Dimitrakopoulou'}, {'name': 'Tom Van Nieuwenhuysen'}, {'name': 'Marthe Boelens'}, {'name': 'Suzan Demuynck'}, {'name': 'Wendy Toussaint'}, {'name': 'David Creytens'}, {'name': 'Pieter Van Vlierberghe'}, {'name': 'Kris Vleminckx'}]\", 'abstract': '<p>Modelling human genetic diseases and cancer in lab animals has been greatly aided by the emergence of genetic engineering tools such as TALENs and CRISPR/Cas9. We have previously demonstrated the ease with which genetically engineered <italic>Xenopus</italic> models (GEXM) can be generated. This included the induction of autochthonous tumour formation by injection of early embryos with Cas9 recombinant protein loaded with sgRNAs targeting multiple tumour suppressor genes. What has been lacking so far is the possibility to propagate the induced cancers via transplantation. In this paper we describe the generation of a <italic>rag2</italic><sup><italic>-/-</italic></sup> knock-out line in <italic>Xenopus tropicalis</italic> that is deficient in functional T- and B-cells. This line was validated by means of an allografting experiment with a primary <italic>tp53</italic><sup><italic>-/-</italic></sup> donor tumour. In addition, we optimized available protocols for sub-lethal gamma irradiation of <italic>X. tropicalis</italic> froglets. Irradiated animals also allowed stable, albeit transient, engraftment of transplanted <italic>tp53</italic><sup><italic>-/-</italic></sup> tumour cells. The novel <italic>X. tropicalis rag2</italic><sup><italic>-/-</italic></sup> line and the irradiated wild type froglets will further expand the experimental toolbox in this diploid amphibian, and help to establish it as a versatile and relevant model for exploring human cancer.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.18.469073', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Adhesion strength between cells regulate non-monotonic growth by a biomechanical feedback mechanism', 'authors': \"[{'name': 'Abdul Malmi-Kakkada'}, {'name': 'Sumit Sinha'}, {'name': 'Xin Li'}, {'name': 'Dave Thirumalai'}]\", 'abstract': '<p>We probe the interplay between intercellular interactions and pressure fluctuations associated with single cells in regulating cell proliferation using simulations of a minimal model for three-dimensional multicellular spheroid (MCS) growth. The emergent spatial variations in the cell division rate, that depends on the location of the cells within the MCS, is regulated by intercellular adhesion strength (f^{ad}). This in turn results in non-monotonic proliferation of cells in the MCS with varying adhesion strength, which accords well with experimental results. A biomechanical feedback mechanism coupling the f^{ad} and cell-dependent pressure fluctuations relative to a threshold value (p_c) determines the onset of a dormant phase, and explains the non-monotonic proliferation response. Increasing f^{ad} from low values enhances cell proliferation because pressure on individual cells is smaller compared to p_c. In contrast, at high f^{ad}, cells readily become dormant and cannot rearrange effectively, leading to arrested cell proliferation. Our work, which shows that proliferation is regulated by pressure-adhesion feedback loop, may be a general feature of tumor growth.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.468912', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Validity of freehand three-dimensional ultrasound system in measurement of three-dimensional surface shape of shoulder muscles', 'authors': \"[{'name': 'Jun Umehara'}, {'name': 'Norio Fukuda'}, {'name': 'Shoji Konda'}, {'name': 'Masaya Hirashima'}]\", 'abstract': '<p>Accurate measurement of muscle morphology is crucial for assessing skeletal muscle capacity. Although the freehand three-dimensional ultrasound (3DUS) system is a promising technique for assessing muscle morphology, its accuracy has been validated mainly in terms of volume by examining lower limb muscles. The purpose of this study was to validate 3DUS in the measurements of 3D surface shape and volume by comparing them with MRI measurements while ensuring the reproducibility of participant posture by focusing on the shoulder muscles. The supraspinatus, infraspinatus, and posterior deltoid muscles of 10 healthy males were scanned using 3DUS and MRI while secured by an immobilization support customized for each participant. A 3D surface model of each muscle was created from the 3DUS and MRI methods, and the agreement between them was assessed. For the muscle volume, the mean difference between the two models was within 0.51 cm<sup>3</sup> for all muscles. For the surface shape, the distances between the closest points of the two models was calculated for every point on the 3DUS surface model. The results showed that the median (third quartile) of the distances was less than 1.21 mm (1.89 mm) for all muscles. These results suggest that, given the above error is permitted, 3DUS can be used as an alternative to MRI in measuring volume and surface shape, even for the shoulder muscles.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.18.469076', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genome and transcriptome architecture of allopolyploid okra (Abelmoschus esculentus)', 'authors': \"[{'name': 'Ronald Nieuwenhuis'}, {'name': 'Thamara Hesselink'}, {'name': 'Hetty C. van den Broeck'}, {'name': 'Jan Cordewener'}, {'name': 'Elio Schijlen'}, {'name': 'Linda Bakker'}, {'name': 'Sara Diaz Trivino'}, {'name': 'Darush Struss'}, {'name': 'Simon-Jan de Hoop'}, {'name': 'Hans de Jong'}, {'name': 'Sander A. Peters'}]\", 'abstract': '<p>We present the first annotated genome assembly of the allopolyploid okra (<italic>Abelmoschus esculentus</italic>). Analysis of telomeric repeats and gene rich regions suggested we obtained whole chromosome and chromosomal arm scaffolds. Besides long distal blocks we also detected short interstitial TTTAGGG telomeric repeats, possibly representing hallmarks of chromosomal speciation upon polyploidization of okra. Ribosomal RNA genes are organized in 5S clusters separated from the 18S-5.8S-28S units, clearly indicating an S-type rRNA gene arrangement. The assembly is consistent with cytogenetic and cytometry observations, identifying 65 chromosomes and 1.45Gb of expected genome size in a haploid sibling. Approximately 57% of the genome consists of repetitive sequence. BUSCO scores and A50 plot statistics indicated a nearly complete genome. Kmer distribution analysis suggests that approximately 75% has a diploid nature, and at least 15% of the genome is heterozygous. We did not observe aberrant meiotic configurations, suggesting there is no recombination among the sub-genomes. BUSCO configurations pointed to the presence of at least 3 sub-genomes. These observations are indicative for an allopolyploid nature of the okra genome. Structural annotation using gene models derived from mapped transcriptome data, generated over 130,000 putative genes. The discovered genes appeared to be located predominantly at the distal ends of scaffolds, gradually decreasing in abundance toward more centrally positioned scaffold domains. In contrast, LTR retrotransposons were more abundant in centrally located scaffold domains, while less frequently represented in the distal ends. This gene and LTR-retrotransposon distribution is consistent with the observed heterochromatin organization of pericentromeric heterochromatin and distal euchromatin. The derived amino acid queries of putative genes were subsequently used for phenol biosynthesis pathway annotation in okra. Comparison against manually curated reference KEGG pathways from related <italic>Malvaceae</italic> species revealed the genetic basis for putative enzyme coding genes that likely enable metabolic reactions involved in the biosynthesis of dietary and therapeutic compounds in okra.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.18.469075', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Olorofim and the azoles are antagonistic in A. fumigatus and functional genomic screens reveal mechanisms of cross resistance', 'authors': \"[{'name': 'Norman van Rhijn'}, {'name': 'Sam Hemmings'}, {'name': 'Clara Valero'}, {'name': 'Jorge Amich'}, {'name': 'Michael J Bromley'}]\", 'abstract': '<p>Aspergillosis, in its various manifestations, is a major cause of morbidity and mortality. Very few classes of antifungal have been approved for clinical use to treat these diseases and resistance to the first line therapeutics is increasing. A new class of antifungals, the orotomides, are currently in development with the first compound in this class olorofim in late-stage clinical trials. In this study, we characterise a network of genes that govern olorofim response in <italic>A. fumigatus</italic>. We reveal that the number of transcription factors that regulate olorofim susceptibility are far fewer than we have previously observed for the azoles and the change in sensitivity observed in these isolates is less extreme. Intriguingly, loss of function in two higher order transcriptional regulators, HapB a member of the heterotrimeric HapB/C/E (CBC) complex or the regulator of nitrogen metabolic genes AreA, leads to cross resistance to both the azoles and olorofim. However, a clinical azole resistant isolate with a point mutation in HapE (<italic>hapE<sup>P88L</sup></italic>) retains sensitivity to olorofim. Our transcriptomic analysis suggests that altered sensitivity to olorofim may emerge via modification of genes involved in the production of pyrimidine biosynthetic precursors. Finally, we also show that the action of the azoles are antagonistic to olorofim <italic>in vitro</italic>.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.15.468716', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Rigid DNA nanotube tethers suppress high frequency noise in dual-trap optical tweezers systems', 'authors': \"[{'name': 'Alan Shaw'}, {'name': 'Rohit Satija'}, {'name': 'Eduardo Antunez de Mayolo De la Matta'}, {'name': 'Susan Marqusee'}, {'name': 'Carlos Bustamante'}]\", 'abstract': '<p>Dual trap optical tweezers are a powerful tool to trap and characterize the biophysical properties of single biomolecules such as the folding pathways of proteins and nucleic acids, and the chemomechanical activity of molecular motors. Despite its vastly successful application, noise from drift and fluctuation of the optics, and Brownian motion of the trapped beads still hinder the technique’s ability to directly visualize folding of small biomolecules or the single nucleotide stepping of polymerases, especially at low forces (&lt;10 pN) and sub-millisecond timescales. Rigid DNA nanotubes have been used to replace the conventional dsDNA linker to reduce optical tweezers noise in the low force range. However, optical tweezers are used to study a wide range of biophysical events, with timescales ranging from microseconds to seconds, and length changes ranging from sub nanometers to tens of nanometers. In this study, we systematically evaluate how noise is distributed across different frequencies in dual trap optical tweezers systems–and show that rigid DNA nanotube tethers suppress only high frequency noise (kHz), while the low frequency noise remains the same when compared to that of dsDNA tethers.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.17.468952', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Multiple trajectory alignment reconstructs disease dynamics for discovery and clinical benefit', 'authors': \"[{'name': 'Amit Frishberg'}, {'name': 'Neta Milman'}, {'name': 'Ayelet Alpert'}, {'name': 'Fabian J Theis'}, {'name': 'Joachim L. Schultze'}, {'name': 'Shai S. Shen-Orr'}]\", 'abstract': '<p>Biological processes are changes occurring concomitantly over time. Therefore, any comparison ignorant of temporal changes is greatly deficient. Even in longitudinal experimental designs, data is usually collected at fixed intervals, whereas process dynamics vary between individuals. We developed TimeAx, which brings time into the equation by building a comparative framework for capturing biological processes dynamics from omic data. We used TimeAx to study influenza infection dynamics and Urothelial bladder cancer tumorigenesis, discovering molecular mechanisms that drive disease progression as well as promoting clinical symptoms, for short and long term biological processes. Specifically, we detected an inflection point, where the tumor progresses into an advanced, pro-metastatic state. Overall, we present a powerful framework for studying high resolution temporal dynamics, providing improved molecular interoperability and clinical benefit.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469147', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Moderate-intensity exercise might be a competitive protocol to prevent obesity and obesity-related metabolic diseases', 'authors': \"[{'name': 'Ryeo-Won Kwon'}, {'name': 'Seung-Jea Lee'}, {'name': 'Tae-Wook Kim'}, {'name': 'Jeong Beom Lee'}]\", 'abstract': '<p>To determine effects of exercise intensity on expression levels of cytokines and neurotransmitters beneficial for the prevention and improvement of obesity and obesity-related metabolic diseases. Expression levels of irisin, fibroblast growth factor-21 (FGF21), adiponectin, free fatty acid (FFA), dopamine (DA), and serotonin (5-HT) levels after moderate-intensity (50% of maximal oxygen uptake (VO<sub>2</sub>max)) and high-intensity treadmill running (80% VO<sub>2</sub>max) for 30 min in 20 healthy men were compared. Blood samples were collected at three time points: before treadmill running (pre-EX), immediately after treadmill running (post-EX), and at 60 min after treadmill running (60 min post-EX). Expression levels of irisin, FGF21, adiponectin, FFA, DA, and 5-HT were increased after 30 min of treadmill running exercise regardless of exercise intensity. Their levels were increased at 60 min post-EX. They showed no significant difference immediately after exercise regardless of exercise intensity. Only irisin, FGF21, FFA, and DA levels showed significant differences between moderate-intensity (50% VO<sub>2</sub>max) group and high-intensity group (80% VO<sub>2</sub>max) at 60 min post-EX. Considering that FFA level at 80% VO<sub>2</sub>max was significantly lower than that at 50% VO<sub>2</sub>max during post-EX and side effects of high-intensity exercise, moderate-intensity exercise would be a more competitive method compared to high-intensity for the prevention and improvement of obesity and obesity-related metabolic diseases.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.19.469193', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Histidine-rich protein 2: a new pathogenic factor of \\nPlasmodium falciparum\\n malaria', 'authors': \"[{'name': 'Takashi Iwasaki'}, {'name': 'Mayu Shimoda'}, {'name': 'Haru Kanayama'}, {'name': 'Tsuyoshi Kawano'}]\", 'abstract': '<p><italic>Plasmodium falciparum</italic> causes serious malaria symptoms; when this protozoan parasite infects human erythrocytes, it produces and secretes large amounts of histidine–rich protein 2 (PfHRP2) into human blood. Thus, PfHRP2 is a well–known diagnostic marker for malaria infection. Here, however, we also identified PfHRP2 as a pathogenic factor produced by <italic>P. falciparum</italic>. PfHRP2 showed cell penetration and cytotoxicity against various human cells. In particular, PfHRP2 showed significant cytotoxicity over 5 days at the same concentration as in <italic>P. falciparum</italic>–infected patients′ blood (90–100 nM). This result is consistent with the mortality rate of <italic>P. falciparum</italic> malaria, which increases rapidly in untreated cases for 3–7 days. In addition, the cell penetration and cytotoxicity of PfHRP2 increased 2.5– and 2.6–fold, respectively, in the absence of serum, which suggests that low serum protein concentrations (occurring during malnutrition, for example) increase the risk of adverse effects from PfHRP2 (consistent with malnutrition increasing the lethality of malaria infection). We also showed that PfHRP2 bound to Ca<sup>2+</sup> ions, localized to intracellular lysosomes, increased lysosomal Ca<sup>2+</sup> levels, and inhibited the basal level of autophagy by inhibiting autolysosome formation. Furthermore, the Ca<sup>2+</sup>–dependent cytotoxicity of PfHRP2 was suppressed by the metal ion chelator ethylenediaminetetraacetic acid (EDTA). In summary, our findings suggest that PfHRP2 acts as a pathogenic factor in <italic>P. falciparum</italic>–infected patients and is associated with the exacerbation of malaria. Furthermore, EDTA is a promising candidate as a therapeutic agent for the suppression of PfHRP2 pathogenicity. Overall, this study provides new insights into <italic>P. falciparum</italic> malaria pathogenesis and treatment.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.21266374', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': False, 'title': 'Studies of parenchymal texture added to mammographic breast density and risk of breast cancer: a systematic review of the methods used in the literature', 'authors': \"[{'name': 'Akila Anandarajah'}, {'name': 'Yongzhen Chen'}, {'name': 'Graham A. Colditz'}, {'name': 'Angela Hardi'}, {'name': 'Carolyn Stoll'}, {'name': 'Shu Jiang'}]\", 'abstract': '<p>This systematic review aimed to assess the methods used to classify mammographic breast parenchymal features in relation to prediction of future breast cancer including the time from mammogram to diagnosis of breast cancer, and methods for the identification of texture features and selection of features for inclusion in analysis. The databases including Medline (Ovid) 1946-, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://Embase.com\">Embase.com</ext-link> 1947-, CINAHL Plus 1937-, Scopus 1823-, Cochrane Library (including CENTRAL), and <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://Clinicaltrials.gov\">Clinicaltrials.gov</ext-link>. were searched through October 2021 to extract published articles in English describing the relationship of parenchymal texture features with risk of breast cancer. Twenty-eight articles published since 2016 were included in the final review. Of these, 7 assessed texture features from film mammograms images, 3 did not report details of the image used, and the others used full field mammograms from Hologic, GE and other manufacturers. The identification of parenchymal texture features varied from using a predefined list to machine-driven identification. Reduction in number of features chosen for analysis in relation to cancer incidence then varied across statistical approaches and machine learning methods. The variation in approach and number of features identified for inclusion in analysis precluded generating a quantitative summary or meta-analysis of the value of these to improve predicting risk of future breast cancers. This updated overview of the state of the art revealed research gaps; based on these, we provide recommendations for future studies using parenchymal features for mammogram images to make use of accumulating image data, and external validation of prediction models that extend to 5 and 10 years to guide clinical risk management. By following these recommendations, we expect to improve risk classification and risk prediction for women to tailor screening and prevention strategies to level of risk.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.18.464426', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Parental control: ecology drives plasticity in parental response to offspring signals', 'authors': \"[{'name': 'Shana Caro'}, {'name': 'Adara C Velasco'}, {'name': 'Tjomme van Mastrigt'}, {'name': 'Kees van Oers'}, {'name': 'Ashleigh S Griffin'}, {'name': 'Stuart A West'}, {'name': 'Camilla A Hinde'}]\", 'abstract': '<p>Different bird species have completely different parent-offspring interactions. When food is plentiful, the chicks that are begging the loudest are fed the most. When food is scarce, bird species instead feed the largest offspring. While this variation could be due to parents responding to signalling differently based on food availability, it could equally be due to offspring adjusting their behaviour, or to variation in information availability. We tested between these competing explanations experimentally, by manipulating food availability in a population of wild great tits, Parus major, while standardising offspring behaviour and size. We found that when food was more plentiful, parents were: (1) more likely to preferentially feed the chicks that were begging the most; and (2) less likely to preferentially feed larger chicks. In addition, we consistently found these same patterns, in a meta-analysis across 57 bird species. Overall, our results suggest that parents have more control over food distribution than offspring do, and that they flexibly adjust how they respond to both offspring signals and cues of offspring quality in response to food availability. Consequently, depending upon environmental conditions, predictably different signalling systems are favoured.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.15.468751', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Transcription Factor Gene Pea3 Regulates Erectile Function in Mice', 'authors': \"[{'name': 'Jarret A.P. Weinrich'}, {'name': 'Aanchal Tyagi'}, {'name': 'Megan C. Kenney'}, {'name': 'Richard J. DiCasoli'}, {'name': 'Julia A. Kaltschmidt'}]\", 'abstract': '<sec><title>Background</title><p>Male mice with homozygous loss of function mutations of the <italic>ETS</italic> transcription factor gene <italic>Pea3</italic> (<italic>Pea3</italic> null) are infertile due to their inability to deposit semen plugs, however the specific deficits in male sexual behaviors that drive this phenotype are unknown.</p></sec><sec><title>Aim</title><p>To investigate the regulatory role of the <italic>Pea3</italic> gene in organizing gross sexual behaviors and erectile functioning during active copulation.</p></sec><sec><title>Methods</title><p>The copulatory behavior of male mice (<italic>Pea3</italic> null and control) with hormonally primed ovariectomized females was monitored via high-speed and high-resolution digital videography to assess for differences in female-directed social behaviors, gross sexual behaviors (mounting, thrusting), and erectile and ejaculatory function.</p></sec><sec><title>Outcomes</title><p><italic>Pea3</italic> null male mice have dramatically reduced erectile function during sexual intercourse, however other aspects of male sexual behaviors are largely intact.</p></sec><sec><title>Results</title><p><italic>Pea3</italic> null male mice exhibit greatly reduced erectile function, with 44% of males displaying no visible erections during mounting behaviors, and none achieving sustained erections. As such, <italic>Pea3</italic> null males are incapable of intromission, and semen plug deposition, despite displaying largely normal female-directed social behaviors, mounting behaviors, and ejaculatory grasping behavior. Additionally, the coordination of the timing of thrusting trains is impaired in <italic>Pea3</italic> null males.</p></sec><sec><title>Clinical Implications</title><p>The identification of the transcription factor <italic>Pea3</italic> in regulating erectile function in mice may provide a useful target for understanding the genetics of male sexual dysfunction in human patients.</p></sec><sec><title>Strengths and Limitations</title><p>High-speed and high-resolution videography allows for a detailed analysis of male sexual behaviors and erectile functioning in <italic>Pea3</italic> null and control mice. How disruption of the <italic>Pea3</italic> gene translates to erectile dysfunction is still unknown.</p></sec><sec><title>Conclusion</title><p>The transcription factor gene <italic>Pea3</italic> regulates the ability to achieve and maintain erections in male mice.</p></sec>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.469012', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'DNA Breaks and Gaps Target Retroviral Integration', 'authors': \"[{'name': 'Gayan Senavirathne'}, {'name': 'Anne Gardner'}, {'name': 'James London'}, {'name': 'Ryan K. Messer'}, {'name': 'Yow Yong Tan'}, {'name': 'Kristine E. Yoder'}, {'name': 'Richard Fishel'}]\", 'abstract': '<p>Integration into a host genome is essential for retrovirus infection and is catalyzed by a nucleoprotein complex (Intasome) containing the virus-encoded integrase (IN) and the reverse transcribed (RT) virus copy DNA (cDNA). Previous studies suggested that integration was limited by intasome-host DNA recognition progressions. Using single molecule Förster resonance energy transfer (smFRET) we show that PFV intasomes pause at nicked and gapped DNA, which targeted site-directed integration without inducing significant intasome conformational alterations. Base excision repair (BER) components that affect retroviral integration <italic>in vivo</italic> produce similar nick/gap intermediates during DNA lesion processing. Intasome pause dynamics was modified by the 5’-nick/gap chemistry, while an 8-oxo-guanine lesion, a mismatch, or a nucleotide insertion that induce backbone flexibility and/or static bends had no effect. These results suggest that dynamic often non-productive intasome-DNA interactions may be modulated to target retroviral integration.</p>', 'posted': '2021-11-17'}, {'doi': '10.1101/2021.11.15.468768', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Pseudomonas synergizes with fluconazole against Candida during treatment of polymicrobial infection', 'authors': \"[{'name': 'Siham Hattab'}, {'name': 'Anna-Maria Dagher'}, {'name': 'Robert T. Wheeler'}]\", 'abstract': '<p>Polymicrobial infections are challenging to treat because we don’t fully understand how pathogens interact during infection and how these interactions affect drug efficacy. <italic>Candida albicans</italic> and <italic>Pseudomonas aeruginosa</italic> are opportunistic pathogens that can be found in similar sites of infection such as in burn wounds and most importantly in the lungs of CF and mechanically ventilated patients. <italic>C. albicans</italic> is particularly difficult to treat because of the paucity of antifungal agents, some of which lack fungicidal activity. In this study, we investigated the efficacy of anti-fungal treatment during <italic>C. albicans</italic>-<italic>P. aeruginosa</italic> co-culture <italic>in vitro</italic> and co-infection in the mucosal zebrafish infection model analogous to the lung. We find that <italic>P. aeruginosa</italic> enhances the activity of fluconazole (FLC), an anti-fungal drug that is fungistatic <italic>in vitro</italic>, to promote both clearance of <italic>C. albicans</italic> during co-infection <italic>in vivo</italic> and fungal killing <italic>in vitro</italic>. This synergy between FLC treatment andbacterial antagonism is partly due to iron piracy, as it is reduced upon iron supplementation and knockout of bacterial siderophores. Our work demonstrates that FLC has enhanced activity in clinically relevant contexts and highlights the need to understand antimicrobial effectiveness in the complex environment of the host with its associated microbial communities.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.15.468624', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Genetic coordination of sperm morphology and seminal fluid proteins promotes male reproductive success in Drosophila melanogaster', 'authors': \"[{'name': 'Jake Galvin'}, {'name': 'Erica L. Larson'}, {'name': 'Sevan Yedigarian'}, {'name': 'Mohammad Rahman'}, {'name': 'Kirill Borziak'}, {'name': 'Michael DeNieu'}, {'name': 'Mollie K. Manier'}]\", 'abstract': '<p>Spermatozoal morphology is highly variable both among and within species and in ways that can significantly impact fertilization success. In <italic>Drosophila melanogaster</italic>, paternity success depends on sperm length of both competing males and length of the female’s primary sperm storage organ. We found that genes upregulated in long sperm testes are enriched for lncRNAs and seminal fluid proteins (Sfps). Transferred in seminal fluid to the female during mating, Sfps are secreted by the male accessory glands (AG) and affect female remating rate, physiology, and behavior with concomitant advantages for male reproductive success. Despite being upregulated in long sperm testes, they have no known function in testis tissue. We found that Sex Peptide and ovulin (Acp26Aa) knockouts resulted in shorter sperm, suggesting that Sfps may regulate sperm length during spermatogenesis. However, knockout of AG function did not affect sperm length, suggesting that AG expression has no influence on spermatogenic processes. We also found that long sperm males are better able to delay female remating, suggesting higher Sfp expression in AG. These results might suggest that long sperm males have a double advantage in sperm competition by both delaying female remating, likely through transfer of more Sfps, and by resisting sperm displacement. However, we also found that this extra advantage does not necessarily translate to more progeny or higher paternity success. Thus, we found that multiple components of the ejaculate coordinate to promote male reproductive success at different stages of reproduction, but the realized fitness advantages in sperm competition are uncertain.</p><sec><title>Significance Statement</title><p>The ejaculate is comprised of sperm produced in the testis and seminal fluid primarily produced in the male accessory glands (AG). These complementary components are both critical for male reproductive success, but they are largely considered to be functionally, genetically, and developmentally independent. In a quest to understand genetic mechanisms of sperm length variation, we found that genes upregulated in long sperm testes are enriched for lncRNAs and seminal fluid proteins (Sfps). Knockout of two Sfps, Sex Peptide and ovulin, results in shorter sperm, though knockout of AG function has no effect. Moreover, long sperm males delay female remating longer. These results suggest sophisticated testis-AG coordination that amplifies male reproductive success, with implications for evolutionary integration of sexually selected traits.</p></sec>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.19.21266532', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Monitoring of SARS-CoV-2 Antibodies Using Dried Blood Spot for At-Home Collection', 'authors': \"[{'name': 'Peyton K. Miesse'}, {'name': 'Bradley B. Collier'}, {'name': 'Russell P. Grant'}]\", 'abstract': '<p>The utilization of vaccines to fight the spread of SARS-CoV-2 has led to a growing need for expansive serological testing. To address this, an EUA approved immunoassay for detection of antibodies to SARS-CoV-2 in venous serum samples was investigated for use with dried blood spot (DBS) samples. Results from self-collected DBS samples demonstrated a 98.1% categorical agreement to venous serum with a correlation (R) of 0.9600 while professionally collected DBS samples demonstrated a categorical agreement of 100.0% with a correlation of 0.9888 to venous serum. Additional studies were performed to stress aspects of at-home DBS collection, including shipping stability, interference effects, and other sample-specific robustness studies. These studies demonstrated a categorical agreement of at least 95.0% and a mean bias less than ±20.0%. Furthermore, the ability to track antibody levels following vaccination with the BioNTech/Pfizer vaccine was demonstrated with self-collected DBS samples from pre-dose (Day 0) out to 19 weeks.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.15.468665', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Random attention can explain apparent object choice behavior in free-walking blowflies', 'authors': \"[{'name': 'José Monteagudo'}, {'name': 'Martin Egelhaaf'}, {'name': 'Jens Peter Lindemann'}]\", 'abstract': '<p>Flies are often observed to approach dark objects. To a naïve observer they seem to pay selective attention to one out of several objects although previous research identified a reflex-like fixation behavior integrating responses to all objects as possible underlying mechanism. In a combination of behavioral experiments and computational modelling, we investigate the choice behavior of flies freely walking towards an arrangement of two objects placed at a variable distance from each other. The walking trajectories are oriented towards one of the objects much earlier than predicted by a simple reactive model. We show that object choice can be explained by a continuous control scheme in combination with a mechanism randomly responding to the position of each object according to a stochastic process. Although this may be viewed as a special form of an attention-like mechanism, the model does not require an explicit decision mechanism or a memory for the drawn decision.</p><sec><title>Summary Statement</title><p>Walking blowflies apparently choose one of two objects to approach. In model simulations, a fixation scheme combined with random attention replicates this behavior without an explicit decision mechanism.</p></sec>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.16.468795', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Stimulation of immunity-linked genes by membrane disruption is linked to Golgi function and the ARF-1 GTPase', 'authors': \"[{'name': 'Christofer M. Welsh'}, {'name': 'Lorissa J. Smulan'}, {'name': 'Matthew J. Fanelli'}, {'name': 'Dominique S. Lui'}, {'name': 'Amy K. Walker'}]\", 'abstract': '<p>Immune-linked genes (ILGs) are activated in response to pathogens but can also be activated by lipid imbalance. Why pathogen attack and metabolic changes both impact ILG activation is unclear. Organelles in the secretory pathway have distinct protein and lipid components and genetically separable stress programs. These stress pathways activate restorative transcriptional programs when lipid ratios become unbalanced or during dysregulated protein folding and trafficking. We find that ILGs are specifically activated when membrane phosphatidylcholine ratios change in the secretory pathway. Consistent with this result, disruption of Golgi function after RNAi targeting the ADP-ribosylation factor ARF-1 also activates ILG expression. Since increased protein secretion is altered by metabolic changes and pathogen responses, our data argue that ILG upregulation is a conserved, coordinated response to changes in trafficking resulting from intrinsic cues (lipid changes) or extrinsic stimulation (during the immune response). These findings uncover important and previously unexplored links between metabolism and the stress response.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.19.469337', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Nonenzymatic loop-closing ligation generates RNA hairpins and is a template-free way to assemble functional RNAs', 'authors': \"[{'name': 'Long-Fei Wu'}, {'name': 'Ziwei Liu'}, {'name': 'Samuel J Roberts'}, {'name': 'Meng Su'}, {'name': 'Jack W Szostak'}, {'name': 'John D Sutherland'}]\", 'abstract': '<p>RNA hairpin loops are the predominant element of secondary structure in functional RNAs. The emergence of primordial functional RNAs, such as ribozymes that fold into complex structures that contain multiple hairpin loops, is generally thought to have been supported by template-directed ligation. However, template inhibition and RNA misfolding problems impede the emergence of function. Here we demonstrate that RNA hairpin loops can be synthesized directly from short RNA duplexes with single-stranded overhangs by nonenzymatic loop-closing ligation chemistry. We show that loop-closing ligation allows full-length functional ribozymes containing a hairpin loop to be assembled free of inhibitory template strands. This approach to the assembly of structurally complex RNAs suggests a plausible pathway for the emergence of functional RNAs before a full-length RNA copying process became available.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.17.21266051', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'The unmitigated profile of COVID-19 infectiousness', 'authors': \"[{'name': 'Ron Sender'}, {'name': 'Yinon M. Bar-On'}, {'name': 'Sang Woo Park'}, {'name': 'Elad Noor'}, {'name': 'Jonathan Dushoffd'}, {'name': 'Ron Milo'}]\", 'abstract': '<p>Quantifying the temporal dynamics of infectiousness of individuals infected with SARS-CoV-2 is crucial for understanding the spread of the COVID-19 pandemic and for analyzing the effectiveness of different mitigation strategies. Many studies have tried to use data from the onset of symptoms of infector-infectee pairs to estimate the infectiousness profile of SARS-CoV-2. However, both statistical and epidemiological biases in the data could lead to an underestimation of the duration of infectiousness. We correct for these biases by curating data from the initial outbreak of the pandemic in China (when mitigation steps were still minimal), and find that the infectiousness profile is wider than previously thought. For example, our estimate for the proportion of transmissions occurring 14 days or more after infection is an order of magnitude higher - namely 19% (95% CI 10%-25%). The inferred generation interval distribution is sensitive to the definition of the period of unmitigated transmission, but estimates that rely on later periods are less reliable due to intervention effects. Nonetheless, the results are robust to other factors such as the model, the assumed growth rate and possible bias of the dataset. Knowing the unmitigated infectiousness profile of infected individuals affects estimates of the effectiveness of self-isolation and quarantine of contacts. The framework presented here can help design better quarantine policies in early stages of future epidemics using data from the initial stages of transmission.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468788', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The sleep EEG envelope: a novel, neuronal firing-based human biomarker', 'authors': \"[{'name': 'Peter Przemyslaw Ujma'}, {'name': 'Martin Dresler'}, {'name': 'Peter Simor'}, {'name': 'Daniel Fabo'}, {'name': 'Istvan Ulbert'}, {'name': 'Lorand Eross'}, {'name': 'Robert Bodizs'}]\", 'abstract': '<p>Sleep EEG reflects instantaneous voltage differences relative to a reference, while its spectrum reflects the degree to which it is comprised of oscillations at various frequencies. In contrast, the envelope of the sleep EEG reflects the instantaneous amplitude of oscillations at specific frequencies, and its spectrum reflects the rhythmicity of the occurrence of these oscillations. The ordinary sleep EEG and its spectrum have been extensively studied and its individual stability and relationship to various demographic characteristics, psychological traits and pathologies is well known. In contrast, the envelope spectrum has not been extensively studied before. In two studies, we explored the generating mechanisms and utility of studying the envelope of the sleep EEG. First, we used human invasive data from cortex-penetrating microelectrodes and subdural grids to demonstrate that the sleep EEG envelope spectrum reflects local neuronal firing. Second, we used a large database of healthy volunteers to demonstrate that the scalp EEG envelope spectrum is highly stable within individuals, especially in NREM sleep, and that it is affected by age and sex. Multivariate models based on a learning algorithm could predict both age (r=0.6) and sex (r=0.5) with considerable accuracy from the EEG envelope spectrum. With age, oscillations characteristically shifted from a 4-5 second rhythm to higher rhythms. The envelope spectrum was not associated with general cognitive ability (IQ). Our results demonstrate that the sleep envelope spectrum is a promising, neuronal firing-based biomarker of various demographic and disease-related phenotypes.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.19.469321', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': \"The proton motive force determines Escherichia coli's robustness to extracellular pH\", 'authors': \"[{'name': 'Guillaume Terradot'}, {'name': 'Ekaterina Krasnopeeva'}, {'name': 'Peter S. Swain'}, {'name': 'Teuta Pilizota'}]\", 'abstract': \"<p>Maintaining intracellular homeostases is a hallmark of life, and key physiological variables, such as cytoplasmic pH, osmotic pressure, and proton motive force (PMF), are typically interdependent. Developing a mathematical model focused on these links, we predict that Escherichia coli uses proton-ion antiporters to generate an out-of-equilibrium plasma membrane potential and so maintain the PMF at the constant levels observed. The strength of the PMF consequently determines the range of extracellular pH over which the cell is able to preserve its near neutral cytoplasmic pH. In support, we concurrently measure the PMF and cytoplasmic pH in single cells and demonstrate both that decreasing the PMF's strength impairs E. coli's ability to maintain its pH and that artificially collapsing the PMF destroys the out-of-equilibrium plasma membrane potential. We further predict the observed ranges of extracellular pH for which three of E. coli's antiporters are expressed, through defining their cost by the rate at which they divert imported protons from generating ATP. Taken together, our results suggest a new perspective on bacterial electrophysiology, where cells regulate the plasma membrane potential by changing the activities of antiporters to maintain both the PMF and cytoplasmic pH.</p>\", 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.18.469049', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Prostaglandin PGE2 receptor EP4 regulates microglial phagocytosis and increases susceptibility to diet-induced obesity', 'authors': \"[{'name': 'Anzela Niraula'}, {'name': 'Rachael D Fasnacht'}, {'name': 'Kelly M Ness'}, {'name': 'Jeremy M Frey'}, {'name': 'Mauricio D Dorfman'}, {'name': 'Joshua P Thaler'}]\", 'abstract': '<p>Background: In rodents, susceptibility to diet-induced obesity requires microglial activation, but the molecular components of this pathway remain incompletely defined. Prostaglandin E2 (PGE2) levels increase in the mediobasal hypothalamus during high fat diet (HFD) feeding, and the PGE2 receptor EP4 regulates microglial activation state and phagocytic activity, suggesting a potential role for microglial EP4 signaling in obesity pathogenesis. Method: Metabolic phenotyping, as assessed by body weight, energy expenditure, glucose, and insulin tolerance, was performed in microglia-specific EP4 knockout (MG-EP4 KO) mice and littermate controls on HFD. Morphological and gene expression analysis of microglia, and a histological survey of microglia-neuron interactions in the arcuate nucleus was performed. Phagocytosis was assessed using in vivo and in vitro pharmacological techniques. Results: Microglial EP4 deletion markedly reduced weight gain and food intake in response to HFD feeding. In correspondence with this lean phenotype, insulin sensitivity was also improved in the HFD-fed MG-EP4 KO mice though glucose tolerance remained surprisingly unaffected. Mechanistically, EP4-deficient microglia showed an attenuated phagocytic state marked by reduced CD68 expression and fewer contacts with POMC neuron soma and processes. These cellular changes observed in the microglial EP4 knockout mice corresponded with an increased density of POMC neurites extending into the paraventricular nucleus. Conclusion: These findings reveal that microglial EP4 signaling promotes body weight gain and insulin resistance during HFD feeding. Furthermore, the data suggest that curbing microglial phagocytic function may preserve POMC cytoarchitecture and PVN input to limit overconsumption during diet-induced obesity.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.10.21266197', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Multi-modal investigation of the schizophrenia-associated 3q29 genomic interval reveals global genetic diversity with unique haplotypes and segments that increase the risk for non-allelic homologous recombination', 'authors': \"[{'name': 'Feyza Yilmaz'}, {'name': 'Umamaheswaran Gurusamy'}, {'name': 'Trenell J. Mosley'}, {'name': 'Yulia Mostovoy'}, {'name': 'Tamim H. Shaikh'}, {'name': 'Michael E. Zwick'}, {'name': 'Pui-Yan Kwok'}, {'name': 'Charles Lee'}, {'name': 'Jennifer G. Mulle'}]\", 'abstract': '<p>Chromosomal rearrangements that alter the copy number of dosage-sensitive genes can result in genomic disorders, such as the 3q29 deletion syndrome. At the 3q29 region, non-allelic homologous recombination (NAHR) between paralogous copies of segmental duplications (SDs) leads to a recurrent ∼1.6 Mbp deletion or duplication, causing neurodevelopmental and psychiatric phenotypes. However, risk factors contributing to NAHR at this locus are not well understood. In this study, we used an optical mapping approach to identify structural variations within the 3q29 interval. We identified 18 novel haplotypes among 161 unaffected individuals and used this information to characterize this region in 18 probands with either the 3q29 deletion or 3q29 duplication syndrome. A significant amount of variation in haplotype prevalence was observed between populations. Within probands, we narrowed down the breakpoints to a ∼5 kbp segment within the SD blocks in 89% of the 3q29 deletion and duplication cases studied. Furthermore, all 3q29 deletion and duplication cases could be categorized into one of five distinct classes based on their breakpoints. Contrary to previous findings for other recurrent deletion and duplication loci, there was no evidence for inversions in either parent of the probands mediating the deletion or duplication seen in this syndrome.</p>', 'posted': '2021-11-15'}, {'doi': '10.1101/2021.11.17.469067', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'LiveCellMiner: A New Tool to Analyze Mitotic Progression', 'authors': \"[{'name': 'Daniel Moreno-Andrés'}, {'name': 'Anuk Bhattacharyya'}, {'name': 'Anja Scheufen'}, {'name': 'Johannes Stegmaier'}]\", 'abstract': '<p>Live-cell imaging has become state of the art to accurately identify the nature of mitotic and cell cycle defects. Low- and high-throughput microscopy setups have yield huge data amounts of cells recorded in different experimental and pathological conditions. Tailored semi-automated and automated image analysis approaches allow the analysis of high-content screening data sets, saving time and avoiding bias. However, they were mostly designed for very specific experimental setups, which restricts their flexibility and usability. The general need for dedicated experiment-specific user-annotated training sets and experiment-specific user-defined segmentation parameters remains a major bottleneck for fully automating the analysis process. In this work we present LiveCellMiner, a highly flexible open-source software tool to automatically extract, analyze and visualize both aggregated and time-resolved image features with potential biological relevance. The software tool allows analysis across high-content data sets obtained in different platforms, in a quantitative and unbiased manner. As proof of principle application, we analyze here the dynamic chromatin and tubulin cytoskeleton features in human cells passing through mitosis highlighting the versatile and flexible potential of this tool set.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468841', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Stimulus-induced narrow-band gamma oscillations in humans can be recorded using open-hardware low-cost EEG amplifier', 'authors': \"[{'name': 'Srividya Pattisapu'}, {'name': 'Supratim Ray'}]\", 'abstract': '<p>Stimulus-induced narrow-band gamma oscillations (30-70 Hz) in human electro - encephalograph (EEG) have been linked to attentional and memory mechanisms and are abnormal in mental health conditions such as autism, schizophrenia and Alzheimer\\'s Disease. This suggests that gamma oscillations could be valuable both as a research tool and an inexpensive, non-invasive biomarker for disease evaluation. However, since the absolute power in EEG decreases rapidly with increasing frequency following a \"1/f\" power law, and the gamma band includes line noise frequency, these oscillations are highly susceptible to instrument noise. Previous studies that recorded stimulus-induced gamma oscillations used expensive research-grade EEG amplifiers to address this issue. While low-cost EEG amplifiers have become popular in Brain Computer Interface applications that mainly rely on low-frequency oscillations (&lt;30 Hz) or steady-state-visually-evoked-potentials, whether they can also be used to measure stimulus-induced gamma oscillations is unknown. We recorded EEG signals using a low-cost, open-source amplifier (OpenBCI) and a traditional, research-grade amplifier (Brain Products GmbH) in male (N = 6) and female (N = 5) subjects (22-29 years) while they viewed full-screen static gratings that are known to induce gamma oscillations. OpenBCI recordings showed gamma response in almost all the subjects who showed a gamma response in Brain Products recordings, and the spectral and temporal profiles of these responses in alpha (8-13 Hz) and gamma bands were highly correlated between OpenBCI and Brain Products recordings. These results suggest that low-cost amplifiers can potentially be used in stimulus induced gamma response detection, making its research, and application in medicine more accessible.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.20.469222', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Epigenetic changes in autoimmune monocytes contribute to disease and can be targeted by JAK inhibition', 'authors': \"[{'name': 'Jorg van Loosdregt'}, {'name': 'Janneke Peeters'}, {'name': 'Arjan Boltjes'}, {'name': 'Rianne Scholman'}, {'name': 'Stephin Vervoort'}, {'name': 'Paul Coffer'}, {'name': 'Michal Mokry'}, {'name': 'Bas Vastert'}, {'name': 'Femke van Wijk'}]\", 'abstract': '<p>How the local inflammatory environment regulates epigenetic changes in the context of autoimmune diseases remains unclear. Here we assessed the transcriptional and active enhancer profile of monocytes derived from the inflamed joints of Juvenile Idiopathic Arthritis (JIA) patients, a model well-suited for studying autoimmune diseases. RNA-sequencing analysis demonstrated that synovial-derived monocytes display an activated phenotype, characterized by increased expression of surface activation markers and multiple chemokines and cytokines. This increased expression is regulated on the epigenetic level, as ChIPsequencing indicated increased H3K27 acetylation of genes upregulated in synovial monocytes. IFN signaling-associated genes are increased and epigenetically altered in synovial monocytes, suggesting a role for IFN in establishing the inflammatory phenotype. Treatment of human synovial monocytes with the Janus-associated kinase (JAK) inhibitor Ruxolitinib, which inhibits IFN signaling, transformed the activated enhancer landscape and reduced disease-associated gene expression, thereby inhibiting the inflammatory phenotype. Taken together, these data provide novel insight into epigenetic regulation of autoimmune disease-patient derived monocytes and suggest that altering the epigenetic profile can be a therapeutic approach for the treatment of autoimmune diseases.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.16.468817', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Host casein kinase 1-mediated phosphorylation modulates phase separation of a rhabdovirus phosphoprotein and virus infection', 'authors': \"[{'name': 'Xiao-Dong Fang'}, {'name': 'Qiang Gao'}, {'name': 'Ying Zang'}, {'name': 'Ji-Hui Qiao'}, {'name': 'Dong-Min Gao'}, {'name': 'Wen-Ya Xu'}, {'name': 'Ying Wang'}, {'name': 'Dawei Li'}, {'name': 'Xian-Bing Wang'}]\", 'abstract': '<p>Liquid–liquid phase separation (LLPS) plays important roles in forming cellular membraneless organelles. However, how host factors regulate LLPS of viral proteins during negative-sense RNA (NSR) virus infections is largely unknown. Here, we used <italic>Barley yellow striate mosaic virus</italic> (BYSMV) as a model to demonstrate regulation of host casein kinase 1 in phase separation and infection of NSR viruses. We first found that the BYSMV phosphoprotein (P) formed spherical granules with liquid properties and recruited viral nucleotide (N) and polymerase (L) proteins <italic>in vivo</italic>. Moreover, the P-formed granules were tethered to the ER/actin network for trafficking and fusion. BYSMV P alone formed droplets and incorporated the N protein and genomic RNA <italic>in vitro</italic>. Interestingly, phase separation of BYSMV P was inhibited by host casein kinase 1 (CK1)-dependent phosphorylation of an intrinsically disordered P protein region. Genetic assays demonstrated that the unphosphorylated mutant of BYSMV P exhibited condensed phase, which promoted virus replication through concentrating the N, L proteins, and genome RNA into viroplasms. Whereas, the phosphorylation-mimic mutant existed in diffuse phase state leading to enhanced virus transcription. Collectively, our results demonstrate that host CK1 modulates phase separation of viral P protein and virus infection.</p>', 'posted': '2021-11-16'}, {'doi': '10.1101/2021.11.21.469448', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Motor plans under uncertainty reflect a trade-off between maximizing reward and success', 'authors': \"[{'name': 'Aaron L Wong'}, {'name': 'Audrey L Green'}, {'name': 'Mitchell W Isaacs'}]\", 'abstract': '<p>When faced with multiple potential movement options, individuals either reach directly to one of the options, or initiate a reach intermediate between the options. It remains unclear why people generate these two types of behaviors. Using the go-before-you-know task (commonly used to study behavior under choice uncertainty), we examined two key questions. First, do these two types of responses reflect distinct movement strategies, or are they simply examples of a more general response to choice uncertainty? If the former, the relative desirability (i.e., weighing the likelihood of successfully hitting the target versus the attainable reward) of the two target options might be computed differently for direct versus intermediate reaches. We showed that indeed, when exogenous reward and success likelihood (i.e., endogenous reward) differ between the two options, direct reaches were more strongly biased by likelihood whereas intermediate movements were more strongly biased by reward. Second, what drives individual differences in how people respond under uncertainty? We found that risk/reward-seeking individuals generated a larger proportion of intermediate reaches and were more sensitive to trial-to-trial changes in reward, suggesting these movements reflect a strategy to maximize reward. In contrast, risk-adverse individuals tended to generate more direct reaches in an attempt to maximize success. Together, these findings suggest that when faced with choice uncertainty, individuals adopt movement strategies consistent with their risk/reward-seeking tendency, preferentially biasing behavior toward exogenous rewards or endogenous success and consequently modulating the relative desirability of the available options.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.21.469446', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Transient inhibition to light explains stronger V1 responses to dark stimuli', 'authors': \"[{'name': 'David St-Amand'}, {'name': 'Curtis L Baker'}]\", 'abstract': '<p>Neurons in the primary visual cortex (V1) receive excitation and inhibition from two different pathways processing lightness (ON) and darkness (OFF). V1 neurons overall respond more strongly to dark than light stimuli (Yeh, Xing and Shapley, 2010; Kremkow et al., 2014), consistent with a preponderance of darker regions in natural images (Ratliff et al., 2010), as well as human psychophysics (Buchner &amp; Baumgartner, 2007). However, it has been unclear whether this \"dark-dominance\" is due to more excitation from the OFF pathway (Jin et al., 2008) or more inhibition from the ON pathway (Taylor et al., 2018). To understand the mechanisms behind dark-dominance, we record electrophysiological responses of individual simple-type V1 neurons to natural image stimuli and then train biologically inspired convolutional neural networks to predict the neuronal responses. Analyzing a sample of 74 neurons (in anesthetized, paralyzed cats) has revealed their responses to be more driven by dark than light stimuli, consistent with previous investigations (Yeh et al., 2010; Kremkow et al., 2013). We show this asymmetry to be predominantly due to slower inhibition to dark stimuli rather than by stronger excitation from the thalamocortical OFF pathway. Consistent with dark-dominant neurons having faster responses than light-dominant neurons (Komban et al., 2014), we find dark-dominance to solely occur in the early latencies of neuronal responses. Neurons that are strongly dark-dominated also tend to be less orientation selective. This novel approach gives us new insight into the dark-dominance phenomenon and provides an avenue to address new questions about excitatory and inhibitory integration in cortical neurons.</p>', 'posted': '2021-11-21'}, {'doi': '10.1101/2021.11.16.468897', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Effect of RNA preservation methods on RNA quantity and quality of field collected avian whole blood', 'authors': \"[{'name': 'Johanna A Harvey'}, {'name': 'Sarah A Knutie'}]\", 'abstract': '<p>A limitation of comparative transcriptomic studies of wild avian populations continues to be sample acquisition and preservation to achieve resulting high-quality RNA (i.e., ribonucleic acids that transfers, translates, and regulates the genetic code from DNA into proteins). Field sampling of wild bird samples provides challenges as RNA degradation progresses quickly and because cryopreservation is often not feasible at remote locations. We collected blood samples from songbirds, as avian blood is nucleated and provides sufficient transcriptionally active material in a small and non-lethal sample, to compare the efficacy of widely available RNA stabilizing buffers, RNAlater (Ambion) and DNA/RNA Shield (Zymo) at differing concentrations along with a dry ice-based flash freezing method (Isopropanol 99% and dry ice mixture, -109°C). Each blood sample was divided among five different preservation treatments (dry ice-based flash freezing, RNAlater with 1:5 or 1:10 dilution, or DNA/RNA Shield with 1:2 or 1:3 dilution). A new protocol was optimized for total RNA extraction from avian blood samples with small starting volumes enabling sampling of small passerines. We quantified quality measures, RNA integrity numbers (RINe), rRNA ratios, and total RNA concentrations. We found that RNA preservation buffers, RNAlater and DNA/RNA Shield at all concentrations, provide sample protection from RNA degradation. We suggest caution against using dry ice-based flash-freezing alone for samples preservation as these samples resulted in lower quality measures then samples in preservation buffer. Total RNA concentration was generally not affected by preservation treatment and may vary due to differences in initial samples volumes and carryover across processing steps.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.16.468878', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'The evidence for anthocyanins in the betalain-pigmented genus Hylocereus is weak', 'authors': \"[{'name': 'Boas Pucker'}, {'name': 'Samuel F. Brockington'}]\", 'abstract': '<p>Here we respond to Zhou et al., 2020 “Combined Transcriptome and Metabolome analysis of Pitaya fruit unveiled the mechanisms underlying Peel and pulp color formation” published in BMC Genomics. Given the evolutionary conserved anthocyanin biosynthesis pathway in betalain-pigmented species, we are open to the idea that species with both anthocyanins and betalains might exist. However, in absence of LC-MS/MS spectra, apparent lack of biological replicates, and no comparison to authentic standards, the findings of Zhou et al., 2020 are not a strong basis to propose the presence of anthocyanins in betalain-pigmented pitaya. In addition, our re-analysis of the datasets indicates the misidentification of important genes and the omission of key anthocyanin synthesis genes <italic>ANS</italic> and <italic>DFR</italic>. Finally, our re-analysis of the RNA-Seq dataset reveals no correlation between anthocyanin biosynthesis gene expression and pigment status.</p>', 'posted': '2021-11-19'}, {'doi': '10.1101/2021.11.20.468818', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Transcriptome and fatty-acid signatures of adipocyte hypertrophy and its non-invasive MR-based characterization in human adipose tissue', 'authors': \"[{'name': 'Julius Honecker'}, {'name': 'Stefan Ruschke'}, {'name': 'Claudine Seeliger'}, {'name': 'Samantha Laber'}, {'name': 'Sophie Strobel'}, {'name': 'Priska Proell'}, {'name': 'Christoffer Nellaker'}, {'name': 'Cecilia Lindgren'}, {'name': 'Ulrich Kulozik'}, {'name': 'Josef Ecker'}, {'name': 'Dimitrios Karampinos'}, {'name': 'Melina Claussnitzer'}, {'name': 'Hans Hauner'}]\", 'abstract': '<p>Adipose tissue is an organ with great plasticity and its hypertrophic expansion is associated with adipocyte dysfunction. How changes in adipocyte morphology are linked to gene expression and which cellular functions are affected remains elusive. We show that adipocyte hypertrophy is associated with transcriptomic changes using RNA-Seq data obtained from adipose tissue and size-separated adipocytes. Genes involved in oxidative phosphorylation, protein biosynthesis and fatty acid metabolism were down-regulated in large adipocytes while genes involved in inflammation were upregulated. For mitochondrial function, a reduction in the expression of thermogenesis related genes and estimated brown/beige adipocyte content was observed in individuals with large adipocytes. As a novel finding the total adipose tissue fatty acid composition was dependent on cell size and depot. MR spectroscopy methods for clinical scanning were developed to characterize adipocyte size and fatty acid composition in a fast and non-invasive manner. Together, the present data provides mechanistic insights on how adipocyte hypertrophy contributes to the manifestation of metabolic disease at the molecular and cellular level. MR spectroscopy was identified as a promising technique for an in-parallel assessment of adipose morphology and fatty acid composition allowing to translate our findings into an improved, non-invasive phenotyping of adipose tissue.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.19.469268', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'A Comparison Study between Autism Spectrum Disorder and Typically Control in Graph Frequency Bands Using Graph and Triadic Interaction Metrics', 'authors': \"[{'name': 'Alireza Talesh Jafadideh'}, {'name': 'Babak Mohammadzadeh Asl'}]\", 'abstract': '<p>Many researchers using many different approaches have attempted to find features discriminating between autism spectrum disorder (ASD) and typically control (TC) subjects. In this study, this attempt has been continued by analyzing global metrics of functional graphs and metrics of functional triadic interactions of the brain in the low, middle, and high-frequency bands (LFB, MFB, and HFB) of the structural graph. The graph signal processing (GSP) provided the combinatorial usage of the functional graph of resting-state fMRI and structural graph of DTI. In comparison to TCs, ASDs had significantly higher clustering coefficients in the MFB, higher efficiencies and strengths in the MFB and HFB, and lower small-world propensity in the HFB. These results show over-connectivity, more global integration, and probably decreased local specialization in ASDs compared to TCs. Triadic analysis showed that the numbers of unbalanced triads were significantly lower for ASDs in the MFB. This finding may show the reason for restricted and repetitive behavior in ASDs. Also, in the MFB and HFB, the numbers of balanced triads and the energies of triadic interactions were significantly higher and lower for ASDs, respectively. These findings may reflect the disruption of the optimum balance between functional integration and specialization. All of these results demonstrated that the significant differences between ASDs and TCs existed in the MFB and HFB of the structural graph when analyzing the global metrics of the functional graph and triadic interaction metrics. In conclusion, the results demonstrate the promising perspective of GSP for attaining discriminative features and new knowledge, especially in the case of ASD.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.19.469302', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Heterogeneous run-and-tumble motion accounts for transient non-Gaussian super-diffusion in haematopoietic multi-potent progenitor cells', 'authors': \"[{'name': 'Benjamin Partridge'}, {'name': 'Sara Gonzalez Anton'}, {'name': 'Reema Khorshed'}, {'name': 'George Adams'}, {'name': 'Constandina Pospori'}, {'name': 'Cristina Lo Celso'}, {'name': 'Chiu Fan Lee'}]\", 'abstract': '<p>Multi-potent progenitor (MPP) cells act as a key intermediary step between haematopoietic stem cells and the entirety of the mature blood cell system. Their eventual fate determination is thought to be achieved through migration in and out of spatially distinct niches. Here we first analyze statistically MPP cell trajectory data obtained from a series of long time-course 3D in-vivo imaging experiments on irradiated mouse calvaria, and report that MPPs display transient super-diffusion with apparent non-Gaussian displacement distributions. Second, we explain these experimental findings using a run-and-tumble model of cell motion which incorporates the observed dynamical heterogeneity of the MPPs. Third, we use our model to extrapolate the dynamics to time-periods currently inaccessible experimentally,  which enables us to quantitatively estimate the time and length scales at which super-diffusion transitions to Fickian diffusion. Our work sheds light on the potential importance of motility in early haematopoietic progenitor function.</p>', 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.19.469265', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Partial prevention of glucocorticoid-induced osteocyte deterioration with osteocrin gene therapy', 'authors': \"[{'name': 'Courtney M Mazur'}, {'name': 'Christian D Castro Andrade'}, {'name': 'Tadatoshi Sato'}, {'name': 'Michael Bruce'}, {'name': 'Mary L Bouxsein'}, {'name': 'Jialiang S Wang'}, {'name': 'Marc N Wein'}]\", 'abstract': \"<p>Glucocorticoid (GC)-induced osteoporosis and subsequent bone fragility are preceded by death and dysfunction at the cellular level. In particular, short-term glucocorticoid excess suppresses osteocyte remodeling of the surrounding bone mineral, causes apoptosis of osteoblasts and osteocytes, and disrupts homeostatic bone remodeling. Preventing apoptosis and preserving osteocyte morphology and function could be effective means of preventing bone loss during glucocorticoid excess. We hypothesized that osteocrin, which preserves osteocyte viability and morphology in other models where osteocyte defects exist, could prevent osteocyte death and dysfunction in a GC excess model. We used a liver-targeted adeno-associated virus (AAV8) to induce osteocrin overexpression in mice one week prior to implantation with prednisolone or placebo pellets. After 28 days, tissues were collected for micro-CT and histological analysis. GC excess caused the expected reduction in cortical bone thickness and osteocyte canalicular length in control AAV8-treated mice, and these effects were blunted in mice overexpressing osteocrin. However, GC-induced changes in cortical porosity, trabecular bone mass, and gene expression were not prevented by osteocrin. While the mechanism of osteocrin's effects on osteocyte morphology warrants further investigation, this study does not support a role for this model of osteocrin supplementation to combat the full skeletal effects of GC excess.</p>\", 'posted': '2021-11-20'}, {'doi': '10.1101/2021.11.17.468889', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'MAPR origins reveal a new class of prokaryotic cytochrome b5 proteins and possible role in eukaryogenesis', 'authors': \"[{'name': 'Daniel Tamarit'}, {'name': 'Sarah Teakel'}, {'name': 'Michealla Marama'}, {'name': 'David Aragão'}, {'name': 'Svetlana Y. Gerdes'}, {'name': 'Jade K. Forwood'}, {'name': 'Thijs J. G. Ettema'}, {'name': 'Michael A. Cahill'}]\", 'abstract': '<p>The multiple functions of PGRMC1, the archetypal heme-binding eukaryotic MAPR family member, include steroidogenic regulation, membrane trafficking, and steroid responsiveness. The interrelationships between these functions are currently poorly understood. Previous work has shown that different MAPR subclasses were present early in eukaryotic evolution, and that tyrosine phosphorylated residues appeared in the eumetazoan ancestor, coincident with a gastrulation organizer. Here we show that MAPR proteins are related to a newly recognized class of prokaryotic cytochrome-b<sub>5</sub> domain proteins. Our first solved structure of this new class exhibits shared MAPR-like folded architecture and heme-binding orientation. We also report that a protein subgroup from Candidate Phyla Radiation bacteria shares MAPR-like heme-interacting tyrosines. Our results support bacterial origins for both PGRMC1 and CYP51A, that catalyze the meiosis-associated 14-demethylation of the first sterol lanosterol from yeast to humans. We propose that eukaryotic acquisition of a membrane-trafficking function related to sterol metabolism was associated with the appearance of MAPR genes early in eukaryotic evolution.</p>', 'posted': '2021-11-18'}, {'doi': '10.1101/2021.11.17.468905', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Gene expression based inference of drug resistance in cancer', 'authors': \"[{'name': 'Smriti Chawla'}, {'name': 'Anja Rockstroh'}, {'name': 'Melanie Lehman'}, {'name': 'Ellca Rather'}, {'name': 'Atishay Jain'}, {'name': 'Anuneet Anand'}, {'name': 'Apoorva Gupta'}, {'name': 'Namrata Bhattacharya'}, {'name': 'Sarita Poonia'}, {'name': 'Priyadarshini Rai'}, {'name': 'Nirjhar Das'}, {'name': 'Angshul Majumdar'}, {'name': 'Jayadeva Jayadeva'}, {'name': 'Gaurav Ahuja'}, {'name': 'Brett G Hollier'}, {'name': 'Colleen C Nelson'}, {'name': 'Debarka Sengupta'}]\", 'abstract': '<p>Inter and intra-tumoral heterogeneity are major stumbling blocks in the treatment of cancer and are responsible for imparting differential drug responses in cancer patients. Recently, the availability of large-scale drug screening datasets has provided an opportunity for predicting appropriate patient-tailored therapies by employing machine learning approaches. In this study, we report a predictive modeling approach to infer treatment response in cancers using gene expression data. In particular, we demonstrate the benefits of considering integrated chemogenomics approach, utilizing the molecular drug descriptors and pathway activity information as opposed to gene expression levels. We performed extensive validation of our approach on tissue-derived single-cell and bulk expression data. Further, we constructed several prostate cancer cell lines and xenografts, exposed to differential treatment conditions to assess the predictability of the outcomes. Our approach was further assessed on pan-cancer RNA-sequencing data from The Cancer Genome Atlas (TCGA) archives, as well as an independent clinical trial study describing the treatment journey of three melanoma patients. To summarise, we benchmarked the proposed approach on cancer RNA-seq data, obtained from cell lines, xenografts, as well as humans. We concluded that pathway-activity patterns in cancer cells are reasonably indicative of drug resistance, and therefore can be leveraged in personalized treatment recommendations.</p>', 'posted': '2021-11-19'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.11.13.468472</td>\n",
       "      <td>502</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pathology</td>\n",
       "      <td>True</td>\n",
       "      <td>A live attenuated influenza virus-vectored int...</td>\n",
       "      <td>[{'name': 'Junyu Chen'}, {'name': 'Pui Wang'},...</td>\n",
       "      <td>&lt;p&gt;Remarkable progress has been made in develo...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.11.18.469078</td>\n",
       "      <td>145</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Metformin Suppresses SARS-CoV-2 in Cell Culture</td>\n",
       "      <td>[{'name': 'Dixit Tandel'}, {'name': 'Haripriya...</td>\n",
       "      <td>&lt;p&gt;People with diabetes are reported to have a...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.11.18.469088</td>\n",
       "      <td>117</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Structural basis for Cas9 off-target activity</td>\n",
       "      <td>[{'name': 'Martin Pacesa'}, {'name': 'Chun-Han...</td>\n",
       "      <td>&lt;p&gt;The target DNA specificity of the CRISPR-as...</td>\n",
       "      <td>2021-11-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.11.19.469228</td>\n",
       "      <td>107</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Pre-metazoan origin of neuropeptide signalling</td>\n",
       "      <td>[{'name': 'Luis Alfonso Yanez-Guerra'}, {'name...</td>\n",
       "      <td>&lt;p&gt;Neuropeptides are a diverse class of signal...</td>\n",
       "      <td>2021-11-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.11.16.468777</td>\n",
       "      <td>99</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>The Delta variant of SARS-CoV-2 maintains high...</td>\n",
       "      <td>[{'name': 'Rayhane Nchioua'}, {'name': 'Annika...</td>\n",
       "      <td>&lt;p&gt;Interferons are a major part of the anti-vi...</td>\n",
       "      <td>2021-11-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.11.15.468391</td>\n",
       "      <td>97</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>An atypical NLR protein modulates the NRC immu...</td>\n",
       "      <td>[{'name': 'Hiroaki Adachi'}, {'name': 'Toshiyu...</td>\n",
       "      <td>&lt;p&gt;The NRC immune receptor network has evolved...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.11.14.21266295</td>\n",
       "      <td>93</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Estimation of R0 for the spread of SARS-CoV-2 ...</td>\n",
       "      <td>[{'name': 'Juan Pablo Prada'}, {'name': 'Luca ...</td>\n",
       "      <td>&lt;p&gt;For SARS-CoV-2, R0 calculations report usua...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.11.17.465477</td>\n",
       "      <td>84</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>kTMP: A New Non-invasive Magnetic Induction Me...</td>\n",
       "      <td>[{'name': 'Ludovica Labruna'}, {'name': 'Chris...</td>\n",
       "      <td>&lt;p&gt;We report a novel subthreshold non-invasive...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.11.18.469090</td>\n",
       "      <td>82</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Mechanics of cell integration in vivo</td>\n",
       "      <td>[{'name': 'Guilherme B. Ventura'}, {'name': 'A...</td>\n",
       "      <td>&lt;p&gt;During embryonic development, regeneration ...</td>\n",
       "      <td>2021-11-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.11.16.468622</td>\n",
       "      <td>80</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The evolution of genomic, transcriptomic, and ...</td>\n",
       "      <td>[{'name': 'Kentaro Ohara'}, {'name': 'Andrè Fi...</td>\n",
       "      <td>&lt;p&gt;The molecular characteristics of metastatic...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.11.17.468999</td>\n",
       "      <td>79</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Reversible bacteriophage resistance by sheddin...</td>\n",
       "      <td>[{'name': 'Veronique Ongenae'}, {'name': 'Adam...</td>\n",
       "      <td>&lt;p&gt;Phages are highly abundant in the environme...</td>\n",
       "      <td>2021-11-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.11.15.468686</td>\n",
       "      <td>71</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Evaluation of methods for the inference of anc...</td>\n",
       "      <td>[{'name': 'Débora Y. C. Brandt'}, {'name': 'Xi...</td>\n",
       "      <td>&lt;p&gt;The ancestral recombination graph (ARG) is ...</td>\n",
       "      <td>2021-11-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.11.18.469106</td>\n",
       "      <td>71</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Transgenic tools for targeted chromosome rearr...</td>\n",
       "      <td>[{'name': 'David L. Stern'}]</td>\n",
       "      <td>&lt;p&gt;Perhaps the most valuable single set of res...</td>\n",
       "      <td>2021-11-18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.11.15.468676</td>\n",
       "      <td>70</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Built on sand: the shaky foundations of simula...</td>\n",
       "      <td>[{'name': 'Helena L. Crowell'}, {'name': 'Sara...</td>\n",
       "      <td>&lt;p&gt;With the emergence of hundreds of single-ce...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.11.14.21265758</td>\n",
       "      <td>54</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Seroprevalence in Tamil Nadu through India’s t...</td>\n",
       "      <td>[{'name': 'T.S. Selvavinayagam'}, {'name': 'A....</td>\n",
       "      <td>&lt;p&gt;Three rounds of population-representative s...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.11.19.468202</td>\n",
       "      <td>53</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Smart soils to observe hidden rhizosphere proc...</td>\n",
       "      <td>[{'name': 'Daniel Patko'}, {'name': 'Qizhi Yan...</td>\n",
       "      <td>&lt;p&gt;Agriculture must reduce green-house gas emi...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.11.15.468761</td>\n",
       "      <td>50</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Microglia do not restrict SARS-CoV-2 replicati...</td>\n",
       "      <td>[{'name': 'Gema M. Olivarria'}, {'name': 'Yuti...</td>\n",
       "      <td>&lt;p&gt;Unlike SARS-CoV-1 and MERS-CoV, infection w...</td>\n",
       "      <td>2021-11-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.11.15.468729</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Physiology</td>\n",
       "      <td>False</td>\n",
       "      <td>NAD precursors cycle between host tissues and ...</td>\n",
       "      <td>[{'name': 'Karthikeyani Chellappa'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Nicotinamide adenine dinucleotide (NAD) is ...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.11.14.21266328</td>\n",
       "      <td>41</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Human genetic evidence supports MAP3K15 inhibi...</td>\n",
       "      <td>[{'name': 'Abhishek Nag'}, {'name': 'Ryan S. D...</td>\n",
       "      <td>&lt;p&gt;Diabetes mellitus is a chronic health condi...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.11.15.21266187</td>\n",
       "      <td>41</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>Controlling SARS-CoV-2 in schools using repeti...</td>\n",
       "      <td>[{'name': 'Andrea Torneri'}, {'name': 'Lander ...</td>\n",
       "      <td>&lt;p&gt;SARS-CoV-2 remains a worldwide emergency. W...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.11.17.469009</td>\n",
       "      <td>39</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Not so fast: Limited validity of deep convolut...</td>\n",
       "      <td>[{'name': 'Guo Jiahui'}, {'name': 'Ma Feilong'...</td>\n",
       "      <td>&lt;p&gt;Deep convolutional neural networks (DCNNs) ...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.11.18.468552</td>\n",
       "      <td>35</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pharmacology and Toxicology</td>\n",
       "      <td>False</td>\n",
       "      <td>Proteolysis Targeting Chimeras With Reduced Of...</td>\n",
       "      <td>[{'name': 'Tuan M Nguyen'}, {'name': 'Arghya D...</td>\n",
       "      <td>&lt;p&gt;Proteolysis Targeting Chimeras (PROTACs), a...</td>\n",
       "      <td>2021-11-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.11.14.468535</td>\n",
       "      <td>35</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Rapid reconstruction of neural circuits using ...</td>\n",
       "      <td>[{'name': 'Joshua L. Lillvis'}, {'name': 'Hide...</td>\n",
       "      <td>&lt;p&gt;Electron microscopy (EM) allows for the rec...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.11.17.469005</td>\n",
       "      <td>33</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Robust temporal map of human in vitro myelopoi...</td>\n",
       "      <td>[{'name': 'Clara Alsinet'}, {'name': 'Maria Pr...</td>\n",
       "      <td>&lt;p&gt;Myeloid cells have a central role in homeos...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.11.15.468770</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Comparative genomics on cultivated and unculti...</td>\n",
       "      <td>[{'name': 'Hang Yu'}, {'name': 'Grayson L. Cha...</td>\n",
       "      <td>&lt;p&gt;Chemolithoautotrophic manganese oxidation h...</td>\n",
       "      <td>2021-11-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.11.18.469135</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Automated assembly scaffolding elevates a new ...</td>\n",
       "      <td>[{'name': 'Michael Alonge'}, {'name': 'Ludivin...</td>\n",
       "      <td>&lt;p&gt;Advancing crop genomics requires efficient ...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.11.19.21265383</td>\n",
       "      <td>29</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>GWAS defines pathogenic signaling pathways and...</td>\n",
       "      <td>[{'name': 'Krzysztof Kiryluk'}, {'name': 'Elen...</td>\n",
       "      <td>&lt;p&gt;IgA nephropathy (IgAN) is a progressive for...</td>\n",
       "      <td>2021-11-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.11.15.468712</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Structures of the Cyanobacterial Phycobilisome</td>\n",
       "      <td>[{'name': 'Paul V. Sauer'}, {'name': 'Maria Ag...</td>\n",
       "      <td>&lt;p&gt;The phycobilisome is an elaborate antenna t...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.11.20.468884</td>\n",
       "      <td>28</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>A glycerol shunt functions as a glucose excess...</td>\n",
       "      <td>[{'name': 'Anfal Al-Mass'}, {'name': 'Pegah Po...</td>\n",
       "      <td>&lt;p&gt;The recently identified glycerol-3-phosphat...</td>\n",
       "      <td>2021-11-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.11.19.21266552</td>\n",
       "      <td>27</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Low neutralizing antibody titers against the M...</td>\n",
       "      <td>[{'name': 'Diego A. Álvarez-Díaz'}, {'name': '...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Global survei...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.11.18.467545</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Slow fluctuations in ongoing brain activity de...</td>\n",
       "      <td>[{'name': 'Maria J. Ribeiro'}, {'name': 'Migue...</td>\n",
       "      <td>&lt;p&gt;In humans, ageing is characterized by decre...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.11.14.468560</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Shade suppresses wound-induced leaf reposition...</td>\n",
       "      <td>[{'name': 'Anne-Sophie Fiorucci'}, {'name': 'O...</td>\n",
       "      <td>&lt;p&gt;Shaded plants challenged with herbivores or...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.11.14.468537</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>Viral E Protein Neutralizes BET Protein-Mediat...</td>\n",
       "      <td>[{'name': 'Irene P. Chen'}, {'name': 'James E....</td>\n",
       "      <td>&lt;p&gt;Inhibitors of Bromodomain and Extra-termina...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.11.15.468735</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Modulation of Cutaneous Responses in the Cunea...</td>\n",
       "      <td>[{'name': 'Qinpu He'}, {'name': 'Christopher S...</td>\n",
       "      <td>&lt;p&gt;To achieve stable and precise movement exec...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.11.17.469032</td>\n",
       "      <td>25</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Anatomical connectivity along the anterior-pos...</td>\n",
       "      <td>[{'name': 'Marshall Axel Dalton'}, {'name': \"A...</td>\n",
       "      <td>&lt;p&gt;The hippocampus supports multiple cognitive...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.11.15.468719</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Structure of the Quenched Cyanobacterial OCP-P...</td>\n",
       "      <td>[{'name': 'Maria Agustina Dominguez-Martin'}, ...</td>\n",
       "      <td>&lt;p&gt;Photoprotection is an essential mechanism i...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.11.18.469072</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>scCODE: an R package for personalized differen...</td>\n",
       "      <td>[{'name': 'jiawei Zou'}, {'name': 'miaochen Wa...</td>\n",
       "      <td>&lt;p&gt;Differential expression (DE) gene detection...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.11.18.468666</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>PURC v2.0: a program for improved sequence inf...</td>\n",
       "      <td>[{'name': 'Peter W Schafran'}, {'name': 'Fay-W...</td>\n",
       "      <td>&lt;p&gt;Inferring the true biological sequences fro...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.11.15.468613</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Live imaging of the co-translational recruitme...</td>\n",
       "      <td>[{'name': 'Silvia Gómez-Puerta'}, {'name': 'Ro...</td>\n",
       "      <td>&lt;p&gt;Endoplasmic reticulum (ER) to nucleus homeo...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.11.15.468621</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Mobilome-driven segregation of the resistome i...</td>\n",
       "      <td>[{'name': 'Laura de Nies'}, {'name': 'Susheel ...</td>\n",
       "      <td>&lt;p&gt;Biological wastewater treatment plants (BWW...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.11.18.469161</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Indirect effects of parental conflict on consp...</td>\n",
       "      <td>[{'name': 'Jenn M Coughlan'}]</td>\n",
       "      <td>&lt;p&gt;Hybrid Seed Inviability (HSI) is a common b...</td>\n",
       "      <td>2021-11-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.11.15.468614</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Wolbachia endosymbionts in two Anopheles speci...</td>\n",
       "      <td>[{'name': 'Shannon Quek'}, {'name': 'Louise Ce...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Wolbachia&lt;/italic&gt; is a genus of ob...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.11.16.468548</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Reconstitution of Spiroplasma swimming by expr...</td>\n",
       "      <td>[{'name': 'Hana Kiyama'}, {'name': 'Shigeyuki ...</td>\n",
       "      <td>&lt;p&gt;Motility is one of the most important featu...</td>\n",
       "      <td>2021-11-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.11.12.468374</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Full protection against all four SARS-CoV-2 va...</td>\n",
       "      <td>[{'name': 'Sapna Sharma'}, {'name': 'Thomas Ve...</td>\n",
       "      <td>&lt;p&gt;Current licensed COVID-19 vaccines are base...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.11.16.467908</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Tuning spherical cells into kinking helices in...</td>\n",
       "      <td>[{'name': 'Carole Lartigue'}, {'name': 'Bastie...</td>\n",
       "      <td>&lt;p&gt;In bacteria, cell shape is determined and m...</td>\n",
       "      <td>2021-11-17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.11.17.469020</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pharmacology and Toxicology</td>\n",
       "      <td>False</td>\n",
       "      <td>Effects of crude oil on juvenile threespine st...</td>\n",
       "      <td>[{'name': 'Kelly S Ireland'}, {'name': 'Kathry...</td>\n",
       "      <td>&lt;p&gt;Aquatic oil spills have resounding effects ...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.11.19.469182</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>TK216 targets microtubules in Ewing sarcoma cells</td>\n",
       "      <td>[{'name': 'Juan Manuel Povedano Selfa'}, {'nam...</td>\n",
       "      <td>&lt;p&gt;Ewing sarcoma (EWS) is a pediatric malignan...</td>\n",
       "      <td>2021-11-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.11.15.468615</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Using OPM-MEG in contrasting magnetic environm...</td>\n",
       "      <td>[{'name': 'Ryan M. Hill'}, {'name': 'Jasen Dev...</td>\n",
       "      <td>&lt;p&gt;Magnetoencephalography (MEG) has been revol...</td>\n",
       "      <td>2021-11-16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.11.18.469186</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>TCR-BERT: learning the grammar of T-cell recep...</td>\n",
       "      <td>[{'name': 'Kevin E. Wu'}, {'name': 'Kathryn E....</td>\n",
       "      <td>&lt;p&gt;The T-cell receptor (TCR) allows T-cells to...</td>\n",
       "      <td>2021-11-20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.11.15.468694</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Structure and functionality of a multimeric hu...</td>\n",
       "      <td>[{'name': 'Mateusz Manicki'}, {'name': 'Halil ...</td>\n",
       "      <td>&lt;p&gt;Coenzyme Q (CoQ, ubiquinone) is a redox-act...</td>\n",
       "      <td>2021-11-15</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.11.13.468472     502  bioRxiv   \n",
       "1     10.1101/2021.11.18.469078     145  bioRxiv   \n",
       "2     10.1101/2021.11.18.469088     117  bioRxiv   \n",
       "3     10.1101/2021.11.19.469228     107  bioRxiv   \n",
       "4     10.1101/2021.11.16.468777      99  bioRxiv   \n",
       "5     10.1101/2021.11.15.468391      97  bioRxiv   \n",
       "6   10.1101/2021.11.14.21266295      93  medRxiv   \n",
       "7     10.1101/2021.11.17.465477      84  bioRxiv   \n",
       "8     10.1101/2021.11.18.469090      82  bioRxiv   \n",
       "9     10.1101/2021.11.16.468622      80  bioRxiv   \n",
       "10    10.1101/2021.11.17.468999      79  bioRxiv   \n",
       "11    10.1101/2021.11.15.468686      71  bioRxiv   \n",
       "12    10.1101/2021.11.18.469106      71  bioRxiv   \n",
       "13    10.1101/2021.11.15.468676      70  bioRxiv   \n",
       "14  10.1101/2021.11.14.21265758      54  medRxiv   \n",
       "15    10.1101/2021.11.19.468202      53  bioRxiv   \n",
       "16    10.1101/2021.11.15.468761      50  bioRxiv   \n",
       "17    10.1101/2021.11.15.468729      41  bioRxiv   \n",
       "18  10.1101/2021.11.14.21266328      41  medRxiv   \n",
       "19  10.1101/2021.11.15.21266187      41  medRxiv   \n",
       "20    10.1101/2021.11.17.469009      39  bioRxiv   \n",
       "21    10.1101/2021.11.18.468552      35  bioRxiv   \n",
       "22    10.1101/2021.11.14.468535      35  bioRxiv   \n",
       "23    10.1101/2021.11.17.469005      33  bioRxiv   \n",
       "24    10.1101/2021.11.15.468770      29  bioRxiv   \n",
       "25    10.1101/2021.11.18.469135      29  bioRxiv   \n",
       "26  10.1101/2021.11.19.21265383      29  medRxiv   \n",
       "27    10.1101/2021.11.15.468712      29  bioRxiv   \n",
       "28    10.1101/2021.11.20.468884      28  bioRxiv   \n",
       "29  10.1101/2021.11.19.21266552      27  medRxiv   \n",
       "30    10.1101/2021.11.18.467545      27  bioRxiv   \n",
       "31    10.1101/2021.11.14.468560      26  bioRxiv   \n",
       "32    10.1101/2021.11.14.468537      26  bioRxiv   \n",
       "33    10.1101/2021.11.15.468735      26  bioRxiv   \n",
       "34    10.1101/2021.11.17.469032      25  bioRxiv   \n",
       "35    10.1101/2021.11.15.468719      24  bioRxiv   \n",
       "36    10.1101/2021.11.18.469072      24  bioRxiv   \n",
       "37    10.1101/2021.11.18.468666      23  bioRxiv   \n",
       "38    10.1101/2021.11.15.468613      22  bioRxiv   \n",
       "39    10.1101/2021.11.15.468621      22  bioRxiv   \n",
       "40    10.1101/2021.11.18.469161      22  bioRxiv   \n",
       "41    10.1101/2021.11.15.468614      21  bioRxiv   \n",
       "42    10.1101/2021.11.16.468548      21  bioRxiv   \n",
       "43    10.1101/2021.11.12.468374      21  bioRxiv   \n",
       "44    10.1101/2021.11.16.467908      21  bioRxiv   \n",
       "45    10.1101/2021.11.17.469020      20  bioRxiv   \n",
       "46    10.1101/2021.11.19.469182      20  bioRxiv   \n",
       "47    10.1101/2021.11.15.468615      20  bioRxiv   \n",
       "48    10.1101/2021.11.18.469186      20  bioRxiv   \n",
       "49    10.1101/2021.11.15.468694      20  bioRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0                               Pathology   True   \n",
       "1                            Microbiology   True   \n",
       "2                            Biochemistry  False   \n",
       "3                                Genomics  False   \n",
       "4                            Microbiology   True   \n",
       "5                           Plant Biology  False   \n",
       "6                            Epidemiology   True   \n",
       "7                            Neuroscience  False   \n",
       "8                              Biophysics  False   \n",
       "9                          Cancer Biology  False   \n",
       "10                           Microbiology  False   \n",
       "11                               Genomics  False   \n",
       "12                               Genetics  False   \n",
       "13                         Bioinformatics  False   \n",
       "14  Infectious Diseases (except HIV/AIDS)   True   \n",
       "15                          Plant Biology  False   \n",
       "16                           Microbiology   True   \n",
       "17                             Physiology  False   \n",
       "18           Genetic and Genomic Medicine  False   \n",
       "19               Public and Global Health   True   \n",
       "20                           Neuroscience  False   \n",
       "21            Pharmacology and Toxicology  False   \n",
       "22                           Neuroscience  False   \n",
       "23                  Developmental Biology  False   \n",
       "24                           Microbiology  False   \n",
       "25                          Plant Biology  False   \n",
       "26           Genetic and Genomic Medicine  False   \n",
       "27                           Biochemistry  False   \n",
       "28                           Biochemistry  False   \n",
       "29  Infectious Diseases (except HIV/AIDS)   True   \n",
       "30                           Neuroscience  False   \n",
       "31                          Plant Biology  False   \n",
       "32                      Molecular Biology   True   \n",
       "33                           Neuroscience  False   \n",
       "34                           Neuroscience  False   \n",
       "35                           Biochemistry  False   \n",
       "36                         Bioinformatics  False   \n",
       "37                         Bioinformatics  False   \n",
       "38                           Cell Biology  False   \n",
       "39                           Microbiology  False   \n",
       "40                   Evolutionary Biology  False   \n",
       "41                           Cell Biology  False   \n",
       "42                      Synthetic Biology  False   \n",
       "43                           Microbiology   True   \n",
       "44                           Microbiology  False   \n",
       "45            Pharmacology and Toxicology  False   \n",
       "46                         Cancer Biology  False   \n",
       "47                           Neuroscience  False   \n",
       "48                         Bioinformatics  False   \n",
       "49                           Biochemistry  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   A live attenuated influenza virus-vectored int...   \n",
       "1     Metformin Suppresses SARS-CoV-2 in Cell Culture   \n",
       "2       Structural basis for Cas9 off-target activity   \n",
       "3      Pre-metazoan origin of neuropeptide signalling   \n",
       "4   The Delta variant of SARS-CoV-2 maintains high...   \n",
       "5   An atypical NLR protein modulates the NRC immu...   \n",
       "6   Estimation of R0 for the spread of SARS-CoV-2 ...   \n",
       "7   kTMP: A New Non-invasive Magnetic Induction Me...   \n",
       "8               Mechanics of cell integration in vivo   \n",
       "9   The evolution of genomic, transcriptomic, and ...   \n",
       "10  Reversible bacteriophage resistance by sheddin...   \n",
       "11  Evaluation of methods for the inference of anc...   \n",
       "12  Transgenic tools for targeted chromosome rearr...   \n",
       "13  Built on sand: the shaky foundations of simula...   \n",
       "14  Seroprevalence in Tamil Nadu through India’s t...   \n",
       "15  Smart soils to observe hidden rhizosphere proc...   \n",
       "16  Microglia do not restrict SARS-CoV-2 replicati...   \n",
       "17  NAD precursors cycle between host tissues and ...   \n",
       "18  Human genetic evidence supports MAP3K15 inhibi...   \n",
       "19  Controlling SARS-CoV-2 in schools using repeti...   \n",
       "20  Not so fast: Limited validity of deep convolut...   \n",
       "21  Proteolysis Targeting Chimeras With Reduced Of...   \n",
       "22  Rapid reconstruction of neural circuits using ...   \n",
       "23  Robust temporal map of human in vitro myelopoi...   \n",
       "24  Comparative genomics on cultivated and unculti...   \n",
       "25  Automated assembly scaffolding elevates a new ...   \n",
       "26  GWAS defines pathogenic signaling pathways and...   \n",
       "27     Structures of the Cyanobacterial Phycobilisome   \n",
       "28  A glycerol shunt functions as a glucose excess...   \n",
       "29  Low neutralizing antibody titers against the M...   \n",
       "30  Slow fluctuations in ongoing brain activity de...   \n",
       "31  Shade suppresses wound-induced leaf reposition...   \n",
       "32  Viral E Protein Neutralizes BET Protein-Mediat...   \n",
       "33  Modulation of Cutaneous Responses in the Cunea...   \n",
       "34  Anatomical connectivity along the anterior-pos...   \n",
       "35  Structure of the Quenched Cyanobacterial OCP-P...   \n",
       "36  scCODE: an R package for personalized differen...   \n",
       "37  PURC v2.0: a program for improved sequence inf...   \n",
       "38  Live imaging of the co-translational recruitme...   \n",
       "39  Mobilome-driven segregation of the resistome i...   \n",
       "40  Indirect effects of parental conflict on consp...   \n",
       "41  Wolbachia endosymbionts in two Anopheles speci...   \n",
       "42  Reconstitution of Spiroplasma swimming by expr...   \n",
       "43  Full protection against all four SARS-CoV-2 va...   \n",
       "44  Tuning spherical cells into kinking helices in...   \n",
       "45  Effects of crude oil on juvenile threespine st...   \n",
       "46  TK216 targets microtubules in Ewing sarcoma cells   \n",
       "47  Using OPM-MEG in contrasting magnetic environm...   \n",
       "48  TCR-BERT: learning the grammar of T-cell recep...   \n",
       "49  Structure and functionality of a multimeric hu...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Junyu Chen'}, {'name': 'Pui Wang'},...   \n",
       "1   [{'name': 'Dixit Tandel'}, {'name': 'Haripriya...   \n",
       "2   [{'name': 'Martin Pacesa'}, {'name': 'Chun-Han...   \n",
       "3   [{'name': 'Luis Alfonso Yanez-Guerra'}, {'name...   \n",
       "4   [{'name': 'Rayhane Nchioua'}, {'name': 'Annika...   \n",
       "5   [{'name': 'Hiroaki Adachi'}, {'name': 'Toshiyu...   \n",
       "6   [{'name': 'Juan Pablo Prada'}, {'name': 'Luca ...   \n",
       "7   [{'name': 'Ludovica Labruna'}, {'name': 'Chris...   \n",
       "8   [{'name': 'Guilherme B. Ventura'}, {'name': 'A...   \n",
       "9   [{'name': 'Kentaro Ohara'}, {'name': 'Andrè Fi...   \n",
       "10  [{'name': 'Veronique Ongenae'}, {'name': 'Adam...   \n",
       "11  [{'name': 'Débora Y. C. Brandt'}, {'name': 'Xi...   \n",
       "12                       [{'name': 'David L. Stern'}]   \n",
       "13  [{'name': 'Helena L. Crowell'}, {'name': 'Sara...   \n",
       "14  [{'name': 'T.S. Selvavinayagam'}, {'name': 'A....   \n",
       "15  [{'name': 'Daniel Patko'}, {'name': 'Qizhi Yan...   \n",
       "16  [{'name': 'Gema M. Olivarria'}, {'name': 'Yuti...   \n",
       "17  [{'name': 'Karthikeyani Chellappa'}, {'name': ...   \n",
       "18  [{'name': 'Abhishek Nag'}, {'name': 'Ryan S. D...   \n",
       "19  [{'name': 'Andrea Torneri'}, {'name': 'Lander ...   \n",
       "20  [{'name': 'Guo Jiahui'}, {'name': 'Ma Feilong'...   \n",
       "21  [{'name': 'Tuan M Nguyen'}, {'name': 'Arghya D...   \n",
       "22  [{'name': 'Joshua L. Lillvis'}, {'name': 'Hide...   \n",
       "23  [{'name': 'Clara Alsinet'}, {'name': 'Maria Pr...   \n",
       "24  [{'name': 'Hang Yu'}, {'name': 'Grayson L. Cha...   \n",
       "25  [{'name': 'Michael Alonge'}, {'name': 'Ludivin...   \n",
       "26  [{'name': 'Krzysztof Kiryluk'}, {'name': 'Elen...   \n",
       "27  [{'name': 'Paul V. Sauer'}, {'name': 'Maria Ag...   \n",
       "28  [{'name': 'Anfal Al-Mass'}, {'name': 'Pegah Po...   \n",
       "29  [{'name': 'Diego A. Álvarez-Díaz'}, {'name': '...   \n",
       "30  [{'name': 'Maria J. Ribeiro'}, {'name': 'Migue...   \n",
       "31  [{'name': 'Anne-Sophie Fiorucci'}, {'name': 'O...   \n",
       "32  [{'name': 'Irene P. Chen'}, {'name': 'James E....   \n",
       "33  [{'name': 'Qinpu He'}, {'name': 'Christopher S...   \n",
       "34  [{'name': 'Marshall Axel Dalton'}, {'name': \"A...   \n",
       "35  [{'name': 'Maria Agustina Dominguez-Martin'}, ...   \n",
       "36  [{'name': 'jiawei Zou'}, {'name': 'miaochen Wa...   \n",
       "37  [{'name': 'Peter W Schafran'}, {'name': 'Fay-W...   \n",
       "38  [{'name': 'Silvia Gómez-Puerta'}, {'name': 'Ro...   \n",
       "39  [{'name': 'Laura de Nies'}, {'name': 'Susheel ...   \n",
       "40                      [{'name': 'Jenn M Coughlan'}]   \n",
       "41  [{'name': 'Shannon Quek'}, {'name': 'Louise Ce...   \n",
       "42  [{'name': 'Hana Kiyama'}, {'name': 'Shigeyuki ...   \n",
       "43  [{'name': 'Sapna Sharma'}, {'name': 'Thomas Ve...   \n",
       "44  [{'name': 'Carole Lartigue'}, {'name': 'Bastie...   \n",
       "45  [{'name': 'Kelly S Ireland'}, {'name': 'Kathry...   \n",
       "46  [{'name': 'Juan Manuel Povedano Selfa'}, {'nam...   \n",
       "47  [{'name': 'Ryan M. Hill'}, {'name': 'Jasen Dev...   \n",
       "48  [{'name': 'Kevin E. Wu'}, {'name': 'Kathryn E....   \n",
       "49  [{'name': 'Mateusz Manicki'}, {'name': 'Halil ...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>Remarkable progress has been made in develo...  2021-11-15  \n",
       "1   <p>People with diabetes are reported to have a...  2021-11-19  \n",
       "2   <p>The target DNA specificity of the CRISPR-as...  2021-11-18  \n",
       "3   <p>Neuropeptides are a diverse class of signal...  2021-11-20  \n",
       "4   <p>Interferons are a major part of the anti-vi...  2021-11-17  \n",
       "5   <p>The NRC immune receptor network has evolved...  2021-11-15  \n",
       "6   <p>For SARS-CoV-2, R0 calculations report usua...  2021-11-15  \n",
       "7   <p>We report a novel subthreshold non-invasive...  2021-11-19  \n",
       "8   <p>During embryonic development, regeneration ...  2021-11-18  \n",
       "9   <p>The molecular characteristics of metastatic...  2021-11-19  \n",
       "10  <p>Phages are highly abundant in the environme...  2021-11-18  \n",
       "11  <p>The ancestral recombination graph (ARG) is ...  2021-11-17  \n",
       "12  <p>Perhaps the most valuable single set of res...  2021-11-18  \n",
       "13  <p>With the emergence of hundreds of single-ce...  2021-11-15  \n",
       "14  <p>Three rounds of population-representative s...  2021-11-15  \n",
       "15  <p>Agriculture must reduce green-house gas emi...  2021-11-19  \n",
       "16  <p>Unlike SARS-CoV-1 and MERS-CoV, infection w...  2021-11-17  \n",
       "17  <p>Nicotinamide adenine dinucleotide (NAD) is ...  2021-11-19  \n",
       "18  <p>Diabetes mellitus is a chronic health condi...  2021-11-15  \n",
       "19  <p>SARS-CoV-2 remains a worldwide emergency. W...  2021-11-15  \n",
       "20  <p>Deep convolutional neural networks (DCNNs) ...  2021-11-19  \n",
       "21  <p>Proteolysis Targeting Chimeras (PROTACs), a...  2021-11-20  \n",
       "22  <p>Electron microscopy (EM) allows for the rec...  2021-11-15  \n",
       "23  <p>Myeloid cells have a central role in homeos...  2021-11-19  \n",
       "24  <p>Chemolithoautotrophic manganese oxidation h...  2021-11-16  \n",
       "25  <p>Advancing crop genomics requires efficient ...  2021-11-19  \n",
       "26  <p>IgA nephropathy (IgAN) is a progressive for...  2021-11-20  \n",
       "27  <p>The phycobilisome is an elaborate antenna t...  2021-11-15  \n",
       "28  <p>The recently identified glycerol-3-phosphat...  2021-11-20  \n",
       "29  <sec><title>Background</title><p>Global survei...  2021-11-19  \n",
       "30  <p>In humans, ageing is characterized by decre...  2021-11-19  \n",
       "31  <p>Shaded plants challenged with herbivores or...  2021-11-15  \n",
       "32  <p>Inhibitors of Bromodomain and Extra-termina...  2021-11-15  \n",
       "33  <p>To achieve stable and precise movement exec...  2021-11-19  \n",
       "34  <p>The hippocampus supports multiple cognitive...  2021-11-19  \n",
       "35  <p>Photoprotection is an essential mechanism i...  2021-11-15  \n",
       "36  <p>Differential expression (DE) gene detection...  2021-11-19  \n",
       "37  <p>Inferring the true biological sequences fro...  2021-11-19  \n",
       "38  <p>Endoplasmic reticulum (ER) to nucleus homeo...  2021-11-15  \n",
       "39  <p>Biological wastewater treatment plants (BWW...  2021-11-15  \n",
       "40  <p>Hybrid Seed Inviability (HSI) is a common b...  2021-11-20  \n",
       "41  <p><italic>Wolbachia</italic> is a genus of ob...  2021-11-15  \n",
       "42  <p>Motility is one of the most important featu...  2021-11-17  \n",
       "43  <p>Current licensed COVID-19 vaccines are base...  2021-11-15  \n",
       "44  <p>In bacteria, cell shape is determined and m...  2021-11-17  \n",
       "45  <p>Aquatic oil spills have resounding effects ...  2021-11-19  \n",
       "46  <p>Ewing sarcoma (EWS) is a pediatric malignan...  2021-11-19  \n",
       "47  <p>Magnetoencephalography (MEG) has been revol...  2021-11-16  \n",
       "48  <p>The T-cell receptor (TCR) allows T-cells to...  2021-11-20  \n",
       "49  <p>Coenzyme Q (CoQ, ubiquinone) is a redox-act...  2021-11-15  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "258 preprints (including 37 covering SARS-CoV-2, 227 from bioRxiv and 31 from medRxiv) published in the last 7 days before 2021-11-22 had been tweeted at least 3 times (maximum 502)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
