{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "22531 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b91c6f0a-1ecd-4143-b228-fe293dea90af&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2ec9b04e-9d10-4666-bed1-e6c5766a79f7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a4188000-ed6f-44e0-9c5b-178cd47fc2e4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a0df8f13-89ff-4f30-be6b-cd845b607a48&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a82f1775-b480-4998-8525-60af2d4ed8f3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=6f74c60e-e62a-405c-a31d-c7f4d18af0af&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=bf711cc2-c8eb-46df-9c45-24fb4f9fa589&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=585ebb62-78e8-4b63-a91a-d9e26a1c6797&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=34660e1f-28ce-4373-b973-39cf68223f4d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c4680606-15a6-47b0-a4e4-4094a98ae19f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b8b3d013-7373-49d8-9f47-f54fc08fea3a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=b27d45ac-d1e1-45d0-8441-9f98b83dae69&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=536fc0bc-cdad-44b6-81bd-bafbf6add68e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=dca50b42-e12a-4b5d-843e-64567d083819&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5723584a-899e-4a01-a4b9-5abd805b5cf3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=ecb7f40e-42d3-4b61-af9d-ab942776f979&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=3aae1c36-2f1c-42ad-a855-72fd5cc2a35a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=09f59c04-6222-4622-a6d0-8e7d788c78f2&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0018.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7d8231f3-5ddb-4ba1-a648-d3ce823ae6cd&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0019.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=ecdf2325-abf1-4c00-821b-67e39d80e281&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0020.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=070a8e7b-2a05-48b3-a7a8-49efb1f90972&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0021.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4a2d3dbe-c9d7-4a75-8fec-1fd99d61c548&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-04&until-occurred-date=2021-10-11\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0022.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.382143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/530097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.20142703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275569.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.21264267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.420448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273300.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.459575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/240317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.28.923532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.426870...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.222299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462346...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.08.191106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.367649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.21.20132449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.21263548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21258385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.413609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.21254026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.017624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462458...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.462880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.206680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.462913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.20093336...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275430.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.163816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.11.451967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20213546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.447951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.327114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.446796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.24.352989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.03.20167395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.413229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.21256049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.21262262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.17.20037689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.271908.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.186684.114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.21261741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.07.20056424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.21262064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463216...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.390773...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.444987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462348...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.448565...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462285...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21256946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.444292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441655...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462462...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.21254678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.461806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.439482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.21264162...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.347054...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.088500...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.20204693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275308.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268581.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.20211755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.26.424423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21249256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.08.20124792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.462574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.21249563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462435...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.225102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.21250840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.459925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.116640...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462293...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.20222661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.437450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.399154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268037.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.384594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.459429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.20235747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438592...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453518...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463337...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.389031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.11.20061473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/608786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262216...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.23.056309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.21262472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.460811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.271874.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.21.914754...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.14.943118...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.24.006197...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21250865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.20086439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/238527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.02.232710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.235549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.30.272237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.462716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.08.20123877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21258418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21262647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.20162420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.21252338...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.443486...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.432552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264335...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.439517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462641...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21254148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.460970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.21255913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462603...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.21257017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259351...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462099...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21263052...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.02.20120014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.17.20133983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21262468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20159228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.21251745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.20155606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.05.237958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21259864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.17.452576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.08.982637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435471...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21262021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.21262042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21253141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.274340.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/176941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/768663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462041...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/276717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.21.21251403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.435319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.20.20073437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21256632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.14.20062463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.06.028712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.29.20248977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.20247700...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.446959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.335083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.460530...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.07.330563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.21249219...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256753...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.21258488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.447074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20209817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463153...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.462986...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462279...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.444789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.385401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460834...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.17.386995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462263...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462682...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.20248876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.19.20198028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.20209114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.449101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463322...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.05.369827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.21262615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264300...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.10.08.463746', 'tweets': 56, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Deep-Learning Super-Resolution Microscopy Reveals Nanometer-Scale Intracellular Dynamics at the Millisecond Temporal Resolution', 'authors': \"[{'name': 'Rong Chen'}, {'name': 'Xiao Tang'}, {'name': 'Zeyu Shen'}, {'name': 'Yusheng Shen'}, {'name': 'Tiantian Li'}, {'name': 'Ji Wang'}, {'name': 'Binbin Cui'}, {'name': 'Yusong Guo'}, {'name': 'Shengwang Du'}, {'name': 'Shuhuai Yao'}]\", 'abstract': '<p>Single-molecule localization microscopy (SMLM) can be used to resolve subcellular structures and achieve a tenfold improvement in spatial resolution compared to that obtained by conventional fluorescence microscopy. However, the separation of single-molecule fluorescence events in thousands of frames dramatically increases the image acquisition time and phototoxicity, impeding the observation of instantaneous intracellular dynamics. Based on deep learning networks, we develop a single-frame super-resolution microscopy (SFSRM) approach that reconstructs a super-resolution image from a single frame of a diffraction-limited image to support live-cell super-resolution imaging at a ~20 nm spatial resolution and a temporal resolution of up to 10 ms over thousands of time points. We demonstrate that our SFSRM method enables the visualization of the dynamics of vesicle transport at a millisecond temporal resolution in the dense and vibrant microtubule network in live cells. Moreover, the well-trained network model can be used with different live-cell imaging systems, such as confocal and light-sheet microscopes, making super-resolution microscopy accessible to nonexperts.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.02.21264267', 'tweets': 39, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Year-long COVID-19 infection reveals within-host evolution of SARS-CoV-2 in a patient with B cell depletion', 'authors': \"[{'name': 'Veronique Nussenblatt'}, {'name': 'Allison E Roder'}, {'name': 'Sanchita Das'}, {'name': 'Emmie de Wit'}, {'name': 'Jung-Ho Youn'}, {'name': 'Stephanie Banakis'}, {'name': 'Alexandra Mushegian'}, {'name': 'Christopher Mederos'}, {'name': 'Wei Wang'}, {'name': 'Matthew Chung'}, {'name': 'Lizzette Pérez-Pérez'}, {'name': 'Tara Palmore'}, {'name': 'Jennifer N. Brudno'}, {'name': 'James N. Kochenderfer'}, {'name': 'Elodie Ghedin'}]\", 'abstract': '<sec><title>Background</title><p>B-cell depleting therapies may lead to protracted disease and prolonged viral shedding in individuals infected with SARS-CoV-2. Viral persistence in the setting of immunosuppression raises concern for viral evolution.</p></sec><sec><title>Methods</title><p>Amplification of sub-genomic transcripts for the E gene (sgE) was done on nasopharyngeal samples over the course of 355 days in a patient infected with SARS-CoV-2 who had previously undergone CAR T cell therapy and had persistently positive SARS-CoV-2 nasopharyngeal swabs. Whole genome sequencing was performed on samples from the patient’s original presentation and 10 months later.</p></sec><sec><title>Results</title><p>Over the course of almost a year, the virus accumulated a unique in-frame deletion in the amino-terminal domain of the spike protein, and complete deletion of ORF7b and ORF8, the first report of its kind in an immunocompromised patient. Also, minority variants that were identified in the early samples—reflecting the heterogeneity of the initial infection—were found to be fixed late in the infection. Remdesivir and high-titer convalescent plasma treatment were given, and the infection was eventually cleared after 335 days of infection.</p></sec><sec><title>Conclusions</title><p>The unique viral mutations found in this study highlight the importance of analyzing viral evolution in protracted SARS-CoV-2 infection, especially in immunosuppressed hosts, and the implication of these mutations in the emergence of viral variants.</p></sec><sec><title>Summary</title><p>We report an immunocompromised patient with persistent symptomatic SARS-CoV-2 infection for 335 days. During this time, the virus accumulated a unique in-frame deletion in the spike, and a complete deletion of ORF7b and ORF8 which is the first report of its kind in an immunocompromised patient.</p></sec>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.06.21264535', 'tweets': 37, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Analytical performance of eleven SARS-CoV-2 antigen-detecting rapid tests for Delta variant', 'authors': \"[{'name': 'Meriem Bekliz'}, {'name': 'Kenneth Adea'}, {'name': 'Manel Essaidi-Laziosi'}, {'name': 'Camille Escadafal'}, {'name': 'Jilian A Sacks'}, {'name': 'Laurent Kaiser'}, {'name': 'Isabella Eckerle'}]\", 'abstract': '<p>Global concerns arose as the emerged and rapidly spreading SARS-CoV-2 Delta variant. To date, few data on routine diagnostic performance for Delta are available. Here, we investigate the analytical performance of eleven commercially available antigen-detecting rapid diagnostic tests (Ag-RDTs) for Delta VOC in comparison with current and earlier VOCs (Alpha, Beta and Gamma) and early pandemic variant using cultured SARS-CoV-2. Comparable sensitivity was observed for Delta for the majority of Ag-RDTs.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.05.463267', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Intercellular transport of RNA can limit heritable epigenetic changes', 'authors': \"[{'name': 'Nathan Shugarts'}, {'name': 'Andrew L. Yi'}, {'name': 'Winnie M. Chan'}, {'name': 'Julia A. Marré'}, {'name': 'Aishwarya Sathya'}, {'name': 'Antony M. Jose'}]\", 'abstract': '<p>RNAs in circulation carry sequence-specific regulatory information between cells in animal, plant, and host-pathogen systems. Double-stranded RNA (dsRNA) delivered into the extracellular space of the nematode <italic>C. elegans</italic> accumulates within the germline and reaches progeny. Here we provide evidence for spatial, temporal, and substrate specificity in the transport of dsRNA from parental circulation to progeny. Temporary loss of dsRNA transport resulted in the persistent accumulation of mRNA from a germline gene. The expression of this gene varied among siblings and even between gonad arms within one animal. Perturbing RNA regulation of the gene created new epigenetic states that lasted for many generations. Thus, one role for the transport of dsRNA into the germline in every generation is to limit heritable changes in gene expression.</p><sec><title>One Sentence Summary</title><p>RNA from parental circulation reduces heritable changes in gene expression.</p></sec>', 'posted': '2021-10-06'}, {'doi': '10.1101/2021.10.04.21264500', 'tweets': 22, 'archive': 'medRxiv', 'subject-area': 'Obstetrics and Gynecology', 'covid': True, 'title': 'Increase in preterm stillbirths and reduction in iatrogenic preterm births for fetal compromise: a multi-centre cohort study of COVID-19 lockdown effects in Melbourne, Australia', 'authors': \"[{'name': 'Lisa Hui'}, {'name': 'Melvin Barrientos Marzan'}, {'name': 'Stephanie Potenza'}, {'name': 'Daniel L. Rolnik'}, {'name': 'Natasha Pritchard'}, {'name': 'Joanne M. Said'}, {'name': 'Kirsten R Palmer'}, {'name': 'Clare L. Whitehead'}, {'name': 'Penelope M. Sheehan'}, {'name': 'Jolyon Ford'}, {'name': 'Ben W. Mol'}, {'name': 'Susan P. Walker'}]\", 'abstract': '<sec><title>Objectives</title><p>The COVID-19 pandemic has been associated with a worsening of perinatal outcomes in many settings due to the combined impacts of maternal COVID-19 disease, disruptions to maternity care, and overloaded health systems. In 2020, Melbourne endured a unique natural experiment where strict lockdown conditions were accompanied by very low COVID-19 case numbers and the maintenance of health service capacity. The aim of this study was to compare stillbirth and preterm birth rates in women who were exposed or unexposed to lockdown restrictions during pregnancy.</p></sec><sec><title>Design</title><p>Retrospective multi-centre cohort study of perinatal outcomes before and during COVID-19 lockdown</p></sec><sec><title>Setting</title><p>Birth outcomes from all 12 public maternity hospitals in metropolitan Melbourne</p></sec><sec><title>Inclusion criteria</title><p>Singleton births without congenital anomalies from 24 weeks’ gestation. The lockdown-exposed cohort were those women for whom weeks 20- 40 of gestation would have occurred during the lockdown period of 23 March 2020 to 14 March 2021. The control cohort comprised all pregnancies in the corresponding periods one and two years prior to the exposed cohort.</p></sec><sec><title>Main outcome measures</title><p>Odds of stillbirth, preterm birth (PTB), birth weight &lt; 3<sup>rd</sup> centile, and iatrogenic PTB for fetal compromise, adjusting for multiple covariates.</p></sec><sec><title>Results</title><p>There were 24,017 births in the exposed and 50,017 births in the control group. There was a significantly higher risk of preterm, but not term, stillbirth in the exposed group compared with the control group (0.26% vs 0.18%, aOR 1.49, 95%CI 1.08 to 2.05, P = 0.015). There was also a significant reduction in preterm birth &lt; 37 weeks (5.93% vs 6.23%, aOR 0.93, 95%CI 0.87 to 0.99, P=0.03), largely mediated by a reduction in iatrogenic PTB for live births (3.01% vs 3.27%, aOR 0.89, 95%CI 0.81 to 0.98, P = 0.015), including iatrogenic PTB for suspected fetal compromise (1.25% vs 1.51%, aOR 0.79, 95%CI 0.69 to 0.91, P= 0.001). There was no significant difference in the spontaneous PTB rate between the exposed and control groups (2.69% vs 2.82%, aOR 0.94, 95%CI 0.86 to 0.1.03, P=0.25).</p></sec><sec><title>Conclusions</title><p>Lockdown restrictions in a high-income setting, in the absence of high rates of COVID-19 disease, were associated with a significant increase in preterm stillbirths, and a significant reduction in iatrogenic PTB for suspected fetal compromise.</p></sec><sec><title>Trial registration</title><p>This study was registered as an observational study with the Australian and New Zealand Clinical Trials Registry (ACTRN12620000878976).</p></sec>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.07.463568', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Molecular rhythm alterations in prefrontal cortex and nucleus accumbens associated with opioid use disorder', 'authors': \"[{'name': 'Xiangning Xue'}, {'name': 'Wei Zong'}, {'name': 'Jill Glausier'}, {'name': 'Sam-Moon Kim'}, {'name': 'Micah Shelton'}, {'name': 'BaDoi N Phan'}, {'name': 'Chaitanya Srinivasan'}, {'name': 'Andreas R Pfenning'}, {'name': 'George Tseng'}, {'name': 'David A Lewis'}, {'name': 'Marianne Seney'}, {'name': 'Ryan Logan'}]\", 'abstract': \"<p>Severe and persistent disruptions to sleep and circadian rhythms are common features of people with opioid use disorder (OUD). Preclinical findings suggest altered molecular rhythms in the brain are involved in opioid reward and dependence. However, whether molecular rhythms are disrupted in brains of people with OUD remained an open question, critical to understanding the role of circadian rhythms in opioid addiction. We previously used subjects' times of death (TOD) as a marker of time of day to investigate transcriptional rhythm alterations in psychiatric disorders. Using TOD and RNA sequencing, we discovered rhythmic transcripts in both the dorsolateral prefrontal cortex (DLPFC) and nucleus accumbens (NAc), key brain areas involved in opioid addiction, were largely distinct between OUD and unaffected comparison subjects. Further, fewer rhythmic transcripts were identified in DLPFC of OUD subjects compared to unaffected subjects, but nearly double the number of rhythmic transcripts were found in the NAc of OUD subjects. In OUD, rhythmic transcripts in the NAc peaked either in the evening or near sunrise, and were associated with dopamine, opioid, and GABAergic neurotransmission. Co-expression network analysis identified several OUD-specific modules in the NAc, enriched for transcripts involved in the modulation of dopamine and GABA synapses, including glutamatergic signaling and extracellular matrices. Integrative analyses with human GWAS revealed that rhythmic transcripts in DLPFC and NAc were enriched for genomic loci associated with sleep duration and insomnia. Overall, our results connect transcriptional rhythm changes in dopamine, opioid, and GABAergic synaptic signaling in human brain to sleep-related phenotypes and OUD.</p>\", 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.08.463671', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'High-throughput mutagenesis identifies mutations and RNA binding proteins controlling CD19 splicing and CART-19 therapy resistance', 'authors': \"[{'name': 'Mariela Cortés-López'}, {'name': 'Laura Schulz'}, {'name': 'Mihaela Enculescu'}, {'name': 'Claudia Paret'}, {'name': 'Bea Spiekermann'}, {'name': 'Anke Busch'}, {'name': 'Anna Orekhova'}, {'name': 'Fridolin Kielisch'}, {'name': 'Mathieu Quesnel-Vallières'}, {'name': 'Manuel Torres-Diz'}, {'name': 'Jörg Faber'}, {'name': 'Yoseph Barash'}, {'name': 'Andrei Thomas-Tikhonenko'}, {'name': 'Kathi Zarnack'}, {'name': 'Stefan Legewie'}, {'name': 'Julian König'}]\", 'abstract': '<p>During CART-19 immunotherapy for B-cell acute lymphoblastic leukaemia (B-ALL), many patients relapse due to loss of the cognate CD19 epitope. Since epitope loss can be caused by aberrant <italic>CD19</italic> exon 2 processing, we herein investigate the regulatory code that controls CD19 splicing. We combine high-throughput mutagenesis with mathematical modelling to quantitatively disentangle the effects of all mutations in the region comprising <italic>CD19</italic> exons 1-3. Thereupon, we identify ~200 single point mutations that alter <italic>CD19</italic> splicing and thus could predispose B-ALL patients to CART-19 resistance. Furthermore, we report almost 100 previously unknown splice isoforms that emerge from cryptic splice sites and likely encode non-functional CD19 proteins. We further identify cis-regulatory elements and trans-acting RNA-binding proteins that control <italic>CD19</italic> splicing (e.g., PTBP1 and SF3B4) and validate that loss of these factors leads to enhanced <italic>CD19</italic> mis-splicing. Our dataset represents a comprehensive resource for potential prognostic factors predicting success of CART-19 therapy.</p>', 'posted': '2021-10-08'}, {'doi': '10.1101/2021.10.07.463475', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Pseudo-dynamic analysis of heart tube formation in the mouse reveals strong regional variability and early left-right asymmetry', 'authors': \"[{'name': 'Isaac Esteban'}, {'name': 'Patrick Schmidt'}, {'name': 'Susana Temino'}, {'name': 'Leif Kobbelt'}, {'name': 'Miguel Torres'}]\", 'abstract': '<p>Understanding organ morphogenesis requires a precise geometrical description of the tissues involved in the process. In highly regulative embryos, like those of mammals, morphological variability hinders the quantitative analysis of morphogenesis. In particular, the study of early heart development in mammals remains a challenging problem, due to imaging limitations and innate complexity. Around embryonic day 7.5 (E7.5), the cardiac crescent folds in an intricate and coordinated manner to produce a pumping linear heart tube at E8.25, followed by heart looping at E8.5. In this work we provide a complete morphological description of this process based on detailed imaging of a temporally dense collection of embryonic heart morphologies. We apply new approaches for morphometric staging and quantification of local morphological variations between specimens at the same stage. We identify hot spots of regionalized variability and identify left-right asymmetry in the inflow region starting at the late cardiac crescent stage, which represents the earliest signs of organ left-right asymmetry in the mammalian embryo. Finally, we generate a 3D+t digital model that provides a framework suitable for co-representation of data from different sources and for the computer modelling of the process.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.07.463556', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Large-scale discovery of candidate type VI secretion effectors with antibacterial activity', 'authors': \"[{'name': 'Alexander Martin Geller'}, {'name': 'David Zlotkin'}, {'name': 'Asaf Levy'}]\", 'abstract': '<p>Type VI secretion systems (T6SS) are common bacterial contractile injection systems that inject toxic effector proteins into neighboring cells. Effector discovery is generally done manually, and computational approaches used for effector discovery depend on genetic linkage to T6SS genes and/or sequence similarity to known effectors. We bioinformatically investigated T6SS in more than 11,832 genomes of Gram negative bacteria. We found that T6SS encoding bacteria are host-associated and pathogenic, enriched in specific human and plant tissues, while depleted in marine, soil, and engineered environments. Analysis of T6SS cores with C-terminal domains (\"evolved\" cores) showed \"evolved\" HCP are rare, overwhelmingly encoded in orphan operons, and are largely restricted to Escherichia. Using the wealth of data generated from our bioinformatic analysis, we developed two algorithms for large-scale discovery of T6SS effector proteins (T6Es). We experimentally validated ten putative antibacterial T6SS effector proteins and one cognate immunity gene from a diverse species. This study provides a systematic genomic perspective of the role of the T6SS in nature, a thorough analysis of T6E evolution and genomic properties, and discovery of a large number of candidate T6Es using new approaches.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.04.462880', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Nanoengineered DNA origami with repurposed TOP1 inhibitors targeting myeloid cells for the mitigation of neuroinflammation', 'authors': \"[{'name': 'Keying Zhu'}, {'name': 'Yang Wang'}, {'name': 'Heela Sarlus'}, {'name': 'Keyi Geng'}, {'name': 'Erik Nutma'}, {'name': 'Jingxian Sun'}, {'name': 'Shin-Yu Kung'}, {'name': 'Cindy Bay'}, {'name': 'Jinming Han'}, {'name': 'Harald Lund'}, {'name': 'Sandra Amor'}, {'name': 'Jun Wang'}, {'name': 'Xingmei Zhang'}, {'name': 'Claudia Kutter'}, {'name': 'André Ortlieb Guerreiro Cacais'}, {'name': 'Björn Högberg'}, {'name': 'Robert A. Harris'}]\", 'abstract': '<p>Targeting myeloid cells, especially microglia, for the treatment of neuroinflammatory diseases such as multiple sclerosis (MS), is underappreciated. Here, we screened a library of compounds and identified the topoisomerase 1 (TOP1) inhibitor camptothecin (CPT) as a promising drug candidate for microglial modulation. CPT and its FDA-approved analog topotecan (TPT) inhibited inflammatory responses in microglia and macrophages, and ameliorated neuroinflammation in mice. Transcriptomic analysis of sorted microglia revealed an altered transcriptional phenotype following TPT treatment, with Ikzf1 identified as a potential target. Importantly, TOP1 expression was found elevated in several neuroinflammatory conditions, including human MS brains. To achieve targeted delivery to myeloid cells we designed a nanosystem using DNA origami and loaded TPT into it (TopoGami). TopoGami also significantly suppressed the inflammatory response in microglia and mitigated disease progression in MS-like mice. Our findings suggest that TOP1 inhibition represents a therapeutic strategy for neuroinflammatory diseases, and the proposed nanosystem may foster future research and drug development with a demand to target myeloid cells.</p>', 'posted': '2021-10-04'}, {'doi': '10.1101/2021.10.04.463138', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Linking contrast sensitivity to cortical magnification in human primary visual cortex', 'authors': \"[{'name': 'Marc M. Himmelberg'}, {'name': 'Jonathan Winawer'}, {'name': 'Marisa Carrasco'}]\", 'abstract': '<p>The size and organization of primary visual cortex (V1) varies across individuals. Across neurotypical adults, V1 size varies more than twofold. Within individuals, surface area per unit of visual field – cortical magnification – varies with eccentricity and polar angle. Contrast sensitivity and cortical magnification covary with eccentricity, therefore it has been hypothesized that cortical magnification, specifically the number of activated V1 neurons, limits contrast sensitivity. Here, we quantify the relation between contrast sensitivity and V1 cortical magnification across observers and polar angle. We measured contrast sensitivity at four cardinal meridians in 29 observers. We then used fMRI to measure the size of V1 in the same observers, and the amount of surface area representing each of the four meridians (wedge-ROIs within 15° polar angle of the meridians, 1 to 8° eccentricity). We found that: First, an observer’s contrast sensitivity (averaged across polar angles) was predicted by the size of V1. Second, contrast sensitivity at each cardinal meridian was correlated with the surface area of the wedge-ROIs centered at the corresponding meridian. Third, increases in contrast sensitivity and cortical magnification at the horizontal compared to vertical meridian (horizontal-vertical anisotropy, ‘HVA’) were strongly correlated: a larger HVA in contrast sensitivity corresponded to a larger HVA in cortical magnification. These results reveal that contrast sensitivity and cortical magnification co-vary across observers and demonstrate a link between perceptual polar angle asymmetries and cortical anatomy. Broadly, the results show a link between visual perception and the idiosyncratic cortical organization of V1 in neurotypical observers.</p><sec><title>SIGNIFICANCE STATEMENT</title><p>Contrast sensitivity is a fundamental property of the human visual system, which indexes the limits of what one can detect or discriminate – the window of visibility. Contrast sensitivity varies with stimulus location on the retina and across observers. These variations are not well understood. Using psychophysics and magnetic resonance imaging, we tested the hypothesis that contrast sensitivity depends on the amount of responsive tissue in primary visual cortex (V1). Individuals with greater contrast sensitivity had a larger V1. Further, within observers, variation in contrast sensitivity across polar angle locations matched the variation in V1 surface area representing those locations. These findings demonstrate a tight link between visual perception and cortical anatomy, both within and among people.</p></sec>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.06.462913', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Minimal required exposures reveal the primacy of awareness in human face perception', 'authors': \"[{'name': 'Renzo C. Lanfranco'}, {'name': 'Andrés Canales-Johnson'}, {'name': 'Axel Cleeremans'}, {'name': 'Hugh Rabagliati'}, {'name': 'David Carmel'}]\", 'abstract': '<p>The human visual system is capable of detecting extremely small light signals, as little as one single photon. But what is the visual system’s minimal exposure threshold to detect meaningful stimuli? Due to hardware limitations, studies examining fast visual processing typically present stimuli for suprathreshold durations and disrupt processing with a mask. Here, we employed a newly developed tachistoscope that enables sub-millisecond presentations. Combining measures of perceptual sensitivity and recordings of neural activity, we find that both the holistic configuration and the emotional expression of a human face have the same minimal required exposure as the minimal required exposure for faces to reach awareness. These findings indicate that conscious awareness is required for processing meaningful visual information.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.08.463652', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'A sense-antisense RNA interaction promotes breast cancer metastasis via regulation of NQO1 expression', 'authors': \"[{'name': 'Bruce Culbertson'}, {'name': 'Kristle Garcia'}, {'name': 'Daniel Markett'}, {'name': 'Hosseinali Asgharian'}, {'name': 'Li Chen'}, {'name': 'Lisa Fish'}, {'name': 'Albertas Navickas'}, {'name': 'Johnny Yu'}, {'name': 'Brian Woo'}, {'name': 'Scott Nanda'}, {'name': 'Joshua Rabinowitz'}, {'name': 'Hani Goodarzi'}]\", 'abstract': \"<p>Antisense RNAs are ubiquitous in human cells, yet the role that they play in healthy and diseased states remains largely unexplored. Here, we developed a computational framework to catalog and profile antisense RNAs and applied it to poorly and highly metastatic breast cancer cell lines. We identified one antisense RNA that plays a functional role in driving breast cancer progression by upregulating the redox enzyme NQO1, and hence named NQO1-antisense RNA or NQO1-AS. This upregulation occurs via a stabilizing interaction between NQO1-AS and its complementary region in the 3'UTR of NQO1 mRNA. By increasing expression of NQO1 protein, breast cancer cells are able to tolerate higher levels of oxidative stress, enabling them to colonize the lung. During this process the cancer cells become dependent on NQO1 to protect them from ferroptosis. We have shown that this dependence can be exploited therapeutically in xenograft models of metastasis. Together, our findings establish a previously unknown role for NQO1-AS in the progression of breast cancer by serving as a post-transcriptional regulator of RNA processing and decay for its sense mRNA.</p>\", 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.07.463578', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'The DNAPKcs long-range C-NHEJ complex is required for blunt DNA end joining when XLF is weakened', 'authors': \"[{'name': 'Metztli Cisneros-Aguirre'}, {'name': 'Felicia Wednesday Lopezcolorado'}, {'name': 'Linda Jillianne Tsai'}, {'name': 'Ragini Bhargava'}, {'name': 'Jeremy M. Stark'}]\", 'abstract': '<p>Canonical non-homologous end joining (C-NHEJ) factors can assemble into a long-range (LR) complex with DNA ends relatively far apart that contains DNAPKcs, XLF, XRCC4, LIG4, and the KU heterodimer and a short-range (SR) complex lacking DNAPKcs that has the ends positioned for ligation. Since the SR complex can form <italic>de novo</italic>, the role of the LR complex (i.e., DNAPKcs) for chromosomal EJ is unclear. We have examined EJ of chromosomal blunt DNA double-strand breaks (DSBs), and found that DNAPKcs is significantly less important than XLF and XRCC4 for such EJ. However, weakening XLF via disrupting interaction interfaces (e.g., disrupting the XLF homodimer interface) causes a marked requirement for DNAPKcs, its kinase activity, and its ABCDE-cluster autophosphorylation sites for blunt DSB EJ. In contrast, other aspects of genome maintenance are sensitive to DNAPKcs kinase inhibition in a manner that is not further enhanced by XLF loss (i.e., suppression of homology-directed repair and structural variants, and IR-resistance). We suggest that DNAPKcs is required to position a weakened XLF in an LR complex that can transition into a functional SR complex for blunt DSB EJ, but also has distinct functions for other aspects of genome maintenance.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.08.463728', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Mistic: an open-source multiplexed image t-SNE viewer', 'authors': \"[{'name': 'Sandhya Prabhakaran'}, {'name': 'Chandler Gatenbee'}, {'name': 'Mark Robertson-Tessi'}, {'name': 'Jeffrey West'}, {'name': 'Amer A Beg'}, {'name': 'Jhanelle E Gray'}, {'name': 'Scott Antonia'}, {'name': 'Robert A Gatenby'}, {'name': 'Alexander R. A. Anderson'}]\", 'abstract': '<p>Understanding the complex ecology of a tumor tissue and the spatio-temporal relationships between its cellular and microenvironment components is becoming a key component of translational research, especially in immune-oncology. The generation and analysis of multiplexed images from patient samples is of paramount importance to facilitate this understanding. In this work, we present Mistic, an open-source multiplexed image t-SNE viewer that enables the simultaneous viewing of multiple 2D images rendered using multiple layout options to provide an overall visual preview of the entire dataset. In particular, the positions of the images can be taken from t-SNE or UMAP coordinates. This grouped view of all the images further aids an exploratory understanding of the specific expression pattern of a given biomarker or collection of biomarkers across all images, helps to identify images expressing a particular phenotype or to select images for subsequent downstream analysis. Currently there is no freely available tool to generate such image t-SNEs. Mistic   is   open-source   and   can   be   downloaded at: https://github.com/MathOnco/Mistic.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.01.21264442', 'tweets': 10, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Hi-C detects genomic structural variants in peripheral blood of pediatric leukemia patients', 'authors': \"[{'name': 'Claire Mallard'}, {'name': 'Michael J Johnston'}, {'name': 'Anna Bobyn'}, {'name': 'Ana Nikolic'}, {'name': 'Bob Argiropoulos'}, {'name': 'Jennifer A Chan'}, {'name': 'Gregory MT Guilcher'}, {'name': 'Marco Gallo'}]\", 'abstract': '<p>B-cell acute lymphoblastic leukemia (B-ALL) is often driven by chromosome translocations that result in recurrent and well-studied gene fusions. Currently, fluorescent in-situ hybridization probes are employed to detect candidate translocations in bone marrow samples from B-ALL patients. Recently Hi-C, a sequencing-based technique originally designed to reconstruct the three-dimensional architecture of the nuclear genome, was shown to effectively recognize structural variants. Here, we demonstrate that Hi-C can be used as a genome-wide assay to detect translocations and other structural variants of potential clinical interest. Structural variants were identified in both bone marrow and peripheral blood samples, including an <italic>ETV6-RUNX1</italic> translocation present in one pediatric B-ALL patient. Our report provides proof-of-principle that Hi-C could be an effective strategy to globally detect driver structural variants in B-ALL peripheral blood specimens, reducing the need for invasive bone marrow biopsies and candidate-based clinical tests.</p>', 'posted': '2021-10-04'}, {'doi': '10.1101/2021.10.08.463575', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Light-induced asymmetries in the embryonic retina are mediated by the vascular system and extracellular matrix', 'authors': \"[{'name': 'Elisabetta Versace'}, {'name': 'Paola Sgado'}, {'name': 'Julia George'}, {'name': 'Jasmine Loveland'}, {'name': 'Joseph Ward'}, {'name': 'Peter Thorpe'}, {'name': 'Lars Juhl Jensen'}, {'name': 'Karen Spencer'}, {'name': 'Silvia Paracchini'}, {'name': 'Giorgio Vallortigara'}]\", 'abstract': '<p>Left-right asymmetries in the nervous system (lateralisation) influence a broad range of behaviours, from social responses to navigation and language. The role and pathways of endogenous and environmental mechanisms in the ontogeny of lateralisation remains to be established. The domestic chick is a model of both endogenous and experience-induced lateralisation driven by light exposure. Following the endogenous rightward rotation of the embryo, the asymmetrical position in the egg results in a greater exposure of the right eye to environmental light. To identify the genetic pathways activated by asymmetric light stimulation, and their time course, we exposed embryos to different light regimes: darkness, 6 hours of light and 24 hours of light. We used RNA-seq to compare gene expression in the right and left retinas and telencephalon. As expected, no differential expression between left and right was present in darkness. We detected differential gene expression in right vs left retina after 6 hours of light exposure. This difference disappeared before 24 hours of light exposure, suggesting that light-induced activation is a self-terminating phenomenon. This transient effect of light exposure was associated with a downregulation of the sensitive-period mediator gene DIO2 (iodothyronine deiodinase 2) in the right retina. No differences between genes expressed in the right vs. left telencephalon were detected. Gene networks associated with lateralisation were connected to vascularisation, cell motility, and the extracellular matrix. Interestingly, we know that the extracellular matrix - including the differentially expressed PDGFRB (platelet-derived growth factor receptor β) gene - is involved in both sensitive periods and in the endogenous chiral mechanism of primary cilia, that drives lateralisation. Our data show a similarity between endogenous and experience-driven lateralisation, identifying functional gene networks that affect lateralisation in a specific time window.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.06.463453', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Multiple regulatory mechanisms control the production of CmrRST, an atypical signal transduction system in Clostridioides difficile', 'authors': \"[{'name': 'Elizabeth M. Garrett'}, {'name': 'Anchal Mehra'}, {'name': 'Ognjen Sekulovic'}, {'name': 'Rita Tamayo'}]\", 'abstract': '<p><italic>Clostridioides difficile</italic>, an intestinal pathogen and leading cause of nosocomial infection, exhibits extensive phenotypic heterogeneity through phase variation by site-specific recombination. The signal transduction system CmrRST, which encodes two response regulators (CmrR and CmrT) and a sensor kinase (CmrS), impacts <italic>C. difficile</italic> cell and colony morphology, surface and swimming motility, biofilm formation, and virulence in an animal model. CmrRST is subject to phase variation through site-specific recombination and reversible inversion of the ‘<italic>cmr</italic> switch’, and expression of <italic>cmrRST</italic> is also regulated by c-di-GMP through a riboswitch. The goal of this study was to determine how the <italic>cmr</italic> switch and c-di-GMP work together to regulate <italic>cmrRST</italic> expression. We generated “phase locked” strains by mutating key residues in the right inverted repeat flanking the <italic>cmr</italic> switch. Phenotypic characterization of these phase locked <italic>cmr</italic>-ON and -OFF strains demonstrates that they cannot switch between rough and smooth colony morphologies, respectively, or other CmrRST-associated phenotypes. Manipulation of c-di-GMP levels in these mutants showed that c-di-GMP promotes <italic>cmrRST</italic> expression and associated phenotypes independent of <italic>cmr</italic> switch orientation. We identified multiple promoters controlling <italic>cmrRST</italic> transcription, including one within the ON orientation of <italic>cmr</italic> switch and another that is positively autoregulated by CmrR. Overall, this work reveals a complex regulatory network that governs <italic>cmrRST</italic> expression and a unique intersection of phase variation and c-di-GMP signaling. These findings suggest that multiple environmental signals impact the production of this signaling transduction system.</p><sec><title>IMPORTANCE</title><p><italic>Clostridioides difficile</italic> is a leading cause of hospital-acquired intestinal infections in the U.S. The CmrRST signal transduction system controls numerous physiological traits and processes in <italic>C. difficile</italic>, including cell and colony morphology, motility, biofilm formation, and virulence. Here we define the complex, multi-level regulation of <italic>cmrRST</italic> expression, including stochastic control through phase variation, modulation by the second messenger c-di-GMP, and positive autoregulation by CmrR. The results of this study suggest that multiple, distinct environmental stimuli and selective pressures must be integrated to appropriately control <italic>cmrRST</italic> expression.</p></sec>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.08.463725', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Mutagenesis-visualization: analysis of site saturation mutagenesis datasets in Python', 'authors': \"[{'name': 'Frank Hidalgo'}, {'name': 'Sage Templeton'}, {'name': 'Che Olavaria Gallegos'}, {'name': 'Joanne Wang'}]\", 'abstract': '<p>Site saturation mutagenesis (SSM) experiments have been transformative in our study of protein function. Despite the rich data generated from such experiments, current tools for processing, analyzing, and visualizing the data offer only a limited set of static plots that are difficult to customize. Furthermore, usage of the tools requires extensive experience and programming. This slows the research process for those in the biological field who are unfamiliar with programming. Here, we introduce mutagenesis-visualization, a Python API for the creation of publication quality figures for SSM datasets which requires no prior Python or statistics experience. The plots can be rendered as native matplotlib objects (easy to stylize) or as Plotly objects (interactive graphs). Additionally, the software offers the possibility to visualize the datasets on Pymol. Availability and implementation: The software can be installed from PyPI or GitHub using the pip package manager and is compatible with Python ≥ 3.8. The documentation can be found at readthedocs and the source code can be found on GitHub.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.05.463266', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'Engineered endosymbionts capable of directing mammalian cell gene expression', 'authors': \"[{'name': 'Cody S. Madsen'}, {'name': 'Ashley V. Makela'}, {'name': 'Emily M. Greeson'}, {'name': 'Jonathan W. Hardy'}, {'name': 'Christopher H. Contag'}]\", 'abstract': '<p>Modular methods for directing mammalian gene expression would enable advances in tissue regeneration, enhance cell-based therapeutics and improve modulation of immune responses. To address this challenge, engineered endosymbionts (EES) that escape endosomal destruction, reside in the cytoplasm of mammalian cells, and secrete proteins that are transported to the nucleus to control host cell gene expression were developed. Microscopy confirmed that EES escape phagosomes, replicate within the cytoplasm, and can secrete reporter proteins into the cytoplasm that were then transported to the nucleus. Synthetic operons encoding the mammalian transcription factors, <italic>Stat-1</italic> and <italic>Klf6</italic> or <italic>Klf4</italic> and <italic>Gata-3</italic> were recombined into the EES genome. Using controlled induction, these EES were shown to direct gene expression in J774A.1 macrophage/monocyte cells and modulate the host cell fates. Expressing mammalian transcription factors from engineered intracellular bacteria as endosymbionts comprises a new tool for directing host cell gene expression for therapeutic and research purposes.</p><sec><title>Graphical abstract</title><fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"463266v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></sec>', 'posted': '2021-10-06'}, {'doi': '10.1101/2021.10.03.462898', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Polyvinyl Chloride Biodegradation Fuels Survival of Invasive Insect Larva and Intestinal Degrading Strain of Klebsiella', 'authors': \"[{'name': 'Zhang Zhe'}, {'name': 'Haoran Peng'}, {'name': 'Dongchen Yang'}, {'name': 'Guoqing Zhang'}, {'name': 'Jinlin Zhang'}, {'name': 'Feng Ju'}]\", 'abstract': '<p>Microbial degradation of polyvinyl chloride (PVC) is eco-friendly and economically attractive, but extremely challenging due to the lack of mechanistic understanding on the degrading strains and enzymes. Motivated by an accidental discovery that the larva of an agricultural invasive pest, <italic>Spodoptera frugiperda</italic>, effectively survived solely on PVC film, we profiled the intestinal microbiota of <italic>S. frugiperda</italic> and screened for PVC-degrading strains. The results showed PVC film feeding significantly changed the larvae intestinal microbiota through selective enrichment of <italic>Enterococcus, Ochrobactrum</italic> and <italic>Klebsiella</italic>. From the larva intestines, we isolated and named a biofilm-forming strain EMBL-1, and experimentally verified it as the first <italic>Klebsiella</italic> bacterium that can actively degrade and utilize PVC based on various classic physicochemical and morphological analyses. We further used multi-omics analyses that complementarily integrate whole genomic, transcriptomic, proteomic, and metabolic insights to identify enzyme-coding genes responsible for PVC degradation and proposed a putative biodegradation pathway by the bacterial strain. All in all, both <italic>S. frugiperda</italic> and its intestinal strain EMBL-1 are discovered to effectively survive on PVC film by exploiting its polymer as a sole energy source. Moreover, this work exemplifying PVC biodegradation provides reference for discovering more degrading microbes and enzymatic resources of other recalcitrant plastics.</p>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.07.463511', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Characterization of divalent cation interactions with AASTY nanodiscs', 'authors': \"[{'name': 'Milena Timcenko'}, {'name': 'Anton A.A. Autzen'}, {'name': 'Henriette E. Autzen'}]\", 'abstract': \"<p>Amphiphilic copolymers show great promise in extracting membrane proteins directly from lipid bilayers into 'native nanodiscs'. However, many such copolymers are polyanionic and sensitive to divalent cations, limiting their applicability towards Ca<sup>2+</sup> or Mg<sup>2+</sup> dependent proteins. Here, we characterize the Ca<sup>2+</sup> and Mg<sup>2+</sup> sensitivity of poly(acrylic acid-co-styrene) (AASTY) copolymers using analytical UV and fluorescent size exclusion chromatography, enabling us to separate signals from nanodiscs, copolymers, and soluble aggregates. Determination of free Ca<sup>2+</sup> ion concentrations in the presence of copolymer shows that divalent cation tolerance is dependent on not only specific characteristics of a copolymer, but also on its concentration. We see that high ionic strength protects against aggregation facilitated by divalent cations, which is prominent in nanodiscs isolated from excess free copolymer through dialysis. Overall, we conclude that the behavior of amphiphilic copolymers in the presence of divalent cations is more complex than precipitation beyond a specific cation concentration.</p>\", 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.05.463216', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Zoology', 'covid': False, 'title': 'Tales of schizomid tails: patterns in schizomid flagellum shape from elliptical Fourier analysis', 'authors': \"[{'name': 'Robert J. Kallal'}, {'name': 'Gustavo Silva de Miranda'}, {'name': 'Erika L. Garcia'}, {'name': 'Hannah M. Wood'}]\", 'abstract': '<p>The arachnid order Schizomida is a relatively understudied group of soil-dwelling predators found on all continents except Antarctica. While efforts to understand their biology are growing, there is still much to know about them. A curious aspect of their morphology is the male flagellum, a sexually dimorphic, tail-like structure which differs in shape across the order and functions in their courtship rituals. The flagellar shape is important for taxonomic classification, yet few efforts have been made to examine shape diversity across the group. Using elliptical Fourier analysis, a type of geometric morphometrics based on outline shape, we quantified shape differences across a combined nearly 550 outlines in the dorsal and lateral views, categorizing them based on genus, family, biogeographic realm, and habitat, with special emphasis on Caribbean and Cuban fauna. We tested for allometric relationships, differences in disparity based on locations and sizes in morphospace among these categories, and for clusters of shapes in morphospace. We found multiple differences in all categories despite apparent overlaps in morphospace, evolutionary allometry, and evidence for discrete clusters in some flagellum shapes. This study can serve as a foundation for further study on the evolution, diversification, and taxonomic utility of the male flagellum.</p>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.07.463524', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Impact of stimulation location relative to grey and white matter on single pulse electrical stimulation responses in the human brain', 'authors': \"[{'name': 'Angelique C Paulk'}, {'name': 'Rina Zelmann'}, {'name': 'Britni Crocker'}, {'name': 'Alik S Widge'}, {'name': 'Darin D Dougherty'}, {'name': 'Emad N Eskandar'}, {'name': 'Daniel S Weisholtz'}, {'name': 'Mark Richardson'}, {'name': 'Rees Cosgrove'}, {'name': 'Sydney S Cash'}]\", 'abstract': '<p>Background: Electrical neuromodulation is an increasingly common therapy for a wide variety of neuropsychiatric diseases. Unfortunately, therapeutic efficacy is inconsistent, possibly a result of our limited understanding of the mechanisms and the massive stimulation parameter space.\\nObjective/Hypothesis: To better understand the role different parameters play in inducing a response, we systematically examined single pulse-induced cortico-cortico evoked potentials (CCEP) as a function of stimulation amplitude, duration and location in the brain and relative to grey and white matter. \\nMethods: We measured voltage peak amplitudes and area under the curve of intracranially recorded stimulation responses as a function of distance from the stimulation site, pulse width, current injected, location relative to grey and white matter, and brain region stimulated (N=52, n=719 stimulation sites). \\nResults: Increasing stimulation pulse width increased response values near the stimulation location. Increasing stimulation amplitude (current) increased responses nonlinearly. Locally (&lt;15 mm from the stimulation site), stimulation closer to the grey matter-white matter boundary induced larger responses. In contrast, for distant sites (&gt;15 mm), white matter stimulation consistently produced larger responses than stimulation in or near grey matter. These relationships were different between cingulate, lateral prefrontal, and lateral temporal cortical stimulation. \\nConclusion: These results demonstrate the importance of location and stimulation parameters in inducing a specific output and indicate that a stronger local response may require stimulation in the grey-white boundary while stimulation in the white matter may be needed for network activation, suggesting that stimulation location can be tailored for a specific outcome, key to informed neuromodulatory therapy.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.07.463402', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Inhibition of SARS-CoV-2 spike protein palmitoylation reduces virus infectivity', 'authors': \"[{'name': 'Ahmed A. Ramadan'}, {'name': 'Karthick Mayilsamy'}, {'name': 'Andrew R. McGill'}, {'name': 'Anandita Ghosh'}, {'name': 'Marc A. Giulianotti'}, {'name': 'Haley M. Donow'}, {'name': 'Shyam S. Mohapatra'}, {'name': 'Subhra Mohapatra'}, {'name': 'Bala Chandran'}, {'name': 'Robert J. Deschenes'}, {'name': 'Arunava Roy'}]\", 'abstract': '<p>Spike glycoproteins of almost all enveloped viruses are known to undergo post-translational attachment of palmitic acid moieties. The precise role of such palmitoylation of the spike protein in membrane fusion and infection is not completely understood. Here, we report that palmitoylation of the first five cysteine residues of the c-terminal cysteine-rich domain of the SARS-CoV-2 spike are indispensable for infection, and palmitoylation deficient spike mutants are defective in trimerization and subsequent membrane fusion. The DHHC9 palmitoyltransferase interacts with and palmitoylates the spike protein in the ER and Golgi, and knockdown of DHHC9 results in reduced fusion and infection of SARS-CoV-2. Two bis-piperazine backbone-based DHHC9 inhibitors inhibit SARS-CoV-2 spike protein palmitoylation and the resulting progeny virion particles released are defective in fusion and infection. This establishes these palmitoyltransferase inhibitors as potential new intervention strategies against SARS-CoV-2.</p>', 'posted': '2021-10-08'}, {'doi': '10.1101/2021.10.03.462970', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'β-amyloid−driven synaptic depression requires PDZ protein interaction at AMPA-receptor subunit GluA3', 'authors': \"[{'name': 'Niels R. Reinders'}, {'name': 'Sophie van der Spek'}, {'name': 'Remco V. Klaassen'}, {'name': 'Karin J. Koymans'}, {'name': 'Ka Wan Li'}, {'name': 'August B. Smit'}, {'name': 'Helmut W. Kessels'}]\", 'abstract': '<p>Soluble oligomeric amyloid-β (Aβ) is a prime suspect to cause cognitive deficits in Alzheimer’s disease and weakens synapses by removing AMPA-type glutamate receptors (AMPARs). We show that synapses of CA1 pyramidal neurons become vulnerable to Aβ when they express AMPAR subunit GluA3. We found that Aβ-oligomers reduce the levels of GluA3 immobilized at spines, indicating they deplete GluA3-containing AMPARs from synapses. These Aβ-driven effects critically depended on the PDZ-binding motif of GluA3. When GluA3 was expressed with a single amino acid mutation in its PDZ-binding motif that prevents GRIP binding, it did not end up at spines and Aβ failed to trigger synaptic depression. GluA3 with a different point mutation in the PDZ-motif that leaves GRIP-binding intact but prevents its endocytosis, was present at spines in normal amounts but was fully resistant to effects of Aβ. Our data indicate that Aβ-mediated synaptic depression requires the removal of GluA3 from synapses. We propose that GRIP-detachment from GluA3 is a critical early step in the cascade of events through which Aβ accumulation causes a loss of synapse.</p>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.06.463384', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Development of a Freeze-Dried CRISPR-Cas12 Sensor for Detecting Wolbachia in the Secondary Science Classroom', 'authors': \"[{'name': 'Grant A Rybnicky'}, {'name': 'Radeen A Dixon'}, {'name': 'Robert M Kuhn'}, {'name': 'Ashty S Karim'}, {'name': 'Michael C Jewett'}]\", 'abstract': '<p>Training the future synthetic biology workforce requires opportunity and exposure to biotechnology concepts and activities in secondary education. Detecting Wolbachia bacteria in arthropods using PCR has become a common way for secondary students to investigate and apply DNA technology in the science classroom. Despite this framework, cutting-edge biotechnologies like CRISPR-based diagnostics have yet to be widely implemented in the classroom. To address this gap, we present a freeze-dried CRISPR-Cas12 sensing reaction to complement traditional DNA technology education and teach synthetic biology concepts. The reactions accurately detect Wolbachia from arthropod-derived PCR samples in under 2 hours and can be stored at room temperature for over a month without appreciable degradation. The reactions are easy-to-use and cost less than $40 to implement for a classroom of 22 students including the cost of reusable equipment. We see this technology as an accessible way to incorporate synthetic biology education into existing biology curriculum, which will expand biology educational opportunities in science, technology, engineering, and mathematics (STEM) education.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.04.21264162', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Inherited variants in CHD3 demonstrate variable expressivity in Snijders Blok-Campeau syndrome', 'authors': \"[{'name': 'Jet van der Spek'}, {'name': 'Joery den Hoed'}, {'name': 'Lot Snijders Blok'}, {'name': 'Alexander J. M. Dingemans'}, {'name': 'Dick Schijven'}, {'name': 'Christoffer Nellaker'}, {'name': 'Hanka Venselaar'}, {'name': 'Tahsin Stefan Barakat'}, {'name': 'E. Martina Bebin'}, {'name': 'Stefanie Beck-Wödl'}, {'name': 'Gea Beunders'}, {'name': 'Natasha J. Brown'}, {'name': 'Theresa Brunet'}, {'name': 'Han G. Brunner'}, {'name': 'Philippe M. Campeau'}, {'name': 'Goran Čuturilo'}, {'name': 'Christian Gilissen'}, {'name': 'Tobias B. Haack'}, {'name': 'Ralf A. Husain'}, {'name': 'Benjamin Kamien'}, {'name': 'Sze Chern Lim'}, {'name': 'Luca Lovrecic'}, {'name': 'Janine Magg'}, {'name': 'Ales Maver'}, {'name': 'Valancy Miranda'}, {'name': 'Danielle C. Monteil'}, {'name': 'Charlotte W. Ockeloen'}, {'name': 'Lynn S. Pais'}, {'name': 'Vasilica Plaiasu'}, {'name': 'Laura Raiti'}, {'name': 'Christopher Richmond'}, {'name': 'Angelika Rieß'}, {'name': 'Eva M. C. Schwaibold'}, {'name': 'Marleen E. H. Simon'}, {'name': 'Stephanie Spranger'}, {'name': 'Tiong Yang Tan'}, {'name': 'Michelle L. Thompson'}, {'name': 'Bert B.A. de Vries'}, {'name': 'Ella J. Wilkins'}, {'name': 'Marjolein H. Willemsen'}, {'name': 'Clyde Francks'}, {'name': 'Lisenka E. L. M. Vissers'}, {'name': 'Simon E. Fisher'}, {'name': 'Tjitske Kleefstra'}]\", 'abstract': '<p>Interpretation of next-generation sequencing data of individuals with an apparent sporadic neurodevelopmental disorder (NDD) often focusses on pathogenic variants in genes associated with NDD, assuming full clinical penetrance with limited variable expressivity. Consequently, inherited variants in genes associated with dominant disorders may be overlooked when the transmitting parent is clinically unaffected. While <italic>de novo</italic> variants explain a substantial proportion of cases with NDDs, a significant number remains undiagnosed possibly explained by coding variants associated with reduced penetrance and variable expressivity. We characterized twenty families with inherited heterozygous missense or protein-truncating variants (PTVs) in <italic>CHD3</italic>, a gene in which <italic>de novo</italic> variants cause Snijders Blok-Campeau syndrome, characterized by intellectual disability, speech delay and recognizable facial features (SNIBCPS). Notably, the majority of the inherited <italic>CHD3</italic> variants were maternally transmitted. Computational facial and human phenotype ontology-based comparisons demonstrated that the phenotypic features of probands with inherited <italic>CHD3</italic> variants overlap with the phenotype previously associated with <italic>de novo</italic> variants in the gene, while carrier parents are mildly or not affected, suggesting variable expressivity. Additionally, similarly reduced expression levels of CHD3 protein in cells of an affected proband and of related healthy carriers with a <italic>CHD3</italic> PTV, suggested that compensation of expression from the wildtype allele is unlikely to be an underlying mechanism. Our results point to a significant role of inherited variation in SNIBCPS, a finding that is critical for correct variant interpretation and genetic counseling and warrants further investigation towards understanding the broader contributions of such variation to the landscape of human disease.</p>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.08.463705', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Inferring the mode and strength of ongoing selection', 'authors': \"[{'name': 'Gustavo Valadares Barroso'}, {'name': 'Kirk Lohmueller'}]\", 'abstract': '<p>Genome sequence data is no longer scarce. The UK Biobank alone comprises 200,000 individual genomes, with more on the way, leading the field of human genetics towards sequencing entire populations. Within the next decades, other model organisms will follow suit, especially domesticated species such as crops and livestock. Having sequences from most individuals in a population will present new challenges for using these data to improve health and agriculture in the pursuit of a sustainable future. Existing population genetic methods are designed to model hundreds of randomly sampled sequences, but are not optimized for extracting the information contained in the larger and richer datasets that are beginning to emerge, with thousands of closely related individuals. Here we develop a new method called TIDES (Trio-based Inference of Dominance and Selection) that uses data from tens of thousands of family trios to make inferences about natural selection acting in a single generation. TIDES further improves on the state-of-the-art by making no assumptions regarding demography, linkage or dominance. We discuss how our method paves the way for studying natural selection from new angles.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.07.463234', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': True, 'title': 'High-throughput Activity Assay for Screening Inhibitors of the SARS-CoV-2 Mac1 Macrodomain', 'authors': \"[{'name': 'Morgan Dasovich'}, {'name': 'Junlin Zhuo'}, {'name': 'Jack A. Goodman'}, {'name': 'Ajit Thomas'}, {'name': 'Robert Lyle McPherson'}, {'name': 'Aravinth Kumar Jayabalan'}, {'name': 'Veronica F. Busa'}, {'name': 'Shang-Jung Cheng'}, {'name': 'Brennan A. Murphy'}, {'name': 'Karli R. Redinger'}, {'name': 'Takashi Tsukamoto'}, {'name': 'Barbara Slusher'}, {'name': 'Jürgen Bosch'}, {'name': 'Huijun Wei'}, {'name': 'Anthony K. L. Leung'}]\", 'abstract': '<p>Macrodomains are a class of conserved ADP-ribosylhydrolases expressed by viruses of pandemic concern, including coronaviruses and alphaviruses. Viral macrodomains are critical for replication and virus-induced pathogenesis; therefore, these enzymes are a promising target for antiviral therapy. However, no potent or selective viral macrodomain inhibitors currently exist, in part due to the lack of a high-throughput assay for this class of enzymes. Here, we developed a high-throughput ADP-ribosylhydrolase assay using the SARS-CoV-2 macrodomain Mac1. We performed a pilot screen which identified dasatinib and dihydralazine as ADP-ribosylhydrolase inhibitors. Importantly, dasatinib does not inhibit MacroD2, the closest Mac1 homolog in humans. Our study demonstrates the feasibility of identifying selective inhibitors based on ADP-ribosylhydrolase activity, paving the way for screening large compound libraries to identify improved macrodomain inhibitors and explore their potential as antiviral therapies for SARS-CoV-2 and future viral threats.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.07.463515', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genomic and antigenic diversity of carried Klebsiella pneumoniae isolates mirrors that of invasive isolates in Blantyre, Malawi', 'authors': \"[{'name': 'Joseph M Lewis'}, {'name': 'Madalitso Mphasa'}, {'name': 'Rachel Banda'}, {'name': 'Mathew A Beale'}, {'name': 'Jane Mallewa'}, {'name': 'Eva Heinz'}, {'name': 'Nicholas R Thomson'}, {'name': 'Nicholas A Feasey'}]\", 'abstract': '<p>Klebsiella pneumoniae is an antimicrobial resistance (AMR) associated pathogen of global importance, and polyvalent vaccines targeting K. pneumoniae O-antigens are in development. Genomes from sub-Saharan Africa (sSA) are underrepresented in global sequencing efforts. We therefore carried out a genomic analysis of extended-spectrum beta-lactamase (ESBL)-producing K. pneumoniae complex isolates colonising adults in Blantyre, Malawi, placed these isolates in a global genomic context, and compared colonising to invasive isolates from the main public hospital in Blantyre. \\n\\n203 isolates from stool and rectal swabs from adults were whole-genome sequenced and compared to a publicly available multicountry collection of 484 K. pneumoniae genomes sampled to cover maximum diversity of the species, 150 previously sequenced Malawian and 66 Kenyan isolates from blood or sterile sites. We inferred phylogenetic relationships and analysed the diversity of genetic loci linked to AMR, virulence, capsule (K-) and LPS O-antigen (O-types). \\n\\nWe find that the diversity of Malawian Klebsiella isolates is representative of the species population structure, but with local success and expansion of sequence types (STs) ST14, ST15, ST340 and ST307. Siderophore and hypermucoidy genes were more frequent in invasive versus carriage isolates (present in 13% vs 1%, p &lt; 0.001) but still generally lacking in most invasive isolates. The population structure and distribution of O-antigen types was similar in Malawian invasive and carriage isolates, with O4 being more common in Malawian isolates (14%) than in previously published studies (2-5%). We conclude that host factors, pathogen opportunity or alternate virulence loci not linked to invasive disease elsewhere are likely to be the major determinants of invasive disease in Malawi. Distinct ST and O-type distributions in Malawi highlights the need for geographically aware sampling to robustly define secular trends in Klebsiella diversity. Colonising and invasive isolates in Blantyre are similar and hence O-typing of colonising Klebsiella isolates may be a rapid and cost-effective approach to describe global diversity and guide vaccine development.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.09.21264771', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Allergy and Immunology', 'covid': True, 'title': 'Taste loss as a distinct symptom of COVID-19: A systematic review and meta-analysis', 'authors': \"[{'name': 'Mackenzie R Hannum'}, {'name': 'Riley Koch'}, {'name': 'Vicente Ramirez'}, {'name': 'Sarah Marks'}, {'name': 'Aurora Toskala'}, {'name': 'Riley Herriman'}, {'name': 'Cailu Lin'}, {'name': 'Paule Valery Joseph'}, {'name': 'Danielle R Reed'}]\", 'abstract': '<p>Chemosensory scientists have been skeptical that reports of COVID–19 taste loss are genuine, in part because, before COVID–19, taste loss was rare and often confused with smell loss. Therefore, to establish the predicted prevalence rate of taste loss in COVID–19 patients, we conducted a systematic review and meta–analysis of 376 papers published in 2020 and 2021, with 241 of the papers meeting all inclusion criteria. Additionally, we explored how methodological differences (direct vs. self-report measures) may affect these estimates. We hypothesized that direct prevalence measures of taste loss would be the most valid because they avoid the taste/smell confusion of self-report. The meta–analysis showed that, among 138,897 COVID–19–positive patients, 39.2% reported taste dysfunction (95% CI: 35.34–43.12%), and the prevalence estimates were slightly but not significantly higher from studies using direct (n = 18) versus self-report (n = 223) methodologies (Q = 0.57, df = 1, p = 0.45). Generally, males reported lower rates of taste loss than did females and taste loss was highest in middle-aged groups. Thus, taste loss is a bona fide symptom COVID–19, meriting further research into the most appropriate direct methods to measure it and its underlying mechanisms.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.06.463447', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genomic and phenotypic characterization of Pseudomonas hygromyciniae, a novel bacterial species discovered from a commercially purchased antibiotic', 'authors': \"[{'name': 'Timothy L. Turner'}, {'name': 'Sumitra D. Mitra'}, {'name': 'Travis J. Kochan'}, {'name': 'Nathan B. Pincus'}, {'name': 'Marine Lebrun-Corbin'}, {'name': 'Bettina Cheung'}, {'name': 'Samuel W. Gatesy'}, {'name': 'Tania Afzal'}, {'name': 'Egon A. Ozer'}, {'name': 'Alan R. Hauser'}]\", 'abstract': '<p>A purchased lot of the antibiotic hygromycin B was found to be contaminated with a novel bacterial species, which we designate <italic>Pseudomonas hygromyciniae</italic>. Characteristics of <italic>P. hygromyciniae</italic> include its ability to use a variety of compounds as carbon sources, its pathogenicity towards lettuce and <italic>Galleria mellonella</italic>, and its ability to inhibit the growth of an <italic>E. coli</italic> strain. <italic>P. hygromyciniae</italic> is unlikely to be a human pathogen, as it did not survive at 37 °C and was not cytotoxic towards a mammalian cell line. The <italic>P. hygromyciniae</italic> strain harbors a novel 250 kb megaplasmid which confers resistance to hygromycin B and contains numerous other genes predicted to encode replication and conjugation machinery. These findings indicate that commercially manufactured antibiotics represent another extreme environment that may support the growth of novel bacterial species.</p><sec><title>IMPORTANCE</title><p>Microbial ecologists have surveyed numerous natural and manmade environments in search of new microbial species. In some instances, these microbes are discovered in harsh conditions, such as deep-sea vents, and their discovery leads to better understanding of how microbes adapt to their environment. Here, we have discovered a new species of bacteria from an extreme manmade environment: a lyophilized, commercially available bottle of the antibiotic hygromycin B.</p></sec>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.09.462574', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Deep learning-based tumor microenvironment segmentation is predictive of tumor mutations and patient survival in non-small-cell lung cancer', 'authors': \"[{'name': 'Łukasz Rączkowski'}, {'name': 'Iwona Paśnik'}, {'name': 'Michał Kukiełka'}, {'name': 'Marcin Nicoś'}, {'name': 'Magdalena A Budzinska'}, {'name': 'Tomasz Kucharczyk'}, {'name': 'Justyna Szumiło'}, {'name': 'Paweł Krawczyk'}, {'name': 'Nicola Crosetto'}, {'name': 'Ewa Szczurek'}]\", 'abstract': '<p>Despite the fact that tumor microenvironment (TME) and gene mutations are the main determinants of progression of the deadliest cancer in the world - lung cancer - their interrelations are not well understood. Digital pathology data provide a unique insight into the spatial composition of the TME. Here, we generated 23,199 image patches from 55 hematoxylin-and-eosin (H&amp;E)-stained lung cancer tissue sections and annotated them into 9 different tissue classes. Using this dataset, we trained a deep neural network and used it to segment 467 lung cancer H&amp;E images downloaded from The Cancer Genome Atlas (TCGA) database. We used the segmented images to compute human interpretable features reflecting the heterogeneous composition of the TME, and successfully utilized them to predict patient survival (c-index 0.723) and cancer gene mutations (largest AUC 73.5% for PDGFRB). Our approach can be generalized to different cancer types to inform precision medicine strategies.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.06.463450', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'High throughput identification of genes conferring resistance or sensitivity to toxic effectors delivered by the type VI secretion system', 'authors': \"[{'name': 'Steven J. Hersch'}, {'name': 'Rehnuma Tabassum Sejuty'}, {'name': 'Kevin Manera'}, {'name': 'Tao G. Dong'}]\", 'abstract': '<p>The type six secretion system (T6SS) is a prevalent bacterial weapon delivering toxic effector proteins into nearby competitors. In addition to immunity genes that protect against a particular effector, alternate yet crucial nonspecific defences have also recently been identified. To systematically identify genes influencing T6SS susceptibility in numerous species, we designed a Tn-Seq-based competition assay. Combined with follow-up analyses using <italic>E. coli</italic> and <italic>V. cholerae</italic> gene knockout collections, we demonstrate that our Tn-Seq competition technique can be used to identify both immunity and non-immunity defences against the T6SS. We also identify <italic>E. coli</italic> proteins that facilitate T6SS-mediated cell death, including metabolic genes such as <italic>cyaA</italic> and <italic>gltA,</italic> where mutant strains were resistant to attack. Our findings act as a proof-of-concept for the technique while also illuminating novel genes of interest. Since Tn-Seq can be applied in numerous species, our method has broad potential for identifying diverse T6SS defence genes across genomes in a high-throughput manner.</p><sec><title>Importance</title><p>The type six secretion system (T6SS) is a molecular poison-tipped spear that bacteria use to kill nearby competitors. To prevent self killing, they use antitoxins called immunity genes that specifically neutralize the poisons. Beyond immunity genes, multiple additional defences have recently been discovered but there are likely many more across the genomes of diverse species. To help discover these novel mechanisms, we designed a high-throughput method that can be used in numerous different species to rapidly identify genes involved in sensitivity to T6SS attacks. Using T6SS ‘killers’ delivering individual poisons and two commonly studied ‘prey’ bacteria, we show proof-of-principle that the technique can discover proteins that make the prey cells more resistant or sensitive to particular poisons. This will greatly improve the speed at which genes influencing the T6SS can be identified and selected for further study in follow-up analyses.</p></sec>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.07.463537', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'The archaeal division protein CdvB1 assembles into polymers that are depolymerized by CdvC', 'authors': \"[{'name': 'Alberto Blanch Jover'}, {'name': 'Nicola De Franceschi'}, {'name': 'Daphna Fenel'}, {'name': 'Winfried Weissenhorn'}, {'name': 'Cees Dekker'}]\", 'abstract': '<p>The Cdv proteins constitute the cell-division system of the Crenarchaea, in a protein machinery that is closely related to the ESCRT system of eukaryotes. The CdvB paralog CdvB1 is believed to play a major role in the constricting ring that is the central actor in cell division in the crenarchaea. Here, we present an <italic>in vitro</italic> study of purified CdvB1 from the crenarchaeon <italic>M. sedula</italic> with a combination of TEM imaging and biochemical assays. We show that CdvB1 self-assembles into filamentous polymers that are depolymerized by the action of the Vps4-homolog ATPase CdvC. Using liposome flotation assays, we show that CdvB1 binds to negatively charged lipid membranes and can be detached from the membrane by the action of CdvC. Interestingly, we find that the polymerization and the membrane binding are mutually exclusive properties of the protein. Our findings provide novel insight into one of the main components of the archaeal cell division machinery.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.07.463504', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'PBP1 of Staphylococcus aureus has multiple essential functions in cell division', 'authors': \"[{'name': 'Katarzyna Wacnik'}, {'name': 'Vincenzo A Rao'}, {'name': 'Xinyue Chen'}, {'name': 'Lucia Lafage'}, {'name': 'Manuel Pazos'}, {'name': 'Simon Booth'}, {'name': 'Waldemar Vollmer'}, {'name': 'Jamie K Hobbs'}, {'name': 'Richard J Lewis'}, {'name': 'Simon J Foster'}]\", 'abstract': '<p>Bacterial cell division is a complex process requiring the coordination of multiple components, to allow the appropriate spatial and temporal control of septum formation and cell scission. Peptidoglycan (PG) is the major structural component of the septum, and our recent studies in the human pathogen <italic>Staphylococcus aureus</italic> have revealed a complex, multi–stage PG architecture that develops during septation. Penicillin binding proteins (PBPs) are essential for the final steps of PG biosynthesis — their transpeptidase activity links together the peptide sidechain of nascent glycan strands together. PBP1 is required for cell division in <italic>S. aureus</italic> and here we demonstrate that it has multiple essential functions associated with its enzymatic activity and as a regulator of division. Loss of PBP1, or just its C–terminal PASTA domains, results in cessation of division at the point of septal plate formation. The PASTA domains can bind PG and thus coordinate the cell division process. The transpeptidase activity of PBP1 is also essential but its loss leads to a strikingly different phenotype of thickened and aberrant septa, which is phenocopied by the morphological effects of adding the PBP1–specific β–lactam, meropenem. Together these results lead to a model for septal PG synthesis where PBP1 enzyme activity is responsible for the characteristic architecture of the septum and PBP1 protein molecules coordinate cell division allowing septal plate formation.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.05.463160', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'The In Vivo Source of Type I and Type III IFNs is Pathogen Dependent', 'authors': \"[{'name': 'Marvin J. Sandoval'}, {'name': 'Hsiang-Chi Tseng'}, {'name': 'Heidi P. Risman'}, {'name': 'Sergey Smirnov'}, {'name': 'Qing Li'}, {'name': 'Jian-Da Lin'}, {'name': 'Amariliz Rivera'}, {'name': 'Russell K. Durbin'}, {'name': 'Sergei V. Kotenko'}, {'name': 'Joan E. Durbin'}]\", 'abstract': '<p>Type I (-α, β) and type III (-λ) interferons (IFNs) are produced in response to virus infection and upregulate a largely overlapping set of IFN stimulated genes which mediate the protective effects of these antiviral cytokines. <italic>In vitro</italic> studies have demonstrated the redundancy of these two cytokine families which activate the same transcription factor, IFN stimulated gene factor 3 (ISGF3), via distinct ligands and receptors. However, <italic>in vivo</italic>, these IFN types do have distinct functions based on receptor distribution, but also ligand availability. Using a newly generated IFN-λ reporter mouse strain we have observed that both type I and type III IFNs are produced in response to respiratory tract infection by Newcastle disease virus (NDV) and influenza A virus (IAV). In the case of NDV these IFNs are synthesized by different cell types. Type I IFNs are produced primarily by alveolar macrophages, type III IFNs are made only by epithelial cells, and production of either is dependent on MAVS. While epithelial cells of the respiratory tract represent the primary target of IAV infection, we found that they did not significantly contribute to IFN-λ production, and IFN-λ protein levels were largely unaffected in the absence of MAVS. Instead we found that pDCs, a cell type known for robust IFN-α production via TLR/MyD88 signaling, were the major producers of IFN-λ during IAV infection, with pDC depletion during influenza infection resulting in significantly reduced levels of both IFN-α and IFN-λ. In addition, we were able to demonstrate that pDCs rely on type I IFN for optimal IFN-λ production. These studies therefore demonstrate that the <italic>in vivo</italic> producers of Type III IFNs in response to respiratory virus infection are pathogen dependent, a finding which may explain the varying levels of cytokine production induced by different viral pathogens.</p>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.06.463290', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'In planta dynamics, transport-biases and endogenous functions of mobile siRNAs in Arabidopsis', 'authors': \"[{'name': 'Christopher A. Brosnan'}, {'name': 'Emanuel A. Devers'}, {'name': 'Alexis Sarazin'}, {'name': 'Peiqi Lim'}, {'name': 'Satu Lehesranta'}, {'name': 'Yrjö Helariutta'}, {'name': 'Olivier Voinnet'}]\", 'abstract': '<p><list list-type=\"bullet\"><list-item><p>In RNA interference (RNAi), small-interfering (si)RNAs processed from double-stranded RNA guide ARGONAUTE(AGO) proteins to silence sequence-complementary RNA/DNA. Plant RNAi can propagate locally and systemically, but despite recent mechanistic advances, basic questions/hurdles remain unaddressed. For instance, RNAi is inferred to diffuse through plasmodesmata, yet how its dynamics <italic>in planta</italic> compares with that of established symplastic-diffusion markers remains unknown. Also unknown is why select siRNA species, or size-classes thereof, are recovered in RNAi-recipient tissues, yet only under some experimental settings. Finally, RNAi shootward movement in micro-grafted Arabidopsis – necessary to study its presumptive transgenerational effects– has not been achieved thus far and endogenous functions of mobile RNAi remain scarcely documented.</p></list-item><list-item><p>Focusing on non-amplified RNAi in <italic>Arabidopsis</italic>, we show here that (<italic>i</italic>) transgenic RNAi-movement, although symplasmic, only partially recapitulates the diffusion pattern of free GFP <italic>in planta</italic>, (<italic>ii</italic>) the presence/absence of specific AGOs in incipient/traversed/recipient tissues likely explains the apparent siRNA-selectivity observed during vascular movement, (<italic>iii</italic>) stress application allows endo-siRNA translocation against the shoot-to-root phloem flow, and (<italic>iv</italic>) mobile endo-siRNAs generated from a single inverted-repeat(<italic>IR</italic>) locus, have the potential to regulate hundreds of transcripts.</p></list-item><list-item><p>Our results close important knowledge-gaps, rationalize previously-noted inconsistencies between mobile RNAi settings, and provide a framework for functional endo-siRNA studies.</p></list-item></list></p>', 'posted': '2021-10-06'}, {'doi': '10.1101/2021.10.06.463337', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Progesterone supplementation in mice leads to microbiome alterations and weight gain in a sex-specific manner', 'authors': \"[{'name': 'Meital Nuriel-Ohayon'}, {'name': 'Anna Belogovski'}, {'name': 'Sharon Komissarov'}, {'name': 'Meirav Ben Izhak'}, {'name': 'Oshrit Shtossel'}, {'name': 'Hadar Neuman'}, {'name': 'Oren Ziv'}, {'name': 'Sondra Turjeman'}, {'name': 'Shai Bel'}, {'name': 'Yoram Louzoun'}, {'name': 'Omry Koren'}]\", 'abstract': '<sec><title>Background</title><p>Progesterone is a steroid hormone produced by the ovaries, involved in pregnancy progression and necessary for successful gestation. We have previously shown that progesterone affects gut microbiota composition and leads to increased relative abundance of <italic>Bifidobacterium</italic>.</p></sec><sec><title>Results</title><p>In non-pregnant female GF mice, levels of progesterone were significantly higher than in SPF mice of the same status. However, no significant differences were observed between GF and SPF males. Females treated with progesterone gained more weight than females treated with a placebo. In contrast to female mice, males treated with progesterone did not gain significantly more weight than males treated with a placebo. Progesterone supplementation led to microbial changes in females but not in males (16S rRNA sequencing). Accordingly, the weight gain observed in female mice treated with progesterone was fully transferable to both male and female germ-free mice via fecal transplantation.</p></sec><sec><title>Conclusions</title><p>We demonstrate that bacteria play a role in regulating progesterone levels in a female-specific manner. Furthermore, weight gain and metabolic changes associated with progesterone may be mediated by the gut microbiota.</p></sec>', 'posted': '2021-10-06'}, {'doi': '10.1101/2021.10.05.463175', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'DetecDiv, a deep-learning platform for automated cell division tracking and replicative lifespan analysis', 'authors': \"[{'name': 'Theo Aspert'}, {'name': 'Didier Hentsch'}, {'name': 'Gilles Charvin'}]\", 'abstract': '<p>Automating the extraction of meaningful temporal information from sequences of microscopy images represents a major challenge to characterize dynamical biological processes. Here, we have developed DetecDiv, a microfluidic-based image acquisition platform combined with deep learning-based software for high-throughput single-cell division tracking. DetecDiv can reconstruct cellular replicative lifespans with an outstanding accuracy and provides comprehensive temporal cellular metrics using time series classification and image semantic segmentation.</p>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.07.463474', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Population genetic bottleneck has only marginal effects on fitness evolution and its repeatability in dioecious C. elegans', 'authors': \"[{'name': 'Karen Bisschop'}, {'name': 'Thomas Blankers'}, {'name': 'Janine Mariën'}, {'name': 'Meike T Wortel'}, {'name': 'Martijn Egas'}, {'name': 'Astrid T. Groot'}, {'name': 'Marcel Visser'}, {'name': 'Jacintha Ellers'}]\", 'abstract': '<p>Evolution is a key process by which populations can adapt to novel conditions, but it is not well understood how predictable this process is. Predictability is expected to depend on the ratio of deterministic and stochastic processes that contribute to evolutionary change and this ratio is modulated by the effective population size. Smaller effective populations harbor less genetic diversity and stochastic processes are generally expected to play a larger role, leading to less repeatable evolutionary trajectories. Empirical insight into the relationship between effective population size and repeatability is limited and biased towards asexual unicellular organisms. Here, we used populations of obligately outcrossing <italic>Caenorhabditis elegans</italic> to test whether fitness increase and selection response were more heterogeneous after a moderate or strong population bottleneck compared to a scenario without bottleneck. Nematodes were exposed to a novel bacterial prey and lower temperature. Population sizes after one week of growth (as a proxy of fitness) were measured before and after 15 generations of evolution. We found that replicates across all (no/moderate/strong bottleneck) treatments evolved higher fitness and no significant difference in average or maximum fitness was found among treatments. Partitioning fitness variance among effects from selection and effects from chance showed that a strong (but not a moderate) bottleneck reduced the relative contribution of selection effects to fitness variation. However, the reduced contribution from selection did not translate to a significant reduction in the repeatability of fitness evolution. Thus, although a strong bottleneck reduced the contribution of deterministic evolutionary change, we found only marginal effects on quantitative measurements of repeatability in evolution. We conclude that the extent to which evolution is predictable may not universally depend on effective population size.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.05.463158', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A spiking neural network model of the Superior Colliculus that is robust to changes in the spatial-temporal input', 'authors': \"[{'name': 'Arezoo Alizadeh'}, {'name': 'A. John Van Opstal'}]\", 'abstract': '<p>Previous studies have indicated that the location of a large neural population in the Superior Colliculus (SC) motor map specifies the amplitude and direction of the saccadic eye-movement vector, while the saccade trajectory and velocity profile are encoded by the population firing rates. We recently proposed a simple spiking neural network model of the SC motor map, based on linear summation of individual spike effects of each recruited neuron, which accounts for many of the observed properties of SC cells in relation to the ensuing eye movement. However, in the model, the cortical input was kept invariant across different saccades. Electrical microstimulation and reversible lesion studies have demonstrated that the saccade properties are quite robust against large changes in supra-threshold SC activation, but that saccade amplitude and peak eye-velocity systematically decrease at low input strengths. These features are not accounted for by the linear spike-vector summation model. Here we show that the model’s input projection strengths and intra-collicular lateral connections can be tuned to generate saccades that follow the experimental results.</p><sec><title>Author statement</title><p>The midbrain SC generates fast saccadic eye movements through a large population of cells within a topographically organized motor map, in which the location, spike count and temporal firing patterns of recruited cells determine saccade metrics and kinematics. According to the dynamic ensemble-coding model, each recruited SC cell contributes to the saccade by linear vector summation of all its spike contributions. Our previous spiking neural network model used invariant cortical inputs to the SC cells for all saccades. We here improved the robustness of the model to large spatial-temporal variations in the input patterns, by tuning its top-down and lateral synaptic connections, to generate saccades with properties observed in electrophysiological experiments.</p></sec>', 'posted': '2021-10-06'}, {'doi': '10.1101/2021.10.07.463499', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genomic and metabolic adaptations of biofilms to ecological windows of opportunities in glacier-fed streams', 'authors': \"[{'name': 'Susheel Bhanu Busi'}, {'name': 'Massimo Bourquin'}, {'name': 'Stilianos Fodelianakis'}, {'name': 'Gregoire Michoud'}, {'name': 'Tyler J Kohler'}, {'name': 'Hannes Peter'}, {'name': 'Paraskevi Pramateftaki'}, {'name': 'Michail Styllas'}, {'name': 'Matteo Tolosano'}, {'name': 'Vincent De Staercke'}, {'name': 'Martina Schon'}, {'name': 'Laura de Nies'}, {'name': 'Ramona Marasco'}, {'name': 'Daniele Daffonchio'}, {'name': 'Leila Ezzat'}, {'name': 'Paul Wilmes'}, {'name': 'Tom J Battin'}]\", 'abstract': '<p>Microorganisms dominate life in cryospheric ecosystems. In glacier-fed streams (GFSs), ecological windows of opportunities allow complex microbial biofilms to develop and transiently form the basis of the food web, thereby controlling key ecosystem processes. Here, using high-resolution metagenomics, we unravel strategies that allow biofilms to seize this opportunity in an ecosystem otherwise characterized by harsh environmental conditions. We found a diverse microbiome spanning the entire tree of life and including a rich virome. Various and co-existing energy acquisition pathways point to diverse niches and the simultaneous exploitation of available resources, likely fostering the establishment of complex biofilms in GFSs during windows of opportunity. The wide occurrence of rhodopsins across metagenome-assembled genomes (MAGs), besides chlorophyll, highlights the role of solar energy capture in these biofilms. Concomitantly, internal carbon and nutrient cycling between photoautotrophs and heterotrophs may help overcome constraints imposed by the high oligotrophy in GFSs. MAGs also revealed mechanisms potentially protecting bacteria against low temperatures and high UV-radiation. The selective pressure of the GFS environment is further highlighted by the phylogenomic analysis, differentiating the representatives of the genus Polaromonas, an important component of the GFS microbiome, from those found in other ecosystems. Our findings reveal key genomic underpinnings of adaptive traits that contribute to the success of complex biofilms to exploit environmental opportunities in GFSs, now rapidly changing owing to global warming.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.08.462716', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Expression dynamics of ARGONAUTE proteins during meiosis in Arabidopsis', 'authors': \"[{'name': 'Cecilia Oliver'}, {'name': 'German Martinez'}]\", 'abstract': '<p>Meiosis is a specialized cell division that is key for reproduction and genetic diversity in sexually reproducing plants. Recently, different RNA silencing pathways have been proposed to carry a specific activity during meiosis, but the pathways involved during this process remain unclear. Here, we explored the subcellular localization of different ARGONAUTE (AGO) proteins, the main effectors of RNA silencing, during male meiosis in Arabidopsis thaliana using immunolocalizations with commercially available antibodies. We detected the presence of AGO proteins associated with posttranscriptional gene silencing (AGO1, 2 and 5) in the cytoplasm or the nucleus, while AGOs associated with transcriptional gene silencing (AGO4 and 9) localized exclusively in the nucleus. These results indicate that the localization of different AGOs correlates with their predicted roles at the transcriptional and posttranscriptional levels and provide an overview of their timing and potential role during meiosis.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.08.463704', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Wind and route choice affect performance of bees flying above versus within a cluttered obstacle field', 'authors': \"[{'name': 'Nicholas Burnett'}, {'name': 'Marc Badger'}, {'name': 'Stacey Combes'}]\", 'abstract': '<p>Bees flying through natural landscapes encounter physical challenges, such as wind and cluttered vegetation. The influence of these factors on the flight performance of bees remains unknown. We analyzed 548 videos of wild-caught honeybees (<italic>Apis mellifera</italic>) flying through an enclosure containing a field of vertical obstacles that bees could fly within (through open corridors, without maneuvering) or above. We examined how obstacle field height, wind presence and direction (headwinds or tailwinds) affected altitude, ground speed, and side-to-side casting (lateral excursions) of bees. When obstacle fields were short, bees flew at altitudes near the midpoint between the tunnel floor and ceiling. When obstacle fields approached or exceeded this midpoint, bees typically, but not always, increased their altitudes to fly over the obstacles. Bees that flew above the obstacle fields exhibited 40% faster ground speeds and 36% larger lateral excursions than bees that flew within the obstacle fields, likely due to the visual feedback from obstacles and narrow space available within the obstacle field. Wind had a strong effect on ground speed and lateral excursions, but not altitude. Bees flew 12-19% faster in tailwinds than in the other wind conditions, but their lateral excursions were 19% larger in any wind, regardless of its direction, than in still air. Our results show that bees flying through complex environments display flexible flight behaviors (e.g., flying above versus within obstacles), which affect flight performance. Similar choices in natural landscapes could have broad implications for foraging efficiency, pollination, and mortality in wild bees.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.08.463691', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Convergent evolution of polyploid genomes from across the eukaryotic tree of life', 'authors': \"[{'name': 'Yue Hao'}, {'name': 'Jonathon Fleming'}, {'name': 'Joanna Petterson'}, {'name': 'Eric Lyons'}, {'name': 'Patrick P. Edger'}, {'name': 'Chris Pires'}, {'name': 'Jeffrey L. Thorne'}, {'name': 'Gavin C. Conant'}]\", 'abstract': '<p>By modeling the homoeologous gene losses that occurred in fifty genomes deriving from ten distinct polyploidy events, we show that the evolutionary forces acting on polyploids are remarkably similar, regardless of whether they occur in flowering plants, ciliates, fishes or yeasts. The models suggest these events were nearly all allopolyploidies, with two distinct progenitors contributing to the modern species. We show that many of the events show a relative rate of duplicate gene loss prior to the first post-polyploidy speciation that is significantly higher than in later phases of their evolution. The relatively low selective constraint seen for the single-copy genes these losses produced lead us to suggest that most of the purely selectively neutral duplicate gene losses occur in the immediate post-polyploid period. We also find ongoing and extensive reciprocal gene losses (RGL; alternative losses of duplicated ancestral genes) between these genomes. With the exception of a handful of closely related taxa, all of these polyploid organisms are separated from each other by tens to thousands of reciprocal gene losses. As a result, it is very unlikely that viable diploid hybrid species could form between these taxa, since matings between such hybrids would tend to produce offspring lacking essential genes. It is therefore possible that the relatively high frequency of recurrent polyploidies in some lineages may be due to the ability of new polyploidies to bypass RGL barriers.</p>', 'posted': '2021-10-10'}, {'doi': '10.1101/2021.10.07.463563', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'LysM receptors in Coffea arabica: identification, characterization, and gene expression in response to Hemileia vastatrix', 'authors': \"[{'name': 'Mariana de Lima Santos'}, {'name': 'Mário Lúcio Vilela de Resende'}, {'name': 'Bárbara Alves dos Santos Ciscon'}, {'name': 'Natália Chagas Freitas'}, {'name': 'Matheus Henrique de Brito Pereira'}, {'name': 'Tharyn Reichel'}, {'name': 'Sandra Marisa Mathioni'}]\", 'abstract': '<p>Pathogen-associated molecular patterns (PAMPs) are recognized by pattern recognition receptors (PRRs) localized on the host plant cell wall. These receptors activate a broad-spectrum and durable defense, which are desired characteristics for disease resistance in plant breeding programs. In this study, candidate sequences for PRRs with lysin motifs (LysM) were investigated in the <italic>Coffea arabica</italic> genome. For this, approaches based on the principle of sequence similarity, conservation of motifs and domains, phylogenetic analysis, and modulation of gene expression in response to <italic>Hemileia vastatrix</italic> were used. The candidate sequences for PRRs in <italic>C. arabica</italic> (<italic>Ca1-LYP, Ca2-LYP, Ca1-CERK1, Ca2-CERK1, Ca-LYK4, Ca1-LYK5</italic> and <italic>Ca2-LYK5</italic>) showed high similarity with the reference PRRs used: <italic>Os-CEBiP, At-CERK1, At-LYK4</italic> and <italic>At-LYK5</italic>. Moreover, the ectodomains of these sequences showed high identity or similarity with the reference sequences, indicating structural and functional conservation. The studied sequences are also phylogenetically related to the reference PRRs described in Arabidopsis, rice, and other plant species. All candidates for receptors had their expression induced after the inoculation with <italic>H. vastatrix</italic>, since the first time of sampling at 6 hours post-inoculation (hpi). At 24 hpi, there was a significant increase in expression, for most of the receptors evaluated, and at 48 hpi, a suppression. The results showed that the candidate sequences for PRRs in the <italic>C. arabica</italic> genome display high homology with fungal PRRs already described in the literature. Besides, they respond to pathogen inoculation and seem to be involved in the perception or signaling of fungal chitin, acting as receptors or coreceptors of this molecule. These findings represent an advance in the understanding of the basal immunity of this species.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.07.463549', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Evolutionary sweeps of subviral parasites and their phage host bring unique parasite variants and disappearance of a phage CRISPR-Cas system', 'authors': \"[{'name': 'Angus Angermeyer'}, {'name': 'Stephanie G. Hays'}, {'name': 'Maria H.T. Nguyen'}, {'name': 'Fatema-tuz Johura'}, {'name': 'Marzia Sultana'}, {'name': 'Munirul Alam'}, {'name': 'Kimberley D. Seed'}]\", 'abstract': '<p><italic>Vibrio cholerae</italic> is a significant threat to global public health in part due to its propensity for large-scale evolutionary sweeps where lineages emerge and are replaced. These sweeps may originate from the Bay of Bengal where bacteriophage predation and the evolution of anti-phage counter defenses is a recurring theme. The bacteriophage ICP1 is a key predator of epidemic <italic>V. cholerae</italic> and is notable for acquiring a CRISPR-Cas system to combat PLE, a defensive subviral parasite encoded by its <italic>V. cholerae</italic> host. Here we describe the discovery of five previously unknown PLE variants, including one found during recent surveillance of patient samples in Bangladesh. We also observed a lineage sweep of PLE negative <italic>V. cholerae</italic> occurring within a patient population in under a year which coincided with a loss of ICP1’s CRISPR-Cas system. These findings reinforce the importance of surveillance to better understand the selective pressures that drive pandemic cholera.</p>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.07.463602', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'CCR5 closes the temporal window for memory linking', 'authors': \"[{'name': 'Yang Shen'}, {'name': 'Miou Zhou'}, {'name': 'Denise Cai'}, {'name': 'Daniel Almeida Filho'}, {'name': 'Giselle Fernandes'}, {'name': 'Ying Cai'}, {'name': 'Nury Kim'}, {'name': 'Deanna Necula'}, {'name': 'Chengbin Zhou'}, {'name': 'Andy Liu'}, {'name': 'Xiaoman Kang'}, {'name': 'Masakazu Kamata'}, {'name': 'Ayal Lavi'}, {'name': 'Shan Huang'}, {'name': 'Tawnie Silva'}, {'name': 'Won Do Heo'}, {'name': 'Alcino J. Silva'}]\", 'abstract': '<p>Real world memories are formed in a particular context and are not acquired or recalled in isolation. Time is a key variable in the organization of memories, since events experienced close in time are more likely to be meaningfully associated, while those experienced with a longer interval are not. How does the brain segregate events that are temporally distinct? Here, we report that a delayed (12-24h) increase in the expression of the C-C chemokine receptor type 5 (CCR5), an immune receptor well known as a co-receptor for HIV infection, following the formation of a contextual memory, determines the duration of the temporal window for associating or linking that memory with subsequent memories. This delayed CCR5 expression in mouse dorsal CA1 (dCA1) neurons results in a decrease in neuronal excitability, which in turn negatively regulates neuronal memory allocation, thus reducing the overlap between dCA1 memory ensembles. Lowering this overlap affects the ability of one memory to trigger the recall of the other, thus closing the temporal window for memory linking. Remarkably, our findings also show that an age-related increase in CCL5/CCR5 expression leads to impairments in memory linking in aged mice, which could be reversed with a CCR5 knockout and an FDA approved drug that inhibits this receptor, a result with significant clinical implications. All together the findings reported here provide the first insights into the molecular and cellular mechanisms that shape the temporal window for memory linking.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.07.463487', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': \"Self-regulation of the brain's Beta rhythm using a brain-computer interface improves inhibitory control\", 'authors': \"[{'name': 'Nadja Enz'}, {'name': 'Jemima Schmidt'}, {'name': 'Kate Nolan'}, {'name': 'Matthew Mitchell'}, {'name': 'Sandra Alvarez Gomez'}, {'name': 'Miryam Alkayyali'}, {'name': 'Pierce Cambay'}, {'name': 'Magdalena Gippert'}, {'name': 'Robert Whelan'}, {'name': 'Kathy L Ruddy'}]\", 'abstract': \"<p>Neural oscillations, or brain rhythms, fluctuate in a manner reflecting ongoing behavior. Whether these fluctuations are instrumental or epiphenomenal to the behavior remains elusive. Attempts to experimentally manipulate neural oscillations exogenously using non-invasive brain stimulation have shown some promise, but difficulty with tailoring stimulation parameters to individuals has hindered progress in this field. We demonstrate here using electroencephalography (EEG) neurofeedback in a brain-computer interface that human participants (n=44) learned over multiple sessions across a 6-day period to self-regulate their Beta rhythm (13-20 Hz) over the right inferior frontal cortex (rIFC). This Beta modulation had observable consequences on cognitive behavior: changes in an individual's inhibitory control performance before and after training were predicted by the magnitude of their training-related change in Beta over rIFC. This was not the case for a control group (n=38) who underwent training of their Alpha rhythm (8-12 Hz). The present data support the view that the right frontal Beta rhythm is a key component of the brain's inhibitory control system. Most importantly, we introduce causality to this relationship for the first time, as experimental modulation of rIFC Beta activity had a measurable impact upon behavior.</p>\", 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.06.463319', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Harmoni: a Method for Eliminating Spurious Interactions due to the Harmonic Components in Neuronal Data', 'authors': \"[{'name': 'Mina Jamshidi Idaji'}, {'name': 'Juanli Zhang'}, {'name': 'Tilman Stephani'}, {'name': 'Guido Nolte'}, {'name': 'Klaus-Robert Mueller'}, {'name': 'Arno Villringer'}, {'name': 'Vadim Nikulin'}]\", 'abstract': \"<p>Cross-frequency synchronization (CFS) has been proposed as a mechanism for integrating spatially and spectrally distributed information in the brain. However, investigating CFS in Magneto- and Electroencephalography (MEG/EEG) is hampered by the presence of spurious neuronal interactions due to the non-sinusoidal waveshape of brain oscillations. Such waveshape gives rise to the presence of oscillatory harmonics mimicking genuine neuronal oscillations. Until recently, however, there has been no methodology for removing these harmonics from neuronal data. In order to address this long-standing challenge, we introduce a novel method (called HARMOnic miNImization - Harmoni) that removes the signal components which can be harmonics of a non-sinusoidal signal. Harmoni's working principle is based on the presence of CFS between harmonic components and the fundamental component of a non-sinusoidal signal. \\nWe extensively tested Harmoni in realistic EEG simulations. The simulated couplings between the source signals represented genuine and spurious CFS and within-frequency phase synchronization. Using diverse evaluation criteria, including ROC analyses, we showed that the within- and cross-frequency spurious interactions are suppressed significantly, while the genuine activities are not affected. Additionally, we applied Harmoni to real resting-state EEG data revealing intricate remote connectivity patterns which are usually masked by the spurious connections. Given the ubiquity of non-sinusoidal neuronal oscillations in electrophysiological recordings, Harmoni is expected to facilitate novel insights into genuine neuronal interactions in various research fields, and can also serve as a steppingstone towards the development of further signal processing methods aiming at refining within- and cross-frequency synchronization in electrophysiological recordings.</p>\", 'posted': '2021-10-08'}, {'doi': '10.1101/2021.10.07.463461', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Comparison between Lotka-Volterra and Multivariate Autoregressive models of ecological interaction systems', 'authors': \"[{'name': 'Daniel Vigário Olivença'}, {'name': 'Jacob D. Davis'}, {'name': 'Eberhard O. Voit'}]\", 'abstract': '<p>Lotka-Volterra (LV) and Multivariate Autoregressive (MAR) models are computational frameworks with different mathematical structures that have both been proposed for the same purpose of extracting governing features of dynamic interactions among coexisting populations of different species from observed time series data. We systematically compare the feasibility of the two modeling approaches, using four synthetically generated datasets and seven ecological datasets from the literature. The overarching result is that LV models outperform MAR models in most cases and are generally superior for representing cases where the dependent variables deviate greatly from their steady states. A large dynamic range is particularly prevalent when the populations are highly abundant, change considerably over time, and exhibit a large signal-to-noise ratio. By contrast, MAR models are better suited for analyses of populations with low abundances and for investigations where the quantification of noise is important. We conclude that the choice of either one or the other modeling framework should be guided by the specific goals of the analysis and the dynamic features of the data.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.07.463455', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Vernalization-triggered expression of the antisense transcript COOLAIR is mediated by CBF genes', 'authors': \"[{'name': 'Myeongjune Jeon'}, {'name': 'Goowon Jeong'}, {'name': 'Youbong Hyun'}, {'name': 'Ilha Lee'}]\", 'abstract': '<p>Many plants undergo vernalization, a long-term winter-triggered acceleration of flowering, to align their flowering time with spring. In Arabidopsis thaliana, this is achieved by silencing a floral repressor, FLOWERING LOCUS C (FLC). COOLAIR, an antisense noncoding RNA expressed at the FLC locus, is induced during the early phase of vernalization, preceding FLC suppression. However, the mechanism by which long-term cold induces COOLAIR is not well understood. Here, we showed that C-repeat (CRT)/dehydration-responsive elements (DREs) at the 3′-end of FLC and CRT/DRE-binding factors (CBFs) are required for vernalization-induced COOLAIR activation. The CBFs bind to CRT/DREs at the 3′-end of FLC, both in vitro and in vivo, and the CBFs levels increased gradually during vernalization. Additionally, vernalization-induced COOLAIR expression was highly suppressed in the cbfs mutant, in which all CBFs were knocked-out. Contrastingly, CBF-overexpressing plants showed COOLAIR upregulation, even at warm temperatures. We propose that COOLAIR is induced by CBFs in the early phase of vernalization but is downregulated as CBFs are evicted from closed FLC chromatin during the late vernalization phase. We also demonstrated that cbfs and COOLAIR mutants have a normal vernalization response, although they show defects in vernalization-induced COOLAIR activation, indicating that COOLAIR is not necessary for this process.</p>', 'posted': '2021-10-09'}, {'doi': '10.1101/2021.10.05.463153', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Biomarker Candidates for Tumors Identified from Deep-Profiled Plasma Stem Predominantly from the Low Abundant Area', 'authors': \"[{'name': 'Marco Tognetti'}, {'name': 'Kamil Sklodowski'}, {'name': 'Sebastian Müller'}, {'name': 'Dominique Kamber'}, {'name': 'Jan Muntel'}, {'name': 'Roland Bruderer'}, {'name': 'Lukas Reiter'}]\", 'abstract': '<p>The plasma proteome has the potential to enable a holistic analysis of the health state of an individual. However, plasma biomarker discovery is difficult due to its high dynamic range and variability. Here, we present a novel automated analytical approach for deep plasma profiling and applied it to a 180-sample cohort of human plasma from lung, breast, colorectal, pancreatic, and prostate cancer.</p><p>Using a controlled quantitative experiment, we demonstrate a 257% increase in protein identification and a 263% increase in significantly differentially abundant proteins over neat plasma.</p><p>In the cohort, we identified 2,732 proteins. Using machine learning, we discovered biomarker candidates such as STAT3 in colorectal cancer and developed models that classify the disease state. For pancreatic cancer, a separation by stage was achieved.</p><p>Importantly, biomarker candidates came predominantly from the low abundance region, demonstrating the necessity to deeply profile because they would have been missed by shallow profiling.</p>', 'posted': '2021-10-06'}, {'doi': '10.1101/2021.10.04.462986', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Stochastic variational variable selection for high-dimensional microbiome data', 'authors': \"[{'name': 'Tung Dang'}, {'name': 'Kie Kumaishi'}, {'name': 'Erika Usui'}, {'name': 'Shungo Kobori'}, {'name': 'Takumi Sato'}, {'name': 'Yusuke Toda'}, {'name': 'Yuji Yamasaki'}, {'name': 'Hisashi Tsujimoto'}, {'name': 'Yasunori Ichihashi'}, {'name': 'Hiroyoshi Iwata'}]\", 'abstract': '<sec><title>Background</title><p>The rapid and accurate identification of a minimal-size core set of representative microbial species plays an important role in the clustering of microbial community data and interpretation of the clustering results. However, the huge dimensionality of microbial metagenomics data sets is a major challenge for the existing methods such as Dirichlet multinomial mixture (DMM) models. In the framework of the existing methods, computational burdens for identifying a small number of representative species from a huge number of observed species remain a challenge.</p></sec><sec><title>Results</title><p>We proposed a novel framework to improve the performance of the widely used DMM approach by combining three ideas: (i) We extended the finite DMM model to the infinite case, via the consideration of Dirichlet process mixtures and estimate the number of clusters as a random variables. (ii) We proposed an indicator variable to identify representative operational taxonomic units that substantially contribute to the differentiation among clusters. (iii) To address the computational burdens of the high-dimensional microbiome data, we proposed are a stochastic variational inference, which approximates the posterior distribution using a controllable distribution called variational distribution, and stochastic optimization algorithms for fast computation. With the proposed method named stochastic variational variable selection (SVVS), we analyzed the root microbiome data collected in our soybean field experiment and the human gut microbiome data from three published data sets of large-scale case-control studies.</p></sec><sec><title>Conclusions</title><p>SVVS demonstrated a better performance and significantly faster computation than existing methods in all cases of testing data sets. In particular, SVVS is the only method that can analyze the massive high-dimensional microbial data with above 50,000 microbial species and 1,000 samples. Furthermore, it was suggested that microbial species selected as a core set played important roles in the recent microbiome studies.</p></sec>', 'posted': '2021-10-05'}, {'doi': '10.1101/2021.10.07.463523', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Genomic analysis of extended-spectrum beta-lactamase (ESBL) producing Escherichia coli colonising adults in Blantyre, Malawi reveals previously undescribed diversity', 'authors': \"[{'name': 'Joseph M. Lewis'}, {'name': 'Madalitso Mphasa'}, {'name': 'Rachel Banda'}, {'name': 'Mathew A. Beale'}, {'name': 'Jane Mallewa'}, {'name': 'Adam P. Roberts'}, {'name': 'Eva Heinz'}, {'name': 'Nicholas R. Thomson'}, {'name': 'Nicholas A Feasey'}]\", 'abstract': '<p><italic>Escherichia coli</italic> is a ubiquitous bacterial species, associated with drug resistant infections; hundreds of thousands of genomes are now available, but are biased towards high-income countries and clinical isolates. Data from sub-Saharan Africa (sSA) are underrepresented in global sequencing efforts and may represent a major source of genetic diversity with respect to transmissible antimicrobial resistance (AMR). We carried out a genomic investigation of extended-spectrum beta-lactamase (ESBL)-producing <italic>E. coli</italic> colonising adults in Blantyre, Malawi to assess the diversity and AMR determinants and to place these isolates in the context of globally available genomes. We carried out short-read whole-genome sequencing of 473 colonising ESBL <italic>E. coli</italic> isolated from stool and placed them in the context of a previous curated species wide collection of 10,146 isolates using the popPUNK clustering algorithm and by constructing a core gene phylogeny. The most frequently identified STs in Malawian isolates were the globally successful ST131 and ST410, and <italic>bla</italic><sub>CTX-M</sub> were the dominant ESBL genes, mirroring global trends. However, 37% of Malawian isolates did not cluster with any isolates in the global collection, and the core gene phylogeny was consistent with local subclades including in ST410 and several phylogroup A lineages. Apparent undescribed diversity in Malawian <italic>E. coli</italic> could be due to local selection pressures or sampling biases in global <italic>E. coli</italic> collections. Taking a one health approach to further sampling of <italic>E. coli</italic> from Malawi and sSA, and principled incorporation into unbiased global collections is necessary to understand local, regional and global transmission of both <italic>E. coli</italic> and priority AMR genes.</p><sec><title>Data Summary</title><p>All data and code to replicate this analysis is available as the <italic>blantyreESBL</italic> v1.0.0 R package (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://doi.org/10.5281/zenodo.5554082\">https://doi.org/10.5281/zenodo.5554082</ext-link>) available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/joelewis101/blantyreESBL\">https://github.com/joelewis101/blantyreESBL</ext-link>. Reads from all isolates sequenced as part of this study have been deposited in the European Nucleotide Archive, and accession numbers (as well as accession numbers of publicly available genomes used in this analysis) are provided in the R package.</p></sec>', 'posted': '2021-10-07'}, {'doi': '10.1101/2021.10.06.463322', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A Helicase-tethered ORC Flip Enables Bidirectional Helicase Loading', 'authors': \"[{'name': 'Shalini Gupta'}, {'name': 'Larry J. Friedman'}, {'name': 'Jeff Gelles'}, {'name': 'Stephen P. Bell'}]\", 'abstract': '<p>Replication origins are licensed by loading two Mcm2-7 helicases around DNA in a head-to-head conformation poised to initiate bidirectional replication. This process requires ORC, Cdc6, and Cdt1. Although different Cdc6 and Cdt1 molecules load each helicase, whether two ORC proteins are required is unclear. Using colocalization single-molecule spectroscopy combined with FRET, we investigated interactions between ORC and Mcm2-7 during helicase loading. We demonstrate that a single ORC molecule can recruit both Mcm2-7/Cdt1 complexes via similar interactions that end upon Cdt1 release. Between the first and second helicase recruitment, we observe a rapid change in interactions between ORC and the first Mcm2-7. In quick succession ORC breaks the interactions mediating first Mcm2-7 recruitment, releases from its initial DNA-binding site, and forms a new interaction with the opposite face of the first Mcm2-7. This rearrangement requires release of the first Cdt1 and tethers ORC as it flips over the first Mcm2-7 to form an inverted Mcm2-7-ORC-DNA complex required for second-helicase recruitment. To ensure correct licensing, this complex is maintained until head-to-head interactions between the two helicases are formed. Our findings reconcile previous observations and reveal a highly-coordinated series of events through which a single ORC molecule can load two oppositely-oriented helicases.</p>', 'posted': '2021-10-06'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.10.08.463746</td>\n",
       "      <td>56</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioengineering</td>\n",
       "      <td>False</td>\n",
       "      <td>Deep-Learning Super-Resolution Microscopy Reve...</td>\n",
       "      <td>[{'name': 'Rong Chen'}, {'name': 'Xiao Tang'},...</td>\n",
       "      <td>&lt;p&gt;Single-molecule localization microscopy (SM...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.10.02.21264267</td>\n",
       "      <td>39</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Year-long COVID-19 infection reveals within-ho...</td>\n",
       "      <td>[{'name': 'Veronique Nussenblatt'}, {'name': '...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;B-cell deplet...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.10.06.21264535</td>\n",
       "      <td>37</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Analytical performance of eleven SARS-CoV-2 an...</td>\n",
       "      <td>[{'name': 'Meriem Bekliz'}, {'name': 'Kenneth ...</td>\n",
       "      <td>&lt;p&gt;Global concerns arose as the emerged and ra...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.10.05.463267</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Intercellular transport of RNA can limit herit...</td>\n",
       "      <td>[{'name': 'Nathan Shugarts'}, {'name': 'Andrew...</td>\n",
       "      <td>&lt;p&gt;RNAs in circulation carry sequence-specific...</td>\n",
       "      <td>2021-10-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.10.04.21264500</td>\n",
       "      <td>22</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Obstetrics and Gynecology</td>\n",
       "      <td>True</td>\n",
       "      <td>Increase in preterm stillbirths and reduction ...</td>\n",
       "      <td>[{'name': 'Lisa Hui'}, {'name': 'Melvin Barrie...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Objectives&lt;/title&gt;&lt;p&gt;The COVID-19 ...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.10.07.463568</td>\n",
       "      <td>18</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Molecular rhythm alterations in prefrontal cor...</td>\n",
       "      <td>[{'name': 'Xiangning Xue'}, {'name': 'Wei Zong...</td>\n",
       "      <td>&lt;p&gt;Severe and persistent disruptions to sleep ...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.10.08.463671</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>High-throughput mutagenesis identifies mutatio...</td>\n",
       "      <td>[{'name': 'Mariela Cortés-López'}, {'name': 'L...</td>\n",
       "      <td>&lt;p&gt;During CART-19 immunotherapy for B-cell acu...</td>\n",
       "      <td>2021-10-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.10.07.463475</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Pseudo-dynamic analysis of heart tube formatio...</td>\n",
       "      <td>[{'name': 'Isaac Esteban'}, {'name': 'Patrick ...</td>\n",
       "      <td>&lt;p&gt;Understanding organ morphogenesis requires ...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.10.07.463556</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Large-scale discovery of candidate type VI sec...</td>\n",
       "      <td>[{'name': 'Alexander Martin Geller'}, {'name':...</td>\n",
       "      <td>&lt;p&gt;Type VI secretion systems (T6SS) are common...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.10.04.462880</td>\n",
       "      <td>13</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Nanoengineered DNA origami with repurposed TOP...</td>\n",
       "      <td>[{'name': 'Keying Zhu'}, {'name': 'Yang Wang'}...</td>\n",
       "      <td>&lt;p&gt;Targeting myeloid cells, especially microgl...</td>\n",
       "      <td>2021-10-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.10.04.463138</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Linking contrast sensitivity to cortical magni...</td>\n",
       "      <td>[{'name': 'Marc M. Himmelberg'}, {'name': 'Jon...</td>\n",
       "      <td>&lt;p&gt;The size and organization of primary visual...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.10.06.462913</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Minimal required exposures reveal the primacy ...</td>\n",
       "      <td>[{'name': 'Renzo C. Lanfranco'}, {'name': 'And...</td>\n",
       "      <td>&lt;p&gt;The human visual system is capable of detec...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.10.08.463652</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A sense-antisense RNA interaction promotes bre...</td>\n",
       "      <td>[{'name': 'Bruce Culbertson'}, {'name': 'Krist...</td>\n",
       "      <td>&lt;p&gt;Antisense RNAs are ubiquitous in human cell...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.10.07.463578</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The DNAPKcs long-range C-NHEJ complex is requi...</td>\n",
       "      <td>[{'name': 'Metztli Cisneros-Aguirre'}, {'name'...</td>\n",
       "      <td>&lt;p&gt;Canonical non-homologous end joining (C-NHE...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.10.08.463728</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Mistic: an open-source multiplexed image t-SNE...</td>\n",
       "      <td>[{'name': 'Sandhya Prabhakaran'}, {'name': 'Ch...</td>\n",
       "      <td>&lt;p&gt;Understanding the complex ecology of a tumo...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.10.01.21264442</td>\n",
       "      <td>10</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Hi-C detects genomic structural variants in pe...</td>\n",
       "      <td>[{'name': 'Claire Mallard'}, {'name': 'Michael...</td>\n",
       "      <td>&lt;p&gt;B-cell acute lymphoblastic leukemia (B-ALL)...</td>\n",
       "      <td>2021-10-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.10.08.463575</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Light-induced asymmetries in the embryonic ret...</td>\n",
       "      <td>[{'name': 'Elisabetta Versace'}, {'name': 'Pao...</td>\n",
       "      <td>&lt;p&gt;Left-right asymmetries in the nervous syste...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.10.06.463453</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Multiple regulatory mechanisms control the pro...</td>\n",
       "      <td>[{'name': 'Elizabeth M. Garrett'}, {'name': 'A...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Clostridioides difficile&lt;/italic&gt;, ...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.10.08.463725</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Mutagenesis-visualization: analysis of site sa...</td>\n",
       "      <td>[{'name': 'Frank Hidalgo'}, {'name': 'Sage Tem...</td>\n",
       "      <td>&lt;p&gt;Site saturation mutagenesis (SSM) experimen...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.10.05.463266</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioengineering</td>\n",
       "      <td>False</td>\n",
       "      <td>Engineered endosymbionts capable of directing ...</td>\n",
       "      <td>[{'name': 'Cody S. Madsen'}, {'name': 'Ashley ...</td>\n",
       "      <td>&lt;p&gt;Modular methods for directing mammalian gen...</td>\n",
       "      <td>2021-10-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.10.03.462898</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Polyvinyl Chloride Biodegradation Fuels Surviv...</td>\n",
       "      <td>[{'name': 'Zhang Zhe'}, {'name': 'Haoran Peng'...</td>\n",
       "      <td>&lt;p&gt;Microbial degradation of polyvinyl chloride...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.10.07.463511</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Characterization of divalent cation interactio...</td>\n",
       "      <td>[{'name': 'Milena Timcenko'}, {'name': 'Anton ...</td>\n",
       "      <td>&lt;p&gt;Amphiphilic copolymers show great promise i...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.10.05.463216</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Zoology</td>\n",
       "      <td>False</td>\n",
       "      <td>Tales of schizomid tails: patterns in schizomi...</td>\n",
       "      <td>[{'name': 'Robert J. Kallal'}, {'name': 'Gusta...</td>\n",
       "      <td>&lt;p&gt;The arachnid order Schizomida is a relative...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.10.07.463524</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Impact of stimulation location relative to gre...</td>\n",
       "      <td>[{'name': 'Angelique C Paulk'}, {'name': 'Rina...</td>\n",
       "      <td>&lt;p&gt;Background: Electrical neuromodulation is a...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.10.07.463402</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>Inhibition of SARS-CoV-2 spike protein palmito...</td>\n",
       "      <td>[{'name': 'Ahmed A. Ramadan'}, {'name': 'Karth...</td>\n",
       "      <td>&lt;p&gt;Spike glycoproteins of almost all enveloped...</td>\n",
       "      <td>2021-10-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.10.03.462970</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>β-amyloid−driven synaptic depression requires ...</td>\n",
       "      <td>[{'name': 'Niels R. Reinders'}, {'name': 'Soph...</td>\n",
       "      <td>&lt;p&gt;Soluble oligomeric amyloid-β (Aβ) is a prim...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.10.06.463384</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Development of a Freeze-Dried CRISPR-Cas12 Sen...</td>\n",
       "      <td>[{'name': 'Grant A Rybnicky'}, {'name': 'Radee...</td>\n",
       "      <td>&lt;p&gt;Training the future synthetic biology workf...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.10.04.21264162</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Inherited variants in CHD3 demonstrate variabl...</td>\n",
       "      <td>[{'name': 'Jet van der Spek'}, {'name': 'Joery...</td>\n",
       "      <td>&lt;p&gt;Interpretation of next-generation sequencin...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.10.08.463705</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Inferring the mode and strength of ongoing sel...</td>\n",
       "      <td>[{'name': 'Gustavo Valadares Barroso'}, {'name...</td>\n",
       "      <td>&lt;p&gt;Genome sequence data is no longer scarce. T...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.10.07.463234</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>True</td>\n",
       "      <td>High-throughput Activity Assay for Screening I...</td>\n",
       "      <td>[{'name': 'Morgan Dasovich'}, {'name': 'Junlin...</td>\n",
       "      <td>&lt;p&gt;Macrodomains are a class of conserved ADP-r...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.10.07.463515</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Genomic and antigenic diversity of carried Kle...</td>\n",
       "      <td>[{'name': 'Joseph M Lewis'}, {'name': 'Madalit...</td>\n",
       "      <td>&lt;p&gt;Klebsiella pneumoniae is an antimicrobial r...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.10.09.21264771</td>\n",
       "      <td>5</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Allergy and Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Taste loss as a distinct symptom of COVID-19: ...</td>\n",
       "      <td>[{'name': 'Mackenzie R Hannum'}, {'name': 'Ril...</td>\n",
       "      <td>&lt;p&gt;Chemosensory scientists have been skeptical...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.10.06.463447</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Genomic and phenotypic characterization of Pse...</td>\n",
       "      <td>[{'name': 'Timothy L. Turner'}, {'name': 'Sumi...</td>\n",
       "      <td>&lt;p&gt;A purchased lot of the antibiotic hygromyci...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.10.09.462574</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Deep learning-based tumor microenvironment seg...</td>\n",
       "      <td>[{'name': 'Łukasz Rączkowski'}, {'name': 'Iwon...</td>\n",
       "      <td>&lt;p&gt;Despite the fact that tumor microenvironmen...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.10.06.463450</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>High throughput identification of genes confer...</td>\n",
       "      <td>[{'name': 'Steven J. Hersch'}, {'name': 'Rehnu...</td>\n",
       "      <td>&lt;p&gt;The type six secretion system (T6SS) is a p...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.10.07.463537</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>The archaeal division protein CdvB1 assembles ...</td>\n",
       "      <td>[{'name': 'Alberto Blanch Jover'}, {'name': 'N...</td>\n",
       "      <td>&lt;p&gt;The Cdv proteins constitute the cell-divisi...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.10.07.463504</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>PBP1 of Staphylococcus aureus has multiple ess...</td>\n",
       "      <td>[{'name': 'Katarzyna Wacnik'}, {'name': 'Vince...</td>\n",
       "      <td>&lt;p&gt;Bacterial cell division is a complex proces...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.10.05.463160</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>The In Vivo Source of Type I and Type III IFNs...</td>\n",
       "      <td>[{'name': 'Marvin J. Sandoval'}, {'name': 'Hsi...</td>\n",
       "      <td>&lt;p&gt;Type I (-α, β) and type III (-λ) interferon...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.10.06.463290</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>In planta dynamics, transport-biases and endog...</td>\n",
       "      <td>[{'name': 'Christopher A. Brosnan'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;&lt;list list-type=\"bullet\"&gt;&lt;list-item&gt;&lt;p&gt;In R...</td>\n",
       "      <td>2021-10-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.10.06.463337</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Progesterone supplementation in mice leads to ...</td>\n",
       "      <td>[{'name': 'Meital Nuriel-Ohayon'}, {'name': 'A...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Progesterone ...</td>\n",
       "      <td>2021-10-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.10.05.463175</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>DetecDiv, a deep-learning platform for automat...</td>\n",
       "      <td>[{'name': 'Theo Aspert'}, {'name': 'Didier Hen...</td>\n",
       "      <td>&lt;p&gt;Automating the extraction of meaningful tem...</td>\n",
       "      <td>2021-10-05</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.10.07.463474</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Population genetic bottleneck has only margina...</td>\n",
       "      <td>[{'name': 'Karen Bisschop'}, {'name': 'Thomas ...</td>\n",
       "      <td>&lt;p&gt;Evolution is a key process by which populat...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.10.05.463158</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>A spiking neural network model of the Superior...</td>\n",
       "      <td>[{'name': 'Arezoo Alizadeh'}, {'name': 'A. Joh...</td>\n",
       "      <td>&lt;p&gt;Previous studies have indicated that the lo...</td>\n",
       "      <td>2021-10-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.10.07.463499</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Genomic and metabolic adaptations of biofilms ...</td>\n",
       "      <td>[{'name': 'Susheel Bhanu Busi'}, {'name': 'Mas...</td>\n",
       "      <td>&lt;p&gt;Microorganisms dominate life in cryospheric...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.10.08.462716</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Expression dynamics of ARGONAUTE proteins duri...</td>\n",
       "      <td>[{'name': 'Cecilia Oliver'}, {'name': 'German ...</td>\n",
       "      <td>&lt;p&gt;Meiosis is a specialized cell division that...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.10.08.463704</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Animal Behavior and Cognition</td>\n",
       "      <td>False</td>\n",
       "      <td>Wind and route choice affect performance of be...</td>\n",
       "      <td>[{'name': 'Nicholas Burnett'}, {'name': 'Marc ...</td>\n",
       "      <td>&lt;p&gt;Bees flying through natural landscapes enco...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.10.08.463691</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Convergent evolution of polyploid genomes from...</td>\n",
       "      <td>[{'name': 'Yue Hao'}, {'name': 'Jonathon Flemi...</td>\n",
       "      <td>&lt;p&gt;By modeling the homoeologous gene losses th...</td>\n",
       "      <td>2021-10-10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.10.07.463563</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>LysM receptors in Coffea arabica: identificati...</td>\n",
       "      <td>[{'name': 'Mariana de Lima Santos'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Pathogen-associated molecular patterns (PAM...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.10.07.463549</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Evolutionary sweeps of subviral parasites and ...</td>\n",
       "      <td>[{'name': 'Angus Angermeyer'}, {'name': 'Steph...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Vibrio cholerae&lt;/italic&gt; is a signi...</td>\n",
       "      <td>2021-10-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.10.07.463602</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>CCR5 closes the temporal window for memory lin...</td>\n",
       "      <td>[{'name': 'Yang Shen'}, {'name': 'Miou Zhou'},...</td>\n",
       "      <td>&lt;p&gt;Real world memories are formed in a particu...</td>\n",
       "      <td>2021-10-09</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.10.08.463746      56  bioRxiv   \n",
       "1   10.1101/2021.10.02.21264267      39  medRxiv   \n",
       "2   10.1101/2021.10.06.21264535      37  medRxiv   \n",
       "3     10.1101/2021.10.05.463267      26  bioRxiv   \n",
       "4   10.1101/2021.10.04.21264500      22  medRxiv   \n",
       "5     10.1101/2021.10.07.463568      18  bioRxiv   \n",
       "6     10.1101/2021.10.08.463671      15  bioRxiv   \n",
       "7     10.1101/2021.10.07.463475      14  bioRxiv   \n",
       "8     10.1101/2021.10.07.463556      14  bioRxiv   \n",
       "9     10.1101/2021.10.04.462880      13  bioRxiv   \n",
       "10    10.1101/2021.10.04.463138      12  bioRxiv   \n",
       "11    10.1101/2021.10.06.462913      12  bioRxiv   \n",
       "12    10.1101/2021.10.08.463652      12  bioRxiv   \n",
       "13    10.1101/2021.10.07.463578      12  bioRxiv   \n",
       "14    10.1101/2021.10.08.463728      11  bioRxiv   \n",
       "15  10.1101/2021.10.01.21264442      10  medRxiv   \n",
       "16    10.1101/2021.10.08.463575      10  bioRxiv   \n",
       "17    10.1101/2021.10.06.463453      10  bioRxiv   \n",
       "18    10.1101/2021.10.08.463725       8  bioRxiv   \n",
       "19    10.1101/2021.10.05.463266       8  bioRxiv   \n",
       "20    10.1101/2021.10.03.462898       8  bioRxiv   \n",
       "21    10.1101/2021.10.07.463511       8  bioRxiv   \n",
       "22    10.1101/2021.10.05.463216       7  bioRxiv   \n",
       "23    10.1101/2021.10.07.463524       7  bioRxiv   \n",
       "24    10.1101/2021.10.07.463402       7  bioRxiv   \n",
       "25    10.1101/2021.10.03.462970       7  bioRxiv   \n",
       "26    10.1101/2021.10.06.463384       7  bioRxiv   \n",
       "27  10.1101/2021.10.04.21264162       6  medRxiv   \n",
       "28    10.1101/2021.10.08.463705       6  bioRxiv   \n",
       "29    10.1101/2021.10.07.463234       6  bioRxiv   \n",
       "30    10.1101/2021.10.07.463515       6  bioRxiv   \n",
       "31  10.1101/2021.10.09.21264771       5  medRxiv   \n",
       "32    10.1101/2021.10.06.463447       5  bioRxiv   \n",
       "33    10.1101/2021.10.09.462574       5  bioRxiv   \n",
       "34    10.1101/2021.10.06.463450       5  bioRxiv   \n",
       "35    10.1101/2021.10.07.463537       5  bioRxiv   \n",
       "36    10.1101/2021.10.07.463504       5  bioRxiv   \n",
       "37    10.1101/2021.10.05.463160       5  bioRxiv   \n",
       "38    10.1101/2021.10.06.463290       5  bioRxiv   \n",
       "39    10.1101/2021.10.06.463337       5  bioRxiv   \n",
       "40    10.1101/2021.10.05.463175       4  bioRxiv   \n",
       "41    10.1101/2021.10.07.463474       4  bioRxiv   \n",
       "42    10.1101/2021.10.05.463158       4  bioRxiv   \n",
       "43    10.1101/2021.10.07.463499       4  bioRxiv   \n",
       "44    10.1101/2021.10.08.462716       4  bioRxiv   \n",
       "45    10.1101/2021.10.08.463704       4  bioRxiv   \n",
       "46    10.1101/2021.10.08.463691       4  bioRxiv   \n",
       "47    10.1101/2021.10.07.463563       3  bioRxiv   \n",
       "48    10.1101/2021.10.07.463549       3  bioRxiv   \n",
       "49    10.1101/2021.10.07.463602       3  bioRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0                          Bioengineering  False   \n",
       "1   Infectious Diseases (except HIV/AIDS)   True   \n",
       "2   Infectious Diseases (except HIV/AIDS)   True   \n",
       "3                                Genetics  False   \n",
       "4               Obstetrics and Gynecology   True   \n",
       "5                            Neuroscience  False   \n",
       "6                       Molecular Biology  False   \n",
       "7                   Developmental Biology  False   \n",
       "8                            Microbiology  False   \n",
       "9                              Immunology  False   \n",
       "10                           Neuroscience  False   \n",
       "11                           Neuroscience  False   \n",
       "12                         Cancer Biology  False   \n",
       "13                      Molecular Biology  False   \n",
       "14                         Cancer Biology  False   \n",
       "15           Genetic and Genomic Medicine  False   \n",
       "16                           Neuroscience  False   \n",
       "17                           Microbiology  False   \n",
       "18                         Bioinformatics  False   \n",
       "19                         Bioengineering  False   \n",
       "20                           Microbiology  False   \n",
       "21                           Biochemistry  False   \n",
       "22                                Zoology  False   \n",
       "23                           Neuroscience  False   \n",
       "24                      Molecular Biology   True   \n",
       "25                           Neuroscience  False   \n",
       "26                      Synthetic Biology  False   \n",
       "27           Genetic and Genomic Medicine  False   \n",
       "28                   Evolutionary Biology  False   \n",
       "29                           Biochemistry   True   \n",
       "30                           Microbiology  False   \n",
       "31                 Allergy and Immunology   True   \n",
       "32                           Microbiology  False   \n",
       "33                         Bioinformatics  False   \n",
       "34                           Microbiology  False   \n",
       "35                           Biochemistry  False   \n",
       "36                           Microbiology  False   \n",
       "37                             Immunology  False   \n",
       "38                          Plant Biology  False   \n",
       "39                           Microbiology  False   \n",
       "40                           Cell Biology  False   \n",
       "41                   Evolutionary Biology  False   \n",
       "42                           Neuroscience  False   \n",
       "43                           Microbiology  False   \n",
       "44                          Plant Biology  False   \n",
       "45          Animal Behavior and Cognition  False   \n",
       "46                   Evolutionary Biology  False   \n",
       "47                          Plant Biology  False   \n",
       "48                           Microbiology  False   \n",
       "49                           Neuroscience  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Deep-Learning Super-Resolution Microscopy Reve...   \n",
       "1   Year-long COVID-19 infection reveals within-ho...   \n",
       "2   Analytical performance of eleven SARS-CoV-2 an...   \n",
       "3   Intercellular transport of RNA can limit herit...   \n",
       "4   Increase in preterm stillbirths and reduction ...   \n",
       "5   Molecular rhythm alterations in prefrontal cor...   \n",
       "6   High-throughput mutagenesis identifies mutatio...   \n",
       "7   Pseudo-dynamic analysis of heart tube formatio...   \n",
       "8   Large-scale discovery of candidate type VI sec...   \n",
       "9   Nanoengineered DNA origami with repurposed TOP...   \n",
       "10  Linking contrast sensitivity to cortical magni...   \n",
       "11  Minimal required exposures reveal the primacy ...   \n",
       "12  A sense-antisense RNA interaction promotes bre...   \n",
       "13  The DNAPKcs long-range C-NHEJ complex is requi...   \n",
       "14  Mistic: an open-source multiplexed image t-SNE...   \n",
       "15  Hi-C detects genomic structural variants in pe...   \n",
       "16  Light-induced asymmetries in the embryonic ret...   \n",
       "17  Multiple regulatory mechanisms control the pro...   \n",
       "18  Mutagenesis-visualization: analysis of site sa...   \n",
       "19  Engineered endosymbionts capable of directing ...   \n",
       "20  Polyvinyl Chloride Biodegradation Fuels Surviv...   \n",
       "21  Characterization of divalent cation interactio...   \n",
       "22  Tales of schizomid tails: patterns in schizomi...   \n",
       "23  Impact of stimulation location relative to gre...   \n",
       "24  Inhibition of SARS-CoV-2 spike protein palmito...   \n",
       "25  β-amyloid−driven synaptic depression requires ...   \n",
       "26  Development of a Freeze-Dried CRISPR-Cas12 Sen...   \n",
       "27  Inherited variants in CHD3 demonstrate variabl...   \n",
       "28  Inferring the mode and strength of ongoing sel...   \n",
       "29  High-throughput Activity Assay for Screening I...   \n",
       "30  Genomic and antigenic diversity of carried Kle...   \n",
       "31  Taste loss as a distinct symptom of COVID-19: ...   \n",
       "32  Genomic and phenotypic characterization of Pse...   \n",
       "33  Deep learning-based tumor microenvironment seg...   \n",
       "34  High throughput identification of genes confer...   \n",
       "35  The archaeal division protein CdvB1 assembles ...   \n",
       "36  PBP1 of Staphylococcus aureus has multiple ess...   \n",
       "37  The In Vivo Source of Type I and Type III IFNs...   \n",
       "38  In planta dynamics, transport-biases and endog...   \n",
       "39  Progesterone supplementation in mice leads to ...   \n",
       "40  DetecDiv, a deep-learning platform for automat...   \n",
       "41  Population genetic bottleneck has only margina...   \n",
       "42  A spiking neural network model of the Superior...   \n",
       "43  Genomic and metabolic adaptations of biofilms ...   \n",
       "44  Expression dynamics of ARGONAUTE proteins duri...   \n",
       "45  Wind and route choice affect performance of be...   \n",
       "46  Convergent evolution of polyploid genomes from...   \n",
       "47  LysM receptors in Coffea arabica: identificati...   \n",
       "48  Evolutionary sweeps of subviral parasites and ...   \n",
       "49  CCR5 closes the temporal window for memory lin...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Rong Chen'}, {'name': 'Xiao Tang'},...   \n",
       "1   [{'name': 'Veronique Nussenblatt'}, {'name': '...   \n",
       "2   [{'name': 'Meriem Bekliz'}, {'name': 'Kenneth ...   \n",
       "3   [{'name': 'Nathan Shugarts'}, {'name': 'Andrew...   \n",
       "4   [{'name': 'Lisa Hui'}, {'name': 'Melvin Barrie...   \n",
       "5   [{'name': 'Xiangning Xue'}, {'name': 'Wei Zong...   \n",
       "6   [{'name': 'Mariela Cortés-López'}, {'name': 'L...   \n",
       "7   [{'name': 'Isaac Esteban'}, {'name': 'Patrick ...   \n",
       "8   [{'name': 'Alexander Martin Geller'}, {'name':...   \n",
       "9   [{'name': 'Keying Zhu'}, {'name': 'Yang Wang'}...   \n",
       "10  [{'name': 'Marc M. Himmelberg'}, {'name': 'Jon...   \n",
       "11  [{'name': 'Renzo C. Lanfranco'}, {'name': 'And...   \n",
       "12  [{'name': 'Bruce Culbertson'}, {'name': 'Krist...   \n",
       "13  [{'name': 'Metztli Cisneros-Aguirre'}, {'name'...   \n",
       "14  [{'name': 'Sandhya Prabhakaran'}, {'name': 'Ch...   \n",
       "15  [{'name': 'Claire Mallard'}, {'name': 'Michael...   \n",
       "16  [{'name': 'Elisabetta Versace'}, {'name': 'Pao...   \n",
       "17  [{'name': 'Elizabeth M. Garrett'}, {'name': 'A...   \n",
       "18  [{'name': 'Frank Hidalgo'}, {'name': 'Sage Tem...   \n",
       "19  [{'name': 'Cody S. Madsen'}, {'name': 'Ashley ...   \n",
       "20  [{'name': 'Zhang Zhe'}, {'name': 'Haoran Peng'...   \n",
       "21  [{'name': 'Milena Timcenko'}, {'name': 'Anton ...   \n",
       "22  [{'name': 'Robert J. Kallal'}, {'name': 'Gusta...   \n",
       "23  [{'name': 'Angelique C Paulk'}, {'name': 'Rina...   \n",
       "24  [{'name': 'Ahmed A. Ramadan'}, {'name': 'Karth...   \n",
       "25  [{'name': 'Niels R. Reinders'}, {'name': 'Soph...   \n",
       "26  [{'name': 'Grant A Rybnicky'}, {'name': 'Radee...   \n",
       "27  [{'name': 'Jet van der Spek'}, {'name': 'Joery...   \n",
       "28  [{'name': 'Gustavo Valadares Barroso'}, {'name...   \n",
       "29  [{'name': 'Morgan Dasovich'}, {'name': 'Junlin...   \n",
       "30  [{'name': 'Joseph M Lewis'}, {'name': 'Madalit...   \n",
       "31  [{'name': 'Mackenzie R Hannum'}, {'name': 'Ril...   \n",
       "32  [{'name': 'Timothy L. Turner'}, {'name': 'Sumi...   \n",
       "33  [{'name': 'Łukasz Rączkowski'}, {'name': 'Iwon...   \n",
       "34  [{'name': 'Steven J. Hersch'}, {'name': 'Rehnu...   \n",
       "35  [{'name': 'Alberto Blanch Jover'}, {'name': 'N...   \n",
       "36  [{'name': 'Katarzyna Wacnik'}, {'name': 'Vince...   \n",
       "37  [{'name': 'Marvin J. Sandoval'}, {'name': 'Hsi...   \n",
       "38  [{'name': 'Christopher A. Brosnan'}, {'name': ...   \n",
       "39  [{'name': 'Meital Nuriel-Ohayon'}, {'name': 'A...   \n",
       "40  [{'name': 'Theo Aspert'}, {'name': 'Didier Hen...   \n",
       "41  [{'name': 'Karen Bisschop'}, {'name': 'Thomas ...   \n",
       "42  [{'name': 'Arezoo Alizadeh'}, {'name': 'A. Joh...   \n",
       "43  [{'name': 'Susheel Bhanu Busi'}, {'name': 'Mas...   \n",
       "44  [{'name': 'Cecilia Oliver'}, {'name': 'German ...   \n",
       "45  [{'name': 'Nicholas Burnett'}, {'name': 'Marc ...   \n",
       "46  [{'name': 'Yue Hao'}, {'name': 'Jonathon Flemi...   \n",
       "47  [{'name': 'Mariana de Lima Santos'}, {'name': ...   \n",
       "48  [{'name': 'Angus Angermeyer'}, {'name': 'Steph...   \n",
       "49  [{'name': 'Yang Shen'}, {'name': 'Miou Zhou'},...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>Single-molecule localization microscopy (SM...  2021-10-09  \n",
       "1   <sec><title>Background</title><p>B-cell deplet...  2021-10-05  \n",
       "2   <p>Global concerns arose as the emerged and ra...  2021-10-07  \n",
       "3   <p>RNAs in circulation carry sequence-specific...  2021-10-06  \n",
       "4   <sec><title>Objectives</title><p>The COVID-19 ...  2021-10-05  \n",
       "5   <p>Severe and persistent disruptions to sleep ...  2021-10-09  \n",
       "6   <p>During CART-19 immunotherapy for B-cell acu...  2021-10-08  \n",
       "7   <p>Understanding organ morphogenesis requires ...  2021-10-09  \n",
       "8   <p>Type VI secretion systems (T6SS) are common...  2021-10-07  \n",
       "9   <p>Targeting myeloid cells, especially microgl...  2021-10-04  \n",
       "10  <p>The size and organization of primary visual...  2021-10-05  \n",
       "11  <p>The human visual system is capable of detec...  2021-10-07  \n",
       "12  <p>Antisense RNAs are ubiquitous in human cell...  2021-10-09  \n",
       "13  <p>Canonical non-homologous end joining (C-NHE...  2021-10-07  \n",
       "14  <p>Understanding the complex ecology of a tumo...  2021-10-09  \n",
       "15  <p>B-cell acute lymphoblastic leukemia (B-ALL)...  2021-10-04  \n",
       "16  <p>Left-right asymmetries in the nervous syste...  2021-10-09  \n",
       "17  <p><italic>Clostridioides difficile</italic>, ...  2021-10-07  \n",
       "18  <p>Site saturation mutagenesis (SSM) experimen...  2021-10-09  \n",
       "19  <p>Modular methods for directing mammalian gen...  2021-10-06  \n",
       "20  <p>Microbial degradation of polyvinyl chloride...  2021-10-05  \n",
       "21  <p>Amphiphilic copolymers show great promise i...  2021-10-07  \n",
       "22  <p>The arachnid order Schizomida is a relative...  2021-10-05  \n",
       "23  <p>Background: Electrical neuromodulation is a...  2021-10-09  \n",
       "24  <p>Spike glycoproteins of almost all enveloped...  2021-10-08  \n",
       "25  <p>Soluble oligomeric amyloid-β (Aβ) is a prim...  2021-10-05  \n",
       "26  <p>Training the future synthetic biology workf...  2021-10-09  \n",
       "27  <p>Interpretation of next-generation sequencin...  2021-10-05  \n",
       "28  <p>Genome sequence data is no longer scarce. T...  2021-10-09  \n",
       "29  <p>Macrodomains are a class of conserved ADP-r...  2021-10-07  \n",
       "30  <p>Klebsiella pneumoniae is an antimicrobial r...  2021-10-07  \n",
       "31  <p>Chemosensory scientists have been skeptical...  2021-10-09  \n",
       "32  <p>A purchased lot of the antibiotic hygromyci...  2021-10-07  \n",
       "33  <p>Despite the fact that tumor microenvironmen...  2021-10-09  \n",
       "34  <p>The type six secretion system (T6SS) is a p...  2021-10-07  \n",
       "35  <p>The Cdv proteins constitute the cell-divisi...  2021-10-07  \n",
       "36  <p>Bacterial cell division is a complex proces...  2021-10-07  \n",
       "37  <p>Type I (-α, β) and type III (-λ) interferon...  2021-10-05  \n",
       "38  <p><list list-type=\"bullet\"><list-item><p>In R...  2021-10-06  \n",
       "39  <sec><title>Background</title><p>Progesterone ...  2021-10-06  \n",
       "40  <p>Automating the extraction of meaningful tem...  2021-10-05  \n",
       "41  <p>Evolution is a key process by which populat...  2021-10-09  \n",
       "42  <p>Previous studies have indicated that the lo...  2021-10-06  \n",
       "43  <p>Microorganisms dominate life in cryospheric...  2021-10-07  \n",
       "44  <p>Meiosis is a specialized cell division that...  2021-10-09  \n",
       "45  <p>Bees flying through natural landscapes enco...  2021-10-09  \n",
       "46  <p>By modeling the homoeologous gene losses th...  2021-10-10  \n",
       "47  <p>Pathogen-associated molecular patterns (PAM...  2021-10-07  \n",
       "48  <p><italic>Vibrio cholerae</italic> is a signi...  2021-10-07  \n",
       "49  <p>Real world memories are formed in a particu...  2021-10-09  "
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "58 preprints (including 6 covering SARS-CoV-2, 52 from bioRxiv and 6 from medRxiv) published in the last 7 days before 2021-10-11 had been tweeted at least 3 times (maximum 56)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
