{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15801 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7ab73589-f8d5-4bb6-b20f-5f76d850ecbb&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5a38f11e-44dd-4189-a176-02263ffad9d4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=ef9cef52-05a1-439e-bdfd-b8f9e4ba35c8&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9ab2a086-0e80-45a1-ba60-fe2d4bd5762c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=12a9aec5-1d99-4d45-9b55-1b65356f653b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=57e6163d-4df6-4b37-91d9-a1f310424b07&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=29e137f5-c9bf-4bd5-987e-3879a51c5f62&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=ec415ddb-c062-4d18-b1cc-b56765f04cbd&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=da9a0220-9124-4857-aaae-a327c83c4452&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5cf9f3d5-2bbe-45d1-bf2c-8b93b4539b96&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=af3576e9-0ec5-49b8-83df-df3d1a0e32d1&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=47689eaa-2ba0-4c6b-b2c5-2962b46fb60d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=1387854f-3487-4f92-826f-ea8f15b83547&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f3a5b040-b0d4-45be-97db-beac9df8a22c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=64cb26eb-267c-4dfa-8a83-88781a55304f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-10-25&until-occurred-date=2021-11-01\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-10-18.csv\n",
      "failed to load preprint_tweets_2021-10-25.csv\n",
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load preprint_tweets_2021-09-13.csv\n",
      "failed to load .DS_Store\n",
      "failed to load preprint_tweets_2021-10-23.csv\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-10-05.csv\n",
      "failed to load preprint_tweets_2021-10-11.csv\n",
      "failed to load preprint_tweets_2021-10-04.csv\n",
      "failed to load preprint_tweets_2021-10-06.csv\n",
      "failed to load preprint_tweets_2021-10-07.csv\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "failed to load preprint_tweets_2021-09-20.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263684...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21262182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263977...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20200329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.465614...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.21262701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.21255275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/530097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465374...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447489...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.09.20184143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.382143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.427303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445986...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264376...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.463909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265476...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.21264535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.420448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275569.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.08.372995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21263691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21263542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.21264827...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.273300.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.446360...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255889...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/240317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.28.923532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.12.439415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461790...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20222778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.02.462868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.437010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466170...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.459575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466024...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465934...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21258385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253770...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263853...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257668...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443555...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466306...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265094...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462538...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449893...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.314815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/702589...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.367649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.413609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465695...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466254...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.11.987958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.12.21250941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461419...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442625...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275503.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462599...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.12.149203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.13.20151233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.465610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.453981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.30.466508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462068...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.10.463851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.451159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.21264771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.21257899...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.21265427...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.11.421644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461778...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264181...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.071498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461389...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.21.885863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461108...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21264623...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.365932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459577...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461843...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465682...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.02.163816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466411...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462198...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.391672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.466009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.14.20062463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465842...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.21265096...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.09.08.459260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21263008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.451464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446137...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.460985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.371609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.462913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20213546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.12.20021386...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.463794...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.465226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461587...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466305...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.10.20021675...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.464848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466284...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461672...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459263...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461425...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459005...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466454...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264208...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/373332...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.21265424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.24.21253992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461830...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.413229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.11.451967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461764...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275989.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.464901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462202...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.464780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462730...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.17.435791...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.21263828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275224.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264014...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.03.20167395...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461774...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.17.20133504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440894...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461579...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.20.432090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461639...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255320...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20131243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21263187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461443...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.20156018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265422...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.435034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21262268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461220...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265127...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.21265508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.21258272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.30.424833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.21263636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.459215...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.465122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461801...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.396002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.465224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.07.21261741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21257910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.20142703...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.03.462938...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275308.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.465965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435416...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.466002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264117...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.465996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20100677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.21254765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466118...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.21263385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.466019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20087965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.20201848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.27.437353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.212233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.18.460924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.427139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.28.21262763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.14.21264762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21252621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.440722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.04.463138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.21262168...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.11.943159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.04.425114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457682...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465493...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.439940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454433...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461867...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462303...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465660...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461732...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.09.462574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/775080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.21259525...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.21263228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.05.895441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.21264863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.11.464002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21264255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265209...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.19.21263548...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.440017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461430...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.459925...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461359...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.09.443246...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459485...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.07.414060...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.14.444173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461851...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.465104...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461933...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461294...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.20204693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466246...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431811...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.427259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.17.427004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.459429...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461517...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.446289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.26.424423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.424451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.02.454825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.461531...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462203...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461865...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.31.466679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.465169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.206680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465513...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460111...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.28.437410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/238527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.21263598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461560...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.21263381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.448644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.462716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462234...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.21264082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.448770...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.05.463175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.21264368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449812...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462158...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.447951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.16.21263477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.446796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448063...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.13.460163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.08.463704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461723...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461293...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462436...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.02.20080036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.389031...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466172...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.339317...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.20086439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.18.344598...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20088260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.460915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461799...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.20240309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.21261298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21263911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.21265610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462061...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461875...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466382...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.456373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.15.20175570...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466339...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466346...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.235549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461400...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.225102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.24.212712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.260109...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.30.272237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.326637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461367...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.17.154104...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.457346...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456235...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428643...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265231...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/850248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21264515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.465103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462459...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.01.438087...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.02.20029124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.460630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.464584...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461550...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.17.20037689...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.439156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256539...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275868.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461910...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21263855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.461055...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268581.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.11.439311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.465023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.22.20040758...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.29.20045880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.25.432716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461160...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.24.21265348...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434607...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253207...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462004...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.463292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.460970...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21258418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.461806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259784...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457612...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462327...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.460453...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462291...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.21264356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21263766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461902...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461648...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459056...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462222...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.21264011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261955...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.462410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.459010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.30.462602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.29.21264312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458567...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461720...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.17.386995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.20229948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.399154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466056...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.20248876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.20248936...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.424067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.26.465931...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.385401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466161...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.27.466182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.09.373860...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.08.371179...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.27.20220905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.19.20214494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.342410...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.20212845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.461349...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.465406...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.06.20207498...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.21264135...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.425393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.21265407...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465589...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.21252931...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.23.465424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263944...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.465130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.22.464858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.21263457...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.465298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265480...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.21265494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.21.21251403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21251809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431706...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.25.465734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.21251133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.21.461312...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21256632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.22.21263515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.427602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.319814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/626507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/768663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461091...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047860...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.460629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.13.039917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.20202267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/592725...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461887...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21264009...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.04.20090506...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.12.091199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462002...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.13.990523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264139...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.17.952663...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/818450...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/856898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264174...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.268037.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.21264211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/135582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/276717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.116756...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.225854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20198796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.303966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.299271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.07.285692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.02.20186734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.31.274993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.26.461864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.25.266536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466332...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.461491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.28.466384...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.21265667...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.17.208199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.465954...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21258047...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466435...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462028...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.462026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.29.466520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.23.21263888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461408...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.21264242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.21.21265334...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.10.17.464398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.21257772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462140...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.17.21263532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257337...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.21257194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.27.461862...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.12.464125...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.13.21264693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461796...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.15.21265037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.10.16.21265095...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.16.464636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.20.460996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.441302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.17.21265101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.07.463455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.21259156...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462769...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.21259519...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462800...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.12.21263442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.449150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.01.462825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21256946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258797...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.14.460388...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.447176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.447094...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.06.463193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.24.461765...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.15.460240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.438521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.21255103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.462062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265279...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265245...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.439482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.438911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265221...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.10.18.21265147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21265239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265223...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.20.21264403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.28.461976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21265230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.25.461805...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.19.21264878...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.439221...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21264783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.10.18.21265057...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.10.26.465776', 'tweets': 202, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Artificial intelligence reveals nuclear pore complexity', 'authors': \"[{'name': 'Shyamal Mosalaganti'}, {'name': 'Agnieszka Obarska-Kosinska'}, {'name': 'Marc Siggel'}, {'name': 'Beata Turonova'}, {'name': 'Christian E. Zimmerli'}, {'name': 'Katarzyna Buczak'}, {'name': 'Florian H. Schmidt'}, {'name': 'Erica Margiotta'}, {'name': 'Marie-Therese Mackmull'}, {'name': 'Wim Hagen'}, {'name': 'Gerhard Hummer'}, {'name': 'Martin Beck'}, {'name': 'Jan Kosinski'}]\", 'abstract': '<p>Nuclear pore complexes (NPCs) mediate nucleocytoplasmic transport. Their intricate 120 MDa architecture remains incompletely understood. Here, we report a near-complete structural model of the human NPC scaffold with explicit membrane and in multiple conformational states. We combined AI-based structure prediction with <italic>in situ</italic> and <italic>in cellulo</italic> cryo-electron tomography and integrative modeling. We show that linker Nups spatially organize the scaffold within and across subcomplexes to establish the higher-order structure. Microsecond-long molecular dynamics simulations suggest that the scaffold is not required to stabilize the inner and outer nuclear membrane fusion, but rather widens the central pore. Our work exemplifies how AI-based modeling can be integrated with <italic>in situ</italic> structural biology to understand subcellular architecture across spatial organization levels.</p><sec><title>One sentence summary</title><p>An AI-based, dynamic model of the human nuclear pore complex reveals how the protein scaffold and the nuclear envelope are coupled inside cells.</p></sec>', 'posted': '2021-10-27'}, {'doi': '10.1101/2021.10.24.465614', 'tweets': 100, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Tension-driven multi-scale self-organisation in human iPSC-derived muscle fibers', 'authors': \"[{'name': 'Qiyan Mao'}, {'name': 'Achyuth Acharya'}, {'name': 'Alejandra Rodrguez-delaRosa'}, {'name': 'Fabio Marchiano'}, {'name': 'Benoit Dehapoit'}, {'name': 'Ziad Al Tanoury'}, {'name': 'Jyoti Rao'}, {'name': 'Margarete Daz-Cuadros'}, {'name': 'Arian Mansur'}, {'name': 'Erica Wagner'}, {'name': 'Claire Chardes'}, {'name': 'Vandana A. Gupta'}, {'name': 'Pierre-Franois Lenne'}, {'name': 'Bianca H. Habermann'}, {'name': 'Olivier Pourquie'}, {'name': 'Frank Schnorrer'}]\", 'abstract': '<p>Human muscle is a hierarchically organised tissue with its contractile cells called myofibers packed into large myofiber bundles. Each myofiber contains periodic myofibrils built by hundreds of contractile sarcomeres that generate large mechanical forces. To better understand the mechanisms that coordinate human muscle morphogenesis from tissue to molecular scales, we adopted a simple <italic>in vitro</italic> system using induced pluripotent stem cell-derived human myogenic precursors. When grown on an unrestricted two-dimensional substrate, developing myofibers spontaneously align and self-organise into higher-order myofiber bundles, which grow and consolidate to stable sizes. Following a transcriptional boost of sarcomeric components, myofibrils assemble into chains of periodic sarcomeres that emerge across the entire myofiber. By directly probing tension we found that tension build-up precedes sarcomere assembly and increases within each assembling myofibril. Furthermore, we found that myofiber ends stably attach to other myofibers using integrin-based attachments and thus myofiber bundling coincides with stable myofiber bundle attachment <italic>in vitro</italic>. A failure in stable myofiber attachment results in a collapse followed by a disassembly of the myofibrils. Overall, our results strongly suggest that mechanical tension across sarcomeric components as well as between differentiating myofibers is key to coordinate the multi- scale self-organisation of muscle morphogenesis.</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.21.21265342', 'tweets': 78, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Linking common and rare disease genetics through gene regulatory networks', 'authors': \"[{'name': 'Olivier B. Bakker'}, {'name': 'Annique Claringbould'}, {'name': 'Harm-Jan Westra'}, {'name': 'Henry Wiersma'}, {'name': 'Floranne Boulogne'}, {'name': 'Urmo Vsa'}, {'name': 'Sophie Mulcahy Symmons'}, {'name': 'Iris H. Jonkers'}, {'name': 'Lude Franke'}, {'name': 'Patrick Deelen'}]\", 'abstract': '<p>Genetic variants identified through genome-wide association studies (GWAS) are typically non-coding and exert small regulatory effects on downstream genes, but which downstream genes are ultimately impacted and how they confer risk remains mostly unclear. Conversely, variants that cause rare Mendelian diseases are often coding and have a more direct impact on disease development. We demonstrate that common and rare genetic diseases can be linked by studying the gene regulatory networks impacted by common disease-associated variants. We implemented this in the Downstreamer method and applied it to 44 GWAS traits and find that predicted downstream key genes are enriched with Mendelian disease genes, e.g. key genes for height are enriched for genes that cause skeletal abnormalities and Ehlers-Danlos syndromes. We find that 82% of these key genes are located outside of GWAS loci, suggesting that they result from complex <italic>trans</italic> regulation rather than being impacted by disease-associated variants in <italic>cis</italic>. Finally, we discuss the challenges in reconstructing gene regulatory networks and provide a roadmap to improve identification of these highly connected genes for common traits and diseases.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.29.466401', 'tweets': 58, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'SARS-CoV-2 spike protein as a bacterial lipopolysaccharide delivery system in an overzealous inflammatory cascade', 'authors': \"[{'name': 'Firdaus Samsudin'}, {'name': 'Palur Raghuvamsi'}, {'name': 'Ganna Petruk'}, {'name': 'Manoj Puthia'}, {'name': 'Jitka Petrlova'}, {'name': 'Paul MacAry'}, {'name': 'Ganesh S Anand'}, {'name': 'Artur Schmidtchen'}, {'name': 'Peter J Bond'}]\", 'abstract': '<p>Accumulating evidence indicates a potential role for bacterial lipopolysaccharide (LPS) in the overactivation of the immune response during SARS-CoV-2 infection. LPS is recognised by Toll-like receptor 4 (TLR4) in innate immunity. Here, we showed that LPS binds to multiple hydrophobic pockets spanning both the S1 and S2 subunits of the SARS-CoV-2 spike (S) protein. LPS binds to the S2 pocket with a lower affinity compared to S1, suggesting its possible role as an intermediate in the TLR4 cascade. Congruently, nuclear factor-kappa B (NF-B) activation in vitro is strongly boosted by S2. In vivo, however, a boosting effect is observed for both S1 and S2, with the former potentially facilitated by proteolysis. Collectively, our study suggests the S protein may act as a delivery system for LPS in host innate immune pathways. The LPS binding pockets are highly conserved across different SARS-CoV-2 variants and therefore represent potential therapeutic targets.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.21265525', 'tweets': 58, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'COVCOG 1: Factors predicting Cognitive Symptoms in Long COVID. A First Publication from the COVID and Cognition Study', 'authors': \"[{'name': 'Panyuan Guo'}, {'name': 'Alvaro Benito Ballesteros'}, {'name': 'Sabine P Yeung'}, {'name': 'Ruby Liu'}, {'name': 'Arka Saha'}, {'name': 'Lyn Curtis'}, {'name': 'Muzaffer Kaser'}, {'name': 'Mark P Haggard'}, {'name': 'Lucy G Cheke'}]\", 'abstract': '<p>Since its first emergence in December 2019, coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has evolved into a global pandemic. Whilst often considered a respiratory disease, a large proportion of COVID-19 patients report neurological symptoms, and there is accumulating evidence for neural damage in some individuals, with recent studies suggesting loss of gray matter in multiple regions, particularly in the left hemisphere. There are a number of mechanisms by which COVID-19 infection may lead to neurological symptoms and structural and functional changes in the brain, and it is reasonable to expect that many of these may translate into cognitive problems. Indeed, cognitive problems are one of the most commonly reported symptoms in those suffering from Long COVIDthe chronic illness following COVID-19 infection that affects between 1025% of sufferers. The COVID and Cognition Study is a part cross-sectional, part longitudinal, study documenting and aiming to understand the cognitive problems in Long COVID. In this first paper from the study, we document the characteristics of our sample of 181 individuals who had suffered COVID-19 infection, and 185 who had not. We explore which factors may be predictive of ongoing symptoms and their severity, as well as conducting an in-depth analysis of symptom profiles. Finally, we explore which factors predict the presence and severity of cognitive symptoms, both throughout the ongoing illness and at the time of testing. The main finding from this first analysis is that that severity of initial illness is a significant predictor of the presence and severity of ongoing symptoms, and that some symptoms during the acute illnessparticularly limb weaknessmay be more common in those that have more severe ongoing symptoms. Symptom profiles can be well described in terms of 5 or 6 factors, reflecting the variety of this highly heterogenous condition suffered by the individual. Specifically, we found that neurological and fatigue symptoms during the initial illness, and that neurological, gastro-intestinal, and cardiopulmonary symptoms during the ongoing illness, predicted experience of cognitive symptoms.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466055', 'tweets': 54, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'Phylogenetic analysis and in silico studies link spike Q675H mutation to SARS-CoV-2 adaptive evolution', 'authors': '[{\\'name\\': \\'Anna Bertelli\\'}, {\\'name\\': \"Pasqualina D\\'Ursi\"}, {\\'name\\': \\'Giovanni Campisi\\'}, {\\'name\\': \\'Serena Messali\\'}, {\\'name\\': \\'Maria Milanesi\\'}, {\\'name\\': \\'Marta Giovanetti\\'}, {\\'name\\': \\'Massimo Ciccozzi\\'}, {\\'name\\': \\'Francesca Caccuri\\'}, {\\'name\\': \\'Arnaldo Caruso\\'}]', 'abstract': '<p>Genotype screening was implemented in Italy and showed a significant prevalence of new SARS-CoV-2 mutants carrying Q675H mutation, near the furin cleavage site of spike protein. Currently, this mutation, which is expressed on different SARS-CoV-2 lineages circulating worldwide, has not been thoughtfully investigated. Therefore, we performed phylogenetic and biocomputational analysis to better understand SARS-CoV-2 Q675H mutants evolutionary relationships with other circulating lineages and Q675H function in its molecular context. Our studies reveal that Q675H spike mutation is the result of parallel evolution because it arose independently in separate evolutionary clades. In silico data show that the Q675H mutation gives rise to a hydrogen-bonds network in the spike polar region delimiting the conformational space of the highly flexible loop containing the furin cleavage site. This results in an optimized directionality of arginine residues involved in interaction of spike with the furin binding pocket, thus improving proteolytic exposure of the viral protein. Furin was found to have a greater affinity for Q675H than Q675 substrate conformations. As a consequence, Q675H mutation is likely to confer a fitness advantage to SARS-CoV-2 by promoting a more efficient viral entry. Interestingly, here we show an ongoing increase in the occurrence of Q675H spike mutation in the most common SARS-CoV-2 variants of concern (VOC). This finding highlights that, VOC are still evolving and start acquiring the Q675H mutation. At the same time, it suggests that our hypothesis of fitness advantage prompted by Q675H could be concrete.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.21265511', 'tweets': 53, 'archive': 'medRxiv', 'subject-area': 'Oncology', 'covid': True, 'title': 'Blood cytokine analysis suggests that SARS-CoV-2 infection results in a sustained tumour promoting environment in cancer patients', 'authors': '[{\\'name\\': \\'Fien HR De Winter\\'}, {\\'name\\': \\'An Hotterbeekx\\'}, {\\'name\\': \\'Manon Huizing\\'}, {\\'name\\': \\'Angelina Konnova\\'}, {\\'name\\': \\'Erik Fransen\\'}, {\\'name\\': \"Bart \\'s Jongers\"}, {\\'name\\': \\'Ravi Kumar Jairam\\'}, {\\'name\\': \\'Vincent Van averbeke\\'}, {\\'name\\': \\'Pieter Moons\\'}, {\\'name\\': \\'Ella Roelant\\'}, {\\'name\\': \\'Debbie Le Blon\\'}, {\\'name\\': \\'Wim Vanden Berghe\\'}, {\\'name\\': \\'Annelies Janssens\\'}, {\\'name\\': \\'Willem Lybaert\\'}, {\\'name\\': \\'Lieselot Croes\\'}, {\\'name\\': \\'Christof Vulsteke\\'}, {\\'name\\': \\'Surbhi Malhotra-Kumar\\'}, {\\'name\\': \\'Herman Goossens\\'}, {\\'name\\': \\'Zwi Berneman\\'}, {\\'name\\': \\'Marc Peeters\\'}, {\\'name\\': \\'Peter van Dam\\'}, {\\'name\\': \\'Samir Kumar-Singh\\'}]', 'abstract': '<p>Cytokines, chemokines and (angiogenic) growth factors (CCGs) have been shown to play an intricate role in the progression of both solid and haematological malignancies. Recent studies have shown that SARS-CoV-2 infection leads to worse outcome in cancer patients, especially in haematological malignancy patients. Here, we investigated how SARS-CoV-2 infection impacts the already altered CCG levels in solid or haematological malignancies, specifically whether there is a protective effect or rather a potentially higher risk for major COVID-19 complications in cancer patients due to elevated CCGs linked to cancer progression. Serially analysing immune responses with 55 CCGs in cancer patients under active treatment with or without SARS-CoV-2 infection, we first showed that cancer patients without SARS-CoV-2 infection (n=54) demonstrate elevated levels of 35 CCGs compared to the non-cancer, non-infected control group of health care workers (n=42). Of the 35 CCGs, 19 were common to both solid and haematological malignancy groups and comprised previously described cytokines such as IL-6, TNF-, IL-1Ra, IL-17A, and VEGF, but also several less well described cytokines/chemokines such as Fractalkine, Tie-2, and T cell chemokine CTACK. Importantly, we show here that 7 CCGs are significantly altered in SARS-CoV-2 exposed cancer patients (n=52). Of these TNF-, IFN-, TSLP and sVCAM-1, identified to be elevated in haematological cancers, are also known tumour-promoting factors. Longitudinal analysis conducted over 3 months showed persistence of several tumour-promoting CCGs in SARS-CoV-2 exposed cancer patients. These data urge for increased vigilance for haematological malignancy patients as a part of long COVID follow-up.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.465200', 'tweets': 52, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Cell-extrinsic autophagy in mature adipocytes regulates anti-inflammatory response to intestinal tissue injury through lipid mobilization', 'authors': \"[{'name': 'Felix Clemens Richter'}, {'name': 'Matthias Friedrich'}, {'name': 'Mathilde Pohin'}, {'name': 'Ghada Alsaleh'}, {'name': 'Irina Guschina'}, {'name': 'Sarah Karin Wideman'}, {'name': 'Errin Johnson'}, {'name': 'Mariana Borsa'}, {'name': 'Klara Piletic'}, {'name': 'Paula Hahn'}, {'name': 'Henk Simon Schipper'}, {'name': 'Claire M. Edwards'}, {'name': 'Fiona Powrie'}, {'name': 'Anna Katharina Simon'}]\", 'abstract': '<p>Autophagy is a critical cellular recycling pathway which is genetically linked to the development of intestinal inflammation in humans. Inflammation drives adipose tissue breakdown and provision of major nutrients such as free fatty acids (FFA). However, the effect of autophagy-mediated FFA release by adipocytes in immune-mediated inflammatory diseases remains unexplored.</p><p>In a mouse model of intestinal inflammation, we found that visceral adipocytes upregulate autophagy at peak inflammation. Adipocyte-specific loss of the key autophagy gene <italic>Atg7</italic> (<italic>Atg7<sup>Ad</sup></italic>) resulted in the exacerbation of intestinal inflammation. TNF-induced lipolysis was impaired in <italic>Atg7</italic>-deficient adipocytes leading to the reduced availability of several FFA species, and decreased expression of the FFA transporter CD36 on adipose tissue macrophages (ATMs). Visceral adipose tissues from <italic>Atg7<sup>Ad</sup></italic> mice released less IL-10 resulting in lower levels of circulating IL-10 in colitis. ATMs present the main source of adipose tissue-derived IL-10 during colitis. <italic>In vitro</italic> assays confirmed that FFA restriction from macrophages reduced CD36 expression and diminished IL-10 production.</p><p>Taken together, our study demonstrates that autophagy-mediated FFA release from adipocytes directs anti-inflammatory responses in ATMs, which in turn conveys protective effects for distant intestinal inflammation.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466091', 'tweets': 43, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Cryo-electron microscopy structure of the H3-H4 octasome without histones H2A and H2B', 'authors': \"[{'name': 'Kayo Nozawa'}, {'name': 'Yoshimasa Takizawa'}, {'name': 'Leonidas Pierrakeas'}, {'name': 'Kazumi Saikusa'}, {'name': 'Satoko Akashi'}, {'name': 'Ed Luk'}, {'name': 'Hitoshi Kurumizaka'}]\", 'abstract': '<p>The canonical nucleosome, which represents the predominant packaging unit in eukaryotic chromatin, has an octameric core made up of two histone H2A-H2B and H3-H4 dimers with ~147 base-pair (bp) DNA wrapping around it. Non-nucleosome particles with alterative histone stoichiometries and DNA wrapping configurations have been found, and they could profoundly influence genome architecture and function. Here we solved the structure of the H3-H4 octasome, which is a nucleosome-like particle with a core made up of four H3-H4 dimers. Two conformations, open and closed, are determined at 3.9  and 3.6  resolutions by cryo-electron microscopy, respectively. The H3-H4 octasome, made up of a di-tetrameric core, is wrapped by ~120 bp DNA in 1.5 negative superhelical turns. The symmetrical halves are connected by a unique H4-H4 interface along the dyad axis. <italic>In vivo</italic> crosslinking of cysteine probes placed at another unique H3-H3 interface demonstrated the existence of the H3-H4 octasome in cells.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.29.466387', 'tweets': 43, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Discovery of a conserved rule behind the assembly of -barrel membrane proteins.', 'authors': \"[{'name': 'Edward M Germany'}, {'name': 'Yue Ding'}, {'name': 'Kenichiro Imai'}, {'name': 'Rebecca S Bamert'}, {'name': 'Rhys A Dunstan'}, {'name': 'Yukari Nakajima'}, {'name': 'XiangFeng Lai'}, {'name': 'Chaille T Webb'}, {'name': 'Christopher J Stubenrauch'}, {'name': 'Kentaro Hidaka'}, {'name': 'Nakajohn Thewasano'}, {'name': 'Kher Shing Tan'}, {'name': 'Hsin-Hui Shen'}, {'name': 'Trevor Lithgow'}, {'name': 'Takuya Shiota'}]\", 'abstract': '<p>Gram-negative bacteria, mitochondria and chloroplasts contain outer membrane proteins (OMPs) characterized by a transmembrane domain with a  barrel structure. Most OMPs have a diagnostic -signal imprinted in the amino acid sequence of the final -strand (-1 strand) of the  barrel. Molecular chaperones of the Omp85 superfamily, such as BamA in the bacterial BAM complex, recognizes the -signal and then catalyze the folding of the OMP into its membrane-embedded  barrel structure. Here, we reconstituted OMP assembly in vitro to study this process with five model OMPs, revealing that a critical assembly signal exists in the fifth -strand from the C-terminus (-5 strand). This signal was shown to drive a critical insertion step of the OMP assembly process but is not required for the initial recognition step between the OMP and BamA. Furthermore, we identified the receptor for this -5 signal as BamD, a second essential subunit of the BAM complex. Distinct binding sites for the -signal and the -5 signal, were identified on BamD. Comparative sequence analysis showed that the -5 signal is a conserved feature in bacterial OMPs, and that mitochondrial OMPs also contains -5 signal motif positioned in the fifth from last strand of their  barrel structure. We propose the -5 rule as a conserved feature of the process of OMP assembly in bacteria and the organelles of eukaryotes.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465865', 'tweets': 41, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Excessive inflammatory and metabolic responses to acute SARS-CoV-2 infection are associated with a distinct gut microbiota composition', 'authors': \"[{'name': 'Werner C. Albrich'}, {'name': 'Tarini Shankar Ghosh'}, {'name': 'Sinead Ahearn-Ford'}, {'name': 'Flora Mikaeloff'}, {'name': 'Nonhlanhla Lunjani'}, {'name': 'Brian Forde'}, {'name': 'Nomie Suh'}, {'name': 'Gian-Reto Kleger'}, {'name': 'Urs Pietsch'}, {'name': 'Manuel Frischknecht'}, {'name': 'Christian Garzoni'}, {'name': 'Rossella Forlenza'}, {'name': 'Mary Horgan'}, {'name': 'Corinna Sadlier'}, {'name': 'Tommaso Rochat Negro'}, {'name': 'Jrme Pugin'}, {'name': 'Hannah Wozniak'}, {'name': 'Andreas Cerny'}, {'name': 'Ujjwal Neogi'}, {'name': 'Paul W. OToole'}, {'name': 'Liam OMahony'}]\", 'abstract': '<p>Protection against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection and associated clinical sequelae requires well-coordinated metabolic and immune responses that limit viral spread and promote recovery of damaged systems. In order to understand potential mechanisms and interactions that influence coronavirus disease 2019 (COVID-19) outcomes, we performed a multi-omics analysis on hospitalised COVID-19 patients and compared those with the most severe outcome (i.e. death) to those with severe non-fatal disease, or mild/moderate disease, that recovered. A distinct subset of 8 cytokines and 140 metabolites in sera identified those with a fatal outcome to infection. In addition, elevated levels of multiple pathobionts and lower levels of protective or anti-inflammatory microbes were observed in the faecal microbiome of those with the poorest clinical outcomes. Weighted gene correlation network analysis (WGCNA) identified modules that associated severity-associated cytokines with tryptophan metabolism, coagulation-linked fibrinopeptides, and bile acids with multiple pathobionts. In contrast, less severe clinical outcomes associated with clusters of anti-inflammatory microbes such as <italic>Bifidobacterium</italic> or <italic>Ruminococcus</italic>, short chain fatty acids (SCFAs) and IL-17A. Our study uncovered distinct mechanistic modules that link host and microbiome processes with fatal outcomes to SARS-CoV-2 infection. These features may be useful to identify at risk individuals, but also highlight a role for the microbiome in modifying hyperinflammatory responses to SARS-CoV-2 and other infectious agents.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466163', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Prime-boost vaccination regimens with INO-4800 and INO-4802 augment and broaden immune responses against SARS-CoV-2 in nonhuman primates', 'authors': \"[{'name': 'Jewell N Walters'}, {'name': 'Blake Schouest'}, {'name': 'Ami Patel'}, {'name': 'Emma L Reuschel'}, {'name': 'Katherine Schultheis'}, {'name': 'Elizabeth Parzych'}, {'name': 'Igor Maricic'}, {'name': 'Ebony N Gary'}, {'name': 'Mansi Purwar'}, {'name': 'Viviane M Andrade'}, {'name': 'Arthur Doan'}, {'name': 'Dustin Elwood'}, {'name': 'Zeena Eblimit'}, {'name': 'Brian Nguyen'}, {'name': 'Drew Frase'}, {'name': 'Faraz I Zaidi'}, {'name': 'Abhijeet Kulkarni'}, {'name': 'Alison Generotti'}, {'name': 'J Joseph Kim'}, {'name': 'Laurent M Humeau'}, {'name': 'Stephanie J Ramos'}, {'name': 'Trevor R.F. Smith'}, {'name': 'David B Weiner'}, {'name': 'Kate E Broderick'}]\", 'abstract': '<p>The enhanced transmissibility and immune evasion associated with emerging SARS-CoV-2 variants demands the development of next-generation vaccines capable of inducing superior protection amid a shifting pandemic landscape. Since a portion of the global population harbors some level of immunity from vaccines based on the original Wuhan-Hu-1 SARS-CoV-2 sequence or natural infection, an important question going forward is whether this immunity can be boosted by next-generation vaccines that target emerging variants while simultaneously maintaining long-term protection against existing strains. Here, we evaluated the immunogenicity of INO-4800, our synthetic DNA vaccine candidate for COVID-19 currently in clinical evaluation, and INO-4802, a next-generation DNA vaccine designed to broadly target emerging SARS-CoV-2 variants, as booster vaccines in nonhuman primates. Rhesus macaques primed over one year prior with the first-generation INO-4800 vaccine were boosted with either INO-4800 or INO-4802 in homologous or heterologous prime-boost regimens. Both boosting schedules led to an expansion of antibody responses which were characterized by improved neutralizing and ACE2 blocking activity across wild-type SARS-CoV-2 as well as multiple variants of concern. These data illustrate the durability of immunity following vaccination with INO-4800 and additionally support the use of either INO-4800 or INO-4802 in prime-boost regimens.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.21265476', 'tweets': 37, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'A Made-in-Canada serology solution for profiling humoral immune responses to SARS-CoV-2 infection and vaccination', 'authors': \"[{'name': 'Karen Colwill'}, {'name': 'Yannick Galipeau'}, {'name': 'Matthew Stuible'}, {'name': 'Christian Gervais'}, {'name': 'Corey Arnold'}, {'name': 'Bhavisha Rathod'}, {'name': 'Kento T. Abe'}, {'name': 'Jenny H. Wang'}, {'name': 'Adrian Pasculescu'}, {'name': 'Mariam Maltseva'}, {'name': 'Lynda Rocheleau'}, {'name': 'Martin Pelchat'}, {'name': 'Mahya Fazel-Zarandi'}, {'name': 'Mariam Iskilova'}, {'name': 'Miriam Barrios-Rodiles'}, {'name': 'Linda Bennett'}, {'name': 'Kevin Yau'}, {'name': 'Franois Cholette'}, {'name': 'Christine Mesa'}, {'name': 'Angel X. Li'}, {'name': 'Aimee Paterson'}, {'name': 'Michelle A. Hladunewich'}, {'name': 'Pamela J. Goodwin'}, {'name': 'Jeffrey L. Wrana'}, {'name': 'Steven J. Drews'}, {'name': 'Samira Mubareka'}, {'name': 'Allison J. McGeer'}, {'name': 'John Kim'}, {'name': 'Marc-Andr Langlois'}, {'name': 'Anne-Claude Gingras'}, {'name': 'Yves Durocher'}]\", 'abstract': '<sec><title>BACKGROUND</title><p>Testing for antibodies against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been instrumental in detecting previous exposures and analyzing vaccine-elicited immune responses. Here, we describe a scalable Made-in-Canada solution that can detect and quantify SARS-CoV-2 antibodies, discriminate between natural infection- and vaccination-induced responses, and assess antibody-mediated inhibition of the spike-angiotensin converting enzyme 2 (ACE2) interaction.</p></sec><sec><title>METHODS</title><p>We developed a set of methods and reagents to detect SARS-CoV-2 antibodies by enzyme-linked immunosorbent assay (ELISA). The main assays focus on the parallel detection of immunoglobulin (Ig)Gs against the spike trimer, its receptor binding domain (RBD), and the nucleocapsid (N) protein. These antigens are complemented by a detection antibody (human anti-IgG fused to horseradish peroxidase (HRP)) and a positive control reference antibody (recombinant IgG against the RBD), permitting intra- and inter-laboratory comparisons. Using this toolkit and commercial reagents, we optimized automated ELISAs on two different high throughput platforms to measure antibody responses to SARS-CoV-2 antigens. The assays were calibrated to a reference standard from the World Health Organization. We also automated a surrogate neutralization (sn)ELISA that measures inhibition of ACE2-Spike or -RBD interactions by antibodies using biotinylated ACE2.</p></sec><sec><title>RESULTS</title><p>Our individual IgG-based ELISAs measure antibody levels in single-point measurements in reference to a standard antibody curve to accurately distinguish non-infected and infected individuals (area under the curve &gt; 0.96 for each assay). Positivity thresholds can be established in individual assays using precision-recall analysis (e.g., by fixing the false positive rate), or more stringently, by scoring against the distribution of the means of negative samples across multiple assays performed over several months. For seroprevalence assessment (in a non-vaccinated cohort), classifying a sample as positive if antibodies were detected for at least 2 of the 3 antigens provided the highest specificity. In vaccinated cohorts, increases in anti-spike and -RBD (but not -N) antibodies are observed. Here, we present detailed protocols to perform these assays using either serum/plasma or dried blood spots both manually and on two automated platforms, and to express the results in international units to facilitate data harmonization and inter-study comparisons. We also demonstrate that the snELISA can be performed automatically at single points, increasing the scalability of this functional assay for large seroprevalence studies.</p></sec><sec><title>INTERPRETATION</title><p>The ability to measure antibodies to three viral antigens and identify neutralizing antibodies capable of disrupting spike-ACE2 interactions in high-throughput assays enables large-scale analyses of humoral immune responses to SARS-CoV-2 infection and vaccination. The Made-in-Canada set of protein reagents, produced at the National Research Council of Canada are publicly available to enable the up-scaling of standardized serological assays, permitting nationwide data comparison and aggregation.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.465989', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Evolutionary instability of collateral susceptibility networks in ciprofloxacin resistant clinical Escherichia coli strains', 'authors': \"[{'name': 'Vidar Srum'}, {'name': 'Emma L. ynes'}, {'name': 'Anna S. Mller'}, {'name': 'Klaus Harms'}, {'name': 'rjan Samuelsen'}, {'name': 'Nicole L. Podnecky'}, {'name': 'Pl J. Johnsen'}]\", 'abstract': '<p>Collateral sensitivity and resistance occur when resistance development towards one antimicrobial either potentiates or deteriorates the effect of others, respectively. Previous reports on collateral effects on susceptibility focus on newly acquired resistance determinants and propose that novel treatment guidelines informed by collateral networks may reduce the evolution, selection and spread of antimicrobial resistance. In this study, we investigate the evolutionary stability of collateral networks in five ciprofloxacin resistant, clinical <italic>Escherichia coli</italic> strains. After 300 generations of experimental evolution without antimicrobials, we show complete fitness restoration in four of five genetic backgrounds and demonstrate evolutionary instability in collateral networks of newly acquired resistance determinants. We show that compensatory mutations reducing efflux expression is the main driver destabilizing initial collateral networks and identify <italic>rpoS</italic> as a putative target for compensatory evolution. Our results add another layer of complexity to future predictions and clinical application of collateral networks.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.25.465810', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Transmission of Klebsiella strains and plasmids within and between Grey-headed flying fox colonies', 'authors': \"[{'name': 'Ben Vezina'}, {'name': 'Louise M. Judd'}, {'name': 'Fiona K. McDougall'}, {'name': 'Wayne S.J. Boardman'}, {'name': 'Michelle L. Power'}, {'name': 'Jane Hawkey'}, {'name': 'Sylvain Brisse'}, {'name': 'Jonathan M. Monk'}, {'name': 'Kathryn E. Holt'}, {'name': 'Kelly L. Wyres'}]\", 'abstract': '<p>The Grey-headed flying fox (<italic>Pteropus poliocephalus</italic>) is an endemic Australian fruit bat, known to carry pathogens with zoonotic potential. We recently showed these bats harbour the bacterial pathogens <italic>Klebsiella pneumoniae</italic> and closely related species in the <italic>K. pneumoniae</italic> species complex (<italic>Kp</italic>SC). However, the dynamics of <italic>Klebsiella</italic> transmission and gene flow within flying fox colonies were not explored and remain poorly understood.</p><p>Here we report a high-resolution genomic comparison of 39 <italic>Kp</italic>SC isolates from Greyheaded flying foxes. Illumina whole genome sequences (n=39) were assembled <italic>de novo</italic> and the Kleborate genotyping tool was used to infer sequence types (STs). Oxford Nanopore sequences were generated for 13 isolates (one for each distinct ST) in order to generate high-quality completed reference genomes. Read mapping and variant calling was used to identify single nucleotide variants (SNVs) within each ST, using the relevant reference genome. <italic>In silico</italic> genome-scale metabolic models were generated to predict and compare substrate usage to 59 previously published <italic>Kp</italic>SC models for isolates from human and environmental sources, which indicated no distinction on the basis of metabolic capabilities.</p><p>High-resolution genome comparisons identified five putative strain transmission clusters (four intra- and one inter-colony, n=2-15 isolates each, 25 pairwise SNVs). Inter-colony transmission of <italic>Klebsiella africana</italic> was found between two flying fox populations located within flying distance. The 13 completed genomes harboured 11 plasmids, all of which showed 37-98% coverage (mean 73%) and 95% identity to those previously reported from human-associated <italic>Kp</italic>SC. Comparison of plasmids from different flying fox associated <italic>Kp</italic>SC indicated an interspecies horizontal plasmid transmission between <italic>K. pneumoniae</italic> and <italic>K. africana</italic> for a 98 kbp plasmid, pFF1003.</p><p>These data indicate that <italic>Kp</italic>SC are able to transmit directly via flying fox populations or indirectly via a common source, and that these isolates can harbour plasmids with similarity to those found in human derived <italic>Kp</italic>SC, indicating gene flow is occurring between isolates from Grey-headed flying fox <italic>Kp</italic>SC and human clinical isolates.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.29.466484', 'tweets': 34, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Clamping of DNA shuts the condensin neck gate', 'authors': \"[{'name': 'Byung-Gil Lee'}, {'name': 'James Rhodes'}, {'name': 'Jan Lwe'}]\", 'abstract': '<p>Condensin is a Structural Maintenance of Chromosomes (SMC) complex needed for the compaction of DNA into chromatids during mitosis. Lengthwise DNA compaction by condensin is facilitated by ATPase-driven loop extrusion, a process that is believed to be the fundamental activity of most, if not all SMC complexes. In order to obtain molecular insights, we obtained cryo-EM structures of yeast condensin in the presence of a slowly-hydrolysable ATP analogue and linear, as well as circular DNAs. The DNAs were shown to be clamped between the engaged heterodimeric SMC ATPase heads and the Ycs4 subunit, in a manner similar to previously reported DNA-bound SMC complex structures. Ycg1, the other non-SMC subunit was only flexibly bound to the complex, while also binding DNA tightly, and often remaining at a distance from the head module. In the clamped state, the DNA is encircled, or topologically entrapped, by the kleisin Brn1 and the two engaged head domains of Smc2 and Smc4, and this tripartite ring is closed at all interfaces, including at the neck of Smc2. We show that the neck gate opens upon head engagement in the absence of DNA, but it remains shut when DNA is present. Our work demonstrates that condensin and other SMC complexes go through similar conformations of the head modules during their ATPase cycle. In contrast, the behaviour of the Ycg1 subunit in the condensin complex might indicate differences in the implementation of the extrusion reactions and our findings will constrain further mechanistic models of loop extrusion by SMC complexes.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465916', 'tweets': 31, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'MLF2 modulates phase separated nuclear envelope condensates that provoke dual proteotoxicity', 'authors': \"[{'name': 'Sarah M Prophet'}, {'name': 'Anthony J Rampello'}, {'name': 'Robert F Niescier'}, {'name': 'Juliana E Shaw'}, {'name': 'Anthony J Koleske'}, {'name': 'Christian Schlieker'}]\", 'abstract': '<p>DYT1 dystonia is a highly debilitating neurological movement disorder arising from mutation in the AAA+ ATPase TorsinA. The hallmark of Torsin dysfunction is nuclear envelope blebbing resulting from defects in nuclear pore complex biogenesis. Whether blebs actively contribute to disease manifestation is presently unknown. We report that FG-nucleoporins in the bleb lumen undergo phase separation and contribute to DYT1 dystonia by provoking two proteotoxic insults. Short-lived ubiquitinated proteins that are normally rapidly degraded in healthy cells partition into the bleb lumen and become stabilized. Additionally, blebs selectively sequester a chaperone network composed of HSP70s and HSP40s. The composition of this chaperone network is altered by the bleb component MLF2. We further demonstrate that MLF2 is a catalyst of phase separation that suppresses the ectopic accumulation of FG-nucleoporins and modulates the selective properties and size of condensates <italic>in vitro</italic>. Our studies identify unprecedented, dual mechanisms of proteotoxicity in the context of liquid-liquid phase separation with direct implications for our understanding of disease etiology and treatment.</p>', 'posted': '2021-10-27'}, {'doi': '10.1101/2021.10.28.466259', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Modelling human zygotic genome activation in 8C-like cells in vitro', 'authors': \"[{'name': 'Jasmin Taubenschmid-Stowers'}, {'name': 'Maria Rostovskaya'}, {'name': 'Fatima Santos'}, {'name': 'Sebastian Ljung'}, {'name': 'Ricard Argelaguet'}, {'name': 'Felix Krueger'}, {'name': 'Jennifer Nichols'}, {'name': 'Wolf Reik'}]\", 'abstract': '<p>The remodelling of the epigenome and transcriptome of the fertilised oocyte to establish totipotency in the zygote and developing embryo is one of the most critical processes in mammalian embryogenesis. Zygotic or embryonic genome activation (ZGA, EGA) in the 2-cell embryo in mouse, and the 8-cell embryo in humans, constitutes the first major wave of transcription. Failure to initiate ZGA leads to developmental defects, and contributes to the high attrition rates of human pre-implantation embryos. Due to limitations in cell numbers and experimental tractability, the mechanisms that regulate human embryonic genome activation in the totipotent embryo remain poorly understood. Here we report the discovery of human 8-cell like cells (8CLCs) specifically among naive embryonic stem cells, but not primed pluripotent cells. 8CLCs express ZGA marker genes such as ZSCAN4, LEUTX and DUXA and their transcriptome closely resembles that of the 8-cell human embryo. 8-cell like cells reactivate 8-cell stage specific transposable elements such as HERVL and MLT2A1 and are characterized by upregulation of the DNA methylation regulator DPPA3. 8CLCs show reduced SOX2 protein, and can be identified based on expression of the novel ZGA-associated protein markers TPRX1 and H3.Y in vitro. Overexpression of the transcription factor DUX4 as well as spliceosome inhibition increase ZGA-like transcription and enhance TPRX1+ 8CLCs formation. Excitingly, the in vitro identified 8CLC marker proteins TPRX1 and H3.Y are also expressed in 8-cell human embryos at the time of genome activation and may thus be relevant in vivo. The discovery of 8CLCs provides a unique opportunity to model and manipulate human ZGA-like transcriptional programs in vitro, and might provide critical functional insights into one of the earliest events in human embryogenesis in vivo.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466356', 'tweets': 26, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Elevated Hoxb5b expands vagal neural crest pool and blocks enteric neuronal development in zebrafish', 'authors': \"[{'name': 'Aubrey G. Adam Howard'}, {'name': 'Aaron C Nguyen'}, {'name': 'Joshua Tworig'}, {'name': 'Priya Ravisankar'}, {'name': 'Eileen Willey Singleton'}, {'name': 'Can Li'}, {'name': 'Grayson Kotzur'}, {'name': 'Joshua S Waxman'}, {'name': 'Rosa A Uribe'}]\", 'abstract': '<p>Neural crest cells (NCCs) are a migratory, transient, and multipotent stem cell population essential to vertebrate embryonic development, contributing to numerous cell lineages in the adult organism. While great strides have been made in elucidating molecular and cellular events that drive NCC specification, comprehensive knowledge of the genetic factors that orchestrate NCC developmental programs is still far from complete. We discovered that elevated Hoxb5b levels promoted an expansion of zebrafish NCCs, which persisted throughout multiple stages of development. Correspondingly, elevated Hoxb5b also specifically expanded expression domains of the vagal NCC markers foxd3 and phox2bb. Increases in NCCs were most apparent after pulsed ectopic Hoxb5b expression at early developmental stages, rather than later during differentiation stages, as determined using a novel transgenic zebrafish line. The increase in vagal NCCs early in development led to supernumerary Phox2b+ enteric neural progenitors, while leaving many other NCC-derived tissues without an overt phenotype. Surprisingly, these NCC-derived enteric progenitors failed to expand properly into sufficient quantities of enterically fated neurons and stalled in the gut tissue. These results suggest that while Hoxb5b participates in vagal NCC development as a driver of progenitor expansion, the supernumerary, ectopically localized NCC fail to initiate expansion programs in timely fashion in the gut.  All together, these data point to a model in which Hoxb5b regulates NCCs both in a tissue specific and temporally restricted manner.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465843', 'tweets': 25, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Diversity, function and evolution of marine microbe genomes', 'authors': \"[{'name': 'Jianwei Chen'}, {'name': 'Yang Guo'}, {'name': 'Yangyang Jia'}, {'name': 'Guilin Liu'}, {'name': 'Denghui Li'}, {'name': 'Dayou Xu'}, {'name': 'Bing Wang'}, {'name': 'Li Zhou'}, {'name': 'Ling Peng'}, {'name': 'Fang Zhao'}, {'name': 'Yuanfang Zhu'}, {'name': 'Jiahui Sun'}, {'name': 'Chen Ye'}, {'name': 'Jun Wang'}, {'name': 'He Zhang'}, {'name': 'Shanshan Liu'}, {'name': 'Inge Seim'}, {'name': 'Xin Liu'}, {'name': 'Xun Xu'}, {'name': 'Huanming Yang'}, {'name': 'Karsten Kristiansen'}, {'name': 'Guangyi Fan'}, {'name': ' '}]\", 'abstract': '<p>Trillions of marine bacterial, archaeal and viral species contribute to the majority diversity of life on Earth. In the current study, we have done a comprehensive review of all the published studies of marine microbiome by re-analyzing most of the available high throughput sequencing data. We collected 17.59 Tb sequencing data from 8,165 metagenomic and prokaryotic samples, and systematically evaluated the genome characters, including genome size, GC content, phylogeny, and the functional and ecological roles of several typical phyla. A genome catalogue of 9,070 high quality genomes and a gene catalogue including 156,209,709 genes were constructed, representing the most integrate marine prokaryotic datasets till now. The genome size of Alphaproteobacteria and Actinobacteria was significant correlated to their GC content. A total of 44,322 biosynthetic gene clusters distributed in 53 types were detected from the reconstructed marine prokaryotic genome catalogue. Phylogenetic annotation of the 8,380 bacterial and 690 archaeal species revealed that most of the known bacterial phyla (99/111), including 62 classes and 181 orders, and four extra unclassified genomes from two candidate novel phyla were detected. In addition, taxonomically unclassified species represented a substantial fraction of 64.56% and 80.29% of the phylogenetic diversity of Bacteria and Archaea respectively. The genomic and ecological features of three groups of Cyanobacteria, luminous bacteria and methane-metabolizing archaea, including inhabitant preference, geolocation distribution and others were through discussed. Our database provides a comprehensive resource for marine microbiome, which would be a valuable reference for studies of marine life origination and evolution, ecology monitor and protection, bioactive compound development.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466170', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Inducible transcriptional condensates drive 3D genome reorganization in the heat shock response', 'authors': \"[{'name': 'Surabhi Chowdhary'}, {'name': 'Amoldeep S. Kainth'}, {'name': 'Sarah Paracha'}, {'name': 'David S. Gross'}, {'name': 'David Pincus'}]\", 'abstract': '<p>Mammalian developmental and disease-associated genes concentrate large quantities of the transcriptional machinery by forming membrane-less compartments known as transcriptional condensates. However, it is unknown whether these structures are evolutionarily conserved, capable of stress-inducible gene activation or involved in 3D genome reorganization. Here, we identify inducible transcriptional condensates in the yeast heat shock response (HSR). HSR condensates are biophysically dynamic spatiotemporal clusters of the sequence-specific transcription factor Heat shock factor 1 (Hsf1) with Mediator and RNA Pol II. Uniquely, HSR condensates drive the coalescence of multiple Hsf1 target genes, even those located on different chromosomes. Binding of the chaperone Hsp70 to a site on Hsf1 represses clustering, while an intrinsically disordered region on Hsf1 promotes condensate formation and intergenic interactions. Mutation of both Hsf1 determinants reprograms HSR condensates to become mammalian-like: constitutively active without intergenic coalescence. These results suggest that transcriptional condensates are ancient and flexible compartments of eukaryotic gene control.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.27.466024', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Tree-based Correlation Screen and Visualization for Exploring Phenotype-Cell Type Association in Multiple Sample Single-Cell RNA-Sequencing Experiments', 'authors': \"[{'name': 'Boyang Zhang'}, {'name': 'Zhicheng Ji'}, {'name': 'Hongkai Ji'}]\", 'abstract': '<p>Single-cell RNA-seq experiments with multiple samples are increasingly used to discover cell types and their molecular features that may influence sample phenotype (e.g. disease). However, analyzing and visualizing the complex cell type-phenotype association remains nontrivial. TreeCorTreat is an open source R package that tackles this problem by using a tree-based correlation screen to analyze and visualize the association between phenotype and transcriptomic features and cell types at multiple cell type resolution levels. With TreeCorTreat, one can conveniently explore and compare different feature types, phenotypic traits, analysis protocols and datasets, and evaluate the impacts of potential confounders.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465912', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'RNA compensation: A positive feedback insulation strategy for RNA-based networks', 'authors': \"[{'name': 'Baiyang Liu'}, {'name': 'Christian Cuba Samaniego'}, {'name': 'Matthew Bennett'}, {'name': 'James Chappell'}, {'name': 'Elisa Franco'}]\", 'abstract': '<p>The lack of signalling modularity of biomolecular systems poses major challenges toward engineering complex networks. An important problem is posed by the consumption of signaling molecules upon circuit interconnection, which makes it possible to control a downstream circuit but compromises the performance of the upstream circuit. This issue has been previously addressed with insulation strategies including high-gain negative feedback and phosphorylation-dephosphorylation reaction cycle. In this paper, we focus on RNA-based circuits and propose a new positive-feedback insulation strategy to mitigate signal consumption. An RNA input is added in tandem with transcription output to compensate the RNA consumption, leading to concentration robustness of the input RNA molecule regardless of the amount of downstream modules. We term this strategy RNA compensation, and it can be applied to systems that have a stringent input-output gain, such as Small Transcription Activating RNAs (STARs). Our analysis shows that RNA compensation not only eliminates the signaling consumption in individual STAR-based regulators, but also improves the composability of STAR cascades and the modularity of RNA bistable systems.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.466255', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Plasticity in stomatal behavior across a gradient of water supply is consistent among field-grown maize inbred lines with varying stomatal patterning', 'authors': \"[{'name': 'Risheng Ding'}, {'name': 'Jiayang Xie'}, {'name': 'Dustin Mayfield-Jones'}, {'name': 'Yanqun Zhang'}, {'name': 'Shaozhong Kang'}, {'name': 'Andrew Leakey'}]\", 'abstract': '<p>Stomata regulate leaf CO2 assimilation (A) and water loss. The Ball-Berry and Medlyn models predict stomatal conductance (gs) with a slope parameter (m or g1) that reflects sensitivity of gs to A, atmospheric CO2 and humidity, and is inversely related to water use efficiency (WUE). This study addressed knowledge gaps about what the values of m and g1 are in C4 crops under field conditions, as well as how they vary among genotypes and with drought stress. m and g1 were unexpectedly consistent in four inbred maize genotypes across a gradient of water supply. This was despite genotypic variation in stomatal patterning, A and gs. m and g1 were strongly correlated with soil water content, moderately correlated with pre-dawn leaf water potential (pd), and weakly correlated with midday leaf water potential (md). This implied that m and g1 respond to long-term water supply more than short-term drought stress. The conserved nature of m and g1 across anatomically diverse genotypes and water supplies suggests there is flexibility in structure-function relationships underpinning WUE. This evidence can guide simulation of maize gs across a range of water supply in the primary maize growing region and inform efforts to improve WUE.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465934', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Open chromatin analysis in Trypanosoma cruzi life forms highlights critical differences in genomic compartments and developmental regulation at tDNA loci', 'authors': \"[{'name': 'Alex Ranieri Jernimo Lima'}, {'name': 'Saloe Bispo Poubel'}, {'name': 'Juliana Nunes Rosn'}, {'name': 'Loyze Paola Oliveira de Lima'}, {'name': 'Hllida Marina Costa-Silva'}, {'name': 'Herbert Guimares de Sousa Silva'}, {'name': 'Camila Silva Gonalves'}, {'name': 'Pedro A. F. Galante'}, {'name': 'Fabiola Holetz'}, {'name': 'Maria Cristina Machado M. Motta'}, {'name': 'Ariel M. Silber'}, {'name': 'M. Carolina Elias'}, {'name': 'Julia Pinheiro Chagas da Cunha'}]\", 'abstract': '<sec><title>Background</title><p>Genomic organization and gene expression regulation in trypanosomes are remarkable because protein-coding genes are organized into codirectional gene clusters with unrelated functions. Moreover, there is no dedicated promoter for each gene, resulting in polycistronic gene transcription, with posttranscriptional control playing a major role. Nonetheless, these parasites harbor epigenetic modifications at critical regulatory genome features that dynamically change among parasite stages, which are not fully understood.</p></sec><sec><title>Results</title><p>Here, we investigated the impact of chromatin changes in a scenario commanded by posttranscriptional control exploring the parasite <italic>Trypanosoma cruzi</italic> and its differentiation program using genome-wide approaches supported by transmission electron microscopy. The integration of FAIRE and MNase-seq data, two complementary epigenomic approaches, enabled us to identify differences in <italic>T. cruzi</italic> genome compartments, putative transcriptional start regions and virulence factors. In addition, we also detected developmental chromatin regulation at tRNA loci (tDNA), which seems to be linked to the translation regulatory mechanism required for parasite differentiation. Strikingly, a positive correlation was observed between active chromatin and steady-state transcription levels.</p></sec><sec><title>Conclusion</title><p>Taken together, our results indicate that chromatin changes reflect the unusual gene expression regulation of trypanosomes and the differences among parasite developmental stages, even in the context of a lack of canonical transcriptional control of protein-coding genes.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466000', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Neurons in human pre-supplementary motor area encode key computations for value-based choice', 'authors': \"[{'name': 'Tomas G. Aquino'}, {'name': 'Jeffrey Cockburn'}, {'name': 'Adam N. Mamelak'}, {'name': 'Ueli Rutishauser'}, {'name': 'John P. ODoherty'}]\", 'abstract': '<p>Adaptive behavior in real-world environments demands that choices integrate over several variables, including the novelty of the options under consideration, their expected value, and uncertainty in value estimation. We recorded neurons from the human pre-supplementary motor area (preSMA), ventromedial prefrontal cortex (vmPFC) and dorsal anterior cingulate to probe how integration over decision variables occurs during decision-making. In contrast to the other areas, preSMA neurons not only represented separate pre-decision variables for each choice option, but also encoded an integrated utility signal and, subsequently, the decision itself. Conversely, post-decision related encoding of variables for the chosen option was more widely distributed and especially prominent in vmPFC. Our findings position the human preSMA as central to the implementation of value-based decisions.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.25.465778', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Cost-effective epigenetic age profiling in shallow methylation sequencing data', 'authors': \"[{'name': 'Alexandre Trapp'}, {'name': 'Vadim N Gladyshev'}]\", 'abstract': '<p>There is a critical need for robust, high-throughput assays of biological aging trajectories. Among various approaches, epigenetic aging clocks emerged as reliable molecular trackers of the aging process. However, current methods for epigenetic age profiling are inherently costly and lack throughput. Here, we leverage the scAge framework for accurate prediction of biological age from very few bisulfite sequencing reads in bulk samples, thereby enabling drastic (100-1,000-fold) reduction in sequencing costs per sample. Our approach permits age assessment based on distinct assortments of CpG sites in different samples, without the need for targeted site enrichment or specialized reagents. We demonstrate the efficacy of this method to quantify the age of mouse blood samples across independent cohorts, identify the effect of calorie restriction as an attenuator of the aging process, and discern rejuvenation upon cellular reprogramming. We propose that this framework may be used for epigenetic age prediction in extremely high-throughput applications, enabling robust, large-scale and inexpensive interventions testing and age profiling.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.29.466408', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'The glycosylated extracellular domain of MUC1 protects against SARS-CoV-2 infection at the respiratory surface', 'authors': \"[{'name': 'Maitrayee Chatterjee'}, {'name': 'Liane Z.X Huang'}, {'name': 'Chunyan Wang'}, {'name': 'Anna Z Mykytyn'}, {'name': 'Bart Westendorp'}, {'name': 'Richard W Wubbolts'}, {'name': 'Berend Jan Bosch'}, {'name': 'Bart Haagmans'}, {'name': 'Jos P.M van Putten'}, {'name': 'Karin Strijbis'}]\", 'abstract': '<p>Mucins play an essential role in protecting the respiratory tract against microbial infections. The heavily O-glycosylated gel-forming mucins MUC5AC and MUC5B eliminate pathogens by mucociliary clearance while transmembrane mucins MUC1, MUC4, and MUC16 restrict microbial invasion at the apical surface of the epithelium. In this study, we determined the impact of host mucins and mucin glycans on SARS-CoV-2 spike-mediated epithelial entry. Human lung epithelial Calu-3 cells have endogenous expression of the SARS-CoV-2 entry receptor ACE2 and express high levels of glycosylated MUC1 on the surface but not MUC4 and MUC16. Removal of the MUC1 extracellular domain (ED) using the O-glycan-specific mucinase StcE greatly enhanced spike binding and viral infection. By contrast, removal of mucin glycans sialic acid and fucose did not impact viral invasion. This study implicates the glycosylated ED of MUC1 as an important component of the host defense that restricts the severity of SARS-CoV-2 infection.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465878', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Chromosome-scale assemblies of Acanthamoeba castellanii genomes provide insights into Legionella pneumophila infection-related chromatin re-organization', 'authors': \"[{'name': 'Cyril Matthey-Doret'}, {'name': 'Morgan J. Colp'}, {'name': 'Pedro Escoll'}, {'name': 'Agns Thierry'}, {'name': 'Bruce Curtis'}, {'name': 'Matt Sarrasin'}, {'name': 'Michael W. Gray'}, {'name': 'B. Franz Lang'}, {'name': 'John M. Archibald'}, {'name': 'Carmen Buchrieser'}, {'name': 'Romain Koszul'}]\", 'abstract': '<p>The unicellular amoeba <italic>Acanthamoeba castellanii</italic> is ubiquitous in aquatic environments, where it preys on bacteria. The organism also hosts bacterial endosymbionts, some of which are parasitic, including human pathogens such as <italic>Chlamydia</italic> and <italic>Legionella</italic> spp. Here we report complete, high quality genome sequences for two extensively studied <italic>A. castellanii</italic> strains, Neff and C3. Combining long- and short-read data with Hi-C, we generated near chromosome-level assemblies for both strains with 90% of the genome contained in 29 scaffolds for the Neff strain and 31 for the C3 strain. Comparative genomics revealed strain-specific functional enrichment, most notably genes related to signal transduction in the C3 strain, and to viral replication in Neff. Furthermore, we characterized the spatial organization of the <italic>A. castellanii</italic> genome and showed that it is reorganized during infection by <italic>Legionella pneumophila.</italic> Infection-dependent chromatin loops were found to be enriched in genes for signal transduction and phosphorylation processes. In genomic regions where chromatin organization changed during <italic>Legionella</italic> infection, we found functional enrichment for genes associated with metabolism, organelle assembly, and cytoskeleton organization, suggesting that changes in chromosomal folding are associated with host cell biology during infection.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466206', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': True, 'title': 'A multi-tissue study of immune gene expression profiling highlights the key role of the nasal epithelium in COVID-19 severity', 'authors': \"[{'name': 'Alberto Gomez-Carballa'}, {'name': 'Irene Rivero-Calle'}, {'name': 'Jacobo Pardo-Seco'}, {'name': 'Jose Gomez-Rial'}, {'name': 'Carmen Rivero-Velasco'}, {'name': 'Nuria Rodriguez-Nunez'}, {'name': 'Gema Barbeito-Castineiras'}, {'name': 'Hugo Perez-Freixo'}, {'name': 'Miriam Cebey-Lopez'}, {'name': 'Ruth Barral-Arca'}, {'name': 'Carmen Rodriguez-Tenreiro'}, {'name': 'Ana Dacosta-Urbieta'}, {'name': 'Xabier Bello'}, {'name': 'Sara Pischedda'}, {'name': 'Maria Jose Curras-Tuala'}, {'name': 'Sandra Viz-Lasheras'}, {'name': 'Federico Martinon-Torres'}, {'name': 'Antonio Salas'}, {'name': ' '}]\", 'abstract': '<p>Background: COVID-19 symptoms range from mild to severe illness; the cause for this differential response to infection remains unknown. Unravelling the immune mechanisms acting at different levels of the colonization process might be key to understand these differences. Methods and findings: We carried out a multi-tissue (nasal, buccal and blood; n = 156) gene expression analysis of immune-related genes from patients affected by different COVID-19 severities, and healthy controls through the nCounter technology. We then used a differential expression approach and pathways analysis to detect tissue specific immune severity signals in COVID-19 patients. Mild and asymptomatic cases showed a powerful innate antiviral response in nasal epithelium, characterized by activation of interferon (IFN) pathway and downstream cascades, successfully controlling the infection at local level. In contrast, weak macrophage/monocyte driven innate antiviral response and lack of IFN signalling activity were shown in severe cases. Consequently, oral mucosa from severe patients showed signals of viral activity, cell arresting and viral dissemination to the lower respiratory tract, which ultimately could explain the exacerbated innate immune response and impaired adaptative immune responses observed at systemic level. Results from saliva transcriptome suggest that the buccal cavity might play a key role in SARS-CoV-2 infection and dissemination in patients with worse prognosis. Conclusions: We found severity-related signatures in patient tissues mainly represented by genes involved in the innate immune system and cytokine/chemokine signalling. Local immune response could be key to determine the course of the systemic response and thus COVID-19 severity. Our findings provide a framework to investigate severity host gene biomarkers and pathways that might be relevant to diagnosis, prognosis, and therapy.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.21265616', 'tweets': 19, 'archive': 'medRxiv', 'subject-area': 'Pediatrics', 'covid': True, 'title': 'Risk factors for severe PCR-positive SARS-CoV-2 infection in hospitalized children: a multicenter cohort study', 'authors': \"[{'name': 'Tilmann Schober'}, {'name': 'Chelsea Caya'}, {'name': 'Michelle Barton'}, {'name': 'Ann Bayliss'}, {'name': 'Ari Bitnun'}, {'name': 'Jennifer Bowes'}, {'name': 'Helena Brenes-Chacon'}, {'name': 'Jared Bullard'}, {'name': 'Suzette Cooke'}, {'name': 'Tammie Dewan'}, {'name': 'Rachel Dwilow'}, {'name': 'Tala El Tal'}, {'name': 'Cheryl Foo'}, {'name': 'Peter Gill'}, {'name': 'Behzad Haghighi Aski'}, {'name': 'Fatima Kakkar'}, {'name': 'Janell Lautermilch'}, {'name': 'Ronald M. Laxer'}, {'name': 'Marie-Astrid Lefebvre'}, {'name': 'Kirk Leifso'}, {'name': 'Nicole Le Saux'}, {'name': 'Alison Lopez'}, {'name': 'Ali Manafi'}, {'name': 'Shaun K. Morris'}, {'name': 'Alireza Nateghian'}, {'name': 'Luc Panetta'}, {'name': 'Dara Petel'}, {'name': 'Dominique Piche'}, {'name': 'Rupeena Purewal'}, {'name': 'Lea Restivo'}, {'name': 'Ashley Roberts'}, {'name': 'Manish Sadarangani'}, {'name': 'Rosie Scuccimarri'}, {'name': 'Alejandra Soriano-Fallas'}, {'name': 'Sarah Tehseen'}, {'name': 'Karina A. Top'}, {'name': 'Rolando Ulloa-Gutierrez'}, {'name': 'Isabelle Viel-Theriaul'}, {'name': 'Jacqueline K. Wong'}, {'name': 'Carmen Yea'}, {'name': 'Ann Yeh'}, {'name': 'Adriana Yock-Corrales'}, {'name': 'Joan Robinson'}, {'name': 'Jesse Papenburg'}]\", 'abstract': \"<p>Importance: Children are less likely than adults to have severe outcomes from SARS-CoV-2 infection and the corresponding risk factors are not well established. \\nObjective: To identify risk factors for severe disease in symptomatic children hospitalized for PCR-positive SARS-CoV-2 infection.\\nDesign: Cohort study, enrollment from February 1, 2020 until May 31, 2021\\nSetting 15 children's hospitals in Canada, Iran, and Costa Rica\\nParticipants: Patients &lt;18 years of age hospitalized with symptomatic SARS-CoV-2 infection, including PCR-positive multisystem inflammatory syndrome in children (MIS-C)\\nExposures: Variables assessed for their association with disease severity included patient demographics, presence of comorbidities, clinical manifestations, laboratory parameters and chest imaging findings.\\nMain Outcomes and Measures: The primary outcome was severe disease defined as a WHO COVID-19 clinical progression scale of 6, i.e., requirement of non-invasive ventilation, high flow nasal cannula, mechanical ventilation, vasopressors, or death. Multivariable logistic regression was used to evaluate factors associated with severe disease.\\nResults: We identified 403 hospitalizations. Median age was 3.78 years (IQR 0.53-10.77). At least one comorbidity was present in 46.4% (187/403) and multiple comorbidities in 18.6% (75/403). Severe disease occurred in 33.8% (102/403). In multivariable analyses, presence of multiple comorbidities (adjusted odds ratio 2.24, 95% confidence interval 1.04-4.81), obesity (2.87, 1.19-6.93), neurological disorder (3.22, 1.37-7.56), anemia, and/or hemoglobinopathy (5.88, 1.30-26.46), shortness of breath (4.37, 2.08-9.16), bacterial and/or viral coinfections (2.26, 1.08-4.73), chest imaging compatible with COVID-19 (2.99, 1.51-5.92), neutrophilia (2.60, 1.35-5.02), and MIS-C diagnosis (3.86, 1.56-9.51) were independent risk factors for severity. Comorbidities, especially obesity (40.9% vs 3.9%, p&lt;0.001), were more frequently present in adolescents 12 years of age. Neurological disorder (3.16, 1.19-8.43) in children &lt;12 years of age and obesity (3.21, 1.15-8.93) in adolescents were the specific comorbidities associated with disease severity in age-stratified adjusted analyses. Sensitivity analyses excluding the 81 cases with MIS-C did not substantially change the identified risk factors. \\nConclusions and Relevance: Pediatric risk factors for severe SARS-CoV-2 infection vary according to age and can potentially guide vaccination programs and treatment approaches in children.</p>\", 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466306', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Vagus nerve stimulation accelerates motor learning through cholinergic modulation', 'authors': \"[{'name': 'Spencer Bowles'}, {'name': 'Jordan Hickman'}, {'name': 'Xiaoyu Peng'}, {'name': 'W. Ryan Williamson'}, {'name': 'Rongchen Huang'}, {'name': 'Kayden Washington'}, {'name': 'Dane Donegan'}, {'name': 'Cristin Welle'}]\", 'abstract': '<p>Vagus nerve stimulation (VNS) is a novel therapeutic option to treat a broad and rapidly expanding set of neurologic conditions. While classically used to treat epilepsy and depression, more recently VNS has received FDA approval as a therapeutic option for stroke rehabilitation and is under preclinical and clinical investigation for other disorders. Despite benefits across a diverse range of neurological disorders, the mechanism for how VNS influences central nervous system circuitry is not well described. A deeper understanding of the influence of VNS on neural circuits and activity is needed to optimize the use of VNS therapy. To define the complex dynamics between VNS, neuronal activity, plasticity, and behavior, we performed chronic VNS in mice during the learning of a dexterous motor task, and leveraged genetic tools to perform optogenetic circuit dissection and longitudinal in vivo imaging calcium activity in cortical neurons. We find VNS has the most robust effect on motor learning when paired with successful movement outcome, while randomized stimulation impairs learning, consistent with VNS serving as a reinforcement cue. In motor cortex, VNS paired with movement outcome selectively modulates neurons that are representing outcome, but not other movement-related neurons. Finally, cholinergic signaling from basal forebrain is required both for VNS-driven improvements in motor learning and the effects on neural activity in M1. This suggests that the effect of VNS on motor learning is mediated by cholinergic signaling, and also presents a novel role for cholinergic signaling in endogenous motor learning. These data imply that VNS therapy may be mediated by augmenting reinforcement cues with precisely-timed cholinergic neuronal activity, presenting strategies for optimizing the use of VNS to treat neurologic conditions.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465991', 'tweets': 18, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'FiberNeat: unsupervised streamline clustering and white matter tract filtering in latent space', 'authors': \"[{'name': 'Bramsh Qamar Chandio'}, {'name': 'Tamoghna Chattopadhyay'}, {'name': 'Conor Owens-Walton'}, {'name': 'Julio E. Villalon Reina'}, {'name': 'Leila Nabulsi'}, {'name': 'Sophia I. Thomopoulos'}, {'name': 'Eleftherios Garyfallidis'}, {'name': 'Paul M. Thompson'}]\", 'abstract': '<p>Whole-brain tractograms generated from diffusion MRI digitally represent the white matter structure of the brain and are composed of millions of streamlines. Such tractograms can have false positive and anatomically implausible streamlines. To obtain anatomically relevant streamlines and tracts, supervised and unsupervised methods can be used for tractogram clustering and tract extraction. Here we propose FiberNeat, an unsupervised streamline clustering and tract filtering method. FiberNeat takes an input set of streamlines that could either be unlabeled clusters or labeled tracts. Individual clusters/tracts are projected into a latent space using nonlinear dimensionality reduction techniques, such as t-SNE and UMAP, to find spurious and outlier streamlines. In addition, outlier streamline clusters are detected using DBSCAN and then removed from the data in streamline space. Quantitative comparisons with expertly delineated tracts show the promise of the approach. This approach can be deployed as a filtering step after tracts are extracted.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.26.465990', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Geological processes mediate a subsurface microbial loop in the deep biosphere', 'authors': \"[{'name': 'Daniel A. Gittins'}, {'name': 'Pierre-Arnaud Desiage'}, {'name': 'Natasha Morrison'}, {'name': 'Jayne E. Rattray'}, {'name': 'Srijak Bhatnagar'}, {'name': 'Anirban Chakraborty'}, {'name': 'Jackie Zorz'}, {'name': 'Carmen Li'}, {'name': 'Oliver Horanszky'}, {'name': 'Margaret A. Cramm'}, {'name': 'Jamie Webb'}, {'name': 'Adam MacDonald'}, {'name': 'Martin Fowler'}, {'name': 'D. Calvin Campbell'}, {'name': 'Casey R. J. Hubert'}]\", 'abstract': '<p>The deep biosphere is the largest microbial habitat on Earth and features abundant bacterial endospores<sup>1,2</sup>. Whereas dormancy and survival at theoretical energy minima are hallmarks of subsurface microbial populations<sup>3</sup>, the roles of fundamental ecological processes like dispersal and selection in these environments are poorly understood<sup>4</sup>. Here we combine geophysics, geochemistry, microbiology and genomics to investigate biogeography in the subsurface, focusing on bacterial endospores in a deep-sea setting characterized by thermogenic hydrocarbon seepage. Thermophilic endospores in permanently cold seabed sediments above petroleum seep conduits were correlated with the presence of hydrocarbons, revealing geofluid-facilitated cell migration pathways originating in deep oil reservoirs. Genomes of thermophilic bacteria highlight adaptations to life in anoxic petroleum systems and reveal that these dormant populations are closely related to oil reservoir microbiomes from around the world. After transport out of the subsurface and into the deep-sea, thermophilic endospores re-enter the geosphere by sedimentation. Viable thermophilic endospores spanning the top several metres of the seabed correspond with total endospore counts that are similar to or exceed the global average. Burial of dormant cells enables their environmental selection in sedimentary formations where new petroleum systems establish, completing a geological microbial loop that circulates living biomass in and out of the deep biosphere.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.465695', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Mother centrioles generate a local pulse of Polo/PLK1 activity to initiate mitotic centrosome assembly', 'authors': \"[{'name': 'Siu-Shing Wong'}, {'name': 'Zachary M. Wilmott'}, {'name': 'Saroj Saurya'}, {'name': 'Felix Y. Zhou'}, {'name': 'Kwai-Yin Chau'}, {'name': 'Alain Goriely'}, {'name': 'Jordan W. Raff'}]\", 'abstract': '<p>Mitotic centrosomes are formed when centrioles recruit large amounts of pericentriolar material (PCM) around themselves. This centrosome maturation requires the centrioles and also Polo/PLK1 protein kinase. The PCM comprises several hundred proteins and, in <italic>Drosophila</italic>, Polo cooperates with the conserved centrosome proteins Spd-2/CEP192 and Cnn/CDK5RAP2 to assemble a PCM scaffold around the mother centriole that then recruits other PCM client proteins. We show here that in <italic>Drosophila</italic> syncytial blastoderm embryos, centrosomal Polo levels rise and fall during the assembly processpeaking, and then starting to decline, even as levels of the PCM scaffold continue to rise. Experiments and mathematical modelling indicate that a centriolar pulse of Polo activity, potentially generated by the interaction between Polo and its centriole receptor Ana1 (CEP295 in humans), could explain these unexpected scaffold assembly dynamics. We propose that centrioles generate a local pulse of Polo activity prior to mitotic entry to initiate centrosome maturation, explaining why centrioles and Polo/PLK1 are normally essential for this process.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.466271', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Cortical adaptation to sound reverberation', 'authors': \"[{'name': 'Aleksandar Z Ivanov'}, {'name': 'Andrew J King'}, {'name': 'Ben Willmore'}, {'name': 'Kerry M M Walker'}, {'name': 'Nicol S Harper'}]\", 'abstract': '<p>In almost every natural environment, sounds are reflected by nearby objects, producing many delayed and distorted copies of the original sound, known as reverberation. Our brains usually cope well with reverberation, allowing us to recognize sound sources regardless of their environments. In contrast, reverberation can cause severe difficulties for speech recognition algorithms and hearing-impaired people. The present study examines how the auditory system copes with reverberation. We trained a linear model to recover a rich set of natural, anechoic sounds from their simulated reverberant counterparts. The model neurons achieved this by extending the inhibitory component of their receptive filters for more reverberant spaces, and did so in a frequency-dependent manner. These predicted effects were observed in the responses of auditory cortical neurons of ferrets in the same simulated reverberant environments. Together, these results suggest that auditory cortical neurons adapt to reverberation by adjusting their filtering properties in a manner consistent with dereverberation.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466254', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Activating natural product synthesis using CRISPR interference and activation systems in Streptomyces', 'authors': \"[{'name': 'Andrea Ameruoso'}, {'name': 'Maria Claudia Villegas Kcam'}, {'name': 'Katherine Piper Cohen'}, {'name': 'James Chappell'}]\", 'abstract': '<p>The rise of antibiotic-resistant bacteria represents a major threat to global health, creating an urgent need to discover new antibiotics. Natural products derived from the genus <italic>Streptomyces</italic> represent a rich and diverse repertoire of chemical molecules from which new antibiotics are likely to be found. However, a major challenge is that the biosynthetic gene clusters (BGCs) responsible for natural product synthesis are often poorly expressed under laboratory culturing conditions, thus preventing isolation and screening of novel chemicals. To address this, we describe a novel approach to activate silent BGCs through rewiring endogenous regulation using synthetic gene regulators based upon CRISPR-Cas. First, we create CRISPR interference (CRISPRi) and CRISPR activation (CRISPRa) systems that allow for highly programmable and effective gene repression and activation in <italic>Streptomyces</italic>. We then harness these tools to activate a silent BGC through perturbing its endogenous regulatory network. Together, this work advances the synthetic regulatory toolbox for <italic>Streptomyces</italic> and facilitates the programmable activation of silent BGCs for novel chemical discovery.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.26.465978', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'PDGFR dimer-specific activation, trafficking and downstream signaling dynamics', 'authors': \"[{'name': 'Madison A. Rogers'}, {'name': 'Katherine A. Fantauzzo'}]\", 'abstract': '<p>Signaling through the platelet-derived growth factor receptors (PDGFRs) plays a critical role in multiple cellular processes during development. The two PDGFRs, PDGFR and PDGFR, dimerize to form homodimers and/or heterodimers. Here, we overcome previous limitations in studying PDGFR dimer-specific dynamics by generating cell lines stably expressing C-terminal fusions of each PDGFR with bimolecular fluorescence complementation (BiFC) fragments corresponding to the N-terminal or C-terminal regions of the Venus fluorescent protein. We find that PDGFR receptors homodimerize more quickly than PDGFR receptors in response to PDGF ligand, with increased levels of autophosphorylation. Further, we demonstrate that PDGFR homodimers are trafficked and degraded more quickly, while PDGFR homodimers are more likely to be recycled back to the cell membrane. We show that PDGFR homodimer activation results in a greater amplitude of phospho-ERK1/2 and phospho-AKT signaling, as well as increased proliferation and migration. Collectively, our findings provide significant insight into how biological specificity is introduced to generate unique responses downstream of PDGFR engagement.</p><sec><title>Summary</title><p>The authors utilize a novel bimolecular fluorescence complementation approach to investigate PDGFR homodimer-specific dynamics. They uncover differences in the timing and extent of receptor dimerization, activation and trafficking, which lead to changes in downstream signaling and cellular activity.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.21265499', 'tweets': 16, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Time of day of vaccination affects SARS-CoV-2 antibody responses in an observational study of healthcare workers', 'authors': '[{\\'name\\': \\'Wei Wang\\'}, {\\'name\\': \\'Peter Balfe\\'}, {\\'name\\': \\'David W Eyre\\'}, {\\'name\\': \\'Sheila F Lumley\\'}, {\\'name\\': \"Denise O\\'Donnell\"}, {\\'name\\': \\'Fiona Warren\\'}, {\\'name\\': \\'Derrick W Crook\\'}, {\\'name\\': \\'Katie Jeffery\\'}, {\\'name\\': \\'Philippa C Matthews\\'}, {\\'name\\': \\'Elizabeth B Klerman\\'}, {\\'name\\': \\'Jane McKeating\\'}]', 'abstract': '<p>The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a global crisis with unprecedented challenges for public health. Vaccinations against SARS-CoV-2 have slowed the incidence of new infections and reduced disease severity. As the time-of-day of vaccination has been reported to influence host immune responses to multiple pathogens, we quantified the influence of SARS-CoV-2 vaccination time, vaccine type, age, sex, and days post-vaccination on anti-Spike antibody responses in healthcare workers. The magnitude of the anti-Spike antibody response associated with the time-of-day of vaccination, vaccine type, participant age, sex, and days post vaccination. These results may be relevant for optimizing SARS-CoV-2 vaccine efficacy.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465984', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'A cautionary tale on proper use of branch-site models to detect convergent positive selection', 'authors': \"[{'name': 'Amanda Kowalczyk'}, {'name': 'Maria Chikina'}, {'name': 'Nathan L Clark'}]\", 'abstract': '<p>Comparative genomics has become a powerful tool to elucidate genotype-phenotype relationships, particularly through the study of convergently acquired phenotypes. By identifying genes under positive selection specifically on branches with the convergent phenotype we can potentially link genes to that phenotype. Such gene scans are often done using branch-site codon models. However, we have observed a recent troubling trend of misinterpretation of branch-site models in which phylogeny-wide positive selection is not distinguished from positive selection specific to convergent branches. Here, we use simulations and real data to demonstrate how failing to discern between these two cases leads to false inferences of positive selection associated with a convergent trait. We then present a  drop-out test solution to distinguish the two cases and thereby truly capture positive selection events associated with convergent phenotypes, thus allowing for further insights into both genetic and phenotypic evolution.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.24.465610', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Neural basis of learning to perceive speech through touch using an acoustic-to-vibrotactile speech sensory substitution', 'authors': \"[{'name': 'Patrick S. Malone'}, {'name': 'Silvio P. Eberhardt'}, {'name': 'Edward T. Auer'}, {'name': 'Richard Klein'}, {'name': 'Lynne E. Bernstein'}, {'name': 'Maximilian Riesenhuber'}]\", 'abstract': '<p>The goal of sensory substitution is to convey the information transduced by one sensory system through a novel sensory modality. One example is vibrotactile (VT) speech, for which acoustic speech is transformed into vibrotactile patterns. Despite an almost century-long history of studying vibrotactile speech, there has been no study of the neural bases of VT speech learning. We here trained hearing adult participants to recognize VT speech syllables. Using fMRI, we showed that both somatosensory (left post-central gyrus) and auditory (right temporal lobe) regions acquire selectivity for VT speech stimuli following training. The right planum temporale in particular was selective for both VT and auditory speech. EEG source-estimated activity revealed temporal dynamics consistent with direct, low-latency engagement of right temporal lobe following activation of the left post-central gyrus. Our results suggest that VT speech learning achieves integration with the auditory speech system by piggybacking onto corresponding auditory speech representations.</p><sec><title>Significance statement</title><p>In sensory substitution, the information conveyed by one sensory system is used to replace the function of another. Blind individuals, for example, can learn to use a visual-to-acoustic sensory substitution device to navigate the world. We tested the hypothesis that sensory substitution can more generally occur in typical individuals, by exploiting the existence of multi-sensory convergence areas in the brain. We trained hearing participants to recognize speech syllables presented as vibrotactile stimulation patterns. Using fMRI and EEG, we show that vibrotactile speech learning integrates with the auditory speech system by piggybacking onto corresponding auditory speech representations. These results suggest that the human brain can bootstrap new senses by leveraging one sensory system to process information normally processed by another.</p></sec>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.28.466308', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Divergent evolutionary trajectories of bryophytes and tracheophytes from a complex common ancestor of land plants', 'authors': \"[{'name': 'Brogan J Harris'}, {'name': 'James W Clark'}, {'name': 'Dominik Schrempf'}, {'name': 'Gergely J Szllsi'}, {'name': 'Philip Donoghue'}, {'name': 'Alistair M Hetherington'}, {'name': 'Tom A Williams'}]\", 'abstract': '<p>The origin of plants and their colonization of land resulted in the transformation of the terrestrial environment. Here we investigate the evolution of the land plants (embryophytes) and their two main lineages, the tracheophytes (vascular plants) and bryophytes (non vascular plants). We used new fossil calibrations, relative lineage dating implied by horizontal gene transfer, and new phylogenomic methods for mapping gene family origins. Distinct rooting strategies resolve tracheophytes and bryophytes as monophyletic sister groups that diverged in the Cambrian, 515-494 Ma. The embryophyte stem is characterised by a burst of gene innovation, while bryophytes subsequently experienced a no less dramatic episode of reductive genome evolution in which they lost genes associated with the elaboration of vasculature and the stomatal complex. Overall, our analyses confirm that extant tracheophytes and bryophytes are both highly derived; as a result, understanding the origin of land plants requires tracing character evolution across the diversity of modern lineages.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.30.466508', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'A neural substrate for Bayesian integration in human parietal cortex', 'authors': \"[{'name': 'Nicholas M. Singletary'}, {'name': 'Jacqueline Gottlieb'}, {'name': 'Guillermo Horga'}]\", 'abstract': \"<p>Making adaptive decisions often requires inferring unobservable states based on unreliable information. Bayesian logic prescribes that individuals form probabilistic beliefs about a state by integrating the likelihood of new evidence with their prior beliefs, but human neuroimaging studies on probability representations have not typically examined this integration process. We developed an inference fMRI task in which participants estimated the posterior probability of a hidden state while we parametrically modulated the prior probability of the state, the likelihood of the supporting evidence, and a monetary penalty for estimation inaccuracy. Consistent with a neural substrate for Bayesian integration, activation in left posterior parietal cortex tracked the estimated posterior probability of the solicited state and its components of prior probability and likelihood, all independently of expected value. This activation further reflected deviations in individual reports from objective probabilities. Thus, this region may provide a neural substrate for humans' ability to approximate Bayesian inference.</p>\", 'posted': '2021-10-31'}, {'doi': '10.1101/2021.10.28.466333', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Deciphering the species-level structure of topologically associating domains', 'authors': \"[{'name': 'Rohit Singh'}, {'name': 'Bonnie Berger'}]\", 'abstract': '<p>Chromosome conformation capture technologies such as Hi-C have revealed a rich hierarchical structure of chromatin, with topologically associating domains (TADs) as a key organizational unit, but experimentally reported TAD architectures, currently determined separately for each cell type, are lacking for many cell/tissue types. A solution to address this issue is to integrate existing epigenetic data across cells and tissue types to develop a species-level consensus map relating genes to TADs. Here, we introduce the TAD Map, a bag-of-genes representation that we use to infer, or \"impute\", TAD architectures for those cells/tissues with limited Hi-C experimental data. The TAD Map enables a systematic analysis of gene coexpression induced by chromatin structure. By overlaying transcriptional data from hundreds of bulk and single-cell assays onto the TAD Map, we assess gene coexpression in TADs and find that expressed genes cluster into fewer TADs than would be expected by chance, and show that time-course and RNA velocity studies further reveal this clustering to be strongest in the early stages of cell differentiation; it is also strong in tumor cells. We provide a probabilistic model to summarize any scRNA-seq transcriptome in terms of its TAD activation profile, which we term a TAD signature, and demonstrate its value for cell type inference, cell fate prediction, and multimodal synthesis. More broadly, our work indicates that the TAD Map\\'s comprehensive, quantitative integration of chromatin structure and scRNA-seq data should play a key role in epigenetic and transcriptomic analyses.\\n\\nSoftware availability: https://tadmap.csail.mit.edu</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.23.21265427', 'tweets': 13, 'archive': 'medRxiv', 'subject-area': 'Occupational and Environmental Health', 'covid': False, 'title': 'Performance of Valved Respirators to Reduce Respiratory Particles Generated by Speaking', 'authors': \"[{'name': 'Jessica M. Hazard'}, {'name': 'Christopher D. Cappa'}]\", 'abstract': '<p>Wearing of face coverings serves two purposes: reducing the concentration of ambient particles inhaled and reducing the emission of respiratory particles generated by the wearer. The efficiency of different face coverings depends on the material, design, and fit. Face coverings such as N95 respirators, when worn properly, are highly efficient at filtering ambient particles during inhalation. Some N95 respirators, as well as other face covering types, include a one-way valve to allow easier exhalation while still maintaining high filtration efficiency towards ambient particles. The extent to which these valves decrease filtration of emitted respiratory particles is, however, not well established. Here, we show that different valved N95s exhibit highly variable filtration efficiencies for exhaled respiratory particles. As such, valved N95s do not provide reliable source control of respired particles and their use should be discouraged in situations where such source control is needed.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466135', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Inhibiting adult neurogenesis differentially affects spatial learning in females and males', 'authors': \"[{'name': 'Timothy P OLeary'}, {'name': 'Baran Askari'}, {'name': 'Bonnie Lee'}, {'name': 'Kathryn Darby'}, {'name': 'Cypress Knudson'}, {'name': 'Alyssa M Ash'}, {'name': 'Desiree R Seib'}, {'name': 'Delane F Espinueva'}, {'name': 'Jason S Snyder'}]\", 'abstract': '<p>Adult hippocampal neurogenesis has been implicated in the spatial processing functions of the hippocampus but ablating neurogenesis does not consistently lead to behavioral deficits in spatial tasks. Parallel studies have shown that adult-born neurons also regulate behavioral responses to stressful and aversive stimuli. We therefore hypothesized that spatial functions of adult-born neurons may be more prominent under conditions of stress, and may differ between males and females given established sex differences in stress responding. To test this we trained intact and neurogenesis-deficient rats in the spatial water maze at temperatures that vary in their degree of aversiveness. At standard temperatures (25C) ablating neurogenesis did not alter learning and memory in either sex, consistent with prior work. However, in cold water (16C), ablating neurogenesis had divergent sex-dependent effects: relative to intact rats, male neurogenesis-deficient rats were slower to escape and female neurogenesis-deficient rats were faster. Neurogenesis promoted temperature-related changes in search strategy in females, but it promoted search strategy stability in males. Females displayed greater recruitment of the dorsal hippocampus than males, particularly at 16C. However, blocking neurogenesis did not alter activity-dependent immediate-early gene expression in either sex. Finally, morphological analyses of retrovirally-labelled neurons revealed greater experience-dependent plasticity in new neurons in males. Neurons had comparable morphology in untrained rats but 16C training increased spine density, and 25C training caused shrinkage of mossy fiber presynaptic terminals, specifically in males. Collectively, these findings indicate that neurogenesis functions in memory are prominent under conditions of stress, they provide the first evidence for sex differences in the behavioral function of newborn neurons, and they suggest possibly distinct roles for neurogenesis in cognition and mental health in males and females.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.28.466256', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Tubular lysosome induction couples animal starvation to healthy aging', 'authors': \"[{'name': 'Tatiana V Villalobos'}, {'name': 'Bhaswati Ghosh'}, {'name': 'Sanaa Alam'}, {'name': 'Tyler J Butsch'}, {'name': 'Brennan M Mercola'}, {'name': 'Cara D Ramos'}, {'name': 'Suman Das'}, {'name': 'Eric D Eymard'}, {'name': 'K. Adam Bohnert'}, {'name': 'Alyssa E Johnson'}]\", 'abstract': '<p>Dietary restriction promotes longevity via autophagy activation. However, changes to lysosomes underlying this effect remain unclear. Using the nematode Caenorhabditis elegans, we show that induction of autophagic tubular lysosomes, which occurs upon dietary restriction or mTOR inhibition, is a critical event linking reduced food intake to lifespan extension. We find that starvation induces tubular lysosomes not only in affected individuals but also in well-fed descendants, and the presence of gut tubular lysosomes in well-fed progeny is predictive of enhanced lifespan. Furthermore, we demonstrate that expression of Drosophila SVIP, a tubular-lysosome activator in flies, artificially induces tubular lysosomes in well-fed worms and improves C. elegans health in old age. These findings identify tubular lysosomes as a new class of lysosomes that couples starvation to healthy aging.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.465682', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Identifying barriers to gene flow and hierarchical conservation units from seascape genomics: a modelling framework applied to a marine predator', 'authors': \"[{'name': 'Germain Boussarie'}, {'name': 'Paolo Momigliano'}, {'name': 'William D. Robbins'}, {'name': 'Lucas Bonnin'}, {'name': 'Jean-Franois Cornu'}, {'name': 'Ccile Fauvelot'}, {'name': 'Jeremy J. Kiszka'}, {'name': 'Stphanie Manel'}, {'name': 'David Mouillot'}, {'name': 'Laurent Vigliola'}]\", 'abstract': '<p>The ongoing decline of large marine vertebrates must be urgently mitigated, particularly under increasing levels of climate change and other anthropogenic pressures. However, characterizing the connectivity among populations remains one of the greatest challenges for the effective conservation of an increasing number of endangered species. Achieving conservation targets requires an understanding of which seascape features influence dispersal and subsequent genetic structure. This is particularly challenging for adult-disperser species, and when distribution-wide sampling is difficult. Here, we developed a two-step modelling framework to investigate how seascape features drive the genetic connectivity of marine species without larval dispersal, to better guide the design of marine protected area networks and corridors. We applied this framework to the endangered grey reef shark, <italic>Carcharhinus amblyrhynchos</italic>, a reef-associated shark distributed across the tropical Indo-Pacific. In the first step, we developed a seascape genomic approach based on isolation-by-resistance models involving circuit theory applied to 515 shark samples, genotyped for 4,491 nuclear single-nucleotide polymorphisms, to explore which parameters drive their population genetic differentiation. We show that deep oceanic areas act as strong barriers to dispersal, while proximity to habitat facilitates dispersal. In the second step, we predicted the resulting genetic differentiation across the entire distribution range of the species, providing both local and global-scale conservation units for future management guidance. We found that grey reef shark populations are more fragmented than expected for such a mobile species, raising concerns about the resilience of isolated populations under high anthropogenic pressures. We recommend the use of this framework to identify barriers to gene flow and to help in the delineation of conservation units at different scales, together with its integration across multiple species when considering marine spatial planning.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.466298', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'Identification of COVID-19 and COPD common key genes and pathways using a protein-protein interaction approach', 'authors': \"[{'name': 'Thiviya S. Thambiraja'}, {'name': 'Kalimuthu Karuppanan'}, {'name': 'Gunasekaran Subramaniam'}, {'name': 'Suresh Kumar'}]\", 'abstract': '<p>Coronavirus disease (COVID-19) is an extremely contagious and cognitive disease that could cause immense hypoxemia. The rise in critically ill patients in epidemic regions has put enormous pressure on hospitals. There is a need to define extreme COVID- 19 clinical determinants to optimize clinical diagnosis and the management system is strong. Chronic obstructive pulmonary disease (COPD) is linked to a rapidly increasing risk of death rates in population pneumonia. In this research, a network of protein-protein interaction (PPI) was developed using constructed datasets of COVID-19 and COPD genes to define the interrelationship between COVID-19 and COPD, how it affects each other, and the genes that are responsible for the process. The PPI network shows the top 10 common overlapping genes, which include IL10, TLR4, TNF, IL6, CXCL8, IL4, ICAM1, IFNG, TLR2, and IL18. These are the genes that COVID-19 and high-risk COPD patients are known to be expressed. These important genes shared by COVID-19 and COPD are involved in pathways such as malaria, African trypanosomiasis, inflammatory bowel disease, Chagas disease, influenza, and tuberculosis.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466320', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Allele frequency dynamics under sex-biased demography and sex-specific inheritance in a pedigreed population', 'authors': \"[{'name': 'Rose M.H. Driscoll'}, {'name': 'Felix E.G. Beaudry'}, {'name': 'Elissa J Cosgrove'}, {'name': 'Reed Bowman'}, {'name': 'John W Fitzpatrick'}, {'name': 'Stephan J Schoech'}, {'name': 'Nancy Chen'}]\", 'abstract': '<p>Sex-biased demography, including sex-biased survival or migration, can impact allele frequency changes across the genome. In particular, we can expect different patterns of genetic variation on autosomes and sex chromosomes due to sex-specific differences in life histories, as well as differences in effective population size, transmission modes, and the strength and mode of selection. Here, we demonstrate the role that sex differences in life history played in shaping short-term evolutionary dynamics across the genome. We used a 25-year pedigree and genomic dataset from a long-studied population of Florida Scrub-Jays (Aphelocoma coerulescens) to directly characterize the relative roles of sex-biased demography and inheritance in shaping genome-wide allele frequency trajectories. We used gene dropping simulations to estimate individual genetic contributions to future generations and to model drift and immigration on the known pedigree. We quantified differential expected genetic contributions of males and females over time, showing the impact of sex-biased dispersal in a monogamous system. Due to female-biased dispersal, more autosomal variation is introduced by female immigrants. However, due to male-biased transmission, more Z variation is introduced by male immigrants. Finally, we partitioned the proportion of variance in allele frequency change through time due to male and female contributions. Overall, most allele frequency change is due to variance in survival and births. Males and females have similar contributions to autosomal allele frequency change, but males have higher contributions to allele frequency change on the Z chromosome. Our work shows the importance of understanding sex-specific demographic processes in accounting for genome-wide allele frequency change in wild populations.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466428', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Assassination Tango: An NLR/NLR-ID immune receptors pair of rapeseed co-operates inside the nucleus to activate cell death', 'authors': \"[{'name': 'Glykeria Mermigka'}, {'name': 'Argyro Amartolou'}, {'name': 'Adriani Mentzelopoulou'}, {'name': 'Niki Astropekaki'}, {'name': 'Panagiotis F. Sarris'}]\", 'abstract': '<p>Plant immunity relies on cell-surface receptors and intracellular NLR immune receptors. Some plant NLRs carry integrated domains (IDs) that that mimic authentic pathogens effector targets. We report here the identification in Brassica napus of the genetically linked NLR/NLR-ID pair, BnRPR1 and BnRPR2. The NLR-ID carries two ID fusions and their mode of action conforms to the proposed integrated sensor/decoy model. The two NLRs interact and the heterocomplex localizes in the plant-cell nucleus and nucleolus. However, the BnRPRs pair does not operate through a negative regulation as it was previously reported for other NLR-IDs. Cell death is induced only upon co-expression of the two proteins and it is dependent on the helper genes EDS1 and NRG1. Truncations of the IDs of BnRPR1 results in cytoplasmic localization and compromises cell death activation. Expression, using the native promoter in Nicotiana species, led to a conditional cell death induction. However, this was not the case for the transgenic Arabidopsis, where no cell death was observed. In summary, we describe a new pair of NLR-IDs with interesting features in relation to its regulation and the cell death activation.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.22.465535', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Rare modification in the ergosterol biosynthesis pathway leads to amphotericin B resistance in Candida auris clinical isolates', 'authors': \"[{'name': 'Milena Kordalewska'}, {'name': 'Kevin D. Guerrero'}, {'name': 'Timothy D. Mikulski'}, {'name': 'Tony N. Elias'}, {'name': 'Rocio Garcia-Rubio'}, {'name': 'Indira Berro'}, {'name': 'Dianne Gardam'}, {'name': 'Christopher H. Heath'}, {'name': 'Anuradha Chowdhary'}, {'name': 'Nelesh Premapragasan Govender'}, {'name': 'David S Perlin'}]\", 'abstract': '<p>We determined amphotericin B (AmB) susceptibility and sequenced key genes of the ergosterol biosynthesis pathway implicated in AmB resistance (<italic>ERG2, ERG3, ERG6, ERG11</italic>) of 321 clinical isolates of <italic>Candida auris</italic>. In antifungal susceptibility testing, 19 (5.9%) isolates were categorized as AmB-resistant (MIC 2 mg/l). Only one AmB-resistant isolate presented a unique non-wild-type <italic>ERG6</italic> genotype that was confirmed to confer amphotericin B resistance (MIC &gt;32 mg/l) when introduced into a susceptible strain (MIC = 0.5 mg/l).</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466411', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'The synthetic triterpenoids CDDO-TFEA and CDDO-Me, but not CDDO, are potent BACH1 inhibitors', 'authors': \"[{'name': 'Laura Casares'}, {'name': 'Rita Moreno'}, {'name': 'Maureen Higgins'}, {'name': 'Sharada Dayalan Naidu'}, {'name': 'Graham Neill'}, {'name': 'Lena Cassin'}, {'name': 'Anders E Kiib'}, {'name': 'Esben B Svenningsen'}, {'name': 'Tadashi Honda'}, {'name': 'Thomas B Poulsen'}, {'name': 'Albena T Dinkova-Kostova'}, {'name': 'David Olagnier'}, {'name': 'Laureano de la Vega'}]\", 'abstract': '<p>The transcription factor BACH1 is a potential target against a variety of chronic conditions linked to oxidative stress and inflammation, and formation of cancer metastasis. However, only a few BACH1 degraders/inhibitors have been described. BACH1 is a transcriptional repressor of heme oxygenase 1 (HMOX1), which is positively regulated by transcription factor NRF2 and is highly inducible by derivatives of the synthetic oleanane triterpenoid 2-cyano-3,12-dioxooleana-1,9(11)-dien-28-oic acid (CDDO). Most of the therapeutic activities of these compounds are due to their anti-inflammatory and antioxidant properties, which are widely attributed to their ability to activate NRF2. However, with such a broad range of action, these drugs may have other molecular targets that have not been fully identified and could also be of importance for their therapeutic profile. Herein we identified BACH1 as a target of CDDO-derivatives, but not CDDO. While both CDDO and CDDO-derivatives activate NRF2 similarly, only CDDO-derivatives inhibit BACH1, which explains the much higher potency of CDDO-derivatives as HMOX1 inducers compared with unmodified CDDO. Notably, we demonstrate that CDDO-derivatives inhibit BACH1 via a novel mechanism that reduces BACH1 nuclear levels while accumulating its cytoplasmic form. Altogether, our study identifies CDDO-derivatives as dual KEAP1/BACH1 inhibitors, providing a rationale for further therapeutic uses of these drugs</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.22.465436', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'An updated framework to account for inter-individual variability when quantifying phenotypic variation', 'authors': \"[{'name': 'Giacomo Puglielli'}, {'name': 'Carlos P. Carmona'}, {'name': 'Laura Varone'}, {'name': 'Lauri Laanisto'}, {'name': 'Carlo Ricotta'}]\", 'abstract': '<p><list list-type=\"order\"><list-item><p>In trait-based ecology, phenotypic variation (<italic>PVar</italic>) is often quantified with measures that express average differences between populations standardized in the range 0-1. A major problem with these measures is that they disregard the within-population trait variability. In addition, most of these measures cannot be decomposed across scales. This can alter their interpretation, thus limiting their applicability.</p></list-item><list-item><p>To overcome these problems, we propose a new measure, the Phenotypic Dissimilarity Index (PhD) that is insensitive to the within-population interindividual trait variability. Likewise, PhD can be used to quantify <italic>PVar</italic> between individuals in a population while accounting for the <italic>PVar</italic> within individuals.</p></list-item><list-item><p>Using simulated and real data, we showed that PhD index correctly quantifies <italic>PVar</italic> when the within-population trait variability is not negligible, as in many ecological studies. By accounting for within-population trait variability, the PhD index generally provides a more parsimonious quantification of <italic>PVar</italic> across an environmental gradient compared to other estimators.</p></list-item><list-item><p>Traits sampled within a species have an inherent variability. Accounting for such variability is essential to understand species phenotypic responses to environmental cues. As such, the PhD index will provide ecologists with an asset to reliably quantify and compare <italic>PVar</italic> within and between species across environmental gradients at different scales. We also provide an R function to calculate the PhD index.</p></list-item></list></p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.26.466009', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Total parenteral nutrition drives glucose metabolism disorders by modulating gut microbiota and its metabolites', 'authors': \"[{'name': 'Haifeng Sun'}, {'name': 'Peng Wang'}, {'name': 'Gulisudumu Maitiabula'}, {'name': 'Li Zhang'}, {'name': 'Jianbo Yang'}, {'name': 'Yupeng Zhang'}, {'name': 'Xuejin Gao'}, {'name': 'Jieshou Li'}, {'name': 'Bin Xue'}, {'name': 'Chao-Jun Li'}, {'name': 'Xinying Wang'}]\", 'abstract': '<p>The occurrence of glucose metabolism disorders is a potentially fatal complication of total parenteral nutrition (TPN). However, the mechanisms of TPN-associated glucose metabolism disorders remain unclarified. Given that the glucose metabolism was related to gut microbiome and TPN could induce the gut microbiota dysbiosis, we hypothesized that gut microbiota and its metabolites played the important roles in TPN-associated glucose metabolism disorders. By performing a cohort study of 256 type 2 IF patients given PN, we found that H-PN (PN&gt;80%) patients exhibited insulin resistance and a higher risk of complications. Then, TPN and microbiome transfer mice model showed that TPN promoted glucose metabolism disorders by inducing gut microbiota dysbiosis; 16S rRNA sequencing demonstrated that the abundance of <italic>Lactobacillacea</italic>e was decreased in mice model and negatively correlated with HOMA-IR and lipopolysaccharide level in TPN patients. Untargeted metabolomics found that indole-3-acetic acid (IAA) was decreased in TPN mice, and the serum level was also decreased in H-PN patients. Furthermore, GLP-1 secretion regulated by IAA and aryl hydrocarbon receptor was reduced in TPN mice and patients; IAA or liraglutide completely prevented glucose metabolism disorders in TPN mice. In conclusion, TPN drives glucose metabolism disorders by inducing alteration of gut microbiota and its metabolites.</p>', 'posted': '2021-10-27'}, {'doi': '10.1101/2021.10.28.466315', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Coloring inside the lines: genomic architecture and evolution of a widespread color pattern in frogs', 'authors': \"[{'name': 'Sandra Goutte'}, {'name': 'Imtiyaz Hariyani'}, {'name': 'Kole Deroy Utzinger'}, {'name': 'Yann Bourgeois'}, {'name': 'Stephane Boissinot'}]\", 'abstract': '<p>Traits shared among distantly related lineages are indicators of common evolutionary constraints, at the ecological, physiological or molecular level. The vertebral stripe is a color pattern that is widespread across the anuran phylogeny. Despite its prevalence in the order, surprisingly little is known about the genetic basis and evolutionary dynamic of this color pattern. Here we combine histology, genome- and transcriptome-wide analyses with order-scale phylogenetic comparative analyses to investigate this common phenotype. We show that the vertebral stripe has evolved hundreds of times in the evolutionary history of anurans and is selected for in terrestrial habitats. Using the Ethiopian Ptychadena radiation as a model system, we demonstrate that variation at the ASIP gene is responsible for the different vertebral stripe phenotypes. Alleles associated to these phenotypes are younger than the split between closely related Ptychadena species, thus indicating that the vertebral stripe results from parallel evolution within the group. Our findings demonstrate that this widespread color pattern evolves rapidly and recurrently in terrestrial anurans, and therefore constitute an ideal system to study repeated evolution.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465975', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Integration of GWAS and TWAS to elucidate the genetic architecture of natural variation for leaf cuticular conductance in maize', 'authors': \"[{'name': 'Meng Lin'}, {'name': 'Pengfei Qiao'}, {'name': 'Susanne Matschi'}, {'name': 'Miguel Vasquez'}, {'name': 'Guillaume P. Ramstein'}, {'name': 'Richard Bourgault'}, {'name': 'Marc Mohammadi'}, {'name': 'Michael J Scanlon'}, {'name': 'Isabel Molina'}, {'name': 'Laurie G. Smith'}, {'name': 'Michael A Gore'}]\", 'abstract': '<p>The cuticle, a hydrophobic layer of cutin and waxes synthesized by plant epidermal cells, is the major barrier to water loss when stomata are closed. Dissecting the genetic architecture of natural variation for maize leaf cuticular conductance (gc) is important for identifying genes relevant to improving crop productivity in drought-prone environments. To this end, we performed an integrated genome- and transcriptome-wide association study (GWAS/TWAS) to identify candidate genes putatively regulating variation in leaf gc. Of the 22 plausible candidate genes identified, five were predicted to be involved in cuticle precursor biosynthesis and export, two in cell wall modification, nine in intracellular membrane trafficking, and seven in the regulation of cuticle development. A gene encoding an INCREASED SALT TOLERANCE1-LIKE1 (ISTL1) protein putatively involved in intracellular protein and membrane trafficking was identified in GWAS and TWAS as the strongest candidate causal gene. A set of maize nested near-isogenic lines that harbor the ISTL1 genomic region from eight donor parents were evaluated for gc, confirming the association between gc and ISTL1 in a haplotype-based association analysis. The findings of this study provide novel insights into the role of regulatory variants in the development of the maize leaf cuticle, and will ultimately assist breeders to develop drought-tolerant maize for target environments.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.26.21265261', 'tweets': 11, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Safety Monitoring of mRNA Vaccines Administered During the Initial 6 Months of the U.S. COVID-19 Vaccination Program: Reports to Vaccine Adverse Events Reporting System (VAERS) and v-safe', 'authors': \"[{'name': 'Hannah G. Rosenblum'}, {'name': 'Julianne M. Gee'}, {'name': 'Ruiling Liu'}, {'name': 'Paige L. Marquez'}, {'name': 'Bicheng Zhang'}, {'name': 'Penelope Strid'}, {'name': 'Winston E. Abara'}, {'name': 'Michael M. McNeil'}, {'name': 'Tanya R. Myers'}, {'name': 'Anne M. Hause'}, {'name': 'John R. Su'}, {'name': 'Bethany Baer'}, {'name': 'David Menschik'}, {'name': 'Lauri E. Markowitz'}, {'name': 'Tom T. Shimabukuro'}, {'name': 'David K. Shay'}]\", 'abstract': '<sec><title>Background</title><p>In December 2020, two mRNA-based COVID-19 vaccines were authorized for use in the United States. Vaccine safety was monitored using the Vaccine Adverse Event Reporting System (VAERS), a passive surveillance system, and v-safe, an active surveillance system.</p></sec><sec><title>Methods</title><p>VAERS and v-safe data during December 14, 2020June 14, 2021 were analyzed. VAERS reports were categorized as non-serious, serious, or death; reporting rates were calculated. Rates of reported deaths were compared to expected mortality rates by age. Proportions of v-safe participants reporting local and systemic reactions or health impacts the week following doses 1 and 2 were determined.</p></sec><sec><title>Findings</title><p>During the analytic period, 298,792,852 doses of mRNA vaccines were administered in the United States. VAERS processed 340,522 reports; 921% were non-serious; 66%, serious, non-death; and 13%, death. Over half of 7,914,583 v-safe participants self-reported local and systemic reactogenicity, more frequently after dose 2. Injection-site pain, fatigue, and headache were commonly reported during days 07 following vaccination. Reactogenicity was reported most frequently one day after vaccination; most reactions were mild. More reports of being unable to work or do normal activities occurred after dose 2 (321%) than dose 1 (119%); &lt;1% of participants reported seeking medical care after vaccination. Rates of deaths reported to VAERS were lower than expected background rates by age group.</p></sec><sec><title>Interpretation</title><p>Safety data from &gt;298 million doses of mRNA COVID-19 vaccine administered in the first 6 months of the U.S. vaccination program show the majority of reported adverse events were mild and short in duration.</p></sec>', 'posted': '2021-10-27'}, {'doi': '10.1101/2021.10.26.465842', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Somatostatin venom analogs evolved by fish-hunting cone snails: From prey capture behavior to identifying drug leads', 'authors': \"[{'name': 'Iris Bea. L. Ramiro'}, {'name': 'Walden E. Bjrn-Yoshimoto'}, {'name': 'Julita S. Imperial'}, {'name': 'Joanna Gajewiak'}, {'name': 'Maren Watkins'}, {'name': 'Dylan Taylor'}, {'name': 'William Resager'}, {'name': 'Beatrix Ueberheide'}, {'name': 'Hans Bruner-Osborne'}, {'name': 'Frank G. Whitby'}, {'name': 'Christopher P. Hill'}, {'name': 'Laurent F. Martin'}, {'name': 'Amol Patwardhan'}, {'name': 'Gisela P. Concepcion'}, {'name': 'Baldomero M. Olivera'}, {'name': 'Helena Safavi-Hemami'}]\", 'abstract': '<p>Somatostatin (SS) is a peptide hormone with diverse physiological roles. By investigating a deep-water clade of fish-hunting cone snails, we show that predator-prey evolution has generated a diverse set of SS analogs, each optimized to elicit specific systemic physiological effects in prey. The increased metabolic stability, distinct SS receptor activation profiles, and chemical diversity of the venom analogs make them suitable leads for therapeutic application, including pain, cancer and endocrine disorders. Our findings not only establish the existence of SS-like peptides in animal venoms, but also serve as a model for the synergy gained from combining molecular phylogenetics and behavioral observations to optimize the discovery of natural products with biomedical potential.</p><sec><title>One Sentence Summary</title><p>Somatostatin drug design by fish-hunting cone snails</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.465851', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Fungal auxin is a quorum-based modulator of blast disease severity', 'authors': \"[{'name': 'Lihong Dong'}, {'name': 'Qing Shen'}, {'name': 'Cheng-yen Chen'}, {'name': 'Lizheng Shen'}, {'name': 'Fan Yang'}, {'name': 'Naweed I. Naqvi'}, {'name': 'Yi Zhen Deng'}]\", 'abstract': '<p>Auxin is an important phytohormone regulating plant growth and development, and can also be produced by microbial pathogens including the rice-blast fungus <italic>Magnaporthe oryzae</italic>. However, the detailed biosynthesis pathway, biological function(s), and cellular distribution of such fungal auxin in <italic>M. oryzae</italic> remain largely unknown. Here, we report a sequential accumulation of intrinsic auxin in the three conidial cells, the infection structure (appressorium), and the invasive hyphae in <italic>M. oryzae</italic>. Such fungus-derived auxin was also secreted out and perceived by the host plants. A mitochondria-associated Indole-3-pyruvate decarboxylase, Ipd1, is essential for auxin/Indole-3-acetic acid biosynthesis in <italic>M. oryzae</italic>. The <italic>ipd1</italic> was defective in pathogenicity whereas overexpression of <italic>IPD1</italic> led to enhanced virulence in rice. Chemical inhibition of fungal IAA biosynthesis, or its increase via external supplementation decreased or increased the severity of blast disease, respectively, in a dose-dependent manner. Furthermore, the IAA produced and secreted by <italic>M. oryzae</italic> governed the incidence and severity of blast disease in a quorum-dependent manner. Appressorium formation, conidial cell death critical for appressorium function, and the transcription of infection-related genes, <italic>MPG1</italic> and <italic>INV1</italic>, directly correlated with cell density and/or IAA levels within the conidial population at the early stages of pathogenic development. Overall, our study revealed that the severity of blast disease is regulated via quorum sensing with intrinsic IAA serving as an associated signal transducer in rice blast.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.465922', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Computational and neural mechanisms of affected beliefs', 'authors': \"[{'name': 'Laura Mller-Pinzler'}, {'name': 'Nora Czekalla'}, {'name': 'Annalina V Mayer'}, {'name': 'Alexander Schrder'}, {'name': 'David S Stolz'}, {'name': 'Frieder Michel Paulus'}, {'name': 'Sren Krach'}]\", 'abstract': '<p>The feedback people receive on their behavior shapes the process of belief formation and self-efficacy in mastering a given task. The neural and computational mechanisms of how the subjective value of these beliefs and corresponding affect bias the learning process are yet unclear. Here we investigate this question during learning of self-efficacy beliefs using fMRI, pupillometry, computational modeling and individual differences in affective experience. Biases in the formation of self-efficacy beliefs were associated with affect, pupil dilation and neural activity within the anterior insula, amygdala, VTA/SN, and mPFC. Specifically, neural and pupil responses map the valence of the prediction errors in correspondence to the experienced affect and learning bias people show during belief formation. Together with the functional connectivity dynamics of the anterior insula within this network our results hint towards neural and computational mechanisms that integrate affect in the process of belief formation.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.28.466260', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'RACK1 associates with RNA-binding proteins Vigilin and SERBP1 to control dengue virus replication', 'authors': \"[{'name': 'Alexis Brugier'}, {'name': 'Mohamed-Lamine Hafirassou'}, {'name': 'Marie Pourcelot'}, {'name': 'Morgane Baldaccini'}, {'name': 'Laurine Couture'}, {'name': 'Vasiliya Kril'}, {'name': 'Beate Kuemmerer'}, {'name': 'Sarah Gallois-Montbrun'}, {'name': 'Lucie Bonnet-Madin'}, {'name': 'Sebastien Pfeffer'}, {'name': 'Pierre-Olivier Vidalain'}, {'name': 'Constance Delaugerre'}, {'name': 'Laurent Meertens'}, {'name': 'Ali Amara'}]\", 'abstract': '<p>Dengue virus (DENV), a re-emerging virus transmitted by Aedes mosquitoes, causes severe pathogenesis in humans. No effective treatment is available against this virus. We recently identified the scaffold protein RACK1 as a component of the DENV replication complex, a macromolecular complex essential for viral genome amplification. Here, we show that RACK1 is important for DENV infection. RACK1 mediates DENV replication through binding to the 40S ribosomal subunit. Mass spectrometry analysis of RACK1 partners coupled to a loss-of-function screen identified the RNA binding proteins Vigilin and SERBP1 as DENV host dependency factors. Vigilin and SERBP1 interact with DENV viral RNA (vRNA), forming a ternary complex with RACK1 to mediate viral replication. Overall, our results indicate that RACK1 recruits Vigilin and SERBP1, linking the DENV vRNA to the translation machinery for optimal translation and replication.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.463794', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Chemogenetic activation of Locus Coeruleus Noradrenergic Neurons Modulates the Default Mode Network', 'authors': \"[{'name': 'Esteban A. Oyarzabal'}, {'name': 'Liming Hsu'}, {'name': 'Manasmita Das'}, {'name': 'Tzu-Hao Harry Chao'}, {'name': 'Jingheng Zhou'}, {'name': 'Sheng Song'}, {'name': 'Weiting Zhang'}, {'name': 'Kathleen G. Smith'}, {'name': 'Natale R. Sciolino'}, {'name': 'Irina Y. Evsyukova'}, {'name': 'Hong Yuan'}, {'name': 'Sung-Ho Lee'}, {'name': 'Guohong Cui'}, {'name': 'Patricia Jensen'}, {'name': 'Yen-Yu Ian Shih'}]\", 'abstract': '<p>The default mode network (DMN) of the brain is involved in cognition, emotion regulation, impulsivity, and balancing between internally and externally focused states. Dysregulation of DMN has been implicated in several neurological and neuropsychiatric disorders. In this study, we used functional magnetic resonance imaging (fMRI), positron emission tomography (PET), and spectral fiber-photometry to investigate the selective neuromodulatory effect of norepinephrine (NE)-releasing noradrenergic neurons in the locus coeruleus (LC) on DMN in mice. Chemogenetic-induced LC-NE activation decreased cerebral blood volume (CBV) and glucose uptake, while increasing synchronous low frequency power within the frontal cortices of the DMN. Fiber-photometry results corroborated these findings, showing LC-NE activation induces NE release, enhances calcium-weighted neuronal spiking, and reduces CBV in the anterior cingulate cortex. These data suggest the lack of conventional stimulus-evoked coupling between neuronal activity and CBV in the frontal DMN. We also demonstrated that chemogenetic activation of LC-NE neurons strengthen functional connectivity within the frontal DMN, and this effect is causally mediated by reduced modulatory inputs from retrosplenial and hippocampal regions to the association cortices of the DMN.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.465226', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': True, 'title': 'A Potential Novel COVID-19 Vaccine With RBD-HR1/HR2 Hexamer Structure', 'authors': \"[{'name': 'Hongbo Liu'}, {'name': 'Xiang Gao'}, {'name': 'Guoyong Wang'}, {'name': 'Jianjun Zhang'}, {'name': 'Jiajie Zhou'}, {'name': 'Tingting Wei'}, {'name': 'Yu Zhang'}, {'name': 'Yujiao Liu'}, {'name': 'Jinhua Piao'}, {'name': 'Qiulei Zhang'}, {'name': 'Yayuan Wang'}, {'name': 'Xin Ma'}, {'name': 'Xiaoting Zhu'}, {'name': 'Yikun Rao'}, {'name': 'Wenjuan Xia'}, {'name': 'Heng Xie'}, {'name': 'Wei Zhang'}]\", 'abstract': '<p>The COVID-19 pandemic and the continued spreading of the SARS-CoV-2 variants have brought a grave public health consequence and severely devastated the global economy with recessions. Vaccination is considered as one of the most promising and efficient methods to end the COVID-19 pandemic and mitigate the disease conditions if infected. Although a few vaccines have been developed with an unprecedented speed, scientists around the world are continuing pursuing the best possible vaccines with innovations. Comparing to the expensive mRNA vaccines and attenuated/inactivated SARS-CoV-2 vaccines, recombinant protein vaccines have certain advantages, including their safety (non-virus components), potential stronger immunogenicity, broader protection, ease of scaling-up production, reduced cost, etc. In this study, we reported a novel COVID-19 vaccine generated with RBD-HR1/HR2 hexamer that was creatively fused with the RBD domain and heptad repeat 1 (HR1) or heptad repeat 2 (HR2) to form a dumbbell-shaped hexamer to target the spike S1 subunit. The novel hexamer COVID-19 vaccine induced high titers of neutralizing antibody in mouse studies (&gt;100,000), and further experiments also showed that the vaccine also induced an alternative antibody to the HR1 region, which probably alleviated the drop of immunogenicity from the frequent mutations of SARS-CoV-2.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466305', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Differential regulation of cranial and cardiac neural crest by Serum Response Factor', 'authors': \"[{'name': 'Colin J Dinsmore'}, {'name': 'Philippe Soriano'}]\", 'abstract': '<p>Serum response factor (SRF) is an essential transcription factor that influences many cellular processes including cell proliferation, migration, and differentiation. SRF directly regulates and is required for immediate early gene (IEG) and actin cytoskeleton-related gene expression. SRF coordinates these competing transcription programs through discrete sets of cofactors, the Ternary Complex Factors (TCFs) and Myocardin Related Transcription Factors (MRTFs). The relative contribution of these two programs to in vivo SRF activity and mutant phenotypes is not fully understood. To study how SRF utilizes its cofactors during development, we generated a knock-in Srf1I allele in mice harboring point mutations that disrupt SRF-MRTF-DNA complex formation but leave SRF-TCF activity unaffected. Homozygous Srf1I/1I mutants die at E10.5 with notable cardiovascular phenotypes, and neural crest conditional mutants succumb at birth to defects of the cardiac outflow tract but display none of the craniofacial phenotypes associated with complete loss of SRF in that lineage. Our studies further support an important role for MRTF mediating SRF function in cardiac neural crest and suggest new mechanisms by which SRF regulates transcription during development.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.29.466147', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'A hold-and-feed mechanism drives directional DNA loop extrusion by condensin', 'authors': \"[{'name': 'Indra A Shaltiel'}, {'name': 'Sumanjit Datta'}, {'name': 'La Lecomte'}, {'name': 'Markus Hassler'}, {'name': 'Marc Kschonsak'}, {'name': 'Sol Bravo'}, {'name': 'Catherine Stober'}, {'name': 'Sebastian Eustermann'}, {'name': 'Christian H Haering'}]\", 'abstract': \"<p>SMC protein complexes structure genomes by extruding DNA loops, but the molecular mechanism that underlies their activity has remained unknown. We show that the active condensin complex entraps the bases of a DNA loop in two separate chambers. Single-molecule and cryo-electron microscopy provide evidence for a power-stroke movement at the first chamber that feeds DNA into the SMC-kleisin ring upon ATP binding, while the second chamber holds on upstream of the same DNA double helix. Unlocking the strict separation of 'motor' and 'anchor' chambers turns condensin from a one-sided into a bidirectional DNA loop extruder. We conclude that the orientation of two topologically bound DNA segments during the course of the SMC reaction cycle determines the directionality of DNA loop extrusion.</p>\", 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466204', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'The return of the rings: evolutionary role of aromatic residues in liquid-liquid phase separation', 'authors': \"[{'name': 'Wen-Lin Ho'}, {'name': 'Jie-rong Huang'}]\", 'abstract': '<p>Aromatic residues appeared relatively late in the evolution of protein sequences. They stabilize the hydrophobic core of globular proteins and are typically absent from intrinsically disordered regions (IDRs). However, recent advances in protein liquid-liquid phase separation (LLPS) studies have shown that aromatic residues in IDRs often act as important stickers, promoting multivalent interactions and the formation of higher-order oligomers. To reconcile this apparent contradiction, we compared levels of sequence disorder in RNA binding proteins and the human proteome and found that aromatic residues appear more frequently than expected in the IDRs of RNA binding proteins, which are often found to undergo LLPS. Phylogenetic analysis shows that aromatic residues are highly conserved among chordates, highlighting their importance in LLPS-driven functional assembly. These results suggest therefore that aromatic residues have contributed twice to evolution: in stabilizing structured proteins and in the assembly of biomolecular condensates.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.25.21265472', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Prediction of recessive inheritance for missense variants in human disease', 'authors': \"[{'name': 'Ben O. Petrazzini'}, {'name': 'Daniel J. Balick'}, {'name': 'Iain S. Forrest'}, {'name': 'Judy Cho'}, {'name': 'Ghislain Rocheleau'}, {'name': 'Daniel M. Jordan'}, {'name': 'Ron Do'}]\", 'abstract': '<p>The prediction of pathogenic human missense variants has improved in recent years, but a more granular level of variant characterization is required. Further axes of information need to be incorporated in order to advance the genotype-to-phenotype map. Recent efforts have developed mode of inheritance prediction tools; however, these lack robust validation and their discrimination performance does not support clinical utility, with evidence of them being fundamentally insensitive to recessive acting diseases. Here, we present MOI-Pred, a three-way variant-level mode of inheritance prediction tool aimed at recessive identification for missense variants. MOI-Pred shows strong ability to discriminate missense variants causing autosomal recessive disease (area under the receiver operating characteristic (AUROC)=0.99 and sensitivity=0.85) in an external validation set. Additionally, we introduce an electronic health record (EHR)-based validation approach using real-world clinical data and show that our recessive predictions are enriched for recessive associations with human diseases, demonstrating utility of our method. Mode of inheritance predictions - pathogenic for autosomal recessive (AR) disease, pathogenic for autosomal dominant (AD) disease, or benign  for all possible missense variants in the human genome are available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/rondolab/MOI-Pred/\">https://github.com/rondolab/MOI-Pred/</ext-link>.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.21265507', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Phenotypic and genetic factors associated with differential consent to record linkage for prescription history in the Australian Genetics of Depression Study', 'authors': \"[{'name': 'Lina Gomez'}, {'name': 'Santiago Daz-Torres'}, {'name': 'Luca Colodro-Conde'}, {'name': 'Luis M. Garcia-Marin'}, {'name': 'Chloe Yap'}, {'name': 'Enda M. Byrne'}, {'name': 'Loic Yengo'}, {'name': 'Penelope A. Lind'}, {'name': 'Naomi R. Wray'}, {'name': 'Sarah E. Medland'}, {'name': 'Ian B. Hickie'}, {'name': 'Michelle K. Lupton'}, {'name': 'Miguel E. Rentera'}, {'name': 'Nicholas G. Martin'}, {'name': 'Adrian I. Campos'}]\", 'abstract': '<p>Samples can be prone to ascertainment and attrition biases.The Australian Genetics of Depression Study is a large publicly recruited cohort (n=20,689) established to increase the understanding of depression and antidepressant treatment response. As part of the recruitment, participants donated a saliva sample and were given the option to consent to linkage of prescription records for research purposes. This study investigates differences between participants who donated a saliva sample or agreed to linkage of their records compared to those who did not. We observed that older, male participants with a higher education were more likely to donate a saliva sample. Self-reported bipolar disorder, ADHD, panic disorder, PTSD, substance use disorder and social anxiety disorder were associated with lower odds of donating a saliva sample whereas anorexia was associated with higher odds of donation. Male and younger participants showed higher odds of agreeing to record linkage. Participants with higher neuroticism scores and those with a history of bipolar disorder were also more likely to agree to record linkage whereas participants with a diagnosis of anorexia were less likely to agree. Increased likelihood of consent was also associated with increased genetic susceptibility to anorexia and reduced genetic risk for depression, and schizophrenia whereas there was no significant genetic effect for neuroticism. Overall, our results show moderate differences among these subsamples. Most current epidemiological studies do not adjust, nor search, for attrition biases at the genetic level. The possibility to do so is a strength of samples such as the AGDS. Our results suggest that analyses can be made more robust by identifying attrition biases both on the phenotypic and genetic level, and either contextualising them as a potential limitation or performing sensitivity analyses adjusting for them.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.466284', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Spontaneous emergence of topologically robust grid cell modules: A multiscale instability theory', 'authors': \"[{'name': 'Mikail Khona'}, {'name': 'Sarthak Chandra'}, {'name': 'Ila Fiete'}]\", 'abstract': '<p>Modular structures in the brain play a central role in compositionality and intelligence, however the general mechanisms driving module emergence have remained elusive. Studying entorhinal grid cells as paradigmatic examples of modular architecture and function, we demonstrate the spontaneous emergence of a small number of discrete spatial and functional modules from an interplay between continuously varying lateral interactions generated by smooth cortical gradients. We derive a comprehensive analytic theory of modularization, revealing that the process is highly generic with its robustness deriving from topological origins. The theory generates universal predictions for the sequence of grid period ratios, furnishing the most accurate explanation of grid cell data to date. Altogether, this work reveals novel principles by which simple bottom-up dynamical interactions lead to macroscopic modular organization.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466299', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Reproduction, seasonal morphology, and juvenile growth in three Malagasy fruit bats', 'authors': \"[{'name': 'Angelo Andrianiaina'}, {'name': 'Santino Andry'}, {'name': 'Anecia Gentles'}, {'name': 'Sarah Guth'}, {'name': 'Jean-Michel Heraud'}, {'name': 'Hafaliana Christian Ranaivoson'}, {'name': 'Ny Anjara Fifi Ravelomanantsoa'}, {'name': 'Timothy Treuer'}, {'name': 'Cara E Brook'}]\", 'abstract': \"<p>The island nation of Madagascar is home to three endemic species of Old World Fruit Bat in the family Pteropodidae: Pteropus rufus, Eidolon dupreanum, and Rousettus madagascariensis, all three of which are IUCN Red Listed under some category of threat. To inform conservation efforts to model population viability for these threatened species, as well understand the mechanisms underpinning persistence of several potentially zoonotic pathogens hosted by these bats, we here define the seasonal limits of a staggered annual birth pulse across the three species. Our field studies in central-eastern Madagascar indicate that this annual birth pulse takes place in September/October for P. rufus, November for E. dupreanum, and December for R. madagascariensis. Juvenile development periods vary across the three Malagasy pteropodids, resulting in near-synchronous weaning of pups for all species in late January-February at the height of the fruiting season for Madagascar, a pattern characteristic of most mammalian frugivores on the island. We here document the size range in morphological traits for the three Malagasy fruit bat species; these traits span the range of those known for pteropodids more broadly, with P. rufus and E. dupreanum among the larger of recorded species and R. madagascariensis among the smaller. All three species demonstrate subtle sexual dimorphism in observed traits with larger-bodied males vs. females. We explore seasonal variation in adult body condition by comparing observed body mass with body mass predicted by forearm length, demonstrating that pregnant females add weight during staggered gestation periods and males lose weight during the nutritionally-deficit Malagasy winter. Finally, we quantify forearm, tibia, and ear length growth rates in juvenile bats, demonstrating both faster growth and more protracted development times for the largest P. rufus species. The longer development period for the already-threatened P. rufus further jeopardizes this species' conservation status as human hunting of bats for subsistence is particularly detrimental to population viability during reproductive periods. The more extreme seasonal variation in the mass to forearm relationship for P. rufus may also modulate immune function, an important consideration given these bats' roles as reservoir hosts for several high profile viral families known to cause severe disease in humans. Our work highlights the importance of longitudinal field studies in collecting critical data for mammalian conservation efforts and human public health alike.</p>\", 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.465742', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Genomic adaptation in the CAZyome and specialised metabolism of the plant-associated Streptomyces violaceusniger clade', 'authors': \"[{'name': 'D. Gayrard'}, {'name': 'M. Veyssire'}, {'name': 'K. Adam'}, {'name': 'Y. Martinez'}, {'name': 'C. Vandecasteele'}, {'name': 'M. Vidal'}, {'name': 'B. Dumas'}, {'name': 'T. Rey'}]\", 'abstract': '<p>Streptomycetes are Gram-positive actinobacteria largely represented in the plant root microbiota. The genetic determinants involved in the presence of <italic>Streptomyces</italic> in the rhizosphere are largely unknown and can rely on the ability to degrade plant-derived compounds such as cell-wall polysaccharides and on the production of specialised metabolites. To address whether <italic>Streptomyces</italic> strains recruited into root microbiota share genomic specificities related to these two functions, we engaged a comparative genomic analysis using a newly sequenced rhizospheric strain, <italic>Streptomyces</italic> sp. AgN23 and strains from the phylogenetically related <italic>S. violaceusniger</italic> clade. This analysis enlightens a shared prominent CAZyome potentially involved in plant polysaccharides degradation and a strong conservation of antimicrobials biosynthetic clusters (rustmicin, mediomycin, niphimycin, nigericin) as well as plant bioactive compounds (nigericin, echosides, elaiophylin). Taken together, our work supports the hypothesis that specific hydrolytic enzymes and specialised metabolites repertoires may play important roles in the development of <italic>Streptomyces</italic> strains in the rhizosphere.</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.29.466454', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Steroid hormone catabolites activate the pyrin inflammasome through a non-canonical mechanism', 'authors': \"[{'name': 'Flora Magnotti'}, {'name': 'Daria Chirita'}, {'name': 'Sarah Dalmon'}, {'name': 'Amandine Martin'}, {'name': 'Pauline Bronnec'}, {'name': 'Jeremy Sousa'}, {'name': 'olivier Helynck'}, {'name': 'Wonyong Lee'}, {'name': 'Daniel Kastner'}, {'name': 'Jae Jin Chae'}, {'name': 'Michael McDermott'}, {'name': 'alexandre Belot'}, {'name': 'Michel Popoff'}, {'name': 'Pascal Seve'}, {'name': 'Sophie Georgin-Lavialle'}, {'name': 'Helene Munier-Lehmann'}, {'name': 'Tu anh TRAN'}, {'name': 'Ellen De Langhe'}, {'name': 'Carine Wouters'}, {'name': 'Yvan Jamilloux'}, {'name': 'Thomas Henry'}]\", 'abstract': '<p>The pyrin inflammasome acts as a guard of RhoA GTPases and is central to immune defences against RhoA-manipulating pathogens. Pyrin activation proceeds in two steps. Yet, the second step is still poorly understood. Using cells constitutively activated for the pyrin step 1, a chemical screen identified etiocholanolone and pregnanolone, two catabolites of testosterone and progesterone, acting at low concentrations as specific step-2 activators. High concentrations of these metabolites fully and rapidly activated pyrin, in a human-specific, B30.2 domain-dependent manner and without inhibiting RhoA. Mutations in MEFV, encoding pyrin, cause two distinct autoinflammatory diseases (PAAND and FMF). Monocytes from PAAND patients, and to a lower extent from FMF patients, displayed increased responses to these metabolites. This study provides a new perspective on pyrin activation, indicates that endogenous steroid catabolites can drive autoinflammation, through the pyrin inflammasome, and explains the \"steroid fever\" described in the late 1950s, upon steroid injection in humans.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.465762', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'A basement membrane discovery pipeline uncovers network complexity, new regulators, and human disease associations', 'authors': \"[{'name': 'Ranjay Jayadev'}, {'name': 'Mychel RPT Morais'}, {'name': 'Jamie M Ellingford'}, {'name': 'Sandhya Srinivasan'}, {'name': 'Richard W Naylor'}, {'name': 'Craig Lawless'}, {'name': 'Anna S Li'}, {'name': 'Jack F Ingham'}, {'name': 'Eric Hastie'}, {'name': 'Qiuyi Chi'}, {'name': 'Maryline Fresquet'}, {'name': 'Nikki-Maria Koudis'}, {'name': 'Huw B Thomas'}, {'name': 'Raymond T OKeefe'}, {'name': 'Emily Williams'}, {'name': 'Antony Adamson'}, {'name': 'Helen M Stuart'}, {'name': 'Siddharth Banka'}, {'name': 'Damian Smedley'}, {'name': 'David R Sherwood'}, {'name': 'Rachel Lennon'}, {'name': ' '}]\", 'abstract': '<p>Basement membranes (BMs) are ubiquitous extracellular matrices whose composition remains elusive, limiting our understanding of BM regulation and function. By developing a bioinformatic and <italic>in vivo</italic> discovery pipeline, we define a network of 222 human proteins localized to BMs. Network analysis and screening in <italic>C. elegans</italic> and zebrafish identified new BM regulators, including ADAMTS, ROBO, and TGF. More than 100 BM-network genes associate with human phenotypes and by screening 63,039 genomes from families with rare disorders, we discovered loss-of-function variants in <italic>LAMA5</italic>, <italic>MPZL2</italic>, and <italic>MATN2,</italic> and show they regulate BM composition and function. This cross-disciplinary study establishes the immense complexity and role of BMs in human health.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.21265424', 'tweets': 9, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': False, 'title': 'Intrahost-diversity of influenza A virus in upper and lower respiratory tract derived samples from a college community', 'authors': \"[{'name': 'Nicolae Sapoval'}, {'name': 'P. Jacob Bueno de Mesquita'}, {'name': 'Yunxi Liu'}, {'name': 'Roger Wang'}, {'name': 'Tian Rui Liu'}, {'name': 'Josie Garza'}, {'name': 'Torrey Williams'}, {'name': 'Carmelli Cadiz'}, {'name': 'Gene S. Tan'}, {'name': 'Harm Van Bakel'}, {'name': 'R. A. Leo Elworth'}, {'name': 'Donald K. Milton'}, {'name': 'Todd J. Treangen'}]\", 'abstract': '<p>Motivation. Influenza is a rapidly mutating RNA virus responsible for annual epidemics causing substantial morbidity, mortality, and economic loss. Characterizing influenza virus mutational diversity and evolutionary processes within and between human hosts can provide tools to help track and understand transmission events. In this study we investigated possible differences between the intrahost genomic content of influenza virus in upper respiratory swabs and exhaled aerosols thought to be enriched for virus from the lower respiratory tract. \\n\\nResults. We examined the sequences of specimens collected from influenza A virus (IAV) infected college community members from December 2012 through May 2013. We analyzed four types of IAV samples m aerosols (N=38), coarse &gt;5m aerosols (N=27), nasopharyngeal (N=53), and oropharyngeal swabs (N=47)) collected from 42 study participants with 60 sampling instances. Eighteen (42.9%) participants had data from four sample types (nasopharyngeal swab, oropharyngeal swab, coarse aerosol, fine aerosol) included in the analysis, 10 (23.8%) had data from 3 sample types, 10 (23.8%) had data from 2 sample types, and 4 (9.5%) had data from one type of sample included in the analysis. We found that 481 (53.3%) consensus single nucleotide polymorphisms are shared by all sample types and 600 (66.5%) are shared by at least three different sample types. We observed that within a single patient consensus and non-consensus single nucleotide variants are shared across all sample types. Finally, we inferred a phylogenetic tree using consensus sequences and found that samples derived from a single patient are monophyletic. \\n \\nConclusions. Single nucleotide polymorphisms did not differentiate between samples with varying origin along the respiratory tree. We found that signatures of variation in non-consensus intrahost single nucleotide variants are host and sample, but not site-specific. We conclude that the genomic information available does not allow us to discern a transmission route. Future investigation into whether any site-specific mutational signatures emerge over a longer period of infection, for example in immunocompromised hosts, can be interesting from the virus evolution perspective.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.464901', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Are reaching and grasping effector-independent? Similarities and differences in reaching and grasping kinematics between the hand and foot', 'authors': \"[{'name': 'Yuqi Liu'}, {'name': 'James Caracoglia'}, {'name': 'Sriparna Sen'}, {'name': 'Erez Freud'}, {'name': 'Ella Striem-Amit'}]\", 'abstract': '<p>While reaching and grasping are highly prevalent <italic>manual</italic> actions, neuroimaging studies provide evidence that their neural representations may be shared between different body parts, i.e. effectors. If these actions are guided by effector-independent mechanisms, similar kinematics should be observed when the action is performed by the hand or by a cortically remote and less experienced effector, such as the foot. We tested this hypothesis with two characteristic components of action: the initial ballistic stage of reaching, and the preshaping of the digits during grasping based on object size. We examined if these kinematic features reflect effector-independent mechanisms by asking participants to reach toward and to grasp objects of different widths with their hand and foot. First, during both reaching and grasping, the velocity profile up to peak velocity matched between the hand and the foot, indicating a shared ballistic acceleration phase. Secondly, maximum grip aperture and time of maximum grip aperture of grasping increased with object size for both effectors, indicating encoding of object size during transport. Differences between the hand and foot were found in the deceleration phase and time of maximum grip aperture, likely due to biomechanical differences and the participants inexperience with foot actions. These findings provide evidence for effector-independent visuomotor mechanisms of reaching and grasping that generalize across body parts.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.26.465788', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Experience-induced remodeling of the hippocampal post-synaptic proteome and phosphoproteome', 'authors': \"[{'name': 'Seok Heo'}, {'name': 'Taewook Kang'}, {'name': 'Alexei M. Bygrave'}, {'name': 'Martin R. Larsen'}, {'name': 'Richard L. Huganir'}]\", 'abstract': '<p>The post synaptic density (PSD) of excitatory synapses contains a highly organized protein network with thousands of proteins and is key node in the regulation of synaptic plasticity. To gain new mechanistic insight into experience-induced changes in the PSD, we examined the global dynamics of the PSD proteome and phosphoproteome in mice following various treatments. Mice were trained using an inhibitory avoidance (IA) task and hippocampal PSD fractions were isolated for quantitative proteomic and phosphoproteomics analysis. We used a sequential enrichment strategy to explore the concurrent events of protein expression and phosphorylation in the hippocampal PSD following IA training (IA) or immediate shock (Shock). We identified more than 6,200 proteins and 3,000 phosphoproteins in the sequential strategy covering a total of 7,429 proteins. On the phosphoproteins we identified a total of 9,589 phosphosites. Strikingly, of the significantly IA-regulated proteins and phosphoproteins, a large fraction of the proteins displayed an overall decrease in phosphorylation level. Bioinformatic analysis of proteins and phosphoproteins that were regulated by IA were annotated for an involvement in regulation of glutamate receptor functionality, calcium signaling, and synaptic plasticity. We also identified synaptic kinases, phosphatases and their respective phosphosites regulated by IA training or immediate shock. Furthermore, we found that AMPA receptor surface expression was regulated by protein phosphatase, Mg<sup>2+</sup>/Mn<sup>2+</sup> dependent 1H (Ppm1h). Together, these results unravel the dynamic remodeling of the PSD upon IA learning or immediate shock and serve as a resource for elucidating the synaptic proteome dynamics induced by experience-dependent plasticity.</p><sec><title>Highlights</title><list list-type=\"bullet\"><list-item><p>The proteome and phosphoproteome of mouse hippocampal PSD fractions were examined using quantitative phosphoproteomics and bioinformatics following inhibitory avoidance training or non-associative immediate shock.</p></list-item><list-item><p>Approximately 6,200 proteins and 3,000 phosphoproteins were identified and quantified in the hippocampal PSD fractions.</p></list-item><list-item><p>IA mediates widespread decreases in the abundance and phosphorylation of proteins in the hippocampal PSD fraction.</p></list-item><list-item><p>Kinases, phosphatases and their phosphorylation status were dynamically and significantly regulated by IA and immediate shock.</p></list-item><list-item><p>Functional validation shows that the protein phosphatase Ppm1h is linked to the regulation of synaptic plasticity <italic>in vitro</italic> and <italic>in vivo</italic>.</p></list-item></list></sec><sec><title>In Brief</title><p>Quantitative proteomics and phosphoproteomics combined with subcellular protein fractionation and bioinformatic analysis identifies a highly dynamic regulation of synaptic protein phosphorylation at the postsynaptic density following IA training and immediate shock.</p></sec>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.25.21265456', 'tweets': 8, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Trends in social exposure to SARS-Cov-2 in France. Evidence from the national socio-epidemiological cohort  EPICOV', 'authors': \"[{'name': 'Josiane Warszawski'}, {'name': 'Laurence Meyer'}, {'name': 'Jeanna-Eve Franck'}, {'name': 'Delphine Rahib'}, {'name': 'Nathalie Lydi'}, {'name': 'Anne Gosselin'}, {'name': 'Emilie Counil'}, {'name': 'Robin Kreling'}, {'name': 'Sophie Novelli'}, {'name': 'Remy Slama'}, {'name': 'Philippe Raynaud'}, {'name': 'Guillaume Bagein'}, {'name': 'Vianney Costemalle'}, {'name': 'Patrick Sillard'}, {'name': 'Toscane Fourie'}, {'name': 'Xavier de Lamballerie'}, {'name': 'Nathalie Bajos'}, {'name': ' '}]\", 'abstract': '<sec><title>Background</title><p>We aimed to study whether social patterns of exposure to SARS-CoV-2 infection changed in France throughout the year 2020, in light to the easing of social contact restrictions.</p></sec><sec><title>Methods</title><p>A population-based cohort of individuals aged 15 years or over was randomly selected from the national tax register to collect socio-economic data, migration history, and living conditions in May and November 2020. Home self-sampling on dried blood was proposed to a 10% random subsample in May and to all in November. A positive anti-SARS-CoV-2 ELISA IgG result against the virus spike protein (ELISA-S) was the primary outcome. The design, including sampling and post-stratification weights, was taken into account in univariate and multivariate analyses.</p></sec><sec><title>Results</title><p>Of the 134,391 participants in May, 107,759 completed the second questionnaire in November, and respectively 12,114 and 63,524 were tested. The national ELISA-S seroprevalence was 4.5% [95%CI: 4.0%-5.1%] in May and 6.2% [5.9%-6.6%] in November. It increased markedly in 18-24-year-old population from 4.8% to 10.0%, and among second-generation immigrants from outside Europe from 5.9% to 14.4%. This group remained strongly associated with seropositivity in November, after controlling for any contextual or individual variables, with an adjusted OR of 2.1 [1.7-2.7], compared to the majority population. In both periods, seroprevalence remained higher in healthcare professions than in other occupations.</p></sec><sec><title>Conclusion</title><p>The risk of Covid-19 infection increased among young people and second-generation migrants between the first and second epidemic waves, in a context of less strict social restrictions, which seems to have reinforced territorialized socialization among peers.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466210', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Selective inhibition of PfATP6 by artemisinins and identification of new classes of inhibitors after expression in yeast', 'authors': \"[{'name': 'Catherine M. Moore'}, {'name': 'Jigang Wang'}, {'name': 'Qingsong Lin'}, {'name': 'Pedro Ferreira'}, {'name': 'Mitchell A. Avery'}, {'name': 'Khaled Elokely'}, {'name': 'Henry M. Staines'}, {'name': 'Sanjeev Krishna'}]\", 'abstract': '<p>Treatment failures with artemisinin combination therapies (ACTs) threaten global efforts to eradicate malaria. They highlight the importance of identifying drug targets and new inhibitors and of studying how existing antimalarial classes work.</p><p>Herein we report the successful development of an heterologous expression-based compound screening tool. Validated drug target <italic>P. falciparum</italic> calcium ATPase6 (PfATP6) and a mammalian ortholog (SERCA1a) were functionally expressed in yeast providing a robust, sensitive, and specific screening tool. Whole-cell and <italic>in vitro</italic> assays consistently demonstrated inhibition and labelling of PfATP6 by artemisinins. Mutations in PfATP6 resulted in fitness costs that were ameliorated in the presence of artemisinin derivatives when studied in the yeast model.</p><p>As previously hypothesised, PfATP6 is a target of artemisinins. Mammalian SERCA1a can be mutated to become more susceptible to artemisinins. The inexpensive, low technology yeast screening platform has identified unrelated classes of druggable PfATP6 inhibitors. Resistance to artemisinins may depend on mechanisms that can concomitantly address multi-targeting by artemisinins and fitness costs of mutations that reduce artemisinin susceptibility.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.23.464780', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Double-check the zebrafish 18s rRNA qPCR primers: they may be wrong', 'authors': \"[{'name': 'Jianing Wang'}, {'name': 'Zhipeng Yang'}, {'name': 'An Xiao'}]\", 'abstract': '<p>A widely used qPCR primer for zebrafish gene <italic>rna18s</italic> (<italic>18s rRNA</italic>, or <italic>18s</italic>), with the sequence of 5-TCGC<underline><bold>ta</bold></underline>GT<underline><bold>t</bold></underline>GGCATCGTTTA<underline><bold>t</bold></underline>G-3, is found to be incorrect. Initially designed for rainbow trout (<italic>Oncorhynchus mykiss</italic>) <italic>rna18s</italic>, the primer has four different nucleotides from the zebrafish sequence 5-TCGC<underline><bold>GG</bold></underline>GT<underline><bold>C</bold></underline>GGCATCGTTTA<underline><bold>C</bold></underline>G-3 (indicated in bold/underlined, lowercase letters for rainbow trout and uppercase letters for zebrafish). Since its first use in zebrafish in 2006, this mismatched primer has been clearly stated to be used in at least 50 publications and may have affected hundreds or more in publications citing them. For a sensitive, quantitative method as qPCR, this error must be corrected as soon as possible in the zebrafish community by using <italic>rna18s</italic> primer sets with accurate sequences, such as those summarized and newly designed in this article.</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.25.465780', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'TGF signaling is required for sclerotome resegmentation during development of the spinal column in Gallus gallus', 'authors': \"[{'name': 'Sade W. Clayton'}, {'name': 'Ronisha McCardell'}, {'name': 'Rosa Serra'}]\", 'abstract': '<p>We previously showed the importance of TGF signaling in development of the mouse axial skeleton. Here, we provide the first direct evidence that TGF signaling is required for resegmentation of the sclerotome using chick embryos. Lipophilic fluorescent tracers, DiO and DiD, were microinjected into adjacent somites of embryos treated with or without TGFR1 inhibitor, SB431542, at developmental day E2.5 (HH16). Lineage tracing of labeled cells was observed over the course of 4 days until the completion of resegmentation at E6.5 (HH32). Vertebrae were malformed and intervertebral discs were small and misshapen in SB431542 injected embryos. Inhibition of TGF signaling resulted in alterations in resegmentation that ranged between full, partial, and slanted shifts in distribution of DiO or DiD labeled cells within vertebrae. Patterning of rostro- caudal markers within sclerotome was disrupted at E3.5 after treatment with SB431542 with rostral domains expressing both rostral and caudal markers. Hypaxial myofibers were also increased in thickness after treatment with the inhibitor. We propose that TGF signaling regulates rostro-caudal polarity and subsequent resegmentation in sclerotome during spinal column development.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.21.21265156', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Pain Medicine', 'covid': False, 'title': 'Acute Pain Pathways: protocol for a prospective cohort study', 'authors': \"[{'name': 'Molly Moore Jeffery'}, {'name': 'Mitra Ahadpour'}, {'name': 'Summer Allen'}, {'name': 'Richardae Araojo'}, {'name': 'Fernanda Bellolio'}, {'name': 'Nancy Chang'}, {'name': 'Laura Ciaccio'}, {'name': 'Lindsay Emanuel'}, {'name': 'Jonathan Fillmore'}, {'name': 'Gregg H. Gilbert'}, {'name': 'Patricia Koussis'}, {'name': 'Christine Lee'}, {'name': 'Heather Lipkind'}, {'name': 'Celeste Mallama'}, {'name': 'Tamra Meyer'}, {'name': 'Megan Moncur'}, {'name': 'Teryl K. Nuckols'}, {'name': 'Michael A. Pacanowski'}, {'name': 'David B. Page'}, {'name': 'Elektra Papadopoulos'}, {'name': 'Jessica D. Ritchie'}, {'name': 'Joseph S. Ross'}, {'name': 'Nilay D. Shah'}, {'name': 'Mat Soukup'}, {'name': 'Christopher O. St. Clair'}, {'name': 'Stephen Tamang'}, {'name': 'Sam S. Torbati'}, {'name': 'Douglas W. Wallace'}, {'name': 'Yueqin Zhao'}, {'name': 'Rebekah Heckmann'}]\", 'abstract': '<sec><title>Introduction</title><p>Opioid analgesics are often used to treat moderate-to-severe acute non-cancer pain; however, there is little high-quality evidence to guide clinician prescribing. An essential element to developing evidence-based guidelines is a better understanding of pain management and pain control among individuals experiencing acute pain for various common diagnoses.</p></sec><sec><title>Methods and analysis</title><p>This multi-center prospective observational study will recruit 1,550 opioid-nave participants with acute pain seen in diverse clinical settings including primary/urgent care, emergency departments, and dental clinics. Participants will be followed for 6 months with the aid of a patient-centered health data aggregating platform that consolidates data from study questionnaires, electronic health record data on health care services received, prescription fill data from pharmacies, and activity and sleep data from a Fitbit activity tracker. Participants will be enrolled to represent diverse races and ethnicities and pain conditions, as well as geographical diversity. Data analysis will focus on assessing patients patterns of pain and opioid analgesic use, along with other pain treatments; associations between patient and condition characteristics and patient-centered outcomes including resolution of pain, satisfaction with care, and long-term use of opioid analgesics; and descriptive analyses of patient management of leftover opioids.</p></sec><sec><title>Ethics and dissemination</title><p>This study has received approval from IRBs at each site. Results will be made available to participants, funders, the research community, and the public.</p></sec><sec><title>Trial registration number</title><p>NCT04509115</p></sec><sec><title>Strengths and limitations of this study</title><list list-type=\"bullet\"><list-item><p>This study addresses a key knowledge gap by recruiting a large and diverse group of patients with acute pain, following them for 6 months, and prospectively assessing their pain experience and acute pain treatment</p></list-item><list-item><p>Patients will be recruited from multiple treatment settings that are important in the treatment of acute pain, including emergency departments, primary care, dental settings, and surgical settings.</p></list-item><list-item><p>A patient-centered health data aggregating platform reduces the patient burden of participating in the trial by allowing participants to complete questionnaires when it is convenient and by automatically collecting data on physical activity, healthcare service use, and prescription fills</p></list-item><list-item><p>All participants will have been offered an opioid prescription, reducing the ability to compare outcomes across opioid and non-opioid pain medications</p></list-item><list-item><p>Despite a large sample size, there may be limited numbers of specific acute pain diagnoses with a sufficient sample to analyze and compare by condition/diagnosis</p></list-item></list></sec>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.25.21265422', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Limited impact of Delta variants mutations in the effectiveness of neutralization conferred by natural infection or COVID-19 vaccines in a Latino population', 'authors': \"[{'name': 'Carlos A. Sariol'}, {'name': 'Crisanta Serrano-Collazo'}, {'name': 'Edwin J. Ortiz'}, {'name': 'Petraleigh Pantoja'}, {'name': 'Lorna Cruz'}, {'name': 'Teresa Arana'}, {'name': 'Dianne Atehortua'}, {'name': 'Christina Pabon-Carrero'}, {'name': 'Ana M. Espino'}]\", 'abstract': '<p>The SARS-CoV-2 pandemic has impacted public health systems all over the world. The Delta variant seems to possess enhanced transmissibility, but no clear evidence suggests it has increased virulence. Our data shows that pre-exposed individuals had similar neutralizing activity against the authentic COVID-19 strain and the Delta and Epsilon variants. After one vaccine dose, the neutralization capacity expands to all tested variants. Healthy vaccinated individuals showed a limited breadth of neutralization. One vaccine dose induced similar neutralizing antibodies against the Delta compared to the authentic strain. However, even after two doses, this capacity only expanded to the Epsilon variant.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.29.21265248', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Safety and immunogenicity of a high-dose quadrivalent influenza vaccine administered concomitantly with a third dose of the mRNA-1273 SARS-CoV-2 vaccine in adults  65 years of age: a Phase II, open-label study', 'authors': \"[{'name': 'Ruvim Izikson'}, {'name': 'Daniel Brune'}, {'name': 'Jean-Sbastien Bolduc'}, {'name': 'Pierre Bourron'}, {'name': 'Marion Fournier'}, {'name': 'Tamala Mallett Moore'}, {'name': 'Aseem Pandey'}, {'name': 'Lucia Perez'}, {'name': 'Nessryne Sater'}, {'name': 'Anju Shrestha'}, {'name': 'Sophie Wague'}, {'name': 'Sandrine I Samson'}]\", 'abstract': '<p><bold>Background</bold>\\n\\nConcomitant seasonal influenza vaccination with a COVID-19 vaccine booster could help to minimise potential disruption to the seasonal influenza vaccination campaign and maximise protection against both diseases among individuals at risk of severe disease and hospitalisation. This study assesses the safety and immunogenicity of concomitant administration of high-dose quadrivalent influenza vaccine (QIV-HD) and a mRNA-1273 vaccine booster dose in older adults.\\n\\n<bold>Methods</bold>\\n\\nThis is an ongoing Phase II, multi-centre, open-label study (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"clintrialgov\" xlink:href=\"NCT04969276\">NCT04969276</ext-link>). We describe interim results up to 21 days after vaccination (July 2021August 2021). Adults aged  65 years living in the community, who were to have received a second mRNA-1273 dose at least five months previously, were randomised (1:1:1) to concomitant QIV-HD and mRNA-1273 vaccination (Coad), QIV-HD alone, or mRNA-1273 vaccine alone. Unsolicited adverse events (AEs) occurring immediately, solicited local and systemic reactions up to day (D)8, and unsolicited AEs, serious AEs (SAEs), AEs of special interest (AESIs) and medically attended AEs (MAAEs) up to D22 were reported. Haemagglutination inhibition (HAI) antibody responses to influenza A/H1N1, A/H3N2, B/Yamagata and B/Victoria strains and SARS CoV-2 binding antibody responses (SARS-CoV-2 Pre-Spike IgG ELISA) were assessed at D1 and D22.\\n\\n<bold>Findings</bold>\\n\\nOf 306 participants randomised, 296 were included for analysis (Coad, n=100; QIV-HD, n=92; mRNA-1273, n=104). Reactogenicity profiles were similar between the Coad and mRNA-1273 groups, with lower reactogenicity rates in the QIV-HD group (frequency [95% CIs] of solicited injection site reactions: 860% [776921], 913% [842960] and 618% [509719] ; solicited systemic reactions: 800% [708873], 837% [751902] and 494% [387602], respectively). Up to D22, unsolicited AEs were reported for 170% and 144% participants in the Coad and mRNA-1273 groups, respectively, with a lower rate (109%) in the QIV-HD group. Seven MAAEs were reported (Coad, n=3; QIV-HD, n=1; mRNA-1273, n=3). There were no SAEs, AESIs or deaths. HAI antibody geometric mean titres (GMTs) increased from D1 to D22 to similar levels for each influenza strain in the Coad and QIV-HD groups (GMTs [95% confidence interval], range across strains: Coad, 286 [233352] to 429 [350525] ; QIV-HD, 315 [257386] to 471 [378588]). SARS-CoV-2 binding antibody geometric mean concentrations (GMCs) also increased to similar levels in the Coad and mRNA-1273 groups (D22 GMCs [95% confidence interval]: 7634 [64459042] and 7904 [68839077], respectively).\\n\\n<bold>Interpretation</bold>\\n\\nNo safety concerns or immune interference were observed for concomitant administration of QIV-HD with mRNA-1273 booster in adults aged  65 years, supporting co-administration recommendations.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.21265127', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Genome-wide association study of liver fat, iron, and extracellular fluid fraction in the UK Biobank', 'authors': \"[{'name': 'Colm ODushlaine'}, {'name': 'Mary Germino'}, {'name': 'Niek Verweij'}, {'name': 'Jonas B. Nielsen'}, {'name': 'Ashish Yadav'}, {'name': 'Christian Benner'}, {'name': 'Joshua D. Backman'}, {'name': 'Nan Lin'}, {'name': 'Gonalo R. Abecasis'}, {'name': 'Aris Baras'}, {'name': 'Manuel A. Ferreira'}, {'name': 'Luca A. Lotta'}, {'name': 'Johnathon R. Walls'}, {'name': 'Prodromos Parasoglou'}, {'name': 'Jonathan L. Marchini'}]\", 'abstract': '<p>Abdominal magnetic resonance imaging (MRI) represents a non-invasive approach allowing the extraction of clinically informative phenotypes. We developed an automated pipeline to segment liver pixels from abdominal MRI images and apply published models to approximate fat fraction, extracellular fluid fraction and iron content in 40,058 MRIs from the UK Biobank. We then conducted a genome-wide association of these traits using imputed variants (N=37,250 individuals, 11,914,698 variants) and exome sequence data (N=35,274 individuals, 8,287,315 variants). For liver fat we identified 8 novel loci in or near genes MARC1, GCKR, ADH1B, MTTP, TRIB1, GPAM, PNPLA2 and APOH. For liver iron we identified 1 novel locus between the genes ASNSD1 and SLC40A1, an iron transporter involved in hemochromatosis. For extracellular fluid fraction we identified 6 novel loci in or near genes AGMAT, NAT2, MRPL4-S1PR2, FADS1, ABO and HFE, with almost all having prior associations to obesity, liver, iron, or lipid traits.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.21265508', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Six-month sequelae of post-vaccination SARS-CoV-2 infection: a retrospective cohort study of 10,024 breakthrough infections', 'authors': \"[{'name': 'Maxime Taquet'}, {'name': 'Quentin Dercon'}, {'name': 'Paul J Harrison'}]\", 'abstract': '<sec><title>Background</title><p>Vaccination has proven effective against infection with SARS-CoV-2, as well as death and hospitalisation following COVID-19 illness. However, little is known about the effect of vaccination on other acute and post-acute outcomes of COVID-19.</p></sec><sec><title>Methods</title><p>Data were obtained from the TriNetX electronic health records network (over 81 million patients mostly in the USA). Using a retrospective cohort study and time-to-event analysis, we compared the incidences of COVID-19 outcomes between individuals who received a COVID-19 vaccine (approved for use in the USA) at least 2 weeks before SARS-CoV-2 infection and propensity score-matched individuals unvaccinated for COVID-19 but who had received an influenza vaccine. Outcomes were ICD-10 codes representing documented COVID-19 sequelae in the 6 months after a confirmed SARS-CoV-2 infection (recorded between January 1 and August 31, 2021). Associations with the number of vaccine doses (1 vs. 2) and age (&lt; 60 vs.  60 years-old) were assessed.</p></sec><sec><title>Results</title><p>Among 10,024 vaccinated individuals with SARS-CoV-2 infection, 9479 were matched to unvaccinated controls. Receiving at least one COVID-19 vaccine dose was associated with a significantly lower risk of respiratory failure, ICU admission, intubation/ventilation, hypoxaemia, oxygen requirement, hypercoagulopathy/venous thromboembolism, seizures, psychotic disorder, and hair loss (each as composite endpoints with death to account for competing risks; HR 0.70-0.83, Bonferroni-corrected p&lt;.05), but not other outcomes, including long-COVID features, renal disease, mood, anxiety, and sleep disorders. Receiving 2 vaccine doses was associated with lower risks for most outcomes. Associations between prior vaccination and outcomes of SARS-CoV-2 infection were marked in those &lt; 60 years-old, whereas no robust associations were observed in those  60 years-old.</p></sec><sec><title>Conclusions</title><p>COVID-19 vaccination is associated with lower risk of several, but not all, COVID-19 sequelae in those with breakthrough SARS-CoV-2 infection. These benefits of vaccination were clear in younger people but not in the over-60s. The findings may inform service planning, contribute to forecasting public health impacts of vaccination programmes, and highlight the need to identify additional interventions for COVID-19 sequelae.</p></sec><sec><title>Funding</title><p>National Institute for Health Research (NIHR) Oxford Health Biomedical Research Centre.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.465782', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Interaction Between Transcribing RNA Polymerase and Topoisomerase I Prevents R-loop Formation in E. coli', 'authors': \"[{'name': 'Dmitry Sutormin'}, {'name': 'Alina Galivondzhyan'}, {'name': 'Olga Musharova'}, {'name': 'Dmitrii Travin'}, {'name': 'Anastasiya Rusanova'}, {'name': 'Kseniya Obraztsova'}, {'name': 'Sergei Borukhov'}, {'name': 'Konstantin Severinov'}]\", 'abstract': '<p>Bacterial topoisomerase I (TopoI) removes excessive negative supercoiling and is thought to relax DNA molecules during transcription, replication and other processes. Using ChIP-Seq, we show that TopoI of <italic>Escherichia coli</italic> (EcTopoI) is co-localized, genome-wide, with RNA polymerase (RNAP) in transcription units. Treatment with transcription elongation inhibitor rifampicin leads to EcTopoI relocation to promoter regions, where RNAP also accumulates. When a 14 kDa RNAP-binding EcTopoI C-terminal domain (CTD) is overexpressed, co-localization of EcTopoI and RNAP along the transcription units is reduced. Pull-down experiments directly show that the two enzymes interact <italic>in vivo</italic>. Using ChIP-Seq and Topo-Seq, we demonstrate that EcTopoI is enriched and in and upstream (within up to 12-15 Kbs) of highly-active transcription units, indicating that EcTopoI relaxes negative supercoiling generated by transcription. Uncoupling of the RNAP-EcTopoI interaction by either overexpression of EcTopoI CTD or deletion of EcTopoI domains involved in the interaction is toxic for cells and leads to excessive negative plasmid supercoiling. Moreover, the CTD overexpression leads to R-loops accumulation genome-wide, indicating that the RNAP-EcTopoI interaction is required for prevention of R-loops formation.</p><sec><title>Article Highlights</title><list list-type=\"bullet\"><list-item><p>TopoI colocalizes genome-wide and interacts with RNAP in <italic>E. coli</italic></p></list-item><list-item><p>Disruption of the interaction between TopoI and RNAP decreases cells viability, leads to hypernegative DNA supercoiling, and R-loops accumulation</p></list-item><list-item><p>TopoI and DNA gyrase are enriched, respectively, upstream and downstream of transcription units in accordance with twin-domain model of Liu and Wang</p></list-item><list-item><p>TopoI recognizes its cleavage sites through a specific motif and by sensing negative supercoiling</p></list-item></list></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.465098', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Adenovirus--defensin complexes induce NLRP3-associated maturation of human phagocytes via TLR4 engagement', 'authors': \"[{'name': 'Karsten Eichholz'}, {'name': 'Tuan Hiep Tran'}, {'name': 'Coraline Chneau'}, {'name': 'Thi Thu Phuong Tran'}, {'name': 'Ocane Paris'}, {'name': 'Martine Pugniere'}, {'name': 'Eric J Kremer'}]\", 'abstract': '<p>Intramuscular delivery of human adenovirus (HAdV)-based vaccines leads to rapid recruitment of neutrophils, which then release antimicrobial peptides/proteins (AMPs). How these AMPs influence vaccine efficacy over the subsequent 24 hr is poorly understood. In this study, we asked if human neutrophil protein 1 (HNP-1), an -defensin that influences the direct and indirect innate immune responses to a range of pathogens, impacts the innate response of human phagocytes to three HAdV species/types (HAdV-C5, -D26, -B35). We show that HNP-1 binds to the capsids, redirects HAdV-C5, -D26, -B35 to Toll-like receptor 4 (TLR4), which leads to internalization, an NLRP3-mediated inflammasome response, and IL-1 release. Surprisingly, IL-1 release was not associated with notable disruption of plasma membrane integrity. These data further our understanding of HAdV innate immunogenicity and may provide pathways to extend HAdV-based vaccines efficacy.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.465122', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Neural population dynamics underlying evidence accumulation in multiple rat brain regions', 'authors': \"[{'name': 'Brian DePasquale'}, {'name': 'Jonathan W Pillow'}, {'name': 'Carlos Brody'}]\", 'abstract': \"<p>Accumulating evidence in service of sensory decision making is a core cognitive function. However, previous work has focused either on the dynamics of neural activity during decision-making or on models of evidence accumulation governing behavior. We unify these two perspectives by introducing an evidence-accumulation framework that simultaneously describes multi-neuron population spiking activity and dynamic stimulus-driven behavior during sensory decision-making. We apply our method to behavioral choices and neural activity recorded from three brain regions - the posterior parietal cortex (PPC), the frontal orienting fields (FOF), and the anterior-dorsal striatum (ADS) - while rats performed a pulse-based accumulation task. The model accurately captures the relationship between stimuli and neural activity, the coordinated activity of neural populations, and the distribution of animal choices in response to the stimulus. Model fits show strikingly distinct accumulation models expressed within each brain region, and that all differ strongly from the accumulation strategy expressed at the level of choices. In particular, the FOF exhibited a suboptimal 'primacy' strategy, where early sensory evidence was favored. Including neural data in the model led to improved prediction of the moment-by-moment value of accumulated evidence and the intended-and ultimately made-choice of the animal. Our approach offers a window into the neural representation of accumulated evidence and provides a principled framework for incorporating neural responses into accumulation models.</p>\", 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.465224', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': True, 'title': 'An iTSC-derived placental model of SARS-CoV-2 infection unveils ACE2-dependent susceptibility in syncytiotrophoblasts', 'authors': \"[{'name': 'Joseph Chen'}, {'name': 'Jessica A Neil'}, {'name': 'Jia Ping Tan'}, {'name': 'Raj Rudraraju'}, {'name': 'Monika Mohenska'}, {'name': 'Yu BY Sun'}, {'name': 'Guizhi Sun'}, {'name': 'Yichen Zhou'}, {'name': 'Ying Li'}, {'name': 'Damien Drew'}, {'name': 'Phill Pymm'}, {'name': 'Wai-Hong Tham'}, {'name': 'Fernando J Rossello'}, {'name': 'Guiying Nie'}, {'name': 'Xiaodong Liu'}, {'name': 'Kanta Subbarao'}, {'name': 'Jose Maria Polo'}]\", 'abstract': '<p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection causing coronavirus disease 2019 (COVID-19) has caused a global health crisis. The primary  site of infection is in the respiratory tract but the virus has been associated with a variety of complications involving the gastrointestinal and cardiovascular systems. Since the virus affects a variety of tissue types, there has been interest in understanding SARS-CoV-2 infection in early development and the placenta. The expression of ACE2 or TMPRSS2, both genes  critical for viral entry, is present in placental-specific cell types such as extravillous trophoblasts (EVTs) and, especially, syncytiotrophoblasts (STs). The potential of SARS-CoV-2 to infect these placental cells and its effect on placental development and function is still unclear. Furthermore, it is crucial to understand the possible mechanism of vertical transmission of SARS-CoV-2 through the placenta. Here, we developed an in vitro model of SARS-CoV-2 infection of placental cell types using induced trophoblast stem cells (iTSCs). This model allowed us to show that STs but not EVTs are infected. Importantly, infected STs lack the expression of key differentiation genes, lack typically observed differentiated morphology and produce significantly lower human chorionic gonadotropin (HCG) compared to non-infected controls. We also show that an anti-ACE2 antibody prevents SARS-CoV-2 infection and restores normal ST differentiation and function. We highlight the establishment of a platform to study SARS-CoV-2 infection in early placental cell types, which will  facilitate investigation of antiviral therapy to protect the placenta during early pregnancy and development.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.21265588', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Designing an evidence-based Bayesian network for estimating the risk versus benefits of AstraZeneca COVID-19 vaccine', 'authors': \"[{'name': 'Helen J Mayfield'}, {'name': 'Colleen L Lau'}, {'name': 'Jane E Sinclair'}, {'name': 'Samuel J Brown'}, {'name': 'Andrew Baird'}, {'name': 'John Litt'}, {'name': 'Aapeli Vuorinen'}, {'name': 'Kirsty R Short'}, {'name': 'Michael Waller'}, {'name': 'Kerrie Mengersen'}]\", 'abstract': '<p>Uncertainty surrounding the risk of developing and dying from Thrombosis and Thromobocytopenia Syndrome (TTS) associated with the AstraZeneca (AZ) COVID-19 vaccine may contribute to vaccine hesitancy. A model is urgently needed to combine and effectively communicate the existing evidence on the risks versus benefits of the AZ vaccine. We developed a Bayesian network to consolidate the existing evidence on risks and benefits of the AZ vaccine, and parameterised the model using data from a range of empirical studies, government reports, and expert advisory groups. Expert judgement was used to interpret the available evidence and determine the structure of the model, relevant variables, data to be included, and how these data were used to inform the model. \\nThe model can be used as a decision support tool to generate scenarios based on age, sex, virus variant and community transmission rates, making it a useful for individuals, clinicians, and researchers to assess the chances of different health outcomes. Model outputs include the risk of dying from TTS following the AZ COVID-19 vaccine, the risk of dying from COVID-19 or COVID-19-associated atypical severe blood clots under different scenarios. Although the model is focused on Australia, it can be easily adaptable to international settings by re-parameterising it with local data. This paper provides detailed description of the model-building methodology, which can used to expand the scope of the model to include other COVID-19 vaccines, booster doses, comorbidities and other health outcomes (e.g., long COVID) to ensure the model remains relevant in the face of constantly changing discussion on risks versus benefits of COVID-19 vaccination.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466388', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Orb-dependent polyadenylation contributes to PLP expression and centrosome scaffold assembly', 'authors': \"[{'name': 'Junnan Fang'}, {'name': 'Dorothy Lerit'}]\", 'abstract': '<p>As the microtubule-organizing centers (MTOCs) of most cells, centrosomes engineer the bipolar mitotic spindle required for error-free mitosis. <italic>Drosophila</italic> Pericentrin (PCNT)-like protein (PLP) is a key centrosome component that directs formation of a pericentriolar material (PCM) scaffold required for PCM organization and MTOC function. Here, we investigate the post-transcriptional regulation of <italic>plp</italic> mRNA. We identify conserved binding sites for cytoplasmic polyadenylation element binding (CPEB) proteins within the <italic>plp</italic> 3-untranslated region and examine the role of the CPEB ortholog, oo18 RNA-binding protein (Orb), in <italic>plp</italic> mRNA regulation. Our data show Orb biochemically interacts with <italic>plp</italic> mRNA and promotes PLP protein expression. Loss of <italic>orb</italic>, but not <italic>orb2</italic>, diminishes PLP levels in embryonic extracts. Consequently, PLP localization to centrosomes and function in PCM scaffolding is compromised in orb mutant embryos, resulting in genome instability and embryonic lethality. Moreover, we find PLP over-expression can restore centrosome scaffolding and rescue the cell division defects caused by orb depletion. Our data suggest Orb modulates PLP expression at the level of <italic>plp</italic> mRNA polyadenylation and showcases the post-transcriptional regulation of core, conserved centrosomal mRNAs as critical for centrosome function.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465946', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': True, 'title': 'Multi-label topic classification for COVID-19 literature annotation using an ensemble model based on PubMedBERT', 'authors': \"[{'name': 'Shubo Tian'}, {'name': 'Jinfeng Zhang'}]\", 'abstract': '<p>The BioCreative VII Track 5 calls for participants to tackle the multi-label classification task for automated topic annotation of COVID-19 literature. In our participation, we evaluated several deep learning models built on PubMedBERT, a pre-trained language model, with different strategies addressing the challenges of the task. Specifically, multi-instance learning was used to deal with the large variation in the lengths of the articles, and focal loss function was used to address the imbalance in the distribution of different topics. We found that the ensemble model performed the best among all the models we have tested. Test results of our submissions showed that our approach was able to achieve satisfactory performance with an F1 score of 0.9247, which is significantly better than the baseline model (F1 score: 0.8678) and the mean of all the submissions (F1 score: 0.8931).</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.465726', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Population structure in the MHC region', 'authors': \"[{'name': 'Andr Silva Marstica'}, {'name': 'Kelly Nunes'}, {'name': 'Erick C. Castelli'}, {'name': 'Nayane S. B. Silva'}, {'name': 'Bruce S. Weir'}, {'name': 'Jrme Goudet'}, {'name': 'Diogo Meyer'}]\", 'abstract': '<p>In his 1972 The apportionment of human diversity, Lewontin showed that, when averaged over loci, genetic diversity is predominantly attributable to differences among individuals within populations. However, selection on specific genes and genomic regions can alter the apportionment of diversity. We examine genetic diversity at the HLA loci, located within the MHC region. HLA genes code for proteins that are critical to adaptive immunity and are well-documented targets of balancing selection. The SNPs within HLA genes show strong signatures of balancing selection on large timescales and are broadly shared among populations, with low F<sub>ST</sub> values. However, when we analyze haplotypes defined by these SNPs (i.e., which define HLA alleles), we find marked differences in frequencies between geographic regions. These differences are not reflected in the F<sub>ST</sub> values because of the extreme polymorphism at HLA loci, illustrating challenges in interpreting F<sub>ST</sub>. Differences in the frequency of HLA alleles among geographic regions are relevant to bone-marrow transplantation, which requires genetic identity at HLA loci between patient and donor. We explore the case of Brazils bone-marrow registry, where a deficit of enrolled volunteers with African ancestry reduces the chance of finding donors for individuals with an MHC region of African ancestry.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.21265544', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'In vitro characterisation and clinical evaluation of the diagnostic accuracy of a new antigen test for SARS-CoV-2 detection.', 'authors': \"[{'name': 'J.J. Montoya'}, {'name': 'J.M. Rubio'}, {'name': 'Y. Ouahid'}, {'name': 'A. Lopez-Lopez'}, {'name': 'A. Madejon'}, {'name': 'A.I. Gil-Garcia'}, {'name': 'R.J. Hannam'}, {'name': 'H.R.E. Butler'}, {'name': 'P. Castan'}]\", 'abstract': '<p>Background and aims: Quick, user-friendly and sensitive diagnostic tools are the key to controlling the spread of the SARS-CoV-2 pandemic in the new epidemiologic landscape. The aim of this work is to characterise a new Covid-19 antigen test that uses an innovative chromatographic Affimer-based technology designed for the qualitative detection of SARS-CoV-2 antigen. As rapid technology to detect Covid-19, the test was extensively characterised in vitro. Once the analytical parameters of performance were set, the test system was challenged in a test field study. The aim of this study was to evaluate its diagnostic accuracy, as compared by the gold standard RT-PCR and other existing lateral flow tests. The study was approved by MiRNAX corporate review board to ensure i) that the test complied with all the ethical requirements, ii) that the rights of participants were protected, and iii) that donors were fully informed about the likelihood that they would not personally benefit from the research. The tests were completed under the frame of Project SENSORNAS RTC-20176501 in collaboration with MiRNAX Biosens Ltd. and Hospital Carlos III and are currently under submission and review from the Ethics Committee of Universidad Autonoma de Madrid.\\nKeywords: respiratory disease COVID-19; SARS-CoV-2; ELISA; RT-PCR; antigen lateral flow test.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.465965', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Impact of a 7-day homogeneous diet on interpersonal variation in human gut microbiomes and metabolomes', 'authors': \"[{'name': 'Leah Guthrie'}, {'name': 'Sean Spencer'}, {'name': 'Dalia Perelman'}, {'name': 'Will Van Treuren'}, {'name': 'Shuo Han'}, {'name': 'Feiqiao Brian Yu'}, {'name': 'Erica Sonnenburg'}, {'name': 'Michael A Fischbach'}, {'name': 'Timothy W. Meyer'}, {'name': 'Justin Sonnenburg'}]\", 'abstract': '<p>Metabolism of dietary compounds by the gut microbiota generates a vast array of microbiome-dependent metabolites (MDMs), which are highly variable between individuals. The uremic MDMs (uMDMs) phenylacetylglutamine (PAG), p-cresol sulfate (PCS) and indoxyl sulfate (IS) accumulate during renal failure and are associated with poor outcomes. Targeted dietary interventions may reduce toxic MDMs generation; however, it is unclear if interindividual differences in diet or gut microbiome dominantly contribute to MDM variance. Here we use a 7-day homogeneous average American diet to standardize dietary precursor availability in 21 healthy individuals. Notably, the coefficient of variation in three uMDMs of interest, PAG, PCS, and IS (primary outcome), did not significantly decrease. The majority of circulating MDMs maintained variation despite identical diets. These results highlight the highly personalized profile of MDMs and limited contribution of short-term dietary heterogeneity, suggesting that dietary modification may need to be paired with microbial therapies to control MDM profiles.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.21265596', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'A Novel Indicator for COVID-19 Pandemic Assessment and Comparison', 'authors': \"[{'name': 'Kanchan Mukherjee'}]\", 'abstract': '<p>Introduction\\nIn an increasingly globalized world, no country can remain immune to the effect of the COVID-19 pandemic. The pandemic has exposed the need for effective public health surveillance in the interest of global health security. However, current indicators are limited in doing a comparative intercountry assessment and comparison because of variation in testing rates and reporting standards. Hence, this study attempts at addressing this gap.\\n\\nMethods \\nThe study proposes incremental change in cases per testing rate (ICTR) as an indicator for doing cross country comparison of the pandemic progress. The equation for calculating this indicator is explained in this study. This is followed by measuring its strength of association and predictive power for determining change in SARS-CoV-2 cases in five countries (USA, UK, India, Pakistan and Bangladesh).\\n\\nResults and discussion\\nICTR was found to have a significantly higher strength of association and predictive power (than the existing indicator- test positivity rate) for determining change in cases over different time periods. Using ICTR, cross country comparison was done for the five countries for15 months to draw deeper insights into the progress of the pandemic. \\n\\nConclusions\\nThe study finds ICTR to be a suitable indicator for inter country comparison and intra country monitoring of the pandemic, which would be useful for global COVID-19 surveillance. \\n\\nKey words: Incremental analysis, SARS-CoV-2, low-middle income country, high income country, testing rate, test positivity rate, surveillance.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465908', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'HIV-1 envelope glycoproteins proteolytic cleavage protects infected cells from ADCC mediated by plasma from infected individuals', 'authors': \"[{'name': 'Jrmie Prvost'}, {'name': 'Halima Medjahed'}, {'name': 'Dani Vzina'}, {'name': 'Hung-Ching Chen'}, {'name': 'Beatrice H Hahn'}, {'name': 'Amos B. Smith'}, {'name': 'Andrs Finzi'}]\", 'abstract': '<p>The HIV-1 envelope glycoprotein (Env) is synthesized in the endoplasmic reticulum as a trimeric gp160 precursor, which requires proteolytic cleavage by a cellular furin protease to mediate virus-cell fusion. Env is conformationally flexible, but controls its transition from the unbound closed conformation (State 1) to downstream CD4-bound conformations (States 2/3), which are required for fusion. In particular, HIV-1 has evolved several mechanisms that reduce the premature opening of Env which exposes highly conserved epitopes recognized by non-neutralizing antibodies (nnAbs) capable of mediating antibody-dependent cellular cytotoxicity (ADCC). Env cleavage decreases its conformational transitions favoring the adoption of the closed conformation. Here we altered the gp160 furin cleavage site to impair Env cleavage and to examine its impact on ADCC responses mediated by plasma from HIV-1-infected individuals. We found that infected primary CD4+ T cells expressing uncleaved, but not wildtype, Env are efficiently recognized by nnAbs and become highly susceptible to ADCC responses mediated by plasma from HIV-1-infected individuals. Thus, HIV-1 limits the exposure of uncleaved Env at the surface of HIV-1-infected cells at least in part to escape ADCC responses.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.466002', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': True, 'title': 'Whole-genome sequencing of Vero E6 (C1008) and comparative analysis of four Vero cell sublines', 'authors': \"[{'name': 'Kazuhiro Konishi'}, {'name': 'Toshiyuki Yamaji'}, {'name': 'Chisato Sakuma'}, {'name': 'Fumio Kasai'}, {'name': 'Toshinori Endo'}, {'name': 'Arihiro Kohara'}, {'name': 'Kentaro Hanada'}, {'name': 'Naoki Osada'}]\", 'abstract': '<p>The Vero cell line is an immortalized cell line established from kidney epithelial cells of the African green monkey. A variety of sublines have been established from the original cell line, which display different characteristics. In this study, we determined the whole-genome sequence of Vero E6 (C1008) and performed comparative analysis among Vero JCRB 0111, Vero CCL-81, Vero 76 and Vero E6. Analysis of the copy number changes and loss of heterozygosity revealed that all sublines share a large deletion and loss of heterozygosity on chromosome 12, which harbors type I interferon and CDKN2 gene clusters. We identified a substantial number of genetic differences among the sublines including single nucleotide variants, indels, and copy number variations. The spectrum of single nucleotide variants indicated a close genetic relationship between Vero JCRB0111 and Vero CCL-81, and between Vero 76 and Vero E6, and a considerable genetic gap between the former two and the latter two lines. In contrast, we confirmed the pattern of genomic integration sites of simian endogenous retroviral sequences, which was consistent among the sublines. We identified subline-specific/enriched loss of function and missense variants, which potentially contribute to the differences in response to viral infection among the Vero sublines. In particular, we focused on Vero E6-specific/enriched variants and identified four genes (IL1RAP, TRIM25, RB1CC1, and ATG2A) that contained missense variants specific or enriched in Vero E6. In addition, we found that V739I variants of ACE2, which functions as the receptor for SARS-CoV-2, were heterozygous in Vero JCRB0111, Vero CCL-81, and Vero 76; however, Vero E6 contained the allele with isoleucine, resulting from the loss of one of the X chromosomes.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.465996', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Camel nanobodies broadly neutralize SARS-CoV-2 variants', 'authors': \"[{'name': 'Jessica Hong'}, {'name': 'Hyung Joon Kwon'}, {'name': 'Raul Cachau'}, {'name': 'Catherine Z Chen'}, {'name': 'Kevin John Butay'}, {'name': 'Zhijian Duan'}, {'name': 'Dan Li'}, {'name': 'Hua Ren'}, {'name': 'Tianyuzhou Liang'}, {'name': 'Jianghai Zhu'}, {'name': 'Venkata P Dandey'}, {'name': 'Negin P. Martin'}, {'name': 'Dominic Esposito'}, {'name': 'Uriel Ortega-Rodriguez'}, {'name': 'Miao Xu'}, {'name': 'Mario J. Borgnia'}, {'name': 'Hang Xie'}, {'name': 'Mitchell Ho'}]\", 'abstract': '<p>With the emergence of SARS-CoV-2 variants, there is urgent need to develop broadly neutralizing antibodies. Here, we isolate two VHH nanobodies (7A3 and 8A2) from dromedary camels by phage display, which have high affinity for the receptor-binding domain (RBD) and broad neutralization activities against SARS-CoV-2 and its emerging variants. Cryo-EM complex structures reveal that 8A2 binds the RBD in its up mode and 7A3 inhibits receptor binding by uniquely targeting a highly conserved and deeply buried site in the spike regardless of the RBD conformational state. 7A3 at a dose of 5 mg/kg efficiently protects K18-hACE2 transgenic mice from the lethal challenge of B.1.351 or B.1.617.2, suggesting that the nanobody has promising therapeutic potentials to curb the COVID-19 surge with emerging SARS-CoV-2 variants.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.466067', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': True, 'title': 'Novel pectin from crude polysaccharide of Syzygium aromaticum against SARS-CoV-2 activities by targeting 3CLpro', 'authors': \"[{'name': 'Can Jin'}, {'name': 'Bo Feng'}, {'name': 'Rong juan Pei'}, {'name': 'Ya qi Ding'}, {'name': 'Mei xia Li'}, {'name': 'Xia Chen'}, {'name': 'Zhen yun Du'}, {'name': 'Yang xiao Ding'}, {'name': 'Chun fan Huang'}, {'name': 'Bo Zhang'}, {'name': 'Xin wen Chen'}, {'name': 'Yi Zang'}, {'name': 'Jia Li'}, {'name': 'Kan Ding'}]\", 'abstract': '<p>To date, COVID-19 is still a severe threat to public health, hence specific effective therapeutic drugs development against SARS-CoV-2 is urgent needed. 3CLpro and PLpro and RdRp are the enzymes required for the SARS-CoV-2 RNA synthesis. Therefore, binding to the enzyme may interfere the enzyme function. Before, we found that sulfated polysaccharide binding to 3CLpro might block the virus replication. Hence, we hypothesize that negative charged pectin glycan may also impede the virus replication. Here we show that 922 crude polysaccharide from Syzygium aromaticum may near completely block SARS-CoV-2 replication. The inhibition rate was 99.9% (EC50 : 0.90 muM). Interestingly, 922 can associates with 3CLpro, PLpro and RdRp. We further show that the homogeneous glycan 922211 from 922 may specifically attenuate 3CL protease activity. The IC50s of 922 and 922211 against 3CLpro are 4.73 plusmn 1.05 muM and 0.18 plusmn 0.01 muM, respectively. Monosaccharide composition analysis reveals that 922211 with molecular weight of 78.7 kDa is composed of rhamnose, galacturonic acid, galactose and arabinose in the molar ratio of 8.21 : 37.81 : 3.58 : 4.49. The structure characterization demonstrated that 922211 is a homogalacturonan linked to RG-I pectin polysaccharide. The linear homogalacturonan part in the backbone may be partly methyl esterified while RG-I type part bearing 1, 4-linked alpha-GalpA, 1, 4-linked alpha-GalpAOMe and 1, 2, 4-linked alpha-Rhap. There are four branches attached to C-1 or C4 position of Rhamnose glycosyl residues on the backbone. The branches are composed of 1, 3-linked beta-Galp, terminal (T)-linked beta-Galp, 1, 5-linked alpha-Araf, T-linked alpha-Araf, 4-linked alpha-GalpA and/or 4-linked beta-GalpA. The above results suggest that 922 and 922211 might be a potential novel leading compound for anti-SARS-CoV-2 new drug development.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.466118', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'How far to build it before they come? Analyzing the use of the Field of Dreams hypothesis to bull kelp restoration', 'authors': \"[{'name': 'Jorge Arroyo-Esquivel'}, {'name': 'Marissa L. Baskett'}, {'name': 'Meredith McPherson'}, {'name': 'Alan Hastings'}]\", 'abstract': '<p>In restoration ecology, the Field of Dreams Hypothesis posits that restoration efforts that create a suitable environment could lead to eventual recovery of the remaining aspects of the ecosystem through natural processes. Natural processes following partial restoration has lead to ecosystem recovery in both terrestrial and aquatic systems. However, understanding the efficacy of a field of dreams approach requires comparison of different approaches to partial restoration in terms of spatial, temporal, and ecological scale to what would happen with more comprehensive restoration efforts. We explore the relative effect of partial restoration and ongoing recovery on restoration efficacy with a dynamical model based on temperate rocky reefs in Northern California. We analyze our model for both the ability and rate of bull kelp forest recovery under different restoration strategies. We compare the efficacy of a partial restoration approach with a more comprehensive restoration effort by exploring how kelp recovery likelihood and rate change with varying intensities of urchin removal and kelp outplanting over different time periods and spatial scales. We find that, for the case of bull kelp forests, setting more favorable initial conditions for kelp recovery through implementing both urchin harvesting and kelp outplanting at the start of the restoration project has a bigger impact on the kelp recovery rate than applying restoration efforts through a longer period of time. Therefore partial restoration efforts, in terms of spatial and temporal scale, can be significantly more effective when applied across multiple ecological scales in terms of both the capacity and rate of achieving the target outcomes.</p><sec><title>Open Research Statement</title><p>The data used in this work is presented in McPherson et al. (2021). The code used to run the simulations can be found in <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/jarroyoe/kelp-restoration-modeling\">https://github.com/jarroyoe/kelp-restoration-modeling</ext-link></p></sec>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.26.466019', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': True, 'title': 'Analyzing SARS CoV-2 Patient Data Using Quantum Supervised Machine Learning', 'authors': \"[{'name': 'Zara Yu'}]\", 'abstract': '<p>The novel coronavirus disease 2019 (COVID-19) has created a serious threat to global health. We developed a new quantum machine learning (QML) assisted diagnostic method that can provide an accurate diagnosis to aid decision processes of medical providers. One of the key elements in our method was to implement the quantum variational method to efficiently classify data, taking crucial multiple correlations among the features into account. We established and fine-tuned this quantum classifier by using a group of data drawn from publicly available COVID-19 cases. We have shown that QML is capable of processing patient information efficiently and accurately for the diagnosis of COVID-19.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466469', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'A Notch-dependent transcriptional mechanism controls expression of temporal patterning factors in Drosophila medulla', 'authors': \"[{'name': 'Alokananda Ray'}, {'name': 'Xin Li'}]\", 'abstract': '<p>Temporal patterning is an important mechanism for generating a great diversity of neuron subtypes from a seemingly homogenous progenitor pool in both vertebrates and invertebrates. Drosophila neuroblasts have been shown to be temporally patterned by sequentially expressed Temporal Transcription Factors (TTFs). These TTFs are proposed to form a transcriptional cascade based on mutant phenotypes, although direct transcriptional regulation between TTFs has not been verified in most cases.  Furthermore, it is not known how the temporal transitions are coupled with generation of the appropriate number of neurons at each stage. We use neuroblasts of the Drosophila optic lobe medulla to address these questions, and show that the expression of TTFs Sloppy-paired 1/ 2 (Slp1/2) is regulated at transcriptional level directly by two other TTFs and the cell-cycle dependent Notch signaling through two cis-regulatory elements.   We also show that supplying transcriptional active Notch can rescue the delayed transition into the Slp stage in cell cycle arrested neuroblasts. Our findings reveal how an interplay between temporal patterning, the neuroblast cell cycle and key signaling pathways such as Notch achieves precise regulation of patterning transcription factor gene expression that is characteristic of these programs.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.466120', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Risk-utility tradeoff shapes memory strategies for evolving patterns', 'authors': \"[{'name': 'Oskar H Schnaack'}, {'name': 'Luca Peliti'}, {'name': 'Armita Nourmohammad'}]\", 'abstract': '<p>Keeping a memory of evolving stimuli is ubiquitous in biology, an example of which is immune memory for evolving pathogens. However, learning and memory storage for dynamic patterns still pose challenges in machine learning. Here, we introduce an analytical energy-based framework to address this problem. By accounting for the tradeoff between utility in keeping a high-affinity memory and the risk in forgetting some of the diverse stimuli, we show that a moderate tolerance for risk enables a repertoire to robustly classify evolving patterns, without much fine tuning. Our approach offers a general guideline for learning and memory storage in systems interacting with diverse and evolving signals.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.28.21265598', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Health Informatics', 'covid': True, 'title': 'Personalized survival probabilities for SARS-CoV-2 positive patients by explainable machine learning', 'authors': \"[{'name': 'Adrian G. Zucco'}, {'name': 'Rudi Agius'}, {'name': 'Rebecka Svanberg'}, {'name': 'Kasper S. Moestrup'}, {'name': 'Ramtin Z. Marandi'}, {'name': 'Cameron Ross MacPherson'}, {'name': 'Jens Lundgren'}, {'name': 'Sisse R. Ostrowski'}, {'name': 'Carsten U. Niemann'}]\", 'abstract': '<p>Interpretable risk assessment of SARS-CoV-2 positive patients can aid clinicians to implement precision medicine. Here we trained a machine learning model to predict mortality within 12 weeks of a first positive SARS-CoV-2 test. By leveraging data on 33,928 confirmed SARS-CoV-2 cases in eastern Denmark, we considered 2,723 variables extracted from electronic health records (EHR) including demographics, diagnoses, medications, laboratory test results and vital parameters. A discrete-time framework for survival modelling enabled us to predict personalized survival curves and explain individual risk factors. Performances of weighted concordance index 0.95 and precision-recall area under the curve 0.71 were measured on the test set. Age, sex, number of medications, previous hospitalizations and lymphocyte counts were identified as top mortality risk factors. Our explainable survival model developed on EHR data also revealed temporal dynamics of the 22 selected risk factors. Upon further validation, this model may allow direct reporting of personalized survival probabilities in routine care.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.21265504', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Global diversity of policy, coverage, and demand of COVID-19 vaccines: a descriptive study', 'authors': \"[{'name': 'Zhiyuan Chen'}, {'name': 'Wen Zheng'}, {'name': 'Qianhui Wu'}, {'name': 'Xinghui Chen'}, {'name': 'Cheng Peng'}, {'name': 'Yuyang Tian'}, {'name': 'Ruijia Sun'}, {'name': 'Minghan Wang'}, {'name': 'Xiaoyu Zhou'}, {'name': 'Zeyao Zhao'}, {'name': 'Guangjie Zhong'}, {'name': 'Xuemei Yan'}, {'name': 'Nuolan Liu'}, {'name': 'Feiran Hao'}, {'name': 'Sihong Zhao'}, {'name': 'Tingyu Zhuang'}, {'name': 'Juan Yang'}, {'name': 'Andrew S. Azman'}, {'name': 'Hongjie Yu'}]\", 'abstract': '<sec><title>Background</title><p>Hundreds of millions of doses of COVID-19 vaccines have been administered globally, but progress in vaccination varies considerably between countries. We aim to provide an overall picture of COVID-19 vaccination campaigns, including policy, coverage, and demand of COVID-19 vaccines.</p></sec><sec><title>Methods</title><p>We conducted a descriptive study of vaccination policy and doses administered data obtained from multiple public sources as of 23 October 2021. We used these data to develop coverage indicators and explore associations of vaccine coverage with socioeconomic and healthcare-related factors. We estimated vaccine demand as numbers of doses required to complete vaccination of countries target populations according to their national immunization program policies.</p></sec><sec><title>Findings</title><p>Use of both mRNA and adenovirus vectored vaccines was the most commonly used COVID-19 vaccines formulary in high-income countries, while adenovirus vectored vaccines were the most widely used vaccines worldwide (176 countries). Almost all countries (98.3%, 173/176) have authorized vaccines for the general public, with 53.4% (94/176) targeting individuals over 12 years and 33.0% (58/176) targeting those 18 years. Forty-one and sixty-seven countries have started additional-dose and booster-dose vaccination programs, respectively. Globally, there have been 116.5 doses administered per 100 target population, although with marked inter-region and inter-country heterogeneity. Completed vaccination series coverage ranged from 0% to more than 95.0% of country target populations, and numbers of doses administered ranged from 0 to 239.6 per 100 target population. Doses administered per 100 total population correlated with healthcare access and quality index (R<sup>2</sup> = 0.58), socio-demographic index (R<sup>2</sup> = 0.56), and GDP per capita (R<sup>2</sup> = 0.65). At least 5.54 billion doses will be required to complete interim vaccination programs  4.65 billion for primary immunization and 0.89 billion for additional/booster programs. Globally, 0.84 and 0.96 dose per individual in the target population are needed for primary immunization and additional/booster programs, respectively.</p></sec><sec><title>Interpretation</title><p>There is wide country-level disparity and inequity in COVID-19 vaccines rollout, suggesting large gaps in immunity, especially in low-income countries.</p></sec><sec><title>Funding</title><p>Key Program of the National Natural Science Foundation of China, the US National Institutes of Health.</p></sec><sec><title>Research in context</title><sec><title>Evidence before this study</title><p>We searched PubMed for articles in any language published up to October 21, 2021, using the following search terms: (COVID-19 OR SARS-CoV-2) AND (vaccination OR vaccine) AND (inequalit* OR inequity OR disparit* OR heterogeneity). We also searched for dashboards associated with vaccine rollout from public websites. We identified several studies on tracking global inequalities of vaccine access, one of which constructed a COVID-19 vaccine dashboard (Our World in Data), and another that explored disparities in COVID-19 vaccination among different-income countries. However, we found no studies that depict global COVID-19 vaccination policies country-by-country and estimate demand for vaccine necessary to completely vaccinate countries designated target populations.</p></sec><sec><title>Added value of this study</title><p>To our knowledge, our study provides the most recent picture of COVID-19 vaccination campaigns, focusing on global vaccination policy and target-population demand. We found a diverse portfolio of vaccines in five technical platforms being administered globally, with 173 countries having authorized administration of vaccines to the general public in various age groups. We observed inter-region and inter-country heterogeneity in one-or-more-dose and full-dose coverage; countries with higher socio-demographic or health resource-related levels had higher coverage. We estimated dose-level demand for completing primary immunization programs and additional/booster dose programs separately.</p></sec><sec><title>Implications of all the available evidence</title><p>Worldwide disparity and inequity of vaccine rollout implies that susceptibility among unvaccinated populations in some countries may impede or reverse pandemic control, especially in face of the emergence of variants and the dilemma of waning antibodies. Our findings suggest that global-level responses to the pandemic - financially, politically, and technically - are needed to overcome complex challenges that lie ahead.</p></sec></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.465873', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'LILRB1 and LILRB2 expression in peripheral blood immune cells at 18 and 24 months of age in infants born from mothers with placental malaria', 'authors': \"[{'name': 'Celia Dechavanne'}, {'name': 'Odilon Nouatin'}, {'name': 'Rafiou Adamou'}, {'name': 'Sofie Edslev'}, {'name': 'Anita Hansen'}, {'name': 'Florian Meurisse'}, {'name': 'Ibrahim Sadissou'}, {'name': 'Erasme Gbaguidi'}, {'name': 'Jacqueline Milet'}, {'name': 'Gilles Cottrell'}, {'name': 'Laure Gineau'}, {'name': 'Audrey Sabbagh'}, {'name': 'Achille Massougbodji'}, {'name': 'Kabirou Moutairou'}, {'name': 'Eduardo A. Donadi'}, {'name': 'Edgardo Carosella'}, {'name': 'Philippe Moreau'}, {'name': 'Ed Remarque'}, {'name': 'Michael Theisen'}, {'name': 'Nathalie Rouas-Freiss'}, {'name': 'Andr Garcia'}, {'name': 'Benoit Favier'}, {'name': 'David Courtin'}]\", 'abstract': '<sec><title>Background</title><p>Placental malaria (PM) is associated with a higher susceptibility of infants to <italic>Plasmodium falciparum (Pf)</italic> malaria. A hypothesis of immune tolerance has been suggested but no clear explanation has been provided so far. Our goal was to investigate the involvement of inhibitory receptors LILRB1 and LILRB2, known to drive immune evasion upon ligation with pathogen and/or host ligands, in PM-induced immune tolerance.</p></sec><sec><title>Methods</title><p>Infants of mothers with or without PM were enrolled in Allada, southern Benin, and followed-up for 24 months. Antibodies with specificity for five blood stage parasite antigens were quantified by ELISA, and the frequency of immune cell subsets was quantified by flow cytometry. LILRB1 or LILRB2 expression was assessed on cells collected at 18 and 24 months of age.</p></sec><sec><title>Results</title><p>Infants born to PM-mothers had a higher risk of developing clinical malaria than those born to mothers without PM (IRR=1.53, p=0.040), and such infants displayed a lower frequency of non-classical monocytes (OR=0.74, p=0.01) that overexpressed LILRB2 (OR=1.36, p=0.002). Moreover, infants born to PM-mothers had lower levels of cytophilic IgG and higher levels of IL-10 during active infection.</p></sec><sec><title>Conclusion</title><p>Modulation of IgG and IL-10 levels could impair monocyte functions (opsonisation/phagocytosis) in infants born to PM-mothers, possibly contributing to their higher susceptibility to malaria. The long-lasting effect of PM on infants monocytes was notable, raising questions about the capacity of ligands such as Rifins or HLA-I molecules to bind to LILRB1 and LILRB2 and to modulate immune responses, and about the reprogramming of neonatal monocytes/macrophages.</p></sec><sec><title>KEY POINTS</title><p>Infants of mothers with placental malaria were more susceptible to clinical malaria than those born to mothers without placental malaria and they displayed a lower frequency of non-classical monocytes that overexpressed LILRB2.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.466359', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Evolution of maternal and early zygotic transcript regulation across Drosophila', 'authors': \"[{'name': 'Charles Omura'}, {'name': 'Susan Lott'}]\", 'abstract': '<p>The complements of mRNAs in early embryonic development are crucial for setting up developmental trajectories in animals. The earliest stages of development are regulated by mRNAs deposited into the egg by the mother, until the zygote can become competent to transcribe its own genome. Previously, we showed that the set of maternally deposited and early transcribed zygotic mRNAs in Drosophila are generally conserved across species, but with some notable variation. We also showed that a majority of regulators of these two types of transcripts are shared. In this study, we examine the differences in regulatory motifs associated with maternal deposition and early zygotic transcription across species of Drosophila. For maternal transcripts, while the regulators are mostly conserved, we find the Drosophila pseudoobscura species subgroup appears to contain numerous novel regulatory motifs unique to these species. These novel motifs are enriched in transposable elements exclusive to this group. As this species group had been previously identified as having the largest divergence in early embryonic transcripts given their divergence time, this change in regulation may be responsible. However, transcripts that are present at the maternal stage only in these species are equally enriched in novel (group-specific) and conserved binding sites, so the novel regulation is not the sole cause of regulatory divergence in these species. At the zygotic stage, we observe a wide variety of species-specific motifs. Additionally, at both stages we observe motifs conserved across species having different effects on gene expression in different species, and regulating different sets of genes in different species. By examining changes in motif content across species, we find that changes in motif content alone is generally insufficient to drive gene expression changes across species.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.465752', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'SCARECROW maintains the stem cell niche in Arabidopsis root by ensuring telomere integrity', 'authors': \"[{'name': 'Bingxin Wang'}, {'name': 'Xiaowen Shi'}, {'name': 'Jingbo Gao'}, {'name': 'Rui Liao'}, {'name': 'Jing Fu'}, {'name': 'Juan Bai'}, {'name': 'Hongchang Cui'}]\", 'abstract': '<p>Stem cells are the ultimate source of the cells in various tissues and organs, and thus are essential to postembryonic plant growth and development. SCARECROW (SCR) is a plant-specific transcription regulator well known for its role in stem-cell renewal in plant roots, but the mechanism by which SCR exerts this function is still unclear. To address this question, we carried out a genetic screen for mutants that no longer express SCR in the stem-cell niche in the Arabidopsis root, and one of the mutants is characterized herein. Using marker-assisted mapping, whole genome sequencing, and complementation tests, we pinpointed the causal mutation in this mutant at <italic>TEN1</italic>, which encodes telomere-end protecting factor. By sequence alignment of TEN1 homologs in a wide range of eukaryotes, we identified two novel motifs. The importance of these motifs was examined through site-directed mutagenesis of a conserved amino acid, and through complementation tests, which showed that G100 is required for TEN1 stability. Interestingly, we found that <italic>TEN1</italic> expression was dramatically reduced in the <italic>scr</italic> mutant. Two components in the same protein complex as TEN1, STN1 and CTC1, were found to be also dramatically downregulated in <italic>scr</italic>, as well as telomerase. Further studies showed that loss of STN1, CTC1 and telomerase also caused defects in the root stem cells. In line with these findings, the <italic>scr</italic> mutant was hypersensitive to DNA damage reagents such as Zeocin. These results together suggest that SCR maintains root stem cells by promoting expression of genes that ensures genome integrity.</p><sec><title>One sentence summary</title><p>This study reveals a role for the root development regulator SCARECROW in maintaining the expression of telomere-end protecting factors and a connection between genome integrity and stem-cell maintenance in Arabidopsis root.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.25.465660', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Multi-omics analysis reveals cross-organism interactions in coral holobiont', 'authors': \"[{'name': 'Toru Maruyama'}, {'name': 'Michihiro Ito'}, {'name': 'Satoshi Wakaoji'}, {'name': 'Yusuke Okubo'}, {'name': 'Keigo Ide'}, {'name': 'Yohei Nishikawa'}, {'name': 'Hiroyuki Fujimura'}, {'name': 'Shoichiro Suda'}, {'name': 'Yoshikatsu Nakano'}, {'name': 'Noriyuki Satoh'}, {'name': 'Chuya Shinzato'}, {'name': 'Kei Yura'}, {'name': 'Haruko Takeyama'}]\", 'abstract': '<p>Corals create an ecosystem, called a holobiont, with intracellular algae (zooxanthellae) and resident bacteria. Zooxanthellae and some bacteria play major roles in the physiological properties of the coral host. However, because of the difficulties in experimental verification of cross-organism interactions, the mechanisms underpinning these interactions are largely unknown. To address this, we here generated and then analyzed multi-omics datasets for corals, zooxanthellae, and bacteria collected at Okinawa, Japan, from November 2014 to September 2016. Using cross-organism co-expression analysis, we successfully characterized the hostalga relationship in the coral holobiont. Specifically, we observed that the coral host dominates the zooxanthellae. The multi-omics analysis also suggested that infection with coral-associated bacteria <italic>Endozoicomonas</italic> likely involves coral-like ephrin ligands, triggering an immune response of the coral host. This study highlights the potential of the multi-omics approach to elucidate coralmicrobe interactions.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.21265615', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Transmission dynamics of SARS-CoV-2 in a strictly-Orthodox Jewish community in the UK', 'authors': \"[{'name': 'William Waites'}, {'name': 'Carl AB Pearson'}, {'name': 'Katherine M Gaskell'}, {'name': 'Thomas House'}, {'name': 'Lorenzo Pellis'}, {'name': 'Marina Johnson'}, {'name': 'Victoria Gould'}, {'name': 'Adam Hunt'}, {'name': 'Neil RH Stone'}, {'name': 'Ben Kasstan'}, {'name': 'Tracey Chantler'}, {'name': 'Sham Lal'}, {'name': 'Chrissy h Roberts'}, {'name': 'David Goldblatt'}, {'name': 'Michael Marks'}, {'name': 'Rosalind M Eggo'}, {'name': ' '}]\", 'abstract': '<p>Some social settings such as households and workplaces, have been identified as high risk for SARS-CoV-2 transmission. Identifying and quantifying the importance of these settings is critical for designing interventions. A tightly-knit religious community in the UK experienced a very large COVID-19 epidemic in 2020, reaching 64.3% seroprevalence within 10 months, and we surveyed this community both for serological status and individual-level attendance at particular settings. Using these data, and a network model of people and places represented as a stochastic graph rewriting system, we estimated the relative contribution of transmission in households, schools and religious institutions to the epidemic, and the relative risk of infection in each of these settings. All congregate settings were important for transmission, with some such as primary schools and places of worship having a higher share of transmission than others. We found that the model needed a higher general-community transmission rate for women (3.3-fold), and lower susceptibility to infection in children to recreate the observed serological data. The precise share of transmission in each place was related to assumptions about the internal structure of those places. Identification of key settings of transmission  can allow public health interventions to be targeted at these locations.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.465104', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Genetic Diversity of Non-Tuberculous Mycobacteria among Symptomatic TB negative Patients in Kenya', 'authors': \"[{'name': 'Zakayo Maingi Maingi'}, {'name': 'Nellie Mukiri'}, {'name': 'Frank Gekara Onyambu'}, {'name': 'Wallace Bulimo'}]\", 'abstract': '<p>Non-Tuberculous Mycobacteria (NTM) transmission to humans occurs through inhalation of dust particles or vaporized water containing NTM leading to pulmonary manifestations. NTM infections are often misdiagnosed for tuberculosis (TB) due to their similar clinical and radiological manifestations. We, therefore, performed a species-level identification of NTM in symptomatic TB negative patients through sequencing of the hsp65 gene. We conducted a cross-sectional study at the National Tuberculosis Reference Laboratory in the period between January to November 2020. One hundred and sixty-six mycobacterial culture-positive samples that tested negative for TB using capilia underwent Polymerase Chain Reaction targeting the hsp65 gene. Isolates showing a band with gel electrophoresis at 441 bp position were sequenced using Sanger technology. Geneious software was used to analyze the obtained sequences, and the National Center for Biotechnology Information gene database identified NTM species for each isolate. A phylogenetic tree was constructed from the DNA sequences and evolutionary distances computed using the general time-reversible method. Pearson chi-square was used to determine the association between NTM infection and participants characteristics. Our study identified 43 different NTM species. The dominant NTM belonged to Mycobacterium avium complex 37 (31%). Slow-growing NTM were the majority at 86 (71%) while rapid-growing NTM were 36 (29%). A significant association (p&lt;0.05) was observed for regions and age, while patient type had a week likelihood of NTM infection. Our study characterized the diversity of NTM in Kenya for the first time and showed a high diversity of NTM species.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466072', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Alu-mediated weak CEBPA binding and slow B cell transdifferentiation in human', 'authors': \"[{'name': 'Ramil Nurtdinov'}, {'name': 'Maria Sanz'}, {'name': 'Amaya Abad'}, {'name': 'Alexandre Esteban'}, {'name': 'Sebastian Ullrich'}, {'name': 'Carme Arnan'}, {'name': 'Rory Johnson'}, {'name': 'Silvia Perez-Lluch'}, {'name': 'Roderic Guigo'}]\", 'abstract': '<p>Many developmental and differentiation processes take substantially longer in human than in mouse. To investigate the molecular mechanisms underlying this phenomenon, here we have specifically focused on the transdifferentiation from B cells to macrophages. The process is triggered by exactly the same molecular mechanism -- the induction by the transcription factor (TF) CEBPA -- but takes three days in mouse and seven in human. In mouse, the speed of this process is known to be associated with Myc expression. We found that in this species, CEBPA binds strongly to the Myc promoter, efficiently down-regulating Myc. In human, in contrast, CEBPA does not bind this promoter, and MYC is indirectly and more slowly down-regulated. Attenuation of CEBPA binding is not specific to the MYC promoter, but a general trait of the human genome across multiple biological conditions. We traced back weak CEBPA binding to the primate-specific Alu repeat expansion. Many Alu repeats carry strong CEBPA binding motifs, which sequester CEBPA, and attenuate CEBPA binding genome-wide. We observed similar CEBPA and MYC dynamics in natural processes regulated by CEBPA, suggesting that CEBPA attenuation could underlie the longer duration in human processes controlled by this factor. Our work highlights the highly complex mode in which biological information is encoded in genome sequences, evolutionarily connecting, in an unexpected way, lineage-specific transposable element expansions to species-specific changes in developmental tempos.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.465825', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Analysis of the Streptococcus mutans proteome during acid and oxidative stress reveals modules of co-expression and an expanded role for the TreR transcriptional regulator', 'authors': \"[{'name': 'Elizabeth L. Tinder'}, {'name': 'Roberta C. Faustoferri'}, {'name': 'Andrew A. Buckley'}, {'name': 'Robert G. Quivey'}, {'name': 'Jonathon L. Baker'}]\", 'abstract': '<p><italic>Streptococcus mutans</italic> promotes a tooth-damaging dysbiosis in the oral microbiota because it can form biofilms and survive acid stress better than most of its ecological competitors, which are typically health-associated. Many of these commensals produce hydrogen peroxide, therefore <italic>S. mutans</italic> must manage both oxidative stress and acid stress with coordinated and complex physiological responses. In this study, the proteome of <italic>S. mutans</italic> was examined during regulated growth in acid and oxidative stresses, as well as in deletion mutants with impaired oxidative stress phenotypes, <italic>nox</italic> and <italic>treR.</italic> 607 proteins exhibited significantly different abundance levels across the conditions tested, and correlation network analysis identified modules of co-expressed proteins that were responsive to the deletion of <italic>nox</italic> and/or <italic>treR</italic>, as well as acid and oxidative stress. The data provided evidence explaining the ROS-sensitive and mutacin-deficient phenotypes exhibited by the <italic>treR</italic> strain. SMU.1069-1070, a poorly understood LytTR system, had elevated abundance in the <italic>treR</italic> strain. <italic>S. mutans</italic> LytTR systems regulate mutacin production and competence, which may explain how TreR affects mutacin production. Furthermore, the gene cluster that produces mutanobactin, a lipopeptide important in ROS tolerance, displayed reduced abundance in the <italic>treR</italic> strain. The role of Nox as a keystone in the oxidative stress response was also emphasized. Crucially, this dataset provides oral health researchers with a proteome atlas that will enable a more complete understanding of the <italic>S. mutans</italic> stress responses that are required for pathogenesis, and facilitate the development of new and improved therapeutic approaches for dental caries.</p><sec><title>Importance</title><p>Dental caries is the most common chronic infectious disease worldwide, and disproportionally affects marginalized socioeconomic groups. <italic>Streptococcus mutans</italic> is a considered a primary etiologic agent of caries, with its pathogenicity dependent on coordinated physiologic stress responses that mitigate the damage caused by the oxidative and acid stress common within dental plaque. In this study, the proteome of <italic>S. mutans</italic> was examined during growth in acidic and oxidative stresses, as well in <italic>nox</italic> and <italic>treR</italic> deletion mutants. 607 proteins were differentially expressed across the strains/growth conditions, and modules of co-expressed proteins were identified, which enabled mapping the acid and oxidative stress responses across <italic>S. mutans</italic> metabolism. The presence of TreR was linked to mutacin production via LytTR system signaling and to oxidative stress via mutanobactin production. The data provided by this study will guide future research elucidating <italic>S. mutans</italic> pathogenesis and developing improved preventative and treatment modalities for dental caries.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.466246', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Detecting ice artefacts in processed macromolecular diffraction data with machine learning', 'authors': \"[{'name': 'Kristopher Nolte'}, {'name': 'Yunyun Gao'}, {'name': 'Sabrina Staeb'}, {'name': 'Philip Kollmansberger'}, {'name': 'Andrea Thorn'}]\", 'abstract': '<p>Contamination with diffraction from ice crystals can negatively affect, or even impede macromolecular structure determination and therefore, detecting the resulting artefacts in diffraction data is crucial. However, once the data have been processed, it can be very difficult to automatically recognize this problem. To address this, a set of convolutional neural networks named Helcaraxe has been developed which can detect ice diffraction artefacts in processed diffraction data from macromolecular crystals. The networks outperform previous algorithms and will be available as part of the AUSPEX webserver and CCP4-distributed software.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.466131', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Artifacts in EEG-based BCI therapies: friend or foe?', 'authors': \"[{'name': 'Eric James McDermott'}, {'name': 'Philipp Raggam'}, {'name': 'Sven Kirsch'}, {'name': 'Paolo Belardinelli'}, {'name': 'Ulf Ziemann'}, {'name': 'Christoph Zrenner'}]\", 'abstract': '<p>EEG-based brain-computer interfaces (BCI) have promising therapeutic potential beyond traditional neurofeedback training, such as enabling personalized and optimized virtual reality (VR) neurorehabilitation paradigms where the timing and parameters of the visual experience is synchronized with specific brain-states. While BCI algorithms are often designed to focus on whichever portion of a signal is most informative, in these brain-state-synchronized applications, it is of critical importance that the resulting decoder is sensitive to physiological brain activity representative of various mental states, and not to artifacts such as those arising from naturalistic movements. In this study, we compare the relative classification accuracy with which different motor tasks can be decoded from both extracted brain activity and artifacts contained in the EEG signal. EEG data was collected from 17 chronic stroke patients while performing six different head, hand, and arm movements in a realistic VR-based neurorehabilitation paradigm. Results show that the artifact component of the EEG signal is significantly more informative than brain activity to the classifier. This finding is consistent across different feature extraction methods and classification pipelines. Whereas informative artifacts are a helpful friend in BCI-based communication applications, they can be a problematic foe in the estimation of physiological brain states.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.466146', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Metacognition, cortical thickness, and tauopathy in aging', 'authors': \"[{'name': 'Kailin Zhuang'}, {'name': 'Xi Chen'}, {'name': 'Suzanne L. Baker'}, {'name': 'William J. Jagust'}]\", 'abstract': \"<p>We investigated two aspects of metacognition and their relationship with cortical thickness and Alzheimer's Disease (AD) biomarkers, amyloid and tau, in cognitively healthy older adults (N=151). The two metacognition measures were self-appraisal rating of task performance and the difference between self-appraisal rating and actual task performance (appraisal discrepancy). All participants underwent neuropsychological testing and 1.5T structural MRI. A subset (N=66) received amyloid-PET with [11C] PiB and tau-PET with [18F] Flortaucipir. We found that worse performers had lower self-appraisal ratings, but still overestimated their performance, consistent with the Dunning-Kruger effect. Self-appraisal rating and appraisal discrepancy revealed distinct relationships with cortical thickness and AD pathology. Greater appraisal discrepancy, indicating overestimation, was related to thinning of inferior-lateral temporal, fusiform, and rostral anterior cingulate cortices. Lower self-appraisal was associated with higher entorhinal and inferior temporal tau. These results suggest that overestimation could implicate structural atrophy beyond AD pathology, while lower self-appraisal could indicate early behavioral alteration due to AD pathology, supporting the notion of subjective cognitive decline prior to objective deficits.</p>\", 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.466159', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Clonal hematopoiesis detection in cancer patients using cell free DNA sequencing', 'authors': '[{\\'name\\': \\'Lauren Fairchild\\'}, {\\'name\\': \\'Jeanne Whalen\\'}, {\\'name\\': \"Katie D\\'Aco\"}, {\\'name\\': \\'Jincheng Wu\\'}, {\\'name\\': \\'Carroll B Gustafson\\'}, {\\'name\\': \\'Nadia Solovieff\\'}, {\\'name\\': \\'Fei Su\\'}, {\\'name\\': \\'Rebecca Leary\\'}, {\\'name\\': \\'Catarina D Campbell\\'}, {\\'name\\': \\'Oscar Alejandro Balbin\\'}]', 'abstract': '<p>In the context of cancer, clonal hematopoiesis of indeterminate potential (CHIP) is associated with the development of therapy-related myeloid neoplasms and shorter overall survival. Cell-free DNA (cfDNA) sequencing is becoming widely adopted for genomic screening of cancer patients but has not been used extensively to determine CHIP status due to a requirement for matched blood and tumor sequencing. Here we present an accurate machine learning approach to determine clonal hematopoiesis (CH) status from cfDNA sequencing alone and apply our model to 4,096 oncology clinical cfDNA samples. Using this method, we determine that 26% of patients in this cohort have evidence of CH and CH is most common in lung cancer patients. Matched RNAseq data shows signals of increased inflammation, especially neutrophil activation, within the tumor microenvironment of CH-positive patients. Additionally, CH patients showed evidence of neutrophil activation systemically, pointing to a potential mechanism of action for the worse outcomes associated with CH status. Neutrophil activation may be one of many mechanisms however, as estrogen positive breast cancer patients harboring TET2 frameshift mutations had worse outcomes but similar neutrophil levels to CH-negative patients.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465969', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'The endoplasmic reticulum adopts two distinct tubule forms', 'authors': \"[{'name': 'Bowen Wang'}, {'name': 'Zhiheng Zhao'}, {'name': 'Michael Xiong'}, {'name': 'Rui Yan'}, {'name': 'Ke Xu'}]\", 'abstract': '<p>The endoplasmic reticulum (ER) is a versatile organelle with diverse functions. Through super-resolution microscopy, we show that the peripheral ER in the mammalian cell adopts two distinct forms of tubules. Whereas an ultrathin form, R1, is consistently covered by ER-membrane curvature-promoting proteins, <italic>e</italic>.<italic>g</italic>., Rtn4 in the native cell, in the second form, R2, Rtn4 and analogs are arranged into two parallel lines at a conserved separation of 105 nm over long ranges. The two tubule forms together account for 90% of the total tubule length in the cell, with either one being dominant in different cell types. The R1-R2 dichotomy and the final tubule geometry are both co-regulated by Rtn4 (and analogs) and the ER sheet-maintaining protein Climp63, which respectively define the edge curvature and lumen height of the R2 tubules to generate a ribbon-like structure of well-defined width. Accordingly, the R2 tubule width correlates positively with the Climp63 intralumenal size. The R1 and R2 tubules undergo active remodeling at the second/sub-second time scales as they differently accommodate proteins, with the former effectively excluding ER-luminal proteins and ER-membrane proteins with large intraluminal domains. We thus uncover a dynamic structural dichotomy for ER tubules with intriguing functional implications.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.21265491', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Genetic and Genomic Medicine', 'covid': False, 'title': 'Loss-of-Function FLNC Variants are Associated with Arrhythmogenic Cardiomyopathy Phenotypes when Identified through Exome Sequencing of a General Clinical Population', 'authors': \"[{'name': 'Eric D Carruth'}, {'name': 'Maria Qureshi'}, {'name': 'Amro Alsaid'}, {'name': 'Melissa A Kelly'}, {'name': 'Hugh Calkins'}, {'name': 'Brittney Murray'}, {'name': 'Crystal Tichnell'}, {'name': 'Amy C Sturm'}, {'name': 'Aris Baras'}, {'name': 'H Lester Kirchner'}, {'name': 'Brandon K Fornwalt'}, {'name': 'Cynthia A James'}, {'name': 'Christopher M Haggerty'}, {'name': ' '}]\", 'abstract': '<p><bold>Background</bold>\\nThe FLNC gene has recently garnered attention as a likely cause of arrhythmogenic cardiomyopathy (ACM), which is considered an actionable genetic condition. However, the association with disease in an unselected clinical population is unknown. We hypothesized that individuals with loss-of-function variants in FLNC (FLNC<sub>LOF</sub>) would have increased odds for ACM-associated phenotypes versus variant-negative controls in the Geisinger MyCode cohort.\\n<bold>Methods</bold>\\nWe identified rare, putative FLNC<sub>LOF</sub> among 171,948 individuals with exome sequencing linked to health records. Associations with ACM phenotypes from available diagnoses and cardiac evaluations were investigated.\\n<bold>Results</bold>\\nSixty individuals (0.03%; median age 58 years [47  70 IQR], 43% male) harbored 27 unique FLNC<sub>LOF</sub>. These individuals had significantly increased odds ratios (OR) for dilated cardiomyopathy (OR:4.9, [95% confidence interval: 2.6  7.6]; p&lt;0.001), supraventricular tachycardia (OR:3.2, [1.1  5.6]; p=0.01), defibrillator implantation (OR:4.6, [1.9  8.4]; p&lt;0.001), and left-dominant ACM (OR:4.2, [1.4  7.9]; p=0.003). Echocardiography revealed reduced left ventricular ejection fraction (52  13% vs. 57  9%; p=0.001) associated with FLNC<sub>LOF</sub>. Overall, at least 9% of FLNC<sub>LOF</sub> carriers demonstrated evidence of penetrant disease.\\n<bold>Conclusions</bold>\\nFLNC<sub>LOF</sub> variants are associated with increased odds of ventricular arrhythmia and dysfunction in an unselected clinical population. These findings support genomic screening of FLNC for actionable secondary findings.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465918', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Extensive sequence divergence of non-coding regions between Aspergillus fumigatus, a major fungal pathogen of humans, and its relatives', 'authors': \"[{'name': 'Alec Brown'}, {'name': 'Matthew E. Mead'}, {'name': 'Jacob L. Steenwyk'}, {'name': 'Gustavo H. Goldman'}, {'name': 'Antonis Rokas'}]\", 'abstract': '<p>Invasive aspergillosis is a deadly fungal disease; more than 400,000 patients are infected worldwide each year and the mortality rate can be as high as 50-95%. Of the 450 species in the genus <italic>Aspergillus</italic> only a few are known to be clinically relevant, with the major pathogen <italic>Aspergillus fumigatus</italic> being responsible for 50% of all invasive mold infections. Genomic comparisons of <italic>A. fumigatus</italic> to other <italic>Aspergillus</italic> species have historically focused on protein-coding regions. However, most <italic>A. fumigatus</italic> genes, including those that modulate its virulence, are also present in non-pathogenic close relatives of <italic>A. fumigatus</italic>. Our hypothesis is that differential gene regulation  mediated through the non-coding regions upstream of genes transcription start sites  contributes to <italic>A. fumigatus</italic> pathogenicity. To begin testing this, we compared non-coding regions up to 500 base pairs upstream of the first codon of single-copy orthologous genes from the two <italic>A. fumigatus</italic> reference strains Af293 and A1163 and eight closely related <italic>Aspergillus</italic> section <italic>Fumigati</italic> species. We found that non-coding regions showed extensive sequence variation and lack of homology across species. By examining the evolutionary rates of both protein-coding and non-coding regions in a subset of orthologous genes with highly conserved non-coding regions across the phylogeny, we identified 418 genes, including 25 genes known to modulate <italic>A. fumigatus</italic> virulence, whose non-coding regions exhibit a different rate of evolution in <italic>A. fumigatus</italic>. Examination of sequence alignments of these non-coding regions revealed numerous instances of insertions, deletions, and other types of mutations of at least a few nucleotides in <italic>A. fumigatus</italic> compared to its close relatives. These results show that closely related <italic>Aspergillus</italic> species that vary greatly in their pathogenicity exhibit extensive non-coding sequence variation and identify numerous changes in non-coding regions of <italic>A. fumigatus</italic> genes known to contribute to virulence.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.28.21265606', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': False, 'title': 'Rapid health evaluation in migrant peoples in transit through Darien, Panama: Protocol for a multi-method qualitative and quantitative study', 'authors': \"[{'name': 'Amanda Gabster'}, {'name': 'Monica Jhanigmal'}, {'name': 'Jennifer Toller Erausquin'}, {'name': 'Jose Antonio Suarez'}, {'name': 'Justo Emilio Pinzon-Espinosa'}, {'name': 'Madeline Baird'}, {'name': 'Jennifer Katz'}, {'name': 'Davis Beltran Henriquez'}, {'name': 'Gonzalo Cabezas Talavero'}, {'name': 'Andres Henao Martinez'}, {'name': 'Carlos Franco-Paredes'}, {'name': 'Nelson Agudelo-Higuita'}, {'name': 'Monica Pachar'}, {'name': 'Jose Anel Gonzalez'}, {'name': 'Juan Miguel Pascale'}, {'name': ' '}]\", 'abstract': '<p>Abstract\\n\\nBackground:\\nThe world is currently unprepared to deal with a the drastic increase in global migration. There is an urgent need to develop programs to protect the well being and health of migrant peoples. Increased population movement is already evident throughout the Americas as exemplified by the rising number of migrant peoples that pass through the Darien neotropical moist broadleaf forest along the border region between Panama and Colombia. The transit of migrant peoples through this area has an increase in the last years. In 2021 an average of 9,400 people entered the region per month compared to 2,000-3,500 people monthly in 2019. Along this trail, there is no access to healthcare, food provision, potable water, or housing. To date, much of what is known about health needs and barriers to healthcare within this population is based on journalistic reports and anecdotes. There is a need for a comprehensive approach to assess the healthcare needs migrant peoples in transit.  This study aims to describe demographic characteristics, mental and physical health status and needs, and experiences of host communities, and to identify opportunities to improve healthcare provision to migrant peoples in transit in Panama. \\n\\nStudy design and methods: \\nThis multi-method study will include qualitative (n=70) and quantitative (n=520) components.  The qualitative component includes interviews with migrant peoples in transit, national and international non-governmental organizations and agencies based in Panama. The quantitative component is a rapid epidemiological study which includes a questionnaire and four clinical screenings: mental health, sexual and reproductive health, general and tropical medicine, and nutrition. \\n\\nConclusion:\\nThis study will contribute to a better understanding of the health status and needs of migrant peoples in transit through Darien, Panama. Findings will be used to allocate resources and provide  targeted healthcare interventions for migrant peoples in transit through Darien, Panama.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.31.466679', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Dimensionality and modularity of adaptive variation: Divergence in threespine stickleback from diverse environments', 'authors': \"[{'name': 'Grant E. Haines'}, {'name': 'Louis Moisan'}, {'name': 'Alison M. Derry'}, {'name': 'Andrew P. Hendry'}]\", 'abstract': '<p>In nature, populations are subjected to a wide variety of environmental conditions that affect fitness and induce adaptive or plastic responses in traits, resulting in phenotypic divergence between populations. The dimensionality of that divergence, however, remains contentious. At the extremes, some contend that populations diverge along a single axis of trait covariance with greatest availability of heritable variation, even if this does not lead a population directly to its fitness optimum. Those at the other extreme argue that selection can push populations towards their fitness optima along multiple phenotype axes simultaneously, resulting in divergence in numerous dimensions. Here, we address this debate using populations of threespine stickleback (<italic>Gasterosteus aculeatus</italic>) in the Cook Inlet region of southern Alaska from lakes with contrasting ecological conditions. We calculated effective dimensionality of divergence in several trait suites (defensive, swimming, and trophic) thought to be under correlated selection pressures, as well as across all traits. We also tested for integration among the trait suites and between each trait suite and the environment. We found that populations in the Cook Inlet radiation exhibit dimensionality of phenotype high enough to preclude a single axis of divergence.</p>', 'posted': '2021-10-31'}, {'doi': '10.1101/2021.10.29.465169', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Genome-wide association study reveals complex genetic architecture of cadmium and mercury accumulation and tolerance traits in Medicago truncatula', 'authors': \"[{'name': 'Tim Paape'}, {'name': 'Benjamin Heiniger'}, {'name': 'Miguel Santo Domingo'}, {'name': 'Michael Clear'}, {'name': 'M. Mercedes Lucas'}, {'name': 'Jose Pueyo'}]\", 'abstract': '<p>Heavy metals are an increasing problem due to contamination from human sources that and can enter the food chain by being taken up by plants. Understanding the genetic basis of accumulation and tolerance in plants is important for reducing the uptake of toxic metals in crops and crop relatives, as well as for removing heavy metals from soils by means of phytoremediation. Following exposure of Medicago truncatula seedlings to cadmium (Cd) and mercury (Hg), we conducted a genome-wide association study using relative root growth (RRG) and leaf accumulation measurements. Cd and Hg accumulation and RRG had heritability ranging 0.44 - 0.72 indicating high genetic diversity for these traits. The Cd and Hg trait associations were broadly distributed throughout the genome, indicated the traits are polygenic and involve several quantitative loci. For all traits, candidate genes included several membrane associated ATP-binding cassette transporters, P-type ATPase transporters, several oxidative stress response genes and stress related UDP-glycosyltransferases. The P-type ATPase transporters and ATP-binding cassette protein-families have roles in vacuole transport of heavy metals, and our findings support their wide use in physiological plant responses to heavy metals and abiotic stresses.  We also found associations between Cd RRG with the genes CAX3 and PDR3, two linked adjacent genes, and leaf accumulation of Hg associated with the genes NRAMP6 and CAX9. When plant genotypes with the most extreme phenotypes were compared, we found significant divergence in genomic regions using population genomics methods that contained metal transport and stress response gene ontologies. Several of these genomic regions show high linkage disequilibrium (LD) among candidate genes suggesting they have evolved together. Minor allele frequency (MAF) and effect size of the most significant SNPs was negatively correlated with large effect alleles being most rare. This is consistent with purifying selection against alleles that increase toxicity and abiotic stress.  Conversely, the alleles with large affect that had higher frequencies that were associated with the exclusion of Cd and Hg. Overall, macroevolutionary conservation of heavy metal and stress response genes is important for improvement of forage crops by harnessing wild genetic variants in gene banks such as the Medicago HapMap collection.</p>', 'posted': '2021-10-31'}, {'doi': '10.1101/2021.10.22.465513', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Cloud-enabled Fiji for reproducible, integrated and modular image processing', 'authors': \"[{'name': 'Ling-Hong Hung'}, {'name': 'Evan Straw'}, {'name': 'Shishir Reddy'}, {'name': 'Zachary Colburn'}, {'name': 'Ka Yee Yeung'}]\", 'abstract': '<sec><title>Summary</title><p>Biomedical image analyses can require many steps processing different types of data. Analysis of increasingly large data sets often exceeds the capacity of local computational resources. We present an easy-to-use and modular cloud platform that allows biomedical researchers to reproducibly execute and share complex analytical workflows to process large image datasets. The workflows and the platform are encapsulated in software containers to ensure reproducibility and facilitate installation of even the most complicated workflows. The platform is both graphical and interactive allowing users to use the viewer of their choice to adjust the image pre-processing and analysis steps to iteratively improve the final results. We demonstrate the utility of our platform via two use cases in focal adhesion and 3D imaging analyses. In particular, our focal adhesion workflow demonstrates integration of Fiji with Jupyter Notebooks. Our 3D imaging use case applies Fiji/BigStitcher to big datasets on the cloud. The accessibility and modularity of the cloud platform democratizes the application and development of complex image analysis workflows.</p></sec>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.29.466423', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Modeling the efficacy of CRISPR gene drive for schistosomiasis control', 'authors': \"[{'name': 'Richard E Grewelle'}, {'name': 'Javier Perez-Saez'}, {'name': 'Josh Tycko'}, {'name': 'Erica KO Namigai'}, {'name': 'Chloe G Rickards'}, {'name': 'Giulio A De Leo'}]\", 'abstract': \"<p>CRISPR gene drives could revolutionize the control of infectious diseases by accelerating the spread of engineered traits that limit parasite transmission in wild populations.  While much effort has been spent developing gene drives in mosquitoes, gene drive technology in molluscs has received little attention despite the role of freshwater snails as obligate, intermediate hosts of parasitic flukes causing schistosomiasis -- a disease of poverty affecting more than 200 million people worldwide.  A successful drive in snails must overcome self-fertilization, which prevents a drive's spread. Simultaneous hermaphroditism is a feature of snails -- distinct from gene drive model organisms -- and is not yet incorporated in gene drive models of disease control.  Here we developed a novel population genetic model accounting for snails' sexual and asexual reproduction, susceptibility to parasite infection regulated by multiple alleles, fitness differences between genotypes, and a range of drive characteristics.  We then integrated this model with an epidemiological model of schistosomiasis transmission and snail population dynamics.  Simulations showed that gene drive establishment can be hindered by a variety of biological and ecological factors, including selfing.  However, our model suggests that, under a range of conditions, gene drive mediated immunity in snails could maintain rapid disease reduction achieved by annual chemotherapy treatment of the human population, leading to long-term elimination.  These results indicate that gene drives, in coordination with existing public health measures, may become a useful tool to reduce schistosomiasis burden in selected transmission settings with effective CRISPR construct design and close evaluation of the genetic and ecological landscape.</p>\", 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.21.21265321', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': False, 'title': 'Accuracy of computer-aided chest X-ray screening in the Kenya National Tuberculosis Prevalence Survey', 'authors': '[{\\'name\\': \\'Brenda Mungai\\'}, {\\'name\\': \"Jane Ong\\'ang\\'o\"}, {\\'name\\': \\'Chu Chang Ku\\'}, {\\'name\\': \\'Marc YR Henrion\\'}, {\\'name\\': \\'Ben Morton\\'}, {\\'name\\': \\'Elizabeth Joekes\\'}, {\\'name\\': \\'Richard Kiplimo\\'}, {\\'name\\': \\'Dickson Kirathe\\'}, {\\'name\\': \\'Enos Masini\\'}, {\\'name\\': \\'Joseph Sitienei\\'}, {\\'name\\': \\'Veronica Manduku\\'}, {\\'name\\': \\'Beatrice Mugi\\'}, {\\'name\\': \\'Stephen Bertel Squire\\'}, {\\'name\\': \\'Peter MacPherson\\'}, {\\'name\\': \\' \\'}]', 'abstract': '<p>Background: Communitybased screening for tuberculosis (TB) could improve detection but is resource intensive. We set out to evaluate the accuracy of computeraided TB screening using digital chest X-ray (CXR) to determine if this approach met target product profiles (TPP) for communitybased screening.\\n\\nMethods: CXR images from participants in the 2016 Kenya National TB Prevalence Survey were evaluated using CAD4TBv6 (Delft Imaging), giving a probabilistic score for pulmonary TB ranging from 0 (low probability) to 99 (high probability). We constructed a Bayesian latent class model to estimate the accuracy of CAD4TBv6 screening compared to bacteriologicallyconfirmed TB across CAD4TBv6 threshold cutoffs, incorporating data on Clinical Officer CXR interpretation, participant demographics (age, sex, TB symptoms, previous TB history), and sputum results. We compared modelestimated sensitivity and specificity of CAD4TBv6 to optimum and minimum TPPs.\\n\\nResults: Of 63,050 prevalence survey participants, 61,848 (98%) had analysable CXR images, and 8,966 (14.5%) underwent sputum bacteriological testing; 298 had bacteriologicallyconfirmed pulmonary TB. Median CAD4TBv6 scores for participants with bacteriologicallyconfirmed TB were significantly higher (72, IQR: 5882.75) compared to participants with bacteriologicallynegative sputum results (49, IQR: 4457, p&lt;0.0001). CAD4TBv6 met the optimum TPP; with the threshold set to achieve a mean sensitivity of 95% (optimum TPP), specificity was 83.3%, (95% credible interval [CrI]: 83.0%83.7%, CAD4TBv6 threshold: 55). There was considerable variation in accuracy by participant characteristics, with older individuals and those with previous TB having lowest specificity. \\n\\nConclusions: CAD4TBv6 met the optimal TPP for TB community screening. To optimise screening accuracy and efficiency of confirmatory sputum testing, we recommend that an adaptive approach to threshold setting is adopted based on participant characteristics.\\n\\u2003</p>', 'posted': '2021-10-30'}, {'doi': '10.1101/2021.10.28.466326', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Efficient, selectable marker free gene targeting in soybean using novel Ochrobactrum haywardense-mediated delivery', 'authors': \"[{'name': 'Sandeep Kumar'}, {'name': 'Zhan-Bin Lui'}, {'name': 'Nathalie Sanyour-Doyel'}, {'name': 'Brian Landerts'}, {'name': 'Andrew Worden'}, {'name': 'Ajith Anand'}, {'name': 'Hyeon-Je Cho'}, {'name': 'Jyothi Bolar'}, {'name': 'Charlotte Harris'}, {'name': 'Lingxia Huang'}, {'name': 'Aiqiu Xing'}, {'name': 'Alexandra Richardson'}]\", 'abstract': '<p>We report robust selectable marker-free gene targeting (GT) system in soybean, one of the most economically important crops. A novel efficient Ochrobactrum haywardense-mediated embryonic axis transformation method was used for the delivery of CRISPR-Cas9 components and donor template to regenerate T0 plants in 6-8 weeks after transformation. This approach generated up to 3.4% targeted insertion of the donor sequence into the target locus in T0 plants, with ~ 90% mutation rate observed at the genomic target site. The GT was demonstrated in two genomic sites using two different donor DNA templates without a need of a selectable marker within the template. High-resolution Southern by Sequencing (SbS) analysis identified T1 plants with precise targeted insertion and without unintended plasmid DNA. Unlike previous low-frequency GT reports in soybean that involved particle bombardment-mediated delivery and extensive selection, the method described here is fast, efficient, reproducible, does not require selectable marker within the donor DNA, and generates non-chimeric plants with heritable GT.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.27.466172', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Dynamic Subcortical Modulators of Human Default Mode Network Function', 'authors': \"[{'name': 'Ben J Harrison'}, {'name': 'Christopher G Davey'}, {'name': 'Hannah S Savage'}, {'name': 'Alec J Jamieson'}, {'name': 'Christine A Leonards'}, {'name': 'Bradford A Moffat'}, {'name': 'Rebecca K Glarin'}, {'name': 'Trevor Steward'}]\", 'abstract': \"<p>The brain's 'default mode network' (DMN) enables flexible switching between internally and externally focused cognition. Precisely how this modulation occurs is not well understood, although may involve key subcortical mechanisms, including hypothesized influences from the basal forebrain (BF) and mediodorsal thalamus (MD). Here, we used ultra-high field (7T) functional magnetic resonance imaging to examine the involvement of the BF and MD across states of task-induced DMN activity modulation. Specifically, we mapped DMN activity suppression ('deactivation') when participants transitioned between rest and externally focused task performance, as well as DMN activity engagement ('activation') when task performance was internally (i.e., self) focused. Consistent with recent rodent studies, the BF showed overall activity suppression with DMN cortical regions when comparing the rest to external task conditions. Further analyses, including dynamic causal modelling, confirmed that the BF drove changes in DMN cortical activity during these rest-to-task transitions. The MD, by comparison, was specifically engaged during internally focused cognition and demonstrated a broad excitatory influence on DMN cortical activation. These results provide the first direct evidence in humans of distinct basal forebrain and thalamic circuit influences on the control of DMN function and suggest novel mechanistic avenues for ongoing translational research.</p>\", 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.21265578', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Primary Care Research', 'covid': True, 'title': 'Impact of the COVID-19 pandemic on routine immunization coverage in children under 2 years old in Ontario, Canada: A retrospective cohort study', 'authors': \"[{'name': 'Catherine Ji'}, {'name': 'Pierre-Philippe Piche-Renaud'}, {'name': 'Jemisha Apajee'}, {'name': 'Ellen Stephenson'}, {'name': 'Milena Forte'}, {'name': 'Jeremy N Friedman'}, {'name': 'Michelle Science'}, {'name': 'Stanley Zlotkin'}, {'name': 'Shaun K Morris'}, {'name': 'Karen Tu'}]\", 'abstract': '<p>Background: The COVID-19 pandemic has caused a disruption in childhood immunization coverage around the world. This study aimed to determine the change in immunization coverage for children under 2 years old in Ontario, Canada, comparing time periods pre-pandemic to during the pandemic.\\nMethods: We conducted an observational retrospective open cohort study, using primary care electronic medical record data from the University of Toronto Practice-Based Research Network (UTOPIAN) database, from January 2019 to December 2020. Children under 2 years old who had at least 2 visits recorded in UTOPIAN were included. We measured up-to-date (UTD) immunization coverage rates, overall and by type of vaccine (DTaP-IPV-Hib, Pneu-C-13, Rot, Men-C-C, MMR, Var), and on-time immunization coverage rates by age milestone (2, 4, 6, 12, 15 and 18 months). We compared average coverage rates over 3 periods of time: January 2019-March 2020 (T1); March-July 2020 (T2); and August-December 2020 (T3).\\nResults: 12,313 children were included. Overall UTD coverage for all children was 71.0% in T1, dropped by 5.7% (95% CI: -6.2, -5.1) in T2, slightly increased in T3 but remained lower than in T1. MMR vaccine UTD coverage slightly decreased in T2 and T3 by approximately 2%. The largest decreases were seen at ages 15-month and 18-month old, with drops in on-time coverage of 14.7% (95% CI: -18.7, -10.6) and 16.4% (95% CI: -20.0, -12.8) respectively during T2. When stratified by sociodemographic characteristics, no specific subgroup of children was found to have been differentially impacted by the pandemic.\\nConclusion: Childhood immunization coverage rates for children under 2 years in Ontario decreased significantly during the early period of the COVID-19 pandemic and only partially recovered during the rest of 2020. Public health and educational interventions for providers and parents are needed to ensure adequate catch-up of delayed/missed immunizations to prevent potential outbreaks of vaccine-preventable diseases.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.21265604', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Transferability of Psychological Interventions from Disaster-Exposed Employees to Healthcare Workers Working during the COVID-19 Pandemic', 'authors': \"[{'name': 'Sean Treacy'}, {'name': 'Shane O Donnell'}, {'name': 'Blanaid Gavin'}, {'name': 'Tamara Schloemer'}, {'name': 'Etain Quigley'}, {'name': 'Dimitrios Adamis'}, {'name': 'Fiona Mc Nicholas'}, {'name': 'John Hayden'}]\", 'abstract': '<p>Background: The COVID-19 Pandemic had a significant negative impact on the mental health of healthcare workers. Evidence-based interventions that could be used to mitigate this impact are lacking in the literature. This review aims to evaluate psychological interventions used for employees following previous disasters and assess the transferability of these interventions to a healthcare setting during the COVID-19 pandemic. \\nMethods: Intervention information from a previously published systematic review of the literature published up to 2015 was extracted, and an additional search of studies published from 2015-2020 was conducted.  Studies were assessed for transferability using a checklist derived from the PIET-T process model. \\nResults: Interventions from eighteen studies were assessed for transferability (including three studies identified in an updated literature search). Interventions established as most transferable included resilience training, meditation/mindfulness interventions, and cognitive behavioural therapy. Psychological debriefing was transferable but as it is contrary to current recommendations is not deemed appropriate for adoption. \\nImplications: Several existing interventions have the potential to be utilised within the COVID-19 context/pandemic. More research needs to be undertaken in this area to assess these interventions upon transfer.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.21265608', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Nursing', 'covid': True, 'title': 'Social discrimination and stigma on the community of health professionals during the Covid-19 pandemic. An ethnographic approach', 'authors': \"[{'name': 'Alexandros Argyriadis'}, {'name': 'Athina Patelarou'}, {'name': 'Vasiliki Kitsona'}, {'name': 'Alexandra Trivli'}, {'name': 'Evridiki Patelarou'}, {'name': 'Agathi Argyriadi'}]\", 'abstract': '<p>Severe Acute Respiratory Syndrome Coronavirus Type 2 (SARS-CoV-2) that caused the pandemic since March 2020, has affected among others, health professionals who work in covid-19 units by facing social discrimination. The aim of this study was to record the experiences of health professionals working in the first line of treatment of the pandemic, to analyse the effects of the pandemic on the interpersonal relationships of health professionals, and to ask about the stigma they faced during their work with people with covid-19.\\nThis is a qualitative study with an ethnographic approach based on 160 semi-structured interviews with health professionals living and working in the Epirus Region, Greece. For the data collection we used semi-structured interviews, discussions and participatory observation.  Specifically, the interviews were conducted on health professionals and more specifically doctors, nurses, rescuers, physiotherapists and administrative staff, working in covid-19 units at the University General Hospital of Ioannina (Reference hospital for Ioannina, in Epirus), which assists in the treatment of patients with covid-19, and in the branch of the rescue department of Ioannina. \\nThe data were analysed in four thematic units based on their common characteristics: a) emotions and experiences of health professionals, b) interpersonal relations of health professionals, c) social exclusion and discrimination, and d) health professionals as patients. The results showed that the main emotions that health professionals experienced when they were moved to covid-19 clinics were fear, anxiety, distress, anger and insecurity. These feelings worsened when their family environment treated them with fear and hesitancy. Their social environment tended to avoid them, leading to a state of self-isolation. To conclude, health professionals faced discriminating behaviors and stigma both from their families and social environment, and from other health professionals. The government struggled to handle the situation in keeping a balance between both the security and well-being of health professionals as it was not prepared for a pandemic like this.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.26.465985', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Ligand-induced unfolding mechanism of an RNA G-quadruplex', 'authors': \"[{'name': 'Susanta Haldar'}, {'name': 'Yashu Zhang'}, {'name': 'Ying Xia'}, {'name': 'Barira Islam'}, {'name': 'Sisi Liu'}, {'name': 'Francesco L Gervasio'}, {'name': 'Adrian J. Mulholland'}, {'name': 'Zo A. E. Waller'}, {'name': 'Dengguo Wei'}, {'name': 'Shozeb Haider'}]\", 'abstract': '<p>The cationic porphyrin, TMPyP4, is a well-established DNA G-quadruplex (G4) binding ligand that can stabilize different topologies via multiple binding modes. However, TMPyP4 has completely opposite destabilizing and unwinding effect on RNA G4 structures. The structural mechanisms that mediate RNA G4 unfolding remains unknown. Here, we report on the TMPyP4-induced RNA G4 unfolding mechanism studied by well-tempered metadynamics (WT-MetaD) with supporting biophysical experiments. The simulations predict a two-state mechanism of TMPyP4 interaction via a groove-bound and a top-face bound conformation. The dynamics of TMPyP4 stacking on the top tetrad disrupts Hoogsteen H-bonds between guanine bases resulting in the consecutive TMPyP4 intercalation from top-to-bottom G-tetrads. The results reveal a striking correlation between computational and experimental approaches and validate WT-MetaD simulations as a powerful tool for studying RNA G4-ligand interactions.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.26.465862', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Transcriptome Dynamics of Floral Organs Approaching Blooming in the Flowering Cherry (Cerasus  yedoensis) Cultivar Somei-Yoshino', 'authors': \"[{'name': 'Kenta Shirasawa'}, {'name': 'Tomoya Esumi'}, {'name': 'Akihiro Itai'}, {'name': 'Sachiko Isobe'}]\", 'abstract': '<p>To gain insights into the genetic mechanisms underlying blooming and petal movement in flowering cherry (<italic>Cerasus</italic>  <italic>yedoensis</italic>), we performed time-course RNA-seq analysis of the floral buds and open-flowers of the most popular flowering cherry cultivar, Somei-Yoshino. Independent biological duplicate samples of floral buds and open-flowers were collected from Somei-Yoshino trees grown at three different locations in Japan. RNA-seq reads obtained from floral bud and open-flower samples collected in the current study (in 2019) and in a previous study (in 2017) were aligned against the genome sequence of Somei-Yoshino to quantify gene transcript levels. Clustering analysis of RNA-seq reads revealed dynamic changes in the transcriptome, with genes in seven modules predominantly expressed at specific time points, ranging from 5 weeks before flowering to 2 weeks after flowering. Based on the identified gene modules and Gene Ontology (GO) terms enriched at different floral stages, we speculate that the genetic mechanisms underlying petal movement and flower opening in cherry involve the processes of development, cell wall organization, reproduction, and metabolism, which are executed by genes encoding transcription factors, phytohormones, transporters, and polysaccharide metabolic enzymes. Furthermore, we propose a method for cherry bloom forecasting, based on gene expression levels at different time points before flowering as RNA markers.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.26.465915', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Hematopoietic stem and progenitor cells improve survival from sepsis by boosting immunomodulatory cells', 'authors': \"[{'name': 'Daniel E. Morales-Mantilla'}, {'name': 'Bailee Kain'}, {'name': 'Duy Le'}, {'name': 'Anthony Flores'}, {'name': 'Silke Paust'}, {'name': 'Katherine King'}]\", 'abstract': '<p>Sepsis accounts for 1 in 5 deaths worldwide, and new therapeutic strategies to reduce sepsis- related mortality are urgently needed. Since hematopoietic stem and progenitor cells (HSPCs) are responsible for producing blood and immune cells, including in response to immunological stress, we explored their potential for treating sepsis. In a mouse model of Group A <italic>Streptococcus</italic> (GAS)-induced sepsis, severe immunological stress was associated with significant depletion of bone marrow HSPCs and mortality within approximately 5-7 days. We hypothesized that the inflammatory environment of GAS infection drives rapid HSPC differentiation and depletion that can be rescued by infusion of donor HSPCs. Indeed, infusion of 10,000 nave HSPCs into GAS-infected mice resulted in rapid myelopoiesis and a 50-60% increase in overall survival. Surprisingly, mice receiving donor HSPCs displayed a similar pathogen load compared to untreated mice. Flow cytometric analysis revealed a significantly increased number of myeloid-derived suppressor cells in HSPC-infused mice, which correlated with reduced inflammatory cytokine levels and restored HSPC levels. These findings suggest that HSPCs play an essential immunomodulatory role that may translate into new therapeutic strategies for sepsis.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.28.21265610', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Pediatrics', 'covid': True, 'title': 'COVID-19 Patients Symptoms: Gastrointestinal Presentations, Comorbidities and Outcome in a Canadian Hospital setting', 'authors': \"[{'name': 'Hassan Brim'}, {'name': 'Michal Moshkovich'}, {'name': 'Nikhil Pai'}, {'name': 'Melanie Figueiredo'}, {'name': 'Emily Hartung'}, {'name': 'Antonio Pizuorno'}, {'name': 'Lee Hill'}, {'name': 'Jelena Popov'}, {'name': 'Eyitope Olaide Awoyemi'}, {'name': 'Waliul I Khan'}, {'name': 'Reza Oskrochi'}, {'name': 'Hassan Ashktorab'}]\", 'abstract': '<p>Background: The Coronavirus disease 2019 (COVID-19) pandemic has had a significant global impact and Canada has seen significant morbidity and mortality nationwide. While public health interventions and provincially regulated health insurance coverage have been credited with minimizing transmission rates in Canada relative to neighboring countries, disease presentation has been presumed to be the same. \\nAim: We sought to determine factors associated with differences in gastrointestinal outcomes in COVID-19 patients at a Canadian hospital.\\nMethods: We collected data from 192 hospital records of COVID-19 patients across seven Hamilton Health Sciences hospitals, a network of academic health centers located in Hamilton, Ontario serving one of the largest metropolitan areas in Canada. Statistical and correlative analysis of symptoms, comorbidities, and mortality were performed.\\nResults: There were 192 patients. The mean age was 57.6 years (SD=21.0). For patients who died (n=27, 14%), mean age was 79.2 years old (SD=10.6) versus 54 years for survivors (SD=20.1). There was a higher mortality among patients with older age (p=0.000), long hospital stay (p=0.004), male patients (p=0.032), and patients in nursing homes (p=0.000). Patients with dyspnea (p=0.028) and hypertension (p=0.004) were more likely to have a poor outcome.  Laboratory test values that were significant in determining outcomes were an elevated INR (p=0.007) and elevated creatinine (p=0.000). Cough and hypertension were the most common symptom and comorbidity, respectively. Diarrhea was the most prevalent (14.5%) gastrointestinal symptom. Impaired liver function was related to negative outcome (LR 5.6; p=0.018). \\nConclusion: In a Canadian cohort, elevated liver enzymes, prolonged INR and elevated creatinine were associated with poor prognosis. Hypertension was also linked to a higher likelihood of negative outcome.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466382', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Impact of antibiotic perturbation on intestinal viral communities in mice', 'authors': \"[{'name': 'Jacqueline Moltzau Anderson'}, {'name': 'Tim Lachnit'}, {'name': 'Simone Lipinski'}, {'name': 'Maren Falk-Paulsen'}, {'name': 'Philip Rosenstiel'}]\", 'abstract': '<p>Viruses and bacteriophages have a strong impact on intestinal barrier function and the composition and functional properties of commensal bacterial communities. To improve our understanding of the role of the enteric intestinal virome, we longitudinally characterized the virome in fecal samples from wild-type (WT) C57BL/6J and knock-out (KO) NOD2 mice in response to an antibiotic perturbation. Sequencing of viral-like-particles (VLPs) demonstrated both a high diversity and high inter-individual variation of the murine gut virome composed of eukaryotic viruses and bacteriophages. Antibiotics also had a significant impact on the gut murine virome causing a delayed resilience independent of genotype. However, compositional shifts in the virome and bacteriome were highly correlated, indicating a possible role of the virome in inflammatory bowel disease. Bacteriophage species may be playing an important role in either upregulating or downregulating the bacterial community, and restoring a healthy virome may therefore be a central goal of microbiota-targeted therapies.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466339', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Emergent phases of ecological diversity and dynamics mapped in microcosms', 'authors': \"[{'name': 'Jiliang Hu'}, {'name': 'Daniel R. Amor'}, {'name': 'Matthieu Barbier'}, {'name': 'Guy Bunin'}, {'name': 'Jeff Gore'}]\", 'abstract': '<p>Natural ecological communities display striking features, such as high biodiversity and a wide range of dynamics, that have been difficult to explain in a unified framework. Using experimental bacterial microcosms, we perform the first direct test of recent complex systems theory predicting that simple aggregate parameters dictate emergent behaviors of the community. As either the number of species or the strength of species interactions is increased, we show that microbial ecosystems transition between distinct qualitative dynamical phases in the predicted order, from a stable equilibrium where all species coexist, to partial coexistence, to emergence of persistent fluctuations in species abundance. Under the same conditions, high biodiversity and fluctuations allow and require each other. Our results demonstrate predictable emergent diversity and dynamics in ecological communities.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466346', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'inPOSE: a flexible toolbox for chromosomal cloning and amplification of bacterial transgenes', 'authors': \"[{'name': 'Ranti Dev Shukla'}, {'name': 'Agnes Zvara'}, {'name': 'Akos Avramucz'}, {'name': 'Alyona Biketova'}, {'name': 'Akos Nyerges'}, {'name': 'Laszlo G. Puskas'}, {'name': 'Tamas Feher'}]\", 'abstract': '<p>Cloning genes and operons encoding heterologous functions in bacterial hosts is almost exclusively carried out today using plasmid vectors. This has multiple drawbacks, including the need for constant selection and the variation in copy numbers. Chromosomal integration of transgenes has always offered a viable alternative, however, to date it has been of limited use due to its tedious nature and to being limited often to a single copy. We introduce here a strategy that uses bacterial insertion sequences, the simplest autonomous transposable elements to insert and amplify genetic cargo into a bacterial chromosome. Transgene insertion can take place either as transposition or homologous recombination, and copy-number amplification is achieved using controlled copy-paste transposition. We display successful use of IS<italic>1</italic> and IS<italic>3 </italic>for this purpose in <italic>Escherichia coli</italic> cells, using various selection markers.  We demonstrate the insertion of selectable genes, an unselectable gene, and a five-gene operon in up to two copies in a single step. We continue with the amplification of the inserted cassette to double-digit copy numbers within two rounds of transposase induction and selection. Finally, we analyze the stability of the cloned genetic constructs in the lack of selection, and find it to be superior to all investigated plasmid-based systems. Due to the ubiquitous nature of transposable elements we believe that with proper design, this strategy can be adapted to numerous further bacterial species.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466358', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'A phosphoregulated RhoGEF feedback loop tunes cortical flow driven amoeboid migration in vivo', 'authors': \"[{'name': 'Benjamin Lin'}, {'name': 'Jonathan Luo'}, {'name': 'Ruth Lehmann'}]\", 'abstract': '<p>Cortical flow driven amoeboid migration utilizes friction from retrograde cortical actin flow to generate motion. Many cell types, including cancer cells, can assemble a cortical flow engine to migrate under confinement and low adhesion in vitro, but it remains unclear whether this engine is endogenously utilized in vivo. Moreover, in the context of a changing environment, it is not known how upstream regulation can set in motion and sustain a mutual feedback between flow and polarity. Here, we establish that Drosophila primordial germ cells (PGCs) utilize cortical flow driven amoeboid migration and that flows are oriented by external cues during developmental homing in vivo. The molecular basis of flow modulation is a phosphoregulated feedback loop involving RhoGEF2, a microtubule plus-end tracking RhoA specific RhoGEF, enriched at the rear of PGCs. RhoGEF2 depletion slows and disorganizes cortical flow, reducing migration speed, while RhoGEF2 activation accelerates cortical flow, thereby augmenting myosin II polarity and migration speed. Both perturbations impair PGC pathfinding, suggesting cortical flows must be tuned for accurate guidance. We surprisingly find that RhoGEF2 polarity and activation are independent of upstream canonical G12/13 signaling. Instead, its PDZ domain and conserved RhoA binding residues in its PH domain are required to establish a positive feedback loop that augments its basal activity. Upstream regulation of this feedback loop occurs via AMPK dependent multisite phosphorylation near a conserved EB1 binding SxIP motif, which releases RhoGEF2 from EB1 dependent inhibition. Thus, we reveal cortical flows as versatile, tunable engines for directed amoeboid migration in vivo.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466240', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'FITTING OF HYPERELASTIC CONSTITUTIVE MODELS IN DIFFERENT SHEEP HEART REGIONS BASED ON BIAXIAL MECHANICAL PROPERTIES', 'authors': \"[{'name': 'Fulufhelo Nemavhola'}, {'name': 'Thanyani Pandelani'}, {'name': 'Harry Ngwangwa'}]\", 'abstract': '<p>Heart failure remains one of the leading causes of death especially among people over the age of 60 years worldwide. To develop effective therapy and suitable replacement materials for the heart muscle it is necessary to understand its biomechanical behaviour under load. This paper investigates the passive mechanical response of the sheep myocardia excised from three different regions of the heart. Due to the relatively higher cost and huge ethical demands in acquisition and testing of real animal heart models, this paper evaluates the fitting performances of five different constitutive models on the myocardial tissue responses. Ten sheep were sacrificed, and their hearts excised and transported within 3h to the testing biomechanical laboratory. The upper sections of the hearts above the short axes were carefully dissected out. Tissues were dissected from the mid-sections of the left ventricle, mid-wall and right ventricle for each heart. The epicardia and endocardia were then carefully sliced off each tissue to leave the myocardia. Stress-strain curves were calculated, filtered and resampled. The results show that Choi-Vito model was found to provide the best fit to the LV, the polynomial (Anisotropic) model to RV, the Four-Fiber Family model to RV, Holzapfel (2000) to RV, Holzapfel (2005) to RV and the Fung model to LV.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466502', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Hidden pathway for cytokine receptor activation: Structural insights into a marine sponge-derived lectin that activates the thrombopoietin receptor via recognition of the fucose moiety', 'authors': \"[{'name': 'Hiromi Watari'}, {'name': 'Hiromu Kageyama'}, {'name': 'Nami Masubuchi'}, {'name': 'Hiroya Nakajima'}, {'name': 'Kako Onodera'}, {'name': 'Pamela J Focia'}, {'name': 'Takumi Oshiro'}, {'name': 'Takashi Matsui'}, {'name': 'Yoshio Kodera'}, {'name': 'Tomohisa Ogawa'}, {'name': 'Takeshi Yokoyama'}, {'name': 'Makoto Hirayama'}, {'name': 'Kanji Hori'}, {'name': 'Douglas M Freymann'}, {'name': 'Norio Komatsu'}, {'name': 'Marito Araki'}, {'name': 'Yoshikazu Tanaka'}, {'name': 'Ryuichi Sakai'}]\", 'abstract': '<p>N-glycan-mediated activation of the thrombopoietin receptor (MPL) under pathological conditions has been implicated in myeloproliferative neoplasms induced by mutant calreticulin, which forms an endogenous receptor-agonist complex that constitutively activates the receptor. However, the molecular basis for this mechanism remains unstudied because no external agonist has been discovered. Here, we describe the structure and function of a marine sponge-derived MPL agonist, thrombocorticin (ThC). ThC-induced activation persists due to limited receptor internalization. Strong synergy between ThC and thrombopoietin suggests that ThC catalyzes the formation of receptor dimers on the cell surface. We show that MPL is subject to sugar-mediated activation and that lectin-mediated activation kinetics differ from cytokine-mediated activation kinetics. Our data demonstrated the potential of lectins to provide deeper insight into human pathogenesis.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466232', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Mandrake: visualising microbial population structure by embedding millions of genomes into a low-dimensional representation', 'authors': \"[{'name': 'John A Lees'}, {'name': 'Gerry Tonkin-Hill'}, {'name': 'Zhirong Yang'}, {'name': 'Jukka Corander'}]\", 'abstract': '<p>In less than a decade, population genomics of microbes has progressed from the effort of sequencing dozens of strains to thousands, or even tens of thousands of strains in a single study. There are now hundreds of thousands of genomes available even for a single bacterial species and the number of genomes is expected to continue to increase at an accelerated pace given the advances in sequencing technology and widespread genomic surveillance initiatives. This explosion of data calls for innovative methods to enable rapid exploration of the structure of a population based on different data modalities, such as multiple sequence alignments, assemblies and estimates of gene content across different genomes. Here we present Mandrake, an efficient implementation of a dimensional reduction method tailored for the needs of large-scale population genomics. Mandrake is capable of visualising population structure from millions of whole genomes and we illustrate its usefulness with several data sets representing major pathogens. Our method is freely available both as an analysis pipeline (https://github.com/johnlees/mandrake) and as a browser-based interactive application (https://gtonkinhill.github.io/mandrake-web/).</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466404', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'In vitro reconstituted quorum sensing pathways enable rapid evaluation of quorum sensing inhibitors', 'authors': \"[{'name': 'Dingchen Yu'}]\", 'abstract': '<p>Quorum sensing, as inner- or inter-species microbial communication process orchestrated by diffusible autoinducers, typically results in collective pathogenic behaviours, being recognized as a promising druggable target for anti-virulence. Here, we reconstituted las and rhl quorum sensing pathways of Pseudomonas aeruginosa, mediated by acyl-homoserine lactones (AHLs) and LuxI/LuxR-family proteins, with fluorescence output in Escherichia coli cell-free expression system, offering a platform to rapidly evaluate quorum sensing inhibitors (QSIs) in vitro. Previously reported small-molecule quorum sensing inhibitors for interfering with P. aeruginosa quorum sensing systems were tested and showed mild to high on-target inhibition as well as off-target toxicity. Of note, quercetin displayed potent on-target inhibition to quorum sensing pathways as well as acceptable off-target toxicity to cell-free expression machinery. Upon our work, cell-free platform is anticipated to further facilitate automated and high-throughput drug screening, bridge in silico and in vivo drug-screening methods, and accelerate the upgrading of antimicrobial arsenal.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466261', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Longitudinal to transverse metachronal wave transitions in an in-vitro model of ciliated bronchial epithelium', 'authors': '[{\\'name\\': \\'Olivier Mesdjian\\'}, {\\'name\\': \\'Chenglei Wang\\'}, {\\'name\\': \\'Simon Gsell\\'}, {\\'name\\': \"Umberto D\\'Ortona\"}, {\\'name\\': \\'Julien Favier\\'}, {\\'name\\': \\'Annie Viallat\\'}, {\\'name\\': \\'Etienne Loiseau\\'}]', 'abstract': '<p>Myriads of cilia beat on ciliated epithelia, which are ubiquitous in life. When ciliary beats are synchronized, metachronal waves emerge, whose direction of propagation depends on the living system in an unexplained way. We show on a reconstructed human bronchial epithelium in-vitro that the direction of propagation is determined by the ability of mucus to be transported at the epithelial surface. Numerical simulations show that longitudinal waves maximise the transport of mucus while transverse waves, observed when the mucus is rigid and still, minimize the energy dissipated by the cilia.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.25.464584', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Molecular identification of rhizobacterial isolates from Prosopis limensis and their effect on the growth of Raphanus sativus under salt stress', 'authors': \"[{'name': 'Rene Flores Clavo'}, {'name': 'Esteban A Valladolid Suyn'}, {'name': 'Karin Reinoza Farroan'}, {'name': 'Cristian Asmat Ortega'}, {'name': 'Gabriel Zuiga Valdera'}, {'name': 'Fabiana Fantinatti Garboggini'}, {'name': 'Sebastian Iglesias-Osores'}, {'name': 'Carmen Rosa Carreo Farfn'}]\", 'abstract': '<p>Soil salinity negatively affects the development of agricultural crops. The utilization of plant growth-promoting rhizobacteria is a biotechnological alternative to mitigate this problem. Rhizobacteria were isolated from the roots and rhizosphere of <italic>Prosopis limensis</italic> Bentham carob to identify them and determine their potential as plant growth-promoters under salt stress. First, ACC deaminase activity was determined in Dworkin &amp; Foster (DF) minimum medium with 3.0 mM ACC as a source of nitrogen; besides, tolerance to sodium chloride was determined in Nutrient Broth (NB) supplemented with 10% NaCI. Then, bacteria displaying ACC deaminase activity and a 10% NaCI tolerance were selected, identified through genomic analysis targeting the 16s rRNA gene, and inoculated into <italic>Raphanus sativus</italic> L. radish seeds to determine the effect on germination percentage under salt stress (80 mM NaCI) and, also on indole production and phosphate solubilization. Isolates were also utilized to evaluate their effect on the growth of radish in saline soils. Genomic analysis showed two bacterial isolates from the genus <italic>Pseudomonas</italic> and one from the genus <italic>Bordetella</italic>: Isolate MW604823 was identified as <italic>Pseudomonas</italic> sp.; isolate MW604824, as <italic>Pseudomonas hunanensis;</italic> and isolate MW604826 as <italic>Bordetella muralis</italic>. Thee isolates demonstrated ACC deaminase activity and tolerance to 10% NaCI. Inoculation of the isolates on radish seeds increased germination percentage compared to the control. The isolates displayed <italic>in vitro</italic> indole production and phosphate solubilization capacity. Moreover, the isolates promoted the growth of radish under salt stress conditions, increasing, leave number, root number, aerial, and root biomass, demonstrating their potential as a biofertilizers.</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.29.465023', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Mispatterned motile cilia beating causes flow blockage in the epileptic brain', 'authors': \"[{'name': 'Regina J Faubel'}, {'name': 'Tim N Feinstein'}, {'name': 'Veronica Santos Canellas'}, {'name': 'Jenna Gaesser'}, {'name': 'Nancy H Beluk'}, {'name': 'Yong Wang'}, {'name': 'Maya Yankova'}, {'name': 'Stephen M King'}, {'name': 'Kalyani Bindu Karunakaran'}, {'name': 'Madhavi K Ganapathiraju'}, {'name': 'Cecilia Wenya Lo'}]\", 'abstract': '<p>Beating of motile cilia at the brain ventricular surface generates rapid flow in an evolutionary conserved pattern mediating the transport of cerebrospinal fluid, but its functional importance has yet to be demonstrated. Here we show disturbance of this transport may contribute to seizure susceptibility. Mice haploinsufficient for FoxJ1, transcription factor regulating motile cilia exhibited cilia-driven flow blockage and increased seizure susceptibility. Mutations in two epilepsy-associated kinases, Cdkl5 and Yes1, in mice resulted in similar cilia-driven flow blockage and increased seizure susceptibility. We showed this arises from disorganized cilia polarity associated with disruption in the highly organized basal body anchoring meshwork. Together these findings suggest mispatterning of cilia-generated flow may contribute to epilepsy and thus might account for seizures unresponsive to current seizure medications.</p>', 'posted': '2021-10-30'}, {'doi': '10.1101/2021.10.24.21265348', 'tweets': 4, 'archive': 'medRxiv', 'subject-area': 'Cardiovascular Medicine', 'covid': False, 'title': 'Factors Associated with Medicine Timing Effects: A Meta-analysis', 'authors': \"[{'name': 'Marc D. Ruben'}, {'name': 'Lauren J. Francey'}, {'name': 'Gang Wu'}, {'name': 'David F. Smith'}, {'name': 'Garret A. FitzGerald'}, {'name': 'Jacob J. Hughey'}, {'name': 'John B. Hogenesch'}]\", 'abstract': '<sec><title>Importance</title><p>Clinical evidence suggests that the time of day of treatment can affect outcomes in many different diseases, but this information is dispersed, imprecise, and heterogeneous. Consequently, practice guidelines and clinical care recommendations seldom specify intervention time.</p></sec><sec><title>Objective</title><p>To understand the sources of variability and summarize clinical findings on the time of day effects of medicine.</p></sec><sec><title>Data Sources</title><p>A systematic search of Pubmed, Google Scholar, and <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://ClinicalTrials.gov\">ClinicalTrials.gov</ext-link> for chronotherapy OR time of administration.</p></sec><sec><title>Study Selection</title><p>Any clinical study since 2000, randomized or observational, that compared the effects of treatment at different times of day. We included pharmacologic or surgical interventions having at least one continuous outcome.</p></sec><sec><title>Data Extraction and Synthesis</title><p>For selected studies, we extracted the mean and variance of each time-of-day treatment group. From these, we computed the standardized mean difference (SMD) as the measure of timing effect. Where a study reported multiple outcomes, we selected a single outcome based on a defined order of priority.</p></sec><sec><title>Main Outcomes and Measures</title><p>We estimated overall pooled effect size and heterogeneity by a random effects model, followed by outlier detection and subgroup analyses to evaluate how study factors, including drug, design, outcome, and source, associate with timing effect.</p></sec><sec><title>Results</title><p>78 studies met the inclusion criteria, comprising 48 distinct interventions over many therapeutic areas. We found an overall effect of time on clinical outcomes but with substantial heterogeneity between studies. Predicted effects range from none to large depending on the study context. Study size, registration status, and source are associated with the magnitude of effect. Larger trials and those that were pre-registered have markedly smaller effects, suggesting that the published record overstates the effects of the timing of medicine on clinical outcomes. In particular, the notion that antihypertensives are more effective if taken at bedtime draws disproportionately from one source in the field, which consistently detects larger effects than the community average. Lastly, among the most highly studied drug timing relationships, aspirins anti-clotting effect stands out, consistently favoring evening over morning dosing.</p></sec><sec><title>Conclusions and Relevance</title><p>While accounts of drug timing effects have focused on <italic>yes/no</italic>, appreciating the range of probable effects may help clarify where circadian medicine meets the threshold for clinical benefit.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466136', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Differential patterns of change in brain connectivity resulting from traumatic brain injury', 'authors': \"[{'name': 'Johan Nakuci'}, {'name': 'Matthew McGuire'}, {'name': 'Ferdinand Schweser'}, {'name': 'David Poulsen'}, {'name': 'Sarah F. Muldoon'}]\", 'abstract': '<sec><title>Background</title><p>Traumatic brain injury (TBI) damages white matter tracts, disrupting brain network structure and communication. There exists a wide heterogeneity in the pattern of structural damage associated with injury, as well as a large heterogeneity in behavioral outcomes. However, little is known about the relationship between changes in network connectivity and clinical outcomes.</p></sec><sec><title>Methods</title><p>We utilize the rat lateral fluid-percussion injury (FPI) model of severe TBI to study differences in brain connectivity in 8 animals that received the insult and 11 animals that received only a craniectomy. Diffusion Tensor Imaging (DTI) is performed 5 weeks after the injury and network theory is used to investigate changes in white matter connectivity.</p></sec><sec><title>Results</title><p>We find that 1) global network measures are not able to distinguish between healthy and injured animals; 2) injury induced alterations predominantly exist in a subset of connections (subnetworks) distributed throughout the brain; and 3) injured animals can be divided into subgroups based on changes in network motifs  measures of local structural connectivity. Additionally, alterations in predicted functional connectivity indicate that the subgroups have different propensities to synchronize brain activity, which could relate to the heterogeneity of clinical outcomes such as the risk of developing post-traumatic epilepsy.</p></sec><sec><title>Discussion</title><p>These results suggest that network measures can be used to quantify progressive changes in brain connectivity due to injury and differentiate among subpopulations with similar injuries but different pathological trajectories.</p></sec><sec><title>Impact Statement</title><p>White matter tracts are important for efficient communication between brain regions and their connectivity pattern underlies proper brain function. Traumatic brain injury (TBI) damages white matter tracts and changes brain connectivity, but how specific changes relate to differences in clinical/behavioral outcomes is not known. Using network theory to study injury related changes in structural connectivity, we find that local measures of network structure can identify subgroups of injured rats with different types of changes in brain structure. Our results suggest that these different patterns of change could relate to differences in clinical outcomes such as the propensity to develop epilepsy.</p></sec>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.27.466094', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Formation and recycling of an active epigenetic mark mediated by cell cycle-specific RNAs', 'authors': \"[{'name': 'Alexander K. Ebralidze'}, {'name': 'Simone Ummarino'}, {'name': 'Mahmoud A. Bassal'}, {'name': 'Haoran Zhang'}, {'name': 'Bogdan Budnik'}, {'name': 'Emanuele Monteleone'}, {'name': 'Dennis Kappei'}, {'name': 'Yanjing V. Liu'}, {'name': 'Danielle E. Tenen'}, {'name': 'Rory Coffey'}, {'name': 'Mee Rie Sheen'}, {'name': 'Yanzhou Zhang'}, {'name': 'Anas Wanet'}, {'name': 'Bon Q. Trinh'}, {'name': 'Valeria Poli'}, {'name': 'Vladimir Espinosa Angarica'}, {'name': 'Roberto Tirado Magallanes'}, {'name': 'Touati Benoukraf'}, {'name': 'Colyn Crane-Robinson'}, {'name': 'Annalisa Di Ruscio'}, {'name': 'Daniel G. Tenen'}]\", 'abstract': '<p>The mechanisms by which epigenetic modifications are established in gene regulatory regions of active genes remain poorly understood. The data presented show that the establishment and recycling of a major epigenetic mark, the acetylated form of the replacement histone H2A.Z, is regulated by cell cycle-specific long noncoding RNAs encoded in regions adjacent to the promoters of active genes. These transcripts, termed <italic>SPEARs</italic> (S Phase EArly RNAs), are induced in early S phase: their expression precedes that of the downstream genes on which they exert their regulatory action. <italic>SPEARs</italic> drive the modification and deposition of the acetylated form of histone H2A.Z by bringing together the replacement histone and the histone acetyl transferase TIP60. This widespread bimodal pathway constitutes a novel RNA-mediated mechanism for the establishment of epigenetic marks and cell-specific epigenetic profiles, thereby providing a unifying explanation for the accuracy and persistence of epigenetic marks on chromatin.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.27.466056', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Automated hyperspectral vegetation index derivation using a hyperparameter optimization framework for high-throughput plant phenotyping', 'authors': \"[{'name': 'Joshua C.O. Koh'}, {'name': 'Bikram P. Banerjee'}, {'name': 'German Spangenberg'}, {'name': 'Surya Kant'}]\", 'abstract': '<p><list list-type=\"bullet\"><list-item><p>Hyperspectral vegetation indices (VIs) are widely deployed in agriculture remote sensing and plant phenotyping to estimate plant biophysical and biochemical traits. However, existing VIs consist mainly of simple 2-band indices which limits the net performance and often do not generalize well for other traits than they were originally designed for.</p></list-item><list-item><p>We present an automated hyperspectral vegetation index (AutoVI) system for the rapid generation of novel 2- to 6-band trait-specific indices in a streamlined process covering model selection, optimization and evaluation driven by the tree parzen estimator algorithm. Its performance was tested in generating novel indices to estimate chlorophyll and sugar contents in wheat.</p></list-item><list-item><p>Results show that AutoVI can rapidly generate complex novel VIs (4-band index) which correlated strongly (R<sup>2</sup> &gt; 0.8) with measured chlorophyll and sugar contents in wheat. AutoVI-derived indices were used as features in simple and stepwise multiple linear regression for chlorophyll and sugar content estimation, and outperformed results achieved with existing 47 VIs and those provided by partial least squares regression.</p></list-item><list-item><p>The AutoVI system can deliver novel trait-specific VIs readily adoptable in high-throughput plant phenotyping platforms and should appeal to plant scientists and breeders. A graphical user interface of AutoVI is herein provided.</p></list-item></list></p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.26.465931', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'A system for CBAF reconstitution reveals roles for BAF47 domains and BCL7 in nucleosome ejection', 'authors': \"[{'name': 'Timothy S. Mulvihill'}, {'name': 'Mary L. Nelson'}, {'name': 'Naveen Verma'}, {'name': 'Kevin B. Jones'}, {'name': 'Bradley R. Cairns'}]\", 'abstract': '<p>Canonical BAF (CBAF) is an essential 12-protein chromatin-remodeling complex that slides and/or ejects nucleosomes using the alternative catalytic ATP-dependent DNA translocases BRG1 or BRM. Currently, the regulation of BRG1/BRM activity and nucleosome ejection remain incompletely understood. To address this, we developed a system for full CBAF reconstitution and purification, and created a novel nucleosome ejection assay. ARID1A and DPF2 were dispensable for assembly and chromatin remodeling activity, contrasting with prior work. The actin-related protein BAF53A and -actin components interacted and enhanced DNA translocation, and were required for BCL7A incorporation, which potentiated ejection. BAF47 also regulated ejection, utilizing two stimulatory domains and an autoinhibitory domain. Finally, we provide evidence for direct nucleosome ejection at low nucleosome density on closed circular arrays. Taken together, we provide powerful new tools for CBAF mechanistic investigation and reveal new roles for several CBAF components.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.27.466161', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'NON-SPECIFIC EFFECTS OF A CINNAMATE-4-HYDROXYLASE INHIBITOR ON AUXIN HOMEOSTASIS', 'authors': \"[{'name': 'Ilias El Houari'}, {'name': 'Petr Klma'}, {'name': 'Alexandra Baekelandt'}, {'name': 'Paul Staswick'}, {'name': 'Veselina Uzunova'}, {'name': 'Charo I. Del Genio'}, {'name': 'Ward Steenackers'}, {'name': 'Petre I. Dobrev'}, {'name': 'Ondrej Novk'}, {'name': 'Richard Napier'}, {'name': 'Jan Petrek'}, {'name': 'Dirk Inz'}, {'name': 'Wout Boerjan'}, {'name': 'Bartel Vanholme'}]\", 'abstract': '<p>Chemical inhibitors are often implemented for the functional characterization of genes to overcome the limitations associated with genetic approaches. Although being a powerful tool, off-target effects of these inhibitors are easily overlooked in a complex biological setting. Here we illustrate the implications of such secondary effects by focusing on piperonylic acid (PA), an inhibitor of CINNAMATE-4-HYDROXYLASE (C4H) that is often used to investigate the involvement of lignin during plant growth and development. When supplied to plants, we found that PA is recognized as a substrate by GRETCHEN HAGEN 3.6 (GH3.6), an amido synthetase involved in the formation of the auxin catabolite indole-3-acetic acid (IAA)-Asp. By competing for the same enzyme, PA interferes with auxin conjugation, resulting in an increase in cellular auxin concentrations. These increased auxin levels likely further contribute to an increase in adventitious rooting previously observed upon PA-treatment. Despite the focus on GH3.6 in this report, PA is conjugated by an array of enzymes and their subsequent reduced activity on native substrates could potentially affect a whole set of physiological processes in the plant. We conclude that surrogate occupation of the endogenous conjugation machinery in the plant by exogenous compounds is likely a more general phenomenon that is rarely considered in pharmacological studies. Our results hereby provide an important basis for future reference in studies using chemical inhibitors.</p>', 'posted': '2021-10-28'}, {'doi': '10.1101/2021.10.27.466182', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Efficacy of anti-microbial gel vapours against aerosolised coronavirus, bacteria, and fungi', 'authors': \"[{'name': 'Parthasarathi Kalaiselvan'}, {'name': 'Muhammad Yasir'}, {'name': 'Mark D. P. Willcox'}, {'name': 'Ajay Kumar Vijay'}]\", 'abstract': '<p>Background: The urban population spends up to 90% of their time indoors. The indoor environment harbours a diverse microbial population including viruses, bacteria, and fungi. Pathogens present in the indoor environment can be transmitted to humans through aerosols. Aim: This study evaluated the efficacy of an antimicrobial gel containing a mix of essential oils against aerosols of bacteria, fungi, and coronavirus. Methods: The antimicrobial gel was allowed to vapourize inside a glass chamber for 10 or 20 minutes. Microbial aerosols of Escerichia coli, Aspergillus flavus spores or murine hepatitis virus MHV 1, a surrogate of SARS CoV-2 was passed through the gel vapours and then collected on a 6-stage Andersen sampler. The number of viable microbes present in the aerosols collected in the different stages were enumerated and compared to number of viable microbes in control microbial aerosols that were not exposed to the gel vapours. Results: Vaporizing the antimicrobial gel for 10 and 20 minutes resulted in a 48% (p = 0.002 Vs. control) and 53% (p = 0.001 Vs. control) reduction in the number of MHV-1 in the aerosols, respectively. The antimicrobial gel vaporised for 10 minutes, reduced the number of viable E. coli by 51% (p = 0.032 Vs. control) and Aspergillus flavus spores by 72% (p=0.008 Vs. control) in the aerosols. Conclusions: The antimicrobial gel may be able to reduce aerosol transmission of microbes. Keywords: Aerosol, Coronavirus, SARS-CoV-2, Indoor Air, Antimicrobial, Essential oils</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.465406', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Application of Cas12j for Streptomyces editing and cluster activation', 'authors': \"[{'name': 'Lee Ling Tan'}, {'name': 'Elena Heng'}, {'name': 'Nadiah Zulkarnain'}, {'name': 'Chuang Yan Leong'}, {'name': 'Veronica Ng'}, {'name': 'Lay Kien Yang'}, {'name': 'Deborah Chwee San Seow'}, {'name': 'Guangrong Peh'}, {'name': 'Yee Hwee Lim'}, {'name': 'Koduru Lokanand'}, {'name': 'Yoganathan Kanagasundaram'}, {'name': 'Siew Bee Ng'}, {'name': 'Fong Tian Wong'}]\", 'abstract': \"<p>In recent years, CRISPR-Cas toolboxes for Streptomyces editing have rapidly accelerated natural product discovery and engineering. However, Cas efficiencies are also oftentimes strain dependent, subsequently a variety of Cas proteins would allow for flexibility and enable genetic manipulation within a wider range of Streptomyces strains. In this work, we have further expanded the Cas toolbox by presenting the first example of Cas12j mediated editing in Streptomyces sp. A34053. In our study, we have also observed significantly improved editing efficiencies with Acidaminococcus sp. Cas12j compared to Cas12a, Francisella tularensis subsp. novicida U112's type V-A Cas (FnCpf1).</p>\", 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.23.21265407', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Occupational and Environmental Health', 'covid': False, 'title': 'Relationship among worktreatment balance, job stress, and work engagement in Japan: A cross-sectional study', 'authors': \"[{'name': 'Kazunori Ikegami'}, {'name': 'Hajime Ando'}, {'name': 'Hisashi Eguchi'}, {'name': 'Mayumi Tsuji'}, {'name': 'Seiichiro Tateishi'}, {'name': 'Koji Mori'}, {'name': 'Keiji Muramatsu'}, {'name': 'Yoshihisa Fujino'}, {'name': 'Akira Ogami'}, {'name': ' '}]\", 'abstract': '<p>There is a drive to support workers undergoing medical treatment who wish to continue working in Japan, known as the worktreatment balance. It is hoped that this support for the worktreatment balance could boost their mental health. This study examines the relationship among the worktreatment balance, job stress, and work engagement. This study was conducted in December 2020 in Japan, with 27,036 participants. We divided the participants into three groups by the receipt state of support for the worktreatment balance: control group (no need the support), unsupported group, and supported group. The scores of the parameters of the job content questionnaire and the Utrecht Work Engagement Scale (UWES-3) were compared among groups using a multilevel regression with age-sex or multivariate-adjusted models. In the two models, the job control score of the unsupported group was significantly lower than that of the control group. The two social support scores of the supported group were significantly higher than those of the control group. The scores of the UWES-3 of the unsupported group were significantly lower than those of the control group. The support of worktreatment balance for workers could have a positive impact on their mental health.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.23.465589', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'UHRF1, NSUN2, and NEIL1 were Detected as Clinical Biomarker Candidates in Prostate Adenocarcinoma with Potential Roles in Disease Pathogenesis', 'authors': \"[{'name': 'Buse Nur Kahyaoullar'}, {'name': 'Turan Demircan'}]\", 'abstract': '<p>Prostate cancer (PCa) is the most commonly diagnosed cancer in men. Expanding evidence suggests a significant association between cancer progression and RNA modifications. However, our knowledge of the link between m5C and hm5C pathways with PCa is limited. Therefore, we aimed to explore the diagnostic and prognostic values of m5C and hm5C regulators in PCa. In this study, genetic alterations in m5C and hm5C regulators were identified using publicly available databases. Differentially expressed genes in these pathways between tumor and nontumor samples, correlation among m5C and hm5C pathway members, and prognostic value of the regulators were evaluated. Furthermore, enrichment of gene ontology (GO) terms and KEGG pathways was carried out. Obtained results unveiled the mRNA level differences as the key genetic alterations for m5C and hm5C regulators between tumor and nontumor samples. <italic>UHRF1, TET3,</italic> and <italic>NEIL1</italic> were significantly upregulated in tumor samples compared to nontumor ones, whereas <italic>EGR1</italic> was significantly downregulated. <italic>UHRF1, DNMT1, NSUN2, NSUN4, C1orf77, C3orf37, WDR77, NEIL1</italic>, and <italic>TDG</italic> genes were identified as candidate prognostic markers of overall survival. The upregulated genes in patient samples with genetic alterations in m5C and hm5C pathways enriched cell cycle-related processes. In summary, our findings suggest that the m5C and hm5C regulators might play a role in PRAD development by activation of proliferation, and the <italic>UHRF1, NSUN2</italic>, and <italic>NEIL1</italic> genes have the potential to be utilized as clinical biomarkers.</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.23.465580', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Identification and assessment of the antibacterial activity of ZnO nanoparticles produced by Vigna mungo and Rhizobacteria', 'authors': \"[{'name': 'Bharat Kwatra'}]\", 'abstract': '<label>1.</label><title>ABSTRACTS</title><p>ZnO nanoparticles have received a lot of interest in recent years due to their unusual features. Antimicrobial properties of ZnO NPs However, the qualities of nanoparticles are determined by their size and form, making them suitable for a variety of applications. The current work looks at the synthesis, characterization, and antibacterial activity of ZnO NPs produced by Vigna Mungo and Rhizobacteria. Rhizobacteria isolated from V. mungo root nodule were morphologically, biochemically, and molecularly examined and identified as Rhizobium sp. strain P4 and Bacillus flexus strain IFO15715. The GC-MS analysis of methanol leaf extract of V. mungo was performed to detect and identify bioactive chemicals, and this indicated phytol as an antibacterial agent, while Squalene and Alpha  tocopherol had antioxidant and anti-tumour properties. Agar well diffusion experiment was used to determine the antibacterial properties of ZnO nanoparticles and Vigna Mungo leaf extract. This approach is widely documented, and standard zones of inhibition for sensitive and resistant values have been defined. The results demonstrated that both methanol extract and zinc oxide nanoparticles have strong antibacterial efficacy against the majority of the pathogens examined. he synthesized nanoparticles from Rhizobium sp. were characterized by analytical techniques like SEM, XRD, FTIR, and UV-Vis.</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.23.465424', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Infants attention to eyes is an independent, heritable trait predicting later verbal competence', 'authors': \"[{'name': 'Charlotte Viktorsson'}, {'name': 'Ana Maria Portugal'}, {'name': 'Danyang Li'}, {'name': 'Maja Rudling'}, {'name': 'Monica Siqueiros Sanchez'}, {'name': 'Kristiina Tammimies'}, {'name': 'Mark J. Taylor'}, {'name': 'Angelica Ronald'}, {'name': 'Terje Falck-Ytter'}]\", 'abstract': '<p>From birth, infants orient preferentially to faces, and when looking at the face, they attend primarily to eyes and mouth  two areas that convey different types of information. Here, in a sample of 535 5-month-old infant twins, we assessed eye (relative to mouth) preference in early infancy. Eye preference was independent from all other concurrent traits measured, and had a moderate-to-high contribution from genetic influences (A = .57; 95% CI: .45, .66). Preference for eyes over mouth at 5 months predicted higher parent ratings of verbal competence in toddlerhood, but did not predict autistic traits. These results suggest that variation in eye looking reflects a type of biological niche picking emerging before infants can select their environments by other means (crawling or walking).</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.22.464858', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Conjugative transfer of naturally occurring plasmids in Mycolicibacterium sp', 'authors': \"[{'name': 'Sergio Morgado'}, {'name': 'Ana Carolina Vicente'}]\", 'abstract': '<p>Conjugation is considered the main horizontal gene transfer (HGT) mechanism in bacterial adaptation and evolution. In the <italic>Mycobacteriaceae</italic> family, <italic>Mycolicibacterium smegmatis</italic> has been used as the model organism for the conjugative transfer of hybrid plasmids. However, the natural conjugation process in any bacteria would involve the transfer of naturally occurring plasmids. Currently, there is a gap in this regard in relation to this abundant environmental genus of <italic>Mycobacteriaceae</italic>. Here, we performed conjugation experiments between wild <italic>Mycolicibacterium</italic> sp. strains involving naturally occurring plasmids (sizes of 21 and 274 kb), and interestingly, evidence of conjugative transfer was obtained. Thus, it is likely that conjugation occurs in <italic>Mycolicibacterium</italic> in the natural environment, representing a source of diversification and evolution in this genus of bacteria.</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.25.21265475', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Health Systems and Quality Improvement', 'covid': True, 'title': 'Optimising the balance of acute and intermediate care capacity for the complex discharge pathway: computer modelling study during COVID-19 recovery in England', 'authors': \"[{'name': 'Z Onen-Dumlu'}, {'name': 'AL Harper'}, {'name': 'PG Forte'}, {'name': 'AL Powell'}, {'name': 'M Pitt'}, {'name': 'C Vasilakis'}, {'name': 'RM Wood'}]\", 'abstract': '<sec><title>Objectives</title><p>While there has been significant research on the pressures facing acute hospitals during the COVID-19 pandemic, there has been less interest in downstream community services which have also been challenged in meeting demand. This study aimed to estimate the theoretical cost-optimal capacity requirement for step down intermediate care services within a major healthcare system in England, at a time when considerable uncertainty remained regarding vaccination uptake and the easing of societal restrictions.</p></sec><sec><title>Methods</title><p>Demand for intermediate care was projected using an epidemiological model (for COVID-19 demand) and regressing upon public mobility (for non-COVID-19 demand). These were inputted to a computer simulation model of patient flow from acute discharge readiness to bedded and home-based Discharge to Assess (D2A) intermediate care services. Cost-optimal capacity was defined as that which yielded the lowest total cost of intermediate care provision and corresponding acute discharge delays.</p></sec><sec><title>Results</title><p>Increased intermediate care capacity is likely to bring about lower system-level costs, with the additional D2A investment more than offset by substantial reductions in costly acute discharge delays (leading also to improved patient outcome and experience). Results suggest that completely eliminating acute bed blocking is unlikely economical (requiring large amounts of downstream capacity), and that health systems should instead target an appropriate tolerance based upon the specific characteristics of the pathway.</p></sec><sec><title>Conclusions</title><p>Computer modelling can be a valuable asset for determining optimal capacity allocation along the complex care pathway. With results supporting a Business Case for increased downstream capacity, this study demonstrates how modelling can be applied in practice and provides a blueprint for use alongside the freely-available model code.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.25.21265480', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Development of health and safety training for Vietnamese American nail salon owners and workers', 'authors': \"[{'name': 'Tran B Huynh'}, {'name': 'Duong T Nguyen'}, {'name': 'Nga Vu'}, {'name': 'Catherine Freeland'}]\", 'abstract': '<sec><title>Background</title><p>Nail salon workers are an underserved worker population that faces multiple barriers to accessing occupational health training and services. We developed a series of occupational health training modules, which were culturally tailored to Vietnamese-speaking workers, covering topics on infection control, musculoskeletal disorder prevention, chemical safety, and labor practices. We delivered the training online (due to COVID-19) to a small group of Vietnamese owners and workers in the Philadelphia metro area to obtain feedback on the training content and potential implementation challenges.</p></sec><sec><title>Methods</title><p>Seven participants (three owners and four workers) were recruited to attend the training. Qualitative feedback was obtained after each training session, followed by a more in-depth interviewer-assisted open-ended questionnaire to gain better understanding of the potential challenges of implementing the recommended changes. The Health Belief Model was used to guide the analysis of the participants responses to identify the perceived benefits and barriers of the training.</p></sec><sec><title>Results</title><p>Themes of perceived benefits of the training were bridging the gap of cosmetology school training, offering practical tips to protect their health at work, and inspiring conversations about work dignity and labor practices. Themes of perceived barriers were availability of affordable safer products and lack of resources, desire to please customers, lack of commitment from owners, and ubiquitous low wage that impacts employees job satisfaction and motivation to change, and difficulty in obtaining a work license.</p></sec><sec><title>Conclusions</title><p>Our study revealed the multitude of social and economic barriers facing immigrant nail salon owners and workers. Potential policies and strategies to overcome some of these structural barriers are discussed for the long-term health protection of nail salon workers.</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.25.21265494', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Health Systems and Quality Improvement', 'covid': True, 'title': 'Health workers Motivators to uptake of the Covid-19 vaccine at Iganga Hospital Eastern Uganda, and Mengo Hospital Kampala Uganda; A qualitative study', 'authors': \"[{'name': 'Lubega Muhamadi'}, {'name': 'Namulema Edith'}, {'name': 'Waako James'}, {'name': 'Nazarius Mbona Tumwesigye'}, {'name': 'Safinah Kisu Museene'}, {'name': 'Stefan Swartling Peterson'}, {'name': 'Anna Mia Ekstrm'}]\", 'abstract': '<p>Uganda continues extensive mobilization and administration of the Covid 19 vaccine to its people albeit some vaccine hesitancy with in the population. Amongst the health workers however, approximately 70% had received their first dose while 40% had received their second dose of the Covid-19 vaccine by September 2021 respectively. These figures represent a recognizable acceptance rate among health workers. Exploring motivators to vaccine uptake among health workers is vital for the governments general population vaccine rollout plan.</p><p>We conducted 12 focus group discussions and 20 in-depth interviews with health workers (vaccinated and unvaccinated) to understand motivators to vaccine acceptance in their own perspective in central and eastern Uganda. Reported motivators to vaccine acceptance included; risk susceptibility/protection, fear of death and/or cost of treatment and experiences of Covid related grief. Other were trust in the vaccine, call to government policy and vaccine success stories elsewhere, real or perceived benefits of vaccination and peer influence.</p><p>We recommend intensified dissemination of health worker tailor made tools/guides for information, education and communication about the Covid 19 vaccine. The tools need to emphasize the elicited themes/motivators. We also recommend use of peers who have taken up the vaccine and survived Covid-19 or got a mild form of the disease to elicit positive peer influence about the vaccine amongst health workers. The information dissemination and peer narratives could be done through the health workers social media platforms, union or association websites, personal statements, editorials or other media.</p>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.25.465697', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Uncertainty and precaution in hunting wolves twice in a year', 'authors': \"[{'name': 'Adrian Treves'}, {'name': 'Naomi X. Louchouarn'}]\", 'abstract': '<p>When humanity confronts the risk of extinction of species, many people invoke precautions, especially in the face of uncertainty. Although precautionary approaches are value judgments, the timing and effect of precautions or lack thereof are scientific questions. We investigated the Wisconsin gray wolves <italic>Canis lupus</italic> facing a second wolf-hunt in November 2021 and use three legal thresholds as the societal value judgments about precautions: (1) the 1999 population goal, 350 wolves, (2) the threshold for statutory listing under the state threatened and endangered species act, 250 wolves; and (3) state extirpation &lt;2 wolves. This allows us to explore the quantitative relationship between precaution and uncertainty. Working from estimates of the state wolf population in April 2021 and an estimate of reproduction to November, we constructed a simple linear model with uninformative priors for the period April 2021-April 2022 including an uncertain wolf-hunt in November 2021. Our first result is that the state government under-counted wolf deaths in the year preceding both wolf-hunts. This emphasizes the need for greater precaution in setting wolf-hunt quotas. We find official recommendations for a quota for the November 2021 wolf-hunt risk undesirable outcomes. Even a quota of zero has a 13% chance of crossing threshold 1. Therefore, a zero death toll would be precautionary. Proponents for high quotas bear the burden of proof that their estimates are accurate, precise, and reproducible. We show how scientists have the tools and concepts for quantifying and explaining the probabilities of crossing thresholds set by laws or other social norms. Refusal by scientists to account for uncertainty opens the door to policy-making based on opinion, anecdote, and political rivalries. We recommend instead that scientists grapple with data gaps by explaining what the uncertainty means for policy and the public including the consequences of being wrong.</p>', 'posted': '2021-10-25'}, {'doi': '10.1101/2021.10.25.465734', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'MACA: Marker-based automatic cell-type annotation for single cell expression data', 'authors': \"[{'name': 'Yang Xu'}, {'name': 'Simon J. Baumgart'}, {'name': 'Christian Stegmann'}, {'name': 'Sikander Hayat'}]\", 'abstract': '<sec><title>Summary</title><p>Accurately identifying cell-types is a critical step in single-cell sequencing analyses. Here, we present marker-based automatic cell-type annotation (MACA), a new tool for annotating single-cell transcriptomics datasets. We developed MACA by testing 4 cell-type scoring methods with 2 public cell-marker databases as reference in 6 single-cell studies. MACA compares favorably to 4 existing marker-based cell-type annotation methods in terms of accuracy and speed. We show that MACA can annotate a large single-nuclei RNA-seq study in minutes on human hearts with ~ 290k cells. MACA scales easily to large datasets and can broadly help experts to annotate cell types in single-cell transcriptomics datasets, and we envision MACA provides a new opportunity for integration and standardization of cell-type annotation across multiple datasets.</p></sec><sec><title>Availability and implementation</title><p>MACA is written in python and released under GNU General Public License v3.0. The source code is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/ImXman/MACA\">https://github.com/ImXman/MACA</ext-link>.</p></sec><sec><title>Contact</title><p>Yang Xu (<email>yxu71@vols.utk.edu</email>), Sikander Hayat (<email>hayat221@gmail.com</email>)</p></sec>', 'posted': '2021-10-26'}, {'doi': '10.1101/2021.10.28.466154', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Reversing anterior insular cortex neuronal hypoexcitability attenuates compulsive behavior in juvenile rats', 'authors': \"[{'name': 'Kshitij Jadhav'}, {'name': 'Aurelien Bernheim'}, {'name': 'Lea Aeschlimann'}, {'name': 'Guylene Kirschmann'}, {'name': 'Isabelle Decosterd'}, {'name': 'Alexander Hoffman'}, {'name': 'Carl R Lupica'}, {'name': 'Benjamin Boutrel'}]\", 'abstract': '<p>Development of self-regulatory competencies during adolescence is partially dependent on normative brain maturation. Here we report that juvenile rats as compared to adults exhibit impulsive and compulsive-like behavioral traits, the latter being associated with lower expression of mRNA levels of the immediate early gene zif268 in the anterior insula (AI). This observation suggests that deficits in AI function in juvenile rats could explain their immature pattern of interoceptive cue integration in rational decision-making and compulsive phenotype. In support of this, here we report hypoexcitability of juvenile layer-V pyramidal neurons in the AI, concomitant with reduced glutamatergic synaptic input to these cells. Chemogenetic activation of the AI attenuated the compulsive trait suggesting that delayed maturation of the AI results in suboptimal integration of sensory and cognitive information in adolescents and this contributes to inflexible behaviors in specific conditions of reward availability.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466226', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Inhibition of Metastatic Uveal Melanoma Cell Proliferation by the Histone Deacetylase 6 Inhibitor, Ricolinostat, is Linked to Altered Microphthalmia-associated Transcription Factor and Phospho-ERK Expression', 'authors': \"[{'name': 'Husvinee Sundaramurthi'}, {'name': 'Sandra Garcia-Mulero'}, {'name': 'Kayleigh Slater'}, {'name': 'Simone Marcone'}, {'name': 'Josep M. Piulats'}, {'name': 'R.William Watson'}, {'name': 'Desmond Tobin'}, {'name': 'Lasse Jensen'}, {'name': 'Breandan N. Kennedy'}]\", 'abstract': '<p>Metastatic uveal melanoma (MUM) is characterized by poor patient survival. Unfortunately, current treatment options demonstrate limited benefits. In this study, we evaluate the efficacy of ACY-1215, a histone deacetylase 6 inhibitor (HDAC6i), to attenuate MUM cell growth in vitro and in vivo, and elucidate the underlying molecular mechanisms. Treatment of OMM2.5 MUM cells with ACY-1215 resulted in a significant (p = 0.0001), dose-dependent reduction in cell survival and proliferation in vitro, and in vivo regression of primary OMM2.5 xenografts in zebrafish larvae. Furthermore, flow cytometry analysis revealed that ACY-1215 significantly arrested the OMM2.5 cell cycle in S phase (p = 0.0006) following 24 hours of treatment and significant apoptosis was triggered in a time- and dose-dependent manner (p = &lt;0.0001). Additionally, ACY-1215 treatment resulted in a significant reduction in OMM2.5 p-ERK expression levels. Through proteome-profiling, attenuation of the microphthalmia-associated transcription factor (MITF) signaling pathway was linked to the observed anti-cancer effects of ACY-1215. In agreement, pharmacological inhibition of MITF signaling with ML329, significantly reduced OMM2.5 cell survival and viability in vitro (p = 0.0001) and in vivo (p = 0.0006). Our findings provide evidence that ACY-1215 and ML329 are efficacious against growth and survival of MUM cells and are potential therapeutic options for MUM.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466245', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'The Loroxanthin cycle: A new type of xanthophyll cycle in green algae (Chlorophyta)', 'authors': \"[{'name': 'Tom E van den Berg'}, {'name': 'Roberta Croce'}]\", 'abstract': '<p>Xanthophyll cycles have proven to be major contributors to photoacclimation for many organisms. This work describes a light-driven xanthophyll cycle operating in the chlorophyte Chlamydomonas reinhardtii and involving the xanthophylls Lutein (L) and Loroxanthin (Lo). Pigments were quantified during a switch from high to low light and at different time points from cells grown in Day/night cycle. Trimeric LHCII was purified from cells acclimated to high or low light and their pigment content and spectroscopic properties were characterized. The Lo/(L+Lo) ratio in the cells varies by a factor of 10 between cells grown in low or high light leading to a change in the Lo/(L+Lo) ratio in trimeric LHCII from 0.5 in low light to 0.07 in high light. Trimeric LhcbMs binding Loroxanthin have 5+/-1% higher excitation energy transfer from carotenoid to Chlorophyll as well as higher thermo- and photostability than trimeric LhcbMs that only bind Lutein. The Loroxanthin cycle operates on long time scales (hours to days) and likely evolved as a shade adaptation. It has many similarities with the Lutein-epoxide - Lutein cycle of plants.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466258', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'DNA methylation patterns differ between free-living Rhizobium leguminosarum RCAM1026 and bacteroids formed in symbiosis with pea (Pisum sativum L.)', 'authors': \"[{'name': 'Alexey M. Afonin'}, {'name': 'Emma S. Gribchenko'}, {'name': 'Evgeni A. Zorin'}, {'name': 'Anton S. Sulima'}, {'name': 'Vladimir A. Zhukov'}]\", 'abstract': '<p>Rhizobium leguminosarum (Rl) is a common name for several genospecies of rhizobia able to form nitrogen-fixing nodules on the roots of pea (Pisum sativum L.) and undergo terminal differentiation into a symbiotic form called bacteroids. In this work, we compared the genomes of the free-living and differentiated forms of the Rl strain RCAM1026 using Oxford Nanopore long reads. No significant genome rearrangements were observed, but the relative abundances of replicons were different between the cell states. GANTC, GGCGCC and GATC methylated motifs have been found in the genome, along with genes for methyltransferases with matching predicted targets. Methylation patterns for the GANTC and GATC motives differed significantly depending on the cell state, which indicates their possible connection to the regulation of symbiotic differentiation. The GGCGCC motif was completely methylated in both bacteria states, and, apparently, is a target for the modification-restriction system. Currently, the methylation patterns in symbiotic bacteria are not extensively studied, so a further investigation of the topic coupled with gene expression data is needed to elucidate the function of differential methylation in terminal differentiation of R. leguminosarum and other rhizobia.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466272', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Computational Fontan Analysis: Preserving accuracy while expediting workflow', 'authors': \"[{'name': 'Xiaolong Liu'}, {'name': 'Seda Aslan'}, {'name': 'Byeol Kim'}, {'name': 'Linnea Warburton'}, {'name': 'Derrick Jackson'}, {'name': 'Abir Muhuri'}, {'name': 'Akshay Subramanian'}, {'name': 'Paige Mass'}, {'name': 'Vincent Cleveland'}, {'name': 'Yue-Hin Loke'}, {'name': 'Narutoshi Hibino'}, {'name': 'Laura Olivieri'}, {'name': 'Axel Krieger'}]\", 'abstract': '<p>Background: Post-operative outcomes of the Fontan operation have been linked to graft shape after implantation. Computational fluid dynamics (CFD) simulations are used to explore different surgical options. The objective of this study is to perform a systematic in vitro validation for investigating the accuracy and efficiency of CFD simulation to predict Fontan hemodynamics. Methods: CFD simulations were performed to measure indexed power loss (iPL) and hepatic flow distribution (HFD) in 10 patient-specific Fontan models, with varying mesh and numerical solvers. The results were compared with a novel in vitro flow loop setup with 3D printed Fontan models. A high-resolution differential pressure sensor was used to measure the pressure drop for validating iPL predictions. Microparticles with particle filtering system were used to measure HFD. The computational time was measured for a representative Fontan model with different mesh sizes and numerical solvers. Results: When compared to in vitro setup, variations in CFD mesh sizes had significant effect on HFD (p = 0.0002) but no significant impact on iPL (p = 0.069). Numerical solvers had no significant impact in both iPL (p = 0.50) and HFD (P = 0.55). A transient solver with 0.5 mm mesh size requires computational time 100 times more than a steady solver with 2.5 mm mesh size to generate similar results. Conclusions: The predictive value of CFD for Fontan planning can be validated against an in vitro flow loop. The prediction accuracy can be affected by the mesh size, model shape complexity and flow competition.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466275', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Direct precise measurement of the stall torque of the flagellar motor in E. coli', 'authors': \"[{'name': 'Bin Wang'}, {'name': 'Guanhua Yue'}, {'name': 'Rongjing Zhang'}, {'name': 'Junhua Yuan'}]\", 'abstract': '<p>The flagellar motor drives the rotation of flagellar filaments, propeling the swimming of flagellated bacteria. The maximum torque the motor generates, the stall torque, is a key characteristics of the motor function. Direct measurements of the stall torque carried out three decades ago suffered from large experimental uncertainties, and subsequently there were only indirect measurements. Here, we applied magnetic tweezer to directly measure the stall torque in E. coli. We precisely calibrated the torsional stiffness of the magnetic tweezer, and performed motor resurrection experiments at stall, accomplishing a precise determination of the stall torque per torque-generating unit (stator unit). From our measurements, each stator passes 2 protons per step, indicating a tight coupling between motor rotation and proton flux.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466295', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'Repeated exposure to chlorpyrifos is associated with a dose-dependent chronic neurobehavioral deficit in adult rats', 'authors': \"[{'name': 'Ana CR Ribeiro'}, {'name': 'Elisa H Hawkins'}, {'name': 'Fay M Jahr'}, {'name': 'Joseph L McClay'}, {'name': 'Laxmikant S Deshpande'}]\", 'abstract': '<p>Organophosphate (OP) chemicals include commonly used pesticides and also chemical warfare agents, and mechanistically they are potent inhibitors of the cholinesterase (ChE) enzyme. While a chronic low-dose OP exposure does not produce acute cholinergic crises, epidemiological studies report long-term neuropsychiatric issues including depression and cognitive impairments in OP-exposed individuals. Chlorpyrifos (CPF) is one of the most widely used pesticides worldwide. Multiple laboratory studies have reported on either the long-term behavioral effect of a single, high-dose CPF or studied sub-chronic behavioral effects particularly the motor and cognitive effects of repeated low-dose CPF exposure. However, studies on chronic mood and depression-related morbidities following repeated sub-threshold CPF doses that would mimic occupationally-relevant OP exposures are lacking. Here, adult male rats were injected with CPF (1, 3, 5, or 10 mg/kg/d, s.c.) for 21-days. Dependent on the CPF dose, ChE activity was inhibited approximately 60-80% in the blood and about 20-50% in the hippocampus at 2-days after the end of CPF exposures. Following an 11-week washout period, CPF-treated rats exhibited a dose-dependent increase in signs of anhedonia (sucrose preference test), anxiety (open-field and elevated plus-maze), and despair (forced swim test) despite a complete recovery of ChE activity at this stage. We speculate that both cholinergic and non-cholinergic mechanisms could play a role in the development of chronic OP-related depressive outcomes. The proposed CPF exposure paradigm could provide an ideal model to further study molecular mechanisms underlying cause and effect relationships between environmental OP exposures and the development of chronic behavioral deficits.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466311', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Sequencing Micronuclei Reveals the Landscape of Chromosomal Instability', 'authors': \"[{'name': 'Catalina Pereira'}, {'name': 'Ana Rita Rebelo'}, {'name': 'Dashiell Massey'}, {'name': 'John C. Schimenti'}, {'name': 'Robert S Weiss'}, {'name': 'Amnon Koren'}]\", 'abstract': '<p>Genome instability (GIN) is a main contributing factor to congenital and somatic diseases, but its sporadic occurrence in individual cell cycles makes it difficult to study mechanistically. One profound manifestation of GIN is the formation of micronuclei (MN), the engulfment of chromosomes or chromosome fragments in their own nuclear structures separate from the main nucleus. Here, we developed MN-seq, an approach for sequencing the DNA contained within micronuclei. We applied MN-seq to mice with mutations in Mcm4 and Rad9a, which disrupt DNA replication, repair, and damage responses. Data analysis and simulations show that centromere presence, fragment length, and a heterogenous landscape of chromosomal fragility all contribute to the patterns of DNA present within MN. In particular, we show that long genes, but also gene-poor regions, are associated with chromosome breaks that lead to the enrichment of particular genomic sequences in MN, in a genetic background-specific manner. Finally, we introduce single-cell micronucleus sequencing (scMN-seq), an approach to sequence the DNA present in MN of individual cells. Together, sequencing micronuclei provides a systematic approach for studying GIN and reveals novel molecular associations with chromosome breakage and segregation.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466332', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Causal Haplotype Block Identification in Plant Genome-Wide Association Studies', 'authors': \"[{'name': 'Xing Wu'}, {'name': 'Wei Jiang'}, {'name': 'Christopher Fragoso'}, {'name': 'Jing Huang'}, {'name': 'Geyu Zhou'}, {'name': 'Hong-yu Zhao'}, {'name': 'Stephen Dellaporta'}]\", 'abstract': '<p>Genome wide association studies (GWAS) can play an essential role in understanding genetic basis of complex traits in plants and animals. Conventional SNP-based linear mixed models (LMM) used in many GWAS that marginally test single nucleotide polymorphisms (SNPs) have successfully identified many loci with major and minor effects. In plants, the relatively small population size in GWAS and the high genetic diversity found many plant species can impede mapping efforts on complex traits. Here we present a novel haplotype-based trait fine-mapping framework, HapFM, to supplement current GWAS methods. HapFM uses genotype data to partition the genome into haplotype blocks, identifies haplotype clusters within each block, and then performs genome-wide haplotype fine-mapping to infer the causal haplotype blocks of trait. We benchmarked HapFM, GEMMA, BSLMM, and GMMAT in both simulation and real plant GWAS datasets. HapFM consistently resulted in higher mapping power than the other GWAS methods in simulations with high polygenicity. Moreover, it resulted in higher mapping resolution, especially in regions of high LD, by identifying small causal blocks in the larger haplotype block. In the Arabidopsis flowering time (FT10) datasets, HapFM identified four novel loci compared to GEMMA results, and its average mapping interval of HapFM was 9.6 times smaller than that of GEMMA. In conclusion, HapFM is tailored for plant GWAS to result in high mapping power on complex traits and improved mapping resolution to facilitate crop improvement.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.28.466384', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Combinatorial viral vector-based and live-attenuated vaccines without an adjuvant to generate broader immune responses to effectively combat pneumonic plague', 'authors': \"[{'name': 'Paul B Kilgore'}, {'name': 'Jian Sha'}, {'name': 'Emily Hendrix'}, {'name': 'Vladimir L Motin'}, {'name': 'Ashok K Chopra'}]\", 'abstract': '<p>Mice immunized with a combination of an adenovirus vector (Ad5-YFV) and live-attenuated (LMA)-based vaccines were evaluated for protective efficacy against pneumonic plague. While the Ad5-YFV vaccine harbors a fusion cassette of three genes encoding YscF, F1, and LcrV, LMA represents a mutant of parental <italic>Yersinia pestis</italic> CO92 deleted for genes encoding Lpp, MsbB, and Ail. Ad5-YFV and LMA were either administered simultaneously (1-dose regimen) or 21 days apart in various order and route of administration combinations (2-dose regimen). The 2-dose regimen induced robust immune responses to provide full protection to animals against parental CO92 and its isogenic F1 (CAF<sup>-</sup>)-deletion mutant challenges during both short- and long-term studies. Mice intranasally (i.n.) immunized with Ad5-YFV first followed by LMA (i.n. or intramuscularly [i.m.]) had higher T- and B- cell proliferative responses and LcrV antibody titers than those in mice vaccinated with LMA (i.n. or i.m.) first ahead of Ad5-YFV (i.n.) during the long-term study. Specifically, the needle- and adjuvant- free vaccine combination (i.n.) is ideal for use in plague endemic regions. Conversely, with a 1-dose regimen, mice vaccinated with Ad5-YFV i.n. and LMA by the i.m. route provided complete protection to animals against CO92 and its CAF<sup>-</sup> mutant challenges, and elicited Th1/Th2, as well as Th17 responses, thus suitable for emergency vaccination during a plague outbreak or bioterrorist attack. This is a first study in which a viral vector-based and live-attenuated vaccines were effectively used in combination representing adjuvant- and/or needle- free immunization, with each vaccine triggering a distinct cellular immune response.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.21265667', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Sports Medicine', 'covid': False, 'title': 'Female football specific energy availability questionnaire and menstrual cycle hormone monitoring', 'authors': \"[{'name': 'Nicola Keay'}, {'name': 'Eddie Craghill'}, {'name': 'Gavin Francis'}]\", 'abstract': '<p>Abstract\\n\\nObjectives\\nThe purpose of this study was to assess the energy availability status of professional female football players with an online Female Football Energy Availability Questionnaire (FFEAQ), combined with the clinical tool to model menstrual cycle hormones using artificial intelligence (AI) techniques. \\nMethods\\nThe Female Football Energy Availability (FFEAQ) was developed based on published questionnaires, with a weighted scoring system to assess risk of Relative Energy Deficiency in Sport (RED-S). For menstrual cycle hormones AI techniques modelled hormone variation over a cycle, using the results from capillary blood samples taken at two time points.\\nResults\\n21 female footballers of professional club level participated in this study, with mean age 22 years [range 16 to 30]. 20 athletes recorded positive scores on the FFEAQ, suggesting a low risk of Relative Energy Deficiency in Sport (RED-S). No players had experienced primary amenorrhoea. 5 athletes reported previous history of secondary amenorrhoea. Amongst the 15 players not taking hormonal contraception, 2 reported current oligomenorrhoea. \\nThe application of AI techniques to model menstrual cycle hormones found that in 1 of the 3 players, subclinical hormone disruption was occurring with this player reporting variable flow of menstruation. Although the other 2 players showed expected menstrual hormone variation, 1 player reported variable flow according to training load, suggestive of subclinical anovulation. At the time of testing training load was low due to pandemic lock down. \\nConclusions\\nThe professional female football athletes in this study were found to be at low risk of RED-S from the FFEAQ. Modelling menstrual cycle hormones using AI techniques indicated that this has the potential to be an effective clinical tool in identifying subtle hormone dysfunction such as subclinical anovulatory cycles in female athletes.</p>', 'posted': '2021-10-30'}, {'doi': '10.1101/2021.10.29.465954', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'R-loop mapping and characterization during Drosophila embryogenesis reveals developmental plasticity in R-loop signatures', 'authors': \"[{'name': 'Alex Munden'}, {'name': 'Mary Lauren Benton'}, {'name': 'John A Capra'}, {'name': 'Jared Nordman'}]\", 'abstract': '<p>R-loops are involved in transcriptional regulation, DNA and histone post-translational modifications, genome replication and genome stability. To what extent R-loop abundance and genome-wide localization is actively regulated during metazoan embryogenesis is unknown. The developmental program of Drosophila embryogenesis provides a powerful system to study address these questions due to its well-characterized developmental program, the sudden onset of zygotic transcription and available genome-wide ChIP and transcription data sets. Here, we measure the overall abundance and genome localization of R-loops in early and late-stage embryos relative to Drosophila cultured cells. We demonstrate that proper absolute R-loop levels change during embryogenesis and that resolution of R-loops is critical for embryonic development. R-loop mapping by strand-specific DRIP-seq revealed that R-loop localization is plastic across development, both in the genes which form R-loops and where they localize relative to gene bodies. Importantly, these changes are not driven by changes in the transcriptional program. Rather, negative GC skew and absolute changes in AT skew are associated with R-loop formation in Drosophila. Furthermore, we demonstrate that while some chromatin binding proteins and histone modification such as H3K27me3 are associated with R-loops throughout development, other chromatin factors associated with R-loops in a developmental specific manner. Our findings highlight the importance and developmental plasticity of R-loops during Drosophila development.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466223', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Onset and stepwise extensions of recombination suppression are common in mating-type chromosomes of Microbotryum anther-smut fungi', 'authors': \"[{'name': 'Marine Duhamel'}, {'name': 'Fantin Carpentier'}, {'name': 'Dominik Begerow'}, {'name': 'Michael Hood'}, {'name': 'Ricardo C Rodriguez de la Vega'}, {'name': 'Tatiana Giraud'}]\", 'abstract': '<p>Sex chromosomes and mating-type chromosomes can display large genomic regions without recombination. Recombination suppression often extended stepwise with time away from the sex- or mating-type-determining genes, generating evolutionary strata of differentiation between alternative sex or mating-type chromosomes. In anther-smut fungi of the <italic>Microbotryum</italic> genus, recombination suppression evolved repeatedly, linking the two mating-type loci and extended multiple times in regions distal to the mating-type genes. Here, we obtained high-quality genome assemblies of alternative mating types for four <italic>Microbotryum</italic> fungi. We found an additional event of independent chromosomal rearrangements bringing the two mating-type loci on the same chromosome followed by recombination suppression linking them. We also found, in a new clade analysed here, that recombination suppression between the two mating-type loci occurred in several steps, with first an ancestral recombination suppression between one of the mating-type locus and its centromere; later, completion of recombination suppression up to the second mating-type locus occurred independently in three species. The estimated dates of recombination suppression between the mating-type loci ranged from 0.15 to 3.58 million years ago. In total, this makes at least nine independent events of linkage between the mating-type loci across the <italic>Microbotryum</italic> genus. Several mating-type locus linkage events occurred through the same types of chromosomal rearrangements, where similar chromosome fissions at centromeres represent convergence in the genomic changes leading to the phenotypic convergence. These findings further highlight <italic>Microbotryum</italic> fungi as excellent models to study the evolution of recombination suppression.</p>', 'posted': '2021-10-30'}, {'doi': '10.1101/2021.10.29.466391', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Characterization of -Glutamyl Peptidases and -Glutamyl Cyclotransferases for Glutathione Degradation in Arabidopsis', 'authors': \"[{'name': 'Takehiro Ito'}, {'name': 'Taisuke Kitaiwa'}, {'name': 'Kosuke Nishizono'}, {'name': 'Minori Umahashi'}, {'name': 'Shunsuke Miyaji'}, {'name': 'Shin-ichiro Agake'}, {'name': 'Kana Kuwahara'}, {'name': 'Tadashi Yokoyama'}, {'name': 'Akiko Maruyama-Nakashita'}, {'name': 'Ryosuke Sugiyama'}, {'name': 'Masami Yokota Hirai'}, {'name': 'Naoko Ohkama-Ohtsu'}]\", 'abstract': '<p>Organic sulfur is stored as glutathione (GSH) in plants. In <italic>Arabidopsis</italic>, -glutamyl cyclotransferases (GGCT2;1, GGCT2;2, and GGCT2;3) degrade cytosolic GSH, but they do not fully explain the rapid GSH turnover. Here, we demonstrate that -glutamyl peptidases, GGP1 and GGP3, play a substantial role in degrading GSH in the cytosol. We conducted yeast complementation assay and activity assay of recombinant proteins to identify the novel GSH degradation enzymes. The expression patterns were investigated by RT-qPCR. GSH concentrations in the mutants were also analyzed. <italic>GGP1</italic> complemented the yeast phenotype. Recombinant GGP1 and GGP3 showed reasonable <italic>K<sub>m</sub></italic> values considering cytosolic GSH concentration, and their activity was comparable to that of GGCTs. The <italic>GGP1</italic> transcript was highly abundant in mature organs such as rosette leaves. The expression of <italic>GGCT2;1</italic> was conspicuously enhanced under sulfur deficiency. GSH concentration was higher in <italic>ggp1</italic> knockout mutants regardless of nutritional conditions; the concentration was higher in <italic>ggct2;1</italic> knockout mutants under sulfur-deficient conditions. We propose a model wherein cytosolic GSH is degraded fundamentally by GGP1. The degradation is accelerated by GGCT2;1 under sulfur deficiency. Given the energy cost throughout the reactions, GGPs could render a more efficient route for GSH degradation than GGCTs.</p>', 'posted': '2021-10-31'}, {'doi': '10.1101/2021.10.29.466435', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Reproducible Propagation of Species-Rich Soil Microbiomes Suggests Robust Underlying Deterministic Principles of Community Formation', 'authors': \"[{'name': 'Senka Causevic'}, {'name': 'Janko Tackmann'}, {'name': 'Vladimir Sentchilo'}, {'name': 'Christian von Mering'}, {'name': 'Jan Roelof van der Meer'}]\", 'abstract': '<p>Microbiomes are typically characterised by high species diversity but it is poorly understood how such system-level complexity can be generated and propagated. Here, we used soils as a relevant model to study microbiome development. Despite inherent stochastic variation in manipulating species-rich communities, both laboratory-mixed medium complexity (21 soil bacterial isolates in equal proportions) and high-diversity natural top-soil communities followed highly reproducible succession paths, maintaining distinct soil microbiome signatures. Development trajectories and compositional states were different for communities propagated in soils than in liquid suspension. Microbiome states were maintained over multiple renewed growth cycles but could be diverged by short-term pollutant exposure. The different but robust trajectories demonstrated that deterministic taxa-inherent characteristics underlie reproducible development and self-organized complexity of soil microbiomes within their environmental boundary conditions. Our findings also have direct implications for potential strategies to achieve controlled restoration of desertified land.</p>', 'posted': '2021-10-29'}, {'doi': '10.1101/2021.10.29.466520', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'The early embryonic transcriptome of a Hawaiian \\tDrosophila picture-wing fly shows evidence of altered gene expression and novel gene evolution', 'authors': \"[{'name': 'Madeline Chenevert'}, {'name': 'Bronwyn Miller'}, {'name': 'Ahmad Karkoutli'}, {'name': 'Anna Rusnak'}, {'name': 'Susan Lott'}, {'name': 'Joel Atallah'}]\", 'abstract': '<p>A massive adaptive radiation on the Hawaiian archipelago has produced approximately one quarter of the fly species in the family Drosophilidae. The Hawaiian <italic>Drosophila</italic> clade has long been recognized as a model system for the study of both the ecology of island endemics and the evolution of developmental mechanisms, but relatively few genomic and transcriptomic datasets are available for this group. We present here a differential expression analysis of the transcriptional profiles of two highly conserved embryonic stages in the Hawaiian picture-wing fly <italic>Drosophila grimshawi</italic>. When we compared our results to previously published datasets across the family Drosophilidae, we identified cases of both gains and losses of gene representation in <italic>D. grimshawi</italic>, including an apparent delay in Hox gene activation. We also found high expression of unannotated genes. Most transcripts of unannotated genes with open reading frames do not have homologs in non-Hawaiian <italic>Drosophila</italic> species, although the vast majority have sequence matches in other genomes of the Hawaiian picture-wing flies. Some of these genes may have arisen from non-coding sequence in the ancestor of Hawaiian flies or during the evolution of the clade. Our results suggests that both the modified use of ancestral genes and the evolution of new ones may occur in rapid radiations.</p>', 'posted': '2021-10-29'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.10.26.465776</td>\n",
       "      <td>202</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Artificial intelligence reveals nuclear pore c...</td>\n",
       "      <td>[{'name': 'Shyamal Mosalaganti'}, {'name': 'Ag...</td>\n",
       "      <td>&lt;p&gt;Nuclear pore complexes (NPCs) mediate nucle...</td>\n",
       "      <td>2021-10-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.10.24.465614</td>\n",
       "      <td>100</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Tension-driven multi-scale self-organisation i...</td>\n",
       "      <td>[{'name': 'Qiyan Mao'}, {'name': 'Achyuth Acha...</td>\n",
       "      <td>&lt;p&gt;Human muscle is a hierarchically organised ...</td>\n",
       "      <td>2021-10-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.10.21.21265342</td>\n",
       "      <td>78</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Genetic and Genomic Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Linking common and rare disease genetics throu...</td>\n",
       "      <td>[{'name': 'Olivier B. Bakker'}, {'name': 'Anni...</td>\n",
       "      <td>&lt;p&gt;Genetic variants identified through genome-...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.10.29.466401</td>\n",
       "      <td>58</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>SARS-CoV-2 spike protein as a bacterial lipopo...</td>\n",
       "      <td>[{'name': 'Firdaus Samsudin'}, {'name': 'Palur...</td>\n",
       "      <td>&lt;p&gt;Accumulating evidence indicates a potential...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.10.26.21265525</td>\n",
       "      <td>58</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>COVCOG 1: Factors predicting Cognitive Symptom...</td>\n",
       "      <td>[{'name': 'Panyuan Guo'}, {'name': 'Alvaro Ben...</td>\n",
       "      <td>&lt;p&gt;Since its first emergence in December 2019,...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.10.27.466055</td>\n",
       "      <td>54</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>True</td>\n",
       "      <td>Phylogenetic analysis and in silico studies li...</td>\n",
       "      <td>[{'name': 'Anna Bertelli'}, {'name': \"Pasquali...</td>\n",
       "      <td>&lt;p&gt;Genotype screening was implemented in Italy...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.10.29.21265511</td>\n",
       "      <td>53</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Oncology</td>\n",
       "      <td>True</td>\n",
       "      <td>Blood cytokine analysis suggests that SARS-CoV...</td>\n",
       "      <td>[{'name': 'Fien HR De Winter'}, {'name': 'An H...</td>\n",
       "      <td>&lt;p&gt;Cytokines, chemokines and (angiogenic) grow...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.10.25.465200</td>\n",
       "      <td>52</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Cell-extrinsic autophagy in mature adipocytes ...</td>\n",
       "      <td>[{'name': 'Felix Clemens Richter'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;Autophagy is a critical cellular recycling ...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.10.27.466091</td>\n",
       "      <td>43</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Cryo-electron microscopy structure of the H3-H...</td>\n",
       "      <td>[{'name': 'Kayo Nozawa'}, {'name': 'Yoshimasa ...</td>\n",
       "      <td>&lt;p&gt;The canonical nucleosome, which represents ...</td>\n",
       "      <td>2021-10-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.10.29.466387</td>\n",
       "      <td>43</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Discovery of a conserved rule behind the assem...</td>\n",
       "      <td>[{'name': 'Edward M Germany'}, {'name': 'Yue D...</td>\n",
       "      <td>&lt;p&gt;Gram-negative bacteria, mitochondria and ch...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.10.26.465865</td>\n",
       "      <td>41</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Excessive inflammatory and metabolic responses...</td>\n",
       "      <td>[{'name': 'Werner C. Albrich'}, {'name': 'Tari...</td>\n",
       "      <td>&lt;p&gt;Protection against severe acute respiratory...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.10.27.466163</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Prime-boost vaccination regimens with INO-4800...</td>\n",
       "      <td>[{'name': 'Jewell N Walters'}, {'name': 'Blake...</td>\n",
       "      <td>&lt;p&gt;The enhanced transmissibility and immune ev...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.10.25.21265476</td>\n",
       "      <td>37</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>A Made-in-Canada serology solution for profi...</td>\n",
       "      <td>[{'name': 'Karen Colwill'}, {'name': 'Yannick ...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;BACKGROUND&lt;/title&gt;&lt;p&gt;Testing for a...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.10.26.465989</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Evolutionary instability of collateral suscept...</td>\n",
       "      <td>[{'name': 'Vidar Srum'}, {'name': 'Emma L. y...</td>\n",
       "      <td>&lt;p&gt;Collateral sensitivity and resistance occur...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.10.25.465810</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Transmission of Klebsiella strains and plasmid...</td>\n",
       "      <td>[{'name': 'Ben Vezina'}, {'name': 'Louise M. J...</td>\n",
       "      <td>&lt;p&gt;The Grey-headed flying fox (&lt;italic&gt;Pteropu...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.10.29.466484</td>\n",
       "      <td>34</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Clamping of DNA shuts the condensin neck gate</td>\n",
       "      <td>[{'name': 'Byung-Gil Lee'}, {'name': 'James Rh...</td>\n",
       "      <td>&lt;p&gt;Condensin is a Structural Maintenance of Ch...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.10.26.465916</td>\n",
       "      <td>31</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>MLF2 modulates phase separated nuclear envelop...</td>\n",
       "      <td>[{'name': 'Sarah M Prophet'}, {'name': 'Anthon...</td>\n",
       "      <td>&lt;p&gt;DYT1 dystonia is a highly debilitating neur...</td>\n",
       "      <td>2021-10-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.10.28.466259</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Modelling human zygotic genome activation in 8...</td>\n",
       "      <td>[{'name': 'Jasmin Taubenschmid-Stowers'}, {'na...</td>\n",
       "      <td>&lt;p&gt;The remodelling of the epigenome and transc...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.10.28.466356</td>\n",
       "      <td>26</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Elevated Hoxb5b expands vagal neural crest poo...</td>\n",
       "      <td>[{'name': 'Aubrey G. Adam Howard'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;Neural crest cells (NCCs) are a migratory, ...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.10.26.465843</td>\n",
       "      <td>25</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Diversity, function and evolution of marine mi...</td>\n",
       "      <td>[{'name': 'Jianwei Chen'}, {'name': 'Yang Guo'...</td>\n",
       "      <td>&lt;p&gt;Trillions of marine bacterial, archaeal and...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.10.27.466170</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Inducible transcriptional condensates drive 3D...</td>\n",
       "      <td>[{'name': 'Surabhi Chowdhary'}, {'name': 'Amol...</td>\n",
       "      <td>&lt;p&gt;Mammalian developmental and disease-associa...</td>\n",
       "      <td>2021-10-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.10.27.466024</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Tree-based Correlation Screen and Visualizatio...</td>\n",
       "      <td>[{'name': 'Boyang Zhang'}, {'name': 'Zhicheng ...</td>\n",
       "      <td>&lt;p&gt;Single-cell RNA-seq experiments with multip...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.10.26.465912</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>RNA compensation: A positive feedback insulati...</td>\n",
       "      <td>[{'name': 'Baiyang Liu'}, {'name': 'Christian ...</td>\n",
       "      <td>&lt;p&gt;The lack of signalling modularity of biomol...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.10.28.466255</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Plasticity in stomatal behavior across a gradi...</td>\n",
       "      <td>[{'name': 'Risheng Ding'}, {'name': 'Jiayang X...</td>\n",
       "      <td>&lt;p&gt;Stomata regulate leaf CO2 assimilation (A) ...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.10.26.465934</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Open chromatin analysis in Trypanosoma cruzi l...</td>\n",
       "      <td>[{'name': 'Alex Ranieri Jernimo Lima'}, {'nam...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Genomic organ...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.10.27.466000</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Neurons in human pre-supplementary motor area ...</td>\n",
       "      <td>[{'name': 'Tomas G. Aquino'}, {'name': 'Jeffre...</td>\n",
       "      <td>&lt;p&gt;Adaptive behavior in real-world environment...</td>\n",
       "      <td>2021-10-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.10.25.465778</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Cost-effective epigenetic age profiling in sha...</td>\n",
       "      <td>[{'name': 'Alexandre Trapp'}, {'name': 'Vadim ...</td>\n",
       "      <td>&lt;p&gt;There is a critical need for robust, high-t...</td>\n",
       "      <td>2021-10-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.10.29.466408</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>The glycosylated extracellular domain of MUC1 ...</td>\n",
       "      <td>[{'name': 'Maitrayee Chatterjee'}, {'name': 'L...</td>\n",
       "      <td>&lt;p&gt;Mucins play an essential role in protecting...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.10.26.465878</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Chromosome-scale assemblies of Acanthamoeba ca...</td>\n",
       "      <td>[{'name': 'Cyril Matthey-Doret'}, {'name': 'Mo...</td>\n",
       "      <td>&lt;p&gt;The unicellular amoeba &lt;italic&gt;Acanthamoeba...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.10.27.466206</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>True</td>\n",
       "      <td>A multi-tissue study of immune gene expression...</td>\n",
       "      <td>[{'name': 'Alberto Gomez-Carballa'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Background: COVID-19 symptoms range from mi...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.10.28.21265616</td>\n",
       "      <td>19</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Pediatrics</td>\n",
       "      <td>True</td>\n",
       "      <td>Risk factors for severe PCR-positive SARS-CoV-...</td>\n",
       "      <td>[{'name': 'Tilmann Schober'}, {'name': 'Chelse...</td>\n",
       "      <td>&lt;p&gt;Importance: Children are less likely than a...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.10.28.466306</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Vagus nerve stimulation accelerates motor lear...</td>\n",
       "      <td>[{'name': 'Spencer Bowles'}, {'name': 'Jordan ...</td>\n",
       "      <td>&lt;p&gt;Vagus nerve stimulation (VNS) is a novel th...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.10.26.465991</td>\n",
       "      <td>18</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>FiberNeat: unsupervised streamline clustering ...</td>\n",
       "      <td>[{'name': 'Bramsh Qamar Chandio'}, {'name': 'T...</td>\n",
       "      <td>&lt;p&gt;Whole-brain tractograms generated from diff...</td>\n",
       "      <td>2021-10-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.10.26.465990</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Geological processes mediate a subsurface micr...</td>\n",
       "      <td>[{'name': 'Daniel A. Gittins'}, {'name': 'Pier...</td>\n",
       "      <td>&lt;p&gt;The deep biosphere is the largest microbial...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.10.26.465695</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Mother centrioles generate a local pulse of Po...</td>\n",
       "      <td>[{'name': 'Siu-Shing Wong'}, {'name': 'Zachary...</td>\n",
       "      <td>&lt;p&gt;Mitotic centrosomes are formed when centrio...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.10.28.466271</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Cortical adaptation to sound reverberation</td>\n",
       "      <td>[{'name': 'Aleksandar Z Ivanov'}, {'name': 'An...</td>\n",
       "      <td>&lt;p&gt;In almost every natural environment, sounds...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.10.28.466254</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Activating natural product synthesis using CRI...</td>\n",
       "      <td>[{'name': 'Andrea Ameruoso'}, {'name': 'Maria ...</td>\n",
       "      <td>&lt;p&gt;The rise of antibiotic-resistant bacteria r...</td>\n",
       "      <td>2021-10-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.10.26.465978</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>PDGFR dimer-specific activation, trafficking a...</td>\n",
       "      <td>[{'name': 'Madison A. Rogers'}, {'name': 'Kath...</td>\n",
       "      <td>&lt;p&gt;Signaling through the platelet-derived grow...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.10.28.21265499</td>\n",
       "      <td>16</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Time of day of vaccination affects SARS-CoV-2 ...</td>\n",
       "      <td>[{'name': 'Wei Wang'}, {'name': 'Peter Balfe'}...</td>\n",
       "      <td>&lt;p&gt;The COVID-19 pandemic caused by severe acut...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.10.26.465984</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A cautionary tale on proper use of branch-site...</td>\n",
       "      <td>[{'name': 'Amanda Kowalczyk'}, {'name': 'Maria...</td>\n",
       "      <td>&lt;p&gt;Comparative genomics has become a powerful ...</td>\n",
       "      <td>2021-10-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.10.24.465610</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Neural basis of learning to perceive speech th...</td>\n",
       "      <td>[{'name': 'Patrick S. Malone'}, {'name': 'Silv...</td>\n",
       "      <td>&lt;p&gt;The goal of sensory substitution is to conv...</td>\n",
       "      <td>2021-10-25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.10.28.466308</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Divergent evolutionary trajectories of bryophy...</td>\n",
       "      <td>[{'name': 'Brogan J Harris'}, {'name': 'James ...</td>\n",
       "      <td>&lt;p&gt;The origin of plants and their colonization...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.10.30.466508</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>A neural substrate for Bayesian integration in...</td>\n",
       "      <td>[{'name': 'Nicholas M. Singletary'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Making adaptive decisions often requires in...</td>\n",
       "      <td>2021-10-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.10.28.466333</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Deciphering the species-level structure of top...</td>\n",
       "      <td>[{'name': 'Rohit Singh'}, {'name': 'Bonnie Ber...</td>\n",
       "      <td>&lt;p&gt;Chromosome conformation capture technologie...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.10.23.21265427</td>\n",
       "      <td>13</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Occupational and Environmental Health</td>\n",
       "      <td>False</td>\n",
       "      <td>Performance of Valved Respirators to Reduce Re...</td>\n",
       "      <td>[{'name': 'Jessica M. Hazard'}, {'name': 'Chri...</td>\n",
       "      <td>&lt;p&gt;Wearing of face coverings serves two purpos...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.10.27.466135</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Inhibiting adult neurogenesis differentially a...</td>\n",
       "      <td>[{'name': 'Timothy P OLeary'}, {'name': 'Bara...</td>\n",
       "      <td>&lt;p&gt;Adult hippocampal neurogenesis has been imp...</td>\n",
       "      <td>2021-10-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.10.28.466256</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Tubular lysosome induction couples animal star...</td>\n",
       "      <td>[{'name': 'Tatiana V Villalobos'}, {'name': 'B...</td>\n",
       "      <td>&lt;p&gt;Dietary restriction promotes longevity via ...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.10.25.465682</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Identifying barriers to gene flow and hierarch...</td>\n",
       "      <td>[{'name': 'Germain Boussarie'}, {'name': 'Paol...</td>\n",
       "      <td>&lt;p&gt;The ongoing decline of large marine vertebr...</td>\n",
       "      <td>2021-10-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.10.28.466298</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>True</td>\n",
       "      <td>Identification of COVID-19 and COPD common key...</td>\n",
       "      <td>[{'name': 'Thiviya S. Thambiraja'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;Coronavirus disease (COVID-19) is an extrem...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.10.28.466320</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Allele frequency dynamics under sex-biased dem...</td>\n",
       "      <td>[{'name': 'Rose M.H. Driscoll'}, {'name': 'Fel...</td>\n",
       "      <td>&lt;p&gt;Sex-biased demography, including sex-biased...</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.10.26.465776     202  bioRxiv   \n",
       "1     10.1101/2021.10.24.465614     100  bioRxiv   \n",
       "2   10.1101/2021.10.21.21265342      78  medRxiv   \n",
       "3     10.1101/2021.10.29.466401      58  bioRxiv   \n",
       "4   10.1101/2021.10.26.21265525      58  medRxiv   \n",
       "5     10.1101/2021.10.27.466055      54  bioRxiv   \n",
       "6   10.1101/2021.10.29.21265511      53  medRxiv   \n",
       "7     10.1101/2021.10.25.465200      52  bioRxiv   \n",
       "8     10.1101/2021.10.27.466091      43  bioRxiv   \n",
       "9     10.1101/2021.10.29.466387      43  bioRxiv   \n",
       "10    10.1101/2021.10.26.465865      41  bioRxiv   \n",
       "11    10.1101/2021.10.27.466163      37  bioRxiv   \n",
       "12  10.1101/2021.10.25.21265476      37  medRxiv   \n",
       "13    10.1101/2021.10.26.465989      37  bioRxiv   \n",
       "14    10.1101/2021.10.25.465810      37  bioRxiv   \n",
       "15    10.1101/2021.10.29.466484      34  bioRxiv   \n",
       "16    10.1101/2021.10.26.465916      31  bioRxiv   \n",
       "17    10.1101/2021.10.28.466259      26  bioRxiv   \n",
       "18    10.1101/2021.10.28.466356      26  bioRxiv   \n",
       "19    10.1101/2021.10.26.465843      25  bioRxiv   \n",
       "20    10.1101/2021.10.27.466170      24  bioRxiv   \n",
       "21    10.1101/2021.10.27.466024      24  bioRxiv   \n",
       "22    10.1101/2021.10.26.465912      23  bioRxiv   \n",
       "23    10.1101/2021.10.28.466255      23  bioRxiv   \n",
       "24    10.1101/2021.10.26.465934      23  bioRxiv   \n",
       "25    10.1101/2021.10.27.466000      22  bioRxiv   \n",
       "26    10.1101/2021.10.25.465778      21  bioRxiv   \n",
       "27    10.1101/2021.10.29.466408      20  bioRxiv   \n",
       "28    10.1101/2021.10.26.465878      20  bioRxiv   \n",
       "29    10.1101/2021.10.27.466206      19  bioRxiv   \n",
       "30  10.1101/2021.10.28.21265616      19  medRxiv   \n",
       "31    10.1101/2021.10.28.466306      19  bioRxiv   \n",
       "32    10.1101/2021.10.26.465991      18  bioRxiv   \n",
       "33    10.1101/2021.10.26.465990      17  bioRxiv   \n",
       "34    10.1101/2021.10.26.465695      17  bioRxiv   \n",
       "35    10.1101/2021.10.28.466271      17  bioRxiv   \n",
       "36    10.1101/2021.10.28.466254      17  bioRxiv   \n",
       "37    10.1101/2021.10.26.465978      16  bioRxiv   \n",
       "38  10.1101/2021.10.28.21265499      16  medRxiv   \n",
       "39    10.1101/2021.10.26.465984      15  bioRxiv   \n",
       "40    10.1101/2021.10.24.465610      15  bioRxiv   \n",
       "41    10.1101/2021.10.28.466308      15  bioRxiv   \n",
       "42    10.1101/2021.10.30.466508      14  bioRxiv   \n",
       "43    10.1101/2021.10.28.466333      14  bioRxiv   \n",
       "44  10.1101/2021.10.23.21265427      13  medRxiv   \n",
       "45    10.1101/2021.10.27.466135      12  bioRxiv   \n",
       "46    10.1101/2021.10.28.466256      12  bioRxiv   \n",
       "47    10.1101/2021.10.25.465682      12  bioRxiv   \n",
       "48    10.1101/2021.10.28.466298      12  bioRxiv   \n",
       "49    10.1101/2021.10.28.466320      12  bioRxiv   \n",
       "\n",
       "                             subject-area  covid  \\\n",
       "0                       Molecular Biology  False   \n",
       "1                   Developmental Biology  False   \n",
       "2            Genetic and Genomic Medicine  False   \n",
       "3                            Microbiology   True   \n",
       "4                            Epidemiology   True   \n",
       "5                          Bioinformatics   True   \n",
       "6                                Oncology   True   \n",
       "7                              Immunology  False   \n",
       "8                            Biochemistry  False   \n",
       "9                            Biochemistry  False   \n",
       "10                             Immunology   True   \n",
       "11                             Immunology   True   \n",
       "12  Infectious Diseases (except HIV/AIDS)   True   \n",
       "13                           Microbiology  False   \n",
       "14                           Microbiology  False   \n",
       "15                      Molecular Biology  False   \n",
       "16                           Cell Biology  False   \n",
       "17                      Molecular Biology  False   \n",
       "18                  Developmental Biology  False   \n",
       "19                           Microbiology  False   \n",
       "20                      Molecular Biology  False   \n",
       "21                         Bioinformatics  False   \n",
       "22                      Synthetic Biology  False   \n",
       "23                          Plant Biology  False   \n",
       "24                           Microbiology  False   \n",
       "25                           Neuroscience  False   \n",
       "26                               Genomics  False   \n",
       "27                           Microbiology   True   \n",
       "28                               Genomics  False   \n",
       "29                               Genomics   True   \n",
       "30                             Pediatrics   True   \n",
       "31                           Neuroscience  False   \n",
       "32                           Neuroscience  False   \n",
       "33                           Microbiology  False   \n",
       "34                           Cell Biology  False   \n",
       "35                           Neuroscience  False   \n",
       "36                      Synthetic Biology  False   \n",
       "37                           Cell Biology  False   \n",
       "38  Infectious Diseases (except HIV/AIDS)   True   \n",
       "39                   Evolutionary Biology  False   \n",
       "40                           Neuroscience  False   \n",
       "41                   Evolutionary Biology  False   \n",
       "42                           Neuroscience  False   \n",
       "43                         Bioinformatics  False   \n",
       "44  Occupational and Environmental Health  False   \n",
       "45                           Neuroscience  False   \n",
       "46                           Cell Biology  False   \n",
       "47                                Ecology  False   \n",
       "48                         Bioinformatics   True   \n",
       "49                   Evolutionary Biology  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   Artificial intelligence reveals nuclear pore c...   \n",
       "1   Tension-driven multi-scale self-organisation i...   \n",
       "2   Linking common and rare disease genetics throu...   \n",
       "3   SARS-CoV-2 spike protein as a bacterial lipopo...   \n",
       "4   COVCOG 1: Factors predicting Cognitive Symptom...   \n",
       "5   Phylogenetic analysis and in silico studies li...   \n",
       "6   Blood cytokine analysis suggests that SARS-CoV...   \n",
       "7   Cell-extrinsic autophagy in mature adipocytes ...   \n",
       "8   Cryo-electron microscopy structure of the H3-H...   \n",
       "9   Discovery of a conserved rule behind the assem...   \n",
       "10  Excessive inflammatory and metabolic responses...   \n",
       "11  Prime-boost vaccination regimens with INO-4800...   \n",
       "12  A Made-in-Canada serology solution for profi...   \n",
       "13  Evolutionary instability of collateral suscept...   \n",
       "14  Transmission of Klebsiella strains and plasmid...   \n",
       "15      Clamping of DNA shuts the condensin neck gate   \n",
       "16  MLF2 modulates phase separated nuclear envelop...   \n",
       "17  Modelling human zygotic genome activation in 8...   \n",
       "18  Elevated Hoxb5b expands vagal neural crest poo...   \n",
       "19  Diversity, function and evolution of marine mi...   \n",
       "20  Inducible transcriptional condensates drive 3D...   \n",
       "21  Tree-based Correlation Screen and Visualizatio...   \n",
       "22  RNA compensation: A positive feedback insulati...   \n",
       "23  Plasticity in stomatal behavior across a gradi...   \n",
       "24  Open chromatin analysis in Trypanosoma cruzi l...   \n",
       "25  Neurons in human pre-supplementary motor area ...   \n",
       "26  Cost-effective epigenetic age profiling in sha...   \n",
       "27  The glycosylated extracellular domain of MUC1 ...   \n",
       "28  Chromosome-scale assemblies of Acanthamoeba ca...   \n",
       "29  A multi-tissue study of immune gene expression...   \n",
       "30  Risk factors for severe PCR-positive SARS-CoV-...   \n",
       "31  Vagus nerve stimulation accelerates motor lear...   \n",
       "32  FiberNeat: unsupervised streamline clustering ...   \n",
       "33  Geological processes mediate a subsurface micr...   \n",
       "34  Mother centrioles generate a local pulse of Po...   \n",
       "35         Cortical adaptation to sound reverberation   \n",
       "36  Activating natural product synthesis using CRI...   \n",
       "37  PDGFR dimer-specific activation, trafficking a...   \n",
       "38  Time of day of vaccination affects SARS-CoV-2 ...   \n",
       "39  A cautionary tale on proper use of branch-site...   \n",
       "40  Neural basis of learning to perceive speech th...   \n",
       "41  Divergent evolutionary trajectories of bryophy...   \n",
       "42  A neural substrate for Bayesian integration in...   \n",
       "43  Deciphering the species-level structure of top...   \n",
       "44  Performance of Valved Respirators to Reduce Re...   \n",
       "45  Inhibiting adult neurogenesis differentially a...   \n",
       "46  Tubular lysosome induction couples animal star...   \n",
       "47  Identifying barriers to gene flow and hierarch...   \n",
       "48  Identification of COVID-19 and COPD common key...   \n",
       "49  Allele frequency dynamics under sex-biased dem...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Shyamal Mosalaganti'}, {'name': 'Ag...   \n",
       "1   [{'name': 'Qiyan Mao'}, {'name': 'Achyuth Acha...   \n",
       "2   [{'name': 'Olivier B. Bakker'}, {'name': 'Anni...   \n",
       "3   [{'name': 'Firdaus Samsudin'}, {'name': 'Palur...   \n",
       "4   [{'name': 'Panyuan Guo'}, {'name': 'Alvaro Ben...   \n",
       "5   [{'name': 'Anna Bertelli'}, {'name': \"Pasquali...   \n",
       "6   [{'name': 'Fien HR De Winter'}, {'name': 'An H...   \n",
       "7   [{'name': 'Felix Clemens Richter'}, {'name': '...   \n",
       "8   [{'name': 'Kayo Nozawa'}, {'name': 'Yoshimasa ...   \n",
       "9   [{'name': 'Edward M Germany'}, {'name': 'Yue D...   \n",
       "10  [{'name': 'Werner C. Albrich'}, {'name': 'Tari...   \n",
       "11  [{'name': 'Jewell N Walters'}, {'name': 'Blake...   \n",
       "12  [{'name': 'Karen Colwill'}, {'name': 'Yannick ...   \n",
       "13  [{'name': 'Vidar Srum'}, {'name': 'Emma L. y...   \n",
       "14  [{'name': 'Ben Vezina'}, {'name': 'Louise M. J...   \n",
       "15  [{'name': 'Byung-Gil Lee'}, {'name': 'James Rh...   \n",
       "16  [{'name': 'Sarah M Prophet'}, {'name': 'Anthon...   \n",
       "17  [{'name': 'Jasmin Taubenschmid-Stowers'}, {'na...   \n",
       "18  [{'name': 'Aubrey G. Adam Howard'}, {'name': '...   \n",
       "19  [{'name': 'Jianwei Chen'}, {'name': 'Yang Guo'...   \n",
       "20  [{'name': 'Surabhi Chowdhary'}, {'name': 'Amol...   \n",
       "21  [{'name': 'Boyang Zhang'}, {'name': 'Zhicheng ...   \n",
       "22  [{'name': 'Baiyang Liu'}, {'name': 'Christian ...   \n",
       "23  [{'name': 'Risheng Ding'}, {'name': 'Jiayang X...   \n",
       "24  [{'name': 'Alex Ranieri Jernimo Lima'}, {'nam...   \n",
       "25  [{'name': 'Tomas G. Aquino'}, {'name': 'Jeffre...   \n",
       "26  [{'name': 'Alexandre Trapp'}, {'name': 'Vadim ...   \n",
       "27  [{'name': 'Maitrayee Chatterjee'}, {'name': 'L...   \n",
       "28  [{'name': 'Cyril Matthey-Doret'}, {'name': 'Mo...   \n",
       "29  [{'name': 'Alberto Gomez-Carballa'}, {'name': ...   \n",
       "30  [{'name': 'Tilmann Schober'}, {'name': 'Chelse...   \n",
       "31  [{'name': 'Spencer Bowles'}, {'name': 'Jordan ...   \n",
       "32  [{'name': 'Bramsh Qamar Chandio'}, {'name': 'T...   \n",
       "33  [{'name': 'Daniel A. Gittins'}, {'name': 'Pier...   \n",
       "34  [{'name': 'Siu-Shing Wong'}, {'name': 'Zachary...   \n",
       "35  [{'name': 'Aleksandar Z Ivanov'}, {'name': 'An...   \n",
       "36  [{'name': 'Andrea Ameruoso'}, {'name': 'Maria ...   \n",
       "37  [{'name': 'Madison A. Rogers'}, {'name': 'Kath...   \n",
       "38  [{'name': 'Wei Wang'}, {'name': 'Peter Balfe'}...   \n",
       "39  [{'name': 'Amanda Kowalczyk'}, {'name': 'Maria...   \n",
       "40  [{'name': 'Patrick S. Malone'}, {'name': 'Silv...   \n",
       "41  [{'name': 'Brogan J Harris'}, {'name': 'James ...   \n",
       "42  [{'name': 'Nicholas M. Singletary'}, {'name': ...   \n",
       "43  [{'name': 'Rohit Singh'}, {'name': 'Bonnie Ber...   \n",
       "44  [{'name': 'Jessica M. Hazard'}, {'name': 'Chri...   \n",
       "45  [{'name': 'Timothy P OLeary'}, {'name': 'Bara...   \n",
       "46  [{'name': 'Tatiana V Villalobos'}, {'name': 'B...   \n",
       "47  [{'name': 'Germain Boussarie'}, {'name': 'Paol...   \n",
       "48  [{'name': 'Thiviya S. Thambiraja'}, {'name': '...   \n",
       "49  [{'name': 'Rose M.H. Driscoll'}, {'name': 'Fel...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>Nuclear pore complexes (NPCs) mediate nucle...  2021-10-27  \n",
       "1   <p>Human muscle is a hierarchically organised ...  2021-10-25  \n",
       "2   <p>Genetic variants identified through genome-...  2021-10-26  \n",
       "3   <p>Accumulating evidence indicates a potential...  2021-10-29  \n",
       "4   <p>Since its first emergence in December 2019,...  2021-10-26  \n",
       "5   <p>Genotype screening was implemented in Italy...  2021-10-29  \n",
       "6   <p>Cytokines, chemokines and (angiogenic) grow...  2021-10-29  \n",
       "7   <p>Autophagy is a critical cellular recycling ...  2021-10-26  \n",
       "8   <p>The canonical nucleosome, which represents ...  2021-10-28  \n",
       "9   <p>Gram-negative bacteria, mitochondria and ch...  2021-10-29  \n",
       "10  <p>Protection against severe acute respiratory...  2021-10-26  \n",
       "11  <p>The enhanced transmissibility and immune ev...  2021-10-29  \n",
       "12  <sec><title>BACKGROUND</title><p>Testing for a...  2021-10-26  \n",
       "13  <p>Collateral sensitivity and resistance occur...  2021-10-26  \n",
       "14  <p>The Grey-headed flying fox (<italic>Pteropu...  2021-10-26  \n",
       "15  <p>Condensin is a Structural Maintenance of Ch...  2021-10-29  \n",
       "16  <p>DYT1 dystonia is a highly debilitating neur...  2021-10-27  \n",
       "17  <p>The remodelling of the epigenome and transc...  2021-10-29  \n",
       "18  <p>Neural crest cells (NCCs) are a migratory, ...  2021-10-29  \n",
       "19  <p>Trillions of marine bacterial, archaeal and...  2021-10-26  \n",
       "20  <p>Mammalian developmental and disease-associa...  2021-10-28  \n",
       "21  <p>Single-cell RNA-seq experiments with multip...  2021-10-29  \n",
       "22  <p>The lack of signalling modularity of biomol...  2021-10-26  \n",
       "23  <p>Stomata regulate leaf CO2 assimilation (A) ...  2021-10-29  \n",
       "24  <sec><title>Background</title><p>Genomic organ...  2021-10-26  \n",
       "25  <p>Adaptive behavior in real-world environment...  2021-10-28  \n",
       "26  <p>There is a critical need for robust, high-t...  2021-10-28  \n",
       "27  <p>Mucins play an essential role in protecting...  2021-10-29  \n",
       "28  <p>The unicellular amoeba <italic>Acanthamoeba...  2021-10-26  \n",
       "29  <p>Background: COVID-19 symptoms range from mi...  2021-10-29  \n",
       "30  <p>Importance: Children are less likely than a...  2021-10-29  \n",
       "31  <p>Vagus nerve stimulation (VNS) is a novel th...  2021-10-29  \n",
       "32  <p>Whole-brain tractograms generated from diff...  2021-10-28  \n",
       "33  <p>The deep biosphere is the largest microbial...  2021-10-26  \n",
       "34  <p>Mitotic centrosomes are formed when centrio...  2021-10-26  \n",
       "35  <p>In almost every natural environment, sounds...  2021-10-29  \n",
       "36  <p>The rise of antibiotic-resistant bacteria r...  2021-10-28  \n",
       "37  <p>Signaling through the platelet-derived grow...  2021-10-26  \n",
       "38  <p>The COVID-19 pandemic caused by severe acut...  2021-10-29  \n",
       "39  <p>Comparative genomics has become a powerful ...  2021-10-28  \n",
       "40  <p>The goal of sensory substitution is to conv...  2021-10-25  \n",
       "41  <p>The origin of plants and their colonization...  2021-10-29  \n",
       "42  <p>Making adaptive decisions often requires in...  2021-10-31  \n",
       "43  <p>Chromosome conformation capture technologie...  2021-10-29  \n",
       "44  <p>Wearing of face coverings serves two purpos...  2021-10-26  \n",
       "45  <p>Adult hippocampal neurogenesis has been imp...  2021-10-28  \n",
       "46  <p>Dietary restriction promotes longevity via ...  2021-10-29  \n",
       "47  <p>The ongoing decline of large marine vertebr...  2021-10-26  \n",
       "48  <p>Coronavirus disease (COVID-19) is an extrem...  2021-10-29  \n",
       "49  <p>Sex-biased demography, including sex-biased...  2021-10-29  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "182 preprints (including 38 covering SARS-CoV-2, 146 from bioRxiv and 36 from medRxiv) published in the last 7 days before 2021-11-01 had been tweeted at least 3 times (maximum 202)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.8"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
