{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "16898 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d250d299-d855-459a-b29b-560aabb2214c&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=2b141f8a-6509-48d5-8ac3-0fc15c77e460&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c27c7c69-7731-44f0-bef4-f1b099941222&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=6e4e4c2a-c99b-47da-a992-a929c9033507&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=f367754e-980e-4900-bd35-e820bc00f0e2&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=91ef9386-6611-452f-bc96-d35b491b6322&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=13624c7f-9589-4b81-bd0e-4cee21478441&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=dfa12362-2a8c-48b5-9737-866bdce82cd2&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=c2330e21-f82f-410e-a089-bbd1c12f8bad&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=a41584ac-21fa-42e3-8966-df4f7ca2a460&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7c8d4569-c4b4-47c4-b9c7-41d92b1bed80&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4e6e984b-b994-4ff3-b500-b708931f6178&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7879b855-5391-4a83-9328-3be6d925e047&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4123195c-e08e-46d2-adf4-b6b57f0da0b7&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=4984075c-8cbb-4d1a-a482-57626c2e208d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=7a1c5113-5c37-4a8e-b6c4-0a44bd60bb30&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-09-06&until-occurred-date=2021-09-13\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load preprint_tweets_2021-09-06.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.20.20108183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20159228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.21263103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.21253183...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.05.459003...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21261912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.21251670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458870...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.16.20023754...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.05.20123133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261562...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.455288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.458941...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.20172288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21257131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.04.20225573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459546...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455491...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.11.430704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/836197...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.05.458952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.214148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.12.20230292...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.07.030650...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.23.353045...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.09.373258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458985...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.21261083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.437224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.29.124115...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/135582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.455988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.25.171850...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.27.424510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446635...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.25.21252363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259583...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20216192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.422737...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456138...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.439272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.441398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459279...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.21261282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.01.405886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.445058...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.07.20145979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454085...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.458611...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.015354...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.01.20135194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.380741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.04.355842...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.20248121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.444624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20087965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.21263123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459322...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.28.20029272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.442687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.428324...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.03.075069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.05.459049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.261586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435343...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.21251857...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459613...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.458105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/846147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.10.276980...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259487...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.20.20108126...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.06.20244780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.20167205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261618...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.431319...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.04.20121434...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.213132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.458711...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.428136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.20205633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453695...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.442096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.438369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459641...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.20248932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.13.039917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.433935...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.18.952978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21263000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.452919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260072...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21252621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.21.21250266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.419044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.21262945...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21253685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.186684.114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.20.20039495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260961...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.437747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454326...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449206...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.09.083907...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/715482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259897...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.434040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.28.424537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.09.443230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.10.376814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.21262415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.454011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458162...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/571380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.441200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.439826...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442822...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430594...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445793...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.431898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.21255699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.21256243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257759...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.441387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.21258378...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275677.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.17.452576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.27.117184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.21.20236018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.20.392225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452462...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.383083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260522...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.28.21262748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.21261501...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.21261496...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.08.982637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.332023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459125...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.10.197426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261806...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.28.270306...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.442971...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261966...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.225854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.20157149...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.06.459062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/292581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.27.21254483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.06.028712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.10.036418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.15.20065995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.14.21255036...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434580...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.05.459032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21255545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458922...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458698...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.21255913...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.440733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.05.20123752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458411...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443540...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20187724...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21261000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450543...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.20222661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.444556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20223958...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20223891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249885...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.21250653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448153...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.451001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.04.451066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21260074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.446451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21256661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.333328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.29.318402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.21258956...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.455157...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.448754...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.23.449568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261600...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.02.21261465...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.11.459880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.217216.116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459722...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.08.459492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.446826...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/782557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.03.21254823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459856...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.447883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.433940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259536...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.09.459569...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.10.459734...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.07.459280...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/233502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458682...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.30.21258079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21262021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453659...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261252...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.02.21261379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/037762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.444019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455920...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.443152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445509...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458361...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.455707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.21262937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.21255690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457884...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.29.21256255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20216630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.11.20150193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.06.20055657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21252814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.425362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.11.20086439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.21253051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.06.238915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.20248382...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.415687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.20246025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.20242495...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.10.20191619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253142...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.407999...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.29.924795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.14.21249839...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.09.20143339...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.093195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.20109231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.26.117325...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20118877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.04.20122481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20124461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.21250840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.07.138883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.21250494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21249554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.14.150490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20136309...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.27.175455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.21249998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.06.20147199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.20197467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.425021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21254000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.335083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.435141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.314815...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.27.20220905...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.09.03.21263103', 'tweets': 170, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'The prevalence of adaptive immunity to COVID-19 and reinfection after recovery – a comprehensive systematic review and meta-analysis of 12 011 447 individuals', 'authors': \"[{'name': 'Tawanda Chivese'}, {'name': 'Joshua T. Matizanadzo'}, {'name': 'Omran A. H. Musa'}, {'name': 'George Hindy'}, {'name': 'Luis Furuya-Kanamori'}, {'name': 'Nazmul Islam'}, {'name': 'Rafal Al-Shebly'}, {'name': 'Rana Shalaby'}, {'name': 'Mohammad Habibullah'}, {'name': 'Talal Al-Marwani'}, {'name': 'Rizeq F Hourani'}, {'name': 'Ahmed D Nawaz'}, {'name': 'Mohammad Z Haider'}, {'name': 'Mohamed M. Emara'}, {'name': 'Farhan Cyprian'}, {'name': 'Suhail A. R. Doi'}]\", 'abstract': '<sec><title>Objectives</title><p>This study aims to estimate the prevalence and longevity of detectable SARS-CoV-2 antibodies as well as memory cells T and B after recovery. In addition, the prevalence of COVID-19 reinfection, and the preventive efficacy of previous infection with SARS-CoV-2 were investigated.</p></sec><sec><title>Methods and analyses</title><p>A synthesis of existing research was conducted. The Cochrane Library for COVID-19 resources, the China Academic Journals Full Text Database, PubMed, and Scopus as well as preprint servers were searched for studies conducted between 1 January 2020 to 1 April 2021. We included studies with the relevant outcomes of interest. All included studies were assessed for methodological quality and pooled estimates of relevant outcomes were obtained in a meta-analysis using a bias adjusted synthesis method. Proportions were synthesized with the Freeman-Tukey double arcsine transformation and binary outcomes using the odds ratio (OR). Heterogeneity between included studies was assessed using the I2 and Cochran’s Q statistics and publication bias was assessed using Doi plots.</p></sec><sec><title>Results</title><p>Fifty-four studies, from 18 countries, with a total of 12 011 447 individuals, followed up to 8 months after recovery were included. At 6-8 months after recovery, the prevalence of SARS-CoV-2 specific immunological memory remained high; IgG – 90.4% (95%CI 72.2-99.9, I<sup>2</sup>=89.0%, 5 studies), CD4+ - 91.7% (95%CI 78.2 – 97.1, one study), and memory B cells 80.6% (95%CI 65.0-90.2, one study) and the pooled prevalence of reinfection was 0.2% (95%CI 0.0 – 0.7, I<sup>2</sup> = 98.8, 9 studies). Individuals previously infected with SARS-CoV-2 had an 81% reduction in odds of a reinfection (OR 0.19, 95% CI 0.1 - 0.3, I<sup>2</sup> = 90.5%, 5 studies).</p></sec><sec><title>Conclusion</title><p>Around 90% of people previously infected with SARS-CoV-2 had evidence of immunological memory to SARS-CoV-2, which was sustained for at least 6-8 months after recovery, and had a low risk of reinfection.</p></sec><sec><title>Registration</title><p>PROSPERO: CRD42020201234</p></sec><sec><title>What is already known on this topic</title><p>Individuals who recover from COVID-19 may have immunity against future infection but the proportion who develop immunity is uncertain. Further, there is uncertainty about the proportion of individuals who get reinfected with COVID-19.</p></sec><sec><title>What this study adds</title><p>Using data from 54 studies with follow up time up to 8 months after recovery, during the period February 2020-February 2021, we found that, post-COVID-19, up to 90% of individuals had antibodies and memory T and B cells against SARS-CoV-2. We also found a pooled prevalence of reinfection of 0.2%, and that infection conferred an 81% decrease in odds of reinfection with SARS-CoV-2, compared to unimmunized individuals without previous COVID-19.</p><p>This review of 12 million individuals presents evidence that most individuals who recover from COVID-19 develop immunological memory to SARS-CoV-2, which was still detectable for up to 8 months. Further, reinfection after recovery from COVID-19 was rare during the first 8 months after recovery, with a prevalence below 1%, while prior infection confers protection with an odds ratio of 0.19 and a preventive efficacy of 80% at a baseline prevalence of 5% for COVID-19 in a community.</p></sec><sec><title>Implications of all the available evidence</title><p>Individuals with a history of COVID-19 infection have immunity against the disease for up to 8 months, although this period could be longer. These individuals could be prioritized last for COVID-19 vaccinations or considered for single dose vaccinations.</p></sec><sec><title>Strengths</title><p>This comprehensive review addresses key questions on prevalent immunological memory and risk of reinfection in individuals with prior confirmed COVID-19 using robust systematic review methods.</p></sec><sec><title>Limitations</title><p>Some of the included studies which examined prevalent immunological memory were small studies which were affected by loss to follow up. The review did not examine evidence for immunity against the new divergent variants, which may be more likely to have immune evasion behaviour and may present a higher risk of reinfection. Lastly, the review did not examine the effect of the severity of COVID-19 on both immunological memory and the risk of reinfection.</p></sec>', 'posted': '2021-09-07'}, {'doi': '10.1101/2021.09.03.458870', 'tweets': 106, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Behavioral state tunes mouse vision to ethological features through pupil dilation', 'authors': \"[{'name': 'Katrin Franke'}, {'name': 'Konstantin F. Willeke'}, {'name': 'Kayla Ponder'}, {'name': 'Mario Galdamez'}, {'name': 'Taliah Muhammad'}, {'name': 'Saumil Patel'}, {'name': 'Emmanouil Froudarakis'}, {'name': 'Jacob Reimer'}, {'name': 'Fabian Sinz'}, {'name': 'Andreas S. Tolias'}]\", 'abstract': '<p>Across animal species, sensory processing dynamically adapts to behavioral context. In the mammalian visual system, sensory neural responses and behavioral performance increase during an active behavioral state characterized by locomotion activity and pupil dilation, whereas preferred stimuli of individual neurons typically remain unchanged. Here, we address how behavioral states modulate stimulus selectivity in the context of colored natural scenes using a combination of large-scale population imaging, behavior, pharmacology, and deep neural network modeling. In visual cortex of awake mice, we identified a consistent shift of individual neuron color preferences towards ultraviolet stimuli during active behavioral periods that was particularly pronounced in the upper visual field. We found that the spectral shift in neural tuning is mediated by pupil dilation, resulting in a dynamic switch from rod-to cone-driven visual responses for constant ambient light levels. We further showed that this shift selectively enhances the discriminability of ultraviolet objects and facilitates the detection of ethological stimuli, such as aerial predators against the ultraviolet background of the twilight sky. Our results suggest a novel functional role for pupil dilation during active behavioral states as a bottom-up mechanism that, together with top-down neuromodulatory mechanisms, dynamically tunes visual representations to different behavioral demands.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.04.21263115', 'tweets': 101, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Durability analysis of the highly effective BNT162b2 vaccine against COVID-19', 'authors': \"[{'name': 'Arjun Puranik'}, {'name': 'Patrick J. Lenehan'}, {'name': 'John C. O’Horo'}, {'name': 'Michiel J.M. Niesen'}, {'name': 'Abinash Virk'}, {'name': 'Melanie D. Swift'}, {'name': 'Walter Kremers'}, {'name': 'A.J. Venkatakrishnan'}, {'name': 'Joel E. Gordon'}, {'name': 'Holly L. Geyer'}, {'name': 'Leigh Lewis Speicher'}, {'name': 'Venky Soundararajan'}, {'name': 'Andrew D. Badley'}]\", 'abstract': '<p>SARS-CoV-2 breakthrough infections have been increasingly reported in fully vaccinated individuals. We conducted a test-negative case-control study to assess the durability of protection after full vaccination with BNT162b2, defined as 14 days after the second dose, against polymerase chain reaction (PCR)-confirmed symptomatic SARS-CoV-2 infection, in a national medical practice between February 1, 2021 and August 22, 2021. We fit conditional logistic regression (CLR) models stratified on residential county and calendar time of testing to assess the association between time elapsed since vaccination and the odds of symptomatic infection or non-COVID-19 hospitalization (negative control), adjusted for several covariates. The primary population included 652 individuals who had a positive symptomatic test after full vaccination with BNT162b2 (cases) and 5,946 individuals with at least one negative symptomatic test after full vaccination (controls). The adjusted odds of symptomatic infection were higher 120 days after full vaccination versus at the date of full vaccination (Odds Ratio [OR]: 3.21, 95% confidence interval [CI]: 1.33-7.74). Importantly, the odds of infection were still lower 150 days after the first BNT162b2 dose as compared to 4 days after the first dose (OR: 0.3, 95% CI: 0.19-0.45), when immune protection approximates the unvaccinated status. Low rates of COVID-19 associated hospitalization or death in this cohort precluded analyses of these severe outcomes. The odds of experiencing a non-COVID-19 hospitalization decreased with time since vaccination, suggesting a possible underestimation of waning protection by this approach due to confounding factors. Taken together, these data constitute an early signal for waning protection against symptomatic illness while also providing reassurance that BNT162b2 continues to protect against symptomatic SARS-CoV-2 infection several months after full vaccination. Continued surveillance of COVID-19 vaccine durability, particularly against severe disease, is critical to guide effective and equitable strategies to respond to the pandemic, including distribution of booster doses, development of new vaccines, and implementation of both pharmaceutical and nonpharmaceutical interventions.</p>', 'posted': '2021-09-07'}, {'doi': '10.1101/2021.09.10.455288', 'tweets': 75, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Generation and diversification of recombinant monoclonal antibodies for studying mitosis', 'authors': \"[{'name': 'Keith F DeLuca'}, {'name': 'Jeanne E Mick'}, {'name': 'Amy L Hodges'}, {'name': 'Wanessa C Lima'}, {'name': 'Lori Sherman'}, {'name': 'Kristin L Schaller'}, {'name': 'Steven M Anderson'}, {'name': 'Ning Zhao'}, {'name': 'Timothy J Stasevich'}, {'name': 'Dileep Varma'}, {'name': 'Gary J Gorbsky'}, {'name': 'Jakob Nilsson'}, {'name': 'Jennifer G. DeLuca'}]\", 'abstract': '<p>Antibodies are indispensable tools used for a large number of applications in both foundational and translational bioscience research; however, there are drawbacks to using traditional antibodies generated in animals. These include a lack of standardization leading to problems with reproducibility, high costs of antibodies purchased from commercial sources, and ethical concerns regarding the large number of animals used to generate antibodies. To address these issues, we have developed practical methodologies and tools for generating low-cost, high-yield preparations of recombinant monoclonal antibodies and antibody fragments directed to protein epitopes from primary sequences. We describe these methods here, as well as approaches to diversify monoclonal antibodies, including customization of antibody species specificity, generation of genetically encoded small antibody fragments, and conversion of single chain antibody fragments (e.g. scFv) into full-length, bivalent antibodies. This study focuses on antibodies directed to epitopes important for mitotic cell division; however, the methods and reagents described here are applicable to antibodies and antibody fragments for use in any field.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.07.458941', 'tweets': 69, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Mapping electromagnetic networks to haemodynamic networks in the human brain', 'authors': \"[{'name': 'Golia Shafiei'}, {'name': 'Sylvain Baillet'}, {'name': 'Bratislav Misic'}]\", 'abstract': '<p>Whole-brain neural communication is typically estimated from statistical associations among electromagnetic or haemodynamic time-series. The relationship between functional network architectures recovered from these two types of neural activity remains unknown. Here we map electromagnetic networks (measured using magnetoencephalography; MEG) to haemodynamic networks (measured using functional magnetic resonance imaging; fMRI). We find that the relationship between the two modalities is regionally heterogeneous and systematically follows the cortical hierarchy, with close correspondence in unimodal cortex and poor correspondence in transmodal cortex, potentially reflecting patterns of laminar differentiation, recurrent subcortical input and neurovascular coupling. Correspondence between the two is largely driven by slower rhythms, particularly the delta (2-4 Hz) and beta (15-29 Hz) frequency band. Moreover, haemodynamic connectivity cannot be explained by electromagnetic activity in a single frequency band, but rather arises from the mixing of multiple neurophysiological rhythms. Collectively, these findings demonstrate highly organized but only partly overlapping patterns of connectivity in MEG and fMRI functional networks, opening fundamentally new avenues for studying the relationship between cortical micro-architecture and multi-modal connectivity patterns.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.07.459342', 'tweets': 59, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'ELeFHAnt: A supervised machine learning approach for label harmonization and annotation of single cell RNA-seq data', 'authors': \"[{'name': 'Konrad Thorner'}, {'name': 'Aaron M. Zorn'}, {'name': 'Praneet Chaturvedi'}]\", 'abstract': '<p>Annotation of single cells has become an important step in the single cell analysis framework. With advances in sequencing technology thousands to millions of cells can be processed to understand the intricacies of the biological system in question. Annotation through manual curation of markers based on a priori knowledge is cumbersome given this exponential growth. There are currently ~200 computational tools available to help researchers automatically annotate single cells using supervised/unsupervised machine learning, cell type markers, or tissue-based markers from bulk RNA-seq. But with the expansion of publicly available data there is also a need for a tool which can help integrate multiple references into a unified atlas and understand how annotations between datasets compare. Here we present ELeFHAnt: <italic>Ensemble learning for harmonization and annotation of single cells</italic>. ELeFHAnt is an easy-to-use R package that employs support vector machine and random forest algorithms together to perform three main functions: 1) <italic>CelltypeAnnotation</italic> 2) <italic>LabelHarmonization</italic> 3) <italic>DeduceRelationship</italic>. <italic>CelltypeAnnotation</italic> is a function to annotate cells in a query Seurat object using a reference Seurat object with annotated cell types. <italic>LabelHarmonization</italic> can be utilized to integrate multiple cell atlases (references) into a unified cellular atlas with harmonized cell types. Finally, <italic>DeduceRelationship</italic> is a function that compares cell types between two scRNA-seq datasets. ELeFHAnt can be accessed from GitHub at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/praneet1988/ELeFHAnt\">https://github.com/praneet1988/ELeFHAnt</ext-link>.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.08.459546', 'tweets': 56, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Humans reconfigure target and distractor processing to address distinct task demands', 'authors': \"[{'name': 'Harrison Ritz'}, {'name': 'Amitai Shenhav'}]\", 'abstract': '<p>When faced with distraction, we can focus more on goal-relevant information (targets) or focus less goal-conflicting information (distractors). How people decide to distribute cognitive control across targets and distractors remains unclear. To help address this question, we developed a parametric attentional control task with a graded manipulation to both target discriminability and distractor interference. We find that participants exert independent control over target and distractor processing. We measured control adjustments through the influence of incentives and previous conflict on target and distractor sensitivity, finding that these have dissociable influences on control. Whereas incentives preferentially led to target enhancement, conflict on the previous trial preferentially led to distractor suppression. These distinct drivers of control altered sensitivity to targets and distractors early in the trial, and were promptly followed by reactive reconfiguration towards task-appropriate feature sensitivity. Finally, we provide a process-level account of these findings by show that these control adjustments are well-captured by an evidence accumulation model with attractor dynamics over feature weights. These results help establish a process-level account of control configuration that provides new insights into how multivariate attentional signals are optimized to achieve task goals.</p>', 'posted': '2021-09-10'}, {'doi': '10.1101/2021.09.08.459528', 'tweets': 54, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Maternal SMCHD1 regulates Hox gene expression and patterning in the mouse embryo', 'authors': \"[{'name': 'Natalia Benetti'}, {'name': 'Quentin Gouil'}, {'name': 'Andres Tapia del Fierro'}, {'name': 'Tamara Beck'}, {'name': 'Kelsey Breslin'}, {'name': 'Andrew Keniry'}, {'name': 'Edwina McGlinn'}, {'name': 'Marnie E. Blewitt'}]\", 'abstract': '<p>Parents transmit genetic and epigenetic information to their offspring. Maternal effect genes regulate the offspring epigenome to ensure normal development. Here we report that the epigenetic regulator SMCHD1 has a maternal effect on <italic>Hox</italic> gene expression and skeletal patterning. Maternal SMCHD1, present in the oocyte and preimplantation embryo, prevents precocious activation of <italic>Hox</italic> genes postimplantation. Without maternal SMCHD1, highly penetrant posterior homeotic transformations occur in the embryo. <italic>Hox</italic> genes are decorated with Polycomb marks H2AK119ub and H3K27me3 from the oocyte throughout early embryonic development; however, loss of maternal SMCHD1 does not alter these marks. Therefore, we propose maternal SMCHD1 acts downstream of Polycomb marks to establish a chromatin state necessary for persistent epigenetic silencing and appropriate <italic>Hox</italic> gene expression later in the developing embryo. This is a striking role for maternal SMCHD1 in long-lived epigenetic effects impacting offspring phenotype.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.10.459833', 'tweets': 49, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Population differentiation of polygenic score predictions under stabilizing selection', 'authors': \"[{'name': 'Sivan Yair'}, {'name': 'Graham Coop'}]\", 'abstract': '<p>Given the many loci uncovered by genome-wide association studies (GWAS), polygenic scores have become central to the drive for genomic medicine and have spread into various areas including evolutionary studies of adaptation. While promising, these scores are fraught with issues of portability across populations, due to the misestimation of effect sizes and missing causal loci across populations not represented in large-scale GWAS. The poor portability of polygenic scores at first seems at odds with the view that much of common genetic variation is shared among populations (Lewontin 1972). Here we investigate one potential cause of this discrepancy: phenotypic stabilizing selection drives the turnover of genetic variation shared between populations at causal loci. Somewhat counter-intuitively, while stabilizing selection to the same optimum phenotype leads to lower phenotypic differentiation among populations, it increases genetic differentiation at GWAS loci and reduces the portability of polygenic scores constructed for unrepresented populations. We also find that stabilizing selection can lead to potentially misleading signals of the differentiation of average polygenic scores among populations. We extend our baseline model to investigate the impact of pleiotropy, gene-by-environment interactions, and directional selection on polygenic score predictions. Our work emphasizes stabilizing selection as a null evolutionary model to understand patterns of allele frequency differentiation and its impact on polygenic score portability and differentiation.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.06.459138', 'tweets': 45, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Mapping of a N-terminal α-helix domain required for human PINK1 stabilisation, Serine228 autophosphorylation and activation in cells', 'authors': \"[{'name': 'Poonam Kakade'}, {'name': 'Hina Ojha'}, {'name': 'Olawale G. Raimi'}, {'name': 'Andrew Shaw'}, {'name': 'Andrew D. Waddell'}, {'name': 'James R. Ault'}, {'name': 'Sophie Burel'}, {'name': 'Kathrin Brockmann'}, {'name': 'Atul Kumar'}, {'name': 'Mohd Syed Ahangar'}, {'name': 'Ewelina M. Krysztofinska'}, {'name': 'Thomas Macartney'}, {'name': 'Richard Bayliss'}, {'name': 'Julia C. Fitzgerald'}, {'name': 'Miratul M. K. Muqit'}]\", 'abstract': '<p>Human autosomal recessive mutations in the <italic>PINK1</italic> gene are causal for Parkinson’s disease (PD). PINK1 encodes a mitochondrial localised protein kinase that is a master-regulator of mitochondrial quality control pathways. Structural studies to date have elaborated the mechanism of how mutations located within the kinase domain disrupt PINK1 function, however, the molecular mechanism of PINK1 mutations located upstream and downstream of the kinase domain are unknown. We have employed mutagenesis studies of human PINK1 in cells to define the minimal region of PINK1, required for optimal ubiquitin phosphorylation, beginning at residue Ile111. Bioinformatic analysis of the region spanning Ile111 to the kinase domain and inspection of the AlphaFold human PINK1 structure model predicts a conserved N-terminal α-helical domain extension (NTE domain) within this region corroborated by hydrogen/deuterium exchange mass spectrometry (HDX-MS) of recombinant insect PINK1 protein. The AlphaFold structure also predicts the NTE domain forms an intramolecular interaction with the C-terminal extension (CTE). Cell-based analysis of human PINK1 reveals that PD-associated mutations (e.g. Q126P), located within the NTE:CTE interface, markedly inhibit stabilization of PINK1; autophosphorylation at Serine228 (Ser228); and Ubiquitin Serine65 (Ser65) phosphorylation. Furthermore, we provide evidence that NTE domain mutants do not affect intrinsic catalytic kinase activity but do disrupt PINK1 stabilisation at the mitochondrial Translocase of outer membrane (TOM) complex. The clinical relevance of our findings is supported by the demonstration of defective stabilization and activation of endogenous PINK1 in human fibroblasts of a patient with early-onset PD due to homozygous PINK1 Q126P mutations. Overall, we define a functional role of the NTE:CTE interface towards PINK1 stabilisation and activation and show that loss of NTE:CTE interactions is a major mechanism of PINK1-associated mutations linked to PD.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.05.458952', 'tweets': 42, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Community science designed ribosomes with beneficial phenotypes', 'authors': '[{\\'name\\': \\'Antje Kruger\\'}, {\\'name\\': \\'Andrew M Watkins\\'}, {\\'name\\': \\'Roger Wellington-Oguri\\'}, {\\'name\\': \\'Jonathan Romano\\'}, {\\'name\\': \\'Camila Kofman\\'}, {\\'name\\': \\'Alysse DeFoe\\'}, {\\'name\\': \\'Yejun Kim\\'}, {\\'name\\': \\'Jeff Anderson-Lee\\'}, {\\'name\\': \\'Eli Fisker\\'}, {\\'name\\': \\'Jill Townley\\'}, {\\'name\\': \"Anne E d\\'Aquino\"}, {\\'name\\': \\'Rhiju Das\\'}, {\\'name\\': \\'Michael C Jewett\\'}, {\\'name\\': \\' \\'}]', 'abstract': '<p>Functional design of ribosomes with mutant ribosomal RNA (rRNA) could expand opportunities for understanding molecular translation, building cells from the bottom-up, and engineering ribosomes with altered capabilities. However, such efforts have been hampered by cell viability constraints, an enormous combinatorial sequence space, and limitations on large-scale, 3D design of RNA structures and functions. To address these challenges, we developed an integrated community science and experimental screening approach for rational design of ribosomes. This approach couples Eterna, an online video game that crowdsources RNA sequence design to community scientists in the form of puzzles, with in vitro ribosome synthesis, assembly, and translation in multiple design-build-test-learn cycles. We applied our framework to discover mutant rRNA sequences that improve protein synthesis in vitro and cell growth in vivo, relative to wild type ribosomes, under diverse environmental conditions. This work provides new insights into ribosome rRNA sequence-function relationships, with implications for synthetic biology.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.06.459169', 'tweets': 42, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PHIST: fast and accurate prediction of prokaryotic hosts from metagenomic viral sequences', 'authors': \"[{'name': 'Andrzej Zielezinski'}, {'name': 'Sebastian Deorowicz'}, {'name': 'Adam Gudyś'}]\", 'abstract': '<sec><title>Summary</title><p>PHIST (Phage-Host Interaction Search Tool) predicts prokaryotic hosts of viruses from their genomic sequences. It improves host prediction accuracy at species level over current alignment-based tools (on average by 3 percentage points) as well as alignment-free and CRISPR-based tools (by 14–20 percentage points). PHIST is also two orders of magnitude faster than alignment-based tools making it suitable for metagenomics studies.</p></sec><sec><title>Availability and implementation</title><p>GNU-licensed C++ code wrapped in Python API available at: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/refresh-bio/phist\">https://github.com/refresh-bio/phist</ext-link></p></sec><sec><title>Contact</title><p><email>andrzej.zielezinski@amu.edu.pl</email>, <email>adam.gudys@polsl.pl</email></p></sec><sec><title>Supplementary information</title><p>Supplementary data are available at publisher Web site.</p></sec>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.11.459888', 'tweets': 34, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Activation of stably silenced genes by recruitment of a synthetic de-methylating module', 'authors': \"[{'name': 'Wing Fuk Chan'}, {'name': 'Christine R. Keenan'}, {'name': 'Timothy M Johanson'}, {'name': 'Rhys S Allan'}]\", 'abstract': '<p>Stably silenced genes that display a high level of CpG dinucleotide methylation are refractory to the current generation of dCas9-based activation systems. To counter this, we created an improved activation system by coupling the catalytic domain of DNA demethylating enzyme TET1 with transcriptional activators (TETact). TETact induces transcription of heavily suppressed non-coding RNA and surface protein, and the reactivation of embryonic haemoglobin genes in non-erythroid cells.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.10.459858', 'tweets': 31, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Engineered bacteria detect tumor DNA in vivo', 'authors': \"[{'name': 'Robert M Cooper'}, {'name': 'Josephine A Wright'}, {'name': 'Jia Q Ng'}, {'name': 'Jarrad M Goyne'}, {'name': 'Nobumi Suzuki'}, {'name': 'Young K Lee'}, {'name': 'Mari Ichinose'}, {'name': 'Georgette Radford'}, {'name': 'Elaine M Thomas'}, {'name': 'Laura Vrbanac'}, {'name': 'Rob Knight'}, {'name': 'Susan L Woods'}, {'name': 'Daniel L Worthley'}, {'name': 'Jeff Hasty'}]\", 'abstract': '<p>In vitro nucleic acid analysis has become a valuable diagnostic tool. However, in vitro measurements have many disadvantages when compared to in vivo techniques. Synthetic bacterial biosensors have been engineered to sense many target signals in vivo, but no biosensor exists to detect specific DNA sequences. Here, we engineered naturally competent Acinetobacter baylyi bacteria to detect engineered donor DNA inserted into the genomes of colorectal cancer (CRC) cells and organoids. The DNA biosensor concept was developed in vitro and then validated in vivo with sensor bacteria delivered orally or rectally to mice that had been injected with orthotopic donor CRC organoids. Horizontal gene transfer occurred from the donor tumor to the sensor bacteria in vivo, conferring antibiotic resistance to the sensor bacteria and allowing their detection in stool. The sensor bacteria differentiated mice with and without CRC. Life detecting life has many implications for future diagnosis, prevention, and treatment of disease. This approach may also be useful in any application that requires the detection of mutations or organisms within environments that are difficult to sample.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.10.459728', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'MetaCoAG: Binning Metagenomic Contigs via Composition, Coverage and Assembly Graphs', 'authors': \"[{'name': 'Vijini Mallawaarachchi'}, {'name': 'Yu Lin'}]\", 'abstract': '<p>Metagenomics binning has allowed us to study and characterize various genetic material of different species and gain insights into microbial communities. While existing binning tools bin metagenomics de novo assemblies, they do not make use of the assembly graphs that produce such assemblies. Here we propose MetaCoAG, a tool that utilizes assembly graphs with the composition and coverage information to bin metagenomic contigs. MetaCoAG uses single-copy marker genes to estimate the number of initial bins, assigns contigs into bins iteratively and adjusts the number of bins dynamically throughout the binning process. Experimental results on simulated and real datasets demonstrate that MetaCoAG significantly outperforms state-of-the-art binning tools, producing more high-quality bins than the second-best tool, with an average median F1-score of 88.40%. To the best of our knowledge, MetaCoAG is the first stand-alone binning tool to make direct use of the assembly graph information. MetaCoAG is available at https://github.com/Vini2/MetaCoAG.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.04.458985', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Spectrum graph-based de-novo sequencing algorithm MaxNovo achieves high peptide identification rates in collisional dissociation MS/MS spectra', 'authors': \"[{'name': 'Petra Gutenbrunner'}, {'name': 'Pelagia Kyriakidou'}, {'name': 'Frido Welker'}, {'name': 'Jürgen Cox'}]\", 'abstract': '<p>We describe MaxNovo, a novel spectrum graph-based peptide de-novo sequencing algorithm integrated into the MaxQuant software. It identifies complete sequences of peptides as well as sequence tags that are incomplete at one or both of the peptide termini. MaxNovo searches for the highest-scoring path in a directed acyclic graph representing the MS/MS spectrum with peaks as nodes and edges as potential sequence constituents consisting of single amino acids or pairs. The raw score is a sum of node and edge weights, plus several reward scores, for instance, for complementary ions or protease compatibility. For search-engine identified peptides, it correlates well with the Andromeda search engine score. We use a particular score normalization and the score difference between the first and second-best solution to define a combined score that integrates all available information. To evaluate its performance, we use a human cell line dataset and take as ground truth all Andromeda-identified MS/MS spectra with an Andromeda score of at least 100. MaxNovo outperforms other software in particular in the high-sensitivity range of precision-coverage plots. We also identify incomplete sequence tags and study their statistical properties. Next, we apply MaxNovo to ion mobility-coupled time of flight data. Here we achieve excellent performance as well, except for potential swaps of the two amino acids closest to the C-terminus, which are not well resolved due to the low end of the mass range in MS/MS spectra in this dataset. We demonstrate the applicability of MaxNovo to palaeoproteomics samples with a Late Pleistocene hominin proteome dataset that was generated using three proteases. Interestingly, we did not use any machine learning in the construction of MaxNovo, but implemented expert domain knowledge directly in the definition of the score. Yet, it performs as good as or better than the leading deep learning-based algorithm.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.09.459620', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Variant surface glycoprotein expression in human African trypanosomiasis patients', 'authors': \"[{'name': 'Jaime So'}, {'name': 'Sarah Sudlow'}, {'name': 'Abeer Sayeed'}, {'name': 'Tanner Grudda'}, {'name': 'Stijn Deborggraeve'}, {'name': 'Dieudonné Mumba Ngoyi'}, {'name': 'Didier Kashiama Desamber'}, {'name': 'Bill Wickstead'}, {'name': 'Veerle Lejon'}, {'name': 'Monica R. Mugnier'}]\", 'abstract': '<p><italic>Trypanosoma brucei gambiense</italic>, an extracellular protozoan parasite, is the primary causative agent of human African Trypanosomiasis. <italic>T. b. gambiense</italic> is endemic to West and Central Africa where it is transmitted by the bite of infected tsetse flies. In the bloodstream of an infected host, the parasite evades antibody recognition by altering the Variant Surface Glycoprotein (VSG) that forms a dense coat on its cell surface through a process known as antigenic variation. Each VSG has a variable N-terminal domain that is exposed to the host and a less variable C-terminal domain that is at least partially hidden from host antibodies. Our lab developed VSG-seq, a targeted RNA-seq method, to study <italic>VSG</italic> expression in <italic>T. brucei</italic>. Studies using VSG-seq to characterize antigenic variation in a mouse model have revealed marked diversity in <italic>VSG</italic> expression within parasite populations, but this finding has not yet been validated in a natural human infection. Here, we used VSG-seq to analyze <italic>VSGs</italic> expressed in the blood of twelve patients infected with <italic>T. b. gambiense</italic>. The number of <italic>VSGs</italic> identified per patient ranged from one to fourteen and, notably, two <italic>VSGs</italic> were shared by more than one patient. Analysis of expressed <italic>VSG</italic> N-terminal domain types revealed that 82% of expressed <italic>VSGs</italic> encoded a type B N-terminus, a bias not seen in datasets from other <italic>T. brucei</italic> subspecies. C-terminal types in <italic>T. b. gambiense</italic> infection were also restricted. These results demonstrate a bias either in the underlying <italic>VSG</italic> repertoire of <italic>T. b. gambiense</italic> or in the selection of <italic>VSGs</italic> from the repertoire during infection. This work demonstrates the feasibility of using VSG-seq to study antigenic variation in human infections and highlights the importance of understanding <italic>VSG</italic> repertoires in the field.</p><sec><title>Author Summary</title><p>Human African Trypanosomiasis is a neglected tropical disease largely caused by the extracellular parasite known as <italic>Trypanosoma brucei gambiense</italic>. To avoid elimination by the host, these parasites repeatedly replace their dense surface coat of Variant Surface Glycoprotein (VSG). Despite the important role of VSGs in prolonging infection, <italic>VSG</italic> expression during natural human infections is poorly understood. A better understanding of natural <italic>VSG</italic> expression dynamics can clarify the mechanisms which <italic>T. brucei</italic> uses to alter its VSG coat and improve how trypanosomiasis is diagnosed in humans. We analyzed the expressed <italic>VSGs</italic> detected in the blood of patients with trypanosomiasis. Our findings indicate that a diverse range of <italic>VSGs</italic> are expressed in both natural and experimental infections.</p></sec>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.07.459228', 'tweets': 22, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Differentially optimized cell-free buffer enables robust expression from unprotected linear DNA in exonuclease-deficient extracts', 'authors': \"[{'name': 'Angelo Cardoso Batista'}, {'name': 'Antoine Levrier'}, {'name': 'Paul Soudier'}, {'name': 'Peter Voyvodic'}, {'name': 'Tatjana Achmedov'}, {'name': 'Tristan Reif-Trauttmansdorff'}, {'name': 'Angelique DeVisch'}, {'name': 'Martin Cohen Gonsaud'}, {'name': 'Jean-Loup Faulon'}, {'name': 'Chase Beisel'}, {'name': 'Jerome Bonnet'}, {'name': 'Manish Kushwaha'}]\", 'abstract': '<p>The use of linear DNA templates in cell-free systems promises to accelerate the prototyping and engineering of synthetic gene circuits. A key challenge is that linear templates are rapidly degraded by exonucleases present in cell extracts. Current approaches tackle the problem by adding exonuclease inhibitors and DNA-binding proteins to protect the linear DNA, requiring additional time- and resource-intensive steps. Here, we delete the <italic>recBCD</italic> exonuclease gene cluster from the <italic>Escherichia coli</italic> BL21 genome. We show that the resulting cell-free systems, with buffers optimized specifically for linear DNA, enable near-plasmid levels of expression from σ70 promoters in linear DNA templates without employing additional protection strategies. When using linear or plasmid DNA templates at the buffer calibration step, we found that the optimal potassium glutamate concentrations obtained when using linear DNA were consistently lower than those obtained when using plasmid DNA for the same extract. We demonstrate the robustness of the exonuclease deficient extracts across seven different batches and a wide range of experimental conditions. Finally, we illustrate the use of the Δ<italic>recBCD</italic> extracts for two applications: toehold switch characterization and enzyme screening. Our work provides a simple, efficient, and costeffective solution for using linear DNA templates in cell-free systems and highlights the importance of specifically tailoring buffer composition for the final experimental setup. Our data also suggest that similar exonuclease deletion strategies can be applied to other species suitable for cell-free synthetic biology.</p>', 'posted': '2021-09-07'}, {'doi': '10.1101/2021.09.06.459145', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'A DCL3 dicing code within Pol IV-RDR2 transcripts diversifies the siRNA pool guiding RNA-directed DNA methylation', 'authors': \"[{'name': 'Andrew Loffer'}, {'name': 'Jasleen Singh'}, {'name': 'Akihito Fukudome'}, {'name': 'Vibhor Mishra'}, {'name': 'Feng Wang'}, {'name': 'Craig S. Pikaard'}]\", 'abstract': '<p>In plants, selfish genetic elements including retrotransposons and DNA viruses are transcriptionally silenced by RNA-directed DNA methylation. Guiding the process are short interfering RNAs (siRNAs) cut by DICER-LIKE 3 (DCL3) from double-stranded precursors of ∼30 bp synthesized by NUCLEAR RNA POLYMERASE IV (Pol IV) and RNA-DEPENDENT RNA POLYMERASE 2 (RDR2). We show that Pol IV initiating nucleotide choice, RDR2 initiation 1-2 nt internal to Pol IV transcript ends and RDR2 terminal transferase activity collectively yield a code that influences which end of the precursor is diced and whether 24 or 23 nt siRNAs are generated from the Pol IV or RDR2-transcribed strands. By diversifying the size, sequence, and strand polarity of siRNAs derived from a given precursor, alternative patterns of DCL3 dicing allow maximal siRNA coverage at methylated target loci.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.08.459370', 'tweets': 20, 'archive': 'bioRxiv', 'subject-area': 'Scientific Communication and Education', 'covid': False, 'title': 'Methylothon: a versatile course-based high school research experience in microbiology and bioinformatics-- with pink bacteria', 'authors': \"[{'name': 'Peyton A. Jones'}, {'name': 'David Frischer'}, {'name': 'Shannon Mueller'}, {'name': 'Thomas Le'}, {'name': 'Anya Schwanes'}, {'name': 'Alekhya Govindaraju'}, {'name': 'Katie Shalvarjian'}, {'name': 'Jean-Baptiste Leducq'}, {'name': 'Christopher J. Marx'}, {'name': 'N. Cecilia Martinez-Gomez'}, {'name': 'Jessica A. Lee'}]\", 'abstract': '<p>Methylothon is an inquiry-based high school learning module in microbial ecology, molecular biology, and bioinformatics that centers around pink-pigmented plant-associated methylotrophic bacteria. Here we present an overview of the module’s learning goals, describe course resources (available for public use on <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://methylothon.com\">http://methylothon.com</ext-link>), and relate lessons learned from adapting Methylothon for remote learning during the pandemic in spring of 2021. The original in-person version of the module allows students to isolate their own strains of methylotrophic bacteria from plants they sample from the environment, to identify these using PCR, sequencing, and phylogenetic analysis, and to contribute their strains to original research in a university lab. The adapted version strengthens the focus on bioinformatics and increases its flexibility and accessibility by making the lab portion optional and adopting free web-based tools. Student feedback and graded assignments from Spring 2021 revealed that the lesson was especially effective at introducing the concepts of BLAST and phylogenetic trees, and that students valued and felt inspired by the opportunity to conduct hands-on work and to participate in community science.</p>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.07.459279', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'eQTLs are key players in the integration of genomic and transcriptomic data for phenotype prediction', 'authors': \"[{'name': 'Abdou Rahmane Wade'}, {'name': 'Harold Duruflé'}, {'name': 'Leopoldo Sanchez'}, {'name': 'Vincent Segura'}]\", 'abstract': '<p>Multi-omics represent a promising link between phenotypes and genome variation. Few studies yet address their integration to understand genetic architecture and improve predictability. Our study used 241 poplar genotypes, phenotyped in two common gardens, with their xylem and cambium RNA sequenced at one site, yielding large phenotypic, genomic and transcriptomic datasets. For each trait, prediction models were built with genotypic or transcriptomic data and compared to concatenation integrating both omics. The advantage of integration varied across traits and, to understand such differences, we made an eQTL analysis to characterize the interplay between the genome and the transcriptome and classify the predicting features into CIS or TRANS relationships. A strong and significant negative correlation was found between the change in predictability and the change in predictor importance for eQTLs (both TRANS and CIS effects) and CIS regulated transcripts, and mostly for traits showing beneficial integration and evaluated in the site of transcriptomic sampling. Consequently, beneficial integration happens when redundancy of predictors is decreased, leaving the stage to other less prominent but complementary predictors. An additional GO enrichment analysis appeared to corroborate such statistical output. To our knowledge, this is a novel finding delineating a promising way to explore data integration.</p><sec><title>One-sentence summary</title><p>Successful multi-omics integration when predicting phenotypes makes redundant the predictors that are linked to ubiquitous connections between the omics, according to biological and statistical approaches</p></sec>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.10.459829', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Dopamine biases sensitivity to personal goals and social influence in self-control over everyday desires', 'authors': \"[{'name': 'Jaime J Castrellon'}, {'name': 'Jacob S Young'}, {'name': 'Linh C Dang'}, {'name': 'Christopher T Smith'}, {'name': 'Ronald L Cowan'}, {'name': 'David H Zald'}, {'name': 'Gregory R Samanez-Larkin'}]\", 'abstract': '<p>People regularly give in to daily temptations in spite of conflict with personal goals. To test hypotheses about neuropharmacological influences on self-control, we used positron emission tomography to measure dopamine D2-like receptors (D2R) and experience sampling surveys to naturalistically track daily desires outside the laboratory in everyday life in a sample of 103 adults. Higher D2R availability in the ventral striatum was associated with increased sensitivity to personal goal conflict but not desire strength in deciding whether to attempt to resist a desire. The influence of D2Rs on sensitivity to personal goal conflict depended on whether desires were experienced in a social context. D2R availability in the midbrain (but not the ventral striatum) influenced whether desires were enacted. These findings provide unique evidence that the dopamine system influences decision making and regulatory behavior and provides new insights into how these mechanisms interact with personal goals and social contexts.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.09.459619', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Single-nucleus transcriptomic analyses reveal microglial activation underlying cerebellar degeneration in Ataxia Telangiectasia', 'authors': \"[{'name': 'Jenny Lai'}, {'name': 'Junho Kim'}, {'name': 'Ailsa M. Jeffries'}, {'name': 'Allie Tolles'}, {'name': 'Thomas W. Chittenden'}, {'name': 'Patrick G. Buckley'}, {'name': 'Timothy W. Yu'}, {'name': 'Michael A. Lodato'}, {'name': 'Eunjung Alice Lee'}]\", 'abstract': '<p>While <italic>ATM</italic> loss-of-function has long been identified as the genetic cause of Ataxia Telangiectasia (AT), how this genetic mutation leads to selective and progressive cerebellar degeneration of Purkinje and granule cells remains unknown. We performed single-nucleus RNA-sequencing of the human cerebellum and prefrontal cortex from individuals with AT and matched unaffected controls to identify AT-associated transcriptomic changes in a cell-type- and brain-region-specific manner. We provide the largest single-nucleus transcriptomic atlas of the adult human cerebellum to-date (126,356 nuclei), identify upregulation of apoptotic and ER stress pathways in Purkinje and granule neurons, and uncover strong downregulation of calcium ion homeostasis genes in Purkinje neurons. Our analysis reveals prominent inflammation of microglia in AT cerebellum with transcriptional signatures similar to aging and neurodegenerative microglia, and suggests that microglia activation precedes Purkinje and granule neuron death in disease progression. Our data implicates a novel role of microglial activation underlying cerebellar degeneration in AT.</p>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.07.459266', 'tweets': 17, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'The temporal dynamics of the sea urchin regulome', 'authors': \"[{'name': 'Roberto Feuda'}]\", 'abstract': '<p>In this work, we used Nanostring N-counter technology, to evaluate the mRNA expression level of more than 330 regulatory genes over 34-time points covering the first three days of development of the sea urchin larvae. The hierarchical clustering of the mRNAs expression levels has identified groups corresponding to the major developmental landmarks (e.g., maternal to zygotic transition and gastrulation). Furthermore, comparison with previous experiments indicates high reproducibility of mRNA level temporal dynamics across batches. Finally, we generated an online tool to visualize gene expression during sea urchin larval development. The site can be accessed at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://nanostring2021.herokuapp.com/\">https://nanostring2021.herokuapp.com/</ext-link>.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.07.459270', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Real age prediction from the transcriptome with RAPToR', 'authors': \"[{'name': 'Romain Bulteau'}, {'name': 'Mirko Francesconi'}]\", 'abstract': '<p>Genome-wide gene expression profiling is a powerful tool for exploratory analyses, providing a high dimensional picture of the state of a biological system. However, uncontrolled variation among samples can obscure and confound the effect of variables of interest. Uncontrolled developmental variation is often a major source of unknown expression variation in developmental systems. Existing methods to sort samples from transcriptomes require many samples to infer developmental trajectories and only provide a relative pseudo-time.</p><p>Here we present RAPToR (<underline><bold>R</bold></underline>eal <underline><bold>A</bold></underline>ge <underline><bold>P</bold></underline>rediction from <underline><bold>T</bold></underline>ranscriptome staging <underline><bold>o</bold></underline>n <underline><bold>R</bold></underline>eference), a simple computational method to estimate the absolute developmental age of even a single sample from its gene expression with up to minutes precision. We achieve this by staging samples on high-resolution reference developmental expression profiles we build from existing time series data. We implemented RAPToR for the most common animal model systems: nematode, fruit fly, zebrafish, and mouse, and demonstrate application for non-model organisms. We show how developmental variation discovered by RAPToR can be exploited to increase power to detect differential expression and to untangle the signal of perturbations of interest even when it is completely confounded with development. We anticipate our RAPToR post-profiling staging strategy will be especially useful in large scale single organism profiling because it eliminates the need for synchronization or for a tedious and potentially difficult step of accurate staging before profiling.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.08.458611', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'CaPTure: Calcium PeakToolbox for analysis of in vitro calcium imaging data', 'authors': \"[{'name': 'Madhavi Tippani'}, {'name': 'Elizabeth A. Pattie'}, {'name': 'Brittany A. Davis'}, {'name': 'Claudia V. Nguyen'}, {'name': 'Yanhong Wang'}, {'name': 'Srinidhi Rao Sripathy'}, {'name': 'Brady J. Maher'}, {'name': 'Keri Martinowich'}, {'name': 'Andrew E. Jaffe'}, {'name': 'Stephanie Cerceo Page'}]\", 'abstract': '<sec><title>Background</title><p>Calcium imaging is a powerful technique for recording cellular activity across large populations of neurons. However, analysis methods capable of single-cell resolution in cultured neurons, especially for cultures derived from human induced pluripotent stem cells (hiPSCs), are lacking. Existing methods lack scalability to accommodate high-throughput comparisons between multiple lines, across developmental timepoints, or across pharmacological manipulations.</p></sec><sec><title>Results</title><p>We developed a scalable, automated Ca<sup>2+</sup> imaging analysis pipeline called CaPTure (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/LieberInstitute/CaPTure\">https://github.com/LieberInstitute/CaPTure</ext-link>). This method detects neurons, classifies and quantifies spontaneous activity, quantifies synchrony metrics, and generates cell- and network-specific metrics that facilitate phenotypic discovery. The method is compatible with parallel processing on computing clusters without requiring significant user input or parameter modification.</p></sec><sec><title>Conclusion</title><p>CaPTure allows for rapid assessment of neuronal activity in cultured cells at cellular resolution, rendering it amenable to high-throughput screening and phenotypic discovery. The platform can be applied to both human- and rodent-derived neurons and is compatible with many imaging systems.</p></sec>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.06.459211', 'tweets': 13, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Assembly formation is stabilized by Parvalbumin neurons and accelerated by Somatostatin neurons', 'authors': \"[{'name': 'Fereshteh Lagzi'}, {'name': 'Martha Canto Bustos'}, {'name': 'Anne-Marie Oswald'}, {'name': 'Brent Doiron'}]\", 'abstract': '<p>Learning entails preserving the features of the external world in the neuronal representations of the brain, and manifests itself in the form of strengthened interactions between neurons within assemblies. Hebbian synaptic plasticity is thought to be one mechanism by which correlations in spiking promote assembly formation during learning. While spike timing dependent plasticity (STDP) rules for excitatory synapses have been well characterized, inhibitory STDP rules remain incomplete, particularly with respect to sub-classes of inhibitory interneurons. Here, we report that in layer 2/3 of the orbitofrontal cortex of mice, inhibition from parvalbumin (PV) interneurons onto excitatory (E) neurons follows a symmetric STDP function and mediates homeostasis in E-neuron firing rates. However, inhibition from somatostatin (SOM) interneurons follows an asymmetric, Hebbian STDP rule. We incorporate these findings in both large scale simulations and mean-field models to investigate how these differences in plasticity impact network dynamics and assembly formation. We find that plasticity of SOM inhibition builds lateral inhibitory connections and increases competition between assemblies. This is reflected in amplified correlations between neurons within assembly and anti-correlations between assemblies. An additional finding is that the emergence of tuned PV inhibition depends on the interaction between SOM and PV STDP rules. Altogether, we show that incorporation of differential inhibitory STDP rules promotes assembly formation through competition, while enhanced inhibition both within and between assemblies protects new representations from degradation after the training input is removed.</p>', 'posted': '2021-09-07'}, {'doi': '10.1101/2021.09.08.459542', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Distal regulation, silencers and a shared combinatorial syntax are hallmarks of animal embryogenesis', 'authors': \"[{'name': 'Paola Cornejo-Páramo'}, {'name': 'Kathrein Roper'}, {'name': 'Sandie M Degnan'}, {'name': 'Bernard M Degnan'}, {'name': 'Emily S Wong'}]\", 'abstract': '<p>The chromatin environment plays a central role in regulating developmental gene expression in metazoans. Yet, the basal regulatory landscape of metazoan embryogenesis is unknown. Here, we generate chromatin accessibility profiles for six embryonic, plus larval and adult stages in the sponge <italic>Amphimedon queenslandica</italic>. These profiles are reproducible within stages, reflect histone modifications, and identify transcription factor (TF) binding sequence motifs predictive of <italic>cis</italic>-regulatory elements during embryogenesis in other metazoans but not the unicellular relative <italic>Capsaspora</italic>. Motif analysis of chromatin accessibility profiles across <italic>Amphimedon</italic> embryogenesis identifies three major developmental periods. As in bilaterian embryogenesis, early development in <italic>Amphimedon</italic> involves activating and repressive chromatin in regions both proximal and distal to transcription start sites. Transcriptionally repressive elements (‘silencers’) are prominent during late embryogenesis and coincide with an increase in <italic>cis</italic>-regulatory regions harbouring metazoan TF binding motifs, and an increase in the expression of metazoan-specific genes. Changes in chromatin state and gene expression in <italic>Amphimedon</italic> suggest the conservation of distal enhancers, dynamically silenced chromatin, and TF-DNA binding specificity in animal embryogenesis.</p>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.04.21263123', 'tweets': 12, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Assessment of COVID-19 intervention strategies in the Nordic countries using genomic epidemiology', 'authors': \"[{'name': 'Sebastian Duchene'}, {'name': 'Leo Featherstone'}, {'name': 'Birgitte Freiesleben de Blasio'}, {'name': 'Edward C. Holmes'}, {'name': 'Jon Bohlin'}, {'name': 'John H.-O. Pettersson'}]\", 'abstract': '<p>The Nordic countries, defined here as Norway, Sweden, Denmark, Finland and Iceland, are known for their comparable demographics and political systems. Since these countries implemented different COVID-19 intervention strategies, they provide a natural laboratory for examining how COVID-19 policies and mitigation strategies affected the propagation, evolution and spread of the SARS-CoV-2 virus. We explored how the duration, the size and number of transmission clusters, defined as country-specific monophyletic groups in a SARS-CoV-2 phylogenetic tree, differed between the Nordic countries. We found that Sweden had the largest number of COVID-19 transmission clusters followed by Denmark, Norway, Finland and Iceland. Moreover, Sweden and Denmark had the largest, and most enduring, transmission clusters followed by Norway, Finland and Iceland. In addition, there was a significant positive association between transmission cluster size and duration, suggesting that the size of transmission clusters could be reduced by rapid and effective contact tracing. Thus, these data indicate that to reduce the general burden of COVID-19 there should be a focus on limiting dense gatherings and their subsequent contacts to keep the number, size and duration of transmission clusters to a minimum. Our results further suggest that although geographical connectivity, population density and openness influence the spread and the size of SARS-CoV-2 transmission clusters, country-specific intervention strategies had the largest single impact.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.07.459322', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Illuminating links between cis-regulators and trans-acting variants in the human prefrontal cortex', 'authors': \"[{'name': 'Shuang Liu'}, {'name': 'Hyejung Won'}, {'name': 'Declan Clarke'}, {'name': 'Nana Matoba'}, {'name': 'Saniya Khullar'}, {'name': 'Yudi Mu'}, {'name': 'Daifeng Wang'}, {'name': 'Mark Gerstein'}]\", 'abstract': '<p>Psychiatric disorders exact immense human and economic tolls in societies globally. Underlying many of these disorders is a complex repertoire of genomic variants that influence the expression of genes involved in pathways and processes in the brain. Identifying such variants and their associated brain functions is thus essential for understanding the molecular underpinnings of psychiatric disorders. Genome-wide association studies (GWASes) have provided many variants associated with these disorders; however, our knowledge of the precise biological mechanisms by which these contribute to disease remains limited. In connection with this, expression quantitative trait loci (eQTLs) have provided useful information linking variants to genes and functions. However, most eQTL studies on human brain have focused exclusively on cis-eQTLs. A complete understanding of disease etiology should also include trans-regulatory mechanisms. Thus, we conduct one of the first genome-wide surveys of trans-eQTLs in the dorsolateral prefrontal cortex (DLPFC) by leveraging the large datasets from the PsychENCODE consortium. We identified ∼80,000 trans-eQTLs. We found that a significant number of these overlap with cis-eQTLs, thereby implicating cis-mediators as key players in trans-acting regulation. We show, furthermore, that trans-regulatory mechanisms provide novel insights into psychiatric disease. Particularly, colocalization analysis between trans-eQTLs and schizophrenia (SCZ) GWAS loci identified 90 novel SCZ risk genes and 23 GWAS loci previously uncharacterized by cis-eQTLs. Moreover, these 90 genes tend to be more central in transcriptome-wide co-expression networks and more susceptible to rare variants than SCZ-risk genes associated by cis-variation.</p>', 'posted': '2021-09-07'}, {'doi': '10.1101/2021.09.08.459552', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'MOLECULAR VIDEOGAMING: SUPER-RESOLVED TRAJECTORY-BASED NANOCLUSTERING ANALYSIS USING SPATIO-TEMPORAL INDEXING', 'authors': \"[{'name': 'Tristan P. Wallis'}, {'name': 'Anmin Jiang'}, {'name': 'Huiyi Hou'}, {'name': 'Rachel S. Gormal'}, {'name': 'Nela Durisic'}, {'name': 'Giuseppe Balistreri'}, {'name': 'Merja Joensuu'}, {'name': 'Frédéric A. Meunier'}]\", 'abstract': '<p>Single-molecule localization microscopy (SMLM) techniques are emerging as vital tools to unravel the nanoscale world of living cells. However, current analysis methods primarily focus on defining spatial nanoclusters based on detection density, but neglect important temporal information such as cluster lifetime and recurrence in “hotspots” on the plasma membrane. Spatial indexing is widely used in videogames to effectively detect interactions between moving geometric objects. Here, we use the R-tree spatial indexing algorithm to perform SMLM data analysis and determine whether the bounding boxes of individual molecular trajectories overlap, as a measure of their potential membership in nanoclusters. Extending the spatial indexing into the time dimension allows unique resolution of spatial nanoclusters into multiple spatiotemporal clusters. We have validated this approach using synthetic and SMLM-derived data. Quantitative characterization of recurring nanoclusters allowed us to demonstrate that both syntaxin1a and Munc18-1 molecules transiently cluster in hotspots on the neurosecretory plasma membrane, offering unprecedented insights into the dynamics of these protein which are essential to neuronal communication. This new analytical tool, named Nanoscale Spatiotemporal Indexing Clustering (NASTIC), has been implemented as a free and open-source Python graphic user interface.</p>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.05.459049', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Vulnerability to cavitation is linked to home climate precipitation across eight eucalypt species', 'authors': \"[{'name': 'David Coleman'}, {'name': 'Andrew Merchant'}, {'name': 'William T. Salter'}]\", 'abstract': '<p>Vulnerability to cavitation in leaves is the result of highly adaptive anatomical and physiological traits that can be linked to water availability in a species’ climate of origin. Despite similar gross leaf morphology, eucalypt species are often confined to specific climate envelopes across the variable rainfall environments of Australia. In this study, we investigate how the progression of cavitation differs among eucalypts and whether this is related to other hydraulic and physical leaf traits. We used the Optical Visualisation technique to capture cavitation progression across the leaves of eight eucalypt species (<italic>Angophora crassifolia, Corymbia tessellaris, Eucalyptus atrata, Eucalyptus grandis, Eucalyptus laevopinea, Eucalyptus longifolia, Eucalyptus macrandra, Eucalyptus tereticornis</italic>) from a wide range of climates and grown in a common garden setting. Vulnerability to cavitation, represented by the leaf water potential required for 50% cavitation of leaf vessels, varied significantly among species (−3.48 MPa to −8.25 MPa) and correlated linearly with home climate precipitation and leaf SLA (<italic>R</italic><sup>2</sup> of 0.64 and 0.75, respectively). P12-P88, the range of water potentials between which 12% to 88% of cavitation occurs, was decoupled from P50 but also correlated with leaf SLA (<italic>R</italic><sup>2</sup> of 0.72). We suggest the magnitude of P12-P88 may be representative of a species’ drought strategy – a large P12-P88 signifying leaves that exhibit drought tolerance (retention of leaves under drought conditions) and a small P12-P88 signifying drought avoidance (leaf shedding after a threshold of drought is reached). Our results agree with other studies that highlight these cavitation metrics as genetically fixed traits. Turgor loss point, on the other hand, may be more plastic, as evidenced by the low variability of this trait across these eucalypt species grown in a common garden environment. Further study will help to establish the SLA-related anatomical traits that impart cavitation resistance and to extend these conclusions to a greater number of species and home climates.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.10.459763', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'clipplotr - a comparative visualisation and analysis tool for CLIP data', 'authors': \"[{'name': 'Anob M Chakrabarti'}, {'name': 'Charlotte Capitanchik'}, {'name': 'Jernej Ule'}, {'name': 'Nicholas Luscombe'}]\", 'abstract': '<p>CLIP technologies are now widely used to study RNA-protein interactions and many datasets are now publicly available. An important first step in CLIP data exploration is the visual inspection and assessment of processed genomic data on selected genes or regions and performing comparisons: either across conditions within a particular project, or incorporating publicly available data. However, the output files produced by data processing pipelines or preprocessed files available to download from data repositories are often not suitable for direct comparison and usually need further processing. Furthermore, to derive biological insight it is usually necessary to visualise CLIP signal alongside other data such as annotations, or orthogonal functional genomic data (e.g. RNA-seq). We have developed a simple, but powerful, command-line tool: clipplotr, which facilitates these visual comparative and integrative analyses with normalisation and smoothing options for CLIP data and the ability to show these alongside reference annotation tracks and functional genomic data. These data can be supplied as input to clipplotr in a range of file formats, which will output a publication quality figure. It is written in R and can both run on a laptop computer independently, or be integrated into computational workflows on a high-performance cluster. Releases, source code and documentation are freely available at: https://github.com/ulelab/clipplotr.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.09.459613', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Distinct, opposing functions for CFIm59 and CFIm68 in mRNA alternative polyadenylation of Pten and in the PI3K/Akt signalling cascade', 'authors': \"[{'name': 'Hsin-Wei Tseng'}, {'name': 'Anthony Mota-Sydor'}, {'name': 'Rania Leventis'}, {'name': 'Ivan Topisirovic'}, {'name': 'Thomas F. Duchaine'}]\", 'abstract': '<p>The precise maintenance of PTEN dosage is crucial for tumor suppression across a wide variety of cancers. Post-transcriptional regulation of <italic>Pten</italic> heavily relies on regulatory elements encoded by its 3’UTR. We previously reported the important diversity of 3’UTR isoforms of <italic>Pten</italic> mRNAs produced through alternative polyadenylation (APA). Here, we reveal the direct regulation of <italic>Pten</italic> APA by the mammalian cleavage factor I (CFIm) complex, which in turn contributes to PTEN protein dosage. CFIm consists of the UGUA-binding CFIm25 and APA regulatory subunits CFIm59 or CFIm68. Deep sequencing analyses of perturbed (KO and KD) cell lines uncovered the differential regulation of <italic>Pten</italic> APA by CFIm59 and CFIm68 and further revealed that their divergent functions have widespread impact for APA in transcriptomes. Differentially regulated genes include numerous factors within the phosphoinositide 3-kinase (PI3K)/protein kinase B (Akt) signalling pathway that PTEN counter-regulates. We further reveal a stratification of APA dysregulation among a subset of <italic>PTEN-driven</italic> cancers, with recurrent alterations among PI3K/Akt pathway genes regulated by CFIm. Our results refine the transcriptome selectivity of the CFIm complex in APA regulation, and the breadth of its impact in <italic>PTEN</italic>-driven cancers.</p>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.06.459120', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'tascCODA: Bayesian tree-aggregated analysis of compositional amplicon and single-cell data', 'authors': \"[{'name': 'Johannes Ostner'}, {'name': 'Salomé Carcy'}, {'name': 'Christian L. Müller'}]\", 'abstract': '<p>Accurate generative statistical modeling of count data is of critical relevance for the analysis of biological datasets from high-throughput sequencing technologies. Important instances include the modeling of microbiome compositions from amplicon sequencing surveys and the analysis of cell type compositions derived from single-cell RNA sequencing. Microbial and cell type abundance data share remarkably similar statistical features, including their inherent compositionality and a natural hierarchical ordering of the individual components from taxonomic or cell lineage tree information, respectively. To this end, we introduce a Bayesian model for <bold>t</bold>ree-aggregated <bold>a</bold>mplicon and <bold>s</bold>ingle-<bold>c</bold>ell <bold>co</bold>mpositional <bold>d</bold>ata <bold>a</bold>nalysis (<monospace>tascCODA</monospace>) that seamlessly integrates hierarchical information and experimental covariate data into the generative modeling of compositional count data. By combining latent parameters based on the tree structure with spike-and-slab Lasso penalization, <monospace>tascCODA</monospace> can determine covariate effects across different levels of the population hierarchy in a data-driven parsimonious way. In the context of differential abundance testing, we validate <monospace>tascCODA</monospace>’s excellent performance on a comprehensive set of synthetic benchmark scenarios. Our analyses on human single-cell RNA-seq data from ulcerative colitis patients and amplicon data from patients with irritable bowel syndrome, respectively, identified aggregated cell type and taxon compositional changes that were more predictive and parsimonious than those proposed by other schemes. We posit that tascCODA<sup>1</sup> constitutes a valuable addition to the growing statistical toolbox for generative modeling and analysis of compositional changes in microbial or cell population data.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.07.459309', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Mutation, selection, and quantitative genetic architecture of susceptibility to bacterial pathogens in C. elegans', 'authors': \"[{'name': 'Lindsay M. Johnson'}, {'name': 'Sayran Saber'}, {'name': 'Md. Monjurul Islam Rifat'}, {'name': 'Sydney Rouse'}, {'name': 'Charles F. Baer'}]\", 'abstract': '<p>Understanding the evolutionary and genetic underpinnings of susceptibility to pathogens is of fundamental importance across a wide swathe of biology. Much theoretical and empirical effort has focused on genetic variants of large effect, but pathogen susceptibility often appears to be a polygenic complex trait. Here we investigate the quantitative genetics of survival over 120 hours of exposure (“susceptibility”) of <italic>C. elegans</italic> to three bacterial pathogens of varying virulence, along with the non-pathogenic OP50 strain of <italic>E. coli</italic>. We compare the genetic (co)variance input by spontaneous mutations accumulated under minimal selection to the standing genetic (co)variance in a set of ∼50 wild isolates. Three conclusions emerge. First, with one exception, mutations increase susceptibility to pathogens, and susceptibility is uncorrelated with fitness in the absence of pathogens. Second, the orientation in trait space of the heritable (co)variance of wild isolates is sufficiently explained by mutation. However, pathogen susceptibility is clearly under purifying, apparently directional, selection of magnitude similar to that of competitive fitness in the MA conditions. The results provide no evidence for fitness tradeoffs between pathogen susceptibility and fitness in the absence of pathogens, nor that balancing selection is important in maintaining genetic variation for susceptibility to these bacterial pathogens.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.07.458711', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Targeting metabolic adaptations in the breast cancer–liver metastatic niche using dietary approaches to improve endocrine therapy efficacy', 'authors': \"[{'name': 'Qianying Zuo'}, {'name': 'Ayca Nazli Mogol'}, {'name': 'Yu-Jeh Liu'}, {'name': 'Ashlie Santaliz Casiano'}, {'name': 'Christine Chien'}, {'name': 'Jenny Drnevich'}, {'name': 'Ozan Berk Imir'}, {'name': 'Eylem Kulkoyluoglu-Cotul'}, {'name': 'Nicole Hwajin Park'}, {'name': 'David J Shapiro'}, {'name': 'Ben Ho Park'}, {'name': 'Yvonne Ziegler'}, {'name': 'Benita S. Katzenellenbogen'}, {'name': 'Evelyn Aranda'}, {'name': 'John D. O’Neill'}, {'name': 'Akshara Singareeka Raghavendra'}, {'name': 'Debu Tripathy'}, {'name': 'Zeynep Madak Erdogan'}]\", 'abstract': '<p>Estrogen receptor-positive (ER<sup>+</sup>) metastatic tumors contribute to nearly 70% of breast cancer-related deaths. Most patients with ER<sup>+</sup> metastatic breast cancer (MBC) undergo treatment with the estrogen receptor agonist fulvestrant (Fulv) as standard of care. Yet, among such patients, metastasis in liver is associated with reduced overall survival compared to other metastasis sites. The factors underlying the reduced responsiveness of liver metastases to ER agonists remain unknown, impeding the development of more effective treatment approaches to improve outcomes for patients with ER<sup>+</sup> liver metastases. We therefore evaluated site-specific changes in MBC cells and determined the mechanisms through which the liver metastatic niche specifically influences ER<sup>+</sup> tumor metabolism and drug resistance. We characterized ER activity of MBC cells both in vitro, using a novel system of tissue-specific extracellular matrix hydrogels representing the stroma of ER<sup>+</sup> tumor metastatic sites (liver, lung and bone), and in vivo, in liver and lung metastasis mouse models. ER<sup>+</sup> metastatic liver tumors and MBC cells grown in liver hydrogels displayed upregulated expression of glucose metabolism enzymes in response to Fulv. Furthermore, differential ERα activity, but not expression, was detected in liver hydrogels. In vivo, increased glucose metabolism led to increased glycogen deposition in liver metastatic tumors, while a fasting-mimicking diet increased efficacy of Fulv treatment to reduce the metastatic burden. Our findings identify a novel mechanism of endocrine resistance driven by the liver tumor microenvironment. These results may guide the development of dietary strategies to circumvent drug resistance in liver metastasis, with potential applicability in other metastatic diseases.</p>', 'posted': '2021-09-07'}, {'doi': '10.1101/2021.09.09.459708', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'HypVW is an HOCl-Sensing Two Component System in Escherichia coli', 'authors': \"[{'name': 'Sara El Hajj'}, {'name': 'Camille Henry'}, {'name': 'Alexandra Vergnes'}, {'name': 'Laurent Loiseau'}, {'name': 'Brasseur Gael'}, {'name': 'Romain Barré'}, {'name': 'Laurent Aussel'}, {'name': 'Benjamin Ezraty'}]\", 'abstract': '<p>Two component systems (TCS) are signalling pathways that allow bacterial cells to sense, respond and adapt to fluctuating environments. Among the classical TCS of Escherichia coli, YedVW has been recently showed to be involved in the regulation of msrPQ, encoding for the periplasmic methionine sulfoxide reductase system. In this study, we demonstrate that hypochlorous acid (HOCl) induces the expression of msrPQ in a YedVW dependant manner, whereas H<sub>2</sub>O<sub>2</sub>, NO and paraquat (a superoxide generator) do not. Therefore, YedV appears to be an HOCl-sensing histidine kinase. Based on this finding, we proposed to rename this system HypVW.\\xa0 Moreover, using a directed mutagenesis approach, we show that Met residues located in the periplasmic loop of HypV (formerly YedV) are important for its activity. Given that HOCl oxidizes preferentially Met residues, we bring evidences that HypV could be activated via the reversible oxidation of its methionine residues, thus conferring to MsrPQ a role in switching HypVW off. Based on these results, we propose that the activation of HypV by HOCl could occur through a Met redox switch. HypVW appears to be the first characterized TCS able to detect HOCl in E. coli. This study represents an important step in understanding the mechanisms of reactive chlorine species resistance in prokaryotes.</p>', 'posted': '2021-09-12'}, {'doi': '10.1101/2021.09.09.459641', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': True, 'title': 'Targeting Stem-loop 1 of the SARS-CoV-2 5’UTR to suppress viral translation and Nsp1 evasion', 'authors': \"[{'name': 'Setu M. Vora'}, {'name': 'Pietro Fontana'}, {'name': 'Valerie Leger'}, {'name': 'Ying Zhang'}, {'name': 'Tian-Min Fu'}, {'name': 'Judy Lieberman'}, {'name': 'Lee Gehrke'}, {'name': 'Ming Shi'}, {'name': 'Longfei Wang'}, {'name': 'Hao Wu'}]\", 'abstract': '<p>SARS-CoV-2 is a highly pathogenic virus that evades anti-viral immunity by interfering with host protein synthesis, mRNA stability, and protein trafficking. The SARS-CoV-2 nonstructural protein 1 (Nsp1) uses its C-terminal domain to block the mRNA entry channel of the 40S ribosome to inhibit host protein synthesis. However, how SARS-CoV-2 circumvents Nsp1-mediated suppression for viral protein synthesis and if the mechanism can be targeted therapeutically remain unclear. Here we show that N- and C-terminal domains of Nsp1 coordinate to drive a tuned ratio of viral to host translation, likely to maintain a certain level of host fitness while maximizing replication. We reveal that the SL1 region of the SARS-CoV-2 5’ UTR is necessary and sufficient to evade Nsp1-mediated translational suppression. Targeting SL1 with locked nucleic acid antisense oligonucleotides (ASOs) inhibits viral translation and makes SARS-CoV-2 5’ UTR vulnerable to Nsp1 suppression, hindering viral replication in vitro at a nanomolar concentration. Thus, SL1 allows Nsp1 to switch infected cells from host to SARS-CoV-2 translation, presenting a therapeutic target against COVID-19 that is conserved among immune-evasive variants. This unique strategy of unleashing a virus’ own virulence mechanism against itself could force a critical trade off between drug resistance and pathogenicity.</p>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.07.459250', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': False, 'title': 'GPCR-G protein selectivity – a unified meta-analysis', 'authors': \"[{'name': 'Alexander S. Hauser'}, {'name': 'Charlotte Avet'}, {'name': 'Claire Normand'}, {'name': 'Arturo Mancini'}, {'name': 'Asuka Inoue'}, {'name': 'Michel Bouvier'}, {'name': 'David E. Gloriam'}]\", 'abstract': '<p>Two-thirds of human hormones and one-third of clinical drugs act on membrane receptors that couple to G proteins to achieve appropriate functional responses. While G protein transducers from literature are annotated in the Guide to Pharmacology database, two recent large-scale datasets now expand the receptor-G protein ‘couplome’. However, these three datasets differ in scope and reported G protein couplings giving different coverage and conclusions on GPCR-G protein signaling. Here, we report a meta-analysis unifying GPCR-G protein coupling, by standardized normalization and consensus support, into a common coupling map. This unravels novel consensus couplings for receptors supported by two independent sources and insights on coupling selectivity of GPCRs and classification of co-coupling G proteins. The coupling protocol, map and selectivity resources will promote advances in receptor research and cellular signaling towards the exploitation of G protein signaling specificity in design of safer drugs.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.08.452919', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'CardIAP: Calcium images analyzer web application', 'authors': \"[{'name': 'Ana Julia Velez Rueda'}, {'name': 'Agustín García Smith'}, {'name': 'Luis Alberto Gonano'}, {'name': 'Maria Silvina Fornasari'}, {'name': 'Gustavo Parisi'}, {'name': 'Leandro Matías Sommese'}]\", 'abstract': '<sec><title>Motivation</title><p>Ionic calcium (Ca2+) plays the role of the second messenger in eukaryotic cells associated with cellular functions of regulation of the cell cycle, such as transport, motility, gene expression, and metabolism (Permyakov and Kretsinger, 2009). The use of fluorometric techniques in isolated cells, loaded with Ca2+ sensitive fluorescent probes allows the quantitative measurement of dynamic events that occur in living, functioning cells. The Cardiomyocytes Images Analyzer Application (CardIAP) covers the need for tools to analyze and retrieve information from confocal microscopy images, in a systematic, accurate, and fast way.</p></sec><sec><title>Results</title><p>Here we present the CardIAP web app, an automated method for the identification of spatio-temporal patterns in a calcium fluorescence imaging sequence. Through this tool, users can analyze single or multiple Ca2+ transients from confocal line-scan images and obtain quantitative information on the dynamic response of the stimulated myocyte.</p><p>Our web application also allows the user the extraction of data on calcium dynamics in downloadable tables and plots, simplifying the calculation of the alternation and discordance indices and their classification. CardIAP could assist in studying the underlying mechanisms of anomalous calcium release phenomena.</p></sec><sec><title>Availability and implementation</title><p>CardIAP is an open-source app, entirely developed in Python, which can be freely accessed and used at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://cardiap.herokuapp.com/\">http://cardiap.herokuapp.com/</ext-link>.</p></sec>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.09.459666', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Multiplex knockout of trichome-regulating MYBs in poplar affects light sensitivity and triterpene accumulation', 'authors': \"[{'name': 'W. Patrick Bewg'}, {'name': 'Scott A Harding'}, {'name': 'Nancy Engle'}, {'name': 'Brajesh N Vaidya'}, {'name': 'Jingyin Yu'}, {'name': 'Ran Zhou'}, {'name': 'Jacob Reeves'}, {'name': 'Thomas W Horn'}, {'name': 'Nirmal Joshee'}, {'name': 'Jerry W Jenkins'}, {'name': 'Shengqiang Shu'}, {'name': 'Kerrie W. Barry'}, {'name': 'Yuko Yoshinaga'}, {'name': 'Jane Grimwood'}, {'name': 'Robert J. Schmitz'}, {'name': 'Jeremy Schmutz'}, {'name': 'Timothy Tschaplinski'}, {'name': 'Chung-Jui Tsai'}]\", 'abstract': '<p>Hair-like trichomes cover the aerial organs of many plant species and act as a barrier between a plant and its environment. They function in defense against biotic and abiotic stresses, while also serving as sites for synthesis and storage of secondary metabolites. Previously, the transcription factor PtaMYB186 was identified as a positive regulator of trichome initiation during early stages of leaf development in Populus tremula x P. alba (IRNA 717-1B4). However, trichome regulation in poplar remains largely unexplored, as does the functional redundancy of duplicated poplar genes. Here, we employed CRISPR/Cas9 to target a consensus region of PtaMYB186 and its close paralogs for knockout. Regeneration of glabrous mutants suggested their essential roles in poplar trichome development. No apparent differences in growth and leaf transpiration rates between the mutants and the controls were observed, but trichomeless poplars showed increased insect pest susceptibility. RNA-seq analysis revealed widespread down-regulation of circadian- and light-responsive genes in the mutants. When exposed to a high light regime, trichomeless mutants accumulated significantly higher levels of photoprotective anthocyanins. Cuticular wax and whole leaf analyses showed a complete absence of triterpenes in the mutants, suggesting biosynthesis and storage of triterpenes in poplar occurs in the non-glandular trichomes. This work also demonstrates that a single gRNA with SNP-aware design is sufficient for multiplex targeting of paralogous genes in outcrossing and/or hybrid species with unexpected copy number variations.</p>', 'posted': '2021-09-10'}, {'doi': '10.1101/2021.09.03.458917', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Strategy-dependent effects of working-memory limitations on human perceptual decision-making', 'authors': \"[{'name': 'Kyra Schapiro'}, {'name': 'Krešimir Josić'}, {'name': 'Zachary P. Kilpatrick'}, {'name': 'Joshua I. Gold'}]\", 'abstract': '<p>Deliberative decisions based on an accumulation of evidence over time depend on working memory, and working memory has limitations, but how these limitations affect deliberative decision-making is not understood. We used human psychophysics to assess the impact of working-memory limitations on the fidelity of a continuous decision variable. Participants decided the average location of multiple visual targets. This computed, continuous decision variable degraded with time and capacity in a manner that depended critically on the strategy used to form the decision variable. This dependence reflected whether the decision variable was computed either: 1) immediately upon observing the evidence, and thus stored as a single value in memory; or 2) at the time of the report, and thus stored as multiple values in memory. These results provide important constraints on how the brain computes and maintains temporally dynamic decision variables.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.10.459864', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Cysteine induces mitochondrial reductive stress in glioblastoma through hydrogen peroxide production', 'authors': \"[{'name': 'Evan K. Noch'}, {'name': 'Laura Palma'}, {'name': 'Isaiah Yim'}, {'name': 'Daniel Barnett'}, {'name': 'Alexander Walsh'}, {'name': 'Bhavneet Bhinder'}, {'name': 'Elisa Benedetti'}, {'name': 'Jan Krumsiek'}, {'name': 'Justin Gurvitch'}, {'name': 'Sumaiyah Khwaja'}, {'name': 'Olivier Elemento'}, {'name': 'Lewis Cantley'}]\", 'abstract': '<p>Glucose and amino acid metabolism are critical for glioblastoma (GBM) growth, but little is known about the specific metabolic alterations in GBM that are targetable with FDA-approved compounds. To investigate tumor metabolism signatures unique to GBM, we interrogated The Cancer Genome Atlas for alterations in glucose and amino acid signatures in GBM relative to other human cancers and found that GBM exhibits the highest levels of cysteine and methionine pathway gene expression of 32 human cancers. Treatment of patient-derived GBM cells with the FDA-approved cysteine compound N-acetylcysteine (NAC) reduce GBM cell growth and mitochondrial oxygen consumption, which was worsened by glucose starvation. Mechanistic experiments revealed that cysteine compounds induce rapid mitochondrial hydrogen peroxide production and reductive stress in GBM cells, an effect blocked by oxidized glutathione, thioredoxin, and redox enzyme overexpression. These findings indicate that GBM is uniquely susceptible to NAC-driven reductive stress and could synergize with glucose-lowering treatments for GBM.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.07.459340', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Atria: An Ultra-fast and Accurate Trimmer for Adapter and Quality Trimming', 'authors': \"[{'name': 'Jiacheng Chuan'}, {'name': 'Aiguo Zhou'}, {'name': 'Lawrence Richard Hale'}, {'name': 'Miao He'}, {'name': 'Xiang Li'}]\", 'abstract': '<sec><title>Background</title><p>As Next Generation Sequencing takes a dominant role in terms of output capacity and sequence length, adapters attached to the reads and low-quality bases hinder the performance of downstream analysis directly and implicitly, such as producing false-positive single nucleotide polymorphisms (SNP), and generating fragmented assemblies. A fast trimming algorithm is in demand to remove adapters precisely, especially in read tails with relatively low quality.</p></sec><sec><title>Findings</title><p>We present a trimming program named Atria. Atria matches the adapters in paired reads and finds possible overlapped regions with a super-fast and carefully designed byte-based matching algorithm (<italic>O(n)</italic> time with <italic>O(1)</italic> space). Atria also implements multi-threading in both sequence processing and file compression and supports single-end reads.</p></sec><sec><title>Conclusions</title><p>Atria performs favorably in various trimming and runtime benchmarks of both simulated and real data with other cutting-edge trimmers. We also provide an ultra-fast and lightweight byte-based matching algorithm. The algorithm can be used in a broad range of short-sequence matching applications, such as primer search and seed scanning before alignment.</p></sec><sec><title>Availability &amp; Implementation</title><p>The Atria executables, source code, and benchmark scripts are available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/cihga39871/Atria\">https://github.com/cihga39871/Atria</ext-link> under the MIT license.</p></sec>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.10.459766', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Combining CRISPR/Cas9 and brain imaging: from genes to proteins to networks', 'authors': \"[{'name': 'Sabina Marciano'}, {'name': 'Tudor M. Ionescu'}, {'name': 'Ran Sing Saw'}, {'name': 'Rachel Y. Cheong'}, {'name': 'Deniz Kirik'}, {'name': 'Andreas Maurer'}, {'name': 'Bernd J. Pichler'}, {'name': 'Kristina Herfert'}]\", 'abstract': \"<p>Understanding how functional connections between brain regions are arranged and influenced by genes, in the healthy and diseased states, is a major goal in neuroscience. Functional connectivity alterations are linked to molecular changes in several neurodegenerative disorders and could serve as early markers of pathology. Yet, the underlying molecular signatures driving the functional alterations remain largely unknown. Here, we combine CRISPR/Cas9 gene-editing with in vivo positron emission tomography (PET) and functional magnetic resonance imaging (fMRI) to investigate the direct link between genes, proteins, and the brain connectome. The extensive knowledge of the Slc18a2 gene encoding the vesicular monoamine transporter (VMAT2), involved in the storage and release of dopamine, makes it an excellent basis for studying the gene networks relationships. We edited Slc18a2 into the substantia nigra pars compacta of adult rats and used in vivo molecular imaging, behavioral, histological, and biochemical assessments to characterize the CRISPR/Cas9-mediated VMAT2 knockdown. Simultaneous PET/fMRI was performed to inspect the functional brain adaptations, beyond the predicted dopaminergic changes. Further, [11C]flumazenil PET was carried out to investigate the dopamine-GABA interplay. We found a regional increase in postsynaptic dopamine receptor availability, preceded by a reorganization of brain networks that adapt to reduced dopamine transmission states by becoming functionally connected and organized. The hyperconnectivity within and between brain networks spreads from the contralateral thalamus and prefrontal cortical regions to the striata and hippocampi. Additionally, impaired striatal dopamine release reduces GABA-A receptor availability, complementing the increased synchrony and functional connectivity between networks observed at rest. Our study reveals that recruiting different brain networks may be an early response to the dopaminergic dysfunction preceding neuronal cell loss, postsynaptic changes, and motor impairment in neurodegenerative disorders such as Parkinson's disease. We anticipate our combinatorial approach to be a starting point to investigate the impact of specific genes on brain molecular and functional dynamics, aiding in the identification of early neurobiological markers and promising therapeutic interventions.</p>\", 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.07.459353', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Concurrent Multimodal Data Acquisition During Brain Scanning is within Reach', 'authors': \"[{'name': 'Rosa Sola Molina'}, {'name': 'Gemma Lamp'}, {'name': 'Laila Hugrass'}, {'name': 'Russell Beaton'}, {'name': 'Marten de Man'}, {'name': 'Lisa Wise'}, {'name': 'David Crewther'}, {'name': 'Melvyn Goodale'}, {'name': 'Sheila Crewther'}]\", 'abstract': '<sec><title>Background</title><p>Previous brain-scanning research exploring the neural mechanisms underpinning visuomotor planning and control has mostly been done without simultaneous motion-tracking and eye-tracking. Employing concurrent methodologies would enhance understanding of the brain mechanisms underlying visuomotor integration of cognitive, visual, ocular, and motor aspects of reaching and grasping behaviours. Therefore, this work presents the methods and validation for a high-speed, multimodal and synchronized system to holistically examine neural processes that are involved in visually-guided movement.</p></sec><sec><title>Methods</title><p>The multimodal methods included high speed 3D motion tracking (Qualisys), 2D eye-tracking (SR Research), and magnetoencephalography (MEG; Elekta) that were synchronized to millisecond precision. Previous MRIs were taken to provide improved spatial localization. The methods section describes the system layout and acquisition parameters to achieve multimodal synchronization. Pilot results presented here are preliminary data from a larger study including 29 participants. Using a pincer grip, five people (3 male, 2 female, ages 30-32) reached for and grasped a translucent dowel 50 times, after it was pseudorandomly illuminated. The object illumination was the Go cue. Seven discrete time points (events) throughout the task were chosen for investigation of simultaneous brain, hand and eye activity associated with specific visual (Go cue), oculomotor (1<sup>st</sup> saccade after Go), motor (Reaction Time; RT, Maximum Velocity: MV, Maximum Grip Width; MGW) or cognitive (Ready, End) mechanisms. Time-frequency analyses were performed on the MEG data sourced from the left precentral gyrus to explore task-related changes time-locked to these chosen events.</p></sec><sec><title>Pilot results</title><p>Basic kinematic parameters including RT, MV, MGW, Movement Time, and Total Time were similar to previous, seminal research by Castiello, Paulignan and Jeannerod, (1991), using a similar task. Although no gaze instructions were given, eye-tracking results indicated volunteers mostly gazed at or near the target object when Ready (72%), and then hardly looked away throughout the rest of the task at the important events sampled here (92% - 98%). At the End event, when lifting the dowel, on average, participants gazed at or near the target object 100% of the time. Although saccades &gt; 100 ms after Go, but prior to RT were made on average in about one fourth (<italic>M</italic> = 13, <italic>SD</italic> = 6) of trials, a mixed model (REML) indicated their latency in timing after the Go was significantly (<italic>F</italic> = 13.376, <italic>p</italic> = .001) associated with RT scores on those trials (<italic>AIC</italic> = 724, <italic>R</italic><sub><italic>m</italic></sub><sup>2</sup> = 0.407, <italic>R</italic><sub><italic>c</italic></sub><sup>2</sup>= 0.420). Neural activity relative to baseline in the beta band was desynchronized for the visually guided reach periods, beginning prior to Go, and remaining sustained until beyond End, after the grasp and lift were executed.</p></sec><sec><title>Conclusion</title><p>This study presents the layout, acquisition parameters and validation for a multimodal, synchronized system designed to record data from the hand, eye and brain simultaneously, with millisecond precision during an ecologically-valid prehension task with physical, 3D objects. The pilot results align with previous research made with single or bimodal data recordings. This multimodal method enables full-brain modelling that can holistically map the precise location and timing of neural activity involved in the visual, oculomotor, motor and cognitive aspects of reach-to-grasp planning and control.</p></sec>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.08.459470', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Classification of antimicrobial mechanism of action using bacterial time-lapse imaging', 'authors': \"[{'name': 'Xudong Ouyang'}, {'name': 'Jelmer Hoeksma'}, {'name': 'Tjalling K. Siersma'}, {'name': 'Leendert W. Hamoen'}, {'name': 'Jeroen den Hertog'}]\", 'abstract': '<p>Antimicrobial resistance is a major threat to human health. Basic knowledge of antimicrobial mechanism of action (MoA) is imperative for patient care and for identification of novel antimicrobials. However, the process of antimicrobial MoA identification is relatively laborious. Here, we developed a simple, quantitative time-lapse fluorescence imaging method, Dynamic Bacterial Morphology Imaging (DBMI), to facilitate this process. It uses a membrane dye and a nucleoid dye to track the morphological changes of single <italic>Bacillus subtilis</italic> cells in response to antimicrobials for up to 60 min. DBMI of bacterial cells facilitated assignment of the MoAs of 14 distinct, known antimicrobial compounds to the five main classes. Using this method, we found that the poorly studied antimicrobial, harzianic acid, a secondary metabolite that we purified from the fungal culture of <italic>Oidiodendron flavum</italic>, targets the cell envelope. We conclude that DBMI is a simple method, which facilitates rapid classification of the MoA of antimicrobials in functionally distinct classes.</p>', 'posted': '2021-09-08'}, {'doi': '10.1101/2021.09.09.459678', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The Cortical Spectrum: a robust structural continuum in primate cerebral cortex revealed by histological staining and magnetic resonance imaging', 'authors': \"[{'name': 'Yohan J. John'}, {'name': 'Basilis Zikopoulos'}, {'name': 'Miguel Ángel García-Cabezas'}, {'name': 'Helen Barbas'}]\", 'abstract': '<p>High-level characterizations of the primate cerebral cortex sit between two extremes: on one end the cortical mantle is seen as a mosaic of structurally and functionally unique areas, and on the other it is seen as a uniform six-layered structure in which functional differences are defined solely by extrinsic connections. Neither of these extremes captures the crucial neuroanatomical finding: that the cortex exhibits systematic gradations in architectonic structure. These gradations have been shown to predict cortico-cortical connectivity, which in turn suggests powerful ways to ground connectomics in anatomical structure, and by extension cortical function. A challenge to more widespread use of this concept is the labor-intensive and invasive nature of histological staining, which is the primary means of recognizing anatomical gradations. Here we show that a novel computational analysis technique can be used to derive a coarse-grained picture of cortical variation. For each of 78 cortical areas spanning the entire cortical mantle of the rhesus macaque, we created a high dimensional set of anatomical features derived from captured images of cortical tissue stained for myelin and SMI-32. The method involved semi-automated de-noising of images, and enabled comparison of brain areas without hand-labeling of features such as layer boundaries. We applied nonmetric multidimensional scaling (NMDS) to the dataset to visualize similarity among cortical areas. This analysis shows a systematic variation between weakly laminated (limbic) cortices and sharply laminated (eulaminate) cortices. We call this smooth continuum the ‘cortical spectrum’. We also show that this spectrum is visible within subsystems of the cortex: the occipital, parietal, temporal, motor, prefrontal, and insular cortices. We compared the NMDS-derived spectrum with a spectrum produced using T1- and T2-weighted magnetic resonance imaging (MRI) data derived from macaque, and found close agreement of the two coarse-graining methods. This evidence suggests that T1/T2 data, routinely obtained in human MRI studies, can be used as an effective proxy for data derived from high-resolution histological methods. More generally, this approach shows that the cortical spectrum is robust to the specific method used to compare cortical areas, and is therefore a powerful tool to understand the principles of organization of the primate cortex.</p>', 'posted': '2021-09-10'}, {'doi': '10.1101/2021.09.11.459906', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'PV neurons improve cortical complex scene analysis by enhancing timing-based coding', 'authors': \"[{'name': 'Jian Carlo Nocon'}, {'name': 'Howard J Gritton'}, {'name': 'Nicholas M James'}, {'name': 'Xue Han'}, {'name': 'Kamal Sen'}]\", 'abstract': '<p>Cortical representations underlying a wide range of cognitive abilities, which employ both rate and spike timing-based coding, emerge from underlying cortical circuits with a tremendous diversity of cell types. However, cell-type specific contributions to cortical coding are not well-understood. Here, we investigate the role of parvalbumin (PV) neurons in cortical complex scene analysis. Many complex scenes contain sensory stimuli, e.g., natural sounds, images, odors or vibrations, which are highly dynamic in time, competing with stimuli at other locations in space. PV neurons are thought to play a fundamental role in sculpting cortical temporal dynamics; yet their specific role in encoding complex scenes via timing-based codes, and the robustness of such temporal representations to spatial competition, have not been investigated. Here, we address these questions in auditory cortex using a cocktail party-like paradigm; integrating electrophysiology, optogenetic manipulations, and a family of novel spike-distance metrics, to dissect the contributions of PV neurons towards rate and timing-based coding. We find that PV neurons improve cortical discrimination of dynamic naturalistic sounds in a cocktail party-like setting by enhancing rapid temporal modulations in rate and spike timing reproducibility. Moreover, this temporal representation is maintained in the face of competing stimuli at other spatial locations, providing a robust code for complex scene analysis. These findings provide novel insights into the specific contributions of PV neurons in cortical coding of complex scenes.</p>', 'posted': '2021-09-12'}, {'doi': '10.1101/2021.09.03.458844', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Improve consensus partitioning via a hierarchical procedure', 'authors': \"[{'name': 'Zuguang Gu'}, {'name': 'Daniel Hübschmann'}]\", 'abstract': '<p>Consensus partitioning is an unsupervised method widely used in high throughput data analysis for revealing subgroups and assigns stability for the classification. However, standard consensus partitioning procedures are weak to identify large numbers of stable subgroups. There are two main issues. 1. Subgroups with small differences are difficult to separate if they are simultaneously detected with subgroups with large differences. And 2. stability of classification generally decreases as the number of subgroups increases. In this work, we proposed a new strategy to solve these two issues by applying consensus partitionings in a hierarchical procedure. We demonstrated hierarchical consensus partitioning can be efficient to reveal more subgroups. We also tested the performance of hierarchical consensus partitioning on revealing a great number of subgroups with a DNA methylation dataset. The hierarchical consensus partitioning is implemented in the R package <italic>cola</italic> with comprehensive functionality for analysis and visualizations. It can also automate the analysis only with a minimum of two lines of code, which generates a detailed HTML report containing the complete analysis.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.09.459524', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Distinct proteostasis states drive pharmacologic chaperone susceptibility for Cystic Fibrosis Transmembrane Conductance Regulator misfolding mutants', 'authors': \"[{'name': 'Eli Fritz McDonald'}, {'name': 'Carleen Mae P. Sabusap'}, {'name': 'Minsoo Kim'}, {'name': 'Lars Plate'}]\", 'abstract': '<p>Pharmacological chaperones represent a class of therapeutic compounds for treating protein misfolding diseases. One of the most prominent examples is the FDA-approved pharmacological chaperone lumacaftor (VX-809), which has transformed cystic fibrosis (CF) therapy. CF is a fatal disease caused by mutations in the cystic fibrosis transmembrane conductance regulator (CFTR). VX-809 corrects folding of F508del CFTR, the most common patient mutation, yet F508del exhibits only mild VX-809 response. In contrast, rarer mutations P67L and L206W are <italic>hyper-responsive</italic> to VX-809, while G85E is <italic>non-responsive</italic>. Despite the clinical success of VX-809, the mechanistic origin for the distinct susceptibility of mutants remains unclear. Here, we use interactomics to characterize the impact of VX-809 on proteostasis interactions of P67L and L206W and compare these to F508del and G85E. We determine <italic>hyper-responsive</italic> mutations P67L and L206W exhibit decreased interactions with proteasomal, and autophagy degradation machinery compared to F508del and G85E. We then show inhibiting the proteasome attenuates P67L and L206W VX-809 response, and inhibiting the lysosome attenuates F508del VX-809 response. Our data suggests a previously unidentified but required role for protein degradation in VX-809 correction. Furthermore, we present an approach for identifying proteostasis characteristics of mutant-specific therapeutic response to pharmacological chaperones.</p>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.06.459125', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Synthetic deconstruction of hunchback regulation by Bicoid', 'authors': \"[{'name': 'Goncalo Fernandes'}, {'name': 'Huy Tran'}, {'name': 'Maxime Andrieu'}, {'name': 'Youssoupha Diaw'}, {'name': 'Carmina Perez-Romero'}, {'name': 'Cécile Fradin'}, {'name': 'Mathieu Coppey'}, {'name': 'Aleksandra M. Walczak'}, {'name': 'Nathalie Dostatni'}]\", 'abstract': '<p>During development, cell identity is established reproducibly among individuals through the expression of specific genes at the correct time and correct location in space. How genes extract and combine both positional and temporal information from different transcription factor (TF) profiles along polarity axes remain largely unexplored. Here, we showcase the classic <italic>hunchback</italic> gene in fruit fly embryos, with focus on 3 of its main TFs: Bicoid, Zelda and Hunchback proteins. We constructed a series of synthetic MS2 reporters, where the numbers and combination of binding sites for each TF are varied. Using live imaging of transcription dynamics by these synthetic reporters and modeling tools, we show that <italic>i)</italic> a Bicoid-only synthetic reporter needs 3 more Bicoid binding sites than found in the <italic>hunchback</italic> promoter to recapitulates almost all spatial features of early <italic>hb</italic> expression but takes more time to reach steady state; <italic>ii)</italic> Hunchback and Zelda binding sites combined with Bicoid sites both reduce the time to reach steady state and increase expression at a different step in the activation process: Zld sites lower the Bicoid threshold required for activation while Hb sites increase the polymerase firing rate and reduce bursting; <italic>iii)</italic> the shift of the Bicoid-only reporter induced by a reduction by half of Bicoid concentrations indicates that the decay length of the Bicoid activity gradient is lower than the decay length of the Bicoid protein gradient. Altogether, this work indicates that Bicoid is the main source of positional information for <italic>hunchback</italic> expression and places back the Bicoid system within the physical limits of an equilibrium model.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.06.459062', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'SETDB1 Fuels the Lung Cancer Phenotype by Modulating Epigenome, 3D Genome Organization and Chromatin Mechanical Properties', 'authors': \"[{'name': 'Vlada V. Zakharova'}, {'name': 'Mikhail D. Magnitov'}, {'name': 'Laurence Del-Maestro'}, {'name': 'Sergey V. Ulianov'}, {'name': 'Alexandros Glentis'}, {'name': 'Burhan Ulyanik'}, {'name': 'Alice Williart'}, {'name': 'Anna Karpukhina'}, {'name': 'Oleg Demidov'}, {'name': 'Veronique Joliot'}, {'name': 'Yegor S. Vassetzky'}, {'name': 'René-Marc Mège'}, {'name': 'Matthieu Piel'}, {'name': 'Sergey V. Razin'}, {'name': 'Slimane Ait-Si-Ali'}]\", 'abstract': '<p>Imbalance in the finely orchestrated system of chromatin-modifying enzymes is a hallmark of many pathologies such as cancers, since causing the affection of the epigenome and transcriptional reprogramming. Here, we demonstrate that a loss-of-function mutation (LOF) of the major histone lysine methyltransferase SETDB1 possessing oncogenic activity in lung cancer cells leads to broad changes in the overall architecture and mechanical properties of the nucleus through genome-wide redistribution of heterochromatin, which perturbs chromatin spatial compartmentalization. Together with the enforced activation of the epithelial expression program, cytoskeleton remodeling, reduced proliferation rate and restricted cellular migration, this leads to the reversed oncogenic potential of lung adenocarcinoma cells. These results emphasize an essential role of chromatin architecture in the determination of oncogenic programs and illustrate a relationship between gene expression, epigenome, 3D genome and nuclear mechanics.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.09.459621', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Reassortment Network of Influenza A Virus', 'authors': \"[{'name': 'Xingfei Gong'}, {'name': 'Mingda Hu'}, {'name': 'Wei Chen'}, {'name': 'Haoyi Yang'}, {'name': 'Boqian Wang'}, {'name': 'Junjie Yue'}, {'name': 'Yuan Jin'}, {'name': 'Long Liang'}, {'name': 'Hongguang Ren'}]\", 'abstract': '<p>Influenza A virus (IAV) genomes are composed of eight single-stranded RNA segments. Genetic exchange through reassortment of the segmented genomes often endows IAVs with new genetic characteristics, which may affect transmissibility and pathogenicity. However, a comprehensive understood of the reassortment history of IAVs remains poorly studied. To this end, we assembled 40296 whole-genome sequences of IAVs for analysis. Using a new clustering method based on Mean Pairwise Distances in the phylogenetic trees, we classified each segment of IAVs into clades. Correspondingly, reassortment events among IAVs were detected by checking the segment clade compositions of related genomes under specific environment factors and time period. We systematically identified 1927 possible reassortment events of IAVs and constructed their reassortment network. Interestingly, minimum spanning tree of the reassortment network reproved that swine act as an intermediate host in the reassortment history of IAVs between avian and human. Moreover, reassortment patterns among related subtypes constructed in this study are consistent with previous studies. Taken together, our genome-wide reassorment analysis of all the IAVs offers an overview of the leaping evolution of the virus and a comprehensive network representing the relationships of IAVs.</p>', 'posted': '2021-09-12'}, {'doi': '10.1101/2021.09.05.459032', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'The potyviral protein 6K1 reduces plant protease activity during Turnip mosaic virus infection', 'authors': \"[{'name': 'Sayanta Bera'}, {'name': 'Gabriella D. Arena'}, {'name': 'Swayamjit Ray'}, {'name': 'Sydney Flannigan'}, {'name': 'Clare L Casteel'}]\", 'abstract': '<p><list list-type=\"bullet\"><list-item><p>Potyviral genomes encode just 11 major proteins and multifunctionality is associated to most of these proteins at different stages of virus life cycle. The potyviral protein 6K1 is required for potyvirus replication at the early stages of viral infection and may mediate cell-to-cell movement at later stages.</p></list-item><list-item><p>Our study demonstrates that the 6K1 protein from <italic>Turnip mosaic virus</italic> (TuMV) reduces the abundance of transcripts related to jasmonic acid biosynthesis and transcripts that encode cysteine protease inhibitors when expressed in <italic>trans</italic> in <italic>Nicotiana benthamiana</italic> relative to controls. Furthermore, 6K1 stability increases when lipoxygenase and cysteine protease activity is inhibited chemically, linking a mechanism to the rapid turnover of 6K1 when expressed in <italic>trans</italic>.</p></list-item><list-item><p>Using transient expression, we show 6K1 is degraded rapidly at early time points in the infection process, whereas at later stages of infection protease activity is reduced and 6K1 becomes more stable, resulting in higher TuMV accumulation in systemic leaves. There was no impact of 6K1 transient expression on TuMV accumulation in local leaves.</p></list-item><list-item><p>Together, these results suggest a novel function for the TuMV 6K1 protein which has not been reported previously and enhances our understanding of the complex interactions occurring between plants and potyviruses.</p></list-item></list></p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.03.458926', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Supervised Phenotype Discovery fromMultimodal Brain Imaging', 'authors': \"[{'name': 'Weikang Gong'}, {'name': 'Song Bai'}, {'name': 'Ying-Qiu Zheng'}, {'name': 'Stephen M. Smith'}, {'name': 'Christian F. Beckmann'}]\", 'abstract': '<p>Data-driven discovery of image-derived phenotypes (IDPs) from large-scale multimodal brain imaging data has enormous potential for neuroscientific and clinical research by linking IDPs to subjects’ demographic, behavioural, clinical and cognitive measures (i.e., non-imaging derived phenotypes or nIDPs). However, current approaches are primarily based on unsupervised approaches, without use of information in nIDPs. In this paper, we proposed Supervised BigFLICA (SuperBigFLICA), a semi-supervised, multimodal, and multi-task fusion approach for IDP discovery, which simultaneously integrates information from multiple imaging modalities as well as multiple nIDPs. SuperBigFLICA is computationally efficient and largely bypasses the need for parameter tuning. Using the UK Biobank brain imaging dataset with around 40,000 subjects and 47 modalities, along with more than 17,000 nIDPs, we showed that SuperBigFLICA enhances the prediction power of nIDPs, bench-marked against IDPs derived by conventional expert-knowledge and unsupervised-learning approaches (with average nIDP prediction accuracy improvements of up to 46%). It also enables learning of generic imaging features that can predict new nIDPs. Further empirical analysis of the SuperBigFLICA algorithm demonstrates its robustness in different prediction tasks and the ability to derive biologically meaningful IDPs in predicting health outcomes and cognitive nIDPs, such as fluid intelligence and hypertension scores.</p>', 'posted': '2021-09-06'}, {'doi': '10.1101/2021.09.11.459880', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'The histone deacetylase activity of HDAC1/2 is required to safeguard zygotic genome activation in mice and cattle', 'authors': \"[{'name': 'Yanna Dang'}, {'name': 'Shuang Li'}, {'name': 'Panpan Zhao'}, {'name': 'Lieying Xiao'}, {'name': 'Lefeng Wang'}, {'name': 'Yan Shi'}, {'name': 'Lei Luo'}, {'name': 'Shaohua Wang'}, {'name': 'Huanan Wang'}, {'name': 'Kun Zhang'}]\", 'abstract': '<p>The genome is transcriptionally inert at fertilization and must be activated through a remarkable developmental process called zygotic genome activation (ZGA). The gene expression pattern formed over the course of ZGA is required for establishing totipotency in early embryos and subsequent development. Substantial epigenetic reprogramming contributes significantly to the pronounced change in gene expression during ZGA, however the mechanism has yet to be resolved. Here, we find histone deacetylase 1 and 2 (HDAC1/2) are critical histone modifiers that regulate ZGA through the histone deacetylase activity. Notably, we show that H3K27ac level declines dramatically during ZGA with a dynamic change in its genome-wide distribution. In mouse embryos, ectopic expression of HDAC1/2 dominant negative mutant leads to a failure of H3K27ac removal and a developmental arrest at 2-cell stage. RNA-seq results reveal a remarkable transcriptomic change with 6565 differentially expressed genes identified. Further analysis shows 64% of down-regulated genes are ZGA genes and 49% of up-regulated genes are developmental genes. Low input ChIP-seq analysis exhibits an increase and decrease of H3K27ac enrichment at the promoter region of up- and down-regulated genes, respectively. Moreover, HDAC1 mutants prohibited removal of broad H3K4me3 domain via impeding the expression of Kdm5s during ZGA. Importantly, the developmental block can be greatly overcome through injection of Kdm5b mRNA and expression of the majority of dysregulated genes partially corrected. Similar functional significance of HDAC1/2 in ZGA is conserved in bovine embryos. Together, we propose that HDAC1/2 is indispensable for mouse and bovine ZGA via creating correct transcriptional repressive and active states.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.10.459722', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'CAMOIP: A Web Server for Comprehensive Analysis on Multi-Omics of Immunotherapy in Pan-cancer', 'authors': \"[{'name': 'Anqi LIN'}, {'name': 'Ting WEI'}, {'name': 'Junyi LIANG'}, {'name': 'Chang QI'}, {'name': 'Mengyao LI'}, {'name': 'Peng LUO'}, {'name': 'Jian Zhang'}]\", 'abstract': '<p>Immune checkpoint inhibitors (ICIs) have completely changed the therapeutic approach for tumor patients. Immunotherapy has also produced much needed data about mutation, expression, and prognosis, providing an unprecedented opportunity for discovering candidate drug targets and screening for immunotherapy-relevant biomarkers. Although existing web tools enable biologists to analyze the expression, mutation, and prognosis data on tumors, they are currently not able to carry out data mining and mechanism analyses related to immune checkpoint therapy. Thus, we developed our own web-based tool called Comprehensive Analysis on Multi-Omics of Immunotherapy in Pan-cancer(CAMOIP), in which we can screen prognostic markers and analyze the mechanisms involved with markers and immunotherapy (more than 4000 patients). The analyses include survival analysis, expression analysis, drug sensitivity analysis, mutational landscape, immune checkpoint analysis, immune related signature analysis, immune cell analysis, immune gene analysis, immunogenicity analysis and gene sets enrichment analysis(GSEA). This comprehensive analysis of biomarkers for immunotherapy can be carried out by a click of CAMOIP, and the software should greatly encourage the further development of immunotherapy. CAMOIP fills the gap between the big data of cancer genomics based on immunotherapy and providing comprehensive information to users, helping to release the value of current ICI-treated data resources. CAMOIP can be found in https://www.camoip.net.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.08.459532', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The Temporal Pattern of Spiking Activity of a Thalamic Neuron are Related to the Amplitude of the Cortical Local Field Potential', 'authors': \"[{'name': 'Hiroshi Tamura'}]\", 'abstract': '<p>Neuron activity in the sensory cortices mainly depends on feedforward thalamic inputs. High-frequency activity of a thalamic input can be temporally integrated by a neuron in the sensory cortex and is likely to induce larger depolarization. However, feedforward inhibition (FFI) and depression of excitatory synaptic transmission in thalamocortical pathways attenuate depolarization induced by the latter part of high-frequency spiking activity and the temporal summation may not be effective. The spiking activity of a thalamic neuron in a specific temporal pattern may circumvent FFI and depression of excitatory synapses. The present study determined the relationship between the temporal pattern of spiking activity of a single thalamic neuron and the degree of cortical activation as well as that between the firing rate of spiking activity of a single thalamic neuron and the degree of cortical activation. Spiking activity of a thalamic neuron was recorded extracellularly from the lateral geniculate nucleus (LGN) in male Long-Evans rats. Degree of cortical activation was assessed by simultaneous recording of local field potential (LFP) from the visual cortex. A specific temporal pattern appearing in three consecutive spikes of an LGN neuron induced larger cortical LFP modulation than high-frequency spiking activity during a short period. These findings indicate that spiking activity of thalamic inputs is integrated by a synaptic mechanism sensitive to an input temporal pattern.</p>', 'posted': '2021-09-10'}, {'doi': '10.1101/2021.09.08.459492', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Combined free energy calculation and machine learning methods for understanding ligand unbinding kinetics', 'authors': \"[{'name': 'Magd Badaoui'}, {'name': 'Pedro J Buigues'}, {'name': 'Dénes Berta'}, {'name': 'Gaurav M. Mandana'}, {'name': 'Hankang Gu'}, {'name': 'Callum J Dickson'}, {'name': 'Viktor Hornak'}, {'name': 'Mitsunori Kato'}, {'name': 'Carla Molteni'}, {'name': 'Simon Parsons'}, {'name': 'Edina Rosta'}]\", 'abstract': '<p>The determination of drug residence times, which define the time an inhibitor is in complex with its target, is a fundamental part of the drug discovery process. Synthesis and experimental pharmacokinetics measurements are, however, expensive, and time-consuming. In this work, we aimed to obtain drug residence times computationally. Furthermore, we propose a novel algorithm to identify molecular design objectives based on ligand unbinding kinetics. We designed an enhanced sampling technique to accurately predict the free energy profiles of the ligand unbinding process, focusing on the free energy barrier for unbinding. Our method first identifies unbinding paths determining a corresponding set of internal coordinates (IC) that form contacts between the protein and the ligand, then iteratively updates these interactions during a series of biased molecular-dynamics (MD) simulations to reveal the ICs important for the whole of the unbinding process. Subsequently, we performed finite temperature string simulations to obtain the free energy barrier for unbinding using the set of ICs as a complex reaction coordinate. Importantly, we also aimed to enable further design of drugs focusing on improved residence times. To this end, we developed a supervised machine learning (ML) approach that uses as input unbiased “downhill” trajectories from the transition state (TS) ensemble of the string unbinding path. We demonstrate that our ML method can identify key ligand-protein interactions driving the system through the TS. Some of the most important drugs for cancer treatment are kinase inhibitors. One of these kinase targets is Cyclin Dependent Kinase 2 (CDK2), an appealing target for anticancer drug development. Here, we tested our method using three different CDK2 inhibitors for potential further development of these compounds. We compared the free energy barriers obtained from our calculations with those observed in available experimental data. We highlighted important interactions at the distal ends of the ligands that can be targeted for improved residence times. Our method provides a new tool to determine unbinding rates, and to identify key structural features of the inhibitors that can be used as starting points for novel design strategies in drug discovery.</p>', 'posted': '2021-09-09'}, {'doi': '10.1101/2021.09.10.459856', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Deficits in decision-making induced by parietal cortex inactivation are compensated at two time scales', 'authors': \"[{'name': 'Danique Jeurissen'}, {'name': 'S Shushruth'}, {'name': 'Yasmine El-Shamayleh'}, {'name': 'Gregory D Horwitz'}, {'name': 'Michael N Shadlen'}]\", 'abstract': '<p>Perceptual decisions arise through the transformation of samples of evidence into a commitment to a proposition or plan of action. Such transformation is thought to involve cortical circuits capable of computation over time scales associated with working memory, attention, and planning. Neurons in the lateral intraparietal area (LIP) are thought to play a role in all of these functions, and much of what is known about the neurobiology of decision making has been influenced by studies of LIP and its network of cortical and subcortical connections. However a causal role of neurons in LIP remains controversial. We used pharmacological and chemogenetic methods to inactivate LIP in one hemisphere of four rhesus monkeys. Inactivation produced clear biases in decisions, but the effects dissipated despite the persistence  of neural inactivation, implying compensation by other unaffected areas. Compensation occurs on a rapid times scale, within an experimental session, and more gradually, across sessions. The findings resolve disparate studies and inform interpretation of focal perturbations of brain function.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.09.459569', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Spontaneous alpha-band oscillations bias subjective contrast perception', 'authors': \"[{'name': 'Elio Balestrieri'}, {'name': 'Niko A. Busch'}]\", 'abstract': \"<p>Perceptual decisions depend both on the features of the incoming stimulus and on the ongoing brain activity at the moment the stimulus is received. Specifically, trial-to-trial fluctuations in cortical excitability have been linked to fluctuations in the amplitude of pre-stimulus alpha oscillations (~8-13 Hz), which are in turn are associated with fluctuations in subjects' tendency to report the detection of a stimulus. It is currently unknown whether alpha oscillations bias post-perceptual decision making, or even bias subjective perception itself. To answer this question, we used a contrast discrimination task in which subjects reported which of two gratings -- one in each hemifield -- was perceived as having a stronger contrast. Our EEG analysis showed that subjective contrast was reduced for the stimulus in the hemifield represented in the hemisphere with relatively stronger pre-stimulus alpha amplitude, reflecting reduced cortical excitability. Furthermore, the strength of this spontaneous hemispheric lateralization was strongly correlated with the magnitude of individual subjects' biases, suggesting that the spontaneous patterns of alpha lateralization play a role in explaining the inter-subject variability in contrast perception. These results indicate that spontaneous fluctuations in cortical excitability, indicted by patterns of pre-stimulus alpha amplitude, affect perceptual decisions by altering the phenomenological perception of the visual world.</p>\", 'posted': '2021-09-10'}, {'doi': '10.1101/2021.09.10.459734', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Deep Learning identifies new morphological patterns of Homologous Recombination Deficiency in luminal breast cancers from whole slide images.', 'authors': \"[{'name': 'Tristan Lazard'}, {'name': 'Guillaume Bataillon'}, {'name': 'Peter Naylor'}, {'name': 'Tatiana Popova'}, {'name': 'Francois-clement Bidard'}, {'name': 'Dominique Stoppa-Lyonnet'}, {'name': 'Marc-Henri Stern'}, {'name': 'Etienne Decenciere'}, {'name': 'Thomas Walter'}, {'name': 'Anne Salomon'}]\", 'abstract': '<p>Homologous Recombination DNA-repair deficiency (HRD) is a well-recognized marker of platinum-salt and PARP inhibitor chemotherapies in ovarian and breast cancers (BC). Causing high genomic instability, HRD is currently determined by BRCA1/2 sequencing or by genomic signatures, but its morphological manifestation is not well understood. Deep Learning (DL) is a powerful machine learning technique that has been recently shown to be capable of predicting genomic signatures from stained tissue slides. However, DL is known to be sensitive to dataset biases and lacks interpretability. Here, we present and evaluate a strategy to control for biases in retrospective cohorts. We train a deep-learning model to predict the HRD in a controlled cohort with unprecedented accuracy (AUC: 0.86) and we develop a new visualization technique that allows for automatic extraction of new morphological features related to HRD. We analyze in detail the extracted morphological patterns that open new hypotheses on the phenotypic impact of HRD.</p>', 'posted': '2021-09-11'}, {'doi': '10.1101/2021.09.07.459280', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'TET1 controls Cxcl1 induction by DNA demethylation and promotes neutrophil recruitment during acute lung injury', 'authors': \"[{'name': 'Kathleen M. Yee'}, {'name': 'Richard W. Shuai'}, {'name': 'Bin Liu'}, {'name': 'Christian A. Huynh'}, {'name': 'Chao Niu'}, {'name': 'Hailey R. Lee'}, {'name': 'Min S. Lee'}, {'name': 'Jirui Wen'}, {'name': 'Jian Zou'}, {'name': 'Jiang Wu'}, {'name': 'Ke Shuai'}]\", 'abstract': '<p>Neutrophils are rapidly recruited from the peripheral blood to the inflammatory site to initiate inflammatory response against pathogenic infections. The process to recruit neutrophils must be properly regulated since the abnormal accumulation of neutrophils can cause organ damage and dysfunction. The acute respiratory distress syndrome (ARDS)/acute lung injury (ALI) is a common cause of respiratory failure that is characterized by the infiltration of neutrophils and epithelial integrity disruption. Indeed, recent studies suggest a pathogenic role of neutrophils in the clinic severity of the coronavirus disease 2019 (COVID-19) ARDS. The chemokine CXCL1, which is rapidly induced by inflammatory stimuli, plays a key role in neutrophil influx during lung inflammation. The molecular basis of <italic>Cxcl1</italic> induction is not fully understood. Here we report that TET1, a member of the ten eleven translocation (TET) methylcytosine dioxygenase protein family, displays a striking specificity in the regulation of gene expression in macrophages. RNA sequencing (RNA-seq) analysis showed that <italic>Tet1</italic> disruption significantly altered the expression of only 48 genes that include <italic>Cxcl1</italic> and several other genes known to be important for cell migration and trafficking in bone marrow derived macrophages (BMDMs) in response to LPS stimulation. TET1 regulates the induction of <italic>Cxcl1</italic> by facilitating the DNA demethylation of the <italic>Cxcl1</italic> promoter. In <italic>Tet1</italic><sup>−/−</sup> mice, the induction of <italic>Cxcl1</italic> was suppressed, resulting in defective neutrophil recruitment to the lung during LPS-induced acute lung injury. Our results identify a novel epigenetic mechanism that selectively controls <italic>Cxcl1</italic> induction and neutrophil recruitment during acute lung injury.</p><sec><title>Key Points</title><list list-type=\"bullet\"><list-item><p>TET1 has a striking specificity in macrophage gene regulation and controls <italic>Cxcl1</italic> induction by inflammatory stimuli via DNA demethylation</p></list-item><list-item><p>Neutrophil recruitment is defective in <italic>Tet1</italic> deficient mice during acute lung injury</p></list-item></list></sec>', 'posted': '2021-09-08'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.09.03.21263103</td>\n",
       "      <td>170</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>The prevalence of adaptive immunity to COVID-1...</td>\n",
       "      <td>[{'name': 'Tawanda Chivese'}, {'name': 'Joshua...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Objectives&lt;/title&gt;&lt;p&gt;This study ai...</td>\n",
       "      <td>2021-09-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.09.03.458870</td>\n",
       "      <td>106</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Behavioral state tunes mouse vision to etholog...</td>\n",
       "      <td>[{'name': 'Katrin Franke'}, {'name': 'Konstant...</td>\n",
       "      <td>&lt;p&gt;Across animal species, sensory processing d...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.09.04.21263115</td>\n",
       "      <td>101</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Durability analysis of the highly effective BN...</td>\n",
       "      <td>[{'name': 'Arjun Puranik'}, {'name': 'Patrick ...</td>\n",
       "      <td>&lt;p&gt;SARS-CoV-2 breakthrough infections have bee...</td>\n",
       "      <td>2021-09-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.09.10.455288</td>\n",
       "      <td>75</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Generation and diversification of recombinant ...</td>\n",
       "      <td>[{'name': 'Keith F DeLuca'}, {'name': 'Jeanne ...</td>\n",
       "      <td>&lt;p&gt;Antibodies are indispensable tools used for...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.09.07.458941</td>\n",
       "      <td>69</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Mapping electromagnetic networks to haemodynam...</td>\n",
       "      <td>[{'name': 'Golia Shafiei'}, {'name': 'Sylvain ...</td>\n",
       "      <td>&lt;p&gt;Whole-brain neural communication is typical...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.09.07.459342</td>\n",
       "      <td>59</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>ELeFHAnt: A supervised machine learning approa...</td>\n",
       "      <td>[{'name': 'Konrad Thorner'}, {'name': 'Aaron M...</td>\n",
       "      <td>&lt;p&gt;Annotation of single cells has become an im...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.09.08.459546</td>\n",
       "      <td>56</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Humans reconfigure target and distractor proce...</td>\n",
       "      <td>[{'name': 'Harrison Ritz'}, {'name': 'Amitai S...</td>\n",
       "      <td>&lt;p&gt;When faced with distraction, we can focus m...</td>\n",
       "      <td>2021-09-10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.09.08.459528</td>\n",
       "      <td>54</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Maternal SMCHD1 regulates Hox gene expression ...</td>\n",
       "      <td>[{'name': 'Natalia Benetti'}, {'name': 'Quenti...</td>\n",
       "      <td>&lt;p&gt;Parents transmit genetic and epigenetic inf...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.09.10.459833</td>\n",
       "      <td>49</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Population differentiation of polygenic score ...</td>\n",
       "      <td>[{'name': 'Sivan Yair'}, {'name': 'Graham Coop'}]</td>\n",
       "      <td>&lt;p&gt;Given the many loci uncovered by genome-wid...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.09.06.459138</td>\n",
       "      <td>45</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Mapping of a N-terminal α-helix domain require...</td>\n",
       "      <td>[{'name': 'Poonam Kakade'}, {'name': 'Hina Ojh...</td>\n",
       "      <td>&lt;p&gt;Human autosomal recessive mutations in the ...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.09.05.458952</td>\n",
       "      <td>42</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Community science designed ribosomes with bene...</td>\n",
       "      <td>[{'name': 'Antje Kruger'}, {'name': 'Andrew M ...</td>\n",
       "      <td>&lt;p&gt;Functional design of ribosomes with mutant ...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.09.06.459169</td>\n",
       "      <td>42</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>PHIST: fast and accurate prediction of prokary...</td>\n",
       "      <td>[{'name': 'Andrzej Zielezinski'}, {'name': 'Se...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Summary&lt;/title&gt;&lt;p&gt;PHIST (Phage-Hos...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.09.11.459888</td>\n",
       "      <td>34</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Activation of stably silenced genes by recruit...</td>\n",
       "      <td>[{'name': 'Wing Fuk Chan'}, {'name': 'Christin...</td>\n",
       "      <td>&lt;p&gt;Stably silenced genes that display a high l...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.09.10.459858</td>\n",
       "      <td>31</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Engineered bacteria detect tumor DNA in vivo</td>\n",
       "      <td>[{'name': 'Robert M Cooper'}, {'name': 'Joseph...</td>\n",
       "      <td>&lt;p&gt;In vitro nucleic acid analysis has become a...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.09.10.459728</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>MetaCoAG: Binning Metagenomic Contigs via Comp...</td>\n",
       "      <td>[{'name': 'Vijini Mallawaarachchi'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Metagenomics binning has allowed us to stud...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.09.04.458985</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Spectrum graph-based de-novo sequencing algori...</td>\n",
       "      <td>[{'name': 'Petra Gutenbrunner'}, {'name': 'Pel...</td>\n",
       "      <td>&lt;p&gt;We describe MaxNovo, a novel spectrum graph...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.09.09.459620</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Variant surface glycoprotein expression in hum...</td>\n",
       "      <td>[{'name': 'Jaime So'}, {'name': 'Sarah Sudlow'...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Trypanosoma brucei gambiense&lt;/itali...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.09.07.459228</td>\n",
       "      <td>22</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Differentially optimized cell-free buffer enab...</td>\n",
       "      <td>[{'name': 'Angelo Cardoso Batista'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;The use of linear DNA templates in cell-fre...</td>\n",
       "      <td>2021-09-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.09.06.459145</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A DCL3 dicing code within Pol IV-RDR2 transcri...</td>\n",
       "      <td>[{'name': 'Andrew Loffer'}, {'name': 'Jasleen ...</td>\n",
       "      <td>&lt;p&gt;In plants, selfish genetic elements includi...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.09.08.459370</td>\n",
       "      <td>20</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Scientific Communication and Education</td>\n",
       "      <td>False</td>\n",
       "      <td>Methylothon: a versatile course-based high sch...</td>\n",
       "      <td>[{'name': 'Peyton A. Jones'}, {'name': 'David ...</td>\n",
       "      <td>&lt;p&gt;Methylothon is an inquiry-based high school...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.09.07.459279</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>eQTLs are key players in the integration of ge...</td>\n",
       "      <td>[{'name': 'Abdou Rahmane Wade'}, {'name': 'Har...</td>\n",
       "      <td>&lt;p&gt;Multi-omics represent a promising link betw...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.09.10.459829</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Dopamine biases sensitivity to personal goals ...</td>\n",
       "      <td>[{'name': 'Jaime J Castrellon'}, {'name': 'Jac...</td>\n",
       "      <td>&lt;p&gt;People regularly give in to daily temptatio...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.09.09.459619</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Single-nucleus transcriptomic analyses reveal ...</td>\n",
       "      <td>[{'name': 'Jenny Lai'}, {'name': 'Junho Kim'},...</td>\n",
       "      <td>&lt;p&gt;While &lt;italic&gt;ATM&lt;/italic&gt; loss-of-function...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.09.07.459266</td>\n",
       "      <td>17</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The temporal dynamics of the sea urchin regulome</td>\n",
       "      <td>[{'name': 'Roberto Feuda'}]</td>\n",
       "      <td>&lt;p&gt;In this work, we used Nanostring N-counter ...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.09.07.459270</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Real age prediction from the transcriptome wit...</td>\n",
       "      <td>[{'name': 'Romain Bulteau'}, {'name': 'Mirko F...</td>\n",
       "      <td>&lt;p&gt;Genome-wide gene expression profiling is a ...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.09.08.458611</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>CaPTure: Calcium PeakToolbox for analysis of i...</td>\n",
       "      <td>[{'name': 'Madhavi Tippani'}, {'name': 'Elizab...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Calcium imagi...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.09.06.459211</td>\n",
       "      <td>13</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Assembly formation is stabilized by Parvalbumi...</td>\n",
       "      <td>[{'name': 'Fereshteh Lagzi'}, {'name': 'Martha...</td>\n",
       "      <td>&lt;p&gt;Learning entails preserving the features of...</td>\n",
       "      <td>2021-09-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.09.08.459542</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Distal regulation, silencers and a shared comb...</td>\n",
       "      <td>[{'name': 'Paola Cornejo-Páramo'}, {'name': 'K...</td>\n",
       "      <td>&lt;p&gt;The chromatin environment plays a central r...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.09.04.21263123</td>\n",
       "      <td>12</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Assessment of COVID-19 intervention strategies...</td>\n",
       "      <td>[{'name': 'Sebastian Duchene'}, {'name': 'Leo ...</td>\n",
       "      <td>&lt;p&gt;The Nordic countries, defined here as Norwa...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.09.07.459322</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Illuminating links between cis-regulators and ...</td>\n",
       "      <td>[{'name': 'Shuang Liu'}, {'name': 'Hyejung Won...</td>\n",
       "      <td>&lt;p&gt;Psychiatric disorders exact immense human a...</td>\n",
       "      <td>2021-09-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.09.08.459552</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>MOLECULAR VIDEOGAMING: SUPER-RESOLVED TRAJECTO...</td>\n",
       "      <td>[{'name': 'Tristan P. Wallis'}, {'name': 'Anmi...</td>\n",
       "      <td>&lt;p&gt;Single-molecule localization microscopy (SM...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.09.05.459049</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Vulnerability to cavitation is linked to home ...</td>\n",
       "      <td>[{'name': 'David Coleman'}, {'name': 'Andrew M...</td>\n",
       "      <td>&lt;p&gt;Vulnerability to cavitation in leaves is th...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.09.10.459763</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>clipplotr - a comparative visualisation and an...</td>\n",
       "      <td>[{'name': 'Anob M Chakrabarti'}, {'name': 'Cha...</td>\n",
       "      <td>&lt;p&gt;CLIP technologies are now widely used to st...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.09.09.459613</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Distinct, opposing functions for CFIm59 and CF...</td>\n",
       "      <td>[{'name': 'Hsin-Wei Tseng'}, {'name': 'Anthony...</td>\n",
       "      <td>&lt;p&gt;The precise maintenance of PTEN dosage is c...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.09.06.459120</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>tascCODA: Bayesian tree-aggregated analysis of...</td>\n",
       "      <td>[{'name': 'Johannes Ostner'}, {'name': 'Salomé...</td>\n",
       "      <td>&lt;p&gt;Accurate generative statistical modeling of...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.09.07.459309</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Mutation, selection, and quantitative genetic ...</td>\n",
       "      <td>[{'name': 'Lindsay M. Johnson'}, {'name': 'Say...</td>\n",
       "      <td>&lt;p&gt;Understanding the evolutionary and genetic ...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.09.07.458711</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Targeting metabolic adaptations in the breast ...</td>\n",
       "      <td>[{'name': 'Qianying Zuo'}, {'name': 'Ayca Nazl...</td>\n",
       "      <td>&lt;p&gt;Estrogen receptor-positive (ER&lt;sup&gt;+&lt;/sup&gt;)...</td>\n",
       "      <td>2021-09-07</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.09.09.459708</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>HypVW is an HOCl-Sensing Two Component System ...</td>\n",
       "      <td>[{'name': 'Sara El Hajj'}, {'name': 'Camille H...</td>\n",
       "      <td>&lt;p&gt;Two component systems (TCS) are signalling ...</td>\n",
       "      <td>2021-09-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.09.09.459641</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>True</td>\n",
       "      <td>Targeting Stem-loop 1 of the SARS-CoV-2 5’UTR ...</td>\n",
       "      <td>[{'name': 'Setu M. Vora'}, {'name': 'Pietro Fo...</td>\n",
       "      <td>&lt;p&gt;SARS-CoV-2 is a highly pathogenic virus tha...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.09.07.459250</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pharmacology and Toxicology</td>\n",
       "      <td>False</td>\n",
       "      <td>GPCR-G protein selectivity – a unified meta-an...</td>\n",
       "      <td>[{'name': 'Alexander S. Hauser'}, {'name': 'Ch...</td>\n",
       "      <td>&lt;p&gt;Two-thirds of human hormones and one-third ...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.09.08.452919</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>CardIAP: Calcium images analyzer web application</td>\n",
       "      <td>[{'name': 'Ana Julia Velez Rueda'}, {'name': '...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Motivation&lt;/title&gt;&lt;p&gt;Ionic calcium...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.09.09.459666</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Multiplex knockout of trichome-regulating MYBs...</td>\n",
       "      <td>[{'name': 'W. Patrick Bewg'}, {'name': 'Scott ...</td>\n",
       "      <td>&lt;p&gt;Hair-like trichomes cover the aerial organs...</td>\n",
       "      <td>2021-09-10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.09.03.458917</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Strategy-dependent effects of working-memory l...</td>\n",
       "      <td>[{'name': 'Kyra Schapiro'}, {'name': 'Krešimir...</td>\n",
       "      <td>&lt;p&gt;Deliberative decisions based on an accumula...</td>\n",
       "      <td>2021-09-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.09.10.459864</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Cysteine induces mitochondrial reductive stres...</td>\n",
       "      <td>[{'name': 'Evan K. Noch'}, {'name': 'Laura Pal...</td>\n",
       "      <td>&lt;p&gt;Glucose and amino acid metabolism are criti...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.09.07.459340</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Atria: An Ultra-fast and Accurate Trimmer for ...</td>\n",
       "      <td>[{'name': 'Jiacheng Chuan'}, {'name': 'Aiguo Z...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;As Next Gener...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.09.10.459766</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Combining CRISPR/Cas9 and brain imaging: from ...</td>\n",
       "      <td>[{'name': 'Sabina Marciano'}, {'name': 'Tudor ...</td>\n",
       "      <td>&lt;p&gt;Understanding how functional connections be...</td>\n",
       "      <td>2021-09-11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.09.07.459353</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Concurrent Multimodal Data Acquisition During ...</td>\n",
       "      <td>[{'name': 'Rosa Sola Molina'}, {'name': 'Gemma...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Previous brai...</td>\n",
       "      <td>2021-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.09.08.459470</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Classification of antimicrobial mechanism of a...</td>\n",
       "      <td>[{'name': 'Xudong Ouyang'}, {'name': 'Jelmer H...</td>\n",
       "      <td>&lt;p&gt;Antimicrobial resistance is a major threat ...</td>\n",
       "      <td>2021-09-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.09.09.459678</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>The Cortical Spectrum: a robust structural con...</td>\n",
       "      <td>[{'name': 'Yohan J. John'}, {'name': 'Basilis ...</td>\n",
       "      <td>&lt;p&gt;High-level characterizations of the primate...</td>\n",
       "      <td>2021-09-10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.09.11.459906</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>PV neurons improve cortical complex scene anal...</td>\n",
       "      <td>[{'name': 'Jian Carlo Nocon'}, {'name': 'Howar...</td>\n",
       "      <td>&lt;p&gt;Cortical representations underlying a wide ...</td>\n",
       "      <td>2021-09-12</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2021.09.03.21263103     170  medRxiv   \n",
       "1     10.1101/2021.09.03.458870     106  bioRxiv   \n",
       "2   10.1101/2021.09.04.21263115     101  medRxiv   \n",
       "3     10.1101/2021.09.10.455288      75  bioRxiv   \n",
       "4     10.1101/2021.09.07.458941      69  bioRxiv   \n",
       "5     10.1101/2021.09.07.459342      59  bioRxiv   \n",
       "6     10.1101/2021.09.08.459546      56  bioRxiv   \n",
       "7     10.1101/2021.09.08.459528      54  bioRxiv   \n",
       "8     10.1101/2021.09.10.459833      49  bioRxiv   \n",
       "9     10.1101/2021.09.06.459138      45  bioRxiv   \n",
       "10    10.1101/2021.09.05.458952      42  bioRxiv   \n",
       "11    10.1101/2021.09.06.459169      42  bioRxiv   \n",
       "12    10.1101/2021.09.11.459888      34  bioRxiv   \n",
       "13    10.1101/2021.09.10.459858      31  bioRxiv   \n",
       "14    10.1101/2021.09.10.459728      29  bioRxiv   \n",
       "15    10.1101/2021.09.04.458985      29  bioRxiv   \n",
       "16    10.1101/2021.09.09.459620      23  bioRxiv   \n",
       "17    10.1101/2021.09.07.459228      22  bioRxiv   \n",
       "18    10.1101/2021.09.06.459145      20  bioRxiv   \n",
       "19    10.1101/2021.09.08.459370      20  bioRxiv   \n",
       "20    10.1101/2021.09.07.459279      19  bioRxiv   \n",
       "21    10.1101/2021.09.10.459829      17  bioRxiv   \n",
       "22    10.1101/2021.09.09.459619      17  bioRxiv   \n",
       "23    10.1101/2021.09.07.459266      17  bioRxiv   \n",
       "24    10.1101/2021.09.07.459270      16  bioRxiv   \n",
       "25    10.1101/2021.09.08.458611      16  bioRxiv   \n",
       "26    10.1101/2021.09.06.459211      13  bioRxiv   \n",
       "27    10.1101/2021.09.08.459542      12  bioRxiv   \n",
       "28  10.1101/2021.09.04.21263123      12  medRxiv   \n",
       "29    10.1101/2021.09.07.459322      11  bioRxiv   \n",
       "30    10.1101/2021.09.08.459552      11  bioRxiv   \n",
       "31    10.1101/2021.09.05.459049      11  bioRxiv   \n",
       "32    10.1101/2021.09.10.459763      10  bioRxiv   \n",
       "33    10.1101/2021.09.09.459613      10  bioRxiv   \n",
       "34    10.1101/2021.09.06.459120      10  bioRxiv   \n",
       "35    10.1101/2021.09.07.459309       9  bioRxiv   \n",
       "36    10.1101/2021.09.07.458711       8  bioRxiv   \n",
       "37    10.1101/2021.09.09.459708       8  bioRxiv   \n",
       "38    10.1101/2021.09.09.459641       8  bioRxiv   \n",
       "39    10.1101/2021.09.07.459250       8  bioRxiv   \n",
       "40    10.1101/2021.09.08.452919       7  bioRxiv   \n",
       "41    10.1101/2021.09.09.459666       7  bioRxiv   \n",
       "42    10.1101/2021.09.03.458917       7  bioRxiv   \n",
       "43    10.1101/2021.09.10.459864       7  bioRxiv   \n",
       "44    10.1101/2021.09.07.459340       7  bioRxiv   \n",
       "45    10.1101/2021.09.10.459766       7  bioRxiv   \n",
       "46    10.1101/2021.09.07.459353       6  bioRxiv   \n",
       "47    10.1101/2021.09.08.459470       6  bioRxiv   \n",
       "48    10.1101/2021.09.09.459678       6  bioRxiv   \n",
       "49    10.1101/2021.09.11.459906       6  bioRxiv   \n",
       "\n",
       "                              subject-area  covid  \\\n",
       "0                             Epidemiology   True   \n",
       "1                             Neuroscience  False   \n",
       "2                             Epidemiology   True   \n",
       "3                             Cell Biology  False   \n",
       "4                             Neuroscience  False   \n",
       "5                           Bioinformatics  False   \n",
       "6                             Neuroscience  False   \n",
       "7                    Developmental Biology  False   \n",
       "8                     Evolutionary Biology  False   \n",
       "9                             Biochemistry  False   \n",
       "10                       Synthetic Biology  False   \n",
       "11                          Bioinformatics  False   \n",
       "12                       Synthetic Biology  False   \n",
       "13                       Synthetic Biology  False   \n",
       "14                          Bioinformatics  False   \n",
       "15                          Bioinformatics  False   \n",
       "16                            Microbiology  False   \n",
       "17                       Synthetic Biology  False   \n",
       "18                       Molecular Biology  False   \n",
       "19  Scientific Communication and Education  False   \n",
       "20                                Genetics  False   \n",
       "21                            Neuroscience  False   \n",
       "22                            Neuroscience  False   \n",
       "23                   Developmental Biology  False   \n",
       "24                          Bioinformatics  False   \n",
       "25                            Neuroscience  False   \n",
       "26                            Neuroscience  False   \n",
       "27                   Developmental Biology  False   \n",
       "28                            Epidemiology   True   \n",
       "29                                Genomics  False   \n",
       "30                              Biophysics  False   \n",
       "31                           Plant Biology  False   \n",
       "32                          Bioinformatics  False   \n",
       "33                            Biochemistry  False   \n",
       "34                          Bioinformatics  False   \n",
       "35                    Evolutionary Biology  False   \n",
       "36                          Cancer Biology  False   \n",
       "37                            Microbiology  False   \n",
       "38                       Molecular Biology   True   \n",
       "39             Pharmacology and Toxicology  False   \n",
       "40                          Bioinformatics  False   \n",
       "41                           Plant Biology  False   \n",
       "42                            Neuroscience  False   \n",
       "43                          Cancer Biology  False   \n",
       "44                          Bioinformatics  False   \n",
       "45                            Neuroscience  False   \n",
       "46                            Neuroscience  False   \n",
       "47                            Microbiology  False   \n",
       "48                            Neuroscience  False   \n",
       "49                            Neuroscience  False   \n",
       "\n",
       "                                                title  \\\n",
       "0   The prevalence of adaptive immunity to COVID-1...   \n",
       "1   Behavioral state tunes mouse vision to etholog...   \n",
       "2   Durability analysis of the highly effective BN...   \n",
       "3   Generation and diversification of recombinant ...   \n",
       "4   Mapping electromagnetic networks to haemodynam...   \n",
       "5   ELeFHAnt: A supervised machine learning approa...   \n",
       "6   Humans reconfigure target and distractor proce...   \n",
       "7   Maternal SMCHD1 regulates Hox gene expression ...   \n",
       "8   Population differentiation of polygenic score ...   \n",
       "9   Mapping of a N-terminal α-helix domain require...   \n",
       "10  Community science designed ribosomes with bene...   \n",
       "11  PHIST: fast and accurate prediction of prokary...   \n",
       "12  Activation of stably silenced genes by recruit...   \n",
       "13       Engineered bacteria detect tumor DNA in vivo   \n",
       "14  MetaCoAG: Binning Metagenomic Contigs via Comp...   \n",
       "15  Spectrum graph-based de-novo sequencing algori...   \n",
       "16  Variant surface glycoprotein expression in hum...   \n",
       "17  Differentially optimized cell-free buffer enab...   \n",
       "18  A DCL3 dicing code within Pol IV-RDR2 transcri...   \n",
       "19  Methylothon: a versatile course-based high sch...   \n",
       "20  eQTLs are key players in the integration of ge...   \n",
       "21  Dopamine biases sensitivity to personal goals ...   \n",
       "22  Single-nucleus transcriptomic analyses reveal ...   \n",
       "23   The temporal dynamics of the sea urchin regulome   \n",
       "24  Real age prediction from the transcriptome wit...   \n",
       "25  CaPTure: Calcium PeakToolbox for analysis of i...   \n",
       "26  Assembly formation is stabilized by Parvalbumi...   \n",
       "27  Distal regulation, silencers and a shared comb...   \n",
       "28  Assessment of COVID-19 intervention strategies...   \n",
       "29  Illuminating links between cis-regulators and ...   \n",
       "30  MOLECULAR VIDEOGAMING: SUPER-RESOLVED TRAJECTO...   \n",
       "31  Vulnerability to cavitation is linked to home ...   \n",
       "32  clipplotr - a comparative visualisation and an...   \n",
       "33  Distinct, opposing functions for CFIm59 and CF...   \n",
       "34  tascCODA: Bayesian tree-aggregated analysis of...   \n",
       "35  Mutation, selection, and quantitative genetic ...   \n",
       "36  Targeting metabolic adaptations in the breast ...   \n",
       "37  HypVW is an HOCl-Sensing Two Component System ...   \n",
       "38  Targeting Stem-loop 1 of the SARS-CoV-2 5’UTR ...   \n",
       "39  GPCR-G protein selectivity – a unified meta-an...   \n",
       "40   CardIAP: Calcium images analyzer web application   \n",
       "41  Multiplex knockout of trichome-regulating MYBs...   \n",
       "42  Strategy-dependent effects of working-memory l...   \n",
       "43  Cysteine induces mitochondrial reductive stres...   \n",
       "44  Atria: An Ultra-fast and Accurate Trimmer for ...   \n",
       "45  Combining CRISPR/Cas9 and brain imaging: from ...   \n",
       "46  Concurrent Multimodal Data Acquisition During ...   \n",
       "47  Classification of antimicrobial mechanism of a...   \n",
       "48  The Cortical Spectrum: a robust structural con...   \n",
       "49  PV neurons improve cortical complex scene anal...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Tawanda Chivese'}, {'name': 'Joshua...   \n",
       "1   [{'name': 'Katrin Franke'}, {'name': 'Konstant...   \n",
       "2   [{'name': 'Arjun Puranik'}, {'name': 'Patrick ...   \n",
       "3   [{'name': 'Keith F DeLuca'}, {'name': 'Jeanne ...   \n",
       "4   [{'name': 'Golia Shafiei'}, {'name': 'Sylvain ...   \n",
       "5   [{'name': 'Konrad Thorner'}, {'name': 'Aaron M...   \n",
       "6   [{'name': 'Harrison Ritz'}, {'name': 'Amitai S...   \n",
       "7   [{'name': 'Natalia Benetti'}, {'name': 'Quenti...   \n",
       "8   [{'name': 'Sivan Yair'}, {'name': 'Graham Coop'}]   \n",
       "9   [{'name': 'Poonam Kakade'}, {'name': 'Hina Ojh...   \n",
       "10  [{'name': 'Antje Kruger'}, {'name': 'Andrew M ...   \n",
       "11  [{'name': 'Andrzej Zielezinski'}, {'name': 'Se...   \n",
       "12  [{'name': 'Wing Fuk Chan'}, {'name': 'Christin...   \n",
       "13  [{'name': 'Robert M Cooper'}, {'name': 'Joseph...   \n",
       "14  [{'name': 'Vijini Mallawaarachchi'}, {'name': ...   \n",
       "15  [{'name': 'Petra Gutenbrunner'}, {'name': 'Pel...   \n",
       "16  [{'name': 'Jaime So'}, {'name': 'Sarah Sudlow'...   \n",
       "17  [{'name': 'Angelo Cardoso Batista'}, {'name': ...   \n",
       "18  [{'name': 'Andrew Loffer'}, {'name': 'Jasleen ...   \n",
       "19  [{'name': 'Peyton A. Jones'}, {'name': 'David ...   \n",
       "20  [{'name': 'Abdou Rahmane Wade'}, {'name': 'Har...   \n",
       "21  [{'name': 'Jaime J Castrellon'}, {'name': 'Jac...   \n",
       "22  [{'name': 'Jenny Lai'}, {'name': 'Junho Kim'},...   \n",
       "23                        [{'name': 'Roberto Feuda'}]   \n",
       "24  [{'name': 'Romain Bulteau'}, {'name': 'Mirko F...   \n",
       "25  [{'name': 'Madhavi Tippani'}, {'name': 'Elizab...   \n",
       "26  [{'name': 'Fereshteh Lagzi'}, {'name': 'Martha...   \n",
       "27  [{'name': 'Paola Cornejo-Páramo'}, {'name': 'K...   \n",
       "28  [{'name': 'Sebastian Duchene'}, {'name': 'Leo ...   \n",
       "29  [{'name': 'Shuang Liu'}, {'name': 'Hyejung Won...   \n",
       "30  [{'name': 'Tristan P. Wallis'}, {'name': 'Anmi...   \n",
       "31  [{'name': 'David Coleman'}, {'name': 'Andrew M...   \n",
       "32  [{'name': 'Anob M Chakrabarti'}, {'name': 'Cha...   \n",
       "33  [{'name': 'Hsin-Wei Tseng'}, {'name': 'Anthony...   \n",
       "34  [{'name': 'Johannes Ostner'}, {'name': 'Salomé...   \n",
       "35  [{'name': 'Lindsay M. Johnson'}, {'name': 'Say...   \n",
       "36  [{'name': 'Qianying Zuo'}, {'name': 'Ayca Nazl...   \n",
       "37  [{'name': 'Sara El Hajj'}, {'name': 'Camille H...   \n",
       "38  [{'name': 'Setu M. Vora'}, {'name': 'Pietro Fo...   \n",
       "39  [{'name': 'Alexander S. Hauser'}, {'name': 'Ch...   \n",
       "40  [{'name': 'Ana Julia Velez Rueda'}, {'name': '...   \n",
       "41  [{'name': 'W. Patrick Bewg'}, {'name': 'Scott ...   \n",
       "42  [{'name': 'Kyra Schapiro'}, {'name': 'Krešimir...   \n",
       "43  [{'name': 'Evan K. Noch'}, {'name': 'Laura Pal...   \n",
       "44  [{'name': 'Jiacheng Chuan'}, {'name': 'Aiguo Z...   \n",
       "45  [{'name': 'Sabina Marciano'}, {'name': 'Tudor ...   \n",
       "46  [{'name': 'Rosa Sola Molina'}, {'name': 'Gemma...   \n",
       "47  [{'name': 'Xudong Ouyang'}, {'name': 'Jelmer H...   \n",
       "48  [{'name': 'Yohan J. John'}, {'name': 'Basilis ...   \n",
       "49  [{'name': 'Jian Carlo Nocon'}, {'name': 'Howar...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <sec><title>Objectives</title><p>This study ai...  2021-09-07  \n",
       "1   <p>Across animal species, sensory processing d...  2021-09-06  \n",
       "2   <p>SARS-CoV-2 breakthrough infections have bee...  2021-09-07  \n",
       "3   <p>Antibodies are indispensable tools used for...  2021-09-11  \n",
       "4   <p>Whole-brain neural communication is typical...  2021-09-08  \n",
       "5   <p>Annotation of single cells has become an im...  2021-09-08  \n",
       "6   <p>When faced with distraction, we can focus m...  2021-09-10  \n",
       "7   <p>Parents transmit genetic and epigenetic inf...  2021-09-08  \n",
       "8   <p>Given the many loci uncovered by genome-wid...  2021-09-11  \n",
       "9   <p>Human autosomal recessive mutations in the ...  2021-09-06  \n",
       "10  <p>Functional design of ribosomes with mutant ...  2021-09-06  \n",
       "11  <sec><title>Summary</title><p>PHIST (Phage-Hos...  2021-09-06  \n",
       "12  <p>Stably silenced genes that display a high l...  2021-09-11  \n",
       "13  <p>In vitro nucleic acid analysis has become a...  2021-09-11  \n",
       "14  <p>Metagenomics binning has allowed us to stud...  2021-09-11  \n",
       "15  <p>We describe MaxNovo, a novel spectrum graph...  2021-09-06  \n",
       "16  <p><italic>Trypanosoma brucei gambiense</itali...  2021-09-09  \n",
       "17  <p>The use of linear DNA templates in cell-fre...  2021-09-07  \n",
       "18  <p>In plants, selfish genetic elements includi...  2021-09-06  \n",
       "19  <p>Methylothon is an inquiry-based high school...  2021-09-09  \n",
       "20  <p>Multi-omics represent a promising link betw...  2021-09-09  \n",
       "21  <p>People regularly give in to daily temptatio...  2021-09-11  \n",
       "22  <p>While <italic>ATM</italic> loss-of-function...  2021-09-09  \n",
       "23  <p>In this work, we used Nanostring N-counter ...  2021-09-08  \n",
       "24  <p>Genome-wide gene expression profiling is a ...  2021-09-08  \n",
       "25  <sec><title>Background</title><p>Calcium imagi...  2021-09-09  \n",
       "26  <p>Learning entails preserving the features of...  2021-09-07  \n",
       "27  <p>The chromatin environment plays a central r...  2021-09-09  \n",
       "28  <p>The Nordic countries, defined here as Norwa...  2021-09-08  \n",
       "29  <p>Psychiatric disorders exact immense human a...  2021-09-07  \n",
       "30  <p>Single-molecule localization microscopy (SM...  2021-09-09  \n",
       "31  <p>Vulnerability to cavitation in leaves is th...  2021-09-06  \n",
       "32  <p>CLIP technologies are now widely used to st...  2021-09-11  \n",
       "33  <p>The precise maintenance of PTEN dosage is c...  2021-09-09  \n",
       "34  <p>Accurate generative statistical modeling of...  2021-09-06  \n",
       "35  <p>Understanding the evolutionary and genetic ...  2021-09-08  \n",
       "36  <p>Estrogen receptor-positive (ER<sup>+</sup>)...  2021-09-07  \n",
       "37  <p>Two component systems (TCS) are signalling ...  2021-09-12  \n",
       "38  <p>SARS-CoV-2 is a highly pathogenic virus tha...  2021-09-09  \n",
       "39  <p>Two-thirds of human hormones and one-third ...  2021-09-08  \n",
       "40  <sec><title>Motivation</title><p>Ionic calcium...  2021-09-09  \n",
       "41  <p>Hair-like trichomes cover the aerial organs...  2021-09-10  \n",
       "42  <p>Deliberative decisions based on an accumula...  2021-09-06  \n",
       "43  <p>Glucose and amino acid metabolism are criti...  2021-09-11  \n",
       "44  <sec><title>Background</title><p>As Next Gener...  2021-09-09  \n",
       "45  <p>Understanding how functional connections be...  2021-09-11  \n",
       "46  <sec><title>Background</title><p>Previous brai...  2021-09-09  \n",
       "47  <p>Antimicrobial resistance is a major threat ...  2021-09-08  \n",
       "48  <p>High-level characterizations of the primate...  2021-09-10  \n",
       "49  <p>Cortical representations underlying a wide ...  2021-09-12  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "65 preprints (including 5 covering SARS-CoV-2, 62 from bioRxiv and 3 from medRxiv) published in the last 7 days before 2021-09-13 had been tweeted at least 3 times (maximum 170)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
