{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&obj-id.prefix=10.1101&source=twitter&rows=0&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(email = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 0,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "24961 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=1c5a467a-d803-45c5-900c-a978a88cecaa&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=5dccc699-f69f-4977-942c-02863a2200b1&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=dacbf722-d96a-4852-a15a-12b1a73a1794&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=9ac969b6-1957-4a5f-9273-5221d4c7671e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=d44892d2-4817-469a-b20b-ea9e6305c7a4&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=a4ed89af-117c-4657-a59d-2c581f0dd27f&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=46305146-154d-4bc8-83f5-73de89ecf176&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=4b7783d5-0c39-4fab-955c-fcf58b9eaefe&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=294cdfbd-4c17-4586-9939-50d652d13af9&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=67171cd9-a0b1-4992-b0bd-2570961ccc1b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=b83414ec-f34c-43b2-8d70-cbc05186b2ee&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=731ff848-5c29-4d99-8a46-bc4ce15d0893&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=607d3dbf-5b04-4508-b8d3-0562196e4603&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0013.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=969496fb-c9bd-4468-b6ee-d2f2a28f9000&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0014.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=12226983-ee40-4c7a-bbab-ecc88e0e912d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0015.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=b165e0e4-bcbd-44e3-9788-f42d3f425256&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0016.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=96ef3b4c-a3e6-4b4c-a621-b4de24978d9e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0017.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=e3738336-b7f9-4594-b1d4-1379f41d3cf6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0018.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=45be0f87-7089-4e54-9cc2-872be9541c2d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0019.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=03eb7317-53b3-4361-89bd-f238f2c8c134&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0020.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=67852494-fde2-4458-9ad9-c3c6ac8026ad&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0021.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=98ef01d9-b10b-4153-85ae-632e9c3b4cc5&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  400\n",
      "API query complete  400\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0022.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=f472591a-54eb-4155-a5bf-2c4b005d50d3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0023.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=Anonymous&cursor=34e37be8-6117-490c-99be-aa93aaffdc04&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-07-26&until-occurred-date=2021-08-02\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0024.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(email = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261130...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260767...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.29.446289...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.434169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21254060...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250617...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453379...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453478...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.01.429219...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261041...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453492...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.31.928630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.21254771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.28.359257...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453573...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453605...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453458...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454514...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.20101253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/658930...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252652...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249840...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453760...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.21261093...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446163...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.21258069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.355651...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453070...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.453189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.03.932046...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453460...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21251283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453747...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453816...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.19.304097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.440404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.451846...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.388819...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.447341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.28.065714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257472...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.20109231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.448967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21256661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.17.423313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.17.20061440...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.453027...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.15.21257017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/794305...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.06.238915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.453258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.395905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.443368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.21257545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.11.425914...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452948...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21250080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.451321...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453678...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21260947...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.20222315...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454071...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275188.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453637...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453666...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453674...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255866...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.18.423358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.30.424837...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.20249026...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.03.282186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.426881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453442...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20159251...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.19.423578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.432674...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/428318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.18.20037994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453574...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/139204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454097...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.19.162032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.03.003178...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.432992...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.436507...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.395079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/685743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.05.446432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.438352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.17.300996...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.452477...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.24.436752...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.432792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.07.372524...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454213...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.15.431212...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.389544...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.25.21252415...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255409...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258542...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.18.440371...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.16.21249632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453521...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.17.440282...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449250...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.14.297143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/507590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.287243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.426286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453727...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.24.962316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257738...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.448940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.21257766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260852...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/710624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.14.21258886...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.21259318...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.04.20243949...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453449...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.24.20248802...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453750...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453512...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.27.175067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.270371.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.265058.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453751...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.446826...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188268...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.16.21255616...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.439243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255714...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431620...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.21252597...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.06.21249256...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.21253051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452075...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.31.428959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.428510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21256128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.21258488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.439071...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.434740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.12.21258835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.426628...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21259929...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.07.21258447...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260972...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454541...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.21261146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.24.20078717...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453653...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.20.957225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454402...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454264...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.07.26.453829...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.10.986059...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.454001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.452189...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260986...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453293...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20177857...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.22.262527...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453650...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453736...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.384743...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258204...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.21255296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21255146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261046...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.432976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454286...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/550640...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.20245993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.07.20245274...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.433590...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.16.20131243...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.017624...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258468...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/596718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453552...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.21253167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.449363...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.17.20069716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442455...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.439231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.266049.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.431006...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.15.21258880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.23.20160895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21250921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.30.181370...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258564...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.453895...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.21257844...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442680...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.14.20062463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261038...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453915...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.434227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257032...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.18.21260710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.450661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.437754...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.14.095414...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.332916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453693...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.20209114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.452934...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/197608...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.01.363861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423475...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.405670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.29.21254554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.447066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.22.440869...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275303.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.14.20245266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454542...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/gr.266429.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445798...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.22.427482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453662...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.06.981076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.428184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.21257505...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.443175...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.21250535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.448993...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20207464...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251383...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/829952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.26.20202267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.347740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21256456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.04.21251134...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21258785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.19.20214494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.01.20144030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453610...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.08.447603...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454589...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.25.21252488...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.04.20167932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.28.428629...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.05.20241927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.447104...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453568...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.324145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.451701...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.21.162396...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453511...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21257602...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255782...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453533...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.443998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.435634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255873...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.22.21259273...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.320515...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.20193508...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.01.20135194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/782557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.416636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447686...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.450983...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.27.401232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.21258358...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454539...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257909...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454269...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20088260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.19.21252062...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20048165...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/706861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/704080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.24.429166...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256963...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.09.442808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.13.439592...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449942...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21258780...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.12.20151084...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.19.100214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.21252328...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.454622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.450238...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.442990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.11.20128520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.21.21254049...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.29.360842...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21253973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.21256242...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.205534...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.440988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.09.21260259...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137299...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.21254494...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.21256571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.03.437766...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.04.076067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21254092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252997...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.08.031807...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.06.028712...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255742...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/504803...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21252086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.05.20088757...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.29.174888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.05.21251182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453339...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.24.20235952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.453664...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.31.275644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2019.12.11.873265...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453308...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453858...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261039...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.19.389916...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.445786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453456...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20234369...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453424...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453821...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.21258647...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453553...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.16.423113...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.23.20018549...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.21258076...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.30.320721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.445979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.20186908...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.274563.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453735...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.04.20020404...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.03.20243352...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.26.453516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.385401...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.12.336677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.06.369355...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.431421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.17.20037432...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453697...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.14.21251715...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453696...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.16.207316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.17.21257092...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453688...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.25.453685...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.12.21249710...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.15.383323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454154...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.21.21260391...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453588...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.29.428808...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.23.350348...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.13.448261...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.11.20229708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.226043...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260946...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.381103...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453260...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.07.26.21261130', 'tweets': 2040, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Vaccine effectiveness when combining the ChAdOx1 vaccine as the first dose with an mRNA COVID-19 vaccine as the second dose', 'authors': \"[{'name': 'Mie Agermose Gram'}, {'name': 'Jens Nielsen'}, {'name': 'Astrid Blicher Schelde'}, {'name': 'Katrine Finderup Nielsen'}, {'name': 'Ida Rask Moustsen-Helms'}, {'name': 'Anne Katrine Bjrkholt Srensen'}, {'name': 'Palle Valentiner-Branth'}, {'name': 'Hanne-Dorthe Emborg'}]\", 'abstract': '<sec><title>Background</title><p>The recommendations in several countries to stop using the ChAdOx1 vaccine has led to vaccine programs combining different vaccine types, which necessitates new knowledge on vaccine effectiveness (VE). In this study, we aimed to estimate the VE when combining the ChAdOx1 vaccine as the first dose and an mRNA vaccine as the second dose.</p></sec><sec><title>Methods</title><p>This nationwide population-based cohort study estimated VE against SARS-CoV-2 infection, all-cause and COVID-19 related hospitalization and death after receiving the ChAdOx1 vaccine as the first dose followed by an mRNA vaccine as the second dose. VE estimates were obtained using a Cox regression with calendar time as underlying time and adjusted for sex, age, comorbidity, heritage and hospital admission. Information on all individuals was extracted and linked from high-quality national registries.</p></sec><sec><title>Results</title><p>A total of 5,542,079 individuals were included in the analyses (97.6% of the total Danish population). A total of 144,360 were vaccinated with the ChAdOx1 vaccine as the first dose and of these 136,551 individuals received an mRNA vaccine as the second dose. A total of 1,691,464 person-years and 83,034 cases of SARS-CoV-2 infection were included. The VE against SARS-CoV-2 infection when combining the ChAdOx1 and an mRNA vaccine was 88% (95% confidence interval (CI): 83; 92) 14 days after the second dose and onwards. There were no COVID-19 related hospitalizations and deaths among the individuals vaccinated with the combination of the ChAdOx1 and an mRNA vaccine during the study period.</p></sec><sec><title>Conclusion</title><p>In conclusion, this study found a reduction in the risk of SARS-CoV-2 infection when combining the ChAdOx1 and an mRNA vaccine, compared with unvaccinated individuals. This is similar to the VE of two doses of an mRNA vaccine. Longer follow-up time is needed to confirm vaccine induced protection against severe events, such as COVID-19 related hospitalization and death.</p></sec>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.23.21261030', 'tweets': 332, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Breakthrough Symptomatic COVID-19 Infections Leading to Long Covid: Report from Long Covid Facebook Group Poll', 'authors': \"[{'name': 'Daisy Massey'}, {'name': 'Diana Berrent'}, {'name': 'Harlan Krumholz'}]\", 'abstract': '<p>Vaccines have been shown to be extremely effective in preventing COVID-19 hospitalizations and deaths. However, a question remains whether vaccine breakthrough cases can still lead to Post-Acute Sequelae of SARS-CoV-2 (PASC), also known as Long Covid. To address this question, the Survivor Corps group, a grassroots COVID-19 organization focused on patient support and research, posted a poll to its 169,900 members that asked about breakthrough cases, Long Covid, and hospitalizations. 1,949 people who self-report being fully vaccinated have responded to date. While robust data are needed in a larger, unbiased sample to extrapolate rates to the population, we analyzed the results of this public poll to determine what people were reporting regarding Long Covid after breakthrough infection and to prompt discussion of how breakthrough cases are measured. The poll was posted in the Survivor Corps Facebook group (169,900 members). Of the 1,949 participants who responded to the poll, 44 reported a symptomatic breakthrough case and 24 of those reported that the case led to symptoms of Long Covid. 1 of these 24 cases was reported to have led to hospitalization in addition to Long Covid.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.27.453996', 'tweets': 113, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Disruption of the TCA cycle reveals an ATF4-dependent integration of redox and amino acid metabolism', 'authors': \"[{'name': 'Dylan G. Ryan'}, {'name': 'Ming Yang'}, {'name': 'Hiran A. Prag'}, {'name': 'Giovanny Rodriguez Blanco'}, {'name': 'Efterpi Nikitopoulou'}, {'name': 'Marc Segarra-Mondejar'}, {'name': 'Christopher A. Powell'}, {'name': 'Tim Young'}, {'name': 'Nils Burger'}, {'name': 'Jan Lj. Miljkovic'}, {'name': 'Michal Minczuk'}, {'name': 'Michael P. Murphy'}, {'name': 'Alexander von Kriegsheim'}, {'name': 'Christian Frezza'}]\", 'abstract': '<p>The Tricarboxylic Acid Cycle (TCA) cycle is arguably the most critical metabolic cycle in physiology and exists as an essential interface coordinating cellular metabolism, bioenergetics, and redox homeostasis. Despite decades of research, a comprehensive investigation into the consequences of TCA cycle dysfunction remains elusive. Here, we targeted two TCA cycle enzymes, fumarate hydratase (FH) and succinate dehydrogenase (SDH), and combined metabolomics, transcriptomics, and proteomics analyses to fully appraise the consequences of TCA cycle inhibition (TCAi) in kidney epithelial cells. Our comparative approach shows that TCAi elicits a convergent rewiring of redox and amino acid metabolism dependent on the activation of ATF4 and the integrated stress response (ISR). Furthermore, we also uncover a divergent metabolic response, whereby acute FHi, but not SDHi, can maintain asparagine levels via reductive carboxylation and maintenance of cytosolic aspartate synthesis. Our work highlights an important interplay between the TCA cycle, redox biology and amino acid homeostasis.</p><sec><title>Highlights</title><list list-type=\"order\"><list-item><p>TCA cycle inhibition promotes GSH synthesis and impairs <italic>de novo</italic> aspartate and proline synthesis</p></list-item><list-item><p>Disruption of mitochondrial thiol redox homeostasis phenocopies TCA cycle inhibition by promoting GSH synthesis and impairing proline and aspartate synthesis</p></list-item><list-item><p>Acute FHi, but not SDHi, can maintain asparagine levels via reductive carboxylation and maintenance of cytosolic aspartate synthesis</p></list-item><list-item><p>TCA cycle inhibition mimics an amino acid deprivation-type response and activates ATF4 via the integrated stress response to maintain redox and amino acid homeostasis</p></list-item></list></sec>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.26.453748', 'tweets': 75, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Identification of a new family of megaphages that are abundant in the marine environment', 'authors': \"[{'name': 'Slawomir Michniewski'}, {'name': 'Branko Rihtman'}, {'name': 'Ryan Cook'}, {'name': 'Michael A. Jones'}, {'name': 'William H Wilson'}, {'name': 'David J Scanlan'}, {'name': 'Andrew Millard'}]\", 'abstract': '<p>Megaphages  bacteriophages harbouring extremely large genomes  have recently been found to be ubiquitous, being described from a variety of microbiomes ranging from the animal gut to soil and freshwater systems. However, no complete marine megaphage has been identified to date. Here, using both short and long read sequencing, we assembled &gt;900 high-quality draft viral genomes from water in the English Channel. One of these genomes included a novel megaphage, Mar_Mega_1 at &gt;650 Kb, making it one of the largest phage genomes assembled to date. Utilising phylogenetic and network approaches, we found this phage represents a new family of bacteriophages. Genomic analysis showed Mar_Mega_1 shares relatively few homologues with its closest relatives, but, as with other mega-phages Mar_Mega_1 contained a variety of auxiliary metabolic genes responsible for carbon metabolism and nucleotide biosynthesis, including isocitrate dehydrogenase [NADP] and nicotinamide-nucleotide amidohydrolase [PncC] which have not previously been identified in megaphages. The results of this study indicate that phages containing extremely large genomes can be found in abundance in the marine environment and augment host metabolism by mechanisms not previously described.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.30.454437', 'tweets': 68, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'Highly efficient SARS-CoV-2 infection of human cardiomyocytes: spike protein-mediated cell fusion and its inhibition', 'authors': \"[{'name': 'Chanakha K. Navaratnarajah'}, {'name': 'David R. Pease'}, {'name': 'Peter Halfmann'}, {'name': 'Biruhalem Taye'}, {'name': 'Alison Barkhymer'}, {'name': 'Kyle G. Howell'}, {'name': 'Jon E. Charlesworth'}, {'name': 'Trace A. Christensen'}, {'name': 'Yoshihiro Kawaoka'}, {'name': 'Roberto Cattaneo'}, {'name': 'Jay W. Schneider'}, {'name': ' '}]\", 'abstract': '<p>Severe cardiovascular complications can occur in coronavirus disease of 2019 (COVID-19) patients. Cardiac damage is attributed mostly to a bystander effect: the aberrant host response to acute respiratory infection. However, direct infection of cardiac tissue by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) also occurs. We examined here the cardiac tropism of SARS-CoV-2 in human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CM) that beat spontaneously. These cardiomyocytes express the angiotensin I converting-enzyme 2 (ACE2) receptor and a subset of the proteases that mediate spike protein cleavage in the lungs, but not transmembrane protease serine 2 (TMPRSS2). Nevertheless, SARS-CoV-2 infection was productive: viral transcripts accounted for about 88% of total mRNA. In the cytoplasm of infected hiPSC-CM, smooth walled exocytic vesicles contained numerous 65-90 nm particles with typical ribonucleocapsid structures, and virus-like particles with knob-like spikes covered the cell surface. To better understand the mechanisms of SARS-CoV-2 spread in hiPSC-CM we engineered an expression vector coding for the spike protein with a monomeric emerald-green fluorescent protein fused to its cytoplasmic tail (S-mEm). Proteolytic processing of S-mEm and the parental spike were equivalent. Live cell imaging tracked spread of S-mEm signal from cell to cell and documented formation of syncytia. A cell-permeable, peptide-based molecule that blocks the catalytic site of furin abolished cell fusion. A spike mutant with the single amino acid change R682S that inactivates the furin cleavage site was fusion inactive. Thus, SARS-CoV-2 can replicate efficiently in hiPSC-CM and furin activation of its spike protein is required for fusion-based cytopathology. This hiPSC-CM platform provides an opportunity for target-based drug discovery in cardiac COVID-19.</p>', 'posted': '2021-07-30'}, {'doi': '10.1101/2021.07.23.453379', 'tweets': 65, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'AlphaPept, a modern and open framework for MS-based proteomics', 'authors': \"[{'name': 'Maximilian T. Strauss'}, {'name': 'Isabell Bludau'}, {'name': 'Wen-Feng Zeng'}, {'name': 'Eugenia Voytik'}, {'name': 'Constantin Ammar'}, {'name': 'Julia Schessner'}, {'name': 'Rajesh Ilango'}, {'name': 'Michelle Gill'}, {'name': 'Florian Meier'}, {'name': 'Sander Willems'}, {'name': 'Matthias Mann'}]\", 'abstract': '<p>In common with other omics technologies, mass spectrometry (MS)-based proteomics produces ever-increasing amounts of raw data, making their efficient analysis a principal challenge. There is a plethora of different computational tools that process the raw MS data and derive peptide and protein identification and quantification. During the last decade, there has been dramatic progress in computer science and software engineering, including collaboration tools that have transformed research and industry. To leverage these advances, we developed AlphaPept, a Python-based open-source framework for efficient processing of large high-resolution MS data sets. Using Numba for just-in-time machine code compilation on CPU and GPU, we achieve hundred-fold speed improvements while maintaining clear syntax and rapid development speed. AlphaPept uses the Python scientific stack of highly optimized packages, reducing the code base to domain-specific tasks while providing access to the latest advances in machine learning. We provide an easy on-ramp for community validation and contributions through the concept of literate programming, implemented in Jupyter Notebooks of the different modules. A framework for continuous integration, testing, and benchmarking enforces solid software engineering principles. Large datasets can rapidly be processed as shown by the analysis of hundreds of cellular proteomes in minutes per file, many-fold faster than the data acquisiton. The AlphaPept framework can be used to build automated processing pipelines using efficient HDF5 based file formats, web-serving functionality and compatibility with downstream analysis tools. Easy access for end-users is provided by one-click installation of the graphical user interface, for advanced users via a modular Python library, and for developers via a fully open GitHub repository.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261041', 'tweets': 58, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'The impact of large mobile air purifiers on aerosol concentration in classrooms and the reduction of airborne transmission of SARS-CoV-2', 'authors': \"[{'name': 'F. F. Duill'}, {'name': 'F. Schulz'}, {'name': 'A. Jain'}, {'name': 'L. Krieger'}, {'name': 'B. van Wachem'}, {'name': 'F. Beyrau'}]\", 'abstract': '<p>In the wake of the SARS-CoV-2 pandemic, an increased risk of infection by virus-containing aerosols indoors is assumed. Especially in schools, the duration of stay is long and the number of people in the rooms is large, increasing the risk of infection. This problem particularly affects schools without pre-installed ventilation systems that are equipped with filters and/or operate with fresh air. Here, the aerosol concentration is reduced by natural ventilation. In this context, we are investigating the effect of large mobile air purifiers (AP) with HEPA filters on particle concentration and their suitability for classroom use in a primary school in Germany. The three tested APs differ significantly in their air outlet characteristics. Measurements of the number of particles, the particle size distribution, and the CO<sub>2</sub> concentration were carried out in the classroom with students (April/May 2021) and with an aerosol generator without students. In this regard, the use of APs leads to a substantial reduction in aerosol particles. At the same time, the three APs are found to have differences in their particle decay rate, noise level, and flow velocity. In addition to the measurements, the effect of various influencing parameters on the potential inhaled particle dose was investigated using a calculation model. The parameters considered include the duration of stay, particle concentration in exhaled air, respiratory flow rate, virus lifetime, ventilation interval, ventilation efficiency, AP volumetric flow, as well as room size. Based on the resulting effect diagrams, significant recommendations can be derived for reducing the risk of infection from virus-laden aerosols. Finally, the measurements were compared to computational fluid dynamics (CFD) modeling, as such tools can aid the optimal placement and configuration of APs and can be used to study the effect of the spread of aerosols from a source in the classroom.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453492', 'tweets': 57, 'archive': 'bioRxiv', 'subject-area': 'Scientific Communication and Education', 'covid': False, 'title': 'Delineating Medical Education: Bibliometric Research Approach(es)', 'authors': \"[{'name': 'Lauren A. Maggio'}, {'name': 'Anton Ninkov'}, {'name': 'Jason R. Frank'}, {'name': 'Joseph A. Costello'}, {'name': 'Anthony R. Artino'}]\", 'abstract': '<sec><title>Background</title><p>The field of medical education remains poorly delineated such that there is no broad consensus of the articles and journals that comprise the field. This lack of consensus has implications for conducting bibliometric studies and other research designs (e.g., systematic reviews); it also challenges the field to compare citation scores in the field and across others and for an individual to identify themselves as a medical education researcher. Other fields have utilized bibliometric field delineation, which is the assigning of articles or journals to a certain field in an effort to define that field.</p></sec><sec><title>Process</title><p>In this <italic>Research Approach</italic>, three bibliometric field delineation approaches -- information retrieval, core journals, and journal co-citation -- are introduced. For each approach, the authors describe their attempt to apply it in the medical education context and identify related strengths and weaknesses. Based on co-citation, the authors propose the Medical Education Journal List 24 (MEJ-24), as a starting point for delineating medical education and invite the community to collaborate on improving and potentially expanding this list.</p></sec><sec><title>Pearls</title><p>As a research approach, field delineation is complicated, and there is no clear best way to delineate the field of medical education. However, recent advances in information and computer science provide potentially more fruitful approaches to deal with the complexity of the field. When considering these emerging approaches, researchers should consider collaborating with bibliometricians.</p><p>Bibliometric approaches rely on available metadata for articles and journals, which necessitates that researchers examine the metadata prior to analysis to understand its strengths and weaknesses, and to assess how this might affect their data interpretation. While using bibliometric approaches for field delineation is valuable, it is important to remember that these techniques are only as good as the research teams interpretation of the data, which suggests that an expanded research approach is needed to better delineate medical education, an approach that includes active discussion within the medical education community.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.21261040', 'tweets': 51, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Novel risk factors for Coronavirus disease-associated mucormycosis (CAM): a case control study during the outbreak in India', 'authors': \"[{'name': 'Umang Arora'}, {'name': 'Megha Priyadarshi'}, {'name': 'Varidh Katiyar'}, {'name': 'Manish Soneja'}, {'name': 'Prerna Garg'}, {'name': 'Ishan Gupta'}, {'name': 'Vishwesh Bharadiya'}, {'name': 'Parul Berry'}, {'name': 'Tamoghna Ghosh'}, {'name': 'Lajjaben Patel'}, {'name': 'Radhika Sarda'}, {'name': 'Shreya Garg'}, {'name': 'Shubham Agarwal'}, {'name': 'Veronica Arora'}, {'name': 'Aishwarya Ramprasad'}, {'name': 'Amit Kumar'}, {'name': 'Rohit Kumar Garg'}, {'name': 'Parul Kodan'}, {'name': 'Neeraj Nischal'}, {'name': 'Gagandeep Singh'}, {'name': 'Pankaj Jorwal'}, {'name': 'Arvind Kumar'}, {'name': 'Upendra Baitha'}, {'name': 'Ved Prakash Meena'}, {'name': 'Animesh Ray'}, {'name': 'Prayas Sethi'}, {'name': 'Immaculata Xess'}, {'name': 'Naval Vikram'}, {'name': 'Sanjeev Sinha'}, {'name': 'Ashutosh Biswas'}, {'name': 'Alok Thakar'}, {'name': 'Sushma Bhatnagar'}, {'name': 'Anjan Trikha'}, {'name': 'Naveet Wig'}]\", 'abstract': '<sec><title>Background</title><p>The epidemiology of the Coronavirus-disease associated mucormycosis (CAM) syndemic is poorly elucidated. We aimed to identify risk factors that may explain the burden of cases and help develop preventive strategies.</p></sec><sec><title>Methods</title><p>We performed a case-control study comparing cases diagnosed with CAM and those who had recovered from COVID-19 without developing mucormycosis (controls). Information on comorbidities, glycemic control, and practices related to COVID-19 prevention and treatment was recorded.</p></sec><sec><title>Results</title><p>352 patients (152 cases and 200 controls) diagnosed with COVID-19 during April-May 2021 were included. In the CAM group, symptoms of mucormycosis began a mean 18.9 (SD 9.1) days after onset of COVID-19, and predominantly rhino-sinus and orbital involvement was present. All, but one, CAM cases carried conventional risk factors of diabetes and steroid use. On multivariable regression, increased odds of CAM were associated with the presence of diabetes (adjusted OR 3.5, 95%CI 1.1-11), use of systemic steroids (aOR 7.7,95% CI 2.4-24.7), prolonged use of cloth and surgical masks (vs no mask, aOR 6.9, 95%CI 1.5-33.1), and repeated nasopharyngeal swab testing during the COVID-19 illness (aOR 1.6,95% CI 1.2-2.2). Zinc therapy, probably due to its utility in immune function, was found to be protective (aOR 0.05, 95%CI 0.01-0.19). Notably, the requirement of oxygen supplementation or hospitalization did not affect the risk of CAM.</p></sec><sec><title>Conclusion</title><p>Judicious use of steroids and stringent glycemic control are vital to preventing mucormycosis. Use of clean masks, preference for N95 masks if available, and minimizing swab testing after the diagnosis of COVID-19 may further reduce the incidence of CAM.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.453458', 'tweets': 46, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Epithelial-mesenchymal plasticity determines estrogen receptor positive (ER+) breast cancer dormancy and reacquisition of an epithelial state drives awakening', 'authors': \"[{'name': 'Patrick Aouad'}, {'name': 'Yueyun Zhang'}, {'name': 'Cline Stibolt'}, {'name': 'Sendurai A. Mani'}, {'name': 'George Sflomos'}, {'name': 'Cathrin Brisken'}]\", 'abstract': '<p>Estrogen receptor -positive (ER+) breast cancers (BCs) represent more than 70% of all breast cancers and pose a particular clinical challenge because they recur up to decades after initial diagnosis and treatment. The mechanisms governing tumor cell dormancy and latent disease remain elusive due to a lack of adequate models. Here, we compare tumor progression of ER+ and triple-negative (TN) BC subtypes with a clinically relevant mouse intraductal xenografting approach (MIND). Both ER+ and TN BC cells disseminate already during the <italic>in situ</italic>stage. However, TN disseminated tumor cells (DTCs) proliferate at the same rate as cells at the primary site and give rise to macro-metastases. ER+ DTCs have low proliferative indices, form only micro-metastases and lose epithelial characteristics. Expression of <italic>CDH1</italic> is decreased whereas the mesenchymal marker <italic>VIM</italic> and the transcription factors, <italic>ZEB1</italic>/<italic>ZEB2,</italic> which control epithelial-mesenchymal plasticity (EMP) are increased. EMP is not detected earlier during ER+ BC development and not required for invasion or metastasis. <italic>In vivo</italic>, forced transition to the epithelial state through ectopic E-cadherin expression overcomes dormancy with increased growth of lung metastases. We conclude that EMP is essential for the generation of a dormant cell state and the development of latent disease. Targeting exit from EMP is of therapeutic potential.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.30.454514', 'tweets': 44, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Length Biases in Single-Cell RNA Sequencing of pre-mRNA', 'authors': \"[{'name': 'Gennady Gorin'}, {'name': 'Lior Pachter'}]\", 'abstract': '<p>Single-molecule pre-mRNA and mRNA sequencing data can be modeled and analyzed using the Markov chain formalism to yield genome-wide insights into transcription. However, quantitative inference with such data requires careful assessment and understanding of noise sources. We find that long pre-mRNA transcripts are over-represented in sequencing data, and explore the mechanistic implications. A biological explanation for this phenomenon within our modeling framework requires unrealistic transcriptional parameters, leading us to posit a length-based model of capture bias. We provide solutions for this model, and use them to find concordant and mechanistically plausible parameter trends across data from multiple single-cell RNA-seq experiments in several species.</p>', 'posted': '2021-07-31'}, {'doi': '10.1101/2021.07.26.453760', 'tweets': 37, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A bacterial dynamin-like protein confers a novel phage resistance strategy on the population level in Bacillus subtilis', 'authors': \"[{'name': 'Lijun Guo'}, {'name': 'Laura Sattler'}, {'name': 'Peter Graumann'}, {'name': 'Marc Bramkamp'}]\", 'abstract': '<p><italic>Bacillus subtilis</italic> DynA is a member of the dynamin superfamily, involved in membrane remodeling processes. DynA was shown to catalyze full membrane fusion and it plays a role in membrane surveillance against antibiotics. We show here that DynA also provides a novel resistance mechanism against phage infection. Cells lacking DynA are efficiently lysed after phage infection and virus replication. DynA does not prevent phage infection and replication in individual cells, but significantly delays host cell lysis, thereby slowing down the release of phage progeny from the host cells. During the process, DynA forms large, almost immobile clusters on the cell membrane that seem to support membrane integrity. Single molecule tracking revealed a shift of freely diffusive molecules within the cytosol towards extended, confined motion at the cell membrane following phage induction. Thus, the bacterial dynamins are the first anti-phage system reported to delay host cell lysis and the last line of defense of a multilayered antiviral defense. DynA is therefore providing protective effects on the population, but not on single cell level.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.21261093', 'tweets': 34, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Protection afforded by the BNT162b2 and mRNA-1273 COVID-19 vaccines in fully vaccinated cohorts with and without prior infection', 'authors': \"[{'name': 'Laith J. Abu-Raddad'}, {'name': 'Hiam Chemaitelly'}, {'name': 'Houssein H. Ayoub'}, {'name': 'Hadi M. Yassine'}, {'name': 'Fatiha M. Benslimane'}, {'name': 'Hebah A. Al Khatib'}, {'name': 'Patrick Tang'}, {'name': 'Mohammad R. Hasan'}, {'name': 'Peter Coyle'}, {'name': 'Zaina Al Kanaani'}, {'name': 'Einas Al Kuwari'}, {'name': 'Andrew Jeremijenko'}, {'name': 'Anvar Hassan Kaleeckal'}, {'name': 'Ali Nizar Latif'}, {'name': 'Riyazuddin Mohammad Shaik'}, {'name': 'Hanan F. Abdul Rahim'}, {'name': 'Gheyath K. Nasrallah'}, {'name': 'Mohamed Ghaith Al Kuwari'}, {'name': 'Adeel A. Butt'}, {'name': 'Hamad Eid Al Romaihi'}, {'name': 'Mohamed H. Al-Thani'}, {'name': 'Abdullatif Al Khal'}, {'name': 'Roberto Bertollini'}]\", 'abstract': '<p>Effect of prior SARS-CoV-2 infection on vaccine protection remains poorly understood. Here, we investigated whether persons vaccinated after a prior infection have better protection against future infection than those vaccinated without prior infection. Effect of prior infection was assessed in Qatars population, where the Alpha (B.1.1.7) and Beta (B.1.351) variants dominate incidence, using two national retrospective, matched-cohort studies, one for the BNT162b2 (Pfizer-BioNTech) vaccine, and one for the mRNA-1273 (Moderna) vaccine. Incidence rates of infection among BNT162b2-vaccinated persons, with and without prior infection, were estimated, respectively, at 1.66 (95% CI: 1.26-2.18) and 11.02 (95% CI: 9.90-12.26) per 10,000 person-weeks. The incidence rate ratio was 0.15 (95% CI: 0.11-0.20). Analogous incidence rates among mRNA-1273-vaccinated persons were estimated at 1.55 (95% CI: 0.86-2.80) and 1.83 (95% CI: 1.07-3.16) per 10,000 person-weeks. The incidence rate ratio was 0.85 (95% CI: 0.34-2.05). Prior infection enhanced protection of those BNT162b2-vaccinated, but not those mRNA-1273-vaccinated. These findings may have implications for dosing, interval between doses, and potential need for booster vaccination.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.27.453460', 'tweets': 29, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'Extracellular mechanical forces drive endocardial cell volume decrease during cardiac valve morphogenesis', 'authors': \"[{'name': 'Julien Vermot'}, {'name': 'Helene Vignes'}, {'name': 'Christina Vagena-Pantoula'}, {'name': 'Mangal Prakash'}, {'name': 'Caren Norden'}, {'name': 'Florian Jug'}]\", 'abstract': '<p>Organ morphogenesis involves dynamic changes of tissue properties at the cellular scale. In addition, cells need to adapt to their mechanical environment through mechanosensitive pathways. How mechanical cues influence cell behaviors during morphogenesis, however, remains poorly understood. Here we studied the influence of mechanical forces during the formation of the atrioventricular canal (AVC) where cardiac valves develop. We show that in zebrafish the AVC forms within a zone of tissue convergence between the atrium and the ventricle which is associated with increased activation of the actomyosin meshwork and endocardial cell orientation changes. We demonstrate that tissue convergence occurs with a major reduction of endocardial cell volume triggered by mechanical forces and the mechanosensitive channels TRPP2/TRPV4. In addition, we show that the extracellular matrix component hyaluronic acid controls cell volume changes. Together, our data suggest that cell volume change is a key cellular feature activated by mechanical forces during cardiovascular morphogenesis. This work further unravels how mechanical forces and\\nextracellular matrix can influence tissue remodeling in developing organs.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.28.454180', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Cryo-EM reconstructions of inhibitor-bound SMG1 kinase reveal an autoinhibitory state dependent on SMG8', 'authors': \"[{'name': 'Lukas M. Langer'}, {'name': 'Fabien Bonneau'}, {'name': 'Yair Gat'}, {'name': 'Elena Conti'}]\", 'abstract': '<p>The PI3K-related kinase (PIKK) SMG1 monitors progression of metazoan nonsense-mediated mRNA decay (NMD) by phosphorylating the RNA helicase UPF1. Previous work has shown that the activity of SMG1 is impaired by small molecule inhibitors, is reduced by the SMG1 interactors SMG8 and SMG9, and is downregulated by the so-called SMG1 insertion domain. However, the molecular basis for this complex regulatory network has remained elusive. Here, we present cryo-electron microscopy reconstructions of human SMG1-9 and SMG1-8-9 complexes bound to either a SMG1 inhibitor or a non-hydrolyzable ATP analogue at overall resolutions ranging from 2.8 to 3.6 . These structures reveal the basis with which a small molecule inhibitor preferentially targets SMG1 over other PIKKs. By comparison with our previously reported substrate-bound structure (Langer et al. 2020), we show that the SMG1 insertion domain can exert an autoinhibitory function by directly blocking the substrate binding path as well as overall access to the SMG1 kinase active site. Together with biochemical analysis, our data indicate that SMG1 autoinhibition is stabilized by the presence of SMG8. Our results explain the specific inhibition of SMG1 by an ATP-competitive small molecule, provide insights into regulation of its kinase activity within the NMD pathway, and expand the understanding of PIKK regulatory mechanisms in general.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.26.453747', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'ABlooper: Fast accurate antibody CDR loop structure prediction with accuracy estimation', 'authors': \"[{'name': 'Brennan Abanades'}, {'name': 'Guy Georges'}, {'name': 'Alexander Bujotzek'}, {'name': 'Charlotte M. Deane'}]\", 'abstract': '<p>Antibodies are a key component of the immune system and have been extensively used as biotherapeutics. Accurate knowledge of their structure is central to understanding their function. The key area for antigen binding and the main area of structural variation in antibodies are concentrated in their six complementarity determining regions (CDRs), with the most variable being the CDR-H3 loop. The sequence and structure variability of CDR-H3 make it particularly challenging to model. Recently, deep learning methods have offered a step change in our ability to predict protein structures. In this work we present ABlooper, an end-to-end equivariant deep-learning based CDR loop structure prediction tool. ABlooper predicts the structure of CDR loops with high accuracy and provides a confidence estimate for each of its predictions. On the models of the Rosetta Antibody Benchmark, ABlooper makes predictions with an average H3 RMSD of 2.45, which drops to 2.02 when considering only its 76% most confident predictions.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453816', 'tweets': 23, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Synaptic Mechanisms of Top-Down Control by The Auditory Cortico-Collicular Pathway', 'authors': \"[{'name': 'Hannah M. Oberle'}, {'name': 'Alex Ford'}, {'name': 'Jordyn Czarny'}, {'name': 'Pierre F. Apostolides'}]\", 'abstract': '<p>Corticofugal projections to evolutionarily ancient, sub-cortical structures are ubiquitous across mammalian sensory systems. These descending pathways enable the neocortex to control ascending sensory representations in a predictive or feedback manner, but the underlying cellular mechanisms are poorly understood. Here we combine optogenetic approaches with <italic>in vivo</italic> and <italic>in vitro</italic> patch-clamp electrophysiology to study the projection from auditory cortex to the inferior colliculus (IC), a major descending auditory pathway that controls IC neuron feature selectivity, plasticity and auditory perceptual learning. Although individual auditory cortico-collicular synapses were generally weak, IC neurons often integrated inputs from multiple corticofugal axons that generated reliable, tonic depolarizations even during prolonged presynaptic activity. Latency measurements <italic>in vivo</italic> showed that descending signals reach the IC within 30 ms of sound onset, which in IC neurons corresponded to the peak of synaptic depolarizations evoked by short sounds. Activating ascending and descending pathways at latencies expected <italic>in vivo</italic> caused a NMDA receptor dependent, supra-linear EPSP summation, indicating that descending signals can non-linearly amplify IC neurons moment-to-moment acoustic responses. Our results shed light upon the synaptic bases of descending sensory control, and imply that heterosynaptic cooperativity contributes to the auditory cortico-collicular pathways role in plasticity and perceptual learning.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453647', 'tweets': 21, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Oncogenic c-Myc induces replication stress by increasing cohesins chromatin occupancy', 'authors': \"[{'name': 'Silvia Peripolli'}, {'name': 'Tanya Singh'}, {'name': 'Harshil Patel'}, {'name': 'Leticia Meneguello'}, {'name': 'Koshiro Kiso'}, {'name': 'Peter Thorpe'}, {'name': 'Cosetta Bertoli'}, {'name': 'Robertus A.M. de Bruin'}]\", 'abstract': '<p>Oncogene-induced replication stress is a major driver of genomic instability in cancer cells, with a central role in both cancer initiation and progression (1). Despite its critical role in cancer development, the mechanisms that lay at the basis of oncogene-induced replication stress remains poorly understood. Here, we investigate the mechanism of c-Myc-induced replication stress. Our data shows that c-Myc induces replication stress by increasing the amount of cohesins bound to chromatin in the G1 phase of the cell cycle. This is independent of previously suggested mechanisms involving deregulation of replication initiation and transcriptional interference. Restoring the amount of chromatin-bound cohesins to control levels, or preventing the accumulation of cohesins at CTCF sites, in cells experiencing oncogenic c-Myc activity prevents replication stress. Increased cohesins chromatin occupancy correlates with a c-Myc-dependent increase in the levels of the cohesion loader Mau2. Preventing c-Myc-induced increase in Mau2 reduces oncogene-induced replication stress. Together our data support a novel mechanism for oncogene-induced replication stress. Since c-Myc activation is a crucial event in many human cancers (2), identifying the mechanisms through which this oncogene promotes replication stress provides critical insights into cancer biology.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453673', 'tweets': 19, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Multivariate mining of an alpaca immune repertoire identifies potent cross-neutralising SARS-CoV-2 nanobodies', 'authors': \"[{'name': 'Leo Hanke'}, {'name': 'Daniel J. Sheward'}, {'name': 'Alec Pankow'}, {'name': 'Laura Perez Vidakovics'}, {'name': 'Vivien Karl'}, {'name': 'Changil Kim'}, {'name': 'Egon Urgard'}, {'name': 'Natalie L. Smith'}, {'name': 'Juan Astorga-Wells'}, {'name': 'Simon Ekstrm'}, {'name': 'Jonathan M. Coquet'}, {'name': 'Gerald M. McInerney'}, {'name': 'Ben Murrell'}]\", 'abstract': '<p>Conventional approaches to isolate and characterize nanobodies are laborious and cumbersome. Here we combine phage display, multivariate enrichment, and novel sequence analysis techniques to annotate an entire nanobody repertoire from an immunized alpaca. We combine this approach with a streamlined screening strategy to identify numerous anti-SARS-CoV-2 nanobodies, and use neutralization assays and Hydrogen/Deuterium exchange coupled to mass spectrometry (HDX-MS) epitope mapping to characterize their potency and specificity. Epitope mapping revealed that the binding site is a key determinant of neutralization potency, rather than affinity alone. The most potent nanobodies bind to the receptor binding motif of the RBD, directly preventing interaction with the host cell receptor ACE2, and we identify two exceptionally potent members of this category (with monomeric IC50s around 13 and 16 ng/ml). Other nanobodies bind to a more conserved epitope on the side of the RBD, and are able to potently neutralize the SARS-CoV-2 founder virus (42 ng/ml), the beta variant (B.1.351/501Y.V2) (35 ng/ml), and also cross-neutralize the more distantly related SARS-CoV-1 (0.46 g/ml). The approach presented here is well suited for the screening of phage libraries to identify functional nanobodies for various biomedical and biochemical applications.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453678', 'tweets': 16, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Reward-dependent selection of feedback gains impacts rapid motor decisions', 'authors': \"[{'name': 'Antoine De Comite'}, {'name': 'Frdric Crevecoeur'}, {'name': 'Philippe Lefvre'}]\", 'abstract': '<p>Expected reward is known to affect planning strategies through modulation of movement vigor. Strikingly, although current theories suggest that movement planning consists in selecting a goal-directed control policy, the influence of reward on feedback control strategies remains unknown. Here we investigated this question in three human reaching experiments. First, we varied the explicit reward associated with the goal target and found an overall increase in movement vigor for higher reward targets, highlighted by larger velocities, feedback responses to external loads, and background muscle activity. Then, assuming that larger feedback gains were used to reject perturbations, we sought to investigate whether this effect hindered online decisions to switch to a new target in the presence of multiple successful goals. We indeed observed idiosyncratic switching strategies dependent on both target rewards and movement vigor, such that the more vigorous movements were less likely to switch to a new goal following perturbations. To gain further insight into a causal influence of movement vigor on rapid motor decisions, we demonstrated that biasing the baseline activity and reflex gains by means of a background load evoked a larger proportion of target switches in the direction opposite to the background load associated with lower muscle activity. Our results highlight the competition between movement vigor and flexibility to switch target during movement.</p><sec><title>Significance statement</title><p>Humans can modulate their movement vigor based on the expected reward. However, a potential influence of reward on feedback control has not been documented. Here we investigated reaching control strategies in different contexts associated with explicit rewards for one or multiple goals, while exposed to external perturbations. We report two strategies: reward could either invigorate feedback gains, or promote flexible switches between goals. The engagement of peripheral circuits in the modulation of feedback gains was confirmed by the application of a background load that biased feedback vigor directionally, which evoked differences in switching behavior in the opposite direction. We conclude that feedback vigor and flexible changes in goal are two competing mechanisms to be selected when one interacts with a dynamic environment.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.21260947', 'tweets': 16, 'archive': 'medRxiv', 'subject-area': 'Respiratory Medicine', 'covid': False, 'title': 'Prospective validation of host transcriptomic biomarkers for pulmonary tuberculosis by real-time PCR', 'authors': \"[{'name': 'Simon C. Mendelsohn'}, {'name': 'Stanley Kimbung Mbandi'}, {'name': 'Andrew Fiore-Gartland'}, {'name': 'Adam Penn-Nicholson'}, {'name': 'Munyaradzi Musvosvi'}, {'name': 'Humphrey Mulenga'}, {'name': 'Michelle Fisher'}, {'name': 'Katie Hadley'}, {'name': 'Mzwandile Erasmus'}, {'name': 'Onke Nombida'}, {'name': 'Michle Tameris'}, {'name': 'Gerhard Walzl'}, {'name': 'Kogieleum Naidoo'}, {'name': 'Gavin Churchyard'}, {'name': 'Mark Hatherill'}, {'name': 'Thomas J. Scriba'}, {'name': ' '}, {'name': ' '}]\", 'abstract': '<p>We tested performance of host-blood transcriptomic tuberculosis (TB) signatures for active case-finding. Among 20,207 HIV-uninfected and 963 HIV-infected adults screened; 2,923 and 861 were enrolled from five South African communities. Eight signatures were measured by microfluidic RT-qPCR and participants were microbiologically-investigated for pulmonary TB at baseline, and actively surveilled for incident disease through 15 months. Diagnostic AUCs for 61 HIV-uninfected (weighted-prevalence 1.1%) and 10 HIV-infected (prevalence 1.2%) prevalent TB cases for the 8 signatures were 0.630.79 and 0.650.88, respectively. Thereafter, 24 HIV-uninfected and 9 HIV-infected participants progressed to incident TB (1.1 and 1.0 per 100 person-years, respectively). Prognostic AUCs through 15-months follow-up were 0.490.66 and 0.540.81, respectively. Prognostic performance for incident TB occurring within 6-12 months in HIV-negative participants was higher for all signatures. None of the signatures met WHO Target Product Profile criteria for a triage test to diagnose asymptomatic TB; most signatures met the criteria for symptomatic TB. Prognostic accuracy of most signatures for incident TB within six months of testing met the criteria for an incipient TB test.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.29.454071', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Age-Related Differences in Ventral Striatal and Default Mode Network Function During Reciprocated Trust', 'authors': \"[{'name': 'Dominic S. Fareri'}, {'name': 'Katherine Hackett'}, {'name': 'Lindsey J. Tepfer'}, {'name': 'Victoria Kelly'}, {'name': 'Nicole Henninger'}, {'name': 'Crystal Reeck'}, {'name': 'Tania Giovannetti'}, {'name': 'David V. Smith'}]\", 'abstract': '<p>Social relationships change across the lifespan as social networks narrow and motivational priorities shift to the present. Interestingly, aging is also associated with changes in executive function, including decision-making abilities, but it remains unclear how age-related changes in both domains interact to impact financial decisions involving other people. To study this problem, we recruited 50 human participants (N<sub>younger</sub> = 26, ages 18-34; N<sub>older</sub> = 24, ages 63-80) to play an economic trust game as the investor with three partners (friend, stranger, and computer) who played the role of investee. Investors underwent functional magnetic resonance imaging (fMRI) during the trust game while investees were seated outside of the scanner. Building on our previous work with younger adults showing both enhanced striatal responses and altered default-mode network (DMN) connectivity as a function of social closeness during reciprocated trust, we predicted that these relations would exhibit age-related differences. We found that striatal responses to reciprocated trust from friends relative to strangers and computers were blunted in older adults relative to younger adults, thus supporting our primary pre-registered hypothesis regarding social closeness. We also found that older adults exhibited enhanced DMN connectivity with the temporoparietal junction (TPJ) during reciprocated trust from friends compared to computers while younger adults exhibited the opposite pattern. Taken together, these results advance our understanding of age-related differences in sensitivity to social closeness in the context of trusting others.</p><sec><title>Significance</title><p>Our social relationships change as we grow older. Despite the ubiquity of these changes, little is known about how they impact our neural and behavioral responses to financial decisions involving other people. We examined this problem in the context of a trust game played with three partners varying in social closeness (friend, stranger, computer) while participants underwent fMRI. We found both blunted striatal activation and enhanced DMN-TPJ connectivity to reciprocated trust from friends relative to strangers and computers in older compared to younger adults. These results suggest that older adults demonstrate altered representations of social closeness within financial exchanges involving trust, which may expose older adults to increased risk for financial exploitation perpetrated by strangers.</p></sec>', 'posted': '2021-07-30'}, {'doi': '10.1101/2021.07.26.453637', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'LAMP2A regulates the loading of proteins into exosomes', 'authors': \"[{'name': 'Joo Vasco Ferreira'}, {'name': 'Ana da Rosa Soares'}, {'name': 'Jos Ramalho'}, {'name': 'Catarina Mximo Carvalho'}, {'name': 'Maria Helena Cardoso'}, {'name': 'Petra Pintado'}, {'name': 'Ana Sofia Carvalho'}, {'name': 'Hans Christian Beck'}, {'name': 'Rune Matthiesen'}, {'name': 'Mnica Zuzarte'}, {'name': 'Henrique Giro'}, {'name': 'Guillaume van Niel'}, {'name': 'Paulo Pereira'}]\", 'abstract': '<p>Exosomes are extracellular vesicles of endosomal origin released by virtually all cell types across metazoans. Exosomes are active vehicles of intercellular communication and can transfer lipids, RNAs and proteins between different cells, tissues or organs. However, the mechanisms that regulate the selective loading of cytosolic proteins into these vesicles are still largely unknow. Here we describe a mechanism whereby proteins containing a pentapeptide sequence, biochemically related to the KFERQ-motif, are loaded into a subpopulation of exosomes in a process that is dependent on the membrane protein LAMP2A. Moreover, this mechanism is independent of the ESCRT machinery components TSG101 and VPS4b and dependent on HSC70, CD63, Alix, Syntenin-1, Rab31 and ceramides. The transcription factor and master regulator of hypoxia HIF1A is loaded into exosomes by this mechanism to transport hypoxia signaling to normoxic cells. Additionally, by tagging fluorescent proteins with KFERQ-like sequences we were able to follow inter-organ transfer of exosomes in zebrafish larvae. Our findings identify LAMP2A as a key component in exosome biogenesis while opening new avenues for exosome engineering by allowing the loading of bioactive proteins by tagging them with KFERQ-like motifs.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453666', 'tweets': 15, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Phylogenetic distribution and expression pattern analyses identified a divergent basal body assembly protein involved in land plant spermatogenesis', 'authors': \"[{'name': 'Shizuka Koshimizu'}, {'name': 'Naoki Minamino'}, {'name': 'Tomoaki Nishiyama'}, {'name': 'Emiko Yoro'}, {'name': 'Kazuo Ebine'}, {'name': 'Keiko Sakakibara'}, {'name': 'Takashi Ueda'}, {'name': 'Kentaro Yano'}]\", 'abstract': '<p>Oogamy is a form of sexual reproduction and evolved independently in animals, fungi, and plants. In streptophyte plants, Charophyceae, Coleochaetophyceae, bryophytes, lycophytes, ferns (monilophytes), and some gymnosperms (Cycads and Ginkgo) utilize spermatozoids as the male gamete. Plant spermatozoids commonly possess characteristic structures such as the spline, which consists of a microtubule array, the multilayered structure (MLS) in which the uppermost layer is continuum of the spline, and multiple flagella. However, the molecular mechanisms underpinning plant spermatogenesis remain to be elucidated. To identify the genes involved in plant spermatogenesis, we performed computational analyses and successfully found deeply divergent <italic>BLD10</italic>s by combining multiple methods and omics-data. We then validated the functions of candidate genes in the liverwort <italic>Marchantia polymorpha</italic> and the moss <italic>Physcomitrium patens</italic> and found that Mp<italic>BLD10</italic> and Pp<italic>BLD10</italic> are required for normal basal body and flagella formation. Mp<italic>bld10</italic> mutants exhibited defects in remodeling of the cytoplasm and nucleus during spermatozoid formation, thus Mp<italic>BLD10</italic> should be involved in chromatin reorganization and elimination of the cytoplasm during spermiogenesis. Streptophyte BLD10s are orthologous to BLD10/CEP135 family proteins, which function in basal body assembly, but we found that BLD10s evolved especially fast in land plants and MpBLD10 might obtain additional functions in spermatozoid formation through the fast molecular evolution. This study provides a successful example of combinatorial study from evolutionary and molecular genetic perspectives that elucidated a function of the key protein of the basal body formation that fast evolved in land plants.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453674', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Role of non-specific interactions in the phase-separation and maturation of macromolecules', 'authors': \"[{'name': 'Rakesh Krishnan'}, {'name': 'Srivastav Ranganathan'}, {'name': 'Samir K Maji'}, {'name': 'Ranjith Padinhateeri'}]\", 'abstract': '<p>Phase separation of biomolecules could be mediated by both specific and non-specific interactions. How the interplay between non-specific and specific interactions along with polymer entropy influences phase separation is an open question. We address this question by simulating self-associating molecules as polymer chains with a short core stretch that forms the specifically interacting functional interface, and longer non-core regions that participate in non-specific/promiscuous interactions. Our results show that the interplay of specific (strength, <italic></italic><sub>sp</sub>) and non-specific interactions (strength, <italic></italic><sub>ns</sub>) could result in phase separation of polymers and its transition to solid-like aggregates (mature state). In the absence of <italic></italic><sub>ns</sub>, the polymer chains do not dwell long enough in the vicinity of each other to undergo phase separation and transition into a mature state. On the other hand, in the limit of strong <italic></italic><sub>ns</sub>, the assemblies cannot transition into the mature state because they get kinetically trapped, suggesting an optimal range of interactions favoring mature multimers. In the scenario where only a fraction (N<sub>frac</sub>) of the non-core regions participate in attractive interactions, we find that a slight modifications to either <italic></italic><sub>ns</sub> or N<sub>frac</sub> can result in a dramatically altered self-assembled states. Using a combination of heterogeneous and homogeneous mix of polymers, we establish how this interplay between interaction energies dictates the propensity of biomolecules to find the correct binding partner at dilute concentrations in crowded environments.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453634', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Crowding-induced phase separation of nuclear transport receptors in FG nucleoporin assemblies', 'authors': \"[{'name': 'Luke K. Davis'}, {'name': 'Ian J. Ford'}, {'name': 'Bart W. Hoogenboom'}]\", 'abstract': '<p>The rapid (&lt; 1 ms) transport of biological material to and from the cell nucleus is regulated by the nuclear pore complex (NPC). At the core of the NPC is a permeability barrier consisting of intrinsically disordered Phe-Gly (FG) nucleoporins (FG Nups). Various types of nuclear transport receptors (NTRs) facilitate transport by partitioning in the FG Nup assembly, overcoming the barrier by their affinity to the FG Nups, and comprise a significant fraction of proteins in the NPC barrier. In previous work, we revealed that the experimental binding of the NTRs NTF2 and  the larger  Imp<italic></italic> to different planar assemblies of FG Nups follows a universal physical law defined by negative cooperativity, which was further validated by a minimal physical model that treated the FG Nups as flexible homopolymers and the NTRs as uniformly cohesive spheres <bold><italic>Zahn et al. (2016</italic></bold>). Here, we build upon our original study by first parametrizing our model to experimental data, and next to predict the effects of crowding by different types of NTRs. We show how varying the amounts of one type of NTR modulates how the other NTR penetrates the FG Nup assembly. Notably, at similar and physiologically relevant NTR concentrations, our model predicts demixed phases of NTF2 and Imp<italic></italic> within the FG Nup assembly. The functional implication of NTR phase separation is that NPCs may sustain separate transport pathways that are determined by inter-NTR competition.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.22.453442', 'tweets': 14, 'archive': 'bioRxiv', 'subject-area': 'Genetics', 'covid': False, 'title': 'Leveraging gene co-regulation to identify gene sets enriched for disease heritability', 'authors': \"[{'name': 'Katherine M. Siewert-Rocks'}, {'name': 'Samuel S. Kim'}, {'name': 'Douglas W. Yao'}, {'name': 'Huwenbo Shi'}, {'name': 'Alkes L. Price'}]\", 'abstract': '<p>Identifying gene sets that are associated to disease can provide valuable biological knowledge, but a fundamental challenge of gene set analyses of GWAS data is linking disease-associated SNPs to genes. Transcriptome-wide association studies (TWAS) can be used to detect associations between the genetically predicted expression of a gene and disease risk, thus implicating candidate disease genes. However, causal disease genes at TWAS-associated loci generally remain unknown due to gene co-regulation, which leads to correlations across genes in predicted expression. We developed a new method, gene co-regulation score (GCSC) regression, to identify gene sets that are enriched for disease heritability explained by the predicted expression of causal disease genes in the gene set. GCSC regresses TWAS chi-square statistics on gene co-regulation scores reflecting correlations in predicted gene expression; GCSC determines that a gene set is enriched for disease heritability if genes with high co-regulation to the gene set have higher TWAS chi-square statistics than genes with low co-regulation to the gene set, beyond what is expected based on co-regulation to all genes. We verified via simulations that GCSC is well-calibrated, and well-powered to identify gene sets that are enriched for disease heritability explained by predicted expression. We applied GCSC to gene expression data from GTEx (48 tissues) and GWAS summary statistics for 43 independent diseases and complex traits (average <italic>N</italic> =344K), analyzing a broad set of biological pathways and specifically expressed gene sets. We identified many enriched gene sets, recapitulating known biology. For Alzheimers disease, we detected evidence of an immune basis, and specifically a role for antigen presentation, in analyses of both biological pathways and specifically expressed gene sets. Our results highlight the advantages of leveraging gene co-regulation within the TWAS framework to identify gene sets associated to disease.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261042', 'tweets': 13, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Robust antibody levels in both diabetic and non-diabetic individuals after BNT162b2 mRNA COVID-19 vaccination', 'authors': \"[{'name': 'Hamad Ali'}, {'name': 'Abdelmohsen Al-Terki'}, {'name': 'Sardar Sindhu'}, {'name': 'Barrak Alahmad'}, {'name': 'Maha Hammad'}, {'name': 'Salman Al-Sabah'}, {'name': 'Mohammad Alghounaim'}, {'name': 'Mohammad H. Jamal'}, {'name': 'Ali Aldei'}, {'name': 'Mohammad J. Mairza'}, {'name': 'Maitham Husain'}, {'name': 'Sriraman Deverajan'}, {'name': 'Rasheed Ahmad'}, {'name': 'Preethi Cherian'}, {'name': 'Irina Alkhairi'}, {'name': 'Abdullah Alkandari'}, {'name': 'Jehad Abubaker'}, {'name': 'Mohamed Abu-Farha'}, {'name': 'Fahd Al-Mulla'}]\", 'abstract': '<p>The emergence of effective vaccines for COVID-19 has been welcomed by the world with great optimism. Given their increased susceptibility to COVID-19, the question arises whether individuals with type-2 diabetes mellitus (T2DM) and other metabolic conditions can respond effectively to the mRNA-based vaccine. We aimed to evaluate the levels of anti-SARS-CoV-2 IgG and neutralizing antibodies in people with T2DM and/or other metabolic risk factors (hypertension and obesity) compared to those without. This study included 262 people that took two doses of BNT162b2 (PfizerBioNTech) mRNA vaccine. Both T2DM and non-diabetic individuals had a robust response to vaccination as demonstrated by their high antibody titers. However, both SARS-CoV-2 IgG and neutralizing antibodies titers were lower in people with T2DM. Their levels were 15449.1 vs. 13859.4BAU/mL for IgG and 87.111.6 vs. 79.719.5% for neutralizing antibodies in individuals without diabetes compared to those with T2DM, respectively. In a multiple linear regression adjusted for individual characteristics, comorbidities, previous COVID-19 infection and duration since second vaccine dose, diabetics had 13.86 BAU/ml (95%CI: 27.08 to 0.64BAU/ml, p=0.041) less IgG antibodies and 4.42% (95%CI: 8.53 to 0.32%, p=0.036) less neutralizing antibodies than non-diabetics. Hypertension and obesity did not show significant changes in antibody titers. Taken together, both type-2 diabetic and non-diabetic individuals elicited strong immune responses to SARS-CoV-2 BNT162b2 mRNA vaccine; nonetheless, lower levels were seen in people with diabetes. Continuous monitoring of the antibody levels might be a good indicator to guide personalized needs for further booster shots to maintain adaptive immunity.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453574', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Bats host the most virulentbut not the most dangerouszoonotic viruses', 'authors': \"[{'name': 'Sarah Guth'}, {'name': 'Nardus Mollentze'}, {'name': 'Katia Renault'}, {'name': 'Daniel G. Streicker'}, {'name': 'Elisa Visher'}, {'name': 'Mike Boots'}, {'name': 'Cara E. Brook'}]\", 'abstract': '<p>Identifying virus characteristics associated with the largest public health impacts on human populations is critical to informing zoonotic risk assessments and surveillance strategies. Efforts to assess zoonotic risk often use trait-based analyses to identify which viral and reservoir host groups are most likely to source zoonoses but have not fully addressed how and why the impacts of zoonotic viruses vary in terms of disease severity (virulence), capacity to spread within human populations (transmissibility), or total human mortality (death burden). We analyzed trends in human case fatality rates, transmission capacities, and total death burdens across a comprehensive dataset of mammalian and avian zoonotic viruses. Bats harbor the most virulent zoonotic viruses even when compared to birds, which alongside bats, have been hypothesized to be special zoonotic reservoirs due to molecular adaptations that support the physiology of flight. Reservoir host groups more closely related to humansin particular, Primatesharbor less virulent, but more highly transmissible viruses. Importantly, disproportionately high human death burden, arguably the most important metric of zoonotic risk, is not associated with any animal reservoir, including bats. Our data demonstrate that mechanisms driving death burdens are diverse and often contradict trait-based predictions. Ultimately, total human mortality is dependent on context-specific epidemiological dynamics, which are shaped by a combination of viral traits and conditions in the animal host population and across and beyond the human-animal interface. Understanding the conditions that predict high zoonotic burden in humans will require longitudinal studies of epidemiological dynamics in wildlife and human populations.</p><sec><title>Significance statement</title><p>The clear need to mitigate zoonotic risk has fueled increased viral discovery in specific reservoir host taxa. We show that a combination of viral and reservoir traits can predict zoonotic virus virulence and transmissibility in humans, supporting the hypothesis that bats harbor exceptionally virulent zoonoses. However, pandemic prevention requires thinking beyond zoonotic capacity, virulence, and transmissibility to consider collective burden on human health. For this, viral discovery targeting specific reservoirs may be inefficient as death burden correlates with viral, not reservoir, traits, and depends on context-specific epidemiological dynamics across and beyond the human-animal interface. These findings suggest that longitudinal studies of viral dynamics in reservoir and spillover host populations may offer the most effective strategy for mitigating zoonotic risk.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.28.454097', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'Integrative analysis of the salt stress response in cyanobacteria', 'authors': \"[{'name': 'Stephan Klaehn'}, {'name': 'Stefan Mikkat'}, {'name': 'Matthias Riediger'}, {'name': 'Jens Georg'}, {'name': 'Wolfgang R Hess'}, {'name': 'Martin Hagemann'}]\", 'abstract': '<p>Microorganisms evolved specific acclimation strategies to thrive in environments of high or fluctuating salinities. Here, salt acclimation in the model cyanobacterium Synechocystis sp. PCC 6803 was analyzed by integrating transcriptomic, proteomic and metabolomic data. A dynamic reorganization of the transcriptome and proteome occurred during the first hours after salt shock, e.g. involving the upregulation of genes to activate compatible solute biochemistry balancing osmotic pressure. The massive accumulation of glucosylglycerol then had a measurable impact on the overall carbon and nitrogen metabolism. In addition, we observed the coordinated induction of putative regulatory RNAs and of several proteins known for their involvement in other stress responses. Overall, salt-induced changes in the proteome and transcriptome showed good correlations, especially among the stably up-regulated proteins and their transcripts. We define an extended salt stimulon comprising proteins directly or indirectly related to compatible solute metabolism, ion and water movements, and a distinct set of regulatory RNAs involved in post-transcriptional regulation. Our comprehensive data set provides the basis for engineering cyanobacterial salt tolerance and to further understand its regulation.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.25.453708', 'tweets': 12, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'The effective family size of immigrant founders predicts their long-term demographic outcome: from Qubec settlers to their 20th-century descendants', 'authors': \"[{'name': 'Damian Labuda'}, {'name': 'Tommy Harding'}, {'name': 'Emmanuel Milot'}, {'name': 'Hlne Vzina'}]\", 'abstract': '<p>Human evolution involves population splits, size fluctuations, founder effects, and admixture. Population history reconstruction based on genetic diversity data routinely relies on simple demographic models while projecting the past. No specific demographic assumptions are needed to understand the genetic structure of the founder population of Qubec. Because genealogy and genetics are intimately related, we used descending genealogies of this population to pursue the fate of its founder lineages. Maternal and paternal lines reflect the transmission of mtDNA and the Y-chromosome, respectively. We followed their transmission in real-time, from the 17<sup>th</sup> century down to its 20<sup>th</sup>-century population. We counted the number of married children of immigrants (i.e., their effective family size, EFS), estimated the proportion of successful immigrants in terms of their survival ratio, and assessed net growth rates and extinction. Likewise, we evaluated the same parameters for their Qubec-born descendants. The survival ratio of the first immigrants was the highest and declined over time in association with the decreasing immigrants EFS. Parents with high EFS left plentiful married progeny, putting EFS as the most important variable determining the parental demographic success throughout time for generations ahead. The 17<sup>th</sup> and 18<sup>th</sup>-century immigrants bear the most remarkable demographic and genetic impact on the 20<sup>th</sup>-century population of Qubec. Lessons learned from Qubec genealogies can teach us about the consequences of founder effects and migrations through real peoples history. The effective family size of immigrant founders predicts their long-term demographic outcome.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.26.453807', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'The benefits of merging two tracking approaches: passive acoustic telemetry combined with active acoustic tracking by transects provide new insights into riverine migration by salmonid smolts', 'authors': \"[{'name': 'Louise Chavarie'}, {'name': 'Hannele M. Honkanen'}, {'name': 'Matthew Newton'}, {'name': 'Jessie M. Lilly'}, {'name': 'Hannah R. Greetham'}, {'name': 'Colin E. Adams'}]\", 'abstract': '<p>The process of smolting is a critical phase in the life-cycle of anadromous salmonids and it has been associated with substantial rates of mortality. Survival during freshwater and marine migration is known to have population level effects, thus an understanding of the patterns of mortality has the potential to yield important insights into population bottlenecks. Despite important advancements in tracking techniques, the specifics of mortality events in anadromous salmonids during their initial migration to sea remains somewhat elusive. Here, we develop a framework combining spatial and temporal detections of smolt riverine migration from two tracking techniques, which enable inferences to be made about mortality locations, causes, and rates. We embed this framework into a fine-scale behaviour study of migration and social structure. In this study, we demonstrate that during their initial riverine transitional phase, smolts were particularly vulnerable to predators. Specifically, avian predation appeared to be the main cause of mortality (42%), although piscine predation events were not trivial (14%). Our results suggested some direct and indirect tagging-induced mortality (e.g., through increased predation vulnerability), which highlights the importance of determining tagging mortality in a telemetry study to ensure adequate interpretation of migration success. There is evidence that predation induced selection on smolt morphology. Unsuccessful river salmon migrants with a phenotype comprising a shorter head and jaw and smaller eye had a higher probability of mortality in the later parts of riverine migration where avian and aquatic predation mortality dominated. In contrast, mortality earlier in river migration was independent of phenotype, most likely a result of tagging effects. Successfully river migrants were found to be interacting with each other, often in a pair or as a trio, indicating that a few individuals of Atlantic salmon and sea trout formed notable intra- and interspecific social associations. However, the heterogeneity of the social associations of successful migrants did not support the assumption that social network features attributed any clear specific benefits of reduced predation risk. Overall, by estimating migration loss and its variability, our study framework should help to guide management actions to mitigate the widespread population declines these species are currently facing.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.452477', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Trial-wise exposure to visual appetitive cues increases physiological arousal but not temporal discounting', 'authors': \"[{'name': 'Kilian Knauth'}, {'name': 'Jan Peters'}]\", 'abstract': '<p>Humans and many animals devalue future rewards as a function of time (temporal discounting). Increased discounting has been linked to various psychiatric conditions, including substance-use-disorders, behavioral addictions and obesity. Despite its high intra-individual stability, temporal discounting is partly under contextual control. One prominent manipulation that has been linked to increases in discounting is the exposure to highly arousing appetitive cues. However, results from trial-wise cue exposure studies appear highly mixed, and changes in physiological arousal were not adequately controlled. Here we tested the effects of appetitive (erotic), aversive and neutral visual cues on temporal discounting in thirty-five healthy male participants. The contribution of single-trial physiological arousal was assessed using comprehensive monitoring of autonomic activity (pupil size, heart rate, electrodermal activity). Physiological arousal was elevated following aversive and in particular erotic cues. In contrast to our pre-registered hypothesis, if anything, we observed <italic>decreased</italic> temporal discounting following erotic cue exposure. Aversive cues tended to increase decision noise. Computational modeling revealed that trial-wise arousal only accounted for minor variance over and above aversive and erotic condition effects, arguing against a general effect of physiological arousal on temporal discounting.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.28.454213', 'tweets': 11, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Mitochondria dysfunction in Charcot Marie Tooth 2B Peripheral Sensory Neuropathy', 'authors': \"[{'name': 'Yingli Gu'}, {'name': 'Flora Guerra'}, {'name': 'Mingzheng Hu'}, {'name': 'Alexander Pope'}, {'name': 'Kijung Sung'}, {'name': 'Wanlin Yang'}, {'name': 'Simone Jetha'}, {'name': 'Thomas A. Shoff'}, {'name': 'Tessanya Gunatilake'}, {'name': 'Owen Dahlkamp'}, {'name': 'Linda Zhixia Shi'}, {'name': 'Fiore Manganelli'}, {'name': 'Maria Nolano'}, {'name': 'Yue Zhou'}, {'name': 'Jianqing Ding'}, {'name': 'Cecilia Bucci'}, {'name': 'Chengbiao Wu'}]\", 'abstract': '<p>Recent evidence has uncovered an important role of Rab7 in regulating mitochondrial morphology and function. Missense mutation(s) of Rab7 underlies the pathogenesis of Charcot Marie Tooth 2B (CMT2B) peripheral neuropathy. Herein, we investigated how mitochondrial morphology and function were impacted by the CMT2B associated Rab7<sup>V162M</sup> mutation in fibroblasts from human CMT2B patients as well as in a knockin mouse model. In contrast to recently published results from studies of using heterologous overexpression systems, our results have demonstrated significant mitochondrial fragmentation in fibroblasts of both human CMT2B patients and CMT2B mouse embryonic fibroblasts (MEFs). Furthermore, we have shown that mitochondria were fragmented and axonal mitochondrial movement was dysregulated in primary cultured E18 dorsal root ganglion (DRG) sensory neurons, but not in E18 hippocampal and cortical primary neurons. We also show that inhibitors to either the mitochondrial fission protein Drp1 or to the nucleotide binding to Rab7 normalized the mitochondrial deficits in both MEFs and E18 cultured DRG neurons. Our study has revealed, for the first time, that expression of CMT2B Rab7 mutation at physiological level enhances Drp1 activity to promote mitochondrial fission, that may potentially underlie selective vulnerability of peripheral sensory neurons in CMT2B pathogenesis.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.26.453733', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'AGO1 regulates major satellite transcripts and H3K9me3 distribution at pericentromeric regions in mESCs', 'authors': \"[{'name': 'Madlen Mller'}, {'name': 'Tara Fh'}, {'name': 'Moritz Schfer'}, {'name': 'Janina Luitz'}, {'name': 'Patrick Stalder'}, {'name': 'Rajika Arora'}, {'name': 'Richard Patryk Ngondo'}, {'name': 'Constance Ciaudo'}]\", 'abstract': '<p>In the past years, several studies reported nuclear roles for the Argonaute (AGO) proteins associating them with transcriptional activation or repression, alternative splicing and, chromatin organization. However, as most of these experiments have been conducted in human cancer cell lines, the nuclear functions of the AGO proteins in mouse early embryonic development still remains elusive. In this study, we investigated possible nuclear functions of the AGO proteins in mouse Embryonic Stem Cells (mESCs). By biochemical assays, we observed that AGO1 and AGO2 are present in a small fraction in the nucleus and even less on chromatin in mESCs. To profile the nuclear interactome of the AGO proteins, we performed immunoprecipitation followed by Mass Spectrometry and identified three novel nuclear interactors for AGO1, namely DNMT3a, HP1<italic></italic>, and ATRX. These interactors are well-known proteins involved in the establishment and maintenance of heterochromatin at pericentromeric regions. Indeed, upon depletion of <italic>Ago1,</italic> we observed a specific redistribution of the heterochromatin protein HP1<italic></italic> and the repressive histone mark H3K9me3, away from pericentromeric regions. Furthermore, these regions are characterized by AT-rich tandem repeats known as major satellite sequences. We demonstrated that major satellite transcripts are strongly upregulated in <italic>Ago1_KO</italic> mESCs. Interestingly, this phenotype was not caused by the loss of genome integrity at pericentromeres, as these could still form normally in <italic>Ago1</italic>_KO mESCs. Lastly, we showed that specific microRNAs loaded in AGO1, regulate the expression of the major satellite transcripts. Overall, our results demonstrate for the first time a novel role for AGO1 in regulating major satellite transcripts and localization of HP1<italic></italic> and H3K9me3 at pericentromeres in mESCs.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453671', 'tweets': 10, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'MetaBinner: a high-performance and stand-alone ensemble binning method to recover individual genomes from complex microbial communities', 'authors': \"[{'name': 'Ziye Wang'}, {'name': 'Pingqin Huang'}, {'name': 'Ronghui You'}, {'name': 'Fengzhu Sun'}, {'name': 'Shanfeng Zhu'}]\", 'abstract': '<p>Binning is an essential procedure during metagenomic data analysis. However, the available individual binning methods usually do not simultaneously fully use different features or biological information. Furthermore, it is challenging to integrate multiple binning results efficiently and effectively. Therefore, we developed an ensemble binner, MetaBinner, which generates component results with multiple types of features and utilizes single-copy gene (SCG) information for k-means initialization. It then utilizes a two-step ensemble strategy based on SCGs to integrate the component results. Extensive experimental results over three large-scale simulated datasets and one real-world dataset demonstrate that MetaBinner outperforms other state-of-the-art individual binners and ensemble binners. MetaBinner is freely available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/ziyewang/MetaBinner\">https://github.com/ziyewang/MetaBinner</ext-link>.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.29.454133', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Deep learning identifies and quantifies recombination hotspot determinants', 'authors': \"[{'name': 'Yu Li'}, {'name': 'Siyuan Chen'}, {'name': 'Trisevgeni Rapakoulia'}, {'name': 'Hiroyuki Kuwahara'}, {'name': 'Kevin Y. Yip'}, {'name': 'Xin Gao'}]\", 'abstract': '<p>Recombination is one of the essential genetic processes for sexually reproducing organisms, which can happen more frequently in some regions, called recombination hotspots. Although several factors, such as PRDM9 binding motifs, are known to be related to the hotspots, their contributions to the recombination hotspots have not been quantified, and other determinants are yet to be elucidated. Here, we develop a computational method, RHSNet, based on deep learning and signal processing, to identify and quantify the hotspot determinants in a purely data-driven manner, utilizing datasets from various studies, populations, sexes, and species. In addition to being able to identify hotspot regions and the well-known determinants accurately, RHSNet is sensitive to the difference between different PRDM9 alleles and different sexes, and can generalize to PRDM9-lacking species. The cross-sex, cross-population, and cross-species studies suggest that the proposed method has the potential to identify and quantify the evolutionary determinant motifs.</p><sec><title>Teaser</title><p>RHSNet can accurately identify and quantify recombination hotspot determinants across different studies, sexes, populations, and species.</p></sec>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.07.23.453597', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Generalizing Bayesian phylogenetics to infer shared evolutionary events', 'authors': \"[{'name': 'Jamie R. Oaks'}, {'name': 'Perry L. Wood'}]\", 'abstract': '<p>Many processes of biological diversification can affect multiple evolutionary lineages. Examples include multiple members of a gene family diverging when a region of a chromosome is duplicated, multiple viral strains diverging at a super-spreading event, and a geological event fragmenting whole communities of species. It is difficult to test for patterns predicted by such processes, because all phylogenetic methods assume that lineages diverge independently. We introduce a general Bayesian framework to relax the assumption of independent divergences during phylogenetic inference, and test for patterns predicted by processes of diversification that affect multiple evolutionary lineages. Using simulations, we find our new method accurately infers shared divergence events when they occur, and performs as well as current methods when divergnces are independent. We apply our new approach to genomic data from two genera of geckos from across the Philippines to test if past changes to the islands landscape caused bursts of speciation. Unlike previous analyses restricted to only pairs of populations, we find evidence for patterns of shared divergences. By generalizing the space of phylogenetic trees in a way that is independent from the likelihood model, our approach opens many avenues for future research.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453744', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Global protein responses of multi-drug resistant plasmid containing Escherichia coli to ampicillin, cefotaxime, imipenem and ciprofloxacin', 'authors': \"[{'name': 'Anatte Margalit'}, {'name': 'James C. Carolan'}, {'name': 'Fiona Walsh'}]\", 'abstract': '<p>Antimicrobial resistance (AMR) and multi-drug resistance (MDR) in pathogenic bacteria are frequently mediated by plasmids. However, plasmids do not exist in isolation but rather require the bacterial host interaction in order to produce the AMR phenotype. This study aimed to utilise mass spectrometry-based proteomics to reveal the plasmid and chromosomally derived protein profile of <italic>Escherichia coli</italic> under antimicrobial stress. This was achieved by comparing the proteomes of <italic>E. coli</italic> containing the MDR pEK499 plasmid, under ampicillin, cefotaxime, imipenem or ciprofloxacin stress with the proteomes of these bacteria grown in the absence of antimicrobial. Our analysis identified statistically significant differentially abundant proteins common to groups exposed to the -lactam antimicrobials but not ciprofloxacin, indicating a -lactam stress response to exposure from this class of drugs, irrespective of -lactam resistance or susceptibility. These include ecotin and free methionine-R-sulfoxide reductase. These data also identified distinct differences in the cellular response to each -lactam. Data arising from comparisons of the proteomes of ciprofloxacin-treated <italic>E. coli</italic> and controls detected an increase in the relative abundance of proteins associated with ribosomes, translation, the TCA-cycle and several proteins associated with detoxification and a decrease in the relative abundances of proteins associated with stress response, including oxidative stress. We identified changes in proteins associated with persister formation in the presence of ciprofloxacin but not the -lactams. The plasmid proteome differed across each treatment and did not follow the pattern of antimicrobial  AMR protein associations. For example, a relative increase in the amount of blaCTX-M-15 in the presence of cefotaxime and ciprofloxacin but not the other -lactams, suggesting regulation of the blaCTX-M-15 protein production. The proteomic data from the this study provided novel insights into the proteins produced from the chromosome and plasmid under different antimicrobial stresses. These data also identified novel proteins not previously associated with AMR or antimicrobials responses in pathogens, which may well represent potential targets of AMR inhibition.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453699', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Supra-physiological levels of gibberellins/DELLAs modify the root cell size/number and the root architecture in root tips of A. thaliana seedlings. Connections to the root hair patterning and abundance', 'authors': \"[{'name': 'Iva McCarthy-Surez'}]\", 'abstract': '<p>A previous study (McCarthy-Surez, 2021) showed that growing <italic>A. thaliana</italic> seedlings for 5 days under excessive levels of gibberellins (GAs)/DELLAs altered the arrangement, shape and frequency of root hairs in root tips. Because no changes in the distribution or number of root hairs occurred when the gai-1 (gibberellin-insensitive-1) DELLA was over-expressed at the root epidermis, it was concluded that the GAs/DELLAs might regulate the root hair patterning and abundance in <italic>A. thaliana</italic> seedlings by acting from the root sub-epidermal tissues. In the present study, microscopy analyses showed that excessive levels of GAs/DELLAs also modified the size and number of root tip cells in <italic>A. thaliana</italic> seedlings. While excessive DELLAs shortened and widened the root epidermal, cortical, endodermal and pericycle cells, excessive GAs, excepting the epidermal cells, generally narrowed them. However, no changes of root cell size occurred when gai-1 was over-expressed at the root epidermis. In addition, high levels of DELLAs often induced extra cells at the root epidermis, cortex, endodermis and pericycle, whereas high levels of GAs sometimes induced extra cells at the root cortex and pericycle. On the other hand, excessive levels of DELLAs enhanced the outgrowth of lateral roots in root tips, unlike excessive levels of GAs. Thus, the results of this study suggest that supra-physiological levels of GAs/DELLAs might modify the size/number of root tip cells by acting from the root sub-epidermal tissues. This, in turn, might impact on the patterning and abundance of root hairs and on the root architecture.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.30.454065', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Subcellular and regional localization of mRNA translation in midbrain dopamine neurons', 'authors': \"[{'name': 'Benjamin D Hobson'}, {'name': 'Linghao Kong'}, {'name': 'Maria Florencia Angelo'}, {'name': 'Ori J Lieberman'}, {'name': 'Eugene V Mosharov'}, {'name': 'Etienne Herzog'}, {'name': 'David Sulzer'}, {'name': 'Peter A Sims'}]\", 'abstract': '<p>Local translation within excitatory and inhibitory neurons is involved in neuronal development and synaptic plasticity. Despite the extensive dendritic and axonal arborizations of central monoaminergic neurons, the subcellular localization of protein synthesis is not well-characterized in these populations. Here, we investigated mRNA localization and translation in midbrain dopaminergic (mDA) neurons, cells with enormous axonal and dendritic projections, both of which exhibit stimulation-evoked dopamine (DA) release. Using highly-sensitive ribosome-bound RNA-sequencing and imaging approaches in mDA axons, we found no evidence for axonal mRNA localization or translation. In contrast, mDA neuronal dendritic projections into the substantia nigra reticulata (SNr) contain ribosomes and mRNAs encoding the major components of DA synthesis, release, and reuptake machinery. Surprisingly, we also observed dendritic localization of mRNAs encoding synaptic vesicle-related proteins, including those involved in vesicular exocytic fusion. Our results are consistent with a role for local translation in the regulation of DA release from dendrites, but not from axons. Our translatome data further defined a molecular signature of the sparse mDA neurons resident in the SNr, including enrichment of Atp2a3/SERCA3, an ER calcium pump previously undescribed in mDA neurons.</p>', 'posted': '2021-08-01'}, {'doi': '10.1101/2021.07.25.453727', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'Miniature inverted-repeat transposable elements drive rapid microRNA diversification in angiosperms', 'authors': \"[{'name': 'Zhonglong Guo'}, {'name': 'Zheng Kuang'}, {'name': 'Yihan Tao'}, {'name': 'Haotian Wang'}, {'name': 'Miaomiao Wan'}, {'name': 'Chen Hao'}, {'name': 'Fei Shen'}, {'name': 'Xiaozeng Yang'}, {'name': 'Lei Li'}]\", 'abstract': '<p>MicroRNAs (miRNAs) are rapidly evolving endogenous small RNAs programing organism function and behavior. Although models for miRNA origination have been proposed based on sporadic cases, the genomic mechanisms driving swift diversification of the miRNA repertoires in plants remain elusive. Here, by comprehensively analyzing 20 phylogenetically representative plant species, we identified miniature inverted-repeat transposable elements (MITEs) as the predominant genomic sources for de novo miRNAs in angiosperms. Our data illustrated a transposition-transcription process whereby properly sized MITEs transposed into active genic regions could be converted into new miRNAs, termed MITE-miRNAs, in as few as 20 generations. We showed that this molecular domestication mechanism leads to a possible evolutionary arms race between the MITEs and the host genomes that rapidly and continuously changes the miRNA repertoires. We found that the MITE-miRNAs are selected for targeting genes associated with plant adaptation and habitat expansion, thereby constituting a genomic innovation potentially underlying angiosperm megadiversity.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453271', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Understanding associative false memories in aging using multivariate analyses', 'authors': \"[{'name': 'Nancy A. Dennis'}, {'name': 'Amy A. Overman'}, {'name': 'Catherine M. Carpenter'}, {'name': 'Courtney R. Gerver'}]\", 'abstract': '<p>Associative memory declines in aging arise, in part, from age-related increases in false memories to recombined lures. Studies have shown that there is a benefit to associative memory if the configural context of associative pairs is maintained from encoding to retrieval. The current study aimed to examine whether this benefit of contextual congruency is reduced in aging, and whether the neural similarity of memory representations between targets and lures underlies age- related increases in false memories. Behaviorally, both age groups benefited from target pairs presented in a visual format that was congruent with how the pair was learned. While no age difference was observed in hits, the typical age-related increase in false alarms was found. Congruent with behavioral results, neither the relationship between target-related patterns of neural activity across memory phases (as measured by ERS) nor the discriminability of target classification as a function of condition at retrieval (as measured by MVPA). However, with regard to false memories, older adults exhibited overall lower pattern similarity for hits and FAs compared to hits and CRs (as measured by RSA). Additionally, Hit-FA RSA correlated with age- related increases in associative FAs across visual, frontal, and parietal cortices. Results suggest that while neural processes supporting associative memory retrieval are dependent on configural congruency between encoding and retrieval, there is no difference as to how congruency affects these processes in aging. Additionally, similarity of target and lure processing may reflect reduced diagnosticity of information processing in aging.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.29.454302', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Small extracellular vesicles from young mice prevent frailty and improve healthspan in old mice', 'authors': \"[{'name': 'Jorge Sanz-Ros'}, {'name': 'Cristina Mas-Bargues'}, {'name': 'Daniel Monlen'}, {'name': 'Mar Dromant'}, {'name': 'Aksinya Derevyanko'}, {'name': 'Ana Guo-Carrin'}, {'name': 'Aurora Romn-Domnguez'}, {'name': 'Nekane Romero-Garca'}, {'name': 'Marta Ingls'}, {'name': 'Mara A. Blasco'}, {'name': 'Jose Via'}, {'name': 'Consuelo Borrs'}]\", 'abstract': '<p>Aging is associated with an increased risk of frailty, disability, comorbidities, institutionalization, falls, fractures, hospitalization, and mortality. Searching for strategies to delay the degenerative changes associated with aging and frailty is interesting. We treated old animals intravenously with small extracellular vesicles (sEVs) derived from adipose mesenchymal stem cells (ADSCs) of young animals, and we found an improvement of several functional parameters usually altered with aging, such as motor coordination, grip strength, fatigue resistance, fur regeneration, and renal function. Frailty index analysis showed that 40% of old control mice were frail, whereas none of the old ADSCs-sEVs treated mice were. Molecular and structural benefits in muscle and kidney accompanied this functional improvement. ADSCs-sEVs induced pro-regenerative effects and a decrease in oxidative stress, inflammation, and senescence markers. The metabolome of treated mice changed to a youth-like pattern. Finally, we gained some insight into the miRNAs contained in sEVs that might be, at least in part, responsible for the effects observed. We propose that young sEVs treatment can be beneficial against frailty and therefore can promote healthy aging.</p>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.07.26.453355', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Inhibition of striatal SEZ6 by miR-3594-5p is a drug-specific marker for late-stage heroin intake escalation', 'authors': \"[{'name': 'Magalie Lenoir'}, {'name': 'Isabella Bondi'}, {'name': 'Loc Clemenceau'}, {'name': 'Isabelle Nondier'}, {'name': 'Margaux Ball'}, {'name': 'Sbastien Jacques'}, {'name': 'Angline Duch'}, {'name': 'Corinne Canestrelli'}, {'name': 'Sverine Martin-Lannere'}, {'name': 'Sophie Mouillet-Richard'}, {'name': 'Jenny M. Gunnersen'}, {'name': 'Serge H. Ahmed'}, {'name': 'Nicolas Marie'}, {'name': 'Florence Noble'}]\", 'abstract': '<p>Escalation of drug use is a hallmark stage in the transition to addiction and uncovering its underlying brain molecular mechanisms constitutes a considerable challenge. Here, we report in rats with extended access to heroin for self-administration that miR-3594-5p was upregulated in the dorsal striatum at late, but not early, stages during escalation when excessive heroin intake plateaued. Striatal miR-3594-5p bound directly to the 3UTR region of <italic>Sez6</italic> transcript and inhibited its expression, thereby decreasing the mature form of the translated SEZ6 protein. This miR-3594-5p/<italic>Sez6</italic> interaction was specific to heroin, as it was not observed with cocaine, and correlated with the severity of heroin intake escalation. Our findings reveal that miRNA alterations during escalation of drug self-administration are spatially and temporally regulated and drug-specific.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453739', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Connectivity between the hippocampus and default mode network during the relief  but not elicitation  of curiosity supports curiosity-enhanced memory enhancements', 'authors': \"[{'name': 'Charlotte Murphy'}, {'name': 'Charan Ranganath'}, {'name': 'Matthias J. Gruber'}]\", 'abstract': '<p>Consistent with the idea that curiosity enhances information seeking, it has been shown that activity within both the dopaminergic circuit and hippocampus supports curiosity-enhanced learning. However, the role of whole-brain mechanisms involved in cognitive control (fronto-parietal network; FPN) and memory integration (default mode network; DMN) that might underpin curiosity states and their effects on memory remain elusive. We hypothesised that the FPN and DMN should distinguish between high- and low-curiosity conditions and be recruited more heavily for later remembered information associated with high-curiosity. Here, we used functional magnetic resonance imaging whilst participants completed a trivia paradigm, in which we presented trivia questions associated with high- and low-curiosity, followed by the associated answer. After a short delay, we tested memory for trivia answers. We adopted a network-based parcellation of the brain into subnetworks of the FPN and DMN to examine how neural activity within, and functional connectivity between, each subnetwork predicts curiosity-enhanced memory. Across elicitation and relief of curiosity, we found focal recruitment of FPNA and widespread recruitment of DMN subnetworks in support of curiosity and curiosity-enhanced memory. Most importantly, during the elicitation of curiosity, functional subcortical connectivity and across cortical networks, but not subcortical-cortical coupling, correlated with curiosity-enhanced memory. However, during the relief of curiosity, coupling between subcortical regions and DMNA emerged in support of curiosity-enhanced memory. Taken together, our results provide the first evidence about how neuromodulatory mechanisms via the hippocampal-dopaminergic circuit trigger states of curiosity and thereby communicate to higher-order cortical regions to facilitate curiosity-enhanced memory.</p><sec><title>Significant statement</title><p>Does neural activity within, and functional connectivity between, the dopaminergic-hippocampal network, fronto-parietal network (FPN), and default mode network (DMN) underpin curiosity states and their effects on memory? Here, we show how the dopaminergic system together with the hippocampus interact specifically with subnetwork DMNA potentially reflecting how subcortical regions support the enhancement of memory intergration of semantic information associated with curiosity. As DMNA (the core DMN subnetwork) was also functionally coupled with the whole DMN network and the semantic control network (FPNA), these findings provide a plausible neuromodulatory mechanism through which hippocampal-dopaminergic input triggers curiosity and then communicates to higher-order brain regions via DMNA to facilitate curiosity-enhanced memory.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453449', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Heat-inactivated modified vaccinia virus Ankara boosts Th1-biased cellular and humoral immune responses as a vaccine adjuvant by activating the STING-mediated cytosolic DNA-sensing pathway', 'authors': \"[{'name': 'Ning Yang'}, {'name': 'Aitor Garzia'}, {'name': 'Cindy Meyer'}, {'name': 'Thomas Tuschl'}, {'name': 'Taha Merghoub'}, {'name': 'Jedd D. Wolchok'}, {'name': 'Liang Deng'}]\", 'abstract': '<sec><title>Background</title><p>Protein or peptide-based subunit vaccines are promising platforms for combating human cancers and infectious diseases. However, one primary concern regarding subunit vaccines is the relatively weak immune responses induced by proteins or peptides. Therefore, developing novel and effective vaccine adjuvants is critical for the success of subunit vaccines. Modified vaccinia virus (MVA) is a safe and effective vaccine against smallpox and monkeypox. In this study, we explored the potential of heat-inactivated MVA (heat-iMVA) as a novel vaccine adjuvant.</p></sec><sec><title>Methods</title><p>We co-administered heat-iMVA with a model antigen, chicken ovalbumin (OVA), either intramuscularly or subcutaneously twice, two weeks apart, and analyzed anti-OVA specific CD8<sup>+</sup> and CD4<sup>+</sup> T cells in the spleens and skin draining lymph nodes (dLNs) and serum anti-OVA IgG1 and IgG2c antibodies. We also compared the adjuvanticity of heat-iMVA with several known vaccine adjuvants, including complete Freunds adjuvant (CFA) and AddaVax, an MF59-like preclinical grade nano-emulsion. In addition, we tested whether co-administration of heat-iMVA plus tumor neoantigen peptides or irradiated tumor cells improves antitumor efficacy in a B16-F10 therapeutic vaccination model. Using Stimulator of Interferon Genes (STING) or Batf3-deficient mice, we evaluated the contribution of the STING pathway and Batf3-dependent CD103<sup>+</sup>/CD8 DCs in heat-iMVA-induced immunity.</p></sec><sec><title>Results</title><p>Co-administration of protein- or peptide-based immunogens with heat-iMVA dramatically enhances Th1-biased cellular and humoral immune responses. This adjuvant effect of heat-iMVA is dependent on the STING-mediated cytosolic DNA-sensing pathway, and the antigen-specific CD8<sup>+</sup> T cell response requires Batf3-dependent CD103<sup>+</sup>/CD8<sup>+</sup> dendritic cells (DCs). Heat-iMVA infection of bone marrow-derived DCs (BMDCs) promoted antigen cross-presentation, whereas live MVA infection did not. RNA-seq analyses revealed that heat-iMVA is a more potent activator of the STING pathway than live MVA. Additionally, combining tumor neoantigen peptides or irradiated tumor cells with heat-iMVA delayed tumor growth and extended the median survival in B16-F10 therapeutic vaccination models.</p></sec><sec><title>Conclusions</title><p>Heat-iMVA induces type I interferon (IFN) production and antigen cross-presentation via a STING-dependent mechanism in DCs. Co-administration of heat-iMVA with peptide antigen generates strong Th1-biased cellular and humoral immunity. Collectively, our results demonstrate that heat-iMVA is a safe and potent vaccine adjuvant.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453750', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'EMDA: A Python package for Electron Microscopy Data Analysis', 'authors': \"[{'name': 'Rangana Warshamanage'}, {'name': 'Keitaro Yamashita'}, {'name': 'Garib N. Murshudov'}]\", 'abstract': '<p>An open-source Python library EMDA for cryo-EM map and model manipulation is presented with a specific focus on validation. The architecture of the library and its programmable and command-line interfaces are presented. The use of several functionalities in the library is presented through several examples. The utility of local correlation as a metric for identifying model errors and unmodeled regions in maps, and how it is used as a metric of map-model validation is demonstrated. The mapping of local correlation to individual atoms, and its use to draw insights on local signal variations are discussed. EMDAs likelihood-based map overlay is demonstrated by carrying out a superposition of two domains in two related structures. The overlay is carried out globally to bring both maps into the same coordinate frame and then locally to estimate the relative movement of domains. Finally, the map magnification refinement in EMDA is presented with an example to highlight the importance of correcting map magnification in structural comparison studies.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453751', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Quantitative Profiling of Adaptation to Cyclin E Overproduction', 'authors': \"[{'name': 'Juanita C. Limas'}, {'name': 'Amiee N. Littlejohn'}, {'name': 'Amy M. House'}, {'name': 'Katarzyna M. Kedziora'}, {'name': 'Dalia Fleifel'}, {'name': 'Brandon L. Mouery'}, {'name': 'Andrea Walens'}, {'name': 'Maria M. Aleman'}, {'name': 'Daniel Dominguez'}, {'name': 'Jeanette Gowen Cook'}]\", 'abstract': '<p>Cyclin E/CDK2 drives cell cycle progression from G1 to S phase. Cyclin E overproduction is toxic to mammalian cells, although the gene encoding cyclin E (<italic>CCNE1</italic>) is overexpressed in some cancers. It is not yet understood how cancer cells tolerate high levels of cyclin E. To address this question, we extensively characterized non-transformed epithelial cells subjected to chronic cyclin E overproduction. Cells overproducing human cyclin E briefly experienced truncated G1 phases, then consistently endured a transient period of DNA replication origin underlicensing, replication stress, and severely impaired proliferation. Individual cells displayed substantial intercellular heterogeneity in both cell cycle dynamics and CDK activity. Each of these phenotypes improved rapidly despite maintaining high cyclin E-associated activity. Transcriptome analysis revealed that adapted cells downregulated a cohort of G1-regulated genes. These cells also shared at least one unique change also found in breast tumors that overproduce cyclin E, expression of the cancer/testis antigen <italic>HORMAD1</italic>. Withdrawing cyclin E induction partially reversed the intermediate licensing phenotype of adapted cells indicating that adaptation is at least partly independent of genetic alterations. This study provides evidence that mammalian cyclin E/CDK inhibits origin licensing by an indirect mechanism through premature S phase onset. It serves as an example of specific oncogene adaptation that can identify key molecular changes during tumorigenesis.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.21261065', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'COVID-Q: validation of the first COVID-19 questionnaire based on patient-rated symptom gravity', 'authors': \"[{'name': 'G Spinato'}, {'name': 'C Fabbris'}, {'name': 'F Conte'}, {'name': 'A Menegaldo'}, {'name': 'L Franz'}, {'name': 'P Gaudioso'}, {'name': 'F Cinetto'}, {'name': 'C Agostini'}, {'name': 'G Costantini'}, {'name': 'P Boscolo-Rizzo'}]\", 'abstract': '<sec><title>Objectives</title><p>The aim of the present study was to develop and validate the CoronaVirus Disease  2019 (COVID-19) Questionnaire (COVID-Q), a novel symptom questionnaire specific for COVID-19 patients, to provide a comprehensive evaluation which may be helpful for physicians.. A secondary goal of the present study was to evaluate the performance of the COVID-Q in identifying subjects at higher risk of being tested positive for COVID-19.</p></sec><sec><title>Material and methods</title><p>Consecutive non-hospitalized adults who underwent nasopharyngeal and throat swab for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) detection at Treviso Hospital in March 2020, were enrolled. Subjects were divided into positive (cases) and negative (controls) in equal number. All of them gave consent and answered the COVID-Q. Patients not able to answer the COVID-Q due to clinical conditions were excluded.</p><p>Parallel Analysis and Principal Component Analysis were used to identify clusters of items measuring the same dimension. The Item Response Theory (IRT)-based analyses evaluated the functioning of item categories, the presence of clusters of local dependence among items, item fit within the model and model fit to the data.</p></sec><sec><title>Results</title><p>Answers obtained from 230 COVID-19 cases (113 males, and 117 females; mean age 55 years, range 20-99 years) and 230 controls (61 males, and 169 females; mean age 46 years, range 21-89) were analyzed. Parallel analysis led to the extraction of six components, which corresponded to as many clinical presentation patterns: asthenia, influenza-like symptoms, ear and nose symptoms, breathing issues, throat symptoms, and anosmia/ageusia. The final IRT models retained 27 items as significant for symptom assessment. The total score on the questionnaire was significantly associated with positivity to the molecular SARS-CoV-2 test: subjects with multiple symptoms were significantly more likely to be affected by COVID-19 (p &lt; .001). Older age and male gender also represented risk factors. Presence of breathing issues and anosmia/ageusia were significantly related to positivity to SARS-CoV-2 (p &lt; 0.001). None of the examined comorbidities had a significant association with COVID-19 diagnosis.</p></sec><sec><title>Conclusion</title><p>According to the analyses, COVID-Q could be validated since the aspects it evaluated were overall significantly related to SARS-CoV-2 infection. The application of the novel COVID-Q to everyday clinical practice may help identifying subjects who are likely to be affected by COVID-19 and address them to a nasopharyngeal swab in order to achieve an early diagnosis.</p></sec><sec><title>What is already known about this topic?</title><p>COVID-19 symptoms are widely known. Lots of studies have been published regarding self-administered questionnaires in order to characterize and know as much as possible regarding this disease. By the way, no specific questionnaires have been validated, yet, and there is no consensus regarding this topic.</p></sec><sec><title>What does this article add?</title><p>This paper shows the COVID-Q, a novel symptom questionnaire specific for COVID-19 patients. The aim is to provide a comprehensive evaluation that may be helpful to clinicians in order to suspect SARS-CoV-2 infection or not.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453582', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Taxonomic repositioning of twelve Xanthomonas campestris, seven Xanthomonas axonopodis and one Pseudomonas cissicola reference pathovars to Xanthomonas citri', 'authors': \"[{'name': 'Kanika Bansal'}, {'name': 'Sanjeet Kumar'}, {'name': 'Prabhu B. Patil'}]\", 'abstract': '<p>Here on the basis of phylo-taxonogenomics criteria we present amended description of 20 pathovars to <italic>Xanthomonas citri</italic> majority (18/20) of which were first reported from India. 7/20 are currently classified as <italic>X. axonopodis</italic>, 12/20 as <italic>X. campestris</italic> and 1/20 as <italic>Pseudomonas cissicola</italic>. Here, we have generated genome sequence data for 4 pathovars and genomes of remaining 16 were used from the published data. Genome based investigation reveals that all these pathovars belong to <italic>X. citri</italic> and not to <italic>X. axonopodis</italic> or <italic>X. campestris</italic> as previously reported. Present proposal is to aid in resolving the taxonomic confusion of the <italic>X. ctiri</italic> pathovars and prevent future usage of invalid names.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453649', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Important Docking study of Structure of Plasmodium falciparum thioredoxin reductase-thioredoxin complex: the case of potent drug Telatinib, a small molecule angiogenesis inhibitor', 'authors': \"[{'name': 'Ivan Vito Ferrari'}]\", 'abstract': '<sec><title>Background</title><p>Over the last decades, malaria parasites have been rapidly developing resistance against antimalarial drugs, which underlines the need for novel drug targets. Thioredoxin reductase (TrxR) is crucially involved in redox homeostasis and essential for Plasmodium falciparum. In this communication, we report first time important Docking study by in Silico approach, using AutoDock Vina. After a selective analysis of over 300 drugs, processed with Pyrx (a Virtual Screening software into the active site of protein (ID PDB 4J56 Thioredoxin reductase 2 Chain A), we noticed excellent value of Binding Energy of Telatinib estimated by Pyrx software. These results are comparable to the crystallized ligand FAD (FLAVIN-ADENINE DINUCLEOTIDE) completed in the above-mentioned protein. Indeed, from the results of Autodock Vina, Telatinib an inhibitor of tyrosine kinases, has excellent a Binding affinity value, ca. 12 kcal/mol.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.30.454541', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Intraspecific interaction of host plant influences local distribution of specialist herbivores through metabolic alterations in leaves', 'authors': \"[{'name': 'Haruna Ohsaki'}, {'name': 'Atsuko Miyagi'}, {'name': 'Maki Kawai-Yamada'}, {'name': 'Akira Yamawo'}]\", 'abstract': '<p>Recent studies suggest that changes in leaf traits due to interactions between plants affect the resource utilisation and distribution of herbivores. However, this has not yet been confirmed experimentally. Here, we investigated the effects of phenotypic plasticity in leaf traits of <italic>Rumex obtusifolius</italic> (host plant) in response to the intra and interspecific interaction on distribution of two leaf beetles, <italic>Gastrophysa atrocyanea</italic> (specialist herbivore) and <italic>Galerucella grisescens</italic> (generalist herbivore). We investigated the local population density of <italic>R. obtusifolius</italic> plants and the presence of leaf beetles on the plants at five study sites. Leaf chemicals (condensed tannins and total phenolics) were compared between aggregated and solitary <italic>R. obtusifolius</italic> plants. To clarify the effects of the interaction environment of <italic>R. obtusifolius</italic> plants on their leaf traits and resource utilisation by leaf beetles, we conducted cultivation and preference experiments. Leaf chemicals (chlorophylls, organic acids, primary metabolites, condensed tannins and total phenolics) and preferences of adult leaf beetles were compared between intraspecific, interspecific plant interaction, or no-interaction treatments. Finally, we evaluated the effects of interaction between <italic>R. obtusifolius</italic> on leaf beetle distribution in mesocosm experiments. In the field, the presence of the specialist leaf beetle, <italic>G. atrocyanea</italic>, was positively correlated with the local population density (rosette overlap ratio) of <italic>R. obtusifolius</italic> plants; however, no correlation was observed in the case of the generalist leaf beetle, <italic>G. grisescens</italic>. In the cultivation experiment, plants in the intraspecific interaction treatment increased their leaf contents of condensed tannins and total phenolics, and <italic>G. atrocyanea</italic> consumed more of these leaves than leaves in other treatments. Similar results were observed in the field. In the mesocosm experiment, larger numbers of <italic>G. atrocyanea</italic> were distributed on <italic>R. obtusifolius</italic> plants exposed to below-ground intraspecific interaction than on plants not exposed to intraspecific interaction. Our results provide experimental evidence that leaf trait changes in response to intraspecific interaction between host plants influence specialist herbivore distribution. This highlights the need to integrate plant plant interactions into our understanding of plant animal interactions.</p>', 'posted': '2021-07-31'}, {'doi': '10.1101/2021.07.26.21261146', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Occupational and Environmental Health', 'covid': True, 'title': 'Laboratory Study of Physical Barrier Efficiency for Worker Protection against SARS-CoV-2 while Standing or Sitting', 'authors': \"[{'name': 'Jacob Bartels'}, {'name': 'Cheryl Fairfield Estill'}, {'name': 'I-Chen Chen'}, {'name': 'Dylan Neu'}]\", 'abstract': '<p>Transparent barriers were installed as a response to the SARS-COV-2 pandemic in many customer-facing industries. Transparent barriers are an engineering control that are utilized to intercept air traveling between customers to workers. Information on the effectiveness of these barriers against aerosols is limited. In this study, a cough simulator was used to represent a cough from a customer. Two optical particle counters were used (one on each side of the barrier, labeled reference and worker) to determine the number of particles that migrated around a transparent barrier. Nine barrier sizes and a no barrier configuration were tested with six replicates each. Tests of these 10 configurations were conducted for both sitting and standing scenarios to represent configurations common to nail salons and grocery stores, respectively. Barrier efficiency was calculated using a ratio of the particle count results (reference/worker). Barriers had better efficiency when they were 9 to 39 cm (3.5 to 15.5) above cough height and at least 91 cm (36) wide, 92% and 93% respectively. Barriers that were 91 cm (36) above table height for both scenarios blocked 71% or more of the particles between 0.350.725 m and 68% for particles between 1 to 3 m. A barrier that blocked an initial cough was effective at reducing particle counts. While the width of barriers was not as significant as height in determining barrier efficiency it was important that a barrier be placed where interactions between customers and workers are most frequent.</p>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.07.27.453653', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Spores and extracellular matrix components impart molecular order in Bacillus subtilis biofilms', 'authors': \"[{'name': 'David N Azulay'}, {'name': 'Oliver Spker'}, {'name': 'Mnar Ghrayeb'}, {'name': 'Michaela Wilsch-Bruninger'}, {'name': 'Ernesto Scoppola'}, {'name': 'Manfred Burghammer'}, {'name': 'Ivo Zizak'}, {'name': 'Luca Bertinetti'}, {'name': 'Yael Politi'}, {'name': 'Liraz Chai'}]\", 'abstract': '<p>Biofilms are surface-associated soft microbial communities, which may be either detrimental or beneficial to their hosting environment. They develop from single cells into mature colonies, that are composed of cells and sometimes (in <italic>Firmicutes</italic> phylum) spores, held together by an extracellular matrix (ECM) of secreted biomolecular components. Biofilm development is a dynamic process, during which cells organize into subgroups, creating functionally distinct regions in space. Specific examples of functional-spatial division in <italic>Bacillus subtilis</italic> biofilms include matrix and spore formation as well as water channels that form beneath wrinkles. An interesting question arising is whether the division of labor in biofilms is also reflected in the molecular-level order across whole biofilms. Using combined X-ray diffraction (XRD)/X-ray fluorescence (XRF), we studied the molecular order in intact biofilms across multiple length scales. We discovered that biofilms display a distinct spatio-temporal XRD signature that depends on highly ordered structures in spores and on cross  sheet structures in matrix components. Spore signal is found especially enhanced with water molecules and metal-ions signals along macroscopic wrinkles, known to act as water channels. Demonstrating <italic>in situ</italic> the link between molecular structures, metal ions distribution and division of labor across whole biofilms in time and space, this study provides new pivotal insight to the understanding biofilm development.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.30.454402', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': True, 'title': 'Structures of SARS-CoV-2 B.1.351 neutralizing antibodies provide insights into cocktail design against concerning variants', 'authors': \"[{'name': 'Shuo Du'}, {'name': 'Pulan Liu'}, {'name': 'Zhiying Zhang'}, {'name': 'Tianhe Xiao'}, {'name': 'Ayijiang Yasimayi'}, {'name': 'Weijin Huang'}, {'name': 'Youchun Wang'}, {'name': 'Yunlong Richard Cao'}, {'name': 'Xiaoliang Sunney Xie'}, {'name': 'Junyu Xiao'}]\", 'abstract': '<p>The spread of the SARS-CoV-2 variants could seriously dampen the global effort to tackle the COVID-19 pandemic. Recently, we investigated the humoral antibody responses of SARS-CoV-2 convalescent patients and vaccinees towards circulating variants, and identified a panel of monoclonal antibodies (mAbs) that could efficiently neutralize the B.1.351 (Beta) variant. Here we investigate how these mAbs target the B.1.351 spike protein using cryo-electron microscopy. In particular, we show that two superpotent mAbs, BD-812 and BD-836, have non-overlapping epitopes on the receptor-binding domain (RBD) of spike. Both block the interaction between RBD and the ACE2 receptor; and importantly, both remain fully efficacious towards the B.1.617.1 (Kappa) and B.1.617.2 (Delta) variants. The BD-812/BD-836 pair could thus serve as an ideal antibody cocktail against the SARS-CoV-2 VOCs.</p>', 'posted': '2021-08-01'}, {'doi': '10.1101/2021.07.29.454288', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Patterns of morphological diversification in the Ramphastoidea reveal the dramatic divergence of toucans from a conserved morphotype', 'authors': \"[{'name': 'Krishnapriya Tamma'}, {'name': 'Anand Krishnan'}, {'name': 'Sushma Reddy'}]\", 'abstract': '<p>Morphological traits offer insights into an organism s ecological niche, species interactions, and patterns of community organisation. Pantropical lineages - animals and plants distributed across the three tropical continental regions of Asia, Africa, and America - provide a way to test how different environments and communities influence morphological diversification. Here, we examined a monophyletic group of frugivorous birds, the barbets and toucans (Ramphastoidea), which diversified independently on three continents, to investigate whether clades in each region exhibit similar (phylogenetically constrained) or distinct (ecologically influenced) patterns of morphological diversification. Our results show that despite differences in community dynamics in these regions, lineage accumulation patterns through time on all three continents are broadly similar, putatively due to phylogenetic niche conservatism. We quantified morphological variation in light of phylogenetic relatedness to further reveal that all barbet lineages across continents occupy a conserved region of morphospace after correcting for variation in size. However, in the Neotropics, one lineage, the toucans, have diverged dramatically from typical barbet space and converged toward (yet are distinct from) the trait space occupied by the distantly related hornbills in Asia and Africa. Additionally, we found no link between climatic variables and morphological traits. We conclude that barbets exhibit a conserved morphotype across continents and have diversified by scaling mainly in body size. However, the absence of other large frugivorous birds may have allowed toucans to diversify into a different region of morphospace of increased bill/wing to tail/wing ratios. By examining different continental lineages of a single monophyletic bird group, we shed light on the contrasting effects of regional ecological factors and phylogenetic constraints on morphological diversification.</p>', 'posted': '2021-07-30'}, {'doi': '10.1101/2021.07.26.453829', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Structure and function of the ROR2 cysteine-rich domain in vertebrate noncanonical WNT5A signaling', 'authors': \"[{'name': 'Samuel C. Griffiths'}, {'name': 'Jia Tan'}, {'name': 'Armin Wagner'}, {'name': 'Levi Blazer'}, {'name': 'Jarret J. Adams'}, {'name': 'Sachdev S. Sidhu'}, {'name': 'Christian Siebold'}, {'name': 'Hsin-Yi Henry Ho'}]\", 'abstract': '<p>The receptor tyrosine kinase ROR2 mediates noncanonical WNT5A signaling to orchestrate tissue morphogenetic processes, and dysfunction of the pathway causes Robinow syndrome, Brachydactyly B and metastatic diseases. The domain(s) and mechanisms required for ROR2 function, however, remain unclear. We solved the crystal structure of the extracellular cysteine-rich (CRD) and Kringle (Kr) domains of ROR2 and found that, unlike other CRDs, the ROR2 CRD lacks the signature hydrophobic pocket that binds lipids/lipid-modified proteins, such as WNTs, suggesting a novel mechanism of receptor action. Functionally, we showed that the ROR2 CRD, but not other domains, is required and minimally sufficient to promote WNT5A signaling, and Robinow mutations in the CRD and the adjacent Kr alter ROR2 function. Moreover, we demonstrated that the activity of the ROR2 CRD requires Frizzled receptors. Thus, ROR2 acts via its CRD to potentiate the function of a receptor supercomplex that includes Frizzleds to transduce WNT5A signals.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.27.454001', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Endorsement and Phylogenetic Analysis of some Fabaceae Plants based on DNA Barcoding gene MatK', 'authors': \"[{'name': 'Nader R. Abdelsalam'}, {'name': 'Mohamed E. Hasan'}, {'name': 'Samar M.A. Rabie'}, {'name': 'Houssam El-Din M.F. El-wakeel'}, {'name': 'Amera F. Zaitoun'}, {'name': 'Rehab Y. Ghareeb'}, {'name': 'Aly Z. Abdelsalam'}, {'name': 'Hesham M. Aly'}, {'name': 'Amira A. Ibrahim'}, {'name': 'Alaa A. Hemeida'}]\", 'abstract': '<p>DNA barcodes have been considered as a tool to facilitate species identification based on their simplicity and high-level accuracy compression to the complexity and subjective biases linked to morphological identification of taxa. MaturaseK gene   MatK  of the chloroplast is very crucial in the plant system which is involved in the group II intron splicing. The main objective of this current study is determining the relative utility of the   MatK  chloroplast gene for barcoding in fifteen legume trees by both single region and multiregional approaches. The chloroplast   MatK  gene sequences were submitted to GenBank and accession numbers (GenBank: <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC602060\">LC602060</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC602154\">LC602154</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC602263\">LC602263</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603347\">LC603347</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603655\">LC603655</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603845\">LC603845</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603846\">LC603846</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC603847\">LC603847</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC604717\">LC604717</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC604718\">LC604718</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC605994\">LC605994</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC604799\">LC604799</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC605995\">LC605995</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC606468\">LC606468</ext-link>, <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"gen\" xlink:href=\"LC606469\">LC606469</ext-link>) were obtained with sequence length ranging from 730 to 1545 nucleotides. These DNA sequences were aligned with database sequence using PROMALS server  ,  Clustal Omega server and Bioedit program. Also,\\xa0 the maximum likelihood and neighbor-joining algorithms for phylogenetic reconstruction using the MEGA-X program were employed. Overall, these results indicated that  the phylogenetic tree  analysis and the evolutionary distances of an individual dataset of each species were agreed with a phylogenetic tree of all each other consisting of two clades, the first clade comprising  (Enterolobium contortisiliquum, Albizia lebbek), Acacia saligna  ,  Leucaena leucocephala,  Dichrostachys Cinerea, (Delonix regia, Parkinsonia aculeata), (Senna surattensis, Cassia fistula, Cassia javanica)  and  Schotia brachypetala  were more closely to each other, respectively. The remaining four species of  Erythrina humeana, (Sophora secundiflora, Dalbergia Sissoo, Tipuana Tipu)  constituted the second clade. Therefore,  MatK  gene is considered promising a candidate for DNA barcoding in plant family Fabaceae and providing a clear relationship between the families. Moreover, their sequences could be successfully utilized in single nucleotide polymorphism (SNP) or part of the sequence as DNA fragment analysis utilizing polymerase chain reaction (PCR) in plant systematic.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.30.454479', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Visuo-vestibular gaze control - conserved subcortical processing', 'authors': \"[{'name': 'Tobias Wibble'}, {'name': 'Tony Pansell'}, {'name': 'Sten Grillner'}, {'name': 'Juan Perez-Fernandez'}]\", 'abstract': '<p>Gaze stabilization compensates for movements of the head or external environment to minimize image blurring, which is critical for visually-guided behaviors. Multisensory information is used to stabilize the visual scene on the retina via the vestibulo-ocular (VOR) and optokinetic (OKR) reflexes. While the organization of neuronal circuits underlying VOR is well described across vertebrates, less is known about the contribution and evolutionary origin of the OKR circuits. Moreover, the integration of these two sensory modalities is still poorly understood. Here, we developed a novel experimental model, the isolated lamprey eye-brain-labyrinth preparation, to analyze the neuronal pathways underlying visuo-vestibular integration which allowed electrophysiological recordings while applying vestibular stimulation using a moving platform, coordinated with visual stimulation via two screens. We show that lampreys exhibit robust visuo-vestibular integration, with optokinetic information processed in the pretectum and integrated with vestibular inputs at several subcortical levels. The enhanced eye movement response to multimodal stimulation favored the vestibular response at increased velocities. The optokinetic signals can be downregulated from tectum. Additionally, saccades are present in the form of nystagmus. The lamprey represents the oldest living group of vertebrates, thus all basic components of the visuo-vestibular control of gaze were present already at the dawn of vertebrate evolution.</p>', 'posted': '2021-08-01'}, {'doi': '10.1101/2021.07.25.452189', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'The first crested duck genome reveals clues to genetic compensation and crest cushion formation', 'authors': \"[{'name': 'Guobin Chang'}, {'name': 'Xiaoya Yuan'}, {'name': 'Qixin Guo'}, {'name': 'Hao Bai'}, {'name': 'Xiaofang Cao'}, {'name': 'Meng Liu'}, {'name': 'Zhixiu Wang'}, {'name': 'Bichun Li'}, {'name': 'Shasha Wang'}, {'name': 'Yong Jiang'}, {'name': 'Zhiquan Wang'}, {'name': 'Yang Zhang'}, {'name': 'Qi Xu'}, {'name': 'Qianqian Song'}, {'name': 'Rui Pan'}, {'name': 'Shenghan Zheng'}, {'name': 'Lingling Qiu'}, {'name': 'Tiantian Gu'}, {'name': 'Xinsheng Wu'}, {'name': 'Yulin Bi'}, {'name': 'Zhengfeng Cao'}, {'name': 'Yu Zhang'}, {'name': 'Yang Chen'}, {'name': 'Hong Li'}, {'name': 'Jianfeng Liu'}, {'name': 'Wangcheng Dai'}, {'name': 'Guohong Chen'}]\", 'abstract': '<p>The Chinese crested (CC) duck is a unique indigenous waterfowl breed with a phenotypic crest trait that affects its high survival rate. Therefore, the CC duck is an ideal model to investigate the genetic compensation response to maintain genetic stability. In the present study, we first generated a chromosome-level genome of CC ducks. Comparative genomics revealed genes related to tissue repair, immune function, and tumors were under strong positive selection, which suggested that these adaptive changes might enhance cancer resistance and immune response to maintain the genetic stability of CC ducks. We sub-assembled a Chinese spot-billed duck genome and detected genome-assembled structure variants among three ducks. Functional analysis revealed that a large number of structural variants were related to the immune system, which strongly suggests the occurrence of genetic compensation in the anti-tumor and immune systems to further support the survival of CC ducks. Moreover, we confirmed that the CC duck originated from the mallard ducks. Finally, we revealed the physiological and genetic basis of crest traits and identified a causative mutation in <italic>TAS2R40</italic> that leads to crest formation. Overall, the findings of this study provide new insights into the role of genetic compensation in adaptive evolution.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261025', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Cardiovascular Medicine', 'covid': True, 'title': 'Clinical echocardiography does not indicate cardiac dysfunction in critically ill Covid-19 patients', 'authors': \"[{'name': 'Henrik Isackson'}, {'name': 'Anders Larsson'}, {'name': 'Miklos Lipcsey'}, {'name': 'Robert Frithiof'}, {'name': 'Frank A. Flachskampf'}, {'name': 'Michael Hultstrm'}]\", 'abstract': '<sec><title>Objective</title><p>We aimed to investigate the acute effects of severe SARS-CoV-2 on myocardial function.</p></sec><sec><title>Methods</title><p>This is an observational study generated from the first 79 patients admitted to intensive care in Uppsala due to respiratory failure with SARS-CoV-2 infection, during the first wave in 2020, included in the PRONMED study. From this group 34 underwent echocardiographic examination of which 25 were included in the study, and compared to 44 non-echo patients. Demographic analysis compared standard parameters and previous morbidities between the echo and non-echo group.</p></sec><sec><title>Results</title><p>Standard echocardiographic parameters were analysed indicating a reduced left ventricular function as assessed by global longitudinal strain and very discrete increases in wall thickness in the group as a whole. A group comparison between the outcomes survival and death was carried out. Right sided dimensions and functional parameters did not indicate major strain. An increased maximum tricuspid valve regurgitation velocity indicating increased pulmonary arterial pressure was significantly associated with death, but failed to maintain significance when corrected for multiple comparison. Biochemical cardiac markers and D-dimer correlated to initiation of echocardiography and mortality. Tricuspid regurgitation maximum velocity was positively correlated with maximum troponin I.</p></sec><sec><title>Conclusions</title><p>These results suggests that there is no clear negative effect on cardiac function in critical SARS-CoV-2 infection. There are indications that pulmonary pressure elevation carries a negative predictive outcome suggesting pulmonary disease as the major driver of mortality. Cardiac biomarkers as well as D-dimer carry a predictive outcome value.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21260986', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': False, 'title': 'A plasma metabolite score of three eicosanoids predicts incident type 2 diabetes  a prospective study in three independent cohorts', 'authors': \"[{'name': 'Karolina Tuomisto'}, {'name': 'Joonatan Palmu'}, {'name': 'Tao Long'}, {'name': 'Jeramie D. Watrous'}, {'name': 'Kysha Mercader'}, {'name': 'Kim A. Lagerborg'}, {'name': 'Allen Andres'}, {'name': 'Marko Salmi'}, {'name': 'Sirpa Jalkanen'}, {'name': 'Ramachandran S. Vasan'}, {'name': 'Michael Inouye'}, {'name': 'Aki S. Havulinna'}, {'name': 'Pekka Jousilahti'}, {'name': 'Teemu J. Niiranen'}, {'name': 'Susan Cheng'}, {'name': 'Mohit Jain'}, {'name': 'Veikko Salomaa'}]\", 'abstract': '<sec><title>Aim</title><p>Peptide markers of inflammation have been associated with the development of type 2 diabetes. The role of upstream, lipid-derived mediators of inflammation such as eicosanoids, remains less clear. The aim was to examine whether eicosanoids are associated with incident type 2 diabetes.</p></sec><sec><title>Methods</title><p>In the FINRISK 2002 study, a population-based sample of Finnish men and women aged 25-74 years, we used directed, non-targeted liquid chromatography  mass spectrometry to identify 545 eicosanoids and related oxylipins in the participants plasma samples (n=8,292). We used multivariable-adjusted Cox regression to examine associations between eicosanoids and incident type 2 diabetes. The findings were replicated in the Framingham Heart Study (FHS, n=2,886) and DILGOM 2007 (n=3,905). Together, these three cohorts had 1070 cases of incident type 2 diabetes.</p></sec><sec><title>Results</title><p>76 eicosanoids were associated individually with incident type 2 diabetes. We identified three eicosanoids independently associated with incident type 2 diabetes using stepwise Cox regression with forward selection and a Bonferroni-corrected inclusion threshold. A three-eicosanoid risk score produced a hazard ratio (HR) of 1.56 (95% confidence interval 1.41-1.72) per one standard deviation (SD) increment for risk of incident diabetes. The HR for comparing the top quartile to the lowest was 2.80 (2.53-3.07). Meta-analysis of the three cohorts yielded a pooled HR per SD of 1.31 (1.05-1.56).</p></sec><sec><title>Conclusion</title><p>Plasma eicosanoid profiles predict incident type 2 diabetes and the clearest signals replicate in three independent cohorts. Our findings give new information on the biology underlying type 2 diabetes and suggest opportunities for early identification of people at risk.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453293', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Age- and sex-related topological organisation of human brain functional networks and their relationship to cognition', 'authors': \"[{'name': 'Heidi Foo'}, {'name': 'Anbupalam Thalamuthu'}, {'name': 'Jiyang Jiang'}, {'name': 'Forrest Koch'}, {'name': 'Karen A. Mather'}, {'name': 'Wei Wen'}, {'name': 'Perminder S. Sachdev'}]\", 'abstract': '<sec><title>BACKGROUND</title><p>Age and sex associated with changes in functional brain network topology and cognition in large population of older adults have been poorly understood. We explored this question further by examining differences in 11 resting-state graph theory measures with respect to age, sex, and their relationships with cognitive performance in 17,127 UK Biobank participants (mean=62.837.41 years).</p></sec><sec><title>METHODS</title><p>Brain connectivity toolbox was used to derive the graph theory measures that assessed network integration, segregation, and strength. Multiple linear regression was performed the relationship between age, sex, cognition, and network measures. Subsequently, multivariate analysis was done to further examine the joint effect of the network measures on cognitive functions.</p></sec><sec><title>RESULTS</title><p>Age was associated with an overall decrease in the effectiveness of network communication (i.e. integration) and loss of functional specialisation (i.e. segregation) of specific brain regions. Sex differences were also observed, with women showing more efficient networks which were less segregated than in men (FDR adjusted <italic>p</italic>&lt;.05). Age-related changes were also more apparent in men than women, which suggests that men may be more vulnerable to cognitive decline with age. Interestingly, while network segregation and strength of limbic network were only nominally associated with cognitive performance, the network measures collectively were significantly associated with cognition (FDR adjusted <italic>p</italic>.002). This may imply that individual measures may be inadequate to capture much of the variance in neural activity or its output and need further refinement.</p></sec><sec><title>CONCLUSION</title><p>The complexity of the functional brain organisation may be shaped by an individuals age and sex, which ultimately may influence cognitive performance of older adults. Age and sex stratification may be used to inform clinical neuroscience research to identify older adults at risk of cognitive dysfunction.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.27.453123', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Electrophysiological measures of visual suppression', 'authors': \"[{'name': 'Daniel H. Baker'}, {'name': 'Greta Vilidaite'}, {'name': 'Alex R. Wade'}]\", 'abstract': '<p>In the early visual system, suppression occurs between neurons representing different stimulus properties. This includes features such as orientation (cross-orientation suppression), eye-of-origin (interocular suppression) and spatial location (surround suppression), which are thought to involve distinct anatomical pathways. We asked if these separate routes to suppression can be differentiated by their pattern of gain control on the contrast response function measured in human participants using steady-state electroencephalography. Changes in contrast gain shift the contrast response function laterally, whereas changes in response gain scale the function vertically. We used a Bayesian hierarchical model to summarise the evidence for each type of gain control. A computational meta-analysis of 16 previous studies found the most evidence for contrast gain effects with overlaid masks, but no clear evidence favouring either response gain or contrast gain for other mask types. We then conducted two new experiments, comparing suppression from four mask types (monocular and dichoptic overlay masks, and aligned and orthogonal surround masks) on responses to sine wave grating patches flickering at 5Hz. At the occipital pole, there was strong evidence for contrast gain effects in all four mask types at the first harmonic frequency (5Hz). Suppression generally became stronger at more lateral electrode sites, but there was little evidence of response gain effects. At the second harmonic frequency (10Hz) suppression was stronger overall, and involved both contrast and response gain effects. Although suppression from different mask types involves distinct anatomical pathways, gain control processes appear to serve a common purpose, which we suggest might be to suppress less reliable inputs.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.24.453650', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Systems Biology', 'covid': False, 'title': 'A network-based approach reveals the dysregulated transcriptional regulation in non-alcohol fatty liver disease', 'authors': \"[{'name': 'Hong Yang'}, {'name': 'Muhammad Arif'}, {'name': 'Meng Yuan'}, {'name': 'Xiangyu Li'}, {'name': 'Ko Eun Shong'}, {'name': 'Hasan Turkez'}, {'name': 'Jens Nielsen'}, {'name': 'Mathias Uhln'}, {'name': 'Jan Born'}, {'name': 'Zhang Cheng'}, {'name': 'Adil Mardinoglu'}]\", 'abstract': '<p>Non-alcohol-related fatty liver disease (NAFLD) is a leading cause of chronic liver disease worldwide. We performed network analysis to investigate the dysregulated biological processes in the disease progression and revealed the molecular mechanism underlying NAFLD. Based on network analysis, we identified a highly conserved disease-associated gene module across three different NAFLD cohorts and highlighted the predominant role of key transcriptional regulators associated with lipid and cholesterol metabolism. In addition, we revealed the detailed metabolic differences between heterogenous NAFLD patients through integrative systems analysis of transcriptomic data and liver-specific genome-scale metabolic model. Furthermore, we identified transcription factors (TFs), including SREBF2, HNF4A, SREBF1, YY1 and KLF13, showing regulation of hepatic expression of genes in the NAFLD-associated modules and validated the TFs using data generated from a mouse NAFLD model. In conclusion, our integrative analysis facilitated our understanding of the regulatory mechanism of these perturbed TFs and associated biological processes.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453736', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Towards an integrative multi-omics workflow', 'authors': \"[{'name': 'Florian Jeanneret'}, {'name': 'Stphane Gazut'}]\", 'abstract': '<p>The advent of high-throughput techniques has greatly enhanced biological discovery. Last years, analysis of multi-omics data has taken the front seat to improve physiological understanding. Handling functional enrichment results from various biological data raises practical questions.</p><p>We propose an integrative workflow to better interpret biological process insights in a multi-omics approach applied to breast cancer data from The Cancer Genome Atlas (TCGA) related to Invasive Ductal Carcinoma (IDC) and Invasive Lobular Carcinoma (ILC). Pathway enrichment by Over Representation Analysis (ORA) and Gene Set Enrichment Analysis (GSEA) has been conducted with both features information from differential expression analysis or selected features from multi-block sPLS-DA methods. Then, comprehensive comparisons of enrichment results have been carried out by looking at classical enrichment analysis, probabilities pooling by Stouffers Z scores method and pathways clustering in biological themes.</p><p>Our work shows that ORA enrichment with selected sPLS-DA features and pathways probabilities pooling by Stouffers method lead to enrichment maps highly associated to physiological knowledge of IDC or ILC phenotypes, better than ORA and GSEA with differential expression driven features.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.21261046', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Infectious Diseases (except HIV/AIDS)', 'covid': True, 'title': 'Respiratory Viral Sequencing Panel identifies SARS-CoV-2 variants, transmission and other co-circulating viruses in Georgia, USA: A Diagnostic and Epidemiologic Tool for Mass Surveillance in COVID-19 Pandemic', 'authors': \"[{'name': 'Nikhil S Sahajpal'}, {'name': 'Ashis K Mondal'}, {'name': 'Allan Njau'}, {'name': 'Zachary Petty'}, {'name': 'Jiani Chen'}, {'name': 'Sudha Ananth'}, {'name': 'Pankaj Ahluwalia'}, {'name': 'Colin Williams'}, {'name': 'Ted M Ross'}, {'name': 'Alka Chaubey'}, {'name': 'Grace DeSantis'}, {'name': 'Gary P. Schroth'}, {'name': 'Justin Bahl'}, {'name': 'Ravindra Kolhe'}]\", 'abstract': '<sec><title>Background</title><p>In the current phase of COVID-19 pandemic, we are facing two serious public health challenges that include deficits in SARS-CoV-2 variant monitoring, and neglect of other co-circulating respiratory viruses. Additionally, accurate assessment of the evolution, extent and dynamics of the outbreak are required to understand the transmission of the virus amongst seemingly unrelated cases and provide critical epidemiological information. To address these challenges, we evaluated a new high-throughput next-generation sequencing (NGS), respiratory viral panel (RVP) that includes 40 viral pathogens with the aim of analyzing viral subtypes, mutational variants of SARS-CoV-2, model to understand the spread of the virus in the state of Georgia, USA, and to assess other circulating viruses in the same population.</p></sec><sec><title>Methods</title><p>This study evaluated a total of 522 samples that included 483 patient samples and 42 synthetic positive control material. The performance metrics were calculated for both clinical and reference control samples by comparing detection results with the RT-PCR assay. The limit of detection (LoD) studies were conducted as per the FDA guidelines. Inference and visualization of the phylogeny of the SARS-CoV-2 sequences were performed through the Nextstrain Command-Line Interface (CLI) tool, utilizing the associated augur and auspice toolkits.</p></sec><sec><title>Results</title><p>The performance metrics calculated using both the clinical samples and the reference controls revealed a PPA, NPA and accuracy of 95.98%, 85.96% and 94.4%, respectively. The LoD was determined to be 10 copies/ml with all 25 replicates detected across two different runs. The clade for pangolin lineage B that contains certain distant variants, including P4715L in ORF1ab, Q57H in ORF 3a and, S84L in ORF8 covarying with the D614G spike protein mutation were the most prevalent, early in the pandemic, in Georgia, USA. In our analysis, isolates from the same county formed paraphyletic groups, which indicated virus transmission between counties.</p></sec><sec><title>Conclusion</title><p>The study demonstrates the clinical and public health utility of the NGS-RVP to identify novel variants that can provide actionable information to prevent or mitigate emerging viral threats, models that provide insights into viral transmission patterns and predict transmission/ resurgence of regional outbreaks and provide critical information on co-circulating respiratory viruses that might be independent factors contributing to the global disease burden.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.28.454217', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Multi-tier mechanics control stromal adaptations in swelling lymph nodes', 'authors': \"[{'name': 'Frank P. Assen'}, {'name': 'Miroslav Hons'}, {'name': 'Robert Hauschild'}, {'name': 'Shayan Shamipour'}, {'name': 'Jun Abe'}, {'name': 'Walter A. Kaufmann'}, {'name': 'Tommaso Costanzo'}, {'name': 'Gabriel Krens'}, {'name': 'Markus Brown'}, {'name': 'Burkhard Ludewig'}, {'name': 'Simon Hippenmeyer'}, {'name': 'Jens V. Stein'}, {'name': 'Carl-Philipp Heisenberg'}, {'name': 'Edouard Hannezo'}, {'name': 'Sanjiv A. Luther'}, {'name': 'Michael Sixt'}]\", 'abstract': '<p>Lymph nodes (LNs) comprise two main structural elements: Fibroblastic reticular cells (FRCs) that form dedicated niches for immune cell interaction and capsular fibroblasts that build a shell around the organ. While LNs are fairly stable in size during homeostatic conditions, immunological challenge causes more than 10-fold increase in size within only a few days. How a solid organ can accommodate such extreme volumetric changes is poorly understood. Here, we characterize the biomechanics of LN swelling on the cellular and organ scale. We identify lymphocyte trapping by influx and proliferation as drivers of an outward pressure force, causing FRCs and their associated conduits to stretch. After an initial phase of relaxation, FRCs sense the resulting strain via cell matrix adhesions, which coordinates local growth and remodeling of the stromal network. While the expanded FRC network adopts its typical configuration, a massive fibrotic reaction of the organ capsule sets in and counters further organ expansion. Thus, different fibroblast populations mechanically control LN swelling in a multi-tier fashion.</p>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.07.29.454261', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': True, 'title': 'COVID-ONE-humoral immune: The One-stop Database for COVID-19-specific Antibody Responses and Clinical Parameters', 'authors': \"[{'name': 'Zhaowei Xu'}, {'name': 'Yang Li'}, {'name': 'Qing Lei'}, {'name': 'Likun Huang'}, {'name': 'Dan-yun Lai'}, {'name': 'Shu-juan Guo'}, {'name': 'He-wei Jiang'}, {'name': 'Hongyan Hou'}, {'name': 'Yun-xiao Zheng'}, {'name': 'Xue-ning Wang'}, {'name': 'Jiaoxiang Wu'}, {'name': 'Ming-liang Ma'}, {'name': 'Bo Zhang'}, {'name': 'Hong Chen'}, {'name': 'Caizheng Yu'}, {'name': 'Jun-biao Xue'}, {'name': 'Hai-nan Zhang'}, {'name': 'Huan Qi'}, {'name': 'Siqi Yu'}, {'name': 'Mingxi Lin'}, {'name': 'Yandi Zhang'}, {'name': 'Xiaosong Lin'}, {'name': 'Zongjie Yao'}, {'name': 'Huiming Sheng'}, {'name': 'Ziyong Sun'}, {'name': 'Feng Wang'}, {'name': 'Xionglin Fan'}, {'name': 'Sheng-ce Tao'}]\", 'abstract': '<p>Coronavirus disease 2019 (COVID-19), which is caused by SARS-CoV-2, varies with regard to symptoms and mortality rates among populations. Humoral immunity plays critical roles in SARS-CoV-2 infection and recovery from COVID-19. However, differences in immune responses and clinical features among COVID-19 patients remain largely unknown. Here, we report a database for COVID-19-specific IgG/IgM immune responses and clinical parameters (COVID-ONE humoral immune). COVID-ONE humoral immunity is based on a dataset that contains the IgG/IgM responses to 21 of 28 known SARS-CoV-2 proteins and 197 spike protein peptides against 2,360 COVID-19 samples collected from 783 patients. In addition, 96 clinical parameters for the 2,360 samples and information for the 783 patients are integrated into the database. Furthermore, COVID-ONE humoral immune provides a dashboard for defining samples and a one-click analysis pipeline for a single group or paired groups. A set of samples of interest is easily defined by adjusting the scale bars of a variety of parameters. After the START button is clicked, one can readily obtain a comprehensive analysis report for further interpretation. COVID-ONE-humoral immune is freely available at www.COVID-ONE.cn.</p>', 'posted': '2021-07-30'}, {'doi': '10.1101/2021.07.29.454286', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Structural basis for the therapeutic advantage of dual and triple agonists at the human GIP, GLP-1 or GCG receptors', 'authors': \"[{'name': 'Fenghui Zhao'}, {'name': 'Qingtong Zhou'}, {'name': 'Zhaotong Cong'}, {'name': 'Kaini Hang'}, {'name': 'Xinyu Zou'}, {'name': 'Chao Zhang'}, {'name': 'Yan Chen'}, {'name': 'Antao Dai'}, {'name': 'Anyi Liang'}, {'name': 'Qianqian Ming'}, {'name': 'Mu Wang'}, {'name': 'Linan Chen'}, {'name': 'Peiyu Xu'}, {'name': 'Rulue Chang'}, {'name': 'Wenbo Feng'}, {'name': 'Tian Xia'}, {'name': 'Yan Zhang'}, {'name': 'Beili Wu'}, {'name': 'Dehua Yang'}, {'name': 'Lihua Zhao'}, {'name': 'H. Eric Xu'}, {'name': 'Ming-Wei Wang'}]\", 'abstract': '<p>Glucose homeostasis, regulated by glucose-dependent insulinotropic polypeptide (GIP), glucagon-like peptide-1 (GLP-1) and glucagon (GCG) is critical to human health. Several multi-targeting agonists at GIPR, GLP-1R or GCGR, developed to maximize metabolic benefits with reduced side-effects, are in clinical trials to treat type 2 diabetes and obesity. To elucidate the molecular mechanisms by which tirzepatide, a GIPR/GLP-1R dualagonist, and peptide 20, a GIPR/GLP-1R/GCGR triagonist, manifest their superior efficacies over monoagonist such as semaglutide, we determined cryo-electron microscopy structures of tirzepatide-bound GIPR and GLP-1R as well as peptide 20-bound GIPR, GLP-1R and GCGR The structures reveal both common and unique features for the dual and triple agonism by illustrating key interactions of clinical relevance at the atomic level. Retention of glucagon function is required to achieve such an advantage over GLP-1 monotherapy. Our findings provide valuable insights into the structural basis of functional versatility and therapeutic supremacy of tirzepatide and peptide 20.</p>', 'posted': '2021-07-30'}, {'doi': '10.1101/2021.07.30.454490', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Roscovitine exacerbates Mycobacterium abscessus infection by reducing NADPH oxidase-dependent neutrophil trafficking', 'authors': \"[{'name': 'Vincent Le Moigne'}, {'name': 'Daniela Rodriguez Rincon'}, {'name': 'Simon Glatigny'}, {'name': 'Christian Dupont'}, {'name': 'Christelle Langevin'}, {'name': 'Amel Ait Ali Said'}, {'name': 'Stephen A Renshaw'}, {'name': 'Andres R Floto'}, {'name': 'Jean-Louis Herrmann'}, {'name': 'Audrey Bernut'}]\", 'abstract': '<p>Persistent neutrophilic inflammation associated with chronic pulmonary infection causes progressive lung injury and eventually death in individuals with cystic fibrosis (CF), a genetic disease caused by bi-allelic mutations in the cystic fibrosis transmembrane conductance regulator (CFTR) gene. We therefore examined whether Roscovitine, a cyclin-dependent kinase inhibitor that (in other conditions) reduces inflammation while promoting host defence, might provide a beneficial effect in the context of CF. Herein, using CFTR-depleted zebrafish larvae as an innovative vertebrate model of CF immuno-pathophysiology, combined with murine and human approaches, we sought to determine the effects of Roscovitine on innate immune responses to tissue injury and pathogens in CF condition. We show that Roscovitine exerts anti-inflammatory and pro-resolution effects in neutrophilic inflammation induced by infection or tail amputation in zebrafish. Roscovitine reduces overactive epithelial ROS-mediated neutrophil trafficking, by reducing DUOX2/NADPH-oxidase activity, and accelerates inflammation resolution by inducing neutrophil apoptosis and reverse migration. Importantly, while Roscovitine efficiently enhances intracellular bacterial killing of Mycobacterium abscessus in human CF macrophages ex vivo, we found that treatment with Roscovitine results in worse infection in mouse and zebrafish models. By interfering with DUOX2/NADPH oxidase-dependent ROS production, Roscovitine reduces the number of neutrophils at infection sites, and consequently compromises granuloma formation and maintenance, favouring extracellular multiplication of M. abscessus and more severe infection. Our findings bring important new understanding of the immune-targeted action of Roscovitine and have significant therapeutic implications for safety targeting inflammation in CF.</p>', 'posted': '2021-07-30'}, {'doi': '10.1101/2021.07.23.453552', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Emotional states as distinct configurations of functional brain networks', 'authors': \"[{'name': 'Rotem Dan'}, {'name': 'Marta Weinstock'}, {'name': 'Gadi Goelman'}]\", 'abstract': '<p>The conceptualization of emotional states as patterns of interactions between large-scale brain networks has recently gained support. Yet, few studies have directly examined the brains network structure during emotional experiences. Here, we investigated the brains functional network organization during experiences of sadness, amusement, and neutral states elicited by movies, in addition to a resting-state. We tested the effects of the experienced emotion on individual variability in the brains functional connectome. Next, for each state, we defined a community structure of the brain and quantified its segregation and integration. Our results show that emotional states increase the similarity between and within individuals in the brains functional connectome. Second, in the brains modular organization, sadness, relative to amusement, was associated with higher integration and increased connectivity of cognitive control networks: the salience and fronto-parietal networks. Importantly, the modular metrics of brain segregation and integration were related to the subjective emotional valence. Last, in both the functional connectome and emotional report, we found higher similarity among women. Our results suggest that the experience of emotion is linked to a reconfiguration of whole-brain distributed, not emotion-specific, functional brain networks and that the brains topological structure carries information about the subjective emotional experience.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453762', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Effect of stimulation time on the expression of human macrophage polarization markers', 'authors': \"[{'name': 'Duygu Unuvar Purcu'}, {'name': 'Asli Korkmaz'}, {'name': 'Sinem Gunalp'}, {'name': 'Derya Goksu Helvaci'}, {'name': 'Yonca Erdal'}, {'name': 'Yavuz Dogan'}, {'name': 'Gerhard Wingender'}, {'name': 'Duygu Sag'}]\", 'abstract': '<p>Macrophages are highly plastic cells that can polarize into functionally distinct subsets <italic>in vivo</italic> and <italic>in vitro</italic> in response to environmental signals. The development of protocols to model macrophage polarization <italic>in vitro</italic> greatly contributes to our understanding of macrophage biology. Macrophages are divided into two main groups: Pro-inflammatory M1 macrophages (classically activated) and anti-inflammatory M2 macrophages (alternatively activated), based on several key surface markers and the production of inflammatory mediators. However, the expression of these common macrophage polarization markers is greatly affected by the stimulation time used. Unfortunately, there is no consensus yet regarding the optimal stimulation times for particular macrophage polarization markers in <italic>in vitro</italic> experiments. This situation is problematic, (i) as analysing a particular marker at a suboptimal time point can lead to false-negative results, and (ii) as it clearly impedes the comparison of different studies. Using human monocyte-derived macrophages (MDMs) <italic>in vitro</italic>, we analysed how the expression of the main polarization markers for M1 (CD64, CD86, CXCL9, CXCL10, HLA-DR, IDO1, IL1, IL12p70, TNF), M2a (CD200R, CD206, CCL17, CCL22, IL-10, TGM2), and M2c (CD163, IL-10, TGF) macrophages changes over time at mRNA and protein levels. Our data establish the most appropriate stimulation time for the analysis of the expression of human macrophage polarization markers <italic>in vitro</italic>. Providing such a reference guide will likely facilitate the investigation of macrophage polarization and its reproducibility.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453290', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PCRMS: a database of predicted cis-regulatory modules and constituent transcription factor binding sites in genomes', 'authors': \"[{'name': 'Pengyu Ni'}, {'name': 'Zhengchang Su'}]\", 'abstract': '<p>More accurate and more complete predictions of <italic>cis</italic>-regulatory modules (CRMs) and constituent transcriptional factor (TF) binding sites (TFBSs) in genomes can facilitate characterizing functions of regulatory sequences. Here, we developed a database PCRMS (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://cci-bioinfo.uncc.edu\">https://cci-bioinfo.uncc.edu</ext-link>) that stores highly accurate and unprecedentedly complete maps of predicted CRMs and TFBSs in the human and mouse genomes. The web interface allows the user to browse CRMs and TFBSs in an organism, find the closest CRMs to a gene, search CRMs around a gene, and find all TFBSs of a TF. PCRMS can be a useful resource for the research community to characterize regulatory genomes.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.28.454169', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'In situ cryo-electron tomography reveals local cellular machineries for axon branch development', 'authors': \"[{'name': 'Hana Nedozralova'}, {'name': 'Nirakar Basnet'}, {'name': 'Iosune Ibiricu'}, {'name': 'Satish Bodakuntla'}, {'name': 'Christian Biertmpfel'}, {'name': 'Naoko Mizuno'}]\", 'abstract': '<p>Neurons are highly polarized cells forming an intricate network of dendrites and axons. They are shaped by the dynamic reorganization of cytoskeleton components and cellular organelles. Axon branching allows to form new paths and increases circuit complexity. However, our understanding of branch formation is sparse due to technical limitations. Using <italic>in situ</italic> cellular cryo-electron tomography on primary mouse neurons, we directly visualized the remodeling of organelles and cytoskeleton structures at axon branches. Strikingly, branched areas functioned as hotspots concentrating organelles to support dynamic activities. Unaligned actin filaments assembled at the base of premature branches and remained while filopodia diminished. Microtubules and ER co-migrated into preformed branches to support outgrowth together with accumulating compact ~500 nm mitochondria and locally clustered ribosomes. We obtained a roadmap of events and present the first direct evidence of local protein synthesis selectively taking place at axon branches, allowing to serve as unique control hubs for axon development and downstream neural network formation.</p>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.07.29.454308', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'HGFAC is a ChREBP Regulated Hepatokine that Enhances Glucose and Lipid Homeostasis', 'authors': \"[{'name': 'Ashot Sargsyan'}, {'name': 'Ludivine Doridot'}, {'name': 'Sarah A. Hannou'}, {'name': 'Wenxin Tong'}, {'name': 'Harini Srinivasan'}, {'name': 'Rachael Ivison'}, {'name': 'Ruby Monn'}, {'name': 'Henry H. Kou'}, {'name': 'Jonathan M. Haldeman'}, {'name': 'Michelle Arlotto'}, {'name': 'Phillip J. White'}, {'name': 'Paul A. Grimsrud'}, {'name': 'Inna Astapova'}, {'name': 'Linus Tsai'}, {'name': 'Mark A. Herman'}]\", 'abstract': '<p>Carbohydrate Responsive Element-Binding Protein (ChREBP) is a carbohydrate sensing transcription factor that regulates both adaptive and maladaptive genomic responses in coordination of systemic fuel homeostasis. Genetic variants in the ChREBP locus associate with diverse metabolic traits in humans, including circulating lipids. To identify novel ChREBP-regulated hepatokines that contribute to its systemic metabolic effects, we integrated ChREBP ChIP-seq analysis in mouse liver with human genetic and genomic data for lipid traits and identified Hepatocyte Growth Factor Activator (HGFAC) as a promising ChREBP-regulated candidate in mice and humans. HGFAC is a protease that activates the pleiotropic hormone Hepatocyte Growth Factor (HGF). We demonstrate that HGFAC KO mice have phenotypes concordant with putative loss-of-function variants in human HGFAC. Moreover, in gain- and loss-of-function genetic mouse models, we demonstrate that HGFAC enhances lipid and glucose homeostasis, in part, through actions to activate hepatic PPAR activity. Together, our studies show that ChREBP mediates an adaptive response to overnutrition via activation of an HGFAC-HGF-PPAR signaling axis in the liver to preserve glucose and lipid homeostasis.</p>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.07.28.453895', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Emotional context sculpts action goal representations in the lateral frontal pole', 'authors': \"[{'name': 'RC Lapate'}, {'name': 'IC Ballard'}, {'name': 'MK Heckner'}, {'name': 'M DEsposito'}]\", 'abstract': '<p>Emotional states provide an ever-present source of contextual information that should inform behavioral goals. Despite the ubiquity of emotional signals in our environment, the neural mechanisms underlying their influence on goal-directed action remains unclear. Prior work suggests that the lateral frontal pole (FPl) is uniquely positioned to integrate affective information into cognitive control representations. We used pattern similarity analysis to examine the content of representations in FPl and interconnected mid-lateral prefrontal and amygdala circuitry. Healthy participants (n=37; n=21 females) were scanned while undergoing an event-related Affective Go/No-Go task, which requires goal-oriented action selection during emotional processing. We found that FPl contained conjunctive emotion-action goal representations that were related to successful cognitive control during emotional processing. These representations differed from conjunctive emotion-action goal representations found in the basolateral amygdala. While robust action goal representations were present in mid-lateral prefrontal cortex, they were not modulated by emotional valence. Finally, converging results from functional connectivity and multivoxel pattern analyses indicated that FPls emotional valence signals likely originated from interconnected subgenual ACC (BA25), which was in turn functionally coupled with the amygdala. Thus, our results identify a key pathway by which internal emotional states influence goal-directed behavior.</p><sec><title>Significance statement</title><p>Optimal functioning in everyday life requires behavioral regulation that flexibly adapts to dynamically changing emotional states. However, precisely how emotional states influence goal-directed action remains unclear. Unveiling the neural architecture that supports emotion-goal integration is critical for our understanding of disorders such as psychopathy, which is characterized by deficits in incorporating emotional cues into goals, as well as mood and anxiety disorders, which are characterized by impaired goal-based emotion regulation. Our study identifies a key circuit through which emotional states influence goal-directed behavior. This circuitry comprised the lateral frontal pole (FPl), which represented integrated emotion-goal information, as well as interconnected amygdala and subgenual ACC, which conveyed emotional signals to FPl.</p></sec>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.07.23.21261038', 'tweets': 5, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Epidemics of chikungunya, Zika, and COVID-19 reveal bias in case-based mapping', 'authors': \"[{'name': 'Fausto Andres Bustos Carrillo'}, {'name': 'Brenda Lopez Mercado'}, {'name': 'Jairo Carey Monterrey'}, {'name': 'Damaris Collado'}, {'name': 'Saira Saborio'}, {'name': 'Tatiana Miranda'}, {'name': 'Carlos Barilla'}, {'name': 'Sergio Ojeda'}, {'name': 'Nery Sanchez'}, {'name': 'Miguel Plazaola'}, {'name': 'Harold Suazo Laguna'}, {'name': 'Douglas Elizondo'}, {'name': 'Sonia Arguello'}, {'name': 'Anna M. Gajewski'}, {'name': 'Hannah E. Maier'}, {'name': 'Krista Latta'}, {'name': 'Bradley Carlson'}, {'name': 'Josefina Coloma'}, {'name': 'Leah Katzelnick'}, {'name': 'Hugh Sturrock'}, {'name': 'Angel Balmaseda'}, {'name': 'Guillermina Kuan'}, {'name': 'Aubree Gordon'}, {'name': 'Eva Harris'}]\", 'abstract': '<p>Explosive epidemics of chikungunya, Zika, and COVID-19 have recently occurred worldwide, all of which featured large proportions of subclinical infections. Spatial studies of infectious disease epidemics typically use symptomatic infections (cases) to estimate incidence rates (cases/total population), often misinterpreting them as infection risks (infections/total population) or disease risks (cases/infected population). We examined these three measures in a pediatric cohort (N3,000) over two chikungunya epidemics and one Zika epidemic and in a household cohort (N=1,793) over one COVID-19 epidemic in Nicaragua. Across different analyses and all epidemics, case incidence rates considerably underestimated both risk-based measures. Spatial infection risk differed from spatial disease risk, and typical case-only approaches precluded a full understanding of the spatial seroprevalence patterns. For epidemics of pathogens that cause many subclinical infections, relying on case-only datasets and misinterpreting incidence rates, as is common, results in substantial bias, a general finding applicable to many pathogens of high human concern.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.27.453915', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'HyperChIP for identifying hypervariable signals across ChIP/ATAC-seq samples', 'authors': \"[{'name': 'Haojie Chen'}, {'name': 'Shiqi Tu'}, {'name': 'Chongze Yuan'}, {'name': 'Feng Tian'}, {'name': 'Yijing Zhang'}, {'name': 'Yihua Sun'}, {'name': 'Zhen Shao'}]\", 'abstract': '<p>With the reduction in sequencing costs, studies become prevalent that profile the chromatin landscape for tens or even hundreds of human individuals by using ChIP/ATAC-seq techniques. Identifying genomic regions with hypervariable ChIP/ATAC-seq signals across given samples is essential for such studies. In particular, the hypervariable regions (HVRs) across tumors from different patients indicate their heterogeneity and can contribute to revealing potential cancer subtypes and the associated epigenetic markers. We present HyperChIP as the first complete statistical tool for the task. HyperChIP uses scaled variances that account for the mean-variance dependence to rank genomic regions, and it increases the statistical power by diminishing the influence of true HVRs on model fitting. Applying it to a large pan-cancer ATAC-seq data set, we found that the identified HVRs not only provided a solid basis to uncover the underlying similarity structure among the involved tumor samples, but also led to the identification of transcription factors pertaining to the similarity structure when coupled with a motif-scanning analysis.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.23.453227', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Ecology', 'covid': False, 'title': 'Disentangling key species interactions in diverse and heterogeneous communities: A Bayesian sparse modeling approach', 'authors': \"[{'name': 'Christopher P. Weiss-Lehman'}, {'name': 'Chhaya M. Werner'}, {'name': 'Catherine H. Bowler'}, {'name': 'Lauren M. Hallett'}, {'name': 'Margaret M. Mayfield'}, {'name': 'Oscar Godoy'}, {'name': 'Lina Aoyama'}, {'name': 'Gyrgy Barabs'}, {'name': 'Chengjin Chu'}, {'name': 'Emma Ladouceur'}, {'name': 'Loralee Larios'}, {'name': 'Lauren G. Shoemaker'}]\", 'abstract': '<label>1</label><title>Abstract</title><p>Modeling species interactions in diverse communities traditionally requires a prohibitively large number of species-interaction coefficients, especially when considering environmental dependence of parameters. We implemented Bayesian variable selection via sparsity-inducing priors on non-linear species abundance models to determine which species-interactions should be retained and which can be represented as an average heterospecific interaction term, reducing the number of model parameters. We evaluated model performance using simulated communities, computing out-of-sample predictive accuracy and parameter recovery across different input sample sizes. We applied our method to a diverse empirical community, allowing us to disentangle the direct role of environmental gradients on species intrinsic growth rates from indirect effects via competitive interactions. We also identified a few neighboring species from the diverse community that had non-generic interactions with our focal species. This sparse modeling approach facilitates exploration of species-interactions in diverse communities while maintaining a manageable number of parameters.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453710', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'A comprehensive collection of transcriptome data to support life cycle analysis of the poplar rust fungus Melampsora larici-populina', 'authors': \"[{'name': 'Pamela Guerillot'}, {'name': 'Emmanuelle Morin'}, {'name': 'Annegret Kohler'}, {'name': 'Clmentine Louet'}, {'name': 'Sbastien Duplessis'}]\", 'abstract': '<p><italic>Melampsora larici-populina</italic> is an obligate biotrophic plant pathogen responsible for the poplar rust disease. This fungus belongs to the taxonomical order Pucciniales and exhibits a complex heteroecious and macrocyclic life cycle, i.e. it has the capacity to infect two unrelated host plants, larch and poplar, and to form five distinct spore types through the year. The <italic>M. larici-populina</italic> genome has been sequenced and annotated in 2011 and since, different transcriptomic analyses were conducted at different stages with oligoarrays and later on with RNA-Seq covering most of its life cycle. Here, we collected published transcriptome data for the poplar rust fungus, transposed them for the version 2 of the genome now available at the Joint Genome Institute Mycocosm website and performed normalization of different oligoarray datasets on one hand, and of RNA-Seq on the other hand. We report a comprehensive normalized dataset for this fungus, made available for the community, which allows life cycle transcriptomics analysis.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453693', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Hoslundia opposita and other Nigerian plants inhibit sickle hemoglobin polymerization and prevent erythrocyte sickling: a virtual screening-guided identification of bioactive plants.', 'authors': \"[{'name': 'Eric O. Olori'}, {'name': 'Olujide Oludayo Olubiyi'}, {'name': 'Chinedum Peace Babalola'}]\", 'abstract': '<p>In sickle cell disease, a hereditary hemoglobinopathy, clinically observed disease presentations are the endpoint of a point mutation involving the substitution of glutamic acid with valine at the position 6 of the beta globin chain. With about 4.4 million people globally being affected, and another 43 million people bearing the sickle cell trait, several research efforts have been made to discover new and affordable treatment and possibly cure for the disease. Africa is endowed with a large flora population and traditional healers and citizens have over time depended on the use of herbs in folkloric medicine for different ailments including sickle cell disease (SCD). Such native knowledge has often formed the basis for different research exploration into the anti-sickling activities of selected African plants. These plants that have been so far investigated for their anti-sickling properties represent about 0.05 % of the 45,000 plant species enriching the flora landscape in Sub-Saharan Africa. Some of these have yielded potent anti-sickling profiles. In the current work we seek to achieve a more extensive search of the African plant diversity with anti-sickling properties: for this we have adopted a hybrid computational-cum-experimental protocol that employed computer-aided drug design (CADD) means for identifying plants with at least one constituent capable of interacting with the sickle hemoglobin, followed by extractive procedures and anti-sickling experiments for validating the predictions. Over two thousand (or 2,000) African natural products, representing over 200 plant species, were first virtually screened against the crystal structure of the dimerized human sickle hemoglobin. The natural products with the best computed sickle hemoglobin interaction energetics were found to belong to five plant species including Catharanthus roseus, Rauvolfia vomitoria, Hoslundia opposita, Lantana camara and Euphorbia hirta. The leaves of these plants were each collected subsequently and subjected to standard processing and extraction procedures. Using both HbSS polymerization inhibition and sickling reversal tests significant reductions in polymerization of erythrocyte hemolysate of the HbSS genotype were observed with the methanolic extracts of the plants, as well as sickling reversal levels of up to 68.50 % (H. opposita) was observed.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.452934', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Modelling time evolution of in blood alcohol concentration', 'authors': \"[{'name': 'Antoni Oliver Gelabert'}]\", 'abstract': '<p>A theoretical model based on mass conservation is proposed to predict blood alcohol concentration (BAC) over time. The proposed model takes as input the body parameters, the data timing for each drink, as well as his alcoholic strength and ingested amount. Three test cases with variations are presented and discussed. The results given are obtained by solving numerically the equation proposed. The solution predicts a well located alcohol peak and a reasonable time of alcohol body cleaning. This model represents a first order approach to a complex problem. Similar scenarios, like, for instance, drug concentrations can be modelled by using the same concept. The proposed model has been implemented on easy-to-use program COGORZA, which source code is available online at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/tonibois/Cogorza\">https://github.com/tonibois/Cogorza</ext-link>.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453662', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Population colonization history influences behavioral responses of European starlings in personality tests', 'authors': \"[{'name': 'Alexandra Rodriguez'}, {'name': 'Martine Hausberger'}, {'name': 'Patricia Le Quilliec'}, {'name': 'Philippe Clergeau'}, {'name': 'Laurence Henry'}]\", 'abstract': '<p>To understand the processes involved in biological invasions, the genetic, morphological, physiological and behavioral characteristics of invasive populations need to be understood.</p><p>Many invasive species have been reported to be flying species. In birds, both invaders and migrants encounter novel situations, therefore one could expect that both groups might react similarly to novelty.</p><p>Here we analyzed the behavioral responses of individuals from three populations of European starling <italic>Sturnus vulgaris</italic>: a population settled for centuries in a rural region, a population that recently colonized an urban area, and a population of winter migrant birds. We conducted a social isolation test, a novel environment test, a novel food test and a novel object test to explore their reactions towards novelty. We identified and characterized different behavioral profiles for each test.</p><p>The group of migratory adults appeared to be less anxious in social isolation than the group of urban young. Urban and migrant groups entered the novel environment sooner than rural birds. Shy, bold and intermediate individuals were observed in all three groups when presented with novel food. Finally, the proportion of shy individuals which did not touch the novel object was higher than the proportion of bold individuals in the rural group. Our study emphasizes that neophilia or boldness present in migrant and invasive populations may facilitate the occupation of novel habitats. Our analysis also suggests that mixed reactions of neophobia ensure behavioral flexibility in a gregarious invasive species.</p><sec><title>Significant statement</title><p>In this paper, we show that an invasive species like European starling, <italic>Sturnus vulgaris</italic>, presents an important flexibility in neophobia and in reaction towards social isolation. These variations depend on the settlement history of populationseven when the birds had been wild-caught as nestlings and hand-raised in standard conditions. This is significant because it highlights possible scenarios of colonization processes.</p><p>We believe that this manuscript is appropriate for publication by Behavioral Ecology and Sociobiology because it places individuals behavior in the core mechanisms of an ecological phenomenom as biological invasions. Our manuscript enlarges the paradigms related to the ways of coping with novelty in animals.</p><p>This manuscript has not been published and is not under consideration for publication elsewhere.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453636', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioengineering', 'covid': False, 'title': 'IRI-CCE: A Platform for Evaluating CRISPR-based Genome Editing Tools and Its Components', 'authors': \"[{'name': 'Rahul Mahadev Shelake'}, {'name': 'Dibyajyoti Pramanik'}, {'name': 'Jae-Yean Kim'}]\", 'abstract': '<p>Rapid assessment of Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-based tools and validation of cloned CRISPR-plasmid vectors are critical aspects for the successful application of genome editing (GE) in different organisms, including plants. Also, the selection of active guide RNAs (gRNAs) is a determining factor to achieve efficient GE at the targeted locus. Incidentally, plasmid vectors capable of ectopic expression of Cas enzyme and sgRNAs in bacteria will facilitate the quick screening and reliability of cloned plasmids and the functionality of designed gRNAs. We report a platform for <italic>in vivo</italic> rapid investigation of CRISPR components in <italic>Escherichia coli</italic> (IRI-CCE), which is compatible with plant-transforming binary vectors comprising plant promoters/terminators. Besides, IRI-CCE analyses reveal distinct features of cytidine (PmCDA1, evoCDA1, APOBEC3A) and adenine (ABE8e) base editors. Finally, we show the IRI-CCE platform as a reliable and rapid method for screening functional gRNAs to achieve successful GE outcomes. Base editor-based CRISPR components expressed by promoters of different strengths led to the establishment of IRI-CCE platform for three major applications: optimization of novel CRISPR tools, validation of cloned CRISPR plasmids, and to know gRNA functionality.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.28.454018', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'The transcriptional corepressor CTBP-1 acts with the SOX family transcription factor EGL-13 to maintain AIA interneuron cell identity in C. elegans', 'authors': \"[{'name': 'Josh Saul'}, {'name': 'Takashi Hirose'}, {'name': 'H. Robert Horvitz'}]\", 'abstract': '<p>Cell identity is characterized by a distinct combination of gene expression, cell morphology and cellular function established as progenitor cells divide and differentiate. Following establishment, cell identities can be unstable and require active and continuous maintenance throughout the remaining life of a cell. Mechanisms underlying the maintenance of cell identities are incompletely understood. Here we show that the gene <italic>ctbp-1,</italic> which encodes the transcriptional corepressor <underline>C</underline>-terminal <underline>b</underline>inding <underline>p</underline>rotein-1 (CTBP-1), is essential for the maintenance of the identities of the two AIA interneurons in the nematode <italic>Caenorhabditis elegans</italic>. <italic>ctbp-1</italic> is not required for the establishment of the AIA cell fate but rather functions cell-autonomously and can act in older worms to maintain proper AIA gene expression, morphology and function. From a screen for suppressors of the <italic>ctbp-1</italic> mutant phenotype, we identified the gene <italic>egl-13,</italic> which encodes a SOX family transcription factor. We found that <italic>egl-13</italic> regulates AIA function and aspects of AIA gene expression, but not AIA morphology. We conclude that the CTBP-1 protein maintains AIA cell identity in part by utilizing EGL-13 to repress transcriptional activity in the AIAs. More generally, we propose that transcriptional corepressors like CTBP-1 might be critical factors in the maintenance of cell identities, harnessing the DNA-binding specificity of transcription factors like EGL-13 to selectively regulate gene expression in a cell-specific manner.</p>', 'posted': '2021-07-28'}, {'doi': '10.1101/2021.07.31.454589', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'The human TRPA1 intrinsic cold and heat sensitivity involves separate channel structures beyond the N-ARD domain', 'authors': \"[{'name': 'Lavanya Moparthi'}, {'name': 'Viktor Sinica'}, {'name': 'Milos Filipovic'}, {'name': 'Viktorie Vlachova'}, {'name': 'Peter Michael Zygmunt'}]\", 'abstract': '<p>The human TRPA1 (hTRPA1) is an intrinsic thermosensitive ion channel responding to both cold and heat, depending on the redox environment. Here, we have studied purified hTRPA1 truncated proteins to gain further insight into the temperature gating of hTRPA1. We found in patch-clamp bilayer recordings that 1-688 hTRPA1, without the N-terminal ankyrin repeat domain (N-ARD), was more sensitive to cold and heat, whereas 1-854 hTRPA1 that is also lacking the S1-S4 voltage sensing-like domain (VSLD) gained sensitivity to cold but lost its heat sensitivity. The thiol reducing agent TCEP abolished the temperature sensitivity of both 1-688 hTRPA1 and 1-854 hTRPA1. Cold and heat activity of 1-688 hTRPA1 and 1-854 hTRPA1 were associated with different structural conformational changes as revealed by intrinsic tryptophan fluorescence measurements. Heat evoked major structural rearrangement of the VSLD as well as the C-terminus domain distal to the transmembrane pore domain S5-S6 (CTD), whereas cold only caused minor conformational changes. As shown for 1-854 hTRPA1, a sudden drop in tryptophan fluorescence occurred within 25-20C indicating a transition between heat and cold conformations of the CTD, and thus it is proposed that the CTD contains a bidirectional temperature switch priming hTRPA1 for either cold or heat. In whole-cell patch clamp electrophysiology experiments, replacement of the cysteines 865, 1021 and 1025 with alanine modulated the cold sensitivity of hTRPA1 when heterologously expressed in HEK293T cells. It is proposed that the hTRPA1 CTD harbors cold and heat sensitive domains allosterically coupled to the S5-S6 pore region and the VSLD, respectively.</p>', 'posted': '2021-08-01'}, {'doi': '10.1101/2021.07.23.453568', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Global, Low Amplitude Cortical State Predicts Response Outcomes in a Selective Detection Task', 'authors': \"[{'name': 'Krista Marrero'}, {'name': 'Krithiga Aruljothi'}, {'name': 'Behzad Zareian'}, {'name': 'Chengchun Gao'}, {'name': 'Zhaoran Zhang'}, {'name': 'Edward Zagha'}]\", 'abstract': '<p>Spontaneous neuronal activity strongly impacts stimulus encoding and behavioral responses. We sought to determine the effects of neocortical prestimulus activity on stimulus detection. We trained mice in a selective whisker detection task, in which they learned to respond (lick) to target stimuli in one whisker field and ignore distractor stimuli in the contralateral whisker field. During expert task performance, we used widefield Ca<sup>2+</sup> imaging to assess prestimulus and post-stimulus neuronal activity broadly across frontal and parietal cortices. We found that lower prestimulus activity correlated with enhanced stimulus detection: lower prestimulus activity predicted response versus no response outcomes and faster reaction times. The activity predictive of trial outcome was distributed through dorsal neocortex, rather than being restricted to whisker or licking regions. Using principal component analysis, we demonstrate that response trials are associated with a distinct and less variable prestimulus neuronal subspace. For single units, prestimulus choice probability was weak yet distributed broadly, with lower than chance choice probability correlating with stronger sensory and motor encoding. These findings support a low amplitude, low variability, optimal prestimulus cortical state for stimulus detection that presents globally and predicts response outcomes for both target and distractor stimuli.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.453511', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PatchPerPixMatch for Automated 3d Search of Neuronal Morphologies in Light Microscopy', 'authors': \"[{'name': 'Lisa Mais'}, {'name': 'Peter Hirsch'}, {'name': 'Claire Managan'}, {'name': 'Kaiyu Wang'}, {'name': 'Konrad Rokicki'}, {'name': 'Robert R. Svirskas'}, {'name': 'Barry J. Dickson'}, {'name': 'Wyatt Korff'}, {'name': 'Gerald M. Rubin'}, {'name': 'Gudrun Ihrke'}, {'name': 'Geoffrey W. Meissner'}, {'name': 'Dagmar Kainmueller'}]\", 'abstract': '<p>Studies of individual neurons in the Drosophila nervous system are facilitated by transgenic lines that sparsely and repeatably label respective neurons of interest. Sparsity can be enhanced by means of intersectional approaches like the split-GAL4 system, which labels the positive intersection of the expression patterns of two (denser) GAL4 lines. To this end, two GAL4 lines have to be identified as labelling a neuron of interest. Current approaches to tackling this task include visual inspection, as well as automated search in 2d projection images, of single cell multi-color flip-out (MCFO) acquisitions of GAL4 expression patterns. There is to date no automated method available that performs full 3d search in MCFO imagery of GAL4 lines, nor one that leverages automated reconstructions of the labelled neuron morphologies. To close this gap, we propose PatchPerPixMatch, a fully automated approach for finding a given neuron morphology in MCFO acquisitions of Gen1 GAL4 lines. PatchPerPixMatch performs automated instance segmentation of MCFO acquisitions, and subsequently searches for a target neuron morphology by minimizing an objective that aims at covering the target with a set of well-fitting segmentation fragments. Patch-PerPixMatch is computationally efficient albeit being full 3d, while also highly robust to inaccuracies in the automated neuron instance segmentation. We are releasing PatchPerPixMatch search results for ~30,000 neuron morphologies from the Drosophila <italic>hemibrain</italic> in ~20,000 MCFO acquisitions of ~3,500 Gen1 GAL4 lines.</p><sec><title>Code</title><p><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/Kainmueller-Lab/PatchPerPixMatch\">https://github.com/Kainmueller-Lab/PatchPerPixMatch</ext-link></p></sec><sec><title>Results</title><p><ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://pppm.janelia.org\">https://pppm.janelia.org</ext-link></p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453642', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Molecular Learning of a Soft-Disks Fluid', 'authors': \"[{'name': 'Luca Zammataro'}]\", 'abstract': '<p>This work is based on the Equivalence between Molecular Dynamics and Neural Network. It provides learning proofs in a Lennard-Jones (LJ) fluid, presented as a network of particles having non-bonded interactions. I describe the fluids learning as the property of an order that emerges as an adaptation in establishing equilibrium with energy and thermal conservation. The experimental section demonstrates the fluid can be trained with logic-gates patterns. The work goes beyond Molecular Computings application, explaining how this model uses its intrinsic minimizing properties in learning and predicting outputs. Finally, it gives hints for a theory on real chemistrys computational universality.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.30.454539', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Engineering adoptive T cell therapy to co-opt Fas ligand-mediated death signaling in ovarian cancer enhances therapeutic efficacy', 'authors': \"[{'name': 'Kristin G. Anderson'}, {'name': 'Shannon K. Oda'}, {'name': 'Breanna M. Bates'}, {'name': 'Madison G. Burnett'}, {'name': 'Magdalia Rodgers Suarez'}, {'name': 'Susan L. Ruskin'}, {'name': 'Philip D. Greenberg'}]\", 'abstract': '<p>Background: In the U.S., more than 50% of ovarian cancer patients die within 5 years of diagnosis, highlighting the need for innovations such as engineered T cell therapies. Mesothelin (Msln) is an attractive immunotherapy target for this cancer, as it is overexpressed by the tumor and contributes to malignant and invasive phenotypes, making antigen loss disadvantageous to the tumor. We previously showed that adoptively transferred T cells engineered to be Msln-specific (TCR1045) preferentially accumulate within established ovarian tumors, delay tumor growth and significantly prolong survival in the ID8VEGF mouse model. However, T cell persistence and anti-tumor activity were not sustained, and we and others have previously detected FasL in the tumor vasculature and the tumor microenvironment (TME) of human and murine ovarian cancers, which can induce apoptosis in infiltrating lymphocytes expressing Fas receptor (Fas). Methods: To concurrently overcome this mechanism for potential immune evasion and enhance T cell responses, we generated an immunomodulatory fusion protein (IFP) containing the Fas extracellular binding domain fused to a 4-1BB co-stimulatory domain, rather than the natural death domain. T cells engineered to express TCR1045 alone or in combination with the IFP were transferred into ID8VEGF-tumor bearing mice and evaluated for persistence, proliferation, anti-tumor cytokine production, and therapeutic efficacy. Results: Relative to T cells modified only to express TCR1045, T cells engineered to express both TCR1045 and a Fas IFP preferentially persisted in the TME of tumor-bearing mice due to improved T cell proliferation and survival. Moreover, adoptive immunotherapy with IFP+ T cells significantly prolonged survival in tumor-bearing mice, relative to TCR1045 T cells lacking the IFP. Conclusions: Fas/FasL signaling can mediate T cell death in the ovarian cancer microenvironment, as well as induce activation-induced cell death, an apoptotic mechanism responsible for regulating T cell expansion. Upregulation of FasL by tumor cells and tumor vasculature represents a mechanism for protecting growing tumors from attack by tumor-infiltrating lymphocytes. As many solid tumors overexpress FasL, an IFP that converts the Fas-mediated death signal into pro-survival and proliferative signals may provide an opportunity to enhance engineered adoptive T cell therapy against many malignancies.</p>', 'posted': '2021-07-31'}, {'doi': '10.1101/2021.07.28.454269', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'SUITOR: selecting the number of mutational signatures through cross-validation', 'authors': \"[{'name': 'Donghyuk Lee'}, {'name': 'Difei Wang'}, {'name': 'Xiaohong R. Yang'}, {'name': 'Jianxin Shi'}, {'name': 'Maria Teresa Landi'}, {'name': 'Bin Zhu'}]\", 'abstract': '<p>For <italic>de novo</italic> mutational signature analysis, the critical first step is to decide how many signatures should be expected in a cancer genomics study. An incorrect number could mislead downstream analyses. Here we present SUITOR (Selecting the nUmber of mutatIonal signaTures thrOugh cRoss-validation), an unsupervised cross-validation method that requires little assumptions and no numerical approximations to select the optimal number of signatures without overfitting the data. <italic>In vitro</italic> studies and <italic>in silico</italic> simulations demonstrated that SUITOR can correctly identify signatures, some of which were missed by other widely used methods. Applied to 2,540 whole-genome sequenced tumors across 22 cancer types, SUITOR selected signatures with the smallest prediction errors and almost all signatures of breast cancer selected by SUITOR were validated in an independent breast cancer study. SUITOR is a powerful tool to select the optimal number of mutational signatures, facilitating downstream analyses with etiological or therapeutic importance.</p>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.08.01.454622', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Synthetic Biology', 'covid': False, 'title': 'Data Storage Based on Combinatorial Synthesis of DNA Shortmers', 'authors': \"[{'name': 'Inbal Preuss'}, {'name': 'Zohar Yakhini'}, {'name': 'Leon Anavy'}]\", 'abstract': '<p>Storage needs represent a significant burden on the economy and the environment. Some of this can potentially be offset by improved density molecular storage. The potential of using DNA for storing data is immense. DNA can be harnessed as a high density, durable archiving medium for compressing and storing the exponentially growing quantities of digital data that mankind generates. Several studies have demonstrated the potential of DNA-based data storage systems. These include exploration of different encoding and error correction schemes and the use of different technologies for DNA synthesis and sequencing. Recently, the use of composite DNA letters has been demonstrated to leverage the inherent redundancy in DNA based storage systems to achieve higher logical density, offering a more cost-effective approach. However, the suggested composite DNA approach is still limited due to its sensitivity to the stochastic nature of the process. Combinatorial assembly methods were also suggested to encode information on DNA in high density, while avoding the challenges of the stochastic system. These are based on enzynatic assembly processes for producing the synthetic DNA. In this paper, we propose a novel method to encode information into DNA molecules using combinatorial encoding and shortmer DNA synthesis, in compatibility with current chemical DNA synthesis technologies. Our approach is based on a set of easily distinguishable DNA shortmers serving as building blocks and allowing for near-zero error rates. We construct an extended combinatorial alphabet in which every letter is a subset of the set of building blocks. We suggest different combinatorial encoding schemes and explore their theoretical properties and practical implications in terms of error probabilities and required sequencing depth. To demonstrate the feasibility of our approach, we implemented an end-to-end computer simulation of a DNA-based storage system, using our suggested combinatorial encodings. We use simulations to assess the performance of the system and the effect of different parameters. Our simulations suggest that our combinatorial approach can potentially achieve up to 6.5-fold increase in the logical density over standard DNA based storage systems, with near zero reconstruction error. Implementing our approach at scale to perform actual synthesis, requires minimal alterations to current technologies. Our work thus suggests that the combination of combinatorial encoding with standard DNA chemical synthesis technologies can potentially improve current solutions, achieving scalable, efficient and cost- effective DNA-based storage.</p>', 'posted': '2021-08-01'}, {'doi': '10.1101/2021.07.29.454399', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Neural-specific alterations in glycosphingolipid biosynthesis and cell signaling associated with two human ganglioside GM3 Synthase Deficiency variants', 'authors': \"[{'name': 'Michelle Dookwah'}, {'name': 'Shannon K Wagner'}, {'name': 'Mayumi Ishihara'}, {'name': 'Seok-ho Yu'}, {'name': 'Heidi Ulrichs'}, {'name': 'Michael J Kulik'}, {'name': 'Nadja Zeltner'}, {'name': 'Stephen Dalton'}, {'name': 'Kevin A Strauss'}, {'name': 'Kazuhiro Aoki'}, {'name': 'Richard Steet'}, {'name': 'Michael Tiemeyer'}]\", 'abstract': '<p>GM3 Synthase Deficiency (GM3SD) is a neurodevelopmental disorder resulting from pathogenic variants in the ST3GAL5 gene, which encodes GM3 synthase, a glycosphingolipid (GSL)-specific sialyltransferase. This enzyme adds a single alpha3-linked sialic acid to the terminal galactose of lactosylceramide (LacCer) to produce the monosialylated ganglioside GM3. In turn, GM3 is extended by other glycosyltransferases to generate nearly all the complex gangliosides enriched in neural tissue. Pathogenic mechanisms that account for neural phenotypes associated with GM3SD are not known. To explore how loss of GM3 impacts neural-specific glycolipid glycosylation and cell signaling, GM3SD patient fibroblasts bearing one of two different ST3GAL5 variants were reprogrammed to induced pluripotent stem cells (iPSCs) and then differentiated to neural crest cells (NCCs). GM3 and GM3-derived gangliosides were undetectable in iPSCs and NCCs from both variants, while LacCer precursor levels were elevated compared to wildtype (WT). NCCs of both variants synthesized elevated levels of neutral lacto- and globo-series, as well as minor alternatively sialylated, GSLs compared to WT. Shifts in ceramide profiles associated with iPSC and NCC GSLs were also detected in GM3SD variants. Altered GSL profiles in the GM3SD cells were accompanied by dynamic changes in the cell surface proteome, protein O-GlcNAcylation, and receptor tyrosine kinase abundance. GM3SD cells also exhibited increased apoptosis and sensitivity to erlotnib, an inhibitor of epidermal growth factor receptor signaling. Pharmacologic inhibition of O-GlcNAcase increased protein O-GlcNAcylation and significantly rescued baseline and erlotnib-induced apoptosis. Collectively, these findings indicate broad effects on cell signaling during differentiation of GM3SD patient-derived iPSCs to NCCs. Thus, human GM3SD cells provide a novel platform to investigate structure/function relationships that connect GSL diversity to cell signaling, cell survival, and neural differentiation.</p>', 'posted': '2021-07-30'}, {'doi': '10.1101/2021.07.22.453339', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Segmenting biological specimens from photos to understand the evolution of UV plumage in passerine birds', 'authors': \"[{'name': 'Yichen He'}, {'name': 'Christopher R. Cooney'}, {'name': 'Zo K. Varley'}, {'name': 'Lara O. Nouri'}, {'name': 'Christopher J. A. Moody'}, {'name': 'Michael D. Jardine'}, {'name': 'Steve Maddock'}, {'name': 'Gavin H. Thomas'}]\", 'abstract': '<p>Ultraviolet (UV) colouration is thought to be an important signalling mechanism in many bird species, yet broad insights regarding the prevalence of UV plumage colouration and the factors promoting its evolution are currently lacking. Here, we develop a novel image segmentation pipeline based on deep learning that considerably outperforms classical (i.e. non-deep learning) segmentation methods, and use this to extract accurate information on whole-body plumage colouration from photographs of &gt;24,000 museum specimens covering &gt;4,500 species of passerine birds. Our results demonstrate that UV reflectance, particularly as a component of other colours, is widespread across the passerine radiation but is strongly phylogenetically conserved. We also find clear evidence in support of the role of light environment in promoting the evolution of UV plumage colouration, and a weak trend towards higher UV plumage reflectance among bird species with ultraviolet rather than violet-sensitive visual systems. Overall, our study provides important broad-scale insight into an enigmatic component of avian colouration, as well as demonstrating that deep learning has considerable promise for allowing new data to be bought to bear on long-standing questions in ecology and evolution.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.453664', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Animal Behavior and Cognition', 'covid': False, 'title': 'Habituation, task solving and memorization may facilitate biological invasions: the Starling example', 'authors': \"[{'name': 'Alexandra Rodriguez'}, {'name': 'Martine Hausberger'}, {'name': 'Philippe Clergeau'}, {'name': 'Laurence Henry'}]\", 'abstract': '<p>Invasions ecology deals more and more with behavioural characteristics of invasive species. Particularly, research have focused on the personality of invaders and on their way of coping with novelty in new habitats. Traits of neophobia may limit individuals in their exploration of novel objects or the consumption of novel foods, they may stop the access to valuable ressources. Actually, in novel environments like cities, food can be unreachable in throwaway dishes with lids or hidden in the garbage or even close to frightening objects. Animals may either left the place and waste these resources, or they can express low neophobia from the beginning and manipulate the objects to reach food. They may also habituate progressively to the context and use the ressources.</p><p>Here we analyzed the behavioural responses of individuals from three populations of European starling <italic>Sturnus vulgaris</italic>: a population anciently settled in a rural region, a population that has recently colonized a urban area and a population of wintering migrant birds.</p><p>We used a series of tests in order to explore if individuals would habituate to a novel object and if they could remember it eight months later. We explored if individuals would be less neophobic when confronted to two novel objects successively and we tested them in a learning task involving a novel object and an attractive food. Our results show that <italic>Sturnus vulgaris</italic> habituates rapidly to novel objects and that categorization facilitates neophobia lost when confronted to two different novel objects. Initially young birds appeared to be more skilled than adults in the learning task. Individuals from this species seem to be able to remember an object durably. We suggest that habituation, task solving and memorization are three mechanisms enhancing biological invasions.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453858', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Crosstalk with keratinocytes causes GNAQ oncogene specificity in melanoma', 'authors': \"[{'name': 'Oscar Urtatiz'}, {'name': 'Amanda Haage'}, {'name': 'Guy Tanentzapf'}, {'name': 'Catherine D. Van Raamsdonk'}]\", 'abstract': '<p>Different melanoma subtypes exhibit specific and non-overlapping sets of oncogene and tumor suppressor mutations, despite a common cell of origin in melanocytes. For example, activation of the G<sub>q/11</sub> signaling pathway is a characteristic initiating event in primary melanomas that arise in the dermis, uveal tract or central nervous system. It is rare in melanomas arising in the epidermis. Here, we present evidence that in the mouse, crosstalk with the epidermal microenvironment actively impairs the survival of melanocytes expressing the GNAQ<sup>Q209L</sup> oncogene, providing a new model for oncogene specificity in cancer. The presence of epidermal cells inhibited cell division and fragmented dendrites of melanocytes expressing GNAQ<sup>Q209L</sup> in culture, while they promoted the growth of normal melanocytes. Differential gene expression analysis of FACS sorted epidermal melanocytes showed that cells expressing GNAQ<sup>Q209L</sup> exhibit an oxidative stress and apoptosis signature previously linked to vitiligo. Furthermore, PLCB4, the direct downstream effector of G<sub>q/11</sub> signaling, is frequently mutated in cutaneous melanoma alongside P53 and NF1. Our results suggest that a deficiency of PLCB4 promotes cutaneous melanomagenesis by reducing GNAQ driven signaling. Hence, our studies reveal the flip side of the GNAQ/PLCB4 signaling pathway, which was hitherto unsuspected. In the future, understanding how epidermal crosstalk restrains the GNAQ<sup>Q209L</sup> oncogene could suggest novel melanoma therapies.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453903', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': '3D Convolutional Neural Networks for Classification of Alzheimers and Parkinsons Disease with T1-Weighted Brain MRI', 'authors': \"[{'name': 'Nikhil J. Dhinagar'}, {'name': 'Sophia I. Thomopoulos'}, {'name': 'Conor Owens-Walton'}, {'name': 'Dimitris Stripelis'}, {'name': 'Jose Luis Ambite'}, {'name': 'Greg Ver Steeg'}, {'name': 'Daniel Weintraub'}, {'name': 'Philip Cook'}, {'name': 'Corey McMillan'}, {'name': 'Paul M. Thompson'}]\", 'abstract': '<p>Parkinsons disease (PD) and Alzheimers disease (AD) are progressive neurodegenerative disorders that affect millions of people worldwide. In this work, we propose a deep learning approach to classify these diseases based on 3D T1-weighted brain MRI. We analyzed several datasets including the Parkinsons Progression Markers Initiative (PPMI), an independent dataset from the University of Pennsylvania School of Medicine (UPenn), the Alzheimers Disease Neuroimaging Initiative (ADNI), and the Open Access Series of Imaging Studies (OASIS) dataset. PPMI and ADNI were partitioned to train (70%), validate (20%), and test (10%) a 3D convolutional neural network (CNN) for PD and AD classification. The UPenn and OASIS datasets were used as independent test sets to evaluate the model performance during inference. We also implemented a random forest classifier as a baseline model by extracting key radiomics features from the same T1-weighted MRI scans. The proposed 3D CNN model was trained from scratch for the classification tasks. For AD classification, the 3D CNN model achieved an ROC-AUC of 0.878 on the ADNI test set and an average ROC-AUC of 0.789 on the OASIS dataset. For PD classification, the proposed 3D CNN model achieved an ROC-AUC of 0.667 on the PPMI test set and an average ROC-AUC of 0.743 on the UPenn dataset. We also found that model performance was largely maintained when using only 25% of the training dataset. The 3D CNN outperformed the random forest classifier for both the PD and AD tasks. The 3D CNN also generalized better on unseen MRI data from different imaging centers. Our results show that the proposed 3D CNN model was less prone to overfitting for AD than for PD classification. This approach shows promise for screening of PD and AD patients using only T1-weighted brain MRI, which is relatively widely available. This model with additional validation could also be used to help differentiate between challenging cases of AD and PD when they present with similarly subtle motor and non-motor symptoms.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.26.453918', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Anaerobic biodegradation of chloroform and dichloromethane with a Dehalobacter enrichment culture', 'authors': \"[{'name': 'Hao Wang'}, {'name': 'Rong Yu'}, {'name': 'Jennifer Webb'}, {'name': 'Peter Dollar'}, {'name': 'David L Freedman'}]\", 'abstract': '<p>Chloroform (CF) and dichloromethane (DCM) are among the more commonly identified chlorinated aliphatic compounds found in contaminated soil and groundwater. Complete dechlorination of CF has been reported under anaerobic conditions by microbes that respire CF to DCM and others that biodegrade DCM. The objectives of this study were to ascertain if a commercially available bioaugmentation enrichment culture (KB-1<sup></sup> Plus) uses an oxidative or fermentative pathway for biodegradation of DCM; and to determine if the products from DCM biodegradation can support organohalide respiration of CF to DCM in the absence of an exogenous electron donor. In various treatments with the KB-1<sup></sup> Plus culture to which <sup>14</sup>C-CF was added, the predominant product was <sup>14</sup>CO<sub>2</sub>, indicating that oxidation is the predominant <sub>\\xa0</sub>pathway for DCM. Recovery of <sup>14</sup>C-DCM when biodegradation was still in progress confirmed that CF first undergoes reductive dechlorination to DCM. <sup>14</sup>C-labeled organic acids, including acetate and propionate, were also recovered, suggesting that synthesis of organic acids provides a sink for the electron equivalents from oxidation of DCM. When the culture was washed to remove organic acids from prior additions of exogenous electron donor and only CF and DCM were added, the culture completely dechlorinated CF. The total amount of DCM added was not sufficient to provide the electron equivalents needed to reduce CF to DCM. Thus, the additional reducing power came via the DCM generated from CF reduction. Nevertheless, the rate of CF consumption was considerably slower in comparison to treatments that received an exogenous electron donor.</p>', 'posted': '2021-07-27'}, {'doi': '10.1101/2021.07.23.21261039', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Geographical concentration of COVID-19 cases by social determinants of health in 16 large metropolitan areas in Canada  a cross-sectional study', 'authors': \"[{'name': 'Yiqing Xia'}, {'name': 'Huiting Ma'}, {'name': 'Gary Moloney'}, {'name': 'Hctor A. Velsquez Garca'}, {'name': 'Monica Sirski'}, {'name': 'Naveed Z. Janjua'}, {'name': 'David Vickers'}, {'name': 'Tyler Williamson'}, {'name': 'Alan Katz'}, {'name': 'Kristy Yu'}, {'name': 'Rafal Kustra'}, {'name': 'David L Buckeridge'}, {'name': 'Marc Brisson'}, {'name': 'Stefan D Baral'}, {'name': 'Sharmistha Mishra'}, {'name': 'Mathieu Maheu-Giroux'}]\", 'abstract': '<sec><title>Background</title><p>There is a growing recognition that strategies to reduce SARS-CoV-2 transmission should be responsive to local transmission dynamics. Studies have revealed inequalities along social determinants of health, but little investigation was conducted surrounding geographic concentration within cities. We quantified social determinants of geographic concentration of COVID-19 cases across sixteen census metropolitan areas (CMA) in four Canadian provinces.</p></sec><sec><title>Methods</title><p>We used surveillance data on confirmed COVID-19 cases at the level of dissemination area. Gini (co-Gini) coefficients were calculated by CMA based on the proportion of the population in ranks of diagnosed cases and each social determinant using census data (income, education, visible minority, recent immigration, suitable housing, and essential workers) and the corresponding share of cases. Heterogeneity was visualized using Lorenz (concentration) curves.</p></sec><sec><title>Results</title><p>Geographic concentration was observed in all CMAs (half of the cumulative cases were concentrated among 21-35% of each citys population): with the greatest geographic heterogeneity in Ontario CMAs (Gini coefficients, 0.32-0.47), followed by British Columbia (0.23-0.36), Manitoba (0.32), and Qubec (0.28-0.37). Cases were disproportionately concentrated in areas with lower income, education attainment, and suitable housing; and higher proportion of visible minorities, recent immigrants, and essential workers. Although a consistent feature across CMAs was concentration by proportion visible minorities, the magnitude of concentration by social determinants varied across CMAs.</p></sec><sec><title>Interpretation</title><p>The feature of geographical concentration of COVID-19 cases was consistent across CMAs, but the pattern by social determinants varied. Geographically-prioritized allocation of resources and services should be tailored to the local drivers of inequalities in transmission in response to SARS-CoV-2s resurgence.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.23.21261034', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Orthopedics', 'covid': False, 'title': 'Running related biomechanical risk factors for overuse injuries in distance runners: A systematic review considering injury specificity and the potentials for future research', 'authors': \"[{'name': 'Steffen Willwacher'}, {'name': 'Markus Kurz'}, {'name': 'Johanna Robbin'}, {'name': 'Matthias Thelen'}, {'name': 'Joseph Hamill'}, {'name': 'Luke Kelly'}, {'name': 'Patrick Mai'}]\", 'abstract': '<sec><title>Objective</title><p>To identify and evaluate the evidence of the most relevant running-related risk factors (RRRFs) for running-related overuse injuries (ROIs) and to suggest future research directions.</p></sec><sec><title>Design</title><p>Systematic review considering prospective and retrospective studies. (PROSPERO_ID: 236832)</p></sec><sec><title>Data sources</title><p>Pubmed. Connected Papers. The search was performed in February 2021.</p></sec><sec><title>Eligibility criteria</title><p>English language. Studies on participants whose primary sport is running addressing the risk for the seven most common ROIs and at least one kinematic, kinetic (including pressure measurements), or electromyographic RRRF. An RRRF needed to be identified in at least one prospective or two retrospective studies.</p></sec><sec><title>Results</title><p>Sixty-two articles fulfilled our eligibility criteria. Levels of evidence for specific ROIs ranged from conflicting to moderate evidence. Running populations and methods applied varied considerably between studies. While some RRRFs appeared for several ROIs, most RRRFs were specific for a particular ROI. The biomechanical measurements performed in many studies would have allowed for consideration of many more RRRFs than have been reported, highlighting a potential for more effective data usage in the future.</p></sec><sec><title>Conclusion</title><p>This study offers a comprehensive overview of RRRFs for the most common ROIs, which might serve as a starting point to develop ROI-specific risk profiles of individual runners. Future work should use macroscopic (big data) approaches involving long-term data collections in the real world and microscopic approaches involving precise stress calculations using recent developments in biomechanical modelling. However, consensus on data collection standards (including the quantification of workload and stress tolerance variables and the reporting of injuries) is warranted.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453381', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Software for Non-Parametric Image Registration of 2-Photon Imaging Data', 'authors': \"[{'name': 'Philipp Flotho'}, {'name': 'Shinobu Nomura'}, {'name': 'Bernd Kuhn'}, {'name': 'Daniel J. Strauss'}]\", 'abstract': '<p>Functional 2-photon microscopy is a key technology for imaging neuronal activity which can, however, contain non-rigid movement artifacts. Despite the established performance of variational <italic>optical flow</italic> (OF) estimation in different computer vision areas and the importance of movement correction for 2-photon applications, no OF-based method for 2-photon imaging is available. We developed the easy-to-use toolbox <italic>Flow-Registration</italic> that outperforms previous alignment tools and allows to align and reconstruct even low <italic>signal-to-noise</italic> 2-photon imaging data.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453821', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PEGR: a flexible management platform for reproducible epigenomic and genomic research', 'authors': \"[{'name': 'Danying Shao'}, {'name': 'Gretta D. Kellogg'}, {'name': 'Ali Nematbakhsh'}, {'name': 'Prashant Kuntala'}, {'name': 'Shaun Mahony'}, {'name': 'B. Franklin Pugh'}, {'name': 'William KM Lai'}]\", 'abstract': '<p>Reproducibility is a significant challenge in (epi)genomic research due to the complexity of experiments composed of traditional biochemistry and informatics. Recent advances have exacerbated this challenge as high-throughput sequencing data is generated at an unprecedented pace. Here we report on our development of a <underline>P</underline>latform for <underline>E</underline>pi-<underline>G</underline>enomic <underline>R</underline>esearch (PEGR), a web-based project management platform that tracks and quality controls experiments from conception to publication-ready figures, compatible with multiple assays and bioinformatic pipelines. It supports rigor and reproducibility for biochemists working at the wet bench, while continuing to fully support reproducibility and reliability for bioinformaticians through integration with the Galaxy platform.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.24.21261076', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Health Informatics', 'covid': True, 'title': 'The U.S. COVID-19 Trends and Impact Survey, 2020-2021: Continuous real-time measurement of COVID-19 symptoms, risks, protective behaviors, testing and vaccination', 'authors': \"[{'name': 'Joshua A. Salomon'}, {'name': 'Alex Reinhart'}, {'name': 'Alyssa Bilinski'}, {'name': 'Eu Jing Chua'}, {'name': 'Wichada La Motte-Kerr'}, {'name': 'Minttu M. Rnn'}, {'name': 'Marissa Reitsma'}, {'name': 'Katherine Ann Morris'}, {'name': 'Sarah LaRocca'}, {'name': 'Tamar Farag'}, {'name': 'Frauke Kreuter'}, {'name': 'Roni Rosenfeld'}, {'name': 'Ryan J. Tibshirani'}]\", 'abstract': '<p>The U.S. COVID-19 Trends and Impact Survey (CTIS) is a large, cross-sectional, Internet-based survey that has operated continuously since April 6, 2020. By inviting a random sample of Facebook active users each day, CTIS collects information about COVID-19 symptoms, risks, mitigating behaviors, mental health, testing, vaccination, and other key priorities. The large scale of the survey  over 20 million responses in its first year of operation  allows tracking of trends over short timescales and allows comparisons at fine demographic and geographic detail. The survey has been repeatedly revised to respond to emerging public health priorities. In this paper, we describe the survey methods and content and give examples of CTIS results that illuminate key patterns and trends and help answer high-priority policy questions relevant to the COVID-19 epidemic and response. These results demonstrate how large online surveys can provide continuous, real-time indicators of important outcomes that are not subject to public health reporting delays and backlogs. The CTIS offers high value as a supplement to official reporting data by supplying essential information about behaviors, attitudes toward policy and preventive measures, economic impacts, and other topics not reported in public health surveillance systems.</p><sec><title>Significance statement</title><p>The U.S. COVID-19 Trends and Impact Survey (CTIS) has operated continuously since April 6, 2020, collecting over 20 million responses. The largest public health survey ever conducted in the United States, CTIS was designed to facilitate detailed demographic and geographic analyses, track trends over time, and accommodate rapid response to emerging priorities. Using examples of CTIS results illuminating trends in symptoms, risks, mitigating behaviors, testing and vaccination in relation to evolving high-priority policy questions over 12 months of the pandemic, we illustrate the value of online surveys for tracking patterns and trends in COVID outcomes as an adjunct to official reporting, and showcase unique insights that would not be visible through traditional public health reporting.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453746', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'EXPRESSION OF A SECRETABLE, CELL-PENETRATING CDKL5 PROTEIN ENHANCES THE EFFICACY OF AAV VECTOR-MEDIATED GENE THERAPY FOR CDKL5 DEFICIENCY DISORDER', 'authors': \"[{'name': 'Giorgio Medici'}, {'name': 'Marianna Tassinari'}, {'name': 'Giuseppe Galvani'}, {'name': 'Stefano Bastianini'}, {'name': 'Laura Gennaccaro'}, {'name': 'Manuela Loi'}, {'name': 'Claudia Fuchs'}, {'name': 'Nicola Mottolese'}, {'name': 'Sara Alvente'}, {'name': 'Chiara Berteotti'}, {'name': 'Helen Rappe Baggett'}, {'name': 'Giovanna Zoccoli'}, {'name': 'Maurizio Giustetto'}, {'name': 'Alysson Muotri'}, {'name': 'Hiroyuki Nakai'}, {'name': 'Stefania Trazzi'}, {'name': 'Elisabetta Ciani'}]\", 'abstract': '<p>No therapy is currently available for CDKL5 (cyclin-dependent kinase-like 5) deficiency disorder (CDD), a severe neurodevelopmental disorder caused by mutations in the CDKL5 gene. Although delivery of a wild-type copy of the mutated gene to cells represents the most curative approach for a monogenic disease, proof-of-concept studies highlight significant efficacy caveats for brain gene therapy. Herein, we used a secretable TATk-CDKL5 protein to enhance the efficiency of a gene therapy for CDD. We found that, although AAVPHP.B_Igk-TATk-CDKL5 and AAVPHP.B_CDKL5 vectors had similar brain infection efficiency, the AAVPHP.B_Igk-TATk-CDKL5 vector led to a higher CDKL5 protein replacement due to secretion and transduction of the TATk-CDKL5 protein into the neighboring cells. Importantly, <italic>Cdkl5</italic> KO mice treated with the AAVPHP.B_Igk-TATk-CDKL5 vector showed a behavioral and neuroanatomical improvement in comparison with vehicle-treated <italic>Cdkl5</italic> KO mice or <italic>Cdkl5</italic> KO mice treated with the AAVPHP.B_CDKL5 vector, indicating that a gene therapy based on a secretable recombinant TATk-CDKL5 protein is more effective at compensating <italic>Cdkl5</italic>-null brain defects than gene therapy based on the expression of the native CDKL5.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453735', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Low affinity integrin states have faster binding kinetics than the high affinity state', 'authors': \"[{'name': 'Jing Li'}, {'name': 'Jiabin Yan'}, {'name': 'Timothy A. Springer'}]\", 'abstract': '<p>Integrin conformational ensembles contain two low-affinity states, bent-closed and extended-closed, and an active, high-affinity, extended-open state. It is widely thought that integrins must be activated before they bind ligand; however, one model holds that activation follows ligand binding. As ligand-binding kinetics are not only rate limiting for cell adhesion but also have important implications for the mechanism of activation, we measure them here for integrins 41 and 51 and show that the low-affinity states bind substantially faster than the high-affinity state. On and off-rate measurements are similar for integrins on cell surfaces and ectodomain fragments. Although the extended-open conformations on-rate is 20-fold slower, its off-rate is 25,000-fold slower, resulting in a large affinity increase. The tighter ligand-binding pocket in the open state may slow its on-rate. These kinetic measurements, together with previous equilibrium measurements of integrin conformational state affinity and relative free energy on intact cells, are key to a definitive understanding of the mechanism of integrin activation.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453679', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Plasmid supercoiling decreases during the dark phase in cyanobacteria: a clarification of the interpretation of chloroquine-agarose gels.', 'authors': \"[{'name': 'Salima Rdiger'}, {'name': 'Anne Rediger'}, {'name': 'Adrian Klsch'}, {'name': 'Dennis Dienst'}, {'name': 'Ilka Maria Axmann'}, {'name': 'Rainer Machn'}]\", 'abstract': \"<p>In cyanobacteria DNA supercoiling varies over the diurnal light/dark cycle and is integrated with the circadian transcription program and (Woelfle et al. [2007], Vijayan et al. [2009], PNAS). Specifically, Woelfle et al. have reported that DNA supercoiling of an endogenous plasmid became progressively higher during prolonged dark phases in Synechococcus elongatus PCC 7942. This is counterintuitive, since higher levels of negative DNA supercoiling are commonly associated with exponential growth and high metabolic flux. Vijayan et al. then have reverted the interpretation of plasmid mobility on agarose gels supplemented with chloroquine diphosphate (CQ), but not further discussed the differences. Here, we set out to clarify this open issue in cyanobacterial DNA supercoiling dynamics. We first re-capitulate Keller's band counting method (1975, PNAS) using CQ instead of ethidium bromide as the intercalating agent. A 500x-1000x higher CQ concentration is required in the DNA relaxation reaction (topoisomerase I) than in the agarose gel buffer to quench all negative supercoiling of pUC19 extracted from Escherichia coli. This is likely due to the dependence of both, the DNA binding affinity of CQ and the induced DNA unwinding angle, on the ionic strength of the buffer. Lower levels of CQ were required to fully relax in vivo pUC19 supercoiling than were used by Woelfle et al. Next, we analyzed the in vivo supercoiling of endogenous plasmids of Synechocystis sp. PCC 6803, at two different CQ concentrations. This clearly indicates that negative supercoiling of plasmids does not increase but decreases in the dark phase, and progressively decreases further in prolonged darkness.</p>\", 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.26.453516', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Resiniferatoxin hampers the nocifensive response of Caenorhabditis elegans to noxious heat, and pathway analysis revealed that the Wnt signaling pathway is involved', 'authors': \"[{'name': 'Jennifer Ben Salem'}, {'name': 'Bruno Nkambeu'}, {'name': 'Dina N Arvanitis'}, {'name': 'Francis Beaudry'}]\", 'abstract': '<p>Resiniferatoxin (RTX) is a metabolite extracted from <italic>Euphorbia resinifera</italic>. RTX is a potent capsaicin analog with specific biological activities resulting from its agonist activity with the transient receptor potential channel vanilloid subfamily member 1 (TRPV1). RTX has been examined as a pain reliever, and more recently, investigated for its ability to desensitize cardiac sensory fibers expressing TRPV1 to improve chronic heart failure (CHF) outcomes using validated animal models. <italic>Caenorhabditis elegans</italic> (<italic>C. elegans</italic>) expresses orthologs of vanilloid receptors activated by capsaicin, producing antinociceptive effects. Thus, we used <italic>C. elegans</italic> to characterize the antinociceptive properties and performed proteomic profiling to uncover specific signaling networks. After exposure to RTX, wild-type (N2) and mutant <italic>C. elegans</italic> were placed on petri dishes divided into quadrants for heat stimulation. The thermal avoidance index was used to phenotype each tested <italic>C. elegans</italic> experimental group. The data revealed for the first time that RTX can hamper the nocifensive response of <italic>C. elegans</italic> to noxious heat (32C  35C). The effect was reversed 6 h after RTX exposure. Additionally, we identified the RTX target, the <italic>C. elegans</italic> transient receptor potential channel OCR-3. The proteomics and pathway enrichment analysis results suggest that Wnt signaling is triggered by the agonistic effects of RTX on <italic>C. elegans</italic> vanilloid receptors.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453707', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Opposing roles of blood-borne monocytes and tissue-resident macrophages in limbal stem cell damage after ocular injury', 'authors': \"[{'name': 'Chengxin Zhou'}, {'name': 'Fengyang Lei'}, {'name': 'Mirja Mittermaier'}, {'name': 'Bruce Ksander'}, {'name': 'Reza Dana'}, {'name': 'Claes H. Dohlman'}, {'name': 'Demetrios G. Vavvas'}, {'name': 'James Chodosh'}, {'name': 'Eleftherios I. Paschalis'}]\", 'abstract': '<p>Limbal stem cell (LSC) deficiency is a frequent and severe complication after chemical injury to the eye. Previous studies have assumed that this is mediated by direct chemical injury. Here we show that LSC loss can occur even without direct chemical injury to LSCs. Elevation of anterior chamber (AC) pH causes acute uveal stress, release of inflammatory cytokines at the basal limbal tissue, and subsequent LSC death. In particular, peripherally-derived CCR2<sup>+</sup>CX3CR1<sup>-</sup>monocytes cause upregulation of tumor necrosis factor alpha (TNF-) at the limbus, causing LSC loss. In contrast to peripherally-derived monocytes, CX3CR1<sup>+</sup>CCR2<sup>-</sup>tissue-resident macrophages are protective. Pharmacological depletion of tissue-resident macrophages prior to injury exacerbates LSC loss and increases LSC vulnerability to apoptosis following exposure to TNF-, despite reduced CCR2<sup>+</sup>cell infiltration into the tissue. Consistently, repopulation of the cornea by new tissue-resident macrophages after depletion reverse LSC susceptibility to inflammatory cell death. This findings have important implication in LSC protection after chemical injury.</p><sec><title>Significance</title><p>Limbal stem cell (LSC) loss after chemical burn is a significant clinical problem that has long been thought to be the result of direct chemical injury. This study demonstrates the existence of competing inflammatory mechanism responsible for LSC death, mediated by infiltrating peripheral monocytes that antagonize and overcome the protective role of tissue-resident macrophages. Thus, enhancing the role of tissue-resident macrophages while suppressing that of infiltrating monocytes could be a novel therapeutic strategy for LSC survival.</p></sec>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453697', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Specialized subsets of innate-like T cells and dendritic cells protect from lethal pneumococcal infection in the lung', 'authors': \"[{'name': 'Mallory Paynich Murray'}, {'name': 'Catherine M. Crosby'}, {'name': 'Paola Marcovecchio'}, {'name': 'Nadine Hartmann'}, {'name': 'Shilpi Chandra'}, {'name': 'Meng Zhao'}, {'name': 'Archana Khurana'}, {'name': 'Sonja P. Zahner'}, {'name': 'Bjrn E. Clausen'}, {'name': 'Fadie T. Coleman'}, {'name': 'Joseph P. Mizgerd'}, {'name': 'Zbigniew Mikulski'}, {'name': 'Mitchell Kronenberg'}]\", 'abstract': '<p>Innate-like T cells, including invariant natural killer T (iNKT) cells, mucosal-associated invariant T (MAIT) cells and  T cells, are present in various barrier tissues, including the lung. They carry out protective responses during infections, but the mechanisms for protection are not completely understood. Here, we investigated their roles during pulmonary infection with <italic>Streptococcus pneumoniae</italic>. Following infection, innate-like T cells rapidly increased in lung tissue, in part through recruitment, but TCR activation and cytokine production occurred mostly in IL-17-producing NKT17 and  T cells. NKT17 cells were preferentially located outside the vasculature prior to infection, as were CD103<sup>+</sup> dendritic cells (cDC1), which were important both for antigen presentation to NKT17 cells and  T cell activation. Whereas IL-17A-producing  T cells also were numerous, GM-CSF was exclusive to NKT17 cells and contributed to iNKT cell-mediated protection. These studies demonstrate how particular cellular interactions and responses of functional subsets of innate-like T cells contribute to protection from pathogenic lung infection.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453696', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Artificial Intelligence Resolves Kinetic Pathways of Magnesium Binding to RNA', 'authors': \"[{'name': 'Jan Neumann'}, {'name': 'Nadine Schwierz'}]\", 'abstract': '<p>Magnesium is an indispensable cofactor in countless vital processes. In order to understand its functional role, the characterization of the binding pathways to biomolecules such as RNA is crucial. Despite the importance, a molecular description is still lacking since the transition from the water-mediated outer-sphere to the direct inner-sphere conformation is on the millisecond timescale and therefore out of reach for conventional simulation techniques. To fill this gap, we use transition path sampling to resolve the binding pathways and to elucidate the role of the solvent in the reaction. The results reveal that the molecular void provoked by the leaving phosphate oxygen of the RNA is immediately filled by an entering water molecule. In addition, water molecules from the first and second hydration shell couple to the concerted exchange. To capture the intimate solute-solvent coupling, we perform a committor analysis as basis for a machine learning algorithm that derives the optimal deep learning model from thousands of scanned architectures using hyperparameter tuning. The results reveal that the properly optimized deep network architecture recognizes the important solvent structures, extracts the relevant information and predicts the commitment probability with high accuracy. Our results provide a quantitative description of solute-solvent coupling which is ubiquitous for kosmotropic ions and governs a large variety of biochemical reactions in aqueous solutions.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453688', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Physiology', 'covid': False, 'title': 'Overexpression of the gene encoding neurosecretory protein GL precursor prevents excessive fat accumulation in the adipose tissue of mice fed a long-term high-fat diet', 'authors': \"[{'name': 'Keisuke Fukumura'}, {'name': 'Yuki Narimatsu'}, {'name': 'Shogo Moriwaki'}, {'name': 'Eiko Iwakoshi-Ukena'}, {'name': 'Megumi Furumitsu'}, {'name': 'Kazuyoshi Ukena'}]\", 'abstract': '<p>We previously identified a novel small hypothalamic protein, neurosecretory protein GL (NPGL), which induces feeding behavior and fat accumulation in rodents depending on their diet. In the present study, we explored the effects of NPGL on feeding behavior and energy metabolism in mice placed on a long-term high-fat diet with 60% calories from fat (HFD 60). Overexpression of the NPGL precursor gene <italic>(Npgl</italic>) over 18 weeks increased food intake and body mass. The weekly body mass gain of <italic>Npgl</italic>-overexpressing mice was higher than that of controls until 7 weeks from induction of overexpression, after which it ceased to be so. Oral glucose tolerance tests showed that <italic>Npgl</italic> overexpression maintained glucose tolerance and increased blood insulin levels, and intraperitoneal insulin tolerance tests showed that it maintained insulin sensitivity. At the experimental endpoint, <italic>Npgl</italic> overexpression was associated with increased mass of the perirenal white adipose tissue (WAT) and decreased mass of the epididymal WAT (eWAT), resulting in little effect on the total WAT mass. These results suggest that under long-term HFD 60 feeding, <italic>Npgl</italic> overexpression may play a role in avoiding metabolic disturbance both by accelerating energy storage and by suppressing excess fat accumulation in certain tissues, such as the eWAT.</p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.25.453685', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'SegmA: Residue Segmentation of cryo-EM density maps', 'authors': \"[{'name': 'Mark Rozanov'}, {'name': 'Haim J. Wolfson'}]\", 'abstract': '<label>1</label><title>Abstract</title><p>The cryo-EM resolution revolution enables the development of algorithms for direct de-novo modelling of protein structures from given cryo-EM density maps. Deep Learning tools have been applied to locate structure patterns, such as rotamers, secondary structures and C<italic></italic> atoms. We present a deep neural network (nicknamed SegmA) for the residue type segmentation of a cryo-EM density map. The network labels voxels in a cryo-EM map by the residue type (amino acid type or nucleic acid) of the sampled macromolecular structure. It also provides a visual representation of the density map by coloring the different types of voxels by their assgned colors. SegmAs algorithm combines a rotation equivariant group convolutional network with a traditional U-net network in a cascade. In addition SegmA estimates the labeling accuracy and reports only labels assigned with high confidence. At resolution of 3<italic></italic> SegmAs accuracy is 80% for nucleotides. Amino acids which can be seen by eye, such as LEU, ARG and PHE, are detected by Segma with about 70% accuracy.</p><p>A web server of the application is under development at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://dev.dcsh7cbr3o89e.amplifyapp.com\">https://dev.dcsh7cbr3o89e.amplifyapp.com</ext-link>. The SegmA open code is available at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/Mark-Rozanov/SegmA_3A/tree/master\">https://github.com/Mark-Rozanov/SegmA_3A/tree/master</ext-link></p>', 'posted': '2021-07-26'}, {'doi': '10.1101/2021.07.28.454154', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Muscarinic receptors mediate motivation via preparatory neural activity in humans', 'authors': \"[{'name': 'John P. Grogan'}, {'name': 'Matthias Raemaekers'}, {'name': 'Maaike M. H. Van Swieten'}, {'name': 'Alexander L. Green'}, {'name': 'Martin J. Gillies'}, {'name': 'Sanjay G. Manohar'}]\", 'abstract': '<p>Motivation depends on dopamine, but might be modulated by acetylcholine which influences dopamine release in the striatum, and amplifies motivation in animal studies. A corresponding effect in humans would be important clinically, since anticholinergic drugs are frequently used in Parkinsons disease, a condition that can also disrupt motivation. Reward and dopamine make us more ready to respond, as indexed by reaction times (RT), and move faster, sometimes termed vigour. These effects may be controlled by preparatory processes that can be tracked using EEG. We measured vigour in a placebo-controlled, double-blinded study of trihexyphenidyl (THP), a muscarinic antagonist, with an incentivised eye movement task and EEG. Participants responded faster and with greater vigour when incentives were high, but THP blunted these motivation effects, suggesting that muscarinic receptors facilitate invigoration by reward. Preparatory EEG build-up (contingent negative variation; CNV) was strengthened by high incentives and by muscarinic blockade. The amplitude of preparatory activity predicted both vigour and RT, although over distinct scalp regions. Frontal activity predicted vigour, whereas a larger, earlier, central component predicted RT. Indeed the incentivisation of RT was partly mediated by the CNV, though vigour was not. Moreover, the CNV mediated the drugs effect on dampening incentives, suggesting that muscarinic receptors underlie the motivational influence on this preparatory activity. Taken together, these findings show that a muscarinic blocker used to treat Parkinsons disease impairs motivated action in healthy people, and that medial frontal preparatory neural activity mediates this for RT.</p>', 'posted': '2021-07-29'}, {'doi': '10.1101/2021.07.23.453588', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'Accelerated epigenetic aging and inflammatory/immunological profile (ipAGE) in patients with chronic kidney disease', 'authors': \"[{'name': 'Igor Yusipov'}, {'name': 'Elena Kondakova'}, {'name': 'Alena Kalyakulina'}, {'name': 'Mikhail Krivonosov'}, {'name': 'Nadezhda Lobanova'}, {'name': 'Maria Giulia Bacalini'}, {'name': 'Claudio Franceschi'}, {'name': 'Maria Vedunova'}, {'name': 'Mikhail Ivanchenko'}]\", 'abstract': '<p>Chronic kidney disease (CKD) is defined by reduced estimated glomerular filtration rate (eGFR). This failure can be related to a phenotype of accelerated aging. In this work we considered 76 subjects with end-stage renal disease (ESRD) and 83 healthy controls. We evaluated two measures that can be informative of the rate of aging, i.e. whole blood DNA methylation using the Illumina Infinium EPIC array and plasma levels of a selection of inflammatory/immunological proteins using Multiplex Immunoassays. We demonstrated accelerated aging in terms of the most common epigenetic age estimators in CKD patients. We developed a new predictor of age based on inflammatory/immunological profile (ipAGE) and confirmed age acceleration in CKD patients. Finally, we evaluated the relationship between epigenetic age predictors and ipAGE and further identified the inflammatory/immunological biomarkers differentially expressed between cases and controls. In summary, our data show an accelerated aging phenotype in CKD patients sustained by inflammatory processes.</p>', 'posted': '2021-07-26'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.07.26.21261130</td>\n",
       "      <td>2040</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Vaccine effectiveness when combining the ChAdO...</td>\n",
       "      <td>[{'name': 'Mie Agermose Gram'}, {'name': 'Jens...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The recommend...</td>\n",
       "      <td>2021-07-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.07.23.21261030</td>\n",
       "      <td>332</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Breakthrough Symptomatic COVID-19 Infections L...</td>\n",
       "      <td>[{'name': 'Daisy Massey'}, {'name': 'Diana Ber...</td>\n",
       "      <td>&lt;p&gt;Vaccines have been shown to be extremely ef...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.07.27.453996</td>\n",
       "      <td>113</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Disruption of the TCA cycle reveals an ATF4-de...</td>\n",
       "      <td>[{'name': 'Dylan G. Ryan'}, {'name': 'Ming Yan...</td>\n",
       "      <td>&lt;p&gt;The Tricarboxylic Acid Cycle (TCA) cycle is...</td>\n",
       "      <td>2021-07-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.07.26.453748</td>\n",
       "      <td>75</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Identification of a new family of megaphages...</td>\n",
       "      <td>[{'name': 'Slawomir Michniewski'}, {'name': 'B...</td>\n",
       "      <td>&lt;p&gt;Megaphages  bacteriophages harbouring extr...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.07.30.454437</td>\n",
       "      <td>68</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Highly efficient SARS-CoV-2 infection of human...</td>\n",
       "      <td>[{'name': 'Chanakha K. Navaratnarajah'}, {'nam...</td>\n",
       "      <td>&lt;p&gt;Severe cardiovascular complications can occ...</td>\n",
       "      <td>2021-07-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.07.23.453379</td>\n",
       "      <td>65</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>AlphaPept, a modern and open framework for MS-...</td>\n",
       "      <td>[{'name': 'Maximilian T. Strauss'}, {'name': '...</td>\n",
       "      <td>&lt;p&gt;In common with other omics technologies, ma...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.07.23.21261041</td>\n",
       "      <td>58</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>The impact of large mobile air purifiers on ae...</td>\n",
       "      <td>[{'name': 'F. F. Duill'}, {'name': 'F. Schulz'...</td>\n",
       "      <td>&lt;p&gt;In the wake of the SARS-CoV-2 pandemic, an ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.07.23.453492</td>\n",
       "      <td>57</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Scientific Communication and Education</td>\n",
       "      <td>False</td>\n",
       "      <td>Delineating Medical Education: Bibliometric Re...</td>\n",
       "      <td>[{'name': 'Lauren A. Maggio'}, {'name': 'Anton...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The field of ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.07.24.21261040</td>\n",
       "      <td>51</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Novel risk factors for Coronavirus disease-ass...</td>\n",
       "      <td>[{'name': 'Umang Arora'}, {'name': 'Megha Priy...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The epidemiol...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.07.22.453458</td>\n",
       "      <td>46</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Epithelial-mesenchymal plasticity determines e...</td>\n",
       "      <td>[{'name': 'Patrick Aouad'}, {'name': 'Yueyun Z...</td>\n",
       "      <td>&lt;p&gt;Estrogen receptor -positive (ER+) breast c...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.07.30.454514</td>\n",
       "      <td>44</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Length Biases in Single-Cell RNA Sequencing of...</td>\n",
       "      <td>[{'name': 'Gennady Gorin'}, {'name': 'Lior Pac...</td>\n",
       "      <td>&lt;p&gt;Single-molecule pre-mRNA and mRNA sequencin...</td>\n",
       "      <td>2021-07-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.07.26.453760</td>\n",
       "      <td>37</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>A bacterial dynamin-like protein confers a nov...</td>\n",
       "      <td>[{'name': 'Lijun Guo'}, {'name': 'Laura Sattle...</td>\n",
       "      <td>&lt;p&gt;&lt;italic&gt;Bacillus subtilis&lt;/italic&gt; DynA is ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.07.25.21261093</td>\n",
       "      <td>34</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Protection afforded by the BNT162b2 and mRNA-1...</td>\n",
       "      <td>[{'name': 'Laith J. Abu-Raddad'}, {'name': 'Hi...</td>\n",
       "      <td>&lt;p&gt;Effect of prior SARS-CoV-2 infection on vac...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.07.27.453460</td>\n",
       "      <td>29</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Extracellular mechanical forces drive endocard...</td>\n",
       "      <td>[{'name': 'Julien Vermot'}, {'name': 'Helene V...</td>\n",
       "      <td>&lt;p&gt;Organ morphogenesis involves dynamic change...</td>\n",
       "      <td>2021-07-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.07.28.454180</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Cryo-EM reconstructions of inhibitor-bound SMG...</td>\n",
       "      <td>[{'name': 'Lukas M. Langer'}, {'name': 'Fabien...</td>\n",
       "      <td>&lt;p&gt;The PI3K-related kinase (PIKK) SMG1 monitor...</td>\n",
       "      <td>2021-07-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.07.26.453747</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>ABlooper: Fast accurate antibody CDR loop stru...</td>\n",
       "      <td>[{'name': 'Brennan Abanades'}, {'name': 'Guy G...</td>\n",
       "      <td>&lt;p&gt;Antibodies are a key component of the immun...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.07.26.453816</td>\n",
       "      <td>23</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Synaptic Mechanisms of Top-Down Control by The...</td>\n",
       "      <td>[{'name': 'Hannah M. Oberle'}, {'name': 'Alex ...</td>\n",
       "      <td>&lt;p&gt;Corticofugal projections to evolutionarily ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.07.25.453647</td>\n",
       "      <td>21</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Oncogenic c-Myc induces replication stress by ...</td>\n",
       "      <td>[{'name': 'Silvia Peripolli'}, {'name': 'Tanya...</td>\n",
       "      <td>&lt;p&gt;Oncogene-induced replication stress is a ma...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.07.25.453673</td>\n",
       "      <td>19</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Multivariate mining of an alpaca immune repert...</td>\n",
       "      <td>[{'name': 'Leo Hanke'}, {'name': 'Daniel J. Sh...</td>\n",
       "      <td>&lt;p&gt;Conventional approaches to isolate and char...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.07.25.453678</td>\n",
       "      <td>16</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Reward-dependent selection of feedback gains i...</td>\n",
       "      <td>[{'name': 'Antoine De Comite'}, {'name': 'Frd...</td>\n",
       "      <td>&lt;p&gt;Expected reward is known to affect planning...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.07.26.21260947</td>\n",
       "      <td>16</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Respiratory Medicine</td>\n",
       "      <td>False</td>\n",
       "      <td>Prospective validation of host transcriptomic ...</td>\n",
       "      <td>[{'name': 'Simon C. Mendelsohn'}, {'name': 'St...</td>\n",
       "      <td>&lt;p&gt;We tested performance of host-blood transcr...</td>\n",
       "      <td>2021-07-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.07.29.454071</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Age-Related Differences in Ventral Striatal an...</td>\n",
       "      <td>[{'name': 'Dominic S. Fareri'}, {'name': 'Kath...</td>\n",
       "      <td>&lt;p&gt;Social relationships change across the life...</td>\n",
       "      <td>2021-07-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.07.26.453637</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>LAMP2A regulates the loading of proteins into ...</td>\n",
       "      <td>[{'name': 'Joo Vasco Ferreira'}, {'name': 'An...</td>\n",
       "      <td>&lt;p&gt;Exosomes are extracellular vesicles of endo...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.07.25.453666</td>\n",
       "      <td>15</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Phylogenetic distribution and expression patte...</td>\n",
       "      <td>[{'name': 'Shizuka Koshimizu'}, {'name': 'Naok...</td>\n",
       "      <td>&lt;p&gt;Oogamy is a form of sexual reproduction and...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.07.25.453674</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Role of non-specific interactions in the phase...</td>\n",
       "      <td>[{'name': 'Rakesh Krishnan'}, {'name': 'Srivas...</td>\n",
       "      <td>&lt;p&gt;Phase separation of biomolecules could be m...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.07.24.453634</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Crowding-induced phase separation of nuclear t...</td>\n",
       "      <td>[{'name': 'Luke K. Davis'}, {'name': 'Ian J. F...</td>\n",
       "      <td>&lt;p&gt;The rapid (&amp;lt; 1 ms) transport of biologic...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.07.22.453442</td>\n",
       "      <td>14</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>False</td>\n",
       "      <td>Leveraging gene co-regulation to identify gene...</td>\n",
       "      <td>[{'name': 'Katherine M. Siewert-Rocks'}, {'nam...</td>\n",
       "      <td>&lt;p&gt;Identifying gene sets that are associated t...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.07.23.21261042</td>\n",
       "      <td>13</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>Robust antibody levels in both diabetic and no...</td>\n",
       "      <td>[{'name': 'Hamad Ali'}, {'name': 'Abdelmohsen ...</td>\n",
       "      <td>&lt;p&gt;The emergence of effective vaccines for COV...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.07.25.453574</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>Bats host the most virulentbut not the most d...</td>\n",
       "      <td>[{'name': 'Sarah Guth'}, {'name': 'Nardus Moll...</td>\n",
       "      <td>&lt;p&gt;Identifying virus characteristics associate...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2021.07.28.454097</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Systems Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Integrative analysis of the salt stress respon...</td>\n",
       "      <td>[{'name': 'Stephan Klaehn'}, {'name': 'Stefan ...</td>\n",
       "      <td>&lt;p&gt;Microorganisms evolved specific acclimation...</td>\n",
       "      <td>2021-07-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2021.07.25.453708</td>\n",
       "      <td>12</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The effective family size of immigrant founder...</td>\n",
       "      <td>[{'name': 'Damian Labuda'}, {'name': 'Tommy Ha...</td>\n",
       "      <td>&lt;p&gt;Human evolution involves population splits,...</td>\n",
       "      <td>2021-07-27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2021.07.26.453807</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Ecology</td>\n",
       "      <td>False</td>\n",
       "      <td>The benefits of merging two tracking approache...</td>\n",
       "      <td>[{'name': 'Louise Chavarie'}, {'name': 'Hannel...</td>\n",
       "      <td>&lt;p&gt;The process of smolting is a critical phase...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2021.07.24.452477</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Trial-wise exposure to visual appetitive cues ...</td>\n",
       "      <td>[{'name': 'Kilian Knauth'}, {'name': 'Jan Pete...</td>\n",
       "      <td>&lt;p&gt;Humans and many animals devalue future rewa...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10.1101/2021.07.28.454213</td>\n",
       "      <td>11</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Mitochondria dysfunction in Charcot Marie Toot...</td>\n",
       "      <td>[{'name': 'Yingli Gu'}, {'name': 'Flora Guerra...</td>\n",
       "      <td>&lt;p&gt;Recent evidence has uncovered an important ...</td>\n",
       "      <td>2021-07-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10.1101/2021.07.26.453733</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>AGO1 regulates major satellite transcripts and...</td>\n",
       "      <td>[{'name': 'Madlen Mller'}, {'name': 'Tara Fh...</td>\n",
       "      <td>&lt;p&gt;In the past years, several studies reported...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10.1101/2021.07.25.453671</td>\n",
       "      <td>10</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>MetaBinner: a high-performance and stand-alone...</td>\n",
       "      <td>[{'name': 'Ziye Wang'}, {'name': 'Pingqin Huan...</td>\n",
       "      <td>&lt;p&gt;Binning is an essential procedure during me...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2021.07.29.454133</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>Deep learning identifies and quantifies recomb...</td>\n",
       "      <td>[{'name': 'Yu Li'}, {'name': 'Siyuan Chen'}, {...</td>\n",
       "      <td>&lt;p&gt;Recombination is one of the essential genet...</td>\n",
       "      <td>2021-07-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2021.07.23.453597</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Generalizing Bayesian phylogenetics to infer s...</td>\n",
       "      <td>[{'name': 'Jamie R. Oaks'}, {'name': 'Perry L....</td>\n",
       "      <td>&lt;p&gt;Many processes of biological diversificatio...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10.1101/2021.07.26.453744</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Global protein responses of multi-drug resista...</td>\n",
       "      <td>[{'name': 'Anatte Margalit'}, {'name': 'James ...</td>\n",
       "      <td>&lt;p&gt;Antimicrobial resistance (AMR) and multi-dr...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10.1101/2021.07.25.453699</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Supra-physiological levels of gibberellins/DEL...</td>\n",
       "      <td>[{'name': 'Iva McCarthy-Surez'}]</td>\n",
       "      <td>&lt;p&gt;A previous study (McCarthy-Surez, 2021) sh...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>10.1101/2021.07.30.454065</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Subcellular and regional localization of mRNA ...</td>\n",
       "      <td>[{'name': 'Benjamin D Hobson'}, {'name': 'Ling...</td>\n",
       "      <td>&lt;p&gt;Local translation within excitatory and inh...</td>\n",
       "      <td>2021-08-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>10.1101/2021.07.25.453727</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Miniature inverted-repeat transposable element...</td>\n",
       "      <td>[{'name': 'Zhonglong Guo'}, {'name': 'Zheng Ku...</td>\n",
       "      <td>&lt;p&gt;MicroRNAs (miRNAs) are rapidly evolving end...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>10.1101/2021.07.26.453271</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Understanding associative false memories in ag...</td>\n",
       "      <td>[{'name': 'Nancy A. Dennis'}, {'name': 'Amy A....</td>\n",
       "      <td>&lt;p&gt;Associative memory declines in aging arise,...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>10.1101/2021.07.29.454302</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Physiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Small extracellular vesicles from young mice p...</td>\n",
       "      <td>[{'name': 'Jorge Sanz-Ros'}, {'name': 'Cristin...</td>\n",
       "      <td>&lt;p&gt;Aging is associated with an increased risk ...</td>\n",
       "      <td>2021-07-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>10.1101/2021.07.26.453355</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Inhibition of striatal SEZ6 by miR-3594-5p is ...</td>\n",
       "      <td>[{'name': 'Magalie Lenoir'}, {'name': 'Isabell...</td>\n",
       "      <td>&lt;p&gt;Escalation of drug use is a hallmark stage ...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10.1101/2021.07.26.453739</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Connectivity between the hippocampus and defau...</td>\n",
       "      <td>[{'name': 'Charlotte Murphy'}, {'name': 'Chara...</td>\n",
       "      <td>&lt;p&gt;Consistent with the idea that curiosity enh...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10.1101/2021.07.24.453449</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Heat-inactivated modified vaccinia virus Ankar...</td>\n",
       "      <td>[{'name': 'Ning Yang'}, {'name': 'Aitor Garzia...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;Protein or pe...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10.1101/2021.07.26.453750</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>EMDA: A Python package for Electron Microscopy...</td>\n",
       "      <td>[{'name': 'Rangana Warshamanage'}, {'name': 'K...</td>\n",
       "      <td>&lt;p&gt;An open-source Python library EMDA for cryo...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10.1101/2021.07.26.453751</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Quantitative Profiling of Adaptation to Cyclin...</td>\n",
       "      <td>[{'name': 'Juanita C. Limas'}, {'name': 'Amiee...</td>\n",
       "      <td>&lt;p&gt;Cyclin E/CDK2 drives cell cycle progression...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10.1101/2021.07.24.21261065</td>\n",
       "      <td>7</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Infectious Diseases (except HIV/AIDS)</td>\n",
       "      <td>True</td>\n",
       "      <td>COVID-Q: validation of the first COVID-19 ques...</td>\n",
       "      <td>[{'name': 'G Spinato'}, {'name': 'C Fabbris'},...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Objectives&lt;/title&gt;&lt;p&gt;The aim of th...</td>\n",
       "      <td>2021-07-26</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0   10.1101/2021.07.26.21261130    2040  medRxiv   \n",
       "1   10.1101/2021.07.23.21261030     332  medRxiv   \n",
       "2     10.1101/2021.07.27.453996     113  bioRxiv   \n",
       "3     10.1101/2021.07.26.453748      75  bioRxiv   \n",
       "4     10.1101/2021.07.30.454437      68  bioRxiv   \n",
       "5     10.1101/2021.07.23.453379      65  bioRxiv   \n",
       "6   10.1101/2021.07.23.21261041      58  medRxiv   \n",
       "7     10.1101/2021.07.23.453492      57  bioRxiv   \n",
       "8   10.1101/2021.07.24.21261040      51  medRxiv   \n",
       "9     10.1101/2021.07.22.453458      46  bioRxiv   \n",
       "10    10.1101/2021.07.30.454514      44  bioRxiv   \n",
       "11    10.1101/2021.07.26.453760      37  bioRxiv   \n",
       "12  10.1101/2021.07.25.21261093      34  medRxiv   \n",
       "13    10.1101/2021.07.27.453460      29  bioRxiv   \n",
       "14    10.1101/2021.07.28.454180      27  bioRxiv   \n",
       "15    10.1101/2021.07.26.453747      24  bioRxiv   \n",
       "16    10.1101/2021.07.26.453816      23  bioRxiv   \n",
       "17    10.1101/2021.07.25.453647      21  bioRxiv   \n",
       "18    10.1101/2021.07.25.453673      19  bioRxiv   \n",
       "19    10.1101/2021.07.25.453678      16  bioRxiv   \n",
       "20  10.1101/2021.07.26.21260947      16  medRxiv   \n",
       "21    10.1101/2021.07.29.454071      15  bioRxiv   \n",
       "22    10.1101/2021.07.26.453637      15  bioRxiv   \n",
       "23    10.1101/2021.07.25.453666      15  bioRxiv   \n",
       "24    10.1101/2021.07.25.453674      14  bioRxiv   \n",
       "25    10.1101/2021.07.24.453634      14  bioRxiv   \n",
       "26    10.1101/2021.07.22.453442      14  bioRxiv   \n",
       "27  10.1101/2021.07.23.21261042      13  medRxiv   \n",
       "28    10.1101/2021.07.25.453574      12  bioRxiv   \n",
       "29    10.1101/2021.07.28.454097      12  bioRxiv   \n",
       "30    10.1101/2021.07.25.453708      12  bioRxiv   \n",
       "31    10.1101/2021.07.26.453807      11  bioRxiv   \n",
       "32    10.1101/2021.07.24.452477      11  bioRxiv   \n",
       "33    10.1101/2021.07.28.454213      11  bioRxiv   \n",
       "34    10.1101/2021.07.26.453733      10  bioRxiv   \n",
       "35    10.1101/2021.07.25.453671      10  bioRxiv   \n",
       "36    10.1101/2021.07.29.454133       9  bioRxiv   \n",
       "37    10.1101/2021.07.23.453597       9  bioRxiv   \n",
       "38    10.1101/2021.07.26.453744       9  bioRxiv   \n",
       "39    10.1101/2021.07.25.453699       9  bioRxiv   \n",
       "40    10.1101/2021.07.30.454065       9  bioRxiv   \n",
       "41    10.1101/2021.07.25.453727       9  bioRxiv   \n",
       "42    10.1101/2021.07.26.453271       9  bioRxiv   \n",
       "43    10.1101/2021.07.29.454302       9  bioRxiv   \n",
       "44    10.1101/2021.07.26.453355       9  bioRxiv   \n",
       "45    10.1101/2021.07.26.453739       8  bioRxiv   \n",
       "46    10.1101/2021.07.24.453449       8  bioRxiv   \n",
       "47    10.1101/2021.07.26.453750       8  bioRxiv   \n",
       "48    10.1101/2021.07.26.453751       8  bioRxiv   \n",
       "49  10.1101/2021.07.24.21261065       7  medRxiv   \n",
       "\n",
       "                              subject-area  covid  \\\n",
       "0                             Epidemiology   True   \n",
       "1                             Epidemiology   True   \n",
       "2                             Biochemistry  False   \n",
       "3                             Microbiology  False   \n",
       "4                             Microbiology   True   \n",
       "5                           Bioinformatics  False   \n",
       "6                 Public and Global Health   True   \n",
       "7   Scientific Communication and Education  False   \n",
       "8                             Epidemiology   True   \n",
       "9                           Cancer Biology  False   \n",
       "10                              Biophysics  False   \n",
       "11                            Microbiology  False   \n",
       "12                            Epidemiology   True   \n",
       "13                   Developmental Biology  False   \n",
       "14                            Biochemistry  False   \n",
       "15                          Bioinformatics  False   \n",
       "16                            Neuroscience  False   \n",
       "17                          Cancer Biology  False   \n",
       "18                              Immunology   True   \n",
       "19                            Neuroscience  False   \n",
       "20                    Respiratory Medicine  False   \n",
       "21                            Neuroscience  False   \n",
       "22                            Cell Biology  False   \n",
       "23                    Evolutionary Biology  False   \n",
       "24                              Biophysics  False   \n",
       "25                              Biophysics  False   \n",
       "26                                Genetics  False   \n",
       "27   Infectious Diseases (except HIV/AIDS)   True   \n",
       "28                                 Ecology  False   \n",
       "29                         Systems Biology  False   \n",
       "30                    Evolutionary Biology  False   \n",
       "31                                 Ecology  False   \n",
       "32                            Neuroscience  False   \n",
       "33                            Neuroscience  False   \n",
       "34                            Cell Biology  False   \n",
       "35                          Bioinformatics  False   \n",
       "36                          Bioinformatics  False   \n",
       "37                    Evolutionary Biology  False   \n",
       "38                            Microbiology  False   \n",
       "39                           Plant Biology  False   \n",
       "40                            Neuroscience  False   \n",
       "41                           Plant Biology  False   \n",
       "42                            Neuroscience  False   \n",
       "43                              Physiology  False   \n",
       "44                            Neuroscience  False   \n",
       "45                            Neuroscience  False   \n",
       "46                              Immunology  False   \n",
       "47                              Biophysics  False   \n",
       "48                            Cell Biology  False   \n",
       "49   Infectious Diseases (except HIV/AIDS)   True   \n",
       "\n",
       "                                                title  \\\n",
       "0   Vaccine effectiveness when combining the ChAdO...   \n",
       "1   Breakthrough Symptomatic COVID-19 Infections L...   \n",
       "2   Disruption of the TCA cycle reveals an ATF4-de...   \n",
       "3   Identification of a new family of megaphages...   \n",
       "4   Highly efficient SARS-CoV-2 infection of human...   \n",
       "5   AlphaPept, a modern and open framework for MS-...   \n",
       "6   The impact of large mobile air purifiers on ae...   \n",
       "7   Delineating Medical Education: Bibliometric Re...   \n",
       "8   Novel risk factors for Coronavirus disease-ass...   \n",
       "9   Epithelial-mesenchymal plasticity determines e...   \n",
       "10  Length Biases in Single-Cell RNA Sequencing of...   \n",
       "11  A bacterial dynamin-like protein confers a nov...   \n",
       "12  Protection afforded by the BNT162b2 and mRNA-1...   \n",
       "13  Extracellular mechanical forces drive endocard...   \n",
       "14  Cryo-EM reconstructions of inhibitor-bound SMG...   \n",
       "15  ABlooper: Fast accurate antibody CDR loop stru...   \n",
       "16  Synaptic Mechanisms of Top-Down Control by The...   \n",
       "17  Oncogenic c-Myc induces replication stress by ...   \n",
       "18  Multivariate mining of an alpaca immune repert...   \n",
       "19  Reward-dependent selection of feedback gains i...   \n",
       "20  Prospective validation of host transcriptomic ...   \n",
       "21  Age-Related Differences in Ventral Striatal an...   \n",
       "22  LAMP2A regulates the loading of proteins into ...   \n",
       "23  Phylogenetic distribution and expression patte...   \n",
       "24  Role of non-specific interactions in the phase...   \n",
       "25  Crowding-induced phase separation of nuclear t...   \n",
       "26  Leveraging gene co-regulation to identify gene...   \n",
       "27  Robust antibody levels in both diabetic and no...   \n",
       "28  Bats host the most virulentbut not the most d...   \n",
       "29  Integrative analysis of the salt stress respon...   \n",
       "30  The effective family size of immigrant founder...   \n",
       "31  The benefits of merging two tracking approache...   \n",
       "32  Trial-wise exposure to visual appetitive cues ...   \n",
       "33  Mitochondria dysfunction in Charcot Marie Toot...   \n",
       "34  AGO1 regulates major satellite transcripts and...   \n",
       "35  MetaBinner: a high-performance and stand-alone...   \n",
       "36  Deep learning identifies and quantifies recomb...   \n",
       "37  Generalizing Bayesian phylogenetics to infer s...   \n",
       "38  Global protein responses of multi-drug resista...   \n",
       "39  Supra-physiological levels of gibberellins/DEL...   \n",
       "40  Subcellular and regional localization of mRNA ...   \n",
       "41  Miniature inverted-repeat transposable element...   \n",
       "42  Understanding associative false memories in ag...   \n",
       "43  Small extracellular vesicles from young mice p...   \n",
       "44  Inhibition of striatal SEZ6 by miR-3594-5p is ...   \n",
       "45  Connectivity between the hippocampus and defau...   \n",
       "46  Heat-inactivated modified vaccinia virus Ankar...   \n",
       "47  EMDA: A Python package for Electron Microscopy...   \n",
       "48  Quantitative Profiling of Adaptation to Cyclin...   \n",
       "49  COVID-Q: validation of the first COVID-19 ques...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Mie Agermose Gram'}, {'name': 'Jens...   \n",
       "1   [{'name': 'Daisy Massey'}, {'name': 'Diana Ber...   \n",
       "2   [{'name': 'Dylan G. Ryan'}, {'name': 'Ming Yan...   \n",
       "3   [{'name': 'Slawomir Michniewski'}, {'name': 'B...   \n",
       "4   [{'name': 'Chanakha K. Navaratnarajah'}, {'nam...   \n",
       "5   [{'name': 'Maximilian T. Strauss'}, {'name': '...   \n",
       "6   [{'name': 'F. F. Duill'}, {'name': 'F. Schulz'...   \n",
       "7   [{'name': 'Lauren A. Maggio'}, {'name': 'Anton...   \n",
       "8   [{'name': 'Umang Arora'}, {'name': 'Megha Priy...   \n",
       "9   [{'name': 'Patrick Aouad'}, {'name': 'Yueyun Z...   \n",
       "10  [{'name': 'Gennady Gorin'}, {'name': 'Lior Pac...   \n",
       "11  [{'name': 'Lijun Guo'}, {'name': 'Laura Sattle...   \n",
       "12  [{'name': 'Laith J. Abu-Raddad'}, {'name': 'Hi...   \n",
       "13  [{'name': 'Julien Vermot'}, {'name': 'Helene V...   \n",
       "14  [{'name': 'Lukas M. Langer'}, {'name': 'Fabien...   \n",
       "15  [{'name': 'Brennan Abanades'}, {'name': 'Guy G...   \n",
       "16  [{'name': 'Hannah M. Oberle'}, {'name': 'Alex ...   \n",
       "17  [{'name': 'Silvia Peripolli'}, {'name': 'Tanya...   \n",
       "18  [{'name': 'Leo Hanke'}, {'name': 'Daniel J. Sh...   \n",
       "19  [{'name': 'Antoine De Comite'}, {'name': 'Frd...   \n",
       "20  [{'name': 'Simon C. Mendelsohn'}, {'name': 'St...   \n",
       "21  [{'name': 'Dominic S. Fareri'}, {'name': 'Kath...   \n",
       "22  [{'name': 'Joo Vasco Ferreira'}, {'name': 'An...   \n",
       "23  [{'name': 'Shizuka Koshimizu'}, {'name': 'Naok...   \n",
       "24  [{'name': 'Rakesh Krishnan'}, {'name': 'Srivas...   \n",
       "25  [{'name': 'Luke K. Davis'}, {'name': 'Ian J. F...   \n",
       "26  [{'name': 'Katherine M. Siewert-Rocks'}, {'nam...   \n",
       "27  [{'name': 'Hamad Ali'}, {'name': 'Abdelmohsen ...   \n",
       "28  [{'name': 'Sarah Guth'}, {'name': 'Nardus Moll...   \n",
       "29  [{'name': 'Stephan Klaehn'}, {'name': 'Stefan ...   \n",
       "30  [{'name': 'Damian Labuda'}, {'name': 'Tommy Ha...   \n",
       "31  [{'name': 'Louise Chavarie'}, {'name': 'Hannel...   \n",
       "32  [{'name': 'Kilian Knauth'}, {'name': 'Jan Pete...   \n",
       "33  [{'name': 'Yingli Gu'}, {'name': 'Flora Guerra...   \n",
       "34  [{'name': 'Madlen Mller'}, {'name': 'Tara Fh...   \n",
       "35  [{'name': 'Ziye Wang'}, {'name': 'Pingqin Huan...   \n",
       "36  [{'name': 'Yu Li'}, {'name': 'Siyuan Chen'}, {...   \n",
       "37  [{'name': 'Jamie R. Oaks'}, {'name': 'Perry L....   \n",
       "38  [{'name': 'Anatte Margalit'}, {'name': 'James ...   \n",
       "39                  [{'name': 'Iva McCarthy-Surez'}]   \n",
       "40  [{'name': 'Benjamin D Hobson'}, {'name': 'Ling...   \n",
       "41  [{'name': 'Zhonglong Guo'}, {'name': 'Zheng Ku...   \n",
       "42  [{'name': 'Nancy A. Dennis'}, {'name': 'Amy A....   \n",
       "43  [{'name': 'Jorge Sanz-Ros'}, {'name': 'Cristin...   \n",
       "44  [{'name': 'Magalie Lenoir'}, {'name': 'Isabell...   \n",
       "45  [{'name': 'Charlotte Murphy'}, {'name': 'Chara...   \n",
       "46  [{'name': 'Ning Yang'}, {'name': 'Aitor Garzia...   \n",
       "47  [{'name': 'Rangana Warshamanage'}, {'name': 'K...   \n",
       "48  [{'name': 'Juanita C. Limas'}, {'name': 'Amiee...   \n",
       "49  [{'name': 'G Spinato'}, {'name': 'C Fabbris'},...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <sec><title>Background</title><p>The recommend...  2021-07-28  \n",
       "1   <p>Vaccines have been shown to be extremely ef...  2021-07-26  \n",
       "2   <p>The Tricarboxylic Acid Cycle (TCA) cycle is...  2021-07-27  \n",
       "3   <p>Megaphages  bacteriophages harbouring extr...  2021-07-26  \n",
       "4   <p>Severe cardiovascular complications can occ...  2021-07-30  \n",
       "5   <p>In common with other omics technologies, ma...  2021-07-26  \n",
       "6   <p>In the wake of the SARS-CoV-2 pandemic, an ...  2021-07-26  \n",
       "7   <sec><title>Background</title><p>The field of ...  2021-07-26  \n",
       "8   <sec><title>Background</title><p>The epidemiol...  2021-07-26  \n",
       "9   <p>Estrogen receptor -positive (ER+) breast c...  2021-07-26  \n",
       "10  <p>Single-molecule pre-mRNA and mRNA sequencin...  2021-07-31  \n",
       "11  <p><italic>Bacillus subtilis</italic> DynA is ...  2021-07-26  \n",
       "12  <p>Effect of prior SARS-CoV-2 infection on vac...  2021-07-26  \n",
       "13  <p>Organ morphogenesis involves dynamic change...  2021-07-27  \n",
       "14  <p>The PI3K-related kinase (PIKK) SMG1 monitor...  2021-07-28  \n",
       "15  <p>Antibodies are a key component of the immun...  2021-07-26  \n",
       "16  <p>Corticofugal projections to evolutionarily ...  2021-07-26  \n",
       "17  <p>Oncogene-induced replication stress is a ma...  2021-07-26  \n",
       "18  <p>Conventional approaches to isolate and char...  2021-07-26  \n",
       "19  <p>Expected reward is known to affect planning...  2021-07-26  \n",
       "20  <p>We tested performance of host-blood transcr...  2021-07-28  \n",
       "21  <p>Social relationships change across the life...  2021-07-30  \n",
       "22  <p>Exosomes are extracellular vesicles of endo...  2021-07-26  \n",
       "23  <p>Oogamy is a form of sexual reproduction and...  2021-07-26  \n",
       "24  <p>Phase separation of biomolecules could be m...  2021-07-26  \n",
       "25  <p>The rapid (&lt; 1 ms) transport of biologic...  2021-07-26  \n",
       "26  <p>Identifying gene sets that are associated t...  2021-07-26  \n",
       "27  <p>The emergence of effective vaccines for COV...  2021-07-26  \n",
       "28  <p>Identifying virus characteristics associate...  2021-07-26  \n",
       "29  <p>Microorganisms evolved specific acclimation...  2021-07-28  \n",
       "30  <p>Human evolution involves population splits,...  2021-07-27  \n",
       "31  <p>The process of smolting is a critical phase...  2021-07-26  \n",
       "32  <p>Humans and many animals devalue future rewa...  2021-07-26  \n",
       "33  <p>Recent evidence has uncovered an important ...  2021-07-28  \n",
       "34  <p>In the past years, several studies reported...  2021-07-26  \n",
       "35  <p>Binning is an essential procedure during me...  2021-07-26  \n",
       "36  <p>Recombination is one of the essential genet...  2021-07-29  \n",
       "37  <p>Many processes of biological diversificatio...  2021-07-26  \n",
       "38  <p>Antimicrobial resistance (AMR) and multi-dr...  2021-07-26  \n",
       "39  <p>A previous study (McCarthy-Surez, 2021) sh...  2021-07-26  \n",
       "40  <p>Local translation within excitatory and inh...  2021-08-01  \n",
       "41  <p>MicroRNAs (miRNAs) are rapidly evolving end...  2021-07-26  \n",
       "42  <p>Associative memory declines in aging arise,...  2021-07-26  \n",
       "43  <p>Aging is associated with an increased risk ...  2021-07-29  \n",
       "44  <p>Escalation of drug use is a hallmark stage ...  2021-07-26  \n",
       "45  <p>Consistent with the idea that curiosity enh...  2021-07-26  \n",
       "46  <sec><title>Background</title><p>Protein or pe...  2021-07-26  \n",
       "47  <p>An open-source Python library EMDA for cryo...  2021-07-26  \n",
       "48  <p>Cyclin E/CDK2 drives cell cycle progression...  2021-07-26  \n",
       "49  <sec><title>Objectives</title><p>The aim of th...  2021-07-26  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "116 preprints (including 17 covering SARS-CoV-2, 100 from bioRxiv and 16 from medRxiv) published in the last 7 days before 2021-08-02 had been tweeted at least 3 times (maximum 2040)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.5"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
