{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tweets from bioRxiv and medRxiv\n",
    "\n",
    "Find and display events from the CSHL DOI prefix 10.1101 from the last 7 days, and find the most tweeted preprints."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('../')\n",
    "sys.path.insert(0, '..')\n",
    "\n",
    "import pandas as pd # data analysis library\n",
    "import json\n",
    "import mrced2 # module to run event data queries\n",
    "import os # some file manipulations\n",
    "import math # some number manipulations\n",
    "import altair.vegalite.v3 as alt # some data visualizations\n",
    "from IPython.display import Markdown as md # some markdown manipulations\n",
    "from datetime import datetime, date, timedelta # some date manipulations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = (date.today() - timedelta(days = 7)).strftime('%Y-%m-%d')\n",
    "end_date = date.today()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&obj-id.prefix=10.1101&source=twitter&rows=500&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n"
     ]
    }
   ],
   "source": [
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.buildQuery({'obj-id.prefix' : prefix, 'source': 'twitter', 'rows': 500,'from-occurred-date' : start_date, 'until-occurred-date' : end_date})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "ed.runQuery(retry = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "12636 events found\n"
     ]
    }
   ],
   "source": [
    "pages = math.ceil(ed.events.getHits() / 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0000.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=0633d756-1645-4176-8937-68fd63f8a7d6&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0001.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=d2e8a6a1-394c-424e-93bc-9704d9a42d01&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0002.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=cc419ae9-d73b-42ea-bab5-75da32336e34&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0003.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=cff31375-7a99-4dfe-bfe4-84b6ffd719d3&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0004.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=ba89a3eb-de00-4901-8d6b-6237752bc60b&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0005.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=5b3f7a0a-7531-4109-9762-f4eb930ddd57&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0006.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=584550d1-4b6a-4d99-93e7-965e29c7f72e&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0007.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=705d8580-94e8-416e-b5f6-9d5d3ca06aad&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0008.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=80e7a7c5-a72a-4c65-9fa5-f258d7a4270d&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0009.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=9ae32045-5660-4901-a489-50baf7702476&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0010.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=345561f5-d08e-494c-9500-53abf53f906a&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0011.json\n",
      "https://api.eventdata.crossref.org/v1/events?mailto=info@front-matter.io&cursor=443eae4c-13f5-4e61-b688-fd25ce662f59&rows=1000&obj-id.prefix=10.1101&source=twitter&from-occurred-date=2021-08-30&until-occurred-date=2021-09-06\n",
      "Event Data query started...\n",
      "API query complete  200\n",
      "output file written to 1101/tweets_0012.json\n"
     ]
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "prefix = \"10.1101\"\n",
    "start_date = date.today() - timedelta(days = 7)\n",
    "end_date = date.today()\n",
    "\n",
    "# find the all result pages for the search\n",
    "ed = mrced2.eventData(mailto = email)\n",
    "ed.getAllPages(pages, {'rows': 1000, 'obj-id.prefix' : prefix, 'source': 'twitter', 'from-occurred-date' : start_date, 'until-occurred-date' : end_date}, fileprefix = '1101/tweets_') "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Initialisation to look at the properties of the results."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "failed to load preprint_tweets_2021-07-29.csv\n",
      "failed to load .DS_Store\n",
      "failed to load .gitkeep\n",
      "failed to load preprint_tweets_2021-08-30.csv\n",
      "failed to load preprint_tweets_2021-08-09.csv\n",
      "failed to load preprint_tweets_2021-08-23.csv\n",
      "failed to load preprint_tweets_2021-08-02.csv\n",
      "failed to load preprint_tweets_2021-08-16.csv\n",
      "failed to load .ipynb_checkpoints\n",
      "failed to load preprint_tweets_2021-07-26.csv\n",
      "output file written to 1101/tweets.json\n"
     ]
    }
   ],
   "source": [
    "jd1 = mrced2.eventRecord() # instance of a class to interpret the events\n",
    "files = os.listdir('1101') # get all the filenames\n",
    "\n",
    "jd1.mergeJsons(files, folder = '1101') # load the json event data from multiple files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "js = json.load(open(\"1101/tweets.json\"))\n",
    "df = pd.json_normalize(js, record_path = ['message', 'events'])\n",
    "gdf = df.groupby(['obj_id']).size().reset_index(name='count').sort_values('count', ascending=False)\n",
    "cdf = gdf[gdf['count'] >= 3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "REST API query started for 10.1101/2021.07.08.21260210...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.452194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.21261387...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.19.21255739...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258176...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261768...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261295...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456982...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.15.21262067...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.10.21260232...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.18.21257385...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21262342...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.21256520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.30.21255934...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21262044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258081...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.21262631...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261991...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457692...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255677...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.16.21257255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.21261912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.21261159...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456173...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257037...\n",
      "REST API query complete  404\n",
      "REST API query started for 10.1101/2021.04.23.21255973...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.24.21261007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.28.225102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21260050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21259924...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261707...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.20.21259209...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.23.424267...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.10.20172288...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.441510...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.15.341479...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.16.21251535...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.452381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.28.20029272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259673...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.450959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.21258690...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455901...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.22.436441...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.30.927871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456196...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.28.21252621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432474...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.10.430571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.21.21255898...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.01.20049528...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261845...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457057...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.21252096...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423721...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.419044...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457186...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067835...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20112334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458411...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.23.21257679...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261596...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456499...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256788...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.11.20229708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456260...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.21262943...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/423079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457079...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21253964...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.03.458877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21260998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.30.20047217...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/135582...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.21259976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.455988...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454333...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456266...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.09.20210146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.09.434621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.26.21259239...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/607390...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21257987...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.06.21258272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.21254398...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251362...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.03.075069...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.22.20137745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.302638...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.439272...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457694...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259277...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.12.422516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261030...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.16.456545...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.04.451053...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21259912...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.19.21249483...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.27.21250604...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.21257129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.05.20054502...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457591...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.21259463...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.24.449781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.02.408575...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21256975...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.25.398172...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261642...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.01.21261297...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.442699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.428240...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.456740...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21258271...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.20205633...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.13.21260393...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.28.441880...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456193...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.22.432340...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455984...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257469...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.380741...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.20179671...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457082...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.12.148833...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.22.20109231...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425392...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.26.20219345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.21.20196428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.15.422900...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.21261876...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.435959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.09.21261704...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.10.21255248...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260872...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.21257080...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.436228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.21261814...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.02.424917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.04.20188417...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.09.085969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256403...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.07.20145979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.453962...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.213132...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.21.261586...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457702...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.21.21257595...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259918...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.23.310623...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.31.21258122...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.27.21261237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.452090...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.17.431670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258025...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.08.332023...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433745...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457563...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.450632...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.25.428136...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458428...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.21259105...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457124...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21260167...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.21261634...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.08.20248932...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262177...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457247...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262264...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.458102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.16.20231399...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.13.039917...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262314...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.433123...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450356...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259439...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.17.21262169...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457182...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.432967...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.442903...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.456015...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455804...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.28.454085...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21256578...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21259864...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.16.21258861...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256847...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.13.295691...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.07.21259779...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.457008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458557...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.06.20124461...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.445019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259785...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261855...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.20.259762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260621...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.452537...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.02.21.957480...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.21259726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.21256470...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.06.425465...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455859...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249131...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.22.914952...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.31.454200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.425008...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.02.21258881...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457718...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.06.21256705...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.21256619...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457556...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.01.31.928630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457601...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.21262065...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.21261013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.30.20217364...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.13.21251670...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.21260236...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21254882...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458226...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.31.437831...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.30.21259787...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456187...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456366...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.18.296657...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.21262789...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.457979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.04.21254906...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21262050...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443888...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449290...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456150...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261921...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.21258147...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/715482...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.07.21256823...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.20249101...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.22.457102...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.21252960...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.22.423783...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.05.434040...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.10.20247205...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.27.433180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.29.454199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.03.442423...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.445532...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.07.21251311...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.26.21257879...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.11.448011...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438262...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438237...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.21252192...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.21262323...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449609...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/571380...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.21255959...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457106...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.21249731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.455451...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.10.20032698...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21258990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.455048...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.18.21252037...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262744...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457776...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.20.20157792...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458373...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.16.20194571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.15.252148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.14.21260307...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/247064...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.10.197426...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.08.280818...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21262687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.21262937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.26.20162420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.21.20236018...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453029...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458270...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21255810...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.02.324368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.16.435733...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252615...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458233...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.04.458950...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.14.383083...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.457871...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/655225...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.07.138883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.23.441180...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.452771...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.15.21260561...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.21258012...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.08.435504...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.04.21261576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.23.432551...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.22.20160341...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.21256649...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.214148...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456368...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.16.341883...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255184...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.433007...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.17.20156034...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.25.21254296...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.20.21254636...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.03.20117994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.456110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.06.081497...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.21260227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.13.20129627...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458520...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.02.20223891...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.02.429298...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21261000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458145...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458516...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.447484...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.457919...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.19.456809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.10.036418...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.18.436013...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21262021...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20216630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.21.20159228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.26.432976...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.20.20215863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.16.20067728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.01.21252622...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.17.452576...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.15.20065995...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.08.443253...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.30.454437...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.09.21257809...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.08.193813...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.11.21257016...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.01.424421...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.01.458571...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.21253098...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.27.20220905...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.20.444927...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.19.21260302...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.12.443786...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.21259010...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21250375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446230...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.437224...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.25.314112...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.25.441313...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.24.441211...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.21249661...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.07.438848...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.09.22.309229...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.05.21249310...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.03.451001...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.03.24.20042606...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455990...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.30.402644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.23.394114...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.438630...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.25.449994...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.26.20140731...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.24.21257729...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.24.457481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458377...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.02.446817...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.06.21261716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.28.20116129...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.450716...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.10.21261726...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.03.21258228...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.09.21255195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.458699...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.11.439381...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.08.19.225854...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.29.458141...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.06.21255017...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.451258...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.31.458283...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.15.440089...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.01.20118877...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.01.21259656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.04.438781...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.20.21260863...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.433755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.28.458046...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.03.21251066...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.02.21252772...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.10.447951...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.04.430128...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.11.455989...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.05.21254656...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.06.21253051...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.05.455249...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.31.20166116...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.27.21255937...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.04.20122481...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.17.448820...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.15.435406...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456143...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.19.21254000...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.30.458162...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.26.21256110...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.12.21253448...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.256164.119...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.18.449086...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.11.21253275...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.10.434828...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.05.20207241...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.23.21254060...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.13.21261969...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.03.454981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.26.433940...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.08.21261763...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.06.21260074...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.21261981...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.08.21252200...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.12.456077...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.04.02.438164...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.03.07.434227...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.456838...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.21.449244...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.05.442868...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.19.21257334...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.27.21260566...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.31.424746...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.03.367375...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.07.06.20147199...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.21.457214...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.18.456832...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.27.21257896...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.13.444019...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.31.426979...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.04.446837...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.13.093195...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.25.445523...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.21260755...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.28.446194...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.26.21250420...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.29.450255...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.23.21249554...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.21262708...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.20.21250152...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.18.20233353...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457748...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.11.378133...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.18.21249998...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.22.21257658...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.05.450926...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.15.426911...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.26.457795...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.22.453345...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.11.13.381467...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.20.457153...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.13.425144...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/037762...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.23.457316...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.12.451825...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21256644...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.275472.121...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.10.21.20208728...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.14.456331...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/074559...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.14.150490...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.08.451480...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.06.18.159434...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.28.21259452...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.04.23.056978...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.23.453473...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.08.415687...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/gr.265058.120...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.08.25.457645...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/292581...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.06.01.446635...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.05.10.21255146...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.01.12.426042...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.09.02.21262965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.08.21251329...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.12.21.423849...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.07.16.452665...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2020.05.01.20087965...\n",
      "REST API query complete  200\n",
      "REST API query started for 10.1101/2021.02.11.430786...\n",
      "REST API query complete  200\n",
      "[{'doi': '10.1101/2021.08.31.458411', 'tweets': 34, 'archive': 'bioRxiv', 'subject-area': 'Plant Biology', 'covid': False, 'title': 'An Arabidopsis root phloem pole cell atlas reveals PINEAPPLE genes as transitioners to autotrophy', 'authors': \"[{'name': 'Sofia Otero'}, {'name': 'Iris Sevilem'}, {'name': 'Pawel Roszak'}, {'name': 'Yipeng Lu'}, {'name': 'Valerio Di Vittori'}, {'name': 'Matthieu Bourdon'}, {'name': 'Lothar Kalmbach'}, {'name': 'Bernhard Blob'}, {'name': 'Jung-ok Heo'}, {'name': 'Federico Peruzzo'}, {'name': 'Thomas Laux'}, {'name': 'Alisdair R. Fernie'}, {'name': 'Hugo Tavares'}, {'name': 'Yka Helariutta'}]\", 'abstract': '<p>Single cell sequencing has recently allowed the generation of exhaustive root cell atlases. However, some cell types are elusive and remain underrepresented. Here, we use a second- generation single cell approach, where we zoom in on the root transcriptome sorting with specific markers to profile the phloem poles at an unprecedented resolution. Our data highlight the similarities among the developmental trajectories and gene regulatory networks communal to protophloem sieve element (PSE) adjacent lineages in relation to PSE enucleation, a key event in phloem biology.</p><p>As a signature for early PSE-adjacent lineages, we have identified a set of DNA-binding with one finger (DOF) transcription factors, the PINEAPPLEs (PAPL), that act downstream of <italic>PHLOEM EARLY DOF</italic> (<italic>PEAR</italic>) genes, and are important to guarantee a proper root nutrition in the transition to autotrophy.</p><p>Our data provide a holistic view of the phloem poles that act as a functional unit in root development.</p>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.31.21262943', 'tweets': 29, 'archive': 'medRxiv', 'subject-area': 'Public and Global Health', 'covid': True, 'title': 'Ensemble forecast of COVID-19 in Karnataka for vulnerability assessment and policy interventions', 'authors': \"[{'name': 'Sashikumaar Ganesan'}, {'name': 'Deepak Subramani'}, {'name': 'Thivin Anandh'}, {'name': 'Divij Ghose'}, {'name': 'Giridhara R Babu'}]\", 'abstract': '<p>We present an ensemble forecast for Wave-3 of COVID-19 in the state of Karnataka, India, using the IISc Population Balance Model for infectious disease spread. The reported data of confirmed, recovered, and deceased cases in Karnataka from 1 July 2020 to 4 July 2021 is utilized to tune the model’s parameters, and an ensemble forecast is done from 5 July 2021 to 30 June 2022. The ensemble is built with 972 members by varying seven critical parameters that quantify the uncertainty in the spread dynamics (antibody waning, viral mutation) and interventions (pharmaceutical, non-pharmaceutical). The probability of Wave-3, the peak date distribution, and the peak caseload distribution are estimated from the ensemble forecast. Our analysis shows that the most significant causal factors are compliance to Covid-appropriate behavior, daily vaccination rate, and the immune escape new variant emergence-time. These causal factors determine when and how severe the Wave-3 of COVID-19 would be in Karnataka. We observe that when compliance to Covid-Appropriate Behavior is good (i.e., lockdown-like compliance), the emergence of new immune-escape variants beyond Sep ‘21 is unlikely to induce a new wave. A new wave is inevitable when compliance to Covid-Appropriate Behavior is only partial. Increasing the daily vaccination rates reduces the peak active caseload at Wave-3. Consequently, the hospitalization, ICU, and Oxygen requirements also decrease. Compared to Wave-2, the ensemble forecast indicates that the number of daily confirmed cases of children (0-17 years) at Wave-3’s peak could be seven times more on average. Our results provide insights to plan science-informed policy interventions and public health response.</p>', 'posted': '2021-09-02'}, {'doi': '10.1101/2021.09.03.458877', 'tweets': 27, 'archive': 'bioRxiv', 'subject-area': 'Pharmacology and Toxicology', 'covid': True, 'title': 'Discovery of highly potent pancoronavirus fusion inhibitors that also effectively inhibit COVID-19 variants from the UK and South Africa', 'authors': \"[{'name': 'Francesca Curreli'}, {'name': 'Shahad Ahmed'}, {'name': 'Sofia M. B. Victor'}, {'name': 'Aleksandra Drelich'}, {'name': 'Siva S. Panda'}, {'name': 'Andrea Altieri'}, {'name': 'Alexander Kurkin'}, {'name': 'Chien-Te Tseng'}, {'name': 'Christopher Hillyer'}, {'name': 'Asim Debnath'}]\", 'abstract': '<p>We report the discovery of a series of benzoic acid-based inhibitors that show highly potent pancoronavirus activity. Some compounds also show complete inhibition of CPE (IC<sub>100</sub>) at 1.25 μM against an authentic SARS-CoV-2 (US_WA-1/2020). Furthermore, the most active inhibitors also potently inhibited variants initially identified in the UK and South Africa. We confirmed that one of the potent inhibitors binds to the prefusion spike protein trimer of SARS-CoV-2 by SPR. Besides, we showed that they inhibit virus-mediated cell-cell fusion. The ADME data show druglike characteristics, and in vivo PK in rats demonstrated excellent half-life (t½) of 11.3 h, mean resident time (MRT) of 14.2 h, and orally bioavailable. Despite the presence of ene-rhodamine moiety, we conclusively demonstrated that these inhibitors target the viral spike protein and are not promiscuous or colloidal aggregators. We expect the lead inhibitors to pave the way for further development to preclinical and clinical candidates.</p>', 'posted': '2021-09-03'}, {'doi': '10.1101/2021.08.30.458146', 'tweets': 24, 'archive': 'bioRxiv', 'subject-area': 'Developmental Biology', 'covid': False, 'title': 'The control of transcriptional memory by stable mitotic bookmarking', 'authors': \"[{'name': 'Maëlle Bellec'}, {'name': 'Jérémy Dufourt'}, {'name': 'George Hunt'}, {'name': 'Hélène Lenden-Hasse'}, {'name': 'Antonio Trullo'}, {'name': 'Amal Zine El Aabidine'}, {'name': 'Marie Lamarque'}, {'name': 'Marissa M Gaskill'}, {'name': 'Heloïse Faure-Gautron'}, {'name': 'Mattias Mannervik'}, {'name': 'Melissa M Harrison'}, {'name': 'Jean-Christophe Andrau'}, {'name': 'Cyril Favard'}, {'name': 'Ovidiu Radulescu'}, {'name': 'Mounia Lagha'}]\", 'abstract': '<p>To maintain cellular identities during development, gene expression profiles must be faithfully propagated through cell generations. The reestablishment of gene expression patterns upon mitotic exit is thought to be mediated, in part, by mitotic bookmarking by transcription factors (TF). However, the mechanisms and functions of TF mitotic bookmarking during early embryogenesis remain poorly understood. In this study, taking advantage of the naturally synchronized mitoses of <italic>Drosophila</italic> early embryos, we provide evidence that the pioneer-like transcription factor GAF acts as stable mitotic bookmarker during zygotic genome activation. We report that GAF remains associated to a large fraction of its interphase targets including at <italic>cis</italic>-regulatory sequences of key developmental genes, with both active and repressive chromatin signatures. GAF mitotic targets are globally accessible during mitosis and are bookmarked via histone acetylation (H4K8ac). By monitoring the kinetics of transcriptional activation in living embryos, we provide evidence that GAF binding establishes competence for rapid activation upon mitotic exit.</p>', 'posted': '2021-08-30'}, {'doi': '10.1101/2021.08.31.458428', 'tweets': 9, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Characterisation of a nucleo-adhesome', 'authors': \"[{'name': 'Adam Byron'}, {'name': 'Billie G. C. Griffith'}, {'name': 'Ana Herrero'}, {'name': 'Alexander E. P. Loftus'}, {'name': 'Emma S. Koeleman'}, {'name': 'John C. Dawson'}, {'name': 'Linda Kogerman'}, {'name': 'Niamh McGivern'}, {'name': 'Jayne Culley'}, {'name': 'Graeme R. Grimes'}, {'name': 'Bryan Serrels'}, {'name': 'Alex von Kriegsheim'}, {'name': 'Valerie G. Brunton'}, {'name': 'Margaret C. Frame'}]\", 'abstract': '<p>In addition to central functions in cell adhesion signalling, integrin-associated proteins have wider roles at sites distal to adhesion receptors. In experimentally defined adhesomes, we noticed that there is clear enrichment of proteins that localise to the nucleus, and conversely, we now report that nuclear proteomes contain a class of adhesome components that localise to the nucleus. We here defined a nucleo-adhesome, providing experimental evidence for a remarkable scale of nuclear localisation of adhesion proteins, establishing a framework for interrogating nuclear adhesion protein functions. In adding to nuclear FAK’s known roles in regulating transcription, we now show that nuclear FAK regulates expression of many adhesion-related proteins that localise to the nucleus and that nuclear FAK binds to the adhesome component and nuclear protein Hic-5. FAK and Hic-C work together in the nucleus, co-regulating a subset of genes transcriptionally. We describe the first nucleo-adhesome using a squamous cancer cell model, and demonstrate the new principle that there are nuclear adhesion protein subcomplexes that cooperate to control transcription.\\n<fig id=\"ufig1\" position=\"float\" fig-type=\"figure\" orientation=\"portrait\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"458428v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></p>', 'posted': '2021-08-31'}, {'doi': '10.1101/2021.08.29.458102', 'tweets': 8, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Exo1-protected DNA nicks direct crossover formation in meiosis', 'authors': \"[{'name': 'Michael Gioia'}, {'name': 'Lisette Payero'}, {'name': 'Gianno Pannafino'}, {'name': 'Jun Jie Chen'}, {'name': 'Sagar Salim'}, {'name': 'Ghanim Fajith'}, {'name': 'Amamah F. Farnaz'}, {'name': 'Sherikat Momoh'}, {'name': 'Michelle Scotland'}, {'name': 'Vandana Raghavan'}, {'name': 'Carol Manhart'}, {'name': 'Akira Shinohara'}, {'name': 'K.T. Nishant'}, {'name': 'Eric Alani'}]\", 'abstract': '<p>In most sexually reproducing organisms crossing over between chromosome homologs during meiosis is critical for the viability of haploid gametes. Most crossovers that form in meiosis in budding yeast result from the biased resolution of double Holliday Junction (dHJ) intermediates. This dHJ resolution step involves the actions Rad2/XPG family nuclease Exo1 and the Mlh1-Mlh3 mismatch repair endonuclease. At present little is known about how these factors act in meiosis at the molecular level. Here we show that Exo1 promotes meiotic crossing over by protecting DNA nicks from ligation. We found that structural elements in Exo1 required for interactions with DNA, such as bending of DNA during nick/flap recognition, are critical for its role in crossing over. Consistent with these observations, meiotic expression of the Rad2/XPG family member Rad27 partially rescued the crossover defect in <italic>exo1</italic> null mutants, and meiotic overexpression of Cdc9 ligase specifically reduced the crossover levels of <italic>exo1</italic> DNA binding mutants to levels approaching the <italic>exo1</italic> null. In addition, our work identified a role for Exo1 in crossover interference that appears independent of its resection activity. Together, these studies provide experimental evidence for Exo1-protected nicks being critical for the formation of meiotic crossovers and their distribution.</p>', 'posted': '2021-08-30'}, {'doi': '10.1101/2021.09.01.458557', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'Functional fluorescently-tagged human profilin', 'authors': \"[{'name': 'Morgan L. Pimm'}, {'name': 'Xinbei Liu'}, {'name': 'Farzana Tuli'}, {'name': 'Ashley Lojko'}, {'name': 'Jessica L. Henty-Ridilla'}]\", 'abstract': '<p>Profilin is an essential regulator of actin and microtubule dynamics and therefore a critical control point for the normal division, motility, and morphology of cells. Most studies of profilin have focused on biochemical investigations using purified protein because high cellular concentrations (121 µM) present challenges for conventional imaging modalities. In addition, past studies that employed direct labeling or conventional fusion protein strategies compromised different facets of profilin function. We engineered a fluorescently-labeled profilin that retains native activities with respect to phosphoinositide lipids, actin monomers, formin-mediated actin assembly, and microtubule polymerization. This fluorescent profilin directly binds to dimers of tubulin (k<sub>D</sub> = 1.7 µM) and the microtubule lattice (k<sub>D</sub> = 10 µM) to stimulate microtubule assembly. In cells, our tagged profilin fully rescues profilin-1<sup>(-/-)</sup> cells from knockout-induced perturbations to cell shape, actin filament architecture, and microtubule arrays. Thus, this labeled profilin-1 is a reliable tool to investigate the dynamic interactions of profilin with actin or microtubules in live cell and in vitro applications.</p>', 'posted': '2021-09-02'}, {'doi': '10.1101/2021.08.30.458230', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Disentangling the flow of signals between populations of neurons', 'authors': \"[{'name': 'Evren Gokcen'}, {'name': 'Anna I. Jasper'}, {'name': 'João D. Semedo'}, {'name': 'Amin Zandvakili'}, {'name': 'Adam Kohn'}, {'name': 'Christian K. Machens'}, {'name': 'Byron M. Yu'}]\", 'abstract': '<p>Technological advances have granted the ability to record from large populations of neurons across multiple brain areas. These recordings may illuminate how communication between areas contributes to brain function, yet a substantial barrier remains: How do we disentangle the concurrent, bidirectional flow of signals between two populations of neurons? We therefore propose here a novel dimensionality reduction framework: Delayed Latents Across Groups (DLAG). DLAG disentangles signals relayed in both directions; identifies how these signals are represented by each population; and characterizes how they evolve over time and trial-to-trial. We demonstrate that DLAG performs well on synthetic datasets similar in scale to current neurophysiological recordings. Then we study simultaneously recorded populations in primate visual areas V1 and V2, where DLAG reveals signatures of concurrent yet selective communication. Our framework lays the foundation for dissecting the intricate flow of signals across populations of neurons, and how this signaling contributes to cortical computation.</p>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.30.458226', 'tweets': 7, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'De Novo Prediction of RNA 3D Structures with Deep Learning', 'authors': \"[{'name': 'Julius Ramakers'}, {'name': 'Christopher Frederik Blum'}, {'name': 'Sabrina König'}, {'name': 'Stefan Harmeling'}, {'name': 'Markus Kollmann'}]\", 'abstract': '<p>We present a Deep Learning approach to predict 3D folding structures of RNAs from their nucleic acid sequence. Our approach combines an autoregressive Deep Generative Model, Monte Carlo Tree Search, and a Score Model to find and rank the most likely folding structures for a given RNA sequence. We confirm the predictive power of our approach by setting new benchmarks for some longer sequences in a simulated blind test of the RNA Puzzles prediction challenge.</p>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.29.21262789', 'tweets': 7, 'archive': 'medRxiv', 'subject-area': 'Epidemiology', 'covid': True, 'title': 'Vaccines alone are no silver bullets: a modeling study on the impact of efficient contact tracing on COVID-19 infection and transmission', 'authors': \"[{'name': 'Dhesi Baha Raja'}, {'name': 'Nur Asheila Abdul Taib'}, {'name': 'Alvin Kuo Jing Teo'}, {'name': 'Vivek Jason Jayaraj'}, {'name': 'Choo-Yee Ting'}]\", 'abstract': '<sec><title>Background</title><p>The computer simulation presented in this study aimed to investigate the effect of contact tracing on COVID-19 transmission and infection in the context of rising vaccination rates.</p></sec><sec><title>Methods</title><p>This study proposed a deterministic SEIRV model with contact tracing and vaccination components. We initialized some parameters using the Malaysian COVID-19 data to inform the model. We defined contact tracing effectiveness as the proportion of contacts of a positive case that was successfully traced and vaccination rate as the proportion of daily doses administered per population in Malaysia. Sensitivity analyses on the untraced and infectious populations were conducted. The study presented in silico findings on multiple scenarios by varying the contact tracing effectiveness and daily vaccination rates.</p></sec><sec><title>Results</title><p>At a vaccination rate of 1.4%, a contact tracing with the effectiveness of 70% could delay the peak of untraced asymptomatic cases by 17 days and reduce the highest number of daily cases by 70% compared with a 30% contact tracing effectiveness. A similar trend was observed for symptomatic cases when a similar experiment setting was used. We also performed sensitivity analyses by using different combinations of contact tracing effectiveness and vaccination rates. In all scenarios, the effect of contact tracing on COVID-19 incidence persisted for both asymptomatic and symptomatic cases.</p></sec><sec><title>Conclusion</title><p>Despite testing only on two public health and social measures (PHSMs), we observed the scenario with low contact tracing and increasing vaccination rates successfully mimicked the current transmission trend in Malaysia. Hence, while vaccines are progressively rolled out, efficient contact tracing must be rapidly implemented concurrently to reach, find, test, isolate, and support the affected populations to bring the pandemic under control.</p></sec>', 'posted': '2021-08-30'}, {'doi': '10.1101/2021.08.31.458373', 'tweets': 6, 'archive': 'bioRxiv', 'subject-area': 'Cell Biology', 'covid': False, 'title': 'A Cytosolic Reductase Pathway is Required for Complete N-Glycosylation of an STT3B-Dependent Acceptor Site', 'authors': \"[{'name': 'Marcel van Lith'}, {'name': 'Marie Anne Pringle'}, {'name': 'Bethany Fleming'}, {'name': 'Giorgia Gaeta'}, {'name': 'Jisu Im'}, {'name': 'Reid Gilmore'}, {'name': 'Neil J Bulleid'}]\", 'abstract': '<p>N-linked glycosylation of proteins entering the secretory pathway is an essential post-translational modification required for protein stability and function. Previously, it has been shown that there is a temporal relationship between protein folding and glycosylation, which influences the occupancy of specific glycosylation sites. Here we use an <italic>in vitro</italic> translation system that reproduces the initial stages of secretory protein translocation, folding and glycosylation under defined redox conditions. We found that the efficiency of glycosylation of hemopexin was dependent upon a robust NADPH-dependent cytosolic reductive pathway, which could also be mimicked by the addition of a membrane impermeable reducing agent. The identified hypoglycosylated acceptor site is adjacent to a cysteine involved in a short range disulfide bond, which has been shown to be dependent on the STT3B-containing oligosaccharyl transferase. We also show that efficient glycosylation at this site is dependent on the STT3A-containing oligosaccharide transferase. Our results provide further insight into the important role of the ER redox conditions in glycosylation site occupancy and demonstrate a link between redox conditions in the cytosol and glycosylation efficiency.</p>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.31.21262937', 'tweets': 6, 'archive': 'medRxiv', 'subject-area': 'Psychiatry and Clinical Psychology', 'covid': False, 'title': 'Impairments in goal-directed action and reversal learning in a proportion of individuals with psychosis: evidence for differential phenotypes in early and persistent psychosis', 'authors': \"[{'name': 'Shuichi Suetani'}, {'name': 'Andrea Baker'}, {'name': 'Kelly Garner'}, {'name': 'Peter Cosgrove'}, {'name': 'Matilda Mackay-Sim'}, {'name': 'Dan Siskind'}, {'name': 'Graham K Murray'}, {'name': 'James G Scott'}, {'name': 'James P Kesby'}]\", 'abstract': '<p>Cognitive impairments in psychosis are one of the strongest predictors of functional decline. Cortico-striatal dysfunction may contribute to both psychosis and cognitive impairment in psychotic illnesses. The decision-making processes underlying goal-directed action and serial reversal learning can be measured and are sensitive to changes reflecting cortico-striatal dysfunction. As such, changes in decision-making performance may assist with predicting functional decline in people with psychosis. We assessed decision-making processes in healthy controls (N=34), and those with early psychosis (N=15) and persistent psychosis (N=45). We subclassified subjects based on intact/impaired goal-directed action. Compared with healthy controls (&lt;20%), a large proportion (58%) of those with persistent psychosis displayed impaired goal-directed action, predicting poor serial reversal learning performance. Computational approaches indicated that those with persistent psychosis were less deterministic in their decision-making. Those with impaired goal-directed action had a decreased capacity to rapidly update their prior beliefs in the face of changing contingencies. In contrast, the early psychosis group included a lower proportion of individuals with impaired goal-directed action (20%) and displayed a different cognitive phenotype from those with persistent psychosis. These findings suggest prominent decision-making deficits, indicative of cortico-striatal dysfunction, are present in a large proportion of people with persistent psychosis while those with early psychosis have relatively intact decision-making processes compared to healthy controls. It is unclear if there is a progressive decline in decision-making processes in some individuals with psychosis or if the presence of decision-making processes in early psychosis is predictive of a persistent trajectory of illness.</p>', 'posted': '2021-09-02'}, {'doi': '10.1101/2021.08.30.458270', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Microbiology', 'covid': False, 'title': 'Global distribution of anaerobic dichloromethane degradation potential', 'authors': \"[{'name': 'Robert W. Murdoch'}, {'name': 'Gao Chen'}, {'name': 'Fadime Kara Murdoch'}, {'name': 'E. Erin Mack'}, {'name': 'Manuel I. Villalobos Solis'}, {'name': 'Robert L. Hettich'}, {'name': 'Frank E. Löffler'}]\", 'abstract': '<p>Anthropogenic activities and natural processes release dichloromethane (DCM), a toxic chemical with substantial ozone-depleting capacity. Specialized anaerobic bacteria metabolize DCM; however, the genetic basis for this process has remained elusive. Comparative genomics of the three known anaerobic DCM-degrading bacterial species revealed a homologous gene cluster, designated the methylene chloride catabolism (<italic>mec</italic>) gene cassette, comprising eight to ten genes with predicted 79.6 – 99.7% amino acid identity. Functional annotation identified genes encoding a corrinoid-dependent methyltransferase system, and shotgun proteomics applied to two DCM-catabolizing cultures revealed high expression of proteins encoded on the <italic>mec</italic> gene cluster during anaerobic growth with DCM. In a DCM-contaminated groundwater plume, the abundance of <italic>mec</italic> genes strongly correlated with DCM concentrations (R<sup>2</sup> = 0.71 – 0.85) indicating their value as process-specific bioremediation biomarkers. <italic>mec</italic> gene clusters were identified in metagenomes representing peat bogs, the deep subsurface, and marine ecosystems including oxygen minimum zones (OMZs), suggesting DCM turnover in diverse habitats. The broad distribution of anaerobic DCM catabolic potential suggests a relevant control function for emissions to the atmosphere, and a role for DCM as a microbial energy source in critical zone environments. The findings imply that the global DCM flux might be far greater than emission measurements suggest.</p><sec><title>Importance</title><p>Dichloromethane (DCM) is an increasing threat to stratospheric ozone with both anthropogenic and natural emission sources. Anaerobic bacterial metabolism of DCM has not yet been taken into consideration as a factor in the global DCM cycle. The discovery of the <italic>mec</italic> gene cassette associated with anaerobic bacterial DCM metabolism and its widespread distribution in environmental systems highlight a strong attenuation potential for DCM. Knowledge of the <italic>mec</italic> cassette offers new opportunities to delineate DCM sources, enables more robust estimates of DCM fluxes, supports refined DCM emission modeling and simulation of the stratospheric ozone layer, reveals a novel, ubiquitous C<sub>1</sub> carbon metabolic system, and provides prognostic and diagnostic tools supporting bioremediation of groundwater aquifers impacted by DCM.</p></sec>', 'posted': '2021-08-31'}, {'doi': '10.1101/2021.08.30.458233', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Temporal inhibition of chromatin looping and enhancer accessibility during neuronal remodeling', 'authors': \"[{'name': 'Dahong Chen'}, {'name': 'Catherine E. McManus'}, {'name': 'Behram Radmanesh'}, {'name': 'Leah H. Matzat'}, {'name': 'Elissa P. Lei'}]\", 'abstract': '<p>During development, looping of an enhancer to a promoter is frequently observed in conjunction with temporal and tissue-specific transcriptional activation. The chromatin insulator-associated protein Shep promotes <italic>Drosophila</italic> post-mitotic neuronal remodeling by repressing transcription of master developmental regulators, such as <italic>brain tumor</italic> (<italic>brat</italic>), specifically in maturing neurons. Since insulator proteins can promote looping, we hypothesized that Shep antagonizes <italic>brat</italic> promoter interaction with an as yet unidentified enhancer. Using chromatin conformation capture and reporter assays, we identified two novel enhancer regions that increase in looping frequency with the <italic>brat</italic> promoter specifically in pupal brains after Shep depletion. The <italic>brat</italic> promoters and enhancers function independently of Shep, ruling out direct repression of these elements. Moreover, ATAC-seq in isolated neurons demonstrated that Shep restricts chromatin accessibility of a key <italic>brat</italic> enhancer as well as other enhancers genome-wide in remodeling pupal but not larval neurons. These enhancers are enriched for chromatin targets of Shep and are located at Shep-inhibited genes, suggesting direct Shep inhibition of enhancer accessibility and gene expression during neuronal remodeling. Our results provide evidence for temporal regulation of chromatin looping and enhancer accessibility during neuronal maturation.</p>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.09.04.458950', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Versatile, do-it-yourself, low-cost spinning disk confocal microscope', 'authors': \"[{'name': 'Aaron R. Halpern'}, {'name': 'Min Yen Lee'}, {'name': 'Marco D. Howard'}, {'name': 'Marcus A. Woodworth'}, {'name': 'Philip R. Nicovich'}, {'name': 'Joshua C. Vaughan'}]\", 'abstract': '<p>Confocal microscopy is an invaluable tool for 3D imaging of biological specimens, however, accessibility is often limited to core facilities due to the high cost of the hardware. We describe an inexpensive do-it-yourself (DIY) spinning disk confocal microscope (SDCM) module based on a commercially fabricated chromium photomask that can be added on to a laser-illuminated epifluorescence microscope. The SDCM achieves strong performance across a wide wavelength range (~400-800 nm) as demonstrated through a series of biological imaging applications that include conventional microscopy (immunofluorescence, small-molecule stains, and fluorescence in situ hybridization) and super-resolution microscopy (single-molecule localization microscopy and expansion microscopy). This low-cost and simple DIY SDCM is well-documented and should help increase accessibility to confocal microscopy for researchers.</p>', 'posted': '2021-09-04'}, {'doi': '10.1101/2021.08.30.457871', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'The Retrieval-related Anterior shift is Moderated by Age and Correlates with Memory Performance', 'authors': \"[{'name': 'Sabina Srokova'}, {'name': 'Paul F Hill'}, {'name': 'Michael D Rugg'}]\", 'abstract': '<p>Recent research suggests that episodic memory is associated with systematic differences in the localization of neural activity observed during memory encoding and retrieval. The retrieval-related anterior shift is a phenomenon whereby the retrieval of a stimulus event (e.g., a scene image) is associated with a peak neural response which is localized more anteriorly than the response elicited when the stimulus is experienced directly. Here, we examine whether the magnitude of the anterior shift, i.e., the distance between encoding- and retrieval-related response peaks, is moderated by age, and also whether the shift is associated with memory performance. Younger and older human subjects of both sexes underwent fMRI as they completed encoding and retrieval tasks on word-face and word-scene pairs. We localized peak scene- and face-selectivity for each individual participant within the face-selective precuneus (PCU) and in three scene-selective (parahippocampal place area [PPA], medial place area [MPA], occipital place area [OPA]) regions of interest (ROIs). In line with recent findings, we identified an anterior shift in PPA and OPA in both age groups and, in older adults only, in MPA and PCU also. Of importance, the magnitude of the anterior shift was larger in older than in younger adults. The shift within the PPA exhibited an age-invariant across-participant negative correlation with source memory performance, such that a smaller displacement between encoding- and retrieval-related neural activity was associated with better performance. These findings provide novel insights into the functional significance of the anterior shift, especially in relation to memory decline in older age.</p>', 'posted': '2021-08-31'}, {'doi': '10.1101/2021.09.02.458731', 'tweets': 5, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': False, 'title': 'Macrophages in the uterus are functionally specialised and continually replenished from the circulation', 'authors': \"[{'name': 'Nicholas A Scott'}, {'name': 'Lamiya Mohiyiddeen'}, {'name': 'Livia Lacerda Mariano'}, {'name': 'Peter T Ruane'}, {'name': 'John D Aplin'}, {'name': 'Olivia C Moran'}, {'name': 'Daniel R Brison'}, {'name': 'Molly A Ingersoll'}, {'name': 'Elizabeth R Mann'}]\", 'abstract': '<p>Macrophages are innate immune cells that fight infection but also regulate tissue regeneration and remodelling. In the uterus, although tissue remodelling is essential for establishment and maintenance of pregnancy, the specialisation of macrophages is not well characterised compared to other mucosal tissues. Here we show that uterine macrophages are functionally specialised, expressing multiple markers of alternative activation associated with tissue remodelling and repair, and responding more highly to the type 2 cytokine IL-4 than other mucosal tissue macrophages. Uterine macrophages were continuously replenished from circulating bone marrow-derived CCR2+ monocytes that fluctuated dramatically in number throughout the reproductive cycle, and had properties distinct from the macrophages that they became, including differential responses to microbial stimulation. Importantly, many of these properties of uterine monocytes and macrophages were conserved between mice and humans. These findings further our understanding of immune regulation of uterine tissue integrity and have important implications for differences in immune responses to infections at different phases of the reproductive cycle.</p>', 'posted': '2021-09-03'}, {'doi': '10.1101/2021.09.01.458520', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Early post-infection treatment of SARS-CoV-2 infected macaques with human convalescent plasma with high neutralizing activity reduces lung inflammation', 'authors': \"[{'name': 'Koen K.A. Van Rompay'}, {'name': 'Katherine J. Olstad'}, {'name': 'Rebecca L. Sammak'}, {'name': 'Joseph Dutra'}, {'name': 'Jennifer K. Watanabe'}, {'name': 'Jodie L. Usachenko'}, {'name': 'Ramya Immareddy'}, {'name': 'Jamin W. Roh'}, {'name': 'Anil Verma'}, {'name': 'Yashavanth Shaan Lakshmanappa'}, {'name': 'Brian A. Schmidt'}, {'name': 'Clara Di Germanio'}, {'name': 'Nabeela Rizvi'}, {'name': 'Mars Stone'}, {'name': 'Graham Simmons'}, {'name': 'Larry J. Dumont'}, {'name': 'A. Mark Allen'}, {'name': 'Sarah Lockwood'}, {'name': 'Rachel E. Pollard'}, {'name': 'Rafael Ramiro de Assis'}, {'name': 'JoAnn L. Yee'}, {'name': 'Peter B. Nham'}, {'name': 'Amir Ardeshir'}, {'name': 'Jesse D. Deere'}, {'name': 'Jean Patterson'}, {'name': 'Aarti Jain'}, {'name': 'Philip L. Felgner'}, {'name': 'Smita S. Iyer'}, {'name': 'Dennis J. Hartigan-O’Connor'}, {'name': 'Michael P. Busch'}, {'name': 'J. Rachel Reader'}]\", 'abstract': '<p>Early in the SARS-CoV-2 pandemic, there was a high level of optimism based on observational studies and small controlled trials that treating hospitalized patients with convalescent plasma from COVID-19 survivors (CCP) would be an important immunotherapy. However, as more data from controlled trials became available, the results became disappointing, with at best moderate evidence of efficacy when CCP with high titers of neutralizing antibodies was used early in infection. To better understand the potential therapeutic efficacy of CCP, and to further validate SARS-CoV-2 infection of macaques as a reliable animal model for testing such strategies, we inoculated 12 adult rhesus macaques with SARS-CoV-2 by intratracheal and intranasal routes. One day later, 8 animals were infused with pooled human CCP with a high titer of neutralizing antibodies (RVPN NT<sub>50</sub> value of 3,003), while 4 control animals received normal human plasma. Animals were monitored for 7 days. Animals treated with CCP had detectable levels of antiviral antibodies after infusion. In comparison to the control animals, they had similar levels of virus replication in the upper and lower respiratory tract, but had significantly reduced interstitial pneumonia, as measured by comprehensive lung histology. By highlighting strengths and weaknesses, data of this study can help to further optimize nonhuman primate models to provide proof-of-concept of intervention strategies, and guide the future use of convalescent plasma against SARS-CoV-2 and potentially other newly emerging respiratory viruses.</p><sec><title>Author summary</title><p>The results of treating SARS-CoV-2 infected hospitalized patients with COVID-19 convalescent plasma (CCP), collected from survivors of natural infection, have been disappointing. The available data from various studies indicate at best moderate clinical benefits only when CCP with high titer of neutralizing antibodies was infused early in infection. The macaque model of SARS-CoV-2 infection can be useful to gain further insights in the value of CCP therapy. In this study, animals were infected with SARS-CoV-2 and the next day, were infused with pooled human convalescent plasma, selected to have a very high titer of neutralizing antibodies. While administration of CCP did not result in a detectable reduction in virus replication in the respiratory tract, it significantly reduced lung inflammation. These data, combined with the results of monoclonal antibody studies, emphasize the need to use products with high titers of neutralizing antibodies, and guide the future development of CCP-based therapies.</p></sec>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.30.458145', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Genomics', 'covid': False, 'title': 'Dicer-to-Argonaute switch controls biogenesis of oncogenic miRNA', 'authors': \"[{'name': 'L. Winchester'}, {'name': 'L. van Bijsterveldt'}, {'name': 'A. Dhawan'}, {'name': 'S. Wigfield'}, {'name': 'C. Triantafyllidis'}, {'name': 'S. Haider'}, {'name': 'A. McIntyre'}, {'name': 'T.C. Humphrey'}, {'name': 'A.L. Harris'}, {'name': 'F.M. Buffa'}]\", 'abstract': '<p>miRNAs are post-transcriptional regulators of gene expression, controlling biological processes from development to pathogenesis. We asked whether the reshaped functional miRNA landscape in cancers is driven by altered transcription of its precursors, or altered biogenesis and maturation of miRNAs. Integrated analysis of genomic and transcriptomic data in 9,111 samples across 10 cancer types and healthy tissues revealed a recurrent genomic switch from DICER-dependent to non-canonical Argonaute-mediated, DICER-independent, miRNA biogenesis. Experimental validation in AGO2-amplified clinical samples and cancer cell lines confirmed that canonical miRNAs can undergo maturation in a DICER-independent manner, and that elevated Argonaute levels promote selective maturation of the oncogenic miR-106b/25 cluster as shown by the altered ratio of mature miRNA to immature pri-miRNA levels. The preferential maturation of these oncogenic miRNAs, whose processing bypasses DICER1, promotes cancer progression and predicts poor prognosis. This highlights the evolution of non-canonical AGO2-dependent oncomiR processing as a novel driver pathway in cancer.</p>', 'posted': '2021-08-31'}, {'doi': '10.1101/2021.09.01.458516', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Immunology', 'covid': True, 'title': 'Absence of severe COVID-19 in patients with clonal mast cells activation disorders: effective anti-SARS-CoV-2 immune response', 'authors': \"[{'name': 'Julien Rossignol'}, {'name': 'Amani Ouedrani'}, {'name': 'Cristina Bulai Livideanu'}, {'name': 'Stéphane Barete'}, {'name': 'Louis Terriou'}, {'name': 'David Launay'}, {'name': 'Richard Lemal'}, {'name': 'Celine Greco'}, {'name': 'Laurent Frenzel'}, {'name': 'Cecile Meni'}, {'name': 'Christine Bodemere-Skandalis'}, {'name': 'Laura Polivka'}, {'name': 'Anne-Florence Collange'}, {'name': 'Hassiba Hachichi'}, {'name': 'Sonia Bouzourine'}, {'name': 'Djazira Nait Messaoud'}, {'name': 'Mathilde Negretto'}, {'name': 'Laurence Vendrame'}, {'name': 'Marguerite Jambou'}, {'name': 'Marie Gousseff'}, {'name': 'Stéphane Durupt'}, {'name': 'Jean-Christophe Lega'}, {'name': 'Jean-Marc Durand'}, {'name': 'Caroline Gaudy'}, {'name': 'Gandhi Damaj'}, {'name': 'Marie-Pierre Gourin'}, {'name': 'Mohamed Hamidou'}, {'name': 'Laurence Bouillet'}, {'name': 'Edwige Le Mouel'}, {'name': 'Alexandre Maria'}, {'name': 'Patricia Zunic'}, {'name': 'Quentin Cabrera'}, {'name': 'Denis Vincent'}, {'name': 'Christian Lavigne'}, {'name': 'Etienne Riviere'}, {'name': 'Clement Gourguechon'}, {'name': 'Anne Brignier'}, {'name': 'Ludovic Lhermitte'}, {'name': 'Thierry Jo Molina'}, {'name': 'Julie Bruneau'}, {'name': 'Julie Agopian'}, {'name': 'Patrice Dubreuil'}, {'name': 'Dana Ranta'}, {'name': 'Alexandre Mania'}, {'name': 'Michel Arock'}, {'name': 'Isabelle Staropoli'}, {'name': 'Olivier Tournilhac'}, {'name': 'Olivier Lortholary'}, {'name': 'Olivier Schwartz'}, {'name': 'Lucienne Chatenoud'}, {'name': 'Olivier Hermine'}]\", 'abstract': '<p>Mast cells are key actors of innate immunity and Th2 adaptive immune response which counterbalance Th1 response, critical for anti-viral immunity. Clonal Mast Cells Activation Disorders (cMCADs) such as mastocytosis and clonal mast cells activation syndrome are characterized by an abnormal mast cells accumulation and/or activation. No data have been published on the anti-viral immune response of patients with cMCADs. The aims of the study were to collected, in a comprehensive way, outcomes of cMCADs patients who experienced a biologically-proven COVID-19 and to characterize both anti-endemic coronaviruses and specific anti-SARS-CoV-2 immune responses in these patients. Clinical follow-up and outcome data were collected prospectively for one year within the French rare disease network CEREMAST encompassing patients from all over the country. Anti-SARS-CoV-2 and anti-endemic coronaviruses specific T-cells were assessed with an enzyme-linked immunospot assay (EliSpot) and anti-SARS-CoV-2 humoral response with dosage of circulating levels of specific IgG, IgA and neutralizing antibodies. Overall, 32 cMCADs patients were identified. None of them required non-invasive or mechanical ventilation; two patients were hospitalized to receive oxygen and steroid therapy. In 21 patients, a characterization of the SARS-CoV-2-specific immune response has been performed. A majority of patients showed a high proportion of circulating SARS-CoV-2-specific interferon (IFN)-γ producing T-cells and high levels of anti-Spike IgG antibodies with neutralizing activity. In addition, no defects in anti-endemic coronaviruses responses were found in patients with cMCADs compared to non-cMCADs controls. Patients with cMCADs frequently showed a spontaneous IFN-γ T-cell production in absence of any stimulation that correlated with circulating basal tryptase levels, a marker of mast cells burden. These findings underscore that patients with cMCADs might be not at risk of severe COVID-19 and the spontaneous IFN-γ production might explain this observation.</p><sec><title>Author Summary</title><p>Mast cells are immune cells involved in many biological processes including the anti-microbial response. However, previous studies suggest that mast cells may have a detrimental role in the response against viruses such as SARS-CoV-2, responsible for COVID-19. When a mutation occurs in mast cells, it can lead to a group of diseases called clonal mast cells activation disorders (cMCADs), characterized by deregulated activation of these cells. Hence, patients with cMCADs might be more susceptible to severe COVID-19 than general population.</p><p>We therefore conducted a 1-year study in France to collect data from all cMCADs patients included in the CEREMAST rare disease French network and who experienced COVID-19. Interestingly, we did not find any severe COVID-19 (i.e. requiring non-invasive or mechanical ventilation) in spite of well-known risk factors for severe COVID-19 in a part of cMCADs patients.</p><p>We then have studied the immune response against SARS-CoV-2 and other endemic coronaviruses in these patients. We did not observe any abnormalities in the immune response either at the level of T and B lymphocytes. These findings underscore that these patients might not be at risk of severe COVID-19 as one might have feared.</p></sec>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.30.457919', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Beyond RuBisCO: Convergent molecular evolution of multiple chloroplast genes in C4 plants', 'authors': \"[{'name': 'Claudio Casola'}, {'name': 'Jingjia Li'}]\", 'abstract': '<sec><title>Background</title><p>The recurrent evolution of the C<sub>4</sub> photosynthetic pathway in angiosperms represents one of the most extraordinary examples of convergent evolution of a complex trait. Comparative genomic analyses have unveiled some of the molecular changes associated with the C<sub>4</sub> pathway. For instance, several key enzymes involved in the transition from C<sub>3</sub> to C<sub>4</sub> photosynthesis have been found to share convergent amino acid replacements along C<sub>4</sub> lineages. However, the extent of convergent replacements potentially associated with the emergence of C<sub>4</sub> plants remains to be fully assessed. Here, we introduced a robust empirical approach to test molecular convergence along a phylogeny including multiple C<sub>3</sub> and C<sub>4</sub> taxa. By analyzing proteins encoded by chloroplast genes, we tested if convergent replacements occurred more frequently than expected in C<sub>4</sub> lineages compared to C<sub>3</sub> lineages. Furthermore, we sought to determine if convergent evolution occurred in multiple chloroplast proteins beside the well-known case of the large RuBisCO subunit encoded by the chloroplast gene <italic>rbcL</italic>.</p></sec><sec><title>Methods</title><p>Our study was based on the comparative analysis of 43 C<sub>4</sub> and 21 C<sub>3</sub> grass species belonging to the PACMAD clade, a focal taxonomic group in many investigations of C<sub>4</sub> evolution. We first used protein sequences of 67 orthologous chloroplast genes to build an accurate phylogeny of these species. Then, we inferred amino acid replacements along 13 C<sub>4</sub> lineages and 9 C<sub>3</sub> lineages using reconstructed protein sequences of their ancestral branches, corresponding to the most recent common ancestor of each lineage. Pairwise comparisons between ancestral branches allowed us to identify both convergent and divergent amino acid replacements between C<sub>4</sub>-C<sub>4</sub>, C<sub>3</sub>-C<sub>3</sub> and C<sub>3</sub>-C<sub>4</sub> lineages.</p></sec><sec><title>Results</title><p>The reconstructed phylogenetic tree of 64 PACMAD grasses was characterized by strong supports in all nodes used for analyses of convergence. We identified 217 convergent replacements and 201 divergent replacements in 45/67 chloroplast proteins in both C<sub>4</sub> and C<sub>3</sub> ancestral branches. Pairs of C<sub>4</sub>-C<sub>4</sub> ancestral branches showed higher levels of convergent replacements than C<sub>3</sub>-C<sub>3</sub> and C<sub>3</sub>-C<sub>4</sub> pairs. Furthermore, we found that more proteins shared unique convergent replacements in C<sub>4</sub> lineages, with both RbcL and RpoC1 (the RNA polymerase beta’ subunit 1) showing a significantly higher convergent/divergent replacements ratio in C<sub>4</sub> branches. Notably, significantly more C<sub>4</sub>-C<sub>4</sub> pairs of ancestral branches showed higher numbers of convergent vs. divergent replacements than C<sub>3</sub>-C<sub>3</sub> and C<sub>3</sub>-C<sub>4</sub> pairs. Our results demonstrated that, in the PACMAD clade, C<sub>4</sub> grasses experienced higher levels of molecular convergence than C<sub>3</sub> species across multiple chloroplast genes. These findings have important implications for both our understanding of the evolution of photosynthesis and the goal of engineering improved crop varieties that integrates components of the C<sub>4</sub> pathway.</p></sec>', 'posted': '2021-08-31'}, {'doi': '10.1101/2021.09.01.458571', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Neuroscience', 'covid': False, 'title': 'Right fronto-parietal networks mediate the neurocognitive benefits of enriched environments.', 'authors': \"[{'name': 'Meadhbh B Brosnan'}, {'name': 'Nir Shalev'}, {'name': 'Jivesh Ramduny'}, {'name': 'Stamatios Sotiropoulos'}, {'name': 'Magdalena Chechlacz'}]\", 'abstract': '<p>Exposure to enriched environments (EE) throughout a lifetime, providing so called reserve, protects against cognitive decline in later years. It has been hypothesised that high levels of alertness necessitated by EE might strengthen the right fronto-parietal networks (FPN) to facilitate this neurocognitive resilience. We have previously shown that EE offset age-related deficits in selective attention by preserving grey matter within right fronto-parietal regions. Here, using neurite orientation dispersion and density imaging (NODDI), we examined the relationship between EE, microstructural properties of fronto-parietal white matter association pathways (three branches of the superior longitudinal fasciculus, SLF), structural brain health (atrophy), and attention (alertness, orienting and executive control) in a group of older adults. We show that EE is associated with a lower orientation dispersion index (ODI) within the right SLF1 which in turn mediates the relationship between EE and alertness, as well as grey- and white-matter atrophy. This suggests that EE may induce white matter plasticity (and prevent age-related dispersion of axons) within the right FPN to facilitate the preservation of neurocognitive health in later years.</p>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.31.458377', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Biochemistry', 'covid': False, 'title': 'Multicenter reverse-phase protein array data integration', 'authors': \"[{'name': 'Leanne de Koning'}, {'name': 'Stephan Bernhardt'}, {'name': 'Kenneth G. Macleod'}, {'name': 'Bérengère Ouine'}, {'name': 'Aurélie Cartier'}, {'name': 'Vonick Sibut'}, {'name': 'Neil O. Carragher'}, {'name': 'Ulrike Korf'}, {'name': 'Bryan Serrels'}, {'name': 'Adam Byron'}]\", 'abstract': '<p>Among the technologies available for protein biomarker discovery and validation, reverse-phase protein array (RPPA) benefits from unequalled sample throughput. Panels of high-quality antibodies enable the quantification by RPPA of protein abundance and posttranslational modifications in biological specimens with high precision and sensitivity. Incorporation of RPPA technology into clinical and drug development pipelines requires robust assays that generate reproducible results across multiple laboratories. We implemented the first international multicenter pilot study to investigate RPPA workflow variability. We characterized the proteomic responses of a series of breast cancer cells to two cancer drugs. This analysis quantified 86,832 sample spots, representing 108 biological samples, arrayed at three independent RPPA platforms. This unique integrated set of data is publicly available as a resource to the proteomic and cancer research communities to catalyse further analysis and investigation. We anticipate that this dataset will form a reference for the comparison of RPPA workflows and reagents, which can be expanded in the future, and will aid the identification of platform-robust treatment-marker antigens in breast cancer cells.</p><p><fig id=\"ufig1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\"><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"458377v1_ufig1\" position=\"float\" orientation=\"portrait\" /></fig></p>', 'posted': '2021-08-31'}, {'doi': '10.1101/2021.09.02.458699', 'tweets': 4, 'archive': 'bioRxiv', 'subject-area': 'Bioinformatics', 'covid': False, 'title': 'PatchMAN docking: Modeling peptide-protein interactions in the context of the receptor surface', 'authors': \"[{'name': 'Alisa Khramushin'}, {'name': 'Tomer Tsaban'}, {'name': 'Julia K Varga'}, {'name': 'Orly Avraham'}, {'name': 'Ora Schueler-Furman'}]\", 'abstract': '<p>Peptide docking can be perceived as a subproblem of protein-protein docking. However, due to the short length and flexible nature of peptides, many do not adopt one defined conformation prior to binding. Therefore, to tackle a peptide docking problem, not only the relative orientation between the two partners, but also the bound conformation of the peptide needs to be modeled. Traditional peptide-centered approaches use information about the peptide sequence to generate a representative conformer ensemble, which can then be rigid body docked to the receptor. Alternatively, one may look at this problem from the viewpoint of the receptor, namely that the protein surface defines the peptide bound conformation.We present PatchMAN (Patch-Motif AligNments), a novel peptide docking approach which uses structural motifs to map the receptor surface with backbone scaffolds extracted from protein structures. On a non-redundant set of protein-peptide complexes, starting from free receptor structures,  PatchMAN successfully models and identifies near-native peptide-protein complexes in 62% / 81% within 2.5Å / 5Å RMSD, with corresponding sampling in 81% / 100% of the cases, outperforming other approaches. PatchMAN leverages the observation that structural units of peptides with their binding pocket can be found not only within interfaces, but also within monomers. We show that the conformation of the bound peptide is sampled based on the structural context of the receptor only, without taking into account any sequence information. Beyond peptide docking, this approach opens exciting new avenues to study principles of peptide-protein association, and to the design of new peptide binders.</p>', 'posted': '2021-09-03'}, {'doi': '10.1101/2021.08.31.458345', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Biophysics', 'covid': False, 'title': 'Cryo-ET of HIV reveals Env positioning on Gag lattice and structural variation among Env trimers', 'authors': \"[{'name': 'Vidya Mangala Prasad'}, {'name': 'Daniel P. Leaman'}, {'name': 'Klaus N. Lovendahl'}, {'name': 'Mark A. Benhaim'}, {'name': 'Edgar A. Hodge'}, {'name': 'Michael B. Zwick'}, {'name': 'Kelly K. Lee'}]\", 'abstract': '<p>HIV-1 Env mediates viral entry into host cells and is the sole target for neutralizing antibodies. However, Env structure and organization in its native virion context has eluded detailed characterization. Here we used cryo-electron tomography to analyze Env in mature and immature HIV-1 particles. Immature particles showed distinct Env positioning relative to the underlying Gag lattice, providing insights into long-standing questions about Env incorporation. A 9.1Å sub-tomogram averaged reconstruction of virion-bound Env in conjunction with structural mass spectrometry revealed unexpected features, including a variable central core of the gp41 subunit, heterogeneous glycosylation between protomers plus a flexible stalk that allows Env tilting and variable exposure of neutralizing epitopes. Together, our results provide an integrative understanding of HIV assembly and structural variation in Env antigen presentation.</p>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.29.458141', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Molecular Biology', 'covid': False, 'title': 'Exploiting flow cytometry for the unbiased quantification of protein inclusions in Caenorhabditis elegans', 'authors': \"[{'name': 'Kristian Claesson'}, {'name': 'Yee Lian Chew'}, {'name': 'Heath Ecroyd'}]\", 'abstract': '<p>The aggregation of proteins into inclusions or plaques is a prominent hallmark of a diverse range of pathologies including neurodegenerative diseases. The quantification of such inclusions in <italic>Caenorhabditis elegans</italic> models of aggregation is usually achieved by fluorescence microscopy or other techniques involving biochemical fractionation of worm lysates. Here, we describe a simple and rapid flow cytometry-based approach that allows fluorescently-tagged inclusions to be enumerated in whole worm lysate in a quantitative and unbiased fashion. We demonstrate that this technique is applicable to multiple <italic>C. elegans</italic> models of aggregation and importantly, can be used to monitor the dynamics of inclusion formation in response to heat shock and during aging. This includes the characterisation of physicochemical properties of inclusions, such as their size, which may reveal how aggregate formation is distinct in different tissues or at different stages of pathology or aging. This new method can be used as a powerful technique for the medium- to high-throughput quantification of inclusions in future studies of genetic or chemical modulators of aggregation in <italic>C. elegans</italic>.</p>', 'posted': '2021-08-31'}, {'doi': '10.1101/2021.08.31.458283', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Cancer Biology', 'covid': False, 'title': 'Breast tumor IGF-1R regulates cell adhesion and metastasis: Alignment of mouse single cell and human breast cancer transcriptomics', 'authors': \"[{'name': 'Alison E. Obr'}, {'name': 'Yung-Jun Chang'}, {'name': 'Virginia Ciliento'}, {'name': 'Alexander Lemenze'}, {'name': 'Krystopher Maingrette'}, {'name': 'Joseph J. Bulatowicz'}, {'name': 'Quan Shang'}, {'name': 'Emily Gallagher'}, {'name': 'Derek LeRoith'}, {'name': 'Teresa L. Wood'}]\", 'abstract': '<p>The acquisition of a metastatic phenotype is the critical event that determines patient survival from breast cancer. Several receptor tyrosine kinases have functions both in promoting and inhibiting metastasis in breast tumors. Although the insulin-like growth factor 1 receptor (IGF-1R) has been considered a target for inhibition in breast cancer, low levels of IGF-1R expression are associated with worse overall patient survival. To determine how reduced IGF-1R impacts tumor phenotype, we used weighted correlation gene network analysis (WCGNA) of METABRIC patient data and identified gene modules specific to cell cycle, adhesion, and immune cell signaling inversely correlated with IGF-1R expression in human breast cancers. Integration of human patient data with data from mouse tumors revealed similar pathways necessary for promoting metastasis in basal-like tumors with reduced signaling or expression of the IGF-1R. Functional analyses revealed the basis for the enhanced metastatic phenotype including alterations in E- and P-cadherins.</p>', 'posted': '2021-09-01'}, {'doi': '10.1101/2021.08.30.458162', 'tweets': 3, 'archive': 'bioRxiv', 'subject-area': 'Evolutionary Biology', 'covid': False, 'title': 'Mutationathon: towards standardization in estimates of pedigree-based germline mutation rates', 'authors': \"[{'name': 'Lucie A. Bergeron'}, {'name': 'Søren Besenbacher'}, {'name': 'Tychele N. Turner'}, {'name': 'Cyril J. Versoza'}, {'name': 'Richard J. Wang'}, {'name': 'Alivia Lee Price'}, {'name': 'Ellie Armstrong'}, {'name': 'Meritxell Riera'}, {'name': 'Jedidiah Carlson'}, {'name': 'Hwei-yen Chen'}, {'name': 'Matthew W. Hahn'}, {'name': 'Kelley Harris'}, {'name': 'April Snøfrid Lo Natalie M Kleppe'}, {'name': 'Elora H. López-Nandam'}, {'name': 'Priya Moorjani'}, {'name': 'Susanne P. Pfeifer'}, {'name': 'George P. Tiley'}, {'name': 'Anne D. Yoder'}, {'name': 'Guojie Zhang'}, {'name': 'Mikkel H. Schierup'}]\", 'abstract': '<p>In the past decade, several studies have estimated the human per-generation germline mutation rate using large pedigrees. More recently, estimates for various non-human species have been published. However, methodological differences among studies in detecting germline mutations and estimating mutation rates make direct comparisons difficult. Here, we describe the many different steps involved in estimating pedigree-based mutation rates, including sampling, sequencing, mapping, variant calling, filtering, and how to appropriately account for false-positive and false-negative rates. For each step, we review the different methods and parameter choices that have been used in the recent literature. Additionally, we present the results from a “Mutationathon”, a competition organized among five research labs to compare germline mutation rate estimates for a single pedigree of rhesus macaques. We report almost a two-fold variation in the final estimated rate among groups using different post-alignment processing, calling, and filtering criteria and provide details into the sources of variation across studies. Though the difference among estimates is not statistically significant, this discrepancy emphasizes the need for standardized methods in mutation rate estimations and the difficulty in comparing rates from different studies. Finally, this work aims to provide guidelines for computational and statistical benchmarks for future studies interested in identifying germline mutations from pedigrees.</p>', 'posted': '2021-08-31'}, {'doi': '10.1101/2021.09.02.21262965', 'tweets': 3, 'archive': 'medRxiv', 'subject-area': 'Intensive Care and Critical Care Medicine', 'covid': True, 'title': 'Whole genome sequencing identifies multiple loci for critical illness caused by COVID-19', 'authors': \"[{'name': 'Athanasios Kousathanas'}, {'name': 'Erola Pairo-Castineira'}, {'name': 'Konrad Rawlik'}, {'name': 'Alex Stuckey'}, {'name': 'Christopher A Odhams'}, {'name': 'Susan Walker'}, {'name': 'Clark D Russell'}, {'name': 'Tomas Malinauskas'}, {'name': 'Jonathan Millar'}, {'name': 'Katherine S Elliott'}, {'name': 'Fiona Griffiths'}, {'name': 'Wilna Oosthuyzen'}, {'name': 'Kirstie Morrice'}, {'name': 'Sean Keating'}, {'name': 'Bo Wang'}, {'name': 'Daniel Rhodes'}, {'name': 'Lucija Klaric'}, {'name': 'Marie Zechner'}, {'name': 'Nick Parkinson'}, {'name': 'Andrew D. Bretherick'}, {'name': 'Afshan Siddiq'}, {'name': 'Peter Goddard'}, {'name': 'Sally Donovan'}, {'name': 'David Maslove'}, {'name': 'Alistair Nichol'}, {'name': 'Malcolm G Semple'}, {'name': 'Tala Zainy'}, {'name': 'Fiona Maleady-Crowe'}, {'name': 'Linda Todd'}, {'name': 'Shahla Salehi'}, {'name': 'Julian Knight'}, {'name': 'Greg Elgar'}, {'name': 'Georgia Chan'}, {'name': 'Prabhu Arumugam'}, {'name': 'Tom A Fowler'}, {'name': 'Augusto Rendon'}, {'name': 'Manu Shankar-Hari'}, {'name': 'Charlotte Summers'}, {'name': 'Paul Elliott'}, {'name': 'Jian Yang'}, {'name': 'Yang Wu'}, {'name': 'Angie Fawkes'}, {'name': 'Lee Murphy'}, {'name': 'Kathy Rowan'}, {'name': 'Chris P Ponting'}, {'name': 'Veronique Vitart'}, {'name': 'James F Wilson'}, {'name': 'Richard H Scott'}, {'name': 'Sara Clohisey'}, {'name': 'Loukas Moutsianas'}, {'name': 'Andy Law'}, {'name': 'Mark J Caulfield'}, {'name': 'J. Kenneth Baillie'}, {'name': ' '}, {'name': ' '}, {'name': ' '}]\", 'abstract': '<p>Critical illness in COVID-19 is caused by inflammatory lung injury, mediated by the host immune system. We and others have shown that host genetic variation influences the development of illness requiring critical care<sup>1</sup> or hospitalisation<sup>2;3;4</sup> following SARS-Co-V2 infection. The GenOMICC (Genetics of Mortality in Critical Care) study is designed to compare genetic variants in critically-ill cases with population controls in order to find underlying disease mechanisms.</p><p>Here, we use whole genome sequencing and statistical fine mapping in 7,491 critically-ill cases compared with 48,400 population controls to discover and replicate 22 independent variants that significantly predispose to life-threatening COVID-19. We identify 15 new independent associations with critical COVID-19, including variants within genes involved in interferon signalling (<italic>IL10RB, PLSCR1</italic>), leucocyte differentiation (<italic>BCL11A</italic>), and blood type antigen secretor status (<italic>FUT2</italic>). Using transcriptome-wide association and colocalisation to infer the effect of gene expression on disease severity, we find evidence implicating expression of multiple genes, including reduced expression of a membrane flippase (<italic>ATP11A</italic>), and increased mucin expression (<italic>MUC1</italic>), in critical disease.</p><p>We show that comparison between critically-ill cases and population controls is highly efficient for genetic association analysis and enables detection of therapeutically-relevant mechanisms of disease. Therapeutic predictions arising from these findings require testing in clinical trials.</p>', 'posted': '2021-09-02'}]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>tweets</th>\n",
       "      <th>archive</th>\n",
       "      <th>subject-area</th>\n",
       "      <th>covid</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2021.08.31.458411</td>\n",
       "      <td>34</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Plant Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>An Arabidopsis root phloem pole cell atlas rev...</td>\n",
       "      <td>[{'name': 'Sofia Otero'}, {'name': 'Iris Sevil...</td>\n",
       "      <td>&lt;p&gt;Single cell sequencing has recently allowed...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2021.08.31.21262943</td>\n",
       "      <td>29</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Public and Global Health</td>\n",
       "      <td>True</td>\n",
       "      <td>Ensemble forecast of COVID-19 in Karnataka for...</td>\n",
       "      <td>[{'name': 'Sashikumaar Ganesan'}, {'name': 'De...</td>\n",
       "      <td>&lt;p&gt;We present an ensemble forecast for Wave-3 ...</td>\n",
       "      <td>2021-09-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2021.09.03.458877</td>\n",
       "      <td>27</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Pharmacology and Toxicology</td>\n",
       "      <td>True</td>\n",
       "      <td>Discovery of highly potent pancoronavirus fusi...</td>\n",
       "      <td>[{'name': 'Francesca Curreli'}, {'name': 'Shah...</td>\n",
       "      <td>&lt;p&gt;We report the discovery of a series of benz...</td>\n",
       "      <td>2021-09-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2021.08.30.458146</td>\n",
       "      <td>24</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>The control of transcriptional memory by stabl...</td>\n",
       "      <td>[{'name': 'Maëlle Bellec'}, {'name': 'Jérémy D...</td>\n",
       "      <td>&lt;p&gt;To maintain cellular identities during deve...</td>\n",
       "      <td>2021-08-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2021.08.31.458428</td>\n",
       "      <td>9</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Characterisation of a nucleo-adhesome</td>\n",
       "      <td>[{'name': 'Adam Byron'}, {'name': 'Billie G. C...</td>\n",
       "      <td>&lt;p&gt;In addition to central functions in cell ad...</td>\n",
       "      <td>2021-08-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2021.08.29.458102</td>\n",
       "      <td>8</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Exo1-protected DNA nicks direct crossover form...</td>\n",
       "      <td>[{'name': 'Michael Gioia'}, {'name': 'Lisette ...</td>\n",
       "      <td>&lt;p&gt;In most sexually reproducing organisms cros...</td>\n",
       "      <td>2021-08-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10.1101/2021.09.01.458557</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Functional fluorescently-tagged human profilin</td>\n",
       "      <td>[{'name': 'Morgan L. Pimm'}, {'name': 'Xinbei ...</td>\n",
       "      <td>&lt;p&gt;Profilin is an essential regulator of actin...</td>\n",
       "      <td>2021-09-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10.1101/2021.08.30.458230</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Disentangling the flow of signals between popu...</td>\n",
       "      <td>[{'name': 'Evren Gokcen'}, {'name': 'Anna I. J...</td>\n",
       "      <td>&lt;p&gt;Technological advances have granted the abi...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2021.08.30.458226</td>\n",
       "      <td>7</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>De Novo Prediction of RNA 3D Structures with D...</td>\n",
       "      <td>[{'name': 'Julius Ramakers'}, {'name': 'Christ...</td>\n",
       "      <td>&lt;p&gt;We present a Deep Learning approach to pred...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10.1101/2021.08.29.21262789</td>\n",
       "      <td>7</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Epidemiology</td>\n",
       "      <td>True</td>\n",
       "      <td>Vaccines alone are no silver bullets: a modeli...</td>\n",
       "      <td>[{'name': 'Dhesi Baha Raja'}, {'name': 'Nur As...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The computer ...</td>\n",
       "      <td>2021-08-30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.1101/2021.08.31.458373</td>\n",
       "      <td>6</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>A Cytosolic Reductase Pathway is Required for ...</td>\n",
       "      <td>[{'name': 'Marcel van Lith'}, {'name': 'Marie ...</td>\n",
       "      <td>&lt;p&gt;N-linked glycosylation of proteins entering...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2021.08.31.21262937</td>\n",
       "      <td>6</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Psychiatry and Clinical Psychology</td>\n",
       "      <td>False</td>\n",
       "      <td>Impairments in goal-directed action and revers...</td>\n",
       "      <td>[{'name': 'Shuichi Suetani'}, {'name': 'Andrea...</td>\n",
       "      <td>&lt;p&gt;Cognitive impairments in psychosis are one ...</td>\n",
       "      <td>2021-09-02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10.1101/2021.08.30.458270</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Microbiology</td>\n",
       "      <td>False</td>\n",
       "      <td>Global distribution of anaerobic dichlorometha...</td>\n",
       "      <td>[{'name': 'Robert W. Murdoch'}, {'name': 'Gao ...</td>\n",
       "      <td>&lt;p&gt;Anthropogenic activities and natural proces...</td>\n",
       "      <td>2021-08-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10.1101/2021.08.30.458233</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Temporal inhibition of chromatin looping and e...</td>\n",
       "      <td>[{'name': 'Dahong Chen'}, {'name': 'Catherine ...</td>\n",
       "      <td>&lt;p&gt;During development, looping of an enhancer ...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10.1101/2021.09.04.458950</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Versatile, do-it-yourself, low-cost spinning d...</td>\n",
       "      <td>[{'name': 'Aaron R. Halpern'}, {'name': 'Min Y...</td>\n",
       "      <td>&lt;p&gt;Confocal microscopy is an invaluable tool f...</td>\n",
       "      <td>2021-09-04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10.1101/2021.08.30.457871</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>The Retrieval-related Anterior shift is Modera...</td>\n",
       "      <td>[{'name': 'Sabina Srokova'}, {'name': 'Paul F ...</td>\n",
       "      <td>&lt;p&gt;Recent research suggests that episodic memo...</td>\n",
       "      <td>2021-08-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.1101/2021.09.02.458731</td>\n",
       "      <td>5</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>False</td>\n",
       "      <td>Macrophages in the uterus are functionally spe...</td>\n",
       "      <td>[{'name': 'Nicholas A Scott'}, {'name': 'Lamiy...</td>\n",
       "      <td>&lt;p&gt;Macrophages are innate immune cells that fi...</td>\n",
       "      <td>2021-09-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10.1101/2021.09.01.458520</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Early post-infection treatment of SARS-CoV-2 i...</td>\n",
       "      <td>[{'name': 'Koen K.A. Van Rompay'}, {'name': 'K...</td>\n",
       "      <td>&lt;p&gt;Early in the SARS-CoV-2 pandemic, there was...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10.1101/2021.08.30.458145</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>False</td>\n",
       "      <td>Dicer-to-Argonaute switch controls biogenesis ...</td>\n",
       "      <td>[{'name': 'L. Winchester'}, {'name': 'L. van B...</td>\n",
       "      <td>&lt;p&gt;miRNAs are post-transcriptional regulators ...</td>\n",
       "      <td>2021-08-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2021.09.01.458516</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Immunology</td>\n",
       "      <td>True</td>\n",
       "      <td>Absence of severe COVID-19 in patients with cl...</td>\n",
       "      <td>[{'name': 'Julien Rossignol'}, {'name': 'Amani...</td>\n",
       "      <td>&lt;p&gt;Mast cells are key actors of innate immunit...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2021.08.30.457919</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Beyond RuBisCO: Convergent molecular evolution...</td>\n",
       "      <td>[{'name': 'Claudio Casola'}, {'name': 'Jingjia...</td>\n",
       "      <td>&lt;sec&gt;&lt;title&gt;Background&lt;/title&gt;&lt;p&gt;The recurrent...</td>\n",
       "      <td>2021-08-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2021.09.01.458571</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>False</td>\n",
       "      <td>Right fronto-parietal networks mediate the neu...</td>\n",
       "      <td>[{'name': 'Meadhbh B Brosnan'}, {'name': 'Nir ...</td>\n",
       "      <td>&lt;p&gt;Exposure to enriched environments (EE) thro...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2021.08.31.458377</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>False</td>\n",
       "      <td>Multicenter reverse-phase protein array data i...</td>\n",
       "      <td>[{'name': 'Leanne de Koning'}, {'name': 'Steph...</td>\n",
       "      <td>&lt;p&gt;Among the technologies available for protei...</td>\n",
       "      <td>2021-08-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10.1101/2021.09.02.458699</td>\n",
       "      <td>4</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>False</td>\n",
       "      <td>PatchMAN docking: Modeling peptide-protein int...</td>\n",
       "      <td>[{'name': 'Alisa Khramushin'}, {'name': 'Tomer...</td>\n",
       "      <td>&lt;p&gt;Peptide docking can be perceived as a subpr...</td>\n",
       "      <td>2021-09-03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2021.08.31.458345</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Biophysics</td>\n",
       "      <td>False</td>\n",
       "      <td>Cryo-ET of HIV reveals Env positioning on Gag ...</td>\n",
       "      <td>[{'name': 'Vidya Mangala Prasad'}, {'name': 'D...</td>\n",
       "      <td>&lt;p&gt;HIV-1 Env mediates viral entry into host ce...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2021.08.29.458141</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Molecular Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Exploiting flow cytometry for the unbiased qua...</td>\n",
       "      <td>[{'name': 'Kristian Claesson'}, {'name': 'Yee ...</td>\n",
       "      <td>&lt;p&gt;The aggregation of proteins into inclusions...</td>\n",
       "      <td>2021-08-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2021.08.31.458283</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Cancer Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Breast tumor IGF-1R regulates cell adhesion an...</td>\n",
       "      <td>[{'name': 'Alison E. Obr'}, {'name': 'Yung-Jun...</td>\n",
       "      <td>&lt;p&gt;The acquisition of a metastatic phenotype i...</td>\n",
       "      <td>2021-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10.1101/2021.08.30.458162</td>\n",
       "      <td>3</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>False</td>\n",
       "      <td>Mutationathon: towards standardization in esti...</td>\n",
       "      <td>[{'name': 'Lucie A. Bergeron'}, {'name': 'Søre...</td>\n",
       "      <td>&lt;p&gt;In the past decade, several studies have es...</td>\n",
       "      <td>2021-08-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10.1101/2021.09.02.21262965</td>\n",
       "      <td>3</td>\n",
       "      <td>medRxiv</td>\n",
       "      <td>Intensive Care and Critical Care Medicine</td>\n",
       "      <td>True</td>\n",
       "      <td>Whole genome sequencing identifies multiple lo...</td>\n",
       "      <td>[{'name': 'Athanasios Kousathanas'}, {'name': ...</td>\n",
       "      <td>&lt;p&gt;Critical illness in COVID-19 is caused by i...</td>\n",
       "      <td>2021-09-02</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            doi  tweets  archive  \\\n",
       "0     10.1101/2021.08.31.458411      34  bioRxiv   \n",
       "1   10.1101/2021.08.31.21262943      29  medRxiv   \n",
       "2     10.1101/2021.09.03.458877      27  bioRxiv   \n",
       "3     10.1101/2021.08.30.458146      24  bioRxiv   \n",
       "4     10.1101/2021.08.31.458428       9  bioRxiv   \n",
       "5     10.1101/2021.08.29.458102       8  bioRxiv   \n",
       "6     10.1101/2021.09.01.458557       7  bioRxiv   \n",
       "7     10.1101/2021.08.30.458230       7  bioRxiv   \n",
       "8     10.1101/2021.08.30.458226       7  bioRxiv   \n",
       "9   10.1101/2021.08.29.21262789       7  medRxiv   \n",
       "10    10.1101/2021.08.31.458373       6  bioRxiv   \n",
       "11  10.1101/2021.08.31.21262937       6  medRxiv   \n",
       "12    10.1101/2021.08.30.458270       5  bioRxiv   \n",
       "13    10.1101/2021.08.30.458233       5  bioRxiv   \n",
       "14    10.1101/2021.09.04.458950       5  bioRxiv   \n",
       "15    10.1101/2021.08.30.457871       5  bioRxiv   \n",
       "16    10.1101/2021.09.02.458731       5  bioRxiv   \n",
       "17    10.1101/2021.09.01.458520       4  bioRxiv   \n",
       "18    10.1101/2021.08.30.458145       4  bioRxiv   \n",
       "19    10.1101/2021.09.01.458516       4  bioRxiv   \n",
       "20    10.1101/2021.08.30.457919       4  bioRxiv   \n",
       "21    10.1101/2021.09.01.458571       4  bioRxiv   \n",
       "22    10.1101/2021.08.31.458377       4  bioRxiv   \n",
       "23    10.1101/2021.09.02.458699       4  bioRxiv   \n",
       "24    10.1101/2021.08.31.458345       3  bioRxiv   \n",
       "25    10.1101/2021.08.29.458141       3  bioRxiv   \n",
       "26    10.1101/2021.08.31.458283       3  bioRxiv   \n",
       "27    10.1101/2021.08.30.458162       3  bioRxiv   \n",
       "28  10.1101/2021.09.02.21262965       3  medRxiv   \n",
       "\n",
       "                                 subject-area  covid  \\\n",
       "0                               Plant Biology  False   \n",
       "1                    Public and Global Health   True   \n",
       "2                 Pharmacology and Toxicology   True   \n",
       "3                       Developmental Biology  False   \n",
       "4                                Cell Biology  False   \n",
       "5                           Molecular Biology  False   \n",
       "6                                Cell Biology  False   \n",
       "7                                Neuroscience  False   \n",
       "8                              Bioinformatics  False   \n",
       "9                                Epidemiology   True   \n",
       "10                               Cell Biology  False   \n",
       "11         Psychiatry and Clinical Psychology  False   \n",
       "12                               Microbiology  False   \n",
       "13                                   Genomics  False   \n",
       "14                                 Biophysics  False   \n",
       "15                               Neuroscience  False   \n",
       "16                                 Immunology  False   \n",
       "17                                 Immunology   True   \n",
       "18                                   Genomics  False   \n",
       "19                                 Immunology   True   \n",
       "20                       Evolutionary Biology  False   \n",
       "21                               Neuroscience  False   \n",
       "22                               Biochemistry  False   \n",
       "23                             Bioinformatics  False   \n",
       "24                                 Biophysics  False   \n",
       "25                          Molecular Biology  False   \n",
       "26                             Cancer Biology  False   \n",
       "27                       Evolutionary Biology  False   \n",
       "28  Intensive Care and Critical Care Medicine   True   \n",
       "\n",
       "                                                title  \\\n",
       "0   An Arabidopsis root phloem pole cell atlas rev...   \n",
       "1   Ensemble forecast of COVID-19 in Karnataka for...   \n",
       "2   Discovery of highly potent pancoronavirus fusi...   \n",
       "3   The control of transcriptional memory by stabl...   \n",
       "4               Characterisation of a nucleo-adhesome   \n",
       "5   Exo1-protected DNA nicks direct crossover form...   \n",
       "6      Functional fluorescently-tagged human profilin   \n",
       "7   Disentangling the flow of signals between popu...   \n",
       "8   De Novo Prediction of RNA 3D Structures with D...   \n",
       "9   Vaccines alone are no silver bullets: a modeli...   \n",
       "10  A Cytosolic Reductase Pathway is Required for ...   \n",
       "11  Impairments in goal-directed action and revers...   \n",
       "12  Global distribution of anaerobic dichlorometha...   \n",
       "13  Temporal inhibition of chromatin looping and e...   \n",
       "14  Versatile, do-it-yourself, low-cost spinning d...   \n",
       "15  The Retrieval-related Anterior shift is Modera...   \n",
       "16  Macrophages in the uterus are functionally spe...   \n",
       "17  Early post-infection treatment of SARS-CoV-2 i...   \n",
       "18  Dicer-to-Argonaute switch controls biogenesis ...   \n",
       "19  Absence of severe COVID-19 in patients with cl...   \n",
       "20  Beyond RuBisCO: Convergent molecular evolution...   \n",
       "21  Right fronto-parietal networks mediate the neu...   \n",
       "22  Multicenter reverse-phase protein array data i...   \n",
       "23  PatchMAN docking: Modeling peptide-protein int...   \n",
       "24  Cryo-ET of HIV reveals Env positioning on Gag ...   \n",
       "25  Exploiting flow cytometry for the unbiased qua...   \n",
       "26  Breast tumor IGF-1R regulates cell adhesion an...   \n",
       "27  Mutationathon: towards standardization in esti...   \n",
       "28  Whole genome sequencing identifies multiple lo...   \n",
       "\n",
       "                                              authors  \\\n",
       "0   [{'name': 'Sofia Otero'}, {'name': 'Iris Sevil...   \n",
       "1   [{'name': 'Sashikumaar Ganesan'}, {'name': 'De...   \n",
       "2   [{'name': 'Francesca Curreli'}, {'name': 'Shah...   \n",
       "3   [{'name': 'Maëlle Bellec'}, {'name': 'Jérémy D...   \n",
       "4   [{'name': 'Adam Byron'}, {'name': 'Billie G. C...   \n",
       "5   [{'name': 'Michael Gioia'}, {'name': 'Lisette ...   \n",
       "6   [{'name': 'Morgan L. Pimm'}, {'name': 'Xinbei ...   \n",
       "7   [{'name': 'Evren Gokcen'}, {'name': 'Anna I. J...   \n",
       "8   [{'name': 'Julius Ramakers'}, {'name': 'Christ...   \n",
       "9   [{'name': 'Dhesi Baha Raja'}, {'name': 'Nur As...   \n",
       "10  [{'name': 'Marcel van Lith'}, {'name': 'Marie ...   \n",
       "11  [{'name': 'Shuichi Suetani'}, {'name': 'Andrea...   \n",
       "12  [{'name': 'Robert W. Murdoch'}, {'name': 'Gao ...   \n",
       "13  [{'name': 'Dahong Chen'}, {'name': 'Catherine ...   \n",
       "14  [{'name': 'Aaron R. Halpern'}, {'name': 'Min Y...   \n",
       "15  [{'name': 'Sabina Srokova'}, {'name': 'Paul F ...   \n",
       "16  [{'name': 'Nicholas A Scott'}, {'name': 'Lamiy...   \n",
       "17  [{'name': 'Koen K.A. Van Rompay'}, {'name': 'K...   \n",
       "18  [{'name': 'L. Winchester'}, {'name': 'L. van B...   \n",
       "19  [{'name': 'Julien Rossignol'}, {'name': 'Amani...   \n",
       "20  [{'name': 'Claudio Casola'}, {'name': 'Jingjia...   \n",
       "21  [{'name': 'Meadhbh B Brosnan'}, {'name': 'Nir ...   \n",
       "22  [{'name': 'Leanne de Koning'}, {'name': 'Steph...   \n",
       "23  [{'name': 'Alisa Khramushin'}, {'name': 'Tomer...   \n",
       "24  [{'name': 'Vidya Mangala Prasad'}, {'name': 'D...   \n",
       "25  [{'name': 'Kristian Claesson'}, {'name': 'Yee ...   \n",
       "26  [{'name': 'Alison E. Obr'}, {'name': 'Yung-Jun...   \n",
       "27  [{'name': 'Lucie A. Bergeron'}, {'name': 'Søre...   \n",
       "28  [{'name': 'Athanasios Kousathanas'}, {'name': ...   \n",
       "\n",
       "                                             abstract      posted  \n",
       "0   <p>Single cell sequencing has recently allowed...  2021-09-01  \n",
       "1   <p>We present an ensemble forecast for Wave-3 ...  2021-09-02  \n",
       "2   <p>We report the discovery of a series of benz...  2021-09-03  \n",
       "3   <p>To maintain cellular identities during deve...  2021-08-30  \n",
       "4   <p>In addition to central functions in cell ad...  2021-08-31  \n",
       "5   <p>In most sexually reproducing organisms cros...  2021-08-30  \n",
       "6   <p>Profilin is an essential regulator of actin...  2021-09-02  \n",
       "7   <p>Technological advances have granted the abi...  2021-09-01  \n",
       "8   <p>We present a Deep Learning approach to pred...  2021-09-01  \n",
       "9   <sec><title>Background</title><p>The computer ...  2021-08-30  \n",
       "10  <p>N-linked glycosylation of proteins entering...  2021-09-01  \n",
       "11  <p>Cognitive impairments in psychosis are one ...  2021-09-02  \n",
       "12  <p>Anthropogenic activities and natural proces...  2021-08-31  \n",
       "13  <p>During development, looping of an enhancer ...  2021-09-01  \n",
       "14  <p>Confocal microscopy is an invaluable tool f...  2021-09-04  \n",
       "15  <p>Recent research suggests that episodic memo...  2021-08-31  \n",
       "16  <p>Macrophages are innate immune cells that fi...  2021-09-03  \n",
       "17  <p>Early in the SARS-CoV-2 pandemic, there was...  2021-09-01  \n",
       "18  <p>miRNAs are post-transcriptional regulators ...  2021-08-31  \n",
       "19  <p>Mast cells are key actors of innate immunit...  2021-09-01  \n",
       "20  <sec><title>Background</title><p>The recurrent...  2021-08-31  \n",
       "21  <p>Exposure to enriched environments (EE) thro...  2021-09-01  \n",
       "22  <p>Among the technologies available for protei...  2021-08-31  \n",
       "23  <p>Peptide docking can be perceived as a subpr...  2021-09-03  \n",
       "24  <p>HIV-1 Env mediates viral entry into host ce...  2021-09-01  \n",
       "25  <p>The aggregation of proteins into inclusions...  2021-08-31  \n",
       "26  <p>The acquisition of a metastatic phenotype i...  2021-09-01  \n",
       "27  <p>In the past decade, several studies have es...  2021-08-31  \n",
       "28  <p>Critical illness in COVID-19 is caused by i...  2021-09-02  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "email = \"info@front-matter.io\"\n",
    "rest = mrced2.restApi(email = email)\n",
    "\n",
    "data = []\n",
    "for index, row in cdf.iterrows():\n",
    "    rest.runQuery(row)\n",
    "    if rest.work is not None and date.fromisoformat(rest.work[\"posted\"]) >= start_date:\n",
    "        data.append(rest.work)\n",
    "print(data)\n",
    "    \n",
    "tdf = pd.DataFrame(data, columns=['doi','tweets','archive','subject-area','covid','title','authors','abstract','posted'])\n",
    "tdf.to_csv('1101/preprint_tweets_' + str(date.today().strftime('%Y-%m-%d')) + '.csv')\n",
    "\n",
    "tdf.head(50)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Tweets of bioRxiv and medRxiv preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "29 preprints (including 6 covering SARS-CoV-2, 25 from bioRxiv and 4 from medRxiv) published in the last 7 days before 2021-09-06 had been tweeted at least 3 times (maximum 34)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_rows = tdf['archive'].count()\n",
    "num_covid = tdf['covid'].value_counts(ascending=True)[1]\n",
    "num_biorxiv = tdf['archive'].value_counts(ascending=True)[1]\n",
    "num_medrxiv = tdf['archive'].value_counts(ascending=True)[0]\n",
    "end_date = date.today().strftime('%Y-%m-%d')\n",
    "max_count = tdf['tweets'].max()\n",
    "\n",
    "md('{} preprints (including {} covering SARS-CoV-2, {} from bioRxiv and {} from medRxiv) published in the last 7 days before {} had been tweeted at least 3 times (maximum {}).'.format(num_rows, num_covid, num_biorxiv, num_medrxiv, end_date, max_count))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.6"
  },
  "nteract": {
   "version": "0.28.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
